
<html lang="en"     class="pb-page"  data-request-id="aef1fd8e-d71d-458c-a17c-bee8d4610ea3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2014.57.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/jm500871s;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8" /></meta><meta name="dc.Creator" content="Anqi  Ma" /></meta><meta name="dc.Creator" content="Wenyu  Yu" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Rachel M.  Bleich" /></meta><meta name="dc.Creator" content="J. Martin  Herold" /></meta><meta name="dc.Creator" content="Kyle V.  Butler" /></meta><meta name="dc.Creator" content="Jacqueline L.  Norris" /></meta><meta name="dc.Creator" content="Victoria  Korboukh" /></meta><meta name="dc.Creator" content="Ashutosh  Tripathy" /></meta><meta name="dc.Creator" content="William P.  Janzen" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Stephen V.  Frye" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20). Monomethylation of H4K20 has been implicated in regulating div..." /></meta><meta name="Description" content="The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20). Monomethylation of H4K20 has been implicated in regulating div..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 25, 2014" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm500871s" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2014 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm500871s" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm500871s" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm500871s" /></link>
        
    
    

<title>Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm500871s" /></meta><meta property="og:title" content="Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0012.jpeg" /></meta><meta property="og:description" content="The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20). Monomethylation of H4K20 has been implicated in regulating diverse biological processes including the DNA damage response. In addition to H4K20, SETD8 monomethylates non-histone substrates including proliferating cell nuclear antigen (PCNA) and promotes carcinogenesis by deregulating PCNA expression. However, selective inhibitors of SETD8 are scarce. The only known selective inhibitor of SETD8 to date is nahuoic acid A, a marine natural product, which is competitive with the cofactor. Here, we report the discovery of the first substrate-competitive inhibitor of SETD8, UNC0379 (1). This small-molecule inhibitor is active in multiple biochemical assays. Its affinity to SETD8 was confirmed by ITC (isothermal titration calorimetry) and SPR (surface plasmon resonance) studies. Importantly, compound 1 is selective for SETD8 over 15 other methyltransferases. We also describe structure–activity relationships (SAR) of this series." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm500871s"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm500871s">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm500871s&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm500871s&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm500871s&amp;href=/doi/10.1021/jm500871s" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2014</span><span class="cit-fg-volume">, 57</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 6822-6833</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/57/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm5008566" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm5009242" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anqi++Ma">Anqi Ma</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wenyu++Yu">Wenyu Yu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rachel+M.++Bleich">Rachel M. Bleich</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Martin++Herold">J. Martin Herold</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kyle+V.++Butler">Kyle V. Butler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline+L.++Norris">Jacqueline L. Norris</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Victoria++Korboukh">Victoria Korboukh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ashutosh++Tripathy">Ashutosh Tripathy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=William+P.++Janzen">William P. Janzen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+V.++Frye">Stephen V. Frye</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, <sup>‡</sup>Department of Pharmacology, School of Medicine, <sup>§</sup>Lineberger Comprehensive Cancer Center, and <sup>∥</sup>Department of Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">⊥</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">#</span> <span class="aff-text">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div><div class="corresp-info"><strong>*</strong>Phone: 919-843-8459. Fax: 919-843-8465. E-mail: <a href="/cdn-cgi/l/email-protection#f3999a929d999a9db3869d90dd969786"><span class="__cf_email__" data-cfemail="5f35363e313536311f2a313c713a3b2a">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm500871s&amp;href=/doi/10.1021%2Fjm500871s" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2014</span></span><span class="cit-volume">, 57</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 6822–6833</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 17, 2014</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 June 2014</li><li><span class="item_label"><b>Published</b> online</span>25 July 2014</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 August 2014</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm500871s" title="DOI URL">https://doi.org/10.1021/jm500871s</a></div><div class="article_header-article-copyright"><strong>Copyright © 2014 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm500871s"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3668</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">51</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm500871s" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Anqi&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Wenyu&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;M. Bleich&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Martin Herold&quot;},{&quot;first_name&quot;:&quot;Kyle&quot;,&quot;last_name&quot;:&quot;V. Butler&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;L. Norris&quot;},{&quot;first_name&quot;:&quot;Victoria&quot;,&quot;last_name&quot;:&quot;Korboukh&quot;},{&quot;first_name&quot;:&quot;Ashutosh&quot;,&quot;last_name&quot;:&quot;Tripathy&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;P. Janzen&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;V. Frye&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2014&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;57&quot;,&quot;pages&quot;:&quot;6822-6833&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm500871s&quot;},&quot;abstract&quot;:&quot;The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20). Monomethylation of H4K20 has been implicated in regulating diverse biological processes including the DNA damage response. In addition to H4K20, SETD8 monomethylates non-histone substrates including proliferating cell nuclear antigen (PCNA) and promotes carcinogenesis by deregulating PCNA expression. However, selective inhibitors of SETD8 are scarce. The only known selective inhibitor of SETD8 to date is nahuoic acid A, a marine natural product, which is competitive with the cofactor. Here, we report the discovery of the first substrate-competitive inhibitor of SETD8, UNC0379 (1). This small-molecule inhibitor is active in multiple biochemical assays. Its affinity to SETD8 was confirmed by ITC (isothermal titration calorimetry) and SPR (surface plasmon resonance) studies. Importantly, compound 1 is selective for SETD8 over 15 other methyltransferases. We also describe structur&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500871s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500871s" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500871s&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500871s" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500871s&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500871s" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm500871s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500871s&amp;href=/doi/10.1021/jm500871s" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm500871s" /></input><a href="/doi/pdf/10.1021/jm500871s" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (1 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm500871s%26sid%3Dliteratum%253Aachs%26pmid%3D25032507%26genre%3Darticle%26aulast%3DMa%26date%3D2014%26atitle%3DDiscovery%2Bof%2Ba%2BSelective%252C%2BSubstrate-Competitive%2BInhibitor%2Bof%2Bthe%2BLysine%2BMethyltransferase%2BSETD8%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D57%26issue%3D15%26spage%3D6822%26epage%3D6833%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291301" title="Salts">Salts</a>,</li><li><a href="/action/doSearch?ConceptID=291011" title="High-performance liquid chromatography">High-performance liquid chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/57/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jmcmar.2014.57.issue-15/production/jmcmar.2014.57.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20). Monomethylation of H4K20 has been implicated in regulating diverse biological processes including the DNA damage response. In addition to H4K20, SETD8 monomethylates non-histone substrates including proliferating cell nuclear antigen (PCNA) and promotes carcinogenesis by deregulating PCNA expression. However, selective inhibitors of SETD8 are scarce. The only known selective inhibitor of SETD8 to date is nahuoic acid A, a marine natural product, which is competitive with the cofactor. Here, we report the discovery of the first substrate-competitive inhibitor of SETD8, UNC0379 (<b>1</b>). This small-molecule inhibitor is active in multiple biochemical assays. Its affinity to SETD8 was confirmed by ITC (isothermal titration calorimetry) and SPR (surface plasmon resonance) studies. Importantly, compound <b>1</b> is selective for SETD8 over 15 other methyltransferases. We also describe structure–activity relationships (SAR) of this series.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11358" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11358" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein lysine methyltransferases (PKMTs, also known as histone lysine methyltransferases (HKMTs)) catalyze the transfer of the methyl group from the cofactor <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to lysine residues of histone and non-histone substrates, leading to lysine mono-, di-, and/or trimethylation.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Histone lysine methylation has been increasingly recognized as a major epigenetic gene regulation mechanism in eukaryotic cells.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6">(1-6)</a> Therefore, PKMTs as a class of potential drug targets have received considerable attention from the medicinal chemistry and chemical biology community. With the exception of DOT1L, PKMTs contain an evolutionarily conserved SET (<i>Su(var)</i>, <i>E(z)</i>, and <i>Trithorax</i>) domain.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> This catalytic domain consists of a substrate binding groove and a cofactor binding site.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> A number of selective small-molecule inhibitors of PKMTs including G9a/GLP, EZH2, SMYD2, DOT1L, and SETD2 have been discovered.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34 ref35">(9-35)</a> These inhibitors are competitive with either the peptide substrate or the cofactor SAM. In addition, a number of selective inhibitors of protein arginine methyltransferases (PRMTs), another class of protein methyltransferases, have been reported.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36-38)</a> Some of these inhibitors have subsequently been utilized as valuable tools for investigating the role of the corresponding methyltransferases in various human diseases.<a onclick="showRef(event, 'ref16 ref19 ref20 ref28 ref29 ref33 ref39 ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref16 ref19 ref20 ref28 ref29 ref33 ref39 ref40 ref41 ref42">(16, 19, 20, 28, 29, 33, 39-42)</a></div><div class="NLM_p">SETD8 (also known as SET8, PR-SET7, or KMT5A (lysine methyltransferase 5A)) is the sole methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20).<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43-45)</a> SETD8 and H4K20me (H4K20 monomethylation) have been implicated in regulating a diverse set of biological processes including the DNA damage response, DNA replication, and mitotic condensation.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> A recent study has shown that (1) SETD8 is physically associated with TWIST, a master regulator of EMT (epithelial–mesenchymal transition), (2) SETD8 and TWIST are functionally interdependent on promoting EMT, (3) SETD8 acts on the promoters of the TWIST target genes such as <i>N-cadherin</i> via exertion of its H4K20 monomethylation activity, and (4) SETD8 expression is positively correlated with metastasis and the expression of TWIST and <i>N-cadherin</i> in breast cancer cells.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In addition to H4K20, SETD8 methylates many non-histone substrates including the tumor suppressor p53 and proliferating cell nuclear antigen (PCNA).<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47, 48)</a> The monomethylation of p53 at lysine 382 (p53K382me1) catalyzed by SETD8 suppresses p53-mediated transcription activation of highly responsive target genes.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> SETD8 and PCNA are coexpressed in lung cancer tissues.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The monomethylation of PCNA at lysine 248 (PCNAK248me1) catalyzed by SETD8 stabilizes PCNA protein, enhances the interaction between PCNA and the flap endonuclease FEN1, and promotes the proliferation of cancer cells.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div><div class="NLM_p">However, selective inhibitors of SETD8 are scarce. To date, nahuoic acid A, a marine natural product, is the only known selective inhibitor of SETD8 (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This inhibitor is competitive with the cofactor SAM and noncompetitive with the peptide substrate. Here we report the discovery of UNC0379 (<b>1</b>), the first substrate-competitive inhibitor of SETD8. Compound <b>1</b> is a synthetic small-molecule inhibitor that displays inhibitory activity in multiple biochemical assays and is selective for SETD8 over 15 other methyltransferases. The binding affinity of compound <b>1</b> to SETD8 was determined using biophysical assays such as ITC (isothermal titration calorimetry) and SPR (surface plasmon resonance) and is largely consistent with its potency in biochemical assays. We describe hit identification, analogue synthesis, structure–activity relationship (SAR) findings, and comprehensive characterization of compound <b>1</b> in a number of biochemical and biophysical assays including mechanism of action and selectivity studies.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0014.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of the known SETD8 inhibitor nahuoic acid A.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31058" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31058" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Discovery of Compound <b>1</b> as a SETD8 Inhibitor</h3><div class="NLM_p">We previously reported that 2,4-diaminoquinazolines are selective, substrate-competitive inhibitors of the lysine methyltransferases G9a and GLP.<a onclick="showRef(event, 'ref10 ref12 ref13 ref14 ref30'); return false;" href="javascript:void(0);" class="ref ref10 ref12 ref13 ref14 ref30">(10, 12-14, 30)</a> To identify a substrate-competitive inhibitor of SETD8, we cross-screened our quinazoline-based inhibitor set, which consists of >150 compounds, against SETD8. From this study, we discovered compound <b>1</b> as an inhibitor of SETD8 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Interestingly, compound <b>1</b> was originally prepared for targeting L3MBTL1, a methyllysine reader protein,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> but showed no appreciable activity for L3MBTL1. On the other hand, compound <b>1</b> displayed inhibitory activity with an IC<sub>50</sub> of 7.3 ± 1.0 μM (<i>n</i> = 2) in a radioactive biochemical assay that measures the transfer of the tritiated methyl group from <sup>3</sup>H-SAM to a peptide substrate catalyzed by SETD8 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). The inhibitory activity of compound <b>1</b> was confirmed in an orthogonal biochemical assay, microfluidic capillary electrophoresis (MCE) assay. This SETD8 MCE assay was developed analogously to the previously reported G9a MCE assay.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Compound <b>1</b> exhibited an IC<sub>50</sub> of 9.0 μM in the SETD8 MCE assay.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>1</b> was identified as an inhibitor of SETD8 by cross-screening a quinazoline-based inhibitor set. (A) Structure of compound <b>1</b>. (B) Concentration–response curve of compound <b>1</b> in the SETD8 radioactive methyl transfer assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Analogue Synthesis</h3><div class="NLM_p">To determine SAR for this promising hit, we designed and synthesized a number of analogues that contain various 2- and 4-substituents at the quinazoline core. We synthesized compounds <b>1</b>–<b>24</b> from commercially available 2,4-dichloro-6,7-dimethoxyquinazoline and corresponding amines in good yields (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a> and Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Using the methods developed previously,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> we displaced the 4-chloro group with the first set of amines at room temperature and the 2-chloro group with the second set of amines under microwave heating conditions to yield the desired 2,4-diamino-6,7-dimethoxyquinazolines.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Typical Synthesis of 2,4-Diamino-6,7-dimethoxyquinazolines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) R<sup>1</sup> amines, THF, <i>N</i>,<i>N</i>-diisopropylethylamine, room temperature; (b) R<sup>2</sup> amines, <i>n</i>-BuOH, DIPEA, microwave, 160 °C.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> SAR in SETD8 Biochemical Assay</h3><div class="NLM_p">The synthesized compounds were then evaluated in the SETD8 radioactive methyl transfer assay. IC<sub>50</sub> values of these compounds in this biochemical assay are summarized in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the 4-Amino Group</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0008.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0009.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed in duplicate.</p></div></div><div></div></div><div class="NLM_p">We first explored the 4-amino group of the quinazoline scaffold (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Reducing the length of the alkyl linker between the two amino groups resulted in the decrease of the potency (compound <b>1</b> versus compounds <b>2</b>–<b>4</b>). The distal amino group can be changed from the pyrrolidinyl to piperidinyl or dimethylamino without any potency loss (compound <b>1</b> versus compounds <b>5</b> and <b>6</b>). Interestingly, the diethylamino analogue (compound <b>7</b>) was somewhat less potent than compounds <b>1</b>, <b>5</b>, and <b>6</b>. We also attempted to replace the piperidinyl group with a piperazinyl or N-substituted piperazinyl group (compound <b>1</b> versus compounds <b>8</b>–<b>12</b>). However, these structural modifications led to a potency loss, suggesting that an additional basic amino group (compounds <b>9</b>–<b>12</b>) or a large substituent (compound <b>8</b>) is detrimental to inhibiting SETD8. In addition, we found that the basicity of the pyrrolidinyl group in compound <b>1</b> was an important contributor to its SETD8 inhibitory activity, as the corresponding amide analogue (compound <b>13</b>) was about 8-fold less potent than compound <b>1</b>. We also explored whether the NH group in compound <b>1</b> was potentially engaged in a hydrogen bond interaction and found that the <i>N</i>-methyl analogue (compound <b>14</b>) was drastically less potent than compound <b>1</b>, suggesting that the hydrogen of the secondary amine likely serves as a hydrogen bond donor.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the 2-Amino Group</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0010.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0011.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed in duplicate.</p></div></div><div></div></div><div class="NLM_p last">We next explored the 2-amino group of the quinazoline scaffold (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The replacement of the pyrrolidinyl group with either the piperidinyl or azepanyl group resulted in a significant loss of the potency (compound <b>1</b> versus compounds <b>15</b> and <b>16</b>), suggesting a large group is disfavored. On the other hand, the dimethylamino group (compound <b>17</b>) did not lead to any significant potency loss. The introduction of an additional basic amino group (compound <b>15</b> versus the unsubstituted piperazinyl analogue <b>18</b> and the <i>N</i>-methylpiperazinyl analogue <b>19</b>) resulted in a complete loss of the potency. We also explored other amino groups such as <i>N</i>-methyl-<i>N</i>-cyclopentylamine (compound <b>20</b>) and <i>N</i>-methyl-<i>N</i>-cyclohexylamine (compound <b>21</b>). Compared with compound <b>1</b>, compounds <b>20</b> and <b>21</b> were about 4-fold less potent. Interestingly, removing the methyl group from either compound <b>20</b> or <b>21</b> led to the complete loss of inhibitory activity (compounds <b>22</b> and <b>23</b>), suggesting that tertiary amino groups are preferred compared with secondary amino groups. In addition, the chloro analogue (compound <b>24</b>) was inactive. Taken together, these results suggest that SETD8 inhibitory activity is very sensitive to the 2-substituent.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> ITC and SPR Studies</h3><div class="NLM_p">We next characterized compound <b>1</b> in biophysical assays. In ITC studies, compound <b>1</b> bound SETD8 with a <i>K</i><sub>D</sub> of 18.3 ± 3.2 μM (<i>n</i> = 3) (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). In SPR studies, compound <b>1</b> behaved as a classic reversible inhibitor with a fast on rate (<i>k</i><sub>a</sub> = 2.18 ± 0.36 × 10<sup>4</sup> 1/Ms) and a fast off rate (<i>k</i><sub>d</sub> = 7.82 ± 0.87 × 10<sup>–1</sup> 1/s) (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). The <i>K</i><sub>D</sub> of compound <b>1</b> was determined to be 36.0 ± 2.3 μM (<i>n</i> = 3). The binding affinity of compound <b>1</b> to SETD8 determined by ITC and SPR is largely consistent with its potency in the biochemical assays.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>1</b> binds SETD8 with a <i>K</i><sub>D</sub> of 18.3 ± 3.2 μM (<i>n</i> = 3) in ITC studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>1</b> exhibits rapid on and off rates in SPR studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> MOA Studies</h3><div class="NLM_p">We next studied the MOA (mechanism of action) of the SETD8 inhibition by compound <b>1</b> via varying concentrations of the H4 peptide substrate or the cofactor SAM. As illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A, IC<sub>50</sub> values of compound <b>1</b> increased linearly with H4 peptide concentrations. On the other hand, IC<sub>50</sub> values of compound <b>1</b> remained constant in the presence of increasing concentrations of SAM (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). These results indicate that compound <b>1</b> is competitive with the peptide substrate and noncompetitive with the cofactor SAM.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MOA studies of compound <b>1</b>. (A) Compound <b>1</b> is competitive with the peptide substrate, as its IC<sub>50</sub> values increased linearly with H4 peptide concentrations. (B) Compound <b>1</b> is noncompetitive with the cofactor SAM, as its IC<sub>50</sub> values remained constant in the presence of increasing concentrations of SAM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Peptide Displacement Assay</h3><div class="NLM_p">To confirm the findings from the MOA studies, we tested compound <b>1</b> and an inactive control (compound <b>14</b>) in a peptide displacement assay using fluorescence polarization (FP), which measures effects of inhibitors on displacing the H4K20me (1–24) peptide with N-terminus labeled by fluorescein isothiocyanate (FITC). As illustrated in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, compound <b>1</b> effectively displaced the FITC-labeled peptide binding to SETD8 with an IC<sub>50</sub> of 37.7 ± 7.2 μM (<i>n</i> = 2). On the other hand, our negative control (compound <b>14</b>) did not displace the peptide in this FP assay. These results further support that compound <b>1</b> is competitive with the peptide substrate.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Peptide displacement assay. Compound <b>1</b> effectively displaced the FITC-labeled H4K20me (1–14) peptide, while an inactive control (compound <b>14</b>) did not.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Selectivity</h3><div class="NLM_p">We next determined the selectivity of inhibitor <b>1</b> for SETD8 over 15 other methyltransferases (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). With the exception of PRC2 (polycomb repressive complex 2), the IC<sub>50</sub> values of compound <b>1</b> for 14 other methyltransferases including G9a and GLP were all above 100 μM. For PRC2, compound <b>1</b> was active only at the two highest concentrations (50 and 100 μM) with an estimated IC<sub>50</sub> value greater than 50 μM. Therefore, inhibitor <b>1</b> is a selective inhibitor of SETD8.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Selectivity of inhibitor <b>1</b> versus 15 other methyltransferases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54540" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54540" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We discovered the first selective, substrate competitive inhibitor of SETD8 by cross-screening our quinazoline-based epigenetic library. Our SAR studies of this series reveal that while the 4-amino moiety can be modified without a significant loss of potency, modifications to the 2-amino moiety are not well tolerated. We characterized compound <b>1</b> in a battery of biochemical, biophysical, MOA, and selectivity assays and found that the SETD8 inhibitory activity of this compound in biochemical assays was largely consistent with its binding affinity to SETD8 determined by ITC and SPR. Our MOA studies indicate that this inhibitor is competitive with the peptide substrate and noncompetitive with the cofactor SAM. This MOA was further supported by our finding that compound <b>1</b> effectively displaced the FITC-labeled H4 peptide in a FP assay. Importantly, this inhibitor is selective for SETD8 over 15 other methyltransferases including G9a and GLP, the two PKMTs that are known for being potently inhibited by quinazolines. These results provide the first evidence that 2,4-diaminoquinazolines can be modified to yield selective, substrate competitive inhibitors of PKMTs other than G9a and GLP.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry General Procedures</h3><div class="NLM_p last">HPLC spectra for all compounds were acquired using an Agilent 6110 series system with UV detector set to 254 nm. Samples were injected (5 μL) onto an Agilent Eclipse Plus 4.6 mm × 50 mm, 1.8 μm C18 column at room temperature. A linear gradient from 10% to 100% B (MeOH + 0.1% acetic acid) in 5.0 min was followed by pumping 100% B for another 2 min with A being H<sub>2</sub>O + 0.1% acetic acid. The flow rate was 1.0 mL/min. Mass spectra data were acquired in positive ion mode using an Agilent 6110 single quadrupole mass spectrometer with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were recorded at Varian Mercury spectrometer with 400 MHz for proton (<sup>1</sup>H NMR) and 100 MHz for carbon (<sup>13</sup>C NMR). Chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with UV detector set to 254 nm. Samples were injected onto a Phenomenex Luna 75 mm × 30 mm, 5 μm C<sub>18</sub> column at room temperature. The flow rate was 30 mL/min. A linear gradient was used with 10% (or 50%) of MeOH (A) in 0.1% TFA in H<sub>2</sub>O (B) to 100% of MeOH (A). HPLC was used to establish the purity of target compounds. All compounds had >95% purity using the HPLC methods described above. High-resolution (positive ion) mass spectrometry (HRMS) for compound <b>1</b> was performed using a Thermo LTqFT mass spectrometer under FT control at 100 000 resolution.</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 6,7-Dimethoxy-2-(pyrrolidin-1-yl)-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>1</b>)</h3><div class="NLM_p last">To a solution of 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 84 mg, 0.17 mmol) and <i>n</i>-butanol (1.5 mL) were added pyrrolidine (commercially available, 56 μL, 0.68 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (89 μL, 0.51 mmol). The resulting solution was stirred inside a microwave at 160 °C for 30 min. After the mixture was cooled, TLC indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was redissolved in CH<sub>2</sub>Cl<sub>2</sub> and washed with brine. The organic layer was dried, concentrated, and purified by HPLC to give the title compound as a TFA salt, white solid (70 mg, yield 64%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.56 (s, 1H), 7.10 (s, 1H), 3.95 (s, 3H), 3.92 (s, 3H), 3.78–3.57 (m, 8H), 3.22–3.16 (m, 2H), 3.10–3.01 (m, 2H), 2.19–1.97 (m, 8H), 1.86–1.73 (m, 4H), 1.55–1.47 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.40, 157.17, 151.52, 148.80, 136.95, 105.04, 103.62, 99.62, 56.93, 56.79, 56.11, 55.08 (four carbons), 42.48, 29.31, 26.87, 25.20, 23.95 (four carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.32 min. MS (ESI): 414 [M + H]<sup>+</sup>. HRMS calcd for C<sub>23</sub>H<sub>35</sub>N<sub>5</sub>O<sub>2</sub> + H, 414.2869; found, 414.2862 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 6,7-Dimethoxy-2-(pyrrolidin-1-yl)-<i>N</i>-(2-(pyrrolidin-1-yl)ethyl)quinazolin-4-amine (<b>2</b>)</h3><div class="NLM_p last">2-Chloro-6,7-dimethoxy-<i>N</i>-(2-(pyrrolidin-1-yl)ethyl)quinazolin-4-amine was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, 1-(2-aminoethyl)pyrrolidine (commercially available), <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. To a solution of 2-chloro-6,7-dimethoxy-<i>N</i>-(2-(pyrrolidin-1-yl)ethyl)quinazolin-4-amine (72 mg, 0.13 mmol) and isopropanol (0.7 mL) were added pyrrolidine (21 μL, 0.26 mmol) and HCl in dioxane (4.0 M, 63 μL, 0.26 mmol). The resulting solution was stirred inside a microwave at 160 °C for 20 min. After the mixture was cooled, TLC indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was redissolved in CH<sub>2</sub>Cl<sub>2</sub>, washed with brine. The organic layer was dried, concentrated, and purified by HPLC to give the title compound <b>2</b> as a TFA salt, white solid (56 mg, yield 73%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.50 (s, 1H), 7.12 (s, 1H), 4.06 (t, <i>J</i> = 6.0 Hz, 2H), 3.96 (s, 3H), 3.92 (s, 3H), 3.82–3.71 (m, 4H), 3.68–3.56 (m, 4H), 3.22–3.10 (m, 2H), 2.22–1.99 (m, 8H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 2.88 min. MS (ESI): 372 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 6,7-Dimethoxy-2-(pyrrolidin-1-yl)-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)quinazolin-4-amine (<b>3</b>)</h3><div class="NLM_p last">2-Chloro-6,7-dimethoxy-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)quinazolin-4-amine was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, 1-(3-aminopropyl)pyrrolidine (commercially available), <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>3</b> was prepared according to the procedure for making <b>2</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(3-(pyrrolidin-1-yl)propyl)quinazolin-4-amine (66 mg, 0.19 mmol), pyrrolidine (30 μL, 0.37 mmol), HCl in dioxane (4.0 M, 93 μL, 0.37 mmol), and isopropanol (1.0 mL). The title compound <b>3</b> was obtained as a TFA salt, brown solid (45 mg, yield 40%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.53 (s, 1H), 7.10 (s, 1H), 3.95 (s, 3H), 3.91 (s, 3H), 3.83–3.60 (m, 8H), 3.35–3.28 (m, 2H), 3.13–3.02 (m, 2H), 2.25–2.00 (m, 10H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.68, 157.34, 151.49, 148.83, 137.08, 105.03, 103.57, 99.57, 56.91, 56.82, 55.11 (four carbons), 53.95, 39.75, 26.51, 23.96 (four carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.24 min. MS (ESI): 386 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 6,7-Dimethoxy-2-(pyrrolidin-1-yl)-<i>N</i>-(4-(pyrrolidin-1-yl)butyl)quinazolin-4-amine (<b>4</b>)</h3><div class="NLM_p last">2-Chloro-6,7-dimethoxy-<i>N</i>-(4-(pyrrolidin-1-yl)butyl)quinazolin-4-amine was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, 1-pyrrolidinebutanamine (commercially available), <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>4</b> was prepared according to the procedure for making <b>1</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(4-(pyrrolidin-1-yl)butyl)quinazolin-4-amine (109 mg, 0.30 mmol), pyrrolidine (99 μL, 1.2 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (105 μL, 0.60 mmol), and <i>n</i>-butanol (1.0 mL). The title compound <b>4</b> was obtained as a TFA salt, white solid (144 mg, yield 77%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.54 (s, 1H), 7.09 (s, 1H), 3.94 (s, 3H), 3.91 (s, 3H), 3.79–3.54 (m, 8H), 3.28–3.19 (m, 2H), 3.12–3.01 (m, 2H), 2.19–1.97 (m, 8H), 1.89–1.79 (m, 4H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.02 min. MS (ESI): 400 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 6,7-Dimethoxy-<i>N</i>-(5-(piperidin-1-yl)pentyl)-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>5</b>)</h3><div class="NLM_p last">2-Chloro-6,7-dimethoxy-<i>N</i>-(5-(piperidin-1-yl)pentyl)quinazolin-4-amine was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, 1-piperidinepentanamine (commercially available), <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>5</b> was prepared according to the procedure for making <b>2</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(piperidin-1-yl)pentyl)quinazolin-4-amine (79 mg, 0.13 mmol), pyrrolidine (21 μL, 0.26 mmol), HCl in dioxane (4.0 M, 63 μL, 0.26 mmol), and isopropanol (0.7 mL). The title compound <b>5</b> was obtained as a TFA salt, brown solid (58 mg, yield 68%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.55 (s, 1H), 7.09 (s, 1H), 3.95 (s, 3H), 3.91 (s, 3H), 3.79–3.57 (m, 6H), 3.56–3.47 (m, 2H), 3.12–3.05 (m, 2H), 2.90 (td, <i>J</i> = 12.5, 2.7 Hz, 2H), 2.22–2.01 (m, 4H), 1.97–1.89 (m, 2H), 1.88–1.70 (m, 7H), 1.58–1.43 (m, 3H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.40, 157.17, 151.53, 148.81, 136.96, 105.05, 103.62, 99.62, 58.12, 56.93, 56.79, 54.27 (four carbons), 42.50, 29.32, 25.29, 24.92, 24.24 (four carbons), 22.72. HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.26 min. MS (ESI): 428 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>N</i><sup>1</sup>-(6,7-Dimethoxy-2-(pyrrolidin-1-yl)quinazolin-4-yl)-<i>N</i><sup>5</sup>,<i>N</i><sup>5</sup>-dimethylpentane-1,5-diamine (<b>6</b>)</h3><div class="NLM_p last"><i>N</i><sup>1</sup>-(2-Chloro-6,7-dimethoxyquinazolin-4-yl)-<i>N</i><sup>5</sup>,<i>N</i><sup>5</sup>-dimethylpentane-1,5-diamine was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, 5-(dimethylamino)amylamine (commercially available), <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>6</b> was prepared according to the procedure for making <b>2</b> from <i>N</i><sup>1</sup>-(2-chloro-6,7-dimethoxyquinazolin-4-yl)-<i>N</i><sup>5</sup>,<i>N</i><sup>5</sup>-dimethylpentane-1,5-diamine (70 mg, 0.15 mmol), pyrrolidine (21 μL, 0.30 mmol), HCl in dioxane (4.0 M, 75 μL, 0.30 mmol), and isopropanol (1.0 mL). The title compound <b>6</b> was obtained as a TFA salt, white solid (57 mg, yield 62%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.55 (s, 1H), 7.09 (s, 1H), 3.95 (s, 3H), 3.91 (s, 3H), 3.78–3.54 (m, 6H), 3.16–3.09 (m, 2H), 2.87 (s, 6H), 2.22–1.98 (m, 4H), 1.89–1.72 (m, 4H), 1.58–1.39 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.41, 157.19, 151.53, 148.82, 136.96, 105.05, 103.62, 99.62, 58.87, 56.93, 56.79, 43.37 (four carbons), 42.48, 29.32 (two carbons), 25.47 (two carbons), 25.06. HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.02 min. MS (ESI): 388 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <i>N</i><sup>1</sup>-(6,7-Dimethoxy-2-(pyrrolidin-1-yl)quinazolin-4-yl)-<i>N</i><sup>5</sup>,<i>N</i><sup>5</sup>-diethylpentane-1,5-diamine (<b>7</b>)</h3><div class="NLM_p last"><i>N</i><sup>1</sup>-(2-Chloro-6,7-dimethoxyquinazolin-4-yl)-<i>N</i><sup>5</sup>,<i>N</i><sup>5</sup>-diethylpentane-1,5-diamine was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, 5-(diethylamino)pentylamine (commercially available), <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>7</b> was prepared according to the procedure for making <b>1</b> from <i>N</i><sup>1</sup>-(2-chloro-6,7-dimethoxyquinazolin-4-yl)-<i>N</i><sup>5</sup>,<i>N</i><sup>5</sup>-diethylpentane-1,5-diamine (62 mg, 0.16 mmol), pyrrolidine (54 μL, 0.64 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (113 μL, 0.64 mmol), and <i>n</i>-butanol (0.2 mL). The title compound <b>7</b> was obtained as a TFA salt, amber solid (79 mg, yield 76%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.53 (s, 1H), 7.07 (s, 1H), 3.93 (s, 3H), 3.90 (s, 3H), 3.77–3.54 (m, 6H), 3.22 (q, <i>J</i> = 7.4 Hz, 4H), 3.16–3.09 (m, 2H), 2.21–2.01 (m, 4H), 1.87–1.72 (m, 4H), 1.57–1.45 (m, 2H), 1.30 (t, <i>J</i> = 7.3 Hz, 6H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.54 min. MS (ESI): 416 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>tert</i>-Butyl 4-(5-((6,7-Dimethoxy-2-(pyrrolidin-1-yl)quinazolin-4-yl)amino)pentyl)piperazine-1-carboxylate (<b>8</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl 4-(5-((2-chloro-6,7-dimethoxyquinazolin-4-yl)amino)pentyl)piperazine-1-carboxylate was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, <i>tert</i>-butyl 4-(5-aminopentyl)piperazine-1-carboxylate (commercially available), <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>8</b> was prepared according to the procedure for making <b>1</b> from <i>tert</i>-butyl 4-(5-((2-chloro-6,7-dimethoxyquinazolin-4-yl)amino)pentyl)piperazine-1-carboxylate (200 mg, 0.40 mmol), pyrrolidine (135 μL, 1.6 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (280 μL, 1.6 mmol), and <i>n</i>-butanol (0.5 mL). The title compound <b>8</b> was obtained as a TFA salt, yellow solid (180 mg, yield 60%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.54 (s, 1H), 7.09 (s, 1H), 4.33–4.08 (m, 2H), 3.95 (s, 3H), 3.91 (s, 3H), 3.77–3.50 (m, 8H), 3.29–3.12 (m, 4H), 3.10–2.90 (m, 2H), 2.21–2.01 (m, 4H), 1.87–1.77 (m, 4H), 1.55–1.49 (m, 2H), 1.47 (s, 9H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.98 min. MS (ESI): 529 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 6,7-Dimethoxy-<i>N</i>-(5-(piperazin-1-yl)pentyl)-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>9</b>)</h3><div class="NLM_p last">To the solution of <i>tert</i>-butyl 4-(5-((6,7-dimethoxy-2-(pyrrolidin-1-yl)quinazolin-4-yl)amino)pentyl)piperazine-1-carboxylate (<b>8</b>) in MeOH and THF was added TFA at room temperature and stirred overnight at 50 °C. LC–MS indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was purified by HPLC to give 14 mg of the title compound <b>9</b> as a TFA salt, white solid (14 mg, yield 80%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.56 (s, 1H), 7.11 (s, 1H), 3.96 (s, 3H), 3.92 (s, 3H), 3.82–3.49 (m, 14H), 3.23–3.17 (m, 2H), 2.21–2.02 (m, 4H), 1.87–1.78 (m, 4H), 1.56–1.47 (m, 2H). HPLC purity: 95%; <i>t</i><sub>R</sub> = 3.50 min. MS (ESI): 429 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 6,7-Dimethoxy-<i>N</i>-(5-(4-methylpiperazin-1-yl)pentyl)-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>10</b>)</h3><div class="NLM_p last">To the solution of 6,7-dimethoxy-<i>N</i>-(5-(piperazin-1-yl)pentyl)-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>9</b>, 80 mg, 0.12 mmol) in MeOH (1.0 mL) were added formaldehyde (commercially available, 39 μL, 1.4 mmol), acetic acid (80 μL, 1.4 mmol), and sodium cyanoborohydride (44 mg, 0.7 mmol) at 0 °C. The resulting mixture was warmed to room temperature and stirred overnight. LC–MS indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was purified by HPLC to give the title compound <b>10</b> as a TFA salt, gray solid (25 mg, yield 31%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.56 (s, 1H), 7.10 (s, 1H), 3.96 (s, 3H), 3.92 (s, 3H), 3.81–3.53 (m, 14H), 3.26–3.19 (m, 2H), 2.97 (s, 3H), 2.22–2.02 (m, 4H), 1.88–1.77 (m, 4H), 1.56–1.46 (m, 2H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.59 min. MS (ESI): 443 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>N</i>-(5-(4-Ethylpiperazin-1-yl)pentyl)-6,7-dimethoxy-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>11</b>)</h3><div class="NLM_p last">Compound <b>11</b> was prepared according to the procedure for making <b>10</b> from 6,7-dimethoxy-<i>N</i>-(5-(piperazin-1-yl)pentyl)-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>9</b>, 80 mg, 0.12 mmol), acetaldehyde (commercially available, 56 μL, 1.4 mmol), acetic acid (80 μL, 1.4 mmol) and sodium cyanoborohydride (44 mg, 0.7 mmol) and MeOH (1.0 mL). The title compound <b>11</b> was obtained as a TFA salt, gray solid (74 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.55 (s, 1H), 7.10 (s, 1H), 3.95 (s, 3H), 3.92 (s, 3H), 3.81–3.55 (m, 14H), 3.35–3.26 (m, 2H), 3.26–3.20 (m, 2H), 2.18–2.02 (m, 4H), 1.88–1.78 (m, 4H), 1.56–1.48 (m, 2H), 1.37 (t, <i>J</i> = 7.3 Hz, 3H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.60 min. MS (ESI): 457 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>N</i>-(5-(4-Isopropylpiperazin-1-yl)pentyl)-6,7-dimethoxy-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>12</b>)</h3><div class="NLM_p last">Compound <b>12</b> was prepared according to the procedure for making <b>10</b> from 6,7-dimethoxy-<i>N</i>-(5-(piperazin-1-yl)pentyl)-2-(pyrrolidin-1-yl)quinazolin-4-amine (<b>9</b>, 80 mg, 0.12 mmol), acetone (commercially available, 103 μL, 1.4 mmol), acetic acid (80 μL, 1.4 mmol) and sodium cyanoborohydride (44 mg, 0.7 mmol) and MeOH (1.0 mL). The title compound <b>12</b> was obtained as a TFA salt, brown solid (61 mg, yield 73%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.56 (s, 1H), 7.10 (s, 1H), 3.96 (s, 3H), 3.92 (s, 3H), 3.82–3.48 (m, 15H), 3.24–3.18 (m, 2H), 2.20–2.03 (m, 4H), 1.88–1.78 (m, 4H), 1.56–1.47 (m, 2H), 1.40 (s, 3H), 1.38 (s, 3H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.61 min. MS (ESI): 471 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 5-((6,7-Dimethoxy-2-(pyrrolidin-1-yl)quinazolin-4-yl)amino)-1-(pyrrolidin-1-yl)pentan-1-one (<b>13</b>)</h3><div class="NLM_p">5-((<i>tert</i>-Butoxycarbonyl)amino)pentanoic acid (commercially available, 0.93 g, 4.3 mmol) and pyrrolidine (238 μL, 2.9 mmol) were dissolved in 23 mL of DMF. To this solution were added <i>N</i>,<i>N</i>-diisopropylethylamine (2.5 mL, 14.5 mmol) and HATU (2.18 g, 5.8 mmol). The resulting solution was stirred at 50 °C for 3 h. After the mixture was cooled, TLC indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was redissolved in CH<sub>2</sub>Cl<sub>2</sub>, washed with brine. The organic layer was dried, concentrated, and purified by ISCO to give <i>tert</i>-butyl (5-oxo-5-(pyrrolidin-1-yl)pentyl)carbamate. To the solution of <i>tert</i>-butyl (5-oxo-5-(pyrrolidin-1-yl)pentyl)carbamate in MeOH and THF was added TFA at room temperature and stirred overnight at room temperature. LC–MS indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the product 5-amino-1-(pyrrolidin-1-yl)pentan-1-one obtained was used for the next step without further purification.</div><div class="NLM_p last">5-((2-Chloro-6,7-dimethoxyquinazolin-4-yl)amino)-1-(pyrrolidin-1-yl)pentan-1-one was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, 5-amino-1-(pyrrolidin-1-yl)pentan-1-one, <i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>13</b> was prepared according to the procedure for making <b>1</b> from 5-((2-chloro-6,7-dimethoxyquinazolin-4-yl)amino)-1-(pyrrolidin-1-yl)pentan-1-one (87 mg, 0.22 mmol), pyrrolidine (74 μL, 0.88 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (154 μL, 0.88 mmol), and <i>n</i>-butanol (0.25 mL). The title compound <b>13</b> was obtained as a TFA salt, brown solid (16 mg, yield 13%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.57 (s, 1H), 7.09 (s, 1H), 3.96 (s, 3H), 3.93 (s, 3H), 3.80–3.59 (m, 6H), 3.48 (t, <i>J</i> = 6.8 Hz, 2H), 3.40 (t, <i>J</i> = 6.9 Hz, 2H), 2.41 (t, <i>J</i> = 7.0 Hz, 2H), 2.20–2.03 (m, 4H), 2.01–1.93 (m, 2H), 1.91–1.84 (m, 2H), 1.83–1.71 (m, 4H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 4.58 min. MS (ESI): 428 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 6,7-Dimethoxy-<i>N</i>-methyl-2-(pyrrolidin-1-yl)-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>14</b>)</h3><div class="NLM_p last">2-Chloro-6,7-dimethoxy-<i>N</i>-methyl-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine was prepared according to the procedure for making <b>24</b> from 2,4-dichloro-6,7-dimethoxylquinazoline, <i>N</i>-methyl-5-(pyrrolidin-1-yl)pentan-1-amine (synthesized according to the precedures preported previously),<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a><i>N</i>,<i>N</i>-diisopropylethylamine, and THF. Compound <b>14</b> was prepared according to the procedure for making <b>2</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-methyl-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (50 mg, 0.13 mmol), pyrrolidine (21 μL, 0.26 mmol), HCl in dioxane (4.0 M, 65 μL, 0.26 mmol), and isopropanol (0.7 mL). The title compound <b>14</b> was obtained as a TFA salt, white solid (25 mg, yield 30%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.47 (s, 1H), 7.17 (s, 1H), 3.97 (s, 3H), 3.91 (s, 3H), 3.90–3.85 (m, 2H), 3.75–3.60 (m, 6H), 3.57 (s, 3H), 3.24–3.17 (m, 2H), 3.11–3.01 (m, 2H), 2.22–2.06 (m, 6H), 2.05–1.97 (m, 2H), 1.96–1.87 (m, 2H), 1.86–1.77 (m, 2H), 1.55–1.45 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 162.30, 157.06, 150.22, 147.38, 139.69, 109.50, 104.02, 99.83, 56.97, 56.79, 56.04 (two carbons), 55.09 (two carbons), 54.45 (two carbons), 41.10, 27.43, 26.91 (two carbons), 25.08 (two carbons), 23.97 (two carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.28 min. MS (ESI): 428 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 6,7-Dimethoxy-2-(piperidin-1-yl)-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>15</b>)</h3><div class="NLM_p last">Compound <b>15</b> was prepared according to the procedure for making <b>2</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 70 mg, 0.19 mmol), piperidine (commercially available, 37 μL, 0.38 mmol), HCl in dioxane (4.0 M, 95 μL, 0.38 mmol), and isopropanol (1.0 mL). The title compound <b>15</b> was obtained as a TFA salt, light yellow solid (101 mg, yield 83%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.54 (s, 1H), 7.09 (s, 1H), 3.94 (s, 3H), 3.91 (s, 3H), 3.88–3.82 (m, 4H), 3.69–3.61 (m, 4H), 3.22–3.16 (m, 2H), 3.10–3.01 (m, 2H), 2.20–2.08 (m, 2H), 2.07–1.96 (m, 2H), 1.85–1.69 (m, 10H), 1.55–1.46 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.55, 157.21, 152.43, 149.01, 137.00, 104.87, 103.58, 99.79, 56.90, 56.81, 56.08, 55.06 (two carbons), 47.16 (two carbons), 42.56, 29.29, 26.87, 26.69 (two carbons), 25.19, 25.13, 23.95 (two carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.22 min. MS (ESI): 428 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2-(Azepan-1-yl)-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>16</b>)</h3><div class="NLM_p last">Compound <b>16</b> was prepared according to the procedure for making <b>2</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 70 mg, 0.19 mmol), hexamethyleneimine (commercially available, 42 μL, 0.38 mmol), HCl in dioxane (4.0 M, 95 μL, 0.38 mmol), and isopropanol (1.0 mL). The title compound <b>16</b> was obtained as a TFA salt, light yellow solid (35 mg, yield 28%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.56 (s, 1H), 7.18 (s, 1H), 3.95 (s, 3H), 3.92 (s, 3H), 3.90–3.70 (m, 4H), 3.70–3.61 (m, 4H), 3.22–3.16 (m, 2H), 3.10–3.01 (m, 2H), 2.19–2.08 (m, 2H), 2.07–1.97 (m, 2H), 1.95–1.86 (m, 4H), 1.85–1.75 (m, 4H), 1.69–1.61 (m, 4H), 1.56–1.47 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.40, 157.23, 152.91, 148.99, 137.10, 104.89, 103.68, 99.85, 56.91, 56.80, 56.10, 55.09 (two carbons), 49.16 (two carbons), 42.59, 29.47, 28.58 (three carbons), 27.85, 26.91, 25.23, 23.95 (two carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.67 min. MS (ESI): 442 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 6,7-Dimethoxy-<i>N</i><sup>2</sup>,<i>N</i><sup>2</sup>-dimethyl-<i>N</i><sup>4</sup>-(5-(pyrrolidin-1-yl)pentyl)quinazoline-2,4-diamine (<b>17</b>)</h3><div class="NLM_p last">To a solution of Pd(OAc)<sub>2</sub> (2 mg, 0.01 mmol), (+)-BINAP (6 mg, 0.01 mmol), and THF (0.5 mL) were added 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 46 mg, 0.10 mmol), dimethylamine hydrochloride (commercially available, 10 mg, 0.12 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (73 mg, 0.24 mmol). The resulting solution was stirred inside a microwave at 140 °C for 30 min. After the mixture was cooled, TLC indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was redissolved in CH<sub>2</sub>Cl<sub>2</sub>, washed with brine. The organic layer was dried, concentrated, and purified by HPLC to give 7.0 mg of the title compound <b>17</b> as a TFA salt, white solid (yield 13%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.57 (s, 1H), 7.15 (s, 1H), 3.97 (s, 3H), 3.92 (s, 3H), 3.71 (t, <i>J</i> = 7.1 Hz, 2H), 3.68–3.60 (m, 2H), 3.31 (s, 6H), 3.23–3.16 (m, 2H), 3.10–3.01 (m, 2H), 2.21–2.09 (m, 2H), 2.07–1.96 (m, 2H), 1.87–1.75 (m, 4H), 1.56–1.48 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.39, 157.35, 153.70, 149.10, 137.07, 104.92, 103.55, 99.79, 56.93, 56.82, 56.13, 55.14 (two carbons), 42.57 (two carbons), 37.96, 29.36, 26.92, 25.23, 23.96 (two carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.04 min. MS (ESI): 388 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 6,7-Dimethoxy-2-(piperazin-1-yl)-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>18</b>)</h3><div class="NLM_p last">Boc-protected compound <b>18</b> was prepared according to the procedure for making <b>1</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 56 mg, 0.11 mmol), 1-Boc-piperazine (commercially available, 82 mg, 0.44 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (57 μL, 0.33 mmol), and <i>n</i>-butanol (0.7 mL). To the resulting mixture was added TFA, and the mixture was stirred overnight at room temperature. LC–MS indicated the completion of the reaction. After removal of the solvent by rotary evaporation, the residue was purified by HPLC to give 40 mg of the title compound <b>18</b> as a TFA salt, white solid (yield 55%, two steps). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.62 (s, 1H), 7.16 (s, 1H), 4.22–4.14 (m, 4H), 3.96 (s, 3H), 3.93 (s, 3H), 3.71 (t, <i>J</i> = 7.0 Hz, 2H), 3.69–3.61 (m, 2H), 3.48–3.38 (m, 4H), 3.23–3.16 (m, 2H), 3.11–3.01 (m, 2H), 2.20–2.09 (m, 2H), 2.07–1.96 (m, 2H), 1.86–1.73 (m, 4H), 1.57–1.48 (m, 2H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 1.76 min. MS (ESI): 429 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 6,7-Dimethoxy-2-(4-methylpiperazin-1-yl)-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>19</b>)</h3><div class="NLM_p last">Compound <b>19</b> was prepared according to the procedure for making <b>1</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 56 mg, 0.11 mmol), 1-methylpiperazine (commercially available, 49 μL, 0.44 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (57 μL, 0.33 mmol), and <i>n</i>-butanol (0.7 mL). The title compound <b>19</b> was obtained as a TFA salt, gray solid (54 mg, yield 73%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.62 (s, 1H), 7.16 (s, 1H), 3.96 (s, 3H), 3.93 (s, 3H), 3.90–3.24 (m, 12H), 3.22–3.16 (m, 2H), 3.10–3.02 (m, 2H), 2.99 (s, 3H), 2.19–2.10 (m, 2H), 2.07–1.96 (m, 2H), 1.86–1.75 (m, 4H), 1.56–1.47 (m, 2H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 0.55 min. MS (ESI): 443 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i><sup>2</sup>-Cyclopentyl-6,7-dimethoxy-<i>N</i><sup>2</sup>-methyl-<i>N</i><sup>4</sup>-(5-(pyrrolidin-1-yl)pentyl)quinazoline-2,4-diamine (<b>20</b>)</h3><div class="NLM_p last">Compound <b>20</b> was prepared according to the procedure for making <b>1</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 56 mg, 0.11 mmol), <i>N</i>-methylcyclopentanamine (commercially available, 44 mg, 0.44 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (57 μL, 0.33 mmol), and <i>n</i>-butanol (0.7 mL). The title compound <b>20</b> was obtained as a TFA salt, yellow solid (40 mg, yield 54%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.57 (s, 1H), 7.18 (s, 1H), 5.15–5.01 (m, 1H), 3.96 (s, 3H), 3.92 (s, 3H), 3.72–3.61 (m, 4H), 3.23–3.16 (m, 2H), 3.14 (s, 3H), 3.10–3.01 (m, 2H), 2.19–2.10 (m, 2H), 2.05–1.93 (m, 4H), 1.89–1.68 (m, 10H), 1.56–1.47 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.35, 157.29, 153.54, 149.07, 137.09, 104.90, 103.75, 99.90, 58.92, 56.92, 56.80, 56.10, 55.10 (three carbons), 42.62, 30.41, 29.65 (two carbons), 29.39, 26.93, 25.28, 25.25, 23.95 (two carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.83 min. MS (ESI): 442 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>N</i><sup>2</sup>-Cyclohexyl-6,7-dimethoxy-<i>N</i><sup>2</sup>-methyl-<i>N</i><sup>4</sup>-(5-(pyrrolidin-1-yl)pentyl)quinazoline-2,4-diamine (<b>21</b>)</h3><div class="NLM_p last">Compound <b>21</b> was prepared according to the procedure for making <b>1</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 56 mg, 0.11 mmol), <i>N</i>-methylcyclohexylamine (commercially available, 44 mg, 0.44 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (57 μL, 0.33 mmol), and <i>n</i>-butanol (0.7 mL). The title compound <b>21</b> was obtained as a TFA salt, brown solid (47 mg, yield 62%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.56 (s, 1H), 7.18 (s, 1H), 4.65–4.49 (m, 1H), 3.95 (s, 3H), 3.92 (s, 3H), 3.71–3.61 (m, 4H), 3.23–3.16 (m, 2H), 3.14 (s, 3H), 3.10–3.01 (m, 2H), 2.21–2.08 (m, 2H), 2.07–1.98 (m, 2H), 1.97–1.90 (m, 2H), 1.86–1.75 (m, 6H), 1.71–1.60 (m, 2H), 1.56–1.42 (m, 4H), 1.40–1.17 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 160.35, 157.24, 153.14, 149.04, 137.14, 104.89, 103.79, 99.93, 57.59, 56.91, 56.80, 56.08, 55.09 (three carbons), 42.68, 30.85 (two carbons), 30.09, 29.40, 26.93 (two carbons), 26.38, 25.34, 23.95 (two carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.54 min. MS (ESI): 456 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i><sup>2</sup>-Cyclopentyl-6,7-dimethoxy-<i>N</i><sup>4</sup>-(5-(pyrrolidin-1-yl)pentyl)quinazoline-2,4-diamine (<b>22</b>)</h3><div class="NLM_p last">Compound <b>22</b> was prepared according to the procedure for making <b>1</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 56 mg, 0.11 mmol), cyclopentylamine (commercially available, 44 μL, 0.44 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (57 μL, 0.33 mmol), and <i>n</i>-butanol (0.7 mL). The title compound <b>22</b> was obtained as a TFA salt, light yellow solid (41 mg, yield 57%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.52 (s, 1H), 6.95 (s, 1H), 4.41–4.28 (m, 1H), 3.95 (s, 3H), 3.90 (s, 3H), 3.73–3.60 (m, 4H), 3.23–3.16 (m, 2H), 3.10–2.99 (m, 2H), 2.20–1.98 (m, 6H), 1.88–1.75 (m, 6H), 1.73–1.60 (m, 4H), 1.56–1.46 (m, 2H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.58 min. MS (ESI): 428 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i><sup>2</sup>-Cyclohexyl-6,7-dimethoxy-<i>N</i><sup>4</sup>-(5-(pyrrolidin-1-yl)pentyl)quinazoline-2,4-diamine (<b>23</b>)</h3><div class="NLM_p last">Compound <b>23</b> was prepared according to the procedure for making <b>1</b> from 2-chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>, 56 mg, 0.11 mmol), <i>N</i>-cyclohexylamine (commercially available, 50 μL, 0.44 mmol), <i>N</i>,<i>N</i>-diisopropylethylamine (57 μL, 0.33 mmol), and <i>n</i>-butanol (0.7 mL). The title compound <b>23</b> was obtained as a TFA salt, brown semisolid (45 mg, yield 61%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.50 (s, 1H), 6.91 (s, 1H), 4.00–3.85 (m, 1H), 3.94 (s, 3H), 3.89 (s, 3H), 3.70–3.61 (m, 4H), 3.23–3.15 (m, 2H), 3.10–3.01 (m, 2H), 2.19–2.10 (m, 2H), 2.06–1.99 (m, 3H), 1.88–1.75 (m, 7H), 1.54–1.28 (m, 8H). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.69 min. MS (ESI): 442 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 2-Chloro-6,7-dimethoxy-<i>N</i>-(5-(pyrrolidin-1-yl)pentyl)quinazolin-4-amine (<b>24</b>)</h3><div class="NLM_p last">To the solution of 2,4-dichloro-6,7-dimethoxyquinazoline (commercially available, 1.00 g, 3.86 mmol) in THF (9.5 mL) was added 5-(pyrrolidin-1-yl)pentan-1-amine (commercially available, 1.33 g, 8.49 mmol), followed by the addition of <i>N</i>,<i>N</i>-diisopropylethylamine (612 μL, 3.51 mmol). And the resulting mixture was stirred at room temperature for 6 h until TLC showed that the starting material had disappeared. Water was added to the reaction mixture, and the resulting solution was extracted with ethyl acetate. The organic layer was washed with brine, dried, and concentrated to give the crude product, which was purified on ISCO using 24 g silica gel column, eluting with hexane–ethyl acetate to give 754 mg of the title compound <b>24</b> as a yellow semisolid (yield 52%). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ 7.61 (s, 1H), 6.99 (s, 1H), 3.98 (s, 3H), 3.96 (s, 3H), 3.73 (t, <i>J</i> = 7.2 Hz, 2H), 3.69–3.61 (m, 2H), 3.24–3.16 (m, 2H), 3.11–3.01 (m, 2H), 2.20–2.09 (m, 2H), 2.07–1.96 (m, 2H), 1.87–1.76 (m, 4H), 1.57–1.47 (m, 2H); <sup>13</sup>C NMR (100 MHz, MeOH-<i>d</i><sub>4</sub>) δ 161.38, 157.90, 152.68, 151.65, 141.15, 107.16, 103.64, 102.16, 57.02, 56.99, 56.08, 55.12 (two carbons), 42.73, 29.27, 26.64, 24.82, 23.95 (two carbons). HPLC purity: >95%; <i>t</i><sub>R</sub> = 3.60 min. MS (ESI): 379 [M + H]<sup>+</sup>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Radioactive Assay (Also Known as Scintillation Proximity Assay)</h3><div class="NLM_p last">SETD8 catalytic domain was expressed in <i>E. coli</i> and purified as reported.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Radioactive assay was developed to screen our chemical library for small molecule inhibitors. Methylation (10 μL) reactions were carried out in a buffer containing 50 mM Tris-HCl (pH 8.0), 10 mM GSH, 0.1%Triton X-100, at room temperature using 50 nM SETD8, 1.5 μM tritium labeled SAM (catalog no. NET155V250UC, PerkinElmer), and 5 μM biotinylated H4 (1–24) peptide substrate (SGRGKGGKGLGKGGAKRHRKVLRDK-biotin) in 384-well plates in the presence of 50 μM compounds. The reactions were then quenched by addition of equal volume of 7.5 M guanidine hydrochloride after a 1 h incubation. Then 40 μL of buffer (20 mM Tris-HCl, pH 8.0) was added into the quenched samples, and all samples were then transferred into a streptavidin/scintillant-coated microplate (catalog no. SMP410, PerkinElmer). The amount of methylated peptide was quantified by tracing the radioactivity (counts per minute) as measured after 1 h using a TopCount plate reader (PerkinElmer). For IC<sub>50</sub> values determination, the compounds were serially diluted 2-fold in DMSO for a total of 11 concentrations, beginning at 0.25 mM and tested in the same condition.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Microfluidic Capillary Electrophoresis Assay</h3><div class="NLM_p">Inhibition of SETD8 methyltransferase activity was analyzed by monitoring decrease in methylation of the fluorescein labeled peptide, TW21 (NH<sub>2</sub>-LGKGGAKRHRKVLRDNIQGITK(5Fam)-OH), essentially as described previously.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p last">The test compounds were solubilized in 100% DMSO to 10 mM and then plated in Greiner 384-well polypropylene plates using 3-fold dilution scheme over 10 points spanning the concentration range from 3 mM to 0.15 μM using TECAN Genesis liquid handler. Then 1 μL of the serial dilution was transferred into compound dilution plate using Nanoscreen MultiMek liquid handling robot. Prior to performing the assay the compounds were diluted 10-fold in 1× assay buffer (20 mM Tris, pH 8.0, 25 mM NaCl, 2 mM DTT, and 0.05% Tween-20), and an amount of 2.5 μL of the resulting dilution was transferred into assay plate by Nanoscreen MultiMek liquid handling robot. To this plate 20 μL of 50 nM SETD8 and 2 μM TW21 peptide cocktail in 1× assay buffer were added using multidrop liquid dispenser. Following a 10 min incubation of the compounds with the enzyme/peptide mix, the reaction was initiated by adding 2.5 μL of 150 μM SAM in 1× buffer. For 100% inhibition controls 1× buffer was added instead of SAM. The reaction was allowed to proceed at room temperature for 120 min, and then the reaction was terminated by adding 35 μL of 0.08 ng/μL Endo-LysC protease solution. Following an additional 1 h incubation the plate was read on a Caliper Life Sciences EZ reader II using upstream voltage −500 V, downstream voltage −1800 V, and pressure of −1.5 psi. The IC<sub>50</sub> values were determined using Screenable Solutions software.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">All ITC measurements were performed at 25 °C using an AutoITC200 microcalorimeter (MicroCal/GE Healthcare). The calorimeter cell (volume 200 μL) was loaded with SETD8 protein in the full salt dialysis buffer (150 mM NaCl) at a concentration of 100 μM. The syringe was loaded with compound <b>1</b> (dissolved in the same buffer) at a concentration of 1 mM. A typical injection protocol included a single 0.2 μL first injection followed by 26 1.5 μL injections of the compound into the calorimeter cell. The spacing between injections was kept at 180 s and the reference power at 8 μcal/s. A control experiment was performed by titrating compound <b>1</b> into buffer under identical settings to determine the heat signals that arose from compound dilution; these were subtracted from the heat signals of protein–compound interaction. The data were analyzed using Origin for ITC, version 7.0, software supplied by the manufacturer and fitted well to a one-site binding model.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Surface Plasmon Resonance</h3><div class="NLM_p last">The interaction between compound <b>1</b> and protein SETD8 was further explored using a ProteOn XPR36 biosensor (Bio-Rad Laboratories, Inc.) at 25 °C. PBS buffer (phosphate buffered saline, pH 7.4) supplemented with 0.005% Tween-20 was used as the running buffer. A GLH (catalog no. 176-5013, Bio-Rad Laboratories, Inc.) sensor chip was first activated by flowing a mixture of 20 mM sulfo-NHS and 20 mM EDC over the chip surface for 5 min at a flow rate of 30 μL/min. SETD8 was then diluted to 20 and 10 μg/mL in 5 mM NaOAc, pH 5.0, and immobilized onto two ligand channels (30 μL/min for 2 min). The surface was then deactivated by flowing 1 M ethanolamine for 5 min at a flow rate 60 μL/min. A blank injection of the running buffer was made, after which four concentrations of the compound in the running buffer (100, 33.3, 11.1, and 3.7 μM) and a running buffer were injected simultaneously at a flow rate of 50 μL/min for 60 s. The sensorgrams obtained at the four concentrations of the compound were fit simultaneously after subtracting a ligand reference (inner spot) and a double compound reference (buffer) using a Langmuir model to obtain on (<i>k</i><sub>a</sub>) and off (<i>k</i><sub>d</sub>) rates. The kinetic <i>K</i><sub>D</sub> was calculated based on the on and off rates for three replicates. An equilibrium analysis of the data was also performed to calculate the <i>K</i><sub>D</sub>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> MOA Studies</h3><div class="NLM_p last">IC<sub>50</sub> values were determined for compound <b>1</b> at varying concentrations of SAM, 50 nM SETD8, 200 μM peptide H4 (1–24) or at varying concentrations of substrate H4 (1–24) peptide, 50 nM of SETD8, 250 μM SAM. The reaction mixtures were incubated 15 min at 23 °C. To stop the enzymatic reactions, an equal volume of 7.5 M guanidine hydrochloride was added and mixed. Then 10 μL of mixture was spotted onto a square of SAM<sup>2</sup> biotin capture membrane (catalog no. V2861, Promega), allowed absorption for 5 min at room temperature, washed with 2 M of NaCl solution several times followed by two deionized water wash, and dried, and scintillation liquid was added and CPM (counts per minute) were read using the scintillation counter.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Peptide Displacement Assay</h3><div class="NLM_p last">H4K20me (1–24) peptide (SGRGKGGKGLGKGGAKRHRKme1VLRD) with N-terminus labelled by FITC was ordered from Tufts University Core Services (Boston, MA). The sample was prepared in a buffer containing 50 mM Tris-HCl (pH 8.0), 10 mM GSH, 0.01% Triton X-100. 50 nM peptide was incubated with 20 μM SETD8 protein, 1 mM <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH), and different concentrations of compounds. The final volume was 10 μL with less than 5% DMSO. The FP measurement was performed in black, 384-well PCR plate (catalog no. 47744-828, VWR), and signal was read with a Synergy H4 microplate reader (BioTek). The polarization values in millipolarization units were measured at an excitation wavelength of 485 nm and an emission wavelength of 528 nm.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Selectivity Assay</h3><div class="NLM_p last">Selectivity of compound <b>1</b> against a panel of methyltransferases was assessed as previously reported.<a onclick="showRef(event, 'ref24 ref36 ref53'); return false;" href="javascript:void(0);" class="ref ref24 ref36 ref53">(24, 36, 53)</a> In brief, the effect of compound <b>1</b> on activity of G9a, SETDB1, GLP, SUV39H2, SETD7, PRMT3, PRMT5-MEP50 complex, PRMT1, SUV420H1, SUV420H2, SMYD2, DNMT1, PRC2 complex, MLL1 complex, and DOT1L was assessed by monitoring the incorporation of tritium-labeled methyl group to lysine or arginine residues of peptide substrates using radioactive assay. Assays were performed in a 20 μL reaction mixture containing <sup>3</sup>H-SAM (catalog no. NET155V250UC, Perkin Elmer) at substrate concentrations close to <i>K</i><sub>m</sub> values for each enzyme. Compound concentrations from 100 nM to 100 μM were used in all selectivity assays. To stop the enzymatic reactions, 7.5 M guanidine hydrochloride was added, followed by 180 μL of buffer (20 mM Tris, pH 8.0), mixed, and then transferred to a 96-well FlashPlate (catalog no. SMP103, PerkinElmer). After mixing, the reaction mixtures in FlashPlates were incubated for 1 h and the CPM were measured using TopCount plate reader (PerkinElmer). The CPM in the absence of compound for each data set were defined as 100% activity. In the absence of the enzyme, the CPM in each data set were defined as background (0%). Data were plotted using SigmaPlot software. For DNMT1 the dsDNA substrate was prepared by annealing two complementary strands (biotintlated forward strand B-GAGCCCGTAAGCCCGTTCAGGTCG and reverse strand CGACCTGAACGGGCTTACGGGCTC), synthesized by Eurofins MWG Operon. For DOT1L, a filter-based assay was used. In this assay, 20 μL of reaction mixtures was incubated at room temperature for 1 h, 100 μL of 10% trichloroacetic acid (TCA) was added, mixed, and transferred to filter plates (catalog no. MSFBN6B10, Millipore). Plates were centrifuged at 2000 rpm (Allegra X-15R; Beckman Coulter, Inc.) for 2 min followed by two additional 10% TCA washes and one ethanol wash (180 μL) followed by centrifugation. Plates were dried, and 100 μL of MicroScint-O (catalog no. 6013611, PerkinElmer) was added to each well, centrifuged, and removed. Then 70 μL of MicroScint-O was added again, and CPM were measured using TopCount plate reader.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47"><a href="/doi/suppl/10.1021/jm500871s">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27817" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27817" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>1</b>. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm500871s/suppl_file/jm500871s_si_001.pdf">jm500871s_si_001.pdf (172.04 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm500871s" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22808" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22808" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#254f4c444b4f4c4b65504b460b404150"><span class="__cf_email__" data-cfemail="a6cccfc7c8cccfc8e6d3c8c588c3c2d3">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anqi Ma</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenyu Yu</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel M. Bleich</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Martin Herold</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kyle V. Butler</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline L. Norris</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Victoria Korboukh</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ashutosh Tripathy</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William P. Janzen</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen V. Frye</span> - <span class="hlFld-Affiliation affiliation">†Center for Integrative Chemical
Biology and Drug Discovery, Division
of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of
Pharmacy, ‡Department of Pharmacology, School of Medicine, §Lineberger Comprehensive Cancer Center, and ∥Department of
Biochemistry and Biophysics, UNC Macromolecular Interactions Facility, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.M. and W.Y. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2753e2688-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12810" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12810" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Raymond C. Trievel at University of Michigan for providing the SETD8 vector, Jacob Stuckey for assisting with ITC studies, and Catherine Simpson for assisting with MCE assay studies. The research described here was supported by Grant R01GM103893 from the National Institute of General Medical Sciences of the National Institutes of Health, the University Cancer Research Fund from University of North Carolina at Chapel Hill, the V Foundation for Cancer Research, and the Structural Genomics Consortium, a registered charity (No. 1097737) that receives funds from the Canada Foundation for Innovation, Eli Lilly Canada, GlaxoSmithKline, the Ontario Ministry of Economic Development and Innovation, the Novartis Research Foundation, Pfizer, AbbVie, Takeda, Janssen, Boehringer Ingelheim, Bayer, and the Wellcome Trust.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i50" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i50"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i51" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i51"> Abbreviations Used</h2><tr><td class="NLM_term">H4K20</td><td class="NLM_def"><p class="first last">histone H4 lysine 20</p></td></tr><tr><td class="NLM_term">H4K20me</td><td class="NLM_def"><p class="first last">histone H4 lysine 20 monomethylation</p></td></tr><tr><td class="NLM_term">PCNA</td><td class="NLM_def"><p class="first last">proliferating cell nuclear antigen</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">HKMT</td><td class="NLM_def"><p class="first last">histone lysine methyltransferase</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SET</td><td class="NLM_def"><p class="first last"><i>Su(var)</i>, <i>E(z)</i>, and <i>Trithorax</i></p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">KMT5A</td><td class="NLM_def"><p class="first last">lysine methyltransferase 5A</p></td></tr><tr><td class="NLM_term">EMT</td><td class="NLM_def"><p class="first last">epithelial–mesenchymal transition</p></td></tr><tr><td class="NLM_term">p53K382me1</td><td class="NLM_def"><p class="first last">monomethylation of p53 at lysine 382</p></td></tr><tr><td class="NLM_term">PCNA</td><td class="NLM_def"><p class="first last">proliferating cell nuclear antigen</p></td></tr><tr><td class="NLM_term">PCNAK248me1</td><td class="NLM_def"><p class="first last">monomethylation of proliferating cell nuclear antigen at lysine 248</p></td></tr><tr><td class="NLM_term">MCE</td><td class="NLM_def"><p class="first last">microfluidic capillary electrophoresis</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term">PRC2</td><td class="NLM_def"><p class="first last">polycomb repressive complex 2</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66460" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66460" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 53 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0lj4_UQjUEfGHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lgDL3VIl-XBkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Helin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span> </span><span class="NLM_article-title">Chromatin proteins and modifications as drug targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">480</span><span class="NLM_x">–</span> <span class="NLM_lpage">488</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnature12751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=24153301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=480-488&author=K.+Helinauthor=D.+Dhanak&title=Chromatin+proteins+and+modifications+as+drug+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin proteins and modifications as drug targets</span></div><div class="casAuthors">Helin, Kristian; Dhanak, Dashyant</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">480-488</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A plethora of groundbreaking studies have demonstrated the importance of chromatin-assocd. proteins and post-translational modifications of histones, proteins and DNA (so-called epigenetic modifications) for transcriptional control and normal development.  Disruption of epigenetic control is a frequent event in disease, and the first epigenetic-based therapies for cancer treatment have been approved.  A generation of new classes of potent and specific inhibitors for several chromatin-assocd. proteins have shown promise in preclin. trials.  Although the biol. of epigenetic regulation is complex, new inhibitors such as these will hopefully be of clin. use in the coming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa6LKkNrgP9bVg90H21EOLACvtfcHk0lgDL3VIl-XBkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E&md5=ca3f12c6aaba31bb571095bacc02550c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature12751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12751%26sid%3Dliteratum%253Aachs%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DChromatin%2520proteins%2520and%2520modifications%2520as%2520drug%2520targets%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D480%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Bernstein, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">The mammalian epigenome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0lgDL3VIl-XBkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0liwulSo8HQPwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0liwulSo8HQPwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laible, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, G.</span><span> </span><span class="NLM_article-title">SET domain proteins modulate chromatin domains in eu- and heterochromatin</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1007%2Fs000180050127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=9487389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADyaK1cXns1CltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=80-93&author=T.+Jenuweinauthor=G.+Laibleauthor=R.+Dornauthor=G.+Reuter&title=SET+domain+proteins+modulate+chromatin+domains+in+eu-+and+heterochromatin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic control of transcription. SET domain proteins modulate chromatin domains in eu- and heterochromatin</span></div><div class="casAuthors">Jenuwein, T.; Laible, G.; Dorn, R.; Reuter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-93</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review with 110 refs. is given as part of a multi-author review on epigenetic control of transcription.  The SET domain is a 130-amino acid, evolutionarily conserved sequence motif present in chromosomal proteins that function in modulating gene activities from yeast to mammals.  Initially identified as members of the Polycomb- and trithorax-group (Pc-6 and trx-G) gene families, which are required to maintain expression boundaries of homeotic selector (HOM-C) genes, SET domain proteins are also involved in position-effect-variegation (PEV), telomeric and centromeric gene silencing, and possibly in detg. chromosome architecture.  These observations implicate SET domain proteins as multifunctional chromatin regulators with activities in both eu- and heterochromatin - a role consistent with their modular structure, which combines the SET domain with addnl. sequence motifs of either a cysteine-rich region/zinc-finger type or the chromo domain.  Multiple functions for chromatin regulators are not restricted to the SET protein family, since many trx-G (but only very few Pc-G) genes are also modifiers of PEV.  Together, these data establish a model in which the modulation of chromatin domains is mechanistically linked with the regulation of key developmental loci (e.g. HOM-C).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXwGgAe0PQ57Vg90H21EOLACvtfcHk0liwulSo8HQPwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1CltA%253D%253D&md5=dd161ab187056ee96b1d1468a0e5ccd1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs000180050127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs000180050127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DLaible%26aufirst%3DG.%26aulast%3DDorn%26aufirst%3DR.%26aulast%3DReuter%26aufirst%3DG.%26atitle%3DSET%2520domain%2520proteins%2520modulate%2520chromatin%2520domains%2520in%2520eu-%2520and%2520heterochromatin%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D1998%26volume%3D54%26spage%3D80%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Xiao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">SET domains and histone methylation</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.sbi.2003.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=14675547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=699-705&author=B.+Xiaoauthor=J.+R.+Wilsonauthor=S.+J.+Gamblin&title=SET+domains+and+histone+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SET domains and histone methylation</span></div><div class="casAuthors">Xiao, Bing; Wilson, Jonathan R.; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">699-705</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The realization that SET domains, which are found in numerous proteins involved in chromatin regulation, catalyze the methylation of lysine residues has led to intense interest in their cellular, biochem. and structural properties.  The structures of five SET domain proteins have been reported over the past year.  SET domains possess a novel fold, and use adjacent domains for both structural stabilization and the completion of their active sites.  The cofactor S-adenosyl-L-methionine and peptide substrates bind on opposite faces of the SET domain.  Remarkably, the sidechain of the target lysine approaches the transferred Me group through a narrow channel that passes through the middle of the domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj4grEHxVvY7Vg90H21EOLACvtfcHk0liwulSo8HQPwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1yjsLw%253D&md5=238b75c760b6bde8a56f128dd8077c30</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2003.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2003.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DSET%2520domains%2520and%2520histone%2520methylation%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2003%26volume%3D13%26spage%3D699%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0lhPk42JM7nUTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lhPk42JM7nUTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a inhibitors: design, synthesis, and structure activity relationships of 2,4-diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+lysine+methyltransferase+G9a+inhibitors%3A+design%2C+synthesis%2C+and+structure+activity+relationships+of+2%2C4-diamino-7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wigle, Tim J.; Wasney, Gregory A.; Dong, Aiping; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Norris, Jacqueline L.; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5844-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers.  Genetic knockdown of G9a inhibits cancer cell growth; and the dimethylation of p53 K373 results in the inactivation of p53.  Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by I ( R1 = Bn; R2 = Me)(BIX01294), a selective small mol. inhibitor of G9a and GLP (G9a-like protein), led to the discovery of I [R1 = Me; R2 = (CH2)3NMe2](II)(UNC0224) as a potent G9a inhibitor with excellent selectivity.  A high resoln. X-ray crystal structure of the G9a-II complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-(dimethylamino)propoxy side chain of II resulted in the discovery of I [R1 = Me; R2 = (CH2)2O(CH2)2NMe2](UNC0321, Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgS0YxdNdNxLVg90H21EOLACvtfcHk0liFAOs38M19_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D&md5=8bd703c0a975a33bf9e5c019f1f95ee6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520structure%2520activity%2520relationships%2520of%25202%252C4-diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0liFAOs38M19_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6139</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lisl-VvvdpUrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0lisl-VvvdpUrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lisl-VvvdpUrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span><span class="NLM_issue">133</span><span class="NLM_x">) </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=16746-16749&issue=133&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis+and+crystallographic+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26issue%3D133%26spage%3D16746%26epage%3D16749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamji, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A small-molecule probe of the histone methyltransferase G9a induces cellular senescence in pancreatic adenocarcinoma</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1157</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb300139y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1152-1157&author=Y.+Yuanauthor=Q.+Wangauthor=J.+Paulkauthor=S.+Kubicekauthor=M.+M.+Kempauthor=D.+J.+Adamsauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=A+small-molecule+probe+of+the+histone+methyltransferase+G9a+induces+cellular+senescence+in+pancreatic+adenocarcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fcb300139y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300139y%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520small-molecule%2520probe%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520induces%2520cellular%2520senescence%2520in%2520pancreatic%2520adenocarcinoma%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1152%26epage%3D1157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lgDBNJicA6Jmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Della Pietra, A.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+Della+Pietraauthor=E.+Diazauthor=L.+V.+Lafranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDella%2BPietra%26aufirst%3DA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLafrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of potent, selective, cell-active inhibitors of the histone lysine methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+potent%2C+selective%2C+cell-active+inhibitors+of+the+histone+lysine+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0lj6YcnGfntWnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520potent%252C%2520selective%252C%2520cell-active%2520inhibitors%2520of%2520the%2520histone%2520lysine%2520methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibáñez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amaya, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span> </span><span class="NLM_article-title">Sinefungin derivatives as inhibitors and structure probes of protein lysine methyltransferase SETD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">18004</span><span class="NLM_x">–</span> <span class="NLM_lpage">18014</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja307060p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=18004-18014&author=W.+Zhengauthor=G.+Ib%C3%A1%C3%B1ezauthor=H.+Wuauthor=G.+Blumauthor=H.+Zengauthor=A.+Dongauthor=F.+Liauthor=T.+Hajianauthor=A.+Allali-Hassaniauthor=M.+F.+Amayaauthor=A.+Siarheyevaauthor=W.+Yuauthor=P.+J.+Brownauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Minauthor=M.+Luo&title=Sinefungin+derivatives+as+inhibitors+and+structure+probes+of+protein+lysine+methyltransferase+SETD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Sinefungin Derivatives as Inhibitors and Structure Probes of Protein Lysine Methyltransferase SETD2</span></div><div class="casAuthors">Zheng, Weihong; Ibanez, Glorymar; Wu, Hong; Blum, Gil; Zeng, Hong; Dong, Aiping; Li, Fengling; Hajian, Taraneh; Allali-Hassani, Abdellah; Amaya, Maria F.; Siarheyeva, Alena; Yu, Wenyu; Brown, Peter J.; Schapira, Matthieu; Vedadi, Masoud; Min, Jinrong; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18004-18014</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic regulation is involved in numerous physiol. and pathogenic processes.  Among the key regulators that orchestrate epigenetic signaling are over 50 human protein lysine methyltransferases (PKMTs).  Interrogation of the functions of individual PKMTs can be facilitated by target-specific PKMT inhibitors.  Given the emerging need for such small mols., an approach was envisioned to identify target-specific methyltransferase inhibitors by screening privileged small-mol. scaffolds against diverse methyltransferases.  In this work, the feasibility of such an approach was demonstrated by identifying the inhibitors of SETD2.  N-Pr sinefungin (Pr-SNF) was shown to interact preferentially with SETD2 by matching the distinct transition-state features of SETD2's catalytically active conformer.  With Pr-SNF as a structure probe, the dual roles of SETD2's post-SET loop in regulating substrate access through a distinct topol. reconfiguration was revealed.  Privileged sinefungin scaffolds are expected to have broad use as structure and chem. probes of methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-sWI1BJjg7Vg90H21EOLACvtfcHk0lj6YcnGfntWnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL&md5=30f5ba653007349df64e095f1c2cf188</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja307060p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307060p%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DIb%25C3%25A1%25C3%25B1ez%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAmaya%26aufirst%3DM.%2BF.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSinefungin%2520derivatives%2520as%2520inhibitors%2520and%2520structure%2520probes%2520of%2520protein%2520lysine%2520methyltransferase%2520SETD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D18004%26epage%3D18014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Qi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardayfio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span> </span><span class="NLM_article-title">Selective inhibition of Ezh2 by a small molecule inhibitor blocks tumor cells proliferation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">21360</span><span class="NLM_x">–</span> <span class="NLM_lpage">21365</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=21360-21365&author=W.+Qiauthor=H.+Chanauthor=L.+Tengauthor=L.+Liauthor=S.+Chuaiauthor=R.+Zhangauthor=J.+Zengauthor=M.+Liauthor=H.+Fanauthor=Y.+Linauthor=J.+Guauthor=O.+Ardayfioauthor=J.-H.+Zhangauthor=X.+Yanauthor=J.+Fangauthor=Y.+Miauthor=M.+Zhangauthor=T.+Zhouauthor=G.+Fengauthor=Z.+Chenauthor=G.+Liauthor=T.+Yangauthor=K.+Zhaoauthor=X.+Liuauthor=Z.+Yuauthor=C.+X.+Luauthor=P.+Atadjaauthor=E.+Li&title=Selective+inhibition+of+Ezh2+by+a+small+molecule+inhibitor+blocks+tumor+cells+proliferation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DArdayfio%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DJ.-H.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DC.%2BX.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DSelective%2520inhibition%2520of%2520Ezh2%2520by%2520a%2520small%2520molecule%2520inhibitor%2520blocks%2520tumor%2520cells%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D21360%26epage%3D21365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chory, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wernimont, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scopton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federation, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wienholds, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fncomms2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23250418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1288&author=W.+Yuauthor=E.+J.+Choryauthor=A.+K.+Wernimontauthor=W.+Tempelauthor=A.+Scoptonauthor=A.+Federationauthor=J.+J.+Marineauauthor=J.+Qiauthor=D.+Barsyte-Lovejoyauthor=J.+Yiauthor=R.+Marcellusauthor=R.+E.+Iacobauthor=J.+R.+Engenauthor=C.+Griffinauthor=A.+Amanauthor=E.+Wienholdsauthor=F.+Liauthor=J.+Pinedaauthor=G.+Estiuauthor=T.+Shatsevaauthor=T.+Hajianauthor=R.+Al-awarauthor=J.+E.+Dickauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+E.+Bradnerauthor=M.+Schapira&title=Catalytic+site+remodelling+of+the+DOT1L+methyltransferase+by+selective+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span></div><div class="casAuthors">Yu Wenyu; Chory Emma J; Wernimont Amy K; Tempel Wolfram; Scopton Alex; Federation Alexander; Marineau Jason J; Qi Jun; Barsyte-Lovejoy Dalia; Yi Joanna; Marcellus Richard; Iacob Roxana E; Engen John R; Griffin Carly; Aman Ahmed; Wienholds Erno; Li Fengling; Pineda Javier; Estiu Guillermina; Shatseva Tatiana; Hajian Taraneh; Al-Awar Rima; Dick John E; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Bradner James E; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1288</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Selective inhibition of protein methyltransferases is a promising new approach to drug discovery.  An attractive strategy towards this goal is the development of compounds that selectively inhibit binding of the cofactor, S-adenosylmethionine, within specific protein methyltransferases.  Here we report the three-dimensional structure of the protein methyltransferase DOT1L bound to EPZ004777, the first S-adenosylmethionine-competitive inhibitor of a protein methyltransferase with in vivo efficacy.  This structure and those of four new analogues reveal remodelling of the catalytic site.  EPZ004777 and a brominated analogue, SGC0946, inhibit DOT1L in vitro and selectively kill mixed lineage leukaemia cells, in which DOT1L is aberrantly localized via interaction with an oncogenic MLL fusion protein.  These data provide important new insight into mechanisms of cell-active S-adenosylmethionine-competitive protein methyltransferase inhibitors, and establish a foundation for the further development of drug-like inhibitors of DOT1L for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcST2GJQO5XUmjDnmmyUf5qOfW6udTcc2eYsIBGkRUHh-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D&md5=f7b501ac4188b14b48fc435637e0e192</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fncomms2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2304%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChory%26aufirst%3DE.%2BJ.%26aulast%3DWernimont%26aufirst%3DA.%2BK.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DScopton%26aufirst%3DA.%26aulast%3DFederation%26aufirst%3DA.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DYi%26aufirst%3DJ.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DWienholds%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPineda%26aufirst%3DJ.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DCatalytic%2520site%2520remodelling%2520of%2520the%2520DOT1L%2520methyltransferase%2520by%2520selective%2520inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D1288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Williams, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalisay, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amphlett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maneerat, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chavez, M. A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matainaho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, R. J.</span><span> </span><span class="NLM_article-title">Nahuoic acid A produced by a streptomyces sp. isolated from a marine sediment is a selective SAM-competitive inhibitor of the histone methyltransferase SETD8</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span></span><div class="citationLinks">[<a href="/doi/10.1021/ol303416k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=414-417&author=D.+E.+Williamsauthor=D.+S.+Dalisayauthor=F.+Liauthor=J.+Amphlettauthor=W.+Maneeratauthor=M.+A.+G.+Chavezauthor=Y.+A.+Wangauthor=T.+Matainahoauthor=W.+Yuauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=R.+J.+Andersen&title=Nahuoic+acid+A+produced+by+a+streptomyces+sp.+isolated+from+a+marine+sediment+is+a+selective+SAM-competitive+inhibitor+of+the+histone+methyltransferase+SETD8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fol303416k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol303416k%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DDalisay%26aufirst%3DD.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DAmphlett%26aufirst%3DJ.%26aulast%3DManeerat%26aufirst%3DW.%26aulast%3DChavez%26aufirst%3DM.%2BA.%2BG.%26aulast%3DWang%26aufirst%3DY.%2BA.%26aulast%3DMatainaho%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DR.%2BJ.%26atitle%3DNahuoic%2520acid%2520A%2520produced%2520by%2520a%2520streptomyces%2520sp.%2520isolated%2520from%2520a%2520marine%2520sediment%2520is%2520a%2520selective%2520SAM-competitive%2520inhibitor%2520of%2520the%2520histone%2520methyltransferase%2520SETD8%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26spage%3D414%26epage%3D417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Synthesis and structure–activity relationship investigation of adenosine-containing inhibitors of histone methyltransferase DOT1L</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">8066</span><span class="NLM_x">–</span> <span class="NLM_lpage">8074</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300917h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=8066-8074&author=J.+L.+Anglinauthor=L.+Dengauthor=Y.+Yaoauthor=G.+Caiauthor=Z.+Liuauthor=H.+Jiangauthor=G.+Chengauthor=P.+Chenauthor=S.+Dongauthor=Y.+Song&title=Synthesis+and+structure%E2%80%93activity+relationship+investigation+of+adenosine-containing+inhibitors+of+histone+methyltransferase+DOT1L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm300917h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300917h%26sid%3Dliteratum%253Aachs%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSynthesis%2520and%2520structure%25E2%2580%2593activity%2520relationship%2520investigation%2520of%2520adenosine-containing%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D8066%26epage%3D8074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacNevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the lysine methyltransferases EZH2 and EZH1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1324</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+MacNevinauthor=F.+Liuauthor=C.+Gaoauthor=X.+P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+lysine+methyltransferases+EZH2+and+EZH1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0lhX7EXKnvSVjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7922</span><span class="NLM_x">–</span> <span class="NLM_lpage">7927</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0ljGGOblqnx9Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Beguelin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teater, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezponda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez-Garcia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoyne, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdel-Wahab, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Licht, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A. M.</span><span> </span><span class="NLM_article-title">EZH2 is required for germinal center formation and somatic EZH2 mutations promote lymphoid transformation</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=677-692&author=W.+Beguelinauthor=R.+Popovicauthor=M.+Teaterauthor=Y.+Jiangauthor=K.+L.+Buntingauthor=M.+Rosenauthor=H.+Shenauthor=S.+N.+Yangauthor=L.+Wangauthor=T.+Ezpondaauthor=E.+Martinez-Garciaauthor=H.+Zhangauthor=Y.+Zhengauthor=S.+K.+Vermaauthor=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+S.+Van+Allerauthor=R.+G.+Krugerauthor=Y.+Liuauthor=C.+F.+McHughauthor=D.+W.+Scottauthor=Y.+R.+Chungauthor=N.+Kelleherauthor=R.+Shaknovichauthor=C.+L.+Creasyauthor=R.+D.+Gascoyneauthor=K.+K.+Wongauthor=L.+Cerchiettiauthor=R.+L.+Levineauthor=O.+Abdel-Wahabauthor=J.+D.+Lichtauthor=O.+Elementoauthor=A.+M.+Melnick&title=EZH2+is+required+for+germinal+center+formation+and+somatic+EZH2+mutations+promote+lymphoid+transformation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DTeater%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DRosen%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DEzponda%26aufirst%3DT.%26aulast%3DMartinez-Garcia%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DScott%26aufirst%3DD.%2BW.%26aulast%3DChung%26aufirst%3DY.%2BR.%26aulast%3DKelleher%26aufirst%3DN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DEZH2%2520is%2520required%2520for%2520germinal%2520center%2520formation%2520and%2520somatic%2520EZH2%2520mutations%2520promote%2520lymphoid%2520transformation%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D677%26epage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8931</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.+P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+vivo+chemical+probe+of+the+lysine+methyltransferases+G9a+and+GLP"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lha8wOgAjD1Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pliushchev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery and development of potent and selective inhibitors of histone methyltransferase G9a</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+L.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+development+of+potent+and+selective+inhibitors+of+histone+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0lha8wOgAjD1Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520development%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D205%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Garapaty-Rao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagnon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Busby, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span> </span><span class="NLM_article-title">Identification of EZH2 and EZH1 small molecule inhibitors with selective impact on diffuse large B cell lymphoma cell growth</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1329</span><span class="NLM_x">–</span> <span class="NLM_lpage">1339</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.chembiol.2013.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=24183969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWmtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1329-1339&author=S.+Garapaty-Raoauthor=C.+Nasveschukauthor=A.+Gagnonauthor=E.+Y.+Chanauthor=P.+Sandyauthor=J.+Busbyauthor=S.+Balasubramanianauthor=R.+Campbellauthor=F.+Zhaoauthor=L.+Bergeronauthor=J.+E.+Audiaauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=P.+Trojer&title=Identification+of+EZH2+and+EZH1+small+molecule+inhibitors+with+selective+impact+on+diffuse+large+B+cell+lymphoma+cell+growth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of EZH2 and EZH1 Small Molecule Inhibitors with Selective Impact on Diffuse Large B Cell Lymphoma Cell Growth</span></div><div class="casAuthors">Garapaty-Rao, Shivani; Nasveschuk, Christopher; Gagnon, Alexandre; Chan, Eric Y.; Sandy, Peter; Busby, Jennifer; Balasubramanian, Srividya; Campbell, Robert; Zhao, Feng; Bergeron, Louise; Audia, James E.; Albrecht, Brian K.; Harmange, Jean-Christophe; Cummings, Richard; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1329-1339</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The histone methyltransferase enhancer of Zeste homolog 2 (EZH2) is a candidate oncogene due to its prevalent overexpression in malignant diseases, including late stage prostate and breast cancers.  The dependency of cancer cells on EZH2 activity is also predicated by recurrent missense mutations residing in the catalytic domain of EZH2 that have been identified in subtypes of diffuse large B cell lymphoma, follicular lymphoma and melanoma.  Herein, we report the identification of a highly selective small mol. inhibitor series of EZH2 and EZH1.  These compds. inhibit wild-type and mutant versions of EZH2 with nanomolar potency, suppress global histone H3-lysine 27 methylation, affect gene expression, and cause selective proliferation defects.  These compds. represent a structurally distinct EZH2 inhibitor chemotype for the exploration of the role of Polycomb Repressive Complex 2-mediated H3K27 methylation in various biol. contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNGhFRO6LRbVg90H21EOLACvtfcHk0lhfvIsqKV5fpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWmtrrP&md5=6079ba01ae1f1964ac1d8108aa5e1227</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DGarapaty-Rao%26aufirst%3DS.%26aulast%3DNasveschuk%26aufirst%3DC.%26aulast%3DGagnon%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DE.%2BY.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DIdentification%2520of%2520EZH2%2520and%2520EZH1%2520small%2520molecule%2520inhibitors%2520with%2520selective%2520impact%2520on%2520diffuse%2520large%2520B%2520cell%2520lymphoma%2520cell%2520growth%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1329%26epage%3D1339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1182%2Fblood-2013-04-497644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23801631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2013&pages=1017-1025&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=A.+Basavapathruniauthor=L.+Jinauthor=P.+A.+Boriack-Sjodinauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Potent+inhibition+of+DOT1L+as+treatment+of+MLL-fusion+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Basavapathruni, Aravind; Jin, Lei; Boriack-Sjodin, P. Ann; Allain, Christina J.; Klaus, Christine R.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1017-1025</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Rearrangements of the MLL gene define a genetically distinct subset of acute leukemias with poor prognosis.  Current treatment options are of limited effectiveness; thus, there is a pressing need for new therapies for this disease.  Genetic and small mol. inhibitor studies have demonstrated that the histone methyltransferase DOT1L is required for the development and maintenance of MLL-rearranged leukemia in model systems.  Here we describe the characterization of EPZ-5676, a potent and selective aminonucleoside inhibitor of DOT1L histone methyltransferase activity.  The compd. has an inhibition const. value of 80 pM, and demonstrates 37000-fold selectivity over all other methyltransferases tested.  In cellular studies, EPZ-5676 inhibited H3K79 methylation and MLL-fusion target gene expression and demonstrated potent cell killing that was selective for acute leukemia lines bearing MLL translocations.  Continuous IV infusion of EPZ-5676 in a rat xenograft model of MLL-rearranged leukemia caused complete tumor regressions that were sustained well beyond the compd. infusion period with no significant wt. loss or signs of toxicity.  EPZ-5676 is therefore a potential treatment of MLL-rearranged leukemia and is under clin. investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT-fqlDzoK07Vg90H21EOLACvtfcHk0lhfvIsqKV5fpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE&md5=bad23c37a5fe886668e25d10b7d9b43a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-497644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-497644%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DPotent%2520inhibition%2520of%2520DOT1L%2520as%2520treatment%2520of%2520MLL-fusion%2520leukemia%26jtitle%3DBlood%26date%3D2013%26volume%3D122%26spage%3D1017%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagnon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garapaty-Rao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J.-C. P.</span><span> </span><span class="NLM_article-title">Discovery and optimization of tetramethylpiperidinyl benzamides as inhibitors of EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">378</span><span class="NLM_x">–</span> <span class="NLM_lpage">383</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml400494b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=378-383&author=C.+G.+Nasveschukauthor=A.+Gagnonauthor=S.+Garapaty-Raoauthor=S.+Balasubramanianauthor=R.+Campbellauthor=C.+Leeauthor=F.+Zhaoauthor=L.+Bergeronauthor=R.+Cummingsauthor=P.+Trojerauthor=J.+E.+Audiaauthor=B.+K.+Albrechtauthor=J.-C.+P.+Harmange&title=Discovery+and+optimization+of+tetramethylpiperidinyl+benzamides+as+inhibitors+of+EZH2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fml400494b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400494b%26sid%3Dliteratum%253Aachs%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DGagnon%26aufirst%3DA.%26aulast%3DGarapaty-Rao%26aufirst%3DS.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.-C.%2BP.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520tetramethylpiperidinyl%2520benzamides%2520as%2520inhibitors%2520of%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D378%26epage%3D383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span> </span><span class="NLM_article-title">Targeting protein lysine methylation and demethylation in cancers</span> <span class="citation_source-journal">Acta Biochim. Biophys. Sin.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">70</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1093%2Fabbs%2Fgmr109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=22194015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=70-79&author=Y.+Heauthor=I.+Korboukhauthor=J.+Jinauthor=J.+Huang&title=Targeting+protein+lysine+methylation+and+demethylation+in+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein lysine methylation and demethylation in cancers</span></div><div class="casAuthors">He, Yunlong; Korboukh, Ilia; Jin, Jian; Huang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biochimica et Biophysica Sinica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">ABBSC2</span>;
        ISSN:<span class="NLM_cas:issn">1672-9145</span>.
    
            (<span class="NLM_cas:orgname">ABBS Editorial Office</span>)
        </div><div class="casAbstract">A review.  During the last decade, we saw an explosion of studies investigating the role of lysine methylation/demethylation of histones and non-histone proteins, such as p53, NF-kappaB, and E2F1.  These Ying-Yang' post-translational modifications are important to fine-tuning the activity of these proteins.  Lysine methylation and demethylation are catalyzed by protein lysine methyltransferases (PKMTs) and protein lysine demethylases (PKDMs).  PKMTs, PKDMs, and their substrates have been shown to play important roles in cancers.  Although the underlying mechanisms of tumorigenesis are still largely unknown, growing evidence is starting to link aberrant regulation of methylation to tumorigenesis.  This review focuses on summarizing the recent progress in understanding of the function of protein lysine methylation, and in the discovery of small mol. inhibitors for PKMTs and PKDMs.  We also discuss the potential and the caveats of targeting protein lysine methylation for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO9Pis9NgA27Vg90H21EOLACvtfcHk0lj71dvTsg4_4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P&md5=bca793734c9f24cf45bbb529a41207db</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmr109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmr109%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520lysine%2520methylation%2520and%2520demethylation%2520in%2520cancers%26jtitle%3DActa%2520Biochim.%2520Biophys.%2520Sin.%26date%3D2012%26volume%3D44%26spage%3D70%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of protein arginine methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=P.+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+allosteric+inhibitor+of+protein+arginine+methyltransferase+3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0lj71dvTsg4_4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0li9Y1ViKcrfuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Yost, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targets in epigenetics: inhibiting the methyl writers of the histone code</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.2174%2F1875397301005010072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21966347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=72-84&author=J.+M.+Yostauthor=I.+Korboukhauthor=F.+Liuauthor=C.+Gaoauthor=J.+Jin&title=Targets+in+epigenetics%3A+inhibiting+the+methyl+writers+of+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targets in epigenetics: inhibiting the methyl writers of the histone code</span></div><div class="casAuthors">Yost, Julianne M.; Korboukh, Ilia; Liu, Feng; Gao, Cen; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-84</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs) are assocd. with the development of various human diseases, including cancer, inflammation, and psychiatric disorders.  Given the significant role of these proteins in human disease, efforts to discover selective small-mol. inhibitors of these enzymes are quickly gaining momentum.  In this review, we focus on the recent progress in the discovery of selective PKMT and PRMT inhibitors.  A future perspective on developing methyltransferase inhibitors is also offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHi_nKenm0brVg90H21EOLACvtfcHk0li9Y1ViKcrfuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP&md5=996a0411f27dcb4db88e5a4885439d94</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010072%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargets%2520in%2520epigenetics%253A%2520inhibiting%2520the%2520methyl%2520writers%2520of%2520the%2520histone%2520code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D72%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pabst, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krosl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yung, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underhill, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvageau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span> </span><span class="NLM_article-title">The methyltransferase G9a regulates HoxA9-dependent transcription in AML</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=317-327&author=B.+Lehnertzauthor=C.+Pabstauthor=L.+Suauthor=M.+Millerauthor=F.+Liuauthor=L.+Yiauthor=R.+Zhangauthor=J.+Kroslauthor=E.+Yungauthor=J.+Kirschnerauthor=P.+Rostenauthor=T.+M.+Underhillauthor=J.+Jinauthor=J.+Hebertauthor=G.+Sauvageauauthor=R.+K.+Humphriesauthor=F.+M.+Rossi&title=The+methyltransferase+G9a+regulates+HoxA9-dependent+transcription+in+AML"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DPabst%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrosl%26aufirst%3DJ.%26aulast%3DYung%26aufirst%3DE.%26aulast%3DKirschner%26aufirst%3DJ.%26aulast%3DRosten%26aufirst%3DP.%26aulast%3DUnderhill%26aufirst%3DT.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHebert%26aufirst%3DJ.%26aulast%3DSauvageau%26aufirst%3DG.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26atitle%3DThe%2520methyltransferase%2520G9a%2520regulates%2520HoxA9-dependent%2520transcription%2520in%2520AML%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D317%26epage%3D327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skutt-Kakaria, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davison, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ou, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loeb, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georges, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torok-Storb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paddison, P. J.</span><span> </span><span class="NLM_article-title">G9a/GLP-dependent histone H3K9me2 patterning during human hematopoietic stem cell lineage commitment</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">2499</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=2499-2511&author=X.+Chenauthor=K.+Skutt-Kakariaauthor=J.+Davisonauthor=Y.+L.+Ouauthor=E.+Choiauthor=P.+Malikauthor=K.+Loebauthor=B.+Woodauthor=G.+Georgesauthor=B.+Torok-Storbauthor=P.+J.+Paddison&title=G9a%2FGLP-dependent+histone+H3K9me2+patterning+during+human+hematopoietic+stem+cell+lineage+commitment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DSkutt-Kakaria%26aufirst%3DK.%26aulast%3DDavison%26aufirst%3DJ.%26aulast%3DOu%26aufirst%3DY.%2BL.%26aulast%3DChoi%26aufirst%3DE.%26aulast%3DMalik%26aufirst%3DP.%26aulast%3DLoeb%26aufirst%3DK.%26aulast%3DWood%26aufirst%3DB.%26aulast%3DGeorges%26aufirst%3DG.%26aulast%3DTorok-Storb%26aufirst%3DB.%26aulast%3DPaddison%26aufirst%3DP.%2BJ.%26atitle%3DG9a%252FGLP-dependent%2520histone%2520H3K9me2%2520patterning%2520during%2520human%2520hematopoietic%2520stem%2520cell%2520lineage%2520commitment%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D2499%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Antignano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kron, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penrod, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gold, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chenery, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNagny, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levings, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span> </span><span class="NLM_article-title">Methyltransferase G9A regulates T cell differentiation during murine intestinal inflammation</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">1945</span><span class="NLM_x">–</span> <span class="NLM_lpage">1955</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=1945-1955&author=F.+Antignanoauthor=K.+Burrowsauthor=M.+R.+Hughesauthor=J.+M.+Hanauthor=K.+J.+Kronauthor=N.+M.+Penrodauthor=M.+J.+Oudhoffauthor=S.+K.+Wangauthor=P.+H.+Minauthor=M.+J.+Goldauthor=A.+L.+Cheneryauthor=M.+J.+Braamauthor=T.+C.+Fungauthor=F.+M.+Rossiauthor=K.+M.+McNagnyauthor=C.+H.+Arrowsmithauthor=M.+Lupienauthor=M.+K.+Levingsauthor=C.+Zaph&title=Methyltransferase+G9A+regulates+T+cell+differentiation+during+murine+intestinal+inflammation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAntignano%26aufirst%3DF.%26aulast%3DBurrows%26aufirst%3DK.%26aulast%3DHughes%26aufirst%3DM.%2BR.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKron%26aufirst%3DK.%2BJ.%26aulast%3DPenrod%26aufirst%3DN.%2BM.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%2BK.%26aulast%3DMin%26aufirst%3DP.%2BH.%26aulast%3DGold%26aufirst%3DM.%2BJ.%26aulast%3DChenery%26aufirst%3DA.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DFung%26aufirst%3DT.%2BC.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26aulast%3DMcNagny%26aufirst%3DK.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DLupien%26aufirst%3DM.%26aulast%3DLevings%26aufirst%3DM.%2BK.%26aulast%3DZaph%26aufirst%3DC.%26atitle%3DMethyltransferase%2520G9A%2520regulates%2520T%2520cell%2520differentiation%2520during%2520murine%2520intestinal%2520inflammation%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2014%26volume%3D124%26spage%3D1945%26epage%3D1955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical tool for in vitro and in vivo precipitation of lysine methyltransferase g9a</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=549-553&author=K.+D.+Konzeauthor=S.+G.+Pattendenauthor=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=J.+M.+Simonauthor=I.+J.+Davisauthor=M.+Vedadiauthor=J.+Jin&title=A+chemical+tool+for+in+vitro+and+in+vivo+precipitation+of+lysine+methyltransferase+g9a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSimon%26aufirst%3DJ.%2BM.%26aulast%3DDavis%26aufirst%3DI.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520tool%2520for%2520in%2520vitro%2520and%2520in%2520vivo%2520precipitation%2520of%2520lysine%2520methyltransferase%2520g9a%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D549%26epage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Nishioka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valenzuela, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steward, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lis, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">PR-Set7 is a nucleosome-specific methyltransferase that modifies lysine 20 of histone H4 and is associated with silent chromatin</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1201</span><span class="NLM_x">–</span> <span class="NLM_lpage">1213</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=1201-1213&author=K.+Nishiokaauthor=J.+C.+Riceauthor=K.+Sarmaauthor=H.+Erdjument-Bromageauthor=J.+Wernerauthor=Y.+Wangauthor=S.+Chuikovauthor=P.+Valenzuelaauthor=P.+Tempstauthor=R.+Stewardauthor=J.+T.+Lisauthor=C.+D.+Allisauthor=D.+Reinberg&title=PR-Set7+is+a+nucleosome-specific+methyltransferase+that+modifies+lysine+20+of+histone+H4+and+is+associated+with+silent+chromatin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNishioka%26aufirst%3DK.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DSarma%26aufirst%3DK.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DWerner%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DValenzuela%26aufirst%3DP.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSteward%26aufirst%3DR.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DPR-Set7%2520is%2520a%2520nucleosome-specific%2520methyltransferase%2520that%2520modifies%2520lysine%252020%2520of%2520histone%2520H4%2520and%2520is%2520associated%2520with%2520silent%2520chromatin%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D9%26spage%3D1201%26epage%3D1213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ketel, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Purification and functional characterization of SET8, a nucleosomal histone H4-lysine 20-specific methyltransferase</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1086</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2FS0960-9822%2802%2900924-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=12121615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1086-1099&author=J.+Fangauthor=Q.+Fengauthor=C.+S.+Ketelauthor=H.+Wangauthor=R.+Caoauthor=L.+Xiaauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=J.+A.+Simonauthor=Y.+Zhang&title=Purification+and+functional+characterization+of+SET8%2C+a+nucleosomal+histone+H4-lysine+20-specific+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and Functional Characterization of SET8, a Nucleosomal Histone H4-Lysine 20-Specific Methyltransferase</span></div><div class="casAuthors">Fang, Jia; Feng, Qin; Ketel, Carrie S.; Wang, Hengbin; Cao, Ru; Xia, Li; Erdjument-Bromage, Hediye; Tempst, Paul; Simon, Jeffrey A.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1086-1099</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Covalent modifications of histone N-terminal tails play fundamental roles in regulating chromatin structure and function.  Extensive studies have established that acetylation of specific lysine residues in the histone tails plays an important role in transcriptional regulation.  Besides acetylation, recent studies have revealed that histone methylation also has significant effects on heterochromatin formation and transcriptional regulation.  Histone methylation occurs on specific arginine and lysine residues of histones H3 and H4.  Thus far, only 2 residues on histone H4 are known to be methylated.  While H4-arginine 3 (H4-R3) methylation is mediated by PRMT1, the enzyme(s) responsible for H4-lysine 20 (H4-K20) methylation is not known.Results: To gain insight into the function of H4-K20 methylation, we set out to identify the enzyme responsible for this modification.  We purified and cloned a novel human SET domain-contg. protein, named SET8, which specifically methylates H4 at K20.  SET8 is a single subunit enzyme and prefers nucleosomal substrates.  We find that H4-K20 methylation occurs in a wide range of higher eukaryotic organisms and that SET8 homologs exist in C. elegans and Drosophila.  We demonstrate that the Drosophila SET8 homolog has the same substrate specificity as its human counterpart.  Importantly, disruption of SET8 in Drosophila reduces levels of H4-K20 methylation in vivo and results in lethality.  Although H4-K20 methylation does not correlate with gene activity, it appears to be regulated during the cell cycle.Conclusions: We identified and characterized an evolutionarily conserved nucleosomal H4-K20-specific methyltransferase and demonstrated its essential role in Drosophila development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7KOsKzI9f47Vg90H21EOLACvtfcHk0lh9WsJnmE7w6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjsLg%253D&md5=6b0cbdd9ffb2feb7b49017dc045669ce</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2802%2900924-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252802%252900924-7%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DKetel%26aufirst%3DC.%2BS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSimon%26aufirst%3DJ.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPurification%2520and%2520functional%2520characterization%2520of%2520SET8%252C%2520a%2520nucleosomal%2520histone%2520H4-lysine%252020-specific%2520methyltransferase%26jtitle%3DCurr.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D1086%26epage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Beck, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">PR-Set7 and H4K20me1: at the crossroads of genome integrity, cell cycle, chromosome condensation, and transcription</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=325-337&author=D.+B.+Beckauthor=H.+Odaauthor=S.+S.+Shenauthor=D.+Reinberg&title=PR-Set7+and+H4K20me1%3A+at+the+crossroads+of+genome+integrity%2C+cell+cycle%2C+chromosome+condensation%2C+and+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DD.%2BB.%26aulast%3DOda%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DS.%2BS.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DPR-Set7%2520and%2520H4K20me1%253A%2520at%2520the%2520crossroads%2520of%2520genome%2520integrity%252C%2520cell%2520cycle%252C%2520chromosome%2520condensation%252C%2520and%2520transcription%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D325%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Yang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Y.</span><span> </span><span class="NLM_article-title">SET8 promotes epithelial–mesenchymal transition and confers TWIST dual transcriptional activities</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=110-123&author=F.+Yangauthor=L.+Sunauthor=Q.+Liauthor=X.+Hanauthor=L.+Leiauthor=H.+Zhangauthor=Y.+Shang&title=SET8+promotes+epithelial%E2%80%93mesenchymal+transition+and+confers+TWIST+dual+transcriptional+activities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DShang%26aufirst%3DY.%26atitle%3DSET8%2520promotes%2520epithelial%25E2%2580%2593mesenchymal%2520transition%2520and%2520confers%2520TWIST%2520dual%2520transcriptional%2520activities%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D110%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kachirskaia, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaguchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appella, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span> </span><span class="NLM_article-title">Modulation of p53 function by SET8-mediated methylation at lysine 382</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">636</span><span class="NLM_x">–</span> <span class="NLM_lpage">646</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=636-646&author=X.+Shiauthor=I.+Kachirskaiaauthor=H.+Yamaguchiauthor=L.+E.+Westauthor=H.+Wenauthor=E.+W.+Wangauthor=S.+Duttaauthor=E.+Appellaauthor=O.+Gozani&title=Modulation+of+p53+function+by+SET8-mediated+methylation+at+lysine+382"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DX.%26aulast%3DKachirskaia%26aufirst%3DI.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DE.%2BW.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DModulation%2520of%2520p53%2520function%2520by%2520SET8-mediated%2520methylation%2520at%2520lysine%2520382%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D636%26epage%3D646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akasu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnuma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atomi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Histone lysine methyltransferase SETD8 promotes carcinogenesis by deregulating PCNA expression</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">3217</span><span class="NLM_x">–</span> <span class="NLM_lpage">3227</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=3217-3227&author=M.+Takawaauthor=H.+S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=M.+Kogureauthor=Y.+Yamaneauthor=Y.+Iwaiauthor=K.+Maejimaauthor=K.+Uedaauthor=A.+Masudaauthor=N.+Dohmaeauthor=H.+I.+Fieldauthor=T.+Tsunodaauthor=T.+Kobayashiauthor=T.+Akasuauthor=M.+Sugiyamaauthor=S.+Ohnumaauthor=Y.+Atomiauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Histone+lysine+methyltransferase+SETD8+promotes+carcinogenesis+by+deregulating+PCNA+expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DAkasu%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DOhnuma%26aufirst%3DS.%26aulast%3DAtomi%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DHistone%2520lysine%2520methyltransferase%2520SETD8%2520promotes%2520carcinogenesis%2520by%2520deregulating%2520PCNA%2520expression%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D3217%26epage%3D3227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nady, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">L3MBTL1 recognition of mono- and dimethylated histones</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1229</span><span class="NLM_x">–</span> <span class="NLM_lpage">1230</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnsmb1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=18026117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1229-1230&author=J.+Minauthor=A.+Allali-Hassaniauthor=N.+Nadyauthor=C.+Qiauthor=H.+Ouyangauthor=Y.+Liuauthor=F.+MacKenzieauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=L3MBTL1+recognition+of+mono-+and+dimethylated+histones"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">L3MBTL1 recognition of mono- and dimethylated histones</span></div><div class="casAuthors">Min, Jinrong; Allali-Hassani, Abdellah; Nady, Nataliya; Qi, Chao; Ouyang, Hui; Liu, Yongsong; MacKenzie, Farrell; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1229-1230</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Crystal structures of the L3MBTL1 MBT repeats in complex with histone H4 peptides dimethylated on Lys20 (H4K20me2) show that only the second of the three MBT repeats can bind mono- and dimethylated histone peptides.  Its binding pocket has similarities to that of 53BP1 and is able to recognize the degree of histone lysine methylation.  An unexpected mode of peptide-mediated dimerization suggests a possible mechanism for chromatin compaction by L3MBTL1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyCuRY6dV9-bVg90H21EOLACvtfcHk0lhJhBiYYPxHnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP&md5=1f1072bf117cb7459a644907ca66b74b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1340%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DNady%26aufirst%3DN.%26aulast%3DQi%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMacKenzie%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DL3MBTL1%2520recognition%2520of%2520mono-%2520and%2520dimethylated%2520histones%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1229%26epage%3D1230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provencher, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span> </span><span class="NLM_article-title">Accessing protein methyltransferase and demethylase enzymology using microfluidic capillary electrophoresis</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">695</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.chembiol.2010.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=20659682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptleitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=695-704&author=T.+J.+Wigleauthor=L.+M.+Provencherauthor=J.+L.+Norrisauthor=J.+Jinauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=W.+P.+Janzen&title=Accessing+protein+methyltransferase+and+demethylase+enzymology+using+microfluidic+capillary+electrophoresis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Accessing Protein Methyltransferase and Demethylase Enzymology Using Microfluidic Capillary Electrophoresis</span></div><div class="casAuthors">Wigle, Tim J.; Provencher, Laurel M.; Norris, Jacqueline L.; Jin, Jian; Brown, Peter J.; Frye, Stephen V.; Janzen, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">695-704</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The discovery of small mols. targeting the >80 enzymes that add (methyltransferases) or remove (demethylases) Me marks from lysine and arginine residues, most notably present in histone tails, may yield unprecedented chemotherapeutic agents and facilitate regenerative medicine.  To better enable chem. exploration of these proteins, a highly quant. microfluidic capillary electrophoresis assay was developed to enable full mechanistic studies of these enzymes and the kinetics of their inhibition.  This technol. separates small biomols., i.e., peptides, based on their charge-to-mass ratio.  Methylation, however, does not alter the charge of peptide substrates.  To overcome this limitation, a methylation-sensitive endoproteinase strategy was employed to sep. methylated from unmethylated peptides.  The assay was validated on a lysine methyltransferase (G9a) and a lysine demethylase (LSD1) and was employed to investigate the inhibition of G9a by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPcF6--FJ3u7Vg90H21EOLACvtfcHk0liVwqZoxv86hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptleitL8%253D&md5=fe13fe9e2120b40edb4ea90d67cb529e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DProvencher%26aufirst%3DL.%2BM.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26atitle%3DAccessing%2520protein%2520methyltransferase%2520and%2520demethylase%2520enzymology%2520using%2520microfluidic%2520capillary%2520electrophoresis%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D695%26epage%3D704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeon, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pae, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. Y.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel T-type calcium channel blockers</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">–</span> <span class="NLM_lpage">475</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=471-475&author=J.+Y.+Choiauthor=H.+N.+Seoauthor=M.+J.+Leeauthor=S.+J.+Parkauthor=J.+Y.+Jeonauthor=J.+H.+Kangauthor=A.+N.+Paeauthor=H.+Rhimauthor=J.+Y.+Lee&title=Synthesis+and+biological+evaluation+of+novel+T-type+calcium+channel+blockers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DSeo%26aufirst%3DH.%2BN.%26aulast%3DLee%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DJeon%26aufirst%3DJ.%2BY.%26aulast%3DKang%26aufirst%3DJ.%2BH.%26aulast%3DPae%26aufirst%3DA.%2BN.%26aulast%3DRhim%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520T-type%2520calcium%2520channel%2520blockers%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D471%26epage%3D475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Structural and functional analysis of SET8, a histone H4 Lys-20 methyltransferase</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1455</span><span class="NLM_x">–</span> <span class="NLM_lpage">1465</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1101%2Fgad.1318405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=15933070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=1455-1465&author=J.+F.+Coutureauthor=E.+Collazoauthor=J.+S.+Brunzelleauthor=R.+C.+Trievel&title=Structural+and+functional+analysis+of+SET8%2C+a+histone+H4+Lys-20+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional analysis of SET8, a histone H4 Lys-20 methyltransferase</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Brunzelle, Joseph S.; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1455-1465</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">SET8 (also known as PR-SET7) is a histone H4-Lys-20-specific methyltransferase that is implicated in cell-cycle-dependent transcriptional silencing and mitotic regulation in metazoans.  Herein we report the crystal structure of human SET8 (hSET8) bound to a histone H4 peptide bearing Lys-20 and the product cofactor S-adenosylhomocysteine.  Histone H4 intercalates in the substrate-binding cleft as an extended parallel β-strand.  Residues preceding Lys-20 in H4 engage in an extensive array of salt bridge, hydrogen bond, and van der Waals interactions with hSET8, while the C-terminal residues bind through predominantly hydrophobic interactions.  Mutational anal. of both the substrate-binding cleft and histone H4 reveals that interactions with residues in the N and C termini of the H4 peptide are crit. for conferring substrate specificity.  Finally, anal. of the product specificity indicates that hSET8 is a monomethylase, consistent with its role in the maintenance of Lys-20 monomethylation during cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphqNcPOAjIbbVg90H21EOLACvtfcHk0liVwqZoxv86hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D&md5=26dd3cdbaee412a5904ba8268b8efb40</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1101%2Fgad.1318405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1318405%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520and%2520functional%2520analysis%2520of%2520SET8%252C%2520a%2520histone%2520H4%2520Lys-20%2520methyltransferase%26jtitle%3DGenes%2520Dev.%26date%3D2005%26volume%3D19%26spage%3D1455%26epage%3D1465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibition of MLL activity by disruption of its interaction with WDR5</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=151-159&author=G.+Senisterraauthor=H.+Wuauthor=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=D.+Barsyte-Lovejoyauthor=L.+Dombrovskiauthor=A.+Dongauthor=K.+T.+Nguyenauthor=D.+Smilauthor=Y.+Bolshanauthor=T.+Hajianauthor=H.+Heauthor=A.+Seitovaauthor=I.+Chauauthor=F.+Liauthor=G.+Podaauthor=J.+F.+Coutureauthor=P.+J.+Brownauthor=R.+Al-Awarauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Small-molecule+inhibition+of+MLL+activity+by+disruption+of+its+interaction+with+WDR5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSeitova%26aufirst%3DA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520MLL%2520activity%2520by%2520disruption%2520of%2520its%2520interaction%2520with%2520WDR5%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D151%26epage%3D159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 51 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Leen N. Vanheer, <span class="NLM_string-name hlFld-ContribAuthor">Björn F.C. Kafsack</span>. </span><span class="cited-content_cbyCitation_article-title">Activity Comparison of Epigenetic Modulators against the Hemoprotozoan Parasites Babesia divergens and Plasmodium falciparum. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.0c00853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.0c00853%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DActivity%252BComparison%252Bof%252BEpigenetic%252BModulators%252Bagainst%252Bthe%252BHemoprotozoan%252BParasites%252BBabesia%252Bdivergens%252Band%252BPlasmodium%252Bfalciparum%26aulast%3DVanheer%26aufirst%3DLeen%2BN.%26date%3D2021%26date%3D2020%26date%3D04122020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Russell A.  Judge</span>, <span class="hlFld-ContribAuthor ">Haizhong  Zhu</span>, <span class="hlFld-ContribAuthor ">Anup K.  Upadhyay</span>, <span class="hlFld-ContribAuthor ">Pierre M.  Bodelle</span>, <span class="hlFld-ContribAuthor ">Charles W.  Hutchins</span>, <span class="hlFld-ContribAuthor ">Maricel  Torrent</span>, <span class="hlFld-ContribAuthor ">Violeta L.  Marin</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">William N.  Pappano</span>, <span class="hlFld-ContribAuthor ">Chaohong  Sun</span>, and <span class="hlFld-ContribAuthor ">Andrew M.  Petros</span>  . </span><span class="cited-content_cbyCitation_article-title">Turning a Substrate Peptide into a Potent Inhibitor for the Histone Methyltransferase SETD8. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>7 </em>
                                    (12)
                                     , 1102-1106. <a href="https://doi.org/10.1021/acsmedchemlett.6b00303" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.6b00303</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.6b00303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.6b00303%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DTurning%252Ba%252BSubstrate%252BPeptide%252Binto%252Ba%252BPotent%252BInhibitor%252Bfor%252Bthe%252BHistone%252BMethyltransferase%252BSETD8%26aulast%3DJudge%26aufirst%3DRussell%2BA.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D03082016%26date%3D11102016%26date%3D17102016%26date%3D08122016%26date%3D11102016%26volume%3D7%26issue%3D12%26spage%3D1102%26epage%3D1106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Wolfram  Tempel</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Fabio  Pittella-Silva</span>, <span class="hlFld-ContribAuthor ">Jason  Shao</span>, <span class="hlFld-ContribAuthor ">Junyi  Wang</span>, <span class="hlFld-ContribAuthor ">Minkui  Luo</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Covalent Inhibitor of the SET Domain-Containing Protein 8 (SETD8) Lysine Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (21)
                                     , 9881-9889. <a href="https://doi.org/10.1021/acs.jmedchem.6b01244" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01244%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BCovalent%252BInhibitor%252Bof%252Bthe%252BSET%252BDomain-Containing%252BProtein%252B8%252B%252528SETD8%252529%252BLysine%252BMethyltransferase%26aulast%3DButler%26aufirst%3DKyle%2BV.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D16082016%26date%3D02112016%26date%3D10112016%26volume%3D59%26issue%3D21%26spage%3D9881%26epage%3D9889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David E.  Williams</span>, <span class="hlFld-ContribAuthor ">Fanny  Izard</span>, <span class="hlFld-ContribAuthor ">Stéphanie  Arnould</span>, <span class="hlFld-ContribAuthor ">Doralyn S.  Dalisay</span>, <span class="hlFld-ContribAuthor ">Cholpisut  Tantapakul</span>, <span class="hlFld-ContribAuthor ">Wisanu  Maneerat</span>, <span class="hlFld-ContribAuthor ">Teatulohi  Matainaho</span>, <span class="hlFld-ContribAuthor ">Eric  Julien</span>, and <span class="hlFld-ContribAuthor ">Raymond J.  Andersen</span>  . </span><span class="cited-content_cbyCitation_article-title">Structures of Nahuoic Acids B–E Produced in Culture by a Streptomyces sp. Isolated from a Marine Sediment and Evidence for the Inhibition of the Histone Methyl Transferase SETD8 in Human Cancer Cells by Nahuoic Acid A. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2016,</strong> <em>81 </em>
                                    (4)
                                     , 1324-1332. <a href="https://doi.org/10.1021/acs.joc.5b02569" title="DOI URL">https://doi.org/10.1021/acs.joc.5b02569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.5b02569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.5b02569%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DStructures%252Bof%252BNahuoic%252BAcids%252BB%2525E2%252580%252593E%252BProduced%252Bin%252BCulture%252Bby%252Ba%252BStreptomyces%252Bsp.%252BIsolated%252Bfrom%252Ba%252BMarine%252BSediment%252Band%252BEvidence%252Bfor%252Bthe%252BInhibition%252Bof%252Bthe%252BHistone%252BMethyl%252BTransferase%252BSETD8%252Bin%252BHuman%252BCancer%252BCells%252Bby%252BNahuoic%252BAcid%252BA%26aulast%3DWilliams%26aufirst%3DDavid%2BE.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D08112015%26date%3D01022016%26date%3D19022016%26date%3D27012016%26volume%3D81%26issue%3D4%26spage%3D1324%26epage%3D1332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cen  Gao</span>, <span class="hlFld-ContribAuthor ">Brandon J.  Margolis</span>, <span class="hlFld-ContribAuthor ">John M.  Strelow</span>, <span class="hlFld-ContribAuthor ">Lewis R.  Vidler</span>, and <span class="hlFld-ContribAuthor ">Mary M.  Mader</span>  . </span><span class="cited-content_cbyCitation_article-title">Beyond PAINs: Chemotype Sensitivity of Protein Methyltransferases in Screens. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>7 </em>
                                    (2)
                                     , 156-161. <a href="https://doi.org/10.1021/acsmedchemlett.5b00375" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.5b00375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.5b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.5b00375%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DBeyond%252BPAINs%25253A%252BChemotype%252BSensitivity%252Bof%252BProtein%252BMethyltransferases%252Bin%252BScreens%26aulast%3DGao%26aufirst%3DCen%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D28092015%26date%3D19112015%26date%3D30112015%26date%3D11022016%26date%3D19112015%26volume%3D7%26issue%3D2%26spage%3D156%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fanwang  Meng</span>, <span class="hlFld-ContribAuthor ">Sufang  Cheng</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Shien  Liu</span>, <span class="hlFld-ContribAuthor ">Yan  Liu</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Shijie  Chen</span>, <span class="hlFld-ContribAuthor ">Junyan  Lu</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Linjuan  Li</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Zhe  Shi</span>, <span class="hlFld-ContribAuthor ">Yu  Zhou</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Wencong  Lu</span>, <span class="hlFld-ContribAuthor ">Hong  Liu</span>, and <span class="hlFld-ContribAuthor ">Cheng  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (20)
                                     , 8166-8181. <a href="https://doi.org/10.1021/acs.jmedchem.5b01154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252BNovel%25252C%252BSelective%252BHistone%252BMethyltransferase%252BSET7%252BInhibitors%252Bby%252BPharmacophore-%252Band%252BDocking-Based%252BVirtual%252BScreening%26aulast%3DMeng%26aufirst%3DFanwang%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D06072015%26date%3D30092015%26date%3D22102015%26date%3D21092015%26volume%3D58%26issue%3D20%26spage%3D8166%26epage%3D8181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Kyle D.  Konze</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Selective Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (4)
                                     , 1596-1629. <a href="https://doi.org/10.1021/jm501234a" title="DOI URL">https://doi.org/10.1021/jm501234a</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm501234a%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D12082014%26date%3D02122014%26date%3D26022015%26date%3D19112014%26volume%3D58%26issue%3D4%26spage%3D1596%26epage%3D1629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gil  Blum</span>, <span class="hlFld-ContribAuthor ">Glorymar  Ibáñez</span>, <span class="hlFld-ContribAuthor ">Xiangjun  Rao</span>, <span class="hlFld-ContribAuthor ">David  Shum</span>, <span class="hlFld-ContribAuthor ">Constantin  Radu</span>, <span class="hlFld-ContribAuthor ">Hakim  Djaballah</span>, <span class="hlFld-ContribAuthor ">Judd C.  Rice</span>, and <span class="hlFld-ContribAuthor ">Minkui  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Inhibitors of SETD8 with Cellular Activity. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2014,</strong> <em>9 </em>
                                    (11)
                                     , 2471-2478. <a href="https://doi.org/10.1021/cb500515r" title="DOI URL">https://doi.org/10.1021/cb500515r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb500515r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb500515r%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DSmall-Molecule%252BInhibitors%252Bof%252BSETD8%252Bwith%252BCellular%252BActivity%26aulast%3DBlum%26aufirst%3DGil%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D15042014%26date%3D19082014%26date%3D03092014%26date%3D21112014%26date%3D19082014%26volume%3D9%26issue%3D11%26spage%3D2471%26epage%3D2478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamakoti P.  Bhat</span>, <span class="hlFld-ContribAuthor ">H.  Ümit Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Or  Gozani</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics and beyond: targeting writers of protein lysine methylation to treat disease. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (4)
                                     , 265-286. <a href="https://doi.org/10.1038/s41573-020-00108-x" title="DOI URL">https://doi.org/10.1038/s41573-020-00108-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-020-00108-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-020-00108-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DEpigenetics%252Band%252Bbeyond%25253A%252Btargeting%252Bwriters%252Bof%252Bprotein%252Blysine%252Bmethylation%252Bto%252Btreat%252Bdisease%26aulast%3DBhat%26aufirst%3DKamakoti%2BP.%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D4%26spage%3D265%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuki  Mizukawa</span>, <span class="hlFld-ContribAuthor ">Mayumi  Ikegami-Kawai</span>, <span class="hlFld-ContribAuthor ">Masako  Horiuchi</span>, <span class="hlFld-ContribAuthor ">Marcel  Kaiser</span>, <span class="hlFld-ContribAuthor ">Masayoshi  Kojima</span>, <span class="hlFld-ContribAuthor ">Seiki  Sakanoue</span>, <span class="hlFld-ContribAuthor ">Seiya  Miyagi</span>, <span class="hlFld-ContribAuthor ">Christian  Nanga Chick</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Togashi</span>, <span class="hlFld-ContribAuthor ">Masayoshi  Tsubuki</span>, <span class="hlFld-ContribAuthor ">Masataka  Ihara</span>, <span class="hlFld-ContribAuthor ">Toyonobu  Usuki</span>, <span class="hlFld-ContribAuthor ">Isamu  Itoh</span>. </span><span class="cited-content_cbyCitation_article-title">Quest for a potent antimalarial drug lead: Synthesis and evaluation of 6,7-dimethoxyquinazoline-2,4-diamines. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>33 </em>, 116018. <a href="https://doi.org/10.1016/j.bmc.2021.116018" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116018%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DQuest%252Bfor%252Ba%252Bpotent%252Bantimalarial%252Bdrug%252Blead%25253A%252BSynthesis%252Band%252Bevaluation%252Bof%252B6%25252C7-dimethoxyquinazoline-2%25252C4-diamines%26aulast%3DMizukawa%26aufirst%3DYuki%26date%3D2021%26volume%3D33%26spage%3D116018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Pisapia</span>, <span class="hlFld-ContribAuthor ">Monica  Viviano</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Tanja  Bartoschik</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Microscale Thermophoresis-Based Method for Screening and Characterizing Inhibitors of the Methyl-Lysine Reader Protein MRG15. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (1)
                                     , 77-87. <a href="https://doi.org/10.1177/2472555220949166" title="DOI URL">https://doi.org/10.1177/2472555220949166</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555220949166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555220949166%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DDevelopment%252Bof%252Ba%252BMicroscale%252BThermophoresis-Based%252BMethod%252Bfor%252BScreening%252Band%252BCharacterizing%252BInhibitors%252Bof%252Bthe%252BMethyl-Lysine%252BReader%252BProtein%252BMRG15%26aulast%3DFeoli%26aufirst%3DAlessandra%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D1%26spage%3D77%26epage%3D87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jianhua  Wu</span>, <span class="hlFld-ContribAuthor ">Kuangyuan  Qiao</span>, <span class="hlFld-ContribAuthor ">Yanming  Du</span>, <span class="hlFld-ContribAuthor ">Xiaoyun  Zhang</span>, <span class="hlFld-ContribAuthor ">Haichao  Cheng</span>, <span class="hlFld-ContribAuthor ">Li  Peng</span>, <span class="hlFld-ContribAuthor ">Zhanjun  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Downregulation of histone methyltransferase SET8 inhibits progression of hepatocellular carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-61402-7" title="DOI URL">https://doi.org/10.1038/s41598-020-61402-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-61402-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-61402-7%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DDownregulation%252Bof%252Bhistone%252Bmethyltransferase%252BSET8%252Binhibits%252Bprogression%252Bof%252Bhepatocellular%252Bcarcinoma%26aulast%3DWu%26aufirst%3DJianhua%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacquelyn A.  Myers</span>, <span class="hlFld-ContribAuthor ">Tyler  Couch</span>, <span class="hlFld-ContribAuthor ">Zachary  Murphy</span>, <span class="hlFld-ContribAuthor ">Jeffrey  Malik</span>, <span class="hlFld-ContribAuthor ">Michael  Getman</span>, <span class="hlFld-ContribAuthor ">Laurie A.  Steiner</span>. </span><span class="cited-content_cbyCitation_article-title">The histone methyltransferase Setd8 alters the chromatin landscape and regulates the expression of key transcription factors during erythroid differentiation. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics & Chromatin</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13072-020-00337-9" title="DOI URL">https://doi.org/10.1186/s13072-020-00337-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13072-020-00337-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13072-020-00337-9%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%2520%2526%2520Chromatin%26atitle%3DThe%252Bhistone%252Bmethyltransferase%252BSetd8%252Balters%252Bthe%252Bchromatin%252Blandscape%252Band%252Bregulates%252Bthe%252Bexpression%252Bof%252Bkey%252Btranscription%252Bfactors%252Bduring%252Berythroid%252Bdifferentiation%26aulast%3DMyers%26aufirst%3DJacquelyn%2BA.%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miku  Wada</span>, <span class="hlFld-ContribAuthor ">Asako  Kukita</span>, <span class="hlFld-ContribAuthor ">Kenbun  Sone</span>, <span class="hlFld-ContribAuthor ">Ryuji  Hamamoto</span>, <span class="hlFld-ContribAuthor ">Syuzo  Kaneko</span>, <span class="hlFld-ContribAuthor ">Masaaki  Komatsu</span>, <span class="hlFld-ContribAuthor ">Yu  Takahashi</span>, <span class="hlFld-ContribAuthor ">Futaba  Inoue</span>, <span class="hlFld-ContribAuthor ">Machiko  Kojima</span>, <span class="hlFld-ContribAuthor ">Harunori  Honjoh</span>, <span class="hlFld-ContribAuthor ">Ayumi  Taguchi</span>, <span class="hlFld-ContribAuthor ">Tomoko  Kashiyama</span>, <span class="hlFld-ContribAuthor ">Yuichiro  Miyamoto</span>, <span class="hlFld-ContribAuthor ">Michihiro  Tanikawa</span>, <span class="hlFld-ContribAuthor ">Tetsushi  Tsuruga</span>, <span class="hlFld-ContribAuthor ">Mayuyo  Mori-Uchino</span>, <span class="hlFld-ContribAuthor ">Osamu  Wada-Hiraike</span>, <span class="hlFld-ContribAuthor ">Yutaka  Osuga</span>, <span class="hlFld-ContribAuthor ">Tomoyuki  Fujii</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Modifier SETD8 as a Therapeutic Target for High-Grade Serous Ovarian Cancer. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 1686. <a href="https://doi.org/10.3390/biom10121686" title="DOI URL">https://doi.org/10.3390/biom10121686</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom10121686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom10121686%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DEpigenetic%252BModifier%252BSETD8%252Bas%252Ba%252BTherapeutic%252BTarget%252Bfor%252BHigh-Grade%252BSerous%252BOvarian%252BCancer%26aulast%3DWada%26aufirst%3DMiku%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D1686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Keita  Ugai</span>, <span class="hlFld-ContribAuthor ">Shuichi  Matsuda</span>, <span class="hlFld-ContribAuthor ">Hideki  Mikami</span>, <span class="hlFld-ContribAuthor ">Ayako  Shimada</span>, <span class="hlFld-ContribAuthor ">Tomoko  Misawa</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Nakamura</span>, <span class="hlFld-ContribAuthor ">Koichiro  Tatsumi</span>, <span class="hlFld-ContribAuthor ">Masahiko  Hatano</span>, <span class="hlFld-ContribAuthor ">Toshihiko  Murayama</span>, <span class="hlFld-ContribAuthor ">Yoshitoshi  Kasuya</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of the SET8 Pathway Ameliorates Lung Fibrosis Even Through Fibroblast Dedifferentiation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fmolb.2020.00192" title="DOI URL">https://doi.org/10.3389/fmolb.2020.00192</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2020.00192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2020.00192%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DInhibition%252Bof%252Bthe%252BSET8%252BPathway%252BAmeliorates%252BLung%252BFibrosis%252BEven%252BThrough%252BFibroblast%252BDedifferentiation%26aulast%3DUgai%26aufirst%3DKeita%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Radoslav  Aleksandrov</span>, <span class="hlFld-ContribAuthor ">Rossitsa  Hristova</span>, <span class="hlFld-ContribAuthor ">Stoyno  Stoynov</span>, <span class="hlFld-ContribAuthor ">Anastas  Gospodinov</span>. </span><span class="cited-content_cbyCitation_article-title">The Chromatin Response to Double-Strand DNA Breaks and Their Repair. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2020,</strong> <em>9 </em>
                                    (8)
                                     , 1853. <a href="https://doi.org/10.3390/cells9081853" title="DOI URL">https://doi.org/10.3390/cells9081853</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells9081853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells9081853%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DThe%252BChromatin%252BResponse%252Bto%252BDouble-Strand%252BDNA%252BBreaks%252Band%252BTheir%252BRepair%26aulast%3DAleksandrov%26aufirst%3DRadoslav%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D8%26spage%3D1853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hope S.  Rugo</span>, <span class="hlFld-ContribAuthor ">Ira  Jacobs</span>, <span class="hlFld-ContribAuthor ">Shikhar  Sharma</span>, <span class="hlFld-ContribAuthor ">Frank  Scappaticci</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Paul</span>, <span class="hlFld-ContribAuthor ">Kristen  Jensen-Pergakes</span>, <span class="hlFld-ContribAuthor ">Gabriel G.  Malouf</span>. </span><span class="cited-content_cbyCitation_article-title">The Promise for Histone Methyltransferase Inhibitors for Epigenetic Therapy in Clinical Oncology: A Narrative Review. </span><span class="cited-content_cbyCitation_journal-name">Advances in Therapy</span><span> <strong>2020,</strong> <em>37 </em>
                                    (7)
                                     , 3059-3082. <a href="https://doi.org/10.1007/s12325-020-01379-x" title="DOI URL">https://doi.org/10.1007/s12325-020-01379-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12325-020-01379-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12325-020-01379-x%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvances%2520in%2520Therapy%26atitle%3DThe%252BPromise%252Bfor%252BHistone%252BMethyltransferase%252BInhibitors%252Bfor%252BEpigenetic%252BTherapy%252Bin%252BClinical%252BOncology%25253A%252BA%252BNarrative%252BReview%26aulast%3DRugo%26aufirst%3DHope%2BS.%26date%3D2020%26date%3D2020%26volume%3D37%26issue%3D7%26spage%3D3059%26epage%3D3082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Richart</span>, <span class="hlFld-ContribAuthor ">Raphaël  Margueron</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging histone methyltransferases in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>56 </em>, 51-62. <a href="https://doi.org/10.1016/j.cbpa.2019.11.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2019.11.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2019.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2019.11.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDrugging%252Bhistone%252Bmethyltransferases%252Bin%252Bcancer%26aulast%3DRichart%26aufirst%3DLaia%26date%3D2020%26volume%3D56%26spage%3D51%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daiki  SHIKATA</span>, <span class="hlFld-ContribAuthor ">Takuto  YAMAMOTO</span>, <span class="hlFld-ContribAuthor ">Shinnosuke  HONDA</span>, <span class="hlFld-ContribAuthor ">Shuntaro  IKEDA</span>, <span class="hlFld-ContribAuthor ">Naojiro  MINAMI</span>. </span><span class="cited-content_cbyCitation_article-title">H4K20 monomethylation inhibition causes loss of genomic integrity in mouse preimplantation embryos. </span><span class="cited-content_cbyCitation_journal-name">Journal of Reproduction and Development</span><span> <strong>2020,</strong> <em>66 </em>
                                    (5)
                                     , 411-419. <a href="https://doi.org/10.1262/jrd.2020-036" title="DOI URL">https://doi.org/10.1262/jrd.2020-036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1262/jrd.2020-036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1262%2Fjrd.2020-036%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Reproduction%2520and%2520Development%26atitle%3DH4K20%252Bmonomethylation%252Binhibition%252Bcauses%252Bloss%252Bof%252Bgenomic%252Bintegrity%252Bin%252Bmouse%252Bpreimplantation%252Bembryos%26aulast%3DSHIKATA%26aufirst%3DDaiki%26date%3D2020%26volume%3D66%26issue%3D5%26spage%3D411%26epage%3D419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shi  Chen</span>, <span class="hlFld-ContribAuthor ">Rafal P  Wiewiora</span>, <span class="hlFld-ContribAuthor ">Fanwang  Meng</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Kun  Qian</span>, <span class="hlFld-ContribAuthor ">Hao  Hu</span>, <span class="hlFld-ContribAuthor ">Hua  Zou</span>, <span class="hlFld-ContribAuthor ">Junyi  Wang</span>, <span class="hlFld-ContribAuthor ">Shijie  Fan</span>, <span class="hlFld-ContribAuthor ">Gil  Blum</span>, <span class="hlFld-ContribAuthor ">Fabio  Pittella-Silva</span>, <span class="hlFld-ContribAuthor ">Kyle A  Beauchamp</span>, <span class="hlFld-ContribAuthor ">Wolfram  Tempel</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Robert J  Skene</span>, <span class="hlFld-ContribAuthor ">Yujun George  Zheng</span>, <span class="hlFld-ContribAuthor ">Peter J  Brown</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">John D  Chodera</span>, <span class="hlFld-ContribAuthor ">Minkui  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">The dynamic conformational landscape of the protein methyltransferase SETD8. </span><span class="cited-content_cbyCitation_journal-name">eLife</span><span> <strong>2019,</strong> <em>8 </em><a href="https://doi.org/10.7554/eLife.45403" title="DOI URL">https://doi.org/10.7554/eLife.45403</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7554/eLife.45403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7554%2FeLife.45403%26sid%3Dliteratum%253Aachs%26jtitle%3DeLife%26atitle%3DThe%252Bdynamic%252Bconformational%252Blandscape%252Bof%252Bthe%252Bprotein%252Bmethyltransferase%252BSETD8%26aulast%3DChen%26aufirst%3DShi%26date%3D2019%26date%3D2019%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frida S.  Herrera-Vázquez</span>, <span class="hlFld-ContribAuthor ">Francisco  Hernández-Luis</span>, <span class="hlFld-ContribAuthor ">José L.  Medina Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Quinazolines as inhibitors of chromatin-associated proteins in histones. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2019,</strong> <em>28 </em>
                                    (4)
                                     , 395-416. <a href="https://doi.org/10.1007/s00044-019-02300-0" title="DOI URL">https://doi.org/10.1007/s00044-019-02300-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-019-02300-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-019-02300-0%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DQuinazolines%252Bas%252Binhibitors%252Bof%252Bchromatin-associated%252Bproteins%252Bin%252Bhistones%26aulast%3DHerrera-V%25C3%25A1zquez%26aufirst%3DFrida%2BS.%26date%3D2019%26date%3D2019%26volume%3D28%26issue%3D4%26spage%3D395%26epage%3D416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guillaume  Guignard</span>, <span class="hlFld-ContribAuthor ">Núria  Llor</span>, <span class="hlFld-ContribAuthor ">David  Pubill</span>, <span class="hlFld-ContribAuthor ">Joan  Bosch</span>, <span class="hlFld-ContribAuthor ">Mercedes  Amat</span>. </span><span class="cited-content_cbyCitation_article-title">Enantioselective Synthesis of the Ethyl Analog of the Marine Alkaloid Haliclorensin C. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (6)
                                     , 1069. <a href="https://doi.org/10.3390/molecules24061069" title="DOI URL">https://doi.org/10.3390/molecules24061069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24061069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24061069%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DEnantioselective%252BSynthesis%252Bof%252Bthe%252BEthyl%252BAnalog%252Bof%252Bthe%252BMarine%252BAlkaloid%252BHaliclorensin%252BC%26aulast%3DGuignard%26aufirst%3DGuillaume%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D6%26spage%3D1069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Small‐Molecule Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 201-220. <a href="https://doi.org/10.1002/9783527809257.ch9" title="DOI URL">https://doi.org/10.1002/9783527809257.ch9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch9%26sid%3Dliteratum%253Aachs%26atitle%3DSelective%252BSmall%2525E2%252580%252590Molecule%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2019%26date%3D2019%26spage%3D201%26epage%3D220%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bethany  Veo</span>, <span class="hlFld-ContribAuthor ">Etienne  Danis</span>, <span class="hlFld-ContribAuthor ">Angela  Pierce</span>, <span class="hlFld-ContribAuthor ">Ismail  Sola</span>, <span class="hlFld-ContribAuthor ">Dong  Wang</span>, <span class="hlFld-ContribAuthor ">Nicholas K.  Foreman</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Natalie  Serkova</span>, <span class="hlFld-ContribAuthor ">Sujatha  Venkataraman</span>, <span class="hlFld-ContribAuthor ">Rajeev  Vibhakar</span>. </span><span class="cited-content_cbyCitation_article-title">Combined functional genomic and chemical screens identify SETD8 as a therapeutic target in MYC-driven medulloblastoma. </span><span class="cited-content_cbyCitation_journal-name">JCI Insight</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1172/jci.insight.122933" title="DOI URL">https://doi.org/10.1172/jci.insight.122933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1172/jci.insight.122933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1172%2Fjci.insight.122933%26sid%3Dliteratum%253Aachs%26jtitle%3DJCI%2520Insight%26atitle%3DCombined%252Bfunctional%252Bgenomic%252Band%252Bchemical%252Bscreens%252Bidentify%252BSETD8%252Bas%252Ba%252Btherapeutic%252Btarget%252Bin%252BMYC-driven%252Bmedulloblastoma%26aulast%3DVeo%26aufirst%3DBethany%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dylan M.  Marchione</span>, <span class="hlFld-ContribAuthor ">Benjamin A.  Garcia</span>, <span class="hlFld-ContribAuthor ">John  Wojcik</span>. </span><span class="cited-content_cbyCitation_article-title">Proteomic approaches for cancer epigenetics research. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Proteomics</span><span> <strong>2019,</strong> <em>16 </em>
                                    (1)
                                     , 33-47. <a href="https://doi.org/10.1080/14789450.2019.1550363" title="DOI URL">https://doi.org/10.1080/14789450.2019.1550363</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14789450.2019.1550363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14789450.2019.1550363%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Proteomics%26atitle%3DProteomic%252Bapproaches%252Bfor%252Bcancer%252Bepigenetics%252Bresearch%26aulast%3DMarchione%26aufirst%3DDylan%2BM.%26date%3D2019%26date%3D2018%26volume%3D16%26issue%3D1%26spage%3D33%26epage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Shuichi  Mori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Chemical Tools for the Regulation and Study of Protein Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1745-1759. <a href="https://doi.org/10.1002/tcr.201800034" title="DOI URL">https://doi.org/10.1002/tcr.201800034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DRecent%252BAdvances%252Bin%252BChemical%252BTools%252Bfor%252Bthe%252BRegulation%252Band%252BStudy%252Bof%252BProtein%252BLysine%252BMethyltransferases%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1745%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zonghong  Li</span>, <span class="hlFld-ContribAuthor ">Hongyang  Liu</span>, <span class="hlFld-ContribAuthor ">Zhangjing  Niu</span>, <span class="hlFld-ContribAuthor ">Wen  Zhong</span>, <span class="hlFld-ContribAuthor ">Miaomiao  Xue</span>, <span class="hlFld-ContribAuthor ">Jifeng  Wang</span>, <span class="hlFld-ContribAuthor ">Fuquan  Yang</span>, <span class="hlFld-ContribAuthor ">Yue  Zhou</span>, <span class="hlFld-ContribAuthor ">Yifa  Zhou</span>, <span class="hlFld-ContribAuthor ">Tao  Xu</span>, <span class="hlFld-ContribAuthor ">Junjie  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Temporal Proteomic Analysis of Pancreatic β-Cells in Response to Lipotoxicity and Glucolipotoxicity. </span><span class="cited-content_cbyCitation_journal-name">Molecular & Cellular Proteomics</span><span> <strong>2018,</strong> <em>17 </em>
                                    (11)
                                     , 2119-2131. <a href="https://doi.org/10.1074/mcp.RA118.000698" title="DOI URL">https://doi.org/10.1074/mcp.RA118.000698</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/mcp.RA118.000698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fmcp.RA118.000698%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520%2526%2520Cellular%2520Proteomics%26atitle%3DTemporal%252BProteomic%252BAnalysis%252Bof%252BPancreatic%252B%2525CE%2525B2-Cells%252Bin%252BResponse%252Bto%252BLipotoxicity%252Band%252BGlucolipotoxicity%26aulast%3DLi%26aufirst%3DZonghong%26date%3D2018%26volume%3D17%26issue%3D11%26spage%3D2119%26epage%3D2131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nam  Bui</span>, <span class="hlFld-ContribAuthor ">Justin K.  Huang</span>, <span class="hlFld-ContribAuthor ">Ana  Bojorquez-Gomez</span>, <span class="hlFld-ContribAuthor ">Katherine  Licon</span>, <span class="hlFld-ContribAuthor ">Kyle S.  Sanchez</span>, <span class="hlFld-ContribAuthor ">Sean N.  Tang</span>, <span class="hlFld-ContribAuthor ">Alex N.  Beckett</span>, <span class="hlFld-ContribAuthor ">Tina  Wang</span>, <span class="hlFld-ContribAuthor ">Wei  Zhang</span>, <span class="hlFld-ContribAuthor ">John Paul  Shen</span>, <span class="hlFld-ContribAuthor ">Jason F.  Kreisberg</span>, <span class="hlFld-ContribAuthor ">Trey  Ideker</span>. </span><span class="cited-content_cbyCitation_article-title">Disruption of
              NSD1
              in Head and Neck Cancer Promotes Favorable Chemotherapeutic Responses Linked to Hypomethylation. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2018,</strong> <em>17 </em>
                                    (7)
                                     , 1585-1594. <a href="https://doi.org/10.1158/1535-7163.MCT-17-0937" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-17-0937</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-17-0937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-17-0937%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DDisruption%252Bof%252BNSD1%252Bin%252BHead%252Band%252BNeck%252BCancer%252BPromotes%252BFavorable%252BChemotherapeutic%252BResponses%252BLinked%252Bto%252BHypomethylation%26aulast%3DBui%26aufirst%3DNam%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D7%26spage%3D1585%26epage%3D1594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>. </span><span class="cited-content_cbyCitation_article-title">Protein methyltransferase inhibitors as precision cancer therapeutics: a decade of discovery. </span><span class="cited-content_cbyCitation_journal-name">Philosophical Transactions of the Royal Society B: Biological Sciences</span><span> <strong>2018,</strong> <em>373 </em>
                                    (1748)
                                     , 20170080. <a href="https://doi.org/10.1098/rstb.2017.0080" title="DOI URL">https://doi.org/10.1098/rstb.2017.0080</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1098/rstb.2017.0080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1098%2Frstb.2017.0080%26sid%3Dliteratum%253Aachs%26jtitle%3DPhilosophical%2520Transactions%2520of%2520the%2520Royal%2520Society%2520B%253A%2520Biological%2520Sciences%26atitle%3DProtein%252Bmethyltransferase%252Binhibitors%252Bas%252Bprecision%252Bcancer%252Btherapeutics%25253A%252Ba%252Bdecade%252Bof%252Bdiscovery%26aulast%3DCopeland%26aufirst%3DRobert%2BA.%26date%3D2018%26date%3D2018%26volume%3D373%26issue%3D1748%26spage%3D20170080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anu  Prakash</span>, <span class="hlFld-ContribAuthor ">Juan  Garcia-Moreno</span>, <span class="hlFld-ContribAuthor ">James  Brown</span>, <span class="hlFld-ContribAuthor ">Emer  Bourke</span>. </span><span class="cited-content_cbyCitation_article-title">Clinically Applicable Inhibitors Impacting Genome Stability. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (5)
                                     , 1166. <a href="https://doi.org/10.3390/molecules23051166" title="DOI URL">https://doi.org/10.3390/molecules23051166</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23051166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23051166%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DClinically%252BApplicable%252BInhibitors%252BImpacting%252BGenome%252BStability%26aulast%3DPrakash%26aufirst%3DAnu%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D5%26spage%3D1166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessio  Marcozzi</span>, <span class="hlFld-ContribAuthor ">Tiziana  Masini</span>, <span class="hlFld-ContribAuthor ">Di  Zhu</span>, <span class="hlFld-ContribAuthor ">Diego  Pesce</span>, <span class="hlFld-ContribAuthor ">Boris  Illarionov</span>, <span class="hlFld-ContribAuthor ">Markus  Fischer</span>, <span class="hlFld-ContribAuthor ">Andreas  Herrmann</span>, <span class="hlFld-ContribAuthor ">Anna K. H.  Hirsch</span>. </span><span class="cited-content_cbyCitation_article-title">Phage Display on the Anti-infective Target 1-Deoxy-
              d
              -xylulose-5-phosphate Synthase Leads to an Acceptor-Substrate Competitive Peptidic Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2018,</strong> <em>19 </em>
                                    (1)
                                     , 58-65. <a href="https://doi.org/10.1002/cbic.201700402" title="DOI URL">https://doi.org/10.1002/cbic.201700402</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201700402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201700402%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DPhage%252BDisplay%252Bon%252Bthe%252BAnti-infective%252BTarget%252B1-Deoxy-%252Bd%252B-xylulose-5-phosphate%252BSynthase%252BLeads%252Bto%252Ban%252BAcceptor-Substrate%252BCompetitive%252BPeptidic%252BInhibitor%26aulast%3DMarcozzi%26aufirst%3DAlessio%26date%3D2018%26date%3D2017%26volume%3D19%26issue%3D1%26spage%3D58%26epage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kimberly D.  Barnash</span>, <span class="hlFld-ContribAuthor ">Kelsey N.  Lamb</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Peptide Technologies in the Development of Chemical Tools for Chromatin-Associated Machinery. </span><span class="cited-content_cbyCitation_journal-name">Drug Development Research</span><span> <strong>2017,</strong> <em>78 </em>
                                    (6)
                                     , 300-312. <a href="https://doi.org/10.1002/ddr.21398" title="DOI URL">https://doi.org/10.1002/ddr.21398</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ddr.21398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fddr.21398%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Development%2520Research%26atitle%3DPeptide%252BTechnologies%252Bin%252Bthe%252BDevelopment%252Bof%252BChemical%252BTools%252Bfor%252BChromatin-Associated%252BMachinery%26aulast%3DBarnash%26aufirst%3DKimberly%2BD.%26date%3D2017%26date%3D2017%26volume%3D78%26issue%3D6%26spage%3D300%26epage%3D312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in developing selective inhibitors of protein methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 100-108. <a href="https://doi.org/10.1016/j.cbpa.2017.06.013" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.06.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.06.013%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Bprogress%252Bin%252Bdeveloping%252Bselective%252Binhibitors%252Bof%252Bprotein%252Bmethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2017%26volume%3D39%26spage%3D100%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ozge  Gursoy-Yuzugullu</span>, <span class="hlFld-ContribAuthor ">Chelsea  Carman</span>, <span class="hlFld-ContribAuthor ">Rodolfo Bortolozo  Serafim</span>, <span class="hlFld-ContribAuthor ">Marios  Myronakis</span>, <span class="hlFld-ContribAuthor ">Valeria  Valente</span>, <span class="hlFld-ContribAuthor ">Brendan D.  Price</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic therapy with inhibitors of histone methylation suppresses DNA damage signaling and increases glioma cell radiosensitivity. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (15)
                                     , 24518-24532. <a href="https://doi.org/10.18632/oncotarget.15543" title="DOI URL">https://doi.org/10.18632/oncotarget.15543</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.15543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.15543%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DEpigenetic%252Btherapy%252Bwith%252Binhibitors%252Bof%252Bhistone%252Bmethylation%252Bsuppresses%252BDNA%252Bdamage%252Bsignaling%252Band%252Bincreases%252Bglioma%252Bcell%252Bradiosensitivity%26aulast%3DGursoy-Yuzugullu%26aufirst%3DOzge%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D15%26spage%3D24518%26epage%3D24532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kenneth D  Bromberg</span>, <span class="hlFld-ContribAuthor ">Taylor R H  Mitchell</span>, <span class="hlFld-ContribAuthor ">Anup K  Upadhyay</span>, <span class="hlFld-ContribAuthor ">Clarissa G  Jakob</span>, <span class="hlFld-ContribAuthor ">Manisha A  Jhala</span>, <span class="hlFld-ContribAuthor ">Kenneth M  Comess</span>, <span class="hlFld-ContribAuthor ">Loren M  Lasko</span>, <span class="hlFld-ContribAuthor ">Conglei  Li</span>, <span class="hlFld-ContribAuthor ">Creighton T  Tuzon</span>, <span class="hlFld-ContribAuthor ">Yujia  Dai</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Mohammad S  Eram</span>, <span class="hlFld-ContribAuthor ">Alexander  Nuber</span>, <span class="hlFld-ContribAuthor ">Niru B  Soni</span>, <span class="hlFld-ContribAuthor ">Vlasios  Manaves</span>, <span class="hlFld-ContribAuthor ">Mikkel A  Algire</span>, <span class="hlFld-ContribAuthor ">Ramzi F  Sweis</span>, <span class="hlFld-ContribAuthor ">Maricel  Torrent</span>, <span class="hlFld-ContribAuthor ">Gunnar  Schotta</span>, <span class="hlFld-ContribAuthor ">Chaohong  Sun</span>, <span class="hlFld-ContribAuthor ">Michael R  Michaelides</span>, <span class="hlFld-ContribAuthor ">Alex R  Shoemaker</span>, <span class="hlFld-ContribAuthor ">Cheryl H  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J  Brown</span>, <span class="hlFld-ContribAuthor ">Vijayaratnam  Santhakumar</span>, <span class="hlFld-ContribAuthor ">Alberto  Martin</span>, <span class="hlFld-ContribAuthor ">Judd C  Rice</span>, <span class="hlFld-ContribAuthor ">Gary G  Chiang</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">William N  Pappano</span>. </span><span class="cited-content_cbyCitation_article-title">The SUV4-20 inhibitor A-196 verifies a role for epigenetics in genomic integrity. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2017,</strong> <em>13 </em>
                                    (3)
                                     , 317-324. <a href="https://doi.org/10.1038/nchembio.2282" title="DOI URL">https://doi.org/10.1038/nchembio.2282</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nchembio.2282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnchembio.2282%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DThe%252BSUV4-20%252Binhibitor%252BA-196%252Bverifies%252Ba%252Brole%252Bfor%252Bepigenetics%252Bin%252Bgenomic%252Bintegrity%26aulast%3DBromberg%26aufirst%3DKenneth%2BD%26date%3D2017%26date%3D2017%26volume%3D13%26issue%3D3%26spage%3D317%26epage%3D324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hiroshi  Tanaka</span>, <span class="hlFld-ContribAuthor ">Shin-ichiro  Takebayashi</span>, <span class="hlFld-ContribAuthor ">Akihisa  Sakamoto</span>, <span class="hlFld-ContribAuthor ">Tomoka  Igata</span>, <span class="hlFld-ContribAuthor ">Yuko  Nakatsu</span>, <span class="hlFld-ContribAuthor ">Noriko  Saitoh</span>, <span class="hlFld-ContribAuthor ">Shinjiro  Hino</span>, <span class="hlFld-ContribAuthor ">Mitsuyoshi  Nakao</span>. </span><span class="cited-content_cbyCitation_article-title">The SETD8/PR-Set7 Methyltransferase Functions as a Barrier to Prevent Senescence-Associated Metabolic Remodeling. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2017,</strong> <em>18 </em>
                                    (9)
                                     , 2148-2161. <a href="https://doi.org/10.1016/j.celrep.2017.02.021" title="DOI URL">https://doi.org/10.1016/j.celrep.2017.02.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2017.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2017.02.021%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DThe%252BSETD8%25252FPR-Set7%252BMethyltransferase%252BFunctions%252Bas%252Ba%252BBarrier%252Bto%252BPrevent%252BSenescence-Associated%252BMetabolic%252BRemodeling%26aulast%3DTanaka%26aufirst%3DHiroshi%26date%3D2017%26volume%3D18%26issue%3D9%26spage%3D2148%26epage%3D2161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronica  Veschi</span>, <span class="hlFld-ContribAuthor ">Zhihui  Liu</span>, <span class="hlFld-ContribAuthor ">Ty C.  Voss</span>, <span class="hlFld-ContribAuthor ">Laurent  Ozbun</span>, <span class="hlFld-ContribAuthor ">Berkley  Gryder</span>, <span class="hlFld-ContribAuthor ">Chunhua  Yan</span>, <span class="hlFld-ContribAuthor ">Ying  Hu</span>, <span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Sharlyn J.  Mazur</span>, <span class="hlFld-ContribAuthor ">Norris  Lam</span>, <span class="hlFld-ContribAuthor ">Barbara K.  Souza</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Giannini</span>, <span class="hlFld-ContribAuthor ">Gordon L.  Hager</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Javed  Khan</span>, <span class="hlFld-ContribAuthor ">Ettore  Appella</span>, <span class="hlFld-ContribAuthor ">Carol J.  Thiele</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic siRNA and Chemical Screens Identify SETD8 Inhibition as a Therapeutic Strategy for p53 Activation in High-Risk Neuroblastoma. </span><span class="cited-content_cbyCitation_journal-name">Cancer Cell</span><span> <strong>2017,</strong> <em>31 </em>
                                    (1)
                                     , 50-63. <a href="https://doi.org/10.1016/j.ccell.2016.12.002" title="DOI URL">https://doi.org/10.1016/j.ccell.2016.12.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ccell.2016.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ccell.2016.12.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Cell%26atitle%3DEpigenetic%252BsiRNA%252Band%252BChemical%252BScreens%252BIdentify%252BSETD8%252BInhibition%252Bas%252Ba%252BTherapeutic%252BStrategy%252Bfor%252Bp53%252BActivation%252Bin%252BHigh-Risk%252BNeuroblastoma%26aulast%3DVeschi%26aufirst%3DVeronica%26date%3D2017%26volume%3D31%26issue%3D1%26spage%3D50%26epage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sandeep  Sundriyal</span>, <span class="hlFld-ContribAuthor ">Patty B.  Chen</span>, <span class="hlFld-ContribAuthor ">Alexandra S.  Lubin</span>, <span class="hlFld-ContribAuthor ">Gregor A.  Lueg</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Andrew J. P.  White</span>, <span class="hlFld-ContribAuthor ">Nicholas A.  Malmquist</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Artur  Scherf</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Fuchter</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine methyltransferase structure activity relationships that allow for segregation of G9a inhibition and anti-Plasmodium activity. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2017,</strong> <em>8 </em>
                                    (5)
                                     , 1069-1092. <a href="https://doi.org/10.1039/C7MD00052A" title="DOI URL">https://doi.org/10.1039/C7MD00052A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MD00052A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MD00052A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DHistone%252Blysine%252Bmethyltransferase%252Bstructure%252Bactivity%252Brelationships%252Bthat%252Ballow%252Bfor%252Bsegregation%252Bof%252BG9a%252Binhibition%252Band%252Banti-Plasmodium%252Bactivity%26aulast%3DSundriyal%26aufirst%3DSandeep%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D5%26spage%3D1069%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ciro  Milite</span>, <span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Monica  Viviano</span>, <span class="hlFld-ContribAuthor ">Donatella  Rescigno</span>, <span class="hlFld-ContribAuthor ">Agostino  Cianciulli</span>, <span class="hlFld-ContribAuthor ">Amodio Luca  Balzano</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Sabrina  Castellano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">The emerging role of lysine methyltransferase SETD8 in human diseases. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-016-0268-4" title="DOI URL">https://doi.org/10.1186/s13148-016-0268-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-016-0268-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-016-0268-4%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DThe%252Bemerging%252Brole%252Bof%252Blysine%252Bmethyltransferase%252BSETD8%252Bin%252Bhuman%252Bdiseases%26aulast%3DMilite%26aufirst%3DCiro%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Zhou</span>, <span class="hlFld-ContribAuthor ">Xin  Che</span>, <span class="hlFld-ContribAuthor ">Guochen  Bao</span>, <span class="hlFld-ContribAuthor ">Na  Wang</span>, <span class="hlFld-ContribAuthor ">Li  Peng</span>, <span class="hlFld-ContribAuthor ">Kimberly D.  Barnash</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Xu  Bai</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and protein methyltransferase activity of a unique set of constrained amine containing compounds. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>26 </em>
                                    (18)
                                     , 4436-4440. <a href="https://doi.org/10.1016/j.bmcl.2016.08.004" title="DOI URL">https://doi.org/10.1016/j.bmcl.2016.08.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2016.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2016.08.004%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bprotein%252Bmethyltransferase%252Bactivity%252Bof%252Ba%252Bunique%252Bset%252Bof%252Bconstrained%252Bamine%252Bcontaining%252Bcompounds%26aulast%3DZhou%26aufirst%3DHao%26date%3D2016%26volume%3D26%26issue%3D18%26spage%3D4436%26epage%3D4440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John M.  Strelow</span>, <span class="hlFld-ContribAuthor ">Min  Xiao</span>, <span class="hlFld-ContribAuthor ">Rachel N.  Cavitt</span>, <span class="hlFld-ContribAuthor ">Nathan C.  Fite</span>, <span class="hlFld-ContribAuthor ">Brandon J.  Margolis</span>, <span class="hlFld-ContribAuthor ">Kyu-Jin  Park</span>. </span><span class="cited-content_cbyCitation_article-title">The Use of Nucleosome Substrates Improves Binding of SAM Analogs to SETD8. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Screening</span><span> <strong>2016,</strong> <em>21 </em>
                                    (8)
                                     , 786-794. <a href="https://doi.org/10.1177/1087057116656596" title="DOI URL">https://doi.org/10.1177/1087057116656596</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1087057116656596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1087057116656596%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Screening%26atitle%3DThe%252BUse%252Bof%252BNucleosome%252BSubstrates%252BImproves%252BBinding%252Bof%252BSAM%252BAnalogs%252Bto%252BSETD8%26aulast%3DStrelow%26aufirst%3DJohn%2BM.%26date%3D2016%26date%3D2016%26volume%3D21%26issue%3D8%26spage%3D786%26epage%3D794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiajun  Zhu</span>, <span class="hlFld-ContribAuthor ">Zhixun  Dou</span>, <span class="hlFld-ContribAuthor ">Morgan A.  Sammons</span>, <span class="hlFld-ContribAuthor ">Arnold J.  Levine</span>, <span class="hlFld-ContribAuthor ">Shelley L.  Berger</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine methylation represses p53 activity in teratocarcinoma cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2016,</strong> <em>113 </em>
                                    (35)
                                     , 9822-9827. <a href="https://doi.org/10.1073/pnas.1610387113" title="DOI URL">https://doi.org/10.1073/pnas.1610387113</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1610387113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1610387113%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DLysine%252Bmethylation%252Brepresses%252Bp53%252Bactivity%252Bin%252Bteratocarcinoma%252Bcancer%252Bcells%26aulast%3DZhu%26aufirst%3DJiajun%26date%3D2016%26date%3D2016%26volume%3D113%26issue%3D35%26spage%3D9822%26epage%3D9827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie  Lopez</span>, <span class="hlFld-ContribAuthor ">Maxime  Leroy</span>, <span class="hlFld-ContribAuthor ">Chantal  Etievant</span>, <span class="hlFld-ContribAuthor ">Fréderic  Ausseil</span>, <span class="hlFld-ContribAuthor ">Alexandre  Gagnon</span>, <span class="hlFld-ContribAuthor ">Paola  Arimondo</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 63-95. <a href="https://doi.org/10.1016/B978-0-12-802208-5.00003-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-802208-5.00003-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802208-5.00003-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802208-5.00003-5%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BDiscovery%252BMethods%26aulast%3DLopez%26aufirst%3DMarie%26date%3D2016%26spage%3D63%26epage%3D95%26pub%3DElsevier%26atitle%3DDrug%252BDiscovery%252Bin%252BCancer%252BEpigenetics%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J.N.  Nichol</span>, <span class="hlFld-ContribAuthor ">D.  Dupéré-Richer</span>, <span class="hlFld-ContribAuthor ">T.  Ezponda</span>, <span class="hlFld-ContribAuthor ">J.D.  Licht</span>, <span class="hlFld-ContribAuthor ">W.H.  Miller</span>. </span><span class="cited-content_cbyCitation_article-title">H3K27 Methylation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 59-95. <a href="https://doi.org/10.1016/bs.acr.2016.05.001" title="DOI URL">https://doi.org/10.1016/bs.acr.2016.05.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.acr.2016.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.acr.2016.05.001%26sid%3Dliteratum%253Aachs%26atitle%3DH3K27%252BMethylation%26aulast%3DNichol%26aufirst%3DJ.N.%26date%3D2016%26spage%3D59%26epage%3D95%26pub%3DElsevier%26date%3D2016%26volume%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Kelsey  Bohn</span>, <span class="hlFld-ContribAuthor ">Christine A.  Hrycyna</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Non-substrate based, small molecule inhibitors of the human isoprenylcysteine carboxyl methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2016,</strong> <em>7 </em>
                                    (5)
                                     , 1016-1021. <a href="https://doi.org/10.1039/C6MD00130K" title="DOI URL">https://doi.org/10.1039/C6MD00130K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6MD00130K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6MD00130K%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DNon-substrate%252Bbased%25252C%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252Bthe%252Bhuman%252Bisoprenylcysteine%252Bcarboxyl%252Bmethyltransferase%26aulast%3DButler%26aufirst%3DKyle%2BV.%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D5%26spage%3D1016%26epage%3D1021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yunlei  Hou</span>, <span class="hlFld-ContribAuthor ">Shasha  Wu</span>, <span class="hlFld-ContribAuthor ">Longsheng  Ma</span>, <span class="hlFld-ContribAuthor ">Jinying  Bai</span>, <span class="hlFld-ContribAuthor ">Zijian  Liu</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and antitumor activity of novel 6,7-dimethoxyquinazoline derivatives containing diaryl urea moiety. </span><span class="cited-content_cbyCitation_journal-name">Chemical Research in Chinese Universities</span><span> <strong>2015,</strong> <em>31 </em>
                                    (5)
                                     , 766-773. <a href="https://doi.org/10.1007/s40242-015-5036-z" title="DOI URL">https://doi.org/10.1007/s40242-015-5036-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s40242-015-5036-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs40242-015-5036-z%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Research%2520in%2520Chinese%2520Universities%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bantitumor%252Bactivity%252Bof%252Bnovel%252B6%25252C7-dimethoxyquinazoline%252Bderivatives%252Bcontaining%252Bdiaryl%252Burea%252Bmoiety%26aulast%3DHou%26aufirst%3DYunlei%26date%3D2015%26date%3D2015%26volume%3D31%26issue%3D5%26spage%3D766%26epage%3D773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunngai  Hui</span>, <span class="hlFld-ContribAuthor ">Tao  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of lysine methyltransferase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2015,</strong> <em>3 </em><a href="https://doi.org/10.3389/fchem.2015.00044" title="DOI URL">https://doi.org/10.3389/fchem.2015.00044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2015.00044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2015.00044%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DSynthesis%252Bof%252Blysine%252Bmethyltransferase%252Binhibitors%26aulast%3DHui%26aufirst%3DChunngai%26date%3D2015%26date%3D2015%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anqi  Ma</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Kyle V.  Butler</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–activity relationship studies of SETD8 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1892-1898. <a href="https://doi.org/10.1039/C4MD00317A" title="DOI URL">https://doi.org/10.1039/C4MD00317A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00317A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00317A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DStructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Bof%252BSETD8%252Binhibitors%26aulast%3DMa%26aufirst%3DAnqi%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1892%26epage%3D1898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0014.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of the known SETD8 inhibitor nahuoic acid A.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>1</b> was identified as an inhibitor of SETD8 by cross-screening a quinazoline-based inhibitor set. (A) Structure of compound <b>1</b>. (B) Concentration–response curve of compound <b>1</b> in the SETD8 radioactive methyl transfer assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Typical Synthesis of 2,4-Diamino-6,7-dimethoxyquinazolines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) R<sup>1</sup> amines, THF, <i>N</i>,<i>N</i>-diisopropylethylamine, room temperature; (b) R<sup>2</sup> amines, <i>n</i>-BuOH, DIPEA, microwave, 160 °C.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>1</b> binds SETD8 with a <i>K</i><sub>D</sub> of 18.3 ± 3.2 μM (<i>n</i> = 3) in ITC studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>1</b> exhibits rapid on and off rates in SPR studies.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MOA studies of compound <b>1</b>. (A) Compound <b>1</b> is competitive with the peptide substrate, as its IC<sub>50</sub> values increased linearly with H4 peptide concentrations. (B) Compound <b>1</b> is noncompetitive with the cofactor SAM, as its IC<sub>50</sub> values remained constant in the presence of increasing concentrations of SAM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Peptide displacement assay. Compound <b>1</b> effectively displaced the FITC-labeled H4K20me (1–14) peptide, while an inactive control (compound <b>14</b>) did not.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/medium/jm-2014-00871s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Selectivity of inhibitor <b>1</b> versus 15 other methyltransferases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-15/jm500871s/20150717/images/large/jm-2014-00871s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500871s&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83346" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83346" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 53 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0lhee0gWMG95iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lhee0gWMG95iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Helin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span> </span><span class="NLM_article-title">Chromatin proteins and modifications as drug targets</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">502</span><span class="NLM_x">, </span> <span class="NLM_fpage">480</span><span class="NLM_x">–</span> <span class="NLM_lpage">488</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnature12751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=24153301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=502&publication_year=2013&pages=480-488&author=K.+Helinauthor=D.+Dhanak&title=Chromatin+proteins+and+modifications+as+drug+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin proteins and modifications as drug targets</span></div><div class="casAuthors">Helin, Kristian; Dhanak, Dashyant</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">502</span>
        (<span class="NLM_cas:issue">7472</span>),
    <span class="NLM_cas:pages">480-488</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A plethora of groundbreaking studies have demonstrated the importance of chromatin-assocd. proteins and post-translational modifications of histones, proteins and DNA (so-called epigenetic modifications) for transcriptional control and normal development.  Disruption of epigenetic control is a frequent event in disease, and the first epigenetic-based therapies for cancer treatment have been approved.  A generation of new classes of potent and specific inhibitors for several chromatin-assocd. proteins have shown promise in preclin. trials.  Although the biol. of epigenetic regulation is complex, new inhibitors such as these will hopefully be of clin. use in the coming years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa6LKkNrgP9bVg90H21EOLACvtfcHk0lj2CtcfH6Tl3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Krt77E&md5=ca3f12c6aaba31bb571095bacc02550c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature12751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12751%26sid%3Dliteratum%253Aachs%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DChromatin%2520proteins%2520and%2520modifications%2520as%2520drug%2520targets%26jtitle%3DNature%26date%3D2013%26volume%3D502%26spage%3D480%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Bernstein, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meissner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">The mammalian epigenome</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">681</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.cell.2007.01.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=17320505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=669-681&author=B.+E.+Bernsteinauthor=A.+Meissnerauthor=E.+S.+Lander&title=The+mammalian+epigenome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The mammalian epigenome</span></div><div class="casAuthors">Bernstein, Bradley E.; Meissner, Alexander; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-681</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. modifications to DNA and histone proteins form a complex regulatory network that modulates chromatin structure and genome function.  The epigenome refers to the complete description of these potentially heritable changes across the genome.  The compn. of the epigenome within a given cell is a function of genetic determinants, lineage, and environment.  With the sequencing of the human genome completed, investigators now seek a comprehensive view of the epigenetic changes that det. how genetic information is made manifest across an incredibly varied background of developmental stages, tissue types, and disease states.  Here we review current research efforts, with an emphasis on large-scale studies, emerging technologies, and challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKJfRRDsxAHbVg90H21EOLACvtfcHk0lj2CtcfH6Tl3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7g%253D&md5=963fb487bbf473ac801e09a558ac0b0e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.033%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DThe%2520mammalian%2520epigenome%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D669%26epage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lj2CtcfH6Tl3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0lj2CtcfH6Tl3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laible, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, G.</span><span> </span><span class="NLM_article-title">SET domain proteins modulate chromatin domains in eu- and heterochromatin</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1007%2Fs000180050127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=9487389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADyaK1cXns1CltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=80-93&author=T.+Jenuweinauthor=G.+Laibleauthor=R.+Dornauthor=G.+Reuter&title=SET+domain+proteins+modulate+chromatin+domains+in+eu-+and+heterochromatin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic control of transcription. SET domain proteins modulate chromatin domains in eu- and heterochromatin</span></div><div class="casAuthors">Jenuwein, T.; Laible, G.; Dorn, R.; Reuter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-93</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review with 110 refs. is given as part of a multi-author review on epigenetic control of transcription.  The SET domain is a 130-amino acid, evolutionarily conserved sequence motif present in chromosomal proteins that function in modulating gene activities from yeast to mammals.  Initially identified as members of the Polycomb- and trithorax-group (Pc-6 and trx-G) gene families, which are required to maintain expression boundaries of homeotic selector (HOM-C) genes, SET domain proteins are also involved in position-effect-variegation (PEV), telomeric and centromeric gene silencing, and possibly in detg. chromosome architecture.  These observations implicate SET domain proteins as multifunctional chromatin regulators with activities in both eu- and heterochromatin - a role consistent with their modular structure, which combines the SET domain with addnl. sequence motifs of either a cysteine-rich region/zinc-finger type or the chromo domain.  Multiple functions for chromatin regulators are not restricted to the SET protein family, since many trx-G (but only very few Pc-G) genes are also modifiers of PEV.  Together, these data establish a model in which the modulation of chromatin domains is mechanistically linked with the regulation of key developmental loci (e.g. HOM-C).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXwGgAe0PQ57Vg90H21EOLACvtfcHk0liPDUFO4hg83A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXns1CltA%253D%253D&md5=dd161ab187056ee96b1d1468a0e5ccd1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs000180050127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs000180050127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DLaible%26aufirst%3DG.%26aulast%3DDorn%26aufirst%3DR.%26aulast%3DReuter%26aufirst%3DG.%26atitle%3DSET%2520domain%2520proteins%2520modulate%2520chromatin%2520domains%2520in%2520eu-%2520and%2520heterochromatin%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D1998%26volume%3D54%26spage%3D80%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Xiao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">SET domains and histone methylation</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">699</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.sbi.2003.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=14675547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=699-705&author=B.+Xiaoauthor=J.+R.+Wilsonauthor=S.+J.+Gamblin&title=SET+domains+and+histone+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SET domains and histone methylation</span></div><div class="casAuthors">Xiao, Bing; Wilson, Jonathan R.; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">699-705</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The realization that SET domains, which are found in numerous proteins involved in chromatin regulation, catalyze the methylation of lysine residues has led to intense interest in their cellular, biochem. and structural properties.  The structures of five SET domain proteins have been reported over the past year.  SET domains possess a novel fold, and use adjacent domains for both structural stabilization and the completion of their active sites.  The cofactor S-adenosyl-L-methionine and peptide substrates bind on opposite faces of the SET domain.  Remarkably, the sidechain of the target lysine approaches the transferred Me group through a narrow channel that passes through the middle of the domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj4grEHxVvY7Vg90H21EOLACvtfcHk0liPDUFO4hg83A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXps1yjsLw%253D&md5=238b75c760b6bde8a56f128dd8077c30</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2003.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2003.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DSET%2520domains%2520and%2520histone%2520methylation%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2003%26volume%3D13%26spage%3D699%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">August, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teodoro, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rea, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homon, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+small-molecule+inhibitor+for+the+G9a+histone+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0liPDUFO4hg83A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520small-molecule%2520inhibitor%2520for%2520the%2520G9a%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozieradzki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a 2,4-diamino-7-aminoalkoxyquinazoline as a potent and selective inhibitor of histone lysine methyltransferase G9a</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">7950</span><span class="NLM_x">–</span> <span class="NLM_lpage">7953</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901543m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=7950-7953&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=G.+A.+Wasneyauthor=A.+Dongauthor=D.+Barsyteauthor=I.+Kozieradzkiauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+2%2C4-diamino-7-aminoalkoxyquinazoline+as+a+potent+and+selective+inhibitor+of+histone+lysine+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a 2,4-Diamino-7-aminoalkoxyquinazoline as a Potent and Selective Inhibitor of Histone Lysine Methyltransferase G9a</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wasney, Gregory A.; Dong, Aiping; Barsyte, Dalia; Kozieradzki, Ivona; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7950-7953</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template led to the discovery of 8 (I) as a potent and selective G9a inhibitor.  A high resoln. X-ray crystal structure of the G9a-8 complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  The cocrystal structure validated our binding hypothesis and will enable structure-based design of novel inhibitors.  8 Is a useful tool for investigating the biol. of G9a and its roles in chromatin remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjWt5XsGXyj7Vg90H21EOLACvtfcHk0lgT4bFrc-o8VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlOiurzN&md5=2d7e34a8c4b5bf1cafd4e01ad2a86de1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm901543m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901543m%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DKozieradzki%26aufirst%3DI.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%25202%252C4-diamino-7-aminoalkoxyquinazoline%2520as%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520lysine%2520methyltransferase%2520G9a%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D7950%26epage%3D7953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">Adding a lysine mimic in the design of potent inhibitors of histone lysine methyltransferases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">400</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=2010&pages=1-7&author=Y.+Changauthor=T.+Ganeshauthor=J.+R.+Hortonauthor=A.+Spannhoffauthor=J.+Liuauthor=A.+Sunauthor=X.+Zhangauthor=M.+T.+Bedfordauthor=Y.+Shinkaiauthor=J.+P.+Snyderauthor=X.+Cheng&title=Adding+a+lysine+mimic+in+the+design+of+potent+inhibitors+of+histone+lysine+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DY.%26aulast%3DGanesh%26aufirst%3DT.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DShinkai%26aufirst%3DY.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAdding%2520a%2520lysine%2520mimic%2520in%2520the%2520design%2520of%2520potent%2520inhibitors%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D400%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Protein lysine methyltransferase G9a inhibitors: design, synthesis, and structure activity relationships of 2,4-diamino-7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5844</span><span class="NLM_x">–</span> <span class="NLM_lpage">5857</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100478y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5844-5857&author=F.+Liuauthor=X.+Chenauthor=A.+Allali-Hassaniauthor=A.+M.+Quinnauthor=T.+J.+Wigleauthor=G.+A.+Wasneyauthor=A.+Dongauthor=G.+Senisterraauthor=I.+Chauauthor=A.+Siarheyevaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Jadhavauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=A.+Simeonovauthor=M.+Vedadiauthor=J.+Jin&title=Protein+lysine+methyltransferase+G9a+inhibitors%3A+design%2C+synthesis%2C+and+structure+activity+relationships+of+2%2C4-diamino-7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Methyltransferase G9a Inhibitors: Design, Synthesis, and Structure Activity Relationships of 2,4-Diamino-7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Chen, Xin; Allali-Hassani, Abdellah; Quinn, Amy M.; Wigle, Tim J.; Wasney, Gregory A.; Dong, Aiping; Senisterra, Guillermo; Chau, Irene; Siarheyeva, Alena; Norris, Jacqueline L.; Kireev, Dmitri B.; Jadhav, Ajit; Herold, J. Martin; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Brown, Peter J.; Simeonov, Anton; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5844-5857</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a, which catalyzes methylation of lysine 9 of histone H3 (H3K9) and lysine 373 (K373) of p53, is overexpressed in human cancers.  Genetic knockdown of G9a inhibits cancer cell growth; and the dimethylation of p53 K373 results in the inactivation of p53.  Initial SAR exploration of the 2,4-diamino-6,7-dimethoxyquinazoline template represented by I ( R1 = Bn; R2 = Me)(BIX01294), a selective small mol. inhibitor of G9a and GLP (G9a-like protein), led to the discovery of I [R1 = Me; R2 = (CH2)3NMe2](II)(UNC0224) as a potent G9a inhibitor with excellent selectivity.  A high resoln. X-ray crystal structure of the G9a-II complex, the first cocrystal structure of G9a with a small mol. inhibitor, was obtained.  On the basis of the structural insights revealed by this cocrystal structure, optimization of the 7-(dimethylamino)propoxy side chain of II resulted in the discovery of I [R1 = Me; R2 = (CH2)2O(CH2)2NMe2](UNC0321, Morrison Ki = 63 pM), which is the first G9a inhibitor with picomolar potency and the most potent G9a inhibitor to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgS0YxdNdNxLVg90H21EOLACvtfcHk0ljdYAHXx3VN5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ekurk%253D&md5=8bd703c0a975a33bf9e5c019f1f95ee6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm100478y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100478y%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProtein%2520lysine%2520methyltransferase%2520G9a%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520structure%2520activity%2520relationships%2520of%25202%252C4-diamino-7-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5844%26epage%3D5857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiMaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+DiMaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0ljdYAHXx3VN5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDiMaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">6139</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200903z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=6139-6150&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=A.+Allali-Hassaniauthor=Y.+Heauthor=J.+M.+Heroldauthor=X.+Chenauthor=C.+M.+Yatesauthor=S.+V.+Fryeauthor=P.+J.+Brownauthor=J.+Huangauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Optimization+of+cellular+activity+of+G9a+inhibitors+7-aminoalkoxy-quinazolines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of cellular activity of G9a inhibitors 7-aminoalkoxy-quinazolines</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Allali-Hassani, Abdellah; He, Yunlong; Herold, J. Martin; Chen, Xin; Yates, Christopher M.; Frye, Stephen V.; Brown, Peter J.; Huang, Jing; Vedadi, Masoud; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6139-6150</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein lysine methyltransferase G9a plays key roles in the transcriptional repression of a variety of genes via dimethylation of lysine 9 on histone H3 (H3K9me2) of chromatin as well as dimethylation of nonhistone proteins including tumor suppressor p53.  We previously reported the discovery of UNC0321, the most potent G9a inhibitor to date, via structure-based design and structure-activity relationship (SAR) exploration of the quinazoline scaffold represented by BIX01294.  Despite its very high in vitro potency, compd. UNC0321 lacks sufficient cellular potency.  The design and synthesis of several generations of new analogs aimed at improving cell membrane permeability while maintaining high in vitro potency resulted in the discovery of a no. of novel G9a inhibitors such as UNC0646 I and UNC0631 II with excellent potency in a variety of cell lines and excellent sepn. of functional potency vs. cell toxicity.  The design, synthesis, and cellular SAR of these potent G9a inhibitors are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC91yKB7S34rVg90H21EOLACvtfcHk0lj4RO_Z09RwQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvVWltLg%253D&md5=f3f54b91e6b24aa248498f9ff1fdb1ae</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm200903z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200903z%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYates%26aufirst%3DC.%2BM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DOptimization%2520of%2520cellular%2520activity%2520of%2520G9a%2520inhibitors%25207-aminoalkoxy-quinazolines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D6139%26epage%3D6150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0lj4RO_Z09RwQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lj4RO_Z09RwQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span><span class="NLM_issue">133</span><span class="NLM_x">) </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=16746-16749&issue=133&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis+and+crystallographic+studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26issue%3D133%26spage%3D16746%26epage%3D16749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemp, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shamji, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A small-molecule probe of the histone methyltransferase G9a induces cellular senescence in pancreatic adenocarcinoma</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1152</span><span class="NLM_x">–</span> <span class="NLM_lpage">1157</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb300139y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1152-1157&author=Y.+Yuanauthor=Q.+Wangauthor=J.+Paulkauthor=S.+Kubicekauthor=M.+M.+Kempauthor=D.+J.+Adamsauthor=A.+F.+Shamjiauthor=B.+K.+Wagnerauthor=S.+L.+Schreiber&title=A+small-molecule+probe+of+the+histone+methyltransferase+G9a+induces+cellular+senescence+in+pancreatic+adenocarcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fcb300139y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300139y%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520small-molecule%2520probe%2520of%2520the%2520histone%2520methyltransferase%2520G9a%2520induces%2520cellular%2520senescence%2520in%2520pancreatic%2520adenocarcinoma%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1152%26epage%3D1157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lgPfJrviTZA0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Della Pietra, A.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+Della+Pietraauthor=E.+Diazauthor=L.+V.+Lafranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDella%2BPietra%26aufirst%3DA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLafrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of potent, selective, cell-active inhibitors of the histone lysine methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+potent%2C+selective%2C+cell-active+inhibitors+of+the+histone+lysine+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0ljo5EpwbP7nmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520potent%252C%2520selective%252C%2520cell-active%2520inhibitors%2520of%2520the%2520histone%2520lysine%2520methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibáñez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amaya, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span> </span><span class="NLM_article-title">Sinefungin derivatives as inhibitors and structure probes of protein lysine methyltransferase SETD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">134</span><span class="NLM_x">, </span> <span class="NLM_fpage">18004</span><span class="NLM_x">–</span> <span class="NLM_lpage">18014</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja307060p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=18004-18014&author=W.+Zhengauthor=G.+Ib%C3%A1%C3%B1ezauthor=H.+Wuauthor=G.+Blumauthor=H.+Zengauthor=A.+Dongauthor=F.+Liauthor=T.+Hajianauthor=A.+Allali-Hassaniauthor=M.+F.+Amayaauthor=A.+Siarheyevaauthor=W.+Yuauthor=P.+J.+Brownauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Minauthor=M.+Luo&title=Sinefungin+derivatives+as+inhibitors+and+structure+probes+of+protein+lysine+methyltransferase+SETD2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Sinefungin Derivatives as Inhibitors and Structure Probes of Protein Lysine Methyltransferase SETD2</span></div><div class="casAuthors">Zheng, Weihong; Ibanez, Glorymar; Wu, Hong; Blum, Gil; Zeng, Hong; Dong, Aiping; Li, Fengling; Hajian, Taraneh; Allali-Hassani, Abdellah; Amaya, Maria F.; Siarheyeva, Alena; Yu, Wenyu; Brown, Peter J.; Schapira, Matthieu; Vedadi, Masoud; Min, Jinrong; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18004-18014</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic regulation is involved in numerous physiol. and pathogenic processes.  Among the key regulators that orchestrate epigenetic signaling are over 50 human protein lysine methyltransferases (PKMTs).  Interrogation of the functions of individual PKMTs can be facilitated by target-specific PKMT inhibitors.  Given the emerging need for such small mols., an approach was envisioned to identify target-specific methyltransferase inhibitors by screening privileged small-mol. scaffolds against diverse methyltransferases.  In this work, the feasibility of such an approach was demonstrated by identifying the inhibitors of SETD2.  N-Pr sinefungin (Pr-SNF) was shown to interact preferentially with SETD2 by matching the distinct transition-state features of SETD2's catalytically active conformer.  With Pr-SNF as a structure probe, the dual roles of SETD2's post-SET loop in regulating substrate access through a distinct topol. reconfiguration was revealed.  Privileged sinefungin scaffolds are expected to have broad use as structure and chem. probes of methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-sWI1BJjg7Vg90H21EOLACvtfcHk0ljo5EpwbP7nmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVOhsrbL&md5=30f5ba653007349df64e095f1c2cf188</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja307060p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307060p%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DIb%25C3%25A1%25C3%25B1ez%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAmaya%26aufirst%3DM.%2BF.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSinefungin%2520derivatives%2520as%2520inhibitors%2520and%2520structure%2520probes%2520of%2520protein%2520lysine%2520methyltransferase%2520SETD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D18004%26epage%3D18014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Qi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardayfio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span> </span><span class="NLM_article-title">Selective inhibition of Ezh2 by a small molecule inhibitor blocks tumor cells proliferation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">21360</span><span class="NLM_x">–</span> <span class="NLM_lpage">21365</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=21360-21365&author=W.+Qiauthor=H.+Chanauthor=L.+Tengauthor=L.+Liauthor=S.+Chuaiauthor=R.+Zhangauthor=J.+Zengauthor=M.+Liauthor=H.+Fanauthor=Y.+Linauthor=J.+Guauthor=O.+Ardayfioauthor=J.-H.+Zhangauthor=X.+Yanauthor=J.+Fangauthor=Y.+Miauthor=M.+Zhangauthor=T.+Zhouauthor=G.+Fengauthor=Z.+Chenauthor=G.+Liauthor=T.+Yangauthor=K.+Zhaoauthor=X.+Liuauthor=Z.+Yuauthor=C.+X.+Luauthor=P.+Atadjaauthor=E.+Li&title=Selective+inhibition+of+Ezh2+by+a+small+molecule+inhibitor+blocks+tumor+cells+proliferation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DArdayfio%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DJ.-H.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DC.%2BX.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DSelective%2520inhibition%2520of%2520Ezh2%2520by%2520a%2520small%2520molecule%2520inhibitor%2520blocks%2520tumor%2520cells%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D21360%26epage%3D21365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chory, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wernimont, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scopton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federation, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wienholds, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fncomms2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23250418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1288&author=W.+Yuauthor=E.+J.+Choryauthor=A.+K.+Wernimontauthor=W.+Tempelauthor=A.+Scoptonauthor=A.+Federationauthor=J.+J.+Marineauauthor=J.+Qiauthor=D.+Barsyte-Lovejoyauthor=J.+Yiauthor=R.+Marcellusauthor=R.+E.+Iacobauthor=J.+R.+Engenauthor=C.+Griffinauthor=A.+Amanauthor=E.+Wienholdsauthor=F.+Liauthor=J.+Pinedaauthor=G.+Estiuauthor=T.+Shatsevaauthor=T.+Hajianauthor=R.+Al-awarauthor=J.+E.+Dickauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+E.+Bradnerauthor=M.+Schapira&title=Catalytic+site+remodelling+of+the+DOT1L+methyltransferase+by+selective+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span></div><div class="casAuthors">Yu Wenyu; Chory Emma J; Wernimont Amy K; Tempel Wolfram; Scopton Alex; Federation Alexander; Marineau Jason J; Qi Jun; Barsyte-Lovejoy Dalia; Yi Joanna; Marcellus Richard; Iacob Roxana E; Engen John R; Griffin Carly; Aman Ahmed; Wienholds Erno; Li Fengling; Pineda Javier; Estiu Guillermina; Shatseva Tatiana; Hajian Taraneh; Al-Awar Rima; Dick John E; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Bradner James E; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1288</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Selective inhibition of protein methyltransferases is a promising new approach to drug discovery.  An attractive strategy towards this goal is the development of compounds that selectively inhibit binding of the cofactor, S-adenosylmethionine, within specific protein methyltransferases.  Here we report the three-dimensional structure of the protein methyltransferase DOT1L bound to EPZ004777, the first S-adenosylmethionine-competitive inhibitor of a protein methyltransferase with in vivo efficacy.  This structure and those of four new analogues reveal remodelling of the catalytic site.  EPZ004777 and a brominated analogue, SGC0946, inhibit DOT1L in vitro and selectively kill mixed lineage leukaemia cells, in which DOT1L is aberrantly localized via interaction with an oncogenic MLL fusion protein.  These data provide important new insight into mechanisms of cell-active S-adenosylmethionine-competitive protein methyltransferase inhibitors, and establish a foundation for the further development of drug-like inhibitors of DOT1L for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcST2GJQO5XUmjDnmmyUf5qOfW6udTcc2ebMU2kFP8gzBrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D&md5=f7b501ac4188b14b48fc435637e0e192</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fncomms2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2304%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChory%26aufirst%3DE.%2BJ.%26aulast%3DWernimont%26aufirst%3DA.%2BK.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DScopton%26aufirst%3DA.%26aulast%3DFederation%26aufirst%3DA.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DYi%26aufirst%3DJ.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DWienholds%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPineda%26aufirst%3DJ.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DCatalytic%2520site%2520remodelling%2520of%2520the%2520DOT1L%2520methyltransferase%2520by%2520selective%2520inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D1288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Williams, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalisay, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amphlett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maneerat, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chavez, M. A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matainaho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, R. J.</span><span> </span><span class="NLM_article-title">Nahuoic acid A produced by a streptomyces sp. isolated from a marine sediment is a selective SAM-competitive inhibitor of the histone methyltransferase SETD8</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">417</span></span><div class="citationLinks">[<a href="/doi/10.1021/ol303416k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=414-417&author=D.+E.+Williamsauthor=D.+S.+Dalisayauthor=F.+Liauthor=J.+Amphlettauthor=W.+Maneeratauthor=M.+A.+G.+Chavezauthor=Y.+A.+Wangauthor=T.+Matainahoauthor=W.+Yuauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=R.+J.+Andersen&title=Nahuoic+acid+A+produced+by+a+streptomyces+sp.+isolated+from+a+marine+sediment+is+a+selective+SAM-competitive+inhibitor+of+the+histone+methyltransferase+SETD8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fol303416k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol303416k%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DD.%2BE.%26aulast%3DDalisay%26aufirst%3DD.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DAmphlett%26aufirst%3DJ.%26aulast%3DManeerat%26aufirst%3DW.%26aulast%3DChavez%26aufirst%3DM.%2BA.%2BG.%26aulast%3DWang%26aufirst%3DY.%2BA.%26aulast%3DMatainaho%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DR.%2BJ.%26atitle%3DNahuoic%2520acid%2520A%2520produced%2520by%2520a%2520streptomyces%2520sp.%2520isolated%2520from%2520a%2520marine%2520sediment%2520is%2520a%2520selective%2520SAM-competitive%2520inhibitor%2520of%2520the%2520histone%2520methyltransferase%2520SETD8%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26spage%3D414%26epage%3D417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Synthesis and structure–activity relationship investigation of adenosine-containing inhibitors of histone methyltransferase DOT1L</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">8066</span><span class="NLM_x">–</span> <span class="NLM_lpage">8074</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300917h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=8066-8074&author=J.+L.+Anglinauthor=L.+Dengauthor=Y.+Yaoauthor=G.+Caiauthor=Z.+Liuauthor=H.+Jiangauthor=G.+Chengauthor=P.+Chenauthor=S.+Dongauthor=Y.+Song&title=Synthesis+and+structure%E2%80%93activity+relationship+investigation+of+adenosine-containing+inhibitors+of+histone+methyltransferase+DOT1L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm300917h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300917h%26sid%3Dliteratum%253Aachs%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSynthesis%2520and%2520structure%25E2%2580%2593activity%2520relationship%2520investigation%2520of%2520adenosine-containing%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D8066%26epage%3D8074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacNevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the lysine methyltransferases EZH2 and EZH1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1324</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+MacNevinauthor=F.+Liuauthor=C.+Gaoauthor=X.+P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+lysine+methyltransferases+EZH2+and+EZH1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0lgkYB5NpqU6dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7922</span><span class="NLM_x">–</span> <span class="NLM_lpage">7927</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0lifr7XZhOL8iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Beguelin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teater, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezponda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez-Garcia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoyne, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdel-Wahab, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Licht, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A. M.</span><span> </span><span class="NLM_article-title">EZH2 is required for germinal center formation and somatic EZH2 mutations promote lymphoid transformation</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=677-692&author=W.+Beguelinauthor=R.+Popovicauthor=M.+Teaterauthor=Y.+Jiangauthor=K.+L.+Buntingauthor=M.+Rosenauthor=H.+Shenauthor=S.+N.+Yangauthor=L.+Wangauthor=T.+Ezpondaauthor=E.+Martinez-Garciaauthor=H.+Zhangauthor=Y.+Zhengauthor=S.+K.+Vermaauthor=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+S.+Van+Allerauthor=R.+G.+Krugerauthor=Y.+Liuauthor=C.+F.+McHughauthor=D.+W.+Scottauthor=Y.+R.+Chungauthor=N.+Kelleherauthor=R.+Shaknovichauthor=C.+L.+Creasyauthor=R.+D.+Gascoyneauthor=K.+K.+Wongauthor=L.+Cerchiettiauthor=R.+L.+Levineauthor=O.+Abdel-Wahabauthor=J.+D.+Lichtauthor=O.+Elementoauthor=A.+M.+Melnick&title=EZH2+is+required+for+germinal+center+formation+and+somatic+EZH2+mutations+promote+lymphoid+transformation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DTeater%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DRosen%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DEzponda%26aufirst%3DT.%26aulast%3DMartinez-Garcia%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DScott%26aufirst%3DD.%2BW.%26aulast%3DChung%26aufirst%3DY.%2BR.%26aulast%3DKelleher%26aufirst%3DN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DEZH2%2520is%2520required%2520for%2520germinal%2520center%2520formation%2520and%2520somatic%2520EZH2%2520mutations%2520promote%2520lymphoid%2520transformation%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D677%26epage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8931</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.+P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+vivo+chemical+probe+of+the+lysine+methyltransferases+G9a+and+GLP"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lifr7XZhOL8iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pliushchev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaelides, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery and development of potent and selective inhibitors of histone methyltransferase G9a</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+L.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+development+of+potent+and+selective+inhibitors+of+histone+methyltransferase+G9a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0ljLXjqfHi7Row"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520development%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D205%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Garapaty-Rao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagnon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandy, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Busby, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span> </span><span class="NLM_article-title">Identification of EZH2 and EZH1 small molecule inhibitors with selective impact on diffuse large B cell lymphoma cell growth</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1329</span><span class="NLM_x">–</span> <span class="NLM_lpage">1339</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.chembiol.2013.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=24183969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWmtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1329-1339&author=S.+Garapaty-Raoauthor=C.+Nasveschukauthor=A.+Gagnonauthor=E.+Y.+Chanauthor=P.+Sandyauthor=J.+Busbyauthor=S.+Balasubramanianauthor=R.+Campbellauthor=F.+Zhaoauthor=L.+Bergeronauthor=J.+E.+Audiaauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=P.+Trojer&title=Identification+of+EZH2+and+EZH1+small+molecule+inhibitors+with+selective+impact+on+diffuse+large+B+cell+lymphoma+cell+growth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of EZH2 and EZH1 Small Molecule Inhibitors with Selective Impact on Diffuse Large B Cell Lymphoma Cell Growth</span></div><div class="casAuthors">Garapaty-Rao, Shivani; Nasveschuk, Christopher; Gagnon, Alexandre; Chan, Eric Y.; Sandy, Peter; Busby, Jennifer; Balasubramanian, Srividya; Campbell, Robert; Zhao, Feng; Bergeron, Louise; Audia, James E.; Albrecht, Brian K.; Harmange, Jean-Christophe; Cummings, Richard; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1329-1339</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The histone methyltransferase enhancer of Zeste homolog 2 (EZH2) is a candidate oncogene due to its prevalent overexpression in malignant diseases, including late stage prostate and breast cancers.  The dependency of cancer cells on EZH2 activity is also predicated by recurrent missense mutations residing in the catalytic domain of EZH2 that have been identified in subtypes of diffuse large B cell lymphoma, follicular lymphoma and melanoma.  Herein, we report the identification of a highly selective small mol. inhibitor series of EZH2 and EZH1.  These compds. inhibit wild-type and mutant versions of EZH2 with nanomolar potency, suppress global histone H3-lysine 27 methylation, affect gene expression, and cause selective proliferation defects.  These compds. represent a structurally distinct EZH2 inhibitor chemotype for the exploration of the role of Polycomb Repressive Complex 2-mediated H3K27 methylation in various biol. contexts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNGhFRO6LRbVg90H21EOLACvtfcHk0ljLXjqfHi7Row"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWmtrrP&md5=6079ba01ae1f1964ac1d8108aa5e1227</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DGarapaty-Rao%26aufirst%3DS.%26aulast%3DNasveschuk%26aufirst%3DC.%26aulast%3DGagnon%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DE.%2BY.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DIdentification%2520of%2520EZH2%2520and%2520EZH1%2520small%2520molecule%2520inhibitors%2520with%2520selective%2520impact%2520on%2520diffuse%2520large%2520B%2520cell%2520lymphoma%2520cell%2520growth%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1329%26epage%3D1339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1182%2Fblood-2013-04-497644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=23801631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2013&pages=1017-1025&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=A.+Basavapathruniauthor=L.+Jinauthor=P.+A.+Boriack-Sjodinauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Potent+inhibition+of+DOT1L+as+treatment+of+MLL-fusion+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibition of DOT1L as treatment of MLL-fusion leukemia</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Basavapathruni, Aravind; Jin, Lei; Boriack-Sjodin, P. Ann; Allain, Christina J.; Klaus, Christine R.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1017-1025</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Rearrangements of the MLL gene define a genetically distinct subset of acute leukemias with poor prognosis.  Current treatment options are of limited effectiveness; thus, there is a pressing need for new therapies for this disease.  Genetic and small mol. inhibitor studies have demonstrated that the histone methyltransferase DOT1L is required for the development and maintenance of MLL-rearranged leukemia in model systems.  Here we describe the characterization of EPZ-5676, a potent and selective aminonucleoside inhibitor of DOT1L histone methyltransferase activity.  The compd. has an inhibition const. value of 80 pM, and demonstrates 37000-fold selectivity over all other methyltransferases tested.  In cellular studies, EPZ-5676 inhibited H3K79 methylation and MLL-fusion target gene expression and demonstrated potent cell killing that was selective for acute leukemia lines bearing MLL translocations.  Continuous IV infusion of EPZ-5676 in a rat xenograft model of MLL-rearranged leukemia caused complete tumor regressions that were sustained well beyond the compd. infusion period with no significant wt. loss or signs of toxicity.  EPZ-5676 is therefore a potential treatment of MLL-rearranged leukemia and is under clin. investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT-fqlDzoK07Vg90H21EOLACvtfcHk0ljLXjqfHi7Row"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWltLfE&md5=bad23c37a5fe886668e25d10b7d9b43a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-497644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-497644%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DPotent%2520inhibition%2520of%2520DOT1L%2520as%2520treatment%2520of%2520MLL-fusion%2520leukemia%26jtitle%3DBlood%26date%3D2013%26volume%3D122%26spage%3D1017%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagnon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garapaty-Rao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J.-C. P.</span><span> </span><span class="NLM_article-title">Discovery and optimization of tetramethylpiperidinyl benzamides as inhibitors of EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">378</span><span class="NLM_x">–</span> <span class="NLM_lpage">383</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml400494b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=378-383&author=C.+G.+Nasveschukauthor=A.+Gagnonauthor=S.+Garapaty-Raoauthor=S.+Balasubramanianauthor=R.+Campbellauthor=C.+Leeauthor=F.+Zhaoauthor=L.+Bergeronauthor=R.+Cummingsauthor=P.+Trojerauthor=J.+E.+Audiaauthor=B.+K.+Albrechtauthor=J.-C.+P.+Harmange&title=Discovery+and+optimization+of+tetramethylpiperidinyl+benzamides+as+inhibitors+of+EZH2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fml400494b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400494b%26sid%3Dliteratum%253Aachs%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DGagnon%26aufirst%3DA.%26aulast%3DGarapaty-Rao%26aufirst%3DS.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.-C.%2BP.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520tetramethylpiperidinyl%2520benzamides%2520as%2520inhibitors%2520of%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D378%26epage%3D383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span> </span><span class="NLM_article-title">Targeting protein lysine methylation and demethylation in cancers</span> <span class="citation_source-journal">Acta Biochim. Biophys. Sin.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">70</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1093%2Fabbs%2Fgmr109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=22194015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=70-79&author=Y.+Heauthor=I.+Korboukhauthor=J.+Jinauthor=J.+Huang&title=Targeting+protein+lysine+methylation+and+demethylation+in+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein lysine methylation and demethylation in cancers</span></div><div class="casAuthors">He, Yunlong; Korboukh, Ilia; Jin, Jian; Huang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biochimica et Biophysica Sinica</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">ABBSC2</span>;
        ISSN:<span class="NLM_cas:issn">1672-9145</span>.
    
            (<span class="NLM_cas:orgname">ABBS Editorial Office</span>)
        </div><div class="casAbstract">A review.  During the last decade, we saw an explosion of studies investigating the role of lysine methylation/demethylation of histones and non-histone proteins, such as p53, NF-kappaB, and E2F1.  These Ying-Yang' post-translational modifications are important to fine-tuning the activity of these proteins.  Lysine methylation and demethylation are catalyzed by protein lysine methyltransferases (PKMTs) and protein lysine demethylases (PKDMs).  PKMTs, PKDMs, and their substrates have been shown to play important roles in cancers.  Although the underlying mechanisms of tumorigenesis are still largely unknown, growing evidence is starting to link aberrant regulation of methylation to tumorigenesis.  This review focuses on summarizing the recent progress in understanding of the function of protein lysine methylation, and in the discovery of small mol. inhibitors for PKMTs and PKDMs.  We also discuss the potential and the caveats of targeting protein lysine methylation for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO9Pis9NgA27Vg90H21EOLACvtfcHk0lj_9LpIK5vdUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htr7P&md5=bca793734c9f24cf45bbb529a41207db</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmr109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmr109%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DTargeting%2520protein%2520lysine%2520methylation%2520and%2520demethylation%2520in%2520cancers%26jtitle%3DActa%2520Biochim.%2520Biophys.%2520Sin.%26date%3D2012%26volume%3D44%26spage%3D70%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of protein arginine methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=P.+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+allosteric+inhibitor+of+protein+arginine+methyltransferase+3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0lj_9LpIK5vdUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lhab40vvxENuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Yost, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targets in epigenetics: inhibiting the methyl writers of the histone code</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.2174%2F1875397301005010072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=21966347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=72-84&author=J.+M.+Yostauthor=I.+Korboukhauthor=F.+Liuauthor=C.+Gaoauthor=J.+Jin&title=Targets+in+epigenetics%3A+inhibiting+the+methyl+writers+of+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targets in epigenetics: inhibiting the methyl writers of the histone code</span></div><div class="casAuthors">Yost, Julianne M.; Korboukh, Ilia; Liu, Feng; Gao, Cen; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-84</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs) are assocd. with the development of various human diseases, including cancer, inflammation, and psychiatric disorders.  Given the significant role of these proteins in human disease, efforts to discover selective small-mol. inhibitors of these enzymes are quickly gaining momentum.  In this review, we focus on the recent progress in the discovery of selective PKMT and PRMT inhibitors.  A future perspective on developing methyltransferase inhibitors is also offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHi_nKenm0brVg90H21EOLACvtfcHk0lhab40vvxENuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP&md5=996a0411f27dcb4db88e5a4885439d94</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010072%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargets%2520in%2520epigenetics%253A%2520inhibiting%2520the%2520methyl%2520writers%2520of%2520the%2520histone%2520code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D72%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pabst, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krosl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yung, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underhill, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvageau, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span> </span><span class="NLM_article-title">The methyltransferase G9a regulates HoxA9-dependent transcription in AML</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=317-327&author=B.+Lehnertzauthor=C.+Pabstauthor=L.+Suauthor=M.+Millerauthor=F.+Liuauthor=L.+Yiauthor=R.+Zhangauthor=J.+Kroslauthor=E.+Yungauthor=J.+Kirschnerauthor=P.+Rostenauthor=T.+M.+Underhillauthor=J.+Jinauthor=J.+Hebertauthor=G.+Sauvageauauthor=R.+K.+Humphriesauthor=F.+M.+Rossi&title=The+methyltransferase+G9a+regulates+HoxA9-dependent+transcription+in+AML"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DPabst%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrosl%26aufirst%3DJ.%26aulast%3DYung%26aufirst%3DE.%26aulast%3DKirschner%26aufirst%3DJ.%26aulast%3DRosten%26aufirst%3DP.%26aulast%3DUnderhill%26aufirst%3DT.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHebert%26aufirst%3DJ.%26aulast%3DSauvageau%26aufirst%3DG.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26atitle%3DThe%2520methyltransferase%2520G9a%2520regulates%2520HoxA9-dependent%2520transcription%2520in%2520AML%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26spage%3D317%26epage%3D327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skutt-Kakaria, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davison, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ou, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loeb, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georges, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torok-Storb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paddison, P. J.</span><span> </span><span class="NLM_article-title">G9a/GLP-dependent histone H3K9me2 patterning during human hematopoietic stem cell lineage commitment</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">2499</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=2499-2511&author=X.+Chenauthor=K.+Skutt-Kakariaauthor=J.+Davisonauthor=Y.+L.+Ouauthor=E.+Choiauthor=P.+Malikauthor=K.+Loebauthor=B.+Woodauthor=G.+Georgesauthor=B.+Torok-Storbauthor=P.+J.+Paddison&title=G9a%2FGLP-dependent+histone+H3K9me2+patterning+during+human+hematopoietic+stem+cell+lineage+commitment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DSkutt-Kakaria%26aufirst%3DK.%26aulast%3DDavison%26aufirst%3DJ.%26aulast%3DOu%26aufirst%3DY.%2BL.%26aulast%3DChoi%26aufirst%3DE.%26aulast%3DMalik%26aufirst%3DP.%26aulast%3DLoeb%26aufirst%3DK.%26aulast%3DWood%26aufirst%3DB.%26aulast%3DGeorges%26aufirst%3DG.%26aulast%3DTorok-Storb%26aufirst%3DB.%26aulast%3DPaddison%26aufirst%3DP.%2BJ.%26atitle%3DG9a%252FGLP-dependent%2520histone%2520H3K9me2%2520patterning%2520during%2520human%2520hematopoietic%2520stem%2520cell%2520lineage%2520commitment%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D2499%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Antignano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kron, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penrod, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gold, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chenery, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNagny, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupien, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levings, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span> </span><span class="NLM_article-title">Methyltransferase G9A regulates T cell differentiation during murine intestinal inflammation</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">1945</span><span class="NLM_x">–</span> <span class="NLM_lpage">1955</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=1945-1955&author=F.+Antignanoauthor=K.+Burrowsauthor=M.+R.+Hughesauthor=J.+M.+Hanauthor=K.+J.+Kronauthor=N.+M.+Penrodauthor=M.+J.+Oudhoffauthor=S.+K.+Wangauthor=P.+H.+Minauthor=M.+J.+Goldauthor=A.+L.+Cheneryauthor=M.+J.+Braamauthor=T.+C.+Fungauthor=F.+M.+Rossiauthor=K.+M.+McNagnyauthor=C.+H.+Arrowsmithauthor=M.+Lupienauthor=M.+K.+Levingsauthor=C.+Zaph&title=Methyltransferase+G9A+regulates+T+cell+differentiation+during+murine+intestinal+inflammation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAntignano%26aufirst%3DF.%26aulast%3DBurrows%26aufirst%3DK.%26aulast%3DHughes%26aufirst%3DM.%2BR.%26aulast%3DHan%26aufirst%3DJ.%2BM.%26aulast%3DKron%26aufirst%3DK.%2BJ.%26aulast%3DPenrod%26aufirst%3DN.%2BM.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%2BK.%26aulast%3DMin%26aufirst%3DP.%2BH.%26aulast%3DGold%26aufirst%3DM.%2BJ.%26aulast%3DChenery%26aufirst%3DA.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DFung%26aufirst%3DT.%2BC.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26aulast%3DMcNagny%26aufirst%3DK.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DLupien%26aufirst%3DM.%26aulast%3DLevings%26aufirst%3DM.%2BK.%26aulast%3DZaph%26aufirst%3DC.%26atitle%3DMethyltransferase%2520G9A%2520regulates%2520T%2520cell%2520differentiation%2520during%2520murine%2520intestinal%2520inflammation%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2014%26volume%3D124%26spage%3D1945%26epage%3D1955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical tool for in vitro and in vivo precipitation of lysine methyltransferase g9a</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">553</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=549-553&author=K.+D.+Konzeauthor=S.+G.+Pattendenauthor=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=J.+M.+Simonauthor=I.+J.+Davisauthor=M.+Vedadiauthor=J.+Jin&title=A+chemical+tool+for+in+vitro+and+in+vivo+precipitation+of+lysine+methyltransferase+g9a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSimon%26aufirst%3DJ.%2BM.%26aulast%3DDavis%26aufirst%3DI.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520tool%2520for%2520in%2520vitro%2520and%2520in%2520vivo%2520precipitation%2520of%2520lysine%2520methyltransferase%2520g9a%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D549%26epage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Nishioka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valenzuela, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steward, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lis, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">PR-Set7 is a nucleosome-specific methyltransferase that modifies lysine 20 of histone H4 and is associated with silent chromatin</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1201</span><span class="NLM_x">–</span> <span class="NLM_lpage">1213</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=1201-1213&author=K.+Nishiokaauthor=J.+C.+Riceauthor=K.+Sarmaauthor=H.+Erdjument-Bromageauthor=J.+Wernerauthor=Y.+Wangauthor=S.+Chuikovauthor=P.+Valenzuelaauthor=P.+Tempstauthor=R.+Stewardauthor=J.+T.+Lisauthor=C.+D.+Allisauthor=D.+Reinberg&title=PR-Set7+is+a+nucleosome-specific+methyltransferase+that+modifies+lysine+20+of+histone+H4+and+is+associated+with+silent+chromatin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNishioka%26aufirst%3DK.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DSarma%26aufirst%3DK.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DWerner%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DValenzuela%26aufirst%3DP.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSteward%26aufirst%3DR.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DPR-Set7%2520is%2520a%2520nucleosome-specific%2520methyltransferase%2520that%2520modifies%2520lysine%252020%2520of%2520histone%2520H4%2520and%2520is%2520associated%2520with%2520silent%2520chromatin%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D9%26spage%3D1201%26epage%3D1213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ketel, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Purification and functional characterization of SET8, a nucleosomal histone H4-lysine 20-specific methyltransferase</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1086</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2FS0960-9822%2802%2900924-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=12121615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1086-1099&author=J.+Fangauthor=Q.+Fengauthor=C.+S.+Ketelauthor=H.+Wangauthor=R.+Caoauthor=L.+Xiaauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=J.+A.+Simonauthor=Y.+Zhang&title=Purification+and+functional+characterization+of+SET8%2C+a+nucleosomal+histone+H4-lysine+20-specific+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and Functional Characterization of SET8, a Nucleosomal Histone H4-Lysine 20-Specific Methyltransferase</span></div><div class="casAuthors">Fang, Jia; Feng, Qin; Ketel, Carrie S.; Wang, Hengbin; Cao, Ru; Xia, Li; Erdjument-Bromage, Hediye; Tempst, Paul; Simon, Jeffrey A.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1086-1099</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Covalent modifications of histone N-terminal tails play fundamental roles in regulating chromatin structure and function.  Extensive studies have established that acetylation of specific lysine residues in the histone tails plays an important role in transcriptional regulation.  Besides acetylation, recent studies have revealed that histone methylation also has significant effects on heterochromatin formation and transcriptional regulation.  Histone methylation occurs on specific arginine and lysine residues of histones H3 and H4.  Thus far, only 2 residues on histone H4 are known to be methylated.  While H4-arginine 3 (H4-R3) methylation is mediated by PRMT1, the enzyme(s) responsible for H4-lysine 20 (H4-K20) methylation is not known.Results: To gain insight into the function of H4-K20 methylation, we set out to identify the enzyme responsible for this modification.  We purified and cloned a novel human SET domain-contg. protein, named SET8, which specifically methylates H4 at K20.  SET8 is a single subunit enzyme and prefers nucleosomal substrates.  We find that H4-K20 methylation occurs in a wide range of higher eukaryotic organisms and that SET8 homologs exist in C. elegans and Drosophila.  We demonstrate that the Drosophila SET8 homolog has the same substrate specificity as its human counterpart.  Importantly, disruption of SET8 in Drosophila reduces levels of H4-K20 methylation in vivo and results in lethality.  Although H4-K20 methylation does not correlate with gene activity, it appears to be regulated during the cell cycle.Conclusions: We identified and characterized an evolutionarily conserved nucleosomal H4-K20-specific methyltransferase and demonstrated its essential role in Drosophila development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7KOsKzI9f47Vg90H21EOLACvtfcHk0liQpufKpuhFOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjsLg%253D&md5=6b0cbdd9ffb2feb7b49017dc045669ce</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2802%2900924-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252802%252900924-7%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DKetel%26aufirst%3DC.%2BS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DSimon%26aufirst%3DJ.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPurification%2520and%2520functional%2520characterization%2520of%2520SET8%252C%2520a%2520nucleosomal%2520histone%2520H4-lysine%252020-specific%2520methyltransferase%26jtitle%3DCurr.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D1086%26epage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Beck, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">PR-Set7 and H4K20me1: at the crossroads of genome integrity, cell cycle, chromosome condensation, and transcription</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">325</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=325-337&author=D.+B.+Beckauthor=H.+Odaauthor=S.+S.+Shenauthor=D.+Reinberg&title=PR-Set7+and+H4K20me1%3A+at+the+crossroads+of+genome+integrity%2C+cell+cycle%2C+chromosome+condensation%2C+and+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DD.%2BB.%26aulast%3DOda%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DS.%2BS.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DPR-Set7%2520and%2520H4K20me1%253A%2520at%2520the%2520crossroads%2520of%2520genome%2520integrity%252C%2520cell%2520cycle%252C%2520chromosome%2520condensation%252C%2520and%2520transcription%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D325%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Yang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Y.</span><span> </span><span class="NLM_article-title">SET8 promotes epithelial–mesenchymal transition and confers TWIST dual transcriptional activities</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=110-123&author=F.+Yangauthor=L.+Sunauthor=Q.+Liauthor=X.+Hanauthor=L.+Leiauthor=H.+Zhangauthor=Y.+Shang&title=SET8+promotes+epithelial%E2%80%93mesenchymal+transition+and+confers+TWIST+dual+transcriptional+activities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DShang%26aufirst%3DY.%26atitle%3DSET8%2520promotes%2520epithelial%25E2%2580%2593mesenchymal%2520transition%2520and%2520confers%2520TWIST%2520dual%2520transcriptional%2520activities%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D110%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kachirskaia, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaguchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appella, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span> </span><span class="NLM_article-title">Modulation of p53 function by SET8-mediated methylation at lysine 382</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">636</span><span class="NLM_x">–</span> <span class="NLM_lpage">646</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=636-646&author=X.+Shiauthor=I.+Kachirskaiaauthor=H.+Yamaguchiauthor=L.+E.+Westauthor=H.+Wenauthor=E.+W.+Wangauthor=S.+Duttaauthor=E.+Appellaauthor=O.+Gozani&title=Modulation+of+p53+function+by+SET8-mediated+methylation+at+lysine+382"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DX.%26aulast%3DKachirskaia%26aufirst%3DI.%26aulast%3DYamaguchi%26aufirst%3DH.%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DE.%2BW.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DModulation%2520of%2520p53%2520function%2520by%2520SET8-mediated%2520methylation%2520at%2520lysine%2520382%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D636%26epage%3D646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akasu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnuma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atomi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Histone lysine methyltransferase SETD8 promotes carcinogenesis by deregulating PCNA expression</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">3217</span><span class="NLM_x">–</span> <span class="NLM_lpage">3227</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=3217-3227&author=M.+Takawaauthor=H.+S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=M.+Kogureauthor=Y.+Yamaneauthor=Y.+Iwaiauthor=K.+Maejimaauthor=K.+Uedaauthor=A.+Masudaauthor=N.+Dohmaeauthor=H.+I.+Fieldauthor=T.+Tsunodaauthor=T.+Kobayashiauthor=T.+Akasuauthor=M.+Sugiyamaauthor=S.+Ohnumaauthor=Y.+Atomiauthor=B.+A.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Histone+lysine+methyltransferase+SETD8+promotes+carcinogenesis+by+deregulating+PCNA+expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DAkasu%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DOhnuma%26aufirst%3DS.%26aulast%3DAtomi%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DHistone%2520lysine%2520methyltransferase%2520SETD8%2520promotes%2520carcinogenesis%2520by%2520deregulating%2520PCNA%2520expression%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D3217%26epage%3D3227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nady, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">L3MBTL1 recognition of mono- and dimethylated histones</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1229</span><span class="NLM_x">–</span> <span class="NLM_lpage">1230</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1038%2Fnsmb1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=18026117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1229-1230&author=J.+Minauthor=A.+Allali-Hassaniauthor=N.+Nadyauthor=C.+Qiauthor=H.+Ouyangauthor=Y.+Liuauthor=F.+MacKenzieauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=L3MBTL1+recognition+of+mono-+and+dimethylated+histones"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">L3MBTL1 recognition of mono- and dimethylated histones</span></div><div class="casAuthors">Min, Jinrong; Allali-Hassani, Abdellah; Nady, Nataliya; Qi, Chao; Ouyang, Hui; Liu, Yongsong; MacKenzie, Farrell; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1229-1230</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Crystal structures of the L3MBTL1 MBT repeats in complex with histone H4 peptides dimethylated on Lys20 (H4K20me2) show that only the second of the three MBT repeats can bind mono- and dimethylated histone peptides.  Its binding pocket has similarities to that of 53BP1 and is able to recognize the degree of histone lysine methylation.  An unexpected mode of peptide-mediated dimerization suggests a possible mechanism for chromatin compaction by L3MBTL1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyCuRY6dV9-bVg90H21EOLACvtfcHk0lg0C4kV1mCd4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtl2hu7rP&md5=1f1072bf117cb7459a644907ca66b74b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1340%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DNady%26aufirst%3DN.%26aulast%3DQi%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMacKenzie%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DL3MBTL1%2520recognition%2520of%2520mono-%2520and%2520dimethylated%2520histones%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1229%26epage%3D1230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Provencher, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span> </span><span class="NLM_article-title">Accessing protein methyltransferase and demethylase enzymology using microfluidic capillary electrophoresis</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">695</span><span class="NLM_x">–</span> <span class="NLM_lpage">704</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1016%2Fj.chembiol.2010.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=20659682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptleitL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=695-704&author=T.+J.+Wigleauthor=L.+M.+Provencherauthor=J.+L.+Norrisauthor=J.+Jinauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=W.+P.+Janzen&title=Accessing+protein+methyltransferase+and+demethylase+enzymology+using+microfluidic+capillary+electrophoresis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Accessing Protein Methyltransferase and Demethylase Enzymology Using Microfluidic Capillary Electrophoresis</span></div><div class="casAuthors">Wigle, Tim J.; Provencher, Laurel M.; Norris, Jacqueline L.; Jin, Jian; Brown, Peter J.; Frye, Stephen V.; Janzen, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">695-704</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The discovery of small mols. targeting the >80 enzymes that add (methyltransferases) or remove (demethylases) Me marks from lysine and arginine residues, most notably present in histone tails, may yield unprecedented chemotherapeutic agents and facilitate regenerative medicine.  To better enable chem. exploration of these proteins, a highly quant. microfluidic capillary electrophoresis assay was developed to enable full mechanistic studies of these enzymes and the kinetics of their inhibition.  This technol. separates small biomols., i.e., peptides, based on their charge-to-mass ratio.  Methylation, however, does not alter the charge of peptide substrates.  To overcome this limitation, a methylation-sensitive endoproteinase strategy was employed to sep. methylated from unmethylated peptides.  The assay was validated on a lysine methyltransferase (G9a) and a lysine demethylase (LSD1) and was employed to investigate the inhibition of G9a by small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPcF6--FJ3u7Vg90H21EOLACvtfcHk0lg0C4kV1mCd4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptleitL8%253D&md5=fe13fe9e2120b40edb4ea90d67cb529e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DProvencher%26aufirst%3DL.%2BM.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26atitle%3DAccessing%2520protein%2520methyltransferase%2520and%2520demethylase%2520enzymology%2520using%2520microfluidic%2520capillary%2520electrophoresis%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D695%26epage%3D704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeon, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pae, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. Y.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel T-type calcium channel blockers</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">–</span> <span class="NLM_lpage">475</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=471-475&author=J.+Y.+Choiauthor=H.+N.+Seoauthor=M.+J.+Leeauthor=S.+J.+Parkauthor=J.+Y.+Jeonauthor=J.+H.+Kangauthor=A.+N.+Paeauthor=H.+Rhimauthor=J.+Y.+Lee&title=Synthesis+and+biological+evaluation+of+novel+T-type+calcium+channel+blockers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DSeo%26aufirst%3DH.%2BN.%26aulast%3DLee%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DS.%2BJ.%26aulast%3DJeon%26aufirst%3DJ.%2BY.%26aulast%3DKang%26aufirst%3DJ.%2BH.%26aulast%3DPae%26aufirst%3DA.%2BN.%26aulast%3DRhim%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520T-type%2520calcium%2520channel%2520blockers%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D471%26epage%3D475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Structural and functional analysis of SET8, a histone H4 Lys-20 methyltransferase</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1455</span><span class="NLM_x">–</span> <span class="NLM_lpage">1465</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm500871s&amp;key=10.1101%2Fgad.1318405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm500871s&amp;key=15933070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm500871s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=1455-1465&author=J.+F.+Coutureauthor=E.+Collazoauthor=J.+S.+Brunzelleauthor=R.+C.+Trievel&title=Structural+and+functional+analysis+of+SET8%2C+a+histone+H4+Lys-20+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional analysis of SET8, a histone H4 Lys-20 methyltransferase</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Brunzelle, Joseph S.; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1455-1465</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">SET8 (also known as PR-SET7) is a histone H4-Lys-20-specific methyltransferase that is implicated in cell-cycle-dependent transcriptional silencing and mitotic regulation in metazoans.  Herein we report the crystal structure of human SET8 (hSET8) bound to a histone H4 peptide bearing Lys-20 and the product cofactor S-adenosylhomocysteine.  Histone H4 intercalates in the substrate-binding cleft as an extended parallel β-strand.  Residues preceding Lys-20 in H4 engage in an extensive array of salt bridge, hydrogen bond, and van der Waals interactions with hSET8, while the C-terminal residues bind through predominantly hydrophobic interactions.  Mutational anal. of both the substrate-binding cleft and histone H4 reveals that interactions with residues in the N and C termini of the H4 peptide are crit. for conferring substrate specificity.  Finally, anal. of the product specificity indicates that hSET8 is a monomethylase, consistent with its role in the maintenance of Lys-20 monomethylation during cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphqNcPOAjIbbVg90H21EOLACvtfcHk0lg0C4kV1mCd4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D&md5=26dd3cdbaee412a5904ba8268b8efb40</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1101%2Fgad.1318405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1318405%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520and%2520functional%2520analysis%2520of%2520SET8%252C%2520a%2520histone%2520H4%2520Lys-20%2520methyltransferase%26jtitle%3DGenes%2520Dev.%26date%3D2005%26volume%3D19%26spage%3D1455%26epage%3D1465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibition of MLL activity by disruption of its interaction with WDR5</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=151-159&author=G.+Senisterraauthor=H.+Wuauthor=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=D.+Barsyte-Lovejoyauthor=L.+Dombrovskiauthor=A.+Dongauthor=K.+T.+Nguyenauthor=D.+Smilauthor=Y.+Bolshanauthor=T.+Hajianauthor=H.+Heauthor=A.+Seitovaauthor=I.+Chauauthor=F.+Liauthor=G.+Podaauthor=J.+F.+Coutureauthor=P.+J.+Brownauthor=R.+Al-Awarauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Small-molecule+inhibition+of+MLL+activity+by+disruption+of+its+interaction+with+WDR5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSeitova%26aufirst%3DA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520MLL%2520activity%2520by%2520disruption%2520of%2520its%2520interaction%2520with%2520WDR5%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D151%26epage%3D159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i47"><a href="/doi/suppl/10.1021/jm500871s">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_53606"></div></div></div></div></div><hr /></hr><p class="last"><sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>1</b>. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm500871s/suppl_file/jm500871s_si_001.pdf">jm500871s_si_001.pdf (172.04 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm500871s&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2014.57.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Fjm500871s%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm500871s" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                1MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679980ab6bbd3c4c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
