
<html lang="en"     class="pb-page"  data-request-id="c9a96d5d-b0d5-4841-b40c-b42a9232fe1d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b00071;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Complexity of Blocking Bivalent Protein–Protein Interactions: Development of a Highly Potent Inhibitor of the Menin–Mixed-Lineage Leukemia Interaction" /></meta><meta name="dc.Creator" content="Dmitry  Borkin" /></meta><meta name="dc.Creator" content="Szymon  Klossowski" /></meta><meta name="dc.Creator" content="Jonathan  Pollock" /></meta><meta name="dc.Creator" content="Hongzhi  Miao" /></meta><meta name="dc.Creator" content="Brian M.  Linhares" /></meta><meta name="dc.Creator" content="Katarzyna  Kempinska" /></meta><meta name="dc.Creator" content="Zhuang  Jin" /></meta><meta name="dc.Creator" content="Trupta  Purohit" /></meta><meta name="dc.Creator" content="Bo  Wen" /></meta><meta name="dc.Creator" content="Miao  He" /></meta><meta name="dc.Creator" content="Duxin  Sun" /></meta><meta name="dc.Creator" content="Tomasz  Cierpicki" /></meta><meta name="dc.Creator" content="Jolanta  Grembecka" /></meta><meta name="dc.Description" content="The protein–protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors. ..." /></meta><meta name="Description" content="The protein–protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors. ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 8, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00071" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00071" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00071" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00071" /></link>
        
    
    

<title>Complexity of Blocking Bivalent Protein–Protein Interactions: Development of a Highly Potent Inhibitor of the Menin–Mixed-Lineage Leukemia Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00071" /></meta><meta property="og:title" content="Complexity of Blocking Bivalent Protein–Protein Interactions: Development of a Highly Potent Inhibitor of the Menin–Mixed-Lineage Leukemia Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0018.jpeg" /></meta><meta property="og:description" content="The protein–protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors. Here, we report the development of a new generation of menin–MLL1 inhibitors identified by structure-based optimization of the thienopyrimidine class of compounds. This work resulted in compound 28 (MI-1481), which showed very potent inhibition of the menin–MLL1 interaction (IC50 = 3.6 nM), representing the most potent reversible menin–MLL1 inhibitor reported to date. The crystal structure of the menin-28 complex revealed a hydrogen bond with Glu366 and hydrophobic interactions, which contributed to strong inhibitory activity of 28. Compound 28 also demonstrates pronounced activity in MLL leukemia cells and in vivo in MLL leukemia models. Thus, 28 is a valuable menin–MLL1 inhibitor that can be used for potential therapeutic applications and in further studies regarding the role of menin in cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00071"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00071">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00071&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00071&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00071&amp;href=/doi/10.1021/acs.jmedchem.8b00071" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 4832-4850</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01826" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00140" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Complexity of Blocking Bivalent Protein–Protein Interactions: Development of a Highly Potent Inhibitor of the Menin–Mixed-Lineage Leukemia Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Dmitry Borkin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dmitry Borkin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dmitry++Borkin">Dmitry Borkin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Szymon Klossowski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Szymon Klossowski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Szymon++Klossowski">Szymon Klossowski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jonathan Pollock</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan Pollock</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan++Pollock">Jonathan Pollock</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongzhi Miao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongzhi Miao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongzhi++Miao">Hongzhi Miao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brian M. Linhares</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brian M. Linhares</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brian+M.++Linhares">Brian M. Linhares</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katarzyna Kempinska</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katarzyna Kempinska</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katarzyna++Kempinska">Katarzyna Kempinska</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhuang Jin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhuang Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhuang++Jin">Zhuang Jin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Trupta Purohit</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Trupta Purohit</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Trupta++Purohit">Trupta Purohit</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bo Wen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bo Wen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Wen">Bo Wen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Miao He</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Miao He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Miao++He">Miao He</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Duxin Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Duxin Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Duxin++Sun">Duxin Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tomasz Cierpicki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tomasz Cierpicki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tomasz++Cierpicki">Tomasz Cierpicki</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jolanta Grembecka</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jolanta Grembecka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#2e4441424f405a4f496e5b43474d46004b4a5b"><span class="__cf_email__" data-cfemail="640e0b08050a1005032411090d070c4a010011">[email protected]</span></a>. Tel.: 734-615-9319. Fax: 734-763-8764.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jolanta++Grembecka">Jolanta Grembecka</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6180-9095" title="Orcid link">http://orcid.org/0000-0002-6180-9095</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00071&amp;href=/doi/10.1021%2Facs.jmedchem.8b00071" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 4832–4850</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 8, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 January 2018</li><li><span class="item_label"><b>Published</b> online</span>8 May 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 June 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00071" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00071</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4832%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDmitry%2BBorkin%252C%2BSzymon%2BKlossowski%252C%2BJonathan%2BPollock%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D11%26contentID%3Dacs.jmedchem.8b00071%26title%3DComplexity%2Bof%2BBlocking%2BBivalent%2BProtein%25E2%2580%2593Protein%2BInteractions%253A%2BDevelopment%2Bof%2Ba%2BHighly%2BPotent%2BInhibitor%2Bof%2Bthe%2BMenin%25E2%2580%2593Mixed-Lineage%2BLeukemia%2BInteraction%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4850%26publicationDate%3DJune%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00071"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2786</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">26</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00071" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Complexity of Blocking Bivalent Protein–Protein Interactions: Development of a Highly Potent Inhibitor of the Menin–Mixed-Lineage Leukemia Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dmitry&quot;,&quot;last_name&quot;:&quot;Borkin&quot;},{&quot;first_name&quot;:&quot;Szymon&quot;,&quot;last_name&quot;:&quot;Klossowski&quot;},{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;Pollock&quot;},{&quot;first_name&quot;:&quot;Hongzhi&quot;,&quot;last_name&quot;:&quot;Miao&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;M. Linhares&quot;},{&quot;first_name&quot;:&quot;Katarzyna&quot;,&quot;last_name&quot;:&quot;Kempinska&quot;},{&quot;first_name&quot;:&quot;Zhuang&quot;,&quot;last_name&quot;:&quot;Jin&quot;},{&quot;first_name&quot;:&quot;Trupta&quot;,&quot;last_name&quot;:&quot;Purohit&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Wen&quot;},{&quot;first_name&quot;:&quot;Miao&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Duxin&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Tomasz&quot;,&quot;last_name&quot;:&quot;Cierpicki&quot;},{&quot;first_name&quot;:&quot;Jolanta&quot;,&quot;last_name&quot;:&quot;Grembecka&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;4832-4850&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00071&quot;},&quot;abstract&quot;:&quot;The protein–protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the MLL1 gene and in solid tumors. Here, we report the development of a new generation of menin–MLL1 inhibitors identified by structure-based optimization of the thienopyrimidine class of compounds. This work resulted in compound 28 (MI-1481), which showed very potent inhibition of the menin–MLL1 interaction (IC50 = 3.6 nM), representing the most potent reversible menin–MLL1 inhibitor reported to date. The crystal structure of the menin-28 complex revealed a hydrogen bond with Glu366 and hydrophobic interactions, which contributed to strong inhibitory activity of 28. Compound 28 also demonstrates pronounced activity in MLL leukemia cells and in vivo in MLL leukemia models. Thus, 28 is a valuable menin–MLL1 inhibitor that can be used for potential therapeutic applications and in further studies regarding the role of menin in cancer.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00071&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00071" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00071&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00071" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00071&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00071" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00071&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00071&amp;href=/doi/10.1021/acs.jmedchem.8b00071" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00071" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00071" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00071%26sid%3Dliteratum%253Aachs%26pmid%3D29738674%26genre%3Darticle%26aulast%3DBorkin%26date%3D2018%26atitle%3DComplexity%2Bof%2BBlocking%2BBivalent%2BProtein%25E2%2580%2593Protein%2BInteractions%253A%2BDevelopment%2Bof%2Ba%2BHighly%2BPotent%2BInhibitor%2Bof%2Bthe%2BMenin%25E2%2580%2593Mixed-Lineage%2BLeukemia%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D11%26spage%3D4832%26epage%3D4850%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291862" title="Indoles">Indoles</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/jmcmar.2018.61.issue-11/20180614/jmcmar.2018.61.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The protein–protein interaction between menin and mixed-lineage leukemia 1 (MLL1) plays an important role in development of acute leukemia with translocations of the <i>MLL1</i> gene and in solid tumors. Here, we report the development of a new generation of menin–MLL1 inhibitors identified by structure-based optimization of the thienopyrimidine class of compounds. This work resulted in compound <b>28</b> (<b>MI-1481</b>), which showed very potent inhibition of the menin–MLL1 interaction (IC<sub>50</sub> = 3.6 nM), representing the most potent reversible menin–MLL1 inhibitor reported to date. The crystal structure of the menin-<b>28</b> complex revealed a hydrogen bond with Glu366 and hydrophobic interactions, which contributed to strong inhibitory activity of <b>28</b>. Compound <b>28</b> also demonstrates pronounced activity in MLL leukemia cells and <i>in vivo</i> in MLL leukemia models. Thus, <b>28</b> is a valuable menin–MLL1 inhibitor that can be used for potential therapeutic applications and in further studies regarding the role of menin in cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Menin is a scaffold protein that is involved in a network of protein–protein interactions, among which the interaction with the mixed-lineage leukemia 1 (MLL1; hereafter referred to simply as MLL) protein or MLL fusion proteins have been recognized as highly relevant to human diseases, specifically to cancer.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The <i>MLL1</i> gene is translocated in 5–10% of acute leukemia cases, and this translocation leads to the expression of MLL fusion proteins, which are critical drivers of leukemogenesis.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> The MLL patients have an approximately 35% five-year survival rate,<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> which reflects poor response to currently available treatments and supports the need for the development of new therapies. The interaction of menin with MLL fusion proteins plays a key role in leukemogenesis, which was demonstrated before using genetic approaches.<a onclick="showRef(event, 'ref2 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref2 ref6 ref7">(2,6,7)</a> The importance of this interaction in MLL leukemia has been further validated by small molecule inhibitors of the menin–MLL interaction that we have reported in earlier studies.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12">(8−12)</a> Furthermore, the interaction of menin with the wild-type MLL plays an important role in solid tumors, including castration resistant prostate cancer,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Ewing sarcoma,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> hepatocellular carcinoma (HCC),<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> and pediatric gliomas.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Therefore, the development of small molecule inhibitors of the menin–MLL interaction with optimized drug-like properties may lead to new therapeutics for acute leukemia and a subset of solid tumors.</div><div class="NLM_p">The development of potent inhibitors with favorable drug-like properties that target protein–protein interactions (PPI) is still a challenge.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> The specific architecture of protein–protein interfaces (e.g., the lack of well-defined pockets and large contact areas) frequently implies the need for increasing the molecular weight of PPI inhibitors, which makes it difficult to achieve strong potency and favorable drug-like properties.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Nevertheless, successful examples of PPI inhibitors exist,<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21−24)</a> including compounds advanced to clinical trials (e.g., Bcl-2 family inhibitors, p53 activators<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a>), thus supporting efforts in this direction. Furthermore, the recent FDA approval of the BCL-2 inhibitor Venetoclax demonstrates that PPI inhibitors can be valuable therapeutics, despite challenges in obtaining favorable drug-like properties for such compounds as a result of inherently higher than average molecular weight (<i>M</i><sub>w</sub> > 800 Da for Venetoclax).</div><div class="NLM_p">The protein–protein interaction between menin and MLL or MLL fusion proteins involves the N-terminal fragment of MLL (MLL<sub>4-43</sub>),<a onclick="showRef(event, 'ref2 ref6'); return false;" href="javascript:void(0);" class="ref ref2 ref6">(2,6)</a> which binds to menin with a strong binding affinity (<i>K</i><sub>d</sub> = 6.8 nM) by utilizing the bivalent binding mode, with MBM1 (menin-binding motif 1, MLL<sub>4–15</sub>) and MBM2 (menin-binding motif 2, MLL<sub>23–43</sub>) motifs involved in the interactions with menin.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Structural studies have revealed that MLL binds to a large central cavity on menin,<a onclick="showRef(event, 'ref9 ref28'); return false;" href="javascript:void(0);" class="ref ref9 ref28">(9,28)</a> thus supporting the existence of pockets to bind small molecules. Indeed, our previous studies have validated that the menin–MLL interaction can be effectively blocked with small molecules that bind to the MLL binding site on menin.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12">(8−12)</a> However, due to the extended menin–MLL interface resulting from the bivalent binding mode of MLL to menin, optimization of the potency and drug-like properties of menin–MLL inhibitors remains challenging.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">Our previous efforts resulted in the development of the thienopyrimidine class of menin–MLL inhibitors, with <b>MI-463</b>, <b>MI-503</b>, and <b>MI-538</b> demonstrating the most potent inhibitory activity, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> These compounds selectively block proliferation of human MLL leukemia cells (GI<sub>50</sub> = 200–500 nM), demonstrate specific mechanism of action and delay leukemia progression in mice models of MLL leukemia.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a><b>MI-503</b> also demonstrated activity in prostate cancer,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Ewing sarcoma,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and hepatocellular carcinoma<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> models. However, menin–MLL inhibitors with improved therapeutic potential over <b>MI-503</b>, namely with increased potency (e.g., IC<sub>50</sub> < 10 nM to make them comparable or more potent than MLL<sub>4-43</sub>, GI<sub>50</sub> < 50 nM and >100-fold selectivity to MLL leukemia cells, strong reduction in leukemia burden and >50% decrease in target gene expression <i>in vivo</i>) are still required. In this study we report structure-based optimization of the <b>MI-463</b> and <b>MI-503</b> menin–MLL inhibitors by exploring a collection of saturated six-membered rings introduced at the indole nitrogen of these compounds to further improve their activity and drug-like properties. These efforts resulted in the identification of compound <b>28</b> (<b>MI-1481</b>), which demonstrated low nanomolar activity (IC<sub>50</sub> = 3.6 nM) for the inhibition of the interaction between menin and the bivalent MLL<sub>4-43</sub> encompassing the entire menin binding motif on MLL. Thus, <b>28</b> demonstrates >10-fold improved inhibitory activity over <b>MI-503</b>, representing the most potent reversible small molecule menin–MLL inhibitor reported to date. Compound <b>28</b> also demonstrated substantially increased activity in MLL leukemia cells (∼10-fold) over the previous generation of reversible menin–MLL inhibitors,<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> high selectivity toward MLL leukemia cells and pronounced <i>in vivo</i> activity in a mice model of MLL leukemia. We also report the co-crystal structure of the menin–<b>28</b> complex, which revealed that both polar and hydrophobic interactions likely contribute to the strong inhibitory activity of this compound. This compound could serve as a valuable chemical probe to further study the role of menin in pathological and physiological states and could lead to the development of a new therapeutic agent for tumors that rely on the menin–MLL interaction.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures and binding mode of the previous generation of menin–MLL inhibitors. (a) Chemical structures of the thienopyrimidine menin–MLL inhibitors <b>MI-463</b>, <b>MI-503</b>, and <b>MI-538</b> together with the inhibitory activity of these compounds measured in the fluorescence polarization (FP) competition assays with menin and the fluorescein-labeled MLL<sub>4–15</sub> or MLL<sub>4-43</sub> peptides. (b) Crystal structure of menin in complex with <b>MI-503</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X5Y">4X5Y</a>). Protein is shown in ribbon representation, and key menin residues involved in interactions with <b>MI-503</b> are shown as sticks. <b>MI-503</b> is shown in stick representation with colors corresponding to the atom type (green, carbons; dark blue, nitrogens; yellow, sulfur; light blue, fluorine). Dashed lines represent hydrogen bonds between <b>MI-503</b> and menin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01416" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01416" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Biochemical Assay To Assess Potent Menin–MLL Inhibitors</h3><div class="NLM_p">We have recently reported the thienopyrimidine class of menin–MLL interaction, with the most potent <b>MI-463</b> (compound <b>1</b>) and <b>MI-503</b> (compound <b>2</b>), <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> These compounds bind to menin with <i>K</i><sub>d</sub> ≈ 10 nM and block the interaction of menin with a short MLL-derived peptide MBM1 (MLL<sub>4-15</sub>) with nanomolar inhibitory activities (IC<sub>50</sub> ≈ 15 nM for both compounds) as measured in the fluorescence polarization (FP) assay, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a. However, in the FP competition assay the low end of inhibitor K<sub>i</sub> values that can be accurately measured is the <i>K</i><sub>d</sub> value of the fluorescent ligand, and for inhibitors more potent than the fluorescent probe the IC<sub>50</sub> values are independent of inhibitor potency but linearly proportional to the <i>K</i><sub>d</sub> value of the fluorescence ligand.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Based on the modest binding affinity of the fluorescein-labeled (FLSN) MBM1 to menin (<i>K</i><sub>d</sub> = 49 nM),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> we concluded that menin–MLL inhibitors with low nanomolar binding affinity or better would reach the detection limit of the competition FP assay that uses FLSN_MBM1. To avoid this issue, we utilized the bivalent fragment of MLL (MLL<sub>4-43</sub>) in the competition FP assay for testing of the new generation of menin–MLL inhibitors. The MLL<sub>4-43</sub> includes the entire menin binding motif (MBM1 and MBM2)<a onclick="showRef(event, 'ref6 ref27'); return false;" href="javascript:void(0);" class="ref ref6 ref27">(6,27)</a> and binds to menin with <i>K</i><sub>d</sub> = 6.8 nM as determined by isothermal titration calorimetry (ITC). First, we determined the binding affinity of the fluorescein-labeled MLL<sub>4-43</sub> peptide (FLSN-MLL<sub>4-43</sub>) to menin in this new FP assay, resulting in <i>K</i><sub>d</sub> = 1.0 nM, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Figure 1</a>, which remains in a good agreement with the results from the ITC experiment.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> We then applied the FP competition assay with FLSN-MLL<sub>4-43</sub> to test the inhibitory activity of <b>MI-463</b> and <b>MI-503</b> menin–MLL inhibitors, resulting in IC<sub>50</sub> values of 32 and 33 nM, respectively, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Because MLL<sub>4–43</sub> encompasses the entire menin binding motif, these data support that the thienopyrimidine compounds (e.g., <b>MI-463</b> and <b>MI-503</b>) can effectively displace the bivalent MLL fragment from menin, which is an important feature in the context of functional studies. Overall, based on these results we concluded that the FP assay utilizing the bivalent MLL<sub>4-43</sub> peptide represents an appropriate approach to provide accurate activity measurements for low nanomolar inhibitors of the menin–MLL interactions. Thus, we used this assay to assess the <i>in vitro</i> activity of all compounds reported in this study.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR and Properties of <b>MI-463</b> Derivatives with Various Rings Introduced at Indole Nitrogen</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0010.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0011.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were measured by fluorescence polarization assay using fluorescein-labeled MLL<sub>4-43</sub>. Average values from two or three independent measurements ± SD are provided.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Growth inhibition (GI<sub>50</sub> values) measured in the MTT cell viability assay after 7 days of treatment of murine bone marrow cells transformed with MLL-AF9.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Half-life of compounds in mouse liver microsomes.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Calculated with ChemBioDraw Ultra 14.0.</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Development of Menin–MLL Inhibitors with Saturated Rings Substituting Indole Nitrogen</h3><div class="NLM_p">In our previous studies, we developed <b>MI-463</b> and its analogues with different substitutions at the indole ring, including aliphatic and aromatic substituents on indole nitrogen (e.g., <b>MI-503</b>), <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a. However, these modifications did not result in increased inhibitory activity as compared to <b>MI-463</b>.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> To further optimize these compounds we performed detailed analysis of the co-crystal structure of menin in complex with <b>MI-503</b>, which revealed three hydrogen bonds (with Tyr276, Trp341, and Glu366) and hydrophobic interactions with menin,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b. Furthermore, we found that the methyl-pyrazole ring substituting indole nitrogen in <b>MI-503</b> is partly solvent exposed, with two negatively charged residues on menin, Glu363 and Glu366, approaching this ring, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b. Based on this analysis, we designed various saturated rings harboring positively charged and/or uncharged polar groups to be introduced at the indole nitrogen of <b>MI-463</b> to form electrostatic interactions or hydrogen bonds with this area of the binding site on menin.</div><div class="NLM_p">First, we explored several saturated rings with short aliphatic linkers substituting indole nitrogen of <b>MI-463</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Introduction of the piperazine or piperidine rings, which resulted in compounds <b>3</b> and <b>4</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, led to a ∼2-fold improved inhibitory activity over <b>MI-463</b> (IC<sub>50</sub> = 15 and 14 nM for <b>3</b> and <b>4</b>, respectively). On the other hand, incorporation of the morpholine (compound <b>5</b>) or morpholinone (compound <b>6</b>) rings connected to indole through an ethyl linker resulted in less pronounced activity improvement (<1.5-fold) over <b>MI-463</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Interestingly, the introduction of the methyl-morpholinone substituent at indole nitrogen (compound <b>7</b>, <b>MI-568</b>) resulted in an over 4-fold improvement in the IC<sub>50</sub> value (IC<sub>50</sub> = 7.5 nM for the racemic mixture of <b>7</b>) as compared to <b>MI-463</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Thus, although <b>7</b> represents the racemic mixture of two enantiomers, this compound demonstrates low nanomolar inhibition of the menin–MLL interaction, outperforming other compounds with various saturated rings introduced to the scaffold, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_p last">We then assessed the cellular activity in MLL leukemia cells and the microsomal stability of compounds <b>3</b>–<b>7</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Notably, compound <b>7</b> demonstrated the most pronounced effect on proliferation of the MLL-AF9 transformed leukemia cells (GI<sub>50</sub> = 50 nM), representing ∼5-fold stronger cellular activity than <b>MI-463</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The cellular activities of compounds <b>3</b>–<b>6</b> are less pronounced and range from 0.15 to 0.81 μM, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Interestingly, despite improved inhibitory activity, compounds <b>3</b> and <b>4</b> showed weaker cell growth inhibition as compared to <b>MI-463</b>, which may have resulted from the positive charge in the piperazine or piperidine rings of these compounds, possibly affecting their cellular permeability. In addition, compounds <b>3</b>, <b>4</b>, and <b>7</b> showed the best microsomal stability measured in mouse liver microsomes (<i>T</i><sub>1/2</sub> > 60 min), representing substantial improvement over <b>MI-463</b> or <b>MI-503</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Furthermore, compound <b>7</b> has also the lowest clogP value, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, thus supporting favorable polarity and solubility. Overall, these results support that the analogues of <b>MI-463</b> with saturated rings introduced at indole nitrogen represent valuable candidates for further optimization.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Optimization of Analogues with Piperazine Ring</h3><div class="NLM_p">Since compound <b>3</b> showed 2-fold improvement in inhibitory activity and substantially increased metabolic stability over <b>MI-463</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, we pursued further optimization of <b>3</b> by introducing substituents to the piperazine ring. First, we introduced aliphatic groups, such as methyl (compound <b>8</b>) and <i>n</i>-propyl (compound <b>9</b>), but they did not improve substantially the <i>in vitro</i> inhibitory activity or cell growth inhibition in MLL-AF9 cells over <b>3</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Analysis of the crystal structure of the menin–<b>MI-503</b> complex revealed the presence of Arg330,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> which could potentially be involved in hydrogen bond/salt bridge with ligands that harbor the appropriate substituent at the indole nitrogen of <b>MI-463</b>. Therefore, we introduced the formamide group to the piperazine ring (compound <b>10</b>), but this modification did not improve inhibitory activity (IC<sub>50</sub> = 13 nM for <b>10</b>), although activity in the MLL-AF9 transformed cells demonstrated a 3-fold improvement for <b>10</b> over <b>3</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. This was likely due to the neutralization of the positive charge in <b>3</b>, which could have positively affected the cellular permeability. Further substitution of the amide with methyl (compound <b>11</b>) did not significantly change the inhibitory activity, while the ethyl analogue (compound <b>12</b>) showed an over 6-fold reduction in inhibitory activity (IC<sub>50</sub> = 83 nM) as compared to <b>10</b>. Interestingly, the analogue with the monofluoromethyl group (compound <b>13</b>, <b>MI-853</b>) demonstrated slightly improved inhibitory activity (IC<sub>50</sub> = 10 nM) over <b>10</b>, while the di- and trifluoromethyl analogues (<b>14</b> and <b>15</b>) were 2–5-fold less potent than <b>10</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Improved <i>in vitro</i> activity of <b>13</b> resulted in more potent cell growth inhibition of the MLL-AF9 transformed cells observed for this compound (GI<sub>50</sub> = 75 nM), <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Furthermore, introduction of a larger hydrophobic group substituting amide in <b>10</b>, which resulted in compounds <b>16</b> and <b>17</b>, led to a 3–4-fold reduced inhibitory activity, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Finally, we introduced the 2-propanamide group (compound <b>18</b>) but did not observe increased inhibitory activity for the racemic mixture of this compound over <b>10</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR and Properties of N-Substituted Analogues of Compound <b>3</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0012.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0013.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were measured by fluorescence polarization assay using fluorescein-labeled MLL <sub>4-43</sub>; average values from two or three independent measurements ± SD are provided.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Growth inhibition (GI<sub>50</sub> values) measured in the MTT cell viability assay after 7 days of treatment of murine bone marrow cells transformed with MLL-AF9.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Half-life of compounds in mouse liver microsomes.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Calculated with ChemBioDraw Ultra 14.0.</p></div></div><div></div></div><div class="NLM_p">To further explore substitutions at the piperazine ring of <b>3</b>, we substituted this ring with sulfonyl or sulfonamide groups (compounds <b>19</b>–<b>23</b>), <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Among these compounds, the analogue with the methyl sulfonamide (compound <b>19</b>) demonstrated the most pronounced inhibitory activity for blocking the menin–MLL interaction (IC<sub>50</sub> = 13 nM), <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. The structure–activity relationship (SAR) for the sulfonyl analogues (compounds <b>19</b>–<b>23</b>) is similar to the corresponding amide derivatives, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, with reduced inhibitory activity upon increasing the size of the aliphatic substituent (e.g., compounds <b>20</b> and <b>21</b>), <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. The sulfamide derivatives <b>22</b> and <b>23</b> showed about 2-fold reduction in inhibitory activity as compared to the methylsulfone analogue <b>19</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_p last">Despite comparable or weaker inhibitory activity in <i>in vitro</i> biochemical assay, the majority of compounds with amides, sulfones or sulfonamides substituting the piperazine ring in <b>3</b> demonstrated stronger cell growth inhibition of the MLL-AF9 transformed cells (GI<sub>50</sub> < 200 nM) as compared to compound <b>3</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. This may have resulted from the reduced basicity of these compounds over <b>3</b>, which could have affected their cellular permeability. Importantly, the piperazine analogues demonstrated good correlation of <i>in vitro</i> inhibitory activity with cell growth inhibition, with compound <b>13</b> showing the strongest inhibitory activity (IC<sub>50</sub> = 10 nM) for blocking the menin–MLL interaction and inhibition of cell proliferation in MLL-AF9 leukemia cells (GI<sub>50</sub> = 75 nM), <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. On the other hand, the microsomal stability of the amide and sulfonamide analogues is reduced as compared to compound <b>3</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, although compounds <b>19</b> and <b>22</b> still retain relatively good microsomal stability (<i>T</i><sub>1/2</sub> > 25 min).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Development of Analogues with Substituted Piperidine Ring</h3><div class="NLM_p">Because unsubstituted piperazine (<b>3</b>) and piperidine (<b>4</b>) derivatives have similar inhibitory activity in blocking the menin–MLL interaction, we also explored several analogues derived from <b>4</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. In contrast to the corresponding piperazine analogues, the introduction of the amide group (compound <b>24</b>) and its further substitution with methyl (compound <b>25</b>) resulted in a 5–10-fold reduction in inhibitory activity as compared to <b>4</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Similarly, the incorporation of the sulfonamide (compound <b>26</b>) or sulfamide (compound <b>27</b>) groups to this scaffold resulted in >2-fold reduced inhibitory activity over <b>4</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Based on these results, we concluded that the reduced basicity of the piperidine series, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, makes these compounds less potent menin–MLL inhibitors than the corresponding piperazine derivatives (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) possibly due to the absence of long-range electrostatic interactions with Glu363 and/or Glu366 side chains, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b. Interestingly, despite weaker <i>in vitro</i> inhibitory activity, all piperidine analogues demonstrated comparable or increased activity in MLL leukemia cells over <b>4</b> (e.g., compound <b>26</b> has over 3-fold improved GI<sub>50</sub> as compared to <b>4</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Furthermore, compounds <b>24</b>-<b>27</b> demonstrate good microsomal stability, with the sulfonyl (<b>26</b>) or sulfonamide (<b>27</b>) analogues exhibiting the longest half-life in mouse liver microsomes (<i>T</i><sub>1/2</sub> > 60 min).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR and Properties of N-Substituted Analogues of Compound <b>4</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0014.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0015.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were measured by fluorescence polarization assay using fluorescein-labeled MLL<sub>4-43</sub>; average values from two or three independent measurements ± SD are provided.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Growth inhibition (GI<sub>50</sub> values) measured in the MTT cell viability assay after 7 days of treatment of murine bone marrow cells transformed with MLL-AF9.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Half-life of compounds in mouse liver microsomes.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Calculated with ChemBioDraw Ultra 14.0.</p></div></div><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structural Studies Reveal the Binding Mode of Piperazine Derivatives to Menin</h3><div class="NLM_p">To understand the molecular basis of the interactions of the piperazine analogues with menin, we co-crystallized menin in complex with <b>13</b>, which represents the most potent compound from this series, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, and solved a high-resolution crystal structure of the complex, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Table 1</a>. The crystal structure revealed that the binding mode of <b>13</b> to menin is similar as that observed for <b>MI-503</b>, with two hydrogen bonds (with Tyr276 and Trp341) and hydrophobic interactions retained in <b>13</b> within the core structure preserved in both compounds, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a. The differences in the binding mode of <b>13</b> versus <b>MI-503</b> were only observed for binding of the substituent introduced at indole nitrogen, which varies between these two compounds, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. The ethyl linker connecting indole with the piperazine ring in <b>13</b> is involved in hydrophobic interactions with the side chains of Glu363, Met322, Val367, and Glu366 on menin, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a. One of the nitrogen atoms in the piperazine ring of <b>13</b> points toward the carboxyl group of Glu366 (distance ∼4.5 Å), supporting the long-range electrostatic interactions, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a. Furthermore, one side of the piperidine ring in <b>13</b> is involved in the hydrophobic contacts with Val367 and Val371 on menin, while the other side of this ring in exposed to the solvent, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a. In addition, the oxygen atom from the fluoroacetamide group of <b>13</b> forms a hydrogen bond with the side chain of Arg330 (3.3 Å), while the fluoromethyl group is involved in the hydrophobic interactions with the side chains of Cys329 and Arg330 with the fluorine atom approaching Cys329 and Gly326, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a. Overall, favorable interactions of the fluoroacetamide group in <b>13</b> with menin result in the most pronounced activity of this compound among all amide or sulfonamide derivatives, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. The crystal structure also demonstrates limited space in the binding site to accommodate hydrophobic groups larger than acetamide, reflected by the reduced activity of such analogues (e.g., compounds <b>12</b> and <b>14</b>–<b>17</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Overall, the crystal structure of menin in complex with <b>13</b> supports that both hydrophobic contacts and hydrogen bond with Arg330 on menin contribute to the pronounced inhibitory activity of this compound.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure-based optimization results in potent menin–MLL inhibitors. (a) Crystal structure of the menin–compound <b>13</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXH">6BXH</a>). Residues on menin involved in the interactions with the piperazine ring and the terminal amide of <b>13</b> are shown in stick representation. Hydrogen bond with Arg330 is marked. (b) Crystal structure of the menin–compound <b>28</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXY">6BXY</a>). Residues on menin involved in the interactions with the morpholinone ring in <b>28</b> are shown in stick representation. Hydrogen bond with Glu366 is marked. (c) Superposition of the crystal structures of menin in complex with <b>MI-503</b>, <b>28</b>, and MLL peptide encompassing MBM1 and MBM2 motifs (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U88">3U88</a>). (d) Comparison of the binding mode of <b>28</b> (left) and <b>29</b> (right, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BY8">6BY8</a>) to menin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Development of Compound <b>28</b> as a Highly Potent Menin–MLL Inhibitor</h3><div class="NLM_p">Exploring different saturated rings introduced at indole nitrogen led to the identification of <b>7</b>, which demonstrated the most potent inhibitory activity for blocking the menin–MLL interaction (IC<sub>50</sub> = 7.5 nM for the racemic mixture), <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. This compound harbors a morpholinone ring introduced to the indole nitrogen through a methyl linker, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. In addition to exhibiting the most potent <i>in vitro</i> activity, compound <b>7</b> also demonstrated the most pronounced cell growth inhibition in the MLL-AF9 leukemia cells (GI<sub>50</sub> = 50 nM), <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. To assess the activity of individual enantiomers of this compound we synthesized both of them, resulting in <b>28</b> (<i>S</i> enantiomer, <b>MI-1481</b>) and <b>29</b> (<i>R</i> enantiomer, <b>MI-1482</b>), <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. The <i>S</i> enantiomers (<b>28</b>) demonstrated a ∼2-fold improvement in inhibitory activity for blocking the menin–MLL interaction (IC<sub>50</sub> = 3.6 nM) over the racemic mixture (compound <b>7</b>), <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Thus, <b>28</b> demonstrates about 10-fold increase in inhibitory activity over <b>MI-463</b> and <b>MI-503</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), representing the most potent reversible menin–MLL inhibitor reported to date. In contrast, the <i>R</i> enantiomer (compound <b>29</b>) demonstrated a ∼35-fold reduction in inhibitory activity in blocking the menin–MLL interaction (IC<sub>50</sub> = 123 nM for <b>29</b>) as compared to <b>28</b>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Thus, <b>28</b> and <b>29</b> provide a valuable pair of enantiomers with distinct inhibitory activity to study menin–MLL inhibition in biological systems.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition of the menin–MLL interaction and cellular activity of compound <b>28</b>. (a) Titration curves from the FP competition assay for inhibition of the menin–MLL<sub>4-43</sub> by <b>28</b> and <b>29</b>. Data represent mean values of duplicate samples ± SD. (b–d) Titration curves from the MTT cell viability assay performed for <b>28</b> or <b>29</b> after 7 days of treatment of MLL-AF9 or Hoxa9/Meis1 (HM-2) transformed murine bone marrow cells treated with <b>28</b> (b); human MLL leukemia cell lines (MV4;11 and MOLM-13) and control cell lines (K562 and U937 without MLL translocation) treated with <b>28</b> (c); and human leukemia MLL cell lines treated with <b>29</b> (d). Cell growth inhibition (GI<sub>50</sub>) values are provided. Data represent mean values of quadruplicates ± SD. The experiments were performed two times. Values are normalized with respect to DMSO-treated samples. (e) qRT-PCR performed in MV4;11 cells after 6 days of treatment with <b>28</b>. Expression of <i>HOXA9</i>, <i>MEIS1</i>, <i>DLX2</i>, and <i>ITGAM</i> was normalized to 18S and referenced to DMSO-treated cells, mean ± SD, <i>n</i> = 2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR and Properties of Enantiomers of Compound <b>7</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0016.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0017.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were measured by fluorescence polarization assay using fluorescein-labeled MLL <sub>4-43</sub>. Average values from two or three independent measurements ± SD are provided.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Growth inhibition (GI<sub>50</sub> values) measured in the MTT cell viability assay after 7 days of treatment of murine bone marrow cells transformed with MLL-AF9.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Half-life of compounds in mouse liver microsomes.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Calculated with ChemBioDraw Ultra 14.0. SI, selectivity index calculated as a ratio of GI<sub>50</sub> values measured in HM-2 cells (control cell line) and MLL-AF9 transformed murine bone marrow cells; ND, not determined.</p></div></div><div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Molecular Basis of Menin–MLL Inhibition by Compound <b>28</b></h3><div class="NLM_p">To understand the molecular basis of the strong inhibition of the menin–MLL interaction by <b>28</b>, we co-crystallized menin in complex with this compound, resulting in a high-resolution crystal structure, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Table 1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b. We found a similar interaction pattern of <b>28</b> with menin to the one observed in the menin-<b>MI-503</b> complex for the structural fragment retained in both compounds (e.g., hydrogen bonds with Tyr276 and Trp341 and similar pattern of hydrophobic interactions), <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b. Again, the differences were found in the binding of substituents introduced at indole nitrogen. Specifically, the methylene group linking indole with the morpholinone ring in <b>28</b> is involved in the hydrophobic interactions with Glu363, Met322, and Val 367, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b. Furthermore, one side of the morpholinone ring in <b>28</b> is involved in hydrophobic contacts with the side chains of Glu363, Glu366, and Val367 on menin, while the nitrogen atom of the amide forms a short hydrogen bond (2.6 Å) with the carboxyl group of Glu366 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). This hydrogen bond, which was not observed for the piperazine analogues (e.g., in the menin–<b>13</b> complex), most likely contributes to the pronounced inhibitory activity of <b>28</b>. The carbonyl carbon and the adjacent methylene group in <b>28</b> are engaged in the hydrophobic contacts with Val371, while the ether oxygen is solvent exposed and does not form major contacts with menin, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b. In contrast, the pyrazole ring introduced at the indole nitrogen of <b>MI-503</b> is mostly solvent exposed, with one hydrogen bond formed with Glu366, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b.</div><div class="NLM_p last">To understand better the improvement in inhibitory activity for <b>28</b> over <b>MI-503</b>, we performed the biolayer interferometry (BLI) studies and compared binding affinities of both compounds to menin. This study demonstrated that the association rates to menin are similar for both compounds; however, the dissociation rate is significantly slower for <b>28</b> over <b>MI-503</b>, resulting in the <i>K</i><sub>d</sub> value ∼3-fold improved for <b>28</b> (<i>K</i><sub>d</sub> = 9 nM) as compared to <b>MI-503</b> (<i>K</i><sub>d</sub> = 24 nM), <a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Figure 2</a>. Furthermore, we have also performed structural analysis of menin interactions with ligands by overlapping the crystal structures of menin in complex with the bivalent MLL peptide encompassing both MBM1 and MBM2 motifs, <b>MI-503</b> and <b>28</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c. This analysis has revealed that both compounds occupy the same region of the binding site on menin where the MBM1 motif binds; however, <b>28</b> approaches more closely the region occupied by MBM2, particularly Arg24 from MLL, as compared to <b>MI-503</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c. Therefore, <b>28</b> might be able to more effectively displace the bivalent MLL peptide from its binding to menin, likely due to stronger interference with the MBM2 portion of MLL, as reflected by ∼10-fold improved inhibitory activity of <b>28</b> over <b>MI-503</b>. Overall, both structural data (e.g., extensive interactions with menin and interference with the MBM2 motif) and binding kinetics support <b>28</b> as a more effective inhibitor of the menin–MLL interaction over the previous generation of menin–MLL inhibitors (e.g., <b>MI-503</b> or <b>MI-463</b>), <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. To better understand the molecular basis of the differential inhibitory activity between both enantiomers of <b>7</b>, we also solved the co-crystal structure of menin in complex with <b>29</b> (<i>R</i> enantiomer of <b>7</b>), <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Table 1</a>. The core structure of <b>29</b> binds to menin in the same way as observed for <b>28</b>, but the orientation of the morpholinone ring in <b>29</b> is different from that of <b>28</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d, which results in significantly reduced contacts of this ring with menin. The most striking difference in the binding mode of <b>29</b> and <b>28</b> is the lack of a hydrogen bond between the amide nitrogen in <b>29</b> and the carboxyl group of Glu366 in menin, which is present when <b>28</b> binds to menin, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b,d. Compound <b>29</b> also has limited hydrophobic contacts with menin as compared to <b>28</b>, as <b>29</b> engages only a small portion of the morpholinone ring in the interactions with Glu366, Val367, and Val371 on menin, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d. Overall, we concluded that limited interactions of the morpholinone ring in <b>29</b> with menin, in particular the lack of a hydrogen bond with Glu366, result in a 35-fold reduction in inhibitory activity of this compound to menin as compared to <b>28</b>.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Compound <b>28</b> Demonstrates Potent On-Target Activity in MLL Leukemia Cells</h3><div class="NLM_p">To assess whether potent inhibition of the menin–MLL interaction by <b>28</b> results in strong activity of this compound in functional assays, we tested <b>28</b> in several leukemia cell lines. Importantly, <b>28</b> markedly reduced cell growth of murine bone marrow cells transformed with MLL-AF9, with a GI<sub>50</sub> value of 34 nM, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b. In contrast, limited activity of this compound was observed in murine bone marrow cells transformed with the control oncogenes Hoxa9/Meis1 (HM-2, GI<sub>50</sub> > 6 μM), <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b. Thus, <b>28</b> demonstrated >200-fold selectivity toward MLL fusion protein transformed cells, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, demonstrating substantial improvement in selectivity over the previous generation of menin–MLL inhibitors (e.g., selectivity index, SI ≈ 27 for <b>MI-503</b> in MLL leukemia cells).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> When tested in human MLL cell lines, <b>28</b> also showed pronounced cell growth inhibition, with GI<sub>50</sub> values of 36 nM in MV4;11 (harboring MLL–AF4 fusion protein) and 61 nM in MOLM13 (harboring MLL–AF9), <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c. Importantly, no substantial cell growth inhibition was observed in the control human leukemia cell lines K562 and U937, which do not harbor MLL translocations (GI<sub>50</sub> > 6 μM), <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c. Therefore, <b>28</b> shows the most pronounced activity in MLL leukemia cells among all reversible menin–MLL inhibitors reported to date, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>,<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> demonstrating ∼7–10-fold improvement in the activity in MLL leukemia cells over <b>MI-463</b> or <b>MI-503</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> As pointed out above, the increased cellular activity of <b>28</b> is associated with ∼10-fold improved inhibitory activity of this compound in blocking the menin–MLL interaction as compared to <b>MI-463</b> or <b>MI-503</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. In contrast, compound <b>29</b>, which represents the <i>R</i> enantiomer of <b>7</b>, showed limited activity in MLL leukemia cells (over 16-fold weaker than <b>28</b>), <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d, in agreement with the substantially reduced inhibitory activity of <b>29</b>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_p last">To validate the on-target activity of <b>28</b> in MLL leukemia cells, we assessed the effect of this compound on the expression level of the MLL fusion target genes: <i>HOXA9</i>, <i>MEIS1</i>, and <i>DLX2</i>, all of which are required for the MLL-fusion protein mediated leukemogenesis. Six days of treatment of the MV4;11 human MLL leukemia cells with <b>28</b> resulted in a substantial downregulation of the MLL fusion protein target genes, with a strong reduction in the expression level of <i>HOXA9</i> and <i>MEIS1</i> at 100 nM of <b>28</b>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e. The effect of <b>28</b> on the expression level of <i>DLX2</i> was even more pronounced, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e. Furthermore, marked upregulation of the <i>ITGAM</i> differentiation marker was observed upon treatment of MV4;11 cells with <b>28</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e), supporting reprogramming of these cells from primitive blasts to more mature blood cells. Overall, <b>28</b> demonstrates a pronounced effect on the expression level of MLL fusion target genes, which strongly supports the on target activity of this compound in MLL leukemia cells.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Compound <b>28</b> Strongly Inhibits Leukemia Progression <i>in Vivo</i></h3><div class="NLM_p">We then performed pharmacokinetic (PK) studies in mice with <b>28</b> and found relatively high exposure of this compound in blood plasma using i.v. (intravenous) and i.p. (intraperitoneal) administration (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Figure 3</a>), however, oral bioavailability of this compound was low (<5%). Based on these results we selected i.p. as a way of administration of <b>28</b> for <i>in vivo</i> efficacy studies. To establish a potential therapeutic value of <b>28</b>, we tested the effect of this compound in a mice model of MLL leukemia. For this purpose, we utilized the xenotransplantation model of the MLL leukemia with MV4;11-luc cells (expressing luciferase) injected into the tail vein of NSG mice. Treatment with <b>28</b> (80 mg/kg, i.p., <i>n</i> = 8) or vehicle (<i>n</i> = 8) was initiated 13 days after transplantation of mice with the MV4;11 cells to account for a compound effect in an advanced leukemia model, and was continued for 21 consecutive days. No reduction in the body weight of mice was observed during the entire treatment period with <b>28</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Figure 4</a>. To assess treatment efficacy, the bioluminescence level in mice was measured at days 0, 7, 14, and 21 after initiating the treatment. Very pronounced (∼75%) and consistent reduction in the luciferase signal was observed over the course of the experiment upon treatment of mice with <b>28</b> as compared to the vehicle treated mice, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a,b. These results demonstrate the pronounced effect <b>28</b> on leukemia progression in an advanced model of MLL leukemia.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>28</b> strongly inhibits leukemia progression in mice. (a,b) Bioluminescence level in MV4;11 leukemia mice measured at days 0, 7, 14, and 21 after initiating treatment with vehicle or <b>28</b>. <i>n</i> = 8 mice per group. (b) Representative images of MV4;11 mice treated with vehicle or <b>28</b> at day 21 of treatment. (c) Spleen weight and pictures of MV4;11 mice after 6 days of treatment with vehicle or <b>28</b> (<i>n</i> = 3 mice/group). (d) Level of hCD45+ cells in MV4;11 mice after 6 days of treatment with vehicle or <b>28</b> measured by flow cytometry (<i>n</i> = 3 mice/group). (e) Expression of <i>HOXA9</i>, <i>MEIS1</i>, and <i>ITGAM</i> in the bone marrow (BM) and spleen samples of the MV4;11 leukemia mice after 6 days of treatment with <b>28</b>. Expression was normalized to GAPDH and referenced to vehicle-treated mice, mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To validate the on-target <i>in vivo</i> activity of <b>28</b> we performed gene expression studies in bone marrow and spleen samples isolated from mice transplanted with MV4;11 cells and treated with <b>28</b> or vehicle. Treatment with <b>28</b> was initiated 21 days after transplantation of mice with the leukemic cells to account for the effect of <b>28</b> in advanced leukemia model. Compound <b>28</b> (or vehicle) was administered for six consecutive days, followed by collection of the bone marrow and spleen samples. Importantly, substantial reduction in the spleen weight and size was observed for mice treated with <b>28</b> compared to the vehicle control, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c. Furthermore, the level of leukemic blasts (human CD45+ cells) was very high (∼75%) in the bone marrow cells isolated from vehicle treated mice, which was indicative of advanced leukemia, while the level of leukemic blasts was substantially reduced (to ∼35%) in mice treated with <b>28</b>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d. We then performed gene expression studies in bone marrow and spleen samples isolated from mice treated with <b>28</b> or vehicle using qRT-PCR and found that treatment with <b>28</b> resulted in a strong reduction (2–5 fold) in the expression level of MLL fusion target genes <i>MEIS1</i> and <i>HOXA9</i>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>e. Furthermore, an over 4–6-fold increase in the expression of <i>ITGAM</i> differentiation marker was observed in the bone marrow and spleen samples upon treatment with <b>28</b>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>e. Overall, the results from gene expression studies demonstrate pronounced, on-target activity of <b>28</b> in the advanced mouse model of MLL leukemia, thus supporting the potential therapeutic value of this compound.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div class="NLM_p">Synthesis of compounds <b>3</b> and <b>4</b> was performed using the synthetic route shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Two major intermediates, 5-formyl-4-methyl-1<i>H</i>-indole-2-carbonitrile (<b>30</b>) and <i>N</i>-(piperidin-4-yl)-6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-amine (<b>33</b>) were obtained according to the procedure described before.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The alcohols <b>31a,b</b> were converted into mesylates followed by the substitution with the aldehyde <b>30</b> in the presence of cesium carbonate, which afforded aldehydes <b>32a,b</b>, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Compounds <b>32a,b</b> were then used in the reductive amination reaction with the secondary amine <b>33</b> utilizing sodium triacetoxyborohydride as the reducing agent to afford compounds <b>34a,b</b>. Removal of the <i>tert</i>-butoxycarbonyl (Boc) group in the presence of tin(IV) chloride resulted in compounds <b>3</b> and <b>4</b>. Compound <b>5</b> bearing the morpholinone ring was obtained using the strategy depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The 4-(2-chloroethyl)morpholine (<b>35</b>) was used for the indole alkylation of aldehyde <b>30</b>, resulting in compound <b>36</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The reductive amination of aldehyde <b>36</b> with the amine <b>33</b> resulted in compound <b>5</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MsCl, TEA, DCM −67 °C to rt; (b) <b>30</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 90–110 °C; (c) <b>33</b>, Na(AcO)<sub>3</sub>BH; TEA, rt; (d) SnCl<sub>4</sub>, tartaric acid.</p></p></figure><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>30</b>, Cs<sub>2</sub>CO<sub>3</sub>, TBAI, DMF, 90 °C; (b) <b>33</b>, Na(AcO)<sub>3</sub>BH; TEA, rt.</p></p></figure><div class="NLM_p">The racemic mixtures of compounds <b>6</b> and <b>7</b> were prepared according to the synthetic route presented in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The intermediates <b>39a</b> and <b>39b</b> were obtained using the methods described before.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Briefly, the commercially available 3-amino-1,2-propanediol (<b>37a</b>) and methyl 4-amino-3-hydroxybutanoate (<b>37b</b>) were subjected to acylation with the chloroacetic acid, resulting in compounds <b>38a</b> and <b>38b</b>, respectively. Direct cyclization of <b>38a</b> with potassium <i>tert-</i>butoxide afforded alcohol <b>39a</b>. Homologous alcohol <b>39b</b> was achieved by reduction of the ester group in <b>38b</b> followed by the analogous cyclization with potassium <i>tert-</i>butoxide as a base (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The alcohols <b>39a,b</b> were then converted to the corresponding mesylates and used for alkylation of the indole <b>30</b> to afford aldehydes <b>40a</b> and <b>40b</b>. Reductive amination reactions of the intermediate <b>33</b> with aldehydes <b>40b</b> or <b>40a</b> led to the target compounds <b>6</b> and <b>7</b>, respectively (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>6</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) chloroacetic acid chloride, MeOH, AcCN, TEA, 0 °C; (b) <i>t-</i>BuOK, amyl alcohol, rt (for <b>38a</b>); (c) LiBH<sub>4</sub>, THF, 0 °C and (d) <i>t</i>-BuOK, amyl alcohol, 0 °C (for <b>38b</b>); (e) MsCl, TEA, DCM, −67 °C to rt; (f) <b>30</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMA, 105 °C; (g) <b>33</b>, Na(AcO)<sub>3</sub>BH; AcOH, 50 °C.</p></p></figure><div class="NLM_p">The synthetic route for compounds <b>8</b> and <b>9</b> harboring the piperazine ring in the structure is shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Briefly, both compounds were prepared from <b>3</b> using either formaline (for <b>8</b>) or propionaldehyde (for <b>9</b>) in the reductive amination reaction, <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The piperazine and piperidine analogues <b>10</b>–<b>27</b> were obtained according to the synthetic strategy depicted in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. In general, the amide series (compounds <b>10</b>–<b>17</b>, <b>24</b>, and <b>25</b>) was obtained by acylation of the amines <b>3</b> or <b>4</b>. The formamides <b>10</b> and <b>24</b> were prepared by formylation of <b>3</b> or <b>4</b>, respectively, with the methyl formate. The corresponding acyl chlorides were used to convert the amine <b>3</b> into compounds <b>11</b>, <b>12</b>, or <b>17</b> with DIPEA serving as a base, <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The analogous reaction was performed with the amine <b>4</b> to afford compound <b>25</b>. Synthesis of the amides <b>13</b>,<b>14</b>, and <b>16</b> was performed by utilizing the coupling reaction between the amine <b>3</b> and the corresponding carboxylic acids in the presence of 2-(1<i>H</i>-benzotriazol-1-yl)-1,1,3,3-tetramethylaminium tetrafluoroborate (TBTU) serving as a condensing reagent, <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Compound <b>15</b> with the trifluoroacetyl moiety was prepared using the triflate anhydride as an acylating agent. Alkylation of the nitrogen in the piperazine ring of <b>3</b> by the 2-bromopropionamide led to the amide <b>18</b>, <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>.</div><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>8</b> and <b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) formaline, Na(AcO)<sub>3</sub>BH, AcCN, rt (for <b>8</b>); (b) <i>n-</i>propanal, Na(AcO)<sub>3</sub>BH; TEA, DCM, rt (for <b>9</b>).</p></p></figure><figure id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0009.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>10</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HCOOMe, reflux (for <b>10</b>, <b>24</b>); (b) R<sup>2</sup>COCl, DIPEA, DCM or AcCN, rt (for <b>11</b>, <b>12</b>, <b>17</b>, <b>25</b>); (c) R<sup>2</sup>COOH, TBTU, THF (for <b>13</b>, <b>14</b>, <b>16</b>); (d) (CF<sub>3</sub>CO)<sub>2</sub>O, DCM, 0 °C to rt (for <b>15</b>); (e) <i>rac-</i>BrCH(CH<sub>3</sub>)CONH<sub>2</sub>, DMF (for <b>18</b>); (f) R<sup>1</sup>SO<sub>2</sub>Cl, DIPEA, DCM, rt (for <b>19–21</b>, <b>23</b>, <b>26</b>); (g) CbzNHSO<sub>2</sub>Cl, DIPEA, DCM; and (h) Pd/C, MeOH, ammonium formate (for <b>22</b>, <b>27</b>).</p></p></figure><div class="NLM_p">The sulfonamide series (compounds <b>19</b>–<b>21</b>, <b>26</b>) was obtained using the sulfonylation reactions with compounds <b>3</b> or <b>4</b> and the appropriate sulfonyl chlorides, <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The two-step synthetic approach was applied to afford compounds <b>22</b> and <b>27</b>. The benzyl (chlorosulfonyl)carbamate was used for sulfonylation of the amines <b>3</b> or <b>4</b>, followed by the cleavage of the Cbz-protected amino group performed with Pd/C and ammonium formate, which led to sulfamides <b>22</b> or <b>27</b>, <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. Conversion of the amine <b>3</b> to compound <b>23</b> was achieved using the methylsulfamoyl chloride in the presence of DIPEA.</div><div class="NLM_p last">Synthesis of compounds <b>28</b> and <b>29</b>, which represent single enantiomers of <b>7</b>, was performed by applying the same methodology as for compound <b>7</b>, <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, through utilizing the nonracemic 3-aminopropane-1,2-diols: <b>37a</b>(<i>S</i>) or <b>37a</b>(<i>R</i>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88432" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88432" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Identification of potent inhibitors targeting protein–protein interactions and optimization of their drug-like properties represents a challenge as such compounds frequently have large molecular weight, which is required for effective blocking of PPI interfaces.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> This is particularly true for compounds that target extended protein–protein interfaces, including inhibitors of the BCL-2 family of proteins<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and others.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> On the other hand, recent FDA approval of the BCL-2 inhibitor Venetoclax demonstrates that PPI inhibitors with large molecular weight (>800 Da) can serve as valuable therapeutics. In the current study, we report optimization of the small-molecule inhibitors of the menin–MLL interaction, with the major goal of improving the potency and drug-like properties of these compounds. The MLL binding site on menin is large (over 5000 Å<sup>3</sup><a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a>) as menin binds long, bivalent MLL peptide (MLL<sub>4-43</sub>). This suggests that compounds with large molecular weight may be required for effective inhibition of the menin–MLL interaction.</div><div class="NLM_p">Using the crystal structure of menin in complex with <b>MI-503</b>, we designed, synthesized, and evaluated a series of new analogues by exploring various saturated rings that substitute indole nitrogen in <b>MI-503</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>-<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. The goal of this work was to increase the inhibitory activity and metabolic stability of menin–MLL inhibitors by introducing substituents with different physicochemical properties (e.g., charged versus uncharged substituents, hydrogen bond donor or acceptor groups, etc.). These efforts resulted in compound <b>28</b>, which demonstrates very potent inhibition of the menin–MLL interaction (IC<sub>50</sub> = 3.6 nM with MLL<sub>4-43</sub>). Therefore, <b>28</b> represents the most potent reversible small molecule inhibitor of the menin–MLL interaction reported to date.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> About 10-fold improved inhibitory activity for <b>28</b> over previous generation of reversible menin–MLL inhibitors (e.g., <b>MI-503</b>, <b>MI-463</b>) results from stronger binding affinity of this compound to menin, but also likely from more pronounced overlap with MBM2 binding to menin, as supported by the structural data, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c. Furthermore, <b>28</b> demonstrates stronger effect (>8-fold) and better selectivity toward MLL leukemia cells than <b>MI-503</b>, and it also has a more pronounced effect on the target gene expression in mouse models of MLL leukemia, Figure 5E.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> On the other hand, increased polarity of <b>28</b> (clogP value reduced) and increased molecular weight (by ∼30 Da) over <b>MI-503</b> (<i>M</i><sub>w</sub> = 597 Da for <b>28</b>) negatively affected oral bioavailability of this compound. These results demonstrate the complexity of blocking bivalent protein–protein interactions with large molecular interfaces, where achievement of potent inhibition requires effective interference with both motifs occupying different regions of the binding site. The increase in the molecular weight of inhibitors targeting bivalent PPIs (e.g., >500 Da) can negatively affect their drug-like properties, making the lead optimization process extensive and complex.</div><div class="NLM_p">Importantly, strong <i>in vitro</i> inhibition induced by <b>28</b> correlates well with pronounced activity of this compound in MLL leukemia cells (GI<sub>50</sub> = 30–60 nM), which represents almost 10-fold improvement over <b>MI-503</b> or any other reversible menin–MLL inhibitor.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> In addition, compound <b>28</b> demonstrates high selectivity (>100-fold) and specific mechanism of action in MLL leukemia cells. Furthermore, the pronounced effect of <b>28</b> in inhibiting leukemia progression in the disseminated model of MLL leukemia combined with the on-target mechanism of action <i>in vivo</i> make this compound a very attractive candidate compound for further studies. During the preparation of this Article, irreversible menin–MLL inhibitors with distinct molecular scaffold were reported,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> but these compounds did not show stronger <i>in vitro</i> inhibitory activity over <b>28</b> despite covalent binding to menin. Furthermore, biological activity of these irreversible menin–MLL inhibitors, especially in the <i>in vivo</i> models of MLL leukemia, and their potential off-target effects remain to be assessed.</div><div class="NLM_p last">Overall, compound <b>28</b> identified in this study represents a valuable molecule to further investigate the role of menin–MLL interaction in normal and pathological states. This compound may also lead to the development of new therapeutic agents for MLL leukemia and possibly solid tumors, including prostate cancer<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Ewing sarcoma,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> HCC,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and other tumors that may rely on the intact menin–MLL interaction. Our work demonstrates that, despite challenges associated with the development of potent PPI inhibitors targeting bivalent protein–protein interfaces, successful identification of such compounds is possible and can lead to valuable chemical tools for the scientific community and potentially to new therapeutic agents.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73165" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73165" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Expression and Purification of Menin</h3><div class="NLM_p last">The expression and purification of menin have been described previously.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> In order to generate biotinylated menin, we introduced Avi-tag sequence (AAALEGLNDIFEAQKIEWHE)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> to the C-terminus of menin. Protein was co-expressed in <i>E. coli</i> BL21 (DE3) cells with the construct encoding BirA enzyme<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and purified using the same protocol as for the intact menin.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Biochemical Characterization of Menin–MLL Inhibitors</h3><div class="NLM_p last">Inhibition of the menin–MLL interaction by small molecules was assessed by fluorescence polarization (FP) assay. The fluorescein-labeled MLL<sub>4-43</sub> peptide (FLSN-MLL<sub>4-43</sub>) at 4 nM, menin at 4 nM and varying concentrations of compounds were used for IC<sub>50</sub> determination in the FP buffer (50 mM Tris, pH 7.5, 50 mM NaCl, 1 mM TCEP, 0.01% BSA). The complex of menin with the FLSN-MLL<sub>4-43</sub> peptide was incubated for 1h. Compounds (5% final DMSO concentration) were then added to the menin-FLSN_MLL<sub>4-43</sub> complex and incubated for 3 h before changes in fluorescence polarization (mP values) were measured using the PHERAstar microplate reader (BMG). The IC<sub>50</sub> values were calculated in Origin 7.0 (OriginLab) by plotting the mP values measured for each compound as a function of compound concentration.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Bio-Layer Interferometry (BLI) Binding Assay</h3><div class="NLM_p last">BLI assay was performed in 96-well microplates at room temperature with continues shaking at 1000 rpm using the Octet Red 96 system (ForteBio, Menlo Park, CA, USA). Biotinylated menin was bound on Super Streptavidin (SSA) biosensors (ForteBio) by dipping sensors into 200 nM biotinylated menin in the assay buffer composed of 50 mM Tris, pH 7.5, 50 mM NaCl, 1 mM TCEP, 0.05% BSA, 0.01% Tween-20 and 2% DMSO. Menin binding was performed for 600 s followed by equilibration in the assay buffer for 1200 s. Compounds (<b>28</b> and <b>MI-503</b>) were used at 150 and 200 nM concentrations and binding was performed for 600 s followed by 1200 s dissociation. Data were fit globally and generated automatically by Octet User Software (ForteBio, Inc.). Analysis of binding kinetics was performed in Prism (GraphPad Software, Inc.). <i>K</i><sub>obs</sub> values were extracted using one phase association equation, and <i>K</i><sub>off</sub> values were extracted using one phase decay equation. Subsequently, <i>K</i><sub>on</sub> was derived using <i>K</i><sub>obs</sub><i>= K</i><sub>on</sub>[Ligand] + <i>K</i><sub>off</sub>, where [Ligand] is free ligand concentration in solution. <i>K</i><sub>D</sub> was calculated as <i>K</i><sub>off</sub>/<i>K</i><sub>on</sub>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Crystallization of Menin Complexes with Small-Molecule Inhibitors</h3><div class="NLM_p last">For co-crystallization experiments 2.5 mg/mL menin was incubated with small molecule inhibitors at 1:3 molar ratio. Crystals were obtained using the sitting drop technique at 10 °C by applying the procedure described previously.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Prior to data collection, crystals were transferred into a cryo-solution containing 20% PEG550 MME and flash-frozen in liquid nitrogen.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Crystallographic Data Collection and Structure Determination</h3><div class="NLM_p last">Diffraction data for menin and menin complexes were collected at the 21-ID-D and 21-ID-F beamlines at the Life Sciences Collaborative Access Team at the Advanced Photon Source. Data were processed with HKL-2000.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Structures of the complexes were determined by molecular replacement using MOLREP with the apo-structure of human menin (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GPQ">4GPQ</a>) as a search model in molecular replacement. The model was refined using REFMAC,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> COOT,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> CCP4 package,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and PHENIX.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>Validation of the structures was performed using MOLPROBITY<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and ADIT.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Details of data processing and refinement are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">Supplementary Table 1</a>. Coordinates and structure factors for menin–inhibitor complexes have been deposited in the Protein Data Bank under the following codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXH">6BXH</a> (menin–compound <b>13</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXY">6BXY</a> (menin–compound <b>28</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BY8">6BY8</a> (menin–compound <b>29</b>).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Viability Assays</h3><div class="NLM_p last">The MLL-AF9, HM-2, and E2A-HLF transformed murine bone marrow cells (BMCs) were prepared as described previously.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> MV4;11, MOLM-13, K562, and U937 cells were cultured in RPMI-1640 medium with 10% FBS, 1% penicillin/streptomycin, and NEAA (nonessential amino acid) solution. For viability assay, MOLM-13 (1 × 10<sup>5</sup>/mL), MV4;11 (1 × 10<sup>5</sup>/mL), K562 (1 × 10<sup>5</sup>/mL), and U937 (1 × 10<sup>5</sup>/mL) human leukemia cells as well as MLL-AF9, HM-2, and E2A-HLF murine bone marrow cells (2.5 × 10<sup>4</sup>/mL) were plated (1 mL/well), treated with compounds or 0.25% DMSO and cultured at 37 °C for 7 days. Media were changed at day four with viable cell number restored to the original concentration and compounds were resupplied. 100 μL of cell suspension were transferred to 96-well plates for each sample in quadruplicates. A Vybrant MTT cell proliferation assay kit (Molecular Probes) was employed. Plates were read for absorbance at 570 nm using a PHERAstar BMG microplate reader. The experiments were performed 2–3 times in quadruplicates with calculation of mean and standard deviation for each condition.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Real-Time PCR</h3><div class="NLM_p last">Effect of <b>28</b> on the expression level of MLL fusion target genes was assessed by real-time quantitative PCR (qRT-PCR) using the protocol described previously.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Microsomal Stability Studies</h3><div class="NLM_p last">The metabolic stability was assessed using CD-1 mouse liver microsomes. One μM each of compounds was incubated with 1.3 mg/mL microsomes and 1.7 mM cofactor NADPH in 0.1 M phosphate buffer (pH = 7.4) containing 3.3 mM MgCl<sub>2</sub> at 37 °C. The DMSO concentration was less than 0.1% in the final incubation system. At 0, 3, 5, 10, 15, 30, 45, and 60 min of incubation, the reactions were stopped by adding 3-fold excess of acetonitrile containing 100 ng/mL of internal standard for quantification. The collected fractions were centrifuged at 3000 <i>g</i> for 20 min to collect the supernatant for LC-MS/MS analysis, from which the amount of compound remaining was determined. The natural log of the amount of compound remaining was plotted against time to determine the disappearance rate and the half-life of tested compounds.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Pharmacokinetic Studies in Mice</h3><div class="NLM_p">All animal experiments in this study were approved by the University of Michigan Committee on Use and Care of Animals and Unit for Laboratory Animal Medicine (ULAM).</div><div class="NLM_p last">The pharmacokinetics of <b>28</b> was determined in female C57BL/6 mice following intravenous (iv) dosing at 15 mg/kg and 30 mg/kg for oral (p.o.) and intraperitoneal (i.p.) dosing. For i.v. and p.o. administration, <b>28</b> was dissolved in the vehicle containing 25% (v/v) DMSO, 25% (v/v) PEG400, and 50% (v/v) PBS. For i.p. administration, <b>28</b> was dissolved in the vehicle containing 20% HP-β-cyclodextrin and 5% Cremophor. Serial blood samples were collected over 7 h, centrifuged at 15 000 rpm for 10 min, and saved for analysis. Plasma concentrations of <b>28</b> were determined by the LC-MS/MS method developed and validated for this study. The LC-MS/MS method consisted of an Agilent 1200 HPLC system, and chromatographic separation of <b>28</b> was achieved using an Agilent Zorbax Extend-C18 column (5 cm × 2.1 mm, 3.5 μm; Waters). An AB Sciex QTrap 3200 mass spectrometer equipped with an electrospray ionization source (ABI-Sciex, Toronto, Canada) in the positive-ion multiple reaction monitoring (MRM) mode was used for detection. Pharmacokinetic parameters were calculated by noncompartmental methods using WinNonlin version 3.2 (Pharsight Corporation, Mountain View, CA, USA).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>In Vivo</i> Efficacy Studies</h3><div class="NLM_p">All animal experiments in this study were approved by the University of Michigan Committee on Use and Care of Animals and Unit for Laboratory Animal Medicine (ULAM).</div><div class="NLM_p">For in vivo efficacy studies, 6- to 8-week-old female NSG mice (Jackson laboratory) were intravenously injected with 1 × 10<sup>7</sup> luciferase-expressing MV4;11 cells. At day 13 post-transplantation, mice were split into vehicle and treatment groups of 8 mice per group and were treated for 21 consecutive days with once daily i.p. injections of vehicle (20% HP-β-cyclodextrin and 5% Cremophor) or 80 mg/kg of <b>28</b> dissolved in vehicle. Imaging of mice was performed at days 0, 7, 14, and 21 of treatment with <b>28</b>. For imaging, mice were kept anesthetized with Isoflurane, and luciferin at 50 mg/kg was administered by intraperitoneal injection. Photonic emission was imaged using the BLI in vivo imaging system (IVIS, 200) with total imaging time of 10 s for each mouse. Survival of mice was monitored to assess treatment efficacy.</div><div class="NLM_p last">For the <i>in vivo</i> gene expression studies, 21 days after transplantation of mice with MV4;11 cells mice were grouped into vehicle or compound <b>28</b> treatment groups (<i>n</i> = 3/group), and the treatment was initiated and continued for 6 consecutive days (80 mg/kg, i.p., once daily). The next day after completing the treatment mice were sacrificed and bone marrow samples were isolated for gene expression studies by qRT-PCR.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Flow Cytometric Analysis of Mice Samples</h3><div class="NLM_p last">To assess levels of leukemia burden in the mice transplanted with leukemic cells, cells from bone marrow, peripheral blood, and spleen were buffered with PBS and 1% FBS for flow cytometry analysis. Red blood cells were lysed with ACK (Lonza). All flow cytometry experiments were performed on a LSR II FACSCanto or FACSAria (BD Biosciences), and all data were analyzed with FlowJo software (Tree Star, Inc.).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> General Chemistry</h3><div class="NLM_p">All commercially available solvents and reagents were used without further preparation unless otherwise indicated. The <sup>1</sup>H and <sup>13</sup>C NMR data were taken on Bruker Avance III 600 MHz or Varian MR400. Chemical shifts are reported in ppm relative to tetramethylsilane or residual solvent signal. Some signals that overlapped with solvents were identified using <sup>1</sup>H–<sup>13</sup>C HSQC spectrum (multiplicity not known). The mass measurements were determined on a Micromass LCT time-of-flight mass spectrometer using positive mode and electrospray ionization. The exact mass measurements were determined on Agilent Q-TOF time-of-flight mass spectrometer using positive ion mode and electrospray ionization. Analytical TLC was performed on Merck TLC aluminum plates precoated with F<sub>254</sub> silica gel 60 (UV, 254 nm, and iodine). The purity analysis of final compounds was determined on Shimadzu Prominence HPLC system (20 series: binary pump, UV/vis at 254 nm, heated column compartment 28 °C), using Restek Ultra C18 (5 μm) 150 mm × 4.6 mm column. LR-MS was recorder on Shimadzu LC-2020 system (DUIS-ESI). The solvents were programmed to run at gradient starting 20% CH<sub>3</sub>CN in water to 80% during 8 min run. If not indicated, the purity of all final compounds was >95% as determined by HPLC via integration of UV spectra at 254 nm. Optical rotation was measured using Jasco P2000 polarimeter (path length = 100 mm). All final (unless indicated otherwise) compounds <b>1</b>–<b>29</b> were tested as hydrochloride salts. The general procedure for HCl salt formation: free base was dissolved in minimum volume of methanol followed by addition of 1 equiv of 1 M HCl in water and thorough drying. During NMR experiment, most of the hydrochloride salts of these compounds exist as a mixture of two forms in solution of either DMSO or MeOH with the approximate ratio of 10:1; NMR is reported for the major rotamer or a free base as indicated.</div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-Methyl-1-(2-(piperazin-1-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>3</b>)</h4><div class="NLM_p last">To a stirred solution of <i>tert</i>-butyl 4-(2-hydroxyethyl)piperazine-1-carboxylate (<b>31a</b>, 1 g 4.3 mmol) and triethylamine (1.5 equiv, 898 μL, 6.45 mmol) in DCM (14 mL) was added methanosulfonyl chloride (1.2 equiv, 399 μL, 5.16 mmol) dropwise at 0 °C. The mixture was allowed to warm to room temperature and stirred for 3 h. The organic phase was diluted with DCM (40 mL), washed with water and brine, and dried over sodium sulfate. Solvent was evaporated (partial decomposition was observed when heated over 35 °C during evaporation), and crude product (1.25 g) was used in the next step. The 5-formyl-4-methyl-1<i>H</i>-indole-2-carbonitrile (<b>30</b>, 300 mg, 1.63 mmol) was suspended in DMF (3.2 mL), and cesium carbonate was added (1.63 g, 4.68 mmol, 3 equiv). The <i>tert</i>-butyl 4-(2-((methylsulfonyl)oxy)ethyl)piperazine-1-carboxylate, from the previous step (753 mg, 2.45 mmol, 1.5 equiv) was then added, and the resulting suspension was stirred at room temperature for 22 h. Reaction was quenched with water and extracted with DCM (3 × 40 mL). Organic phase was evaporated with silica gel and separated using flash chromatography (silica gel, hexanes:ethyl acetate 3:1 to 1:1, v:v). 316 mg of compound <i>tert</i>-butyl 4-(2-(2-cyano-5-formyl-4-methyl-1<i>H</i>-indol-1-yl)ethyl)piperazine-1-carboxylate (<b>32b</b>) was isolated (48% yield). MS (<i>m</i>/<i>z</i>) found for [M+H<sup>+</sup>]: 397 Da. Compound <b>32b</b> (880 mg, 2.79 mmol, 1 equiv) and amine <b>33</b> (1.01 g, 3.06 mmol, 1.1 equiv) were then dissolved in DCM (28 mL). Triethylamine was added (1.6 mL, 4 equiv) followed by sodium triacetoxyborohydride (1.2 g, 5.6 mmol, 2 equiv). Reaction was stirred overnight at room temperature. Mixture was diluted with DCM (50 mL) and washed with 1 M aqueous NaOH. Organic phase was dried over sodium sulfate and evaporated with silica gel. Purified using flash chromatography (silica gel, dichloromethane:methanol with 5% ammonia, 0% to 5%, v:v). 1.2 g of product <b>34b</b> (<i>tert</i>-butyl 4-(2-(2-cyano-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indol-1-yl)ethyl)piperazine-1-carboxylate) was isolated (62% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>4</sub>/methanol-<i>d</i><sub>4</sub>) δ 8.29 (s, 1H), 7.31–7.39 (m, 2H), 7.17–7.24 (m, 2H), 4.34 (t, <i>J</i> =  6.2 Hz, 2H), 4.10–4.19 (m, 1H), 3.65 (q, <i>J</i> = 10.27 Hz, 2H), 3.61–3.70 (m, 2H), 3.33–3.39 (m, 4H), 2.91–3.10 (m, 2H), 2.72 (t, <i>J</i> = 6.42 Hz, 2H), 2.54 (s, 3H), 2.38–2.46 (m, 4H), 2.19–2.38 (m, 2H), 1.95–2.06 (m, 2H), 1.55–1.72 (m, 2H), 1.40 (s, 9H); MS (<i>m</i>/<i>z</i>) found for [M+H<sup>+</sup>]: 697 Da. Compound <b>34b</b> (180 mg, 0.26 mmol) was then dissolved in acetonitrile (2.6 mL) and tin(IV) chloride (677 mg, 304 μL, 2.6 mmol, 10 equiv) was added at 0 °C. The reaction was stirred for 1h at room temperature and then cooled again to 0 °C. Ammonia solution (30%<sub>aq</sub>) was added slowly and DCM was evaporated. Water was added (2 mL) followed by tartaric acid (750 mg). The mixture was sonicated until the solution became clear. The product was extracted with DCM:MeOH (10:1). Purified on preparative TLC (silica gel, dichloromethane:methanol with 5% ammonia, 5%, v:v). 88 mg of compound <b>3</b> was isolated (57%). <sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>) δ 8.47 (s, 1H), 7.35 (d, <i>J</i> = 8.44 Hz, 1H), 7.18 (s, 1H), 7.14–7.17 (m, 1H), 7.10 (s, 1H), 5.21 (d, <i>J</i> = 7.70 Hz, 1H), 4.33 (t, <i>J</i> = 6.60 Hz, 2H), 4.14–4.29 (m, 1H), 3.57–3.68 (m, 4H), 2.87–2.97 (m, 6H), 2.66–2.74 (m, 2H), 2.54 (s, 3H), 2.46–2.53 (m, 4H), 2.22–2.30 (m, 2H), 2.04–2.12 (m, 2H), 1.53–1.65 (m, 2H); <sup>13</sup>C NMR (151 MHz, CHLOROFORM-d) δ 166.8, 156.1, 154.3, 136.4, 131.2, 129.0, 128.6, 128.0, 127.3, 124.7 (<i>J</i> = 277 Hz) 118.6, 116.4, 114.0, 111.6, 109.8, 107.2, 60.1, 57.8, 53.9, 52.3, 51.3, 48.0, 45.5, 43.3, 35.7 (<i>J</i> = 32 Hz), 32.4, 15.1; HRMS (ESI): [M+H<sup>+</sup>] calculated 597.2736; found 597.2731.</div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-Methyl-1-(2-morpholinoethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>5</b>)</h4><div class="NLM_p last">Compound <b>30</b> (84 mg, 0.46 mmol, 1 equiv) and 4-(2-chloroethyl)morpholine (<b>35</b>) (126 mg, 0.68 mmol, 1.5 equiv) were dissolved in DMF (0.92 mL). TBAI (85 mg, 0.23 mmol, 0.5 equiv) was added followed by cesium carbonate (905 mg, 1.76 mmol, 6 equiv). The suspension was heated at 60 °C for 1h. DMF was evaporated with silica gel and product was isolated using flash chromatography (silica gel, hexane:ethyl acetate, 1:1, v:v) to afford 63 mg of compound <b>36</b> (47% yield). Compound <b>36</b> (22 mg, 0.07 mmol) was then suspended in DCM (0.7 mL) and compound <b>33</b> (32 mg, 0.09 mmol, 1.2eq) and triethylamine (20 μL, 0.11 mmol, 1.4 equiv) were added. The suspension was stirred for 15 min and then sodium triacetoxyborohydride (23 mg, 0.11 mmol, 1.5 equiv) was added. The reaction was stirred overnight. DCM was added to dilute the solution and organic phase was washed with 1 M aqueous NaOH and brine. DCM was evaporated and the product was recrystallized from MeOH affording 30 mg of compound <b>5</b> (67% yield). <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.35 (s, 1H), 7.56 (m, 3H), 7.45 (s, 1H), 4.54 (m, 2H), 4.49 (m, 3H), 3.87 (q, <i>J</i> = 11 Hz, 2H), 3.65 (m, 6H), 2.79 (m, 2H), 2.70 (s, 3H), 2.53 (m, 4H), 2.35 (m, 2H), 1.99 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 167.1, 157.7, 154.9, 138.8, 135.1, 130.6, 130.0, 128.8, 126.6 (q, <i>J</i> = 276 Hz), 121.9, 121.2, 118.3, 114.6, 113.4, 112.6, 110.6, 67.9, 59.0, 58.8, 55.0, 53.1, 47.2, 44.2, 35.9, 35.7 (q, <i>J</i> = 33 Hz), 30.1, 15.9. HRMS (ESI): [M+H<sup>+</sup>] calculated 598.2570; found 598.2573.</div></div><div id="sec4_12_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-Methyl-1-(2-(4-propylpiperazin-1-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>9</b>)</h4><div class="NLM_p last">Compound <b>3</b> (20 mg, 0.03 mmol, 1 equiv) and propionaldehyde (3 μL, 0.034 mmol, 1.2 equiv) were dissolved in DCM (0.3 mL). After stirring for 15 min, sodium triacetoxyborohydride (13 mg, 0.06 mmol, 2 equiv) was added. The mixture was directly evaporated with silica gel and compound <b>9</b> (10 mg, 56% yield) was purified using flash chromatography (silica gel, dichloromethane:methanol with 5% aqueous ammonia, 5%, v:v). <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.36 (s, 1H), 7.63 (m, 1H), 7.56 (m, 2H), 7.56 (m, 2H), 7.45 (s, 1H), 4.55 (m, 2H), 4.49 (m, 2H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.62 (m, 2H), 3.49 (m, 2H), 3.06 (m, 5H), 2.84 (m, 2H), 2.70 (s, 3H), 2.55 (m, 2H), 2.34 (m, 2H), 2.04 (m, 2H), 1.76 (m, 2H), 1.00 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 176.4, 166.6, 157.7, 154.7, 154.7, 138.7, 135.1, 130.8, 130.1, 128.7, 126.8 (q, <i>J</i> = 276 Hz), 121.8, 121.1, 118.2, 114.8, 113.3, 112.6, 110.4, 59.4, 58.8, 57.7, 53.3, 52.9, 51.3, 47.1, 44.4, 35.6 (q, <i>J</i> = 33 Hz), 29.9, 22.1, 18.5, 15.8, 11.1. HRMS (ESI): [M+H<sup>+</sup>] calculated 639.3200; found 639.3198.</div></div><div id="sec4_12_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 1-(2-(4-Formylpiperazin-1-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>10</b>)</h4><div class="NLM_p last">Compound <b>3</b> (750 mg, 1.25 mmol) was dissolved in methyl formate (10 mL) and stirred overnight. Solvent was evaporated with silica gel and compound <b>10</b> (700 mg, 89% yield) was isolated using flash chromatography (silica gel, dichloromethane:methanol with 5% aqueous ammonia, 5%, v:v). <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.34 (s, 1H), 7.98 (m, 1H), 7.61 (m, 1H), 7.55 (m, 2H), 7.45 (s, 1H), 4.55 (m, 2H), 4.49 (m, 3H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.64 (m, 2H), 3.48 (m, 2H), 3.43 (m, 2H), 3.35 (m, 2H), 2.85 (m, 2H), 2.70 (s, 3H), 2.58 (m, 2H), 2.50 (m, 2H), 2.34 (m, 2H), 2.03 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 167.0, 163.2, 157.8, 154.9, 138.9, 135.2, 130.8, 130.1, 128.9, 126.8 (q, <i>J</i> = 276 Hz), 121.9, 121.2, 118.3, 114.7, 113.4, 112.6, 110.6, 59.0, 58.2, 54.9, 53.8, 53.0, 47.2, 46.7, 44.4, 40.9, 35.7 (q, <i>J</i> = 33 Hz), 30.1, 16.0. HRMS (ESI): [M+H<sup>+</sup>] calculated 625.2679; found 625.2682.</div></div><div id="sec4_12_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> General Procedure for Synthesis of Compounds <b>11</b> and <b>12</b></h4><div class="NLM_p last">Compound <b>3</b> (20 mg, 0.03 mmol) was dissolved in DCM or AcCN (0.2 mL), and DIPEA (12 μL, 0.05 mmol) was added. Mixture was cooled to 0 °C and corresponding acid chloride was added. Mixture was allowed to warm to room temperature and was stirred until no starting material was detected by TLC (30 min).</div></div><div id="sec4_12_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-(2-(4-Acetylpiperazin-1-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>11</b>)</h4><div class="NLM_p last">Reaction carried out in AcCN with acetyl chloride. Isolated 16 mg (86% yield). <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.38 (s, 1H), 7.69 (m, 2H), 7.58 (s, 1H), 7.53 (s, 1H), 4.56 (m, 2H), 4.49 (m, 1H), 3.87 (q, <i>J</i> = 11 Hz, 2H), 3.80 (m, 2H), 3.63 (m, 2H), 3.35 (m, 2H), 3.19 (m, 4H), 2.71 (s, 3H), 2.34 (m, 2H), 2.13 (s, 3H), 2.04 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): 176.5, 171.9, 165.9, 157.7, 154.4, 139.0, 135.5, 131.3, 130.5, 128.9, 126.8 (q, <i>J</i> = 276 Hz), 122.0, 121.7, 118.3, 114.6, 114.2, 111.9, 110.6, 58.9, 56.5, 53.8, 53.6, 53.0, 47.4, 45.5, 42.0, 40.7, 35.7 (q, <i>J</i> = 33 Hz), 30.0, 27.4, 22.2, 21.1, 16.0. HRMS (ESI): [M+H<sup>+</sup>] calculated 639.2836; found 639.2841.</div></div><div id="sec4_12_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-Methyl-1-(2-(4-propionylpiperazin-1-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>12</b>)</h4><div class="NLM_p last">The reaction was carried out in DCM with propionic acid chloride. Isolated 19 mg (99% yield). <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.35 (s, 1H), 7.61 (m, 2H), 7.54 (s, 1H), 7.48 (s, 1H), 4.61 (m, 2H), 4.55 (m, 2H), 4.47 (m, 1H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.62 (m, 2H), 3.60 (m, 2H), 3.35 (m, 2H), 2.70 (s, 3H), 2.34 (m, 2H), 2.40 (q, <i>J</i> = 7.6 Hz, 2H), 2.35 (m, 2H), 2.00 (m, 2H), 1.10 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): 174.8, 166.8, 157.7, 154.7, 138.8, 135.2, 130.8, 130.0, 128.8, 126.8 (q, <i>J</i> = 276 Hz), 121.7, 121.2, 118.2, 114.4, 110.5, 58.8, 54.3, 54.0, 52.9, 47.1, 35.7 (q, <i>J</i> = 33 Hz), 30.0, 27.1, 15.8, 9.8. HRMS (ESI): [M+H<sup>+</sup>] calculated 653.2992; found 653.2989.</div></div><div id="sec4_12_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> General Procedure for Synthesis of Compounds <b>13</b> and <b>14</b></h4><div class="NLM_p last">Compound <b>3</b> (30 mg, 0.05 mmol) was dissolved in THF (0.5 mL). Fluoroacetic acid (3.1 μL, 0.06 mmol, 1.2 equiv), difluoroacetic acid (3.8 μL, 0.06 mmol, 1.2 equiv) or 2-cyclopentylacetic acid (70 mg, 0.5 mmol, 1.1 equiv) were added followed by TBTU (22.8 mg, 0.07 mmol, 1.3 equiv for <b>13</b> and <b>14</b>) or EDCI (11 mg, 0.06 mmol, 1.1 equiv for <b>16</b>). Reaction was carried out at room temperature until the starting material was not detected by LC-MS. (0.5–24 h). Crude mixture was loaded on preparative TLC plate and purified using DCM:methanol (20:1, v:v).</div></div><div id="sec4_12_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 1-(2-(4-(2-Fluoroacetyl)piperazin-1-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>13</b>)</h4><div class="NLM_p last">Reaction time: 3 h. Isolated 24 mg, 73% yield. <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.34 (s, 1H), 7.59 (m, 1H), 7.54 (m, 2H), 7.44 (s, 1H), 5.07 (d, <i>J</i> = 40 Hz, 2H), 4.54 (m, 2H), 4.47 (m, 3H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.63 (m, 2H), 3.53 (m, 2H), 3.35 (m, 2H), 2.77 (m, 2H), 2.70 (s, 3H), 2.50 (m, 2H), 2.45 (m, 2H), 2.34 (m, 2H), 1.99 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 168.2 (d, <i>J</i> = 40 Hz), 167.3, 158.0, 155.1, 139.0, 135.3, 130.8, 130.2, 129.0, 126.9 (q, <i>J</i> = 276 Hz), 122.0, 121.3, 118.4, 114.9, 113.4, 112.9, 110.7, 80.5 (d, <i>J</i> = 40 Hz), 59.1, 58.5, 54.5, 54.2, 53.2, 47.4, 45.5, 44.8, 43.1, 35.7 (q, <i>J</i> = 33 Hz), 30.2, 16.1. HRMS (ESI): [M+H<sup>+</sup>] calculated 657.2742; found 657.2745.</div></div><div id="sec4_12_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 1-(2-(4-(2,2-Difluoroacetyl)piperazin-1-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>14</b>)</h4><div class="NLM_p last">Reaction time: 24 h. Purified two times on the prep-TLC plate, isolated 15 mg, 32% yield. <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.34 (s, 1H), 7.56 (m, 3H), 7.43 (s, 1H), 6.43 (t, <i>J</i> = 53 Hz, 1H), 4.47 (m, 5H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.56 (m, 2H), 3.23 (m, 2H), 2.77 (m, 2H), 2.69 (s, 3H), 2.52 (m, 2H), 2.47 (m, 2H), 2.30 (m, 2H), 2.00 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 167.2, 162.6 (d, <i>J</i> = 25 Hz), 162.5, 157.9, 155.0, 138.8, 135.0, 130.6, 130.1, 130.1, 128.9, 126.9 (q, <i>J</i> = 276 Hz), 121.9, 118.3, 114.8, 113.2, 112.7, 111.4, 110.4, 109.8, 108.2, 58.3, 54.5, 54.0, 46.2, 44.7, 43.7, 39.0, 35.7 (q, <i>J</i> = 33 Hz), 15.9. HRMS (ESI): [M + H<sup>+</sup>] calculated 675.2647; found 675.2643.</div></div><div id="sec4_12_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-Methyl-1-(2-(4-(2,2,2-trifluoroacetyl)piperazin-1-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>15</b>)</h4><div class="NLM_p last">Compound <b>3</b> (30 mg, 0.05 mmol) was dissolved in DCM (0.25 mL) and cooled to 0 °C. The triflate anhydride (7 μL, 0.05 mmol) in DCM (0.25 mL) was added at 0 °C. The mixture was warmed to room temperature and solvents were evaporated with silica gel. Compound <b>15</b> (21 mg, 60% yield) was purified using flash chromatography (silica gel, DCM:methanol). <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.35 (s, 1H), 7.55(m, 3H), 7.43 (s, 1H), 4.50 (m, 5H), 3.85 (q, <i>J</i> = 11 Hz, 2H), 3.61 (m, 4H), 2.83 (m, 2H), 2.70 (s, 3H), 2.57 (m, 4H), 2.33 (m, 2H), 2.00 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 166.6, 157.7, 154.7, 138.8, 135.2, 130.6, 130.1, 128.8, 121.8, 121.1, 118.2, 114.6, 113.3, 112.6, 110.5, 58.9, 58.0, 54.2, 53.7, 52.9, 47.1, 46.8, 44.3, 35.7 (m), 30.0, 15.8; HRMS (ESI): [M+H<sup>+</sup>] calculated 693.2553; found 693.2558.</div></div><div id="sec4_12_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-(2-(4-(2-Cyclopentylacetyl)piperazin-1-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>16</b>)</h4><div class="NLM_p last">Compound <b>16</b> was prepared using the same general procedure as described for compounds <b>13</b> and <b>14</b>. Reaction time: 30 min. Isolated 19 mg, 54% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.36 (s, 1H), 7.55–7.62 (m, 3H), 7.45 (s, 1H), 4.50–4.60 (m, 2H), 4.45–4.53 (m, 3H), 3.88 (q, <i>J</i> = 10.6 Hz, 2H), 3.63 (d, <i>J</i> = 12.8 Hz, 2H), 3.49–3.57 (m, 4H), 3.30–3.43 (m, 2H, overlapped with the residual methanol signal), 2.78 (t, <i>J</i> = 5.5 Hz,. 2H), 2.72 (s, 3H), 2.50 (t, <i>J</i> = 5.14 Hz, 2H), 2.42 (t, <i>J</i> = 5.14 Hz, 2H), 2.39 (d, <i>J</i> = 7.34 Hz, 2H), 2.27–2.37 (m, 2H), 2.13–2.24 (m, 1H), 1.94–2.10 (m, 2H), 1.77–1.87 (m, 2H), 1.62–1.69 (m, 2H), 1.52–1.61 (m, 2H), 1.14–1.25 (m, 2H); <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 173.8, 167.1, 157.7, 154.9, 138.8, 135.0, 130.5, 130.0, 128.8, 126.6 (<i>J</i> = 277 Hz), 121.9, 121.2, 118.2, 114.6, 113.2, 112.6, 110.5, 58.9, 58.3, 54.7, 54.3, 52.9, 47.0, 44.6, 42.8, 39.9, 38.2, 35.7 (<i>J</i> = 32 Hz), 33.5, 30.0, 25.8, 15.8; HRMS (ESI): [M+H<sup>+</sup>] calculated 707.3462; found 707.3466.</div></div><div id="sec4_12_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1-(2-(4-(3-Cyclopentylpropanoyl)piperazin-1-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>17</b>)</h4><div class="NLM_p last">Compound <b>17</b> was prepared using the same synthetic procedure as described for compounds <b>11</b> and <b>12</b>. The reaction was carried out in DCM with 3-cyclopentylpropanoyl chloride. Reaction time: 30 min. Isolated 13 mg, 52% yield. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.36 (s, 1H), 7.50–7.64 (m, 3H), 7.44 (s, 1H), 4.48 (t, <i>J</i> = 5.9 Hz, 2H), 4.33–4.47 (m, 3H), 3.88 (q, <i>J</i> = 10.6 Hz, 2H), 3.48–3.60 (m, 6H), 3.10–3.24 (m, 2H), 2.78 (t, <i>J</i> = 5.9 Hz,. 2H), 2.70 (s, 3H), 2.48–2.54 (m, 2H), 2.41–2.45 (m, 2H), 2.36–2.40 (m, 2H), 2.26–2.33 (m, 2H), 2.21–2.26 (m, 1H), 1.93–2.12 (m, 2H), 1.77–1.86 (m, 2H), 1.62–1.69 (m, 2H), 1.51–1.61 (m, 4H), 1.08–1.19 (m, 2H); <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i><sub>4</sub>) δ 179.5, 174.3, 167.1, 157.7, 154.9, 138.7, 134.6, 130.5, 129.9, 128.8, 126.6 (<i>J</i> = 276.20 Hz) 121.8, 118.2, 114.7, 113.1, 112.4, 110.2, 58.3, 54.7, 54.2, 47.0, 44.6, 42.8, 41.1, 35.7 (<i>J</i> = 31.9 Hz), 35.9, 33.5, 33.5, 33.4, 33.3, 32.9, 26.1, 15.7; HRMS (ESI): [M+H<sup>+</sup>] calculated 721.3618; found 721.3619.</div></div><div id="sec4_12_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-(4-(2-(2-Cyano-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indol-1-yl)ethyl)piperazin-1-yl)propanamide (<b>18</b>)</h4><div class="NLM_p last">Compound <b>3</b> (57 mg, 0.1 mmol) was dissolved in DCM (0.5 mL) and 2-bromopropionamide (7 mg, 0.5eq) was added. After 24 h the reaction was diluted with water (despite the presence of starting bromide) and extracted with DCM:MeOH (10:1, v:v). Compound <b>18</b> was purified using preparative TLC (silica gel, DCM:MeOH:NH<sub>4</sub>OH, 10:1:0.05, v:v:v). Twelve mg (18% yield) of the product was isolated. <sup>1</sup>H NMR (600 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.33 (s, 1H), 7.55 (s, 1H), 7.46–7.54 (m, 2H), 7.40 (s, 1H), 4.45 (t, <i>J</i> = 6.2 Hz, 2H), 4.37–4.43 (m, 1H), 4.29–4.36 (m, 2H), 3.85 (q, <i>J</i> = 10.6 Hz, 2H), 3.47 (d, <i>J</i> = 12.8 Hz, 2H), 2.97–3.15 (m, 3H), 2.72–2.80 (m, 2H), 2.66 (s, 3H) 2.61–2.69 (m, 2H), 2.49–2.61 (m, 6H), 2.21–2.30 (m, 2H), 1.87–2.03 (m, 2H), 1.24 (d, <i>J</i> = 6.97 Hz, 3H); <sup>13</sup>C NMR (151 MHz, methanol-<i>d</i>4) δ 178.4, 167.0, 157.7, 154.9, 138.6, 134.4, 130.5, 129.8, 128.7, 126.4 (<i>J</i> = 276.2 Hz), 123.20, 121.8, 118.2, 114.6, 113.1, 112.1, 110.1; 65.2, 59.2, 58.3, 54.2, 52.8, 51.0, 47.5, 44.4, 35.8 (<i>J</i> = 31.8 Hz), 30.8, 30.3, 15.7, 14.3; HRMS (ESI): [M+H<sup>+</sup>] calculated 668.3107; found 668.3098</div></div><div id="sec4_12_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> General Procedure for Synthesis of Compounds <b>19</b>, <b>20</b>, and <b>21</b></h4><div class="NLM_p last">Compound <b>3</b> (1 equiv) and DIPEA (1.5 equiv) were dissolved in DCM (6 mL/mmol). At 0 °C the corresponding sulfonyl or sulfamoyl chloride (1 equiv) was added and reaction was left to warm to room temperature. The crude reaction was loaded on the silica gel column and purified using DCM:methanol.</div></div><div id="sec4_12_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-Methyl-1-(2-(4-(methylsulfonyl)piperazin-1-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>19</b>)</h4><div class="NLM_p last">The reaction was performed with the methanesulfonyl chloride. Isolated 13 mg, 58% yield. <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.36 (s, 1H), 7.59 (m, 2H), 7.54 (s, 1H), 7.46 (s, 1H), 4.55 (m, 4H), 4.47 (m, 1H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.63 (m, 2H), 3.34 (m, 2H), 3.25 (m, 4H), 2.94 (m, 2H), 2.84 (s, 3H), 2.73 (m, 4H), 2.70 (s, 3H), 2.4 (m, 2H), 2.00 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 166.8, 157.8, 154.8, 138.9, 135.3, 130.9, 130.2, 128.9, 126.8 (q, <i>J</i> = 276 Hz), 121.9, 121.3, 118.3, 114.6, 113.7, 112.5, 110.6, 59.0, 57.7, 53.9, 53.0, 47.3, 46.6, 44.0, 35.7 (q, <i>J</i> = 33 Hz), 34.9, 30.1, 15.9. HRMS (ESI): [M+H<sup>+</sup>] calculated 675.2506; found 675.2503.</div></div><div id="sec4_12_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 4-Methyl-1-(2-(4-(ethylsulfonyl)piperazin-1-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>20</b>)</h4><div class="NLM_p last">The reaction was performed with the ethanesulfonyl chloride. Isolated 20 mg, 58% yield. <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.34 (s, 1H), 7.60 (m, 1H), 7.54 (m, 2H), 7.44 (s, 1H), 4.54 (m, 2H), 4.49 (m, 3H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.63 (m, 2H), 3.34 (m, 2H), 3.26 (m, 4H), 3.02 (q, <i>J</i> = 7.6 Hz, 2H), 2.85 (m, 2H), 2.70 (s, 3H), 2.60 (m, 4H), 2.34 (m, 2H), 2.01 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 167.2, 157.9, 155.1, 139.0, 135.4, 130.9, 130.3, 129.0, 126.8 (q, <i>J</i> = 276 Hz), 122.0, 121.3, 118.4, 114.9, 113.6, 112.8, 110.7, 59.1, 58.2, 54.4, 53.2, 47.4, 46.8, 44.9, 44.5, 35.9 (q, <i>J</i> = 33 Hz), 30.2, 16.1, 8.3. HRMS (ESI): [M+H<sup>+</sup>] calculated 689.2662; found 689.2658.</div></div><div id="sec4_12_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-Methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1-(2-(4-((trifluoromethyl)sulfonyl)piperazin-1-yl)ethyl)-1<i>H</i>-indole-2-carbonitrile (<b>21</b>)</h4><div class="NLM_p last">Trifluoromethanosulfonyl chloride was added to the reaction mixture at −78 °C. Isolated 9 mg, 25% yield. <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.34 (s, 1H), 7.54 (m, 3H), 7.43 (s, 1H), 4.46 (m, 2H), 4.42 (m, 3H), 3.86 (q, <i>J</i> = 11 Hz, 2H), 3.53 (m, 2H), 3.45 (m, 2H), 3.19 (m, 4H), 2.80 (m, 2H), 2.68 (s, 3H), 2.55 (m, 4H), 2.28 (m, 2H), 1.97 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 167.0, 157.7, 154.9, 138.6, 134.7, 130.5, 129.9, 128.8, 126.8 (q, <i>J</i> = 276 Hz), 122.6, 121.8, 120.5, 118.2, 114.8, 113.1, 112.5, 110.2, 58.1, 54.0, 44.5, 35.6 (q, <i>J</i> = 33 Hz), 15.7. HRMS (ESI): [M+H<sup>+</sup>] calculated 729.2223; found 729.2218.</div></div><div id="sec4_12_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 4-(2-(2-Cyano-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indol-1-yl)ethyl)piperazine-1-sulfonamide (<b>22</b>)</h4><div class="NLM_p last">Compound <b>3</b> (59 mg, 0.1 mmol) and DIPEA (25 μL, 0.15 mmol, 1.5 equiv) were dissolved in DCM (1 mL). Benzyl (chlorosulfonyl)carbamate, CbzNHSO<sub>2</sub>Cl) (26 mg, 0.11 mmol, 1.1 equiv), prepared as described before<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was then added and reaction was carried out at room temperature for 30 min. Crude mixture was loaded on preparative TLC plate and Cbz-protected compound <b>22</b> was purified (silica gel, DCM:MeOH:NH<sub>3</sub>OH, 10:1:0.5 v:v:v). The product was then dissolved in MeOH (2.25 mL) and Pd/C (6 mg) and ammonium formate (9 mg, 0.13 mmol, 4 equiv) were added. The mixture was heated at 60 °C for 12 h. Mixture was filtered through Celite and evaporated. Purified using preparative TLC (silica gel, DCM:MeOH:NH<sub>3</sub>OH, 10:1:0.5 v:v:v). 5.6 mg (8% yield) of compound <b>22</b> was isolated. <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.51 (s, 1H), 7.76 (m, 1H), 7.67 (m, 2H), 7.58 (s, 1H), 4.56 (m, 4H), 4.42 (m, 3H), 3.92 (q, <i>J</i> = 11 Hz, 2H), 3.63 (m, 4H), 3.58 (m, 2H), 3.51 (m, 4H), 3.35 (m, 2H), 2.71 (s, 3H), 2.34 (m, 2H), 2.08 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 157.7, 153.7, 152.7, 139.1, 135.6, 131.6, 129.9, 129.7, 129.5, 129.0, 126.6 (q, <i>J</i> = 276 Hz), 122.3, 121.8, 118.5, 115.1, 114.0, 111.6, 110.6, 69.2, 63.3, 58.9, 55.6, 53.1, 52.9, 47.8, 45.1, 41.1, 35.5 (q, <i>J</i> = 33 Hz), 33.2, 30.9, 29.9, 26.2, 23.9, 16.0. HRMS (ESI): [M+H<sup>+</sup>] calculated 676.2458; found 676.2463.</div></div><div id="sec4_12_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-(2-(2-Cyano-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indol-1-yl)ethyl)-<i>N</i>-methylpiperazine-1-sulfonamide (<b>23</b>, Tested as a Free Base)</h4><div class="NLM_p last">Compound <b>23</b> was prepared using a general synthetic procedure as described for compounds <b>19</b>–<b>21</b>. Reaction performed with the methylsulfamoyl chloride added to the reaction mixture at the room temperature. Isolated 11 mg, 23% yield. <sup>1</sup>H NMR CD<sub>3</sub>OD (600 MHz): δ 8.30 (s, 1H), 7.52 (s, 1H), 7.39 (m, 3H), 4.41 (m, 2H), 4.19 (m, 1H), 3.83 (q, <i>J</i> = 11 Hz, 2H), 3.76 (m, 2H), 3.12 (m, 4H), 3.07 (m, 2H), 2.75 (m, 2H), 2.61 (s, 3H), 2.59 (s, 3H), 2.51 (m, 4H), 2.39 (m, 2H), 2.03 (m, 2H). <sup>13</sup>C NMR CD<sub>3</sub>OD (150 MHz): δ 166.9, 157.9, 155.1, 138.2, 132.9, 130.5, 129.7, 129.6, 128.9, 126.8 (q, <i>J</i> = 276 Hz), 122.0, 118.3, 115.2, 112.8, 111.5, 109.2, 60.7, 58.3, 54.0, 53.8, 47.2, 44.5, 35.5 (q, <i>J</i> = 33 Hz), 32.3, 30.9, 29.7, 15.5, 15.5. HRMS (ESI): [M+H<sup>+</sup>] calculated 690.2615; found 690.2611.</div></div><div id="sec4_12_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>S</i>)-4-Methyl-1-((5-oxomorpholin-2-yl)methyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>28</b>)</h4><div class="NLM_p">(<i>S</i>)-3-aminopropane-1,2-diol (<b>37a</b>, 5 g, 55 mmol, 1 equiv) was dissolved in methanol (20 mL) and the solution was diluted by AcCN (60 mL) and triethylamine was added, stirred and cooled at −20 °C. Chloroacetyl chloride was added slowly into the reaction mixture over 1 h and the temperature was controlled to −15 °C. The reaction mixture was slowly warmed to −10 °C and stirred for 1 h, then warmed to room temperature and stirred overnight. Solvents were removed and the residue was washed by ethyl acetate (800 mL), filtered and concentrated to give the desired compound (<i>S</i>)-<b>38a</b>. (<i>S</i>)-2-Chloro-<i>N</i>-(2,3-dihydroxypropyl)acetamide, (<i>S</i>)-<b>38a</b>, from the previous step (2.0 g, 12 mmol) in <i>tert-</i>amyl alcohol (32 mL) was added dropwise to the solution of <i>t-</i>BuOK (4.7 g, 42 mmol, 3.5 equiv) in <i>tert-</i>amyl alcohol (45 mL) at the room temperature for 2.5 h. The mixture was stirred for an additional 1 h. 1 M aqueous HCl (15 mL) was added to the mixture (to adjust pH to ∼2) and the mixture was stirred for additional 30 min. Solvents were removed and the (<i>S</i>)-6-(hydroxymethyl)morpholin-3-one (<i>S</i>)-<b>39a</b> was purified using column chromatography (silica gel, hexane:ethyl acetate then DCM:MeOH, 0–10% v:v). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.90 (br s, 1H), 4.82 (t, <i>J</i> = 5.69 Hz, 1H), 3.92–4.10 (m, 2H), 3.60–3.71 (m, 1H), 3.45–3.53 (m, 1H), 3.36–3.45 (m, 1H), 3.18 (td, <i>J</i> = 3.85, 12.10 Hz, 1H), 3.02–3.13 (m, 1H). Compound (<i>S</i>)-<b>39a</b> (1 g, 7.6 mmol) was dissolved in DCM (12 mL) and TEA (1.4 mL, 9.9 mmol, 1.3 equiv) was added. Mixture was cooled to 0 °C and methanesulfonyl chloride (823 μL, 9.9 mmol, 1.3 equiv) was added dropwise. The mixture was warmed to room temperature and stirred overnight. Solvent was evaporated and the residue was purified using column chromatography (silica gel, DCM:MeOH, 0–10% v:v) providing 1.2 g of the(S)-(5-oxomorpholin-2-yl)methylmethanesulfonate, which was directly used in the next step. The mesylate from the previous step (1.2 g) was dissolved in DMA (22 mL) and cesium carbonate (7.5 g, 22.8 mmol, 4 equiv) and compound <b>30</b> (1.1 g, 5.7 mmol, 1 equiv) were added. Mixture was stirred at 105 °C for 4 h. After cooling to room temperature, ethyl acetate (150 mL) was added along with water (3 × 15 mL). Organic phase was then separated and concentrated. The residue was purified using column chromatography (silica gel, DCM:MeOH, 0–10% v:v) resulting in 420 mg of compound (<i>S</i>)-<b>40a</b>, (<i>S</i>)-5-formyl-4-methyl-1-((5-oxomorpholin-2-yl)methyl)-1<i>H</i>-indole-2-carbonitrile, MS (<i>m</i>/<i>z</i>) found for [M+AcCN+H<sup>+</sup>]: 339 Da. Compound (<i>S</i>)-<b>40a</b> (400 mg, 1.4 mmol) was dissolved in THF (48 mL) followed by addition of compound <b>33</b> (638 mg, 2.0 mmol, 1.5 equiv) and acetic acid (116 μL). Resulting mixture was stirred at 50 °C for 30 min, then cooled to room temperature. Sodium triacetoxyborohydride (572 mg, 2.69 mmol, 2 equiv) was then added and the mixture was stirred for 5 h at 50 °C. Cooled mixture was then partitioned between EtOAc (60 mL) and 1 M aqueous NaOH (25 mL). Organic phase was concentrated and product was purified by column chromatography (silica gel, DCM:MeOH, 0–10% v:v). 520 mg (65% yield) of compound <b>28</b> was obtained. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.68 (br s, 1H), 8.38 (s, 1H), 8.20–8.29 (m, 1H), 8.00–8.03 (m, 1H), 7.76–7.81 (m, 1H), 7.75 (br s, 1H), 7.62–7.66 (m, 2H), 4.57–4.62 (m, 1H), 4.31–4.45 (m, 4H), 4.01–4.10 (m, 3H), 4.00 (d, <i>J</i> = 16.4 Hz, 1H), 3.89 (d, <i>J</i> = 16.5 Hz, 1H), 3.35–3.47 (m, 3H, overlapped with water), 3.21–3.32 (m, 2H), 3.18 (t, <i>J</i> = 11.37 Hz, 1H), 2.64 (s, 3H), 2.01–2.14 (m, 4H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 155.7, 153.4, 137.3, 133.3, 130.0, 127.2, 126.5, 125.4 (<i>J</i> = 277 Hz) 121.5, 120.4, 116.2, 113.5, 113.2, 110.2, 109.3, 71.5, 66.9, 56.1, 50.6, 50.5, 46.8, 45.7, 42.7, 33.8 (<i>J</i> =  31 Hz), 28.2, 15.5; <sup>1</sup>H NMR (600 MHz, METHANOL-d4) δ 8.41 (s, 1H), 7.57–7.67 (m, 3H), 7.49 (s, 1H), 4.61 (dd, <i>J</i> = 2.75, 15.22 Hz, 1H), 4.56 (s, 2H), 4.47–4.54 (m, 2H), 4.14–4.21 (m, 1H), 4.15 (AB, d, <i><u class="uu">J</u></i> = 16.51 Hz, 1H), 4.00 (AB, d, <i><u class="uu">J</u></i> = 16.51 Hz, 1H), 3.83–3.93 (m, 2H), 3.65 (d, <i>J</i> = 11.74 Hz, 2H), 3.48 (dd, <i>J</i> = 2.38, 12.29 Hz, 1H), 3.33–3.41 (m, 3H, partially overlapped with MeOH), 2.71 (s, 3H), 2.35 (bd, <i>J</i> = 13.20 Hz, 2H), 2.06 (q, <i>J</i> = 12.47 Hz, 2H); <sup>13</sup>C NMR (151 MHz, METHANOL-<i>d</i><sub>4</sub>) δ 170.6, 165.1, 157.6, 153.9, 139.6, 135.0, 130.8, 130.5, 128.6, 126.6 (<i>J</i> = 276 Hz), 125.6, 121.9, 121.2, 118.3, 114.2, 114.0, 112.5, 73.3, 68.0, 58.8, 52.9, 52.9, 48.2, 47.3, 44.4, 35.6 (<i>J</i> = 32 Hz), 29.9, 15.8; [α]<sup>24</sup><sub>D</sub> −17.5 (<i>c</i> 1.0, MeOH); HR-MS (ESI): [M+H<sup>+</sup>] calculated 598.2212, found 598.2208.</div><div class="NLM_p last">Synthesis of compounds <b>4</b>, <b>6–8</b>, <b>24–27</b>, and <b>29</b> was outsourced to the Contract Research Organization (CRO). These compounds were prepared using the general methodology presented in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>, <a class="ref internalNav" href="#sch3" aria-label="3">3</a>, <a class="ref internalNav" href="#sch4" aria-label="4">4</a>, and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>. Final compounds were characterized in house, and the purity was confirmed by HPLC before their biological activity was evaluated.</div></div><div id="sec4_12_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 4-Methyl-1-(2-(piperidin-4-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>4</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.02 (br s, 1H), 9.08 (br s, 1H), 8.90 (br s, 1H), 8.79 (br s, 1H), 8.49 (s, 1H), 7.74–7.91 (m, 2H), 7.65 (s, 1H), 7.57 (d, <i>J</i> = 8.80 Hz, 1H), 4.31–4.47 (m, 5H), 4.08 (q, <i>J</i> = 10.88 Hz, 2H), 3.41 (d, <i>J</i> = 11.37 Hz, 2H), 3.18–3.30 (m, 4H), 2.80 (q, <i>J</i> = 11.74 Hz, 2H), 2.65 (s, 3H), 2.04–2.22 (m, 4H), 1.88 (d, <i>J</i> = 12.84 Hz, 2H), 1.70 (q, <i>J</i> = 6.72 Hz, 2H), 1.55–1.65 (m, 1H), 1.36–1.48 (m, 2H) <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.2, 151.6, 136.8, 133.5, 130.2, 129.9, 128.3, 126.6, 125.4 (<i>J</i> = 276 Hz), 121.7, 120.3, 116.5, 113.5, 113.1, 108.8, 108.8, 56.0, 50.4, 46.2, 42.8, 42.7, 35.9, 33.7 (<i>J</i> = 31 Hz) 30.7, 28.0, 28.0, 15.6; HRMS (ESI): [M+H<sup>+</sup>] calculated 596.2783; found 596.2776.</div></div><div id="sec4_12_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 4-Methyl-1-(2-(5-oxomorpholin-2-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>6</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.91 (br s, 1H), 8.72–8.88 (m, 1H), 8.43–8.59 (m, 1H), 7.91–7.98 (m, 1H), 7.86 (s, 1H), 7.82 (d, <i>J</i> = 8.80 Hz, 1H), 7.65 (s, 1H), 7.53 (d, <i>J</i> = 8.80 Hz, 1H), 4.33–4.57 (m, 5H), 4.09 (q, <i>J</i> =  11.0 Hz, 2H), 4.05 (d, <i>J</i> = 16.1 Hz, 1H), 3.92 (d, <i>J</i> = 16.1 Hz, 1H), 3.49–3.58 (m, 1H), 3.36–3.46 (m, 2H), 3.19–3.30 (m, 2H), 3.11–3.18 (m, 1H), 3.02–3.10 (m, 1H), 2.65 (s, 3H), 2.06–2.26 (m, 4H), 1.99–2.07 (m, 1H), 1.85–1.95 (m, 1H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.0, 155.3, 151.6, 136.9, 133.5, 130.1, 128.3, 126.6, 125.4 (<i>J</i> = 277 Hz), 121.7, 120.3, 116.5, 113.4, 113.1, 109.1, 108.8, 70.0, 66.8, 56.0, 50.5, 50.4, 46.2, 44.9, 41.6, 33.7 (<i>J</i> = 31 Hz), 32.3, 28.0, 15.6 HRMS (ESI): [M+H<sup>+</sup>] calculated 612.2369; found 612.2369.</div></div><div id="sec4_12_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 4-Methyl-1-((5-oxomorpholin-2-yl)methyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>7</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.67 (br s, 1H), 8.52–8.63 (m, 1H), 8.45 (s, 1H), 8.01 (d, <i>J</i> = 3.67 Hz, 1H), 7.74–7.83 (m, 2H), 7.59–7.67 (m, 2H), 4.59 (dd, <i>J</i> = 2.75, 15.22 Hz, 1H), 4.33–4.47 (m, 4H), 4.03–4.14 (m, 3H), 3.99 (d, <i>J</i> = 16.1 Hz, 1H), 3.89 (d, <i>J</i> = 16.5 Hz, 1H), 3.36–3.47 (m, 3H), 3.21–3.33 (m, 2H), 3.18 (t, <i>J</i> = 11.37 Hz, 1H), 2.64 (s, 3H), 2.03–2.24 (m, 4H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 155.5, 152.4, 137.4, 133.3, 130.0, 127.9, 126.6, 125.4 (<i>J</i> = 277 Hz) 121.6, 120.4, 116.4, 113.5, 113.2, 110.2, 109.3, 71.5, 66.9, 56.1, 50.6, 50.5, 46.8, 46.0, 42.7, 33.7 (<i>J</i> =  31 Hz), 28.1, 15.5; HRMS (ESI): [M+H<sup>+</sup>] calculated 598.2212; found 598.2205.</div></div><div id="sec4_12_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 4-Methyl-1-(2-(4-propylpiperazin-1-yl)methyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>8</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-d6) δ 10.66 (br s,1H), 8.38 (s, 1H), 8.25 (d, <i>J</i> = 6.60 Hz, 1H), 7.78 (d, <i>J</i> = 8.80 Hz, 1H), 7.74 (s, 1H), 7.58–7.70 (m, 2H), 4.50 (m, 2H), 4.40 (d, <i>J</i> = 5.14 Hz, 2H), 4.31–4.39 (m, 1H), 4.05 (q, <i>J</i> = 11.13 Hz, 2H), signals overlapped with water: 3.39 (HSQC), 3.36 (HSQC), 3.20–3.29 (m, 2H), 3.02–3.15 (m, 2H), 2.93–3.02 (m, 2H), 2.80–2.90 (m, 2H), 2.75 (s, 3H), 2.64 (s, 3H), 2.01–2.22 (m, 4H); <sup>13</sup>C NMR (151 MHz, DMSO-d6) δ 155.6, 136.8, 133.4, 130.0, 126.6, 121.4, 116.2, 109.0, 69.1, 56.7 (HSQC) 56.1, 52.0, 50.6, 49.3, 45.8, 42.7 (HSQC), 41.9, 33.6, 28.2, 15.5; HRMS (ESI): [M+H<sup>+</sup>] calculated 611.2892; found 611.2873.</div></div><div id="sec4_12_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 1-(2-(1-Formylpiperidin-4-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>24</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.42 (br s, 1H), 8.36–8.43 (m, 1H), 8.22 (d, <i>J</i> = 7.34 Hz, 1H), 7.96 (s, 1H), 7.71–7.77 (m, 2H), 7.65 (s, 1H), 7.58 (d, <i>J</i> = 8.80 Hz, 1H), 4.37–4.45 (m, 4H), 4.29–4.36 (m, 1H), 4.13 (d, <i>J</i> = 13.20 Hz, 1H), 4.05 (q, <i>J</i> = 11.00 Hz, 2H), 3.66 (d, <i>J</i> = 13.20 Hz, 1H), 3.42 (m, 2H, HSQC, overlapped with water) 3.20–3.29 (m, 2H), 2.94–3.03 (m, 1H), 2.64 (s, 3H), 2.54–2.61 (m, 1H), 2.08–2.20 (m, 2H), 1.96–2.07 (m, 2H), 1.73–1.84 (m, 2H), 1.71 (q, <i>J</i> = 6.97 Hz, 2H), 1.52–1.62 (m, 1H), 0.96–1.16 (m, 2H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.6, 155.7, 153.3, 136.8, 133.5, 130.0, 127.3, 126.6, 125.5 (<i>J</i> = 277 Hz), 121.4, 120.3, 116.2, 113.5, 113.1, 108.9, 108.9, 56.2, 50.6, 45.7, 44.8, 42.8, 38.8, 36.2, 33.8 (<i>J</i> = 30 Hz), 33.2, 32.1, 30.8, 28.2, 15.5; HRMS (ESI): [M+Na<sup>+</sup>] calculated 646.2552; found 646.2544.</div></div><div id="sec4_12_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(2-(1-Acetylpiperidin-4-yl)ethyl)-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>25</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.22 (br s, 1H), 8.36 (s, 1H), 8.11 (d, <i>J</i> = 6.97 Hz, 1H), 7.69–7.74 (m, 2H), 7.65 (s, 1H), 7.58 (d, <i>J</i> = 8.80 Hz, 1H), 4.36–4.46 (m, 4H), 4.30–4.35 (m, 2H), 4.05 (q, <i>J</i> = 10.9 Hz, 2H), 3.78 (d, <i>J</i> = 13.2 Hz, 1H), 2.97 (t, <i>J</i> = 13.2 Hz, 1H), 2.63 (s, 3H), 2.44–2.49 (m, 1H), 2.10–2.15 (m, 2H), 1.97 (s, 3H), 1.92–2.02 (m, 2H), 1.76 (d, <i>J</i> = 13.2 Hz, 1H), 1.66–1.74 (m, 3H), 1.48–1.57 (m, 1H), 1.15 (m, 1H), 0.96–1.07 (m, 1H); HSQC (<sup>1</sup>H–<sup>13</sup>C): signals overlapped with water: 3.43 (CH<sub>2</sub>), 3.25 (CH<sub>2</sub>); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i>6<sub>)</sub> δ 167.8, 165.6, 155.7, 153.7, 136.8, 133.5, 130.0, 127.2, 126.6, 125.50 (<i>J</i> = 277 Hz), 121.4, 120.2, 116.9, 116.2, 113.5, 113.0, 108.9 56.3, 50.7, 45.7, 45.6, 42.9, 40.8, 36.2, 33.8 (<i>J</i> = 32 Hz), 32.9, 32.0, 31.3, 28.3, 21.2, 15.5 HRMS (ESI): [M+Na<sup>+</sup>] calculated 660.2703; found 660.2699.</div></div><div id="sec4_12_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 4-Methyl-1-(2-(1-(methylsulfonyl)piperidin-4-yl)ethyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>26</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.75 (br s, 1H), 8.60 (br s, 1H), 8.46 (s, 1H), 7.81 (s, 1H), 7.79 (d, <i>J</i> = 8.80 Hz, 1H), 7.65 (s, 1H), 7.58 (d, <i>J</i> = 8.80 Hz, 1H), 4.32–4.45 (m, 5H), 4.08 (q, <i>J</i> = 11.00 Hz, 2H), 3.54 (d, <i>J</i> = 12.1 Hz, 2H), 3.42 (d, <i>J</i> = 12.1 Hz, 2H), 3.20–3.31 (m, 2H), 2.83 (s, 3H), 2.65–2.69 (m, 2H), 2.64 (s, 3H), 2.04–2.24 (m, 4H), 1.79–1.87 (m, 2H), 1.69–1.78 (m, 2H), 1.40 (m, 1H), 1.22–1.30 (m, 2H) <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.4, 152.2, 136.8, 133.5, 130.1, 126.6, 126.4, 121.6, 120.3, 116.4, 113.5, 113.1, 108.9, 108.9, 108.9, 56.1, 50.5, 46.1, 45.3, 42.8, 35.9, 34.1, 33.7 (<i>J</i> =  30 Hz), 32.1, 30.9, 28.1, 15.5; HRMS (ESI): [M+H<sup>+</sup>] calculated 674.2559; found 674.2548.</div></div><div id="sec4_12_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 4-(2-(2-Cyano-4-methyl-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indol-1-yl)ethyl)piperidine-1-sulfonamide (<b>27</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.58 (br s, 1H), 8.42 (s, 1H), 8.34–8.41 (m, 1H), 7.73–7.83 (m, 2H), 7.65 (s, 1H), 7.58 (d, <i>J</i> = 8.80 Hz, 1H), 6.65 (br s, 2H), 4.32–4.46 (m, 5H), 4.06 (q, <i>J</i> = 11.13 Hz, 2H), 3.38–3.47 (m, 4H), 3.17–3.35 (m, 2H), 2.64 (s, 3H), 2.44–2.48 (m, 2H), 2.02–2.21 (m, 4H), 1.77–1.88 (m, 2H), 1.66–1.75 (m, 2H), 1.30–1.40 (m, 1H), 1.19–1.30 (m, 2H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.6, 152.8, 136.8, 133.5, 130.0, 127.6, 126.6, 126.4, 124.5, 121.5, 120.3, 116.3, 113.5, 113.1, 108.9, 56.1, 50.6, 45.9, 42.9, 40.1, 36.0, 33.75 (<i>J</i> = 31 Hz), 32.3, 30.6, 28.2, 15.5; HRMS (ESI): [M+H<sup>+</sup>] calculated 675.2511; found 675.2505.</div></div><div id="sec4_12_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>R</i>)-4-Methyl-1-((5-oxomorpholin-2-yl)methyl)-5-((4-((6-(2,2,2-trifluoroethyl)thieno[2,3-<i>d</i>]pyrimidin-4-yl)amino)piperidin-1-yl)methyl)-1<i>H</i>-indole-2-carbonitrile (<b>29</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.68 (br s, 1H), 8.48–8.56 (m, 1H), 8.44 (s, 1H), 8.01 (d, <i>J</i> = 3.67 Hz, 1H), 7.75–7.81 (m, 2H), 7.61–7.66 (m, 2H), 4.59 (dd, <i>J</i> = 2.75, 15.22 Hz, 1H), 4.33–4.45 (m, 4H), 4.03–4.11 (m, 3H), 4.00 (d, <i>J</i> = 16.5 Hz, 1H), 3.89 (d, <i>J</i> = 16.5 Hz, 1H), 3.37–3.47 (m, 3H), 3.21–3.32 (m, 2H), 3.18 (t, <i>J</i> = 11.37 Hz, 1H), 2.64 (s, 3H), 2.06–2.15 (m, 4H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 155.5, 152.5, 137.4, 133.3, 130.0, 127.8, 126.4, 125.4 (<i>J</i> = 276 Hz) 121.6, 120.4, 116.4, 113.5, 113.2, 110.2, 109.3, 71.5, 66.9, 56.1, 50.6, 50.5, 46.8, 46.0, 42.7, 33.7 (<i>J</i> =  32 Hz), 28.1, 15.5; [α]<sup>24</sup><sub>D</sub> 18.1 (<i>c</i> 1.0, MeOH); HR-MS (ESI): [M+H<sup>+</sup>] calculated 598.2212, found 598.2207.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00071" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00071" class="ext-link">10.1021/acs.jmedchem.8b00071</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary Table 1, crystallography data; Supplementary Figure 1, <i>K</i><sub>d</sub> results from the FP assay for MLL<sub>4-43</sub> binding to menin; Supplementary Figure 2, BLI studies for menin–MLL inhibitors; Supplementary Figure 3, PK results for <b>28</b>; Supplementary Figure 4, mice body weight from <i>in vivo</i> efficacy studies with <b>28</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf">jm8b00071_si_001.pdf (431.98 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_002.csv">jm8b00071_si_002.csv (2.98 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinate files have been deposited in PDB with the following codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXH">6BXH</a> (menin-compound <b>13</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXY">6BXY</a> (menin-compound <b>28</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BY8">6BY8</a> (menin-compound <b>29</b>).</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00071" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12177" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12177" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jolanta Grembecka</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6180-9095" title="Orcid link">http://orcid.org/0000-0002-6180-9095</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#761c191a171802171136031b1f151e58131203"><span class="__cf_email__" data-cfemail="6d0702010c03190c0a2d1800040e0543080918">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dmitry Borkin</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Szymon Klossowski</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan Pollock</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongzhi Miao</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian M. Linhares</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katarzyna Kempinska</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhuang Jin</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Trupta Purohit</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Wen</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Miao He</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duxin Sun</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tomasz Cierpicki</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.B. and S.K. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Drs. Grembecka and Cierpicki receive research support from Kura Oncology, Inc. They have also an equity ownership in the company. Other co-authors declare no potential conflict of interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d49e5091-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was funded by the National Institutes of Health (NIH) grants R01 (1R01CA160467) to J.G., R01 (1R01CA200660) and R01 (1R01CA201204), a Leukemia and Lymphoma Society LLS Scholar (1215-14) to J.G., LLS TAP (Therapy Acceleration Program) to J.G., American Cancer Society Research Scholar grant RSG-13-130-01-CDD to J.G., and the Department of Pathology, University of Michigan. Use of the Advanced Photon Source was supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences, under contract number DE-AC02-06CH11357. Use of the LS-CAT Sector 21 was supported by the Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor for the support of this research program (grant 085P1000817). The mouse work was performed under oversight of UCUCA at the University of Michigan.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed lineage leukemia</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">NEAA</td><td class="NLM_def"><p class="first last">nonessential amino acids</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">AcCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">AcOH</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N,N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-(dimethylamino)propyl)-<i>N</i>′-ethylcarbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TBTU</td><td class="NLM_def"><p class="first last"><i>O</i>-(benzotriazol-1-yl)-<i>N,N,N′,N′</i>-tetramethyluronium tetrafluoroborate</p></td></tr><tr><td class="NLM_term">TBAI</td><td class="NLM_def"><p class="first last">tetrabutylammonium iodide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34621" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34621" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Challenges and opportunities in targeting the menin–MLL interaction</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.4155/fmc.13.214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.4155%2Ffmc.13.214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=24635524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=447-462&issue=4&author=T.+Cierpickiauthor=J.+Grembecka&title=Challenges+and+opportunities+in+targeting+the+menin%E2%80%93MLL+interaction&doi=10.4155%2Ffmc.13.214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and opportunities in targeting the menin-MLL interaction</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-462</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Menin is an essential co-factor of oncogenic MLL fusion proteins and the menin-MLL interaction is crit. for development of acute leukemia in vivo.  Targeting the menin-MLL interaction with small mols. represents an attractive strategy to develop new anticancer agents.  Recent developments, including detn. of menin crystal structure and development of potent small mol. and peptidomimetic inhibitors, demonstrate the feasibility of targeting the menin-MLL interaction.  On the other hand, biochem. and structural studies revealed that MLL binds to menin in a complex bivalent mode engaging two MLL motifs, and therefore inhibition of this protein-protein interaction represents a challenge.  This review summarizes the most recent achievements in targeting the menin-MLL interaction as well as discusses potential benefits of blocking menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovLrSLXU5DNbVg90H21EOLACvtfcHk0liK-q6hg9sRxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D&md5=97777abe7b2fd8df3daf390d9b019721</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.214%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DChallenges%2520and%2520opportunities%2520in%2520targeting%2520the%2520menin%25E2%2580%2593MLL%2520interaction%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D4%26spage%3D447%26epage%3D462%26doi%3D10.4155%2Ffmc.13.214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>123</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.cell.2005.09.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=207-218&issue=2&author=A.+Yokoyamaauthor=T.+C.+Somervailleauthor=K.+S.+Smithauthor=O.+Rozenblatt-Rosenauthor=M.+Meyersonauthor=M.+L.+Cleary&title=The+menin+tumor+suppressor+protein+is+an+essential+oncogenic+cofactor+for+MLL-associated+leukemogenesis&doi=10.1016%2Fj.cell.2005.09.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26aulast%3DSmith%26aufirst%3DK.%2BS.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DThe%2520menin%2520tumor%2520suppressor%2520protein%2520is%2520an%2520essential%2520oncogenic%2520cofactor%2520for%2520MLL-associated%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D123%26issue%3D2%26spage%3D207%26epage%3D218%26doi%3D10.1016%2Fj.cell.2005.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marschalek, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of leukemogenesis by MLL fusion proteins</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>152</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2141.2010.08459.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1111%2Fj.1365-2141.2010.08459.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=21118195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2011&pages=141-154&issue=2&author=R.+Marschalek&title=Mechanisms+of+leukemogenesis+by+MLL+fusion+proteins&doi=10.1111%2Fj.1365-2141.2010.08459.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of leukemogenesis by MLL fusion proteins</span></div><div class="casAuthors">Marschalek, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-154</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Infant acute leukemia is characterised by specific genetic rearrangements and a rapid onset of disease shortly after birth.  The vast majority of these cases bear rearranged MLL alleles.  However, many facets of MLL-rearranged leukemia are largely unknown.  Basically, there exists a fundamental and evolutionary conserved relationship between the family of MLL/Trithorax proteins and the regulation of HOX gene clusters.  Therefore, direct MLL fusion proteins are per se able to deregulate HOX genes, except when reciprocal MLL fusion proteins come into play.  This reviews discusses (i) the current situation in MLL-rearranged leukemia, (ii) the mol. and genetic tools to functionally investigate the many different MLL fusions, (iii) the latency of disease development, (iv) a novel cancer mechanism that has been recently uncovered when different MLL fusion protein complexes were characterized, (v) mutated signalling pathways in MLL-rearranged leukemia and (vi) presents new ideas on how a given MLL fusion protein may modulate existing signalling pathways in leukemic cells.  The hypothesis is posed that the many different fusion partners of MLL are critically distinct entities for which specific inhibitors should be identified in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeSBrUEMt1JLVg90H21EOLACvtfcHk0liK-q6hg9sRxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D&md5=f0330065cb216625a242a2508d5d1c93</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2010.08459.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2010.08459.x%26sid%3Dliteratum%253Aachs%26aulast%3DMarschalek%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520leukemogenesis%2520by%2520MLL%2520fusion%2520proteins%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2011%26volume%3D152%26issue%3D2%26spage%3D141%26epage%3D154%26doi%3D10.1111%2Fj.1365-2141.2010.08459.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimartino, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Mll rearrangements in haematological malignancies: lessons from clinical and biological studies</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2141.1999.01439.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1046%2Fj.1365-2141.1999.01439.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10468849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1999&pages=614-626&issue=3&author=J.+F.+Dimartinoauthor=M.+L.+Cleary&title=Mll+rearrangements+in+haematological+malignancies%3A+lessons+from+clinical+and+biological+studies&doi=10.1046%2Fj.1365-2141.1999.01439.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MLL rearrangements in haematological malignancies: Lessons from clinical and biological studies</span></div><div class="casAuthors">DiMartino, Jorge F.; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">614-626</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 96 refs., covering the MLL family of 11q23 chromosomal rearrangements, assocn. between 11q23 rearrangements and leukemia phenotype, MLL rearrangements in the absence of leukemia, prognostic implications of 11q23 rearrangements, etiol. of MLL rearrangements, MLL rearrangements leading to leukemia with short latency, structure and probable function of the MLL protein, and possible mechanisms of leukemogenesis by MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOiR8FH8-xQbVg90H21EOLACvtfcHk0lhe5Hg6fOBeWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D&md5=845845da43dc70b59322261edb88bb6e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2141.1999.01439.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2141.1999.01439.x%26sid%3Dliteratum%253Aachs%26aulast%3DDimartino%26aufirst%3DJ.%2BF.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMll%2520rearrangements%2520in%2520haematological%2520malignancies%253A%2520lessons%2520from%2520clinical%2520and%2520biological%2520studies%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D1999%26volume%3D106%26issue%3D3%26spage%3D614%26epage%3D626%26doi%3D10.1046%2Fj.1365-2141.1999.01439.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span> <span> </span><span class="NLM_article-title">MLL: how complex does it get?</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>95</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1002/jcb.20430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fjcb.20430" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2005&pages=234-242&issue=2&author=R.+Popovicauthor=N.+J.+Zeleznik-Le&title=MLL%3A+how+complex+does+it+get%3F&doi=10.1002%2Fjcb.20430"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fjcb.20430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.20430%26sid%3Dliteratum%253Aachs%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26atitle%3DMLL%253A%2520how%2520complex%2520does%2520it%2520get%253F%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2005%26volume%3D95%26issue%3D2%26spage%3D234%26epage%3D242%26doi%3D10.1002%2Fjcb.20430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caslini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Osta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Interaction of MLL amino terminal sequences with menin is required for transformation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">7275</span>– <span class="NLM_lpage">7283</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-2369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1158%2F0008-5472.CAN-06-2369" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=7275-7283&issue=15&author=C.+Casliniauthor=Z.+Yangauthor=M.+El-Ostaauthor=T.+A.+Milneauthor=R.+K.+Slanyauthor=J.+L.+Hess&title=Interaction+of+MLL+amino+terminal+sequences+with+menin+is+required+for+transformation&doi=10.1158%2F0008-5472.CAN-06-2369"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-2369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-2369%26sid%3Dliteratum%253Aachs%26aulast%3DCaslini%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DEl-Osta%26aufirst%3DM.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DSlany%26aufirst%3DR.%2BK.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DInteraction%2520of%2520MLL%2520amino%2520terminal%2520sequences%2520with%2520menin%2520is%2520required%2520for%2520transformation%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26issue%3D15%26spage%3D7275%26epage%3D7283%26doi%3D10.1158%2F0008-5472.CAN-06-2369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Menin critically links MLL proteins with LEDGF on cancer-associated target genes</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2008.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.ccr.2008.05.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=36-46&issue=1&author=A.+Yokoyamaauthor=M.+L.+Cleary&title=Menin+critically+links+MLL+proteins+with+LEDGF+on+cancer-associated+target+genes&doi=10.1016%2Fj.ccr.2008.05.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMenin%2520critically%2520links%2520MLL%2520proteins%2520with%2520LEDGF%2520on%2520cancer-associated%2520target%2520genes%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D14%26issue%3D1%26spage%3D36%26epage%3D46%26doi%3D10.1016%2Fj.ccr.2008.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorenson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1038/nchembio.773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnchembio.773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=22286128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=277-284&issue=3&author=J.+Grembeckaauthor=S.+Heauthor=A.+Shiauthor=T.+Purohitauthor=A.+G.+Munteanauthor=R.+J.+Sorensonauthor=H.+D.+Showalterauthor=M.+J.+Muraiauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=J.+L.+Hessauthor=T.+Cierpicki&title=Menin-MLL+inhibitors+reverse+oncogenic+activity+of+MLL+fusion+proteins+in+leukemia&doi=10.1038%2Fnchembio.773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span></div><div class="casAuthors">Grembecka, Jolanta; He, Shihan; Shi, Aibin; Purohit, Trupta; Muntean, Andrew G.; Sorenson, Roderick J.; Showalter, Hollis D.; Murai, Marcelo J.; Belcher, Amalia M.; Hartley, Thomas; Hess, Jay L.; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Translocations involving the mixed lineage leukemia (MLL) gene result in human acute leukemias with very poor prognosis.  The leukemogenic activity of MLL fusion proteins is critically dependent on their direct interaction with menin, a product of the multiple endocrine neoplasia (MEN1) gene.  Here we present what are to our knowledge the first small-mol. inhibitors of the menin-MLL fusion protein interaction that specifically bind menin with nanomolar affinities.  These compds. effectively reverse MLL fusion protein-mediated leukemic transformation by downregulating the expression of target genes required for MLL fusion protein oncogenic activity.  They also selectively block proliferation and induce both apoptosis and differentiation of leukemia cells harboring MLL translocations.  Identification of these compds. provides a new tool for better understanding MLL-mediated leukemogenesis and represents a new approach for studying the role of menin as an oncogenic cofactor of MLL fusion proteins.  Our findings also highlight a new therapeutic strategy for aggressive leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_e6VYyS5l7Vg90H21EOLACvtfcHk0lhe5Hg6fOBeWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D&md5=55eb9f353031dfef4c1a281b0ac28cba</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.773%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DA.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DSorenson%26aufirst%3DR.%2BJ.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMenin-MLL%2520inhibitors%2520reverse%2520oncogenic%2520activity%2520of%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26issue%3D3%26spage%3D277%26epage%3D284%26doi%3D10.1038%2Fnchembio.773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of the bivalent menin–MLL interaction by small molecules in leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">4461</span>– <span class="NLM_lpage">4469</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&issue=23&author=A.+Shiauthor=M.+J.+Muraiauthor=S.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+insights+into+inhibition+of+the+bivalent+menin%E2%80%93MLL+interaction+by+small+molecules+in+leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520the%2520bivalent%2520menin%25E2%2580%2593MLL%2520interaction%2520by%2520small%2520molecules%2520in%2520leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26issue%3D23%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the Menin-MLL interaction blocks progression of MLL leukemia in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.ccell.2015.02.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=589-602&issue=4&author=D.+Borkinauthor=S.+Heauthor=H.+Miaoauthor=K.+Kempinskaauthor=J.+Pollockauthor=J.+Chaseauthor=T.+Purohitauthor=B.+Malikauthor=T.+Zhaoauthor=J.+Wangauthor=B.+Wenauthor=H.+Zongauthor=M.+Jonesauthor=G.+Danet-Desnoyersauthor=M.+L.+Guzmanauthor=M.+Talpazauthor=D.+L.+Bixbyauthor=D.+Sunauthor=J.+L.+Hessauthor=A.+G.+Munteanauthor=I.+Maillardauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+Menin-MLL+interaction+blocks+progression+of+MLL+leukemia+in+vivo&doi=10.1016%2Fj.ccell.2015.02.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DChase%26aufirst%3DJ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZong%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DBixby%26aufirst%3DD.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DMaillard%26aufirst%3DI.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520Menin-MLL%2520interaction%2520blocks%2520progression%2520of%2520MLL%2520leukemia%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26issue%3D4%26spage%3D589%26epage%3D602%26doi%3D10.1016%2Fj.ccell.2015.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">High-affinity small-molecule inhibitors of the menin-mixed lineage leukemia (MLL) interaction closely mimic a natural protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1543</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1021/jm401868d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401868d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1543-1556&issue=4&author=S.+Heauthor=T.+J.+Senterauthor=J.+Pollockauthor=C.+Hanauthor=S.+K.+Upadhyayauthor=T.+Purohitauthor=R.+D.+Gogliottiauthor=C.+W.+Lindsleyauthor=T.+Cierpickiauthor=S.+R.+Staufferauthor=J.+Grembecka&title=High-affinity+small-molecule+inhibitors+of+the+menin-mixed+lineage+leukemia+%28MLL%29+interaction+closely+mimic+a+natural+protein-protein+interaction&doi=10.1021%2Fjm401868d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm401868d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401868d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSenter%26aufirst%3DT.%2BJ.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DS.%2BK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DHigh-affinity%2520small-molecule%2520inhibitors%2520of%2520the%2520menin-mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520interaction%2520closely%2520mimic%2520a%2520natural%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D4%26spage%3D1543%26epage%3D1556%26doi%3D10.1021%2Fjm401868d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Property focused structure-based optimization of small molecule inhibitors of the protein-protein interaction between menin and Mixed Lineage Leukemia (MLL)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=892-913&issue=3&author=D.+Borkinauthor=J.+Pollockauthor=K.+Kempinskaauthor=T.+Purohitauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=H.+Miaoauthor=S.+Shuklaauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Property+focused+structure-based+optimization+of+small+molecule+inhibitors+of+the+protein-protein+interaction+between+menin+and+Mixed+Lineage+Leukemia+%28MLL%29&doi=10.1021%2Facs.jmedchem.5b01305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01305%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DProperty%2520focused%2520structure-based%2520optimization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520protein-protein%2520interaction%2520between%2520menin%2520and%2520Mixed%2520Lineage%2520Leukemia%2520%2528MLL%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D3%26spage%3D892%26epage%3D913%26doi%3D10.1021%2Facs.jmedchem.5b01305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malik, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asangani, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieslik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prensner, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escara-Wilke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stender, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niknafs, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palanisamy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunju, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yocum, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellacheruvu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nesvizhskii, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanasekaran, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the MLL complex in castration-resistant prostate cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">344</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/nm.3830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnm.3830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=25822367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=344-352&issue=4&author=R.+Malikauthor=A.+P.+Khanauthor=I.+A.+Asanganiauthor=M.+Cieslikauthor=J.+R.+Prensnerauthor=X.+Wangauthor=M.+K.+Iyerauthor=X.+Jiangauthor=D.+Borkinauthor=J.+Escara-Wilkeauthor=R.+Stenderauthor=Y.+M.+Wuauthor=Y.+S.+Niknafsauthor=X.+Jingauthor=Y.+Qiaoauthor=N.+Palanisamyauthor=L.+P.+Kunjuauthor=P.+M.+Krishnamurthyauthor=A.+K.+Yocumauthor=D.+Mellacheruvuauthor=A.+I.+Nesvizhskiiauthor=X.+Caoauthor=S.+M.+Dhanasekaranauthor=F.+Y.+Fengauthor=J.+Grembeckaauthor=T.+Cierpickiauthor=A.+M.+Chinnaiyan&title=Targeting+the+MLL+complex+in+castration-resistant+prostate+cancer&doi=10.1038%2Fnm.3830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the MLL complex in castration-resistant prostate cancer</span></div><div class="casAuthors">Malik, Rohit; Khan, Amjad P.; Asangani, Irfan A.; Cieslik, Marcin; Prensner, John R.; Wang, Xiaoju; Iyer, Matthew K.; Jiang, Xia; Borkin, Dmitry; Escara-Wilke, June; Stender, Rachell; Wu, Yi-Mi; Niknafs, Yashar S.; Jing, Xiaojun; Qiao, Yuanyuan; Palanisamy, Nallasivam; Kunju, Lakshmi P.; Krishnamurthy, Pranathi M.; Yocum, Anastasia K.; Mellacheruvu, Dattatreya; Nesvizhskii, Alexey I.; Cao, Xuhong; Dhanasekaran, Saravana M.; Feng, Felix Y.; Grembecka, Jolanta; Cierpicki, Tomasz; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">344-352</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Resistance to androgen deprivation therapies and increased androgen receptor (AR) activity are major drivers of castration-resistant prostate cancer (CRPC).  Although prior work has focused on targeting AR directly, co-activators of AR signaling, which may represent new therapeutic targets, are relatively underexplored.  Here we demonstrate that the mixed-lineage leukemia protein (MLL) complex, a well-known driver of MLL fusion-pos. leukemia, acts as a co-activator of AR signaling.  AR directly interacts with the MLL complex via the menin-MLL subunit.  Menin expression is higher in CRPC than in both hormone-naive prostate cancer and benign prostate tissue, and high menin expression correlates with poor overall survival of individuals diagnosed with prostate cancer.  Treatment with a small-mol. inhibitor of menin-MLL interaction blocks AR signaling and inhibits the growth of castration-resistant tumors in vivo in mice.  Taken together, this work identifies the MLL complex as a crucial co-activator of AR and a potential therapeutic target in advanced prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmvjojfsVQ97Vg90H21EOLACvtfcHk0liQaUcvhxNoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D&md5=31d39514cc6f6cf723146a70676f4021</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnm.3830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3830%26sid%3Dliteratum%253Aachs%26aulast%3DMalik%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DA.%2BP.%26aulast%3DAsangani%26aufirst%3DI.%2BA.%26aulast%3DCieslik%26aufirst%3DM.%26aulast%3DPrensner%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DIyer%26aufirst%3DM.%2BK.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DEscara-Wilke%26aufirst%3DJ.%26aulast%3DStender%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DNiknafs%26aufirst%3DY.%2BS.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DPalanisamy%26aufirst%3DN.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DKrishnamurthy%26aufirst%3DP.%2BM.%26aulast%3DYocum%26aufirst%3DA.%2BK.%26aulast%3DMellacheruvu%26aufirst%3DD.%26aulast%3DNesvizhskii%26aufirst%3DA.%2BI.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DTargeting%2520the%2520MLL%2520complex%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26issue%3D4%26spage%3D344%26epage%3D352%26doi%3D10.1038%2Fnm.3830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Svoboda, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Noord, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krook, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, E. R.</span></span> <span> </span><span class="NLM_article-title">Tumorigenicity of Ewing sarcoma is critically dependent on the trithorax proteins MLL1 and menin</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.18632%2Foncotarget.13444" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=458-471&issue=1&author=L.+K.+Svobodaauthor=N.+Baileyauthor=R.+A.+Van+Noordauthor=M.+A.+Krookauthor=A.+Harrisauthor=C.+Cramerauthor=B.+Jasmanauthor=R.+M.+Patelauthor=D.+Thomasauthor=D.+Borkinauthor=T.+Cierpickiauthor=J.+Grembeckaauthor=E.+R.+Lawlor&title=Tumorigenicity+of+Ewing+sarcoma+is+critically+dependent+on+the+trithorax+proteins+MLL1+and+menin&doi=10.18632%2Foncotarget.13444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13444%26sid%3Dliteratum%253Aachs%26aulast%3DSvoboda%26aufirst%3DL.%2BK.%26aulast%3DBailey%26aufirst%3DN.%26aulast%3DVan%2BNoord%26aufirst%3DR.%2BA.%26aulast%3DKrook%26aufirst%3DM.%2BA.%26aulast%3DHarris%26aufirst%3DA.%26aulast%3DCramer%26aufirst%3DC.%26aulast%3DJasman%26aufirst%3DB.%26aulast%3DPatel%26aufirst%3DR.%2BM.%26aulast%3DThomas%26aufirst%3DD.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DLawlor%26aufirst%3DE.%2BR.%26atitle%3DTumorigenicity%2520of%2520Ewing%2520sarcoma%2520is%2520critically%2520dependent%2520on%2520the%2520trithorax%2520proteins%2520MLL1%2520and%2520menin%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D458%26epage%3D471%26doi%3D10.18632%2Foncotarget.13444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G. H.</span></span> <span> </span><span class="NLM_article-title">Menin promotes hepatocellular carcinogenesis and epigenetically up-regulates Yap1 transcription</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">17480</span>– <span class="NLM_lpage">17485</span>, <span class="refDoi"> DOI: 10.1073/pnas.1312022110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1073%2Fpnas.1312022110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=17480-17485&issue=43&author=B.+Xuauthor=S.+H.+Liauthor=R.+Zhengauthor=S.+B.+Gaoauthor=L.+H.+Dingauthor=Z.+Y.+Yinauthor=X.+Linauthor=Z.+J.+Fengauthor=S.+Zhangauthor=X.+M.+Wangauthor=G.+H.+Jin&title=Menin+promotes+hepatocellular+carcinogenesis+and+epigenetically+up-regulates+Yap1+transcription&doi=10.1073%2Fpnas.1312022110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1312022110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1312022110%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DS.%2BH.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DS.%2BB.%26aulast%3DDing%26aufirst%3DL.%2BH.%26aulast%3DYin%26aufirst%3DZ.%2BY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%2BJ.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BM.%26aulast%3DJin%26aufirst%3DG.%2BH.%26atitle%3DMenin%2520promotes%2520hepatocellular%2520carcinogenesis%2520and%2520epigenetically%2520up-regulates%2520Yap1%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D43%26spage%3D17480%26epage%3D17485%26doi%3D10.1073%2Fpnas.1312022110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klossowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the menin–MLL interaction leads to transcriptional repression of PEG10 and blocks hepatocellular carcinoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-17-0580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1158%2F1535-7163.MCT-17-0580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=29142068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Giuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=26-38&issue=1&author=K.+Kempinskaauthor=B.+Malikauthor=D.+Borkinauthor=S.+Klossowskiauthor=S.+Shuklaauthor=H.+Miaoauthor=J.+Wangauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+menin%E2%80%93MLL+interaction+leads+to+transcriptional+repression+of+PEG10+and+blocks+hepatocellular+carcinoma&doi=10.1158%2F1535-7163.MCT-17-0580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of the Menin-MLL Interaction Leads to Transcriptional Repression of PEG10 and Blocks Hepatocellular Carcinoma</span></div><div class="casAuthors">Kempinska, Katarzyna; Malik, Bhavna; Borkin, Dmitry; Klossowski, Szymon; Shukla, Shirish; Miao, Hongzhi; Wang, Jingya; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26-38</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) accounts for approx. 85% of malignant liver tumors and results in 600,000 deaths each year, emphasizing the need for new therapies.  Upregulation of menin was reported in HCC patients and high levels of menin correlate with poor patient prognosis.  The protein-protein interaction between menin and histone methyltransferase mixed lineage leukemia 1 (MLL1) plays an important role in the development of HCC, implying that pharmacol. inhibition of this interaction could lead to new therapeutic strategy for the HCC patients.  Here, we demonstrate that the menin-MLL inhibitor MI-503 shows antitumor activity in in vitro and in vivo models of HCC and reveals the potential mechanism of menin contribution to HCC.  Treatment with MI-503 selectively kills various HCC cell lines and this effect is significantly enhanced by a combination of MI-503 with sorafenib, the std.-of-care therapy for HCC.  Furthermore, MI-503 reduces sphere formation and cell migration in in vitro HCC models.  When applied in vivo, MI-503 gives a strong antitumor effect both as a single agent and in combination with sorafenib in mice xenograft models of HCC.  Mechanistically, treatment with MI-503 downregulates expression of several genes known to play a crit. role in proliferation and migration of HCC cells, including PEG10, and displaces the menin-MLL1 complex from the PEG10 promoter, resulting in reduced H3K4 methylation and transcriptional repression.  Overall, our studies reveal a mechanistic link between menin and genes involved in HCC and demonstrate that pharmacol. inhibition of the menin-MLL interaction might represent a promising therapeutic approach for HCC.  Mol Cancer Ther; 17(1); 26-38. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI6J4bAkPP2rVg90H21EOLACvtfcHk0liQaUcvhxNoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Giuw%253D%253D&md5=e4675b46b6306a72d5fe6148e610b224</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-17-0580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-17-0580%26sid%3Dliteratum%253Aachs%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DKlossowski%26aufirst%3DS.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520menin%25E2%2580%2593MLL%2520interaction%2520leads%2520to%2520transcriptional%2520repression%2520of%2520PEG10%2520and%2520blocks%2520hepatocellular%2520carcinoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26issue%3D1%26spage%3D26%26epage%3D38%26doi%3D10.1158%2F1535-7163.MCT-17-0580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Funato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabar, V.</span></span> <span> </span><span class="NLM_article-title">Use of human embryonic stem cells to model pediatric gliomas with H3.3K27M histone mutation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span> (<span class="NLM_issue">6216</span>),  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1533</span>, <span class="refDoi"> DOI: 10.1126/science.1253799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1126%2Fscience.1253799" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=1529-1533&issue=6216&author=K.+Funatoauthor=T.+Majorauthor=P.+W.+Lewisauthor=C.+D.+Allisauthor=V.+Tabar&title=Use+of+human+embryonic+stem+cells+to+model+pediatric+gliomas+with+H3.3K27M+histone+mutation&doi=10.1126%2Fscience.1253799"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscience.1253799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1253799%26sid%3Dliteratum%253Aachs%26aulast%3DFunato%26aufirst%3DK.%26aulast%3DMajor%26aufirst%3DT.%26aulast%3DLewis%26aufirst%3DP.%2BW.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DTabar%26aufirst%3DV.%26atitle%3DUse%2520of%2520human%2520embryonic%2520stem%2520cells%2520to%2520model%2520pediatric%2520gliomas%2520with%2520H3.3K27M%2520histone%2520mutation%26jtitle%3DScience%26date%3D2014%26volume%3D346%26issue%3D6216%26spage%3D1529%26epage%3D1533%26doi%3D10.1126%2Fscience.1253799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span> <span> </span><span class="NLM_article-title">Protein-protein interactions as targets for small molecule drug discovery</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>84</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1002/bip.20608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fbip.20608" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2006&pages=535-552&issue=6&author=D.+C.+Fry&title=Protein-protein+interactions+as+targets+for+small+molecule+drug+discovery&doi=10.1002%2Fbip.20608"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fbip.20608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20608%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BC.%26atitle%3DProtein-protein%2520interactions%2520as%2520targets%2520for%2520small%2520molecule%2520drug%2520discovery%26jtitle%3DBiopolymers%26date%3D2006%26volume%3D84%26issue%3D6%26spage%3D535%26epage%3D552%26doi%3D10.1002%2Fbip.20608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, P.</span></span> <span> </span><span class="NLM_article-title">Small-molecule protein-protein interaction inhibitors: therapeutic potential in light of molecular size, chemical space, and ligand binding efficiency considerations</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1002/iub.383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fiub.383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=20979208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=724-731&issue=10&author=P.+Buchwald&title=Small-molecule+protein-protein+interaction+inhibitors%3A+therapeutic+potential+in+light+of+molecular+size%2C+chemical+space%2C+and+ligand+binding+efficiency+considerations&doi=10.1002%2Fiub.383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule protein-protein interaction inhibitors: therapeutic potential in light of molecular size, chemical space, and ligand binding efficiency considerations</span></div><div class="casAuthors">Buchwald, Peter</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">724-731</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  As the ultimate function of proteins depends to a great extent on their binding partners, protein-protein interactions (PPIs) represent a treasure trove of possible new therapeutic targets.  Unfortunately, interfaces involved in PPIs are not well-suited for effective small mol. binding.  Nevertheless, successful examples of small-mol. PPI inhibitors (PPIIs) are beginning to accumulate, and the sheer no. of PPIs that form the human interactome implies that, despite the relative unsuitability of PPIs to serve as "druggable" targets, small-mol. PPIIs can still provide novel pharmacol. tools and new innovative drugs in at least some areas.  Here, after some illustrative examples, accumulating information on the binding efficiency, mol. size, and chem. space requirements will be briefly reviewed.  Therapeutic success can only be achieved if these considerations are incorporated into the search process and if careful medicinal chem. approaches are used to address the absorption, distribution, metab., and excretion requirements of larger mols. that are often needed for this target class due to the lower efficiency of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPrt8JCmQ-mLVg90H21EOLACvtfcHk0lhIZgpt_SQC4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM&md5=ae27f95b3ef88b20ad2973927eaf4c91</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fiub.383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.383%26sid%3Dliteratum%253Aachs%26aulast%3DBuchwald%26aufirst%3DP.%26atitle%3DSmall-molecule%2520protein-protein%2520interaction%2520inhibitors%253A%2520therapeutic%2520potential%2520in%2520light%2520of%2520molecular%2520size%252C%2520chemical%2520space%252C%2520and%2520ligand%2520binding%2520efficiency%2520considerations%26jtitle%3DIUBMB%2520Life%26date%3D2010%26volume%3D62%26issue%3D10%26spage%3D724%26epage%3D731%26doi%3D10.1002%2Fiub.383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azzarito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of alpha-helix-mediated protein-protein interactions using designed molecules</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">173</span>, <span class="refDoi"> DOI: 10.1038/nchem.1568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnchem.1568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=23422557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=161-173&issue=3&author=V.+Azzaritoauthor=K.+Longauthor=N.+S.+Murphyauthor=A.+J.+Wilson&title=Inhibition+of+alpha-helix-mediated+protein-protein+interactions+using+designed+molecules&doi=10.1038%2Fnchem.1568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of α-helix-mediated protein-protein interactions using designed molecules</span></div><div class="casAuthors">Azzarito, Valeria; Long, Kerya; Murphy, Natasha S.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-173</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inhibition of protein-protein interactions (PPIs) represents a significant challenge because it is unclear how they can be effectively and selectively targeted using small mols.  Achieving this goal is crit. given the defining role of these interactions in biol. processes.  A rational approach to inhibitor design based on the secondary structure at the interface is the focus of much research, and different classes of designed ligands have emerged, some of which effectively and selectively disrupt targeted PPIs.  This Review discusses the relevance of PPIs and, in particular, the importance of α-helix-mediated PPIs to chem. biol. and drug discovery with a focus on designing inhibitors, including constrained peptides, foldamers and proteomimetic-derived ligands.  In doing so, key challenges and major advances in developing generic approaches for the elaboration of PPI inhibitors are highlighted.  The challenges faced in developing such ligands as drug leads - and how criteria applied to these may differ from conventional small-mol. drugs - are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpsQAFaEkKq7Vg90H21EOLACvtfcHk0lhIZgpt_SQC4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D&md5=161f88aa1c745dee9a95b1fab340b9fe</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1568%26sid%3Dliteratum%253Aachs%26aulast%3DAzzarito%26aufirst%3DV.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DN.%2BS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DInhibition%2520of%2520alpha-helix-mediated%2520protein-protein%2520interactions%2520using%2520designed%2520molecules%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26issue%3D3%26spage%3D161%26epage%3D173%26doi%3D10.1038%2Fnchem.1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span> <span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span> (<span class="NLM_issue">7172</span>),  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&issue=7172&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lgQHqU00ngt4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26issue%3D7172%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The road less traveled: modulating signal transduction enzymes by inhibiting their protein-protein interactions</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2009.05.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.cbpa.2009.05.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=19553156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVWlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=284-290&issue=3&author=M.+R.+Arkinauthor=A.+Whitty&title=The+road+less+traveled%3A+modulating+signal+transduction+enzymes+by+inhibiting+their+protein-protein+interactions&doi=10.1016%2Fj.cbpa.2009.05.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The road less traveled: Modulating signal transduction enzymes by inhibiting their protein-protein interactions</span></div><div class="casAuthors">Arkin, Michelle R.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">284-290</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The biol. functions of intracellular signaling enzymes typically depend on multiple protein-protein interactions (PPI) with substrates, scaffolding proteins, and other cytoplasmic mols.  Blocking these interactions provides an alternative means to modulate signaling activity without fully ablating the catalytic activity of the target.  Several recent reports describe small-mol. antagonists that target PPI sites on signaling enzymes.  These findings suggest that such sites may often be druggable.  However, the hypothesis that targeting such sites might confer on the resulting inhibitors improved properties of efficacy and/or tolerability, while appealing, remains largely untested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbJ1DqXR5rDLVg90H21EOLACvtfcHk0lgQHqU00ngt4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVWlur8%253D&md5=93e46c2f0d7bd7b16454df7bdc00b792</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2009.05.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2009.05.125%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520road%2520less%2520traveled%253A%2520modulating%2520signal%2520transduction%2520enzymes%2520by%2520inhibiting%2520their%2520protein-protein%2520interactions%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D13%26issue%3D3%26spage%3D284%26epage%3D290%26doi%3D10.1016%2Fj.cbpa.2009.05.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Features of protein-protein interactions that translate into potent inhibitors: topology, surface area and affinity</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">e16</span>, <span class="refDoi"> DOI: 10.1017/erm.2012.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1017%2Ferm.2012.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=22831787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=e16&author=M.+C.+Smithauthor=J.+E.+Gestwicki&title=Features+of+protein-protein+interactions+that+translate+into+potent+inhibitors%3A+topology%2C+surface+area+and+affinity&doi=10.1017%2Ferm.2012.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Features of protein-protein interactions that translate into potent inhibitors: topology, surface area and affinity</span></div><div class="casAuthors">Smith, Matthew C.; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e16/1-e16/20</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) control the assembly of multi-protein complexes and, thus, these contacts have enormous potential as drug targets.  However, the field has produced a mix of both exciting success stories and frustrating challenges. Here, we review known examples and explore how the phys. features of a PPI, such as its affinity, hotspots, off-rates, buried surface area and topol., might influence the chances of success in finding inhibitors.  This anal. suggests that concise, tight binding PPIs are most amenable to inhibition.  However, it is also clear that emerging tech. methods are expanding the repertoire of 'druggable' protein contacts and increasing the odds against difficult targets.  In particular, natural product-like compd. libraries, high throughput screens specifically designed for PPIs and approaches that favor discovery of allosteric inhibitors appear to be attractive routes.  The first group of PPI inhibitors has entered clin. trials, further motivating the need to understand the challenges and opportunities in pursuing these types of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6DSZAfVfqDLVg90H21EOLACvtfcHk0lgQHqU00ngt4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjtL7L&md5=a24bcc721f4c6ebfc8fae029c79f6426</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1017%2Ferm.2012.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252Ferm.2012.10%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DM.%2BC.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DFeatures%2520of%2520protein-protein%2520interactions%2520that%2520translate%2520into%2520potent%2520inhibitors%253A%2520topology%252C%2520surface%2520area%2520and%2520affinity%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2012%26volume%3D14%26spage%3De16%26doi%3D10.1017%2Ferm.2012.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interactions in hematologic malignancies: still a challenge or a great opportunity for future therapies?</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>263</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1111/imr.12244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1111%2Fimr.12244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=25510283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFSnsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=2015&pages=279-301&issue=1&author=T.+Cierpickiauthor=J.+Grembecka&title=Targeting+protein-protein+interactions+in+hematologic+malignancies%3A+still+a+challenge+or+a+great+opportunity+for+future+therapies%3F&doi=10.1111%2Fimr.12244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions in hematologic malignancies: still a challenge or a great opportunity for future therapies?</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">279-301</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Summary : Over the past several years, there has been an increasing research effort focused on inhibition of protein-protein interactions (PPIs) to develop novel therapeutic approaches for cancer, including hematol. malignancies.  These efforts have led to development of small mol. inhibitors of PPIs, some of which already advanced to the stage of clin. trials while others are at different stages of preclin. optimization, emphasizing PPIs as an emerging and attractive class of drug targets.  Here, we review several examples of recently developed inhibitors of PPIs highly relevant to hematol. cancers.  We address the existing skepticism about feasibility of targeting PPIs and emphasize potential therapeutic benefit from blocking PPIs in hematol. malignancies.  We then use these examples to discuss the approaches for successful identification of PPI inhibitors and provide anal. of the protein-protein interfaces, with the goal to address 'druggability' of new PPIs relevant to hematol.  We discuss lessons learned to improve the success of targeting new PPIs and evaluate prospects and limits of the research in this field.  We conclude that not all PPIs are equally tractable for blocking by small mols., and detailed anal. of PPI interfaces is crit. for selection of those with the highest chance of success.  Together, our anal. uncovers patterns that should help to advance drug discovery in hematol. malignancies by successful targeting of new PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhINiOkTGpPrVg90H21EOLACvtfcHk0lg7evpcGOEgCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFSnsrzE&md5=eded19171161abe72de4aab023bbfc58</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fimr.12244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimr.12244%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DTargeting%2520protein-protein%2520interactions%2520in%2520hematologic%2520malignancies%253A%2520still%2520a%2520challenge%2520or%2520a%2520great%2520opportunity%2520for%2520future%2520therapies%253F%26jtitle%3DImmunol.%2520Rev.%26date%3D2015%26volume%3D263%26issue%3D1%26spage%3D279%26epage%3D301%26doi%3D10.1111%2Fimr.12244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeting apoptosis pathways for new cancer therapeutics</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-010713-141310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1146%2Fannurev-med-010713-141310" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2014&pages=139-155&author=L.+Baiauthor=S.+Wang&title=Targeting+apoptosis+pathways+for+new+cancer+therapeutics&doi=10.1146%2Fannurev-med-010713-141310"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-010713-141310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-010713-141310%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeting%2520apoptosis%2520pathways%2520for%2520new%2520cancer%2520therapeutics%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2014%26volume%3D65%26spage%3D139%26epage%3D155%26doi%3D10.1146%2Fannurev-med-010713-141310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nero, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holien, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wielens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span> <span> </span><span class="NLM_article-title">Oncogenic protein interfaces: small molecules, big challenges</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nrc3690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnrc3690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=24622521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=248-262&issue=4&author=T.+L.+Neroauthor=C.+J.+Mortonauthor=J.+K.+Holienauthor=J.+Wielensauthor=M.+W.+Parker&title=Oncogenic+protein+interfaces%3A+small+molecules%2C+big+challenges&doi=10.1038%2Fnrc3690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic protein interfaces: small molecules, big challenges</span></div><div class="casAuthors">Nero, Tracy L.; Morton, Craig J.; Holien, Jessica K.; Wielens, Jerome; Parker, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">248-262</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Historically, targeting protein-protein interactions with small mols. was not thought possible because the corresponding interfaces were considered mostly flat and featureless and therefore 'undruggable'.  Instead, such interactions were targeted with larger mols., such as peptides and antibodies.  However, the past decade has seen encouraging breakthroughs through the refinement of existing techniques and the development of new ones, together with the identification and exploitation of unexpected aspects of protein-protein interaction surfaces.  In this Review, we describe some of the latest techniques to discover modulators of protein-protein interactions and how current drug discovery approaches have been adapted to successfully target these interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlk_D7e8hVV7Vg90H21EOLACvtfcHk0lg7evpcGOEgCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D&md5=7aad54df0c10c6cdb9b1a7ae0f15bfe9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrc3690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3690%26sid%3Dliteratum%253Aachs%26aulast%3DNero%26aufirst%3DT.%2BL.%26aulast%3DMorton%26aufirst%3DC.%2BJ.%26aulast%3DHolien%26aufirst%3DJ.%2BK.%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DParker%26aufirst%3DM.%2BW.%26atitle%3DOncogenic%2520protein%2520interfaces%253A%2520small%2520molecules%252C%2520big%2520challenges%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26issue%3D4%26spage%3D248%26epage%3D262%26doi%3D10.1038%2Fnrc3690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of the mixed lineage leukemia-menin interaction: implications for targeting mixed lineage leukemias</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">52</span>),  <span class="NLM_fpage">40690</span>– <span class="NLM_lpage">40698</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.172783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1074%2Fjbc.M110.172783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=40690-40698&issue=52&author=J.+Grembeckaauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=T.+Cierpicki&title=Molecular+basis+of+the+mixed+lineage+leukemia-menin+interaction%3A+implications+for+targeting+mixed+lineage+leukemias&doi=10.1074%2Fjbc.M110.172783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.172783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.172783%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMolecular%2520basis%2520of%2520the%2520mixed%2520lineage%2520leukemia-menin%2520interaction%253A%2520implications%2520for%2520targeting%2520mixed%2520lineage%2520leukemias%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D52%26spage%3D40690%26epage%3D40698%26doi%3D10.1074%2Fjbc.M110.172783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of menin reveals binding site for mixed lineage leukemia (MLL) protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">31742</span>– <span class="NLM_lpage">31748</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.258186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1074%2Fjbc.M111.258186" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=31742-31748&issue=36&author=M.+J.+Muraiauthor=M.+Chruszczauthor=G.+Reddyauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Crystal+structure+of+menin+reveals+binding+site+for+mixed+lineage+leukemia+%28MLL%29+protein&doi=10.1074%2Fjbc.M111.258186"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.258186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.258186%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520menin%2520reveals%2520binding%2520site%2520for%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26issue%3D36%26spage%3D31742%26epage%3D31748%26doi%3D10.1074%2Fjbc.M111.258186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=12854996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&issue=1&author=X.+Huang&title=Fluorescence+polarization+competition+assay%3A+the+range+of+resolvable+inhibitor+potency+is+limited+by+the+affinity+of+the+fluorescent+ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span></div><div class="casAuthors">Huang, Xinyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">For the development of fluorescence polarization (FP) competition assays, there is a widespread belief that tight-binding fluorescent ligands should be avoided to identify inhibitors of low or intermediate potency in the screening of small-mol. compd. libraries.  It is demonstrated herein that this statement is a misconception; in fact, the higher the affinity of the fluorescent ligand, the wider the range of inhibitor potency that can be resolved.  An approx. est. for the low end of inhibitor Ki values that can be resolved is the Kd value of the fluorescent ligand.  Because FP competition assays are typically conducted under nonstoichiometric titrn. conditions, it is suggested that a fluorescent ligand of highest affinity that also has an adequate quantum yield to satisfy such conditions be selected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUYlJzQaNj7Vg90H21EOLACvtfcHk0lgLYPd8hiOjFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D&md5=7c706c96b6e534486f6780c93a0d441b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DFluorescence%2520polarization%2520competition%2520assay%253A%2520the%2520range%2520of%2520resolvable%2520inhibitor%2520potency%2520is%2520limited%2520by%2520the%2520affinity%2520of%2520the%2520fluorescent%2520ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26issue%3D1%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamagnan, G.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis of (+)-(S,S)-reboxetine via a new (S)-2-(hydroxymethyl)morpholine preparation</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">939</span>, <span class="refDoi"> DOI: 10.1021/ol050059g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol050059g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=937-939&issue=5&author=E.+Brennerauthor=R.+M.+Baldwinauthor=G.+Tamagnan&title=Asymmetric+synthesis+of+%28%2B%29-%28S%2CS%29-reboxetine+via+a+new+%28S%29-2-%28hydroxymethyl%29morpholine+preparation&doi=10.1021%2Fol050059g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol050059g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol050059g%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DE.%26aulast%3DBaldwin%26aufirst%3DR.%2BM.%26aulast%3DTamagnan%26aufirst%3DG.%26atitle%3DAsymmetric%2520synthesis%2520of%2520%2528%252B%2529-%2528S%252CS%2529-reboxetine%2520via%2520a%2520new%2520%2528S%2529-2-%2528hydroxymethyl%2529morpholine%2520preparation%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26issue%3D5%26spage%3D937%26epage%3D939%26doi%3D10.1021%2Fol050059g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, M.</span>; <span class="NLM_string-name">Seekamp, C. K.</span>; <span class="NLM_string-name">Zhang, X.</span>; <span class="NLM_string-name">Tarrant, J. G.</span>; <span class="NLM_string-name">Lee, C.-W.</span>; <span class="NLM_string-name">Kaiser, B.</span>; <span class="NLM_string-name">Zablocki, M. M.</span>; <span class="NLM_string-name">Chenard, B. L.</span>; <span class="NLM_string-name">Doller, D.</span>; <span class="NLM_string-name">Yamaguchi, Y.</span>; <span class="NLM_string-name">Wustrow, D. J.</span>; <span class="NLM_string-name">Li, G.</span></span> <span> </span><span class="NLM_article-title">Aminopiperidines and Realted Compounds</span>. Int. Patent Appl. <span class="NLM_patent">WO2008016811A2</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=M.+Ghosh&author=C.+K.+Seekamp&author=X.+Zhang&author=J.+G.+Tarrant&author=C.-W.+Lee&author=B.+Kaiser&author=M.+M.+Zablocki&author=B.+L.+Chenard&author=D.+Doller&author=Y.+Yamaguchi&author=D.+J.+Wustrow&author=G.+Li&title=Aminopiperidines+and+Realted+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DM.%26atitle%3DAminopiperidines%2520and%2520Realted%2520Compounds%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the first-in-class, highly potent irreversible inhibitor targeting the menin–MLL protein-protein interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&issue=6&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fernandez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+first-in-class%2C+highly+potent+irreversible+inhibitor+targeting+the+menin%E2%80%93MLL+protein-protein+interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520first-in-class%252C%2520highly%2520potent%2520irreversible%2520inhibitor%2520targeting%2520the%2520menin%25E2%2580%2593MLL%2520protein-protein%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26issue%3D6%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fairhead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howarth, M.</span></span> <span> </span><span class="NLM_article-title">Site-specific biotinylation of purified proteins using BirA</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1266</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2272-7_12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1007%2F978-1-4939-2272-7_12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=25560075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1Sitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1266&publication_year=2015&pages=171-184&author=M.+Fairheadauthor=M.+Howarth&title=Site-specific+biotinylation+of+purified+proteins+using+BirA&doi=10.1007%2F978-1-4939-2272-7_12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Specific Biotinylation of Purified Proteins Using BirA</span></div><div class="casAuthors">Fairhead, Michael; Howarth, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1266</span>
        (<span class="NLM_cas:issue">Site-Specific Protein Labeling</span>),
    <span class="NLM_cas:pages">171-184</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The binding between biotin and streptavidin or avidin is one of the strongest known non-covalent biol. interactions.  The (strept)avidin-biotin interaction has been widely used for decades in biol. research and biotechnol.  Therefore labeling of purified proteins by biotin is a powerful way to achieve protein capture, immobilization, and functionalization, as well as multimerizing or bridging mols.  Chem. biotinylation often generates heterogeneous products, which may have impaired function.  Enzymic biotinylation with E. coli biotin ligase (BirA) is highly specific in covalently attaching biotin to the 15 amino acid AviTag peptide, giving a homogeneous product with high yield.  AviTag can conveniently be added genetically at the N-terminus, C-terminus, or in exposed loops of a target protein.  We describe here procedures for AviTag insertion by inverse PCR, purifn. of BirA fused to glutathione-S-transferase (GST-BirA) from E. coli , BirA biotinylation of purified protein, and gel-shift anal. by SDS-PAGE to quantify the extent of biotinylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptd53yZ7pVCrVg90H21EOLACvtfcHk0lgLYPd8hiOjFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1Sitbg%253D&md5=06168aa71460dff05856cab89576e15d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2272-7_12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2272-7_12%26sid%3Dliteratum%253Aachs%26aulast%3DFairhead%26aufirst%3DM.%26aulast%3DHowarth%26aufirst%3DM.%26atitle%3DSite-specific%2520biotinylation%2520of%2520purified%2520proteins%2520using%2520BirA%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1266%26spage%3D171%26epage%3D184%26doi%3D10.1007%2F978-1-4939-2272-7_12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhpiapgCvMDWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&issue=3&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhpiapgCvMDWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26issue%3D3%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&issue=12&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhpiapgCvMDWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D12%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Collaborative
Computational Project, N.</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&issue=5&author=Collaborative%0AComputational+Project%2C+N.&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26issue%3D5%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liGLYiNsK7_pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaver-Fay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snoeyink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">Web Server</span>),  <span class="NLM_fpage">W375</span>– <span class="NLM_lpage">W383</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=17452350" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&issue=Web+Server&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids&doi=10.1093%2Fnar%2Fgkm216"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26issue%3DWeb%2520Server%26spage%3DW375%26epage%3DW383%26doi%3D10.1093%2Fnar%2Fgkm216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guranovic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J. D.</span></span> <span> </span><span class="NLM_article-title">Automated and accurate deposition of structures solved by X-ray diffraction to the Protein Data Bank</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1833</span>– <span class="NLM_lpage">1839</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444904019419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=15388930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnslOltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1833-1839&issue=10&author=H.+Yangauthor=V.+Guranovicauthor=S.+Duttaauthor=Z.+Fengauthor=H.+M.+Bermanauthor=J.+D.+Westbrook&title=Automated+and+accurate+deposition+of+structures+solved+by+X-ray+diffraction+to+the+Protein+Data+Bank&doi=10.1107%2FS0907444904019419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Automated and accurate deposition of structures solved by X-ray diffraction to the Protein Data Bank</span></div><div class="casAuthors">Yang, Huanwang; Guranovic, Vladimir; Dutta, Shuchismita; Feng, Zukang; Berman, Helen M.; Westbrook, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1833-1839</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The RCSB Protein Data Bank (PDB) has a no. of options for deposition of structural data and has developed software tools to facilitate the process.  In addn. to ADIT and the PDB Validation Suite, a new software application, pdb_ext., has been designed to promote automatic data deposition of structures solved by x-ray diffraction.  The pdb_ext. software can ext. information about data redn., phasing, mol. replacement, d. modification and refinement from the output files produced by many x-ray crystallog. applications.  The options, procedures and tools for accurate and automated PDB data deposition are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpASfaztZ8sFbVg90H21EOLACvtfcHk0liGLYiNsK7_pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnslOltbo%253D&md5=8df9343c464ef1804d31041009bc2f44</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019419%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGuranovic%26aufirst%3DV.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26atitle%3DAutomated%2520and%2520accurate%2520deposition%2520of%2520structures%2520solved%2520by%2520X-ray%2520diffraction%2520to%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D10%26spage%3D1833%26epage%3D1839%26doi%3D10.1107%2FS0907444904019419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitwala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basrur, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenitoba-Johnson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">The PAF complex synergizes with MLL fusion proteins at HOX loci to promote leukemogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.ccr.2010.04.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=609-621&issue=6&author=A.+G.+Munteanauthor=J.+Tanauthor=K.+Sitwalaauthor=Y.+Huangauthor=J.+Bronsteinauthor=J.+A.+Connellyauthor=V.+Basrurauthor=K.+S.+Elenitoba-Johnsonauthor=J.+L.+Hess&title=The+PAF+complex+synergizes+with+MLL+fusion+proteins+at+HOX+loci+to+promote+leukemogenesis&doi=10.1016%2Fj.ccr.2010.04.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DSitwala%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DBronstein%26aufirst%3DJ.%26aulast%3DConnelly%26aufirst%3DJ.%2BA.%26aulast%3DBasrur%26aufirst%3DV.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520PAF%2520complex%2520synergizes%2520with%2520MLL%2520fusion%2520proteins%2520at%2520HOX%2520loci%2520to%2520promote%2520leukemogenesis%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26issue%3D6%26spage%3D609%26epage%3D621%26doi%3D10.1016%2Fj.ccr.2010.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madigan, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maring, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rockway, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, D. J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of HCV NS5B polymerase: synthesis and structure-activity relationships of N-alkyl-4-hydroxyquinolon-3-yl-benzothiadiazine sulfamides</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3367</span>– <span class="NLM_lpage">3370</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.bmcl.2006.04.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=3367-3370&issue=13&author=A.+C.+Kruegerauthor=D.+L.+Madiganauthor=W.+W.+Jiangauthor=W.+M.+Katiauthor=D.+Liuauthor=Y.+Liuauthor=C.+J.+Maringauthor=S.+Masseauthor=K.+F.+McDanielauthor=T.+Middletonauthor=H.+Moauthor=A.+Mollaauthor=D.+Montgomeryauthor=J.+K.+Prattauthor=T.+W.+Rockwayauthor=R.+Zhangauthor=D.+J.+Kempf&title=Inhibitors+of+HCV+NS5B+polymerase%3A+synthesis+and+structure-activity+relationships+of+N-alkyl-4-hydroxyquinolon-3-yl-benzothiadiazine+sulfamides&doi=10.1016%2Fj.bmcl.2006.04.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DKrueger%26aufirst%3DA.%2BC.%26aulast%3DMadigan%26aufirst%3DD.%2BL.%26aulast%3DJiang%26aufirst%3DW.%2BW.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaring%26aufirst%3DC.%2BJ.%26aulast%3DMasse%26aufirst%3DS.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DMiddleton%26aufirst%3DT.%26aulast%3DMo%26aufirst%3DH.%26aulast%3DMolla%26aufirst%3DA.%26aulast%3DMontgomery%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DRockway%26aufirst%3DT.%2BW.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26atitle%3DInhibitors%2520of%2520HCV%2520NS5B%2520polymerase%253A%2520synthesis%2520and%2520structure-activity%2520relationships%2520of%2520N-alkyl-4-hydroxyquinolon-3-yl-benzothiadiazine%2520sulfamides%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26issue%3D13%26spage%3D3367%26epage%3D3370%26doi%3D10.1016%2Fj.bmcl.2006.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 26 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Meng Zhang, Angelo Aguilar, Shilin Xu, Liyue Huang, Krishnapriya Chinnaswamy, Taryn Sleger, Bo Wang, Stefan Gross, Brandon N. Nicolay, Sebastien Ronseaux, Kaitlin Harvey, Yu Wang, Donna McEachern, Paul D. Kirchhoff, Zhaomin Liu, Jeanne Stuckey, Adriana E. Tron, Tao Liu, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of M-1121 as an Orally Active Covalent Inhibitor of Menin-MLL Interaction Capable of Achieving Complete and Long-Lasting Tumor Regression. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10333-10349. <a href="https://doi.org/10.1021/acs.jmedchem.1c00789" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00789</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00789&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00789%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BM-1121%252Bas%252Ban%252BOrally%252BActive%252BCovalent%252BInhibitor%252Bof%252BMenin-MLL%252BInteraction%252BCapable%252Bof%252BAchieving%252BComplete%252Band%252BLong-Lasting%252BTumor%252BRegression%26aulast%3DZhang%26aufirst%3DMeng%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D30042021%26date%3D01072021%26volume%3D64%26issue%3D14%26spage%3D10333%26epage%3D10349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Harikesh Kumar, Gurudayal Prajapati, Atul Dubey, Ravi Sankar Ampapathi, <span class="NLM_string-name hlFld-ContribAuthor">Pintu Kumar Mandal</span>. </span><span class="cited-content_cbyCitation_article-title">Intramolecular 6-exo-dig Post-Ugi Cyclization of N-Substituted 2-Alkynamides: Direct Access to Functionalized Morpholinone Glycoconjugates. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (23)
                                     , 9258-9262. <a href="https://doi.org/10.1021/acs.orglett.0c03486" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c03486</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c03486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c03486%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DIntramolecular%252B6-exo-dig%252BPost-Ugi%252BCyclization%252Bof%252BN-Substituted%252B2-Alkynamides%25253A%252BDirect%252BAccess%252Bto%252BFunctionalized%252BMorpholinone%252BGlycoconjugates%26aulast%3DKumar%26aufirst%3DHarikesh%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19102020%26date%3D17112020%26volume%3D22%26issue%3D23%26spage%3D9258%26epage%3D9262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Angelo Aguilar, Ke Zheng, Tianfeng Xu, Shilin Xu, Liyue Huang, Ester Fernandez-Salas, Liu Liu, Denzil Bernard, Kaitlin P. Harvey, Caroline Foster, Donna McEachern, Jeanne Stuckey, Krishnapriya Chinnaswamy, James Delproposto, Jeff W. Kampf, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Discovery of M-89 as a Highly Potent Inhibitor of the Menin-Mixed Lineage Leukemia (Menin-MLL) Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (13)
                                     , 6015-6034. <a href="https://doi.org/10.1021/acs.jmedchem.9b00021" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00021%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDiscovery%252Bof%252BM-89%252Bas%252Ba%252BHighly%252BPotent%252BInhibitor%252Bof%252Bthe%252BMenin-Mixed%252BLineage%252BLeukemia%252B%252528Menin-MLL%252529%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DAguilar%26aufirst%3DAngelo%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04012019%26date%3D22062019%26volume%3D62%26issue%3D13%26spage%3D6015%26epage%3D6034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Structure, function and inhibition of critical protein–protein interactions involving mixed lineage leukemia 1 and its fusion oncoproteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01057-7" title="DOI URL">https://doi.org/10.1186/s13045-021-01057-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01057-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01057-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DStructure%25252C%252Bfunction%252Band%252Binhibition%252Bof%252Bcritical%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Binvolving%252Bmixed%252Blineage%252Bleukemia%252B1%252Band%252Bits%252Bfusion%252Boncoproteins%26aulast%3DLi%26aufirst%3DXin%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alvina I.  Khamidullina</span>, <span class="hlFld-ContribAuthor ">Ekaterina A.  Varlamova</span>, <span class="hlFld-ContribAuthor ">Nour Alhuda  Hammoud</span>, <span class="hlFld-ContribAuthor ">Margarita A.  Yastrebova</span>, <span class="hlFld-ContribAuthor ">Alexandra V.  Bruter</span>. </span><span class="cited-content_cbyCitation_article-title">Gene Transcription as a Therapeutic Target in Leukemia. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (14)
                                     , 7340. <a href="https://doi.org/10.3390/ijms22147340" title="DOI URL">https://doi.org/10.3390/ijms22147340</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22147340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22147340%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DGene%252BTranscription%252Bas%252Ba%252BTherapeutic%252BTarget%252Bin%252BLeukemia%26aulast%3DKhamidullina%26aufirst%3DAlvina%2BI.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D14%26spage%3D7340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew J.  Henley</span>, <span class="hlFld-ContribAuthor ">Angela N.  Koehler</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in targeting ‘undruggable’ transcription factors with small molecules. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>172 </em><a href="https://doi.org/10.1038/s41573-021-00199-0" title="DOI URL">https://doi.org/10.1038/s41573-021-00199-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-021-00199-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-021-00199-0%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DAdvances%252Bin%252Btargeting%252B%2525E2%252580%252598undruggable%2525E2%252580%252599%252Btranscription%252Bfactors%252Bwith%252Bsmall%252Bmolecules%26aulast%3DHenley%26aufirst%3DMatthew%2BJ.%26date%3D2021%26date%3D2021%26volume%3D172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ezgi  Ozyerli‐Goknar</span>, <span class="hlFld-ContribAuthor ">Sheikh  Nizamuddin</span>, <span class="hlFld-ContribAuthor ">H. T. Marc  Timmers</span>. </span><span class="cited-content_cbyCitation_article-title">A Box of Chemistry to Inhibit the MEN1 Tumor Suppressor Gene Promoting Leukemia. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (9)
                                     , 1391-1402. <a href="https://doi.org/10.1002/cmdc.202000972" title="DOI URL">https://doi.org/10.1002/cmdc.202000972</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000972%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DA%252BBox%252Bof%252BChemistry%252Bto%252BInhibit%252Bthe%252BMEN1%252BTumor%252BSuppressor%252BGene%252BPromoting%252BLeukemia%26aulast%3DOzyerli%25E2%2580%2590Goknar%26aufirst%3DEzgi%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D9%26spage%3D1391%26epage%3D1402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clara  Libbrecht</span>, <span class="hlFld-ContribAuthor ">Hongbo M.  Xie</span>, <span class="hlFld-ContribAuthor ">Molly C.  Kingsley</span>, <span class="hlFld-ContribAuthor ">Jessica N.  Haladyna</span>, <span class="hlFld-ContribAuthor ">Simone S.  Riedel</span>, <span class="hlFld-ContribAuthor ">Fatemeh  Alikarami</span>, <span class="hlFld-ContribAuthor ">Alexandra  Lenard</span>, <span class="hlFld-ContribAuthor ">Gerard M.  McGeehan</span>, <span class="hlFld-ContribAuthor ">Patricia  Ernst</span>, <span class="hlFld-ContribAuthor ">Kathrin M.  Bernt</span>. </span><span class="cited-content_cbyCitation_article-title">Menin is necessary for long term maintenance of meningioma-1 driven leukemia. </span><span class="cited-content_cbyCitation_journal-name">Leukemia</span><span> <strong>2021,</strong> <em>35 </em>
                                    (5)
                                     , 1405-1417. <a href="https://doi.org/10.1038/s41375-021-01146-z" title="DOI URL">https://doi.org/10.1038/s41375-021-01146-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41375-021-01146-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41375-021-01146-z%26sid%3Dliteratum%253Aachs%26jtitle%3DLeukemia%26atitle%3DMenin%252Bis%252Bnecessary%252Bfor%252Blong%252Bterm%252Bmaintenance%252Bof%252Bmeningioma-1%252Bdriven%252Bleukemia%26aulast%3DLibbrecht%26aufirst%3DClara%26date%3D2021%26date%3D2021%26volume%3D35%26issue%3D5%26spage%3D1405%26epage%3D1417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunhua  Wan</span>, <span class="hlFld-ContribAuthor ">Sylvia  Mahara</span>, <span class="hlFld-ContribAuthor ">Claire  Sun</span>, <span class="hlFld-ContribAuthor ">Anh  Doan</span>, <span class="hlFld-ContribAuthor ">Hui Kheng  Chua</span>, <span class="hlFld-ContribAuthor ">Dakang  Xu</span>, <span class="hlFld-ContribAuthor ">Jia  Bian</span>, <span class="hlFld-ContribAuthor ">Yue  Li</span>, <span class="hlFld-ContribAuthor ">Danxi  Zhu</span>, <span class="hlFld-ContribAuthor ">Dhanya  Sooraj</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>, <span class="hlFld-ContribAuthor ">Ron  Firestein</span>. </span><span class="cited-content_cbyCitation_article-title">Genome-scale CRISPR-Cas9 screen of Wnt/β-catenin signaling identifies therapeutic targets for colorectal cancer. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2021,</strong> <em>7 </em>
                                    (21)
                                     , eabf2567. <a href="https://doi.org/10.1126/sciadv.abf2567" title="DOI URL">https://doi.org/10.1126/sciadv.abf2567</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.abf2567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.abf2567%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DGenome-scale%252BCRISPR-Cas9%252Bscreen%252Bof%252BWnt%25252F%2525CE%2525B2-catenin%252Bsignaling%252Bidentifies%252Btherapeutic%252Btargets%252Bfor%252Bcolorectal%252Bcancer%26aulast%3DWan%26aufirst%3DChunhua%26date%3D2021%26date%3D2021%26volume%3D7%26issue%3D21%26spage%3Deabf2567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamakoti P.  Bhat</span>, <span class="hlFld-ContribAuthor ">H.  Ümit Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Or  Gozani</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics and beyond: targeting writers of protein lysine methylation to treat disease. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (4)
                                     , 265-286. <a href="https://doi.org/10.1038/s41573-020-00108-x" title="DOI URL">https://doi.org/10.1038/s41573-020-00108-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-020-00108-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-020-00108-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DEpigenetics%252Band%252Bbeyond%25253A%252Btargeting%252Bwriters%252Bof%252Bprotein%252Blysine%252Bmethylation%252Bto%252Btreat%252Bdisease%26aulast%3DBhat%26aufirst%3DKamakoti%2BP.%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D4%26spage%3D265%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria Luisa  Brandi</span>, <span class="hlFld-ContribAuthor ">Sunita K  Agarwal</span>, <span class="hlFld-ContribAuthor ">Nancy D  Perrier</span>, <span class="hlFld-ContribAuthor ">Kate E  Lines</span>, <span class="hlFld-ContribAuthor ">Gerlof D  Valk</span>, <span class="hlFld-ContribAuthor ">Rajesh V  Thakker</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple Endocrine Neoplasia Type 1: Latest Insights. </span><span class="cited-content_cbyCitation_journal-name">Endocrine Reviews</span><span> <strong>2021,</strong> <em>42 </em>
                                    (2)
                                     , 133-170. <a href="https://doi.org/10.1210/endrev/bnaa031" title="DOI URL">https://doi.org/10.1210/endrev/bnaa031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1210/endrev/bnaa031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1210%2Fendrev%2Fbnaa031%26sid%3Dliteratum%253Aachs%26jtitle%3DEndocrine%2520Reviews%26atitle%3DMultiple%252BEndocrine%252BNeoplasia%252BType%252B1%25253A%252BLatest%252BInsights%26aulast%3DBrandi%26aufirst%3DMaria%2BLuisa%26date%3D2021%26date%3D2020%26volume%3D42%26issue%3D2%26spage%3D133%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongzhi  Miao</span>, <span class="hlFld-ContribAuthor ">EunGi  Kim</span>, <span class="hlFld-ContribAuthor ">Dong  Chen</span>, <span class="hlFld-ContribAuthor ">Trupta  Purohit</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kempinska</span>, <span class="hlFld-ContribAuthor ">James  Ropa</span>, <span class="hlFld-ContribAuthor ">Szymon  Klossowski</span>, <span class="hlFld-ContribAuthor ">Winifred  Trotman</span>, <span class="hlFld-ContribAuthor ">Gwenn  Danet-Desnoyers</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>. </span><span class="cited-content_cbyCitation_article-title">Combinatorial treatment with menin and FLT3 inhibitors induces complete remission in AML models with activating FLT3 mutations. </span><span class="cited-content_cbyCitation_journal-name">Blood</span><span> <strong>2020,</strong> <em>136 </em>
                                    (25)
                                     , 2958-2963. <a href="https://doi.org/10.1182/blood.2020006575" title="DOI URL">https://doi.org/10.1182/blood.2020006575</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1182/blood.2020006575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1182%2Fblood.2020006575%26sid%3Dliteratum%253Aachs%26jtitle%3DBlood%26atitle%3DCombinatorial%252Btreatment%252Bwith%252Bmenin%252Band%252BFLT3%252Binhibitors%252Binduces%252Bcomplete%252Bremission%252Bin%252BAML%252Bmodels%252Bwith%252Bactivating%252BFLT3%252Bmutations%26aulast%3DMiao%26aufirst%3DHongzhi%26date%3D2020%26volume%3D136%26issue%3D25%26spage%3D2958%26epage%3D2963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Margarita M.  Dzama</span>, <span class="hlFld-ContribAuthor ">Marlene  Steiner</span>, <span class="hlFld-ContribAuthor ">Johanna  Rausch</span>, <span class="hlFld-ContribAuthor ">Daniel  Sasca</span>, <span class="hlFld-ContribAuthor ">Jonas  Schönfeld</span>, <span class="hlFld-ContribAuthor ">Kerstin  Kunz</span>, <span class="hlFld-ContribAuthor ">Martha C.  Taubert</span>, <span class="hlFld-ContribAuthor ">Gerard M.  McGeehan</span>, <span class="hlFld-ContribAuthor ">Chun-Wei  Chen</span>, <span class="hlFld-ContribAuthor ">Annalisa  Mupo</span>, <span class="hlFld-ContribAuthor ">Patricia  Hähnel</span>, <span class="hlFld-ContribAuthor ">Matthias  Theobald</span>, <span class="hlFld-ContribAuthor ">Thomas  Kindler</span>, <span class="hlFld-ContribAuthor ">Richard P.  Koche</span>, <span class="hlFld-ContribAuthor ">George S.  Vassiliou</span>, <span class="hlFld-ContribAuthor ">Scott A.  Armstrong</span>, <span class="hlFld-ContribAuthor ">Michael W. M.  Kühn</span>. </span><span class="cited-content_cbyCitation_article-title">Synergistic targeting of
              FLT3
              mutations in AML via combined menin-MLL and FLT3 inhibition. </span><span class="cited-content_cbyCitation_journal-name">Blood</span><span> <strong>2020,</strong> <em>136 </em>
                                    (21)
                                     , 2442-2456. <a href="https://doi.org/10.1182/blood.2020005037" title="DOI URL">https://doi.org/10.1182/blood.2020005037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1182/blood.2020005037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1182%2Fblood.2020005037%26sid%3Dliteratum%253Aachs%26jtitle%3DBlood%26atitle%3DSynergistic%252Btargeting%252Bof%252BFLT3%252Bmutations%252Bin%252BAML%252Bvia%252Bcombined%252Bmenin-MLL%252Band%252BFLT3%252Binhibition%26aulast%3DDzama%26aufirst%3DMargarita%2BM.%26date%3D2020%26volume%3D136%26issue%3D21%26spage%3D2442%26epage%3D2456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nok-Hei Mickey  Wong</span>, <span class="hlFld-ContribAuthor ">Chi Wai Eric  So</span>. </span><span class="cited-content_cbyCitation_article-title">Novel therapeutic strategies for MLL-rearranged leukemias. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span> <strong>2020,</strong> <em>1863 </em>
                                    (9)
                                     , 194584. <a href="https://doi.org/10.1016/j.bbagrm.2020.194584" title="DOI URL">https://doi.org/10.1016/j.bbagrm.2020.194584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagrm.2020.194584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagrm.2020.194584%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Gene%2520Regulatory%2520Mechanisms%26atitle%3DNovel%252Btherapeutic%252Bstrategies%252Bfor%252BMLL-rearranged%252Bleukemias%26aulast%3DWong%26aufirst%3DNok-Hei%2BMickey%26date%3D2020%26volume%3D1863%26issue%3D9%26spage%3D194584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Szymon  Klossowski</span>, <span class="hlFld-ContribAuthor ">Hongzhi  Miao</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kempinska</span>, <span class="hlFld-ContribAuthor ">Tao  Wu</span>, <span class="hlFld-ContribAuthor ">Trupta  Purohit</span>, <span class="hlFld-ContribAuthor ">EunGi  Kim</span>, <span class="hlFld-ContribAuthor ">Brian M.  Linhares</span>, <span class="hlFld-ContribAuthor ">Dong  Chen</span>, <span class="hlFld-ContribAuthor ">Gloria  Jih</span>, <span class="hlFld-ContribAuthor ">Eric  Perkey</span>, <span class="hlFld-ContribAuthor ">Huang  Huang</span>, <span class="hlFld-ContribAuthor ">Miao  He</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Yi  Wang</span>, <span class="hlFld-ContribAuthor ">Ke  Yu</span>, <span class="hlFld-ContribAuthor ">Stanley Chun-Wei  Lee</span>, <span class="hlFld-ContribAuthor ">Gwenn  Danet-Desnoyers</span>, <span class="hlFld-ContribAuthor ">Winifred  Trotman</span>, <span class="hlFld-ContribAuthor ">Malathi  Kandarpa</span>, <span class="hlFld-ContribAuthor ">Anitria  Cotton</span>, <span class="hlFld-ContribAuthor ">Omar  Abdel-Wahab</span>, <span class="hlFld-ContribAuthor ">Hongwei  Lei</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, <span class="hlFld-ContribAuthor ">Monica  Guzman</span>, <span class="hlFld-ContribAuthor ">Luke  Peterson</span>, <span class="hlFld-ContribAuthor ">Tanja  Gruber</span>, <span class="hlFld-ContribAuthor ">Sarah  Choi</span>, <span class="hlFld-ContribAuthor ">Duxin  Sun</span>, <span class="hlFld-ContribAuthor ">Pingda  Ren</span>, <span class="hlFld-ContribAuthor ">Lian-Sheng  Li</span>, <span class="hlFld-ContribAuthor ">Yi  Liu</span>, <span class="hlFld-ContribAuthor ">Francis  Burrows</span>, <span class="hlFld-ContribAuthor ">Ivan  Maillard</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>. </span><span class="cited-content_cbyCitation_article-title">Menin inhibitor MI-3454 induces remission in MLL1-rearranged and NPM1-mutated models of leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Clinical Investigation</span><span> <strong>2020,</strong> <em>130 </em>
                                    (2)
                                     , 981-997. <a href="https://doi.org/10.1172/JCI129126" title="DOI URL">https://doi.org/10.1172/JCI129126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1172/JCI129126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1172%2FJCI129126%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Clinical%2520Investigation%26atitle%3DMenin%252Binhibitor%252BMI-3454%252Binduces%252Bremission%252Bin%252BMLL1-rearranged%252Band%252BNPM1-mutated%252Bmodels%252Bof%252Bleukemia%26aulast%3DKlossowski%26aufirst%3DSzymon%26date%3D2020%26date%3D2020%26volume%3D130%26issue%3D2%26spage%3D981%26epage%3D997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krzysztof  Brzezinka</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Nevedomskaya</span>, <span class="hlFld-ContribAuthor ">Ralf  Lesche</span>, <span class="hlFld-ContribAuthor ">Andrea  Haegebarth</span>, <span class="hlFld-ContribAuthor ">Antonius  ter Laak</span>, <span class="hlFld-ContribAuthor ">Amaury E.  Fernández-Montalván</span>, <span class="hlFld-ContribAuthor ">Uwe  Eberspaecher</span>, <span class="hlFld-ContribAuthor ">Nicolas D.  Werbeck</span>, <span class="hlFld-ContribAuthor ">Ursula  Moenning</span>, <span class="hlFld-ContribAuthor ">Stephan  Siegel</span>, <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>, <span class="hlFld-ContribAuthor ">Ashley L.  Eheim</span>, <span class="hlFld-ContribAuthor ">Carlo  Stresemann</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of the Menin-MLL Interaction as Therapeutic Cancer Target. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     , 201. <a href="https://doi.org/10.3390/cancers12010201" title="DOI URL">https://doi.org/10.3390/cancers12010201</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12010201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12010201%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DCharacterization%252Bof%252Bthe%252BMenin-MLL%252BInteraction%252Bas%252BTherapeutic%252BCancer%252BTarget%26aulast%3DBrzezinka%26aufirst%3DKrzysztof%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1%26spage%3D201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Hongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic machinery: Emerging novel allosteric inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2019,</strong> <em>204 </em>, 107406. <a href="https://doi.org/10.1016/j.pharmthera.2019.107406" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2019.107406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2019.107406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2019.107406%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bepigenetic%252Bmachinery%25253A%252BEmerging%252Bnovel%252Ballosteric%252Binhibitors%26aulast%3DYe%26aufirst%3DFei%26date%3D2019%26volume%3D204%26spage%3D107406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Connor M.  Blair</span>, <span class="hlFld-ContribAuthor ">Nicola M.  Walsh</span>, <span class="hlFld-ContribAuthor ">Bruce H.  Littman</span>, <span class="hlFld-ContribAuthor ">Frank W.  Marcoux</span>, <span class="hlFld-ContribAuthor ">George S.  Baillie</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting B-Raf inhibitor resistant melanoma with novel cell penetrating peptide disrupters of PDE8A – C-Raf. </span><span class="cited-content_cbyCitation_journal-name">BMC Cancer</span><span> <strong>2019,</strong> <em>19 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12885-019-5489-4" title="DOI URL">https://doi.org/10.1186/s12885-019-5489-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12885-019-5489-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12885-019-5489-4%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Cancer%26atitle%3DTargeting%252BB-Raf%252Binhibitor%252Bresistant%252Bmelanoma%252Bwith%252Bnovel%252Bcell%252Bpenetrating%252Bpeptide%252Bdisrupters%252Bof%252BPDE8A%252B%2525E2%252580%252593%252BC-Raf%26aulast%3DBlair%26aufirst%3DConnor%2BM.%26date%3D2019%26date%3D2019%26volume%3D19%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krzysztof  Brzezinka</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Nevedomskaya</span>, <span class="hlFld-ContribAuthor ">Ralf  Lesche</span>, <span class="hlFld-ContribAuthor ">Michael  Steckel</span>, <span class="hlFld-ContribAuthor ">Ashley L.  Eheim</span>, <span class="hlFld-ContribAuthor ">Andrea  Haegebarth</span>, <span class="hlFld-ContribAuthor ">Carlo  Stresemann</span>. </span><span class="cited-content_cbyCitation_article-title">Functional diversity of inhibitors tackling the differentiation blockage of MLL-rearranged leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2019,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-019-0749-y" title="DOI URL">https://doi.org/10.1186/s13045-019-0749-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-019-0749-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-019-0749-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DFunctional%252Bdiversity%252Bof%252Binhibitors%252Btackling%252Bthe%252Bdifferentiation%252Bblockage%252Bof%252BMLL-rearranged%252Bleukemia%26aulast%3DBrzezinka%26aufirst%3DKrzysztof%26date%3D2019%26date%3D2019%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Ye</span>, <span class="hlFld-ContribAuthor ">Jie  Zha</span>, <span class="hlFld-ContribAuthor ">Yuanfei  Shi</span>, <span class="hlFld-ContribAuthor ">Yin  Li</span>, <span class="hlFld-ContribAuthor ">Delin  Yuan</span>, <span class="hlFld-ContribAuthor ">Qinwei  Chen</span>, <span class="hlFld-ContribAuthor ">Fusheng  Lin</span>, <span class="hlFld-ContribAuthor ">Zhihong  Fang</span>, <span class="hlFld-ContribAuthor ">Yong  Yu</span>, <span class="hlFld-ContribAuthor ">Yun  Dai</span>, <span class="hlFld-ContribAuthor ">Bing  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Co-inhibition of HDAC and MLL-menin interaction targets MLL-rearranged acute myeloid leukemia cells via disruption of DNA damage checkpoint and DNA repair. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-019-0723-0" title="DOI URL">https://doi.org/10.1186/s13148-019-0723-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-019-0723-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-019-0723-0%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DCo-inhibition%252Bof%252BHDAC%252Band%252BMLL-menin%252Binteraction%252Btargets%252BMLL-rearranged%252Bacute%252Bmyeloid%252Bleukemia%252Bcells%252Bvia%252Bdisruption%252Bof%252BDNA%252Bdamage%252Bcheckpoint%252Band%252BDNA%252Brepair%26aulast%3DYe%26aufirst%3DJing%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John H.  Bushweller</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting transcription factors in cancer — from undruggable to reality. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2019,</strong> <em>19 </em>
                                    (11)
                                     , 611-624. <a href="https://doi.org/10.1038/s41568-019-0196-7" title="DOI URL">https://doi.org/10.1038/s41568-019-0196-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-019-0196-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-019-0196-7%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DTargeting%252Btranscription%252Bfactors%252Bin%252Bcancer%252B%2525E2%252580%252594%252Bfrom%252Bundruggable%252Bto%252Breality%26aulast%3DBushweller%26aufirst%3DJohn%2BH.%26date%3D2019%26date%3D2019%26volume%3D19%26issue%3D11%26spage%3D611%26epage%3D624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas  Mercher</span>, <span class="hlFld-ContribAuthor ">Juerg  Schwaller</span>. </span><span class="cited-content_cbyCitation_article-title">Pediatric Acute Myeloid Leukemia (AML): From Genes to Models Toward Targeted Therapeutic Intervention. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pediatrics</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fped.2019.00401" title="DOI URL">https://doi.org/10.3389/fped.2019.00401</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fped.2019.00401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffped.2019.00401%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pediatrics%26atitle%3DPediatric%252BAcute%252BMyeloid%252BLeukemia%252B%252528AML%252529%25253A%252BFrom%252BGenes%252Bto%252BModels%252BToward%252BTargeted%252BTherapeutic%252BIntervention%26aulast%3DMercher%26aufirst%3DThomas%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michaela  Kuhlen</span>, <span class="hlFld-ContribAuthor ">Jan-Henning  Klusmann</span>, <span class="hlFld-ContribAuthor ">Jessica I.  Hoell</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Approaches to Treating Pediatric Leukemias. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pediatrics</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fped.2019.00368" title="DOI URL">https://doi.org/10.3389/fped.2019.00368</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fped.2019.00368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffped.2019.00368%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pediatrics%26atitle%3DMolecular%252BApproaches%252Bto%252BTreating%252BPediatric%252BLeukemias%26aulast%3DKuhlen%26aufirst%3DMichaela%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony K. N.  Chan</span>, <span class="hlFld-ContribAuthor ">Chun-Wei  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Rewiring the Epigenetic Networks in MLL-Rearranged Leukemias: Epigenetic Dysregulation and Pharmacological Interventions. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fcell.2019.00081" title="DOI URL">https://doi.org/10.3389/fcell.2019.00081</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2019.00081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2019.00081%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DRewiring%252Bthe%252BEpigenetic%252BNetworks%252Bin%252BMLL-Rearranged%252BLeukemias%25253A%252BEpigenetic%252BDysregulation%252Band%252BPharmacological%252BInterventions%26aulast%3DChan%26aufirst%3DAnthony%2BK.%2BN.%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaowen  Wang</span>, <span class="hlFld-ContribAuthor ">Mingbo  Su</span>, <span class="hlFld-ContribAuthor ">You  Li</span>, <span class="hlFld-ContribAuthor ">Tongchao  Liu</span>, <span class="hlFld-ContribAuthor ">Yujie  Wang</span>, <span class="hlFld-ContribAuthor ">Yabing  Chen</span>, <span class="hlFld-ContribAuthor ">Le  Tang</span>, <span class="hlFld-ContribAuthor ">Yu-Peng  He</span>, <span class="hlFld-ContribAuthor ">Xiaoguang  Ding</span>, <span class="hlFld-ContribAuthor ">Fang  Yu</span>, <span class="hlFld-ContribAuthor ">Jingkang  Shen</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Yubo  Zhou</span>, <span class="hlFld-ContribAuthor ">Yue-Lei  Chen</span>, <span class="hlFld-ContribAuthor ">Bing  Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Tranylcypromine and 6-trifluoroethyl thienopyrimidine hybrid as LSD1 inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (6)
                                     , 844-847. <a href="https://doi.org/10.1016/j.bmcl.2019.01.017" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.01.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.01.017%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DTranylcypromine%252Band%252B6-trifluoroethyl%252Bthienopyrimidine%252Bhybrid%252Bas%252BLSD1%252Binhibitor%26aulast%3DWang%26aufirst%3DXiaowen%26date%3D2019%26volume%3D29%26issue%3D6%26spage%3D844%26epage%3D847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures and binding mode of the previous generation of menin–MLL inhibitors. (a) Chemical structures of the thienopyrimidine menin–MLL inhibitors <b>MI-463</b>, <b>MI-503</b>, and <b>MI-538</b> together with the inhibitory activity of these compounds measured in the fluorescence polarization (FP) competition assays with menin and the fluorescein-labeled MLL<sub>4–15</sub> or MLL<sub>4-43</sub> peptides. (b) Crystal structure of menin in complex with <b>MI-503</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X5Y">4X5Y</a>). Protein is shown in ribbon representation, and key menin residues involved in interactions with <b>MI-503</b> are shown as sticks. <b>MI-503</b> is shown in stick representation with colors corresponding to the atom type (green, carbons; dark blue, nitrogens; yellow, sulfur; light blue, fluorine). Dashed lines represent hydrogen bonds between <b>MI-503</b> and menin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure-based optimization results in potent menin–MLL inhibitors. (a) Crystal structure of the menin–compound <b>13</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXH">6BXH</a>). Residues on menin involved in the interactions with the piperazine ring and the terminal amide of <b>13</b> are shown in stick representation. Hydrogen bond with Arg330 is marked. (b) Crystal structure of the menin–compound <b>28</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXY">6BXY</a>). Residues on menin involved in the interactions with the morpholinone ring in <b>28</b> are shown in stick representation. Hydrogen bond with Glu366 is marked. (c) Superposition of the crystal structures of menin in complex with <b>MI-503</b>, <b>28</b>, and MLL peptide encompassing MBM1 and MBM2 motifs (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U88">3U88</a>). (d) Comparison of the binding mode of <b>28</b> (left) and <b>29</b> (right, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BY8">6BY8</a>) to menin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition of the menin–MLL interaction and cellular activity of compound <b>28</b>. (a) Titration curves from the FP competition assay for inhibition of the menin–MLL<sub>4-43</sub> by <b>28</b> and <b>29</b>. Data represent mean values of duplicate samples ± SD. (b–d) Titration curves from the MTT cell viability assay performed for <b>28</b> or <b>29</b> after 7 days of treatment of MLL-AF9 or Hoxa9/Meis1 (HM-2) transformed murine bone marrow cells treated with <b>28</b> (b); human MLL leukemia cell lines (MV4;11 and MOLM-13) and control cell lines (K562 and U937 without MLL translocation) treated with <b>28</b> (c); and human leukemia MLL cell lines treated with <b>29</b> (d). Cell growth inhibition (GI<sub>50</sub>) values are provided. Data represent mean values of quadruplicates ± SD. The experiments were performed two times. Values are normalized with respect to DMSO-treated samples. (e) qRT-PCR performed in MV4;11 cells after 6 days of treatment with <b>28</b>. Expression of <i>HOXA9</i>, <i>MEIS1</i>, <i>DLX2</i>, and <i>ITGAM</i> was normalized to 18S and referenced to DMSO-treated cells, mean ± SD, <i>n</i> = 2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>28</b> strongly inhibits leukemia progression in mice. (a,b) Bioluminescence level in MV4;11 leukemia mice measured at days 0, 7, 14, and 21 after initiating treatment with vehicle or <b>28</b>. <i>n</i> = 8 mice per group. (b) Representative images of MV4;11 mice treated with vehicle or <b>28</b> at day 21 of treatment. (c) Spleen weight and pictures of MV4;11 mice after 6 days of treatment with vehicle or <b>28</b> (<i>n</i> = 3 mice/group). (d) Level of hCD45+ cells in MV4;11 mice after 6 days of treatment with vehicle or <b>28</b> measured by flow cytometry (<i>n</i> = 3 mice/group). (e) Expression of <i>HOXA9</i>, <i>MEIS1</i>, and <i>ITGAM</i> in the bone marrow (BM) and spleen samples of the MV4;11 leukemia mice after 6 days of treatment with <b>28</b>. Expression was normalized to GAPDH and referenced to vehicle-treated mice, mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>3</b> and <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MsCl, TEA, DCM −67 °C to rt; (b) <b>30</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 90–110 °C; (c) <b>33</b>, Na(AcO)<sub>3</sub>BH; TEA, rt; (d) SnCl<sub>4</sub>, tartaric acid.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>30</b>, Cs<sub>2</sub>CO<sub>3</sub>, TBAI, DMF, 90 °C; (b) <b>33</b>, Na(AcO)<sub>3</sub>BH; TEA, rt.</p></p></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>6</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) chloroacetic acid chloride, MeOH, AcCN, TEA, 0 °C; (b) <i>t-</i>BuOK, amyl alcohol, rt (for <b>38a</b>); (c) LiBH<sub>4</sub>, THF, 0 °C and (d) <i>t</i>-BuOK, amyl alcohol, 0 °C (for <b>38b</b>); (e) MsCl, TEA, DCM, −67 °C to rt; (f) <b>30</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMA, 105 °C; (g) <b>33</b>, Na(AcO)<sub>3</sub>BH; AcOH, 50 °C.</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>8</b> and <b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) formaline, Na(AcO)<sub>3</sub>BH, AcCN, rt (for <b>8</b>); (b) <i>n-</i>propanal, Na(AcO)<sub>3</sub>BH; TEA, DCM, rt (for <b>9</b>).</p></p></figure><figure data-id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/medium/jm-2018-00071a_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0009.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>10</b>–<b>27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-11/acs.jmedchem.8b00071/20180608/images/large/jm-2018-00071a_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00071&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HCOOMe, reflux (for <b>10</b>, <b>24</b>); (b) R<sup>2</sup>COCl, DIPEA, DCM or AcCN, rt (for <b>11</b>, <b>12</b>, <b>17</b>, <b>25</b>); (c) R<sup>2</sup>COOH, TBTU, THF (for <b>13</b>, <b>14</b>, <b>16</b>); (d) (CF<sub>3</sub>CO)<sub>2</sub>O, DCM, 0 °C to rt (for <b>15</b>); (e) <i>rac-</i>BrCH(CH<sub>3</sub>)CONH<sub>2</sub>, DMF (for <b>18</b>); (f) R<sup>1</sup>SO<sub>2</sub>Cl, DIPEA, DCM, rt (for <b>19–21</b>, <b>23</b>, <b>26</b>); (g) CbzNHSO<sub>2</sub>Cl, DIPEA, DCM; and (h) Pd/C, MeOH, ammonium formate (for <b>22</b>, <b>27</b>).</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46452" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46452" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Challenges and opportunities in targeting the menin–MLL interaction</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.4155/fmc.13.214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.4155%2Ffmc.13.214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=24635524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=447-462&issue=4&author=T.+Cierpickiauthor=J.+Grembecka&title=Challenges+and+opportunities+in+targeting+the+menin%E2%80%93MLL+interaction&doi=10.4155%2Ffmc.13.214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and opportunities in targeting the menin-MLL interaction</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">447-462</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Menin is an essential co-factor of oncogenic MLL fusion proteins and the menin-MLL interaction is crit. for development of acute leukemia in vivo.  Targeting the menin-MLL interaction with small mols. represents an attractive strategy to develop new anticancer agents.  Recent developments, including detn. of menin crystal structure and development of potent small mol. and peptidomimetic inhibitors, demonstrate the feasibility of targeting the menin-MLL interaction.  On the other hand, biochem. and structural studies revealed that MLL binds to menin in a complex bivalent mode engaging two MLL motifs, and therefore inhibition of this protein-protein interaction represents a challenge.  This review summarizes the most recent achievements in targeting the menin-MLL interaction as well as discusses potential benefits of blocking menin in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovLrSLXU5DNbVg90H21EOLACvtfcHk0liFmc0sGaFomg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtLg%253D&md5=97777abe7b2fd8df3daf390d9b019721</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.214%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DChallenges%2520and%2520opportunities%2520in%2520targeting%2520the%2520menin%25E2%2580%2593MLL%2520interaction%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26issue%3D4%26spage%3D447%26epage%3D462%26doi%3D10.4155%2Ffmc.13.214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">The menin tumor suppressor protein is an essential oncogenic cofactor for MLL-associated leukemogenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>123</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.09.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.cell.2005.09.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=207-218&issue=2&author=A.+Yokoyamaauthor=T.+C.+Somervailleauthor=K.+S.+Smithauthor=O.+Rozenblatt-Rosenauthor=M.+Meyersonauthor=M.+L.+Cleary&title=The+menin+tumor+suppressor+protein+is+an+essential+oncogenic+cofactor+for+MLL-associated+leukemogenesis&doi=10.1016%2Fj.cell.2005.09.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.09.025%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26aulast%3DSmith%26aufirst%3DK.%2BS.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DThe%2520menin%2520tumor%2520suppressor%2520protein%2520is%2520an%2520essential%2520oncogenic%2520cofactor%2520for%2520MLL-associated%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D123%26issue%3D2%26spage%3D207%26epage%3D218%26doi%3D10.1016%2Fj.cell.2005.09.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marschalek, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of leukemogenesis by MLL fusion proteins</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>152</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2141.2010.08459.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1111%2Fj.1365-2141.2010.08459.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=21118195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2011&pages=141-154&issue=2&author=R.+Marschalek&title=Mechanisms+of+leukemogenesis+by+MLL+fusion+proteins&doi=10.1111%2Fj.1365-2141.2010.08459.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of leukemogenesis by MLL fusion proteins</span></div><div class="casAuthors">Marschalek, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-154</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Infant acute leukemia is characterised by specific genetic rearrangements and a rapid onset of disease shortly after birth.  The vast majority of these cases bear rearranged MLL alleles.  However, many facets of MLL-rearranged leukemia are largely unknown.  Basically, there exists a fundamental and evolutionary conserved relationship between the family of MLL/Trithorax proteins and the regulation of HOX gene clusters.  Therefore, direct MLL fusion proteins are per se able to deregulate HOX genes, except when reciprocal MLL fusion proteins come into play.  This reviews discusses (i) the current situation in MLL-rearranged leukemia, (ii) the mol. and genetic tools to functionally investigate the many different MLL fusions, (iii) the latency of disease development, (iv) a novel cancer mechanism that has been recently uncovered when different MLL fusion protein complexes were characterized, (v) mutated signalling pathways in MLL-rearranged leukemia and (vi) presents new ideas on how a given MLL fusion protein may modulate existing signalling pathways in leukemic cells.  The hypothesis is posed that the many different fusion partners of MLL are critically distinct entities for which specific inhibitors should be identified in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeSBrUEMt1JLVg90H21EOLACvtfcHk0liAbcZMWHR7fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVGjtro%253D&md5=f0330065cb216625a242a2508d5d1c93</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2141.2010.08459.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2141.2010.08459.x%26sid%3Dliteratum%253Aachs%26aulast%3DMarschalek%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520leukemogenesis%2520by%2520MLL%2520fusion%2520proteins%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2011%26volume%3D152%26issue%3D2%26spage%3D141%26epage%3D154%26doi%3D10.1111%2Fj.1365-2141.2010.08459.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimartino, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Mll rearrangements in haematological malignancies: lessons from clinical and biological studies</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1046/j.1365-2141.1999.01439.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1046%2Fj.1365-2141.1999.01439.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10468849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1999&pages=614-626&issue=3&author=J.+F.+Dimartinoauthor=M.+L.+Cleary&title=Mll+rearrangements+in+haematological+malignancies%3A+lessons+from+clinical+and+biological+studies&doi=10.1046%2Fj.1365-2141.1999.01439.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MLL rearrangements in haematological malignancies: Lessons from clinical and biological studies</span></div><div class="casAuthors">DiMartino, Jorge F.; Cleary, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">614-626</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 96 refs., covering the MLL family of 11q23 chromosomal rearrangements, assocn. between 11q23 rearrangements and leukemia phenotype, MLL rearrangements in the absence of leukemia, prognostic implications of 11q23 rearrangements, etiol. of MLL rearrangements, MLL rearrangements leading to leukemia with short latency, structure and probable function of the MLL protein, and possible mechanisms of leukemogenesis by MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOiR8FH8-xQbVg90H21EOLACvtfcHk0liAbcZMWHR7fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFSjt7g%253D&md5=845845da43dc70b59322261edb88bb6e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1046%2Fj.1365-2141.1999.01439.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1365-2141.1999.01439.x%26sid%3Dliteratum%253Aachs%26aulast%3DDimartino%26aufirst%3DJ.%2BF.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMll%2520rearrangements%2520in%2520haematological%2520malignancies%253A%2520lessons%2520from%2520clinical%2520and%2520biological%2520studies%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D1999%26volume%3D106%26issue%3D3%26spage%3D614%26epage%3D626%26doi%3D10.1046%2Fj.1365-2141.1999.01439.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span> <span> </span><span class="NLM_article-title">MLL: how complex does it get?</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>95</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1002/jcb.20430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fjcb.20430" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2005&pages=234-242&issue=2&author=R.+Popovicauthor=N.+J.+Zeleznik-Le&title=MLL%3A+how+complex+does+it+get%3F&doi=10.1002%2Fjcb.20430"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fjcb.20430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.20430%26sid%3Dliteratum%253Aachs%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26atitle%3DMLL%253A%2520how%2520complex%2520does%2520it%2520get%253F%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2005%26volume%3D95%26issue%3D2%26spage%3D234%26epage%3D242%26doi%3D10.1002%2Fjcb.20430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caslini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Osta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Interaction of MLL amino terminal sequences with menin is required for transformation</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">7275</span>– <span class="NLM_lpage">7283</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-2369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1158%2F0008-5472.CAN-06-2369" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=7275-7283&issue=15&author=C.+Casliniauthor=Z.+Yangauthor=M.+El-Ostaauthor=T.+A.+Milneauthor=R.+K.+Slanyauthor=J.+L.+Hess&title=Interaction+of+MLL+amino+terminal+sequences+with+menin+is+required+for+transformation&doi=10.1158%2F0008-5472.CAN-06-2369"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-2369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-2369%26sid%3Dliteratum%253Aachs%26aulast%3DCaslini%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DEl-Osta%26aufirst%3DM.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DSlany%26aufirst%3DR.%2BK.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DInteraction%2520of%2520MLL%2520amino%2520terminal%2520sequences%2520with%2520menin%2520is%2520required%2520for%2520transformation%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26issue%3D15%26spage%3D7275%26epage%3D7283%26doi%3D10.1158%2F0008-5472.CAN-06-2369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. L.</span></span> <span> </span><span class="NLM_article-title">Menin critically links MLL proteins with LEDGF on cancer-associated target genes</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2008.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.ccr.2008.05.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=36-46&issue=1&author=A.+Yokoyamaauthor=M.+L.+Cleary&title=Menin+critically+links+MLL+proteins+with+LEDGF+on+cancer-associated+target+genes&doi=10.1016%2Fj.ccr.2008.05.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DYokoyama%26aufirst%3DA.%26aulast%3DCleary%26aufirst%3DM.%2BL.%26atitle%3DMenin%2520critically%2520links%2520MLL%2520proteins%2520with%2520LEDGF%2520on%2520cancer-associated%2520target%2520genes%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D14%26issue%3D1%26spage%3D36%26epage%3D46%26doi%3D10.1016%2Fj.ccr.2008.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorenson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1038/nchembio.773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnchembio.773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=22286128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=277-284&issue=3&author=J.+Grembeckaauthor=S.+Heauthor=A.+Shiauthor=T.+Purohitauthor=A.+G.+Munteanauthor=R.+J.+Sorensonauthor=H.+D.+Showalterauthor=M.+J.+Muraiauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=J.+L.+Hessauthor=T.+Cierpicki&title=Menin-MLL+inhibitors+reverse+oncogenic+activity+of+MLL+fusion+proteins+in+leukemia&doi=10.1038%2Fnchembio.773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Menin-MLL inhibitors reverse oncogenic activity of MLL fusion proteins in leukemia</span></div><div class="casAuthors">Grembecka, Jolanta; He, Shihan; Shi, Aibin; Purohit, Trupta; Muntean, Andrew G.; Sorenson, Roderick J.; Showalter, Hollis D.; Murai, Marcelo J.; Belcher, Amalia M.; Hartley, Thomas; Hess, Jay L.; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Translocations involving the mixed lineage leukemia (MLL) gene result in human acute leukemias with very poor prognosis.  The leukemogenic activity of MLL fusion proteins is critically dependent on their direct interaction with menin, a product of the multiple endocrine neoplasia (MEN1) gene.  Here we present what are to our knowledge the first small-mol. inhibitors of the menin-MLL fusion protein interaction that specifically bind menin with nanomolar affinities.  These compds. effectively reverse MLL fusion protein-mediated leukemic transformation by downregulating the expression of target genes required for MLL fusion protein oncogenic activity.  They also selectively block proliferation and induce both apoptosis and differentiation of leukemia cells harboring MLL translocations.  Identification of these compds. provides a new tool for better understanding MLL-mediated leukemogenesis and represents a new approach for studying the role of menin as an oncogenic cofactor of MLL fusion proteins.  Our findings also highlight a new therapeutic strategy for aggressive leukemias with MLL rearrangements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_e6VYyS5l7Vg90H21EOLACvtfcHk0lhFf9chAFZPOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOrsrY%253D&md5=55eb9f353031dfef4c1a281b0ac28cba</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.773%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DA.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DSorenson%26aufirst%3DR.%2BJ.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMenin-MLL%2520inhibitors%2520reverse%2520oncogenic%2520activity%2520of%2520MLL%2520fusion%2520proteins%2520in%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26issue%3D3%26spage%3D277%26epage%3D284%26doi%3D10.1038%2Fnchembio.773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into inhibition of the bivalent menin–MLL interaction by small molecules in leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">4461</span>– <span class="NLM_lpage">4469</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&issue=23&author=A.+Shiauthor=M.+J.+Muraiauthor=S.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+insights+into+inhibition+of+the+bivalent+menin%E2%80%93MLL+interaction+by+small+molecules+in+leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520inhibition%2520of%2520the%2520bivalent%2520menin%25E2%2580%2593MLL%2520interaction%2520by%2520small%2520molecules%2520in%2520leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26issue%3D23%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danet-Desnoyers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bixby, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the Menin-MLL interaction blocks progression of MLL leukemia in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.ccell.2015.02.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2015&pages=589-602&issue=4&author=D.+Borkinauthor=S.+Heauthor=H.+Miaoauthor=K.+Kempinskaauthor=J.+Pollockauthor=J.+Chaseauthor=T.+Purohitauthor=B.+Malikauthor=T.+Zhaoauthor=J.+Wangauthor=B.+Wenauthor=H.+Zongauthor=M.+Jonesauthor=G.+Danet-Desnoyersauthor=M.+L.+Guzmanauthor=M.+Talpazauthor=D.+L.+Bixbyauthor=D.+Sunauthor=J.+L.+Hessauthor=A.+G.+Munteanauthor=I.+Maillardauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+Menin-MLL+interaction+blocks+progression+of+MLL+leukemia+in+vivo&doi=10.1016%2Fj.ccell.2015.02.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DChase%26aufirst%3DJ.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZong%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DDanet-Desnoyers%26aufirst%3DG.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DBixby%26aufirst%3DD.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DMaillard%26aufirst%3DI.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520Menin-MLL%2520interaction%2520blocks%2520progression%2520of%2520MLL%2520leukemia%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D27%26issue%3D4%26spage%3D589%26epage%3D602%26doi%3D10.1016%2Fj.ccell.2015.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogliotti, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">High-affinity small-molecule inhibitors of the menin-mixed lineage leukemia (MLL) interaction closely mimic a natural protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1543</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1021/jm401868d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401868d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1543-1556&issue=4&author=S.+Heauthor=T.+J.+Senterauthor=J.+Pollockauthor=C.+Hanauthor=S.+K.+Upadhyayauthor=T.+Purohitauthor=R.+D.+Gogliottiauthor=C.+W.+Lindsleyauthor=T.+Cierpickiauthor=S.+R.+Staufferauthor=J.+Grembecka&title=High-affinity+small-molecule+inhibitors+of+the+menin-mixed+lineage+leukemia+%28MLL%29+interaction+closely+mimic+a+natural+protein-protein+interaction&doi=10.1021%2Fjm401868d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm401868d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401868d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSenter%26aufirst%3DT.%2BJ.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DC.%26aulast%3DUpadhyay%26aufirst%3DS.%2BK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DGogliotti%26aufirst%3DR.%2BD.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DStauffer%26aufirst%3DS.%2BR.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DHigh-affinity%2520small-molecule%2520inhibitors%2520of%2520the%2520menin-mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520interaction%2520closely%2520mimic%2520a%2520natural%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D4%26spage%3D1543%26epage%3D1556%26doi%3D10.1021%2Fjm401868d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Property focused structure-based optimization of small molecule inhibitors of the protein-protein interaction between menin and Mixed Lineage Leukemia (MLL)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01305</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01305" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=892-913&issue=3&author=D.+Borkinauthor=J.+Pollockauthor=K.+Kempinskaauthor=T.+Purohitauthor=X.+Liauthor=B.+Wenauthor=T.+Zhaoauthor=H.+Miaoauthor=S.+Shuklaauthor=M.+Heauthor=D.+Sunauthor=T.+Cierpickiauthor=J.+Grembecka&title=Property+focused+structure-based+optimization+of+small+molecule+inhibitors+of+the+protein-protein+interaction+between+menin+and+Mixed+Lineage+Leukemia+%28MLL%29&doi=10.1021%2Facs.jmedchem.5b01305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01305%26sid%3Dliteratum%253Aachs%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DPollock%26aufirst%3DJ.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DProperty%2520focused%2520structure-based%2520optimization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520protein-protein%2520interaction%2520between%2520menin%2520and%2520Mixed%2520Lineage%2520Leukemia%2520%2528MLL%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D3%26spage%3D892%26epage%3D913%26doi%3D10.1021%2Facs.jmedchem.5b01305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malik, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asangani, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieslik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prensner, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escara-Wilke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stender, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niknafs, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palanisamy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunju, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yocum, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellacheruvu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nesvizhskii, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanasekaran, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the MLL complex in castration-resistant prostate cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">344</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/nm.3830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnm.3830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=25822367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=344-352&issue=4&author=R.+Malikauthor=A.+P.+Khanauthor=I.+A.+Asanganiauthor=M.+Cieslikauthor=J.+R.+Prensnerauthor=X.+Wangauthor=M.+K.+Iyerauthor=X.+Jiangauthor=D.+Borkinauthor=J.+Escara-Wilkeauthor=R.+Stenderauthor=Y.+M.+Wuauthor=Y.+S.+Niknafsauthor=X.+Jingauthor=Y.+Qiaoauthor=N.+Palanisamyauthor=L.+P.+Kunjuauthor=P.+M.+Krishnamurthyauthor=A.+K.+Yocumauthor=D.+Mellacheruvuauthor=A.+I.+Nesvizhskiiauthor=X.+Caoauthor=S.+M.+Dhanasekaranauthor=F.+Y.+Fengauthor=J.+Grembeckaauthor=T.+Cierpickiauthor=A.+M.+Chinnaiyan&title=Targeting+the+MLL+complex+in+castration-resistant+prostate+cancer&doi=10.1038%2Fnm.3830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the MLL complex in castration-resistant prostate cancer</span></div><div class="casAuthors">Malik, Rohit; Khan, Amjad P.; Asangani, Irfan A.; Cieslik, Marcin; Prensner, John R.; Wang, Xiaoju; Iyer, Matthew K.; Jiang, Xia; Borkin, Dmitry; Escara-Wilke, June; Stender, Rachell; Wu, Yi-Mi; Niknafs, Yashar S.; Jing, Xiaojun; Qiao, Yuanyuan; Palanisamy, Nallasivam; Kunju, Lakshmi P.; Krishnamurthy, Pranathi M.; Yocum, Anastasia K.; Mellacheruvu, Dattatreya; Nesvizhskii, Alexey I.; Cao, Xuhong; Dhanasekaran, Saravana M.; Feng, Felix Y.; Grembecka, Jolanta; Cierpicki, Tomasz; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">344-352</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Resistance to androgen deprivation therapies and increased androgen receptor (AR) activity are major drivers of castration-resistant prostate cancer (CRPC).  Although prior work has focused on targeting AR directly, co-activators of AR signaling, which may represent new therapeutic targets, are relatively underexplored.  Here we demonstrate that the mixed-lineage leukemia protein (MLL) complex, a well-known driver of MLL fusion-pos. leukemia, acts as a co-activator of AR signaling.  AR directly interacts with the MLL complex via the menin-MLL subunit.  Menin expression is higher in CRPC than in both hormone-naive prostate cancer and benign prostate tissue, and high menin expression correlates with poor overall survival of individuals diagnosed with prostate cancer.  Treatment with a small-mol. inhibitor of menin-MLL interaction blocks AR signaling and inhibits the growth of castration-resistant tumors in vivo in mice.  Taken together, this work identifies the MLL complex as a crucial co-activator of AR and a potential therapeutic target in advanced prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmvjojfsVQ97Vg90H21EOLACvtfcHk0liB8pxiuvVH4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVOnsrY%253D&md5=31d39514cc6f6cf723146a70676f4021</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnm.3830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3830%26sid%3Dliteratum%253Aachs%26aulast%3DMalik%26aufirst%3DR.%26aulast%3DKhan%26aufirst%3DA.%2BP.%26aulast%3DAsangani%26aufirst%3DI.%2BA.%26aulast%3DCieslik%26aufirst%3DM.%26aulast%3DPrensner%26aufirst%3DJ.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DIyer%26aufirst%3DM.%2BK.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DEscara-Wilke%26aufirst%3DJ.%26aulast%3DStender%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DNiknafs%26aufirst%3DY.%2BS.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DPalanisamy%26aufirst%3DN.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DKrishnamurthy%26aufirst%3DP.%2BM.%26aulast%3DYocum%26aufirst%3DA.%2BK.%26aulast%3DMellacheruvu%26aufirst%3DD.%26aulast%3DNesvizhskii%26aufirst%3DA.%2BI.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DTargeting%2520the%2520MLL%2520complex%2520in%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26issue%3D4%26spage%3D344%26epage%3D352%26doi%3D10.1038%2Fnm.3830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Svoboda, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Noord, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krook, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, E. R.</span></span> <span> </span><span class="NLM_article-title">Tumorigenicity of Ewing sarcoma is critically dependent on the trithorax proteins MLL1 and menin</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">471</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.18632%2Foncotarget.13444" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=458-471&issue=1&author=L.+K.+Svobodaauthor=N.+Baileyauthor=R.+A.+Van+Noordauthor=M.+A.+Krookauthor=A.+Harrisauthor=C.+Cramerauthor=B.+Jasmanauthor=R.+M.+Patelauthor=D.+Thomasauthor=D.+Borkinauthor=T.+Cierpickiauthor=J.+Grembeckaauthor=E.+R.+Lawlor&title=Tumorigenicity+of+Ewing+sarcoma+is+critically+dependent+on+the+trithorax+proteins+MLL1+and+menin&doi=10.18632%2Foncotarget.13444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13444%26sid%3Dliteratum%253Aachs%26aulast%3DSvoboda%26aufirst%3DL.%2BK.%26aulast%3DBailey%26aufirst%3DN.%26aulast%3DVan%2BNoord%26aufirst%3DR.%2BA.%26aulast%3DKrook%26aufirst%3DM.%2BA.%26aulast%3DHarris%26aufirst%3DA.%26aulast%3DCramer%26aufirst%3DC.%26aulast%3DJasman%26aufirst%3DB.%26aulast%3DPatel%26aufirst%3DR.%2BM.%26aulast%3DThomas%26aufirst%3DD.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DLawlor%26aufirst%3DE.%2BR.%26atitle%3DTumorigenicity%2520of%2520Ewing%2520sarcoma%2520is%2520critically%2520dependent%2520on%2520the%2520trithorax%2520proteins%2520MLL1%2520and%2520menin%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D458%26epage%3D471%26doi%3D10.18632%2Foncotarget.13444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, G. H.</span></span> <span> </span><span class="NLM_article-title">Menin promotes hepatocellular carcinogenesis and epigenetically up-regulates Yap1 transcription</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">17480</span>– <span class="NLM_lpage">17485</span>, <span class="refDoi"> DOI: 10.1073/pnas.1312022110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1073%2Fpnas.1312022110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=17480-17485&issue=43&author=B.+Xuauthor=S.+H.+Liauthor=R.+Zhengauthor=S.+B.+Gaoauthor=L.+H.+Dingauthor=Z.+Y.+Yinauthor=X.+Linauthor=Z.+J.+Fengauthor=S.+Zhangauthor=X.+M.+Wangauthor=G.+H.+Jin&title=Menin+promotes+hepatocellular+carcinogenesis+and+epigenetically+up-regulates+Yap1+transcription&doi=10.1073%2Fpnas.1312022110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1312022110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1312022110%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DS.%2BH.%26aulast%3DZheng%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DS.%2BB.%26aulast%3DDing%26aufirst%3DL.%2BH.%26aulast%3DYin%26aufirst%3DZ.%2BY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%2BJ.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BM.%26aulast%3DJin%26aufirst%3DG.%2BH.%26atitle%3DMenin%2520promotes%2520hepatocellular%2520carcinogenesis%2520and%2520epigenetically%2520up-regulates%2520Yap1%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26issue%3D43%26spage%3D17480%26epage%3D17485%26doi%3D10.1073%2Fpnas.1312022110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klossowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic inhibition of the menin–MLL interaction leads to transcriptional repression of PEG10 and blocks hepatocellular carcinoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-17-0580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1158%2F1535-7163.MCT-17-0580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=29142068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Giuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=26-38&issue=1&author=K.+Kempinskaauthor=B.+Malikauthor=D.+Borkinauthor=S.+Klossowskiauthor=S.+Shuklaauthor=H.+Miaoauthor=J.+Wangauthor=T.+Cierpickiauthor=J.+Grembecka&title=Pharmacologic+inhibition+of+the+menin%E2%80%93MLL+interaction+leads+to+transcriptional+repression+of+PEG10+and+blocks+hepatocellular+carcinoma&doi=10.1158%2F1535-7163.MCT-17-0580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic Inhibition of the Menin-MLL Interaction Leads to Transcriptional Repression of PEG10 and Blocks Hepatocellular Carcinoma</span></div><div class="casAuthors">Kempinska, Katarzyna; Malik, Bhavna; Borkin, Dmitry; Klossowski, Szymon; Shukla, Shirish; Miao, Hongzhi; Wang, Jingya; Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26-38</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) accounts for approx. 85% of malignant liver tumors and results in 600,000 deaths each year, emphasizing the need for new therapies.  Upregulation of menin was reported in HCC patients and high levels of menin correlate with poor patient prognosis.  The protein-protein interaction between menin and histone methyltransferase mixed lineage leukemia 1 (MLL1) plays an important role in the development of HCC, implying that pharmacol. inhibition of this interaction could lead to new therapeutic strategy for the HCC patients.  Here, we demonstrate that the menin-MLL inhibitor MI-503 shows antitumor activity in in vitro and in vivo models of HCC and reveals the potential mechanism of menin contribution to HCC.  Treatment with MI-503 selectively kills various HCC cell lines and this effect is significantly enhanced by a combination of MI-503 with sorafenib, the std.-of-care therapy for HCC.  Furthermore, MI-503 reduces sphere formation and cell migration in in vitro HCC models.  When applied in vivo, MI-503 gives a strong antitumor effect both as a single agent and in combination with sorafenib in mice xenograft models of HCC.  Mechanistically, treatment with MI-503 downregulates expression of several genes known to play a crit. role in proliferation and migration of HCC cells, including PEG10, and displaces the menin-MLL1 complex from the PEG10 promoter, resulting in reduced H3K4 methylation and transcriptional repression.  Overall, our studies reveal a mechanistic link between menin and genes involved in HCC and demonstrate that pharmacol. inhibition of the menin-MLL interaction might represent a promising therapeutic approach for HCC.  Mol Cancer Ther; 17(1); 26-38. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI6J4bAkPP2rVg90H21EOLACvtfcHk0lg5BHUux2L2CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Giuw%253D%253D&md5=e4675b46b6306a72d5fe6148e610b224</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-17-0580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-17-0580%26sid%3Dliteratum%253Aachs%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DMalik%26aufirst%3DB.%26aulast%3DBorkin%26aufirst%3DD.%26aulast%3DKlossowski%26aufirst%3DS.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DPharmacologic%2520inhibition%2520of%2520the%2520menin%25E2%2580%2593MLL%2520interaction%2520leads%2520to%2520transcriptional%2520repression%2520of%2520PEG10%2520and%2520blocks%2520hepatocellular%2520carcinoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26issue%3D1%26spage%3D26%26epage%3D38%26doi%3D10.1158%2F1535-7163.MCT-17-0580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Funato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabar, V.</span></span> <span> </span><span class="NLM_article-title">Use of human embryonic stem cells to model pediatric gliomas with H3.3K27M histone mutation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>346</i></span> (<span class="NLM_issue">6216</span>),  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1533</span>, <span class="refDoi"> DOI: 10.1126/science.1253799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1126%2Fscience.1253799" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=1529-1533&issue=6216&author=K.+Funatoauthor=T.+Majorauthor=P.+W.+Lewisauthor=C.+D.+Allisauthor=V.+Tabar&title=Use+of+human+embryonic+stem+cells+to+model+pediatric+gliomas+with+H3.3K27M+histone+mutation&doi=10.1126%2Fscience.1253799"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscience.1253799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1253799%26sid%3Dliteratum%253Aachs%26aulast%3DFunato%26aufirst%3DK.%26aulast%3DMajor%26aufirst%3DT.%26aulast%3DLewis%26aufirst%3DP.%2BW.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DTabar%26aufirst%3DV.%26atitle%3DUse%2520of%2520human%2520embryonic%2520stem%2520cells%2520to%2520model%2520pediatric%2520gliomas%2520with%2520H3.3K27M%2520histone%2520mutation%26jtitle%3DScience%26date%3D2014%26volume%3D346%26issue%3D6216%26spage%3D1529%26epage%3D1533%26doi%3D10.1126%2Fscience.1253799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span> <span> </span><span class="NLM_article-title">Protein-protein interactions as targets for small molecule drug discovery</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>84</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1002/bip.20608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fbip.20608" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2006&pages=535-552&issue=6&author=D.+C.+Fry&title=Protein-protein+interactions+as+targets+for+small+molecule+drug+discovery&doi=10.1002%2Fbip.20608"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fbip.20608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20608%26sid%3Dliteratum%253Aachs%26aulast%3DFry%26aufirst%3DD.%2BC.%26atitle%3DProtein-protein%2520interactions%2520as%2520targets%2520for%2520small%2520molecule%2520drug%2520discovery%26jtitle%3DBiopolymers%26date%3D2006%26volume%3D84%26issue%3D6%26spage%3D535%26epage%3D552%26doi%3D10.1002%2Fbip.20608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, P.</span></span> <span> </span><span class="NLM_article-title">Small-molecule protein-protein interaction inhibitors: therapeutic potential in light of molecular size, chemical space, and ligand binding efficiency considerations</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1002/iub.383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fiub.383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=20979208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=724-731&issue=10&author=P.+Buchwald&title=Small-molecule+protein-protein+interaction+inhibitors%3A+therapeutic+potential+in+light+of+molecular+size%2C+chemical+space%2C+and+ligand+binding+efficiency+considerations&doi=10.1002%2Fiub.383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule protein-protein interaction inhibitors: therapeutic potential in light of molecular size, chemical space, and ligand binding efficiency considerations</span></div><div class="casAuthors">Buchwald, Peter</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">724-731</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  As the ultimate function of proteins depends to a great extent on their binding partners, protein-protein interactions (PPIs) represent a treasure trove of possible new therapeutic targets.  Unfortunately, interfaces involved in PPIs are not well-suited for effective small mol. binding.  Nevertheless, successful examples of small-mol. PPI inhibitors (PPIIs) are beginning to accumulate, and the sheer no. of PPIs that form the human interactome implies that, despite the relative unsuitability of PPIs to serve as "druggable" targets, small-mol. PPIIs can still provide novel pharmacol. tools and new innovative drugs in at least some areas.  Here, after some illustrative examples, accumulating information on the binding efficiency, mol. size, and chem. space requirements will be briefly reviewed.  Therapeutic success can only be achieved if these considerations are incorporated into the search process and if careful medicinal chem. approaches are used to address the absorption, distribution, metab., and excretion requirements of larger mols. that are often needed for this target class due to the lower efficiency of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPrt8JCmQ-mLVg90H21EOLACvtfcHk0lg5BHUux2L2CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM&md5=ae27f95b3ef88b20ad2973927eaf4c91</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fiub.383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.383%26sid%3Dliteratum%253Aachs%26aulast%3DBuchwald%26aufirst%3DP.%26atitle%3DSmall-molecule%2520protein-protein%2520interaction%2520inhibitors%253A%2520therapeutic%2520potential%2520in%2520light%2520of%2520molecular%2520size%252C%2520chemical%2520space%252C%2520and%2520ligand%2520binding%2520efficiency%2520considerations%26jtitle%3DIUBMB%2520Life%26date%3D2010%26volume%3D62%26issue%3D10%26spage%3D724%26epage%3D731%26doi%3D10.1002%2Fiub.383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azzarito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of alpha-helix-mediated protein-protein interactions using designed molecules</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">173</span>, <span class="refDoi"> DOI: 10.1038/nchem.1568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnchem.1568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=23422557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=161-173&issue=3&author=V.+Azzaritoauthor=K.+Longauthor=N.+S.+Murphyauthor=A.+J.+Wilson&title=Inhibition+of+alpha-helix-mediated+protein-protein+interactions+using+designed+molecules&doi=10.1038%2Fnchem.1568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of α-helix-mediated protein-protein interactions using designed molecules</span></div><div class="casAuthors">Azzarito, Valeria; Long, Kerya; Murphy, Natasha S.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-173</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inhibition of protein-protein interactions (PPIs) represents a significant challenge because it is unclear how they can be effectively and selectively targeted using small mols.  Achieving this goal is crit. given the defining role of these interactions in biol. processes.  A rational approach to inhibitor design based on the secondary structure at the interface is the focus of much research, and different classes of designed ligands have emerged, some of which effectively and selectively disrupt targeted PPIs.  This Review discusses the relevance of PPIs and, in particular, the importance of α-helix-mediated PPIs to chem. biol. and drug discovery with a focus on designing inhibitors, including constrained peptides, foldamers and proteomimetic-derived ligands.  In doing so, key challenges and major advances in developing generic approaches for the elaboration of PPI inhibitors are highlighted.  The challenges faced in developing such ligands as drug leads - and how criteria applied to these may differ from conventional small-mol. drugs - are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpsQAFaEkKq7Vg90H21EOLACvtfcHk0lg5BHUux2L2CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D&md5=161f88aa1c745dee9a95b1fab340b9fe</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1568%26sid%3Dliteratum%253Aachs%26aulast%3DAzzarito%26aufirst%3DV.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DN.%2BS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DInhibition%2520of%2520alpha-helix-mediated%2520protein-protein%2520interactions%2520using%2520designed%2520molecules%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26issue%3D3%26spage%3D161%26epage%3D173%26doi%3D10.1038%2Fnchem.1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span> <span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span> (<span class="NLM_issue">7172</span>),  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&issue=7172&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lgfRTBxAJK25Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26issue%3D7172%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The road less traveled: modulating signal transduction enzymes by inhibiting their protein-protein interactions</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2009.05.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.cbpa.2009.05.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=19553156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVWlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=284-290&issue=3&author=M.+R.+Arkinauthor=A.+Whitty&title=The+road+less+traveled%3A+modulating+signal+transduction+enzymes+by+inhibiting+their+protein-protein+interactions&doi=10.1016%2Fj.cbpa.2009.05.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The road less traveled: Modulating signal transduction enzymes by inhibiting their protein-protein interactions</span></div><div class="casAuthors">Arkin, Michelle R.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">284-290</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The biol. functions of intracellular signaling enzymes typically depend on multiple protein-protein interactions (PPI) with substrates, scaffolding proteins, and other cytoplasmic mols.  Blocking these interactions provides an alternative means to modulate signaling activity without fully ablating the catalytic activity of the target.  Several recent reports describe small-mol. antagonists that target PPI sites on signaling enzymes.  These findings suggest that such sites may often be druggable.  However, the hypothesis that targeting such sites might confer on the resulting inhibitors improved properties of efficacy and/or tolerability, while appealing, remains largely untested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbJ1DqXR5rDLVg90H21EOLACvtfcHk0lgfRTBxAJK25Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVWlur8%253D&md5=93e46c2f0d7bd7b16454df7bdc00b792</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2009.05.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2009.05.125%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520road%2520less%2520traveled%253A%2520modulating%2520signal%2520transduction%2520enzymes%2520by%2520inhibiting%2520their%2520protein-protein%2520interactions%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D13%26issue%3D3%26spage%3D284%26epage%3D290%26doi%3D10.1016%2Fj.cbpa.2009.05.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Features of protein-protein interactions that translate into potent inhibitors: topology, surface area and affinity</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">e16</span>, <span class="refDoi"> DOI: 10.1017/erm.2012.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1017%2Ferm.2012.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=22831787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=e16&author=M.+C.+Smithauthor=J.+E.+Gestwicki&title=Features+of+protein-protein+interactions+that+translate+into+potent+inhibitors%3A+topology%2C+surface+area+and+affinity&doi=10.1017%2Ferm.2012.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Features of protein-protein interactions that translate into potent inhibitors: topology, surface area and affinity</span></div><div class="casAuthors">Smith, Matthew C.; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Reviews in Molecular Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e16/1-e16/20</span>CODEN:
                <span class="NLM_cas:coden">ERMMFS</span>;
        ISSN:<span class="NLM_cas:issn">1462-3994</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) control the assembly of multi-protein complexes and, thus, these contacts have enormous potential as drug targets.  However, the field has produced a mix of both exciting success stories and frustrating challenges. Here, we review known examples and explore how the phys. features of a PPI, such as its affinity, hotspots, off-rates, buried surface area and topol., might influence the chances of success in finding inhibitors.  This anal. suggests that concise, tight binding PPIs are most amenable to inhibition.  However, it is also clear that emerging tech. methods are expanding the repertoire of 'druggable' protein contacts and increasing the odds against difficult targets.  In particular, natural product-like compd. libraries, high throughput screens specifically designed for PPIs and approaches that favor discovery of allosteric inhibitors appear to be attractive routes.  The first group of PPI inhibitors has entered clin. trials, further motivating the need to understand the challenges and opportunities in pursuing these types of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6DSZAfVfqDLVg90H21EOLACvtfcHk0lgfRTBxAJK25Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjtL7L&md5=a24bcc721f4c6ebfc8fae029c79f6426</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1017%2Ferm.2012.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252Ferm.2012.10%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DM.%2BC.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DFeatures%2520of%2520protein-protein%2520interactions%2520that%2520translate%2520into%2520potent%2520inhibitors%253A%2520topology%252C%2520surface%2520area%2520and%2520affinity%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2012%26volume%3D14%26spage%3De16%26doi%3D10.1017%2Ferm.2012.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interactions in hematologic malignancies: still a challenge or a great opportunity for future therapies?</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>263</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1111/imr.12244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1111%2Fimr.12244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=25510283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFSnsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=2015&pages=279-301&issue=1&author=T.+Cierpickiauthor=J.+Grembecka&title=Targeting+protein-protein+interactions+in+hematologic+malignancies%3A+still+a+challenge+or+a+great+opportunity+for+future+therapies%3F&doi=10.1111%2Fimr.12244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions in hematologic malignancies: still a challenge or a great opportunity for future therapies?</span></div><div class="casAuthors">Cierpicki, Tomasz; Grembecka, Jolanta</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">279-301</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Summary : Over the past several years, there has been an increasing research effort focused on inhibition of protein-protein interactions (PPIs) to develop novel therapeutic approaches for cancer, including hematol. malignancies.  These efforts have led to development of small mol. inhibitors of PPIs, some of which already advanced to the stage of clin. trials while others are at different stages of preclin. optimization, emphasizing PPIs as an emerging and attractive class of drug targets.  Here, we review several examples of recently developed inhibitors of PPIs highly relevant to hematol. cancers.  We address the existing skepticism about feasibility of targeting PPIs and emphasize potential therapeutic benefit from blocking PPIs in hematol. malignancies.  We then use these examples to discuss the approaches for successful identification of PPI inhibitors and provide anal. of the protein-protein interfaces, with the goal to address 'druggability' of new PPIs relevant to hematol.  We discuss lessons learned to improve the success of targeting new PPIs and evaluate prospects and limits of the research in this field.  We conclude that not all PPIs are equally tractable for blocking by small mols., and detailed anal. of PPI interfaces is crit. for selection of those with the highest chance of success.  Together, our anal. uncovers patterns that should help to advance drug discovery in hematol. malignancies by successful targeting of new PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhINiOkTGpPrVg90H21EOLACvtfcHk0ljzgd8p-TcZCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFSnsrzE&md5=eded19171161abe72de4aab023bbfc58</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fimr.12244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimr.12244%26sid%3Dliteratum%253Aachs%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DGrembecka%26aufirst%3DJ.%26atitle%3DTargeting%2520protein-protein%2520interactions%2520in%2520hematologic%2520malignancies%253A%2520still%2520a%2520challenge%2520or%2520a%2520great%2520opportunity%2520for%2520future%2520therapies%253F%26jtitle%3DImmunol.%2520Rev.%26date%3D2015%26volume%3D263%26issue%3D1%26spage%3D279%26epage%3D301%26doi%3D10.1111%2Fimr.12244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeting apoptosis pathways for new cancer therapeutics</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-010713-141310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1146%2Fannurev-med-010713-141310" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2014&pages=139-155&author=L.+Baiauthor=S.+Wang&title=Targeting+apoptosis+pathways+for+new+cancer+therapeutics&doi=10.1146%2Fannurev-med-010713-141310"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-010713-141310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-010713-141310%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeting%2520apoptosis%2520pathways%2520for%2520new%2520cancer%2520therapeutics%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2014%26volume%3D65%26spage%3D139%26epage%3D155%26doi%3D10.1146%2Fannurev-med-010713-141310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nero, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holien, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wielens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, M. W.</span></span> <span> </span><span class="NLM_article-title">Oncogenic protein interfaces: small molecules, big challenges</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nrc3690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1038%2Fnrc3690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=24622521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=248-262&issue=4&author=T.+L.+Neroauthor=C.+J.+Mortonauthor=J.+K.+Holienauthor=J.+Wielensauthor=M.+W.+Parker&title=Oncogenic+protein+interfaces%3A+small+molecules%2C+big+challenges&doi=10.1038%2Fnrc3690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic protein interfaces: small molecules, big challenges</span></div><div class="casAuthors">Nero, Tracy L.; Morton, Craig J.; Holien, Jessica K.; Wielens, Jerome; Parker, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">248-262</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Historically, targeting protein-protein interactions with small mols. was not thought possible because the corresponding interfaces were considered mostly flat and featureless and therefore 'undruggable'.  Instead, such interactions were targeted with larger mols., such as peptides and antibodies.  However, the past decade has seen encouraging breakthroughs through the refinement of existing techniques and the development of new ones, together with the identification and exploitation of unexpected aspects of protein-protein interaction surfaces.  In this Review, we describe some of the latest techniques to discover modulators of protein-protein interactions and how current drug discovery approaches have been adapted to successfully target these interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlk_D7e8hVV7Vg90H21EOLACvtfcHk0ljzgd8p-TcZCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktV2ktrY%253D&md5=7aad54df0c10c6cdb9b1a7ae0f15bfe9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrc3690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3690%26sid%3Dliteratum%253Aachs%26aulast%3DNero%26aufirst%3DT.%2BL.%26aulast%3DMorton%26aufirst%3DC.%2BJ.%26aulast%3DHolien%26aufirst%3DJ.%2BK.%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DParker%26aufirst%3DM.%2BW.%26atitle%3DOncogenic%2520protein%2520interfaces%253A%2520small%2520molecules%252C%2520big%2520challenges%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26issue%3D4%26spage%3D248%26epage%3D262%26doi%3D10.1038%2Fnrc3690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belcher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of the mixed lineage leukemia-menin interaction: implications for targeting mixed lineage leukemias</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span> (<span class="NLM_issue">52</span>),  <span class="NLM_fpage">40690</span>– <span class="NLM_lpage">40698</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.172783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1074%2Fjbc.M110.172783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=40690-40698&issue=52&author=J.+Grembeckaauthor=A.+M.+Belcherauthor=T.+Hartleyauthor=T.+Cierpicki&title=Molecular+basis+of+the+mixed+lineage+leukemia-menin+interaction%3A+implications+for+targeting+mixed+lineage+leukemias&doi=10.1074%2Fjbc.M110.172783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.172783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.172783%26sid%3Dliteratum%253Aachs%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DBelcher%26aufirst%3DA.%2BM.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DMolecular%2520basis%2520of%2520the%2520mixed%2520lineage%2520leukemia-menin%2520interaction%253A%2520implications%2520for%2520targeting%2520mixed%2520lineage%2520leukemias%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26issue%3D52%26spage%3D40690%26epage%3D40698%26doi%3D10.1074%2Fjbc.M110.172783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of menin reveals binding site for mixed lineage leukemia (MLL) protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">31742</span>– <span class="NLM_lpage">31748</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.258186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1074%2Fjbc.M111.258186" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=31742-31748&issue=36&author=M.+J.+Muraiauthor=M.+Chruszczauthor=G.+Reddyauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Crystal+structure+of+menin+reveals+binding+site+for+mixed+lineage+leukemia+%28MLL%29+protein&doi=10.1074%2Fjbc.M111.258186"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.258186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.258186%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520menin%2520reveals%2520binding%2520site%2520for%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26issue%3D36%26spage%3D31742%26epage%3D31748%26doi%3D10.1074%2Fjbc.M111.258186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span> <span> </span><span class="NLM_article-title">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=12854996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&issue=1&author=X.+Huang&title=Fluorescence+polarization+competition+assay%3A+the+range+of+resolvable+inhibitor+potency+is+limited+by+the+affinity+of+the+fluorescent+ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span></div><div class="casAuthors">Huang, Xinyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">For the development of fluorescence polarization (FP) competition assays, there is a widespread belief that tight-binding fluorescent ligands should be avoided to identify inhibitors of low or intermediate potency in the screening of small-mol. compd. libraries.  It is demonstrated herein that this statement is a misconception; in fact, the higher the affinity of the fluorescent ligand, the wider the range of inhibitor potency that can be resolved.  An approx. est. for the low end of inhibitor Ki values that can be resolved is the Kd value of the fluorescent ligand.  Because FP competition assays are typically conducted under nonstoichiometric titrn. conditions, it is suggested that a fluorescent ligand of highest affinity that also has an adequate quantum yield to satisfy such conditions be selected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUYlJzQaNj7Vg90H21EOLACvtfcHk0lgVM-TteQFzlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D&md5=7c706c96b6e534486f6780c93a0d441b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DFluorescence%2520polarization%2520competition%2520assay%253A%2520the%2520range%2520of%2520resolvable%2520inhibitor%2520potency%2520is%2520limited%2520by%2520the%2520affinity%2520of%2520the%2520fluorescent%2520ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26issue%3D1%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamagnan, G.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis of (+)-(S,S)-reboxetine via a new (S)-2-(hydroxymethyl)morpholine preparation</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">937</span>– <span class="NLM_lpage">939</span>, <span class="refDoi"> DOI: 10.1021/ol050059g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol050059g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=937-939&issue=5&author=E.+Brennerauthor=R.+M.+Baldwinauthor=G.+Tamagnan&title=Asymmetric+synthesis+of+%28%2B%29-%28S%2CS%29-reboxetine+via+a+new+%28S%29-2-%28hydroxymethyl%29morpholine+preparation&doi=10.1021%2Fol050059g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol050059g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol050059g%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DE.%26aulast%3DBaldwin%26aufirst%3DR.%2BM.%26aulast%3DTamagnan%26aufirst%3DG.%26atitle%3DAsymmetric%2520synthesis%2520of%2520%2528%252B%2529-%2528S%252CS%2529-reboxetine%2520via%2520a%2520new%2520%2528S%2529-2-%2528hydroxymethyl%2529morpholine%2520preparation%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26issue%3D5%26spage%3D937%26epage%3D939%26doi%3D10.1021%2Fol050059g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, M.</span>; <span class="NLM_string-name">Seekamp, C. K.</span>; <span class="NLM_string-name">Zhang, X.</span>; <span class="NLM_string-name">Tarrant, J. G.</span>; <span class="NLM_string-name">Lee, C.-W.</span>; <span class="NLM_string-name">Kaiser, B.</span>; <span class="NLM_string-name">Zablocki, M. M.</span>; <span class="NLM_string-name">Chenard, B. L.</span>; <span class="NLM_string-name">Doller, D.</span>; <span class="NLM_string-name">Yamaguchi, Y.</span>; <span class="NLM_string-name">Wustrow, D. J.</span>; <span class="NLM_string-name">Li, G.</span></span> <span> </span><span class="NLM_article-title">Aminopiperidines and Realted Compounds</span>. Int. Patent Appl. <span class="NLM_patent">WO2008016811A2</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=M.+Ghosh&author=C.+K.+Seekamp&author=X.+Zhang&author=J.+G.+Tarrant&author=C.-W.+Lee&author=B.+Kaiser&author=M.+M.+Zablocki&author=B.+L.+Chenard&author=D.+Doller&author=Y.+Yamaguchi&author=D.+J.+Wustrow&author=G.+Li&title=Aminopiperidines+and+Realted+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DM.%26atitle%3DAminopiperidines%2520and%2520Realted%2520Compounds%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the first-in-class, highly potent irreversible inhibitor targeting the menin–MLL protein-protein interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&issue=6&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fernandez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+first-in-class%2C+highly+potent+irreversible+inhibitor+targeting+the+menin%E2%80%93MLL+protein-protein+interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520first-in-class%252C%2520highly%2520potent%2520irreversible%2520inhibitor%2520targeting%2520the%2520menin%25E2%2580%2593MLL%2520protein-protein%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26issue%3D6%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fairhead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howarth, M.</span></span> <span> </span><span class="NLM_article-title">Site-specific biotinylation of purified proteins using BirA</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1266</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2272-7_12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1007%2F978-1-4939-2272-7_12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=25560075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1Sitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1266&publication_year=2015&pages=171-184&author=M.+Fairheadauthor=M.+Howarth&title=Site-specific+biotinylation+of+purified+proteins+using+BirA&doi=10.1007%2F978-1-4939-2272-7_12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Specific Biotinylation of Purified Proteins Using BirA</span></div><div class="casAuthors">Fairhead, Michael; Howarth, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1266</span>
        (<span class="NLM_cas:issue">Site-Specific Protein Labeling</span>),
    <span class="NLM_cas:pages">171-184</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The binding between biotin and streptavidin or avidin is one of the strongest known non-covalent biol. interactions.  The (strept)avidin-biotin interaction has been widely used for decades in biol. research and biotechnol.  Therefore labeling of purified proteins by biotin is a powerful way to achieve protein capture, immobilization, and functionalization, as well as multimerizing or bridging mols.  Chem. biotinylation often generates heterogeneous products, which may have impaired function.  Enzymic biotinylation with E. coli biotin ligase (BirA) is highly specific in covalently attaching biotin to the 15 amino acid AviTag peptide, giving a homogeneous product with high yield.  AviTag can conveniently be added genetically at the N-terminus, C-terminus, or in exposed loops of a target protein.  We describe here procedures for AviTag insertion by inverse PCR, purifn. of BirA fused to glutathione-S-transferase (GST-BirA) from E. coli , BirA biotinylation of purified protein, and gel-shift anal. by SDS-PAGE to quantify the extent of biotinylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptd53yZ7pVCrVg90H21EOLACvtfcHk0ljkMTqw3jTzGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1Sitbg%253D&md5=06168aa71460dff05856cab89576e15d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2272-7_12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2272-7_12%26sid%3Dliteratum%253Aachs%26aulast%3DFairhead%26aufirst%3DM.%26aulast%3DHowarth%26aufirst%3DM.%26atitle%3DSite-specific%2520biotinylation%2520of%2520purified%2520proteins%2520using%2520BirA%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1266%26spage%3D171%26epage%3D184%26doi%3D10.1007%2F978-1-4939-2272-7_12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0ljkMTqw3jTzGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&issue=3&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljkMTqw3jTzGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26issue%3D3%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&issue=12&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhYZ4eZhLK7xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D12%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Collaborative
Computational Project, N.</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&issue=5&author=Collaborative%0AComputational+Project%2C+N.&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26issue%3D5%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&issue=2&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhYZ4eZhLK7xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D2%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaver-Fay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snoeyink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">Web Server</span>),  <span class="NLM_fpage">W375</span>– <span class="NLM_lpage">W383</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=17452350" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&issue=Web+Server&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids&doi=10.1093%2Fnar%2Fgkm216"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26issue%3DWeb%2520Server%26spage%3DW375%26epage%3DW383%26doi%3D10.1093%2Fnar%2Fgkm216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guranovic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J. D.</span></span> <span> </span><span class="NLM_article-title">Automated and accurate deposition of structures solved by X-ray diffraction to the Protein Data Bank</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1833</span>– <span class="NLM_lpage">1839</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1107%2FS0907444904019419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=15388930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnslOltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1833-1839&issue=10&author=H.+Yangauthor=V.+Guranovicauthor=S.+Duttaauthor=Z.+Fengauthor=H.+M.+Bermanauthor=J.+D.+Westbrook&title=Automated+and+accurate+deposition+of+structures+solved+by+X-ray+diffraction+to+the+Protein+Data+Bank&doi=10.1107%2FS0907444904019419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Automated and accurate deposition of structures solved by X-ray diffraction to the Protein Data Bank</span></div><div class="casAuthors">Yang, Huanwang; Guranovic, Vladimir; Dutta, Shuchismita; Feng, Zukang; Berman, Helen M.; Westbrook, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1833-1839</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The RCSB Protein Data Bank (PDB) has a no. of options for deposition of structural data and has developed software tools to facilitate the process.  In addn. to ADIT and the PDB Validation Suite, a new software application, pdb_ext., has been designed to promote automatic data deposition of structures solved by x-ray diffraction.  The pdb_ext. software can ext. information about data redn., phasing, mol. replacement, d. modification and refinement from the output files produced by many x-ray crystallog. applications.  The options, procedures and tools for accurate and automated PDB data deposition are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpASfaztZ8sFbVg90H21EOLACvtfcHk0ljRJmvGn5QddQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnslOltbo%253D&md5=8df9343c464ef1804d31041009bc2f44</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019419%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGuranovic%26aufirst%3DV.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%2BD.%26atitle%3DAutomated%2520and%2520accurate%2520deposition%2520of%2520structures%2520solved%2520by%2520X-ray%2520diffraction%2520to%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26issue%3D10%26spage%3D1833%26epage%3D1839%26doi%3D10.1107%2FS0907444904019419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muntean, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitwala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basrur, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenitoba-Johnson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">The PAF complex synergizes with MLL fusion proteins at HOX loci to promote leukemogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.ccr.2010.04.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=609-621&issue=6&author=A.+G.+Munteanauthor=J.+Tanauthor=K.+Sitwalaauthor=Y.+Huangauthor=J.+Bronsteinauthor=J.+A.+Connellyauthor=V.+Basrurauthor=K.+S.+Elenitoba-Johnsonauthor=J.+L.+Hess&title=The+PAF+complex+synergizes+with+MLL+fusion+proteins+at+HOX+loci+to+promote+leukemogenesis&doi=10.1016%2Fj.ccr.2010.04.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DMuntean%26aufirst%3DA.%2BG.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DSitwala%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DBronstein%26aufirst%3DJ.%26aulast%3DConnelly%26aufirst%3DJ.%2BA.%26aulast%3DBasrur%26aufirst%3DV.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520PAF%2520complex%2520synergizes%2520with%2520MLL%2520fusion%2520proteins%2520at%2520HOX%2520loci%2520to%2520promote%2520leukemogenesis%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26issue%3D6%26spage%3D609%26epage%3D621%26doi%3D10.1016%2Fj.ccr.2010.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madigan, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maring, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rockway, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, D. J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of HCV NS5B polymerase: synthesis and structure-activity relationships of N-alkyl-4-hydroxyquinolon-3-yl-benzothiadiazine sulfamides</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3367</span>– <span class="NLM_lpage">3370</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;key=10.1016%2Fj.bmcl.2006.04.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=3367-3370&issue=13&author=A.+C.+Kruegerauthor=D.+L.+Madiganauthor=W.+W.+Jiangauthor=W.+M.+Katiauthor=D.+Liuauthor=Y.+Liuauthor=C.+J.+Maringauthor=S.+Masseauthor=K.+F.+McDanielauthor=T.+Middletonauthor=H.+Moauthor=A.+Mollaauthor=D.+Montgomeryauthor=J.+K.+Prattauthor=T.+W.+Rockwayauthor=R.+Zhangauthor=D.+J.+Kempf&title=Inhibitors+of+HCV+NS5B+polymerase%3A+synthesis+and+structure-activity+relationships+of+N-alkyl-4-hydroxyquinolon-3-yl-benzothiadiazine+sulfamides&doi=10.1016%2Fj.bmcl.2006.04.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DKrueger%26aufirst%3DA.%2BC.%26aulast%3DMadigan%26aufirst%3DD.%2BL.%26aulast%3DJiang%26aufirst%3DW.%2BW.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaring%26aufirst%3DC.%2BJ.%26aulast%3DMasse%26aufirst%3DS.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DMiddleton%26aufirst%3DT.%26aulast%3DMo%26aufirst%3DH.%26aulast%3DMolla%26aufirst%3DA.%26aulast%3DMontgomery%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DRockway%26aufirst%3DT.%2BW.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26atitle%3DInhibitors%2520of%2520HCV%2520NS5B%2520polymerase%253A%2520synthesis%2520and%2520structure-activity%2520relationships%2520of%2520N-alkyl-4-hydroxyquinolon-3-yl-benzothiadiazine%2520sulfamides%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26issue%3D13%26spage%3D3367%26epage%3D3370%26doi%3D10.1016%2Fj.bmcl.2006.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X5Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X5Y','PDB','4X5Y'); return false;">PDB: 4X5Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXH','PDB','6BXH'); return false;">PDB: 6BXH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXY','PDB','6BXY'); return false;">PDB: 6BXY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U88" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3U88','PDB','3U88'); return false;">PDB: 3U88</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BY8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BY8','PDB','6BY8'); return false;">PDB: 6BY8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GPQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GPQ','PDB','4GPQ'); return false;">PDB: 4GPQ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i67"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00071">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_23160"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00071">10.1021/acs.jmedchem.8b00071</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary Table 1, crystallography data; Supplementary Figure 1, <i>K</i><sub>d</sub> results from the FP assay for MLL<sub>4-43</sub> binding to menin; Supplementary Figure 2, BLI studies for menin–MLL inhibitors; Supplementary Figure 3, PK results for <b>28</b>; Supplementary Figure 4, mice body weight from <i>in vivo</i> efficacy studies with <b>28</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_001.pdf">jm8b00071_si_001.pdf (431.98 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00071/suppl_file/jm8b00071_si_002.csv">jm8b00071_si_002.csv (2.98 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinate files have been deposited in PDB with the following codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXH">6BXH</a> (menin-compound <b>13</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BXY">6BXY</a> (menin-compound <b>28</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BY8">6BY8</a> (menin-compound <b>29</b>).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00071&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00071%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00071" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b6eadcc5d93e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
