
<html lang="en"     class="pb-page"  data-request-id="7e42f5f8-6591-44cd-8592-033b05bd50ed"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm3011838;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2012.55.issue-20;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Pharmacokinetic Optimization of Class-Selective Histone Deacetylase Inhibitors and Identification of Associated Candidate Predictive Biomarkers of Hepatocellular Carcinoma Tumor Response" /></meta><meta name="dc.Creator" content="Jason C.  Wong" /></meta><meta name="dc.Creator" content="Guozhi  Tang" /></meta><meta name="dc.Creator" content="Xihan  Wu" /></meta><meta name="dc.Creator" content="Chungen  Liang" /></meta><meta name="dc.Creator" content="Zhenshan  Zhang" /></meta><meta name="dc.Creator" content="Lei  Guo" /></meta><meta name="dc.Creator" content="Zhenghong  Peng" /></meta><meta name="dc.Creator" content="Weixing  Zhang" /></meta><meta name="dc.Creator" content="Xianfeng  Lin" /></meta><meta name="dc.Creator" content="Zhanguo  Wang" /></meta><meta name="dc.Creator" content="Jianghua  Mei" /></meta><meta name="dc.Creator" content="Junli  Chen" /></meta><meta name="dc.Creator" content="Song  Pan" /></meta><meta name="dc.Creator" content="Nan  Zhang" /></meta><meta name="dc.Creator" content="Yongfu  Liu" /></meta><meta name="dc.Creator" content="Mingwei  Zhou" /></meta><meta name="dc.Creator" content="Lichun  Feng" /></meta><meta name="dc.Creator" content="Weili  Zhao" /></meta><meta name="dc.Creator" content="Shijie  Li" /></meta><meta name="dc.Creator" content="Chao  Zhang" /></meta><meta name="dc.Creator" content="Meifang  Zhang" /></meta><meta name="dc.Creator" content="Yiping  Rong" /></meta><meta name="dc.Creator" content="Tai-Guang  Jin" /></meta><meta name="dc.Creator" content="Xiongwen  Zhang" /></meta><meta name="dc.Creator" content="Shuang  Ren" /></meta><meta name="dc.Creator" content="Ying  Ji" /></meta><meta name="dc.Creator" content="Rong  Zhao" /></meta><meta name="dc.Creator" content="Jin  She" /></meta><meta name="dc.Creator" content="Yi  Ren" /></meta><meta name="dc.Creator" content="Chunping  Xu" /></meta><meta name="dc.Creator" content="Dawei  Chen" /></meta><meta name="dc.Creator" content="Jie  Cai" /></meta><meta name="dc.Creator" content="Song  Shan" /></meta><meta name="dc.Creator" content="Desi  Pan" /></meta><meta name="dc.Creator" content="Zhiqiang  Ning" /></meta><meta name="dc.Creator" content="Xianping  Lu" /></meta><meta name="dc.Creator" content="Taiping  Chen" /></meta><meta name="dc.Creator" content="Yun  He" /></meta><meta name="dc.Creator" content="Li  Chen" /></meta><meta name="dc.Description" content="Herein, we describe the pharmacokinetic optimization of a series of class-selective histone deacetylase (HDAC) inhibitors and the subsequent identification of candidate predictive biomarkers of hep..." /></meta><meta name="Description" content="Herein, we describe the pharmacokinetic optimization of a series of class-selective histone deacetylase (HDAC) inhibitors and the subsequent identification of candidate predictive biomarkers of hep..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 12, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm3011838" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2012 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm3011838" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm3011838" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm3011838" /></link>
        
    
    

<title>Pharmacokinetic Optimization of Class-Selective Histone Deacetylase Inhibitors and Identification of Associated Candidate Predictive Biomarkers of Hepatocellular Carcinoma Tumor Response | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm3011838" /></meta><meta property="og:title" content="Pharmacokinetic Optimization of Class-Selective Histone Deacetylase Inhibitors and Identification of Associated Candidate Predictive Biomarkers of Hepatocellular Carcinoma Tumor Response" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0023.jpeg" /></meta><meta property="og:description" content="Herein, we describe the pharmacokinetic optimization of a series of class-selective histone deacetylase (HDAC) inhibitors and the subsequent identification of candidate predictive biomarkers of hepatocellular carcinoma (HCC) tumor response for our clinical lead using patient-derived HCC tumor xenograft models. Through a combination of conformational constraint and scaffold hopping, we lowered the in vivo clearance (CL) and significantly improved the bioavailability (F) and exposure (AUC) of our HDAC inhibitors while maintaining selectivity toward the class I HDAC family with particular potency against HDAC1, resulting in clinical lead 5 (HDAC1 IC50 = 60 nM, mouse CL = 39 mL/min/kg, mouse F = 100%, mouse AUC after single oral dose at 10 mg/kg = 6316 h·ng/mL). We then evaluated 5 in a biomarker discovery pilot study using patient-derived tumor xenograft models, wherein two out of the three models responded to treatment. By comparing tumor response status to compound tumor exposure, induction of acetylated histone H3, candidate gene expression changes, and promoter DNA methylation status from all three models at various time points, we identified preliminary candidate response prediction biomarkers that warrant further validation in a larger cohort of patient-derived tumor models and through confirmatory functional studies." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm3011838"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm3011838">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm3011838&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm3011838&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm3011838&amp;href=/doi/10.1021/jm3011838" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 8903-8925</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm301172v" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm301210j" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Pharmacokinetic Optimization of Class-Selective Histone Deacetylase Inhibitors and Identification of Associated Candidate Predictive Biomarkers of Hepatocellular Carcinoma Tumor Response</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jason+C.++Wong">Jason C. Wong</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guozhi++Tang">Guozhi Tang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xihan++Wu">Xihan Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chungen++Liang">Chungen Liang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhenshan++Zhang">Zhenshan Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Guo">Lei Guo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhenghong++Peng">Zhenghong Peng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Weixing++Zhang">Weixing Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xianfeng++Lin">Xianfeng Lin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhanguo++Wang">Zhanguo Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jianghua++Mei">Jianghua Mei</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Junli++Chen">Junli Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Song++Pan">Song Pan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nan++Zhang">Nan Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yongfu++Liu">Yongfu Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mingwei++Zhou">Mingwei Zhou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lichun++Feng">Lichun Feng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Weili++Zhao">Weili Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shijie++Li">Shijie Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chao++Zhang">Chao Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Meifang++Zhang">Meifang Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yiping++Rong">Yiping Rong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tai-Guang++Jin">Tai-Guang Jin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiongwen++Zhang">Xiongwen Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shuang++Ren">Shuang Ren</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Ji">Ying Ji</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rong++Zhao">Rong Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jin++She">Jin She</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yi++Ren">Yi Ren</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chunping++Xu">Chunping Xu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dawei++Chen">Dawei Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Cai">Jie Cai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Song++Shan">Song Shan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Desi++Pan">Desi Pan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhiqiang++Ning">Zhiqiang Ning</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xianping++Lu">Xianping Lu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Taiping++Chen">Taiping Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yun++He">Yun He</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Li++Chen">Li Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Roche R&D Center (China) Ltd., 720 Cai Lun Road, Building 5, Pudong, Shanghai 201203, P. R. China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Crown Bioscience Inc., Light Muller Building, Changping Sector of Zhongguancun Science Park, No.21 Huoju Street, Changping District, Beijing 102200, P. R. China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Chipscreen Biosciences Ltd., 2-601, BIO-Incubator, Gaoxin C, first Ave., Shenzhen High-Tech Industrial Park, Nanshan District, Shenzhen, Guangdong 518057, P. R. China</span></div><div class="corresp-info"><strong>*</strong>Phone: 1-510-710-4129. E-mail: <a href="/cdn-cgi/l/email-protection#c0aab7afaea780a1acb5adeeada9b4eea5a4b5"><span class="__cf_email__" data-cfemail="2248554d4c4562434e574f0c4f4b560c474657">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm3011838&amp;href=/doi/10.1021%2Fjm3011838" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 8903–8925</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 12, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 August 2012</li><li><span class="item_label"><b>Published</b> online</span>12 October 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 October 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm3011838" title="DOI URL">https://doi.org/10.1021/jm3011838</a></div><div class="article_header-article-copyright"><strong>Copyright © 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8903%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJason%2BC.%2BWong%252C%2BGuozhi%2BTang%252C%2BXihan%2BWu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D55%26issueNum%3D20%26contentID%3Djm3011838%26title%3DPharmacokinetic%2BOptimization%2Bof%2BClass-Selective%2BHistone%2BDeacetylase%2BInhibitors%2Band%2BIdentification%2Bof%2BAssociated%2BCandidate%2BPredictive%2BBiomarkers%2Bof%2BHepatocellular%2BCarcinoma%2BTumor%2BResponse%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8925%26publicationDate%3DOctober%2B2012">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm3011838"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3528</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">23</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm3011838" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Pharmacokinetic Optimization of Class-Selective Histone Deacetylase Inhibitors and Identification of Associated Candidate Predictive Biomarkers of Hepatocellular Carcinoma Tumor Response&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;C. Wong&quot;},{&quot;first_name&quot;:&quot;Guozhi&quot;,&quot;last_name&quot;:&quot;Tang&quot;},{&quot;first_name&quot;:&quot;Xihan&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Chungen&quot;,&quot;last_name&quot;:&quot;Liang&quot;},{&quot;first_name&quot;:&quot;Zhenshan&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Zhenghong&quot;,&quot;last_name&quot;:&quot;Peng&quot;},{&quot;first_name&quot;:&quot;Weixing&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Xianfeng&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Zhanguo&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jianghua&quot;,&quot;last_name&quot;:&quot;Mei&quot;},{&quot;first_name&quot;:&quot;Junli&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Song&quot;,&quot;last_name&quot;:&quot;Pan&quot;},{&quot;first_name&quot;:&quot;Nan&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yongfu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Mingwei&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Lichun&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Weili&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Shijie&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Chao&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Meifang&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yiping&quot;,&quot;last_name&quot;:&quot;Rong&quot;},{&quot;first_name&quot;:&quot;Tai-Guang&quot;,&quot;last_name&quot;:&quot;Jin&quot;},{&quot;first_name&quot;:&quot;Xiongwen&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Shuang&quot;,&quot;last_name&quot;:&quot;Ren&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Ji&quot;},{&quot;first_name&quot;:&quot;Rong&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Jin&quot;,&quot;last_name&quot;:&quot;She&quot;},{&quot;first_name&quot;:&quot;Yi&quot;,&quot;last_name&quot;:&quot;Ren&quot;},{&quot;first_name&quot;:&quot;Chunping&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Dawei&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Cai&quot;},{&quot;first_name&quot;:&quot;Song&quot;,&quot;last_name&quot;:&quot;Shan&quot;},{&quot;first_name&quot;:&quot;Desi&quot;,&quot;last_name&quot;:&quot;Pan&quot;},{&quot;first_name&quot;:&quot;Zhiqiang&quot;,&quot;last_name&quot;:&quot;Ning&quot;},{&quot;first_name&quot;:&quot;Xianping&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Taiping&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yun&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Chen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;8903-8925&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm3011838&quot;},&quot;abstract&quot;:&quot;Herein, we describe the pharmacokinetic optimization of a series of class-selective histone deacetylase (HDAC) inhibitors and the subsequent identification of candidate predictive biomarkers of hepatocellular carcinoma (HCC) tumor response for our clinical lead using patient-derived HCC tumor xenograft models. Through a combination of conformational constraint and scaffold hopping, we lowered the in vivo clearance (CL) and significantly improved the bioavailability (F) and exposure (AUC) of our HDAC inhibitors while maintaining selectivity toward the class I HDAC family with particular potency against HDAC1, resulting in clinical lead 5 (HDAC1 IC50 = 60 nM, mouse CL = 39 mL/min/kg, mouse F = 100%, mouse AUC after single oral dose at 10 mg/kg = 6316 h·ng/mL). We then evaluated 5 in a biomarker discovery pilot study using patient-derived tumor xenograft models, wherein two out of the three models responded to treatment. By comparing tumor response status to compound tumor exposure, induction of acetylated histo&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3011838&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011838" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3011838&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011838" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3011838&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011838" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm3011838&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011838&amp;href=/doi/10.1021/jm3011838" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm3011838" /></input><a href="/doi/pdf/10.1021/jm3011838" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm3011838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm3011838%26sid%3Dliteratum%253Aachs%26pmid%3D23061376%26genre%3Darticle%26aulast%3DWong%26date%3D2012%26atitle%3DPharmacokinetic%2BOptimization%2Bof%2BClass-Selective%2BHistone%2BDeacetylase%2BInhibitors%2Band%2BIdentification%2Bof%2BAssociated%2BCandidate%2BPredictive%2BBiomarkers%2Bof%2BHepatocellular%2BCarcinoma%2BTumor%2BResponse%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D20%26spage%3D8903%26epage%3D8925%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292164" title="Pyrrolidines">Pyrrolidines</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291304" title="Organic reactions">Organic reactions</a>,</li><li><a href="/action/doSearch?ConceptID=292153" title="Tumors">Tumors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jmcmar.2012.55.issue-20/production/jmcmar.2012.55.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Herein, we describe the pharmacokinetic optimization of a series of class-selective histone deacetylase (HDAC) inhibitors and the subsequent identification of candidate predictive biomarkers of hepatocellular carcinoma (HCC) tumor response for our clinical lead using patient-derived HCC tumor xenograft models. Through a combination of conformational constraint and scaffold hopping, we lowered the in vivo clearance (CL) and significantly improved the bioavailability (F) and exposure (AUC) of our HDAC inhibitors while maintaining selectivity toward the class I HDAC family with particular potency against HDAC1, resulting in clinical lead <b>5</b> (HDAC1 IC<sub>50</sub> = 60 nM, mouse CL = 39 mL/min/kg, mouse <i>F</i> = 100%, mouse AUC after single oral dose at 10 mg/kg = 6316 h·ng/mL). We then evaluated <b>5</b> in a biomarker discovery pilot study using patient-derived tumor xenograft models, wherein two out of the three models responded to treatment. By comparing tumor response status to compound tumor exposure, induction of acetylated histone H3, candidate gene expression changes, and promoter DNA methylation status from all three models at various time points, we identified preliminary candidate response prediction biomarkers that warrant further validation in a larger cohort of patient-derived tumor models and through confirmatory functional studies.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>#</span> Author Present Address</h4><p class="last">14 Moon Beam Drive, Mountain View, CA 94043.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Hepatocellular carcinoma (HCC) is among the leading causes of cancer death worldwide with an overall 5-year median survival rate that remains below 12% in the United States.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Most HCC patients are first diagnosed when the disease has reached an advanced and/or unresectable stage where either systemic or local chemotherapy are the standard treatment options.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Currently, systemic treatment with the multitargeted kinase inhibitor sorafenib is considered the standard of care for advanced HCC, providing a median overall survival of 10.7 months as compared to 7.9 months with placebo.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In light of the poor overall survival rates associated with HCC and the limited efficacy of sorafenib, there is a great need to identify new targets and corresponding drug candidates for HCC treatment.</div><div class="NLM_p">The histone deacetylases (HDACs) are a family of Zn<sup>2+</sup>-dependent hydrolases that remove the acetyl group from the acetylated ε-amino side chain of lysine residues in histones and in a wide variety of other post-translationally acetylated proteins. The HDAC family of enzymes can be divided into two major groups, the Rpd3/Hda1-like NAD-independent lysine deacetylases and the NAD-dependent sirtuins.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The Rpd3/Hda1-like deacetylase group has been further organized into class I enzymes (HDACs 1, 2, 3, and 8) and class II enzymes (HDACs 4, 5, 6, 7, 9, 10) based on sequence similarity to the yeast HDACs Rpd3 and Hda1, respectively.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">With respect to HDACs as potential targets for HCC treatment, HDAC1 in particular has been found to be significantly overexpressed in HCC patient tumor samples and its expression in this context is negatively correlated to overall survival.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Moreover, hepatitis B virus (HBV), which has been strongly implicated in HCC pathogenesis, has been demonstrated to induce HDAC1 overexpression through the HBV HBx protein in hepatocellular carcinoma cells, leading to an enhancement of pro-tumor hypoxia signaling.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Functionally, small molecule HDAC inhibition in HCC cells induces antiproliferative and pro-apoptotic effects that correlate with in-cell target inhibition as assessed by classical pharmacodynamic markers such as p21 and acetylated histone H3.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> Moreover, in certain cellular contexts, HDAC inhibition can reactivate the expression of tumor suppressor genes such as TMS1/ASC<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and TFPI-2<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> that have been epigenetically silenced through promoter CpG island DNA hypermethylation which ultimately recruits multiprotein complexes including HDAC1 to enforce a transcriptionally repressed heterochromatin state.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Furthermore, in preclinical in vivo models, HDAC inhibitors can significantly inhibit the growth of established HCC tumors.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> As a result of the evidence linking HDAC activity with HCC pathogenesis and progression, several pan-HDAC inhibitors (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A) are currently in or recently have been in early stage clinical trials in HCC either as single agents or in combination with sorafenib or traditional chemotherapy.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative HDAC inhibitors. (A) HDAC inhibitors that have been or are being evaluated in the clinic for HCC therapy; (B) <i>o</i>-aminoanilide HDAC inhibitors under clinical development.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of the evidence implicating HDAC1 in HCC in particular, and in contrast to the pan-HDAC inhibitors currently under clinical evaluation in HCC, we set out to develop a selective class I HDAC inhibitor based on the <i>o</i>-aminoanilide pharmacophore<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B) with particular potency toward HDAC1 for HCC therapy. Furthermore, consistent with our general strategy of developing companion predictive and/or pharmacodynamic biomarkers for our drug discovery programs, we incorporated biomarker hypothesis generation<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> as part of our preclinical candidate research and selection efforts, as detailed below, to ultimately inform and improve our future clinical trial design. In particular, in light of the relatively limited clinical efficacy of HDACi in solid tumor treatment thus far, we envisioned that the identification of predictive biomarkers of tumor response could help guide the selection of HCC patients more likely to benefit from treatment with our class I-selective HDACi in either a monotherapy or combination therapy setting.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> While other researchers have previously sought to identify predictive biomarkers for HDACi in various cell models and/or clinical isolates,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> to our knowledge the research disclosed herein is the first published example employing mRNA and DNA methylation profiling in patient-derived xenograft models to identify candidate predictive biomarkers of HCC tumor response for a class I-selective HDAC inhibitor.</div><div class="NLM_p">We previously identified compound <b>1</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) as an <i>o</i>-aminoanilide-based selective class I HDAC inhibitor with particular potency against HDAC1 and good cell-based activity as assessed by p21 reporter gene assay, endogenous p21 and acetylated histone H3 induction assays, and antiproliferative assays in various cancer cell lines.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> While <b>1</b> and its closely related analogues had reasonable enzyme- and cell-based activities, these compounds were not suitable for further drug candidate development due to high in vivo clearance and limited bioavailability. We therefore undertook a two-stage campaign to optimize compound <b>1</b>. In the first stage, we attempted to replace the conformationally flexible and metabolically labile morpholine side chain with more compact and rigid moieties to balance the overall properties of our lead series. In the second stage, motivated by the modest, but still suboptimal, property improvement attained by replacing the morpholine side chain, we further rigidified our lead structure through a scaffold-hopping approach.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This two-staged effort ultimately led to the discovery of clinical candidate compound <b>5</b> and its evaluation in primary patient tumor-derived HCC xenograft models for the identification of candidate predictive tumor response biomarkers, as detailed below.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Summary of compound <b>1</b> optimization history leading to identification of clinical lead <b>5</b> and the associated <i>trans</i>-3,4-disubstituted pyrrolidine lead series. P21_RP3 is the relative fold increase in luciferase signal in the p21 reporter gene assay compared to clinical HDACi entinostat (both test compound and comparator were evaluated at 3 μM in duplicate or triplicate) and normalized by internal GFP transfection control. CLh (MLM) is the calculated hepatic clearance based on mouse liver microsome intrinsic clearance. CL (mouse) is the in vivo systemic clearance measured by intravenous pharmacokinetic study in BALB/C nude mice. F is the bioavailability determined by single dose intravenous (2 mpk) and oral (10 mpk) pharmacokinetic study in nude mice. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for further details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64308" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64308" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> General Comments on Stereochemistry</h3><div class="NLM_p last">Unless otherwise noted, compounds were synthesized as racemic mixtures.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> General Synthesis of Morpholine Side-Chain Replacement Analogues of Compound <b>1</b> (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>)</h3><div class="NLM_p">Synthesis of morpholine side-chain replacement analogues of compound <b>1</b> began with esterification of commercially available 2-(4-bromophenyl)-2-hydroxyacetate in methanol with sulfuric acid as catalyst to obtain methyl ester <b>I</b>. Subsequent Heck coupling of <b>I</b> with acrylamide <b>III</b> (obtained by acylation of Boc-protected aniline <b>II</b> with the mixed anhydride of acrylic acid) using tri-<i>o</i>-tolylphosphine as ligand provided cinnamide <b>IV</b> in moderate yield. Mesylation of <b>IV</b> followed by nucleophic substitution on the resulting mesylates with alkyl amines gave α-amino esters which were hydrolyzed to the corresponding acids and then coupled with various amines using carbodiimide chemistry to obtain α-amino amides <b>VI</b>. Final Boc group deprotection in methanolic HCl gave target compounds <b>2</b>–<b>4</b> and <b>6</b>–<b>10</b> (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Table S1).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Morpholine Side-Chain Replacement Analogues of Compound <b>1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) H<sub>2</sub>SO<sub>4</sub>, MeOH, 50 °C, 3 h, 77%; (b) Boc<sub>2</sub>O, THF, rt, 16 h, 77%; (c) acrylic acid, <i>N</i>-methylmorpholine, isobutyl chloroformate, DCM, 0 °C, 30 min and then add intermediate <b>II</b> at reflux over 30 min, stir 2 h, 34%; (d) tri-<i>o</i>-tolylphosphine, TEA, Pd<sub>2</sub>(dba)<sub>3</sub>, DMF, 100 °C, 6 h, 60%; (e) MsCl, TEA, DCM, 0 °C, 1 h; (f) R<sup>2</sup>NH<sub>2</sub> or R<sup>2</sup>R<sup>3</sup>NH, TEA, DCM, rt, 16 h; (g) LiOH, MeOH, rt, 5 h, 81% (three steps); (h) R<sup>1</sup>NH<sub>2</sub>, PyBrop, DIPEA, DCM, rt, 12 h; (i) 1.25 M HCl in MeOH, rt, 2 h, and then NaHCO<sub>3</sub>, 20–25% (two steps).</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> General Synthesis of R<sup>1</sup> Analogues of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>)</h3><div class="NLM_p">Synthesis of R<sup>1</sup> analogues of the <i>rac</i>-(<i>trans</i>-3,4)-disubstituted pyrrolidine series began with iminium ylide cycloaddition<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> of commercially available (<i>E</i>)-3-(4-bromophenyl)acrylic acid ethyl ester with sarcosine and paraformaldehyde to obtain pyrrolidine <b>VII</b>. Pyrrolidine <b>VII</b> was then reacted with acrylamide <b>III</b> using Heck coupling conditions as described above to synthesize cinnamide <b>VIII</b>. Hydrolysis of <b>VIII</b> using lithium hydroxide in methanol furnished the corresponding carboxylic acid which was then coupled with various amine/aniline building blocks and subsequently subjected to methanolic HCl to provide final compounds <b>11</b>–<b>42</b> as racemic mixtures of <i>trans</i> diastereomers.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0004.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of R<sup>1</sup> Analogues of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) Sarcosine, paraformaldehyde, toluene, reflux, 4 h, 46%; (b) <b>III</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 110 °C (sealed tube), 16 h, 87%; (c), LiOH, MeOH, rt, 16 h; (d) R<sup>1</sup> (substituted amine, aniline, or aminopyridine), HATU, TEA, DMF:DCM (1:10), rt, 3 h, 88% (two steps); (e) 1 N HCl in MeOH, rt, 4 h, 43%.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> General Synthesis of R<sup>2</sup> Analogues of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series via Alkylation, Epoxidation, Reductive Amination, Acylation, or Nucleophilic Substitution (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>)</h3><div class="NLM_p">Synthesis of R<sup>2</sup> analogues of the <i>rac</i>-(<i>trans</i>-3,4)-disubstituted pyrrolidine series in which the R<sup>2</sup> group was installed via alkylation, epoxidation, reductive amination, acylation, or nucleophilic substitution began with orthoboric acid-catalyzed iminium ylide cycloaddition of commercially available (<i>E</i>)-3-(4-bromophenyl)acrylic acid ethyl ester with <i>N</i>-(methoxymethyl)-<i>N</i>-(trimethylsilylmethyl)benzylamine in DCM to afford <i>N</i>-benzylpyrrolidine <b>IX</b>. Except for the synthesis of final compound <b>58</b>, where <i>N</i>-benzyl pyrrolidine <b>IX</b> was directly reacted with acrylamide <b>III</b> to obtain cinnamide <b>XII</b>, the <i>N</i>-benzyl group was then removed in a two-step process involving conversion of the <i>N</i>-benzyl group to trichloroethyl carbamate with trichloroethyl chloroformate followed by carbamate cleavage mediated by zinc metal in acetic acid to provide pyrrolidine <b>X</b>. Pyrrolidine <b>X</b> was then converted to R<sup>2</sup>-substituted pyrrolidine <b>XI</b> via one of five general methods depending on the nature of the R<sup>2</sup> substituent. To obtain the R<sup>2</sup>-substituted pyrrolidines <b>XIa</b> corresponding to final compounds <b>45</b>, <b>46</b>, <b>49</b>, <b>50</b>, and <b>54</b>, pyrrolidine <b>X</b> was alkylated under microwave conditions with potassium iodide as catalyst with various R<sup>2</sup>X building blocks, where X was a suitable leaving group such as bromide or tosylate. Overalkylation was generally not an issue in these cases presumably due to the deactivating effect that these R<sup>2</sup> substituents had on the nucleophilicity of the resulting tertiary pyrrolidine amine. To obtain the corresponding pyrrolidine intermediate (<b>XIb</b>) for compound <b>51</b>, pyrrolidine <b>X</b> underwent S<sub>N</sub>2 reaction with 2,2-dimethyloxirane in ethanol. For pyrrolidine intermediates <b>XIc</b> corresponding to final compounds <b>43</b>, <b>44</b>, and <b>59</b>, pyrrolidine <b>X</b> underwent reductive amination with acetaldehyde, acetone, and tetrahydropyran-4-one, respectively. For the pyrrolidine intermediate <b>XId</b> corresponding to final compounds <b>55</b> and <b>56</b>, pyrrolidine <b>X</b> was acylated with triphosgene and the appropriate alkyl amine. For the pyrrolidine intermediate <b>XIe</b> corresponding to final compound <b>57</b>, pyrrolidine <b>X</b> underwent nucleophilic aromatic substitution with 2-chloropyrimidine. Pyrrolidine intermediates <b>XIa</b>–<b>e</b> from all five general methods were then coupled with acrylamide <b>III</b> via the Heck conditions described above to obtain cinnamide esters <b>XII</b>. Esters <b>XII</b> were then hydrolyzed to the corresponding cinnamide acids with lithium hydroxide in methanol followed by HATU-mediated condensation with 4-chloroaniline and final deprotection in methanolic HCl to afford final compounds <b>43</b>–<b>46</b>, <b>49</b>–<b>51</b>, and <b>54</b>–<b>59</b> as racemic mixtures of <i>trans</i> diastereomers.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of R<sup>2</sup> Analogues of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series via Alkylation, Epoxidation, Reductive Amination, Acylation, or Nucleophilic Aromatic Substitution<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) <i>N</i>-(Methoxymethyl)-<i>N</i>-(trimethylsilylmethyl)benzylamine, orthoboric acid, DCM, rt, 2 d then 45 °C for 2 h, 71%; (b) 2,2,2-trichloroethyl chloroformate, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 60 °C, 3 h, 66%; (c) Zn, acetic acid, rt, 3 h; (d) for alkylation (R<sup>2</sup>X where X is halide, tosylate, or mesylate): K<sub>2</sub>CO<sub>3</sub>, KI, R<sup>2</sup>X, DMF, 130 °C (microwave) 20 min; for epoxidation (2,2-dimethyloxirane): 2,2-dimethyloxirane, EtOH, 80 °C (sealed tube), 5 h, 96%; for reductive amination (R<sup>2</sup> = R<sup>3</sup>R<sup>4</sup>CH after reduction from R<sup>3</sup>R<sup>4</sup>CO): R<sup>3</sup>R<sup>4</sup>CO, NaBH(OAc)<sub>3</sub>, catalytic acetic acid, MeOH, rt, 16 h, 64%; for acylation: triphosgene and R<sup>3</sup>R<sup>4</sup>NH (R<sup>2</sup> = CONR<sup>3</sup>R<sup>4</sup>), DIPEA, THF, 0 °C to rt, 12 h; for nucleophilic aromatic substitution (2-chloropyrimidine): 2-chloropyrimidine, DIPEA, <i>i</i>-PrOH, 130 °C (microwave), 10 min, 60%; (e) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 100 °C, 3 h; (f) R<sup>1</sup> (substituted amine, aniline, or aminopyridine), LiOH, MeOH, rt, 16 h; (g) HATU, TEA, DMF:DCM (1:10), rt, 3 h; (h) 1 N HCl in MeOH, rt, 4 h.</p></p></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Analogue <b>53</b> (R<sup>2</sup> = 2-hydroxy-1,1-dimethylethyl) (Scheme <a class="ref internalNav" href="#sch4" aria-label="4">4</a>)</h3><div class="NLM_p">Synthesis of compound <b>53</b> began with one-pot hydrolysis and Boc protection of pyrrolidine ester <b>X</b> to obtain the pyrrolidine acid <b>XIII</b>. Carbodiimide-mediated coupling of 4-chloroaniline with acid <b>XIII</b> furnished pyrrolidine amide <b>XIV</b>. Amide <b>XIV</b> was deprotected in methanolic HCl to yield pyrrolidine amine HCl salt, which was then reacted with 2-bromo-2-methylpropionic acid methyl ester in DMF and subsequently reduced with lithium borohydride in THF to the corresponding alcohol <b>XV</b>. Heck coupling of acrylamide <b>III</b> with <b>XV</b> provided the corresponding cinnamide which was then deprotected in methanolic HCl to give compound <b>53</b>.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0006.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series Analogue <b>53</b> (R<sup>2</sup> = 2-hydroxy-1,1-dimethylethyl)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) NaOH, CH<sub>3</sub>CN, rt, 2 h then Na<sub>2</sub>CO<sub>3</sub>, Boc<sub>2</sub>O, rt, 16 h, 50%; (b) 4-chloroaniline, HOBt, EDCI, DCM, rt, 16 h; (c) 1.25 M HCl in MeOH, rt, 3 h then Na<sub>2</sub>CO<sub>3</sub>; (d) 2-bromo-2-methylpropionic acid methyl ester, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 5 h; (e) LiBH<sub>4</sub>, THF, rt, 2 h (17%, four steps); (f) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 100 °C, 3 h; (g) 1.25 M HCl, MeOH, rt, 3 h (24%, two steps).</p></p></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Analogue <b>52</b> (R<sup>2</sup> = <i>tert</i>-butyl) (Scheme <a class="ref internalNav" href="#sch5" aria-label="5">5</a>)</h3><div class="NLM_p">Synthesis of compound <b>52</b> began with iminium ylide cycloaddition of commercially available (<i>E</i>)-3-(4-bromophenyl)acrylic acid ethyl ester with <i>N</i>-<i>tert</i>-butylglycine hydrochloride salt and paraformaldehyde in refluxing toluene to obtain <i>N</i>-<i>tert</i>-butylpyrrolidine <b>XVI</b>. Heck coupling of <b>XVI</b> with acrylamide <b>III</b> then furnished cinnamide ester <b>XVII</b>. Ester <b>XVII</b> was hydrolyzed with lithium hydroxide in THF, then the corresponding acid underwent HATU-mediated coupling with 4-chloroaniline followed by Boc deprotection with methanolic HCl to obtain final compound <b>52</b>.</div><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0007.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Analogue <b>52</b> (R<sup>2</sup> = <i>tert</i>-butyl)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) <i>N</i>-<i>tert</i>-butylglycine hydrochloride, paraformaldehyde, toluene, reflux, 4 h, 12%; (b) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 110 °C (sealed tube), 12 h, 53%; (c) LiOH, THF, rt, 16 h; (d) 4-chloroaniline, HATU, TEA, DCM, rt, 16 h, 66% (two steps); (e) 1.25 M HCl in MeOH, rt, 4 h, 27%.</p></p></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Analogues <b>47</b> and <b>48</b> (R<sup>2</sup> = 1,1-difluoroethyl and 1,1,1-trifluoroethyl) (Scheme <a class="ref internalNav" href="#sch6" aria-label="6">6</a>)</h3><div class="NLM_p">Synthesis of compounds <b>47</b> and <b>48</b> began with acylation of pyrrolidine <b>X</b> with difluoroacetyl chloride and triflic anhydride, respectively, followed by selective amide reduction with borane in THF to yield the corresponding <i>N</i>-alkyl pyrrolidines <b>XVIII</b>. <i>N</i>-Alkyl pyrrolidines <b>XVIII</b> were then treated with lithium hydroxide followed by PyBrop-mediated coupling of 4-chloroaniline to the resulting acids to obtain pyrrolidine amides <b>XIX</b>. These pyrrolidine intermediates <b>XIX</b> then underwent Heck coupling with acrylamide <b>III</b> followed by Boc deprotection in methanolic HCl to afford final compounds <b>47</b> and <b>48</b>.</div><figure id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0008.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Analogues <b>47</b> and <b>48</b> (R<sup>2</sup> = 1,1-difluoroethyl and 1,1,1-trifluoroethyl)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) For Y = CF<sub>3</sub>: triflic anhydride, TEA, DCM, 0 °C, 30 min and then to rt, 1 h, 68%; for Y = CF<sub>2</sub>H: difluoroacetyl chloride, DCM, rt, 1 h, 97%; (b) for Y = CF<sub>3</sub>: BH<sub>3</sub>, THF, 0 °C, 2 h, 30%; for Y = CF<sub>2</sub>H: BH<sub>3</sub>, THF, 60 °C, 5 h, 63% (c) NaOH H<sub>2</sub>O/MeOH, rt, 4 h; (d) 4-chloroaniline, PyBrop, DIPEA, DCM, rt, 16 h, 90%; (e) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 110 °C (sealed tube), 4 h, 86%; (f) TFA, DCM, rt, 2 h.</p></p></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine 5-Vinylpyridin-2-yl Linker Analogue <b>64</b> (Scheme <a class="ref internalNav" href="#sch7" aria-label="7">7</a>)</h3><div class="NLM_p">Synthesis of compound <b>64</b> began with isopropyl magnesium chloride-mediated formylation of 5-bromo-2-iodopyridine with DMF followed by HWE reaction to furnish substituted ethyl acrylate <b>XX</b>. Iminium ylide cycloaddition of ethyl acrylate <b>XX</b> with sarcosine and paraformaldehyde in refluxing toluene produced the <i>N</i>-methyl pyrrolidine ester <b>XXI</b>. Intermediate <b>XXI</b> was then converted to final compound <b>64</b> in a series of transformations analogous to those for the conversion of intermediate <b>XI</b> to <i>rac</i>-(<i>trans</i>-3,4)-disubstitued pyrrolidine R<sup>2</sup> analogues (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>).</div><figure id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0009.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine 5-Vinylpyridin-2-yl Linker Analogue <b>64</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) <i>i</i>-PrMgCl, THF, −20 °C, 2 h and then add DMF, 0 °C to rt, 1 h, 90%; (b) triethyl phosphonoacetate, TEA, LiBr, CH<sub>3</sub>CN, rt, 16 h, 67%; (c) sarcosine, paraformaldehyde, toluene, reflux, 4 h, 77%; (d) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 100 °C, 3 h, 75%; (e) LiOH, THF/H<sub>2</sub>O (3:1), rt, 5 h, 86%; (f) 4-chloroaniline, HATU, TEA, DCM, rt, 16 h, 54%; (g) 2 N HCl, MeOH, rt, 4 h.</p></p></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine 6-Vinylpyridin-3-yl Linker Analogue <b>61</b> (Scheme <a class="ref internalNav" href="#sch8" aria-label="8">8</a>)</h3><div class="NLM_p">Synthesis of compound <b>61</b> began with protection of 4-bromobenzaldehyde as its [1,3]dioxolane acetal followed by Heck reaction with ethyl acrylate to furnish substituted ethyl acrylate <b>XXII</b>. Iminium ylide cycloaddition of <b>XXII</b> with sarcosine and paraformaldehyde in refluxing toluene yielded <i>N</i>-methylpyrrolidine <b>XXIII</b>. Removal of the acetal protecting group using lithium chloride in a DMSO–H<sub>2</sub>O solvent system under microwave conditions furnished the corresponding aldehyde which was then converted to (<i>E</i>)-3-pyridin-2-ylacrylic acid <i>tert</i>-butyl ester <b>XXIV</b> via HWE reaction with diethyl <i>ter</i>t-butyl phosphonoacetate. Selective lithium hydroxide-mediated hydrolysis of the ethyl ester followed by HATU-mediated coupling to 4-chloroaniline gave pyrrolidine amide <b>XXV</b>. Cleavage of the <i>tert</i>-butyl ester with TFA followed by coupling to 1,2-diaminobenzene using HATU furnished target compound <b>61</b>.</div><figure id="sch8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0010.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine 6-Vinylpyridin-3-yl Linker Analogue <b>61</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) Ethane-1,2-diol, TsOH, toluene, reflux, 6 h; (b) ethyl acrylate, Pd<sub>2</sub>(dba)<sub>3</sub>, tris-<i>o</i>-tolylphosphine, TEA, 110 °C, 3 h; (c) <i>N</i>-methylglycine, paraformaldehyde, toluene, reflux, 6 h; (d) LiCl, DMSO/H<sub>2</sub>O (3:1), 150 °C (microwave), 10 min; (e) diethyl <i>tert</i>-butyl phosphonoacetate, <i>n</i>-butyllithium, THF, −78 to 0 °C, 2 h; (f) LiOH, THF/H<sub>2</sub>O (3:1), rt, 3 h; (g) 4-chloroaniline, HATU, TEA, DCM, rt, 24 h; (h) TFA, DCM, 0 °C to rt, 2 h; (i) 1,2-diaminobenzene, HATU, TEA, DCM, rt, 12 h.</p></p></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Synthesis of Compounds <b>5</b>, <b>60</b>, <b>62</b>, and <b>63</b></h3><div class="NLM_p last">Schemes and experimental methods for the synthesis of <i>rac</i>-(<i>trans</i>-3,4)-disubstituted pyrrolidine 2-fluoro-4-vinylphenyl linker analogue <b>60</b>, <i>rac</i>-(<i>trans</i>-3,4)-disubstituted pyrrolidine 5-vinylpyrazin-2-yl linker analogue <b>62</b>, and <i>rac</i>-(<i>trans</i>-3,4)-disubstituted pyrrolidine 5-vinylpyradizin-2-yl linker analogue <b>63</b> can be found in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. A scheme and experimental methods for the enantioselective synthesis of compounds <b>5</b> (3<i>R</i>,4<i>S</i>) and <b>65</b> (3<i>S</i>,4<i>R</i>), the corresponding enantiomers of <i>rac-</i>(<i>trans</i>-3,4)-disubstituted pyrrolidine <b>15</b>, are available in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Lead Optimization Process and Parameters</h3><div class="NLM_p last">We selected the p21 reporter gene assay and liver microsome stability assay as a primary screening system to guide our medicinal chemistry efforts because the goal of our optimization campaign was to improve compound metabolic stability while maintaining in-cell activity against the class I HDACs comparable or superior to entinostat (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B), a clinical stage <i>o</i>-aminoanilide HDAC inhibitor. Representative compounds from structurally distinct subseries were also tested in a solubility assay to focus optimization efforts on subseries with comparable or improved solubility relative to <b>1</b> so as to broaden future formulation and dosing administration options. Compounds with good in-cell HDAC inhibitory activity, comparable or improved solubility relative to <b>1</b>, and improved in vitro metabolic stability were then assessed in single-dose intravenous and oral pharmacokinetic studies in rodents (mostly BALB/C nude mice). Compounds with good pharmacokinetic profiles were then further prioritized through cytochrome P450 (CYP) and human ether-à-go-go-related gene (hERG) counter-screening. The optimized leads arising from this process were then spot-checked in a cell-free biochemical HDAC inhibition panel before proceeding to in vivo efficacy and biomarker studies.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Modification of the Morpholine Side Chain</h3><div class="NLM_p">Metabolite identification studies indicated that the 2-morpholin-4-ylethylamino side chain is a major site of oxidative metabolism for compound <b>1</b> (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details). We initially explored two approaches to reduce oxidative metabolism at this site: (1) replacement of the morpholine moiety; (2) conformational restriction of the alkyl amino side chain. Whereas replacement of the morpholine moiety with a simple primary hydroxyl group to generate <b>2</b> led to improved microsomal stability (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), not surprisingly the removal of the basic amine also lowered compound solubility. Alternatively, side-chain cyclization (compound <b>3</b>) combined with addition of a solubilizing dialkylamino group to give <b>4</b> led to both improved in vitro metabolic stability and vastly improved solubility (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Notably, five-membered ring cyclization was found to be superior to four- or six-membered ring cyclization in terms of p21 induction activity (see compounds <b>6</b>–<b>10</b> in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Table S1). Despite the improved in vitro metabolic stability and solubility of <b>4</b>, this compound still had high in vivo clearance, albeit lower than compound <b>1</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Therefore, our subsequent efforts focused on a distinct <i>trans</i>-3,4-disubstituted pyrrolidine-based scaffold inspired by our previous experience with the <i>trans</i>-3,4-disubstituted pyrrolidine core in another lead optimization program and supported by docked conformational alignment modeling with <b>4</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><figure id="fig3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docked conformational alignment by Glide software (see Experimental Section) of a prototypical <i>trans</i>-3,4-disubstituted pyrrolidine scaffold core (blue sticks) with that of the pyrrolidine-based side-chain analogues of <b>4</b> (yellow sticks). For clarity, HDAC1 homology model used for docking is not shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Systematic <i>trans</i>-3,4-Disubstituted Pyrrolidine Optimization</h3><div class="NLM_p">As briefly summarized above, due to the difficulties we encountered in optimizing either compound <b>1</b> or its side-chain-cyclized analogues, we focused the majority of our efforts on systematically exploring and optimizing the <i>trans</i>-3,4-disubstituted pyrrolidine scaffold. In this respect, we sequentially varied the R<sup>1</sup>, R<sup>2</sup>, and linker (L) positions (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>) to identify an optimal combination of R<sup>1</sup>, R<sup>2</sup>, and linker substructures to achieve both good in-cell target inhibition and a pharmacokinetic profile suitable for in vivo efficacy and biomarker studies.</div><figure id="fig4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Generalized structure of the <i>trans</i>-3,4-disubstituted pyrrolidine series.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> R<sup>1</sup> Position SAR</h3><div class="NLM_p">Our optimization effort began with the exploration of various R<sup>1</sup> substituents while fixing R<sup>2</sup> and the linker region as a simple methyl group and the standard cinnamide linker, respectively. Compounds with aromatic R<sup>1</sup> substituents containing electron-withdrawing groups in the para position of the aromatic ring generally had reasonable in vitro microsomal stability (compounds <b>12</b>, <b>13</b>, <b>16</b>, <b>19</b>, <b>20</b> in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) whereas the <i>p</i>-fluoro and <i>p</i>-chloro substituted compounds <b>12</b> and <b>13</b> were preferred for obtaining both moderate microsomal stability and in-cell target activity. In contrast, meta and ortho monosubstituted aromatic R<sup>1</sup> substituents were well-tolerated in terms of target activity but did not result in good microsomal stability (compounds <b>21</b>–<b>25</b>). Combining meta or ortho substituents with <i>p</i>-fluoro or <i>p</i>-chloro substituents (compounds <b>26</b>–<b>33</b>) did not lead to any obvious improvement in activity or microsomal stability as compared to the monosubstituted <i>p</i>-fluoro or <i>p</i>-chloro derivatives <b>12</b> and <b>13</b>. Simple pyridinyl R<sup>1</sup> analogues <b>35</b> and <b>37</b> also displayed reasonable activity and stability, whereas pyridinyl R<sup>1</sup> analogue <b>34</b> had decreased in-cell target activity. Finally, we explored various aliphatic R<sup>1</sup> substituents but found that none of these compounds (<b>39</b>–<b>42</b>) had both robust in-cell target activity and good microsomal stability. In summary, these results indicated that the optimal choice of substituent for the R<sup>1</sup> position is either a <i>p</i>-fluoro- or <i>p</i>-chloro-substituted phenyl ring. Accordingly, in our subsequent studies of the R<sup>2</sup> and linker region SAR, we generally fixed the R<sup>1</sup> position as one of these two substituents.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure–Activity and Structure–Property Relationships for <i>trans</i>-3,4-Disubstituted Pyrrolidines in the R<sup>1</sup> Region</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0013.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0014.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">P21 RP3 is the relative fold increase in luciferase signal in the p21 reporter gene assay compared to clinical HDACi entinostat (both test compound and comparator were evaluated at 3 μM in duplicate or triplicate) and normalized by internal GFP transfection control.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">HLM and MLM are the human and mouse hepatic clearance in units of mL/min/kg as calculated from the intrinsic microsome clearance (see Experimental Section for details).</p></div></div><div></div></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> R<sup>2</sup> Position SAR</h3><div class="NLM_p">While fixing R<sup>1</sup> and the linker region as a <i>p</i>-chloro-substituted phenyl ring and cinnamide linker, respectively, we explored a range of saturated and unsaturated R<sup>2</sup> substituents (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), with the goal of probing both the steric requirements of this region as well as the effect of modulating the basicity of the pyrrolidine nitrogen on the overall properties of the resulting analogues. In terms of steric requirements, we found that a relatively wide range of substituents were well-tolerated in this region, from simple methyl substitution (compound <b>13</b>), to <i>tert</i>-butyl (compound <b>52</b>), pyrimidinyl (compound <b>57</b>), and tetrahydropyranyl (compound <b>59</b>) substitutions. Moreover, electron-withdrawing alkoxy substituents (for example, compounds <b>49</b>, <b>51</b>, <b>53</b>) were also generally well-tolerated in terms of both in-cell target activity and microsomal stability. In contrast, acyl substituents such as those found in compounds <b>55</b> and <b>56</b> were less well-tolerated, either leading to suboptimal p21 induction activity or poorer microsomal stability, respectively. Overall, we were unable to fully distinguish beneficial from harmful R<sup>2</sup> modifications on the basis of in vitro assays alone, so we selected representative R<sup>2</sup> analogues with good p21 induction activity and microsomal stability for single-dose pharmacokinetic evaluation (see below).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure–Activity and Structure–Property Relationships for <i>trans</i>-3,4-Disubstituted Pyrrolidines in the R<sup>2</sup> Region</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0015.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0016.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">P21 RP3 is the relative fold increase in luciferase signal in the p21 reporter gene assay compared to clinical HDACi entinostat (both test compound and comparator were evaluated at 3 μM in duplicate or triplicate) and normalized by internal GFP transfection control.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">HLM and MLM are the human and mouse hepatic clearance in units of mL/min/kg as calculated from the intrinsic microsome clearance (see Experimental Section for details).</p></div></div><div></div></div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Linker Region SAR</h3><div class="NLM_p">While fixing R<sup>1</sup> and R<sup>2</sup> as a <i>p</i>-chloro-substituted phenyl ring and simple methyl group, respectively, we explored various fine-tuning modifications of the cinnamide phenyl ring in the linker region (L), including a nitrogen walk (compounds <b>61</b>–<b>64</b>) and fluorine substitution (compound <b>60</b>) (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Because these relatively conservative changes were all well-tolerated in terms of both in-cell target activity and microsomal stability, we selected representative compounds for single-dose pharmacokinetic evaluation (see below).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structure–Activity and Structure–Property Relationships for Linker Variation (L) within the <i>trans</i>-3,4-Disubstituted Pyrrolidine Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0017.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0018.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">P21 RP3 is the relative fold increase in luciferase signal in the p21 reporter gene assay compared to clinical HDACi entinostat (both test compound and comparator were evaluated at 3 μM in duplicate or triplicate) and normalized by internal GFP transfection control.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">HLM and MLM are the human and mouse hepatic clearance in units of mL/min/kg as calculated from the intrinsic microsome clearance (see Experimental Section for details).</p></div><div class="footnote" id="t3fn3"><sup>Table c</sup><p class="last">Estimated to be at least 100% based on 4-F-phenyl analogue and SAR correlation between 4-F-phenyl and 4-Cl-phenyl subseries (data not shown).</p></div></div><div></div></div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Pharmacokinetic Evaluation of Representative <i>trans</i>-3,4-Disubstituted Pyrrolidines</h3><div class="NLM_p">We selected 14 <i>trans</i>-3,4-disubstituted pyrrolidines, with various R<sup>1</sup>, R<sup>2</sup>, and linker region moieties for intravenous and oral single-dose pharmacokinetic studies (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>) based on the aforementioned in vitro SAR results. Compounds <b>12</b>, <b>44</b>, <b>63</b>, and <b>64</b> had particularly high in vivo clearance generally associated with low to moderate exposure. We presumed that the high in vivo clearance of these compounds was due to clearance pathways acting on these compounds that were not present in the in vitro microsomal assay system. Compounds <b>53</b> and <b>59</b> had moderate in vivo clearance but relatively low bioavailability resulting in moderate exposure. Compounds <b>26</b> and <b>52</b> had moderate bioavailability but also moderately high in vivo clearance. In the case of compound <b>52</b>, we presumed that the relatively high lipophilicity of this compound (cLogP = 5.4) relative to the other analogues under investigation contributed to its higher-than-expected in vivo clearance. The remaining compounds, <b>13</b>, <b>5</b> (3<i>R</i>,4<i>S</i>) and its corresponding enantiomer <b>65</b> (3<i>S</i>,4<i>R</i>), <b>49</b>, <b>51</b>, and <b>60</b>, demonstrated reasonable overall pharmacokinetic properties, resulting in good exposure and prompting us to conduct more detailed in vitro selectivity assays to select potential preclinical leads for in vivo efficacy and biomarker studies.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Pharmacokinetic Parameters for Selected <i>trans</i>-3,4-Disubstituted Pyrrolidines with Acceptable p21 Activity and Mouse Liver Microsome Stability</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC (h·ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">CL (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub> (L)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">170</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char=".">624</td><td class="colsep0 rowsep0" align="char" char=".">127</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">324</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">2946</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">8.6</td><td class="colsep0 rowsep0" align="char" char=".">75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b> (3<i>R</i>,4<i>S</i>)</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="char" char=".">893</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">6316</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65</b> (3<i>S</i>,4<i>R</i>)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">450</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">2544</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">9.2</td><td class="colsep0 rowsep0" align="char" char=".">89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">262</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="char" char=".">911</td><td class="colsep0 rowsep0" align="char" char=".">53</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">331</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">2077</td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="char" char=".">957</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">7096</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="char" char=".">58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char=".">1100</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">5653</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">113</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">813</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">337</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">743</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="left">0.17</td><td class="colsep0 rowsep0" align="char" char=".">315</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">1002</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="char" char=".">828</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">3566</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63</b></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">116</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="char" char=".">19</td><td class="colsep0 rowsep0" align="char" char=".">9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64</b></td><td class="colsep0 rowsep0" align="left">0.17</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">204</td><td class="colsep0 rowsep0" align="char" char=".">79</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">19</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Compounds were dosed in nude mice at 10 mpk po and 2 mpk iv except for <b>12</b> and <b>64</b> which were dosed at 5 mpk po and 1 mpk iv and <b>63</b> which was dosed in rat rather than nude mouse. More details for PK studies can be found in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>.</p></div></div></div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> In Vitro Selectivity Panel Assessment for Selected <i>trans</i>-3,4-Disubstituted Pyrrolidines</h3><div class="NLM_p">To select compounds not only suitable for in vivo efficacy and biomarker studies but also having good potential to ultimately progress to the clinic, we evaluated the CYP and hERG inhibition profiles for the aforementioned compounds <b>5</b>, <b>65</b>, <b>49</b>, <b>51</b>, and <b>60</b> that had reasonable pharmacokinetic properties (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). Compounds <b>49</b> and <b>51</b> demonstrated significant inhibition (less than 10 μM) of several CYP isoforms and were therefore deprioritized. Compounds <b>5</b>, <b>65</b>, and <b>60</b> were found to have acceptable CYP inhibition profiles and were thus assessed in an HDAC isoform panel selectivity assay. Consistent with our expectation of compounds with the <i>o</i>-aminoanilide zinc-binding group, these molecules were confirmed to be selective inhibitors of HDAC isoforms 1, 2, and 3. Moreover, compound <b>5</b> was particularly active toward HDAC1. Therefore, based on the overall in vitro and in vivo properties of <b>5</b>, <b>65</b>, and <b>60</b>, we selected compound <b>5</b> as our lead for in vitro and in vivo efficacy and biomarker studies as well as further preclinical development.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. In Vitro Selectivity of Representative <i>trans</i>-3,4-Disubstituted Pyrrolidines toward HDACs, CYPs, and hERG</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="11" align="center">HDAC isoform IC<sub>50</sub><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="5" align="center">CYP isoform IC<sub>50</sub><a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">1</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">3</th><th class="colsep0 rowsep0" align="center">4</th><th class="colsep0 rowsep0" align="center">5</th><th class="colsep0 rowsep0" align="center">6</th><th class="colsep0 rowsep0" align="center">7</th><th class="colsep0 rowsep0" align="center">8</th><th class="colsep0 rowsep0" align="center">9</th><th class="colsep0 rowsep0" align="center">10</th><th class="colsep0 rowsep0" align="center">11</th><th class="colsep0 rowsep0" align="center">3A4</th><th class="colsep0 rowsep0" align="center">2D6</th><th class="colsep0 rowsep0" align="center">2C9</th><th class="colsep0 rowsep0" align="center">2C19</th><th class="colsep0 rowsep0" align="center">1A2</th><th class="colsep0 rowsep0" align="center">hERG IC<sub>50</sub><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">0.06</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65</b></td><td class="colsep0 rowsep0" align="char" char=".">0.14</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="char" char=".">0.20</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">0.22</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">ND</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">ND</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">HDAC enzyme inhibition assays were performed by Reaction Biology Corporation according to their standard assay protocols (<a href="http://www.reactionbiology.com" class="extLink">www.reactionbiology.com</a>, see Experimental Section for additional information). ND = not determined.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">CYP isoform inhibition assays were performed using the LCMS method.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">hERG inhibition assay was performed using PatchXpress methodology.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> All IC<sub>50</sub> measurements are reported in units of μM. ND = not determined.</p></div></div></div></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> In Vitro Efficacy and Mechanistic Studies of Compound <b>5</b></h3><div class="NLM_p">While the intended primary indication for our class I HDAC-selective inhibitors was hepatocellular carcinoma, we decided to use both hepatocellular carcinoma (HCC) and gastric cancer (GC) cell lines to increase our confidence in the basic in vitro efficacy and cell-based mechanistic and functional studies for <b>5</b>. Therefore, the antiproliferative activity of <b>5</b> was evaluated in a panel of HCC and GC cell lines, using the clinical HDAC inhibitor entinostat and the FDA-approved HDAC inhibitor vorinostat as comparators. As assessed by the concentration required to inhibit 90% of cell growth at 72 h, <b>5</b> demonstrated better antiproliferative activity than either entinostat or vorinostat in all cell lines tested (Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>). Mechanistically, the antiproliferative effect of <b>5</b> correlated with the dose-dependent induction of endogenous acetylated histone H3 and p21 after 24 h treatment within a concentration range consistent with the GI<sub>90</sub> observed in the same cell line (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). Functionally, induction of endogenous acetylated histone H3 and p21 corresponded with a dose-dependent decrease in S phase cells at 24 h (consistent with G<sub>1</sub>/G<sub>2</sub> cell cycle arrest typically observed for HDAC inhibitors) and subsequent induction of apoptosis at 48 h (representative cell lines, Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C). Taken together, these results demonstrate that the potent antiproliferative efficacy of <b>5</b> is due to HDAC-dependent in-cell induction of histone acetylation leading to concomitant increase in the expression of endogenous cyclin-dependent kinase inhibitor p21, ultimately resulting in cell cycle arrest and apoptosis.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Antiproliferative Activity of Compound <b>5</b> versus Clinical Comparators Entinostat and Vorinostat in Asian Hepatocellular Carcinoma and Gastric Cancer Cell Lines</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center" char="."><b>5</b> (GI<sub>90</sub>)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">entinostat (GI<sub>90</sub>)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">vorinostat (GI<sub>90</sub>)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMMC-7721</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BEL-7402</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BEL-7404</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SKHEP1</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">QGY-7701</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SGC-7901</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">9.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MGC8–03</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">8.9</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BGC-823</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HGC-27</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">GI<sub>90</sub> is the concentration (reported in μM) required to inhibit cell growth by 90% after 72 h compound treatment using the water-soluble tetrazolium (WST) dye assay with eight-point dose–response (half-log serial dilutions in triplicate). See Experimental Section for further details.</p></div></div></div><figure id="fig5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0019.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mechanistic and functional effects of compound <b>5</b> in representative hepatocellular carcinoma and gastric cancer cell lines. (A) Dose-dependent induction of endogenous p21 and acetylated histone H3 in response to 24 h treatment with clinical HDACi comparator mocetinostat or compound <b>5</b> in SGC-7901 cells. (B) Dose-dependent induction of cell cycle arrest by compound <b>5</b> in SGC-7901 cells at 24 h. (C) Induction of apoptosis at 48 h by compound <b>5</b> at 5 μM in SMMC-7721 cells. See Experimental Section for details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> In Vivo Efficacy and PK/PD Studies of <b>5</b></h3><div class="NLM_p">Having demonstrated that <b>5</b> can potently inhibit HCC and GC cancer cell growth in vitro, we next set out to assess its tumor growth inhibitory effects in subcutaneous cell line and primary tumor-derived xenograft models.</div><div class="NLM_p">We first evaluated <b>5</b> in the HCC cell line-derived xenograft model BEL-7404. At an oral dose of 50 mg/kg once daily or 100 mg/kg every other day, <b>5</b> significantly inhibited the growth of established BEL-7404 tumors (tumor growth inhibition rate >60%) with acceptable body weight loss (less than 10% relative to the mean starting weight of the mice). Terminal pharmacokinetic analysis indicated that both dosing schedules resulted in total trough concentrations of <b>5</b> in the tumor that were 4-fold above the in vitro GI<sub>90</sub> for the corresponding BEL-7404 cell line. This corresponded to consistent pharmacodynamic induction of acetylated histone H3 in the tumor from zero to 24 h after administration of the last dose of <b>5</b>, as assessed by Western blot (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). Taken together, these results demonstrate that <b>5</b> has significant in vivo antitumor activity and, while not ruling out unforeseen off-target effects, suggest that this efficacy is linked to target engagement in the tumor as assessed by pharmacokinetic and pharmacodynamic measurements.</div><figure id="fig6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0020.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. In vivo antitumor efficacy and PK/PD correlation for compound <b>5</b>. (A) Effect of compound <b>5</b> on growth of an established BEL-7404 tumor in nude mice (<i>n</i> = 9–10) over four weeks. (B) Concentration of compound <b>5</b> in tumor tissue over 24 h period subsequent to the last dose. (C) Acetylated histone H3 induction in tumor tissue sampled at various time points after sacrificing mice subsequent to the last dose of compound <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With preliminary in vivo efficacy results for <b>5</b> in hand, we proceeded to conduct a pilot study with <b>5</b> to assess the feasibility of using primary tumor-derived xenograft models to identify candidate predictive biomarkers that could indicate which tumors (or patients) are more likely to respond to treatment with <b>5</b>, with the future goal of developing a companion diagnostic for patient selection. We chose to conduct this pilot study with primary tumor-derived subcutaneous xenograft models because, compared to traditional cell line-derived subcutaneous xenograft models, primary tumor-derived models better reflect the histology of the original patient tumor and are presumably therefore more likely to resemble the genetic and epigenetic status of the original tumor.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><div class="NLM_p">We selected three patient-derived HCC tumor models with different associated clinical and molecular characteristics to assess response toward <b>5</b> and to identify candidate predictive biomarkers (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Table S5). Each model was treated with <b>5</b> at 100 mpk every other day (the efficacious dose in cell line-derived liver cancer xenografts) for approximately four weeks. In regards to biomarker hypothesis generation, while we recognized that global expression and methyl-ome profiling are ideal for unbiased biomarker discovery, practical considerations led us to choose instead the candidate gene approach for our pilot study. Thus, the promoter CpG methylation status and mRNA expression level for 22 candidate genes (Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>) were assessed on treatment days 0, 7, 15 and the last day of the study (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for more details on the study design) using Methyl-Profiler PCR arrays allowing for parallel analysis of up to 24 genes at a time (our rationale for the selection of these 22 candidate genes is detailed below). Histone-H3 acetylation and pharmacokinetic parameters were assessed by immunohistochemistry and compound <b>5</b> bioanalysis, respectively, on the last day of the study.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Twenty-Two Candidate Genes Evaluated as Potential Biomarkers for Tumor Response Prediction</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">candidate gene category description<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">gene ID</th><th class="colsep0 rowsep0" align="center">function<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">methylation frequency in HCC patients,<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a> %</th><th class="colsep0 rowsep0" align="center">number of patients evaluated<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">detected in microarray?</th><th class="colsep0 rowsep0" align="center">up-regulated by <b>5</b> in microarray?</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TSGs methylated in patients; up-regulated by <b>5</b></td><td class="colsep0 rowsep0" align="left">CACNA1G</td><td class="colsep0 rowsep0" align="left">proliferation</td><td class="colsep0 rowsep0" align="left">20–40</td><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCND2</td><td class="colsep0 rowsep0" align="left">cell cycle</td><td class="colsep0 rowsep0" align="left">60–80</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CDKN1C</td><td class="colsep0 rowsep0" align="left">cell cycle</td><td class="colsep0 rowsep0" align="left">20–40</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CDKN2B</td><td class="colsep0 rowsep0" align="left">cell cycle</td><td class="colsep0 rowsep0" align="left">40–60</td><td class="colsep0 rowsep0" align="left">301</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CTGF</td><td class="colsep0 rowsep0" align="left">adhesion</td><td class="colsep0 rowsep0" align="left">0–20</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">FN1</td><td class="colsep0 rowsep0" align="left">adhesion</td><td class="colsep0 rowsep0" align="left">40–60</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SOCS1</td><td class="colsep0 rowsep0" align="left">apoptosis</td><td class="colsep0 rowsep0" align="left">60–80</td><td class="colsep0 rowsep0" align="left">250</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">THBS1</td><td class="colsep0 rowsep0" align="left">adhesion</td><td class="colsep0 rowsep0" align="left">0–20</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TIMP3</td><td class="colsep0 rowsep0" align="left">invasion</td><td class="colsep0 rowsep0" align="left">20–40</td><td class="colsep0 rowsep0" align="left">238</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TSGs methylated in patients; not up-regulated by <b>5</b></td><td class="colsep0 rowsep0" align="left">RASSF1</td><td class="colsep0 rowsep0" align="left">cell cycle</td><td class="colsep0 rowsep0" align="left">60–80</td><td class="colsep0 rowsep0" align="left">648</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TFPI2</td><td class="colsep0 rowsep0" align="left">invasion</td><td class="colsep0 rowsep0" align="left">40–60</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CDKN2A</td><td class="colsep0 rowsep0" align="left">cell cycle</td><td class="colsep0 rowsep0" align="left">40–60</td><td class="colsep0 rowsep0" align="left">1380</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CDH1</td><td class="colsep0 rowsep0" align="left">adhesion</td><td class="colsep0 rowsep0" align="left">20–40</td><td class="colsep0 rowsep0" align="left">574</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TSGs methylated in patients; not detected in microarray</td><td class="colsep0 rowsep0" align="left">SPINT2</td><td class="colsep0 rowsep0" align="left">cell movement</td><td class="colsep0 rowsep0" align="left">80–100</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HIC1</td><td class="colsep0 rowsep0" align="left">cell cycle</td><td class="colsep0 rowsep0" align="left">80–100</td><td class="colsep0 rowsep0" align="left">67</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PYCARD</td><td class="colsep0 rowsep0" align="left">apoptosis</td><td class="colsep0 rowsep0" align="left">20–40</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">up-regulated by <b>5</b>; no patient methylation information available</td><td class="colsep0 rowsep0" align="left">IGFBP3</td><td class="colsep0 rowsep0" align="left">apoptosis</td><td class="colsep0 rowsep0" align="left">not known</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CEBPA</td><td class="colsep0 rowsep0" align="left">differentiation</td><td class="colsep0 rowsep0" align="left">not known</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CDKN1A</td><td class="colsep0 rowsep0" align="left">cell cycle</td><td class="colsep0 rowsep0" align="left">not known</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SDC2</td><td class="colsep0 rowsep0" align="left">proliferation</td><td class="colsep0 rowsep0" align="left">not known</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">Y</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">oncogenes down-regulated by <b>5</b></td><td class="colsep0 rowsep0" align="left">WNT5A</td><td class="colsep0 rowsep0" align="left">growth</td><td class="colsep0 rowsep0" align="left">not known</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">down-regulated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CFLAR</td><td class="colsep0 rowsep0" align="left">apoptosis</td><td class="colsep0 rowsep0" align="left">not known</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">down-regulated</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup>a</sup><p class="last">The candidate genes were grouped into five categories according to their promoter CpG island methylation status in HCC patient tissues, their response to treatment with compound <b>5</b> in in vitro microarray studies, their detect-ability in our microarray studies, and their functional classification as tumor suppressor genes (TSGs) or oncogenes (OGs).</p></div><div class="footnote" id="t7fn2"><sup>b</sup><p class="last">Microarray data were analyzed by Ingenuity Pathway Analysis.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></p></div><div class="footnote" id="t7fn3"><sup>c</sup><p class="last">Information from PubMeth public database of promoter CpG methylation profiling in cancer tissues.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div></div></div><div class="NLM_p">The 22 candidate genes were selected based upon the results of microarray gene expression profiling of <b>5</b> in liver cancer cell lines, qRT-PCR confirmation of treatment-responsive genes, and a public database of liver cancer tumor suppressor genes found to be methylated frequently in patients<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for further details). Broadly, the candidate genes were grouped into five categories: (1) tumor suppressor genes (TSGs) with frequent CpG island promoter methylation in patient tissue and up-regulated by <b>5</b> in microarray profiling studies; (2) TSGs with frequent CpG island promoter methylation in patient tissue and not up-regulated by <b>5</b> in microarray profiling studies; (3) TSGs with frequent CpG island promoter methylation in patient tissue but not detected in microarray profiling studies due to low hybridization signal; (4) TSGs not known to be methylated in patient tissue but up-regulated by <b>5</b> in microarray profiling studies; (5) candidate oncogenes (OGs) down-regulated by <b>5</b> in microarray profiling studies (Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>). We hypothesized that responsive tumors would have a qualitatively distinct epigenetic signature from nonresponsive tumors and that such a signature, if confirmed through functional studies and a larger tumor cohort, could potentially be used to select patients more likely to respond to treatment with <b>5</b>.</div><div class="NLM_p">Patient tumor models B and C demonstrated a significant response to <b>5</b> (∼50% tumor growth inhibition) while patient tumor model A did not respond at all to compound treatment (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A). PK/PD analysis revealed that the tumor exposure of <b>5</b> and the induction of acetylated histone H3 resulting from target engagement (as measured by immunohistochemistry over multiple time points) were somewhat lower in the nonresponding model (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C). This result suggests that the apparent lack of response toward <b>5</b> for model A may have been in part or in whole due to differences in compound exposure and target engagement in this model compared to the other two models.</div><figure id="fig7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0021.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo efficacy with PK/PD correlation for compound <b>5</b> in three patient tumor-derived xenograft models (top row = patient A; middle row = patient B; bottom row = patient C): (A) tumor growth curves for patients A–C treated with vehicle (15% Kleptose, pH 4.5 buffered with tartaric acid) or compound <b>5</b> Q2D PO 100 mpk. (B) Tumor concentration of compound <b>5</b> vs time for patients A–C (log 10 scale). Each square represents the concentration of <b>5</b> measured from a single mouse tumor. (C) Acetylated histone H3 IHC score (0–3) vs time for patients A–C (representative IHC sections scored as 0, 1, 2, or 3 available in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure S6). Each square represents the IHC score from a single mouse tumor. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for further detail.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Analysis of the CpG methylation status of the 22 candidate genes in the three primary tumor models revealed a significant difference in the methylation pattern between the responding models and the nonresponding model. In particular, in the responding models B and C, only one and two genes had CpG promoter methylation, respectively. In contrast, in the nonresponding model A, seven out of the twenty-two candidate genes were methylated (Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a>). These results suggest that the severity of baseline epigenetic silencing of these candidate (mostly tumor suppressor) genes may be predictive of tumor response to <b>5</b>, or at least may contribute to the refractory nature of nonresponding model A. Hypothetically, if the nonresponsiveness of model A were functionally confirmed to be due to heavy promoter DNA methylation, then one potential approach to overcome this therapeutic resistance would be to treat patients with such heavy baseline methylation with a combination of HDAC and DNA methyltransferase inhibitors.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Promoter CpG Island Methylation Status for Genes Methylated in at Least One out of Three Patients on Day 0, Start of Weeks 2 and 3, and Last Day of Study</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">gene ID<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">patient A<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">patient B<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">patient C<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CACNA1G</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CCND2</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CDKN1C</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SOCS1</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RASSF1</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">X</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TFPI2</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SPINT2</td><td class="colsep0 rowsep0" align="left">X</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">X</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup>a</sup><p class="last">Only 7 out of 22 candidate genes are listed (the remaining 15 genes, CDKN2B, CTGF, FN1, THBS1, TIMP3, CDKNA2, CDH1, HIC1, PYCARD, IGFBP3, CEBPA, CDKN1A, SDC2, WNT5A, CFLAR, were not methylated in any of the three patients).</p></div><div class="footnote" id="t8fn2"><sup>b</sup><p class="last">“X” = methylated as determined by methylation-specific PCR; “–” = not methylated as determined by methylation-specific PCR. Methylation status did not change over the course of the study. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for details.</p></div></div></div><div class="NLM_p">With regard to the ability of <b>5</b> to reactivate epigenetically silenced tumor suppressor genes, one hypermethylated TSG in one responder, SPINT2, was significantly up-regulated upon treatment with <b>5</b>. While a few other genes were up-regulated in both responding models at one or more time points (e.g., TIMP3, p21/CDKN1A), only the SPINT2 gene was both hypermethylated and responsive to treatment with <b>5</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). Intriguingly, SPINT2 is an inhibitor of the HGF/MET pathway and has been identified as a TSG in certain cancers.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Future follow-up studies to these pilot experiments should evaluate the utility of SPINT2 as a potential biomarker in additional primary HCC tumor models as well as probe the functional consequences of SPINT2 reactivation. Validation of the SPINT2 biomarker would suggest a rational means of selecting patients for combination therapy with <b>5</b> and MET kinase inhibitors.</div><figure id="fig8" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0022.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. mRNA expression levels of candidate genes with absolute fold change greater than two at any of three time points (day 7, day 15, and last day of study) in the three patient models. CCND2, CDKN2B, FN1, RASSF1, CDKN2A, HIC1, CEBPA, SDC2, and CFLAR are not shown because these genes did not change significantly throughout the course of the experiment. Expression levels were determined by qRT-PCR, normalized to GAPDH, and reported as fold-change relative to day 0 (expression level before treatment). See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for further details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In summary, the pilot epigenetic biomarker study of <b>5</b> in three primary liver cancer xenograft models indicated a potential correlation between baseline promoter methylation status of key tumor suppressor genes and tumor response, with heavy promoter methylation potentially predictive of poor response and suggesting combination therapy with DNA methyltransferase inhibitors. Furthermore, in one responder, a specific tumor suppressor gene with CpG island hypermethylation, SPINT2, was reactivated in response to compound treatment. The expression and promoter DNA methylation status of this gene deserves further assessment as a potential biomarker for response prediction and for combination therapy of <b>5</b> with MET kinase inhibitors. These preliminary findings should be interpreted in the context of the PK/PD data, which indicated somewhat lower compound <b>5</b> tumor exposure and target modulation for the nonresponding model.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15415" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15415" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Through a combination of scaffold hopping and structure–property relationship studies, we have developed an orally bioavailable HDAC inhibitor with selectivity toward the class I HDAC family and particular potency toward HDAC1. We demonstrated that this compound is more potent at inhibiting liver cancer cell growth in vitro as compared to clinical HDAC inhibitors entinostat and vorinostat and that its antiproliferative effects correlate to markers of in-cell target engagement such as acetylated histone H3 and p21/CDKN1A. Moreover, these in vitro effects translated into significant efficacy in both cell-based and patient tumor-derived xenograft models that appeared to correlate well to compound exposure levels and to changes in the same pharmacodynamic markers used to assess in-cell target engagement. One out of the three patient tumor models did not respond to treatment with our optimized HDAC inhibitor and, compared to the two responding models, had much higher levels of DNA methylation in the promoter regions of a panel of tumor suppressor genes known to be frequently epigenetically silenced in liver cancer patients. Furthermore, in one of the responding models, we identified SPINT2, an endogenous inhibitor of the progrowth HGF/MET pathway, as an epigenetically silenced gene whose expression could be reactivated by treatment with our HDAC inhibitor. Future research should seek to confirm the potential clinical utility of our candidate DNA methylation and tumor suppressor gene biomarkers in a larger cohort of patient tumor-derived models, through functional studies of these biomarkers, and in the context of combination therapy with either MET kinase inhibitors, DNA methyltransferase inhibitors, or the current standard of care in HCC, sorafenib.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05026" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05026" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Molecular Docking</h3><div class="NLM_p last">OPLS_2005 force field of MacroModel<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was used to optimize the initial geometries of the compounds with a final root-mean-square gradient of 0.01 kcal/mol/Å. The default values of the optimization parameters and thresholds were retained. Glide XP was used for compound docking because of its robust energetic terms on metal–ligand interaction.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> All torsion angles of compound were released to freely rotate, and 10000 docking runs were adopted during the XP docking with NE2 atom of His178 and zinc ion as docking constraints after trying different constraint combinations. Docking poses were clustered based on heavy atom RMSD values (1 Å), and top-ranked poses of each compound were selected and viewed graphically within the Maestro of Schrödinger program (Schrödinger, L.L.C., New York, NY), and finally displayed in Pymol (Schrödinger, L.L.C., New York, NY).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Microsomal Stability Assay</h3><div class="NLM_p last">Each incubated mixture contained 0.5 mg/mL liver microsome (human or mouse), 100 mM potassium phosphate buffer (pH = 7.4), 10 mM NADPH, and 1 μM of test compound in a total volume of 400 μL. After prewarming at 37 °C for 10 min, the NADPH was added to initiate the reaction. The reaction was terminated after 0, 3, 6, 9, 15, and 30 min by adding 150 μL of 100 ng/mL of tolbutamide (internal standard) in ice-cold methanol into 300 μL of incubation mixtures. Samples were then centrifuged at 4000 rpm for 10 min at 4 °C. The supernatant was then analyzed by LC-MS/MS.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Solubility Assay</h3><div class="NLM_p last">DMSO stock solution (10 mM) of compound was prepared and divided into two portions. For one portion, the DMSO was evaporated to dryness and the remaining was redissolved in 50 mM phosphate buffer (pH 6.5). The mixture was stirred and shaken for 1 and 2 h, respectively. After the mixture was allowed to stand overnight, the solution was filtered before HPLC analysis. The other portion was used for the calibration curve preparation by diluting the DMSO stock solution using the same PBS buffer mentioned above to a series of solutions with concentration range of 50–500 μM.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Cytochrome P450 Inhibition Assay</h3><div class="NLM_p last">A cocktail method that enables simultaneous incubation and measurement of compound inhibitory activity against CYP isoforms CYP3A4, CYP2D6, CYP2C9, CYP2C19, and CYP1A2 was developed with modification from previously reported methods.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Each incubated mixture contained 0.125 mg/mL human liver microsome (protein content), 5 mM MgCl<sub>2</sub>, 100 mM potassium phosphate buffer (pH= 7.4), substrate cocktail, various concentrations of test compound, and 2 mM NADPH in a total volume of 200 μL. The final concentrations of each CYP substrate were at the reported literature <i>K</i><sub>m</sub> values<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Table S2 for further details). Before addition of NADPH to initiate the reaction, mixtures were prewarmed at 37 °C for 10 min, and the reaction was terminated after 15 min of incubation by adding 100 μL of ice cold methanol containing 3 μM dextrorphan as an internal standard (IS). Samples were then centrifuged at 4000 rpm for 10 min at 4 °C. The supernatant was then analyzed by LC-MS/MS.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> hERG Inhibition Assay</h3><div class="NLM_p last">The inhibitory effect of compounds against the hERG channel was measured on an automated patch clamp system.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> CHO cells expressing hERG were loaded into the system. Cell solutions and buffer solutions were transferred to specially designed chips, where cells were attached to each well. Pressure changes were applied in each well until a gigaseal status and a whole cell configuration were achieved. After a brief stabilization process, a predefined pulse procedure was applied in each well, opening hERG channels and triggering tail currents. The peak of a tail current was measured. Compounds at various concentrations were then coincubated with the cell, and the same pulse procedure was repeated to generate tail current data. Amoxicillin and amitriptyline were employed as negative and positive controls, respectively. Generally, a compound was tested at a minimum of two concentrations in duplicate, and thus IC<sub>20</sub> or IC<sub>50</sub> can be estimated to evaluate compound inhibitory potential on hERG channels.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> PK Analyses</h3><div class="NLM_p last">All compounds were evaluated in BALB/C nude mice (15–20 g, <i>N</i> = 16 per arm) except for compound <b>63</b> which was tested in male Sprague–Dawley rats (250 ± 20 g; <i>N</i> = 3). Compounds were administered by iv bolus at 2 mg/kg in 5% DMSO, 40% PEG400, and 55% saline and/or PO (gavage) at 10 mg/kg in 0.445% microcrystalline cellulose microsuspension. In each species, blood samples (150 μL) were collected at eight time points (po) or nine time points (iv) into sodium heparin centrifuge tubes, and plasma samples were then isolated by centrifugation and stored at −20 °C prior to analysis. Plasma concentrations were determined by LCMS/MS and data were analyzed by noncompartmental methods using WinNonlin Pro (Pharsight Corp., Mountain View, CA).</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Biology General Comments</h3><div class="NLM_p last">Brief descriptions of the biological protocols used to generate data in this manuscript can be found below. For further details regarding experimental protocols, cell line origins, culture conditions, and reagent manufacturers, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> HDAC Enzyme Inhibition Assay</h3><div class="NLM_p last">HDAC enzyme inhibition assays were conducted by Reaction Biology Corporation as described (<a href="http://www.reactionbiology.com" class="extLink">www.reactionbiology.com</a>) using a 10-point dose response curve with half-log serial dilutions, fluorogenic peptide from p53 residues 379–382 (RHKKAc) at 50 μM as substrate, and trichostatin A as a positive control.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> p21 Reporter Gene Assay</h3><div class="NLM_p last">Compounds were tested for their ability to induce p21 gene expression using a reporter gene assay involving HeLa cells transfected with a p21 promoter-luciferase construct. The p21 promoter contained the Sp1/Sp3 binding site for HDAC but not the upstream p53 binding site. Briefly, the day before transfection, HeLa cells were seeded at 11 000 cells/well in a 96-well culture plate and incubated at 37 °C in 5% CO<sub>2</sub> overnight. For transfection, the medium was removed and replaced with 100 μL/well transfection media previously prepared as follows: 5 μL of serum-free DMEM, 0.15 μL of Fugene 6 reagent, 40 ng of p21-luc, 10 ng of GFP were mixed gently and incubated at room temperature for 30 min, and then 98 μL of DMEM (with 10% FBS, 1% penicillin, and streptomycin) was added to the DNA:Fugene 6 reagent complex and mixed gently. After the cells were incubated for 24 h at 37 °C in 5% CO<sub>2</sub>, fresh media and test compounds were added to the wells and the cells further incubated for 15 h at 37 °C in 5% CO<sub>2</sub>. Cells were lysed by adding 80 μL/well of a cell culture lysis reagent (Promega). A 50 μL amount of each lysate was taken for GFP detection using an excitation wavelength of 486 nm and detection at 527 nm. A 100 μL amount of Luciferase assay reagent (Promega) was then added to every 20 μL of cell lysate for luminometer detection.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Cell Proliferation (water-soluble tetrazolium dye) Assay</h3><div class="NLM_p last">Cells were seeded in 96-well culture plates (200 μL/well at different seeding concentrations depending on cell type) and incubated overnight at 37 °C in 5% CO<sub>2</sub>. After adding compound dilutions (typically eight-point half-log serial dilutions in triplicate) to the cells (DMSO concentration kept below 0.5%), the cells were incubated at 37 °C in 5% CO<sub>2</sub> for 72 h. The effects on proliferation were determined by addition of CCK-8 reagent (Dojindo) according to the manufacturer’s instruction, followed by incubation for 2 h at 37 °C in 5% CO<sub>2</sub>, and finally recording the absorbance at 450 nm using an ELISA plate reader.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Cell Cycle Analysis</h3><div class="NLM_p last">An amount of 1 × 10<sup>5</sup> SGC-7901 cells was cultured overnight prior to treatment at about 50% confluency. Cells were treated with indicated concentrations of mocetinostat or <b>5</b> for 24 h. Cells were trypsinized and fixed with chilled 70% ethanol for 1 h at 4 °C. The cells were then resuspended in 500 μL of propidium iodide (PI) solution (50 μg/mL PI, 0.1 mg/mL RNase A and 0.05% Triton X-100) and incubated for 1 h at room temperature. Cells were washed three times with PBS and suspended in 500 μL of PBS. PI positive cells were quantified using flow cytometer and CellQuest software (BD Biosciences).</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Apoptosis Assay</h3><div class="NLM_p last">An amount of 1 × 10<sup>5</sup> SMMC-7721 cells was treated with 5 μM of mocetinostat or <b>5</b> for 48 h. Cells were trypsinized, washed twice with cold PBS, and then resuspended in 1X Annexin V binding buffer. A 100 μL amount of the cell solution was incubated with 5 μL of FITC/Annexin V for 15 min at room temperature. Cell samples were resuspended in 400 μL of 1X Annexin V binding buffer, and FITC/Annexin V positive cells were quantified using flow cytometer and CellQuest software (BD Biosciences).</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Western Blotting for p21 and Acetylated Histone H3 from Cells</h3><div class="NLM_p last">An amount of 1 × 10<sup>6</sup> SGC-7901 cells was seeded overnight and then incubated with indicated concentrations of mocetinostat and <b>5</b> for 24 h. Cell extracts were prepared by lysing cultured cells into a mammalian protein extraction reagent supplemented with EDTA-free protease inhibitor on ice for 15 min. Supernatants were collected following centrifugation of lysed cells (15 000 g) for 10 min at 4 °C. For analysis of total cell lysates, 30 μg of total protein per sample was resolved by SDSPAGE and transferred onto nitrocellulose membrane. Membranes with immobilized proteins were probed with indicated antibodies. The reactive protein bands were visualized using enhanced chemiluminescence (ECL) detection system.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> BEL-7404 Xenograft Efficacy Study</h3><div class="NLM_p">The use and care of experimental animals was approved by the Institutional Animal Care and Use Committee, Roche R&D Center (China). Female BALB/c nu/nu nude mice aged 6–7 weeks were obtained from Beijing Academy of Military Medical Science, Green violet blue Lab Animal Technology Co. Ltd. (Beijing, China), and housed in specific pathogen-free (SPF) conditions. Animals were held for a minimum of three days for acclimation prior to beginning the study. BEL-7404 suspended in serum-free RPMI-1640 (Invitrogen, Carlsbad, CA) was injected into the right flank of the nude mouse at 2.0 × 10<sup>6</sup> cells/mouse. After the tumor was established and reached 100–150 mm<sup>3</sup>, the mice were randomized into 10 mice per group and started to receive dosing of compound <b>5</b> (either QD PO 50 mpk or Q2D PO 100 mpk prepared in (15%) Kleptose-tartaric buffer (pH = 4.5)). The tumor growth was recorded three times a week from the measurement of length (L) and width (W) with caliper and calculated as tumor volume (TV, mm<sup>3</sup>) = 0.5 × L × W × W (mm). The tumor growth inhibition (TGI) was calculated as TGI% = (1 – (mean tumor volume of the treatment group on the first day – mean tumor volume of the treatment group on the end day)/(mean tumor volume of the control group on the first day – mean tumor volume of the control group on the end day)) × 100%.</div><div class="NLM_p last">For PD biomarker evaluation, 50 mg of frozen tumor sample was dissected on dry ice and lysed in 500 μL of tissue protein extraction reagent (Thermo Scientific) supplemented with EDTA-free protease inhibitors in MagNA lyser green beads and homogenized at 5000 rpm for 4 × 25 s (4 °C) using MagNA lyser (Roche). Tissue proteins were obtained from the supernatant of tissue homogenate by centrifugation at 15 000 rpm for 15 min (4 °C) and analyzed by Western blot (50 μg of total protein per sample).</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Patient Tumor-Derived Xenograft Model and PK/PD Analysis</h3><div class="NLM_p last">The use and care of experimental animals was approved by the Institutional Animal Care and Use Committee, Roche R&D Center (China). Female BALB/c nu/nu nude mice aged 6–7 weeks were obtained from Shanghai SLAC Laboratory Animal Company (Shanghai, China) and housed in specific pathogen-free (SPF) conditions. The tumor fragments (3 × 3 × 3 mm<sup>3</sup>) derived from three HCC cancer patients were harvested for tumor inoculation. Under sterilized conditions, fragments were subcutaneously implanted into the right flank of nude mice for tumor development. Mice were randomized (<i>n</i> = 10 per dosing arm and 15 for biomarker satellite arm) and treatment initiated when tumors reached approximately 200 (150–250) mm<sup>3</sup> as indicated (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for more study design details). Measurements were taken in the same way as in the BEL-7404 model study. Single-dose administration was also given as indicated on the last day of treatment, and the tumors were isolated at assigned time points (0 to 24 h postdosing) and cut into two pieces. The first piece from each tumor was subjected to pharmacokinetic analysis (see <a class="ref internalNav" href="#sec5_6" aria-label="PK Analysis">PK Analysis</a>). The second piece from each tumor was subjected to immunohistochemistry. The 8 μm paraffin embedded tumor tissue slides were heated in a dry oven at 60 °C for 1 h and then deparaffinized and hydrolyzed by a Leica autostainer XL ST5010 at room temperature. Antigen retrieval was carried out at 95–99 °C in citrate buffer (0.01 M, pH 6.0) heated by a medical microwave at 92–98 °C for 5 min and then cooled to room temperature. Then peroxidase-blocking reagent (DAKO, DK-2600, Glostrup Denmark) was applied to quench the endogenous peroxidase. The slides were incubated in blocking serum for 20 min and then covered by 100 of μL rabbit antihuman Ac-H3 (Lys9) antibody (Cell Signaling #9671, 1:200 in TBS/T) at 4 °C overnight. The slides were covered by 100 μL of DAKO Envision+/HRP, rabbit/mouse after being washed thoroughly with TBS. One hundred microliters of substrate–chromogen solution (DAB) was applied to the slides and incubated at room temperature for 10 min. The slides were rinsed gently with tap water for 15 min. Counterstain and dehydration were performed with an XL ST5010 at room temperature. Finally, the slides were mounted with Permount mounting medium (Fisher Scientific, Miami, FL). The scoring was carried out by two pathologists independently (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for representative images and scores).</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> mRNA and Methylation Analysis by Real Time PCR</h3><div class="NLM_p">Human xenograft total RNA was extracted with TRIzol (Invitrogen), treated with DNAaseI (Progmega), and further purified by RNA clean kit (Qiagen) before PCR. The 22-gene customized qRT-PCR array was designed by SABiosciences. Genomic DNA control, positive PCR control, and reverse transcription control were included in the array to monitor the PCR quality. GAPDH and β-glucuronidase were housekeeping genes used as internal controls. The reaction system and conditions were suggested by SABiosciences protocols. Bio-Rad Chromo4 was used for the qPCR reaction and detection. Amplification plots and dissociation curves were checked for reaction validity. ΔCt values were calculated against the average of two housekeeping genes, and then the fold changes were calculated using lower expressed samples as control.</div><div class="NLM_p last">DNA methylation status was detected by using the customized “Methyl-Profiler” DNA methylation PCR array system (SABioscience). The experiments and analysis were performed by following the manufacturer’s manual. Briefly, the real time PCR primers for the 22 genes were designed, covering CpG islands in the DNA promoter regions. Four separate restriction enzyme digestion reactions were carried out to differentiate the methylation levels. Genomic DNA was extracted with a DNeasy kit (Qiagen) before the restriction digestions with the Methyl-Profiler enzyme kit (SABiosciences). The real-time PCR was performed under the suggested conditions from SABiosciences on a Bio-Rad Chromo4. The relative amount of each methylated DNA fraction is calculated using the standard ΔCT method, normalizing the amount of DNA in each digestion to the total amount of input DNA in the mock digestion. The quality control for the digestions was checked by the refractory DNA percentage during the analysis. Methylation superarray data were analyzed following Web site instructions (<a href="http://www.sabiosciences.com/dna_methylation_data_analysis.php" class="extLink">http://www.sabiosciences.com/dna_methylation_data_analysis.php</a>).</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Synthetic Chemistry General Comments</h3><div class="NLM_p last">All starting materials were obtained commercially and were used without further purification. All reported yields are of isolated products and are not optimized. All final compounds were purified by preparative HPLC on a reverse phase column using Waters XBridge OBD Phenyl (30 × 100 mm, 5 μm) column or OBD RP18 (30 × 100 mm, 5 μm) column. LC/MS spectra were obtained using a MicroMass Platform LC (Waters alliance 2795-ZQ2000). For further details regarding preparative HPLC and LCMS protocols, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. NMR spectra were obtained using a Bruker AVANCE 400 MHz NMR spectrometer. All final compounds have purities greater than 95% based upon LC/MS and <sup>1</sup>H NMR. Enantiomeric excess was determined by chiral HPLC (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details).</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> (2-Aminophenyl)carbamic Acid <i>tert</i>-Butyl Ester (<b>II</b>)</h3><div class="NLM_p last">To a solution of <i>o</i>-phenylenediamine (54.0 g, 0.500 mol) in THF (500 mL) was added (Boc)<sub>2</sub>O (109 g, 0.500 mol) in THF (150 mL) dropwise, and the mixture was stirred at room temperature overnight. After concentration under vacuum, the residue was diluted with ethyl acetate/petroleum ether = 1/4 (v/v) (150 mL), and the precipitate was collected. The mother liquor was concentrated, and the crude product was recrystallized with ethyl acetate/petroleum ether = 1/4 (v/v). The combined solids were dried in vacuo at 40 °C for 4 h. An off-white solid (80 g, 77%) was obtained. MS: calcd 209 (MH+), exp 209 (MH+). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ ppm 7.25 (m, 1H), 7.00 (m, 1H), 6.77 (m, 2H), 6.29 (broad m, 1H), 3.60 (broad m, 2H), 1.51 (s, 9H).</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> <i>N</i>-(2-(Acryloylamino)phenyl)carbamic Acid <i>tert</i>-Butyl Ester (<b>III</b>)</h3><div class="NLM_p last">To a solution of acrylic acid (2.50 g, 34.7 mmol) in dichloromethane (80 mL) at 0 °C was added <i>N</i>-methylmorpholine (4.73 g, 46.8 mmol), followed by isobutyl chloroformate (6.37 g, 46.8 mmol). After 30 min, a solution of (2-aminophenyl)carbamic acid <i>tert</i>-butyl ester (<b>II</b>) (5.80 g, 27.8 mmol) in dichloromethane (50 mL) was added dropwise to the refluxing reaction mixture over 30 min. After the reaction was completed (2 h later), the reaction mixture was allowed to cool down to room temperature, poured into ice–water, and extracted with dichloromethane (30 mL × 3). The organic layer was washed with water, dilute sodium bicarbonate solution, 0.1 M HCl, water, and brine in turn. The organic layer was dried over sodium sulfate, filtered, and concentrated in vacuo. The crude solid was recrystallized from ethyl acetate/petroleum ether = 1/4 (v/v) to obtain the desired product (2.5 g, 34%). MS: calcd 263 (MH+), exp 263 (MH+). <sup>1</sup>H NMR (<i>d</i><sub><i>6</i></sub>-DMSO, 400 MHz) δ ppm 9.66 (s, 1 H), 8.45 (s, 1 H), 7.47 - 7.63 (m, 2 H), 7.03 - 7.23 (m, 2 H), 6.42 - 6.57 (m, 1 H), 6.27 (dd, J = 17, 2 Hz, 1 H), 5.79 (dd, J = 10, 2 Hz, 1 H), 1.46 (s, 9 H).</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> (<i>E</i>)-{4-[2-(2-<i>tert</i>-Butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}hydroxyacetic Acid Methyl Ester (<b>IV</b>)</h3><div class="NLM_p">To a solution of (4-bromophenyl)hydroxyacetic acid (31.4 g, 136 mmol) in methanol (200 mL) was slowly added concentrated sulfuric acid (10 mL) dropwise, and then the reaction was heated to 50 °C for 3 h. The reaction was neutralized with sodium carbonate, concentrated by rotary evaporation, then extracted with ethyl acetate. Desired product <b>I</b> was recrystallized from hexanes/ethyl acetate (25.6 g, 77%) and used directly in the following step.</div><div class="NLM_p last">A mixture of <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>) (23 g, 87.7 mmol), methyl-2-(4-bromophenyl)-2-hydroxyacetate (25.6 g, 104.5 mmol), tri-<i>o</i>-tolylphosphine (2.8 g, 9.2 mmol), Et<sub>3</sub>N (35.8 g, 353.8 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (4.3 g, 4.7 mmol) in DMF (400 mL) was heated at 100 °C for 6 h under N<sub>2</sub> atmosphere and monitored by TLC. After being cooled to room temperature, the mixture was poured into a saturated aqueous solution of NH<sub>4</sub>Cl and extracted with ethyl acetate. The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, concentrated in vacuo, and purified by flash column chromatography (ethyl acetate/petroleum ether 1:2) to obtain pale yellow solid (<b>IV</b>) (22.4 g, 60%). MS: calcd 427 (MH+), exp 427 (MH+). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ ppm 9.71 (s, 1H), 8.48 (s, 1H), 7.64 (d, 2H, <i>J</i> = 8.0 Hz), 7.59 (d, 1H, <i>J</i> = 15.6 Hz), 7.57 (m, 1H), 7.48 (d, 2H, <i>J</i> = 8.0 Hz), 7.14 (m, 2H), 6.92 (d, 1H, <i>J</i> = 15.6, 6.17 (d, 1H, <i>J</i> = 5.2 Hz), 5.21 (d, 1H, <i>J</i> = 4.8 Hz), 3.64 (s, 3H), 1.47 (s, 9H).</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> (<i>E</i>)-{4-[2-(2-<i>tert</i>-Butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-((<i>S</i>)-3-hydroxypyrrolidin-1-yl)acetic Acid Methyl Ester (<b>V</b>)</h3><div class="NLM_p">To a solution of {4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}hydroxyacetic acid methyl ester (<b>IV</b>) (9.00 g, 21.1 mmol) and triethylamine (3.20 g, 31.6 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (135 mL) cooled to −5 °C was added dropwise methanesulfonyl chloride (3.14 g, 27.4 mmol) under N<sub>2</sub> atmosphere. The reaction was stirred at 0 °C until the starting material had been consumed according to TLC (about 1 h). The mixture was washed with water (90 mL) and brine (90 mL), dried with MgSO<sub>4</sub>, filtered, and evaporated in vacuo to obtain 11.2 g (quantitative yield) of {4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}methanesulfonyloxyacetic acid methyl ester as light yellow crystals which were used in the following step without further purification. MS: calcd 505 (MH+), exp 505 (MH+).</div><div class="NLM_p last">To a solution of {4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}methanesulfonyloxyacetic acid methyl ester (1.26 g, 2.5 mmol) and Et<sub>3</sub>N (0.76 g, 7.5 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (15 mL) was added (<i>S</i>)-3-hydroxypyrrolidine (0.27 g, 3 mmol). This mixture was stirred overnight at room temperature, diluted with CH<sub>2</sub>Cl<sub>2</sub> (50 mL), washed with saturated aqueous NaHCO<sub>3</sub> solution, water and brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to obtain a yellow solid (<b>V</b>) which was directly used in the next reaction without further purification. MS: calcd 496 (MH+), exp 496 (MH+).</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> (<i>E</i>)-<i>N</i>-(2-Aminophenyl)-3-{4-[(4-bromophenylcarbamoyl)-((<i>S</i>)-3-hydroxypyrrolidin-1-yl)methyl]phenyl}acrylamide (<b>3</b>)</h3><div class="NLM_p">To a solution of {4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-(3-hydroxypyrrolidin-1-yl)acetic acid methyl ester (<b>V</b>) (1.23 g, 2.5 mmol) in MeOH (30 mL) was added aqueous LiOH (1 N, 10 mL). This mixture was stirred for about 5 h at room temperature and then evaporated to remove most of the MeOH. The aqueous layer was acidified with concentrated HCl to pH 5–6. The acidified aqueous layer was extracted with ethyl acetate (3 × 30 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to obtain 0.85 g (81%) of {4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-(3-hydroxypyrrolidin-1-yl)acetic acid as a yellow solid product used directly in the following step. MS: calcd 482 (MH+), exp 482 (MH+).</div><div class="NLM_p last">To a solution of {4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-(3-hydroxypyrrolidin-1-yl)acetic acid (2.3 g, 4.78 mmol), PyBrop (3 g, 6.4 mmol) and DIPEA (1 mL, 5.68 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) was added 4-bromoaniline (1 g, 5.8 mmol). This mixture was stirred overnight at room temperature and then diluted with CH<sub>2</sub>Cl<sub>2</sub>, washed with saturated aqueous NaHCO<sub>3</sub> solution, water, and brine, dried with Na<sub>2</sub>SO<sub>4</sub>, and evaporated to obtain a yellow residue (<b>VI</b>). Hydrochloric acid in methanol (1.25 M, 30 mL) was added to the residue, the mixture was stirred for about 3 h, and then NaHCO<sub>3</sub> was added to the reaction system. After filtration of solids, the crude mixture was purified by preparative HPLC to obtain 520 mg of compound <b>3</b> as a pale-yellow solid (20%). MS: calcd 535 (MH+), exp 535 (MH+). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ ppm 7.83 (d, 2H, <i>J</i> = 8.0 Hz), 7.81 (d, 1H, <i>J</i> = 15.6 Hz), 7.75 (d, 2H, <i>J</i> = 8.0 Hz), 7.55 (d, 2H, <i>J</i> = 8.8 Hz), 7.49 (d, 2H, <i>J</i> = 8.8 Hz), 7.40–7.30 (m, 4H), 6.98 (d, 1H, <i>J</i> = 15.6 Hz), 5.24 (m, 1H), 4.60 (broad s, 1H), 4.00–3.80 (broad m, 2H), 3.60–3.20 (broad m, 2H), 2.35–2.15 (broad m, 2H).</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-methylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>VII</b>)</h3><div class="NLM_p last">A mixture of sarcosine (44.5 g, 0.5 mol, 2.5 equiv), paraformaldehyde (60 g, 2 mol, 10 equiv), and (<i>E</i>)-3-(4-bromophenyl)acrylic acid ethyl ester (50 g, 196 mmol, 1 equiv) was heated under reflux in toluene (500 mL). The H<sub>2</sub>O formed was removed with the aid of a Dean–Stark trap. After 4 h, the cooled mixture was filtered. The filtrate was concentrated. The residue was purified by chromatography on silica gel eluted with hexane–EtOAc to give 28 g of product (yield was 46%) and 26 g of starting material ethyl cinnamate (52%). MS: calcd (MH+) 312, exp (MH+) 312. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ ppm 7.48 (d, 1H, <i>J</i> = 8.4 Hz), 7.29 (d, 2H, <i>J</i> = 8.4 Hz), 4.15 (q, 2H, 6.8 Hz), 3.51 (m, 1H), 3.30 (m, 2H), 3.20 (m, 2H), 2.72 (s, 3H), 1.21 (t, 3H, 6.8 Hz).</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-Butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-methylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>VIII</b>)</h3><div class="NLM_p last">The mixture of compound <i>rac</i>-(<i>trans-</i>3,4)-4-(4-bromophenyl)-1-methylpyrrolidine-3-carboxylic acid ethyl ester (<b>VII</b>) (10.3 g, 33 mmol), <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>) (9.51 g, 36.3 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (300 mg, 0.33 mmol), and P(<i>o</i>-tolyl)<sub>3</sub> (1.0 g, 3.3 mmol) in DMF (50 mL) and TEA (9 mL, 66 mmol) was stirred at 110 °C under N<sub>2</sub> in a sealed tube overnight. LC-MS indicated that the reaction was completed. The cooled mixture was partitioned between water and ethyl acetate. The organic phase was dried and concentrated. The residue was purified by chromatography on silica gel eluted by dichloromethane to give 14.22 g of desired product (87%). MS: calcd (MH+) 494, exp (MH+) 494.</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> <i>rac</i>-(<i>trans-3,4</i>)-4-{4-[2-(2-Aminophenylcarbamoyl)vinyl]phenyl}-1-methylpyrrolidine-3-carboxylic Acid (4-fluorophenyl)amide (<b>12</b>)</h3><div class="NLM_p">Lithium hydroxide monohydrate (2.42 g, 57.6 mmol) in water (20 mL) was added to a solution of <i>rac</i>-(<i>trans-</i>3,4)-4-{4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-methylpyrrolidine-3-carboxylic acid ethyl ester (<b>VIII</b>) (14.22 g, 28.8 mmol) in methanol (80 mL). Then the mixture was stirred at room temperature overnight. LC-MS indicated that the starting material was consumed. The mixture was adjusted to pH = 6–8 with 6 N HCl. The solvent was removed to give crude product <i>rac</i>-(<i>trans-</i>3,4)-4-{4-[2-(2-aminophenylcarbamoyl)vinyl]phenyl}-1-methylpyrrolidine-3-carboxylic acid which was used in the next step without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ ppm 7.81 (d, 1H, <i>J</i> = 15.6 Hz), 7.76 (d, 2H, <i>J</i> = 7.6 Hz), 7.70 (d, 1H, <i>J</i> = 7.6 Hz), 7.62 (d, 1H, <i>J</i> = 7.6 Hz), 7.58 (d, 2H, <i>J</i> = 8.0 Hz), 7.36 (t, 1H, <i>J</i> = 7.2 Hz), 7.29 (t, 1H, <i>J</i> = 7.2 Hz), 6.97 (d, 1H, <i>J</i> = 15.6 Hz), 4.02–3.84 (m, 2H), 3.82–3.64 (m, 2H), 3.33–3.26 (m, 2H), 3.12 (s, 3H), 1.63 (s, 9H).</div><div class="NLM_p">HATU (13.3 g, 35 mmol) was added to a solution of compound <i>rac</i>-(<i>trans-</i>3,4)-4-{4-[2-(2-aminophenylcarbamoyl)vinyl]phenyl}-1-methylpyrrolidine-3-carboxylic acid (crude material, 28.8 mmol) and 4-fluorophenylamine (3.55 g, 32 mmol) in TEA (10 mL, 72 mmol), DMF (30 mL), and dichloromethane (300 mL) at rt. Then the mixture was stirred at rt for 3 h. LC-MS indicated that the reaction was completed. The mixture was partitioned between water and dichloromethane. The organic phase was dried and concentrated. The residue was purified by chromatography on silica gel to give 14.1 g of <i>rac</i>-[2-(3-{4-[(<i>trans</i>-3,4)-4-(4-fluorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]phenyl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester (88% over two steps). MS: calcd (MH+) 559, exp (MH+) 559.</div><div class="NLM_p last"><i>rac</i>-[2-(3-{4-[(<i>trans</i>-3,4)-4-(4-Fluorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]phenyl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester (14.5 g, 25.3 mmol) was dissolved in 150 mL of MeOH (with 1 N HCl) and stirred for 4 h. LC-MS indicated that the reaction was completed. The solvent was removed, and the residue was basified with TEA and purified by prep-HPLC to give 5 g of product <b>12</b> (43%). MS: calcd (MH+) 459, exp (MH+) 459. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ ppm 7.63–7.58 (m, 3H), 7.53–7.50 (m, 2H), 7.40 (d, 2H, <i>J</i> = 8.0 Hz), 7.21 (d, 1H, <i>J</i> = 7.6 Hz), 7.08–7.00 (m, 3H), 6.90–6.74 (m, 3H), 3.85 (m, 1H), 3.21–3.11 (m, 3H), 2.97–2.93 (m, 2H), 2.50 (s, 3H).</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> <i>rac</i>-(<i>trans</i>-3,4)-1-Benzyl-4-(4-bromophenyl)pyrrolidine-3-carboxylic Acid Ethyl Ester (<b>IX</b>)</h3><div class="NLM_p last">To a mixture of ethyl <i>trans</i>-4-bromocinnamate (0.95 g, 3.73 mmol) and <i>N</i>-(methoxymethyl)-<i>N</i>-(trimethylsilylmethyl)benzylamine (1.06 g, 4.48 mmol) in dichloromethane (10 mL) was added orthoboric acid (23 mg, 0.37 mmol), and the mixture was stirred at room temperature for 2 days and heated at 45 °C for 2 h. The resulting mixture was concentrated in vacuo, and the crude product was purified by flash chromatography (PET:EtOAc = 10:1) to get product as colorless oil (1.08 g, 71%). MS: calcd 388 (MH+), exp 388 (MH+).</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)pyrrolidine-3-carboxylic Acid Ethyl Ester (<b>X</b>)</h3><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-1-benzyl-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>IX</b>) (3.97 g, 10.2 mmol) in acetonitrile (40 mL) was added K<sub>2</sub>CO<sub>3</sub> (2.25 g, 16.3 mmol) followed by 2,2,2-trichloroethyl chloroformate (3.03 g, 14.3 mmol), and the mixture was heated at 60 °C for 3 h. The mixture was diluted with ethyl acetate and filtered. The filtrate was concentrated in vacuo, and the crude product was purified by flash chromatography (Pet:EtOAc = 10:1) to get <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-1,3-dicarboxylic acid 3-ethyl ester 1-(2,2,2-trichloroethyl) ester as a colorless oil (3.2 g, 66.2%). MS: calcd 472 (MH+), exp 472 (MH+).</div><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-1,3-dicarboxylic acid 3-ethyl ester 1-(2,2,2-trichloroethyl) ester (3.2 g, 6.76 mmol) in acetic acid (15 mL) was added Zn (1.28 g, 19.6 mmol) under Ar, and the mixture was stirred for 3 h at room temperature. The resulting mixture was diluted with dichloromethane and filtered. The filtrate was concentrated in vacuo. The crude product <b>X</b> (2.5 g) was obtained and used directly without further purification. MS: calcd 298 (MH+), exp (MH+) 298. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz), 7.45 (d, 2H, <i>J</i> = 8.4 Hz), 7.21 (d, 2H, <i>J</i> = 8.4 Hz), 4.09 (t, 2H, <i>J</i> = 7.2 Hz), 3.47–3.33 (m, 3H), 3.26–3.21 (m, 1H), 3.09–3.05 (m, 1H), 2.89–2.84 (m, 1H), 1.16 (t, 3H, <i>J</i> = 7.2 Hz).</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Representative Alkylation Procedure for R<sup>2</sup> Analogues of the 3,4-Disubstituted Pyrrolidine Series: <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-(2-fluoroethyl)pyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XIa</b>)</h3><div class="NLM_p last">A 20 mL microwave process vial was charged with <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>X</b>) (1.49 g, 5 mmol), K<sub>2</sub>CO<sub>3</sub> (1.38 g, 10 mmol), KI (41.5 mg, 0.25 mmol), and 1-bromo-2-fluoroethane (0.76 g, 6 mmol) in DMF (15 mL). The vial was sealed and heated at 130 °C for 20 min under microwave irradiation. After cooling, the solution was poured into water (30 mL), and the aqueous layer was extracted with ethyl acetate (3 × 50 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated. The crude product was purified by column chromatography (PE:EtOAc = 1:1) to give product as a yellow solid. MS: calcd 344 (MH+), exp 344 (MH+).</div></div><div id="sec5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Representative Epoxidation Procedure for R<sup>2</sup> Analogues of the 3,4-Disubstituted Pyrrolidine Series: <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-(2-hydroxy-2-methylpropyl)pyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XIb</b>)</h3><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>X</b>) (5.0 g, 16.78 mmol) in ethanol (60 mL) was added 2,2-dimethyloxirane (8 mL). This mixture was heated for about 5 h at 80 °C in a sealed tube until the starting material had been consumed and then evaporated in vacuo to give 6.0 g (96%) of crude product as a yellow oil. MS: calcd 370 (MH+), exp 370 (MH+).</div></div><div id="sec5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Representative Reductive Amination Procedure for R<sup>2</sup> Analogues of the 3,4-Disubstituted Pyrrolidine Series: <i>rac</i>-(<i>trans-</i>3,4)-4-(4-Bromophenyl)-1-isopropylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XIc</b>)</h3><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans-</i>3,4)-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>X</b>) (2.5 g, 8.39 mmol) in methanol were added acetone (9.73 g, 16.78 mmol), NaBH(OAc)<sub>3</sub> (3.56 g, 16.78 mmol), and catalytic acetic acid. This mixture was stirred at rt overnight until the starting material had been consumed. Water was added, and the organic phase was washed by brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give 1.8 g of product (yield 64%). MS: calcd 340 (MH+), exp 340 (MH+).</div></div><div id="sec5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Representative Acylation Procedure for R<sup>2</sup> Analogues of the 3,4-Disubstituted Pyrrolidine Series: <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-ethylcarbamoylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XId</b>)</h3><div class="NLM_p last">To a solution of DIPEA (0.87 mL, 5 mmol) in dry THF was added triphosgene (530 mg, 1.8 mmol) at 0 °C, the resulting solution was stirred for 5 min, and then a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>X</b>) (1.485 g, 5 mmol) in THF and DIPEA (0.87 mL, 5 mmol) was added dropwise. After the mixture was stirring for an additional 20 min, ethylamine (450 mg, 10 mmol) was added dropwise and stirred overnight at room temperature before it was poured into H<sub>2</sub>O (25 mL) and extracted with EtOAc (2 × 15 mL). The organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give a crude product which was purified by flash column chromatography to yield the title compound (1.14 g, 62%) as a white solid. MS: calcd 369 (MH+), exp 369 (MH+).</div></div><div id="sec5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Representative Nucleophilic Aromatic Substitution Procedure for R<sup>2</sup> Analogues of the 3,4-Disubstituted Pyrrolidine Series: <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[(<i>E</i>)-2-(2-Aminophenylcarbamoyl)vinyl]phenyl}-1-pyrimidin-2-ylpyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>XIe</b>)</h3><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans-</i>3,4)-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>X</b>) (466 mg, 1.56 mmol) in 2-propanol (6 mL) were added DIPEA (412 μL, 2.34 mmol) and 2-chloropyrimidine (228 mg, 2 mmol). The solution was heated to 130 °C for 10 min by microwave and then concentrated in vacuo and purified by flash chromatography to obtain 357 mg of product as an oil (60%). MS: calcd 376 (MH+), exp 376 (MH+).</div></div><div id="sec5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-Butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-(2-fluoroethyl)pyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XII</b>)</h3><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)-1-(2-fluoroethyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>XIa</b>) (2.0 g, 5.81 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (265.8 mg, 0.29 mmol), tri-(<i>o</i>-tolyl)phosphine (88 mg, 0.29 mmol), and Et<sub>3</sub>N (17.5 g, 17.4 mmol) in DMF (20 mL) was added <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>) (1.82 g, 6.97 mmol). This mixture was stirred at 100 °C for about 3 h until the starting material had been consumed, and then the reaction mixture was cooled and filtered. The solution was poured into water (40 mL) and extracted with ethyl acetate (3 × 60 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated. The crude product was purified by column chromatography (PE:EtOAc = 1:1) to give yellow solid product. MS: calcd 526 (MH+), exp 526 (MH+).</div></div><div id="sec5_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-Aminophenylcarbamoyl)vinyl]phenyl}-1-(2-fluoroethyl)pyrrolidine-3-carboxylic Acid (4-chlorophenyl)amide (<b>46</b>)</h3><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-(2-fluoroethyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>XII</b>) (1.05 g, 2.0 mmol) in MeOH/H<sub>2</sub>O (20 mL/6 mL) was added LiOH·H<sub>2</sub>O (252 mg, 6.0 mmol). This mixture was stirred at room temperature overnight and then evaporated to remove most of the MeOH. The mixture was extracted with ethyl acetate (3 × 50 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to get yellow solid product <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-(2-fluoroethyl)pyrrolidine-3-carboxylic acid. MS: calcd 498 (MH+), exp 498 (MH+).</div><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-(2-fluoroethyl)pyrrolidine-3-carboxylic acid (300 mg, 0.6 mmol), HATU (342 mg, 0.9 mmol), and Et<sub>3</sub>N (181.8 mg, 1.8 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (15 mL) was added 4-chloroaniline (114.3 mg, 0.9 mmol). The reaction was stirred at room temperature overnight. The mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub> (15 mL), washed with water (15 mL) and brine (15 mL), dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated in vacuo to obtain light yellow residue which was used without further purification. MS: calcd 607 (MH+), exp 607 (MH+).</div><div class="NLM_p last">Hydrochloric acid in methanol (1.25 M, 5 mL) was added to the <i>rac</i>-[2-(3-{4-[(<i>trans</i>-3,4)-4-(4-chlorophenylcarbamoyl)-1-(2-fluoroethyl)pyrrolidin-3-yl]phenyl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester residue, the mixture was stirred for about 3 h, and then NaHCO<sub>3</sub> was added to the reaction system. After filtration of solids, the crude mixture was purified by preparative HPLC to obtain light yellow solid. MS: calcd 507 (MH+), exp 507 (MH+). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz), 7.66 (d, 1H, <i>J</i> = 15.6 Hz), 7.60 (d, 2H, <i>J</i> = 8.0 Hz), 7.53 (d, 2H, <i>J</i> = 8.8 Hz), 7.42 (d, 2H, <i>J</i> = 8.0 Hz), 7.21 (d, 1H, <i>J</i> = 8.0 Hz), 7.08–7.04 (m, 3H), 6.90–6.82 (m, 2H), 6.78–6.74 (m, 1H), 4.69 (t, 1H, <i>J</i> = 4.8 Hz), 4.57 (t, 1H, <i>J</i> = 4.8 Hz), 3.84–3.78 (m, 1H), 3.30–3.16 (m, 3H), 3.03–2.87 (m, 4H).</div></div><div id="sec5_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)pyrrolidine-1,3-dicarboxylic Acid 1-<i>tert</i>-Butyl Ester (<b>XIII</b>)</h3><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester (<b>X</b>) (5.0 g, 16.8 mmol) in 20 mL of acetonitrile was added 20 mL of 2.5 M NaOH, and the resulted mixture was stirred at room temperature for 2 h. Then Na<sub>2</sub>CO<sub>3</sub> (3.56 g 33.6 mmol) and Boc<sub>2</sub>O (11 g, 50.4 mmol) were added, and the mixture was stirred at rt overnight. After removal of acetonitrile by vacuum, the residue was adjusted to pH 3. The solid was collected and triturated in hexane twice and dried by vacuum (50% yield). MS: calcd 370 (MH+), exp 370 (MH+).</div></div><div id="sec5_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> <i>rac</i>-(<i>trans</i>-3,4)-3-(4-Bromophenyl)-4-(4-chlorophenylcarbamoyl)pyrrolidine-1-carboxylic Acid <i>tert</i>-Butyl Ester (<b>XIV</b>)</h3><div class="NLM_p last">A suspension of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-1,3-dicarboxylic acid 1-<i>tert</i>-butyl ester (<b>XIII</b>) (3.1 g, 8.4 mmol), 4-chloroaniline (1.6 g, 12.6 mmol), HOBt (1.7 g, 12.6 mmol), and EDCI (2.4g, 12.6 mmol) in 50 mL of dichloromethane was stirred at rt overnight, and the reaction mixture was washed with 1 M NaOH, water, and brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to give yellow oil, which was used directly in the next step without purification. MS: calcd 479 (MH+), exp 479 (MH+).</div></div><div id="sec5_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-(2-hydroxy-1,1-dimethylethyl)pyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>XV</b>)</h3><div class="NLM_p"><i>rac</i>-(<i>trans</i>-3,4)-3-(4-Bromophenyl)-4-(4-chlorophenylcarbamoyl)pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester (<b>XIV</b>) was dissolved in 15 mL of HCl/MeOH (1.25 M) and stirred at rt for 3 h. The reaction mixture was treated with saturated Na<sub>2</sub>CO<sub>3</sub> solution and was extracted with dichloromethane. The organic layer was concentrated to give yellow oil, which was used without further purification. MS: calcd 379 (MH+), exp 379 (MH+).</div><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)pyrrolidine-3-carboxylic acid (4-chlorophenyl)amide (1.12 g, 2.98 mmol) in 5 mL of DMF were added K<sub>2</sub>CO<sub>3</sub> (1.24 g, 8.9 mmol) and 2-bromo-2-methylpropionic acid methyl ester (0.81 g, 4.47 mmol), and the mixture was heated at 50 °C for 5 h. The reaction mixture was diluted with water and extracted with EtOAc, and the organic layer was washed with water and brine. After removal of solvent, the residue was dissolved in 30 mL of dry THF under N<sub>2</sub> atmosphere, and then LiBH<sub>4</sub> (2 M in THF, 2.9 mL, 5.8 mmol) was added dropwise at room temperature over 5 min. The resulting mixture was kept at this temperature for 2 h and then quenched by adding water carefully. The mixture was extracted with EtOAc, and the crude product was purified by column chromatography. A 0.23 g amount of product <b>XV</b> was obtained with a yield of 17%. MS: calcd 451 (MH+), exp 451 (MH+).</div></div><div id="sec5_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-Aminophenylcarbamoyl)vinyl]phenyl}-1-(2-hydroxy-1,1-dimethylethyl)pyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>53</b>)</h3><div class="NLM_p last">The mixture of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)-1-(2-hydroxy-1,1-dimethylethyl)pyrrolidine-3-carboxylic acid (4-chlorophenyl)amide (0.23 g, 0.5 mmol) (<b>XV</b>), <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester <i>tert</i>-butyl ester (<b>III</b>) (0.14 g, 0.53 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (37 mg, 0.04 mmol), and tri-(<i>o</i>-tolyl)phosphine (27 mg, 0.08 mmol) in DMF (3 mL) and TEA (0.35 mL, 2.5 mmol) was stirred at 100 °C under N<sub>2</sub> in a sealed tube for 3 h. The cooled mixture was partitioned between water and ethyl acetate. The organic phase was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give crude yellow oil. The crude oil was dissolved in 3 mL of HCl/MeOH (1.25 M) and stirred at room temperature for 3 h. The resultant mixture was neutralized with NH<sub>3</sub>/EtOH solution and purified by prep-HPLC to obtain 66 mg yellow solid as product. Yield was 24% over two steps. MS: calcd 533 (MH+), exp 533 (MH+). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz), 10.09 (s, 1H), 9.36 (s, 1H), 7.50–7.61 (m, 5H), 7.33–7.39 (m, 5H), 6.84–6.94 (m, 2H), 6.75 (d, 1H, <i>J</i> = 8.0 Hz), 6.58 (t, 1H, <i>J</i> = 7.2 Hz), 4.93 (s, 2H), 4.46 (s, 1H), 3.62 (q, 1H, <i>J</i> = 7.6 Hz), 3.30 (m, 2H), 3.21–3.29 (m, 2H), 3.04 (q, 1H, <i>J</i> = 7.6 Hz), 2.78–2.88 (m, 2H), 1.03 (s, 3H), 1.01 (s, 3H)</div></div><div id="sec5_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-<i>tert</i>-butylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XVI</b>)</h3><div class="NLM_p last">A mixture of <i>N</i>-<i>tert</i>-butylglycine hydrochloride salt (0.9 g, 6.8 mmol), paraformaldehyde (1.5 g, 50 mmol), and ethyl cinnamate (0.9 g, 35 mmol) was heated under reflux in toluene (30 mL). The H<sub>2</sub>O formed was removed with the aid of a Dean–Stark trap. After 4 h, the cooled mixture was filtered. The filtrate was concentrated. The residue was purified by chromatography on silica gel eluted with hexane–EtOAc to give 300 mg of product (yield was 12%). MS: calcd 354 (MH+), exp 354 (MH+).</div></div><div id="sec5_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-Butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-<i>tert</i>-butylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XVII</b>)</h3><div class="NLM_p last">A mixture of <b>XVI</b> (0.5 g, 1.4 mmol), <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>) (0.5 g, 1.9 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (100 mg, 0.01 mmol), and P(<i>o</i>-tolyl)<sub>3</sub> (100 mg, 0.33 mmol) in DMF (5 mL) and TEA (1 mL) was stirred at 110 °C under N<sub>2</sub> in a sealed tube overnight. LC-MS indicated that the reaction was completed. The cooled mixture was partitioned between water and ethyl acetate. The organic phase was dried and concentrated. The residue was purified by chromatography on silica gel eluted by dichloromethane to give 0.4 g of product (yield was 53%). MS: calcd 536 (MH+), exp 536 (MH+).</div></div><div id="sec5_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-Aminophenylcarbamoyl)vinyl]phenyl}-1-<i>tert</i>-butylpyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>52</b>)</h3><div class="NLM_p">A solution of lithium hydroxide monohydrate (2.1 g, 50 mmol) in water (50 mL) was added to a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-<i>tert</i>-butylpyrrolidine-3-carboxylic acid ethyl ester (<b>XVII</b>) (0.4 g, 0.74 mmol) in THF (50 mL), and the mixture was stirred at room temperature overnight. LC-MS indicated that the starting material was consumed. The mixture was adjusted to pH = 6–8 with 6 N HCl. The solvent was removed to give crude <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-<i>tert</i>-butylpyrrolidine-3-carboxylic acid, which was used in the next step without further purification. MS: calcd 508 (MH+), exp 508 (MH+).</div><div class="NLM_p">HATU (0.2 g, 0.52 mmol) was added to a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]phenyl}-1-<i>tert</i>-butylpyrrolidine-3-carboxylic acid (crude material, 0.2 mmol) and 4-chloroaniline (0.1 g, 0.78 mmol) in TEA (0.1 mL), and dichloromethane (7 mL) at rt, and the mixture was stirred at rt overnight. LC-MS indicated that the reaction was completed. The mixture was partitioned between water and dichloromethane. The organic phase was dried and concentrated. The residue was purified by chromatography on silica gel to give 80 mg of <i>rac</i>-[2-(3-{4-[(<i>trans</i>-3,4)-1-<i>tert</i>-butyl-4-(4-chlorophenylcarbamoyl)pyrrolidin-3-yl]phenyl}acryloylamino)phenyl]carbamic acid tert-butyl ester (yield was 66%). MS: calcd 617 (MH+), exp 617 (MH+).</div><div class="NLM_p last"><i>rac</i>-[2-(3-{4-[(<i>trans</i>-3,4)-1-<i>tert</i>-Butyl-4-(4-chlorophenylcarbamoyl)pyrrolidin-3-yl]phenyl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester (80 mg, 0.13 mmol) was dissolved in MeOH (1.25 M HCl, 10 mL) and stirred for 4 h. LC-MS indicated that the reaction was completed. The solvent was removed, and the residue was basified with TEA and purified by prep-HPLC to give 18 mg of product. MS: calcd (MH+) 517, exp (MH+) 517. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz), 7.64 (d, 1H, <i>J</i> = 15.4 Hz), 7.55 (d, 2H, <i>J</i> = 8.0 Hz), 7.48 (d, 2H, <i>J</i> = 8.8 Hz), 7.39 (d, 2H, <i>J</i> = 8.0 Hz), 7.25 (d, 2H, <i>J</i> = 8.8 Hz), 7.19 (d, 1H, <i>J</i> = 8.0 Hz), 7.04 (t, 1H, <i>J</i> = 7.2 Hz), 6.87 (d, 1H, <i>J</i> = 8.0 Hz), 6.82 (d, 1H, <i>J</i> = 15.6 Hz), 6.74 (t, 1H, <i>J</i> = 7.6 Hz), 3.72 (m, 1H), 3.18–3.07 (m, 3H), 2.91 (m, 2H), 1.16 (s, 9H).</div></div><div id="sec5_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-(2,2,2-trifluoroethyl)pyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XVIII</b>)</h3><div class="NLM_p">To a suspension of <i>rac</i>-(<i>trans</i>-3,4)-(4-(4-bromophenyl)pyrrolidine-3-carboxylic acid ethyl ester hydrochloride (<b>X</b>) (1.10 g, 3.28 mmol) in DCM (10 mL) was added Et<sub>3</sub>N (1.4 mL, 9.86 mmol). This mixture was stirred for about 5 min at room temperature and then cooled to 0 °C. To the resulting mixture was added trifluoroacetate anhydride (0.9 g, 4.28 mmol). The mixture was stirred for 30 min at 0 °C, warmed to room temperature, and stirred for 1 h. The resulting mixture was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash chromatography (Pet:EtOAc = 10:1) to get <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)-1-(2,2,2-trifluoroacetyl)pyrrolidine-3-carboxylic acid ethyl ester as a colorless oil product (0.88 g, 68.1%).</div><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)-1-(2,2,2-trifluoroacetyl)pyrrolidine-3-carboxylic acid ethyl ester (0.82 g, 2.08 mmol) was added a solution of borane in THF (3.3 mL, 3.3 mmol) at 0 °C. Then the mixture was refluxed for 2 h. The resulting mixture was cooled and quenched by aqueous HCl (6 N). The mixture was concentrated in vacuo to remove THF. The residue was basified by solid Na<sub>2</sub>CO<sub>3</sub> and then extracted with EtOAc. The organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. The crude product was purified by flash chromatography (Pet:EtOAc = 20:1) to get colorless oil product <b>XVIII</b> (230 mg, 30%); 188 mg of starting material was recovered. MS: calcd 380 (MH+), exp 380 (MH+). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz), 7.45 (d, 2H, <i>J</i> = 8.4 Hz), 7.23 (d, 2H, <i>J</i> = 8.4 Hz), 4.13–4.19 (m, 2H), 3.61–3.66 (m, 1H), 3.34–3.37 (m, 1H), 3.04–3.24 (m, 6H), 1.24 (t, 3H, <i>J</i> = 6.8 Hz).</div></div><div id="sec5_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> <i>rac</i>-(<i>trans</i>-3,4)-4-(4-Bromophenyl)-1-(2,2,2-trifluoroethyl)pyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>XIX</b>)</h3><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)-1-(2,2,2-trifluoroethyl)pyrrolidine-3-carboxylic acid ethyl ester (440 mg, 1.16 mmol) in methanol (3 mL) was added aqueous NaOH (2.9 mL, 2 M). The mixture was stirred at rt for 4 h. Then the mixture was concentrated in vacuo to remove methanol, and the residue was acidified by aqueous HCl and lyophilized to offer a colorless solid. MS: calcd 352 (MH+), exp 352 (MH+), calcd 350 (M – H), exp 350 (M – H).</div><div class="NLM_p last">To a suspension of the crude acid in DCM (10 mL) were added PyBrop (811 mg, 1.74 mmol), 4-chlorophenylamine (177 mg, 1.39 mmol), and DIPEA (0.5 mL, 2.84 mmol). The reaction was stirred at room temperature overnight and diluted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL). The mixture was washed with water (10 mL) and brine (10 mL), dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash chromatography on silica gel (Pet:EtOAc = 5:1–2:1) to get the product (483 mg, 90%). MS: calcd 461 (MH+), exp 461 (MH+).</div></div><div id="sec5_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> <i>rac</i>-(<i>trans</i>-3,4)-4-{4-[2-(2-Aminophenylcarbamoyl)vinyl]phenyl}-1-(2,2,2-trifluoroethyl)pyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>48</b>)</h3><div class="NLM_p">The mixture of <i>rac</i>-(<i>trans</i>-3,4)-4-(4-bromophenyl)-1-(2,2,2-trifluoroethyl)pyrrolidine-3-carboxylic acid (4-chlorophenyl)amide (<b>XIX</b>) (483 mg, 1.05 mmol), <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (302 mg, 1.15 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (28.8 mg, 0.03 mmol), and P(o-tolyl)<sub>3</sub> (38.4 mg, 0.12 mmol) in DMF (50 mL) and TEA (0.42 mL, 3 mmol) was heated at 110 °C under Ar in a sealed tube for 4 h. The cooled mixture was concentrated in vacuo to remove DMF. The residue was purified by chromatography on silica gel (DCM:methanol = 40:1–20:1) to give <i>rac</i>-[2-(3-{4-[(<i>trans</i>-3,4)-4-(4-chlorophenylcarbamoyl)-1-(2,2,2-trifluoroethyl)pyrrolidin-3-yl]phenyl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester (580 mg, 86%). MS: calcd 643(MH+), exp 643 (MH+).</div><div class="NLM_p last">To a solution of <i>rac</i>-[2-(3-{4-[(<i>trans</i>-3,4)-4-(4-chlorophenylcarbamoyl)-1-(2,2,2-trifluoroethyl)pyrrolidin-3-yl]phenyl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester (580 mg, 0.9 mmol) in DCM (6 mL) was added trifluoroacetic acid (2 mL), and the mixture was stirred for 2 h. The solvent was removed, and the residue was basified with TEA and purified by prep-HPLC. MS: calcd 543 (MH+), exp 543 (MH+). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz), 7.66 (d, 1H, <i>J</i> = 15.6 Hz), 7.60 (d, 2H, <i>J</i> = 8.0 Hz), 7.53 (d, 2H, <i>J</i> = 8.4 Hz), 7.42 (d, 2H, <i>J</i> = 8.0 Hz), 7.29 (d, 2H, <i>J</i> = 8.4 Hz), 7.21 (d, 1H, <i>J</i> = 7.6 Hz), 7.08–7.04 (m, 1H), 6.90–6.82 (m, 2H), 6.78–6.74 (m, 1H), 3.92–3.60 (m, 1H), 3.44–3.38 (m, 2H), 3.29–3.24 (m, 4H), 3.10–3.06 (m, 1H).</div></div><div id="sec5_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> 3-(5-Bromopyridin-2-yl)acrylic Acid Ethyl Ester (<b>XX</b>)</h3><div class="NLM_p">To a solution of 5-bromo-2-iodopyridine (14.2 g, 50 mmol) in dry THF (160 mL) was added dropwise isopropyl magnesium chloride (30 mL, 60 mmol) at −20 °C under N<sub>2</sub>. After the addition, the reaction mixture was stirred for about 2 h at the same temperature, and then DMF (5.11 g, 70 mmol) was added while keeping the temperature under 0 °C. The reaction temperature was allowed to rise to rt, the mixture was stirred for 1 h and quenched with saturated ammonium chloride (10 mL), and then the solution pH was adjusted to 7–8 with 2.0 M HCl. The solvent was removed and extracted with ethyl acetate (2 × 200 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to obtain 8.5 g (90%) of 5-bromopyridine-2-carbaldehyde as a yellow solid product which was used directly in the next step without further purification. MS: calcd 186 (MH+), exp 186 (MH+).</div><div class="NLM_p last">To a solution of triethyl phosphonoacetate (10.2 g, 45.7 mmol), Et<sub>3</sub>N (7 mL, 50.3 mmol), and lithium bromide (4.3 g, 50.3 mol) in CH<sub>3</sub>CN (120 mL) was added 5-bromopyridine-2-carbaldehyde (8.5 g, 45.7 mmol). This mixture was stirred overnight at room temperature. When LC-MS indicated the reaction was complete, it was quenched with water (10 mL). Excess CH<sub>3</sub>CN was removed in vacuo, and then the remaining mixture was extracted with EtOAc (2 × 100 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated. The crude product was purified by column chromatography (PE: EtOAc = 5:1) to give 7.8 g (67%) of a yellow solid product. MS: calcd 256 (MH+), exp 256 (MH+).</div></div><div id="sec5_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> <i>rac</i>-(<i>trans</i>-3,4)-4-(5-Bromopyridin-2-yl)-1-methylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XXI</b>)</h3><div class="NLM_p last">To a solution of 3-(5-bromopyridin-2-yl)acrylic acid ethyl ester (<b>XX</b>) (3 g, 11.7 mmol) and paraformaldehyde (4.22 g, 140.4 mol) in toluene (200 mL) was added sarcosine (2.1 g, 23.4 mmol). This mixture was heated to reflux for 4 h, using Dean–Stark trap to remove the water formed in the reaction. LC-MS indicated that the reaction was complete. After cooling, the solution was filtered, and the solid was washed with EtOAc (2 × 50 mL). The combined organic layer was washed with water and brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated. The crude product was purified by column chromatography (PE: EtOAc = 3:1) to give 2.8 g (77%) of a white solid product. MS: calcd 313 (MH+), exp 313 (MH+).</div></div><div id="sec5_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> <i>rac</i>-(<i>trans</i>-3,4)-4-{5-[2-(2-Aminophenylcarbamoyl)vinyl]pyridin-2-yl}-1-methylpyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>64</b>)</h3><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(5-bromopyridin-2-yl)-1-methylpyrrolidine-3-carboxylic acid ethyl ester (<b>XXI</b>) (2.1 g, 6.7 mmol) in dried DMF (50 mL) were added <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>) (2.1 g, 8.05 mmol), tris-<i>o</i>-tolylphosphine (0.41 g, 1.34 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (0.62 g, 0.67 mmol), and Et<sub>3</sub>N (2.7 g, 26.8 mmol). The resulting mixture was stirred for 3 h at 100 °C under argon. After cooling, the mixture was filtered, and the solvent was removed, diluted with EtOAc (100 mL), washed with water and brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to give an oil. The crude product was purified by column chromatography (PE: EtOAc = 1:1) to give 2.5 g (75%) of <i>rac</i>-(<i>trans</i>-3,4)-4-{5-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]pyridin-2-yl}-1-methylpyrrolidine-3-carboxylic acid ethyl ester as a white solid product. MS: calcd 495 (MH+), exp 495 (MH+).</div><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-{5-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]pyridin-2-yl}-1-methylpyrrolidine-3-carboxylic acid ethyl ester (2.5 g, 5 mmol) in THF/H<sub>2</sub>O (45 mL/15 mL) was added LiOH monohydrate (1.05 g, 25 mmol). This mixture was stirred for about 5 h at room temperature and then evaporated to remove most of the THF. The aqueous solution was acidified with 2 N HCl to pH 6–7 and then extracted with ethyl acetate (3 × 70 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to obtain 2.0 g (85.6%) of <i>rac</i>-(<i>trans</i>-3,4)-4-{5-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]pyridin-2-yl}-1-methylpyrrolidine-3-carboxylic acid as a yellow solid product. MS: calcd 467 (MH+), exp 467 (MH+).</div><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-{5-[2-(2-<i>tert</i>-butoxycarbonylaminophenylcarbamoyl)vinyl]pyridin-2-yl}-1-methylpyrrolidine-3-carboxylic acid (350 mg, 0.75 mmol), 4-chloroaniline (115 mg, 0.9 mmol), and Et<sub>3</sub>N (227 mg, 2.25 mol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) was added HATU (430 mg, 1.125 mmol). This mixture was stirred overnight at room temperature and then diluted with CH<sub>2</sub>Cl<sub>2</sub> (50 mL), washed with saturated aqueous NaHCO<sub>3</sub> solution, water, and brine, dried with Na<sub>2</sub>SO<sub>4</sub>, and evaporated to obtain 0.23 g (53%) of <i>rac</i>-2-(3-{6-[(<i>trans</i>-3,4)-4-(4-chlorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]pyridin-3-yl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester as a yellow solid. MS: calcd 576 (MH+), exp 576 (MH+).</div><div class="NLM_p last">The solution of <i>rac</i>-2-(3-{6-[(<i>trans</i>-3,4)-4-(4-chlorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]pyridin-3-yl}acryloylamino)phenyl]carbamic acid <i>tert</i>-butyl ester (168 mg, 0.3 mmol) in 2 N HCl in MeOH (10 mL) was stirred for 4 h at room temperature, and then the pH of the solution was adjusted to 8–9 with NaHCO<sub>3</sub> solution. Then the mixture was diluted with water (50 mL) and extracted with EtOAc (2 × 50 mL). The combined organic layer was washed with brine, dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated. The crude product was purified by preparative HPLC to give a yellow solid product. MS: calcd 476 (MH+), exp 476 (MH+). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz), 10.16 (s, 1H), 9.44 (s, 1H), 8.76 (s, 1H), 7.97 (d, 1H <i>J</i> = 8.0 Hz,), 7.63–7.56 broad m, 3H), 7.41 (d, 1H, <i>J</i> = 8.4 Hz), 7.36–7.30 (broad m, 3H), 6.98–6.91 (broad m, 2H), 6.76 (d, 1H, <i>J</i> = 7.2 Hz), 6.58 (t, 1H, <i>J</i> = 7.2 Hz), 4.95 (broad s, 2H), 3.95 (m, 1H), 3.48 (m, 1H), 3.03 (broad m, 2H), 2.79 (m, 2H), 2.33 (s, 3H)</div></div><div id="sec5_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Ethyl 3-(2-[1,3]-Dioxolanpyridinyl)acrylate (<b>XXII</b>)</h3><div class="NLM_p">To a solution of 5-bromopyridine-2-aldehyde (3.72 g, 20 mmol) in toluene (60 mL) were added TsOH monohydrate (190 mg, 1 mmol) and ethane-1,2-diol (2.48 g, 40 mmol). The mixture was heated to reflux for 6 h. The solvent was removed, and the residue was dissolved in ethyl acetate and washed with sodium bicarbonate and brine. The organic layer was dried with sodium sulfate and concentrated to give the crude product 5-bromo-2-[1,3]dioxolan-2-ylpyridine which was used in the next step without further purification. MS: calcd 230 (MH+), exp 230 (MH+).</div><div class="NLM_p last">To a solution of 5-bromo-2-[1,3]dioxolan-2-ylpyridine (460 mg, 2.0 mmol) in DMF (10 mL) were added ethyl acrylate (240 mg, 2.4 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (183 mg, 0.2 mmol), tris-<i>o</i>-tolylphosphine (122 mg, 0.4 mmol), and Et<sub>3</sub>N (606 mg, 6 mmol). The mixture was heated to 110 °C for 3 h under protection of nitrogen. The mixture was diluted with ethyl acetate and washed with brine. The organic layer was dried with sodium sulfate and concentrated. The residue was purified by chromatography to give the pure product. MS: calcd 250 (MH+), exp 250 (MH+).</div></div><div id="sec5_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> <i>rac</i>-(<i>trans</i>-3,4)-4-(6-[1,3]Dioxolan-2-ylpyridin-3-yl)-1-methylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XXIII</b>)</h3><div class="NLM_p last">To a solution of ethyl 3-{2-[1,3]-dioxolanpyridinyl}acrylate (6 g, 24 mmol) in toluene (600 mL) was added <i>N</i>-methylglycine (10.7 g, 120 mmol) and paraformaldehyde (14.4 g, 480 mmol). The mixture was heated to reflux for 6 h. After cooling, the solvent was removed and the residue was purified by chromatography to give the pure product. MS: calcd 307 (MH+), exp 307 (MH+).</div></div><div id="sec5_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> <i>rac</i>-(<i>trans</i>-3,4)-4-[6-(2-<i>tert</i>-Butoxycarbonylvinyl)pyridin-3-yl]-1-methylpyrrolidine-3-carboxylic Acid Ethyl Ester (<b>XXIV</b>)</h3><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(6-[1,3]dioxolan-2-ylpyridin-3-yl)-1-methylpyrrolidine-3-carboxylic acid ethyl ester (612 mg, 2 mmol) in DMSO/H<sub>2</sub>O (12 mL/4 mL) was added LiCl (509 mg, 12 mmol). The mixture was sealed in a microwave tube and heated to 150 °C for 10 min under microwave conditions. The mixture was diluted with ethyl acetate and washed with brine. The organic layer was dried with sodium sulfate and concentrated. The residue was purified by chromatography to give the pure product <i>rac</i>-(<i>trans</i>-3,4)-4-(6-formylpyridin-3-yl)-1-methylpyrrolidine-3-carboxylic acid ethyl ester. MS: calcd 263 (MH+), exp 263 (MH+).</div><div class="NLM_p last">To the solution of <i>rac</i>-(<i>trans</i>-3,4)-4-(6-formylpyridin-3-yl)-1-methylpyrrolidine-3-carboxylic acid ethyl ester (1.31 g, 5 mmol) in THF (35 mL) was added diethyl <i>tert</i>-butyl phosphonoacetate (1.78 g, 7 mmol) at rt. <i>n</i>-Butyllithium (4 mL, 8 mmol) was added dropwise to the above solution at −78 °C. The solution was allowed to rise to 0 °C slowly and stirred at 0 °C for 2 h. The reaction was quenched with saturated ammonium chloride. The solvent was removed, and the residue was dissolved in ethyl acetate. The organic layer was washed with brine and dried with sodium sulfate. The solvent was concentrated, and the residue was purified by chromatography to give the pure product. MS: calcd 361 (MH+), exp 361 (MH+).</div></div><div id="sec5_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> <i>rac</i>-3-{5-[(<i>trans</i>-3,4)-4-(4-Chlorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]pyridin-2-yl}acrylic acid <i>tert</i>-Butyl Ester (<b>XXV</b>)</h3><div class="NLM_p">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-[6-(2-<i>tert</i>-butoxycarbonylvinyl)pyridin-3-yl]-1-methylpyrrolidine-3-carboxylic acid ethyl ester (500 mg, 1.39 mmol) in THF (9 mL) was added lithium hydroxide monohydrate (234 mg, 5.56 mmol) and water (3 mL). The mixture was stirred at rt for 3 h. The solution was neutralized to pH 6–7. The solvent was removed, and the residue (<i>rac</i>-(<i>trans</i>-3,4)-4-[6-(2-<i>tert</i>-butoxycarbonylvinyl)pyridin-3-yl]-1-methylpyrrolidine-3-carboxylic acid) was used directly in the next step. MS: calcd 333 (MH+), exp 333 (MH+).</div><div class="NLM_p last">To a solution of <i>rac</i>-(<i>trans</i>-3,4)-4-[6-(2-<i>tert</i>-butoxycarbonylvinyl)pyridin-3-yl]-1-methylpyrrolidine-3-carboxylic acid (430 mg, 1.3 mmol) in dichloromethane (10 mL) were added HATU (990 mg, 2.6 mmol), Et<sub>3</sub>N (525 mg, 5.2 mmol), and 4-chloroaniline (249 mg, 1.95 mmol). The mixture was stirred at rt for 24 h. The solvent was removed, and the residue was dissolved in ethyl acetate. The organic layer was washed with brine and dried with sodium sulfate. The solvent was removed, and the residue was purified by chromatography to give the pure product. MS: calcd 442 (MH+), exp 442 (MH+).</div></div><div id="sec5_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> <i>rac</i>-(<i>trans</i>-3,4)-4-{6-[2-(2-Aminophenylcarbamoyl)vinyl]pyridin-3-yl}-1-methylpyrrolidine-3-carboxylic Acid (4-Chlorophenyl)amide (<b>61</b>)</h3><div class="NLM_p">To a solution of <i>rac</i>-3-{5-[(<i>trans</i>-3,4)-4-(4-chlorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]pyridin-2-yl}acrylic acid <i>tert</i>-butyl ester (172 mg, 0.5 mmol) in dichloromethane (5 mL) was added trifluoroacetic acid (570 mg, 5 mmol) at 0 °C. The solution was allowed to rise to rt and stirred for 2 h. The solvent was removed under reduced pressure to give the crude product (<i>rac</i>-3-{5-[(<i>trans</i>-<i>3,4</i>)-4-(4-chlorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]pyridin-2-yl}acrylic acid) which was used directly in next step. MS: calcd 386 (MH+), exp 386 (MH+).</div><div class="NLM_p last">To a solution of <i>rac</i>-3-{5-[(<i>trans</i>-3,4)-4-(4-chlorophenylcarbamoyl)-1-methylpyrrolidin-3-yl]pyridin-2-yl}acrylic acid (54 mg, 0.14 mmol) in dichloromethane (3 mL) were added 1,2-diaminobenzene (46 mg, 0.42 mmol), HATU (80 mg, 0.21 mmol), and Et<sub>3</sub>N (85 mg, 0.84 mmol). The mixture was stirred at rt for 12 h. The solvent was removed, and the residue was dissolved in ethyl acetate. The solution was washed with brine and dried with sodium sulfate. The solvent was concentrated to give the residue, which was purified by preparative HPLC to give pure product. MS: calcd 476 (MH+), exp 476 (MH+). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz), 8.57 (s, 1H), 7.88 (d, 1H, <i>J</i> = 8.4 Hz), 7.68–7.62 (m, 2H), 7.53 (d, 2H, <i>J</i> = 8.8 Hz), 7.28 (d, 2H, <i>J</i> = 8.8 Hz), 7.22–7.18 (m, 2H), 7.05 (t, 1H, <i>J</i> = 7.6 Hz), 6.87 (d, 1H, <i>J</i> = 8.0 Hz), 6.76 (t, 1H, <i>J</i> = 7.6 Hz), 3.90 (q, 1H, <i>J</i> = 8.0 Hz), 3.32–3.18 (broad m, 3H), 3.01 (t, 2H, <i>J</i> = 8.4 Hz), 2.54 (s, 3H).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i95"><a href="/doi/suppl/10.1021/jm3011838">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05705" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05705" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Additional information regarding compound characterization (LCMS and <sup>1</sup>H NMR), supplemental SAR data, and experimental details for the CYP inhibition assay, metabolite identification, cell lines and biological reagents, patient-derived xenograft model study design, microarray profiling study of compound <b>5</b>, and immunohistochemistry scoring images. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm3011838/suppl_file/jm3011838_si_001.pdf">jm3011838_si_001.pdf (716.9 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm3011838" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75236" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75236" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason C. Wong</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#78120f17161f3819140d155615110c561d1c0d"><span class="__cf_email__" data-cfemail="59332e36373e1938352c347734302d773c3d2c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guozhi Tang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xihan Wu</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chungen Liang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhenshan Zhang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Guo</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhenghong Peng</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weixing Zhang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xianfeng Lin</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhanguo Wang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianghua Mei</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junli Chen</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Song Pan</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nan Zhang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongfu Liu</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingwei Zhou</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lichun Feng</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weili Zhao</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shijie Li</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao Zhang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meifang Zhang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yiping Rong</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tai-Guang Jin</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiongwen Zhang</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuang Ren</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Ji</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rong Zhao</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jin She</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi Ren</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunping Xu</span> - <span class="hlFld-Affiliation affiliation">Crown Bioscience
Inc., Light Muller Building, Changping Sector of Zhongguancun
Science Park, No.21 Huoju Street, Changping District, Beijing 102200,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dawei Chen</span> - <span class="hlFld-Affiliation affiliation">Crown Bioscience
Inc., Light Muller Building, Changping Sector of Zhongguancun
Science Park, No.21 Huoju Street, Changping District, Beijing 102200,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Cai</span> - <span class="hlFld-Affiliation affiliation">Crown Bioscience
Inc., Light Muller Building, Changping Sector of Zhongguancun
Science Park, No.21 Huoju Street, Changping District, Beijing 102200,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Song Shan</span> - <span class="hlFld-Affiliation affiliation">Chipscreen Biosciences
Ltd., 2-601, BIO-Incubator, Gaoxin C, first Ave., Shenzhen
High-Tech Industrial Park, Nanshan District, Shenzhen, Guangdong 518057,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Desi Pan</span> - <span class="hlFld-Affiliation affiliation">Chipscreen Biosciences
Ltd., 2-601, BIO-Incubator, Gaoxin C, first Ave., Shenzhen
High-Tech Industrial Park, Nanshan District, Shenzhen, Guangdong 518057,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhiqiang Ning</span> - <span class="hlFld-Affiliation affiliation">Chipscreen Biosciences
Ltd., 2-601, BIO-Incubator, Gaoxin C, first Ave., Shenzhen
High-Tech Industrial Park, Nanshan District, Shenzhen, Guangdong 518057,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xianping Lu</span> - <span class="hlFld-Affiliation affiliation">Chipscreen Biosciences
Ltd., 2-601, BIO-Incubator, Gaoxin C, first Ave., Shenzhen
High-Tech Industrial Park, Nanshan District, Shenzhen, Guangdong 518057,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Taiping Chen</span> - <span class="hlFld-Affiliation affiliation">Crown Bioscience
Inc., Light Muller Building, Changping Sector of Zhongguancun
Science Park, No.21 Huoju Street, Changping District, Beijing 102200,
P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yun He</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Chen</span> - <span class="hlFld-Affiliation affiliation">Roche R&D Center
(China) Ltd., 720 Cai Lun Road, Building 5, Pudong,
Shanghai 201203, P. R. China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d1123e6114-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i96">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank members of the following departments of the Roche R&D Center (China) Ltd. for research support: (1) the department of analytical chemistry (Wenzhi Cheng, Lucy Lu, Sterling Yuan, Harkin Gao, Dr. Penny Ding, Dr. Fred Li) for support on purification and characterization of compounds; (2) the DMPK department (Hongxia Qiu, Yuxia Zhang, Jian Xin) for conducting microsome, solubility, and CYP inhibition studies; (3) the compound management and inventory group (Lana Liu, Bill Wang, Qinghong Lv, Dr. Lynn Lin) for support on obtaining and tracking synthetic reagents and formatting final compounds for biological assays.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i97" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i97"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i98" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i98"> Abbreviations Used</h2><tr><td class="NLM_term">AFP</td><td class="NLM_def"><p class="first last">α-fetoprotein</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under curve</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last">COOC(CH<sub>3</sub>)<sub>3</sub></p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">systemic clearance rate</p></td></tr><tr><td class="NLM_term">CYP</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">dba</td><td class="NLM_def"><p class="first last">dibenzylideneacetone</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMPK</td><td class="NLM_def"><p class="first last">distribution, metabolism, pharmacokinetics</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">F</td><td class="NLM_def"><p class="first last">bioavailability</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo-[4,5-<i>b</i>]pyridinium hexafluorophosphate 3-oxide</p></td></tr><tr><td class="NLM_term">HBV</td><td class="NLM_def"><p class="first last">hepatitis B virus</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">HDACi</td><td class="NLM_def"><p class="first last">histone deacetylase inhibitor</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-à-go-go related gene</p></td></tr><tr><td class="NLM_term">HLM</td><td class="NLM_def"><p class="first last">human liver microsomes</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">HWE</td><td class="NLM_def"><p class="first last">Horner–Wadsworth–Emmons</p></td></tr><tr><td class="NLM_term">IHC</td><td class="NLM_def"><p class="first last">immunohistochemistry</p></td></tr><tr><td class="NLM_term"><i>i</i>-PrOH</td><td class="NLM_def"><p class="first last">2-propanol</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsomes</p></td></tr><tr><td class="NLM_term">mpk</td><td class="NLM_def"><p class="first last">milligrams per kilogram</p></td></tr><tr><td class="NLM_term">Ms</td><td class="NLM_def"><p class="first last">mesyl</p></td></tr><tr><td class="NLM_term">NAD</td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamics</p></td></tr><tr><td class="NLM_term">PE</td><td class="NLM_def"><p class="first last">petroleum ether</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PO</td><td class="NLM_def"><p class="first last">oral administration</p></td></tr><tr><td class="NLM_term">Q2D</td><td class="NLM_def"><p class="first last">once every two days</p></td></tr><tr><td class="NLM_term">QD</td><td class="NLM_def"><p class="first last">every day</p></td></tr><tr><td class="NLM_term">qRT-PCR</td><td class="NLM_def"><p class="first last">quantitative real time polymerase chain reaction</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">structure–property relationship</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TSG</td><td class="NLM_def"><p class="first last">tumor suppressor gene</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution at steady state</p></td></tr><tr><td class="NLM_term">WST</td><td class="NLM_def"><p class="first last">water-soluble tetrazolium dye</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07288" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07288" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 32 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">El-Seraq, H. B.</span><span> </span><span class="NLM_article-title">Hepatocellular carcinoma</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">365</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1127</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1056%2FNEJMra1001683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=21992124" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=1118-1127&author=H.+B.+El-Seraq&title=Hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1001683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1001683%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Seraq%26aufirst%3DH.%2BB.%26atitle%3DHepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D1118%26epage%3D1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Bruix, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llovet, J. M.</span><span> </span><span class="NLM_article-title">Prognostic prediction and treatment strategy in hepatocellular carcinoma</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">519</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2002&pages=519-524&author=J.+Bruixauthor=J.+M.+Llovet&title=Prognostic+prediction+and+treatment+strategy+in+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBruix%26aufirst%3DJ.%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26atitle%3DPrognostic%2520prediction%2520and%2520treatment%2520strategy%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2002%26volume%3D35%26spage%3D519%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Llovet, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ricci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzaferro, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gane, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanc, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Oliveira, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raoul, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeuzem, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolondi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greten, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galle, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitz, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borbath, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Häussinger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannaris, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moscovici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voliotis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruix, J.</span><span> </span><span class="NLM_article-title">Sorafenib in advanced hepatocellular carcinoma</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">378</span><span class="NLM_x">–</span> <span class="NLM_lpage">390</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1056%2FNEJMoa0708857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18650514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFWjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2008&pages=378-390&author=J.+M.+Llovetauthor=S.+Ricciauthor=V.+Mazzaferroauthor=P.+Hilgardauthor=E.+Ganeauthor=J.+F.+Blancauthor=A.+C.+de+Oliveiraauthor=A.+Santoroauthor=J.+L.+Raoulauthor=A.+Fornerauthor=M.+Schwartzauthor=C.+Portaauthor=S.+Zeuzemauthor=L.+Bolondiauthor=T.+F.+Gretenauthor=P.+R.+Galleauthor=J.+F.+Seitzauthor=I.+Borbathauthor=D.+H%C3%A4ussingerauthor=T.+Giannarisauthor=M.+Shanauthor=M.+Moscoviciauthor=D.+Voliotisauthor=J.+Bruix&title=Sorafenib+in+advanced+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Sorafenib in advanced hepatocellular carcinoma</span></div><div class="casAuthors">Llovet, Josep M.; Ricci, Sergio; Mazzaferro, Vincenzo; Hilgard, Philip; Gane, Edward; Blanc, Jean-Frederic; Cosme de Oliveira, Andre; Santoro, Armando; Raoul, Jean-Luc; Forner, Alejandro; Schwartz, Myron; Porta, Camillo; Zeuzem, Stefan; Bolondi, Luigi; Greten, Tim F.; Galle, Peter R.; Seitz, Jean-Francois; Borbath, Ivan; Haussinger, Dieter; Giannaris, Tom; Shan, Minghua; Moscovici, Marius; Voliotis, Dimitris; Bruix, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">378-390</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">No effective systemic therapy exists for patients with advanced hepatocellular carcinoma.  A preliminary study suggested that sorafenib, an oral multi-kinase inhibitor of the vascular endothelial growth factor receptor, the platelet-derived growth factor receptor, and Raf may be effective in hepatocellular carcinoma.  In this multicenter, phase 3, double-blind, placebo-controlled trial, we randomly assigned 602 patients with advanced hepatocellular carcinoma who had not received previous systemic treatment to receive either sorafenib (at a dose of 400 mg twice daily) or placebo.  Primary outcomes were overall survival and the time to symptomatic progression.  Secondary outcomes included the time to radiol. progression and safety.  At the second planned interim anal., 321 deaths had occurred, and the study was stopped.  Median overall survival was 10.7 mo in the sorafenib group and 7.9 mo in the placebo group (hazard ratio in the sorafenib group, 0.69; 95% confidence interval, 0.55 to 0.87; P < 0.001).  There was no significant difference between the two groups in the median time to symptomatic progression (4.1 mo vs. 4.9 mo, resp., P = 0.77). The median time to radiol. progression was 5.5 mo in the sorafenib group and 2.8 mo in the placebo group (P < 0.001).  Seven patients in the sorafenib group (2%) and two patients in the placebo group (1%) had a partial response; no patients had a complete response.  Diarrhea, wt. loss, hand-foot skin reaction, and hypophosphatemia were more frequent in the sorafenib group.  In patients with advanced hepatocellular carcinoma, median survival and the time to radiol. progression were nearly 3 mo longer for patients treated with sorafenib than for those given placebo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6mcSPrj1TibVg90H21EOLACvtfcHk0lgj6NRhN7g0Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFWjsL8%253D&md5=579650dc323f35ad2415523c9827bf9a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0708857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0708857%26sid%3Dliteratum%253Aachs%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26aulast%3DRicci%26aufirst%3DS.%26aulast%3DMazzaferro%26aufirst%3DV.%26aulast%3DHilgard%26aufirst%3DP.%26aulast%3DGane%26aufirst%3DE.%26aulast%3DBlanc%26aufirst%3DJ.%2BF.%26aulast%3Dde%2BOliveira%26aufirst%3DA.%2BC.%26aulast%3DSantoro%26aufirst%3DA.%26aulast%3DRaoul%26aufirst%3DJ.%2BL.%26aulast%3DForner%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DM.%26aulast%3DPorta%26aufirst%3DC.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DBolondi%26aufirst%3DL.%26aulast%3DGreten%26aufirst%3DT.%2BF.%26aulast%3DGalle%26aufirst%3DP.%2BR.%26aulast%3DSeitz%26aufirst%3DJ.%2BF.%26aulast%3DBorbath%26aufirst%3DI.%26aulast%3DH%25C3%25A4ussinger%26aufirst%3DD.%26aulast%3DGiannaris%26aufirst%3DT.%26aulast%3DShan%26aufirst%3DM.%26aulast%3DMoscovici%26aufirst%3DM.%26aulast%3DVoliotis%26aufirst%3DD.%26aulast%3DBruix%26aufirst%3DJ.%26atitle%3DSorafenib%2520in%2520advanced%2520hepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2008%26volume%3D359%26spage%3D378%26epage%3D390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Yang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, E.</span><span> </span><span class="NLM_article-title">The Rpd3/Hda1 family of lysine deacetylases: from bacteria and yeast to mice and men</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">206</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1038%2Fnrm2346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18292778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlGnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=206-218&author=X.+J.+Yangauthor=E.+Seto&title=The+Rpd3%2FHda1+family+of+lysine+deacetylases%3A+from+bacteria+and+yeast+to+mice+and+men"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Rpd3/Hda1 family of lysine deacetylases: from bacteria and yeast to mice and men</span></div><div class="casAuthors">Yang, Xiang-Jiao; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">206-218</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein lysine deacetylases have a pivotal role in numerous biol. processes and can be divided into the Rpd3/Hda1 and sirtuin families, each having members in diverse organisms including prokaryotes.  In vertebrates, the Rpd3/Hda1 family contains 11 members, traditionally referred to as histone deacetylases (HDAC) 1-11, which are further grouped into classes I, II and IV.  Whereas most class I HDACs are subunits of multiprotein nuclear complexes that are crucial for transcriptional repression and epigenetic landscaping, class II members regulate cytoplasmic processes or function as signal transducers that shuttle between the cytoplasm and the nucleus.  Little is known about class IV HDAC11, although its evolutionary conservation implies a fundamental role in various organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRiVyZexxGn7Vg90H21EOLACvtfcHk0lii4NvzlD4QFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlGnurk%253D&md5=f11f89567feadf9cd38b3852dc44c382</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrm2346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2346%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BJ.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DThe%2520Rpd3%252FHda1%2520family%2520of%2520lysine%2520deacetylases%253A%2520from%2520bacteria%2520and%2520yeast%2520to%2520mice%2520and%2520men%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2008%26volume%3D9%26spage%3D206%26epage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Grozinger, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">Deacetylase enzymes: biological functions and the use of small-molecule inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">16</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2FS1074-5521%2802%2900092-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=11841934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=3-16&author=C.+M.+Grozingerauthor=S.+L.+Schreiber&title=Deacetylase+enzymes%3A+biological+functions+and+the+use+of+small-molecule+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Deacetylase enzymes: Biological functions and the use of small-molecule inhibitors</span></div><div class="casAuthors">Grozinger, Christina M.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-16</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This review discusses the mechanism of deacetylation by the histone deacetylases (HDAC) and the Sir2 (silent information regulator)-like family of NAD-dependent deacetylases, or sirtuins.  The use of specific small-mol. inhibitors of the deacetylase activity of HDAC and sirtuins may provide an effective technique for elucidating the biol. roles of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvpPshiJErULVg90H21EOLACvtfcHk0lii4NvzlD4QFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2isLo%253D&md5=1eabb4f886e5777735ba14edadfaa332</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2802%2900092-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252802%252900092-3%26sid%3Dliteratum%253Aachs%26aulast%3DGrozinger%26aufirst%3DC.%2BM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DDeacetylase%2520enzymes%253A%2520biological%2520functions%2520and%2520the%2520use%2520of%2520small-molecule%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D3%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Rikimaru, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taketomi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirabe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maehara, Y.</span><span> </span><span class="NLM_article-title">Clinical significance of histone deacetylase 1 expression in patients with hepatocellular carcinoma</span> <span class="citation_source-journal">Oncology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=69-74&author=T.+Rikimaruauthor=A.+Taketomiauthor=Y.+Yamashitaauthor=K.+Shirabeauthor=T.+Hamatsuauthor=M.+Shimadaauthor=Y.+Maehara&title=Clinical+significance+of+histone+deacetylase+1+expression+in+patients+with+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRikimaru%26aufirst%3DT.%26aulast%3DTaketomi%26aufirst%3DA.%26aulast%3DYamashita%26aufirst%3DY.%26aulast%3DShirabe%26aufirst%3DK.%26aulast%3DHamatsu%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DM.%26aulast%3DMaehara%26aufirst%3DY.%26atitle%3DClinical%2520significance%2520of%2520histone%2520deacetylase%25201%2520expression%2520in%2520patients%2520with%2520hepatocellular%2520carcinoma%26jtitle%3DOncology%26date%3D2007%26volume%3D72%26spage%3D69%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Yoo, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Na, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seong, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. O.</span><span> </span><span class="NLM_article-title">Hepatitis B virus X protein induces the expression of MTA1 and HDAC1, which enhances hypoxia signaling in hepatocellular carcinoma cells</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">3405</span><span class="NLM_x">–</span> <span class="NLM_lpage">3413</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3405-3413&author=Y.+G.+Yooauthor=T.+Y.+Naauthor=H.+W.+Seoauthor=J.+K.+Seongauthor=C.+K.+Parkauthor=Y.+K.+Shinauthor=M.+O.+Lee&title=Hepatitis+B+virus+X+protein+induces+the+expression+of+MTA1+and+HDAC1%2C+which+enhances+hypoxia+signaling+in+hepatocellular+carcinoma+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DY.%2BG.%26aulast%3DNa%26aufirst%3DT.%2BY.%26aulast%3DSeo%26aufirst%3DH.%2BW.%26aulast%3DSeong%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BK.%26aulast%3DShin%26aufirst%3DY.%2BK.%26aulast%3DLee%26aufirst%3DM.%2BO.%26atitle%3DHepatitis%2520B%2520virus%2520X%2520protein%2520induces%2520the%2520expression%2520of%2520MTA1%2520and%2520HDAC1%252C%2520which%2520enhances%2520hypoxia%2520signaling%2520in%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3405%26epage%3D3413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, H.</span><span> </span><span class="NLM_article-title">Antitumor activity of Chidamide in hepatocellular carcinoma cell lines</span> <span class="citation_source-journal">Mol. Med. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1503</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=22484326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVKmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=1503-1508&author=H.+Wangauthor=Y.+Guoauthor=M.+Fuauthor=X.+Liangauthor=X.+Zhangauthor=R.+Wangauthor=C.+Linauthor=H.+Qian&title=Antitumor+activity+of+Chidamide+in+hepatocellular+carcinoma+cell+lines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity of Chidamide in hepatocellular carcinoma cell lines</span></div><div class="casAuthors">Wang, Haijuan; Guo, Yi; Fu, Ming; Liang, Xiao; Zhang, Xueyan; Wang, Renzhi; Lin, Chen; Qian, Haili</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1503-1508</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Chidamide, the structural analog of MS-275, is a novel and promising histone deacetylase (HDAC) inhibitor for use in cancer therapy.  To investigate its effects on cancer cell growth, MTT assay was performed in 10 human cancer cell lines.  The data showed that the IC50 of Chidamide ranged from 1 to 13 μM, which was comparable to that of MS-275 in half of the tested cell lines.  Furthermore, the growth curve indicated that cell growth was gradually inhibited with an increase in Chidamide dosage, and the inhibition was reversed after drug removal in two hepatocelluclar carcinoma cell lines, BEL-7402 and HCC-9204.  To det. cell cycle and apoptosis, FACS was carried out in the BEL-7402 and HCC-9204 cells treated with Chidamide.  A decrease in the cell population at S phase and an increase in the cell population at G1 phase occurred in a dose-dependent manner.  In addn., Chidamide induced apoptosis and up-regulated p21 mRNA expression.  These results suggest that Chidamide may arrest the cell cycle and inhibit the growth of hepatocellular carcinoma cells through up-regulation of p21.  Further studies are required to clarify the antitumor activity of Chidamide in vivo and its mechanism in anticancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4L7-7ko2hsrVg90H21EOLACvtfcHk0lh7mh9g-8fYpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVKmsbg%253D&md5=67c8639c2d449c485b791669a8aadc99</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DQian%26aufirst%3DH.%26atitle%3DAntitumor%2520activity%2520of%2520Chidamide%2520in%2520hepatocellular%2520carcinoma%2520cell%2520lines%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2012%26volume%3D5%26spage%3D1503%26epage%3D1508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Ma, B. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sung, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poon, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lui, V. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeo, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, A. T.</span><span> </span><span class="NLM_article-title">The preclinical activity of the histone deacetylase inhibitor PXD101 (belinostat) in hepatocellular carcinoma cell lines</span> <span class="citation_source-journal">Invest. New Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=107-114&author=B.+B.+Maauthor=F.+Sungauthor=Q.+Taoauthor=F.+F.+Poonauthor=V.+W.+Luiauthor=W.+Yeoauthor=S.+L.+Chanauthor=A.+T.+Chan&title=The+preclinical+activity+of+the+histone+deacetylase+inhibitor+PXD101+%28belinostat%29+in+hepatocellular+carcinoma+cell+lines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DB.%2BB.%26aulast%3DSung%26aufirst%3DF.%26aulast%3DTao%26aufirst%3DQ.%26aulast%3DPoon%26aufirst%3DF.%2BF.%26aulast%3DLui%26aufirst%3DV.%2BW.%26aulast%3DYeo%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DS.%2BL.%26aulast%3DChan%26aufirst%3DA.%2BT.%26atitle%3DThe%2520preclinical%2520activity%2520of%2520the%2520histone%2520deacetylase%2520inhibitor%2520PXD101%2520%2528belinostat%2529%2520in%2520hepatocellular%2520carcinoma%2520cell%2520lines%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2010%26volume%3D28%26spage%3D107%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Zhang, C</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, G</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liew, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, D.</span><span> </span><span class="NLM_article-title">Transcriptional silencing of the TMS1/ASC tumour suppressor gene by an epigenetic mechanism in hepatocellular carcinoma cells</span> <span class="citation_source-journal">J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">212</span><span class="NLM_x">, </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">142</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1002%2Fpath.2173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=17471463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1Gkt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=212&publication_year=2007&pages=134-142&author=C+Zhangauthor=H+Liauthor=G+Zhouauthor=Q+Zhangauthor=T+Zhangauthor=J+Liauthor=J+Zhangauthor=J+Houauthor=C.+T.+Liewauthor=D.+Yin&title=Transcriptional+silencing+of+the+TMS1%2FASC+tumour+suppressor+gene+by+an+epigenetic+mechanism+in+hepatocellular+carcinoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional silencing of the TMS1/ASC tumour suppressor gene by an epigenetic mechanism in hepatocellular carcinoma cells</span></div><div class="casAuthors">Zhang, C.; Li, H.; Zhou, G.; Zhang, Q.; Zhang, T.; Li, J.; Zhang, J.; Hou, J.; Liew, C. T.; Yin, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-142</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">DNA methylation and histone modifications have emerged as key mechanisms in transcriptional regulation.  The target of methylation-induced silencing 1 (TMS1) is a bipartite protein.  Recent studies have indicated that methylation-assocd. silencing of TMS1 occurs in many cancers.  However, whether and how TMS1 is regulated by epigenetic mechanisms in cancers remains unknown.  In this study we showed that methylation of the TMS1 promoter occurred in five of six hepatocellular carcinoma (HCC) cell lines.  TMS1 expression was reduced in four HCC cell lines and correlated with methylation status.  Furthermore, the TMS1 promoter was completely methylated and mRNA expression was undetectable.  TMS1 expression could be restored by 5-aza-2'-deoxycitidine (5-Aza-dC) (a DNA methyltransferase inhibitor) or trichostatin A (TSA) (a histone deacetylase inhibitor) alone and the promoter methylation was partially reversible.  TSA was more efficient than 5-Aza-dC in inducing TMS1 expression, and the combination of 5-Aza-dC and TSA resulted in markedly synergistic reactivation of the gene and completely reversed promoter methylation.  Interestingly, TMS1 promoter methylation-assocd. gene silencing was accompanied by histone H3 Lysine 9 (H3K9) hypoacetylation and trimethylation.  5-Aza-dC and/or TSA also had some effect on conversion of methylated to acetylated H3K9 in restoring TMS1.  This conversion was dynamic at the TMS1 promoter and a decrease in H3K9 trimethylation preceded an increase in H3K9 acetylation after 5-Aza-dC and/or TSA treatment.  Our results thus suggest that epigenetic inactivation of TMS1 expression is regulated by promoter hypermethylation and H3K9 modifications in a coordinated way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDySdFI4BelrVg90H21EOLACvtfcHk0lh7mh9g-8fYpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1Gkt7s%253D&md5=2c2c41bb60a712c01bd2089eb9205348</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fpath.2173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.2173%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC%26aulast%3DLi%26aufirst%3DH%26aulast%3DZhou%26aufirst%3DG%26aulast%3DZhang%26aufirst%3DQ%26aulast%3DZhang%26aufirst%3DT%26aulast%3DLi%26aufirst%3DJ%26aulast%3DZhang%26aufirst%3DJ%26aulast%3DHou%26aufirst%3DJ%26aulast%3DLiew%26aufirst%3DC.%2BT.%26aulast%3DYin%26aufirst%3DD.%26atitle%3DTranscriptional%2520silencing%2520of%2520the%2520TMS1%252FASC%2520tumour%2520suppressor%2520gene%2520by%2520an%2520epigenetic%2520mechanism%2520in%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DJ.%2520Pathol.%26date%3D2007%26volume%3D212%26spage%3D134%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Wong, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, O. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tung, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ching, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, I. O.</span><span> </span><span class="NLM_article-title">Tissue factor pathway inhibitor-2 as a frequently silenced tumor suppressor gene in hepatocellular carcinoma</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1138</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2007&pages=1129-1138&author=C.+M.+Wongauthor=Y.+L.+Ngauthor=J.+M.+Leeauthor=C.+C.+Wongauthor=O.+F.+Cheungauthor=C.+Y.+Chanauthor=E.+K.+Tungauthor=Y.+P.+Chingauthor=I.+O.+Ng&title=Tissue+factor+pathway+inhibitor-2+as+a+frequently+silenced+tumor+suppressor+gene+in+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.%2BM.%26aulast%3DNg%26aufirst%3DY.%2BL.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DWong%26aufirst%3DC.%2BC.%26aulast%3DCheung%26aufirst%3DO.%2BF.%26aulast%3DChan%26aufirst%3DC.%2BY.%26aulast%3DTung%26aufirst%3DE.%2BK.%26aulast%3DChing%26aufirst%3DY.%2BP.%26aulast%3DNg%26aufirst%3DI.%2BO.%26atitle%3DTissue%2520factor%2520pathway%2520inhibitor-2%2520as%2520a%2520frequently%2520silenced%2520tumor%2520suppressor%2520gene%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2007%26volume%3D45%26spage%3D1129%26epage%3D1138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">DNA methylation and gene silencing in cancer</span> <span class="citation_source-journal">Nat. Clin. Pract. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">S4</span><span class="NLM_x">–</span> <span class="NLM_lpage">S11</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=S4-S11&author=S.+B.+Baylin&title=DNA+methylation+and+gene+silencing+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DDNA%2520methylation%2520and%2520gene%2520silencing%2520in%2520cancer%26jtitle%3DNat.%2520Clin.%2520Pract.%2520Oncol.%26date%3D2005%26volume%3D2%26spage%3DS4%26epage%3DS11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Lu, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulp, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Z. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. S.</span><span> </span><span class="NLM_article-title">Efficacy of a novel histone deacetylase inhibitor in murine models of hepatocellular carcinoma</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1130</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=1119-1130&author=Y.+S.+Luauthor=Y.+Kashidaauthor=S.+K.+Kulpauthor=Y.+C.+Wangauthor=D.+Wangauthor=J.+H.+Hungauthor=M.+Tangauthor=Z.+Z.+Linauthor=T.+J.+Chenauthor=A.+L.+Chengauthor=C.+S.+Chen&title=Efficacy+of+a+novel+histone+deacetylase+inhibitor+in+murine+models+of+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%2BS.%26aulast%3DKashida%26aufirst%3DY.%26aulast%3DKulp%26aufirst%3DS.%2BK.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DHung%26aufirst%3DJ.%2BH.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DZ.%2BZ.%26aulast%3DChen%26aufirst%3DT.%2BJ.%26aulast%3DCheng%26aufirst%3DA.%2BL.%26aulast%3DChen%26aufirst%3DC.%2BS.%26atitle%3DEfficacy%2520of%2520a%2520novel%2520histone%2520deacetylase%2520inhibitor%2520in%2520murine%2520models%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2007%26volume%3D46%26spage%3D1119%26epage%3D1130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span>See for example: clinical trials.gov identifiers NCT00823290, NCT00943449, and NCT00537121 at <a href="http://www.clinicaltrials.gov" class="extLink">www.clinicaltrials.gov</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=See+for+example%3A+clinical+trials.gov+identifiers+NCT00823290%2C+NCT00943449%2C+and+NCT00537121+at+www.clinicaltrials.gov."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Miller, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witter, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvedere, S.</span><span> </span><span class="NLM_article-title">Histone deacetylase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">5097</span><span class="NLM_x">–</span> <span class="NLM_lpage">5116</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0303094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5097-5116&author=T.+A.+Millerauthor=D.+J.+Witterauthor=S.+Belvedere&title=Histone+deacetylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors</span></div><div class="casAuthors">Miller, Thomas A.; Witter, David J.; Belvedere, Sandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5097-5116</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the medicinal chem. and structure-activity relationships underlying advances in histone deacetylase (HDAC) class I/II inhibitor discovery, design, and optimization.  HDAC structural studies and HDAC inhibitory activity of small-mol. hydroxamic acids, carboxylates, benzamides, electrophilic ketones, and cyclic peptides are described.  Miscellaneousness and selectivity of the class I/II HDACs are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU5UlOsBUF7rVg90H21EOLACvtfcHk0lgm2FJtcDFPaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D&md5=d2d0a95d87d699334c1630df5deda38a</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1021%2Fjm0303094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303094%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DT.%2BA.%26aulast%3DWitter%26aufirst%3DD.%2BJ.%26aulast%3DBelvedere%26aufirst%3DS.%26atitle%3DHistone%2520deacetylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5097%26epage%3D5116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Wong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">Structural biasing elements for in-cell histone deacetylase paralog selectivity</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">5586</span><span class="NLM_x">–</span> <span class="NLM_lpage">5587</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0341440" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=5586-5587&author=J.+C.+Wongauthor=R.+Hongauthor=S.+L.+Schreiber&title=Structural+biasing+elements+for+in-cell+histone+deacetylase+paralog+selectivity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1021%2Fja0341440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0341440%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DHong%26aufirst%3DR.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DStructural%2520biasing%2520elements%2520for%2520in-cell%2520histone%2520deacetylase%2520paralog%2520selectivity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D5586%26epage%3D5587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Ning, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X. P.</span><span> </span><span class="NLM_article-title">Chidamide (CS055/HBI-8000): a new histone deacetylase inhibitor of the benzamide class with antitumor activity and the ability to enhance immune cell-mediated tumor cell cytotoxicity</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">901</span><span class="NLM_x">–</span> <span class="NLM_lpage">909</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1007%2Fs00280-011-1766-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=22080169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVCrsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2012&pages=901-909&author=Z.+Q.+Ningauthor=Z.+B.+Liauthor=M.+J.+Newmanauthor=S.+Shanauthor=X.+H.+Wangauthor=D.+S.+Panauthor=J.+Zhangauthor=M.+Dongauthor=X.+Duauthor=X.+P.+Lu&title=Chidamide+%28CS055%2FHBI-8000%29%3A+a+new+histone+deacetylase+inhibitor+of+the+benzamide+class+with+antitumor+activity+and+the+ability+to+enhance+immune+cell-mediated+tumor+cell+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Chidamide (CS055/HBI-8000): a new histone deacetylase inhibitor of the benzamide class with antitumor activity and the ability to enhance immune cell-mediated tumor cell cytotoxicity</span></div><div class="casAuthors">Ning, Zhi-Qiang; Li, Zhi-Bin; Newman, Michael J.; Shan, Song; Wang, Xin-Hao; Pan, De-Si; Zhang, Jin; Dong, Mei; Du, Xin; Lu, Xian-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">901-909</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose. Chidamide (CS055/HBI-8000) is a new histone deacetylase (HDAC) inhibitor of the benzamide class currently under clin. development in cancer indications.  This study reports the in vitro and in vivo antitumor characteristics of the compd.  Methods. Selectivity and potency of chidamide in inhibition of HDAC isotypes were analyzed by using a panel of human recombinant HDAC proteins.  Tumor cell lines either in culture or inoculated in nude mice were used for the evaluation of the compd.'s antitumor activity.  To investigate the immune cell-mediated antitumor effect, isolated peripheral blood mononuclear cells from healthy donors were treated with chidamide, and cytotoxicity and expression of relevant surface proteins were analyzed.  Microarray gene expression studies were performed on peripheral white blood cells from two T-cell lymphoma patients treated with chidamide.  Results. Chidamide was found to be a low nanomolar inhibitor of HDAC1, 2, 3, and 10, the HDAC isotypes well documented to be assocd. with the malignant phenotype.  Significant and broad spectrum in vitro and in vivo antitumor activity, including a wide therapeutic index, was obsd.  Chidamide was also shown to enhance the cytotoxic effect of human peripheral mononuclear cells ex vivo on K562 target cells, accompanied by the upregulation of proteins involved in NK cell functions.  Furthermore, the expression of a no. of genes involved in immune cell-mediated antitumor activity was obsd. to be upregulated in peripheral white blood cells from two T-cell lymphoma patients who responded to chidamide administration.  Conclusions. The results presented in this study provide evidence that chidamide has potential applicability for the treatment of a variety of tumor types, either as a single agent or in combination therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqPlwFoZiAfrVg90H21EOLACvtfcHk0lg7jhJ5Po0RYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVCrsr4%253D&md5=a0055f5a84ad0a9d8d1bd907339cab0f</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.1007%2Fs00280-011-1766-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-011-1766-x%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DZ.%2BQ.%26aulast%3DLi%26aufirst%3DZ.%2BB.%26aulast%3DNewman%26aufirst%3DM.%2BJ.%26aulast%3DShan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BH.%26aulast%3DPan%26aufirst%3DD.%2BS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DM.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%2BP.%26atitle%3DChidamide%2520%2528CS055%252FHBI-8000%2529%253A%2520a%2520new%2520histone%2520deacetylase%2520inhibitor%2520of%2520the%2520benzamide%2520class%2520with%2520antitumor%2520activity%2520and%2520the%2520ability%2520to%2520enhance%2520immune%2520cell-mediated%2520tumor%2520cell%2520cytotoxicity%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2012%26volume%3D69%26spage%3D901%26epage%3D909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Yap, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandhu, S. K</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Bono, J. S.</span><span> </span><span class="NLM_article-title">Envisioning the future of early anticancer drug development</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">514</span><span class="NLM_x">–</span> <span class="NLM_lpage">523</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1038%2Fnrc2870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=20535131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntF2ltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=514-523&author=T.+A.+Yapauthor=S.+K+Sandhuauthor=P.+Workmanauthor=J.+S.+de+Bono&title=Envisioning+the+future+of+early+anticancer+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Envisioning the future of early anticancer drug development</span></div><div class="casAuthors">Yap, Timothy A.; Sandhu, Shahneen K.; Workman, Paul; de Bono, Johann S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">514-523</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The development of novel molecularly targeted cancer therapeutics remains slow and expensive with many late-stage failures.  There is an urgent need to accelerate this process by improving early clin. anticancer drug evaluation through modern and rational trial designs that incorporate predictive, pharmacokinetic, pharmacodynamic, pharmacogenomic and intermediate end-point biomarkers.  In this article, we discuss current approaches and propose strategies that will potentially maximize benefit to patients and expedite the regulatory approvals of new anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2o6My3QVmz7Vg90H21EOLACvtfcHk0lg7jhJ5Po0RYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntF2ltr4%253D&md5=accb8dc22085dbea6da09c0b3d33c3ed</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc2870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2870%26sid%3Dliteratum%253Aachs%26aulast%3DYap%26aufirst%3DT.%2BA.%26aulast%3DSandhu%26aufirst%3DS.%2BK%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26atitle%3DEnvisioning%2520the%2520future%2520of%2520early%2520anticancer%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D514%26epage%3D523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Venugopal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, T. R.</span><span> </span><span class="NLM_article-title">Developing histone deacetylase inhibitors as anti-cancer therapeutics</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1658</span><span class="NLM_x">–</span> <span class="NLM_lpage">1671</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.2174%2F092986711795471284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=21428881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpslOiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1658-1671&author=B.+Venugopalauthor=T.+R.+Evans&title=Developing+histone+deacetylase+inhibitors+as+anti-cancer+therapeutics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Developing histone deacetylase inhibitors as anti-cancer therapeutics</span></div><div class="casAuthors">Venugopal, B.; Evans, T. R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1658-1671</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Post translational modification of histones and non-histone proteins by acetylation play a key role in tumorigenesis.  Histone deacetylases (HDACs) are enzymes involved in remodelling of chromatin by deacetylating the lysine residues and play a pivotal role in epigenetic regulation of gene expression.  An aberrant activity of HDACs has been documented in several types of cancers and HDACs have emerged as an attractive therapeutic target.  HDAC inhibitors (HDACi) are a structurally diverse group of anti-cancer agents which have a potential role in regulation of gene expression and induction of cell death, cell cycle arrest, and differentiation by altering the acetylation status of histone and non-histone proteins.  HDACi have pleiotropic effects on malignant cells and have demonstrated potent anti-cancer activity in pre-clin. studies.  A no. of clin. trials of HDACi as a monotherapy and/or in combination with conventional and novel chemotherapeutic drugs in solid and haematol. tumors have been published with variable efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolqVbj8pZQj7Vg90H21EOLACvtfcHk0lg7jhJ5Po0RYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpslOiu7o%253D&md5=4b530e4130570ddea322f5b0d5314b22</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F092986711795471284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711795471284%26sid%3Dliteratum%253Aachs%26aulast%3DVenugopal%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DT.%2BR.%26atitle%3DDeveloping%2520histone%2520deacetylase%2520inhibitors%2520as%2520anti-cancer%2520therapeutics%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D1658%26epage%3D1671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Khan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fotheringham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stimson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pezzella, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duvic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">HR23B is a biomarker for tumor sensitivity to HDAC inhibitor-based therapy</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">6532</span><span class="NLM_x">–</span> <span class="NLM_lpage">6537</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=6532-6537&author=O.+Khanauthor=S.+Fotheringhamauthor=V.+Woodauthor=L.+Stimsonauthor=C.+Zhangauthor=F.+Pezzellaauthor=M.+Duvicauthor=D.+J.+Kerrauthor=N.+B.+La+Thangue&title=HR23B+is+a+biomarker+for+tumor+sensitivity+to+HDAC+inhibitor-based+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DO.%26aulast%3DFotheringham%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DV.%26aulast%3DStimson%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DPezzella%26aufirst%3DF.%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DKerr%26aufirst%3DD.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DHR23B%2520is%2520a%2520biomarker%2520for%2520tumor%2520sensitivity%2520to%2520HDAC%2520inhibitor-based%2520therapy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D6532%26epage%3D6537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit18b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Miyanaga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gemma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kataoka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seike, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawakami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uesaka, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakae, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kudoh, S.</span><span> </span><span class="NLM_article-title">Antitumor activity of histone deacetylase inhibitors in non-small cell lung cancer cells: development of a molecular predictive model</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1923</span><span class="NLM_x">–</span> <span class="NLM_lpage">1930</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1158%2F1535-7163.MCT-07-2140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18606719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCgtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=1923-1930&author=A.+Miyanagaauthor=A.+Gemmaauthor=R.+Noroauthor=K.+Kataokaauthor=K.+Matsudaauthor=M.+Naraauthor=T.+Okanoauthor=M.+Seikeauthor=A.+Yoshimuraauthor=A.+Kawakamiauthor=H.+Uesakaauthor=H.+Nakaeauthor=S.+Kudoh&title=Antitumor+activity+of+histone+deacetylase+inhibitors+in+non-small+cell+lung+cancer+cells%3A+development+of+a+molecular+predictive+model"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity of histone deacetylase inhibitors in non-small cell lung cancer cells: development of a molecular predictive model</span></div><div class="casAuthors">Miyanaga, Akihiko; Gemma, Akihiko; Noro, Rintaro; Kataoka, Kiyoko; Matsuda, Kuniko; Nara, Michiya; Okano, Tetsuya; Seike, Masahiro; Yoshimura, Akinobu; Kawakami, Akiko; Uesaka, Haruka; Nakae, Hiroki; Kudoh, Shoji</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1923-1930</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">To ascertain the potential for histone deacetylase (HDAC) inhibitor-based treatment in non-small cell lung cancer (NSCLC), we analyzed the antitumor effects of trichostatin A (TSA) and suberoylanilide hydroxamic acid (vorinostat) in a panel of 16 NSCLC cell lines via 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide assay.  TSA and vorinostat both displayed strong antitumor activities in 50% of NSCLC cell lines, suggesting the need for the use of predictive markers to select patients receiving this treatment.  There was a strong correlation between the responsiveness to TSA and vorinostat (P < 0.0001).  To identify a mol. model of sensitivity to HDAC inhibitor treatment in NSCLC, we conducted a gene expression profiling study using cDNA arrays on the same set of cell lines and related the cytotoxic activity of TSA to corresponding gene expression pattern using a modified National Cancer Institute program.  In addn., pathway anal. was done with Pathway Architect software.  We used nine genes, which were identified by gene-drug sensitivity correlation and pathway anal., to build a support vector machine algorithm model by which sensitive cell lines were distinguished from resistant cell lines.  The prediction performance of the support vector machine model was validated by an addnl. nine cell lines, resulting in a prediction value of 100% with respect to detg. response to TSA and vorinostat.  Our results suggested that (a) HDAC inhibitors may be promising anticancer drugs to NSCLC and (b) the nine-gene classifier is useful in predicting drug sensitivity to HDAC inhibitors and may contribute to achieving individualized therapy for NSCLC patients. [Mol Cancer Ther 2008;7(7):1923-30].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgFNT3a7EIWbVg90H21EOLACvtfcHk0lij8ksubxaSIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCgtbY%253D&md5=e6a40b0bff8000734628b77edb9dd2cf</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2140%26sid%3Dliteratum%253Aachs%26aulast%3DMiyanaga%26aufirst%3DA.%26aulast%3DGemma%26aufirst%3DA.%26aulast%3DNoro%26aufirst%3DR.%26aulast%3DKataoka%26aufirst%3DK.%26aulast%3DMatsuda%26aufirst%3DK.%26aulast%3DNara%26aufirst%3DM.%26aulast%3DOkano%26aufirst%3DT.%26aulast%3DSeike%26aufirst%3DM.%26aulast%3DYoshimura%26aufirst%3DA.%26aulast%3DKawakami%26aufirst%3DA.%26aulast%3DUesaka%26aufirst%3DH.%26aulast%3DNakae%26aufirst%3DH.%26aulast%3DKudoh%26aufirst%3DS.%26atitle%3DAntitumor%2520activity%2520of%2520histone%2520deacetylase%2520inhibitors%2520in%2520non-small%2520cell%2520lung%2520cancer%2520cells%253A%2520development%2520of%2520a%2520molecular%2520predictive%2520model%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D1923%26epage%3D1930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit18c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Dejligbjerg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grauslund, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tjørnelund, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhl Jensen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sehested, M.</span><span> </span><span class="NLM_article-title">Identification of predictive biomarkers for the histone deacetylase inhibitor belinostat in a panel of human cancer cell lines</span> <span class="citation_source-journal">Cancer Biomark.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18c&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18503161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18c&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A280%3ADC%252BD1czjvFylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=101-109&author=M.+Dejligbjergauthor=M.+Grauslundauthor=I.+J.+Christensenauthor=J.+Tj%C3%B8rnelundauthor=P.+Buhl+Jensenauthor=M.+Sehested&title=Identification+of+predictive+biomarkers+for+the+histone+deacetylase+inhibitor+belinostat+in+a+panel+of+human+cancer+cell+lines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18cR"><div class="casContent"><span class="casTitleNuber">18c</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of predictive biomarkers for the histone deacetylase inhibitor belinostat in a panel of human cancer cell lines</span></div><div class="casAuthors">Dejligbjerg Marielle; Grauslund Morten; Christensen Ib Jarle; Tjornelund Jette; Buhl Jensen Peter; Sehested Maxwell</div><div class="citationInfo"><span class="NLM_cas:title">Cancer biomarkers : section A of Disease markers</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-9</span>
        ISSN:<span class="NLM_cas:issn">1574-0153</span>.
    </div><div class="casAbstract">Histone deacetylase inhibitors (HDACi) are promising epigenetic cancer chemotherapeutics rapidly approaching clinical use.  HDACi increases acetylation levels of histone and non-histone proteins and causes an alteration in gene-expression levels, ultimately resulting in proliferation arrest or apoptosis of especially cancer cells.  However, the precise mechanism of action of this class of therapeutics and the genes implicated in sensitivity remain obscure.  Hence, there is a need for identifying predictive biomarkers.  In this study, we examined the gene-expression levels of selected possible HDACi biomarkers, as suggested in the literature.  This was correlated with the inherent sensitivity towards the HDACi belinostat in a panel of 18 wild-type cancer cell lines with up to a 30-fold difference in chemosensitivity, which matched IC50 data from the NCI60 screen.  Of 16 genes examined, 4 showed a correlation in their expression levels to belinostat sensitivity: Ornithine decarboxylase (ODC1), v-ski sarcoma viral oncogene homolog (SKI), signal transducer and activator of transcription 1 (STAT1), and thymidylate synthetase (TYMS).  Including ODC and SKI simultaneously further strengthened the model.  Further, there was no correlation between sensitivity and intracellular belinostat uptake or with histone and tubulin acetylation.  Therefore, the genes identified in this study may be potential biomarkers for predicting clinical HDACi sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxd7G-wJtc6-kdrW1FkXG-fW6udTcc2eYTlTFhvOd1Abntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czjvFylug%253D%253D&md5=ed1743ce984ab02f86d8e11e8f8d46fc</span></div><a href="/servlet/linkout?suffix=cit18c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDejligbjerg%26aufirst%3DM.%26aulast%3DGrauslund%26aufirst%3DM.%26aulast%3DChristensen%26aufirst%3DI.%2BJ.%26aulast%3DTj%25C3%25B8rnelund%26aufirst%3DJ.%26aulast%3DBuhl%2BJensen%26aufirst%3DP.%26aulast%3DSehested%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520predictive%2520biomarkers%2520for%2520the%2520histone%2520deacetylase%2520inhibitor%2520belinostat%2520in%2520a%2520panel%2520of%2520human%2520cancer%2520cell%2520lines%26jtitle%3DCancer%2520Biomark.%26date%3D2008%26volume%3D4%26spage%3D101%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit18d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Fantin, V. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loboda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paweletz, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gooden, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrington, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randolph, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ware, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Constitutive activation of signal transducers and activators of transcription predicts vorinostat resistance in cutaneous T-cell lymphoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">3785</span><span class="NLM_x">–</span> <span class="NLM_lpage">3794</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=3785-3794&author=V.+R.+Fantinauthor=A.+Lobodaauthor=C.+P.+Paweletzauthor=R.+C.+Hendricksonauthor=J.+W.+Pierceauthor=J.+A.+Rothauthor=L.+Liauthor=F.+Goodenauthor=S.+Korenchukauthor=X.+S.+Houauthor=E.+A.+Harringtonauthor=S.+Randolphauthor=J.+F.+Reillyauthor=C.+M.+Wareauthor=M.+E.+Kadinauthor=S.+R.+Frankelauthor=V.+M.+Richon&title=Constitutive+activation+of+signal+transducers+and+activators+of+transcription+predicts+vorinostat+resistance+in+cutaneous+T-cell+lymphoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DLoboda%26aufirst%3DA.%26aulast%3DPaweletz%26aufirst%3DC.%2BP.%26aulast%3DHendrickson%26aufirst%3DR.%2BC.%26aulast%3DPierce%26aufirst%3DJ.%2BW.%26aulast%3DRoth%26aufirst%3DJ.%2BA.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DGooden%26aufirst%3DF.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DX.%2BS.%26aulast%3DHarrington%26aufirst%3DE.%2BA.%26aulast%3DRandolph%26aufirst%3DS.%26aulast%3DReilly%26aufirst%3DJ.%2BF.%26aulast%3DWare%26aufirst%3DC.%2BM.%26aulast%3DKadin%26aufirst%3DM.%2BE.%26aulast%3DFrankel%26aufirst%3DS.%2BR.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DConstitutive%2520activation%2520of%2520signal%2520transducers%2520and%2520activators%2520of%2520transcription%2520predicts%2520vorinostat%2520resistance%2520in%2520cutaneous%2520T-cell%2520lymphoma%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D3785%26epage%3D3794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit18e"><span><span class="NLM_label">(e) </span><div class="note"><p class="first last">For a general review on the subject of HDACi and biomarkers, refer to:</p></div><span class="NLM_contrib-group">Stimson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Biomarkers for predicting clinical responses to HDAC inhibitors</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18e&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2Fj.canlet.2009.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18e&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=19362413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18e&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsF2jt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2009&pages=177-183&author=L.+Stimsonauthor=N.+B.+La+Thangue&title=Biomarkers+for+predicting+clinical+responses+to+HDAC+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18eR"><div class="casContent"><span class="casTitleNuber">18e</span><div class="casTitle"><span class="NLM_cas:atitle">Biomarkers for predicting clinical responses to HDAC inhibitors</span></div><div class="casAuthors">Stimson, Lindsay; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-183</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histone and non-histone proteins by acetylation are known to play a key role in tumorigenesis.  Pharmacol. manipulation of acetylation has been possible with the identification of small mol. inhibitors of histone deacetylases (HDAC), the enzymes responsible for deacetylating lysine residues.  An explosion of drug discovery efforts in recent years has led to the development of an extensive group of HDAC inhibitors, many of which have been shown pre-clin. to have potent anti-tumor activity.  Clin. trials using these agents are now underway, with Vorinostat (suberoylanilide hydroxamic acid) having been approved by the FDA for treating cutaneous T-cell lymphoma (CTCL) in patients with progressive, persistent or recurrent disease.  This review discusses how biomarkers are being identified and used to expand our knowledge of the mechanisms by which HDAC inhibitors exhibit their anti-cancer effects.  In the longer term, biomarkers will provide a means towards achieving patient stratification in tumor types that will respond favorably to HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtCGFKlhhe8bVg90H21EOLACvtfcHk0lij8ksubxaSIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsF2jt7o%253D&md5=a925ab1ae931b77b309225ab086923de</span></div><a href="/servlet/linkout?suffix=cit18e&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2009.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2009.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DStimson%26aufirst%3DL.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DBiomarkers%2520for%2520predicting%2520clinical%2520responses%2520to%2520HDAC%2520inhibitors%26jtitle%3DCancer%2520Lett.%26date%3D2009%26volume%3D280%26spage%3D177%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Wong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ning, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span> </span><span class="NLM_article-title">Application of p21 and klf2 reporter gene assays to identify selective histone deacetylase inhibitors for cancer therapy</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=110-116&author=J.+C.+Wongauthor=L.+Guoauthor=Z.+H.+Pengauthor=W.+X.+Zhangauthor=N.+Zhangauthor=W.+Laiauthor=Z.+S.+Zhangauthor=S.+Songauthor=D.+S.+Panauthor=Z.+Q.+Ningauthor=X.+P.+Luauthor=X.+W.+Zhangauthor=C.+Zhangauthor=J.+Liauthor=C.+M.+Xieauthor=Y.+Heauthor=L.+Chen&title=Application+of+p21+and+klf2+reporter+gene+assays+to+identify+selective+histone+deacetylase+inhibitors+for+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DZ.%2BH.%26aulast%3DZhang%26aufirst%3DW.%2BX.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DLai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DZ.%2BS.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DD.%2BS.%26aulast%3DNing%26aufirst%3DZ.%2BQ.%26aulast%3DLu%26aufirst%3DX.%2BP.%26aulast%3DZhang%26aufirst%3DX.%2BW.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DC.%2BM.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DApplication%2520of%2520p21%2520and%2520klf2%2520reporter%2520gene%2520assays%2520to%2520identify%2520selective%2520histone%2520deacetylase%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D110%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Zhao, H.</span><span> </span><span class="NLM_article-title">Scaffold selection and scaffold hopping in lead generation: a medicinal chemistry perspective</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2Fj.drudis.2006.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=17275735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtleisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=149-155&author=H.+Zhao&title=Scaffold+selection+and+scaffold+hopping+in+lead+generation%3A+a+medicinal+chemistry+perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold selection and scaffold hopping in lead generation: a medicinal chemistry perspective</span></div><div class="casAuthors">Zhao, Hongyu</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">149-155</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Hit selection and lead generation are crucial for the success of the resource-demanding lead-optimization phase in drug discovery, and represent a major research area of medicinal chem. today.  Ligand-binding efficiency, ligand complexity, ligand-target profile complementarity and chem. tractability are important parameters in hit selection.  As synthesis and assay throughput improve, a large no. of analogs based on the same scaffold can be rapidly synthesized and tested.  Consequently, more chem. resources could be devoted to scaffold modifications to expand the candidate pool in lead generation.  Most recently discovered druggable targets are promiscuous toward lipophilic ligands, and the hydrophobic portions of hit compds. should be preferentially modified in analog and scaffold design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_G4HUVC1fQrVg90H21EOLACvtfcHk0liQsl9cD_cLCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtleisbo%253D&md5=a9ffb13a77e3bb18d5c432f3286d4608</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DH.%26atitle%3DScaffold%2520selection%2520and%2520scaffold%2520hopping%2520in%2520lead%2520generation%253A%2520a%2520medicinal%2520chemistry%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D149%26epage%3D155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Joucla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortier, J.</span><span> </span><span class="NLM_article-title">2,5-Unsubstituted pyrrolidines from formaldehyde and amino acids through in situ azomethine-ylide 1,3-dipolar cycloaddition to alkenes</span> <span class="citation_source-journal">J. Chem. Soc., Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1566</span><span class="NLM_x">–</span> <span class="NLM_lpage">1567</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1985&pages=1566-1567&author=M.+Jouclaauthor=J.+Mortier&title=2%2C5-Unsubstituted+pyrrolidines+from+formaldehyde+and+amino+acids+through+in+situ+azomethine-ylide+1%2C3-dipolar+cycloaddition+to+alkenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJoucla%26aufirst%3DM.%26aulast%3DMortier%26aufirst%3DJ.%26atitle%3D2%252C5-Unsubstituted%2520pyrrolidines%2520from%2520formaldehyde%2520and%2520amino%2520acids%2520through%2520in%2520situ%2520azomethine-ylide%25201%252C3-dipolar%2520cycloaddition%2520to%2520alkenes%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Chem.%2520Commun.%26date%3D1985%26volume%3D22%26spage%3D1566%26epage%3D1567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Evans, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mito, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, D.</span><span> </span><span class="NLM_article-title">Scope and mechanism of enantioselective Michael additions of 1,3-dicarbonyl compounds to nitroalkenes catalyzed by nickel(II)-diamine complexes</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">11583</span><span class="NLM_x">–</span> <span class="NLM_lpage">11592</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0735913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=11583-11592&author=D.+A.+Evansauthor=S.+Mitoauthor=D.+Seidel&title=Scope+and+mechanism+of+enantioselective+Michael+additions+of+1%2C3-dicarbonyl+compounds+to+nitroalkenes+catalyzed+by+nickel%28II%29-diamine+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja0735913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0735913%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DD.%2BA.%26aulast%3DMito%26aufirst%3DS.%26aulast%3DSeidel%26aufirst%3DD.%26atitle%3DScope%2520and%2520mechanism%2520of%2520enantioselective%2520Michael%2520additions%2520of%25201%252C3-dicarbonyl%2520compounds%2520to%2520nitroalkenes%2520catalyzed%2520by%2520nickel%2528II%2529-diamine%2520complexes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D11583%26epage%3D11592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budhu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajdu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keohane, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbach, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milligan, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shei, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chrebet, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergstrom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Card, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandala, S. M.</span><span> </span><span class="NLM_article-title">2-Aryl(pyrrolidin-4-yl)acetic acids are potent agonists of sphingosine-1-phosphate (S1P) receptors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3564</span><span class="NLM_x">–</span> <span class="NLM_lpage">3568</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=3564-3568&author=L.+Yanauthor=R.+Budhuauthor=P.+Huoauthor=C.+L.+Lynchauthor=J.+J.+Haleauthor=S.+G.+Millsauthor=R.+Hajduauthor=C.+A.+Keohaneauthor=M.+J.+Rosenbachauthor=J.+A.+Milliganauthor=G.+J.+Sheiauthor=G.+Chrebetauthor=J.+Bergstromauthor=D.+Cardauthor=S.+M.+Mandala&title=2-Aryl%28pyrrolidin-4-yl%29acetic+acids+are+potent+agonists+of+sphingosine-1-phosphate+%28S1P%29+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DL.%26aulast%3DBudhu%26aufirst%3DR.%26aulast%3DHuo%26aufirst%3DP.%26aulast%3DLynch%26aufirst%3DC.%2BL.%26aulast%3DHale%26aufirst%3DJ.%2BJ.%26aulast%3DMills%26aufirst%3DS.%2BG.%26aulast%3DHajdu%26aufirst%3DR.%26aulast%3DKeohane%26aufirst%3DC.%2BA.%26aulast%3DRosenbach%26aufirst%3DM.%2BJ.%26aulast%3DMilligan%26aufirst%3DJ.%2BA.%26aulast%3DShei%26aufirst%3DG.%2BJ.%26aulast%3DChrebet%26aufirst%3DG.%26aulast%3DBergstrom%26aufirst%3DJ.%26aulast%3DCard%26aufirst%3DD.%26aulast%3DMandala%26aufirst%3DS.%2BM.%26atitle%3D2-Aryl%2528pyrrolidin-4-yl%2529acetic%2520acids%2520are%2520potent%2520agonists%2520of%2520sphingosine-1-phosphate%2520%2528S1P%2529%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D3564%26epage%3D3568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span>For more information about HuPrime tumor-derived xenograft models versus traditional cell line-derived xenograft models, see the website of Crown Biosciences (Santa Clara, CA) at <a href="http://www.crownbio.com" class="extLink">www.crownbio.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+more+information+about+HuPrime+tumor-derived+xenograft+models+versus+traditional+cell+line-derived+xenograft+models%2C+see+the+website+of+Crown+Biosciences+%28Santa+Clara%2C+CA%29+at+www.crownbio.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span>For more information, see <a href="http://www.ingenuity.com" class="extLink">www.ingenuity.com</a> (Ingenuity Systems, Redwood City, CA).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+more+information%2C+see+www.ingenuity.com+%28Ingenuity+Systems%2C+Redwood+City%2C+CA%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span>See PubMeth website: <a href="http://methycancer.psych.ac.cn/MethyCancer.do" class="extLink">http://methycancer.psych.ac.cn/MethyCancer.do</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=See+PubMeth+website%3A+http%3A%2F%2Fmethycancer.psych.ac.cn%2FMethyCancer.do."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Cameron, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachman, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myöhänen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1038%2F5047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=9916800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1999&pages=103-107&author=E.+E.+Cameronauthor=K.+E.+Bachmanauthor=S.+My%C3%B6h%C3%A4nenauthor=J.+G.+Hermanauthor=S.+B.+Baylin&title=Synergy+of+demethylation+and+histone+deacetylase+inhibition+in+the+re-expression+of+genes+silenced+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer</span></div><div class="casAuthors">Cameron, Elizabeth E.; Bachman, Kurtis E.; Myohanen, Sanna; Herman, James G.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Densely methylated DNA assocs. with transcriptionally repressive chromatin characterized by the presence of underacetylated histones.  Recently, these 2 epigenetic processes were dynamically linked.  The methyl-CpG-binding protein MeCP2 appears to reside in a complex with histone deacetylase activity.  MeCP2 can mediate formation of transcriptionally repressive chromatin on methylated promoter templates in vitro, and this process can be reversed by trichostatin A (TSA), a specific inhibitor of histone deacetylase.  Little is known about the relative roles of methylation and histone deacetylase activity in the stable inhibition of transcription on densely methylated endogenous promoters, such as those for silenced alleles of imprinted genes, genes on the female inactive X chromosome, and tumor-suppressor genes inactivated in cancer cells.  The authors show that the hypermethylated genes MLH1, TIMP3, CDKN2B (INK4B, p15), and CDKN2A (INK4, p16) cannot be transcriptionally reactivated with TSA alone in tumor cells in which the authors have shown that TSA alone can upregulate the expression of non-methylated genes.  Following minimal demethylation and slight gene reactivation in the presence of low dose 5-aza-2' deoxycytidine (5Aza-dC), however, TSA treatment results in robust re-expression of each gene.  TSA does not contribute to demethylation of the genes, and none of the treatments alter the chromatin structure assocd. with the hypermethylated promoters.  Thus, although DNA methylation and histone deacetylation appear to act as synergistic layers for the silencing of genes in cancer, dense CpG island methylation is dominant for the stable maintenance of a silent state at these loci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZCmJwXaY4o7Vg90H21EOLACvtfcHk0lhmLYBUfu_elw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D&md5=5f0666dac86ee091d5e06ab0b9403502</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2F5047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F5047%26sid%3Dliteratum%253Aachs%26aulast%3DCameron%26aufirst%3DE.%2BE.%26aulast%3DBachman%26aufirst%3DK.%2BE.%26aulast%3DMy%25C3%25B6h%25C3%25A4nen%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DJ.%2BG.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSynergy%2520of%2520demethylation%2520and%2520histone%2520deacetylase%2520inhibition%2520in%2520the%2520re-expression%2520of%2520genes%2520silenced%2520in%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D1999%26volume%3D21%26spage%3D103%26epage%3D107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Morris, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulrahman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maina, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiesener, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teh, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Latif, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maher, E. R.</span><span> </span><span class="NLM_article-title">Tumor suppressor activity and epigenetic inactivation of hepatocyte growth factor activator inhibitor type 2/SPINT2 in papillary and clear cell renal cell carcinoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">4598</span><span class="NLM_x">–</span> <span class="NLM_lpage">4606</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=4598-4606&author=M.+R.+Morrisauthor=D.+Gentleauthor=M.+Abdulrahmanauthor=E.+N.+Mainaauthor=K.+Guptaauthor=R.+E.+Banksauthor=M.+S.+Wiesenerauthor=T.+Kishidaauthor=M.+Yaoauthor=B.+Tehauthor=F.+Latifauthor=E.+R.+Maher&title=Tumor+suppressor+activity+and+epigenetic+inactivation+of+hepatocyte+growth+factor+activator+inhibitor+type+2%2FSPINT2+in+papillary+and+clear+cell+renal+cell+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DM.%2BR.%26aulast%3DGentle%26aufirst%3DD.%26aulast%3DAbdulrahman%26aufirst%3DM.%26aulast%3DMaina%26aufirst%3DE.%2BN.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DBanks%26aufirst%3DR.%2BE.%26aulast%3DWiesener%26aufirst%3DM.%2BS.%26aulast%3DKishida%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTeh%26aufirst%3DB.%26aulast%3DLatif%26aufirst%3DF.%26aulast%3DMaher%26aufirst%3DE.%2BR.%26atitle%3DTumor%2520suppressor%2520activity%2520and%2520epigenetic%2520inactivation%2520of%2520hepatocyte%2520growth%2520factor%2520activator%2520inhibitor%2520type%25202%252FSPINT2%2520in%2520papillary%2520and%2520clear%2520cell%2520renal%2520cell%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D4598%26epage%3D4606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span>MacroModel, formerly known as BatchMin, is a general-purpose molecular mechanics program available from Schrödinger, L.L.C., New York. MacroModel was developed in the laboratories of Prof. Clark Still (Columbia University).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MacroModel%2C+formerly+known+as+BatchMin%2C+is+a+general-purpose+molecular+mechanics+program+available+from+Schr%C3%B6dinger%2C+L.L.C.%2C+New+York.+MacroModel+was+developed+in+the+laboratories+of+Prof.+Clark+Still+%28Columbia+University%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D.</span><span> </span><span class="NLM_article-title">Extra precision Glide: Docking and scoring incorporating a model of hydrophobic enclosure for protein–ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+Mainz&title=Extra+precision+Glide%3A+Docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein%E2%80%93ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%26atitle%3DExtra%2520precision%2520Glide%253A%2520Docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein%25E2%2580%2593ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Testino, S. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patonay, G.</span><span> </span><span class="NLM_article-title">High-throughput inhibition screening of major human cytochrome P450 enzymes using an in vitro cocktail and liquid chromatography-tandem mass spectrometry</span> <span class="citation_source-journal">J. Pharm. Biomed. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1459</span><span class="NLM_x">–</span> <span class="NLM_lpage">1467</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2FS0731-7085%2802%2900480-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=12467917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFKhsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2003&pages=1459-1467&author=S.+A.+J.+Testinoauthor=G.+Patonay&title=High-throughput+inhibition+screening+of+major+human+cytochrome+P450+enzymes+using+an+in+vitro+cocktail+and+liquid+chromatography-tandem+mass+spectrometry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31aR"><div class="casContent"><span class="casTitleNuber">31a</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput inhibition screening of major human cytochrome P450 enzymes using an in vitro cocktail and liquid chromatography-tandem mass spectrometry</span></div><div class="casAuthors">Testino, Samuel A.; Patonay, Gabor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1459-1467</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A method has been developed for the high-throughput inhibition screening of the major human cytochrome P 450 (CYP) enzymes (CYP1A2, CYP2C9, CYP2C19, CYP2D6, and CYP3A4) using an in vitro substrate cocktail and liq. chromatog.-tandem mass spectrometry (LC-MS-MS).  A cocktail consisting of the selective substrates phenacetin (CYP1A2), tolbutamide (CYP2C9), omeprazole (CYP2C19), bufuralol (CYP2D6), and midazolam (CYP3A4) was incubated with human liver microsomes.  The metabolic reactions were terminated with methanol contg. dextrorphan as an internal std.  Following centrifugation, the supernatant was analyzed by LC-MS-MS employing a fast gradient.  The concns. of the substrate metabolites-paracetamol, 4-hydroxytolbutamide, 5-hydroxyomeprazole, 1'-hydroxybufuralol, and 1'-hydroxymidazolam-in each sample were detd. by LC-MS-MS in a single assay.  The method was validated by incubating known CYP inhibitors (furafylline, CYP1A2; sulfaphenazole, CYP2C9; s-mephenytoin, CYP2C19; quinidine, CYP2D6; and troleandomycin, CYP3A4) with the individual substrates they were known to inhibit and with the substrate cocktail.  IC50s (μM) detd. using the substrate cocktail were in good agreement with those obtained with individual substrates (furafylline, 2.9 vs. 2.0; sulfaphenazole, 0.75 vs. 0.72; s-mephenytoin, 170 vs. 180; quinidine, 0.17 vs. 0.24; troleandomycin, 2.6 vs. 3.2) and with previously reported values in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYVLv3U7pcprVg90H21EOLACvtfcHk0lhmLYBUfu_elw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFKhsLw%253D&md5=68a59dc476e91abc7130c232cd816cb5</span></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=10.1016%2FS0731-7085%2802%2900480-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0731-7085%252802%252900480-6%26sid%3Dliteratum%253Aachs%26aulast%3DTestino%26aufirst%3DS.%2BA.%2BJ.%26aulast%3DPatonay%26aufirst%3DG.%26atitle%3DHigh-throughput%2520inhibition%2520screening%2520of%2520major%2520human%2520cytochrome%2520P450%2520enzymes%2520using%2520an%2520in%2520vitro%2520cocktail%2520and%2520liquid%2520chromatography-tandem%2520mass%2520spectrometry%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2003%26volume%3D30%26spage%3D1459%26epage%3D1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit31b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Walsky, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Validated assays for human cytochrome P450 activities</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1124%2Fdmd.32.6.647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=15155557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktlOgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=647-660&author=R.+L.+Walskyauthor=R.+S.+Obach&title=Validated+assays+for+human+cytochrome+P450+activities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31bR"><div class="casContent"><span class="casTitleNuber">31b</span><div class="casTitle"><span class="NLM_cas:atitle">Validated assays for human cytochrome P450 activities</span></div><div class="casAuthors">Walsky, Robert L.; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">647-660</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The measurement of the effect of new chem. entities on human cytochrome P 450 marker activities using in vitro experimentation represents an important exptl. approach in drug development.  In vitro drug interaction data can be used in guiding the design of clin. drug interaction studies, or, when no effect is obsd. in vitro, the data can be used in place of an in vivo study to claim that no interaction will occur in vivo.  To make such a claim, it must be assured that the in vitro expts. are performed with abs. confidence in the methods used and data obtained.  To meet this need, 12 semiautomated assays for human P 450 marker substrate activities have been developed and validated using approaches described in the GLP (good lab. practices) as per the code of U.S. Federal Regulations.  The assays that were validated are: phenacetin O-deethylase (CYP1A2), coumarin 7-hydroxylase (CYP2A6), bupropion hydroxylase (CYP2B6), amodiaquine N-deethylase (CYP2C8), diclofenac 4'-hydroxylase and tolbutamide methylhydroxylase (CYP2C9), (S)-mephenytoin 4'-hydroxylase (CYP2C19), dextromethorphan O-demethylase (CYP2D6), chlorzoxazone 6-hydroxylase (CYP2E1), felodipine dehydrogenase, testosterone 6β-hydroxylase, and midazolam 1'-hydroxylase (CYP3A4 and CYP3A5).  High-pressure liq. chromatog.-tandem mass spectrometry, using stable isotope-labeled internal stds., has been applied as the anal. method.  This anal. approach, through its high sensitivity and selectivity, has permitted the use of very low incubation concns. of microsomal protein (0.01-0.2 mg/mL).  Anal. assay accuracy and precision values were excellent.  Enzyme kinetic and inhibition parameters obtained using these methods demonstrated high precision and were within the range of values previously reported in the scientific literature.  These methods should prove useful in the routine assessments of the potential for new drug candidates to elicit pharmacokinetic drug interactions via inhibition of cytochrome P 450 activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwF04TGeyYbbVg90H21EOLACvtfcHk0liilkOsdErWIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktlOgt74%253D&md5=35532b907c15f1053aa8233dab2ec581</span></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.6.647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.6.647%26sid%3Dliteratum%253Aachs%26aulast%3DWalsky%26aufirst%3DR.%2BL.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DValidated%2520assays%2520for%2520human%2520cytochrome%2520P450%2520activities%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26spage%3D647%26epage%3D660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit31c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Weaver, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beattie, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riley, R. J.</span><span> </span><span class="NLM_article-title">Cytochrome P450 inhibition using recombinant proteins and mass spectrometry/multiple reaction monitoring technology in a cassette incubation</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">955</span><span class="NLM_x">–</span> <span class="NLM_lpage">966</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=955-966&author=R.+Weaverauthor=K.+S.+Grahamauthor=I.+G.+Beattieauthor=R.+J.+Riley&title=Cytochrome+P450+inhibition+using+recombinant+proteins+and+mass+spectrometry%2Fmultiple+reaction+monitoring+technology+in+a+cassette+incubation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWeaver%26aufirst%3DR.%26aulast%3DGraham%26aufirst%3DK.%2BS.%26aulast%3DBeattie%26aufirst%3DI.%2BG.%26aulast%3DRiley%26aufirst%3DR.%2BJ.%26atitle%3DCytochrome%2520P450%2520inhibition%2520using%2520recombinant%2520proteins%2520and%2520mass%2520spectrometry%252Fmultiple%2520reaction%2520monitoring%2520technology%2520in%2520a%2520cassette%2520incubation%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2003%26volume%3D31%26spage%3D955%26epage%3D966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Liang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, H.</span><span> </span><span class="NLM_article-title">Automated electrophysiology in the preclinical evaluation of drugs for potential QT prolongation</span> <span class="citation_source-journal">J. Pharmacol. Toxicol. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2Fj.vascn.2005.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=15936217" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2005&pages=123-135&author=G.+Liangauthor=H.+Guthrie&title=Automated+electrophysiology+in+the+preclinical+evaluation+of+drugs+for+potential+QT+prolongation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.vascn.2005.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.vascn.2005.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DG.%26aulast%3DGuthrie%26aufirst%3DH.%26atitle%3DAutomated%2520electrophysiology%2520in%2520the%2520preclinical%2520evaluation%2520of%2520drugs%2520for%2520potential%2520QT%2520prolongation%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2005%26volume%3D52%26spage%3D123%26epage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15a','cit15b','cit15c'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18a','cit18b','cit18c','cit18d','cit18e'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31a','cit31b','cit31c'],'ref32':['cit32']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 23 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhiyao Zheng, Angela van der Werf, Marie Deliaval, <span class="NLM_string-name hlFld-ContribAuthor">Nicklas Selander</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Fluorinated Amide Derivatives via a Radical N-Perfluoroalkylation–Defluorination Pathway. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (7)
                                     , 2791-2796. <a href="https://doi.org/10.1021/acs.orglett.0c00768" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c00768</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c00768%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSynthesis%252Bof%252BFluorinated%252BAmide%252BDerivatives%252Bvia%252Ba%252BRadical%252BN-Perfluoroalkylation%2525E2%252580%252593Defluorination%252BPathway%26aulast%3DZheng%26aufirst%3DZhiyao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28022020%26date%3D25032020%26volume%3D22%26issue%3D7%26spage%3D2791%26epage%3D2796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maxim Epifanov, Paul J. Foth, Frances Gu, Charlotte Barrillon, Sahil S. Kanani, Carolyn S. Higman, Jason E. Hein, <span class="NLM_string-name hlFld-ContribAuthor">Glenn M. Sammis</span>. </span><span class="cited-content_cbyCitation_article-title">One-Pot 1,1-Dihydrofluoroalkylation of Amines Using Sulfuryl Fluoride. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2018,</strong> <em>140 </em>
                                    (48)
                                     , 16464-16468. <a href="https://doi.org/10.1021/jacs.8b11309" title="DOI URL">https://doi.org/10.1021/jacs.8b11309</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b11309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b11309%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DOne-Pot%252B1%25252C1-Dihydrofluoroalkylation%252Bof%252BAmines%252BUsing%252BSulfuryl%252BFluoride%26aulast%3DEpifanov%26aufirst%3DMaxim%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D19102018%26date%3D19112018%26date%3D15112018%26volume%3D140%26issue%3D48%26spage%3D16464%26epage%3D16468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucía Gandolfi  Donadío</span>, <span class="hlFld-ContribAuthor ">Mariana A.  Galetti</span>, <span class="hlFld-ContribAuthor ">Gianluca  Giorgi</span>, <span class="hlFld-ContribAuthor ">Marcello  Rasparini</span>, and <span class="hlFld-ContribAuthor ">Maria J.  Comin</span>  . </span><span class="cited-content_cbyCitation_article-title">Anti-Selective Organocatalytic Michael Addition between Phenylacetaldehyde and Nitrostyrene. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2016,</strong> <em>81 </em>
                                    (17)
                                     , 7952-7957. <a href="https://doi.org/10.1021/acs.joc.6b01061" title="DOI URL">https://doi.org/10.1021/acs.joc.6b01061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.6b01061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.6b01061%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DAnti-Selective%252BOrganocatalytic%252BMichael%252BAddition%252Bbetween%252BPhenylacetaldehyde%252Band%252BNitrostyrene%26aulast%3DDonad%25C3%25ADo%26aufirst%3DLuc%25C3%25ADa%2BGandolfi%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D06052016%26date%3D01082016%26date%3D02092016%26date%3D21072016%26volume%3D81%26issue%3D17%26spage%3D7952%26epage%3D7957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junli  Chen</span>, <span class="hlFld-ContribAuthor ">Tianqi  Chen</span>, <span class="hlFld-ContribAuthor ">Qiupeng  Hu</span>, <span class="hlFld-ContribAuthor ">Kurt  Püntener</span>, <span class="hlFld-ContribAuthor ">Yi  Ren</span>, <span class="hlFld-ContribAuthor ">Jin  She</span>, <span class="hlFld-ContribAuthor ">Zhengming  Du</span>, and <span class="hlFld-ContribAuthor ">Michelangelo  Scalone</span>  . </span><span class="cited-content_cbyCitation_article-title">Practical Asymmetric Hydrogenation-Based Synthesis of a Class-Selective Histone Deacetylase Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Organic Process Research & Development</span><span> <strong>2014,</strong> <em>18 </em>
                                    (12)
                                     , 1702-1713. <a href="https://doi.org/10.1021/op500250b" title="DOI URL">https://doi.org/10.1021/op500250b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/op500250b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fop500250b%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Process%2520Research%2520%2526%2520Development%26atitle%3DPractical%252BAsymmetric%252BHydrogenation-Based%252BSynthesis%252Bof%252Ba%252BClass-Selective%252BHistone%252BDeacetylase%252BInhibitor%26aulast%3DChen%26aufirst%3DJunli%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D05082014%26date%3D29102014%26date%3D19122014%26volume%3D18%26issue%3D12%26spage%3D1702%26epage%3D1713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guozhi  Tang</span>, <span class="hlFld-ContribAuthor ">Jason C.  Wong</span>, <span class="hlFld-ContribAuthor ">Weixing  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhanguo  Wang</span>, <span class="hlFld-ContribAuthor ">Nan  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhenghong  Peng</span>, <span class="hlFld-ContribAuthor ">Zhenshan  Zhang</span>, <span class="hlFld-ContribAuthor ">Yiping  Rong</span>, <span class="hlFld-ContribAuthor ">Shijie  Li</span>, <span class="hlFld-ContribAuthor ">Meifang  Zhang</span>, <span class="hlFld-ContribAuthor ">Lingjie  Yu</span>, <span class="hlFld-ContribAuthor ">Teng  Feng</span>, <span class="hlFld-ContribAuthor ">Xiongwen  Zhang</span>, <span class="hlFld-ContribAuthor ">Xihan  Wu</span>, <span class="hlFld-ContribAuthor ">Jim Z.  Wu</span>, and <span class="hlFld-ContribAuthor ">Li  Chen</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of a Novel Aminotetralin Class of HDAC6 and HDAC8 Selective Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (19)
                                     , 8026-8034. <a href="https://doi.org/10.1021/jm5008962" title="DOI URL">https://doi.org/10.1021/jm5008962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm5008962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm5008962%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Ba%252BNovel%252BAminotetralin%252BClass%252Bof%252BHDAC6%252Band%252BHDAC8%252BSelective%252BInhibitors%26aulast%3DTang%26aufirst%3DGuozhi%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D12062014%26date%3D30092014%26date%3D09102014%26date%3D19092014%26volume%3D57%26issue%3D19%26spage%3D8026%26epage%3D8034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin E.  Blass</span>. </span><span class="cited-content_cbyCitation_article-title">In vitro ADME and in vivo pharmacokinetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 305-369. <a href="https://doi.org/10.1016/B978-0-12-817214-8.00006-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-817214-8.00006-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-817214-8.00006-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-817214-8.00006-3%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252Bvitro%252BADME%252Band%252Bin%252Bvivo%252Bpharmacokinetics%26aulast%3DBlass%26aufirst%3DBenjamin%2BE.%26date%3D2021%26spage%3D305%26epage%3D369%26pub%3DElsevier%26atitle%3DBasic%252BPrinciples%252Bof%252BDrug%252BDiscovery%252Band%252BDevelopment%26date%3D2021%26volume%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin E.  Blass</span>. </span><span class="cited-content_cbyCitation_article-title">Case studies in drug discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 625-664. <a href="https://doi.org/10.1016/B978-0-12-817214-8.00014-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-817214-8.00014-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-817214-8.00014-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-817214-8.00014-2%26sid%3Dliteratum%253Aachs%26atitle%3DCase%252Bstudies%252Bin%252Bdrug%252Bdiscovery%26aulast%3DBlass%26aufirst%3DBenjamin%2BE.%26date%3D2021%26spage%3D625%26epage%3D664%26pub%3DElsevier%26atitle%3DBasic%252BPrinciples%252Bof%252BDrug%252BDiscovery%252Band%252BDevelopment%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chu  Tang</span>, <span class="hlFld-ContribAuthor ">Yang  Du</span>, <span class="hlFld-ContribAuthor ">Qian  Liang</span>, <span class="hlFld-ContribAuthor ">Zhen  Cheng</span>, <span class="hlFld-ContribAuthor ">Jie  Tian</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Novel Histone Deacetylase-Targeted Near-Infrared Probe for Hepatocellular Carcinoma Imaging and Fluorescence Image-Guided Surgery. </span><span class="cited-content_cbyCitation_journal-name">Molecular Imaging and Biology</span><span> <strong>2020,</strong> <em>22 </em>
                                    (3)
                                     , 476-485. <a href="https://doi.org/10.1007/s11307-019-01389-4" title="DOI URL">https://doi.org/10.1007/s11307-019-01389-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11307-019-01389-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11307-019-01389-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Imaging%2520and%2520Biology%26atitle%3DDevelopment%252Bof%252Ba%252BNovel%252BHistone%252BDeacetylase-Targeted%252BNear-Infrared%252BProbe%252Bfor%252BHepatocellular%252BCarcinoma%252BImaging%252Band%252BFluorescence%252BImage-Guided%252BSurgery%26aulast%3DTang%26aufirst%3DChu%26date%3D2020%26date%3D2019%26volume%3D22%26issue%3D3%26spage%3D476%26epage%3D485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kunal  Nepali</span>, <span class="hlFld-ContribAuthor ">Ting-Yu  Chang</span>, <span class="hlFld-ContribAuthor ">Mei-Jung  Lai</span>, <span class="hlFld-ContribAuthor ">Kai-Cheng  Hsu</span>, <span class="hlFld-ContribAuthor ">Yun  Yen</span>, <span class="hlFld-ContribAuthor ">Tony Eight  Lin</span>, <span class="hlFld-ContribAuthor ">Sung-Bau  Lee</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Purine/purine isoster based scaffolds as new derivatives of benzamide class of HDAC inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>196 </em>, 112291. <a href="https://doi.org/10.1016/j.ejmech.2020.112291" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112291</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112291%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPurine%25252Fpurine%252Bisoster%252Bbased%252Bscaffolds%252Bas%252Bnew%252Bderivatives%252Bof%252Bbenzamide%252Bclass%252Bof%252BHDAC%252Binhibitors%26aulast%3DNepali%26aufirst%3DKunal%26date%3D2020%26volume%3D196%26spage%3D112291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bruna  Costa</span>, <span class="hlFld-ContribAuthor ">Marta F.  Estrada</span>, <span class="hlFld-ContribAuthor ">Raquel V.  Mendes</span>, <span class="hlFld-ContribAuthor ">Rita  Fior</span>. </span><span class="cited-content_cbyCitation_article-title">Zebrafish Avatars towards Personalized Medicine—A Comparative Review between Avatar Models. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2020,</strong> <em>9 </em>
                                    (2)
                                     , 293. <a href="https://doi.org/10.3390/cells9020293" title="DOI URL">https://doi.org/10.3390/cells9020293</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells9020293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells9020293%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DZebrafish%252BAvatars%252Btowards%252BPersonalized%252BMedicine%2525E2%252580%252594A%252BComparative%252BReview%252Bbetween%252BAvatar%252BModels%26aulast%3DCosta%26aufirst%3DBruna%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D2%26spage%3D293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Li</span>, <span class="hlFld-ContribAuthor ">Fang  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoxue  Chen</span>, <span class="hlFld-ContribAuthor ">Jie  Wang</span>, <span class="hlFld-ContribAuthor ">Yonglong  Zhao</span>, <span class="hlFld-ContribAuthor ">Yongjun  Li</span>, <span class="hlFld-ContribAuthor ">Bin  He</span>. </span><span class="cited-content_cbyCitation_article-title">Zinc-dependent Deacetylase (HDAC) Inhibitors with Different Zinc Binding Groups. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (3)
                                     , 223-241. <a href="https://doi.org/10.2174/1568026619666190122144949" title="DOI URL">https://doi.org/10.2174/1568026619666190122144949</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666190122144949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666190122144949%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DZinc-dependent%252BDeacetylase%252B%252528HDAC%252529%252BInhibitors%252Bwith%252BDifferent%252BZinc%252BBinding%252BGroups%26aulast%3DLi%26aufirst%3DYan%26date%3D2019%26volume%3D19%26issue%3D3%26spage%3D223%26epage%3D241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Diamantis I.  Tsilimigras</span>, <span class="hlFld-ContribAuthor ">Ioannis  Ntanasis-Stathopoulos</span>, <span class="hlFld-ContribAuthor ">Demetrios  Moris</span>, <span class="hlFld-ContribAuthor ">Eleftherios  Spartalis</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Pawlik</span>. </span><span class="cited-content_cbyCitation_article-title">Histone deacetylase inhibitors in hepatocellular carcinoma: A therapeutic perspective. </span><span class="cited-content_cbyCitation_journal-name">Surgical Oncology</span><span> <strong>2018,</strong> <em>27 </em>
                                    (4)
                                     , 611-618. <a href="https://doi.org/10.1016/j.suronc.2018.07.015" title="DOI URL">https://doi.org/10.1016/j.suronc.2018.07.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.suronc.2018.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.suronc.2018.07.015%26sid%3Dliteratum%253Aachs%26jtitle%3DSurgical%2520Oncology%26atitle%3DHistone%252Bdeacetylase%252Binhibitors%252Bin%252Bhepatocellular%252Bcarcinoma%25253A%252BA%252Btherapeutic%252Bperspective%26aulast%3DTsilimigras%26aufirst%3DDiamantis%2BI.%26date%3D2018%26volume%3D27%26issue%3D4%26spage%3D611%26epage%3D618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joseph B.  Sweeney</span>, <span class="hlFld-ContribAuthor ">Julien  Doulcet</span>, <span class="hlFld-ContribAuthor ">Bimod  Thapa</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 3-Substituted Pyrrolidines via Palladium-Catalyzed Hydroarylation. </span><span class="cited-content_cbyCitation_journal-name">iScience</span><span> <strong>2018,</strong> <em>9 </em>, 328-336. <a href="https://doi.org/10.1016/j.isci.2018.10.025" title="DOI URL">https://doi.org/10.1016/j.isci.2018.10.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.isci.2018.10.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.isci.2018.10.025%26sid%3Dliteratum%253Aachs%26jtitle%3DiScience%26atitle%3DSynthesis%252Bof%252B3-Substituted%252BPyrrolidines%252Bvia%252BPalladium-Catalyzed%252BHydroarylation%26aulast%3DSweeney%26aufirst%3DJoseph%2BB.%26date%3D2018%26volume%3D9%26spage%3D328%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Keith G.  Andrews</span>, <span class="hlFld-ContribAuthor ">Radmila  Faizova</span>, <span class="hlFld-ContribAuthor ">Ross M.  Denton</span>. </span><span class="cited-content_cbyCitation_article-title">A practical and catalyst-free trifluoroethylation reaction of amines using trifluoroacetic acid. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2017,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/ncomms15913" title="DOI URL">https://doi.org/10.1038/ncomms15913</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/ncomms15913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fncomms15913%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DA%252Bpractical%252Band%252Bcatalyst-free%252Btrifluoroethylation%252Breaction%252Bof%252Bamines%252Busing%252Btrifluoroacetic%252Bacid%26aulast%3DAndrews%26aufirst%3DKeith%2BG.%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christina Susanne  Mullins</span>, <span class="hlFld-ContribAuthor ">Stephanie  Bock</span>, <span class="hlFld-ContribAuthor ">Mathias  Krohn</span>, <span class="hlFld-ContribAuthor ">Michael  Linnebacher</span>. </span><span class="cited-content_cbyCitation_article-title">Generation of Xenotransplants from Human Cancer Biopsies to Assess Anti-cancer Activities of HDACi. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 217-229. <a href="https://doi.org/10.1007/978-1-4939-6527-4_16" title="DOI URL">https://doi.org/10.1007/978-1-4939-6527-4_16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-6527-4_16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-6527-4_16%26sid%3Dliteratum%253Aachs%26atitle%3DGeneration%252Bof%252BXenotransplants%252Bfrom%252BHuman%252BCancer%252BBiopsies%252Bto%252BAssess%252BAnti-cancer%252BActivities%252Bof%252BHDACi%26aulast%3DMullins%26aufirst%3DChristina%2BSusanne%26date%3D2017%26date%3D2016%26spage%3D217%26epage%3D229%26pub%3DSpringer%2520New%2520York%26atitle%3DHDAC%25252FHAT%252BFunction%252BAssessment%252Band%252BInhibitor%252BDevelopment%26aulast%3DKr%25C3%25A4mer%26aufirst%3DOliver%2BH.%26date%3D2017%26volume%3D1510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emanuele  Amata</span>, <span class="hlFld-ContribAuthor ">Nicholas D.  Bland</span>, <span class="hlFld-ContribAuthor ">Robert K.  Campbell</span>, <span class="hlFld-ContribAuthor ">Michael P.  Pollastri</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of pyrrolidine and pyrazolone derivatives as inhibitors of trypanosomal phosphodiesterase B1 (TbrPDEB1). </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2015,</strong> <em>56 </em>
                                    (21)
                                     , 2832-2835. <a href="https://doi.org/10.1016/j.tetlet.2015.04.061" title="DOI URL">https://doi.org/10.1016/j.tetlet.2015.04.061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2015.04.061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2015.04.061%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DEvaluation%252Bof%252Bpyrrolidine%252Band%252Bpyrazolone%252Bderivatives%252Bas%252Binhibitors%252Bof%252Btrypanosomal%252Bphosphodiesterase%252BB1%252B%252528TbrPDEB1%252529%26aulast%3DAmata%26aufirst%3DEmanuele%26date%3D2015%26volume%3D56%26issue%3D21%26spage%3D2832%26epage%3D2835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin E.  Blass</span>. </span><span class="cited-content_cbyCitation_article-title">In vitro ADME and In vivo Pharmacokinetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 245-306. <a href="https://doi.org/10.1016/B978-0-12-411508-8.00006-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-411508-8.00006-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-411508-8.00006-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-411508-8.00006-2%26sid%3Dliteratum%253Aachs%26atitle%3DIn%2525C2%2525A0vitro%252BADME%252Band%252BIn%2525C2%2525A0vivo%252BPharmacokinetics%26aulast%3DBlass%26aufirst%3DBenjamin%2BE.%26date%3D2015%26spage%3D245%26epage%3D306%26pub%3DElsevier%26atitle%3DBasic%252BPrinciples%252Bof%252BDrug%252BDiscovery%252Band%252BDevelopment%26date%3D2015%26volume%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin E.  Blass</span>. </span><span class="cited-content_cbyCitation_article-title">Case Studies in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 499-529. <a href="https://doi.org/10.1016/B978-0-12-411508-8.00013-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-411508-8.00013-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-411508-8.00013-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-411508-8.00013-X%26sid%3Dliteratum%253Aachs%26atitle%3DCase%252BStudies%252Bin%252BDrug%252BDiscovery%26aulast%3DBlass%26aufirst%3DBenjamin%2BE.%26date%3D2015%26spage%3D499%26epage%3D529%26pub%3DElsevier%26atitle%3DBasic%252BPrinciples%252Bof%252BDrug%252BDiscovery%252Band%252BDevelopment%26date%3D2015%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prerna  Malaney</span>, <span class="hlFld-ContribAuthor ">Santo V.  Nicosia</span>, <span class="hlFld-ContribAuthor ">Vrushank  Davé</span>. </span><span class="cited-content_cbyCitation_article-title">One mouse, one patient paradigm: New avatars of personalized cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2014,</strong> <em>344 </em>
                                    (1)
                                     , 1-12. <a href="https://doi.org/10.1016/j.canlet.2013.10.010" title="DOI URL">https://doi.org/10.1016/j.canlet.2013.10.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2013.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2013.10.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DOne%252Bmouse%25252C%252Bone%252Bpatient%252Bparadigm%25253A%252BNew%252Bavatars%252Bof%252Bpersonalized%252Bcancer%252Btherapy%26aulast%3DMalaney%26aufirst%3DPrerna%26date%3D2014%26volume%3D344%26issue%3D1%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Esther C.Y.  Lee</span>, <span class="hlFld-ContribAuthor ">Meihua  Tu</span>, <span class="hlFld-ContribAuthor ">Benjamin D.  Stevens</span>, <span class="hlFld-ContribAuthor ">Jianwei  Bian</span>, <span class="hlFld-ContribAuthor ">Gary  Aspnes</span>, <span class="hlFld-ContribAuthor ">Christian  Perreault</span>, <span class="hlFld-ContribAuthor ">Matthew F.  Sammons</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Wright</span>, <span class="hlFld-ContribAuthor ">John  Litchfield</span>, <span class="hlFld-ContribAuthor ">Amit S.  Kalgutkar</span>, <span class="hlFld-ContribAuthor ">Raman  Sharma</span>, <span class="hlFld-ContribAuthor ">Mary T.  Didiuk</span>, <span class="hlFld-ContribAuthor ">David C.  Ebner</span>, <span class="hlFld-ContribAuthor ">Kevin J.  Filipski</span>, <span class="hlFld-ContribAuthor ">Janice  Brown</span>, <span class="hlFld-ContribAuthor ">Karen  Atkinson</span>, <span class="hlFld-ContribAuthor ">Jeffrey A.  Pfefferkorn</span>, <span class="hlFld-ContribAuthor ">Angel  Guzman-Perez</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a novel conformationally constrained glucagon receptor antagonist. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (3)
                                     , 839-844. <a href="https://doi.org/10.1016/j.bmcl.2013.12.090" title="DOI URL">https://doi.org/10.1016/j.bmcl.2013.12.090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2013.12.090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2013.12.090%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252Ba%252Bnovel%252Bconformationally%252Bconstrained%252Bglucagon%252Breceptor%252Bantagonist%26aulast%3DLee%26aufirst%3DEsther%2BC.Y.%26date%3D2014%26volume%3D24%26issue%3D3%26spage%3D839%26epage%3D844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fatemeh Mazraati  Tajabadi</span>, <span class="hlFld-ContribAuthor ">Marc R.  Campitelli</span>, <span class="hlFld-ContribAuthor ">Ronald J.  Quinn</span>. </span><span class="cited-content_cbyCitation_article-title">Scaffold Flatness: Reversing the Trend. </span><span class="cited-content_cbyCitation_journal-name">Springer Science Reviews</span><span> <strong>2013,</strong> <em>1 </em>
                                    (1-2)
                                     , 141-151. <a href="https://doi.org/10.1007/s40362-013-0014-7" title="DOI URL">https://doi.org/10.1007/s40362-013-0014-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s40362-013-0014-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs40362-013-0014-7%26sid%3Dliteratum%253Aachs%26jtitle%3DSpringer%2520Science%2520Reviews%26atitle%3DScaffold%252BFlatness%25253A%252BReversing%252Bthe%252BTrend%26aulast%3DTajabadi%26aufirst%3DFatemeh%2BMazraati%26date%3D2013%26date%3D2013%26volume%3D1%26issue%3D1-2%26spage%3D141%26epage%3D151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam L  Cohen</span>, <span class="hlFld-ContribAuthor ">Stephen R  Piccolo</span>, <span class="hlFld-ContribAuthor ">Luis  Cheng</span>, <span class="hlFld-ContribAuthor ">Rafaella  Soldi</span>, <span class="hlFld-ContribAuthor ">Bing  Han</span>, <span class="hlFld-ContribAuthor ">W Evan  Johnson</span>, <span class="hlFld-ContribAuthor ">Andrea H  Bild</span>. </span><span class="cited-content_cbyCitation_article-title">Genomic pathway analysis reveals that EZH2 and HDAC4 represent mutually exclusive epigenetic pathways across human cancers. </span><span class="cited-content_cbyCitation_journal-name">BMC Medical Genomics</span><span> <strong>2013,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/1755-8794-6-35" title="DOI URL">https://doi.org/10.1186/1755-8794-6-35</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/1755-8794-6-35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2F1755-8794-6-35%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Medical%2520Genomics%26atitle%3DGenomic%252Bpathway%252Banalysis%252Breveals%252Bthat%252BEZH2%252Band%252BHDAC4%252Brepresent%252Bmutually%252Bexclusive%252Bepigenetic%252Bpathways%252Bacross%252Bhuman%252Bcancers%26aulast%3DCohen%26aufirst%3DAdam%2BL%26date%3D2013%26date%3D2013%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonas  Ecker</span>, <span class="hlFld-ContribAuthor ">Olaf  Witt</span>, <span class="hlFld-ContribAuthor ">Till  Milde</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting of histone deacetylases in brain tumors. </span><span class="cited-content_cbyCitation_journal-name">CNS Oncology</span><span> <strong>2013,</strong> <em>2 </em>
                                    (4)
                                     , 359-376. <a href="https://doi.org/10.2217/cns.13.24" title="DOI URL">https://doi.org/10.2217/cns.13.24</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/cns.13.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fcns.13.24%26sid%3Dliteratum%253Aachs%26jtitle%3DCNS%2520Oncology%26atitle%3DTargeting%252Bof%252Bhistone%252Bdeacetylases%252Bin%252Bbrain%252Btumors%26aulast%3DEcker%26aufirst%3DJonas%26date%3D2013%26volume%3D2%26issue%3D4%26spage%3D359%26epage%3D376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative HDAC inhibitors. (A) HDAC inhibitors that have been or are being evaluated in the clinic for HCC therapy; (B) <i>o</i>-aminoanilide HDAC inhibitors under clinical development.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Summary of compound <b>1</b> optimization history leading to identification of clinical lead <b>5</b> and the associated <i>trans</i>-3,4-disubstituted pyrrolidine lead series. P21_RP3 is the relative fold increase in luciferase signal in the p21 reporter gene assay compared to clinical HDACi entinostat (both test compound and comparator were evaluated at 3 μM in duplicate or triplicate) and normalized by internal GFP transfection control. CLh (MLM) is the calculated hepatic clearance based on mouse liver microsome intrinsic clearance. CL (mouse) is the in vivo systemic clearance measured by intravenous pharmacokinetic study in BALB/C nude mice. F is the bioavailability determined by single dose intravenous (2 mpk) and oral (10 mpk) pharmacokinetic study in nude mice. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for further details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Morpholine Side-Chain Replacement Analogues of Compound <b>1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) H<sub>2</sub>SO<sub>4</sub>, MeOH, 50 °C, 3 h, 77%; (b) Boc<sub>2</sub>O, THF, rt, 16 h, 77%; (c) acrylic acid, <i>N</i>-methylmorpholine, isobutyl chloroformate, DCM, 0 °C, 30 min and then add intermediate <b>II</b> at reflux over 30 min, stir 2 h, 34%; (d) tri-<i>o</i>-tolylphosphine, TEA, Pd<sub>2</sub>(dba)<sub>3</sub>, DMF, 100 °C, 6 h, 60%; (e) MsCl, TEA, DCM, 0 °C, 1 h; (f) R<sup>2</sup>NH<sub>2</sub> or R<sup>2</sup>R<sup>3</sup>NH, TEA, DCM, rt, 16 h; (g) LiOH, MeOH, rt, 5 h, 81% (three steps); (h) R<sup>1</sup>NH<sub>2</sub>, PyBrop, DIPEA, DCM, rt, 12 h; (i) 1.25 M HCl in MeOH, rt, 2 h, and then NaHCO<sub>3</sub>, 20–25% (two steps).</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0004.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of R<sup>1</sup> Analogues of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) Sarcosine, paraformaldehyde, toluene, reflux, 4 h, 46%; (b) <b>III</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 110 °C (sealed tube), 16 h, 87%; (c), LiOH, MeOH, rt, 16 h; (d) R<sup>1</sup> (substituted amine, aniline, or aminopyridine), HATU, TEA, DMF:DCM (1:10), rt, 3 h, 88% (two steps); (e) 1 N HCl in MeOH, rt, 4 h, 43%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of R<sup>2</sup> Analogues of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series via Alkylation, Epoxidation, Reductive Amination, Acylation, or Nucleophilic Aromatic Substitution<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) <i>N</i>-(Methoxymethyl)-<i>N</i>-(trimethylsilylmethyl)benzylamine, orthoboric acid, DCM, rt, 2 d then 45 °C for 2 h, 71%; (b) 2,2,2-trichloroethyl chloroformate, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 60 °C, 3 h, 66%; (c) Zn, acetic acid, rt, 3 h; (d) for alkylation (R<sup>2</sup>X where X is halide, tosylate, or mesylate): K<sub>2</sub>CO<sub>3</sub>, KI, R<sup>2</sup>X, DMF, 130 °C (microwave) 20 min; for epoxidation (2,2-dimethyloxirane): 2,2-dimethyloxirane, EtOH, 80 °C (sealed tube), 5 h, 96%; for reductive amination (R<sup>2</sup> = R<sup>3</sup>R<sup>4</sup>CH after reduction from R<sup>3</sup>R<sup>4</sup>CO): R<sup>3</sup>R<sup>4</sup>CO, NaBH(OAc)<sub>3</sub>, catalytic acetic acid, MeOH, rt, 16 h, 64%; for acylation: triphosgene and R<sup>3</sup>R<sup>4</sup>NH (R<sup>2</sup> = CONR<sup>3</sup>R<sup>4</sup>), DIPEA, THF, 0 °C to rt, 12 h; for nucleophilic aromatic substitution (2-chloropyrimidine): 2-chloropyrimidine, DIPEA, <i>i</i>-PrOH, 130 °C (microwave), 10 min, 60%; (e) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 100 °C, 3 h; (f) R<sup>1</sup> (substituted amine, aniline, or aminopyridine), LiOH, MeOH, rt, 16 h; (g) HATU, TEA, DMF:DCM (1:10), rt, 3 h; (h) 1 N HCl in MeOH, rt, 4 h.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0006.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Series Analogue <b>53</b> (R<sup>2</sup> = 2-hydroxy-1,1-dimethylethyl)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) NaOH, CH<sub>3</sub>CN, rt, 2 h then Na<sub>2</sub>CO<sub>3</sub>, Boc<sub>2</sub>O, rt, 16 h, 50%; (b) 4-chloroaniline, HOBt, EDCI, DCM, rt, 16 h; (c) 1.25 M HCl in MeOH, rt, 3 h then Na<sub>2</sub>CO<sub>3</sub>; (d) 2-bromo-2-methylpropionic acid methyl ester, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 5 h; (e) LiBH<sub>4</sub>, THF, rt, 2 h (17%, four steps); (f) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 100 °C, 3 h; (g) 1.25 M HCl, MeOH, rt, 3 h (24%, two steps).</p></p></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0007.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Analogue <b>52</b> (R<sup>2</sup> = <i>tert</i>-butyl)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) <i>N</i>-<i>tert</i>-butylglycine hydrochloride, paraformaldehyde, toluene, reflux, 4 h, 12%; (b) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 110 °C (sealed tube), 12 h, 53%; (c) LiOH, THF, rt, 16 h; (d) 4-chloroaniline, HATU, TEA, DCM, rt, 16 h, 66% (two steps); (e) 1.25 M HCl in MeOH, rt, 4 h, 27%.</p></p></figure><figure data-id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0008.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine Analogues <b>47</b> and <b>48</b> (R<sup>2</sup> = 1,1-difluoroethyl and 1,1,1-trifluoroethyl)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) For Y = CF<sub>3</sub>: triflic anhydride, TEA, DCM, 0 °C, 30 min and then to rt, 1 h, 68%; for Y = CF<sub>2</sub>H: difluoroacetyl chloride, DCM, rt, 1 h, 97%; (b) for Y = CF<sub>3</sub>: BH<sub>3</sub>, THF, 0 °C, 2 h, 30%; for Y = CF<sub>2</sub>H: BH<sub>3</sub>, THF, 60 °C, 5 h, 63% (c) NaOH H<sub>2</sub>O/MeOH, rt, 4 h; (d) 4-chloroaniline, PyBrop, DIPEA, DCM, rt, 16 h, 90%; (e) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 110 °C (sealed tube), 4 h, 86%; (f) TFA, DCM, rt, 2 h.</p></p></figure><figure data-id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0009.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine 5-Vinylpyridin-2-yl Linker Analogue <b>64</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) <i>i</i>-PrMgCl, THF, −20 °C, 2 h and then add DMF, 0 °C to rt, 1 h, 90%; (b) triethyl phosphonoacetate, TEA, LiBr, CH<sub>3</sub>CN, rt, 16 h, 67%; (c) sarcosine, paraformaldehyde, toluene, reflux, 4 h, 77%; (d) <i>N</i>-(2-(acryloylamino)phenyl)carbamic acid <i>tert</i>-butyl ester (<b>III</b>), Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>o</i>-tolylphosphine, TEA, DMF, 100 °C, 3 h, 75%; (e) LiOH, THF/H<sub>2</sub>O (3:1), rt, 5 h, 86%; (f) 4-chloroaniline, HATU, TEA, DCM, rt, 16 h, 54%; (g) 2 N HCl, MeOH, rt, 4 h.</p></p></figure><figure data-id="sch8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0010.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of <i>rac</i>-(<i>trans</i>-3,4)-Disubstituted Pyrrolidine 6-Vinylpyridin-3-yl Linker Analogue <b>61</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) Ethane-1,2-diol, TsOH, toluene, reflux, 6 h; (b) ethyl acrylate, Pd<sub>2</sub>(dba)<sub>3</sub>, tris-<i>o</i>-tolylphosphine, TEA, 110 °C, 3 h; (c) <i>N</i>-methylglycine, paraformaldehyde, toluene, reflux, 6 h; (d) LiCl, DMSO/H<sub>2</sub>O (3:1), 150 °C (microwave), 10 min; (e) diethyl <i>tert</i>-butyl phosphonoacetate, <i>n</i>-butyllithium, THF, −78 to 0 °C, 2 h; (f) LiOH, THF/H<sub>2</sub>O (3:1), rt, 3 h; (g) 4-chloroaniline, HATU, TEA, DCM, rt, 24 h; (h) TFA, DCM, 0 °C to rt, 2 h; (i) 1,2-diaminobenzene, HATU, TEA, DCM, rt, 12 h.</p></p></figure><figure data-id="fig3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docked conformational alignment by Glide software (see Experimental Section) of a prototypical <i>trans</i>-3,4-disubstituted pyrrolidine scaffold core (blue sticks) with that of the pyrrolidine-based side-chain analogues of <b>4</b> (yellow sticks). For clarity, HDAC1 homology model used for docking is not shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Generalized structure of the <i>trans</i>-3,4-disubstituted pyrrolidine series.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0019.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mechanistic and functional effects of compound <b>5</b> in representative hepatocellular carcinoma and gastric cancer cell lines. (A) Dose-dependent induction of endogenous p21 and acetylated histone H3 in response to 24 h treatment with clinical HDACi comparator mocetinostat or compound <b>5</b> in SGC-7901 cells. (B) Dose-dependent induction of cell cycle arrest by compound <b>5</b> in SGC-7901 cells at 24 h. (C) Induction of apoptosis at 48 h by compound <b>5</b> at 5 μM in SMMC-7721 cells. See Experimental Section for details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0020.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. In vivo antitumor efficacy and PK/PD correlation for compound <b>5</b>. (A) Effect of compound <b>5</b> on growth of an established BEL-7404 tumor in nude mice (<i>n</i> = 9–10) over four weeks. (B) Concentration of compound <b>5</b> in tumor tissue over 24 h period subsequent to the last dose. (C) Acetylated histone H3 induction in tumor tissue sampled at various time points after sacrificing mice subsequent to the last dose of compound <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0021.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. In vivo efficacy with PK/PD correlation for compound <b>5</b> in three patient tumor-derived xenograft models (top row = patient A; middle row = patient B; bottom row = patient C): (A) tumor growth curves for patients A–C treated with vehicle (15% Kleptose, pH 4.5 buffered with tartaric acid) or compound <b>5</b> Q2D PO 100 mpk. (B) Tumor concentration of compound <b>5</b> vs time for patients A–C (log 10 scale). Each square represents the concentration of <b>5</b> measured from a single mouse tumor. (C) Acetylated histone H3 IHC score (0–3) vs time for patients A–C (representative IHC sections scored as 0, 1, 2, or 3 available in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Figure S6). Each square represents the IHC score from a single mouse tumor. See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for further detail.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/medium/jm-2012-011838_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0022.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. mRNA expression levels of candidate genes with absolute fold change greater than two at any of three time points (day 7, day 15, and last day of study) in the three patient models. CCND2, CDKN2B, FN1, RASSF1, CDKN2A, HIC1, CEBPA, SDC2, and CFLAR are not shown because these genes did not change significantly throughout the course of the experiment. Expression levels were determined by qRT-PCR, normalized to GAPDH, and reported as fold-change relative to day 0 (expression level before treatment). See <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a> for further details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-20/jm3011838/production/images/large/jm-2012-011838_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011838&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53503" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53503" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 32 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">El-Seraq, H. B.</span><span> </span><span class="NLM_article-title">Hepatocellular carcinoma</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">365</span><span class="NLM_x">, </span> <span class="NLM_fpage">1118</span><span class="NLM_x">–</span> <span class="NLM_lpage">1127</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1056%2FNEJMra1001683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=21992124" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=1118-1127&author=H.+B.+El-Seraq&title=Hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1001683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1001683%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Seraq%26aufirst%3DH.%2BB.%26atitle%3DHepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D1118%26epage%3D1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Bruix, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llovet, J. M.</span><span> </span><span class="NLM_article-title">Prognostic prediction and treatment strategy in hepatocellular carcinoma</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">519</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2002&pages=519-524&author=J.+Bruixauthor=J.+M.+Llovet&title=Prognostic+prediction+and+treatment+strategy+in+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBruix%26aufirst%3DJ.%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26atitle%3DPrognostic%2520prediction%2520and%2520treatment%2520strategy%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2002%26volume%3D35%26spage%3D519%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Llovet, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ricci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzaferro, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilgard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gane, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanc, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Oliveira, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raoul, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeuzem, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolondi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greten, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galle, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitz, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borbath, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Häussinger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannaris, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moscovici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voliotis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruix, J.</span><span> </span><span class="NLM_article-title">Sorafenib in advanced hepatocellular carcinoma</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">378</span><span class="NLM_x">–</span> <span class="NLM_lpage">390</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1056%2FNEJMoa0708857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18650514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFWjsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2008&pages=378-390&author=J.+M.+Llovetauthor=S.+Ricciauthor=V.+Mazzaferroauthor=P.+Hilgardauthor=E.+Ganeauthor=J.+F.+Blancauthor=A.+C.+de+Oliveiraauthor=A.+Santoroauthor=J.+L.+Raoulauthor=A.+Fornerauthor=M.+Schwartzauthor=C.+Portaauthor=S.+Zeuzemauthor=L.+Bolondiauthor=T.+F.+Gretenauthor=P.+R.+Galleauthor=J.+F.+Seitzauthor=I.+Borbathauthor=D.+H%C3%A4ussingerauthor=T.+Giannarisauthor=M.+Shanauthor=M.+Moscoviciauthor=D.+Voliotisauthor=J.+Bruix&title=Sorafenib+in+advanced+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Sorafenib in advanced hepatocellular carcinoma</span></div><div class="casAuthors">Llovet, Josep M.; Ricci, Sergio; Mazzaferro, Vincenzo; Hilgard, Philip; Gane, Edward; Blanc, Jean-Frederic; Cosme de Oliveira, Andre; Santoro, Armando; Raoul, Jean-Luc; Forner, Alejandro; Schwartz, Myron; Porta, Camillo; Zeuzem, Stefan; Bolondi, Luigi; Greten, Tim F.; Galle, Peter R.; Seitz, Jean-Francois; Borbath, Ivan; Haussinger, Dieter; Giannaris, Tom; Shan, Minghua; Moscovici, Marius; Voliotis, Dimitris; Bruix, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">378-390</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">No effective systemic therapy exists for patients with advanced hepatocellular carcinoma.  A preliminary study suggested that sorafenib, an oral multi-kinase inhibitor of the vascular endothelial growth factor receptor, the platelet-derived growth factor receptor, and Raf may be effective in hepatocellular carcinoma.  In this multicenter, phase 3, double-blind, placebo-controlled trial, we randomly assigned 602 patients with advanced hepatocellular carcinoma who had not received previous systemic treatment to receive either sorafenib (at a dose of 400 mg twice daily) or placebo.  Primary outcomes were overall survival and the time to symptomatic progression.  Secondary outcomes included the time to radiol. progression and safety.  At the second planned interim anal., 321 deaths had occurred, and the study was stopped.  Median overall survival was 10.7 mo in the sorafenib group and 7.9 mo in the placebo group (hazard ratio in the sorafenib group, 0.69; 95% confidence interval, 0.55 to 0.87; P < 0.001).  There was no significant difference between the two groups in the median time to symptomatic progression (4.1 mo vs. 4.9 mo, resp., P = 0.77). The median time to radiol. progression was 5.5 mo in the sorafenib group and 2.8 mo in the placebo group (P < 0.001).  Seven patients in the sorafenib group (2%) and two patients in the placebo group (1%) had a partial response; no patients had a complete response.  Diarrhea, wt. loss, hand-foot skin reaction, and hypophosphatemia were more frequent in the sorafenib group.  In patients with advanced hepatocellular carcinoma, median survival and the time to radiol. progression were nearly 3 mo longer for patients treated with sorafenib than for those given placebo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6mcSPrj1TibVg90H21EOLACvtfcHk0lhzHGqnWJMlBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFWjsL8%253D&md5=579650dc323f35ad2415523c9827bf9a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0708857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0708857%26sid%3Dliteratum%253Aachs%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26aulast%3DRicci%26aufirst%3DS.%26aulast%3DMazzaferro%26aufirst%3DV.%26aulast%3DHilgard%26aufirst%3DP.%26aulast%3DGane%26aufirst%3DE.%26aulast%3DBlanc%26aufirst%3DJ.%2BF.%26aulast%3Dde%2BOliveira%26aufirst%3DA.%2BC.%26aulast%3DSantoro%26aufirst%3DA.%26aulast%3DRaoul%26aufirst%3DJ.%2BL.%26aulast%3DForner%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DM.%26aulast%3DPorta%26aufirst%3DC.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DBolondi%26aufirst%3DL.%26aulast%3DGreten%26aufirst%3DT.%2BF.%26aulast%3DGalle%26aufirst%3DP.%2BR.%26aulast%3DSeitz%26aufirst%3DJ.%2BF.%26aulast%3DBorbath%26aufirst%3DI.%26aulast%3DH%25C3%25A4ussinger%26aufirst%3DD.%26aulast%3DGiannaris%26aufirst%3DT.%26aulast%3DShan%26aufirst%3DM.%26aulast%3DMoscovici%26aufirst%3DM.%26aulast%3DVoliotis%26aufirst%3DD.%26aulast%3DBruix%26aufirst%3DJ.%26atitle%3DSorafenib%2520in%2520advanced%2520hepatocellular%2520carcinoma%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2008%26volume%3D359%26spage%3D378%26epage%3D390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Yang, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, E.</span><span> </span><span class="NLM_article-title">The Rpd3/Hda1 family of lysine deacetylases: from bacteria and yeast to mice and men</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">206</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1038%2Fnrm2346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18292778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlGnurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=206-218&author=X.+J.+Yangauthor=E.+Seto&title=The+Rpd3%2FHda1+family+of+lysine+deacetylases%3A+from+bacteria+and+yeast+to+mice+and+men"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Rpd3/Hda1 family of lysine deacetylases: from bacteria and yeast to mice and men</span></div><div class="casAuthors">Yang, Xiang-Jiao; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">206-218</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein lysine deacetylases have a pivotal role in numerous biol. processes and can be divided into the Rpd3/Hda1 and sirtuin families, each having members in diverse organisms including prokaryotes.  In vertebrates, the Rpd3/Hda1 family contains 11 members, traditionally referred to as histone deacetylases (HDAC) 1-11, which are further grouped into classes I, II and IV.  Whereas most class I HDACs are subunits of multiprotein nuclear complexes that are crucial for transcriptional repression and epigenetic landscaping, class II members regulate cytoplasmic processes or function as signal transducers that shuttle between the cytoplasm and the nucleus.  Little is known about class IV HDAC11, although its evolutionary conservation implies a fundamental role in various organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRiVyZexxGn7Vg90H21EOLACvtfcHk0lhzHGqnWJMlBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlGnurk%253D&md5=f11f89567feadf9cd38b3852dc44c382</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrm2346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2346%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%2BJ.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DThe%2520Rpd3%252FHda1%2520family%2520of%2520lysine%2520deacetylases%253A%2520from%2520bacteria%2520and%2520yeast%2520to%2520mice%2520and%2520men%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2008%26volume%3D9%26spage%3D206%26epage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Grozinger, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">Deacetylase enzymes: biological functions and the use of small-molecule inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">16</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2FS1074-5521%2802%2900092-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=11841934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=3-16&author=C.+M.+Grozingerauthor=S.+L.+Schreiber&title=Deacetylase+enzymes%3A+biological+functions+and+the+use+of+small-molecule+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Deacetylase enzymes: Biological functions and the use of small-molecule inhibitors</span></div><div class="casAuthors">Grozinger, Christina M.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-16</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This review discusses the mechanism of deacetylation by the histone deacetylases (HDAC) and the Sir2 (silent information regulator)-like family of NAD-dependent deacetylases, or sirtuins.  The use of specific small-mol. inhibitors of the deacetylase activity of HDAC and sirtuins may provide an effective technique for elucidating the biol. roles of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvpPshiJErULVg90H21EOLACvtfcHk0lhzHGqnWJMlBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2isLo%253D&md5=1eabb4f886e5777735ba14edadfaa332</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2802%2900092-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252802%252900092-3%26sid%3Dliteratum%253Aachs%26aulast%3DGrozinger%26aufirst%3DC.%2BM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DDeacetylase%2520enzymes%253A%2520biological%2520functions%2520and%2520the%2520use%2520of%2520small-molecule%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D3%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Rikimaru, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taketomi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirabe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maehara, Y.</span><span> </span><span class="NLM_article-title">Clinical significance of histone deacetylase 1 expression in patients with hepatocellular carcinoma</span> <span class="citation_source-journal">Oncology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">74</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=69-74&author=T.+Rikimaruauthor=A.+Taketomiauthor=Y.+Yamashitaauthor=K.+Shirabeauthor=T.+Hamatsuauthor=M.+Shimadaauthor=Y.+Maehara&title=Clinical+significance+of+histone+deacetylase+1+expression+in+patients+with+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRikimaru%26aufirst%3DT.%26aulast%3DTaketomi%26aufirst%3DA.%26aulast%3DYamashita%26aufirst%3DY.%26aulast%3DShirabe%26aufirst%3DK.%26aulast%3DHamatsu%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DM.%26aulast%3DMaehara%26aufirst%3DY.%26atitle%3DClinical%2520significance%2520of%2520histone%2520deacetylase%25201%2520expression%2520in%2520patients%2520with%2520hepatocellular%2520carcinoma%26jtitle%3DOncology%26date%3D2007%26volume%3D72%26spage%3D69%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Yoo, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Na, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seong, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. O.</span><span> </span><span class="NLM_article-title">Hepatitis B virus X protein induces the expression of MTA1 and HDAC1, which enhances hypoxia signaling in hepatocellular carcinoma cells</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">3405</span><span class="NLM_x">–</span> <span class="NLM_lpage">3413</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=3405-3413&author=Y.+G.+Yooauthor=T.+Y.+Naauthor=H.+W.+Seoauthor=J.+K.+Seongauthor=C.+K.+Parkauthor=Y.+K.+Shinauthor=M.+O.+Lee&title=Hepatitis+B+virus+X+protein+induces+the+expression+of+MTA1+and+HDAC1%2C+which+enhances+hypoxia+signaling+in+hepatocellular+carcinoma+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DY.%2BG.%26aulast%3DNa%26aufirst%3DT.%2BY.%26aulast%3DSeo%26aufirst%3DH.%2BW.%26aulast%3DSeong%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BK.%26aulast%3DShin%26aufirst%3DY.%2BK.%26aulast%3DLee%26aufirst%3DM.%2BO.%26atitle%3DHepatitis%2520B%2520virus%2520X%2520protein%2520induces%2520the%2520expression%2520of%2520MTA1%2520and%2520HDAC1%252C%2520which%2520enhances%2520hypoxia%2520signaling%2520in%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D3405%26epage%3D3413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, H.</span><span> </span><span class="NLM_article-title">Antitumor activity of Chidamide in hepatocellular carcinoma cell lines</span> <span class="citation_source-journal">Mol. Med. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1503</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=22484326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVKmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=1503-1508&author=H.+Wangauthor=Y.+Guoauthor=M.+Fuauthor=X.+Liangauthor=X.+Zhangauthor=R.+Wangauthor=C.+Linauthor=H.+Qian&title=Antitumor+activity+of+Chidamide+in+hepatocellular+carcinoma+cell+lines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity of Chidamide in hepatocellular carcinoma cell lines</span></div><div class="casAuthors">Wang, Haijuan; Guo, Yi; Fu, Ming; Liang, Xiao; Zhang, Xueyan; Wang, Renzhi; Lin, Chen; Qian, Haili</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1503-1508</span>CODEN:
                <span class="NLM_cas:coden">MMROA5</span>;
        ISSN:<span class="NLM_cas:issn">1791-2997</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Chidamide, the structural analog of MS-275, is a novel and promising histone deacetylase (HDAC) inhibitor for use in cancer therapy.  To investigate its effects on cancer cell growth, MTT assay was performed in 10 human cancer cell lines.  The data showed that the IC50 of Chidamide ranged from 1 to 13 μM, which was comparable to that of MS-275 in half of the tested cell lines.  Furthermore, the growth curve indicated that cell growth was gradually inhibited with an increase in Chidamide dosage, and the inhibition was reversed after drug removal in two hepatocelluclar carcinoma cell lines, BEL-7402 and HCC-9204.  To det. cell cycle and apoptosis, FACS was carried out in the BEL-7402 and HCC-9204 cells treated with Chidamide.  A decrease in the cell population at S phase and an increase in the cell population at G1 phase occurred in a dose-dependent manner.  In addn., Chidamide induced apoptosis and up-regulated p21 mRNA expression.  These results suggest that Chidamide may arrest the cell cycle and inhibit the growth of hepatocellular carcinoma cells through up-regulation of p21.  Further studies are required to clarify the antitumor activity of Chidamide in vivo and its mechanism in anticancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4L7-7ko2hsrVg90H21EOLACvtfcHk0liWwSFIpIcANg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVKmsbg%253D&md5=67c8639c2d449c485b791669a8aadc99</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DQian%26aufirst%3DH.%26atitle%3DAntitumor%2520activity%2520of%2520Chidamide%2520in%2520hepatocellular%2520carcinoma%2520cell%2520lines%26jtitle%3DMol.%2520Med.%2520Rep.%26date%3D2012%26volume%3D5%26spage%3D1503%26epage%3D1508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Ma, B. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sung, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poon, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lui, V. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeo, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, A. T.</span><span> </span><span class="NLM_article-title">The preclinical activity of the histone deacetylase inhibitor PXD101 (belinostat) in hepatocellular carcinoma cell lines</span> <span class="citation_source-journal">Invest. New Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=107-114&author=B.+B.+Maauthor=F.+Sungauthor=Q.+Taoauthor=F.+F.+Poonauthor=V.+W.+Luiauthor=W.+Yeoauthor=S.+L.+Chanauthor=A.+T.+Chan&title=The+preclinical+activity+of+the+histone+deacetylase+inhibitor+PXD101+%28belinostat%29+in+hepatocellular+carcinoma+cell+lines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DB.%2BB.%26aulast%3DSung%26aufirst%3DF.%26aulast%3DTao%26aufirst%3DQ.%26aulast%3DPoon%26aufirst%3DF.%2BF.%26aulast%3DLui%26aufirst%3DV.%2BW.%26aulast%3DYeo%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DS.%2BL.%26aulast%3DChan%26aufirst%3DA.%2BT.%26atitle%3DThe%2520preclinical%2520activity%2520of%2520the%2520histone%2520deacetylase%2520inhibitor%2520PXD101%2520%2528belinostat%2529%2520in%2520hepatocellular%2520carcinoma%2520cell%2520lines%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2010%26volume%3D28%26spage%3D107%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Zhang, C</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, G</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liew, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, D.</span><span> </span><span class="NLM_article-title">Transcriptional silencing of the TMS1/ASC tumour suppressor gene by an epigenetic mechanism in hepatocellular carcinoma cells</span> <span class="citation_source-journal">J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">212</span><span class="NLM_x">, </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">142</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1002%2Fpath.2173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=17471463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1Gkt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=212&publication_year=2007&pages=134-142&author=C+Zhangauthor=H+Liauthor=G+Zhouauthor=Q+Zhangauthor=T+Zhangauthor=J+Liauthor=J+Zhangauthor=J+Houauthor=C.+T.+Liewauthor=D.+Yin&title=Transcriptional+silencing+of+the+TMS1%2FASC+tumour+suppressor+gene+by+an+epigenetic+mechanism+in+hepatocellular+carcinoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional silencing of the TMS1/ASC tumour suppressor gene by an epigenetic mechanism in hepatocellular carcinoma cells</span></div><div class="casAuthors">Zhang, C.; Li, H.; Zhou, G.; Zhang, Q.; Zhang, T.; Li, J.; Zhang, J.; Hou, J.; Liew, C. T.; Yin, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-142</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">DNA methylation and histone modifications have emerged as key mechanisms in transcriptional regulation.  The target of methylation-induced silencing 1 (TMS1) is a bipartite protein.  Recent studies have indicated that methylation-assocd. silencing of TMS1 occurs in many cancers.  However, whether and how TMS1 is regulated by epigenetic mechanisms in cancers remains unknown.  In this study we showed that methylation of the TMS1 promoter occurred in five of six hepatocellular carcinoma (HCC) cell lines.  TMS1 expression was reduced in four HCC cell lines and correlated with methylation status.  Furthermore, the TMS1 promoter was completely methylated and mRNA expression was undetectable.  TMS1 expression could be restored by 5-aza-2'-deoxycitidine (5-Aza-dC) (a DNA methyltransferase inhibitor) or trichostatin A (TSA) (a histone deacetylase inhibitor) alone and the promoter methylation was partially reversible.  TSA was more efficient than 5-Aza-dC in inducing TMS1 expression, and the combination of 5-Aza-dC and TSA resulted in markedly synergistic reactivation of the gene and completely reversed promoter methylation.  Interestingly, TMS1 promoter methylation-assocd. gene silencing was accompanied by histone H3 Lysine 9 (H3K9) hypoacetylation and trimethylation.  5-Aza-dC and/or TSA also had some effect on conversion of methylated to acetylated H3K9 in restoring TMS1.  This conversion was dynamic at the TMS1 promoter and a decrease in H3K9 trimethylation preceded an increase in H3K9 acetylation after 5-Aza-dC and/or TSA treatment.  Our results thus suggest that epigenetic inactivation of TMS1 expression is regulated by promoter hypermethylation and H3K9 modifications in a coordinated way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDySdFI4BelrVg90H21EOLACvtfcHk0liWwSFIpIcANg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1Gkt7s%253D&md5=2c2c41bb60a712c01bd2089eb9205348</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fpath.2173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.2173%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC%26aulast%3DLi%26aufirst%3DH%26aulast%3DZhou%26aufirst%3DG%26aulast%3DZhang%26aufirst%3DQ%26aulast%3DZhang%26aufirst%3DT%26aulast%3DLi%26aufirst%3DJ%26aulast%3DZhang%26aufirst%3DJ%26aulast%3DHou%26aufirst%3DJ%26aulast%3DLiew%26aufirst%3DC.%2BT.%26aulast%3DYin%26aufirst%3DD.%26atitle%3DTranscriptional%2520silencing%2520of%2520the%2520TMS1%252FASC%2520tumour%2520suppressor%2520gene%2520by%2520an%2520epigenetic%2520mechanism%2520in%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DJ.%2520Pathol.%26date%3D2007%26volume%3D212%26spage%3D134%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Wong, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, O. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tung, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ching, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, I. O.</span><span> </span><span class="NLM_article-title">Tissue factor pathway inhibitor-2 as a frequently silenced tumor suppressor gene in hepatocellular carcinoma</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1138</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2007&pages=1129-1138&author=C.+M.+Wongauthor=Y.+L.+Ngauthor=J.+M.+Leeauthor=C.+C.+Wongauthor=O.+F.+Cheungauthor=C.+Y.+Chanauthor=E.+K.+Tungauthor=Y.+P.+Chingauthor=I.+O.+Ng&title=Tissue+factor+pathway+inhibitor-2+as+a+frequently+silenced+tumor+suppressor+gene+in+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.%2BM.%26aulast%3DNg%26aufirst%3DY.%2BL.%26aulast%3DLee%26aufirst%3DJ.%2BM.%26aulast%3DWong%26aufirst%3DC.%2BC.%26aulast%3DCheung%26aufirst%3DO.%2BF.%26aulast%3DChan%26aufirst%3DC.%2BY.%26aulast%3DTung%26aufirst%3DE.%2BK.%26aulast%3DChing%26aufirst%3DY.%2BP.%26aulast%3DNg%26aufirst%3DI.%2BO.%26atitle%3DTissue%2520factor%2520pathway%2520inhibitor-2%2520as%2520a%2520frequently%2520silenced%2520tumor%2520suppressor%2520gene%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2007%26volume%3D45%26spage%3D1129%26epage%3D1138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">DNA methylation and gene silencing in cancer</span> <span class="citation_source-journal">Nat. Clin. Pract. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">S4</span><span class="NLM_x">–</span> <span class="NLM_lpage">S11</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=S4-S11&author=S.+B.+Baylin&title=DNA+methylation+and+gene+silencing+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DDNA%2520methylation%2520and%2520gene%2520silencing%2520in%2520cancer%26jtitle%3DNat.%2520Clin.%2520Pract.%2520Oncol.%26date%3D2005%26volume%3D2%26spage%3DS4%26epage%3DS11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Lu, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulp, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Z. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. S.</span><span> </span><span class="NLM_article-title">Efficacy of a novel histone deacetylase inhibitor in murine models of hepatocellular carcinoma</span> <span class="citation_source-journal">Hepatology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1130</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=1119-1130&author=Y.+S.+Luauthor=Y.+Kashidaauthor=S.+K.+Kulpauthor=Y.+C.+Wangauthor=D.+Wangauthor=J.+H.+Hungauthor=M.+Tangauthor=Z.+Z.+Linauthor=T.+J.+Chenauthor=A.+L.+Chengauthor=C.+S.+Chen&title=Efficacy+of+a+novel+histone+deacetylase+inhibitor+in+murine+models+of+hepatocellular+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%2BS.%26aulast%3DKashida%26aufirst%3DY.%26aulast%3DKulp%26aufirst%3DS.%2BK.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DHung%26aufirst%3DJ.%2BH.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DZ.%2BZ.%26aulast%3DChen%26aufirst%3DT.%2BJ.%26aulast%3DCheng%26aufirst%3DA.%2BL.%26aulast%3DChen%26aufirst%3DC.%2BS.%26atitle%3DEfficacy%2520of%2520a%2520novel%2520histone%2520deacetylase%2520inhibitor%2520in%2520murine%2520models%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2007%26volume%3D46%26spage%3D1119%26epage%3D1130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span>See for example: clinical trials.gov identifiers NCT00823290, NCT00943449, and NCT00537121 at <a href="http://www.clinicaltrials.gov" class="extLink">www.clinicaltrials.gov</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=See+for+example%3A+clinical+trials.gov+identifiers+NCT00823290%2C+NCT00943449%2C+and+NCT00537121+at+www.clinicaltrials.gov."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Miller, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witter, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvedere, S.</span><span> </span><span class="NLM_article-title">Histone deacetylase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">5097</span><span class="NLM_x">–</span> <span class="NLM_lpage">5116</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0303094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5097-5116&author=T.+A.+Millerauthor=D.+J.+Witterauthor=S.+Belvedere&title=Histone+deacetylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors</span></div><div class="casAuthors">Miller, Thomas A.; Witter, David J.; Belvedere, Sandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5097-5116</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the medicinal chem. and structure-activity relationships underlying advances in histone deacetylase (HDAC) class I/II inhibitor discovery, design, and optimization.  HDAC structural studies and HDAC inhibitory activity of small-mol. hydroxamic acids, carboxylates, benzamides, electrophilic ketones, and cyclic peptides are described.  Miscellaneousness and selectivity of the class I/II HDACs are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU5UlOsBUF7rVg90H21EOLACvtfcHk0lg9M3B2nB9QYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D&md5=d2d0a95d87d699334c1630df5deda38a</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1021%2Fjm0303094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303094%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DT.%2BA.%26aulast%3DWitter%26aufirst%3DD.%2BJ.%26aulast%3DBelvedere%26aufirst%3DS.%26atitle%3DHistone%2520deacetylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5097%26epage%3D5116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Wong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">Structural biasing elements for in-cell histone deacetylase paralog selectivity</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">5586</span><span class="NLM_x">–</span> <span class="NLM_lpage">5587</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0341440" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=5586-5587&author=J.+C.+Wongauthor=R.+Hongauthor=S.+L.+Schreiber&title=Structural+biasing+elements+for+in-cell+histone+deacetylase+paralog+selectivity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1021%2Fja0341440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0341440%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DHong%26aufirst%3DR.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DStructural%2520biasing%2520elements%2520for%2520in-cell%2520histone%2520deacetylase%2520paralog%2520selectivity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D5586%26epage%3D5587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Ning, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X. P.</span><span> </span><span class="NLM_article-title">Chidamide (CS055/HBI-8000): a new histone deacetylase inhibitor of the benzamide class with antitumor activity and the ability to enhance immune cell-mediated tumor cell cytotoxicity</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">901</span><span class="NLM_x">–</span> <span class="NLM_lpage">909</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1007%2Fs00280-011-1766-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=22080169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVCrsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2012&pages=901-909&author=Z.+Q.+Ningauthor=Z.+B.+Liauthor=M.+J.+Newmanauthor=S.+Shanauthor=X.+H.+Wangauthor=D.+S.+Panauthor=J.+Zhangauthor=M.+Dongauthor=X.+Duauthor=X.+P.+Lu&title=Chidamide+%28CS055%2FHBI-8000%29%3A+a+new+histone+deacetylase+inhibitor+of+the+benzamide+class+with+antitumor+activity+and+the+ability+to+enhance+immune+cell-mediated+tumor+cell+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Chidamide (CS055/HBI-8000): a new histone deacetylase inhibitor of the benzamide class with antitumor activity and the ability to enhance immune cell-mediated tumor cell cytotoxicity</span></div><div class="casAuthors">Ning, Zhi-Qiang; Li, Zhi-Bin; Newman, Michael J.; Shan, Song; Wang, Xin-Hao; Pan, De-Si; Zhang, Jin; Dong, Mei; Du, Xin; Lu, Xian-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">901-909</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose. Chidamide (CS055/HBI-8000) is a new histone deacetylase (HDAC) inhibitor of the benzamide class currently under clin. development in cancer indications.  This study reports the in vitro and in vivo antitumor characteristics of the compd.  Methods. Selectivity and potency of chidamide in inhibition of HDAC isotypes were analyzed by using a panel of human recombinant HDAC proteins.  Tumor cell lines either in culture or inoculated in nude mice were used for the evaluation of the compd.'s antitumor activity.  To investigate the immune cell-mediated antitumor effect, isolated peripheral blood mononuclear cells from healthy donors were treated with chidamide, and cytotoxicity and expression of relevant surface proteins were analyzed.  Microarray gene expression studies were performed on peripheral white blood cells from two T-cell lymphoma patients treated with chidamide.  Results. Chidamide was found to be a low nanomolar inhibitor of HDAC1, 2, 3, and 10, the HDAC isotypes well documented to be assocd. with the malignant phenotype.  Significant and broad spectrum in vitro and in vivo antitumor activity, including a wide therapeutic index, was obsd.  Chidamide was also shown to enhance the cytotoxic effect of human peripheral mononuclear cells ex vivo on K562 target cells, accompanied by the upregulation of proteins involved in NK cell functions.  Furthermore, the expression of a no. of genes involved in immune cell-mediated antitumor activity was obsd. to be upregulated in peripheral white blood cells from two T-cell lymphoma patients who responded to chidamide administration.  Conclusions. The results presented in this study provide evidence that chidamide has potential applicability for the treatment of a variety of tumor types, either as a single agent or in combination therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqPlwFoZiAfrVg90H21EOLACvtfcHk0lhYe-V3Bg6HMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVCrsr4%253D&md5=a0055f5a84ad0a9d8d1bd907339cab0f</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.1007%2Fs00280-011-1766-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-011-1766-x%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DZ.%2BQ.%26aulast%3DLi%26aufirst%3DZ.%2BB.%26aulast%3DNewman%26aufirst%3DM.%2BJ.%26aulast%3DShan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BH.%26aulast%3DPan%26aufirst%3DD.%2BS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DM.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%2BP.%26atitle%3DChidamide%2520%2528CS055%252FHBI-8000%2529%253A%2520a%2520new%2520histone%2520deacetylase%2520inhibitor%2520of%2520the%2520benzamide%2520class%2520with%2520antitumor%2520activity%2520and%2520the%2520ability%2520to%2520enhance%2520immune%2520cell-mediated%2520tumor%2520cell%2520cytotoxicity%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2012%26volume%3D69%26spage%3D901%26epage%3D909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Yap, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandhu, S. K</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Bono, J. S.</span><span> </span><span class="NLM_article-title">Envisioning the future of early anticancer drug development</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">514</span><span class="NLM_x">–</span> <span class="NLM_lpage">523</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1038%2Fnrc2870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=20535131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntF2ltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=514-523&author=T.+A.+Yapauthor=S.+K+Sandhuauthor=P.+Workmanauthor=J.+S.+de+Bono&title=Envisioning+the+future+of+early+anticancer+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Envisioning the future of early anticancer drug development</span></div><div class="casAuthors">Yap, Timothy A.; Sandhu, Shahneen K.; Workman, Paul; de Bono, Johann S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">514-523</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The development of novel molecularly targeted cancer therapeutics remains slow and expensive with many late-stage failures.  There is an urgent need to accelerate this process by improving early clin. anticancer drug evaluation through modern and rational trial designs that incorporate predictive, pharmacokinetic, pharmacodynamic, pharmacogenomic and intermediate end-point biomarkers.  In this article, we discuss current approaches and propose strategies that will potentially maximize benefit to patients and expedite the regulatory approvals of new anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2o6My3QVmz7Vg90H21EOLACvtfcHk0lhYe-V3Bg6HMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntF2ltr4%253D&md5=accb8dc22085dbea6da09c0b3d33c3ed</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc2870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2870%26sid%3Dliteratum%253Aachs%26aulast%3DYap%26aufirst%3DT.%2BA.%26aulast%3DSandhu%26aufirst%3DS.%2BK%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26atitle%3DEnvisioning%2520the%2520future%2520of%2520early%2520anticancer%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D514%26epage%3D523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Venugopal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, T. R.</span><span> </span><span class="NLM_article-title">Developing histone deacetylase inhibitors as anti-cancer therapeutics</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1658</span><span class="NLM_x">–</span> <span class="NLM_lpage">1671</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.2174%2F092986711795471284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=21428881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpslOiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1658-1671&author=B.+Venugopalauthor=T.+R.+Evans&title=Developing+histone+deacetylase+inhibitors+as+anti-cancer+therapeutics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Developing histone deacetylase inhibitors as anti-cancer therapeutics</span></div><div class="casAuthors">Venugopal, B.; Evans, T. R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1658-1671</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Post translational modification of histones and non-histone proteins by acetylation play a key role in tumorigenesis.  Histone deacetylases (HDACs) are enzymes involved in remodelling of chromatin by deacetylating the lysine residues and play a pivotal role in epigenetic regulation of gene expression.  An aberrant activity of HDACs has been documented in several types of cancers and HDACs have emerged as an attractive therapeutic target.  HDAC inhibitors (HDACi) are a structurally diverse group of anti-cancer agents which have a potential role in regulation of gene expression and induction of cell death, cell cycle arrest, and differentiation by altering the acetylation status of histone and non-histone proteins.  HDACi have pleiotropic effects on malignant cells and have demonstrated potent anti-cancer activity in pre-clin. studies.  A no. of clin. trials of HDACi as a monotherapy and/or in combination with conventional and novel chemotherapeutic drugs in solid and haematol. tumors have been published with variable efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolqVbj8pZQj7Vg90H21EOLACvtfcHk0lhYe-V3Bg6HMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpslOiu7o%253D&md5=4b530e4130570ddea322f5b0d5314b22</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F092986711795471284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711795471284%26sid%3Dliteratum%253Aachs%26aulast%3DVenugopal%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DT.%2BR.%26atitle%3DDeveloping%2520histone%2520deacetylase%2520inhibitors%2520as%2520anti-cancer%2520therapeutics%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D1658%26epage%3D1671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Khan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fotheringham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stimson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pezzella, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duvic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">HR23B is a biomarker for tumor sensitivity to HDAC inhibitor-based therapy</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">6532</span><span class="NLM_x">–</span> <span class="NLM_lpage">6537</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=6532-6537&author=O.+Khanauthor=S.+Fotheringhamauthor=V.+Woodauthor=L.+Stimsonauthor=C.+Zhangauthor=F.+Pezzellaauthor=M.+Duvicauthor=D.+J.+Kerrauthor=N.+B.+La+Thangue&title=HR23B+is+a+biomarker+for+tumor+sensitivity+to+HDAC+inhibitor-based+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DO.%26aulast%3DFotheringham%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DV.%26aulast%3DStimson%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DPezzella%26aufirst%3DF.%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DKerr%26aufirst%3DD.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DHR23B%2520is%2520a%2520biomarker%2520for%2520tumor%2520sensitivity%2520to%2520HDAC%2520inhibitor-based%2520therapy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D6532%26epage%3D6537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit18b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Miyanaga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gemma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noro, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kataoka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seike, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawakami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uesaka, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakae, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kudoh, S.</span><span> </span><span class="NLM_article-title">Antitumor activity of histone deacetylase inhibitors in non-small cell lung cancer cells: development of a molecular predictive model</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1923</span><span class="NLM_x">–</span> <span class="NLM_lpage">1930</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1158%2F1535-7163.MCT-07-2140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18606719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18b&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCgtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=1923-1930&author=A.+Miyanagaauthor=A.+Gemmaauthor=R.+Noroauthor=K.+Kataokaauthor=K.+Matsudaauthor=M.+Naraauthor=T.+Okanoauthor=M.+Seikeauthor=A.+Yoshimuraauthor=A.+Kawakamiauthor=H.+Uesakaauthor=H.+Nakaeauthor=S.+Kudoh&title=Antitumor+activity+of+histone+deacetylase+inhibitors+in+non-small+cell+lung+cancer+cells%3A+development+of+a+molecular+predictive+model"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18bR"><div class="casContent"><span class="casTitleNuber">18b</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity of histone deacetylase inhibitors in non-small cell lung cancer cells: development of a molecular predictive model</span></div><div class="casAuthors">Miyanaga, Akihiko; Gemma, Akihiko; Noro, Rintaro; Kataoka, Kiyoko; Matsuda, Kuniko; Nara, Michiya; Okano, Tetsuya; Seike, Masahiro; Yoshimura, Akinobu; Kawakami, Akiko; Uesaka, Haruka; Nakae, Hiroki; Kudoh, Shoji</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1923-1930</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">To ascertain the potential for histone deacetylase (HDAC) inhibitor-based treatment in non-small cell lung cancer (NSCLC), we analyzed the antitumor effects of trichostatin A (TSA) and suberoylanilide hydroxamic acid (vorinostat) in a panel of 16 NSCLC cell lines via 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide assay.  TSA and vorinostat both displayed strong antitumor activities in 50% of NSCLC cell lines, suggesting the need for the use of predictive markers to select patients receiving this treatment.  There was a strong correlation between the responsiveness to TSA and vorinostat (P < 0.0001).  To identify a mol. model of sensitivity to HDAC inhibitor treatment in NSCLC, we conducted a gene expression profiling study using cDNA arrays on the same set of cell lines and related the cytotoxic activity of TSA to corresponding gene expression pattern using a modified National Cancer Institute program.  In addn., pathway anal. was done with Pathway Architect software.  We used nine genes, which were identified by gene-drug sensitivity correlation and pathway anal., to build a support vector machine algorithm model by which sensitive cell lines were distinguished from resistant cell lines.  The prediction performance of the support vector machine model was validated by an addnl. nine cell lines, resulting in a prediction value of 100% with respect to detg. response to TSA and vorinostat.  Our results suggested that (a) HDAC inhibitors may be promising anticancer drugs to NSCLC and (b) the nine-gene classifier is useful in predicting drug sensitivity to HDAC inhibitors and may contribute to achieving individualized therapy for NSCLC patients. [Mol Cancer Ther 2008;7(7):1923-30].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgFNT3a7EIWbVg90H21EOLACvtfcHk0lg9fllVsFW4LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCgtbY%253D&md5=e6a40b0bff8000734628b77edb9dd2cf</span></div><a href="/servlet/linkout?suffix=cit18b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2140%26sid%3Dliteratum%253Aachs%26aulast%3DMiyanaga%26aufirst%3DA.%26aulast%3DGemma%26aufirst%3DA.%26aulast%3DNoro%26aufirst%3DR.%26aulast%3DKataoka%26aufirst%3DK.%26aulast%3DMatsuda%26aufirst%3DK.%26aulast%3DNara%26aufirst%3DM.%26aulast%3DOkano%26aufirst%3DT.%26aulast%3DSeike%26aufirst%3DM.%26aulast%3DYoshimura%26aufirst%3DA.%26aulast%3DKawakami%26aufirst%3DA.%26aulast%3DUesaka%26aufirst%3DH.%26aulast%3DNakae%26aufirst%3DH.%26aulast%3DKudoh%26aufirst%3DS.%26atitle%3DAntitumor%2520activity%2520of%2520histone%2520deacetylase%2520inhibitors%2520in%2520non-small%2520cell%2520lung%2520cancer%2520cells%253A%2520development%2520of%2520a%2520molecular%2520predictive%2520model%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D1923%26epage%3D1930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit18c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Dejligbjerg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grauslund, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tjørnelund, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhl Jensen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sehested, M.</span><span> </span><span class="NLM_article-title">Identification of predictive biomarkers for the histone deacetylase inhibitor belinostat in a panel of human cancer cell lines</span> <span class="citation_source-journal">Cancer Biomark.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18c&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=18503161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18c&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A280%3ADC%252BD1czjvFylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=101-109&author=M.+Dejligbjergauthor=M.+Grauslundauthor=I.+J.+Christensenauthor=J.+Tj%C3%B8rnelundauthor=P.+Buhl+Jensenauthor=M.+Sehested&title=Identification+of+predictive+biomarkers+for+the+histone+deacetylase+inhibitor+belinostat+in+a+panel+of+human+cancer+cell+lines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18cR"><div class="casContent"><span class="casTitleNuber">18c</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of predictive biomarkers for the histone deacetylase inhibitor belinostat in a panel of human cancer cell lines</span></div><div class="casAuthors">Dejligbjerg Marielle; Grauslund Morten; Christensen Ib Jarle; Tjornelund Jette; Buhl Jensen Peter; Sehested Maxwell</div><div class="citationInfo"><span class="NLM_cas:title">Cancer biomarkers : section A of Disease markers</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-9</span>
        ISSN:<span class="NLM_cas:issn">1574-0153</span>.
    </div><div class="casAbstract">Histone deacetylase inhibitors (HDACi) are promising epigenetic cancer chemotherapeutics rapidly approaching clinical use.  HDACi increases acetylation levels of histone and non-histone proteins and causes an alteration in gene-expression levels, ultimately resulting in proliferation arrest or apoptosis of especially cancer cells.  However, the precise mechanism of action of this class of therapeutics and the genes implicated in sensitivity remain obscure.  Hence, there is a need for identifying predictive biomarkers.  In this study, we examined the gene-expression levels of selected possible HDACi biomarkers, as suggested in the literature.  This was correlated with the inherent sensitivity towards the HDACi belinostat in a panel of 18 wild-type cancer cell lines with up to a 30-fold difference in chemosensitivity, which matched IC50 data from the NCI60 screen.  Of 16 genes examined, 4 showed a correlation in their expression levels to belinostat sensitivity: Ornithine decarboxylase (ODC1), v-ski sarcoma viral oncogene homolog (SKI), signal transducer and activator of transcription 1 (STAT1), and thymidylate synthetase (TYMS).  Including ODC and SKI simultaneously further strengthened the model.  Further, there was no correlation between sensitivity and intracellular belinostat uptake or with histone and tubulin acetylation.  Therefore, the genes identified in this study may be potential biomarkers for predicting clinical HDACi sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxd7G-wJtc6-kdrW1FkXG-fW6udTcc2eb13PCy1Xc2D7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czjvFylug%253D%253D&md5=ed1743ce984ab02f86d8e11e8f8d46fc</span></div><a href="/servlet/linkout?suffix=cit18c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDejligbjerg%26aufirst%3DM.%26aulast%3DGrauslund%26aufirst%3DM.%26aulast%3DChristensen%26aufirst%3DI.%2BJ.%26aulast%3DTj%25C3%25B8rnelund%26aufirst%3DJ.%26aulast%3DBuhl%2BJensen%26aufirst%3DP.%26aulast%3DSehested%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520predictive%2520biomarkers%2520for%2520the%2520histone%2520deacetylase%2520inhibitor%2520belinostat%2520in%2520a%2520panel%2520of%2520human%2520cancer%2520cell%2520lines%26jtitle%3DCancer%2520Biomark.%26date%3D2008%26volume%3D4%26spage%3D101%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit18d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Fantin, V. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loboda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paweletz, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gooden, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrington, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randolph, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reilly, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ware, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frankel, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Constitutive activation of signal transducers and activators of transcription predicts vorinostat resistance in cutaneous T-cell lymphoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">3785</span><span class="NLM_x">–</span> <span class="NLM_lpage">3794</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=3785-3794&author=V.+R.+Fantinauthor=A.+Lobodaauthor=C.+P.+Paweletzauthor=R.+C.+Hendricksonauthor=J.+W.+Pierceauthor=J.+A.+Rothauthor=L.+Liauthor=F.+Goodenauthor=S.+Korenchukauthor=X.+S.+Houauthor=E.+A.+Harringtonauthor=S.+Randolphauthor=J.+F.+Reillyauthor=C.+M.+Wareauthor=M.+E.+Kadinauthor=S.+R.+Frankelauthor=V.+M.+Richon&title=Constitutive+activation+of+signal+transducers+and+activators+of+transcription+predicts+vorinostat+resistance+in+cutaneous+T-cell+lymphoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DLoboda%26aufirst%3DA.%26aulast%3DPaweletz%26aufirst%3DC.%2BP.%26aulast%3DHendrickson%26aufirst%3DR.%2BC.%26aulast%3DPierce%26aufirst%3DJ.%2BW.%26aulast%3DRoth%26aufirst%3DJ.%2BA.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DGooden%26aufirst%3DF.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DX.%2BS.%26aulast%3DHarrington%26aufirst%3DE.%2BA.%26aulast%3DRandolph%26aufirst%3DS.%26aulast%3DReilly%26aufirst%3DJ.%2BF.%26aulast%3DWare%26aufirst%3DC.%2BM.%26aulast%3DKadin%26aufirst%3DM.%2BE.%26aulast%3DFrankel%26aufirst%3DS.%2BR.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DConstitutive%2520activation%2520of%2520signal%2520transducers%2520and%2520activators%2520of%2520transcription%2520predicts%2520vorinostat%2520resistance%2520in%2520cutaneous%2520T-cell%2520lymphoma%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D3785%26epage%3D3794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit18e"><span><span class="NLM_label">(e) </span><div class="note"><p class="first last">For a general review on the subject of HDACi and biomarkers, refer to:</p></div><span class="NLM_contrib-group">Stimson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Biomarkers for predicting clinical responses to HDAC inhibitors</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18e&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2Fj.canlet.2009.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18e&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=19362413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18e&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsF2jt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2009&pages=177-183&author=L.+Stimsonauthor=N.+B.+La+Thangue&title=Biomarkers+for+predicting+clinical+responses+to+HDAC+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18eR"><div class="casContent"><span class="casTitleNuber">18e</span><div class="casTitle"><span class="NLM_cas:atitle">Biomarkers for predicting clinical responses to HDAC inhibitors</span></div><div class="casAuthors">Stimson, Lindsay; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-183</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histone and non-histone proteins by acetylation are known to play a key role in tumorigenesis.  Pharmacol. manipulation of acetylation has been possible with the identification of small mol. inhibitors of histone deacetylases (HDAC), the enzymes responsible for deacetylating lysine residues.  An explosion of drug discovery efforts in recent years has led to the development of an extensive group of HDAC inhibitors, many of which have been shown pre-clin. to have potent anti-tumor activity.  Clin. trials using these agents are now underway, with Vorinostat (suberoylanilide hydroxamic acid) having been approved by the FDA for treating cutaneous T-cell lymphoma (CTCL) in patients with progressive, persistent or recurrent disease.  This review discusses how biomarkers are being identified and used to expand our knowledge of the mechanisms by which HDAC inhibitors exhibit their anti-cancer effects.  In the longer term, biomarkers will provide a means towards achieving patient stratification in tumor types that will respond favorably to HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtCGFKlhhe8bVg90H21EOLACvtfcHk0lh8DbT7Snk3WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsF2jt7o%253D&md5=a925ab1ae931b77b309225ab086923de</span></div><a href="/servlet/linkout?suffix=cit18e&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2009.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2009.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DStimson%26aufirst%3DL.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DBiomarkers%2520for%2520predicting%2520clinical%2520responses%2520to%2520HDAC%2520inhibitors%26jtitle%3DCancer%2520Lett.%26date%3D2009%26volume%3D280%26spage%3D177%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Wong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ning, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span> </span><span class="NLM_article-title">Application of p21 and klf2 reporter gene assays to identify selective histone deacetylase inhibitors for cancer therapy</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=110-116&author=J.+C.+Wongauthor=L.+Guoauthor=Z.+H.+Pengauthor=W.+X.+Zhangauthor=N.+Zhangauthor=W.+Laiauthor=Z.+S.+Zhangauthor=S.+Songauthor=D.+S.+Panauthor=Z.+Q.+Ningauthor=X.+P.+Luauthor=X.+W.+Zhangauthor=C.+Zhangauthor=J.+Liauthor=C.+M.+Xieauthor=Y.+Heauthor=L.+Chen&title=Application+of+p21+and+klf2+reporter+gene+assays+to+identify+selective+histone+deacetylase+inhibitors+for+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DZ.%2BH.%26aulast%3DZhang%26aufirst%3DW.%2BX.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DLai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DZ.%2BS.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DD.%2BS.%26aulast%3DNing%26aufirst%3DZ.%2BQ.%26aulast%3DLu%26aufirst%3DX.%2BP.%26aulast%3DZhang%26aufirst%3DX.%2BW.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DC.%2BM.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DApplication%2520of%2520p21%2520and%2520klf2%2520reporter%2520gene%2520assays%2520to%2520identify%2520selective%2520histone%2520deacetylase%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D110%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Zhao, H.</span><span> </span><span class="NLM_article-title">Scaffold selection and scaffold hopping in lead generation: a medicinal chemistry perspective</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2Fj.drudis.2006.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=17275735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtleisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=149-155&author=H.+Zhao&title=Scaffold+selection+and+scaffold+hopping+in+lead+generation%3A+a+medicinal+chemistry+perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold selection and scaffold hopping in lead generation: a medicinal chemistry perspective</span></div><div class="casAuthors">Zhao, Hongyu</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">149-155</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Hit selection and lead generation are crucial for the success of the resource-demanding lead-optimization phase in drug discovery, and represent a major research area of medicinal chem. today.  Ligand-binding efficiency, ligand complexity, ligand-target profile complementarity and chem. tractability are important parameters in hit selection.  As synthesis and assay throughput improve, a large no. of analogs based on the same scaffold can be rapidly synthesized and tested.  Consequently, more chem. resources could be devoted to scaffold modifications to expand the candidate pool in lead generation.  Most recently discovered druggable targets are promiscuous toward lipophilic ligands, and the hydrophobic portions of hit compds. should be preferentially modified in analog and scaffold design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_G4HUVC1fQrVg90H21EOLACvtfcHk0lh8DbT7Snk3WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtleisbo%253D&md5=a9ffb13a77e3bb18d5c432f3286d4608</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DH.%26atitle%3DScaffold%2520selection%2520and%2520scaffold%2520hopping%2520in%2520lead%2520generation%253A%2520a%2520medicinal%2520chemistry%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D149%26epage%3D155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Joucla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mortier, J.</span><span> </span><span class="NLM_article-title">2,5-Unsubstituted pyrrolidines from formaldehyde and amino acids through in situ azomethine-ylide 1,3-dipolar cycloaddition to alkenes</span> <span class="citation_source-journal">J. Chem. Soc., Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1566</span><span class="NLM_x">–</span> <span class="NLM_lpage">1567</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1985&pages=1566-1567&author=M.+Jouclaauthor=J.+Mortier&title=2%2C5-Unsubstituted+pyrrolidines+from+formaldehyde+and+amino+acids+through+in+situ+azomethine-ylide+1%2C3-dipolar+cycloaddition+to+alkenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJoucla%26aufirst%3DM.%26aulast%3DMortier%26aufirst%3DJ.%26atitle%3D2%252C5-Unsubstituted%2520pyrrolidines%2520from%2520formaldehyde%2520and%2520amino%2520acids%2520through%2520in%2520situ%2520azomethine-ylide%25201%252C3-dipolar%2520cycloaddition%2520to%2520alkenes%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Chem.%2520Commun.%26date%3D1985%26volume%3D22%26spage%3D1566%26epage%3D1567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Evans, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mito, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, D.</span><span> </span><span class="NLM_article-title">Scope and mechanism of enantioselective Michael additions of 1,3-dicarbonyl compounds to nitroalkenes catalyzed by nickel(II)-diamine complexes</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">11583</span><span class="NLM_x">–</span> <span class="NLM_lpage">11592</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0735913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=11583-11592&author=D.+A.+Evansauthor=S.+Mitoauthor=D.+Seidel&title=Scope+and+mechanism+of+enantioselective+Michael+additions+of+1%2C3-dicarbonyl+compounds+to+nitroalkenes+catalyzed+by+nickel%28II%29-diamine+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja0735913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0735913%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DD.%2BA.%26aulast%3DMito%26aufirst%3DS.%26aulast%3DSeidel%26aufirst%3DD.%26atitle%3DScope%2520and%2520mechanism%2520of%2520enantioselective%2520Michael%2520additions%2520of%25201%252C3-dicarbonyl%2520compounds%2520to%2520nitroalkenes%2520catalyzed%2520by%2520nickel%2528II%2529-diamine%2520complexes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D11583%26epage%3D11592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budhu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hale, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajdu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keohane, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbach, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milligan, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shei, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chrebet, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergstrom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Card, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandala, S. M.</span><span> </span><span class="NLM_article-title">2-Aryl(pyrrolidin-4-yl)acetic acids are potent agonists of sphingosine-1-phosphate (S1P) receptors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">3564</span><span class="NLM_x">–</span> <span class="NLM_lpage">3568</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=3564-3568&author=L.+Yanauthor=R.+Budhuauthor=P.+Huoauthor=C.+L.+Lynchauthor=J.+J.+Haleauthor=S.+G.+Millsauthor=R.+Hajduauthor=C.+A.+Keohaneauthor=M.+J.+Rosenbachauthor=J.+A.+Milliganauthor=G.+J.+Sheiauthor=G.+Chrebetauthor=J.+Bergstromauthor=D.+Cardauthor=S.+M.+Mandala&title=2-Aryl%28pyrrolidin-4-yl%29acetic+acids+are+potent+agonists+of+sphingosine-1-phosphate+%28S1P%29+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DL.%26aulast%3DBudhu%26aufirst%3DR.%26aulast%3DHuo%26aufirst%3DP.%26aulast%3DLynch%26aufirst%3DC.%2BL.%26aulast%3DHale%26aufirst%3DJ.%2BJ.%26aulast%3DMills%26aufirst%3DS.%2BG.%26aulast%3DHajdu%26aufirst%3DR.%26aulast%3DKeohane%26aufirst%3DC.%2BA.%26aulast%3DRosenbach%26aufirst%3DM.%2BJ.%26aulast%3DMilligan%26aufirst%3DJ.%2BA.%26aulast%3DShei%26aufirst%3DG.%2BJ.%26aulast%3DChrebet%26aufirst%3DG.%26aulast%3DBergstrom%26aufirst%3DJ.%26aulast%3DCard%26aufirst%3DD.%26aulast%3DMandala%26aufirst%3DS.%2BM.%26atitle%3D2-Aryl%2528pyrrolidin-4-yl%2529acetic%2520acids%2520are%2520potent%2520agonists%2520of%2520sphingosine-1-phosphate%2520%2528S1P%2529%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D3564%26epage%3D3568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span>For more information about HuPrime tumor-derived xenograft models versus traditional cell line-derived xenograft models, see the website of Crown Biosciences (Santa Clara, CA) at <a href="http://www.crownbio.com" class="extLink">www.crownbio.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+more+information+about+HuPrime+tumor-derived+xenograft+models+versus+traditional+cell+line-derived+xenograft+models%2C+see+the+website+of+Crown+Biosciences+%28Santa+Clara%2C+CA%29+at+www.crownbio.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span>For more information, see <a href="http://www.ingenuity.com" class="extLink">www.ingenuity.com</a> (Ingenuity Systems, Redwood City, CA).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+more+information%2C+see+www.ingenuity.com+%28Ingenuity+Systems%2C+Redwood+City%2C+CA%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span>See PubMeth website: <a href="http://methycancer.psych.ac.cn/MethyCancer.do" class="extLink">http://methycancer.psych.ac.cn/MethyCancer.do</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=See+PubMeth+website%3A+http%3A%2F%2Fmethycancer.psych.ac.cn%2FMethyCancer.do."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Cameron, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachman, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myöhänen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herman, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1038%2F5047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=9916800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1999&pages=103-107&author=E.+E.+Cameronauthor=K.+E.+Bachmanauthor=S.+My%C3%B6h%C3%A4nenauthor=J.+G.+Hermanauthor=S.+B.+Baylin&title=Synergy+of+demethylation+and+histone+deacetylase+inhibition+in+the+re-expression+of+genes+silenced+in+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer</span></div><div class="casAuthors">Cameron, Elizabeth E.; Bachman, Kurtis E.; Myohanen, Sanna; Herman, James G.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Densely methylated DNA assocs. with transcriptionally repressive chromatin characterized by the presence of underacetylated histones.  Recently, these 2 epigenetic processes were dynamically linked.  The methyl-CpG-binding protein MeCP2 appears to reside in a complex with histone deacetylase activity.  MeCP2 can mediate formation of transcriptionally repressive chromatin on methylated promoter templates in vitro, and this process can be reversed by trichostatin A (TSA), a specific inhibitor of histone deacetylase.  Little is known about the relative roles of methylation and histone deacetylase activity in the stable inhibition of transcription on densely methylated endogenous promoters, such as those for silenced alleles of imprinted genes, genes on the female inactive X chromosome, and tumor-suppressor genes inactivated in cancer cells.  The authors show that the hypermethylated genes MLH1, TIMP3, CDKN2B (INK4B, p15), and CDKN2A (INK4, p16) cannot be transcriptionally reactivated with TSA alone in tumor cells in which the authors have shown that TSA alone can upregulate the expression of non-methylated genes.  Following minimal demethylation and slight gene reactivation in the presence of low dose 5-aza-2' deoxycytidine (5Aza-dC), however, TSA treatment results in robust re-expression of each gene.  TSA does not contribute to demethylation of the genes, and none of the treatments alter the chromatin structure assocd. with the hypermethylated promoters.  Thus, although DNA methylation and histone deacetylation appear to act as synergistic layers for the silencing of genes in cancer, dense CpG island methylation is dominant for the stable maintenance of a silent state at these loci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZCmJwXaY4o7Vg90H21EOLACvtfcHk0liSqFiPMv4GvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D&md5=5f0666dac86ee091d5e06ab0b9403502</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2F5047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F5047%26sid%3Dliteratum%253Aachs%26aulast%3DCameron%26aufirst%3DE.%2BE.%26aulast%3DBachman%26aufirst%3DK.%2BE.%26aulast%3DMy%25C3%25B6h%25C3%25A4nen%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DJ.%2BG.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSynergy%2520of%2520demethylation%2520and%2520histone%2520deacetylase%2520inhibition%2520in%2520the%2520re-expression%2520of%2520genes%2520silenced%2520in%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D1999%26volume%3D21%26spage%3D103%26epage%3D107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Morris, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gentle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulrahman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maina, E. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiesener, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kishida, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teh, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Latif, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maher, E. R.</span><span> </span><span class="NLM_article-title">Tumor suppressor activity and epigenetic inactivation of hepatocyte growth factor activator inhibitor type 2/SPINT2 in papillary and clear cell renal cell carcinoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">4598</span><span class="NLM_x">–</span> <span class="NLM_lpage">4606</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=4598-4606&author=M.+R.+Morrisauthor=D.+Gentleauthor=M.+Abdulrahmanauthor=E.+N.+Mainaauthor=K.+Guptaauthor=R.+E.+Banksauthor=M.+S.+Wiesenerauthor=T.+Kishidaauthor=M.+Yaoauthor=B.+Tehauthor=F.+Latifauthor=E.+R.+Maher&title=Tumor+suppressor+activity+and+epigenetic+inactivation+of+hepatocyte+growth+factor+activator+inhibitor+type+2%2FSPINT2+in+papillary+and+clear+cell+renal+cell+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DM.%2BR.%26aulast%3DGentle%26aufirst%3DD.%26aulast%3DAbdulrahman%26aufirst%3DM.%26aulast%3DMaina%26aufirst%3DE.%2BN.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DBanks%26aufirst%3DR.%2BE.%26aulast%3DWiesener%26aufirst%3DM.%2BS.%26aulast%3DKishida%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DTeh%26aufirst%3DB.%26aulast%3DLatif%26aufirst%3DF.%26aulast%3DMaher%26aufirst%3DE.%2BR.%26atitle%3DTumor%2520suppressor%2520activity%2520and%2520epigenetic%2520inactivation%2520of%2520hepatocyte%2520growth%2520factor%2520activator%2520inhibitor%2520type%25202%252FSPINT2%2520in%2520papillary%2520and%2520clear%2520cell%2520renal%2520cell%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D4598%26epage%3D4606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span>MacroModel, formerly known as BatchMin, is a general-purpose molecular mechanics program available from Schrödinger, L.L.C., New York. MacroModel was developed in the laboratories of Prof. Clark Still (Columbia University).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MacroModel%2C+formerly+known+as+BatchMin%2C+is+a+general-purpose+molecular+mechanics+program+available+from+Schr%C3%B6dinger%2C+L.L.C.%2C+New+York.+MacroModel+was+developed+in+the+laboratories+of+Prof.+Clark+Still+%28Columbia+University%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D.</span><span> </span><span class="NLM_article-title">Extra precision Glide: Docking and scoring incorporating a model of hydrophobic enclosure for protein–ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+Mainz&title=Extra+precision+Glide%3A+Docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein%E2%80%93ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%26atitle%3DExtra%2520precision%2520Glide%253A%2520Docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein%25E2%2580%2593ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Testino, S. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patonay, G.</span><span> </span><span class="NLM_article-title">High-throughput inhibition screening of major human cytochrome P450 enzymes using an in vitro cocktail and liquid chromatography-tandem mass spectrometry</span> <span class="citation_source-journal">J. Pharm. Biomed. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1459</span><span class="NLM_x">–</span> <span class="NLM_lpage">1467</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2FS0731-7085%2802%2900480-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=12467917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31a&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFKhsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2003&pages=1459-1467&author=S.+A.+J.+Testinoauthor=G.+Patonay&title=High-throughput+inhibition+screening+of+major+human+cytochrome+P450+enzymes+using+an+in+vitro+cocktail+and+liquid+chromatography-tandem+mass+spectrometry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31aR"><div class="casContent"><span class="casTitleNuber">31a</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput inhibition screening of major human cytochrome P450 enzymes using an in vitro cocktail and liquid chromatography-tandem mass spectrometry</span></div><div class="casAuthors">Testino, Samuel A.; Patonay, Gabor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1459-1467</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A method has been developed for the high-throughput inhibition screening of the major human cytochrome P 450 (CYP) enzymes (CYP1A2, CYP2C9, CYP2C19, CYP2D6, and CYP3A4) using an in vitro substrate cocktail and liq. chromatog.-tandem mass spectrometry (LC-MS-MS).  A cocktail consisting of the selective substrates phenacetin (CYP1A2), tolbutamide (CYP2C9), omeprazole (CYP2C19), bufuralol (CYP2D6), and midazolam (CYP3A4) was incubated with human liver microsomes.  The metabolic reactions were terminated with methanol contg. dextrorphan as an internal std.  Following centrifugation, the supernatant was analyzed by LC-MS-MS employing a fast gradient.  The concns. of the substrate metabolites-paracetamol, 4-hydroxytolbutamide, 5-hydroxyomeprazole, 1'-hydroxybufuralol, and 1'-hydroxymidazolam-in each sample were detd. by LC-MS-MS in a single assay.  The method was validated by incubating known CYP inhibitors (furafylline, CYP1A2; sulfaphenazole, CYP2C9; s-mephenytoin, CYP2C19; quinidine, CYP2D6; and troleandomycin, CYP3A4) with the individual substrates they were known to inhibit and with the substrate cocktail.  IC50s (μM) detd. using the substrate cocktail were in good agreement with those obtained with individual substrates (furafylline, 2.9 vs. 2.0; sulfaphenazole, 0.75 vs. 0.72; s-mephenytoin, 170 vs. 180; quinidine, 0.17 vs. 0.24; troleandomycin, 2.6 vs. 3.2) and with previously reported values in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYVLv3U7pcprVg90H21EOLACvtfcHk0lhBo4q_n9DPBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFKhsLw%253D&md5=68a59dc476e91abc7130c232cd816cb5</span></div><a href="/servlet/linkout?suffix=cit31a&amp;dbid=16384&amp;doi=10.1016%2FS0731-7085%2802%2900480-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0731-7085%252802%252900480-6%26sid%3Dliteratum%253Aachs%26aulast%3DTestino%26aufirst%3DS.%2BA.%2BJ.%26aulast%3DPatonay%26aufirst%3DG.%26atitle%3DHigh-throughput%2520inhibition%2520screening%2520of%2520major%2520human%2520cytochrome%2520P450%2520enzymes%2520using%2520an%2520in%2520vitro%2520cocktail%2520and%2520liquid%2520chromatography-tandem%2520mass%2520spectrometry%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2003%26volume%3D30%26spage%3D1459%26epage%3D1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit31b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Walsky, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Validated assays for human cytochrome P450 activities</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1124%2Fdmd.32.6.647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=15155557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31b&amp;dbid=32&amp;doi=10.1021%2Fjm3011838&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktlOgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=647-660&author=R.+L.+Walskyauthor=R.+S.+Obach&title=Validated+assays+for+human+cytochrome+P450+activities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31bR"><div class="casContent"><span class="casTitleNuber">31b</span><div class="casTitle"><span class="NLM_cas:atitle">Validated assays for human cytochrome P450 activities</span></div><div class="casAuthors">Walsky, Robert L.; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">647-660</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The measurement of the effect of new chem. entities on human cytochrome P 450 marker activities using in vitro experimentation represents an important exptl. approach in drug development.  In vitro drug interaction data can be used in guiding the design of clin. drug interaction studies, or, when no effect is obsd. in vitro, the data can be used in place of an in vivo study to claim that no interaction will occur in vivo.  To make such a claim, it must be assured that the in vitro expts. are performed with abs. confidence in the methods used and data obtained.  To meet this need, 12 semiautomated assays for human P 450 marker substrate activities have been developed and validated using approaches described in the GLP (good lab. practices) as per the code of U.S. Federal Regulations.  The assays that were validated are: phenacetin O-deethylase (CYP1A2), coumarin 7-hydroxylase (CYP2A6), bupropion hydroxylase (CYP2B6), amodiaquine N-deethylase (CYP2C8), diclofenac 4'-hydroxylase and tolbutamide methylhydroxylase (CYP2C9), (S)-mephenytoin 4'-hydroxylase (CYP2C19), dextromethorphan O-demethylase (CYP2D6), chlorzoxazone 6-hydroxylase (CYP2E1), felodipine dehydrogenase, testosterone 6β-hydroxylase, and midazolam 1'-hydroxylase (CYP3A4 and CYP3A5).  High-pressure liq. chromatog.-tandem mass spectrometry, using stable isotope-labeled internal stds., has been applied as the anal. method.  This anal. approach, through its high sensitivity and selectivity, has permitted the use of very low incubation concns. of microsomal protein (0.01-0.2 mg/mL).  Anal. assay accuracy and precision values were excellent.  Enzyme kinetic and inhibition parameters obtained using these methods demonstrated high precision and were within the range of values previously reported in the scientific literature.  These methods should prove useful in the routine assessments of the potential for new drug candidates to elicit pharmacokinetic drug interactions via inhibition of cytochrome P 450 activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwF04TGeyYbbVg90H21EOLACvtfcHk0lhBo4q_n9DPBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktlOgt74%253D&md5=35532b907c15f1053aa8233dab2ec581</span></div><a href="/servlet/linkout?suffix=cit31b&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.6.647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.6.647%26sid%3Dliteratum%253Aachs%26aulast%3DWalsky%26aufirst%3DR.%2BL.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DValidated%2520assays%2520for%2520human%2520cytochrome%2520P450%2520activities%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26spage%3D647%26epage%3D660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit31c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Weaver, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beattie, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riley, R. J.</span><span> </span><span class="NLM_article-title">Cytochrome P450 inhibition using recombinant proteins and mass spectrometry/multiple reaction monitoring technology in a cassette incubation</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">955</span><span class="NLM_x">–</span> <span class="NLM_lpage">966</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2003&pages=955-966&author=R.+Weaverauthor=K.+S.+Grahamauthor=I.+G.+Beattieauthor=R.+J.+Riley&title=Cytochrome+P450+inhibition+using+recombinant+proteins+and+mass+spectrometry%2Fmultiple+reaction+monitoring+technology+in+a+cassette+incubation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWeaver%26aufirst%3DR.%26aulast%3DGraham%26aufirst%3DK.%2BS.%26aulast%3DBeattie%26aufirst%3DI.%2BG.%26aulast%3DRiley%26aufirst%3DR.%2BJ.%26atitle%3DCytochrome%2520P450%2520inhibition%2520using%2520recombinant%2520proteins%2520and%2520mass%2520spectrometry%252Fmultiple%2520reaction%2520monitoring%2520technology%2520in%2520a%2520cassette%2520incubation%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2003%26volume%3D31%26spage%3D955%26epage%3D966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Liang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, H.</span><span> </span><span class="NLM_article-title">Automated electrophysiology in the preclinical evaluation of drugs for potential QT prolongation</span> <span class="citation_source-journal">J. Pharmacol. Toxicol. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm3011838&amp;key=10.1016%2Fj.vascn.2005.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm3011838&amp;key=15936217" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2005&pages=123-135&author=G.+Liangauthor=H.+Guthrie&title=Automated+electrophysiology+in+the+preclinical+evaluation+of+drugs+for+potential+QT+prolongation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.vascn.2005.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.vascn.2005.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DG.%26aulast%3DGuthrie%26aufirst%3DH.%26atitle%3DAutomated%2520electrophysiology%2520in%2520the%2520preclinical%2520evaluation%2520of%2520drugs%2520for%2520potential%2520QT%2520prolongation%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2005%26volume%3D52%26spage%3D123%26epage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i95"><a href="/doi/suppl/10.1021/jm3011838">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_14310"></div></div></div></div></div><hr /></hr><p class="last">Additional information regarding compound characterization (LCMS and <sup>1</sup>H NMR), supplemental SAR data, and experimental details for the CYP inhibition assay, metabolite identification, cell lines and biological reagents, patient-derived xenograft model study design, microarray profiling study of compound <b>5</b>, and immunohistochemistry scoring images. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm3011838/suppl_file/jm3011838_si_001.pdf">jm3011838_si_001.pdf (716.9 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm3011838&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm3011838%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-20%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm3011838" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799350bccc53d12","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
