
<html lang="en"     class="pb-page"  data-request-id="cf660727-0c12-4684-8ffc-394f1842676e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01967;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-11"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Conjugation of Short Peptides to Dibenzodiazepinone-Type Muscarinic Acetylcholine Receptor Ligands Determines M2R Selectivity" /></meta><meta name="dc.Creator" content="Andrea  Pegoli" /></meta><meta name="dc.Creator" content="David  Wifling" /></meta><meta name="dc.Creator" content="Corinna G.  Gruber" /></meta><meta name="dc.Creator" content="Xueke  She" /></meta><meta name="dc.Creator" content="Harald  Hübner" /></meta><meta name="dc.Creator" content="Günther  Bernhardt" /></meta><meta name="dc.Creator" content="Peter  Gmeiner" /></meta><meta name="dc.Creator" content="Max  Keller" /></meta><meta name="dc.Description" content="Muscarinic acetylcholine receptors (MRs), comprising five subtypes (M1R–M5R) in humans, exhibit a high degree of structural similarity. Therefore, subtype-selective MR agonists and antagonists are ..." /></meta><meta name="Description" content="Muscarinic acetylcholine receptors (MRs), comprising five subtypes (M1R–M5R) in humans, exhibit a high degree of structural similarity. Therefore, subtype-selective MR agonists and antagonists are ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 10, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01967" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01967" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01967" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01967" /></link>
        
    
    

<title>Conjugation of Short Peptides to Dibenzodiazepinone-Type Muscarinic Acetylcholine Receptor Ligands Determines M2R Selectivity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01967" /></meta><meta property="og:title" content="Conjugation of Short Peptides to Dibenzodiazepinone-Type Muscarinic Acetylcholine Receptor Ligands Determines M2R Selectivity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0008.jpeg" /></meta><meta property="og:description" content="Muscarinic acetylcholine receptors (MRs), comprising five subtypes (M1R–M5R) in humans, exhibit a high degree of structural similarity. Therefore, subtype-selective MR agonists and antagonists are lacking. We present an approach to highly M2R-selective MR antagonists based on the conjugation of di- or tripeptides to M2R-preferring dibenzodiazepinone-type MR antagonists. M2R selectivity was dependent on the peptide sequence and on the type of linker. The introduction of basic amino acids resulted in improved M2R selectivity (e.g., UR-AP148 (48): pKi (hM2R) of 8.97, ratio of Ki M1R/M2R/M3R/M4R/M5R of 49:1:6500:60:400) compared to reported pyridobenzo- and dibenzodiazepinone-type MR ligands. A supposed dualsteric binding mode of the DIBA–peptide conjugates, such as 48, at MRs was supported by molecular dynamics simulations." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01967"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01967">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01967&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01967&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01967&amp;href=/doi/10.1021/acs.jmedchem.8b01967" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 5358-5369</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01709" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b02027" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Conjugation of Short Peptides to Dibenzodiazepinone-Type Muscarinic Acetylcholine Receptor Ligands Determines M<sub>2</sub>R Selectivity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Andrea Pegoli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Pegoli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Pegoli">Andrea Pegoli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Wifling</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Wifling</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Wifling">David Wifling</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Corinna G. Gruber</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Corinna G. Gruber</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Corinna+G.++Gruber">Corinna G. Gruber</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xueke She</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xueke She</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xueke++She">Xueke She</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Harald Hübner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Harald Hübner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Pharmacy, Medicinal Chemistry, Friedrich Alexander University, Nikolaus-Fiebiger-Straße 10, D-91058 Erlangen, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Harald++H%C3%BCbner">Harald Hübner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Günther Bernhardt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Günther Bernhardt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=G%C3%BCnther++Bernhardt">Günther Bernhardt</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6491-9874" title="Orcid link">http://orcid.org/0000-0001-6491-9874</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Gmeiner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Gmeiner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Pharmacy, Medicinal Chemistry, Friedrich Alexander University, Nikolaus-Fiebiger-Straße 10, D-91058 Erlangen, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Gmeiner">Peter Gmeiner</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4127-197X" title="Orcid link">http://orcid.org/0000-0002-4127-197X</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Max Keller</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Max Keller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#6a070b1244010f06060f182a1f18440e0f"><span class="__cf_email__" data-cfemail="92fff3eabcf9f7fefef7e0d2e7e0bcf6f7">[email protected]</span></a>. Phone: (+49) 941-9433329. Fax: (+49) 941-9434820.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Max++Keller">Max Keller</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8095-8627" title="Orcid link">http://orcid.org/0000-0002-8095-8627</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01967&amp;href=/doi/10.1021%2Facs.jmedchem.8b01967" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 5358–5369</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 10, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 December 2018</li><li><span class="item_label"><b>Published</b> online</span>10 May 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 June 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01967" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01967</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5358%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAndrea%2BPegoli%252C%2BDavid%2BWifling%252C%2BCorinna%2BG.%2BGruber%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D11%26contentID%3Dacs.jmedchem.8b01967%26title%3DConjugation%2Bof%2BShort%2BPeptides%2Bto%2BDibenzodiazepinone-Type%2BMuscarinic%2BAcetylcholine%2BReceptor%2BLigands%2BDetermines%2BM2R%2BSelectivity%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5369%26publicationDate%3DJune%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01967"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">859</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">7</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01967" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Conjugation of Short Peptides to Dibenzodiazepinone-Type Muscarinic Acetylcholine Receptor Ligands Determines M2R Selectivity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Pegoli&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Wifling&quot;},{&quot;first_name&quot;:&quot;Corinna&quot;,&quot;last_name&quot;:&quot;G. Gruber&quot;},{&quot;first_name&quot;:&quot;Xueke&quot;,&quot;last_name&quot;:&quot;She&quot;},{&quot;first_name&quot;:&quot;Harald&quot;,&quot;last_name&quot;:&quot;Hübner&quot;},{&quot;first_name&quot;:&quot;Günther&quot;,&quot;last_name&quot;:&quot;Bernhardt&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Gmeiner&quot;},{&quot;first_name&quot;:&quot;Max&quot;,&quot;last_name&quot;:&quot;Keller&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;5358-5369&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01967&quot;},&quot;abstract&quot;:&quot;Muscarinic acetylcholine receptors (MRs), comprising five subtypes (M1R–M5R) in humans, exhibit a high degree of structural similarity. Therefore, subtype-selective MR agonists and antagonists are lacking. We present an approach to highly M2R-selective MR antagonists based on the conjugation of di- or tripeptides to M2R-preferring dibenzodiazepinone-type MR antagonists. M2R selectivity was dependent on the peptide sequence and on the type of linker. The introduction of basic amino acids resulted in improved M2R selectivity (e.g., UR-AP148 (48): pKi (hM2R) of 8.97, ratio of Ki M1R/M2R/M3R/M4R/M5R of 49:1:6500:60:400) compared to reported pyridobenzo- and dibenzodiazepinone-type MR ligands. A supposed dualsteric binding mode of the DIBA–peptide conjugates, such as 48, at MRs was supported by molecular dynamics simulations.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01967&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01967" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01967&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01967" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01967&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01967" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01967&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01967&amp;href=/doi/10.1021/acs.jmedchem.8b01967" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01967" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01967" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01967%26sid%3Dliteratum%253Aachs%26pmid%3D31074983%26genre%3Darticle%26aulast%3DPegoli%26date%3D2019%26atitle%3DConjugation%2Bof%2BShort%2BPeptides%2Bto%2BDibenzodiazepinone-Type%2BMuscarinic%2BAcetylcholine%2BReceptor%2BLigands%2BDetermines%2BM2R%2BSelectivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D11%26spage%3D5358%26epage%3D5369%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/jmcmar.2019.62.issue-11/20190613/jmcmar.2019.62.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Muscarinic acetylcholine receptors (MRs), comprising five subtypes (M<sub>1</sub>R–M<sub>5</sub>R) in humans, exhibit a high degree of structural similarity. Therefore, subtype-selective MR agonists and antagonists are lacking. We present an approach to highly M<sub>2</sub>R-selective MR antagonists based on the conjugation of di- or tripeptides to M<sub>2</sub>R-preferring dibenzodiazepinone-type MR antagonists. M<sub>2</sub>R selectivity was dependent on the peptide sequence and on the type of linker. The introduction of basic amino acids resulted in improved M<sub>2</sub>R selectivity (e.g., UR-AP148 (<b>48</b>): p<i>K</i><sub>i</sub> (hM<sub>2</sub>R) of 8.97, ratio of <i>K</i><sub>i</sub> M<sub>1</sub>R/M<sub>2</sub>R/M<sub>3</sub>R/M<sub>4</sub>R/M<sub>5</sub>R of 49:1:6500:60:400) compared to reported pyridobenzo- and dibenzodiazepinone-type MR ligands. A supposed dualsteric binding mode of the DIBA–peptide conjugates, such as <b>48</b>, at MRs was supported by molecular dynamics simulations.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43227" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43227" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In humans, muscarinic acetylcholine receptors (MRs) comprise five receptor subtypes (M<sub>1</sub>–M<sub>5</sub>), all belonging to the class A of G-protein-coupled receptors. MRs are expressed in the CNS and in the periphery, being involved in the regulation of cognition and smooth muscle contraction, respectively.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Due to the high structural similarity of MRs, most pronounced in the region of the orthosteric binding site,<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> subtype-selective MR agonists and antagonists are lacking. Such compounds are considered to improve pharmacotherapy. For example, in the CNS, antagonism at presynaptic M<sub>2</sub>Rs has been suggested as an approach to enhance cholinergic function in Alzheimer’s disease patients.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7">(4−7)</a> However, for that purpose, highly selective M<sub>2</sub>R antagonists are needed, because concomitant blocking of postsynaptic M<sub>1</sub>Rs must be strictly prevented.</div><div class="NLM_p">To date, to the best of our knowledge, only one compound class (diaryl sulfones linked to a piperidinyl moiety) has afforded M<sub>2</sub>R ligands with pronounced selectivity (>100-fold in binding affinity) over the other MR subtypes (compound <b>1a</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Strikingly, the amino group of the anthranoyl moiety in <b>1a</b> seems to be crucial for high M<sub>2</sub>R selectivity, which becomes obvious from the considerably lower M<sub>2</sub>R selectivity of congener <b>1b</b>,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> bearing a methyl instead of the amino group in <b>1a</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Another well studied class of M<sub>2</sub> subtype-preferring MR antagonists are pyridobenzodiazepinone-type (e.g., <b>2</b>–<b>4</b>,<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) and dibenzodiazepinone-type (e.g., <b>5</b>–<b>10</b>,<a onclick="showRef(event, 'ref10 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref10 ref13 ref14 ref15 ref16">(10,13−16)</a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) tricyclic MR ligands. The structures and MR binding data of selected tricyclic M<sub>2</sub>R antagonists with high M<sub>2</sub>R affinity (p<i>K</i><sub>i</sub> > 7.5) (<b>2</b>–<b>10</b>), including the most M<sub>2</sub>R-selective dibenzodiazepinone-type MR ligand reported in literature to date (<b>8</b>),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures and MR binding data of two described diarylsulfone-type M<sub>2</sub>R-selective MR ligands <b>(1a</b>, <b>1b</b>) and of a selection of reported M<sub>2</sub>R-preferring tricyclic MR antagonists (compounds <b>2</b>–<b>10</b>). In the table, references are as follows: (a) Clader et al. (reported <i>K</i><sub>i</sub> values (M<sub>2</sub>R) were converted to p<i>K</i><sub>i</sub> values; IC<sub>50</sub>, pIC<sub>50</sub>, <i>K</i><sub>i</sub>, or p<i>K</i><sub>i</sub> values for the MR subtypes M<sub>1</sub>, M<sub>3</sub>, M<sub>4</sub>, and M<sub>5</sub> were not given; instead, only the ratios <i>K</i><sub>i</sub>(M<sub>1,3,4,5</sub>R)/<i>K</i><sub>i</sub>(M<sub>2</sub>R) were provided);<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> (b) Dörje et al.;<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (c) Keller et al. (data reported as pIC<sub>50</sub>; data were reanalyzed to obtain p<i>K</i><sub>i</sub> values);<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (d) Doods et al.;<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (e) Doods et al.;<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (f) Maggio et al. (reported <i>K</i><sub>i</sub> values were converted to p<i>K</i><sub>i</sub> values);<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (g) Gitler et al. (reported <i>K</i><sub>i</sub> values were converted to p<i>K</i><sub>i</sub> values);<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (h) She et al.;<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (i) Pegoli et al.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Except for <b>5</b>, data refer to human receptors. In the case of <b>5</b>, M<sub>2</sub> and M<sub>4</sub> were rat receptors, whereas M<sub>1</sub> and M<sub>3</sub> receptors were not specified.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Recently, investigations using tritiated analogs of the dibenzodiazepinone-type MR antagonists <b>7</b>, <b>9</b>, and <b>10</b>, showing low to moderate M<sub>2</sub>R selectivity (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), revealed that these compounds bind to the M<sub>2</sub>R in a dualsteric manner with the tricyclic moiety residing in the orthosteric binding pocket and the “side chains” reaching the vestibule (common allosteric site) of the receptor.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Moreover, these studies demonstrated that the attachment of various bulky moieties to the dibenzodiazepinone core structure “A” (cf. box in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was well tolerated with respect to M<sub>2</sub>R binding (e.g., <b>8</b>–<b>10</b>).<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a></div><div class="NLM_p">As the dualsteric ligand approach is considered useful to develop MR ligands with improved subtype selectivity,<a onclick="showRef(event, 'ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20">(17−20)</a> we wanted to explore how the attachment of putatively allosterically interacting peptide moieties to the dibenzodiazepinone core structure influences M<sub>2</sub>R affinity and selectivity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). For this purpose, various di- and tripeptides were conjugated to the dibenzodiazepinone pharmacophore via two linkers differing in length and polarity, resulting in “DIBA–peptide conjugates”, which were characterized with respect to binding at MRs M<sub>1</sub>–M<sub>5</sub>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Rationale of the present study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43367" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43367" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The selection of amino acids, used to prepare the di- and tripeptide moieties of the DIBA–peptide conjugates, was driven by the fact that acidic amino acids, present at the extracellular surface of MRs, vary in number and position (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S8, Supporting Information</a>).<a onclick="showRef(event, 'ref3 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref3 ref21 ref22">(3,21,22)</a> Therefore, Lys and Arg were chosen as potential interaction partners of these acidic amino acids of the receptor. Gln was used as a nonbasic, polar amino acid, capable of forming hydrogen bonds, as well as the aromatic amino acids Tyr and Trp to potentially address Tyr83<sup>2.64</sup>, Tyr177<sup>ECL2.51</sup>, and Trp422<sup>7.35</sup> located within the vestibule of the M<sub>2</sub>R (cf. <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S8A, Supporting Information</a>). Additionally, Ala and Leu were incorporated as nonpolar amino acids. Neither the selection of individual amino acids nor their combination was guided by computational chemistry (docking, simulations) for the following reasons: docking studies alone, requiring considerably less computing time compared to simulations, have limited predictive value; docking studies with subsequent time-consuming simulations were at that stage of the project not considered reasonable to be performed for numerous potential DIBA–peptide conjugates. We rather selected amino acid sequences covering the following combinations of physicochemically different types of amino acid residues: aromatic/aromatic (Trp/Tyr), polar/hydrophobic (Gln/Leu), basic/aromatic (Lys/Tyr), and basic/basic (Lys/Arg or Arg/Arg) (cf. <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In the tripeptides, Ala was chosen as a “spacer” amino acid to increase the distance between the aforementioned amino acid pairs.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0006.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of DIBA–Peptide Conjugates <b>28</b>–<b>53</b> and Dibenzodiazepinone Derivatives <b>55</b>, <b>56</b>, and <b>58</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) Fmoc-strategy SPPS, amino acid coupling: Fmoc-aa, HBTU, HOBt, DIPEA, DMF/NMP (8:2 v/v), rt or 35 °C, 2 × 60 min (“double coupling”). Fmoc deprotection: 20% piperidine in DMF/NMP (8:2 v/v), rt, 2 × 10 min. (b) (1) EDC, HOBt, DIPEA, DMF, 5 °C, 3 h, (2) TFA/H<sub>2</sub>O (95:5 v/v), rt, 3 h, 27–66%; (c) (1) EDC, HOBt, DMF, 5 °C, 3 h, (2) TFA/H<sub>2</sub>O (95:5 v/v), rt, 3 h, 30–75%; (d) acetic anhydride, DIPEA, DMF, rt, 2 h, 78%; (e) DIPEA, DMF, rt, 30 min, 80%.</p></p></figure><div class="NLM_p">The estimation of the appropriate length of the linker, by which the dibenzodiazepinone scaffold and the peptide moiety are held together, was supported by the reported crystal structure of the M<sub>2</sub>R bound to the nonselective MR antagonist 3-quinuclidinyl benzilate (QNB) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON">3UON</a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>): recently, induced fit docking of the dibenzodiazepinone derivative <b>7</b> (cf. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) to this crystal structure, followed by MD simulation (2 μs), yielded a binding pose of <b>7</b> (included in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S14, Supporting Information</a>),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> which was used to identify the putative position of the tricyclic dibenzodiazepinone moiety of the DIBA–peptide conjugates in the orthosteric binding site of the M<sub>2</sub>R. The chosen lengths of the two linkers were supposed to allow for a dualsteric binding mode of the DIBA–peptide conjugates, i.e. the dibenzodiazepinone pharmacophore presumably residing in the orthosteric binding site, and the peptide moiety being located in the vestibule or at the surface of the receptor.</div><div class="NLM_p">The N-terminally acetylated peptides <b>11</b>–<b>25</b> were synthesized on a chlorotrityl resin according to the Fmoc strategy. After coupling and Fmoc-deprotection of the final amino acids, the N-termini were acetylated by treatment of the resins with acetic anhydride. The peptides were cleaved off the resin under mild acidic conditions to maintain protecting groups at the side chains (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Coupling of the C-termini of <b>11</b>–<b>21</b> to amines <b>26</b> and <b>27</b>, followed by cleavage of protecting groups, afforded the DIBA–peptide conjugates <b>28</b>–<b>49</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Likewise, conjugation of peptides <b>22</b>–<b>25</b> to amine <b>27</b>, followed by deprotection, yielded compounds <b>50</b>–<b>53</b>. Whereas pronounced epimerization occurred during amide bond formation between the peptides and amine <b>26</b> or <b>27</b> when using HBTU/HOBt as coupling reagent and an excess of DIPEA (data not shown), epimerization was markedly reduced or precluded by using EDC/HOBt as coupling reagent and avoiding excessive DIPEA (in the case of amine <b>26</b>) or omitting DIPEA (in the case of amine <b>27</b>). Coupling of Boc-Arg(Pbf)-OH (<b>54</b>) to amine <b>27</b>, followed by deprotection, afforded <b>55</b>, which was acetylated to give <b>56</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Propionylation of amine <b>27</b> yielded the reference compound <b>58</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><div class="NLM_p">M<sub>1</sub>–M<sub>5</sub> receptor affinities of <b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, and <b>58</b> were determined by competition binding with the orthosteric MR antagonist [<sup>3</sup>H]NMS using genetically engineered CHO cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) (displacement curves are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figures S1–S5, Supporting Information</a>, and curve slopes are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Table S1, Supporting Information</a>). The subset of compounds with the longer linker, containing a basic piperazine moiety (<b>28</b>–<b>38</b>), exhibited higher affinities at all MR subtypes compared to the subset of derivatives containing the shorter linker (<b>39</b>–<b>53</b>, <b>55</b>, <b>56</b>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. M<sub>1</sub>–M<sub>5</sub> Receptor Binding Data (p<i>K</i><sub>i</sub>) of <b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, and <b>58</b> Obtained from Equilibrium Competition Binding Experiments with [<sup>3</sup>H]NMS at Intact CHO-hM<sub><i>x</i></sub>R Cells (<i>x</i> = 1–5)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center">p<i>K</i><sub>i</sub></th><th class="rowsep1 colsep0" colspan="4" align="center">selectivity toward M<sub>2</sub>R (ratio <i>K</i><sub>i</sub>(M<sub>1,3,4,5</sub>R)/<i>K</i><sub>i</sub>(M<sub>2</sub>R))</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmpd.</th><th class="colsep0 rowsep0" align="center">structure<a class="ref internalNav" href="#tbl1-fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">M<sub>1</sub>R</th><th class="colsep0 rowsep0" align="center">M<sub>2</sub>R</th><th class="colsep0 rowsep0" align="center">M<sub>3</sub>R</th><th class="colsep0 rowsep0" align="center">M<sub>4</sub>R</th><th class="colsep0 rowsep0" align="center">M<sub>5</sub>R</th><th class="colsep0 rowsep0" align="center">M<sub>1</sub></th><th class="colsep0 rowsep0" align="center">M<sub>3</sub></th><th class="colsep0 rowsep0" align="center">M<sub>4</sub></th><th class="colsep0 rowsep0" align="center">M<sub>5</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">Ac-Ala-Ala-NH-X</td><td class="colsep0 rowsep0" align="left">8.50 ± 0.05</td><td class="colsep0 rowsep0" align="left">9.37 ± 0.05</td><td class="colsep0 rowsep0" align="left">6.81 ± 0.09</td><td class="colsep0 rowsep0" align="left">8.61 ± 0.10</td><td class="colsep0 rowsep0" align="left">6.34 ± 0.07</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">370</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">1100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">Ac-Trp-Tyr-NH-X</td><td class="colsep0 rowsep0" align="left">9.06 ± 0.17</td><td class="colsep0 rowsep0" align="left">9.83 ± 0.12</td><td class="colsep0 rowsep0" align="left">7.87 ± 0.09</td><td class="colsep0 rowsep0" align="left">8.97 ± 0.13</td><td class="colsep0 rowsep0" align="left">7.41 ± 0.04</td><td class="colsep0 rowsep0" align="left">6.3</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">7.0</td><td class="colsep0 rowsep0" align="left">220</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">Ac-Trp-Ala-Tyr-NH-X</td><td class="colsep0 rowsep0" align="left">8.64 ± 0.05</td><td class="colsep0 rowsep0" align="left">10.1 ± 0.03</td><td class="colsep0 rowsep0" align="left">7.45 ± 0.06</td><td class="colsep0 rowsep0" align="left">8.79 ± 0.04</td><td class="colsep0 rowsep0" align="left">7.51 ± 0.10</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">410</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">390</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">Ac-Gln-Leu-NH-X</td><td class="colsep0 rowsep0" align="left">8.62 ± 0.04</td><td class="colsep0 rowsep0" align="left">9.62 ± 0.10</td><td class="colsep0 rowsep0" align="left">7.21 ± 0.13</td><td class="colsep0 rowsep0" align="left">8.92 ± 0.15</td><td class="colsep0 rowsep0" align="left">6.53 ± 0.13</td><td class="colsep0 rowsep0" align="left">9.6</td><td class="colsep0 rowsep0" align="left">270</td><td class="colsep0 rowsep0" align="left">5.4</td><td class="colsep0 rowsep0" align="left">1300</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">Ac-Gln-Ala-Leu-NH-X</td><td class="colsep0 rowsep0" align="left">8.53 ± 0.07</td><td class="colsep0 rowsep0" align="left">9.45 ± 0.05</td><td class="colsep0 rowsep0" align="left">6.82 ± 0.09</td><td class="colsep0 rowsep0" align="left">8.45 ± 0.11</td><td class="colsep0 rowsep0" align="left">6.62 ± 0.10</td><td class="colsep0 rowsep0" align="left">8.4</td><td class="colsep0 rowsep0" align="left">440</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">710</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">Ac-Lys-Tyr-NH-X</td><td class="colsep0 rowsep0" align="left">8.85 ± 0.02</td><td class="colsep0 rowsep0" align="left">10.1 ± 0.20</td><td class="colsep0 rowsep0" align="left">7.56 ± 0.01</td><td class="colsep0 rowsep0" align="left">8.95 ± 0.16</td><td class="colsep0 rowsep0" align="left">7.15 ± 0.15</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">260</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">780</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">Ac-Lys-Ala-Tyr-NH-X</td><td class="colsep0 rowsep0" align="left">8.90 ± 0.15</td><td class="colsep0 rowsep0" align="left">9.76 ± 0.08</td><td class="colsep0 rowsep0" align="left">7.35 ± 0.07</td><td class="colsep0 rowsep0" align="left">8.90 ± 0.13</td><td class="colsep0 rowsep0" align="left">7.02 ± 0.14</td><td class="colsep0 rowsep0" align="left">8.3</td><td class="colsep0 rowsep0" align="left">240</td><td class="colsep0 rowsep0" align="left">8.1</td><td class="colsep0 rowsep0" align="left">560</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">Ac-Tyr-Lys-NH-X</td><td class="colsep0 rowsep0" align="left">8.67 ± 0.17</td><td class="colsep0 rowsep0" align="left">10.1 ± 0.03</td><td class="colsep0 rowsep0" align="left">7.79 ± 0.06</td><td class="colsep0 rowsep0" align="left">9.04 ± 0.12</td><td class="colsep0 rowsep0" align="left">6.85 ± 0.05</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">220</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">1900</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">Ac-Tyr-Ala-Lys-NH-X</td><td class="colsep0 rowsep0" align="left">8.63 ± 0.10</td><td class="colsep0 rowsep0" align="left">9.94 ± 0.02</td><td class="colsep0 rowsep0" align="left">7.55 ± 0.13</td><td class="colsep0 rowsep0" align="left">8.49 ± 0.03</td><td class="colsep0 rowsep0" align="left">7.53 ± 0.08</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">290</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">260</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">Ac-Lys-Arg-NH-X</td><td class="colsep0 rowsep0" align="left">8.91 ± 0.10</td><td class="colsep0 rowsep0" align="left">10.2 ± 0.06</td><td class="colsep0 rowsep0" align="left">7.31 ± 0.06</td><td class="colsep0 rowsep0" align="left">8.69 ± 0.17</td><td class="colsep0 rowsep0" align="left">6.95 ± 0.11</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">800</td><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">1900</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b> (UR-AP158)</td><td class="colsep0 rowsep0" align="left">Ac-Lys-Ala-Arg-NH-X</td><td class="colsep0 rowsep0" align="left">8.64 ± 0.03</td><td class="colsep0 rowsep0" align="left">10.2 ± 0.03</td><td class="colsep0 rowsep0" align="left">6.96 ± 0.07</td><td class="colsep0 rowsep0" align="left">8.68 ± 0.12</td><td class="colsep0 rowsep0" align="left">6.53 ± 0.09</td><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">1800</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">4900</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">Ac-Ala-Ala-NH-Y</td><td class="colsep0 rowsep0" align="left">6.52 ± 0.09</td><td class="colsep0 rowsep0" align="left">7.64 ± 0.04</td><td class="colsep0 rowsep0" align="left">5.86 ± 0.03</td><td class="colsep0 rowsep0" align="left">7.10 ± 0.09</td><td class="colsep0 rowsep0" align="left">5.87 ± 0.11</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="left">62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">Ac-Trp-Tyr-NH-Y</td><td class="colsep0 rowsep0" align="left">6.81 ± 0.12</td><td class="colsep0 rowsep0" align="left">7.48 ± 0.05</td><td class="colsep0 rowsep0" align="left">5.80 ± 0.04</td><td class="colsep0 rowsep0" align="left">7.03 ± 0.05</td><td class="colsep0 rowsep0" align="left">7.23 ± 0.12</td><td class="colsep0 rowsep0" align="left">5.0</td><td class="colsep0 rowsep0" align="left">46</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">Ac-Trp-Ala-Tyr-NH-Y</td><td class="colsep0 rowsep0" align="left">7.03 ± 0.06</td><td class="colsep0 rowsep0" align="left">7.70 ± 0.05</td><td class="colsep0 rowsep0" align="left">5.94 ± 0.04</td><td class="colsep0 rowsep0" align="left">7.43 ± 0.11</td><td class="colsep0 rowsep0" align="left">6.92 ± 0.09</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="left">56</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">6.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">Ac-Gln-Leu-NH-Y</td><td class="colsep0 rowsep0" align="left">5.98 ± 0.07</td><td class="colsep0 rowsep0" align="left">6.51 ± 0.02</td><td class="colsep0 rowsep0" align="left"><5</td><td class="colsep0 rowsep0" align="left">5.97 ± 0.08</td><td class="colsep0 rowsep0" align="left">5.82 ± 0.04</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">>27</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left">4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">Ac-Gln-Ala-Leu-NH-Y</td><td class="colsep0 rowsep0" align="left">6.27 ± 0.12</td><td class="colsep0 rowsep0" align="left">6.62 ± 0.10</td><td class="colsep0 rowsep0" align="left"><5</td><td class="colsep0 rowsep0" align="left">6.38 ± 0.11</td><td class="colsep0 rowsep0" align="left">5.92 ± 0.04</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">>36</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">4.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">Ac-Lys-Tyr-NH-Y</td><td class="colsep0 rowsep0" align="left">7.52 ± 0.10</td><td class="colsep0 rowsep0" align="left">8.74 ± 0.08</td><td class="colsep0 rowsep0" align="left">5.79 ± 0.09</td><td class="colsep0 rowsep0" align="left">7.97 ± 0.08</td><td class="colsep0 rowsep0" align="left">7.13 ± 0.12</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">720</td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">Ac-Lys-Ala-Tyr-NH-Y</td><td class="colsep0 rowsep0" align="left">7.23 ± 0.05</td><td class="colsep0 rowsep0" align="left">8.37 ± 0.02</td><td class="colsep0 rowsep0" align="left">6.18 ± 0.05</td><td class="colsep0 rowsep0" align="left">7.63 ± 0.10</td><td class="colsep0 rowsep0" align="left">6.65 ± 0.05</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">160</td><td class="colsep0 rowsep0" align="left">5.8</td><td class="colsep0 rowsep0" align="left">53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">Ac-Tyr-Lys-NH-Y</td><td class="colsep0 rowsep0" align="left">7.06 ± 0.01</td><td class="colsep0 rowsep0" align="left">7.33 ± 0.04</td><td class="colsep0 rowsep0" align="left">5.00 ± 0.01</td><td class="colsep0 rowsep0" align="left">6.53 ± 0.08</td><td class="colsep0 rowsep0" align="left">6.15 ± 0.02</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">210</td><td class="colsep0 rowsep0" align="left">6.6</td><td class="colsep0 rowsep0" align="left">15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">Ac-Tyr-Ala-Lys-NH-Y</td><td class="colsep0 rowsep0" align="left">6.75 ± 0.08</td><td class="colsep0 rowsep0" align="left">8.03 ± 0.10</td><td class="colsep0 rowsep0" align="left">5.30 ± 0.04</td><td class="colsep0 rowsep0" align="left">7.24 ± 0.05</td><td class="colsep0 rowsep0" align="left">6.68 ± 0.09</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">550</td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b> (UR-AP148)</td><td class="colsep0 rowsep0" align="left">Ac-Lys-Arg-NH-Y</td><td class="colsep0 rowsep0" align="left">7.38 ± 0.14</td><td class="colsep0 rowsep0" align="left">8.97 ± 0.02</td><td class="colsep0 rowsep0" align="left">5.17 ± 0.07</td><td class="colsep0 rowsep0" align="left">7.21 ± 0.06</td><td class="colsep0 rowsep0" align="left">6.37 ± 0.07</td><td class="colsep0 rowsep0" align="left">49</td><td class="colsep0 rowsep0" align="left">6500</td><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">400</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="left">Ac-Lys-Ala-Arg-NH-Y</td><td class="colsep0 rowsep0" align="left">6.94 ± 0.03</td><td class="colsep0 rowsep0" align="left">8.11 ± 0.02</td><td class="colsep0 rowsep0" align="left">5.57 ± 0.06</td><td class="colsep0 rowsep0" align="left">7.05 ± 0.11</td><td class="colsep0 rowsep0" align="left">6.37 ± 0.08</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">350</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="left">Ac-Arg-Lys-NH-Y</td><td class="colsep0 rowsep0" align="left">7.02 ± 0.08</td><td class="colsep0 rowsep0" align="left">7.92 ± 0.18</td><td class="colsep0 rowsep0" align="left">5.26 ± 0.13</td><td class="colsep0 rowsep0" align="left">6.59 ± 0.06</td><td class="colsep0 rowsep0" align="left">6.30 ± 0.19</td><td class="colsep0 rowsep0" align="left">6.5</td><td class="colsep0 rowsep0" align="left">400</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="left">Ac-Arg-Arg-NH-Y</td><td class="colsep0 rowsep0" align="left">7.70 ± 0.11</td><td class="colsep0 rowsep0" align="left">8.77 ± 0.03</td><td class="colsep0 rowsep0" align="left">6.02 ± 0.19</td><td class="colsep0 rowsep0" align="left">7.53 ± 0.16</td><td class="colsep0 rowsep0" align="left">6.79 ± 0.05</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">710</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">Ac-Arg-Lys-Gly-NH-Y</td><td class="colsep0 rowsep0" align="left">7.64 ± 0.05</td><td class="colsep0 rowsep0" align="left">8.83 ± 0.10</td><td class="colsep0 rowsep0" align="left">6.11 ± 0.16</td><td class="colsep0 rowsep0" align="left">7.76 ± 0.03</td><td class="colsep0 rowsep0" align="left">6.44 ± 0.13</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">620</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">260</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">Ac-Lys-Arg-Gly-NH-Y</td><td class="colsep0 rowsep0" align="left">7.49 ± 0.12</td><td class="colsep0 rowsep0" align="left">8.38 ± 0.05</td><td class="colsep0 rowsep0" align="left">6.19 ± 0.15</td><td class="colsep0 rowsep0" align="left">7.56 ± 0.10</td><td class="colsep0 rowsep0" align="left">6.70 ± 0.10</td><td class="colsep0 rowsep0" align="left">8.2</td><td class="colsep0 rowsep0" align="left">180</td><td class="colsep0 rowsep0" align="left">6.9</td><td class="colsep0 rowsep0" align="left">49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="left">H-Arg-NH-Y</td><td class="colsep0 rowsep0" align="left">6.97 ± 0.07</td><td class="colsep0 rowsep0" align="left">8.27 ± 0.12</td><td class="colsep0 rowsep0" align="left">5.90 ± 0.17</td><td class="colsep0 rowsep0" align="left">7.25 ± 0.11</td><td class="colsep0 rowsep0" align="left">5.62 ± 0.11</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">260</td><td class="colsep0 rowsep0" align="left">9.8</td><td class="colsep0 rowsep0" align="left">410</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">Ac-Arg-NH-Y</td><td class="colsep0 rowsep0" align="left">7.01 ± 0.09</td><td class="colsep0 rowsep0" align="left">7.95 ± 0.09</td><td class="colsep0 rowsep0" align="left">5.83 ± 0.14</td><td class="colsep0 rowsep0" align="left">6.83 ± 0.10</td><td class="colsep0 rowsep0" align="left">6.01 ± 0.11</td><td class="colsep0 rowsep0" align="left">8.4</td><td class="colsep0 rowsep0" align="left">140</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="left">propanoyl-NH-Y</td><td class="colsep0 rowsep0" align="left">7.07 ± 0.05</td><td class="colsep0 rowsep0" align="left">7.10 ± 0.05</td><td class="colsep0 rowsep0" align="left">5.27 ± 0.08</td><td class="colsep0 rowsep0" align="left">6.68 ± 0.08</td><td class="colsep0 rowsep0" align="left">5.15 ± 0.14</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">69</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">98</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Presented are mean values ± SEM from at least three independent experiments (each performed in triplicate). <i>K</i><sub>d</sub> values (reported previously<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a>)/concentrations of [<sup>3</sup>H]NMS: M<sub>1</sub>, 0.15/0.2 nM; M<sub>2</sub>, 0.090/0.2 nM; M<sub>3</sub>, 0.089/0.2 nM; M<sub>4</sub>, 0.035/0.1 nM; M<sub>5</sub>, 0.24/0.3 nM. For the most M<sub>2</sub>R selective members of the presented series of compounds lab codes are given.</p></div><div class="footnote" id="tbl1-fn1"><sup><sup>b</sup></sup><p class="last">For X and Y, see <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</p></div></div></div><div class="NLM_p">Strikingly, within the “long-linker” subset of compounds (<b>28</b>–<b>38</b>), the ranges of p<i>K</i><sub>i</sub> values determined for M<sub>1</sub>R, M<sub>2</sub>R, and M<sub>4</sub>R binding were very narrow (p<i>K</i><sub>i</sub>: M<sub>1</sub>R 8.50–9.06, M<sub>2</sub>R 9.37–10.2, M<sub>4</sub>R 8.45–9.04; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). By contrast, the “short-linker” DIBA–peptide conjugates (<b>39</b>–<b>53</b>) exhibited wider ranges of MR affinities (e.g., p<i>K</i><sub>i</sub> M<sub>2</sub>R: 6.4–8.97), suggesting that in the cases of the M<sub>1</sub>, M<sub>2</sub>, and M<sub>4</sub> receptors, the SAR of the “long-linker” series of compounds did not correlate with the SAR of the respective “short-linker” counterparts. Indeed, this became obvious by plotting the p<i>K</i><sub>i</sub> values (M<sub>1</sub>R, M<sub>2</sub>R, M<sub>4</sub>R) of compounds <b>28</b>–<b>38</b> (“long-linker” DIBA–peptide conjugates) against the p<i>K</i><sub>i</sub> values of <b>39</b>–<b>49</b> (corresponding “short-linker” DIBA–peptide conjugates) and linear regression analysis (<i>R</i><sup>2</sup> < 0.44, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S6, Supporting Information</a>). Likewise, the SARs of both linker series were also nonparallel in the case of the M<sub>3</sub>R (<i>R</i><sup>2</sup> = 0.061, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S6, Supporting Information</a>). Merely the M<sub>5</sub>R binding data revealed a slight linear correlation between the two linker series (<i>R</i><sup>2</sup> = 0.71, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S6, Supporting Information</a>). Consequently, the missing correlation between the SARs of the two series of compounds (long vs. short linker) is also reflected by disagreeing MR selectivity profiles of individual pairs of compounds differing only in the type of the linker, i.e. bearing identical peptide and dibenzodiazepinone moieties (e.g., <b>30</b> vs. <b>41</b>, selectivity profiles: <i>K</i><sub>i</sub> M<sub>1</sub>R/M<sub>2</sub>R/M<sub>3</sub>R/M<sub>4</sub>R/M<sub>5</sub>R of 27:1:410:19:390 vs 4.7:1:56:1.9:6.2; cf. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The degree of correlation between the selectivity profiles of such pairs of DIBA peptide conjugates also became obvious by plotting the p<i>K</i><sub>i</sub> values (M<sub>1</sub>R–M<sub>5</sub>R) of the “short-linker” compound against the p<i>K</i><sub>i</sub> values (M<sub>1</sub>R–M<sub>5</sub>R) of the “long-linker” counterpart (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S7, Supporting Information</a>): for the compound pairs <b>29</b>/<b>40</b>, <b>30</b>/<b>41</b>, <b>33</b>/<b>44</b>, <b>35</b>/<b>46</b>, and <b>36</b>/<b>47</b> the correlation was rather low (<i>R</i><sup>2</sup> < 0.71), and for the pairs <b>28</b>/<b>39</b>, <b>34</b>/<b>45</b>, <b>37</b>/<b>48</b>, and <b>38</b>/<b>49</b> a moderate linear correlation was found (<i>R</i><sup>2</sup> = 0.83–0.89).</div><div class="NLM_p">The DIBA–peptide derivative with the lowest pronounced M<sub>2</sub>R selectivity, containing Gln, Ala, and Leu, was found in the “short-linker” subset of compounds (<b>43</b>, cf. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The “long-linker” derivative <b>38</b> (UR-AP158) and the “short-linker” derivative <b>48</b> (UR-AP148) exhibited the highest M<sub>2</sub>R affinities within the respective compound subsets as well as higher M<sub>2</sub>R selectivities (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) compared to reported, so-called “M<sub>2</sub>R-selective” pyridobenzo- and dibenzodiazepinone-type MR ligands (e.g., <b>2</b> and <b>5</b>, cf. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16">(9−16)</a> As both <b>38</b> and <b>48</b> contain two basic amino acids (Lys, Arg), these results suggest that positively charged residues are advantageous for M<sub>2</sub>R binding, being in agreement with reports on orthosteric and allosteric M<sub>2</sub>R ligands.<a onclick="showRef(event, 'ref19 ref21 ref23'); return false;" href="javascript:void(0);" class="ref ref19 ref21 ref23">(19,21,23)</a></div><div class="NLM_p">For the majority of the presented compounds, the M<sub>2</sub>R selectivity toward the M<sub>1</sub>R and M<sub>4</sub>R was less pronounced than toward the M<sub>3</sub>R and M<sub>5</sub>R (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), being consistent with the selectivity profile of previously described, tricyclic M<sub>2</sub>R-preferring MR antagonists (cf. selectivity profiles of <b>2</b>, <b>3a</b>, <b>4</b>, and <b>7</b>–<b>10</b> given in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The modulatory effect of the peptidyl moiety on M<sub>2</sub>R selectivity was confirmed by the low M<sub>2</sub>R selectivity of reference compound <b>58</b>, which contains a small propionyl group instead of the peptide moiety (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">For <b>38</b> and <b>48</b>, showing the highest M<sub>2</sub>R selectivity, M<sub>2</sub>R antagonism was investigated and confirmed in an IP1 accumulation assay<a onclick="showRef(event, 'ref15 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref15 ref24 ref25">(15,24,25)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The pIC<sub>50</sub> values of <b>38</b> and <b>48</b> amounted to 7.98 and 7.03, respectively. Moreover, plasma stabilities were studied for <b>38</b>, <b>48</b>, and <b>52</b>, which proved to be highly stable (>99% intact compound after 24 h, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Investigation of M<sub>2</sub>R agonism and antagonism of compounds <b>38</b> and <b>48</b> in an IP1 accumulation assay using HEK-hM<sub>2</sub>-Gα<sub>qi5-HA</sub> cells. (A) Concentration-dependent effects of CCh, <b>38</b>, and <b>48</b> on the accumulation of IP1. <b>38</b> and <b>48</b> elicited no response. pEC<sub>50</sub> of CCh (mean ± SEM) is 6.97 ± 0.03. Data represent mean values ± SEM from 14 (CCh) or 4 (<b>38</b>, <b>48</b>) independent experiments (performed in duplicate). (B) Concentration-dependent inhibition of the CCh (0.3 μM) induced IP1 accumulation by the MR antagonists atropine, <b>38</b>, and <b>48</b>. pIC<sub>50</sub> values (mean ± SEM) are the following: <b>38</b>, 7.98 ± 0.19; <b>48</b>, 7.03 ± 0.06; atropine, 8.09 ± 0.03. Data represent mean values ± SEM from four (<b>38</b>), five (atropine), or six (<b>48</b>) independent experiments (performed in duplicate).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Stabilities of <b>38</b>, <b>48</b>, and <b>52</b> in Human Plasma/PBS (1:2 v/v) at 37 °C<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">% intact peptide in plasma after the specified incubation time</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">1 h</th><th class="colsep0 rowsep0" align="center">6 h</th><th class="colsep0 rowsep0" align="center">24 h</th><th class="colsep0 rowsep0" align="center">48 h</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="left">99 ± 3</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">83 ± 6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The initial concentrations of compounds <b>38</b>, <b>48</b>, and <b>52</b> in plasma/PBS (1:2 v/v) were 100 μM. Data represent the mean (±SEM) of three independent experiments (SEM not given when no decomposition was detected). LC–MS analysis revealed the Lys-Gly amide bond in <b>52</b> as the major cleavage site (for structure of <b>52</b>, see <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> or <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</p></div></div></div><div class="NLM_p">In order to visualize the interaction of <b>38</b> and <b>48</b> with MRs by computational chemistry, MD simulations (20 μs) were performed with all MR subtypes bound to <b>38</b> or <b>48</b> (except for the M<sub>3</sub>R bound to <b>38</b>). The lowest free energy (MM-PBSA) binding poses obtained from the MD simulations and the time courses over the simulations are depicted in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> (M<sub>2</sub>R) and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figures S9–S12 (Supporting Information)</a> (M<sub>1</sub>R, M<sub>3</sub>R–M<sub>5</sub>R). The simulations revealed a dualsteric binding mode of <b>38</b> and <b>48</b> at the M<sub>1</sub>R, M<sub>2</sub>R, M<sub>4</sub>R, and M<sub>5</sub>R with the tricyclic dibenzodiazepinone moiety residing in the orthosteric binding pocket (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figures S9, S11 and S12, Supporting Information</a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. MD simulations (20 μs) of the hM<sub>2</sub>R (inactive state, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON">3UON</a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>) bound to <b>38</b> (A, C) or <b>48</b> (B, D). (A, B) Lowest free energy (MM-PBSA) poses of <b>38</b> (green) and <b>48</b> (purple), respectively, obtained from analysis of the MD simulation trajectories. Amino acids involved in H-bonding, salt bridges (both indicated as yellow dashed lines), or π–π interactions (green dashed lines) with the ligands are labeled: in (A), D103<sup>3.32</sup> (HB, SB), E172<sup>ECL2</sup> (HB, SB), E175<sup>ECL2</sup> (HB, SB), C176<sup>ECL2</sup> (HB), I178<sup>ECL2</sup> (HB), Y403<sup>6.51</sup> (π–π), N404<sup>6.52</sup> (HB), N419<sup>ECL3</sup> (HB), and Y426<sup>7.39</sup> (HB); in (B), Y80<sup>2.61</sup> (HB), D103<sup>3.32</sup> (HB, SB), Y104<sup>3.33</sup> (HB), E172<sup>ECL2</sup> (HB, SB), E175<sup>ECL2</sup> (HB, SB), I178<sup>ECL2</sup> (HB). (C, D) Time courses of the 20 μs MD simulations of the hM<sub>2</sub>R bound to <b>38</b> (C) or <b>48</b> (D) showing superimposed snapshots collected every 1 μs. HB = hydrogen bond. SB = salt bridge.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This was consistent with previously reported investigations on the binding mode of <b>7</b>, <b>9</b>, and <b>10</b> at the M<sub>2</sub>R <a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> and in agreement with the fact that <b>38</b> and <b>48</b> were capable of fully displacing the orthosteric ligand [<sup>3</sup>H]NMS from M<sub>2</sub>Rs when using a high radioligand concentration (i.e., receptor occupancy by [<sup>3</sup>H]NMS > 95%) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), indicating a competitive mechanism between the orthosteric radioligand [<sup>3</sup>H]NMS and the DIBA–peptide conjugates <b>38</b> and <b>48</b>. An MD simulation of the M<sub>3</sub>R, bound to <b>38</b>, was not performed as preceding induced-fit docking afforded no reasonable binding pose for <b>38</b>. In the case of <b>48</b>, the initial dualsteric binding pose of <b>48</b> at the M<sub>3</sub>R started to disappear during the simulation due to an upward movement of the dibenzodiazepinone moiety (“unbinding” process, cf. <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figure S10B, Supporting Information</a>), being in agreement with the low binding affinity of <b>48</b> at the M<sub>3</sub>R (p<i>K</i><sub>i</sub> = 5.17, cf. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of the DIBA–peptide conjugates <b>38</b> and <b>48</b> on M<sub>2</sub>R equilibrium binding of [<sup>3</sup>H]NMS (<i>K</i><sub>d</sub> = 0.09 nM) using a radioligand concentration of 0.2 nM (filled symbols, data taken from <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figures S2 and S4</a>) or 2 nM (open symbols). Experiments were performed at intact CHO-hM<sub>2</sub>R cells at 22 °C. Unspecific binding was determined in the presence of atropine (500-fold excess to [<sup>3</sup>H]NMS). Data, representing mean values ± SEM from at least four independent experiments (each performed in triplicate), were analyzed by four-parameter logistic fits. The use of a high radioligand concentration, resulting in a high receptor occupancy by [<sup>3</sup>H]NMS, can “unmask” a potential noncompetitive, negative allosteric mechanism as previously demonstrated, e.g., for the negative allosteric M<sub>2</sub>R modulator W84 (indicated by an elevated lower curve plateau and a missing rightward shift of the curves; cf. Figure 8C in Pegoli et al., 2017<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>). Here, the determination of the effects of <b>38</b> and <b>48</b> on M<sub>2</sub>R equilibrium binding of [<sup>3</sup>H]NMS yielded curves that were rightward shifted and exhibited lower curve plateaus not different from zero. This indicates a competitive mechanism between the orthosteric MR antagonist NMS and <b>38</b> as well as NMS and <b>48</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A detailed discussion of the binding modes of <b>38</b> and <b>48</b>, suggested by MD simulations, is provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Supporting Information</a>. It should be mentioned that the coordinates of the recently reported crystal structure of the M<sub>2</sub>R in complex with <b>2</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZKB">5ZKB</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>) were not available at the time the presented MD simulations with the M<sub>2</sub>R were performed. As the dibenzodiazepinone substructure of the investigated DIBA–peptide conjugates is structurally similar to compound <b>2</b> (structure shown <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), the recently reported M<sub>2</sub>R structure<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> might be considered as template for future computational studies of M<sub>2</sub>R binding of tricyclic muscarinic receptor antagonists.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Attachment of di- and tripeptides to dibenzodiazepinone-type MR ligands via two different linkers yielded DIBA–peptide conjugates (“long-linker” series <b>28</b>–<b>38</b>, “short-linker” series <b>39</b>–<b>53</b>) with medium to high M<sub>2</sub>R affinity. M<sub>1</sub>R–M<sub>5</sub>R binding data of these compounds revealed that M<sub>2</sub>R selectivity is dependent on the linker type and can be tuned by the peptide sequence. The study suggests basic residues to be favorable with respect to M<sub>2</sub>R selectivity. The following conclusions can be drawn from MR affinities and MD simulations, the latter confirming a dualsteric binding mode at MRs; whereas the longer linker, containing a basic piperazine moiety, is favorable with respect to high M<sub>2</sub>R affinity, the shorter aliphatic linker is favorable in terms of tuning MR selectivity by the peptide moiety.</div><div class="NLM_p">The presented type of selective M<sub>2</sub>R antagonists (compounds <b>38</b> and <b>48</b>) requires, in contrast to M<sub>2</sub>R ligands such as <b>1a</b> (cf. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), structural optimization with respect to a potential application as therapeutics acting in the CNS. This may be achieved by a bioisosteric replacement of strongly basic moieties, such as guanidine groups, by moderately basic surrogates. As MR antagonists such <b>38</b> or <b>48</b> represent the cold forms of potential tritiated selective M<sub>2</sub>R ligands, accessible by replacement of the acetyl residue by a commercially available tritiated acetyl moiety, the presented compound class is superior to <b>1a</b> in terms of providing convenient access to radiolabeled M<sub>2</sub>R-selective ligands, which represent useful molecular tools.</div><div class="NLM_p last">To the best of our knowledge, the presented approach, i.e., triggering receptor subtype selectivity of small-molecule, non-peptide receptor ligands by attachment of short peptides, has not been described before. This concept can potentially be applied to other receptor families with respect to the identification of lead structures for the development of dualsteric ligands with improved subtype selectivity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02445" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02445" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> General Experimental Conditions</h3><div class="NLM_p last">Acetonitrile (HPLC gradient grade) was obtained from Honeywell (Seelze, Germany) or Sigma-Aldrich (Taufkirchen, Germany). DMF, atropine, EDC, DIPEA, HFIP, 1-methyl-<span class="smallcaps smallerCapital">d</span>-tryptophan (<b>59</b>), and TFA were purchased from Sigma-Aldrich. Absolute EtOH was obtained from Honeywell. CH<sub>2</sub>Cl<sub>2</sub>, HOBt, DMF for peptide synthesis and NMP for peptide synthesis were obtained from Acros Organics/Fisher Scientific (Nidderau, Germany). Boc-Arg(Pbf)-OH (<b>54</b>) was purchased from Carbolution Chemicals (St. Ingbert, Germany). Fmoc-Ala-OH, Fmoc-Arg(Pbf)-OH, Fmoc-Gln-(Trt)-OH, Fmoc-Lys(Boc)-OH, Fmoc-Trp(Boc)-OH, Fmoc-Tyr(tBu)-OH), acetic anhydride, and solid supports for SPPS (H-Ala-, H-Arg(Pbf)-, H-Leu-, H-Lys(Boc)-, H-Tyr(tBu)-, and H-Gly-2-ClTrt PS resin) were purchased from Merck Chemicals (Darmstadt, Germany). Deuterated solvents were from Deutero (Kastellaun, Germany). HBTU and piperidine were from Iris Biotech (Marktredwitz, Germany). The syntheses of compounds <b>26</b>,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a><b>27</b>,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and <b>57</b><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> were described elsewhere. Millipore water was used for the preparation of buffers and HPLC eluents. 1.5 or 2 mL polypropylene reaction vessels with screw cap (Süd-Laborbedarf, Gauting, Germany) were used for small scale reactions for the investigation of plasma stabilities and for the preparation and storage of stock solutions. Temperature-controlled reactions and plasma stabilities were performed in 1.5 mL reaction vessels using a Thermocell mixing block MB-102 from Bioer (Hangzhou, China). NMR spectra were recorded on a Bruker Avance III HD 600 equipped with a cryogenic probe (14.1 T, <sup>1</sup>H, 600.1 MHz; <sup>13</sup>C, 150.9 MHz) (Bruker, Karlsruhe, Germany). Abbreviations for the multiplicities of the signals are s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), and brs (broad singlet). HRMS was performed on an Agilent 6540 UHD Accurate-Mass Q-TOF LC/MS system (Agilent Technologies, Santa Clara, CA USA) using an ESI source. Optical rotations at 589 nm (Na D line) were measured on a polarimeter P8000-T equipped with an electronic Peltier thermostat PT31 (A. KRÜSS Optronic, Hamburg, Germany) using a thermostated (20 °C) microcuvette (layer thickness of 1 dm, volume of 0.9 mL) and acetonitrile/water (3:7 v/v) as solvent. Preparative HPLC was performed with a system from Knauer (Berlin, Germany), composed of two K-1800 pumps and a K-2001 detector, or with a Prep 150 LC system from Waters (Eschborn, Germany) consisting of a 2545 binary gradient module, a 2489 UV/visible detector, and a Waters fraction collector III. When using acetonitrile as organic solvent, a Kinetex-XB C18 (5 μm, 250 mm × 21 mm; Phenomenex, Aschaffenburg, Germany) was used as stationary phase at a flow rate of 18 mL/min (<b>30</b>–<b>33</b>, <b>35</b>–<b>38</b>, <b>41</b>, <b>42</b>, <b>44</b>–<b>53</b>, <b>55</b>, <b>56</b>) or 20 mL/min (<b>28</b>, <b>29</b>, <b>34</b>, <b>39</b>, <b>40</b>, <b>43</b>, <b>58</b>). When using MeOH as organic solvent, a Gemini-NX C18 (5 μm, 250 mm × 21 mm; Phenomenex) served as stationary phase at a flow rate of 15 mL/min (compounds <b>28</b> and <b>39</b>). Mixtures of 0.1% aq TFA and acetonitrile or mixtures of 0.1% aq TFA and MeOH were used as mobile phase. The detection wavelength was 220 nm. The solvent of collected fractions was removed by lyophilization. Analytical HPLC analysis was performed with a system from Agilent Technologies composed of a 1290 Infinity binary pump equipped with a degasser, a 1290 Infinity autosampler, a 1290 Infinity thermostated column compartment, a 1260 Infinity diode array detector, and a 1260 Infinity fluorescence detector. A Kinetex-XB C18 (2.6 μm, 100 × 3 mm; Phenomenex) was used as stationary phase at a flow rate of 0.5 mL/min (<b>28</b>–<b>49</b>) or 0.6 mL/min (<b>50</b>–<b>53</b>, <b>55</b>, <b>56</b>, <b>58</b>). Mixtures of 0.04% aq TFA (A) and acetonitrile (B) or A and MeOH (C) were used as mobile phase. The following linear gradients were applied. Compounds <b>28</b>–<b>48</b>, <b>50</b>–<b>53</b>, <b>55</b>, and <b>56</b>: 0–20 min, A/B 90:10 to 70:30; 20–22 min, 70:30 to 5:95; 22–26 min, 5:95. Compound <b>58</b>: 0–12 min, A/B 90:10 to 70:30; 12–16 min, 70:30 to 5:95; 16–20 min, 5:95. Compounds <b>28</b>, <b>29</b>, <b>31</b>, <b>32</b>, <b>34</b>, <b>39</b>, <b>42</b>, <b>43</b>, and <b>45</b>: 0–20 min, A/C 85:15 to 52:48; 20–23 min, 52:48 to 5:95; 23–27 min, 5:95. Compound <b>40</b>: 0–20 min, A/C 85:15 to 35:65; 20–23 min, 35:65 to 5:95; 23–27 min, 5:95. For all analytical HPLC runs the oven temperature was set to 25 °C, the injection volume was 20 μL, and detection was performed at 220 nm. Compound concentrations were between 40 and 150 μM.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Compound Characterization</h3><div class="NLM_p">Target compounds (<b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, <b>58</b>) were characterized by <sup>1</sup>H NMR, <sup>13</sup>C NMR, and 2D NMR (<sup>1</sup>H COSY, HSQC, HMBC) spectroscopy, HRMS, and RP-HPLC analysis. Additionally, specific optical rotation was determined for compounds <b>38</b>, <b>48</b>, and <b>52</b>. The purity (HPLC, 220 nm) of all target compounds amounted to ≥97% (chromatograms shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p"><i>Annotation Concerning the <sup>1</sup>H and <sup>13</sup>C NMR Spectra of <b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, <b>58</b>.</i> Due to a slow rotation about the exocyclic dibenzodiazepinone amide group on the NMR time scale, two isomers (ratios provided in the experimental protocols) were evident in the <sup>1</sup>H and <sup>13</sup>C NMR spectra.</div><div class="NLM_p last"><i>Annotation Concerning the <sup>1</sup>H NMR Spectra (Solvent: DMSO-<i>d</i><sub>6</sub>) of <b>50</b>–<b>53</b>, <b>55</b>, and <b>56</b></i>. In order to integrate signals interfering with the broad water signal at ∼3.5 ppm, spectra were additionally recorded in DMSO-<i>d</i><sub>6</sub>/D<sub>2</sub>O (4:1 v/v) (spectra shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Screening for Pan-Assay Interference Compounds (PAINS)</h3><div class="NLM_p last">Screening of all target compounds (<b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, <b>58</b>) for PAINS via the public tool <a href="http://zinc15.docking.org/patterns/home" class="extLink">http://zinc15.docking.org/patterns/home</a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> gave no hits.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Solid-Phase Peptide Synthesis (Fmoc Strategy) of N-Terminally Acetylated, Side Chain Protected Peptides <b>11</b>–<b>25</b></h3><div class="NLM_p last">5 mL BD Discardit II syringes (Becton Dickinson, Heidelberg, Germany) or 5 mL B. Braun syringes (ALMO-Erzeugnisse Erwin Busch, Bad Arolsen, Germany) equipped with a polyethylene frit (average pore size 35 μm; Roland Vetter Laborbedarf, Ammerbuch, Germany) were used as reaction vessels. DMF/NMP (8:2 v/v) was used as solvent throughout (∼2.2 mL per 1 mmol Fmoc-aa). Protected amino acids (5-fold excess) were preactivated with HBTU/HOBt/DIPEA (5/5/10 equiv) in DMF/NMP (8:2) at rt for 5 min and added to the resin. The vessel was shaken at rt or 35 °C for 60 min. The coupling procedure was repeated (“double coupling”). After completed coupling of an Fmoc-aa, the resin was washed with DMF/NMP (8:2) (4×) followed by Fmoc deprotection with 20% piperidine in DMF/NMP (8:2) at rt (2 × 10 min) and washing of the resin with DMF/NMP (8:2) (5×). After coupling and Fmoc-deprotection of the final Fmoc-aa, a mixture formed by acetic anhydride (5 equiv) and DIPEA (10 equiv) in DMF/NMP (8:2) was added and the vessel was shaken at rt for 30 min followed by washing with DMF/NMP (8:2) (4×) and CH<sub>2</sub>Cl<sub>2</sub> (treated with K<sub>2</sub>CO<sub>3</sub>) (6×). The side chain protected peptides were cleaved off by treatment of the resin with CH<sub>2</sub>Cl<sub>2</sub>/HFIP (4:1 v/v) at rt (2 × 20 min). The resin was separated by filtration and washed once with CH<sub>2</sub>Cl<sub>2</sub>/HFIP (4:1) (∼1 mL). The filtrates were combined, the volatiles were evaporated, and the residue was dissolved in acetonitrile/water (2:8 or 3:7 v/v) (∼2 mL) followed by the addition of water (∼50 mL) and lyophilization. Compounds <b>11</b>–<b>25</b> were used in the next step without purification. The chemical identities and purities of <b>11</b>–<b>25</b> were assessed by HRMS and analytical HPLC (data not shown).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> General Procedure for the Synthesis of Compounds <b>28</b>–<b>53</b></h3><div class="NLM_p last">The coupling of the side chain protected peptides <b>11</b>–<b>25</b> to amine <b>26</b> or <b>27</b> was performed in 1.5 or 2 mL polypropylene reaction vessels with screw cap. HOBt (1.2 equiv) and EDC (1.2 equiv) were added to a cooled (5 °C) solution of the respective side chain-protected peptide (1 equiv) in DMF (0.1–0.4 mL). In the case of compounds derived from <b>26</b>, a solution of <b>26</b> (tetrakis(hydrotrifluoroacetate), 1–1.2 equiv) and DIPEA (3 equiv) in DMF (0.1 mL) was immediately added and stirring was continued at 5 °C for 3 h. For the synthesis of compounds derived from <b>27</b> a solution of <b>27</b> (free base, 1–1.2 equiv) in DMF (0.1 mL) was immediately added, and stirring was continued at 5 °C for 3 h. 1% aq TFA (0.1–0.4 mL, depending on the amount of DMF used to dissolve the peptide) was added, and the mixture was subjected to preparative HPLC to isolate the protected intermediates (conditions not given). The latter compounds were dissolved in TFA/CH<sub>2</sub>Cl<sub>2</sub>/H<sub>2</sub>O (10:10:1 v/v/v) (products without Arg) or TFA/H<sub>2</sub>O (95:5 v/v) (products containing Arg). The mixtures were stirred at rt for 3 h, CH<sub>2</sub>Cl<sub>2</sub> (∼20 mL) was added, and the volatiles were removed by evaporation. The final products were purified by preparative HPLC.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Experimental Synthetic Protocols and Analytical Data of Compounds <b>55</b>, <b>56</b>, and <b>58</b>. (<i>S</i>)-2-Amino-5-guanidino-<i>N</i>-(4-(1-(2-oxo-2-(11-oxo-10,11-dihydro-5<i>H</i>-dibenzo[<i>b</i>,<i>e</i>][1,4]diazepin-5-yl)ethyl)piperidin-4-yl)butyl)pentanamide Tris(hydrotrifluoroacetate) (<b>55</b>)</h3><div class="NLM_p last">Compound <b>55</b> was prepared from amine <b>27</b> (55.6 mg, 0.137 mmol) and compound <b>54</b> (60 mg, 0.114 mmol) according to the general procedure for the synthesis of <b>28</b>–<b>53</b>. Purification by preparative HPLC (gradient 0–25 min, 0.1% aq TFA/acetonitrile 81:19 to 72:28, <i>t</i><sub>R</sub> = 7 min) yielded <b>55</b> as a white fluffy solid (67 mg, 0.074 mmol, 65%). Ratio of configurational isomers evident in the NMR spectra: ∼1.5:1. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 1.13–1.29 (m, 4H), 1.30–1.52 (m, 7H), 1.64–1.71 (m, 2H), 1.71–1.83 (m, 2H), 2.83–2.96 (m, 1H), 2.96–3.04 (m, 1H), 3.04–3.14 (m, 4H), 3.28–3.35 (m, 0.8H), 3.55–3–63 (m, 1.2H), 3.65–3.73 (m, 1H), 3.75 (d, 0.6H, <i>J</i> 17 Hz), 3.90 (d, 0.4H, <i>J</i> 17 Hz), 4.32–4.48 (m, 1H), 6.95–7.49 (m, 7H), 7.49–7.56 (m, 1.4H), 7.56–7.63 (m, 0.6H), 7.63–7.79 (m, 2H), 7.79–7.92 (m, 2H), 8.11–8.24 (m, 3H), 8.41–8.48 (m, 1H), 9.57 (brs, 1H), 10.73 (s, 0.4H), 10.78 (s, 0.6H). <sup>13</sup>C NMR (150.9 MHz, DMOS-<i>d</i><sub>6</sub>): δ (ppm) 22.32, 24.20, 28.31, 28.74, 28.98, 32.81, 34.95, 38.67, 40.08, 51.94, 52.80, 53.24, 55.96, 116.05 (TFA), 118.04 (TFA), 121.91, 122.34, 124.93, 125.51, 126.75, 127.27, 127.71, 128.75, 129.00, 129.71, 130.04, 130.38, 130.97, 131.63, 133.07, 133.80, 134.68, 135.77, 140.99, 156.84, 158.3 (q, <i>J</i> 32 Hz, carbonyl group of TFA), 163.70, 164.35, 165.78, 166.07, 168.05, 171.50. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for [C<sub>30</sub>H<sub>43</sub>N<sub>8</sub>O<sub>3</sub>]<sup>+</sup> 563.3453, found 563.3462. RP-HPLC (220 nm): 99% (<i>t</i><sub>R</sub> = 6.2 min, <i>k</i> = 7.1). C<sub>30</sub>H<sub>42</sub>N<sub>8</sub>O<sub>3</sub>·C<sub>6</sub>H<sub>3</sub>F<sub>9</sub>O<sub>6</sub> (562.72 + 342.07).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> (<i>S</i>)-2-Acetamido-5-guanidino-<i>N</i>-(4-(1-(2-oxo-2-(11-oxo-10,11-dihydro-5<i>H</i>-dibenzo[<i>b</i>,<i>e</i>][1,4]diazepin-5-yl)ethyl)piperidin-4-yl)butyl)pentanamide Bis(hydrotrifluoroacetate) (<b>56</b>)</h3><div class="NLM_p last">Acetyl anhydride (3.6 μL, 0.038 mmol) was added to a solution of <b>55</b> (25 mg, 0.032 mmol) and DIPEA (28 μL, 0.16 mmol) in DMF (0.3 mL), and the mixture was stirred at rt for 2 h. Isolation of the product by preparative HPLC (gradient, 0–25 min, 0.1% aq TFA/acetonitrile 81:19 to 72:28, <i>t</i><sub>R</sub> = 9 min) afforded <b>56</b> as a white fluffy solid (20.8 mg, 0.025 mmol, 78%). Ratio of configurational isomers evident in the NMR spectra: ∼1.5:1. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ (ppm) 1.13–1.26 (m, 4H), 1.30–1.50 (m, 8H), 1.57–1.66 (m, 1H), 1.69–1.82 (m, 2H), 1.83 (s, 3H), 2.85–2.96 (m, 1H), 2.96–3.14 (m, 5H), 3.28–3.35 (m, 1H), 3.57–3.61 (m, 1H), 3.76 (d, 0.6H, <i>J</i> 16 Hz), 3.91 (d, 0.4H, <i>J</i> 16 Hz), 4.13–4.21 (m, 1H), 4.33–4.48 (m, 1H), 7.03 (brs, 2H), 7.20–7.31 (m, 2H), 7.31–7.39 (m, 1.6H), 7.39–7.49 (m, 1.4H), 7.49–7.55 (m, 1.4H), 7.55–7.62 (m, 0.6H), 7.62–7.79 (m, 3H), 7.79–7.85 (m, 0.6H), 7.85–7.93 (m, 1.4H), 7.95–8.02 (m, 1H), 9.55 (brs, 1H), 10.73 (s, 0.4H), 10.78 (s, 0.6H). <sup>13</sup>C NMR (150.9 MHz, DMOS-<i>d</i><sub>6</sub>): δ (ppm) 22.49, 25.16, 28.69, 29.09, 29.26, 32.74, 34.90, 38.31, 40.33, 52.13, 52.65, 52.78, 53.23, 55.93, 116.05 (TFA), 118.10 (TFA), 121.88, 122.30, 124.89, 125.47, 126.74, 127.25, 127.70, 128.25, 128.70, 128.95, 129.69, 130.00, 130.36, 131.61, 133.03, 133.76, 134.68, 135.77, 140.97, 156.77, 158.2 (broad signal, carbonyl group of TFA), 163.67, 164.17, 165.73, 166.03, 169.19, 171.22. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for [C<sub>32</sub>H<sub>45</sub>N<sub>8</sub>O<sub>4</sub>]<sup>+</sup> 605.3558, found 605.3560. RP-HPLC (220 nm): 98% (<i>t</i><sub>R</sub> = 8.0 min, <i>k</i> = 9.5). C<sub>32</sub>H<sub>44</sub>N<sub>8</sub>O<sub>4</sub>·C<sub>4</sub>H<sub>2</sub>F<sub>6</sub>O<sub>4</sub> (604.76 + 228.05).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> <i>N</i>-(4-(1-(2-Oxo-2-(11-oxo-10,11-dihydro-5<i>H</i>-dibenzo[<i>b</i>,<i>e</i>][1,4]diazepin-5-yl)ethyl)piperidin-4-yl)butyl)propionamide Hydrotrifluoroacetate (<b>58</b>)</h3><div class="NLM_p last">DIPEA (12 μL, 0.071 mmol) and a solution of <b>57</b> (12 mg, 0.071 mmol) in DMF (30 μL) were added to a solution of <b>27</b> (28 mg, 0.071 mmol) in DMF (0.25 mL), and the mixture was stirred at rt for 30 min followed by the addition of 10% aq TFA (0.14 mL). Purification by preparative HPLC (gradient, 0–30 min, 0.1% aq TFA/acetonitrile 90:10 to 57:43, <i>t</i><sub>R</sub> = 18 min) yielded <b>58</b> as a white fluffy solid (33 mg, 0.057 mmol, 80%). Ratio of configurational isomers evident in the NMR spectra: ∼1.5:1. <sup>1</sup>H NMR (600 MHz, MeOH-<i>d</i><sub>4</sub>): δ (ppm) 1.11 (t, 3H, <i>J</i> 7.7 Hz), 1.27–1.37 (m, 4H), 1.38–1.56 (m, 5H), 1.85–2.00 (m, 2H), 2.17 (q, 2H, <i>J</i> 7.7 Hz), 2.87–2.96 (m, 1H), 2.99–3.08 (m, 1H), 3.15 (t, 2H, <i>J</i> 7.1 Hz), 3.39–3.49 (m, 1H), 3.67–3.83 (m, 2H), 4.39 (d, 0.6H, <i>J</i> 17 Hz), 4.43 (d, 0.4H, <i>J</i> 17 Hz), 7.23–7.31 (m, 0.8H), 7.31–7.36 (m, 1.2H), 7.36–7.41 (m, 0.4H), 7.44–7.55 (m, 2.2H), 7.59–7.71 (m, 2H), 7.73–7.78 (m, 0.4H), 7.90 (d, 0.6H, <i>J</i> 8.1 Hz), 7.97 (d, 0.4H, <i>J</i> 7.8 Hz). <sup>13</sup>C NMR (150.9 MHz, DMOS-<i>d</i><sub>6</sub>): δ (ppm) 10.59, 24.66, 30.22, 30.41, 30.47, 34.39, 36.31, 40.07, 54.95, 55.01, 55.37, 57.94, 58.04, 123.08, 123.64, 126.84, 127.51, 127.87, 128.49, 128.89, 129.45, 130.10, 130.54, 130.88, 131.20, 131.70, 131.97, 132.34, 133.01, 133.43, 134.57, 134.94, 135.45, 135.73, 137.04, 141.02, 142.68, 162.1 (broad signal, carbonyl group of TFA), 164.96, 165.45, 168.56, 168.81, 177.02. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for [C<sub>27</sub>H<sub>35</sub>N<sub>4</sub>O<sub>3</sub>]<sup>+</sup> 463.2709, found 463.2729. RP-HPLC (220 nm): 99% (<i>t</i><sub>R</sub> = 8.3 min, <i>k</i> = 8.0). C<sub>27</sub>H<sub>34</sub>N<sub>4</sub>O<sub>3</sub>·C<sub>2</sub>HF<sub>3</sub>O<sub>2</sub> (462.59 + 114.02).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Cell Culture</h3><div class="NLM_p last">CHO-K9 cells, stably transfected with the DNA of human muscarinic receptors M<sub>1</sub>–M<sub>5</sub> (obtained from Missouri S&T cDNA Resource Center; Rolla, MO) were cultured in HAM’s F12 medium supplemented with fetal calf serum (Biochrom, Berlin, Germany) (10%) and G418 (Biochrom) (750 μg/mL). HEK-293 cells were maintained in DMEM/HAM’s F12 (Thermo Scientific) supplemented with fetal bovine serum (Thermo Scientific) (10%) and penicillin/streptomycin (Thermo Scientific) (10 000 U/mL).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> IP1 Accumulation Assay</h3><div class="NLM_p last">The measurement of M<sub>2</sub>R stimulated activation of the G-protein mediated pathway was performed as previously described applying the IP-One HTRF assay (Cisbio, Codolet, France) according to the manufacturer’s protocol.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In brief, HEK-293 cells were grown to a confluency of approximately 70% and transiently co-transfected with the cDNAs of the human M<sub>2</sub> receptor (Missouri S&T cDNA Rescourse Center) and the hybrid G-protein Gα<sub>qi5-HA</sub>(Gα<sub>q</sub> protein with the last five amino acids at the C-terminus replaced by the corresponding sequence of Gα<sub>i</sub>; gift from the J. David Gladstone Institutes, San Francisco, CA),<a onclick="showRef(event, 'ref19 ref29'); return false;" href="javascript:void(0);" class="ref ref19 ref29">(19,29)</a> using TransIT-293 Mirus transfection reagent (MoBiTec, Goettingen, Germany). After 1 day, cells were detached from the culture dish with Versene (Life Technologies, Darmstadt, Germany), seeded into black 384-well plates (10 000 cells/well) (Greiner Bio-One, Frickenhausen, Germany) and maintained for 24 h at 37 °C. After incubation with the test compounds, dissolved in stimulation buffer, at 37 °C for 1 h the detection reagents were added (IP1-d2 conjugate and anti IP1cryptate TB conjugate, each dissolved in lysis buffer), and incubation was continued at rt for 60 min. Time resolved fluorescence resonance energy transfer (HTRF) was determined using a Clariostar plate reader (BMG, Ortenberg, Germany) measuring fluorescence at 620 (±10) nm and 670 (±10) nm (excitation at 330 nm). Antagonistic activities of <b>38</b> and <b>48</b> were determined after preincubation of the cells with <b>38</b> or <b>48</b> for 30 min, followed by addition of CCh (final concentration: 0.3 μM) and continued incubation at 37 °C for 1 h.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Radioligand Competition Binding</h3><div class="NLM_p last">Competition binding experiments with [<sup>3</sup>H]NMS (specific activity: 80 Ci/mmol; purchased from American Radiolabeled Chemicals (St. Louis, MO) via Hartman Analytics (Braunschweig, Germany) were performed at intact CHO-hM<sub><i>x</i></sub>R cells (<i>x</i> = 1–5) as described previously with minor modifications.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Cells were seeded in tissue culture treated white 96-well plates with clear bottom (Corning Incorporated Life Sciences, Tewksbury, MA; Corning catalog no. 3610) 1 or 2 days prior to the experiment. Depending on the level of receptor expression,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> the confluency of the cells was 30–50% (M<sub>1</sub>R), 50–70% (M<sub>3</sub>R, M<sub>4</sub>R, M<sub>5</sub>R), or 90–100% (M<sub>2</sub>R) on the day of the experiment. All experiments were performed at 22 ± 1 °C. The culture medium was removed by suction, the cells were washed with phosphate buffered saline (PBS) (200 μL) and covered with 160 μL of Leibovitz’s L15 medium (Life Technologies) supplemented with 1% bovine serum albumin (Serva, Heidelberg, Germany), in the following referred to as L15 medium. For total binding L15 medium (20 μL) and L15 medium (20 μL) containing [<sup>3</sup>H]NMS (10-fold concentrated) were added. To determine unspecific binding and the effect of a compound of interest on [<sup>3</sup>H]NMS equilibrium binding, L15 medium (20 μL) containing atropine or the compound of interest (10-fold concentrated) and L15 medium (20 μL) containing [<sup>3</sup>H]NMS (10-fold concentrated) were added. Samples were incubated for 3 h under gentle shaking. After incubation, the liquid was removed by suction, the cells were washed twice with ice-cold PBS (200 μL), and lysis solution (urea (8 M), acetic acid (3 M) and Triton-X-100 (1%) in water) (25 μL) was added. The plates were shaken for 30 min, liquid scintillator (Optiphase Supermix, PerkinElmer, Rodgau, Germany) (200 μL) was added, and the plates were sealed with a transparent sealing tape (permanent seal for microplates, PerkinElmer, product no. 1450-461). The plates were turned upside down several times in order to achieve complete mixing of scintillator and lysis solution. The samples were kept in the dark for at least 1 h prior to the measurement of radioactivity (dpm) with a MicroBeta2 plate counter (PerkinElmer). Applied concentrations of [<sup>3</sup>H]NMS were 0.2 nM (M<sub>1</sub>R, M<sub>2</sub>R, M<sub>3</sub>R), 0.1 nM (M<sub>4</sub>R), 0.3 nM (M<sub>5</sub>R), or 2 nM (M<sub>2</sub>R). Unspecific binding was determined in the presence of atropine (1000-fold excess to [<sup>3</sup>H]NMS) and amounted to <15% of total binding.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Investigation of the Stability of <b>38</b>, <b>48</b>, and <b>52</b> in Human Plasma</h3><div class="NLM_p">The metabolic stabilities of <b>38</b>, <b>48</b>, and <b>52</b> were investigated in human blood plasma/PBS pH 7.4 (1:2 v/v) (in the following referred to as plasma/PBS) at 37 °C. 1-Methyl-<span class="smallcaps smallerCapital">d</span>-tryptophan (<b>59</b>) was used as internal standard. As the purity (RP-HPLC, 220 nm) of <b>59</b> was <95% (data not shown), it was purified by preparative HPLC to give a purity of >99%. Plasma was obtained by the collection of human blood from a healthy donor in 5.5 mL heparinized plasma-monovettes, followed by centrifugation at 1200<i>g</i> at 4 °C for 10 min. The supernatants were pooled in two 50 mL Falcon tubes and centrifuged again at 1200<i>g</i> at 4 °C for 10 min. The plasma was aliquoted and stored at −80 °C.</div><div class="NLM_p">For the addition of the compounds to plasma/PBS, 5 mM stock solutions in acetonitrile/0.04% aqueous TFA (3:7 v/v) were used. Recoveries were determined for concentrations of 80 μM and 4 μM and an internal standard concentration of 10 μM. For this purpose, <b>59</b> and the DIBA–peptide conjugates were added to plasma/PBS (total volume: 70 μL), immediately followed by vortexing (∼10 s) and precipitation of protein by the addition of EtOH/acetonitrile (1:1 v/v) (140 μL). The mixture was vortexed for 5 min and centrifuged at 16 100<i>g</i> at 4 °C for 10 min. Aliquots (180 μL) of the supernatant were transferred into 1.5 mL polypropylene reaction vessels containing 10% aqueous TFA (5 μL). The volatiles were removed in a vacuum concentrator under reduced pressure at 40 °C (∼60 min), and the residue was taken up in acetonitrile/0.04% aqueous TFA (1:9 v/v) (90 μL) by vortexing (2 min). The samples were filtered using a 0.2 μm RC-membrane filter (Phenomenex, Aschaffenburg, Germany) and analyzed by RP-HPLC using the analytical HPLC system and the conditions as described under general experimental conditions but applying the following gradient: 0–12 min, A/B 90:10 to 70:30; 12–16 min, 70:30 to 5:95; 16–20 min, 5:95. On the day of an experiment four-point calibrations were performed for the respective DIBA–peptide conjugates and the internal standard. Peak areas representing 100% recovery were obtained by analyzing 53.3 μM and 2.67 μM “DIBA–peptide” solutions as well as a 6.67 μM solution of <b>59</b> (in duplicate each). All peak areas were transformed into concentrations (μM), and percent recoveries of <b>38</b>, <b>48</b>, <b>52</b>, and <b>59</b> were calculated based on the average values of the 100% reference samples (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Table S2, Supporting Information</a>). Recovery ratios were obtained by dividing the recovery of the DIBA–peptide conjugate by the recovery of <b>59</b> for each individual sample (<i>n</i> = 3 or 4, cf. <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Table S2, Supporting Information</a>).</div><div class="NLM_p">For the investigation of the stabilities in plasma, the DIBA–peptide conjugates (<b>38</b>, <b>48</b>, <b>52</b>) and <b>59</b> were added to plasma/PBS at a concentration of 100 μM and 10 μM, respectively (in triplicate each). After 1, 6, 24, and 48 h of incubation under shaking at 37 °C, aliquots (70 μL) were taken and processed as described above for the determination of recoveries. On the day of an experiment four-point calibrations were performed for the respective DIBA–peptide conjugates and <b>59</b>. Peak areas representing 100% recovery were obtained by analyzing 66.7 μM “DIBA–peptide” solutions and a 6.67 μM solution of <b>59</b> (in duplicate). On the basis of the calibration, the peak areas of the 100% references and of the samples were transformed into concentrations (μM). Recoveries of <b>38</b>, <b>48</b>, and <b>52</b> were calculated by multiplying the recovery of <b>59</b>, obtained for each individual sample, with the recovery ratio obtained for the concentration of 80 μM (see above). The plasma concentrations of the DIBA–peptide conjugates were obtained by dividing the determined “DIBA–peptide” concentration by the respective recovery. Degradation (%) of <b>38</b>, <b>48</b>, and <b>52</b> was calculated based on the average values of the 100% reference samples.</div><div class="NLM_p last">Note: Recovery ratios obtained for a concentration of 4 μM were not used as <b>38</b>, <b>48</b>, and <b>52</b> exhibited high stabilities (lowest concentration in plasma of ∼80 μM (<b>52</b>, 48 h)).</div></div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Molecular Dynamics Simulations. MR Models</h3><div class="NLM_p last">In the case of the hM<sub>2</sub>R, a described model derived from the M<sub>2</sub>R crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON">3UON</a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>) was used.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Models of the hM<sub>1</sub>R, hM<sub>3</sub>R, and hM<sub>4</sub>R were obtained by refinement of reported crystal structures (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CXV">5CXV</a>,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DAJ">4DAJ</a>,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DSG">5DSG</a>,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> respectively) using Modeller 9.18.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30−32)</a> The homology model of the hM<sub>5</sub>R was prepared based on the M<sub>2</sub>R crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON">3UON</a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>) using Modeller 9.18. Replacement of ICL3 of the hM<sub>1</sub>R, hM<sub>3</sub>R, hM<sub>4</sub>R, and hM<sub>5</sub>R by eight alanines was performed using the respective coordinates of the described hM<sub>2</sub>R model.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The receptor models (M<sub>1</sub>R, M<sub>2</sub>R, M<sub>3</sub>R, M<sub>4</sub>R, M<sub>5</sub>R) contained a disulfide bridge between Cys<sup>3.25</sup> and Cys<sup>ECL2.50</sup> as well as between the two cysteines in ECL3. Coordinates other than ligand and receptor molecules were removed. The Protein Preparation Wizard (Schrödinger LLC, Portland, OR, USA) was used to further refine the receptor models: The N- and C-termini were capped by the introduction of acetyl and methylamide groups, respectively, and amino acid side chains containing hydrogen bond donors and acceptors were optimized for hydrogen bonding. Histidine residues were simulated in the uncharged form as the N<sup>ε</sup>–H tautomer. Residues other than histidine were simulated in their dominant protonation state at pH 7. The above-mentioned disulfide bonds were maintained and a sodium ion was placed next to D<sup>2.50</sup>.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Induced-Fit Docking</h3><div class="NLM_p last">For induced-fit docking, performed to find the initial ligand binding poses for MD simulations, ligand (<b>38</b>, <b>48</b>) geometries were energetically optimized using the LigPrep module (Schrödinger LLC). Amine (piperidine, piperazine (singly protonated), Lys) and guanidine (Arg) groups in <b>38</b> and <b>48</b> were protonated resulting in net charges of +4 and +3, respectively. For initial docking, two strategies were pursued: Y<sup>3.33</sup>, Y<sup>6.51</sup>, and Y<sup>7.39</sup> were temporarily mutated to alanine (Gridgen_Recep_Vscale = 0.7), or mutations were omitted (Gridgen_Recep_Vscale = 0.5). Ligands were docked within a box of 46 × 46 × 46 Å<sup>3</sup> around the crystallographic binding poses of the cocrystallized ligands. Re-docking was performed in the extended precision mode. From the obtained induced-fit docking results of each MR subtype/ligand combination, one pose (ligand–receptor complex) was selected as input structure for the MD simulation based on low/minimal MM-GBSA scores (Schrödinger LLC, implicit membrane model) and reasonability of the ligand binding pose.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> MD Simulations</h3><div class="NLM_p">The selected ligand–receptor complexes were aligned to the aforementioned crystal structure entries in the orientations of proteins in membranes (OPM) database<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> using the protein structure alignment tool (Schrödinger LLC). Ligand–receptor complexes were inserted into a hydrated palmitoyloleoylphosphatidylcholine (POPC) bilayer (comprising 160 POPC molecules) as described for <b>7</b>.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The systems contained about 61 000–68 000 atoms and the initial box sizes were approximately 82 × 82 × (99–110) Å<sup>3</sup>. The parameters (geometry, partial charges) of the protein structure, ligands (<b>38</b>, <b>48</b>), lipids, inorganic ions, and water were assigned as reported.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> MD simulations were executed on Nvidia GTX 1080(Ti) GPUs using the CUDA version of PMEMD,<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> implemented in AMBER18 (AMBER 2018, University of California, San Francisco, CA, USA). After minimization, the systems were heated from 0 to 100 K in the <i>NVT</i> ensemble during 20 ps and from 100 to 310 K in the <i>NPT</i> ensemble during 100 ps, applying harmonic restraints of 5 kcal·mol<sup>–1</sup>·Å<sup>–1</sup> to non-hydrogen atoms of protein, lipids, and ligand. The temperature and pressure coupling parameters used for the equilibration at 310 K (<i>NPT</i> ensemble) were the same as described for MD simulations of the hM<sub>2</sub>R bound to <b>7</b>.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> During the 10 ns equilibration period, harmonic restraints on receptor and ligand non-hydrogen atoms were reduced stepwise (0.5 kcal·mol<sup>–1</sup>·Å<sup>–1</sup> every 0.5 ns) to 2.5 kcal·mol<sup>–1</sup>·Å<sup>–1</sup> within 3 ns. While removing restraints on ligand atoms, harmonic restraints on receptor mainchain atoms were further reduced stepwise (0.5 kcal·mol<sup>–1</sup>·Å<sup>–1</sup> every 0.5 ns) to 0.5 kcal·mol<sup>–1</sup>·Å<sup>–1</sup> from 3 to 5 ns. After 5 ns, harmonic restraints on receptor mainchain atoms were removed; i.e., the residual equilibration period (5 ns) was run without restraints. The interaction cutoff was set to 9.0 Å, and long-range electrostatics were computed using the particle mesh Ewald (PME) method.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> To enable a frame step size of 4 fs, bonds involving hydrogen atoms were constrained using SHAKE<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and hydrogen mass repartitioning (HMR)<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> was applied. The final frame of the equilibration period was used as input for the simulations over 20.25 μs. After the equilibration period of 10 ns, the Berendsen barostat was replaced by the Monte Carlo barostat algorithm.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div class="NLM_p last">Data were collected every 500 ps. For analyses, the first 250 ns of the simulations were omitted and every twentieth frame (10 ns step size) was considered. To determine the frame corresponding to the lowest binding free energy, MM-PBSA calculations of the MD simulation (20 μs) trajectories were performed using AMBER18. Ligand–receptor interactions were analyzed using PLIP 1.4.2.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> For time course illustrations, frames were collected every 1 μs (20 frames). Figures showing molecular structures (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figures S8–S15</a>) were generated with PyMOL Molecular Graphics system, version 2.2.0 (Schrödinger LLC).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Data Processing</h3><div class="NLM_p last">Specific optical rotations were calculated according to [α]<sub class="stack">D</sub><sup class="stack">20</sup> = α/(<i>c</i><i>l</i>) (α = measured angle of rotation [deg], <i>c</i> = concentration [g cm<sup>–3</sup>], <i>l</i> = length of the cuvette [dm]). Retention (capacity) factors were calculated from retention times (<i>t</i><sub>R</sub>) according to <i>k</i> = (<i>t</i><sub>R</sub> – <i>t</i><sub>0</sub>)/<i>t</i><sub>0</sub> (<i>t</i><sub>0</sub> = dead time). Data from radioligand ([<sup>3</sup>H]NMS) competition binding assays (except for <b>48</b>, M<sub>3</sub>R) and from the IP1 accumulation assay were processed as reported previously, but pIC<sub>50</sub> values obtained from the IP1 assay (antagonist mode) were not converted to p<i>K</i><sub>b</sub> values.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In the case of competition binding studies with [<sup>3</sup>H]NMS and <b>48</b> at the M<sub>3</sub>R, pIC<sub>50</sub> values were obtained from a linear Hill plot (log(<i>B</i>/(<i>B</i><sub>0</sub> – <i>B</i>)) plotted against log(concentration of <b>48</b>); <i>B</i> = specifically bound [<sup>3</sup>H]NMS in the presence of <b>48</b>, <i>B</i><sub>0</sub> = specifically bound [<sup>3</sup>H]NMS in the absence of <b>48</b>; log(concentration of <b>48</b>) at “log(<i>B</i>/(<i>B</i><sub>0</sub> – <i>B</i>)) = 0” equals the pIC<sub>50</sub>). <i>K</i><sub>i</sub> values for the calculation of relative MR affinities (M<sub>2</sub>R selectivity, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) were obtained by transforming pIC<sub>50</sub> values from individual competition binding experiments with [<sup>3</sup>H]NMS to IC<sub>50</sub> values followed by conversion to <i>K</i><sub>i</sub> values according to the Cheng–Prusoff equation<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and calculation of mean <i>K</i><sub>i</sub> values.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01967" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82677" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82677" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01967" class="ext-link">10.1021/acs.jmedchem.8b01967</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Analytical data of compounds <b>28</b>–<b>53</b>; Figures S1–S8; MD simulations with the M<sub>1</sub>R, M<sub>2</sub>R, M<sub>4</sub>R, and M<sub>5</sub>R bound to <b>38</b> and with the M<sub>1</sub>R–M<sub>5</sub>R bound to <b>48</b> (Figures S9–S15); Tables S1 and S2; RP-HPLC chromatograms of <b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, and <b>58</b>; <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, and <b>58</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and muscarinic receptor binding data (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Coordinates of the homology model of the hM<sub>5</sub>R, which was prepared based on a reported crystal structure of the M<sub>2</sub>R (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON">3UON</a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf">jm8b01967_si_001.pdf (4.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_002.csv">jm8b01967_si_002.csv (4.22 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_003.pdb">jm8b01967_si_003.pdb (181.78 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01967" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61723" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61723" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Max Keller</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8095-8627" title="Orcid link">http://orcid.org/0000-0002-8095-8627</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dcb1bda4f2b7b9b0b0b9ae9ca9aef2b8b9"><span class="__cf_email__" data-cfemail="721f130a5c19171e1e17003207005c1617">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Pegoli</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</span>; 
    <span>Present Address:
                        A.P.: Ramboll Environment & Health GmbH, Werinherstraße 79, D-81541 Munich, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Wifling</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Corinna G. Gruber</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xueke She</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</span>; 
    <span>Present Address:
                        X.S.: Moffitt Cancer Center, 12902 USF Magnolia Drive, Tampa, FL, U.S</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harald Hübner</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Pharmacy, Medicinal Chemistry, Friedrich Alexander University, Nikolaus-Fiebiger-Straße 10, D-91058 Erlangen, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Günther Bernhardt</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacy, Faculty of Chemistry and Pharmacy, University of Regensburg, Universitätsstraße 31, D-93053 Regensburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6491-9874" title="Orcid link">http://orcid.org/0000-0001-6491-9874</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Gmeiner</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Pharmacy, Medicinal Chemistry, Friedrich Alexander University, Nikolaus-Fiebiger-Straße 10, D-91058 Erlangen, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4127-197X" title="Orcid link">http://orcid.org/0000-0002-4127-197X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.P., D.W., and C.G.G. contributed equally. A.P., C.G.G., M.K., and X.S. performed the syntheses and the analytical characterization of chemical compounds. A.P., C.G.G., and M.K. performed radioligand competition binding experiments and analyzed the data. H.H. and P.G. performed IP1 accumulation assays and analyzed the data. C.G.G. investigated plasma stabilities. D.W. performed molecular docking and MD simulations and processed the data. D.W., A.P., and M.K. designed the MR ligands. M.K. initiated and planned the project. M.K. and G.B. supervised the research. M.K., A.P., D.W., G.B., and C.G.G. wrote the manuscript. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Authors will release the atomic coordinates upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Susanne Bollwein, Brigitte Wenzl, Lisa Schindler, and Maria Beer-Krön for excellent technical assistance, Armin Buschauer for providing laboratory equipment and for helpful discussions, and the Leibniz Rechenzentrum (LRZ) in Munich for providing software (Schrödinger Suite) and computing resources. This work was funded by the Graduate Training Program (Graduiertenkolleg) Grant GRK1910 of the Deutsche Forschungsgemeinschaft and by the China Scholarship Council.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CCh</td><td class="NLM_def"><p class="first last">carbachol</p></td></tr><tr><td class="NLM_term">CHO</td><td class="NLM_def"><p class="first last">Chinese hamster ovary</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">HBTU</td><td class="NLM_def"><p class="first last"><i>O</i>-(1<i>H-</i>benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HFIP</td><td class="NLM_def"><p class="first last">1,1,1,3,3,3-hexafluoro-2-propanol</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole hydrate</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">IP1</td><td class="NLM_def"><p class="first last">inositol monophosphate</p></td></tr><tr><td class="NLM_term"><i>k</i></td><td class="NLM_def"><p class="first last">retention (or capacity) factor (HPLC)</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>d</sub></td><td class="NLM_def"><p class="first last">dissociation constant obtained from a saturation binding experiment</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>i</sub></td><td class="NLM_def"><p class="first last">dissociation constant obtained from a competition binding experiment</p></td></tr><tr><td class="NLM_term">MR</td><td class="NLM_def"><p class="first last">muscarinic receptor</p></td></tr><tr><td class="NLM_term">M<sub><i>x</i></sub>R</td><td class="NLM_def"><p class="first last">muscarinic receptor subtype <i>x</i></p></td></tr><tr><td class="NLM_term">NMS</td><td class="NLM_def"><p class="first last"><i>N</i>-methylscopolamine</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate buffered saline</p></td></tr><tr><td class="NLM_term">PS</td><td class="NLM_def"><p class="first last">polystyrene</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>R</sub></td><td class="NLM_def"><p class="first last">retention time</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span> <span> </span><span class="NLM_article-title">Overview of muscarinic receptor subtypes</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1007/978-3-642-23274-9_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1007%2F978-3-642-23274-9_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChsb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2012&pages=3-28&author=R.+M.+Eglen&title=Overview+of+muscarinic+receptor+subtypes&doi=10.1007%2F978-3-642-23274-9_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of muscarinic receptor subtypes</span></div><div class="casAuthors">Eglen, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue">Muscarinic Receptors</span>),
    <span class="NLM_cas:pages">3-28</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The physiol. role of muscarinic receptors is highly complex and, although not completely understood, has become clearer over the last decade.  Recent pharmacol. evidence with novel compds., together with data from transgenic mice, suggests that all five subtypes have defined functions in the nervous system as well as mediating the non neuronal, hormonal actions of acetylcholine.  Numerous novel agonists, allosteric regulators, and antagonists have now been identified with authentic subtype specificity in vitro and in vivo.  These compds. provide addnl. pharmacol. opportunities for selective subtype modulation as well as a new generation of muscarinic receptor-based therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGsQ_5yP8Tg7Vg90H21EOLACvtfcHk0lh4ZEEOt_PuJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChsb7K&md5=e9acb43ac95866f476329d7f0b119c33</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-23274-9_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-642-23274-9_1%26sid%3Dliteratum%253Aachs%26aulast%3DEglen%26aufirst%3DR.%2BM.%26atitle%3DOverview%2520of%2520muscarinic%2520receptor%2520subtypes%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2012%26volume%3D208%26spage%3D3%26epage%3D28%26doi%3D10.1007%2F978-3-642-23274-9_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Allosteric modulation of muscarinic acetylcholine receptors</span>. <i>Curr. Neuropharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.2174/157015907781695946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.2174%2F157015907781695946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=19305798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKqsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=157-167&author=K.+J.+Gregoryauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=Allosteric+modulation+of+muscarinic+acetylcholine+receptors&doi=10.2174%2F157015907781695946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of muscarinic acetylcholine receptors</span></div><div class="casAuthors">Gregory, Karen J.; Sexton, Patrick M.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Current Neuropharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">CNUEAN</span>;
        ISSN:<span class="NLM_cas:issn">1570-159X</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Muscarinic acetylcholine receptors (mAChRs) are prototypical Family A G protein coupled-receptors.  The five mAChR subtypes are widespread throughout the periphery and the central nervous system and, accordingly, are widely involved in a variety of both physiol. and pathophysiol. processes.  There currently remains an unmet need for better therapeutic agents that can selectively target a given mAChR subtype to the relative exclusion of others.  The main reason for the lack of such selective mAChR ligands is the high sequence homol. within the acetylcholine-binding site (orthosteric site) across all mAChRs.  However, the mAChRs possess at least one, and likely two, extracellular allosteric binding sites that can recognize small mol. allosteric modulators to regulate the binding and function of orthosteric ligands.  Extensive studies of prototypical mAChR modulators, such as gallamine and alcuronium, have provided strong pharmacol. evidence, and assocd. structure-activity relationships (SAR), for a "common" allosteric site on all five mAChRs.  These studies are also supported by mutagenesis expts. implicating the second extracellular loop and the interface between the third extracellular loop and the top of transmembrane domain 7 as contributing to the common allosteric site.  Other studies are also delineating the pharmacol. of a second allosteric site, recognized by compds. such as staurosporine.  In addn., allosteric agonists, such as McN-A-343, AC-42 and N-desmethylclozapine, have also been identified.  Current challenges to the field include the ability to effectively detect and validate allosteric mechanisms, and to quantify allosteric effects on binding affinity and signaling efficacy to inform allosteric modulator SAR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMjqjcw6KlILVg90H21EOLACvtfcHk0lg6z5JXbqgyRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKqsLnP&md5=a30c0102440ffbdc1fc6efe9da084521</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F157015907781695946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157015907781695946%26sid%3Dliteratum%253Aachs%26aulast%3DGregory%26aufirst%3DK.%2BJ.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAllosteric%2520modulation%2520of%2520muscarinic%2520acetylcholine%2520receptors%26jtitle%3DCurr.%2520Neuropharmacol.%26date%3D2007%26volume%3D5%26spage%3D157%26epage%3D167%26doi%3D10.2174%2F157015907781695946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawaratne, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bures, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachhawat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of the M<sub>1</sub> and M<sub>4</sub> muscarinic acetylcholine receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>531</i></span>,  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">340</span>, <span class="refDoi"> DOI: 10.1038/nature17188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature17188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=26958838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlGjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=531&publication_year=2016&pages=335-340&author=D.+M.+Thalauthor=B.+Sunauthor=D.+Fengauthor=V.+Nawaratneauthor=K.+Leachauthor=C.+C.+Felderauthor=M.+G.+Buresauthor=D.+A.+Evansauthor=W.+I.+Weisauthor=P.+Bachhawatauthor=T.+S.+Kobilkaauthor=P.+M.+Sextonauthor=B.+K.+Kobilkaauthor=A.+Christopoulos&title=Crystal+structures+of+the+M1+and+M4+muscarinic+acetylcholine+receptors&doi=10.1038%2Fnature17188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the M1 and M4 muscarinic acetylcholine receptors</span></div><div class="casAuthors">Thal, David M.; Sun, Bingfa; Feng, Dan; Nawaratne, Vindhya; Leach, Katie; Felder, Christian C.; Bures, Mark G.; Evans, David A.; Weis, William I.; Bachhawat, Priti; Kobilka, Tong Sun; Sexton, Patrick M.; Kobilka, Brian K.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">531</span>
        (<span class="NLM_cas:issue">7594</span>),
    <span class="NLM_cas:pages">335-340</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Muscarinic M1-M5 acetylcholine receptors are G protein-coupled receptors (GPCRs) that regulate many vital functions of the central and peripheral nervous systems.  In particular, the M1 and M4 receptor subtypes have emerged as attractive drug targets for treatments of neurol. disorders, such as Alzheimer's disease and schizophrenia, but the high conservation of the acetylcholine-binding pocket has spurred current research into targeting allosteric sites on these receptors.  Here we report the crystal structures of the M1 and M4 muscarinic receptors bound to the inverse agonist, tiotropium.  Comparison of these structures with each other, as well as with the previously reported M2 and M3 receptor structures, reveals differences in the orthosteric and allosteric binding sites that contribute to a role in drug selectivity at this important receptor family.  We also report identification of a cluster of residues that form a network linking the orthosteric and allosteric sites of the M4 receptor, which provides new insight into how allosteric modulation may be transmitted between the two spatially distinct domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3oy2Ml852CrVg90H21EOLACvtfcHk0lg6z5JXbqgyRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlGjurg%253D&md5=8ed3f453737ad0291cf5b2c088f528da</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature17188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17188%26sid%3Dliteratum%253Aachs%26aulast%3DThal%26aufirst%3DD.%2BM.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DNawaratne%26aufirst%3DV.%26aulast%3DLeach%26aufirst%3DK.%26aulast%3DFelder%26aufirst%3DC.%2BC.%26aulast%3DBures%26aufirst%3DM.%2BG.%26aulast%3DEvans%26aufirst%3DD.%2BA.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DBachhawat%26aufirst%3DP.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DCrystal%2520structures%2520of%2520the%2520M1%2520and%2520M4%2520muscarinic%2520acetylcholine%2520receptors%26jtitle%3DNature%26date%3D2016%26volume%3D531%26spage%3D335%26epage%3D340%26doi%3D10.1038%2Fnature17188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choppin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, N.</span></span> <span> </span><span class="NLM_article-title">Therapeutic opportunities from muscarinic receptor research</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.1016/S0165-6147(00)01737-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2FS0165-6147%2800%2901737-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=11479003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsFyksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=409-414&author=R.+M.+Eglenauthor=A.+Choppinauthor=N.+Watson&title=Therapeutic+opportunities+from+muscarinic+receptor+research&doi=10.1016%2FS0165-6147%2800%2901737-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic opportunities from muscarinic receptor research</span></div><div class="casAuthors">Eglen, R. M.; Choppin, A.; Watson, N.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">409-414</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Muscarinic acetylcholine receptor subtypes have been the subjects of research for at least a quarter of a century.  Nonetheless, there are few selective muscarinic receptor ligands presently used as therapeutics.  The extensive development of muscarinic M1 receptor agonists for the treatment of cognitive dysfunction has culminated in a series of unsuccessful drug candidates, which reflects a lack of understanding of the disease and the role played by muscarinic cholinergic transmission.  Paradoxically, the most successful antagonist approved for use in urinary incontinence is the nonselective muscarinic receptor antagonist tolterodine.  This deficit in subtype-selective ligands could be circumvented by the development of transgenic mice, each lacking functional M1, M2, M3, M4 or M5 receptors.  In this article, the current status of muscarinic receptor research is critically assessed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZkhW-d5bhbVg90H21EOLACvtfcHk0lg6z5JXbqgyRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsFyksLw%253D&md5=ca0c63af01d912edc947a0f8d106656d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0165-6147%2800%2901737-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0165-6147%252800%252901737-5%26sid%3Dliteratum%253Aachs%26aulast%3DEglen%26aufirst%3DR.%2BM.%26aulast%3DChoppin%26aufirst%3DA.%26aulast%3DWatson%26aufirst%3DN.%26atitle%3DTherapeutic%2520opportunities%2520from%2520muscarinic%2520receptor%2520research%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2001%26volume%3D22%26spage%3D409%26epage%3D414%26doi%3D10.1016%2FS0165-6147%2800%2901737-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheardown, M. J.</span></span> <span> </span><span class="NLM_article-title">Muscarinic M<sub>1</sub> receptor agonists and M<sub>2</sub> receptor antagonists as therapeutic targets in Alzheimer’s disease</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">870</span>, <span class="refDoi"> DOI: 10.1517/13543776.12.6.863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1517%2F13543776.12.6.863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD38XkvVWlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=863-870&author=M.+J.+Sheardown&title=Muscarinic+M1+receptor+agonists+and+M2+receptor+antagonists+as+therapeutic+targets+in+Alzheimer%E2%80%99s+disease&doi=10.1517%2F13543776.12.6.863"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic M1 receptor agonists and M2 receptor antagonists as therapeutic targets in Alzheimer's disease</span></div><div class="casAuthors">Sheardown, Malcolm J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">863-870</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease is the most common form of senile dementia and is of increasing medical and economic importance.  One of the earliest findings in the systematic study of Alzheimer's disease was the progressive loss of acetylcholinergic neurons.  This lead to the so-called cholinergic hypothesis of Alzheimer's disease and provoked a search for cholinergic replacement therapies.  This has resulted in compds. capable of inhibiting the catabolism of acetylcholine (acetylcholinesterase inhibitors) being licensed for the treatment of Alzheimer's disease.  However, the search for directly acting, selective, muscarinic M1 receptor agonists and M2 receptor antagonists, that are capable of increasing acetylcholine levels in the neocortex, has not yet proved successful.  For the muscarinic agonists this failure has been largely due to the difficulty in finding genuinely selective compds.  As a result, there has been a redn. in the no. published patents in this area.  This review discusses the rationale, progress and the possible future for these two potential approaches to the therapy of Alzheimer's disease esp. in light of some recent patents contg. selective compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp7DYdk9clZLVg90H21EOLACvtfcHk0lgMIsgTzPpIlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XkvVWlsLc%253D&md5=7b55f8667befd78dcb877b6cc3c55a7f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1517%2F13543776.12.6.863&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.12.6.863%26sid%3Dliteratum%253Aachs%26aulast%3DSheardown%26aufirst%3DM.%2BJ.%26atitle%3DMuscarinic%2520M1%2520receptor%2520agonists%2520and%2520M2%2520receptor%2520antagonists%2520as%2520therapeutic%2520targets%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2002%26volume%3D12%26spage%3D863%26epage%3D870%26doi%3D10.1517%2F13543776.12.6.863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clader, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Muscarinic receptor agonists and antagonists in the treatment of Alzheimer’s disease</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3353</span>– <span class="NLM_lpage">3361</span>, <span class="refDoi"> DOI: 10.2174/138161205774370762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.2174%2F138161205774370762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=16250841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWrs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=3353-3361&author=J.+W.+Claderauthor=Y.+Wang&title=Muscarinic+receptor+agonists+and+antagonists+in+the+treatment+of+Alzheimer%E2%80%99s+disease&doi=10.2174%2F138161205774370762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic receptor agonists and antagonists in the treatment of Alzheimer's disease</span></div><div class="casAuthors">Clader, John W.; Wang, Yuguang</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3353-3361</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  One of the consistent findings in the brains of Alzheimer's Disease (AD) patients is loss of cholinergic function.  The cholinergic approach to treatment of AD involves counteracting this loss in cholinergic activity by pharmacol. intervention to increase cholinergic transmission.  The cognitive effects of acetylcholine are mediated via the muscarinic M1 receptor.  Direct stimulation of this receptor using muscarinic M1 agonists improves cognition in animal models and improves performance in cognitive tests in Alzheimer's patients.  Alternatively, antagonists of central presynaptic M2 receptors improves cognition by increasing the central release of acetylcholine.  Both approaches require high selectivity for one muscarinic receptor sub-type both for efficacy and to avoid cholinergic side effects.  In this review summarizes recent progress in the identification and characterization of selective muscarinic receptor ligands for the treatment of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2_wdSDisNRLVg90H21EOLACvtfcHk0lgMIsgTzPpIlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWrs7vP&md5=d0d18c0299921abfbf6b815b223e743a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F138161205774370762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161205774370762%26sid%3Dliteratum%253Aachs%26aulast%3DClader%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DMuscarinic%2520receptor%2520agonists%2520and%2520antagonists%2520in%2520the%2520treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2005%26volume%3D11%26spage%3D3353%26epage%3D3361%26doi%3D10.2174%2F138161205774370762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Citron, M.</span></span> <span> </span><span class="NLM_article-title">Alzheimer’s disease: strategies for disease modification</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1038/nrd2896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnrd2896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=20431570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=387-398&author=M.+Citron&title=Alzheimer%E2%80%99s+disease%3A+strategies+for+disease+modification&doi=10.1038%2Fnrd2896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease: strategies for disease modification</span></div><div class="casAuthors">Citron, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">387-398</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease is the largest unmet medical need in neurol.  Current drugs improve symptoms, but do not have profound disease-modifying effects.  However, in recent years, several approaches aimed at inhibiting disease progression have advanced to clin. trials.  Among these, strategies targeting the prodn. and clearance of the amyloid-β peptide - a cardinal feature of Alzheimer's disease that is thought to be important in disease pathogenesis - are the most advanced.  Approaches aimed at modulating the abnormal aggregation of tau filaments (another key feature of the disease), and those targeting metabolic dysfunction, are also being evaluated in the clinic.  This article discusses recent progress with each of these strategies, with a focus on anti-amyloid strategies, highlighting the lessons learned and the challenges that remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeM0fsdJQ-z7Vg90H21EOLACvtfcHk0lgMIsgTzPpIlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWjsrw%253D&md5=281ba27137d2a1fa173d5dde1dcbe772</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2896%26sid%3Dliteratum%253Aachs%26aulast%3DCitron%26aufirst%3DM.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%253A%2520strategies%2520for%2520disease%2520modification%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D387%26epage%3D398%26doi%3D10.1038%2Fnrd2896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clader, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billard, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binch, H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachowicz, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCombie, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vice, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenlee, W. J.</span></span> <span> </span><span class="NLM_article-title">Muscarinic M<sub>2</sub> antagonists: anthranilamide derivatives with exceptional selectivity and in vivo activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2003.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2Fj.bmc.2003.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=14723952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVOksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=319-326&author=J.+W.+Claderauthor=W.+Billardauthor=H.+Binchauthor=L.+Y.+Chenauthor=G.+Crosbyauthor=R.+A.+Duffyauthor=J.+Fordauthor=J.+A.+Kozlowskiauthor=J.+E.+Lachowiczauthor=S.+Liauthor=C.+Liuauthor=S.+W.+McCombieauthor=S.+Viceauthor=G.+Zhouauthor=W.+J.+Greenlee&title=Muscarinic+M2+antagonists%3A+anthranilamide+derivatives+with+exceptional+selectivity+and+in+vivo+activity&doi=10.1016%2Fj.bmc.2003.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic M2 antagonists: anthranilamide derivatives with exceptional selectivity and in vivo activity</span></div><div class="casAuthors">Clader, John W.; Billard, William; Binch, Herbert; Chen, Lian-Yong; Crosby, Gordon; Duffy, Ruth A.; Ford, Jennifer; Kozlowski, Joseph A.; Lachowicz, Jean E.; Li, Shengjian; Liu, Charles; McCombie, Stuart W.; Vice, Susan; Zhou, Gowei; Greenlee, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">319-326</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) assocs. with the consisting findings in brains of AD patients of a loss of cholinergic markers, including levels of acetylcholine.  The cholinergic approach to treatment of AD involves counteracting this loss in cholinergic activity by pharmacol. intervention to increase cholinergic transmission.  While the use of acetylcholinesterase inhibitors remains the most common approach to cholinergic therapy, an alternative involves inhibition of central presynaptic M2 receptors, thus causing an increase in acetylcholine release.  The authors present anthranilamide analogs of formula (I) and (II) that are potent and highly selective muscarinic M2 antagonists that also show good oral bioavailability and in vivo activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHDdxde5nOVbVg90H21EOLACvtfcHk0lgTjXLVYACLXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVOksA%253D%253D&md5=9b696ecafbf680510d2ab78fe6829a8f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2003.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2003.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DClader%26aufirst%3DJ.%2BW.%26aulast%3DBillard%26aufirst%3DW.%26aulast%3DBinch%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DL.%2BY.%26aulast%3DCrosby%26aufirst%3DG.%26aulast%3DDuffy%26aufirst%3DR.%2BA.%26aulast%3DFord%26aufirst%3DJ.%26aulast%3DKozlowski%26aufirst%3DJ.%2BA.%26aulast%3DLachowicz%26aufirst%3DJ.%2BE.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DMcCombie%26aufirst%3DS.%2BW.%26aulast%3DVice%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26atitle%3DMuscarinic%2520M2%2520antagonists%253A%2520anthranilamide%2520derivatives%2520with%2520exceptional%2520selectivity%2520and%2520in%2520vivo%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D319%26epage%3D326%26doi%3D10.1016%2Fj.bmc.2003.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dörje, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrecht, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutschler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brann, M.</span></span> <span> </span><span class="NLM_article-title">Antagonist binding profiles of five cloned human muscarinic receptor subtypes</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>256</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">733</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1994002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A280%3ADyaK3M7jt1CrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=256&publication_year=1991&pages=727-733&author=F.+D%C3%B6rjeauthor=J.+Wessauthor=G.+Lambrechtauthor=R.+Tackeauthor=E.+Mutschlerauthor=M.+Brann&title=Antagonist+binding+profiles+of+five+cloned+human+muscarinic+receptor+subtypes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Antagonist binding profiles of five cloned human muscarinic receptor subtypes</span></div><div class="casAuthors">Dorje F; Wess J; Lambrecht G; Tacke R; Mutschler E; Brann M R</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of pharmacology and experimental therapeutics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">256</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">727-33</span>
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">A variety of muscarinic antagonists are currently used as tools to pharmacologically subclassify muscarinic receptors into M1, M2 and M3 subtypes.  In the present study, we have determined the affinity profiles of several of these antagonists at five cloned human muscarinic receptors (m1-m5) stably expressed in Chinese hamster ovary cells (CHO-K1).  At all five receptors, the (R)-enantiomers of trihexyphenidyl and hexbutinol displayed considerably higher affinities (up to 525-fold) than their corresponding (S)-isomers.  The stereoselectivity ratios [inhibition constant(S)/inhibition constant(R)] for both pairs of enantiomers were lowest at m2 receptors, suggesting that less stringent configurational demands are made by this receptor subtype.  The "M1-selective" antagonist (R)-trihexyphenidyl displayed high affinities for m1 and m4 receptors.  The "M2-selective" antagonists himbacine, (+-)-5,11-dihydro-11- ([(2-[(dipropylamino)methyl]-1- piperidinyl)ethyl)amino]carbonyl)-6H-pyrido(2,3-b)(1,4)benzodiazepine-6- one (AF-DX 384), 11-[4-[4-(diethylamino)butyl]-1-piperidinyl)acetyl)-5,11- dihydro-6H-pyrido(2,3-b) (1,4)benzodiazepine-6-one (AQ-RA 741) and (+)-(11-[2-[(diethylamino) methyl]-1-piperidinyl)acetyl)-5,11-di-hydro-6H-pyrido(2,3-b)(1,4) benzodiazepine-6-one [AF-DX 250; the (+)-enantiomer of AF-DX 116] exhibited high affinities for m2 and m4, intermediate affinities for m1 and m3 and low affinities for m5 receptors.  This selectivity profile was most prominent for AQ-RA 741, which displayed 195- and 129-fold higher affinities for m2 and m4 receptors than for m5 receptors.(ABSTRACT TRUNCATED AT 250 WORDS)</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4swN6h2ItNhKDYG3bNImrfW6udTcc2eY3gNSgQU7v3Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3M7jt1CrtQ%253D%253D&md5=24fb5162f94b5e0163c9ae9ca2a5a582</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25B6rje%26aufirst%3DF.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DLambrecht%26aufirst%3DG.%26aulast%3DTacke%26aufirst%3DR.%26aulast%3DMutschler%26aufirst%3DE.%26aulast%3DBrann%26aufirst%3DM.%26atitle%3DAntagonist%2520binding%2520profiles%2520of%2520five%2520cloned%2520human%2520muscarinic%2520receptor%2520subtypes%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1991%26volume%3D256%26spage%3D727%26epage%3D733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quirion, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihm, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entzeroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavi, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladinsky, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bechtel, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ensinger, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendla, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of CNS-active M<sub>2</sub> antagonists: novel structures and pharmacology</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1016/0024-3205(93)90307-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0024-3205%2893%2990307-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8441330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK3sXhtl2jtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1993&pages=497-503&author=H.+N.+Doodsauthor=R.+Quirionauthor=G.+Mihmauthor=W.+Engelauthor=K.+Rudolfauthor=M.+Entzerothauthor=G.+B.+Schiaviauthor=H.+Ladinskyauthor=W.+D.+Bechtelauthor=H.+A.+Ensingerauthor=K.+D.+Mendlaauthor=W.+Eberlein&title=Therapeutic+potential+of+CNS-active+M2+antagonists%3A+novel+structures+and+pharmacology&doi=10.1016%2F0024-3205%2893%2990307-O"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of CNS-active M2 antagonists: novel structures and pharmacology</span></div><div class="casAuthors">Doods, H. N.; Quirion, R.; Mihm, G.; Engel, W.; Rudolf, K.; Entzeroth, M.; Schiavi, G. B.; Ladinsky, H.; Bechtel, W. D.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">497-503</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    </div><div class="casAbstract">A review with 13 refs. of the author's work.  It is believed, that of the five muscarinic receptor subtypes so far identified in the brain, M2 receptors are located presynaptically in the cortex and hippocampus and upon the brain, M2 receptors are located presynaptically in the cortex and hippocampus and upon stimulation inhibit the release of acetylcholine.  Based on this hypothesis, we initiated a drug discovery program with the aim of identifying selective and centrally active M2 antagonists which are capable of enhancing cholinergic transmission.  These efforts resulted in the successful design and synthesis of novel muscarinic antagonists able to cross the blood brain barrier.  However, these compds. show few peripheral effects and possess a superior M2 vs. M1 selectivity.  The prototype of this novel class of M2 selective compds., BIBN 99, could be a valuable tool to test the hypothesis that lipophilic M2 antagonists show beneficial effects in the treatment of cognitive disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouGDQBiyG9E7Vg90H21EOLACvtfcHk0lgTjXLVYACLXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhtl2jtbs%253D&md5=1c0d1472e7b20059e4458eba712c69a4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2F0024-3205%2893%2990307-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0024-3205%252893%252990307-O%26sid%3Dliteratum%253Aachs%26aulast%3DDoods%26aufirst%3DH.%2BN.%26aulast%3DQuirion%26aufirst%3DR.%26aulast%3DMihm%26aufirst%3DG.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DSchiavi%26aufirst%3DG.%2BB.%26aulast%3DLadinsky%26aufirst%3DH.%26aulast%3DBechtel%26aufirst%3DW.%2BD.%26aulast%3DEnsinger%26aufirst%3DH.%2BA.%26aulast%3DMendla%26aufirst%3DK.%2BD.%26aulast%3DEberlein%26aufirst%3DW.%26atitle%3DTherapeutic%2520potential%2520of%2520CNS-active%2520M2%2520antagonists%253A%2520novel%2520structures%2520and%2520pharmacology%26jtitle%3DLife%2520Sci.%26date%3D1993%26volume%3D52%26spage%3D497%26epage%3D503%26doi%3D10.1016%2F0024-3205%2893%2990307-O" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entzeroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihm, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span> <span> </span><span class="NLM_article-title">Characterization of BIBN 99: a lipophilic and selective muscarinic M<sub>2</sub> receptor antagonist</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>242</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/0014-2999(93)90005-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0014-2999%2893%2990005-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8223934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2cXktlw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=242&publication_year=1993&pages=23-30&author=H.+Doodsauthor=M.+Entzerothauthor=H.+Zieglerauthor=G.+Schiaviauthor=W.+Engelauthor=G.+Mihmauthor=K.+Rudolfauthor=W.+Eberlein&title=Characterization+of+BIBN+99%3A+a+lipophilic+and+selective+muscarinic+M2+receptor+antagonist&doi=10.1016%2F0014-2999%2893%2990005-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of BIBN 99: A lipophilic and selective muscarinic M2 receptor antagonist</span></div><div class="casAuthors">Doods, Henri; Entzeroth, Michael; Ziegler, Harald; Schiavi, Giovanni; Engel, Wolfhard; Mihm, Gerhard; Rudolf, Klaus; Eberlein, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">242</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    </div><div class="casAbstract">The present study was designed to characterize the receptor selectivity profile of the novel muscarinic M2 receptor antagonist BIBN 99 (5,11-dihydro-8-chloro-11-[[4-[3-[(2,2-dimethyl-1-oxopentyl)ethylamino]propyl]-1-piperidinyl]acetyl]-6H-pyrido[2,3-b][1,4]benzodiazepin-6-one).  In radioligand binding studies BIBN 99 showed high affinity for m2/M2 sites (pK1 = 7.52/7.57), intermediate affinity for m4 sites (pKi = 6.76) and low affinity for m1/M1 (pK1 = 5.97/6.17), m3/M3 (pKi = 6.11/6.04) and m5 sites (pKi = 5.84).  Functional studies in vitro showed BIBN 99 to be a competitive antagonist and to have an 11- to 25-fold higher affinity for M2 receptors than for putative M1 receptors in the rabbit vas deferens or M3 receptors in guinea-pig trachea.  In vivo studies revealed that BIBN 99 is able to cross the blood-brain barrier, and although showing an approx. 3-fold higher affinity for M2 binding sites BIBN 99 appeared to be 7- to 18-fold less potent than AF-DX 116 in inhibiting muscarinic agonist or vagally induced bradycardia in rats and guinea-pigs.  The results show that BIBN 99 is the first lipophilic muscarinic M2 receptor antagonist to have remarkable M2 vs. M1 selectivity (30-fold).  In addn., BIBN 99 possesses central nervous system activity and only minor peripheral cardiac effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8mHfHIPQPGbVg90H21EOLACvtfcHk0li0cg3B3S-v5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktlw%253D&md5=434d8f19eabb285348c60180a550db25</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2893%2990005-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252893%252990005-3%26sid%3Dliteratum%253Aachs%26aulast%3DDoods%26aufirst%3DH.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DZiegler%26aufirst%3DH.%26aulast%3DSchiavi%26aufirst%3DG.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DMihm%26aufirst%3DG.%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DEberlein%26aufirst%3DW.%26atitle%3DCharacterization%2520of%2520BIBN%252099%253A%2520a%2520lipophilic%2520and%2520selective%2520muscarinic%2520M2%2520receptor%2520antagonist%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1993%26volume%3D242%26spage%3D23%26epage%3D30%26doi%3D10.1016%2F0014-2999%2893%2990005-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maggio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedeschi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchiorre, C.</span></span> <span> </span><span class="NLM_article-title">Binding profile of the selective muscarinic receptor antagonist tripitramine</span>. <i>Eur. J. Pharmacol., Mol. Pharmacol. Sect.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1016/0922-4106(94)90075-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0922-4106%2894%2990075-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=7805774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2cXmsVKjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1994&pages=459-462&author=R.+Maggioauthor=P.+Barbierauthor=M.+L.+Bolognesiauthor=A.+Minariniauthor=D.+Tedeschiauthor=C.+Melchiorre&title=Binding+profile+of+the+selective+muscarinic+receptor+antagonist+tripitramine&doi=10.1016%2F0922-4106%2894%2990075-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Binding profile of the selective muscarinic receptor antagonist tripitramine</span></div><div class="casAuthors">Maggio, Roberto; Barbier, Pascaline; Bolognesi, Maria L.; Minarini, Anna; Tedeschi, Daniele; Melchiorre, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology, Molecular Pharmacology Section</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-62</span>CODEN:
                <span class="NLM_cas:coden">EJPPET</span>;
        ISSN:<span class="NLM_cas:issn">0922-4106</span>.
    </div><div class="casAbstract">The binding selectivity of the muscarinic antagonist tripitramine has been tested on the five cloned human muscarinic receptor subtypes (Hm1 to Hm5) expressed in chinese hamster ovary (CHO-K1) cells.  The results indicate that tripitramine binds to the muscarinic Hm2 receptor with a Ki value of 0.27 ± 0.02 nM.  Tripitramine distinguishes Hm2 vs. Hm4 by a factor of 24 and vs. Hm3 and Hm5 by a factor of 142 and 125, resp.  A lower affinity ratio, about 6-fold, was found between muscarinic Hm2 and Hm1 receptors.  A comparative study with the well-known selective muscarinic M2 receptor antagonist methoctramine indicates that tripitramine has gained both potency and selectivity for the muscarinic Hm2 receptor subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlP4SfCTKVJrVg90H21EOLACvtfcHk0li0cg3B3S-v5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsVKjs7k%253D&md5=6067bf0125d1ec322d40b59cb585a652</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0922-4106%2894%2990075-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0922-4106%252894%252990075-2%26sid%3Dliteratum%253Aachs%26aulast%3DMaggio%26aufirst%3DR.%26aulast%3DBarbier%26aufirst%3DP.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DMinarini%26aufirst%3DA.%26aulast%3DTedeschi%26aufirst%3DD.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DBinding%2520profile%2520of%2520the%2520selective%2520muscarinic%2520receptor%2520antagonist%2520tripitramine%26jtitle%3DEur.%2520J.%2520Pharmacol.%252C%2520Mol.%2520Pharmacol.%2520Sect.%26date%3D1994%26volume%3D268%26spage%3D459%26epage%3D462%26doi%3D10.1016%2F0922-4106%2894%2990075-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reba, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, V. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzeszotarski, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgold, J.</span></span> <span> </span><span class="NLM_article-title">A novel m2-selective muscarinic antagonist: binding characteristics and autoradiographic distribution in rat brain</span>. <i>Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/0006-8993(92)90141-U</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0006-8993%2892%2990141-U" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1393548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK38XkvVegsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=1992&pages=253-260&author=M.+S.+Gitlerauthor=R.+C.+Rebaauthor=V.+I.+Cohenauthor=W.+J.+Rzeszotarskiauthor=J.+Baumgold&title=A+novel+m2-selective+muscarinic+antagonist%3A+binding+characteristics+and+autoradiographic+distribution+in+rat+brain&doi=10.1016%2F0006-8993%2892%2990141-U"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A novel M2-selective muscarinic antagonist:  binding characteristics and autoradiographic distribution in rat brain</span></div><div class="casAuthors">Gitler, Miriam S.; Reba, Richard C.; Cohen, Victor I.; Rzeszotarski, Waclaw J.; Baumgold, Jesse</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">253-60</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    </div><div class="casAbstract">Although several M2-selective muscarinic antagonists have been described, they are not particularly potent.  Thus, the development of potent M2-selective compds. remains an important goal.  The authors now report that a bio-isoster of AQ-RA 741 is both one order of magnitude more potent and slightly more selective than previously described compds.  DIBA (I), a di-benzo deriv. of AQ-RA 741, in which the pyridine of the tricycle is replaced with a benzene ring, had Ki values of 4, 0.3, 11 and 2 nM at M1 through M4 receptors, resp.  These values were detd. in competition studies with [3H]N-methylscopolamine ([3H]NMS) in membranes from transfected A9 L cells (M1 and M3), rat heart (M2) and NG108-15 cells (M4).  AQ-RA 741 had Ki values of 34, 4, 86 and 15 nm at each of these receptors.  The autoradiog. distribution of I binding sites was detd. by competition studies of [3H]NMS in rat brain.  At low concn., I reduced [3H]NMS binding most significantly from superior colliculi, thalamus, hypothalamus, pontine nucleus, and interpeduncular nucleus, and not appreciably from caudate nucleus, cerebral cortical regions, or hippocampus, consistent with its binding to m2 receptors.  These data indicate that I is the most potent, M2-selective muscarinic antagonist yet described.  I should therefore become a useful probe in future studies of muscarinic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFUtuhX4_TGbVg90H21EOLACvtfcHk0li0cg3B3S-v5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XkvVegsb4%253D&md5=c738f525755ed004c458acce0e29018b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2F0006-8993%2892%2990141-U&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-8993%252892%252990141-U%26sid%3Dliteratum%253Aachs%26aulast%3DGitler%26aufirst%3DM.%2BS.%26aulast%3DReba%26aufirst%3DR.%2BC.%26aulast%3DCohen%26aufirst%3DV.%2BI.%26aulast%3DRzeszotarski%26aufirst%3DW.%2BJ.%26aulast%3DBaumgold%26aufirst%3DJ.%26atitle%3DA%2520novel%2520m2-selective%2520muscarinic%2520antagonist%253A%2520binding%2520characteristics%2520and%2520autoradiographic%2520distribution%2520in%2520rat%2520brain%26jtitle%3DBrain%2520Res.%26date%3D1992%26volume%3D582%26spage%3D253%26epage%3D260%26doi%3D10.1016%2F0006-8993%2892%2990141-U" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tränkle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pegoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. W.</span></span> <span> </span><span class="NLM_article-title">M<sub>2</sub> Subtype preferring dibenzodiazepinone-type muscarinic receptor ligands: Effect of chemical homo-dimerization on orthosteric (and allosteric?) binding</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3970</span>– <span class="NLM_lpage">3990</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2Fj.bmc.2015.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=25650309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Ogtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=3970-3990&author=M.+Kellerauthor=C.+Tr%C3%A4nkleauthor=X.+Sheauthor=A.+Pegoliauthor=G.+Bernhardtauthor=A.+Buschauerauthor=R.+W.+Read&title=M2+Subtype+preferring+dibenzodiazepinone-type+muscarinic+receptor+ligands%3A+Effect+of+chemical+homo-dimerization+on+orthosteric+%28and+allosteric%3F%29+binding&doi=10.1016%2Fj.bmc.2015.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">M2 Subtype preferring dibenzodiazepinone-type muscarinic receptor ligands: Effect of chemical homo-dimerization on orthosteric (and allosteric?) binding</span></div><div class="casAuthors">Keller, Max; Traenkle, Christian; She, Xueke; Pegoli, Andrea; Bernhardt, Guenther; Buschauer, Armin; Read, Roger W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3970-3990</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of new dibenzodiazepinone-type muscarinic receptor ligands, including two homo-dimeric compds., was prepd.  Sixteen representative compds. were characterized in equil. binding studies with [3H]N-methylscopolamine ([3H]NMS) at the muscarinic receptor subtype M2, and seven selected compds. were addnl. investigated at M1, M3, M4 and M5 with respect to receptor subtype selectivity.  The side chain of the known M2 preferring muscarinic receptor antagonist DIBA was widely varied with respect to chain length and type of the basic group (amine, imidazole, guanidine and piperazine).  Most of the structural changes were well tolerated with respect to muscarinic receptor binding, detd. by displacement of [3H]NMS.  Compds. investigated at all subtypes shared a similar selectivity profile, which can be summarized as M2 > M1 ≈ M4 > M3 ≈ M5 (46, 50, 57, 62-64) and M2 > M1 ≈ M4 > M3 > M5 (1, 58).  The homo-dimeric dibenzodiazepinone derivs. UNSW-MK250 (63) and UNSW-MK262 (64) exhibited the highest M2 receptor affinities (pIC50 = 9.0 and 9.2, resp.).  At the M2 receptor a steep curve slope of -2 was found for the dimeric ligand 63, which cannot be described according to the law of mass action, suggesting a more complex mechanism of binding.  In addn. to equil. binding studies, for selected ligands, we detd. pEC50,diss, an est. of affinity to the allosteric site of M2 receptors occupied with [3H]NMS.  Compds. 58 and 62-64 were capable of retarding [3H]NMS dissocn. by a factor >10 (Emax,diss >92%), with highest potency (pEC50,diss = 5.56) residing in the dimeric compd. 64.  As the monomeric counterpart of 64 was 100 times less potent (62: pEC50,diss = 3.59), these data suggest that chem. dimerization of dibenzodiazepinone-type M receptor ligands can enhance allosteric binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8D4k-_1Hnb7Vg90H21EOLACvtfcHk0ljJPtov44-9iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Ogtr4%253D&md5=00c26092810d90955e92c09ade5cec1d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DTr%25C3%25A4nkle%26aufirst%3DC.%26aulast%3DShe%26aufirst%3DX.%26aulast%3DPegoli%26aufirst%3DA.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26aulast%3DRead%26aufirst%3DR.%2BW.%26atitle%3DM2%2520Subtype%2520preferring%2520dibenzodiazepinone-type%2520muscarinic%2520receptor%2520ligands%253A%2520Effect%2520of%2520chemical%2520homo-dimerization%2520on%2520orthosteric%2520%2528and%2520allosteric%253F%2529%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D3970%26epage%3D3990%26doi%3D10.1016%2Fj.bmc.2015.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pegoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wifling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span> <span> </span><span class="NLM_article-title">Radiolabeled dibenzodiazepinone-type antagonists give evidence of dualsteric binding at the M<sub>2</sub> muscarinic acetylcholine receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3314</span>– <span class="NLM_lpage">3334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01892</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01892" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVaqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3314-3334&author=A.+Pegoliauthor=X.+Sheauthor=D.+Wiflingauthor=H.+H%C3%BCbnerauthor=G.+Bernhardtauthor=P.+Gmeinerauthor=M.+Keller&title=Radiolabeled+dibenzodiazepinone-type+antagonists+give+evidence+of+dualsteric+binding+at+the+M2+muscarinic+acetylcholine+receptor&doi=10.1021%2Facs.jmedchem.6b01892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Radiolabeled Dibenzodiazepinone-Type Antagonists Give Evidence of Dualsteric Binding at the M2 Muscarinic Acetylcholine Receptor</span></div><div class="casAuthors">Pegoli, Andrea; She, Xueke; Wifling, David; Huebner, Harald; Bernhardt, Guenther; Gmeiner, Peter; Keller, Max</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3314-3334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dualsteric ligand approach, aiming at ligands with improved subtype selectivity, has been increasingly applied to muscarinic receptors (MRs).  In this article we present the synthesis and characterization of a M2R subtype-preferring radiolabeled dibenzodiazepinone-type antagonist [3H]UNSW-MK259, (I) and its homodimeric analog [3H]UR-AP060, (II).  Satn. binding studies at the M2R, using the orthosteric antagonist atropine to det. unspecific binding, proved that the monomeric and the dimeric compd. bind to the orthosteric binding site (apparent Kd: 0.87 and 0.24 nM, resp.).  Various binding studies with I and II at the M2R, for instance, satn. binding expts. in the presence of the allosteric MR modulators W84 or LY2119620 (Schild-like anal.) suggested a competitive mechanism between the allosteric modulator and the dibenzodiazepinone derivs., and thus a dualsteric binding mode of both I and II.  This was consistent with the results of M2R MD simulations (≥2 μs) performed with I and II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMeA4UbzK06LVg90H21EOLACvtfcHk0ljJPtov44-9iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVaqsbs%253D&md5=1dcfeac950b89fc293968ceed01e2500</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01892%26sid%3Dliteratum%253Aachs%26aulast%3DPegoli%26aufirst%3DA.%26aulast%3DShe%26aufirst%3DX.%26aulast%3DWifling%26aufirst%3DD.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DKeller%26aufirst%3DM.%26atitle%3DRadiolabeled%2520dibenzodiazepinone-type%2520antagonists%2520give%2520evidence%2520of%2520dualsteric%2520binding%2520at%2520the%2520M2%2520muscarinic%2520acetylcholine%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3314%26epage%3D3334%26doi%3D10.1021%2Facs.jmedchem.6b01892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pegoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huebner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span> <span> </span><span class="NLM_article-title">Heterodimerization of dibenzodiazepinone-type muscarinic acetylcholine receptor ligands leads to increased M<sub>2</sub>R affinity and selectivity</span>. <i>ACS Omega.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">6741</span>– <span class="NLM_lpage">6754</span>, <span class="refDoi"> DOI: 10.1021/acsomega.7b01085</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b01085" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Cru7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=6741-6754&author=X.+Sheauthor=A.+Pegoliauthor=J.+Mayrauthor=H.+Huebnerauthor=G.+Bernhardtauthor=P.+Gmeinerauthor=M.+Keller&title=Heterodimerization+of+dibenzodiazepinone-type+muscarinic+acetylcholine+receptor+ligands+leads+to+increased+M2R+affinity+and+selectivity&doi=10.1021%2Facsomega.7b01085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Heterodimerization of Dibenzodiazepinone-Type Muscarinic Acetylcholine Receptor Ligands Leads to Increased M2R Affinity and Selectivity</span></div><div class="casAuthors">She, Xueke; Pegoli, Andrea; Mayr, Judith; Huebner, Harald; Bernhardt, Guenther; Gmeiner, Peter; Keller, Max</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6741-6754</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In search for selective ligands for the muscarinic acetylcholine receptor (MR) subtype M2, the dimeric ligand approach, that is combining two pharmacophores in one and the same mol., was pursued.  Different types (agonists, antagonists, orthosteric, and allosteric) of monomeric MR ligands were combined by various linkers with a dibenzodiazepinone-type MR antagonist, affording five types of heterodimeric compds. ("DIBA-xanomeline," "DIBA-TBPB," "DIBA-77-LH-28-1," "DIBA-propantheline," and "DIBA-4-DAMP"), which showed high M2R affinities (pKi > 8.3).  The heterodimeric ligand UR-SK75 (46) exhibited the highest M2R affinity and selectivity [pKi (M1R-M5R): 8.84, 10.14, 7.88, 8.59, and 7.47].  Two tritium-labeled dimeric derivs. ("DIBA-xanomeline"-type: [3H]UR-SK71 ([3H]44) and "DIBA-TBPB"-type: [3H]UR-SK59 ([3H]64)) were prepd. to investigate their binding modes at hM2R.  Satn.-binding expts. showed that these compds. address the orthosteric binding site of the M2R.  The investigation of the effect of various allosteric MR modulators [gallamine (13), W84 (14), and LY2119620 (15)] on the equil. (13-15) or satn. (14) binding of [3H]64 suggested a competitive mechanism between [3H]64 and the investigated allosteric ligands, and consequently a dualsteric binding mode of 64 at the M2R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq4nnRJHnLF7Vg90H21EOLACvtfcHk0lhX9_jtucO1Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Cru7zN&md5=4f7d67758e6e919d62541c5bec83e21d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b01085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b01085%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DX.%26aulast%3DPegoli%26aufirst%3DA.%26aulast%3DMayr%26aufirst%3DJ.%26aulast%3DHuebner%26aufirst%3DH.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DKeller%26aufirst%3DM.%26atitle%3DHeterodimerization%2520of%2520dibenzodiazepinone-type%2520muscarinic%2520acetylcholine%2520receptor%2520ligands%2520leads%2520to%2520increased%2520M2R%2520affinity%2520and%2520selectivity%26jtitle%3DACS%2520Omega.%26date%3D2017%26volume%3D2%26spage%3D6741%26epage%3D6754%26doi%3D10.1021%2Facsomega.7b01085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Amici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dallanoce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzgrabe, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trankle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, K.</span></span> <span> </span><span class="NLM_article-title">Allosteric ligands for G protein-coupled receptors: a novel strategy with attractive therapeutic opportunities</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1002/med.20166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1002%2Fmed.20166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=19557759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVOlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=463-549&author=M.+De%0AAmiciauthor=C.+Dallanoceauthor=U.+Holzgrabeauthor=C.+Trankleauthor=K.+Mohr&title=Allosteric+ligands+for+G+protein-coupled+receptors%3A+a+novel+strategy+with+attractive+therapeutic+opportunities&doi=10.1002%2Fmed.20166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric ligands for G protein-coupled receptors: a novel strategy with attractive therapeutic opportunities</span></div><div class="casAuthors">De Amici, Marco; Dallanoce, Clelia; Holzgrabe, Ulrike; Trankle, Christian; Mohr, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">463-549</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Allosteric receptor ligands bind to a recognition site that is distinct from the binding site of the endogenous messenger mol.  As a consequence, allosteric agents may attach to receptors that are already transmitter-bound.  Ternary complex formation opens an avenue to qual. new drug actions at G protein-coupled receptors (GPCRs), in particular receptor subtype selective potentiation of endogenous transmitter action.  Consequently, suitable exploitation of allosteric recognition sites as alternative mol. targets could pave the way to a drug discovery paradigm different from those aimed at mimicking or blocking the effects of endogenous (orthosteric) receptor activators.  The no. of allosteric ligands reported to modulate GPCR function is steadily increasing and some have already reached routine clin. use.  This review aims at introducing into this fascinating field of drug discovery and at providing an overview about the achievements that have already been made.  Various case examples will be discussed in the framework of GPCR classification (family A, B, and C receptors).  In addn., the behavior at muscarinic receptors of hybrid derivs. incorporating both an allosteric and an orthosteric fragment in a common mol. skeleton will be illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorICLQyOlQ3bVg90H21EOLACvtfcHk0lhX9_jtucO1Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVOlsbY%253D&md5=99697b812d5dac4e90d9e6bcc46a639d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fmed.20166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20166%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BAmici%26aufirst%3DM.%26aulast%3DDallanoce%26aufirst%3DC.%26aulast%3DHolzgrabe%26aufirst%3DU.%26aulast%3DTrankle%26aufirst%3DC.%26aulast%3DMohr%26aufirst%3DK.%26atitle%3DAllosteric%2520ligands%2520for%2520G%2520protein-coupled%2520receptors%253A%2520a%2520novel%2520strategy%2520with%2520attractive%2520therapeutic%2520opportunities%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2010%26volume%3D30%26spage%3D463%26epage%3D549%26doi%3D10.1002%2Fmed.20166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert Lane, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">The best of both worlds? Bitopic orthosteric/allosteric ligands of g protein-coupled receptors</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1146%2Fannurev-pharmtox-010611-134514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=21910627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsV2ntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=153-178&author=C.+Valantauthor=J.+Robert+Laneauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=The+best+of+both+worlds%3F+Bitopic+orthosteric%2Fallosteric+ligands+of+g+protein-coupled+receptors&doi=10.1146%2Fannurev-pharmtox-010611-134514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The best of both worlds? Bitopic orthosteric/allosteric ligands of G protein-coupled receptors</span></div><div class="casAuthors">Valant, Celine; Lane, J. Robert; Sexton, Patrick M.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">153-178</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  It is now acknowledged that G protein-coupled receptors, the largest class of drug targets, adopt multiple active states that can be preferentially stabilized by orthosteric ligands or allosteric modulators, thus giving rise to the phenomenon of pathway-biased signaling.  In the past few years, researchers have begun to explore the potential of linking orthosteric and allosteric pharmacophores to yield bitopic hybrid ligands.  This approach is an extension of the more traditional bivalent ligand concept and shares some of the same challenges, including the choice and role of the linker between the two pharmacophores and the validation of mechanism of action.  Nonetheless, the promise of bitopic ligands is the generation of novel chem. tools that have improved affinity and/or selectivity profiles.  Previously identified functionally selective compds. (and medicines) also may act via a bitopic mechanism, suggesting that the phenomenon is more widespread than currently appreciated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQs4WlX5wY8LVg90H21EOLACvtfcHk0lhpKvboLDkyAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsV2ntrc%253D&md5=334839005a548a9739df466e3d09e1ae</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134514%26sid%3Dliteratum%253Aachs%26aulast%3DValant%26aufirst%3DC.%26aulast%3DRobert%2BLane%26aufirst%3DJ.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DThe%2520best%2520of%2520both%2520worlds%253F%2520Bitopic%2520orthosteric%252Fallosteric%2520ligands%2520of%2520g%2520protein-coupled%2520receptors%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2012%26volume%3D52%26spage%3D153%26epage%3D178%26doi%3D10.1146%2Fannurev-pharmtox-010611-134514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ring, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Activation and allosteric modulation of a muscarinic acetylcholine receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>504</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1038/nature12735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature12735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=24256733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=101-106&author=A.+C.+Kruseauthor=A.+M.+Ringauthor=A.+Manglikauthor=J.+Huauthor=K.+Huauthor=K.+Eitelauthor=H.+H%C3%BCbnerauthor=E.+Pardonauthor=C.+Valantauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=C.+C.+Felderauthor=P.+Gmeinerauthor=J.+Steyaertauthor=W.+I.+Weisauthor=K.+C.+Garciaauthor=J.+Wessauthor=B.+K.+Kobilka&title=Activation+and+allosteric+modulation+of+a+muscarinic+acetylcholine+receptor&doi=10.1038%2Fnature12735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Activation and allosteric modulation of a muscarinic acetylcholine receptor</span></div><div class="casAuthors">Kruse, Andrew C.; Ring, Aaron M.; Manglik, Aashish; Hu, Jianxin; Hu, Kelly; Eitel, Katrin; Huebner, Harald; Pardon, Els; Valant, Celine; Sexton, Patrick M.; Christopoulos, Arthur; Felder, Christian C.; Gmeiner, Peter; Steyaert, Jan; Weis, William I.; Garcia, K. Christopher; Wess, Juergen; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7478</span>),
    <span class="NLM_cas:pages">101-106</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite recent advances in crystallog. and the availability of G-protein-coupled receptor (GPCR) structures, little is known about the mechanism of their activation process, as only the β2 adrenergic receptor (β2AR) and rhodopsin have been crystd. in fully active conformations.  Here we report the structure of an agonist-bound, active state of the human M2 muscarinic acetylcholine receptor stabilized by a G-protein mimetic camelid antibody fragment isolated by conformational selection using yeast surface display.  In addn. to the expected changes in the intracellular surface, the structure reveals larger conformational changes in the extracellular region and orthosteric binding site than obsd. in the active states of the β2AR and rhodopsin.  We also report the structure of the M2 receptor simultaneously bound to the orthosteric agonist iperoxo and the pos. allosteric modulator LY2119620.  This structure reveals that LY2119620 recognizes a largely pre-formed binding site in the extracellular vestibule of the iperoxo-bound receptor, inducing a slight contraction of this outer binding pocket.  These structures offer important insights into the activation mechanism and allosteric modulation of muscarinic receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOuUgQSB7Dd7Vg90H21EOLACvtfcHk0lhpKvboLDkyAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLjP&md5=ecddab0ab467321ac243b86bb82b71f8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature12735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12735%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DRing%26aufirst%3DA.%2BM.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DK.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DFelder%26aufirst%3DC.%2BC.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DGarcia%26aufirst%3DK.%2BC.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DActivation%2520and%2520allosteric%2520modulation%2520of%2520a%2520muscarinic%2520acetylcholine%2520receptor%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D101%26epage%3D106%26doi%3D10.1038%2Fnature12735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Advances in G protein-coupled receptor allostery: from function to structure</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">478</span>, <span class="refDoi"> DOI: 10.1124/mol.114.094342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1124%2Fmol.114.094342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=25061106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslamurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=463-478&author=A.+Christopoulos&title=Advances+in+G+protein-coupled+receptor+allostery%3A+from+function+to+structure&doi=10.1124%2Fmol.114.094342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in G protein-coupled receptor allostery: from function to structure</span></div><div class="casAuthors">Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">463-478, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  It is now widely accepted that G protein-coupled receptors (GPCRs) are highly dynamic proteins that adopt multiple active states linked to distinct functional outcomes.  Furthermore, these states can be differentially stabilized not only by orthosteric ligands but also by allosteric ligands acting at spatially distinct binding sites.  The key pharmacol. characteristics of GPCR allostery include improved selectivity due to either greater sequence divergence between receptor subtypes and/or subtype-selective cooperativity, a ceiling level to the effect, probe dependence (whereby the magnitude and direction of the allosteric effect change with the nature of the interacting ligands) and the potential for biased signaling.  Recent chem. biol. developments are beginning to demonstrate how the incorporation of anal. pharmacol. and operational modeling into the exptl. workflow can enrich structure-activity studies of allostery and bias and have also led to the discovery of a new class of hybrid orthosteric/allosteric (bitopic) mols.  The potential for endogenous allosteric modulators to play a role in physiol. and disease remains to be fully appreciated but will likely represent an important area for future studies.  Finally, breakthroughs in structural and computational biol. are beginning to unravel the mechanistic basis of GPCR allosteric modulation at the mol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8oX6HPCpaG7Vg90H21EOLACvtfcHk0lgGV5g-Kmf0yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslamurjI&md5=dba22b1ab717a25b9a2f9a9c888df9e6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.094342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.094342%26sid%3Dliteratum%253Aachs%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAdvances%2520in%2520G%2520protein-coupled%2520receptor%2520allostery%253A%2520from%2520function%2520to%2520structure%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26spage%3D463%26epage%3D478%26doi%3D10.1124%2Fmol.114.094342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yurugi-Kobayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiroishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span> <span> </span><span class="NLM_article-title">Structure of the human M<sub>2</sub> muscarinic acetylcholine receptor bound to an antagonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature10753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature10753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=22278061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ejsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=547-551&author=K.+Hagaauthor=A.+C.+Kruseauthor=H.+Asadaauthor=T.+Yurugi-Kobayashiauthor=M.+Shiroishiauthor=C.+Zhangauthor=W.+I.+Weisauthor=T.+Okadaauthor=B.+K.+Kobilkaauthor=T.+Hagaauthor=T.+Kobayashi&title=Structure+of+the+human+M2+muscarinic+acetylcholine+receptor+bound+to+an+antagonist&doi=10.1038%2Fnature10753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human M2 muscarinic acetylcholine receptor bound to an antagonist</span></div><div class="casAuthors">Haga, Kazuko; Kruse, Andrew C.; Asada, Hidetsugu; Yurugi-Kobayashi, Takami; Shiroishi, Mitsunori; Zhang, Cheng; Weis, William I.; Okada, Tetsuji; Kobilka, Brian K.; Haga, Tatsuya; Kobayashi, Takuya</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">547-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The X-ray crystal structure of the M2 muscarinic acetylcholine receptor, which is essential for the physiol. control of cardiovascular function, is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosewFev-FNErVg90H21EOLACvtfcHk0lgGV5g-Kmf0yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ejsro%253D&md5=c33b8d9303d58db7dfdd4c18ec4f597b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature10753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10753%26sid%3Dliteratum%253Aachs%26aulast%3DHaga%26aufirst%3DK.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DAsada%26aufirst%3DH.%26aulast%3DYurugi-Kobayashi%26aufirst%3DT.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DHaga%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStructure%2520of%2520the%2520human%2520M2%2520muscarinic%2520acetylcholine%2520receptor%2520bound%2520to%2520an%2520antagonist%26jtitle%3DNature%26date%3D2012%26volume%3D482%26spage%3D547%26epage%3D551%26doi%3D10.1038%2Fnature10753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosemond, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure and dynamics of the M<sub>3</sub> muscarinic acetylcholine receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.1038/nature10867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature10867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=22358844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=552-556&author=A.+C.+Kruseauthor=J.+Huauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+M.+Rosenbaumauthor=E.+Rosemondauthor=H.+F.+Greenauthor=T.+Liuauthor=P.+S.+Chaeauthor=R.+O.+Drorauthor=D.+E.+Shawauthor=W.+I.+Weisauthor=J.+Wessauthor=B.+K.+Kobilka&title=Structure+and+dynamics+of+the+M3+muscarinic+acetylcholine+receptor&doi=10.1038%2Fnature10867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of the M3 muscarinic acetylcholine receptor</span></div><div class="casAuthors">Kruse, Andrew C.; Hu, Jianxin; Pan, Albert C.; Arlow, Daniel H.; Rosenbaum, Daniel M.; Rosemond, Erica; Green, Hillary F.; Liu, Tong; Chae, Pil Seok; Dror, Ron O.; Shaw, David E.; Weis, William I.; Wess, Jurgen; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">552-556</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acetylcholine, the first neurotransmitter to be identified, exerts many of its physiol. actions via activation of a family of G-protein-coupled receptors (GPCRs) known as muscarinic acetylcholine receptors (mAChRs).  Although the five mAChR subtypes (M1-M5) share a high degree of sequence homol., they show pronounced differences in G-protein coupling preference and the physiol. responses they mediate.  Unfortunately, despite decades of effort, no therapeutic agents endowed with clear mAChR subtype selectivity have been developed to exploit these differences.  We describe here the structure of the Gq/11-coupled M3 mAChR ('M3 receptor', from rat) bound to the bronchodilator drug tiotropium and identify the binding mode for this clin. important drug.  This structure, together with that of the Gi/o-coupled M2 receptor, offers possibilities for the design of mAChR subtype-selective ligands.  Importantly, the M3 receptor structure allows a structural comparison between two members of a mammalian GPCR subfamily displaying different G-protein coupling selectivities.  Furthermore, mol. dynamics simulations suggest that tiotropium binds transiently to an allosteric site en route to the binding pocket of both receptors.  These simulations offer a structural view of an allosteric binding mode for an orthosteric GPCR ligand and provide addnl. opportunities for the design of ligands with different affinities or binding kinetics for different mAChR subtypes.  Our findings not only offer insights into the structure and function of one of the most important GPCR families, but may also facilitate the design of improved therapeutics targeting these crit. receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozveVSiaQvz7Vg90H21EOLACvtfcHk0ljm1riJ9K0R8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D&md5=31bb89b1d14cf122353fced8f905e996</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature10867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10867%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DRosemond%26aufirst%3DE.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520and%2520dynamics%2520of%2520the%2520M3%2520muscarinic%2520acetylcholine%2520receptor%26jtitle%3DNature%26date%3D2012%26volume%3D482%26spage%3D552%26epage%3D556%26doi%3D10.1038%2Fnature10867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valcourt, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahmani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nature12595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature12595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=24121438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=295-299&author=R.+O.+Drorauthor=H.+F.+Greenauthor=C.+Valantauthor=D.+W.+Borhaniauthor=J.+R.+Valcourtauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=M.+Canalsauthor=J.+R.+Laneauthor=R.+Rahmaniauthor=J.+B.+Baellauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=D.+E.+Shaw&title=Structural+basis+for+modulation+of+a+G-protein-coupled+receptor+by+allosteric+drugs&doi=10.1038%2Fnature12595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span></div><div class="casAuthors">Dror, Ron O.; Green, Hillary F.; Valant, Celine; Borhani, David W.; Valcourt, James R.; Pan, Albert C.; Arlow, Daniel H.; Canals, Meritxell; Lane, J. Robert; Rahmani, Raphael; Baell, Jonathan B.; Sexton, Patrick M.; Christopoulos, Arthur; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7475</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of G-protein-coupled receptor (GPCR) allosteric modulators, an active area of modern pharmaceutical research, has proved challenging because neither the binding modes nor the mol. mechanisms of such drugs are known.  Here we det. binding sites, bound conformations and specific drug-receptor interactions for several allosteric modulators of the M2 muscarinic acetylcholine receptor (M2 receptor), a prototypical family A GPCR, using at.-level simulations in which the modulators spontaneously assoc. with the receptor.  Despite substantial structural diversity, all modulators form cation-π interactions with clusters of arom. residues in the receptor extracellular vestibule, approx. 15 Å from the classical, 'orthosteric' ligand-binding site.  We validate the obsd. modulator binding modes through radioligand binding expts. on receptor mutants designed, on the basis of our simulations, either to increase or to decrease modulator affinity.  Simulations also revealed mechanisms that contribute to pos. and neg. allosteric modulation of classical ligand binding, including coupled conformational changes of the two binding sites and electrostatic interactions between ligands in these sites.  These observations enabled the design of chem. modifications that substantially alter a modulator's allosteric effects.  Our findings thus provide a structural basis for the rational design of allosteric modulators targeting muscarinic and possibly other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTlvAcaBf-1rVg90H21EOLACvtfcHk0ljm1riJ9K0R8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN&md5=2fa40ecd46ce62125ce4a12c18270b1f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature12595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12595%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DValcourt%26aufirst%3DJ.%2BR.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DRahmani%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520basis%2520for%2520modulation%2520of%2520a%2520G-protein-coupled%2520receptor%2520by%2520allosteric%2520drugs%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D295%26epage%3D299%26doi%3D10.1038%2Fnature12595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stossel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huebner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Structure-based design and discovery of new M<sub>2</sub> receptor agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9239</span>– <span class="NLM_lpage">9250</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslGnu7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9239-9250&author=I.+Fishauthor=A.+Stosselauthor=K.+Eitelauthor=C.+Valantauthor=S.+Alboldauthor=H.+Huebnerauthor=D.+Mollerauthor=M.+J.+Clarkauthor=R.+K.+Sunaharaauthor=A.+Christopoulosauthor=B.+K.+Shoichetauthor=P.+Gmeiner&title=Structure-based+design+and+discovery+of+new+M2+receptor+agonists&doi=10.1021%2Facs.jmedchem.7b01113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design and Discovery of New M2 Receptor Agonists</span></div><div class="casAuthors">Fish, Inbar; Stossel, Anne; Eitel, Katrin; Valant, Celine; Albold, Sabine; Huebner, Harald; Moller, Dorothee; Clark, Mary J.; Sunahara, Roger K.; Christopoulos, Arthur; Shoichet, Brian K.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9239-9250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Muscarinic receptor agonists are characterized by apparently strict restraints on their tertiary or quaternary amine and their distance to an ester or related center.  On the basis of the active state crystal structure of the muscarinic M2 receptor in complex with iperoxo, the authors explored potential agonists that lacked the highly conserved functionalities of previously known ligands.  Using structure-guided pharmacophore design followed by docking, the authors found two agonists (compds. 3 (2-(3-methoxyphenyl)-N,N,N-trimethylethan-1-aminium formate) and 17 (2-(benzofuran-6-yl)-N,N,N-trimethylethan-1-aminium formate)), out of 19 docked and synthesized compds., that fit the receptor well and were predicted to form a hydrogen-bond conserved among known agonists.  Structural optimization led to compd. 28 (2-(2,3-dihydrobenzofuran-6-yl)-N,N,N-trimethylethan-1-aminium formate), which was 4-fold more potent than its parent 3.  Fortified by the discovery of this new scaffold, the authors sought a broader range of chemotypes by docking 2.2 million fragments, which revealed another three micromolar agonists unrelated either to 28 or known muscarinics.  Even pockets as tightly defined and as deeply studied as that of the muscarinic reveal opportunities for the structure-based design and the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Yk4WSO0I3LVg90H21EOLACvtfcHk0ljwsw-hv8fCZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslGnu7nN&md5=113c2c9b16bc816ddde88634a129da09</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01113%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DI.%26aulast%3DStossel%26aufirst%3DA.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DAlbold%26aufirst%3DS.%26aulast%3DHuebner%26aufirst%3DH.%26aulast%3DMoller%26aufirst%3DD.%26aulast%3DClark%26aufirst%3DM.%2BJ.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DStructure-based%2520design%2520and%2520discovery%2520of%2520new%2520M2%2520receptor%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9239%26epage%3D9250%26doi%3D10.1021%2Facs.jmedchem.7b01113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaake, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartuschat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaindl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampp, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Structure-guided development of selective M<sub>3</sub> muscarinic acetylcholine receptor antagonists</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">12046</span>– <span class="NLM_lpage">12050</span>, <span class="refDoi"> DOI: 10.1073/pnas.1813988115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1073%2Fpnas.1813988115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=30404914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFanu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=12046-12050&author=H.+Liuauthor=J.+Hofmannauthor=I.+Fishauthor=B.+Schaakeauthor=K.+Eitelauthor=A.+Bartuschatauthor=J.+Kaindlauthor=H.+Ramppauthor=A.+Banerjeeauthor=H.+Hubnerauthor=M.+J.+Clarkauthor=S.+G.+Vincentauthor=J.+T.+Fisherauthor=M.+R.+Heinrichauthor=K.+Hirataauthor=X.+Liuauthor=R.+K.+Sunaharaauthor=B.+K.+Shoichetauthor=B.+K.+Kobilkaauthor=P.+Gmeiner&title=Structure-guided+development+of+selective+M3+muscarinic+acetylcholine+receptor+antagonists&doi=10.1073%2Fpnas.1813988115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided development of selective M3 muscarinic acetylcholine receptor antagonists</span></div><div class="casAuthors">Liu, Hongtao; Hofmann, Josefa; Fish, Inbar; Schaake, Benjamin; Eitel, Katrin; Bartuschat, Amelie; Kaindl, Jonas; Rampp, Hannelore; Banerjee, Ashutosh; HA1/4bner, Harald; Clark, Mary J.; Vincent, Sandra G.; Fisher, John T.; Heinrich, Markus R.; Hirata, Kunio; Liu, Xiangyu; Sunahara, Roger K.; Shoichet, Brian K.; Kobilka, Brian K.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12046-12050</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Drugs that treat chronic obstructive pulmonary disease by antagonizing the M3 muscarinic acetylcholine receptor (M3R) have had a significant effect on health, but can suffer from their lack of selectivity against the M2R subtype, which modulates heart rate.  Beginning with the crystal structures of M2R and M3R, we exploited a single amino acid difference in their orthosteric binding pockets using mol. docking and structure-based design.  The resulting M3R antagonists had up to 100-fold selectivity over M2R in affinity and over 1,000-fold selectivity in vivo.  The crystal structure of the M3R-selective antagonist in complex with M3R corresponded closely to the docking-predicted geometry, providing a template for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdR9T0wReSnbVg90H21EOLACvtfcHk0lhk0unkzTaxhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFanu7nE&md5=69c76953444dfbde4a3559507b4f8f4a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1813988115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1813988115%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHofmann%26aufirst%3DJ.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DSchaake%26aufirst%3DB.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DBartuschat%26aufirst%3DA.%26aulast%3DKaindl%26aufirst%3DJ.%26aulast%3DRampp%26aufirst%3DH.%26aulast%3DBanerjee%26aufirst%3DA.%26aulast%3DHubner%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DM.%2BJ.%26aulast%3DVincent%26aufirst%3DS.%2BG.%26aulast%3DFisher%26aufirst%3DJ.%2BT.%26aulast%3DHeinrich%26aufirst%3DM.%2BR.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DStructure-guided%2520development%2520of%2520selective%2520M3%2520muscarinic%2520acetylcholine%2520receptor%2520antagonists%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3D12046%26epage%3D12050%26doi%3D10.1073%2Fpnas.1813988115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawaramoto, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaidehi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into the subtype-selective antagonist binding to the M<sub>2</sub> muscarinic receptor</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1150</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0152-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fs41589-018-0152-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=30420692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Sgur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1150-1158&author=R.+Sunoauthor=S.+Leeauthor=S.+Maedaauthor=S.+Yasudaauthor=K.+Yamashitaauthor=K.+Hirataauthor=S.+Horitaauthor=M.+S.+Tawaramotoauthor=H.+Tsujimotoauthor=T.+Murataauthor=M.+Kinoshitaauthor=M.+Yamamotoauthor=B.+K.+Kobilkaauthor=N.+Vaidehiauthor=S.+Iwataauthor=T.+Kobayashi&title=Structural+insights+into+the+subtype-selective+antagonist+binding+to+the+M2+muscarinic+receptor&doi=10.1038%2Fs41589-018-0152-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the subtype-selective antagonist binding to the M2 muscarinic receptor</span></div><div class="casAuthors">Suno, Ryoji; Lee, Sangbae; Maeda, Shoji; Yasuda, Satoshi; Yamashita, Keitaro; Hirata, Kunio; Horita, Shoichiro; Tawaramoto, Maki S.; Tsujimoto, Hirokazu; Murata, Takeshi; Kinoshita, Masahiro; Yamamoto, Masaki; Kobilka, Brian K.; Vaidehi, Nagarajan; Iwata, So; Kobayashi, Takuya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1150-1158</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Human muscarinic receptor M2 is one of the five subtypes of muscarinic receptors belonging to the family of G-protein-coupled receptors.  Muscarinic receptors are targets for multiple neurodegenerative diseases.  The challenge has been designing subtype-selective ligands against one of the five muscarinic receptors.  We report high-resoln. structures of a thermostabilized mutant M2 receptor bound to a subtype-selective antagonist AF-DX 384 and a nonselective antagonist NMS.  The thermostabilizing mutation S110R in M2 was predicted using a theor. strategy previously developed in our group.  Comparison of the crystal structures and pharmacol. properties of the M2 receptor shows that the Arg in the S110R mutant mimics the stabilizing role of the sodium cation, which is known to allosterically stabilize inactive state(s) of class A GPCRs.  Mol. dynamics simulations reveal that tightening of the ligand-residue contacts in M2 receptors compared to M3 receptors leads to subtype selectivity of AF-DX 384.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd5Qa0G7cW67Vg90H21EOLACvtfcHk0lhk0unkzTaxhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Sgur7N&md5=d99b0d7d43a8b17f6467682a7dd67f1c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0152-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0152-y%26sid%3Dliteratum%253Aachs%26aulast%3DSuno%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DMaeda%26aufirst%3DS.%26aulast%3DYasuda%26aufirst%3DS.%26aulast%3DYamashita%26aufirst%3DK.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DHorita%26aufirst%3DS.%26aulast%3DTawaramoto%26aufirst%3DM.%2BS.%26aulast%3DTsujimoto%26aufirst%3DH.%26aulast%3DMurata%26aufirst%3DT.%26aulast%3DKinoshita%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DVaidehi%26aufirst%3DN.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520the%2520subtype-selective%2520antagonist%2520binding%2520to%2520the%2520M2%2520muscarinic%2520receptor%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D1150%26epage%3D1158%26doi%3D10.1038%2Fs41589-018-0152-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutzler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">Guanidine-acylguanidine bioisosteric approach in the design of radioligands: synthesis of a tritium-labeled N(G)-propionylargininamide ([<sup>3</sup>H]-UR-MK114) as a highly potent and selective neuropeptide Y Y<sub>1</sub> receptor antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">8168</span>– <span class="NLM_lpage">8172</span>, <span class="refDoi"> DOI: 10.1021/jm801018u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801018u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVaht73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=8168-8172&author=M.+Kellerauthor=N.+Popauthor=C.+Hutzlerauthor=A.+G.+Beck-Sickingerauthor=G.+Bernhardtauthor=A.+Buschauer&title=Guanidine-acylguanidine+bioisosteric+approach+in+the+design+of+radioligands%3A+synthesis+of+a+tritium-labeled+N%28G%29-propionylargininamide+%28%5B3H%5D-UR-MK114%29+as+a+highly+potent+and+selective+neuropeptide+Y+Y1+receptor+antagonist&doi=10.1021%2Fjm801018u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Guanidine-Acylguanidine Bioisosteric Approach in the Design of Radioligands: Synthesis of a Tritium-Labeled NG-Propionylargininamide ([3H]-UR-MK114) as a Highly Potent and Selective Neuropeptide Y Y1 Receptor Antagonist</span></div><div class="casAuthors">Keller, Max; Pop, Nathalie; Hutzler, Christoph; Beck-Sickinger, Annette G.; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8168-8172</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthesis and characterization of (R)-Nα-(2,2-diphenylacetyl)-N-(4-hydroxybenzyl)-Nω-([2,3-3H]-propanoyl)argininamide (I, [3H]-UR-MK114), an easily accessible tritium-labeled NPY Y1 receptor (Y1R) antagonist (KB: 0.8 nM, calcium assay, HEL cells) derived from the (R)-argininamide BIBP 3226, is reported.  The radioligand binds with high affinity (KD, satn. : 1.2 nM, kinetic expts.: 1.1 nM, SK-N-MC cells) and selectivity for Y1R over Y2, Y4, and Y5 receptors.  The title compd. is a useful pharmacol. tool for the detn. of Y1R ligand affinities, quantification of Y1R binding sites, and autoradiog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreypW1UupwE7Vg90H21EOLACvtfcHk0lgOrfUSV1aFPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVaht73I&md5=db217c63381985eb5cfa3a51bc37aaa4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm801018u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801018u%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DPop%26aufirst%3DN.%26aulast%3DHutzler%26aufirst%3DC.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DGuanidine-acylguanidine%2520bioisosteric%2520approach%2520in%2520the%2520design%2520of%2520radioligands%253A%2520synthesis%2520of%2520a%2520tritium-labeled%2520N%2528G%2529-propionylargininamide%2520%2528%255B3H%255D-UR-MK114%2529%2520as%2520a%2520highly%2520potent%2520and%2520selective%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D8168%26epage%3D8172%26doi%3D10.1021%2Fjm801018u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The ecstasy and agony of assay interference compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2165</span>– <span class="NLM_lpage">2168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00229</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00229" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2165-2168&author=C.+Aldrichauthor=C.+Bertozziauthor=G.+I.+Georgauthor=L.+Kiesslingauthor=C.+Lindsleyauthor=D.+Liottaauthor=K.+M.+Merzauthor=A.+Schepartzauthor=S.+Wang&title=The+ecstasy+and+agony+of+assay+interference+compounds&doi=10.1021%2Facs.jmedchem.7b00229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The Ecstasy and Agony of Assay Interference Compounds</span></div><div class="casAuthors">Aldrich, Courtney; Bertozzi, Carolyn; Georg, Gunda I.; Kiessling, Laura; Lindsley, Craig; Liotta, Dennis; Merz, Kenneth M.; Schepartz, Alanna; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2165-2168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ecstasy of discovering a new hit from screening can lead to a highly productive research effort to discover new bioactive compds.  However, in too many cases this ecstasy is followed by the agony of realizing that the compds. are not active against the desired target.  Many of these false hits are Pan Assay INterference compds. (PAINS) 1 or colloidal aggregators.  Whether the screen is conducted in silico or in the lab. and whether screening libraries, natural products, or drugs are used, all discovery efforts that rely on some form of screening to identify bioactivity are susceptible to this phenomenon.  Studies that omit crit. controls against exptl. artifacts caused by PAINS may waste years of research effort as useless compds. are progressed.  The American Chem. Society (ACS) is eager to alert the scientific community to this problem and to recommend protocols that will eliminate the publication of research articles based on compds. with artificial activity.  This editorial aims to summarize relevant concepts and to set the framework by which relevant ACS journals will address this issue going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJUf5u51VEpbVg90H21EOLACvtfcHk0liL-MA7iMxQ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D&md5=64bf087659a47fe7769208217ef8c989</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00229%26sid%3Dliteratum%253Aachs%26aulast%3DAldrich%26aufirst%3DC.%26aulast%3DBertozzi%26aufirst%3DC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DKiessling%26aufirst%3DL.%26aulast%3DLindsley%26aufirst%3DC.%26aulast%3DLiotta%26aufirst%3DD.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DSchepartz%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520ecstasy%2520and%2520agony%2520of%2520assay%2520interference%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2165%26epage%3D2168%26doi%3D10.1021%2Facs.jmedchem.7b00229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conklin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farfel, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lustig, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, H. R.</span></span> <span> </span><span class="NLM_article-title">Substitution of three amino acids switches receptor specificity of Gq alpha to that of Gi alpha</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>363</i></span>,  <span class="NLM_fpage">274</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1038/363274a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2F363274a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8387644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1Sltbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=1993&pages=274-276&author=B.+R.+Conklinauthor=Z.+Farfelauthor=K.+D.+Lustigauthor=D.+Juliusauthor=H.+R.+Bourne&title=Substitution+of+three+amino+acids+switches+receptor+specificity+of+Gq+alpha+to+that+of+Gi+alpha&doi=10.1038%2F363274a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Substitution of three amino acids switches receptor specificity of Gqα to that of Giα</span></div><div class="casAuthors">Conklin, Bruce R.; Farfel, Zvi; Lustig, Kevin D.; Julius, David; Bourne, Henry R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">6426</span>),
    <span class="NLM_cas:pages">274-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">C-terminal amino acids of the Gq protein α-subunit were replaced with the corresponding residues of Gi2 protein α-subunit to create αq/αi2 chimeras that can mediate stimulation of phospholipase C by receptors otherwise coupled exclusively to Gi.  A min. of 3 αi2 amino acids, including a glycine 3 residues from the C terminus, suffices to switch the receptor specificity of the αq/αi2 chimeras.  Apparently, a C-terminal turn, centered on this glycine, plays an important part in specifying receptor interactions of G proteins in the Gi/G0/Gz family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq-ZuaQSMBYLVg90H21EOLACvtfcHk0liL-MA7iMxQ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1Sltbk%253D&md5=f2253edf096f40af1d6ae91c4823612f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2F363274a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F363274a0%26sid%3Dliteratum%253Aachs%26aulast%3DConklin%26aufirst%3DB.%2BR.%26aulast%3DFarfel%26aufirst%3DZ.%26aulast%3DLustig%26aufirst%3DK.%2BD.%26aulast%3DJulius%26aufirst%3DD.%26aulast%3DBourne%26aufirst%3DH.%2BR.%26atitle%3DSubstitution%2520of%2520three%2520amino%2520acids%2520switches%2520receptor%2520specificity%2520of%2520Gq%2520alpha%2520to%2520that%2520of%2520Gi%2520alpha%26jtitle%3DNature%26date%3D1993%26volume%3D363%26spage%3D274%26epage%3D276%26doi%3D10.1038%2F363274a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marti-Renom, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Comparative protein structure modeling of genes and genomes</span>. <i>Annu. Rev. Biophys. Biomol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1146/annurev.biophys.29.1.291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1146%2Fannurev.biophys.29.1.291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10940251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2000&pages=291-325&author=M.+A.+Marti-Renomauthor=A.+C.+Stuartauthor=A.+Fiserauthor=R.+Sanchezauthor=F.+Meloauthor=A.+Sali&title=Comparative+protein+structure+modeling+of+genes+and+genomes&doi=10.1146%2Fannurev.biophys.29.1.291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein structure modeling of genes and genomes</span></div><div class="casAuthors">Marti-Renom, Marc A.; Stuart, Ashley C.; Fiser, Andras; Sanchez, Roberto; Melo, Francisco; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">291-325</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 212 refs.  Comparative modeling predicts the three-dimensional structure of a given protein sequence (target) based primarily on its alignment to one or more proteins of known structure (templates).  The prediction process consists of fold assignment, target-template alignment, model building, and model evaluation.  The no. of protein sequences that can be modeled and the accuracy of the predictions are increasing steadily because of the growth in the no. of known protein structures and because of the improvements in the modeling software.  Further advances are necessary in recognizing weak sequence-structure similarities, aligning sequences with structures, modeling of rigid body shifts, distortions, loops and side chains, as well as detecting errors in a model.  Despite these problems, it is currently possible to model with useful accuracy significant parts of approx. one third of all known protein sequences.  The use of individual comparative models in biol. is already rewarding and increasingly widespread.  A major new challenge for comparative modeling is the integration of it with the torrents of data from genome sequencing projects as well as from functional and structural genomics.  In particular, there is a need to develop an automated, rapid, robust, sensitive, and accurate comparative modeling pipeline applicable to whole genomes.  Such large-scale modeling is likely to encourage new kinds of applications for the many resulting models, based on their large no. and completeness at the level of the family, organism, or functional network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq45B6Amw-zi7Vg90H21EOLACvtfcHk0liL-MA7iMxQ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D&md5=d97ffe6c47c4a3bae2bbbf14c045afdd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.29.1.291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.29.1.291%26sid%3Dliteratum%253Aachs%26aulast%3DMarti-Renom%26aufirst%3DM.%2BA.%26aulast%3DStuart%26aufirst%3DA.%2BC.%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMelo%26aufirst%3DF.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520of%2520genes%2520and%2520genomes%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2000%26volume%3D29%26spage%3D291%26epage%3D325%26doi%3D10.1146%2Fannurev.biophys.29.1.291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Comparative protein modelling by satisfaction of spatial restraints</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1006%2Fjmbi.1993.1626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8254673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+Saliauthor=T.+L.+Blundell&title=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints&doi=10.1006%2Fjmbi.1993.1626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein modeling by satisfaction of spatial restraints</span></div><div class="casAuthors">Sali, Andrej; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-815</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe a comparative protein modeling method designed to find the most probable structure for a sequence given its alignment with related structures.  The three-dimensional (3D) model is obtained by optimally satisfying spatial restraints derived from the alignment and expressed as probability d. functions (pdfs) for the features restrained.  For example, the probabilities for main-chain conformations of a modelled residue may be restrained by its residue type, main-chain conformation of an equiv. residue in a related protein, and the local similarity between the two sequences.  Several such pdfs are obtained from the correlations between structural features in 17 families of homologous proteins which have been aligned on the basis of their 3D structures.  The pdfs restrain Cα-Cα distances, main-chain N-O distances, main-chain and side-chain dihedral angles.  A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database.  The 3D model of a protein is obtained by optimization of the mol. pdf such that the model violates the input restraints as little as possible.  The mol. pdf is derived as a combination of pdfs restraining individual spatial features of the whole mol.  The optimization procedure is a variable target function method that applies the conjugate gradients algorithm to positions of all non-hydrogen atoms.  The method is automated and is illustrated by the modeling of trypsin from two other serine proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC4wGhwZvib7Vg90H21EOLACvtfcHk0liL-MA7iMxQ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D&md5=d4a3c39b2205e36221dc187a3d1a478b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1626%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modelling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815%26doi%3D10.1006%2Fjmbi.1993.1626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Modeling of loops in protein structures</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1753</span>– <span class="NLM_lpage">1773</span>, <span class="refDoi"> DOI: 10.1110/ps.9.9.1753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1110%2Fps.9.9.1753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=11045621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2000&pages=1753-1773&author=A.+Fiserauthor=R.+K.+Doauthor=A.+Sali&title=Modeling+of+loops+in+protein+structures&doi=10.1110%2Fps.9.9.1753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling of loops in protein structures</span></div><div class="casAuthors">Fiser, Andras; Do, Richard Kinh Gian; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1753-1773</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Comparative protein structure prediction is limited mostly by the errors in alignment and loop modeling.  We describe here a new automated modeling technique that significantly improves the accuracy of loop predictions in protein structures.  The positions of all nonhydrogen atoms of the loop are optimized in a fixed environment with respect to a pseudo energy function.  The energy is a sum of many spatial restraints that include the bond length, bond angle, and improper dihedral angle terms from the CHARMM-22 force field, statistical preferences for the main-chain and side-chain dihedral angles, and statistical preferences for nonbonded at. contacts that depend on the two atom types, their distance through space, and sepn. in sequence.  The energy function is optimized with the method of conjugate gradients combined with mol. dynamics and simulated annealing.  Typically, the predicted loop conformation corresponds to the lowest energy conformation among 500 independent optimizations.  Predictions were made for 40 loops of known structure at each length from 1 to 14 residues.  The accuracy of loop predictions is evaluated as a function of thoroughness of conformational sampling, loop length, and structural properties of native loops.  When accuracy is measured by local superposition of the model on the native loop, 100, 90, and 30% of 4-, 8-, and 12-residue loop predictions, resp., had <2 Å RMSD error for the mainchain N, Cα, C, and O atoms; the av. accuracies were 0.59±0.05, 1.16±0.10, and 2.61±0.16 Å, resp.  To simulate real comparative modeling problems, the method was also evaluated by predicting loops of known structure in only approx. correct environments with errors typical of comparative modeling without misalignment.  When the RMSD distortion of the main-chain stem atoms is 2.5 Å, the av. loop prediction error increased by 180, 25, and 3% for 4-, 8-, and 12-residue loops, resp.  The accuracy of the lowest energy prediction for a given loop can be estd. from the structural variability among a no. of low energy predictions.  The relative value of the present method is gauged by (1) comparing it with one of the most successful previously described methods, and (2) describing its accuracy in recent blind predictions of protein structure.  Finally, it is shown that the av. accuracy of prediction is limited primarily by the accuracy of the energy function rather than by the extent of conformational sampling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsbG2px_r_krVg90H21EOLACvtfcHk0ljVMB0FCHcwFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D&md5=0bcdbfbddf54ecca0e3fee95ecdc3f8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1110%2Fps.9.9.1753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.9.9.1753%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DR.%2BK.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModeling%2520of%2520loops%2520in%2520protein%2520structures%26jtitle%3DProtein%2520Sci.%26date%3D2000%26volume%3D9%26spage%3D1753%26epage%3D1773%26doi%3D10.1110%2Fps.9.9.1753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanborn, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husbands, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynor, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structural insights into micro-opioid receptor activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>524</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1038/nature14886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature14886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=26245379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=524&publication_year=2015&pages=315-321&author=W.+Huangauthor=A.+Manglikauthor=A.+J.+Venkatakrishnanauthor=T.+Laeremansauthor=E.+N.+Feinbergauthor=A.+L.+Sanbornauthor=H.+E.+Katoauthor=K.+E.+Livingstonauthor=T.+S.+Thorsenauthor=R.+C.+Klingauthor=S.+Granierauthor=P.+Gmeinerauthor=S.+M.+Husbandsauthor=J.+R.+Traynorauthor=W.+I.+Weisauthor=J.+Steyaertauthor=R.+O.+Drorauthor=B.+K.+Kobilka&title=Structural+insights+into+micro-opioid+receptor+activation&doi=10.1038%2Fnature14886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into μ-opioid receptor activation</span></div><div class="casAuthors">Huang, Weijiao; Manglik, Aashish; Venkatakrishnan, A. J.; Laeremans, Toon; Feinberg, Evan N.; Sanborn, Adrian L.; Kato, Hideaki E.; Livingston, Kathryn E.; Thorsen, Thor S.; Kling, Ralf C.; Granier, Sebastien; Gmeiner, Peter; Husbands, Stephen M.; Traynor, John R.; Weis, William I.; Steyaert, Jan; Dror, Ron O.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">524</span>
        (<span class="NLM_cas:issue">7565</span>),
    <span class="NLM_cas:pages">315-321</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the μ-opioid receptor (μOR) is responsible for the efficacy of the most effective analgesics.  To shed light on the structural basis for μOR activation, here we report a 2.1 Å X-ray crystal structure of the murine μOR bound to the morphinan agonist BU72 and a G protein mimetic camelid antibody fragment Nb39 (nanobody 39).  The BU72-stabilized changes in the μOR binding pocket are subtle and differ from those obsd. for agonist-bound structures of the β2-adrenergic receptor (β2AR) and the M2 muscarinic receptor (M2R).  Comparison with active β2AR reveals a common rearrangement in the packing of three conserved amino acids in the core of the μOR, and mol. dynamics simulations illustrate how the ligand-binding pocket is conformationally linked to this conserved triad.  Addnl., an extensive polar network between the ligand-binding pocket and the cytoplasmic domains appears to play a similar role in signal propagation for all three G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkFUN1wheV07Vg90H21EOLACvtfcHk0ljVMB0FCHcwFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM&md5=b9af3e1351b04539379532c127c02651</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature14886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14886%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DVenkatakrishnan%26aufirst%3DA.%2BJ.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DFeinberg%26aufirst%3DE.%2BN.%26aulast%3DSanborn%26aufirst%3DA.%2BL.%26aulast%3DKato%26aufirst%3DH.%2BE.%26aulast%3DLivingston%26aufirst%3DK.%2BE.%26aulast%3DThorsen%26aufirst%3DT.%2BS.%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DGranier%26aufirst%3DS.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DHusbands%26aufirst%3DS.%2BM.%26aulast%3DTraynor%26aufirst%3DJ.%2BR.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructural%2520insights%2520into%2520micro-opioid%2520receptor%2520activation%26jtitle%3DNature%26date%3D2015%26volume%3D524%26spage%3D315%26epage%3D321%26doi%3D10.1038%2Fnature14886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lomize, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomize, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogozheva, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosberg, H. I.</span></span> <span> </span><span class="NLM_article-title">OPM: orientations of proteins in membranes database</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">625</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btk023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1093%2Fbioinformatics%2Fbtk023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=16397007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=623-625&author=M.+A.+Lomizeauthor=A.+L.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+I.+Mosberg&title=OPM%3A+orientations+of+proteins+in+membranes+database&doi=10.1093%2Fbioinformatics%2Fbtk023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">OPM: Orientations of Proteins in Membranes database</span></div><div class="casAuthors">Lomize, Mikhail A.; Lomize, Andrei L.; Pogozheva, Irina D.; Mosberg, Henry I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">623-625</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Orientations of Proteins in Membranes (OPM) database provides a collection of transmembrane, monotopic, and peripheral proteins from the Protein Data Bank whose spatial arrangements in the lipid bilayer have been calcd. theor. and compared with exptl. data.  The database allows anal., sorting, and searching of membrane proteins based on their structural classification, species, destination membrane, nos. of transmembrane segments and subunits, nos. of secondary structures, and the calcd. hydrophobic thickness or tilt angle with respect to the bilayer normal.  All coordinate files with the calcd. membrane boundaries are available for downloading.  The Internet site for the database is: http://opm.phar.umich.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1WjpHiq7e8LVg90H21EOLACvtfcHk0li2nNlsGP6Xxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D&md5=8a9590a32e912509a53ca0bfc7ede4ad</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtk023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtk023%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DM.%2BA.%26aulast%3DLomize%26aufirst%3DA.%2BL.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26atitle%3DOPM%253A%2520orientations%2520of%2520proteins%2520in%2520membranes%2520database%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3D623%26epage%3D625%26doi%3D10.1093%2Fbioinformatics%2Fbtk023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 2. explicit solvent particle mesh Ewald</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3878</span>– <span class="NLM_lpage">3888</span>, <span class="refDoi"> DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3878-3888&author=R.+Salomon-Ferrerauthor=A.+W.+Gotzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+2.+explicit+solvent+particle+mesh+Ewald&doi=10.1021%2Fct400314y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0li2nNlsGP6Xxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DGotz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520explicit%2520solvent%2520particle%2520mesh%2520Ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">SPFP: Speed without compromise - a mixed precision model for GPU accelerated molecular dynamics simulations</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">374</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2013&pages=374-380&author=S.+Le%0AGrandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&title=SPFP%3A+Speed+without+compromise+-+a+mixed+precision+model+for+GPU+accelerated+molecular+dynamics+simulations&doi=10.1016%2Fj.cpc.2012.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0lhUBuagoICcmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520compromise%2520-%2520a%2520mixed%2520precision%2520model%2520for%2520GPU%2520accelerated%2520molecular%2520dynamics%2520simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+an+N%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lhUBuagoICcmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520an%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lhUBuagoICcmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A. E.</span></span> <span> </span><span class="NLM_article-title">Long-time-step molecular dynamics through hydrogen mass repartitioning</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1864</span>– <span class="NLM_lpage">1874</span>, <span class="refDoi"> DOI: 10.1021/ct5010406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5010406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1864-1874&author=C.+W.+Hopkinsauthor=S.+Le+Grandauthor=R.+C.+Walkerauthor=A.+E.+Roitberg&title=Long-time-step+molecular+dynamics+through+hydrogen+mass+repartitioning&doi=10.1021%2Fct5010406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Long-Time-Step Molecular Dynamics through Hydrogen Mass Repartitioning</span></div><div class="casAuthors">Hopkins, Chad W.; Le Grand, Scott; Walker, Ross C.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1864-1874</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies have shown that the method of hydrogen mass repartitioning (HMR) is a potentially useful tool for accelerating mol. dynamics (MD) simulations.  By repartitioning the mass of heavy atoms into the bonded hydrogen atoms, it is possible to slow the highest-frequency motions of the macromol. under study, thus allowing the time step of the simulation to be increased by up to a factor of 2.  In this communication, we investigate further how this mass repartitioning allows the simulation time step to be increased in a stable fashion without significantly increasing discretization error.  To this end, we ran a set of simulations with different time steps and mass distributions on a three-residue peptide to get a comprehensive view of the effect of mass repartitioning and time step increase on a system whose accessible phase space is fully explored in a relatively short amt. of time.  We next studied a 129-residue protein, hen egg white lysozyme (HEWL), to verify that the obsd. behavior extends to a larger, more-realistic, system.  Results for the protein include structural comparisons from MD trajectories, as well as comparisons of pKa calcns. via const.-pH MD.  We also calcd. a potential of mean force (PMF) of a dihedral rotation for the MTS [(1-oxyl-2,2,5,5-tetramethyl-pyrroline-3-methyl)methanethiosulfonate] spin label via umbrella sampling with a set of regular MD trajectories, as well as a set of mass-repartitioned trajectories with a time step of 4 fs.  Since no significant difference in kinetics or thermodn. is obsd. by the use of fast HMR trajectories, further evidence is provided that long-time-step HMR MD simulations are a viable tool for accelerating MD simulations for mols. of biochem. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfG1v1aGj1NrVg90H21EOLACvtfcHk0lj8fN3KIcqddg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D&md5=6ca75b544b63523481de7836d11a3c1b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct5010406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5010406%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DC.%2BW.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DLong-time-step%2520molecular%2520dynamics%2520through%2520hydrogen%2520mass%2520repartitioning%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D1864%26epage%3D1874%26doi%3D10.1021%2Fct5010406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chow, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D. M.</span></span> <span> </span><span class="NLM_article-title">Isothermal-isobaric molecular dynamics simulations with Monte Carlo volume sampling</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1016/0010-4655(95)00059-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0010-4655%2895%2900059-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2MXps1Wrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1995&pages=283-289&author=K.-H.+Chowauthor=D.+M.+Ferguson&title=Isothermal-isobaric+molecular+dynamics+simulations+with+Monte+Carlo+volume+sampling&doi=10.1016%2F0010-4655%2895%2900059-O"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Isothermal-isobaric molecular dynamics simulations with Monte Carlo volume sampling</span></div><div class="casAuthors">Chow, Kim-Hung; Ferguson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">283-9</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A new pressure coupling method is described that combines Monte Carlo vol.-space sampling with traditional mol. dynamics calcns. to simulate the phys. properties of mol. systems under std. conditions.  The pressure is maintained by accepting or rejecting vol. moves of newly propagated configurations using the Metropolis algorithm with probability P(ΔV) = min(1, exp((-1/kT0){ΔE+P0ΔV-NkT0 ln[(V+ΔV)/V]})).  The sample is further coupled to an external temp. bath using a variation of Andersen's stochastic collisions.  The computational approach was implemented and applied to simulate the equil. properties of dense neon at various pressures.  The results show the method to be both reliable and efficient for maintaining isothermal-isobaric conditions and should prove effective for studying the pressure dependence of phys. properties in mol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqBZv_rh25XbVg90H21EOLACvtfcHk0lj8fN3KIcqddg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXps1Wrt7s%253D&md5=7c04ed3236a63fbadd0d0acb4e9d42ac</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2F0010-4655%2895%2900059-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0010-4655%252895%252900059-O%26sid%3Dliteratum%253Aachs%26aulast%3DChow%26aufirst%3DK.-H.%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26atitle%3DIsothermal-isobaric%2520molecular%2520dynamics%2520simulations%2520with%2520Monte%2520Carlo%2520volume%2520sampling%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D1995%26volume%3D91%26spage%3D283%26epage%3D289%26doi%3D10.1016%2F0010-4655%2895%2900059-O" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salentin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupt, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adasme, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, M.</span></span> <span> </span><span class="NLM_article-title">PLIP: fully automated protein-ligand interaction profiler</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">W443</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1093%2Fnar%2Fgkv315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=25873628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W443-447&author=S.+Salentinauthor=S.+Schreiberauthor=V.+J.+Hauptauthor=M.+F.+Adasmeauthor=M.+Schroeder&title=PLIP%3A+fully+automated+protein-ligand+interaction+profiler&doi=10.1093%2Fnar%2Fgkv315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PLIP: fully automated protein-ligand interaction profiler</span></div><div class="casAuthors">Salentin, Sebastian; Schreiber, Sven; Haupt, V. Joachim; Adasme, Melissa F.; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W443-W447</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The characterization of interactions in protein-ligand complexes is essential for research in structural bioinformatics, drug discovery and biol.  However, comprehensive tools are not freely available to the research community.  Here, we present the protein-ligand interaction profiler (PLIP), a novel web service for fully automated detection and visualization of relevant non-covalent protein-ligand contacts in 3D structures, freely available at projects.biotec.tu-dresden.de/plip-web.  The input is either a Protein Data Bank structure, a protein or ligand name, or a custom protein-ligand complex (e.g. from docking).  In contrast to other tools, the rule-based PLIP algorithm does not require any structure prepn.  It returns a list of detected interactions on single atom level, covering seven interaction types (hydrogen bonds, hydrophobic contacts, pi-stacking, pi-cation interactions, salt bridges, water bridges and halogen bonds).  PLIP stands out by offering publication-ready images, PyMOL sesions files to generate custom images and parsable result files to facilitate successive data processing.  The full python source code is available for download on the website.  PLIP's command-line mode allows for high-throughput interaction profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgMd-_C9BUCbVg90H21EOLACvtfcHk0lj8fN3KIcqddg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbrI&md5=4878881a7bd715e5585682876e102e65</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv315%26sid%3Dliteratum%253Aachs%26aulast%3DSalentin%26aufirst%3DS.%26aulast%3DSchreiber%26aufirst%3DS.%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DAdasme%26aufirst%3DM.%2BF.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DPLIP%253A%2520fully%2520automated%2520protein-ligand%2520interaction%2520profiler%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3DW443%26epage%3D447%26doi%3D10.1093%2Fnar%2Fgkv315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the inhibition constant (<i>K</i><sub>I</sub>) and the concentration of inhibitor which causes 50 per cent inhibition (<i>I</i><sub>50</sub>) of an enzymic reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0liCiGpEJzSecQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 7 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xueke She, Andrea Pegoli, Corinna G. Gruber, David Wifling, Jessica Carpenter, Harald Hübner, Mengya Chen, Jianfei Wan, Günther Bernhardt, Peter Gmeiner, Nicholas D. Holliday, <span class="NLM_string-name hlFld-ContribAuthor">Max Keller</span>. </span><span class="cited-content_cbyCitation_article-title">Red-Emitting Dibenzodiazepinone Derivatives as Fluorescent Dualsteric Probes for the Muscarinic Acetylcholine M2 Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (8)
                                     , 4133-4154. <a href="https://doi.org/10.1021/acs.jmedchem.9b02172" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02172</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02172%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRed-Emitting%252BDibenzodiazepinone%252BDerivatives%252Bas%252BFluorescent%252BDualsteric%252BProbes%252Bfor%252Bthe%252BMuscarinic%252BAcetylcholine%252BM2%252BReceptor%26aulast%3DShe%26aufirst%3DXueke%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D29122019%26date%3D13042020%26date%3D01042020%26volume%3D63%26issue%3D8%26spage%3D4133%26epage%3D4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Corinna. G.  Weinhart</span>, <span class="hlFld-ContribAuthor ">David  Wifling</span>, <span class="hlFld-ContribAuthor ">Maximilian. F.  Schmidt</span>, <span class="hlFld-ContribAuthor ">Eduard  Neu</span>, <span class="hlFld-ContribAuthor ">Carina  Höring</span>, <span class="hlFld-ContribAuthor ">Timothy  Clark</span>, <span class="hlFld-ContribAuthor ">Peter  Gmeiner</span>, <span class="hlFld-ContribAuthor ">Max  Keller</span>. </span><span class="cited-content_cbyCitation_article-title">Dibenzodiazepinone-type muscarinic receptor antagonists conjugated to basic peptides: Impact of the linker moiety and unnatural amino acids on M2R selectivity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>213 </em>, 113159. <a href="https://doi.org/10.1016/j.ejmech.2021.113159" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113159</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113159%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDibenzodiazepinone-type%252Bmuscarinic%252Breceptor%252Bantagonists%252Bconjugated%252Bto%252Bbasic%252Bpeptides%25253A%252BImpact%252Bof%252Bthe%252Blinker%252Bmoiety%252Band%252Bunnatural%252Bamino%252Bacids%252Bon%252BM2R%252Bselectivity%26aulast%3DWeinhart%26aufirst%3DCorinna.%2BG.%26date%3D2021%26volume%3D213%26spage%3D113159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frauke  Antoni</span>, <span class="hlFld-ContribAuthor ">David  Wifling</span>, <span class="hlFld-ContribAuthor ">Günther  Bernhardt</span>. </span><span class="cited-content_cbyCitation_article-title">Water-soluble inhibitors of ABCG2 (BCRP) – A fragment-based and computational approach. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112958. <a href="https://doi.org/10.1016/j.ejmech.2020.112958" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112958</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112958%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DWater-soluble%252Binhibitors%252Bof%252BABCG2%252B%252528BCRP%252529%252B%2525E2%252580%252593%252BA%252Bfragment-based%252Band%252Bcomputational%252Bapproach%26aulast%3DAntoni%26aufirst%3DFrauke%26date%3D2021%26volume%3D210%26spage%3D112958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Corinna G.  Gruber</span>, <span class="hlFld-ContribAuthor ">Andrea  Pegoli</span>, <span class="hlFld-ContribAuthor ">Christoph  Müller</span>, <span class="hlFld-ContribAuthor ">Lukas  Grätz</span>, <span class="hlFld-ContribAuthor ">Xueke  She</span>, <span class="hlFld-ContribAuthor ">Max  Keller</span>. </span><span class="cited-content_cbyCitation_article-title">Differently fluorescence-labelled dibenzodiazepinone-type muscarinic acetylcholine receptor ligands with high M
              2
              R affinity. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (7)
                                     , 823-832. <a href="https://doi.org/10.1039/D0MD00137F" title="DOI URL">https://doi.org/10.1039/D0MD00137F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00137F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00137F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DDifferently%252Bfluorescence-labelled%252Bdibenzodiazepinone-type%252Bmuscarinic%252Bacetylcholine%252Breceptor%252Bligands%252Bwith%252Bhigh%252BM%252B2%252BR%252Baffinity%26aulast%3DGruber%26aufirst%3DCorinna%2BG.%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D7%26spage%3D823%26epage%3D832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonas  Buschmann</span>, <span class="hlFld-ContribAuthor ">Theresa  Seiler</span>, <span class="hlFld-ContribAuthor ">Günther  Bernhardt</span>, <span class="hlFld-ContribAuthor ">Max  Keller</span>, <span class="hlFld-ContribAuthor ">David  Wifling</span>. </span><span class="cited-content_cbyCitation_article-title">Argininamide-type neuropeptide Y Y
              1
              receptor antagonists: the nature of
              N
              ω
              -carbamoyl substituents determines Y
              1
              R binding mode and affinity. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (2)
                                     , 274-282. <a href="https://doi.org/10.1039/C9MD00538B" title="DOI URL">https://doi.org/10.1039/C9MD00538B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00538B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00538B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DArgininamide-type%252Bneuropeptide%252BY%252BY%252B1%252Breceptor%252Bantagonists%25253A%252Bthe%252Bnature%252Bof%252BN%252B%2525CF%252589%252B-carbamoyl%252Bsubstituents%252Bdetermines%252BY%252B1%252BR%252Bbinding%252Bmode%252Band%252Baffinity%26aulast%3DBuschmann%26aufirst%3DJonas%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D2%26spage%3D274%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Wifling</span>, <span class="hlFld-ContribAuthor ">Christopher  Pfleger</span>, <span class="hlFld-ContribAuthor ">Jonas  Kaindl</span>, <span class="hlFld-ContribAuthor ">Passainte  Ibrahim</span>, <span class="hlFld-ContribAuthor ">Ralf C.  Kling</span>, <span class="hlFld-ContribAuthor ">Armin  Buschauer</span>, <span class="hlFld-ContribAuthor ">Holger  Gohlke</span>, <span class="hlFld-ContribAuthor ">Timothy  Clark</span>. </span><span class="cited-content_cbyCitation_article-title">Basal Histamine H
              4
              Receptor Activation: Agonist Mimicry by the Diphenylalanine Motif. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2019,</strong> <em>25 </em>
                                    (64)
                                     , 14613-14624. <a href="https://doi.org/10.1002/chem.201902801" title="DOI URL">https://doi.org/10.1002/chem.201902801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201902801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201902801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DBasal%252BHistamine%252BH%252B4%252BReceptor%252BActivation%25253A%252BAgonist%252BMimicry%252Bby%252Bthe%252BDiphenylalanine%252BMotif%26aulast%3DWifling%26aufirst%3DDavid%26date%3D2019%26date%3D2019%26volume%3D25%26issue%3D64%26spage%3D14613%26epage%3D14624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  He</span>, <span class="hlFld-ContribAuthor ">Zhi  Zheng</span>, <span class="hlFld-ContribAuthor ">Yajie  Liu</span>, <span class="hlFld-ContribAuthor ">Jiajie  Qiao</span>, <span class="hlFld-ContribAuthor ">Xinying  Zhang</span>, <span class="hlFld-ContribAuthor ">Xuesen  Fan</span>. </span><span class="cited-content_cbyCitation_article-title">Selective synthesis of β-nitrated N-heterocycles and
              N
              -nitroso-2-alkoxyamine aldehydes from inactivated cyclic amines promoted by
              t
              BuONO and oxoammonium salt. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (82)
                                     , 12372-12375. <a href="https://doi.org/10.1039/C9CC05963F" title="DOI URL">https://doi.org/10.1039/C9CC05963F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CC05963F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CC05963F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DSelective%252Bsynthesis%252Bof%252B%2525CE%2525B2-nitrated%252BN-heterocycles%252Band%252BN%252B-nitroso-2-alkoxyamine%252Baldehydes%252Bfrom%252Binactivated%252Bcyclic%252Bamines%252Bpromoted%252Bby%252Bt%252BBuONO%252Band%252Boxoammonium%252Bsalt%26aulast%3DHe%26aufirst%3DYan%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D82%26spage%3D12372%26epage%3D12375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures and MR binding data of two described diarylsulfone-type M<sub>2</sub>R-selective MR ligands <b>(1a</b>, <b>1b</b>) and of a selection of reported M<sub>2</sub>R-preferring tricyclic MR antagonists (compounds <b>2</b>–<b>10</b>). In the table, references are as follows: (a) Clader et al. (reported <i>K</i><sub>i</sub> values (M<sub>2</sub>R) were converted to p<i>K</i><sub>i</sub> values; IC<sub>50</sub>, pIC<sub>50</sub>, <i>K</i><sub>i</sub>, or p<i>K</i><sub>i</sub> values for the MR subtypes M<sub>1</sub>, M<sub>3</sub>, M<sub>4</sub>, and M<sub>5</sub> were not given; instead, only the ratios <i>K</i><sub>i</sub>(M<sub>1,3,4,5</sub>R)/<i>K</i><sub>i</sub>(M<sub>2</sub>R) were provided);<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> (b) Dörje et al.;<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (c) Keller et al. (data reported as pIC<sub>50</sub>; data were reanalyzed to obtain p<i>K</i><sub>i</sub> values);<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (d) Doods et al.;<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (e) Doods et al.;<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (f) Maggio et al. (reported <i>K</i><sub>i</sub> values were converted to p<i>K</i><sub>i</sub> values);<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (g) Gitler et al. (reported <i>K</i><sub>i</sub> values were converted to p<i>K</i><sub>i</sub> values);<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (h) She et al.;<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (i) Pegoli et al.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Except for <b>5</b>, data refer to human receptors. In the case of <b>5</b>, M<sub>2</sub> and M<sub>4</sub> were rat receptors, whereas M<sub>1</sub> and M<sub>3</sub> receptors were not specified.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Rationale of the present study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0006.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of DIBA–Peptide Conjugates <b>28</b>–<b>53</b> and Dibenzodiazepinone Derivatives <b>55</b>, <b>56</b>, and <b>58</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions. (a) Fmoc-strategy SPPS, amino acid coupling: Fmoc-aa, HBTU, HOBt, DIPEA, DMF/NMP (8:2 v/v), rt or 35 °C, 2 × 60 min (“double coupling”). Fmoc deprotection: 20% piperidine in DMF/NMP (8:2 v/v), rt, 2 × 10 min. (b) (1) EDC, HOBt, DIPEA, DMF, 5 °C, 3 h, (2) TFA/H<sub>2</sub>O (95:5 v/v), rt, 3 h, 27–66%; (c) (1) EDC, HOBt, DMF, 5 °C, 3 h, (2) TFA/H<sub>2</sub>O (95:5 v/v), rt, 3 h, 30–75%; (d) acetic anhydride, DIPEA, DMF, rt, 2 h, 78%; (e) DIPEA, DMF, rt, 30 min, 80%.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Investigation of M<sub>2</sub>R agonism and antagonism of compounds <b>38</b> and <b>48</b> in an IP1 accumulation assay using HEK-hM<sub>2</sub>-Gα<sub>qi5-HA</sub> cells. (A) Concentration-dependent effects of CCh, <b>38</b>, and <b>48</b> on the accumulation of IP1. <b>38</b> and <b>48</b> elicited no response. pEC<sub>50</sub> of CCh (mean ± SEM) is 6.97 ± 0.03. Data represent mean values ± SEM from 14 (CCh) or 4 (<b>38</b>, <b>48</b>) independent experiments (performed in duplicate). (B) Concentration-dependent inhibition of the CCh (0.3 μM) induced IP1 accumulation by the MR antagonists atropine, <b>38</b>, and <b>48</b>. pIC<sub>50</sub> values (mean ± SEM) are the following: <b>38</b>, 7.98 ± 0.19; <b>48</b>, 7.03 ± 0.06; atropine, 8.09 ± 0.03. Data represent mean values ± SEM from four (<b>38</b>), five (atropine), or six (<b>48</b>) independent experiments (performed in duplicate).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. MD simulations (20 μs) of the hM<sub>2</sub>R (inactive state, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON">3UON</a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>) bound to <b>38</b> (A, C) or <b>48</b> (B, D). (A, B) Lowest free energy (MM-PBSA) poses of <b>38</b> (green) and <b>48</b> (purple), respectively, obtained from analysis of the MD simulation trajectories. Amino acids involved in H-bonding, salt bridges (both indicated as yellow dashed lines), or π–π interactions (green dashed lines) with the ligands are labeled: in (A), D103<sup>3.32</sup> (HB, SB), E172<sup>ECL2</sup> (HB, SB), E175<sup>ECL2</sup> (HB, SB), C176<sup>ECL2</sup> (HB), I178<sup>ECL2</sup> (HB), Y403<sup>6.51</sup> (π–π), N404<sup>6.52</sup> (HB), N419<sup>ECL3</sup> (HB), and Y426<sup>7.39</sup> (HB); in (B), Y80<sup>2.61</sup> (HB), D103<sup>3.32</sup> (HB, SB), Y104<sup>3.33</sup> (HB), E172<sup>ECL2</sup> (HB, SB), E175<sup>ECL2</sup> (HB, SB), I178<sup>ECL2</sup> (HB). (C, D) Time courses of the 20 μs MD simulations of the hM<sub>2</sub>R bound to <b>38</b> (C) or <b>48</b> (D) showing superimposed snapshots collected every 1 μs. HB = hydrogen bond. SB = salt bridge.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/medium/jm-2018-01967s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of the DIBA–peptide conjugates <b>38</b> and <b>48</b> on M<sub>2</sub>R equilibrium binding of [<sup>3</sup>H]NMS (<i>K</i><sub>d</sub> = 0.09 nM) using a radioligand concentration of 0.2 nM (filled symbols, data taken from <a href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf" class="ext-link">Figures S2 and S4</a>) or 2 nM (open symbols). Experiments were performed at intact CHO-hM<sub>2</sub>R cells at 22 °C. Unspecific binding was determined in the presence of atropine (500-fold excess to [<sup>3</sup>H]NMS). Data, representing mean values ± SEM from at least four independent experiments (each performed in triplicate), were analyzed by four-parameter logistic fits. The use of a high radioligand concentration, resulting in a high receptor occupancy by [<sup>3</sup>H]NMS, can “unmask” a potential noncompetitive, negative allosteric mechanism as previously demonstrated, e.g., for the negative allosteric M<sub>2</sub>R modulator W84 (indicated by an elevated lower curve plateau and a missing rightward shift of the curves; cf. Figure 8C in Pegoli et al., 2017<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>). Here, the determination of the effects of <b>38</b> and <b>48</b> on M<sub>2</sub>R equilibrium binding of [<sup>3</sup>H]NMS yielded curves that were rightward shifted and exhibited lower curve plateaus not different from zero. This indicates a competitive mechanism between the orthosteric MR antagonist NMS and <b>38</b> as well as NMS and <b>48</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-11/acs.jmedchem.8b01967/20190607/images/large/jm-2018-01967s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01967&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25651" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25651" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span> <span> </span><span class="NLM_article-title">Overview of muscarinic receptor subtypes</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1007/978-3-642-23274-9_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1007%2F978-3-642-23274-9_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChsb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2012&pages=3-28&author=R.+M.+Eglen&title=Overview+of+muscarinic+receptor+subtypes&doi=10.1007%2F978-3-642-23274-9_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of muscarinic receptor subtypes</span></div><div class="casAuthors">Eglen, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue">Muscarinic Receptors</span>),
    <span class="NLM_cas:pages">3-28</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The physiol. role of muscarinic receptors is highly complex and, although not completely understood, has become clearer over the last decade.  Recent pharmacol. evidence with novel compds., together with data from transgenic mice, suggests that all five subtypes have defined functions in the nervous system as well as mediating the non neuronal, hormonal actions of acetylcholine.  Numerous novel agonists, allosteric regulators, and antagonists have now been identified with authentic subtype specificity in vitro and in vivo.  These compds. provide addnl. pharmacol. opportunities for selective subtype modulation as well as a new generation of muscarinic receptor-based therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGsQ_5yP8Tg7Vg90H21EOLACvtfcHk0lgci0e1EYISWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChsb7K&md5=e9acb43ac95866f476329d7f0b119c33</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-23274-9_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-642-23274-9_1%26sid%3Dliteratum%253Aachs%26aulast%3DEglen%26aufirst%3DR.%2BM.%26atitle%3DOverview%2520of%2520muscarinic%2520receptor%2520subtypes%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2012%26volume%3D208%26spage%3D3%26epage%3D28%26doi%3D10.1007%2F978-3-642-23274-9_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Allosteric modulation of muscarinic acetylcholine receptors</span>. <i>Curr. Neuropharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.2174/157015907781695946</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.2174%2F157015907781695946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=19305798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKqsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=157-167&author=K.+J.+Gregoryauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=Allosteric+modulation+of+muscarinic+acetylcholine+receptors&doi=10.2174%2F157015907781695946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of muscarinic acetylcholine receptors</span></div><div class="casAuthors">Gregory, Karen J.; Sexton, Patrick M.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Current Neuropharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">CNUEAN</span>;
        ISSN:<span class="NLM_cas:issn">1570-159X</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Muscarinic acetylcholine receptors (mAChRs) are prototypical Family A G protein coupled-receptors.  The five mAChR subtypes are widespread throughout the periphery and the central nervous system and, accordingly, are widely involved in a variety of both physiol. and pathophysiol. processes.  There currently remains an unmet need for better therapeutic agents that can selectively target a given mAChR subtype to the relative exclusion of others.  The main reason for the lack of such selective mAChR ligands is the high sequence homol. within the acetylcholine-binding site (orthosteric site) across all mAChRs.  However, the mAChRs possess at least one, and likely two, extracellular allosteric binding sites that can recognize small mol. allosteric modulators to regulate the binding and function of orthosteric ligands.  Extensive studies of prototypical mAChR modulators, such as gallamine and alcuronium, have provided strong pharmacol. evidence, and assocd. structure-activity relationships (SAR), for a "common" allosteric site on all five mAChRs.  These studies are also supported by mutagenesis expts. implicating the second extracellular loop and the interface between the third extracellular loop and the top of transmembrane domain 7 as contributing to the common allosteric site.  Other studies are also delineating the pharmacol. of a second allosteric site, recognized by compds. such as staurosporine.  In addn., allosteric agonists, such as McN-A-343, AC-42 and N-desmethylclozapine, have also been identified.  Current challenges to the field include the ability to effectively detect and validate allosteric mechanisms, and to quantify allosteric effects on binding affinity and signaling efficacy to inform allosteric modulator SAR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMjqjcw6KlILVg90H21EOLACvtfcHk0lhgGxZzA_VMjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKqsLnP&md5=a30c0102440ffbdc1fc6efe9da084521</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F157015907781695946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157015907781695946%26sid%3Dliteratum%253Aachs%26aulast%3DGregory%26aufirst%3DK.%2BJ.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAllosteric%2520modulation%2520of%2520muscarinic%2520acetylcholine%2520receptors%26jtitle%3DCurr.%2520Neuropharmacol.%26date%3D2007%26volume%3D5%26spage%3D157%26epage%3D167%26doi%3D10.2174%2F157015907781695946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawaratne, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bures, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachhawat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of the M<sub>1</sub> and M<sub>4</sub> muscarinic acetylcholine receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>531</i></span>,  <span class="NLM_fpage">335</span>– <span class="NLM_lpage">340</span>, <span class="refDoi"> DOI: 10.1038/nature17188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature17188" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=26958838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC28XktlGjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=531&publication_year=2016&pages=335-340&author=D.+M.+Thalauthor=B.+Sunauthor=D.+Fengauthor=V.+Nawaratneauthor=K.+Leachauthor=C.+C.+Felderauthor=M.+G.+Buresauthor=D.+A.+Evansauthor=W.+I.+Weisauthor=P.+Bachhawatauthor=T.+S.+Kobilkaauthor=P.+M.+Sextonauthor=B.+K.+Kobilkaauthor=A.+Christopoulos&title=Crystal+structures+of+the+M1+and+M4+muscarinic+acetylcholine+receptors&doi=10.1038%2Fnature17188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the M1 and M4 muscarinic acetylcholine receptors</span></div><div class="casAuthors">Thal, David M.; Sun, Bingfa; Feng, Dan; Nawaratne, Vindhya; Leach, Katie; Felder, Christian C.; Bures, Mark G.; Evans, David A.; Weis, William I.; Bachhawat, Priti; Kobilka, Tong Sun; Sexton, Patrick M.; Kobilka, Brian K.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">531</span>
        (<span class="NLM_cas:issue">7594</span>),
    <span class="NLM_cas:pages">335-340</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Muscarinic M1-M5 acetylcholine receptors are G protein-coupled receptors (GPCRs) that regulate many vital functions of the central and peripheral nervous systems.  In particular, the M1 and M4 receptor subtypes have emerged as attractive drug targets for treatments of neurol. disorders, such as Alzheimer's disease and schizophrenia, but the high conservation of the acetylcholine-binding pocket has spurred current research into targeting allosteric sites on these receptors.  Here we report the crystal structures of the M1 and M4 muscarinic receptors bound to the inverse agonist, tiotropium.  Comparison of these structures with each other, as well as with the previously reported M2 and M3 receptor structures, reveals differences in the orthosteric and allosteric binding sites that contribute to a role in drug selectivity at this important receptor family.  We also report identification of a cluster of residues that form a network linking the orthosteric and allosteric sites of the M4 receptor, which provides new insight into how allosteric modulation may be transmitted between the two spatially distinct domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3oy2Ml852CrVg90H21EOLACvtfcHk0lhgGxZzA_VMjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktlGjurg%253D&md5=8ed3f453737ad0291cf5b2c088f528da</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature17188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17188%26sid%3Dliteratum%253Aachs%26aulast%3DThal%26aufirst%3DD.%2BM.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DNawaratne%26aufirst%3DV.%26aulast%3DLeach%26aufirst%3DK.%26aulast%3DFelder%26aufirst%3DC.%2BC.%26aulast%3DBures%26aufirst%3DM.%2BG.%26aulast%3DEvans%26aufirst%3DD.%2BA.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DBachhawat%26aufirst%3DP.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DCrystal%2520structures%2520of%2520the%2520M1%2520and%2520M4%2520muscarinic%2520acetylcholine%2520receptors%26jtitle%3DNature%26date%3D2016%26volume%3D531%26spage%3D335%26epage%3D340%26doi%3D10.1038%2Fnature17188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choppin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, N.</span></span> <span> </span><span class="NLM_article-title">Therapeutic opportunities from muscarinic receptor research</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.1016/S0165-6147(00)01737-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2FS0165-6147%2800%2901737-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=11479003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsFyksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=409-414&author=R.+M.+Eglenauthor=A.+Choppinauthor=N.+Watson&title=Therapeutic+opportunities+from+muscarinic+receptor+research&doi=10.1016%2FS0165-6147%2800%2901737-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic opportunities from muscarinic receptor research</span></div><div class="casAuthors">Eglen, R. M.; Choppin, A.; Watson, N.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">409-414</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Muscarinic acetylcholine receptor subtypes have been the subjects of research for at least a quarter of a century.  Nonetheless, there are few selective muscarinic receptor ligands presently used as therapeutics.  The extensive development of muscarinic M1 receptor agonists for the treatment of cognitive dysfunction has culminated in a series of unsuccessful drug candidates, which reflects a lack of understanding of the disease and the role played by muscarinic cholinergic transmission.  Paradoxically, the most successful antagonist approved for use in urinary incontinence is the nonselective muscarinic receptor antagonist tolterodine.  This deficit in subtype-selective ligands could be circumvented by the development of transgenic mice, each lacking functional M1, M2, M3, M4 or M5 receptors.  In this article, the current status of muscarinic receptor research is critically assessed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsZkhW-d5bhbVg90H21EOLACvtfcHk0lgiVnP65a5xNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsFyksLw%253D&md5=ca0c63af01d912edc947a0f8d106656d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0165-6147%2800%2901737-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0165-6147%252800%252901737-5%26sid%3Dliteratum%253Aachs%26aulast%3DEglen%26aufirst%3DR.%2BM.%26aulast%3DChoppin%26aufirst%3DA.%26aulast%3DWatson%26aufirst%3DN.%26atitle%3DTherapeutic%2520opportunities%2520from%2520muscarinic%2520receptor%2520research%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2001%26volume%3D22%26spage%3D409%26epage%3D414%26doi%3D10.1016%2FS0165-6147%2800%2901737-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheardown, M. J.</span></span> <span> </span><span class="NLM_article-title">Muscarinic M<sub>1</sub> receptor agonists and M<sub>2</sub> receptor antagonists as therapeutic targets in Alzheimer’s disease</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">870</span>, <span class="refDoi"> DOI: 10.1517/13543776.12.6.863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1517%2F13543776.12.6.863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD38XkvVWlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=863-870&author=M.+J.+Sheardown&title=Muscarinic+M1+receptor+agonists+and+M2+receptor+antagonists+as+therapeutic+targets+in+Alzheimer%E2%80%99s+disease&doi=10.1517%2F13543776.12.6.863"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic M1 receptor agonists and M2 receptor antagonists as therapeutic targets in Alzheimer's disease</span></div><div class="casAuthors">Sheardown, Malcolm J.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">863-870</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Ashley Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease is the most common form of senile dementia and is of increasing medical and economic importance.  One of the earliest findings in the systematic study of Alzheimer's disease was the progressive loss of acetylcholinergic neurons.  This lead to the so-called cholinergic hypothesis of Alzheimer's disease and provoked a search for cholinergic replacement therapies.  This has resulted in compds. capable of inhibiting the catabolism of acetylcholine (acetylcholinesterase inhibitors) being licensed for the treatment of Alzheimer's disease.  However, the search for directly acting, selective, muscarinic M1 receptor agonists and M2 receptor antagonists, that are capable of increasing acetylcholine levels in the neocortex, has not yet proved successful.  For the muscarinic agonists this failure has been largely due to the difficulty in finding genuinely selective compds.  As a result, there has been a redn. in the no. published patents in this area.  This review discusses the rationale, progress and the possible future for these two potential approaches to the therapy of Alzheimer's disease esp. in light of some recent patents contg. selective compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp7DYdk9clZLVg90H21EOLACvtfcHk0lgiVnP65a5xNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XkvVWlsLc%253D&md5=7b55f8667befd78dcb877b6cc3c55a7f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1517%2F13543776.12.6.863&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.12.6.863%26sid%3Dliteratum%253Aachs%26aulast%3DSheardown%26aufirst%3DM.%2BJ.%26atitle%3DMuscarinic%2520M1%2520receptor%2520agonists%2520and%2520M2%2520receptor%2520antagonists%2520as%2520therapeutic%2520targets%2520in%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2002%26volume%3D12%26spage%3D863%26epage%3D870%26doi%3D10.1517%2F13543776.12.6.863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clader, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Muscarinic receptor agonists and antagonists in the treatment of Alzheimer’s disease</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3353</span>– <span class="NLM_lpage">3361</span>, <span class="refDoi"> DOI: 10.2174/138161205774370762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.2174%2F138161205774370762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=16250841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWrs7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=3353-3361&author=J.+W.+Claderauthor=Y.+Wang&title=Muscarinic+receptor+agonists+and+antagonists+in+the+treatment+of+Alzheimer%E2%80%99s+disease&doi=10.2174%2F138161205774370762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic receptor agonists and antagonists in the treatment of Alzheimer's disease</span></div><div class="casAuthors">Clader, John W.; Wang, Yuguang</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">3353-3361</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  One of the consistent findings in the brains of Alzheimer's Disease (AD) patients is loss of cholinergic function.  The cholinergic approach to treatment of AD involves counteracting this loss in cholinergic activity by pharmacol. intervention to increase cholinergic transmission.  The cognitive effects of acetylcholine are mediated via the muscarinic M1 receptor.  Direct stimulation of this receptor using muscarinic M1 agonists improves cognition in animal models and improves performance in cognitive tests in Alzheimer's patients.  Alternatively, antagonists of central presynaptic M2 receptors improves cognition by increasing the central release of acetylcholine.  Both approaches require high selectivity for one muscarinic receptor sub-type both for efficacy and to avoid cholinergic side effects.  In this review summarizes recent progress in the identification and characterization of selective muscarinic receptor ligands for the treatment of Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2_wdSDisNRLVg90H21EOLACvtfcHk0lh11W6hJ8qklA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWrs7vP&md5=d0d18c0299921abfbf6b815b223e743a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F138161205774370762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161205774370762%26sid%3Dliteratum%253Aachs%26aulast%3DClader%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DMuscarinic%2520receptor%2520agonists%2520and%2520antagonists%2520in%2520the%2520treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2005%26volume%3D11%26spage%3D3353%26epage%3D3361%26doi%3D10.2174%2F138161205774370762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Citron, M.</span></span> <span> </span><span class="NLM_article-title">Alzheimer’s disease: strategies for disease modification</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1038/nrd2896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnrd2896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=20431570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=387-398&author=M.+Citron&title=Alzheimer%E2%80%99s+disease%3A+strategies+for+disease+modification&doi=10.1038%2Fnrd2896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Alzheimer's disease: strategies for disease modification</span></div><div class="casAuthors">Citron, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">387-398</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease is the largest unmet medical need in neurol.  Current drugs improve symptoms, but do not have profound disease-modifying effects.  However, in recent years, several approaches aimed at inhibiting disease progression have advanced to clin. trials.  Among these, strategies targeting the prodn. and clearance of the amyloid-β peptide - a cardinal feature of Alzheimer's disease that is thought to be important in disease pathogenesis - are the most advanced.  Approaches aimed at modulating the abnormal aggregation of tau filaments (another key feature of the disease), and those targeting metabolic dysfunction, are also being evaluated in the clinic.  This article discusses recent progress with each of these strategies, with a focus on anti-amyloid strategies, highlighting the lessons learned and the challenges that remain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeM0fsdJQ-z7Vg90H21EOLACvtfcHk0lh11W6hJ8qklA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWjsrw%253D&md5=281ba27137d2a1fa173d5dde1dcbe772</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2896%26sid%3Dliteratum%253Aachs%26aulast%3DCitron%26aufirst%3DM.%26atitle%3DAlzheimer%25E2%2580%2599s%2520disease%253A%2520strategies%2520for%2520disease%2520modification%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D387%26epage%3D398%26doi%3D10.1038%2Fnrd2896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clader, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billard, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binch, H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlowski, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachowicz, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCombie, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vice, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenlee, W. J.</span></span> <span> </span><span class="NLM_article-title">Muscarinic M<sub>2</sub> antagonists: anthranilamide derivatives with exceptional selectivity and in vivo activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2003.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2Fj.bmc.2003.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=14723952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVOksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=319-326&author=J.+W.+Claderauthor=W.+Billardauthor=H.+Binchauthor=L.+Y.+Chenauthor=G.+Crosbyauthor=R.+A.+Duffyauthor=J.+Fordauthor=J.+A.+Kozlowskiauthor=J.+E.+Lachowiczauthor=S.+Liauthor=C.+Liuauthor=S.+W.+McCombieauthor=S.+Viceauthor=G.+Zhouauthor=W.+J.+Greenlee&title=Muscarinic+M2+antagonists%3A+anthranilamide+derivatives+with+exceptional+selectivity+and+in+vivo+activity&doi=10.1016%2Fj.bmc.2003.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic M2 antagonists: anthranilamide derivatives with exceptional selectivity and in vivo activity</span></div><div class="casAuthors">Clader, John W.; Billard, William; Binch, Herbert; Chen, Lian-Yong; Crosby, Gordon; Duffy, Ruth A.; Ford, Jennifer; Kozlowski, Joseph A.; Lachowicz, Jean E.; Li, Shengjian; Liu, Charles; McCombie, Stuart W.; Vice, Susan; Zhou, Gowei; Greenlee, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">319-326</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) assocs. with the consisting findings in brains of AD patients of a loss of cholinergic markers, including levels of acetylcholine.  The cholinergic approach to treatment of AD involves counteracting this loss in cholinergic activity by pharmacol. intervention to increase cholinergic transmission.  While the use of acetylcholinesterase inhibitors remains the most common approach to cholinergic therapy, an alternative involves inhibition of central presynaptic M2 receptors, thus causing an increase in acetylcholine release.  The authors present anthranilamide analogs of formula (I) and (II) that are potent and highly selective muscarinic M2 antagonists that also show good oral bioavailability and in vivo activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHDdxde5nOVbVg90H21EOLACvtfcHk0lg6qLXJj0OZIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVOksA%253D%253D&md5=9b696ecafbf680510d2ab78fe6829a8f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2003.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2003.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DClader%26aufirst%3DJ.%2BW.%26aulast%3DBillard%26aufirst%3DW.%26aulast%3DBinch%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DL.%2BY.%26aulast%3DCrosby%26aufirst%3DG.%26aulast%3DDuffy%26aufirst%3DR.%2BA.%26aulast%3DFord%26aufirst%3DJ.%26aulast%3DKozlowski%26aufirst%3DJ.%2BA.%26aulast%3DLachowicz%26aufirst%3DJ.%2BE.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DMcCombie%26aufirst%3DS.%2BW.%26aulast%3DVice%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DGreenlee%26aufirst%3DW.%2BJ.%26atitle%3DMuscarinic%2520M2%2520antagonists%253A%2520anthranilamide%2520derivatives%2520with%2520exceptional%2520selectivity%2520and%2520in%2520vivo%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D319%26epage%3D326%26doi%3D10.1016%2Fj.bmc.2003.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dörje, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrecht, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutschler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brann, M.</span></span> <span> </span><span class="NLM_article-title">Antagonist binding profiles of five cloned human muscarinic receptor subtypes</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>256</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">733</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1994002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A280%3ADyaK3M7jt1CrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=256&publication_year=1991&pages=727-733&author=F.+D%C3%B6rjeauthor=J.+Wessauthor=G.+Lambrechtauthor=R.+Tackeauthor=E.+Mutschlerauthor=M.+Brann&title=Antagonist+binding+profiles+of+five+cloned+human+muscarinic+receptor+subtypes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Antagonist binding profiles of five cloned human muscarinic receptor subtypes</span></div><div class="casAuthors">Dorje F; Wess J; Lambrecht G; Tacke R; Mutschler E; Brann M R</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of pharmacology and experimental therapeutics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">256</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">727-33</span>
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">A variety of muscarinic antagonists are currently used as tools to pharmacologically subclassify muscarinic receptors into M1, M2 and M3 subtypes.  In the present study, we have determined the affinity profiles of several of these antagonists at five cloned human muscarinic receptors (m1-m5) stably expressed in Chinese hamster ovary cells (CHO-K1).  At all five receptors, the (R)-enantiomers of trihexyphenidyl and hexbutinol displayed considerably higher affinities (up to 525-fold) than their corresponding (S)-isomers.  The stereoselectivity ratios [inhibition constant(S)/inhibition constant(R)] for both pairs of enantiomers were lowest at m2 receptors, suggesting that less stringent configurational demands are made by this receptor subtype.  The "M1-selective" antagonist (R)-trihexyphenidyl displayed high affinities for m1 and m4 receptors.  The "M2-selective" antagonists himbacine, (+-)-5,11-dihydro-11- ([(2-[(dipropylamino)methyl]-1- piperidinyl)ethyl)amino]carbonyl)-6H-pyrido(2,3-b)(1,4)benzodiazepine-6- one (AF-DX 384), 11-[4-[4-(diethylamino)butyl]-1-piperidinyl)acetyl)-5,11- dihydro-6H-pyrido(2,3-b) (1,4)benzodiazepine-6-one (AQ-RA 741) and (+)-(11-[2-[(diethylamino) methyl]-1-piperidinyl)acetyl)-5,11-di-hydro-6H-pyrido(2,3-b)(1,4) benzodiazepine-6-one [AF-DX 250; the (+)-enantiomer of AF-DX 116] exhibited high affinities for m2 and m4, intermediate affinities for m1 and m3 and low affinities for m5 receptors.  This selectivity profile was most prominent for AQ-RA 741, which displayed 195- and 129-fold higher affinities for m2 and m4 receptors than for m5 receptors.(ABSTRACT TRUNCATED AT 250 WORDS)</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT4swN6h2ItNhKDYG3bNImrfW6udTcc2ea7qvinnLMbzLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3M7jt1CrtQ%253D%253D&md5=24fb5162f94b5e0163c9ae9ca2a5a582</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25B6rje%26aufirst%3DF.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DLambrecht%26aufirst%3DG.%26aulast%3DTacke%26aufirst%3DR.%26aulast%3DMutschler%26aufirst%3DE.%26aulast%3DBrann%26aufirst%3DM.%26atitle%3DAntagonist%2520binding%2520profiles%2520of%2520five%2520cloned%2520human%2520muscarinic%2520receptor%2520subtypes%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D1991%26volume%3D256%26spage%3D727%26epage%3D733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quirion, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihm, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entzeroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavi, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladinsky, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bechtel, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ensinger, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendla, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of CNS-active M<sub>2</sub> antagonists: novel structures and pharmacology</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1016/0024-3205(93)90307-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0024-3205%2893%2990307-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8441330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK3sXhtl2jtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1993&pages=497-503&author=H.+N.+Doodsauthor=R.+Quirionauthor=G.+Mihmauthor=W.+Engelauthor=K.+Rudolfauthor=M.+Entzerothauthor=G.+B.+Schiaviauthor=H.+Ladinskyauthor=W.+D.+Bechtelauthor=H.+A.+Ensingerauthor=K.+D.+Mendlaauthor=W.+Eberlein&title=Therapeutic+potential+of+CNS-active+M2+antagonists%3A+novel+structures+and+pharmacology&doi=10.1016%2F0024-3205%2893%2990307-O"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of CNS-active M2 antagonists: novel structures and pharmacology</span></div><div class="casAuthors">Doods, H. N.; Quirion, R.; Mihm, G.; Engel, W.; Rudolf, K.; Entzeroth, M.; Schiavi, G. B.; Ladinsky, H.; Bechtel, W. D.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">497-503</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    </div><div class="casAbstract">A review with 13 refs. of the author's work.  It is believed, that of the five muscarinic receptor subtypes so far identified in the brain, M2 receptors are located presynaptically in the cortex and hippocampus and upon the brain, M2 receptors are located presynaptically in the cortex and hippocampus and upon stimulation inhibit the release of acetylcholine.  Based on this hypothesis, we initiated a drug discovery program with the aim of identifying selective and centrally active M2 antagonists which are capable of enhancing cholinergic transmission.  These efforts resulted in the successful design and synthesis of novel muscarinic antagonists able to cross the blood brain barrier.  However, these compds. show few peripheral effects and possess a superior M2 vs. M1 selectivity.  The prototype of this novel class of M2 selective compds., BIBN 99, could be a valuable tool to test the hypothesis that lipophilic M2 antagonists show beneficial effects in the treatment of cognitive disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouGDQBiyG9E7Vg90H21EOLACvtfcHk0lgGMDV78jTE1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhtl2jtbs%253D&md5=1c0d1472e7b20059e4458eba712c69a4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2F0024-3205%2893%2990307-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0024-3205%252893%252990307-O%26sid%3Dliteratum%253Aachs%26aulast%3DDoods%26aufirst%3DH.%2BN.%26aulast%3DQuirion%26aufirst%3DR.%26aulast%3DMihm%26aufirst%3DG.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DSchiavi%26aufirst%3DG.%2BB.%26aulast%3DLadinsky%26aufirst%3DH.%26aulast%3DBechtel%26aufirst%3DW.%2BD.%26aulast%3DEnsinger%26aufirst%3DH.%2BA.%26aulast%3DMendla%26aufirst%3DK.%2BD.%26aulast%3DEberlein%26aufirst%3DW.%26atitle%3DTherapeutic%2520potential%2520of%2520CNS-active%2520M2%2520antagonists%253A%2520novel%2520structures%2520and%2520pharmacology%26jtitle%3DLife%2520Sci.%26date%3D1993%26volume%3D52%26spage%3D497%26epage%3D503%26doi%3D10.1016%2F0024-3205%2893%2990307-O" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doods, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entzeroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiavi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihm, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, W.</span></span> <span> </span><span class="NLM_article-title">Characterization of BIBN 99: a lipophilic and selective muscarinic M<sub>2</sub> receptor antagonist</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>242</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/0014-2999(93)90005-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0014-2999%2893%2990005-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8223934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2cXktlw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=242&publication_year=1993&pages=23-30&author=H.+Doodsauthor=M.+Entzerothauthor=H.+Zieglerauthor=G.+Schiaviauthor=W.+Engelauthor=G.+Mihmauthor=K.+Rudolfauthor=W.+Eberlein&title=Characterization+of+BIBN+99%3A+a+lipophilic+and+selective+muscarinic+M2+receptor+antagonist&doi=10.1016%2F0014-2999%2893%2990005-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of BIBN 99: A lipophilic and selective muscarinic M2 receptor antagonist</span></div><div class="casAuthors">Doods, Henri; Entzeroth, Michael; Ziegler, Harald; Schiavi, Giovanni; Engel, Wolfhard; Mihm, Gerhard; Rudolf, Klaus; Eberlein, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">242</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    </div><div class="casAbstract">The present study was designed to characterize the receptor selectivity profile of the novel muscarinic M2 receptor antagonist BIBN 99 (5,11-dihydro-8-chloro-11-[[4-[3-[(2,2-dimethyl-1-oxopentyl)ethylamino]propyl]-1-piperidinyl]acetyl]-6H-pyrido[2,3-b][1,4]benzodiazepin-6-one).  In radioligand binding studies BIBN 99 showed high affinity for m2/M2 sites (pK1 = 7.52/7.57), intermediate affinity for m4 sites (pKi = 6.76) and low affinity for m1/M1 (pK1 = 5.97/6.17), m3/M3 (pKi = 6.11/6.04) and m5 sites (pKi = 5.84).  Functional studies in vitro showed BIBN 99 to be a competitive antagonist and to have an 11- to 25-fold higher affinity for M2 receptors than for putative M1 receptors in the rabbit vas deferens or M3 receptors in guinea-pig trachea.  In vivo studies revealed that BIBN 99 is able to cross the blood-brain barrier, and although showing an approx. 3-fold higher affinity for M2 binding sites BIBN 99 appeared to be 7- to 18-fold less potent than AF-DX 116 in inhibiting muscarinic agonist or vagally induced bradycardia in rats and guinea-pigs.  The results show that BIBN 99 is the first lipophilic muscarinic M2 receptor antagonist to have remarkable M2 vs. M1 selectivity (30-fold).  In addn., BIBN 99 possesses central nervous system activity and only minor peripheral cardiac effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8mHfHIPQPGbVg90H21EOLACvtfcHk0lgGMDV78jTE1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktlw%253D&md5=434d8f19eabb285348c60180a550db25</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2893%2990005-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252893%252990005-3%26sid%3Dliteratum%253Aachs%26aulast%3DDoods%26aufirst%3DH.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DZiegler%26aufirst%3DH.%26aulast%3DSchiavi%26aufirst%3DG.%26aulast%3DEngel%26aufirst%3DW.%26aulast%3DMihm%26aufirst%3DG.%26aulast%3DRudolf%26aufirst%3DK.%26aulast%3DEberlein%26aufirst%3DW.%26atitle%3DCharacterization%2520of%2520BIBN%252099%253A%2520a%2520lipophilic%2520and%2520selective%2520muscarinic%2520M2%2520receptor%2520antagonist%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1993%26volume%3D242%26spage%3D23%26epage%3D30%26doi%3D10.1016%2F0014-2999%2893%2990005-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maggio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tedeschi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchiorre, C.</span></span> <span> </span><span class="NLM_article-title">Binding profile of the selective muscarinic receptor antagonist tripitramine</span>. <i>Eur. J. Pharmacol., Mol. Pharmacol. Sect.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1016/0922-4106(94)90075-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0922-4106%2894%2990075-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=7805774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2cXmsVKjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1994&pages=459-462&author=R.+Maggioauthor=P.+Barbierauthor=M.+L.+Bolognesiauthor=A.+Minariniauthor=D.+Tedeschiauthor=C.+Melchiorre&title=Binding+profile+of+the+selective+muscarinic+receptor+antagonist+tripitramine&doi=10.1016%2F0922-4106%2894%2990075-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Binding profile of the selective muscarinic receptor antagonist tripitramine</span></div><div class="casAuthors">Maggio, Roberto; Barbier, Pascaline; Bolognesi, Maria L.; Minarini, Anna; Tedeschi, Daniele; Melchiorre, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology, Molecular Pharmacology Section</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-62</span>CODEN:
                <span class="NLM_cas:coden">EJPPET</span>;
        ISSN:<span class="NLM_cas:issn">0922-4106</span>.
    </div><div class="casAbstract">The binding selectivity of the muscarinic antagonist tripitramine has been tested on the five cloned human muscarinic receptor subtypes (Hm1 to Hm5) expressed in chinese hamster ovary (CHO-K1) cells.  The results indicate that tripitramine binds to the muscarinic Hm2 receptor with a Ki value of 0.27 ± 0.02 nM.  Tripitramine distinguishes Hm2 vs. Hm4 by a factor of 24 and vs. Hm3 and Hm5 by a factor of 142 and 125, resp.  A lower affinity ratio, about 6-fold, was found between muscarinic Hm2 and Hm1 receptors.  A comparative study with the well-known selective muscarinic M2 receptor antagonist methoctramine indicates that tripitramine has gained both potency and selectivity for the muscarinic Hm2 receptor subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlP4SfCTKVJrVg90H21EOLACvtfcHk0lgGMDV78jTE1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmsVKjs7k%253D&md5=6067bf0125d1ec322d40b59cb585a652</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0922-4106%2894%2990075-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0922-4106%252894%252990075-2%26sid%3Dliteratum%253Aachs%26aulast%3DMaggio%26aufirst%3DR.%26aulast%3DBarbier%26aufirst%3DP.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DMinarini%26aufirst%3DA.%26aulast%3DTedeschi%26aufirst%3DD.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DBinding%2520profile%2520of%2520the%2520selective%2520muscarinic%2520receptor%2520antagonist%2520tripitramine%26jtitle%3DEur.%2520J.%2520Pharmacol.%252C%2520Mol.%2520Pharmacol.%2520Sect.%26date%3D1994%26volume%3D268%26spage%3D459%26epage%3D462%26doi%3D10.1016%2F0922-4106%2894%2990075-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reba, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, V. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzeszotarski, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgold, J.</span></span> <span> </span><span class="NLM_article-title">A novel m2-selective muscarinic antagonist: binding characteristics and autoradiographic distribution in rat brain</span>. <i>Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/0006-8993(92)90141-U</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0006-8993%2892%2990141-U" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1393548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK38XkvVegsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=1992&pages=253-260&author=M.+S.+Gitlerauthor=R.+C.+Rebaauthor=V.+I.+Cohenauthor=W.+J.+Rzeszotarskiauthor=J.+Baumgold&title=A+novel+m2-selective+muscarinic+antagonist%3A+binding+characteristics+and+autoradiographic+distribution+in+rat+brain&doi=10.1016%2F0006-8993%2892%2990141-U"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A novel M2-selective muscarinic antagonist:  binding characteristics and autoradiographic distribution in rat brain</span></div><div class="casAuthors">Gitler, Miriam S.; Reba, Richard C.; Cohen, Victor I.; Rzeszotarski, Waclaw J.; Baumgold, Jesse</div><div class="citationInfo"><span class="NLM_cas:title">Brain Research</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">253-60</span>CODEN:
                <span class="NLM_cas:coden">BRREAP</span>;
        ISSN:<span class="NLM_cas:issn">0006-8993</span>.
    </div><div class="casAbstract">Although several M2-selective muscarinic antagonists have been described, they are not particularly potent.  Thus, the development of potent M2-selective compds. remains an important goal.  The authors now report that a bio-isoster of AQ-RA 741 is both one order of magnitude more potent and slightly more selective than previously described compds.  DIBA (I), a di-benzo deriv. of AQ-RA 741, in which the pyridine of the tricycle is replaced with a benzene ring, had Ki values of 4, 0.3, 11 and 2 nM at M1 through M4 receptors, resp.  These values were detd. in competition studies with [3H]N-methylscopolamine ([3H]NMS) in membranes from transfected A9 L cells (M1 and M3), rat heart (M2) and NG108-15 cells (M4).  AQ-RA 741 had Ki values of 34, 4, 86 and 15 nm at each of these receptors.  The autoradiog. distribution of I binding sites was detd. by competition studies of [3H]NMS in rat brain.  At low concn., I reduced [3H]NMS binding most significantly from superior colliculi, thalamus, hypothalamus, pontine nucleus, and interpeduncular nucleus, and not appreciably from caudate nucleus, cerebral cortical regions, or hippocampus, consistent with its binding to m2 receptors.  These data indicate that I is the most potent, M2-selective muscarinic antagonist yet described.  I should therefore become a useful probe in future studies of muscarinic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFUtuhX4_TGbVg90H21EOLACvtfcHk0lg6TPoOAG3EVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XkvVegsb4%253D&md5=c738f525755ed004c458acce0e29018b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2F0006-8993%2892%2990141-U&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-8993%252892%252990141-U%26sid%3Dliteratum%253Aachs%26aulast%3DGitler%26aufirst%3DM.%2BS.%26aulast%3DReba%26aufirst%3DR.%2BC.%26aulast%3DCohen%26aufirst%3DV.%2BI.%26aulast%3DRzeszotarski%26aufirst%3DW.%2BJ.%26aulast%3DBaumgold%26aufirst%3DJ.%26atitle%3DA%2520novel%2520m2-selective%2520muscarinic%2520antagonist%253A%2520binding%2520characteristics%2520and%2520autoradiographic%2520distribution%2520in%2520rat%2520brain%26jtitle%3DBrain%2520Res.%26date%3D1992%26volume%3D582%26spage%3D253%26epage%3D260%26doi%3D10.1016%2F0006-8993%2892%2990141-U" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tränkle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pegoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. W.</span></span> <span> </span><span class="NLM_article-title">M<sub>2</sub> Subtype preferring dibenzodiazepinone-type muscarinic receptor ligands: Effect of chemical homo-dimerization on orthosteric (and allosteric?) binding</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3970</span>– <span class="NLM_lpage">3990</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2Fj.bmc.2015.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=25650309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Ogtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=3970-3990&author=M.+Kellerauthor=C.+Tr%C3%A4nkleauthor=X.+Sheauthor=A.+Pegoliauthor=G.+Bernhardtauthor=A.+Buschauerauthor=R.+W.+Read&title=M2+Subtype+preferring+dibenzodiazepinone-type+muscarinic+receptor+ligands%3A+Effect+of+chemical+homo-dimerization+on+orthosteric+%28and+allosteric%3F%29+binding&doi=10.1016%2Fj.bmc.2015.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">M2 Subtype preferring dibenzodiazepinone-type muscarinic receptor ligands: Effect of chemical homo-dimerization on orthosteric (and allosteric?) binding</span></div><div class="casAuthors">Keller, Max; Traenkle, Christian; She, Xueke; Pegoli, Andrea; Bernhardt, Guenther; Buschauer, Armin; Read, Roger W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3970-3990</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of new dibenzodiazepinone-type muscarinic receptor ligands, including two homo-dimeric compds., was prepd.  Sixteen representative compds. were characterized in equil. binding studies with [3H]N-methylscopolamine ([3H]NMS) at the muscarinic receptor subtype M2, and seven selected compds. were addnl. investigated at M1, M3, M4 and M5 with respect to receptor subtype selectivity.  The side chain of the known M2 preferring muscarinic receptor antagonist DIBA was widely varied with respect to chain length and type of the basic group (amine, imidazole, guanidine and piperazine).  Most of the structural changes were well tolerated with respect to muscarinic receptor binding, detd. by displacement of [3H]NMS.  Compds. investigated at all subtypes shared a similar selectivity profile, which can be summarized as M2 > M1 ≈ M4 > M3 ≈ M5 (46, 50, 57, 62-64) and M2 > M1 ≈ M4 > M3 > M5 (1, 58).  The homo-dimeric dibenzodiazepinone derivs. UNSW-MK250 (63) and UNSW-MK262 (64) exhibited the highest M2 receptor affinities (pIC50 = 9.0 and 9.2, resp.).  At the M2 receptor a steep curve slope of -2 was found for the dimeric ligand 63, which cannot be described according to the law of mass action, suggesting a more complex mechanism of binding.  In addn. to equil. binding studies, for selected ligands, we detd. pEC50,diss, an est. of affinity to the allosteric site of M2 receptors occupied with [3H]NMS.  Compds. 58 and 62-64 were capable of retarding [3H]NMS dissocn. by a factor >10 (Emax,diss >92%), with highest potency (pEC50,diss = 5.56) residing in the dimeric compd. 64.  As the monomeric counterpart of 64 was 100 times less potent (62: pEC50,diss = 3.59), these data suggest that chem. dimerization of dibenzodiazepinone-type M receptor ligands can enhance allosteric binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8D4k-_1Hnb7Vg90H21EOLACvtfcHk0lg6TPoOAG3EVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Ogtr4%253D&md5=00c26092810d90955e92c09ade5cec1d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DTr%25C3%25A4nkle%26aufirst%3DC.%26aulast%3DShe%26aufirst%3DX.%26aulast%3DPegoli%26aufirst%3DA.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26aulast%3DRead%26aufirst%3DR.%2BW.%26atitle%3DM2%2520Subtype%2520preferring%2520dibenzodiazepinone-type%2520muscarinic%2520receptor%2520ligands%253A%2520Effect%2520of%2520chemical%2520homo-dimerization%2520on%2520orthosteric%2520%2528and%2520allosteric%253F%2529%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D3970%26epage%3D3990%26doi%3D10.1016%2Fj.bmc.2015.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pegoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wifling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span> <span> </span><span class="NLM_article-title">Radiolabeled dibenzodiazepinone-type antagonists give evidence of dualsteric binding at the M<sub>2</sub> muscarinic acetylcholine receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3314</span>– <span class="NLM_lpage">3334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01892</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01892" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVaqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3314-3334&author=A.+Pegoliauthor=X.+Sheauthor=D.+Wiflingauthor=H.+H%C3%BCbnerauthor=G.+Bernhardtauthor=P.+Gmeinerauthor=M.+Keller&title=Radiolabeled+dibenzodiazepinone-type+antagonists+give+evidence+of+dualsteric+binding+at+the+M2+muscarinic+acetylcholine+receptor&doi=10.1021%2Facs.jmedchem.6b01892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Radiolabeled Dibenzodiazepinone-Type Antagonists Give Evidence of Dualsteric Binding at the M2 Muscarinic Acetylcholine Receptor</span></div><div class="casAuthors">Pegoli, Andrea; She, Xueke; Wifling, David; Huebner, Harald; Bernhardt, Guenther; Gmeiner, Peter; Keller, Max</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3314-3334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dualsteric ligand approach, aiming at ligands with improved subtype selectivity, has been increasingly applied to muscarinic receptors (MRs).  In this article we present the synthesis and characterization of a M2R subtype-preferring radiolabeled dibenzodiazepinone-type antagonist [3H]UNSW-MK259, (I) and its homodimeric analog [3H]UR-AP060, (II).  Satn. binding studies at the M2R, using the orthosteric antagonist atropine to det. unspecific binding, proved that the monomeric and the dimeric compd. bind to the orthosteric binding site (apparent Kd: 0.87 and 0.24 nM, resp.).  Various binding studies with I and II at the M2R, for instance, satn. binding expts. in the presence of the allosteric MR modulators W84 or LY2119620 (Schild-like anal.) suggested a competitive mechanism between the allosteric modulator and the dibenzodiazepinone derivs., and thus a dualsteric binding mode of both I and II.  This was consistent with the results of M2R MD simulations (≥2 μs) performed with I and II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMeA4UbzK06LVg90H21EOLACvtfcHk0lgvTCHFtPOXVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVaqsbs%253D&md5=1dcfeac950b89fc293968ceed01e2500</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01892%26sid%3Dliteratum%253Aachs%26aulast%3DPegoli%26aufirst%3DA.%26aulast%3DShe%26aufirst%3DX.%26aulast%3DWifling%26aufirst%3DD.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DKeller%26aufirst%3DM.%26atitle%3DRadiolabeled%2520dibenzodiazepinone-type%2520antagonists%2520give%2520evidence%2520of%2520dualsteric%2520binding%2520at%2520the%2520M2%2520muscarinic%2520acetylcholine%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3314%26epage%3D3334%26doi%3D10.1021%2Facs.jmedchem.6b01892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pegoli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huebner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span> <span> </span><span class="NLM_article-title">Heterodimerization of dibenzodiazepinone-type muscarinic acetylcholine receptor ligands leads to increased M<sub>2</sub>R affinity and selectivity</span>. <i>ACS Omega.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">6741</span>– <span class="NLM_lpage">6754</span>, <span class="refDoi"> DOI: 10.1021/acsomega.7b01085</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b01085" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Cru7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=6741-6754&author=X.+Sheauthor=A.+Pegoliauthor=J.+Mayrauthor=H.+Huebnerauthor=G.+Bernhardtauthor=P.+Gmeinerauthor=M.+Keller&title=Heterodimerization+of+dibenzodiazepinone-type+muscarinic+acetylcholine+receptor+ligands+leads+to+increased+M2R+affinity+and+selectivity&doi=10.1021%2Facsomega.7b01085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Heterodimerization of Dibenzodiazepinone-Type Muscarinic Acetylcholine Receptor Ligands Leads to Increased M2R Affinity and Selectivity</span></div><div class="casAuthors">She, Xueke; Pegoli, Andrea; Mayr, Judith; Huebner, Harald; Bernhardt, Guenther; Gmeiner, Peter; Keller, Max</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6741-6754</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In search for selective ligands for the muscarinic acetylcholine receptor (MR) subtype M2, the dimeric ligand approach, that is combining two pharmacophores in one and the same mol., was pursued.  Different types (agonists, antagonists, orthosteric, and allosteric) of monomeric MR ligands were combined by various linkers with a dibenzodiazepinone-type MR antagonist, affording five types of heterodimeric compds. ("DIBA-xanomeline," "DIBA-TBPB," "DIBA-77-LH-28-1," "DIBA-propantheline," and "DIBA-4-DAMP"), which showed high M2R affinities (pKi > 8.3).  The heterodimeric ligand UR-SK75 (46) exhibited the highest M2R affinity and selectivity [pKi (M1R-M5R): 8.84, 10.14, 7.88, 8.59, and 7.47].  Two tritium-labeled dimeric derivs. ("DIBA-xanomeline"-type: [3H]UR-SK71 ([3H]44) and "DIBA-TBPB"-type: [3H]UR-SK59 ([3H]64)) were prepd. to investigate their binding modes at hM2R.  Satn.-binding expts. showed that these compds. address the orthosteric binding site of the M2R.  The investigation of the effect of various allosteric MR modulators [gallamine (13), W84 (14), and LY2119620 (15)] on the equil. (13-15) or satn. (14) binding of [3H]64 suggested a competitive mechanism between [3H]64 and the investigated allosteric ligands, and consequently a dualsteric binding mode of 64 at the M2R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq4nnRJHnLF7Vg90H21EOLACvtfcHk0lgvTCHFtPOXVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Cru7zN&md5=4f7d67758e6e919d62541c5bec83e21d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b01085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b01085%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DX.%26aulast%3DPegoli%26aufirst%3DA.%26aulast%3DMayr%26aufirst%3DJ.%26aulast%3DHuebner%26aufirst%3DH.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DKeller%26aufirst%3DM.%26atitle%3DHeterodimerization%2520of%2520dibenzodiazepinone-type%2520muscarinic%2520acetylcholine%2520receptor%2520ligands%2520leads%2520to%2520increased%2520M2R%2520affinity%2520and%2520selectivity%26jtitle%3DACS%2520Omega.%26date%3D2017%26volume%3D2%26spage%3D6741%26epage%3D6754%26doi%3D10.1021%2Facsomega.7b01085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Amici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dallanoce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzgrabe, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trankle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, K.</span></span> <span> </span><span class="NLM_article-title">Allosteric ligands for G protein-coupled receptors: a novel strategy with attractive therapeutic opportunities</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1002/med.20166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1002%2Fmed.20166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=19557759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVOlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=463-549&author=M.+De%0AAmiciauthor=C.+Dallanoceauthor=U.+Holzgrabeauthor=C.+Trankleauthor=K.+Mohr&title=Allosteric+ligands+for+G+protein-coupled+receptors%3A+a+novel+strategy+with+attractive+therapeutic+opportunities&doi=10.1002%2Fmed.20166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric ligands for G protein-coupled receptors: a novel strategy with attractive therapeutic opportunities</span></div><div class="casAuthors">De Amici, Marco; Dallanoce, Clelia; Holzgrabe, Ulrike; Trankle, Christian; Mohr, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">463-549</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Allosteric receptor ligands bind to a recognition site that is distinct from the binding site of the endogenous messenger mol.  As a consequence, allosteric agents may attach to receptors that are already transmitter-bound.  Ternary complex formation opens an avenue to qual. new drug actions at G protein-coupled receptors (GPCRs), in particular receptor subtype selective potentiation of endogenous transmitter action.  Consequently, suitable exploitation of allosteric recognition sites as alternative mol. targets could pave the way to a drug discovery paradigm different from those aimed at mimicking or blocking the effects of endogenous (orthosteric) receptor activators.  The no. of allosteric ligands reported to modulate GPCR function is steadily increasing and some have already reached routine clin. use.  This review aims at introducing into this fascinating field of drug discovery and at providing an overview about the achievements that have already been made.  Various case examples will be discussed in the framework of GPCR classification (family A, B, and C receptors).  In addn., the behavior at muscarinic receptors of hybrid derivs. incorporating both an allosteric and an orthosteric fragment in a common mol. skeleton will be illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorICLQyOlQ3bVg90H21EOLACvtfcHk0lgTjwAKe_Zg9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVOlsbY%253D&md5=99697b812d5dac4e90d9e6bcc46a639d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fmed.20166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20166%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BAmici%26aufirst%3DM.%26aulast%3DDallanoce%26aufirst%3DC.%26aulast%3DHolzgrabe%26aufirst%3DU.%26aulast%3DTrankle%26aufirst%3DC.%26aulast%3DMohr%26aufirst%3DK.%26atitle%3DAllosteric%2520ligands%2520for%2520G%2520protein-coupled%2520receptors%253A%2520a%2520novel%2520strategy%2520with%2520attractive%2520therapeutic%2520opportunities%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2010%26volume%3D30%26spage%3D463%26epage%3D549%26doi%3D10.1002%2Fmed.20166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert Lane, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">The best of both worlds? Bitopic orthosteric/allosteric ligands of g protein-coupled receptors</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1146%2Fannurev-pharmtox-010611-134514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=21910627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsV2ntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=153-178&author=C.+Valantauthor=J.+Robert+Laneauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=The+best+of+both+worlds%3F+Bitopic+orthosteric%2Fallosteric+ligands+of+g+protein-coupled+receptors&doi=10.1146%2Fannurev-pharmtox-010611-134514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The best of both worlds? Bitopic orthosteric/allosteric ligands of G protein-coupled receptors</span></div><div class="casAuthors">Valant, Celine; Lane, J. Robert; Sexton, Patrick M.; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">153-178</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  It is now acknowledged that G protein-coupled receptors, the largest class of drug targets, adopt multiple active states that can be preferentially stabilized by orthosteric ligands or allosteric modulators, thus giving rise to the phenomenon of pathway-biased signaling.  In the past few years, researchers have begun to explore the potential of linking orthosteric and allosteric pharmacophores to yield bitopic hybrid ligands.  This approach is an extension of the more traditional bivalent ligand concept and shares some of the same challenges, including the choice and role of the linker between the two pharmacophores and the validation of mechanism of action.  Nonetheless, the promise of bitopic ligands is the generation of novel chem. tools that have improved affinity and/or selectivity profiles.  Previously identified functionally selective compds. (and medicines) also may act via a bitopic mechanism, suggesting that the phenomenon is more widespread than currently appreciated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQs4WlX5wY8LVg90H21EOLACvtfcHk0lgTjwAKe_Zg9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsV2ntrc%253D&md5=334839005a548a9739df466e3d09e1ae</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134514%26sid%3Dliteratum%253Aachs%26aulast%3DValant%26aufirst%3DC.%26aulast%3DRobert%2BLane%26aufirst%3DJ.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DThe%2520best%2520of%2520both%2520worlds%253F%2520Bitopic%2520orthosteric%252Fallosteric%2520ligands%2520of%2520g%2520protein-coupled%2520receptors%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2012%26volume%3D52%26spage%3D153%26epage%3D178%26doi%3D10.1146%2Fannurev-pharmtox-010611-134514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ring, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Activation and allosteric modulation of a muscarinic acetylcholine receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>504</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1038/nature12735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature12735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=24256733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=101-106&author=A.+C.+Kruseauthor=A.+M.+Ringauthor=A.+Manglikauthor=J.+Huauthor=K.+Huauthor=K.+Eitelauthor=H.+H%C3%BCbnerauthor=E.+Pardonauthor=C.+Valantauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=C.+C.+Felderauthor=P.+Gmeinerauthor=J.+Steyaertauthor=W.+I.+Weisauthor=K.+C.+Garciaauthor=J.+Wessauthor=B.+K.+Kobilka&title=Activation+and+allosteric+modulation+of+a+muscarinic+acetylcholine+receptor&doi=10.1038%2Fnature12735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Activation and allosteric modulation of a muscarinic acetylcholine receptor</span></div><div class="casAuthors">Kruse, Andrew C.; Ring, Aaron M.; Manglik, Aashish; Hu, Jianxin; Hu, Kelly; Eitel, Katrin; Huebner, Harald; Pardon, Els; Valant, Celine; Sexton, Patrick M.; Christopoulos, Arthur; Felder, Christian C.; Gmeiner, Peter; Steyaert, Jan; Weis, William I.; Garcia, K. Christopher; Wess, Juergen; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7478</span>),
    <span class="NLM_cas:pages">101-106</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite recent advances in crystallog. and the availability of G-protein-coupled receptor (GPCR) structures, little is known about the mechanism of their activation process, as only the β2 adrenergic receptor (β2AR) and rhodopsin have been crystd. in fully active conformations.  Here we report the structure of an agonist-bound, active state of the human M2 muscarinic acetylcholine receptor stabilized by a G-protein mimetic camelid antibody fragment isolated by conformational selection using yeast surface display.  In addn. to the expected changes in the intracellular surface, the structure reveals larger conformational changes in the extracellular region and orthosteric binding site than obsd. in the active states of the β2AR and rhodopsin.  We also report the structure of the M2 receptor simultaneously bound to the orthosteric agonist iperoxo and the pos. allosteric modulator LY2119620.  This structure reveals that LY2119620 recognizes a largely pre-formed binding site in the extracellular vestibule of the iperoxo-bound receptor, inducing a slight contraction of this outer binding pocket.  These structures offer important insights into the activation mechanism and allosteric modulation of muscarinic receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOuUgQSB7Dd7Vg90H21EOLACvtfcHk0lgTjwAKe_Zg9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyisLjP&md5=ecddab0ab467321ac243b86bb82b71f8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature12735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12735%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DRing%26aufirst%3DA.%2BM.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DK.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DFelder%26aufirst%3DC.%2BC.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DGarcia%26aufirst%3DK.%2BC.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DActivation%2520and%2520allosteric%2520modulation%2520of%2520a%2520muscarinic%2520acetylcholine%2520receptor%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D101%26epage%3D106%26doi%3D10.1038%2Fnature12735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Advances in G protein-coupled receptor allostery: from function to structure</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">478</span>, <span class="refDoi"> DOI: 10.1124/mol.114.094342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1124%2Fmol.114.094342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=25061106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslamurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=463-478&author=A.+Christopoulos&title=Advances+in+G+protein-coupled+receptor+allostery%3A+from+function+to+structure&doi=10.1124%2Fmol.114.094342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in G protein-coupled receptor allostery: from function to structure</span></div><div class="casAuthors">Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">463-478, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  It is now widely accepted that G protein-coupled receptors (GPCRs) are highly dynamic proteins that adopt multiple active states linked to distinct functional outcomes.  Furthermore, these states can be differentially stabilized not only by orthosteric ligands but also by allosteric ligands acting at spatially distinct binding sites.  The key pharmacol. characteristics of GPCR allostery include improved selectivity due to either greater sequence divergence between receptor subtypes and/or subtype-selective cooperativity, a ceiling level to the effect, probe dependence (whereby the magnitude and direction of the allosteric effect change with the nature of the interacting ligands) and the potential for biased signaling.  Recent chem. biol. developments are beginning to demonstrate how the incorporation of anal. pharmacol. and operational modeling into the exptl. workflow can enrich structure-activity studies of allostery and bias and have also led to the discovery of a new class of hybrid orthosteric/allosteric (bitopic) mols.  The potential for endogenous allosteric modulators to play a role in physiol. and disease remains to be fully appreciated but will likely represent an important area for future studies.  Finally, breakthroughs in structural and computational biol. are beginning to unravel the mechanistic basis of GPCR allosteric modulation at the mol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8oX6HPCpaG7Vg90H21EOLACvtfcHk0liRjtZWOU6JQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslamurjI&md5=dba22b1ab717a25b9a2f9a9c888df9e6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.094342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.094342%26sid%3Dliteratum%253Aachs%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAdvances%2520in%2520G%2520protein-coupled%2520receptor%2520allostery%253A%2520from%2520function%2520to%2520structure%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26spage%3D463%26epage%3D478%26doi%3D10.1124%2Fmol.114.094342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yurugi-Kobayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiroishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span> <span> </span><span class="NLM_article-title">Structure of the human M<sub>2</sub> muscarinic acetylcholine receptor bound to an antagonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature10753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature10753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=22278061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ejsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=547-551&author=K.+Hagaauthor=A.+C.+Kruseauthor=H.+Asadaauthor=T.+Yurugi-Kobayashiauthor=M.+Shiroishiauthor=C.+Zhangauthor=W.+I.+Weisauthor=T.+Okadaauthor=B.+K.+Kobilkaauthor=T.+Hagaauthor=T.+Kobayashi&title=Structure+of+the+human+M2+muscarinic+acetylcholine+receptor+bound+to+an+antagonist&doi=10.1038%2Fnature10753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the human M2 muscarinic acetylcholine receptor bound to an antagonist</span></div><div class="casAuthors">Haga, Kazuko; Kruse, Andrew C.; Asada, Hidetsugu; Yurugi-Kobayashi, Takami; Shiroishi, Mitsunori; Zhang, Cheng; Weis, William I.; Okada, Tetsuji; Kobilka, Brian K.; Haga, Tatsuya; Kobayashi, Takuya</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">547-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The X-ray crystal structure of the M2 muscarinic acetylcholine receptor, which is essential for the physiol. control of cardiovascular function, is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosewFev-FNErVg90H21EOLACvtfcHk0liRjtZWOU6JQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ejsro%253D&md5=c33b8d9303d58db7dfdd4c18ec4f597b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature10753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10753%26sid%3Dliteratum%253Aachs%26aulast%3DHaga%26aufirst%3DK.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DAsada%26aufirst%3DH.%26aulast%3DYurugi-Kobayashi%26aufirst%3DT.%26aulast%3DShiroishi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DHaga%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStructure%2520of%2520the%2520human%2520M2%2520muscarinic%2520acetylcholine%2520receptor%2520bound%2520to%2520an%2520antagonist%26jtitle%3DNature%26date%3D2012%26volume%3D482%26spage%3D547%26epage%3D551%26doi%3D10.1038%2Fnature10753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosemond, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure and dynamics of the M<sub>3</sub> muscarinic acetylcholine receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>482</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.1038/nature10867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature10867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=22358844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2012&pages=552-556&author=A.+C.+Kruseauthor=J.+Huauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=D.+M.+Rosenbaumauthor=E.+Rosemondauthor=H.+F.+Greenauthor=T.+Liuauthor=P.+S.+Chaeauthor=R.+O.+Drorauthor=D.+E.+Shawauthor=W.+I.+Weisauthor=J.+Wessauthor=B.+K.+Kobilka&title=Structure+and+dynamics+of+the+M3+muscarinic+acetylcholine+receptor&doi=10.1038%2Fnature10867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of the M3 muscarinic acetylcholine receptor</span></div><div class="casAuthors">Kruse, Andrew C.; Hu, Jianxin; Pan, Albert C.; Arlow, Daniel H.; Rosenbaum, Daniel M.; Rosemond, Erica; Green, Hillary F.; Liu, Tong; Chae, Pil Seok; Dror, Ron O.; Shaw, David E.; Weis, William I.; Wess, Jurgen; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">7386</span>),
    <span class="NLM_cas:pages">552-556</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acetylcholine, the first neurotransmitter to be identified, exerts many of its physiol. actions via activation of a family of G-protein-coupled receptors (GPCRs) known as muscarinic acetylcholine receptors (mAChRs).  Although the five mAChR subtypes (M1-M5) share a high degree of sequence homol., they show pronounced differences in G-protein coupling preference and the physiol. responses they mediate.  Unfortunately, despite decades of effort, no therapeutic agents endowed with clear mAChR subtype selectivity have been developed to exploit these differences.  We describe here the structure of the Gq/11-coupled M3 mAChR ('M3 receptor', from rat) bound to the bronchodilator drug tiotropium and identify the binding mode for this clin. important drug.  This structure, together with that of the Gi/o-coupled M2 receptor, offers possibilities for the design of mAChR subtype-selective ligands.  Importantly, the M3 receptor structure allows a structural comparison between two members of a mammalian GPCR subfamily displaying different G-protein coupling selectivities.  Furthermore, mol. dynamics simulations suggest that tiotropium binds transiently to an allosteric site en route to the binding pocket of both receptors.  These simulations offer a structural view of an allosteric binding mode for an orthosteric GPCR ligand and provide addnl. opportunities for the design of ligands with different affinities or binding kinetics for different mAChR subtypes.  Our findings not only offer insights into the structure and function of one of the most important GPCR families, but may also facilitate the design of improved therapeutics targeting these crit. receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozveVSiaQvz7Vg90H21EOLACvtfcHk0liRjtZWOU6JQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xis1eqs7k%253D&md5=31bb89b1d14cf122353fced8f905e996</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature10867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10867%26sid%3Dliteratum%253Aachs%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DRosemond%26aufirst%3DE.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520and%2520dynamics%2520of%2520the%2520M3%2520muscarinic%2520acetylcholine%2520receptor%26jtitle%3DNature%26date%3D2012%26volume%3D482%26spage%3D552%26epage%3D556%26doi%3D10.1038%2Fnature10867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valcourt, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahmani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nature12595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature12595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=24121438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=295-299&author=R.+O.+Drorauthor=H.+F.+Greenauthor=C.+Valantauthor=D.+W.+Borhaniauthor=J.+R.+Valcourtauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=M.+Canalsauthor=J.+R.+Laneauthor=R.+Rahmaniauthor=J.+B.+Baellauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=D.+E.+Shaw&title=Structural+basis+for+modulation+of+a+G-protein-coupled+receptor+by+allosteric+drugs&doi=10.1038%2Fnature12595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span></div><div class="casAuthors">Dror, Ron O.; Green, Hillary F.; Valant, Celine; Borhani, David W.; Valcourt, James R.; Pan, Albert C.; Arlow, Daniel H.; Canals, Meritxell; Lane, J. Robert; Rahmani, Raphael; Baell, Jonathan B.; Sexton, Patrick M.; Christopoulos, Arthur; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7475</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of G-protein-coupled receptor (GPCR) allosteric modulators, an active area of modern pharmaceutical research, has proved challenging because neither the binding modes nor the mol. mechanisms of such drugs are known.  Here we det. binding sites, bound conformations and specific drug-receptor interactions for several allosteric modulators of the M2 muscarinic acetylcholine receptor (M2 receptor), a prototypical family A GPCR, using at.-level simulations in which the modulators spontaneously assoc. with the receptor.  Despite substantial structural diversity, all modulators form cation-π interactions with clusters of arom. residues in the receptor extracellular vestibule, approx. 15 Å from the classical, 'orthosteric' ligand-binding site.  We validate the obsd. modulator binding modes through radioligand binding expts. on receptor mutants designed, on the basis of our simulations, either to increase or to decrease modulator affinity.  Simulations also revealed mechanisms that contribute to pos. and neg. allosteric modulation of classical ligand binding, including coupled conformational changes of the two binding sites and electrostatic interactions between ligands in these sites.  These observations enabled the design of chem. modifications that substantially alter a modulator's allosteric effects.  Our findings thus provide a structural basis for the rational design of allosteric modulators targeting muscarinic and possibly other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTlvAcaBf-1rVg90H21EOLACvtfcHk0lhjoCrsFdJ2PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN&md5=2fa40ecd46ce62125ce4a12c18270b1f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature12595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12595%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DValcourt%26aufirst%3DJ.%2BR.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DRahmani%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520basis%2520for%2520modulation%2520of%2520a%2520G-protein-coupled%2520receptor%2520by%2520allosteric%2520drugs%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D295%26epage%3D299%26doi%3D10.1038%2Fnature12595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stossel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huebner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Structure-based design and discovery of new M<sub>2</sub> receptor agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9239</span>– <span class="NLM_lpage">9250</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01113</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01113" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslGnu7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9239-9250&author=I.+Fishauthor=A.+Stosselauthor=K.+Eitelauthor=C.+Valantauthor=S.+Alboldauthor=H.+Huebnerauthor=D.+Mollerauthor=M.+J.+Clarkauthor=R.+K.+Sunaharaauthor=A.+Christopoulosauthor=B.+K.+Shoichetauthor=P.+Gmeiner&title=Structure-based+design+and+discovery+of+new+M2+receptor+agonists&doi=10.1021%2Facs.jmedchem.7b01113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design and Discovery of New M2 Receptor Agonists</span></div><div class="casAuthors">Fish, Inbar; Stossel, Anne; Eitel, Katrin; Valant, Celine; Albold, Sabine; Huebner, Harald; Moller, Dorothee; Clark, Mary J.; Sunahara, Roger K.; Christopoulos, Arthur; Shoichet, Brian K.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9239-9250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Muscarinic receptor agonists are characterized by apparently strict restraints on their tertiary or quaternary amine and their distance to an ester or related center.  On the basis of the active state crystal structure of the muscarinic M2 receptor in complex with iperoxo, the authors explored potential agonists that lacked the highly conserved functionalities of previously known ligands.  Using structure-guided pharmacophore design followed by docking, the authors found two agonists (compds. 3 (2-(3-methoxyphenyl)-N,N,N-trimethylethan-1-aminium formate) and 17 (2-(benzofuran-6-yl)-N,N,N-trimethylethan-1-aminium formate)), out of 19 docked and synthesized compds., that fit the receptor well and were predicted to form a hydrogen-bond conserved among known agonists.  Structural optimization led to compd. 28 (2-(2,3-dihydrobenzofuran-6-yl)-N,N,N-trimethylethan-1-aminium formate), which was 4-fold more potent than its parent 3.  Fortified by the discovery of this new scaffold, the authors sought a broader range of chemotypes by docking 2.2 million fragments, which revealed another three micromolar agonists unrelated either to 28 or known muscarinics.  Even pockets as tightly defined and as deeply studied as that of the muscarinic reveal opportunities for the structure-based design and the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Yk4WSO0I3LVg90H21EOLACvtfcHk0lhjoCrsFdJ2PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslGnu7nN&md5=113c2c9b16bc816ddde88634a129da09</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01113%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DI.%26aulast%3DStossel%26aufirst%3DA.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DAlbold%26aufirst%3DS.%26aulast%3DHuebner%26aufirst%3DH.%26aulast%3DMoller%26aufirst%3DD.%26aulast%3DClark%26aufirst%3DM.%2BJ.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DStructure-based%2520design%2520and%2520discovery%2520of%2520new%2520M2%2520receptor%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9239%26epage%3D9250%26doi%3D10.1021%2Facs.jmedchem.7b01113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaake, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eitel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartuschat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaindl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampp, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Structure-guided development of selective M<sub>3</sub> muscarinic acetylcholine receptor antagonists</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">12046</span>– <span class="NLM_lpage">12050</span>, <span class="refDoi"> DOI: 10.1073/pnas.1813988115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1073%2Fpnas.1813988115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=30404914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFanu7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=12046-12050&author=H.+Liuauthor=J.+Hofmannauthor=I.+Fishauthor=B.+Schaakeauthor=K.+Eitelauthor=A.+Bartuschatauthor=J.+Kaindlauthor=H.+Ramppauthor=A.+Banerjeeauthor=H.+Hubnerauthor=M.+J.+Clarkauthor=S.+G.+Vincentauthor=J.+T.+Fisherauthor=M.+R.+Heinrichauthor=K.+Hirataauthor=X.+Liuauthor=R.+K.+Sunaharaauthor=B.+K.+Shoichetauthor=B.+K.+Kobilkaauthor=P.+Gmeiner&title=Structure-guided+development+of+selective+M3+muscarinic+acetylcholine+receptor+antagonists&doi=10.1073%2Fpnas.1813988115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided development of selective M3 muscarinic acetylcholine receptor antagonists</span></div><div class="casAuthors">Liu, Hongtao; Hofmann, Josefa; Fish, Inbar; Schaake, Benjamin; Eitel, Katrin; Bartuschat, Amelie; Kaindl, Jonas; Rampp, Hannelore; Banerjee, Ashutosh; HA1/4bner, Harald; Clark, Mary J.; Vincent, Sandra G.; Fisher, John T.; Heinrich, Markus R.; Hirata, Kunio; Liu, Xiangyu; Sunahara, Roger K.; Shoichet, Brian K.; Kobilka, Brian K.; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12046-12050</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Drugs that treat chronic obstructive pulmonary disease by antagonizing the M3 muscarinic acetylcholine receptor (M3R) have had a significant effect on health, but can suffer from their lack of selectivity against the M2R subtype, which modulates heart rate.  Beginning with the crystal structures of M2R and M3R, we exploited a single amino acid difference in their orthosteric binding pockets using mol. docking and structure-based design.  The resulting M3R antagonists had up to 100-fold selectivity over M2R in affinity and over 1,000-fold selectivity in vivo.  The crystal structure of the M3R-selective antagonist in complex with M3R corresponded closely to the docking-predicted geometry, providing a template for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdR9T0wReSnbVg90H21EOLACvtfcHk0liPWAJFEIBiNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFanu7nE&md5=69c76953444dfbde4a3559507b4f8f4a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1813988115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1813988115%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHofmann%26aufirst%3DJ.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DSchaake%26aufirst%3DB.%26aulast%3DEitel%26aufirst%3DK.%26aulast%3DBartuschat%26aufirst%3DA.%26aulast%3DKaindl%26aufirst%3DJ.%26aulast%3DRampp%26aufirst%3DH.%26aulast%3DBanerjee%26aufirst%3DA.%26aulast%3DHubner%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DM.%2BJ.%26aulast%3DVincent%26aufirst%3DS.%2BG.%26aulast%3DFisher%26aufirst%3DJ.%2BT.%26aulast%3DHeinrich%26aufirst%3DM.%2BR.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DStructure-guided%2520development%2520of%2520selective%2520M3%2520muscarinic%2520acetylcholine%2520receptor%2520antagonists%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3D12046%26epage%3D12050%26doi%3D10.1073%2Fpnas.1813988115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawaramoto, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinoshita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaidehi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span> <span> </span><span class="NLM_article-title">Structural insights into the subtype-selective antagonist binding to the M<sub>2</sub> muscarinic receptor</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1150</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0152-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fs41589-018-0152-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=30420692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Sgur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1150-1158&author=R.+Sunoauthor=S.+Leeauthor=S.+Maedaauthor=S.+Yasudaauthor=K.+Yamashitaauthor=K.+Hirataauthor=S.+Horitaauthor=M.+S.+Tawaramotoauthor=H.+Tsujimotoauthor=T.+Murataauthor=M.+Kinoshitaauthor=M.+Yamamotoauthor=B.+K.+Kobilkaauthor=N.+Vaidehiauthor=S.+Iwataauthor=T.+Kobayashi&title=Structural+insights+into+the+subtype-selective+antagonist+binding+to+the+M2+muscarinic+receptor&doi=10.1038%2Fs41589-018-0152-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the subtype-selective antagonist binding to the M2 muscarinic receptor</span></div><div class="casAuthors">Suno, Ryoji; Lee, Sangbae; Maeda, Shoji; Yasuda, Satoshi; Yamashita, Keitaro; Hirata, Kunio; Horita, Shoichiro; Tawaramoto, Maki S.; Tsujimoto, Hirokazu; Murata, Takeshi; Kinoshita, Masahiro; Yamamoto, Masaki; Kobilka, Brian K.; Vaidehi, Nagarajan; Iwata, So; Kobayashi, Takuya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1150-1158</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Human muscarinic receptor M2 is one of the five subtypes of muscarinic receptors belonging to the family of G-protein-coupled receptors.  Muscarinic receptors are targets for multiple neurodegenerative diseases.  The challenge has been designing subtype-selective ligands against one of the five muscarinic receptors.  We report high-resoln. structures of a thermostabilized mutant M2 receptor bound to a subtype-selective antagonist AF-DX 384 and a nonselective antagonist NMS.  The thermostabilizing mutation S110R in M2 was predicted using a theor. strategy previously developed in our group.  Comparison of the crystal structures and pharmacol. properties of the M2 receptor shows that the Arg in the S110R mutant mimics the stabilizing role of the sodium cation, which is known to allosterically stabilize inactive state(s) of class A GPCRs.  Mol. dynamics simulations reveal that tightening of the ligand-residue contacts in M2 receptors compared to M3 receptors leads to subtype selectivity of AF-DX 384.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd5Qa0G7cW67Vg90H21EOLACvtfcHk0liPWAJFEIBiNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Sgur7N&md5=d99b0d7d43a8b17f6467682a7dd67f1c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0152-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0152-y%26sid%3Dliteratum%253Aachs%26aulast%3DSuno%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DMaeda%26aufirst%3DS.%26aulast%3DYasuda%26aufirst%3DS.%26aulast%3DYamashita%26aufirst%3DK.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DHorita%26aufirst%3DS.%26aulast%3DTawaramoto%26aufirst%3DM.%2BS.%26aulast%3DTsujimoto%26aufirst%3DH.%26aulast%3DMurata%26aufirst%3DT.%26aulast%3DKinoshita%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DVaidehi%26aufirst%3DN.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStructural%2520insights%2520into%2520the%2520subtype-selective%2520antagonist%2520binding%2520to%2520the%2520M2%2520muscarinic%2520receptor%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D1150%26epage%3D1158%26doi%3D10.1038%2Fs41589-018-0152-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutzler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschauer, A.</span></span> <span> </span><span class="NLM_article-title">Guanidine-acylguanidine bioisosteric approach in the design of radioligands: synthesis of a tritium-labeled N(G)-propionylargininamide ([<sup>3</sup>H]-UR-MK114) as a highly potent and selective neuropeptide Y Y<sub>1</sub> receptor antagonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">8168</span>– <span class="NLM_lpage">8172</span>, <span class="refDoi"> DOI: 10.1021/jm801018u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801018u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVaht73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=8168-8172&author=M.+Kellerauthor=N.+Popauthor=C.+Hutzlerauthor=A.+G.+Beck-Sickingerauthor=G.+Bernhardtauthor=A.+Buschauer&title=Guanidine-acylguanidine+bioisosteric+approach+in+the+design+of+radioligands%3A+synthesis+of+a+tritium-labeled+N%28G%29-propionylargininamide+%28%5B3H%5D-UR-MK114%29+as+a+highly+potent+and+selective+neuropeptide+Y+Y1+receptor+antagonist&doi=10.1021%2Fjm801018u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Guanidine-Acylguanidine Bioisosteric Approach in the Design of Radioligands: Synthesis of a Tritium-Labeled NG-Propionylargininamide ([3H]-UR-MK114) as a Highly Potent and Selective Neuropeptide Y Y1 Receptor Antagonist</span></div><div class="casAuthors">Keller, Max; Pop, Nathalie; Hutzler, Christoph; Beck-Sickinger, Annette G.; Bernhardt, Guenther; Buschauer, Armin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8168-8172</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthesis and characterization of (R)-Nα-(2,2-diphenylacetyl)-N-(4-hydroxybenzyl)-Nω-([2,3-3H]-propanoyl)argininamide (I, [3H]-UR-MK114), an easily accessible tritium-labeled NPY Y1 receptor (Y1R) antagonist (KB: 0.8 nM, calcium assay, HEL cells) derived from the (R)-argininamide BIBP 3226, is reported.  The radioligand binds with high affinity (KD, satn. : 1.2 nM, kinetic expts.: 1.1 nM, SK-N-MC cells) and selectivity for Y1R over Y2, Y4, and Y5 receptors.  The title compd. is a useful pharmacol. tool for the detn. of Y1R ligand affinities, quantification of Y1R binding sites, and autoradiog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreypW1UupwE7Vg90H21EOLACvtfcHk0liPWAJFEIBiNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVaht73I&md5=db217c63381985eb5cfa3a51bc37aaa4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm801018u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801018u%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DPop%26aufirst%3DN.%26aulast%3DHutzler%26aufirst%3DC.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DBernhardt%26aufirst%3DG.%26aulast%3DBuschauer%26aufirst%3DA.%26atitle%3DGuanidine-acylguanidine%2520bioisosteric%2520approach%2520in%2520the%2520design%2520of%2520radioligands%253A%2520synthesis%2520of%2520a%2520tritium-labeled%2520N%2528G%2529-propionylargininamide%2520%2528%255B3H%255D-UR-MK114%2529%2520as%2520a%2520highly%2520potent%2520and%2520selective%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D8168%26epage%3D8172%26doi%3D10.1021%2Fjm801018u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The ecstasy and agony of assay interference compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2165</span>– <span class="NLM_lpage">2168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00229</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00229" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2165-2168&author=C.+Aldrichauthor=C.+Bertozziauthor=G.+I.+Georgauthor=L.+Kiesslingauthor=C.+Lindsleyauthor=D.+Liottaauthor=K.+M.+Merzauthor=A.+Schepartzauthor=S.+Wang&title=The+ecstasy+and+agony+of+assay+interference+compounds&doi=10.1021%2Facs.jmedchem.7b00229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The Ecstasy and Agony of Assay Interference Compounds</span></div><div class="casAuthors">Aldrich, Courtney; Bertozzi, Carolyn; Georg, Gunda I.; Kiessling, Laura; Lindsley, Craig; Liotta, Dennis; Merz, Kenneth M.; Schepartz, Alanna; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2165-2168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ecstasy of discovering a new hit from screening can lead to a highly productive research effort to discover new bioactive compds.  However, in too many cases this ecstasy is followed by the agony of realizing that the compds. are not active against the desired target.  Many of these false hits are Pan Assay INterference compds. (PAINS) 1 or colloidal aggregators.  Whether the screen is conducted in silico or in the lab. and whether screening libraries, natural products, or drugs are used, all discovery efforts that rely on some form of screening to identify bioactivity are susceptible to this phenomenon.  Studies that omit crit. controls against exptl. artifacts caused by PAINS may waste years of research effort as useless compds. are progressed.  The American Chem. Society (ACS) is eager to alert the scientific community to this problem and to recommend protocols that will eliminate the publication of research articles based on compds. with artificial activity.  This editorial aims to summarize relevant concepts and to set the framework by which relevant ACS journals will address this issue going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJUf5u51VEpbVg90H21EOLACvtfcHk0lhf96MxgWWvBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D&md5=64bf087659a47fe7769208217ef8c989</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00229%26sid%3Dliteratum%253Aachs%26aulast%3DAldrich%26aufirst%3DC.%26aulast%3DBertozzi%26aufirst%3DC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DKiessling%26aufirst%3DL.%26aulast%3DLindsley%26aufirst%3DC.%26aulast%3DLiotta%26aufirst%3DD.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DSchepartz%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520ecstasy%2520and%2520agony%2520of%2520assay%2520interference%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2165%26epage%3D2168%26doi%3D10.1021%2Facs.jmedchem.7b00229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conklin, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farfel, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lustig, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, H. R.</span></span> <span> </span><span class="NLM_article-title">Substitution of three amino acids switches receptor specificity of Gq alpha to that of Gi alpha</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>363</i></span>,  <span class="NLM_fpage">274</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1038/363274a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2F363274a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8387644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1Sltbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=1993&pages=274-276&author=B.+R.+Conklinauthor=Z.+Farfelauthor=K.+D.+Lustigauthor=D.+Juliusauthor=H.+R.+Bourne&title=Substitution+of+three+amino+acids+switches+receptor+specificity+of+Gq+alpha+to+that+of+Gi+alpha&doi=10.1038%2F363274a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Substitution of three amino acids switches receptor specificity of Gqα to that of Giα</span></div><div class="casAuthors">Conklin, Bruce R.; Farfel, Zvi; Lustig, Kevin D.; Julius, David; Bourne, Henry R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">6426</span>),
    <span class="NLM_cas:pages">274-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">C-terminal amino acids of the Gq protein α-subunit were replaced with the corresponding residues of Gi2 protein α-subunit to create αq/αi2 chimeras that can mediate stimulation of phospholipase C by receptors otherwise coupled exclusively to Gi.  A min. of 3 αi2 amino acids, including a glycine 3 residues from the C terminus, suffices to switch the receptor specificity of the αq/αi2 chimeras.  Apparently, a C-terminal turn, centered on this glycine, plays an important part in specifying receptor interactions of G proteins in the Gi/G0/Gz family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq-ZuaQSMBYLVg90H21EOLACvtfcHk0lhf96MxgWWvBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkt1Sltbk%253D&md5=f2253edf096f40af1d6ae91c4823612f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2F363274a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F363274a0%26sid%3Dliteratum%253Aachs%26aulast%3DConklin%26aufirst%3DB.%2BR.%26aulast%3DFarfel%26aufirst%3DZ.%26aulast%3DLustig%26aufirst%3DK.%2BD.%26aulast%3DJulius%26aufirst%3DD.%26aulast%3DBourne%26aufirst%3DH.%2BR.%26atitle%3DSubstitution%2520of%2520three%2520amino%2520acids%2520switches%2520receptor%2520specificity%2520of%2520Gq%2520alpha%2520to%2520that%2520of%2520Gi%2520alpha%26jtitle%3DNature%26date%3D1993%26volume%3D363%26spage%3D274%26epage%3D276%26doi%3D10.1038%2F363274a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marti-Renom, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Comparative protein structure modeling of genes and genomes</span>. <i>Annu. Rev. Biophys. Biomol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1146/annurev.biophys.29.1.291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1146%2Fannurev.biophys.29.1.291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10940251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2000&pages=291-325&author=M.+A.+Marti-Renomauthor=A.+C.+Stuartauthor=A.+Fiserauthor=R.+Sanchezauthor=F.+Meloauthor=A.+Sali&title=Comparative+protein+structure+modeling+of+genes+and+genomes&doi=10.1146%2Fannurev.biophys.29.1.291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein structure modeling of genes and genomes</span></div><div class="casAuthors">Marti-Renom, Marc A.; Stuart, Ashley C.; Fiser, Andras; Sanchez, Roberto; Melo, Francisco; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">291-325</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review with 212 refs.  Comparative modeling predicts the three-dimensional structure of a given protein sequence (target) based primarily on its alignment to one or more proteins of known structure (templates).  The prediction process consists of fold assignment, target-template alignment, model building, and model evaluation.  The no. of protein sequences that can be modeled and the accuracy of the predictions are increasing steadily because of the growth in the no. of known protein structures and because of the improvements in the modeling software.  Further advances are necessary in recognizing weak sequence-structure similarities, aligning sequences with structures, modeling of rigid body shifts, distortions, loops and side chains, as well as detecting errors in a model.  Despite these problems, it is currently possible to model with useful accuracy significant parts of approx. one third of all known protein sequences.  The use of individual comparative models in biol. is already rewarding and increasingly widespread.  A major new challenge for comparative modeling is the integration of it with the torrents of data from genome sequencing projects as well as from functional and structural genomics.  In particular, there is a need to develop an automated, rapid, robust, sensitive, and accurate comparative modeling pipeline applicable to whole genomes.  Such large-scale modeling is likely to encourage new kinds of applications for the many resulting models, based on their large no. and completeness at the level of the family, organism, or functional network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq45B6Amw-zi7Vg90H21EOLACvtfcHk0li-I8IewUF6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlsFOnur0%253D&md5=d97ffe6c47c4a3bae2bbbf14c045afdd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.29.1.291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.29.1.291%26sid%3Dliteratum%253Aachs%26aulast%3DMarti-Renom%26aufirst%3DM.%2BA.%26aulast%3DStuart%26aufirst%3DA.%2BC.%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DMelo%26aufirst%3DF.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520of%2520genes%2520and%2520genomes%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2000%26volume%3D29%26spage%3D291%26epage%3D325%26doi%3D10.1146%2Fannurev.biophys.29.1.291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Comparative protein modelling by satisfaction of spatial restraints</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1006%2Fjmbi.1993.1626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=8254673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+Saliauthor=T.+L.+Blundell&title=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints&doi=10.1006%2Fjmbi.1993.1626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein modeling by satisfaction of spatial restraints</span></div><div class="casAuthors">Sali, Andrej; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-815</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe a comparative protein modeling method designed to find the most probable structure for a sequence given its alignment with related structures.  The three-dimensional (3D) model is obtained by optimally satisfying spatial restraints derived from the alignment and expressed as probability d. functions (pdfs) for the features restrained.  For example, the probabilities for main-chain conformations of a modelled residue may be restrained by its residue type, main-chain conformation of an equiv. residue in a related protein, and the local similarity between the two sequences.  Several such pdfs are obtained from the correlations between structural features in 17 families of homologous proteins which have been aligned on the basis of their 3D structures.  The pdfs restrain Cα-Cα distances, main-chain N-O distances, main-chain and side-chain dihedral angles.  A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database.  The 3D model of a protein is obtained by optimization of the mol. pdf such that the model violates the input restraints as little as possible.  The mol. pdf is derived as a combination of pdfs restraining individual spatial features of the whole mol.  The optimization procedure is a variable target function method that applies the conjugate gradients algorithm to positions of all non-hydrogen atoms.  The method is automated and is illustrated by the modeling of trypsin from two other serine proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC4wGhwZvib7Vg90H21EOLACvtfcHk0li-I8IewUF6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D&md5=d4a3c39b2205e36221dc187a3d1a478b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1626%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modelling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815%26doi%3D10.1006%2Fjmbi.1993.1626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Modeling of loops in protein structures</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1753</span>– <span class="NLM_lpage">1773</span>, <span class="refDoi"> DOI: 10.1110/ps.9.9.1753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1110%2Fps.9.9.1753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=11045621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2000&pages=1753-1773&author=A.+Fiserauthor=R.+K.+Doauthor=A.+Sali&title=Modeling+of+loops+in+protein+structures&doi=10.1110%2Fps.9.9.1753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling of loops in protein structures</span></div><div class="casAuthors">Fiser, Andras; Do, Richard Kinh Gian; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1753-1773</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Comparative protein structure prediction is limited mostly by the errors in alignment and loop modeling.  We describe here a new automated modeling technique that significantly improves the accuracy of loop predictions in protein structures.  The positions of all nonhydrogen atoms of the loop are optimized in a fixed environment with respect to a pseudo energy function.  The energy is a sum of many spatial restraints that include the bond length, bond angle, and improper dihedral angle terms from the CHARMM-22 force field, statistical preferences for the main-chain and side-chain dihedral angles, and statistical preferences for nonbonded at. contacts that depend on the two atom types, their distance through space, and sepn. in sequence.  The energy function is optimized with the method of conjugate gradients combined with mol. dynamics and simulated annealing.  Typically, the predicted loop conformation corresponds to the lowest energy conformation among 500 independent optimizations.  Predictions were made for 40 loops of known structure at each length from 1 to 14 residues.  The accuracy of loop predictions is evaluated as a function of thoroughness of conformational sampling, loop length, and structural properties of native loops.  When accuracy is measured by local superposition of the model on the native loop, 100, 90, and 30% of 4-, 8-, and 12-residue loop predictions, resp., had <2 Å RMSD error for the mainchain N, Cα, C, and O atoms; the av. accuracies were 0.59±0.05, 1.16±0.10, and 2.61±0.16 Å, resp.  To simulate real comparative modeling problems, the method was also evaluated by predicting loops of known structure in only approx. correct environments with errors typical of comparative modeling without misalignment.  When the RMSD distortion of the main-chain stem atoms is 2.5 Å, the av. loop prediction error increased by 180, 25, and 3% for 4-, 8-, and 12-residue loops, resp.  The accuracy of the lowest energy prediction for a given loop can be estd. from the structural variability among a no. of low energy predictions.  The relative value of the present method is gauged by (1) comparing it with one of the most successful previously described methods, and (2) describing its accuracy in recent blind predictions of protein structure.  Finally, it is shown that the av. accuracy of prediction is limited primarily by the accuracy of the energy function rather than by the extent of conformational sampling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsbG2px_r_krVg90H21EOLACvtfcHk0li-I8IewUF6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXntFamsL8%253D&md5=0bcdbfbddf54ecca0e3fee95ecdc3f8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1110%2Fps.9.9.1753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.9.9.1753%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DR.%2BK.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DModeling%2520of%2520loops%2520in%2520protein%2520structures%26jtitle%3DProtein%2520Sci.%26date%3D2000%26volume%3D9%26spage%3D1753%26epage%3D1773%26doi%3D10.1110%2Fps.9.9.1753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanborn, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husbands, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynor, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structural insights into micro-opioid receptor activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>524</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1038/nature14886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1038%2Fnature14886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=26245379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=524&publication_year=2015&pages=315-321&author=W.+Huangauthor=A.+Manglikauthor=A.+J.+Venkatakrishnanauthor=T.+Laeremansauthor=E.+N.+Feinbergauthor=A.+L.+Sanbornauthor=H.+E.+Katoauthor=K.+E.+Livingstonauthor=T.+S.+Thorsenauthor=R.+C.+Klingauthor=S.+Granierauthor=P.+Gmeinerauthor=S.+M.+Husbandsauthor=J.+R.+Traynorauthor=W.+I.+Weisauthor=J.+Steyaertauthor=R.+O.+Drorauthor=B.+K.+Kobilka&title=Structural+insights+into+micro-opioid+receptor+activation&doi=10.1038%2Fnature14886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into μ-opioid receptor activation</span></div><div class="casAuthors">Huang, Weijiao; Manglik, Aashish; Venkatakrishnan, A. J.; Laeremans, Toon; Feinberg, Evan N.; Sanborn, Adrian L.; Kato, Hideaki E.; Livingston, Kathryn E.; Thorsen, Thor S.; Kling, Ralf C.; Granier, Sebastien; Gmeiner, Peter; Husbands, Stephen M.; Traynor, John R.; Weis, William I.; Steyaert, Jan; Dror, Ron O.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">524</span>
        (<span class="NLM_cas:issue">7565</span>),
    <span class="NLM_cas:pages">315-321</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the μ-opioid receptor (μOR) is responsible for the efficacy of the most effective analgesics.  To shed light on the structural basis for μOR activation, here we report a 2.1 Å X-ray crystal structure of the murine μOR bound to the morphinan agonist BU72 and a G protein mimetic camelid antibody fragment Nb39 (nanobody 39).  The BU72-stabilized changes in the μOR binding pocket are subtle and differ from those obsd. for agonist-bound structures of the β2-adrenergic receptor (β2AR) and the M2 muscarinic receptor (M2R).  Comparison with active β2AR reveals a common rearrangement in the packing of three conserved amino acids in the core of the μOR, and mol. dynamics simulations illustrate how the ligand-binding pocket is conformationally linked to this conserved triad.  Addnl., an extensive polar network between the ligand-binding pocket and the cytoplasmic domains appears to play a similar role in signal propagation for all three G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkFUN1wheV07Vg90H21EOLACvtfcHk0lj4MApIWmvR1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM&md5=b9af3e1351b04539379532c127c02651</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature14886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14886%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DVenkatakrishnan%26aufirst%3DA.%2BJ.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DFeinberg%26aufirst%3DE.%2BN.%26aulast%3DSanborn%26aufirst%3DA.%2BL.%26aulast%3DKato%26aufirst%3DH.%2BE.%26aulast%3DLivingston%26aufirst%3DK.%2BE.%26aulast%3DThorsen%26aufirst%3DT.%2BS.%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DGranier%26aufirst%3DS.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DHusbands%26aufirst%3DS.%2BM.%26aulast%3DTraynor%26aufirst%3DJ.%2BR.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructural%2520insights%2520into%2520micro-opioid%2520receptor%2520activation%26jtitle%3DNature%26date%3D2015%26volume%3D524%26spage%3D315%26epage%3D321%26doi%3D10.1038%2Fnature14886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lomize, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomize, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogozheva, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosberg, H. I.</span></span> <span> </span><span class="NLM_article-title">OPM: orientations of proteins in membranes database</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">625</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btk023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1093%2Fbioinformatics%2Fbtk023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=16397007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=623-625&author=M.+A.+Lomizeauthor=A.+L.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+I.+Mosberg&title=OPM%3A+orientations+of+proteins+in+membranes+database&doi=10.1093%2Fbioinformatics%2Fbtk023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">OPM: Orientations of Proteins in Membranes database</span></div><div class="casAuthors">Lomize, Mikhail A.; Lomize, Andrei L.; Pogozheva, Irina D.; Mosberg, Henry I.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">623-625</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Orientations of Proteins in Membranes (OPM) database provides a collection of transmembrane, monotopic, and peripheral proteins from the Protein Data Bank whose spatial arrangements in the lipid bilayer have been calcd. theor. and compared with exptl. data.  The database allows anal., sorting, and searching of membrane proteins based on their structural classification, species, destination membrane, nos. of transmembrane segments and subunits, nos. of secondary structures, and the calcd. hydrophobic thickness or tilt angle with respect to the bilayer normal.  All coordinate files with the calcd. membrane boundaries are available for downloading.  The Internet site for the database is: http://opm.phar.umich.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1WjpHiq7e8LVg90H21EOLACvtfcHk0lj4MApIWmvR1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVeksro%253D&md5=8a9590a32e912509a53ca0bfc7ede4ad</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtk023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtk023%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DM.%2BA.%26aulast%3DLomize%26aufirst%3DA.%2BL.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26atitle%3DOPM%253A%2520orientations%2520of%2520proteins%2520in%2520membranes%2520database%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3D623%26epage%3D625%26doi%3D10.1093%2Fbioinformatics%2Fbtk023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine microsecond molecular dynamics simulations with AMBER on GPUs. 2. explicit solvent particle mesh Ewald</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3878</span>– <span class="NLM_lpage">3888</span>, <span class="refDoi"> DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3878-3888&author=R.+Salomon-Ferrerauthor=A.+W.+Gotzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&title=Routine+microsecond+molecular+dynamics+simulations+with+AMBER+on+GPUs.+2.+explicit+solvent+particle+mesh+Ewald&doi=10.1021%2Fct400314y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0ljDXJV7cWXhLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DGotz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520microsecond%2520molecular%2520dynamics%2520simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520explicit%2520solvent%2520particle%2520mesh%2520Ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">SPFP: Speed without compromise - a mixed precision model for GPU accelerated molecular dynamics simulations</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">374</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2013&pages=374-380&author=S.+Le%0AGrandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&title=SPFP%3A+Speed+without+compromise+-+a+mixed+precision+model+for+GPU+accelerated+molecular+dynamics+simulations&doi=10.1016%2Fj.cpc.2012.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0ljDXJV7cWXhLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520compromise%2520-%2520a%2520mixed%2520precision%2520model%2520for%2520GPU%2520accelerated%2520molecular%2520dynamics%2520simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+an+N%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0liHcbJDUq4yLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520an%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0liHcbJDUq4yLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A. E.</span></span> <span> </span><span class="NLM_article-title">Long-time-step molecular dynamics through hydrogen mass repartitioning</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1864</span>– <span class="NLM_lpage">1874</span>, <span class="refDoi"> DOI: 10.1021/ct5010406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5010406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1864-1874&author=C.+W.+Hopkinsauthor=S.+Le+Grandauthor=R.+C.+Walkerauthor=A.+E.+Roitberg&title=Long-time-step+molecular+dynamics+through+hydrogen+mass+repartitioning&doi=10.1021%2Fct5010406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Long-Time-Step Molecular Dynamics through Hydrogen Mass Repartitioning</span></div><div class="casAuthors">Hopkins, Chad W.; Le Grand, Scott; Walker, Ross C.; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1864-1874</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previous studies have shown that the method of hydrogen mass repartitioning (HMR) is a potentially useful tool for accelerating mol. dynamics (MD) simulations.  By repartitioning the mass of heavy atoms into the bonded hydrogen atoms, it is possible to slow the highest-frequency motions of the macromol. under study, thus allowing the time step of the simulation to be increased by up to a factor of 2.  In this communication, we investigate further how this mass repartitioning allows the simulation time step to be increased in a stable fashion without significantly increasing discretization error.  To this end, we ran a set of simulations with different time steps and mass distributions on a three-residue peptide to get a comprehensive view of the effect of mass repartitioning and time step increase on a system whose accessible phase space is fully explored in a relatively short amt. of time.  We next studied a 129-residue protein, hen egg white lysozyme (HEWL), to verify that the obsd. behavior extends to a larger, more-realistic, system.  Results for the protein include structural comparisons from MD trajectories, as well as comparisons of pKa calcns. via const.-pH MD.  We also calcd. a potential of mean force (PMF) of a dihedral rotation for the MTS [(1-oxyl-2,2,5,5-tetramethyl-pyrroline-3-methyl)methanethiosulfonate] spin label via umbrella sampling with a set of regular MD trajectories, as well as a set of mass-repartitioned trajectories with a time step of 4 fs.  Since no significant difference in kinetics or thermodn. is obsd. by the use of fast HMR trajectories, further evidence is provided that long-time-step HMR MD simulations are a viable tool for accelerating MD simulations for mols. of biochem. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfG1v1aGj1NrVg90H21EOLACvtfcHk0lhOi_wPAcxC5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksFSrsL0%253D&md5=6ca75b544b63523481de7836d11a3c1b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct5010406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5010406%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DC.%2BW.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DLong-time-step%2520molecular%2520dynamics%2520through%2520hydrogen%2520mass%2520repartitioning%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D1864%26epage%3D1874%26doi%3D10.1021%2Fct5010406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chow, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D. M.</span></span> <span> </span><span class="NLM_article-title">Isothermal-isobaric molecular dynamics simulations with Monte Carlo volume sampling</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1016/0010-4655(95)00059-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0010-4655%2895%2900059-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaK2MXps1Wrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1995&pages=283-289&author=K.-H.+Chowauthor=D.+M.+Ferguson&title=Isothermal-isobaric+molecular+dynamics+simulations+with+Monte+Carlo+volume+sampling&doi=10.1016%2F0010-4655%2895%2900059-O"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Isothermal-isobaric molecular dynamics simulations with Monte Carlo volume sampling</span></div><div class="casAuthors">Chow, Kim-Hung; Ferguson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">283-9</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A new pressure coupling method is described that combines Monte Carlo vol.-space sampling with traditional mol. dynamics calcns. to simulate the phys. properties of mol. systems under std. conditions.  The pressure is maintained by accepting or rejecting vol. moves of newly propagated configurations using the Metropolis algorithm with probability P(ΔV) = min(1, exp((-1/kT0){ΔE+P0ΔV-NkT0 ln[(V+ΔV)/V]})).  The sample is further coupled to an external temp. bath using a variation of Andersen's stochastic collisions.  The computational approach was implemented and applied to simulate the equil. properties of dense neon at various pressures.  The results show the method to be both reliable and efficient for maintaining isothermal-isobaric conditions and should prove effective for studying the pressure dependence of phys. properties in mol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqBZv_rh25XbVg90H21EOLACvtfcHk0lhOi_wPAcxC5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXps1Wrt7s%253D&md5=7c04ed3236a63fbadd0d0acb4e9d42ac</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2F0010-4655%2895%2900059-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0010-4655%252895%252900059-O%26sid%3Dliteratum%253Aachs%26aulast%3DChow%26aufirst%3DK.-H.%26aulast%3DFerguson%26aufirst%3DD.%2BM.%26atitle%3DIsothermal-isobaric%2520molecular%2520dynamics%2520simulations%2520with%2520Monte%2520Carlo%2520volume%2520sampling%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D1995%26volume%3D91%26spage%3D283%26epage%3D289%26doi%3D10.1016%2F0010-4655%2895%2900059-O" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salentin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haupt, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adasme, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, M.</span></span> <span> </span><span class="NLM_article-title">PLIP: fully automated protein-ligand interaction profiler</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">W443</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1093%2Fnar%2Fgkv315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=25873628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W443-447&author=S.+Salentinauthor=S.+Schreiberauthor=V.+J.+Hauptauthor=M.+F.+Adasmeauthor=M.+Schroeder&title=PLIP%3A+fully+automated+protein-ligand+interaction+profiler&doi=10.1093%2Fnar%2Fgkv315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PLIP: fully automated protein-ligand interaction profiler</span></div><div class="casAuthors">Salentin, Sebastian; Schreiber, Sven; Haupt, V. Joachim; Adasme, Melissa F.; Schroeder, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W443-W447</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The characterization of interactions in protein-ligand complexes is essential for research in structural bioinformatics, drug discovery and biol.  However, comprehensive tools are not freely available to the research community.  Here, we present the protein-ligand interaction profiler (PLIP), a novel web service for fully automated detection and visualization of relevant non-covalent protein-ligand contacts in 3D structures, freely available at projects.biotec.tu-dresden.de/plip-web.  The input is either a Protein Data Bank structure, a protein or ligand name, or a custom protein-ligand complex (e.g. from docking).  In contrast to other tools, the rule-based PLIP algorithm does not require any structure prepn.  It returns a list of detected interactions on single atom level, covering seven interaction types (hydrogen bonds, hydrophobic contacts, pi-stacking, pi-cation interactions, salt bridges, water bridges and halogen bonds).  PLIP stands out by offering publication-ready images, PyMOL sesions files to generate custom images and parsable result files to facilitate successive data processing.  The full python source code is available for download on the website.  PLIP's command-line mode allows for high-throughput interaction profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgMd-_C9BUCbVg90H21EOLACvtfcHk0lhOi_wPAcxC5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbrI&md5=4878881a7bd715e5585682876e102e65</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv315%26sid%3Dliteratum%253Aachs%26aulast%3DSalentin%26aufirst%3DS.%26aulast%3DSchreiber%26aufirst%3DS.%26aulast%3DHaupt%26aufirst%3DV.%2BJ.%26aulast%3DAdasme%26aufirst%3DM.%2BF.%26aulast%3DSchroeder%26aufirst%3DM.%26atitle%3DPLIP%253A%2520fully%2520automated%2520protein-ligand%2520interaction%2520profiler%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3DW443%26epage%3D447%26doi%3D10.1093%2Fnar%2Fgkv315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the inhibition constant (<i>K</i><sub>I</sub>) and the concentration of inhibitor which causes 50 per cent inhibition (<i>I</i><sub>50</sub>) of an enzymic reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0liZGXJT0EcVlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON','PDB','3UON'); return false;">PDB: 3UON</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZKB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZKB','PDB','5ZKB'); return false;">PDB: 5ZKB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CXV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CXV','PDB','5CXV'); return false;">PDB: 5CXV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DAJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DAJ','PDB','4DAJ'); return false;">PDB: 4DAJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DSG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DSG','PDB','5DSG'); return false;">PDB: 5DSG</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i25"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01967">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_33371"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01967">10.1021/acs.jmedchem.8b01967</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Analytical data of compounds <b>28</b>–<b>53</b>; Figures S1–S8; MD simulations with the M<sub>1</sub>R, M<sub>2</sub>R, M<sub>4</sub>R, and M<sub>5</sub>R bound to <b>38</b> and with the M<sub>1</sub>R–M<sub>5</sub>R bound to <b>48</b> (Figures S9–S15); Tables S1 and S2; RP-HPLC chromatograms of <b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, and <b>58</b>; <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>28</b>–<b>53</b>, <b>55</b>, <b>56</b>, and <b>58</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and muscarinic receptor binding data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_002.csv">CSV</a>)</p></li><li><p class="inline">Coordinates of the homology model of the hM<sub>5</sub>R, which was prepared based on a reported crystal structure of the M<sub>2</sub>R (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UON">3UON</a><a class="ref ref21" href="javascript:void(0);">(21)</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_001.pdf">jm8b01967_si_001.pdf (4.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_002.csv">jm8b01967_si_002.csv (4.22 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01967/suppl_file/jm8b01967_si_003.pdb">jm8b01967_si_003.pdb (181.78 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01967&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01967%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-11" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01967" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997fc9ce403c94","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
