
<html lang="en"     class="pb-page"  data-request-id="cdba6d94-14da-48b6-8842-24f3d26a3d49"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c00413"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A New Class of Dengue and West Nile Virus Protease Inhibitors with Submicromolar Activity in Reporter Gene DENV-2 Protease and Viral Replication Assays" /></meta><meta name="dc.Creator" content="Nikos  Kühl" /></meta><meta name="dc.Creator" content="Dominik  Graf" /></meta><meta name="dc.Creator" content="Josephine  Bock" /></meta><meta name="dc.Creator" content="Mira A. M.  Behnam" /></meta><meta name="dc.Creator" content="Mila-Mareen  Leuthold" /></meta><meta name="dc.Creator" content="Christian D.  Klein" /></meta><meta name="dc.Description" content="Dengue and West Nile virus are rapidly spreading global pathogens for which no specific therapeutic treatments are available. One of the promising targets for drug discovery against dengue and othe..." /></meta><meta name="Description" content="Dengue and West Nile virus are rapidly spreading global pathogens for which no specific therapeutic treatments are available. One of the promising targets for drug discovery against dengue and othe..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 1, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00413" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00413" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00413" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00413" /></link>
        
    
    

<title>A New Class of Dengue and West Nile Virus Protease Inhibitors with Submicromolar Activity in Reporter Gene DENV-2 Protease and Viral Replication Assays | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00413" /></meta><meta property="og:title" content="A New Class of Dengue and West Nile Virus Protease Inhibitors with Submicromolar Activity in Reporter Gene DENV-2 Protease and Viral Replication Assays" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0016.jpeg" /></meta><meta property="og:description" content="Dengue and West Nile virus are rapidly spreading global pathogens for which no specific therapeutic treatments are available. One of the promising targets for drug discovery against dengue and other flaviviruses is the viral serine protease NS2B-NS3. We present the design, synthesis, and in vitro and cellular characterization of a novel chemotype of potent small-molecule non-peptidic dengue protease inhibitors derived from 4-benzyloxyphenylglycine. A newly developed luciferase-based DENV-2 protease reporter system in HeLa cells (DENV2proHeLa) was employed to determine the activity of the compounds in a cellular environment. Specificity and selectivity of the DENV2proHeLa system were confirmed by viral titer reduction assays. The compounds reach low micromolar to upper nanomolar inhibitory potency in cell-based assays, are selective against other serine proteases, and do not show relevant cytotoxicity. An extensive structure–activity relationship study provides a perspective for further drug development against flaviviral infections." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00413"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00413">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00413&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00413&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00413&amp;href=/doi/10.1021/acs.jmedchem.0c00413" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 8179-8197</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00408" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00426" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A New Class of Dengue and West Nile Virus Protease Inhibitors with Submicromolar Activity in Reporter Gene DENV-2 Protease and Viral Replication Assays</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Nikos Kühl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nikos Kühl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Institute of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nikos++K%C3%BChl">Nikos Kühl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominik Graf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominik Graf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Institute of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominik++Graf">Dominik Graf</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Josephine Bock</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Josephine Bock</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Institute of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Josephine++Bock">Josephine Bock</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mira A. M. Behnam</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mira A. M. Behnam</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Institute of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mira+A.+M.++Behnam">Mira A. M. Behnam</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5839-7675" title="Orcid link">http://orcid.org/0000-0002-5839-7675</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mila-Mareen Leuthold</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mila-Mareen Leuthold</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Institute of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mila-Mareen++Leuthold">Mila-Mareen Leuthold</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7698-2360" title="Orcid link">http://orcid.org/0000-0001-7698-2360</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Christian D. Klein</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian D. Klein</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Institute of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#94f7bafff8f1fdfad4e1fafdb9fcf1fdf0f1f8f6f1e6f3baf0f1"><span class="__cf_email__" data-cfemail="a4c78acfc8c1cdcae4d1cacd89ccc1cdc0c1c8c6c1d6c38ac0c1">[email protected]</span></a>. Tel.: +49 6221 544875. Fax: +49 6221 546430.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian+D.++Klein">Christian D. Klein</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3522-9182" title="Orcid link">http://orcid.org/0000-0003-3522-9182</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00413&amp;href=/doi/10.1021%2Facs.jmedchem.0c00413" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 8179–8197</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 1, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 March 2020</li><li><span class="item_label"><b>Published</b> online</span>1 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00413" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00413</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8179%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DNikos%2BK%25C3%25BChl%252C%2BDominik%2BGraf%252C%2BJosephine%2BBock%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D15%26contentID%3Dacs.jmedchem.0c00413%26title%3DA%2BNew%2BClass%2Bof%2BDengue%2Band%2BWest%2BNile%2BVirus%2BProtease%2BInhibitors%2Bwith%2BSubmicromolar%2BActivity%2Bin%2BReporter%2BGene%2BDENV-2%2BProtease%2Band%2BViral%2BReplication%2BAssays%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8197%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00413"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1660</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00413" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A New Class of Dengue and West Nile Virus Protease Inhibitors with Submicromolar Activity in Reporter Gene DENV-2 Protease and Viral Replication Assays&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Nikos&quot;,&quot;last_name&quot;:&quot;Kühl&quot;},{&quot;first_name&quot;:&quot;Dominik&quot;,&quot;last_name&quot;:&quot;Graf&quot;},{&quot;first_name&quot;:&quot;Josephine&quot;,&quot;last_name&quot;:&quot;Bock&quot;},{&quot;first_name&quot;:&quot;Mira&quot;,&quot;last_name&quot;:&quot;A. M. Behnam&quot;},{&quot;first_name&quot;:&quot;Mila-Mareen&quot;,&quot;last_name&quot;:&quot;Leuthold&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;D. Klein&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8179-8197&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00413&quot;},&quot;abstract&quot;:&quot;Dengue and West Nile virus are rapidly spreading global pathogens for which no specific therapeutic treatments are available. One of the promising targets for drug discovery against dengue and other flaviviruses is the viral serine protease NS2B-NS3. We present the design, synthesis, and in vitro and cellular characterization of a novel chemotype of potent small-molecule non-peptidic dengue protease inhibitors derived from 4-benzyloxyphenylglycine. A newly developed luciferase-based DENV-2 protease reporter system in HeLa cells (DENV2proHeLa) was employed to determine the activity of the compounds in a cellular environment. Specificity and selectivity of the DENV2proHeLa system were confirmed by viral titer reduction assays. The compounds reach low micromolar to upper nanomolar inhibitory potency in cell-based assays, are selective against other serine proteases, and do not show relevant cytotoxicity. An extensive structure–activity relationship study provides a perspective for further drug development agains&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00413&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00413" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00413&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00413" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00413&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00413" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00413&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00413&amp;href=/doi/10.1021/acs.jmedchem.0c00413" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00413" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00413" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00413%26sid%3Dliteratum%253Aachs%26pmid%3D32605372%26genre%3Darticle%26aulast%3DK%25C3%25BChl%26date%3D2020%26atitle%3DA%2BNew%2BClass%2Bof%2BDengue%2Band%2BWest%2BNile%2BVirus%2BProtease%2BInhibitors%2Bwith%2BSubmicromolar%2BActivity%2Bin%2BReporter%2BGene%2BDENV-2%2BProtease%2Band%2BViral%2BReplication%2BAssays%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D15%26spage%3D8179%26epage%3D8197%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291719" title="Redox reactions">Redox reactions</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/jmcmar.2020.63.issue-15/20200813/jmcmar.2020.63.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Dengue and West Nile virus are rapidly spreading global pathogens for which no specific therapeutic treatments are available. One of the promising targets for drug discovery against dengue and other flaviviruses is the viral serine protease NS2B-NS3. We present the design, synthesis, and in vitro and cellular characterization of a novel chemotype of potent small-molecule non-peptidic dengue protease inhibitors derived from 4-benzyloxyphenylglycine. A newly developed luciferase-based DENV-2 protease reporter system in HeLa cells (DENV2proHeLa) was employed to determine the activity of the compounds in a cellular environment. Specificity and selectivity of the DENV2proHeLa system were confirmed by viral titer reduction assays. The compounds reach low micromolar to upper nanomolar inhibitory potency in cell-based assays, are selective against other serine proteases, and do not show relevant cytotoxicity. An extensive structure–activity relationship study provides a perspective for further drug development against flaviviral infections.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30862" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30862" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The global incidence of the mosquito-borne viral disease dengue has grown rapidly in the past years. A recent analysis estimates 390 million infections per year of which about one third are clinically recognized.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Dengue virus (DENV) infections can result in serious and life-threatening diseases like dengue fever, hemorrhagic fever, and dengue shock syndrome.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Despite an increasing impact of dengue fever, there are no specific therapeutic treatments available. Although a vaccine is approved in several countries, the vaccination of seronegative humans is not recommended.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> Hence, there is an unmet medical need for antiviral agents against infections with DENV and other flaviviruses like West Nile and Zika virus.</div><div class="NLM_p">Similar to DENV, the mosquito-borne West Nile Virus (WNV) is a pathogen of global occurrence. Although most infections with WNV are asymptomatic, many patients develop flu-like symptoms or more severe diseases like encephalitis. Unfortunately, neither an effective antiviral therapy nor a vaccine is available.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">As witnessed by the success of marketed drugs that target viral enzymes in hepatitis C virus (HCV)<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and in human immunodeficiency virus (HIV),<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> viral proteases are attractive targets for the development of antiviral drugs.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The flaviviral protease complex (NS2B-NS3) in DENV, WNV, etc. is responsible for cleavage of the viral polyprotein into separate functional proteins and is therefore essential for virus replication.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a></div><div class="NLM_p">The flaviviral proteases are serine proteases with a chymotrypsin-like fold and have a preference for substrates with dibasic motifs in P1 and P2.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Therefore, it is not surprising that most of the inhibitors developed so far contain highly charged recognition elements, frequently with a di- or polybasic motif. Combination of basic peptide sequences with an electrophilic group such as aldehyde or boronic acid, which are capable to form covalent bonds with the catalytic serine, results in high-affinity inhibitors.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> The efficacy of such inhibitors in cellular assay systems is often limited by pharmacokinetic liabilities due to high molecular weight and pronounced polarity of the highly basic side chains (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). In most of the accomplished DENV and WNV protease inhibitor studies, cellular activities were not considered in the compound design. Structure–activity relationships were mainly focused on inhibition of isolated viral proteases. This approach has, to our knowledge, not yet delivered any DENV or WNV protease inhibitors with potential for clinical application. Whereas several classes of inhibitors reach IC<sub>50</sub> values in the nanomolar range under biochemical assay conditions, the EC<sub>50</sub> values from cell-based assays are often significantly higher. The biochemical and cellular assay data collected in our group over the past years indicate a lack of correlation between these readouts. In other words, while the biochemical assay is highly useful to efficiently screen a large number of compounds and analyze structure-activity relationships on the molecular target interaction level, its utility to predict antiviral activity is limited. To address this shortcoming, we developed an HTS-capable intracellular DENV-2 protease screening assay based on a luciferase reporter system (DENV2proHeLa) that allows the determination of protease activity in a cellular context and integrates the simultaneous assessment of membrane permeability, metabolic stability, and cytotoxicity. Furthermore, the protease in DENV2proHeLa cells is capable of interacting with human host proteins and the ER membrane and can therefore be studied in a more biologically meaningful environment as compared to an isolated protease in a biochemical assay. For a selection of previously reported compounds from our group, we found a good correlation between data generated in the DENV2proHeLa and virus titer reduction assays.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Representative examples of previously reported peptidic and peptide hybrid inhibitors and their efficacy in a DENV-2 virus titer reduction assay.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Peptide backbones are marked in orange. (B) General structure of the inhibitor series presented here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last"><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B depicts the structure of the newly discovered compound class. Compounds in this series contain only one basic residue and have a much smaller molecular mass than most previously reported flaviviral protease inhibitors of similar or higher potency. These factors are expected to lead to increased bioavailability and higher activity in cellular assays.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> In contrast to the peptide-boronic acid inhibitors reported before,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> which have high target affinities but also interact strongly with off-targets such as thrombin, the new class of compounds do not incorporate an electrophilic element for covalent binding to the catalytic serine. Inherited from previous work is a phenylglycine (Phg) or 4-benzyloxyphenylglycine element.<a onclick="showRef(event, 'ref15 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref15 ref19 ref20">(15,19,20)</a> However, in the current compound class, this amino acid is coupled to lipophilic alkyl chains or ring systems via an amide bond and not to other peptidic elements as described before. The phenylglycine carboxylic acid moiety is connected via a second amide bond to a phenyl group that is preferably further substituted with basic residues. We herein present the synthesis, in vitro and cellular structure–activity relationships, off-target selectivity, and stability evaluation of this novel compound class.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">All final compounds and intermediates were obtained by solid-phase peptide synthesis according to the Fmoc protocol. The 4-benzyloxyphenylglycine derivatives (<b>59</b>–<b>67</b>) were obtained by O-alkylation of the corresponding 4-hydroxyphenylglycine as previously described.<a onclick="showRef(event, 'ref15 ref21'); return false;" href="javascript:void(0);" class="ref ref15 ref21">(15,21)</a> Phenylglycine derivatives were <i>N</i>-Fmoc protected (compounds <b>68</b>–<b>76</b>) as described by Dener et al.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Peptide coupling reactions were performed with COMU/TMP (on resin) or HATU / TMP (in solution) since it was shown that under those conditions, the α-carbon racemization of the phenylglycine derivative can be minimized to a negligible extent.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> As further detailed below, stereochemical purity was verified for the compounds presented here.</div><div class="NLM_p"><a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> depicts the synthetic approach for the on-resin-synthesis of compounds and their guanidinylation reactions. 6-Aminonicotinonitrile was reduced to the respective amine <b>77</b> using palladium on carbon under hydrogen atmosphere and selectively <i>N</i>-Fmoc protected using Fmoc-OSu under nitrogen atmosphere. The aniline nitrogen of the <i>N</i>-Fmoc protected 4-(aminomethyl)aniline or 5-(aminomethyl)pyridin-2-amine was attached to chlorotrityl chloride resin under basic conditions. After Fmoc deprotection, the respective compound was obtained by solid-phase peptide synthesis according to the Fmoc strategy. After cleavage from the resin with 5% TFA in DCM, the resulting aniline was converted to the respective di-Boc-guanidine using <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine, which is known for efficient guanidinylation reactions in peptide synthesis.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> This procedure remained unsuccessful for the 2-aminopyridine <b>12</b>, which was transformed into the respective guanidine moiety by treatment with <i>N</i>,<i>N</i>′-di-Boc-<i>S</i>-methylisothiourea and mercuric chloride in THF, as described before.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The Boc-protected guanidine moiety was deprotected using 50% TFA in DCM. All other compounds with guanidine groups were synthesized under similar conditions. The aniline compound <b>11</b> was reacted with sodium cyanate and acetic acid to afford the urea analog <b>24</b>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of amine-, guanidine-, and urea-containing compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Pd/C, conc. HCl, H<sub>2</sub>, EtOH, 24 h; (b) Fmoc-OSu, TEA, DMF, MeCN; (c) TMP, DMF, 16 h; (d) DCM/MeOH/TMP (80:15:5 v/v); (e) piperidine, DMF; (f) phenylglycine derivative, COMU, TMP, DMF; (g) N-terminal capping group, COMU, TMP, DMF; (h) 5% TFA in DCM; (i) <i>N</i>,<i>N</i>′-di-Boc-<i>S</i>-methylisothiourea, TEA, HgCl<sub>2</sub>, THF, 24 h; (j) TFA, DCM (1:1); (k) NaOCN, acetic acid, H<sub>2</sub>O (l) <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine, DMAP, DIPEA, MeOH, 48 h.</p></p></figure><div class="NLM_p">The synthesis of <b>9</b> and <b>18</b> is provided in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. 4-(Aminomethyl)benzonitrile (<b>56</b>) was synthesized from 4-cyanobenzylbromide, which was substituted to the azide and directly reduced to the respective amine with tin(II)chloride dihydrate in 94% overall yield. The benzamidine-containing compound <b>18</b> was obtained via the respective nitrile <b>9</b>: first, (<i>S</i>)-2-(4-(benzyloxy)phenyl)-2-hexanamidoacetic acid (<b>57</b>) was synthesized on chlorotrityl chloride resin and coupled to 4-(aminomethyl)benzonitrile with HATU/TMP in solution. The resulting compound <b>9</b> was treated with hydroxylamine to achieve the amidoxime intermediate, which was directly reduced using tin(II)chloride to the respective benzamidine <b>18</b>. Syntheses of caps <b>80</b>–<b>83</b> and other intermediates are described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supplementary Information</a>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>9</b> and <b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) NaN<sub>3</sub>, DMF, 60 °C; (b) SnCl<sub>2</sub>·H<sub>2</sub>O, MeOH, 0 °C - > rt; (c) NaOH; (d) HATU, TMP, DCM/DMF, 0 °C - > rt; (e) TFA, DCM; (f) NH<sub>2</sub>OH·HCl, TMP, THF, 60 °C; (g) SnCl<sub>2</sub>·H<sub>2</sub>O, DMF, 80 °C, 72 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structure–Activity Relationships</h3><div class="NLM_p">Four series of compounds were evaluated in biochemical enzymatic assays against DENV-2 and WNV proteases. Aprotinin and compounds <b>MB-8</b> and <b>MB-53</b> were used as inhibitor controls (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). All three compounds are known inhibitors of flaviviral proteases with high potencies in biochemical assays.<a onclick="showRef(event, 'ref15 ref19 ref26'); return false;" href="javascript:void(0);" class="ref ref15 ref19 ref26">(15,19,26)</a> Thrombin and trypsin were evaluated as potential off-targets. All compounds with low cytotoxicity (>80% signal compared to nontreated control) at the screening concentration were further evaluated in the DENV2proHeLa assay. The SAR exploration was focused on basic C-terminal moieties, different substitutions of the benzyl residue, and variation of lipophilic N-terminal caps. From a medicinal chemistry perspective, our strategy—the creation of compound libraries by building block assembly and derivatization—provides a fast track to diverse compounds, which are not easily accessible otherwise, considering for example the use of the racemization-prone phenylglycine in synthetic procedures. While this approach is inspired from amino acid assembly for peptides, it can be effectively applied on other chemical building blocks for lead optimization.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibitory Activity of Compounds with Different C-Terminal Residues against Isolated DENV and WNV Proteases, in the Cellular DENV Protease Reporter Assay (DENV2proHeLa) and Cytotoxicity in HeLa Cells</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0009.gif" alt="" id="gr9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0010.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 at 50 μM substrate concentration.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values against WNV NS2B-NS3 at 50 μM substrate concentration.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">EC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 in HeLa cells.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Measured in HeLa cells.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Measurements with aprotinin were performed with a concentration of 30 nM for DENV-2 and WNV proteases. If inhibition ≤10% = no inhibition (n.i.). n.d. = not determined. All measurements were carried out in triplicate.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Intracellular DENV Protease Reporter Gene System (DENV2proHeLa)</h3><div class="NLM_p">In order to study inhibitory activity in a target-specific, intracellular high-throughput format, we developed a DENV serotype 2 protease reporter assay in HeLa cells. This reporter system was recently used to evaluate prodrug activation by the DENV protease.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p">In order to preserve the natural conformation and interaction of NS2B and NS3<sub>pro</sub>, NS2B and NS3 are present as full-length proteins in the DENV2proHeLa system. The <i>Renilla reniformis</i> luciferase was used as the reporter enzyme. The N-terminus of the luciferase is connected via a dengue protease cleavage site to the oxygen-dependent degradation domain (ODD) of HIF1α (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The ODD domain initiates rapid proteolytic degradation of the fused protein by ubiquitination of PEST sequences within this domain.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31 ref32">(28−32)</a> The linkage of a luciferase reporter to an HIF-1α ODD degradation domain, its degradation by the proteasome, and the use for compound screenings in cells and in vivo were shown before.<a onclick="showRef(event, 'ref33 ref34 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35 ref36 ref37">(33−37)</a> The natural recognition sites NS2A/NS2B, NS3/NS4A, and NS4B/NS5 of the flaviviral polyprotein were explored as cleavage sites for the linker.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> When co-transfected with a mammalian expression plasmid coding for the DENV protease, incorporation of the NS2A/NS2B cleavage site in the linker region resulted in the most significant increase in luciferase activity, as compared to other polyprotein cleavage sites (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Figure S1</a>). This increase was not observed when a DENV protease construct without catalytic competence (S135A mutant) was co-transfected. Recently, sensors of cellular flavivirus NS2B-NS3 cleavage activity have been generated by linking GFP-based reporters with protease cleavage sites to a quenching peptide. Those sensors reported a fluorescence increase in stably transduced cells upon DENV-2 infection.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. DENV-2 reporter gene assay (DENV2proHeLa). (A) Schematic representation of the reporter construct used for co-transfection with the DENV-2 protease. (B) Relationship of inhibitory potencies at single concentration measurements in DENV2proHeLa versus virus titer reduction assay (compounds screened at their respective highest nontoxic concentration). (C) Relationship of EC<sub>50</sub> values from the virus titer reduction assay of known compounds versus EC<sub>50</sub> values derived from DENV2proHeLa. Compound identifiers and values are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supporting Information</a>. Graphs 2B and 2C were generated using Prism 6.01 (Graphpad Software, Inc.). Statistical parameters: 2B: <i>R</i><sup>2</sup> = 0.76, Sy.x = 13.89, <i>p</i> < 0.0001; 2C: <i>R</i><sup>2</sup> = 0.75, Sy.x = 0.24, <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The stable co-transfection of the DENV-2 NS2B/NS3 protease and the ODD-luciferase reporter construct resulted in a cell clone (DENV2proHeLa) with satisfying characteristics and wild-type morphology. Integration of the two sequences was achieved without destruction of relevant genes or regulatory sequences within the genome, and sequencing of the amplified inserts proved the integrity of both constructs. Several published inhibitors with known antiviral activity were used to further explore the characteristics of the system and its utility for compound screening (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_p last">Treatment of DENV2proHeLa cells with well-characterized inhibitors of the DENV protease had a pronounced effect on the luciferase signal, already after 24 h of treatment. EC<sub>50</sub> values determined in the DENV2proHeLa system were highly reproducible, operator-independent, and correlated well with previous data from virus titer reduction assays (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C). The statistical parameters <i>Z</i>′ factor and signal window were calculated for four independent experiments (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Figure S2</a> for further details). The <i>Z</i>′ factor was above 0.5 for all experiments, indicating good applicability for high-throughput screening. EC<sub>50</sub> values, exemplary dose–response curves, and structures of published covalent and noncovalent inhibitors that were used for the correlation are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supplementary Information</a>. Hence, we conclude that DENV2proHeLa is a valuable system to evaluate compounds targeted against the DENV protease in a high-throughput format before proceeding to a virus titer reduction assay.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Variation of the C-Terminal Residue</h3><div class="NLM_p last">Basic residues were primarily evaluated at this position (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The lysine and alkylamine analogs <b>1</b>–<b>4</b> showed no or minor inhibition in the biochemical assays, similar to nonbasic or weakly basic compounds (e.g., <b>6</b>, <b>7</b>, <b>9</b>, <b>11</b>). The inhibition increased with higher basicity of the residues. Especially phenylalkylamines or phenylguanidines led to potent inhibitors. <i>K</i><sub>i</sub> values of 3.8 μM for <b>14</b> and 9.8 μM for <b>20</b> at DENV-2 protease were determined using Cheng–Prusoff plots (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supplementary Information</a>). Compounds <b>24</b> and <b>25</b> can form similar hydrogen bonds with the protease and were designed as nonbasic guanidine replacements. However, a basic character is required at this position since a significant decrease of inhibition in all assays was measured. Surprisingly, the amidine-containing compound <b>18</b> had weak activity. Meta-substituted derivatives had lower potency compared to their para-substituted derivatives (e.g., <b>15</b>, <b>23</b>). A methylene group between the amide nitrogen and the phenyl ring was needed for optimal activity (<b>19</b> vs <b>20</b>). Most compounds had similar activities in the biochemical and the cellular protease assays, but compounds <b>13</b>, <b>14</b>, and <b>19</b> could not be measured due to cytotoxicity at the screening concentration. Some of the compounds had higher activity in the cellular environment than in an isolated protease assay, an observation that was made before by other groups and was regarded as a consequence of the artificial composition of the biochemical assay,<a onclick="showRef(event, 'ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42 ref43">(40−43)</a> which is carried out at basic pH in the presence of detergents and polyols.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Other hypothetical explanations for increased cellular activity can be an intracellular accumulation or the requirement of a nearly complete processing of the viral polyprotein for viral infectivity.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Noteworthy is the upper-nanomolar DENV2proHeLa EC<sub>50</sub> of compound <b>20</b>. The <i>N</i>,<i>N</i>-dimethylamine derivative <b>16</b> showed no inhibition in the biochemical assays but a high inhibition in the cell-based assay, which could be due to intracellular demethylation to yield the monomethylamine derivative of compound <b>14</b>. The latter compound was a potent inhibitor in the biochemical assays, but could not be evaluated in cells due to its low CC<sub>50</sub> of 4.8 μM. Various heterocycles were evaluated: The 1<i>H</i>-indazole-containing compound <b>8</b> had an EC<sub>50</sub> of around 7 μM. The pyridine-containing compounds <b>7</b> and <b>12</b> inhibited 50% of the protease at 12.5 μM in cells, whereas 1-(pyridin-2-yl)guanidine-containing <b>21</b> had an EC<sub>50</sub> of 6 μM.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Variation of the Benzyloxyphenylglycine Moiety</h3><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> presents combinations of the C-terminal 4-phenylguanidine with variable benzyloxyphenylglycine moieties. The benzyloxy residue is essential for activity. The phenylglycine-containing compound <b>26</b> was inactive in all assays. Furthermore, the unsubstituted <span class="smallcaps smallerCapital">d</span>-derivative (<b>27</b>) had a 20-fold lower inhibition compared to the <span class="smallcaps smallerCapital">l</span>-isomer (<b>20</b>). This is in strong contrast to previously published inhibitor structures, which showed about 3-fold higher activity for <span class="smallcaps smallerCapital">d</span>-derivatives<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and may be seen as an indication for a fundamentally different binding mode of the present new chemotype. The electron-withdrawing halogen substituent containing compounds in <span class="smallcaps smallerCapital">l</span>- and <span class="smallcaps smallerCapital">d</span>-configurations had a 2- to 5-fold higher inhibitory potency in biochemical assays (<b>28</b>, <b>29</b>, <b>32</b>, <b>33</b>). However, substitutions at the benzyloxy residue were not well tolerated in the DENV2proHeLa assay. Other substituents like 3-methoxy, 4-cyano, or 4-<i>tert</i>-butyl decreased the activity of the compounds in all assays.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Activity of Compounds with Variations in the Benzyloxyphenylglycine Moiety against Isolated DENV and WNV Protease, in the Cellular DENV Protease Reporter Assay (DENV2proHeLa) and Cytotoxicity in HeLa Cells</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0011.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0012.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 at 50 μM substrate concentration.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values against WNV NS2B-NS3 at 50 μM substrate concentration.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">EC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 in HeLa cells.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Measured in HeLa cells. If inhibition ≤10% = no inhibition (n.i.). n.d. = not determined. All measurements were carried out in triplicate.</p></div></div><div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Variation of the N-Terminal Cap</h3><div class="NLM_p">Mostly, lipophilic alkyl and phenyl residues were evaluated as N-terminal caps. The length of the alkyl chains had a major influence on the activity, with clear dependencies between lipophilicity and activities in the biochemical and cellular assays. The minor inhibition by the acetyl analog in the cell-based assay could be increased by extending the chain length. The hexanoic acid cap (<b>20</b>) showed a nearly 6-fold higher activity compared to the valeric acid cap (<b>36</b>). The heptanoic and octanoic acid caps gave comparable EC<sub>50</sub> values of 0.52 and 0.83 μM, respectively. Those compounds are therefore novel promising structures with high activity in a cellular environment without relevant cytotoxicity. The positive effect of chain length on potency tapered off in the C10 region and at more complex (cyclo-) aliphatic caps. A benzoic acid cap led to low inhibition, whereas cyclohexyloxy-, phenyloxy-, and benzyloxybenzoic acid derivatives showed a 2- or 3-fold higher inhibition in the biochemical assays compared to compound <b>20</b>. For these inhibitors, comparable EC<sub>50</sub> values were measured in the cellular protease assay. With a 4-chloro substituent (<b>46</b>), a nearly 5-fold higher inhibition in biochemical assays was achieved, whereas the 3-methoxy substituent (<b>47</b>) led to comparable inhibition to the unsubstituted derivative (<b>45</b>). Similar values were observed in the cellular protease assay for those derivatives except for <b>46</b>, which was significantly less potent.</div><div class="NLM_p last">An up to 9-fold higher activity was observed with the biphenyl-4-carboxylic acid and 2-(biphenyl-4-yl)acetic acid caps in the DENV biochemical assay, and an up to 7-fold higher activity was observed in the WNV assay. Note that 2-(biphenyl-4-yl)acetic acid (INN: felbinac) is a clinically used cyclooxygenase inhibitor.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The incorporation of biphenyl-2-carboxylic acid as a cap led to 2-fold lower inhibition compared to that of compound <b>20</b>. Most improvements of the biphenyl caps could not be followed up in the DENV2proHeLa assay, where the long alkyl chain cap-containing compounds achieved the highest activities (e.g., <b>20</b>). Especially compound <b>52</b> showed a big deviation between the biochemical vs cellular protease assays: The analog with a 2-(biphenyl-4-yl)acetic acid (= felbinac) cap had the highest in vitro activity but reached only an EC<sub>50</sub> of 20 μM in the DENV2proHeLa system. However, future work may be aimed at the exploration of the wide diversity of clinically used arylacetic and arylpropionic acids as N-terminal capping moieties. Some of the caps resulted in cytotoxicity at 50 μM (<b>42</b>, <b>43</b>, <b>45</b>, <b>46</b>, <b>47</b>), but at the DENV2proHeLa screening concentration (12.5 μM), no cytotoxicity was observed.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Fragment Merging</h3><div class="NLM_p">Selected fragments of the compounds described in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> were merged, thus combining the most active residues in the biochemical assays (see <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Compound <b>53</b> showed promising inhibition in biochemical assays but could not be further evaluated due to its cytotoxicity. Compounds <b>54</b> and <b>55</b> are combinations of the most active fragments in the biochemical assays. The combination of 2,6-dichloro-(4-benzyloxy)-phenylglycine, 2-(biphenyl-4-yl)acetic acid, and the phenylguanidine moiety afforded compound <b>55</b> with upper nanomolar potencies in biochemical assays. <i>K</i><sub>i</sub> values were measured in DENV and WNV biochemical assays and were found to be 0.46 and 1.0 μM, respectively. In spite of the promising potency in in vitro assays, compound <b>55</b> had significantly lower activity (EC<sub>50</sub>: 39 μM) in the DENV2proHeLa assay. The same observation was made for compound <b>54</b>. The high molecular masses of these compounds (683 and 667 g/mol) may result in an inferior pharmacokinetic profile and low membrane permeability. Compounds <b>32</b> and <b>52</b>, which contain the same structural moieties as compound <b>55</b>, were also significantly less potent in the cell-based assay in comparison to compound <b>20</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Inhibitory Activity of Compounds with Variation of the N-Terminal Cap against Isolated DENV and WNV Protease, in the Cellular DENV Protease Reporter Assay (DENV2proHeLa) and Cytotoxicity in HeLa Cells</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0013.gif" alt="" id="gr13" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0014.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 at 50 μM substrate concentration.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values against WNV NS2B-NS3 at 50 μM substrate concentration.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">EC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 in HeLa cells.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Measured in HeLa cells.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">Measured with HPLC-based protease assay. If inhibition ≤10% = no inhibition (n.i.). n.d. = not determined. All measurements were carried out in triplicate.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Inhibitory Activity of Merged Compounds against Isolated DENV and WNV Protease, in the Cellular DENV Protease Reporter Assay (DENV2proHeLa) and Cytotoxicity in HeLa Cells</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0015.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 at 50 μM substrate concentration.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values against WNV NS2B-NS3 at 50 μM substrate concentration.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">EC<sub>50</sub> values against DENV NS2B-NS3 serotype 2 in HeLa cells.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Measured in HeLa cells. If inhibition ≤10% = no inhibition (n.i.). n.d. = not determined. All measurements were carried out in triplicate.</p></div></div><div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Membrane Permeability</h3><div class="NLM_p">The passive membrane permeabilities of compounds <b>20</b>, <b>32</b>, <b>37</b>, <b>38</b>, <b>39</b>, <b>42</b>, <b>43</b>, <b>44</b>, <b>45</b>, <b>47</b>, <b>52</b>, <b>54</b>, and <b>55</b> and a set of drug references were evaluated using the precoated trilayer parallel artificial membrane permeability assay (PAMPA).<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> The permeability of the reference compounds was in accordance with literature values,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> but compounds <b>32</b>, <b>39</b>, <b>45</b>, <b>47</b>, <b>52</b>, <b>54</b>, and <b>55</b> appeared to be nonpermeable (see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). This offers an explanation for the lower activity of compounds <b>32</b>, <b>52</b>, <b>54</b>, and <b>55</b> in the DENV2proHeLa system compared to their inhibition in the biochemical DENV assay, although an active transport or endocytotic uptake of these basic compounds is possible, especially in hepatocytes such as the Huh-7 cell line used for the viral replication experiments. Compounds <b>37</b>, <b>38</b>, <b>42</b>, <b>43</b>, and <b>44</b>, which had high activities in the DENV2proHeLa system, are somewhat permeable but significantly less than the tested standards caffeine, carbamazepine, and phenytoin. The most active inhibitor in the DENV2proHeLa assay (compound <b>20)</b> showed significantly higher membrane permeability than the other compounds. Notably, the molecular masses of the cell-active compounds <b>20</b> (502 g/mol), <b>37</b> (516 g/mol), and <b>38</b> (530 g/mol) are very close to the Lipinski criteria,<a onclick="showRef(event, 'ref17 ref48'); return false;" href="javascript:void(0);" class="ref ref17 ref48">(17,48)</a> whereas compounds <b>32</b>, <b>39</b>, <b>42</b>, <b>43</b>, <b>44</b>, <b>45</b>, <b>47</b>, <b>52</b>, <b>54</b>, and <b>55</b> violate this rule.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Passive Membrane Permeability of Selected Compounds Determined by PAMPA</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="char" char="." /></col><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" align="center" char="."><i>M</i> (g/mol)</th><th class="colsep0 rowsep0" align="center"><i>C</i><sub>Acc</sub> (μM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="±"><i>C</i><sub>Don</sub> (μM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="."><i>P<sub>e</sub></i> (10<sup>–6</sup> cm/s)<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i> (%)<a class="ref internalNav" href="#t5fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">caffeine</td><td class="colsep0 rowsep0" align="char" char=".">195</td><td class="colsep0 rowsep0" align="center">31.9 ± 3.7</td><td class="colsep0 rowsep0" align="char" char="±">159.5 ± 21.5</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">9.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">carbamazepine</td><td class="colsep0 rowsep0" align="char" char=".">236</td><td class="colsep0 rowsep0" align="center">35.7 ± 5.2</td><td class="colsep0 rowsep0" align="char" char="±">171.6 ± 6.7</td><td class="colsep0 rowsep0" align="char" char=".">8.1</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">phenytoin</td><td class="colsep0 rowsep0" align="char" char=".">252</td><td class="colsep0 rowsep0" align="center">23.4 ± 1.7</td><td class="colsep0 rowsep0" align="char" char="±">149.6 ± 1.4</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">17.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">MB-53</td><td class="colsep0 rowsep0" align="char" char=".">645</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">502</td><td class="colsep0 rowsep0" align="center">7.5 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">171.0 ± 11.4</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>32</b></td><td class="colsep0 rowsep0" align="char" char=".">569</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>37</b></td><td class="colsep0 rowsep0" align="char" char=".">516</td><td class="colsep0 rowsep0" align="center">1.9 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">167.3 ± 16.8</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">15.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>38</b></td><td class="colsep0 rowsep0" align="char" char=".">530</td><td class="colsep0 rowsep0" align="center">1.5 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">164.5 ± 18.4</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">17.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>39</b></td><td class="colsep0 rowsep0" align="char" char=".">558</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>42</b></td><td class="colsep0 rowsep0" align="char" char=".">606</td><td class="colsep0 rowsep0" align="center">1.6 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">177.4 ± 16.2</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>43</b></td><td class="colsep0 rowsep0" align="char" char=".">600</td><td class="colsep0 rowsep0" align="center">1.2 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">154.5 ± 19.1</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">22.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>44</b></td><td class="colsep0 rowsep0" align="char" char=".">600</td><td class="colsep0 rowsep0" align="center">1.1 ± 0.0</td><td class="colsep0 rowsep0" align="char" char="±">168.9 ± 17.8</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">15.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>45</b></td><td class="colsep0 rowsep0" align="char" char=".">614</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>47</b></td><td class="colsep0 rowsep0" align="char" char=".">644</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>52</b></td><td class="colsep0 rowsep0" align="char" char=".">598</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>54</b></td><td class="colsep0 rowsep0" align="char" char=".">683</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>55</b></td><td class="colsep0 rowsep0" align="char" char=".">667</td><td class="colsep0 rowsep0" colspan="4" align="center">not permeable in the detectable range</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Concentration of the compound in the acceptor plate after 5 h of incubation.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Concentration of the compound in the donor plate after 5 h of incubation.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Permeability of the compound calculated according to the literature.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last">Mass retention of the compound calculated according to the literature.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></p></div></div></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Selectivity and Binding Mode</h3><div class="NLM_p">To determine the selectivity of the inhibitors, biochemical assays with the serine proteases thrombin and trypsin were performed (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supporting Information</a>). None of the compounds showed significant inhibition of thrombin at 25 μM, which stands in marked contrast to some of the flaviviral protease inhibitor series before, in particular the peptide boronic acids.<a onclick="showRef(event, 'ref14 ref49'); return false;" href="javascript:void(0);" class="ref ref14 ref49">(14,49)</a> Three of the compounds were moderately active against trypsin in a screen (above 50% inhibition at 50 μM). The IC<sub>50</sub> of compounds <b>18</b>, <b>27</b>, and <b>48</b> against trypsin were determined to be 45, 44, and 31 μM, respectively. All other compounds had low off-target activities.</div><div class="NLM_p">Cheng–Prusoff plots of several inhibitors in biochemical assays with DENV and WNV proteases indicate a competitive inhibition mechanisms for this inhibitor series (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supporting Information</a>). Furthermore, a tryptophan quenching assay<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> was used to study the binding mode of a selected compound at DENV-2 protease. For that purpose,4 compound <b>51</b> was designed because 2,2′-bithiophene-5-carboxylic acid quenches tryptophan fluorescence effectively.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Compound <b>51</b> had an IC<sub>50</sub> of approximately 6 μM against DENV and WNV in in vitro assays. The results of the fluorescence quenching experiments indicate a binding of the inhibitor in the active site of the enzyme. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the addition of compound <b>51</b> to DENV-2 protease results in concentration-dependent quenching of the fluorescence of tryptophan residues of the protease. Aprotinin is a known competitive inhibitor for DENV protease and was also co-crystallized with DENV-3 protease.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Aprotinin displaces compound <b>51</b> from the DENV-2 protease, providing further evidence for specific binding at the active site (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Addition of compound <b>51</b> to DENV-2 protease led to concentration-dependent quenching of the fluorescence of the tryptophan residues in the substrate-binding region. The fluorescence is partially restored when aprotinin is added. NATA (<i>N</i>-acetyl-<span class="smallcaps smallerCapital">l</span>-tryptophanamide) is used as control for inner filter effects.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Compound <b>51</b> can be used as a quencher for the protease autofluorescence, particularly in displacement assays for compounds that lack moieties capable of interacting with tryptophan fluorescence.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In comparison to commercially available inhibitors for the flaviviral protease (e.g., aprotinin, ARDP0006), the inhibitor offers an advantage through its high potency in both biochemical and cellular assays.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Enantiomeric Purity of Compounds</h3><div class="NLM_p last">The acidity of the α-proton in phenylglycine results in a relatively high racemization tendency of this building block.<a onclick="showRef(event, 'ref23 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref23 ref51 ref52">(23,51,52)</a> Stereochemical purity and stability of the compounds were assessed by chiral HPLC. Exemplary compounds were analyzed, and enantiomeric purity was above 95% in all cases (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supplementary Information</a> for more details). Additionally, compound <b>20</b> was tested for its racemization tendency in assay buffers used in this study. Racemization was negligible after incubation for 96 h at 37 °C in phosphate-buffered saline (pH 7.4), biochemical DENV assay buffer (pH 9.0), and DENV2proHeLa medium (pH 7.0). A highly basic sodium hydroxide solution (pH 13) and an incubation of 96 h were required for full racemization of the tested compound (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supplementary Information</a>).</div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Docking Studies</h3><div class="NLM_p">Docking studies were performed using GOLD 5.1<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and the X-ray structure of DENV-3 protease with a resolution of 2.3 Å reported by Noble et al. (PDB: 3U1I).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The crystal structure of the DENV-3 protease in complex with a tetrapeptidyl aldehyde was removed from the dimer structure. All water molecules, sulfate ions, and the covalently bound peptidyl aldehyde inhibitor were removed. The structure by Noble et al. was selected because no X-ray structure with an inhibitor bound to the closed form of DENV-2 protease is available and serotypes 2 and 3 have a high degree of conservation of the active sites. Docking studies were focused on compounds <b>20</b> and <b>43</b>. Compound <b>20</b> was chosen for docking studies because of its high activity in the cellular DENV2proHeLa assay. Inhibitor <b>43</b> showed high potencies in biochemical and cellular assays and contains a 4-phenoxybenzoic acid cap. The docking studies were done mainly to evaluate whether both the flexible hexanoic acid cap and the large, relatively rigid 4-phenoxybenzoic acid cap show a tendency to bind in the same binding area in the active site of the enzyme, and the simulations support this notion (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The positively charged phenylguanidine moiety is placed in the S1 pocket, where it forms an ionic interaction with the negatively charged aspartate residue on the bottom of the pocket. It is known that basic residues are preferred in this pocket.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (4-Benzyloxy)-<span class="smallcaps smallerCapital">l</span>-phenylglycine is located in the S2 pocket. The phenyl ring of the phenylglycine is in close contact with the catalytic histidine with a possibility to form Π–Π-stacking interactions. Phenylglycine and its derivatives were already described as preferred moieties in HCV protease inhibitors at this position.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> The hexanoic acid (compound <b>20</b>) and the 4-phenoxybenzoic acid cap (compound <b>43</b>) were placed in the S3 and S4 pockets where mainly lipophilic interactions occur.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking studies of compounds <b>20</b> and <b>43</b> at DENV-3 protease (PDB: 3U1I). (A) Ribbon model of the protease with chosen side chains shown in stick representation. The protease domain is colored in red, and the cofactor is colored in blue. Distances between the ligand and protein atoms are shown in black. (B) Docking pose of compound <b>20</b>. (C) Docking pose of compound <b>43</b>. The protein surface in pictures B and C is colored by amino acid hydrophobicity, with light blue indicating hydrophilic and orange indicating hydrophobic regions. The ligands are represented as sticks. The positions of the specificity pockets are indicated. Figures were generated using the Chimera software 1.11.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Antiviral Activity in Virus Titer Reduction Assay</h3><div class="NLM_p">The compounds with the highest potency in the DENV2proHeLa assay were further evaluated in a virus titer reduction assay using DENV-2-infected human hepatic carcinoma cells (Huh-7). The previously reported DENV-2 inhibitor <b>MB-53</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> was used as the reference compound. No cytotoxic effects were observed at the chosen concentration ranges. All tested compounds showed reduction of viral titers at a concentration of 12.5 μM (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Whereas the pyridin-2-yl guanidine derivative <b>21</b> and the 4-benzyloxybenzoic acid derivatives <b>45</b> and <b>47</b> showed moderate activity, treatment with compounds <b>20</b>, <b>22</b>, <b>37</b>, <b>43</b>, and <b>49</b> reduced viral titers to less than 25% of the control. In addition, antiviral dose–response curves were obtained for three compounds: <b>20</b>, <b>38</b>, and <b>43</b> reduced viral titers at least by 50% at 12.5 μM (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) and had EC<sub>50</sub> values between 0.49 and 3 μM in DENV2proHeLa. When comparing the cellular data of compounds <b>20</b> and compounds <b>36</b>–<b>39</b>, which differ only in the length of their alkyl chain, maximum potency was obtained for a C6 chain with a drop in potencies for longer alkyl chains. In accordance with inhibition in DENV2proHeLa, all three compounds reduced the virus titer with EC<sub>50</sub> values in the low micromolar range. Compound <b>20</b> had the most promising EC<sub>50</sub> of 0.89 μM. In contrast, compounds <b>38</b> and <b>43</b> had EC<sub>50</sub> values of 7.2 and 9.3 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibitory activity of selected compounds and the reference compound <b>MB-53</b> at a concentration of 12.5 μM in the DENV-2 virus titer reduction assay. Untreated infected Huh-7 cells were used as control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Dose–response curves of compounds determined in a viral titer reduction assay: concentration-dependent DENV-2 virus titer reduction and EC<sub>50</sub> values for selected compounds. (A) Dose–response curve of compound <b>20</b> (<i>R</i><sup>2</sup> = 0.97). (B) Dose–response curve of compound <b>38</b> (<i>R</i><sup>2</sup> = 0.97). (C) Dose–response curve of compound <b>43</b> (<i>R</i><sup>2</sup> = 0.93). (D) Dose–response curve of reference compound <b>MB-53</b> (<i>R</i><sup>2</sup> = 0.77).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54016" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54016" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We here describe the design, synthesis, and testing of small-molecule DENV protease inhibitors with drug-like properties. Several derivatives have nanomolar activity in a cellular DENV-2 protease reporter assay (DENV2proHeLa) and stable stereochemistry under physiological and assay conditions. The high activity of selected compounds was further confirmed in viral titer reduction assays. Compound <b>20</b>, with proven passive membrane permeability, had submicromolar activity in both cellular assays. Most of the analogs presented here are nontoxic up to 50 μM and have low off-target activity. Future efforts in medicinal chemistry will aim at the optimization of cellular activity of the compound class presented here, such as the development of a prodrug strategy for the guanidinophenyl residue.</div><div class="NLM_p">In contrast to, for example, boronic acids or some other classes of flaviviral protease inhibitors, the synthetic access to the analogs presented here is relatively facile. This favors the exploration of additional derivatives and makes them readily available. Besides their use as lead compounds for further drug discovery, these compounds can serve as reference inhibitors in both biochemical and cellular DENV-2 protease assay systems. Viral replication assays are usually performed under variable conditions (e.g., MOI, cell type, incubation time, medium composition) by different laboratories,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> impeding the comparison of results, antiviral activity, and assay conditions (e.g., pH, presence and type of detergent or other additives). Considering this background, reference compounds with good synthetic accessibility provide an opportunity to calibrate results from various laboratories and assay setups against a common standard.</div><div class="NLM_p last">In vitro target affinity is frequently difficult to translate into systems of higher complexity. In a cellular environment, pharmacokinetic effects determine the concentration at the site of target binding and the target is under the influence of biophysical factors that are difficult or impossible to reproduce under biochemical assay conditions, like molecular crowding, redox state, membrane binding, or formation of higher-order assemblies and structures. Therefore, it is attractive to employ cellular reporter gene assays, such as the DENV2proHeLa system, already at the early stages of compound characterization. The DENV2proHeLa incorporates pharmacokinetic effects and allows the evaluation of target engagement under physiological conditions, which includes the association of viral target(s) with organelles and (macro-) molecules of the host cell. In comparison to biochemical assays, the DENV2proHeLa results have an increased correlation to authentic virus titer reduction, which is considered the in vitro gold standard in antiviral drug discovery. The present study profited considerably from the ODD-↓-luciferase reporter gene protease assay, which was capable of identifying the nanomolar activity of the most promising lead structures in cells. The DENV2proHeLa assay also detected compounds with cellular activity that showed insignificant inhibition against the isolated DENV-2 protease: This indicates that factors beyond pharmacokinetics (such as folding of the target protein, interaction with other cellular components, etc.) influence the expression of cellular activity. The DENV2proHeLa assay can be performed in a high-throughput format and is significantly less time-consuming than virus titer reduction assays. We therefore consider it to be a useful bridge between the in vitro biochemical and viral replication assays. The presented paradigm can also be applied to the establishment of analogous assays for other intracellular proteases, such as the Corona virus protease, which is currently pursued in our group.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Reagents and Solvents</h3><div class="NLM_p last">All chemicals for the synthesis of precursors were obtained from Sigma-Aldrich (Germany), Alfa Aesar (Germany), Thermo Fisher Scientific (Germany), Acros Organics (Belgium), Thermo Fisher Scientific (United States), and TCI Europe (Belgium) and were of analytical grade. The amino acids were purchased from Carbolution Chemicals (Germany), Alfa Aesar (Germany), and TCI Europe (Belgium) or synthesized according to procedures described. COMU, HATU, and 2-chlorotrityl chloride resin (200–400 mesh); loading capacity: 1.6 mmol Cl<sup>–</sup>/g resin; cross-linked with 1% DVB was purchased from Carbolution Chemicals (Germany). All chemicals and solvents were purchased from commercial suppliers and were used without further purification.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Equipment and Analytical Methods</h3><div class="NLM_p">The progress of the reactions was determined by thin-layer chromatography (TLC) on Merck silica gel plates 60F<sub>254</sub> (UV detection). Flash chromatography was performed on a Biotage Isolera One purification system using silica gel (0.060–0.200 mm) cartridges (KP-Sil) with UV detection at 254 and 280 nm. NMR spectra were recorded on a Varian NMR instrument at 300 or 500 MHz, 300 K; chemical shifts (δ) are given in parts per million (ppm). Residual peaks of nondeuterated solvents were used as the internal standard: CDCl<sub>3</sub> (δ ppm: 7.26), D<sub>2</sub>O (δ ppm: 4.79), CD<sub>3</sub>OD (δ ppm: 3.31), DMSO-<i>d</i><sub>6</sub> (δ ppm: 2.50), CD<sub>3</sub>CN (δ ppm: 1.94), and acetone-<i>d</i><sub>6</sub> (δ ppm: 2.05). Coupling constants (<i>J</i>) are given in hertz (Hz). Multiplicity is reported as s (singlet), d (doublet), t (triplet), q (quartet), p (pentet), sext (sextet), dd (doublet of doublet), m (multiplet), and br (broad). Mass spectra (HR-ESI) of all compounds were measured on a Bruker micrOTOF-Q II instrument. Analysis of the compounds and intermediates was carried out using solutions in water, methanol, acetonitrile, or water/methanol (1:1). Melting points of selected compounds were determined with a Büchi M-560 melting point apparatus and are uncorrected.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure for Peptide Coupling in Solution</h4><div class="NLM_p last">The respective carboxylic acid (1.0 equiv) and HATU (1.2 equiv) were suspended in DCM (2–5 mL). The mixture was cooled to 0 °C, and the respective amine (1.2 equiv) was added prior to the dropwise addition of TMP (1.3–3.2 equiv). DMF was added until all solids were dissolved, and the mixture was stirred at room temperature for 2 h. The reaction was quenched with the addition of a solution of TFA in DCM (5%, 5 mL). All solvents were removed in vacuo, and the resulting residue was dissolved in ethyl acetate. The ethyl acetate phase was washed twice with 1 N HCl, 0.05 N NaOH, and water. The combined organic phases were dried over anhydrous MgSO<sub>4</sub>, concentrated under reduced pressure, and purified by preparative HPLC. After purification, all organic solvents were evaporated and the compounds were freeze-dried in water.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>S</i>)-<i>N</i>-(2-(Benzylamino)-1-(4-(benzyloxy)phenyl)-2-oxoethyl)hexanamide (<b>6</b>)</h4><div class="NLM_p last"><b>6</b> was synthesized according to the general procedure for peptide coupling in solution using <b>57</b> (21 mg, 0.06 mmol), HATU (27 mg, 0.07 mmol), benzylamine (0.008 mL, 0.07 mmol), and TMP (0.010 mL, 0.08 mmol). Obtained as a white powder (14 mg, 54% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 7.83 (t, <i>J</i> = 5.9 Hz, 1H), 7.53 (d, <i>J</i> = 7.6 Hz, 1H), 7.47 (d, <i>J</i> = 6.7 Hz, 2H), 7.43–7.29 (m, 5H), 7.29–7.15 (m, 5H), 6.96 (d, <i>J</i> = 8.7 Hz, 2H), 5.51 (d, <i>J</i> = 7.7 Hz, 1H), 5.12 (s, 2H), 4.39 (d, <i>J</i> = 6.2 Hz, 2H), 2.25 (t, <i>J</i> = 7.5 Hz, 2H), 1.66–1.52 (m, 2H), 1.36–1.21 (m, 4H), 0.86 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>33</sub>N<sub>2</sub>O<sub>3</sub>: 445.2486, found: 445.2487.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-oxo-2-(pyridin-4-ylmethylamino)ethyl)hexanamide (<b>7</b>)</h4><div class="NLM_p last"><b>7</b> was synthesized according to the general procedure for peptide coupling in solution using <b>57</b> (28 mg, 0.08 mmol), HATU (36 mg, 0.10 mmol), aminomethylpyridine (0.010 mL, 0.10 mmol), and TMP (0.013 mL, 0.10 mmol). Obtained as a white powder (26 mg, 74% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.45–8.34 (m, 1H), 8.08–7.85 (m, 2H), 7.81–7.62 (m, 2H), 7.48 (d, <i>J</i> = 7.1 Hz, 2H), 7.45–7.28 (m, 5H), 7.01 (d, <i>J</i> = 8.4 Hz, 2H), 5.53 (d, <i>J</i> = 6.4 Hz, 1H), 5.14 (s, 2H), 4.65 (s, 2H), 2.28 (t, <i>J</i> = 7.5 Hz, 2H), 1.67–1.51 (m, 2H), 1.35–1.17 (m, 4H), 0.84 (t, <i>J</i> = 6.5 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>32</sub>N<sub>3</sub>O<sub>3</sub>: 446.2438, found: 446.2432.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-cyanobenzylamino)-2-oxoethyl)hexanamide (<b>9</b>)</h4><div class="NLM_p last"><b>9</b> was synthesized according to the general procedure for peptide coupling in solution using <b>57</b> (72 mg, 0.20 mmol), HATU (92 mg, 0.24 mmol), <b>56</b> (32 mg, 0.24 mmol), and TMP (0.035 mL, 0.26 mmol). Obtained as a white powder (49 mg, 52% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.10–8.00 (m, 1H), 7.64 (d, <i>J</i> = 8.2 Hz, 2H), 7.48 (d, <i>J</i> = 7.6 Hz, 2H), 7.44–7.32 (m, 8H), 6.98 (d, <i>J</i> = 8.7 Hz, 2H), 5.51–5.48 (m, 1H), 5.13 (s, 2H), 4.48 (s, 2H), 2.26 (t, <i>J</i> = 7.3 Hz, 2H), 1.75–1.64 (m, 2H), 1.35–1.21 (m, 4H), 0.90 (t, <i>J</i> = 6.5 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>32</sub>N<sub>3</sub>O<sub>3</sub>: 470.2438, found: 470.2439.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>S</i>)-<i>N</i>-(2-(1<i>H</i>-Indazol-5-ylamino)-1-(4-(benzyloxy)phenyl)-2-oxoethyl)hexanamide (<b>8</b>)</h4><div class="NLM_p last"><b>8</b> was synthesized according to the general procedure for peptide coupling in solution using <b>57</b> (38 mg, 0.11 mmol), HATU (49 mg, 0.13 mmol), 1<i>H</i>-indazol-5-amine (17 mg, 0.13 mmol), and TMP (0.018 mL, 0.14 mmol). Obtained as a brown powder (22 mg, 35% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.50 (s, 1H), 8.19 (s, 1H), 7.98 (s, 1H), 7.64 (d, <i>J</i> = 8.3 Hz, 1H), 7.55–7.27 (m, 10H), 6.99 (d, <i>J</i> = 8.6 Hz, 2H), 5.66 (d, <i>J</i> = 7.4 Hz, 1H), 5.11 (s, 2H), 2.30 (t, <i>J</i> = 7.5 Hz, 2H), 1.69–1.53 (m, 2H), 1.39–1.21 (m, 4H), 0.86 (t, <i>J</i> = 6.4 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>31</sub>N<sub>4</sub>O<sub>3</sub>: 471.2391, found: 471.2396.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> General Procedure for Guanidinylation of Amines</h4><div class="NLM_p last">The respective amine (1.0 equiv), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (1.5 equiv), DMAP (0.3 equiv), and DIPEA (1.0 equiv) were dissolved or suspended in MeOH (5–20 mL). The mixture was allowed to stir for 2 or 3 days at room temperature. The solvent was removed in vacuo, and the resulting residue was dissolved in ethyl acetate or DCM. The organic phase was washed with 0.1 N HCl, 1 N NaOH, and water. The combined organic phase was dried over anhydrous MgSO<sub>4</sub> and concentrated under reduced pressure. The resulting solid or oil was dissolved in TFA/DCM (1:1, 5–10 mL) and was stirred for 2–3 h. The mixture was concentrated and further co-evaporated with several portions of toluene. The residue was purified by preparative HPLC and freeze-dried in water.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(6-guanidinohexylamino)-2-oxoethyl)hexanamide (<b>5</b>)</h4><div class="NLM_p last"><b>5</b> was synthesized according to the general procedure for guanidinylation from <b>4</b> (10 mg, 0.02 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (9 mg, 0.03 mmol), DMAP (1 mg, 0.01 mmol), and DIPEA (0.003 mL, 0.02 mmol). Obtained as a white powder (TFA salt, 6 mg, 60% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.26 (br. s, 1H), 7.71–7.51 (m, 6H), 7.47 (d, <i>J</i> = 7.1 Hz, 2H), 7.43–7.27 (m, 5H), 6.97 (d, <i>J</i> = 8.7 Hz, 2H), 5.40 (d, <i>J</i> = 7.4 Hz, 1H), 5.11 (s, 2H), 3.29–3.08 (m, 4H), 2.27 (t, <i>J</i> = 7.5 Hz, 2H), 1.68–1.53 (m, 4H), 1.54–1.42 (m, 2H), 1.41–1.20 (m, 8H), 0.86 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>42</sub>N<sub>5</sub>O<sub>3</sub>: 496.3282, found: 496.3281.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinophenylamino)-2-oxoethyl)hexanamide (<b>19</b>)</h4><div class="NLM_p last"><b>19</b> was synthesized according to the general procedure for guanidinylation from <b>10</b> (19 mg, 0.04 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (20 mg, 0.06 mmol), DMAP (2 mg, 0.01 mmol), and DIPEA (0.007 mL, 0.04 mmol). Obtained as a white powder (TFA salt, 15 mg, 58% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.84 (s, 1H), 9.80 (s, 1H), 7.84 (s, 4H), 7.78 (d, <i>J</i> = 7.1 Hz, 1H), 7.68 (d, <i>J</i> = 8.9 Hz, 2H), 7.49–7.26 (m, 7H), 7.16 (d, <i>J</i> = 8.7 Hz, 2H), 7.00 (d, <i>J</i> = 8.6 Hz, 2H), 5.63 (d, <i>J</i> = 7.2 Hz, 1H), 5.12 (s, 2H), 2.31 (t, <i>J</i> = 7.4 Hz, 2H), 1.67–1.54 (m, 2H), 1.37–1.23 (m, 4H), 0.86 (t, <i>J</i> = 6.7 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>34</sub>N<sub>5</sub>O<sub>3</sub>: 488.2656, found: 488.2647.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)hexanamide (<b>20</b>)</h4><div class="NLM_p last"><b>20</b> was synthesized according to the general procedure for guanidinylation from <b>11</b> (129 mg, 0.28 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (131 mg, 0.42 mmol), DMAP (10 mg, 0.08 mmol), and DIPEA (0.048 mL, 0.28 mmol). Obtained as a light yellow powder (TFA salt, 89 mg, 51% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 8.69 (t, <i>J</i> = 6.0 Hz, 1H), 8.38 (d, <i>J</i> = 6.6 Hz, 1H), 7.44 (d, <i>J</i> = 7.4 Hz, 2H), 7.41–7.24 (m, 7H), 7.18 (d, <i>J</i> = 8.3 Hz, 2H), 7.01 (d, <i>J</i> = 8.6 Hz, 2H), 5.35 (s, 1H), 5.12 (s, 2H), 4.41 (s, 2H), 2.28 (t, <i>J</i> = 7.5 Hz, 2H), 1.62 (p, <i>J</i> = 7.1 Hz, 2H), 1.37–1.27 (m, 4H), 0.91 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (APT, 126 MHz, CD<sub>3</sub>OD) δ 176.0, 173.3, 160.2, 158.2, 139.8, 138.6, 134.8, 130.8, 130.1, 129.7, 129.5, 128.9, 128.5, 126.7, 116.2, 71.0, 58.7, 43.5, 36.6, 32.5, 26.6, 23.5, 14.3. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>36</sub>N<sub>5</sub>O<sub>3</sub>: 502.2813, found: 502.2802.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (<i>S</i>)-<i>N</i>-(1-(4-(benzyloxy)phenyl)-2-(4-(guanidinomethyl)benzylamino)-2-oxoethyl)hexanamide (<b>22</b>)</h4><div class="NLM_p last"><b>22</b> was synthesized according to the general procedure for guanidinylation from <b>14</b> (10 mg, 0.02 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (10 mg, 0.03 mmol), DMAP (1 mg, 0.01 mmol), and DIPEA (0.004 mL, 0.02 mmol). Obtained as a white powder (TFA salt, 8 mg, 62% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.84 (s, 1H), 7.93 (t, <i>J</i> = 5.8 Hz, 1H), 7.86–7.66 (br. s, 4H), 7.58 (d, <i>J</i> = 7.6 Hz, 1H), 7.47 (d, <i>J</i> = 7.0 Hz, 2H), 7.43–7.31 (m, 5H), 7.30–7.18 (m, 4H), 6.97 (d, <i>J</i> = 8.7 Hz, 2H), 5.54–5.46 (m, 1H), 5.12 (s, 2H), 4.51–4.42 (m , 2H), 4.41–4.30 (m, 2H), 2.25 (t, <i>J</i> = 7.5 Hz, 2H), 1.65–1.51 (m, 2H), 1.36–1.20 (m, 4H), 0.85 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>38</sub>N<sub>5</sub>O<sub>3</sub>: 516.2969, found: 516.2965.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(3-(guanidinomethyl)benzylamino)-2-oxoethyl)hexanamide (<b>23</b>)</h4><div class="NLM_p last"><b>23</b> was synthesized according to the general procedure for guanidinylation from <b>15</b> (15 mg, 0.03 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (15 mg, 0.05 mmol), DMAP (1 mg, 0.01 mmol), and DIPEA (0.005 mL, 0.03 mmol). Obtained as a white powder (TFA salt, 13 mg, 65% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.62 (s, 1H), 8.01 (t, <i>J</i> = 6.1 Hz, 1H), 7.82 (br. s, 4H), 7.71 (d, <i>J</i> = 6.7 Hz, 1H), 7.47 (d, <i>J</i> = 7.0 Hz, 2H), 7.43–7.18 (m, 8H), 7.14 (d, <i>J</i> = 7.2 Hz, 1H), 6.99 (d, <i>J</i> = 8.7 Hz, 2H), 5.47 (d, <i>J</i> = 7.1 Hz, 1H), 5.13 (s, 2H), 4.55–4.25 (m, 4H), 2.29 (t, <i>J</i> = 7.5 Hz, 2H), 1.66–1.52 (m, 2H), 1.38–1.20 (m, 4H), 0.85 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>38</sub>N<sub>5</sub>O<sub>3</sub>: 516.2969, found: 516.2970.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>S</i>)-<i>N</i>-(2-(4-Guanidinobenzylamino)-2-oxo-1-phenylethyl)hexanamide (<b>26</b>)</h4><div class="NLM_p last"><b>26</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (64 mg, 0.14 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (63 mg, 0.20 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.023 mL, 0.14 mmol). Obtained as a white powder (TFA salt, 44 mg, 64% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.90 (br. s, 1H), 8.09 (t, <i>J</i> = 5.7 Hz, 1H), 7.81 (br. s, 4H), 7.71 (d, <i>J</i> = 7.2 Hz, 1H), 7.47 (d, <i>J</i> = 6.3 Hz, 2H), 7.38–7.24 (m, 5H), 7.20 (d, <i>J</i> = 8.2 Hz, 2H), 5.60 (d, <i>J</i> = 7.3 Hz, 1H), 4.50–4.32 (m, 2H), 2.28 (t, <i>J</i> = 7.5 Hz, 2H), 1.59 (p, <i>J</i> = 7.3 Hz, 2H), 1.36–1.20 (m, 4H), 0.85 (t, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.0, 171.2, 161.3, 139.8, 139.1, 134.8, 129.6, 129.3, 128.7, 128.4, 126.0, 58.0, 42.9, 36.3, 32.2, 26.1, 23.1, 14.3. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub>: 396.2394, found: 396.2385.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>R</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)hexanamide (<b>27</b>)</h4><div class="NLM_p last"><b>27</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (81 mg, 0.14 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (66 mg, 0.21 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.024 mL, 0.14 mmol). Obtained as a white powder (TFA salt, 61 mg, 70% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.00 (br. s, 1H), 8.01 (t, <i>J</i> = 6.1 Hz, 1H), 7.85 (br. s, 4H), 7.63 (d, <i>J</i> = 7.2 Hz, 1H), 7.47 (d, <i>J</i> = 7.1 Hz, 2H), 7.43–7.29 (m, 5H), 7.26 (d, <i>J</i> = 8.4 Hz, 2H), 7.19 (d, <i>J</i> = 8.3 Hz, 2H), 6.98 (d, <i>J</i> = 8.6 Hz, 2H), 5.52 (d, <i>J</i> = 7.2 Hz, 1H), 5.12 (s, 2H), 4.48–4.32 (m, 2H), 2.26 (t, <i>J</i> = 7.5 Hz, 2H), 1.67–1.50 (p, <i>J</i> = 7.4 Hz, 2H), 1.36–1.20 (m, 4H), 0.85 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.9, 171.5, 159.4, 158.7, 139.1, 138.4, 134.9, 132.1, 130.5, 129.7, 129.3, 128.6, 128.4, 125.9, 115.6, 70.4, 57.4, 42.8, 36.3, 32.2, 26.1, 23.1, 14.3. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>36</sub>N<sub>5</sub>O<sub>3</sub>: 502.2813, found: 502.2842.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>S</i>)-<i>N</i>-(2-(4-Guanidinobenzylamino)-2-oxo-1-(4-(4-(trifluoromethyl)benzyloxy)phenyl)ethyl)hexanamide (<b>28</b>)</h4><div class="NLM_p last"><b>28</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (150 mg, 0.23 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (110 mg, 0.35 mmol), DMAP (9 mg, 0.07 mmol), and DIPEA (0.040 mL, 0.23 mmol). Obtained as a light yellow powder (TFA salt, 53 mg, 67% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.77 (s, 1H), 8.13 (t, <i>J</i> = 5.9 Hz, 1H), 7.76 (br. s, 4H), 7.75–7.66 (m, 5H), 7.41 (d, <i>J</i> = 8.7 Hz, 2H), 7.28 (d, <i>J</i> = 8.4 Hz, 2H), 7.20 (d, <i>J</i> = 8.4 Hz, 2H), 7.00 (d, <i>J</i> = 8.7 Hz, 2H), 5.60–5.50 (m, 1H), 5.24 (s, 2H), 4.49–4.30 (m, 2H), 2.27 (t, <i>J</i> = 7.5 Hz, 2H), 1.57 (p, <i>J</i> = 7.3 Hz, 2H), 1.35–1.16 (m, 4H), 0.85 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>35</sub>F<sub>3</sub>N<sub>5</sub>O<sub>3</sub>: 570.2687, found: 570.2708.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>R</i>)-<i>N</i>-(2-(4-Guanidinobenzylamino)-2-oxo-1-(4-(4-(trifluoromethyl)benzyloxy)phenyl)ethyl)hexanamide (<b>29</b>)</h4><div class="NLM_p last"><b>29</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (64 mg, 0.10 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (47 mg, 0.15 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.017 mL, 0.10 mmol). Obtained as a pale yellow powder (TFA salt, 47 mg, 69% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.90 (s, 1H), 8.12 (t, <i>J</i> = 5.9 Hz, 1H), 7.83 (br. s, 4H), 7.78–7.66 (m, 5H), 7.41 (d, <i>J</i> = 8.7 Hz, 2H), 7.27 (d, <i>J</i> = 8.3 Hz, 2H), 7.20 (d, <i>J</i> = 8.3 Hz, 2H), 7.00 (d, <i>J</i> = 8.7 Hz, 2H), 5.61–5.52 (m, 1H), 5.24 (s, 2H), 4.49–4.30 (m, 2H), 2.27 (t, <i>J</i> = 7.5 Hz, 2H), 1.65–1.49 (m, 2H), 1.34–1.19 (m, 4H), 0.85 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>35</sub>F<sub>3</sub>N<sub>5</sub>O<sub>3</sub>: 570.2687, found: 570.2681.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>S</i>)-<i>N</i>-(2-(4-Guanidinobenzylamino)-1-(4-(3-methoxybenzyloxy)phenyl)-2-oxoethyl)hexanamide (<b>30</b>)</h4><div class="NLM_p last"><b>30</b> was synthesized according to the general procedure for from the respective aniline (73.5 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (57 mg, 0.18 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.021 mL, 0.12 mmol). Obtained as a light yellow powder (TFA salt, 29 mg, 37% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.04 (s, 1H), 7.97 (t, <i>J</i> = 6.2 Hz, 1H), 7.84 (br. s, 4H), 7.59 (d, <i>J</i> = 6.7 Hz, 1H), 7.37 (d, <i>J</i> = 8.7 Hz, 2H), 7.34–7.24 (m, 3H), 7.21 (d, <i>J</i> = 8.6 Hz, 2H), 7.06–6.94 (m, 4H), 6.92–6.85 (m, 1H), 5.53–5.47 (m, 1H), 5.10 (s, 2H), 4.49–4.33 (m, 2H), 3.80 (s, 3H), 2.26 (t, <i>J</i> = 7.5 Hz, 2H), 1.58 (p, <i>J</i> = 7.3 Hz, 2H), 1.36–1.21 (m, 4H), 0.85 (t, <i>J</i> = 6.9 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>38</sub>N<sub>5</sub>O<sub>4</sub>: 532.2918, found: 532.2927.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>S</i>)-<i>N</i>-(1-(4-(4-Cyanobenzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)hexanamide (<b>31</b>)</h4><div class="NLM_p last"><b>31</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (70 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (54 mg, 0.17 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.020 mL, 0.12 mmol). Obtained as a light yellow powder (TFA salt, 43 mg, 58% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.74 (s, 1H), 8.04 (t, <i>J</i> = 5.9 Hz, 1H), 7.81 (d, <i>J</i> = 8.3 Hz, 2H), 7.75 (br. s, 4H), 7.71–7.62 (m, 3H), 7.40 (d, <i>J</i> = 8.7 Hz, 2H), 7.32–7.16 (m, 4H), 7.00 (d, <i>J</i> = 8.7 Hz, 2H), 5.56–5.49 (m, 1H), 5.25 (s, 2H), 4.50–4.31 (m, 2H), 2.27 (t, <i>J</i> = 7.5 Hz, 2H), 1.58 (p, <i>J</i> = 7.4 Hz, 2H), 1.37–1.20 (m, 4H), 0.85 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.1, 171.4, 159.0, 157.7, 144.0, 139.3, 134.7, 133.2, 132.5, 129.8, 129.4, 128.8, 126.0, 119.2, 115.6, 112.3, 69.5, 57.4, 42.8, 36.3, 32.2, 26.1, 23.1, 14.3. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>35</sub>N<sub>6</sub>O<sub>3</sub>: 527.2765, found: 527.2774.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>S</i>)-<i>N</i>-(1-(4-(2,6-Dichlorobenzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)hexanamide (<b>32</b>)</h4><div class="NLM_p last"><b>32</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (68 mg, 0.11 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (49 mg, 0.16 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.018 mL, 0.11 mmol). Obtained as a light yellow powder (TFA salt, 39 mg, 54% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.97 (s, 1H), 8.03 (t, <i>J</i> = 5.9 Hz, 1H), 7.85 (br. s, 4H), 7.65 (d, <i>J</i> = 7.2 Hz, 1H), 7.56–7.40 (m, 5H), 7.29 (d, <i>J</i> = 8.3 Hz, 2H), 7.22 (d, <i>J</i> = 8.2 Hz, 2H), 7.05 (d, <i>J</i> = 8.7 Hz, 2H), 5.59–5.51 (m, 1H), 5.32 (s, 2H), 4.50–4.34 (m, 2H), 2.28 (t, <i>J</i> = 7.5 Hz, 2H), 1.59 (p, <i>J</i> = 7.2 Hz, 2H), 1.38–1.21 (m, 4H), 0.86 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.9, 171.5, 159.6, 157.8, 139.2, 137.5, 134.9, 133.2, 132.6, 132.1, 129.8, 129.6, 129.4, 126.0, 115.5, 65.9, 57.4, 42.8, 36.3, 32.2, 26.1, 23.1, 14.3. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>3</sub>: 570.2033, found: 570.2033.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-<i>N</i>-(1-(4-(3,4-Dichlorobenzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)hexanamide (<b>33</b>)</h4><div class="NLM_p last"><b>33</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (77 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (56 mg, 0.18 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.021 mL, 0.12 mmol). Obtained as a yellow powder (TFA salt, 41 mg, 50% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.96 (s, 1H), 8.03 (t, <i>J</i> = 6.0 Hz, 1H), 7.85 (br. s, 4H), 7.76–7.66 (m, 2H), 7.60 (d, <i>J</i> = 8.3 Hz, 1H), 7.46 (d, <i>J</i> = 8.2 Hz, 1H), 7.40 (d, <i>J</i> = 8.6 Hz, 2H), 7.28 (d, <i>J</i> = 8.4 Hz, 2H), 7.21 (d, <i>J</i> = 8.4 Hz, 2H), 6.99 (d, <i>J</i> = 8.7 Hz, 2H), 5.56–5.49 (m, 1H), 5.16 (s, 2H), 4.49–4.31 (m, 2H), 2.27 (t, <i>J</i> = 7.5 Hz, 2H), 1.65–1.51 (m, 2H), 1.35–1.20 (m, 4H), 0.84 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>3</sub>: 570.2033, found: 570.2059.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>S</i>)-<i>N</i>-(1-(4-(4-<i>tert</i>-Butylbenzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)hexanamide (<b>34</b>)</h4><div class="NLM_p last"><b>34</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (69 mg, 0.11 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (51 mg, 0.17 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.018 mL, 0.11 mmol). Obtained as a light brown powder (TFA salt, 9 mg, 12% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.00 (s, 1H), 8.03–7.96 (m, 1H), 7.92 (t, <i>J</i> = 5.5 Hz, 1H), 7.84 (br. s, 4H), 7.56–7.33 (m, 2H), 7.31–7.02 (m, 8H), 6.80 (d, <i>J</i> = 8.3 Hz, 2H), 5.44 (d, <i>J</i> = 7.5 Hz, 1H), 5.32 (s, 2H), 4.44–4.36 (m, 2H), 2.24 (t, <i>J</i> = 7.5 Hz, 2H), 1.64–1.50 (m, 2H), 1.39–1.29 (m, 4H), 1.27 (s, 9H), 0.86 (t, <i>J</i> = 6.9 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>44</sub>N<sub>5</sub>O<sub>3</sub>: 558.3439, found: 558.3443.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>S</i>)-2-Acetamido-2-(4-(benzyloxy)phenyl)-<i>N</i>-(4-guanidinobenzyl)acetamide (<b>35</b>)</h4><div class="NLM_p last"><b>35</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (61 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (55 mg, 0.18 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.020 mL, 0.12 mmol). Obtained as a white powder (TFA salt, 34 mg, 52% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.77 (s, 1H), 8.09 (t, <i>J</i> = 5.5 Hz, 1H), 7.89–7.72 (m, 5H), 7.46 (d, <i>J</i> = 7.2 Hz, 2H), 7.43–7.30 (m, 5H), 7.27 (d, <i>J</i> = 8.1 Hz, 2H), 7.19 (d, <i>J</i> = 7.9 Hz, 2H), 6.98 (d, <i>J</i> = 7.9 Hz, 2H), 5.53–5.46 (m, 1H), 5.12 (s, 2H), 4.49–4.29 (m, 2H), 1.96 (s, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.5, 170.2, 159.5, 157.7, 139.2, 138.4, 134.7, 131.8, 129.8, 129.3, 129.3, 128.7, 128.4, 126.1, 115.7, 70.4, 57.6, 42.8, 22.7. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>28</sub>N<sub>5</sub>O<sub>3</sub>: 446.2187, found: 446.2182.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)pentanamide (<b>36</b>)</h4><div class="NLM_p last"><b>36</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (66 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (55 mg, 0.18 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.020 mL, 0.12 mmol). Obtained as a white powder (TFA salt, 36 mg, 51% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.87 (s, 1H), 8.01 (t, <i>J</i> = 5.6 Hz, 1H), 7.80 (br. s, 4H), 7.64 (d, <i>J</i> = 7.0 Hz, 1H), 7.47 (d, <i>J</i> = 7.3 Hz, 2H), 7.43–7.31 (m, 5H), 7.26 (d, <i>J</i> = 8.0 Hz, 2H), 7.19 (d, <i>J</i> = 7.9 Hz, 2H), 6.98 (d, <i>J</i> = 7.8 Hz, 2H), 5.52 (d, <i>J</i> = 5.2 Hz, 1H), 5.12 (s, 2H), 4.48–4.31 (m, 2H), 2.27 (t, <i>J</i> = 7.4 Hz, 2H), 1.55 (p, <i>J</i> = 7.5 Hz, 2H), 1.30 (sext, <i>J</i> = 7.6 Hz, 2H); 0.86 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.9, 171.5, 159.4, 157.8, 139.2, 138.4, 134.8, 132.0, 129.7, 129.3, 129.3, 128.6, 128.4, 126.0, 115.6, 70.4, 57.4, 42.8, 36.1, 28.5, 23.0, 14.1. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>34</sub>N<sub>5</sub>O<sub>3</sub>: 488.2656, found: 488.2647.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)heptanamide (<b>37</b>)</h4><div class="NLM_p last"><b>37</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (68 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (54 mg, 0.17 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.020 mL, 0.12 mmol). Obtained as a pale yellow powder (TFA salt, 46 mg, 63% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.11 (s, 1H), 7.98 (t, <i>J</i> = 6.2 Hz, 1H), 7.90 (br. s, 4H), 7.59 (d, <i>J</i> = 7.0 Hz, 1H), 7.47 (d, <i>J</i> = 7.2 Hz, 2H), 7.43–7.30 (m, 5H), 7.26 (d, <i>J</i> = 8.4 Hz, 2H), 7.20 (d, <i>J</i> = 8.5 Hz, 2H), 6.98 (d, <i>J</i> = 8.7 Hz, 2H), 5.54–5.48 (m, 1H), 5.13 (s, 2H), 4.48–4.32 (m, 2H), 2.26 (t, <i>J</i> = 7.5 Hz, 2H), 1.58 (p, <i>J</i> = 7.4 Hz, 2H), 1.37–1.20 (m, 6H), 0.86 (t, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 172.7, 171.4, 159.4, 157.6, 139.2, 138.4, 134.9, 132.2, 129.7, 129.7, 129.3, 128.6, 128.4, 125.9, 115.6, 70.4, 57.3, 42.8, 36.4, 32.3, 26.4, 23.2, 14.3. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>38</sub>N<sub>5</sub>O<sub>3</sub>: 516.2969, found: 516.2969.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)octanamide (<b>38</b>)</h4><div class="NLM_p last"><b>38</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (70 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (54 mg, 0.18 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.020 mL, 0.12 mmol). Obtained as a white powder (TFA salt, 44 mg, 59% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.89 (s, 1H), 8.05 (t, <i>J</i> = 5.6 Hz, 1H), 7.81 (br. s, 4H), 7.67 (d, <i>J</i> = 6.6 Hz, 1H), 7.47 (d, <i>J</i> = 7.5 Hz, 2H), 7.43–7.29 (m, 5H), 7.26 (d, <i>J</i> = 7.5 Hz, 2H), 7.19 (d, <i>J</i> = 8.0 Hz, 2H), 6.98 (d, <i>J</i> = 7.5 Hz, 2H), 5.53 (d, <i>J</i> = 5.7 Hz, 1H), 5.12 (s, 2H), 4.48–4.31 (m, 2H), 2.27 (t, <i>J</i> = 7.1 Hz, 2H), 1.65–1.50 (m, 2H), 1.36–1.19 (m, 8H), 0.86 (t, <i>J</i> = 5.9 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.0, 171.5, 159.4, 157.8, 139.2, 138.4, 134.8, 132.0, 129.7, 129.3, 128.6, 128.4, 126.0, 115.6, 70.4, 57.4, 42.8, 36.4, 32.5, 30.0, 29.8, 26.4, 23.3, 14.4. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>40</sub>N<sub>5</sub>O<sub>3</sub>: 530.3126, found: 530.3128.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)decanamide (<b>39</b>)</h4><div class="NLM_p last"><b>39</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (95 mg, 0.15 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (70 mg, 0.23 mmol), DMAP (6 mg, 0.05 mmol), and DIPEA (0.026 mL, 0.15 mmol). Obtained as a white powder (TFA salt, 59 mg, 58% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.96 (s, 1H), 8.06 (t, <i>J</i> = 5.6 Hz, 1H), 7.86 (s, 4H), 7.68 (d, <i>J</i> = 7.1 Hz, 1H), 7.47 (d, <i>J</i> = 7.2 Hz, 2H), 7.43–7.29 (m, 5H), 7.26 (d, <i>J</i> = 8.2 Hz, 2H), 7.19 (d, <i>J</i> = 8.3 Hz, 2H), 6.98 (d, <i>J</i> = 8.6 Hz, 2H), 5.57–5.51 (m, 1H), 5.12 (s, 2H), 4.48–4.30 (m, 2H), 2.27 (t, <i>J</i> = 7.4 Hz, 2H), 1.58 (p, <i>J</i> = 6.9 Hz, 2H), 1.37–1.17 (m, 12H), 0.87 (t, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 173.1, 171.5, 159.4, 157.8, 139.1, 138.3, 134.8, 132.0, 129.7, 129.3, 128.6, 128.4, 125.9, 115.7, 70.3, 57.2, 42.8, 36.4, 32.6, 31.0, 30.3, 30.0, 30.0, 26.4, 23.3, 14.4. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>44</sub>N<sub>5</sub>O<sub>3</sub>: 558.3439, found: 558.3449.</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (<i>S</i>)-2-(4-(Benzyloxy)phenyl)-2-(2-cyclohexylacetamido)-<i>N</i>-(4-guanidinobenzyl)acetamide (<b>40</b>)</h4><div class="NLM_p last"><b>40</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (85 mg, 0.14 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (66 mg, 0.21 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.024 mL, 0.14 mmol). Obtained as a light yellow powder (TFA salt, 65 mg, 71% yield). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 7.49–7.36 (m, 7H), 7.31 (d, <i>J</i> = 8.6 Hz, 2H), 7.19 (d, <i>J</i> = 8.5 Hz, 2H), 7.03 (d, <i>J</i> = 8.8 Hz, 2H), 5.39 (s, 1H), 5.74 (s, 2H), 4.42 (s, 2H), 2.18 (d, <i>J</i> = 7.1 Hz, 2H), 1.85–1.61 (m, 6H), 1.38–1.11 (m, 3H), 1.09–0.90 (m, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>38</sub>N<sub>5</sub>O<sub>3</sub>: 528.2969, found: 528.2972.</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)benzamide (<b>41</b>)</h4><div class="NLM_p last"><b>41</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (84 mg, 0.14 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (69 mg, 0.22 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.025 mL, 0.14 mmol). Obtained as a pale yellow powder (TFA salt, 34 mg, 38% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.81 (s, 1H), 8.17–8.06 (m, 2H), 7.93 (d, <i>J</i> = 7.3 Hz, 2H), 7.79 (br. s, 4H), 7.59–7.26 (m, 12H), 7.20 (d, <i>J</i> = 8.2 Hz, 2H), 7.01 (d, <i>J</i> = 8.6 Hz, 2H), 5.79–5.66 (m, 1H), 5.13 (s, 2H), 4.54–4.35 (m, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>30</sub>N<sub>5</sub>O<sub>3</sub>: 508.2343, found: 508.2357.</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)-4-(cyclohexyloxy)benzamide (<b>42</b>)</h4><div class="NLM_p last"><b>42</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (89 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (58 mg, 0.19 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.021 mL, 0.12 mmol). Obtained as a pale red powder (TFA salt, 58 mg, 62% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.56 (s, 1H), 8.18 (t, <i>J</i> = 5.4 Hz, 1H), 8.01 (d, <i>J</i> = 6.7 Hz, 1H), 7.88 (d, <i>J</i> = 8.5 Hz, 2H), 7.70 (br. s, 4H), 7.53–7.25 (m, 9H), 7.18 (d, <i>J</i> = 8.0 Hz, 2H), 6.99 (d, <i>J</i> = 8.9 Hz, 2H), 6.96 (d, <i>J</i> = 8.7 Hz, 2H), 5.77–5.64 (m, 1H), 5.12 (s, 2H), 4.52–4.32 (m, 3H), 2.01–1.70 (m, 4H), 1.63–1.23 (m, 6H). <sup>13</sup>C NMR (75 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.7, 166.7, 161.5, 159.5, 157.7, 139.2, 138.3, 134.6, 131.8, 130.2, 130.0, 129.4, 129.3, 128.6, 128.4, 126.9, 126.1, 116.0, 115.7, 75.7, 70.4, 58.0, 32.3, 26.2, 24.2. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>40</sub>N<sub>5</sub>O<sub>4</sub>: 606.3075, found: 606.3082.</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)-4-phenoxybenzamide (<b>43</b>)</h4><div class="NLM_p last"><b>43</b> was synthesized according to general procedure for guanidinylation from the respective aniline (98 mg, 0.15 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (68 mg, 0.22 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.025 mL, 0.15 mmol). Obtained as a light orange powder (TFA salt, 66 mg, 64% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.46 (s, 1H), 8.19 (t, <i>J</i> = 6.0 Hz, 1H), 8.13 (s, <i>J</i> = 6.9 Hz, 1H), 7.95 (d, <i>J</i> = 8.7, 2H), 7.65 (s, 4H), 7.53–7.26 (m, 11H), 7.25–7.15 (m, 3H), 7.07 (s, <i>J</i> = 7.8 Hz, 2H), 7.00 (d, <i>J</i> = 8.6 Hz, 4H), 5.78–5.69 (m, 1H), 5.12 (s, 2H), 4.52–4.34 (m, 2H). <sup>13</sup>C NMR (126 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.5, 166.3, 161.3, 159.6, 157.7, 157.0, 139.1, 138.3, 134.7, 131.7, 131.0, 130.4, 130.0, 129.7, 129.4, 129.3, 128.6, 128.4, 126.0, 125.1, 120.5, 118.3, 115.7, 70.4, 58.0, 43.0, 29.8. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>34</sub>N<sub>5</sub>O<sub>4</sub>: 600.2605, found: 600.2628.</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)-3-phenoxybenzamide (<b>44</b>)</h4><div class="NLM_p last"><b>44</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (104 mg, 0.16 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (72 mg, 0.23 mmol), DMAP (6 mg, 0.05 mmol), and DIPEA (0.027 mL, 0.16 mmol). Obtained as a light orange powder (TFA salt, 77 mg, 69% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.79 (s, 1H), 8.18 (d, <i>J</i> = 6.9 Hz, 1H), 8.08 (t, <i>J</i> = 5.9 Hz, 1H), 7.77 (br. s, 4H), 7.70 (d, <i>J</i> = 7.7 Hz, 1H), 7.61–7.54 (m, 1H), 7.52–7.25 (m, 12H), 7.24–7.10 (m, 4H), 7.01 (t, <i>J</i> = 7.6 Hz, 4H), 5.74–5.66 (m, 1H), 5.13 (s, 2H), 4.52–4.34 (m, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>34</sub>N<sub>5</sub>O<sub>4</sub>: 600.2605, found: 600.2626.</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>S</i>)-4-(Benzyloxy)-<i>N</i>-(1-(4-(benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)benzamide (<b>45</b>)</h4><div class="NLM_p last"><b>45</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (102 mg, 0.15 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (69 mg, 0.22 mmol), DMAP (5 mg, 0.05 mmol), and DIPEA (0.025 mL, 0.15 mmol). Obtained as a pale yellow powder (TFA salt, 61 mg, 57% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.89 (s, 1H), 8.11 (t, <i>J</i> = 6.0 Hz, 1H), 7.99 (d, <i>J</i> = 7.0 Hz, 1H), 7.91 (d, <i>J</i> = 8.8 Hz, 2H), 7.83 (br. s, 4H), 4.54–7.23 (m, 14H), 7.20 (d, <i>J</i> = 8.4 Hz, 2H), 7.06 (d, <i>J</i> = 8.8 Hz, 2H), 7.00 (d, <i>J</i> = 8.7 Hz, 2H), 5.75–5.67 (m, 1H), 5.18 (s, 2H), 5.13 (s, 2H), 4.52–4.34 (m, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>36</sub>N<sub>5</sub>O<sub>4</sub>: 614.2762, found: 614.2777.</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)-4-(4-chlorobenzyloxy)benzamide (<b>46</b>)</h4><div class="NLM_p last"><b>46</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (82 mg, 0.11 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (53 mg, 0.17 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.020 mL, 0.12 mmol). Obtained as a pale orange powder (TFA salt, 60 mg, 69% yield). <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.09 (s, 1H), 8.02 (t, <i>J</i> = 5.7 Hz, 1H), 7.98–7.90 (m, 3H), 7.85 (br. s, 4H), 7.51 (d, <i>J</i> = 8.3 Hz, 2H), 7.50–7.45 (m, 4H), 7.43 (d, <i>J</i> = 8.4 Hz, 2H), 7.39 (t, <i>J</i> = 7.5 Hz, 2H), 7.35–7.27 (m, 3H), 7.21 (d, <i>J</i> = 8.3 Hz, 2H), 7.07 (d, <i>J</i> = 8.7 Hz, 2H), 7.00 (d, <i>J</i> = 8.6 Hz, 2H), 5.70 (d, <i>J</i> = 7.1 Hz, 1H), 5.20 (s, 2H), 5.13 (s, 2H), 4.50–4.40 (m, 2H). <sup>13</sup>C NMR (APT, 126 MHz, acetone-<i>d</i><sub>6</sub>) δ 174.8, 171.4, 166.2, 162.1, 159.5, 139.2, 138.4, 136.9, 134.9, 134.1, 132.1, 130.2, 130.1, 130.0, 129.4, 129.4, 129.3, 128.7, 128.4, 127.9, 126.0, 115.7, 115.3, 70.4, 69.8, 57.9, 42.9. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>35</sub>ClN<sub>5</sub>O<sub>4</sub>: 648.2372, found: 648.2380.</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)-4-(3-methoxybenzyloxy)benzamide (<b>47</b>)</h4><div class="NLM_p last"><b>47</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (76 mg, 0.11 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (49 mg, 0.16 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.018 mL, 0.11 mmol). Obtained as a pale yellow powder (TFA salt, 44 mg 55% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.84 (s, 1H), 8.12 (t, <i>J</i> = 5.9 Hz, 1H), 8.00 (d, <i>J</i> = 7.2 Hz, 1H), 7.90 (d, <i>J</i> = 8.8 Hz, 2H), 7.80 (br. s, 4H), 7.47 (d, <i>J</i> = 8.6 Hz, 4H), 7.43–7.23 (m, 6H), 7.19 (d, <i>J</i> = 8.2 Hz, 2H), 7.10–6.95 (m, 6H), 6.92–6.86 (m, 1H), 5.76–5.68 (m, 1H), 5.16 (s, 2H), 5.12 (s, 2H), 4.52–4.32 (m, 2H), 3.79 (s, 3H). <sup>13</sup>C NMR (75 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.6, 166.6, 162.4, 161.0, 159.6, 157.8, 139.5, 139.3, 138.4, 134.8, 132.0, 130.5, 130.2, 130.1, 129.4, 129.4, 128.7, 128.5, 126.1, 120.5, 115.7, 115.4, 114.0, 70.8, 58.1, 55.6, 43.0. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>38</sub>H<sub>38</sub>N<sub>5</sub>O<sub>5</sub>: 644.2867, found: 644.2873.</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)biphenyl-4-carboxamide (<b>48</b>)</h4><div class="NLM_p last"><b>48</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (85 mg, 0.13 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (61 mg, 0.20 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.022 mL, 0.13 mmol). Obtained as a yellow powder (TFA salt, 40 mg, 44% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.16 (d, <i>J</i> = 7.3 Hz, 1H), 8.10 (t, <i>J</i> = 5.9 Hz, 1H), 8.03 (d, <i>J</i> = 8.3 Hz, 2H), 7.80–7.66 (m, 4H), 7.55–7.25 (m, 12H), 7.21 (d, <i>J</i> = 8.5 Hz, 2H), 7.02 (d, <i>J</i> = 8.7 Hz, 2H), 5.80–5.73 (m, 1H), 5.14 (s, 2H), 4.55–4.37 (m, 2H). <sup>13</sup>C NMR (APT, 75 MHz, acetone-<i>d</i><sub>6</sub>) δ 171.8, 171.3, 159.6, 159.0, 144.8, 140.8, 139.2, 138.4, 134.9, 134.0, 131.9, 130.0, 129.9, 129.4, 129.3, 128.9, 128.9, 128.7, 128.4, 127.9, 127.6, 126.0, 115.7, 70.4, 58.0, 42.9. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>34</sub>N<sub>5</sub>O<sub>3</sub>: 584.2656, found: 584.2682.</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)-4′-chlorobiphenyl-4-carboxamide (<b>49</b>)</h4><div class="NLM_p last"><b>49</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (71 mg, 0.10 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (48 mg, 0.15 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.018 mL, 0.10 mmol). Obtained as a pale orange powder (TFA salt, 62 mg 83% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.14 (s, 1H), 8.16 (d, <i>J</i> = 6.9 Hz, 1H), 8.11–7.99 (m, 3H), 7.90 (br. s, 4H), 7.78–7.71 (m, 4H), 7.54–7.45 (m, 6H), 7.44–7.27 (m, 5H), 7.22 (d, <i>J</i> = 8.5 Hz, 2H), 7.02 (d, <i>J</i> = 8.8 Hz, 2H), 5.75 (d, <i>J</i> = 6.8 Hz, 1H), 5.14 (s, 2H), 4.54–4.39 (m, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>33</sub>ClN<sub>5</sub>O<sub>3</sub>: 618.2266, found: 618.2265.</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)biphenyl-2-carboxamide (<b>50</b>)</h4><div class="NLM_p last"><b>50</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (65 mg, 0.10 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (46 mg, 0.15 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.017 mL, 0.10 mmol). Obtained as a pale yellow powder (TFA salt, 49 mg, 71% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 11.04 (s, 1H), 7.86 (br. s, 4H), 7.79 (t, <i>J</i> = 6.0 Hz, 1H), 7.56 (d, <i>J</i> = 7.5 Hz, 1H), 7.54–7.18 (m, 18H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 6.92 (d, <i>J</i> = 8.8 Hz, 2H), 5.54–5.47 (m, 1H), 5.14 (s, 2H), 4.45–4.26 (m, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>36</sub>H<sub>34</sub>N<sub>5</sub>O<sub>3</sub>: 584.2656, found: 584.2641.</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-guanidinobenzylamino)-2-oxoethyl)-2,2′-bithiophene-5-carboxamide (<b>51</b>)</h4><div class="NLM_p last"><b>51</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (119 mg, 0.18 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (83 mg, 0.27 mmol), DMAP (7 mg, 0.05 mmol), and DIPEA (0.030 mL, 0.18 mmol). Obtained as a yellow powder (TFA salt, 48 mg, 38% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.75 (s, 1H), 8.18–8.04 (m, 2H), 7.84–7.68 (m, 5H), 7.53–7.16 (m, 14H), 7.14–7.08 (m, 1H), 7.01 (d, <i>J</i> = 8.7 Hz, 2H), 5.74–5.62 (m, 1H), 5.13 (s, 2H), 4.55–4.34 (m, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>30</sub>N<sub>5</sub>O<sub>3</sub>S<sub>2</sub>: 596.1785, found: 596.1793.</div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (<i>S</i>)-2-(4-(Benzyloxy)phenyl)-2-(2-(biphenyl-4-yl)acetamido)-<i>N</i>-(4-guanidinobenzyl)acetamide (<b>52</b>)</h4><div class="NLM_p last"><b>52</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (87 mg, 0.13 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (62 mg, 0.20 mmol), DMAP (5 mg, 0.04 mmol), and DIPEA (0.022 mL, 0.13 mmol). Obtained as a pale yellow powder (TFA salt, 48 mg, 52% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 7.58 (d, <i>J</i> = 7.2 Hz, 2H), 7.54 (d, <i>J</i> = 8.3 Hz, 2H), 7.46–7.39 (m, 4H), 7.39–7.27 (m, 8H), 7.25 (d, <i>J</i> = 8.5 Hz, 2H), 7.12 (d, <i>J</i> = 8.5 Hz, 2H), 6.99 (d, <i>J</i> = 8.8 Hz, 2H), 5.36 (s, 1H), 5.10 (s, 2H), 4.41–4.32 (m, 2H), 3.64 (s, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD) δ 173.6, 173.1, 160.3, 158.1, 142.1, 141.1, 139.7, 138.6, 135.9, 134.8, 130.7, 130.6, 130.1, 129.9, 129.7, 129.5, 128.9, 128.5, 128.3, 128.1, 127.9, 126.7, 122.5, 116.3, 70.9, 58.9, 49.0, 43.5, 42.9. HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>36</sub>N<sub>5</sub>O<sub>3</sub>: 598.2813, found: 598.2831.</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (<i>R</i>)-<i>N</i>-(1-(4-(3,4-Dichlorobenzyloxy)phenyl)-2-(4-(guanidinomethyl)benzylamino)-2-oxoethyl)-4-phenoxybenzamide (<b>54</b>)</h4><div class="NLM_p last"><b>54</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (27 mg, 0.04 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (17 mg, 0.05 mmol), DMAP (1 mg, 0.01 mmol), and DIPEA (0.006 mL, 0.04 mmol). Obtained as a yellow powder (TFA salt, 10 mg, 34% yield). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>CN) δ 11.84 (s, 1H), 7.98 (br. s, 4H), 7.83 (d, <i>J</i> = 8.8 Hz, 2H), 7.66–7.62 (m, 1H), 7.55 (d, <i>J</i> = 8.5 Hz, 2H), 7.46–7.34 (m, 6H), 7.27–7.18 (m, 5H), 7.07 (d, <i>J</i> = 7.5 Hz, 2H), 7.03–6.95 (m, 4H), 5.49 (d, <i>J</i> = 7.0 Hz, 1H), 5.08 (s, 2H), 4.35 (d, <i>J</i> = 6.5 Hz, 2H), 4.30 (d, <i>J</i> = 5.9 Hz, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>4</sub>: 682.1982, found: 682.1975.</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>S</i>)-2-(2-(Biphenyl-4-yl)acetamido)-2-(4-(2,6-dichlorobenzyloxy)phenyl)-<i>N</i>-(4-guanidinobenzyl)acetamide (<b>55</b>)</h4><div class="NLM_p last"><b>55</b> was synthesized according to the general procedure for guanidinylation from the respective aniline (85 mg, 0.12 mmol), <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine (54 mg, 0.17 mmol), DMAP (4 mg, 0.03 mmol), and DIPEA (0.020 mL, 0.12 mmol). Obtained as a pale yellow powder (TFA salt, 72 mg, 80% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 7.64 (d, <i>J</i> = 7.6 Hz, 2H), 7.58 (d, <i>J</i> = 7.3 Hz, 2H), 7.52 (d, <i>J</i> = 7.6 Hz, 2H), 7.49–7.38 (m, 7H), 7.37–7.13 (m, 5H), 7.04 (d, <i>J</i> = 7.4 Hz, 2H), 5.54 (s, 1H), 5.31 (s, 2H), 4.48–4.32 (m, 2H), 3.68 (s, 2H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>37</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>3</sub>: 666.2033, found: 666.2020.</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-((6-guanidinopyridin-3-yl)methylamino)-2-oxoethyl)hexanamide (<b>21</b>)</h4><div class="NLM_p last"><b>21</b> was synthesized in analogy to a previously described synthesis.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> To a solution of <b>12</b> (10 mg, 0.02 mmol) and <i>N</i>,<i>N</i>′-di-(<i>tert</i>-butoxycarbonyl)-<i>S</i>-methylisothiourea (7 mg, 0.02 mmol) in THF (5 mL) was added triethylamin (0.008 mL, 0.06 mmol). The solution was cooled on an ice-water bath, and mercuric chloride (5 mg, 0.02 mmol) was added. After 15 min, the stirred suspension was allowed to warm to room temperature and stirred till no leftover <b>12</b> was observed by TLC analysis. The solvent was removed in vacuo, and the residue was suspended in ethyl acetate and filtered through Celite. The filtrate was washed several times with water and brine, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The resulting residue was dissolved in TFA/DCM (1:1, 10 mL) and allowed to stir for 3 h. The mixture was concentrated and further co-evaporated with several portions of toluene. The residue was purified by preparative HPLC and freeze-dried in water. Obtained as a pale yellow powder (TFA salt, 5 mg, 45% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.76 (s, 1H), 8.18 (s, 1H), 7.96 (t, <i>J</i> = 5.9 Hz, 1H), 7.78 (br. s, 4H), 7.72 (d, <i>J</i> = 8.6 Hz, 1H), 7.53 (d, <i>J</i> = 5.7 Hz, 1H), 7.47 (d, <i>J</i> = 7.1 Hz, 2H), 7.43–7.29 (m, 6H), 6.96 (d, <i>J</i> = 8.7 Hz, 2H), 5.49–5.40 (m, 1H), 5.11 (s, 2H), 4.42–4.29 (m, 2H), 2.24 (t, <i>J</i> = 7.1 Hz, 2H), 1.67–1.51 (m, 2H), 1.36–1.23 (m, 4H), 0.85 (t, <i>J</i> = 7.2 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>35</sub>N<sub>6</sub>O<sub>3</sub>: 503.2765, found: 503.2755.</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-carbamimidoylbenzylamino)-2-oxoethyl)hexanamide (<b>18</b>)</h4><div class="NLM_p last">To a solution of compound <b>9</b> (68 mg, 0.14 mmol) in THF (10 mL) were added 2,4,6-trimethylpyridine (0.057 mL, 0.43 mmol) and [NH<sub>3</sub>OH]Cl (30 mg, 0.43 mmol), and the mixture was stirred at 60 °C overnight. The solvent was removed under reduced pressure to afford an oil, which was dissolved in ethyl acetate. The solution was washed two times with 0.05 M aqueous NaOH and water, dried over MgSO<sub>4</sub>, and concentrated in vacuo<i>.</i> The resulting residue was mixed with SnCl<sub>2</sub>·H<sub>2</sub>O (259 mg, 1.15 mmol) in DMF (15 mL) to result in a slurry, which was stirred for 3 days under reflux in a nitrogen atmosphere. The mixture was filtered through Celite and concentrated to provide <b>18</b>, which was further purified with reverse-phase liquid chromatography and freeze-dried in water (TFA salt, 21 mg, 24% over 2 steps). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD /D<sub>2</sub>O) δ 7.88 (d, <i>J</i> = 9.0 Hz, 2H), 7.70 (d, <i>J</i> = 8.5 Hz, 2H), 7.49–7.37 (m, 5H), 7.17 (d, <i>J</i> = 9.0 Hz, 2H), 7.00 (d, <i>J</i> = 8.6 Hz, 2H), 5.35 (s, 1H), 5.22 (s, 2H), 4.48 (s, 2H), 2.27 (t, <i>J</i> = 7.5 Hz, 2H), 1.68–1.54 (m, 2H), 1.39–1.26 (m, 4H), 0.88 (t, <i>J</i> = 6.7 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>O<sub>3</sub>: 487.2704, found: 487.2711.</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-((dimethylamino)methyl)benzylamino)-2-oxoethyl)hexanamide (<b>16</b>)</h4><div class="NLM_p last">Compound <b>14</b> (10 mg, 0.02 mmol) was dissolved in MeOH (2 mL), and formalin (37% formaldehyde in water solution, 0.006 mL) was added. After 10 min of stirring, sodium cyanoborohydride (4 mg, 0.06 mmol) was added as a solution in MeOH (2 mL). The mixture was left to stir for 5 h, and concentrated HCl (0.5 mL) was added. The solvent was removed under reduced pressure, and the resulting residue was redissolved in ethyl acetate. The organic phase was washed tow times with 1 N aqueous NaOH and brine, dried over anhydrous MgSO<sub>4</sub>, and concentrated in vacuo<i>.</i> The residue was further purified with reverse-phase liquid chromatography and freeze-dried in water to afford a white powder (TFA salt, 10 mg, 77% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.00 (t, <i>J</i> = 6.0 Hz, 1H), 7.60 (d, <i>J</i> = 7.4 Hz, 1H), 7.52–7.44 (m, 4H), 7.44–7.32 (m, 5H), 7.29 (d, <i>J</i> = 8.0 Hz, 2H), 6.96 (d, <i>J</i> = 8.7 Hz, 2H), 5.55–5.49 (m, 1H), 5.12 (s, 2H), 4.43 (d, <i>J</i> = 4.2 Hz, 2H), 4.30 (s, 2H), 2.80 (s, 6H), 2.26 (t, <i>J</i> = 7.5 Hz, 2H), 1.64–1.51 (m, 2H), 1.37–1.20 (m, 4H), 0.85 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>40</sub>N<sub>3</sub>O<sub>3</sub>: 502.3064, found: 502.3068.</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-oxo-2-(4-ureidobenzylamino)ethyl)hexanamide (<b>24</b>)</h4><div class="NLM_p last">To a solution of acetic acid (1 mL) in water (0.250 mL) was added compound <b>11</b> (14 mg, 0.02 mmol). To this mixture, a solution of sodium cyanate (3 mg, 0.05 mmol) in water was added. A gray precipitate was formed, and the suspension was stirred overnight. The mixture was cooled with an ice-water bath and filtered. The collected solid was washed several times with cold water and dried in vacuo. The solid was further purified with reverse-phase liquid chromatography and freeze-dried in water to afford a white powder (10 mg, 83% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.04 (br. s, 1H), 7.71 (t, <i>J</i> = 6.1 Hz, 1H), 7.55–7.43 (m, 3H), 7.44–7.29 (m, 7H), 7.06 (d, <i>J</i> = 8.4 Hz, 2H), 6.95 (d, <i>J</i> = 8.7 Hz, 2H), 5.52–5.46 (m, 1H), 5.40 (br. s, 2H), 5.11 (s, 2H), 4.30 (d, <i>J</i> = 5.9 Hz, 2H), 2.25 (t, <i>J</i> = 7.5 Hz, 2H), 1.64–1.51 (m, 2H), 1.35–1.21 (m, 4H), 0.86 (t, <i>J</i> = 6.8 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>O<sub>4</sub>: 503.2653, found: 503.2649.</div></div><div id="sec4_2_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (<i>S</i>)-<i>N</i>-(1-(4-(Benzyloxy)phenyl)-2-(4-(hydrazinecarbonyl)benzylamino)-2-oxoethyl)hexanamide (<b>25</b>)</h4><div class="NLM_p last">The respective carboxylic acid was synthesized with SPPS on 2-chlorotrityl chloride resin (<b>58</b>) (28 mg, 0.06 mmol) and dissolved in one drop of dry THF under a nitrogen atmosphere. To this solution, carbonyldiimidazole (14 mg, 0.09 mmol) in dry THF (0.5 mL) was added. The mixture was stirred for 2 h at room temperature, and hydrazine monohydrate (7 mg, 21 mmol) in dry THF (0.5 mL) was added. The mixture was further stirred for 5 h, and the solvent was removed under reduced pressure. The resulting residue was dissolved in ethyl acetate and was washed two times with 0.05 N aqueous NaOH and water. The organic phase was concentrated in vacuo, filtered over anhydrous MgSO<sub>4</sub>, and was purified with reverse-phase liquid chromatography. The purified product was redissolved in water and freeze-dried to afford a white powder (11 mg, 38% yield). <sup>1</sup>H NMR (300 MHz, acetone-<i>d</i><sub>6</sub>) δ 7.93 (t, <i>J</i> = 5.9 Hz, 1H), 7.75 (d, <i>J</i> = 6.9 Hz, 2H), 7.54 (d, <i>J</i> = 7.9 Hz, 1H), 7.47 (d, <i>J</i> = 7.1 Hz, 2H), 7.43–7.22 (m, 7H), 6.98 (d, <i>J</i> = 8.6 Hz, 2H), 5.50 (d, <i>J</i> = 7.4 Hz, 1H), 5.12 (s, 2H), 4.45 (d, <i>J</i> = 6.1 Hz, 2H), 2.26 (t, <i>J</i> = 7.4 Hz, 2H), 1.64–1.51 (m, 2H), 1.33–1.22 (m, 4H), 0.86 (t, <i>J</i> = 6.9 Hz, 3H). HRMS (ESI): <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>O<sub>4</sub>: 503.2653, found: 503.2656.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> General Procedure for the Synthesis of Inhibitors and Intermediates on the Solid Support</h3><div class="NLM_p last">All peptides, peptide hybrid sequences, and intermediates (e.g., (<i>S</i>)-2-(4-(benzyloxy)phenyl)-2-hexanamidoacetic acid (<b>57</b>), (<i>S</i>)-4-((2-(4-(benzyloxy)phenyl)-2-hexanamidoacetamido)methyl)benzoic acid (<b>58</b>)) were assembled by stepwise solid-phase synthesis on CTC resin using the standard Fmoc-strategy, as previously described.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> Solid-phase synthesis was done manually in plastic syringes equipped with a frit; all steps were performed at room temperature under continuous shaking. The first amino acid or respective building block (1.5–3.0 equiv) was added to the resin in a syringe and DMF (1 mL), and TMP (1.5–3.0 equiv) was added. The resin was shaken overnight and capped before the Fmoc deprotection step. Therefore, the resin was washed with 3× DMF, 3× DCM, and again 3× DMF and a capping solution (DCM/MeOH/TMP 80:15:5 (v/v), 1 mL per 100 mg resin) was added two times for 20 min. For Fmoc deprotection, a piperidine solution (10–20% in DMF) was added two times for 10 and 5 min. Following each deprotection or coupling step, the resin was washed with 3× DMF, 3× DCM, and again 3× DMF. COMU/TMP was used for all coupling steps. In detail, the coupling solution contained the <i>N</i><sub>α</sub>-Fmoc-protected amino acid or N-terminal carboxylic acid capping group (3.0 equiv), COMU (3.0 equiv), and TMP (3.9 equiv) in DMF (∼1.0 mL per 100 mg of resin). The solution was added to the resin, and the reaction was shaken for 60–120 min. Afterward, the resin was washed as described before. Fmoc deprotection and coupling steps were iteratively repeated until the desired sequence was obtained. The resin loaded with the finished compound or intermediate was washed five times with diethyl ether and dried under reduced pressure. The final product was cleaved off the resin three times with TFA/DCM solution (5:95, 1–2 mL per 100 mg resin), and the mixture was shaken for at least 30 min. The cleavage solution was dispensed into a round-bottom flask, dried under reduced pressure, and co-evaporated with toluene several times. All crude compounds were purified by preparative HPLC on an ÄKTA Purifier, GE Healthcare (Germany), with an RP-18 pre and main column (Rephospher, Dr. Maisch GmbH, Germany, C18-DE, 5 μm, 30 mm × 16 mm and 120 mm × 16 mm). The following conditions were used: eluent A, water (0.1% TFA); eluent B, methanol (0.1% TFA); flow rate, 8 mL/min; and gradient, 10% B (2.5 min), 100% B (23.5 min), 100% B (26 min), 10% B (26.1 min), and 10% B (30 min). Detection was performed at 214, 254, and 280 nm. After purification, methanol was evaporated, and the compounds and intermediates were freeze-dried in water and stored at −20 °C.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> DENV and WNV Protease Relative Inhibition Assay</h3><div class="NLM_p last">The DENV and WNV protease relative inhibition assays were performed as described before.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> Continuous enzymatic assays were performed in black 96-well V-bottom plates (Greiner Bio-One, Germany) using a BMG Labtech Fluostar OPTIMA microtiter fluorescence plate reader at an excitation wavelength of 320 nm and a monitored emission wavelength of 405 nm. Stock solutions of the inhibitors (10 mM in DMSO) were diluted to a final concentration of 50 μM in triplicate and preincubated for 15 min with the DENV protease (100 nM) or WNV protease (150 nM) in the assay buffer (50 mM Tris–HCl pH 9, ethylene glycol (10% v/v), and 0.0016% Brij 58). The reaction was then initiated by the addition of the FRET substrate (final concentration 50 μM) to obtain a final assay volume of 100 μL per well. The enzymatic activity was monitored for 15 min and determined as a slope of relative fluorescence units per second (RFU/s) for each concentration. Percentage inhibition was calculated relative to a positive control (without the inhibitor), as a mean of the triplicates and respective standard deviation.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> DENV and WNV Protease Assays for IC<sub>50</sub> Determination</h3><div class="NLM_p last">IC<sub>50</sub> for DENV and WNV inhibitors were performed as described previously.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> Continuous enzymatic assays were performed in black 96-well V-bottom plates (Greiner Bio-One, Germany) using a BMG Labtech Fluostar OPTIMA Microtiter fluorescence plate reader at an excitation wavelength of 320 nm and a monitored emission wavelength of 405 nm. Eight inhibitor concentrations covering the range 0–3, 0–6, 0–10, 0–15, 0–30, or 0–50 μM were chosen for analysis. Dilutions were made starting from 10 mM stock solutions in DMSO, and final concentrations were measured in triplicate. The inhibitors were preincubated for 15 min with the DENV protease (100 nM) or WNV protease (150 nM) in the assay buffer (50 mM Tris–HCl pH 9.0, ethylene glycol (10% v/v), and 0.0016% Brij 58). The enzymatic reaction was then initiated by the addition of the FRET substrate (final concentration 50 μM) to obtain a final assay volume of 100 μL per well. The enzymatic activity was monitored for 15 min and determined as a slope of relative fluorescence units per second (RFU/s) for each concentration. The mean and the standard deviation of the triplicates plotted against the corresponding concentration were used to determine the IC<sub>50</sub> values on Prism 6.01 (Graphpad Software, Inc.) using nonlinear dose–response curves with variable slopes.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Tryptophan Fluorescence Quenching Assay</h3><div class="NLM_p last">The tryptophan fluorescence quenching assay was performed as described by Bodenreider et al.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> DENV protease (0.2 μM) was titrated with different amounts (0, 1.25, 2.5, 5, 10, 20, 30, 40, 50 μM) of inhibitor in assay buffer (50 mM Tris–HCl pH 9.0, ethylene glycol (10% v/v), and 0.0016% Brij 58), and the samples were incubated for 1 h at room temperature. For the fluorescence displacement experiment, the protease and NATA controls were incubated with the inhibitor (50 μM) and aprotinin (10 μM) under the same conditions. Fluorescence of the enzymes (0.2 μM DENV) in the presence of aprotinin (10 μM), without the inhibitor, was also determined and used to correct the results of aprotinin displacement, in order to exclude aprotinin intrinsic fluorescence or inner filter effects. Fluorescence emission at 340 nm was monitored on a Tecan Safire II instrument (excitation at 280 nm). All experiments were performed in triplicate, and the values were obtained in relative fluorescence units (RFU). Tryptophan quenching was plotted as a curve of the mean and standard deviation of the values against the respective concentrations of the inhibitor in comparison to the fluorescence displacement experiment using Prism 6.01 (Graphpad Software, Inc.).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Cell Culture</h3><div class="NLM_p last">If not stated otherwise, HeLa, Huh-7, and Vero E6 cells were maintained in DMEM supplemented with 100 U/mL penicillin, 100 μg/mL streptomycin, and 10% heat-inactivated FCS. During infection of Huh-7 cells, DMEM was supplemented with 10 mM HEPES.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Cloning Strategy</h3><div class="NLM_p last">For construction of the reporter plasmids, the oxygen-dependent degradation domain of hypoxia-inducible factor 1-alpha isoform 1 (HIF1α, corresponding sequence to the amino acids 392 to 715) was cloned from human cDNA into a pRL-sv40 vector backbone (Promega) using Phusion DNA polymerase (NEB). All cloning steps were done by overlap extension PCR. Sequences of primes are found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supplementary Information</a>. For amplification of the degradation domain, the primers ODD_fwd and ODD_rev_A/B/C were used. The sequences that were introduced represent the DENV protease NS2A/NS2B cleavage site (ODD_rev_A), the NS3/NS4A cleavage site (ODD_rev_B), and the NS4B/NS5 cleavage (ODD_rev_C). For stable transfection of the reporter plasmid, a puromycin resistance cassette following an EMCV IRES sequence was cloned into the vector downstream of the <i>Renilla reniformis</i> luciferase ORF using the primers RF_IRES-Puro_1 and RF_IRES-Puro_2. As a template for amplification of the protease, DENV genomic DNA was used. The sequence was cloned into the pEF1α-IRES-Neo vector (Addgene plasmid #28019) using the primer pair RF_NS2B3_1 and RF_NS2B3_2.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Transfection of Mammalian Cells and Stable Cell Line Generation</h3><div class="NLM_p last">All transfections were done using the Lipofectamine 3000 transfection reagent (Life technologies). The plasmid DNA was diluted in Opti-MEM with 0.2 μL of P3000 reagent per 100 ng of DNA and mixed with 0.3 μL of Lipofectamine 3000 transfection reagent per 100 ng of DNA, which was also diluted in Opti-MEM. Ten minutes after mixing, the cell suspension was added and seeded into the respective plates or flasks. For stable transfections, selected antibiotics were added 24 h post-transfection. In brief, 0.8 μg/mL puromycin and 800 μg/mL G418 (Geneticin) were added and the cells were further incubated in DMEM supplemented with 10% FBS for at least 14 days at 37 °C. Afterward, surviving cells were detached, seeded as single cells, and grown out in selected medium in order to generate single-cell clones.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Measurement of Luminescence (DENV2proHeLa)</h3><div class="NLM_p last">Stable cells were seeded into 96-well plates with a density of 2 × 10<sup>4</sup> cells per well and treated immediately with inhibitors of DENV protease in a final volume of 100 μL. After incubation for 24 h at 37 °C, the medium was removed and cells were lysed by adding 25 μL lysis buffer (Promega) for 15 min at room temperature. Luciferase activity was recorded using a FLUOstar omega plate reader (BMG labtech) with injections of 100 μL of coelenterazine (2.75 μM in PBS) per well. Luminescence was recorded for 5 s. Measurements for <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B were performed with 100 μL of coelenterazine (1 μM in PBS) per well, and luminescence was recorded for 10 s. Each concentration was assayed in triplicate. Percent inhibition was calculated using an untreated control. For EC<sub>50</sub> calculations, data were fitted and calculated with Prism 6.01 (Graphpad Software, Inc.) using a four-parameter nonlinear dose–response curve with background subtraction (wells without cells).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Virus Titer Reduction Assay</h3><div class="NLM_p last">Huh-7 cells were seeded into 96-well plates at a density of 1 × 10<sup>4</sup> cells in 50 μL per well and incubated overnight. The next day, cells were infected with DENV serotype 2 for 2 h at an MOI of 1 before medium change and compound addition. Infected cells were then incubated for 48 h in the presence of compound in triplicate wells. In order to determine the EC<sub>50</sub>, a range of serially diluted compound concentrations starting at 50 μM was used. After 48 h of incubation, supernatants were harvested and triplicates were pooled. These virus-containing supernatants were used to determine the virus titer (reduction) by a plaque assay, as described elsewhere.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Briefly, VeroE6 cells were seeded into 24-well plates at a density of 2.5 × 10<sup>5</sup> cells/well and the next day infected for 1 h with 10-fold serial dilutions of the virus supernatant ranging from 10<sup>–1</sup> to 10<sup>–6</sup>. After medium exchange and addition of the plaque medium, VeroE6 cells were incubated for 7 days. Cells were subsequently fixed with formaldehyde, and plaques were visualized by crystal violet stain. At a suitable dilution, plaques were counted, and the virus titer was calculated and plotted against the respective compound concentration. EC<sub>50</sub> values were derived from fitting data to a dose–response curve (variable-slope, nonlinear regression model) using Prism 6.01 (Graphpad Software, Inc.).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Cytotoxicity</h3><div class="NLM_p last">Cell viability in Huh-7 or Hela cells in the presence of compound dilutions was determined using CellTiter-Blue (Promega) according to the manufacturer’s instructions. Plates were prepared in parallel to the cell-based DENV reporter or virus titer reduction assay with analogous treatment. Each concentration was assayed in triplicate.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Molecular Modeling</h3><div class="NLM_p last">The structures of the ligands were sketched using ChemBioDraw Ultra v.11.0 (CambridgeSoft Corporation, USA), minimized using the MM2 method. The calculations were performed on an Intel(R) Core(TM) 2 Quad CPU Q9450 @ 2.66 GHz running open SuSE 11.0, using GOLD 5.1 and its graphical interface Hermes 1.4.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The crystal structure of the NS2B–NS3 protease of DENV serotype 3 in complex with a tetrapeptidyl aldehyde was extracted from the dimer structure 3U1I.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> All water molecules and sulfate ions were removed, and the structure was further prepared using Dock Prep (Chimera).<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> All hydrogens were added, and the ligand was extracted from the structure. The binding site was defined in a radius of 10 Å around the extracted ligand, and automatic cavity detection was used. A total of 10 different solutions were calculated for every ligand, and the docking results were aligned and visualized using Chimera 11.1.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00413" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74540" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74540" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00413?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00413</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Expression and purification of the viral proteases, primers used for cloning of reporter constructs and the DENV-2 protease, reporter constructs used for co-transfection, assay validation, inhibitors used for assay correlation, DENV2proHeLa dose–response curves, kinetic studies, characterization of compound <b>51</b>, HPLC-based DENV and WNV protease assays, inhibitory activity of compounds against thrombin and trypsin, structures of reference compounds, PAMPA, FRET substrate synthesis, synthesis and analytical data of precursors, analytical data of inhibitors, HPLC purity of inhibitors, enantiomeric purity of inhibitors, and selected <sup>1</sup>H und <sup>13</sup>C NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and activity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf">jm0c00413_si_001.pdf (3.57 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_002.csv">jm0c00413_si_002.csv (9.14 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00413" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian D. Klein</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Institute
of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3522-9182" title="Orcid link">http://orcid.org/0000-0003-3522-9182</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8feca1e4e3eae6e1cffae1e6a2e7eae6ebeae3edeafde8a1ebea"><span class="__cf_email__" data-cfemail="6e0d4005020b07002e1b000743060b070a0b020c0b1c09400a0b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nikos Kühl</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Institute
of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominik Graf</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Institute
of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Josephine Bock</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Institute
of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mira A. M. Behnam</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Institute
of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5839-7675" title="Orcid link">http://orcid.org/0000-0002-5839-7675</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mila-Mareen Leuthold</span> - <span class="hlFld-Affiliation affiliation">Medicinal Chemistry, Institute
of Pharmacy and Molecular Biotechnology IPMB, Heidelberg University, Im Neuenheimer Feld 364, 69120 Heidelberg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7698-2360" title="Orcid link">http://orcid.org/0000-0001-7698-2360</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): A patent application, which covers the chemical structures and medical application of the compounds described in this work, was filed by Heidelberg University (PCT/EP2020/056001).<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15288" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15288" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Deutsche Forschungsgemeinschaft under grant no. 1356/3-2. We thank Heiko Rudy for measuring ESI high-resolution spectra and Natascha Stefan and Tobias Timmermann for technical support. We thank Felix Streicher for synthesis of <b>60</b>, <b>61</b>, <b>62</b>, <b>63</b>, and <b>64</b>. We thank Prof. Ralf Bartenschlager and Marie Bartenschlager for providing DENV-2 stock solutions and experimental input. Additional financial support for the development of the DENV2proHeLa system into a high-throughput-screening tool was received from the Health-Holland Kickstarter initiative.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS</td></tr><tr><td class="NLM_term">COMU</td><td class="NLM_def"><p class="first last">(1-cyano-2-ethoxy-2-oxoethylidenaminooxy)dimethylamino-morpholino-carbenium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DENV</td><td class="NLM_def"><p class="first last">dengue virus</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">endoplasmic reticulum</p></td></tr><tr><td class="NLM_term">FCS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">Fmoc-OSu</td><td class="NLM_def"><p class="first last">9-fluorenylmethyl-<i>N</i>-succinimidyl carbonate</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">(1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxidohexafluorophosphate</p></td></tr><tr><td class="NLM_term">HCV</td><td class="NLM_def"><p class="first last">hepatitis C virus</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">HIF1α</td><td class="NLM_def"><p class="first last">hypoxia-inducible factor 1-alpha</p></td></tr><tr><td class="NLM_term">HIV</td><td class="NLM_def"><p class="first last">human immunodeficiency virus</p></td></tr><tr><td class="NLM_term">MOI</td><td class="NLM_def"><p class="first last">multiplicity of infection</p></td></tr><tr><td class="NLM_term">NSAID</td><td class="NLM_def"><p class="first last">nonsteroidal anti-inflammatory drug</p></td></tr><tr><td class="NLM_term">ODD</td><td class="NLM_def"><p class="first last">oxygen degradable domain</p></td></tr><tr><td class="NLM_term">PEST</td><td class="NLM_def"><p class="first last">protein sequence rich in proline, glutamic acid, serine, and threonine</p></td></tr><tr><td class="NLM_term">SPPS</td><td class="NLM_def"><p class="first last">solid-phase peptide synthesis</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroaceticacid</p></td></tr><tr><td class="NLM_term">TMP</td><td class="NLM_def"><p class="first last">2,4,6-trimethylpyridine</p></td></tr><tr><td class="NLM_term">WNV</td><td class="NLM_def"><p class="first last">West Nile virus</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58046" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58046" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gething, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messina, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farlow, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyes, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownstein, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoen, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankoh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaenisch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wint, G. R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrar, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, S. I.</span></span> <span> </span><span class="NLM_article-title">The Global Distribution and Burden of Dengue</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>496</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/nature12060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fnature12060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=23563266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltlOnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=504-507&author=S.+Bhattauthor=P.+W.+Gethingauthor=O.+J.+Bradyauthor=J.+P.+Messinaauthor=A.+W.+Farlowauthor=C.+L.+Moyesauthor=J.+M.+Drakeauthor=J.+S.+Brownsteinauthor=A.+G.+Hoenauthor=O.+Sankohauthor=M.+F.+Myersauthor=D.+B.+Georgeauthor=T.+Jaenischauthor=G.+R.+W.+Wintauthor=C.+P.+Simmonsauthor=T.+W.+Scottauthor=J.+J.+Farrarauthor=S.+I.+Hay&title=The+Global+Distribution+and+Burden+of+Dengue&doi=10.1038%2Fnature12060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The global distribution and burden of dengue</span></div><div class="casAuthors">Bhatt, Samir; Gething, Peter W.; Brady, Oliver J.; Messina, Jane P.; Farlow, Andrew W.; Moyes, Catherine L.; Drake, John M.; Brownstein, John S.; Hoen, Anne G.; Sankoh, Osman; Myers, Monica F.; George, Dylan B.; Jaenisch, Thomas; Wint, G. R. William; Simmons, Cameron P.; Scott, Thomas W.; Farrar, Jeremy J.; Hay, Simon I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue">7446</span>),
    <span class="NLM_cas:pages">504-507</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dengue is a systemic viral infection transmitted between humans by Aedes mosquitoes.  For some patients, dengue is a life-threatening illness.  There are currently no licensed vaccines or specific therapeutics, and substantial vector control efforts have not stopped its rapid emergence and global spread.  The contemporary worldwide distribution of the risk of dengue virus infection and its public health burden are poorly known.  Here we undertake an exhaustive assembly of known records of dengue occurrence worldwide, and use a formal modeling framework to map the global distribution of dengue risk.  We then pair the resulting risk map with detailed longitudinal information from dengue cohort studies and population surfaces to infer the public health burden of dengue in 2010.  We predict dengue to be ubiquitous throughout the tropics, with local spatial variations in risk influenced strongly by rainfall, temp. and the degree of urbanization.  Using cartog. approaches, we est. there to be 390 million (95% credible interval 284-528) dengue infections per yr, of which 96 million (67-136) manifest apparently (any level of clin. or subclin. severity).  This infection total is more than three times the dengue burden est. of the World Health Organization.  Stratification of our ests. by country allows comparison with national dengue reporting, after taking into account the probability of an apparent infection being formally reported.  The most notable differences are discussed.  These new risk maps and infection ests. provide novel insights into the global, regional and national public health burden imposed by dengue.  We anticipate that they will provide a starting point for a wider discussion about the global impact of this disease and will help to guide improvements in disease control strategies using vaccine, drug and vector control methods, and in their economic evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVDkrHXysP5bVg90H21EOLACvtfcHk0lgbDNalgEITmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltlOnsLs%253D&md5=5b52dcfdd70064e2492f482bac1194b1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature12060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12060%26sid%3Dliteratum%253Aachs%26aulast%3DBhatt%26aufirst%3DS.%26aulast%3DGething%26aufirst%3DP.%2BW.%26aulast%3DBrady%26aufirst%3DO.%2BJ.%26aulast%3DMessina%26aufirst%3DJ.%2BP.%26aulast%3DFarlow%26aufirst%3DA.%2BW.%26aulast%3DMoyes%26aufirst%3DC.%2BL.%26aulast%3DDrake%26aufirst%3DJ.%2BM.%26aulast%3DBrownstein%26aufirst%3DJ.%2BS.%26aulast%3DHoen%26aufirst%3DA.%2BG.%26aulast%3DSankoh%26aufirst%3DO.%26aulast%3DMyers%26aufirst%3DM.%2BF.%26aulast%3DGeorge%26aufirst%3DD.%2BB.%26aulast%3DJaenisch%26aufirst%3DT.%26aulast%3DWint%26aufirst%3DG.%2BR.%2BW.%26aulast%3DSimmons%26aufirst%3DC.%2BP.%26aulast%3DScott%26aufirst%3DT.%2BW.%26aulast%3DFarrar%26aufirst%3DJ.%2BJ.%26aulast%3DHay%26aufirst%3DS.%2BI.%26atitle%3DThe%2520Global%2520Distribution%2520and%2520Burden%2520of%2520Dengue%26jtitle%3DNature%26date%3D2013%26volume%3D496%26spage%3D504%26epage%3D507%26doi%3D10.1038%2Fnature12060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardosa, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. G.</span></span> <span> </span><span class="NLM_article-title">Of Cascades and Perfect Storms: The Immunopathogenesis of Dengue Haemorrhagic Fever-Dengue Shock Syndrome (DHF/DSS)</span>. <i>Immunology & Cell Biology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1038/sj.icb.7100008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fsj.icb.7100008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17130899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhslOgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2007&pages=43-45&author=T.+Pangauthor=M.+J.+Cardosaauthor=M.+G.+Guzman&title=Of+Cascades+and+Perfect+Storms%3A+The+Immunopathogenesis+of+Dengue+Haemorrhagic+Fever-Dengue+Shock+Syndrome+%28DHF%2FDSS%29&doi=10.1038%2Fsj.icb.7100008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Of cascades and perfect storms: the immunopathogenesis of dengue haemorrhagic fever-dengue shock syndrome (DHF/DSS)</span></div><div class="casAuthors">Pang, Tikki; Cardosa, Mary Jane; Guzman, Maria G.</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-45</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past four decades has witnessed a consolidation of the original observations made in the 1970s that dengue haemorrhagic fever (DHF) and dengue shock syndrome (DSS) have an immunol. basis.  Following reinfection with a dengue virus of different serotype, severe disease is linked to high levels of antibody-enhanced viral replication early in illness which is followed by a cascade of memory T-cell activation and a 'storm' of inflammatory cytokines and other chem. mediators.  These compds. are released mainly from T cells, monocytes/macrophages and endothelial cells, and ultimately cause an increase in vascular permeability.  The consolidation of the evidence has been largely due to several important prospective sero-epidemiol. studies in areas endemic for DHF/DSS, which have shown that risk of severe disease is significantly higher in secondary dengue infections.  These advances have underscored the fact that DHF/DSS pathogenesis is a complex, multifactorial process involving cocirculation of various dengue virus serotypes and the interplay of host and viral factors that influence disease severity.  The continued search to define risk factors in susceptible populations must be combined with the new techniques of mol. virol. and innovative approaches in vaccine design to achieve the ultimate objective of developing a safe and effective vaccine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_zZX_80c847Vg90H21EOLACvtfcHk0liijiYG5rKWJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhslOgsLc%253D&md5=b7a4c7c5e3853e56f2405bd43a613878</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.icb.7100008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.icb.7100008%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DT.%26aulast%3DCardosa%26aufirst%3DM.%2BJ.%26aulast%3DGuzman%26aufirst%3DM.%2BG.%26atitle%3DOf%2520Cascades%2520and%2520Perfect%2520Storms%253A%2520The%2520Immunopathogenesis%2520of%2520Dengue%2520Haemorrhagic%2520Fever-Dengue%2520Shock%2520Syndrome%2520%2528DHF%252FDSS%2529%26jtitle%3DImmunology%2520%2526%2520Cell%2520Biology%26date%3D2007%26volume%3D85%26spage%3D43%26epage%3D45%26doi%3D10.1038%2Fsj.icb.7100008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halstead, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artsob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsperger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelegrino, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoksan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeling, R. W.</span></span> <span> </span><span class="NLM_article-title">Dengue: A Continuing Global Threat</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">S7</span>– <span class="NLM_lpage">S16</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fnrmicro2460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=21079655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVansb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=S7-S16&author=M.+G.+Guzmanauthor=S.+B.+Halsteadauthor=H.+Artsobauthor=P.+Buchyauthor=J.+Farrarauthor=D.+J.+Gublerauthor=E.+Hunspergerauthor=A.+Kroegerauthor=H.+S.+Margolisauthor=E.+Mart%C3%ADnezauthor=M.+B.+Nathanauthor=J.+L.+Pelegrinoauthor=C.+Simmonsauthor=S.+Yoksanauthor=R.+W.+Peeling&title=Dengue%3A+A+Continuing+Global+Threat&doi=10.1038%2Fnrmicro2460"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue: a continuing global threat</span></div><div class="casAuthors">Guzman, Maria G.; Halstead, Scott B.; Artsob, Harvey; Buchy, Philippe; Farrar, Jeremy; Gubler, Duane J.; Hunsperger, Elizabeth; Kroeger, Axel; Margolis, Harold S.; Martinez, Eric; Nathan, Michael B.; Pelegrino, Jose Luis; Simmons, Cameron; Yoksan, Sutee; Peeling, Rosanna W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12_supp</span>),
    <span class="NLM_cas:pages">S7-S16</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dengue fever and dengue haemorrhagic fever are important arthropod-borne viral diseases.  Each year, there are ∼50 million dengue infections and ∼500,000 individuals are hospitalized with dengue haemorrhagic fever, mainly in Southeast Asia, the Pacific and the Americas.  Illness is produced by any of the four dengue virus serotypes.  A global strategy aimed at increasing the capacity for surveillance and outbreak response, changing behaviors and reducing the disease burden using integrated vector management in conjunction with early and accurate diagnosis has been advocated.  Antiviral drugs and vaccines that are currently under development could also make an important contribution to dengue control in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTeD-NBdwYqbVg90H21EOLACvtfcHk0liijiYG5rKWJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVansb7J&md5=f11ccf80ba80a850682348ec75ddae19</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2460%26sid%3Dliteratum%253Aachs%26aulast%3DGuzman%26aufirst%3DM.%2BG.%26aulast%3DHalstead%26aufirst%3DS.%2BB.%26aulast%3DArtsob%26aufirst%3DH.%26aulast%3DBuchy%26aufirst%3DP.%26aulast%3DFarrar%26aufirst%3DJ.%26aulast%3DGubler%26aufirst%3DD.%2BJ.%26aulast%3DHunsperger%26aufirst%3DE.%26aulast%3DKroeger%26aufirst%3DA.%26aulast%3DMargolis%26aufirst%3DH.%2BS.%26aulast%3DMart%25C3%25ADnez%26aufirst%3DE.%26aulast%3DNathan%26aufirst%3DM.%2BB.%26aulast%3DPelegrino%26aufirst%3DJ.%2BL.%26aulast%3DSimmons%26aufirst%3DC.%26aulast%3DYoksan%26aufirst%3DS.%26aulast%3DPeeling%26aufirst%3DR.%2BW.%26atitle%3DDengue%253A%2520A%2520Continuing%2520Global%2520Threat%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2010%26volume%3D8%26spage%3DS7%26epage%3DS16%26doi%3D10.1038%2Fnrmicro2460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span> <i>World Health Organisation. GACVS Statement on Dengvaxia ® (CYD-TDV)</i>. <a href="https://www.who.int/vaccine_safety/committee/GACVS-StatementonDengvaxia-CYD-TDV/en/" class="extLink">https://www.who.int/vaccine_safety/committee/GACVS-StatementonDengvaxia-CYD-TDV/en/</a> (accessed Mar 31, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+World+Health+Organisation.+GACVS+Statement+on+Dengvaxia+%C2%AE+%28CYD-TDV%29.+https%3A%2F%2Fwww.who.int%2Fvaccine_safety%2Fcommittee%2FGACVS-StatementonDengvaxia-CYD-TDV%2Fen%2F+%28accessed+Mar+31%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DWorld%2520Health%2520Organisation.%2520GACVS%2520Statement%2520on%2520Dengvaxia%2520%25C2%25AE%2520%2528CYD-TDV%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span> <i>World Health Organisation.
Revised SAGE Recommendation on Use of
Dengue Vaccine</i>. <a href="https://www.who.int/immunization/diseases/dengue/revised_SAGE_recommendations_dengue_vaccines_apr2018/en/" class="extLink">https://www.who.int/immunization/diseases/dengue/revised_SAGE_recommendations_dengue_vaccines_apr2018/en/</a> (accessed Mar 31, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+World+Health+Organisation.%0ARevised+SAGE+Recommendation+on+Use+of%0ADengue+Vaccine.+https%3A%2F%2Fwww.who.int%2Fimmunization%2Fdiseases%2Fdengue%2Frevised_SAGE_recommendations_dengue_vaccines_apr2018%2Fen%2F+%28accessed+Mar+31%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DWorld%2520Health%2520Organisation.%250ARevised%2520SAGE%2520Recommendation%2520on%2520Use%2520of%250ADengue%2520Vaccine" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Normile, D.</span></span> <span> </span><span class="NLM_article-title">Safety Concerns Derail Dengue Vaccination Program</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1126/science.358.6370.1514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1126%2Fscience.358.6370.1514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=29269451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVGjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=1514-1515&author=D.+Normile&title=Safety+Concerns+Derail+Dengue+Vaccination+Program&doi=10.1126%2Fscience.358.6370.1514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Safety concerns derail dengue vaccination program</span></div><div class="casAuthors">Normile, Dennis</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6370</span>),
    <span class="NLM_cas:pages">1514-1515</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpRDA_BKZRLVg90H21EOLACvtfcHk0ljhxuRjBN1O7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVGjtQ%253D%253D&md5=a83e5957db20c0945ac434269c0b34b5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.358.6370.1514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.358.6370.1514%26sid%3Dliteratum%253Aachs%26aulast%3DNormile%26aufirst%3DD.%26atitle%3DSafety%2520Concerns%2520Derail%2520Dengue%2520Vaccination%2520Program%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3D1514%26epage%3D1515%26doi%3D10.1126%2Fscience.358.6370.1514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suthar, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, M.,  Jr.</span></span> <span> </span><span class="NLM_article-title">West Nile Virus Infection and Immunity</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2950</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fnrmicro2950" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=23321534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslGqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=115-128&author=M.+S.+Sutharauthor=M.+S.+Diamondauthor=M.+Gale&title=West+Nile+Virus+Infection+and+Immunity&doi=10.1038%2Fnrmicro2950"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">West Nile virus infection and immunity</span></div><div class="casAuthors">Suthar, Mehul S.; Diamond, Michael S.; Gale, Michael, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-128</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  West Nile virus (WNV) is an emerging neurotropic flavivirus that is transmitted to humans through the bite of an infected mosquito.  WNV has disseminated broadly in the Western hemisphere and now poses a significant public health risk.  The continuing spread of WNV, combined with the lack of specific therapeutics or vaccines to combat or prevent infection, imparts a pressing need to identify the viral and host processes that control the outcome of and immunity to WNV infection.  Here, we provide an overview of recent research that has revealed the virus-host interface controlling WNV infection and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzAOghp0axGbVg90H21EOLACvtfcHk0ljhxuRjBN1O7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslGqtA%253D%253D&md5=d6a1b96007511a54045764d79fdb22aa</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2950%26sid%3Dliteratum%253Aachs%26aulast%3DSuthar%26aufirst%3DM.%2BS.%26aulast%3DDiamond%26aufirst%3DM.%2BS.%26aulast%3DGale%26aufirst%3DM.%26atitle%3DWest%2520Nile%2520Virus%2520Infection%2520and%2520Immunity%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2013%26volume%3D11%26spage%3D115%26epage%3D128%26doi%3D10.1038%2Fnrmicro2950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCauley, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudd, M. T.</span></span> <span> </span><span class="NLM_article-title">Hepatitis C Virus NS3/4A Protease Inhibitors</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2016.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.coph.2016.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=27544488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ygurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=84-92&author=J.+A.+McCauleyauthor=M.+T.+Rudd&title=Hepatitis+C+Virus+NS3%2F4A+Protease+Inhibitors&doi=10.1016%2Fj.coph.2016.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis C virus NS3/4a protease inhibitors</span></div><div class="casAuthors">McCauley, John A.; Rudd, Michael T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">84-92</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) infection is a major health issue around the world and HCV NS3/4a protease inhibitors have been the focus of intensive research for the past 20 years.  From the first identification of substrate-derived peptide inhibitors to the complex, macrocyclic compds., including paritaprevir and grazoprevir, that are currently available, the field has used structure-based design to confront the issues of potency, resistance and pharmacokinetics.  Numerous breakthrough structures from a multitude of companies have led to compds. that are now key components of combination therapies with cure rates of >90%.  Herein, we detail the compds. that have advanced to clin. trials including their design and their impact on the NS3/4a protease field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecX2r0UguZLVg90H21EOLACvtfcHk0ljhxuRjBN1O7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ygurrJ&md5=1d7d626d8c10ab9d619e8caa14a3f9a5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2016.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2016.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DMcCauley%26aufirst%3DJ.%2BA.%26aulast%3DRudd%26aufirst%3DM.%2BT.%26atitle%3DHepatitis%2520C%2520Virus%2520NS3%252F4A%2520Protease%2520Inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2016%26volume%3D30%26spage%3D84%26epage%3D92%26doi%3D10.1016%2Fj.coph.2016.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span> <span> </span><span class="NLM_article-title">Anti-HIV Drugs: 25 Compounds Approved within 25 Years after the Discovery of HIV</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">320</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2008.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.ijantimicag.2008.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19108994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=307-320&author=E.+De%0AClercq&title=Anti-HIV+Drugs%3A+25+Compounds+Approved+within+25+Years+after+the+Discovery+of+HIV&doi=10.1016%2Fj.ijantimicag.2008.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV drugs: 25 compounds approved within 25 years after the discovery of HIV</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-320</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In 2008, 25 years after the human immunodeficiency virus (HIV) was discovered as the then tentative etiol. agent of acquired immune deficiency syndrome (AIDS), exactly 25 anti-HIV compds. have been formally approved for clin. use in the treatment of AIDS.  These compds. fall into six categories: nucleoside reverse transcriptase inhibitors (NRTIs: zidovudine, didanosine, zalcitabine, stavudine, lamivudine, abacavir and emtricitabine); nucleotide reverse transcriptase inhibitors (NtRTIs: tenofovir); non-nucleoside reverse transcriptase inhibitors (NNRTIs: nevirapine, delavirdine, efavirenz and etravirine); protease inhibitors (PIs: saquinavir, ritonavir, indinavir, nelfinavir, amprenavir, lopinavir, atazanavir, fosamprenavir, tipranavir and darunavir); cell entry inhibitors [fusion inhibitors (FIs: enfuvirtide) and co-receptor inhibitors (CRIs: maraviroc)]; and integrase inhibitors (INIs: raltegravir).  These compds. should be used in drug combination regimens to achieve the highest possible benefit, tolerability and compliance and to diminish the risk of resistance development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsT6dqAjvKIrVg90H21EOLACvtfcHk0ljhxuRjBN1O7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D&md5=6d8263455e9c912d353ef2ea230a5e15</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2008.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2008.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DAnti-HIV%2520Drugs%253A%252025%2520Compounds%2520Approved%2520within%252025%2520Years%2520after%2520the%2520Discovery%2520of%2520HIV%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2009%26volume%3D33%26spage%3D307%26epage%3D320%26doi%3D10.1016%2Fj.ijantimicag.2008.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.-Y.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of Dengue Virus Enzymes: Towards Rational Design of Therapeutics</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.antiviral.2012.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=22995600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12gtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2012&pages=115-126&author=C.+G.+Nobleauthor=P.-Y.+Shi&title=Structural+Biology+of+Dengue+Virus+Enzymes%3A+Towards+Rational+Design+of+Therapeutics&doi=10.1016%2Fj.antiviral.2012.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of dengue virus enzymes: Towards rational design of therapeutics</span></div><div class="casAuthors">Noble, Christian G.; Shi, Pei-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-126</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Development of anti-dengue therapy represents an urgent un-met medical need.  Towards antiviral therapy, recent advances in crystal structures of DENV enzymes have led to the possibility of structure-based rational design of inhibitors for anti-dengue therapy.  These include (i) the structure of the active' form of the DENV protease in complex with a peptide substrate; (ii) the structure of DENV methyltransferase bound to an inhibitor that selectively suppresses viral methyltransferase, but not human methyltransferases; (iii) the structure of DENV RNA-dependent RNA polymerase in complex with a small-mol. compd.  This review summarizes the structural biol. of these three key enzymes (protease, methyltransferase, and polymerase) that are essential for DENV replication.  The new structural information has provided new avenues for development of anti-dengue therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG75WRQ0Bn7LVg90H21EOLACvtfcHk0liKbCp2eGCPtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12gtLfF&md5=42f5387c37a813b1cdf9737ccc213d54</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DC.%2BG.%26aulast%3DShi%26aufirst%3DP.-Y.%26atitle%3DStructural%2520Biology%2520of%2520Dengue%2520Virus%2520Enzymes%253A%2520Towards%2520Rational%2520Design%2520of%2520Therapeutics%26jtitle%3DAntiviral%2520Res.%26date%3D2012%26volume%3D96%26spage%3D115%26epage%3D126%26doi%3D10.1016%2Fj.antiviral.2012.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Biochemistry and Medicinal Chemistry of the Dengue Virus Protease</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">11348</span>– <span class="NLM_lpage">11381</span>, <span class="refDoi"> DOI: 10.1021/cr500233q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500233q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Cns77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=11348-11381&author=C.+Nitscheauthor=S.+Hollowayauthor=T.+Schirmeisterauthor=C.+D.+Klein&title=Biochemistry+and+Medicinal+Chemistry+of+the+Dengue+Virus+Protease&doi=10.1021%2Fcr500233q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemistry and Medicinal Chemistry of the Dengue Virus Protease</span></div><div class="casAuthors">Nitsche, Christoph; Holloway, Steven; Schirmeister, Tanja; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">11348-11381</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the present work, we will review the protease of dengue virus from several directions, with a particular focus on those aspects related to medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVn6Afm5NC07Vg90H21EOLACvtfcHk0liKbCp2eGCPtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Cns77P&md5=5f4764c07d55dd760f95305019ddf84c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcr500233q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500233q%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DHolloway%26aufirst%3DS.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DBiochemistry%2520and%2520Medicinal%2520Chemistry%2520of%2520the%2520Dengue%2520Virus%2520Protease%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D11348%26epage%3D11381%26doi%3D10.1021%2Fcr500233q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumanut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tully, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiricek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priestle, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, S. G.</span></span> <span> </span><span class="NLM_article-title">Functional Profiling of Recombinant NS3 Proteases from All Four Serotypes of Dengue Virus Using Tetrapeptide and Octapeptide Substrate Libraries</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">28766</span>– <span class="NLM_lpage">28774</span>, <span class="refDoi"> DOI: 10.1074/jbc.M500588200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.M500588200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=15932883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvFSnt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=28766-28774&author=J.+Liauthor=S.+P.+Limauthor=D.+Beerauthor=V.+Patelauthor=D.+Wenauthor=C.+Tumanutauthor=D.+C.+Tullyauthor=J.+A.+Williamsauthor=J.+Jiricekauthor=J.+P.+Priestleauthor=J.+L.+Harrisauthor=S.+G.+Vasudevan&title=Functional+Profiling+of+Recombinant+NS3+Proteases+from+All+Four+Serotypes+of+Dengue+Virus+Using+Tetrapeptide+and+Octapeptide+Substrate+Libraries&doi=10.1074%2Fjbc.M500588200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Profiling of Recombinant NS3 Proteases from All Four Serotypes of Dengue Virus Using Tetrapeptide and Octapeptide Substrate Libraries</span></div><div class="casAuthors">Li, Jun; Lim, Siew Pheng; Beer, David; Patel, Viral; Wen, Daying; Tumanut, Christine; Tully, David C.; Williams, Jennifer A.; Jiricek, Jan; Priestle, John P.; Harris, Jennifer L.; Vasudevan, Subhash G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">28766-28774</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Regulated proteolysis by the two-component NS2B/NS3 protease of dengue virus is essential for virus replication and the maturation of infectious virions.  The functional similarity between the NS2B/NS3 proteases from the four genetically and antigenically distinct serotypes was addressed by characterizing the differences in their substrate specificity using tetrapeptide and octapeptide libraries in a positional scanning format, each contg. 130,321 substrates.  The proteases from different serotypes were shown to be functionally homologous based on the similarity of their substrate cleavage preferences.  A strong preference for basic amino acid residues (Arg/Lys) at the P1 positions was obsd., whereas the preferences for the P2-4 sites were in the order of Arg > Thr > Gln/Asn/Lys for P2, Lys > Arg > Asn for P3, and Nle > Leu > Lys > Xaa for P4.  The prime site substrate specificity was for small and polar amino acids in P1' and P3'.  In contrast, the P2' and P4' substrate positions showed minimal activity.  The influence of the P2 and P3 amino acids on ground state binding and the P4 position for transition state stabilization was identified through single substrate kinetics with optimal and suboptimal substrate sequences.  The specificities obsd. for dengue NS2B/NS3 have features in common with the physiol. cleavage sites in the dengue polyprotein; however, all sites reveal previously unrecognized suboptimal sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH2hcNat1IwrVg90H21EOLACvtfcHk0liKbCp2eGCPtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvFSnt7w%253D&md5=7e7bd5ab26af480759378090e7e73e66</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M500588200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M500588200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLim%26aufirst%3DS.%2BP.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DTumanut%26aufirst%3DC.%26aulast%3DTully%26aufirst%3DD.%2BC.%26aulast%3DWilliams%26aufirst%3DJ.%2BA.%26aulast%3DJiricek%26aufirst%3DJ.%26aulast%3DPriestle%26aufirst%3DJ.%2BP.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DVasudevan%26aufirst%3DS.%2BG.%26atitle%3DFunctional%2520Profiling%2520of%2520Recombinant%2520NS3%2520Proteases%2520from%2520All%2520Four%2520Serotypes%2520of%2520Dengue%2520Virus%2520Using%2520Tetrapeptide%2520and%2520Octapeptide%2520Substrate%2520Libraries%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D28766%26epage%3D28774%26doi%3D10.1074%2Fjbc.M500588200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeyaraj, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngew, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span> <span> </span><span class="NLM_article-title">Peptide Inhibitors of Dengue Virus NS3 Protease. Part 1: Warhead</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.09.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmcl.2005.09.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=16246553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Citr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=36-39&author=Z.+Yinauthor=S.+J.+Patelauthor=W.-L.+Wangauthor=G.+Wangauthor=W.-L.+Chanauthor=K.+R.+R.+Raoauthor=J.+Alamauthor=D.+A.+Jeyarajauthor=X.+Ngewauthor=V.+Patelauthor=D.+Beerauthor=S.+P.+Limauthor=S.+G.+Vasudevanauthor=T.+H.+Keller&title=Peptide+Inhibitors+of+Dengue+Virus+NS3+Protease.+Part+1%3A+Warhead&doi=10.1016%2Fj.bmcl.2005.09.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide inhibitors of dengue virus NS3 protease. Part 1: Warhead</span></div><div class="casAuthors">Yin, Zheng; Patel, Sejal J.; Wang, Wei-Ling; Wang, Gang; Chan, Wai-Ling; Rao, K. R. Ranga; Alam, Jenefer; Jeyaraj, Duraiswamy A.; Ngew, Xinyi; Patel, Viral; Beer, David; Lim, Siew Pheng; Vasudevan, Subhash G.; Keller, Thomas H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-39</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Substrate-based tetrapeptide inhibitors with various warheads were designed, synthesized, and evaluated against the Dengue virus NS3 protease.  Effective inhibition was achieved by peptide inhibitors with electrophilic warheads such as aldehyde, trifluoromethyl ketone, and boronic acid.  Boronic acid Bz-Nle-Lys-Arg-Arg-B(OH)2 has the highest affinity, exhibiting a Ki of 43 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZaCOf0CvDFrVg90H21EOLACvtfcHk0ljFLKOHOJQbbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Citr3M&md5=1ac1823a7945aa3d9c7e6edca23ed9f0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.09.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.09.062%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DPatel%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DW.-L.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DChan%26aufirst%3DW.-L.%26aulast%3DRao%26aufirst%3DK.%2BR.%2BR.%26aulast%3DAlam%26aufirst%3DJ.%26aulast%3DJeyaraj%26aufirst%3DD.%2BA.%26aulast%3DNgew%26aufirst%3DX.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%2BP.%26aulast%3DVasudevan%26aufirst%3DS.%2BG.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26atitle%3DPeptide%2520Inhibitors%2520of%2520Dengue%2520Virus%2520NS3%2520Protease.%2520Part%25201%253A%2520Warhead%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D36%26epage%3D39%26doi%3D10.1016%2Fj.bmcl.2005.09.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigel, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Peptide–Boronic Acid Inhibitors of Flaviviral Proteases: Medicinal Chemistry and Structural Biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01021</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01021" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGrtLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=511-516&author=C.+Nitscheauthor=L.+Zhangauthor=L.+F.+Weigelauthor=J.+Schilzauthor=D.+Grafauthor=R.+Bartenschlagerauthor=R.+Hilgenfeldauthor=C.+D.+Klein&title=Peptide%E2%80%93Boronic+Acid+Inhibitors+of+Flaviviral+Proteases%3A+Medicinal+Chemistry+and+Structural+Biology&doi=10.1021%2Facs.jmedchem.6b01021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide-Boronic Acid Inhibitors of Flaviviral Proteases: Medicinal Chemistry and Structural Biology</span></div><div class="casAuthors">Nitsche, Christoph; Zhang, Linlin; Weigel, Lena F.; Schilz, Jonas; Graf, Dominik; Bartenschlager, Ralf; Hilgenfeld, Rolf; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">511-516</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A thousand-fold affinity gain is achieved by introduction of a C-terminal boronic acid moiety into dipeptidic inhibitors of the Zika, West Nile and dengue virus proteases.  The resulting compds. have Ki values in the two-digit nanomolar range, are not cytotoxic, and inhibit virus replication.  Structure-activity relationships and a high resoln. x-ray co-crystal structure with West Nile virus protease provide a basis for the design of optimized covalent-reversible inhibitors aimed at emerging flaviviral pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT1SPIPsFvFLVg90H21EOLACvtfcHk0ljFLKOHOJQbbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGrtLvE&md5=a9671140deb2c9014ae8e1429cb5074f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01021%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWeigel%26aufirst%3DL.%2BF.%26aulast%3DSchilz%26aufirst%3DJ.%26aulast%3DGraf%26aufirst%3DD.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DPeptide%25E2%2580%2593Boronic%2520Acid%2520Inhibitors%2520of%2520Flaviviral%2520Proteases%253A%2520Medicinal%2520Chemistry%2520and%2520Structural%2520Biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D511%26epage%3D516%26doi%3D10.1021%2Facs.jmedchem.6b01021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotos, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of Nanomolar Dengue and West Nile Virus Protease Inhibitors Containing a 4-Benzyloxyphenylglycine Residue</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9354</span>– <span class="NLM_lpage">9370</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01441</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01441" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9354-9370&author=M.+A.+M.+Behnamauthor=D.+Grafauthor=R.+Bartenschlagerauthor=D.+P.+Zlotosauthor=C.+D.+Klein&title=Discovery+of+Nanomolar+Dengue+and+West+Nile+Virus+Protease+Inhibitors+Containing+a+4-Benzyloxyphenylglycine+Residue&doi=10.1021%2Facs.jmedchem.5b01441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Nanomolar Dengue and West Nile Virus Protease Inhibitors Containing a 4-Benzyloxyphenylglycine Residue</span></div><div class="casAuthors">Behnam, Mira A. M.; Graf, Dominik; Bartenschlager, Ralf; Zlotos, Darius P.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9354-9370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dengue virus (DENV) and West Nile Virus (WNV) NS2B-NS3 proteases are attractive targets for the development of dual-acting therapeutics against these arboviral pathogens.  We present the synthesis and extensive biol. evaluation of inhibitors that contain benzyl ethers of 4-hydroxyphenylglycine as non-natural peptidic building blocks synthesized via a copper-complex intermediate.  A three-step optimization strategy, beginning with fragment growth of the C-terminal 4-hydroxyphenylglycine to the benzyloxy ether, followed by C- and N-terminal optimization, and finally fragment merging generated compds. with in vitro affinities in the low nanomolar range.  The most promising deriv. reached Ki values of 12 nM at the DENV-2 and 39 nM at the WNV proteases.  Several of the newly discovered protease inhibitors yielded a significant redn. of dengue and West Nile virus titers in cell-based assays of virus replication, with an EC50 value of 3.4 μM at DENV-2 and 15.5 μM at WNV for the most active analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZA17Sr9XobVg90H21EOLACvtfcHk0liZFmvjGNWbZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7nK&md5=056ebd78d73cce7686a80752c69d3791</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01441%26sid%3Dliteratum%253Aachs%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DGraf%26aufirst%3DD.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DZlotos%26aufirst%3DD.%2BP.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DDiscovery%2520of%2520Nanomolar%2520Dengue%2520and%2520West%2520Nile%2520Virus%2520Protease%2520Inhibitors%2520Containing%2520a%25204-Benzyloxyphenylglycine%2520Residue%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9354%26epage%3D9370%26doi%3D10.1021%2Facs.jmedchem.5b01441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bastos
Lima, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Sherif, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vechi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Dual Inhibitors of the Dengue and West Nile Virus NS2B–NS3 Proteases: Synthesis, Biological Evaluation and Docking Studies of Novel Peptide-Hybrids</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5748</span>– <span class="NLM_lpage">5755</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmc.2015.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=26233795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2ks7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=5748-5755&author=A.+Bastos%0ALimaauthor=M.+A.+M.+Behnamauthor=Y.+El+Sherifauthor=C.+Nitscheauthor=S.+M.+Vechiauthor=C.+D.+Klein&title=Dual+Inhibitors+of+the+Dengue+and+West+Nile+Virus+NS2B%E2%80%93NS3+Proteases%3A+Synthesis%2C+Biological+Evaluation+and+Docking+Studies+of+Novel+Peptide-Hybrids&doi=10.1016%2Fj.bmc.2015.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Dual inhibitors of the dengue and West Nile virus NS2B-NS3 proteases: Synthesis, biological evaluation and docking studies of novel peptide-hybrids</span></div><div class="casAuthors">Bastos Lima, Allan; Behnam, Mira A. M.; El Sherif, Yasmin; Nitsche, Christoph; Vechi, Sergio M.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5748-5755</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dengue virus (DENV) and West Nile virus (WNV) are mosquito-borne arboviruses responsible for causing acute systemic diseases and severe health conditions in humans.  The discovery of therapies capable to prevent infections or treat infected individuals remains an important challenge, since no vaccine or specific efficient treatment could be developed so far.  In this context, we present herein the synthesis, characterization, biol. evaluation and docking studies of novel peptide-hybrids based on 2,4-thiazolidinedione scaffolds contg. non-polar groups.  The most promising compd. has an IC50 of 0.75 μM against WNV protease, which represents a seventyfold improvement in activity compared to our previously reported compds.  Exptl. results and docking studies are in agreement with the hypothesis that a non-polar group in the scaffold is important to obtain interactions between the inhibitors and a hydrophobic pocket in the substrate recognition region of the DENV and WNV NS2B-NS3 serine proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxjuAszpkdcrVg90H21EOLACvtfcHk0liZFmvjGNWbZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2ks7nI&md5=69934d8829047f4e0f7a4951969f7e25</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DBastos%2BLima%26aufirst%3DA.%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DEl%2BSherif%26aufirst%3DY.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DVechi%26aufirst%3DS.%2BM.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DDual%2520Inhibitors%2520of%2520the%2520Dengue%2520and%2520West%2520Nile%2520Virus%2520NS2B%25E2%2580%2593NS3%2520Proteases%253A%2520Synthesis%252C%2520Biological%2520Evaluation%2520and%2520Docking%2520Studies%2520of%2520Novel%2520Peptide-Hybrids%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D5748%26epage%3D5755%26doi%3D10.1016%2Fj.bmc.2015.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and Computational Approaches to Estimate Solubility and Permeability in Drug Discovery and Development Settings</span>. <i>Adv. Drug Delivery Rev</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+Computational+Approaches+to+Estimate+Solubility+and+Permeability+in+Drug+Discovery+and+Development+Settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520Computational%2520Approaches%2520to%2520Estimate%2520Solubility%2520and%2520Permeability%2520in%2520Drug%2520Discovery%2520and%2520Development%2520Settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev%26date%3D1997%26volume%3D46%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rayan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldblum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Understanding Drug-Likeness</span>. <i>Wiley Interdiscip. Rev.: Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1002/wcms.52</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fwcms.52" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFWrtrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=760-781&author=O.+Ursuauthor=A.+Rayanauthor=A.+Goldblumauthor=T.+I.+Oprea&title=Understanding+Drug-Likeness&doi=10.1002%2Fwcms.52"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding drug-likeness</span></div><div class="casAuthors">Ursu, Oleg; Rayan, Anwar; Goldblum, Amiram; Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: Computational Molecular Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">760-781</span>CODEN:
                <span class="NLM_cas:coden">WIRCAH</span>;
        ISSN:<span class="NLM_cas:issn">1759-0884</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  "Drug-likeness", a qual. property of chems. assigned by experts committee vote, is wedely integrated into the early stages of lead and drug discovery.  Its conceptual evolution paralleled work related to Pfizer's "rule of five" and lead-likeness, and is placed within this framework.  The discrimination between "drugs" (represented by a collection of pharmaceutically relevant small mols., some of which are marketed drugs) and "nondrugs" (typically, chem. reagents) is possible using a wide variety of statistical tools and chem. descriptor systems.  Here we summarize 18 papers focused on drug-likeness, and provide a comprehensive overview of progress in the field.  Tools that est. drug-likeness are valuable in the early stages of lead discovery, and can be used to filter out compds. with undersirable properties from screening libraries and to prioritize hits from primary screens.  As the goal is, most often, to develop orally available drugs, it is also useful to optimize drug-like pharmacokinetic properties.  We examine tools that evaluate drug-likeness and some of their shortcomings, challenges facing these tools, and address the following issues: What is the definition of drug-likeness and how can it be utilized to reduce attrition rate in drug discovery.  How difficult is it to distinguish drugs from nondrugs.  Are nondrug datasets reliable.  Can we est. oral drug-likeness.  We discuss a drug-like filter and recent advances in the prediction of oral drug-likeness.  The heuristic aspect of drug-likeness is also addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeTi02F5W647Vg90H21EOLACvtfcHk0liZFmvjGNWbZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFWrtrbO&md5=3cd46a7a7a082d6d3730e6bbf16712d2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fwcms.52&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.52%26sid%3Dliteratum%253Aachs%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DRayan%26aufirst%3DA.%26aulast%3DGoldblum%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DUnderstanding%2520Drug-Likeness%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%253A%2520Comput.%2520Mol.%2520Sci.%26date%3D2011%26volume%3D1%26spage%3D760%26epage%3D781%26doi%3D10.1002%2Fwcms.52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vechi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">C-Terminal Residue Optimization and Fragment Merging: Discovery of a Potent Peptide-Hybrid Inhibitor of Dengue Protease</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1037</span>– <span class="NLM_lpage">1042</span>, <span class="refDoi"> DOI: 10.1021/ml500245v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500245v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1037-1042&author=M.+A.+M.+Behnamauthor=C.+Nitscheauthor=S.+M.+Vechiauthor=C.+D.+Klein&title=C-Terminal+Residue+Optimization+and+Fragment+Merging%3A+Discovery+of+a+Potent+Peptide-Hybrid+Inhibitor+of+Dengue+Protease&doi=10.1021%2Fml500245v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">C-Terminal Residue Optimization and Fragment Merging: Discovery of a Potent Peptide-Hybrid Inhibitor of Dengue Protease</span></div><div class="casAuthors">Behnam, Mira A. M.; Nitsche, Christoph; Vechi, Sergio M.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1037-1042</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dengue virus protease is a promising target for the development of antiviral drugs.  We describe here a two-step rational optimization that led to the discovery of the potent inhibitor 35 with nanomolar binding affinity at dengue protease serotype 2 (IC50 = 0.6 μM, Ki = 0.4 μM).  First, a large no. of natural and non-natural amino acids were screened at the C-terminal position of the previously reported, canonical peptide sequence (Cap-Arg-Lys-Nle-NH2).  Compared to the ref. compd. 1 (Bz-Arg-Lys-Nle-NH2, IC50 = 13.3 μM), a 4-fold higher inhibitory potential was obsd. with the incorporation of a C-terminal phenylglycine (compd. 9, IC50 = 3.3 μM).  Second, we applied fragment merging of 9 with the previously reported thiazolidinedione peptide hybrid 33 (IC50 = 2.5 μM).  This approach led to the fusion of two inhibitor-fragments with micromolar affinity into a 20-fold more potent, competitive inhibitor of dengue protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru7uUNG4dKDbVg90H21EOLACvtfcHk0ljvIgi13DOetg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbrI&md5=ff1a6a4a5c8b5427dd5d6555a45976ed</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fml500245v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500245v%26sid%3Dliteratum%253Aachs%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DVechi%26aufirst%3DS.%2BM.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DC-Terminal%2520Residue%2520Optimization%2520and%2520Fragment%2520Merging%253A%2520Discovery%2520of%2520a%2520Potent%2520Peptide-Hybrid%2520Inhibitor%2520of%2520Dengue%2520Protease%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1037%26epage%3D1042%26doi%3D10.1021%2Fml500245v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuiper, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellmon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medapureddy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muzzarelli, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amblard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovari, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinazi, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovari, L. C.</span></span> <span> </span><span class="NLM_article-title">Increased Activity of Unlinked Zika Virus NS2B/NS3 Protease Compared to Linked Zika Virus Protease</span>. <i>Biochem. Biophys. Res. CommunBiochemical and Biophysical Research Communications</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">673</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2017.03.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bbrc.2017.03.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=28342876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVajsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2017&pages=668-673&author=B.+D.+Kuiperauthor=K.+Slaterauthor=N.+Spellmonauthor=J.+Holcombauthor=P.+Medapureddyauthor=K.+M.+Muzzarelliauthor=Z.+Yangauthor=R.+Ovadiaauthor=F.+Amblardauthor=I.+A.+Kovariauthor=R.+F.+Schinaziauthor=L.+C.+Kovari&title=Increased+Activity+of+Unlinked+Zika+Virus+NS2B%2FNS3+Protease+Compared+to+Linked+Zika+Virus+Protease&doi=10.1016%2Fj.bbrc.2017.03.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Increased activity of unlinked Zika virus NS2B/NS3 protease compared to linked Zika virus protease</span></div><div class="casAuthors">Kuiper, Benjamin D.; Slater, Kristin; Spellmon, Nicholas; Holcomb, Joshua; Medapureddy, Prasanna; Muzzarelli, Kendall M.; Yang, Zhe; Ovadia, Reuben; Amblard, Franck; Kovari, Iulia A.; Schinazi, Raymond F.; Kovari, Ladislau C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">668-673</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Zika virus (ZIKV) is a flavivirus spread by daytime-active Aedes spp. mosquitoes such as A. aegypti and A. albopictus.  Previously thought to be a mild infection, the latest ZIKV outbreak in the Americas is causally assocd. with more severe symptoms as well as severe birth defects, such as microcephaly.  Currently no vaccine or antiviral exists.  However, recent progress has demonstrated the viral NS2B/NS3 protease may be a suitable target for the development of small-mol. antiviral agents.  To better understand the ZIKV protease, we expressed, purified, and characterized unlinked and linked NS2B/NS3 protease corresponding to an isolate from the recent outbreak in Puerto Rico.  Unlinked ZIKV protease is more active and binds substrate with greater affinity than linked ZIKV protease.  Therefore, we propose that unlinked ZIKV protease be used when evaluating or designing ZIKV protease inhibitors.  Addnl., potent inhibitors of related viral proteases, like West Nile Virus and Dengue virus, may serve as advanced starting points to identify and develop ZIKV protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS3UjrJ1aaW7Vg90H21EOLACvtfcHk0ljvIgi13DOetg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVajsr4%253D&md5=09a78e1627358550764961eb7d93ed26</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.03.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.03.108%26sid%3Dliteratum%253Aachs%26aulast%3DKuiper%26aufirst%3DB.%2BD.%26aulast%3DSlater%26aufirst%3DK.%26aulast%3DSpellmon%26aufirst%3DN.%26aulast%3DHolcomb%26aufirst%3DJ.%26aulast%3DMedapureddy%26aufirst%3DP.%26aulast%3DMuzzarelli%26aufirst%3DK.%2BM.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DOvadia%26aufirst%3DR.%26aulast%3DAmblard%26aufirst%3DF.%26aulast%3DKovari%26aufirst%3DI.%2BA.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26aulast%3DKovari%26aufirst%3DL.%2BC.%26atitle%3DIncreased%2520Activity%2520of%2520Unlinked%2520Zika%2520Virus%2520NS2B%252FNS3%2520Protease%2520Compared%2520to%2520Linked%2520Zika%2520Virus%2520Protease%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520CommunBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26date%3D2017%26volume%3D492%26spage%3D668%26epage%3D673%26doi%3D10.1016%2Fj.bbrc.2017.03.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wünsch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fries, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwick, A.</span></span> <span> </span><span class="NLM_article-title">Beiträge zur Peptidsynthese, V. Darstellung und Peptidsynthetische Verwendung von O-Benzyl-L-Tyrosin</span>. <i>Chem. Ber.</i> <span class="NLM_year" style="font-weight: bold;">1958</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1002/cber.19580910312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fcber.19580910312" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1958&pages=542-547&author=E.+W%C3%BCnschauthor=G.+Friesauthor=A.+Zwick&title=Beitr%C3%A4ge+zur+Peptidsynthese%2C+V.+Darstellung+und+Peptidsynthetische+Verwendung+von+O-Benzyl-L-Tyrosin&doi=10.1002%2Fcber.19580910312"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcber.19580910312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.19580910312%26sid%3Dliteratum%253Aachs%26aulast%3DW%25C3%25BCnsch%26aufirst%3DE.%26aulast%3DFries%26aufirst%3DG.%26aulast%3DZwick%26aufirst%3DA.%26atitle%3DBeitr%25C3%25A4ge%2520zur%2520Peptidsynthese%252C%2520V.%2520Darstellung%2520und%2520Peptidsynthetische%2520Verwendung%2520von%2520O-Benzyl-L-Tyrosin%26jtitle%3DChem.%2520Ber.%26date%3D1958%26volume%3D91%26spage%3D542%26epage%3D547%26doi%3D10.1002%2Fcber.19580910312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dener, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantauzzi, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kshirsagar, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, A. B.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Syntheses of Fmoc-Protected Non-Proteogenic Amino Acids: Useful Building Blocks for Combinatorial Libraries</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1021/op010204f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/op010204f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFOhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=445-449&author=J.+M.+Denerauthor=P.+P.+Fantauzziauthor=T.+A.+Kshirsagarauthor=D.+E.+Kellyauthor=A.+B.+Wolfe&title=Large-Scale+Syntheses+of+Fmoc-Protected+Non-Proteogenic+Amino+Acids%3A+Useful+Building+Blocks+for+Combinatorial+Libraries&doi=10.1021%2Fop010204f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale syntheses of Fmoc-protected non-proteogenic amino acids: useful building blocks for combinatorial libraries</span></div><div class="casAuthors">Dener, Jeffrey M.; Fantauzzi, Pascal P.; Kshirsagar, Tushar A.; Kelly, Daphne E.; Wolfe, Aaron B.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">445-449</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Convenient and reliable large-scale procedures for the protection of various amino acids with N-(9-fluorenylmethoxycarbonyl)oxysuccinimide (Fmoc-OSu) are described.  Com. available 4-aminomethylbenzoic acid and trans-4-(aminomethyl)cyclohexanecarboxylic acid were converted into their corresponding Fmoc-derivs. in excellent yields without the need for an extractive workup.  In addn., Fmoc-cis-β-amino acids were also prepd., employing a [2 + 2]-cycloaddn. strategy between a cyclic olefin and N-chlorosulfonyl isocyanate (CSI).  The resulting N-chlorosulfonyl β-lactams were reduced to the parent β-lactams with sodium sulfite and then converted to the cis-β-amino acid hydrochlorides by exposure to aq. hydrochloric acid.  The resulting cis-β-amino acids were converted to their Fmoc-derivs. under conditions similar to those developed for the com. available amino acids.  Differences in the conditions employed between these β-amino acids and the com. derivs. were obsd., primarily in the nature of the base required for the reaction.  A possible rationale for the differences in behavior is described.  These Fmoc-amino acid derivs. are valuable intermediates for the solid-phase synthesis of combinatorial libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr31UZPt5I5-LVg90H21EOLACvtfcHk0lgyRcZiXsfS6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFOhu7w%253D&md5=8d76a0c09d27d3eac2ff1a29217da4da</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fop010204f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fop010204f%26sid%3Dliteratum%253Aachs%26aulast%3DDener%26aufirst%3DJ.%2BM.%26aulast%3DFantauzzi%26aufirst%3DP.%2BP.%26aulast%3DKshirsagar%26aufirst%3DT.%2BA.%26aulast%3DKelly%26aufirst%3DD.%2BE.%26aulast%3DWolfe%26aufirst%3DA.%2BB.%26atitle%3DLarge-Scale%2520Syntheses%2520of%2520Fmoc-Protected%2520Non-Proteogenic%2520Amino%2520Acids%253A%2520Useful%2520Building%2520Blocks%2520for%2520Combinatorial%2520Libraries%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2001%26volume%3D5%26spage%3D445%26epage%3D449%26doi%3D10.1021%2Fop010204f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundermann, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Phenylglycine Racemization in Fmoc-Based Solid-Phase Peptide Synthesis: Stereochemical Stability Is Achieved by Choice of Reaction Conditions</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2325</span>– <span class="NLM_lpage">2329</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2017.04.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.tetlet.2017.04.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2017&pages=2325-2329&author=C.+Liangauthor=M.+A.+M.+Behnamauthor=T.+R.+Sundermannauthor=C.+D.+Klein&title=Phenylglycine+Racemization+in+Fmoc-Based+Solid-Phase+Peptide+Synthesis%3A+Stereochemical+Stability+Is+Achieved+by+Choice+of+Reaction+Conditions&doi=10.1016%2Fj.tetlet.2017.04.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylglycine racemization in Fmoc-based solid-phase peptide synthesis: Stereochemical stability is achieved by choice of reaction conditions</span></div><div class="casAuthors">Liang, Chen; Behnam, Mira A. M.; Sundermann, Tom R.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2325-2329</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Phenylglycine-contg. peptides have broad applications in medicinal chem., but their synthetic accessibility is complicated by the risk of epimerization during solid-phase peptide synthesis (SPPS).  Phenylglycine is therefore often considered a troublesome residue.  This work studies the extent of Phg racemization under different Fmoc-SPPS reaction conditions.  It is shown that the base-catalyzed coupling of Fmoc-Phg is the crit. step for racemization.  However, racemization can be reduced to a negligible level if DEPBT or COMU combined with TMP or DMP are employed during this step.  Resin-bound peptides are remarkably resistant against epimerization during extended incubation under basic conditions and the free peptides were stable in buffer solns. used for biol. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaQeBw87P6bVg90H21EOLACvtfcHk0lgyRcZiXsfS6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKhsbw%253D&md5=d6ad6038198c08fcf1484d350f2dbd16</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2017.04.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2017.04.047%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DSundermann%26aufirst%3DT.%2BR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DPhenylglycine%2520Racemization%2520in%2520Fmoc-Based%2520Solid-Phase%2520Peptide%2520Synthesis%253A%2520Stereochemical%2520Stability%2520Is%2520Achieved%2520by%2520Choice%2520of%2520Reaction%2520Conditions%26jtitle%3DTetrahedron%2520Lett.%26date%3D2017%26volume%3D58%26spage%3D2325%26epage%3D2329%26doi%3D10.1016%2Fj.tetlet.2017.04.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernatowicz, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsueda, G. R.</span></span> <span> </span><span class="NLM_article-title">1H-Pyrazole-1-Carboxamidine Hydrochloride an Attractive Reagent for Guanylation of Amines and Its Application to Peptide Synthesis</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2497</span>– <span class="NLM_lpage">2502</span>, <span class="refDoi"> DOI: 10.1021/jo00034a059</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00034a059" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK38XhslCksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1992&pages=2497-2502&author=M.+S.+Bernatowiczauthor=Y.+Wuauthor=G.+R.+Matsueda&title=1H-Pyrazole-1-Carboxamidine+Hydrochloride+an+Attractive+Reagent+for+Guanylation+of+Amines+and+Its+Application+to+Peptide+Synthesis&doi=10.1021%2Fjo00034a059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">1H-Pyrazole-1-carboxamidine hydrochloride an attractive reagent for guanylation of amines and its application to peptide synthesis</span></div><div class="casAuthors">Bernatowicz, Michael S.; Wu, Youling; Matsueda, Gary R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2497-502</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">1H-Pyrazole-1-carboxamidine hydrochloride (I) was conveniently prepd. in 95% yield by reaction of equimolar amts. of pyrazole and cyanamide in refluxing HCl/p-dioxane.  Guanylation of sterically unhindered primary and secondary aliph. amines (e.g. cyclohexylamine, ethanolamine, piperidine, glycine) in DMF at room temp. for 4-8 h gave 71-88% isolated yields of guanidines.  Arom. amines (e.g. aniline, 4-methoxyaniline) required longer reaction times or more forcing conditions.  Other potentially nucleophilic functionalities (hydroxyl, carboxyl, thiol, imidazole) failed to react with I.  The successful prepn. of H-Gly-His-Arg-Pro-Leu-Asp-Lys-NH2 (N-terminal heptapeptide from the β-chain of human fibrin) by guanylation of an appropriately protected Orn3-peptidyl-polystyrene resin precursor demonstrated the feasibility of using I as a guanylating reagent in an Orn → Arg strategy of solid phase peptide synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyfr8dJ0yWzLVg90H21EOLACvtfcHk0lgyRcZiXsfS6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhslCksbw%253D&md5=e48ccfec15ddfe89575b20e18a3cff4b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjo00034a059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00034a059%26sid%3Dliteratum%253Aachs%26aulast%3DBernatowicz%26aufirst%3DM.%2BS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DMatsueda%26aufirst%3DG.%2BR.%26atitle%3D1H-Pyrazole-1-Carboxamidine%2520Hydrochloride%2520an%2520Attractive%2520Reagent%2520for%2520Guanylation%2520of%2520Amines%2520and%2520Its%2520Application%2520to%2520Peptide%2520Synthesis%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1992%26volume%3D57%26spage%3D2497%26epage%3D2502%26doi%3D10.1021%2Fjo00034a059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donovan, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozas, I.</span></span> <span> </span><span class="NLM_article-title">Pyridin-2-yl Guanidine Derivatives: Conformational Control Induced by Intramolecular Hydrogen-Bonding Interactions</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">9216</span>– <span class="NLM_lpage">9227</span>, <span class="refDoi"> DOI: 10.1021/jo200954c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo200954c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OnsLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=9216-9227&author=B.+Kellyauthor=D.+H.+O%E2%80%99Donovanauthor=J.+O%E2%80%99Brienauthor=T.+McCabeauthor=F.+Blancoauthor=I.+Rozas&title=Pyridin-2-yl+Guanidine+Derivatives%3A+Conformational+Control+Induced+by+Intramolecular+Hydrogen-Bonding+Interactions&doi=10.1021%2Fjo200954c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridin-2-yl guanidine derivatives: Conformational control induced by intramolecular hydrogen-bonding interactions</span></div><div class="casAuthors">Kelly, Brendan; O'Donovan, Daniel H.; O'Brien, John; McCabe, Thomas; Blanco, Fernando; Rozas, Isabel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9216-9227</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and conformational anal. of a series of pyridin-2-yl guanidine derivs. using NMR, X-ray crystallog., and B3LYP/6-31+G** theor. studies are reported.  A remarkable difference was obsd. in the 1H NMR spectra of the guanidinium salts as compared with their N,N'-di-Boc protected and neutral analogs.  This difference corresponds to a 180° change in the dihedral angle between the guanidine/ium moiety and the pyridine ring in the salts as compared to the Boc-protected derivs., a conclusion that was supported by theor. studies, X-ray data, and NMR anal.  Moreover, our data sustain the existence of two intramol. hydrogen-bonding systems: (i) between the pyridine N1 atom and the guanidinium protons in the salts and (ii) within the tert-Bu carbamate groups of the Boc-protected derivs.  To verify that the obsd. conformational control arises from these intramol. interactions, a new series of N-Boc-N'-propyl-substituted pyridin-2-yl guanidines were also prepd. and studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfrhi-HXC_BLVg90H21EOLACvtfcHk0lil4aUjqlNz9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OnsLvF&md5=3981dd5b64a899381e64118c392c2e8a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjo200954c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo200954c%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DD.%2BH.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DJ.%26aulast%3DMcCabe%26aufirst%3DT.%26aulast%3DBlanco%26aufirst%3DF.%26aulast%3DRozas%26aufirst%3DI.%26atitle%3DPyridin-2-yl%2520Guanidine%2520Derivatives%253A%2520Conformational%2520Control%2520Induced%2520by%2520Intramolecular%2520Hydrogen-Bonding%2520Interactions%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D9216%26epage%3D9227%26doi%3D10.1021%2Fjo200954c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seh, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P. Y.</span></span> <span> </span><span class="NLM_article-title">Ligand-Bound Structures of the Dengue Virus Protease Reveal the Active Conformation</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1128/JVI.06225-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FJVI.06225-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=22031935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsVKks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=438-446&author=C.+G.+Nobleauthor=C.+C.+Sehauthor=A.+T.+Chaoauthor=P.+Y.+Shi&title=Ligand-Bound+Structures+of+the+Dengue+Virus+Protease+Reveal+the+Active+Conformation&doi=10.1128%2FJVI.06225-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-bound structures of the dengue virus protease reveal the active conformation</span></div><div class="casAuthors">Noble, Christian G.; Seh, Cheah Chen; Chao, Alexander T.; Shi, Pei Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">438-446</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue is a mosquito-borne viral hemorrhagic disease that is a major threat to human health in tropical and subtropical regions.  Here we report crystal structures of a peptide covalently bound to dengue virus serotype 3 (DENV-3) protease as well as the serine-protease inhibitor aprotinin bound to the same enzyme.  These structures reveal, for the first time, a catalytically active, closed conformation of the DENV protease.  In the presence of the peptide, the DENV-3 protease forms the closed conformation in which the hydrophilic β-hairpin region of NS2B wraps around the NS3 protease core, in a manner analogous to the structure of West Nile virus (WNV) protease.  Our results confirm that flavivirus proteases form the closed conformation during proteolysis, as previously proposed for WNV.  The current DENV-3 protease structures reveal the detailed interactions at the P4' to P3 sites of the substrate.  The new structural information explains the sequence preference, particularly for long basic residues in the nonprime side, as well as the difference in substrate specificity between the WNV and DENV proteases at the prime side.  Structural anal. of the DENV-3 protease-peptide complex revealed a pocket that is formed by residues from NS2B and NS3; this pocket also exists in the WNV NS2B/NS3 protease structure and could be targeted for potential antivirus development.  The structural information presented in the current study is invaluable for the design of specific inhibitors of DENV protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEQKa3k5XF7rVg90H21EOLACvtfcHk0lil4aUjqlNz9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsVKks7c%253D&md5=f1b30da53856fe70d4c129fab8d54f35</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FJVI.06225-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.06225-11%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DC.%2BG.%26aulast%3DSeh%26aufirst%3DC.%2BC.%26aulast%3DChao%26aufirst%3DA.%2BT.%26aulast%3DShi%26aufirst%3DP.%2BY.%26atitle%3DLigand-Bound%2520Structures%2520of%2520the%2520Dengue%2520Virus%2520Protease%2520Reveal%2520the%2520Active%2520Conformation%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D438%26epage%3D446%26doi%3D10.1128%2FJVI.06225-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boldescu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streicher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimoglo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Synthesis, Biological Evaluation, and Molecular Docking of Combretastatin and Colchicine Derivatives and Their HCE1-Activated Prodrugs as Antiviral Agents</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fcmdc.201800641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=30605241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCrsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=469-483&author=M.+Richterauthor=V.+Boldescuauthor=D.+Grafauthor=F.+Streicherauthor=A.+Dimogloauthor=R.+Bartenschlagerauthor=C.+D.+Klein&title=Synthesis%2C+Biological+Evaluation%2C+and+Molecular+Docking+of+Combretastatin+and+Colchicine+Derivatives+and+Their+HCE1-Activated+Prodrugs+as+Antiviral+Agents&doi=10.1002%2Fcmdc.201800641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological evaluation, and molecular docking of combretastatin and colchicine derivatives and their hCE1-activated prodrugs as antiviral agents</span></div><div class="casAuthors">Richter, Michael; Boldescu, Veaceslav; Graf, Dominik; Streicher, Felix; Dimoglo, Anatoli; Bartenschlager, Ralf; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-483</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Recent studies indicate that tubulin can be a host factor for vector-borne flaviviruses like dengue (DENV) and Zika (ZIKV), and inhibitors of tubulin polymn. such as colchicine have been demonstrated to decrease virus replication.  However, toxicity limits the application of these compds.  Herein we report prodrugs based on combretastatin and colchicine derivs. that contain an ester cleavage site for human carboxylesterase, a highly abundant enzyme in monocytes and hepatocytes targeted by DENV.  Relative to their parent compds., the cytotoxicity of these prodrugs was reduced by several orders of magnitude.  All synthesized prodrugs contg. a leucine ester were hydrolyzed by the esterase in vitro.  In contrast to previous reports, the phenylglycine esters were not cleaved by human carboxylesterase.  The antiviral activity of combretastatin, colchicine, and selected prodrugs against DENV and ZIKV in cell culture was obsd. at low micromolar and sub-micromolar concns.  In addn., docking studies were performed to understand the binding mode of the studied compds. to tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsPhT5W1m43bVg90H21EOLACvtfcHk0lil4aUjqlNz9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCrsr8%253D&md5=877296cfa9d8b10725b73d7a0f0f3ea8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800641%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DBoldescu%26aufirst%3DV.%26aulast%3DGraf%26aufirst%3DD.%26aulast%3DStreicher%26aufirst%3DF.%26aulast%3DDimoglo%26aufirst%3DA.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DSynthesis%252C%2520Biological%2520Evaluation%252C%2520and%2520Molecular%2520Docking%2520of%2520Combretastatin%2520and%2520Colchicine%2520Derivatives%2520and%2520Their%2520HCE1-Activated%2520Prodrugs%2520as%2520Antiviral%2520Agents%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D469%26epage%3D483%26doi%3D10.1002%2Fcmdc.201800641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kain, S. R.</span></span> <span> </span><span class="NLM_article-title">Generation of Destabilized Green Fluorescent Protein as a Transcription Reporter</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">34970</span>– <span class="NLM_lpage">34975</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.52.34970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.273.52.34970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=9857028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK1MXisFajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=34970-34975&author=X.+Liauthor=X.+Zhaoauthor=Y.+Fangauthor=X.+Jiangauthor=T.+Duongauthor=C.+Fanauthor=C.-C.+Huangauthor=S.+R.+Kain&title=Generation+of+Destabilized+Green+Fluorescent+Protein+as+a+Transcription+Reporter&doi=10.1074%2Fjbc.273.52.34970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of destabilized green fluorescent protein as a transcription reporter</span></div><div class="casAuthors">Li, Xianqiang; Zhao, Xiaoning; Fang, Yu; Jiang, Xin; Duong, Tommy; Fan, Connie; Huang, Chiao-Chain; Kain, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">34970-34975</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The green fluorescent protein (GFP) is a widely used reporter in gene expression and protein localization studies.  GFP is a stable protein; this property allows its accumulation and easy detection in cells.  However, this stability also limits its application in studies that require rapid reporter turnover.  We created a destabilized GFP for use in such studies by fusing amino acids 422-461 of the degrdn. domain of mouse ornithine decarboxylase (MODC) to the C-terminal end of an enhanced variant of GFP (EGFP).  The fusion protein, unlike EGFP, was unstable in the presence of cycloheximide and had a fluorescence half-life of 2 h.  Western blot anal. indicated that the fluorescence decay of EGFP-MODC-(422-461) was correlated with degrdn. of the fusion protein.  We mutated key amino acids in the PEST sequence of EGFP-MODC-(422-461) and identified several mutants with variable half-lives.  The suitability of destabilized EGFP as a transcription reporter was tested by linking it to NFκB binding sequences and monitoring tumor necrosis factor α-mediated NFκB activation.  We obtained time course induction and dose response kinetics similar to secreted alk. phosphatase obtained in transfected cells.  This result did not occur when unmodified EGFP was used as the reporter.  Because of its autofluorescence, destabilized EGFP can be used to directly correlate gene induction with biochem. change, such as NFκB translocation to the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnMsl5eDXgJ7Vg90H21EOLACvtfcHk0ljWjAaNCcWyfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisFajsA%253D%253D&md5=4557c9ff881135e40b55c90877e6b9a0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.52.34970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.52.34970%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DDuong%26aufirst%3DT.%26aulast%3DFan%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DC.-C.%26aulast%3DKain%26aufirst%3DS.%2BR.%26atitle%3DGeneration%2520of%2520Destabilized%2520Green%2520Fluorescent%2520Protein%2520as%2520a%2520Transcription%2520Reporter%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D34970%26epage%3D34975%26doi%3D10.1074%2Fjbc.273.52.34970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rechsteiner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, S. W.</span></span> <span> </span><span class="NLM_article-title">PEST Sequences and Regulation by Proteolysis</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">271</span>, <span class="refDoi"> DOI: 10.1016/S0968-0004(96)10031-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2FS0968-0004%2896%2910031-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=8755249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK28XksVymtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1996&pages=267-271&author=M.+Rechsteinerauthor=S.+W.+Rogers&title=PEST+Sequences+and+Regulation+by+Proteolysis&doi=10.1016%2FS0968-0004%2896%2910031-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">PEST sequences and regulation by proteolysis</span></div><div class="casAuthors">Rechsteiner, Martin; Rogers, Scott W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">267-271</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Trends Journals</span>)
        </div><div class="casAbstract">A review with 48 refs.  In 1986, we proposed that polypeptide sequences enriched in proline (P), glutamic acid (E), serine (S) and threonine (T) target proteins for rapid destruction.  For much of the past decade there were only sporadic exptl. tests of the hypothesis.  This situation changed markedly during the past two years with a no. of papers providing strong evidence that PEST regions do, in fact, serve as proteolytic signals.  The properties of PEST regions and some interesting examples of the conditional nature of such signals are briefly discussed.  Most of the article, however, focuses on recent exptl. support for the hypothesis and on mechanisms responsible for the rapid degrdn. of proteins that contain PEST regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJfQ1FYbkGtLVg90H21EOLACvtfcHk0ljWjAaNCcWyfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XksVymtr8%253D&md5=14cd790123efab12ba61c6226ca0c986</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2896%2910031-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252896%252910031-1%26sid%3Dliteratum%253Aachs%26aulast%3DRechsteiner%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DS.%2BW.%26atitle%3DPEST%2520Sequences%2520and%2520Regulation%2520by%2520Proteolysis%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1996%26volume%3D21%26spage%3D267%26epage%3D271%26doi%3D10.1016%2FS0968-0004%2896%2910031-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunn, H. F.</span></span> <span> </span><span class="NLM_article-title">Regulation of Hypoxia-Inducible Factor 1alpha Is Mediated by an O2-Dependent Degradation Domain Via the Ubiquitin-Proteasome Pathway</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">7987</span>– <span class="NLM_lpage">7992</span>, <span class="refDoi"> DOI: 10.1073/pnas.95.14.7987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1073%2Fpnas.95.14.7987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=9653127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK1cXks1SlsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=7987-7992&author=L.+E.+Huangauthor=J.+Guauthor=M.+Schauauthor=H.+F.+Bunn&title=Regulation+of+Hypoxia-Inducible+Factor+1alpha+Is+Mediated+by+an+O2-Dependent+Degradation+Domain+Via+the+Ubiquitin-Proteasome+Pathway&doi=10.1073%2Fpnas.95.14.7987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of hypoxia-inducible factor 1α is mediated by an O2-dependent degradation domain via the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Huang, L. Eric; Gu, Jie; Schau, Maureen; Bunn, H. Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7987-7992</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hypoxia induces a group of physiol. important genes such as erythropoietin and vascular endothelial growth factor.  These genes are transcriptionally up-regulated by hypoxia-inducible factor 1 (HIF-1), a global regulator that belongs to the basic helix-loop-helix PAS family.  Although HIF-1 is a heterodimer composed of α and β subunits, its activity is primarily detd. by hypoxia-induced stabilization of HIF-1α which is otherwise rapidly degraded in oxygenated cells.  The authors report the identification of an oxygen-dependent degrdn. (ODD) domain within HIF-1α that controls its degrdn. by the ubiquitin-proteasome pathway.  The ODD domain consists of ≈200 amino acid residues, located in the central region of HIF-1α.  Because portions of the domain independently confer degrdn. of HIF-1α, deletion of this entire region is required to give rise to a stable HIF-1α, capable of heterodimerization, DNA-binding, and transactivation in the absence of hypoxic signaling.  Conversely, the ODD domain alone confers oxygen-dependent instability when fused to a stable protein, Gal4.  Hence, the ODD domain plays a pivotal role for regulating HIF-1 activity and thereby may provide a means of controlling gene expression by changes in oxygen tension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGXW78adyIx7Vg90H21EOLACvtfcHk0ljWjAaNCcWyfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1SlsrY%253D&md5=2d2765f29149ea4e78964f5721922e19</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.14.7987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.14.7987%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%2BE.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DSchau%26aufirst%3DM.%26aulast%3DBunn%26aufirst%3DH.%2BF.%26atitle%3DRegulation%2520of%2520Hypoxia-Inducible%2520Factor%25201alpha%2520Is%2520Mediated%2520by%2520an%2520O2-Dependent%2520Degradation%2520Domain%2520Via%2520the%2520Ubiquitin-Proteasome%2520Pathway%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D7987%26epage%3D7992%26doi%3D10.1073%2Fpnas.95.14.7987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-W.</span></span> <span> </span><span class="NLM_article-title">Hypoxia-Inducible Factor (HIF-1)α: Its Protein Stability and Biological Functions</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1038/emm.2004.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Femm.2004.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=15031665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1Grs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2004&pages=1-12&author=J.-W.+Leeauthor=S.-H.+Baeauthor=J.-W.+Jeongauthor=S.-H.+Kimauthor=K.-W.+Kim&title=Hypoxia-Inducible+Factor+%28HIF-1%29%CE%B1%3A+Its+Protein+Stability+and+Biological+Functions&doi=10.1038%2Femm.2004.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia-inducible factor (HIF-1)α: its protein stability and biological functions</span></div><div class="casAuthors">Lee, Ji-Won; Bae, Seong-Hui; Jeong, Joo-Won; Kim, Se-Hee; Kim, Kyu-Won</div><div class="citationInfo"><span class="NLM_cas:title">Experimental and Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1226-3613</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Medical Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Hypoxia-inducible factor (HIF-1) is an oxygen-dependent transcriptional activator, which plays crucial roles in the angiogenesis of tumors and mammalian development.  HIF-1 consists of a constitutively expressed HIF-1β subunit and one of three subunits (HIF-1α, HIF-2α or HIF-3α).  The stability and activity of HIF-1α are regulated by various post-translational modifications, hydroxylation, acetylation, and phosphorylation.  Therefore, HIF-1α interacts with several protein factors including PHD, pVHL, ARD-1, and p300/CBP.  Under normoxia, the HIF-1α subunit is rapidly degraded via the von Hippel-Lindau tumor suppressor gene product (pVHL)-mediated ubiquitin-proteasome pathway.  The assocn. of pVHL and HIF-1α under normoxic conditions is triggered by the hydroxylation of prolines and the acetylation of lysine within a polypeptide segment known as the oxygen-dependent degrdn. (ODD) domain.  On the contrary, in the hypoxia condition, HIF-1α subunit becomes stable and interacts with coactivators such as p300/CBP to modulate its transcriptional activity.  Eventually, HIF-1 acts as a master regulator of numerous hypoxia-inducible genes under hypoxic conditions.  The target genes of HIF-1 are esp. related to angiogenesis, cell proliferation/survival, and glucose/iron metab.  Moreover, it was reported that the activation of HIF-1α is closely assocd. with a variety of tumors and oncogenic pathways.  Hence, the blocking of HIF-1α itself or HIF-1α interacting proteins inhibit tumor growth.  Based on these findings, HIF-1 can be a prime target for anticancer therapies.  This review summarizes the mol. mechanism of HIF-1α stability, the bi- ol. functions of HIF-1 and its potential applications of cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkaNBwWoBA-bVg90H21EOLACvtfcHk0lizQLFG8TzkVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1Grs7g%253D&md5=49fb7514a11c4b30d0e803520242969d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Femm.2004.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2004.1%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.-W.%26aulast%3DBae%26aufirst%3DS.-H.%26aulast%3DJeong%26aufirst%3DJ.-W.%26aulast%3DKim%26aufirst%3DS.-H.%26aulast%3DKim%26aufirst%3DK.-W.%26atitle%3DHypoxia-Inducible%2520Factor%2520%2528HIF-1%2529%25CE%25B1%253A%2520Its%2520Protein%2520Stability%2520and%2520Biological%2520Functions%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2004%26volume%3D36%26spage%3D1%26epage%3D12%26doi%3D10.1038%2Femm.2004.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, P. J.</span></span> <span> </span><span class="NLM_article-title">Independent Function of Two Destruction Domains in Hypoxia-Inducible Factor-Alpha Chains Activated by Prolyl Hydroxylation</span>. <i>EMBO J</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5197</span>– <span class="NLM_lpage">5206</span>, <span class="refDoi"> DOI: 10.1093/emboj/20.18.5197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1093%2Femboj%2F20.18.5197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=11566883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD3MXns1Ghs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=5197-5206&author=N.+Massonauthor=C.+Willamauthor=P.+H.+Maxwellauthor=C.+W.+Pughauthor=P.+J.+Ratcliffe&title=Independent+Function+of+Two+Destruction+Domains+in+Hypoxia-Inducible+Factor-Alpha+Chains+Activated+by+Prolyl+Hydroxylation&doi=10.1093%2Femboj%2F20.18.5197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Independent function of two destruction domains in hypoxia-inducible factor-α chains activated by prolyl hydroxylation</span></div><div class="casAuthors">Masson, Norma; Willam, Carsten; Maxwell, Patrick H.; Pugh, Christopher W.; Ratcliffe, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5197-5206</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Oxygen-dependent proteolytic destruction of hypoxia-inducible factor-α (HIF-α) subunits plays a central role in regulating transcriptional responses to hypoxia.  Recent studies have defined a key function for the von Hippel-Lindau tumor suppressor E3 ubiquitin ligase (VHLE3) in this process, and have defined an interaction with HIF-1α that is regulated by prolyl hydroxylation.  Here we show that two independent regions within the HIF-α oxygen-dependent degrdn. domain (ODDD) are targeted for ubiquitylation by VHLE3 in a manner dependent upon prolyl hydroxylation.  In a series of in vitro and in vivo assays, we demonstrate the independent and non-redundant operation of each site in regulation of the HIF system.  Both sites contain a common core motif, but differ both in overall sequence and in the conditions under which they bind to the VHLE3 ligase complex.  The definition of two independent destruction domains implicates a more complex system of pVHL-HIF-α interactions, but reinforces the role of prolyl hydroxylation as an oxygen-dependent destruction signal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLKrBnPdE6lrVg90H21EOLACvtfcHk0lizQLFG8TzkVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXns1Ghs78%253D&md5=e4f167d66a64f515de9c43e53a7f546d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F20.18.5197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F20.18.5197%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DN.%26aulast%3DWillam%26aufirst%3DC.%26aulast%3DMaxwell%26aufirst%3DP.%2BH.%26aulast%3DPugh%26aufirst%3DC.%2BW.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26atitle%3DIndependent%2520Function%2520of%2520Two%2520Destruction%2520Domains%2520in%2520Hypoxia-Inducible%2520Factor-Alpha%2520Chains%2520Activated%2520by%2520Prolyl%2520Hydroxylation%26jtitle%3DEMBO%2520J%26date%3D2001%26volume%3D20%26spage%3D5197%26epage%3D5206%26doi%3D10.1093%2Femboj%2F20.18.5197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smirnova, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakhman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payappilly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazakov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez-Guzman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaisina, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratan, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazaryan, I. G.</span></span> <span> </span><span class="NLM_article-title">Utilization of an in Vivo Reporter for High Throughput Identification of Branched Small Molecule Regulators of Hypoxic Adaptation</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.chembiol.2010.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=20416509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFOiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=380-391&author=N.+A.+Smirnovaauthor=I.+Rakhmanauthor=N.+Morozauthor=M.+Bassoauthor=J.+Payappillyauthor=S.+Kazakovauthor=F.+Hernandez-Guzmanauthor=I.+N.+Gaisinaauthor=A.+P.+Kozikowskiauthor=R.+R.+Ratanauthor=I.+G.+Gazaryan&title=Utilization+of+an+in+Vivo+Reporter+for+High+Throughput+Identification+of+Branched+Small+Molecule+Regulators+of+Hypoxic+Adaptation&doi=10.1016%2Fj.chembiol.2010.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Utilization of an In Vivo Reporter for High Throughput Identification of Branched Small Molecule Regulators of Hypoxic Adaptation</span></div><div class="casAuthors">Smirnova, Natalya A.; Rakhman, Ilay; Moroz, Natalia; Basso, Manuela; Payappilly, Jimmy; Kazakov, Sergey; Hernandez-Guzman, Francisco; Gaisina, Irina N.; Kozikowski, Alan P.; Ratan, Rajiv R.; Gazaryan, Irina G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">380-391</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Small mols. inhibiting hypoxia inducible factor (HIF) prolyl hydroxylases (PHDs) are the focus of drug development efforts directed toward the treatment of ischemia and metabolic imbalance.  A cell-based reporter produced by fusing HIF-1α oxygen degradable domain (ODD) to luciferase was shown to work as a capture assay monitoring stability of the overexpressed luciferase-labeled HIF PHD substrate under conditions more physiol. than in vitro test tubes.  High throughput screening identified novel catechol and oxyquinoline pharmacophores with a "branching motif" immediately adjacent to a Fe-binding motif that fits selectively into the HIF PHD active site in in silico models.  In accord with their structure-activity relationship in the primary screen, the best "hits" stabilize HIF1α, upregulate known HIF target genes in a human neuronal line, and exert neuroprotective effects in established model of oxidative stress in cortical neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTX3nine48HLVg90H21EOLACvtfcHk0lizQLFG8TzkVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFOiu7g%253D&md5=6d590af4c5d184e21268bb7a352ce387</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DSmirnova%26aufirst%3DN.%2BA.%26aulast%3DRakhman%26aufirst%3DI.%26aulast%3DMoroz%26aufirst%3DN.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DPayappilly%26aufirst%3DJ.%26aulast%3DKazakov%26aufirst%3DS.%26aulast%3DHernandez-Guzman%26aufirst%3DF.%26aulast%3DGaisina%26aufirst%3DI.%2BN.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DGazaryan%26aufirst%3DI.%2BG.%26atitle%3DUtilization%2520of%2520an%2520in%2520Vivo%2520Reporter%2520for%2520High%2520Throughput%2520Identification%2520of%2520Branched%2520Small%2520Molecule%2520Regulators%2520of%2520Hypoxic%2520Adaptation%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D380%26epage%3D391%26doi%3D10.1016%2Fj.chembiol.2010.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Safran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flippin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günzler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.</span></span> <span> </span><span class="NLM_article-title">Mouse Model for Noninvasive Imaging of HIF Prolyl Hydroxylase Activity: Assessment of an Oral Agent That Stimulates Erythropoietin Production</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1073/pnas.0509459103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1073%2Fpnas.0509459103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=16373502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=105-110&author=M.+Safranauthor=W.+Y.+Kimauthor=F.+O%E2%80%99Connellauthor=L.+Flippinauthor=V.+G%C3%BCnzlerauthor=J.+W.+Hornerauthor=R.+A.+DePinhoauthor=W.+G.+Kaelin&title=Mouse+Model+for+Noninvasive+Imaging+of+HIF+Prolyl+Hydroxylase+Activity%3A+Assessment+of+an+Oral+Agent+That+Stimulates+Erythropoietin+Production&doi=10.1073%2Fpnas.0509459103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse model for noninvasive imaging of HIF prolyl hydroxylase activity: Assessment of an oral agent that stimulates erythropoietin production</span></div><div class="casAuthors">Safran, Michal; Kim, William Y.; O'Connell, Fionnuala; Flippin, Lee; Gunzler, Volkmar; Horner, James W.; DePinho, Ronald A.; Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-110</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many human diseases are characterized by the development of tissue hypoxia.  Inadequate oxygenation can cause cellular dysfunction and death.  Tissues use many strategies, including induction of angiogenesis and alterations in metab., to survive under hypoxic conditions.  The heterodimeric transcription factor hypoxia-inducible factor (HIF) is a master regulator of genes that promote adaptation to hypoxia.  HIF activity is linked to oxygen availability because members of the EGLN family hydroxylate HIFα subunits on specific prolyl residues when oxygen is present, which marks them for ubiquitination and proteasomal degrdn.  The authors created a mouse that ubiquitously expresses a bioluminescent reporter consisting of firefly luciferase fused to a region of HIF that is sufficient for oxygen-dependent degrdn.  The authors' validation studies suggest that this mouse will be useful for monitoring hypoxic tissues and evaluating therapeutic agents that stabilize HIF.  One such agent, the HIF prolyl hydroxylase inhibitor FG-4383, was active in the liver and kidney after systemic administration as detd. by bioluminescence imaging, transcription profiling, and prodn. of erythropoietin, indicating that the HIF transcriptional program can be manipulated in vivo with orally active org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3yVklADN01LVg90H21EOLACvtfcHk0ljvAA941JUtnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms12juw%253D%253D&md5=c39a622219baf85cb3f7d69e0cca5392</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0509459103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0509459103%26sid%3Dliteratum%253Aachs%26aulast%3DSafran%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DW.%2BY.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DF.%26aulast%3DFlippin%26aufirst%3DL.%26aulast%3DG%25C3%25BCnzler%26aufirst%3DV.%26aulast%3DHorner%26aufirst%3DJ.%2BW.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DMouse%2520Model%2520for%2520Noninvasive%2520Imaging%2520of%2520HIF%2520Prolyl%2520Hydroxylase%2520Activity%253A%2520Assessment%2520of%2520an%2520Oral%2520Agent%2520That%2520Stimulates%2520Erythropoietin%2520Production%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D105%26epage%3D110%26doi%3D10.1073%2Fpnas.0509459103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimbrel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span> <span> </span><span class="NLM_article-title">In Vivo Pharmacodynamic Imaging of Proteasome Inhibition</span>. <i>Mol. Imaging</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">147</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.2310%2F7290.2009.00007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19723471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptVemtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=140-147&author=E.+A.+Kimbrelauthor=T.+N.+Davisauthor=J.+E.+Bradnerauthor=A.+L.+Kung&title=In+Vivo+Pharmacodynamic+Imaging+of+Proteasome+Inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo pharmacodynamic imaging of proteasome inhibition</span></div><div class="casAuthors">Kimbrel, Erin A.; Davis, Tina N.; Bradner, James E.; Kung, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Imaging</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">140-147</span>CODEN:
                <span class="NLM_cas:coden">MIOMBP</span>;
        ISSN:<span class="NLM_cas:issn">1535-3508</span>.
    
            (<span class="NLM_cas:orgname">BC Decker Inc.</span>)
        </div><div class="casAbstract">Inhibiting the proteolytic activity of the 26S proteasome has been shown to have selective apoptotic effects on cancer cells and to be clin. efficacious in certain malignancies.  There is an unmet medical need for addnl. proteasome inhibitors, and their development will be facilitated by surrogate markers of proteasome function.  Toward this end, ectopic fusion of the destruction domain from ornithine decarboxylase (ODC) to reporter proteins is often used for assessing proteasome function.  For luciferase-based reporters, we hypothesized that the oxygen-dependent destruction domain (ODD) from hypoxia-inducible factor 1α (HIF-1α) may provide improved sensitivity over luciferase-ODC, owing to its extremely rapid turnover by the proteasome (HIF-1α has a half-life of less than 5 min).  In the current study, we show that ODD-luciferase affords a greater dynamic range and faster kinetics than luciferase-ODC in sensing proteasome inhibition in vitro.  Importantly, ODD-luciferase also serves as an effective in vivo marker of proteasome function in xenograft tumor models, with inhibition being detected by noninvasive imaging within 3 h of bortezomib administration.  These data establish ODD-luciferase as a surrogate marker of proteasome function that can be used both in vitro and in vivo for the development of novel proteasome inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomH3K_-EPUwrVg90H21EOLACvtfcHk0ljvAA941JUtnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptVemtbY%253D&md5=25301db1b4f83a1778539374bba4fe00</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2310%2F7290.2009.00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2310%252F7290.2009.00007%26sid%3Dliteratum%253Aachs%26aulast%3DKimbrel%26aufirst%3DE.%2BA.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKung%26aufirst%3DA.%2BL.%26atitle%3DIn%2520Vivo%2520Pharmacodynamic%2520Imaging%2520of%2520Proteasome%2520Inhibition%26jtitle%3DMol.%2520Imaging%26date%3D2009%26volume%3D8%26spage%3D140%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kizaka-Kondoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itasaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morinibu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinomiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraoka, M.</span></span> <span> </span><span class="NLM_article-title">The Combination of Hypoxia-Response Enhancers and an Oxygen-Dependent Proteolytic Motif Enables Real-Time Imaging of Absolute HIF-1 Activity in Tumor Xenografts</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>360</i></span>,  <span class="NLM_fpage">791</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2007.06.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bbrc.2007.06.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17624305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVWmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2007&pages=791-796&author=H.+Haradaauthor=S.+Kizaka-Kondohauthor=S.+Itasakaauthor=K.+Shibuyaauthor=A.+Morinibuauthor=K.+Shinomiyaauthor=M.+Hiraoka&title=The+Combination+of+Hypoxia-Response+Enhancers+and+an+Oxygen-Dependent+Proteolytic+Motif+Enables+Real-Time+Imaging+of+Absolute+HIF-1+Activity+in+Tumor+Xenografts&doi=10.1016%2Fj.bbrc.2007.06.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The combination of hypoxia-response enhancers and an oxygen-dependent proteolytic motif enables real-time imaging of absolute HIF-1 activity in tumor xenografts</span></div><div class="casAuthors">Harada, Hiroshi; Kizaka-Kondoh, Shinae; Itasaka, Satoshi; Shibuya, Keiko; Morinibu, Akiyo; Shinomiya, Kazumi; Hiraoka, Masahiro</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">360</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">791-796</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The transcriptional activity of hypoxia-inducible factor-1 (HIF-1) is assocd. with tumor malignancies; therefore, it is important to comprehend its dynamism in solid tumors.  However, a mol. imaging strategy to accurately access it remains to be developed.  We constructed here a novel HIF-1-dependent reporter gene, 5HREp-ODD-luc, in which 5 copies of the hypoxia-response element (5HRE) enhance expression of the oxygen-dependent degrdn. (ODD) domain and luciferase (luc) fusion under hypoxia.  Because the ODD domain caused the oxygen-dependent degrdn. of the ODD-Luc protein, the novel reporter gene showed little leak of luminescence under normoxia.  Such a property caused an increase of the hypoxia-responsiveness up to about 4.7×104-fold.  Moreover, the ODD domain caused rapid degrdn. of the ODD-Luc protein under normoxia, the luminescence reflected the dynamism of HIF-1 activity in real-time.  The superiority of the novel reporter gene will surely accelerate anal. of the intratumoral HIF-1 activity during tumor progression and cancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLe6_xAOLmnrVg90H21EOLACvtfcHk0ljvAA941JUtnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVWmt7o%253D&md5=8e4daaa98bca82b5adc2487e29986a10</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.06.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.06.149%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DH.%26aulast%3DKizaka-Kondoh%26aufirst%3DS.%26aulast%3DItasaka%26aufirst%3DS.%26aulast%3DShibuya%26aufirst%3DK.%26aulast%3DMorinibu%26aufirst%3DA.%26aulast%3DShinomiya%26aufirst%3DK.%26aulast%3DHiraoka%26aufirst%3DM.%26atitle%3DThe%2520Combination%2520of%2520Hypoxia-Response%2520Enhancers%2520and%2520an%2520Oxygen-Dependent%2520Proteolytic%2520Motif%2520Enables%2520Real-Time%2520Imaging%2520of%2520Absolute%2520HIF-1%2520Activity%2520in%2520Tumor%2520Xenografts%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D360%26spage%3D791%26epage%3D796%26doi%3D10.1016%2Fj.bbrc.2007.06.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrosky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span> <span> </span><span class="NLM_article-title">Use of the ODD-Luciferase Transgene for the Non-Invasive Imaging of Spontaneous Tumors in Mice</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e18269</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0018269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1371%2Fjournal.pone.0018269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=21479246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXks1Klu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=S.+J.+Goldmanauthor=E.+Chenauthor=R.+Taylorauthor=S.+Zhangauthor=W.+Petroskyauthor=M.+Reissauthor=S.+Jin&title=Use+of+the+ODD-Luciferase+Transgene+for+the+Non-Invasive+Imaging+of+Spontaneous+Tumors+in+Mice&doi=10.1371%2Fjournal.pone.0018269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the ODD-luciferase transgene for the non-invasive imaging of spontaneous tumors in mice</span></div><div class="casAuthors">Goldman, Scott J.; Chen, Elizabeth; Taylor, Robert; Zhang, Sheng; Petrosky, Whitney; Reiss, Michael; Jin, Shengkan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e18269</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">In humans, imaging of tumors provides rapid, accurate assessment of tumor growth and location.  In lab. animals, however, the imaging of spontaneously occurring tumors continues to pose many tech. and logistical problems.  Recently a mouse model was generated in which a chimeric protein consisting of HIF-1α oxygen-dependent degrdn. domain (ODD) fused to luciferase was ubiquitously expressed in all tissues.  Hypoxic stress leads to the accumulation of ODD-luciferase in the tissues of this mouse model which can be identified by non-invasive bioluminescence measurement.  Since solid tumors often contain hypoxic regions, we performed proof-of-principle expts. testing whether this transgenic mouse model may be used as a universal platform for non-invasive imaging anal. of spontaneous solid tumors.  ODD-luciferase transgenic mice were bred with MMTV-neu/beclin1+/- mice.  Upon injection of luciferin, bioluminescent background of normal tissues in the transgenic mice and bioluminescent signals from spontaneously mammary carcinomas were measured non-invasively with an IVIS Spectrum imaging station.  Tumor vols. were measured manually and the histol. of tumor tissues was analyzed.  Our results show that spontaneous mammary tumors in ODD-luciferase transgenic mice generate substantial bioluminescent signals, which are clearly discernable from background tissue luminescence.  Moreover, we demonstrate a strong quant. correlation between the bioluminescent tumor contour and the vol. of palpable tumors.  We further demonstrate that shrinkage of the vol. of spontaneous tumors in response to chemotherapeutic treatment can be detd. quant. using this system.  Finally, we show that the growth and development of spontaneous tumors can be monitored longitudinally over several weeks.  Thus, our results suggest that this model could potentially provide a practical, reliable, and cost-effective non-invasive quant. method for imaging spontaneous solid tumors in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_88YwpPfkF7Vg90H21EOLACvtfcHk0ljBBLPdnFRM2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXks1Klu7g%253D&md5=5a8d2c5365c07bf8e3c0760603cff6fb</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0018269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0018269%26sid%3Dliteratum%253Aachs%26aulast%3DGoldman%26aufirst%3DS.%2BJ.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DPetrosky%26aufirst%3DW.%26aulast%3DReiss%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DS.%26atitle%3DUse%2520of%2520the%2520ODD-Luciferase%2520Transgene%2520for%2520the%2520Non-Invasive%2520Imaging%2520of%2520Spontaneous%2520Tumors%2520in%2520Mice%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0018269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bera, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. L.</span></span> <span> </span><span class="NLM_article-title">Functional Characterization of Cis and Trans Activity of the Flavivirus NS2B-NS3 Protease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">12883</span>– <span class="NLM_lpage">12892</span>, <span class="refDoi"> DOI: 10.1074/jbc.M611318200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.M611318200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17337448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVSitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=12883-12892&author=A.+K.+Beraauthor=R.+J.+Kuhnauthor=J.+L.+Smith&title=Functional+Characterization+of+Cis+and+Trans+Activity+of+the+Flavivirus+NS2B-NS3+Protease&doi=10.1074%2Fjbc.M611318200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Characterization of cis and trans Activity of the Flavivirus NS2B-NS3 Protease</span></div><div class="casAuthors">Bera, Aloke K.; Kuhn, Richard J.; Smith, Janet L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">12883-12892</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Flaviviruses are serious human pathogens for which treatments are generally lacking.  The proteolytic maturation of the 375-kDa viral polyprotein is one target for antiviral development.  The flavivirus serine protease consists of the N-terminal domain of the multifunctional nonstructural protein 3 (NS3) and an essential 40-residue cofactor (NS2B40) within viral protein NS2B.  The NS2B-NS3 protease is responsible for all cytoplasmic cleavage events in viral polyprotein maturation.  This study describes the first biochem. characterization of flavivirus protease activity using full-length NS3.  Recombinant proteases were created by fusion of West Nile virus (WNV) NS2B40 to full-length WNV NS3.  The protease catalyzed two autolytic cleavages.  The NS2B/NS3 junction was cleaved before protein purifn.  A second site at Arg459↓Gly460 within the C-terminal helicase region of NS3 was cleaved more slowly.  Autolytic cleavage reactions also occurred in NS2B-NS3 recombinant proteins from yellow fever virus, dengue virus types 2 and 4, and Japanese encephalitis virus.  Cis and trans cleavages were distinguished using a noncleavable WNV protease variant and two types of substrates as follows: an inactive variant of recombinant WNV NS2B-NS3, and cyan and yellow fluorescent proteins fused by a dodecamer peptide encompassing a natural cleavage site.  With these materials, the autolytic cleavages were found to be intramol. only.  Autolytic cleavage of the helicase site was insensitive to protein diln., confirming that autolysis is intramol.  Formation of an active protease was found to require neither cleavage of NS2B from NS3 nor a free NS3 N terminus.  Evidence was also obtained for product inhibition of the protease by the cleaved C terminus of NS2B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppQqtdjn7TJrVg90H21EOLACvtfcHk0ljBBLPdnFRM2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVSitbc%253D&md5=2ac2dc45954b392fcc4738d653276117</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M611318200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M611318200%26sid%3Dliteratum%253Aachs%26aulast%3DBera%26aufirst%3DA.%2BK.%26aulast%3DKuhn%26aufirst%3DR.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BL.%26atitle%3DFunctional%2520Characterization%2520of%2520Cis%2520and%2520Trans%2520Activity%2520of%2520the%2520Flavivirus%2520NS2B-NS3%2520Protease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D12883%26epage%3D12892%26doi%3D10.1074%2Fjbc.M611318200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arias-Arias, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora-Rodríguez, R.</span></span> <span> </span><span class="NLM_article-title">A Fluorescence-Activatable Reporter of Flavivirus NS2B–NS3 Protease Activity Enables Live Imaging of Infection in Single Cells and Viral Plaques</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2226</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA119.011319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.RA119.011319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=31919100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosFChsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=2212-2226&author=J.+L.+Arias-Ariasauthor=D.+J.+MacPhersonauthor=M.+E.+Hillauthor=J.+A.+Hardyauthor=R.+Mora-Rodr%C3%ADguez&title=A+Fluorescence-Activatable+Reporter+of+Flavivirus+NS2B%E2%80%93NS3+Protease+Activity+Enables+Live+Imaging+of+Infection+in+Single+Cells+and+Viral+Plaques&doi=10.1074%2Fjbc.RA119.011319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence-activatable reporter of flavivirus NS2B-NS3 protease activity enables live imaging of infection in single cells and viral plaques</span></div><div class="casAuthors">Arias-Arias, Jorge L.; MacPherson, Derek J.; Hill, Maureen E.; Hardy, Jeanne A.; Mora-Rodriguez, Rodrigo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2212-2227</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">1083-351X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The genus Flavivirus in the family Flaviviridae comprises many medically important viruses, such as dengue virus (DENV), Zika virus (ZIKV), and yellow fever virus.  The quest for therapeutic targets to combat flavivirus infections requires a better understanding of the kinetics of virus- host interactions during infections with native viral strains.  However, this is precluded by limitations of current cell-based systems for monitoring flavivirus infection in living cells.  The authors report the construction of fluorescence-activatable sensors to detect the activities of flavivirus NS2B-NS3 serine proteases in living cells.  The system consists of GFP-based reporters that become fluorescent upon cleavage by recombinant DENV-2/ZIKV proteases in vitro.  A version of this sensor contg. the flavivirus internal NS3 cleavage site linker reported the highest fluorescence activation in stably transduced mammalian cells upon DENV-2/ ZIKV infection.  Moreover, the onset of fluorescence correlated with viral protease activity.  A far-red version of this flavivirus sensor had the best signal-to-noise ratio in a fluorescent Dulbecco's plaque assay, leading to the construction of a multireporter platform combining the flavivirus sensor with reporter dyes for detection of chromatin condensation and cell death, enabling studies of viral plaque formation with single-cell resoln.  Finally, the application of this platform enabled the study of cell-population kinetics of infection and cell death by DENV-2, ZIKV, and yellow fever virus.  The authors anticipate that future studies of viral infection kinetics with this reporter system will enable basic studies of virus- host interactions and facilitate future applications in antiviral drug research to manage flavivirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqadWoYTjPOPbVg90H21EOLACvtfcHk0ljBBLPdnFRM2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosFChsLo%253D&md5=773be52f5c7530a0ce87c78a443780e0</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA119.011319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA119.011319%26sid%3Dliteratum%253Aachs%26aulast%3DArias-Arias%26aufirst%3DJ.%2BL.%26aulast%3DMacPherson%26aufirst%3DD.%2BJ.%26aulast%3DHill%26aufirst%3DM.%2BE.%26aulast%3DHardy%26aufirst%3DJ.%2BA.%26aulast%3DMora-Rodr%25C3%25ADguez%26aufirst%3DR.%26atitle%3DA%2520Fluorescence-Activatable%2520Reporter%2520of%2520Flavivirus%2520NS2B%25E2%2580%2593NS3%2520Protease%2520Activity%2520Enables%2520Live%2520Imaging%2520of%2520Infection%2520in%2520Single%2520Cells%2520and%2520Viral%2520Plaques%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D2212%26epage%3D2226%26doi%3D10.1074%2Fjbc.RA119.011319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snitko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weidner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weißbrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spannaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acosta, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodem, J.</span></span> <span> </span><span class="NLM_article-title">Novel Dengue Virus NS2B/NS3 Protease Inhibitors</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1100</span>– <span class="NLM_lpage">1109</span>, <span class="refDoi"> DOI: 10.1128/AAC.03543-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FAAC.03543-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=25487800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1KrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=1100-1109&author=H.+Wuauthor=S.+Bockauthor=M.+Snitkoauthor=T.+Bergerauthor=T.+Weidnerauthor=S.+Hollowayauthor=M.+Kanitzauthor=W.+E.+Diederichauthor=H.+Steuberauthor=C.+Walterauthor=D.+Hofmannauthor=B.+Wei%C3%9Fbrichauthor=R.+Spannausauthor=E.+G.+Acostaauthor=R.+Bartenschlagerauthor=B.+Engelsauthor=T.+Schirmeisterauthor=J.+Bodem&title=Novel+Dengue+Virus+NS2B%2FNS3+Protease+Inhibitors&doi=10.1128%2FAAC.03543-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Novel dengue virus NS2B/NS3 protease inhibitors</span></div><div class="casAuthors">Wu, Hongmei; Bock, Stefanie; Snitko, Mariya; Berger, Thilo; Weidner, Thomas; Holloway, Steven; Kanitz, Manuel; Diederich, Wibke E.; Steuber, Holger; Walter, Christof; Hofmann, Daniela; Weissbrich, Benedikt; Spannaus, Ralf; Acosta, Eliana G.; Bartenschlager, Ralf; Engels, Bernd; Schirmeister, Tanja; Bodem, Jochen</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1100-1109, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue fever is a severe, widespread, and neglected disease with more than 2 million diagnosed infections per yr.  The dengue virus NS2B/NS3 protease (PR) represents a prime target for rational drug design.  At the moment, there are no clin. PR inhibitors (PIs) available.  We have identified diaryl (thio)ethers as candidates for a novel class of PIs.  Here, we report the selective and noncompetitive inhibition of the serotype 2 and 3 dengue virus PR in vitro and in cells by benzothiazole derivs. exhibiting 50% inhibitory concns. (IC50s) in the low-micromolar range.  Inhibition of replication of DENV serotypes 1 to 3 was specific, since all substances influenced neither hepatitis C virus (HCV) nor HIV-1 replication.  Mol. docking suggests binding at a specific allosteric binding site.  In addn. to the in vitro assays, a cell-based PR assay was developed to test these substances in a replication-independent way.  The new compds. inhibited the DENV PR with IC50s in the low-micromolar or submicromolar range in cells.  Furthermore, these novel PIs inhibit viral replication at submicromolar concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvLZ7EVXcgu7Vg90H21EOLACvtfcHk0liPJ6XPLVgUqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1KrtbY%253D&md5=b41b4e2566cd1d1115a33edf42962930</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FAAC.03543-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.03543-14%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBock%26aufirst%3DS.%26aulast%3DSnitko%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DT.%26aulast%3DWeidner%26aufirst%3DT.%26aulast%3DHolloway%26aufirst%3DS.%26aulast%3DKanitz%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DW.%2BE.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DWalter%26aufirst%3DC.%26aulast%3DHofmann%26aufirst%3DD.%26aulast%3DWei%25C3%259Fbrich%26aufirst%3DB.%26aulast%3DSpannaus%26aufirst%3DR.%26aulast%3DAcosta%26aufirst%3DE.%2BG.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DEngels%26aufirst%3DB.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DBodem%26aufirst%3DJ.%26atitle%3DNovel%2520Dengue%2520Virus%2520NS2B%252FNS3%2520Protease%2520Inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D1100%26epage%3D1109%26doi%3D10.1128%2FAAC.03543-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, A.</span></span> <span> </span><span class="NLM_article-title">Novel Dengue Virus-Specific NS2B/NS3 Protease Inhibitor, BP2109, Discovered by a High-Throughput Screening Assay</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1128/AAC.00855-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FAAC.00855-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=20937790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVehu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=229-238&author=C.-C.+Yangauthor=Y.-C.+Hsiehauthor=S.-J.+Leeauthor=S.-H.+Wuauthor=C.-L.+Liaoauthor=C.-H.+Tsaoauthor=Y.-S.+Chaoauthor=J.-H.+Chernauthor=C.-P.+Wuauthor=A.+Yueh&title=Novel+Dengue+Virus-Specific+NS2B%2FNS3+Protease+Inhibitor%2C+BP2109%2C+Discovered+by+a+High-Throughput+Screening+Assay&doi=10.1128%2FAAC.00855-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Novel dengue virus-specific NS2B/NS3 protease inhibitor, BP2109, discovered by a high-throughput screening assay</span></div><div class="casAuthors">Yang, Chi-Chen; Hsieh, Yi-Chen; Lee, Shiow-Ju; Wu, Szu-Huei; Liao, Ching-Len; Tsao, Chang-Huei; Chao, Yu-Sheng; Chern, Jyh-Haur; Wu, Chung-Pu; Yueh, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">229-238</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) causes disease globally, with an estd. 25 to 100 million new infections per yr.  At present, no effective vaccine is available, and treatment is supportive.  In this study, we identified BP2109, a potent and selective small-mol. inhibitor of the DENV NS2B/NS3 protease, by a high-throughput screening assay using a recombinant protease complex consisting of the central hydrophilic portion of NS2B and the N terminus of the protease domain.  BP2109 inhibited DENV (serotypes 1 to 4), but not Japanese encephalitis virus (JEV), replication and viral RNA synthesis without detectable cytotoxicity.  The compd. inhibited recombinant DENV-2 NS2B/NS3 protease with a 50% inhibitory concn. (IC50) of 15.43 ± 2.12 μM and reduced the reporter expression of the DENV-2 replicon with a 50% effective concn. (EC50) of 0.17 ± 0.01 μM.  Sequencing analyses of several individual clones derived from BP2109-resistant DENV-2 RNAs revealed that two amino acid substitutions (R55K and E80K) are found in the region of NS2B, a cofactor of the NS2B/NS3 protease complex.  The introduction of R55K and E80K double mutations into the dengue virus NS2B/NS3 protease and a dengue virus replicon construct conferred 10.3- and 73.8-fold resistance to BP2109, resp.  The E80K mutation was further detd. to be the key mutation conferring dengue virus replicon resistance (61.3-fold) to BP2109, whereas the R55K mutation alone did not affect resistance to BP2109.  Both the R55K and E80K mutations are located in the central hydrophilic portion of the NS2B cofactor, where extensive interactions with the NS3pro domain exist.  Thus, our data provide evidence that BP2109 likely inhibits DENV by a novel mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRD8XjZuvDP7Vg90H21EOLACvtfcHk0liPJ6XPLVgUqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVehu7k%253D&md5=e1a0728b4d1077399a005f7a84dad1e2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FAAC.00855-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00855-10%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.-C.%26aulast%3DHsieh%26aufirst%3DY.-C.%26aulast%3DLee%26aufirst%3DS.-J.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DLiao%26aufirst%3DC.-L.%26aulast%3DTsao%26aufirst%3DC.-H.%26aulast%3DChao%26aufirst%3DY.-S.%26aulast%3DChern%26aufirst%3DJ.-H.%26aulast%3DWu%26aufirst%3DC.-P.%26aulast%3DYueh%26aufirst%3DA.%26atitle%3DNovel%2520Dengue%2520Virus-Specific%2520NS2B%252FNS3%2520Protease%2520Inhibitor%252C%2520BP2109%252C%2520Discovered%2520by%2520a%2520High-Throughput%2520Screening%2520Assay%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D229%26epage%3D238%26doi%3D10.1128%2FAAC.00855-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiaang, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, A.</span></span> <span> </span><span class="NLM_article-title">A Novel Dengue Virus Inhibitor, BP13944, Discovered by High-Throughput Screening with Dengue Virus Replicon Cells Selects for Resistance in the Viral NS2B/NS3 Protease</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1128/AAC.01281-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FAAC.01281-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=24145533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFCitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=110-119&author=C.-C.+Yangauthor=H.-S.+Huauthor=R.-H.+Wuauthor=S.-H.+Wuauthor=S.-J.+Leeauthor=W.-T.+Jiaangauthor=J.-H.+Chernauthor=Z.-S.+Huangauthor=H.-N.+Wuauthor=C.-M.+Changauthor=A.+Yueh&title=A+Novel+Dengue+Virus+Inhibitor%2C+BP13944%2C+Discovered+by+High-Throughput+Screening+with+Dengue+Virus+Replicon+Cells+Selects+for+Resistance+in+the+Viral+NS2B%2FNS3+Protease&doi=10.1128%2FAAC.01281-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A novel dengue virus inhibitor, BP13944, discovered by high-throughput screening with dengue virus replicon cells selects for resistance in the viral NS2B/NS3 protease</span></div><div class="casAuthors">Yang, Chi-Chen; Hu, Han-Shu; Wu, Ren-Huang; Wu, Szu-Huei; Lee, Shiow-Ju; Jiaang, Weir-Torn; Chern, Jyh-Haur; Huang, Zhi-Shun; Wu, Huey-Nan; Chang, Chung-Ming; Yueh, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">110-119, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) causes disease globally, resulting in an estd. 25 to 100 million new infections per yr.  No effective DENV vaccine is available, and the current treatment is only supportive.  Thus, there is an urgent need to develop therapeutic agents to cure this epidemic disease.  In the present study, we identified a potential small-mol. inhibitor, BP13944, via high-throughput screening (HTS) of 60,000 compds. using a stable cell line harboring an efficient luciferase replicon of DENV serotype 2 (DENV-2).  BP13944 reduced the expression of the DENV replicon reporter in cells, showing a 50% effective concn. (EC50) of 1.03 ± 0.09 μM.  Without detectable cytotoxicity, the compd. inhibited replication or viral RNA synthesis in all four serotypes of DENV but not in Japanese encephalitis virus (JEV).  Sequencing analyses of several individual clones derived from BP13944-resistant RNAs purified from cells harboring the DENV-2 replicon revealed a consensus amino acid substitution (E66G) in the region of the NS3 protease domain.  Introduction of E66G into the DENV replicon, an infectious DENV cDNA clone, and recombinant NS2B/NS3 protease constructs conferred 15.2-, 17.2-, and 3.1-fold resistance to BP13944, resp.  Our results identify an effective small-mol. inhibitor, BP13944, which likely targets the DENV NS3 protease.  BP13944 could be considered part of a more effective treatment regime for inhibiting DENV in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtuEbfgxAqMLVg90H21EOLACvtfcHk0liPJ6XPLVgUqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFCitb8%253D&md5=445bed8eb7106964aaaf626a318c999f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FAAC.01281-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01281-13%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.-C.%26aulast%3DHu%26aufirst%3DH.-S.%26aulast%3DWu%26aufirst%3DR.-H.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DLee%26aufirst%3DS.-J.%26aulast%3DJiaang%26aufirst%3DW.-T.%26aulast%3DChern%26aufirst%3DJ.-H.%26aulast%3DHuang%26aufirst%3DZ.-S.%26aulast%3DWu%26aufirst%3DH.-N.%26aulast%3DChang%26aufirst%3DC.-M.%26aulast%3DYueh%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Dengue%2520Virus%2520Inhibitor%252C%2520BP13944%252C%2520Discovered%2520by%2520High-Throughput%2520Screening%2520with%2520Dengue%2520Virus%2520Replicon%2520Cells%2520Selects%2520for%2520Resistance%2520in%2520the%2520Viral%2520NS2B%252FNS3%2520Protease%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D110%26epage%3D119%26doi%3D10.1128%2FAAC.01281-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of Dengue Virus Protease Inhibitors</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2009.02.190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.antiviral.2009.02.190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19428601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVKrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2009&pages=110-114&author=S.+M.+Tomlinsonauthor=R.+D.+Malmstromauthor=A.+Russoauthor=N.+Muellerauthor=Y.-P.+Pangauthor=S.+J.+Watowich&title=Structure-Based+Discovery+of+Dengue+Virus+Protease+Inhibitors&doi=10.1016%2Fj.antiviral.2009.02.190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of dengue virus protease inhibitors</span></div><div class="casAuthors">Tomlinson, Suzanne M.; Malmstrom, Robert D.; Russo, Andrew; Mueller, Niklaus; Pang, Yuan-Ping; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">110-114</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dengue virus belongs to the family Flaviviridae and is a major emerging pathogen for which the development of vaccines and antiviral therapy has seen little success.  The NS3 viral protease is a potential target for antiviral drugs since it is required for virus replication.  The goal of this study was to identify novel dengue virus (type 2; DEN2V) protease inhibitors for eventual development as effective anti-flaviviral drugs.  The EUDOC docking program was used to computationally screen a small-mol. library for compds. that dock into the P1 pocket and the catalytic site of the DEN2V NS3 protease domain apo-structure and the Bowman-Birk inhibitor-bound structure.  The top 20 computer-identified hits that demonstrated the most favorable scoring energies were selected for in vitro assessment of protease inhibition.  Preliminary protease activity assays demonstrated that more than half of the tested compds. were sol. and exhibited in vitro inhibition of the DEN2V protease.  Two of these compds. also inhibited viral replication in cell culture expts., and thus are promising compds. for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4lR431in7_rVg90H21EOLACvtfcHk0ljIsLbSKiCV7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVKrsbg%253D&md5=4d527895dc1cf644918c5b8f51ae0399</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2009.02.190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2009.02.190%26sid%3Dliteratum%253Aachs%26aulast%3DTomlinson%26aufirst%3DS.%2BM.%26aulast%3DMalmstrom%26aufirst%3DR.%2BD.%26aulast%3DRusso%26aufirst%3DA.%26aulast%3DMueller%26aufirst%3DN.%26aulast%3DPang%26aufirst%3DY.-P.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Dengue%2520Virus%2520Protease%2520Inhibitors%26jtitle%3DAntiviral%2520Res.%26date%3D2009%26volume%3D82%26spage%3D110%26epage%3D114%26doi%3D10.1016%2Fj.antiviral.2009.02.190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steuer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinonen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Optimization of Assay Conditions for Dengue Virus Protease: Effect of Various Polyols and Nonionic Detergents</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1108</span>, <span class="refDoi"> DOI: 10.1177/1087057109344115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1177%2F1087057109344115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19726784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVOltrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=1102-1108&author=C.+Steuerauthor=K.+H.+Heinonenauthor=L.+Kattnerauthor=C.+D.+Klein&title=Optimization+of+Assay+Conditions+for+Dengue+Virus+Protease%3A+Effect+of+Various+Polyols+and+Nonionic+Detergents&doi=10.1177%2F1087057109344115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of assay conditions for dengue virus protease: effect of various polyols and nonionic detergents</span></div><div class="casAuthors">Steuer, Christian; Heinonen, Karl H.; Kattner, Lars; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1108</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The aim of this work was to perform a systematic study of the effect of nonionic detergents on the activity of the dengue virus NS2B-NS3 protease.  To ensure a high activity of the protease, the assay procedures for the dengue virus and other flaviviral proteases published to date are performed in the presence of up to 35% glycerol, which does not represent the cellular physicochem. environment.  In addn., the high viscosity of glycerol-contg. solns. leads to various exptl. problems in miniaturized assays.  Using an internally quenched peptide substrate, the authors show that glycerol is not essential for enzymic activity if certain nonionic detergents are added to the assay buffer.  In addn., nonionic detergents may help to avoid false-pos. screening results caused by "promiscuous" inhibitors.  Other polyalcs. can substitute glycerol and have less effect on the viscosity of the assay buffer.  The assay was used to screen a compd. library and allowed the identification of small-mol. nonpeptidic inhibitors of dengue NS3 protease.  Finally, the authors discuss the mode of action of nonionic detergents and the influence that they may have on the conformational properties of the NS2B-NS3 protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG3ngqZ602BbVg90H21EOLACvtfcHk0ljIsLbSKiCV7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVOltrrP&md5=c6acb8d910b47af78048e59b633c6353</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F1087057109344115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109344115%26sid%3Dliteratum%253Aachs%26aulast%3DSteuer%26aufirst%3DC.%26aulast%3DHeinonen%26aufirst%3DK.%2BH.%26aulast%3DKattner%26aufirst%3DL.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DOptimization%2520of%2520Assay%2520Conditions%2520for%2520Dengue%2520Virus%2520Protease%253A%2520Effect%2520of%2520Various%2520Polyols%2520and%2520Nonionic%2520Detergents%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D1102%26epage%3D1108%26doi%3D10.1177%2F1087057109344115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altman, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barthel, H. R.</span></span> <span> </span><span class="NLM_article-title">Topical Therapies for Osteoarthritis</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1279</span>, <span class="refDoi"> DOI: 10.2165/11592550-000000000-00000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.2165%2F11592550-000000000-00000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=21770475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlakur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=1259-1279&author=R.+D.+Altmanauthor=H.+R.+Barthel&title=Topical+Therapies+for+Osteoarthritis&doi=10.2165%2F11592550-000000000-00000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Topical therapies for osteoarthritis</span></div><div class="casAuthors">Altman, Roy D.; Barthel, H. Richard</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1259-1279</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Adis Data Information BV</span>)
        </div><div class="casAbstract">This review discusses the pharmacol., analgesic efficacy, safety and tolerability of topical NSAIDs, salicylates and capsaicin for the management of osteoarthritis (OA) pain.  Topical therapies present a valuable therapeutic option for OA pain management, with substantial evidence supporting the efficacy and safety of topical NSAIDs, but less robust support for capsaicin and salicylates.  We define topical therapies as those intended to act locally, in contrast to transdermal therapies intended to act systemically.  Oral therapies for patients with mild to moderate OA pain include paracetamol (acetaminophen) and NSAIDs.  Paracetamol has only weak efficacy at therapeutic doses and is hepatotoxic at doses >3.25 g/day.  NSAIDs have demonstrated efficacy in patients with OA, but are assocd. with dose-, duration- and age-dependent risks of gastrointestinal, cardiovascular, renal, haematol. and hepatic adverse events (AEs), as well as clin. meaningful drug interactions.  To minimize AE risks, treatment guidelines for OA suggest minimizing NSAID exposure by prescribing the lowest ED for the shortest duration of time.  Systemic NSAID exposure may also be limited by prescribing topical NSAIDs, particularly in patients with OA pain limited to a few superficial joints.  Topical NSAIDs have been available in Europe for decades and were introduced to provide localized analgesia with minimal systemic NSAID exposure.  Guidelines of the American Academy of Orthopaedic Surgeons, European League Against Rheumatism (EULAR), Osteoarthritis Research Society International, and National Institute for Health and Clin. Excellence (NICE) state that topical NSAIDs may be considered for patients with mild to moderate OA of the knee or hand, particularly in patients with few affected joints and/or a history of sensitivity to oral NSAIDs.  In fact, the EULAR and NICE guidelines state that topical NSAIDs should be considered before oral therapies.  Clin. trials of topical NSAIDs, most notably formulations of diclofenac and ketoprofen, have shown efficacy significantly superior to placebo and similar to oral NSAIDs.  Most topical NSAIDs (piroxicam being the exception) have shown improved safety and tolerability compared with oral NSAIDs.  Topical salicylates and capsaicin are available in the US without a prescription, but neither has shown substantial efficacy in clin. trials, and both have potential to cause serious AEs.  Accidental poisonings have been reported with salicylates, and concerns exist that capsaicin-induced nerve desensitization is not fully reversible and that its autonomic nerve effects may increase the risk of skin ulcers in diabetic patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdu0AHAnb-yLVg90H21EOLACvtfcHk0ljIsLbSKiCV7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlakur%252FE&md5=ec9dec1bfe57012584cb5d901537be9c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.2165%2F11592550-000000000-00000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F11592550-000000000-00000%26sid%3Dliteratum%253Aachs%26aulast%3DAltman%26aufirst%3DR.%2BD.%26aulast%3DBarthel%26aufirst%3DH.%2BR.%26atitle%3DTopical%2520Therapies%2520for%2520Osteoarthritis%26jtitle%3DDrugs%26date%3D2011%26volume%3D71%26spage%3D1259%26epage%3D1279%26doi%3D10.2165%2F11592550-000000000-00000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murawski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespi, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balimane, P. V.</span></span> <span> </span><span class="NLM_article-title">A Novel Design of Artificial Membrane for Improving the PAMPA Model</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1511</span>– <span class="NLM_lpage">1520</span>, <span class="refDoi"> DOI: 10.1007/s11095-007-9517-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1007%2Fs11095-007-9517-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=18185985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmslWntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2008&pages=1511-1520&author=X.+Chenauthor=A.+Murawskiauthor=K.+Patelauthor=C.+L.+Crespiauthor=P.+V.+Balimane&title=A+Novel+Design+of+Artificial+Membrane+for+Improving+the+PAMPA+Model&doi=10.1007%2Fs11095-007-9517-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Design of Artificial Membrane for Improving the Parallel Artificial Membrane Permeability Assay Model</span></div><div class="casAuthors">Chen, Xiaoxi; Murawski, Anthony; Patel, Karishma; Crespi, Charles L.; Balimane, Praveen V.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1511-1520</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose Since the first demonstration of parallel artificial membrane permeability assay (PAMPA), the artificial membrane has been traditionally prepd. by impregnating a porous filter with a soln. of lipid mixt.  While the lipid soln.-based method is simple and seems to provide good predictability for many compds., it is challenged by several shortcomings including reproducibility, stability, mass retention and the incorrect prediction of a group of highly permeable compds. including caffeine and antipyrine.  Here we present the validation of a novel artificial membrane formed by constructing a lipid/oil/lipid tri-layer in the porous filter.  Methods Permeability values obtained from traditional and new artificial membrane were compared for their correlation with Caco-2 and human absorption values.  Mass retention, stability and org. solvent compatibility of the new artificial membrane were studied.  Results The new artificial membrane correctly predicts the permeability of the traditionally under-predicted compds. and improves the correlation with Caco-2 and human absorption values.  Furthermore, the new artificial membrane reduces the mass retention of compds. that are highly retained by the traditional artificial membrane.  The new artificial membrane is also found to be robust enough to sustain long term storage and has good compatibility with org. solvents.  Conclusions The new artificial membrane provides an improved PAMPA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgXqaJqeAUybVg90H21EOLACvtfcHk0liAuZ-boz3aNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmslWntLo%253D&md5=b076352ee4cc2935c97afb5437cdd496</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs11095-007-9517-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-007-9517-8%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMurawski%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DCrespi%26aufirst%3DC.%2BL.%26aulast%3DBalimane%26aufirst%3DP.%2BV.%26atitle%3DA%2520Novel%2520Design%2520of%2520Artificial%2520Membrane%2520for%2520Improving%2520the%2520PAMPA%2520Model%26jtitle%3DPharm.%2520Res.%26date%3D2008%26volume%3D25%26spage%3D1511%26epage%3D1520%26doi%3D10.1007%2Fs11095-007-9517-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0liAuZ-boz3aNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N. J.</span>; <span class="NLM_string-name">Hinner, M. J.</span></span> <span> </span><span class="NLM_article-title">Getting across the Cell Membrane: An Overview for Small Molecules, Peptides, and Proteins</span>. In  <i>Site-Specific Protein Labeling: Methods and Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautier, A.</span>; <span class="NLM_string-name">Hinner, M. J.</span></span>, Eds. <span class="NLM_publisher-name">Springer New York</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">53</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1007%2F978-1-4939-2272-7_3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=29-53&author=N.+J.+Yang&author=M.+J.+Hinnerauthor=A.+Gautier&author=M.+J.+Hinner&title=Site-Specific+Protein+Labeling%3A+Methods+and+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2272-7_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-2272-7_3%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DN.%2BJ.%26atitle%3DGetting%2520across%2520the%2520Cell%2520Membrane%253A%2520An%2520Overview%2520for%2520Small%2520Molecules%252C%2520Peptides%252C%2520and%2520Proteins%26btitle%3DSite-Specific%2520Protein%2520Labeling%253A%2520Methods%2520and%2520Protocols%26aulast%3DGautier%26aufirst%3DA.%26pub%3DSpringer%2520New%2520York%26date%3D2015%26spage%3D29%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Zika Virus NS2B-NS3 Protease in Complex with a Boronate Inhibitor</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">503</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1126/science.aag2419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1126%2Fscience.aag2419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=27386922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ehtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=503-505&author=J.+Leiauthor=G.+Hansenauthor=C.+Nitscheauthor=C.+D.+Kleinauthor=L.+Zhangauthor=R.+Hilgenfeld&title=Crystal+Structure+of+Zika+Virus+NS2B-NS3+Protease+in+Complex+with+a+Boronate+Inhibitor&doi=10.1126%2Fscience.aag2419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of Zika virus NS2B-NS3 protease in complex with a boronate inhibitor</span></div><div class="casAuthors">Lei, Jian; Hansen, Guido; Nitsche, Christoph; Klein, Christian D.; Zhang, Linlin; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6298</span>),
    <span class="NLM_cas:pages">503-505</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The ongoing Zika virus (ZIKV) outbreak is linked to severe neurol. disorders.  ZIKV relies on its NS2B/NS3 protease for polyprotein processing; hence, this enzyme is an attractive drug target.  The 2.7 angstrom; crystal structure of ZIKV protease in complex with a peptidomimetic boronic acid inhibitor reveals a cyclic diester between the boronic acid and glycerol.  The P2 4-aminomethylphenylalanine moiety of the inhibitor forms a salt-bridge with the nonconserved Asp83 of NS2B; ion-pairing between Asp83 and the P2 residue of the substrate likely accounts for the enzyme's high catalytic efficiency.  The unusual dimer of the ZIKV protease:inhibitor complex seen in the crystal may provide a model for assemblies formed at high local concns. of protease at the endoplasmatic reticulum membrane, the site of polyprotein processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqIQ6GCMppIbVg90H21EOLACvtfcHk0liAuZ-boz3aNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ehtL%252FM&md5=32c40f1d5060b22f4cd2e781433fdee4</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1126%2Fscience.aag2419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aag2419%26sid%3Dliteratum%253Aachs%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520Zika%2520Virus%2520NS2B-NS3%2520Protease%2520in%2520Complex%2520with%2520a%2520Boronate%2520Inhibitor%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D503%26epage%3D505%26doi%3D10.1126%2Fscience.aag2419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodenreider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonntag, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shochat, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otting, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span> <span> </span><span class="NLM_article-title">A Fluorescence Quenching Assay to Discriminate between Specific and Nonspecific Inhibitors of Dengue Virus Protease</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2009.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.ab.2009.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19682971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GmsrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2009&pages=195-204&author=C.+Bodenreiderauthor=D.+Beerauthor=T.+H.+Kellerauthor=S.+Sonntagauthor=D.+Wenauthor=L.+Yapauthor=Y.+H.+Yauauthor=S.+G.+Shochatauthor=D.+Huangauthor=T.+Zhouauthor=A.+Caflischauthor=X.-C.+Suauthor=K.+Ozawaauthor=G.+Ottingauthor=S.+G.+Vasudevanauthor=J.+Lescarauthor=S.+P.+Lim&title=A+Fluorescence+Quenching+Assay+to+Discriminate+between+Specific+and+Nonspecific+Inhibitors+of+Dengue+Virus+Protease&doi=10.1016%2Fj.ab.2009.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence quenching assay to discriminate between specific and nonspecific inhibitors of dengue virus protease</span></div><div class="casAuthors">Bodenreider, Christophe; Beer, David; Keller, Thomas H.; Sonntag, Sebastian; Wen, Daying; Yap, Li Jian; Yau, Yin Hoe; Shochat, Susana Geifman; Huang, Danzhi; Zhou, Ting; Caflisch, Amedeo; Su, Xun-Cheng; Ozawa, Kiyoshi; Otting, Gottfried; Vasudevan, Subhash G.; Lescar, Julien; Lim, Siew Pheng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">195-204</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In drug discovery, the occurrence of false positives is a major hurdle in the search for lead compds. that can be developed into drugs.  A small-mol.-wt. compd. that inhibits dengue virus protease at low micromolar levels was identified in a screening campaign.  Binding to the enzyme was confirmed by isothermal titrn. calorimetry (ITC) and NMR.  However, a structure-activity relationship study that ensued did not yield more potent leads.  To further characterize the parental compd. and its analogs, we developed a high-speed, low-cost, quant. fluorescence quenching assay.  We obsd. that specific analogs quenched dengue protease fluorescence and showed variation in IC50 values.  In contrast, nonspecifically binding compds. did not quench its fluorescence and showed similar IC50 values with steep dose-response curves.  We validated the assay using single Trp-to-Ala protease mutants and the competitive protease inhibitor aprotinin.  Specific compds. detected in the binding assay were further analyzed by competitive ITC, NMR, and surface plasmon resonance, and the assay's utility in comparison with these biophys. methods is discussed.  The sensitivity of this assay makes it highly useful for hit finding and validation in drug discovery.  Furthermore, the technique can be readily adapted for studying other protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIeUje_AwhZ7Vg90H21EOLACvtfcHk0ljRjRX7nuW4lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GmsrfI&md5=ca035ecd54f7c72e77d03ed5afe69d18</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2009.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2009.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DBodenreider%26aufirst%3DC.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DSonntag%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DYap%26aufirst%3DL.%26aulast%3DYau%26aufirst%3DY.%2BH.%26aulast%3DShochat%26aufirst%3DS.%2BG.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DSu%26aufirst%3DX.-C.%26aulast%3DOzawa%26aufirst%3DK.%26aulast%3DOtting%26aufirst%3DG.%26aulast%3DVasudevan%26aufirst%3DS.%2BG.%26aulast%3DLescar%26aufirst%3DJ.%26aulast%3DLim%26aufirst%3DS.%2BP.%26atitle%3DA%2520Fluorescence%2520Quenching%2520Assay%2520to%2520Discriminate%2520between%2520Specific%2520and%2520Nonspecific%2520Inhibitors%2520of%2520Dengue%2520Virus%2520Protease%26jtitle%3DAnal.%2520Biochem.%26date%3D2009%26volume%3D395%26spage%3D195%26epage%3D204%26doi%3D10.1016%2Fj.ab.2009.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brieke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cryle, M. J.</span></span> <span> </span><span class="NLM_article-title">A Facile Fmoc Solid Phase Synthesis Strategy to Access Epimerization-Prone Biosynthetic Intermediates of Glycopeptide Antibiotics</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2454</span>– <span class="NLM_lpage">2457</span>, <span class="refDoi"> DOI: 10.1021/ol500840f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol500840f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmt1ShsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=2454-2457&author=C.+Briekeauthor=M.+J.+Cryle&title=A+Facile+Fmoc+Solid+Phase+Synthesis+Strategy+to+Access+Epimerization-Prone+Biosynthetic+Intermediates+of+Glycopeptide+Antibiotics&doi=10.1021%2Fol500840f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A Facile Fmoc Solid Phase Synthesis Strategy To Access Epimerization-Prone Biosynthetic Intermediates of Glycopeptide Antibiotics</span></div><div class="casAuthors">Brieke, Clara; Cryle, Max J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2454-2457</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A rapid protocol based on Fmoc-chem. for the solid-phase peptide synthesis of vancomycin- and teicoplanin-type peptides is described.  Epimerization of highly racemization-prone arylglycine derivs. is suppressed through optimized Fmoc-deprotection and coupling conditions.  Starting from easily accessible Fmoc-protected amino acids, this strategy enables the enantioselective synthesis of peptides corresponding to intermediates found in vancomycin and teicoplanin biosynthesis with excellent purity and in high yields (38%-71%).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Z8GnYCsJU7Vg90H21EOLACvtfcHk0ljRjRX7nuW4lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmt1ShsbY%253D&md5=cee6106a787cbb147bcdcd07c66caf2e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fol500840f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol500840f%26sid%3Dliteratum%253Aachs%26aulast%3DBrieke%26aufirst%3DC.%26aulast%3DCryle%26aufirst%3DM.%2BJ.%26atitle%3DA%2520Facile%2520Fmoc%2520Solid%2520Phase%2520Synthesis%2520Strategy%2520to%2520Access%2520Epimerization-Prone%2520Biosynthetic%2520Intermediates%2520of%2520Glycopeptide%2520Antibiotics%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D2454%26epage%3D2457%26doi%3D10.1021%2Fol500840f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elsawy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewage, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, B.</span></span> <span> </span><span class="NLM_article-title">Racemisation of N-Fmoc Phenylglycine under Mild Microwave-SPPS and Conventional Stepwise SPPS Conditions: Attempts to Develop Strategies for Overcoming This</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">302</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1002/psc.2398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fpsc.2398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=22451378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC38XksV2hu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=302-311&author=M.+A.+Elsawyauthor=C.+Hewageauthor=B.+Walker&title=Racemisation+of+N-Fmoc+Phenylglycine+under+Mild+Microwave-SPPS+and+Conventional+Stepwise+SPPS+Conditions%3A+Attempts+to+Develop+Strategies+for+Overcoming+This&doi=10.1002%2Fpsc.2398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Racemisation of N-Fmoc phenylglycine under mild microwave-SPPS and conventional stepwise SPPS conditions: attempts to develop strategies for overcoming this</span></div><div class="casAuthors">Elsawy, Mohamed A.; Hewage, Chandralal; Walker, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">302-311</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">We have been engaged in the microwave-solid phase peptide synthesis (SPPS) synthesis of the phenylglycine (Phg)-contg. pentapeptide H-Ala-Val-Pro-Phg-Tyr-NH2 previously demonstrated to bind to the so-called BIR3 domain of the anti-apoptotic protein XIAP.  Anal. of the target peptide by a combination of RP-HPLC, ESI-MS, and NMR revealed the presence of two diastereoisomers arising out of the racemization of the Phg residue, with the percentage of the LLLDL component assessed as 49%.  We performed the synthesis of peptide H-Ala-Val-Pro-Phg-Tyr-NH2 using different microwave and conventional stepwise SPPS conditions in attempts to reduce the level of racemization of the Phg residue and to det. at which part of the synthetic cycle the epimerization had occurred.  We detd. that racemization occurred mainly during the Fmoc-group removal and, to a much lesser extent, during activation/coupling of the Fmoc-Phg-OH residue.  We were able to obtain the desired peptide with a 71% diastereomeric purity (29% LLLDL as impurity) by utilizing microwave-assisted SPPS at 50 °C and power 22 W, when the triazine-derived coupling reagent DMTMM-BF4 was used, together with NMM as an activator base, for the incorporation of this residue and 20% piperidine as an Fmoc-deprotection base.  In contrast, the phenylalanine analog of the above peptide, H-Ala-Val-Pro-Phe-Tyr-NH2, was always obtained as a single diastereoisomer by using a range of std. coupling and deprotection conditions.  Our findings suggest that the racemization of Fmoc-Phg-OH, under both microwave-SPPS and stepwise conventional SPPS syntheses conditions, is very facile but can be limited through the use of the above stated conditions.  Copyright © 2012 European Peptide Society and John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHTPUtPSTeELVg90H21EOLACvtfcHk0ljRjRX7nuW4lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksV2hu7s%253D&md5=aef27234860e27672c7d6b76797c661d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2398%26sid%3Dliteratum%253Aachs%26aulast%3DElsawy%26aufirst%3DM.%2BA.%26aulast%3DHewage%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DB.%26atitle%3DRacemisation%2520of%2520N-Fmoc%2520Phenylglycine%2520under%2520Mild%2520Microwave-SPPS%2520and%2520Conventional%2520Stepwise%2520SPPS%2520Conditions%253A%2520Attempts%2520to%2520Develop%2520Strategies%2520for%2520Overcoming%2520This%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2012%26volume%3D18%26spage%3D302%26epage%3D311%26doi%3D10.1002%2Fpsc.2398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for Flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0ljlcFzbeXr68Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Örtqvist, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Åkerblom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gossas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fransson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helena Danielson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandström, A.</span></span> <span> </span><span class="NLM_article-title">Phenylglycine as a Novel P2 Scaffold in Hepatitis C Virus NS3 Protease Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1448</span>– <span class="NLM_lpage">1474</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2006.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmc.2006.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17113777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FhtVertg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=1448-1474&author=P.+%C3%96rtqvistauthor=S.+D.+Petersonauthor=E.+%C3%85kerblomauthor=T.+Gossasauthor=Y.+A.+Sabnisauthor=R.+Franssonauthor=G.+Lindebergauthor=U.+Helena+Danielsonauthor=A.+Karl%C3%A9nauthor=A.+Sandstr%C3%B6m&title=Phenylglycine+as+a+Novel+P2+Scaffold+in+Hepatitis+C+Virus+NS3+Protease+Inhibitors&doi=10.1016%2Fj.bmc.2006.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylglycine as a novel P2 scaffold in hepatitis C virus NS3 protease inhibitors</span></div><div class="casAuthors">Ortqvist Pernilla; Peterson Shane D; Kerblom Eva; Gossas Thomas; Sabnis Yogesh A; Fransson Rebecca; Lindeberg Gunnar; Helena Danielson U; Karlen Anders; Sandstrom Anja</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1448-74</span>
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    </div><div class="casAbstract">Molecular modeling and inhibitory potencies of tetrapeptide protease inhibitors of HCV NS3 proposed phenylglycine as a new promising P2 residue.  The results suggest that phenylglycine might be capable of interacting with the NS3 (protease-helicase/NTPase) in ways not possible for the common P2 proline-based inhibitors.  Thus, a series of tripeptides, both linear and macrocyclic, based on p-hydroxy-phenylglycine in the P2 position were prepared and their inhibitory effect determined.  When the p-hydroxy group was replaced by methoxy, isoquinolin-, or quinolinyloxy functions, inhibitors with improved potencies were obtained.  The P2 phenylglycine-based inhibitors were further optimized by C-terminal extension to acyl sulfonamides and by P1-P3 cyclization, which gave products with inhibition constants in the nanomolar range ( approximately 75nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ3vpve6fi1Ar5oR8RchjrrfW6udTcc2eZMXZ3d6uth67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FhtVertg%253D%253D&md5=3c7a55b59840dc761e76ad0c1a029c8d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.11.003%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596rtqvist%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DS.%2BD.%26aulast%3D%25C3%2585kerblom%26aufirst%3DE.%26aulast%3DGossas%26aufirst%3DT.%26aulast%3DSabnis%26aufirst%3DY.%2BA.%26aulast%3DFransson%26aufirst%3DR.%26aulast%3DLindeberg%26aufirst%3DG.%26aulast%3DHelena%2BDanielson%26aufirst%3DU.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26aulast%3DSandstr%25C3%25B6m%26aufirst%3DA.%26atitle%3DPhenylglycine%2520as%2520a%2520Novel%2520P2%2520Scaffold%2520in%2520Hepatitis%2520C%2520Virus%2520NS3%2520Protease%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D1448%26epage%3D1474%26doi%3D10.1016%2Fj.bmc.2006.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lampa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenberg, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vema, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Åkerblom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandström, A.</span></span> <span> </span><span class="NLM_article-title">Improved P2 Phenylglycine-Based Hepatitis C Virus NS3 Protease Inhibitors with Alkenylic Prime-Side Substituents</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5413</span>– <span class="NLM_lpage">5424</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.05.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmc.2010.05.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=20541424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXoslans78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=5413-5424&author=A.+Lampaauthor=A.+E.+Ehrenbergauthor=S.+S.+Gustafssonauthor=A.+Vemaauthor=E.+%C3%85kerblomauthor=G.+Lindebergauthor=A.+Karl%C3%A9nauthor=U.+H.+Danielsonauthor=A.+Sandstr%C3%B6m&title=Improved+P2+Phenylglycine-Based+Hepatitis+C+Virus+NS3+Protease+Inhibitors+with+Alkenylic+Prime-Side+Substituents&doi=10.1016%2Fj.bmc.2010.05.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Improved P2 phenylglycine-based hepatitis C virus NS3 protease inhibitors with alkenylic prime-side substituents</span></div><div class="casAuthors">Lampa, Anna; Ehrenberg, Angelica E.; Gustafsson, Sofia S.; Vema, Aparna; Aakerblom, Eva; Lindeberg, Gunnar; Karlen, Anders; Danielson, U. Helena; Sandstroem, Anja</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5413-5424</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phenylglycine has proven to be a useful P2 residue in HCV NS3 protease inhibitors.  A novel π-π-interaction between the phenylglycine and the catalytic H57 residue of the protease was postulated, and it was hypothesized that the introduction of a vinyl on the phenylglycine might strengthen this π-π-interaction.  Thus, a series of acyclic vinylated phenylglycine-based HCV NS3 protease inhibitors, e.g., I, was synthesized and their inhibitory potency evaluated.  Surprisingly, inhibitors based on both D- and L-phenylglycine were found to be effective inhibitors, with a slight preference for the D-epimers.  Furthermore, prime-side alkenylic extension of the C-terminal acylsulfonamide group gave significantly improved inhibitors with potencies in the nanomolar range (∼35 nM), potencies which were retained on mutant variants of the protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1S6r1UZnWFrVg90H21EOLACvtfcHk0ljlcFzbeXr68Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXoslans78%253D&md5=53f795b5eba3e0b2d27e74fd504229a0</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.05.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.05.027%26sid%3Dliteratum%253Aachs%26aulast%3DLampa%26aufirst%3DA.%26aulast%3DEhrenberg%26aufirst%3DA.%2BE.%26aulast%3DGustafsson%26aufirst%3DS.%2BS.%26aulast%3DVema%26aufirst%3DA.%26aulast%3D%25C3%2585kerblom%26aufirst%3DE.%26aulast%3DLindeberg%26aufirst%3DG.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DSandstr%25C3%25B6m%26aufirst%3DA.%26atitle%3DImproved%2520P2%2520Phenylglycine-Based%2520Hepatitis%2520C%2520Virus%2520NS3%2520Protease%2520Inhibitors%2520with%2520Alkenylic%2520Prime-Side%2520Substituents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D5413%26epage%3D5424%26doi%3D10.1016%2Fj.bmc.2010.05.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera—a Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lhTbqTNqAqgQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postnikova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWald, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logue, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiuliis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelotti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honko, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbrook, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olinger, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hensley, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahrling, P. B.</span></span> <span> </span><span class="NLM_article-title">Testing Therapeutics in Cell-Based Assays: Factors That Influence the Apparent Potency of Drugs</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e0194880</span>– <span class="NLM_lpage">e0194880</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0194880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1371%2Fjournal.pone.0194880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=29566079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslenur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0194880-e0194880&author=E.+Postnikovaauthor=Y.+Congauthor=L.+E.+DeWaldauthor=J.+Dyallauthor=S.+Yuauthor=B.+J.+Hartauthor=H.+Zhouauthor=R.+Grossauthor=J.+Logueauthor=Y.+Caiauthor=N.+Deiuliisauthor=J.+Michelottiauthor=A.+N.+Honkoauthor=R.+S.+Bennettauthor=M.+R.+Holbrookauthor=G.+G.+Olingerauthor=L.+E.+Hensleyauthor=P.+B.+Jahrling&title=Testing+Therapeutics+in+Cell-Based+Assays%3A+Factors+That+Influence+the+Apparent+Potency+of+Drugs&doi=10.1371%2Fjournal.pone.0194880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Testing therapeutics in cell-based assays: factors that influence the apparent potency of drugs</span></div><div class="casAuthors">Postnikova, Elena; Cong, Yu; DeWald, Lisa Evans; Dyall, Julie; Yu, Shuiqing; Hart, Brit J.; Zhou, Huanying; Gross, Robin; Logue, James; Cai, Yingyun; Deiuliis, Nicole; Michelotti, Julia; Honko, Anna N.; Bennett, Richard S.; Holbrook, Michael R.; Olinger, Gene G.; Hensley, Lisa E.; Jahrling, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0194880/1-e0194880/18</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Identifying effective antivirals for treating Ebola virus disease (EVD) and minimizing transmission of such disease is crit.  A variety of cell-based assays have been developed for evaluating compds. for activity against Ebola virus.  However, very few reports discuss the variable assay conditions that can affect the results obtained from these drug screens.  Here, we describe variable conditions tested during the development of our cell-based drug screen assays designed to identify compds. with anti-Ebola virus activity using established cell lines and human primary cells.  The effect of multiple assay readouts and variable assay conditions, including virus input, time of infection, and the cell passage no., were compared, and the impact on the effective concn. for 50% and/ or 90% inhibition (EC50, EC90) was evaluated using the FDA-approved compd., toremifene citrate.  In these studies, we show that altering cell-based assay conditions can have an impact on apparent drug potency as measured by the EC50.  These results further support the importance of developing std. operating procedures for generating reliable and reproducible in vitro data sets for potential antivirals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5MSjHQoAi4bVg90H21EOLACvtfcHk0lhTbqTNqAqgQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslenur7M&md5=a92120e9cba3ec410044b4301c8de218</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0194880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0194880%26sid%3Dliteratum%253Aachs%26aulast%3DPostnikova%26aufirst%3DE.%26aulast%3DCong%26aufirst%3DY.%26aulast%3DDeWald%26aufirst%3DL.%2BE.%26aulast%3DDyall%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DHart%26aufirst%3DB.%2BJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DGross%26aufirst%3DR.%26aulast%3DLogue%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DDeiuliis%26aufirst%3DN.%26aulast%3DMichelotti%26aufirst%3DJ.%26aulast%3DHonko%26aufirst%3DA.%2BN.%26aulast%3DBennett%26aufirst%3DR.%2BS.%26aulast%3DHolbrook%26aufirst%3DM.%2BR.%26aulast%3DOlinger%26aufirst%3DG.%2BG.%26aulast%3DHensley%26aufirst%3DL.%2BE.%26aulast%3DJahrling%26aufirst%3DP.%2BB.%26atitle%3DTesting%2520Therapeutics%2520in%2520Cell-Based%2520Assays%253A%2520Factors%2520That%2520Influence%2520the%2520Apparent%2520Potency%2520of%2520Drugs%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26spage%3De0194880%26epage%3De0194880%26doi%3D10.1371%2Fjournal.pone.0194880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreier, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Thiazolidinone–Peptide Hybrids as Dengue Virus Protease Inhibitors with Antiviral Activity in Cell Culture</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8389</span>– <span class="NLM_lpage">8403</span>, <span class="refDoi"> DOI: 10.1021/jm400828u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400828u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8389-8403&author=C.+Nitscheauthor=V.+N.+Schreierauthor=M.+A.+M.+Behnamauthor=A.+Kumarauthor=R.+Bartenschlagerauthor=C.+D.+Klein&title=Thiazolidinone%E2%80%93Peptide+Hybrids+as+Dengue+Virus+Protease+Inhibitors+with+Antiviral+Activity+in+Cell+Culture&doi=10.1021%2Fjm400828u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Thiazolidinone-Peptide Hybrids as Dengue Virus Protease Inhibitors with Antiviral Activity in Cell Culture</span></div><div class="casAuthors">Nitsche, Christoph; Schreier, Verena N.; Behnam, Mira A. M.; Kumar, Anil; Bartenschlager, Ralf; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8389-8403</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protease of dengue virus is a promising target for antiviral drug discovery.  We here report a new generation of peptide-hybrid inhibitors of dengue protease that incorporate N-substituted 5-arylidenethiazolidinone heterocycles (rhodanines and thiazolidinediones) as N-terminal capping groups of the peptide moiety.  The compds. were extensively characterized with respect to inhibition of various proteases, inhibition mechanisms, membrane permeability, antiviral activity, and cytotoxicity in cell culture.  A sulfur/oxygen exchange in position 2 of the capping heterocycle (thiazolidinedione-capped vs rhodanine-capped peptide hybrids) has a significant effect on these properties and activities.  The most promising in vitro affinities were obsd. for thiazolidinedione-based peptide hybrids contg. hydrophobic groups with Ki values between 1.5 and 1.8 μM and competitive inhibition mechanisms.  Rhodanine-capped peptide hybrids with hydrophobic substituents have, in correlation with their membrane permeability, a more pronounced antiviral activity in cell culture than the thiazolidinediones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq_iHlDnsmD7Vg90H21EOLACvtfcHk0lhTbqTNqAqgQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrfJ&md5=ab9c63ae3682c8621347f7fd8145bb24</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm400828u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400828u%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DSchreier%26aufirst%3DV.%2BN.%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DThiazolidinone%25E2%2580%2593Peptide%2520Hybrids%2520as%2520Dengue%2520Virus%2520Protease%2520Inhibitors%2520with%2520Antiviral%2520Activity%2520in%2520Cell%2520Culture%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8389%26epage%3D8403%26doi%3D10.1021%2Fjm400828u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span>; <span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Fluorimetric and HPLC-Based Dengue Virus Protease Assays Using a FRET Substrate</span>. In  <i>Antiviral Methods and Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, E. Y.</span></span>, Ed. <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">Totowa, New Jersey</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">236</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1007%2F978-1-62703-484-5_18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=221-236&author=C.+Nitsche&author=C.+D.+Kleinauthor=E.+Y.+Gong&title=Antiviral+Methods+and+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2F978-1-62703-484-5_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-62703-484-5_18%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26atitle%3DFluorimetric%2520and%2520HPLC-Based%2520Dengue%2520Virus%2520Protease%2520Assays%2520Using%2520a%2520FRET%2520Substrate%26btitle%3DAntiviral%2520Methods%2520and%2520Protocols%26aulast%3DGong%26aufirst%3DE.%2BY.%26pub%3DHumana%2520Press%26date%3D2013%26spage%3D221%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiramel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatel-Chaix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvisi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora-Rodríguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamacher-Brady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span> <span> </span><span class="NLM_article-title">Dengue Virus Inhibition of Autophagic Flux and Dependency of Viral Replication on Proteasomal Degradation of the Autophagy Receptor p62</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">8026</span>– <span class="NLM_lpage">8041</span>, <span class="refDoi"> DOI: 10.1128/JVI.00787-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FJVI.00787-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=26018155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyku7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=8026-8041&author=P.+Metzauthor=A.+Chiramelauthor=L.+Chatel-Chaixauthor=G.+Alvisiauthor=P.+Bankheadauthor=R.+Mora-Rodr%C3%ADguezauthor=G.+Longauthor=A.+Hamacher-Bradyauthor=N.+R.+Bradyauthor=R.+Bartenschlager&title=Dengue+Virus+Inhibition+of+Autophagic+Flux+and+Dependency+of+Viral+Replication+on+Proteasomal+Degradation+of+the+Autophagy+Receptor+p62&doi=10.1128%2FJVI.00787-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue virus inhibition of autophagic flux and dependency of viral replication on proteasomal degradation of the autophagy receptor p62</span></div><div class="casAuthors">Metz, Philippe; Chiramel, Abhilash; Chatel-Chaix, Laurent; Alvisi, Gualtiero; Bankhead, Peter; Mora-Rodriguez, Rodrigo; Long, Gang; Hamacher-Brady, Anne; Brady, Nathan R.; Bartenschlager, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8026-8041</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Autophagic flux involves formation of autophagosomes and their degrdn. by lysosomes.  Autophagy can either promote or restrict viral replication.  In the case of Dengue virus (DENV), several studies report that autophagy supports the viral replication cycle, and describe an increase of autophagic vesicles (AVs) following infection.  However, it is unknown how autophagic flux is altered to result in increased AVs.  To address this question and gain insight into the role of autophagy during DENV infection, we established an unbiased, image-based flow cytometry approach to quantify autophagic flux under normal growth conditions and in response to activation by nutrient deprivation or the mTOR inhibitor Torin1.  We found that DENV induced an initial activation of autophagic flux, followed by inhibition of general and specific autophagy.  Early after infection, basal and activated autophagic flux was enhanced.  However, during established replication, basal and Torin1-activated autophagic flux was blocked, while autophagic flux activated by nutrient deprivation was reduced, indicating a block to AV formation and reduced AV degrdn. capacity.  During late infection AV levels increased as a result of inefficient fusion of autophagosomes with lysosomes.  In addn., endolysosomal trafficking was suppressed, while lysosomal activities were increased.  We further detd. that DENV infection progressively reduced levels of the autophagy receptor SQSTM1/p62 via proteasomal degrdn.  Importantly, stable overexpression of p62 significantly suppressed DENV replication, suggesting a novel role for p62 as a viral restriction factor.  Overall, our findings indicate that in the course of DENV infection, autophagy shifts from a supporting to an antiviral role, which is countered by DENV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAZ45b04tNDrVg90H21EOLACvtfcHk0lii7104ns6Pww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyku7%252FO&md5=67cbffe54070d331d8057cf1565f5022</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1128%2FJVI.00787-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00787-15%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DP.%26aulast%3DChiramel%26aufirst%3DA.%26aulast%3DChatel-Chaix%26aufirst%3DL.%26aulast%3DAlvisi%26aufirst%3DG.%26aulast%3DBankhead%26aufirst%3DP.%26aulast%3DMora-Rodr%25C3%25ADguez%26aufirst%3DR.%26aulast%3DLong%26aufirst%3DG.%26aulast%3DHamacher-Brady%26aufirst%3DA.%26aulast%3DBrady%26aufirst%3DN.%2BR.%26aulast%3DBartenschlager%26aufirst%3DR.%26atitle%3DDengue%2520Virus%2520Inhibition%2520of%2520Autophagic%2520Flux%2520and%2520Dependency%2520of%2520Viral%2520Replication%2520on%2520Proteasomal%2520Degradation%2520of%2520the%2520Autophagy%2520Receptor%2520p62%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D8026%26epage%3D8041%26doi%3D10.1128%2FJVI.00787-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Nikos Kühl, Mila M. Leuthold, Mira A. M. Behnam, <span class="NLM_string-name hlFld-ContribAuthor">Christian D. Klein</span>. </span><span class="cited-content_cbyCitation_article-title">Beyond Basicity: Discovery of Nonbasic DENV-2 Protease Inhibitors with Potent Activity in Cell Culture. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4567-4587. <a href="https://doi.org/10.1021/acs.jmedchem.0c02042" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02042%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBeyond%252BBasicity%25253A%252BDiscovery%252Bof%252BNonbasic%252BDENV-2%252BProtease%252BInhibitors%252Bwith%252BPotent%252BActivity%252Bin%252BCell%252BCulture%26aulast%3DK%25C3%25BChl%26aufirst%3DNikos%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24112020%26date%3D14042021%26volume%3D64%26issue%3D8%26spage%3D4567%26epage%3D4587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tonko  Dražić</span>, <span class="hlFld-ContribAuthor ">Nikos  Kühl</span>, <span class="hlFld-ContribAuthor ">Mila M.  Leuthold</span>, <span class="hlFld-ContribAuthor ">Mira A. M.  Behnam</span>, <span class="hlFld-ContribAuthor ">Christian D.  Klein</span>. </span><span class="cited-content_cbyCitation_article-title">Efficiency Improvements and Discovery of New Substrates for a SARS-CoV-2 Main Protease FRET Assay. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>371 </em>, 247255522110206. <a href="https://doi.org/10.1177/24725552211020681" title="DOI URL">https://doi.org/10.1177/24725552211020681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/24725552211020681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F24725552211020681%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DEfficiency%252BImprovements%252Band%252BDiscovery%252Bof%252BNew%252BSubstrates%252Bfor%252Ba%252BSARS-CoV-2%252BMain%252BProtease%252BFRET%252BAssay%26aulast%3DDra%25C5%25BEi%25C4%2587%26aufirst%3DTonko%26date%3D2021%26date%3D2021%26volume%3D371%26spage%3D247255522110206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Piotr  Rejnhardt</span>, <span class="hlFld-ContribAuthor ">Marek  Daszkiewicz</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal structure and vibrational spectra of salts of 1H-pyrazole-1-carboxamidine and its protonation route. </span><span class="cited-content_cbyCitation_journal-name">Structural Chemistry</span><span> <strong>2021,</strong> <em>32 </em>
                                    (2)
                                     , 539-551. <a href="https://doi.org/10.1007/s11224-020-01671-0" title="DOI URL">https://doi.org/10.1007/s11224-020-01671-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11224-020-01671-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11224-020-01671-0%26sid%3Dliteratum%253Aachs%26jtitle%3DStructural%2520Chemistry%26atitle%3DCrystal%252Bstructure%252Band%252Bvibrational%252Bspectra%252Bof%252Bsalts%252Bof%252B1H-pyrazole-1-carboxamidine%252Band%252Bits%252Bprotonation%252Broute%26aulast%3DRejnhardt%26aufirst%3DPiotr%26date%3D2021%26date%3D2020%26volume%3D32%26issue%3D2%26spage%3D539%26epage%3D551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marcin  Skoreński</span>, <span class="hlFld-ContribAuthor ">Renata  Grzywa</span>, <span class="hlFld-ContribAuthor ">Marcin  Sieńczyk</span>. </span><span class="cited-content_cbyCitation_article-title">Viral Proteases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-9. <a href="https://doi.org/10.1007/978-3-030-21573-6_146-1" title="DOI URL">https://doi.org/10.1007/978-3-030-21573-6_146-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-21573-6_146-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-21573-6_146-1%26sid%3Dliteratum%253Aachs%26atitle%3DViral%252BProteases%26aulast%3DSkore%25C5%2584ski%26aufirst%3DMarcin%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D9%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DEncyclopedia%252Bof%252BMolecular%252BPharmacology%26aulast%3DOffermanns%26aufirst%3DStefan%26date%3D2020%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Saan  Voss</span>, <span class="hlFld-ContribAuthor ">Christoph  Nitsche</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the protease of West Nile virus. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>2 </em><a href="https://doi.org/10.1039/D1MD00080B" title="DOI URL">https://doi.org/10.1039/D1MD00080B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00080B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00080B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bthe%252Bprotease%252Bof%252BWest%252BNile%252Bvirus%26aulast%3DVoss%26aufirst%3DSaan%26date%3D2021%26date%3D2021%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Representative examples of previously reported peptidic and peptide hybrid inhibitors and their efficacy in a DENV-2 virus titer reduction assay.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Peptide backbones are marked in orange. (B) General structure of the inhibitor series presented here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of amine-, guanidine-, and urea-containing compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Pd/C, conc. HCl, H<sub>2</sub>, EtOH, 24 h; (b) Fmoc-OSu, TEA, DMF, MeCN; (c) TMP, DMF, 16 h; (d) DCM/MeOH/TMP (80:15:5 v/v); (e) piperidine, DMF; (f) phenylglycine derivative, COMU, TMP, DMF; (g) N-terminal capping group, COMU, TMP, DMF; (h) 5% TFA in DCM; (i) <i>N</i>,<i>N</i>′-di-Boc-<i>S</i>-methylisothiourea, TEA, HgCl<sub>2</sub>, THF, 24 h; (j) TFA, DCM (1:1); (k) NaOCN, acetic acid, H<sub>2</sub>O (l) <i>N</i>,<i>N</i>′-di-Boc-1<i>H</i>-pyrazole-1-carboxamidine, DMAP, DIPEA, MeOH, 48 h.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>9</b> and <b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) NaN<sub>3</sub>, DMF, 60 °C; (b) SnCl<sub>2</sub>·H<sub>2</sub>O, MeOH, 0 °C - > rt; (c) NaOH; (d) HATU, TMP, DCM/DMF, 0 °C - > rt; (e) TFA, DCM; (f) NH<sub>2</sub>OH·HCl, TMP, THF, 60 °C; (g) SnCl<sub>2</sub>·H<sub>2</sub>O, DMF, 80 °C, 72 h.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. DENV-2 reporter gene assay (DENV2proHeLa). (A) Schematic representation of the reporter construct used for co-transfection with the DENV-2 protease. (B) Relationship of inhibitory potencies at single concentration measurements in DENV2proHeLa versus virus titer reduction assay (compounds screened at their respective highest nontoxic concentration). (C) Relationship of EC<sub>50</sub> values from the virus titer reduction assay of known compounds versus EC<sub>50</sub> values derived from DENV2proHeLa. Compound identifiers and values are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf" class="ext-link">Supporting Information</a>. Graphs 2B and 2C were generated using Prism 6.01 (Graphpad Software, Inc.). Statistical parameters: 2B: <i>R</i><sup>2</sup> = 0.76, Sy.x = 13.89, <i>p</i> < 0.0001; 2C: <i>R</i><sup>2</sup> = 0.75, Sy.x = 0.24, <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Addition of compound <b>51</b> to DENV-2 protease led to concentration-dependent quenching of the fluorescence of the tryptophan residues in the substrate-binding region. The fluorescence is partially restored when aprotinin is added. NATA (<i>N</i>-acetyl-<span class="smallcaps smallerCapital">l</span>-tryptophanamide) is used as control for inner filter effects.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking studies of compounds <b>20</b> and <b>43</b> at DENV-3 protease (PDB: 3U1I). (A) Ribbon model of the protease with chosen side chains shown in stick representation. The protease domain is colored in red, and the cofactor is colored in blue. Distances between the ligand and protein atoms are shown in black. (B) Docking pose of compound <b>20</b>. (C) Docking pose of compound <b>43</b>. The protein surface in pictures B and C is colored by amino acid hydrophobicity, with light blue indicating hydrophilic and orange indicating hydrophobic regions. The ligands are represented as sticks. The positions of the specificity pockets are indicated. Figures were generated using the Chimera software 1.11.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibitory activity of selected compounds and the reference compound <b>MB-53</b> at a concentration of 12.5 μM in the DENV-2 virus titer reduction assay. Untreated infected Huh-7 cells were used as control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/medium/jm0c00413_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Dose–response curves of compounds determined in a viral titer reduction assay: concentration-dependent DENV-2 virus titer reduction and EC<sub>50</sub> values for selected compounds. (A) Dose–response curve of compound <b>20</b> (<i>R</i><sup>2</sup> = 0.97). (B) Dose–response curve of compound <b>38</b> (<i>R</i><sup>2</sup> = 0.97). (C) Dose–response curve of compound <b>43</b> (<i>R</i><sup>2</sup> = 0.93). (D) Dose–response curve of reference compound <b>MB-53</b> (<i>R</i><sup>2</sup> = 0.77).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00413/20200806/images/large/jm0c00413_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00413&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i86">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gething, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messina, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farlow, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyes, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownstein, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoen, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankoh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaenisch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wint, G. R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrar, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, S. I.</span></span> <span> </span><span class="NLM_article-title">The Global Distribution and Burden of Dengue</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>496</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/nature12060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fnature12060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=23563266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltlOnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=504-507&author=S.+Bhattauthor=P.+W.+Gethingauthor=O.+J.+Bradyauthor=J.+P.+Messinaauthor=A.+W.+Farlowauthor=C.+L.+Moyesauthor=J.+M.+Drakeauthor=J.+S.+Brownsteinauthor=A.+G.+Hoenauthor=O.+Sankohauthor=M.+F.+Myersauthor=D.+B.+Georgeauthor=T.+Jaenischauthor=G.+R.+W.+Wintauthor=C.+P.+Simmonsauthor=T.+W.+Scottauthor=J.+J.+Farrarauthor=S.+I.+Hay&title=The+Global+Distribution+and+Burden+of+Dengue&doi=10.1038%2Fnature12060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The global distribution and burden of dengue</span></div><div class="casAuthors">Bhatt, Samir; Gething, Peter W.; Brady, Oliver J.; Messina, Jane P.; Farlow, Andrew W.; Moyes, Catherine L.; Drake, John M.; Brownstein, John S.; Hoen, Anne G.; Sankoh, Osman; Myers, Monica F.; George, Dylan B.; Jaenisch, Thomas; Wint, G. R. William; Simmons, Cameron P.; Scott, Thomas W.; Farrar, Jeremy J.; Hay, Simon I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue">7446</span>),
    <span class="NLM_cas:pages">504-507</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dengue is a systemic viral infection transmitted between humans by Aedes mosquitoes.  For some patients, dengue is a life-threatening illness.  There are currently no licensed vaccines or specific therapeutics, and substantial vector control efforts have not stopped its rapid emergence and global spread.  The contemporary worldwide distribution of the risk of dengue virus infection and its public health burden are poorly known.  Here we undertake an exhaustive assembly of known records of dengue occurrence worldwide, and use a formal modeling framework to map the global distribution of dengue risk.  We then pair the resulting risk map with detailed longitudinal information from dengue cohort studies and population surfaces to infer the public health burden of dengue in 2010.  We predict dengue to be ubiquitous throughout the tropics, with local spatial variations in risk influenced strongly by rainfall, temp. and the degree of urbanization.  Using cartog. approaches, we est. there to be 390 million (95% credible interval 284-528) dengue infections per yr, of which 96 million (67-136) manifest apparently (any level of clin. or subclin. severity).  This infection total is more than three times the dengue burden est. of the World Health Organization.  Stratification of our ests. by country allows comparison with national dengue reporting, after taking into account the probability of an apparent infection being formally reported.  The most notable differences are discussed.  These new risk maps and infection ests. provide novel insights into the global, regional and national public health burden imposed by dengue.  We anticipate that they will provide a starting point for a wider discussion about the global impact of this disease and will help to guide improvements in disease control strategies using vaccine, drug and vector control methods, and in their economic evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVDkrHXysP5bVg90H21EOLACvtfcHk0ljk-RvtxH9npQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltlOnsLs%253D&md5=5b52dcfdd70064e2492f482bac1194b1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature12060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12060%26sid%3Dliteratum%253Aachs%26aulast%3DBhatt%26aufirst%3DS.%26aulast%3DGething%26aufirst%3DP.%2BW.%26aulast%3DBrady%26aufirst%3DO.%2BJ.%26aulast%3DMessina%26aufirst%3DJ.%2BP.%26aulast%3DFarlow%26aufirst%3DA.%2BW.%26aulast%3DMoyes%26aufirst%3DC.%2BL.%26aulast%3DDrake%26aufirst%3DJ.%2BM.%26aulast%3DBrownstein%26aufirst%3DJ.%2BS.%26aulast%3DHoen%26aufirst%3DA.%2BG.%26aulast%3DSankoh%26aufirst%3DO.%26aulast%3DMyers%26aufirst%3DM.%2BF.%26aulast%3DGeorge%26aufirst%3DD.%2BB.%26aulast%3DJaenisch%26aufirst%3DT.%26aulast%3DWint%26aufirst%3DG.%2BR.%2BW.%26aulast%3DSimmons%26aufirst%3DC.%2BP.%26aulast%3DScott%26aufirst%3DT.%2BW.%26aulast%3DFarrar%26aufirst%3DJ.%2BJ.%26aulast%3DHay%26aufirst%3DS.%2BI.%26atitle%3DThe%2520Global%2520Distribution%2520and%2520Burden%2520of%2520Dengue%26jtitle%3DNature%26date%3D2013%26volume%3D496%26spage%3D504%26epage%3D507%26doi%3D10.1038%2Fnature12060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardosa, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. G.</span></span> <span> </span><span class="NLM_article-title">Of Cascades and Perfect Storms: The Immunopathogenesis of Dengue Haemorrhagic Fever-Dengue Shock Syndrome (DHF/DSS)</span>. <i>Immunology & Cell Biology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1038/sj.icb.7100008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fsj.icb.7100008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17130899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhslOgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2007&pages=43-45&author=T.+Pangauthor=M.+J.+Cardosaauthor=M.+G.+Guzman&title=Of+Cascades+and+Perfect+Storms%3A+The+Immunopathogenesis+of+Dengue+Haemorrhagic+Fever-Dengue+Shock+Syndrome+%28DHF%2FDSS%29&doi=10.1038%2Fsj.icb.7100008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Of cascades and perfect storms: the immunopathogenesis of dengue haemorrhagic fever-dengue shock syndrome (DHF/DSS)</span></div><div class="casAuthors">Pang, Tikki; Cardosa, Mary Jane; Guzman, Maria G.</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-45</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past four decades has witnessed a consolidation of the original observations made in the 1970s that dengue haemorrhagic fever (DHF) and dengue shock syndrome (DSS) have an immunol. basis.  Following reinfection with a dengue virus of different serotype, severe disease is linked to high levels of antibody-enhanced viral replication early in illness which is followed by a cascade of memory T-cell activation and a 'storm' of inflammatory cytokines and other chem. mediators.  These compds. are released mainly from T cells, monocytes/macrophages and endothelial cells, and ultimately cause an increase in vascular permeability.  The consolidation of the evidence has been largely due to several important prospective sero-epidemiol. studies in areas endemic for DHF/DSS, which have shown that risk of severe disease is significantly higher in secondary dengue infections.  These advances have underscored the fact that DHF/DSS pathogenesis is a complex, multifactorial process involving cocirculation of various dengue virus serotypes and the interplay of host and viral factors that influence disease severity.  The continued search to define risk factors in susceptible populations must be combined with the new techniques of mol. virol. and innovative approaches in vaccine design to achieve the ultimate objective of developing a safe and effective vaccine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_zZX_80c847Vg90H21EOLACvtfcHk0ljk-RvtxH9npQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhslOgsLc%253D&md5=b7a4c7c5e3853e56f2405bd43a613878</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.icb.7100008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.icb.7100008%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DT.%26aulast%3DCardosa%26aufirst%3DM.%2BJ.%26aulast%3DGuzman%26aufirst%3DM.%2BG.%26atitle%3DOf%2520Cascades%2520and%2520Perfect%2520Storms%253A%2520The%2520Immunopathogenesis%2520of%2520Dengue%2520Haemorrhagic%2520Fever-Dengue%2520Shock%2520Syndrome%2520%2528DHF%252FDSS%2529%26jtitle%3DImmunology%2520%2526%2520Cell%2520Biology%26date%3D2007%26volume%3D85%26spage%3D43%26epage%3D45%26doi%3D10.1038%2Fsj.icb.7100008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halstead, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artsob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsperger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nathan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelegrino, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoksan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeling, R. W.</span></span> <span> </span><span class="NLM_article-title">Dengue: A Continuing Global Threat</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">S7</span>– <span class="NLM_lpage">S16</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fnrmicro2460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=21079655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVansb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=S7-S16&author=M.+G.+Guzmanauthor=S.+B.+Halsteadauthor=H.+Artsobauthor=P.+Buchyauthor=J.+Farrarauthor=D.+J.+Gublerauthor=E.+Hunspergerauthor=A.+Kroegerauthor=H.+S.+Margolisauthor=E.+Mart%C3%ADnezauthor=M.+B.+Nathanauthor=J.+L.+Pelegrinoauthor=C.+Simmonsauthor=S.+Yoksanauthor=R.+W.+Peeling&title=Dengue%3A+A+Continuing+Global+Threat&doi=10.1038%2Fnrmicro2460"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue: a continuing global threat</span></div><div class="casAuthors">Guzman, Maria G.; Halstead, Scott B.; Artsob, Harvey; Buchy, Philippe; Farrar, Jeremy; Gubler, Duane J.; Hunsperger, Elizabeth; Kroeger, Axel; Margolis, Harold S.; Martinez, Eric; Nathan, Michael B.; Pelegrino, Jose Luis; Simmons, Cameron; Yoksan, Sutee; Peeling, Rosanna W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12_supp</span>),
    <span class="NLM_cas:pages">S7-S16</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dengue fever and dengue haemorrhagic fever are important arthropod-borne viral diseases.  Each year, there are ∼50 million dengue infections and ∼500,000 individuals are hospitalized with dengue haemorrhagic fever, mainly in Southeast Asia, the Pacific and the Americas.  Illness is produced by any of the four dengue virus serotypes.  A global strategy aimed at increasing the capacity for surveillance and outbreak response, changing behaviors and reducing the disease burden using integrated vector management in conjunction with early and accurate diagnosis has been advocated.  Antiviral drugs and vaccines that are currently under development could also make an important contribution to dengue control in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTeD-NBdwYqbVg90H21EOLACvtfcHk0ljk-RvtxH9npQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVansb7J&md5=f11ccf80ba80a850682348ec75ddae19</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2460%26sid%3Dliteratum%253Aachs%26aulast%3DGuzman%26aufirst%3DM.%2BG.%26aulast%3DHalstead%26aufirst%3DS.%2BB.%26aulast%3DArtsob%26aufirst%3DH.%26aulast%3DBuchy%26aufirst%3DP.%26aulast%3DFarrar%26aufirst%3DJ.%26aulast%3DGubler%26aufirst%3DD.%2BJ.%26aulast%3DHunsperger%26aufirst%3DE.%26aulast%3DKroeger%26aufirst%3DA.%26aulast%3DMargolis%26aufirst%3DH.%2BS.%26aulast%3DMart%25C3%25ADnez%26aufirst%3DE.%26aulast%3DNathan%26aufirst%3DM.%2BB.%26aulast%3DPelegrino%26aufirst%3DJ.%2BL.%26aulast%3DSimmons%26aufirst%3DC.%26aulast%3DYoksan%26aufirst%3DS.%26aulast%3DPeeling%26aufirst%3DR.%2BW.%26atitle%3DDengue%253A%2520A%2520Continuing%2520Global%2520Threat%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2010%26volume%3D8%26spage%3DS7%26epage%3DS16%26doi%3D10.1038%2Fnrmicro2460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span> <i>World Health Organisation. GACVS Statement on Dengvaxia ® (CYD-TDV)</i>. <a href="https://www.who.int/vaccine_safety/committee/GACVS-StatementonDengvaxia-CYD-TDV/en/" class="extLink">https://www.who.int/vaccine_safety/committee/GACVS-StatementonDengvaxia-CYD-TDV/en/</a> (accessed Mar 31, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+World+Health+Organisation.+GACVS+Statement+on+Dengvaxia+%C2%AE+%28CYD-TDV%29.+https%3A%2F%2Fwww.who.int%2Fvaccine_safety%2Fcommittee%2FGACVS-StatementonDengvaxia-CYD-TDV%2Fen%2F+%28accessed+Mar+31%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DWorld%2520Health%2520Organisation.%2520GACVS%2520Statement%2520on%2520Dengvaxia%2520%25C2%25AE%2520%2528CYD-TDV%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span> <i>World Health Organisation.
Revised SAGE Recommendation on Use of
Dengue Vaccine</i>. <a href="https://www.who.int/immunization/diseases/dengue/revised_SAGE_recommendations_dengue_vaccines_apr2018/en/" class="extLink">https://www.who.int/immunization/diseases/dengue/revised_SAGE_recommendations_dengue_vaccines_apr2018/en/</a> (accessed Mar 31, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+World+Health+Organisation.%0ARevised+SAGE+Recommendation+on+Use+of%0ADengue+Vaccine.+https%3A%2F%2Fwww.who.int%2Fimmunization%2Fdiseases%2Fdengue%2Frevised_SAGE_recommendations_dengue_vaccines_apr2018%2Fen%2F+%28accessed+Mar+31%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DWorld%2520Health%2520Organisation.%250ARevised%2520SAGE%2520Recommendation%2520on%2520Use%2520of%250ADengue%2520Vaccine" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Normile, D.</span></span> <span> </span><span class="NLM_article-title">Safety Concerns Derail Dengue Vaccination Program</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">1514</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1126/science.358.6370.1514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1126%2Fscience.358.6370.1514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=29269451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVGjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=1514-1515&author=D.+Normile&title=Safety+Concerns+Derail+Dengue+Vaccination+Program&doi=10.1126%2Fscience.358.6370.1514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Safety concerns derail dengue vaccination program</span></div><div class="casAuthors">Normile, Dennis</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6370</span>),
    <span class="NLM_cas:pages">1514-1515</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpRDA_BKZRLVg90H21EOLACvtfcHk0liKae8tQMky4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVGjtQ%253D%253D&md5=a83e5957db20c0945ac434269c0b34b5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.358.6370.1514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.358.6370.1514%26sid%3Dliteratum%253Aachs%26aulast%3DNormile%26aufirst%3DD.%26atitle%3DSafety%2520Concerns%2520Derail%2520Dengue%2520Vaccination%2520Program%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3D1514%26epage%3D1515%26doi%3D10.1126%2Fscience.358.6370.1514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suthar, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, M.,  Jr.</span></span> <span> </span><span class="NLM_article-title">West Nile Virus Infection and Immunity</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1038/nrmicro2950</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Fnrmicro2950" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=23321534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslGqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=115-128&author=M.+S.+Sutharauthor=M.+S.+Diamondauthor=M.+Gale&title=West+Nile+Virus+Infection+and+Immunity&doi=10.1038%2Fnrmicro2950"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">West Nile virus infection and immunity</span></div><div class="casAuthors">Suthar, Mehul S.; Diamond, Michael S.; Gale, Michael, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-128</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  West Nile virus (WNV) is an emerging neurotropic flavivirus that is transmitted to humans through the bite of an infected mosquito.  WNV has disseminated broadly in the Western hemisphere and now poses a significant public health risk.  The continuing spread of WNV, combined with the lack of specific therapeutics or vaccines to combat or prevent infection, imparts a pressing need to identify the viral and host processes that control the outcome of and immunity to WNV infection.  Here, we provide an overview of recent research that has revealed the virus-host interface controlling WNV infection and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzAOghp0axGbVg90H21EOLACvtfcHk0liD7HtdtSvIBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslGqtA%253D%253D&md5=d6a1b96007511a54045764d79fdb22aa</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2950%26sid%3Dliteratum%253Aachs%26aulast%3DSuthar%26aufirst%3DM.%2BS.%26aulast%3DDiamond%26aufirst%3DM.%2BS.%26aulast%3DGale%26aufirst%3DM.%26atitle%3DWest%2520Nile%2520Virus%2520Infection%2520and%2520Immunity%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2013%26volume%3D11%26spage%3D115%26epage%3D128%26doi%3D10.1038%2Fnrmicro2950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCauley, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudd, M. T.</span></span> <span> </span><span class="NLM_article-title">Hepatitis C Virus NS3/4A Protease Inhibitors</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2016.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.coph.2016.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=27544488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ygurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=84-92&author=J.+A.+McCauleyauthor=M.+T.+Rudd&title=Hepatitis+C+Virus+NS3%2F4A+Protease+Inhibitors&doi=10.1016%2Fj.coph.2016.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis C virus NS3/4a protease inhibitors</span></div><div class="casAuthors">McCauley, John A.; Rudd, Michael T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">84-92</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) infection is a major health issue around the world and HCV NS3/4a protease inhibitors have been the focus of intensive research for the past 20 years.  From the first identification of substrate-derived peptide inhibitors to the complex, macrocyclic compds., including paritaprevir and grazoprevir, that are currently available, the field has used structure-based design to confront the issues of potency, resistance and pharmacokinetics.  Numerous breakthrough structures from a multitude of companies have led to compds. that are now key components of combination therapies with cure rates of >90%.  Herein, we detail the compds. that have advanced to clin. trials including their design and their impact on the NS3/4a protease field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecX2r0UguZLVg90H21EOLACvtfcHk0liD7HtdtSvIBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ygurrJ&md5=1d7d626d8c10ab9d619e8caa14a3f9a5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2016.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2016.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DMcCauley%26aufirst%3DJ.%2BA.%26aulast%3DRudd%26aufirst%3DM.%2BT.%26atitle%3DHepatitis%2520C%2520Virus%2520NS3%252F4A%2520Protease%2520Inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2016%26volume%3D30%26spage%3D84%26epage%3D92%26doi%3D10.1016%2Fj.coph.2016.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span> <span> </span><span class="NLM_article-title">Anti-HIV Drugs: 25 Compounds Approved within 25 Years after the Discovery of HIV</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">320</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2008.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.ijantimicag.2008.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19108994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=307-320&author=E.+De%0AClercq&title=Anti-HIV+Drugs%3A+25+Compounds+Approved+within+25+Years+after+the+Discovery+of+HIV&doi=10.1016%2Fj.ijantimicag.2008.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV drugs: 25 compounds approved within 25 years after the discovery of HIV</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-320</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In 2008, 25 years after the human immunodeficiency virus (HIV) was discovered as the then tentative etiol. agent of acquired immune deficiency syndrome (AIDS), exactly 25 anti-HIV compds. have been formally approved for clin. use in the treatment of AIDS.  These compds. fall into six categories: nucleoside reverse transcriptase inhibitors (NRTIs: zidovudine, didanosine, zalcitabine, stavudine, lamivudine, abacavir and emtricitabine); nucleotide reverse transcriptase inhibitors (NtRTIs: tenofovir); non-nucleoside reverse transcriptase inhibitors (NNRTIs: nevirapine, delavirdine, efavirenz and etravirine); protease inhibitors (PIs: saquinavir, ritonavir, indinavir, nelfinavir, amprenavir, lopinavir, atazanavir, fosamprenavir, tipranavir and darunavir); cell entry inhibitors [fusion inhibitors (FIs: enfuvirtide) and co-receptor inhibitors (CRIs: maraviroc)]; and integrase inhibitors (INIs: raltegravir).  These compds. should be used in drug combination regimens to achieve the highest possible benefit, tolerability and compliance and to diminish the risk of resistance development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsT6dqAjvKIrVg90H21EOLACvtfcHk0liD7HtdtSvIBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D&md5=6d8263455e9c912d353ef2ea230a5e15</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2008.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2008.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DAnti-HIV%2520Drugs%253A%252025%2520Compounds%2520Approved%2520within%252025%2520Years%2520after%2520the%2520Discovery%2520of%2520HIV%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2009%26volume%3D33%26spage%3D307%26epage%3D320%26doi%3D10.1016%2Fj.ijantimicag.2008.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.-Y.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of Dengue Virus Enzymes: Towards Rational Design of Therapeutics</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.antiviral.2012.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=22995600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12gtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2012&pages=115-126&author=C.+G.+Nobleauthor=P.-Y.+Shi&title=Structural+Biology+of+Dengue+Virus+Enzymes%3A+Towards+Rational+Design+of+Therapeutics&doi=10.1016%2Fj.antiviral.2012.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of dengue virus enzymes: Towards rational design of therapeutics</span></div><div class="casAuthors">Noble, Christian G.; Shi, Pei-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-126</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Development of anti-dengue therapy represents an urgent un-met medical need.  Towards antiviral therapy, recent advances in crystal structures of DENV enzymes have led to the possibility of structure-based rational design of inhibitors for anti-dengue therapy.  These include (i) the structure of the active' form of the DENV protease in complex with a peptide substrate; (ii) the structure of DENV methyltransferase bound to an inhibitor that selectively suppresses viral methyltransferase, but not human methyltransferases; (iii) the structure of DENV RNA-dependent RNA polymerase in complex with a small-mol. compd.  This review summarizes the structural biol. of these three key enzymes (protease, methyltransferase, and polymerase) that are essential for DENV replication.  The new structural information has provided new avenues for development of anti-dengue therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG75WRQ0Bn7LVg90H21EOLACvtfcHk0lj-_MIT5tTZjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12gtLfF&md5=42f5387c37a813b1cdf9737ccc213d54</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DC.%2BG.%26aulast%3DShi%26aufirst%3DP.-Y.%26atitle%3DStructural%2520Biology%2520of%2520Dengue%2520Virus%2520Enzymes%253A%2520Towards%2520Rational%2520Design%2520of%2520Therapeutics%26jtitle%3DAntiviral%2520Res.%26date%3D2012%26volume%3D96%26spage%3D115%26epage%3D126%26doi%3D10.1016%2Fj.antiviral.2012.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Biochemistry and Medicinal Chemistry of the Dengue Virus Protease</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">11348</span>– <span class="NLM_lpage">11381</span>, <span class="refDoi"> DOI: 10.1021/cr500233q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500233q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Cns77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=11348-11381&author=C.+Nitscheauthor=S.+Hollowayauthor=T.+Schirmeisterauthor=C.+D.+Klein&title=Biochemistry+and+Medicinal+Chemistry+of+the+Dengue+Virus+Protease&doi=10.1021%2Fcr500233q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemistry and Medicinal Chemistry of the Dengue Virus Protease</span></div><div class="casAuthors">Nitsche, Christoph; Holloway, Steven; Schirmeister, Tanja; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">11348-11381</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the present work, we will review the protease of dengue virus from several directions, with a particular focus on those aspects related to medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVn6Afm5NC07Vg90H21EOLACvtfcHk0lj-_MIT5tTZjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Cns77P&md5=5f4764c07d55dd760f95305019ddf84c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcr500233q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500233q%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DHolloway%26aufirst%3DS.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DBiochemistry%2520and%2520Medicinal%2520Chemistry%2520of%2520the%2520Dengue%2520Virus%2520Protease%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D11348%26epage%3D11381%26doi%3D10.1021%2Fcr500233q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumanut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tully, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiricek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priestle, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, S. G.</span></span> <span> </span><span class="NLM_article-title">Functional Profiling of Recombinant NS3 Proteases from All Four Serotypes of Dengue Virus Using Tetrapeptide and Octapeptide Substrate Libraries</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">28766</span>– <span class="NLM_lpage">28774</span>, <span class="refDoi"> DOI: 10.1074/jbc.M500588200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.M500588200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=15932883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvFSnt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=28766-28774&author=J.+Liauthor=S.+P.+Limauthor=D.+Beerauthor=V.+Patelauthor=D.+Wenauthor=C.+Tumanutauthor=D.+C.+Tullyauthor=J.+A.+Williamsauthor=J.+Jiricekauthor=J.+P.+Priestleauthor=J.+L.+Harrisauthor=S.+G.+Vasudevan&title=Functional+Profiling+of+Recombinant+NS3+Proteases+from+All+Four+Serotypes+of+Dengue+Virus+Using+Tetrapeptide+and+Octapeptide+Substrate+Libraries&doi=10.1074%2Fjbc.M500588200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Profiling of Recombinant NS3 Proteases from All Four Serotypes of Dengue Virus Using Tetrapeptide and Octapeptide Substrate Libraries</span></div><div class="casAuthors">Li, Jun; Lim, Siew Pheng; Beer, David; Patel, Viral; Wen, Daying; Tumanut, Christine; Tully, David C.; Williams, Jennifer A.; Jiricek, Jan; Priestle, John P.; Harris, Jennifer L.; Vasudevan, Subhash G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">28766-28774</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Regulated proteolysis by the two-component NS2B/NS3 protease of dengue virus is essential for virus replication and the maturation of infectious virions.  The functional similarity between the NS2B/NS3 proteases from the four genetically and antigenically distinct serotypes was addressed by characterizing the differences in their substrate specificity using tetrapeptide and octapeptide libraries in a positional scanning format, each contg. 130,321 substrates.  The proteases from different serotypes were shown to be functionally homologous based on the similarity of their substrate cleavage preferences.  A strong preference for basic amino acid residues (Arg/Lys) at the P1 positions was obsd., whereas the preferences for the P2-4 sites were in the order of Arg > Thr > Gln/Asn/Lys for P2, Lys > Arg > Asn for P3, and Nle > Leu > Lys > Xaa for P4.  The prime site substrate specificity was for small and polar amino acids in P1' and P3'.  In contrast, the P2' and P4' substrate positions showed minimal activity.  The influence of the P2 and P3 amino acids on ground state binding and the P4 position for transition state stabilization was identified through single substrate kinetics with optimal and suboptimal substrate sequences.  The specificities obsd. for dengue NS2B/NS3 have features in common with the physiol. cleavage sites in the dengue polyprotein; however, all sites reveal previously unrecognized suboptimal sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH2hcNat1IwrVg90H21EOLACvtfcHk0lj-_MIT5tTZjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvFSnt7w%253D&md5=7e7bd5ab26af480759378090e7e73e66</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M500588200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M500588200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLim%26aufirst%3DS.%2BP.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DTumanut%26aufirst%3DC.%26aulast%3DTully%26aufirst%3DD.%2BC.%26aulast%3DWilliams%26aufirst%3DJ.%2BA.%26aulast%3DJiricek%26aufirst%3DJ.%26aulast%3DPriestle%26aufirst%3DJ.%2BP.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DVasudevan%26aufirst%3DS.%2BG.%26atitle%3DFunctional%2520Profiling%2520of%2520Recombinant%2520NS3%2520Proteases%2520from%2520All%2520Four%2520Serotypes%2520of%2520Dengue%2520Virus%2520Using%2520Tetrapeptide%2520and%2520Octapeptide%2520Substrate%2520Libraries%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D28766%26epage%3D28774%26doi%3D10.1074%2Fjbc.M500588200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeyaraj, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngew, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span> <span> </span><span class="NLM_article-title">Peptide Inhibitors of Dengue Virus NS3 Protease. Part 1: Warhead</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.09.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmcl.2005.09.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=16246553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Citr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=36-39&author=Z.+Yinauthor=S.+J.+Patelauthor=W.-L.+Wangauthor=G.+Wangauthor=W.-L.+Chanauthor=K.+R.+R.+Raoauthor=J.+Alamauthor=D.+A.+Jeyarajauthor=X.+Ngewauthor=V.+Patelauthor=D.+Beerauthor=S.+P.+Limauthor=S.+G.+Vasudevanauthor=T.+H.+Keller&title=Peptide+Inhibitors+of+Dengue+Virus+NS3+Protease.+Part+1%3A+Warhead&doi=10.1016%2Fj.bmcl.2005.09.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide inhibitors of dengue virus NS3 protease. Part 1: Warhead</span></div><div class="casAuthors">Yin, Zheng; Patel, Sejal J.; Wang, Wei-Ling; Wang, Gang; Chan, Wai-Ling; Rao, K. R. Ranga; Alam, Jenefer; Jeyaraj, Duraiswamy A.; Ngew, Xinyi; Patel, Viral; Beer, David; Lim, Siew Pheng; Vasudevan, Subhash G.; Keller, Thomas H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-39</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Substrate-based tetrapeptide inhibitors with various warheads were designed, synthesized, and evaluated against the Dengue virus NS3 protease.  Effective inhibition was achieved by peptide inhibitors with electrophilic warheads such as aldehyde, trifluoromethyl ketone, and boronic acid.  Boronic acid Bz-Nle-Lys-Arg-Arg-B(OH)2 has the highest affinity, exhibiting a Ki of 43 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZaCOf0CvDFrVg90H21EOLACvtfcHk0lgR6hhJW-xGlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Citr3M&md5=1ac1823a7945aa3d9c7e6edca23ed9f0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.09.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.09.062%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DPatel%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DW.-L.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DChan%26aufirst%3DW.-L.%26aulast%3DRao%26aufirst%3DK.%2BR.%2BR.%26aulast%3DAlam%26aufirst%3DJ.%26aulast%3DJeyaraj%26aufirst%3DD.%2BA.%26aulast%3DNgew%26aufirst%3DX.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%2BP.%26aulast%3DVasudevan%26aufirst%3DS.%2BG.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26atitle%3DPeptide%2520Inhibitors%2520of%2520Dengue%2520Virus%2520NS3%2520Protease.%2520Part%25201%253A%2520Warhead%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D36%26epage%3D39%26doi%3D10.1016%2Fj.bmcl.2005.09.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigel, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Peptide–Boronic Acid Inhibitors of Flaviviral Proteases: Medicinal Chemistry and Structural Biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01021</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01021" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGrtLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=511-516&author=C.+Nitscheauthor=L.+Zhangauthor=L.+F.+Weigelauthor=J.+Schilzauthor=D.+Grafauthor=R.+Bartenschlagerauthor=R.+Hilgenfeldauthor=C.+D.+Klein&title=Peptide%E2%80%93Boronic+Acid+Inhibitors+of+Flaviviral+Proteases%3A+Medicinal+Chemistry+and+Structural+Biology&doi=10.1021%2Facs.jmedchem.6b01021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide-Boronic Acid Inhibitors of Flaviviral Proteases: Medicinal Chemistry and Structural Biology</span></div><div class="casAuthors">Nitsche, Christoph; Zhang, Linlin; Weigel, Lena F.; Schilz, Jonas; Graf, Dominik; Bartenschlager, Ralf; Hilgenfeld, Rolf; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">511-516</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A thousand-fold affinity gain is achieved by introduction of a C-terminal boronic acid moiety into dipeptidic inhibitors of the Zika, West Nile and dengue virus proteases.  The resulting compds. have Ki values in the two-digit nanomolar range, are not cytotoxic, and inhibit virus replication.  Structure-activity relationships and a high resoln. x-ray co-crystal structure with West Nile virus protease provide a basis for the design of optimized covalent-reversible inhibitors aimed at emerging flaviviral pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT1SPIPsFvFLVg90H21EOLACvtfcHk0lgR6hhJW-xGlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGrtLvE&md5=a9671140deb2c9014ae8e1429cb5074f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01021%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWeigel%26aufirst%3DL.%2BF.%26aulast%3DSchilz%26aufirst%3DJ.%26aulast%3DGraf%26aufirst%3DD.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DPeptide%25E2%2580%2593Boronic%2520Acid%2520Inhibitors%2520of%2520Flaviviral%2520Proteases%253A%2520Medicinal%2520Chemistry%2520and%2520Structural%2520Biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D511%26epage%3D516%26doi%3D10.1021%2Facs.jmedchem.6b01021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotos, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of Nanomolar Dengue and West Nile Virus Protease Inhibitors Containing a 4-Benzyloxyphenylglycine Residue</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9354</span>– <span class="NLM_lpage">9370</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01441</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01441" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9354-9370&author=M.+A.+M.+Behnamauthor=D.+Grafauthor=R.+Bartenschlagerauthor=D.+P.+Zlotosauthor=C.+D.+Klein&title=Discovery+of+Nanomolar+Dengue+and+West+Nile+Virus+Protease+Inhibitors+Containing+a+4-Benzyloxyphenylglycine+Residue&doi=10.1021%2Facs.jmedchem.5b01441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Nanomolar Dengue and West Nile Virus Protease Inhibitors Containing a 4-Benzyloxyphenylglycine Residue</span></div><div class="casAuthors">Behnam, Mira A. M.; Graf, Dominik; Bartenschlager, Ralf; Zlotos, Darius P.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9354-9370</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dengue virus (DENV) and West Nile Virus (WNV) NS2B-NS3 proteases are attractive targets for the development of dual-acting therapeutics against these arboviral pathogens.  We present the synthesis and extensive biol. evaluation of inhibitors that contain benzyl ethers of 4-hydroxyphenylglycine as non-natural peptidic building blocks synthesized via a copper-complex intermediate.  A three-step optimization strategy, beginning with fragment growth of the C-terminal 4-hydroxyphenylglycine to the benzyloxy ether, followed by C- and N-terminal optimization, and finally fragment merging generated compds. with in vitro affinities in the low nanomolar range.  The most promising deriv. reached Ki values of 12 nM at the DENV-2 and 39 nM at the WNV proteases.  Several of the newly discovered protease inhibitors yielded a significant redn. of dengue and West Nile virus titers in cell-based assays of virus replication, with an EC50 value of 3.4 μM at DENV-2 and 15.5 μM at WNV for the most active analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreZA17Sr9XobVg90H21EOLACvtfcHk0lgR6hhJW-xGlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7nK&md5=056ebd78d73cce7686a80752c69d3791</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01441%26sid%3Dliteratum%253Aachs%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DGraf%26aufirst%3DD.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DZlotos%26aufirst%3DD.%2BP.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DDiscovery%2520of%2520Nanomolar%2520Dengue%2520and%2520West%2520Nile%2520Virus%2520Protease%2520Inhibitors%2520Containing%2520a%25204-Benzyloxyphenylglycine%2520Residue%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9354%26epage%3D9370%26doi%3D10.1021%2Facs.jmedchem.5b01441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bastos
Lima, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Sherif, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vechi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Dual Inhibitors of the Dengue and West Nile Virus NS2B–NS3 Proteases: Synthesis, Biological Evaluation and Docking Studies of Novel Peptide-Hybrids</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5748</span>– <span class="NLM_lpage">5755</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmc.2015.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=26233795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2ks7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=5748-5755&author=A.+Bastos%0ALimaauthor=M.+A.+M.+Behnamauthor=Y.+El+Sherifauthor=C.+Nitscheauthor=S.+M.+Vechiauthor=C.+D.+Klein&title=Dual+Inhibitors+of+the+Dengue+and+West+Nile+Virus+NS2B%E2%80%93NS3+Proteases%3A+Synthesis%2C+Biological+Evaluation+and+Docking+Studies+of+Novel+Peptide-Hybrids&doi=10.1016%2Fj.bmc.2015.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Dual inhibitors of the dengue and West Nile virus NS2B-NS3 proteases: Synthesis, biological evaluation and docking studies of novel peptide-hybrids</span></div><div class="casAuthors">Bastos Lima, Allan; Behnam, Mira A. M.; El Sherif, Yasmin; Nitsche, Christoph; Vechi, Sergio M.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5748-5755</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dengue virus (DENV) and West Nile virus (WNV) are mosquito-borne arboviruses responsible for causing acute systemic diseases and severe health conditions in humans.  The discovery of therapies capable to prevent infections or treat infected individuals remains an important challenge, since no vaccine or specific efficient treatment could be developed so far.  In this context, we present herein the synthesis, characterization, biol. evaluation and docking studies of novel peptide-hybrids based on 2,4-thiazolidinedione scaffolds contg. non-polar groups.  The most promising compd. has an IC50 of 0.75 μM against WNV protease, which represents a seventyfold improvement in activity compared to our previously reported compds.  Exptl. results and docking studies are in agreement with the hypothesis that a non-polar group in the scaffold is important to obtain interactions between the inhibitors and a hydrophobic pocket in the substrate recognition region of the DENV and WNV NS2B-NS3 serine proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxjuAszpkdcrVg90H21EOLACvtfcHk0lgjIXJr8eZlcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2ks7nI&md5=69934d8829047f4e0f7a4951969f7e25</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DBastos%2BLima%26aufirst%3DA.%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DEl%2BSherif%26aufirst%3DY.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DVechi%26aufirst%3DS.%2BM.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DDual%2520Inhibitors%2520of%2520the%2520Dengue%2520and%2520West%2520Nile%2520Virus%2520NS2B%25E2%2580%2593NS3%2520Proteases%253A%2520Synthesis%252C%2520Biological%2520Evaluation%2520and%2520Docking%2520Studies%2520of%2520Novel%2520Peptide-Hybrids%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D5748%26epage%3D5755%26doi%3D10.1016%2Fj.bmc.2015.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and Computational Approaches to Estimate Solubility and Permeability in Drug Discovery and Development Settings</span>. <i>Adv. Drug Delivery Rev</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+Computational+Approaches+to+Estimate+Solubility+and+Permeability+in+Drug+Discovery+and+Development+Settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520Computational%2520Approaches%2520to%2520Estimate%2520Solubility%2520and%2520Permeability%2520in%2520Drug%2520Discovery%2520and%2520Development%2520Settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev%26date%3D1997%26volume%3D46%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rayan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldblum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Understanding Drug-Likeness</span>. <i>Wiley Interdiscip. Rev.: Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1002/wcms.52</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fwcms.52" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFWrtrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=760-781&author=O.+Ursuauthor=A.+Rayanauthor=A.+Goldblumauthor=T.+I.+Oprea&title=Understanding+Drug-Likeness&doi=10.1002%2Fwcms.52"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding drug-likeness</span></div><div class="casAuthors">Ursu, Oleg; Rayan, Anwar; Goldblum, Amiram; Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: Computational Molecular Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">760-781</span>CODEN:
                <span class="NLM_cas:coden">WIRCAH</span>;
        ISSN:<span class="NLM_cas:issn">1759-0884</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  "Drug-likeness", a qual. property of chems. assigned by experts committee vote, is wedely integrated into the early stages of lead and drug discovery.  Its conceptual evolution paralleled work related to Pfizer's "rule of five" and lead-likeness, and is placed within this framework.  The discrimination between "drugs" (represented by a collection of pharmaceutically relevant small mols., some of which are marketed drugs) and "nondrugs" (typically, chem. reagents) is possible using a wide variety of statistical tools and chem. descriptor systems.  Here we summarize 18 papers focused on drug-likeness, and provide a comprehensive overview of progress in the field.  Tools that est. drug-likeness are valuable in the early stages of lead discovery, and can be used to filter out compds. with undersirable properties from screening libraries and to prioritize hits from primary screens.  As the goal is, most often, to develop orally available drugs, it is also useful to optimize drug-like pharmacokinetic properties.  We examine tools that evaluate drug-likeness and some of their shortcomings, challenges facing these tools, and address the following issues: What is the definition of drug-likeness and how can it be utilized to reduce attrition rate in drug discovery.  How difficult is it to distinguish drugs from nondrugs.  Are nondrug datasets reliable.  Can we est. oral drug-likeness.  We discuss a drug-like filter and recent advances in the prediction of oral drug-likeness.  The heuristic aspect of drug-likeness is also addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeTi02F5W647Vg90H21EOLACvtfcHk0lgjIXJr8eZlcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFWrtrbO&md5=3cd46a7a7a082d6d3730e6bbf16712d2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fwcms.52&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.52%26sid%3Dliteratum%253Aachs%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DRayan%26aufirst%3DA.%26aulast%3DGoldblum%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DUnderstanding%2520Drug-Likeness%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%253A%2520Comput.%2520Mol.%2520Sci.%26date%3D2011%26volume%3D1%26spage%3D760%26epage%3D781%26doi%3D10.1002%2Fwcms.52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vechi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">C-Terminal Residue Optimization and Fragment Merging: Discovery of a Potent Peptide-Hybrid Inhibitor of Dengue Protease</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1037</span>– <span class="NLM_lpage">1042</span>, <span class="refDoi"> DOI: 10.1021/ml500245v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500245v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1037-1042&author=M.+A.+M.+Behnamauthor=C.+Nitscheauthor=S.+M.+Vechiauthor=C.+D.+Klein&title=C-Terminal+Residue+Optimization+and+Fragment+Merging%3A+Discovery+of+a+Potent+Peptide-Hybrid+Inhibitor+of+Dengue+Protease&doi=10.1021%2Fml500245v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">C-Terminal Residue Optimization and Fragment Merging: Discovery of a Potent Peptide-Hybrid Inhibitor of Dengue Protease</span></div><div class="casAuthors">Behnam, Mira A. M.; Nitsche, Christoph; Vechi, Sergio M.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1037-1042</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dengue virus protease is a promising target for the development of antiviral drugs.  We describe here a two-step rational optimization that led to the discovery of the potent inhibitor 35 with nanomolar binding affinity at dengue protease serotype 2 (IC50 = 0.6 μM, Ki = 0.4 μM).  First, a large no. of natural and non-natural amino acids were screened at the C-terminal position of the previously reported, canonical peptide sequence (Cap-Arg-Lys-Nle-NH2).  Compared to the ref. compd. 1 (Bz-Arg-Lys-Nle-NH2, IC50 = 13.3 μM), a 4-fold higher inhibitory potential was obsd. with the incorporation of a C-terminal phenylglycine (compd. 9, IC50 = 3.3 μM).  Second, we applied fragment merging of 9 with the previously reported thiazolidinedione peptide hybrid 33 (IC50 = 2.5 μM).  This approach led to the fusion of two inhibitor-fragments with micromolar affinity into a 20-fold more potent, competitive inhibitor of dengue protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru7uUNG4dKDbVg90H21EOLACvtfcHk0ljxeWtP7w4mTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbrI&md5=ff1a6a4a5c8b5427dd5d6555a45976ed</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fml500245v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500245v%26sid%3Dliteratum%253Aachs%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DVechi%26aufirst%3DS.%2BM.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DC-Terminal%2520Residue%2520Optimization%2520and%2520Fragment%2520Merging%253A%2520Discovery%2520of%2520a%2520Potent%2520Peptide-Hybrid%2520Inhibitor%2520of%2520Dengue%2520Protease%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1037%26epage%3D1042%26doi%3D10.1021%2Fml500245v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuiper, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellmon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medapureddy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muzzarelli, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amblard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovari, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinazi, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovari, L. C.</span></span> <span> </span><span class="NLM_article-title">Increased Activity of Unlinked Zika Virus NS2B/NS3 Protease Compared to Linked Zika Virus Protease</span>. <i>Biochem. Biophys. Res. CommunBiochemical and Biophysical Research Communications</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">673</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2017.03.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bbrc.2017.03.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=28342876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVajsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2017&pages=668-673&author=B.+D.+Kuiperauthor=K.+Slaterauthor=N.+Spellmonauthor=J.+Holcombauthor=P.+Medapureddyauthor=K.+M.+Muzzarelliauthor=Z.+Yangauthor=R.+Ovadiaauthor=F.+Amblardauthor=I.+A.+Kovariauthor=R.+F.+Schinaziauthor=L.+C.+Kovari&title=Increased+Activity+of+Unlinked+Zika+Virus+NS2B%2FNS3+Protease+Compared+to+Linked+Zika+Virus+Protease&doi=10.1016%2Fj.bbrc.2017.03.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Increased activity of unlinked Zika virus NS2B/NS3 protease compared to linked Zika virus protease</span></div><div class="casAuthors">Kuiper, Benjamin D.; Slater, Kristin; Spellmon, Nicholas; Holcomb, Joshua; Medapureddy, Prasanna; Muzzarelli, Kendall M.; Yang, Zhe; Ovadia, Reuben; Amblard, Franck; Kovari, Iulia A.; Schinazi, Raymond F.; Kovari, Ladislau C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">668-673</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Zika virus (ZIKV) is a flavivirus spread by daytime-active Aedes spp. mosquitoes such as A. aegypti and A. albopictus.  Previously thought to be a mild infection, the latest ZIKV outbreak in the Americas is causally assocd. with more severe symptoms as well as severe birth defects, such as microcephaly.  Currently no vaccine or antiviral exists.  However, recent progress has demonstrated the viral NS2B/NS3 protease may be a suitable target for the development of small-mol. antiviral agents.  To better understand the ZIKV protease, we expressed, purified, and characterized unlinked and linked NS2B/NS3 protease corresponding to an isolate from the recent outbreak in Puerto Rico.  Unlinked ZIKV protease is more active and binds substrate with greater affinity than linked ZIKV protease.  Therefore, we propose that unlinked ZIKV protease be used when evaluating or designing ZIKV protease inhibitors.  Addnl., potent inhibitors of related viral proteases, like West Nile Virus and Dengue virus, may serve as advanced starting points to identify and develop ZIKV protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS3UjrJ1aaW7Vg90H21EOLACvtfcHk0ljxeWtP7w4mTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVajsr4%253D&md5=09a78e1627358550764961eb7d93ed26</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2017.03.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2017.03.108%26sid%3Dliteratum%253Aachs%26aulast%3DKuiper%26aufirst%3DB.%2BD.%26aulast%3DSlater%26aufirst%3DK.%26aulast%3DSpellmon%26aufirst%3DN.%26aulast%3DHolcomb%26aufirst%3DJ.%26aulast%3DMedapureddy%26aufirst%3DP.%26aulast%3DMuzzarelli%26aufirst%3DK.%2BM.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DOvadia%26aufirst%3DR.%26aulast%3DAmblard%26aufirst%3DF.%26aulast%3DKovari%26aufirst%3DI.%2BA.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26aulast%3DKovari%26aufirst%3DL.%2BC.%26atitle%3DIncreased%2520Activity%2520of%2520Unlinked%2520Zika%2520Virus%2520NS2B%252FNS3%2520Protease%2520Compared%2520to%2520Linked%2520Zika%2520Virus%2520Protease%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520CommunBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26date%3D2017%26volume%3D492%26spage%3D668%26epage%3D673%26doi%3D10.1016%2Fj.bbrc.2017.03.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wünsch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fries, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwick, A.</span></span> <span> </span><span class="NLM_article-title">Beiträge zur Peptidsynthese, V. Darstellung und Peptidsynthetische Verwendung von O-Benzyl-L-Tyrosin</span>. <i>Chem. Ber.</i> <span class="NLM_year" style="font-weight: bold;">1958</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1002/cber.19580910312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fcber.19580910312" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1958&pages=542-547&author=E.+W%C3%BCnschauthor=G.+Friesauthor=A.+Zwick&title=Beitr%C3%A4ge+zur+Peptidsynthese%2C+V.+Darstellung+und+Peptidsynthetische+Verwendung+von+O-Benzyl-L-Tyrosin&doi=10.1002%2Fcber.19580910312"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcber.19580910312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.19580910312%26sid%3Dliteratum%253Aachs%26aulast%3DW%25C3%25BCnsch%26aufirst%3DE.%26aulast%3DFries%26aufirst%3DG.%26aulast%3DZwick%26aufirst%3DA.%26atitle%3DBeitr%25C3%25A4ge%2520zur%2520Peptidsynthese%252C%2520V.%2520Darstellung%2520und%2520Peptidsynthetische%2520Verwendung%2520von%2520O-Benzyl-L-Tyrosin%26jtitle%3DChem.%2520Ber.%26date%3D1958%26volume%3D91%26spage%3D542%26epage%3D547%26doi%3D10.1002%2Fcber.19580910312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dener, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantauzzi, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kshirsagar, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, A. B.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Syntheses of Fmoc-Protected Non-Proteogenic Amino Acids: Useful Building Blocks for Combinatorial Libraries</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1021/op010204f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/op010204f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjvFOhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=445-449&author=J.+M.+Denerauthor=P.+P.+Fantauzziauthor=T.+A.+Kshirsagarauthor=D.+E.+Kellyauthor=A.+B.+Wolfe&title=Large-Scale+Syntheses+of+Fmoc-Protected+Non-Proteogenic+Amino+Acids%3A+Useful+Building+Blocks+for+Combinatorial+Libraries&doi=10.1021%2Fop010204f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale syntheses of Fmoc-protected non-proteogenic amino acids: useful building blocks for combinatorial libraries</span></div><div class="casAuthors">Dener, Jeffrey M.; Fantauzzi, Pascal P.; Kshirsagar, Tushar A.; Kelly, Daphne E.; Wolfe, Aaron B.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">445-449</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Convenient and reliable large-scale procedures for the protection of various amino acids with N-(9-fluorenylmethoxycarbonyl)oxysuccinimide (Fmoc-OSu) are described.  Com. available 4-aminomethylbenzoic acid and trans-4-(aminomethyl)cyclohexanecarboxylic acid were converted into their corresponding Fmoc-derivs. in excellent yields without the need for an extractive workup.  In addn., Fmoc-cis-β-amino acids were also prepd., employing a [2 + 2]-cycloaddn. strategy between a cyclic olefin and N-chlorosulfonyl isocyanate (CSI).  The resulting N-chlorosulfonyl β-lactams were reduced to the parent β-lactams with sodium sulfite and then converted to the cis-β-amino acid hydrochlorides by exposure to aq. hydrochloric acid.  The resulting cis-β-amino acids were converted to their Fmoc-derivs. under conditions similar to those developed for the com. available amino acids.  Differences in the conditions employed between these β-amino acids and the com. derivs. were obsd., primarily in the nature of the base required for the reaction.  A possible rationale for the differences in behavior is described.  These Fmoc-amino acid derivs. are valuable intermediates for the solid-phase synthesis of combinatorial libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr31UZPt5I5-LVg90H21EOLACvtfcHk0ljxeWtP7w4mTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjvFOhu7w%253D&md5=8d76a0c09d27d3eac2ff1a29217da4da</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fop010204f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fop010204f%26sid%3Dliteratum%253Aachs%26aulast%3DDener%26aufirst%3DJ.%2BM.%26aulast%3DFantauzzi%26aufirst%3DP.%2BP.%26aulast%3DKshirsagar%26aufirst%3DT.%2BA.%26aulast%3DKelly%26aufirst%3DD.%2BE.%26aulast%3DWolfe%26aufirst%3DA.%2BB.%26atitle%3DLarge-Scale%2520Syntheses%2520of%2520Fmoc-Protected%2520Non-Proteogenic%2520Amino%2520Acids%253A%2520Useful%2520Building%2520Blocks%2520for%2520Combinatorial%2520Libraries%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2001%26volume%3D5%26spage%3D445%26epage%3D449%26doi%3D10.1021%2Fop010204f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundermann, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Phenylglycine Racemization in Fmoc-Based Solid-Phase Peptide Synthesis: Stereochemical Stability Is Achieved by Choice of Reaction Conditions</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2325</span>– <span class="NLM_lpage">2329</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2017.04.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.tetlet.2017.04.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2017&pages=2325-2329&author=C.+Liangauthor=M.+A.+M.+Behnamauthor=T.+R.+Sundermannauthor=C.+D.+Klein&title=Phenylglycine+Racemization+in+Fmoc-Based+Solid-Phase+Peptide+Synthesis%3A+Stereochemical+Stability+Is+Achieved+by+Choice+of+Reaction+Conditions&doi=10.1016%2Fj.tetlet.2017.04.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylglycine racemization in Fmoc-based solid-phase peptide synthesis: Stereochemical stability is achieved by choice of reaction conditions</span></div><div class="casAuthors">Liang, Chen; Behnam, Mira A. M.; Sundermann, Tom R.; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2325-2329</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Phenylglycine-contg. peptides have broad applications in medicinal chem., but their synthetic accessibility is complicated by the risk of epimerization during solid-phase peptide synthesis (SPPS).  Phenylglycine is therefore often considered a troublesome residue.  This work studies the extent of Phg racemization under different Fmoc-SPPS reaction conditions.  It is shown that the base-catalyzed coupling of Fmoc-Phg is the crit. step for racemization.  However, racemization can be reduced to a negligible level if DEPBT or COMU combined with TMP or DMP are employed during this step.  Resin-bound peptides are remarkably resistant against epimerization during extended incubation under basic conditions and the free peptides were stable in buffer solns. used for biol. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaQeBw87P6bVg90H21EOLACvtfcHk0lhGapBpVE7BQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKhsbw%253D&md5=d6ad6038198c08fcf1484d350f2dbd16</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2017.04.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2017.04.047%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DSundermann%26aufirst%3DT.%2BR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DPhenylglycine%2520Racemization%2520in%2520Fmoc-Based%2520Solid-Phase%2520Peptide%2520Synthesis%253A%2520Stereochemical%2520Stability%2520Is%2520Achieved%2520by%2520Choice%2520of%2520Reaction%2520Conditions%26jtitle%3DTetrahedron%2520Lett.%26date%3D2017%26volume%3D58%26spage%3D2325%26epage%3D2329%26doi%3D10.1016%2Fj.tetlet.2017.04.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernatowicz, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsueda, G. R.</span></span> <span> </span><span class="NLM_article-title">1H-Pyrazole-1-Carboxamidine Hydrochloride an Attractive Reagent for Guanylation of Amines and Its Application to Peptide Synthesis</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2497</span>– <span class="NLM_lpage">2502</span>, <span class="refDoi"> DOI: 10.1021/jo00034a059</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00034a059" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK38XhslCksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1992&pages=2497-2502&author=M.+S.+Bernatowiczauthor=Y.+Wuauthor=G.+R.+Matsueda&title=1H-Pyrazole-1-Carboxamidine+Hydrochloride+an+Attractive+Reagent+for+Guanylation+of+Amines+and+Its+Application+to+Peptide+Synthesis&doi=10.1021%2Fjo00034a059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">1H-Pyrazole-1-carboxamidine hydrochloride an attractive reagent for guanylation of amines and its application to peptide synthesis</span></div><div class="casAuthors">Bernatowicz, Michael S.; Wu, Youling; Matsueda, Gary R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2497-502</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">1H-Pyrazole-1-carboxamidine hydrochloride (I) was conveniently prepd. in 95% yield by reaction of equimolar amts. of pyrazole and cyanamide in refluxing HCl/p-dioxane.  Guanylation of sterically unhindered primary and secondary aliph. amines (e.g. cyclohexylamine, ethanolamine, piperidine, glycine) in DMF at room temp. for 4-8 h gave 71-88% isolated yields of guanidines.  Arom. amines (e.g. aniline, 4-methoxyaniline) required longer reaction times or more forcing conditions.  Other potentially nucleophilic functionalities (hydroxyl, carboxyl, thiol, imidazole) failed to react with I.  The successful prepn. of H-Gly-His-Arg-Pro-Leu-Asp-Lys-NH2 (N-terminal heptapeptide from the β-chain of human fibrin) by guanylation of an appropriately protected Orn3-peptidyl-polystyrene resin precursor demonstrated the feasibility of using I as a guanylating reagent in an Orn → Arg strategy of solid phase peptide synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyfr8dJ0yWzLVg90H21EOLACvtfcHk0lhGapBpVE7BQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhslCksbw%253D&md5=e48ccfec15ddfe89575b20e18a3cff4b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjo00034a059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00034a059%26sid%3Dliteratum%253Aachs%26aulast%3DBernatowicz%26aufirst%3DM.%2BS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DMatsueda%26aufirst%3DG.%2BR.%26atitle%3D1H-Pyrazole-1-Carboxamidine%2520Hydrochloride%2520an%2520Attractive%2520Reagent%2520for%2520Guanylation%2520of%2520Amines%2520and%2520Its%2520Application%2520to%2520Peptide%2520Synthesis%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1992%26volume%3D57%26spage%3D2497%26epage%3D2502%26doi%3D10.1021%2Fjo00034a059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donovan, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozas, I.</span></span> <span> </span><span class="NLM_article-title">Pyridin-2-yl Guanidine Derivatives: Conformational Control Induced by Intramolecular Hydrogen-Bonding Interactions</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">9216</span>– <span class="NLM_lpage">9227</span>, <span class="refDoi"> DOI: 10.1021/jo200954c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo200954c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OnsLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=9216-9227&author=B.+Kellyauthor=D.+H.+O%E2%80%99Donovanauthor=J.+O%E2%80%99Brienauthor=T.+McCabeauthor=F.+Blancoauthor=I.+Rozas&title=Pyridin-2-yl+Guanidine+Derivatives%3A+Conformational+Control+Induced+by+Intramolecular+Hydrogen-Bonding+Interactions&doi=10.1021%2Fjo200954c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Pyridin-2-yl guanidine derivatives: Conformational control induced by intramolecular hydrogen-bonding interactions</span></div><div class="casAuthors">Kelly, Brendan; O'Donovan, Daniel H.; O'Brien, John; McCabe, Thomas; Blanco, Fernando; Rozas, Isabel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9216-9227</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and conformational anal. of a series of pyridin-2-yl guanidine derivs. using NMR, X-ray crystallog., and B3LYP/6-31+G** theor. studies are reported.  A remarkable difference was obsd. in the 1H NMR spectra of the guanidinium salts as compared with their N,N'-di-Boc protected and neutral analogs.  This difference corresponds to a 180° change in the dihedral angle between the guanidine/ium moiety and the pyridine ring in the salts as compared to the Boc-protected derivs., a conclusion that was supported by theor. studies, X-ray data, and NMR anal.  Moreover, our data sustain the existence of two intramol. hydrogen-bonding systems: (i) between the pyridine N1 atom and the guanidinium protons in the salts and (ii) within the tert-Bu carbamate groups of the Boc-protected derivs.  To verify that the obsd. conformational control arises from these intramol. interactions, a new series of N-Boc-N'-propyl-substituted pyridin-2-yl guanidines were also prepd. and studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfrhi-HXC_BLVg90H21EOLACvtfcHk0lhGapBpVE7BQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OnsLvF&md5=3981dd5b64a899381e64118c392c2e8a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjo200954c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo200954c%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DD.%2BH.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DJ.%26aulast%3DMcCabe%26aufirst%3DT.%26aulast%3DBlanco%26aufirst%3DF.%26aulast%3DRozas%26aufirst%3DI.%26atitle%3DPyridin-2-yl%2520Guanidine%2520Derivatives%253A%2520Conformational%2520Control%2520Induced%2520by%2520Intramolecular%2520Hydrogen-Bonding%2520Interactions%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D9216%26epage%3D9227%26doi%3D10.1021%2Fjo200954c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seh, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P. Y.</span></span> <span> </span><span class="NLM_article-title">Ligand-Bound Structures of the Dengue Virus Protease Reveal the Active Conformation</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">446</span>, <span class="refDoi"> DOI: 10.1128/JVI.06225-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FJVI.06225-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=22031935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsVKks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=438-446&author=C.+G.+Nobleauthor=C.+C.+Sehauthor=A.+T.+Chaoauthor=P.+Y.+Shi&title=Ligand-Bound+Structures+of+the+Dengue+Virus+Protease+Reveal+the+Active+Conformation&doi=10.1128%2FJVI.06225-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-bound structures of the dengue virus protease reveal the active conformation</span></div><div class="casAuthors">Noble, Christian G.; Seh, Cheah Chen; Chao, Alexander T.; Shi, Pei Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">438-446</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue is a mosquito-borne viral hemorrhagic disease that is a major threat to human health in tropical and subtropical regions.  Here we report crystal structures of a peptide covalently bound to dengue virus serotype 3 (DENV-3) protease as well as the serine-protease inhibitor aprotinin bound to the same enzyme.  These structures reveal, for the first time, a catalytically active, closed conformation of the DENV protease.  In the presence of the peptide, the DENV-3 protease forms the closed conformation in which the hydrophilic β-hairpin region of NS2B wraps around the NS3 protease core, in a manner analogous to the structure of West Nile virus (WNV) protease.  Our results confirm that flavivirus proteases form the closed conformation during proteolysis, as previously proposed for WNV.  The current DENV-3 protease structures reveal the detailed interactions at the P4' to P3 sites of the substrate.  The new structural information explains the sequence preference, particularly for long basic residues in the nonprime side, as well as the difference in substrate specificity between the WNV and DENV proteases at the prime side.  Structural anal. of the DENV-3 protease-peptide complex revealed a pocket that is formed by residues from NS2B and NS3; this pocket also exists in the WNV NS2B/NS3 protease structure and could be targeted for potential antivirus development.  The structural information presented in the current study is invaluable for the design of specific inhibitors of DENV protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEQKa3k5XF7rVg90H21EOLACvtfcHk0lhAsKt4LutI6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsVKks7c%253D&md5=f1b30da53856fe70d4c129fab8d54f35</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FJVI.06225-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.06225-11%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DC.%2BG.%26aulast%3DSeh%26aufirst%3DC.%2BC.%26aulast%3DChao%26aufirst%3DA.%2BT.%26aulast%3DShi%26aufirst%3DP.%2BY.%26atitle%3DLigand-Bound%2520Structures%2520of%2520the%2520Dengue%2520Virus%2520Protease%2520Reveal%2520the%2520Active%2520Conformation%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D438%26epage%3D446%26doi%3D10.1128%2FJVI.06225-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boldescu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graf, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streicher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimoglo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Synthesis, Biological Evaluation, and Molecular Docking of Combretastatin and Colchicine Derivatives and Their HCE1-Activated Prodrugs as Antiviral Agents</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fcmdc.201800641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=30605241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCrsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=469-483&author=M.+Richterauthor=V.+Boldescuauthor=D.+Grafauthor=F.+Streicherauthor=A.+Dimogloauthor=R.+Bartenschlagerauthor=C.+D.+Klein&title=Synthesis%2C+Biological+Evaluation%2C+and+Molecular+Docking+of+Combretastatin+and+Colchicine+Derivatives+and+Their+HCE1-Activated+Prodrugs+as+Antiviral+Agents&doi=10.1002%2Fcmdc.201800641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological evaluation, and molecular docking of combretastatin and colchicine derivatives and their hCE1-activated prodrugs as antiviral agents</span></div><div class="casAuthors">Richter, Michael; Boldescu, Veaceslav; Graf, Dominik; Streicher, Felix; Dimoglo, Anatoli; Bartenschlager, Ralf; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-483</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Recent studies indicate that tubulin can be a host factor for vector-borne flaviviruses like dengue (DENV) and Zika (ZIKV), and inhibitors of tubulin polymn. such as colchicine have been demonstrated to decrease virus replication.  However, toxicity limits the application of these compds.  Herein we report prodrugs based on combretastatin and colchicine derivs. that contain an ester cleavage site for human carboxylesterase, a highly abundant enzyme in monocytes and hepatocytes targeted by DENV.  Relative to their parent compds., the cytotoxicity of these prodrugs was reduced by several orders of magnitude.  All synthesized prodrugs contg. a leucine ester were hydrolyzed by the esterase in vitro.  In contrast to previous reports, the phenylglycine esters were not cleaved by human carboxylesterase.  The antiviral activity of combretastatin, colchicine, and selected prodrugs against DENV and ZIKV in cell culture was obsd. at low micromolar and sub-micromolar concns.  In addn., docking studies were performed to understand the binding mode of the studied compds. to tubulin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsPhT5W1m43bVg90H21EOLACvtfcHk0lhAsKt4LutI6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCrsr8%253D&md5=877296cfa9d8b10725b73d7a0f0f3ea8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800641%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DBoldescu%26aufirst%3DV.%26aulast%3DGraf%26aufirst%3DD.%26aulast%3DStreicher%26aufirst%3DF.%26aulast%3DDimoglo%26aufirst%3DA.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DSynthesis%252C%2520Biological%2520Evaluation%252C%2520and%2520Molecular%2520Docking%2520of%2520Combretastatin%2520and%2520Colchicine%2520Derivatives%2520and%2520Their%2520HCE1-Activated%2520Prodrugs%2520as%2520Antiviral%2520Agents%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D469%26epage%3D483%26doi%3D10.1002%2Fcmdc.201800641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kain, S. R.</span></span> <span> </span><span class="NLM_article-title">Generation of Destabilized Green Fluorescent Protein as a Transcription Reporter</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">34970</span>– <span class="NLM_lpage">34975</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.52.34970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.273.52.34970" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=9857028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK1MXisFajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=34970-34975&author=X.+Liauthor=X.+Zhaoauthor=Y.+Fangauthor=X.+Jiangauthor=T.+Duongauthor=C.+Fanauthor=C.-C.+Huangauthor=S.+R.+Kain&title=Generation+of+Destabilized+Green+Fluorescent+Protein+as+a+Transcription+Reporter&doi=10.1074%2Fjbc.273.52.34970"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of destabilized green fluorescent protein as a transcription reporter</span></div><div class="casAuthors">Li, Xianqiang; Zhao, Xiaoning; Fang, Yu; Jiang, Xin; Duong, Tommy; Fan, Connie; Huang, Chiao-Chain; Kain, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">34970-34975</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The green fluorescent protein (GFP) is a widely used reporter in gene expression and protein localization studies.  GFP is a stable protein; this property allows its accumulation and easy detection in cells.  However, this stability also limits its application in studies that require rapid reporter turnover.  We created a destabilized GFP for use in such studies by fusing amino acids 422-461 of the degrdn. domain of mouse ornithine decarboxylase (MODC) to the C-terminal end of an enhanced variant of GFP (EGFP).  The fusion protein, unlike EGFP, was unstable in the presence of cycloheximide and had a fluorescence half-life of 2 h.  Western blot anal. indicated that the fluorescence decay of EGFP-MODC-(422-461) was correlated with degrdn. of the fusion protein.  We mutated key amino acids in the PEST sequence of EGFP-MODC-(422-461) and identified several mutants with variable half-lives.  The suitability of destabilized EGFP as a transcription reporter was tested by linking it to NFκB binding sequences and monitoring tumor necrosis factor α-mediated NFκB activation.  We obtained time course induction and dose response kinetics similar to secreted alk. phosphatase obtained in transfected cells.  This result did not occur when unmodified EGFP was used as the reporter.  Because of its autofluorescence, destabilized EGFP can be used to directly correlate gene induction with biochem. change, such as NFκB translocation to the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnMsl5eDXgJ7Vg90H21EOLACvtfcHk0lhAsKt4LutI6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisFajsA%253D%253D&md5=4557c9ff881135e40b55c90877e6b9a0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.52.34970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.52.34970%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DDuong%26aufirst%3DT.%26aulast%3DFan%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DC.-C.%26aulast%3DKain%26aufirst%3DS.%2BR.%26atitle%3DGeneration%2520of%2520Destabilized%2520Green%2520Fluorescent%2520Protein%2520as%2520a%2520Transcription%2520Reporter%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D34970%26epage%3D34975%26doi%3D10.1074%2Fjbc.273.52.34970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rechsteiner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, S. W.</span></span> <span> </span><span class="NLM_article-title">PEST Sequences and Regulation by Proteolysis</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">271</span>, <span class="refDoi"> DOI: 10.1016/S0968-0004(96)10031-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2FS0968-0004%2896%2910031-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=8755249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK28XksVymtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1996&pages=267-271&author=M.+Rechsteinerauthor=S.+W.+Rogers&title=PEST+Sequences+and+Regulation+by+Proteolysis&doi=10.1016%2FS0968-0004%2896%2910031-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">PEST sequences and regulation by proteolysis</span></div><div class="casAuthors">Rechsteiner, Martin; Rogers, Scott W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">267-271</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Trends Journals</span>)
        </div><div class="casAbstract">A review with 48 refs.  In 1986, we proposed that polypeptide sequences enriched in proline (P), glutamic acid (E), serine (S) and threonine (T) target proteins for rapid destruction.  For much of the past decade there were only sporadic exptl. tests of the hypothesis.  This situation changed markedly during the past two years with a no. of papers providing strong evidence that PEST regions do, in fact, serve as proteolytic signals.  The properties of PEST regions and some interesting examples of the conditional nature of such signals are briefly discussed.  Most of the article, however, focuses on recent exptl. support for the hypothesis and on mechanisms responsible for the rapid degrdn. of proteins that contain PEST regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJfQ1FYbkGtLVg90H21EOLACvtfcHk0ljnIqxe8cIrXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XksVymtr8%253D&md5=14cd790123efab12ba61c6226ca0c986</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2896%2910031-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252896%252910031-1%26sid%3Dliteratum%253Aachs%26aulast%3DRechsteiner%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DS.%2BW.%26atitle%3DPEST%2520Sequences%2520and%2520Regulation%2520by%2520Proteolysis%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1996%26volume%3D21%26spage%3D267%26epage%3D271%26doi%3D10.1016%2FS0968-0004%2896%2910031-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunn, H. F.</span></span> <span> </span><span class="NLM_article-title">Regulation of Hypoxia-Inducible Factor 1alpha Is Mediated by an O2-Dependent Degradation Domain Via the Ubiquitin-Proteasome Pathway</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">7987</span>– <span class="NLM_lpage">7992</span>, <span class="refDoi"> DOI: 10.1073/pnas.95.14.7987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1073%2Fpnas.95.14.7987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=9653127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK1cXks1SlsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=7987-7992&author=L.+E.+Huangauthor=J.+Guauthor=M.+Schauauthor=H.+F.+Bunn&title=Regulation+of+Hypoxia-Inducible+Factor+1alpha+Is+Mediated+by+an+O2-Dependent+Degradation+Domain+Via+the+Ubiquitin-Proteasome+Pathway&doi=10.1073%2Fpnas.95.14.7987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of hypoxia-inducible factor 1α is mediated by an O2-dependent degradation domain via the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Huang, L. Eric; Gu, Jie; Schau, Maureen; Bunn, H. Franklin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7987-7992</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hypoxia induces a group of physiol. important genes such as erythropoietin and vascular endothelial growth factor.  These genes are transcriptionally up-regulated by hypoxia-inducible factor 1 (HIF-1), a global regulator that belongs to the basic helix-loop-helix PAS family.  Although HIF-1 is a heterodimer composed of α and β subunits, its activity is primarily detd. by hypoxia-induced stabilization of HIF-1α which is otherwise rapidly degraded in oxygenated cells.  The authors report the identification of an oxygen-dependent degrdn. (ODD) domain within HIF-1α that controls its degrdn. by the ubiquitin-proteasome pathway.  The ODD domain consists of ≈200 amino acid residues, located in the central region of HIF-1α.  Because portions of the domain independently confer degrdn. of HIF-1α, deletion of this entire region is required to give rise to a stable HIF-1α, capable of heterodimerization, DNA-binding, and transactivation in the absence of hypoxic signaling.  Conversely, the ODD domain alone confers oxygen-dependent instability when fused to a stable protein, Gal4.  Hence, the ODD domain plays a pivotal role for regulating HIF-1 activity and thereby may provide a means of controlling gene expression by changes in oxygen tension.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGXW78adyIx7Vg90H21EOLACvtfcHk0ljnIqxe8cIrXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1SlsrY%253D&md5=2d2765f29149ea4e78964f5721922e19</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.14.7987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.14.7987%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%2BE.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DSchau%26aufirst%3DM.%26aulast%3DBunn%26aufirst%3DH.%2BF.%26atitle%3DRegulation%2520of%2520Hypoxia-Inducible%2520Factor%25201alpha%2520Is%2520Mediated%2520by%2520an%2520O2-Dependent%2520Degradation%2520Domain%2520Via%2520the%2520Ubiquitin-Proteasome%2520Pathway%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D7987%26epage%3D7992%26doi%3D10.1073%2Fpnas.95.14.7987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-W.</span></span> <span> </span><span class="NLM_article-title">Hypoxia-Inducible Factor (HIF-1)α: Its Protein Stability and Biological Functions</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.1038/emm.2004.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1038%2Femm.2004.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=15031665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1Grs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2004&pages=1-12&author=J.-W.+Leeauthor=S.-H.+Baeauthor=J.-W.+Jeongauthor=S.-H.+Kimauthor=K.-W.+Kim&title=Hypoxia-Inducible+Factor+%28HIF-1%29%CE%B1%3A+Its+Protein+Stability+and+Biological+Functions&doi=10.1038%2Femm.2004.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia-inducible factor (HIF-1)α: its protein stability and biological functions</span></div><div class="casAuthors">Lee, Ji-Won; Bae, Seong-Hui; Jeong, Joo-Won; Kim, Se-Hee; Kim, Kyu-Won</div><div class="citationInfo"><span class="NLM_cas:title">Experimental and Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1226-3613</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Medical Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Hypoxia-inducible factor (HIF-1) is an oxygen-dependent transcriptional activator, which plays crucial roles in the angiogenesis of tumors and mammalian development.  HIF-1 consists of a constitutively expressed HIF-1β subunit and one of three subunits (HIF-1α, HIF-2α or HIF-3α).  The stability and activity of HIF-1α are regulated by various post-translational modifications, hydroxylation, acetylation, and phosphorylation.  Therefore, HIF-1α interacts with several protein factors including PHD, pVHL, ARD-1, and p300/CBP.  Under normoxia, the HIF-1α subunit is rapidly degraded via the von Hippel-Lindau tumor suppressor gene product (pVHL)-mediated ubiquitin-proteasome pathway.  The assocn. of pVHL and HIF-1α under normoxic conditions is triggered by the hydroxylation of prolines and the acetylation of lysine within a polypeptide segment known as the oxygen-dependent degrdn. (ODD) domain.  On the contrary, in the hypoxia condition, HIF-1α subunit becomes stable and interacts with coactivators such as p300/CBP to modulate its transcriptional activity.  Eventually, HIF-1 acts as a master regulator of numerous hypoxia-inducible genes under hypoxic conditions.  The target genes of HIF-1 are esp. related to angiogenesis, cell proliferation/survival, and glucose/iron metab.  Moreover, it was reported that the activation of HIF-1α is closely assocd. with a variety of tumors and oncogenic pathways.  Hence, the blocking of HIF-1α itself or HIF-1α interacting proteins inhibit tumor growth.  Based on these findings, HIF-1 can be a prime target for anticancer therapies.  This review summarizes the mol. mechanism of HIF-1α stability, the bi- ol. functions of HIF-1 and its potential applications of cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkaNBwWoBA-bVg90H21EOLACvtfcHk0ljnIqxe8cIrXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1Grs7g%253D&md5=49fb7514a11c4b30d0e803520242969d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Femm.2004.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2004.1%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.-W.%26aulast%3DBae%26aufirst%3DS.-H.%26aulast%3DJeong%26aufirst%3DJ.-W.%26aulast%3DKim%26aufirst%3DS.-H.%26aulast%3DKim%26aufirst%3DK.-W.%26atitle%3DHypoxia-Inducible%2520Factor%2520%2528HIF-1%2529%25CE%25B1%253A%2520Its%2520Protein%2520Stability%2520and%2520Biological%2520Functions%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2004%26volume%3D36%26spage%3D1%26epage%3D12%26doi%3D10.1038%2Femm.2004.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, P. J.</span></span> <span> </span><span class="NLM_article-title">Independent Function of Two Destruction Domains in Hypoxia-Inducible Factor-Alpha Chains Activated by Prolyl Hydroxylation</span>. <i>EMBO J</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5197</span>– <span class="NLM_lpage">5206</span>, <span class="refDoi"> DOI: 10.1093/emboj/20.18.5197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1093%2Femboj%2F20.18.5197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=11566883" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD3MXns1Ghs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=5197-5206&author=N.+Massonauthor=C.+Willamauthor=P.+H.+Maxwellauthor=C.+W.+Pughauthor=P.+J.+Ratcliffe&title=Independent+Function+of+Two+Destruction+Domains+in+Hypoxia-Inducible+Factor-Alpha+Chains+Activated+by+Prolyl+Hydroxylation&doi=10.1093%2Femboj%2F20.18.5197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Independent function of two destruction domains in hypoxia-inducible factor-α chains activated by prolyl hydroxylation</span></div><div class="casAuthors">Masson, Norma; Willam, Carsten; Maxwell, Patrick H.; Pugh, Christopher W.; Ratcliffe, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5197-5206</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Oxygen-dependent proteolytic destruction of hypoxia-inducible factor-α (HIF-α) subunits plays a central role in regulating transcriptional responses to hypoxia.  Recent studies have defined a key function for the von Hippel-Lindau tumor suppressor E3 ubiquitin ligase (VHLE3) in this process, and have defined an interaction with HIF-1α that is regulated by prolyl hydroxylation.  Here we show that two independent regions within the HIF-α oxygen-dependent degrdn. domain (ODDD) are targeted for ubiquitylation by VHLE3 in a manner dependent upon prolyl hydroxylation.  In a series of in vitro and in vivo assays, we demonstrate the independent and non-redundant operation of each site in regulation of the HIF system.  Both sites contain a common core motif, but differ both in overall sequence and in the conditions under which they bind to the VHLE3 ligase complex.  The definition of two independent destruction domains implicates a more complex system of pVHL-HIF-α interactions, but reinforces the role of prolyl hydroxylation as an oxygen-dependent destruction signal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLKrBnPdE6lrVg90H21EOLACvtfcHk0ljnIqxe8cIrXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXns1Ghs78%253D&md5=e4f167d66a64f515de9c43e53a7f546d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F20.18.5197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F20.18.5197%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DN.%26aulast%3DWillam%26aufirst%3DC.%26aulast%3DMaxwell%26aufirst%3DP.%2BH.%26aulast%3DPugh%26aufirst%3DC.%2BW.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26atitle%3DIndependent%2520Function%2520of%2520Two%2520Destruction%2520Domains%2520in%2520Hypoxia-Inducible%2520Factor-Alpha%2520Chains%2520Activated%2520by%2520Prolyl%2520Hydroxylation%26jtitle%3DEMBO%2520J%26date%3D2001%26volume%3D20%26spage%3D5197%26epage%3D5206%26doi%3D10.1093%2Femboj%2F20.18.5197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smirnova, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakhman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payappilly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazakov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez-Guzman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaisina, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratan, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazaryan, I. G.</span></span> <span> </span><span class="NLM_article-title">Utilization of an in Vivo Reporter for High Throughput Identification of Branched Small Molecule Regulators of Hypoxic Adaptation</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.chembiol.2010.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=20416509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFOiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=380-391&author=N.+A.+Smirnovaauthor=I.+Rakhmanauthor=N.+Morozauthor=M.+Bassoauthor=J.+Payappillyauthor=S.+Kazakovauthor=F.+Hernandez-Guzmanauthor=I.+N.+Gaisinaauthor=A.+P.+Kozikowskiauthor=R.+R.+Ratanauthor=I.+G.+Gazaryan&title=Utilization+of+an+in+Vivo+Reporter+for+High+Throughput+Identification+of+Branched+Small+Molecule+Regulators+of+Hypoxic+Adaptation&doi=10.1016%2Fj.chembiol.2010.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Utilization of an In Vivo Reporter for High Throughput Identification of Branched Small Molecule Regulators of Hypoxic Adaptation</span></div><div class="casAuthors">Smirnova, Natalya A.; Rakhman, Ilay; Moroz, Natalia; Basso, Manuela; Payappilly, Jimmy; Kazakov, Sergey; Hernandez-Guzman, Francisco; Gaisina, Irina N.; Kozikowski, Alan P.; Ratan, Rajiv R.; Gazaryan, Irina G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">380-391</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Small mols. inhibiting hypoxia inducible factor (HIF) prolyl hydroxylases (PHDs) are the focus of drug development efforts directed toward the treatment of ischemia and metabolic imbalance.  A cell-based reporter produced by fusing HIF-1α oxygen degradable domain (ODD) to luciferase was shown to work as a capture assay monitoring stability of the overexpressed luciferase-labeled HIF PHD substrate under conditions more physiol. than in vitro test tubes.  High throughput screening identified novel catechol and oxyquinoline pharmacophores with a "branching motif" immediately adjacent to a Fe-binding motif that fits selectively into the HIF PHD active site in in silico models.  In accord with their structure-activity relationship in the primary screen, the best "hits" stabilize HIF1α, upregulate known HIF target genes in a human neuronal line, and exert neuroprotective effects in established model of oxidative stress in cortical neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTX3nine48HLVg90H21EOLACvtfcHk0lh6jE1l-VMuFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFOiu7g%253D&md5=6d590af4c5d184e21268bb7a352ce387</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DSmirnova%26aufirst%3DN.%2BA.%26aulast%3DRakhman%26aufirst%3DI.%26aulast%3DMoroz%26aufirst%3DN.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DPayappilly%26aufirst%3DJ.%26aulast%3DKazakov%26aufirst%3DS.%26aulast%3DHernandez-Guzman%26aufirst%3DF.%26aulast%3DGaisina%26aufirst%3DI.%2BN.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DGazaryan%26aufirst%3DI.%2BG.%26atitle%3DUtilization%2520of%2520an%2520in%2520Vivo%2520Reporter%2520for%2520High%2520Throughput%2520Identification%2520of%2520Branched%2520Small%2520Molecule%2520Regulators%2520of%2520Hypoxic%2520Adaptation%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D380%26epage%3D391%26doi%3D10.1016%2Fj.chembiol.2010.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Safran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flippin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günzler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.</span></span> <span> </span><span class="NLM_article-title">Mouse Model for Noninvasive Imaging of HIF Prolyl Hydroxylase Activity: Assessment of an Oral Agent That Stimulates Erythropoietin Production</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1073/pnas.0509459103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1073%2Fpnas.0509459103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=16373502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=105-110&author=M.+Safranauthor=W.+Y.+Kimauthor=F.+O%E2%80%99Connellauthor=L.+Flippinauthor=V.+G%C3%BCnzlerauthor=J.+W.+Hornerauthor=R.+A.+DePinhoauthor=W.+G.+Kaelin&title=Mouse+Model+for+Noninvasive+Imaging+of+HIF+Prolyl+Hydroxylase+Activity%3A+Assessment+of+an+Oral+Agent+That+Stimulates+Erythropoietin+Production&doi=10.1073%2Fpnas.0509459103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse model for noninvasive imaging of HIF prolyl hydroxylase activity: Assessment of an oral agent that stimulates erythropoietin production</span></div><div class="casAuthors">Safran, Michal; Kim, William Y.; O'Connell, Fionnuala; Flippin, Lee; Gunzler, Volkmar; Horner, James W.; DePinho, Ronald A.; Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-110</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many human diseases are characterized by the development of tissue hypoxia.  Inadequate oxygenation can cause cellular dysfunction and death.  Tissues use many strategies, including induction of angiogenesis and alterations in metab., to survive under hypoxic conditions.  The heterodimeric transcription factor hypoxia-inducible factor (HIF) is a master regulator of genes that promote adaptation to hypoxia.  HIF activity is linked to oxygen availability because members of the EGLN family hydroxylate HIFα subunits on specific prolyl residues when oxygen is present, which marks them for ubiquitination and proteasomal degrdn.  The authors created a mouse that ubiquitously expresses a bioluminescent reporter consisting of firefly luciferase fused to a region of HIF that is sufficient for oxygen-dependent degrdn.  The authors' validation studies suggest that this mouse will be useful for monitoring hypoxic tissues and evaluating therapeutic agents that stabilize HIF.  One such agent, the HIF prolyl hydroxylase inhibitor FG-4383, was active in the liver and kidney after systemic administration as detd. by bioluminescence imaging, transcription profiling, and prodn. of erythropoietin, indicating that the HIF transcriptional program can be manipulated in vivo with orally active org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3yVklADN01LVg90H21EOLACvtfcHk0lh6jE1l-VMuFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms12juw%253D%253D&md5=c39a622219baf85cb3f7d69e0cca5392</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0509459103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0509459103%26sid%3Dliteratum%253Aachs%26aulast%3DSafran%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DW.%2BY.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DF.%26aulast%3DFlippin%26aufirst%3DL.%26aulast%3DG%25C3%25BCnzler%26aufirst%3DV.%26aulast%3DHorner%26aufirst%3DJ.%2BW.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DMouse%2520Model%2520for%2520Noninvasive%2520Imaging%2520of%2520HIF%2520Prolyl%2520Hydroxylase%2520Activity%253A%2520Assessment%2520of%2520an%2520Oral%2520Agent%2520That%2520Stimulates%2520Erythropoietin%2520Production%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D105%26epage%3D110%26doi%3D10.1073%2Fpnas.0509459103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kimbrel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span> <span> </span><span class="NLM_article-title">In Vivo Pharmacodynamic Imaging of Proteasome Inhibition</span>. <i>Mol. Imaging</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">147</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.2310%2F7290.2009.00007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19723471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptVemtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=140-147&author=E.+A.+Kimbrelauthor=T.+N.+Davisauthor=J.+E.+Bradnerauthor=A.+L.+Kung&title=In+Vivo+Pharmacodynamic+Imaging+of+Proteasome+Inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo pharmacodynamic imaging of proteasome inhibition</span></div><div class="casAuthors">Kimbrel, Erin A.; Davis, Tina N.; Bradner, James E.; Kung, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Imaging</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">140-147</span>CODEN:
                <span class="NLM_cas:coden">MIOMBP</span>;
        ISSN:<span class="NLM_cas:issn">1535-3508</span>.
    
            (<span class="NLM_cas:orgname">BC Decker Inc.</span>)
        </div><div class="casAbstract">Inhibiting the proteolytic activity of the 26S proteasome has been shown to have selective apoptotic effects on cancer cells and to be clin. efficacious in certain malignancies.  There is an unmet medical need for addnl. proteasome inhibitors, and their development will be facilitated by surrogate markers of proteasome function.  Toward this end, ectopic fusion of the destruction domain from ornithine decarboxylase (ODC) to reporter proteins is often used for assessing proteasome function.  For luciferase-based reporters, we hypothesized that the oxygen-dependent destruction domain (ODD) from hypoxia-inducible factor 1α (HIF-1α) may provide improved sensitivity over luciferase-ODC, owing to its extremely rapid turnover by the proteasome (HIF-1α has a half-life of less than 5 min).  In the current study, we show that ODD-luciferase affords a greater dynamic range and faster kinetics than luciferase-ODC in sensing proteasome inhibition in vitro.  Importantly, ODD-luciferase also serves as an effective in vivo marker of proteasome function in xenograft tumor models, with inhibition being detected by noninvasive imaging within 3 h of bortezomib administration.  These data establish ODD-luciferase as a surrogate marker of proteasome function that can be used both in vitro and in vivo for the development of novel proteasome inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomH3K_-EPUwrVg90H21EOLACvtfcHk0ljl_qgj-3bY6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptVemtbY%253D&md5=25301db1b4f83a1778539374bba4fe00</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2310%2F7290.2009.00007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2310%252F7290.2009.00007%26sid%3Dliteratum%253Aachs%26aulast%3DKimbrel%26aufirst%3DE.%2BA.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKung%26aufirst%3DA.%2BL.%26atitle%3DIn%2520Vivo%2520Pharmacodynamic%2520Imaging%2520of%2520Proteasome%2520Inhibition%26jtitle%3DMol.%2520Imaging%26date%3D2009%26volume%3D8%26spage%3D140%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kizaka-Kondoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itasaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morinibu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinomiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraoka, M.</span></span> <span> </span><span class="NLM_article-title">The Combination of Hypoxia-Response Enhancers and an Oxygen-Dependent Proteolytic Motif Enables Real-Time Imaging of Absolute HIF-1 Activity in Tumor Xenografts</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>360</i></span>,  <span class="NLM_fpage">791</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2007.06.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bbrc.2007.06.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17624305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVWmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2007&pages=791-796&author=H.+Haradaauthor=S.+Kizaka-Kondohauthor=S.+Itasakaauthor=K.+Shibuyaauthor=A.+Morinibuauthor=K.+Shinomiyaauthor=M.+Hiraoka&title=The+Combination+of+Hypoxia-Response+Enhancers+and+an+Oxygen-Dependent+Proteolytic+Motif+Enables+Real-Time+Imaging+of+Absolute+HIF-1+Activity+in+Tumor+Xenografts&doi=10.1016%2Fj.bbrc.2007.06.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The combination of hypoxia-response enhancers and an oxygen-dependent proteolytic motif enables real-time imaging of absolute HIF-1 activity in tumor xenografts</span></div><div class="casAuthors">Harada, Hiroshi; Kizaka-Kondoh, Shinae; Itasaka, Satoshi; Shibuya, Keiko; Morinibu, Akiyo; Shinomiya, Kazumi; Hiraoka, Masahiro</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">360</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">791-796</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The transcriptional activity of hypoxia-inducible factor-1 (HIF-1) is assocd. with tumor malignancies; therefore, it is important to comprehend its dynamism in solid tumors.  However, a mol. imaging strategy to accurately access it remains to be developed.  We constructed here a novel HIF-1-dependent reporter gene, 5HREp-ODD-luc, in which 5 copies of the hypoxia-response element (5HRE) enhance expression of the oxygen-dependent degrdn. (ODD) domain and luciferase (luc) fusion under hypoxia.  Because the ODD domain caused the oxygen-dependent degrdn. of the ODD-Luc protein, the novel reporter gene showed little leak of luminescence under normoxia.  Such a property caused an increase of the hypoxia-responsiveness up to about 4.7×104-fold.  Moreover, the ODD domain caused rapid degrdn. of the ODD-Luc protein under normoxia, the luminescence reflected the dynamism of HIF-1 activity in real-time.  The superiority of the novel reporter gene will surely accelerate anal. of the intratumoral HIF-1 activity during tumor progression and cancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLe6_xAOLmnrVg90H21EOLACvtfcHk0ljl_qgj-3bY6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVWmt7o%253D&md5=8e4daaa98bca82b5adc2487e29986a10</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.06.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.06.149%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DH.%26aulast%3DKizaka-Kondoh%26aufirst%3DS.%26aulast%3DItasaka%26aufirst%3DS.%26aulast%3DShibuya%26aufirst%3DK.%26aulast%3DMorinibu%26aufirst%3DA.%26aulast%3DShinomiya%26aufirst%3DK.%26aulast%3DHiraoka%26aufirst%3DM.%26atitle%3DThe%2520Combination%2520of%2520Hypoxia-Response%2520Enhancers%2520and%2520an%2520Oxygen-Dependent%2520Proteolytic%2520Motif%2520Enables%2520Real-Time%2520Imaging%2520of%2520Absolute%2520HIF-1%2520Activity%2520in%2520Tumor%2520Xenografts%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D360%26spage%3D791%26epage%3D796%26doi%3D10.1016%2Fj.bbrc.2007.06.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrosky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span> <span> </span><span class="NLM_article-title">Use of the ODD-Luciferase Transgene for the Non-Invasive Imaging of Spontaneous Tumors in Mice</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e18269</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0018269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1371%2Fjournal.pone.0018269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=21479246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXks1Klu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=S.+J.+Goldmanauthor=E.+Chenauthor=R.+Taylorauthor=S.+Zhangauthor=W.+Petroskyauthor=M.+Reissauthor=S.+Jin&title=Use+of+the+ODD-Luciferase+Transgene+for+the+Non-Invasive+Imaging+of+Spontaneous+Tumors+in+Mice&doi=10.1371%2Fjournal.pone.0018269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the ODD-luciferase transgene for the non-invasive imaging of spontaneous tumors in mice</span></div><div class="casAuthors">Goldman, Scott J.; Chen, Elizabeth; Taylor, Robert; Zhang, Sheng; Petrosky, Whitney; Reiss, Michael; Jin, Shengkan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e18269</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">In humans, imaging of tumors provides rapid, accurate assessment of tumor growth and location.  In lab. animals, however, the imaging of spontaneously occurring tumors continues to pose many tech. and logistical problems.  Recently a mouse model was generated in which a chimeric protein consisting of HIF-1α oxygen-dependent degrdn. domain (ODD) fused to luciferase was ubiquitously expressed in all tissues.  Hypoxic stress leads to the accumulation of ODD-luciferase in the tissues of this mouse model which can be identified by non-invasive bioluminescence measurement.  Since solid tumors often contain hypoxic regions, we performed proof-of-principle expts. testing whether this transgenic mouse model may be used as a universal platform for non-invasive imaging anal. of spontaneous solid tumors.  ODD-luciferase transgenic mice were bred with MMTV-neu/beclin1+/- mice.  Upon injection of luciferin, bioluminescent background of normal tissues in the transgenic mice and bioluminescent signals from spontaneously mammary carcinomas were measured non-invasively with an IVIS Spectrum imaging station.  Tumor vols. were measured manually and the histol. of tumor tissues was analyzed.  Our results show that spontaneous mammary tumors in ODD-luciferase transgenic mice generate substantial bioluminescent signals, which are clearly discernable from background tissue luminescence.  Moreover, we demonstrate a strong quant. correlation between the bioluminescent tumor contour and the vol. of palpable tumors.  We further demonstrate that shrinkage of the vol. of spontaneous tumors in response to chemotherapeutic treatment can be detd. quant. using this system.  Finally, we show that the growth and development of spontaneous tumors can be monitored longitudinally over several weeks.  Thus, our results suggest that this model could potentially provide a practical, reliable, and cost-effective non-invasive quant. method for imaging spontaneous solid tumors in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_88YwpPfkF7Vg90H21EOLACvtfcHk0ljjT6GTgIxBug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXks1Klu7g%253D&md5=5a8d2c5365c07bf8e3c0760603cff6fb</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0018269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0018269%26sid%3Dliteratum%253Aachs%26aulast%3DGoldman%26aufirst%3DS.%2BJ.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DPetrosky%26aufirst%3DW.%26aulast%3DReiss%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DS.%26atitle%3DUse%2520of%2520the%2520ODD-Luciferase%2520Transgene%2520for%2520the%2520Non-Invasive%2520Imaging%2520of%2520Spontaneous%2520Tumors%2520in%2520Mice%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0018269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bera, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. L.</span></span> <span> </span><span class="NLM_article-title">Functional Characterization of Cis and Trans Activity of the Flavivirus NS2B-NS3 Protease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">12883</span>– <span class="NLM_lpage">12892</span>, <span class="refDoi"> DOI: 10.1074/jbc.M611318200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.M611318200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17337448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVSitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=12883-12892&author=A.+K.+Beraauthor=R.+J.+Kuhnauthor=J.+L.+Smith&title=Functional+Characterization+of+Cis+and+Trans+Activity+of+the+Flavivirus+NS2B-NS3+Protease&doi=10.1074%2Fjbc.M611318200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Characterization of cis and trans Activity of the Flavivirus NS2B-NS3 Protease</span></div><div class="casAuthors">Bera, Aloke K.; Kuhn, Richard J.; Smith, Janet L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">12883-12892</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Flaviviruses are serious human pathogens for which treatments are generally lacking.  The proteolytic maturation of the 375-kDa viral polyprotein is one target for antiviral development.  The flavivirus serine protease consists of the N-terminal domain of the multifunctional nonstructural protein 3 (NS3) and an essential 40-residue cofactor (NS2B40) within viral protein NS2B.  The NS2B-NS3 protease is responsible for all cytoplasmic cleavage events in viral polyprotein maturation.  This study describes the first biochem. characterization of flavivirus protease activity using full-length NS3.  Recombinant proteases were created by fusion of West Nile virus (WNV) NS2B40 to full-length WNV NS3.  The protease catalyzed two autolytic cleavages.  The NS2B/NS3 junction was cleaved before protein purifn.  A second site at Arg459↓Gly460 within the C-terminal helicase region of NS3 was cleaved more slowly.  Autolytic cleavage reactions also occurred in NS2B-NS3 recombinant proteins from yellow fever virus, dengue virus types 2 and 4, and Japanese encephalitis virus.  Cis and trans cleavages were distinguished using a noncleavable WNV protease variant and two types of substrates as follows: an inactive variant of recombinant WNV NS2B-NS3, and cyan and yellow fluorescent proteins fused by a dodecamer peptide encompassing a natural cleavage site.  With these materials, the autolytic cleavages were found to be intramol. only.  Autolytic cleavage of the helicase site was insensitive to protein diln., confirming that autolysis is intramol.  Formation of an active protease was found to require neither cleavage of NS2B from NS3 nor a free NS3 N terminus.  Evidence was also obtained for product inhibition of the protease by the cleaved C terminus of NS2B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppQqtdjn7TJrVg90H21EOLACvtfcHk0ljjT6GTgIxBug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVSitbc%253D&md5=2ac2dc45954b392fcc4738d653276117</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M611318200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M611318200%26sid%3Dliteratum%253Aachs%26aulast%3DBera%26aufirst%3DA.%2BK.%26aulast%3DKuhn%26aufirst%3DR.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BL.%26atitle%3DFunctional%2520Characterization%2520of%2520Cis%2520and%2520Trans%2520Activity%2520of%2520the%2520Flavivirus%2520NS2B-NS3%2520Protease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D12883%26epage%3D12892%26doi%3D10.1074%2Fjbc.M611318200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arias-Arias, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora-Rodríguez, R.</span></span> <span> </span><span class="NLM_article-title">A Fluorescence-Activatable Reporter of Flavivirus NS2B–NS3 Protease Activity Enables Live Imaging of Infection in Single Cells and Viral Plaques</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2226</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA119.011319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1074%2Fjbc.RA119.011319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=31919100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosFChsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2020&pages=2212-2226&author=J.+L.+Arias-Ariasauthor=D.+J.+MacPhersonauthor=M.+E.+Hillauthor=J.+A.+Hardyauthor=R.+Mora-Rodr%C3%ADguez&title=A+Fluorescence-Activatable+Reporter+of+Flavivirus+NS2B%E2%80%93NS3+Protease+Activity+Enables+Live+Imaging+of+Infection+in+Single+Cells+and+Viral+Plaques&doi=10.1074%2Fjbc.RA119.011319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence-activatable reporter of flavivirus NS2B-NS3 protease activity enables live imaging of infection in single cells and viral plaques</span></div><div class="casAuthors">Arias-Arias, Jorge L.; MacPherson, Derek J.; Hill, Maureen E.; Hardy, Jeanne A.; Mora-Rodriguez, Rodrigo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2212-2227</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">1083-351X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The genus Flavivirus in the family Flaviviridae comprises many medically important viruses, such as dengue virus (DENV), Zika virus (ZIKV), and yellow fever virus.  The quest for therapeutic targets to combat flavivirus infections requires a better understanding of the kinetics of virus- host interactions during infections with native viral strains.  However, this is precluded by limitations of current cell-based systems for monitoring flavivirus infection in living cells.  The authors report the construction of fluorescence-activatable sensors to detect the activities of flavivirus NS2B-NS3 serine proteases in living cells.  The system consists of GFP-based reporters that become fluorescent upon cleavage by recombinant DENV-2/ZIKV proteases in vitro.  A version of this sensor contg. the flavivirus internal NS3 cleavage site linker reported the highest fluorescence activation in stably transduced mammalian cells upon DENV-2/ ZIKV infection.  Moreover, the onset of fluorescence correlated with viral protease activity.  A far-red version of this flavivirus sensor had the best signal-to-noise ratio in a fluorescent Dulbecco's plaque assay, leading to the construction of a multireporter platform combining the flavivirus sensor with reporter dyes for detection of chromatin condensation and cell death, enabling studies of viral plaque formation with single-cell resoln.  Finally, the application of this platform enabled the study of cell-population kinetics of infection and cell death by DENV-2, ZIKV, and yellow fever virus.  The authors anticipate that future studies of viral infection kinetics with this reporter system will enable basic studies of virus- host interactions and facilitate future applications in antiviral drug research to manage flavivirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqadWoYTjPOPbVg90H21EOLACvtfcHk0ljjT6GTgIxBug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosFChsLo%253D&md5=773be52f5c7530a0ce87c78a443780e0</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA119.011319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA119.011319%26sid%3Dliteratum%253Aachs%26aulast%3DArias-Arias%26aufirst%3DJ.%2BL.%26aulast%3DMacPherson%26aufirst%3DD.%2BJ.%26aulast%3DHill%26aufirst%3DM.%2BE.%26aulast%3DHardy%26aufirst%3DJ.%2BA.%26aulast%3DMora-Rodr%25C3%25ADguez%26aufirst%3DR.%26atitle%3DA%2520Fluorescence-Activatable%2520Reporter%2520of%2520Flavivirus%2520NS2B%25E2%2580%2593NS3%2520Protease%2520Activity%2520Enables%2520Live%2520Imaging%2520of%2520Infection%2520in%2520Single%2520Cells%2520and%2520Viral%2520Plaques%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2020%26volume%3D295%26spage%3D2212%26epage%3D2226%26doi%3D10.1074%2Fjbc.RA119.011319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snitko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weidner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weißbrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spannaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acosta, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmeister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodem, J.</span></span> <span> </span><span class="NLM_article-title">Novel Dengue Virus NS2B/NS3 Protease Inhibitors</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1100</span>– <span class="NLM_lpage">1109</span>, <span class="refDoi"> DOI: 10.1128/AAC.03543-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FAAC.03543-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=25487800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1KrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=1100-1109&author=H.+Wuauthor=S.+Bockauthor=M.+Snitkoauthor=T.+Bergerauthor=T.+Weidnerauthor=S.+Hollowayauthor=M.+Kanitzauthor=W.+E.+Diederichauthor=H.+Steuberauthor=C.+Walterauthor=D.+Hofmannauthor=B.+Wei%C3%9Fbrichauthor=R.+Spannausauthor=E.+G.+Acostaauthor=R.+Bartenschlagerauthor=B.+Engelsauthor=T.+Schirmeisterauthor=J.+Bodem&title=Novel+Dengue+Virus+NS2B%2FNS3+Protease+Inhibitors&doi=10.1128%2FAAC.03543-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Novel dengue virus NS2B/NS3 protease inhibitors</span></div><div class="casAuthors">Wu, Hongmei; Bock, Stefanie; Snitko, Mariya; Berger, Thilo; Weidner, Thomas; Holloway, Steven; Kanitz, Manuel; Diederich, Wibke E.; Steuber, Holger; Walter, Christof; Hofmann, Daniela; Weissbrich, Benedikt; Spannaus, Ralf; Acosta, Eliana G.; Bartenschlager, Ralf; Engels, Bernd; Schirmeister, Tanja; Bodem, Jochen</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1100-1109, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue fever is a severe, widespread, and neglected disease with more than 2 million diagnosed infections per yr.  The dengue virus NS2B/NS3 protease (PR) represents a prime target for rational drug design.  At the moment, there are no clin. PR inhibitors (PIs) available.  We have identified diaryl (thio)ethers as candidates for a novel class of PIs.  Here, we report the selective and noncompetitive inhibition of the serotype 2 and 3 dengue virus PR in vitro and in cells by benzothiazole derivs. exhibiting 50% inhibitory concns. (IC50s) in the low-micromolar range.  Inhibition of replication of DENV serotypes 1 to 3 was specific, since all substances influenced neither hepatitis C virus (HCV) nor HIV-1 replication.  Mol. docking suggests binding at a specific allosteric binding site.  In addn. to the in vitro assays, a cell-based PR assay was developed to test these substances in a replication-independent way.  The new compds. inhibited the DENV PR with IC50s in the low-micromolar or submicromolar range in cells.  Furthermore, these novel PIs inhibit viral replication at submicromolar concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvLZ7EVXcgu7Vg90H21EOLACvtfcHk0lhY7vRo5LkCmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1KrtbY%253D&md5=b41b4e2566cd1d1115a33edf42962930</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FAAC.03543-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.03543-14%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DBock%26aufirst%3DS.%26aulast%3DSnitko%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DT.%26aulast%3DWeidner%26aufirst%3DT.%26aulast%3DHolloway%26aufirst%3DS.%26aulast%3DKanitz%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DW.%2BE.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DWalter%26aufirst%3DC.%26aulast%3DHofmann%26aufirst%3DD.%26aulast%3DWei%25C3%259Fbrich%26aufirst%3DB.%26aulast%3DSpannaus%26aufirst%3DR.%26aulast%3DAcosta%26aufirst%3DE.%2BG.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DEngels%26aufirst%3DB.%26aulast%3DSchirmeister%26aufirst%3DT.%26aulast%3DBodem%26aufirst%3DJ.%26atitle%3DNovel%2520Dengue%2520Virus%2520NS2B%252FNS3%2520Protease%2520Inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D1100%26epage%3D1109%26doi%3D10.1128%2FAAC.03543-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, A.</span></span> <span> </span><span class="NLM_article-title">Novel Dengue Virus-Specific NS2B/NS3 Protease Inhibitor, BP2109, Discovered by a High-Throughput Screening Assay</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1128/AAC.00855-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FAAC.00855-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=20937790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVehu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=229-238&author=C.-C.+Yangauthor=Y.-C.+Hsiehauthor=S.-J.+Leeauthor=S.-H.+Wuauthor=C.-L.+Liaoauthor=C.-H.+Tsaoauthor=Y.-S.+Chaoauthor=J.-H.+Chernauthor=C.-P.+Wuauthor=A.+Yueh&title=Novel+Dengue+Virus-Specific+NS2B%2FNS3+Protease+Inhibitor%2C+BP2109%2C+Discovered+by+a+High-Throughput+Screening+Assay&doi=10.1128%2FAAC.00855-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Novel dengue virus-specific NS2B/NS3 protease inhibitor, BP2109, discovered by a high-throughput screening assay</span></div><div class="casAuthors">Yang, Chi-Chen; Hsieh, Yi-Chen; Lee, Shiow-Ju; Wu, Szu-Huei; Liao, Ching-Len; Tsao, Chang-Huei; Chao, Yu-Sheng; Chern, Jyh-Haur; Wu, Chung-Pu; Yueh, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">229-238</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) causes disease globally, with an estd. 25 to 100 million new infections per yr.  At present, no effective vaccine is available, and treatment is supportive.  In this study, we identified BP2109, a potent and selective small-mol. inhibitor of the DENV NS2B/NS3 protease, by a high-throughput screening assay using a recombinant protease complex consisting of the central hydrophilic portion of NS2B and the N terminus of the protease domain.  BP2109 inhibited DENV (serotypes 1 to 4), but not Japanese encephalitis virus (JEV), replication and viral RNA synthesis without detectable cytotoxicity.  The compd. inhibited recombinant DENV-2 NS2B/NS3 protease with a 50% inhibitory concn. (IC50) of 15.43 ± 2.12 μM and reduced the reporter expression of the DENV-2 replicon with a 50% effective concn. (EC50) of 0.17 ± 0.01 μM.  Sequencing analyses of several individual clones derived from BP2109-resistant DENV-2 RNAs revealed that two amino acid substitutions (R55K and E80K) are found in the region of NS2B, a cofactor of the NS2B/NS3 protease complex.  The introduction of R55K and E80K double mutations into the dengue virus NS2B/NS3 protease and a dengue virus replicon construct conferred 10.3- and 73.8-fold resistance to BP2109, resp.  The E80K mutation was further detd. to be the key mutation conferring dengue virus replicon resistance (61.3-fold) to BP2109, whereas the R55K mutation alone did not affect resistance to BP2109.  Both the R55K and E80K mutations are located in the central hydrophilic portion of the NS2B cofactor, where extensive interactions with the NS3pro domain exist.  Thus, our data provide evidence that BP2109 likely inhibits DENV by a novel mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRD8XjZuvDP7Vg90H21EOLACvtfcHk0lik9M907YEzSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVehu7k%253D&md5=e1a0728b4d1077399a005f7a84dad1e2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FAAC.00855-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00855-10%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.-C.%26aulast%3DHsieh%26aufirst%3DY.-C.%26aulast%3DLee%26aufirst%3DS.-J.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DLiao%26aufirst%3DC.-L.%26aulast%3DTsao%26aufirst%3DC.-H.%26aulast%3DChao%26aufirst%3DY.-S.%26aulast%3DChern%26aufirst%3DJ.-H.%26aulast%3DWu%26aufirst%3DC.-P.%26aulast%3DYueh%26aufirst%3DA.%26atitle%3DNovel%2520Dengue%2520Virus-Specific%2520NS2B%252FNS3%2520Protease%2520Inhibitor%252C%2520BP2109%252C%2520Discovered%2520by%2520a%2520High-Throughput%2520Screening%2520Assay%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D229%26epage%3D238%26doi%3D10.1128%2FAAC.00855-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiaang, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chern, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yueh, A.</span></span> <span> </span><span class="NLM_article-title">A Novel Dengue Virus Inhibitor, BP13944, Discovered by High-Throughput Screening with Dengue Virus Replicon Cells Selects for Resistance in the Viral NS2B/NS3 Protease</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1128/AAC.01281-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FAAC.01281-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=24145533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFCitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=110-119&author=C.-C.+Yangauthor=H.-S.+Huauthor=R.-H.+Wuauthor=S.-H.+Wuauthor=S.-J.+Leeauthor=W.-T.+Jiaangauthor=J.-H.+Chernauthor=Z.-S.+Huangauthor=H.-N.+Wuauthor=C.-M.+Changauthor=A.+Yueh&title=A+Novel+Dengue+Virus+Inhibitor%2C+BP13944%2C+Discovered+by+High-Throughput+Screening+with+Dengue+Virus+Replicon+Cells+Selects+for+Resistance+in+the+Viral+NS2B%2FNS3+Protease&doi=10.1128%2FAAC.01281-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A novel dengue virus inhibitor, BP13944, discovered by high-throughput screening with dengue virus replicon cells selects for resistance in the viral NS2B/NS3 protease</span></div><div class="casAuthors">Yang, Chi-Chen; Hu, Han-Shu; Wu, Ren-Huang; Wu, Szu-Huei; Lee, Shiow-Ju; Jiaang, Weir-Torn; Chern, Jyh-Haur; Huang, Zhi-Shun; Wu, Huey-Nan; Chang, Chung-Ming; Yueh, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">110-119, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Dengue virus (DENV) causes disease globally, resulting in an estd. 25 to 100 million new infections per yr.  No effective DENV vaccine is available, and the current treatment is only supportive.  Thus, there is an urgent need to develop therapeutic agents to cure this epidemic disease.  In the present study, we identified a potential small-mol. inhibitor, BP13944, via high-throughput screening (HTS) of 60,000 compds. using a stable cell line harboring an efficient luciferase replicon of DENV serotype 2 (DENV-2).  BP13944 reduced the expression of the DENV replicon reporter in cells, showing a 50% effective concn. (EC50) of 1.03 ± 0.09 μM.  Without detectable cytotoxicity, the compd. inhibited replication or viral RNA synthesis in all four serotypes of DENV but not in Japanese encephalitis virus (JEV).  Sequencing analyses of several individual clones derived from BP13944-resistant RNAs purified from cells harboring the DENV-2 replicon revealed a consensus amino acid substitution (E66G) in the region of the NS3 protease domain.  Introduction of E66G into the DENV replicon, an infectious DENV cDNA clone, and recombinant NS2B/NS3 protease constructs conferred 15.2-, 17.2-, and 3.1-fold resistance to BP13944, resp.  Our results identify an effective small-mol. inhibitor, BP13944, which likely targets the DENV NS3 protease.  BP13944 could be considered part of a more effective treatment regime for inhibiting DENV in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtuEbfgxAqMLVg90H21EOLACvtfcHk0lik9M907YEzSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFCitb8%253D&md5=445bed8eb7106964aaaf626a318c999f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FAAC.01281-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01281-13%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.-C.%26aulast%3DHu%26aufirst%3DH.-S.%26aulast%3DWu%26aufirst%3DR.-H.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DLee%26aufirst%3DS.-J.%26aulast%3DJiaang%26aufirst%3DW.-T.%26aulast%3DChern%26aufirst%3DJ.-H.%26aulast%3DHuang%26aufirst%3DZ.-S.%26aulast%3DWu%26aufirst%3DH.-N.%26aulast%3DChang%26aufirst%3DC.-M.%26aulast%3DYueh%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Dengue%2520Virus%2520Inhibitor%252C%2520BP13944%252C%2520Discovered%2520by%2520High-Throughput%2520Screening%2520with%2520Dengue%2520Virus%2520Replicon%2520Cells%2520Selects%2520for%2520Resistance%2520in%2520the%2520Viral%2520NS2B%252FNS3%2520Protease%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D110%26epage%3D119%26doi%3D10.1128%2FAAC.01281-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of Dengue Virus Protease Inhibitors</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2009.02.190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.antiviral.2009.02.190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19428601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVKrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2009&pages=110-114&author=S.+M.+Tomlinsonauthor=R.+D.+Malmstromauthor=A.+Russoauthor=N.+Muellerauthor=Y.-P.+Pangauthor=S.+J.+Watowich&title=Structure-Based+Discovery+of+Dengue+Virus+Protease+Inhibitors&doi=10.1016%2Fj.antiviral.2009.02.190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of dengue virus protease inhibitors</span></div><div class="casAuthors">Tomlinson, Suzanne M.; Malmstrom, Robert D.; Russo, Andrew; Mueller, Niklaus; Pang, Yuan-Ping; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">110-114</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dengue virus belongs to the family Flaviviridae and is a major emerging pathogen for which the development of vaccines and antiviral therapy has seen little success.  The NS3 viral protease is a potential target for antiviral drugs since it is required for virus replication.  The goal of this study was to identify novel dengue virus (type 2; DEN2V) protease inhibitors for eventual development as effective anti-flaviviral drugs.  The EUDOC docking program was used to computationally screen a small-mol. library for compds. that dock into the P1 pocket and the catalytic site of the DEN2V NS3 protease domain apo-structure and the Bowman-Birk inhibitor-bound structure.  The top 20 computer-identified hits that demonstrated the most favorable scoring energies were selected for in vitro assessment of protease inhibition.  Preliminary protease activity assays demonstrated that more than half of the tested compds. were sol. and exhibited in vitro inhibition of the DEN2V protease.  Two of these compds. also inhibited viral replication in cell culture expts., and thus are promising compds. for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4lR431in7_rVg90H21EOLACvtfcHk0ljxE8D4bepDkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVKrsbg%253D&md5=4d527895dc1cf644918c5b8f51ae0399</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2009.02.190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2009.02.190%26sid%3Dliteratum%253Aachs%26aulast%3DTomlinson%26aufirst%3DS.%2BM.%26aulast%3DMalmstrom%26aufirst%3DR.%2BD.%26aulast%3DRusso%26aufirst%3DA.%26aulast%3DMueller%26aufirst%3DN.%26aulast%3DPang%26aufirst%3DY.-P.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Dengue%2520Virus%2520Protease%2520Inhibitors%26jtitle%3DAntiviral%2520Res.%26date%3D2009%26volume%3D82%26spage%3D110%26epage%3D114%26doi%3D10.1016%2Fj.antiviral.2009.02.190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steuer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinonen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Optimization of Assay Conditions for Dengue Virus Protease: Effect of Various Polyols and Nonionic Detergents</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1102</span>– <span class="NLM_lpage">1108</span>, <span class="refDoi"> DOI: 10.1177/1087057109344115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1177%2F1087057109344115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19726784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVOltrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=1102-1108&author=C.+Steuerauthor=K.+H.+Heinonenauthor=L.+Kattnerauthor=C.+D.+Klein&title=Optimization+of+Assay+Conditions+for+Dengue+Virus+Protease%3A+Effect+of+Various+Polyols+and+Nonionic+Detergents&doi=10.1177%2F1087057109344115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of assay conditions for dengue virus protease: effect of various polyols and nonionic detergents</span></div><div class="casAuthors">Steuer, Christian; Heinonen, Karl H.; Kattner, Lars; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1102-1108</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The aim of this work was to perform a systematic study of the effect of nonionic detergents on the activity of the dengue virus NS2B-NS3 protease.  To ensure a high activity of the protease, the assay procedures for the dengue virus and other flaviviral proteases published to date are performed in the presence of up to 35% glycerol, which does not represent the cellular physicochem. environment.  In addn., the high viscosity of glycerol-contg. solns. leads to various exptl. problems in miniaturized assays.  Using an internally quenched peptide substrate, the authors show that glycerol is not essential for enzymic activity if certain nonionic detergents are added to the assay buffer.  In addn., nonionic detergents may help to avoid false-pos. screening results caused by "promiscuous" inhibitors.  Other polyalcs. can substitute glycerol and have less effect on the viscosity of the assay buffer.  The assay was used to screen a compd. library and allowed the identification of small-mol. nonpeptidic inhibitors of dengue NS3 protease.  Finally, the authors discuss the mode of action of nonionic detergents and the influence that they may have on the conformational properties of the NS2B-NS3 protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG3ngqZ602BbVg90H21EOLACvtfcHk0ljxE8D4bepDkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVOltrrP&md5=c6acb8d910b47af78048e59b633c6353</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F1087057109344115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109344115%26sid%3Dliteratum%253Aachs%26aulast%3DSteuer%26aufirst%3DC.%26aulast%3DHeinonen%26aufirst%3DK.%2BH.%26aulast%3DKattner%26aufirst%3DL.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DOptimization%2520of%2520Assay%2520Conditions%2520for%2520Dengue%2520Virus%2520Protease%253A%2520Effect%2520of%2520Various%2520Polyols%2520and%2520Nonionic%2520Detergents%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D1102%26epage%3D1108%26doi%3D10.1177%2F1087057109344115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altman, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barthel, H. R.</span></span> <span> </span><span class="NLM_article-title">Topical Therapies for Osteoarthritis</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1279</span>, <span class="refDoi"> DOI: 10.2165/11592550-000000000-00000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.2165%2F11592550-000000000-00000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=21770475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlakur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=1259-1279&author=R.+D.+Altmanauthor=H.+R.+Barthel&title=Topical+Therapies+for+Osteoarthritis&doi=10.2165%2F11592550-000000000-00000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Topical therapies for osteoarthritis</span></div><div class="casAuthors">Altman, Roy D.; Barthel, H. Richard</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1259-1279</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Adis Data Information BV</span>)
        </div><div class="casAbstract">This review discusses the pharmacol., analgesic efficacy, safety and tolerability of topical NSAIDs, salicylates and capsaicin for the management of osteoarthritis (OA) pain.  Topical therapies present a valuable therapeutic option for OA pain management, with substantial evidence supporting the efficacy and safety of topical NSAIDs, but less robust support for capsaicin and salicylates.  We define topical therapies as those intended to act locally, in contrast to transdermal therapies intended to act systemically.  Oral therapies for patients with mild to moderate OA pain include paracetamol (acetaminophen) and NSAIDs.  Paracetamol has only weak efficacy at therapeutic doses and is hepatotoxic at doses >3.25 g/day.  NSAIDs have demonstrated efficacy in patients with OA, but are assocd. with dose-, duration- and age-dependent risks of gastrointestinal, cardiovascular, renal, haematol. and hepatic adverse events (AEs), as well as clin. meaningful drug interactions.  To minimize AE risks, treatment guidelines for OA suggest minimizing NSAID exposure by prescribing the lowest ED for the shortest duration of time.  Systemic NSAID exposure may also be limited by prescribing topical NSAIDs, particularly in patients with OA pain limited to a few superficial joints.  Topical NSAIDs have been available in Europe for decades and were introduced to provide localized analgesia with minimal systemic NSAID exposure.  Guidelines of the American Academy of Orthopaedic Surgeons, European League Against Rheumatism (EULAR), Osteoarthritis Research Society International, and National Institute for Health and Clin. Excellence (NICE) state that topical NSAIDs may be considered for patients with mild to moderate OA of the knee or hand, particularly in patients with few affected joints and/or a history of sensitivity to oral NSAIDs.  In fact, the EULAR and NICE guidelines state that topical NSAIDs should be considered before oral therapies.  Clin. trials of topical NSAIDs, most notably formulations of diclofenac and ketoprofen, have shown efficacy significantly superior to placebo and similar to oral NSAIDs.  Most topical NSAIDs (piroxicam being the exception) have shown improved safety and tolerability compared with oral NSAIDs.  Topical salicylates and capsaicin are available in the US without a prescription, but neither has shown substantial efficacy in clin. trials, and both have potential to cause serious AEs.  Accidental poisonings have been reported with salicylates, and concerns exist that capsaicin-induced nerve desensitization is not fully reversible and that its autonomic nerve effects may increase the risk of skin ulcers in diabetic patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdu0AHAnb-yLVg90H21EOLACvtfcHk0ljxE8D4bepDkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlakur%252FE&md5=ec9dec1bfe57012584cb5d901537be9c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.2165%2F11592550-000000000-00000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F11592550-000000000-00000%26sid%3Dliteratum%253Aachs%26aulast%3DAltman%26aufirst%3DR.%2BD.%26aulast%3DBarthel%26aufirst%3DH.%2BR.%26atitle%3DTopical%2520Therapies%2520for%2520Osteoarthritis%26jtitle%3DDrugs%26date%3D2011%26volume%3D71%26spage%3D1259%26epage%3D1279%26doi%3D10.2165%2F11592550-000000000-00000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murawski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespi, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balimane, P. V.</span></span> <span> </span><span class="NLM_article-title">A Novel Design of Artificial Membrane for Improving the PAMPA Model</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1511</span>– <span class="NLM_lpage">1520</span>, <span class="refDoi"> DOI: 10.1007/s11095-007-9517-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1007%2Fs11095-007-9517-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=18185985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmslWntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2008&pages=1511-1520&author=X.+Chenauthor=A.+Murawskiauthor=K.+Patelauthor=C.+L.+Crespiauthor=P.+V.+Balimane&title=A+Novel+Design+of+Artificial+Membrane+for+Improving+the+PAMPA+Model&doi=10.1007%2Fs11095-007-9517-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Design of Artificial Membrane for Improving the Parallel Artificial Membrane Permeability Assay Model</span></div><div class="casAuthors">Chen, Xiaoxi; Murawski, Anthony; Patel, Karishma; Crespi, Charles L.; Balimane, Praveen V.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1511-1520</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose Since the first demonstration of parallel artificial membrane permeability assay (PAMPA), the artificial membrane has been traditionally prepd. by impregnating a porous filter with a soln. of lipid mixt.  While the lipid soln.-based method is simple and seems to provide good predictability for many compds., it is challenged by several shortcomings including reproducibility, stability, mass retention and the incorrect prediction of a group of highly permeable compds. including caffeine and antipyrine.  Here we present the validation of a novel artificial membrane formed by constructing a lipid/oil/lipid tri-layer in the porous filter.  Methods Permeability values obtained from traditional and new artificial membrane were compared for their correlation with Caco-2 and human absorption values.  Mass retention, stability and org. solvent compatibility of the new artificial membrane were studied.  Results The new artificial membrane correctly predicts the permeability of the traditionally under-predicted compds. and improves the correlation with Caco-2 and human absorption values.  Furthermore, the new artificial membrane reduces the mass retention of compds. that are highly retained by the traditional artificial membrane.  The new artificial membrane is also found to be robust enough to sustain long term storage and has good compatibility with org. solvents.  Conclusions The new artificial membrane provides an improved PAMPA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgXqaJqeAUybVg90H21EOLACvtfcHk0lhD4VpdQnaH8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmslWntLo%253D&md5=b076352ee4cc2935c97afb5437cdd496</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs11095-007-9517-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-007-9517-8%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMurawski%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DCrespi%26aufirst%3DC.%2BL.%26aulast%3DBalimane%26aufirst%3DP.%2BV.%26atitle%3DA%2520Novel%2520Design%2520of%2520Artificial%2520Membrane%2520for%2520Improving%2520the%2520PAMPA%2520Model%26jtitle%3DPharm.%2520Res.%26date%3D2008%26volume%3D25%26spage%3D1511%26epage%3D1520%26doi%3D10.1007%2Fs11095-007-9517-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lhD4VpdQnaH8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N. J.</span>; <span class="NLM_string-name">Hinner, M. J.</span></span> <span> </span><span class="NLM_article-title">Getting across the Cell Membrane: An Overview for Small Molecules, Peptides, and Proteins</span>. In  <i>Site-Specific Protein Labeling: Methods and Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautier, A.</span>; <span class="NLM_string-name">Hinner, M. J.</span></span>, Eds. <span class="NLM_publisher-name">Springer New York</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">53</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1007%2F978-1-4939-2272-7_3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=29-53&author=N.+J.+Yang&author=M.+J.+Hinnerauthor=A.+Gautier&author=M.+J.+Hinner&title=Site-Specific+Protein+Labeling%3A+Methods+and+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2272-7_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-2272-7_3%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DN.%2BJ.%26atitle%3DGetting%2520across%2520the%2520Cell%2520Membrane%253A%2520An%2520Overview%2520for%2520Small%2520Molecules%252C%2520Peptides%252C%2520and%2520Proteins%26btitle%3DSite-Specific%2520Protein%2520Labeling%253A%2520Methods%2520and%2520Protocols%26aulast%3DGautier%26aufirst%3DA.%26pub%3DSpringer%2520New%2520York%26date%3D2015%26spage%3D29%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Zika Virus NS2B-NS3 Protease in Complex with a Boronate Inhibitor</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">503</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1126/science.aag2419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1126%2Fscience.aag2419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=27386922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ehtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=503-505&author=J.+Leiauthor=G.+Hansenauthor=C.+Nitscheauthor=C.+D.+Kleinauthor=L.+Zhangauthor=R.+Hilgenfeld&title=Crystal+Structure+of+Zika+Virus+NS2B-NS3+Protease+in+Complex+with+a+Boronate+Inhibitor&doi=10.1126%2Fscience.aag2419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of Zika virus NS2B-NS3 protease in complex with a boronate inhibitor</span></div><div class="casAuthors">Lei, Jian; Hansen, Guido; Nitsche, Christoph; Klein, Christian D.; Zhang, Linlin; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6298</span>),
    <span class="NLM_cas:pages">503-505</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The ongoing Zika virus (ZIKV) outbreak is linked to severe neurol. disorders.  ZIKV relies on its NS2B/NS3 protease for polyprotein processing; hence, this enzyme is an attractive drug target.  The 2.7 angstrom; crystal structure of ZIKV protease in complex with a peptidomimetic boronic acid inhibitor reveals a cyclic diester between the boronic acid and glycerol.  The P2 4-aminomethylphenylalanine moiety of the inhibitor forms a salt-bridge with the nonconserved Asp83 of NS2B; ion-pairing between Asp83 and the P2 residue of the substrate likely accounts for the enzyme's high catalytic efficiency.  The unusual dimer of the ZIKV protease:inhibitor complex seen in the crystal may provide a model for assemblies formed at high local concns. of protease at the endoplasmatic reticulum membrane, the site of polyprotein processing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqIQ6GCMppIbVg90H21EOLACvtfcHk0lhD4VpdQnaH8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ehtL%252FM&md5=32c40f1d5060b22f4cd2e781433fdee4</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1126%2Fscience.aag2419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aag2419%26sid%3Dliteratum%253Aachs%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520Zika%2520Virus%2520NS2B-NS3%2520Protease%2520in%2520Complex%2520with%2520a%2520Boronate%2520Inhibitor%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D503%26epage%3D505%26doi%3D10.1126%2Fscience.aag2419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodenreider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonntag, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shochat, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otting, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span> <span> </span><span class="NLM_article-title">A Fluorescence Quenching Assay to Discriminate between Specific and Nonspecific Inhibitors of Dengue Virus Protease</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2009.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.ab.2009.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=19682971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GmsrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2009&pages=195-204&author=C.+Bodenreiderauthor=D.+Beerauthor=T.+H.+Kellerauthor=S.+Sonntagauthor=D.+Wenauthor=L.+Yapauthor=Y.+H.+Yauauthor=S.+G.+Shochatauthor=D.+Huangauthor=T.+Zhouauthor=A.+Caflischauthor=X.-C.+Suauthor=K.+Ozawaauthor=G.+Ottingauthor=S.+G.+Vasudevanauthor=J.+Lescarauthor=S.+P.+Lim&title=A+Fluorescence+Quenching+Assay+to+Discriminate+between+Specific+and+Nonspecific+Inhibitors+of+Dengue+Virus+Protease&doi=10.1016%2Fj.ab.2009.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence quenching assay to discriminate between specific and nonspecific inhibitors of dengue virus protease</span></div><div class="casAuthors">Bodenreider, Christophe; Beer, David; Keller, Thomas H.; Sonntag, Sebastian; Wen, Daying; Yap, Li Jian; Yau, Yin Hoe; Shochat, Susana Geifman; Huang, Danzhi; Zhou, Ting; Caflisch, Amedeo; Su, Xun-Cheng; Ozawa, Kiyoshi; Otting, Gottfried; Vasudevan, Subhash G.; Lescar, Julien; Lim, Siew Pheng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">195-204</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In drug discovery, the occurrence of false positives is a major hurdle in the search for lead compds. that can be developed into drugs.  A small-mol.-wt. compd. that inhibits dengue virus protease at low micromolar levels was identified in a screening campaign.  Binding to the enzyme was confirmed by isothermal titrn. calorimetry (ITC) and NMR.  However, a structure-activity relationship study that ensued did not yield more potent leads.  To further characterize the parental compd. and its analogs, we developed a high-speed, low-cost, quant. fluorescence quenching assay.  We obsd. that specific analogs quenched dengue protease fluorescence and showed variation in IC50 values.  In contrast, nonspecifically binding compds. did not quench its fluorescence and showed similar IC50 values with steep dose-response curves.  We validated the assay using single Trp-to-Ala protease mutants and the competitive protease inhibitor aprotinin.  Specific compds. detected in the binding assay were further analyzed by competitive ITC, NMR, and surface plasmon resonance, and the assay's utility in comparison with these biophys. methods is discussed.  The sensitivity of this assay makes it highly useful for hit finding and validation in drug discovery.  Furthermore, the technique can be readily adapted for studying other protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIeUje_AwhZ7Vg90H21EOLACvtfcHk0lgtFUBQeuKGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GmsrfI&md5=ca035ecd54f7c72e77d03ed5afe69d18</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2009.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2009.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DBodenreider%26aufirst%3DC.%26aulast%3DBeer%26aufirst%3DD.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DSonntag%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DD.%26aulast%3DYap%26aufirst%3DL.%26aulast%3DYau%26aufirst%3DY.%2BH.%26aulast%3DShochat%26aufirst%3DS.%2BG.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DSu%26aufirst%3DX.-C.%26aulast%3DOzawa%26aufirst%3DK.%26aulast%3DOtting%26aufirst%3DG.%26aulast%3DVasudevan%26aufirst%3DS.%2BG.%26aulast%3DLescar%26aufirst%3DJ.%26aulast%3DLim%26aufirst%3DS.%2BP.%26atitle%3DA%2520Fluorescence%2520Quenching%2520Assay%2520to%2520Discriminate%2520between%2520Specific%2520and%2520Nonspecific%2520Inhibitors%2520of%2520Dengue%2520Virus%2520Protease%26jtitle%3DAnal.%2520Biochem.%26date%3D2009%26volume%3D395%26spage%3D195%26epage%3D204%26doi%3D10.1016%2Fj.ab.2009.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brieke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cryle, M. J.</span></span> <span> </span><span class="NLM_article-title">A Facile Fmoc Solid Phase Synthesis Strategy to Access Epimerization-Prone Biosynthetic Intermediates of Glycopeptide Antibiotics</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2454</span>– <span class="NLM_lpage">2457</span>, <span class="refDoi"> DOI: 10.1021/ol500840f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol500840f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmt1ShsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=2454-2457&author=C.+Briekeauthor=M.+J.+Cryle&title=A+Facile+Fmoc+Solid+Phase+Synthesis+Strategy+to+Access+Epimerization-Prone+Biosynthetic+Intermediates+of+Glycopeptide+Antibiotics&doi=10.1021%2Fol500840f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A Facile Fmoc Solid Phase Synthesis Strategy To Access Epimerization-Prone Biosynthetic Intermediates of Glycopeptide Antibiotics</span></div><div class="casAuthors">Brieke, Clara; Cryle, Max J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2454-2457</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A rapid protocol based on Fmoc-chem. for the solid-phase peptide synthesis of vancomycin- and teicoplanin-type peptides is described.  Epimerization of highly racemization-prone arylglycine derivs. is suppressed through optimized Fmoc-deprotection and coupling conditions.  Starting from easily accessible Fmoc-protected amino acids, this strategy enables the enantioselective synthesis of peptides corresponding to intermediates found in vancomycin and teicoplanin biosynthesis with excellent purity and in high yields (38%-71%).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Z8GnYCsJU7Vg90H21EOLACvtfcHk0lgtFUBQeuKGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmt1ShsbY%253D&md5=cee6106a787cbb147bcdcd07c66caf2e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fol500840f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol500840f%26sid%3Dliteratum%253Aachs%26aulast%3DBrieke%26aufirst%3DC.%26aulast%3DCryle%26aufirst%3DM.%2BJ.%26atitle%3DA%2520Facile%2520Fmoc%2520Solid%2520Phase%2520Synthesis%2520Strategy%2520to%2520Access%2520Epimerization-Prone%2520Biosynthetic%2520Intermediates%2520of%2520Glycopeptide%2520Antibiotics%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D2454%26epage%3D2457%26doi%3D10.1021%2Fol500840f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elsawy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewage, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, B.</span></span> <span> </span><span class="NLM_article-title">Racemisation of N-Fmoc Phenylglycine under Mild Microwave-SPPS and Conventional Stepwise SPPS Conditions: Attempts to Develop Strategies for Overcoming This</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">302</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1002/psc.2398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fpsc.2398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=22451378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC38XksV2hu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=302-311&author=M.+A.+Elsawyauthor=C.+Hewageauthor=B.+Walker&title=Racemisation+of+N-Fmoc+Phenylglycine+under+Mild+Microwave-SPPS+and+Conventional+Stepwise+SPPS+Conditions%3A+Attempts+to+Develop+Strategies+for+Overcoming+This&doi=10.1002%2Fpsc.2398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Racemisation of N-Fmoc phenylglycine under mild microwave-SPPS and conventional stepwise SPPS conditions: attempts to develop strategies for overcoming this</span></div><div class="casAuthors">Elsawy, Mohamed A.; Hewage, Chandralal; Walker, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">302-311</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">We have been engaged in the microwave-solid phase peptide synthesis (SPPS) synthesis of the phenylglycine (Phg)-contg. pentapeptide H-Ala-Val-Pro-Phg-Tyr-NH2 previously demonstrated to bind to the so-called BIR3 domain of the anti-apoptotic protein XIAP.  Anal. of the target peptide by a combination of RP-HPLC, ESI-MS, and NMR revealed the presence of two diastereoisomers arising out of the racemization of the Phg residue, with the percentage of the LLLDL component assessed as 49%.  We performed the synthesis of peptide H-Ala-Val-Pro-Phg-Tyr-NH2 using different microwave and conventional stepwise SPPS conditions in attempts to reduce the level of racemization of the Phg residue and to det. at which part of the synthetic cycle the epimerization had occurred.  We detd. that racemization occurred mainly during the Fmoc-group removal and, to a much lesser extent, during activation/coupling of the Fmoc-Phg-OH residue.  We were able to obtain the desired peptide with a 71% diastereomeric purity (29% LLLDL as impurity) by utilizing microwave-assisted SPPS at 50 °C and power 22 W, when the triazine-derived coupling reagent DMTMM-BF4 was used, together with NMM as an activator base, for the incorporation of this residue and 20% piperidine as an Fmoc-deprotection base.  In contrast, the phenylalanine analog of the above peptide, H-Ala-Val-Pro-Phe-Tyr-NH2, was always obtained as a single diastereoisomer by using a range of std. coupling and deprotection conditions.  Our findings suggest that the racemization of Fmoc-Phg-OH, under both microwave-SPPS and stepwise conventional SPPS syntheses conditions, is very facile but can be limited through the use of the above stated conditions.  Copyright © 2012 European Peptide Society and John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHTPUtPSTeELVg90H21EOLACvtfcHk0lgtFUBQeuKGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XksV2hu7s%253D&md5=aef27234860e27672c7d6b76797c661d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2398%26sid%3Dliteratum%253Aachs%26aulast%3DElsawy%26aufirst%3DM.%2BA.%26aulast%3DHewage%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DB.%26atitle%3DRacemisation%2520of%2520N-Fmoc%2520Phenylglycine%2520under%2520Mild%2520Microwave-SPPS%2520and%2520Conventional%2520Stepwise%2520SPPS%2520Conditions%253A%2520Attempts%2520to%2520Develop%2520Strategies%2520for%2520Overcoming%2520This%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2012%26volume%3D18%26spage%3D302%26epage%3D311%26doi%3D10.1002%2Fpsc.2398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for Flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0liMZWqyQn0Vsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Örtqvist, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Åkerblom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gossas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fransson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helena Danielson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandström, A.</span></span> <span> </span><span class="NLM_article-title">Phenylglycine as a Novel P2 Scaffold in Hepatitis C Virus NS3 Protease Inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1448</span>– <span class="NLM_lpage">1474</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2006.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmc.2006.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=17113777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FhtVertg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=1448-1474&author=P.+%C3%96rtqvistauthor=S.+D.+Petersonauthor=E.+%C3%85kerblomauthor=T.+Gossasauthor=Y.+A.+Sabnisauthor=R.+Franssonauthor=G.+Lindebergauthor=U.+Helena+Danielsonauthor=A.+Karl%C3%A9nauthor=A.+Sandstr%C3%B6m&title=Phenylglycine+as+a+Novel+P2+Scaffold+in+Hepatitis+C+Virus+NS3+Protease+Inhibitors&doi=10.1016%2Fj.bmc.2006.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylglycine as a novel P2 scaffold in hepatitis C virus NS3 protease inhibitors</span></div><div class="casAuthors">Ortqvist Pernilla; Peterson Shane D; Kerblom Eva; Gossas Thomas; Sabnis Yogesh A; Fransson Rebecca; Lindeberg Gunnar; Helena Danielson U; Karlen Anders; Sandstrom Anja</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1448-74</span>
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    </div><div class="casAbstract">Molecular modeling and inhibitory potencies of tetrapeptide protease inhibitors of HCV NS3 proposed phenylglycine as a new promising P2 residue.  The results suggest that phenylglycine might be capable of interacting with the NS3 (protease-helicase/NTPase) in ways not possible for the common P2 proline-based inhibitors.  Thus, a series of tripeptides, both linear and macrocyclic, based on p-hydroxy-phenylglycine in the P2 position were prepared and their inhibitory effect determined.  When the p-hydroxy group was replaced by methoxy, isoquinolin-, or quinolinyloxy functions, inhibitors with improved potencies were obtained.  The P2 phenylglycine-based inhibitors were further optimized by C-terminal extension to acyl sulfonamides and by P1-P3 cyclization, which gave products with inhibition constants in the nanomolar range ( approximately 75nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ3vpve6fi1Ar5oR8RchjrrfW6udTcc2eaJ22KGX5C0Wbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FhtVertg%253D%253D&md5=3c7a55b59840dc761e76ad0c1a029c8d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2006.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2006.11.003%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596rtqvist%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DS.%2BD.%26aulast%3D%25C3%2585kerblom%26aufirst%3DE.%26aulast%3DGossas%26aufirst%3DT.%26aulast%3DSabnis%26aufirst%3DY.%2BA.%26aulast%3DFransson%26aufirst%3DR.%26aulast%3DLindeberg%26aufirst%3DG.%26aulast%3DHelena%2BDanielson%26aufirst%3DU.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26aulast%3DSandstr%25C3%25B6m%26aufirst%3DA.%26atitle%3DPhenylglycine%2520as%2520a%2520Novel%2520P2%2520Scaffold%2520in%2520Hepatitis%2520C%2520Virus%2520NS3%2520Protease%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2007%26volume%3D15%26spage%3D1448%26epage%3D1474%26doi%3D10.1016%2Fj.bmc.2006.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lampa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenberg, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vema, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Åkerblom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandström, A.</span></span> <span> </span><span class="NLM_article-title">Improved P2 Phenylglycine-Based Hepatitis C Virus NS3 Protease Inhibitors with Alkenylic Prime-Side Substituents</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5413</span>– <span class="NLM_lpage">5424</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.05.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1016%2Fj.bmc.2010.05.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=20541424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXoslans78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=5413-5424&author=A.+Lampaauthor=A.+E.+Ehrenbergauthor=S.+S.+Gustafssonauthor=A.+Vemaauthor=E.+%C3%85kerblomauthor=G.+Lindebergauthor=A.+Karl%C3%A9nauthor=U.+H.+Danielsonauthor=A.+Sandstr%C3%B6m&title=Improved+P2+Phenylglycine-Based+Hepatitis+C+Virus+NS3+Protease+Inhibitors+with+Alkenylic+Prime-Side+Substituents&doi=10.1016%2Fj.bmc.2010.05.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Improved P2 phenylglycine-based hepatitis C virus NS3 protease inhibitors with alkenylic prime-side substituents</span></div><div class="casAuthors">Lampa, Anna; Ehrenberg, Angelica E.; Gustafsson, Sofia S.; Vema, Aparna; Aakerblom, Eva; Lindeberg, Gunnar; Karlen, Anders; Danielson, U. Helena; Sandstroem, Anja</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5413-5424</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phenylglycine has proven to be a useful P2 residue in HCV NS3 protease inhibitors.  A novel π-π-interaction between the phenylglycine and the catalytic H57 residue of the protease was postulated, and it was hypothesized that the introduction of a vinyl on the phenylglycine might strengthen this π-π-interaction.  Thus, a series of acyclic vinylated phenylglycine-based HCV NS3 protease inhibitors, e.g., I, was synthesized and their inhibitory potency evaluated.  Surprisingly, inhibitors based on both D- and L-phenylglycine were found to be effective inhibitors, with a slight preference for the D-epimers.  Furthermore, prime-side alkenylic extension of the C-terminal acylsulfonamide group gave significantly improved inhibitors with potencies in the nanomolar range (∼35 nM), potencies which were retained on mutant variants of the protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1S6r1UZnWFrVg90H21EOLACvtfcHk0liMZWqyQn0Vsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXoslans78%253D&md5=53f795b5eba3e0b2d27e74fd504229a0</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.05.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.05.027%26sid%3Dliteratum%253Aachs%26aulast%3DLampa%26aufirst%3DA.%26aulast%3DEhrenberg%26aufirst%3DA.%2BE.%26aulast%3DGustafsson%26aufirst%3DS.%2BS.%26aulast%3DVema%26aufirst%3DA.%26aulast%3D%25C3%2585kerblom%26aufirst%3DE.%26aulast%3DLindeberg%26aufirst%3DG.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DSandstr%25C3%25B6m%26aufirst%3DA.%26atitle%3DImproved%2520P2%2520Phenylglycine-Based%2520Hepatitis%2520C%2520Virus%2520NS3%2520Protease%2520Inhibitors%2520with%2520Alkenylic%2520Prime-Side%2520Substituents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D5413%26epage%3D5424%26doi%3D10.1016%2Fj.bmc.2010.05.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera—a Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lgVsndDGJw-cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postnikova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWald, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logue, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deiuliis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelotti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honko, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbrook, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olinger, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hensley, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahrling, P. B.</span></span> <span> </span><span class="NLM_article-title">Testing Therapeutics in Cell-Based Assays: Factors That Influence the Apparent Potency of Drugs</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e0194880</span>– <span class="NLM_lpage">e0194880</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0194880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1371%2Fjournal.pone.0194880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=29566079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslenur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0194880-e0194880&author=E.+Postnikovaauthor=Y.+Congauthor=L.+E.+DeWaldauthor=J.+Dyallauthor=S.+Yuauthor=B.+J.+Hartauthor=H.+Zhouauthor=R.+Grossauthor=J.+Logueauthor=Y.+Caiauthor=N.+Deiuliisauthor=J.+Michelottiauthor=A.+N.+Honkoauthor=R.+S.+Bennettauthor=M.+R.+Holbrookauthor=G.+G.+Olingerauthor=L.+E.+Hensleyauthor=P.+B.+Jahrling&title=Testing+Therapeutics+in+Cell-Based+Assays%3A+Factors+That+Influence+the+Apparent+Potency+of+Drugs&doi=10.1371%2Fjournal.pone.0194880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Testing therapeutics in cell-based assays: factors that influence the apparent potency of drugs</span></div><div class="casAuthors">Postnikova, Elena; Cong, Yu; DeWald, Lisa Evans; Dyall, Julie; Yu, Shuiqing; Hart, Brit J.; Zhou, Huanying; Gross, Robin; Logue, James; Cai, Yingyun; Deiuliis, Nicole; Michelotti, Julia; Honko, Anna N.; Bennett, Richard S.; Holbrook, Michael R.; Olinger, Gene G.; Hensley, Lisa E.; Jahrling, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0194880/1-e0194880/18</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Identifying effective antivirals for treating Ebola virus disease (EVD) and minimizing transmission of such disease is crit.  A variety of cell-based assays have been developed for evaluating compds. for activity against Ebola virus.  However, very few reports discuss the variable assay conditions that can affect the results obtained from these drug screens.  Here, we describe variable conditions tested during the development of our cell-based drug screen assays designed to identify compds. with anti-Ebola virus activity using established cell lines and human primary cells.  The effect of multiple assay readouts and variable assay conditions, including virus input, time of infection, and the cell passage no., were compared, and the impact on the effective concn. for 50% and/ or 90% inhibition (EC50, EC90) was evaluated using the FDA-approved compd., toremifene citrate.  In these studies, we show that altering cell-based assay conditions can have an impact on apparent drug potency as measured by the EC50.  These results further support the importance of developing std. operating procedures for generating reliable and reproducible in vitro data sets for potential antivirals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5MSjHQoAi4bVg90H21EOLACvtfcHk0lgVsndDGJw-cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslenur7M&md5=a92120e9cba3ec410044b4301c8de218</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0194880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0194880%26sid%3Dliteratum%253Aachs%26aulast%3DPostnikova%26aufirst%3DE.%26aulast%3DCong%26aufirst%3DY.%26aulast%3DDeWald%26aufirst%3DL.%2BE.%26aulast%3DDyall%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DHart%26aufirst%3DB.%2BJ.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DGross%26aufirst%3DR.%26aulast%3DLogue%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DDeiuliis%26aufirst%3DN.%26aulast%3DMichelotti%26aufirst%3DJ.%26aulast%3DHonko%26aufirst%3DA.%2BN.%26aulast%3DBennett%26aufirst%3DR.%2BS.%26aulast%3DHolbrook%26aufirst%3DM.%2BR.%26aulast%3DOlinger%26aufirst%3DG.%2BG.%26aulast%3DHensley%26aufirst%3DL.%2BE.%26aulast%3DJahrling%26aufirst%3DP.%2BB.%26atitle%3DTesting%2520Therapeutics%2520in%2520Cell-Based%2520Assays%253A%2520Factors%2520That%2520Influence%2520the%2520Apparent%2520Potency%2520of%2520Drugs%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26spage%3De0194880%26epage%3De0194880%26doi%3D10.1371%2Fjournal.pone.0194880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreier, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnam, M. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Thiazolidinone–Peptide Hybrids as Dengue Virus Protease Inhibitors with Antiviral Activity in Cell Culture</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8389</span>– <span class="NLM_lpage">8403</span>, <span class="refDoi"> DOI: 10.1021/jm400828u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400828u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8389-8403&author=C.+Nitscheauthor=V.+N.+Schreierauthor=M.+A.+M.+Behnamauthor=A.+Kumarauthor=R.+Bartenschlagerauthor=C.+D.+Klein&title=Thiazolidinone%E2%80%93Peptide+Hybrids+as+Dengue+Virus+Protease+Inhibitors+with+Antiviral+Activity+in+Cell+Culture&doi=10.1021%2Fjm400828u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Thiazolidinone-Peptide Hybrids as Dengue Virus Protease Inhibitors with Antiviral Activity in Cell Culture</span></div><div class="casAuthors">Nitsche, Christoph; Schreier, Verena N.; Behnam, Mira A. M.; Kumar, Anil; Bartenschlager, Ralf; Klein, Christian D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8389-8403</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protease of dengue virus is a promising target for antiviral drug discovery.  We here report a new generation of peptide-hybrid inhibitors of dengue protease that incorporate N-substituted 5-arylidenethiazolidinone heterocycles (rhodanines and thiazolidinediones) as N-terminal capping groups of the peptide moiety.  The compds. were extensively characterized with respect to inhibition of various proteases, inhibition mechanisms, membrane permeability, antiviral activity, and cytotoxicity in cell culture.  A sulfur/oxygen exchange in position 2 of the capping heterocycle (thiazolidinedione-capped vs rhodanine-capped peptide hybrids) has a significant effect on these properties and activities.  The most promising in vitro affinities were obsd. for thiazolidinedione-based peptide hybrids contg. hydrophobic groups with Ki values between 1.5 and 1.8 μM and competitive inhibition mechanisms.  Rhodanine-capped peptide hybrids with hydrophobic substituents have, in correlation with their membrane permeability, a more pronounced antiviral activity in cell culture than the thiazolidinediones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq_iHlDnsmD7Vg90H21EOLACvtfcHk0lgVsndDGJw-cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGrsrfJ&md5=ab9c63ae3682c8621347f7fd8145bb24</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm400828u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400828u%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26aulast%3DSchreier%26aufirst%3DV.%2BN.%26aulast%3DBehnam%26aufirst%3DM.%2BA.%2BM.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DKlein%26aufirst%3DC.%2BD.%26atitle%3DThiazolidinone%25E2%2580%2593Peptide%2520Hybrids%2520as%2520Dengue%2520Virus%2520Protease%2520Inhibitors%2520with%2520Antiviral%2520Activity%2520in%2520Cell%2520Culture%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8389%26epage%3D8403%26doi%3D10.1021%2Fjm400828u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span>; <span class="NLM_string-name">Klein, C. D.</span></span> <span> </span><span class="NLM_article-title">Fluorimetric and HPLC-Based Dengue Virus Protease Assays Using a FRET Substrate</span>. In  <i>Antiviral Methods and Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, E. Y.</span></span>, Ed. <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">Totowa, New Jersey</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">236</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1007%2F978-1-62703-484-5_18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=221-236&author=C.+Nitsche&author=C.+D.+Kleinauthor=E.+Y.+Gong&title=Antiviral+Methods+and+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2F978-1-62703-484-5_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-62703-484-5_18%26sid%3Dliteratum%253Aachs%26aulast%3DNitsche%26aufirst%3DC.%26atitle%3DFluorimetric%2520and%2520HPLC-Based%2520Dengue%2520Virus%2520Protease%2520Assays%2520Using%2520a%2520FRET%2520Substrate%26btitle%3DAntiviral%2520Methods%2520and%2520Protocols%26aulast%3DGong%26aufirst%3DE.%2BY.%26pub%3DHumana%2520Press%26date%3D2013%26spage%3D221%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiramel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatel-Chaix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvisi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora-Rodríguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamacher-Brady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span> <span> </span><span class="NLM_article-title">Dengue Virus Inhibition of Autophagic Flux and Dependency of Viral Replication on Proteasomal Degradation of the Autophagy Receptor p62</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">8026</span>– <span class="NLM_lpage">8041</span>, <span class="refDoi"> DOI: 10.1128/JVI.00787-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=10.1128%2FJVI.00787-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=26018155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyku7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=8026-8041&author=P.+Metzauthor=A.+Chiramelauthor=L.+Chatel-Chaixauthor=G.+Alvisiauthor=P.+Bankheadauthor=R.+Mora-Rodr%C3%ADguezauthor=G.+Longauthor=A.+Hamacher-Bradyauthor=N.+R.+Bradyauthor=R.+Bartenschlager&title=Dengue+Virus+Inhibition+of+Autophagic+Flux+and+Dependency+of+Viral+Replication+on+Proteasomal+Degradation+of+the+Autophagy+Receptor+p62&doi=10.1128%2FJVI.00787-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Dengue virus inhibition of autophagic flux and dependency of viral replication on proteasomal degradation of the autophagy receptor p62</span></div><div class="casAuthors">Metz, Philippe; Chiramel, Abhilash; Chatel-Chaix, Laurent; Alvisi, Gualtiero; Bankhead, Peter; Mora-Rodriguez, Rodrigo; Long, Gang; Hamacher-Brady, Anne; Brady, Nathan R.; Bartenschlager, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8026-8041</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Autophagic flux involves formation of autophagosomes and their degrdn. by lysosomes.  Autophagy can either promote or restrict viral replication.  In the case of Dengue virus (DENV), several studies report that autophagy supports the viral replication cycle, and describe an increase of autophagic vesicles (AVs) following infection.  However, it is unknown how autophagic flux is altered to result in increased AVs.  To address this question and gain insight into the role of autophagy during DENV infection, we established an unbiased, image-based flow cytometry approach to quantify autophagic flux under normal growth conditions and in response to activation by nutrient deprivation or the mTOR inhibitor Torin1.  We found that DENV induced an initial activation of autophagic flux, followed by inhibition of general and specific autophagy.  Early after infection, basal and activated autophagic flux was enhanced.  However, during established replication, basal and Torin1-activated autophagic flux was blocked, while autophagic flux activated by nutrient deprivation was reduced, indicating a block to AV formation and reduced AV degrdn. capacity.  During late infection AV levels increased as a result of inefficient fusion of autophagosomes with lysosomes.  In addn., endolysosomal trafficking was suppressed, while lysosomal activities were increased.  We further detd. that DENV infection progressively reduced levels of the autophagy receptor SQSTM1/p62 via proteasomal degrdn.  Importantly, stable overexpression of p62 significantly suppressed DENV replication, suggesting a novel role for p62 as a viral restriction factor.  Overall, our findings indicate that in the course of DENV infection, autophagy shifts from a supporting to an antiviral role, which is countered by DENV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAZ45b04tNDrVg90H21EOLACvtfcHk0ljoYfwILyhTfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyku7%252FO&md5=67cbffe54070d331d8057cf1565f5022</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1128%2FJVI.00787-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00787-15%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DP.%26aulast%3DChiramel%26aufirst%3DA.%26aulast%3DChatel-Chaix%26aufirst%3DL.%26aulast%3DAlvisi%26aufirst%3DG.%26aulast%3DBankhead%26aufirst%3DP.%26aulast%3DMora-Rodr%25C3%25ADguez%26aufirst%3DR.%26aulast%3DLong%26aufirst%3DG.%26aulast%3DHamacher-Brady%26aufirst%3DA.%26aulast%3DBrady%26aufirst%3DN.%2BR.%26aulast%3DBartenschlager%26aufirst%3DR.%26atitle%3DDengue%2520Virus%2520Inhibition%2520of%2520Autophagic%2520Flux%2520and%2520Dependency%2520of%2520Viral%2520Replication%2520on%2520Proteasomal%2520Degradation%2520of%2520the%2520Autophagy%2520Receptor%2520p62%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D8026%26epage%3D8041%26doi%3D10.1128%2FJVI.00787-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i83"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00413">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04343"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00413?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00413</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Expression and purification of the viral proteases, primers used for cloning of reporter constructs and the DENV-2 protease, reporter constructs used for co-transfection, assay validation, inhibitors used for assay correlation, DENV2proHeLa dose–response curves, kinetic studies, characterization of compound <b>51</b>, HPLC-based DENV and WNV protease assays, inhibitory activity of compounds against thrombin and trypsin, structures of reference compounds, PAMPA, FRET substrate synthesis, synthesis and analytical data of precursors, analytical data of inhibitors, HPLC purity of inhibitors, enantiomeric purity of inhibitors, and selected <sup>1</sup>H und <sup>13</sup>C NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and activity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_001.pdf">jm0c00413_si_001.pdf (3.57 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00413/suppl_file/jm0c00413_si_002.csv">jm0c00413_si_002.csv (9.14 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00413&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00413" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00413" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b4d0fce8d18f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
