
<html lang="en"     class="pb-page"  data-request-id="e1287456-ff90-4f4a-b3b1-7391161cefc2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-7;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c02160;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6" /></meta><meta name="dc.Creator" content="Yudao  Shen" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Magdalena M.  Szewczyk" /></meta><meta name="dc.Creator" content="Levon  Halabelian" /></meta><meta name="dc.Creator" content="Irene  Chau" /></meta><meta name="dc.Creator" content="Mohammad S.  Eram" /></meta><meta name="dc.Creator" content="Carlo  Dela Seña" /></meta><meta name="dc.Creator" content="Kwang-Su  Park" /></meta><meta name="dc.Creator" content="Fanye  Meng" /></meta><meta name="dc.Creator" content="He  Chen" /></meta><meta name="dc.Creator" content="Hong  Zeng" /></meta><meta name="dc.Creator" content="Aiping  Dong" /></meta><meta name="dc.Creator" content="Hong  Wu" /></meta><meta name="dc.Creator" content="Viacheslav V.  Trush" /></meta><meta name="dc.Creator" content="David  McLeod" /></meta><meta name="dc.Creator" content="Carlos A.  Zepeda-Velázquez" /></meta><meta name="dc.Creator" content="Robert M.  Campbell" /></meta><meta name="dc.Creator" content="Mary M.  Mader" /></meta><meta name="dc.Creator" content="Brian M.  Watson" /></meta><meta name="dc.Creator" content="Matthieu  Schapira" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Rima  Al-Awar" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="H. Ümit  Kaniskan" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Protein arginine methyltransferase 6 (PRMT6) catalyzes monomethylation and asymmetric dimethylation of arginine residues in various proteins, plays important roles in biological processes, and is a..." /></meta><meta name="Description" content="Protein arginine methyltransferase 6 (PRMT6) catalyzes monomethylation and asymmetric dimethylation of arginine residues in various proteins, plays important roles in biological processes, and is a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 16, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02160" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02160" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02160" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02160" /></link>
        
    
    

<title>A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02160" /></meta><meta property="og:title" content="A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0010.jpeg" /></meta><meta property="og:description" content="Protein arginine methyltransferase 6 (PRMT6) catalyzes monomethylation and asymmetric dimethylation of arginine residues in various proteins, plays important roles in biological processes, and is associated with multiple cancers. To date, a highly selective PRMT6 inhibitor has not been reported. Here we report the discovery and characterization of a first-in-class, highly selective allosteric inhibitor of PRMT6, (R)-2 (SGC6870). (R)-2 is a potent PRMT6 inhibitor (IC50 = 77 ± 6 nM) with outstanding selectivity for PRMT6 over a broad panel of other methyltransferases and nonepigenetic targets. Notably, the crystal structure of the PRMT6–(R)-2 complex and kinetic studies revealed (R)-2 binds a unique, induced allosteric pocket. Additionally, (R)-2 engages PRMT6 and potently inhibits its methyltransferase activity in cells. Moreover, (R)-2’s enantiomer, (S)-2 (SGC6870N), is inactive against PRMT6 and can be utilized as a negative control. Collectively, (R)-2 is a well-characterized PRMT6 chemical probe and a valuable tool for further investigating PRMT6 functions in health and disease." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02160"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02160">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02160&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02160&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02160&amp;href=/doi/10.1021/acs.jmedchem.0c02160" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3697-3706</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00473" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.0c01371" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yudao Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yudao Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yudao++Shen">Yudao Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fengling Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fengling Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Magdalena M. Szewczyk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Magdalena M. Szewczyk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena+M.++Szewczyk">Magdalena M. Szewczyk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Levon Halabelian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Levon Halabelian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Levon++Halabelian">Levon Halabelian</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4361-3619" title="Orcid link">http://orcid.org/0000-0003-4361-3619</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Irene Chau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Irene Chau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Irene++Chau">Irene Chau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mohammad S. Eram</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mohammad S. Eram</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mohammad+S.++Eram">Mohammad S. Eram</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlo Dela Seña</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlo Dela Seña</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlo++Dela+Se%C3%B1a">Carlo Dela Seña</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kwang-Su Park</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kwang-Su Park</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kwang-Su++Park">Kwang-Su Park</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fanye Meng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fanye Meng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fanye++Meng">Fanye Meng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">He Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">He Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=He++Chen">He Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong Zeng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong Zeng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Zeng">Hong Zeng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aiping Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aiping Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aiping++Dong">Aiping Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Wu">Hong Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Viacheslav V. Trush</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Viacheslav V. Trush</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Viacheslav+V.++Trush">Viacheslav V. Trush</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David McLeod</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David McLeod</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Ontario Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++McLeod">David McLeod</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlos A. Zepeda-Velázquez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlos A. Zepeda-Velázquez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Ontario Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlos+A.++Zepeda-Vel%C3%A1zquez">Carlos A. Zepeda-Velázquez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert M. Campbell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert M. Campbell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Eli Lilly and Company, Lilly Research Laboratories, Indianapolis, Indiana 46225, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+M.++Campbell">Robert M. Campbell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mary M. Mader</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mary M. Mader</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Eli Lilly and Company, Lilly Research Laboratories, Indianapolis, Indiana 46225, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mary+M.++Mader">Mary M. Mader</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brian M. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brian M. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Eli Lilly and Company, Lilly Research Laboratories, Indianapolis, Indiana 46225, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brian+M.++Watson">Brian M. Watson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthieu Schapira</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthieu Schapira</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthieu++Schapira">Matthieu Schapira</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link">http://orcid.org/0000-0002-1047-3309</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheryl H. Arrowsmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheryl H. Arrowsmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rima Al-Awar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rima Al-Awar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Ontario Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rima++Al-Awar">Rima Al-Awar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dalia Barsyte-Lovejoy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dalia Barsyte-Lovejoy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">H. Ümit Kaniskan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">H. Ümit Kaniskan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=H.+%C3%9Cmit++Kaniskan">H. Ümit Kaniskan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5327-832X" title="Orcid link">http://orcid.org/0000-0001-5327-832X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter J. Brown</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. Brown</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#0171647564736b2f63736e766f4174756e736e6f756e2f6260"><span class="__cf_email__" data-cfemail="7b0b1e0f1e0911551909140c153b0e0f140914150f1455181a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masoud Vedadi</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masoud Vedadi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ff92d1899a9b9e9b96bf8a8b908d90918b90d19c9e"><span class="__cf_email__" data-cfemail="b0dd9ec6d5d4d1d4d9f0c5c4dfc2dfdec4df9ed3d1">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c6acafa7a8e8acafa886abb5b5abe8a3a2b3"><span class="__cf_email__" data-cfemail="402a29212e6e2a292e002d33332d6e252435">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02160&amp;href=/doi/10.1021%2Facs.jmedchem.0c02160" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3697–3706</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 16, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 December 2020</li><li><span class="item_label"><b>Published</b> online</span>16 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02160" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02160</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3697%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYudao%2BShen%252C%2BFengling%2BLi%252C%2BMagdalena%2BM.%2BSzewczyk%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D7%26contentID%3Dacs.jmedchem.0c02160%26title%3DA%2BFirst-in-Class%252C%2BHighly%2BSelective%2Band%2BCell-Active%2BAllosteric%2BInhibitor%2Bof%2BProtein%2BArginine%2BMethyltransferase%2B6%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3706%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02160"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1522</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02160" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yudao&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;M. Szewczyk&quot;},{&quot;first_name&quot;:&quot;Levon&quot;,&quot;last_name&quot;:&quot;Halabelian&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;Chau&quot;},{&quot;first_name&quot;:&quot;Mohammad&quot;,&quot;last_name&quot;:&quot;S. Eram&quot;},{&quot;first_name&quot;:&quot;Carlo&quot;,&quot;last_name&quot;:&quot;Dela Seña&quot;},{&quot;first_name&quot;:&quot;Kwang-Su&quot;,&quot;last_name&quot;:&quot;Park&quot;},{&quot;first_name&quot;:&quot;Fanye&quot;,&quot;last_name&quot;:&quot;Meng&quot;},{&quot;first_name&quot;:&quot;He&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Zeng&quot;},{&quot;first_name&quot;:&quot;Aiping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Viacheslav&quot;,&quot;last_name&quot;:&quot;V. Trush&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;McLeod&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;A. Zepeda-Velázquez&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;M. Campbell&quot;},{&quot;first_name&quot;:&quot;Mary&quot;,&quot;last_name&quot;:&quot;M. Mader&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;M. Watson&quot;},{&quot;first_name&quot;:&quot;Matthieu&quot;,&quot;last_name&quot;:&quot;Schapira&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Rima&quot;,&quot;last_name&quot;:&quot;Al-Awar&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;H.&quot;,&quot;last_name&quot;:&quot;Ümit Kaniskan&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3697-3706&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02160&quot;},&quot;abstract&quot;:&quot;Protein arginine methyltransferase 6 (PRMT6) catalyzes monomethylation and asymmetric dimethylation of arginine residues in various proteins, plays important roles in biological processes, and is associated with multiple cancers. To date, a highly selective PRMT6 inhibitor has not been reported. Here we report the discovery and characterization of a first-in-class, highly selective allosteric inhibitor of PRMT6, (R)-2 (SGC6870). (R)-2 is a potent PRMT6 inhibitor (IC50 = 77 ± 6 nM) with outstanding selectivity for PRMT6 over a broad panel of other methyltransferases and nonepigenetic targets. Notably, the crystal structure of the PRMT6–(R)-2 complex and kinetic studies revealed (R)-2 binds a unique, induced allosteric pocket. Additionally, (R)-2 engages PRMT6 and potently inhibits its methyltransferase activity in cells. Moreover, (R)-2’s enantiomer, (S)-2 (SGC6870N), is inactive against PRMT6 and can be utilized as a negative control. Collectively, (R)-2 is a well-characterized PRMT6 chemical probe and a valu&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02160&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02160" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02160&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02160" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02160&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02160" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02160&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02160&amp;href=/doi/10.1021/acs.jmedchem.0c02160" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02160" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02160" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02160%26sid%3Dliteratum%253Aachs%26pmid%3D33591753%26genre%3Darticle%26aulast%3DShen%26date%3D2021%26atitle%3DA%2BFirst-in-Class%252C%2BHighly%2BSelective%2Band%2BCell-Active%2BAllosteric%2BInhibitor%2Bof%2BProtein%2BArginine%2BMethyltransferase%2B6%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D7%26spage%3D3697%26epage%3D3706%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/jmcmar.2021.64.issue-7/20210408/jmcmar.2021.64.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein arginine methyltransferase 6 (PRMT6) catalyzes monomethylation and asymmetric dimethylation of arginine residues in various proteins, plays important roles in biological processes, and is associated with multiple cancers. To date, a highly selective PRMT6 inhibitor has not been reported. Here we report the discovery and characterization of a first-in-class, highly selective allosteric inhibitor of PRMT6, <i>(R)-</i><b>2</b> (SGC6870). <i>(R)-</i><b>2</b> is a potent PRMT6 inhibitor (IC<sub>50</sub> = 77 ± 6 nM) with outstanding selectivity for PRMT6 over a broad panel of other methyltransferases and nonepigenetic targets. Notably, the crystal structure of the PRMT6–<i>(R)-</i><b>2</b> complex and kinetic studies revealed <i>(R)-</i><b>2</b> binds a unique, induced allosteric pocket. Additionally, <i>(R)-</i><b>2</b> engages PRMT6 and potently inhibits its methyltransferase activity in cells. Moreover, <i>(R)-</i><b>2</b>’s enantiomer, <i>(S)-</i><b>2</b> (SGC6870N), is inactive against PRMT6 and can be utilized as a negative control. Collectively, <i>(R)</i>-<b>2</b> is a well-characterized PRMT6 chemical probe and a valuable tool for further investigating PRMT6 functions in health and disease.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02828" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02828" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein arginine methyltransferases (PRMTs) catalyze monomethylation and symmetric or asymmetric dimethylation of arginine residues at histone and/or nonhistone substrates.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> PRMTs are involved in multiple cellular processes, including regulation of cell cycle, cell pluripotency, estrogen-stimulated transcription, DNA repair, and modulation of pre-mRNA splicing.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Aberrant expression of PRMTs are associated with various types of cancer, such as leukemiaand breast, lung, prostate, and bladder cancers.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> PRMTs are commonly classified into three types.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Type I PRMTs, including PRMT1, PRMT3, PRMT4 (CARM1), PRMT6, and PRMT8, catalyze monomethylation and asymmetric dimethylation of arginine residues. Type II PRMTs, including PRMT5 and PRMT9, catalyze monomethylation and symmetric dimethylation of arginine residues. PRMT7 as the only type III PRMT catalyzes monomethylation of arginine residues. All PRMTs can methylate multiple protein substrates and some PRMTs share the same substrates. For example, H4R3 can be methylated by PRMT1, PRMT5, PRMT6, and PRMT7;<a onclick="showRef(event, 'ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9">(6−9)</a> TP53 can be methylated by PRMT3 and PRMT5;<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> and H3R2 can be methylated by PRMT5, PRMT6, and PRMT7.<a onclick="showRef(event, 'ref5 ref12'); return false;" href="javascript:void(0);" class="ref ref5 ref12">(5,12)</a> To further understand the roles of each PRMT in cell biology, highly selective and cell-active inhibitors of each PRMT are needed. In addition to being invaluable chemical tools, selective PRMT inhibitors could also serve as leads for drug discovery programs. To date, highly selective inhibitors for PRMT3,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> CARM1,<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> PRMT5,<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20">(16−20)</a> and PRMT7<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> have been reported. However, it remains challenging to develop highly selective inhibitors for other PRMTs, including PRMT6, due to high homology in the conserved catalytic core shared among type I PRMTs.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> PRMT6 is associated with proliferative phenotypes,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and it contributes to negative regulation of global DNA methylation in cancer cells.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> However, its function is still poorly understood. While several PRMT6 inhibitors have been reported to date, a highly selective and cell-active PRMT6 inhibitor that can be utilized as a PRMT6 chemical probe has not yet been reported. Previously, we reported the type I PRMT pan inhibitor MS023,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> PRMT6 covalent inhibitor MS117 with modest selectivity,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and PRMT4/6 dual inhibitor MS049 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In addition, EPZ020411<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> was reported as a PRMT6 inhibitor with limited selectivity (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Here, we report the first highly selective and cell-active allosteric inhibitor of PRMT6, <i>(R)-</i><b>2</b>, which is one of the only few PRMT allosteric inhibitors, including our previously reported PRMT3 allosteric inhibitor, SGC707,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and a recently reported PRMT5 allosteric inhibitor.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> We also developed <i>(S)-</i><b>2</b>, which is the enantiomer of <i>(R)-</i><b>2</b> but is inactive for PRMT6, as a negative control of <i>(R)-</i><b>2</b> for chemical biology studies.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. The structures and potencies of reported PRMT6 inhibitors</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14417" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14417" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Discoveries of <i>(R)-</i>2 and <i>(S)-</i>2</h3><div class="NLM_p">Our quest for the discovery of a selective PRMT6 inhibitor began with the screening of a diverse library of 5000 compounds, from which a submicromolar inhibitor <i>(±)</i>-<b>1</b> (IC<sub>50</sub> = 755 ± 87 nM) was identified (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Optically pure <i>(R)-</i><b>1</b> enantiomer and <i>(S)-</i><b>1</b> enantiomer were further chirally separated. The <i>(R)-</i><b>1</b> enantiomer was identified as the active isomer (IC<sub>50</sub> = 388 ± 77 nM), while the <i>(S)-</i><b>1</b> enantiomer was inactive (IC<sub>50</sub> > 100 μM) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). To discover the first highly selective chemical probe of PRMT6, we conducted structure–activity relationship (SAR) studies and successively optimized the amide group, benzodiazepine ring substituent, and pendant ring moiety of compound <i>(±)-</i><b>1</b>. Design, synthesis, and testing of more than 60 derivatives of <i>(±)-</i><b>1</b> led to the identification of compound <i>(±)</i>-<b>2</b>, which contains 3,5-dimethyl phenyl and thiophene rings, with the most potent inhibitory effect of PRMT6 (IC<sub>50</sub> = 214 ± 37 nM). The synthesis of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> was outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Intermediate <b>3</b> was prepared through the addition reaction between 2,6-dimethyl-4<i>H</i>-benzo[<i>d</i>][1,3]oxazin-4-one and (3,5-dimethylphenyl)magnesium bromide, deprotection of acetal group in the presence of hydrochloric acid, and cyclization reaction with Oxazolidine-2,5-dione, successively. Intermediate <b>3</b> was further reduced to afford diazepine in the presence of sodium cyanoborohydride, followed by amide coupling reaction with 5-bromothiophene-2-carboxylic acid to afford racemic compound <i>(±)-</i><b>2</b>. Optically pure enantiomer <i>(R)-</i><b>2</b> and enantiomer <i>(S)-</i><b>2</b> were then obtained by chiral separation. Similar to compound <i>(±)-</i><b>1,</b> the <i>(R)-</i>enantiomer of <i>(±)-</i><b>2</b>, <i>(R)-</i><b>2</b>, showed potent PRMT6 inhibition (IC<sub>50</sub> = 77 ± 6 nM) while the <i>(S)-</i>enantiomer, <i>(S)-</i><b>2</b>, was inactive (IC<sub>50</sub> > 50 μM) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Discovery of the PRMT6 chemical probe <i>(R)-</i><b>2</b>. (A) Structure–activity relationship studies and chiral separation led to the discovery of <i>(R)-</i><b>2</b> as a potent PRMT6 inhibitor and its enantiomer, <i>(S)-</i><b>2</b>, as a negative control. (B) IC<sub>50</sub> determination of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b>, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <i>(R)-</i><b>2</b> and Compound <i>(S)-</i><b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) dichloromethane, −78 °C to rt; (b) HCl, MeOH/H<sub>2</sub>O, reflux; (c) oxazolidine-2,5-dione, CF<sub>3</sub>COOH, triethylamine, toluene, 60–85 °C; (d) NaBH<sub>3</sub>CN, AcOH, MeOH, rt; (e) 5-bromothiophene-2-carboxylic acid, EDCI, HOAt, NMM, DMSO, rt; (f) chiral separation.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Selectivity Assessment</h3><div class="NLM_p">Next, we assessed the selectivity of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> against a total of 33 methyltransferases, including 8 PRMTs, 21 protein lysine methyltransferases (PKMTs), 3 DNA methyltransferases (DNMTs), and 1 RNA methyltransferase. <i>(R)-</i><b>2</b> at both 1 and 10 μM potently inhibited PRMT6, but did not significantly inhibit other 32 methyltransferases (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S1</a>). As expected, negative control <i>(S)-</i><b>2</b> did not significantly inhibit any of the 33 methyltransferases, including PRMT6, at 1 or 10 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S1</a>). Furthermore, <i>(R)-</i><b>2</b> was remarkably selective for PRMT6 over a broad range of nonepigenetic targets, including G protein-coupled receptors (GPCRs), ion channels, transporters, and other enzymes at 1 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S2</a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selectivity assessment of <i>(R)-</i><b>2</b> (SGC6870) at 1 μM (gray) and 10 μM (black) and <i>(S)-</i><b>2</b> (SGC6870N) at 1 μM (green) and 10 μM (blue) against 8 PRMTs, 21 protein lysine methyltransferases (PKMTs), 3 DNA methyltransferases (DNMTs), and 1 RNA methyltransferase, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Mechanism of Action</h3><div class="NLM_p">Typically, when the addition of an inhibitor leads to quick equilibrium, the IC<sub>50</sub> values can be determined very reproducibly regardless of small variations in the preparation of reaction mixtures, and time of inhibitor incubation with protein. However, as we further assessed the inhibitory effect of <i>(R)-</i><b>2</b> in different conditions, we observed significant differences in IC<sub>50</sub> values. We then investigated a possible time-dependent inhibition of PRMT6 by <i>(R)-</i><b>2</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S3</a>). Inhibitory effect of <i>(R)-</i><b>2</b> was indeed time-dependent with more than 100-fold decrease in the IC<sub>50</sub> value after 2 h of compound preincubation with PRMT6 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S3</a>). This indicates that <i>(R)-</i><b>2</b> binds slowly and requires significant time to reach equilibrium before steady state turnover initiated by the addition of the substrate (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S3</a>). To confirm that the time-dependent inhibition of PRMT6 by <i>(R)-</i><b>2</b> is not due to covalent binding of the compound, we tested samples of PRMT6 incubated with <i>(R)-</i><b>2</b> for 2 h by mass spectrometry. No covalent modification of PRMT6 by <i>(R)-</i><b>2</b> was observed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Figure S1</a>). Using 2 h preincubation, we investigated the mechanism of action of <i>(R)-</i><b>2</b> by determining the IC<sub>50</sub> values at various concentrations of the cofactor (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C) or substrate (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). Our data supported a noncompetitive pattern of inhibition with respect to both the S-adenosyl methionine (SAM) cofactor and peptide substrate as no changes in IC<sub>50</sub> values were observed as the cofactor or substrate concentrations were increased (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D). These mechanism of action results suggest that <i>(R)-</i><b>2</b> may be an allosteric inhibitor binding away from the catalytic active site.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mechanism of action. (A) IC<sub>50</sub> values were determined at various compound-protein incubation times ranging from 15 to 120 min. The value obtained with no incubation time was used as a control (values are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S3</a>). (B) Part of the plot in Figure A is magnified for better view of the decrease in the IC<sub>50</sub> value at longer incubation times. (C) IC<sub>50</sub> values of <i>(R)-</i><b>2</b> at increasing concentrations of SAM (up to 30 μM) at fixed concentration of peptide substrate (3 μM of H4 1–24 peptide). (D) IC<sub>50</sub> values of <i>(R)-</i><b>2</b> at increasing concentration of peptide (up to 8 μM) at fixed concentration of SAM (12 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Cocrystal Structures</h3><div class="NLM_p">To further investigate the mechanism of inhibition, we solved the crystal structures of PRMT6 in complex with <i>(R)-</i><b>2</b> (PDB: 6W6D) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S4</a>) and <i>(R)-</i><b>1</b> (PDB: 5WCF) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Figure S2A and Table S4</a>), respectively. Structural alignment of PRMT6-<i>(R)-</i><b>2</b>-SAM complex, and previously reported PRMT6-MS023-SAH<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and PRMT6-SAH<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> complexes revealed <i>(R)-</i><b>2</b> binds in a unique pocket distinct from either the substrate or SAM binding pocket (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B) consistent with our kinetic data. A nine-amino acid loop consisting of residues Gly158 through Met166 flipped toward and downsized the substrate-binding pocket, creating a new pocket for accommodating <i>(R)-</i><b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Notably, this loop movement was highly localized, and no other large-scale changes were observed in the complex. Several key interactions were observed between <i>(R)-</i><b>2</b> and residues in this new site (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). The endo-amide on the diazepine ring formed a hydrogen bond with the backbone nitrogen of Gly158, and the exo-amide formed a hydrogen bond with the backbone nitrogen of Gly160. In addition, the thiophene group formed a T-shaped π-π stacking interaction with Tyr159, and another T-shaped π-π stacking interaction was also observed between the dimethylphenyl group and Trp156. Furthermore, the methylphenyl portion of the benzodiazepine moiety points to a hydrophobic pocket consisting of Leu267, Val271, Leu343, and Pro345 residues. Residue Ala321 is one of the key components creating the newly formed binding pocket for the bromothiophene group of <i>(R)-</i><b>2</b> and bromofuran group of <i>(R)-</i><b>1</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Figure S2B</a>). We mutated it to Ile, Gln, and Met bearing bulkier side chains, respectively. While these directed mutations did not significantly impact the substrate or SAM binding affinity to PRMT6 or PRMT6 catalytic activity itself (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), they did impair PRMT6 inhibitory potency of <i>(R)-</i><b>1</b> by 4–12-fold (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The structural alignment of PRMT6-<i>(R)-</i><b>2</b> and PMRT6-<i>(R)-</i><b>1</b> complexes revealed that two inhibitors possess the same binding mode (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Figure S2B</a>). Taken together, these results show both <i>(R)-</i><b>2</b> and <i>(R)-</i><b>1</b> bind to the induced PRMT6 allosteric pocket. As previously reported, electron donating or withdrawing substituents of aromatic rings impact the T-shaped π-π stacking interactions.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Compared with <i>(R)-</i><b>1</b>, the enhanced inhibitory potency of <i>(R)-</i><b>2</b> could be partially due to higher electron density of thiophene and dimethylphenyl groups of <i>(R)-</i><b>2</b> than furan and fluorophenyl of <i>(R)-</i><b>1</b>, as a result of stronger T-shaped π-π stacking interactions between <i>(R)-</i><b>2</b> and PRMT6 residues Tyr159 and Trp156. Collectively, both structural and kinetic data consistently revealed that <i>(R)-</i><b>2</b> is an allosteric PRMT6 inhibitor.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of PRMT6 in complex with <i>(R)-</i><b>2</b>. <b>(</b>A) Cocrystal structure of PRMT6 (tinted blue) in complex with <i>(R)-</i><b>2</b> (green), and SAM (orange), (PDB: 6W6D). Dashed yellow lines indicate key hydrogen bonds. (B) Structural alignments of complexes PRMT6-<i>(R)-</i><b>2</b>-SAM (cyan), PRMT6-MS023-SAH (blue) (PDB: 5E8R) and PRMT6-SAH (magenta) (PDB: 4C05)<b>.</b> Movement of loop consisting of nine residues, EWMGYGLLH, indicates that <i>(R)-</i>2 binds to a novel induced allosteric pocket of PRMT6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Activity Assessment on PRMT6-Wild Type (1–375) and Its Mutants (17, 19, 20)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameters</th><th class="colsep0 rowsep0" align="center">wild type</th><th class="colsep0 rowsep0" align="center">PRMT6–17</th><th class="colsep0 rowsep0" align="center">PRMT6–19</th><th class="colsep0 rowsep0" align="center">PRMT6–20</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mutation</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">A321I</td><td class="colsep0 rowsep0" align="left">A321Q</td><td class="colsep0 rowsep0" align="left">A321M</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>m</sub> B-H<sub>4</sub>(1–24), μM</td><td class="colsep0 rowsep0" align="left">0.17</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.18</td><td class="colsep0 rowsep0" align="left">0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>k</i><sub>cat</sub>, h<sup>–1</sup></td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>m</sub> SAM, μM</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">4</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Potency of <i>(R)-</i><b>1</b> against PRMT6-Wild Type (1–375) and Its Mutants (17, 19, 20)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">protein</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">hill slope</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT6-Wt</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT6–17</td><td class="colsep0 rowsep0" align="left">46</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT6–19</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT6–20</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">0.8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The IC<sub>50</sub> values in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> were generated using 20 min preincubation.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Evaluation in Cellular Assays</h3><div class="NLM_p">Next, given that <i>(R)-</i><b>2</b> is a highly selective PRMT6 inhibitor in biochemical assays, we did not assess its cellular selectivity for PRMT6, but evaluated its cellular potency on ectopically expressed PRMT6 in HEK293T cells and used a catalytically inactive PRMT6 mutant (V86K/D88A) as a positive control. HEK293T cells were treated with <i>(R)-</i><b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A) or <i>(S)-</i><b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B) at titrated concentrations for 20 h. <i>(R)-</i><b>2</b> potently and concentration-dependently reduced cellular levels of H3R2me2a (IC<sub>50</sub> = 0.9 ± 0.1 μM) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) and H4R3me2a (IC<sub>50</sub> = 0.6 ± 0.1 μM) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D), both of which are known substrates of PRMT6.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Furthermore, <i>(R)-</i><b>2</b> did not show significant toxicity to HEK293T cells at up to 10 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). As expected, ectopical expression of the PRMT6 catalytically inactive V86K/D88A mutant led to near complete reduction of H3R2me2a and H4R3me2a markers (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Consistent with its poor potency against PRMT6 in the biochemical assay, <i>(S)-</i><b>2</b> did not significantly reduce cellular levels of H4R3me2a or H3R2me2a at up to 10 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). However, at 30 μM, significant toxicity to HEK293T cells was observed (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of PRMT6-dependent H3R2 and H4R3 asymmetric di-methylation in cells. HEK293T cells were transfected with Flag-tagged PRMT6 and treated with indicated compounds for 20 h. The flag-tagged PRMT6 catalytically inactive mutant V86K/D88A (mut) was used as a positive control. (A) Western blot representation of the effect of <i>(R)-</i><b>2</b> on PRMT6 activity. (B) Western blot representation of the effect of <i>(S)-</i><b>2</b> on PRMT6 activity. (C) IC<sub>50</sub> determination of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> at reducing H3R2me2a. The graph represents nonlinear fit of H3R2me2a fluorescence intensities normalized to intensities of H3, <i>n</i> = 4. (D) IC<sub>50</sub> determination of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> at reducing H4R3me2a. The graph represents nonlinear fit of H4R3me2a fluorescence intensities normalized to intensities of H4, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Finally, we further investigated the effect of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> on cell growth in three different cell lines, HEK293T (embryonic kidney), PNT2 (prostate), and MCF-7 (breast cancer). Neither <i>(R)-</i><b>2</b> nor <i>(S)-</i><b>2</b> showed any significant toxicity to these three cell lines at concentrations up to 10 μM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B) after 3 days treatment. Therefore, <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> can be utilized in cellular studies at up to 10 μM concentration.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibitory effect of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> on cell viability (A) Effect of <i>(R)-</i><b>2</b> on MCF-7, PNT2, and HEK293T cell viability, <i>n</i> = 3. (B) Inhibitory effect of <i>(S)-</i><b>2</b> on MCF-7, PNT2, and HEK293T cell viability, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we discovered the first highly selective, potent and cell-active PRMT6 allosteric inhibitor, <i>(R)-</i><b>2</b>. The crystal structure of the PRMT6-<i>(R)-</i><b>2</b> complex and kinetic studies revealed that <i>(R)-</i><b>2</b> binds to a unique, induced allosteric pocket, rendering this inhibitor highly selective for PRMT6 over 32 other methyltransferases, as well as a broad range of common drug targets. In addition, <i>(R)-</i><b>2</b> can significantly inhibit PRMT6 activity in cells with submicromolar potency, but did not display significant cellular toxicity at concentrations up to 10 μM. Furthermore, we demonstrated that the enantiomer of <i>(R)-</i><b>2</b>, <i>(S)-</i><b>2</b>, showed very poor inhibitory effect on PRMT6 activity in both biochemical and cell-based assays and can be used as a negative control. Collectively, <i>(R)-</i><b>2</b> is a valuable chemical probe of PRMT6 and together with <i>(S)-</i><b>2</b> is an excellent tool compound to further investigate physiological and pathophysiological functions of PRMT6.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37702" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37702" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry General Procedure</h3><div class="NLM_p last">All commercially available chemical reagents were directly used without further purification. A Teledyne ISCO CombiFlash Rf<sup>+</sup> instrument equipped with a variable wavelength UV detector and a fraction collector was used to conduct flash column chromatography. Redi<i>SepR</i>f HP C18 and flash silica columns were used for purification. The purity of each tested compound was >95%, determined based on the HPLC spectrum, which was generated on an Agilent 1200 series system with a DAD detector and a 2.1 mm × 150 mm Zorbax 300SB-C18 5 μm column.  0.8 μL of the sample (methanol solution) was injected onto a C18 column at room temperature, and the flow rate of eluents was 0.4 mL/min. The eluents were as follows: water containing 0.1% formic acid was designated as solvent A, while acetonitrile containing 0.1% formic acid was designated as solvent B. The linear gradient was set such that 1% B was used from 0 to 1 min, 1–99% B from 1 to 4 min, and 99% B from 4 to 8 min. High-resolution mass spectra (HRMS) data were acquired in positive ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were acquired on Bruker Avance-III 800 MHz spectrometer (800 MHz <sup>1</sup>H NMR, 201 MHz <sup>13</sup>C NMR). Chemical shifts are reported in ppm (δ). The enantiomers were separated by prep SFC using an AS-H column (2 × 25 cm) and mobile phase of 25% MeOH/CO<sub>2</sub> with 0.1% DEA at 70 mL·min<sup>–1</sup> (100 bar).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 5-(3,5-dimethylphenyl)-7-methyl-1,3-dihydro-2<i>H</i>-benzo[1,4]diazepin-2-one (3)</h3><div class="NLM_p last">To the solution of 2,6-dimethyl-4<i>H</i>-benzo[<i>d</i>][1,3]oxazin-4-one (3.5 g, 20 mmol) in 60 mL of dichloromethane at −78 °C, was added dropwise 3,5-dimethylphenylmagnesium bromide (0.5 M in THF, 48 mL, 24 mmol). The resulting mixture was slowly warmed to room temperature and stirred overnight. Then, 30 mL of saturated aqueous ammonium chloride was carefully added, and the mixture was extracted with dichloromethane (60 mL × 3). The organic phase was dried over anhydrous sodium sulfate and concentrated under reduced pressure. The resulting residue was dissolved in 60 mL of methanol and 10 mL of hydrochloric acid (36.5–38.0%). The solution was heated to reflux for 6 h. The cooled solution was then carefully basified to pH = 8 by saturated aqueous sodium bicarbonate and concentrated under reduced pressure. The resulting mixture was extracted with ethyl acetate (50 mL × 3). The organic phase was dried over anhydrous sodium sulfate and concentrated under reduced pressure. The resulting residue was dissolved in 80 mL of toluene. To this solution, was added trifluoroacetic acid (1.7 mL, 22 mmol) and added portion wise oxazolidine-2,5-dione (2.6 g, 26 mmol). The reaction was heated to 60 °C for 30 min. Then triethylamine (3.1 mL, 22 mmol) was added, and the mixture was heated to 80 °C for another 30 min. To the cooled mixture was carefully added 20 mL of saturated aqueous sodium bicarbonate and the mixture was extracted with ethyl acetate (50 mL × 3). The organic phase was dried over anhydrous sodium sulfate and concentrated under reduced pressure. The resulting residue was purified by flash chromatography on silica gel column with eluent (EtOAc/hexane: 0–50%) to give white solid (2.7 g, yield 49%). <sup>1</sup>H NMR (800 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.43 (s, 1H), 7.36 (d, <i>J</i> = 8.4 Hz, 1H), 7.15 (d, <i>J</i> = 8.3 Hz, 1H), 7.11 (s, 1H), 7.06 (s, 2H), 7.01 (s, 1H), 4.46–3.66 (m, 2H), 2.27 (s, 6H), 2.25 (s, 3H). <sup>13</sup>C NMR (201 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 170.3, 139.8, 137.7, 137.7, 132.8, 132.2, 131.9, 130.8, 127.5, 126.9, 121.5, 57.3, 21.3, 20.7. MS (ESI) <i>m/z</i> 279.2 [M + H]<sup>+</sup>. HRMS (TOF) <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>19</sub>N<sub>2</sub>O<sup>+</sup> 279.1492, found 279.1509.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 4-(5-bromothiophene-2-carbonyl)-5-(3,5-dimethylphenyl)-7-methyl-1,3,4,5-tetrahydro-2<i>H</i>-benzo[<i>e</i>][1,4]diazepin-2-one (<i>(±)-</i>2)</h3><div class="NLM_p">To the solution of 5-(3,5-dimethylphenyl)-7-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>e</i>][1,4]diazepin-2-one (2.7 g, 9.7 mmol) in 40 mL of methanol was added portion wise sodium cyanoborohydride (1.8 g, 28 mmol) and acetic acid (5.5 mL, 97 mmol). The mixture was stirred at room temperature overnight, and the volatile was removed under reduced pressure. The residue was carefully treated by 20 mL of saturated aqueous sodium bicarbonate and extracted with ethyl acetate (50 mL × 3). The organic phase was dried over anhydrous sodium sulfate and concentrated under reduced pressure. To the solution of residue above in 50 mL of dimethyl sulfoxide was added 5-bromothiophene-2-carboxylic acid (2 g, 10 mmol), EDCI (2.4 g, 13 mmol), HOAt (1.7 g, 13 mmol), and <i>N</i>-methylmorpholine (2.8 mL, 26 mmol). The resulting solution was stirred at room temperature for 8 h and extracted with ethyl acetate (150 mL) and water (100 mL). The organic phase was washed by another 100 mL of water for twice and concentrated. The residue was purified by flash chromatography on silica gel column with eluent (EtOAc/hexane: 0–50%) to give a pale yellow solid (2.7 g, yield 59%).</div><div class="NLM_p">The enantiomers were separated by prep SFC using an AS-H column (2 × 25 cm) and mobile phase of 25% MeOH/CO<sub>2</sub> with 0.1% DEA at 70 mL.min<sup>–1</sup> (100 bar). Then, 15 mg aliquots in EtOH:DCM (1:1; 1 mL) were injected and the absorbance was monitored at 220 nm. Analytical SFC was used to determine the purity of combined separated fractions using an AS-H column (0.46 × 25 cm) and mobile phase of 40% MeOH/CO<sub>2</sub> with 0.1% DEA at 3 mL·min<sup>–1</sup> (120 bar). Peak 1 eluting at 2.44 min using the analytical conditions corresponded to <i>(S)-</i><b>2</b>, whereas Peak 2 eluting at 3.34 min corresponded to <i>(R)-</i><b>2</b>. Lastly, 1.0 g of each enantiomer was obtained from chiral separation of 2.2 g of racemic material.</div><div class="NLM_p"><i>(R)-</i><b>2:</b><sup>1</sup>H NMR (800 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.19–9.82 (m, 1H), 7.73–7.49 (m, 1H), 7.41–7.33 (m, 2H), 7.19 (d, <i>J</i> = 8.1 Hz, 1H), 7.00–6.81 (m, 2H), 6.67 (s, 2H), 6.59–6.31 (m, 1H), 4.48–3.82 (m, 2H), 2.32 (s, 3H), 2.19 (s, 6H). <sup>13</sup>C NMR (201 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 161.5, 140.2, 139.8, 137.8, 134.9, 134.4, 132.5, 131.6, 130.3, 129.8, 129.1, 124.42, 124.41, 117.6, 62.6, 51.4, 21.5, 20.7. HPLC >95% pure, <i>t</i><sub>R</sub><i>=</i> 5.39 min; MS (ESI) <i>m/z</i> 469.1 [M + H]<sup>+</sup>. HRMS <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>22</sub>BrN<sub>2</sub>O<sub>2</sub>S<sup>+</sup> 469.0580, found 469.0574. [α]<sub>D</sub><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> +275.2° (<i>c</i> 4.55, MeOH).</div><div class="NLM_p"><i>(S)-</i><b>2:</b><sup>1</sup>H NMR (800 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.22–9.85 (m, 1H), 7.60 (s, 1H), 7.49–7.10 (m, 3H), 7.01–6.80 (m, 2H), 6.67 (s, 2H), 6.60–6.27 (m, 1H), 4.55–3.75 (m, 2H), 2.32 (s, 3H), 2.18 (s, 6H). <sup>13</sup>C NMR (201 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.1, 161.5, 140.2, 139.8, 137.8, 134.9, 134.4, 132.5, 131.6, 130.4, 129.8, 129.1, 124.42, 122.40, 117.6, 62.6, 51.3, 21.5, 20.7. HPLC >95% pure, <i>t</i><sub>R</sub><i>=</i> 5.37 min; MS (ESI) <i>m/z</i> 469.1 [M + H]<sup>+</sup>. HRMS <i>m/z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>22</sub>BrN<sub>2</sub>O<sub>2</sub>S<sup>+</sup> 469.0580, found 469.0590. [α]<sub>D</sub><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> -268.2° (<i>c</i> 4.77, MeOH).</div><div class="NLM_p last">Due to the amide rotamers of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b>, which can interconvert at ambient temperature, broadening of NMR peaks was observed.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Crystallization and Structure Determination</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Crystallization</h4><div class="NLM_p last">Human PRMT6 protein was expressed and purified according to the previously published protocol.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> PRMT6 at 5.6 mg/mL was mixed with fivefold molar excess of S-adenosyl-L-homocysteine (SAH) or S-Adenosyl methionine (SAM) and 3-fold molar excess of (<i>R</i>)-<b>1</b> or <i>(R)-</i><b>2</b> (dissolved from a previously prepared 100 mM DMSO stock solution). Diffraction quality crystals were obtained by setting a 96-well vapor-diffusion sitting drops at room temperature, in a precipitant solution containing 0.1 M sodium malonate pH 7.0, 12% (w/v) PEG 3350. The PRMT6-<i>(R)-</i><b>2</b>/(<i>R</i>)-<b>1</b> crystal was cryo-protected by immersing it into a precipitant solution supplemented with 10% glycerol and then into paratone, and cryo-cooled in liquid nitrogen.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Structure Determination</h4><div class="NLM_p last">X-ray diffraction data for PRMT6 + <i>(R)-</i><b>2</b>/<i>(R)-</i><b>1</b> were collected at 100 K at beam line 24IDE of Advanced Photon Source (APS), Argonne National Laboratory and CMCF 08ID-1 of Canadian Light Source. Both data sets were processed using the HKL-3000 suite.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The structure of PRMT6 + <i>(R)-</i><b>2</b> was directly refined against PDB entry 6W6D as template. The structure of PRMT6 + <i>(R)-</i><b>1</b> was directly refined against PDB entry 5WCF as template. REFMAC was used for structure refinement.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> GRADE was used to generate all restrains for compound refinement.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Graphics program COOT was used for model building and visualization.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Molprobity was used for structure validation.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> PRMT6 Mutation</h3><div class="NLM_p last">Three PRMT6 mutants were generated by mutating the alanine residue at position 321 of the wild type PRMT6 (PRMT6-WT) (residues 1–375) to isoleucine, glutamine and methionine of mutants 17, 19, and 20, respectively.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Kinetic Characterization of PRMT6-WT and PRMT6-Mutants</h3><div class="NLM_p last">Using the optimized assay conditions (25 nM of PRMT6-wt or PRMT6-mutants, 20 mM Tris–HCl pH 7.5, 0.01% Triton X-100, and 10 mM DTT) , the kinetic parameters were determined for biotinylated H4 (B-H4) (residues 1–24) by varying the peptide concentration (0–1.75 μM) and keeping the AdoMet at a saturation concentration of 10 μM. Apparent Km values were determined for AdoMet in reactions with 2 μM of B-H4 by varying the AdoMet concentration (up to 12 μM). The reactions were started by the addition of a mixture of <sup>3</sup>H-AdoMet and unlabeled AdoMet. The reaction mixtures were incubated for 20 min at 23 °C and quenched by adding 10 μL of 7.5 M guanidinium hydrochloride. Ten microliters of the quenched reaction mixture was spotted onto streptavidin-coated membrane squares (SAM2® Biotin capture membrane, Promega). The membrane was washed three times in 2 M NaCl for 2 min each time, and then in water three times for 30 s each. The membrane was dried at 50 °C for 1 h, and then each spotted square was cut from the membrane and placed in a scintillation vial. The amount of methylated peptide was quantified by tracing the radioactivity (cpm) as counted by a TriCarb liquid scintillation counter (PerkinElmer Life Sciences).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> IC<sub>50</sub> Determination of <i>(R)-</i>1 against PRMT6-WT and PRMT6-Mutants</h3><div class="NLM_p last">IC<sub>50</sub> values of <i>(R)-</i><b>1</b> (data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) were determined using the radiometric method.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The preincubation time was 20 min.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Biochemical Assays</h3><div class="NLM_p last">IC<sub>50</sub> values inhibiting PRMT6 activity were also determined using the radiometric method.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The preincubation time was 2 h for all biochemical assays except the ones for <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> as noted above.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Mass Spectrometry Analysis for Assessing the Covalent Binding</h3><div class="NLM_p last">PRMT6 was incubated with 20 molar excess of the compound <i>(R)-</i><b>2</b> for 1 h at RT, followed by adding 0.1% trifluoroacetic acid (aq.). The samples were separated over a HPLC column over a 5–95% acetonitrile/water gradient and analyzed using an Agilent LC/MSD Time-of-Flight Mass Spectrometer equipped with an electrospray ionization source.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Selectivity Assays</h3><div class="NLM_p last">The effect of compounds on 33 methyltransferase (MT) activities was assessed as previously described.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Selectivity assay of 44 nonepigenetic targets (kinases, GPCRs, ion channels, and transporters) were conducted by Eurofins (<a href="https://www.eurofinsdiscoveryservices.com/cms/cms-content/services/adme-tox" class="extLink">https://www.eurofinsdiscoveryservices.com/cms/cms-content/services/adme-tox</a>).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Cellular PRMT6 Assay</h3><div class="NLM_p last">HEK293T cells were grown in 12-well plates (2e5cells/well) in DMEM supplemented with 10% FBS, penicillin (100 units/mL), and streptomycin (100 μg/mL). Next day cells were transfected with FLAG-tagged PRMT6/mutant V86K, D88A PRMT6 (1 μg of DNA per well) using jetPRIME transfection reagent (Polyplus-Transfection), following manufacturer instructions. After 4 h media were removed, and cells were treated with compounds. After 20 h, the media was removed, and cells were lysed in 100 μL of lysis buffer (in mM: 20 Tris–HCl pH = 8, 150 NaCl, 1 EDTA, 10 MgCl<sub>2</sub>, 0.5% Triton-X100, 12.5 U/mL benzonase (Sigma), complete EDTA-free protease inhibitor cocktail (Roche). After 1 min incubation at RT, SDS was added to the final 1% concentration. Total cell lysates were resolved in 4–12% Bis-Tris protein gels (Invitrogen) with MOPS buffer (Invitrogen) and transferred in for 1.5 h (80 V) onto PVDF membrane (Millipore) in Tris-glycine transfer buffer containing 20% MeOH and 0.05% SDS. Blots were blocked for 1 h in blocking buffer (5% milk in 0.1% Tween 20 PBS) and incubated with primary antibodies: mouse anti-H4 (1:1000, Abcam #174628), rabbit anti-H4R3me2a (1:1000 Active Motif #39705), mouse anti-FLAG (1:5000, Sigma #F1804), mouse anti-H3 (1:1000, Abcam #174628), and rabbit anti-H3R2me2a (1:1000, Millipore #04–808) in blocking buffer o/n at 4 °C. After five washes with 0.1% Tween 20 PBS, the blots were incubated with goat-anti rabbit (IR800 conjugated, LiCor #926–32,211) and donkey anti-mouse (IR 680, LiCor #926–68,072) antibodies (1:5000) in Odyssey Blocking Buffer (LiCor) for 1 h at RT and washed five times with 0.1% Tween 20 PBS. The signal was read on an Odyssey scanner (LiCor) at 800 and 700 nm.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cell Viability Assay</h3><div class="NLM_p last">1 × 10<sup>4</sup> cells (HEK293T, PNT2 and MCF7) were seeded on 96-well plates. After 24 h, the serial diluted compounds (<i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b>) from 10 μM were treated for 3 days. Cell viability was evaluated by CCK-8 (Cell counting kit-8, WST-8). Briefly, 10 μL/well of CCK-8 was treated and then incubated for 4 h at 37 °C. The absorbance was recorded by Infinite F PLEX plate reader (TECAN, Morrisville, NC, USA) at 450 nm. GraphPad Prism 8 was used for data analysis, and the experimental results are shown as the average ± SD from three replicate experiments.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Accession Code</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78572" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78572" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The structures presented in this article have been deposited in the Protein Data Bank under accession code 5WCF for PRMT6-<i>(R)-</i><b>1</b> complex and 6W6D for PRMT6-<i>(R)-</i><b>2</b>. Authors will release the atomic coordinates upon article publication.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02160" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33781" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02160?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02160</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Selectivity data, mass spectrometry data, effect of preincubation on PRMT6 inhibitory potency of <i>(R)-</i><b>2</b>, crystal structure of PRMT6 with <i>(R)-</i><b>1</b>, crystallographic statistics and collection parameters, spectral characterization of compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf">jm0c02160_si_001.pdf (5.72 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_002.csv">jm0c02160_si_002.csv (0.82 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02160" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#621207160710084c00100d150c2217160d100d0c160d4c0103"><span class="__cf_email__" data-cfemail="d2a2b7a6b7a0b8fcb0a0bda5bc92a7a6bda0bdbca6bdfcb1b3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#cea3e0b8abaaafaaa78ebbbaa1bca1a0baa1e0adaf"><span class="__cf_email__" data-cfemail="234e0d55464742474a6356574c514c4d574c0d4042">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#274d4e4649094d4e49674a54544a09424352"><span class="__cf_email__" data-cfemail="ed87848c83c3878483ad809e9e80c3888998">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yudao Shen</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena M. Szewczyk</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Levon Halabelian</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4361-3619" title="Orcid link">http://orcid.org/0000-0003-4361-3619</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene Chau</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammad S. Eram</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlo Dela Seña</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kwang-Su Park</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fanye Meng</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">He Chen</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Zeng</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aiping Dong</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Wu</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Viacheslav V. Trush</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David McLeod</span> - <span class="hlFld-Affiliation affiliation">Ontario
Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos A. Zepeda-Velázquez</span> - <span class="hlFld-Affiliation affiliation">Ontario
Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert M. Campbell</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly
and Company, Lilly Research Laboratories, Indianapolis, Indiana 46225, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mary M. Mader</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly
and Company, Lilly Research Laboratories, Indianapolis, Indiana 46225, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian M. Watson</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly
and Company, Lilly Research Laboratories, Indianapolis, Indiana 46225, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthieu Schapira</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link">http://orcid.org/0000-0002-1047-3309</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rima Al-Awar</span> - <span class="hlFld-Affiliation affiliation">Ontario
Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">H. Ümit Kaniskan</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5327-832X" title="Orcid link">http://orcid.org/0000-0001-5327-832X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): R. M. Campbell, M. M. Mader and B. M. Watson were employees of Eli Lilly and Company during the time this research was performed.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part by the grants R01GM122749 (to J.J.) and P30CA196521 (to J.J.) from the U.S. National Institutes of Health and an endowed professorship from the Icahn School of Medicine at Mount Sinai (to J.J.). The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada through Ontario Genomics Institute [OGI-055], Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA, Darmstadt, Germany, MSD, Novartis Pharma AG, Ontario Ministry of Research, Innovation and Science (MRIS), Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome [106169/ZZ14/Z]. Eli Lilly and Company was a funder of SGC during the discovery and characterization of <i>(R)-</i><b>1</b>. This work utilized the AVANCE NEO 600 MHz NMR Spectrometer System that was upgraded with funding from a National Institutes of Health SIG grant 1S10OD025132-01A1. We also thank Alice Shi Ming Li for contributions to biological characterization of compounds.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">After this paper was published ASAP February 16, 2021, authors Aiping Dong, Hong Wu, and Viacheslav V. Trush were added to the author list. The revised version was reposted March 24, 2021.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last">S-adenosylmethionine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-aza-benzo-triazole</p></td></tr><tr><td class="NLM_term">NMM</td><td class="NLM_def"><p class="first last"><i>N</i>-methylmorpholine</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last">N-Methyl-2-Pyrrolidone</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">H3R2me2a</td><td class="NLM_def"><p class="first last">H3R2 asymmetric dimethylation</p></td></tr><tr><td class="NLM_term">H4R3me2a</td><td class="NLM_def"><p class="first last">H4R3 asymmetric dimethylation.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06273" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06273" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0ljgk4CH0yoGnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+U.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0ljgk4CH0yoGnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0lgqqUfCdJRi6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNutt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span> <span> </span><span class="NLM_article-title">PTEN arginine methylation by PRMT6 suppresses PI3K-AKT signaling and modulates pre-mRNA splicing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6868</span>– <span class="NLM_lpage">6877</span>, <span class="refDoi"> DOI: 10.1073/pnas.1811028116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1073%2Fpnas.1811028116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=30886105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsFeqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=6868-6877&author=J.+Fengauthor=Y.+Dangauthor=W.+Zhangauthor=X.+Zhaoauthor=C.+Zhangauthor=Z.+Houauthor=Y.+Jinauthor=M.+A.+McNuttauthor=A.+R.+Marksauthor=Y.+Yin&title=PTEN+arginine+methylation+by+PRMT6+suppresses+PI3K-AKT+signaling+and+modulates+pre-mRNA+splicing&doi=10.1073%2Fpnas.1811028116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PTEN arginine methylation by PRMT6 suppresses PI3K-AKT signaling and modulates pre-mRNA splicing</span></div><div class="casAuthors">Feng, Jiawen; Dang, Yaping; Zhang, Weiqi; Zhao, Xuyang; Zhang, Cong; Hou, Zhiyuan; Jin, Yan; McNutt, Michael A.; Marks, Andrew R.; Yin, Yuxin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6868-6877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Arginine methylation is a ubiquitous posttranslational modification that regulates crit. cellular processes including signal transduction and pre-mRNA splicing.  Here, we report that the tumor-suppressor PTEN is methylated by protein arginine methyltransferase 6 (PRMT6).  Mass-spectrometry anal. reveals that PTEN is dimethylated at arginine 159 (R159).  We found that PTEN is mutated at R159 in cancers, and the PTEN mutant R159K loses its capability to inhibit the PI3K-AKT cascade.  Furthermore, PRMT6 is phys. assocd. with PTEN, promotes asym. dimethylation of PTEN, and regulates the PI3K-AKT cascade through PTEN R159 methylation.  In addn., using transcriptome analyses, we found that PTEN R159 methylation is involved in modulation of pre-mRNA alternative splicing.  Our results demonstrate that PTEN is functionally regulated by arginine methylation.  We propose that PTEN arginine methylation modulates pre-mRNA alternative splicing and influences diverse physiol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoObdTCJWaRtLVg90H21EOLACvtfcHk0lgqqUfCdJRi6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsFeqsLk%253D&md5=8c7dede72240882358a7fad8a661fadf</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1811028116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1811028116%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHou%26aufirst%3DZ.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DMcNutt%26aufirst%3DM.%2BA.%26aulast%3DMarks%26aufirst%3DA.%2BR.%26aulast%3DYin%26aufirst%3DY.%26atitle%3DPTEN%2520arginine%2520methylation%2520by%2520PRMT6%2520suppresses%2520PI3K-AKT%2520signaling%2520and%2520modulates%2520pre-mRNA%2520splicing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D6868%26epage%3D6877%26doi%3D10.1073%2Fpnas.1811028116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0lgqqUfCdJRi6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brame, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">Methylation of histone H4 at arginine 3 occurs in vivo and is mediated by the nuclear receptor coactivator PRMT1</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">996</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(01)00294-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2FS0960-9822%2801%2900294-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=11448779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltVGjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=996-1000&author=B.+D.+Strahlauthor=S.+D.+Briggsauthor=C.+J.+Brameauthor=J.+A.+Caldwellauthor=S.+S.+Kohauthor=H.+Maauthor=R.+G.+Cookauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=M.+R.+Stallcupauthor=C.+D.+Allis&title=Methylation+of+histone+H4+at+arginine+3+occurs+in+vivo+and+is+mediated+by+the+nuclear+receptor+coactivator+PRMT1&doi=10.1016%2FS0960-9822%2801%2900294-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H4 at arginine 3 occurs in vivo and is mediated by the nuclear receptor coactivator PRMT1</span></div><div class="casAuthors">Strahl, B. D.; Briggs, S. D.; Brame, C. J.; Caldwell, J. A.; Koh, S. S.; Ma, H.; Cook, R. G.; Shabanowitz, J.; Hunt, D. F.; Stallcup, M. R.; Allis, C. D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">996-1000</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone amino termini play an important role in modulating chromatin structure and function [1-3].  Lysine methylation of histones has been well documented [4, 5], and recently this modification has been linked to cellular processes involving gene transcription and heterochromatin assembly [6-9].  However, the existence of arginine methylation on histones has remained unclear.  Recent discoveries of protein arginine methyltransferases, CARM1 and PRMT1, as transcriptional coactivators for nuclear receptors suggest that histones may be physiol. targets of these enzymes as part of a poorly defined transcriptional activation pathway [10-12].  Here we show by using mass spectrometry that histone H4, isolated from asynchronously growing human 293T cells, is methylated at arginine 3 (Arg-3) in vivo.  In support, a novel antibody directed against histone H4 methylated at Arg-3 independently demonstrates the in vivo occurrence of this modification and reveals that H4 Arg-3 methylation is highly conserved throughout eukaryotes.  Finally, we show that PRMT1 is the major, if not exclusive, H4 Arg-3 methyltransfase in human 293T cells.  These findings suggest a role for arginine methylation of histones in the transcription process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOj66WOabZpLVg90H21EOLACvtfcHk0linsq1UlQItwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltVGjtbc%253D&md5=a3c9c1c279890250a315bb787226223f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2801%2900294-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252801%252900294-9%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBrame%26aufirst%3DC.%2BJ.%26aulast%3DCaldwell%26aufirst%3DJ.%2BA.%26aulast%3DKoh%26aufirst%3DS.%2BS.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DCook%26aufirst%3DR.%2BG.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMethylation%2520of%2520histone%2520H4%2520at%2520arginine%25203%2520occurs%2520in%2520vivo%2520and%2520is%2520mediated%2520by%2520the%2520nuclear%2520receptor%2520coactivator%2520PRMT1%26jtitle%3DCurr.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D996%26epage%3D1000%26doi%3D10.1016%2FS0960-9822%2801%2900294-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Girardot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirasawa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kacem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fritsch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kota, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipponi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbrizio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ait-Si-Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feil, R.</span></span> <span> </span><span class="NLM_article-title">PRMT5-mediated histone H4 arginine-3 symmetrical dimethylation marks chromatin at G + C-rich regions of the mouse genome</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">248</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1093%2Fnar%2Fgkt884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=24097435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvFGk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=235-248&author=M.+Girardotauthor=R.+Hirasawaauthor=S.+Kacemauthor=L.+Fritschauthor=J.+Pontisauthor=S.+K.+Kotaauthor=D.+Filipponiauthor=E.+Fabbrizioauthor=C.+Sardetauthor=F.+Lohmannauthor=S.+Kadamauthor=S.+Ait-Si-Aliauthor=R.+Feil&title=PRMT5-mediated+histone+H4+arginine-3+symmetrical+dimethylation+marks+chromatin+at+G+%2B+C-rich+regions+of+the+mouse+genome&doi=10.1093%2Fnar%2Fgkt884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5-mediated histone H4 arginine-3 symmetrical dimethylation marks chromatin at G + C-rich regions of the mouse genome</span></div><div class="casAuthors">Girardot, Michael; Hirasawa, Ryutaro; Kacem, Salim; Fritsch, Lauriane; Pontis, Julien; Kota, Satya K.; Filipponi, Doria; Fabbrizio, Eric; Sardet, Claude; Lohmann, Felix; Kadam, Shilpa; Ait-Si-Ali, Slimane; Feil, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-248</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Sym. dimethylation on arginine-3 of histone H4 (H4R3me2s) has been reported to occur at several repressed genes, but its specific regulation and genomic distribution remained unclear.  Here, we show that the type-II protein arginine methyltransferase PRMT5 controls H4R3me2s in mouse embryonic fibroblasts (MEFs).  In these differentiated cells, we find that the genome-wide pattern of H4R3me2s is highly similar to that in embryonic stem cells.  In both the cell types, H4R3me2s peaks are detected predominantly at G + C-rich regions.  Promoters are consistently marked by H4R3me2s, independently of transcriptional activity.  Remarkably, H4R3me2s is mono-allelic at imprinting control regions (ICRs), at which it marks the same parental allele as H3K9me3, H4K20me3 and DNA methylation.  These repressive chromatin modifications are regulated independently, however, since PRMT5-depletion in MEFs resulted in loss of H4R3me2s, without affecting H3K9me3, H4K20me3 or DNA methylation.  Conversely, depletion of ESET (KMT1E) or SUV420H1/H2 (KMT5B/C) affected H3K9me3 and H4K20me3, resp., without altering H4R3me2s at ICRs.  Combined, our data indicate that PRMT5-mediated H4R3me2s uniquely marks the mammalian genome, mostly at G + C-rich regions, and independently from transcriptional activity or chromatin repression.  Furthermore, comparative bioinformatics analyses suggest a putative role of PRMT5-mediated H4R3me2s in chromatin configuration in the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgFVMfko7xHbVg90H21EOLACvtfcHk0linsq1UlQItwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvFGk&md5=d5cff330abf484a6476d0aa8d0a8fe5e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt884%26sid%3Dliteratum%253Aachs%26aulast%3DGirardot%26aufirst%3DM.%26aulast%3DHirasawa%26aufirst%3DR.%26aulast%3DKacem%26aufirst%3DS.%26aulast%3DFritsch%26aufirst%3DL.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DKota%26aufirst%3DS.%2BK.%26aulast%3DFilipponi%26aufirst%3DD.%26aulast%3DFabbrizio%26aufirst%3DE.%26aulast%3DSardet%26aufirst%3DC.%26aulast%3DLohmann%26aufirst%3DF.%26aulast%3DKadam%26aufirst%3DS.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DFeil%26aufirst%3DR.%26atitle%3DPRMT5-mediated%2520histone%2520H4%2520arginine-3%2520symmetrical%2520dimethylation%2520marks%2520chromatin%2520at%2520G%2520%252B%2520C-rich%2520regions%2520of%2520the%2520mouse%2520genome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D235%26epage%3D248%26doi%3D10.1093%2Fnar%2Fgkt884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Lorenzo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Histone arginine methylation</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>585</i></span>,  <span class="NLM_fpage">2024</span>– <span class="NLM_lpage">2031</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2010.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.febslet.2010.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=21074527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFagtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2024-2031&author=A.+Di%0ALorenzoauthor=M.+T.+Bedford&title=Histone+arginine+methylation&doi=10.1016%2Fj.febslet.2010.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone arginine methylation</span></div><div class="casAuthors">Di Lorenzo, Alessandra; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">585</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2024-2031</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation is a common posttranslational modification (PTM).  This type of PTM occurs on both nuclear and cytoplasmic proteins, and is particularly abundant on shuttling proteins.  In this review, we will focus on one aspect of this PTM: the diverse roles that arginine methylation of the core histone tails play in regulating chromatin function.  A family of nine protein arginine methyltransferases (PRMTs) catalyze methylation reactions, and a subset target histones.  Importantly, arginine methylation of histone tails can promote or prevent the docking of key transcriptional effector mols., thus playing a central role in the orchestration of the histone code.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtVnjeIHcVcrVg90H21EOLACvtfcHk0linsq1UlQItwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFagtbc%253D&md5=ab36ef630cd6ea1541ae6bca50e85685</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2010.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2010.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLorenzo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DHistone%2520arginine%2520methylation%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26spage%3D2024%26epage%3D2031%26doi%3D10.1016%2Fj.febslet.2010.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 7 regulates cellular response to DNA damage by methylating promoter histones H2A and H4 of the polymerase delta catalytic subunit gene, POLD1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">29801</span>– <span class="NLM_lpage">29814</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.378281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1074%2Fjbc.M112.378281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=22761421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1CgtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=29801-29814&author=V.+Karkhanisauthor=L.+Wangauthor=S.+Taeauthor=Y.+J.+Huauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Protein+arginine+methyltransferase+7+regulates+cellular+response+to+DNA+damage+by+methylating+promoter+histones+H2A+and+H4+of+the+polymerase+delta+catalytic+subunit+gene%2C+POLD1&doi=10.1074%2Fjbc.M112.378281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 7 regulates cellular response to DNA damage by methylating promoter histones H2A and H4 of the polymerase δ catalytic subunit gene, POLD1</span></div><div class="casAuthors">Karkhanis, Vrajesh; Wang, Li; Tae, Sookil; Hu, Yu-Jie; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">29801-29814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Covalent modification of histones by protein arginine methyltransferases (PRMTs) impacts genome organization and gene expression.  In this report, we show that PRMT7 interacts with the BRG1-based hSWI/SNF chromatin remodeling complex and specifically methylates histone H2A Arg-3 (H2AR3) and histone H4 Arg-3 (H4R3).  To elucidate the biol. function of PRMT7, we knocked down its expression in NIH 3T3 cells and analyzed global gene expression.  Our findings show that PRMT7 neg. regulates expression of genes involved in DNA repair, including ALKBH5, APEX2, POLD1, and POLD2.  Chromatin immunopptn. (ChIP) revealed that PRMT7 and dimethylated H2AR3 and H4R3 are enriched at target DNA repair genes in parental cells, whereas PRMT7 knockdown caused a significant decrease in PRMT7 recruitment and H2AR3/H4R3 methylation.  Decreased PRMT7 expression also resulted in derepression of target DNA repair genes and enhanced cell resistance to DNA-damaging agents.  Furthermore, we show that BRG1 co-localizes with PRMT7 on target promoters and that expression of a catalytically inactive form of BRG1 results in derepression of PRMT7 target DNA repair genes.  Remarkably, reducing expression of individual PRMT7 target DNA repair genes showed that only the catalytic subunit of DNA polymerase, POLD1, was able to resensitize PRMT7 knock-down cells to DNA-damaging agents.  These results provide evidence for the important role played by PRMT7 in epigenetic regulation of DNA repair genes and cellular response to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqANrVNaylqb7Vg90H21EOLACvtfcHk0ljUlW2DEpRXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1CgtrrE&md5=4f7305ced8d1fe2407ec49e2c658511c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.378281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.378281%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DTae%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%2BJ.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25207%2520regulates%2520cellular%2520response%2520to%2520DNA%2520damage%2520by%2520methylating%2520promoter%2520histones%2520H2A%2520and%2520H4%2520of%2520the%2520polymerase%2520delta%2520catalytic%2520subunit%2520gene%252C%2520POLD1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D29801%26epage%3D29814%26doi%3D10.1074%2Fjbc.M112.378281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weintraub, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiio, Y.</span></span> <span> </span><span class="NLM_article-title">Proteomic dissection of the von Hippel-Lindau (VHL) interactome</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5175</span>– <span class="NLM_lpage">5182</span>, <span class="refDoi"> DOI: 10.1021/pr200642c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr200642c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5175-5182&author=Y.+Laiauthor=M.+Songauthor=K.+Hakalaauthor=S.+T.+Weintraubauthor=Y.+Shiio&title=Proteomic+dissection+of+the+von+Hippel-Lindau+%28VHL%29+interactome&doi=10.1021%2Fpr200642c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span></div><div class="casAuthors">Lai, Yanlai; Song, Meihua; Hakala, Kevin; Weintraub, Susan T.; Shiio, Yuzuru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5175-5182</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) tumor suppressor gene encodes a component of a ubiquitin ligase complex contg. elongin B, elongin C, cullin 2, and Rbx1, which acts as a neg. regulator of hypoxia inducible factor (HIF).  VHL ubiquitinates and degrades the alpha subunits of HIF, and this is proposed to suppress tumorigenesis and tumor angiogenesis.  Several lines of evidence also suggest important roles for HIF-independent VHL functions in the maintenance of primary cilium, extracellular matrix formation, and tumor suppression.  We undertook a series of proteomic analyses to gain a comprehensive picture of the VHL-interacting proteins.  We found that the ARF tumor suppressor interacts with VHL30, a longer VHL isoform, but not with VHL19, a shorter VHL isoform.  ARF was found to release VHL30 from the E3 ligase complex, promoting the binding of VHL30 to a protein arginine methyltransferase, PRMT3.  Our anal. of the VHL19 interactome also uncovered that VHL19 displays an affinity to collagens and their biosynthesis enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqTYZ6RvZ7erVg90H21EOLACvtfcHk0ljUlW2DEpRXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL&md5=51500af3ec4256311dbd835c47a90200</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fpr200642c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200642c%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHakala%26aufirst%3DK.%26aulast%3DWeintraub%26aufirst%3DS.%2BT.%26aulast%3DShiio%26aufirst%3DY.%26atitle%3DProteomic%2520dissection%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520interactome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5175%26epage%3D5182%26doi%3D10.1021%2Fpr200642c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chitnis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsugoe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein-Szanto, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rustgi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">PRMT5 is required for lymphomagenesis triggered by multiple oncogenic drivers</span>. <i>Cancer Discov.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">303</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-14-0625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1158%2F2159-8290.CD-14-0625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=25582697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVGktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=288-303&author=Y.+Liauthor=N.+Chitnisauthor=H.+Nakagawaauthor=Y.+Kitaauthor=S.+Natsugoeauthor=Y.+Yangauthor=Z.+Liauthor=M.+Wasikauthor=A.+J.+Klein-Szantoauthor=A.+K.+Rustgiauthor=J.+A.+Diehl&title=PRMT5+is+required+for+lymphomagenesis+triggered+by+multiple+oncogenic+drivers&doi=10.1158%2F2159-8290.CD-14-0625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 Is Required for Lymphomagenesis Triggered by Multiple Oncogenic Drivers</span></div><div class="casAuthors">Li, Yan; Chitnis, Nilesh; Nakagawa, Hiroshi; Kita, Yoshiaki; Natsugoe, Shoji; Yang, Yi; Li, Zihai; Wasik, Mariusz; Klein-Szanto, Andres J. P.; Rustgi, Anil K.; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-303</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PRMT5 cooperates with oncogenic drivers such as cyclin D1 to promote lymphomagenesis via p53 methylation, which alters p53 chromatin occupancy and inhibits proapoptotic p53 target genes.  Protein arginine methyltransferase 5 (PRMT5) has been implicated as a key modulator of lymphomagenesis.  Whether PRMT5 has overt oncogenic function in the context of leukemia/lymphoma and whether it represents a therapeutic target remains to be established.  We demonstrate that inactivation of PRMT5 inhibits colony-forming activity by multiple oncogenic drivers, including cyclin D1, c-MYC, NOTCH1, and MLL-AF9.  Furthermore, we demonstrate that PRMT5 overexpression specifically cooperates with cyclin D1 to drive lymphomagenesis in a mouse model, revealing inherent neoplastic activity.  Mol. anal. of lymphomas revealed that arginine methylation of p53 selectively suppresses expression of crucial proapoptotic and antiproliferative target genes, thereby sustaining tumor cell self-renewal and proliferation and bypassing the need for the acquisition of inactivating p53 mutations.  Critically, anal. of human tumor specimens reveals a strong correlation between cyclin D1 overexpression and p53 methylation, supporting the biomedical relevance of this pathway.  Significance: We have identified and functionally validated a crucial role for PRMT5 for the inhibition of p53-dependent tumor suppression in response to oncogenic insults.  The requisite role for PRMT5 in the context of multiple lymphoma/leukemia oncogenic drivers suggests a mol. rationale for therapeutic development.  Cancer Discov; 5(3); 288-303. ©2015 AACR.  This article is highlighted in the In This Issue feature, p.  213.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCRiYvtZPoW7Vg90H21EOLACvtfcHk0ljUlW2DEpRXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVGktb8%253D&md5=c2d346e514c484969af530af53310350</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0625%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChitnis%26aufirst%3DN.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DNatsugoe%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWasik%26aufirst%3DM.%26aulast%3DKlein-Szanto%26aufirst%3DA.%2BJ.%26aulast%3DRustgi%26aufirst%3DA.%2BK.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DPRMT5%2520is%2520required%2520for%2520lymphomagenesis%2520triggered%2520by%2520multiple%2520oncogenic%2520drivers%26jtitle%3DCancer%2520Discov.%26date%3D2015%26volume%3D5%26spage%3D288%26epage%3D303%26doi%3D10.1158%2F2159-8290.CD-14-0625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Migliori, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phalke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaratne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capasso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackstock, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fnsmb.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=22231400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=136-144&author=V.+Miglioriauthor=J.+Mullerauthor=S.+Phalkeauthor=D.+Lowauthor=M.+Bezziauthor=W.+C.+Mokauthor=S.+K.+Sahuauthor=J.+Gunaratneauthor=P.+Capassoauthor=C.+Bassiauthor=V.+Cecatielloauthor=A.+De+Marcoauthor=W.+Blackstockauthor=V.+Kuznetsovauthor=B.+Amatiauthor=M.+Mapelliauthor=E.+Guccione&title=Symmetric+dimethylation+of+H3R2+is+a+newly+identified+histone+mark+that+supports+euchromatin+maintenance&doi=10.1038%2Fnsmb.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span></div><div class="casAuthors">Migliori, Valentina; Muller, Julius; Phalke, Sameer; Low, Diana; Bezzi, Marco; Mok, Wei Chuen; Sahu, Sanjeeb Kumar; Gunaratne, Jayantha; Capasso, Paola; Bassi, Christian; Cecatiello, Valentina; De Marco, Ario; Blackstock, Walter; Kuznetsov, Vladimir; Amati, Bruno; Mapelli, Marina; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-144</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The asym. dimethylation of histone H3 arginine 2 (H3R2me2a) acts as a repressive mark that antagonizes trimethylation of H3 lysine 4.  Here we report that H3R2 is also sym. dimethylated (H3R2me2s) by PRMT5 and PRMT7 and present in euchromatic regions.  Profiling of H3-tail interactors by SILAC MS revealed that H3R2me2s excludes binding of RBBP7, a central component of co-repressor complexes Sin3a, NURD and PRC2.  Conversely H3R2me2s enhances binding of WDR5, a common component of the coactivator complexes MLL, SET1A, SET1B, NLS1 and ATAC.  The interaction of histone H3 with WDR5 distinguishes H3R2me2s from H3R2me2a, which impedes the recruitment of WDR5 to chromatin.  The crystallog. structure of WDR5 and the H3R2me2s peptide elucidates the mol. determinants of this high affinity interaction.  Our findings identify H3R2me2s as a previously unknown mark that keeps genes poised in euchromatin for transcriptional activation upon cell-cycle withdrawal and differentiation in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDe73jpv41ObVg90H21EOLACvtfcHk0lhF4e59HLYEfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D&md5=c7b15ed7ee8f4819b45d488171e89ac4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2209%26sid%3Dliteratum%253Aachs%26aulast%3DMigliori%26aufirst%3DV.%26aulast%3DMuller%26aufirst%3DJ.%26aulast%3DPhalke%26aufirst%3DS.%26aulast%3DLow%26aufirst%3DD.%26aulast%3DBezzi%26aufirst%3DM.%26aulast%3DMok%26aufirst%3DW.%2BC.%26aulast%3DSahu%26aufirst%3DS.%2BK.%26aulast%3DGunaratne%26aufirst%3DJ.%26aulast%3DCapasso%26aufirst%3DP.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DDe%2BMarco%26aufirst%3DA.%26aulast%3DBlackstock%26aufirst%3DW.%26aulast%3DKuznetsov%26aufirst%3DV.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DMapelli%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DSymmetric%2520dimethylation%2520of%2520H3R2%2520is%2520a%2520newly%2520identified%2520histone%2520mark%2520that%2520supports%2520euchromatin%2520maintenance%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D136%26epage%3D144%26doi%3D10.1038%2Fnsmb.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Vedadi, M. A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span>. <i>Angew. Chem. Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5166</span>– <span class="NLM_lpage">5170</span>, <span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=25728001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A280%3ADC%252BC2MrpvVCjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+U.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+Yuauthor=M.+S.+Eramauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+Liauthor=E.+Dobrovetskyauthor=A.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+Zhaoauthor=J.+Jin&title=Vedadi%2C+M.+A+potent%2C+selective+and+cell-active+allosteric+inhibitor+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan H Umit; Szewczyk Magdalena M; Yu Zhengtian; Eram Mohammad S; Yang Xiaobao; Schmidt Keith; Luo Xiao; Dai Miao; He Feng; Zang Irene; Lin Ying; Kennedy Steven; Li Fengling; Dobrovetsky Elena; Dong Aiping; Smil David; Min Sun-Joon; Landon Melissa; Lin-Jones Jennifer; Huang Xi-Ping; Roth Bryan L; Schapira Matthieu; Atadja Peter; Barsyte-Lovejoy Dalia; Arrowsmith Cheryl H; Brown Peter J; Zhao Kehao; Jin Jian; Vedadi Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PRMT3 catalyzes the asymmetric dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biology.  Here we report the discovery of the first PRMT3 chemical probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochemical, biophysical, and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50 =31±2 nM, KD =53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chemical probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTrgkSPlfW-0Ptos8R3JUhXfW6udTcc2eb6EAlrgZJq17ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrpvVCjuw%253D%253D&md5=6a72abd6ed4b82dbc4187a51f6d75157</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DVedadi%252C%2520M.%2520A%2520potent%252C%2520selective%2520and%2520cell-active%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%2520Engl.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela
Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunitomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cary, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span> <span> </span><span class="NLM_article-title">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18480</span>– <span class="NLM_lpage">18493</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.18632%2Foncotarget.24883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=29719619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=18480-18493&author=K.+Nakayamaauthor=M.+M.+Szewczykauthor=C.+Dela%0ASenaauthor=H.+Wuauthor=A.+Dongauthor=H.+Zengauthor=F.+Liauthor=R.+F.+de+Freitasauthor=M.+S.+Eramauthor=M.+Schapiraauthor=Y.+Babaauthor=M.+Kunitomoauthor=D.+R.+Caryauthor=M.+Tawadaauthor=A.+Ohashiauthor=Y.+Imaedaauthor=K.+S.+Saikatenduauthor=C.+E.+Grimshawauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=A.+Kibaauthor=D.+Tomitaauthor=P.+J.+Brown&title=TP-064%2C+a+potent+and+selective+small+molecule+inhibitor+of+PRMT4+for+multiple+myeloma&doi=10.18632%2Foncotarget.24883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span></div><div class="casAuthors">Nakayama Kazuhide; Baba Yuji; Kunitomo Mihoko; Cary Douglas R; Ohashi Akihiro; Imaeda Yasuhiro; Kiba Atsushi; Tomita Daisuke; Szewczyk Magdalena M; Dela Sena Carlo; Wu Hong; Dong Aiping; Zeng Hong; Li Fengling; de Freitas Renato Ferreira; Eram Mohammad S; Schapira Matthieu; Vedadi Masoud; Arrowsmith Cheryl H; Barsyte-Lovejoy Dalia; Brown Peter J; Schapira Matthieu; Vedadi Masoud; Tawada Michiko; Saikatendu Kumar Singh; Grimshaw Charles E; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">18480-18493</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methyltransferase (PRMT) 4 (also known as coactivator-associated arginine methyltransferase 1; CARM1) is involved in a variety of biological processes and is considered as a candidate oncogene owing to its overexpression in several types of cancer.  Selective PRMT4 inhibitors are useful tools for clarifying the molecular events regulated by PRMT4 and for validating PRMT4 as a therapeutic target.  Here, we report the discovery of TP-064, a potent, selective, and cell-active chemical probe of human PRMT4 and its co-crystal structure with PRMT4.  TP-064 inhibited the methyltransferase activity of PRMT4 with high potency (half-maximal inhibitory concentration, IC50 < 10 nM) and selectivity over other PRMT family proteins, and reduced arginine dimethylation of the PRMT4 substrates BRG1-associated factor 155 (BAF155; IC50= 340 ± 30 nM) and Mediator complex subunit 12 (MED12; IC50 = 43 ± 10 nM).  TP-064 treatment inhibited the proliferation of a subset of multiple myeloma cell lines, with affected cells arrested in G1 phase of the cell cycle.  TP-064 and its negative control (TP-064N) will be valuable tools to further investigate the biology of PRMT4 and the therapeutic potential of PRMT4 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTsNG0-q3aOlw20zIbnCCI3fW6udTcc2eb6EAlrgZJq17ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D&md5=f910a11dc3aa023564499ba2e6209e31</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24883%26sid%3Dliteratum%253Aachs%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3Dde%2BFreitas%26aufirst%3DR.%2BF.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBaba%26aufirst%3DY.%26aulast%3DKunitomo%26aufirst%3DM.%26aulast%3DCary%26aufirst%3DD.%2BR.%26aulast%3DTawada%26aufirst%3DM.%26aulast%3DOhashi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DKiba%26aufirst%3DA.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DTP-064%252C%2520a%2520potent%2520and%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520PRMT4%2520for%2520multiple%2520myeloma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D18480%26epage%3D18493%26doi%3D10.18632%2Foncotarget.24883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela Sena, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivcharuk, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibanez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazutis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">A chemical probe of CARM1 alters epigenetic plasticity against breast cancer cell invasion</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e47110</span> <span class="refDoi"> DOI: 10.7554/eLife.47110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.7554%2FeLife.47110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=31657716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlalt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&author=X.+C.+Caiauthor=T.+Zhangauthor=E.+J.+Kimauthor=M.+Jiangauthor=K.+Wangauthor=J.+Wangauthor=S.+Chenauthor=N.+Zhangauthor=H.+Wuauthor=F.+Liauthor=C.+C.+Dela+Senaauthor=H.+Zengauthor=V.+Vivcharukauthor=X.+Niuauthor=W.+Zhengauthor=J.+P.+Leeauthor=Y.+Chenauthor=D.+Barsyteauthor=M.+Szewczykauthor=T.+Hajianauthor=G.+Ibanezauthor=A.+Dongauthor=L.+Dombrovskiauthor=Z.+Zhangauthor=H.+Dengauthor=J.+Minauthor=C.+H.+Arrowsmithauthor=L.+Mazutisauthor=L.+Shiauthor=M.+Vedadiauthor=P.+J.+Brownauthor=J.+Xiangauthor=L.+X.+Qinauthor=W.+Xuauthor=M.+Luo&title=A+chemical+probe+of+CARM1+alters+epigenetic+plasticity+against+breast+cancer+cell+invasion&doi=10.7554%2FeLife.47110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe of CARM1 alters epigenetic plasticity against breast cancer cell invasion</span></div><div class="casAuthors">Cai, Xiao-Chuan; Zhang, Tuo; Kim, Eui-jun; Jiang, Ming; Wang, Ke; Wang, Junyi; Chen, Shi; Zhang, Nawei; Wu, Hong; Li, Fengling; Dela Sena, Carlo C.; Zeng, Hong; Vivcharuk, Victor; Niu, Xiang; Zheng, Weihong; Lee, Jonghan P.; Chen, Yuling; Barsyte, Dalia; Szewczyk, Magda; Hajian, Taraneh; Ibanez, Glorymar; Dong, Aiping; Dombrovski, Ludmila; Zhang, Zhenyu; Deng, Haiteng; Min, Jinrong; Arrowsmith, Cheryl H.; Mazutis, Linas; Shi, Lei; Vedadi, Masoud; Brown, Peter J.; Xiang, Jenny; Qin, Li-Xuan; Xu, Wei; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e47110/1-e47110/42</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">CARM1 is a cancer-relevant protein arginine methyltransferase that regulates many aspects of transcription.  Its pharmacol. inhibition is a promising anti-cancer strategy.  Here SKI-73 (6a in this work) is presented as a CARM1 chem. probe with pro-drug properties.  SKI-73 (6a) can rapidly penetrate cell membranes and then be processed into active inhibitors, which are retained intracellularly with 10-fold enrichment for several days.  These compds. were characterized for their potency, selectivity, modes of action, and on-target engagement.  SKI-73 (6a) recapitulates the effect of CARM1 knockout against breast cancer cell invasion.  Single-cell RNA-seq anal. revealed that the SKI-73(6a)-assocd. redn. of invasiveness acts by altering epigenetic plasticity and suppressing the invasion-prone subpopulation.  Interestingly, SKI-73 (6a) and CARM1 knockout alter the epigenetic plasticity with remarkable difference, suggesting distinct modes of action for small-mol. and genetic perturbations.  We therefore discovered a CARM1-addiction mechanism of cancer metastasis and developed a chem. probe to target this process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpArjVrLFUaObVg90H21EOLACvtfcHk0ljUvuaQodteMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlalt7jF&md5=644ad9780e1504c8793e3ae5195b5df1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.7554%2FeLife.47110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.47110%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DX.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DE.%2BJ.%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDela%2BSena%26aufirst%3DC.%2BC.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DVivcharuk%26aufirst%3DV.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DJ.%2BP.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DIbanez%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DMazutis%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DL.%2BX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DA%2520chemical%2520probe%2520of%2520CARM1%2520alters%2520epigenetic%2520plasticity%2520against%2520breast%2520cancer%2520cell%2520invasion%26jtitle%3DElife%26date%3D2019%26volume%3D8%26doi%3D10.7554%2FeLife.47110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0ljUvuaQodteMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1033</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1033-1038&author=H.+Linauthor=M.+Wangauthor=Y.+W.+Zhangauthor=S.+Tongauthor=R.+A.+Lealauthor=R.+Shettyauthor=K.+Vaddiauthor=J.+I.+Luengo&title=Discovery+of+potent+and+selective+covalent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1021%2Facsmedchemlett.9b00074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Protein Arginine Methyltransferase 5 (PRMT5) Inhibitors</span></div><div class="casAuthors">Lin Hong; Wang Min; Zhang Yang W; Leal Raul A; Shetty Rupa; Vaddi Kris; Luengo Juan I; Tong Shuilong</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1033-1038</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is known to symmetrically dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 possesses a cysteine (C449) in the active site, unique to PRMT5.  Therefore, covalent PRMT5 inhibition is an attractive chemical approach.  Herein, we report an exciting discovery of a series of novel hemiaminals that under physiological conditions can be converted to aldehydes and react with C449 to form covalent adducts, which presumably undergo an unprecedented elimination to form the thiol-vinyl ethers, as indicated by electron density in the co-crystal structure of the PRMT5/MEP50 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdNDVSKB6iezW1zwg2yUK5fW6udTcc2eblN49qFRybbrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D&md5=4b5b08dc3e3615144ece564f70bf3bd0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00074%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BW.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DLeal%26aufirst%3DR.%2BA.%26aulast%3DShetty%26aufirst%3DR.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1033%26epage%3D1038%26doi%3D10.1021%2Facsmedchemlett.9b00074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 ± 3 nM and 25 ± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0lh7P2fwsJmnIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 μM.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0lgklLnnh2qErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181601</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1371%2Fjournal.pone.0181601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=28806746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=G.+M.+Kongauthor=M.+Yuauthor=Z.+Guauthor=Z.+Chenauthor=R.+M.+Xuauthor=D.+O%E2%80%99Bryantauthor=Z.+Wang&title=Selective+small-chemical+inhibitors+of+protein+arginine+methyltransferase+5+with+anti-lung+cancer+activity&doi=10.1371%2Fjournal.pone.0181601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span></div><div class="casAuthors">Kong, Gui-Mei; Yu, Min; Gu, Zhongping; Chen, Zhi; Xu, Rui-Ming; O'Bryant, Deon; Wang, Zhengxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181601/1-e0181601/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) plays crit. roles in a wide variety of biol. processes, including tumorigenesis.  By screening a library of small chem. compds., we identified eight compds. that selectively inhibit the PRMT5 enzymic activity, with IC50 values ranging from 0.1 to 6 μM.  Mol. docking simulation and site directed mutagenesis indicated that identified compds. target the substrate-binding site in PRMT5.  Treatment of lung cancer cells with identified inhibitors led to inhibition of the sym. arginine methylation of SmD3 and histones and the cellular proliferation.  Oral administration of the inhibitor demonstrated antitumor activity in a lung tumor xenograft model.  Thus, identified PRMT5-specific small-mol. inhibitors would help elucidate the biol. roles of PRMT5 and serve as lead compds. for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrmbn8I9YMLVg90H21EOLACvtfcHk0lgklLnnh2qErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D&md5=0306ed1351aac8a04d286df255c6e432</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181601%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DG.%2BM.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DO%25E2%2580%2599Bryant%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DSelective%2520small-chemical%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520with%2520anti-lung%2520cancer%2520activity%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Organ, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukushi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seña, C. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takizawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of PRMT7 links arginine monomethylation to the cellular stress response</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">2396</span> <span class="refDoi"> DOI: 10.1038/s41467-020-16271-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fs41467-020-16271-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=32409666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXps1OltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=M.+M.+Szewczykauthor=Y.+Ishikawaauthor=S.+Organauthor=N.+Sakaiauthor=F.+Liauthor=S.+Acklooauthor=M.+Eramauthor=D.+Dilworthauthor=H.+Fukushiauthor=R.+Hardingauthor=C.+C.+D.+Se%C3%B1aauthor=T.+Sugoauthor=K.+Hayashiauthor=D.+Macleodauthor=C.+Zepedaauthor=S.+Takagiauthor=R.+Al-Awarauthor=S.+Richardauthor=M.+Takizawaauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=P.+J.+Brownauthor=H.+Naraauthor=D.+Barsyte-Lovejoy&title=Pharmacological+inhibition+of+PRMT7+links+arginine+monomethylation+to+the+cellular+stress+response&doi=10.1038%2Fs41467-020-16271-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of PRMT7 links arginine monomethylation to the cellular stress response</span></div><div class="casAuthors">Szewczyk, Magdalena M.; Ishikawa, Yoshinori; Organ, Shawna; Sakai, Nozomu; Li, Fengling; Halabelian, Levon; Ackloo, Suzanne; Couzens, Amber L.; Eram, Mohammad; Dilworth, David; Fukushi, Hideto; Harding, Rachel; dela Sena, Carlo C.; Sugo, Tsukasa; Hayashi, Kozo; McLeod, David; Zepeda, Carlos; Aman, Ahmed; Sanchez-Osuna, Maria; Bonneil, Eric; Takagi, Shinji; Al-Awar, Rima; Tyers, Mike; Richard, Stephane; Takizawa, Masayuki; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Vedadi, Masoud; Brown, Peter J.; Nara, Hiroshi; Barsyte-Lovejoy, Dalia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2396</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Protein arginine methyltransferases (PRMTs) regulate diverse biol. processes and are increasingly being recognized for their potential as drug targets.  Here we report the discovery of a potent, selective, and cell-active chem. probe for PRMT7.  SGC3027 is a cell permeable prodrug, which in cells is converted to SGC8158, a potent, SAM-competitive PRMT7 inhibitor.  Inhibition or knockout of cellular PRMT7 results in drastically reduced levels of arginine monomethylated HSP70 family stress-assocd. proteins.  Structural and biochem. analyses reveal that PRMT7-driven in vitro methylation of HSP70 at R469 requires an ATP-bound, open conformation of HSP70.  In cells, SGC3027 inhibits methylation of both constitutive and inducible forms of HSP70, and leads to decreased tolerance for perturbations of proteostasis including heat shock and proteasome inhibitors.  These results demonstrate a role for PRMT7 and arginine methylation in stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rIt3-Hwdr7Vg90H21EOLACvtfcHk0lgklLnnh2qErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXps1OltLk%253D&md5=337393269b76b0373d467ff6e4108f08</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16271-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16271-z%26sid%3Dliteratum%253Aachs%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DOrgan%26aufirst%3DS.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DEram%26aufirst%3DM.%26aulast%3DDilworth%26aufirst%3DD.%26aulast%3DFukushi%26aufirst%3DH.%26aulast%3DHarding%26aufirst%3DR.%26aulast%3DSe%25C3%25B1a%26aufirst%3DC.%2BC.%2BD.%26aulast%3DSugo%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DK.%26aulast%3DMacleod%26aufirst%3DD.%26aulast%3DZepeda%26aufirst%3DC.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DTakizawa%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26atitle%3DPharmacological%2520inhibition%2520of%2520PRMT7%2520links%2520arginine%2520monomethylation%2520to%2520the%2520cellular%2520stress%2520response%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-16271-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">3537</span>– <span class="NLM_lpage">3543</span>, <span class="refDoi"> DOI: 10.1074/jbc.M108786200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1074%2Fjbc.M108786200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=11724789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=3537-3543&author=A.+Frankelauthor=N.+Yadavauthor=J.+Leeauthor=T.+L.+Branscombeauthor=S.+Clarkeauthor=M.+T.+Bedford&title=The+novel+human+protein+arginine+N-methyltransferase+PRMT6+is+a+nuclear+enzyme+displaying+unique+substrate+specificity&doi=10.1074%2Fjbc.M108786200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span></div><div class="casAuthors">Frankel, Adam; Yadav, Neelu; Lee, Jaeho; Branscombe, Tina L.; Clarke, Steven; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3537-3543</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein arginine methylation is a prevalent posttranslational modification in eukaryotic cells that has been implicated in signal transduction, the metab. of nascent pre-RNA, and the transcriptional activation processes.  In searching the human genome for protein arginine N-methyltransferase (PRMT) family members, a novel gene has been found on chromosome 1 that encodes for an apparent methyltransferase, PRMT6.  The polypeptide chain of PRMT6 is 41.9 kDa consisting of a catalytic core sequence common to other PRMT enzymes.  Expressed as a glutathione S-transferase fusion protein, PRMT6 demonstrates type I PRMT activity, capable of forming both ω-NG-monomethylarginine and asym. ω-NG,NG-dimethylarginine derivs. on the recombinant glycine- and arginine-rich substrate in a processive manner with a specific activity of 144 pmol Me groups transferred min-1 mg-1 enzyme.  A comparison of substrate specificity reveals that PRMT6 is functionally distinct from two previously characterized type I enzymes, PRMT1 and PRMT4.  In addn., PRMT6 displays automethylation activity; it is the first PRMT to do so.  This novel human PRMT, which resides solely in the nucleus when fused to the green fluorescent protein, joins a family of enzymes with diverse functions within cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_K0kYqP2NbVg90H21EOLACvtfcHk0ljIFjCswDSrnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D&md5=1faaf27f44d72742749bdee2998ec9c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M108786200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M108786200%26sid%3Dliteratum%253Aachs%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520novel%2520human%2520protein%2520arginine%2520N-methyltransferase%2520PRMT6%2520is%2520a%2520nuclear%2520enzyme%2520displaying%2520unique%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D3537%26epage%3D3543%26doi%3D10.1074%2Fjbc.M108786200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lorenzo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastian, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Mouse models of overexpression reveal distinct oncogenic roles for different type I protein arginine methyltransferases</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-18-1995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1158%2F0008-5472.CAN-18-1995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=30352814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotFGhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=21-32&author=J.+Baoauthor=A.+Di+Lorenzoauthor=K.+Linauthor=Y.+Luauthor=Y.+Zhongauthor=M.+M.+Sebastianauthor=W.+J.+Mullerauthor=Y.+Yangauthor=M.+T.+Bedford&title=Mouse+models+of+overexpression+reveal+distinct+oncogenic+roles+for+different+type+I+protein+arginine+methyltransferases&doi=10.1158%2F0008-5472.CAN-18-1995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse models of overexpression reveal distinct oncogenic roles for different type i protein arginine methyltransferases</span></div><div class="casAuthors">Bao, Jianqiang; Di Lorenzo, Alessandra; Lin, Kevin; Lu, Yue; Zhong, Yi; Sebastian, Manu M.; Muller, William J.; Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-32</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMT) are generally not mutated in diseased states, but they are overexpressed in a no. of cancers, including breast cancer.  To address the possible roles of PRMT overexpression in mammary gland tumorigenesis, we generated Cre-activated PRMT1, CARM1, and PRMT6 overexpression mouse models.  These three enzymes are the primary type I PRMTs and are responsible for the majority of the asym. arginine methylation deposited in the cells.  Using either a keratin 5-Cre recombinase (K5-Cre) cross or an MMTV-NIC mouse, we investigated the impact of PRMT overexpression alone or in the context of a HER2-driven model of breast cancer, resp.  The overexpression of all three PRMTs induced hyper-branching of the mammary glands and increased Ki-67 staining.  When combined with the MMTV-NIC model, these in vivo expts. provided the first genetic evidence implicating elevated levels of these three PRMTs in mammary gland tumorigenesis, albeit with variable degrees of tumor promotion and latency.  In addn., these mouse models provided valuable tools for exploring the biol. roles and mol. mechanisms of PRMT overexpression in the mammary gland.  For example, transcriptome anal. of purified mammary epithelial cells isolated from bigenic NIC-PRMT1Tg and NIC-PRMT6Tg mice revealed a deregulated PI3K-AKT pathway.  In the future, these PRMTTg lines can be leveraged to investigate the roles of arginine methylation in other tissues and tumor model systems using different tissue-specific Cre crosses, and they can also be used for testing the in vivo efficacy of small mol. inhibitors that target these PRMT function in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2aAowvjAR3rVg90H21EOLACvtfcHk0ljIFjCswDSrnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotFGhsL0%253D&md5=8a8799e61fa2a62e4a93db7a74d8b403</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-18-1995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-18-1995%26sid%3Dliteratum%253Aachs%26aulast%3DBao%26aufirst%3DJ.%26aulast%3DDi%2BLorenzo%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DSebastian%26aufirst%3DM.%2BM.%26aulast%3DMuller%26aufirst%3DW.%2BJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DMouse%2520models%2520of%2520overexpression%2520reveal%2520distinct%2520oncogenic%2520roles%2520for%2520different%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D21%26epage%3D32%26doi%3D10.1158%2F0008-5472.CAN-18-1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veland, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayatri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span> <span> </span><span class="NLM_article-title">The arginine methyltransferase PRMT6 regulates DNA methylation and contributes to global DNA hypomethylation in cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3390</span>– <span class="NLM_lpage">3397</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.11.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.celrep.2017.11.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=29262320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=3390-3397&author=N.+Velandauthor=S.+Hardikarauthor=Y.+Zhongauthor=S.+Gayatriauthor=J.+Danauthor=B.+D.+Strahlauthor=S.+B.+Rothbartauthor=M.+T.+Bedfordauthor=T.+Chen&title=The+arginine+methyltransferase+PRMT6+regulates+DNA+methylation+and+contributes+to+global+DNA+hypomethylation+in+cancer&doi=10.1016%2Fj.celrep.2017.11.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Arginine Methyltransferase PRMT6 Regulates DNA Methylation and Contributes to Global DNA Hypomethylation in Cancer</span></div><div class="casAuthors">Veland, Nicolas; Hardikar, Swanand; Zhong, Yi; Gayatri, Sitaram; Dan, Jiameng; Strahl, Brian D.; Rothbart, Scott B.; Bedford, Mark T.; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3390-3397</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">DNA methylation plays crucial roles in chromatin structure and gene expression.  Aberrant DNA methylation patterns, including global hypomethylation and regional hypermethylation, are assocd. with cancer and implicated in oncogenic events.  How DNA methylation is regulated in developmental and cellular processes and dysregulated in cancer is poorly understood.  Here, we show that PRMT6, a protein arginine methyltransferase responsible for asym. dimethylation of histone H3 arginine 2 (H3R2me2a), neg. regulates DNA methylation and that PRMT6 upregulation contributes to global DNA hypomethylation in cancer.  Mechanistically, PRMT6 overexpression impairs chromatin assocn. of UHRF1, an accessory factor of DNMT1, resulting in passive DNA demethylation.  The effect is likely due to elevated H3R2me2a, which inhibits the interaction between UHRF1 and histone H3.  Our work identifies a mechanistic link between protein arginine methylation and DNA methylation, which is disrupted in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeK-EYH5WxEbVg90H21EOLACvtfcHk0ljIFjCswDSrnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbK&md5=4ed7f5ed35723fadde97d9a5e3e99ea5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.11.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.11.082%26sid%3Dliteratum%253Aachs%26aulast%3DVeland%26aufirst%3DN.%26aulast%3DHardikar%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DGayatri%26aufirst%3DS.%26aulast%3DDan%26aufirst%3DJ.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DRothbart%26aufirst%3DS.%2BB.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520arginine%2520methyltransferase%2520PRMT6%2520regulates%2520DNA%2520methylation%2520and%2520contributes%2520to%2520global%2520DNA%2520hypomethylation%2520in%2520cancer%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D3390%26epage%3D3397%26doi%3D10.1016%2Fj.celrep.2017.11.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela
Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+Shenauthor=M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+Liauthor=K.+V.+Butlerauthor=H.+U.+Kaniskanauthor=B.+A.+Speedauthor=C.+Dela%0ASenaauthor=A.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0li2S19zsQL0wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabelian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class protein arginine methyltransferase 6 (PRMT6) covalent inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5477</span>– <span class="NLM_lpage">5487</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5477-5487&author=Y.+Shenauthor=F.+Liauthor=M.+M.+Szewczykauthor=L.+Halabelianauthor=K.+S.+Parkauthor=I.+Chauauthor=A.+Dongauthor=H.+Zengauthor=H.+Chenauthor=F.+Mengauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+first-in-class+protein+arginine+methyltransferase+6+%28PRMT6%29+covalent+inhibitor&doi=10.1021%2Facs.jmedchem.0c00406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor</span></div><div class="casAuthors">Shen, Yudao; Li, Fengling; Szewczyk, Magdalena M.; Halabelian, Levon; Park, Kwang-su; Chau, Irene; Dong, Aiping; Zeng, Hong; Chen, He; Meng, Fanye; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5477-5487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biol. processes assocd. with multiple cancers.  Well-characterized potent, selective, and cell-active PRMT6 inhibitors are invaluable tools for testing biol. and therapeutic hypotheses.  Although there are several known reversible PRMT6 inhibitors, covalent PRMT6 inhibitors have not been reported.  Based on a cocrystal structure of PRMT6-MS023 (a type I PRMT inhibitor), we discovered the first potent and cell-active irreversible PRMT6 inhibitor, 4 (MS117)(I).  The covalent binding mode of compd. 4 to PRMT6 was confirmed by mass spectrometry and kinetic studies and by a cocrystal structure.  Compd. 4 did not covalently modify other closely related PRMTs, potently inhibited PRMT6 in cells, and was selective for PRMT6 over other methyltransferases.  We also developed two structurally similar control compds., 5 (MS167) and 7 (MS168).  We provide these valuable chem. tools to the scientific community for further studying PRMT6 physiol. and pathophysiol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTU1W0CZjN17Vg90H21EOLACvtfcHk0li2S19zsQL0wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D&md5=cb244ab5a38d04320439c05048998b2f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DHalabelian%26aufirst%3DL.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520protein%2520arginine%2520methyltransferase%25206%2520%2528PRMT6%2529%2520covalent%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5477%26epage%3D5487%26doi%3D10.1021%2Facs.jmedchem.0c00406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradei, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: identification of the first PRMT6 tool compound</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0lj6sVaiVH3ryw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palte, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacey, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansueto, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reutershan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siliphaivanh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sondey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machacek, M. R.</span></span> <span> </span><span class="NLM_article-title">Allosteric modulation of protein arginine methyltransferase 5 (PRMT5)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1688</span>– <span class="NLM_lpage">1693</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00525</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00525" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFehtb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1688-1693&author=R.+L.+Palteauthor=S.+E.+Schneiderauthor=M.+D.+Altmanauthor=R.+P.+Hayesauthor=S.+Kawamuraauthor=B.+M.+Laceyauthor=M.+S.+Mansuetoauthor=M.+Reutershanauthor=P.+Siliphaivanhauthor=C.+Sondeyauthor=H.+Xuauthor=Z.+Xuauthor=Y.+Yeauthor=M.+R.+Machacek&title=Allosteric+modulation+of+protein+arginine+methyltransferase+5+%28PRMT5%29&doi=10.1021%2Facsmedchemlett.9b00525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Modulation of Protein Arginine Methyltransferase 5 (PRMT5)</span></div><div class="casAuthors">Palte, Rachel L.; Schneider, Sebastian E.; Altman, Michael D.; Hayes, Robert P.; Kawamura, Shuhei; Lacey, Brian M.; Mansueto, My Sam; Reutershan, Michael; Siliphaivanh, Phieng; Sondey, Christopher; Xu, Haiyan; Xu, Zangwei; Ye, Yingchun; Machacek, Michelle R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1688-1693</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) belongs to a family of enzymes that regulate the posttranslational modification of histones and other proteins via methylation of arginine.  Methylation of histones is linked to an increase in transcription and regulates a manifold of functions such as signal transduction and transcriptional regulation.  PRMT5 has been shown to be upregulated in the tumor environment of several cancer types, and the inhibition of PRMT5 activity was identified as a potential way to reduce tumor growth.  Previously, four different modes of PRMT5 inhibition were known-competing (covalently or non-covalently) with the essential cofactor S-adenosyl methionine (SAM), blocking the substrate binding pocket, or blocking both simultaneously.  Herein we describe an unprecedented conformation of PRMT5 in which the formation of an allosteric binding pocket abrogates the enzyme's canonical binding site and present the discovery of potent small mol. allosteric PRMT5 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFkbnJkLKiL7Vg90H21EOLACvtfcHk0lj6sVaiVH3ryw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFehtb7P&md5=75a8f75e60a3e20444af90b79c5af810</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00525%26sid%3Dliteratum%253Aachs%26aulast%3DPalte%26aufirst%3DR.%2BL.%26aulast%3DSchneider%26aufirst%3DS.%2BE.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DHayes%26aufirst%3DR.%2BP.%26aulast%3DKawamura%26aufirst%3DS.%26aulast%3DLacey%26aufirst%3DB.%2BM.%26aulast%3DMansueto%26aufirst%3DM.%2BS.%26aulast%3DReutershan%26aufirst%3DM.%26aulast%3DSiliphaivanh%26aufirst%3DP.%26aulast%3DSondey%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYe%26aufirst%3DY.%26aulast%3DMachacek%26aufirst%3DM.%2BR.%26atitle%3DAllosteric%2520modulation%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1688%26epage%3D1693%26doi%3D10.1021%2Facsmedchemlett.9b00525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mailliot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troffer-Charlier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianferani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavarelli, J.</span></span> <span> </span><span class="NLM_article-title">Functional insights from high resolution structures of mouse protein arginine methyltransferase 6</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2015.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.jsb.2015.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=26094878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFSjtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2015&pages=175-183&author=L.+Bonnefondauthor=J.+Stojkoauthor=J.+Mailliotauthor=N.+Troffer-Charlierauthor=V.+Curaauthor=J.+M.+Wurtzauthor=S.+Cianferaniauthor=J.+Cavarelli&title=Functional+insights+from+high+resolution+structures+of+mouse+protein+arginine+methyltransferase+6&doi=10.1016%2Fj.jsb.2015.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Functional insights from high resolution structures of mouse protein arginine methyltransferase 6</span></div><div class="casAuthors">Bonnefond, Luc; Stojko, Johann; Mailliot, Justine; Troffer-Charlier, Nathalie; Cura, Vincent; Wurtz, Jean-Marie; Cianferani, Sarah; Cavarelli, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-183</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">PRMT6 is a protein arginine methyltransferase involved in transcriptional regulation, human immunodeficiency virus pathogenesis, DNA base excision repair, and cell cycle progression.  Like other PRMTs, PRMT6 is overexpressed in several cancer types and is therefore considered as a potential anti-cancer drug target.  In the present study, we described six crystal structures of PRMT6 from Mus musculus, solved and refined at 1.34 Å for the highest resoln. structure.  The crystal structures revealed that the folding of the helix αX is required to stabilize a productive active site before methylation of the bound peptide can occur.  In the absence of cofactor, metal cations can be found in the catalytic pocket at the expected position of the guanidinium moiety of the target arginine substrate.  Using mass spectrometry under native conditions, we show that PRMT6 dimer binds two cofactor and a single H4 peptide mols.  Finally, we characterized a new site of in vitro automethylation of mouse PRMT6 at position 7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIfUGr0Ak2b7Vg90H21EOLACvtfcHk0lj6sVaiVH3ryw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFSjtLjF&md5=306f52f65b771a7e6351d0c21c11a570</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2015.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2015.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DBonnefond%26aufirst%3DL.%26aulast%3DStojko%26aufirst%3DJ.%26aulast%3DMailliot%26aufirst%3DJ.%26aulast%3DTroffer-Charlier%26aufirst%3DN.%26aulast%3DCura%26aufirst%3DV.%26aulast%3DWurtz%26aufirst%3DJ.%2BM.%26aulast%3DCianferani%26aufirst%3DS.%26aulast%3DCavarelli%26aufirst%3DJ.%26atitle%3DFunctional%2520insights%2520from%2520high%2520resolution%2520structures%2520of%2520mouse%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2015%26volume%3D191%26spage%3D175%26epage%3D183%26doi%3D10.1016%2Fj.jsb.2015.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sinnokrot, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherrill, C. D.</span></span> <span> </span><span class="NLM_article-title">Substituent effects in pi-pi interactions: sandwich and T-shaped configurations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">7690</span>– <span class="NLM_lpage">7697</span>, <span class="refDoi"> DOI: 10.1021/ja049434a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja049434a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVejur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=7690-7697&author=M.+O.+Sinnokrotauthor=C.+D.+Sherrill&title=Substituent+effects+in+pi-pi+interactions%3A+sandwich+and+T-shaped+configurations&doi=10.1021%2Fja049434a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Substituent Effects in π-π Interactions: Sandwich and T-Shaped Configurations</span></div><div class="casAuthors">Sinnokrot, Mutasem Omar; Sherrill, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7690-7697</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sandwich and T-shaped configurations of benzene dimer, benzene-phenol, benzene-toluene, benzene-fluorobenzene, and benzene-benzonitrile are studied by coupled-cluster theory to elucidate how substituents tune π-π interactions.  All substituted sandwich dimers bind more strongly than benzene dimer, whereas the T-shaped configurations bind more or less favorably depending on the substituent.  Symmetry-adapted perturbation theory (SAPT) indicates that electrostatic, dispersion, induction, and exchange-repulsion contributions are all significant to the overall binding energies, and all but induction are important in detg. relative energies.  Models of π-π interactions based solely on electrostatics, such as the Hunter-Sanders rules, do not seem capable of explaining the energetic ordering of the dimers considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFBZHgFWNA5rVg90H21EOLACvtfcHk0lijfiKYUYoXWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVejur4%253D&md5=f9c4afdd96f98ecc1d46f93a0ce0f17a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja049434a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja049434a%26sid%3Dliteratum%253Aachs%26aulast%3DSinnokrot%26aufirst%3DM.%2BO.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26atitle%3DSubstituent%2520effects%2520in%2520pi-pi%2520interactions%253A%2520sandwich%2520and%2520T-shaped%2520configurations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D7690%26epage%3D7697%26doi%3D10.1021%2Fja049434a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchcock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1176</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0lijfiKYUYoXWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution--from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%2D%2Dfrom+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lijfiKYUYoXWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution--from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lh5gasaVJKxqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in BUSTER</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1107/S0907444911056058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444911056058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=22505257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=368-380&author=O.+S.+Smartauthor=T.+O.+Womackauthor=C.+Flensburgauthor=P.+Kellerauthor=W.+Paciorekauthor=A.+Sharffauthor=C.+Vonrheinauthor=G.+Bricogne&title=Exploiting+structure+similarity+in+refinement%3A+automated+NCS+and+target-structure+restraints+in+BUSTER&doi=10.1107%2FS0907444911056058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in BUSTER</span></div><div class="casAuthors">Smart, Oliver S.; Womack, Thomas O.; Flensburg, Claus; Keller, Peter; Paciorek, Wlodek; Sharff, Andrew; Vonrhein, Clemens; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">368-380</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Maximum-likelihood X-ray macromol. structure refinement in BUSTER has been extended with restraints facilitating the exploitation of structural similarity.  The similarity can be between two or more chains within the structure being refined, thus favoring NCS, or to a distinct 'target' structure that remains fixed during refinement.  The local structural similarity restraints (LSSR) approach considers all distances less than 5.5 Å between pairs of atoms in the chain to be restrained.  For each, the difference from the distance between the corresponding atoms in the related chain is found.  LSSR applies a restraint penalty on each difference.  A functional form that reaches a plateau for large differences is used to avoid the restraints distorting parts of the structure that are not similar.  Because LSSR are local, there is no need to sep. out domains.  Some restraint pruning is still necessary, but this has been automated.  LSSR have been available to academic users of BUSTER since 2009 with the easy-to-use and options.  The use of LSSR is illustrated in the re-refinement of PDB entries , where enables the correct ligand-binding structure to be found, and , where contributes to the location of an addnl. copy of the cyclic peptide ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu-qo0DGjn3bVg90H21EOLACvtfcHk0lh5gasaVJKxqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D&md5=bea89f86f489e8dbd00a9866a3d7dab9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444911056058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911056058%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26aulast%3DWomack%26aufirst%3DT.%2BO.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DExploiting%2520structure%2520similarity%2520in%2520refinement%253A%2520automated%2520NCS%2520and%2520target-structure%2520restraints%2520in%2520BUSTER%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D368%26epage%3D380%26doi%3D10.1107%2FS0907444911056058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lh5gasaVJKxqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MOLPROBITY: structure validation and all-atom contact analysis for nucleic acids and their complexes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W615</span>– <span class="NLM_lpage">W619</span>, <span class="refDoi"> DOI: 10.1093/nar/gkh398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1093%2Fnar%2Fgkh398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=15215462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W615-W619&author=I.+W.+Davisauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MOLPROBITY%3A+structure+validation+and+all-atom+contact+analysis+for+nucleic+acids+and+their+complexes&doi=10.1093%2Fnar%2Fgkh398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MolPROBITY: Structure validation and all-atom contact analysis for nucleic acids and their complexes</span></div><div class="casAuthors">Davis, Ian W.; Murray, Laura Weston; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W615-W619</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MolProbity is a general-purpose web service offering quality validation for three-dimensional (3D) structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact anal. of any steric problems within the mols. and can calc. and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addn. and full optimization of all hydrogen atoms, both polar and nonpolar.  The results are reported in multiple forms: as overall numeric scores, as lists, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KING viewer.  This service is available free to all users at http://kinemage.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9I64e-HQbnbVg90H21EOLACvtfcHk0lhco5xoLhZqBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D&md5=2050f7acf49743addde69c5ce9c9e2e4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh398%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMOLPROBITY%253A%2520structure%2520validation%2520and%2520all-atom%2520contact%2520analysis%2520for%2520nucleic%2520acids%2520and%2520their%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW615%26epage%3DW619%26doi%3D10.1093%2Fnar%2Fgkh398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vhuiyan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of arginine asymmetrical dimethylation by PRMT6</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>473</i></span>,  <span class="NLM_fpage">3049</span>– <span class="NLM_lpage">3063</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1042%2FBCJ20160537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=27480107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2016&pages=3049-3063&author=H.+Wuauthor=W.+Zhengauthor=M.+S.+Eramauthor=M.+Vhuiyanauthor=A.+Dongauthor=H.+Zengauthor=H.+Heauthor=P.+Brownauthor=A.+Frankelauthor=M.+Vedadiauthor=M.+Luoauthor=J.+Min&title=Structural+basis+of+arginine+asymmetrical+dimethylation+by+PRMT6&doi=10.1042%2FBCJ20160537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of arginine asymmetrical dimethylation by PRMT6</span></div><div class="casAuthors">Wu, Hong; Zheng, Weihong; Eram, Mohammad S.; Vhuiyan, Mynol; Dong, Aiping; Zeng, Hong; He, Hao; Brown, Peter; Frankel, Adam; Vedadi, Masoud; Luo, Minkui; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3049-3063</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">PRMT6 is a type I protein arginine methyltransferase, generating the asym. dimethylarginine mark on proteins such as histone H3R2.  Asym. dimethylation of histone H3R2 by PRMT6 acts as a repressive mark that antagonizes trimethylation of H3 lysine 4 by the MLL histone H3K4 methyltransferase.  PRMT6 is overexpressed in several cancer types, including prostate, bladder and lung cancers; therefore, it is of great interest to develop potent and selective inhibitors for PRMT6.  Here, we report the synthesis of a potent bisubstrate inhibitor GMS [6'-methyleneamine sinefungin, an analog of sinefungin (SNF)], and the crystal structures of human PRMT6 in complex, resp., with S-adenosyl- L-homocysteine (SAH) and the bisubstrate inhibitor GMS that shed light on the significantly improved inhibition effect of GMS on methylation activity of PRMT6 compared with SAH and an S-adenosyl-L-methionine competitive methyltransferase inhibitor SNF.  In addn., we also crystd. PRMT6 in complex with SAH and a short arginine-contg. peptide.  Based on the structural information here and available in the PDB database, we proposed a mechanism that can rationalize the distinctive arginine methylation product specificity of different types of arginine methyltransferases and pinpoint the structural determinant of such a specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo061VtGpp_KLVg90H21EOLACvtfcHk0lhco5xoLhZqBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D&md5=dcacb06a69eb1a4856bea9b0d8458112</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160537%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DVhuiyan%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520arginine%2520asymmetrical%2520dimethylation%2520by%2520PRMT6%26jtitle%3DBiochem.%2520J.%26date%3D2016%26volume%3D473%26spage%3D3049%26epage%3D3063%26doi%3D10.1042%2FBCJ20160537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2110</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lhco5xoLhZqBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=30604761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=19&author=S.+Scheerauthor=S.+Acklooauthor=T.+S.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+A.+Wardauthor=A.+M.+Lewisauthor=J.+P.+Northropauthor=P.+L.+Richardsonauthor=H.+U.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+A.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+V.+M.+Huberauthor=D.+D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+J.+Brownauthor=C.+H.+Arrowsmith&title=A+chemical+biology+toolbox+to+study+protein+methyltransferases+and+epigenetic+signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span></div><div class="casAuthors">Scheer, Sebastian; Ackloo, Suzanne; Medina, Tiago S.; Schapira, Matthieu; Li, Fengling; Ward, Jennifer A.; Lewis, Andrew M.; Northrop, Jeffrey P.; Richardson, Paul L.; Kaniskan, H. Umit; Shen, Yudao; Liu, Jing; Smil, David; McLeod, David; Zepeda-Velazquez, Carlos A.; Luo, Minkui; Jin, Jian; Barsyte-Lovejoy, Dalia; Huber, Kilian V. M.; De Carvalho, Daniel D.; Vedadi, Masoud; Zaph, Colby; Brown, Peter J.; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) comprise a major class of epigenetic regulatory enzymes with therapeutic relevance.  Here we present a collection of chem. probes and assocd. reagents and data to elucidate the function of human and murine PMTs in cellular studies.  Our collection provides inhibitors and antagonists that together modulate most of the key regulatory methylation marks on histones H3 and H4, providing an important resource for modulating cellular epigenomes.  We describe a comprehensive and comparative characterization of the probe collection with respect to their potency, selectivity, and mode of inhibition.  We demonstrate the utility of this collection in CD4+ T cell differentiation assays revealing the potential of individual probes to alter multiple T cell subpopulations which may have implications for T cell-mediated processes such as inflammation and immuno-oncol.  In particular, we demonstrate a role for DOT1L in limiting Th1 cell differentiation and maintaining lineage integrity.  This chem. probe collection and assocd. data form a resource for the study of methylation-mediated signaling in epigenetics, inflammation and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA5rgeiWDvrVg90H21EOLACvtfcHk0lhP7ERJDPWnGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D&md5=008fec084426171f5c91657e54b73900</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%2BA.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DA%2520chemical%2520biology%2520toolbox%2520to%2520study%2520protein%2520methyltransferases%2520and%2520epigenetic%2520signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D19%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Bing Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Modulation: Dynamics is Double-“E”dged. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3694-3696. <a href="https://doi.org/10.1021/acs.jmedchem.1c00473" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00473</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00473%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAllosteric%252BModulation%25253A%252BDynamics%252Bis%252BDouble-%2525E2%252580%25259CE%2525E2%252580%25259Ddged%26aulast%3DXiong%26aufirst%3DBing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D15032021%26date%3D24032021%26volume%3D64%26issue%3D7%26spage%3D3694%26epage%3D3696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joshua J.  Hamey</span>, <span class="hlFld-ContribAuthor ">Sinja  Rakow</span>, <span class="hlFld-ContribAuthor ">Caroline  Bouchard</span>, <span class="hlFld-ContribAuthor ">Johanna M.  Senst</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>, <span class="hlFld-ContribAuthor ">Uta‐Maria  Bauer</span>, <span class="hlFld-ContribAuthor ">Marc R.  Wilkins</span>, <span class="hlFld-ContribAuthor ">Gene  Hart‐Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic investigation of PRMT6 substrate recognition reveals broad specificity with a preference for an RG motif or basic and bulky residues. </span><span class="cited-content_cbyCitation_journal-name">The FEBS Journal</span><span> <strong>2021,</strong> <em>20 </em><a href="https://doi.org/10.1111/febs.15837" title="DOI URL">https://doi.org/10.1111/febs.15837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.15837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.15837%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FEBS%2520Journal%26atitle%3DSystematic%252Binvestigation%252Bof%252BPRMT6%252Bsubstrate%252Brecognition%252Breveals%252Bbroad%252Bspecificity%252Bwith%252Ba%252Bpreference%252Bfor%252Ban%252BRG%252Bmotif%252Bor%252Bbasic%252Band%252Bbulky%252Bresidues%26aulast%3DHamey%26aufirst%3DJoshua%2BJ.%26date%3D2021%26date%3D2021%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. The structures and potencies of reported PRMT6 inhibitors</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Discovery of the PRMT6 chemical probe <i>(R)-</i><b>2</b>. (A) Structure–activity relationship studies and chiral separation led to the discovery of <i>(R)-</i><b>2</b> as a potent PRMT6 inhibitor and its enantiomer, <i>(S)-</i><b>2</b>, as a negative control. (B) IC<sub>50</sub> determination of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b>, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compound <i>(R)-</i><b>2</b> and Compound <i>(S)-</i><b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) dichloromethane, −78 °C to rt; (b) HCl, MeOH/H<sub>2</sub>O, reflux; (c) oxazolidine-2,5-dione, CF<sub>3</sub>COOH, triethylamine, toluene, 60–85 °C; (d) NaBH<sub>3</sub>CN, AcOH, MeOH, rt; (e) 5-bromothiophene-2-carboxylic acid, EDCI, HOAt, NMM, DMSO, rt; (f) chiral separation.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Selectivity assessment of <i>(R)-</i><b>2</b> (SGC6870) at 1 μM (gray) and 10 μM (black) and <i>(S)-</i><b>2</b> (SGC6870N) at 1 μM (green) and 10 μM (blue) against 8 PRMTs, 21 protein lysine methyltransferases (PKMTs), 3 DNA methyltransferases (DNMTs), and 1 RNA methyltransferase, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mechanism of action. (A) IC<sub>50</sub> values were determined at various compound-protein incubation times ranging from 15 to 120 min. The value obtained with no incubation time was used as a control (values are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf" class="ext-link">Table S3</a>). (B) Part of the plot in Figure A is magnified for better view of the decrease in the IC<sub>50</sub> value at longer incubation times. (C) IC<sub>50</sub> values of <i>(R)-</i><b>2</b> at increasing concentrations of SAM (up to 30 μM) at fixed concentration of peptide substrate (3 μM of H4 1–24 peptide). (D) IC<sub>50</sub> values of <i>(R)-</i><b>2</b> at increasing concentration of peptide (up to 8 μM) at fixed concentration of SAM (12 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of PRMT6 in complex with <i>(R)-</i><b>2</b>. <b>(</b>A) Cocrystal structure of PRMT6 (tinted blue) in complex with <i>(R)-</i><b>2</b> (green), and SAM (orange), (PDB: 6W6D). Dashed yellow lines indicate key hydrogen bonds. (B) Structural alignments of complexes PRMT6-<i>(R)-</i><b>2</b>-SAM (cyan), PRMT6-MS023-SAH (blue) (PDB: 5E8R) and PRMT6-SAH (magenta) (PDB: 4C05)<b>.</b> Movement of loop consisting of nine residues, EWMGYGLLH, indicates that <i>(R)-</i>2 binds to a novel induced allosteric pocket of PRMT6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of PRMT6-dependent H3R2 and H4R3 asymmetric di-methylation in cells. HEK293T cells were transfected with Flag-tagged PRMT6 and treated with indicated compounds for 20 h. The flag-tagged PRMT6 catalytically inactive mutant V86K/D88A (mut) was used as a positive control. (A) Western blot representation of the effect of <i>(R)-</i><b>2</b> on PRMT6 activity. (B) Western blot representation of the effect of <i>(S)-</i><b>2</b> on PRMT6 activity. (C) IC<sub>50</sub> determination of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> at reducing H3R2me2a. The graph represents nonlinear fit of H3R2me2a fluorescence intensities normalized to intensities of H3, <i>n</i> = 4. (D) IC<sub>50</sub> determination of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> at reducing H4R3me2a. The graph represents nonlinear fit of H4R3me2a fluorescence intensities normalized to intensities of H4, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/medium/jm0c02160_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibitory effect of <i>(R)-</i><b>2</b> and <i>(S)-</i><b>2</b> on cell viability (A) Effect of <i>(R)-</i><b>2</b> on MCF-7, PNT2, and HEK293T cell viability, <i>n</i> = 3. (B) Inhibitory effect of <i>(S)-</i><b>2</b> on MCF-7, PNT2, and HEK293T cell viability, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c02160/20210331/images/large/jm0c02160_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02160&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32088" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32088" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0liwQhwSFYDDiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+U.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0li63h6nuhPg1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0li63h6nuhPg1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNutt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span> <span> </span><span class="NLM_article-title">PTEN arginine methylation by PRMT6 suppresses PI3K-AKT signaling and modulates pre-mRNA splicing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6868</span>– <span class="NLM_lpage">6877</span>, <span class="refDoi"> DOI: 10.1073/pnas.1811028116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1073%2Fpnas.1811028116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=30886105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmsFeqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=6868-6877&author=J.+Fengauthor=Y.+Dangauthor=W.+Zhangauthor=X.+Zhaoauthor=C.+Zhangauthor=Z.+Houauthor=Y.+Jinauthor=M.+A.+McNuttauthor=A.+R.+Marksauthor=Y.+Yin&title=PTEN+arginine+methylation+by+PRMT6+suppresses+PI3K-AKT+signaling+and+modulates+pre-mRNA+splicing&doi=10.1073%2Fpnas.1811028116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PTEN arginine methylation by PRMT6 suppresses PI3K-AKT signaling and modulates pre-mRNA splicing</span></div><div class="casAuthors">Feng, Jiawen; Dang, Yaping; Zhang, Weiqi; Zhao, Xuyang; Zhang, Cong; Hou, Zhiyuan; Jin, Yan; McNutt, Michael A.; Marks, Andrew R.; Yin, Yuxin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6868-6877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Arginine methylation is a ubiquitous posttranslational modification that regulates crit. cellular processes including signal transduction and pre-mRNA splicing.  Here, we report that the tumor-suppressor PTEN is methylated by protein arginine methyltransferase 6 (PRMT6).  Mass-spectrometry anal. reveals that PTEN is dimethylated at arginine 159 (R159).  We found that PTEN is mutated at R159 in cancers, and the PTEN mutant R159K loses its capability to inhibit the PI3K-AKT cascade.  Furthermore, PRMT6 is phys. assocd. with PTEN, promotes asym. dimethylation of PTEN, and regulates the PI3K-AKT cascade through PTEN R159 methylation.  In addn., using transcriptome analyses, we found that PTEN R159 methylation is involved in modulation of pre-mRNA alternative splicing.  Our results demonstrate that PTEN is functionally regulated by arginine methylation.  We propose that PTEN arginine methylation modulates pre-mRNA alternative splicing and influences diverse physiol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoObdTCJWaRtLVg90H21EOLACvtfcHk0li63h6nuhPg1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmsFeqsLk%253D&md5=8c7dede72240882358a7fad8a661fadf</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1811028116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1811028116%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHou%26aufirst%3DZ.%26aulast%3DJin%26aufirst%3DY.%26aulast%3DMcNutt%26aufirst%3DM.%2BA.%26aulast%3DMarks%26aufirst%3DA.%2BR.%26aulast%3DYin%26aufirst%3DY.%26atitle%3DPTEN%2520arginine%2520methylation%2520by%2520PRMT6%2520suppresses%2520PI3K-AKT%2520signaling%2520and%2520modulates%2520pre-mRNA%2520splicing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D6868%26epage%3D6877%26doi%3D10.1073%2Fpnas.1811028116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0ljuDFjjNLygOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brame, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">Methylation of histone H4 at arginine 3 occurs in vivo and is mediated by the nuclear receptor coactivator PRMT1</span>. <i>Curr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">996</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1016/S0960-9822(01)00294-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2FS0960-9822%2801%2900294-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=11448779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltVGjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=996-1000&author=B.+D.+Strahlauthor=S.+D.+Briggsauthor=C.+J.+Brameauthor=J.+A.+Caldwellauthor=S.+S.+Kohauthor=H.+Maauthor=R.+G.+Cookauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=M.+R.+Stallcupauthor=C.+D.+Allis&title=Methylation+of+histone+H4+at+arginine+3+occurs+in+vivo+and+is+mediated+by+the+nuclear+receptor+coactivator+PRMT1&doi=10.1016%2FS0960-9822%2801%2900294-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H4 at arginine 3 occurs in vivo and is mediated by the nuclear receptor coactivator PRMT1</span></div><div class="casAuthors">Strahl, B. D.; Briggs, S. D.; Brame, C. J.; Caldwell, J. A.; Koh, S. S.; Ma, H.; Cook, R. G.; Shabanowitz, J.; Hunt, D. F.; Stallcup, M. R.; Allis, C. D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">996-1000</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone amino termini play an important role in modulating chromatin structure and function [1-3].  Lysine methylation of histones has been well documented [4, 5], and recently this modification has been linked to cellular processes involving gene transcription and heterochromatin assembly [6-9].  However, the existence of arginine methylation on histones has remained unclear.  Recent discoveries of protein arginine methyltransferases, CARM1 and PRMT1, as transcriptional coactivators for nuclear receptors suggest that histones may be physiol. targets of these enzymes as part of a poorly defined transcriptional activation pathway [10-12].  Here we show by using mass spectrometry that histone H4, isolated from asynchronously growing human 293T cells, is methylated at arginine 3 (Arg-3) in vivo.  In support, a novel antibody directed against histone H4 methylated at Arg-3 independently demonstrates the in vivo occurrence of this modification and reveals that H4 Arg-3 methylation is highly conserved throughout eukaryotes.  Finally, we show that PRMT1 is the major, if not exclusive, H4 Arg-3 methyltransfase in human 293T cells.  These findings suggest a role for arginine methylation of histones in the transcription process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOj66WOabZpLVg90H21EOLACvtfcHk0ljuDFjjNLygOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltVGjtbc%253D&md5=a3c9c1c279890250a315bb787226223f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2801%2900294-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252801%252900294-9%26sid%3Dliteratum%253Aachs%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBrame%26aufirst%3DC.%2BJ.%26aulast%3DCaldwell%26aufirst%3DJ.%2BA.%26aulast%3DKoh%26aufirst%3DS.%2BS.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DCook%26aufirst%3DR.%2BG.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMethylation%2520of%2520histone%2520H4%2520at%2520arginine%25203%2520occurs%2520in%2520vivo%2520and%2520is%2520mediated%2520by%2520the%2520nuclear%2520receptor%2520coactivator%2520PRMT1%26jtitle%3DCurr.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D996%26epage%3D1000%26doi%3D10.1016%2FS0960-9822%2801%2900294-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Girardot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirasawa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kacem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fritsch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kota, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipponi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbrizio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ait-Si-Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feil, R.</span></span> <span> </span><span class="NLM_article-title">PRMT5-mediated histone H4 arginine-3 symmetrical dimethylation marks chromatin at G + C-rich regions of the mouse genome</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">248</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1093%2Fnar%2Fgkt884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=24097435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvFGk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=235-248&author=M.+Girardotauthor=R.+Hirasawaauthor=S.+Kacemauthor=L.+Fritschauthor=J.+Pontisauthor=S.+K.+Kotaauthor=D.+Filipponiauthor=E.+Fabbrizioauthor=C.+Sardetauthor=F.+Lohmannauthor=S.+Kadamauthor=S.+Ait-Si-Aliauthor=R.+Feil&title=PRMT5-mediated+histone+H4+arginine-3+symmetrical+dimethylation+marks+chromatin+at+G+%2B+C-rich+regions+of+the+mouse+genome&doi=10.1093%2Fnar%2Fgkt884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5-mediated histone H4 arginine-3 symmetrical dimethylation marks chromatin at G + C-rich regions of the mouse genome</span></div><div class="casAuthors">Girardot, Michael; Hirasawa, Ryutaro; Kacem, Salim; Fritsch, Lauriane; Pontis, Julien; Kota, Satya K.; Filipponi, Doria; Fabbrizio, Eric; Sardet, Claude; Lohmann, Felix; Kadam, Shilpa; Ait-Si-Ali, Slimane; Feil, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-248</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Sym. dimethylation on arginine-3 of histone H4 (H4R3me2s) has been reported to occur at several repressed genes, but its specific regulation and genomic distribution remained unclear.  Here, we show that the type-II protein arginine methyltransferase PRMT5 controls H4R3me2s in mouse embryonic fibroblasts (MEFs).  In these differentiated cells, we find that the genome-wide pattern of H4R3me2s is highly similar to that in embryonic stem cells.  In both the cell types, H4R3me2s peaks are detected predominantly at G + C-rich regions.  Promoters are consistently marked by H4R3me2s, independently of transcriptional activity.  Remarkably, H4R3me2s is mono-allelic at imprinting control regions (ICRs), at which it marks the same parental allele as H3K9me3, H4K20me3 and DNA methylation.  These repressive chromatin modifications are regulated independently, however, since PRMT5-depletion in MEFs resulted in loss of H4R3me2s, without affecting H3K9me3, H4K20me3 or DNA methylation.  Conversely, depletion of ESET (KMT1E) or SUV420H1/H2 (KMT5B/C) affected H3K9me3 and H4K20me3, resp., without altering H4R3me2s at ICRs.  Combined, our data indicate that PRMT5-mediated H4R3me2s uniquely marks the mammalian genome, mostly at G + C-rich regions, and independently from transcriptional activity or chromatin repression.  Furthermore, comparative bioinformatics analyses suggest a putative role of PRMT5-mediated H4R3me2s in chromatin configuration in the nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgFVMfko7xHbVg90H21EOLACvtfcHk0li2HLArTTkojQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvFGk&md5=d5cff330abf484a6476d0aa8d0a8fe5e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt884%26sid%3Dliteratum%253Aachs%26aulast%3DGirardot%26aufirst%3DM.%26aulast%3DHirasawa%26aufirst%3DR.%26aulast%3DKacem%26aufirst%3DS.%26aulast%3DFritsch%26aufirst%3DL.%26aulast%3DPontis%26aufirst%3DJ.%26aulast%3DKota%26aufirst%3DS.%2BK.%26aulast%3DFilipponi%26aufirst%3DD.%26aulast%3DFabbrizio%26aufirst%3DE.%26aulast%3DSardet%26aufirst%3DC.%26aulast%3DLohmann%26aufirst%3DF.%26aulast%3DKadam%26aufirst%3DS.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DFeil%26aufirst%3DR.%26atitle%3DPRMT5-mediated%2520histone%2520H4%2520arginine-3%2520symmetrical%2520dimethylation%2520marks%2520chromatin%2520at%2520G%2520%252B%2520C-rich%2520regions%2520of%2520the%2520mouse%2520genome%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D235%26epage%3D248%26doi%3D10.1093%2Fnar%2Fgkt884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Lorenzo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Histone arginine methylation</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>585</i></span>,  <span class="NLM_fpage">2024</span>– <span class="NLM_lpage">2031</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2010.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.febslet.2010.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=21074527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFagtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2024-2031&author=A.+Di%0ALorenzoauthor=M.+T.+Bedford&title=Histone+arginine+methylation&doi=10.1016%2Fj.febslet.2010.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone arginine methylation</span></div><div class="casAuthors">Di Lorenzo, Alessandra; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">585</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2024-2031</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation is a common posttranslational modification (PTM).  This type of PTM occurs on both nuclear and cytoplasmic proteins, and is particularly abundant on shuttling proteins.  In this review, we will focus on one aspect of this PTM: the diverse roles that arginine methylation of the core histone tails play in regulating chromatin function.  A family of nine protein arginine methyltransferases (PRMTs) catalyze methylation reactions, and a subset target histones.  Importantly, arginine methylation of histone tails can promote or prevent the docking of key transcriptional effector mols., thus playing a central role in the orchestration of the histone code.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtVnjeIHcVcrVg90H21EOLACvtfcHk0li2HLArTTkojQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFagtbc%253D&md5=ab36ef630cd6ea1541ae6bca50e85685</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2010.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2010.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLorenzo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DHistone%2520arginine%2520methylation%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26spage%3D2024%26epage%3D2031%26doi%3D10.1016%2Fj.febslet.2010.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karkhanis, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbalzano, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sif, S.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferase 7 regulates cellular response to DNA damage by methylating promoter histones H2A and H4 of the polymerase delta catalytic subunit gene, POLD1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">29801</span>– <span class="NLM_lpage">29814</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.378281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1074%2Fjbc.M112.378281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=22761421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1CgtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=29801-29814&author=V.+Karkhanisauthor=L.+Wangauthor=S.+Taeauthor=Y.+J.+Huauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Protein+arginine+methyltransferase+7+regulates+cellular+response+to+DNA+damage+by+methylating+promoter+histones+H2A+and+H4+of+the+polymerase+delta+catalytic+subunit+gene%2C+POLD1&doi=10.1074%2Fjbc.M112.378281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 7 regulates cellular response to DNA damage by methylating promoter histones H2A and H4 of the polymerase δ catalytic subunit gene, POLD1</span></div><div class="casAuthors">Karkhanis, Vrajesh; Wang, Li; Tae, Sookil; Hu, Yu-Jie; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">29801-29814</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Covalent modification of histones by protein arginine methyltransferases (PRMTs) impacts genome organization and gene expression.  In this report, we show that PRMT7 interacts with the BRG1-based hSWI/SNF chromatin remodeling complex and specifically methylates histone H2A Arg-3 (H2AR3) and histone H4 Arg-3 (H4R3).  To elucidate the biol. function of PRMT7, we knocked down its expression in NIH 3T3 cells and analyzed global gene expression.  Our findings show that PRMT7 neg. regulates expression of genes involved in DNA repair, including ALKBH5, APEX2, POLD1, and POLD2.  Chromatin immunopptn. (ChIP) revealed that PRMT7 and dimethylated H2AR3 and H4R3 are enriched at target DNA repair genes in parental cells, whereas PRMT7 knockdown caused a significant decrease in PRMT7 recruitment and H2AR3/H4R3 methylation.  Decreased PRMT7 expression also resulted in derepression of target DNA repair genes and enhanced cell resistance to DNA-damaging agents.  Furthermore, we show that BRG1 co-localizes with PRMT7 on target promoters and that expression of a catalytically inactive form of BRG1 results in derepression of PRMT7 target DNA repair genes.  Remarkably, reducing expression of individual PRMT7 target DNA repair genes showed that only the catalytic subunit of DNA polymerase, POLD1, was able to resensitize PRMT7 knock-down cells to DNA-damaging agents.  These results provide evidence for the important role played by PRMT7 in epigenetic regulation of DNA repair genes and cellular response to DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqANrVNaylqb7Vg90H21EOLACvtfcHk0li2HLArTTkojQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1CgtrrE&md5=4f7305ced8d1fe2407ec49e2c658511c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.378281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.378281%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DTae%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%2BJ.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25207%2520regulates%2520cellular%2520response%2520to%2520DNA%2520damage%2520by%2520methylating%2520promoter%2520histones%2520H2A%2520and%2520H4%2520of%2520the%2520polymerase%2520delta%2520catalytic%2520subunit%2520gene%252C%2520POLD1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D29801%26epage%3D29814%26doi%3D10.1074%2Fjbc.M112.378281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hakala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weintraub, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiio, Y.</span></span> <span> </span><span class="NLM_article-title">Proteomic dissection of the von Hippel-Lindau (VHL) interactome</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5175</span>– <span class="NLM_lpage">5182</span>, <span class="refDoi"> DOI: 10.1021/pr200642c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr200642c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=5175-5182&author=Y.+Laiauthor=M.+Songauthor=K.+Hakalaauthor=S.+T.+Weintraubauthor=Y.+Shiio&title=Proteomic+dissection+of+the+von+Hippel-Lindau+%28VHL%29+interactome&doi=10.1021%2Fpr200642c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic Dissection of the von Hippel-Lindau (VHL) Interactome</span></div><div class="casAuthors">Lai, Yanlai; Song, Meihua; Hakala, Kevin; Weintraub, Susan T.; Shiio, Yuzuru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5175-5182</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) tumor suppressor gene encodes a component of a ubiquitin ligase complex contg. elongin B, elongin C, cullin 2, and Rbx1, which acts as a neg. regulator of hypoxia inducible factor (HIF).  VHL ubiquitinates and degrades the alpha subunits of HIF, and this is proposed to suppress tumorigenesis and tumor angiogenesis.  Several lines of evidence also suggest important roles for HIF-independent VHL functions in the maintenance of primary cilium, extracellular matrix formation, and tumor suppression.  We undertook a series of proteomic analyses to gain a comprehensive picture of the VHL-interacting proteins.  We found that the ARF tumor suppressor interacts with VHL30, a longer VHL isoform, but not with VHL19, a shorter VHL isoform.  ARF was found to release VHL30 from the E3 ligase complex, promoting the binding of VHL30 to a protein arginine methyltransferase, PRMT3.  Our anal. of the VHL19 interactome also uncovered that VHL19 displays an affinity to collagens and their biosynthesis enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqTYZ6RvZ7erVg90H21EOLACvtfcHk0li2HLArTTkojQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12jsbzL&md5=51500af3ec4256311dbd835c47a90200</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fpr200642c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr200642c%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DHakala%26aufirst%3DK.%26aulast%3DWeintraub%26aufirst%3DS.%2BT.%26aulast%3DShiio%26aufirst%3DY.%26atitle%3DProteomic%2520dissection%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520interactome%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2011%26volume%3D10%26spage%3D5175%26epage%3D5182%26doi%3D10.1021%2Fpr200642c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chitnis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsugoe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein-Szanto, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rustgi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, J. A.</span></span> <span> </span><span class="NLM_article-title">PRMT5 is required for lymphomagenesis triggered by multiple oncogenic drivers</span>. <i>Cancer Discov.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">303</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-14-0625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1158%2F2159-8290.CD-14-0625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=25582697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVGktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=288-303&author=Y.+Liauthor=N.+Chitnisauthor=H.+Nakagawaauthor=Y.+Kitaauthor=S.+Natsugoeauthor=Y.+Yangauthor=Z.+Liauthor=M.+Wasikauthor=A.+J.+Klein-Szantoauthor=A.+K.+Rustgiauthor=J.+A.+Diehl&title=PRMT5+is+required+for+lymphomagenesis+triggered+by+multiple+oncogenic+drivers&doi=10.1158%2F2159-8290.CD-14-0625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 Is Required for Lymphomagenesis Triggered by Multiple Oncogenic Drivers</span></div><div class="casAuthors">Li, Yan; Chitnis, Nilesh; Nakagawa, Hiroshi; Kita, Yoshiaki; Natsugoe, Shoji; Yang, Yi; Li, Zihai; Wasik, Mariusz; Klein-Szanto, Andres J. P.; Rustgi, Anil K.; Diehl, J. Alan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-303</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PRMT5 cooperates with oncogenic drivers such as cyclin D1 to promote lymphomagenesis via p53 methylation, which alters p53 chromatin occupancy and inhibits proapoptotic p53 target genes.  Protein arginine methyltransferase 5 (PRMT5) has been implicated as a key modulator of lymphomagenesis.  Whether PRMT5 has overt oncogenic function in the context of leukemia/lymphoma and whether it represents a therapeutic target remains to be established.  We demonstrate that inactivation of PRMT5 inhibits colony-forming activity by multiple oncogenic drivers, including cyclin D1, c-MYC, NOTCH1, and MLL-AF9.  Furthermore, we demonstrate that PRMT5 overexpression specifically cooperates with cyclin D1 to drive lymphomagenesis in a mouse model, revealing inherent neoplastic activity.  Mol. anal. of lymphomas revealed that arginine methylation of p53 selectively suppresses expression of crucial proapoptotic and antiproliferative target genes, thereby sustaining tumor cell self-renewal and proliferation and bypassing the need for the acquisition of inactivating p53 mutations.  Critically, anal. of human tumor specimens reveals a strong correlation between cyclin D1 overexpression and p53 methylation, supporting the biomedical relevance of this pathway.  Significance: We have identified and functionally validated a crucial role for PRMT5 for the inhibition of p53-dependent tumor suppression in response to oncogenic insults.  The requisite role for PRMT5 in the context of multiple lymphoma/leukemia oncogenic drivers suggests a mol. rationale for therapeutic development.  Cancer Discov; 5(3); 288-303. ©2015 AACR.  This article is highlighted in the In This Issue feature, p.  213.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCRiYvtZPoW7Vg90H21EOLACvtfcHk0lgXK8Plwx4xSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVGktb8%253D&md5=c2d346e514c484969af530af53310350</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0625%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChitnis%26aufirst%3DN.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DKita%26aufirst%3DY.%26aulast%3DNatsugoe%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWasik%26aufirst%3DM.%26aulast%3DKlein-Szanto%26aufirst%3DA.%2BJ.%26aulast%3DRustgi%26aufirst%3DA.%2BK.%26aulast%3DDiehl%26aufirst%3DJ.%2BA.%26atitle%3DPRMT5%2520is%2520required%2520for%2520lymphomagenesis%2520triggered%2520by%2520multiple%2520oncogenic%2520drivers%26jtitle%3DCancer%2520Discov.%26date%3D2015%26volume%3D5%26spage%3D288%26epage%3D303%26doi%3D10.1158%2F2159-8290.CD-14-0625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Migliori, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phalke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaratne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capasso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackstock, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span> <span> </span><span class="NLM_article-title">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fnsmb.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=22231400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=136-144&author=V.+Miglioriauthor=J.+Mullerauthor=S.+Phalkeauthor=D.+Lowauthor=M.+Bezziauthor=W.+C.+Mokauthor=S.+K.+Sahuauthor=J.+Gunaratneauthor=P.+Capassoauthor=C.+Bassiauthor=V.+Cecatielloauthor=A.+De+Marcoauthor=W.+Blackstockauthor=V.+Kuznetsovauthor=B.+Amatiauthor=M.+Mapelliauthor=E.+Guccione&title=Symmetric+dimethylation+of+H3R2+is+a+newly+identified+histone+mark+that+supports+euchromatin+maintenance&doi=10.1038%2Fnsmb.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Symmetric dimethylation of H3R2 is a newly identified histone mark that supports euchromatin maintenance</span></div><div class="casAuthors">Migliori, Valentina; Muller, Julius; Phalke, Sameer; Low, Diana; Bezzi, Marco; Mok, Wei Chuen; Sahu, Sanjeeb Kumar; Gunaratne, Jayantha; Capasso, Paola; Bassi, Christian; Cecatiello, Valentina; De Marco, Ario; Blackstock, Walter; Kuznetsov, Vladimir; Amati, Bruno; Mapelli, Marina; Guccione, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-144</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The asym. dimethylation of histone H3 arginine 2 (H3R2me2a) acts as a repressive mark that antagonizes trimethylation of H3 lysine 4.  Here we report that H3R2 is also sym. dimethylated (H3R2me2s) by PRMT5 and PRMT7 and present in euchromatic regions.  Profiling of H3-tail interactors by SILAC MS revealed that H3R2me2s excludes binding of RBBP7, a central component of co-repressor complexes Sin3a, NURD and PRC2.  Conversely H3R2me2s enhances binding of WDR5, a common component of the coactivator complexes MLL, SET1A, SET1B, NLS1 and ATAC.  The interaction of histone H3 with WDR5 distinguishes H3R2me2s from H3R2me2a, which impedes the recruitment of WDR5 to chromatin.  The crystallog. structure of WDR5 and the H3R2me2s peptide elucidates the mol. determinants of this high affinity interaction.  Our findings identify H3R2me2s as a previously unknown mark that keeps genes poised in euchromatin for transcriptional activation upon cell-cycle withdrawal and differentiation in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDe73jpv41ObVg90H21EOLACvtfcHk0lgXK8Plwx4xSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1ShtA%253D%253D&md5=c7b15ed7ee8f4819b45d488171e89ac4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2209%26sid%3Dliteratum%253Aachs%26aulast%3DMigliori%26aufirst%3DV.%26aulast%3DMuller%26aufirst%3DJ.%26aulast%3DPhalke%26aufirst%3DS.%26aulast%3DLow%26aufirst%3DD.%26aulast%3DBezzi%26aufirst%3DM.%26aulast%3DMok%26aufirst%3DW.%2BC.%26aulast%3DSahu%26aufirst%3DS.%2BK.%26aulast%3DGunaratne%26aufirst%3DJ.%26aulast%3DCapasso%26aufirst%3DP.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DDe%2BMarco%26aufirst%3DA.%26aulast%3DBlackstock%26aufirst%3DW.%26aulast%3DKuznetsov%26aufirst%3DV.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DMapelli%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26atitle%3DSymmetric%2520dimethylation%2520of%2520H3R2%2520is%2520a%2520newly%2520identified%2520histone%2520mark%2520that%2520supports%2520euchromatin%2520maintenance%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D136%26epage%3D144%26doi%3D10.1038%2Fnsmb.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Vedadi, M. A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span>. <i>Angew. Chem. Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5166</span>– <span class="NLM_lpage">5170</span>, <span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=25728001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A280%3ADC%252BC2MrpvVCjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+U.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+Yuauthor=M.+S.+Eramauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+Liauthor=E.+Dobrovetskyauthor=A.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+Zhaoauthor=J.+Jin&title=Vedadi%2C+M.+A+potent%2C+selective+and+cell-active+allosteric+inhibitor+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan H Umit; Szewczyk Magdalena M; Yu Zhengtian; Eram Mohammad S; Yang Xiaobao; Schmidt Keith; Luo Xiao; Dai Miao; He Feng; Zang Irene; Lin Ying; Kennedy Steven; Li Fengling; Dobrovetsky Elena; Dong Aiping; Smil David; Min Sun-Joon; Landon Melissa; Lin-Jones Jennifer; Huang Xi-Ping; Roth Bryan L; Schapira Matthieu; Atadja Peter; Barsyte-Lovejoy Dalia; Arrowsmith Cheryl H; Brown Peter J; Zhao Kehao; Jin Jian; Vedadi Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PRMT3 catalyzes the asymmetric dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biology.  Here we report the discovery of the first PRMT3 chemical probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochemical, biophysical, and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50 =31±2 nM, KD =53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chemical probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTrgkSPlfW-0Ptos8R3JUhXfW6udTcc2ea5E93C4UTn4rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MrpvVCjuw%253D%253D&md5=6a72abd6ed4b82dbc4187a51f6d75157</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DVedadi%252C%2520M.%2520A%2520potent%252C%2520selective%2520and%2520cell-active%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%2520Engl.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela
Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunitomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cary, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span> <span> </span><span class="NLM_article-title">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18480</span>– <span class="NLM_lpage">18493</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.18632%2Foncotarget.24883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=29719619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=18480-18493&author=K.+Nakayamaauthor=M.+M.+Szewczykauthor=C.+Dela%0ASenaauthor=H.+Wuauthor=A.+Dongauthor=H.+Zengauthor=F.+Liauthor=R.+F.+de+Freitasauthor=M.+S.+Eramauthor=M.+Schapiraauthor=Y.+Babaauthor=M.+Kunitomoauthor=D.+R.+Caryauthor=M.+Tawadaauthor=A.+Ohashiauthor=Y.+Imaedaauthor=K.+S.+Saikatenduauthor=C.+E.+Grimshawauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=A.+Kibaauthor=D.+Tomitaauthor=P.+J.+Brown&title=TP-064%2C+a+potent+and+selective+small+molecule+inhibitor+of+PRMT4+for+multiple+myeloma&doi=10.18632%2Foncotarget.24883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span></div><div class="casAuthors">Nakayama Kazuhide; Baba Yuji; Kunitomo Mihoko; Cary Douglas R; Ohashi Akihiro; Imaeda Yasuhiro; Kiba Atsushi; Tomita Daisuke; Szewczyk Magdalena M; Dela Sena Carlo; Wu Hong; Dong Aiping; Zeng Hong; Li Fengling; de Freitas Renato Ferreira; Eram Mohammad S; Schapira Matthieu; Vedadi Masoud; Arrowsmith Cheryl H; Barsyte-Lovejoy Dalia; Brown Peter J; Schapira Matthieu; Vedadi Masoud; Tawada Michiko; Saikatendu Kumar Singh; Grimshaw Charles E; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">18480-18493</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methyltransferase (PRMT) 4 (also known as coactivator-associated arginine methyltransferase 1; CARM1) is involved in a variety of biological processes and is considered as a candidate oncogene owing to its overexpression in several types of cancer.  Selective PRMT4 inhibitors are useful tools for clarifying the molecular events regulated by PRMT4 and for validating PRMT4 as a therapeutic target.  Here, we report the discovery of TP-064, a potent, selective, and cell-active chemical probe of human PRMT4 and its co-crystal structure with PRMT4.  TP-064 inhibited the methyltransferase activity of PRMT4 with high potency (half-maximal inhibitory concentration, IC50 < 10 nM) and selectivity over other PRMT family proteins, and reduced arginine dimethylation of the PRMT4 substrates BRG1-associated factor 155 (BAF155; IC50= 340 ± 30 nM) and Mediator complex subunit 12 (MED12; IC50 = 43 ± 10 nM).  TP-064 treatment inhibited the proliferation of a subset of multiple myeloma cell lines, with affected cells arrested in G1 phase of the cell cycle.  TP-064 and its negative control (TP-064N) will be valuable tools to further investigate the biology of PRMT4 and the therapeutic potential of PRMT4 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTsNG0-q3aOlw20zIbnCCI3fW6udTcc2eYo6t8Lh4RDWrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D&md5=f910a11dc3aa023564499ba2e6209e31</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24883%26sid%3Dliteratum%253Aachs%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3Dde%2BFreitas%26aufirst%3DR.%2BF.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBaba%26aufirst%3DY.%26aulast%3DKunitomo%26aufirst%3DM.%26aulast%3DCary%26aufirst%3DD.%2BR.%26aulast%3DTawada%26aufirst%3DM.%26aulast%3DOhashi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DKiba%26aufirst%3DA.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DTP-064%252C%2520a%2520potent%2520and%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520PRMT4%2520for%2520multiple%2520myeloma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D18480%26epage%3D18493%26doi%3D10.18632%2Foncotarget.24883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela Sena, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivcharuk, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibanez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazutis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">A chemical probe of CARM1 alters epigenetic plasticity against breast cancer cell invasion</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e47110</span> <span class="refDoi"> DOI: 10.7554/eLife.47110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.7554%2FeLife.47110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=31657716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlalt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&author=X.+C.+Caiauthor=T.+Zhangauthor=E.+J.+Kimauthor=M.+Jiangauthor=K.+Wangauthor=J.+Wangauthor=S.+Chenauthor=N.+Zhangauthor=H.+Wuauthor=F.+Liauthor=C.+C.+Dela+Senaauthor=H.+Zengauthor=V.+Vivcharukauthor=X.+Niuauthor=W.+Zhengauthor=J.+P.+Leeauthor=Y.+Chenauthor=D.+Barsyteauthor=M.+Szewczykauthor=T.+Hajianauthor=G.+Ibanezauthor=A.+Dongauthor=L.+Dombrovskiauthor=Z.+Zhangauthor=H.+Dengauthor=J.+Minauthor=C.+H.+Arrowsmithauthor=L.+Mazutisauthor=L.+Shiauthor=M.+Vedadiauthor=P.+J.+Brownauthor=J.+Xiangauthor=L.+X.+Qinauthor=W.+Xuauthor=M.+Luo&title=A+chemical+probe+of+CARM1+alters+epigenetic+plasticity+against+breast+cancer+cell+invasion&doi=10.7554%2FeLife.47110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe of CARM1 alters epigenetic plasticity against breast cancer cell invasion</span></div><div class="casAuthors">Cai, Xiao-Chuan; Zhang, Tuo; Kim, Eui-jun; Jiang, Ming; Wang, Ke; Wang, Junyi; Chen, Shi; Zhang, Nawei; Wu, Hong; Li, Fengling; Dela Sena, Carlo C.; Zeng, Hong; Vivcharuk, Victor; Niu, Xiang; Zheng, Weihong; Lee, Jonghan P.; Chen, Yuling; Barsyte, Dalia; Szewczyk, Magda; Hajian, Taraneh; Ibanez, Glorymar; Dong, Aiping; Dombrovski, Ludmila; Zhang, Zhenyu; Deng, Haiteng; Min, Jinrong; Arrowsmith, Cheryl H.; Mazutis, Linas; Shi, Lei; Vedadi, Masoud; Brown, Peter J.; Xiang, Jenny; Qin, Li-Xuan; Xu, Wei; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e47110/1-e47110/42</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">CARM1 is a cancer-relevant protein arginine methyltransferase that regulates many aspects of transcription.  Its pharmacol. inhibition is a promising anti-cancer strategy.  Here SKI-73 (6a in this work) is presented as a CARM1 chem. probe with pro-drug properties.  SKI-73 (6a) can rapidly penetrate cell membranes and then be processed into active inhibitors, which are retained intracellularly with 10-fold enrichment for several days.  These compds. were characterized for their potency, selectivity, modes of action, and on-target engagement.  SKI-73 (6a) recapitulates the effect of CARM1 knockout against breast cancer cell invasion.  Single-cell RNA-seq anal. revealed that the SKI-73(6a)-assocd. redn. of invasiveness acts by altering epigenetic plasticity and suppressing the invasion-prone subpopulation.  Interestingly, SKI-73 (6a) and CARM1 knockout alter the epigenetic plasticity with remarkable difference, suggesting distinct modes of action for small-mol. and genetic perturbations.  We therefore discovered a CARM1-addiction mechanism of cancer metastasis and developed a chem. probe to target this process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpArjVrLFUaObVg90H21EOLACvtfcHk0ljx-Cz9pjXJhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlalt7jF&md5=644ad9780e1504c8793e3ae5195b5df1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.7554%2FeLife.47110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.47110%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DX.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DE.%2BJ.%26aulast%3DJiang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDela%2BSena%26aufirst%3DC.%2BC.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DVivcharuk%26aufirst%3DV.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DJ.%2BP.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBarsyte%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DIbanez%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DMazutis%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DL.%2BX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DA%2520chemical%2520probe%2520of%2520CARM1%2520alters%2520epigenetic%2520plasticity%2520against%2520breast%2520cancer%2520cell%2520invasion%26jtitle%3DElife%26date%3D2019%26volume%3D8%26doi%3D10.7554%2FeLife.47110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0ljWODMbBXwKaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective covalent protein arginine methyltransferase 5 (PRMT5) inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1033</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1033-1038&author=H.+Linauthor=M.+Wangauthor=Y.+W.+Zhangauthor=S.+Tongauthor=R.+A.+Lealauthor=R.+Shettyauthor=K.+Vaddiauthor=J.+I.+Luengo&title=Discovery+of+potent+and+selective+covalent+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitors&doi=10.1021%2Facsmedchemlett.9b00074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Covalent Protein Arginine Methyltransferase 5 (PRMT5) Inhibitors</span></div><div class="casAuthors">Lin Hong; Wang Min; Zhang Yang W; Leal Raul A; Shetty Rupa; Vaddi Kris; Luengo Juan I; Tong Shuilong</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1033-1038</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is known to symmetrically dimethylate numerous cytosolic and nuclear proteins that are involved in a variety of cellular processes.  Recent findings have revealed its potential as a cancer therapeutic target.  PRMT5 possesses a cysteine (C449) in the active site, unique to PRMT5.  Therefore, covalent PRMT5 inhibition is an attractive chemical approach.  Herein, we report an exciting discovery of a series of novel hemiaminals that under physiological conditions can be converted to aldehydes and react with C449 to form covalent adducts, which presumably undergo an unprecedented elimination to form the thiol-vinyl ethers, as indicated by electron density in the co-crystal structure of the PRMT5/MEP50 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdNDVSKB6iezW1zwg2yUK5fW6udTcc2eYTMkLqZpogGbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzotVeltQ%253D%253D&md5=4b5b08dc3e3615144ece564f70bf3bd0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00074%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%2BW.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DLeal%26aufirst%3DR.%2BA.%26aulast%3DShetty%26aufirst%3DR.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520covalent%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1033%26epage%3D1038%26doi%3D10.1021%2Facsmedchemlett.9b00074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a potent and selective inhibitor of arginine methyltransferase 5, PRMT5, with antitumor activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+potent+and+selective+inhibitor+of+arginine+methyltransferase+5%2C+PRMT5%2C+with+antitumor+activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 ± 3 nM and 25 ± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0ljWODMbBXwKaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520arginine%2520methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520antitumor%2520activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Potent, selective, and cell active protein arginine methyltransferase 5 (PRMT5) inhibitor developed by structure-based virtual screening and hit optimization</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6289</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6289-6304&author=R.+Maoauthor=J.+Shaoauthor=K.+Zhuauthor=Y.+Zhangauthor=H.+Dingauthor=C.+Zhangauthor=Z.+Shiauthor=H.+Jiangauthor=D.+Sunauthor=W.+Duanauthor=C.+Luo&title=Potent%2C+selective%2C+and+cell+active+protein+arginine+methyltransferase+5+%28PRMT5%29+inhibitor+developed+by+structure-based+virtual+screening+and+hit+optimization&doi=10.1021%2Facs.jmedchem.7b00587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></div><div class="casAuthors">Mao, Ruifeng; Shao, Jingwei; Zhu, Kongkai; Zhang, Yuanyuan; Ding, Hong; Zhang, Chenhua; Shi, Zhe; Jiang, Hualiang; Sun, Dequn; Duan, Wenhu; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6289-6304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr. PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies.  Besides, PRMT5 has been validated as anticancer target in mantle cell lymphoma.  In this study, the authors found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization.  The identified compd. I (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases.  The direct binding of I to PRMT5 was validated by surface plasmon resonance expts., with a Kd of 0.987 μM.  Kinetic expts. indicated that I was a SAM competitive inhibitor other than substrate.  In addn., I showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to inhibition of PRMT5 mediated SmD3 methylation. I may represent a promising lead compd. to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI1jnM_kEhRrVg90H21EOLACvtfcHk0lj9haVeydrM1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitr7E&md5=5e3687166e2e6b5558409806e3c4b9e5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DR.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DDuan%26aufirst%3DW.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPotent%252C%2520selective%252C%2520and%2520cell%2520active%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibitor%2520developed%2520by%2520structure-based%2520virtual%2520screening%2520and%2520hit%2520optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6289%26epage%3D6304%26doi%3D10.1021%2Facs.jmedchem.7b00587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Bryant, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181601</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1371%2Fjournal.pone.0181601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=28806746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=G.+M.+Kongauthor=M.+Yuauthor=Z.+Guauthor=Z.+Chenauthor=R.+M.+Xuauthor=D.+O%E2%80%99Bryantauthor=Z.+Wang&title=Selective+small-chemical+inhibitors+of+protein+arginine+methyltransferase+5+with+anti-lung+cancer+activity&doi=10.1371%2Fjournal.pone.0181601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-chemical inhibitors of protein arginine methyltransferase 5 with anti-lung cancer activity</span></div><div class="casAuthors">Kong, Gui-Mei; Yu, Min; Gu, Zhongping; Chen, Zhi; Xu, Rui-Ming; O'Bryant, Deon; Wang, Zhengxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181601/1-e0181601/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) plays crit. roles in a wide variety of biol. processes, including tumorigenesis.  By screening a library of small chem. compds., we identified eight compds. that selectively inhibit the PRMT5 enzymic activity, with IC50 values ranging from 0.1 to 6 μM.  Mol. docking simulation and site directed mutagenesis indicated that identified compds. target the substrate-binding site in PRMT5.  Treatment of lung cancer cells with identified inhibitors led to inhibition of the sym. arginine methylation of SmD3 and histones and the cellular proliferation.  Oral administration of the inhibitor demonstrated antitumor activity in a lung tumor xenograft model.  Thus, identified PRMT5-specific small-mol. inhibitors would help elucidate the biol. roles of PRMT5 and serve as lead compds. for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrmbn8I9YMLVg90H21EOLACvtfcHk0lj9haVeydrM1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2rtbk%253D&md5=0306ed1351aac8a04d286df255c6e432</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181601%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DG.%2BM.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DO%25E2%2580%2599Bryant%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DSelective%2520small-chemical%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520with%2520anti-lung%2520cancer%2520activity%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Organ, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukushi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seña, C. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takizawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of PRMT7 links arginine monomethylation to the cellular stress response</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">2396</span> <span class="refDoi"> DOI: 10.1038/s41467-020-16271-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fs41467-020-16271-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=32409666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXps1OltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=M.+M.+Szewczykauthor=Y.+Ishikawaauthor=S.+Organauthor=N.+Sakaiauthor=F.+Liauthor=S.+Acklooauthor=M.+Eramauthor=D.+Dilworthauthor=H.+Fukushiauthor=R.+Hardingauthor=C.+C.+D.+Se%C3%B1aauthor=T.+Sugoauthor=K.+Hayashiauthor=D.+Macleodauthor=C.+Zepedaauthor=S.+Takagiauthor=R.+Al-Awarauthor=S.+Richardauthor=M.+Takizawaauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=P.+J.+Brownauthor=H.+Naraauthor=D.+Barsyte-Lovejoy&title=Pharmacological+inhibition+of+PRMT7+links+arginine+monomethylation+to+the+cellular+stress+response&doi=10.1038%2Fs41467-020-16271-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of PRMT7 links arginine monomethylation to the cellular stress response</span></div><div class="casAuthors">Szewczyk, Magdalena M.; Ishikawa, Yoshinori; Organ, Shawna; Sakai, Nozomu; Li, Fengling; Halabelian, Levon; Ackloo, Suzanne; Couzens, Amber L.; Eram, Mohammad; Dilworth, David; Fukushi, Hideto; Harding, Rachel; dela Sena, Carlo C.; Sugo, Tsukasa; Hayashi, Kozo; McLeod, David; Zepeda, Carlos; Aman, Ahmed; Sanchez-Osuna, Maria; Bonneil, Eric; Takagi, Shinji; Al-Awar, Rima; Tyers, Mike; Richard, Stephane; Takizawa, Masayuki; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Vedadi, Masoud; Brown, Peter J.; Nara, Hiroshi; Barsyte-Lovejoy, Dalia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2396</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Protein arginine methyltransferases (PRMTs) regulate diverse biol. processes and are increasingly being recognized for their potential as drug targets.  Here we report the discovery of a potent, selective, and cell-active chem. probe for PRMT7.  SGC3027 is a cell permeable prodrug, which in cells is converted to SGC8158, a potent, SAM-competitive PRMT7 inhibitor.  Inhibition or knockout of cellular PRMT7 results in drastically reduced levels of arginine monomethylated HSP70 family stress-assocd. proteins.  Structural and biochem. analyses reveal that PRMT7-driven in vitro methylation of HSP70 at R469 requires an ATP-bound, open conformation of HSP70.  In cells, SGC3027 inhibits methylation of both constitutive and inducible forms of HSP70, and leads to decreased tolerance for perturbations of proteostasis including heat shock and proteasome inhibitors.  These results demonstrate a role for PRMT7 and arginine methylation in stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rIt3-Hwdr7Vg90H21EOLACvtfcHk0lixl_QM6CDCSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXps1OltLk%253D&md5=337393269b76b0373d467ff6e4108f08</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16271-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16271-z%26sid%3Dliteratum%253Aachs%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DOrgan%26aufirst%3DS.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DEram%26aufirst%3DM.%26aulast%3DDilworth%26aufirst%3DD.%26aulast%3DFukushi%26aufirst%3DH.%26aulast%3DHarding%26aufirst%3DR.%26aulast%3DSe%25C3%25B1a%26aufirst%3DC.%2BC.%2BD.%26aulast%3DSugo%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DK.%26aulast%3DMacleod%26aufirst%3DD.%26aulast%3DZepeda%26aufirst%3DC.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DTakizawa%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26atitle%3DPharmacological%2520inhibition%2520of%2520PRMT7%2520links%2520arginine%2520monomethylation%2520to%2520the%2520cellular%2520stress%2520response%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-16271-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">3537</span>– <span class="NLM_lpage">3543</span>, <span class="refDoi"> DOI: 10.1074/jbc.M108786200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1074%2Fjbc.M108786200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=11724789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=3537-3543&author=A.+Frankelauthor=N.+Yadavauthor=J.+Leeauthor=T.+L.+Branscombeauthor=S.+Clarkeauthor=M.+T.+Bedford&title=The+novel+human+protein+arginine+N-methyltransferase+PRMT6+is+a+nuclear+enzyme+displaying+unique+substrate+specificity&doi=10.1074%2Fjbc.M108786200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span></div><div class="casAuthors">Frankel, Adam; Yadav, Neelu; Lee, Jaeho; Branscombe, Tina L.; Clarke, Steven; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3537-3543</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein arginine methylation is a prevalent posttranslational modification in eukaryotic cells that has been implicated in signal transduction, the metab. of nascent pre-RNA, and the transcriptional activation processes.  In searching the human genome for protein arginine N-methyltransferase (PRMT) family members, a novel gene has been found on chromosome 1 that encodes for an apparent methyltransferase, PRMT6.  The polypeptide chain of PRMT6 is 41.9 kDa consisting of a catalytic core sequence common to other PRMT enzymes.  Expressed as a glutathione S-transferase fusion protein, PRMT6 demonstrates type I PRMT activity, capable of forming both ω-NG-monomethylarginine and asym. ω-NG,NG-dimethylarginine derivs. on the recombinant glycine- and arginine-rich substrate in a processive manner with a specific activity of 144 pmol Me groups transferred min-1 mg-1 enzyme.  A comparison of substrate specificity reveals that PRMT6 is functionally distinct from two previously characterized type I enzymes, PRMT1 and PRMT4.  In addn., PRMT6 displays automethylation activity; it is the first PRMT to do so.  This novel human PRMT, which resides solely in the nucleus when fused to the green fluorescent protein, joins a family of enzymes with diverse functions within cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_K0kYqP2NbVg90H21EOLACvtfcHk0lixl_QM6CDCSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D&md5=1faaf27f44d72742749bdee2998ec9c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M108786200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M108786200%26sid%3Dliteratum%253Aachs%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520novel%2520human%2520protein%2520arginine%2520N-methyltransferase%2520PRMT6%2520is%2520a%2520nuclear%2520enzyme%2520displaying%2520unique%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D3537%26epage%3D3543%26doi%3D10.1074%2Fjbc.M108786200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lorenzo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastian, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Mouse models of overexpression reveal distinct oncogenic roles for different type I protein arginine methyltransferases</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-18-1995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1158%2F0008-5472.CAN-18-1995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=30352814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotFGhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=21-32&author=J.+Baoauthor=A.+Di+Lorenzoauthor=K.+Linauthor=Y.+Luauthor=Y.+Zhongauthor=M.+M.+Sebastianauthor=W.+J.+Mullerauthor=Y.+Yangauthor=M.+T.+Bedford&title=Mouse+models+of+overexpression+reveal+distinct+oncogenic+roles+for+different+type+I+protein+arginine+methyltransferases&doi=10.1158%2F0008-5472.CAN-18-1995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse models of overexpression reveal distinct oncogenic roles for different type i protein arginine methyltransferases</span></div><div class="casAuthors">Bao, Jianqiang; Di Lorenzo, Alessandra; Lin, Kevin; Lu, Yue; Zhong, Yi; Sebastian, Manu M.; Muller, William J.; Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-32</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMT) are generally not mutated in diseased states, but they are overexpressed in a no. of cancers, including breast cancer.  To address the possible roles of PRMT overexpression in mammary gland tumorigenesis, we generated Cre-activated PRMT1, CARM1, and PRMT6 overexpression mouse models.  These three enzymes are the primary type I PRMTs and are responsible for the majority of the asym. arginine methylation deposited in the cells.  Using either a keratin 5-Cre recombinase (K5-Cre) cross or an MMTV-NIC mouse, we investigated the impact of PRMT overexpression alone or in the context of a HER2-driven model of breast cancer, resp.  The overexpression of all three PRMTs induced hyper-branching of the mammary glands and increased Ki-67 staining.  When combined with the MMTV-NIC model, these in vivo expts. provided the first genetic evidence implicating elevated levels of these three PRMTs in mammary gland tumorigenesis, albeit with variable degrees of tumor promotion and latency.  In addn., these mouse models provided valuable tools for exploring the biol. roles and mol. mechanisms of PRMT overexpression in the mammary gland.  For example, transcriptome anal. of purified mammary epithelial cells isolated from bigenic NIC-PRMT1Tg and NIC-PRMT6Tg mice revealed a deregulated PI3K-AKT pathway.  In the future, these PRMTTg lines can be leveraged to investigate the roles of arginine methylation in other tissues and tumor model systems using different tissue-specific Cre crosses, and they can also be used for testing the in vivo efficacy of small mol. inhibitors that target these PRMT function in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2aAowvjAR3rVg90H21EOLACvtfcHk0lixl_QM6CDCSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotFGhsL0%253D&md5=8a8799e61fa2a62e4a93db7a74d8b403</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-18-1995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-18-1995%26sid%3Dliteratum%253Aachs%26aulast%3DBao%26aufirst%3DJ.%26aulast%3DDi%2BLorenzo%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DSebastian%26aufirst%3DM.%2BM.%26aulast%3DMuller%26aufirst%3DW.%2BJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DMouse%2520models%2520of%2520overexpression%2520reveal%2520distinct%2520oncogenic%2520roles%2520for%2520different%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D21%26epage%3D32%26doi%3D10.1158%2F0008-5472.CAN-18-1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veland, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayatri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span> <span> </span><span class="NLM_article-title">The arginine methyltransferase PRMT6 regulates DNA methylation and contributes to global DNA hypomethylation in cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3390</span>– <span class="NLM_lpage">3397</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.11.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.celrep.2017.11.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=29262320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=3390-3397&author=N.+Velandauthor=S.+Hardikarauthor=Y.+Zhongauthor=S.+Gayatriauthor=J.+Danauthor=B.+D.+Strahlauthor=S.+B.+Rothbartauthor=M.+T.+Bedfordauthor=T.+Chen&title=The+arginine+methyltransferase+PRMT6+regulates+DNA+methylation+and+contributes+to+global+DNA+hypomethylation+in+cancer&doi=10.1016%2Fj.celrep.2017.11.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Arginine Methyltransferase PRMT6 Regulates DNA Methylation and Contributes to Global DNA Hypomethylation in Cancer</span></div><div class="casAuthors">Veland, Nicolas; Hardikar, Swanand; Zhong, Yi; Gayatri, Sitaram; Dan, Jiameng; Strahl, Brian D.; Rothbart, Scott B.; Bedford, Mark T.; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3390-3397</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">DNA methylation plays crucial roles in chromatin structure and gene expression.  Aberrant DNA methylation patterns, including global hypomethylation and regional hypermethylation, are assocd. with cancer and implicated in oncogenic events.  How DNA methylation is regulated in developmental and cellular processes and dysregulated in cancer is poorly understood.  Here, we show that PRMT6, a protein arginine methyltransferase responsible for asym. dimethylation of histone H3 arginine 2 (H3R2me2a), neg. regulates DNA methylation and that PRMT6 upregulation contributes to global DNA hypomethylation in cancer.  Mechanistically, PRMT6 overexpression impairs chromatin assocn. of UHRF1, an accessory factor of DNMT1, resulting in passive DNA demethylation.  The effect is likely due to elevated H3R2me2a, which inhibits the interaction between UHRF1 and histone H3.  Our work identifies a mechanistic link between protein arginine methylation and DNA methylation, which is disrupted in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeK-EYH5WxEbVg90H21EOLACvtfcHk0lixl_QM6CDCSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbK&md5=4ed7f5ed35723fadde97d9a5e3e99ea5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.11.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.11.082%26sid%3Dliteratum%253Aachs%26aulast%3DVeland%26aufirst%3DN.%26aulast%3DHardikar%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DGayatri%26aufirst%3DS.%26aulast%3DDan%26aufirst%3DJ.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DRothbart%26aufirst%3DS.%2BB.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520arginine%2520methyltransferase%2520PRMT6%2520regulates%2520DNA%2520methylation%2520and%2520contributes%2520to%2520global%2520DNA%2520hypomethylation%2520in%2520cancer%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D3390%26epage%3D3397%26doi%3D10.1016%2Fj.celrep.2017.11.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela
Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+Shenauthor=M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+Liauthor=K.+V.+Butlerauthor=H.+U.+Kaniskanauthor=B.+A.+Speedauthor=C.+Dela%0ASenaauthor=A.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0lhN51CArUe37g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabelian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class protein arginine methyltransferase 6 (PRMT6) covalent inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5477</span>– <span class="NLM_lpage">5487</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5477-5487&author=Y.+Shenauthor=F.+Liauthor=M.+M.+Szewczykauthor=L.+Halabelianauthor=K.+S.+Parkauthor=I.+Chauauthor=A.+Dongauthor=H.+Zengauthor=H.+Chenauthor=F.+Mengauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+first-in-class+protein+arginine+methyltransferase+6+%28PRMT6%29+covalent+inhibitor&doi=10.1021%2Facs.jmedchem.0c00406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor</span></div><div class="casAuthors">Shen, Yudao; Li, Fengling; Szewczyk, Magdalena M.; Halabelian, Levon; Park, Kwang-su; Chau, Irene; Dong, Aiping; Zeng, Hong; Chen, He; Meng, Fanye; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5477-5487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biol. processes assocd. with multiple cancers.  Well-characterized potent, selective, and cell-active PRMT6 inhibitors are invaluable tools for testing biol. and therapeutic hypotheses.  Although there are several known reversible PRMT6 inhibitors, covalent PRMT6 inhibitors have not been reported.  Based on a cocrystal structure of PRMT6-MS023 (a type I PRMT inhibitor), we discovered the first potent and cell-active irreversible PRMT6 inhibitor, 4 (MS117)(I).  The covalent binding mode of compd. 4 to PRMT6 was confirmed by mass spectrometry and kinetic studies and by a cocrystal structure.  Compd. 4 did not covalently modify other closely related PRMTs, potently inhibited PRMT6 in cells, and was selective for PRMT6 over other methyltransferases.  We also developed two structurally similar control compds., 5 (MS167) and 7 (MS168).  We provide these valuable chem. tools to the scientific community for further studying PRMT6 physiol. and pathophysiol. functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTU1W0CZjN17Vg90H21EOLACvtfcHk0lhN51CArUe37g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gsbs%253D&md5=cb244ab5a38d04320439c05048998b2f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DHalabelian%26aufirst%3DL.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520protein%2520arginine%2520methyltransferase%25206%2520%2528PRMT6%2529%2520covalent%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5477%26epage%3D5487%26doi%3D10.1021%2Facs.jmedchem.0c00406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradei, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: identification of the first PRMT6 tool compound</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0lgdvng7vQY0mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palte, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacey, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansueto, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reutershan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siliphaivanh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sondey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machacek, M. R.</span></span> <span> </span><span class="NLM_article-title">Allosteric modulation of protein arginine methyltransferase 5 (PRMT5)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1688</span>– <span class="NLM_lpage">1693</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00525</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00525" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFehtb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1688-1693&author=R.+L.+Palteauthor=S.+E.+Schneiderauthor=M.+D.+Altmanauthor=R.+P.+Hayesauthor=S.+Kawamuraauthor=B.+M.+Laceyauthor=M.+S.+Mansuetoauthor=M.+Reutershanauthor=P.+Siliphaivanhauthor=C.+Sondeyauthor=H.+Xuauthor=Z.+Xuauthor=Y.+Yeauthor=M.+R.+Machacek&title=Allosteric+modulation+of+protein+arginine+methyltransferase+5+%28PRMT5%29&doi=10.1021%2Facsmedchemlett.9b00525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Modulation of Protein Arginine Methyltransferase 5 (PRMT5)</span></div><div class="casAuthors">Palte, Rachel L.; Schneider, Sebastian E.; Altman, Michael D.; Hayes, Robert P.; Kawamura, Shuhei; Lacey, Brian M.; Mansueto, My Sam; Reutershan, Michael; Siliphaivanh, Phieng; Sondey, Christopher; Xu, Haiyan; Xu, Zangwei; Ye, Yingchun; Machacek, Michelle R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1688-1693</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) belongs to a family of enzymes that regulate the posttranslational modification of histones and other proteins via methylation of arginine.  Methylation of histones is linked to an increase in transcription and regulates a manifold of functions such as signal transduction and transcriptional regulation.  PRMT5 has been shown to be upregulated in the tumor environment of several cancer types, and the inhibition of PRMT5 activity was identified as a potential way to reduce tumor growth.  Previously, four different modes of PRMT5 inhibition were known-competing (covalently or non-covalently) with the essential cofactor S-adenosyl methionine (SAM), blocking the substrate binding pocket, or blocking both simultaneously.  Herein we describe an unprecedented conformation of PRMT5 in which the formation of an allosteric binding pocket abrogates the enzyme's canonical binding site and present the discovery of potent small mol. allosteric PRMT5 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFkbnJkLKiL7Vg90H21EOLACvtfcHk0lgdvng7vQY0mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFehtb7P&md5=75a8f75e60a3e20444af90b79c5af810</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00525%26sid%3Dliteratum%253Aachs%26aulast%3DPalte%26aufirst%3DR.%2BL.%26aulast%3DSchneider%26aufirst%3DS.%2BE.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DHayes%26aufirst%3DR.%2BP.%26aulast%3DKawamura%26aufirst%3DS.%26aulast%3DLacey%26aufirst%3DB.%2BM.%26aulast%3DMansueto%26aufirst%3DM.%2BS.%26aulast%3DReutershan%26aufirst%3DM.%26aulast%3DSiliphaivanh%26aufirst%3DP.%26aulast%3DSondey%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYe%26aufirst%3DY.%26aulast%3DMachacek%26aufirst%3DM.%2BR.%26atitle%3DAllosteric%2520modulation%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1688%26epage%3D1693%26doi%3D10.1021%2Facsmedchemlett.9b00525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mailliot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troffer-Charlier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianferani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavarelli, J.</span></span> <span> </span><span class="NLM_article-title">Functional insights from high resolution structures of mouse protein arginine methyltransferase 6</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2015.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1016%2Fj.jsb.2015.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=26094878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFSjtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2015&pages=175-183&author=L.+Bonnefondauthor=J.+Stojkoauthor=J.+Mailliotauthor=N.+Troffer-Charlierauthor=V.+Curaauthor=J.+M.+Wurtzauthor=S.+Cianferaniauthor=J.+Cavarelli&title=Functional+insights+from+high+resolution+structures+of+mouse+protein+arginine+methyltransferase+6&doi=10.1016%2Fj.jsb.2015.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Functional insights from high resolution structures of mouse protein arginine methyltransferase 6</span></div><div class="casAuthors">Bonnefond, Luc; Stojko, Johann; Mailliot, Justine; Troffer-Charlier, Nathalie; Cura, Vincent; Wurtz, Jean-Marie; Cianferani, Sarah; Cavarelli, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">175-183</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">PRMT6 is a protein arginine methyltransferase involved in transcriptional regulation, human immunodeficiency virus pathogenesis, DNA base excision repair, and cell cycle progression.  Like other PRMTs, PRMT6 is overexpressed in several cancer types and is therefore considered as a potential anti-cancer drug target.  In the present study, we described six crystal structures of PRMT6 from Mus musculus, solved and refined at 1.34 Å for the highest resoln. structure.  The crystal structures revealed that the folding of the helix αX is required to stabilize a productive active site before methylation of the bound peptide can occur.  In the absence of cofactor, metal cations can be found in the catalytic pocket at the expected position of the guanidinium moiety of the target arginine substrate.  Using mass spectrometry under native conditions, we show that PRMT6 dimer binds two cofactor and a single H4 peptide mols.  Finally, we characterized a new site of in vitro automethylation of mouse PRMT6 at position 7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIfUGr0Ak2b7Vg90H21EOLACvtfcHk0ljoiuq1AM--Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFSjtLjF&md5=306f52f65b771a7e6351d0c21c11a570</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2015.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2015.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DBonnefond%26aufirst%3DL.%26aulast%3DStojko%26aufirst%3DJ.%26aulast%3DMailliot%26aufirst%3DJ.%26aulast%3DTroffer-Charlier%26aufirst%3DN.%26aulast%3DCura%26aufirst%3DV.%26aulast%3DWurtz%26aufirst%3DJ.%2BM.%26aulast%3DCianferani%26aufirst%3DS.%26aulast%3DCavarelli%26aufirst%3DJ.%26atitle%3DFunctional%2520insights%2520from%2520high%2520resolution%2520structures%2520of%2520mouse%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2015%26volume%3D191%26spage%3D175%26epage%3D183%26doi%3D10.1016%2Fj.jsb.2015.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sinnokrot, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherrill, C. D.</span></span> <span> </span><span class="NLM_article-title">Substituent effects in pi-pi interactions: sandwich and T-shaped configurations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">7690</span>– <span class="NLM_lpage">7697</span>, <span class="refDoi"> DOI: 10.1021/ja049434a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja049434a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVejur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=7690-7697&author=M.+O.+Sinnokrotauthor=C.+D.+Sherrill&title=Substituent+effects+in+pi-pi+interactions%3A+sandwich+and+T-shaped+configurations&doi=10.1021%2Fja049434a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Substituent Effects in π-π Interactions: Sandwich and T-Shaped Configurations</span></div><div class="casAuthors">Sinnokrot, Mutasem Omar; Sherrill, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7690-7697</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sandwich and T-shaped configurations of benzene dimer, benzene-phenol, benzene-toluene, benzene-fluorobenzene, and benzene-benzonitrile are studied by coupled-cluster theory to elucidate how substituents tune π-π interactions.  All substituted sandwich dimers bind more strongly than benzene dimer, whereas the T-shaped configurations bind more or less favorably depending on the substituent.  Symmetry-adapted perturbation theory (SAPT) indicates that electrostatic, dispersion, induction, and exchange-repulsion contributions are all significant to the overall binding energies, and all but induction are important in detg. relative energies.  Models of π-π interactions based solely on electrostatics, such as the Hunter-Sanders rules, do not seem capable of explaining the energetic ordering of the dimers considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFBZHgFWNA5rVg90H21EOLACvtfcHk0ljoiuq1AM--Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVejur4%253D&md5=f9c4afdd96f98ecc1d46f93a0ce0f17a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja049434a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja049434a%26sid%3Dliteratum%253Aachs%26aulast%3DSinnokrot%26aufirst%3DM.%2BO.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26atitle%3DSubstituent%2520effects%2520in%2520pi-pi%2520interactions%253A%2520sandwich%2520and%2520T-shaped%2520configurations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D7690%26epage%3D7697%26doi%3D10.1021%2Fja049434a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchcock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1176</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0ljoiuq1AM--Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution--from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%2D%2Dfrom+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0liqn25GdWpZHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution--from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liqn25GdWpZHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in BUSTER</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1107/S0907444911056058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444911056058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=22505257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=368-380&author=O.+S.+Smartauthor=T.+O.+Womackauthor=C.+Flensburgauthor=P.+Kellerauthor=W.+Paciorekauthor=A.+Sharffauthor=C.+Vonrheinauthor=G.+Bricogne&title=Exploiting+structure+similarity+in+refinement%3A+automated+NCS+and+target-structure+restraints+in+BUSTER&doi=10.1107%2FS0907444911056058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in BUSTER</span></div><div class="casAuthors">Smart, Oliver S.; Womack, Thomas O.; Flensburg, Claus; Keller, Peter; Paciorek, Wlodek; Sharff, Andrew; Vonrhein, Clemens; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">368-380</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Maximum-likelihood X-ray macromol. structure refinement in BUSTER has been extended with restraints facilitating the exploitation of structural similarity.  The similarity can be between two or more chains within the structure being refined, thus favoring NCS, or to a distinct 'target' structure that remains fixed during refinement.  The local structural similarity restraints (LSSR) approach considers all distances less than 5.5 Å between pairs of atoms in the chain to be restrained.  For each, the difference from the distance between the corresponding atoms in the related chain is found.  LSSR applies a restraint penalty on each difference.  A functional form that reaches a plateau for large differences is used to avoid the restraints distorting parts of the structure that are not similar.  Because LSSR are local, there is no need to sep. out domains.  Some restraint pruning is still necessary, but this has been automated.  LSSR have been available to academic users of BUSTER since 2009 with the easy-to-use and options.  The use of LSSR is illustrated in the re-refinement of PDB entries , where enables the correct ligand-binding structure to be found, and , where contributes to the location of an addnl. copy of the cyclic peptide ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu-qo0DGjn3bVg90H21EOLACvtfcHk0liqn25GdWpZHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D&md5=bea89f86f489e8dbd00a9866a3d7dab9</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444911056058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911056058%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26aulast%3DWomack%26aufirst%3DT.%2BO.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DExploiting%2520structure%2520similarity%2520in%2520refinement%253A%2520automated%2520NCS%2520and%2520target-structure%2520restraints%2520in%2520BUSTER%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D368%26epage%3D380%26doi%3D10.1107%2FS0907444911056058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lj2Hl_-cfW1MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MOLPROBITY: structure validation and all-atom contact analysis for nucleic acids and their complexes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W615</span>– <span class="NLM_lpage">W619</span>, <span class="refDoi"> DOI: 10.1093/nar/gkh398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1093%2Fnar%2Fgkh398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=15215462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W615-W619&author=I.+W.+Davisauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MOLPROBITY%3A+structure+validation+and+all-atom+contact+analysis+for+nucleic+acids+and+their+complexes&doi=10.1093%2Fnar%2Fgkh398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">MolPROBITY: Structure validation and all-atom contact analysis for nucleic acids and their complexes</span></div><div class="casAuthors">Davis, Ian W.; Murray, Laura Weston; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W615-W619</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MolProbity is a general-purpose web service offering quality validation for three-dimensional (3D) structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact anal. of any steric problems within the mols. and can calc. and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addn. and full optimization of all hydrogen atoms, both polar and nonpolar.  The results are reported in multiple forms: as overall numeric scores, as lists, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KING viewer.  This service is available free to all users at http://kinemage.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9I64e-HQbnbVg90H21EOLACvtfcHk0lj2Hl_-cfW1MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D&md5=2050f7acf49743addde69c5ce9c9e2e4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh398%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMOLPROBITY%253A%2520structure%2520validation%2520and%2520all-atom%2520contact%2520analysis%2520for%2520nucleic%2520acids%2520and%2520their%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW615%26epage%3DW619%26doi%3D10.1093%2Fnar%2Fgkh398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vhuiyan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of arginine asymmetrical dimethylation by PRMT6</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>473</i></span>,  <span class="NLM_fpage">3049</span>– <span class="NLM_lpage">3063</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1042%2FBCJ20160537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=27480107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2016&pages=3049-3063&author=H.+Wuauthor=W.+Zhengauthor=M.+S.+Eramauthor=M.+Vhuiyanauthor=A.+Dongauthor=H.+Zengauthor=H.+Heauthor=P.+Brownauthor=A.+Frankelauthor=M.+Vedadiauthor=M.+Luoauthor=J.+Min&title=Structural+basis+of+arginine+asymmetrical+dimethylation+by+PRMT6&doi=10.1042%2FBCJ20160537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of arginine asymmetrical dimethylation by PRMT6</span></div><div class="casAuthors">Wu, Hong; Zheng, Weihong; Eram, Mohammad S.; Vhuiyan, Mynol; Dong, Aiping; Zeng, Hong; He, Hao; Brown, Peter; Frankel, Adam; Vedadi, Masoud; Luo, Minkui; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3049-3063</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">PRMT6 is a type I protein arginine methyltransferase, generating the asym. dimethylarginine mark on proteins such as histone H3R2.  Asym. dimethylation of histone H3R2 by PRMT6 acts as a repressive mark that antagonizes trimethylation of H3 lysine 4 by the MLL histone H3K4 methyltransferase.  PRMT6 is overexpressed in several cancer types, including prostate, bladder and lung cancers; therefore, it is of great interest to develop potent and selective inhibitors for PRMT6.  Here, we report the synthesis of a potent bisubstrate inhibitor GMS [6'-methyleneamine sinefungin, an analog of sinefungin (SNF)], and the crystal structures of human PRMT6 in complex, resp., with S-adenosyl- L-homocysteine (SAH) and the bisubstrate inhibitor GMS that shed light on the significantly improved inhibition effect of GMS on methylation activity of PRMT6 compared with SAH and an S-adenosyl-L-methionine competitive methyltransferase inhibitor SNF.  In addn., we also crystd. PRMT6 in complex with SAH and a short arginine-contg. peptide.  Based on the structural information here and available in the PDB database, we proposed a mechanism that can rationalize the distinctive arginine methylation product specificity of different types of arginine methyltransferases and pinpoint the structural determinant of such a specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo061VtGpp_KLVg90H21EOLACvtfcHk0lj2Hl_-cfW1MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ylsLc%253D&md5=dcacb06a69eb1a4856bea9b0d8458112</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160537%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DVhuiyan%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520arginine%2520asymmetrical%2520dimethylation%2520by%2520PRMT6%26jtitle%3DBiochem.%2520J.%26date%3D2016%26volume%3D473%26spage%3D3049%26epage%3D3063%26doi%3D10.1042%2FBCJ20160537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2110</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lj2Hl_-cfW1MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=30604761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=19&author=S.+Scheerauthor=S.+Acklooauthor=T.+S.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+A.+Wardauthor=A.+M.+Lewisauthor=J.+P.+Northropauthor=P.+L.+Richardsonauthor=H.+U.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+A.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+V.+M.+Huberauthor=D.+D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+J.+Brownauthor=C.+H.+Arrowsmith&title=A+chemical+biology+toolbox+to+study+protein+methyltransferases+and+epigenetic+signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span></div><div class="casAuthors">Scheer, Sebastian; Ackloo, Suzanne; Medina, Tiago S.; Schapira, Matthieu; Li, Fengling; Ward, Jennifer A.; Lewis, Andrew M.; Northrop, Jeffrey P.; Richardson, Paul L.; Kaniskan, H. Umit; Shen, Yudao; Liu, Jing; Smil, David; McLeod, David; Zepeda-Velazquez, Carlos A.; Luo, Minkui; Jin, Jian; Barsyte-Lovejoy, Dalia; Huber, Kilian V. M.; De Carvalho, Daniel D.; Vedadi, Masoud; Zaph, Colby; Brown, Peter J.; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) comprise a major class of epigenetic regulatory enzymes with therapeutic relevance.  Here we present a collection of chem. probes and assocd. reagents and data to elucidate the function of human and murine PMTs in cellular studies.  Our collection provides inhibitors and antagonists that together modulate most of the key regulatory methylation marks on histones H3 and H4, providing an important resource for modulating cellular epigenomes.  We describe a comprehensive and comparative characterization of the probe collection with respect to their potency, selectivity, and mode of inhibition.  We demonstrate the utility of this collection in CD4+ T cell differentiation assays revealing the potential of individual probes to alter multiple T cell subpopulations which may have implications for T cell-mediated processes such as inflammation and immuno-oncol.  In particular, we demonstrate a role for DOT1L in limiting Th1 cell differentiation and maintaining lineage integrity.  This chem. probe collection and assocd. data form a resource for the study of methylation-mediated signaling in epigenetics, inflammation and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA5rgeiWDvrVg90H21EOLACvtfcHk0ljf0NGXnOfP_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D&md5=008fec084426171f5c91657e54b73900</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%2BA.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DA%2520chemical%2520biology%2520toolbox%2520to%2520study%2520protein%2520methyltransferases%2520and%2520epigenetic%2520signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D19%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i30"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02160">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_21417"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02160?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02160</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Selectivity data, mass spectrometry data, effect of preincubation on PRMT6 inhibitory potency of <i>(R)-</i><b>2</b>, crystal structure of PRMT6 with <i>(R)-</i><b>1</b>, crystallographic statistics and collection parameters, spectral characterization of compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_001.pdf">jm0c02160_si_001.pdf (5.72 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02160/suppl_file/jm0c02160_si_002.csv">jm0c02160_si_002.csv (0.82 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02160&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-7%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02160%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02160" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992e37bdfe3cee","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
