
<html lang="en"     class="pb-page"  data-request-id="cfa7c207-729a-4e35-a6c1-e1c448aa0264"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.8b01721;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-5;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Binding to an Unusual Inactive Kinase Conformation by Highly Selective Inhibitors of Inositol-Requiring Enzyme 1α Kinase-Endoribonuclease" /></meta><meta name="dc.Creator" content="Giampiero  Colombano" /></meta><meta name="dc.Creator" content="John J.  Caldwell" /></meta><meta name="dc.Creator" content="Thomas P.  Matthews" /></meta><meta name="dc.Creator" content="Chitra  Bhatia" /></meta><meta name="dc.Creator" content="Amar  Joshi" /></meta><meta name="dc.Creator" content="Tatiana  McHardy" /></meta><meta name="dc.Creator" content="Ngai Yi  Mok" /></meta><meta name="dc.Creator" content="Yvette  Newbatt" /></meta><meta name="dc.Creator" content="Lisa  Pickard" /></meta><meta name="dc.Creator" content="Jade  Strover" /></meta><meta name="dc.Creator" content="Somaieh  Hedayat" /></meta><meta name="dc.Creator" content="Michael I.  Walton" /></meta><meta name="dc.Creator" content="Stephanie M.  Myers" /></meta><meta name="dc.Creator" content="Alan M.  Jones" /></meta><meta name="dc.Creator" content="Harry  Saville" /></meta><meta name="dc.Creator" content="Craig  McAndrew" /></meta><meta name="dc.Creator" content="Rosemary  Burke" /></meta><meta name="dc.Creator" content="Suzanne A.  Eccles" /></meta><meta name="dc.Creator" content="Faith E.  Davies" /></meta><meta name="dc.Creator" content="Richard  Bayliss" /></meta><meta name="dc.Creator" content="Ian  Collins" /></meta><meta name="dc.Description" content="A series of imidazo[1,2-b]pyridazin-8-amine kinase inhibitors were discovered to allosterically inhibit the endoribonuclease function of the dual kinase-endoribonuclease inositol-requiring enzyme 1..." /></meta><meta name="Description" content="A series of imidazo[1,2-b]pyridazin-8-amine kinase inhibitors were discovered to allosterically inhibit the endoribonuclease function of the dual kinase-endoribonuclease inositol-requiring enzyme 1..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 19, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01721" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01721" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01721" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01721" /></link>
        
    
    

<title>Binding to an Unusual Inactive Kinase Conformation by Highly Selective Inhibitors of Inositol-Requiring Enzyme 1α Kinase-Endoribonuclease | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01721" /></meta><meta property="og:title" content="Binding to an Unusual Inactive Kinase Conformation by Highly Selective Inhibitors of Inositol-Requiring Enzyme 1α Kinase-Endoribonuclease" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0007.jpeg" /></meta><meta property="og:description" content="A series of imidazo[1,2-b]pyridazin-8-amine kinase inhibitors were discovered to allosterically inhibit the endoribonuclease function of the dual kinase-endoribonuclease inositol-requiring enzyme 1α (IRE1α), a key component of the unfolded protein response in mammalian cells and a potential drug target in multiple human diseases. Inhibitor optimization gave compounds with high kinome selectivity that prevented endoplasmic reticulum stress-induced IRE1α oligomerization and phosphorylation, and inhibited endoribonuclease activity in human cells. X-ray crystallography showed the inhibitors to bind to a previously unreported and unusually disordered conformation of the IRE1α kinase domain that would be incompatible with back-to-back dimerization of the IRE1α protein and activation of the endoribonuclease function. These findings increase the repertoire of known IRE1α protein conformations and can guide the discovery of highly selective ligands for the IRE1α kinase site that allosterically inhibit the endoribonuclease." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01721"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01721">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01721&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01721&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01721&amp;href=/doi/10.1021/acs.jmedchem.8b01721" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2447-2465</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01714" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01775" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Binding to an Unusual Inactive Kinase Conformation by Highly Selective Inhibitors of Inositol-Requiring Enzyme 1α Kinase-Endoribonuclease</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Giampiero Colombano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Giampiero Colombano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Giampiero++Colombano">Giampiero Colombano</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John J. Caldwell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John J. Caldwell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+J.++Caldwell">John J. Caldwell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas P. Matthews</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas P. Matthews</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+P.++Matthews">Thomas P. Matthews</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4765-7734" title="Orcid link">http://orcid.org/0000-0002-4765-7734</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chitra Bhatia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chitra Bhatia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cell Biology, University of Leicester, Leicester LE1 7RH, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chitra++Bhatia">Chitra Bhatia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Amar Joshi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amar Joshi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cell Biology, University of Leicester, Leicester LE1 7RH, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amar++Joshi">Amar Joshi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tatiana McHardy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tatiana McHardy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tatiana++McHardy">Tatiana McHardy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ngai Yi Mok</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ngai Yi Mok</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ngai+Yi++Mok">Ngai Yi Mok</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2827-3735" title="Orcid link">http://orcid.org/0000-0002-2827-3735</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yvette Newbatt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yvette Newbatt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yvette++Newbatt">Yvette Newbatt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lisa Pickard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lisa Pickard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lisa++Pickard">Lisa Pickard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jade Strover</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jade Strover</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Molecular Pathology, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jade++Strover">Jade Strover</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Somaieh Hedayat</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Somaieh Hedayat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Somaieh++Hedayat">Somaieh Hedayat</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael I. Walton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael I. Walton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+I.++Walton">Michael I. Walton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephanie M. Myers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephanie M. Myers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephanie+M.++Myers">Stephanie M. Myers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alan M. Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alan M. Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alan+M.++Jones">Alan M. Jones</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Harry Saville</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Harry Saville</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Harry++Saville">Harry Saville</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Craig McAndrew</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Craig McAndrew</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Craig++McAndrew">Craig McAndrew</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rosemary Burke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rosemary Burke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rosemary++Burke">Rosemary Burke</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Suzanne A. Eccles</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Suzanne A. Eccles</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Suzanne+A.++Eccles">Suzanne A. Eccles</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Faith E. Davies</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Faith E. Davies</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Molecular Pathology, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Faith+E.++Davies">Faith E. Davies</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard Bayliss</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard Bayliss</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cell Biology, University of Leicester, Leicester LE1 7RH, U.K.</div><div class="loa-info-affiliations-info">School of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Bayliss">Richard Bayliss</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0604-2773" title="Orcid link">http://orcid.org/0000-0003-0604-2773</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ian Collins</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ian Collins</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SW7 3RP, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#167f77783875797a7a7f7865567f756438777538637d"><span class="__cf_email__" data-cfemail="86efe7e8a8e5e9eaeaefe8f5c6efe5f4a8e7e5a8f3ed">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ian++Collins">Ian Collins</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8143-8498" title="Orcid link">http://orcid.org/0000-0002-8143-8498</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01721&amp;href=/doi/10.1021%2Facs.jmedchem.8b01721" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2447–2465</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 19, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 November 2018</li><li><span class="item_label"><b>Published</b> online</span>19 February 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 March 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01721" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01721</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01721"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2929</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01721" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Binding to an Unusual Inactive Kinase Conformation by Highly Selective Inhibitors of Inositol-Requiring Enzyme 1α Kinase-Endoribonuclease&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Giampiero&quot;,&quot;last_name&quot;:&quot;Colombano&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;J. Caldwell&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;P. Matthews&quot;},{&quot;first_name&quot;:&quot;Chitra&quot;,&quot;last_name&quot;:&quot;Bhatia&quot;},{&quot;first_name&quot;:&quot;Amar&quot;,&quot;last_name&quot;:&quot;Joshi&quot;},{&quot;first_name&quot;:&quot;Tatiana&quot;,&quot;last_name&quot;:&quot;McHardy&quot;},{&quot;first_name&quot;:&quot;Ngai&quot;,&quot;last_name&quot;:&quot;Yi Mok&quot;},{&quot;first_name&quot;:&quot;Yvette&quot;,&quot;last_name&quot;:&quot;Newbatt&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;Pickard&quot;},{&quot;first_name&quot;:&quot;Jade&quot;,&quot;last_name&quot;:&quot;Strover&quot;},{&quot;first_name&quot;:&quot;Somaieh&quot;,&quot;last_name&quot;:&quot;Hedayat&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;I. Walton&quot;},{&quot;first_name&quot;:&quot;Stephanie&quot;,&quot;last_name&quot;:&quot;M. Myers&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;M. Jones&quot;},{&quot;first_name&quot;:&quot;Harry&quot;,&quot;last_name&quot;:&quot;Saville&quot;},{&quot;first_name&quot;:&quot;Craig&quot;,&quot;last_name&quot;:&quot;McAndrew&quot;},{&quot;first_name&quot;:&quot;Rosemary&quot;,&quot;last_name&quot;:&quot;Burke&quot;},{&quot;first_name&quot;:&quot;Suzanne&quot;,&quot;last_name&quot;:&quot;A. Eccles&quot;},{&quot;first_name&quot;:&quot;Faith&quot;,&quot;last_name&quot;:&quot;E. Davies&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Bayliss&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;Collins&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;2447-2465&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01721&quot;},&quot;abstract&quot;:&quot;A series of imidazo[1,2-b]pyridazin-8-amine kinase inhibitors were discovered to allosterically inhibit the endoribonuclease function of the dual kinase-endoribonuclease inositol-requiring enzyme 1α (IRE1α), a key component of the unfolded protein response in mammalian cells and a potential drug target in multiple human diseases. Inhibitor optimization gave compounds with high kinome selectivity that prevented endoplasmic reticulum stress-induced IRE1α oligomerization and phosphorylation, and inhibited endoribonuclease activity in human cells. X-ray crystallography showed the inhibitors to bind to a previously unreported and unusually disordered conformation of the IRE1α kinase domain that would be incompatible with back-to-back dimerization of the IRE1α protein and activation of the endoribonuclease function. These findings increase the repertoire of known IRE1α protein conformations and can guide the discovery of highly selective ligands for the IRE1α kinase site that allosterically inhibit the endoribonucl&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01721&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01721" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01721&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01721" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01721&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01721" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01721&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01721&amp;href=/doi/10.1021/acs.jmedchem.8b01721" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01721" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01721" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01721%26sid%3Dliteratum%253Aachs%26pmid%3D30779566%26genre%3Darticle%26aulast%3DColombano%26date%3D2019%26atitle%3DBinding%2Bto%2Ban%2BUnusual%2BInactive%2BKinase%2BConformation%2Bby%2BHighly%2BSelective%2BInhibitors%2Bof%2BInositol-Requiring%2BEnzyme%2B1%25CE%25B1%2BKinase-Endoribonuclease%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D5%26spage%3D2447%26epage%3D2465%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=291774" title="Nuclear magnetic resonance spectroscopy">Nuclear magnetic resonance spectroscopy</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/jmcmar.2019.62.issue-5/20190314/jmcmar.2019.62.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A series of imidazo[1,2-<i>b</i>]pyridazin-8-amine kinase inhibitors were discovered to allosterically inhibit the endoribonuclease function of the dual kinase-endoribonuclease inositol-requiring enzyme 1α (IRE1α), a key component of the unfolded protein response in mammalian cells and a potential drug target in multiple human diseases. Inhibitor optimization gave compounds with high kinome selectivity that prevented endoplasmic reticulum stress-induced IRE1α oligomerization and phosphorylation, and inhibited endoribonuclease activity in human cells. X-ray crystallography showed the inhibitors to bind to a previously unreported and unusually disordered conformation of the IRE1α kinase domain that would be incompatible with back-to-back dimerization of the IRE1α protein and activation of the endoribonuclease function. These findings increase the repertoire of known IRE1α protein conformations and can guide the discovery of highly selective ligands for the IRE1α kinase site that allosterically inhibit the endoribonuclease.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30031" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30031" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The dual kinase-endoribonuclease inositol-requiring enzyme 1α (IRE1α; ERN1) is a central and conserved component of the unfolded protein response (UPR) activated by eukaryotic cells in reaction to endoplasmic reticulum (ER) stress, caused by an excess of misfolded proteins in the ER lumen.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Activation of the UPR reduces protein translation and increases the protein-folding capacity. If this initial response does not alleviate ER stress, prolonged activation of the UPR signaling network leads to apoptosis. Thus, the balance of downstream anti- and proapoptotic signaling pathways activated by the UPR determines cell fate in the presence of acute or chronic proteotoxic stress.</div><div class="NLM_p">IRE1α is an ER membrane resident protein with serine/threonine kinase and endoribonuclease (RNase) functions contained in its cytosolic domain.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The accumulation of unfolded or misfolded proteins in the ER lumen triggers oligomerization of IRE1α and kinase autophosphorylation, resulting in activation of the RNase.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> The RNase specifically cleaves the messenger RNA (mRNA) coding for the unspliced transcription factor X-box protein 1 (XBP1u) to remove a 26 base pair intron, and subsequent ligation produces the mRNA of the active, spliced form (XBP1s) of the transcription factor.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> XBP1s-dependent transcription includes upregulation of ER-resident molecular chaperones required to relieve aberrant protein folding.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In contrast, nonspecific cleavage of other mRNAs localized at the ER membrane by IRE1α RNase (termed regulated IRE1-dependent decay) is proposed to exert an apoptotic effect.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Additionally, the interaction of TRAF2 with oligomerized phospho-IRE1α elicits signaling through the JNK/c-JUN axis, independent of IRE1α RNase activity.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In mammalian cells, IRE1α functions with the transcriptional repression and regulation of apoptosis effected by the other UPR elements, ATF6 and PERK, to alleviate ER stress or to direct cell death.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a></div><div class="NLM_p">Dysregulation of IRE1α-XBP1s signaling is implicated in a number of human diseases, including cancer, diabetes, lipidemia, inflammatory disease, and neurodegeneration.<a onclick="showRef(event, 'ref1 ref11'); return false;" href="javascript:void(0);" class="ref ref1 ref11">(1,11)</a> Signaling through IRE1α-XBP1s has been proposed as important for the survival of myeloma cells, as well as breast and prostate cancers.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> Recently, in an ovarian cancer in vivo model, XBP1s function has been shown to suppress proper presentation of tumor-associated antigens by tumor-infiltrating dendritic cells, thus determining the degree of T-cell antitumor immune response and suggesting that modulation of XBP1 splicing in normal tissue may be an alternative approach for an anticancer therapy.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p">There is a clear need for selective chemical tools to explore the consequences of IRE1α inhibition by different mechanisms in cancer and nontransformed cells and to study the IRE1α-XBP1s pathway in other human diseases. Given the diverse nature of IRE1α signaling outputs, inhibitors targeting the kinase or RNase functions, or both, may have different consequences for cell fate, which may further vary according to cell context. Reversible covalent binders targeting the RNase active site directly have been reported from a salicylaldehyde scaffold<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> and have been shown to inhibit cancer cell growth.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> The pharmacology of inhibitors binding at the IRE1α kinase site is complicated by the possibility of allosteric inhibition or activation of the RNase function depending on the conformation of the kinase domain and oligomerization state that is stabilized.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Allosteric inhibition of the RNase function has been associated with type II kinase site ligands that bind the DFG-out conformation of the kinase<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> or other compounds that displace the αC-helix of the kinase from the active conformation.<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22−24)</a> While a series of highly selective, kinase site binding, allosteric inhibitors of IRE1α RNase have shown no direct cytotoxic effect on a range of cancer cell lines,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> compounds with this profile remain of interest for other therapeutic approaches that target nonmalignant cells.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In contrast to type II and αC-helix displacing scaffolds, reported type I IRE1α kinase site ligands promote dimerization and activate the RNase function.<a onclick="showRef(event, 'ref6 ref21'); return false;" href="javascript:void(0);" class="ref ref6 ref21">(6,21)</a> The identification of new activators and inhibitors of IRE1α RNase and understanding the allosteric mechanisms linking the kinase and RNase sites are therefore of continued importance for the development of new therapeutics.</div><div class="NLM_p">We previously described the type I imidazo[1,2-<i>b</i>]pyridazine kinase inhibitor <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) that allosterically activates IRE1α endoribonuclease in vitro.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> While screening chemical scaffolds based on <b>1</b>, we unexpectedly discovered the related type I imidazo[1,2-<i>b</i>]pyridazin-8-amine <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) to be an inhibitor of both the kinase and RNase functions of IRE1α. In this study, we show how <b>2</b> and more potent analogues bind to a previously undescribed inactive conformation of monomeric IRE1α involving major rearrangements of key secondary structure in the kinase domain, including disordering of the αC-helix and disruption of the hydrophobic spine of the protein, which together confer high IRE1α kinase binding selectivity and inhibition of both kinase and RNase functions. We confirm that these allosteric modulators disrupt all downstream signaling through IRE1α in human cells, thus validating targeting of this unusual binding mode as a new way to generate selective and cell active inhibitors of IRE1α.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structures of the IRE1α RNase activator <b>1</b> and the structurally related RNase inhibitor <b>2</b>. (B) Structures of selected published IRE1α kinase-RNase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Synthetic Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28373" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28373" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We established short synthetic routes to prepare analogues of <b>2</b> using the differential reactivity of the polyhalogenated imidazo[1,2-<i>b</i>]pyridazine <b>3</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Selective displacement of the 8-bromo substituent introduced (cyclopropylmethyl)amine and was followed by Suzuki reactions of the 3-bromo group of <b>4</b> to give compounds <b>5</b>–<b>12</b>. The 6-chloro substituent of <b>3</b> and <b>4</b> might be expected to undergo competing nucleophilic substitution or palladium insertion but proved notably stable under the conditions studied.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> To replace the indazol-5-yl substituent of <b>2</b> with 2-(<i>N</i>-phenyl)amino-1<i>H</i>-benzo[<i>d</i>]imidazole, the intermediate <b>13</b> was prepared from 5-bromo-2-chloro-1<i>H</i>-benzo[<i>d</i>]imidazole by displacement of the 2-chloro group with aniline and N-protection, before palladium-catalyzed installation of the boronate (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Compound <b>13</b> was obtained and used as a mixture of <i>N</i>-Boc 2-aminobenzimidazole regioisomers, with complete N-deprotection observed in the subsequent Suzuki reaction. 2-(Methylamino)benzimidazole <b>14</b> did not require N-protection for successful reaction. Other N<i>-</i>methylated 2-aminobenzimidazoles <b>16</b> and <b>18</b> were synthesized from the nitrophenyl boronates <b>15</b> and <b>17</b>, respectively, by nitro group reduction and condensation with isothiocyanatobenzene.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>2</b>–<b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 2-chloroacetaldehyde, EtOH, 50 °C, 21 h, 78%; (ii) <i>N</i>-bromosuccinimide, CH<sub>3</sub>Cl, 0 °C to room temperature (rt), 16 h, 84%; (iii) cyclopropylmethanamine, tetrahydrofuran (THF), rt, 3 h, 93%; (iv) heteroaryl boronate, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, dioxane, 120–135 °C, 15–22 h, 5–51%; (v) (a) PhNH<sub>2</sub>, 180 °C (microwave), 1 h; (b) (Boc)<sub>2</sub>O, 4-dimethylaminopyridine (DMAP), THF, rt, 16 h; (c) bis(pinacolato)diboron, KOAc, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, 1,4-dioxane, 100 °C, 16 h, 53% over three steps; (vi) PhNHMe, 180 °C (microwave), 1 h, 86%; (vii) bis(pinacolato)diboron, KOAc, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, 1,4-dioxane, 100 °C, 16 h, 36%; (viii) isothiocyanatobenzene, 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDC), THF, 50 °C, 5 h, 17–45%.</p></p></figure><div class="NLM_p">The synthesis was adapted to vary the 8-amino substituent of the imidazo[1,2-<i>b</i>]pyridazine by reacting a range of amines with substrate <b>3</b> to give <b>19</b>–<b>21</b> and <b>27</b>–<b>29</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). For the unsubstituted 1,4-diaminocyclohexanes <b>22</b> and <b>23</b>, N-protection of the terminal amine, to ensure selectivity, and higher temperatures were required in the nucleophilic substitution step. The more elaborate 1,4-diaminocyclohexane substituents incorporated into <b>24</b>–<b>26</b> were made prior to reaction with <b>3</b>. Fluorine substituents were introduced into the 2-(<i>N</i>-phenyl)amino-1<i>H</i>-benzo[<i>d</i>]imidazole group starting from the appropriate fluorinated 4-bromobenzene-1,2-diamines via condensation with isothiocyanatobenzene, and the route was telescoped to generate compounds <b>30</b>–<b>32</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Compounds <b>19</b>–<b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) amine, THF, rt, 16 h, 60–97%; (ii) heteroaryl boronate, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 130 °C, 16 h, 4–55%; (iii) <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)carbamate, THF, 50 °C, 16 h, quantitative; (iv) HCl, 1,4-dioxane, rt, 2 h, 99%; (v) (a) LiAlH<sub>4</sub>, THF, 0 °C to reflux, 4 h; (b) ethyl 1,3-dioxoisoindoline-2-carboxylate, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 15 h; (c) <b>3</b>, Et<sub>3</sub>N, 1,4-dioxane–dimethyl sulfoxide (DMSO), 100 °C, 22 h, 52% over three steps; (vi) 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)-ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 130 °C, 21 h, then NH<sub>2</sub>NH<sub>2</sub>·H<sub>2</sub>O, 100 °C, 48 h, 34%; (vii) amine, THF, 50 °C, 16 h, 29–85%; (viii) (a) <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)carbamate, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 20 h; (b) HCl–1,4-dioxane, MeOH, rt, 2 h, 37% over two steps; (ix) <b>3</b>, Et<sub>3</sub>N, 1,4-dioxane-DMSO, 50 °C, 23 h, 58%; (x) (a) isothiocyanatobenzene, EDC, THF, 50 °C, 5 h; (b) (Boc)<sub>2</sub>O, DMAP, THF, rt, 16 h; (c) bis(pinacolato)diboron, KOAc, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, 1,4-dioxane, 135 °C, 24 h; (d) 3-bromo-6-chloro-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)-ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 135 °C, 16 h, 2–5% over four steps.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84030" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84030" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compound <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) was discovered through screening of type I kinase scaffolds related to the IRE1α kinase inhibitor-RNase activator <b>1</b><sup>6</sup>. Unexpectedly, we determined that while inhibiting the kinase activity with similar potency to <b>1</b>, compound <b>2</b> was an inhibitor of the RNase function in vitro, suggesting a different allosteric effect on the RNase compared to typical type I IRE1α kinase inhibitors.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The small size of <b>2</b> compared to reported IRE1α kinase-RNase inhibitors from type II- or type I-extended scaffolds (e.g., KIRA3 and compound <b>16</b>, respectively; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) was also attractive as a potential starting point for inhibitor optimization.<a onclick="showRef(event, 'ref21 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref23 ref24">(21,23,24)</a> We hypothesized that the orientation of <b>2</b> within the IRE1α adenosine 5′-triphosphate (ATP) site would be similar to that determined for <b>1</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z7H">4Z7H</a>).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> We anticipated that the key hinge-binding hydrogen bond from N-1 of the imidazo[1,2-<i>b</i>]pyridazine would be maintained and that a new hydrogen bond could be formed to the hinge region from the 8-amino group, replacing the CH···O interaction seen for <b>1</b>. In this orientation, the indazole substituent would be directed toward the αC-helix, taking the place of the benzamide moiety in <b>1</b>, while the cyclopropylmethyl group would be directed out toward solvent. The imidazo[1,2-<i>b</i>]pyridazin-8-amine hinge-binding group has recently been shown to adopt such a binding mode to the kinase, monopolar spindle 1 (MPS1).<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. In Vitro Binding and Inhibition of IRE1α Kinase and RNase Activities by <b>2</b>, <b>5</b>–<b>12</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">IRE1α S724 autophosphorylation IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IRE1α ATP-site binding IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">IRE1α RNase activity IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b><sup>6</sup></td><td class="colsep0 rowsep0" align="left">0.218 (±0.15)<sup>6</sup></td><td class="colsep0 rowsep0" align="left">1.18 (±0.17)</td><td class="colsep0 rowsep0" align="left">activator EC<sub>50</sub> 0.143<sup>6</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">0.303 (±0.15)</td><td class="colsep0 rowsep0" align="left">1.60 (±0.91)</td><td class="colsep0 rowsep0" align="left">8.90 (±0.21)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">n.d.<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">4.06 (±0.74)</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">0.89 (±0.14)</td><td class="colsep0 rowsep0" align="left">46% @10 μM<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">0.81 (±0.4)</td><td class="colsep0 rowsep0" align="left">0.84 (±0.42)</td><td class="colsep0 rowsep0" align="left">2.67 (±1.89)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">2.7 (±1.1)</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of recombinant G547 IRE1α kinase-endoribonuclease extended (KEN) domain pS274 autophosphorylation measured in dissociation-enhanced lanthanide fluorescence immunoassay (DELFIA) format, mean (±standard deviation (SD)) for <i>n</i> ≥ 3.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Inhibition of ATP-site LanthaScreen tracer binding to recombinant dephosphorylated G547 IRE1α KEN domain, mean (±SD) for <i>n</i> ≥ 3.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Inhibition of G547 IRE1α-dependent cleavage of a fluorescence resonance energy transfer (FRET)-labeled stem-loop RNA containing the XBP1 cleavage site, mean (±SD) for <i>n</i> ≥ 3.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">n.d. = not determined.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">±4%, <i>n</i> = 4.</p></div></div></div><div class="NLM_p">We used in vitro biochemical assays to measure compound binding and effects on IRE1α autophosphorylation and endoribonuclease activities, to investigate the structure–activity relationships around <b>2</b> and test the proposed binding model. Recombinant IRE1α protein corresponding to the cytoplasmic kinase-endoribonuclease extended (KEN) domain (G547 IRE1α) was produced separately in phosphorylated and dephosphorylated states, as confirmed by intact protein mass spectrometry (MS) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> We used our previously described DELFIA format to determine the inhibition of recombinant IRE1α S274 autophosphorylation.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> To measure compound binding to IRE1α, we developed a LanthaScreen Eu Kinase binding FRET assay, which monitored the displacement of an ATP-site tracer. To determine the allosteric effect of the kinase inhibitors on IRE1α RNase activity, we used a FRET-based assay to measure the cleavage of a short stem-loop RNA sequence derived from the endogenous cleavage site in XBP1u mRNA (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S2</a>).<a onclick="showRef(event, 'ref21 ref30'); return false;" href="javascript:void(0);" class="ref ref21 ref30">(21,30)</a></div><div class="NLM_p">We initially investigated the role of the indazol-5-yl substituent in <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The availability and correct positioning of hydrogen bond donor/acceptor functionality was critical, with neither the <i>N</i>-methyl (<b>5</b>) nor regioisomeric (<b>6</b>) analogues showing activity. Only weak inhibition of IRE1α autophosphorylation in the sensitive DELFIA format was observed for the indol-5-yl derivative (<b>7</b>). The benzimidazol-5-yl analogue (<b>8</b>) also inhibited IRE1α autophosphorylation but showed significantly weaker ATP-site binding in the LanthaScreen assay and no RNase inhibition. For the RNase assay, this may in part reflect the reduced potency of <b>8</b> compared to <b>2</b>. The weak potency for ATP-site binding assay was unexpected, but based on the promising inhibition of IRE1α autophosphorylation, we pursued further substitution of the benzimidazol-5-yl scaffold.</div><div class="NLM_p">We assumed that the indazol-5-yl substituent of <b>2</b> potentially interacted with the same region of the IRE1α active site as the benzamide of <b>1</b>, in particular with residues such as D711 from the DFG motif or K599. Compound <b>1</b> also interacted with a water molecule that was in contact with E612 from the αC-helix.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Therefore, we investigated 2-(<i>N</i>-phenyl)amino substitution of the benzimidazole <b>8</b>. The 2-(<i>N</i>-phenyl)aminobenzimidazole motif is a known isostere for the diaryl urea motif more commonly found in type II kinase inhibitors, as it is capable of mimicking the hydrogen-bonding pattern made by type II urea inhibitors in the inactive DFG-out conformation of the kinase domain.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The addition of aryl urea groups or their isosteres to convert type I kinase inhibitor scaffolds to type II molecules is a well-established approach.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Gratifyingly, we observed that the 2-(<i>N</i>-phenyl)aminobenzimidazole <b>9</b> had similar kinase site binding affinity to <b>2</b>, and improved on the inhibition of IRE1α RNase activity. Consistent with a role for the 2-(<i>N</i>-phenyl)aminoimidazole functionality in forming multiple hydrogen bonds, N-methylation at any position reduced inhibitory activity (see <b>10</b>–<b>12</b>).</div><div class="NLM_p">With two variations of the imidazo[1,2-<i>b</i>]pyridazin-8-amine core identified that led to RNase inhibition, the effects of varying the 8-amino group on potency and physicochemical properties were studied. The aqueous (aq) buffer solubilities of <b>2</b> (<i>K</i><sub>Sol</sub> = 0.8 μM) and <b>9</b> (<i>K</i><sub>Sol</sub> = 2.9 μM) were low, consistent with their measured lipophilicities (<b>2</b>, Log <i>D</i><sub>7.4</sub> = 3.4; <b>9</b>, Log <i>D</i><sub>7.4</sub> = 4.1). We therefore sought to introduce polar or ionizable groups to improve solubility and hypothesized that the vector defined by the cyclopropylmethyl group should be directed toward solvent. Importantly, studies leading to the IRE1α inhibitor compound <b>16</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) found that appending cyclic amine or cyclohexylamine substituents directed out of the ATP-binding pocket was essential for achieving high potency.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Our presumed binding mode of the core scaffolds <b>2</b> and <b>9</b> oriented the 8-amino substituent in a similar direction. We investigated a range of substituents, including cyclic amines analogous to those described in the development of compound <b>16</b>, with selected examples highlighted in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. In Vitro Binding and Inhibition of IRE1α Kinase and RNase Activities by <b>19</b>–<b>32</b> and Published Compound KIRA3</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">IRE1α S724 autophosphorylation IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IRE1α ATP-site binding IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">IRE1α RNase activity IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">0.19 (±0.11)</td><td class="colsep0 rowsep0" align="left">0.25 (±0.07)</td><td class="colsep0 rowsep0" align="left">0.48 (±0.19)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">0.77 (±0.13)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">0.83 (±0.19)</td><td class="colsep0 rowsep0" align="left">0.13<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">0.47 (±0.09)</td><td class="colsep0 rowsep0" align="left">0.87 (±0.24)</td><td class="colsep0 rowsep0" align="left">0.16<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">0.32 (±0.09)</td><td class="colsep0 rowsep0" align="left">0.11 (±0.04)</td><td class="colsep0 rowsep0" align="left">0.072 (±0.011)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">n.d.<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">0.15 (±0.05)</td><td class="colsep0 rowsep0" align="left">0.14 (±0.02)</td><td class="colsep0 rowsep0" align="left">0.033<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">0.19 (±0.01)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">0.11 (±0.01)</td><td class="colsep0 rowsep0" align="left">0.122 (±0.026)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">0.24 (±0.09)</td><td class="colsep0 rowsep0" align="left">0.087 (±0.022)</td><td class="colsep0 rowsep0" align="left">0.027 (±0.007)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">0.20 (±0.05)</td><td class="colsep0 rowsep0" align="left">2.0 (±0.94)</td><td class="colsep0 rowsep0" align="left">0.51<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">0.13 (±0.06)</td><td class="colsep0 rowsep0" align="left">0.67 (±0.23)</td><td class="colsep0 rowsep0" align="left">0.18<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">0.13 (±0.03)</td><td class="colsep0 rowsep0" align="left">0.39 (±0.11)</td><td class="colsep0 rowsep0" align="left">0.09 (±0.05)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">0.15 (±0.02)</td><td class="colsep0 rowsep0" align="left">0.33 (±0.04)</td><td class="colsep0 rowsep0" align="left">0.10 (±0.05)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">0.16 (±0.06)</td><td class="colsep0 rowsep0" align="left">0.27 (±0.17)</td><td class="colsep0 rowsep0" align="left">0.08 (±0.05)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">0.16 (±0.05)</td><td class="colsep0 rowsep0" align="left">0.48 (±0.05)</td><td class="colsep0 rowsep0" align="left">0.17 (±0.09)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KIRA3</td><td class="colsep0 rowsep0" align="left">0.29 (±0.06)</td><td class="colsep0 rowsep0" align="left">0.40 (±0.16)</td><td class="colsep0 rowsep0" align="left">0.45 (±0.10)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of recombinant G547 IRE1α KEN domain pS274 autophosphorylation measured in DELFIA format, mean (±SD) for <i>n</i> ≥ 3.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Inhibition of ATP-site LanthaScreen tracer binding to recombinant dephosphorylated G547 IRE1α KEN, mean (±SD) for <i>n</i> ≥ 3.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Inhibition of G547 IRE1α-dependent cleavage of a FRET-labeled stem-loop RNA containing the XBP1 cleavage site, mean (±SD) for <i>n</i> ≥ 3.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last"><i>n</i> = 2.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">Single determination.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last">n.d. = not determined.</p></div></div></div><div class="NLM_p">Introduction of polar groups generally maintained the dual kinase-RNase inhibitory profile of the compounds while enhancing potency. Thus, replacement of the cyclopropylmethyl substituent by tetrahydropyran-4-yl resulted in submicromolar inhibitors of IRE1α RNase function, <b>19</b> and <b>27</b>. Aqueous solubility remained low (<b>19</b>, <i>K</i><sub>Sol</sub> = 2.2 μM; <b>27</b>, <i>K</i><sub>Sol</sub> = 2.4 μM), but was increased in the basic <i>N</i>-methylpiperidin-4-yl derivatives <b>20</b> (<i>K</i><sub>Sol</sub> = 58 μM, Log <i>D</i><sub>7.4</sub> = 1.8) and <b>28</b> (<i>K</i><sub>Sol</sub> = 41 μM, Log <i>D</i><sub>7.4</sub> = 2.7), which also showed improvements in RNase inhibition.</div><div class="NLM_p">For the 3-(indazol-5-yl) substituted analogues, extension of the piperidine substituent had a negligible effect on compound affinity or potency (see <b>21</b>). However, the <i>trans</i>-1,4-diaminocyclohexane analogue <b>22</b> showed enhanced binding and RNase inhibition. Notably, <i>N</i>-8-methylation to give <b>23</b> led to loss of activity, consistent with the assumed kinase hinge-binding role of the 8-amino substituent. Further substitution of the <i>trans</i>-1,4-diaminocyclohexane group was tolerated, exemplified by <b>24</b>–<b>26</b>, and led to potent in vitro inhibitors of IRE1α RNase activity with acceptable aqueous solubility (e.g., <b>26</b>, <i>K</i><sub>Sol</sub> = 80 μM, Log <i>D</i><sub>7.4</sub> = 1.8).</div><div class="NLM_p">For the 3-(2-(<i>N</i>-phenyl)aminobenzimidazol-5-yl) substituted compounds, the (<i>N</i>-methylpiperidin-4-yl)methyl analogue <b>29</b> provided an enhancement in binding and RNase inhibitory potency without the need for further substitution. We examined fluorination of the benzimidazole group but found only marginal (<2-fold) gains in affinity and potency (see <b>30</b>–<b>32</b>). The most potent RNase inhibitor <b>31</b> from this scaffold showed moderate solubility (<i>K</i><sub>Sol</sub> = 38 μM), reflecting the high lipophilicity (Log <i>D</i><sub>7.4</sub> = 3.7).</div><div class="NLM_p">Cellular inhibition of IRE1α RNase function was measured for selected compounds using a bioluminescent reporter assay in HEK293 cells stably expressing an XBP1 luciferase fusion mRNA to quantify XBP1 mRNA splicing (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). In the absence of ER stress, translation of the XBP1u mRNA terminates before the luciferase sequence, while exogenous induction of ER stress leads to IRE1α-dependent XBP1 mRNA splicing and results in tandem translation of the XBP1s-luciferase reporter protein. The imidazo[1,2-<i>b</i>]pyridazine <b>2</b> moderately inhibited tunicamycin-induced mRNA splicing. Enhancement of the in vitro potency and aqueous solubility of the scaffold, to give <b>22</b> and <b>26</b>, was accompanied by increased cellular activity. Likewise, the most potent and soluble 3-(2-(<i>N</i>-phenyl)aminobenzimidazol-5-yl) substituted analogues <b>29</b> and <b>31</b> gave submicromolar inhibitors of IRE1α activity in cells. The optimization of in vitro potency in this series involved addition of basic groups, which may have impaired cell permeability and countered gains in potency for some of the more basic molecules. The cytotoxicity of compounds in the cell line was monitored to preclude interference in the assay, and no compound showed high cytotoxicity in the HEK293 cell line over the duration of the assay.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Cellular Activities of Selected IRE1α Inhibitors</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">XBP1 luciferase fusion mRNA splicing in HEK293 cells IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">cytotoxicity in HEK293 reporter cells EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">4.2 (±1.7)</td><td class="colsep0 rowsep0" align="left">>49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">2.9 (±0.87)</td><td class="colsep0 rowsep0" align="left">28 (±6.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">1.12 (±0.24)</td><td class="colsep0 rowsep0" align="left">>49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">0.89 (±0.1)</td><td class="colsep0 rowsep0" align="left">12<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">0.71 (±0.33)</td><td class="colsep0 rowsep0" align="left">9.0 (±1.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KIRA3</td><td class="colsep0 rowsep0" align="left">0.76<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">>49</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of tunicamycin-induced IRE1α-dependent splicing of an XBP1u-luciferase mRNA reporter stably expressed in HEK293 cells, mean (±SD) for <i>n</i> ≥ 2.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Cytotoxicity in HEK293 cells stably expressing an XBP1u-luciferase mRNA reporter, measured using the Alamar Blue format, mean (±SD) for <i>n</i> ≥ 2.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Single determination.</p></div></div></div><div class="NLM_p">Compounds <b>2</b>, <b>26</b>, and <b>31</b> were profiled for broad kinase selectivity, as determined by inhibition of probe binding to recombinant human protein and lipid kinase domains at a concentration of 1 μM (KINOME<i>scan</i>, Eurofins; <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Table S1</a>). Compound <b>2</b> gave >70% inhibition of 26/455 wild-type and mutant kinases tested. A distinct selectivity pattern was seen, with specific CMGC (CLK1, CLK2, CLK4, DYRK1A, DYRK1B, HIPK4) and TK (FLT3 wild type and mutants, KIT wild type and mutants, PDGFRB) subfamily members showing particularly high inhibition (>90%). The more potent derivative <b>26</b> maintained a very similar selectivity pattern, with additional kinases in the CMGC (DYRK2, HIPK1, HIPK2, HIPK3) and TK (PDGFRA) subfamilies related to those inhibited by <b>2</b> showing potent (>90%) inhibition. Overall, 54/455 wild-type and mutant kinases tested showed >70% inhibition, consistent with the increased potency of <b>26</b>. In contrast, the 2-(<i>N</i>-phenyl)aminobenzimidazole <b>31</b> had a different selectivity pattern and very high selectivity with >70% inhibition of only 4/455 kinases. While two of the CMGC kinases inhibited by <b>2</b> and <b>26</b> (CLK1 and CLK4) were inhibited to a lesser extent by <b>31</b>, the most potently inhibited kinase was another CGMC family member PCTK2 (97% inhibition at 1 μM test concentration). Interaction with wild-type and mutant TK subfamily kinases was minimal for <b>31</b>. The inhibitors <b>26</b> and <b>31</b> were, respectively, 9- and 100-fold selective for IRE1α over the IRE1β isoform, which has 80% sequence identity in the kinase domain,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> in parallel LanthaScreen binding assays (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S3</a>).</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding profiles of compounds <b>2</b>, <b>26</b>, and <b>31</b> tested at 1 μM concentrations of inhibitors against 455 wild-type and mutant human protein and lipid kinases (KINOME<i>scan</i>, Eurofins) (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Table S1</a>). Kinases where probe binding was inhibited by >90% at 1 μM test compound concentration are listed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To better understand the origins of the potency and high selectivity of the inhibitors, we determined the X-ray structure of <b>2</b> bound to the cytosolic kinase-endoribonuclease (KEN) domain of IRE1α (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>; <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). The complex <b>2</b>-IRE1α crystallized as a monomer in the unit cell. The kinase domain adopted an unusual conformation in which the αC-helix was remarkably disordered such that electron density was not modeled for most of this important secondary structural component. The DFG motif occupied an intermediate position between the canonical DFG-in and DFG-out conformations, which we term “DFG-up” for convenience. Thus, F712 was displaced from the “in” position that characterizes the active conformation of the regulatory hydrophobic spine of the kinase. A related intermediate conformation of the DFG motif has been observed in complexes of ligands bound to mutated Aurora A kinases.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> In the current structure, the residue Y628, also part of the hydrophobic spine in IRE1α, was directed down toward the DFG motif, forming hydrogen bonds to D711 (2.7 Å) and the bound ligand. This contrasted with the upward orientation of Y628 seen when the regulatory spine is intact and the IRE1α kinase domain is in the active conformation.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The DFG-up, Y628-down conformation of <b>2</b>-IRE1α was accompanied by an unusual folding of the activation loop in an antiparallel double β-strand across the face of the ATP site, forming a region of β-sheet with the P-loop that enclosed the bound ligand. The residue H723 in the activation loop appeared to anchor this folding, with direct hydrogen bonding to E651 (2.8 Å) in the kinase domain C-terminus and a water-mediated hydrogen bond observed between H723 and L577 in the P-loop. Folding of the activation loop across the active site is an autoinhibitory mechanism for some kinases. For example, structures of imidazo[1,2-<i>b</i>]pyridazines and other ligands bound to the kinase MPS1 show a single strand of the activation loop folded across the active site parallel to the P-loop, potentially blocking the interaction with substrate peptides.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> However, in these cases, the kinase adopts a DFG-in conformation on ligand binding and the αC-helix is preserved.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of IRE1α inhibitor binding modes. (A) Part of the crystal structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>) of <b>2</b> (green sticks) bound to IRE1α KEN domain (gray image) showing the conformations of the DFG motif (cyan), activation loop segment (pale cyan), Y628 (blue), K599 (gray), and the observed part of the αC-helix (red). (B) Binding of <b>2</b> (green sticks) in the IRE1α ATP site (gray sticks) with key residues labeled. Hydrogen bonds are shown as black dotted lines and selected water molecules as red spheres. (C) Details of the interactions of <b>2</b> (green sticks) with the DFG motif and Y628 in the IRE1 ATP site (gray image) with key residues labeled. Hydrogen bonds are shown as black dotted lines and selected water molecules as red spheres. (D) Comparison of <b>2</b> (purple), <b>1</b> (blue; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z7H">4Z7H</a>), Compound <b>16</b> (light blue; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4U6R">4U6R</a>), and GSK2850163 (red; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YZ9">4YZ9</a>) binding to IRE1α KEN domain. Structures are overlaid on the hinge region (E643-L644-C645; lines), and the αC-helix (L602-S619; cartoon) is shown in the three structures in which it is present. (E) Schematic overlay showing the relationship between the DFG-out, DFG-up, and DFG-in conformations and the corresponding orientations of the observed activation loop segments (L714-S729) for GSK2850163-IRE1α (red), <b>2</b>-IRE1 (purple), compound <b>16</b>-IRE1α (light blue), and <b>1</b>-IRE1α (blue). (F) Comparison of components of the N-terminal kinase domain dimerization interface for the DFG-in, active conformation of <b>1</b>-IRE1α (blue and blue-white protomers) and DFG-up, and inactive conformation of <b>2</b>-IRE1α (purple). Structures are aligned on the kinase C-lobe (shown) and endonuclease C-terminal domain (not shown). Bound ligands are rendered as sticks; (G) model of <b>9</b> (green sticks) bound to IRE1 (gray image) based on the conformation observed for <b>2</b>-IRE1. Predicted hydrogen bonds are shown as black dotted lines. (H) Detail showing the movement of Y628 as positioned in <b>2</b>-IRE1 (gray image) required to accommodate the modeled binding of <b>9</b> (green sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Within the unusual kinase ATP-site conformation, the ligand <b>2</b> bound at the hinge region, with the imidazo[1,2-<i>b</i>]pyridazine ring sandwiched between L714 in the activation loop and V586 in the β2-strand, C-terminal to the P-loop (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Canonical hydrogen bonds were made from the imidazo[1,2-<i>b</i>]pyridazine N1 and 8-NH to the C645 NH (3.1 Å) and C═O (3.0 Å) functionalities, respectively. This closely resembled the binding modes described for 8-amino-imidazo[1,2-<i>b</i>]pyridazines in other kinases.<a onclick="showRef(event, 'ref28 ref29 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref37 ref38">(28,29,37,38)</a> An additional hydrogen bond (2.8 Å) was observed between the indazole NH and Y628, presumably contributing to stabilizing the tyrosine “down” conformation. The indazole occupied a hydrophobic pocket formed by the gatekeeper residue I642, F712 of the DFG motif, F725 from the activation loop, and the catalytic lysine K599. The 6-chloro substituent of <b>2</b> packed closely against the β-sheet surface formed by L577 and G578 of the P-loop and H723 and S724 of the activation loop. The cyclopropylmethyl side chain of <b>2</b> reached the edge of the ATP site and was oriented to provide a vector out into solvent.</div><div class="NLM_p">The unusual binding mode seen for <b>2</b>-IRE1α was independently reproduced with a close analogue, <b>33</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HV0">6HV0</a>), in which the cyclopropylmethyl group was replaced by an isopropyl chain (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S4</a>), supporting the likely biological relevance of the structure. Furthermore, the observed binding mode was consistent with the structure–activity relationships described above for the indazole-substituted 8-amino-imidazo[1,2-<i>b</i>]pyridazines, specifically the requirement for a hydrogen bond donor corresponding to the indazole N(1)–H (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and the tolerance of a wide range of substitution of the 8-amino group on the imidazo[1,2-<i>b</i>]pyridazine core (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Modeling of the more potent compound <b>26</b> into the conformation of IRE1α observed in the crystal structures reproduced this binding mode, with the suggestion of an additional hydrogen bond from the cyclohexylamine to E651 on the surface of the protein (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p">The unusual DFG-up inactive conformation of the kinase domain of <b>2</b>- and <b>33</b>-IRE1α differed considerably from other active and inactive kinase conformations reported for ligand-bound structures of IRE1α (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D).<a onclick="showRef(event, 'ref4 ref6 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref4 ref6 ref22 ref23">(4,6,22,23)</a> We previously described the IRE1α kinase inhibitor-RNase activator <b>1</b> that stabilized a classical DFG-in, active conformation of the kinase with the activation loop extended away from the active site, mimicking IRE1α autophosphorylation and activating the RNase function through competent dimerization of the endonuclease domain.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In contrast, a large sulfonamide inhibitor (named compound <b>16</b>), although it also bound to a DFG-in conformation of the kinase, was found to cause a shift of the αC-helix that was incompatible with the formation of the RNase domain dimer.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> A type III inhibitor of IRE1α, GSK2850163, has been described that occupied a pocket adjacent to the hinge region of the kinase and promoted a DFG-out kinase conformation, with the activation loop directed across the active site and the αC-helix further shifted and rotated away from the active conformation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In the latter two cases, although the active kinase conformation was disrupted, the αC-helix remained intact. For <b>2</b> and <b>33</b>, our data showed a striking disruption of the αC-helix, as well as a distinctive refolding of the activation loop around the bound inhibitor. This conformation is incompatible with the back-to-back dimerization required to form an active endoribonuclease domain dimer (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). The dimerization interface in the kinase domain is formed by the loops β(−1)−β(0) and β(2)−β(3) from each monomer.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Compared to the active IRE1α dimer structure represented by <b>1</b>-IRE1α, the binding of <b>2</b> and <b>33</b> shifted the position of the kinase domain relative to the endonuclease domain such that dimerization at the kinase domain would lead to steric clashes of the endoribonuclease domains.</div><div class="NLM_p">We were unable to obtain a crystal structure of a 2-aminobenzimidazole-substituted analogue bound to IRE1α. Modeling of <b>9</b>, the closest analogue to <b>2</b>, into the structure of <b>2</b>-IRE1 showed that alignment of the imidazo[1,2-<i>b</i>]pyridazine cores of the two inhibitors resulted in a clash between the 2-aminobenzimidazole substituent and residue Y628. When Y628 was allowed to move from the “down” orientation observed in <b>2</b>-IRE1 to the “up” conformation observed in other IRE1α structures, this minimal change resulted in a satisfactory binding mode for the prototypical 2-(<i>N</i>-phenyl)aminobenzimidazole <b>9</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). The model retained binding of the imidazo[1,2-<i>b</i>]pyridazine at the hinge region and proposed a 180° rotation of the 2-aminobenzimidazole relative to the indazole substituent of <b>2</b> such that a new hydrogen bond was formed between the 2-amino group and D711 of the DFG motif. The terminal phenyl group occupied the space vacated by Y628 and packed against this residue now in its upward conformation. The introduction of a hydrogen bond to the DFG motif was consistent with precedent for other 2-aminobenzimidazole kinase inhibitors, where this functionality is often an isostere for urea-based inhibitors that bind to DFG-out conformations, as we had originally speculated in developing the series. Moreover, the modeled hydrogen bond to D711 was consistent with the observed structure–activity relationships, where methylation of the 2-amino group or imidazole nitrogens reduced activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The combination of multiple kinase conformational disruptions accompanying binding of the indazole and aminobenzimidazole scaffolds can account for the high IRE1α kinase selectivity of <b>26</b> and <b>31</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). For <b>26</b>, a discrete set of kinases in the CAMK and TK subfamilies were also inhibited. For this ligand, we cannot exclude the possibility that an alternative binding mode where the indazole contributes the donor–acceptor hinge-binding motif may be accessible in some kinases.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> Interestingly, while the majority of residues forming the binding site of <b>2</b> and <b>33</b> to IRE1α are identical in the sequence of the isoform IRE1β, the histidine H723 in IRE1α is replaced by a cysteine residue (C671) in IREβ (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S3B</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Since hydrogen bonding of the imidazole side chain of H723 to E651 appeared important for the unusual activation loop fold in the structures <b>2</b> and <b>33</b> bound to IRE1α, we speculate that the absence of this residue may in part be involved in the observed selectivity of the compounds for IRE1α over IRE1β. The other major local difference (A646 in the hinge corresponds to R594 in IRE1β<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a>) involves a side chain directed away from the binding site.</div><div class="NLM_p">To confirm the functional consequences of the unusual binding modes of the inhibitors, we investigated the effects of <b>26</b> and <b>31</b> on ER stress-induced IRE1α signaling outputs in human cells. The effects of the compounds on IRE1α oligomerization were determined in HEK293 cells expressing doxycycline-inducible green fluorescent protein (GFP)-labeled IRE1α that forms quantifiable fluorescent foci in the ER membrane under stress conditions<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). GFP-IRE1α foci produced upon treatment with tunicamycin were dose-dependently reduced by <b>26</b> (IC<sub>50</sub> = 1.88 ± 0.09 μM) and <b>31</b> (IC<sub>50</sub> = 0.74 ± 0.03 μM), indicating inhibition of IRE1α oligomer formation. No stimulation of GFP-IRE1α foci formation was observed for either compound on its own. Thus, the compounds are full inhibitors of oligomerization in cells and do not show partial activation of IRE1α through occupation of the nucleotide-binding site.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compounds <b>26</b> and <b>31</b> inhibit IRE1α oligomerization, IRE1α autophosphorylation, IRE1α RNase activity, XBP1s protein expression, and XBP1-dependent transcription in human cells. (A) Inhibition of tunicamycin-induced IRE1α oligomerization, measured by fluorescent foci formation in HEK293 cells stably transfected with doxycycline-inducible GFP-IRE1α after 5 h treatment with inhibitors (0–10 μM) in the absence or presence of tunicamycin (quantification of image fields from <i>n</i> = 2 experiments plotted separately). Dox = doxycycline, T = tunicamycin (10 μg/mL). (B) Inhibition of tunicamycin-induced pS724 IRE1α autophosphorylation as measured by capillary electrophoresis immunoassay (simple Western) relative to total IRE1α. Data shown for a single experiment representative of <i>n</i> = 3. (C) Inhibition of tunicamycin-induced XBP1s protein expression in H929 cells as measured by immunofluorescent assay (quantification of image fields from <i>n</i> > 3 experiments). (D) Inhibition of tunicamycin-induced XBP1s-dependent transcription of DNAJB9 mRNA as measured by real-time quantitative polymerase chain reaction (RT-qPCR). Data shown for a single experiment representative of <i>n</i> = 3. Tm = tunicamycin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Compounds <b>26</b> and <b>31</b> inhibited both tunicamycin- and thapsigargin-induced IRE1α-dependent splicing of XBP1 luciferase fusion mRNA in HEK293 cells (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S6</a>) with equivalent potencies (IC<sub>50</sub> 0.68–1.63 μM). In parallel, inhibition of tunicamycin-induced production of endogenous XBP1s mRNA was demonstrated in H929 myeloma cells using RT-qPCR at similar concentrations (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S7</a>). The expression of the spliced transcription factor XBP1s following ER stress was measured by immunofluorescent staining in H929 myeloma cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). Tunicamycin-induced expression of XBP1s protein was inhibited by <b>26</b> and <b>31</b> with similar potencies to the inhibition of IRE1α oligomerization and RNase activities in cells. The expected downstream effect on XBP1s-dependent transcription in H929 cells was confirmed by monitoring production of the mRNA coding for DNAJB9, an ER-resident molecular chaperone regulated by XBP1s<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). Inhibition of IRE1α autophosphorylation in H929 cells was also observed on treatment with <b>26</b> and <b>31</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">S8</a>), although full blockade appeared to require higher concentrations than the oligomerization and RNase functions. This is consistent with the proposed model for IRE1α activation, where conformationally productive oligomerization is the critical step in activating the endoribonuclease,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> while autophosphorylation contributes to stabilizing the active oligomers<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and XBP1-independent signaling to JNK through the binding of TRAF2.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Previous studies by our group<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and others<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> indicate that the trans-autophosphorylation of IRE1α proceeds through a face-to-face encounter of IRE1α protomers distinct from the back-to-back arrangement essential for activation of the RNase function.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Through screening analogues of a type I IRE1α kinase inhibitor that activates the RNase function through binding to a classical DFG-in kinase conformation, we unexpectedly discovered a closely related series of imidazo[1,2-<i>b</i>]pyridazin-8-amine kinase inhibitors that also inhibited the RNase. This surprising finding challenged our initial assumption that type II or type III kinase inhibitors binding to a DFG-out conformation of the kinase would be needed to prevent IRE1α dimerization and activation of the RNase.</div><div class="NLM_p">We increased the potency of the imidazo[1,2-<i>b</i>]pyridazin-8-amines using design hypotheses that assumed a type I binding mode of the core scaffold, leading to a series of indazoles, while further substitution generated 2-phenyl-benzimidazoles designed as potential type II inhibitors. However, crystal structures of the indazoles <b>2</b> and <b>33</b> bound to IRE1α showed the protein to adopt a highly unusual conformation, with the DFG motif in an intermediate “up” position and substantial disordering of the αC-helix. These structures and models derived from them accounted for the structure–activity relationships of the inhibitors, showing how a type I scaffold could promote a previously unreported conformation of the kinase domain that is incompatible with RNase domain back-to-back dimerization required for activation of the RNase function. The binding mode emphasized the role of the hydrophobic spine residue Y628 of IRE1α in compound binding and kinase conformational control.<a onclick="showRef(event, 'ref6 ref42'); return false;" href="javascript:void(0);" class="ref ref6 ref42">(6,42)</a></div><div class="NLM_p">Other kinase signaling pathways have been observed to be modulated through transactivation of dimers controlled allosterically by kinase site occupation.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Notably, ligand-promoted heterodimerization of inhibitor-bound BRAF with CRAF results in paradoxical activation of RAF signaling by some BRAF inhibitors.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> Kinase site inhibitors that can simultaneously suppress the catalytic function of kinase monomers and their dimerization with uninhibited partners are therefore a potentially useful therapeutic modality.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Certain pseudokinases that lack intrinsic enzyme function also exploit dimerization to regulate signaling pathways and may be amenable to pharmacological intervention through nucleotide-binding-site ligands that prevent dimerization.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><div class="NLM_p">The most advanced compound we studied showed nanomolar potency for biochemical inhibition of the kinase and RNase and, importantly, was highly selective for IRE1α over other kinases. It is possible that the high selectivity of <b>31</b> reflects binding to the unusual IRE1α kinase conformation incorporating multiple disruptions, as modeled for the analogue <b>9</b>. Although elements of the conformational disruption of the IRE1α kinase domain have been seen separately in other contexts,<a onclick="showRef(event, 'ref22 ref23 ref28 ref29 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref28 ref29 ref35 ref36">(22,23,28,29,35,36)</a> to our knowledge, they have not been observed occurring simultaneously. Our findings thus increase the repertoire of conformations discovered for IRE1α and provide a means to achieve high specificity for IRE1α inhibition over other kinases, with reliable translation of kinase ATP-site occupation into RNase inhibition.</div><div class="NLM_p last">We confirmed that compounds engaging the newly discovered IRE1α conformation resulted in the inhibition of ER stress-induced signaling through IRE1α in multiple human cell types, resulting in inhibition of XBP1-dependent transcription. We determined substantial inhibition of the IRE1α signaling pathways to occur in cells at concentrations of <b>26</b> and <b>31</b> between ca. 1 and 2 μM in short-term assays. Selective IRE1α kinase-endoribonuclease inhibitors may have utility in situations where direct cytotoxicity is not the desired outcome of IRE1α-XBP1s inhibition, such as the modulation of XBP1s-dependent suppression of dendritic immune cell function. The identification of compounds <b>26</b> and <b>31</b> and their binding mode demonstrates a new means to achieve highly selective inhibition of IRE1α, which may be important in such contexts where the biology of untransformed host cells is targeted.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88834" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88834" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> IRE1α Autophosphorylation Assay</h3><div class="NLM_p last">Inhibition of IRE1α autophosphorylation was measured using a dissociation-enhanced lanthanide fluorescence immunoassay (DELFIA) as described previously.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Briefly, dephosphorylated IRE1α G547-L977 (700 nM) was incubated for 25 min with 100 μM ATP in 15 μL of assay buffer (40 mM Tris (pH 7.5), 20 mM MgCl<sub>2</sub>, and 1 mM 1,4-dithiothreitol (DTT)). The assay was stopped by the addition of 40 mM ethylenediaminetetraacetic acid. Samples were transferred to a 384-well high-binding plate and incubated overnight at 4 °C. The plates were washed three times in 0.1% Tween-20, followed by blocking in 5% nonfat skimmed milk in phosphate-buffered saline (PBS) for 30 min at 37 °C. The plates were washed again before application of α-pS724 phospho-specific IRE1α primary antibody<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (160 pg/mL) in PBS to each well and incubated for 1.5 h at 37 °C. The plates were washed before addition of a europium-labeled α-rabbit secondary antibody (PerkinElmer Life Sciences). Another wash step was followed by the addition of DELFIA enhancement solution (PerkinElmer Life Sciences). Assay plates were read on an EnVision multimode plate reader (PerkinElmer Life Sciences).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> IRE1α RNase Assay</h3><div class="NLM_p last">Inhibition of IRE1α endoribonuclease activity was measured using a FRET de-repression assay monitoring cleavage of a 29-nucleotide stem-loop RNA containing the XBP1 cleavage site sequence and labeled with a fluorescence emitter (fluorescein amidite (FAM)) and a fluorescence quencher (Black Hole Quencher (BHQ)) at the 5′ and 3′ ends, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Figure S2</a>), as described previously.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Briefly, varying volumes of compound in DMSO or DMSO alone were added to a low-volume 384-well plate (3676, Corning) to give final concentrations ranging from 100 μM to 0.313 nM using an Echo acoustic liquid dispenser (Labcyte, CA). Nonphosphorylated IRE1α G547-L977 was added to a final concentration of 200 nM. After incubating for 30 min at 30 °C, hairpin RNA XBP1 substrate mimic labeled with fluorescein and Black Hole Quencher (5′ FAM-GAACAAGAUAUCCGCA-GCAUAUACAGUUC-3′ BHQ, Eurofins MWG Operon, Germany) was added to 100 nM final concentration. After incubation for a further 15 min at 30 °C, the fluorescein fluorescence was measured on an EnVision multimode plate reader (PerkinElmer Life Sciences). Fluorescence in the presence of compound was expressed relative to that of DMSO alone (no compound).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> IRE1α ATP-Site Binding Assay</h3><div class="NLM_p last">Compound binding to IRE1α was measured using a LanthaScreen Eu Kinase binding FRET assay monitoring the displacement of an ATP-site tracer by inhibitors. Briefly, a master mix of (7.5 μL) 50 nM dephosphorylated IRE1α G547-L977, 2 nM biotinylated anti-his-tag antibody (PV6089, Thermo Fisher Scientific), 2 nM streptavidin–europium (PV5899, Thermo Fisher Scientific) diluted in assay buffer (PV3189, Thermo Fisher Scientific), and 1 mM 1,4-dithiothreitol (DTT) (D0632, Sigma-Aldrich) was incubated with 7.5 μL of 75 nM tracer (Tracer 236, Thermo Fisher Scientific) in assay buffer for 60 min at rt. During the incubation, the 384-well white low-volume plate (3674, Corning) was covered with foil. Assay plates were read on an EnVision multimode plate reader (PerkinElmer Life Sciences).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> X-ray Crystallography of <b>2</b>-IRE1α (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>)</h3><div class="NLM_p last">Determination of the structure of <b>2</b>-IRE1α was conducted by Proteros Biostructures GmbH (Martinsried, Germany). Expression, purification, and crystallization of IRE1α with compound were performed by analogy to previously reported protocols.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Crystals were flash-frozen and data were collected at 100 K at the Swiss Light Source (PXI/X06SA, PILATUS 6 M detector; SLS, Villigen, Switzerland) using cryogenic conditions. Data were processed using the programs XDS and XSCALE.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The structure of apo-IRE1α was solved by molecular replacement using a previously solved structure of IRE1α as a search model. Subsequent model building and refinement was performed according to standard protocols with the software packages CCP4<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and COOT.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> For the calculation of the free <i>R</i>-factor, 5.4% of measured reflections were excluded from the refinement procedure (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Table S2</a>). The ligand parameterization and generation of the corresponding library files were carried out with CORINA (Molecular Networks, GmbH). The water model was built with the “Find waters” algorithm of COOT by placing water molecules in peaks of the Fo–Fc map contoured at 3.0, followed by refinement with REFMAC5<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and checking all waters with the validation tool of COOT. The criteria for the list of suspicious waters were: <i>B</i>-factor greater than 80 Å<sup>2</sup>, 2Fo–Fc map less than 1.2σ, and distance to the closest contact less than 2.3 Å or more than 3.5 Å. The suspicious water molecules and those in the ligand binding site (distance to ligand less than 10 Å) were checked manually. The occupancy of side chains in negative peaks in the Fo–Fc map (contoured at −3.0σ) was set to zero and subsequently to 0.5 if a positive peak occurred after the next refinement cycle.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> In Silico Chemistry</h3><div class="NLM_p">The enzyme–inhibitor crystal structure of <b>2</b>-IRE1α (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>) was prepared for modeling using Protein Preparation Wizard in Maestro,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and all water molecules, apart from that interacting with residues L577 and H723, were removed. To propose predicted binding modes of the inhibitors, Glide (grid-based ligand docking with energetics)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> was used for the docking experiments. The receptor grid (model A; for <b>26</b> bound to IRE1α) was defined by a grid box of 25 × 25 × 25 Å<sup>3</sup> with a default inner box (10 × 10 × 10 Å<sup>3</sup>) centered on the ligand <b>2</b> in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>. For the “Y628-up” conformation (model B; for <b>9</b> bound to IRE1α), the side-chain orientation of residue Tyr628 in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a> was flipped from the observed rotamer to the most probable rotamer without steric clashes according to the Schrödinger in-built rotamer library, energy-minimized, and a Glide receptor grid subsequently prepared as described above.</div><div class="NLM_p">Ligands were prepared using LigPrep<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> applying the OPLS_2005 force field with possible tautomeric and ionization states in the pH range 5.0–9.0 generated. Using Glide Extra Precision (XP) settings, flexible docking of ligands was conducted with the minimal constraints defined as satisfying at least one H-bonding interaction with either residue R643 or C645 (kinase hinge region interactions), analogous to the interactions observed in the structure of <b>2</b>-IRE1α. Model A was used for the binding mode predictions of <b>26</b>, whereas model B was applied to predict the binding mode of <b>9</b>.</div><div class="NLM_p last">Hit compound <b>2</b> was analyzed for potential pan-assay interference behaviors using open access in silico tools (<a href="http://zinc15.docking.org/patterns/home" class="extLink">http://zinc15.docking.org/patterns/home</a>), and no substructural alerts were returned. Specific inhibition of IRE1α kinase was demonstrated in two independent assay formats for <b>2</b> and active analogues (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Synthetic Chemistry</h3><div class="NLM_p">Reactions were carried out under N<sub>2</sub>. Organic solutions were dried over MgSO<sub>4</sub> or Na<sub>2</sub>SO<sub>4</sub>. Starting materials and solvents were purchased from commercial suppliers and were used without further purification. Reactions heated by microwave irradiation were carried out using a Biotage Initiator microwave reactor. Reactions in sealed tubes were carried out in pressure vessels with shield protection. Ion-exchange chromatography was performed using ISOLUTE Flash SCX-II (acidic) or Flash NH2 (basic) resin cartridges. Silica column chromatography was performed using Biotage SP1 or Isolera medium-pressure chromatography systems, using prepacked silica gel cartridges (normal phase, Biotage SNAP KP-Si; reverse phase, Biotage SNAP Ultra C18). Preparative high-performance liquid chromatography (HPLC) was carried out at rt using a 1200 Series Preparative HPLC (Agilent, Santa Clara) over a 15 min gradient elution from 60:40 to 0:100 water/MeOH (both modified with 0.1% formic acid) at flow rates of 5, 20, or 40 mL/min depending on the column size used. Standard injections of 500 μL to 2 mL (with needle wash) of the sample were made onto ACE 5 C18-PFP columns (5 μm, 250 × 10/250 × 21.2/250 × 30 mm<sup>2</sup>, Advanced Chromatography Technologies, Aberdeen, U.K.). UV–vis spectra were acquired at 254 nm on a 1200 Series Prep Scale diode array detector (Agilent, Santa Clara). NMR spectra were recorded on Bruker AMX500 or AV600 instruments using internal deuterium locks. Chemical shifts (δ) are reported relative to tetramethylsilane (δ = 0) and/or referenced to the solvent in which they were measured. Compounds were assessed for purity by tandem HPLC–MS. Combined HPLC–MS analyses were recorded using an Agilent 6210 time-of-flight (ToF) HPLC–MS with a Merck Purospher STAR column (RP-18e, 30 × 4 mm<sup>2</sup>) or a Waters Xevo G2QToF HPLC–MS with a Phenomenex Kinetex C18 column (30 × 2.1 mm<sup>2</sup>, 2.6 μm, 100 A). Analytical separation was carried out at 40 °C with UV detection at 254 nm, and ionization was by positive-ion electrospray. The mobile phase was a mixture of MeOH (solvent A) and water (solvent B), both containing formic acid at 0.1%. Gradient elution from 10:90 (A/B) to 90:10 (A/B) was carried out over 2 min (flow rate: Agilent, 3 mL/min; Xevo, 0.5 mL/min) or 4 min (flow rate: Agilent, 1.5 mL/min; Xevo, 0.3 mL/min). Biologically evaluated compounds gave >95% purity as determined by these methods (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Table S2</a>).</div><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>2</b>)</h4><div class="NLM_p last">4-Bromo-6-chloropyridazin-3-amine (10.0 g, 48.1 mmol) and 2-chloroacetaldehyde (31.8 mL, 250 mmol; 50% w/w aqueous solution) in EtOH (67 mL) were stirred at 50 °C for 22 h. The mixture was cooled to rt, concentrated, and partitioned between CH<sub>2</sub>Cl<sub>2</sub> (300 mL) and saturated aqueous NaHCO<sub>3</sub> (200 mL). The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (150 mL). The combined organic layers were dried, filtered, and evaporated to dryness to give 8-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazine (11.2 g, quantitative) that was taken to the next step without further purification. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.01 (d, <i>J</i> = 1.2 Hz, 1H), 7.84 (d, <i>J</i> = 1.1 Hz, 1H), 7.40 (s, 1H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 145.9, 137.0, 134.8, 124.2, 121.2, 118.8; liquid chromatography mass spectrometry (LCMS) (2 min) electrospray ionization (ESI), (<i>m</i>/<i>z</i>) <i>t</i><sub>R</sub> = 1.04 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 233 (M + H<sup>+</sup>). <i>N</i>-Bromosuccinimide (5.17 g, 29.0 mmol) was added to a solution of 8-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazine (4.50 g, 19.4 mmol) in CHCl<sub>3</sub> (59 mL) at 0 °C. The mixture was warmed to rt and stirred for 18 h. The mixture was partitioned between saturated aqueous NaHCO<sub>3</sub> (150 mL) and CH<sub>2</sub>Cl<sub>2</sub> (100 mL × 2). The combined organic layers were dried, filtered, and evaporated to dryness. Silica column chromatography, eluting with 30% EtOAc–hexanes, gave 3,8-dibromo-6-chloroimidazo[1,2-<i>b</i>]pyridazine (<b>3</b>) (5.05 g, 84%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.85 (s, 1H), 7.46 (s, 1H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 147.0, 137.5, 135.3, 124.3, 121.4, 103.2; LCMS (2.0 min) <i>t</i><sub>R</sub> = 1.36 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 311 (M + H<sup>+</sup>). Cyclopropanemethylamine (4.82 mL, 56.2 mmol) was added to a stirred solution of <b>3</b> (3.50 g, 11.24 mmol) in THF (56 mL) at rt. After 3 h, the mixture was evaporated and partitioned between saturated aqueous NaHCO<sub>3</sub> (200 mL) and CH<sub>2</sub>Cl<sub>2</sub> (100 mL × 2). The combined organic layers were washed with brine (100 mL), dried, and evaporated to dryness. Silica column chromatography, eluting with 15–20% EtOAc–hexanes, gave 3-bromo-6-chloro-<i>N</i>-(cyclopropylmethyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>4</b>) (3.20 g, 93%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.49 (s, 1H), 6.04 (s, 1H), 6.00 (br t, <i>J</i> = 5.3 Hz, 1H), 3.17 (dd, <i>J</i> = 5.2, 7.1 Hz, 2H), 1.20–1.17 (m, 1H), 0.69–0.66 (m, 2H), 0.36–0.33 (m, 2H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 149.5, 142.9, 133.1, 130.9, 101.1, 92.2, 47.8, 10.0, 3.8 (2C); LCMS (2.0 min) <i>t</i><sub>R</sub> = 1.63 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 303 (M + H<sup>+</sup>). N<sub>2</sub> was bubbled for 10 min through a mixture of <b>4</b> (0.096 g, 0.318 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole (0.121 g, 0.350 mmol), and aq Na<sub>2</sub>CO<sub>3</sub> (2 M; 0.637 mL, 1.27 mmol) in 1,4-dioxane (1.59 mL). Pd(OAc)<sub>2</sub> (7.1 mg, 0.032 mmol) and 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene (0.017 g, 0.032 mmol) were added and the mixture was heated at 135 °C in a sealed tube for 22 h. The mixture was cooled and purified by ion-exchange chromatography on acidic resin (5 g), eluting with 2 M NH<sub>3</sub> in MeOH. The fractions containing product were concentrated and purified by silica column chromatography, eluting with 5% MeOH–CH<sub>2</sub>Cl<sub>2</sub>, to give <b>2</b> (0.047 g, 44%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.18 (s, 1H), 8.51 (t, <i>J</i> = 1.1 Hz, 1H), 8.19 (t, <i>J</i> = 1.2 Hz, 1H), 8.07–7.79 (m, 3H), 7.66 (dt, <i>J</i> = 8.9, 0.9 Hz, 1H), 6.34 (s, 1H), 3.32 (s, 2H), 1.23–1.14 (m, 1H), 0.51–0.45 (m, 2H), 0.35–0.27 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 147.6, 143.7, 139.1, 134.2, 132.9, 129.0, 128.8, 125.5, 122.9, 120.8, 118.4, 110.4, 90.6, 46.1, 10.2, 3.4 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.50 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 339 (M + H<sup>+</sup>); high-resolution mass spectrometry (HRMS) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>16</sub>N<sub>6</sub><sup>35</sup>Cl (M + H) 339.1125, found 338.1133.</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(1-methyl-1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>5</b>)</h4><div class="NLM_p last">A mixture of 1-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole (0.086 g, 0.333 mmol), <b>4</b> (0.084 g, 0.278 mmol), aqueous Na<sub>2</sub>CO<sub>3</sub> (2 M; 0.56 mL, 1.11 mmol), and Pd(OAc)<sub>2</sub> (6.23 mg, 0.028 mmol), and 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene (0.013 g, 0.028 mmol) in 1,4-dioxane (1.4 mL) was heated at 120 °C in a sealed tube for 22 h. The mixture was cooled, absorbed onto silica gel, and purified by silica column chromatography, eluting with 70% EtOAc–hexanes, to give <b>5</b> (0.046 g, 47%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.52 (dd, <i>J</i> = 1.6, 0.8 Hz, 1H), 8.17 (d, <i>J</i> = 1.0 Hz, 1H), 8.02–7.93 (m, 3H), 7.76 (dt, <i>J</i> = 9.0, 0.9 Hz, 1H), 6.34 (s, 1H), 4.09 (s, 3H), 3.31–3.17 (m, 2H), 1.21–1.15 (m, 1H), 0.51–0.44 (m, 2H), 0.35–0.26 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 148.1, 144.1, 139.3, 133.5, 133.4, 129.5, 129.1, 125.9, 124.0, 121.4, 118.9, 110.5, 91.1, 46.5, 35.9, 10.7, 3.9 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.56 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 353 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>N<sub>6</sub><sup>35</sup>Cl (M + H) 353.1281, found 353.1275.</div></div><div id="sec5_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(1<i>H</i>-indazol-6-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>6</b>)</h4><div class="NLM_p last">N<sub>2</sub> was bubbled for 10 min through a mixture of <b>4</b> (0.073 g, 0.242 mmol), 6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole (0.065 g, 0.266 mmol), and aqueous Na<sub>2</sub>CO<sub>3</sub> (2 M; 0.48 mL, 0.968 mmol) in 1,4-dioxane (1.21 mL). Pd(OAc)<sub>2</sub> (5.4 mg, 0.024 mmol) and 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene (0.013 g, 0.024 mmol) were added and the mixture was heated at 135 °C in a sealed tube for 15 h. The mixture was cooled and purified by ion-exchange chromatography on acidic resin (5 g), eluting with 2 M NH<sub>3</sub> in MeOH. The fractions containing product were concentrated and purified by silica column chromatography, eluting with 5% MeOH–CH<sub>2</sub>Cl<sub>2</sub>, to give <b>6</b> (0.042 g, 51%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.32–13.16 (m, 1H), 8.49 (q, <i>J</i> = 1.1 Hz, 1H), 8.12–8.08 (m, 2H), 8.03 (t, <i>J</i> = 6.1 Hz, 1H), 7.86 (dd, <i>J</i> = 8.5, 0.9 Hz, 1H), 7.71 (dt, <i>J</i> = 8.5, 1.0 Hz, 1H), 6.40 (d, <i>J</i> = 1.5 Hz, 1H), 3.34 (s, 2H), 1.24–1.14 (m, 1H), 0.53–0.45 (m, 2H), 0.35–0.28 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 147.7, 143.7, 140.0, 139.9, 133.5, 130.0, 128.3, 126.0, 121.9, 120.8, 119.5, 107.0, 90.9, 46.1, 10.2, 3.4 (2C); LCMS (4 min), <i>t</i><sub>R</sub> = 3.21 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 339 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>6</sub><sup>35</sup>Cl (M + H) 339.1119, found 339.1115.</div></div><div id="sec5_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(1<i>H</i>-indol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>7</b>)</h4><div class="NLM_p last">Prepared from <b>4</b> and (1<i>H</i>-indol-5-yl)boronic acid using the method described for <b>6</b> to give <b>7</b> (50%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.23 (s, 1H), 8.29–8.20 (m, 1H), 7.88 (t, <i>J</i> = 6.1 Hz, 1H), 7.84 (s, 1H), 7.68 (dd, <i>J</i> = 1.7, 8.5 Hz, 1H), 7.50 (dt, <i>J</i> = 0.9, 8.6 Hz, 1H), 7.40 (t, <i>J</i> = 2.8 Hz, 1H), 6.53 (ddd, <i>J</i> = 0.9, 1.9, 3.1 Hz, 1H), 6.30 (s, 1H), 3.25 (s, 2H), 1.19 (dd, <i>J</i> = 2.0, 4.9 Hz, 1H), 0.52–0.45 (m, 2H), 0.34–0.28 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 147.4, 143.6, 135.4, 132.5, 129.9, 128.4, 127.6, 126.2, 120.5, 119.3, 118.6, 111.6, 101.6, 90.3, 46.1, 10.3, 3.4 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.52 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 338 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>5</sub><sup>35</sup>Cl (M + H) 338.1172, found 338.1161.</div></div><div id="sec5_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>8</b>)</h4><div class="NLM_p last">Prepared from <b>4</b> and 6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazole using the method described for <b>6</b> with purification by preparative HPLC to give <b>8</b> (5%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (d, <i>J</i> = 36.4 Hz, 1H), 8.39 (d, <i>J</i> = 8.9 Hz, 1H), 8.28 (s, 1H), 7.95 (d, <i>J</i> = 15.4 Hz, 2H), 7.88–7.60 (m, 2H), 6.34 (s, 1H), 3.25 (s, 2H), 1.23–1.14 (m, 1H), 0.52–0.45 (m, 2H), 0.35–0.28 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 148.0, 144.1, 143.4, 133.4, 129.5 (2C), 122.7, 121.6, 91.0, 46.5, 10.7, 3.9 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.19 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 339 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>6</sub><sup>35</sup>Cl (M + H) 339.1125, found 339.1127.</div></div><div id="sec5_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>9</b>)</h4><div class="NLM_p last">A mixture of 5-bromo-2-chloro-1<i>H</i>-benzo[<i>d</i>]imidazole (5.00 g, 21.6 mmol) and aniline (10.1 mL, 110 mmol) was heated by microwave irradiation at 180 °C for 60 min. The mixture was cooled, diluted with EtOAc (60 mL), and washed with water (60 mL) and brine (60 mL). The organic layer was dried and concentrated. Silica chromatography, eluting with 0–2% MeOH–CH<sub>2</sub>Cl<sub>2</sub>, gave 5-bromo-<i>N</i>-phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (4.67 g, 75%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.02 (s, 1H), 9.54 (s, 1H), 7.76–7.69 (m, 2H), 7.46 (s, 1H), 7.35–7.27 (m, 2H), 7.24 (d, <i>J</i> = 8.2 Hz, 1H), 7.11 (dd, <i>J</i> = 2.0, 8.4 Hz, 1H), 6.98–6.91 (m, 1H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.4, 140.5, 128.8 (2C), 122.9, 122.2, 120.9, 118.1, 117.9, 117.3 (2C), 112.1, 111.0; LCMS (2 min) <i>t</i><sub>R</sub> = 0.95 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 288 and 290 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>10</sub>N<sub>3</sub><sup>79</sup>Br (M + H) 288.0136, found 288.0148. DMAP (0.4 g, 3.24 mmol) and di-<i>tert</i>-butyl-dicarbonate (8.84 g, 40.5 mmol) were added to a solution of 5-bromo-<i>N</i>-phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (4.67 g, 16.2 mmol) in dry THF (160 mL). The mixture was stirred at rt for 16 h and then concentrated. The residue was dissolved in EtOAc (100 mL) and washed with water (100 mL) and brine (100 mL). The organic layer was dried and concentrated. Silica chromatography, eluting with 0–2% MeOH-CH<sub>2</sub>Cl<sub>2</sub>, gave <i>tert</i>-butyl 5-bromo-2-((<i>tert</i>-butoxycarbonyl)(phenyl)amino)-1<i>H</i>-benzo[<i>d</i>]imidazole-1-carboxylate (7.9 g, quantitative) as a mixture of three di-Boc-protected isomers, which was used directly in the next step. LCMS (2 min) <i>t</i><sub>R</sub> = 1.69, 1.87, 1.91 min; <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 488 and 490 (M + H<sup>+</sup>). A portion of the material (5.4 g, 11.1 mmol) was dissolved in dry 1,4-dioxane (110 mL) and stirred at rt. Bis(pinacolato)diboron (3.37 g, 13.3 mmol), 1,1′-bis(diphenylphosphino)ferrocene-palladium(II) dichloride (0.81 g, 1.11 mmol), and KOAc (2.17 g, 22.1 mmol) were added and the mixture was heated at 100 °C for 15 h. The mixture was cooled, filtered through celite, and concentrated. The residue was dissolved in EtOAc (50 mL) and washed with water (50 mL) and brine (50 mL). The organic layer was dried and concentrated. Silica chromatography, eluting with 0–10% EtOAc–cyclohexane, gave <i>tert</i>-butyl 2-((<i>tert</i>-butoxycarbonyl)(phenyl)amino)-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazole-1-carboxylate <b>13</b> (4.40 g, 74%) as a mixture of three di-Boc protected isomers, which was used directly in the next step. LCMS (2 min) <i>t</i><sub>R</sub> = 1.70 min, 1.78 min, 1.82 min; <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 536 (M + H<sup>+</sup>). The mixture of isomers of <b>13</b> was reacted with <b>4</b> using the method described for <b>6</b> to give <b>9</b> (33%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.14 and 11.01 (2 × s, 1H, NH); 9.53 and 9.50 (2 × s, 1H, NH), 8.06 (dd, <i>J</i> = 1.7, 11.6 Hz, 1H), 7.93–7.84 (m, 2H), 7.82–7.74 (m, 2H), 7.62 (td, <i>J</i> = 1.8, 7.9 Hz, 1H), 7.41 (dd, <i>J</i> = 8.2, 28.3 Hz, 1H), 7.33 (td, <i>J</i> = 2.9, 8.0 Hz, 2H), 6.94 (td, <i>J</i> = 1.2, 7.2 Hz, 1H), 6.31 (s, 1H), 3.25 (s, 2H), 1.23–1.15 (m, 1H), 0.53–0.45 (m, 2H), 0.35–0.29 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.4, 151.2, 147.5, 143.6, 143.2, 143.1, 140.7, 132.9, 132.7, 132.6, 129.7 (2C), 128.8, 128.5, 120.9, 120.7, 120.0, 119.9, 119.0, 117.2 (2C), 117.1, 115.8, 114.2, 109.5, 107.7, 90.3, 46.1, 10.2, 3.4 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.30 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 430 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>20</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 430.1547, found 430.1537.</div></div><div id="sec5_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(2-(methyl(phenyl)amino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>10</b>)</h4><div class="NLM_p last">A mixture of 5-bromo-2-chloro-1<i>H</i>-benzo[<i>d</i>]imidazole (0.700 g, 3.02 mmol) and <i>N</i>-methylaniline (0.98 mL, 9.07 mmol) in EtOH (7 mL) was heated under microwave irradiation at 180 °C for 30 min. The mixture was cooled and concentrated. Silica chromatography, eluting with 10% EtOAc–cyclohexane and then with 50% EtOH–EtOAc, gave 5-bromo-<i>N</i>-methyl-<i>N</i>-phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (0.800 g, 88%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.85 (br s, 1H), 7.54 (d, <i>J</i> = 3.1 Hz, 4H), 7.45 (d, <i>J</i> = 1.6 Hz, 1H), 7.41 (tt, <i>J</i> = 2.7, 6.0 Hz, 1H), 7.30–7.23 (m, 2H), 3.56 (s, 3H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 152.1, 142.4, 130.1 (2C), 127.3, 125.4 (2C), 124.5, 114.5, 113.6, 113.3 (2 quaternary C not observed; N<u class="uu">C</u>H<sub>3</sub> obscured by solvent but detected in heteronuclear single quantum coherence NMR); LCMS (2 min) <i>t</i><sub>R</sub> = 0.94 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 302 and 304 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>12</sub>N<sub>3</sub><sup>79</sup>Br (M + H) 302.0293, found 302.0300. A mixture of 5-bromo-<i>N</i>-methyl-<i>N</i>-phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (0.800 g, 2.65 mmol), bis(pinacolato)diboron (0.690 g, 2.73 mmol), 1,1′-bis(diphenylphosphino)ferrocene-palladium(II) dichloride (0.190 g, 0.260 mmol), and KOAc (0.520 g, 5.30 mmol) in dry 1,4-dioxane (32 mL) was heated at 100 °C for 15 h. The mixture was cooled, filtered through celite, and concentrated. Ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub> in MeOH, gave <i>N</i>-methyl-<i>N</i>-phenyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>14</b>) (0.29 g, 31%), which was used in the next step without further purification. LCMS (2 min) <i>t</i><sub>R</sub> = 1.03 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 350 (M + H<sup>+</sup>). The crude product <b>14</b> was reacted with <b>4</b> using the method described for <b>6</b> to give <b>10</b> (24%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.45 (s, 1H), 8.01 (d, <i>J</i> = 1.6 Hz, 1H), 7.90 (t, <i>J</i> = 6.1 Hz, 1H), 7.84 (s, 1H), 7.57 (dd, <i>J</i> = 1.7, 8.2 Hz, 1H), 7.49–7.42 (m, 4H), 7.32 (d, <i>J</i> = 8.1 Hz, 1H), 7.27–7.20 (m, 1H), 6.30 (s, 1H), 3.34 (s, 3H), 3.24 (s, 2H), 1.24–1.11 (m, 1H), 0.52–0.43 (m, 2H), 0.35–0.26 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 154.8, 147.4, 144.7, 143.6, 132.6, 129.6, 129.4 (2C), 128.4, 124.8, 123.9 (2C), 119.5, 90.3, 46.1, 38.8, 10.2, 3.4 (2C), 5 quaternary C not observed; LCMS (2 min) <i>t</i><sub>R</sub> = 1.25 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 444 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>22</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 444.1703, found 444.1703.</div></div><div id="sec5_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(1-methyl-2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>11</b>)</h4><div class="NLM_p last">Isothiocyanatobenzene (0.20 mL, 0.23 mmol) and EDC (0.56 g, 2.9 mmol) were added to a stirred solution of <i>N</i><sup>1</sup>-methyl-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzene-1,2-diamine (<b>15</b>)<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> in dry THF (16 mL) at rt. The mixture was heated to 50 °C for 16 h. The mixture was concentrated and the residue was partitioned between water (30 mL) and EtOAc (30 mL). The organic layer was washed with brine (30 mL), dried, and concentrated. Ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub>–MeOH, gave 1-methyl-<i>N</i>-phenyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>16</b>) (0.25 g, 45%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.94 (s, 1H), 7.82–7.90 (m, 2H), 7.65 (d, <i>J</i> = 0.9 Hz, 1H), 7.41 (dd, <i>J</i> = 1.1, 7.9 Hz, 1H), 7.29–7.35 (m, 3H), 6.96 (tt, <i>J</i> = 1.2, 7.4 Hz, 1H), 3.72 (s, 3H), 1.30 (s, 12H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 150.8, 141.4, 140.8, 136.8, 128.6 (2C), 126.3, 122.3, 121.1, 120.1, 117.9 (2C), 107.7, 83.2 (2C), 29.1, 24.7 (4C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.10 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 350 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>25</sub>BN<sub>3</sub>O<sub>2</sub> (M + H) 350.2043, found 350.2035. The product <b>16</b> was combined with <b>4</b> using the method described for <b>6</b> with purification by preparative HPLC to give <b>11</b> (15%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.99 (s, 1H), 8.12 (dd, <i>J</i> = 0.5, 1.6 Hz, 1H), 7.94–7.87 (m, 4H), 7.71 (dd, <i>J</i> = 1.6, 8.3 Hz, 1H), 7.42 (d, <i>J</i> = 8.2 Hz, 1H), 7.38–7.30 (m, 2H), 6.97 (tt, <i>J</i> = 1.1, 7.3 Hz, 1H), 6.32 (s, 1H), 3.76 (s, 3H), 3.25 (s, 1H), 1.29–1.13 (m, 2H), 0.52–0.43 (m, 2H), 0.35–0.28 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.1, 147.5, 143.7, 141.8, 140.8, 134.0, 132.7, 129.5, 128.7, 128.6 (2C), 121.3, 121.2, 118.9, 118.0 (2C), 114.3, 108.2, 90.4, 46.1, 29.2, 10.2, 3.4 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.41 min, (ESI<sup>+</sup>) 444 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>23</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 444.1703, found 444.1706.</div></div><div id="sec5_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 6-Chloro-<i>N</i>-(cyclopropylmethyl)-3-(1-methyl-2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-6-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>12</b>)</h4><div class="NLM_p last">Isothiocyanatobenzene was combined with <i>N</i><sup>1</sup>-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)benzene-1,2-diamine (<b>17</b>)<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> using the method described for <b>16</b> to give 1-methyl-<i>N</i>-phenyl-6-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>18</b>) (17%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.00 (s, 1H), 7.81–7.93 (m, 2H), 7.56 (d, <i>J</i> = 1.0 Hz, 1H), 7.42 (dd, <i>J</i> = 1.1, 7.8 Hz, 1H), 7.28–7.38 (m, 3H), 6.97 (tt, <i>J</i> = 1.1, 7.3 Hz, 1H), 3.73 (s, 3H), 1.31 (s, 12H); LCMS (2 min) <i>t</i><sub>R</sub> = 1.25 min, 350 (M + H<sup>+</sup>). The product <b>18</b> was combined with <b>4</b> using the method described for <b>6</b> to give <b>12</b> (38%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.02 (s, 1H), 7.97 (d, <i>J</i> = 1.6 Hz, 1H), 7.96–7.91 (m, 2H), 7.91–7.87 (m, 2H), 7.81 (dd, <i>J</i> = 1.7, 8.3 Hz, 1H), 7.50–7.46 (m, 1H), 7.38–7.30 (m, 2H), 6.97 (tt, <i>J</i> = 1.1, 7.2 Hz, 1H), 6.32 (s, 1H), 3.77 (s, 3H), 3.25 (s, 2H), 1.26–1.13 (m, 1H), 0.52–0.45 (m, 2H), 0.35–0.28 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.3, 147.5, 143.6, 141.5, 140.7, 134.4, 132.7, 129.5, 128.7, 128.6 (2C), 121.3, 120.2, 120.0, 118.1 (2C), 115.9, 106.3, 90.4, 46.1, 29.1, 10.2, 3.5 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.41 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 444 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>23</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 444.1703, found 444.1704.</div></div><div id="sec5_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 6-Chloro-3-(1<i>H</i>-indazol-5-yl)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>19</b>)</h4><div class="NLM_p last">A mixture of <b>3</b> (150 mg, 0.482 mmol) and tetrahydro-2<i>H</i>-pyran-4-amine (73 mg, 1.5 equiv, 0.723 mmol) in THF (1.2 mL) was stirred at rt for 20 h. The mixture was diluted with THF (3 mL), and polystyrene–benzaldehyde resin (0.50 g, 0.723 mmol) was added. The suspension was gently stirred for 18 h at rt and then purified by ion-exchange chromatography on acidic resin (5 g), eluting with MeOH, 20% MeOH–CH<sub>2</sub>Cl<sub>2</sub>, and then 1:1 CH<sub>2</sub>Cl<sub>2</sub>/7 M NH<sub>3</sub> in MeOH. The basic fractions were concentrated and dried in vacuo to give 3-bromo-6-chloro-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (144 mg, 90%), which was used directly in the next step. A sample of the crude material (60 mg, 0.18 mmol) was added to a mixture of 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole (53 mg, 0.216 mmol), 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene (8.54 mg, 0.018 mmol), Pd(OAc)<sub>2</sub> (4.04 mg, 0.018 mmol), and aq Na<sub>2</sub>CO<sub>3</sub> (2 M; 0.360 mL, 0.720 mmol) in 1,4-dioxane (1.5 mL). The mixture was heated in a sealed tube at 130 °C for 18 h and then cooled and purified by ion-exchange chromatography on acidic resin (2 g), eluting with MeOH and then with 1:1 CH<sub>2</sub>Cl<sub>2</sub>/7 M NH<sub>3</sub> in MeOH. The basic fractions were concentrated. Preparative HPLC, followed by preparative thin-layer chromatography, eluting with 1% NH<sub>3</sub> (aq) in 7% MeOH–CH<sub>2</sub>Cl<sub>2</sub>, gave <b>19</b> (2.90 mg, 4%). <sup>1</sup>H NMR (500 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.54 (dd, <i>J</i> = 0.9, 1.6 Hz, 1H), 8.14 (d, <i>J</i> = 1.0 Hz, 1H), 7.95 (dd, <i>J</i> = 1.6, 8.8 Hz, 1H), 7.81 (s, 1H), 7.63 (dt, <i>J</i> = 1.0, 8.9 Hz, 1H), 6.34 (s, 1H), 4.01 (dt, <i>J</i> = 3.7, 11.9 Hz, 2H), 3.84 (d, <i>J</i> = 10.4 Hz, 1H), 3.60 (td, <i>J</i> = 2.2, 11.6 Hz, 2H), 2.06 (ddt, <i>J</i> = 2.3, 4.6, 12.2 Hz, 2H), 1.69 (dtd, <i>J</i> = 4.3, 10.8, 13.2 Hz, 2H); <sup>13</sup>C NMR (126 MHz, MeOH-<i>d</i><sub>4</sub>) δ 149.9, 143.4, 135.6, 134.5, 132.4, 131.1, 129.5, 127.6, 124.4, 122.6, 120.5, 111.3, 93.2, 67.4, 49.6 (2C), 33.3 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.35 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 369 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>N<sub>6</sub>O<sup>35</sup>Cl (M + H) 369.1225, found 369.1216.</div></div><div id="sec5_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 6-Chloro-3-(1<i>H</i>-indazol-5-yl)-<i>N</i>-(1-methylpiperidin-4-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>20</b>)</h4><div class="NLM_p last">Prepared from <b>3</b>, 1-methylpiperidin-4-amine, and 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole using the method described for <b>19</b> to give <b>20</b> (11% over two steps). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.2 (s, 1H), 8.51 (s, 1H), 8.20 (s, 1H), 8.00 (s, 1H), 7.98 (s, 1H), 7.74 (br d, <i>J</i> = 8.5 Hz, 1H), 7.67 (br d, <i>J</i> = 8.8 Hz, 1H), 6.40 (s, 1H), 3.66 (br s, 1H), 2.95–2.93 (m, 2H), 2.33 (br s, 5H), 1.93–1.90 (m, 2H), 1.82–1.76 (m, 2H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 148.1, 143.3, 139.6, 134.6, 133.3, 129.4, 129.3, 125.9, 123.4, 121.2, 118.9, 110.9, 91.4, 55.4, 54.2, 45.4, 34.2, 31.8 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 0.94 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 382 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 382.1547, found 382.1542.</div></div><div id="sec5_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 6-Chloro-3-(1<i>H</i>-indazol-5-yl)-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>21</b>)</h4><div class="NLM_p last">Prepared from <b>3</b>, (1-methylpiperidin-4-yl)methanamine, and 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole using the method described for <b>19</b> to give <b>21</b> (10% over two steps). <sup>1</sup>H NMR (500 MHz, MeOH-d<sub>4</sub>) δ 8.53 (t, <i>J</i> = 0.7 Hz, 1H), 8.14 (d, <i>J</i> = 0.9 Hz, 1H), 7.94 (dd, <i>J</i> = 1.6 and 8.8 Hz, 1H), 7.78 (s, 1H), 7.62 (d, <i>J</i> = 8.8 Hz, 1H), 6.20 (s, 1H), 3.23 (d, <i>J</i> = 7.0 Hz, 2H), 2.94 (br d, <i>J</i> = 12 Hz, 2H), 2.31 (s, 3H), 2.10–2.05 (m, 2H), 1.86 (br d, <i>J</i> = 13 Hz, 2H), 1.74–1.70 (m, 1H), 1.42–1.34 (m, 2H); <sup>13</sup>C NMR (126 MHz, Methanol-d4) δ 148.4, 143.4, 139.6, 134.1, 133.0, 129.5, 128.1, 126.1, 122.9, 121.2, 118.9, 110.0, 91.4, 54.8, 44.8, 34.5, 29.2; LCMS (2 min) <i>t</i><sub>R</sub> = 1.09 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 396 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 396.1703, found 396.1707.</div></div><div id="sec5_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (1<i>r</i>,4<i>r</i>)-<i>N</i><sup>1</sup>-(6-Chloro-3-(1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-yl)cyclohexane-1,4-diamine (<b>22</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl ((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)carbamate (0.929 g, 4.34 mmol) was added to a stirred solution of <b>3</b> (0.450 g, 1.45 mmol) in dimethylformamide (10 mL) at rt. The reaction was heated at 50 °C for 16 h, then cooled and partitioned between Na<sub>2</sub>CO<sub>3</sub> aq and EtOAc. The organic layer was washed with brine, dried, and concentrated. Silica chromatography, eluting with 0–40% EtOAc–cyclohexane, gave <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-((3-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazin-8-yl)amino)cyclohexyl)-carbamate (0.643 g, 100%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.48 (s, 1H), 6.16 (s, 1H), 5.97 (d, <i>J</i> = 7.7 Hz, 1H), 4.08 (d, <i>J</i> = 13.4 Hz, 1H), 3.79 (s, 1H), 3.56 (s, 1H), 3.11 (ddd, <i>J</i> = 13.1, 9.6, 3.4 Hz, 1H), 3.03 (dd, <i>J</i> = 13.3, 8.5 Hz, 1H), 2.16–2.09 (m, 1H), 1.84 (d, <i>J</i> = 14.3 Hz, 1H), 1.75–1.64 (m, 2H), 1.48 (s, 9H); LCMS (2 min) <i>t</i><sub>R</sub> = 1.57 min <i>m</i>/<i>z</i>, (ESI<sup>+</sup>) 446 (M + H<sup>+</sup>). A mixture of <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-((3-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazin-8-yl)amino)cyclohexyl)-carbamate (0.100 g, 0.225 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole (0.055 g, 0.225 mmol), Pd(OAc)<sub>2</sub> (5.05 mg, 0.022 mmol), 1,1′-bis(di-<i>t</i>-butylphosphino)ferrocene (0.011 g, 0.022 mmol), and Na<sub>2</sub>CO<sub>3</sub> aq (2 M; 0.45 mL, 0.900 mmol) in 1,4-dioxane (2 mL) was heated at 130 °C in a sealed tube for 24 h. The mixture was cooled, filtered through celite, and absorbed onto silica. Chromatography, eluting with 0–50% EtOAc–cyclohexane, gave <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-((6-chloro-3-(1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-yl)amino)cyclohexyl)carbamate (0.060 g, 55%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.50 (s, 1H), 8.22 (s, 1H), 7.94 (d, <i>J</i> = 8.7 Hz, 1H), 7.74 (s, 1H), 7.61 (d, <i>J</i> = 8.8 Hz, 1H), 6.18 (d, <i>J</i> = 8.0 Hz, 1H), 6.04 (d, <i>J</i> = 7.0 Hz, 1H), 4.54–4.47 (m, 1H), 3.54 (s, 1H), 3.47–3.37 (m, 1H), 2.52–2.12 (m, 4H), 1.48 (s, 9H), 1.37–1.24 (m, 4H); LCMS (2 min) <i>t</i><sub>R</sub> = 1.56 min <i>m</i>/<i>z</i>, (ESI<sup>+</sup>) 482 (M + H<sup>+</sup>). <i>tert</i>-Butyl ((1<i>r</i>,4<i>r</i>)-4-((6-chloro-3-(1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-yl)amino)cyclohexyl)carbamate (30 mg, 0.062 mmol) was dissolved in HCl–1,4-dioxane (4 M; 1 mL) and stirred at rt for 2 h. The mixture was concentrated. Ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub>–MeOH, followed by preparative HPLC gave <b>22</b> (24 mg, 99%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.50 (t, <i>J</i> = 1.2 Hz, 1H), 8.19 (d, <i>J</i> = 1.0 Hz, 1H), 7.96–7.89 (m, 2H), 7.69–7.62 (m, 1H), 7.55 (s, 1H), 6.34 (s, 1H), 3.52 (s, 1H), 3.17 (br s, 3H), 2.57 (tt, <i>J</i> = 10.5, 3.6 Hz, 1H), 1.99–1.71 (m, 4H), 1.50 (q, <i>J</i> = 13.3, 12.7 Hz, 2H), 1.25–1.15 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.6, 147.7, 142.78, 139.3, 134.0, 132.8, 128.9, 125.4, 122.9, 120.8, 118.4, 110.6, 90.9, 49.7, 48.5, 29.5 (2C), 29.3 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.11 min, <i>m</i>/<i>z</i>, (ESI<sup>+</sup>), 382 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>20</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 382.1547, found 382.1538.</div></div><div id="sec5_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (1<i>r</i>,4<i>r</i>)-<i>N</i><sup>1</sup>-(6-Chloro-3-(1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-yl)-<i>N</i><sup>1</sup>-methylcyclohexane-1,4-diamine (<b>23</b>)</h4><div class="NLM_p last">LiAlH<sub>4</sub> in THF (1 M; 25 mL, 25.0 mmol) was added to a solution of <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)carbamate (1.30 g, 6.07 mmol) in THF (5 mL) at 0 °C. The mixture was heated to reflux for 4 h, forming a thick white suspension. The reaction was cooled, diluted with Et<sub>2</sub>O (80 mL), and water (0.95 mL) was slowly added dropwise, followed by 15% w/v NaOH aq (0.95 mL). After stirring for 1 h, water (2.85 mL) was added dropwise. The mixture was filtered through celite, washed with Et<sub>2</sub>O, and the filtrate was concentrated to give crude (1<i>r</i>,4<i>r</i>)-<i>N</i><sup>1</sup>-methylcyclohexane-1,4-diamine (0.597 g). A solution of the crude amine, Et<sub>3</sub>N (1.3 mL, 9.31 mmol), and ethyl 1,3-dioxoisoindoline-2-carboxylate (0.928 g, 4.23 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (42 mL) was stirred at rt for 15 h. Saturated (NH<sub>4</sub>)<sub>2</sub>CO<sub>3</sub> aq (50 mL) was added, and the mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 30 mL). The combined organic layers were dried and concentrated to give crude 2-((1<i>r</i>,4<i>r</i>)-4-(methylamino)cyclohexyl)isoindoline-1,3-dione (0.99 g). A mixture of the crude amine, <b>3</b> (0.918 g, 2.95 mmol), and Et<sub>3</sub>N (0.82 mL, 5.90 mmol) in 1,4-dioxane (3.7 mL) and DMSO (3.7 mL) was heated at 100 °C for 21 h. The mixture was cooled and diluted with CH<sub>2</sub>Cl<sub>2</sub> (15 mL). The solid was collected and washed with dichloromethane (DCM, 5 mL) to give 2-((1<i>r</i>,4<i>r</i>)-4-((3-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazin-8-yl)(methyl)amino)cyclohexyl)isoindoline-1,3-dione (1.06 g, 52% over three steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.88–7.82 (m, 4H), 7.80 (s, 1H), 6.26 (s, 1H), 5.76 (br s, 1H), 4.18–4.08 (m, 1H), 3.10 (br s, 3H), 2.39–2.25 (m, 2H), 1.97–1.77 (m, 6H); LCMS (4 min) <i>t</i><sub>R</sub> = 3.53 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 490 (M + H<sup>+</sup>). A mixture of 2-((1<i>r</i>,4<i>r</i>)-4-((3-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazin-8-yl)(methyl)amino)cyclohexyl)-isoindoline-1,3-dione (0.210 g, 0.430 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole (0.126 g, 0.516 mmol), Pd(OAc)<sub>2</sub> (9.6 mg, 0.043 mmol), 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene (0.020 g, 0.043 mmol), and Na<sub>2</sub>CO<sub>3</sub> aq (2 M; 0.86 mL, 1.72 mmol) in 1,4-dioxane (2.1 mL) was heated at 130 °C for 21 h. The reaction mixture was cooled and NH<sub>2</sub>NH<sub>2</sub>·H<sub>2</sub>O (0.5 mL) was added. The mixture was heated at 100 °C for 48 h. The mixture was cooled and purified by ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub> in MeOH. Reversed-phase silica chromatography, eluting with 10–90% MeOH–water containing 0.1% HCO<sub>2</sub>H, followed by normal phase chromatography, eluting with 2–20% EtOH–CH<sub>2</sub>Cl<sub>2</sub>, gave <b>23</b> (0.058 g, 34%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (s, 1H), 8.47–8.44 (m, 1H), 8.20 (d, <i>J</i> = 0.9 Hz, 1H), 7.96 (s, 1H), 7.90 (dd, <i>J</i> = 8.7, 1.6 Hz, 1H), 7.67 (d, <i>J</i> = 8.7 Hz, 1H), 6.19 (s, 1H), 5.70 (br s, 1H), 3.07 (br s, 3H), 2.57 (tt, <i>J</i> = 11.1, 3.9 Hz, 1H), 1.88–1.81 (m, 2H), 1.76–1.70 (m, 4H), 1.62 (br s, 2H), 1.32–1.18 (m, 2H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 147.6, 145.1, 139.7, 134.5, 133.7, 129.3, 128.9, 126.3, 123.3, 121.2, 119.4, 110.9, 94.1, 57.9, 50.0, 35.8 (2C), 32.7, 28.9 (2C); LCMS (4 min) <i>t</i><sub>R</sub> = 2.26 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 396 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>22</sub>N<sub>7</sub><sup>35</sup>Cl (M + H) 396.1703, found 396.1700.</div></div><div id="sec5_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(<i>trans</i>-4-((6-Chloro-3-(1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-yl)amino)cyclohexyl)acetamide (<b>24</b>)</h4><div class="NLM_p last">Prepared from <b>3</b>, <i>N</i>-((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)acetamide, and 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole using the methods described for <b>19</b>, except that the first step was conducted at 50 °C, to give <b>24</b> (35% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.18 (s, 1H), 8.50 (dt, <i>J</i> = 0.8, 1.6 Hz, 1H), 8.19 (t, <i>J</i> = 1.3 Hz, 1H), 7.96–7.90 (m, 2H), 7.80 (d, <i>J</i> = 7.7 Hz, 1H), 7.65 (dt, <i>J</i> = 0.9, 8.7 Hz, 1H), 7.61 (d, <i>J</i> = 8.8 Hz, 1H), 6.42 (s, 1H), 3.60 (s, 1H), 3.50 (dtt, <i>J</i> = 4.1, 8.1, 11.9 Hz, 1H), 1.93 (t, <i>J</i> = 8.2 Hz, 2H), 1.79 (s, 5H), 1.56 (d, <i>J</i> = 12.8 Hz, 2H), 1.41–1.27 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.3, 147.7, 142.8, 139.1, 134.2, 132.9, 128.9, 128.8, 125.5, 122.9, 120.8, 118.4, 110.4, 90.7, 50.1, 47.2, 30.9 (2C), 30.3 (2C), 22.8; LCMS (2 min), <i>t</i><sub>R</sub> = 1.32 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 424 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>23</sub>N<sub>7</sub>O<sup>35</sup>Cl (M + H) 424.1647, found 424.1635.</div></div><div id="sec5_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 6-Chloro-3-(1<i>H</i>-indazol-5-yl)-<i>N</i>-(<i>trans</i>-4-morpholinocyclohexyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine, Formate Salt (<b>25</b>)</h4><div class="NLM_p last">Prepared from <b>3</b>, (1<i>r</i>,4<i>r</i>)-4-morpholinocyclohexan-1-amine, and 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole using the methods described for <b>19</b>, except that the first step was conducted at 50 °C, to give <b>25</b> (4% over two steps), isolated as the formate salt from preparative HPLC. <sup>1</sup>H NMR (500 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.56–8.52 (m, 1H), 8.36 (s, 1H), 8.14 (d, <i>J</i> = 1.0 Hz, 1H), 7.94 (dd, <i>J</i> = 1.6, 8.8 Hz, 1H), 7.79 (s, 1H, formate), 7.63 (d, <i>J</i> = 8.6 Hz, 1H), 6.26 (s, 1H), 3.88 (t, <i>J</i> = 4.7 Hz, 4H), 3.55–3.51 (m, 1H), 3.13 (t, <i>J</i> = 4.6 Hz, 4H), 3.00 (ddd, <i>J</i> = 3.3, 8.4, 12.1 Hz, 1H), 2.28–2.15 (m, 4H), 1.63 (qd, <i>J</i> = 3.2, 12.6 Hz, 2H), 1.46 (qd, <i>J</i> = 2.9, 13.1 Hz, 2H); <sup>13</sup>C NMR (126 MHz, MeOH-<i>d</i><sub>4</sub>) δ 167.8 (formate), 149.8, 143.5, 141.1, 135.5, 134.4, 131.0, 129.6, 127.5, 124.4, 122.6, 120.4, 111.4, 93.1, 65.9 (2C), 65.4, 51.4, 50.4 (2C), 31.4 (2C), 26.6 (2C); LCMS (2 min), <i>t</i><sub>R</sub> = 1.00 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 452 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>27</sub>N<sub>7</sub>O<sup>35</sup>Cl (M + H) 452.1966, found 452.1961.</div></div><div id="sec5_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-(((1<i>r</i>,4<i>r</i>)-4-((6-Chloro-3-(1<i>H</i>-indazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-yl)amino)cyclohexyl)amino)-1-morpholinoethan-1-one (<b>26</b>)</h4><div class="NLM_p last">A solution of 2-chloro-1-morpholinoethanone (0.66 mL, 5.05 mmol), <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)carbamate (902 mg, 4.21 mmol), and Et<sub>3</sub>N (2.1 mL, 15.1 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (35 mL) was stirred at rt for 20 h. The mixture was partitioned between water (50 mL) and CH<sub>2</sub>Cl<sub>2</sub> (30 mL × 2). The combined organic layers were dried and concentrated. The resulting crude material was dissolved in MeOH (42 mL), and HCl in 1,4-dioxane (4 M; 42 mL, 168 mmol) was added. After 2 h at rt, the mixture was concentrated and purified by ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub> in MeOH, to give 2-(((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)amino)-1-morpholinoethan-1-one (0.375 g, 37%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 3.71–3.59 (m, 7H), 3.49–3.36 (m, 4H), 2.72–2.58 (m, 3H), 2.42–2.33 (m, 1H), 1.94–1.78 (m, 4H), 1.24–1.04 (m, 4H). A sample of the material (0.370 g, 1.55 mmol), <b>3</b> (0.440 g, 1.41 mmol), and Et<sub>3</sub>N (0.39 mL, 2.83 mmol) in 1,4-dioxane (1.8 mL) and DMSO (1.8 mL) was heated at 50 °C for 23 h. The mixture was cooled and purified by ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub> in MeOH, and then by silica chromatography, eluting with 2–20% EtOH–CH<sub>2</sub>Cl<sub>2</sub>, to give 2-(((1<i>r</i>,4<i>r</i>)-4-((3-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazin-8-yl)amino)cyclohexyl)amino)-1-morpholinoethan-1-one (0.38 g, 58%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.44 (s, 1H), 6.04 (s, 1H), 5.80 (d, <i>J</i> = 8.1 Hz, 1H), 3.75–3.62 (m, 7H), 3.47 (s, 2H), 3.43–3.40 (m, 3H), 2.56–2.47 (m, 1H), 2.23–2.15 (m, 2H), 2.10–2.01 (m, 2H), 1.43–1.29 (m, 4H); LCMS (2 min) <i>t</i><sub>R</sub> = 0.97 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 473 (M + H<sup>+</sup>). A sample of the material (0.365 g, 0.774 mmol), 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole (0.227 g, 0.928 mmol), Pd(OAc)<sub>2</sub> (17.0 mg, 0.077 mmol), 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene (37.0 mg, 0.077 mmol), and Na<sub>2</sub>CO<sub>3</sub> aq (2 M; 1.55 mL, 3.09 mmol) were mixed in 1,4-dioxane (3.9 mL) and heated at 135 °C for 19 h. The mixture was cooled and purified by ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub> in MeOH, then by silica chromatography, eluting with 2–20% MeOH-CH<sub>2</sub>Cl<sub>2</sub>, and then by reversed-phase chromatography, eluting with 10–70% MeOH–water with 0.1% formic acid, to give <b>26</b> (0.069 g, 18%). <sup>1</sup>H NMR (500 MHz, MeOH-<i>d</i><sub>4</sub>) δ 8.49 (dd, <i>J</i> = 1.6, 0.9 Hz, 1H), 8.11 (d, <i>J</i> = 1.0 Hz, 1H), 7.89 (dd, <i>J</i> = 8.8, 1.6 Hz, 1H), 7.72 (s, 1H), 7.59 (d, <i>J</i> = 8.7 Hz, 1H), 6.17 (s, 1H), 3.67–3.63 (m, 4H), 3.59–3.55 (m, 2H), 3.46 (s, 2H), 3.45–3.42 (m, 3H), 2.50–2.41 (m, 1H), 2.13–2.06 (m, 2H), 2.02–1.96 (m, 2H), 1.40–1.24 (m, 4H); <sup>13</sup>C NMR (126 MHz, MeOH-<i>d</i><sub>4</sub>) δ 171.2, 149.8, 143.5, 141.0, 135.5, 134.4, 130.9, 129.4, 127.5, 124.4, 122.6, 120.3, 111.3, 92.9, 67.7, 67.6, 57.2, 52.2, 48.0, 46.1, 43.4, 32.0 (2C), 31.6 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.05 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 509 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>30</sub>N<sub>8</sub>O<sub>2</sub><sup>35</sup>Cl (M + H) 509.2180, found 509.2187.</div></div><div id="sec5_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 6-Chloro-3-(2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-<i>N</i>-(tetrahydro-2<i>H</i>-pyran-4-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>27</b>)</h4><div class="NLM_p last">Prepared from <b>4</b>, tetrahydro-2<i>H</i>-pyran-4-amine, and <b>13</b> using the method described for <b>19</b> to give <b>27</b> (14% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 and 11.01 (2 × s, 1H, NH); 9.53 and 9.50 (2 × s, 1H, NH), 8.06 (dd, <i>J</i> = 1.7, 12.9 Hz, 1H), 7.87 (d, <i>J</i> = 11.0 Hz, 1H), 7.82–7.75 (m, 2H), 7.71 (dd, <i>J</i> = 6.7, 8.7 Hz, 1H), 7.64–7.58 (m, 1H), 7.41 (dd, <i>J</i> = 8.3, 28.4 Hz, 1H), 7.36–7.29 (m, 2H), 6.94 (tt, <i>J</i> = 1.2, 7.3 Hz, 1H), 6.43 (s, 1H), 3.83–3.96 (m, 2H), 3.45 (td, <i>J</i> = 2.0, 11.7 Hz, 2H), 3.33 (s, 1H), 1.92–1.79 (m, 2H), 1.78–1.64 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.4, 151.2, 147.5, 143.2, 143.1, 142.6, 140.7, 132.9, 132.6, 129.7, 128.8 (2C), 128.5, 120.9, 120.7, 120.0, 117.2 (2C), 117.1, 115.8, 114.2, 109.5, 107.7, 90.6, 66.0 (2C), 48.2, 31.8 (2C); LCMS (4 min) <i>t</i><sub>R</sub> = 2.60 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 460 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>22</sub>N<sub>7</sub>O<sup>35</sup>Cl (M + H) 460.1653, found 460.1651.</div></div><div id="sec5_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-Chloro-<i>N</i>-(1-methylpiperidin-4-yl)-3-(2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>28</b>)</h4><div class="NLM_p last">Prepared from <b>3</b>, 1-methylpiperidin-4-amine, and <b>13</b> using the method described for <b>19</b> to give <b>28</b> (5% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.14 and 11.02 (2 × s, 1H, NH), 9.54 and 9.51 (2 × s, 1H, NH), 8.11–8.10 (m, 1H), 7.86 (d, <i>J</i> = 10.6 Hz, 1H), 7.78 (d, <i>J</i> = 7.9 Hz, 2H), 7.65–7.57 (m, 2H), 7.41 (dd, <i>J</i> = 8.3, 28.5 Hz, 1H), 7.36–7.28 (m, 2H), 6.97–6.90 (m, 1H), 6.35 (s, 1H), 3.57 (d, <i>J</i> = 2.3 Hz, 1H), 2.80 (d, <i>J</i> = 11.1 Hz, 2H), 2.21 (s, 3H), 2.08 (t, <i>J</i> = 11.7 Hz, 2H), 1.86 (d, <i>J</i> = 12.2 Hz, 2H), 1.80–1.67 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.4, 151.2, 147.5, 143.2, 143.1, 142.8, 140.7, 132.9, 132.6, 129.7, 128.8 (2C), 128.4, 120.9, 120.7, 119.9, 119.0, 117.2 (2C), 117.1, 115.8, 114.2, 109.5, 107.7, 90.6, 54.1 (2C), 48.8, 45.8, 30.7 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 0.86 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 473 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>25</sub>N<sub>8</sub><sup>35</sup>Cl (M + H) 473.1969, found 473.1961.</div></div><div id="sec5_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 6-Chloro-<i>N</i>-((1-methylpiperidin-4-yl)methyl)-3-(2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>29</b>)</h4><div class="NLM_p last">Prepared from <b>3</b>, (1-methylpiperidin-4-yl)methanamine, and <b>13</b> using the method described for <b>19</b> to give <b>29</b> (13% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.19 and 11.07 (2 × s, 1H, NH), 9.57 and 9.54 (2 × s, 1H, NH), 8.05 (d, <i>J</i> = 13.4 Hz, 1H), 7.93 (s, 1H), 7.85 (d, <i>J</i> = 9.2 Hz, 1H), 7.79 (d, <i>J</i> = 8.0 Hz, 2H), 7.61 (d, <i>J</i> = 8.2 Hz, 1H), 7.40 (dd, <i>J</i> = 8.2, 9.4 Hz, 1H), 7.32 (t, <i>J</i> = 7.8 Hz, 2H), 6.93 (tt, <i>J</i> = 1.2, 7.3 Hz, 1H), 6.30 (s, 1H), 3.23 (s, 2H), 2.79 (d, <i>J</i> = 11.2 Hz, 2H), 2.17 (s, 3H), 1.96–1.81 (m, 2H), 1.74–1.60 (m, 3H), 1.32–1.16 (m, 2H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.4, 151.3, 147.5, 143.9, 143.2, 143.1, 140.8, 132.9, 132.6, 129.7, 128.8 (2C), 128.4, 120.7, 119.9, 119.0, 117.2 (2C), 115.8, 114.2, 109.5, 107.7, 90.3, 54.9 (2C), 47.2, 45.9, 34.2, 29.4 (2C); LCMS (2 min) <i>t</i><sub>R</sub> = 1.03 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 487 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>27</sub>N<sub>8</sub><sup>35</sup>Cl (M + H) 487.2125, found 487.2130.</div></div><div id="sec5_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 6-Chloro-3-(7-fluoro-2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>30</b>)</h4><div class="NLM_p last">Prepared from 5-bromo-3-fluorobenzene-1,2-diamine and 3-bromo-6-chloro-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine using the method described for <b>31</b> to give <b>30</b> (5% over 4 steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.67 (s, 1H), 9.82 (s, 1H), 7.97 (t, <i>J</i> = 6.5 Hz, 1H), 7.94 (s, 2H), 7.83–7.72 (m, 2H), 7.54 (d, <i>J</i> = 12.2 Hz, 1H), 7.39–7.29 (m, 2H), 6.96 (tt, <i>J</i> = 1.2, 7.3 Hz, 1H), 6.33 (s, 1H), 3.24 (s, 2H), 2.80 (d, <i>J</i> = 11.2 Hz, 2H), 1.88 (t, <i>J</i> = 11.5 Hz, 2H), 2.18 (s, 3H), 1.77–1.61 (m, 3H), 1.32–1.17 (m, 2H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 152.0, 148.1, 144.3, 140.9, 133.4, 129.5, 129.3 (2C), 128.9, 121.5, 120.8, 117.9 (2C), 105.8 (d, <i>J</i><sub>C–F</sub> = 16.6 Hz), 104.5, 91.0, 55.5 (2C), 47.8, 46.6, 34.8, 30.0 (2C) (3 quaternary C not observed); LCMS (2 min) <i>t</i><sub>R</sub> = 1.13 min, <i>m</i>/<i>z</i> (ES<sup>+</sup>) 505 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>26</sub>N<sub>8</sub>F<sup>35</sup>Cl (M + H) 505.2031, found 505.2036.</div></div><div id="sec5_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 6-Chloro-3-(6-fluoro-2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>31</b>)</h4><div class="NLM_p last">A mixture of (1-methylpiperidin-4-yl)methanamine (1.24 g, 9.64 mmol), <b>3</b> (2.00 g, 6.42 mmol), and Et<sub>3</sub>N (1.79 mL, 12.9 mmol) in 1,4-dioxane (4 mL) and DMSO (4 mL) was stirred at 50 °C for 18 h. The mixture was cooled, diluted with EtOAc, and the organic layer was washed sequentially with water and brine. The organic layer was dried and concentrated. Ion-exchange chromatography on acidic resin, eluting with 2 M NH<sub>3</sub> in MeOH, followed by silica chromatography, eluting with 0–5% MeOH–CH<sub>2</sub>Cl<sub>2</sub> and then with 5–10% MeOH–DCM + 0.2 M NH<sub>3</sub>, gave 3-bromo-6-chloro-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (2.00 g, 87%), which was used directly in subsequent steps. LCMS (2 min) <i>t</i><sub>R</sub> = 0.84 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 360 (M + H<sup>+</sup>). Separately, isothiocyanatobenzene (1.22 mL, 10.2 mmol) and EDC (3.37 g, 17.5 mmol) were added to a stirred solution of 4-bromo-5-fluorobenzene-1,2-diamine (2.00 g, 9.75 mmol) in dry THF (100 mL) at rt. The mixture was heated at 50 °C for 4 h, cooled, and concentrated. The residue was partitioned between water (100 mL) and EtOAc (100 mL). The organic layer was washed with brine (100 mL), dried, and concentrated. Ion-exchange chromatography on acidic resin, eluting with MeOH, gave 5-bromo-6-fluoro-<i>N</i>-phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine that was used directly in the next step (2.53 g, 85%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.10 and 10.99 (2 × s, 1H, NH), 9.61 and 9.58 (2 × s, 1H, NH), 7.74–7.60 (m, 2H), 7.60–7.45 (m, 1H), 7.35–7.23 (m, 3H), 6.95 (t, <i>J</i> = 7.3 Hz, 1H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 154.6, 152.5, 152.0, 143.4, 140.3, 130.4, 128.8 (2C), 121.15, 121.1, 118.5, 117.4 (2C), 117.3, 112.3, 103.2 (d, <i>J</i> = 26.0 Hz), 97.8 (d, <i>J</i> = 26.0 Hz), (1 quaternary C not observed); LCMS (2 min) <i>t</i><sub>R</sub> = 1.01 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 306 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>9</sub>N<sub>3</sub>F<sup>79</sup>Br (M + H) 306.0042, found 306.0047. A mixture of 5-bromo-6-fluoro-<i>N</i>-phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (2.53 g, 8.26 mmol), (Boc)<sub>2</sub>O (4.50 g, 20.7 mmol), and DMAP (0.20 g, 1.65 mmol) in THF (80 mL) was stirred at rt for 16 h. The mixture was concentrated, and the residue was partitioned between EtOAc (100 mL) and water (100 mL). The organic layer was washed with brine (100 mL), dried, and concentrated. Silica chromatography, eluting with 0–10% EtOAc–cyclohexane, gave the product as a mixture of three different di-Boc-protected positional isomers (1.94 g), which was used as such in the next step. LCMS (2 min) <i>t</i><sub>R</sub> = 1.68, 1.79, and 1.92 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 506 (M + H<sup>+</sup>). This material (1.94 g) was stirred at rt in 1,4-dioxane (42 mL), and bis(pinacolato)diboron (1.17 g, 4.59 mmol), PdCl<sub>2</sub>dppf (0.28 g, 0.38 mmol), and KOAc (0.75 g, 7.65 mmol) were added. The mixture was heated at 120 °C for 24 h, then cooled and filtered through celite. The filtrate was concentrated and dissolved in THF (80 mL). DMAP (0.093 g, 0.77 mmol) and (Boc)<sub>2</sub>O (1.67 g, 7.66 mmol) were added, and the mixture was stirred for 6 h at rt. Solvent was removed by evaporation, and the residue was partitioned between water (100 mL) and EtOAc (100 mL). The organic layer was washed with brine (100 mL), dried, and concentrated. Silica chromatography, eluting with 0–50% EtOAc-cyclohexane, gave the product as a mixture of three different di-Boc-protected positional isomers (1.08 g), which was used as such in the next step. LCMS (2 min) <i>t</i><sub>R</sub> = 1.67, 1.78, and 1.81 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 554 (M + H<sup>+</sup>). A mixture of this material (1.08 g), 3-bromo-6-chloro-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (0.63 g, 1.76 mmol) and Na<sub>2</sub>CO<sub>3</sub> aq (2 M; 3.51 mL, 7.03 mmol) in dry 1,4-dioxane (18 mL) was purged with N<sub>2</sub> for 10 min. Pd(OAc)<sub>2</sub> (0.079 g, 0.35 mmol) and 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene (0.17 g, 0.35 mmol) were added and the tube was sealed and stirred at 135 °C for 15 h. The mixture was cooled, filtered through celite pad, and concentrated. The residue was purified by silica chromatography, eluting with 0–5% MeOH–CH<sub>2</sub>Cl<sub>2</sub> and then with 5–10% MeOH–CH<sub>2</sub>Cl<sub>2</sub> + 1% 0.2 M NH<sub>3</sub> aq, to give <b>31</b> (100 mg, 3% over three steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.14 (s, 1H), 9.62 (s, 1H), 8.00 (d, <i>J</i> = 6.5 Hz, 1H), 7.82 (d, <i>J</i> = 6.6 Hz, 1H), 7.79–7.74 (m, 2H), 7.70 (d, <i>J</i> = 2.5 Hz, 1H), 7.35–7.31 (m, 2H), 7.28 (d, <i>J</i> = 11.1 Hz, 1H), 6.98–6.92 (m, 1H), 6.31 (s, 1H), 3.23 (s, 2H), 2.75 (d, <i>J</i> = 10.8 Hz, 2H), 2.13 (s, 3H), 1.87–1.73 (m, 2H), 1.73–1.59 (m, 3H), 1.31–1.16 (m, 2H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 154.1 (d, <i>J</i><sub>C–F</sub> = 257.2 Hz), 148.1, 144.3, 140.9, 132.9, 131.0, 129.8, 129.3 (2C), 124.5, 121.5, 117.9 (2C), 110.0, 102.8, 91.1, 55.5 (2C), 47.8, 46.6, 34.8, 30.1 (2C) (3 quaternary C not observed); LCMS (2 min) <i>t</i><sub>R</sub> = 0.91 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 505 (M + H<sup>+</sup>); ESI <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>26</sub>N<sub>8</sub>F<sup>35</sup>Cl (M + H) 505.2031, found 505.2036.</div></div><div id="sec5_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 6-Chloro-3-(4-fluoro-2-(phenylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine (<b>32</b>)</h4><div class="NLM_p last">Prepared from 4-bromo-3-fluorobenzene-1,2-diamine and 3-bromo-6-chloro-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine using the method described for <b>31</b> to give <b>32</b> (2% over four steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (s, 1H), 9.62 (s, 1H), 7.99 (t, <i>J</i> = 6.3 Hz, 1H), 7.81–7.74 (m, 2H), 7.69 (d, <i>J</i> = 1.4 Hz, 1H), 7.38 (t, <i>J</i> = 7.3 Hz, 1H), 7.36–7.32 (m, 2H), 7.25 (d, <i>J</i> = 8.2 Hz, 1H), 6.96 (tt, <i>J</i> = 1.2, 7.3 Hz, 1H), 6.29 (s, 1H), 3.23 (s, 2H), 2.79–2.68 (m, 2H), 2.13 (s, 3H), 1.86–1.74 (m, 2H), 1.73–1.60 (m, 3H), 1.30–1.16 (m, 2H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 151.9, 148.1, 144.3, 140.9, 133.0, 130.7, 129.3 (2C), 124.8, 122.0, 121.4, 117.8 (2C), 107.9, 106.2, 91.1, 55.4 (2C), 47.7, 46.4, 34.7, 29.9 (2C) (3 quaternary C not observed); LCMS (2 min) <i>t</i><sub>R</sub> = 1.14 min, <i>m</i>/<i>z</i> (ESI<sup>+</sup>) 505 (M + H<sup>+</sup>); HRMS <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>26</sub>N<sub>8</sub>F<sup>35</sup>Cl (M + H) 505.2031, found 505.2013.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01721" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25255" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25255" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01721" class="ext-link">10.1021/acs.jmedchem.8b01721</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Mass spectrometry characterization of IRE1α proteins; schematic of the fluorescence-labeled stem-loop RNA used to measure inhibition of IRE1α RNase activity; selectivity of compounds <b>26</b> and <b>31</b> for IRE1α vs IRE1β kinase inhibition; kinase selectivity profiles of compounds <b>2</b>, <b>26</b>, and <b>31</b>; chemical structure and IRE1α inhibition data for <b>33</b>; crystal structure of <b>33</b>-IRE1α; model of <b>26</b> bound to IRE1α; microscopic images showing inhibition of stress-induced GFP-IRE1α foci formation in HEK293 cells by <b>26</b> and <b>31</b>; titrations of inhibition of stress-induced splicing of XBP1-Luc reporter mRNA in HEK293 cells by <b>26</b> and <b>31;</b> titrations of inhibition of stress-induced XBP1s mRNA expression in NCI-H929 cells by <b>26</b> and <b>31</b>; quantification of inhibition of pS724 IRE1α relative to total IRE1α using capillary electrophoresis immunoassay in H929 cells by <b>26</b> and <b>31</b>; microscopic images showing inhibition of stress-induced expression of XBP1s in NCI-H929 cells by <b>26</b> and <b>31</b>; titrations of the inhibition of IRE1α autophosphorylation, IRE1α ATP-site binding, and IRE1α RNase by compounds <b>2</b>, <b>26</b>, and <b>31</b> in in vitro assays; selectivity profiles of <b>2</b>, <b>26</b>, and <b>31</b> tested against 455 human kinases at 1000 nM; HPLC purities of compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b>; experimental method for the generation of an HEK293TN XBP1 luciferase reporter cell line; experimental methods to determine inhibition of an XBP1 luciferase reporter in HEK293TN cells and assess compound cytotoxicity to HEK293TN cells; experimental method for the quantification of XBP1s expression in NCI-H929 cells by immunofluorescence; experimental method for the quantification of XBP1s and DNAJB9 mRNA expression in NCI-H929 cells by qPCR; experimental method for the quantification of the inhibition of IRE1α-GFP oligomerization in T-Rex293-IRE1-3F6HGFP cells; experimental methods for the quantification of total IRE1α and pIRE1α expression in NCI-H929 cells; experimental method for the determination of the crystal structure of <b>33</b>-IRE1α; summary of crystallographic data for <b>2</b>-IRE1α and <b>33</b>-IRE1α; experimental methods for the synthesis of <b>33</b>; copies of <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b>; copies of HPLC traces for compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b>; and molecular formula strings for compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf">jm8b01721_si_001.pdf (5.62 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_002.csv">jm8b01721_si_002.csv (2.01 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID Codes: <b>2</b>-IRE1α, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>; <b>33</b>-IRE1α, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HV0">6HV0</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01721" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian Collins</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8143-8498" title="Orcid link">http://orcid.org/0000-0002-8143-8498</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#761f17185815191a1a1f1805361f150458171558031d"><span class="__cf_email__" data-cfemail="2049414e0e434f4c4c494e53604943520e41430e554b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giampiero Colombano</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John J. Caldwell</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas P. Matthews</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4765-7734" title="Orcid link">http://orcid.org/0000-0002-4765-7734</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chitra Bhatia</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cell Biology, University
of Leicester, Leicester LE1 7RH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amar Joshi</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cell Biology, University
of Leicester, Leicester LE1 7RH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tatiana McHardy</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ngai Yi Mok</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2827-3735" title="Orcid link">http://orcid.org/0000-0002-2827-3735</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yvette Newbatt</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa Pickard</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jade Strover</span> - <span class="hlFld-Affiliation affiliation">Division of Molecular Pathology and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Somaieh Hedayat</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael I. Walton</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephanie M. Myers</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan M. Jones</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harry Saville</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Craig McAndrew</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosemary Burke</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suzanne A. Eccles</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SW7 3RP, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Faith E. Davies</span> - <span class="hlFld-Affiliation affiliation">Division of Molecular Pathology and , The Institute of Cancer Research, London SW7 3RP, U.K.</span>; 
    <span>Present Address:
                        UAMS Myeloma Institute, 4018 W Capitol Avenue, Little Rock, Arkansas 72205, United States (F.E.D.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Bayliss</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cell Biology, University
of Leicester, Leicester LE1 7RH, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0604-2773" title="Orcid link">http://orcid.org/0000-0003-0604-2773</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors who are, or have been, employed by The Institute of Cancer Research are subject to a Rewards to Inventors Scheme which may reward contributors to a program that is subsequently licensed. The Institute of Cancer Research has a commercial interest in the development of inhibitors of IRE1-alpha.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6426-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55522" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55522" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Cancer Research UK grants C309/A11566 (to the Cancer Therapeutics Unit, ICR), C20826/A12 (to F.E.D.), C24461/A13231 and C24461/A12772 (to R.B.), Janssen Biotech Inc., and The Institute of Cancer Research. The authors thank Dr. Sebastian Naud for the original synthesis of <b>2</b>; Dr. Amin Mirza, Dr. Maggie Liu, and Meirion Richards for assistance in the structural characterization of compounds; Prof. Peter Walter for providing GFP-IRE1α HEK293 cells; Miri Jwa for details of the GFP-IRE1α oligomerization assay; Dr. Martin Augustin (Proteros Biostructures GmbH) for crystallography data; and Drs. Olivia Rossanese, Julian Blagg, Matthias Versele, and Ian Stansfield for helpful discussions and their interest in this work.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hetz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span> <span> </span><span class="NLM_article-title">The unfolded protein response and cell fate control</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.molcel.2017.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=29107536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehsL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2018&pages=169-181&author=C.+Hetzauthor=F.+R.+Papa&title=The+unfolded+protein+response+and+cell+fate+control&doi=10.1016%2Fj.molcel.2017.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Unfolded Protein Response and Cell Fate Control</span></div><div class="casAuthors">Hetz, Claudio; Papa, Feroz R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-181</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The secretory capacity of a cell is constantly challenged by physiol. demands and pathol. perturbations.  To adjust and match the protein-folding capacity of the endoplasmic reticulum (ER) to changing secretory needs, cells employ a dynamic intracellular signaling pathway known as the unfolded protein response (UPR).  Homeostatic activation of the UPR enforces adaptive programs that modulate and augment key aspects of the entire secretory pathway, whereas maladaptive UPR outputs trigger apoptosis.  Here, we discuss recent advances into how the UPR integrates information about the intensity and duration of ER stress stimuli in order to control cell fate.  These findings are timely and significant because they inform an evolving mechanistic understanding of a wide variety of human diseases, including diabetes mellitus, neurodegeneration, and cancer, thus opening up the potential for new therapeutic modalities to treat these diverse diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprBdkjctuvj7Vg90H21EOLACvtfcHk0lgdVxvHXR8afw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehsL7K&md5=59ef80187888b4c876e22bd63c8c0d7f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DHetz%26aufirst%3DC.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26atitle%3DThe%2520unfolded%2520protein%2520response%2520and%2520cell%2520fate%2520control%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D69%26spage%3D169%26epage%3D181%26doi%3D10.1016%2Fj.molcel.2017.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corazzari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagliardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fimia, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piacentini, M.</span></span> <span> </span><span class="NLM_article-title">Endoplasmic reticulum stress, unfolded protein response and cancer cell fate</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">78</span>, <span class="refDoi"> DOI: 10.3389/fonc.2017.00078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.3389%2Ffonc.2017.00078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28491820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC1crmtVCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=78&author=M.+Corazzariauthor=M.+Gagliardiauthor=G.+M.+Fimiaauthor=M.+Piacentini&title=Endoplasmic+reticulum+stress%2C+unfolded+protein+response+and+cancer+cell+fate&doi=10.3389%2Ffonc.2017.00078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Endoplasmic Reticulum Stress, Unfolded Protein Response, and Cancer Cell Fate</span></div><div class="casAuthors">Corazzari Marco; Corazzari Marco; Fimia Gian Maria; Piacentini Mauro; Gagliardi Mara; Piacentini Mauro; Fimia Gian Maria</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">78</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Perturbation of endoplasmic reticulum (ER) homeostasis results in a stress condition termed "ER stress" determining the activation of a finely regulated program defined as unfolded protein response (UPR) and whose primary aim is to restore this organelle's physiological activity.  Several physiological and pathological stimuli deregulate normal ER activity causing UPR activation, such as hypoxia, glucose shortage, genome instability, and cytotoxic compounds administration.  Some of these stimuli are frequently observed during uncontrolled proliferation of transformed cells, resulting in tumor core formation and stage progression.  Therefore, it is not surprising that ER stress is usually induced during solid tumor development and stage progression, becoming an hallmark of such malignancies.  Several UPR components are in fact deregulated in different tumor types, and accumulating data indicate their active involvement in tumor development/progression.  However, although the UPR program is primarily a pro-survival process, sustained and/or prolonged stress may result in cell death induction.  Therefore, understanding the mechanism(s) regulating the cell survival/death decision under ER stress condition may be crucial in order to specifically target tumor cells and possibly circumvent or overcome tumor resistance to therapies.  In this review, we discuss the role played by the UPR program in tumor initiation, progression and resistance to therapy, highlighting the recent advances that have improved our understanding of the molecular mechanisms that regulate the survival/death switch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPyD_CTZi66dNTrvPZP1IpfW6udTcc2eb6kdgFC-YgG7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crmtVCjtQ%253D%253D&md5=3f762464fa3a8ccc1e3151a84f936859</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2017.00078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2017.00078%26sid%3Dliteratum%253Aachs%26aulast%3DCorazzari%26aufirst%3DM.%26aulast%3DGagliardi%26aufirst%3DM.%26aulast%3DFimia%26aufirst%3DG.%2BM.%26aulast%3DPiacentini%26aufirst%3DM.%26atitle%3DEndoplasmic%2520reticulum%2520stress%252C%2520unfolded%2520protein%2520response%2520and%2520cancer%2520cell%2520fate%26jtitle%3DFront.%2520Oncol.%26date%3D2017%26volume%3D7%26spage%3D78%26doi%3D10.3389%2Ffonc.2017.00078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neculai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dever, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Structure of the dual enzyme Ire1 reveals the basis for catalysis and regulation in nonconventional RNA splicing</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.10.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.cell.2007.10.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=18191223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFehtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2008&pages=89-100&author=K.+P.+Leeauthor=M.+Deyauthor=D.+Neculaiauthor=C.+Caoauthor=T.+E.+Deverauthor=F.+Sicheri&title=Structure+of+the+dual+enzyme+Ire1+reveals+the+basis+for+catalysis+and+regulation+in+nonconventional+RNA+splicing&doi=10.1016%2Fj.cell.2007.10.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the dual enzyme Ire1 reveals the basis for catalysis and regulation in nonconventional RNA splicing</span></div><div class="casAuthors">Lee, Kenneth P. K.; Dey, Madhusudan; Neculai, Dante; Cao, Chune; Dever, Thomas E.; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ire1 is an ancient transmembrane sensor of ER stress with dual protein kinase and RNase activities.  In response to ER stress, Ire1 catalyzes the splicing of target mRNAs in a spliceosome-independent manner.  We have detd. the crystal structure of the dual catalytic region of Ire1 at 2.4 Å resoln., revealing the fusion of a domain, which we term the KEN domain, to the protein kinase domain.  Dimerization of the kinase domain composes a large catalytic surface on the KEN domain which carries out RNase function.  We further show that signal induced trans-autophosphorylation of the kinase domain permits unfettered binding of nucleotide, which in turn promotes dimerization to compose the RNase active site.  Comparison of Ire1 to a topol. disparate RNase reveals the convergent evolution of their catalytic mechanism.  These findings provide a basis for understanding the mechanism of action of RNaseL and other pseudokinases, which represent 10% of the human kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPv617JGYBAbVg90H21EOLACvtfcHk0lgOA4F9MNsoZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFehtbg%253D&md5=bc4f1a82988030a84b7cb4201437905d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.10.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.10.057%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%2BP.%26aulast%3DDey%26aufirst%3DM.%26aulast%3DNeculai%26aufirst%3DD.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DDever%26aufirst%3DT.%2BE.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DStructure%2520of%2520the%2520dual%2520enzyme%2520Ire1%2520reveals%2520the%2520basis%2520for%2520catalysis%2520and%2520regulation%2520in%2520nonconventional%2520RNA%2520splicing%26jtitle%3DCell%26date%3D2008%26volume%3D132%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.cell.2007.10.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korennykh, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egea, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korostelev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finer-Moore, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">The unfolded protein response signals through high-order assembly of Ire1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>457</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1038/nature07661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature07661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=19079236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=2009&pages=687-693&author=A.+V.+Korennykhauthor=P.+F.+Egeaauthor=A.+A.+Korostelevauthor=J.+Finer-Mooreauthor=C.+Zhangauthor=K.+M.+Shokatauthor=R.+M.+Stroudauthor=P.+Walter&title=The+unfolded+protein+response+signals+through+high-order+assembly+of+Ire1&doi=10.1038%2Fnature07661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The unfolded protein response signals through high-order assembly of Ire1</span></div><div class="casAuthors">Korennykh, Alexei V.; Egea, Pascal F.; Korostelev, Andrei A.; Finer-Moore, Janet; Zhang, Chao; Shokat, Kevan M.; Stroud, Robert M.; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">457</span>
        (<span class="NLM_cas:issue">7230</span>),
    <span class="NLM_cas:pages">687-693</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aberrant folding of proteins in the endoplasmic reticulum activates the bifunctional transmembrane kinase/endoribonuclease Ire1.  Ire1 excises an intron from HAC1 mRNA in yeasts and Xbp1 mRNA in metazoans encoding homologous transcription factors.  This non-conventional mRNA splicing event initiates the unfolded protein response, a transcriptional program that relieves the endoplasmic reticulum stress.  Here we show that oligomerization is central to Ire1 function and is an intrinsic attribute of its cytosolic domains.  We obtained the 3.2-Å crystal structure of the oligomer of the Ire1 cytosolic domains in complex with a kinase inhibitor that acts as a potent activator of the Ire1 RNase.  The structure reveals a rod-shaped assembly that has no known precedence among kinases.  This assembly positions the kinase domain for trans-autophosphorylation, orders the RNase domain, and creates an interaction surface for binding of the mRNA substrate.  Activation of Ire1 through oligomerization expands the mechanistic repertoire of kinase-based signaling receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomyUZIZFiup7Vg90H21EOLACvtfcHk0lgOA4F9MNsoZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ehtrw%253D&md5=51c74f5b87a912f4a9c1394a28357616</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature07661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07661%26sid%3Dliteratum%253Aachs%26aulast%3DKorennykh%26aufirst%3DA.%2BV.%26aulast%3DEgea%26aufirst%3DP.%2BF.%26aulast%3DKorostelev%26aufirst%3DA.%2BA.%26aulast%3DFiner-Moore%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DThe%2520unfolded%2520protein%2520response%2520signals%2520through%2520high-order%2520assembly%2520of%2520Ire1%26jtitle%3DNature%26date%3D2009%26volume%3D457%26spage%3D687%26epage%3D693%26doi%3D10.1038%2Fnature07661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagratuni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Santisteban, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrews, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aherne, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearl, L. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the Ire1 autophosphorylation complex and implications for the unfolded protein response</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">894</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1038/emboj.2011.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Femboj.2011.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21317875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=894-905&author=M.+M.+Aliauthor=T.+Bagratuniauthor=E.+L.+Davenportauthor=P.+R.+Nowakauthor=M.+C.+Silva-Santistebanauthor=A.+Hardcastleauthor=C.+McAndrewsauthor=M.+G.+Rowlandsauthor=G.+J.+Morganauthor=W.+Aherneauthor=I.+Collinsauthor=F.+E.+Daviesauthor=L.+H.+Pearl&title=Structure+of+the+Ire1+autophosphorylation+complex+and+implications+for+the+unfolded+protein+response&doi=10.1038%2Femboj.2011.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Ire1 autophosphorylation complex and implications for the unfolded protein response</span></div><div class="casAuthors">Ali, Maruf M. U.; Bagratuni, Tina; Davenport, Emma L.; Nowak, Piotr R.; Silva-Santisteban, M. Cris; Hardcastle, Anthea; McAndrews, Craig; Rowlands, Martin G.; Morgan, Gareth J.; Aherne, Wynne; Collins, Ian; Davies, Faith E.; Pearl, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">894-905</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ire1 (Ern1) is an unusual transmembrane protein kinase essential for the endoplasmic reticulum (ER) unfolded protein response (UPR).  Activation of Ire1 by assocn. of its N-terminal ER luminal domains promotes autophosphorylation by its cytoplasmic kinase domain, leading to activation of the C-terminal RNase domain, which splices Xbp1 mRNA, generating an active Xbp1s transcriptional activator.  We have detd. the crystal structure of the cytoplasmic portion of dephosphorylated human Ire1α bound to ADP, revealing the 'phosphoryl-transfer'-competent dimeric face-to-face complex, which precedes and is distinct from the back-to-back RNase 'active' conformation described for yeast Ire1.  We show that the Xbp1-specific RNase activity depends on autophosphorylation, and that ATP-competitive inhibitors staurosporin and sunitinib, which inhibit autophosphorylation in vitro, also inhibit Xbp1 splicing in vivo.  Furthermore, we demonstrate that activated Ire1α is a competent protein kinase, able to phosphorylate a heterologous peptide substrate.  These studies identify human Ire1α as a target for development of ATP-competitive inhibitors that will modulate the UPR in human cells, which has particular relevance for myeloma and other secretory malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQfUxv1Xx4JbVg90H21EOLACvtfcHk0lh8qLXumeoiwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtbw%253D&md5=b92aa98d4f38d5d46776e744ebe61980</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Femboj.2011.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2011.18%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DM.%2BM.%26aulast%3DBagratuni%26aufirst%3DT.%26aulast%3DDavenport%26aufirst%3DE.%2BL.%26aulast%3DNowak%26aufirst%3DP.%2BR.%26aulast%3DSilva-Santisteban%26aufirst%3DM.%2BC.%26aulast%3DHardcastle%26aufirst%3DA.%26aulast%3DMcAndrews%26aufirst%3DC.%26aulast%3DRowlands%26aufirst%3DM.%2BG.%26aulast%3DMorgan%26aufirst%3DG.%2BJ.%26aulast%3DAherne%26aufirst%3DW.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DDavies%26aufirst%3DF.%2BE.%26aulast%3DPearl%26aufirst%3DL.%2BH.%26atitle%3DStructure%2520of%2520the%2520Ire1%2520autophosphorylation%2520complex%2520and%2520implications%2520for%2520the%2520unfolded%2520protein%2520response%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D894%26epage%3D905%26doi%3D10.1038%2Femboj.2011.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of human IRE1 activation through dimerization and ligand binding</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">13019</span>– <span class="NLM_lpage">13035</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.18632%2Foncotarget.3864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=25968568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC2MfjsFSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=13019-13035&author=A.+Joshiauthor=Y.+Newbattauthor=P.+C.+McAndrewauthor=M.+Stubbsauthor=R.+Burkeauthor=M.+W.+Richardsauthor=C.+Bhatiaauthor=J.+J.+Caldwellauthor=T.+McHardyauthor=I.+Collinsauthor=R.+Bayliss&title=Molecular+mechanisms+of+human+IRE1+activation+through+dimerization+and+ligand+binding&doi=10.18632%2Foncotarget.3864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms of human IRE1 activation through dimerization and ligand binding</span></div><div class="casAuthors">Joshi Amar; Richards Mark W; Bhatia Chitra; Bayliss Richard; Joshi Amar; Richards Mark W; Bhatia Chitra; Bayliss Richard; Newbatt Yvette; McAndrew P Craig; Stubbs Mark; Burke Rosemary; Caldwell John J; McHardy Tatiana; Collins Ian</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">13019-35</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">IRE1 transduces the unfolded protein response by splicing XBP1 through its C-terminal cytoplasmic kinase-RNase region.  IRE1 autophosphorylation is coupled to RNase activity through formation of a back-to-back dimer, although the conservation of the underlying molecular mechanism is not clear from existing structures.  We have crystallized human IRE1 in a back-to-back conformation only previously seen for the yeast homologue.  In our structure the kinase domain appears primed for catalysis but the RNase domains are disengaged.  Structure-function analysis reveals that IRE1 is autoinhibited through a Tyr-down mechanism related to that found in the unrelated Ser/Thr protein kinase Nek7.  We have developed a compound that potently inhibits human IRE1 kinase activity while stimulating XBP1 splicing.  A crystal structure of the inhibitor bound to IRE1 shows an increased ordering of the kinase activation loop.  The structures of hIRE in apo and ligand-bound forms are consistent with a previously proposed model of IRE1 regulation in which formation of a back-to-back dimer coupled to adoption of a kinase-active conformation drive RNase activation.  The structures provide opportunities for structure-guided design of IRE1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKXDQm87QVYFw2ofyr07jXfW6udTcc2eb1a2ykzIhu_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfjsFSktQ%253D%253D&md5=6005c96ebea157a65583d679acec5cbf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3864%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DA.%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DMcAndrew%26aufirst%3DP.%2BC.%26aulast%3DStubbs%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DBhatia%26aufirst%3DC.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DMcHardy%26aufirst%3DT.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DMolecular%2520mechanisms%2520of%2520human%2520IRE1%2520activation%2520through%2520dimerization%2520and%2520ligand%2520binding%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D13019%26epage%3D13035%26doi%3D10.18632%2Foncotarget.3864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, K.</span></span> <span> </span><span class="NLM_article-title">XBP1 mRNA is induced by ATF6 and spliced by IRE1 in response to ER stress to produce a highly active transcription factor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">891</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(01)00611-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2FS0092-8674%2801%2900611-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=11779464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtlKlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2001&pages=881-891&author=H.+Yoshidaauthor=T.+Matsuiauthor=A.+Yamamotoauthor=T.+Okadaauthor=K.+Mori&title=XBP1+mRNA+is+induced+by+ATF6+and+spliced+by+IRE1+in+response+to+ER+stress+to+produce+a+highly+active+transcription+factor&doi=10.1016%2FS0092-8674%2801%2900611-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">XBP1 mRNA is induced by ATF6 and spliced by IRE1 in response to ER stress to produce a highly active transcription factor</span></div><div class="casAuthors">Yoshida, Hiderou; Matsui, Toshie; Yamamoto, Akira; Okada, Tetsuya; Mori, Kazutoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">881-891</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In yeast, the transmembrane protein kinase/endoribonuclease Ire1p activated by endoplasmic reticulum stress cleaves HAC1 mRNA, leading to prodn. of the transcription factor Hac1p that activates the unfolded protein response (UPR).  In mammals, no Hac1p counterpart has yet been discovered despite the presence of Ire1p homologs in the endoplasmic reticulum.  Instead, the transcription factor ATF6 specific to the mammalian UPR is regulated by intramembrane proteolysis.  Here, we identified the transcription factor XBP1, a target of ATF6, as a mammalian substrate of such an unconventional mRNA splicing system and showed that only the spliced form of XBP1 can activate the UPR efficiently.  Our results reveal features of the UPR conserved during evolution and clarify the relationship between IRE1- and ATF6-dependent pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo67siPWlzG7LVg90H21EOLACvtfcHk0lgUtd96dxb7dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtlKlsg%253D%253D&md5=e873cf424078a4964d0335c2529775f2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900611-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900611-0%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DK.%26atitle%3DXBP1%2520mRNA%2520is%2520induced%2520by%2520ATF6%2520and%2520spliced%2520by%2520IRE1%2520in%2520response%2520to%2520ER%2520stress%2520to%2520produce%2520a%2520highly%2520active%2520transcription%2520factor%26jtitle%3DCell%26date%3D2001%26volume%3D107%26spage%3D881%26epage%3D891%26doi%3D10.1016%2FS0092-8674%2801%2900611-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwakoshi, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span> <span> </span><span class="NLM_article-title">XBP-1 regulates a subset of endoplasmic reticulum resident chaperone genes in the unfolded protein response</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">7448</span>– <span class="NLM_lpage">7459</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.21.7448-7459.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1128%2FMCB.23.21.7448-7459.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=14559994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosFOgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=7448-7459&author=A.+H.+Leeauthor=N.+N.+Iwakoshiauthor=L.+H.+Glimcher&title=XBP-1+regulates+a+subset+of+endoplasmic+reticulum+resident+chaperone+genes+in+the+unfolded+protein+response&doi=10.1128%2FMCB.23.21.7448-7459.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">XBP-1 regulates a subset of endoplasmic reticulum resident chaperone genes in the unfolded protein response</span></div><div class="casAuthors">Lee, Ann-Hwee; Iwakoshi, Neal N.; Glimcher, Laurie H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7448-7459</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mammalian unfolded protein response (UPR) protects the cell against the stress of misfolded proteins in the endoplasmic reticulum (ER).  We have investigated here the contribution of the UPR transcription factors XBP-1, ATF6α, and ATF6β to UPR target gene expression.  Gene profiling of cell lines lacking these factors yielded several XBP-1-dependent UPR target genes, all of which appear to act in the ER.  These included the DnaJ/Hsp40-like genes, p58IPK, ERdj4, and HEDJ, as well as EDEM, protein disulfide isomerase-P5, and ribosome-assocd. membrane protein 4 (RAMP4), whereas expression of BiP was only modestly dependent on XBP-1.  Surprisingly, given previous reports that enforced expression of ATF6α induced a subset of UPR target genes, cells deficient in ATF6α, ATF6β, or both had minimal defects in upregulating UPR target genes by gene profiling anal., suggesting the presence of compensatory mechanism(s) for ATF6 in the UPR.  Since cells lacking both XBP-1 and ATF6α had significantly impaired induction of select UPR target genes and ERSE reporter activation, XBP-1 and ATF6α may serve partially redundant functions.  No UPR target genes that required ATF6β were identified, nor, in contrast to XBP-1 and ATF6α, did the activity of the UPRE or ERSE promoters require ATF6β, suggesting a minor role for it during the UPR.  Collectively, these results suggest that the IRE1/XBP-1 pathway is required for efficient protein folding, maturation, and degrdn. in the ER and imply the existence of subsets of UPR target genes as defined by their dependence on XBP-1.  Further, our observations suggest the existence of addnl., as-yet-unknown, key regulators of the UPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9zXJxAEOnKbVg90H21EOLACvtfcHk0lgUtd96dxb7dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosFOgsLY%253D&md5=952311f7921081765f45ae2ad02331c6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.21.7448-7459.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.21.7448-7459.2003%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BH.%26aulast%3DIwakoshi%26aufirst%3DN.%2BN.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26atitle%3DXBP-1%2520regulates%2520a%2520subset%2520of%2520endoplasmic%2520reticulum%2520resident%2520chaperone%2520genes%2520in%2520the%2520unfolded%2520protein%2520response%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D7448%26epage%3D7459%26doi%3D10.1128%2FMCB.23.21.7448-7459.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Decay of endoplasmic reticulum-localized mRNAs during the unfolded protein response</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1126/science.1129631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1126%2Fscience.1129631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=16825573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFWjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=104-107&author=J.+Hollienauthor=J.+S.+Weissman&title=Decay+of+endoplasmic+reticulum-localized+mRNAs+during+the+unfolded+protein+response&doi=10.1126%2Fscience.1129631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Decay of Endoplasmic Reticulum-Localized mRNAs During the Unfolded Protein Response</span></div><div class="casAuthors">Hollien, Julie; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5783</span>),
    <span class="NLM_cas:pages">104-107</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The unfolded protein response (UPR) allows the endoplasmic reticulum (ER) to recover from the accumulation of misfolded proteins, in part by increasing its folding capacity.  Inositol-requiring enzyme-1 (IRE1) promotes this remodeling by detecting misfolded ER proteins and activating a transcription factor, X-box-binding protein 1, through endonucleolytic cleavage of its mRNA.  Here, we report that IRE1 independently mediates the rapid degrdn. of a specific subset of mRNAs, based both on their localization to the ER membrane and on the amino acid sequence they encode.  This response is well suited to complement other UPR mechanisms because it could selectively halt prodn. of proteins that challenge the ER and clear the translocation and folding machinery for the subsequent remodeling process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIQKpC2dcLGbVg90H21EOLACvtfcHk0ljLBM5OcWo7LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFWjurc%253D&md5=7c9d3114078742b3cbe3457fb91643fd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1129631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1129631%26sid%3Dliteratum%253Aachs%26aulast%3DHollien%26aufirst%3DJ.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DDecay%2520of%2520endoplasmic%2520reticulum-localized%2520mRNAs%2520during%2520the%2520unfolded%2520protein%2520response%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D104%26epage%3D107%26doi%3D10.1126%2Fscience.1129631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ron, D.</span></span> <span> </span><span class="NLM_article-title">Coupling of stress in the ER to activation of JNK protein kinases by transmembrane protein kinase IRE1</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1126/science.287.5453.664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1126%2Fscience.287.5453.664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10650002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVyhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=664-666&author=F.+Uranoauthor=X.+Wangauthor=A.+Bertolottiauthor=Y.+Zhangauthor=P.+Chungauthor=H.+P.+Hardingauthor=D.+Ron&title=Coupling+of+stress+in+the+ER+to+activation+of+JNK+protein+kinases+by+transmembrane+protein+kinase+IRE1&doi=10.1126%2Fscience.287.5453.664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Coupling of stress in the ER to activation of JNK protein kinases by transmembrane protein kinase IRE1</span></div><div class="casAuthors">Urano, Fumihiko; Wang, Xiaozhong; Bertolotti, Anne; Zhang, Yuhong; Chung, Peter; Harding, Heather P.; Ron, David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">5453</span>),
    <span class="NLM_cas:pages">664-666</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Malfolded proteins in the endoplasmic reticulum (ER) induce cellular stress and activate c-Jun amino-terminal kinases (JNKs or SAPKs).  Mammalian homologs of yeast IRE1, which activate chaperone genes in response to ER stress, also activated JNK, and IRE1α-/- fibroblasts were impaired in JNK activation by ER stress.  The cytoplasmic part of IRE1 bound TRAF2, an adaptor protein that couples plasma membrane receptors to JNK activation.  Dominant-neg. TRAF2 inhibited activation of JNK by IRE1.  Activation of JNK by endogenous signals initiated in the ER proceeds by a pathway similar to that initiated by cell surface receptors in response to extracellular signals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOjeif2GC19LVg90H21EOLACvtfcHk0ljLBM5OcWo7LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVyhuw%253D%253D&md5=e8bac4aee53f9359a6f20b93cb4e0185</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.287.5453.664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.287.5453.664%26sid%3Dliteratum%253Aachs%26aulast%3DUrano%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBertolotti%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DP.%26aulast%3DHarding%26aufirst%3DH.%2BP.%26aulast%3DRon%26aufirst%3DD.%26atitle%3DCoupling%2520of%2520stress%2520in%2520the%2520ER%2520to%2520activation%2520of%2520JNK%2520protein%2520kinases%2520by%2520transmembrane%2520protein%2520kinase%2520IRE1%26jtitle%3DScience%26date%3D2000%26volume%3D287%26spage%3D664%26epage%3D666%26doi%3D10.1126%2Fscience.287.5453.664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, R. J.</span></span> <span> </span><span class="NLM_article-title">Protein misfolding in the endoplasmic reticulum as a conduit to human disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature17041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature17041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=26791723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1OjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=326-335&author=M.+Wangauthor=R.+J.+Kaufman&title=Protein+misfolding+in+the+endoplasmic+reticulum+as+a+conduit+to+human+disease&doi=10.1038%2Fnature17041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein misfolding in the endoplasmic reticulum as a conduit to human disease</span></div><div class="casAuthors">Wang, Miao; Kaufman, Randal J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">326-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In eukaryotic cells, the endoplasmic reticulum is essential for the folding and trafficking of proteins that enter the secretory pathway.  Environmental insults or increased protein synthesis often lead to protein misfolding in the organelle, the accumulation of misfolded or unfolded proteins - known as endoplasmic reticulum stress - and the activation of the adaptive unfolded protein response to restore homeostasis.  If protein misfolding is not resolved, cells death occurs.  Endoplasmic reticulum stress and activation of the unfolded protein response help to det. cell fate and function.  Furthermore, endoplasmic reticulum stress contributes to the etiol. of many human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCscbs52xg07Vg90H21EOLACvtfcHk0ljLBM5OcWo7LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1OjurY%253D&md5=2b7470a4f2ef81181d5f2c1c5e629488</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature17041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17041%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DKaufman%26aufirst%3DR.%2BJ.%26atitle%3DProtein%2520misfolding%2520in%2520the%2520endoplasmic%2520reticulum%2520as%2520a%2520conduit%2520to%2520human%2520disease%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D326%26epage%3D335%26doi%3D10.1038%2Fnature17041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukhdeo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protopopova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkus, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protopopov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span> <span> </span><span class="NLM_article-title">The differentiation and stress response factor XBP-1 drives multiple myeloma pathogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2007.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.ccr.2007.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=17418411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksFWjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=349-360&author=D.+R.+Carrascoauthor=K.+Sukhdeoauthor=M.+Protopopovaauthor=R.+Sinhaauthor=M.+Enosauthor=D.+E.+Carrascoauthor=M.+Zhengauthor=M.+Maniauthor=J.+Hendersonauthor=G.+S.+Pinkusauthor=N.+Munshiauthor=J.+Hornerauthor=E.+V.+Ivanovaauthor=A.+Protopopovauthor=K.+C.+Andersonauthor=G.+Tononauthor=R.+A.+DePinho&title=The+differentiation+and+stress+response+factor+XBP-1+drives+multiple+myeloma+pathogenesis&doi=10.1016%2Fj.ccr.2007.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The differentiation and stress response factor XBP-1 drives multiple myeloma pathogenesis</span></div><div class="casAuthors">Carrasco, Daniel R.; Sukhdeo, Kumar; Protopopova, Marina; Sinha, Raktim; Enos, Miriam; Carrasco, Daniel E.; Zheng, Mei; Mani, Mala; Henderson, Joel; Pinkus, Geraldine S.; Munshi, Nikhil; Horner, James; Ivanova, Elena V.; Protopopov, Alexei; Anderson, Kenneth C.; Tonon, Giovanni; DePinho, Ronald A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-360</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) evolves from a highly prevalent premalignant condition termed MGUS.  The factors underlying the malignant transformation of MGUS are unknown.  We report a MGUS/MM phenotype in transgenic mice with Eμ-directed expression of the XBP-1 spliced isoform (XBP-1s), a factor governing unfolded protein/ER stress response and plasma-cell development.  Eμ-XBP-1s elicited elevated serum Ig and skin alterations.  With age, Eμ-xbp-1s transgenics develop features diagnostic of human MM, including bone lytic lesions and subendothelial Ig deposition.  Furthermore, transcriptional profiles of Eμ-xbp-1s lymphoid and MM cells show aberrant expression of known human MM dysregulated genes.  The similarities of this model with the human disease, coupled with documented frequent XBP-1s overexpression in human MM, serve to implicate XBP-1s dysregulation in MM pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGRySDE6u1uLVg90H21EOLACvtfcHk0lioMr4pWcHLkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksFWjsro%253D&md5=8daad15d72f9924c73be3251ac6877ab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2007.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2007.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DCarrasco%26aufirst%3DD.%2BR.%26aulast%3DSukhdeo%26aufirst%3DK.%26aulast%3DProtopopova%26aufirst%3DM.%26aulast%3DSinha%26aufirst%3DR.%26aulast%3DEnos%26aufirst%3DM.%26aulast%3DCarrasco%26aufirst%3DD.%2BE.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DMani%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DJ.%26aulast%3DPinkus%26aufirst%3DG.%2BS.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DHorner%26aufirst%3DJ.%26aulast%3DIvanova%26aufirst%3DE.%2BV.%26aulast%3DProtopopov%26aufirst%3DA.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DTonon%26aufirst%3DG.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26atitle%3DThe%2520differentiation%2520and%2520stress%2520response%2520factor%2520XBP-1%2520drives%2520multiple%2520myeloma%2520pathogenesis%26jtitle%3DCancer%2520Cell%26date%3D2007%26volume%3D11%26spage%3D349%26epage%3D360%26doi%3D10.1016%2Fj.ccr.2007.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iliopoulos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatziapostolou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adoro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landis, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span> <span> </span><span class="NLM_article-title">XBP1 promotes triple-negative breast cancer by controlling the HIF1α pathway</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1038/nature13119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature13119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=24670641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsFOqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=103-107&author=X.+Chenauthor=D.+Iliopoulosauthor=Q.+Zhangauthor=Q.+Tangauthor=M.+B.+Greenblattauthor=M.+Hatziapostolouauthor=E.+Limauthor=W.+L.+Tamauthor=M.+Niauthor=Y.+Chenauthor=J.+Maiauthor=H.+Shenauthor=D.+Z.+Huauthor=S.+Adoroauthor=B.+Huauthor=M.+Songauthor=C.+Tanauthor=M.+D.+Landisauthor=M.+Ferrariauthor=S.+J.+Shinauthor=M.+Brownauthor=J.+C.+Changauthor=X.+S.+Liuauthor=L.+H.+Glimcher&title=XBP1+promotes+triple-negative+breast+cancer+by+controlling+the+HIF1%CE%B1+pathway&doi=10.1038%2Fnature13119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">XBP1 promotes triple-negative breast cancer by controlling the HIF1α pathway</span></div><div class="casAuthors">Chen, Xi; Iliopoulos, Dimitrios; Zhang, Qing; Tang, Qianzi; Greenblatt, Matthew B.; Hatziapostolou, Maria; Lim, Elgene; Tam, Wai Leong; Ni, Min; Chen, Yiwen; Mai, Junhua; Shen, Haifa; Hu, Dorothy Z.; Adoro, Stanley; Hu, Bella; Song, Minkyung; Tan, Chen; Landis, Melissa D.; Ferrari, Mauro; Shin, Sandra J.; Brown, Myles; Chang, Jenny C.; Liu, X. Shirley; Glimcher, Laurie H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7494</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells induce a set of adaptive response pathways to survive in the face of stressors due to inadequate vascularization.  One such adaptive pathway is the unfolded protein (UPR) or endoplasmic reticulum (ER) stress response mediated in part by the ER-localized transmembrane sensor IRE1 (ref. 2) and its substrate XBP1 (ref. 3).  Previous studies report UPR activation in various human tumors, but the role of XBP1 in cancer progression in mammary epithelial cells is largely unknown.  Triple-neg. breast cancer (TNBC)-a form of breast cancer in which tumor cells do not express the genes for estrogen receptor, progesterone receptor and HER2 (also called ERBB2 or NEU)-is a highly aggressive malignancy with limited treatment options.  Here we report that XBP1 is activated in TNBC and has a pivotal role in the tumorigenicity and progression of this human breast cancer subtype.  In breast cancer cell line models, depletion of XBP1 inhibited tumor growth and tumor relapse and reduced the CD44highCD24low population.  Hypoxia-inducing factor 1α (HIF1α) is known to be hyperactivated in TNBCs.  Genome-wide mapping of the XBP1 transcriptional regulatory network revealed that XBP1 drives TNBC tumorigenicity by assembling a transcriptional complex with HIF1α that regulates the expression of HIF1α targets via the recruitment of RNA polymerase II.  Anal. of independent cohorts of patients with TNBC revealed a specific XBP1 gene expression signature that was highly correlated with HIF1α and hypoxia-driven signatures and that strongly assocd. with poor prognosis.  Our findings reveal a key function for the XBP1 branch of the UPR in TNBC and indicate that targeting this pathway may offer alternative treatment strategies for this aggressive subtype of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFeZj_3EFRXbVg90H21EOLACvtfcHk0lioMr4pWcHLkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsFOqtLY%253D&md5=7f5b8d5b2699ce4e92c8a54e6b33091f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature13119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13119%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DIliopoulos%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTang%26aufirst%3DQ.%26aulast%3DGreenblatt%26aufirst%3DM.%2BB.%26aulast%3DHatziapostolou%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DE.%26aulast%3DTam%26aufirst%3DW.%2BL.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DMai%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DD.%2BZ.%26aulast%3DAdoro%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DLandis%26aufirst%3DM.%2BD.%26aulast%3DFerrari%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DS.%2BJ.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DJ.%2BC.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26atitle%3DXBP1%2520promotes%2520triple-negative%2520breast%2520cancer%2520by%2520controlling%2520the%2520HIF1%25CE%25B1%2520pathway%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D103%26epage%3D107%26doi%3D10.1038%2Fnature13119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Storm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnoldussen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saatcioglu, F.</span></span> <span> </span><span class="NLM_article-title">Prostate cancer and the unfolded protein response</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">54051</span>– <span class="NLM_lpage">54066</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.18632%2Foncotarget.9912" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=54051-54066&author=M.+Stormauthor=X.+Shengauthor=Y.+J.+Arnoldussenauthor=F.+Saatcioglu&title=Prostate+cancer+and+the+unfolded+protein+response&doi=10.18632%2Foncotarget.9912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9912%26sid%3Dliteratum%253Aachs%26aulast%3DStorm%26aufirst%3DM.%26aulast%3DSheng%26aufirst%3DX.%26aulast%3DArnoldussen%26aufirst%3DY.%2BJ.%26aulast%3DSaatcioglu%26aufirst%3DF.%26atitle%3DProstate%2520cancer%2520and%2520the%2520unfolded%2520protein%2520response%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D54051%26epage%3D54066%26doi%3D10.18632%2Foncotarget.9912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cubillos-Ruiz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silberman, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perales-Puchalt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettigole, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellenson, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span> <span> </span><span class="NLM_article-title">ER stress sensor XBP1 controls anti-tumor immunity by disrupting dendritic cell homeostasis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">1527</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.05.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.cell.2015.05.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=26073941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVeiu77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2015&pages=1527-1538&author=J.+R.+Cubillos-Ruizauthor=P.+C.+Silbermanauthor=M.+R.+Rutkowskiauthor=S.+Chopraauthor=A.+Perales-Puchaltauthor=M.+Songauthor=S.+Zhangauthor=S.+E.+Bettigoleauthor=D.+Guptaauthor=K.+Holcombauthor=L.+H.+Ellensonauthor=T.+Caputoauthor=A.+H.+Leeauthor=J.+R.+Conejo-Garciaauthor=L.+H.+Glimcher&title=ER+stress+sensor+XBP1+controls+anti-tumor+immunity+by+disrupting+dendritic+cell+homeostasis&doi=10.1016%2Fj.cell.2015.05.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ER stress sensor XBP1 controls anti-tumor immunity by disrupting dendritic cell homeostasis</span></div><div class="casAuthors">Cubillos-Ruiz, Juan R.; Silberman, Pedro C.; Rutkowski, Melanie R.; Chopra, Sahil; Perales-Puchalt, Alfredo; Song, Minkyung; Zhang, Sheng; Bettigole, Sarah E.; Gupta, Divya; Holcomb, Kevin; Ellenson, Lora H.; Caputo, Thomas; Lee, Ann-Hwee; Conejo-Garcia, Jose R.; Glimcher, Laurie H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1527-1538</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Dendritic cells (DCs) are required to initiate and sustain T cell-dependent anti-cancer immunity.  However, tumors often evade immune control by crippling normal DC function.  The endoplasmic reticulum (ER) stress response factor XBP1 promotes intrinsic tumor growth directly, but whether it also regulates the host anti-tumor immune response is not known.  Here we show that constitutive activation of XBP1 in tumor-assocd. DCs (tDCs) drives ovarian cancer (OvCa) progression by blunting anti-tumor immunity.  XBP1 activation, fueled by lipid peroxidn. byproducts, induced a triglyceride biosynthetic program in tDCs leading to abnormal lipid accumulation and subsequent inhibition of tDC capacity to support anti-tumor T cells.  Accordingly, DC-specific XBP1 deletion or selective nanoparticle-mediated XBP1 silencing in tDCs restored their immunostimulatory activity in situ and extended survival by evoking protective type 1 anti-tumor responses.  Targeting the ER stress response should concomitantly inhibit tumor growth and enhance anti-cancer immunity, thus offering a unique approach to cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo12xOn4E_-B7Vg90H21EOLACvtfcHk0lj8YwVxjo-K7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVeiu77J&md5=00e069a6bfd44b300645617cd534f121</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.05.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.05.025%26sid%3Dliteratum%253Aachs%26aulast%3DCubillos-Ruiz%26aufirst%3DJ.%2BR.%26aulast%3DSilberman%26aufirst%3DP.%2BC.%26aulast%3DRutkowski%26aufirst%3DM.%2BR.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DPerales-Puchalt%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DBettigole%26aufirst%3DS.%2BE.%26aulast%3DGupta%26aufirst%3DD.%26aulast%3DHolcomb%26aufirst%3DK.%26aulast%3DEllenson%26aufirst%3DL.%2BH.%26aulast%3DCaputo%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DA.%2BH.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26atitle%3DER%2520stress%2520sensor%2520XBP1%2520controls%2520anti-tumor%2520immunity%2520by%2520disrupting%2520dendritic%2520cell%2520homeostasis%26jtitle%3DCell%26date%3D2015%26volume%3D161%26spage%3D1527%26epage%3D1538%26doi%3D10.1016%2Fj.cell.2015.05.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volkmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuga, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brumm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriebel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der-Sarkissian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweitzer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malyankar, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiovitti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canny, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. B.</span></span> <span> </span><span class="NLM_article-title">Potent and selective inhibitors of the inositol-requiring enzyme 1 endoribonuclease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">12743</span>– <span class="NLM_lpage">12755</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.199737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1074%2Fjbc.M110.199737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21303903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=12743-12755&author=K.+Volkmannauthor=J.+L.+Lucasauthor=D.+Vugaauthor=X.+Wangauthor=D.+Brummauthor=C.+Stilesauthor=D.+Kriebelauthor=A.+Der-Sarkissianauthor=K.+Krishnanauthor=C.+Schweitzerauthor=Z.+Liuauthor=U.+M.+Malyankarauthor=D.+Chiovittiauthor=M.+Cannyauthor=D.+Durocherauthor=F.+Sicheriauthor=J.+B.+Patterson&title=Potent+and+selective+inhibitors+of+the+inositol-requiring+enzyme+1+endoribonuclease&doi=10.1074%2Fjbc.M110.199737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Inhibitors of the Inositol-requiring Enzyme 1 Endoribonuclease</span></div><div class="casAuthors">Volkmann, Kori; Lucas, Julie L.; Vuga, Danka; Wang, Xiao-Ping; Brumm, Duane; Stiles, Caryn; Kriebel, David; Der-Sarkissian, Ani; Krishnan, Kris; Schweitzer, Colleen; Liu, Zheng; Malyankar, Uriel M.; Chiovitti, David; Canny, Marella; Durocher, Dan; Sicheri, Frank; Patterson, John B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">12743-12755</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Inositol-requiring enzyme 1 (IRE1) is the most highly conserved signaling node of the unfolded protein response (UPR) and represents a potential therapeutic target for a no. of diseases assocd. with endoplasmic reticulum stress.  IRE1 activates the XBP-1 transcription factor by site-specific cleavage of two hairpin loops within its mRNA to facilitate its nonconventional splicing and alternative translation.  We screened for inhibitors using a construct contg. the unique cytosolic kinase and endoribonuclease domains of human IRE1α (hIRE1α-cyto) and a mini-XBP-1 stem-loop RNA as the substrate.  One class compds. was salicylaldehyde analogs from the hydrolyzed product of salicylaldimines in the library.  Salicylaldehyde analogs were active in inhibiting the site-specific cleavage of several mini-XBP-1 stem-loop RNAs in a dose-dependent manner.  Salicyaldehyde analogs were also active in inhibiting yeast Ire1 but had little activity inhibiting RNase L or the unrelated RNases A and T1.  Kinetic anal. revealed that one potent salicylaldehyde analog, 3-ethoxy-5,6-dibromosalicylaldehyde, is a non-competitive inhibitor with respect to the XBP-1 RNA substrate.  Surface plasmon resonance studies confirmed this compd. bound to IRE1 in a specific, reversible and dose-dependent manner.  Salicylaldehydes inhibited XBP-1 splicing induced pharmacol. in human cells.  These compds. also blocked transcriptional up-regulation of known XBP-1 targets as well as mRNAs targeted for degrdn. by IRE1.  Finally, the salicylaldehyde analog 3-methoxy-6-bromosalicylaldehyde strongly inhibited XBP-1 splicing in an in vivo model of acute endoplasmic reticulum stress.  To our knowledge, salicylaldehyde analogs are the first reported specific IRE1 endoribonuclease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrguJWBkL4i1bVg90H21EOLACvtfcHk0liMS37-HxKGIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOjsLg%253D&md5=e81641293ee0b2bd0a620d54d78241df</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.199737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.199737%26sid%3Dliteratum%253Aachs%26aulast%3DVolkmann%26aufirst%3DK.%26aulast%3DLucas%26aufirst%3DJ.%2BL.%26aulast%3DVuga%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBrumm%26aufirst%3DD.%26aulast%3DStiles%26aufirst%3DC.%26aulast%3DKriebel%26aufirst%3DD.%26aulast%3DDer-Sarkissian%26aufirst%3DA.%26aulast%3DKrishnan%26aufirst%3DK.%26aulast%3DSchweitzer%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DMalyankar%26aufirst%3DU.%2BM.%26aulast%3DChiovitti%26aufirst%3DD.%26aulast%3DCanny%26aufirst%3DM.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DPatterson%26aufirst%3DJ.%2BB.%26atitle%3DPotent%2520and%2520selective%2520inhibitors%2520of%2520the%2520inositol-requiring%2520enzyme%25201%2520endoribonuclease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D12743%26epage%3D12755%26doi%3D10.1074%2Fjbc.M110.199737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cross, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxendale, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ron, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, H. P.</span></span> <span> </span><span class="NLM_article-title">The molecular basis for selective inhibition of unconventional mRNA splicing by an IRE1-binding small molecule</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">E869</span>– <span class="NLM_lpage">E878</span>, <span class="refDoi"> DOI: 10.1073/pnas.1115623109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1073%2Fpnas.1115623109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=22315414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVCgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=E869-E878&author=B.+C.+Crossauthor=P.+J.+Bondauthor=P.+G.+Sadowskiauthor=B.+K.+Jhaauthor=J.+Zakauthor=J.+M.+Goodmanauthor=R.+H.+Silvermanauthor=T.+A.+Neubertauthor=I.+R.+Baxendaleauthor=D.+Ronauthor=H.+P.+Harding&title=The+molecular+basis+for+selective+inhibition+of+unconventional+mRNA+splicing+by+an+IRE1-binding+small+molecule&doi=10.1073%2Fpnas.1115623109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular basis for selective inhibition of unconventional mRNA splicing by an IRE1-binding small molecule</span></div><div class="casAuthors">Cross, Benedict C. S.; Bond, Peter J.; Sadowski, Pawel G.; Jha, Babal Kant; Zak, Jaroslav; Goodman, Jonathan M.; Silverman, Robert H.; Neubert, Thomas A.; Baxendale, Ian R.; Ron, David; Harding, Heather P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">E869-E878, SE869/1-SE869/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">IRE1 couples endoplasmic reticulum unfolded protein load to RNA cleavage events that culminate in the sequence-specific splicing of the Xbp1 mRNA and in the regulated degrdn. of diverse membrane-bound mRNAs.  We report on the identification of a small mol. inhibitor that attains its selectivity by forming an unusually stable Schiff base with lysine 907 in the IRE1 endonuclease domain, explained by solvent inaccessibility of the imine bond in the enzyme-inhibitor complex.  The inhibitor (abbreviated 4μ8C) blocks substrate access to the active site of IRE1 and selectively inactivates both Xbp1 splicing and IRE1-mediated mRNA degrdn.  Surprisingly, inhibition of IRE1 endonuclease activity does not sensitize cells to the consequences of acute endoplasmic reticulum stress, but rather interferes with the expansion of secretory capacity.  Thus, the chem. reactivity and sterics of a unique residue in the endonuclease active site of IRE1 can be exploited by selective inhibitors to interfere with protein secretion in pathol. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdJR15M_rc7rVg90H21EOLACvtfcHk0ljDmpg9ywSn0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVCgtb8%253D&md5=f6a05686d1b4a8a25d33fea41270f3ee</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1115623109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1115623109%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DB.%2BC.%26aulast%3DBond%26aufirst%3DP.%2BJ.%26aulast%3DSadowski%26aufirst%3DP.%2BG.%26aulast%3DJha%26aufirst%3DB.%2BK.%26aulast%3DZak%26aufirst%3DJ.%26aulast%3DGoodman%26aufirst%3DJ.%2BM.%26aulast%3DSilverman%26aufirst%3DR.%2BH.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DBaxendale%26aufirst%3DI.%2BR.%26aulast%3DRon%26aufirst%3DD.%26aulast%3DHarding%26aufirst%3DH.%2BP.%26atitle%3DThe%2520molecular%2520basis%2520for%2520selective%2520inhibition%2520of%2520unconventional%2520mRNA%2520splicing%2520by%2520an%2520IRE1-binding%2520small%2520molecule%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3DE869%26epage%3DE878%26doi%3D10.1073%2Fpnas.1115623109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ming, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span> <span> </span><span class="NLM_article-title">A novel chemical, STF-083010, reverses tamoxifen-related drug resistance in breast cancer by inhibiting IRE1/XBP1</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">40692</span>– <span class="NLM_lpage">40703</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.5827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.18632%2Foncotarget.5827" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=26517687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC28zmvFSrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=40692-40703&author=J.+Mingauthor=S.+Ruanauthor=M.+Wangauthor=D.+Yeauthor=N.+Fanauthor=Q.+Mengauthor=B.+Tianauthor=T.+Huang&title=A+novel+chemical%2C+STF-083010%2C+reverses+tamoxifen-related+drug+resistance+in+breast+cancer+by+inhibiting+IRE1%2FXBP1&doi=10.18632%2Foncotarget.5827"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A novel chemical, STF-083010, reverses tamoxifen-related drug resistance in breast cancer by inhibiting IRE1/XBP1</span></div><div class="casAuthors">Ming Jie; Ruan Shengnan; Wang Mengyi; Ye Dan; Huang Tao; Fan Ningning; Meng Qingyu; Tian Bo; Tian Bo</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">40692-703</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent studies show that the unfolded protein response (UPR) within the endoplasmic reticulum is correlated with breast cancer drug resistance.  In particular, human X-box binding protein-1(XBP1), a transcription factor which participates in UPR stress signaling, is reported to correlate with poor clinical responsiveness to tamoxifen.  In this study, we develop a tamoxifen-resistant MCF-7 cell line by treating the cell line with low concentration of tamoxifen, and we find that XBP1 is indeed up-regulated at both the mRNA and protein levels compared to normal MCF-7 cells.  STF-083010, a novel inhibitor which specifically blocks the XBP1 splicing, reestablishes tamoxifen sensitivity to resistant MCF-7 cells.  Moreover, co-treatment with STF-083010 and tamoxifen can significantly delay breast cancer progression in a xenograft mammary tumor model.  We next investigate the expression of XBP1s in over 170 breast cancer patients' samples and the results demonstrate that XBP1s expression level is highly correlated with overall survival in the ER+ subgroup, but not in the ER- subgroup, suggesting a potential therapeutic application of XBP1 inhibitors in ER+breast cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRKRBjVR00jFhfKb6ELK8t0fW6udTcc2ebMGoLSQHfW8Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zmvFSrsQ%253D%253D&md5=c7d700c77fbea029b419a0e702aeb48e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.5827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.5827%26sid%3Dliteratum%253Aachs%26aulast%3DMing%26aufirst%3DJ.%26aulast%3DRuan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DN.%26aulast%3DMeng%26aufirst%3DQ.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DT.%26atitle%3DA%2520novel%2520chemical%252C%2520STF-083010%252C%2520reverses%2520tamoxifen-related%2520drug%2520resistance%2520in%2520breast%2520cancer%2520by%2520inhibiting%2520IRE1%252FXBP1%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D40692%26epage%3D40703%26doi%3D10.18632%2Foncotarget.5827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papandreou, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denko, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Melckebeke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lust, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koong, A. C.</span></span> <span> </span><span class="NLM_article-title">Identification of an Ire1alpha endonuclease specific inhibitor with cytotoxic activity against human multiple myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1314</span>, <span class="refDoi"> DOI: 10.1182/blood-2010-08-303099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1182%2Fblood-2010-08-303099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21081713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFGks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2011&pages=1311-1314&author=I.+Papandreouauthor=N.+C.+Denkoauthor=M.+Olsonauthor=H.+Van+Melckebekeauthor=S.+Lustauthor=A.+Tamauthor=D.+E.+Solow-Corderoauthor=D.+M.+Bouleyauthor=F.+Offnerauthor=M.+Niwaauthor=A.+C.+Koong&title=Identification+of+an+Ire1alpha+endonuclease+specific+inhibitor+with+cytotoxic+activity+against+human+multiple+myeloma&doi=10.1182%2Fblood-2010-08-303099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an Ire1alpha endonuclease specific inhibitor with cytotoxic activity against human multiple myeloma</span></div><div class="casAuthors">Papandreou, Ioanna; Denko, Nicholas C.; Olson, Michael; Van Melckebeke, Heleen; Lust, Sofie; Tam, Arvin; Solow-Cordero, David E.; Bouley, Donna M.; Offner, Fritz; Niwa, Maho; Koong, Albert C.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1311-1314</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Activation of the adaptive Ire1-XBP1 pathway has been identified in many solid tumors and hematol. malignancies, including multiple myeloma (MM).  Here, we report the identification of STF-083010, a novel small-mol. inhibitor of Ire1.  STF-083010 inhibited Ire1 endonuclease activity, without affecting its kinase activity, after endoplasmic reticulum stress both in vitro and in vivo.  Treatment with STF-083010 showed significant antimyeloma activity in model human MM xenografts.  Similarly, STF-083010 was preferentially toxic to freshly isolated human CD138+ MM cells compared with other similarly isolated cell populations.  The identification of this novel Ire1 inhibitor supports the hypothesis that the Ire1-XBP1 axis is a promising target for anticancer therapy, esp. in the context of MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozlJFXgajWgLVg90H21EOLACvtfcHk0ljDmpg9ywSn0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFGks7c%253D&md5=a237ec467f838e4185df327ec9dfb969</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2010-08-303099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2010-08-303099%26sid%3Dliteratum%253Aachs%26aulast%3DPapandreou%26aufirst%3DI.%26aulast%3DDenko%26aufirst%3DN.%2BC.%26aulast%3DOlson%26aufirst%3DM.%26aulast%3DVan%2BMelckebeke%26aufirst%3DH.%26aulast%3DLust%26aufirst%3DS.%26aulast%3DTam%26aufirst%3DA.%26aulast%3DSolow-Cordero%26aufirst%3DD.%2BE.%26aulast%3DBouley%26aufirst%3DD.%2BM.%26aulast%3DOffner%26aufirst%3DF.%26aulast%3DNiwa%26aufirst%3DM.%26aulast%3DKoong%26aufirst%3DA.%2BC.%26atitle%3DIdentification%2520of%2520an%2520Ire1alpha%2520endonuclease%2520specific%2520inhibitor%2520with%2520cytotoxic%2520activity%2520against%2520human%2520multiple%2520myeloma%26jtitle%3DBlood%26date%3D2011%26volume%3D117%26spage%3D1311%26epage%3D1314%26doi%3D10.1182%2Fblood-2010-08-303099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrolecki, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talukdar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartig, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaipparettu, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putluri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Pharmacological targeting of MYC-regulated IRE1/XBP1 pathway suppresses MYC-driven breast cancer</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1283</span>– <span class="NLM_lpage">1299</span>, <span class="refDoi"> DOI: 10.1172/JCI95873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1172%2FJCI95873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=29480818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC1MrltlCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2018&pages=1283-1299&author=N.+Zhaoauthor=J.+Caoauthor=L.+Xuauthor=Q.+Tangauthor=L.+E.+Dobroleckiauthor=X.+Lvauthor=M.+Talukdarauthor=Y.+Luauthor=X.+Wangauthor=D.+Z.+Huauthor=Q.+Shiauthor=Y.+Xiangauthor=Y.+Wangauthor=X.+Liuauthor=W.+Buauthor=Y.+Jiangauthor=M.+Liauthor=Y.+Gongauthor=Z.+Sunauthor=H.+Yingauthor=B.+Yuanauthor=X.+Linauthor=X.+H.+Fengauthor=S.+M.+Hartigauthor=F.+Liauthor=H.+Shenauthor=Y.+Chenauthor=L.+Hanauthor=Q.+Zengauthor=J.+B.+Pattersonauthor=B.+A.+Kaipparettuauthor=N.+Putluriauthor=F.+Sicheriauthor=J.+M.+Rosenauthor=M.+T.+Lewisauthor=X.+Chen&title=Pharmacological+targeting+of+MYC-regulated+IRE1%2FXBP1+pathway+suppresses+MYC-driven+breast+cancer&doi=10.1172%2FJCI95873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of MYC-regulated IRE1/XBP1 pathway suppresses MYC-driven breast cancer</span></div><div class="casAuthors">Zhao Na; Cao Jin; Xu Longyong; Dobrolecki Lacey E; Lv Xiangdong; Lu Yang; Wang Xiaoran; Shi Qing; Liu Xia; Bu Wen; Gong Yingyun; Sun Zheng; Feng Xin-Hua; Hartig Sean M; Li Feng; Putluri Nagireddy; Rosen Jeffrey M; Lewis Michael T; Chen Xi; Zhao Na; Cao Jin; Xu Longyong; Dobrolecki Lacey E; Lv Xiangdong; Wang Xiaoran; Shi Qing; Liu Xia; Bu Wen; Lewis Michael T; Chen Xi; Zhao Na; Cao Jin; Xu Longyong; Dobrolecki Lacey E; Lv Xiangdong; Lu Yang; Wang Xiaoran; Shi Qing; Liu Xia; Kaipparettu Benny Abraham; Rosen Jeffrey M; Lewis Michael T; Chen Xi; Tang Qianzi; Li Mingzhou; Talukdar Manisha; Sicheri Frank; Talukdar Manisha; Sicheri Frank; Hu Dorothy Z; Xiang Yu; Han Leng; Wang Yunfei; Chen Yiwen; Jiang Yi; Gong Yingyun; Sun Zheng; Ying Haoqiang; Yuan Bo; Feng Xin-Hua; Lin Xia; Feng Xin-Hua; Shen Haifa; Zeng Qingping; Patterson John B; Sicheri Frank</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1283-1299</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The unfolded protein response (UPR) is a cellular homeostatic mechanism that is activated in many human cancers and plays pivotal roles in tumor progression and therapy resistance.  However, the molecular mechanisms for UPR activation and regulation in cancer cells remain elusive.  Here, we show that oncogenic MYC regulates the inositol-requiring enzyme 1 (IRE1)/X-box binding protein 1 (XBP1) branch of the UPR in breast cancer via multiple mechanisms.  We found that MYC directly controls IRE1 transcription by binding to its promoter and enhancer.  Furthermore, MYC forms a transcriptional complex with XBP1, a target of IRE1, and enhances its transcriptional activity.  Importantly, we demonstrate that XBP1 is a synthetic lethal partner of MYC.  Silencing of XBP1 selectively blocked the growth of MYC-hyperactivated cells.  Pharmacological inhibition of IRE1 RNase activity with small molecule inhibitor 8866 selectively restrained the MYC-overexpressing tumor growth in vivo in a cohort of preclinical patient-derived xenograft models and genetically engineered mouse models.  Strikingly, 8866 substantially enhanced the efficacy of docetaxel chemotherapy, resulting in rapid regression of MYC-overexpressing tumors.  Collectively, these data establish the synthetic lethal interaction of the IRE1/XBP1 pathway with MYC hyperactivation and provide a potential therapy for MYC-driven human breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTs0p5pZIKXSjiVMCqkKxVfW6udTcc2eZ09YrMBeCh87ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrltlCkug%253D%253D&md5=a401b8d714c025a1ca3e6b7eecf994fc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1172%2FJCI95873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI95873%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DN.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DQ.%26aulast%3DDobrolecki%26aufirst%3DL.%2BE.%26aulast%3DLv%26aufirst%3DX.%26aulast%3DTalukdar%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DD.%2BZ.%26aulast%3DShi%26aufirst%3DQ.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DBu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DX.%2BH.%26aulast%3DHartig%26aufirst%3DS.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DPatterson%26aufirst%3DJ.%2BB.%26aulast%3DKaipparettu%26aufirst%3DB.%2BA.%26aulast%3DPutluri%26aufirst%3DN.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DRosen%26aufirst%3DJ.%2BM.%26aulast%3DLewis%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DPharmacological%2520targeting%2520of%2520MYC-regulated%2520IRE1%252FXBP1%2520pathway%2520suppresses%2520MYC-driven%2520breast%2520cancer%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2018%26volume%3D128%26spage%3D1283%26epage%3D1299%26doi%3D10.1172%2FJCI95873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hari, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhhatarai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backes, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schürer, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span> <span> </span><span class="NLM_article-title">Divergent allosteric control of the IRE1α endoribonuclease using kinase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">982</span>– <span class="NLM_lpage">989</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnchembio.1094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=23086298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=982-989&author=L.+Wangauthor=B.+G.+Pereraauthor=S.+B.+Hariauthor=B.+Bhhataraiauthor=B.+J.+Backesauthor=M.+A.+Seeligerauthor=S.+C.+Sch%C3%BCrerauthor=S.+A.+Oakesauthor=F.+R.+Papaauthor=D.+J.+Maly&title=Divergent+allosteric+control+of+the+IRE1%CE%B1+endoribonuclease+using+kinase+inhibitors&doi=10.1038%2Fnchembio.1094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent allosteric control of the IRE1α endoribonuclease using kinase inhibitors</span></div><div class="casAuthors">Wang, Likun; Perera, B. Gayani K.; Hari, Sanjay B.; Bhhatarai, Barun; Backes, Bradley J.; Seeliger, Markus A.; Schuerer, Stephan C.; Oakes, Scott A.; Papa, Feroz R.; Maly, Dustin J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">982-989</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Under endoplasmic reticulum stress, unfolded protein accumulation leads to activation of the endoplasmic reticulum transmembrane kinase/endoRNase IRE1α.  IRE1α oligomerizes, autophosphorylates and initiates splicing of XBP1 mRNA, thus triggering the unfolded protein response (UPR).  Here we show that IRE1α's kinase-controlled RNase can be regulated in two distinct modes with kinase inhibitors: one class of ligands occupies IRE1α's kinase ATP-binding site to activate RNase-mediated XBP1 mRNA splicing even without upstream endoplasmic reticulum stress, whereas a second class can inhibit the RNase through the same ATP-binding site, even under endoplasmic reticulum stress.  Thus, alternative kinase conformations stabilized by distinct classes of ATP-competitive inhibitors can cause allosteric switching of IRE1α's RNase-either on or off.  As dysregulation of the UPR has been implicated in a variety of cell degenerative and neoplastic disorders, small-mol. control over IRE1α should advance efforts to understand the UPR's role in pathophysiol. and to develop drugs for endoplasmic reticulum stress-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5edxS7gfwNrVg90H21EOLACvtfcHk0lggTCsxC6GE7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLbO&md5=a6daf1ddb2ccdec77035e2e3c70d674c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1094%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPerera%26aufirst%3DB.%2BG.%26aulast%3DHari%26aufirst%3DS.%2BB.%26aulast%3DBhhatarai%26aufirst%3DB.%26aulast%3DBackes%26aufirst%3DB.%2BJ.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26aulast%3DOakes%26aufirst%3DS.%2BA.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26atitle%3DDivergent%2520allosteric%2520control%2520of%2520the%2520IRE1%25CE%25B1%2520endoribonuclease%2520using%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D982%26epage%3D989%26doi%3D10.1038%2Fnchembio.1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallwood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhry, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuster, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerding, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeYoung, M. P.</span></span> <span> </span><span class="NLM_article-title">Long-range inhibitor-induced conformational regulation of human IRE1α endoribonuclease activity</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">1011</span>– <span class="NLM_lpage">1023</span>, <span class="refDoi"> DOI: 10.1124/mol.115.100917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1124%2Fmol.115.100917" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=1011-1023&author=N.+O.+Conchaauthor=A.+Smallwoodauthor=W.+Bonnetteauthor=R.+Totoritisauthor=G.+Zhangauthor=K.+Federowiczauthor=J.+Yangauthor=H.+Qiauthor=S.+Chenauthor=N.+Campobassoauthor=A.+E.+Choudhryauthor=L.+E.+Shusterauthor=K.+A.+Evansauthor=J.+Ralphauthor=S.+Sweitzerauthor=D.+A.+Heerdingauthor=C.+A.+Buserauthor=D.+S.+Suauthor=M.+P.+DeYoung&title=Long-range+inhibitor-induced+conformational+regulation+of+human+IRE1%CE%B1+endoribonuclease+activity&doi=10.1124%2Fmol.115.100917"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fmol.115.100917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.115.100917%26sid%3Dliteratum%253Aachs%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DSmallwood%26aufirst%3DA.%26aulast%3DBonnette%26aufirst%3DW.%26aulast%3DTotoritis%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DFederowicz%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DChoudhry%26aufirst%3DA.%2BE.%26aulast%3DShuster%26aufirst%3DL.%2BE.%26aulast%3DEvans%26aufirst%3DK.%2BA.%26aulast%3DRalph%26aufirst%3DJ.%26aulast%3DSweitzer%26aufirst%3DS.%26aulast%3DHeerding%26aufirst%3DD.%2BA.%26aulast%3DBuser%26aufirst%3DC.%2BA.%26aulast%3DSu%26aufirst%3DD.%2BS.%26aulast%3DDeYoung%26aufirst%3DM.%2BP.%26atitle%3DLong-range%2520inhibitor-induced%2520conformational%2520regulation%2520of%2520human%2520IRE1%25CE%25B1%2520endoribonuclease%2520activity%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D88%26spage%3D1011%26epage%3D1023%26doi%3D10.1124%2Fmol.115.100917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malwitz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasker, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellamaggiore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeckel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Materna-Reichelt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span> <span> </span><span class="NLM_article-title">Unfolded protein response in cancer: IRE1α inhibition by selective kinase ligands does not impair tumor cell viability</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1021/ml500315b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500315b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wksr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=68-72&author=P.+E.+Harringtonauthor=K.+Biswasauthor=D.+Malwitzauthor=A.+S.+Taskerauthor=C.+Mohrauthor=K.+L.+Andrewsauthor=K.+Dellamaggioreauthor=R.+Kendallauthor=H.+Beckmannauthor=P.+Jaeckelauthor=S.+Materna-Reicheltauthor=J.+R.+Allenauthor=J.+R.+Lipford&title=Unfolded+protein+response+in+cancer%3A+IRE1%CE%B1+inhibition+by+selective+kinase+ligands+does+not+impair+tumor+cell+viability&doi=10.1021%2Fml500315b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Unfolded Protein Response in Cancer: IRE1α Inhibition by Selective Kinase Ligands Does Not Impair Tumor Cell Viability</span></div><div class="casAuthors">Harrington, Paul E.; Biswas, Kaustav; Malwitz, David; Tasker, Andrew S.; Mohr, Christopher; Andrews, Kristin L.; Dellamaggiore, Ken; Kendall, Richard; Beckmann, Holger; Jaeckel, Peter; Materna-Reichelt, Silvia; Allen, Jennifer R.; Lipford, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-72</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The kinase/endonuclease inositol requiring enzyme 1 (IRE1α), one of the sensors of unfolded protein accumulation in the endoplasmic reticulum that triggers the unfolded protein response (UPR), has been investigated as an anticancer target.  We identified potent allosteric inhibitors of IRE1α endonuclease activity that bound to the kinase site on the enzyme.  Structure-activity relationship (SAR) studies led to 16 and 18, which were selective in kinase screens and were potent against recombinant IRE1α endonuclease as well as cellular IRE1α.  The first X-ray crystal structure of a kinase inhibitor (16) bound to hIRE1α was obtained.  Screening of native tumor cell lines (>300) against selective IRE1α inhibitors failed to demonstrate any effect on cellular viability.  These results suggest that IRE1α activity is not essential for viability in most tumor cell lines, in vitro, and that interfering with the survival functions of the UPR may not be an effective strategy to block tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1dx6SPyC8t7Vg90H21EOLACvtfcHk0lggTCsxC6GE7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wksr7J&md5=58812b61336303299493b8a3e204fa66</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml500315b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500315b%26sid%3Dliteratum%253Aachs%26aulast%3DHarrington%26aufirst%3DP.%2BE.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DMalwitz%26aufirst%3DD.%26aulast%3DTasker%26aufirst%3DA.%2BS.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DDellamaggiore%26aufirst%3DK.%26aulast%3DKendall%26aufirst%3DR.%26aulast%3DBeckmann%26aufirst%3DH.%26aulast%3DJaeckel%26aufirst%3DP.%26aulast%3DMaterna-Reichelt%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DJ.%2BR.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26atitle%3DUnfolded%2520protein%2520response%2520in%2520cancer%253A%2520IRE1%25CE%25B1%2520inhibition%2520by%2520selective%2520kinase%2520ligands%2520does%2520not%2520impair%2520tumor%2520cell%2520viability%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D68%26epage%3D72%26doi%3D10.1021%2Fml500315b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meza-Acevedo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobillot, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gliedt, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hari, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backes, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span> <span> </span><span class="NLM_article-title">Structural and functional analysis of the allosteric inhibition of IRE1α with ATP-competitive ligands</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2195</span>– <span class="NLM_lpage">2205</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFyksbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2195-2205&author=H.+C.+Feldmanauthor=M.+Tongauthor=L.+Wangauthor=R.+Meza-Acevedoauthor=T.+A.+Gobillotauthor=I.+Lebedevauthor=M.+J.+Gliedtauthor=S.+B.+Hariauthor=A.+K.+Mitraauthor=B.+J.+Backesauthor=F.+R.+Papaauthor=M.+A.+Seeligerauthor=D.+J.+Maly&title=Structural+and+functional+analysis+of+the+allosteric+inhibition+of+IRE1%CE%B1+with+ATP-competitive+ligands&doi=10.1021%2Facschembio.5b00940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional analysis of the allosteric inhibition of IRE1α with ATP-competitive ligands</span></div><div class="casAuthors">Feldman, Hannah C.; Tong, Michael; Wang, Likun; Meza-Acevedo, Rosa; Gobillot, Theodore A.; Lebedev, Ivan; Gliedt, Micah J.; Hari, Sanjay B.; Mitra, Arinjay K.; Backes, Bradley J.; Papa, Feroz R.; Seeliger, Markus A.; Maly, Dustin J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2195-2205</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accumulation of unfolded proteins under endoplasmic reticulum (ER) stress leads to the activation of the multidomain protein sensor, inositol-requiring enzyme 1α (IRE1α) as part of the unfolded protein response (UPR).  The clustering of IRE1α lumenal domains in the presence of unfolded proteins promotes kinase trans-autophosphorylation in the cytosol and subsequent RNase domain activation.  Interestingly, there is an allosteric relation between the kinase and RNase domains of IRE1α, which allows ATP-competitive inhibitors to modulate the activity of the RNase domain.  Here, the authors use kinase inhibitors to study how ATP-binding site conformation affects the activity of the RNase domain of IRE1α.  The authors found that diverse ATP-competitive inhibitors of IRE1α promoted dimerization and activation of RNase activity despite blocking kinase autophosphorylation.  In contrast, a subset of ATP-competitive ligands, which the authors call kinase-inhibiting RNase attenuators (KIRAs), allosterically inactivate the RNase domain through the kinase domain by stabilizing monomeric IRE1α.  Further insight into how ATP-competitive inhibitors are able to divergently modulate the RNase domain through the kinase domain was gained by obtaining the 1st structure of apo human IRE1α in the RNase active back-to-back dimer conformation.  Comparison of this structure with other existing structures of IRE1α and integration of the extensive structure-activity relation (SAR) data led the authors to formulate a model to rationalize how ATP-binding site ligands are able to control the IRE1α oligomeric state and subsequent RNase domain activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCwB7RD3BZDbVg90H21EOLACvtfcHk0liLm2Ah-8uI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFyksbo%253D&md5=c97eb1c65a217ce17093089d8f9fd6ee</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00940%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DH.%2BC.%26aulast%3DTong%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeza-Acevedo%26aufirst%3DR.%26aulast%3DGobillot%26aufirst%3DT.%2BA.%26aulast%3DLebedev%26aufirst%3DI.%26aulast%3DGliedt%26aufirst%3DM.%2BJ.%26aulast%3DHari%26aufirst%3DS.%2BB.%26aulast%3DMitra%26aufirst%3DA.%2BK.%26aulast%3DBackes%26aufirst%3DB.%2BJ.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26atitle%3DStructural%2520and%2520functional%2520analysis%2520of%2520the%2520allosteric%2520inhibition%2520of%2520IRE1%25CE%25B1%2520with%2520ATP-competitive%2520ligands%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2195%26epage%3D2205%26doi%3D10.1021%2Facschembio.5b00940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann-Petersen, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehdizadeh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colon-Negron, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meza-Acevedo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backes, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluestone, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span> <span> </span><span class="NLM_article-title">Targeting ABL-IRE1α signaling spares ER-stressed pancreatic β cells to reverse autoimmune diabetes</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2017.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.cmet.2017.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28380378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1ylsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=883-897&author=S.+Moritaauthor=S.+A.+Villaltaauthor=H.+C.+Feldmanauthor=A.+C.+Registerauthor=W.+Rosenthalauthor=I.+T.+Hoffmann-Petersenauthor=M.+Mehdizadehauthor=R.+Ghoshauthor=L.+Wangauthor=K.+Colon-Negronauthor=R.+Meza-Acevedoauthor=B.+J.+Backesauthor=D.+J.+Malyauthor=J.+A.+Bluestoneauthor=F.+R.+Papa&title=Targeting+ABL-IRE1%CE%B1+signaling+spares+ER-stressed+pancreatic+%CE%B2+cells+to+reverse+autoimmune+diabetes&doi=10.1016%2Fj.cmet.2017.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting ABL-IRE1α Signaling Spares ER-Stressed Pancreatic β Cells to Reverse Autoimmune Diabetes</span></div><div class="casAuthors">Morita, Shuhei; Villalta, S. Armando; Feldman, Hannah C.; Register, Ames C.; Rosenthal, Wendy; Hoffmann-Petersen, Ingeborg T.; Mehdizadeh, Morvarid; Ghosh, Rajarshi; Wang, Likun; Colon-Negron, Kevin; Meza-Acevedo, Rosa; Backes, Bradley J.; Maly, Dustin J.; Bluestone, Jeffrey A.; Papa, Feroz R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">883-897.e8</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In cells experiencing unrelieved endoplasmic reticulum (ER) stress, the ER transmembrane kinase/endoribonuclease (RNase)-IRE1α-endonucleolytically degrades ER-localized mRNAs to promote apoptosis.  Here we find that the ABL family of tyrosine kinases rheostatically enhances IRE1α's enzymic activities, thereby potentiating ER stress-induced apoptosis.  During ER stress, cytosolic ABL kinases localize to the ER membrane, where they bind, scaffold, and hyperactivate IRE1α's RNase.  Imatinib-an anti-cancer tyrosine kinase inhibitor-antagonizes the ABL-IRE1α interaction, blunts IRE1α RNase hyperactivity, reduces pancreatic β cell apoptosis, and reverses type 1 diabetes (T1D) in the non-obese diabetic (NOD) mouse model.  A mono-selective kinase inhibitor that allosterically attenuates IRE1α's RNase-KIRA8-also efficaciously reverses established diabetes in NOD mice by sparing β cells and preserving their physiol. function.  Our data support a model wherein ER-stressed β cells contribute to their own demise during T1D pathogenesis and implicate the ABL-IRE1α axis as a drug target for the treatment of an autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGZtS35KHrIrVg90H21EOLACvtfcHk0liLm2Ah-8uI5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1ylsbg%253D&md5=be7692b3c222ac9eb9b1d1ca68a77e47</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2017.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2017.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DMorita%26aufirst%3DS.%26aulast%3DVillalta%26aufirst%3DS.%2BA.%26aulast%3DFeldman%26aufirst%3DH.%2BC.%26aulast%3DRegister%26aufirst%3DA.%2BC.%26aulast%3DRosenthal%26aufirst%3DW.%26aulast%3DHoffmann-Petersen%26aufirst%3DI.%2BT.%26aulast%3DMehdizadeh%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DColon-Negron%26aufirst%3DK.%26aulast%3DMeza-Acevedo%26aufirst%3DR.%26aulast%3DBackes%26aufirst%3DB.%2BJ.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26aulast%3DBluestone%26aufirst%3DJ.%2BA.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26atitle%3DTargeting%2520ABL-IRE1%25CE%25B1%2520signaling%2520spares%2520ER-stressed%2520pancreatic%2520%25CE%25B2%2520cells%2520to%2520reverse%2520autoimmune%2520diabetes%26jtitle%3DCell%2520Metab.%26date%3D2017%26volume%3D25%26spage%3D883%26epage%3D897%26doi%3D10.1016%2Fj.cmet.2017.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Dodd, D.
S.</span>; <span class="NLM_string-name">Huynh, T.
N.</span>; <span class="NLM_string-name">Lin, J.</span>; <span class="NLM_string-name">Liu, C.</span>; <span class="NLM_string-name">Mussari, C. P.</span>; <span class="NLM_string-name">Tokarski, J. S.</span>; <span class="NLM_string-name">Tortolani, D. R.</span>; <span class="NLM_string-name">Wrobleski, S. T.</span>; <span class="NLM_string-name">Lin, S.</span></span> <span> </span><span class="NLM_article-title">Preparation of Fused Heterocyclic Compounds Useful as Kinase Modulators</span>. <span class="NLM_patent">US20080045536</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=W.+Vaccaro&author=Z.+Chen&author=D.%0AS.+Dodd&author=T.%0AN.+Huynh&author=J.+Lin&author=C.+Liu&author=C.+P.+Mussari&author=J.+S.+Tokarski&author=D.+R.+Tortolani&author=S.+T.+Wrobleski&author=S.+Lin&title=Preparation+of+Fused+Heterocyclic+Compounds+Useful+as+Kinase+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVaccaro%26aufirst%3DW.%26atitle%3DPreparation%2520of%2520Fused%2520Heterocyclic%2520Compounds%2520Useful%2520as%2520Kinase%2520Modulators%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. F.</span>; <span class="NLM_string-name">Ba-Maung, N. Y.</span>; <span class="NLM_string-name">Erickson, S. A.</span>; <span class="NLM_string-name">Fidanze, S. D.</span>; <span class="NLM_string-name">Mantei, R. A.</span>; <span class="NLM_string-name">Sheppard, G. S.</span>; <span class="NLM_string-name">Sorensen, B. K.</span>; <span class="NLM_string-name">Wang, G. T.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Bell, R.
L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrimidine Inhibitors of Kinase Activity</span>. International Application No. <span class="NLM_patent">WO2010138575</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=R.+F.+Clark&author=N.+Y.+Ba-Maung&author=S.+A.+Erickson&author=S.+D.+Fidanze&author=R.+A.+Mantei&author=G.+S.+Sheppard&author=B.+K.+Sorensen&author=G.+T.+Wang&author=J.+Wang&author=R.%0AL.+Bell&title=Preparation+of+Pyrimidine+Inhibitors+of+Kinase+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DR.%2BF.%26atitle%3DPreparation%2520of%2520Pyrimidine%2520Inhibitors%2520of%2520Kinase%2520Activity%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kusakabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ide, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashizume, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tagashira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanazawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishioka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dohi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kido, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of imidazo[1,2-b]pyridazine derivatives: selective and orally available Mps1 (TTK) kinase inhibitors exhibiting remarkable antiproliferative activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1760</span>– <span class="NLM_lpage">1775</span>, <span class="refDoi"> DOI: 10.1021/jm501599u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501599u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2it7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1760-1775&author=K.+Kusakabeauthor=N.+Ideauthor=Y.+Daigoauthor=T.+Itohauthor=T.+Yamamotoauthor=H.+Hashizumeauthor=K.+Nozuauthor=H.+Yoshidaauthor=G.+Tadanoauthor=S.+Tagashiraauthor=K.+Higashinoauthor=Y.+Okanoauthor=Y.+Satoauthor=M.+Inoueauthor=M.+Iguchiauthor=T.+Kanazawaauthor=Y.+Ishiokaauthor=K.+Dohiauthor=Y.+Kidoauthor=S.+Sakamotoauthor=S.+Andoauthor=M.+Maedaauthor=M.+Higakiauthor=Y.+Babaauthor=Y.+Nakamura&title=Discovery+of+imidazo%5B1%2C2-b%5Dpyridazine+derivatives%3A+selective+and+orally+available+Mps1+%28TTK%29+kinase+inhibitors+exhibiting+remarkable+antiproliferative+activity&doi=10.1021%2Fjm501599u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Imidazo[1,2-b]pyridazine Derivatives: Selective and Orally Available Mps1 (TTK) Kinase Inhibitors Exhibiting Remarkable Antiproliferative Activity</span></div><div class="casAuthors">Kusakabe, Ken-ichi; Ide, Nobuyuki; Daigo, Yataro; Itoh, Takeshi; Yamamoto, Takahiko; Hashizume, Hiroshi; Nozu, Kohei; Yoshida, Hiroshi; Tadano, Genta; Tagashira, Sachie; Higashino, Kenichi; Okano, Yousuke; Sato, Yuji; Inoue, Makiko; Iguchi, Motofumi; Kanazawa, Takayuki; Ishioka, Yukichi; Dohi, Keiji; Kido, Yasuto; Sakamoto, Shingo; Ando, Shigeru; Maeda, Masahiro; Higaki, Masayo; Baba, Yoshiyasu; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1760-1775</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monopolar spindle 1 (Mps1) is an attractive oncol. target due to its high expression level in cancer cells as well as the correlation of its expression levels with histol. grades of cancers.  An imidazo[1,2-a]pyrazine 10a was identified during an HTS campaign.  Although 10a exhibited good biochem. activity, its moderate cellular as well as antiproliferative activities needed to be improved.  The cocrystal structure of an analog of 10a guided our lead optimization to introduce substituents at the 6-position of the scaffold, giving the 6-aryl substituted 21b which had improved cellular activity but no oral bioavailability in rat.  Property-based optimization at the 6-position and a scaffold change led to the discovery of the imidazo[1,2-b]pyridazine-based 27f, an extremely potent (cellular Mps1 IC50 = 0.70 nM, A549 IC50 = 6.0 nM), selective Mps1 inhibitor over 192 kinases, which could be orally administered and was active in vivo.  This 27f demonstrated remarkable antiproliferative activity in the nanomolar range against various tissue cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8D9nRl5vWabVg90H21EOLACvtfcHk0lhjHaV6uQpOSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2it7g%253D&md5=bee62a50247a363dc3e35fcdeac6a64f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm501599u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501599u%26sid%3Dliteratum%253Aachs%26aulast%3DKusakabe%26aufirst%3DK.%26aulast%3DIde%26aufirst%3DN.%26aulast%3DDaigo%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DHashizume%26aufirst%3DH.%26aulast%3DNozu%26aufirst%3DK.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DTadano%26aufirst%3DG.%26aulast%3DTagashira%26aufirst%3DS.%26aulast%3DHigashino%26aufirst%3DK.%26aulast%3DOkano%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DM.%26aulast%3DIguchi%26aufirst%3DM.%26aulast%3DKanazawa%26aufirst%3DT.%26aulast%3DIshioka%26aufirst%3DY.%26aulast%3DDohi%26aufirst%3DK.%26aulast%3DKido%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DS.%26aulast%3DAndo%26aufirst%3DS.%26aulast%3DMaeda%26aufirst%3DM.%26aulast%3DHigaki%26aufirst%3DM.%26aulast%3DBaba%26aufirst%3DY.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520imidazo%255B1%252C2-b%255Dpyridazine%2520derivatives%253A%2520selective%2520and%2520orally%2520available%2520Mps1%2520%2528TTK%2529%2520kinase%2520inhibitors%2520exhibiting%2520remarkable%2520antiproliferative%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1760%26epage%3D1775%26doi%3D10.1021%2Fjm501599u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uitdehaag, J. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Man, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willemsen-Seegers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinsen, M. B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libouban, M. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterrenburg, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wit, J. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Vetter, J. R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Roos, J. A. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buijsman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, G. J. R.</span></span> <span> </span><span class="NLM_article-title">Target residence time-guided optimization on TTK kinase results in inhibitors with potent anti-proliferative activity</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2230</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.jmb.2017.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28539250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Wiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2211-2230&author=J.+C.+M.+Uitdehaagauthor=J.+de+Manauthor=N.+Willemsen-Seegersauthor=M.+B.+W.+Prinsenauthor=M.+A.+A.+Liboubanauthor=J.+G.+Sterrenburgauthor=J.+J.+P.+de+Witauthor=J.+R.+F.+de%0AVetterauthor=J.+A.+D.+M.+de+Roosauthor=R.+C.+Buijsmanauthor=G.+J.+R.+Zaman&title=Target+residence+time-guided+optimization+on+TTK+kinase+results+in+inhibitors+with+potent+anti-proliferative+activity&doi=10.1016%2Fj.jmb.2017.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Target Residence Time-Guided Optimization on TTK Kinase Results in Inhibitors with Potent Anti-Proliferative Activity</span></div><div class="casAuthors">Uitdehaag, Joost C. M.; de Man, Jos; Willemsen-Seegers, Nicole; Prinsen, Martine B. W.; Libouban, Marion A. A.; Sterrenburg, Jan Gerard; de Wit, Joeri J. P.; de Vetter, Judith R. F.; de Roos, Jeroen A. D. M.; Buijsman, Rogier C.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2211-2230</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The protein kinase threonine tyrosine kinase (TTK; also known as Mps1) is a crit. component of the spindle assembly checkpoint and a promising drug target for the treatment of aggressive cancers, such as triple neg. breast cancer.  While the first TTK inhibitors have entered clin. trials, little is known about how the inhibition of TTK with small-mol. compds. affects cellular activity.  The authors studied the selective TTK inhibitor NTRC 0066-0, which was developed in the authors' own lab., together with 11 TTK inhibitors developed by other companies, including Mps-BAY2b, BAY 1161909, BAY 1217389 (Bayer), TC-Mps1-12 (Shionogi), and MPI-0479605 (Myrexis).  Parallel testing shows that the cellular activity of these TTK inhibitors correlates with their binding affinity to TTK and, more strongly, with target residence time.  TTK inhibitors are therefore an example where target residence time dets. activity in in vitro cellular assays.  X-ray structures and thermal stability expts. reveal that the most potent compds. induce a shift of the glycine-rich loop as a result of binding to the catalytic lysine at position 553.  This "lysine trap" disrupts the catalytic machinery.  Based on these insights, the authors developed TTK inhibitors, based on a (5,6-dihydro)pyrimido[4,5-e]indolizine scaffold, with longer target residence times, which further exploit an allosteric pocket surrounding Lys 553.  Their binding mode is new for kinase inhibitors and can be classified as hybrid Type I/Type III.  These inhibitors have very potent anti-proliferative activity that rivals classic cytotoxic therapy.  The authors' findings will open up new avenues for more applications for TTK inhibitors in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT_6mZVMoVTrVg90H21EOLACvtfcHk0ljP6QatOl5nFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Wiuro%253D&md5=c02b149e673dec7b0212b37a4c7b0831</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%2BM.%26aulast%3Dde%2BMan%26aufirst%3DJ.%26aulast%3DWillemsen-Seegers%26aufirst%3DN.%26aulast%3DPrinsen%26aufirst%3DM.%2BB.%2BW.%26aulast%3DLibouban%26aufirst%3DM.%2BA.%2BA.%26aulast%3DSterrenburg%26aufirst%3DJ.%2BG.%26aulast%3Dde%2BWit%26aufirst%3DJ.%2BJ.%2BP.%26aulast%3Dde%2BVetter%26aufirst%3DJ.%2BR.%2BF.%26aulast%3Dde%2BRoos%26aufirst%3DJ.%2BA.%2BD.%2BM.%26aulast%3DBuijsman%26aufirst%3DR.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%2BR.%26atitle%3DTarget%2520residence%2520time-guided%2520optimization%2520on%2520TTK%2520kinase%2520results%2520in%2520inhibitors%2520with%2520potent%2520anti-proliferative%2520activity%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2211%26epage%3D2230%26doi%3D10.1016%2Fj.jmb.2017.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itzhak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strover, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span> <span> </span><span class="NLM_article-title">Multiple autophosphorylations significantly enhance the endoribonuclease activity of human inositol requiring enzyme 1α</span>. <i>BMC Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1186/1471-2091-15-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1186%2F1471-2091-15-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=24524643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFaqtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3&author=D.+Itzhakauthor=M.+Brightauthor=P.+McAndrewauthor=A.+Mirzaauthor=Y.+Newbattauthor=J.+Stroverauthor=M.+Widyaauthor=A.+Thompsonauthor=G.+Morganauthor=I.+Collinsauthor=F.+Davies&title=Multiple+autophosphorylations+significantly+enhance+the+endoribonuclease+activity+of+human+inositol+requiring+enzyme+1%CE%B1&doi=10.1186%2F1471-2091-15-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple autophosphorylations significantly enhance the endoribonuclease activity of human inositol requiring enzyme 1α</span></div><div class="casAuthors">Itzhak, Daniel; Bright, Michael; McAndrew, Peter; Mirza, Amin; Newbatt, Yvette; Strover, Jade; Widya, Marcella; Thompson, Andrew; Morgan, Gareth; Collins, Ian; Davies, Faith</div><div class="citationInfo"><span class="NLM_cas:title">BMC Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3/1-3/7</span>CODEN:
                <span class="NLM_cas:coden">BBMIB3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Endoplasmic reticulum stress, caused by the presence of misfolded proteins, activates the stress sensor inositol-requiring enzyme 1α (IRE1α).  The resulting increase in IRE1α RNase activity causes sequence-specific cleavage of X-box binding protein 1 (XBP1) mRNA, resulting in upregulation of the unfolded protein response and cellular adaptation to stress.  The precise mechanism of human IRE1α activation is currently unclear.  The role of IRE1α kinase activity is disputed, as results from the generation of various kinase-inactivating mutations in either yeast or human cells are discordant.  Kinase activity can also be made redundant by small mols. which bind the ATP binding site.  We set out to uncover a role for IRE1α kinase activity using wild-type cytosolic protein constructs.  Results: We show that concn.-dependent oligomerization is sufficient to cause IRE1α cytosolic domain RNase activity in vitro.  We demonstrate a role for the kinase activity by showing that autophosphorylation enhances RNase activity.  Inclusion of the IRE1α linker domain in protein constructs allows hyperphosphorylation and further enhancement of RNase activity, highlighting the importance of kinase activity.  We show that IRE1α phosphorylation status correlates with an increased propensity to form oligomeric complexes and that forced dimerization causes great enhancement in RNase activity.  In addn. we demonstrate that even when IRE1α is forced to dimerize, by a GST-tag, phospho-enhancement of activity is still obsd.  Conclusions: Taken together these expts. support the hypothesis that phosphorylation is important in modulating IRE1α RNase activity which is achieved by increasing the propensity of IRE1α to dimerize.  This work supports the development of IRE1α kinase inhibitors for use in the treatment of secretory cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohh5JaMayEnbVg90H21EOLACvtfcHk0ljP6QatOl5nFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFaqtL0%253D&md5=c773198e9f1deb86fccb2076fd2603b7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1186%2F1471-2091-15-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2091-15-3%26sid%3Dliteratum%253Aachs%26aulast%3DItzhak%26aufirst%3DD.%26aulast%3DBright%26aufirst%3DM.%26aulast%3DMcAndrew%26aufirst%3DP.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DStrover%26aufirst%3DJ.%26aulast%3DWidya%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DDavies%26aufirst%3DF.%26atitle%3DMultiple%2520autophosphorylations%2520significantly%2520enhance%2520the%2520endoribonuclease%2520activity%2520of%2520human%2520inositol%2520requiring%2520enzyme%25201%25CE%25B1%26jtitle%3DBMC%2520Biochem.%26date%3D2014%26volume%3D15%26spage%3D3%26doi%3D10.1186%2F1471-2091-15-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strover, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L.</span></span> <span> </span><span class="NLM_article-title">Identification of autophosphorylation inhibitors of the inositol-requiring enzyme 1 alpha (IRE1α) by high-throughput screening using a DELFIA assay</span>. <i>J. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1177/1087057112465647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1177%2F1087057112465647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=23139381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OktrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=298-308&author=Y.+Newbattauthor=A.+Hardcastleauthor=P.+C.+McAndrewauthor=J.+A.+Stroverauthor=A.+Mirzaauthor=G.+J.+Morganauthor=R.+Burkeauthor=F.+E.+Daviesauthor=I.+Collinsauthor=R.+L.+van+Montfort&title=Identification+of+autophosphorylation+inhibitors+of+the+inositol-requiring+enzyme+1+alpha+%28IRE1%CE%B1%29+by+high-throughput+screening+using+a+DELFIA+assay&doi=10.1177%2F1087057112465647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of autophosphorylation inhibitors of the inositol-requiring enzyme 1 alpha (IRE1α) by high-throughput screening using a DELFIA assay</span></div><div class="casAuthors">Newbatt, Yvette; Hardcastle, Anthea; McAndrew, P. Craig; Strover, Jade A.; Mirza, Amin; Morgan, Gareth J.; Burke, Rosemary; Davies, Faith E.; Collins, Ian; van Montfort, Rob L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-308, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Inositol-requiring enzyme 1 alpha (IRE1α) is a transmembrane sensor protein with both kinase and RNase activity, which plays a crucial role in the unfolded protein response (UPR).  Protein misfolding in the endoplasmic reticulum (ER) lumen triggers dimerization and subsequent trans-autophosphorylation of IRE1α.  This leads to the activation of its endoribonuclease (RNase) domain and splicing of the mRNA of the transcriptional activator XBP1, ultimately generating an active XBP1 (XBP1s) implicated in multiple myeloma survival.  Previously, we have identified human IRE1α as a target for the development of kinase inhibitors that could modulate the UPR in human cells, which has particular relevance for multiple myeloma and other secretory malignancies.  Here we describe the development and validation of a 384-well high-throughput screening assay using DELFIA technol. that is specific for IRE1α autophosphorylation.  Using this format, a focused library of 2312 potential kinase inhibitors was screened, and several novel IRE1α kinase inhibitor scaffolds were identified that could potentially be developed toward new therapies to treat multiple myeloma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsN6-SmyMtXbVg90H21EOLACvtfcHk0ljCnVWekxA9jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OktrfM&md5=97f3f0b8ef6c1ff9b35af0f4165486bc</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1177%2F1087057112465647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112465647%26sid%3Dliteratum%253Aachs%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DHardcastle%26aufirst%3DA.%26aulast%3DMcAndrew%26aufirst%3DP.%2BC.%26aulast%3DStrover%26aufirst%3DJ.%2BA.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DG.%2BJ.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DDavies%26aufirst%3DF.%2BE.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%26atitle%3DIdentification%2520of%2520autophosphorylation%2520inhibitors%2520of%2520the%2520inositol-requiring%2520enzyme%25201%2520alpha%2520%2528IRE1%25CE%25B1%2529%2520by%2520high-throughput%2520screening%2520using%2520a%2520DELFIA%2520assay%26jtitle%3DJ.%2520Biomol.%2520Screen.%26date%3D2013%26volume%3D18%26spage%3D298%26epage%3D308%26doi%3D10.1177%2F1087057112465647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potashman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bready, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMelfi, T. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPietro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbaum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neervannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polverino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Plas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of orally active benzimidazoles and benzoxazoles as vascular endothelial growth factor-2 receptor tyrosine kinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">4351</span>– <span class="NLM_lpage">4373</span>, <span class="refDoi"> DOI: 10.1021/jm070034i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070034i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovF2mtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=4351-4373&author=M.+H.+Potashmanauthor=J.+Breadyauthor=A.+Coxonauthor=T.+M.+DeMelfiauthor=L.+DiPietroauthor=N.+Doerrauthor=D.+Elbaumauthor=J.+Estradaauthor=P.+Gallantauthor=J.+Germainauthor=Y.+Guauthor=J.+C.+Harmangeauthor=S.+A.+Kaufmanauthor=R.+Kendallauthor=J.+L.+Kimauthor=G.+N.+Kumarauthor=A.+M.+Longauthor=S.+Neervannanauthor=V.+F.+Patelauthor=A.+Polverinoauthor=P.+Roseauthor=S.+van+der%0APlasauthor=D.+Whittingtonauthor=R.+Zanonauthor=H.+Zhao&title=Design%2C+synthesis%2C+and+evaluation+of+orally+active+benzimidazoles+and+benzoxazoles+as+vascular+endothelial+growth+factor-2+receptor+tyrosine+kinase+inhibitors&doi=10.1021%2Fjm070034i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of Orally Active Benzimidazoles and Benzoxazoles as Vascular Endothelial Growth Factor-2 Receptor Tyrosine Kinase Inhibitors</span></div><div class="casAuthors">Potashman, Michele H.; Bready, James; Coxon, Angela; DeMelfi, Thomas M., Jr.; DiPietro, Lucian; Doerr, Nicholas; Elbaum, Daniel; Estrada, Juan; Gallant, Paul; Germain, Julie; Gu, Yan; Harmange, Jean-Christophe; Kaufman, Stephen A.; Kendall, Rick; Kim, Joseph L.; Kumar, Gondi N.; Long, Alexander M.; Neervannan, Seshadri; Patel, Vinod F.; Polverino, Anthony; Rose, Paul; Van der Plas, Simon; Whittington, Douglas; Zanon, Roger; Zhao, Huilin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4351-4373</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">inhibition of the VEGF signaling pathway has become a valuable approach in the treatment of cancers.  guided by X-ray crystallog. and mol. modeling, a series of 2-aminobenzimidazoles and 2-aminobenzoxazoles were identified as potent inhibitors of the vascular endothelial growth factor-2 receptor tyrosine kinase VEGFR-2 (also known as KDR, kinase insert domain receptor) in both enzymic and HUVEC cellular proliferation assays.  in this report we describe the synthesis and structure-activity relationship of a series of 2-aminobenzimidazoles and benzoxazoles, culminating in the identification of a nonracemic pyrrolidinylmethoxyphenylaminobenzoxazolyloxy pyridinecarboxamide as a potent and selective VEGFR-2 inhibitor displaying a good pharmacokinetic profile.  the nonracemic pyrrolidinylmethoxyphenylaminobenzoxazolyloxy pyridinecarboxamide also demonstrated efficacy in both the murine matrigel model for vascular permeability (79% inhibition obsd. at 100 mg/kg) and the rat corneal angiogenesis model (ED50 = 16.3 mg/kg).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqG7laVqJmUbVg90H21EOLACvtfcHk0ljCnVWekxA9jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovF2mtro%253D&md5=e4639afa8ba460a8679dcc75bc270d8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm070034i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070034i%26sid%3Dliteratum%253Aachs%26aulast%3DPotashman%26aufirst%3DM.%2BH.%26aulast%3DBready%26aufirst%3DJ.%26aulast%3DCoxon%26aufirst%3DA.%26aulast%3DDeMelfi%26aufirst%3DT.%2BM.%26aulast%3DDiPietro%26aufirst%3DL.%26aulast%3DDoerr%26aufirst%3DN.%26aulast%3DElbaum%26aufirst%3DD.%26aulast%3DEstrada%26aufirst%3DJ.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DGermain%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKaufman%26aufirst%3DS.%2BA.%26aulast%3DKendall%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DKumar%26aufirst%3DG.%2BN.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DNeervannan%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DV.%2BF.%26aulast%3DPolverino%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BPlas%26aufirst%3DS.%26aulast%3DWhittington%26aufirst%3DD.%26aulast%3DZanon%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%2520orally%2520active%2520benzimidazoles%2520and%2520benzoxazoles%2520as%2520vascular%2520endothelial%2520growth%2520factor-2%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D4351%26epage%3D4373%26doi%3D10.1021%2Fjm070034i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1038/nchembio799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations&doi=10.1038%2Fnchembio799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0liYRAq_5L9RIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iwawaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosoda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamigori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura-Furuwatari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, K.</span></span> <span> </span><span class="NLM_article-title">Translational control by the ER transmembrane kinase/ribonuclease IRE1 under ER stress</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">158</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1038/35055065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2F35055065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=11175748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1KmsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2001&pages=158-164&author=T.+Iwawakiauthor=A.+Hosodaauthor=T.+Okudaauthor=Y.+Kamigoriauthor=C.+Nomura-Furuwatariauthor=Y.+Kimataauthor=A.+Tsuruauthor=K.+Kohno&title=Translational+control+by+the+ER+transmembrane+kinase%2Fribonuclease+IRE1+under+ER+stress&doi=10.1038%2F35055065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Translational control by the ER transmembrane kinase/ribonuclease IRE1 under ER stress</span></div><div class="casAuthors">Iwawaki, Takao; Hosoda, Akira; Okuda, Tetsuo; Kamigori, Yusuke; Nomura-Furuwatari, Chizumi; Kimata, Yukio; Tsuru, Akio; Kohno, Kenji</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">158-164</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Under conditions of endoplasmic reticulum (ER) stress, mammalian cells induce both translational repression and the unfolded protein response that transcriptionally activates genes encoding ER-resident mol. chaperones.  To date, the only known pathway for translational repression in response to ER stress has been the phosphorylation of eIF-2α by the double-stranded RNA-activated protein kinase (PKR) or the transmembrane PKR-like ER kinase (PERK).  Here we report another pathway in which the ER transmembrane kinase/RNase IRE1β induces translational repression through 28S rRNA cleavage in response to ER stress.  The evidence suggests that both pathways are important for efficient translational repression during the ER stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn7Fzg6AubS7Vg90H21EOLACvtfcHk0liYRAq_5L9RIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1KmsLg%253D&md5=39608203eb79b14e442aa31b6789bf45</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F35055065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35055065%26sid%3Dliteratum%253Aachs%26aulast%3DIwawaki%26aufirst%3DT.%26aulast%3DHosoda%26aufirst%3DA.%26aulast%3DOkuda%26aufirst%3DT.%26aulast%3DKamigori%26aufirst%3DY.%26aulast%3DNomura-Furuwatari%26aufirst%3DC.%26aulast%3DKimata%26aufirst%3DY.%26aulast%3DTsuru%26aufirst%3DA.%26aulast%3DKohno%26aufirst%3DK.%26atitle%3DTranslational%2520control%2520by%2520the%2520ER%2520transmembrane%2520kinase%252Fribonuclease%2520IRE1%2520under%2520ER%2520stress%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2001%26volume%3D3%26spage%3D158%26epage%3D164%26doi%3D10.1038%2F35055065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosmopoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atrash, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of an Aurora-A mutant that mimics Aurora-B bound to MLN8054: insights into selectivity and drug design</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>427</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1042/BJ20091530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1042%2FBJ20091530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20067443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2010&pages=19-28&author=C.+A.+Dodsonauthor=M.+Kosmopoulouauthor=M.+W.+Richardsauthor=B.+Atrashauthor=V.+Bavetsiasauthor=J.+Blaggauthor=R.+Bayliss&title=Crystal+structure+of+an+Aurora-A+mutant+that+mimics+Aurora-B+bound+to+MLN8054%3A+insights+into+selectivity+and+drug+design&doi=10.1042%2FBJ20091530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35aR"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of an Aurora-A mutant that mimics Aurora-B bound to MLN8054: insights into selectivity and drug design</span></div><div class="casAuthors">Dodson, Charlotte A.; Kosmopoulou, Magda; Richards, Mark W.; Atrash, Butrus; Bavetsias, Vassilios; Blagg, Julian; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">427</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-28</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The prodn. of selective protein kinase inhibitors is often frustrated by the similarity of the enzyme active sites.  For this reason, it is challenging to design inhibitors that discriminate between the three Aurora kinases, which are important targets in cancer drug discovery.  We have used a triple-point mutant of Aurora-A (AurAx3) which mimics the active site of Aurora-B to investigate the structural basis of MLN8054 selectivity.  The bias toward Aurora-A inhibition by MLN8054 is fully recapitulated by AurAx3 in vitro.  X-ray crystal structures of the complex suggest that the basis for the discrimination is electrostatic repulsion due to the T217E substitution, which we have confirmed using a single-point mutant.  The activation loop of Aurora-A in the AurAx3-MLN8054 complex exhibits an unusual conformation in which Asp274 and Phe275 side chains point into the interior of the protein.  There is to our knowledge no documented precedent for this conformation, which we have termed DFG-up.  The sequence requirements of the DFG-up conformation suggest that it might be accessible to only a fraction of kinases.  MLN8054 thus circumvents the problem of highly homologous active sites.  Binding of MLN8054 to Aurora-A switches the character of a pocket within the active site from polar to a hydrophobic pocket, similar to what is obsd. in the structure of Aurora-A bound to a compd. that induces DFG-out.  We propose that targeting this pocket may be a productive route in the design of selective kinase inhibitors and describe the structural basis for the rational design of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdJs5KY2VkabVg90H21EOLACvtfcHk0lgKzJ7SrkzJCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D&md5=fdf1a176a5fdf5fcd71fd57c3ef92817</span></div><a href="/servlet/linkout?suffix=cit35a&amp;dbid=16384&amp;doi=10.1042%2FBJ20091530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091530%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DC.%2BA.%26aulast%3DKosmopoulou%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520an%2520Aurora-A%2520mutant%2520that%2520mimics%2520Aurora-B%2520bound%2520to%2520MLN8054%253A%2520insights%2520into%2520selectivity%2520and%2520drug%2520design%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26spage%3D19%26epage%3D28%26doi%3D10.1042%2FBJ20091530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="note"><p class="first last">Erratum in:</p></div><div class="NLM_citation" id="cit35a1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosmopoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atrash, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>427</i></span>,  <span class="NLM_fpage">551</span>, <span class="refDoi"> DOI: 10.1042/BJ4270551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35a1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1042%2FBJ4270551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dodson%2C+C.+A.%3B+Kosmopoulou%2C+M.%3B+Richards%2C+M.+W.%3B+Atrash%2C+B.%3B+Bavetsias%2C+V.%3B+Blagg%2C+J.%3B+Bayliss%2C+R.+Biochem.+J.+2010%2C+427%2C+551%2C+10.1042%2FBJ4270551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35a1&amp;dbid=16384&amp;doi=10.1042%2FBJ4270551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ4270551%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DC.%2BA.%26aulast%3DKosmopoulou%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBayliss%26aufirst%3DR.%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26spage%3D551%26doi%3D10.1042%2FBJ4270551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas-Martin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dotson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drummond, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawson, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">A class of 2,4-bisanilinopyrimidine Aurora A inhibitors with unusually high selectivity against Aurora B</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3300</span>– <span class="NLM_lpage">3307</span>, <span class="refDoi"> DOI: 10.1021/jm9000314</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9000314" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=3300-3307&author=I.+Aliagas-Martinauthor=D.+Burdickauthor=L.+Corsonauthor=J.+Dotsonauthor=J.+Drummondauthor=C.+Fieldsauthor=O.+W.+Huangauthor=T.+Hunsakerauthor=T.+Kleinheinzauthor=E.+Kruegerauthor=J.+Liangauthor=J.+Moffatauthor=G.+Phillipsauthor=R.+Pulkauthor=T.+E.+Rawsonauthor=M.+Ultschauthor=L.+Walkerauthor=C.+Wiesmannauthor=B.+Zhangauthor=B.+Y.+Zhuauthor=A.+G.+Cochran&title=A+class+of+2%2C4-bisanilinopyrimidine+Aurora+A+inhibitors+with+unusually+high+selectivity+against+Aurora+B&doi=10.1021%2Fjm9000314"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm9000314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9000314%26sid%3Dliteratum%253Aachs%26aulast%3DAliagas-Martin%26aufirst%3DI.%26aulast%3DBurdick%26aufirst%3DD.%26aulast%3DCorson%26aufirst%3DL.%26aulast%3DDotson%26aufirst%3DJ.%26aulast%3DDrummond%26aufirst%3DJ.%26aulast%3DFields%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DKrueger%26aufirst%3DE.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DG.%26aulast%3DPulk%26aufirst%3DR.%26aulast%3DRawson%26aufirst%3DT.%2BE.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DL.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DB.%2BY.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DA%2520class%2520of%25202%252C4-bisanilinopyrimidine%2520Aurora%2520A%2520inhibitors%2520with%2520unusually%2520high%2520selectivity%2520against%2520Aurora%2520B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D3300%26epage%3D3307%26doi%3D10.1021%2Fjm9000314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moslin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santella, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncia, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strnad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedicord, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zupa-Fernandez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arienzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muckelbauer, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tredup, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aranibar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, D. S.</span></span> <span> </span><span class="NLM_article-title">Identification of imidazo[1,2-b]pyridazine TYK2 pseudokinase ligands as potent and selective allosteric inhibitors of TYK2 signalling</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1039/C6MD00560H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1039%2FC6MD00560H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=30108788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyru7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=700-712&author=R.+Moslinauthor=D.+Gardnerauthor=J.+Santellaauthor=Y.+Zhangauthor=J.+V.+Dunciaauthor=C.+Liuauthor=J.+Linauthor=J.+S.+Tokarskiauthor=J.+Strnadauthor=D.+Pedicordauthor=J.+Chenauthor=Y.+Blatauthor=A.+Zupa-Fernandezauthor=L.+Chengauthor=H.+Sunauthor=C.+Chaudhryauthor=C.+Huangauthor=C.+D%E2%80%99Arienzoauthor=J.+S.+Sackauthor=J.+K.+Muckelbauerauthor=C.+Changauthor=J.+Tredupauthor=D.+Xieauthor=N.+Aranibarauthor=J.+R.+Burkeauthor=P.+H.+Carterauthor=D.+S.+Weinstein&title=Identification+of+imidazo%5B1%2C2-b%5Dpyridazine+TYK2+pseudokinase+ligands+as+potent+and+selective+allosteric+inhibitors+of+TYK2+signalling&doi=10.1039%2FC6MD00560H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of imidazo[1,2-b]pyridazine TYK2 pseudokinase ligands as potent and selective allosteric inhibitors of TYK2 signalling</span></div><div class="casAuthors">Moslin, R.; Gardner, D.; Santella, J.; Zhang, Y.; Duncia, J. V.; Liu, C.; Lin, J.; Tokarski, J. S.; Strnad, J.; Pedicord, D.; Chen, J.; Blat, Y.; Zupa-Fernandez, A.; Cheng, L.; Sun, H.; Chaudhry, C.; Huang, C.; D'Arienzo, C.; Sack, J. S.; Muckelbauer, J. K.; Chang, C.; Tredup, J.; Xie, D.; Aranibar, N.; Burke, J. R.; Carter, P. H.; Weinstein, D. S.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">700-712</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As a member of the Janus (JAK) family of non-receptor tyrosine kinases, TYK2 mediates the signaling of pro-inflammatory cytokines including IL-12, IL-23 and type 1 interferon (IFN), and therefore represents an attractive potential target for treating the various immuno-inflammatory diseases in which these cytokines have been shown to play a role.  Following up on our previous report that ligands to the pseudokinase domain (JH2) of TYK2 suppress cytokine-mediated receptor activation of the catalytic (JH1) domain, the imidazo[1,2-b]pyridazine (IZP) 7 was identified as a promising hit compd.  Through iterative modification of each of the substituents of the IZP scaffold, the cellular potency was improved while maintaining selectivity over the JH1 domain.  These studies led to the discovery of the JH2-selective TYK2 inhibitor 29, which provided encouraging systemic exposures after oral dosing in mice.  Phosphodiesterase 4 (PDE4) was identified as an off-target and potential liability of the IZP ligands, and selectivity for TYK2 JH2 over this enzyme was obtained by elaborating along selectivity vectors detd. from analyses of X-ray co-crystal structures of representative ligands of the IZP class bound to both proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobX5_XfvLHsLVg90H21EOLACvtfcHk0librWFDwl5geQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyru7jN&md5=67b06413866ee6096810903f62edd901</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC6MD00560H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00560H%26sid%3Dliteratum%253Aachs%26aulast%3DMoslin%26aufirst%3DR.%26aulast%3DGardner%26aufirst%3DD.%26aulast%3DSantella%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDuncia%26aufirst%3DJ.%2BV.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DTokarski%26aufirst%3DJ.%2BS.%26aulast%3DStrnad%26aufirst%3DJ.%26aulast%3DPedicord%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBlat%26aufirst%3DY.%26aulast%3DZupa-Fernandez%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DChaudhry%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Arienzo%26aufirst%3DC.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DMuckelbauer%26aufirst%3DJ.%2BK.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DTredup%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DD.%26aulast%3DAranibar%26aufirst%3DN.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DWeinstein%26aufirst%3DD.%2BS.%26atitle%3DIdentification%2520of%2520imidazo%255B1%252C2-b%255Dpyridazine%2520TYK2%2520pseudokinase%2520ligands%2520as%2520potent%2520and%2520selective%2520allosteric%2520inhibitors%2520of%2520TYK2%2520signalling%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D700%26epage%3D712%26doi%3D10.1039%2FC6MD00560H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luk, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, C.</span></span> <span> </span><span class="NLM_article-title">Rational design of highly selective spleen tyrosine kinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">10414</span>– <span class="NLM_lpage">10423</span>, <span class="refDoi"> DOI: 10.1021/jm301367c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301367c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10414-10423&author=M.+C.+Lucasauthor=D.+M.+Goldsteinauthor=J.+C.+Hermannauthor=A.+Kuglstatterauthor=W.+Liuauthor=K.+C.+Lukauthor=F.+Padillaauthor=M.+Sladeauthor=A.+G.+Villasenorauthor=J.+Wannerauthor=W.+Xieauthor=X.+Zhangauthor=C.+Liao&title=Rational+design+of+highly+selective+spleen+tyrosine+kinase+inhibitors&doi=10.1021%2Fjm301367c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm301367c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301367c%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DM.%2BC.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DKuglstatter%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLuk%26aufirst%3DK.%2BC.%26aulast%3DPadilla%26aufirst%3DF.%26aulast%3DSlade%26aufirst%3DM.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DWanner%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLiao%26aufirst%3DC.%26atitle%3DRational%2520design%2520of%2520highly%2520selective%2520spleen%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10414%26epage%3D10423%26doi%3D10.1021%2Fjm301367c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougan, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komandla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyrick, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of a small molecule inhibitor of Bruton’s tyrosine kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5437</span>– <span class="NLM_lpage">5444</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00734</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00734" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVans7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5437-5444&author=C.+R.+Smithauthor=D.+R.+Douganauthor=M.+Komandlaauthor=T.+Kanouniauthor=B.+Knightauthor=J.+D.+Lawsonauthor=M.+Sabatauthor=E.+R.+Taylorauthor=P.+Vuauthor=C.+Wyrick&title=Fragment-based+discovery+of+a+small+molecule+inhibitor+of+Bruton%E2%80%99s+tyrosine+kinase&doi=10.1021%2Facs.jmedchem.5b00734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of a Small Molecule Inhibitor of Bruton's Tyrosine Kinase</span></div><div class="casAuthors">Smith, Christopher R.; Dougan, Douglas R.; Komandla, Mallareddy; Kanouni, Toufike; Knight, Beverly; Lawson, J. David; Sabat, Mark; Taylor, Ewan R.; Vu, Phong; Wyrick, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5437-5444</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery and optimization of a series of 4-aminocinnoline-3-carboxamide inhibitors of Bruton's tyrosine kinase are reported.  A fragment-based screening approach incorporating x-ray cocrystallog. was used to identify a cinnoline fragment and characterize its binding mode in the ATP binding site of Btk.  Optimization of the fragment hit resulted in the identification of a lead compd. I which reduced paw swelling in a dose- and exposure-dependent fashion in a rat model of collagen-induced arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozENTW7pHlFrVg90H21EOLACvtfcHk0lgbnAUpBpelcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVans7rF&md5=d6626fabcc3e2aa6a9180dc702a5030c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00734%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BR.%26aulast%3DDougan%26aufirst%3DD.%2BR.%26aulast%3DKomandla%26aufirst%3DM.%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DKnight%26aufirst%3DB.%26aulast%3DLawson%26aufirst%3DJ.%2BD.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DE.%2BR.%26aulast%3DVu%26aufirst%3DP.%26aulast%3DWyrick%26aufirst%3DC.%26atitle%3DFragment-based%2520discovery%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5437%26epage%3D5444%26doi%3D10.1021%2Facs.jmedchem.5b00734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paavilainen, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafimova, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Electrophilic fragment-based design of reversible covalent kinase inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">5298</span>– <span class="NLM_lpage">5301</span>, <span class="refDoi"> DOI: 10.1021/ja401221b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja401221b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=5298-5301&author=R.+M.+Millerauthor=V.+O.+Paavilainenauthor=S.+Krishnanauthor=I.+M.+Serafimovaauthor=J.+Taunton&title=Electrophilic+fragment-based+design+of+reversible+covalent+kinase+inhibitors&doi=10.1021%2Fja401221b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophilic Fragment-Based Design of Reversible Covalent Kinase Inhibitors</span></div><div class="casAuthors">Miller, Rand M.; Paavilainen, Ville O.; Krishnan, Shyam; Serafimova, Iana M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5298-5301</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based ligand design and covalent targeting of noncatalytic cysteines have been employed to develop potent and selective kinase inhibitors.  Here, we combine these approaches, starting with a panel of low-mol.-wt., heteroaryl-substituted cyanoacrylamides, which we have previously shown to form reversible covalent bonds with cysteine thiols.  Using this strategy, we identify electrophilic fragments with sufficient ligand efficiency and selectivity to serve as starting points for the first reported inhibitors of the MSK1 C-terminal kinase domain.  Guided by X-ray co-crystal structures, indazole fragment 1 was elaborated to afford 12 (RMM-46), a reversible covalent inhibitor that exhibits high ligand efficiency and selectivity for MSK/RSK-family kinases.  At nanomolar concns., 12 blocked activation of cellular MSK and RSK, as well as downstream phosphorylation of the crit. transcription factor, CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrax7wBNWOT8rVg90H21EOLACvtfcHk0lgbnAUpBpelcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D&md5=c6100b56ce6f587f6afb834aeafec351</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja401221b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja401221b%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DElectrophilic%2520fragment-based%2520design%2520of%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D5298%26epage%3D5301%26doi%3D10.1021%2Fja401221b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korennykh, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrman, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">Mammalian endoplasmic reticulum stress sensor IRE1 signals by dynamic clustering</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">16113</span>– <span class="NLM_lpage">16118</span>, <span class="refDoi"> DOI: 10.1073/pnas.1010580107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1073%2Fpnas.1010580107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20798350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2ju7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=16113-16118&author=H.+Liauthor=A.+V.+Korennykhauthor=S.+L.+Behrmanauthor=P.+Walter&title=Mammalian+endoplasmic+reticulum+stress+sensor+IRE1+signals+by+dynamic+clustering&doi=10.1073%2Fpnas.1010580107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian endoplasmic reticulum stress sensor IRE1 signals by dynamic clustering</span></div><div class="casAuthors">Li, Han; Korennykh, Alexei V.; Behrman, Shannon L.; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">16113-16118, S16113/1-S16113/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Accumulation of misfolded proteins in the endoplasmic reticulum (ER) triggers the unfolded protein response (UPR), an intracellular signaling pathway that adjusts the protein folding capacity of the ER according to need.  If homeostasis in the ER protein folding environment cannot be reestablished, cells commit to apoptosis.  The ER-resident transmembrane kinase-endo-RNase inositol-requiring enzyme 1 (IRE1) is the best characterized UPR signal transduction mol.  In yeast, Ire1 oligomerizes upon activation in response to an accumulation of misfolded proteins in the ER.  Here we show that the salient mechanistic features of IRE1 activation are conserved: mammalian IRE1 oligomerizes in the ER membrane and oligomerization correlates with the onset of IRE1 phosphorylation and RNase activity.  Moreover, the kinase/RNase module of human IRE1 activates cooperatively in vitro, indicating that formation of oligomers larger than four IRE1 mols. takes place upon activation.  High-order IRE1 oligomerization thus emerges as a conserved mechanism of IRE1 signaling.  IRE1 signaling attenuates after prolonged ER stress.  IRE1 then enters a refractive state even if ER stress remains unmitigated.  Attenuation includes dissoln. of IRE1 clusters, IRE1 dephosphorylation, and decline in endo-RNase activity.  Thus IRE1 activity is governed by a timer that may be important in switching the UPR from the initially cytoprotective phase to the apoptotic mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhiPftLtEiZLVg90H21EOLACvtfcHk0ljlNwFq4qogKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2ju7bJ&md5=ecf826507d91fbd659134634eba800c9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1010580107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1010580107%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DKorennykh%26aufirst%3DA.%2BV.%26aulast%3DBehrman%26aufirst%3DS.%2BL.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DMammalian%2520endoplasmic%2520reticulum%2520stress%2520sensor%2520IRE1%2520signals%2520by%2520dynamic%2520clustering%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D16113%26epage%3D16118%26doi%3D10.1073%2Fpnas.1010580107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haq, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span> <span> </span><span class="NLM_article-title">The Ys and wherefores of protein kinase autoinhibition</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1854</i></span>,  <span class="NLM_fpage">1586</span>– <span class="NLM_lpage">1594</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2015.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.bbapap.2015.04.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1854&publication_year=2015&pages=1586-1594&author=R.+Baylissauthor=T.+Haqauthor=S.+Yeoh&title=The+Ys+and+wherefores+of+protein+kinase+autoinhibition&doi=10.1016%2Fj.bbapap.2015.04.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2015.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2015.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DHaq%26aufirst%3DT.%26aulast%3DYeoh%26aufirst%3DS.%26atitle%3DThe%2520Ys%2520and%2520wherefores%2520of%2520protein%2520kinase%2520autoinhibition%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2015%26volume%3D1854%26spage%3D1586%26epage%3D1594%26doi%3D10.1016%2Fj.bbapap.2015.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thevakumaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Dimerization-induced allostery in protein kinase regulation</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2014.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.tibs.2014.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=25220378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=475-486&author=H.+Lavoieauthor=J.+J.+Liauthor=N.+Thevakumaranauthor=M.+Therrienauthor=F.+Sicheri&title=Dimerization-induced+allostery+in+protein+kinase+regulation&doi=10.1016%2Fj.tibs.2014.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Dimerization-induced allostery in protein kinase regulation</span></div><div class="casAuthors">Lavoie, Hugo; Li, John J.; Thevakumaran, Neroshan; Therrien, Marc; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">475-486</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The ability of protein kinases to switch between inactive and active states is crit. to control the outputs of cellular signaling pathways.  In several protein kinases, the conformation of helix αC is a key hub on which regulatory inputs converge to induce catalytic switching.  An emerging mechanism involved in regulating helix αC orientation is the allosteric coupling with kinase domain surfaces involved in homo- or heterodimerization.  In this review, we discuss dimerization-mediated regulation of the rapidly accelerated fibrosarcoma (RAF) and eIF2α kinase families and draw parallels with the analogous behavior of the epidermal growth factor receptor (EGFR) and serine/threonine-protein kinase endoribonuclease 1 (IRE1)/RNase L (RNAse L) kinase families.  Given that resistance to RAF-targeted therapeutics often stems from dimerization-dependent mechanisms, we suggest that a better understanding of dimerization-induced allostery may assist in developing alternate therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaIF5kAf47DbVg90H21EOLACvtfcHk0ljlNwFq4qogKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7N&md5=34d4079ea8c7906193dc4f2384bf6364</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DLavoie%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DThevakumaran%26aufirst%3DN.%26aulast%3DTherrien%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DDimerization-induced%2520allostery%2520in%2520protein%2520kinase%2520regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26spage%3D475%26epage%3D486%26doi%3D10.1016%2Fj.tibs.2014.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandhuber, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlam, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seshagiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeppen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span> <span> </span><span class="NLM_article-title">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">435</span>, <span class="refDoi"> DOI: 10.1038/nature08833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature08833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20130576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=431-435&author=G.+Hatzivassiliouauthor=K.+Songauthor=I.+Yenauthor=B.+J.+Brandhuberauthor=D.+J.+Andersonauthor=R.+Alvaradoauthor=M.+J.+Ludlamauthor=D.+Stokoeauthor=S.+L.+Gloorauthor=G.+Vigersauthor=T.+Moralesauthor=I.+Aliagasauthor=B.+Liuauthor=S.+Siderisauthor=K.+P.+Hoeflichauthor=B.+S.+Jaiswalauthor=S.+Seshagiriauthor=H.+Koeppenauthor=M.+Belvinauthor=L.+S.+Friedmanauthor=S.+Malek&title=RAF+inhibitors+prime+wild-type+RAF+to+activate+the+MAPK+pathway+and+enhance+growth&doi=10.1038%2Fnature08833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span></div><div class="casAuthors">Hatzivassiliou, Georgia; Song, Kyung; Yen, Ivana; Brandhuber, Barbara J.; Anderson, Daniel J.; Alvarado, Ryan; Ludlam, Mary J. C.; Stokoe, David; Gloor, Susan L.; Vigers, Guy; Morales, Tony; Aliagas, Ignacio; Liu, Bonnie; Sideris, Steve; Hoeflich, Klaus P.; Jaiswal, Bijay S.; Seshagiri, Somasekar; Koeppen, Hartmut; Belvin, Marcia; Friedman, Lori S.; Malek, Shiva</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7287</span>),
    <span class="NLM_cas:pages">431-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activating mutations in KRAS and BRAF are found in more than 30% of all human tumors and 40% of melanoma, resp., thus targeting this pathway could have broad therapeutic effects.  Small mol. ATP-competitive RAF kinase inhibitors have potent antitumor effects on mutant BRAF(V600E) tumors but, in contrast to mitogen-activated protein kinase kinase (MEK) inhibitors, are not potent against RAS mutant tumor models, despite RAF functioning as a key effector downstream of RAS and upstream of MEK.  Here we show that ATP-competitive RAF inhibitors have two opposing mechanisms of action depending on the cellular context.  In BRAF(V600E) tumors, RAF inhibitors effectively block the mitogen-activated protein kinase (MAPK) signalling pathway and decrease tumor growth.  Notably, in KRAS mutant and RAS/RAF wild-type tumors, RAF inhibitors activate the RAF-MEK-ERK pathway in a RAS-dependent manner, thus enhancing tumor growth in some xenograft models.  Inhibitor binding activates wild-type RAF isoforms by inducing dimerization, membrane localization and interaction with RAS-GTP.  These events occur independently of kinase inhibition and are, instead, linked to direct conformational effects of inhibitors on the RAF kinase domain.  On the basis of these findings, we demonstrate that ATP-competitive kinase inhibitors can have opposing functions as inhibitors or activators of signalling pathways, depending on the cellular context.  Furthermore, this work provides new insights into the therapeutic use of ATP-competitive RAF inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGounpcMETzVoLVg90H21EOLACvtfcHk0ljKnaWt1-vOzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D&md5=eb48bdf352ee9c9e182e63e38fd820a5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature08833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08833%26sid%3Dliteratum%253Aachs%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DBrandhuber%26aufirst%3DB.%2BJ.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DAlvarado%26aufirst%3DR.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DGloor%26aufirst%3DS.%2BL.%26aulast%3DVigers%26aufirst%3DG.%26aulast%3DMorales%26aufirst%3DT.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DJaiswal%26aufirst%3DB.%2BS.%26aulast%3DSeshagiri%26aufirst%3DS.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26atitle%3DRAF%2520inhibitors%2520prime%2520wild-type%2520RAF%2520to%2520activate%2520the%2520MAPK%2520pathway%2520and%2520enhance%2520growth%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D431%26epage%3D435%26doi%3D10.1038%2Fnature08833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poulikakos, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">RAF inhibitors transactivate RAF dimers and ERK signalling in cells with wild-type BRAF</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1038/nature08902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature08902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20179705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsFWjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=427-430&author=P.+I.+Poulikakosauthor=C.+Zhangauthor=G.+Bollagauthor=K.+M.+Shokatauthor=N.+Rosen&title=RAF+inhibitors+transactivate+RAF+dimers+and+ERK+signalling+in+cells+with+wild-type+BRAF&doi=10.1038%2Fnature08902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">RAF inhibitors transactivate RAF dimers and ERK signalling in cells with wild-type BRAF</span></div><div class="casAuthors">Poulikakos, Poulikos I.; Zhang, Chao; Bollag, Gideon; Shokat, Kevan M.; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7287</span>),
    <span class="NLM_cas:pages">427-430</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumors with mutant BRAF are dependent on the RAF-MEK-ERK signaling pathway for their growth.  We found that ATP-competitive RAF inhibitors inhibit ERK signaling in cells with mutant BRAF, but unexpectedly enhance signaling in cells with wild-type BRAF.  Here we demonstrate the mechanistic basis for these findings.  We used chem. genetic methods to show that drug-mediated transactivation of RAF dimers is responsible for paradoxical activation of the enzyme by inhibitors.  Induction of ERK signaling requires direct binding of the drug to the ATP-binding site of one kinase of the dimer and is dependent on RAS activity.  Drug binding to one member of RAF homodimers (CRAF-CRAF) or heterodimers (CRAF-BRAF) inhibits one protomer, but results in transactivation of the drug-free protomer.  In BRAF(V600E) tumors, RAS is not activated, thus transactivation is minimal and ERK signaling is inhibited in cells exposed to RAF inhibitors.  These results indicate that RAF inhibitors will be effective in tumors in which BRAF is mutated.  Furthermore, because RAF inhibitors do not inhibit ERK signaling in other cells, the model predicts that they would have a higher therapeutic index and greater antitumor activity than mitogen-activated protein kinase (MEK) inhibitors, but could also cause toxicity due to MEK/ERK activation.  These predictions have been borne out in a recent clin. trial of the RAF inhibitor PLX4032 (refs 4, 5).  The model indicates that promotion of RAF dimerization by elevation of wild-type RAF expression or RAS activity could lead to drug resistance in mutant BRAF tumors.  In agreement with this prediction, RAF inhibitors do not inhibit ERK signaling in cells that coexpress BRAF(V600E) and mutant RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMlm9feupyNrVg90H21EOLACvtfcHk0ljKnaWt1-vOzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsFWjsrs%253D&md5=8323a558886ccbb8f7d85b22b9bfbc15</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnature08902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08902%26sid%3Dliteratum%253Aachs%26aulast%3DPoulikakos%26aufirst%3DP.%2BI.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DRAF%2520inhibitors%2520transactivate%2520RAF%2520dimers%2520and%2520ERK%2520signalling%2520in%2520cells%2520with%2520wild-type%2520BRAF%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D427%26epage%3D430%26doi%3D10.1038%2Fnature08902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arora, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Michele, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermarliere, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Heerde, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linders, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span> <span> </span><span class="NLM_article-title">Structural investigation of B-Raf paradox breaker and inducer inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1818</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1021/jm501667n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501667n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1818-1831&author=R.+Aroraauthor=M.+Di+Micheleauthor=E.+Stesauthor=E.+Vandermarliereauthor=L.+Martensauthor=K.+Gevaertauthor=E.+Van+Heerdeauthor=J.+T.+Lindersauthor=D.+Brehmerauthor=E.+Jacobyauthor=P.+Bonnet&title=Structural+investigation+of+B-Raf+paradox+breaker+and+inducer+inhibitors&doi=10.1021%2Fjm501667n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Investigation of B-Raf Paradox Breaker and Inducer Inhibitors</span></div><div class="casAuthors">Arora, Rohit; Di Michele, Michela; Stes, Elisabeth; Vandermarliere, Elien; Martens, Lennart; Gevaert, Kris; Van Heerde, Erika; Linders, Joannes T. M.; Brehmer, Dirk; Jacoby, Edgar; Bonnet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1818-1831</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The V600E missense mutation in B-Raf kinase leads to an anomalous regulation of the MAPK pathway, uncontrolled cell proliferation, and initiation of tumorigenesis.  While the ATP-competitive B-Raf inhibitors block the MAPK pathway in B-Raf mutant cells, they induce conformational changes to wild-type B-Raf kinase domain leading to heterodimerization with C-Raf causing a paradoxical hyperactivation of MAPK pathway.  A new class of inhibitors (paradox breakers) has been developed that inhibit B-RafV600E activity without agonistically affecting the MAPK pathway in wild-type B-Raf cells.  In this study, we explore the structural, conformational, and cellular effects on the B-Raf kinase domain upon binding of paradox breakers and inducers.  Our results indicate that a subtle structural difference between paradox inducers and breakers leads to significant conformational differences when complexed with B-Raf.  This study provides a novel insight into the activation of B-Raf by ATP-competitive inhibitors and can aid in the design of more potent and selective inhibitors without agonistic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDJw-VSU5CabVg90H21EOLACvtfcHk0lidZHazd3HLkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtr8%253D&md5=c79cfd000d525f553a5247b9e917b3e7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm501667n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501667n%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DR.%26aulast%3DDi%2BMichele%26aufirst%3DM.%26aulast%3DStes%26aufirst%3DE.%26aulast%3DVandermarliere%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DL.%26aulast%3DGevaert%26aufirst%3DK.%26aulast%3DVan%2BHeerde%26aufirst%3DE.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DBonnet%26aufirst%3DP.%26atitle%3DStructural%2520investigation%2520of%2520B-Raf%2520paradox%2520breaker%2520and%2520inducer%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1818%26epage%3D1831%26doi%3D10.1021%2Fjm501667n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulkes, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyers, P. A.</span></span> <span> </span><span class="NLM_article-title">Pseudokinases: update on their functions and evaluation as new drug targets</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">265</span>, <span class="refDoi"> DOI: 10.4155/fmc-2016-0207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.4155%2Ffmc-2016-0207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28097887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslWnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=245-265&author=D.+P.+Byrneauthor=D.+M.+Foulkesauthor=P.+A.+Eyers&title=Pseudokinases%3A+update+on+their+functions+and+evaluation+as+new+drug+targets&doi=10.4155%2Ffmc-2016-0207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudokinases: update on their functions and evaluation as new drug targets</span></div><div class="casAuthors">Byrne, Dominic P.; Foulkes, Daniel M.; Eyers, Patrick A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">245-265</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The pseudokinase complement of the human kinase superfamily consists of approx. 60 signaling proteins, which lacks one or more of the amino acids typically required to correctly align ATP and metal ions, and phosphorylate protein substrates.  Recent studies in the pseudokinase field have begun to expose the biol. relevance of pseudokinases, which are now thought to perform a diverse range of physiol. roles and are connected to a multitude of human diseases, including cancer.  In this review, we discuss how and why members of the 'pseudokinome' represent important new targets for drug discovery, and describe how knowledge of protein structure and function provides informative clues to help guide the rational chem. design or repurposing of inhibitors to target pseudokinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqInUA03dIObVg90H21EOLACvtfcHk0lidZHazd3HLkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslWnurg%253D&md5=290fd229e4f7e5fefc308b2fae35881d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0207%26sid%3Dliteratum%253Aachs%26aulast%3DByrne%26aufirst%3DD.%2BP.%26aulast%3DFoulkes%26aufirst%3DD.%2BM.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26atitle%3DPseudokinases%253A%2520update%2520on%2520their%2520functions%2520and%2520evaluation%2520as%2520new%2520drug%2520targets%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D245%26epage%3D265%26doi%3D10.4155%2Ffmc-2016-0207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raundhal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaurio, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konrad, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettigole, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowley, M. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerliani, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenkov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motorykin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfredi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamarin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinovich, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubillos-Ruiz, J. R.</span></span> <span> </span><span class="NLM_article-title">IRE1α-XBP1 controls T cell function in ovarian cancer by regulating mitochondrial activity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>562</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0597-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fs41586-018-0597-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=30305738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=562&publication_year=2018&pages=423-428&author=M.+Songauthor=T.+A.+Sandovalauthor=C.+S.+Chaeauthor=S.+Chopraauthor=C.+Tanauthor=M.+R.+Rutkowskiauthor=M.+Raundhalauthor=R.+A.+Chaurioauthor=K.+K.+Payneauthor=C.+Konradauthor=S.+E.+Bettigoleauthor=H.+R.+Shinauthor=M.+J.+P.+Crowleyauthor=J.+P.+Cerlianiauthor=A.+V.+Kossenkovauthor=I.+Motorykinauthor=S.+Zhangauthor=G.+Manfrediauthor=D.+Zamarinauthor=K.+Holcombauthor=P.+C.+Rodriguezauthor=G.+A.+Rabinovichauthor=J.+R.+Conejo-Garciaauthor=L.+H.+Glimcherauthor=J.+R.+Cubillos-Ruiz&title=IRE1%CE%B1-XBP1+controls+T+cell+function+in+ovarian+cancer+by+regulating+mitochondrial+activity&doi=10.1038%2Fs41586-018-0597-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">IRE1α-XBP1 controls T cell function in ovarian cancer by regulating mitochondrial activity</span></div><div class="casAuthors">Song, Minkyung; Sandoval, Tito A.; Chae, Chang-Suk; Chopra, Sahil; Tan, Chen; Rutkowski, Melanie R.; Raundhal, Mahesh; Chaurio, Ricardo A.; Payne, Kyle K.; Konrad, Csaba; Bettigole, Sarah E.; Shin, Hee Rae; Crowley, Michael J. P.; Cerliani, Juan P.; Kossenkov, Andrew V.; Motorykin, Ievgen; Zhang, Sheng; Manfredi, Giovanni; Zamarin, Dmitriy; Holcomb, Kevin; Rodriguez, Paulo C.; Rabinovich, Gabriel A.; Conejo-Garcia, Jose R.; Glimcher, Laurie H.; Cubillos-Ruiz, Juan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">562</span>
        (<span class="NLM_cas:issue">7727</span>),
    <span class="NLM_cas:pages">423-428</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Tumors evade immune control by creating hostile microenvironments that perturb T cell metab. and effector function.  However, it remains unclear how intra-tumoral T cells integrate and interpret metabolic stress signals.  Here the authors report that ovarian cancer -an aggressive malignancy that is refractory to std. treatments and current immunotherapies -induces endoplasmic reticulum stress and activates the IRE1α-XBP1 arm of the unfolded protein response in T cells to control their mitochondrial respiration and anti-tumor function.  In T cells isolated from specimens collected from patients with ovarian cancer, up-regulation of XBP1 was assocd. with decreased infiltration of T cells into tumors and with reduced IFNG mRNA expression.  Malignant ascites fluid obtained from patients with ovarian cancer inhibited glucose uptake and caused N-linked protein glycosylation defects in T cells, which triggered IRE1α-XBP1 activation that suppressed mitochondrial activity and IFNγ prodn.  Mechanistically, induction of XBP1 regulated the abundance of glutamine carriers and thus limited the influx of glutamine that is necessary to sustain mitochondrial respiration in T cells under glucose-deprived conditions.  Restoring N-linked protein glycosylation, abrogating IRE1α-XBP1 activation or enforcing expression of glutamine transporters enhanced mitochondrial respiration in human T cells exposed to ovarian cancer ascites.  XBP1-deficient T cells in the metastatic ovarian cancer milieu exhibited global transcriptional reprogramming and improved effector capacity.  Accordingly, mice that bear ovarian cancer and lack XBP1 selectively in T cells demonstrate superior anti-tumor immunity, delayed malignant progression and increased overall survival.  Controlling endoplasmic reticulum stress or targeting IRE1α-XBP1 signaling may help to restore the metabolic fitness and anti-tumor capacity of T cells in cancer hosts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0WsJG_HHAh7Vg90H21EOLACvtfcHk0lidZHazd3HLkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtLrK&md5=f6280f187633b2498f2172703cc19768</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0597-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0597-x%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DM.%26aulast%3DSandoval%26aufirst%3DT.%2BA.%26aulast%3DChae%26aufirst%3DC.%2BS.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DRutkowski%26aufirst%3DM.%2BR.%26aulast%3DRaundhal%26aufirst%3DM.%26aulast%3DChaurio%26aufirst%3DR.%2BA.%26aulast%3DPayne%26aufirst%3DK.%2BK.%26aulast%3DKonrad%26aufirst%3DC.%26aulast%3DBettigole%26aufirst%3DS.%2BE.%26aulast%3DShin%26aufirst%3DH.%2BR.%26aulast%3DCrowley%26aufirst%3DM.%2BJ.%2BP.%26aulast%3DCerliani%26aufirst%3DJ.%2BP.%26aulast%3DKossenkov%26aufirst%3DA.%2BV.%26aulast%3DMotorykin%26aufirst%3DI.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DManfredi%26aufirst%3DG.%26aulast%3DZamarin%26aufirst%3DD.%26aulast%3DHolcomb%26aufirst%3DK.%26aulast%3DRodriguez%26aufirst%3DP.%2BC.%26aulast%3DRabinovich%26aufirst%3DG.%2BA.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26aulast%3DCubillos-Ruiz%26aufirst%3DJ.%2BR.%26atitle%3DIRE1%25CE%25B1-XBP1%2520controls%2520T%2520cell%2520function%2520in%2520ovarian%2520cancer%2520by%2520regulating%2520mitochondrial%2520activity%26jtitle%3DNature%26date%3D2018%26volume%3D562%26spage%3D423%26epage%3D428%26doi%3D10.1038%2Fs41586-018-0597-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lj7URIonG_X2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lj7URIonG_X2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhiRFTRCLbpDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhiRFTRCLbpDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span> <i>Maestro</i>, version 9.3; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Maestro%2C+version+9.3%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span> <i>Glide</i>, version 5.8; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+version+5.8%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span> <i>LigPrep</i>, version 2.5; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+version+2.5%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhegalova, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezin, M. Y.</span></span> <span> </span><span class="NLM_article-title">Synthesis of nitric oxide probes with fluorescence lifetime sensitivity</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">8228</span>– <span class="NLM_lpage">8234</span>, <span class="refDoi"> DOI: 10.1039/c3ob41498a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1039%2Fc3ob41498a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=24166035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslOmsbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=8228-8234&author=N.+G.+Zhegalovaauthor=G.+Gonzalesauthor=M.+Y.+Berezin&title=Synthesis+of+nitric+oxide+probes+with+fluorescence+lifetime+sensitivity&doi=10.1039%2Fc3ob41498a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of nitric oxide probes with fluorescence lifetime sensitivity</span></div><div class="casAuthors">Zhegalova, Natalia G.; Gonzales, Garrett; Berezin, Mikhail Y.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">8228-8234</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors present the rationale, synthesis and evaluation of the 1st activatable fluorescent probe that uses fluorescence lifetime change for detection of nitric oxide.  The new probe I (R = R' = H) features a near-IR polymethine skeleton with a diaminobenzene functionality incorporated into the meso-position.  The probe is partially quenched, and upon reaction with nitric oxide shows an increase in the fluorescence lifetime from 1.08 ns to 1.24 ns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUAAtEJ72VTrVg90H21EOLACvtfcHk0lhUZgOpzFFpag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslOmsbzF&md5=62e199c9a6b4bc72464a983f72ad3370</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2Fc3ob41498a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob41498a%26sid%3Dliteratum%253Aachs%26aulast%3DZhegalova%26aufirst%3DN.%2BG.%26aulast%3DGonzales%26aufirst%3DG.%26aulast%3DBerezin%26aufirst%3DM.%2BY.%26atitle%3DSynthesis%2520of%2520nitric%2520oxide%2520probes%2520with%2520fluorescence%2520lifetime%2520sensitivity%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D8228%26epage%3D8234%26doi%3D10.1039%2Fc3ob41498a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35a','cit35a1'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jeremy A. Turkett, Anna E. Ringuette, Craig W. Lindsley, <span class="NLM_string-name hlFld-ContribAuthor">Aaron M. Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Substituted 6,7-Dihydro-5H-pyrrolo[2,3-c]pyridazines/pyrazines via Catalyst-Free Tandem Hydroamination–Aromatic Substitution. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>85 </em>
                                    (9)
                                     , 6123-6130. <a href="https://doi.org/10.1021/acs.joc.9b03463" title="DOI URL">https://doi.org/10.1021/acs.joc.9b03463</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b03463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b03463%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Bof%252BSubstituted%252B6%25252C7-Dihydro-5H-pyrrolo%25255B2%25252C3-c%25255Dpyridazines%25252Fpyrazines%252Bvia%252BCatalyst-Free%252BTandem%252BHydroamination%2525E2%252580%252593Aromatic%252BSubstitution%26aulast%3DTurkett%26aufirst%3DJeremy%2BA.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D23122019%26date%3D09042020%26date%3D31032020%26volume%3D85%26issue%3D9%26spage%3D6123%26epage%3D6130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hannah C. Feldman, Venkata Narayana Vidadala, Zachary E. Potter, Feroz R. Papa, Bradley J. Backes, <span class="NLM_string-name hlFld-ContribAuthor">Dustin J. Maly</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Chemical Toolset for Studying the Paralog-Specific Function of IRE1. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (12)
                                     , 2595-2605. <a href="https://doi.org/10.1021/acschembio.9b00482" title="DOI URL">https://doi.org/10.1021/acschembio.9b00482</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00482%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDevelopment%252Bof%252Ba%252BChemical%252BToolset%252Bfor%252BStudying%252Bthe%252BParalog-Specific%252BFunction%252Bof%252BIRE1%26aulast%3DFeldman%26aufirst%3DHannah%2BC.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D17062019%26date%3D26092019%26date%3D14102019%26volume%3D14%26issue%3D12%26spage%3D2595%26epage%3D2605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Monica C.  Pillon</span>, <span class="hlFld-ContribAuthor ">Jacob  Gordon</span>, <span class="hlFld-ContribAuthor ">Meredith N.  Frazier</span>, <span class="hlFld-ContribAuthor ">Robin E.  Stanley</span>. </span><span class="cited-content_cbyCitation_article-title">HEPN RNases – an emerging class of functionally distinct RNA processing and degradation enzymes. </span><span class="cited-content_cbyCitation_journal-name">Critical Reviews in Biochemistry and Molecular Biology</span><span> <strong>2021,</strong> <em>56 </em>
                                    (1)
                                     , 88-108. <a href="https://doi.org/10.1080/10409238.2020.1856769" title="DOI URL">https://doi.org/10.1080/10409238.2020.1856769</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10409238.2020.1856769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10409238.2020.1856769%26sid%3Dliteratum%253Aachs%26jtitle%3DCritical%2520Reviews%2520in%2520Biochemistry%2520and%2520Molecular%2520Biology%26atitle%3DHEPN%252BRNases%252B%2525E2%252580%252593%252Ban%252Bemerging%252Bclass%252Bof%252Bfunctionally%252Bdistinct%252BRNA%252Bprocessing%252Band%252Bdegradation%252Benzymes%26aulast%3DPillon%26aufirst%3DMonica%2BC.%26date%3D2021%26date%3D2020%26volume%3D56%26issue%3D1%26spage%3D88%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">K. Alison  Rinderspacher</span>. </span><span class="cited-content_cbyCitation_article-title">Six-membered ring systems: diazines and benzo derivatives. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 431-466. <a href="https://doi.org/10.1016/B978-0-323-89812-6.00013-4" title="DOI URL">https://doi.org/10.1016/B978-0-323-89812-6.00013-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-323-89812-6.00013-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-323-89812-6.00013-4%26sid%3Dliteratum%253Aachs%26atitle%3DSix-membered%252Bring%252Bsystems%25253A%252Bdiazines%252Band%252Bbenzo%252Bderivatives%26aulast%3DRinderspacher%26aufirst%3DK.%2BAlison%26date%3D2021%26spage%3D431%26epage%3D466%26pub%3DElsevier%26date%3D2021%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Diana Pelizzari  Raymundo</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Doultsinos</span>, <span class="hlFld-ContribAuthor ">Xavier  Guillory</span>, <span class="hlFld-ContribAuthor ">Antonio  Carlesso</span>, <span class="hlFld-ContribAuthor ">Leif A.  Eriksson</span>, <span class="hlFld-ContribAuthor ">Eric  Chevet</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological Targeting of IRE1 in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Trends in Cancer</span><span> <strong>2020,</strong> <em>6 </em>
                                    (12)
                                     , 1018-1030. <a href="https://doi.org/10.1016/j.trecan.2020.07.006" title="DOI URL">https://doi.org/10.1016/j.trecan.2020.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.trecan.2020.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.trecan.2020.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Cancer%26atitle%3DPharmacological%252BTargeting%252Bof%252BIRE1%252Bin%252BCancer%26aulast%3DRaymundo%26aufirst%3DDiana%2BPelizzari%26date%3D2020%26volume%3D6%26issue%3D12%26spage%3D1018%26epage%3D1030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zekeriya  Duzgun</span>, <span class="hlFld-ContribAuthor ">Zuhal  Eroglu</span>. </span><span class="cited-content_cbyCitation_article-title">Role of cardiac drugs and flavonoids on the IRE1–JNK pathway as revealed by re-ranked molecular docking scores, MM/PBSA and umbrella sampling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>425 </em>, 1-23. <a href="https://doi.org/10.1080/07391102.2020.1851299" title="DOI URL">https://doi.org/10.1080/07391102.2020.1851299</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1851299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1851299%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DRole%252Bof%252Bcardiac%252Bdrugs%252Band%252Bflavonoids%252Bon%252Bthe%252BIRE1%2525E2%252580%252593JNK%252Bpathway%252Bas%252Brevealed%252Bby%252Bre-ranked%252Bmolecular%252Bdocking%252Bscores%25252C%252BMM%25252FPBSA%252Band%252Bumbrella%252Bsampling%26aulast%3DDuzgun%26aufirst%3DZekeriya%26date%3D2020%26date%3D2020%26volume%3D425%26spage%3D1%26epage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alan M.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Dialling‐In New Reactivity into the Shono‐type Anodic Oxidation Reaction. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2020,</strong> <em>52 </em><a href="https://doi.org/10.1002/tcr.202000116" title="DOI URL">https://doi.org/10.1002/tcr.202000116</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.202000116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.202000116%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DDialling%2525E2%252580%252590In%252BNew%252BReactivity%252Binto%252Bthe%252BShono%2525E2%252580%252590type%252BAnodic%252BOxidation%252BReaction%26aulast%3DJones%26aufirst%3DAlan%2BM.%26date%3D2020%26date%3D2020%26volume%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Garcia-Marin  Javier</span>. </span><span class="cited-content_cbyCitation_article-title">Computational insight into the selective allosteric inhibition for PTP1B versus TCPTP: a molecular modelling study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>31 </em>, 1-12. <a href="https://doi.org/10.1080/07391102.2020.1790421" title="DOI URL">https://doi.org/10.1080/07391102.2020.1790421</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1790421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1790421%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DComputational%252Binsight%252Binto%252Bthe%252Bselective%252Ballosteric%252Binhibition%252Bfor%252BPTP1B%252Bversus%252BTCPTP%25253A%252Ba%252Bmolecular%252Bmodelling%252Bstudy%26aulast%3DJavier%26aufirst%3DGarcia-Marin%26date%3D2020%26date%3D2020%26volume%3D31%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yantao  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in the Synthesis of Methylated Products through Indirect Approaches. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2020,</strong> <em>362 </em>
                                    (5)
                                     , 998-1014. <a href="https://doi.org/10.1002/adsc.201901240" title="DOI URL">https://doi.org/10.1002/adsc.201901240</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.201901240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.201901240%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DAdvances%252Bin%252Bthe%252BSynthesis%252Bof%252BMethylated%252BProducts%252Bthrough%252BIndirect%252BApproaches%26aulast%3DChen%26aufirst%3DYantao%26date%3D2020%26date%3D2019%26volume%3D362%26issue%3D5%26spage%3D998%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structures of the IRE1α RNase activator <b>1</b> and the structurally related RNase inhibitor <b>2</b>. (B) Structures of selected published IRE1α kinase-RNase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Compounds <b>2</b>–<b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 2-chloroacetaldehyde, EtOH, 50 °C, 21 h, 78%; (ii) <i>N</i>-bromosuccinimide, CH<sub>3</sub>Cl, 0 °C to room temperature (rt), 16 h, 84%; (iii) cyclopropylmethanamine, tetrahydrofuran (THF), rt, 3 h, 93%; (iv) heteroaryl boronate, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, dioxane, 120–135 °C, 15–22 h, 5–51%; (v) (a) PhNH<sub>2</sub>, 180 °C (microwave), 1 h; (b) (Boc)<sub>2</sub>O, 4-dimethylaminopyridine (DMAP), THF, rt, 16 h; (c) bis(pinacolato)diboron, KOAc, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, 1,4-dioxane, 100 °C, 16 h, 53% over three steps; (vi) PhNHMe, 180 °C (microwave), 1 h, 86%; (vii) bis(pinacolato)diboron, KOAc, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, 1,4-dioxane, 100 °C, 16 h, 36%; (viii) isothiocyanatobenzene, 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDC), THF, 50 °C, 5 h, 17–45%.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Compounds <b>19</b>–<b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) amine, THF, rt, 16 h, 60–97%; (ii) heteroaryl boronate, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 130 °C, 16 h, 4–55%; (iii) <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)carbamate, THF, 50 °C, 16 h, quantitative; (iv) HCl, 1,4-dioxane, rt, 2 h, 99%; (v) (a) LiAlH<sub>4</sub>, THF, 0 °C to reflux, 4 h; (b) ethyl 1,3-dioxoisoindoline-2-carboxylate, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 15 h; (c) <b>3</b>, Et<sub>3</sub>N, 1,4-dioxane–dimethyl sulfoxide (DMSO), 100 °C, 22 h, 52% over three steps; (vi) 5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-indazole, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)-ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 130 °C, 21 h, then NH<sub>2</sub>NH<sub>2</sub>·H<sub>2</sub>O, 100 °C, 48 h, 34%; (vii) amine, THF, 50 °C, 16 h, 29–85%; (viii) (a) <i>tert</i>-butyl ((1<i>r</i>,4<i>r</i>)-4-aminocyclohexyl)carbamate, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 20 h; (b) HCl–1,4-dioxane, MeOH, rt, 2 h, 37% over two steps; (ix) <b>3</b>, Et<sub>3</sub>N, 1,4-dioxane-DMSO, 50 °C, 23 h, 58%; (x) (a) isothiocyanatobenzene, EDC, THF, 50 °C, 5 h; (b) (Boc)<sub>2</sub>O, DMAP, THF, rt, 16 h; (c) bis(pinacolato)diboron, KOAc, Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub>, 1,4-dioxane, 135 °C, 24 h; (d) 3-bromo-6-chloro-<i>N</i>-((1-methylpiperidin-4-yl)methyl)imidazo[1,2-<i>b</i>]pyridazin-8-amine, Pd(OAc)<sub>2</sub>, 1,1′-bis(di-<i>tert</i>-butylphosphino)-ferrocene, 2 M Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 135 °C, 16 h, 2–5% over four steps.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding profiles of compounds <b>2</b>, <b>26</b>, and <b>31</b> tested at 1 μM concentrations of inhibitors against 455 wild-type and mutant human protein and lipid kinases (KINOME<i>scan</i>, Eurofins) (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf" class="ext-link">Table S1</a>). Kinases where probe binding was inhibited by >90% at 1 μM test compound concentration are listed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of IRE1α inhibitor binding modes. (A) Part of the crystal structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>) of <b>2</b> (green sticks) bound to IRE1α KEN domain (gray image) showing the conformations of the DFG motif (cyan), activation loop segment (pale cyan), Y628 (blue), K599 (gray), and the observed part of the αC-helix (red). (B) Binding of <b>2</b> (green sticks) in the IRE1α ATP site (gray sticks) with key residues labeled. Hydrogen bonds are shown as black dotted lines and selected water molecules as red spheres. (C) Details of the interactions of <b>2</b> (green sticks) with the DFG motif and Y628 in the IRE1 ATP site (gray image) with key residues labeled. Hydrogen bonds are shown as black dotted lines and selected water molecules as red spheres. (D) Comparison of <b>2</b> (purple), <b>1</b> (blue; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z7H">4Z7H</a>), Compound <b>16</b> (light blue; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4U6R">4U6R</a>), and GSK2850163 (red; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YZ9">4YZ9</a>) binding to IRE1α KEN domain. Structures are overlaid on the hinge region (E643-L644-C645; lines), and the αC-helix (L602-S619; cartoon) is shown in the three structures in which it is present. (E) Schematic overlay showing the relationship between the DFG-out, DFG-up, and DFG-in conformations and the corresponding orientations of the observed activation loop segments (L714-S729) for GSK2850163-IRE1α (red), <b>2</b>-IRE1 (purple), compound <b>16</b>-IRE1α (light blue), and <b>1</b>-IRE1α (blue). (F) Comparison of components of the N-terminal kinase domain dimerization interface for the DFG-in, active conformation of <b>1</b>-IRE1α (blue and blue-white protomers) and DFG-up, and inactive conformation of <b>2</b>-IRE1α (purple). Structures are aligned on the kinase C-lobe (shown) and endonuclease C-terminal domain (not shown). Bound ligands are rendered as sticks; (G) model of <b>9</b> (green sticks) bound to IRE1 (gray image) based on the conformation observed for <b>2</b>-IRE1. Predicted hydrogen bonds are shown as black dotted lines. (H) Detail showing the movement of Y628 as positioned in <b>2</b>-IRE1 (gray image) required to accommodate the modeled binding of <b>9</b> (green sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/medium/jm-2018-01721z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compounds <b>26</b> and <b>31</b> inhibit IRE1α oligomerization, IRE1α autophosphorylation, IRE1α RNase activity, XBP1s protein expression, and XBP1-dependent transcription in human cells. (A) Inhibition of tunicamycin-induced IRE1α oligomerization, measured by fluorescent foci formation in HEK293 cells stably transfected with doxycycline-inducible GFP-IRE1α after 5 h treatment with inhibitors (0–10 μM) in the absence or presence of tunicamycin (quantification of image fields from <i>n</i> = 2 experiments plotted separately). Dox = doxycycline, T = tunicamycin (10 μg/mL). (B) Inhibition of tunicamycin-induced pS724 IRE1α autophosphorylation as measured by capillary electrophoresis immunoassay (simple Western) relative to total IRE1α. Data shown for a single experiment representative of <i>n</i> = 3. (C) Inhibition of tunicamycin-induced XBP1s protein expression in H929 cells as measured by immunofluorescent assay (quantification of image fields from <i>n</i> > 3 experiments). (D) Inhibition of tunicamycin-induced XBP1s-dependent transcription of DNAJB9 mRNA as measured by real-time quantitative polymerase chain reaction (RT-qPCR). Data shown for a single experiment representative of <i>n</i> = 3. Tm = tunicamycin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01721/20190325/images/large/jm-2018-01721z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01721&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31806" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31806" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hetz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span> <span> </span><span class="NLM_article-title">The unfolded protein response and cell fate control</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.molcel.2017.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=29107536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehsL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2018&pages=169-181&author=C.+Hetzauthor=F.+R.+Papa&title=The+unfolded+protein+response+and+cell+fate+control&doi=10.1016%2Fj.molcel.2017.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Unfolded Protein Response and Cell Fate Control</span></div><div class="casAuthors">Hetz, Claudio; Papa, Feroz R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-181</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The secretory capacity of a cell is constantly challenged by physiol. demands and pathol. perturbations.  To adjust and match the protein-folding capacity of the endoplasmic reticulum (ER) to changing secretory needs, cells employ a dynamic intracellular signaling pathway known as the unfolded protein response (UPR).  Homeostatic activation of the UPR enforces adaptive programs that modulate and augment key aspects of the entire secretory pathway, whereas maladaptive UPR outputs trigger apoptosis.  Here, we discuss recent advances into how the UPR integrates information about the intensity and duration of ER stress stimuli in order to control cell fate.  These findings are timely and significant because they inform an evolving mechanistic understanding of a wide variety of human diseases, including diabetes mellitus, neurodegeneration, and cancer, thus opening up the potential for new therapeutic modalities to treat these diverse diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprBdkjctuvj7Vg90H21EOLACvtfcHk0ljGV3jiSAwmhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehsL7K&md5=59ef80187888b4c876e22bd63c8c0d7f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DHetz%26aufirst%3DC.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26atitle%3DThe%2520unfolded%2520protein%2520response%2520and%2520cell%2520fate%2520control%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D69%26spage%3D169%26epage%3D181%26doi%3D10.1016%2Fj.molcel.2017.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corazzari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagliardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fimia, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piacentini, M.</span></span> <span> </span><span class="NLM_article-title">Endoplasmic reticulum stress, unfolded protein response and cancer cell fate</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">78</span>, <span class="refDoi"> DOI: 10.3389/fonc.2017.00078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.3389%2Ffonc.2017.00078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28491820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC1crmtVCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=78&author=M.+Corazzariauthor=M.+Gagliardiauthor=G.+M.+Fimiaauthor=M.+Piacentini&title=Endoplasmic+reticulum+stress%2C+unfolded+protein+response+and+cancer+cell+fate&doi=10.3389%2Ffonc.2017.00078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Endoplasmic Reticulum Stress, Unfolded Protein Response, and Cancer Cell Fate</span></div><div class="casAuthors">Corazzari Marco; Corazzari Marco; Fimia Gian Maria; Piacentini Mauro; Gagliardi Mara; Piacentini Mauro; Fimia Gian Maria</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">78</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Perturbation of endoplasmic reticulum (ER) homeostasis results in a stress condition termed "ER stress" determining the activation of a finely regulated program defined as unfolded protein response (UPR) and whose primary aim is to restore this organelle's physiological activity.  Several physiological and pathological stimuli deregulate normal ER activity causing UPR activation, such as hypoxia, glucose shortage, genome instability, and cytotoxic compounds administration.  Some of these stimuli are frequently observed during uncontrolled proliferation of transformed cells, resulting in tumor core formation and stage progression.  Therefore, it is not surprising that ER stress is usually induced during solid tumor development and stage progression, becoming an hallmark of such malignancies.  Several UPR components are in fact deregulated in different tumor types, and accumulating data indicate their active involvement in tumor development/progression.  However, although the UPR program is primarily a pro-survival process, sustained and/or prolonged stress may result in cell death induction.  Therefore, understanding the mechanism(s) regulating the cell survival/death decision under ER stress condition may be crucial in order to specifically target tumor cells and possibly circumvent or overcome tumor resistance to therapies.  In this review, we discuss the role played by the UPR program in tumor initiation, progression and resistance to therapy, highlighting the recent advances that have improved our understanding of the molecular mechanisms that regulate the survival/death switch.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPyD_CTZi66dNTrvPZP1IpfW6udTcc2eb5NjTb7vpzbLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crmtVCjtQ%253D%253D&md5=3f762464fa3a8ccc1e3151a84f936859</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2017.00078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2017.00078%26sid%3Dliteratum%253Aachs%26aulast%3DCorazzari%26aufirst%3DM.%26aulast%3DGagliardi%26aufirst%3DM.%26aulast%3DFimia%26aufirst%3DG.%2BM.%26aulast%3DPiacentini%26aufirst%3DM.%26atitle%3DEndoplasmic%2520reticulum%2520stress%252C%2520unfolded%2520protein%2520response%2520and%2520cancer%2520cell%2520fate%26jtitle%3DFront.%2520Oncol.%26date%3D2017%26volume%3D7%26spage%3D78%26doi%3D10.3389%2Ffonc.2017.00078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neculai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dever, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Structure of the dual enzyme Ire1 reveals the basis for catalysis and regulation in nonconventional RNA splicing</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.10.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.cell.2007.10.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=18191223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFehtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2008&pages=89-100&author=K.+P.+Leeauthor=M.+Deyauthor=D.+Neculaiauthor=C.+Caoauthor=T.+E.+Deverauthor=F.+Sicheri&title=Structure+of+the+dual+enzyme+Ire1+reveals+the+basis+for+catalysis+and+regulation+in+nonconventional+RNA+splicing&doi=10.1016%2Fj.cell.2007.10.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the dual enzyme Ire1 reveals the basis for catalysis and regulation in nonconventional RNA splicing</span></div><div class="casAuthors">Lee, Kenneth P. K.; Dey, Madhusudan; Neculai, Dante; Cao, Chune; Dever, Thomas E.; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ire1 is an ancient transmembrane sensor of ER stress with dual protein kinase and RNase activities.  In response to ER stress, Ire1 catalyzes the splicing of target mRNAs in a spliceosome-independent manner.  We have detd. the crystal structure of the dual catalytic region of Ire1 at 2.4 Å resoln., revealing the fusion of a domain, which we term the KEN domain, to the protein kinase domain.  Dimerization of the kinase domain composes a large catalytic surface on the KEN domain which carries out RNase function.  We further show that signal induced trans-autophosphorylation of the kinase domain permits unfettered binding of nucleotide, which in turn promotes dimerization to compose the RNase active site.  Comparison of Ire1 to a topol. disparate RNase reveals the convergent evolution of their catalytic mechanism.  These findings provide a basis for understanding the mechanism of action of RNaseL and other pseudokinases, which represent 10% of the human kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPv617JGYBAbVg90H21EOLACvtfcHk0liIDRlEcvoKjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFehtbg%253D&md5=bc4f1a82988030a84b7cb4201437905d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.10.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.10.057%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%2BP.%26aulast%3DDey%26aufirst%3DM.%26aulast%3DNeculai%26aufirst%3DD.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DDever%26aufirst%3DT.%2BE.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DStructure%2520of%2520the%2520dual%2520enzyme%2520Ire1%2520reveals%2520the%2520basis%2520for%2520catalysis%2520and%2520regulation%2520in%2520nonconventional%2520RNA%2520splicing%26jtitle%3DCell%26date%3D2008%26volume%3D132%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.cell.2007.10.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korennykh, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egea, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korostelev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finer-Moore, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">The unfolded protein response signals through high-order assembly of Ire1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>457</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1038/nature07661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature07661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=19079236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=457&publication_year=2009&pages=687-693&author=A.+V.+Korennykhauthor=P.+F.+Egeaauthor=A.+A.+Korostelevauthor=J.+Finer-Mooreauthor=C.+Zhangauthor=K.+M.+Shokatauthor=R.+M.+Stroudauthor=P.+Walter&title=The+unfolded+protein+response+signals+through+high-order+assembly+of+Ire1&doi=10.1038%2Fnature07661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The unfolded protein response signals through high-order assembly of Ire1</span></div><div class="casAuthors">Korennykh, Alexei V.; Egea, Pascal F.; Korostelev, Andrei A.; Finer-Moore, Janet; Zhang, Chao; Shokat, Kevan M.; Stroud, Robert M.; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">457</span>
        (<span class="NLM_cas:issue">7230</span>),
    <span class="NLM_cas:pages">687-693</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aberrant folding of proteins in the endoplasmic reticulum activates the bifunctional transmembrane kinase/endoribonuclease Ire1.  Ire1 excises an intron from HAC1 mRNA in yeasts and Xbp1 mRNA in metazoans encoding homologous transcription factors.  This non-conventional mRNA splicing event initiates the unfolded protein response, a transcriptional program that relieves the endoplasmic reticulum stress.  Here we show that oligomerization is central to Ire1 function and is an intrinsic attribute of its cytosolic domains.  We obtained the 3.2-Å crystal structure of the oligomer of the Ire1 cytosolic domains in complex with a kinase inhibitor that acts as a potent activator of the Ire1 RNase.  The structure reveals a rod-shaped assembly that has no known precedence among kinases.  This assembly positions the kinase domain for trans-autophosphorylation, orders the RNase domain, and creates an interaction surface for binding of the mRNA substrate.  Activation of Ire1 through oligomerization expands the mechanistic repertoire of kinase-based signaling receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomyUZIZFiup7Vg90H21EOLACvtfcHk0liZKHOKD2dH2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ehtrw%253D&md5=51c74f5b87a912f4a9c1394a28357616</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature07661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07661%26sid%3Dliteratum%253Aachs%26aulast%3DKorennykh%26aufirst%3DA.%2BV.%26aulast%3DEgea%26aufirst%3DP.%2BF.%26aulast%3DKorostelev%26aufirst%3DA.%2BA.%26aulast%3DFiner-Moore%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DThe%2520unfolded%2520protein%2520response%2520signals%2520through%2520high-order%2520assembly%2520of%2520Ire1%26jtitle%3DNature%26date%3D2009%26volume%3D457%26spage%3D687%26epage%3D693%26doi%3D10.1038%2Fnature07661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagratuni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Santisteban, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrews, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aherne, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearl, L. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the Ire1 autophosphorylation complex and implications for the unfolded protein response</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">894</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1038/emboj.2011.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Femboj.2011.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21317875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=894-905&author=M.+M.+Aliauthor=T.+Bagratuniauthor=E.+L.+Davenportauthor=P.+R.+Nowakauthor=M.+C.+Silva-Santistebanauthor=A.+Hardcastleauthor=C.+McAndrewsauthor=M.+G.+Rowlandsauthor=G.+J.+Morganauthor=W.+Aherneauthor=I.+Collinsauthor=F.+E.+Daviesauthor=L.+H.+Pearl&title=Structure+of+the+Ire1+autophosphorylation+complex+and+implications+for+the+unfolded+protein+response&doi=10.1038%2Femboj.2011.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Ire1 autophosphorylation complex and implications for the unfolded protein response</span></div><div class="casAuthors">Ali, Maruf M. U.; Bagratuni, Tina; Davenport, Emma L.; Nowak, Piotr R.; Silva-Santisteban, M. Cris; Hardcastle, Anthea; McAndrews, Craig; Rowlands, Martin G.; Morgan, Gareth J.; Aherne, Wynne; Collins, Ian; Davies, Faith E.; Pearl, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">894-905</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ire1 (Ern1) is an unusual transmembrane protein kinase essential for the endoplasmic reticulum (ER) unfolded protein response (UPR).  Activation of Ire1 by assocn. of its N-terminal ER luminal domains promotes autophosphorylation by its cytoplasmic kinase domain, leading to activation of the C-terminal RNase domain, which splices Xbp1 mRNA, generating an active Xbp1s transcriptional activator.  We have detd. the crystal structure of the cytoplasmic portion of dephosphorylated human Ire1α bound to ADP, revealing the 'phosphoryl-transfer'-competent dimeric face-to-face complex, which precedes and is distinct from the back-to-back RNase 'active' conformation described for yeast Ire1.  We show that the Xbp1-specific RNase activity depends on autophosphorylation, and that ATP-competitive inhibitors staurosporin and sunitinib, which inhibit autophosphorylation in vitro, also inhibit Xbp1 splicing in vivo.  Furthermore, we demonstrate that activated Ire1α is a competent protein kinase, able to phosphorylate a heterologous peptide substrate.  These studies identify human Ire1α as a target for development of ATP-competitive inhibitors that will modulate the UPR in human cells, which has particular relevance for myeloma and other secretory malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQfUxv1Xx4JbVg90H21EOLACvtfcHk0liZKHOKD2dH2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtbw%253D&md5=b92aa98d4f38d5d46776e744ebe61980</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Femboj.2011.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2011.18%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DM.%2BM.%26aulast%3DBagratuni%26aufirst%3DT.%26aulast%3DDavenport%26aufirst%3DE.%2BL.%26aulast%3DNowak%26aufirst%3DP.%2BR.%26aulast%3DSilva-Santisteban%26aufirst%3DM.%2BC.%26aulast%3DHardcastle%26aufirst%3DA.%26aulast%3DMcAndrews%26aufirst%3DC.%26aulast%3DRowlands%26aufirst%3DM.%2BG.%26aulast%3DMorgan%26aufirst%3DG.%2BJ.%26aulast%3DAherne%26aufirst%3DW.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DDavies%26aufirst%3DF.%2BE.%26aulast%3DPearl%26aufirst%3DL.%2BH.%26atitle%3DStructure%2520of%2520the%2520Ire1%2520autophosphorylation%2520complex%2520and%2520implications%2520for%2520the%2520unfolded%2520protein%2520response%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D894%26epage%3D905%26doi%3D10.1038%2Femboj.2011.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of human IRE1 activation through dimerization and ligand binding</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">13019</span>– <span class="NLM_lpage">13035</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.18632%2Foncotarget.3864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=25968568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC2MfjsFSktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=13019-13035&author=A.+Joshiauthor=Y.+Newbattauthor=P.+C.+McAndrewauthor=M.+Stubbsauthor=R.+Burkeauthor=M.+W.+Richardsauthor=C.+Bhatiaauthor=J.+J.+Caldwellauthor=T.+McHardyauthor=I.+Collinsauthor=R.+Bayliss&title=Molecular+mechanisms+of+human+IRE1+activation+through+dimerization+and+ligand+binding&doi=10.18632%2Foncotarget.3864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms of human IRE1 activation through dimerization and ligand binding</span></div><div class="casAuthors">Joshi Amar; Richards Mark W; Bhatia Chitra; Bayliss Richard; Joshi Amar; Richards Mark W; Bhatia Chitra; Bayliss Richard; Newbatt Yvette; McAndrew P Craig; Stubbs Mark; Burke Rosemary; Caldwell John J; McHardy Tatiana; Collins Ian</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">13019-35</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">IRE1 transduces the unfolded protein response by splicing XBP1 through its C-terminal cytoplasmic kinase-RNase region.  IRE1 autophosphorylation is coupled to RNase activity through formation of a back-to-back dimer, although the conservation of the underlying molecular mechanism is not clear from existing structures.  We have crystallized human IRE1 in a back-to-back conformation only previously seen for the yeast homologue.  In our structure the kinase domain appears primed for catalysis but the RNase domains are disengaged.  Structure-function analysis reveals that IRE1 is autoinhibited through a Tyr-down mechanism related to that found in the unrelated Ser/Thr protein kinase Nek7.  We have developed a compound that potently inhibits human IRE1 kinase activity while stimulating XBP1 splicing.  A crystal structure of the inhibitor bound to IRE1 shows an increased ordering of the kinase activation loop.  The structures of hIRE in apo and ligand-bound forms are consistent with a previously proposed model of IRE1 regulation in which formation of a back-to-back dimer coupled to adoption of a kinase-active conformation drive RNase activation.  The structures provide opportunities for structure-guided design of IRE1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKXDQm87QVYFw2ofyr07jXfW6udTcc2ea8IgseYZZ45rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfjsFSktQ%253D%253D&md5=6005c96ebea157a65583d679acec5cbf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3864%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DA.%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DMcAndrew%26aufirst%3DP.%2BC.%26aulast%3DStubbs%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DBhatia%26aufirst%3DC.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DMcHardy%26aufirst%3DT.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DMolecular%2520mechanisms%2520of%2520human%2520IRE1%2520activation%2520through%2520dimerization%2520and%2520ligand%2520binding%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D13019%26epage%3D13035%26doi%3D10.18632%2Foncotarget.3864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, K.</span></span> <span> </span><span class="NLM_article-title">XBP1 mRNA is induced by ATF6 and spliced by IRE1 in response to ER stress to produce a highly active transcription factor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">881</span>– <span class="NLM_lpage">891</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(01)00611-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2FS0092-8674%2801%2900611-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=11779464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtlKlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2001&pages=881-891&author=H.+Yoshidaauthor=T.+Matsuiauthor=A.+Yamamotoauthor=T.+Okadaauthor=K.+Mori&title=XBP1+mRNA+is+induced+by+ATF6+and+spliced+by+IRE1+in+response+to+ER+stress+to+produce+a+highly+active+transcription+factor&doi=10.1016%2FS0092-8674%2801%2900611-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">XBP1 mRNA is induced by ATF6 and spliced by IRE1 in response to ER stress to produce a highly active transcription factor</span></div><div class="casAuthors">Yoshida, Hiderou; Matsui, Toshie; Yamamoto, Akira; Okada, Tetsuya; Mori, Kazutoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">881-891</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In yeast, the transmembrane protein kinase/endoribonuclease Ire1p activated by endoplasmic reticulum stress cleaves HAC1 mRNA, leading to prodn. of the transcription factor Hac1p that activates the unfolded protein response (UPR).  In mammals, no Hac1p counterpart has yet been discovered despite the presence of Ire1p homologs in the endoplasmic reticulum.  Instead, the transcription factor ATF6 specific to the mammalian UPR is regulated by intramembrane proteolysis.  Here, we identified the transcription factor XBP1, a target of ATF6, as a mammalian substrate of such an unconventional mRNA splicing system and showed that only the spliced form of XBP1 can activate the UPR efficiently.  Our results reveal features of the UPR conserved during evolution and clarify the relationship between IRE1- and ATF6-dependent pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo67siPWlzG7LVg90H21EOLACvtfcHk0lhtLvdd_YtkIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtlKlsg%253D%253D&md5=e873cf424078a4964d0335c2529775f2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900611-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900611-0%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DMatsui%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DK.%26atitle%3DXBP1%2520mRNA%2520is%2520induced%2520by%2520ATF6%2520and%2520spliced%2520by%2520IRE1%2520in%2520response%2520to%2520ER%2520stress%2520to%2520produce%2520a%2520highly%2520active%2520transcription%2520factor%26jtitle%3DCell%26date%3D2001%26volume%3D107%26spage%3D881%26epage%3D891%26doi%3D10.1016%2FS0092-8674%2801%2900611-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwakoshi, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span> <span> </span><span class="NLM_article-title">XBP-1 regulates a subset of endoplasmic reticulum resident chaperone genes in the unfolded protein response</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">7448</span>– <span class="NLM_lpage">7459</span>, <span class="refDoi"> DOI: 10.1128/MCB.23.21.7448-7459.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1128%2FMCB.23.21.7448-7459.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=14559994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosFOgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=7448-7459&author=A.+H.+Leeauthor=N.+N.+Iwakoshiauthor=L.+H.+Glimcher&title=XBP-1+regulates+a+subset+of+endoplasmic+reticulum+resident+chaperone+genes+in+the+unfolded+protein+response&doi=10.1128%2FMCB.23.21.7448-7459.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">XBP-1 regulates a subset of endoplasmic reticulum resident chaperone genes in the unfolded protein response</span></div><div class="casAuthors">Lee, Ann-Hwee; Iwakoshi, Neal N.; Glimcher, Laurie H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7448-7459</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mammalian unfolded protein response (UPR) protects the cell against the stress of misfolded proteins in the endoplasmic reticulum (ER).  We have investigated here the contribution of the UPR transcription factors XBP-1, ATF6α, and ATF6β to UPR target gene expression.  Gene profiling of cell lines lacking these factors yielded several XBP-1-dependent UPR target genes, all of which appear to act in the ER.  These included the DnaJ/Hsp40-like genes, p58IPK, ERdj4, and HEDJ, as well as EDEM, protein disulfide isomerase-P5, and ribosome-assocd. membrane protein 4 (RAMP4), whereas expression of BiP was only modestly dependent on XBP-1.  Surprisingly, given previous reports that enforced expression of ATF6α induced a subset of UPR target genes, cells deficient in ATF6α, ATF6β, or both had minimal defects in upregulating UPR target genes by gene profiling anal., suggesting the presence of compensatory mechanism(s) for ATF6 in the UPR.  Since cells lacking both XBP-1 and ATF6α had significantly impaired induction of select UPR target genes and ERSE reporter activation, XBP-1 and ATF6α may serve partially redundant functions.  No UPR target genes that required ATF6β were identified, nor, in contrast to XBP-1 and ATF6α, did the activity of the UPRE or ERSE promoters require ATF6β, suggesting a minor role for it during the UPR.  Collectively, these results suggest that the IRE1/XBP-1 pathway is required for efficient protein folding, maturation, and degrdn. in the ER and imply the existence of subsets of UPR target genes as defined by their dependence on XBP-1.  Further, our observations suggest the existence of addnl., as-yet-unknown, key regulators of the UPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9zXJxAEOnKbVg90H21EOLACvtfcHk0ljcOxqzw1EwCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosFOgsLY%253D&md5=952311f7921081765f45ae2ad02331c6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.21.7448-7459.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.21.7448-7459.2003%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BH.%26aulast%3DIwakoshi%26aufirst%3DN.%2BN.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26atitle%3DXBP-1%2520regulates%2520a%2520subset%2520of%2520endoplasmic%2520reticulum%2520resident%2520chaperone%2520genes%2520in%2520the%2520unfolded%2520protein%2520response%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D7448%26epage%3D7459%26doi%3D10.1128%2FMCB.23.21.7448-7459.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Decay of endoplasmic reticulum-localized mRNAs during the unfolded protein response</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1126/science.1129631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1126%2Fscience.1129631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=16825573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFWjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=104-107&author=J.+Hollienauthor=J.+S.+Weissman&title=Decay+of+endoplasmic+reticulum-localized+mRNAs+during+the+unfolded+protein+response&doi=10.1126%2Fscience.1129631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Decay of Endoplasmic Reticulum-Localized mRNAs During the Unfolded Protein Response</span></div><div class="casAuthors">Hollien, Julie; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5783</span>),
    <span class="NLM_cas:pages">104-107</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The unfolded protein response (UPR) allows the endoplasmic reticulum (ER) to recover from the accumulation of misfolded proteins, in part by increasing its folding capacity.  Inositol-requiring enzyme-1 (IRE1) promotes this remodeling by detecting misfolded ER proteins and activating a transcription factor, X-box-binding protein 1, through endonucleolytic cleavage of its mRNA.  Here, we report that IRE1 independently mediates the rapid degrdn. of a specific subset of mRNAs, based both on their localization to the ER membrane and on the amino acid sequence they encode.  This response is well suited to complement other UPR mechanisms because it could selectively halt prodn. of proteins that challenge the ER and clear the translocation and folding machinery for the subsequent remodeling process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIQKpC2dcLGbVg90H21EOLACvtfcHk0ljcOxqzw1EwCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFWjurc%253D&md5=7c9d3114078742b3cbe3457fb91643fd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.1129631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1129631%26sid%3Dliteratum%253Aachs%26aulast%3DHollien%26aufirst%3DJ.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DDecay%2520of%2520endoplasmic%2520reticulum-localized%2520mRNAs%2520during%2520the%2520unfolded%2520protein%2520response%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D104%26epage%3D107%26doi%3D10.1126%2Fscience.1129631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ron, D.</span></span> <span> </span><span class="NLM_article-title">Coupling of stress in the ER to activation of JNK protein kinases by transmembrane protein kinase IRE1</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1126/science.287.5453.664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1126%2Fscience.287.5453.664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10650002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVyhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=664-666&author=F.+Uranoauthor=X.+Wangauthor=A.+Bertolottiauthor=Y.+Zhangauthor=P.+Chungauthor=H.+P.+Hardingauthor=D.+Ron&title=Coupling+of+stress+in+the+ER+to+activation+of+JNK+protein+kinases+by+transmembrane+protein+kinase+IRE1&doi=10.1126%2Fscience.287.5453.664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Coupling of stress in the ER to activation of JNK protein kinases by transmembrane protein kinase IRE1</span></div><div class="casAuthors">Urano, Fumihiko; Wang, Xiaozhong; Bertolotti, Anne; Zhang, Yuhong; Chung, Peter; Harding, Heather P.; Ron, David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">5453</span>),
    <span class="NLM_cas:pages">664-666</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Malfolded proteins in the endoplasmic reticulum (ER) induce cellular stress and activate c-Jun amino-terminal kinases (JNKs or SAPKs).  Mammalian homologs of yeast IRE1, which activate chaperone genes in response to ER stress, also activated JNK, and IRE1α-/- fibroblasts were impaired in JNK activation by ER stress.  The cytoplasmic part of IRE1 bound TRAF2, an adaptor protein that couples plasma membrane receptors to JNK activation.  Dominant-neg. TRAF2 inhibited activation of JNK by IRE1.  Activation of JNK by endogenous signals initiated in the ER proceeds by a pathway similar to that initiated by cell surface receptors in response to extracellular signals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOjeif2GC19LVg90H21EOLACvtfcHk0ljcOxqzw1EwCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVyhuw%253D%253D&md5=e8bac4aee53f9359a6f20b93cb4e0185</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.287.5453.664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.287.5453.664%26sid%3Dliteratum%253Aachs%26aulast%3DUrano%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBertolotti%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DP.%26aulast%3DHarding%26aufirst%3DH.%2BP.%26aulast%3DRon%26aufirst%3DD.%26atitle%3DCoupling%2520of%2520stress%2520in%2520the%2520ER%2520to%2520activation%2520of%2520JNK%2520protein%2520kinases%2520by%2520transmembrane%2520protein%2520kinase%2520IRE1%26jtitle%3DScience%26date%3D2000%26volume%3D287%26spage%3D664%26epage%3D666%26doi%3D10.1126%2Fscience.287.5453.664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, R. J.</span></span> <span> </span><span class="NLM_article-title">Protein misfolding in the endoplasmic reticulum as a conduit to human disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature17041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature17041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=26791723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1OjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=326-335&author=M.+Wangauthor=R.+J.+Kaufman&title=Protein+misfolding+in+the+endoplasmic+reticulum+as+a+conduit+to+human+disease&doi=10.1038%2Fnature17041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein misfolding in the endoplasmic reticulum as a conduit to human disease</span></div><div class="casAuthors">Wang, Miao; Kaufman, Randal J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">326-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In eukaryotic cells, the endoplasmic reticulum is essential for the folding and trafficking of proteins that enter the secretory pathway.  Environmental insults or increased protein synthesis often lead to protein misfolding in the organelle, the accumulation of misfolded or unfolded proteins - known as endoplasmic reticulum stress - and the activation of the adaptive unfolded protein response to restore homeostasis.  If protein misfolding is not resolved, cells death occurs.  Endoplasmic reticulum stress and activation of the unfolded protein response help to det. cell fate and function.  Furthermore, endoplasmic reticulum stress contributes to the etiol. of many human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCscbs52xg07Vg90H21EOLACvtfcHk0lgaCQHsVaxk6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1OjurY%253D&md5=2b7470a4f2ef81181d5f2c1c5e629488</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature17041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17041%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DKaufman%26aufirst%3DR.%2BJ.%26atitle%3DProtein%2520misfolding%2520in%2520the%2520endoplasmic%2520reticulum%2520as%2520a%2520conduit%2520to%2520human%2520disease%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D326%26epage%3D335%26doi%3D10.1038%2Fnature17041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukhdeo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protopopova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkus, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protopopov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePinho, R. A.</span></span> <span> </span><span class="NLM_article-title">The differentiation and stress response factor XBP-1 drives multiple myeloma pathogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2007.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.ccr.2007.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=17418411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksFWjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=349-360&author=D.+R.+Carrascoauthor=K.+Sukhdeoauthor=M.+Protopopovaauthor=R.+Sinhaauthor=M.+Enosauthor=D.+E.+Carrascoauthor=M.+Zhengauthor=M.+Maniauthor=J.+Hendersonauthor=G.+S.+Pinkusauthor=N.+Munshiauthor=J.+Hornerauthor=E.+V.+Ivanovaauthor=A.+Protopopovauthor=K.+C.+Andersonauthor=G.+Tononauthor=R.+A.+DePinho&title=The+differentiation+and+stress+response+factor+XBP-1+drives+multiple+myeloma+pathogenesis&doi=10.1016%2Fj.ccr.2007.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The differentiation and stress response factor XBP-1 drives multiple myeloma pathogenesis</span></div><div class="casAuthors">Carrasco, Daniel R.; Sukhdeo, Kumar; Protopopova, Marina; Sinha, Raktim; Enos, Miriam; Carrasco, Daniel E.; Zheng, Mei; Mani, Mala; Henderson, Joel; Pinkus, Geraldine S.; Munshi, Nikhil; Horner, James; Ivanova, Elena V.; Protopopov, Alexei; Anderson, Kenneth C.; Tonon, Giovanni; DePinho, Ronald A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-360</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) evolves from a highly prevalent premalignant condition termed MGUS.  The factors underlying the malignant transformation of MGUS are unknown.  We report a MGUS/MM phenotype in transgenic mice with Eμ-directed expression of the XBP-1 spliced isoform (XBP-1s), a factor governing unfolded protein/ER stress response and plasma-cell development.  Eμ-XBP-1s elicited elevated serum Ig and skin alterations.  With age, Eμ-xbp-1s transgenics develop features diagnostic of human MM, including bone lytic lesions and subendothelial Ig deposition.  Furthermore, transcriptional profiles of Eμ-xbp-1s lymphoid and MM cells show aberrant expression of known human MM dysregulated genes.  The similarities of this model with the human disease, coupled with documented frequent XBP-1s overexpression in human MM, serve to implicate XBP-1s dysregulation in MM pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGRySDE6u1uLVg90H21EOLACvtfcHk0lgaCQHsVaxk6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksFWjsro%253D&md5=8daad15d72f9924c73be3251ac6877ab</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2007.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2007.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DCarrasco%26aufirst%3DD.%2BR.%26aulast%3DSukhdeo%26aufirst%3DK.%26aulast%3DProtopopova%26aufirst%3DM.%26aulast%3DSinha%26aufirst%3DR.%26aulast%3DEnos%26aufirst%3DM.%26aulast%3DCarrasco%26aufirst%3DD.%2BE.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DMani%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DJ.%26aulast%3DPinkus%26aufirst%3DG.%2BS.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DHorner%26aufirst%3DJ.%26aulast%3DIvanova%26aufirst%3DE.%2BV.%26aulast%3DProtopopov%26aufirst%3DA.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DTonon%26aufirst%3DG.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26atitle%3DThe%2520differentiation%2520and%2520stress%2520response%2520factor%2520XBP-1%2520drives%2520multiple%2520myeloma%2520pathogenesis%26jtitle%3DCancer%2520Cell%26date%3D2007%26volume%3D11%26spage%3D349%26epage%3D360%26doi%3D10.1016%2Fj.ccr.2007.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iliopoulos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatziapostolou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adoro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landis, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span> <span> </span><span class="NLM_article-title">XBP1 promotes triple-negative breast cancer by controlling the HIF1α pathway</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1038/nature13119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature13119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=24670641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsFOqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=103-107&author=X.+Chenauthor=D.+Iliopoulosauthor=Q.+Zhangauthor=Q.+Tangauthor=M.+B.+Greenblattauthor=M.+Hatziapostolouauthor=E.+Limauthor=W.+L.+Tamauthor=M.+Niauthor=Y.+Chenauthor=J.+Maiauthor=H.+Shenauthor=D.+Z.+Huauthor=S.+Adoroauthor=B.+Huauthor=M.+Songauthor=C.+Tanauthor=M.+D.+Landisauthor=M.+Ferrariauthor=S.+J.+Shinauthor=M.+Brownauthor=J.+C.+Changauthor=X.+S.+Liuauthor=L.+H.+Glimcher&title=XBP1+promotes+triple-negative+breast+cancer+by+controlling+the+HIF1%CE%B1+pathway&doi=10.1038%2Fnature13119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">XBP1 promotes triple-negative breast cancer by controlling the HIF1α pathway</span></div><div class="casAuthors">Chen, Xi; Iliopoulos, Dimitrios; Zhang, Qing; Tang, Qianzi; Greenblatt, Matthew B.; Hatziapostolou, Maria; Lim, Elgene; Tam, Wai Leong; Ni, Min; Chen, Yiwen; Mai, Junhua; Shen, Haifa; Hu, Dorothy Z.; Adoro, Stanley; Hu, Bella; Song, Minkyung; Tan, Chen; Landis, Melissa D.; Ferrari, Mauro; Shin, Sandra J.; Brown, Myles; Chang, Jenny C.; Liu, X. Shirley; Glimcher, Laurie H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7494</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells induce a set of adaptive response pathways to survive in the face of stressors due to inadequate vascularization.  One such adaptive pathway is the unfolded protein (UPR) or endoplasmic reticulum (ER) stress response mediated in part by the ER-localized transmembrane sensor IRE1 (ref. 2) and its substrate XBP1 (ref. 3).  Previous studies report UPR activation in various human tumors, but the role of XBP1 in cancer progression in mammary epithelial cells is largely unknown.  Triple-neg. breast cancer (TNBC)-a form of breast cancer in which tumor cells do not express the genes for estrogen receptor, progesterone receptor and HER2 (also called ERBB2 or NEU)-is a highly aggressive malignancy with limited treatment options.  Here we report that XBP1 is activated in TNBC and has a pivotal role in the tumorigenicity and progression of this human breast cancer subtype.  In breast cancer cell line models, depletion of XBP1 inhibited tumor growth and tumor relapse and reduced the CD44highCD24low population.  Hypoxia-inducing factor 1α (HIF1α) is known to be hyperactivated in TNBCs.  Genome-wide mapping of the XBP1 transcriptional regulatory network revealed that XBP1 drives TNBC tumorigenicity by assembling a transcriptional complex with HIF1α that regulates the expression of HIF1α targets via the recruitment of RNA polymerase II.  Anal. of independent cohorts of patients with TNBC revealed a specific XBP1 gene expression signature that was highly correlated with HIF1α and hypoxia-driven signatures and that strongly assocd. with poor prognosis.  Our findings reveal a key function for the XBP1 branch of the UPR in TNBC and indicate that targeting this pathway may offer alternative treatment strategies for this aggressive subtype of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFeZj_3EFRXbVg90H21EOLACvtfcHk0lg-QTaFqISwgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsFOqtLY%253D&md5=7f5b8d5b2699ce4e92c8a54e6b33091f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature13119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13119%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DIliopoulos%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTang%26aufirst%3DQ.%26aulast%3DGreenblatt%26aufirst%3DM.%2BB.%26aulast%3DHatziapostolou%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DE.%26aulast%3DTam%26aufirst%3DW.%2BL.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DMai%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DD.%2BZ.%26aulast%3DAdoro%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DLandis%26aufirst%3DM.%2BD.%26aulast%3DFerrari%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DS.%2BJ.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DJ.%2BC.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26atitle%3DXBP1%2520promotes%2520triple-negative%2520breast%2520cancer%2520by%2520controlling%2520the%2520HIF1%25CE%25B1%2520pathway%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D103%26epage%3D107%26doi%3D10.1038%2Fnature13119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Storm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnoldussen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saatcioglu, F.</span></span> <span> </span><span class="NLM_article-title">Prostate cancer and the unfolded protein response</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">54051</span>– <span class="NLM_lpage">54066</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.18632%2Foncotarget.9912" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=54051-54066&author=M.+Stormauthor=X.+Shengauthor=Y.+J.+Arnoldussenauthor=F.+Saatcioglu&title=Prostate+cancer+and+the+unfolded+protein+response&doi=10.18632%2Foncotarget.9912"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9912%26sid%3Dliteratum%253Aachs%26aulast%3DStorm%26aufirst%3DM.%26aulast%3DSheng%26aufirst%3DX.%26aulast%3DArnoldussen%26aufirst%3DY.%2BJ.%26aulast%3DSaatcioglu%26aufirst%3DF.%26atitle%3DProstate%2520cancer%2520and%2520the%2520unfolded%2520protein%2520response%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D54051%26epage%3D54066%26doi%3D10.18632%2Foncotarget.9912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cubillos-Ruiz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silberman, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perales-Puchalt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettigole, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellenson, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span> <span> </span><span class="NLM_article-title">ER stress sensor XBP1 controls anti-tumor immunity by disrupting dendritic cell homeostasis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">1527</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.05.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.cell.2015.05.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=26073941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVeiu77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2015&pages=1527-1538&author=J.+R.+Cubillos-Ruizauthor=P.+C.+Silbermanauthor=M.+R.+Rutkowskiauthor=S.+Chopraauthor=A.+Perales-Puchaltauthor=M.+Songauthor=S.+Zhangauthor=S.+E.+Bettigoleauthor=D.+Guptaauthor=K.+Holcombauthor=L.+H.+Ellensonauthor=T.+Caputoauthor=A.+H.+Leeauthor=J.+R.+Conejo-Garciaauthor=L.+H.+Glimcher&title=ER+stress+sensor+XBP1+controls+anti-tumor+immunity+by+disrupting+dendritic+cell+homeostasis&doi=10.1016%2Fj.cell.2015.05.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ER stress sensor XBP1 controls anti-tumor immunity by disrupting dendritic cell homeostasis</span></div><div class="casAuthors">Cubillos-Ruiz, Juan R.; Silberman, Pedro C.; Rutkowski, Melanie R.; Chopra, Sahil; Perales-Puchalt, Alfredo; Song, Minkyung; Zhang, Sheng; Bettigole, Sarah E.; Gupta, Divya; Holcomb, Kevin; Ellenson, Lora H.; Caputo, Thomas; Lee, Ann-Hwee; Conejo-Garcia, Jose R.; Glimcher, Laurie H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1527-1538</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Dendritic cells (DCs) are required to initiate and sustain T cell-dependent anti-cancer immunity.  However, tumors often evade immune control by crippling normal DC function.  The endoplasmic reticulum (ER) stress response factor XBP1 promotes intrinsic tumor growth directly, but whether it also regulates the host anti-tumor immune response is not known.  Here we show that constitutive activation of XBP1 in tumor-assocd. DCs (tDCs) drives ovarian cancer (OvCa) progression by blunting anti-tumor immunity.  XBP1 activation, fueled by lipid peroxidn. byproducts, induced a triglyceride biosynthetic program in tDCs leading to abnormal lipid accumulation and subsequent inhibition of tDC capacity to support anti-tumor T cells.  Accordingly, DC-specific XBP1 deletion or selective nanoparticle-mediated XBP1 silencing in tDCs restored their immunostimulatory activity in situ and extended survival by evoking protective type 1 anti-tumor responses.  Targeting the ER stress response should concomitantly inhibit tumor growth and enhance anti-cancer immunity, thus offering a unique approach to cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo12xOn4E_-B7Vg90H21EOLACvtfcHk0lg-QTaFqISwgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVeiu77J&md5=00e069a6bfd44b300645617cd534f121</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.05.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.05.025%26sid%3Dliteratum%253Aachs%26aulast%3DCubillos-Ruiz%26aufirst%3DJ.%2BR.%26aulast%3DSilberman%26aufirst%3DP.%2BC.%26aulast%3DRutkowski%26aufirst%3DM.%2BR.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DPerales-Puchalt%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DBettigole%26aufirst%3DS.%2BE.%26aulast%3DGupta%26aufirst%3DD.%26aulast%3DHolcomb%26aufirst%3DK.%26aulast%3DEllenson%26aufirst%3DL.%2BH.%26aulast%3DCaputo%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DA.%2BH.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26atitle%3DER%2520stress%2520sensor%2520XBP1%2520controls%2520anti-tumor%2520immunity%2520by%2520disrupting%2520dendritic%2520cell%2520homeostasis%26jtitle%3DCell%26date%3D2015%26volume%3D161%26spage%3D1527%26epage%3D1538%26doi%3D10.1016%2Fj.cell.2015.05.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volkmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuga, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brumm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriebel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der-Sarkissian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweitzer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malyankar, U. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiovitti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canny, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. B.</span></span> <span> </span><span class="NLM_article-title">Potent and selective inhibitors of the inositol-requiring enzyme 1 endoribonuclease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">12743</span>– <span class="NLM_lpage">12755</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.199737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1074%2Fjbc.M110.199737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21303903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=12743-12755&author=K.+Volkmannauthor=J.+L.+Lucasauthor=D.+Vugaauthor=X.+Wangauthor=D.+Brummauthor=C.+Stilesauthor=D.+Kriebelauthor=A.+Der-Sarkissianauthor=K.+Krishnanauthor=C.+Schweitzerauthor=Z.+Liuauthor=U.+M.+Malyankarauthor=D.+Chiovittiauthor=M.+Cannyauthor=D.+Durocherauthor=F.+Sicheriauthor=J.+B.+Patterson&title=Potent+and+selective+inhibitors+of+the+inositol-requiring+enzyme+1+endoribonuclease&doi=10.1074%2Fjbc.M110.199737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective Inhibitors of the Inositol-requiring Enzyme 1 Endoribonuclease</span></div><div class="casAuthors">Volkmann, Kori; Lucas, Julie L.; Vuga, Danka; Wang, Xiao-Ping; Brumm, Duane; Stiles, Caryn; Kriebel, David; Der-Sarkissian, Ani; Krishnan, Kris; Schweitzer, Colleen; Liu, Zheng; Malyankar, Uriel M.; Chiovitti, David; Canny, Marella; Durocher, Dan; Sicheri, Frank; Patterson, John B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">12743-12755</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Inositol-requiring enzyme 1 (IRE1) is the most highly conserved signaling node of the unfolded protein response (UPR) and represents a potential therapeutic target for a no. of diseases assocd. with endoplasmic reticulum stress.  IRE1 activates the XBP-1 transcription factor by site-specific cleavage of two hairpin loops within its mRNA to facilitate its nonconventional splicing and alternative translation.  We screened for inhibitors using a construct contg. the unique cytosolic kinase and endoribonuclease domains of human IRE1α (hIRE1α-cyto) and a mini-XBP-1 stem-loop RNA as the substrate.  One class compds. was salicylaldehyde analogs from the hydrolyzed product of salicylaldimines in the library.  Salicylaldehyde analogs were active in inhibiting the site-specific cleavage of several mini-XBP-1 stem-loop RNAs in a dose-dependent manner.  Salicyaldehyde analogs were also active in inhibiting yeast Ire1 but had little activity inhibiting RNase L or the unrelated RNases A and T1.  Kinetic anal. revealed that one potent salicylaldehyde analog, 3-ethoxy-5,6-dibromosalicylaldehyde, is a non-competitive inhibitor with respect to the XBP-1 RNA substrate.  Surface plasmon resonance studies confirmed this compd. bound to IRE1 in a specific, reversible and dose-dependent manner.  Salicylaldehydes inhibited XBP-1 splicing induced pharmacol. in human cells.  These compds. also blocked transcriptional up-regulation of known XBP-1 targets as well as mRNAs targeted for degrdn. by IRE1.  Finally, the salicylaldehyde analog 3-methoxy-6-bromosalicylaldehyde strongly inhibited XBP-1 splicing in an in vivo model of acute endoplasmic reticulum stress.  To our knowledge, salicylaldehyde analogs are the first reported specific IRE1 endoribonuclease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrguJWBkL4i1bVg90H21EOLACvtfcHk0lhSE9nke2hMQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOjsLg%253D&md5=e81641293ee0b2bd0a620d54d78241df</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.199737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.199737%26sid%3Dliteratum%253Aachs%26aulast%3DVolkmann%26aufirst%3DK.%26aulast%3DLucas%26aufirst%3DJ.%2BL.%26aulast%3DVuga%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBrumm%26aufirst%3DD.%26aulast%3DStiles%26aufirst%3DC.%26aulast%3DKriebel%26aufirst%3DD.%26aulast%3DDer-Sarkissian%26aufirst%3DA.%26aulast%3DKrishnan%26aufirst%3DK.%26aulast%3DSchweitzer%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DMalyankar%26aufirst%3DU.%2BM.%26aulast%3DChiovitti%26aufirst%3DD.%26aulast%3DCanny%26aufirst%3DM.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DPatterson%26aufirst%3DJ.%2BB.%26atitle%3DPotent%2520and%2520selective%2520inhibitors%2520of%2520the%2520inositol-requiring%2520enzyme%25201%2520endoribonuclease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D12743%26epage%3D12755%26doi%3D10.1074%2Fjbc.M110.199737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cross, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxendale, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ron, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, H. P.</span></span> <span> </span><span class="NLM_article-title">The molecular basis for selective inhibition of unconventional mRNA splicing by an IRE1-binding small molecule</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">E869</span>– <span class="NLM_lpage">E878</span>, <span class="refDoi"> DOI: 10.1073/pnas.1115623109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1073%2Fpnas.1115623109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=22315414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVCgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=E869-E878&author=B.+C.+Crossauthor=P.+J.+Bondauthor=P.+G.+Sadowskiauthor=B.+K.+Jhaauthor=J.+Zakauthor=J.+M.+Goodmanauthor=R.+H.+Silvermanauthor=T.+A.+Neubertauthor=I.+R.+Baxendaleauthor=D.+Ronauthor=H.+P.+Harding&title=The+molecular+basis+for+selective+inhibition+of+unconventional+mRNA+splicing+by+an+IRE1-binding+small+molecule&doi=10.1073%2Fpnas.1115623109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular basis for selective inhibition of unconventional mRNA splicing by an IRE1-binding small molecule</span></div><div class="casAuthors">Cross, Benedict C. S.; Bond, Peter J.; Sadowski, Pawel G.; Jha, Babal Kant; Zak, Jaroslav; Goodman, Jonathan M.; Silverman, Robert H.; Neubert, Thomas A.; Baxendale, Ian R.; Ron, David; Harding, Heather P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">E869-E878, SE869/1-SE869/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">IRE1 couples endoplasmic reticulum unfolded protein load to RNA cleavage events that culminate in the sequence-specific splicing of the Xbp1 mRNA and in the regulated degrdn. of diverse membrane-bound mRNAs.  We report on the identification of a small mol. inhibitor that attains its selectivity by forming an unusually stable Schiff base with lysine 907 in the IRE1 endonuclease domain, explained by solvent inaccessibility of the imine bond in the enzyme-inhibitor complex.  The inhibitor (abbreviated 4μ8C) blocks substrate access to the active site of IRE1 and selectively inactivates both Xbp1 splicing and IRE1-mediated mRNA degrdn.  Surprisingly, inhibition of IRE1 endonuclease activity does not sensitize cells to the consequences of acute endoplasmic reticulum stress, but rather interferes with the expansion of secretory capacity.  Thus, the chem. reactivity and sterics of a unique residue in the endonuclease active site of IRE1 can be exploited by selective inhibitors to interfere with protein secretion in pathol. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdJR15M_rc7rVg90H21EOLACvtfcHk0liIVl1KMezw4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVCgtb8%253D&md5=f6a05686d1b4a8a25d33fea41270f3ee</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1115623109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1115623109%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DB.%2BC.%26aulast%3DBond%26aufirst%3DP.%2BJ.%26aulast%3DSadowski%26aufirst%3DP.%2BG.%26aulast%3DJha%26aufirst%3DB.%2BK.%26aulast%3DZak%26aufirst%3DJ.%26aulast%3DGoodman%26aufirst%3DJ.%2BM.%26aulast%3DSilverman%26aufirst%3DR.%2BH.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DBaxendale%26aufirst%3DI.%2BR.%26aulast%3DRon%26aufirst%3DD.%26aulast%3DHarding%26aufirst%3DH.%2BP.%26atitle%3DThe%2520molecular%2520basis%2520for%2520selective%2520inhibition%2520of%2520unconventional%2520mRNA%2520splicing%2520by%2520an%2520IRE1-binding%2520small%2520molecule%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3DE869%26epage%3DE878%26doi%3D10.1073%2Fpnas.1115623109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ming, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span> <span> </span><span class="NLM_article-title">A novel chemical, STF-083010, reverses tamoxifen-related drug resistance in breast cancer by inhibiting IRE1/XBP1</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">40692</span>– <span class="NLM_lpage">40703</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.5827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.18632%2Foncotarget.5827" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=26517687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC28zmvFSrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=40692-40703&author=J.+Mingauthor=S.+Ruanauthor=M.+Wangauthor=D.+Yeauthor=N.+Fanauthor=Q.+Mengauthor=B.+Tianauthor=T.+Huang&title=A+novel+chemical%2C+STF-083010%2C+reverses+tamoxifen-related+drug+resistance+in+breast+cancer+by+inhibiting+IRE1%2FXBP1&doi=10.18632%2Foncotarget.5827"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A novel chemical, STF-083010, reverses tamoxifen-related drug resistance in breast cancer by inhibiting IRE1/XBP1</span></div><div class="casAuthors">Ming Jie; Ruan Shengnan; Wang Mengyi; Ye Dan; Huang Tao; Fan Ningning; Meng Qingyu; Tian Bo; Tian Bo</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">40692-703</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent studies show that the unfolded protein response (UPR) within the endoplasmic reticulum is correlated with breast cancer drug resistance.  In particular, human X-box binding protein-1(XBP1), a transcription factor which participates in UPR stress signaling, is reported to correlate with poor clinical responsiveness to tamoxifen.  In this study, we develop a tamoxifen-resistant MCF-7 cell line by treating the cell line with low concentration of tamoxifen, and we find that XBP1 is indeed up-regulated at both the mRNA and protein levels compared to normal MCF-7 cells.  STF-083010, a novel inhibitor which specifically blocks the XBP1 splicing, reestablishes tamoxifen sensitivity to resistant MCF-7 cells.  Moreover, co-treatment with STF-083010 and tamoxifen can significantly delay breast cancer progression in a xenograft mammary tumor model.  We next investigate the expression of XBP1s in over 170 breast cancer patients' samples and the results demonstrate that XBP1s expression level is highly correlated with overall survival in the ER+ subgroup, but not in the ER- subgroup, suggesting a potential therapeutic application of XBP1 inhibitors in ER+breast cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRKRBjVR00jFhfKb6ELK8t0fW6udTcc2eaEbgVPikoiBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zmvFSrsQ%253D%253D&md5=c7d700c77fbea029b419a0e702aeb48e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.5827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.5827%26sid%3Dliteratum%253Aachs%26aulast%3DMing%26aufirst%3DJ.%26aulast%3DRuan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DN.%26aulast%3DMeng%26aufirst%3DQ.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DT.%26atitle%3DA%2520novel%2520chemical%252C%2520STF-083010%252C%2520reverses%2520tamoxifen-related%2520drug%2520resistance%2520in%2520breast%2520cancer%2520by%2520inhibiting%2520IRE1%252FXBP1%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D40692%26epage%3D40703%26doi%3D10.18632%2Foncotarget.5827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papandreou, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denko, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Melckebeke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lust, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouley, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koong, A. C.</span></span> <span> </span><span class="NLM_article-title">Identification of an Ire1alpha endonuclease specific inhibitor with cytotoxic activity against human multiple myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1314</span>, <span class="refDoi"> DOI: 10.1182/blood-2010-08-303099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1182%2Fblood-2010-08-303099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21081713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFGks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2011&pages=1311-1314&author=I.+Papandreouauthor=N.+C.+Denkoauthor=M.+Olsonauthor=H.+Van+Melckebekeauthor=S.+Lustauthor=A.+Tamauthor=D.+E.+Solow-Corderoauthor=D.+M.+Bouleyauthor=F.+Offnerauthor=M.+Niwaauthor=A.+C.+Koong&title=Identification+of+an+Ire1alpha+endonuclease+specific+inhibitor+with+cytotoxic+activity+against+human+multiple+myeloma&doi=10.1182%2Fblood-2010-08-303099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an Ire1alpha endonuclease specific inhibitor with cytotoxic activity against human multiple myeloma</span></div><div class="casAuthors">Papandreou, Ioanna; Denko, Nicholas C.; Olson, Michael; Van Melckebeke, Heleen; Lust, Sofie; Tam, Arvin; Solow-Cordero, David E.; Bouley, Donna M.; Offner, Fritz; Niwa, Maho; Koong, Albert C.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1311-1314</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Activation of the adaptive Ire1-XBP1 pathway has been identified in many solid tumors and hematol. malignancies, including multiple myeloma (MM).  Here, we report the identification of STF-083010, a novel small-mol. inhibitor of Ire1.  STF-083010 inhibited Ire1 endonuclease activity, without affecting its kinase activity, after endoplasmic reticulum stress both in vitro and in vivo.  Treatment with STF-083010 showed significant antimyeloma activity in model human MM xenografts.  Similarly, STF-083010 was preferentially toxic to freshly isolated human CD138+ MM cells compared with other similarly isolated cell populations.  The identification of this novel Ire1 inhibitor supports the hypothesis that the Ire1-XBP1 axis is a promising target for anticancer therapy, esp. in the context of MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozlJFXgajWgLVg90H21EOLACvtfcHk0liWeTPIEQokHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFGks7c%253D&md5=a237ec467f838e4185df327ec9dfb969</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2010-08-303099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2010-08-303099%26sid%3Dliteratum%253Aachs%26aulast%3DPapandreou%26aufirst%3DI.%26aulast%3DDenko%26aufirst%3DN.%2BC.%26aulast%3DOlson%26aufirst%3DM.%26aulast%3DVan%2BMelckebeke%26aufirst%3DH.%26aulast%3DLust%26aufirst%3DS.%26aulast%3DTam%26aufirst%3DA.%26aulast%3DSolow-Cordero%26aufirst%3DD.%2BE.%26aulast%3DBouley%26aufirst%3DD.%2BM.%26aulast%3DOffner%26aufirst%3DF.%26aulast%3DNiwa%26aufirst%3DM.%26aulast%3DKoong%26aufirst%3DA.%2BC.%26atitle%3DIdentification%2520of%2520an%2520Ire1alpha%2520endonuclease%2520specific%2520inhibitor%2520with%2520cytotoxic%2520activity%2520against%2520human%2520multiple%2520myeloma%26jtitle%3DBlood%26date%3D2011%26volume%3D117%26spage%3D1311%26epage%3D1314%26doi%3D10.1182%2Fblood-2010-08-303099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrolecki, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talukdar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartig, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaipparettu, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putluri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Pharmacological targeting of MYC-regulated IRE1/XBP1 pathway suppresses MYC-driven breast cancer</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1283</span>– <span class="NLM_lpage">1299</span>, <span class="refDoi"> DOI: 10.1172/JCI95873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1172%2FJCI95873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=29480818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A280%3ADC%252BC1MrltlCkug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2018&pages=1283-1299&author=N.+Zhaoauthor=J.+Caoauthor=L.+Xuauthor=Q.+Tangauthor=L.+E.+Dobroleckiauthor=X.+Lvauthor=M.+Talukdarauthor=Y.+Luauthor=X.+Wangauthor=D.+Z.+Huauthor=Q.+Shiauthor=Y.+Xiangauthor=Y.+Wangauthor=X.+Liuauthor=W.+Buauthor=Y.+Jiangauthor=M.+Liauthor=Y.+Gongauthor=Z.+Sunauthor=H.+Yingauthor=B.+Yuanauthor=X.+Linauthor=X.+H.+Fengauthor=S.+M.+Hartigauthor=F.+Liauthor=H.+Shenauthor=Y.+Chenauthor=L.+Hanauthor=Q.+Zengauthor=J.+B.+Pattersonauthor=B.+A.+Kaipparettuauthor=N.+Putluriauthor=F.+Sicheriauthor=J.+M.+Rosenauthor=M.+T.+Lewisauthor=X.+Chen&title=Pharmacological+targeting+of+MYC-regulated+IRE1%2FXBP1+pathway+suppresses+MYC-driven+breast+cancer&doi=10.1172%2FJCI95873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of MYC-regulated IRE1/XBP1 pathway suppresses MYC-driven breast cancer</span></div><div class="casAuthors">Zhao Na; Cao Jin; Xu Longyong; Dobrolecki Lacey E; Lv Xiangdong; Lu Yang; Wang Xiaoran; Shi Qing; Liu Xia; Bu Wen; Gong Yingyun; Sun Zheng; Feng Xin-Hua; Hartig Sean M; Li Feng; Putluri Nagireddy; Rosen Jeffrey M; Lewis Michael T; Chen Xi; Zhao Na; Cao Jin; Xu Longyong; Dobrolecki Lacey E; Lv Xiangdong; Wang Xiaoran; Shi Qing; Liu Xia; Bu Wen; Lewis Michael T; Chen Xi; Zhao Na; Cao Jin; Xu Longyong; Dobrolecki Lacey E; Lv Xiangdong; Lu Yang; Wang Xiaoran; Shi Qing; Liu Xia; Kaipparettu Benny Abraham; Rosen Jeffrey M; Lewis Michael T; Chen Xi; Tang Qianzi; Li Mingzhou; Talukdar Manisha; Sicheri Frank; Talukdar Manisha; Sicheri Frank; Hu Dorothy Z; Xiang Yu; Han Leng; Wang Yunfei; Chen Yiwen; Jiang Yi; Gong Yingyun; Sun Zheng; Ying Haoqiang; Yuan Bo; Feng Xin-Hua; Lin Xia; Feng Xin-Hua; Shen Haifa; Zeng Qingping; Patterson John B; Sicheri Frank</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1283-1299</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The unfolded protein response (UPR) is a cellular homeostatic mechanism that is activated in many human cancers and plays pivotal roles in tumor progression and therapy resistance.  However, the molecular mechanisms for UPR activation and regulation in cancer cells remain elusive.  Here, we show that oncogenic MYC regulates the inositol-requiring enzyme 1 (IRE1)/X-box binding protein 1 (XBP1) branch of the UPR in breast cancer via multiple mechanisms.  We found that MYC directly controls IRE1 transcription by binding to its promoter and enhancer.  Furthermore, MYC forms a transcriptional complex with XBP1, a target of IRE1, and enhances its transcriptional activity.  Importantly, we demonstrate that XBP1 is a synthetic lethal partner of MYC.  Silencing of XBP1 selectively blocked the growth of MYC-hyperactivated cells.  Pharmacological inhibition of IRE1 RNase activity with small molecule inhibitor 8866 selectively restrained the MYC-overexpressing tumor growth in vivo in a cohort of preclinical patient-derived xenograft models and genetically engineered mouse models.  Strikingly, 8866 substantially enhanced the efficacy of docetaxel chemotherapy, resulting in rapid regression of MYC-overexpressing tumors.  Collectively, these data establish the synthetic lethal interaction of the IRE1/XBP1 pathway with MYC hyperactivation and provide a potential therapy for MYC-driven human breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSTs0p5pZIKXSjiVMCqkKxVfW6udTcc2eaZ4NGb4wjkM7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrltlCkug%253D%253D&md5=a401b8d714c025a1ca3e6b7eecf994fc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1172%2FJCI95873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI95873%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DN.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DQ.%26aulast%3DDobrolecki%26aufirst%3DL.%2BE.%26aulast%3DLv%26aufirst%3DX.%26aulast%3DTalukdar%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DD.%2BZ.%26aulast%3DShi%26aufirst%3DQ.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DBu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DX.%2BH.%26aulast%3DHartig%26aufirst%3DS.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DPatterson%26aufirst%3DJ.%2BB.%26aulast%3DKaipparettu%26aufirst%3DB.%2BA.%26aulast%3DPutluri%26aufirst%3DN.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DRosen%26aufirst%3DJ.%2BM.%26aulast%3DLewis%26aufirst%3DM.%2BT.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DPharmacological%2520targeting%2520of%2520MYC-regulated%2520IRE1%252FXBP1%2520pathway%2520suppresses%2520MYC-driven%2520breast%2520cancer%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2018%26volume%3D128%26spage%3D1283%26epage%3D1299%26doi%3D10.1172%2FJCI95873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hari, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhhatarai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backes, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schürer, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oakes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span> <span> </span><span class="NLM_article-title">Divergent allosteric control of the IRE1α endoribonuclease using kinase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">982</span>– <span class="NLM_lpage">989</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnchembio.1094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=23086298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=982-989&author=L.+Wangauthor=B.+G.+Pereraauthor=S.+B.+Hariauthor=B.+Bhhataraiauthor=B.+J.+Backesauthor=M.+A.+Seeligerauthor=S.+C.+Sch%C3%BCrerauthor=S.+A.+Oakesauthor=F.+R.+Papaauthor=D.+J.+Maly&title=Divergent+allosteric+control+of+the+IRE1%CE%B1+endoribonuclease+using+kinase+inhibitors&doi=10.1038%2Fnchembio.1094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent allosteric control of the IRE1α endoribonuclease using kinase inhibitors</span></div><div class="casAuthors">Wang, Likun; Perera, B. Gayani K.; Hari, Sanjay B.; Bhhatarai, Barun; Backes, Bradley J.; Seeliger, Markus A.; Schuerer, Stephan C.; Oakes, Scott A.; Papa, Feroz R.; Maly, Dustin J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">982-989</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Under endoplasmic reticulum stress, unfolded protein accumulation leads to activation of the endoplasmic reticulum transmembrane kinase/endoRNase IRE1α.  IRE1α oligomerizes, autophosphorylates and initiates splicing of XBP1 mRNA, thus triggering the unfolded protein response (UPR).  Here we show that IRE1α's kinase-controlled RNase can be regulated in two distinct modes with kinase inhibitors: one class of ligands occupies IRE1α's kinase ATP-binding site to activate RNase-mediated XBP1 mRNA splicing even without upstream endoplasmic reticulum stress, whereas a second class can inhibit the RNase through the same ATP-binding site, even under endoplasmic reticulum stress.  Thus, alternative kinase conformations stabilized by distinct classes of ATP-competitive inhibitors can cause allosteric switching of IRE1α's RNase-either on or off.  As dysregulation of the UPR has been implicated in a variety of cell degenerative and neoplastic disorders, small-mol. control over IRE1α should advance efforts to understand the UPR's role in pathophysiol. and to develop drugs for endoplasmic reticulum stress-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5edxS7gfwNrVg90H21EOLACvtfcHk0lgZCeSi8dgnGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmtLbO&md5=a6daf1ddb2ccdec77035e2e3c70d674c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1094%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPerera%26aufirst%3DB.%2BG.%26aulast%3DHari%26aufirst%3DS.%2BB.%26aulast%3DBhhatarai%26aufirst%3DB.%26aulast%3DBackes%26aufirst%3DB.%2BJ.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26aulast%3DOakes%26aufirst%3DS.%2BA.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26atitle%3DDivergent%2520allosteric%2520control%2520of%2520the%2520IRE1%25CE%25B1%2520endoribonuclease%2520using%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D982%26epage%3D989%26doi%3D10.1038%2Fnchembio.1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallwood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhry, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuster, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerding, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeYoung, M. P.</span></span> <span> </span><span class="NLM_article-title">Long-range inhibitor-induced conformational regulation of human IRE1α endoribonuclease activity</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">1011</span>– <span class="NLM_lpage">1023</span>, <span class="refDoi"> DOI: 10.1124/mol.115.100917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1124%2Fmol.115.100917" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=1011-1023&author=N.+O.+Conchaauthor=A.+Smallwoodauthor=W.+Bonnetteauthor=R.+Totoritisauthor=G.+Zhangauthor=K.+Federowiczauthor=J.+Yangauthor=H.+Qiauthor=S.+Chenauthor=N.+Campobassoauthor=A.+E.+Choudhryauthor=L.+E.+Shusterauthor=K.+A.+Evansauthor=J.+Ralphauthor=S.+Sweitzerauthor=D.+A.+Heerdingauthor=C.+A.+Buserauthor=D.+S.+Suauthor=M.+P.+DeYoung&title=Long-range+inhibitor-induced+conformational+regulation+of+human+IRE1%CE%B1+endoribonuclease+activity&doi=10.1124%2Fmol.115.100917"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fmol.115.100917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.115.100917%26sid%3Dliteratum%253Aachs%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DSmallwood%26aufirst%3DA.%26aulast%3DBonnette%26aufirst%3DW.%26aulast%3DTotoritis%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DFederowicz%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DChoudhry%26aufirst%3DA.%2BE.%26aulast%3DShuster%26aufirst%3DL.%2BE.%26aulast%3DEvans%26aufirst%3DK.%2BA.%26aulast%3DRalph%26aufirst%3DJ.%26aulast%3DSweitzer%26aufirst%3DS.%26aulast%3DHeerding%26aufirst%3DD.%2BA.%26aulast%3DBuser%26aufirst%3DC.%2BA.%26aulast%3DSu%26aufirst%3DD.%2BS.%26aulast%3DDeYoung%26aufirst%3DM.%2BP.%26atitle%3DLong-range%2520inhibitor-induced%2520conformational%2520regulation%2520of%2520human%2520IRE1%25CE%25B1%2520endoribonuclease%2520activity%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D88%26spage%3D1011%26epage%3D1023%26doi%3D10.1124%2Fmol.115.100917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malwitz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasker, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellamaggiore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeckel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Materna-Reichelt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span> <span> </span><span class="NLM_article-title">Unfolded protein response in cancer: IRE1α inhibition by selective kinase ligands does not impair tumor cell viability</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1021/ml500315b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500315b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wksr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=68-72&author=P.+E.+Harringtonauthor=K.+Biswasauthor=D.+Malwitzauthor=A.+S.+Taskerauthor=C.+Mohrauthor=K.+L.+Andrewsauthor=K.+Dellamaggioreauthor=R.+Kendallauthor=H.+Beckmannauthor=P.+Jaeckelauthor=S.+Materna-Reicheltauthor=J.+R.+Allenauthor=J.+R.+Lipford&title=Unfolded+protein+response+in+cancer%3A+IRE1%CE%B1+inhibition+by+selective+kinase+ligands+does+not+impair+tumor+cell+viability&doi=10.1021%2Fml500315b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Unfolded Protein Response in Cancer: IRE1α Inhibition by Selective Kinase Ligands Does Not Impair Tumor Cell Viability</span></div><div class="casAuthors">Harrington, Paul E.; Biswas, Kaustav; Malwitz, David; Tasker, Andrew S.; Mohr, Christopher; Andrews, Kristin L.; Dellamaggiore, Ken; Kendall, Richard; Beckmann, Holger; Jaeckel, Peter; Materna-Reichelt, Silvia; Allen, Jennifer R.; Lipford, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-72</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The kinase/endonuclease inositol requiring enzyme 1 (IRE1α), one of the sensors of unfolded protein accumulation in the endoplasmic reticulum that triggers the unfolded protein response (UPR), has been investigated as an anticancer target.  We identified potent allosteric inhibitors of IRE1α endonuclease activity that bound to the kinase site on the enzyme.  Structure-activity relationship (SAR) studies led to 16 and 18, which were selective in kinase screens and were potent against recombinant IRE1α endonuclease as well as cellular IRE1α.  The first X-ray crystal structure of a kinase inhibitor (16) bound to hIRE1α was obtained.  Screening of native tumor cell lines (>300) against selective IRE1α inhibitors failed to demonstrate any effect on cellular viability.  These results suggest that IRE1α activity is not essential for viability in most tumor cell lines, in vitro, and that interfering with the survival functions of the UPR may not be an effective strategy to block tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1dx6SPyC8t7Vg90H21EOLACvtfcHk0lhpb59jNLrMFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wksr7J&md5=58812b61336303299493b8a3e204fa66</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml500315b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500315b%26sid%3Dliteratum%253Aachs%26aulast%3DHarrington%26aufirst%3DP.%2BE.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DMalwitz%26aufirst%3DD.%26aulast%3DTasker%26aufirst%3DA.%2BS.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DDellamaggiore%26aufirst%3DK.%26aulast%3DKendall%26aufirst%3DR.%26aulast%3DBeckmann%26aufirst%3DH.%26aulast%3DJaeckel%26aufirst%3DP.%26aulast%3DMaterna-Reichelt%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DJ.%2BR.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26atitle%3DUnfolded%2520protein%2520response%2520in%2520cancer%253A%2520IRE1%25CE%25B1%2520inhibition%2520by%2520selective%2520kinase%2520ligands%2520does%2520not%2520impair%2520tumor%2520cell%2520viability%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D68%26epage%3D72%26doi%3D10.1021%2Fml500315b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meza-Acevedo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobillot, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gliedt, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hari, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backes, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span> <span> </span><span class="NLM_article-title">Structural and functional analysis of the allosteric inhibition of IRE1α with ATP-competitive ligands</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2195</span>– <span class="NLM_lpage">2205</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFyksbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2195-2205&author=H.+C.+Feldmanauthor=M.+Tongauthor=L.+Wangauthor=R.+Meza-Acevedoauthor=T.+A.+Gobillotauthor=I.+Lebedevauthor=M.+J.+Gliedtauthor=S.+B.+Hariauthor=A.+K.+Mitraauthor=B.+J.+Backesauthor=F.+R.+Papaauthor=M.+A.+Seeligerauthor=D.+J.+Maly&title=Structural+and+functional+analysis+of+the+allosteric+inhibition+of+IRE1%CE%B1+with+ATP-competitive+ligands&doi=10.1021%2Facschembio.5b00940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional analysis of the allosteric inhibition of IRE1α with ATP-competitive ligands</span></div><div class="casAuthors">Feldman, Hannah C.; Tong, Michael; Wang, Likun; Meza-Acevedo, Rosa; Gobillot, Theodore A.; Lebedev, Ivan; Gliedt, Micah J.; Hari, Sanjay B.; Mitra, Arinjay K.; Backes, Bradley J.; Papa, Feroz R.; Seeliger, Markus A.; Maly, Dustin J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2195-2205</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accumulation of unfolded proteins under endoplasmic reticulum (ER) stress leads to the activation of the multidomain protein sensor, inositol-requiring enzyme 1α (IRE1α) as part of the unfolded protein response (UPR).  The clustering of IRE1α lumenal domains in the presence of unfolded proteins promotes kinase trans-autophosphorylation in the cytosol and subsequent RNase domain activation.  Interestingly, there is an allosteric relation between the kinase and RNase domains of IRE1α, which allows ATP-competitive inhibitors to modulate the activity of the RNase domain.  Here, the authors use kinase inhibitors to study how ATP-binding site conformation affects the activity of the RNase domain of IRE1α.  The authors found that diverse ATP-competitive inhibitors of IRE1α promoted dimerization and activation of RNase activity despite blocking kinase autophosphorylation.  In contrast, a subset of ATP-competitive ligands, which the authors call kinase-inhibiting RNase attenuators (KIRAs), allosterically inactivate the RNase domain through the kinase domain by stabilizing monomeric IRE1α.  Further insight into how ATP-competitive inhibitors are able to divergently modulate the RNase domain through the kinase domain was gained by obtaining the 1st structure of apo human IRE1α in the RNase active back-to-back dimer conformation.  Comparison of this structure with other existing structures of IRE1α and integration of the extensive structure-activity relation (SAR) data led the authors to formulate a model to rationalize how ATP-binding site ligands are able to control the IRE1α oligomeric state and subsequent RNase domain activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCwB7RD3BZDbVg90H21EOLACvtfcHk0lhpb59jNLrMFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFyksbo%253D&md5=c97eb1c65a217ce17093089d8f9fd6ee</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00940%26sid%3Dliteratum%253Aachs%26aulast%3DFeldman%26aufirst%3DH.%2BC.%26aulast%3DTong%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeza-Acevedo%26aufirst%3DR.%26aulast%3DGobillot%26aufirst%3DT.%2BA.%26aulast%3DLebedev%26aufirst%3DI.%26aulast%3DGliedt%26aufirst%3DM.%2BJ.%26aulast%3DHari%26aufirst%3DS.%2BB.%26aulast%3DMitra%26aufirst%3DA.%2BK.%26aulast%3DBackes%26aufirst%3DB.%2BJ.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26atitle%3DStructural%2520and%2520functional%2520analysis%2520of%2520the%2520allosteric%2520inhibition%2520of%2520IRE1%25CE%25B1%2520with%2520ATP-competitive%2520ligands%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2195%26epage%3D2205%26doi%3D10.1021%2Facschembio.5b00940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann-Petersen, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehdizadeh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colon-Negron, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meza-Acevedo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backes, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluestone, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papa, F. R.</span></span> <span> </span><span class="NLM_article-title">Targeting ABL-IRE1α signaling spares ER-stressed pancreatic β cells to reverse autoimmune diabetes</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2017.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.cmet.2017.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28380378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1ylsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=883-897&author=S.+Moritaauthor=S.+A.+Villaltaauthor=H.+C.+Feldmanauthor=A.+C.+Registerauthor=W.+Rosenthalauthor=I.+T.+Hoffmann-Petersenauthor=M.+Mehdizadehauthor=R.+Ghoshauthor=L.+Wangauthor=K.+Colon-Negronauthor=R.+Meza-Acevedoauthor=B.+J.+Backesauthor=D.+J.+Malyauthor=J.+A.+Bluestoneauthor=F.+R.+Papa&title=Targeting+ABL-IRE1%CE%B1+signaling+spares+ER-stressed+pancreatic+%CE%B2+cells+to+reverse+autoimmune+diabetes&doi=10.1016%2Fj.cmet.2017.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting ABL-IRE1α Signaling Spares ER-Stressed Pancreatic β Cells to Reverse Autoimmune Diabetes</span></div><div class="casAuthors">Morita, Shuhei; Villalta, S. Armando; Feldman, Hannah C.; Register, Ames C.; Rosenthal, Wendy; Hoffmann-Petersen, Ingeborg T.; Mehdizadeh, Morvarid; Ghosh, Rajarshi; Wang, Likun; Colon-Negron, Kevin; Meza-Acevedo, Rosa; Backes, Bradley J.; Maly, Dustin J.; Bluestone, Jeffrey A.; Papa, Feroz R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">883-897.e8</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In cells experiencing unrelieved endoplasmic reticulum (ER) stress, the ER transmembrane kinase/endoribonuclease (RNase)-IRE1α-endonucleolytically degrades ER-localized mRNAs to promote apoptosis.  Here we find that the ABL family of tyrosine kinases rheostatically enhances IRE1α's enzymic activities, thereby potentiating ER stress-induced apoptosis.  During ER stress, cytosolic ABL kinases localize to the ER membrane, where they bind, scaffold, and hyperactivate IRE1α's RNase.  Imatinib-an anti-cancer tyrosine kinase inhibitor-antagonizes the ABL-IRE1α interaction, blunts IRE1α RNase hyperactivity, reduces pancreatic β cell apoptosis, and reverses type 1 diabetes (T1D) in the non-obese diabetic (NOD) mouse model.  A mono-selective kinase inhibitor that allosterically attenuates IRE1α's RNase-KIRA8-also efficaciously reverses established diabetes in NOD mice by sparing β cells and preserving their physiol. function.  Our data support a model wherein ER-stressed β cells contribute to their own demise during T1D pathogenesis and implicate the ABL-IRE1α axis as a drug target for the treatment of an autoimmune disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGZtS35KHrIrVg90H21EOLACvtfcHk0liGUhPVIUdXIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1ylsbg%253D&md5=be7692b3c222ac9eb9b1d1ca68a77e47</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2017.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2017.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DMorita%26aufirst%3DS.%26aulast%3DVillalta%26aufirst%3DS.%2BA.%26aulast%3DFeldman%26aufirst%3DH.%2BC.%26aulast%3DRegister%26aufirst%3DA.%2BC.%26aulast%3DRosenthal%26aufirst%3DW.%26aulast%3DHoffmann-Petersen%26aufirst%3DI.%2BT.%26aulast%3DMehdizadeh%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DColon-Negron%26aufirst%3DK.%26aulast%3DMeza-Acevedo%26aufirst%3DR.%26aulast%3DBackes%26aufirst%3DB.%2BJ.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26aulast%3DBluestone%26aufirst%3DJ.%2BA.%26aulast%3DPapa%26aufirst%3DF.%2BR.%26atitle%3DTargeting%2520ABL-IRE1%25CE%25B1%2520signaling%2520spares%2520ER-stressed%2520pancreatic%2520%25CE%25B2%2520cells%2520to%2520reverse%2520autoimmune%2520diabetes%26jtitle%3DCell%2520Metab.%26date%3D2017%26volume%3D25%26spage%3D883%26epage%3D897%26doi%3D10.1016%2Fj.cmet.2017.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Dodd, D.
S.</span>; <span class="NLM_string-name">Huynh, T.
N.</span>; <span class="NLM_string-name">Lin, J.</span>; <span class="NLM_string-name">Liu, C.</span>; <span class="NLM_string-name">Mussari, C. P.</span>; <span class="NLM_string-name">Tokarski, J. S.</span>; <span class="NLM_string-name">Tortolani, D. R.</span>; <span class="NLM_string-name">Wrobleski, S. T.</span>; <span class="NLM_string-name">Lin, S.</span></span> <span> </span><span class="NLM_article-title">Preparation of Fused Heterocyclic Compounds Useful as Kinase Modulators</span>. <span class="NLM_patent">US20080045536</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=W.+Vaccaro&author=Z.+Chen&author=D.%0AS.+Dodd&author=T.%0AN.+Huynh&author=J.+Lin&author=C.+Liu&author=C.+P.+Mussari&author=J.+S.+Tokarski&author=D.+R.+Tortolani&author=S.+T.+Wrobleski&author=S.+Lin&title=Preparation+of+Fused+Heterocyclic+Compounds+Useful+as+Kinase+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVaccaro%26aufirst%3DW.%26atitle%3DPreparation%2520of%2520Fused%2520Heterocyclic%2520Compounds%2520Useful%2520as%2520Kinase%2520Modulators%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. F.</span>; <span class="NLM_string-name">Ba-Maung, N. Y.</span>; <span class="NLM_string-name">Erickson, S. A.</span>; <span class="NLM_string-name">Fidanze, S. D.</span>; <span class="NLM_string-name">Mantei, R. A.</span>; <span class="NLM_string-name">Sheppard, G. S.</span>; <span class="NLM_string-name">Sorensen, B. K.</span>; <span class="NLM_string-name">Wang, G. T.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Bell, R.
L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyrimidine Inhibitors of Kinase Activity</span>. International Application No. <span class="NLM_patent">WO2010138575</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=R.+F.+Clark&author=N.+Y.+Ba-Maung&author=S.+A.+Erickson&author=S.+D.+Fidanze&author=R.+A.+Mantei&author=G.+S.+Sheppard&author=B.+K.+Sorensen&author=G.+T.+Wang&author=J.+Wang&author=R.%0AL.+Bell&title=Preparation+of+Pyrimidine+Inhibitors+of+Kinase+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DR.%2BF.%26atitle%3DPreparation%2520of%2520Pyrimidine%2520Inhibitors%2520of%2520Kinase%2520Activity%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kusakabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ide, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daigo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashizume, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tagashira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iguchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanazawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishioka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dohi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kido, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of imidazo[1,2-b]pyridazine derivatives: selective and orally available Mps1 (TTK) kinase inhibitors exhibiting remarkable antiproliferative activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1760</span>– <span class="NLM_lpage">1775</span>, <span class="refDoi"> DOI: 10.1021/jm501599u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501599u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2it7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1760-1775&author=K.+Kusakabeauthor=N.+Ideauthor=Y.+Daigoauthor=T.+Itohauthor=T.+Yamamotoauthor=H.+Hashizumeauthor=K.+Nozuauthor=H.+Yoshidaauthor=G.+Tadanoauthor=S.+Tagashiraauthor=K.+Higashinoauthor=Y.+Okanoauthor=Y.+Satoauthor=M.+Inoueauthor=M.+Iguchiauthor=T.+Kanazawaauthor=Y.+Ishiokaauthor=K.+Dohiauthor=Y.+Kidoauthor=S.+Sakamotoauthor=S.+Andoauthor=M.+Maedaauthor=M.+Higakiauthor=Y.+Babaauthor=Y.+Nakamura&title=Discovery+of+imidazo%5B1%2C2-b%5Dpyridazine+derivatives%3A+selective+and+orally+available+Mps1+%28TTK%29+kinase+inhibitors+exhibiting+remarkable+antiproliferative+activity&doi=10.1021%2Fjm501599u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Imidazo[1,2-b]pyridazine Derivatives: Selective and Orally Available Mps1 (TTK) Kinase Inhibitors Exhibiting Remarkable Antiproliferative Activity</span></div><div class="casAuthors">Kusakabe, Ken-ichi; Ide, Nobuyuki; Daigo, Yataro; Itoh, Takeshi; Yamamoto, Takahiko; Hashizume, Hiroshi; Nozu, Kohei; Yoshida, Hiroshi; Tadano, Genta; Tagashira, Sachie; Higashino, Kenichi; Okano, Yousuke; Sato, Yuji; Inoue, Makiko; Iguchi, Motofumi; Kanazawa, Takayuki; Ishioka, Yukichi; Dohi, Keiji; Kido, Yasuto; Sakamoto, Shingo; Ando, Shigeru; Maeda, Masahiro; Higaki, Masayo; Baba, Yoshiyasu; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1760-1775</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monopolar spindle 1 (Mps1) is an attractive oncol. target due to its high expression level in cancer cells as well as the correlation of its expression levels with histol. grades of cancers.  An imidazo[1,2-a]pyrazine 10a was identified during an HTS campaign.  Although 10a exhibited good biochem. activity, its moderate cellular as well as antiproliferative activities needed to be improved.  The cocrystal structure of an analog of 10a guided our lead optimization to introduce substituents at the 6-position of the scaffold, giving the 6-aryl substituted 21b which had improved cellular activity but no oral bioavailability in rat.  Property-based optimization at the 6-position and a scaffold change led to the discovery of the imidazo[1,2-b]pyridazine-based 27f, an extremely potent (cellular Mps1 IC50 = 0.70 nM, A549 IC50 = 6.0 nM), selective Mps1 inhibitor over 192 kinases, which could be orally administered and was active in vivo.  This 27f demonstrated remarkable antiproliferative activity in the nanomolar range against various tissue cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8D9nRl5vWabVg90H21EOLACvtfcHk0ljpG1-yhTTFug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2it7g%253D&md5=bee62a50247a363dc3e35fcdeac6a64f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm501599u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501599u%26sid%3Dliteratum%253Aachs%26aulast%3DKusakabe%26aufirst%3DK.%26aulast%3DIde%26aufirst%3DN.%26aulast%3DDaigo%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DHashizume%26aufirst%3DH.%26aulast%3DNozu%26aufirst%3DK.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DTadano%26aufirst%3DG.%26aulast%3DTagashira%26aufirst%3DS.%26aulast%3DHigashino%26aufirst%3DK.%26aulast%3DOkano%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DM.%26aulast%3DIguchi%26aufirst%3DM.%26aulast%3DKanazawa%26aufirst%3DT.%26aulast%3DIshioka%26aufirst%3DY.%26aulast%3DDohi%26aufirst%3DK.%26aulast%3DKido%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DS.%26aulast%3DAndo%26aufirst%3DS.%26aulast%3DMaeda%26aufirst%3DM.%26aulast%3DHigaki%26aufirst%3DM.%26aulast%3DBaba%26aufirst%3DY.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520imidazo%255B1%252C2-b%255Dpyridazine%2520derivatives%253A%2520selective%2520and%2520orally%2520available%2520Mps1%2520%2528TTK%2529%2520kinase%2520inhibitors%2520exhibiting%2520remarkable%2520antiproliferative%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1760%26epage%3D1775%26doi%3D10.1021%2Fjm501599u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uitdehaag, J. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Man, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willemsen-Seegers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinsen, M. B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libouban, M. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterrenburg, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wit, J. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Vetter, J. R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Roos, J. A. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buijsman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, G. J. R.</span></span> <span> </span><span class="NLM_article-title">Target residence time-guided optimization on TTK kinase results in inhibitors with potent anti-proliferative activity</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2230</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.jmb.2017.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28539250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Wiuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2211-2230&author=J.+C.+M.+Uitdehaagauthor=J.+de+Manauthor=N.+Willemsen-Seegersauthor=M.+B.+W.+Prinsenauthor=M.+A.+A.+Liboubanauthor=J.+G.+Sterrenburgauthor=J.+J.+P.+de+Witauthor=J.+R.+F.+de%0AVetterauthor=J.+A.+D.+M.+de+Roosauthor=R.+C.+Buijsmanauthor=G.+J.+R.+Zaman&title=Target+residence+time-guided+optimization+on+TTK+kinase+results+in+inhibitors+with+potent+anti-proliferative+activity&doi=10.1016%2Fj.jmb.2017.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Target Residence Time-Guided Optimization on TTK Kinase Results in Inhibitors with Potent Anti-Proliferative Activity</span></div><div class="casAuthors">Uitdehaag, Joost C. M.; de Man, Jos; Willemsen-Seegers, Nicole; Prinsen, Martine B. W.; Libouban, Marion A. A.; Sterrenburg, Jan Gerard; de Wit, Joeri J. P.; de Vetter, Judith R. F.; de Roos, Jeroen A. D. M.; Buijsman, Rogier C.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2211-2230</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The protein kinase threonine tyrosine kinase (TTK; also known as Mps1) is a crit. component of the spindle assembly checkpoint and a promising drug target for the treatment of aggressive cancers, such as triple neg. breast cancer.  While the first TTK inhibitors have entered clin. trials, little is known about how the inhibition of TTK with small-mol. compds. affects cellular activity.  The authors studied the selective TTK inhibitor NTRC 0066-0, which was developed in the authors' own lab., together with 11 TTK inhibitors developed by other companies, including Mps-BAY2b, BAY 1161909, BAY 1217389 (Bayer), TC-Mps1-12 (Shionogi), and MPI-0479605 (Myrexis).  Parallel testing shows that the cellular activity of these TTK inhibitors correlates with their binding affinity to TTK and, more strongly, with target residence time.  TTK inhibitors are therefore an example where target residence time dets. activity in in vitro cellular assays.  X-ray structures and thermal stability expts. reveal that the most potent compds. induce a shift of the glycine-rich loop as a result of binding to the catalytic lysine at position 553.  This "lysine trap" disrupts the catalytic machinery.  Based on these insights, the authors developed TTK inhibitors, based on a (5,6-dihydro)pyrimido[4,5-e]indolizine scaffold, with longer target residence times, which further exploit an allosteric pocket surrounding Lys 553.  Their binding mode is new for kinase inhibitors and can be classified as hybrid Type I/Type III.  These inhibitors have very potent anti-proliferative activity that rivals classic cytotoxic therapy.  The authors' findings will open up new avenues for more applications for TTK inhibitors in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT_6mZVMoVTrVg90H21EOLACvtfcHk0ljpG1-yhTTFug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Wiuro%253D&md5=c02b149e673dec7b0212b37a4c7b0831</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%2BM.%26aulast%3Dde%2BMan%26aufirst%3DJ.%26aulast%3DWillemsen-Seegers%26aufirst%3DN.%26aulast%3DPrinsen%26aufirst%3DM.%2BB.%2BW.%26aulast%3DLibouban%26aufirst%3DM.%2BA.%2BA.%26aulast%3DSterrenburg%26aufirst%3DJ.%2BG.%26aulast%3Dde%2BWit%26aufirst%3DJ.%2BJ.%2BP.%26aulast%3Dde%2BVetter%26aufirst%3DJ.%2BR.%2BF.%26aulast%3Dde%2BRoos%26aufirst%3DJ.%2BA.%2BD.%2BM.%26aulast%3DBuijsman%26aufirst%3DR.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%2BR.%26atitle%3DTarget%2520residence%2520time-guided%2520optimization%2520on%2520TTK%2520kinase%2520results%2520in%2520inhibitors%2520with%2520potent%2520anti-proliferative%2520activity%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2211%26epage%3D2230%26doi%3D10.1016%2Fj.jmb.2017.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itzhak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strover, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span> <span> </span><span class="NLM_article-title">Multiple autophosphorylations significantly enhance the endoribonuclease activity of human inositol requiring enzyme 1α</span>. <i>BMC Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1186/1471-2091-15-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1186%2F1471-2091-15-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=24524643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsFaqtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=3&author=D.+Itzhakauthor=M.+Brightauthor=P.+McAndrewauthor=A.+Mirzaauthor=Y.+Newbattauthor=J.+Stroverauthor=M.+Widyaauthor=A.+Thompsonauthor=G.+Morganauthor=I.+Collinsauthor=F.+Davies&title=Multiple+autophosphorylations+significantly+enhance+the+endoribonuclease+activity+of+human+inositol+requiring+enzyme+1%CE%B1&doi=10.1186%2F1471-2091-15-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple autophosphorylations significantly enhance the endoribonuclease activity of human inositol requiring enzyme 1α</span></div><div class="casAuthors">Itzhak, Daniel; Bright, Michael; McAndrew, Peter; Mirza, Amin; Newbatt, Yvette; Strover, Jade; Widya, Marcella; Thompson, Andrew; Morgan, Gareth; Collins, Ian; Davies, Faith</div><div class="citationInfo"><span class="NLM_cas:title">BMC Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3/1-3/7</span>CODEN:
                <span class="NLM_cas:coden">BBMIB3</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Endoplasmic reticulum stress, caused by the presence of misfolded proteins, activates the stress sensor inositol-requiring enzyme 1α (IRE1α).  The resulting increase in IRE1α RNase activity causes sequence-specific cleavage of X-box binding protein 1 (XBP1) mRNA, resulting in upregulation of the unfolded protein response and cellular adaptation to stress.  The precise mechanism of human IRE1α activation is currently unclear.  The role of IRE1α kinase activity is disputed, as results from the generation of various kinase-inactivating mutations in either yeast or human cells are discordant.  Kinase activity can also be made redundant by small mols. which bind the ATP binding site.  We set out to uncover a role for IRE1α kinase activity using wild-type cytosolic protein constructs.  Results: We show that concn.-dependent oligomerization is sufficient to cause IRE1α cytosolic domain RNase activity in vitro.  We demonstrate a role for the kinase activity by showing that autophosphorylation enhances RNase activity.  Inclusion of the IRE1α linker domain in protein constructs allows hyperphosphorylation and further enhancement of RNase activity, highlighting the importance of kinase activity.  We show that IRE1α phosphorylation status correlates with an increased propensity to form oligomeric complexes and that forced dimerization causes great enhancement in RNase activity.  In addn. we demonstrate that even when IRE1α is forced to dimerize, by a GST-tag, phospho-enhancement of activity is still obsd.  Conclusions: Taken together these expts. support the hypothesis that phosphorylation is important in modulating IRE1α RNase activity which is achieved by increasing the propensity of IRE1α to dimerize.  This work supports the development of IRE1α kinase inhibitors for use in the treatment of secretory cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohh5JaMayEnbVg90H21EOLACvtfcHk0lgXQnqpom_Wmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsFaqtL0%253D&md5=c773198e9f1deb86fccb2076fd2603b7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1186%2F1471-2091-15-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2091-15-3%26sid%3Dliteratum%253Aachs%26aulast%3DItzhak%26aufirst%3DD.%26aulast%3DBright%26aufirst%3DM.%26aulast%3DMcAndrew%26aufirst%3DP.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DStrover%26aufirst%3DJ.%26aulast%3DWidya%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DDavies%26aufirst%3DF.%26atitle%3DMultiple%2520autophosphorylations%2520significantly%2520enhance%2520the%2520endoribonuclease%2520activity%2520of%2520human%2520inositol%2520requiring%2520enzyme%25201%25CE%25B1%26jtitle%3DBMC%2520Biochem.%26date%3D2014%26volume%3D15%26spage%3D3%26doi%3D10.1186%2F1471-2091-15-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strover, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L.</span></span> <span> </span><span class="NLM_article-title">Identification of autophosphorylation inhibitors of the inositol-requiring enzyme 1 alpha (IRE1α) by high-throughput screening using a DELFIA assay</span>. <i>J. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1177/1087057112465647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1177%2F1087057112465647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=23139381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OktrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=298-308&author=Y.+Newbattauthor=A.+Hardcastleauthor=P.+C.+McAndrewauthor=J.+A.+Stroverauthor=A.+Mirzaauthor=G.+J.+Morganauthor=R.+Burkeauthor=F.+E.+Daviesauthor=I.+Collinsauthor=R.+L.+van+Montfort&title=Identification+of+autophosphorylation+inhibitors+of+the+inositol-requiring+enzyme+1+alpha+%28IRE1%CE%B1%29+by+high-throughput+screening+using+a+DELFIA+assay&doi=10.1177%2F1087057112465647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of autophosphorylation inhibitors of the inositol-requiring enzyme 1 alpha (IRE1α) by high-throughput screening using a DELFIA assay</span></div><div class="casAuthors">Newbatt, Yvette; Hardcastle, Anthea; McAndrew, P. Craig; Strover, Jade A.; Mirza, Amin; Morgan, Gareth J.; Burke, Rosemary; Davies, Faith E.; Collins, Ian; van Montfort, Rob L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-308, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Inositol-requiring enzyme 1 alpha (IRE1α) is a transmembrane sensor protein with both kinase and RNase activity, which plays a crucial role in the unfolded protein response (UPR).  Protein misfolding in the endoplasmic reticulum (ER) lumen triggers dimerization and subsequent trans-autophosphorylation of IRE1α.  This leads to the activation of its endoribonuclease (RNase) domain and splicing of the mRNA of the transcriptional activator XBP1, ultimately generating an active XBP1 (XBP1s) implicated in multiple myeloma survival.  Previously, we have identified human IRE1α as a target for the development of kinase inhibitors that could modulate the UPR in human cells, which has particular relevance for multiple myeloma and other secretory malignancies.  Here we describe the development and validation of a 384-well high-throughput screening assay using DELFIA technol. that is specific for IRE1α autophosphorylation.  Using this format, a focused library of 2312 potential kinase inhibitors was screened, and several novel IRE1α kinase inhibitor scaffolds were identified that could potentially be developed toward new therapies to treat multiple myeloma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsN6-SmyMtXbVg90H21EOLACvtfcHk0lgXQnqpom_Wmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OktrfM&md5=97f3f0b8ef6c1ff9b35af0f4165486bc</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1177%2F1087057112465647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112465647%26sid%3Dliteratum%253Aachs%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DHardcastle%26aufirst%3DA.%26aulast%3DMcAndrew%26aufirst%3DP.%2BC.%26aulast%3DStrover%26aufirst%3DJ.%2BA.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DG.%2BJ.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DDavies%26aufirst%3DF.%2BE.%26aulast%3DCollins%26aufirst%3DI.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%26atitle%3DIdentification%2520of%2520autophosphorylation%2520inhibitors%2520of%2520the%2520inositol-requiring%2520enzyme%25201%2520alpha%2520%2528IRE1%25CE%25B1%2529%2520by%2520high-throughput%2520screening%2520using%2520a%2520DELFIA%2520assay%26jtitle%3DJ.%2520Biomol.%2520Screen.%26date%3D2013%26volume%3D18%26spage%3D298%26epage%3D308%26doi%3D10.1177%2F1087057112465647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potashman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bready, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMelfi, T. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPietro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbaum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neervannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polverino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Plas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of orally active benzimidazoles and benzoxazoles as vascular endothelial growth factor-2 receptor tyrosine kinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">4351</span>– <span class="NLM_lpage">4373</span>, <span class="refDoi"> DOI: 10.1021/jm070034i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070034i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovF2mtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=4351-4373&author=M.+H.+Potashmanauthor=J.+Breadyauthor=A.+Coxonauthor=T.+M.+DeMelfiauthor=L.+DiPietroauthor=N.+Doerrauthor=D.+Elbaumauthor=J.+Estradaauthor=P.+Gallantauthor=J.+Germainauthor=Y.+Guauthor=J.+C.+Harmangeauthor=S.+A.+Kaufmanauthor=R.+Kendallauthor=J.+L.+Kimauthor=G.+N.+Kumarauthor=A.+M.+Longauthor=S.+Neervannanauthor=V.+F.+Patelauthor=A.+Polverinoauthor=P.+Roseauthor=S.+van+der%0APlasauthor=D.+Whittingtonauthor=R.+Zanonauthor=H.+Zhao&title=Design%2C+synthesis%2C+and+evaluation+of+orally+active+benzimidazoles+and+benzoxazoles+as+vascular+endothelial+growth+factor-2+receptor+tyrosine+kinase+inhibitors&doi=10.1021%2Fjm070034i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of Orally Active Benzimidazoles and Benzoxazoles as Vascular Endothelial Growth Factor-2 Receptor Tyrosine Kinase Inhibitors</span></div><div class="casAuthors">Potashman, Michele H.; Bready, James; Coxon, Angela; DeMelfi, Thomas M., Jr.; DiPietro, Lucian; Doerr, Nicholas; Elbaum, Daniel; Estrada, Juan; Gallant, Paul; Germain, Julie; Gu, Yan; Harmange, Jean-Christophe; Kaufman, Stephen A.; Kendall, Rick; Kim, Joseph L.; Kumar, Gondi N.; Long, Alexander M.; Neervannan, Seshadri; Patel, Vinod F.; Polverino, Anthony; Rose, Paul; Van der Plas, Simon; Whittington, Douglas; Zanon, Roger; Zhao, Huilin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4351-4373</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">inhibition of the VEGF signaling pathway has become a valuable approach in the treatment of cancers.  guided by X-ray crystallog. and mol. modeling, a series of 2-aminobenzimidazoles and 2-aminobenzoxazoles were identified as potent inhibitors of the vascular endothelial growth factor-2 receptor tyrosine kinase VEGFR-2 (also known as KDR, kinase insert domain receptor) in both enzymic and HUVEC cellular proliferation assays.  in this report we describe the synthesis and structure-activity relationship of a series of 2-aminobenzimidazoles and benzoxazoles, culminating in the identification of a nonracemic pyrrolidinylmethoxyphenylaminobenzoxazolyloxy pyridinecarboxamide as a potent and selective VEGFR-2 inhibitor displaying a good pharmacokinetic profile.  the nonracemic pyrrolidinylmethoxyphenylaminobenzoxazolyloxy pyridinecarboxamide also demonstrated efficacy in both the murine matrigel model for vascular permeability (79% inhibition obsd. at 100 mg/kg) and the rat corneal angiogenesis model (ED50 = 16.3 mg/kg).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqG7laVqJmUbVg90H21EOLACvtfcHk0lj5-vMsTtjjiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovF2mtro%253D&md5=e4639afa8ba460a8679dcc75bc270d8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm070034i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070034i%26sid%3Dliteratum%253Aachs%26aulast%3DPotashman%26aufirst%3DM.%2BH.%26aulast%3DBready%26aufirst%3DJ.%26aulast%3DCoxon%26aufirst%3DA.%26aulast%3DDeMelfi%26aufirst%3DT.%2BM.%26aulast%3DDiPietro%26aufirst%3DL.%26aulast%3DDoerr%26aufirst%3DN.%26aulast%3DElbaum%26aufirst%3DD.%26aulast%3DEstrada%26aufirst%3DJ.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DGermain%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKaufman%26aufirst%3DS.%2BA.%26aulast%3DKendall%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DKumar%26aufirst%3DG.%2BN.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DNeervannan%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DV.%2BF.%26aulast%3DPolverino%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BPlas%26aufirst%3DS.%26aulast%3DWhittington%26aufirst%3DD.%26aulast%3DZanon%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%2520orally%2520active%2520benzimidazoles%2520and%2520benzoxazoles%2520as%2520vascular%2520endothelial%2520growth%2520factor-2%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D4351%26epage%3D4373%26doi%3D10.1021%2Fjm070034i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1038/nchembio799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations&doi=10.1038%2Fnchembio799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0lj5-vMsTtjjiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iwawaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosoda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamigori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura-Furuwatari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, K.</span></span> <span> </span><span class="NLM_article-title">Translational control by the ER transmembrane kinase/ribonuclease IRE1 under ER stress</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">158</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1038/35055065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2F35055065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=11175748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1KmsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2001&pages=158-164&author=T.+Iwawakiauthor=A.+Hosodaauthor=T.+Okudaauthor=Y.+Kamigoriauthor=C.+Nomura-Furuwatariauthor=Y.+Kimataauthor=A.+Tsuruauthor=K.+Kohno&title=Translational+control+by+the+ER+transmembrane+kinase%2Fribonuclease+IRE1+under+ER+stress&doi=10.1038%2F35055065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Translational control by the ER transmembrane kinase/ribonuclease IRE1 under ER stress</span></div><div class="casAuthors">Iwawaki, Takao; Hosoda, Akira; Okuda, Tetsuo; Kamigori, Yusuke; Nomura-Furuwatari, Chizumi; Kimata, Yukio; Tsuru, Akio; Kohno, Kenji</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">158-164</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Under conditions of endoplasmic reticulum (ER) stress, mammalian cells induce both translational repression and the unfolded protein response that transcriptionally activates genes encoding ER-resident mol. chaperones.  To date, the only known pathway for translational repression in response to ER stress has been the phosphorylation of eIF-2α by the double-stranded RNA-activated protein kinase (PKR) or the transmembrane PKR-like ER kinase (PERK).  Here we report another pathway in which the ER transmembrane kinase/RNase IRE1β induces translational repression through 28S rRNA cleavage in response to ER stress.  The evidence suggests that both pathways are important for efficient translational repression during the ER stress response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn7Fzg6AubS7Vg90H21EOLACvtfcHk0ljGPP_TYG7Pyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1KmsLg%253D&md5=39608203eb79b14e442aa31b6789bf45</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2F35055065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35055065%26sid%3Dliteratum%253Aachs%26aulast%3DIwawaki%26aufirst%3DT.%26aulast%3DHosoda%26aufirst%3DA.%26aulast%3DOkuda%26aufirst%3DT.%26aulast%3DKamigori%26aufirst%3DY.%26aulast%3DNomura-Furuwatari%26aufirst%3DC.%26aulast%3DKimata%26aufirst%3DY.%26aulast%3DTsuru%26aufirst%3DA.%26aulast%3DKohno%26aufirst%3DK.%26atitle%3DTranslational%2520control%2520by%2520the%2520ER%2520transmembrane%2520kinase%252Fribonuclease%2520IRE1%2520under%2520ER%2520stress%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2001%26volume%3D3%26spage%3D158%26epage%3D164%26doi%3D10.1038%2F35055065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35a"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosmopoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atrash, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of an Aurora-A mutant that mimics Aurora-B bound to MLN8054: insights into selectivity and drug design</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>427</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1042/BJ20091530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1042%2FBJ20091530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20067443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2010&pages=19-28&author=C.+A.+Dodsonauthor=M.+Kosmopoulouauthor=M.+W.+Richardsauthor=B.+Atrashauthor=V.+Bavetsiasauthor=J.+Blaggauthor=R.+Bayliss&title=Crystal+structure+of+an+Aurora-A+mutant+that+mimics+Aurora-B+bound+to+MLN8054%3A+insights+into+selectivity+and+drug+design&doi=10.1042%2FBJ20091530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35aR"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of an Aurora-A mutant that mimics Aurora-B bound to MLN8054: insights into selectivity and drug design</span></div><div class="casAuthors">Dodson, Charlotte A.; Kosmopoulou, Magda; Richards, Mark W.; Atrash, Butrus; Bavetsias, Vassilios; Blagg, Julian; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">427</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-28</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The prodn. of selective protein kinase inhibitors is often frustrated by the similarity of the enzyme active sites.  For this reason, it is challenging to design inhibitors that discriminate between the three Aurora kinases, which are important targets in cancer drug discovery.  We have used a triple-point mutant of Aurora-A (AurAx3) which mimics the active site of Aurora-B to investigate the structural basis of MLN8054 selectivity.  The bias toward Aurora-A inhibition by MLN8054 is fully recapitulated by AurAx3 in vitro.  X-ray crystal structures of the complex suggest that the basis for the discrimination is electrostatic repulsion due to the T217E substitution, which we have confirmed using a single-point mutant.  The activation loop of Aurora-A in the AurAx3-MLN8054 complex exhibits an unusual conformation in which Asp274 and Phe275 side chains point into the interior of the protein.  There is to our knowledge no documented precedent for this conformation, which we have termed DFG-up.  The sequence requirements of the DFG-up conformation suggest that it might be accessible to only a fraction of kinases.  MLN8054 thus circumvents the problem of highly homologous active sites.  Binding of MLN8054 to Aurora-A switches the character of a pocket within the active site from polar to a hydrophobic pocket, similar to what is obsd. in the structure of Aurora-A bound to a compd. that induces DFG-out.  We propose that targeting this pocket may be a productive route in the design of selective kinase inhibitors and describe the structural basis for the rational design of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdJs5KY2VkabVg90H21EOLACvtfcHk0ljGPP_TYG7Pyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyntr8%253D&md5=fdf1a176a5fdf5fcd71fd57c3ef92817</span></div><a href="/servlet/linkout?suffix=cit35a&amp;dbid=16384&amp;doi=10.1042%2FBJ20091530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091530%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DC.%2BA.%26aulast%3DKosmopoulou%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520an%2520Aurora-A%2520mutant%2520that%2520mimics%2520Aurora-B%2520bound%2520to%2520MLN8054%253A%2520insights%2520into%2520selectivity%2520and%2520drug%2520design%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26spage%3D19%26epage%3D28%26doi%3D10.1042%2FBJ20091530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="note"><p class="first last">Erratum in:</p></div><div class="NLM_citation" id="rightTab-cit35a1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosmopoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atrash, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavetsias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>427</i></span>,  <span class="NLM_fpage">551</span>, <span class="refDoi"> DOI: 10.1042/BJ4270551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35a1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1042%2FBJ4270551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dodson%2C+C.+A.%3B+Kosmopoulou%2C+M.%3B+Richards%2C+M.+W.%3B+Atrash%2C+B.%3B+Bavetsias%2C+V.%3B+Blagg%2C+J.%3B+Bayliss%2C+R.+Biochem.+J.+2010%2C+427%2C+551%2C+10.1042%2FBJ4270551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35a1&amp;dbid=16384&amp;doi=10.1042%2FBJ4270551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ4270551%26sid%3Dliteratum%253Aachs%26aulast%3DDodson%26aufirst%3DC.%2BA.%26aulast%3DKosmopoulou%26aufirst%3DM.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DAtrash%26aufirst%3DB.%26aulast%3DBavetsias%26aufirst%3DV.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBayliss%26aufirst%3DR.%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D427%26spage%3D551%26doi%3D10.1042%2FBJ4270551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas-Martin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dotson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drummond, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawson, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">A class of 2,4-bisanilinopyrimidine Aurora A inhibitors with unusually high selectivity against Aurora B</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3300</span>– <span class="NLM_lpage">3307</span>, <span class="refDoi"> DOI: 10.1021/jm9000314</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9000314" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=3300-3307&author=I.+Aliagas-Martinauthor=D.+Burdickauthor=L.+Corsonauthor=J.+Dotsonauthor=J.+Drummondauthor=C.+Fieldsauthor=O.+W.+Huangauthor=T.+Hunsakerauthor=T.+Kleinheinzauthor=E.+Kruegerauthor=J.+Liangauthor=J.+Moffatauthor=G.+Phillipsauthor=R.+Pulkauthor=T.+E.+Rawsonauthor=M.+Ultschauthor=L.+Walkerauthor=C.+Wiesmannauthor=B.+Zhangauthor=B.+Y.+Zhuauthor=A.+G.+Cochran&title=A+class+of+2%2C4-bisanilinopyrimidine+Aurora+A+inhibitors+with+unusually+high+selectivity+against+Aurora+B&doi=10.1021%2Fjm9000314"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm9000314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9000314%26sid%3Dliteratum%253Aachs%26aulast%3DAliagas-Martin%26aufirst%3DI.%26aulast%3DBurdick%26aufirst%3DD.%26aulast%3DCorson%26aufirst%3DL.%26aulast%3DDotson%26aufirst%3DJ.%26aulast%3DDrummond%26aufirst%3DJ.%26aulast%3DFields%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DKrueger%26aufirst%3DE.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DG.%26aulast%3DPulk%26aufirst%3DR.%26aulast%3DRawson%26aufirst%3DT.%2BE.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DWalker%26aufirst%3DL.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DB.%2BY.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DA%2520class%2520of%25202%252C4-bisanilinopyrimidine%2520Aurora%2520A%2520inhibitors%2520with%2520unusually%2520high%2520selectivity%2520against%2520Aurora%2520B%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D3300%26epage%3D3307%26doi%3D10.1021%2Fjm9000314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moslin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santella, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncia, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strnad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedicord, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zupa-Fernandez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arienzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muckelbauer, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tredup, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aranibar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, D. S.</span></span> <span> </span><span class="NLM_article-title">Identification of imidazo[1,2-b]pyridazine TYK2 pseudokinase ligands as potent and selective allosteric inhibitors of TYK2 signalling</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1039/C6MD00560H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1039%2FC6MD00560H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=30108788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyru7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=700-712&author=R.+Moslinauthor=D.+Gardnerauthor=J.+Santellaauthor=Y.+Zhangauthor=J.+V.+Dunciaauthor=C.+Liuauthor=J.+Linauthor=J.+S.+Tokarskiauthor=J.+Strnadauthor=D.+Pedicordauthor=J.+Chenauthor=Y.+Blatauthor=A.+Zupa-Fernandezauthor=L.+Chengauthor=H.+Sunauthor=C.+Chaudhryauthor=C.+Huangauthor=C.+D%E2%80%99Arienzoauthor=J.+S.+Sackauthor=J.+K.+Muckelbauerauthor=C.+Changauthor=J.+Tredupauthor=D.+Xieauthor=N.+Aranibarauthor=J.+R.+Burkeauthor=P.+H.+Carterauthor=D.+S.+Weinstein&title=Identification+of+imidazo%5B1%2C2-b%5Dpyridazine+TYK2+pseudokinase+ligands+as+potent+and+selective+allosteric+inhibitors+of+TYK2+signalling&doi=10.1039%2FC6MD00560H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of imidazo[1,2-b]pyridazine TYK2 pseudokinase ligands as potent and selective allosteric inhibitors of TYK2 signalling</span></div><div class="casAuthors">Moslin, R.; Gardner, D.; Santella, J.; Zhang, Y.; Duncia, J. V.; Liu, C.; Lin, J.; Tokarski, J. S.; Strnad, J.; Pedicord, D.; Chen, J.; Blat, Y.; Zupa-Fernandez, A.; Cheng, L.; Sun, H.; Chaudhry, C.; Huang, C.; D'Arienzo, C.; Sack, J. S.; Muckelbauer, J. K.; Chang, C.; Tredup, J.; Xie, D.; Aranibar, N.; Burke, J. R.; Carter, P. H.; Weinstein, D. S.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">700-712</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As a member of the Janus (JAK) family of non-receptor tyrosine kinases, TYK2 mediates the signaling of pro-inflammatory cytokines including IL-12, IL-23 and type 1 interferon (IFN), and therefore represents an attractive potential target for treating the various immuno-inflammatory diseases in which these cytokines have been shown to play a role.  Following up on our previous report that ligands to the pseudokinase domain (JH2) of TYK2 suppress cytokine-mediated receptor activation of the catalytic (JH1) domain, the imidazo[1,2-b]pyridazine (IZP) 7 was identified as a promising hit compd.  Through iterative modification of each of the substituents of the IZP scaffold, the cellular potency was improved while maintaining selectivity over the JH1 domain.  These studies led to the discovery of the JH2-selective TYK2 inhibitor 29, which provided encouraging systemic exposures after oral dosing in mice.  Phosphodiesterase 4 (PDE4) was identified as an off-target and potential liability of the IZP ligands, and selectivity for TYK2 JH2 over this enzyme was obtained by elaborating along selectivity vectors detd. from analyses of X-ray co-crystal structures of representative ligands of the IZP class bound to both proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobX5_XfvLHsLVg90H21EOLACvtfcHk0lhE99PFlDpMNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyru7jN&md5=67b06413866ee6096810903f62edd901</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC6MD00560H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00560H%26sid%3Dliteratum%253Aachs%26aulast%3DMoslin%26aufirst%3DR.%26aulast%3DGardner%26aufirst%3DD.%26aulast%3DSantella%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDuncia%26aufirst%3DJ.%2BV.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DTokarski%26aufirst%3DJ.%2BS.%26aulast%3DStrnad%26aufirst%3DJ.%26aulast%3DPedicord%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBlat%26aufirst%3DY.%26aulast%3DZupa-Fernandez%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DChaudhry%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Arienzo%26aufirst%3DC.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DMuckelbauer%26aufirst%3DJ.%2BK.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DTredup%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DD.%26aulast%3DAranibar%26aufirst%3DN.%26aulast%3DBurke%26aufirst%3DJ.%2BR.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DWeinstein%26aufirst%3DD.%2BS.%26atitle%3DIdentification%2520of%2520imidazo%255B1%252C2-b%255Dpyridazine%2520TYK2%2520pseudokinase%2520ligands%2520as%2520potent%2520and%2520selective%2520allosteric%2520inhibitors%2520of%2520TYK2%2520signalling%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D700%26epage%3D712%26doi%3D10.1039%2FC6MD00560H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermann, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luk, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, C.</span></span> <span> </span><span class="NLM_article-title">Rational design of highly selective spleen tyrosine kinase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">10414</span>– <span class="NLM_lpage">10423</span>, <span class="refDoi"> DOI: 10.1021/jm301367c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301367c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10414-10423&author=M.+C.+Lucasauthor=D.+M.+Goldsteinauthor=J.+C.+Hermannauthor=A.+Kuglstatterauthor=W.+Liuauthor=K.+C.+Lukauthor=F.+Padillaauthor=M.+Sladeauthor=A.+G.+Villasenorauthor=J.+Wannerauthor=W.+Xieauthor=X.+Zhangauthor=C.+Liao&title=Rational+design+of+highly+selective+spleen+tyrosine+kinase+inhibitors&doi=10.1021%2Fjm301367c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm301367c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301367c%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DM.%2BC.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DKuglstatter%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLuk%26aufirst%3DK.%2BC.%26aulast%3DPadilla%26aufirst%3DF.%26aulast%3DSlade%26aufirst%3DM.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DWanner%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLiao%26aufirst%3DC.%26atitle%3DRational%2520design%2520of%2520highly%2520selective%2520spleen%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10414%26epage%3D10423%26doi%3D10.1021%2Fjm301367c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougan, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komandla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyrick, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of a small molecule inhibitor of Bruton’s tyrosine kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5437</span>– <span class="NLM_lpage">5444</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00734</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00734" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVans7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5437-5444&author=C.+R.+Smithauthor=D.+R.+Douganauthor=M.+Komandlaauthor=T.+Kanouniauthor=B.+Knightauthor=J.+D.+Lawsonauthor=M.+Sabatauthor=E.+R.+Taylorauthor=P.+Vuauthor=C.+Wyrick&title=Fragment-based+discovery+of+a+small+molecule+inhibitor+of+Bruton%E2%80%99s+tyrosine+kinase&doi=10.1021%2Facs.jmedchem.5b00734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of a Small Molecule Inhibitor of Bruton's Tyrosine Kinase</span></div><div class="casAuthors">Smith, Christopher R.; Dougan, Douglas R.; Komandla, Mallareddy; Kanouni, Toufike; Knight, Beverly; Lawson, J. David; Sabat, Mark; Taylor, Ewan R.; Vu, Phong; Wyrick, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5437-5444</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery and optimization of a series of 4-aminocinnoline-3-carboxamide inhibitors of Bruton's tyrosine kinase are reported.  A fragment-based screening approach incorporating x-ray cocrystallog. was used to identify a cinnoline fragment and characterize its binding mode in the ATP binding site of Btk.  Optimization of the fragment hit resulted in the identification of a lead compd. I which reduced paw swelling in a dose- and exposure-dependent fashion in a rat model of collagen-induced arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozENTW7pHlFrVg90H21EOLACvtfcHk0liPBrcZC8wSZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVans7rF&md5=d6626fabcc3e2aa6a9180dc702a5030c</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00734%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BR.%26aulast%3DDougan%26aufirst%3DD.%2BR.%26aulast%3DKomandla%26aufirst%3DM.%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DKnight%26aufirst%3DB.%26aulast%3DLawson%26aufirst%3DJ.%2BD.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DE.%2BR.%26aulast%3DVu%26aufirst%3DP.%26aulast%3DWyrick%26aufirst%3DC.%26atitle%3DFragment-based%2520discovery%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520Bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5437%26epage%3D5444%26doi%3D10.1021%2Facs.jmedchem.5b00734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paavilainen, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafimova, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Electrophilic fragment-based design of reversible covalent kinase inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">5298</span>– <span class="NLM_lpage">5301</span>, <span class="refDoi"> DOI: 10.1021/ja401221b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja401221b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=5298-5301&author=R.+M.+Millerauthor=V.+O.+Paavilainenauthor=S.+Krishnanauthor=I.+M.+Serafimovaauthor=J.+Taunton&title=Electrophilic+fragment-based+design+of+reversible+covalent+kinase+inhibitors&doi=10.1021%2Fja401221b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophilic Fragment-Based Design of Reversible Covalent Kinase Inhibitors</span></div><div class="casAuthors">Miller, Rand M.; Paavilainen, Ville O.; Krishnan, Shyam; Serafimova, Iana M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5298-5301</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based ligand design and covalent targeting of noncatalytic cysteines have been employed to develop potent and selective kinase inhibitors.  Here, we combine these approaches, starting with a panel of low-mol.-wt., heteroaryl-substituted cyanoacrylamides, which we have previously shown to form reversible covalent bonds with cysteine thiols.  Using this strategy, we identify electrophilic fragments with sufficient ligand efficiency and selectivity to serve as starting points for the first reported inhibitors of the MSK1 C-terminal kinase domain.  Guided by X-ray co-crystal structures, indazole fragment 1 was elaborated to afford 12 (RMM-46), a reversible covalent inhibitor that exhibits high ligand efficiency and selectivity for MSK/RSK-family kinases.  At nanomolar concns., 12 blocked activation of cellular MSK and RSK, as well as downstream phosphorylation of the crit. transcription factor, CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrax7wBNWOT8rVg90H21EOLACvtfcHk0liPBrcZC8wSZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D&md5=c6100b56ce6f587f6afb834aeafec351</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja401221b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja401221b%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DElectrophilic%2520fragment-based%2520design%2520of%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D5298%26epage%3D5301%26doi%3D10.1021%2Fja401221b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korennykh, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrman, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">Mammalian endoplasmic reticulum stress sensor IRE1 signals by dynamic clustering</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">16113</span>– <span class="NLM_lpage">16118</span>, <span class="refDoi"> DOI: 10.1073/pnas.1010580107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1073%2Fpnas.1010580107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20798350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2ju7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=16113-16118&author=H.+Liauthor=A.+V.+Korennykhauthor=S.+L.+Behrmanauthor=P.+Walter&title=Mammalian+endoplasmic+reticulum+stress+sensor+IRE1+signals+by+dynamic+clustering&doi=10.1073%2Fpnas.1010580107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian endoplasmic reticulum stress sensor IRE1 signals by dynamic clustering</span></div><div class="casAuthors">Li, Han; Korennykh, Alexei V.; Behrman, Shannon L.; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">16113-16118, S16113/1-S16113/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Accumulation of misfolded proteins in the endoplasmic reticulum (ER) triggers the unfolded protein response (UPR), an intracellular signaling pathway that adjusts the protein folding capacity of the ER according to need.  If homeostasis in the ER protein folding environment cannot be reestablished, cells commit to apoptosis.  The ER-resident transmembrane kinase-endo-RNase inositol-requiring enzyme 1 (IRE1) is the best characterized UPR signal transduction mol.  In yeast, Ire1 oligomerizes upon activation in response to an accumulation of misfolded proteins in the ER.  Here we show that the salient mechanistic features of IRE1 activation are conserved: mammalian IRE1 oligomerizes in the ER membrane and oligomerization correlates with the onset of IRE1 phosphorylation and RNase activity.  Moreover, the kinase/RNase module of human IRE1 activates cooperatively in vitro, indicating that formation of oligomers larger than four IRE1 mols. takes place upon activation.  High-order IRE1 oligomerization thus emerges as a conserved mechanism of IRE1 signaling.  IRE1 signaling attenuates after prolonged ER stress.  IRE1 then enters a refractive state even if ER stress remains unmitigated.  Attenuation includes dissoln. of IRE1 clusters, IRE1 dephosphorylation, and decline in endo-RNase activity.  Thus IRE1 activity is governed by a timer that may be important in switching the UPR from the initially cytoprotective phase to the apoptotic mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhiPftLtEiZLVg90H21EOLACvtfcHk0lh1aolPsPM-hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2ju7bJ&md5=ecf826507d91fbd659134634eba800c9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1010580107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1010580107%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DKorennykh%26aufirst%3DA.%2BV.%26aulast%3DBehrman%26aufirst%3DS.%2BL.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DMammalian%2520endoplasmic%2520reticulum%2520stress%2520sensor%2520IRE1%2520signals%2520by%2520dynamic%2520clustering%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D16113%26epage%3D16118%26doi%3D10.1073%2Fpnas.1010580107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haq, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, S.</span></span> <span> </span><span class="NLM_article-title">The Ys and wherefores of protein kinase autoinhibition</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1854</i></span>,  <span class="NLM_fpage">1586</span>– <span class="NLM_lpage">1594</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2015.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.bbapap.2015.04.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1854&publication_year=2015&pages=1586-1594&author=R.+Baylissauthor=T.+Haqauthor=S.+Yeoh&title=The+Ys+and+wherefores+of+protein+kinase+autoinhibition&doi=10.1016%2Fj.bbapap.2015.04.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2015.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2015.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DR.%26aulast%3DHaq%26aufirst%3DT.%26aulast%3DYeoh%26aufirst%3DS.%26atitle%3DThe%2520Ys%2520and%2520wherefores%2520of%2520protein%2520kinase%2520autoinhibition%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2015%26volume%3D1854%26spage%3D1586%26epage%3D1594%26doi%3D10.1016%2Fj.bbapap.2015.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thevakumaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span> <span> </span><span class="NLM_article-title">Dimerization-induced allostery in protein kinase regulation</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2014.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1016%2Fj.tibs.2014.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=25220378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=475-486&author=H.+Lavoieauthor=J.+J.+Liauthor=N.+Thevakumaranauthor=M.+Therrienauthor=F.+Sicheri&title=Dimerization-induced+allostery+in+protein+kinase+regulation&doi=10.1016%2Fj.tibs.2014.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Dimerization-induced allostery in protein kinase regulation</span></div><div class="casAuthors">Lavoie, Hugo; Li, John J.; Thevakumaran, Neroshan; Therrien, Marc; Sicheri, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">475-486</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The ability of protein kinases to switch between inactive and active states is crit. to control the outputs of cellular signaling pathways.  In several protein kinases, the conformation of helix αC is a key hub on which regulatory inputs converge to induce catalytic switching.  An emerging mechanism involved in regulating helix αC orientation is the allosteric coupling with kinase domain surfaces involved in homo- or heterodimerization.  In this review, we discuss dimerization-mediated regulation of the rapidly accelerated fibrosarcoma (RAF) and eIF2α kinase families and draw parallels with the analogous behavior of the epidermal growth factor receptor (EGFR) and serine/threonine-protein kinase endoribonuclease 1 (IRE1)/RNase L (RNAse L) kinase families.  Given that resistance to RAF-targeted therapeutics often stems from dimerization-dependent mechanisms, we suggest that a better understanding of dimerization-induced allostery may assist in developing alternate therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaIF5kAf47DbVg90H21EOLACvtfcHk0lh1aolPsPM-hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKnur7N&md5=34d4079ea8c7906193dc4f2384bf6364</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DLavoie%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DThevakumaran%26aufirst%3DN.%26aulast%3DTherrien%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DDimerization-induced%2520allostery%2520in%2520protein%2520kinase%2520regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26spage%3D475%26epage%3D486%26doi%3D10.1016%2Fj.tibs.2014.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandhuber, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarado, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlam, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiswal, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seshagiri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeppen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span> <span> </span><span class="NLM_article-title">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">435</span>, <span class="refDoi"> DOI: 10.1038/nature08833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature08833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20130576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=431-435&author=G.+Hatzivassiliouauthor=K.+Songauthor=I.+Yenauthor=B.+J.+Brandhuberauthor=D.+J.+Andersonauthor=R.+Alvaradoauthor=M.+J.+Ludlamauthor=D.+Stokoeauthor=S.+L.+Gloorauthor=G.+Vigersauthor=T.+Moralesauthor=I.+Aliagasauthor=B.+Liuauthor=S.+Siderisauthor=K.+P.+Hoeflichauthor=B.+S.+Jaiswalauthor=S.+Seshagiriauthor=H.+Koeppenauthor=M.+Belvinauthor=L.+S.+Friedmanauthor=S.+Malek&title=RAF+inhibitors+prime+wild-type+RAF+to+activate+the+MAPK+pathway+and+enhance+growth&doi=10.1038%2Fnature08833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span></div><div class="casAuthors">Hatzivassiliou, Georgia; Song, Kyung; Yen, Ivana; Brandhuber, Barbara J.; Anderson, Daniel J.; Alvarado, Ryan; Ludlam, Mary J. C.; Stokoe, David; Gloor, Susan L.; Vigers, Guy; Morales, Tony; Aliagas, Ignacio; Liu, Bonnie; Sideris, Steve; Hoeflich, Klaus P.; Jaiswal, Bijay S.; Seshagiri, Somasekar; Koeppen, Hartmut; Belvin, Marcia; Friedman, Lori S.; Malek, Shiva</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7287</span>),
    <span class="NLM_cas:pages">431-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activating mutations in KRAS and BRAF are found in more than 30% of all human tumors and 40% of melanoma, resp., thus targeting this pathway could have broad therapeutic effects.  Small mol. ATP-competitive RAF kinase inhibitors have potent antitumor effects on mutant BRAF(V600E) tumors but, in contrast to mitogen-activated protein kinase kinase (MEK) inhibitors, are not potent against RAS mutant tumor models, despite RAF functioning as a key effector downstream of RAS and upstream of MEK.  Here we show that ATP-competitive RAF inhibitors have two opposing mechanisms of action depending on the cellular context.  In BRAF(V600E) tumors, RAF inhibitors effectively block the mitogen-activated protein kinase (MAPK) signalling pathway and decrease tumor growth.  Notably, in KRAS mutant and RAS/RAF wild-type tumors, RAF inhibitors activate the RAF-MEK-ERK pathway in a RAS-dependent manner, thus enhancing tumor growth in some xenograft models.  Inhibitor binding activates wild-type RAF isoforms by inducing dimerization, membrane localization and interaction with RAS-GTP.  These events occur independently of kinase inhibition and are, instead, linked to direct conformational effects of inhibitors on the RAF kinase domain.  On the basis of these findings, we demonstrate that ATP-competitive kinase inhibitors can have opposing functions as inhibitors or activators of signalling pathways, depending on the cellular context.  Furthermore, this work provides new insights into the therapeutic use of ATP-competitive RAF inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGounpcMETzVoLVg90H21EOLACvtfcHk0ljsUGkRq86-gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D&md5=eb48bdf352ee9c9e182e63e38fd820a5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature08833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08833%26sid%3Dliteratum%253Aachs%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DBrandhuber%26aufirst%3DB.%2BJ.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DAlvarado%26aufirst%3DR.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DGloor%26aufirst%3DS.%2BL.%26aulast%3DVigers%26aufirst%3DG.%26aulast%3DMorales%26aufirst%3DT.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DJaiswal%26aufirst%3DB.%2BS.%26aulast%3DSeshagiri%26aufirst%3DS.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26atitle%3DRAF%2520inhibitors%2520prime%2520wild-type%2520RAF%2520to%2520activate%2520the%2520MAPK%2520pathway%2520and%2520enhance%2520growth%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D431%26epage%3D435%26doi%3D10.1038%2Fnature08833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poulikakos, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">RAF inhibitors transactivate RAF dimers and ERK signalling in cells with wild-type BRAF</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>464</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1038/nature08902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fnature08902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20179705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsFWjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=427-430&author=P.+I.+Poulikakosauthor=C.+Zhangauthor=G.+Bollagauthor=K.+M.+Shokatauthor=N.+Rosen&title=RAF+inhibitors+transactivate+RAF+dimers+and+ERK+signalling+in+cells+with+wild-type+BRAF&doi=10.1038%2Fnature08902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">RAF inhibitors transactivate RAF dimers and ERK signalling in cells with wild-type BRAF</span></div><div class="casAuthors">Poulikakos, Poulikos I.; Zhang, Chao; Bollag, Gideon; Shokat, Kevan M.; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7287</span>),
    <span class="NLM_cas:pages">427-430</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumors with mutant BRAF are dependent on the RAF-MEK-ERK signaling pathway for their growth.  We found that ATP-competitive RAF inhibitors inhibit ERK signaling in cells with mutant BRAF, but unexpectedly enhance signaling in cells with wild-type BRAF.  Here we demonstrate the mechanistic basis for these findings.  We used chem. genetic methods to show that drug-mediated transactivation of RAF dimers is responsible for paradoxical activation of the enzyme by inhibitors.  Induction of ERK signaling requires direct binding of the drug to the ATP-binding site of one kinase of the dimer and is dependent on RAS activity.  Drug binding to one member of RAF homodimers (CRAF-CRAF) or heterodimers (CRAF-BRAF) inhibits one protomer, but results in transactivation of the drug-free protomer.  In BRAF(V600E) tumors, RAS is not activated, thus transactivation is minimal and ERK signaling is inhibited in cells exposed to RAF inhibitors.  These results indicate that RAF inhibitors will be effective in tumors in which BRAF is mutated.  Furthermore, because RAF inhibitors do not inhibit ERK signaling in other cells, the model predicts that they would have a higher therapeutic index and greater antitumor activity than mitogen-activated protein kinase (MEK) inhibitors, but could also cause toxicity due to MEK/ERK activation.  These predictions have been borne out in a recent clin. trial of the RAF inhibitor PLX4032 (refs 4, 5).  The model indicates that promotion of RAF dimerization by elevation of wild-type RAF expression or RAS activity could lead to drug resistance in mutant BRAF tumors.  In agreement with this prediction, RAF inhibitors do not inhibit ERK signaling in cells that coexpress BRAF(V600E) and mutant RAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMlm9feupyNrVg90H21EOLACvtfcHk0ljsUGkRq86-gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsFWjsrs%253D&md5=8323a558886ccbb8f7d85b22b9bfbc15</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnature08902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08902%26sid%3Dliteratum%253Aachs%26aulast%3DPoulikakos%26aufirst%3DP.%2BI.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DRAF%2520inhibitors%2520transactivate%2520RAF%2520dimers%2520and%2520ERK%2520signalling%2520in%2520cells%2520with%2520wild-type%2520BRAF%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D427%26epage%3D430%26doi%3D10.1038%2Fnature08902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arora, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Michele, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermarliere, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Heerde, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linders, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span> <span> </span><span class="NLM_article-title">Structural investigation of B-Raf paradox breaker and inducer inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1818</span>– <span class="NLM_lpage">1831</span>, <span class="refDoi"> DOI: 10.1021/jm501667n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501667n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1818-1831&author=R.+Aroraauthor=M.+Di+Micheleauthor=E.+Stesauthor=E.+Vandermarliereauthor=L.+Martensauthor=K.+Gevaertauthor=E.+Van+Heerdeauthor=J.+T.+Lindersauthor=D.+Brehmerauthor=E.+Jacobyauthor=P.+Bonnet&title=Structural+investigation+of+B-Raf+paradox+breaker+and+inducer+inhibitors&doi=10.1021%2Fjm501667n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Investigation of B-Raf Paradox Breaker and Inducer Inhibitors</span></div><div class="casAuthors">Arora, Rohit; Di Michele, Michela; Stes, Elisabeth; Vandermarliere, Elien; Martens, Lennart; Gevaert, Kris; Van Heerde, Erika; Linders, Joannes T. M.; Brehmer, Dirk; Jacoby, Edgar; Bonnet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1818-1831</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The V600E missense mutation in B-Raf kinase leads to an anomalous regulation of the MAPK pathway, uncontrolled cell proliferation, and initiation of tumorigenesis.  While the ATP-competitive B-Raf inhibitors block the MAPK pathway in B-Raf mutant cells, they induce conformational changes to wild-type B-Raf kinase domain leading to heterodimerization with C-Raf causing a paradoxical hyperactivation of MAPK pathway.  A new class of inhibitors (paradox breakers) has been developed that inhibit B-RafV600E activity without agonistically affecting the MAPK pathway in wild-type B-Raf cells.  In this study, we explore the structural, conformational, and cellular effects on the B-Raf kinase domain upon binding of paradox breakers and inducers.  Our results indicate that a subtle structural difference between paradox inducers and breakers leads to significant conformational differences when complexed with B-Raf.  This study provides a novel insight into the activation of B-Raf by ATP-competitive inhibitors and can aid in the design of more potent and selective inhibitors without agonistic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDJw-VSU5CabVg90H21EOLACvtfcHk0liJjWLCyYx3HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtr8%253D&md5=c79cfd000d525f553a5247b9e917b3e7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm501667n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501667n%26sid%3Dliteratum%253Aachs%26aulast%3DArora%26aufirst%3DR.%26aulast%3DDi%2BMichele%26aufirst%3DM.%26aulast%3DStes%26aufirst%3DE.%26aulast%3DVandermarliere%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DL.%26aulast%3DGevaert%26aufirst%3DK.%26aulast%3DVan%2BHeerde%26aufirst%3DE.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DBonnet%26aufirst%3DP.%26atitle%3DStructural%2520investigation%2520of%2520B-Raf%2520paradox%2520breaker%2520and%2520inducer%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1818%26epage%3D1831%26doi%3D10.1021%2Fjm501667n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulkes, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyers, P. A.</span></span> <span> </span><span class="NLM_article-title">Pseudokinases: update on their functions and evaluation as new drug targets</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">265</span>, <span class="refDoi"> DOI: 10.4155/fmc-2016-0207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.4155%2Ffmc-2016-0207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=28097887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslWnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=245-265&author=D.+P.+Byrneauthor=D.+M.+Foulkesauthor=P.+A.+Eyers&title=Pseudokinases%3A+update+on+their+functions+and+evaluation+as+new+drug+targets&doi=10.4155%2Ffmc-2016-0207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudokinases: update on their functions and evaluation as new drug targets</span></div><div class="casAuthors">Byrne, Dominic P.; Foulkes, Daniel M.; Eyers, Patrick A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">245-265</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The pseudokinase complement of the human kinase superfamily consists of approx. 60 signaling proteins, which lacks one or more of the amino acids typically required to correctly align ATP and metal ions, and phosphorylate protein substrates.  Recent studies in the pseudokinase field have begun to expose the biol. relevance of pseudokinases, which are now thought to perform a diverse range of physiol. roles and are connected to a multitude of human diseases, including cancer.  In this review, we discuss how and why members of the 'pseudokinome' represent important new targets for drug discovery, and describe how knowledge of protein structure and function provides informative clues to help guide the rational chem. design or repurposing of inhibitors to target pseudokinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqInUA03dIObVg90H21EOLACvtfcHk0liJjWLCyYx3HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslWnurg%253D&md5=290fd229e4f7e5fefc308b2fae35881d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0207%26sid%3Dliteratum%253Aachs%26aulast%3DByrne%26aufirst%3DD.%2BP.%26aulast%3DFoulkes%26aufirst%3DD.%2BM.%26aulast%3DEyers%26aufirst%3DP.%2BA.%26atitle%3DPseudokinases%253A%2520update%2520on%2520their%2520functions%2520and%2520evaluation%2520as%2520new%2520drug%2520targets%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D245%26epage%3D265%26doi%3D10.4155%2Ffmc-2016-0207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raundhal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaurio, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konrad, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettigole, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowley, M. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerliani, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenkov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motorykin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfredi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamarin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holcomb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinovich, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glimcher, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubillos-Ruiz, J. R.</span></span> <span> </span><span class="NLM_article-title">IRE1α-XBP1 controls T cell function in ovarian cancer by regulating mitochondrial activity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>562</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0597-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1038%2Fs41586-018-0597-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=30305738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=562&publication_year=2018&pages=423-428&author=M.+Songauthor=T.+A.+Sandovalauthor=C.+S.+Chaeauthor=S.+Chopraauthor=C.+Tanauthor=M.+R.+Rutkowskiauthor=M.+Raundhalauthor=R.+A.+Chaurioauthor=K.+K.+Payneauthor=C.+Konradauthor=S.+E.+Bettigoleauthor=H.+R.+Shinauthor=M.+J.+P.+Crowleyauthor=J.+P.+Cerlianiauthor=A.+V.+Kossenkovauthor=I.+Motorykinauthor=S.+Zhangauthor=G.+Manfrediauthor=D.+Zamarinauthor=K.+Holcombauthor=P.+C.+Rodriguezauthor=G.+A.+Rabinovichauthor=J.+R.+Conejo-Garciaauthor=L.+H.+Glimcherauthor=J.+R.+Cubillos-Ruiz&title=IRE1%CE%B1-XBP1+controls+T+cell+function+in+ovarian+cancer+by+regulating+mitochondrial+activity&doi=10.1038%2Fs41586-018-0597-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">IRE1α-XBP1 controls T cell function in ovarian cancer by regulating mitochondrial activity</span></div><div class="casAuthors">Song, Minkyung; Sandoval, Tito A.; Chae, Chang-Suk; Chopra, Sahil; Tan, Chen; Rutkowski, Melanie R.; Raundhal, Mahesh; Chaurio, Ricardo A.; Payne, Kyle K.; Konrad, Csaba; Bettigole, Sarah E.; Shin, Hee Rae; Crowley, Michael J. P.; Cerliani, Juan P.; Kossenkov, Andrew V.; Motorykin, Ievgen; Zhang, Sheng; Manfredi, Giovanni; Zamarin, Dmitriy; Holcomb, Kevin; Rodriguez, Paulo C.; Rabinovich, Gabriel A.; Conejo-Garcia, Jose R.; Glimcher, Laurie H.; Cubillos-Ruiz, Juan R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">562</span>
        (<span class="NLM_cas:issue">7727</span>),
    <span class="NLM_cas:pages">423-428</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Tumors evade immune control by creating hostile microenvironments that perturb T cell metab. and effector function.  However, it remains unclear how intra-tumoral T cells integrate and interpret metabolic stress signals.  Here the authors report that ovarian cancer -an aggressive malignancy that is refractory to std. treatments and current immunotherapies -induces endoplasmic reticulum stress and activates the IRE1α-XBP1 arm of the unfolded protein response in T cells to control their mitochondrial respiration and anti-tumor function.  In T cells isolated from specimens collected from patients with ovarian cancer, up-regulation of XBP1 was assocd. with decreased infiltration of T cells into tumors and with reduced IFNG mRNA expression.  Malignant ascites fluid obtained from patients with ovarian cancer inhibited glucose uptake and caused N-linked protein glycosylation defects in T cells, which triggered IRE1α-XBP1 activation that suppressed mitochondrial activity and IFNγ prodn.  Mechanistically, induction of XBP1 regulated the abundance of glutamine carriers and thus limited the influx of glutamine that is necessary to sustain mitochondrial respiration in T cells under glucose-deprived conditions.  Restoring N-linked protein glycosylation, abrogating IRE1α-XBP1 activation or enforcing expression of glutamine transporters enhanced mitochondrial respiration in human T cells exposed to ovarian cancer ascites.  XBP1-deficient T cells in the metastatic ovarian cancer milieu exhibited global transcriptional reprogramming and improved effector capacity.  Accordingly, mice that bear ovarian cancer and lack XBP1 selectively in T cells demonstrate superior anti-tumor immunity, delayed malignant progression and increased overall survival.  Controlling endoplasmic reticulum stress or targeting IRE1α-XBP1 signaling may help to restore the metabolic fitness and anti-tumor capacity of T cells in cancer hosts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0WsJG_HHAh7Vg90H21EOLACvtfcHk0lirN1D7gZnrlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtLrK&md5=f6280f187633b2498f2172703cc19768</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0597-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0597-x%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DM.%26aulast%3DSandoval%26aufirst%3DT.%2BA.%26aulast%3DChae%26aufirst%3DC.%2BS.%26aulast%3DChopra%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DRutkowski%26aufirst%3DM.%2BR.%26aulast%3DRaundhal%26aufirst%3DM.%26aulast%3DChaurio%26aufirst%3DR.%2BA.%26aulast%3DPayne%26aufirst%3DK.%2BK.%26aulast%3DKonrad%26aufirst%3DC.%26aulast%3DBettigole%26aufirst%3DS.%2BE.%26aulast%3DShin%26aufirst%3DH.%2BR.%26aulast%3DCrowley%26aufirst%3DM.%2BJ.%2BP.%26aulast%3DCerliani%26aufirst%3DJ.%2BP.%26aulast%3DKossenkov%26aufirst%3DA.%2BV.%26aulast%3DMotorykin%26aufirst%3DI.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DManfredi%26aufirst%3DG.%26aulast%3DZamarin%26aufirst%3DD.%26aulast%3DHolcomb%26aufirst%3DK.%26aulast%3DRodriguez%26aufirst%3DP.%2BC.%26aulast%3DRabinovich%26aufirst%3DG.%2BA.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DGlimcher%26aufirst%3DL.%2BH.%26aulast%3DCubillos-Ruiz%26aufirst%3DJ.%2BR.%26atitle%3DIRE1%25CE%25B1-XBP1%2520controls%2520T%2520cell%2520function%2520in%2520ovarian%2520cancer%2520by%2520regulating%2520mitochondrial%2520activity%26jtitle%3DNature%26date%3D2018%26volume%3D562%26spage%3D423%26epage%3D428%26doi%3D10.1038%2Fs41586-018-0597-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lirN1D7gZnrlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liiT9FpTSYXgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liiT9FpTSYXgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljp-GBk8xb_fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span> <i>Maestro</i>, version 9.3; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Maestro%2C+version+9.3%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span> <i>Glide</i>, version 5.8; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide%2C+version+5.8%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span> <i>LigPrep</i>, version 2.5; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LigPrep%2C+version+2.5%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhegalova, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezin, M. Y.</span></span> <span> </span><span class="NLM_article-title">Synthesis of nitric oxide probes with fluorescence lifetime sensitivity</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">8228</span>– <span class="NLM_lpage">8234</span>, <span class="refDoi"> DOI: 10.1039/c3ob41498a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=10.1039%2Fc3ob41498a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=24166035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslOmsbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=8228-8234&author=N.+G.+Zhegalovaauthor=G.+Gonzalesauthor=M.+Y.+Berezin&title=Synthesis+of+nitric+oxide+probes+with+fluorescence+lifetime+sensitivity&doi=10.1039%2Fc3ob41498a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of nitric oxide probes with fluorescence lifetime sensitivity</span></div><div class="casAuthors">Zhegalova, Natalia G.; Gonzales, Garrett; Berezin, Mikhail Y.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">8228-8234</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors present the rationale, synthesis and evaluation of the 1st activatable fluorescent probe that uses fluorescence lifetime change for detection of nitric oxide.  The new probe I (R = R' = H) features a near-IR polymethine skeleton with a diaminobenzene functionality incorporated into the meso-position.  The probe is partially quenched, and upon reaction with nitric oxide shows an increase in the fluorescence lifetime from 1.08 ns to 1.24 ns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUAAtEJ72VTrVg90H21EOLACvtfcHk0ljp-GBk8xb_fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslOmsbzF&md5=62e199c9a6b4bc72464a983f72ad3370</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2Fc3ob41498a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob41498a%26sid%3Dliteratum%253Aachs%26aulast%3DZhegalova%26aufirst%3DN.%2BG.%26aulast%3DGonzales%26aufirst%3DG.%26aulast%3DBerezin%26aufirst%3DM.%2BY.%26atitle%3DSynthesis%2520of%2520nitric%2520oxide%2520probes%2520with%2520fluorescence%2520lifetime%2520sensitivity%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D8228%26epage%3D8234%26doi%3D10.1039%2Fc3ob41498a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z7H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z7H','PDB','4Z7H'); return false;">PDB: 4Z7H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1','PDB','6HX1'); return false;">PDB: 6HX1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4U6R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4U6R','PDB','4U6R'); return false;">PDB: 4U6R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YZ9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YZ9','PDB','4YZ9'); return false;">PDB: 4YZ9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HV0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HV0','PDB','6HV0'); return false;">PDB: 6HV0</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i41"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01721">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_17777"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01721">10.1021/acs.jmedchem.8b01721</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Mass spectrometry characterization of IRE1α proteins; schematic of the fluorescence-labeled stem-loop RNA used to measure inhibition of IRE1α RNase activity; selectivity of compounds <b>26</b> and <b>31</b> for IRE1α vs IRE1β kinase inhibition; kinase selectivity profiles of compounds <b>2</b>, <b>26</b>, and <b>31</b>; chemical structure and IRE1α inhibition data for <b>33</b>; crystal structure of <b>33</b>-IRE1α; model of <b>26</b> bound to IRE1α; microscopic images showing inhibition of stress-induced GFP-IRE1α foci formation in HEK293 cells by <b>26</b> and <b>31</b>; titrations of inhibition of stress-induced splicing of XBP1-Luc reporter mRNA in HEK293 cells by <b>26</b> and <b>31;</b> titrations of inhibition of stress-induced XBP1s mRNA expression in NCI-H929 cells by <b>26</b> and <b>31</b>; quantification of inhibition of pS724 IRE1α relative to total IRE1α using capillary electrophoresis immunoassay in H929 cells by <b>26</b> and <b>31</b>; microscopic images showing inhibition of stress-induced expression of XBP1s in NCI-H929 cells by <b>26</b> and <b>31</b>; titrations of the inhibition of IRE1α autophosphorylation, IRE1α ATP-site binding, and IRE1α RNase by compounds <b>2</b>, <b>26</b>, and <b>31</b> in in vitro assays; selectivity profiles of <b>2</b>, <b>26</b>, and <b>31</b> tested against 455 human kinases at 1000 nM; HPLC purities of compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b>; experimental method for the generation of an HEK293TN XBP1 luciferase reporter cell line; experimental methods to determine inhibition of an XBP1 luciferase reporter in HEK293TN cells and assess compound cytotoxicity to HEK293TN cells; experimental method for the quantification of XBP1s expression in NCI-H929 cells by immunofluorescence; experimental method for the quantification of XBP1s and DNAJB9 mRNA expression in NCI-H929 cells by qPCR; experimental method for the quantification of the inhibition of IRE1α-GFP oligomerization in T-Rex293-IRE1-3F6HGFP cells; experimental methods for the quantification of total IRE1α and pIRE1α expression in NCI-H929 cells; experimental method for the determination of the crystal structure of <b>33</b>-IRE1α; summary of crystallographic data for <b>2</b>-IRE1α and <b>33</b>-IRE1α; experimental methods for the synthesis of <b>33</b>; copies of <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b>; copies of HPLC traces for compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b>; and molecular formula strings for compounds <b>2</b>, <b>5</b>–<b>12</b>, and <b>19</b>–<b>33</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_001.pdf">jm8b01721_si_001.pdf (5.62 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01721/suppl_file/jm8b01721_si_002.csv">jm8b01721_si_002.csv (2.01 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID Codes: <b>2</b>-IRE1α, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HX1">6HX1</a>; <b>33</b>-IRE1α, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HV0">6HV0</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01721&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01721%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-5%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01721" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b3be2a62198b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
