
<html lang="en"     class="pb-page"  data-request-id="0f6eb796-f650-4036-813e-baab96514de0"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.9b00742;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2019.62.issue-24;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Synthesis and Pharmacological Evaluation of Triazolopyrimidinone Derivatives as Noncompetitive, Intracellular Antagonists for CC Chemokine Receptors 2 and 5" /></meta><meta name="dc.Creator" content="Natalia V.  Ortiz Zacarías" /></meta><meta name="dc.Creator" content="Jacobus P. D.  van Veldhoven" /></meta><meta name="dc.Creator" content="Lisa S.  den Hollander" /></meta><meta name="dc.Creator" content="Burak  Dogan" /></meta><meta name="dc.Creator" content="Joseph  Openy" /></meta><meta name="dc.Creator" content="Ya-Yun  Hsiao" /></meta><meta name="dc.Creator" content="Eelke B.  Lenselink" /></meta><meta name="dc.Creator" content="Laura H.  Heitman" /></meta><meta name="dc.Creator" content="Adriaan P.  IJzerman" /></meta><meta name="dc.Description" content="CC chemokine receptors 2 (CCR2) and 5 (CCR5) are involved in many inflammatory diseases; however, most CCR2 and CCR5 clinical candidates have been unsuccessful. (Pre)clinical evidence suggests that..." /></meta><meta name="Description" content="CC chemokine receptors 2 (CCR2) and 5 (CCR5) are involved in many inflammatory diseases; however, most CCR2 and CCR5 clinical candidates have been unsuccessful. (Pre)clinical evidence suggests that..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 19, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00742" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00742" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00742" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00742" /></link>
        
    
    

<title>Synthesis and Pharmacological Evaluation of Triazolopyrimidinone Derivatives as Noncompetitive, Intracellular Antagonists for CC Chemokine Receptors 2 and 5 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00742" /></meta><meta property="og:title" content="Synthesis and Pharmacological Evaluation of Triazolopyrimidinone Derivatives as Noncompetitive, Intracellular Antagonists for CC Chemokine Receptors 2 and 5" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0009.jpeg" /></meta><meta property="og:description" content="CC chemokine receptors 2 (CCR2) and 5 (CCR5) are involved in many inflammatory diseases; however, most CCR2 and CCR5 clinical candidates have been unsuccessful. (Pre)clinical evidence suggests that dual CCR2/CCR5 inhibition might be more effective in the treatment of such multifactorial diseases. In this regard, the highly conserved intracellular binding site in chemokine receptors provides a new avenue for the design of multitarget ligands. In this study, we synthesized and evaluated the biological activity of a series of triazolopyrimidinone derivatives in CCR2 and CCR5. Radioligand binding assays first showed that they bind to the intracellular site of CCR2, and in combination with functional assays on CCR5, we explored structure–affinity/activity relationships in both receptors. Although most compounds were CCR2-selective, 39 and 43 inhibited β-arrestin recruitment in CCR5 with high potency. Moreover, these compounds displayed an insurmountable mechanism of inhibition in both receptors, which holds promise for improved efficacy in inflammatory diseases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00742"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00742">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00742&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00742&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00742&amp;href=/doi/10.1021/acs.jmedchem.9b00742" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 11035-11053</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00753" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00857" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthesis and Pharmacological Evaluation of Triazolopyrimidinone Derivatives as Noncompetitive, Intracellular Antagonists for CC Chemokine Receptors 2 and 5</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Natalia V. Ortiz Zacarías</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Natalia V. Ortiz Zacarías</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Natalia+V.++Ortiz+Zacar%C3%ADas">Natalia V. Ortiz Zacarías</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jacobus P. D. van Veldhoven</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jacobus P. D. van Veldhoven</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jacobus+P.+D.++van+Veldhoven">Jacobus P. D. van Veldhoven</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lisa S. den Hollander</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lisa S. den Hollander</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lisa+S.++den+Hollander">Lisa S. den Hollander</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Burak Dogan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Burak Dogan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Burak++Dogan">Burak Dogan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph Openy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph Openy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph++Openy">Joseph Openy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ya-Yun Hsiao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ya-Yun Hsiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ya-Yun++Hsiao">Ya-Yun Hsiao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eelke B. Lenselink</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eelke B. Lenselink</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eelke+B.++Lenselink">Eelke B. Lenselink</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura H. Heitman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura H. Heitman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura+H.++Heitman">Laura H. Heitman</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Adriaan P. IJzerman</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adriaan P. IJzerman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Drug Discovery and Safety, Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +31 (0) 71 527 4651. E-mail: <a href="/cdn-cgi/l/email-protection#96fffcecf3e4fbf7f8d6faf7f5f2e4b8faf3fff2f3f8e3f8ffe0b8f8fa"><span class="__cf_email__" data-cfemail="8be2e1f1eef9e6eae5cbe7eae8eff9a5e7eee2efeee5fee5e2fda5e5e7">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adriaan+P.++IJzerman">Adriaan P. IJzerman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1182-2259" title="Orcid link">http://orcid.org/0000-0002-1182-2259</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00742&amp;href=/doi/10.1021%2Facs.jmedchem.9b00742" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 11035–11053</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 19, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 May 2019</li><li><span class="item_label"><b>Published</b> online</span>19 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00742" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00742</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc text">with CC-BY-NC-ND<a href="/page/policy/authorchoice_ccbyncnd_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00742"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1572</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00742" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthesis and Pharmacological Evaluation of Triazolopyrimidinone Derivatives as Noncompetitive, Intracellular Antagonists for CC Chemokine Receptors 2 and 5&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Natalia&quot;,&quot;last_name&quot;:&quot;V. Ortiz Zacarías&quot;},{&quot;first_name&quot;:&quot;Jacobus&quot;,&quot;last_name&quot;:&quot;P. D. van Veldhoven&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;S. den Hollander&quot;},{&quot;first_name&quot;:&quot;Burak&quot;,&quot;last_name&quot;:&quot;Dogan&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;Openy&quot;},{&quot;first_name&quot;:&quot;Ya-Yun&quot;,&quot;last_name&quot;:&quot;Hsiao&quot;},{&quot;first_name&quot;:&quot;Eelke&quot;,&quot;last_name&quot;:&quot;B. Lenselink&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;H. Heitman&quot;},{&quot;first_name&quot;:&quot;Adriaan&quot;,&quot;last_name&quot;:&quot;P. IJzerman&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;11035-11053&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00742&quot;},&quot;abstract&quot;:&quot;CC chemokine receptors 2 (CCR2) and 5 (CCR5) are involved in many inflammatory diseases; however, most CCR2 and CCR5 clinical candidates have been unsuccessful. (Pre)clinical evidence suggests that dual CCR2/CCR5 inhibition might be more effective in the treatment of such multifactorial diseases. In this regard, the highly conserved intracellular binding site in chemokine receptors provides a new avenue for the design of multitarget ligands. In this study, we synthesized and evaluated the biological activity of a series of triazolopyrimidinone derivatives in CCR2 and CCR5. Radioligand binding assays first showed that they bind to the intracellular site of CCR2, and in combination with functional assays on CCR5, we explored structure–affinity/activity relationships in both receptors. Although most compounds were CCR2-selective, 39 and 43 inhibited β-arrestin recruitment in CCR5 with high potency. Moreover, these compounds displayed an insurmountable mechanism of inhibition in both receptors, which holds promis&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00742&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00742" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00742&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00742" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00742&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00742" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00742&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00742&amp;href=/doi/10.1021/acs.jmedchem.9b00742" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00742" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00742" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00742%26sid%3Dliteratum%253Aachs%26pmid%3D31742400%26genre%3Darticle%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26date%3D2019%26atitle%3DSynthesis%2Band%2BPharmacological%2BEvaluation%2Bof%2BTriazolopyrimidinone%2BDerivatives%2Bas%2BNoncompetitive%252C%2BIntracellular%2BAntagonists%2Bfor%2BCC%2BChemokine%2BReceptors%2B2%2Band%2B5%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D24%26spage%3D11035%26epage%3D11053%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (7)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/jmcmar.2019.62.issue-24/20191226/jmcmar.2019.62.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">CC chemokine receptors 2 (CCR2) and 5 (CCR5) are involved in many inflammatory diseases; however, most CCR2 and CCR5 clinical candidates have been unsuccessful. (Pre)clinical evidence suggests that dual CCR2/CCR5 inhibition might be more effective in the treatment of such multifactorial diseases. In this regard, the highly conserved intracellular binding site in chemokine receptors provides a new avenue for the design of multitarget ligands. In this study, we synthesized and evaluated the biological activity of a series of triazolopyrimidinone derivatives in CCR2 and CCR5. Radioligand binding assays first showed that they bind to the intracellular site of CCR2, and in combination with functional assays on CCR5, we explored structure–affinity/activity relationships in both receptors. Although most compounds were CCR2-selective, <b>39</b> and <b>43</b> inhibited β-arrestin recruitment in CCR5 with high potency. Moreover, these compounds displayed an insurmountable mechanism of inhibition in both receptors, which holds promise for improved efficacy in inflammatory diseases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">CC chemokine receptors 2 (CCR2) and 5 (CCR5) are two membrane-bound G protein-coupled receptors (GPCRs), which belong to the subfamily of chemokine receptors. Chemokine receptors are widely expressed in leukocytes, and thus, they regulate different homeostatic and inflammatory leukocyte functions upon interaction with their endogenous chemokines.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> In general, chemokine receptors interact with multiple endogenous chemokines, such as CCL2, CCL7, and CCL8 in the case of CCR2, and CCL3, CCL4, and CCL5 in the case of CCR5.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Furthermore, most chemokines can interact with multiple chemokine receptors, allowing for a very complex and fine-tuned system.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Dysregulation of this system has been linked to the development of several pathophysiological conditions. For example, both CCR2 and CCR5 have been implicated in many inflammatory and immune diseases such as rheumatoid arthritis, multiple sclerosis, atherosclerosis, diabetes mellitus, and psoriasis,<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> rendering these proteins attractive targets for the pharmaceutical industry. As a result, many efforts have been made to bring CCR2 and CCR5 small-molecule antagonists into the clinic although with limited success. Only maraviroc, an HIV-1 entry inhibitor selectively targeting CCR5, has been approved by the FDA and EMA,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> while all other drug candidates have failed in clinical trials.</div><div class="NLM_p">Recently, it has been suggested that the development of multitarget drugs (designed to interact with multiple receptors) represents a more effective approach in the treatment of complex multifactorial diseases.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Thus, dual targeting of CCR2 and CCR5 emerges as a potentially more efficacious strategy in diseases where both receptors are involved. Indeed, combined CCR2/CCR5 inhibition has resulted in beneficial effects in several preclinical disease models and clinical studies, further supporting the use of dual antagonists.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10−12)</a> In this regard, several antagonists with dual CCR2/CCR5 activity have been reported in the past years, including the first dual antagonist TAK-779 and the clinical candidate cenicriviroc.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> All of these antagonists bind to the extracellular region of CCR2 and CCR5, in a site overlapping with the chemokine’s binding pocket.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Yet the crystal structures of CCR2 and CCR9 have demonstrated that chemokine receptors can also be targeted with intracellular allosteric modulators.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> These intracellular ligands offer a number of advantages, such as noncompetitive binding and, as a consequence, insurmountable inhibition, which is particularly important due to the high local concentration of chemokines during pathological conditions.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> In addition, the high conservation of this intracellular site allows for the design of multitarget antagonists.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> Several high-affinity intracellular ligands have been already identified for CCR2<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> but not for CCR5, although intracellular compounds developed for CCR2 or CCR4 have been reported to bind CCR5 with much lower potency.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a></div><div class="NLM_p">In the current study we first report that previously patented CCR2 antagonists with a triazolopyrimidinone scaffold, such as compound <b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>),<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> bind to the intracellular site of the receptor with high affinity. In addition, we show that this compound is able to inhibit CCR5 with moderate activity, suggesting a potential dual CCR2/CCR5 activity for this class of compounds. Thus, a series of novel and previously reported triazolopyrimidinone derivatives were synthesized according to published methods<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> in order to obtain structure–affinity/activity relationships (SARs) in both CCR2 and CCR5. Radioligand binding assays and functional assays were used to evaluate their affinity toward CCR2 and activity toward CCR5. In addition, characterization of two selected compounds (<b>39</b> and <b>43</b>) in a [<sup>35</sup>S]GTPγS binding assay demonstrated that these compounds inhibit both receptors in a noncompetitive, insurmountable manner. Finally, selected compounds were docked into the CCR2 crystal structure in order to shed light on the binding mode of these derivatives, in comparison to that of the crystallized CCR2-RA-[<i>R</i>].<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In summary, our findings provide some insight on the CCR2/CCR5 selectivity profile of triazolopyrimidinone derivatives, as well as on the structural requirements for the design of multitarget or selective intracellular ligands for these receptors.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of the orthosteric CCR2/CCR5 antagonist TAK-779 and the CCR2 intracellular ligands CCR2-RA-[<i>R</i>], SD-24, JNJ-27141491 and the triazolopyrimidinone derivative <b>8</b>. [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] was used in radioligand binding assays for CCR2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07123" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07123" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">Triazolo-pyrimidinone derivatives <b>6</b>–<b>43</b> were synthesized using a three-step synthesis approach as described by Bengtsson et al.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). First, if not commercially available, the β-keto esters <b>1a</b>–<b>n</b> were synthesized from ethyl acetoacetate <b>1a</b> and the respective bromo- or iodoalkanes <b>2f</b>–<b>h</b>,<b>j</b>,<b>k</b> or benzyl bromide <b>2n</b>. Benzylation of the β-keto esters <b>1a</b>–<b>n</b> with the corresponding R<sup>1</sup>-substituted benzyl bromides (<b>3a</b>–<b>v</b>), at reflux, resulted in a series of benzylated β-keto esters <b>4aa</b>–<b>na</b>, <b>4bb</b>–<b>bq</b>, <b>4eq</b>–<b>ev</b> in yields between 8% and 97% (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Table S1</a>). Finally a cyclization reaction of the benzylated β-keto esters <b>4aa</b>–<b>na</b>, <b>4bb</b>–<b>bq</b>, <b>4eq</b>–<b>ev</b> with the commercially available 3,5-diaminotriazole <b>5c</b> in ionic liquid BMIM-PF6 (1-butyl-3-methylimidazolium hexafluorophosphate) at 200 °C under microwave irradiation resulted in final compounds <b>6</b>, <b>9</b>–<b>43</b> in yields ranging from 4% to 83%. Final compound <b>7</b> (R<sup>2</sup> = H) was synthesized using H<sub>3</sub>PO<sub>4</sub> in ethanol conditions and <b>8</b> (R<sup>2</sup> = Me) in <i>p</i>-toluenesulfonic acid monohydrate conditions.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Scheme of the Triazolopyrimidinone Derivatives <b>6</b>–<b>43</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) NaH, <i>n</i>-BuLi, THF, overnight, 0 °C to rt (<b>1a</b>–<b>e</b>,<b>i</b>,<b>l</b>,<b>m</b> were commercially available); (ii) DIPEA, LiCl, THF, reflux, overnight; (iii) (<b>8</b>–<b>43</b>, R<sup>2</sup> = NH<sub>2</sub>) BMIM-PF6, 200 °C, 1 h or (<b>6</b>, R<sup>2</sup> = H<b>)</b> H<sub>3</sub>PO<sub>4</sub>, EtOH, 170 °C, 10 h or (<b>7</b>, R<sup>2</sup> = Me) <i>p</i>-toluenesulfonic acid monohydrate, 180 °C, 30 min.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Biology</h3><div class="NLM_p">We have previously identified several CCR2 intracellular ligands belonging to different chemical scaffolds, such as CCR2-RA-[<i>R</i>], SD-24, and JNJ-27141491 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> In contrast to CCR2 orthosteric ligands, these intracellular ligands lack a basic nitrogen and have lower molecular weights, unsaturated systems with haloarenes, and acidic groups capable of forming hydrogen bonds.<a onclick="showRef(event, 'ref18 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref20">(18,20)</a> Other CCR2 antagonists with similar features have been described in the literature, including the triazolo- or pyrazolopyrimidinone derivatives described in two different patents.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> To test whether they also bind to the intracellular site of the receptor, we synthesized “example 1” from the patent by Bengtsson et al.,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> corresponding to the triazolopyrimidinone derivative <b>8</b> in our study (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Using a [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] binding assay as previously described,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> we found that compound <b>8</b> fully displaced [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] binding from U2OS cells stably expressing hCCR2b (U2OS-CCR2) with high affinity and a pseudo-Hill slope (<i>n</i><sub>H</sub>) close to unity, indicating a competitive interaction with [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] for the intracellular binding site. <b>8</b> displaced [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] with a p<i>K</i><sub>i</sub> of 8.90 ± 0.04 (<i>K</i><sub>i</sub> = 1.3 nM, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), consistent with its previously reported activity in a CCR2 calcium flux assay (IC<sub>50</sub> = 16 nM).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of ligands in U2OS-CCR2 and U2OS-CCR5. (a) [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] displacement by increasing concentrations of triazolopyrimidinone derivatives <b>8</b>, <b>39</b>, and <b>43</b> in U2OS-CCR2 at 25 °C. Data are normalized to specific binding in the absence of compound (set as 100%). (b) Inhibition of CCL2-stimulated β-arrestin recruitment in U2OS-CCR2 by increasing concentrations of compounds <b>8</b>, <b>39</b>, and <b>43</b>, after stimulation with an EC<sub>80</sub> concentration of CCL2 (set as 100%). (c) Inhibition of CCL3-stimulated β-arrestin recruitment in U2OS-CCR5 by increasing concentrations of compounds <b>8</b>, <b>39</b>, and <b>43</b>, after stimulation with an EC<sub>80</sub> concentration of CCL3 (set as 100%). All data are from single, representative experiments performed in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Characterization of Compounds <b>6</b>–<b>23</b> in hCCR2 and hCCR5<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0006.gif" alt="" id="GRAPHIC-d7e683-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">hCCR2</th><th class="colsep0 rowsep0" align="center">hCCR5</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">p<i>K</i><sub>i</sub> ± SEM (<i>K</i><sub>i</sub>, nM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">pIC<sub>50</sub> ± SEM (IC<sub>50</sub>, nM) or inhibition at 1 μM (%)<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">8.76 ± 0.01 (1.7)</td><td class="colsep0 rowsep0" align="left">60%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">8.46 ± 0.05 (3.5)</td><td class="colsep0 rowsep0" align="left">35%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.90 ± 0.04 (1.3)</td><td class="colsep0 rowsep0" align="left">6.24 ± 0.004 (571)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.27 ± 0.10 (5.9)</td><td class="colsep0 rowsep0" align="left">35%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">2-Me</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">7.81 ± 0.05 (15.7)</td><td class="colsep0 rowsep0" align="left">28%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">2-Cl</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">7.84 ± 0.03 (14.5)</td><td class="colsep0 rowsep0" align="left">27%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">2-OMe</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">6.98 ± 0.04 (104.6)</td><td class="colsep0 rowsep0" align="left">–20%<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">3-Me</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.61 ± 0.03 (2.5)</td><td class="colsep0 rowsep0" align="left">62%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">3-F</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.53 ± 0.18 (3.4)</td><td class="colsep0 rowsep0" align="left">43%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">3-Br</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">9.08 ± 0.06 (0.9)</td><td class="colsep0 rowsep0" align="left">60%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">3-I</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">9.06 ± 0.02 (0.9)</td><td class="colsep0 rowsep0" align="left">66%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">7.89 ± 0.07 (13.0)</td><td class="colsep0 rowsep0" align="left">–27%<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.26 ± 0.09 (5.9)</td><td class="colsep0 rowsep0" align="left">36%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">4-Me</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.46 ± 0.03 (3.5)</td><td class="colsep0 rowsep0" align="left">–57%<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">4-F</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.39 ± 0.03 (4.1)</td><td class="colsep0 rowsep0" align="left">31%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.74 ± 0.05 (1.8)</td><td class="colsep0 rowsep0" align="left">31%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">4-Br</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">8.84 ± 0.02 (1.5)</td><td class="colsep0 rowsep0" align="left">14%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">4-OMe</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">7.68 ± 0.05 (20.9)</td><td class="colsep0 rowsep0" align="left">–28%</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data are presented as the mean p<i>K</i><sub>i</sub>/pIC<sub>50</sub> ± standard error of the mean (SEM) and mean <i>K</i><sub>i</sub>/IC<sub>50</sub> (nM) of at least three independent experiments performed in duplicate.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">p<i>K</i><sub>i</sub> values from the displacement of ∼6 nM [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] from U2OS cells stably expressing CCR2, at 25 °C.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Percent inhibition of β-arrestin recruitment in U2OS cells stably expressing CCR5 by 1 μM compound, in the presence of CCL3 (pEC<sub>80</sub> = 7.9). pIC<sub>50</sub> values were determined for compounds displaying more than 70% inhibition. % Inhibition values are presented as means values of at least two independent experiments, performed in duplicate.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">No inhibition was observed at the concentration of 1 μM; instead some CCL3 stimulation was measured.</p></div></div></div><div class="NLM_p">Previous studies have shown that some of these intracellular ligands are able to bind and inhibit multiple chemokine receptors, enabling the design of selective and multitarget inhibitors.<a onclick="showRef(event, 'ref19 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref21 ref22">(19,21,22)</a> In this regard, CCR5 is the closest homolog to CCR2, with >90% sequence similarity of their intracellular binding pockets. From the main interactions of CCR2-RA-[<i>R</i>] to CCR2, only Val244<sup>6×36</sup> is exchanged to Leu236<sup>6×36</sup> in CCR5<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (residues named according to structure-based Ballesteros—Weinstein nomenclature<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>). Thus, we investigated whether compound <b>8</b> is also able to inhibit the highly homologous CCR5. However, the much lower affinity of [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] for CCR5 compared to CCR2 hindered us from performing radioligand binding assays.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Thus, we assessed the CCR5 activity of <b>8</b> with a functional β-arrestin recruitment assay after stimulation with CCL3, one of the endogenous agonists of CCR5. For this assay, we also included the intracellular ligands CCR2-RA-[<i>R</i>], SD-24 and JNJ-27141491, as well as the CCR2/CCR5 orthosteric antagonist TAK-779 as a positive control (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), since it is a potent CCR5 antagonist in a variety of functional assays.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a></div><div class="NLM_p">In this assay, CCL3 induced β-arrestin recruitment to U2OS cells stably expressing hCCR5 (U2OS-CCR5) with a pEC<sub>50</sub> of 8.3 ± 0.08 (6 nM) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S1a</a>), similar to values reported in the literature.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> As expected, TAK-779 was able to completely inhibit β-arrestin recruitment induced by an EC<sub>80</sub> concentration of CCL3 (pEC<sub>80</sub> = 7.9 ± 0.08), when tested at a single concentration of 1 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S1b</a>). In contrast, none of the intracellular ligands were able to fully inhibit CCL3-induced β-arrestin recruitment to the same level as TAK-779; in fact, only compound <b>8</b> displayed more than 70% inhibition when tested at 1 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S1b</a>), while CCR2-RA-[<i>R</i>], SD-24, and JNJ-27141491 led to approximately 50% inhibition or less at the same concentration of 1 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S1b</a>). Consistent with this low inhibition in CCR5, it was previously shown that CCR2-RA-[<i>R</i>], JNJ-27141491, and SD-24 inhibited inositol phosphate (IP) formation in CCR5 with 7- to 22-fold lower potency compared to CCR2 inhibition.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Preincubation of U2OS-CCR5 cells with increasing concentrations of TAK-779, before exposure to CCL3, resulted in an inhibitory potency (IC<sub>50</sub>) of 6 nM, consistent with previously reported values (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Table S2</a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Also in agreement with a previous study,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> the reference intracellular ligand CCR2-RA-[<i>R</i>] inhibited CCL3-induced β-arrestin recruitment with an IC<sub>50</sub> value of 703 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Table S2</a>). Moreover, while TAK-779 inhibited CCL3-induced β-arrestin recruitment with a pseudo-Hill slope close to unity (<i>n</i><sub>H</sub> = −1.1), CCR2-RA-[<i>R</i>] inhibition showed a significantly higher Hill slope (<i>n</i><sub>H</sub> = −2.4), indicative of two different binding sites for CCL3 and CCR2-RA-[<i>R</i>] (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Table S2</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">As compound <b>8</b> was the best CCR5 inhibitor in this assay, displaying an IC<sub>50</sub> value of 571 nM and a Hill slope of −2.2 ± 0.3 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), we then synthesized several triazolopyrimidinone derivatives to explore their structure–affinity/activity relationships (SARs) in CCR2 and CCR5. All synthesized triazolopyrimidinone derivatives were evaluated in [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] binding assays to determine their binding affinity for CCR2 and in β-arrestin recruitment assays to determine their activity toward CCR5 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). In CCR5, compounds were first screened at a concentration of 1 μM, as we were only interested in dual-targeting compounds with moderate to high potencies (IC<sub>50</sub> < 1 μM). For the same reason, only those that displayed >70% inhibition at this concentration were further evaluated in a concentration–inhibition curve to determine their potency. For better comparison, compounds <b>8</b>, <b>39</b>, and <b>43</b> were also tested in a CCR2 β-arrestin recruitment assay as previously described (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Finally, we determined the mechanism of inhibition of <b>39</b> and <b>43</b> in both CCR2 and CCR5 using a [<sup>35</sup>S]GTPγS binding assay (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of compounds <b>39</b> and <b>43</b> as insurmountable, negative allosteric modulators using a [<sup>35</sup>S]GTPγS binding assay in hCCR2 and hCCR5. Effect of increasing concentrations of <b>39</b> and <b>43</b> in a CCL2-stimulated [<sup>35</sup>S]GTPγS binding in U2OS-CCR2 (a, b) or in a CCL3-stimulated [<sup>35</sup>S]GTPγS binding in U2OS-CCR5 (c, d), at 25 °C. Parameters obtained from the concentration–response curves (pEC<sub>50</sub>, <i>E</i><sub>max</sub>) are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Data are presented as mean ± SEM values of three experiments performed in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Affinity/Activity Relationships (SARs) in CCR2 and CCR5</h3><div class="NLM_p">Analysis of the triazolopyrimidinone derivatives started by modifying the amino group (R<sup>2</sup>) of the triazolo moiety (R<sup>2</sup>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compared to <b>8</b>, removing the amino group (<b>6</b>) resulted in a similar affinity toward CCR2, in agreement with the similar reported IC<sub>50</sub> values of approximately 20 nM for both compounds, when tested in a calcium flux assay.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> However, in CCR5 <b>6</b> displayed a lower potency, as the inhibition of CCL3-stimulated recruitment of β-arrestin decreased to 60%, compared to 76% inhibition by <b>8</b>. The introduction of a methyl group in R<sup>2</sup> (<b>7</b>) was less favorable for both receptors, as both affinity for CCR2 and activity to CCR5 were reduced compared to <b>8</b>. As compound <b>8</b> displayed the highest affinity/activity for both receptors, we decided to keep the amino group in R<sup>2</sup> and explore different phenyl substituents (R<sup>1</sup>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), taking <b>8</b> as the starting point.</div><div class="NLM_p">Compared to <b>8</b>, the unsubstituted <b>9</b> showed a 5-fold decrease in affinity toward CCR2, while in CCR5 it was only able to inhibit 35% of the receptor response at 1 μM. Next, we investigated the effect of several benzyl modifications, including the influence of different substituent positions (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In the case of CCR2, meta-substituted derivatives yielded the highest affinities in this series of compounds (<b>13</b>–<b>18</b>), whereas ortho-substituted derivatives yielded the lowest (<b>10</b>–<b>12</b>). None of the ortho-substitutions led to an improvement in affinity over <b>8</b> or the unsubstituted <b>9</b>. Introduction of a methyl (<b>10</b>) or a chloro (<b>11</b>) group in this position resulted in affinities lower than 10 nM, while the introduction of an electron-donating methoxy group further reduced the affinity to 105 nM (<b>12</b>), displaying the lowest CCR2 affinity in this series (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Moving the methyl group to meta (<b>13</b>) or para (<b>19</b>) position slightly improved the CCR2 binding affinity compared to <b>9</b>, achieving the highest affinity in meta position (<b>19</b>, 3 nM). Similarly, moving the methoxy group to meta or para position resulted in improved affinities following the meta > para > ortho order; however, the affinities remained lower than 10 nM (<b>17</b>, 13 nM; <b>23</b>, 21 nM), with no improvement over <b>9</b>. This is consistent with functional data reported in the patent by Bengtsson et al., where similar compounds with a methoxybenzyl moiety displayed a loss of CCR2 activity compared to the unsubstituted-phenyl analogue.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Substitution of the meta methoxy group by an electron-withdrawing CF<sub>3</sub> group resulted in improved affinity over <b>17</b> (<b>18</b>, 6 nM) but no improvement over the unsubstituted <b>9</b>.</div><div class="NLM_p">The effect of introducing different halogen groups was first investigated in the meta position. Overall, an increase in size and lipophilicity from fluoro to iodo resulted in improved binding affinities toward CCR2 (F, <b>14</b> < Cl, <b>8</b> < Br, <b>15</b> ≈ I, <b>16</b>). In fact, compounds <b>15</b> and <b>16</b> displayed the highest affinities in this series of derivatives (<b>15</b>, 0.8 nM; <b>16</b>, 0.9 nM). Moving the halogen substituents to the para position resulted in a similar trend in affinity (F, <b>20</b> < Cl, <b>21</b> < Br, <b>22</b>); however, their affinities were lower compared to the meta-substituted analogues. Of note, compounds with a fluorine atom in meta (<b>14</b>) or para (<b>20</b>) position displayed lower affinities than compounds with a methyl group in the equivalent position (<b>13</b> and <b>19</b>). To gain more insight in a potential relationship between affinity and lipophilicity as observed in the halogen series, calculated log <i>P</i> values (cLogP) of compounds <b>8</b>–<b>23</b>, with R<sup>1</sup> modifications, were plotted against their p<i>K</i><sub>i</sub> values in CCR2. This analysis revealed only a slight correlation between these two parameters for this set of compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S2a</a>); however, this correlation was lost when all synthesized derivatives were included in this plot (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S2b</a>), indicating that this is not a general trend.</div><div class="NLM_p">In the case of CCR5, meta-substituted derivatives also outperformed their ortho- and para-substituted analogues, with some compounds displaying >60% inhibition at 1 μM; in contrast, ortho- and para-substitution resulted in compounds with low (≤31%) to marginal efficacy in CCR5, suggesting that substituents in ortho or para position are not tolerated in CCR5. Similarly as in CCR2, the introduction of a methoxy group was unfavorable, as it led to a complete loss of activity in CCR5 when tested at 1 μM (<b>12</b>, <b>17</b>, and <b>23</b>), regardless of the position, whereas electron-withdrawing groups in meta position (<b>18</b>, R<sup>2</sup> = CF<sub>3</sub>) did not bring any improvement over the unsubstituted <b>9</b>. Except for compound <b>14</b> bearing a <i>meta</i>-fluoro, which showed less than 45% inhibition, all other compounds bearing halogens in meta position led to >60% inhibition; the same was achieved when a methyl group was placed in this position (<b>13</b>). Overall, these data indicate that meta-substituents, especially halogens, are preferred to achieve dual CCR2/CCR5 activity, while ortho- and para-substituents lead to a lower affinity but higher selectivity toward CCR2.</div><div class="NLM_p">As none of the other substituents in R<sup>2</sup> led to a significant improvement in CCR5 activity over compound <b>8</b>, we decided to continue with this compound and investigate the effect of replacing the cyclopropyl moiety in R<sup>3</sup>. On the basis of the chemical structure of <b>8</b> and CCR2-RA-[<i>R</i>] (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), we hypothesized that the cyclopropyl group in <b>8</b> interacts with Val244<sup>6×36</sup> in CCR2 in a similar manner as the cyclohexyl group of CCR2-RA-[<i>R</i>].<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Thus, several triazolopyrimidinone derivatives were synthesized with different alkyl chains and aromatic groups in this position in order to investigate their SARs (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Starting with the effect of alkyl substituents, we observed that increasing the size and flexibility of the alkyl chain from <i>n</i> = 1 (methyl) to <i>n</i> = 4 (butyl) resulted in a parallel increase in CCR2 affinity (17 nM for R<sup>3</sup> = Me (<b>24</b>); ∼4 nM for R<sup>3</sup> = Et (<b>25</b>) and R<sup>3</sup> = Pr (<b>26</b>); 2 nM for R<sup>3</sup> = Bu (<b>28</b>)). However, further elongation of the chain length (<i>n</i> = 5–7) led to a progressive drop in affinity (7 nM for R<sup>3</sup> = Pent (<b>30</b>); 22 nM for R<sup>3</sup> = Hex (<b>32</b>); 178 nM for R<sup>3</sup> = Hept (<b>28</b>)), indicating that linear alkyl chains longer than five carbons might not fit in this hydrophobic pocket. The same trend was observed for CCR5 activity, as only the <i>n</i>-propyl (<b>26</b>) and <i>n</i>-butyl (<b>28</b>) substituted compound led to >60% inhibition, albeit without improvement over <b>8</b> (<b>28</b>, 519 nM). Moreover, introduction of a hexyl or heptyl group resulted in CCL3 stimulation instead of inhibition, which was not further investigated. Increasing bulkiness via branching of alkyl groups or substitution with aliphatic rings enhanced the affinity toward CCR2, indicating that these substituents might provide a better interaction with the receptor. For instance, the introduction of both isopropyl (<b>27</b>) and cyclopropyl (<b>8</b>) groups led to an improvement in CCR2 affinity compared to the linear analogue <b>26</b>. Moreover, compound <b>27</b> with an isopropyl substituent also yielded a 2-fold increase in CCR5 potency compared to the cyclic analogue <b>8</b>, displaying the highest potency in this series of compounds (<b>27</b>, 281 nM). In line with this trend, we observed that replacing the linear pentyl group (<b>30</b>) with a cyclopentyl group (<b>31</b>) was also beneficial for CCR2, as this derivative showed a 4.5-fold increased affinity compared to <b>30</b> (<b>31</b>, 1.6 nM). In CCR5, <b>31</b> inhibited the CCL3-induced response with a potency of 388 nM, showing a slight improvement over compound <b>8</b>. In contrast, the introduction of a 2-ethylbutyl group (<b>29</b>) resulted in reduced affinity/activity toward both CCR2 and CCR5. These data suggest that the isopropyl group is the preferred R<sup>3</sup> substituent when designing CCR2/CCR5 dual antagonists, as this substituent led to the highest potency in CCR5 while maintaining a high affinity for CCR2. Next, inspired by our work on CCR1/CCR2 selectivity of pyrrolone derivatives,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> we investigated whether aromatic substituents are tolerated in this position. As expected from previous studies,<a onclick="showRef(event, 'ref19 ref30'); return false;" href="javascript:void(0);" class="ref ref19 ref30">(19,30)</a> the introduction of aromatic groups decreased 20-fold (<b>34</b>, 27 nM), 40-fold (<b>36</b>, 52 nM), and 122-fold (<b>35</b>, 159 nM) the affinity for CCR2 compared to <b>8</b>. When tested in CCR5, all derivatives showed a complete loss of activity at 1 μM, indicating that aromatic groups are not favorable for selectivity or dual activity.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Characterization of Compounds <b>24</b>–<b>36</b> in hCCR2 and hCCR5<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0007.gif" alt="" id="GRAPHIC-d7e1849-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">hCCR2</th><th class="colsep0 rowsep0" align="center">hCCR5</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">p<i>K</i><sub>i</sub> ± SEM (<i>K</i><sub>i</sub>, nM)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">pIC<sub>50</sub> ± SEM (IC<sub>50</sub>, nM) or inhibition at 1 μM (%)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left"><i>c</i>Pr</td><td class="colsep0 rowsep0" align="left">8.90 ± 0.04 (1.3)</td><td class="colsep0 rowsep0" align="left">6.24 ± 0.004 (571)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">7.78 ± 0.07 (17.2)</td><td class="colsep0 rowsep0" align="left">–35%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">8.40 ± 0.07 (4.0)</td><td class="colsep0 rowsep0" align="left">29%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">Pr</td><td class="colsep0 rowsep0" align="left">8.46 ± 0.07 (3.6)</td><td class="colsep0 rowsep0" align="left">64%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">8.72 ± 0.05 (1.9)</td><td class="colsep0 rowsep0" align="left">6.56 ± 0.05 (281)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">Bu</td><td class="colsep0 rowsep0" align="left">8.64 ± 0.03 (2.3)</td><td class="colsep0 rowsep0" align="left">6.29 ± 0.05 (519)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">2-EtBu</td><td class="colsep0 rowsep0" align="left">8.20 ± 0.04 (6.4)</td><td class="colsep0 rowsep0" align="left">29%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">Pent</td><td class="colsep0 rowsep0" align="left">8.14 ± 0.03 (7.2)</td><td class="colsep0 rowsep0" align="left">38%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left"><i>c</i>Pent</td><td class="colsep0 rowsep0" align="left">8.81 ± 0.04 (1.6)</td><td class="colsep0 rowsep0" align="left">6.43 ± 0.08 (388)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">Hex</td><td class="colsep0 rowsep0" align="left">7.66 ± 0.02 (22.0)</td><td class="colsep0 rowsep0" align="left">–63%<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">Hept</td><td class="colsep0 rowsep0" align="left">6.76 ± 0.05 (178.1)</td><td class="colsep0 rowsep0" align="left">–265%<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">7.64 ± 0.17 (26.7)</td><td class="colsep0 rowsep0" align="left">–41%<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">4-MePh</td><td class="colsep0 rowsep0" align="left">6.81 ± 0.07 (158.8)</td><td class="colsep0 rowsep0" align="left">–13%<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>CH<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">7.29 ± 0.05 (52.3)</td><td class="colsep0 rowsep0" align="left">–42%<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Data are presented as the mean p<i>K</i><sub>i</sub>/pIC<sub>50</sub> ± standard error of the mean (SEM) and mean <i>K</i><sub>i</sub>/IC<sub>50</sub> (nM) of at least three independent experiments performed in duplicate.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">p<i>K</i><sub>i</sub> values from the displacement of ∼6 nM [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] from U2OS cells stably expressing CCR2, at 25 °C.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Percent inhibition of β-arrestin recruitment in U2OS cells stably expressing CCR5 by 1 μM compound, in the presence of CCL3 (pEC<sub>80</sub> = 7.9). pIC<sub>50</sub> values were determined for compounds displaying more than 70% inhibition. % Inhibition values are presented as mean values of at least two independent experiments, performed in duplicate.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">No inhibition was observed at the concentration of 1 μM; instead some CCL3 stimulation was measured.</p></div></div></div><div class="NLM_p">With the aim of finding dual CCR2/CCR5 intracellular inhibitors, we kept the isopropyl moiety in R<sup>3</sup> and investigated the effect of having a disubstituted phenyl moiety in R<sup>1</sup> by exploring different positions and combinations of chlorine and bromine atoms (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). First and similar to <b>8</b>, we kept the cyclopropyl moiety in R<sup>3</sup> and combined it with dichlorination in meta and para positions (<b>37</b>). Compared to the monosubstituted analogues <b>8</b> and <b>21</b>, this compound yielded an even higher affinity to CCR2 (<b>37</b>, 0.4 nM); moreover, its ability to inhibit CCL3-induced response in CCR5 was also improved, as the potency increased to 214 nM. By replacing the cyclopropyl of <b>37</b> with an isopropyl group (<b>38</b>), we retained affinity for CCR2 (0.6 nM), but the potency for CCR5 increased by almost 2-fold (132 nM), in agreement with the higher potency observed in <b>27</b> versus <b>8</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Moving one chlorine atom to the ortho position, while keeping one in the adjacent meta position, yielded compound <b>39</b> with slightly lower affinity for CCR2 but even higher potency in CCR5 (<b>39</b>, 84 nM), indicating that although ortho substituents are not preferred in monosubstituted derivatives, they are still tolerated when placed in combination with halogens in other positions. However, placing the two halogens in the second and fifth positions was clearly detrimental for both receptors (<b>40</b>); in CCR2, the affinity decreased by almost 40-fold, while in CCR2, the compound was only able to inhibit 20% of the CCR5 response. Placing the two halogens in the symmetrical third and fifth positions restored the affinity/activity in both receptors (<b>41</b>, 2.2 nM in CCR2 and 336 nM in CCR5). Replacing the two chlorine atoms of <b>41</b> by bromine atoms yielded derivative <b>42</b>, which retained affinity toward CCR2 but led to decrease in CCR5 activity, as this compound was not able to inhibit >70% of the CCL3-induced response. Finally, the combination of a bromo in meta position with a chloro in para position (<b>42</b>) improved both the affinity and activity to both receptors to similar levels as <b>37</b>, in the case of CCR2, and <b>38</b> in the case of CCR5, indicating that halogens in adjacent positions are more favorable for activity in these receptors. Of note, compounds <b>37</b>, <b>38</b>, and <b>43</b> displayed the highest affinities to CCR2 in this study, while <b>38</b>, <b>39</b>, and <b>43</b> displayed the highest potencies to CCR5.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Characterization of Compounds <b>37</b>–<b>43</b> in hCCR2 and hCCR5<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0008.gif" alt="" id="GRAPHIC-d7e2305-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">hCCR2</th><th class="colsep0 rowsep0" align="center">hCCR5</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">p<i>K</i><sub>i</sub> ± SEM (<i>K</i><sub>i</sub>, nM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">pIC<sub>50</sub> ± SEM (IC<sub>50</sub>, nM) or inhibition at 1 μM (%)<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">3,4-diCl</td><td class="colsep0 rowsep0" align="left"><i>c</i>Pr</td><td class="colsep0 rowsep0" align="left">9.35 ± 0.05 (0.4)</td><td class="colsep0 rowsep0" align="left">6.67 ± 0.03 (214)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">3,4-diCl</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">9.22 ± 0.05 (0.6)</td><td class="colsep0 rowsep0" align="left">6.91 ± 0.09 (132)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">2,3-diCl</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">8.81 ± 0.07 (1.6)</td><td class="colsep0 rowsep0" align="left">7.09 ± 0.07 (84)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">2,5-diCl</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">7.65 ± 0.03 (22.5)</td><td class="colsep0 rowsep0" align="left">20%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">3,5-diCl</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">8.66 ± 0.05 (2.2)</td><td class="colsep0 rowsep0" align="left">6.49 ± 0.06 (336)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">3,5-diBr</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">8.68 ± 0.01 (2.1)</td><td class="colsep0 rowsep0" align="left">64%</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">3-Br, 4-Cl</td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">9.42 ± 0.02 (0.4)</td><td class="colsep0 rowsep0" align="left">6.95 ± 0.04 (115)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Data are presented as mean p<i>K</i><sub>i</sub>/pIC<sub>50</sub> ± standard error of the mean (SEM) and mean <i>K</i><sub>i</sub>/IC<sub>50</sub> (nM) of at least three independent experiments performed in duplicate.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">p<i>K</i><sub>i</sub> values from the displacement of ∼6 nM [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] from U2OS cells stably expressing CCR2, at 25 °C.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Percent inhibition of β-arrestin recruitment in U2OS cells stably expressing CCR5 by 1 μM compound, in the presence of CCL3 (pEC<sub>80</sub> = 7.9). pIC<sub>50</sub> values were determined for compounds displaying more than 70% inhibition. % Inhibition values are presented as mean values of at least two independent experiments, performed in duplicate.</p></div></div></div><div class="NLM_p last">It is important to note that so far we are comparing data not only between two different receptors but also between two different assays: (i) a radioligand binding assay for CCR2, in the absence of agonist, which allows the determination of true affinities (p<i>K</i><sub>i</sub> values); (ii) a functional assay for CCR5 in the presence of an EC<sub>80</sub> concentration of CCL3, without further correction of their IC<sub>50</sub> values. To better compare the activities in both receptors, we selected starting compound <b>8</b> as well as compounds <b>39</b> and <b>43</b> (with the highest potency on CCR5 and the highest affinity for CCR2, respectively) and tested these in a previously described β-arrestin recruitment assay for CCR2.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In this assay, compound <b>8</b> inhibited CCL2-stimulated β-arrestin recruitment with a potency of 10 nM and a Hill slope of −2.7, in agreement with its allosteric binding mode. Compound <b>39</b> inhibited β-arrestin recruitment in CCR2 with a lower potency of 21 nM, while compound <b>43</b> displayed a higher potency of 4 nM, consistent with their affinities. In addition, their Hill slopes (<i>n</i><sub>H</sub> = −2.5 for <b>39</b>; <i>n</i><sub>H</sub> = −3.4 for <b>43</b>) are also indicative of a noncompetitive form of inhibition, a further confirmation of their allosteric binding site located in the intracellular region of CCR2 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Table S3</a>). Of note, the Hill slopes in CCR5 were comparable to those in CCR2 (<i>n</i><sub>H</sub> = −3.7 for <b>39</b>; <i>n</i><sub>H</sub> = −4.4 for <b>43</b>), i.e., indicating an allosteric interaction at CCR5 as well. Comparing the IC<sub>50</sub> values obtained with the functional assays in both receptors, we observe a 4-fold difference between CCR2 and CCR5 in the case of <b>39</b>, making it a potential dual-antagonist for both receptors. In contrast, the potencies in CCR2 and CCR5 differ by 29-fold in the case of <b>43</b>, indicating a higher selectivity toward CCR2. Yet, network studies have suggested that partial inhibition by a low-affinity binder might be sufficient to effectively modulate cellular pathways <i>in vivo</i>;<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> thus further studies are needed to establish the optimal activity ratio for these receptors in order to achieve <i>in vivo</i> efficacy.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Mechanism of Inhibition of Selected Compounds</h3><div class="NLM_p">Selected compounds <b>39</b> and <b>43</b> were also tested in a [<sup>35</sup>S]GTPγS binding assay in both CCR2 and CCR5 in order to determine their mechanism of inhibition. In the case of CCR2, we have shown that these ligands fully displace [<sup>3</sup>H]-CCR2-RA-[<i>R</i>], indicating that triazolopyrimidinone derivatives bind in the same intracellular binding site. Thus, these compounds were expected to show noncompetitive, insurmountable antagonism to (orthosteric) chemokine ligands, as previously demonstrated in CCR2 with CCR2-RA-[<i>R</i>]<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and JNJ-27141491.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> To verify this, <b>39</b> and <b>43</b> were characterized in a previously described [<sup>35</sup>S]GTPγS binding assay on U2OS-CCR2 membranes.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In this assay, CCL2-stimulation of [<sup>35</sup>S]GTPγS binding in CCR2 was examined in the absence or presence of fixed concentrations of <b>39</b> and <b>43</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,b). In the absence of antagonist, increasing concentrations of CCL2 induced [<sup>35</sup>S]GTPγS binding with an EC<sub>50</sub> of 8 nM, in line with previously described parameters.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Co-incubation of CCL2 with <b>39</b> or <b>43</b> caused a significant reduction in the maximal response of CCL2 (<i>E</i><sub>max</sub>) at all three antagonist concentrations tested. The lowest concentrations of antagonist did not affect the potency of CCL2, while higher concentrations significantly reduced the potency of CCL2 (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,b). Of note, both compounds were also tested in the absence of CCL2 at a single concentration of 1 μM to determine potential inverse agonism. At this concentration they only reduced the basal [<sup>35</sup>S]GTPγS binding levels by 7–8% (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S3</a>), providing too small a window to accurately determine their potencies as inverse agonists. Previously, we reported that some intracellular pyrrolone derivatives were also able to decrease the CCR2 basal activity in this assay; however, the effect seemed to be dependent on the assay conditions, such as GDP concentrations.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Thus, more studies are needed to investigate whether the observed CCR2 constitutive activity is biologically relevant.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Effects of Compounds <b>39</b> and <b>43</b> in Chemokine-Stimulated [<sup>35</sup>S]GTPγS Binding<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">receptor</th><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">pEC<sub>50</sub> ± SEM (EC<sub>50</sub>, nM)</th><th class="colsep0 rowsep0" align="center"><i>E</i><sub>max</sub> ± SEM (%)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hCCR2</td><td class="colsep0 rowsep0" align="left">CCL2</td><td class="colsep0 rowsep0" align="left">8.10 ± 0.06 (8)</td><td class="colsep0 rowsep0" align="left">107 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL2 + 10 nM <b>39</b></td><td class="colsep0 rowsep0" align="left">7.89 ± 0.04 (13)</td><td class="colsep0 rowsep0" align="left">91 ± 1**</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL2 + 30 nM <b>39</b></td><td class="colsep0 rowsep0" align="left">7.60 ± 0.07 (26)**</td><td class="colsep0 rowsep0" align="left">75 ± 4****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL2 + 100 nM <b>39</b></td><td class="colsep0 rowsep0" align="left">7.27 ± 0.10 (56)****</td><td class="colsep0 rowsep0" align="left">50 ± 3****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL2 + 1 nM <b>43</b></td><td class="colsep0 rowsep0" align="left">7.91 ± 0.10 (13)</td><td class="colsep0 rowsep0" align="left">72 ± 4****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL2 + 3 nM <b>43</b></td><td class="colsep0 rowsep0" align="left">7.53 ± 0.12 (32)***</td><td class="colsep0 rowsep0" align="left">51 ± 5****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL2 + 10 nM <b>43</b></td><td class="colsep0 rowsep0" align="left">6.87 ± 0.13 (148)****</td><td class="colsep0 rowsep0" align="left">33 ± 3****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hCCR5</td><td class="colsep0 rowsep0" align="left">CCL3</td><td class="colsep0 rowsep0" align="left">8.42 ± 0.06 (4)</td><td class="colsep0 rowsep0" align="left">108 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL3 + 100 nM <b>39</b></td><td class="colsep0 rowsep0" align="left">8.35 ± 0.09 (5)</td><td class="colsep0 rowsep0" align="left">79 ± 5****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL3 + 300 nM <b>39</b></td><td class="colsep0 rowsep0" align="left">8.14 ± 0.12 (8)</td><td class="colsep0 rowsep0" align="left">56 ± 2****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL3 + 1000 nM <b>39</b></td><td class="colsep0 rowsep0" align="left">8.14 ± 0.17 (9)</td><td class="colsep0 rowsep0" align="left">25 ± 4****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL3 + 30 nM <b>43</b></td><td class="colsep0 rowsep0" align="left">8.30 ± 0.05 (5)</td><td class="colsep0 rowsep0" align="left">81 ± 3****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL3 + 100 nM <b>43</b></td><td class="colsep0 rowsep0" align="left">8.21 ± 0.05 (6)</td><td class="colsep0 rowsep0" align="left">58 ± 1****</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CCL3 + 300 nM <b>43</b></td><td class="colsep0 rowsep0" align="left">8.05 ± 0.06 (9)*</td><td class="colsep0 rowsep0" align="left">35 ± 2****</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Data represent the mean ± standard error of the mean (SEM) of three independent experiments performed in duplicate. One-way ANOVA with Dunnett’s post hoc test was used to analyze differences in pEC<sub>50</sub> and <i>E</i><sub>max</sub> values against CCL2 or CCL3 controls.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Maximum effect (<i>E</i><sub>max</sub>) of CCL2 or CCL3 measured in the absence or presence of fixed concentrations of compound <b>39</b> and <b>43</b> in CCR2 or CCR5, respectively.</p></div></div></div><div class="NLM_p last">To confirm our hypothesis that these two compounds also bind to an allosteric site in CCR5, i.e., the intracellular binding site, we next analyzed the effect of <b>39</b> and <b>43</b> on CCL3-induced [<sup>35</sup>S]GTPγS binding in U2OS-CCR5 membranes. In agreement with previous studies, CCL3 stimulated [<sup>35</sup>S]GTPγS binding in CCR5 with a potency of 4 nM.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Similarly as in CCR2, the two compounds were able to significantly suppress the maximal response induced by CCL3 at all concentrations tested (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c,d). However, in contrast to CCR2, the potency of CCL3 was only significantly reduced with the highest concentration of <b>43</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Such depression of the maximal response with or without a decrease of agonist potency is typical of insurmountable antagonists,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> indicating that <b>39</b> and <b>43</b> behave as insurmountable antagonists at both CCR2 and CCR5. Of note, insurmountable antagonism can be generally achieved by two different mechanisms: allosteric binding or slow binding kinetics, i.e., slow equilibration of a competitive antagonist.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> However, insurmountable inhibition due to hemiequilibrium is only evident in preincubation experiments, where the receptor is preincubated with the antagonist before exposure to the agonist.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In contrast, allosteric binding leads to insurmountable inhibition in co-incubation experiments, as performed in this study. These data further support our hypothesis that <b>39</b> and <b>43</b> bind to an allosteric binding site in CCR5, most probably located intracellularly.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Docking Study</h3><div class="NLM_p">To further investigate the binding mode of triazolopyrimidinone derivatives, compounds <b>8</b>, <b>39</b>, <b>40</b>, and <b>43</b> were docked into a CCR2b model based on the crystal structure of CCR2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1A">5T1A</a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">S4</a>).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Due to the close proximity to the intracellular binding site, several residues from the intracellular loop 3 (ICL3) had to be modeled based on the crystal structure of CCR5 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MBS">4MBS</a>),<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> since they were mutated in the original CCR2 crystal structure to further stabilize the receptor. As seen in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a, <b>43</b> was predicted to adopt a similar binding pose as that of the previously cocrystallized ligand CCR2-RA-[<i>R</i>].<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The disubstituted phenyl group of <b>43</b> was constrained to overlap with the corresponding phenyl group of CCR2-RA-[<i>R</i>], since the disubstituted aromatic rings of JNJ-27141491 and SD-24 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) were also predicted to overlay with the phenyl group of CCR2-RA-[<i>R</i>] in a previous study.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, the bromine group of compound <b>43</b> is predicted to form a halogen bond with the backbone of Val<sup>1×53</sup>, which might contribute to the higher affinity of <b>43</b> versus CCR2-RA-[<i>R</i>] (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). The positions of the halogens in this phenyl group might also explain the results of our SAR study. For instance, compound <b>40</b> with no halogen group on position 3 is not able to form halogen bonds with the receptor; in addition, one of the chlorine groups seems to point upward toward Tyr<sup>7×53</sup>, resulting in a sterically unfavorable position and thus in the observed lower activity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Similar to <b>43</b>, both <b>8</b> and <b>39</b> contain a chlorine group in position 3, promoting the formation of a halogen bond with the backbone of Val<sup>1×53</sup>, which might result in the improved activity compared to <b>40</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figure S4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, bromine displays a larger σ-hole and therefore a higher halogen bond strength compared to chlorine,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> which might result in the higher affinity of <b>43</b>, containing a bromine group, compared to <b>8</b> or <b>39</b> containing a chlorine. As the SAR follows a similar trend in CCR5, these data suggest that the intracellular ligands share similar interactions between the aromatic group and both receptors (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Proposed binding mode of <b>43</b> in hCCR2b. (a) Overlay of <b>43</b> with the CCR2 intracellular ligand CCR2-RA-[<i>R</i>], showing that <b>43</b> interacts in a similar manner as CCR2-RA-[<i>R</i>]. (b) Docking of <b>43</b>, displaying the interactions with CCR2. The amino group in R<sup>2</sup> makes an extra hydrogen-bond interaction with E<sup>8×48</sup>, while the bromine group in R<sup>1</sup> makes an extra halogen bond with the backbone of V<sup>1×53</sup>, which might contribute to the improved affinity of <b>43</b> to this receptor. Model of hCCR2 is based on the crystal structure of CCR2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1A">5T1A</a>),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and amino acid residues are labeled according to their structure-based Ballesteros–Weinstein numbers.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, the isopropyl group present in compounds <b>39</b>, <b>40</b>, and <b>43</b> is predicted to bind in the same position as the cyclohexyl moiety of CCR2-RA-[<i>R</i>], although it seems to make less interactions with Val<sup>6×36</sup> perhaps due to the slightly different ligand orientation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). Previous studies have confirmed the crucial role of Val<sup>6×36</sup> for binding affinity of some intracellular ligands in CCR2, as mutation of this residue to alanine completely abolished binding of CCR2-RA-[<i>R</i>] to the receptor.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Moreover, this residue might be involved in target selectivity, as the main difference between the intracellular pockets of CCR2 and CCR5 is the single substitution of Val<sup>6×36</sup> by Leu<sup>6×36</sup>. The steric hindrance introduced by this substitution might thus be responsible for the reduction in activity of CCR2-RA-[<i>R</i>] and the triazolopyrimidinone derivatives toward CCR5 compared to CCR2.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Indeed, in the case of CCR5, only small aliphatic groups were tolerated in R<sup>3</sup> position, such as cyclopropyl or isopropyl (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), while bigger aliphatic groups resulted in improved selectivity toward CCR2. In line with the role of Val<sup>6×36</sup> as determinant of selectivity, a previous SAR analysis of pyrrolone derivatives in CCR1, which also contains a leucine in position 6 × 36, showed that aromatic groups in the equivalent R<sup>3</sup> position provide CCR1 selectivity versus CCR2, as aromatic groups are not tolerated in this position in CCR2<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">The binding pose of <b>43</b> seems to be stabilized by a network of hydrogen bonds between the triazolopyrimidinone core and residues E<sup>8×48</sup>, Lys<sup>8×49</sup>, F<sup>8×50</sup>, and R<sup>3×50</sup> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). Although the core of CCR2-RA-[<i>R</i>] and <b>43</b> binds with a different orientation, the carboxy group of both is overlaid in the same position, interacting with the backbones of Lys<sup>8×49</sup> and F<sup>8×50</sup>. Moreover, the secondary and tertiary amino groups present in the triazolopyrimidinone core also form hydrogen bonds with the backbones of Lys<sup>8×49</sup> and Glu<sup>8×48</sup>, as well as with the side chains of Arg<sup>3×50</sup>. Finally, the primary amino group in position R<sup>2</sup> of compound <b>43</b> also makes an extra hydrogen bond with the side chain of E<sup>8×48</sup>. Such extended network of hydrogen bond interactions is not present with CCR2-RA-[<i>R</i>], and thus it might also be responsible for the higher affinity of <b>43</b> in CCR2, compared to CCR2-RA-[<i>R</i>]. In addition, the SAR data suggest that this interaction is also crucial in CCR5, as the removal of this amino group (compounds <b>6</b> and <b>7</b>) was detrimental for CCR5 activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Previous studies have confirmed the importance of residues 8 × 49 and/or 8 × 50 in chemokine receptors for the binding of several intracellular ligands. For example, alanine mutations of Lys<sup>8×49</sup> and F<sup>8×50</sup> in CCR2 caused a 10-fold reduction or a complete loss of affinity of intracellular ligands, respectively, compared to the wild-type receptor.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In CXCR2, alanine mutation of Lys<sup>8×49</sup> led to a reduced affinity of three different intracellular ligands, while the mutation F<sup>8×50</sup>A only affected one of the ligands tested, indicating a different binding mode.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Moreover, Lys<sup>8×49</sup> has been suggested as a key residue for target selectivity between CXCR1 and CXCR2, as it is exchanged by Asn<sup>8×49</sup> in CXCR1.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In addition, the crystal structure of CCR9 in complex with vercirnon<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> also shows a binding interaction between the ligand and Arg323<sup>8×49</sup> and Phe324<sup>8×50</sup>.</div><div class="NLM_p last">Overall, these data suggest that although the intracellular pockets of CCR2 and CCR5 are quite conserved, the design of multitarget compounds is not quite straightforward. Moreover, several of these residues have been shown to be involved in Gα<sub>i</sub> coupling in recent cryoelectron microscopy (cryo-EM)-derived GPCR structures, including residues 3 × 50, 6 × 29, 6 × 32 to 6 × 37, 8 × 47, and 8 × 49.<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39−41)</a> Similarly, homologous residues are also involved in direct interactions between rhodopsin and arrestin,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> suggesting a direct interference of these intracellular ligands with the Gα<sub>i</sub> protein and β-arrestin binding site, and the possibility of fine-tuning residue interactions for the design of biased ligands. On the basis of the SAR analysis and the docking study, these compounds could be further optimized by exploring the triazolopyrimidinone core. For instance, pyrazolopyrimidinones have also been described for CCR2,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> which might also display CCR5 activity. In addition, exploring other halogen combinations at the phenyl group in R<sup>1</sup> or other small, bulky aliphatic groups in R<sup>3</sup> such as cyclobutyl might lead to compounds with improved dual activity. Although out of the scope of this manuscript, ligands such as <b>39</b> with high potency in CCR5 could be investigated as potential tools to further study binding interactions in CCR5, i.e., by obtaining a radiolabeled tool compound or by obtaining a CCR5 crystal structure in complex with an intracellular ligand. Finally, these ligands can be used in future experiments designed to investigate their functional effects both <i>in vitro</i> and <i>in vivo</i>, to validate the target combination, and to establish the required level of target modulation.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64186" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64186" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this study we first confirmed that the triazolopyrimidinone derivative <b>8</b> binds to the intracellular pocket of CCR2 in a similar manner as the reference intracellular ligand CCR2-RA-[<i>R</i>]. Moreover, compound <b>8</b> was also able to inhibit CCR5 in a functional β-arrestin recruitment assay; thus, we took this compound as a starting point for the synthesis of a series of novel and previously described triazolopyrimidinone derivatives. Using [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] binding assays and functional β-arrestin recruitment assays, we explored structure–affinity/activity relationships (SARs) in both receptors. Overall, these compounds were mostly selective toward CCR2; however, CCR5 activity was increased with the combination of a primary amino group in R<sup>2</sup> position, an isopropyl moiety in R<sup>3</sup>, and two halogens placed in adjacent positions at the phenyl group in R<sup>1</sup>. Overall, these findings indicate that even though the intracellular pockets of CCR2 and CCR5 are highly conserved, selectivity of intracellular ligands can be fine-tuned, allowing the design of either selective or multitarget ligands. Evaluation of compounds <b>39</b> and <b>43</b> in a [<sup>35</sup>S]GTPγS binding assay indicates that both compounds display a noncompetitive, insurmountable mode of inhibition in CCR2 and CCR5, which might represent a therapeutic advantage in inflammatory diseases characterized by a high local concentration of endogenous chemokines, such as multiple sclerosis and rheumatoid arthritis. Thus, in diseases where selective chemokine receptor antagonists have been largely unsuccessful, the development of multitarget, intracellular ligands for CCR2 and CCR5 is warranted to further study the effects of multitarget versus selective inhibition, as these ligands may represent a novel therapeutic option in these diseases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03236" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03236" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry. General Methods</h3><div class="NLM_p last">All solvents and reagents used were of analytical grade and from commercial sources. Demineralized water was used in all cases, unless stated otherwise, and is simply referred to as H<sub>2</sub>O. Microwave-based synthesis was carried out using a Biotage Initiator equipment (Biotage, Sweden). All reactions were monitored by thin-layer chromatography (TLC) using aluminum plates coated with silica gel 60 F<sub>254</sub> (Merck), and compounds were visualized under ultraviolet light at 254 nm or via KMnO<sub>4</sub> staining. Column chromatography for compound purification was performed using silica gel (Merck Millipore) with particle size 0.04–0.63 mm. Chemical identity of final compounds was established using <sup>1</sup>H NMR and liquid chromatography–mass spectrometry (LC–MS). <sup>1</sup>H NMR spectra were recorded on a Bruker AV 400 liquid spectrometer (<sup>1</sup>H NMR, 400 MHz) at room temperature. Compound <b>39</b>, as one of the most soluble and one of the best CCR2/CCR5 antagonists, was fully characterized (<sup>1</sup>H NMR, <sup>13</sup>C NMR, and attached proton test (APT)) on a Bruker AV500 spectrometer at 80 °C. The corresponding NMR spectra of compound <b>39</b> are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">Figures S5–S7</a>. Chemical shifts (δ) are reported in parts per million (ppm), and coupling constants (<i>J</i>) in Hz. Liquid chromatography–mass spectrometry (LC–MS) of final compounds was performed using a Thermo Finnigan Surveyor LCQ Advantage Max LC–MS system and a Gemini C18 Phenomenex column (50 mm × 4.6 mm, 3 μm). Analytical purity of the compounds was determined using a Shimadzu high pressure liquid chromatography (HPLC) equipment with a Phenomenex Gemini column (3 x C18 110A column, 50 mm × 4.6 mm, 3 μm). A flow rate of 1.3 mL/min and an elution gradient of 10–90% MeCN/H<sub>2</sub>O (0.1% TFA) were used. The absorbance of the UV spectrophotometer was set at 254 nm. All compounds tested in biological assays showed a single peak at the designated retention time and were ≥95% pure. Sample preparations for HPLC and LC–MS were as follows unless stated otherwise: 0.3 mg/mL of compound was dissolved in a 1:1:1 mixture of H<sub>2</sub>O:MeOH:<i><sup>t</sup></i>BuOH. Of note, some compounds required DMSO and heat to ensure proper dissolution. None of the final compounds were identified as potential pan-assay interference compounds (PAINS) after assessment with the free ADME-Tox filtering tool (FAF-Drugs4),<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> which uses three different PAINS filters based on Baell et al.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedure 1: Synthesis of β-Keto Esters <b>1f</b>–<b>h</b>,<b>j</b>,<b>k</b>,<b>n</b><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></h3><div class="NLM_p last">In a flame-dried round-bottom flask under a nitrogen atmosphere, ethyl acetoacetate (2.53 mL, 20.0 mmol, 1.00 equiv) was added dropwise to a suspension of NaH (880 mg, 22.1 mmol 1.10 equiv) in dry THF (5 mL) at 0 °C while stirring. After 20 min, <i>n</i>-butyllithium (20 0.0 mmol, 2.50 M solution in pentane,1.00 equiv) was added dropwise to the mixture and stirred for further 30 min. The respective alkyl halide <b>2f</b>–<b>h</b>,<b>j</b>,<b>k</b> or benzyl bromide <b>2n</b> (1.20 equiv) was subsequently added dropwise over a period of 10 min to the dianion solution after which the solution was allowed to reach rt. After 14 h, the reaction was quenched by the addition of saturated NH<sub>4</sub>Cl (aq, 80 mL). The mixture was subsequently extracted with diethyl ether (2 × 120 mL). The combined organic fractions were washed with brine (80 mL) and dried over MgSO<sub>4</sub> followed by concentration in vacuo. The crude products were purified by flash chromatography (CH<sub>2</sub>Cl<sub>2</sub>/petroleum ether and/or EtOAc/petroleum ether as the eluent) to give the title compounds <b>1f</b>–<b>h</b>,<b>j</b>,<b>k</b>,<b>n</b> as oils. Compounds <b>1a</b>–<b>e</b>,<b>i</b>,<b>l</b>,<b>m</b> were commercially available.</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Ethyl-3-oxoheptanoate (<b>1f</b>)<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></h3><div class="NLM_p last">Compound <b>1f</b> was <b>s</b>ynthesized according to general procedure 1, using 1-bromopropane (<b>2f</b>, 2.00 mL, 22.0 mmol, 1.10 equiv) as starting compound. Compound was purified by silica column chromatography (1–30% EtOAc in petroleum ether). Yield: 36% (1.25 g) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.19 (q, <i>J</i> = 7.2 Hz, 2H), 3.44 (s, 2H), 2.54 (t, <i>J</i> = 7.4 Hz, 2H), 1.65–1.55 (m, 2H), 1.38–1.25 (m, 7H), 0.88 (t, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Ethyl 5-Ethyl-3-oxoheptanoate (<b>1g</b>)</h3><div class="NLM_p last">Compound <b>1g</b> was <b>s</b>ynthesized according to general procedure 1, using 3-bromopentane (<b>2g</b>, 3.00 mL, 24.2 mmol, 1.21 equiv) as starting compound. Compound was purified by silica column chromatography (1–30% EtOAc in petroleum ether). Yield: 18% (630 mg) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.17 (q, <i>J</i> = 7.2 Hz, 2H), 3.40 (s, 2H), 2.43 (d, <i>J</i> = 6.8 Hz, 2H), 1.88–1.73 (m, 1H), 1.33–1.21 (m, 7H), 0.82 (t, <i>J</i> = 7.4 Hz, 6H) ppm.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Ethyl 3-Oxooctanoate (<b>1h</b>)<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></h3><div class="NLM_p last">Compound <b>1h</b> was <b>s</b>ynthesized according to general procedure 1, using 1-bromobutane (<b>2h</b>, 2.59 mL, 24.1 mmol, 1.21 equiv) as starting compound. Compound was purified by silica column chromatography (1–30% EtOAc in petroleum ether). Yield: 38% (1.41 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.19 (q, <i>J</i> = 7.2 Hz, 2H), 3.43 (s, 2H), 2.53 (t, <i>J</i> = 7.2 Hz, 2H), 1.65–1.55 (m, 2H), 1.37–1.12 (m, 7H), 0.89 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Ethyl 3-Oxononanoate (<b>1j</b>)<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></h3><div class="NLM_p last">Compound <b>1j</b> was <b>s</b>ynthesized according to general procedure 1, using 1-iodopentane (<b>2j</b>, 3.13 mL, 24.0 mmol, 1.20 equiv) as starting compound. Compound was purified by silica column chromatography (1–30% EtOAc in petroleum ether). Yield: 44% (1.76 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.19 (q, <i>J</i> = 7.2 Hz, 2H), 3.43 (s, 2H), 2.53 (t, <i>J</i> = 7.4 Hz, 2H), 1.54–1.64 (m, 2H), 1.33–1.24 (m, 9H), 0.88 (t, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Ethyl 3-Oxodecanoate (<b>1k</b>)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></h3><div class="NLM_p last">Compound <b>1k</b> was synthesized according to general procedure 1 using 1-bromohexane (<b>2k</b>, 3.36 mL, 25.0 mmol, 1.24 equiv) as starting compound. Compound was purified by silica column chromatography (30–100% CH<sub>2</sub>Cl<sub>2</sub> in petroleum ether). Yield: 15% (625 mg) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.19 (q, <i>J</i> = 7.2 Hz, 2H), 3.43 (s, 2H), 2.53 (t, <i>J</i> = 7.4 Hz, 2H), 1.64–1.53 (m, 2H), 1.33–1.24 (m, 11H), 0.88 (t, <i>J</i> = 7.5 Hz, 3H) ppm.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Ethyl 3-Oxo-5-phenylpentanoate (<b>1n</b>)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></h3><div class="NLM_p last">Compound <b>1n</b> was synthesized according to general procedure 1, using benzyl bromide (<b>2n</b>, 2.90 mL, 24.4 mmol, 1.22 equiv) as starting compound. Compound was purified by silica column chromatography (1–30% EtOAc in petroleum ether). Yield: 20% (1.06 g) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33–7.23 (m, 2H), 7.23–7.12 (m, 3H), 4.17 (q, <i>J</i> = 7.2 Hz, 2H), 3.42 (s, 2H), 2.98–2.81 (m, 4H), 1.26 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure 2. Benzylated β-Keto Esters <b>4aa</b>–<b>na</b>, <b>4bb</b>–<b>bq</b>, <b>4eq</b>–<b>ev</b><a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">LiCl (1.00 equiv) was slurried in anhydrous THF (1 mL/mmol <b>1a</b>–<b>n</b>) in a flame-dried round-bottom flask and under an atmosphere of nitrogen. The desired β-keto ester <b>1a</b>–<b>n</b> (1.00 equiv) was added and followed by DIPEA (2.00 equiv) and the respective benzylic halide <b>3a</b>–<b>v</b> (1.20 equiv). The reaction mixture was refluxed for 20 h, after which the reaction was completed as indicated by TLC (5–10% EtOAc in petroleum ether). THF was removed in vacuo, the crude dissolved in EtOAc (30 mL), and this organic layer was washed with citric acid (5%, 25 mL) followed by saturated NaHCO<sub>3</sub> (25 mL) and brine (25 mL). The organic layer was subsequently dried over MgSO<sub>4</sub>, and concentrated in vacuo to afford the crude product. The crude product was purified by flash column chromatography (5–10% EtOAc/petroleum ether) to yield the corresponding benzylated β-keto esters <b>4aa</b>–<b>na</b>, <b>4bb</b>–<b>bq</b>, <b>4eq</b>–<b>ev</b>.</div></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Ethyl 2-(3-Chlorobenzyl)-3-oxobutanoate (<b>4aa</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxobutanoate <b>1a</b> (0.37 mL, 2.92 mmol, 1.20 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.32 mmol, 2.43 mmol, 1.00 equiv), DIPEA (0.85 mL, 4.86 mmol, 2.00 equiv), LiCl (103 mg, 2.43 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 70% (433 mg) as a colorless oil. <sup>1</sup>H NMR: (400 MHz, CDCl<sub>3</sub>): δ 7.21–7.14 (m, 3H), 7.09–7.04 (m, 1H), 4.16 (q, <i>J</i> = 7.2 Hz, 2H), 3.75 (t, <i>J</i> = 8.0 Hz, 1H), 3.17–3.07 (m, 2H), 2.21 (s, 3H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Ethyl 3-cyclopropyl-2-(3-chlorobenzyl)-3-oxopropanoate (<b>4ba</b>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (7.56 mL, 51.2 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (7.06 mL, 53.8 mmol, 1.05 equiv), DIPEA (17.8 mL, 102 mmol, 2.00 equiv), LiCl (2.17 g, 51.22 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 71% (10.2 g) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.21–7.l6 (m, 3H), 7.10–7.05 (m, 1H), 4.17 (qd, <i>J</i> = 7.2, 1.6 Hz, 2H), 3.89 (t, <i>J</i> = 7.6 Hz, 1H), 3.15 (dd, <i>J</i> = 7.2 Hz, 2H), 2.08–2.02 (m, 1H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H), 1.11–1.01 (m, 2H), 0.98–0.88 (m, 2H) ppm.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Ethyl 2-Benzyl-3-cyclopropyl-3-oxopropanoate (<b>4bb</b>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.52 mL, 3.51 mmol, 1.20 equiv), benzyl bromide <b>3b</b> (0.347 mL, 2.92 mmol, 1.00 equiv), DIPEA (1.02 mL, 5.84 mmol, 2.00 equiv), LiCl (124 mg, 2.92 mmol, 1.00 equiv), 4 mL of dry THF. Yield: 26% (105 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.29–7.25 (m, 2H), 7.22–7.18 (m, 3H), 4.20–4.12 (m, 2H), 3.91 (t, <i>J</i> = 7.6 Hz, 1H), 3.20 (d, <i>J</i> = 7.6 Hz, 2H), 2.08–2.02 (m, 1H), 1.20 (t, <i>J</i> = 7.2 Hz, 3H), 1.06–1.03 (m, 2H), 0.94–0.84 (m, 2H) ppm.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Ethyl 3-Cyclopropyl-2-(2-methylbenzyl)-3-oxopropanoate (<b>4bc</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.63 mL, 4.27 mmol, 1.20 equiv), 2-methylbenzyl chloride <b>3c</b> (0.48 mL, 3.56 mmol, 1.00 equiv), DIPEA (1.24 mL, 7.12 mmol, 2.00 equiv), LiCl (151 mg, 3.56 mmol, 1.00 equiv), 4 mL of dry THF. Yield: 80% (742 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.15–7.09 (m, 4H), 4.16 (qd, <i>J</i> = 7.2, 1.2 Hz, 2H), 3.91 (t, <i>J</i> = 7.2 Hz, 1H), 3.20 (d, <i>J</i> = 7.6 Hz, 2H), 2.34 (s, 3H), 2.05–2.00 (m, 1H), 1.20 (t, <i>J</i> = 7.2 Hz, 3H), 1.07–1.02 (m, 2H), 0.95–0.84 (m, 2H) ppm.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Ethyl 3-Cyclopropyl-2-(2-chlorobenzyl)-3-oxopropanoate (<b>4bd</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.43 mL, 2.92 mmol, 1.20 equiv), 2-chlorobenzyl bromide <b>3d</b> (0.32 mL, 2.43 mmol, 1.00 equiv), DIPEA (0.85 mL, 4.86 mmol, 2.00 equiv), LiCl (103 mg, 2.43 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 78% (532 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.36–7.33 (m, 1H), 7.26–7.24 (m, 1H), 7.18–7.14 (m, 2H), 4.20–4.08 (m, 3H), 3.37–3.22 (m, 2H), 2.10–2.03 (m, 1H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 1.09–1.01 (m, 2H), 0.97–0.87 (m, 2H) ppm.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Ethyl 3-Cyclopropyl-2-(2-methoxybenzyl)-3-oxopropanoate (<b>4be</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.33 mL, 2.26 mmol, 1.20 equiv), 2-methoxybenzyl bromide<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a><b>3e</b> (378 mg, 1.88 mmol, 1.00 equiv), DIPEA (0.66 mL, 3.76 mmol, 2.00 equiv), LiCl (79.7 mg, 1.88 mmol, 1.00 equiv), 5 mL of dry THF. Silica column chromatography in 8:1:1 petroleum ether:EtOAc:CH<sub>2</sub>Cl<sub>2</sub>. Yield: 36% (185 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.20 (td, <i>J</i> = 6.4, 1.6 Hz, 1H), 7.13 (dd, <i>J</i> = 6.0, 1.2 Hz, 1H), 6.86–6.26 (m, 2H), 4.17–4.09 (m, 2H), 4.04 (dd, <i>J</i> = 6.8, 1.6 Hz, 1H), 3.84 (s, 3H), 3.42–3.11 (m, 2H), 2.07–2.02 (m, 1H), 1.19 (t, <i>J</i> = 6.0 Hz, 3H), 1.05–1.02 (m, 2H), 0.92–0.85 (m, 2H) ppm.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Ethyl 3-Cyclopropyl-2-(3-methylbenzyl)-3-oxopropanoate (<b>4bf</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.47 mL, 3.24 mmol, 1.20 equiv), 3-methylbenzyl bromide <b>3f</b> (0.36 mL, 2.70 mmol, 1.00 equiv), DIPEA (0.94 mL, 5.40 mmol, 2.00 equiv), LiCl (114 mg, 2.70 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 74% (521 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.17 (t, <i>J</i> = 7.6 Hz, 1H), 7.02–6.96 (m, 3H), 4.19–4.10 (m, 2H), 3.90 (t, <i>J</i> = 7.6 Hz, 1H), 3.15 (d, <i>J</i> = 7.6 Hz, 2H), 2.30 (s, 3H), 2.07–2.01 (m, 1H), 1.19 (t, <i>J</i> = 7.2 Hz, 3H), 1.04–0.99 (m, 2H), 0.93–0.82 (m, 2H) ppm.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Ethyl 3-Cyclopropyl-2-(3-fluorobenzyl)-3-oxopropanoate (<b>4bg</b>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.550 mL, 3.73 mmol, 1.31 equiv), 3-fluorobenzyl bromide <b>3g</b> (0.350 mL, 2.85 mmol, 1.00 equiv), DIPEA (0.940 mL, 5.39 mmol, 1.89 equiv), LiCl (0.140 g, 2.70 mmol, 0.947 equiv), 5 mL of dry THF. Yield: 100% (770 mg) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.26–7.16 (m, 1H), 6.97 (d, <i>J</i> = 8.0 Hz, 1H), 6.94–6.84 (m, 2H), 4.19–4.11 (m, 2H), 3.92 (t, <i>J</i> = 7.6 Hz, 1H), 3.18 (dd, <i>J</i> = 7.6, 1.6 Hz, 2H), 2.10–2.04 (m, 1H), 1.20 (t, <i>J</i> = 7.2 Hz, 3H), 1.07–0.98 (m, 2H), 0.93–0.83 (m, 2H) ppm.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Ethyl 2-(3-Bromobenzyl)-3-cyclopropyl-3-oxopropanoate (<b>4bh</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.35 mL, 2.40 mmol, 1.20 equiv), 3-bromobenzyl bromide <b>3h</b> (500 mg, 2.00 mmol, 1.00 equiv), DIPEA (0.70 mL, 4.00 mmol, 2.00 equiv), LiCl (85 mg, 2.00 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 47% (303 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.36–7.32 (m, 2H), 7.17–7.10 (m, 2H), 4.17 (q, <i>J</i> = 7.2 Hz, 2H), 3.89 (t, <i>J</i> = 7.2 Hz, 1H), 3.15 (d, <i>J</i> = 8.0 Hz, 2H), 2.08–2.01 (m, 1H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H), 1.11–1.01 (m, 2H), 0.98–0.86 (m, 2H) ppm.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Ethyl 3-Cyclopropyl-2-(3-iodobenzyl)-3-oxopropanoate (<b>4bi</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.550 mL, 3.73 mmol, 1.38 equiv), 3-iodobenzyl bromide <b>3i</b> (0.802 g, 2.70 mmol, 1.00 equiv), DIPEA (0.940 mL, 5.39 mmol, 2.00 equiv), LiCl (0.150 g, 2.70 mmol, 1.00 equiv), 5 mL of dry THF. Yield: quantitative (1.28 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.55–7.51 (m, 2H), 7.15 (d, <i>J</i> = 7.6 Hz, 1H), 6.99 (t, <i>J</i> = 7.6 Hz, 1H), 4.16 (qd, <i>J</i> = 7.2, 1.6 Hz, 2H), 3.87 (t, <i>J</i> = 7.2 Hz, 1H), 3.11 (dd, <i>J</i> = 8.0, 2.0 Hz, 2H), 2.06–2.00 (m, 1H), 1.20 (t, <i>J</i> = 6.8 Hz, 3H), 1.11–1.00 (m, 2H), 0.98–0.81 (m, 2H) ppm.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Ethyl 3-Cyclopropyl-2-(3-methoxybenzyl)-3-oxopropanoate (<b>4bj</b>)<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.44 mL, 3.00 mmol, 1.20 equiv), 3-methoxybenzyl bromide <b>3j</b> (0.35 mL, 2.50 mmol, 1.00 equiv), DIPEA (0.87 mL, 5.00 mmol, 2.00 equiv), LiCl (106 mg, 2.50 mmol, 1.00 equiv), 5 mL of dry THF. Silica column chromatography in 8:1:1 petroleum ether:EtOAc:CH<sub>2</sub>Cl<sub>2</sub>. Yield: 42% (294 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.19 (td, <i>J</i> = 7.2 Hz, 1.2 Hz, 1H), 6.79–6.74 (m, 3H), 4.21–4.13 (m, 2H), 3.91 (t, <i>J</i> = 7.6 Hz, 1H), 3.78 (s, 3H), 3.17 (d, <i>J</i> = 7.6 Hz, 2H), 2.08–2.02 (m, 1H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H), 1.06–1.01 (m, 2H), 0.96–0.85 (m, 2H) ppm.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Ethyl 3-Cyclopropyl-3-oxo-2-(3-(trifluoromethyl)benzyl)propanoate (<b>4bk</b>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.37 mL, 2.51 mmol, 1.20 equiv), 3-(trifluoromethyl)benzyl bromide <b>3k</b> (0.32 mL, 2.09 mmol, 1.00 equiv), DIPEA (0.73 mL, 4.18 mmol, 2.00 equiv), LiCl (89 mg, 2.09 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 31% (214 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.50–7.44 (m, 2H), 7.42–7.38 (m, 2H), 4.17 (q, <i>J</i> = 6.8 Hz, 2H), 3.92 (t, <i>J</i> = 8.0 Hz, 1H), 3.25 (d, <i>J</i> = 7.2 Hz, 2H), 2.10–2.02 (m, 1H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 1.12–1.01 (m, 2H), 0.99–0.86 (m, 2H) ppm.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Ethyl 3-Cyclopropyl-2-(4-methylbenzyl)-3-oxopropanoate (<b>4bl</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.48 mL, 3.24 mmol, 1.20 equiv), 4-methylbenzyl bromide <b>3l</b> (500 mg, 2.70 mmol, 1.00 equiv), DIPEA (0.94 mL, 5.40 mmol, 2.00 equiv), LiCl (114 mg, 2.70 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 74% (521 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.08 (s, 4H), 4.22–4.09 (m, 2H), 3.88 (t, <i>J</i> = 7.6 Hz, 1H), 3.15 (d, <i>J</i> = 7.6 Hz, 2H), 2.30 (s, 3H), 2.07–2.02 (m, 1H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 1.07–1.02 (m, 2H), 0.96–0.83 (m, 2H) ppm.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Ethyl 3-Cyclopropyl-2-(4-fluorobenzyl)-3-oxopropanoate (<b>4bm</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.890 mL, 6.04 mmol, 1.14 equiv), 1-(bromomethyl)-4-fluorobenzene <b>3m</b> (1.57 g, 5.29 mmol, 1.00 equiv), DIPEA (1.05 mL, 6.00 mmol, 1.13 equiv), LiCl (0.130 g, 3.00 mmol, 0.58 equiv), 5 mL of dry THF. Yield: 56% (780 mg) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.18–7.12 (m, 2H), 6.96 (tt, <i>J</i> = 8.8, 2.0 Hz, 2H), 4.22–4.10 (m, 2H), 3.87 (t, <i>J</i> = 8.0 Hz, 1H), 3.16 (d, <i>J</i> = 7.6 Hz, 2H), 2.08–2.00 (m, 1H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 1.09- 0.99 (m, 2H), 0.97–0.84 (m, 2H) ppm.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Ethyl 2-(4-Chlorobenzyl)-3-cyclopropyl-3-oxopropanoate (<b>4bn</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.43 mL, 2.92 mmol, 1.20 equiv), 4-chlorobenzyl bromide <b>3n</b> (500 mg, 2.43 mmol, 1.00 equiv), DIPEA (0.85 mL, 4.86 mmol, 2.00 equiv), LiCl (103 mg, 2.43 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 66% (451 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.24 (dt <i>J</i> = 8.8, 2.0 Hz,, 2H), 7.13 (dt, <i>J</i> = 8.4, 2.0 Hz, 2H), 4.21–4.11 (m, 2H), 3.87 (dd, <i>J</i> = 8.0, 0.8 Hz, 1H), 3.15 (dd, <i>J</i> = 6.8, 1.2 Hz, 2H), 2.08–2.01 (m, 1H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 1.09–1.01 (m, 2H), 0.97–0.86 (m, 2H) ppm.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Ethyl 2-(4-Bromobenzyl)-3-cyclopropyl-3-oxopropanoate (<b>4bo</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.880 mL, 5.97 mmol, 1.49 equiv), 1-(bromomethyl)-4-bromobenzene <b>3o</b> (1.00 g, 4.00 mmol, 1.00 equiv), DIPEA (1.39 mL, 8.00 mmol, 2.00 equiv), LiCl (0.170 g, 4.00 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 67% (0.880 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.39 (d, <i>J</i> = 8.4 Hz, 2H), 7.07 (d, <i>J</i> = 8.4 Hz, 2H), 4.22–4.10 (m, 2H), 3.87 (t, <i>J</i> = 8.0 Hz, 1H), 3.14 (d, <i>J</i> = 8.2 Hz, 2H), 2.08–1.96 (m, 1H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H), 1.10–0.99 (m, 2H), 0.99–0.85 (m, 2H) ppm.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Ethyl 3-Cyclopropyl-2-(4-methoxybenzyl)-3-oxopropanoate (<b>4bp</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.57 mL, 3.83 mmol, 1.20 equiv), 4-methoxybenzyl bromide <b>3p</b> (0.46 mL, 3.19 mmol, 1.00 equiv), DIPEA (1.11 mL, 6.38 mmol, 2.00 equiv), LiCl (135 mg, 3.19 mmol, 1.00 equiv), 5 mL of dry THF. Silica column chromatography in 7:1:2 petroleum ether:EtOAc:CH<sub>2</sub>Cl<sub>2</sub>. Yield: 52% (454 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.10 (dt, <i>J</i> = 8.8, 2.0 Hz, 2H), 6.81 (dt, <i>J</i> = 8.8, 2.4 Hz, 2H), 4.20–4.10 (m, 2H), 3.86 (t, <i>J</i> = 7.6 Hz, 1H), 3.78 (s, 3H), 3.13 (d, <i>J</i> = 7.6 Hz, 2H), 2.07–2.01 (m, 1H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 1.08–1.01 (m, 2H), 0.95–0.83 (m, 2H) ppm.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Ethyl 3-Cyclopropyl-2-(3,4-dichlorobenzyl)-3-oxopropanoate (<b>4bq</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopropyl-3-oxopropanoate <b>1b</b> (0.60 mL, 4.06 mmol, 1.00 equiv), 3,4-dichlorobenzyl bromide <b>3q</b> (0.62 mL, 4.27 mmol, 1.05 equiv), DIPEA (1.42 mL, 8.13 mmol, 2.00 equiv), LiCl (172 mg, 4.06 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 39% (493 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33 (d, <i>J</i> = 8.4 Hz, 1H), 7.30 (d, <i>J</i> = 1.6 Hz, 1H), 7.03 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 4.22–4.12 (m, 2H), 3.87 (t, <i>J</i> = 7.6 Hz, 1H), 3.18–3.09 (m, 2H), 2.09–2.02 (m, 1H), 1.23 (t, <i>J</i> = 7.2 Hz, 3H), 1.09–1.01 (m, 2H), 0.95–0.85 (m, 2H) ppm.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Ethyl 2-(3-Chlorobenzyl)-3-oxopentanoate (<b>4ca</b>)<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxopentanoate <b>1c</b> (0.42 mL, 2.92 mmol, 1.20 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.32 mL, 2.43 mmol, 1.00 equiv), NaH (117 mg, 4.86 mmol, 2.00 equiv), 5 mL of dry THF. Yield: 52% (340 mg) as a colorless oil. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.20–7.14 (m, 3H), 7.09–7.03 (m, 1H), 4.15 (qd, <i>J</i> = 7.5, 0.8 Hz, 2H), 3.76 (t, <i>J</i> = 7.5 Hz, 1H), 3.17–3.08 (m, 2H), 2.60 (dqd, <i>J</i> = 18.0, 7.0, 1.0 Hz, 1H), 2.37 (dqd, <i>J</i> = 18.5, 7.0, 0.5 Hz, 1H), 1.20 (td, <i>J</i> = 7.0, 1.0 Hz, 3H), 1.01 (td, <i>J</i> = 7.5, 1.0 Hz, 3H) ppm.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Ethyl 2-(3-Chlorobenzyl)-3-oxohexanoate (<b>4da</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxohexanoate <b>1d</b> (0.46 mL, 2.92 mmol, 1.20 equiv), 2.43 mmol of 3-chlorobenzyl bromide <b>3a</b>, DIPEA (0.85 mmol, 4.86 mmol, 2.00 equiv), LiCl (103 mg, 2.43 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 76% (522 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.22–7.14 (m, 3H), 7.08–7.04 (m, 1H), 4.15 (q, <i>J</i> = 7.2 Hz, 2H), 3.75 (t, <i>J</i> = 7.6 Hz, 1H), 3.18–3.07 (m, 2H), 2.54 (dt, J = 17.6, 7.2 Hz, 1H), 2.35 (dt, <i>J</i> = 17.2, 7.2 Hz, 1H), 1.57 (sextet, <i>J</i> = 7.2 Hz, 2H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 0.85 (t, <i>J</i> = 7.6 Hz, 3H) ppm.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Ethyl 2-(3-Chlorobenzyl)-4-methyl-3-oxopentanoate (<b>4ea</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 4-methyl-3-oxopentanoate <b>1e</b> (0.51 mL, 3.16 mmol, 1.20 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.35 mL, 2.63 mmol, 1.00 equiv), DIPEA (0.95 mL, 5.26 mmol, 2.00 equiv), LiCl (111 mg, 2.63 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 86% (640 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.20–7.14 (m, 3H), 7.07–7.04 (m, 1H), 4.14 (qd, <i>J</i> = 7.2, 0.8 Hz, 2H), 3.91 (t, <i>J</i> = 7.6 Hz, 1H), 3.17–3.07 (m, 2H), 2.66 (septet, <i>J</i> = 6.8 Hz, 1H), 1.21 (t, <i>J</i> = 7.2 Hz, 3H), 1.07 (d, <i>J</i> = 6.8 Hz, 3H), 0.91 (d, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Ethyl 2-(3,4-Dichlorobenzyl)-4-methyl-3-oxopentanoate (<b>4eq</b>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 4-methyl-3-oxopentanoate <b>1e</b> (5.00 mL, 31.0 mmol, 1.00 equiv), 3,4-dichlorobenzyl bromide <b>3q</b> (5.41 mL, 37.2 mmol, 1.20 equiv), DIPEA (10.8 mL, 62.0 mmol, 2.00 equiv), LiCl (1.31 g, 30.9 mmol, 1.00 equiv), 50 mL of dry THF. Yield: 33% (3.20 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33 (d, <i>J</i> = 8.4 Hz, 1H), 7.29–7.25 (m, 1H), 7.01 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 4.15 (qd, <i>J</i> = 7.2, 1.6 Hz, 2H), 3.89 (t, <i>J</i> = 7.6 Hz, 1H), 3.16–3.04 (m, 2H), 2.69 (heptet, <i>J</i> = 6.8 Hz, 1H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H), 1.07 (d, <i>J</i> = 6.4 Hz, 3H), 0.93 (d, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Ethyl 2-(2,3-Dichlorobenzyl)-4-methyl-3-oxopentanoate (<b>4er</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 4-methyl-3-oxopentanoate <b>1e</b> (0.670 mL, 4.61 mmol, 1.00 equiv), 2,3-dichlorobenzyl bromide <b>3r</b> (1.00 g, 4.17 mmol, 0.90 equiv), DIPEA (1.45 mL, 8.34 mmol, 1.81 equiv), LiCl (180 mg, 0.90 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 8% (110 mg) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.34 (dd, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.15 (dd, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.09 (t, <i>J</i> = 7.6 Hz, 1H), 4.20–4.03 (m, 3H), 3.34–3.22 (m, 2H), 2.71 (heptet, <i>J</i> = 6.8 Hz, 1H), 1.19 (t, <i>J</i> = 7.2 Hz, 3H), 1.07 (d, <i>J</i> = 6.8 Hz, 3H), 0.90 (d, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div><div id="sec4_3_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Ethyl 2-(2,5-Dichlorobenzyl)-4-methyl-3-oxopentanoate (<b>4es</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 4-methyl-3-oxopentanoate <b>1e</b> (0.204 mL, 1.39 mmol, 1.00 equiv), 2,5-dichlorobenzyl bromide <b>3s</b> (500 mg, 2.08 mmol, 1.5 equiv), DIPEA (0.242 mL, 1.39 mmol, 1.00 equiv), LiCl (60 mg, 1.39 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 71% (315 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.27 (d, <i>J</i> = 8.8 Hz, 1H), 7.22 (d, <i>J</i> = 2.4 Hz, 1H), 7.15 (dd, <i>J</i> = 8.4, 2.4 Hz, 1H), 4.18–4.12 (m, 2H), 4.10 (t, <i>J</i> = 7.2 Hz, 1H), 3.26–3.16(m, 2H), 2.73 (heptet, <i>J</i> = 6.8 Hz, 1H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H), 1.08 (d, <i>J</i> = 6.8 Hz, 3H), 0.93 (d, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Ethyl 2-(3,5-Dichlorobenzyl)-4-methyl-3-oxopentanoate (<b>4et</b>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 4-methyl-3-oxopentanoate <b>1e</b> (0.480 mL, 3.30 mmol, 1.00 equiv), 3,5-dichlorobenzyl bromide <b>3t</b> (0.440 mL, 3.12 mmol, 1.00 equiv), DIPEA (1.05 mL, 6.00 mmol, 1.82 equiv), LiCl (0.310 g, 3.00 mmol, 0.91 equiv), 5 mL of dry THF. Yield: 16% (150 mg) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.13 (t, <i>J</i> = 2.0 Hz, 1H), 6.99 (d, <i>J</i> = 2.0 Hz, 2H), 4.08 (q, <i>J</i> = 7.2 Hz, 2H), 3.81 (t, <i>J</i> = 7.6 Hz, 1H), 3.07–2.97 (m, 2H), 2.63 (heptet, <i>J</i> = 6.8 Hz, 1H), 1.15 (t, <i>J</i> = 7.2 Hz, 3H), 1.00 (d, <i>J</i> = 6.8 Hz, 3H), 0.87 (d, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div><div id="sec4_3_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Ethyl 2-(3,5-Dibromobenzyl)-4-methyl-3-oxopentanoate (<b>4eu</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 4-methyl-3-oxopentanoate <b>1e</b> (0.480 mL, 3.30 mmol, 1.00 equiv), 3,5-dibromobenzyl bromide <b>3u</b> (1.04 g, 3.16 mmol, 0.958 equiv), DIPEA (1.05 mL, 6.00 mmol, 1.82 equiv), LiCl (0.130 g, 3.00 mmol, 0.909 equiv), 5 mL of dry. THF. Yield: 29% (0.350 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.27 (s, 3H), 4.15 (q, <i>J</i> = 7.2 Hz, 2H), 3.87 (t, <i>J</i> = 7.2 Hz, 1H), 3.13–3.05 (m, 2H), 2.70 (heptet, <i>J</i> = 6.8 Hz, 1H), 1.24 (t, <i>J</i> = 6.4 Hz, 3H), 1.08 (d, <i>J</i> = 6.8 Hz, 3H), 0.95 (d, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div><div id="sec4_3_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Ethyl 2-(3-Bromo-4-chlorobenzyl)-4-methyl-3-oxopentanoate (<b>4ev</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 4-methyl-3-oxopentanoate <b>1e</b> (0.480 mL, 3.30 mmol, 1.00 equiv), 3-bromo-4-chlorobenzyl bromide <b>3v</b> (0.900 g, 3.16 mmol, 0.958 equiv), DIPEA (1.05 mL, 6.00 mmol, 1.82 equiv), LiCl (0.130 g, 3.00 mmol, 0.909 equiv), 5 mL of dry THF. Yield: 34% (0.360 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.44 (d, <i>J</i> = 2.0 Hz, 1H), 7.33 (d, <i>J</i> = 8.0 Hz, 1H), 7.06 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 4.15 (qd, <i>J</i> = 7.2, 1.6 Hz, 2H), 3.88 (t, <i>J</i> = 7.6 Hz, 1H), 3.14–3.04 (m, 2H), 2.69 (heptet, <i>J</i> = 6.8 Hz, 1H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H), 1.08 (d, <i>J</i> = 6.8 Hz, 3H), 0.94 (d, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div><div id="sec4_3_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Ethyl 2-(3-Chlorobenzyl)-3-oxoheptanoate (<b>4fa</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxoheptanoate <b>1f</b> (0.47 g, 2.72 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.33 mL, 2.51 mmol, 0.922 equiv), DIPEA (1.05 mL, 6.00 mmol, 2.21 equiv), LiCl (0.130 g, 3.00 mmol, 1.10 equiv), 5 mL of dry THF. Yield: 67% (0.550 g) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.20–7.15 (m, 3H), 7.07–7.03 (m, 1H), 7.17–4.10 (m, 2H), 3.75 (t, <i>J</i> = 7.6 Hz, 1H), 3.16–3.06 (m, 2H), 2.55 (dt, <i>J</i> = 17.2, 7.2 Hz, 1H), 2.35 (dt, <i>J</i> = 17.6, 7.2 Hz, 1H), 1.50 (pentet, <i>J</i> = 7.6 Hz, 2H), 1.28–1.19 (m, 5H), 0.85 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Ethyl 2-(3-Chlorobenzyl)-5-ethyl-3-oxoheptanoate (<b>4ga</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 5-ethyl-3-oxoheptanoate <b>1g</b> (0.600 g, 3.00 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.470 mL, 3.57 mmol, 1.19 equiv), DIPEA (1.05 mL, 6.00 mmol, 2.00 equiv), LiCl (0.130 g, 3.00 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 31% (0.300 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.20–7.14 (m, 3H), 7.06–7.05 (m, 1H), 4.18–4.12 (m, 2H), 3.75 (t, <i>J</i> = 7.6 Hz, 1H), 3.17–3.06 (m, 2H), 2.45 (dd, J = 17.2, 6.8 Hz, 1H), 2.29 (dd, <i>J</i> = 17.2, 6.0 Hz, 1H), 1.76 (heptet, <i>J</i> = 6.4 Hz, 1H), 1.30–1.17 (m, 7H), 0.78 (t, <i>J</i> = 7.6 Hz, 6H) ppm.</div></div><div id="sec4_3_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Ethyl 2-(3-Chlorobenzyl)-3-oxooctanoate (<b>4ha</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxooctanoate <b>1h</b> (0.560 g, 3.01 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.470 mL, 3.57 mmol, 1.18 equiv), DIPEA (1.05 mL, 6.00 mmol, 1.99 equiv), LiCl (0.130 g, 3.00 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 21% (0.197 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.22–7.15 (m, 3H), 7.07–7.04 (m, 1H), 4.15 (q, <i>J</i> = 7.2 Hz, 2H), 3.75 (t, <i>J</i> = 7.6 Hz, 1H), 3.17–3.07 (m, 2H), 2.54 (dt, <i>J</i> = 17.2, 7.2 Hz, 1H), 2.36 (dt, <i>J</i> = 17.2, 7.2 Hz, 1H), 1.52 (pentet, <i>J</i> = 7.4 Hz, 2H), 1.31–1.13 (m, 7H), 0.86 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Ethyl 2-(3-Chlorobenzyl)-3-cyclopentyl-3-oxopropanoate (<b>4ia</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-cyclopentyl-3-oxopropanoate <b>1i</b> (0.376 g, 2.04 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.240 mL, 1.83 mmol, 0.888 equiv), DIPEA (0.530 mL, 3.03 mmol, 1.47 equiv), LiCl (0.067 g, 1.57 mmol, 0.772 equiv), 5 mL of dry THF. Silica column chromatography (20–50% CH<sub>2</sub>Cl<sub>2</sub>/petroleum ether). Yield: 85% (0.483 g) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.20–7.15 (m, 3H), 7.08–7.02 (m, 1H), 4.13 (q, <i>J</i> = 7.2 Hz, 2H), 3.85 (t, <i>J</i> = 7.6 Hz, 1H), 3.15–3.05 (m, 2H), 2.94 (pentet, <i>J</i> = 8.0 Hz, 1H), 1.87–1.35 (m, 8H), 1.20 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Ethyl 2-(3-Chlorobenzyl)-3-oxononanoate (<b>4ja</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxononanoate <b>1j</b> (0.601 g, 3.00 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.325 mL, 2.48 mmol, 0.825 equiv), DIPEA (1.05 mL, 6.00 mmol, 2.00 equiv), LiCl (0.130 g, 3.00 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 97% (0.780 g) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.20–7.15 (m, 3H), 7.07–7.01 (m, 1H), 4.14 (q, <i>J</i> = 6.8 Hz, 2H), 3.75 (t, <i>J</i> = 7.6 Hz, 1H), 3.17–3.06 (m, 2H), 2.53 (dt, J = 17.6, 7.2 Hz, 1H), 2.35 (dt, <i>J</i> = 17.6, 7.2 Hz, 1H), 1.50 (pentet, <i>J</i> = 7.2 Hz, 2H), 1.30–1.15 (m, 9H), 0.85 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Ethyl 2-(3-Chlorobenzyl)-3-oxodecanoate (<b>4ka</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxodecanoate <b>1k</b> (0.650 g, 3.03 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.475 mL, 3.60 mmol, 1.20 equiv), DIPEA (1.05 mL, 6.00 mmol, 2.00 equiv), LiCl (0.128 g, 3.00 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 50% (0.511 g) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.21–7.16 (m, 3H), 7.07–7.04 (m, 1H), 4.15 (q, <i>J</i> = 7.2 Hz, 2H), 3.75 (t, <i>J</i> = 7.6 Hz, 1H), 3.18–3.06 (m, 2H), 2.54 (dt, <i>J</i> = 17.6, 7.2 Hz, 1H), 2.36 (dt, <i>J</i> = 17.6, 7.6 Hz, 1H), 1.51 (pentet, <i>J</i> = 6.8 Hz, 2H), 1.31–1.17 (m, 11H), 0.87 (t, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div><div id="sec4_3_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Ethyl 2-(3-Chlorobenzyl)-3-oxo-3-phenylpropanoate (<b>4la</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxo-3-phenylpropanoate <b>1l</b> (0.498 mL, 2.92 mmol, 1.20 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.320 mL, 2.43 mmol, 1.00 equiv), DIPEA (0.847 mL, 4.86, 2.00 equiv), LiCl (103 mg; 2.43 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 85% (686 mg) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.92 (dd, <i>J</i> = 8.0, 0.8, 2H), 7.46 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.35 (t, <i>J</i> = 7.6 Hz, 2H), 7.20 (s, 1H), 7.10–7.05 (m, 3H), 4.62 (t, <i>J</i> = 7.2 Hz, 1H), 4.05–3.95 (m, 2H), 3.31–3.20 (m, 2H), 1.01 (t, <i>J</i> = 7.2, 0.8 Hz, 3H) ppm.</div></div><div id="sec4_3_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Ethyl 2-(3-Chlorobenzyl)-3-oxo-3-(<i>p</i>-tolyl)propanoate (<b>4ma</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxo-3-(<i>p</i>-tolyl)propanoate <b>1m</b> (2.30 g, 11.15 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (2.29 mL, 11.15 mmol, 1.00 equiv), DIPEA (3.88 mL, 22.3 mmol, 2.00 equiv), LiCl (473 mg; 11.15 mmol, 1.00 equiv), 30 mL of dry THF. Yield: 86% (3.18 g) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.87 (d, <i>J</i> = 8.0 Hz, 2H), 7.28–7.22 (m, 3H), 7.19–7.14 (m, 2H), 7.12–7.09 (m, 1H), 4.57 (t, <i>J</i> = 7.6 Hz, 1H), 4.13–4.05 (m, 2H), 3.33–3.23 (m, 2H), 2.41 (s, 3H), 1.12 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Ethyl 2-(3-Chlorobenzyl)-3-oxo-5-phenylpentanoate (<b>4na</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 2, using the following reagents: ethyl 3-oxo-5-phenylpentanoate <b>1n</b> (0.661 g, 3.00 mmol, 1.00 equiv), 3-chlorobenzyl bromide <b>3a</b> (0.475 mL, 3.65 mmol, 1.20 equiv), DIPEA (1.05 mL, 6.00 mmol, 2.00 equiv), LiCl (0.128 g, 3.00 mmol, 1.00 equiv), 5 mL of dry THF. Yield: 67% (0.690 g) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.32–7.26 (m, 2H), 7.24–7.19 (m, 3H), 7.16 (t, <i>J</i> = 7.2 Hz, 3H), 4.13 (q, <i>J</i> = 7.2 Hz, 2H), 3.75 (t, <i>J</i> = 7.6 Hz, 1H), 3.13 (d, <i>J</i> = 7.6 Hz, 2H), 2.99–2.85 (m, 3H), 2.75–2.67 (m, 1H), 1.20 (t, <i>J</i> = 7.2 Hz, 3H) ppm.</div></div><div id="sec4_3_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Procedure for the Synthesis of 6-(3-Chlorobenzyl)-5-cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>6</b>)<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a></h3><div class="NLM_p last">In a sealed microwave tube 3-amino-1,2,4-triazole <b>5a</b> (66 mg, 0.78 mmol, 1.1 equiv), <b>4ba</b> (200 mg, 0.71 mmol, 1.00 equiv), and H<sub>3</sub>PO<sub>4</sub> (96 μL, 1.42 mmol, 2.00 equiv) were heated at 170 °C in 1 mL of EtOH in the microwave for 10 h. The reaction mixture was poured in water (30 mL), the pH was adjusted to pH = 12 (1 M NaOH aq), and the organics were extracted with EtOAc (3 × 30 mL). The combined extracts were dried over MgSO<sub>4</sub> and the solvents evaporated in vacuo, resulting in 165 mg of crude mixture. The pure product was obtained by column chromatography (5% CH<sub>3</sub>OH in CH<sub>2</sub>Cl<sub>2</sub>) followed by prep HPLC gradient 10–90% CH<sub>3</sub>CN/water + 0.1%TFA yielding 4% (9 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.14 (s, 1H), 7.28–7.25 (m, 1H), 7.22–7.15 (m, 3H), 7.17–7.08 (m, 3H), 4.16 (s, 2H), 2.22–2.13 (m, 1H), 1.36–1.30 (m, 2H), 1.20–1.15 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>13</sub>ClN<sub>4</sub>O [M + H]<sup>+</sup> 301.09, found 301.1. HPLC: 6.566 min, purity 97%.</div></div><div id="sec4_3_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Procedure for the Synthesis of 6-(3-Chlorobenzyl)-5-cyclopropyl-2-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>7</b>)</h3><div class="NLM_p last">In a sealed microwave tube 3-amino-5-methyl-1,2,4-triazole <b>5b</b> (126 mg, 1.28 mmol, 1.2 equiv), <b>4ba</b> (300 mg, 1.07 mmol, 1.0 equiv), and <i>p</i>-toluenesulfonic acid monohydrate (102 mg, 0.53 mmol, 0.5 equiv) were heated for 30 min at 180 °C in the microwave. As visualized by TLC, <b>4ba</b> was consumed and mainly one product was formed (Rf 0.5 in 5% CH<sub>3</sub>OH in CH<sub>2</sub>Cl<sub>2</sub>). The crude product was purified by column chromatography (3% CH<sub>3</sub>OH in CH<sub>2</sub>Cl<sub>2</sub>) yielding 31% (96 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub> + drop MeOD): δ 7.23 (s, 1H), 7.21–7.13 (m, 3H), 6.26 (br s, 2H), 4.11 (s, 2H), 2.47 (s, 2H), 2.05–1.96 (m, 1H), 1.14–1.07 (m, 2H), 1.05–1.01 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>15</sub>ClN<sub>4</sub>O [M + H]<sup>+</sup> 315.10, found 315.1. HPLC: 6.826 min, purity 96%.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> General Procedure 3. Triazolopyrimidinones <b>8</b>–<b>43</b><a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">The synthesis of compounds <b>8</b>–<b>43</b> was according to the following procedure: In a microwave tube a mixture of the corresponding benzylated β-keto ester <b>4aa</b>–<b>na</b>, <b>4bb</b>–<b>bq</b>, <b>4eq</b>–<b>ev</b> (1.00 equiv), triazole <b>5c</b> (2.00 equiv) and 1-butyl-3-methylimidazolium hexafluorophosphate (BMIM-PF6, 1 mL or 6.00 equiv) was heated at 200 °C in a microwave reactor for an hour. Afterward, the reaction was allowed to cool to room temperature and stirred in a mixture of CH<sub>2</sub>Cl<sub>2</sub> (30 mL), H<sub>2</sub>O (10 mL), and 5–10% aqueous citric acid (1 mL) for 20–30 min. The resulting mixture was filtered over a glass filter, and the residue was washed with hot methanol. Finally, the precipitate was collected and dried <i>in vacuo</i> to yield the pure compounds.</div></div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 2-Amino-6-(3-chlorobenzyl)-5-cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>8</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (216 mg, 2.18 mmol, 2.00 equiv), <b>4ba</b> (306 mg, 1.09 mmol, 1.00 equiv), and BMIM-PF6 (1.35 mL, 6.54 mmol, 6.00 equiv). Yield: 36% (125 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.16 (br s, 1H), 7.35–7.24 (m,2H), 7.24–7.14 (m, 2H), 6.24 (br s, 2H), 3.96 (s, 2H), 2.14–2.01 (m, 1H), 1.07–0.94 (m, 2H), 0.94–0.81 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 316.10, found 316.13. HPLC: 6.350 min, purity 98%.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 2-Amino-6-benzyl-5-cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>9</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (276 mg, 2.84 mmol, 2.00 equiv), <b>4bb</b> (350 mg, 1.42 mmol, 1.00 equiv) and BMIM-PF6 (1.75 mL, 8.52 mmol, 6.00 equiv). Yield: 26% (105 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.11 (br s, 1H), 7.26–7.20 (m, 4H), 7.17–7.10 (m, 1H), 6.20 (br s, 2H), 3.96 (s, 2H), 2.10–2.03 (m, 1H), 1.04–0.95 (m, 2H), 0.94–0.88 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 282.14, found 282.13. HPLC: 5.655 min, purity 97%.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 2-Amino-5-cyclopropyl-6-(2-methylbenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>10</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (190 mg, 1.96 mmol, 2.00 equiv), <b>4bc</b> (255 mg, 0.98 mmol, 1.00 equiv), and BMIM-PF6 (1.21 mL, 5.88 mmol, 6.00 equiv). Yield: 32% (93 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.17 (br s, 1H), 7.15 (d, <i>J</i> = 6.8 Hz, 1H), 7.08–6.99 (m, 2H), 6.81 (d, <i>J</i> = 6.8 Hz, 1H), 6.21 (br s, 2H), 3.86 (s, 2H), 2.36 (s, 3H), 1.90–1.81 (m, 1H), 1.04–0.94 (m, 2H), 0.88–0.81 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>17</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 296.15, found 296.13. HPLC: 6.074 min, purity 98%.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 2-Amino-6-(2-chlorobenzyl)-5-cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>11</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (173 mg, 1.78 mmol, 2.00 equiv), <b>4bd</b> (250 mg, 0.89 mmol, 1.00 equiv), and BMIM-PF6 (1.01 mL, 5.34 mmol, 6.00 equiv). Yield: 37% (104 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.14 (br s, 1H), 7.47–7.42 (m, 1H), 7.24–7.17 (m, 2H), 7.01–6.96 (m, 1H), 6.20 (br s, 2H), 4.00 (s, 2H), 1.90–1.80 (m, 1H), 0.99–0.93 (m, 2H), 0.89–0.80 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 316.10, found 316.13. HPLC: 6.411 min, purity 95%.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 2-Amino-5-cyclopropyl-6-(2-methoxybenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>12</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (130 mg, 1.34 mmol, 2.00 equiv), <b>4be</b> (185 mg, 0.67 mmol, 1.00 equiv), and BMIM-PF6 (0.83 mL, 4.02 mmol, 6.00 equiv). Yield: 43% (89 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.11 (br s, 1H), 7.15 (t, <i>J</i> = 6.0 Hz, 1H), 6.95 (d, <i>J</i> = 6.4 Hz, 1H), 6.86 (d, <i>J</i> = 5.6 Hz, 1H), 6.78 (t, <i>J</i> = 5.6 Hz, 1H), 6.18 (br s, 2H), 3.86 (s, 2H), 3.83 (s, 3H), 1.98–1.86 (m, 1H), 1.03–0.91 (m, 2H), 0.90–0.79 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>17</sub>N<sub>5</sub>O2 [M + H]<sup>+</sup> 312.15, found 312.2. HPLC: 5.832 min, purity 98%.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 2-Amino-5-cyclopropyl-6-(3-methylbenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>13</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (216 mg, 2.22 mmol, 2.00 equiv), <b>4bf</b> (289 mg, 1.11 mmol, 1.00 equiv), and BMIM-PF6 (1.37 mL, 6.66 mmol, 6.00 equiv). Yield: 35% (114 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 11.98 (br s, 1H), 7.12 (t, <i>J</i> = 7.6 Hz, 1H), 7.04 (s, 1H), 7.01 (d, <i>J</i> = 7.2 Hz, 1H), 6.95 (d, <i>J</i> = 7.2 Hz, 1H), 6.17 (br s, 2H), 3.91 (s, 2H), 2.23 (s, 3H), 2.10–2.00 (m, 1H), 1.04–0.94 (m, 2H), 0.94–0.83 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>17</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 296.15, found 296.13. HPLC: 6.154 min, purity 97%.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 2-Amino-5-cyclopropyl-6-(3-fluorobenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>14</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (190 mg, 1.92 mmol, 1.45 equiv), <b>4bg</b> (350 mg, 1.32 mmol, 1.00 equiv), and BMIM-PF6 (1.14 mL, 5.54 mmol, 4.19 equiv). Yield: 15% (61 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 7.40–7.20 (m, 1H), 7.16–6.85 (m, 3H), 5.81 (br s, 2H), 4.01 (s, 2H), 2.12–1.95 (m, 1H), 1.10–0.85 (m, 4H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>FN<sub>5</sub>O [M + H]<sup>+</sup> 300.13, found 300.1. HPLC: 5.812 min, purity 97%.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 2-Amino-6-(3-bromobenzyl)-5-cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>15</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (169 mg, 1.74 mmol, 2.00 equiv), <b>4bh</b> (283 mg, 0.87 mmol, 1.00 equiv), and BMIM-PF6 (1.07 mL, 5.22 mmol, 6.00 equiv). Yield: 43% (135 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.21 (br s, 1H), 7.43 (s, 1H), 7.34 (d, <i>J</i> = 7.2 Hz, 1H), 7.25 (d, <i>J</i> = 7.6 Hz, 1H), 7.20 (t, <i>J</i> = 7.6 Hz, 1H), 6.26 (br s, 2H), 3.96 (s, 2H), 2.12–2.04 (m, 1H), 1.04–0.94 (m, 2H), 0.94–0.85 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>BrN<sub>5</sub>O [M + H]<sup>+</sup> 360.05, found 360.2. HPLC: 6.529 min, purity 99%.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 2-Amino-5-cyclopropyl-6-(3-iodobenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>16</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (190 mg, 1.92 mmol, 2.32 equiv), <b>4bi</b> (320 mg, 0.86 mmol, 1.00 equiv), and BMIM-PF6 (1.14 mL, 4.25 mmol, 6.00 equiv). Yield: 26% (92 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.18 (br s, 1H), 7.62 (s, 4H), 7.51 (d, <i>J</i> = 7.6 Hz, 1H), 7.25 (d, <i>J</i> = 7.6 Hz, 1H), 7.05 (t, <i>J</i> = 7.6 Hz, 1H), 6.22 (br s, 2H), 3.92 (s, 2H), 2.12–2.07 (m, 1H), 1.01–0.94 (m, 2H), 0.93–0.86 (m, 2H). LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>IN<sub>5</sub>O [M + H]<sup>+</sup> 408.03, found 408.1. HPLC: 7.042 min, purity 96%.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 2-Amino-5-cyclopropyl-6-(3-methoxybenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>17</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (146 mg, 1.50 mmol, 2.00 equiv), <b>4bj</b> (207 mg, 0.75 mmol, 1.00 equiv), BMIM-PF6 (0.93 mL, 4.50 mmol, 6.00 equiv). Yield: 21% (48 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 11.94 (br s, 1H), 7.15 (t, <i>J</i> = 8.0 Hz, 1H), 6.82–6.77 (m, 2H), 6.72 (d, <i>J</i> = 7.6 Hz, 1H), 6.16 (br s, 2H), 3.93 (s, 2H), 3.69 (s, 3H), 2.11–2.03 (m, 1H), 1.04–0.96 (m, 2H), 0.96–0.85 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>17</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 312.15, found 312.13. HPLC: 5.675 min, purity 96%.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 2-Amino-5-cyclopropyl-6-(3-(trifluoromethyl)benzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>18</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (150 mg, 1.54 mmol, 2.00 equiv), <b>4bk</b> (242 mg, 0.77 mmol, 1.00 equiv), and BMIM-PF6 (0.95 mL, 4.62 mmol, 6.00 equiv). Yield 6% (17 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.18 (br s, 1H), 7.61 (s, 1H), 7.57–7.46 (m, 3H), 6.24 (br s, 2H), 4.06 (s, 2H), 2.17–2.08 (m, 1H), 1.03–0.95 (m, 2H), 0.94–0.86 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>F<sub>3</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 350.13, found 350.27. HPLC: 6.727 min, purity 97%.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> 2-Amino-5-cyclopropyl-6-(4-methylbenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>19</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (197 mg, 2.03 mmol, 2.00 equiv), <b>4bl</b> (266 mg, 1.02 mmol, 1.00 equiv), and BMIM-PF6 (1.25 mL, 6.09 mmol, 6.00 equiv). Yield: 39% (115 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.06 (br s, 1H), 7.10 (d, <i>J</i> = 8.0 Hz, 2H), 7.03 (d, <i>J</i> = 8.0 Hz, 2H), 6.18 (br s, 2H), 3.90 (s, 2H), 2.23 (s, 3H), 2.08–2.01 (m, 1H), 1.01–0.94 (m, 2H), 0.94–0.87 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>17</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 296.15, found 296.13. HPLC: 6.279 min, purity 97%.</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> 2-Amino-5-cyclopropyl-6-(4-fluorobenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>20</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (150 mg, 1.51 mmol, 2.00 equiv), <b>4bm</b> (200 mg, 0.756 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 6.39 equiv). Yield: 35% (79 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.13 (s, 1H), 7.28 (dd, <i>J</i> = 9.6, 2.4 Hz, 2H), 7.07 (t, <i>J</i> = 8.8 Hz, 2H), 6.21 (s br, 2H), 3.95 (s, 2H), 2.15–2.02 (m, 1H), 1.05–0.85 (m, 4H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>FN<sub>5</sub>O [M + H]<sup>+</sup> 300.13, found 300.2. HPLC: 6.394 min, purity 95%.</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> 2-Amino-6-(4-chlorobenzyl)-5-cyclopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>21</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (179 mg, 1.84 mmol, 2.00 equiv), <b>4bn</b> (258 mg, 0.92 mmol, 1.00 equiv), and BMIM-PF6 (1.14 mL, 5.52 mmol, 6.00 equiv). Yield: 33% (105 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.18 (br s, 1H), 7.28 (d, <i>J</i> = 8.4 Hz, 2H), 7.25 (d, <i>J</i> = 8.4 Hz, 2H), 6.24 (br s, 2H), 3.94 (s, 2H), 2.10–2.00 (m, 1H), 1.04–0.94 (m, 2H), 0.93–0.81 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 316.10, found 316.13. HPLC: 6.393 min, purity 96%.</div></div><div id="sec4_4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> 2-Amino-5-cyclopropyl-6-(4-bromobenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>22</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (120 mg, 1.21 mmol, 2.00 equiv), <b>4bo</b> (200 mg, 0.615 mmol, 1.00 equiv), and BMIM-PF6 (1,00 mL, 4.86 mmol, 7.90 equiv). Yield: 21% (47 mg) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.15 (s, 1H), 7.42 (d, <i>J</i> = 8.4 Hz, 2H), 7.20 (d, <i>J</i> = 8.0 Hz, 2H), 6.24 (br s, 2H), 3.92 (s, 2H), 2.10–2.00 (m, 1H), 1.08–0.75 (m, 4H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>14</sub>BrN<sub>5</sub>O [M + H]<sup>+</sup> 360.05, found 360.1. HPLC: 6.474 min, purity 96%.</div></div><div id="sec4_4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> 2-Amino-5-cyclopropyl-6-(4-methoxybenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>23</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (165 mg, 1.70 mmol, 2.00 equiv), <b>4bp</b> (235 mg, 0.85 mmol, 1.00 equiv), and BMIM-PF6 (1.05 mL, 5.10 mmol, 6.00 equiv). Yield: 41% (109 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.07 (br s, 1H), 7.15 (d, <i>J</i> = 8.4 Hz, 2H), 6.80 (d, <i>J</i> = 8.8 Hz, 2H), 6.19 (br s, 2H), 3.88 (s, 2H), 3.69 (s, 3H), 2.12–2.01 (m, 1H), 1.04–0.95 (m, 2H), 0.94–0.86 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>17</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 312.15, found 312.07. HPLC: 5.753 min, purity 96%.</div></div><div id="sec4_4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> 2-Amino-6-(3-chlorobenzyl)-5-methyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>24</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (94 mg, 0.95 mmol, 2.00 equiv), <b>4aa</b> (120 mg, 0.47 mmol, 1.00 equiv), and BMIM-PF6 (0.58 mL, 2.82 mmol, 6.00 equiv). Yield: 57% (77 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.56 (s, 1H), 7.30–7.21 (m, 3H), 7.17 (d, <i>J</i> = 7.2 Hz, 1H), 5.99 (s, 2H), 3.78 (s, 2H), 2.25 (s, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>12</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 290.08, found 290.1. HPLC: 5.709 min, purity 99%.</div></div><div id="sec4_4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> 2-Amino-6-(3-chlorobenzyl)-5-ethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>25</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (141 mg, 1.43 mmol, 2.00 equiv), <b>4ca</b> (192 mg, 0.71 mmol, 1.00 equiv), and BMIM-PF6 (0.88 mL, 4.28 mmol, 6.00 equiv). Yield: 21% (46 mg) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO): δ 12.57 (s, 1H) 7.29–7.51 (m, 2H), 7.22 (d, <i>J</i> = 8.0 Hz, 1H), 7.16 (d, <i>J</i> = 7.5 Hz, 1H), 6.02 (s, 2H), 3.81 (s, 2H), 2.57 (q, <i>J</i> = 7.5 Hz, 2H), 1.05 (t, J= 7.5 Hz, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>14</sub>H<sub>14</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 304.10, found 304.2. HPLC: 6.156 min, purity 95%.</div></div><div id="sec4_4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> 2-Amino-6-(3-chlorobenzyl)-5-propyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>26</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (210 mg, 2.12 mmol, 2.00 equiv), <b>4da</b> (300 mg, 1.06 mmol, 1.00 equiv), and BMIM-PF6 (1.31 mL, 6.36 mmol, 6.00 equiv). Yield: 22% (76 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.56 (s, 1H), 7.29–7.20 (m, 3H), 7.16 (d, <i>J</i> = 7.6 Hz, 1H), 6.01 (s, 2H), 3.81 (s, 2H), 2.53 (t, <i>J</i> = 7.6 Hz, 2H), 1.45 (sextet, <i>J</i> = 7.6 Hz, 2H), 0.85 (t, <i>J</i> = 7.2 Hz, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>16</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 318.11, found 318.2. HPLC: 6.660 min, purity 99%.</div></div><div id="sec4_4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> 2-Amino-6-(3-chlorobenzyl)-5-isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>27</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (284 mg, 2.87 mmol, 2.00 equiv), <b>1ea</b> (407 mg, 1.44 mmol, 1.00 equiv), and BMIM-PF6 (1.76 mL, 8.63 mmol, 6.00 equiv). Yield: 21% (96 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.33 (s, 1H), 7.28 (t, <i>J</i> = 8.0 Hz, 1H), 7.35 (s, 1H), 7.21 (d, <i>J</i> = 8.0 Hz, 1H), 7.14 (d, <i>J</i> = 7.6 Hz, 1H), 6.04 (s, 2H) 3.89 (s, 2H), 3.18 (septet, J= 6.8 Hz, 1H), 1.12 (d, J= 6.8 Hz, 6H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>16</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 318.11, found 318.2. HPLC: 6.596 min, purity 99%.</div></div><div id="sec4_4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> 2-Amino-5-butyl-6-(3-chlorobenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>28</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (250 mg, 1.51 mmol, 2.14 equiv), <b>1fa</b> (210 mg, 0.708 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 6.86 equiv). Yield: 20% (47 mg) as a white solid. Purity: 98%. <sup>1</sup>H NMR (400 MHz, DMSO) δ 12.54 (s, 1H), 7.33–7.19 (m, 3H), 7.16 (d, <i>J</i> = 7.3 Hz, 1H), 6.02 (s, 2H), 3.82 (s, 2H), 2.60–2.50 (m, 2H), 1.45–1.32 (m, 2H), 1.32–1.20 (m, 2H), 0.81 (t, <i>J</i> = 7.1 Hz, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>18</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 332.13, found 332.3. HPLC: 7.027 min, purity 98%.</div></div><div id="sec4_4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> 2-Amino-6-(3-chlorobenzyl)-5-(2-ethylbutyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>29</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (130 mg, 1.31 mmol, 2.13 equiv), <b>4ga</b> (200 mg, 0.616 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 7.89 equiv). Yield: 9% (20 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.49 (br s, 1H), 7.28 (t, <i>J</i> = 7.2 Hz, 1H), 7.24–7.18 (m, 2H), 7.14 (d, <i>J</i> = 7.2 Hz, 1H), 5.91 (br s, 2H), 3.82 (s, 2H), 1.58 (s, 2H), 1.30–1.15 (m, 5H), 0.77 (t, <i>J</i> = 6.8 Hz, 6H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>22</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 360.16, found 360.3. HPLC: 7.640 min, purity 97%.</div></div><div id="sec4_4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> 2-Amino-6-(3-chlorobenzyl)-5-pentyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>30</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (120 mg, 1.21 mmol, 1.88 equiv), <b>4ha</b> (200 mg, 0.643 mmol; 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 7.56 equiv). Yield: 24% (52 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.43 (br s, 1H), 7.35–7.10 (m, 4H), 6.02 (br s, 2H), 3.82 (s, 2H), 1.45–1.15 (m, 6H), 0.79 (s, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>20</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 346.15, found 346.2. HPLC: 7.376 min, purity 98%.</div></div><div id="sec4_4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> 2-Amino-6-(3-chlorobenzyl)-5-cyclopentyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>31</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (130 mg, 1.31 mmol, 2.00 equiv), <b>4ia</b> (200 mg, 0.647 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 7.51 equiv). Yield: 38% (86 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.36 (br s, 1H), 7.38–7.07 (m, 4H), 6.07 (br s, 2H), 3.90 (s, 2H), 3.23 (m, 1H), 1.90–1.45 (m, 8H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>18</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 344.13, found 344.1. HPLC: 7.049 min, purity 99%.</div></div><div id="sec4_4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> 2-Amino-6-(3-chlorobenzyl)-5-hexyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>32</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (86 mg, 0.86 mmol, 2.00 equiv), <b>4ja</b> (140 mg, 0.431 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 11.3 equiv). Yield: 83% (128 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.50 (s, 1H), 7.33–7.19 (m, 3H), 7.15 (d, <i>J</i> = 6.8 Hz, 1H), 6.02 (s, 2H), 3.82 (s, 2H), 2.60–2.40 (m, 2H), 1.44–1.32 (m, 2H), 1.30–1.08 (m, 6H), 0.82 (t, <i>J</i> = 6.5 Hz, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>22</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 360.16, found 360.3. HPLC: 7.843 min, purity 97%.</div></div><div id="sec4_4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 2-Amino-6-(3-chlorobenzyl)-5-heptyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>33</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (116 mg, 1.18 mmol, 2.00 equiv), <b>4ka</b> (200 mg, 0.591 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 8.22 equiv). Yield: 50% (108 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.55 (s, 1H), 7.30–7.19 (m, 3H), 7.15 (d, <i>J</i> = 6.4 Hz, 1H), 6.01 (s, 1H), 3.82 (s, 2H), 1.46–1.32 (m, 2H), 1.27–1.10 (m, 8H), 0.83 (t, <i>J</i> = 6.4 Hz, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>24</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 374.18, found 374.3. HPLC: 8.200 min, purity 98%.</div></div><div id="sec4_4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> 2-Amino-6-(3-chlorobenzyl)-5-phenyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>34</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (177 mg, 1.79 mmol, 2.00 equiv), <b>4la</b> (296 mg, 0.89 mmol, 1.00 equiv), and BMIM-PF6 (1.11 mL, 5.36 mmol, 6.00 equiv). Yield: 30% (95 mg) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO): δ 12.83 (s, 1H), 7.53–7.47 (m, 3H), 7.42 (dd, <i>J</i> = 8.0, 1.5 Hz, 2H), 7.20 (t, <i>J</i> = 8.0 Hz, 1H), 7.17 (d, <i>J</i> = 7.0 Hz, 1H), 7.03 (s, 1H), 6.94 (d, <i>J</i> = 7.0 Hz, 1H), 6.12 (s, 2H), 3.64 (s, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>14</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 352.10, found 352.2. HPLC: 6.946 min, purity 99%.</div></div><div id="sec4_4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> 2-Amino-6-(3-chlorobenzyl)-5-(<i>p</i>-tolyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>35</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (181 mg, 1.82 mmol, 2.00 equiv), <b>4ma</b> (314 mg, 0.937 mmol, 1.00 equiv), and BMIM-PF6 (1.5 mL, 7.28 mmol, 7.77 equiv). Yield: 33% (108 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.80 (s, 1H), 7.33–7.28 (m, 4H), 7.26–7.14 (m, 2H), 7.05 (s, 1H), 6.96 (d, <i>J</i> = 7.1 Hz, 1H), 6.11 (s, 2H), 3.64 (s, 2H), 2.36 (s, 3H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>16</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 366.20, found 366.2. HPLC: 7.342 min, purity 98%.</div></div><div id="sec4_4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Procedure for the synthesis of 2-Amino-6-(3-chlorobenzyl)-5-phenethyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>36</b>)</h3><div class="NLM_p last">A mixture of <b>4na</b> (213 mg, 0.57 mmol, 1.00 equiv), 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (116 mg, 1.16 mmol, 2.00 equiv), and <i>ortho</i>-phosphoric acid 85% (59.6 μL, 0.906 mmol,1.58 equiv) in EtOH (1 mL) was stirred for 1 min at 20 °C and then heated at 175 °C under microwave irradiation for 3 h. The reaction mixture was allowed to cool to room temperature, and CH<sub>2</sub>Cl<sub>2</sub> (30 mL), water (10 mL), and aqueous citric acid (5%, 1 mL) were added. The resulting precipitate was stirred for 20 min, filtered, and the residue was washed with hot methanol, collected, and dried in vacuo to afford the title compound. Yield: 22% (49 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.67 (s, 1H), 7.32–7.18 (m, 6H), 7.17–7.10 (m, 3H), 6.06 (br s, 2H), 3.76 (s, 2H), 2.84–2.78 (m, 2H), 2.70–2.66 (m, 2H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>ClN<sub>5</sub>O [M + H]<sup>+</sup> 380.13, found 380.2. HPLC: 7.392 min, purity 97%.</div></div><div id="sec4_4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> 2-Amino-5-cyclopropyl-6-(3,4-dichlorobenzyl)[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>37</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (157 mg, 1.58 mmol, 2.00 equiv), <b>4bq</b> (250 mg, 0.79 mmol, 1.00 equiv), and BMIM-PF6 (1.3 mL, 4.74 mmol, 6.00 equiv). Yield: 31% (86 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.26 (br s, 1H), 7.52–7.47 (m, 2H), 7.22 (d, <i>J</i> = 8.4 Hz, 2H), 6.26 (br s, 2H), 3.95 (s, 2H), 2.12–2.01 (m, 1H), 1.01–0.85 (m, 4H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 350.06, found 350.1. HPLC: 6.836 min, purity 96%.</div></div><div id="sec4_4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> 2-Amino-6-(3,4-dichlorobenzyl)-5-isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>38</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (125 mg, 1.26 mmol, 2.00 equiv), <b>4eq</b> (200 mg, 0.63 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 7.71 equiv). Yield: 34% (76 mg), as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.23 (br s, 1H), 7.54–7.43 (m, 2H), 7.17 (d, <i>J</i> = 8.0 Hz, 1H), 6.09 (s, 2H), 3.88 (s, 2H), 3.22–3.13 (m, 1H), 1.12 (d, <i>J</i> = 6.4 Hz, 6H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 352.08, found 352.3. HPLC: 7.008 min, purity 97%.</div></div><div id="sec4_4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> 2-Amino-6-(2,3-dichlorobenzyl)-5-isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>39</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (125 mg, 1.26 mmol, 2.00 equiv), <b>4er</b> (200 mg, 0.630 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 7.71 equiv). Yield: 48% (106 mg) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 80 °C): δ 7.44 (dd, <i>J</i> = 7.9, 1.5 Hz, 1H), 7.21 (t, <i>J</i> = 7.9 Hz, 1H), 6.97 (dd, <i>J</i> = 7.2, 1.5 Hz, 1H), 5.83 (s br, 2H), 3.98 (s, 2H), 2.96 (pentet, <i>J</i> = 6.9 Hz, 1H), 1.16 (d, <i>J</i> = 6.9 Hz, 6H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.8, 156.0, 155.6, 149.7, 140.1, 131.4, 130.5, 127.7, 127.4, 126.9, 104.5, 29.4, 28.3, 19.6 ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 352.08, found 352.2. HPLC: 7.060 min, purity 97%.</div></div><div id="sec4_4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> 2-Amino-6-(2,5-dichlorobenzyl)-5-isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>40</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (125 mg, 1.26 mmol, 2.00 equiv), <b>4es</b> (200 mg, 0.630 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 7.71 equiv). Yield: 24% (54 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.41 (br s, 1H), 7.70–7.42 (m, 1H), 7.40–7.19 (m, 1H), 7.16–6.85 (m, 1H), 6.08 (br s, 2H), 3.88 (s, 2H), 3.01–2.83 (m, 1H), 1.14 (s, 6H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 352.08, found 352.2. HPLC: 6.938 min, purity 96%.</div></div><div id="sec4_4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> 2-Amino-6-(3,5-dichlorobenzyl)-5-isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>41</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (96 mg, 0.97 mmol, 2 equiv), <b>4et</b> (153 mg, 0.48 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 10 equiv). Yield: 18% (31 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.33 (s, 1H), 7.41 (s, 1H), 7.26 (s, 2H), 6.07 (br s, 2H), 3.89 (s, 2H), 1.13 (s, 6H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 352.08, found 352.1. HPLC: 7.103 min, purity 98%.</div></div><div id="sec4_4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> 2-Amino-6-(3,5-dibromobenzyl)-5-isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin 7(4<i>H</i>)-one (<b>42</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (97 mg, 0.98 mmol, 2.00 equiv), <b>4eu</b> (200 mg, 0.49 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 9.91 equiv). Yield: 7% (14 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 12.33 (br s, 1H), 7.63 (s, 1H), 7.42 (s, 2H), 6.07 (br s, 2H), 3.88 (s, 2H), 1.13 (s, 6H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>Br<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup> 439.97, found 440.1. HPLC: 7.337 min, purity 98%.</div></div><div id="sec4_4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> 2-Amino-6-(3-bromo-4-chloro-benzyl)-5-isopropyl[1,2,4]triazolo[1,5-<i>a</i>]pyrimidin-7(4<i>H</i>)-one (<b>43</b>)</h3><div class="NLM_p last">Compound was synthesized according to general procedure 3, using the following reagents: 3,5-diamino-4<i>H</i>-1,2,4-triazole <b>5c</b> (109 mg, 1.10 mmol, 2.00 equiv), <b>4ev</b> (200 mg, 0.553 mmol, 1.00 equiv), and BMIM-PF6 (1.00 mL, 4.86 mmol, 8.83 equiv). Yield: 24% (53 mg) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): δ 7.60 (d, <i>J</i> = 2.0 Hz, 1H), 7.49 (d, <i>J</i> = 8.0 Hz, 1H), 7.21 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 6.08 (br s, 2H), 3.87 (s, 2H), 3.22–3.11 (m, 1H), 1.12 (d, <i>J</i> = 7.2 Hz, 6H) ppm. LC–MS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>15</sub>H<sub>15</sub>BrClN<sub>5</sub>O [M + H]<sup>+</sup> 396.02, found 396.1. HPLC: 7.080 min, purity 98%.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> Chemicals and Reagents</h3><div class="NLM_p last">The human recombinant chemokines CCL2 and CCL3 were purchased from PeproTech (Rocky Hill, NJ). TAK-779 was obtained from NIH AIDS reagent program (Germantown, MD, catalogue number 4983). All triazolopyrimidinone derivatives were synthesized in-house. Guanosine 5′-<i>O</i>-[γ-thio]triphosphate ([<sup>35</sup>S[GTPγS) (specific activity 1250 Ci/mmol) was purchased from PerkinElmer (Waltham, MA), while [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] (specific activity 59.6 Ci mmol<sup>–1</sup>) was custom-labeled by Vitrax (Placentia, CA). Bovine serum albumin was purchased from Sigma-Aldrich (St. Louis, MO). Bicinchoninic acid (BCA) and Pierce BCA protein assay kit were purchased from Pierce Biotechnology (Thermo Scientific, Rockford, IL). Tango U2OS cells stably expressing human CCR2b (U2OS-CCR2) or human CCR5 (U2OS-CCR5) were purchased from Invitrogen (Carlsbad, CA). All other chemicals were obtained from standard commercial sources.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> Cell Culture</h3><div class="NLM_p last">Both U2OS-CCR2b and U2OS-CCR5 cells were cultured in McCoy’s 5A medium supplemented with 10% (v/v) fetal calf serum, 2 mM glutamine, 0.1 mM nonessential amino acids, 25 mM HEPES, 1 mM sodium pyruvate, 200 IU/mL penicillin, 200 μg/mL streptomycin, 100 μg/mL G418, 40–50 μg/mL hygromycin, and 125 μg/mL zeocin. Cells were grown until 80% confluence and cultured twice-weekly on 10 or 15 cm ⌀ plates by trypsinization. Dialyzed fetal calf serum was used when culturing cells for functional assays or as a last step before membrane preparation.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> Membrane Preparation</h3><div class="NLM_p last">Membranes from U2OS-CCR2 or U2OS-CCR5 cells were prepared as previously described for CCR2.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Briefly, U2OS-CCR2 or U2OS-CCR5 cells were scraped from confluent 15 cm ⌀ plates using phosphate buffered saline (PBS) and subsequently centrifuged at 3000 rpm for 5 min. Pellets were then resuspended in ice-cold Tris buffer (50 mM Tris-HCl, 5 mM MgCl<sub>2</sub>, pH 7.4) before homogenization with an Ultra Turrax homogenizer (IKA-Werke GmbH & Co. KG, Staufen, Germany). Membranes and cytosolic contents were separated using an Optima LE-80 K ultracentrifuge (Beckman Coulter, Inc., Fullerton, CA) at 31 000 rpm for 20 min at 4 °C. After a second cycle of homogenization and centrifugation, the final pellet was resuspended and homogenized in ice-cold Tris buffer, aliquoted, and stored at −80 °C. Finally, membrane protein concentrations were determined using a BCA protein determination assay, as described by the manufacturer (Pierce BCA protein assay kit).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] Binding Assays</h3><div class="NLM_p last">For [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] displacement assays, U2OS-CCR2b membrane homogenates (15–20 μg of total protein) were incubated with ∼6 nM [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] and at least 6 increasing concentrations of competing ligand in a final volume of 100 μL of assay buffer (50 mM Tris-HCl, 5 mM MgCl<sub>2</sub>, 0.1% CHAPS, pH 7.4). Ligands were diluted to the desired concentration with an HP D300 digital dispenser (Tecan, Giessen, The Netherlands). Total radioligand binding did not exceed 10% of the amount added to prevent ligand depletion, and nonspecific binding was determined using 10 μM CCR2-RA-[<i>R</i>]. After 2 h at 25 °C, incubation was terminated by rapid filtration through a 96-well GF/B filterplate on a PerkinElmer FilterMate harvester, using ice-cold wash buffer (50 mM Tris-HCl buffer supplemented with 5 mM MgCl<sub>2</sub> and 0.05% CHAPS, pH 7.4). Filters were washed 10 times with ice-cold wash buffer and subsequently dried at 55 °C for 30 min. After addition of 25 μL of Microscint scintillation cocktail (PerkinElmer), the filter-bound radioactivity was measured by scintillation spectrometry using the P-E 2450 Microbeta<sup>2</sup> counter (PerkinElmer).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> Tango β-Arrestin Recruitment Assay</h3><div class="NLM_p last">β-Arrestin recruitment was measured using the Tango CCR2-<i>bla</i> or CCR5-<i>bla</i> U2OS cell-based assay (Invitrogen) according to the manufacturer’s protocol. Briefly, U2OS-CCR2b or U2OS-CCR5 cells were grown until approximately 80% confluence and detached by trypsinization. Cells were recovered by centrifugation at 1000 rpm for 5 min, resuspended in assay medium (FreeStyle Expression Medium, Invitrogen) to a density of 10 000 cells per well and seeded into black-wall, clear-bottom, 384-well assay plates (Corning). For agonist assays, cells were exposed to increasing concentrations of CCL2 or CCL3 for CCR2 or CCR5, respectively, for 16 h at 37 °C and 5% CO<sub>2</sub>. For antagonist assays, compounds were first diluted in assay medium containing a final DMSO concentration of 0.5% or lower. Cells were then preincubated with either 1 μM (for single-point inhibition experiments) or increasing concentrations of antagonist for 30 min at room temperature, before a 16 hour co-incubation with an EC<sub>80</sub> concentration of CCL2 (5 nM) or CCL3 (14 nM) at 37 °C and 5% CO<sub>2</sub>. After 16 h cells were loaded in the dark with 8 μL of LiveBLAzer-FRET B/G substrate (Invitrogen) and incubated for 2 h at room temperature. Finally, fluorescence emission at 460 and 535 nm was measured in an EnVision multilabel plate reader (PerkinElmer) after excitation at 400 nm. The ratio of emission at 460 and 535 nm was calculated for each well.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> [<sup>35</sup>S]GTPγS Binding Assay</h3><div class="NLM_p last">To determine the mechanism of inhibition [<sup>35</sup>S]GTPγS binding assays were performed. In CCR2 the [<sup>35</sup>S]GTPγS binding assay was performed as previously described.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> In the case of CCR5, an amount of 10 μg of U2OS-CCR5 membranes with 0.25 mg/mL saponin was preincubated with 5 μM GDP, increasing concentrations of CCL3, and three different antagonist concentrations for 30 min at 25 °C. All dilutions were made in assay buffer containing 50 mM Tris-HCl (pH 7.4), 10 mM MgCl<sub>2</sub>, 10 mM NaCl, 1 mM EDTA, and 0.05% BSA. [<sup>35</sup>S]GTPγS (0.3 nM) was added, and the mixture was co-incubated for an additional 90 min at 25 °C before harvesting. Incubation was stopped by dilution with ice-cold 50 mM Tris-HCl buffer with 5 mM MgCl<sub>2</sub>. Separation of bound and unbound radioligand was performed by rapid filtration through a 96-well GF/B filter plate as described in “[<sup>3</sup>H]-CCR2-RA-[<i>R</i>] Binding Assays”.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> Data Analysis</h3><div class="NLM_p last">All experiments were analyzed using nonlinear regression curve fitting program Prism 7 (GraphPad, San Diego, CA). EC<sub>50</sub>, EC<sub>80</sub>, <i>E</i><sub>max</sub>, and IC<sub>50</sub> values from functional assays were obtained by nonlinear regression analysis. All values obtained are the mean ± SEM of at least three separate experiments performed in duplicate, unless stated otherwise. For radioligand binding assays, <i>K</i><sub>i</sub> values were determined using the Cheng–Prussoff equation using a <i>K</i><sub>D</sub> of 6.3 nM for the radioligand.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> Computational Modeling</h3><div class="NLM_p last">The inactive crystal structure of hCCR2b with BMS-681 and CCR2-RA-[<i>R</i>] (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1A">5T1A</a>)<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> was used as the basis for docking compounds <b>8</b>, <b>39</b>, <b>40</b>, and <b>43</b>. Docking was performed in the Schrodinger suite,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> as previously described for the docking of CCR2-RA-[<i>R</i>].<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Before docking, the CCR2b crystal structure was prepared by replacing the residues between L226<sup>5×62</sup> and R240<sup>6×32</sup>, which correspond to the M2 muscarinic acetylcholine receptor, with the CCR2b sequence using prime<a onclick="showRef(event, 'ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref56 ref57">(55−57)</a> and CCR5 as template (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MBS">4MBS</a>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Induced fit docking, with a substructure restraint on the right-hand phenyl (R<sup>1</sup>, SMARTS: “c1ccccc1”) was used to dock compound <b>43</b> in the hCCR2b model.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> Compounds <b>8</b>, <b>39</b>, and <b>40</b> were docked using regular docking (Glide-SP) on the basis of the model of compound <b>43</b>. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b, and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">S4</a> were rendered using PyMOL.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i108"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00742" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87553" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87553" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b00742?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00742</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Characterization of ligands in a U2OS-CCR5 β-arrestin-recruitment assay (Figure S1); correlation between log <i>P</i> (cLogP) and affinity (p<i>K</i><sub>i</sub>) values in CCR2 (Figure S2); characterization of compounds <b>39</b> and <b>43</b> as potential inverse agonists in hCCR2 (Figure S3); docking of compounds <b>8</b>, <b>39</b>, <b>40</b>, and <b>43</b> (Figure S4); list of intermediate compounds <b>4aa</b>–<b>na</b>, <b>4bb</b>–<b>bq</b>, <b>4eq</b>–<b>ev</b> (Table S1); functional activity of TAK-779 and CCR2-RA-[<i>R</i>] in hCCR5, using a CCL3-induced β-arrestin recruitment assay (Table S2); and functional activity of compounds <b>8</b>, <b>39</b>, and <b>43</b> in hCCR2, using a CCL2-induced β-arrestin recruitment assay (Table S3) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Model of CCR2b (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_007.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Model of compound <b>8</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Model of compound <b>39</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Model of compound <b>40</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Model of compound <b>43</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_006.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf">jm9b00742_si_001.pdf (1.53 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_002.csv">jm9b00742_si_002.csv (2.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_003.pdb">jm9b00742_si_003.pdb (3.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_004.pdb">jm9b00742_si_004.pdb (4.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_005.pdb">jm9b00742_si_005.pdb (4.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_006.pdb">jm9b00742_si_006.pdb (4.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_007.pdb">jm9b00742_si_007.pdb (377.37 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00742" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65736" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65736" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adriaan P. IJzerman</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1182-2259" title="Orcid link">http://orcid.org/0000-0002-1182-2259</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3b5251415e49565a557b575a585f4915575e525f5e554e55524d155557"><span class="__cf_email__" data-cfemail="543d3e2e312639353a1438353730267a38313d30313a213a3d227a3a38">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalia V. Ortiz Zacarías</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacobus P. D. van Veldhoven</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa S. den Hollander</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Burak Dogan</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph Openy</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ya-Yun Hsiao</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eelke B. Lenselink</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura H. Heitman</span> - <span class="hlFld-Affiliation affiliation">Division of Drug Discovery and Safety,
Leiden Academic Centre for Drug Research, Leiden University, P.O. Box 9502, 2300 RA Leiden, The Netherlands</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i110">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67332" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67332" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The following reagent was obtained through the NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH: TAK-779.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CCR2</td><td class="NLM_def"><p class="first last">CC chemokine receptor 2</p></td></tr><tr><td class="NLM_term">CCR5</td><td class="NLM_def"><p class="first last">CC chemokine receptor 5</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–affinity/activity relationship</p></td></tr><tr><td class="NLM_term">U2OS</td><td class="NLM_def"><p class="first last">human osteosarcoma cell</p></td></tr><tr><td class="NLM_term">IP</td><td class="NLM_def"><p class="first last">inositol phosphate</p></td></tr><tr><td class="NLM_term">[<sup>35</sup>S]GTPγS</td><td class="NLM_def"><p class="first last">guanosine 5′-<i>O</i>-(3-[<sup>35</sup>S]thio)triphosphate</p></td></tr><tr><td class="NLM_term">BCA</td><td class="NLM_def"><p class="first last">bicinchoninic acid</p></td></tr><tr><td class="NLM_term">cryo-EM</td><td class="NLM_def"><p class="first last">cryoelectron microscopy</p></td></tr><tr><td class="NLM_term">NEAA</td><td class="NLM_def"><p class="first last">nonessential amino acid</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate buffered saline</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i112">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60536" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60536" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachelerie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combadiere, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farber, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparre-Ulrich, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luster, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Power, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkilde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sozzani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thelen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshie, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnik, A.</span></span> <span> </span><span class="NLM_article-title">International union of basic and clinical pharmacology. LXXXIX. Update on the extended family of chemokine receptors and introducing a new nomenclature for atypical chemokine receptors</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1124/pr.113.007724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fpr.113.007724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24218476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2014&pages=1-79&issue=1&author=F.+Bachelerieauthor=A.+Ben-Baruchauthor=A.+M.+Burkhardtauthor=C.+Combadiereauthor=J.+M.+Farberauthor=G.+J.+Grahamauthor=R.+Horukauthor=A.+H.+Sparre-Ulrichauthor=M.+Locatiauthor=A.+D.+Lusterauthor=A.+Mantovaniauthor=K.+Matsushimaauthor=P.+M.+Murphyauthor=R.+Nibbsauthor=H.+Nomiyamaauthor=C.+A.+Powerauthor=A.+E.+Proudfootauthor=M.+M.+Rosenkildeauthor=A.+Rotauthor=S.+Sozzaniauthor=M.+Thelenauthor=O.+Yoshieauthor=A.+Zlotnik&title=International+union+of+basic+and+clinical+pharmacology.+LXXXIX.+Update+on+the+extended+family+of+chemokine+receptors+and+introducing+a+new+nomenclature+for+atypical+chemokine+receptors&doi=10.1124%2Fpr.113.007724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">International union of basic and clinical pharmacology LXXXIX. Update on the extended family of chemokine receptors and introducing a new nomenclature for atypical chemokine receptors</span></div><div class="casAuthors">Bachelerie, Francoise; Ben-Baruch, Adit; Burkhardt, Amanda M.; Combadiere, Christophe; Farber, Joshua M.; Graham, Gerard J.; Horuk, Richard; Sparre-Ulrich, Alexander Hovard; Locati, Massimo; Luster, Andrew D.; Mantovani, Alberto; Matsushima, Kouji; Murphy, Philip M.; Nibbs, Robert; Nomiyama, Hisayuki; Power, Christine A.; Proudfoot, Amanda E. I.; Rosenkilde, Mette M.; Rot, Antal; Sozzani, Silvano; Thelen, Marcus; Yoshie, Osamu; Zlotnik, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-79, 80 pp.</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Sixteen years ago, the Nomenclature Committee of the International Union of Pharmacol. approved a system for naming human seven-transmembrane (7TM) G protein-coupled chemokine receptors, the large family of leukocyte chemoattractant receptors that regulates immune system development and function, in large part by mediating leukocyte trafficking.  This was announced in Pharmacol. Reviews in a major overview of the first decade of research in this field.  Since then, several new receptors have been discovered, and major advances have been made for the others in many areas, including structural biol., signal transduction mechanisms, biol., and pharmacol.  New and diverse roles have been identified in infection, immunity, inflammation, development, cancer, and other areas.  The first two drugs acting at chemokine receptors have been approved by the U.S. Food and Drug Administration (FDA), maraviroc targeting CCR5 in human immunodeficiency virus (HIV)/AIDS, and plerixafor targeting CXCR4 for stem cell mobilization for transplantation in cancer, and other candidates are now undergoing pivotal clin. trials for diverse disease indications.  In addn., a subfamily of atypical chemokine receptors has emerged that may signal through arrestins instead of G proteins to act as chemokine scavengers, and many microbial and invertebrate G protein-coupled chemokine receptors and sol. chemokine-binding proteins have been described.  Here, we review this extended family of chemokine receptors and chemokine-binding proteins at the basic, translational, and clin. levels, including an update on drug development.  We also introduce a new nomenclature for atypical chemokine receptors with the stem ACKR (atypical chemokine receptor) approved by the Nomenclature Committee of the International Union of Pharmacol. and the Human Genome Nomenclature Committee.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7EhMCJlz-zLVg90H21EOLACvtfcHk0lgnyFHekgjavw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D&md5=44a2c14ddf2508ada59121610c5682cf</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fpr.113.007724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.113.007724%26sid%3Dliteratum%253Aachs%26aulast%3DBachelerie%26aufirst%3DF.%26aulast%3DBen-Baruch%26aufirst%3DA.%26aulast%3DBurkhardt%26aufirst%3DA.%2BM.%26aulast%3DCombadiere%26aufirst%3DC.%26aulast%3DFarber%26aufirst%3DJ.%2BM.%26aulast%3DGraham%26aufirst%3DG.%2BJ.%26aulast%3DHoruk%26aufirst%3DR.%26aulast%3DSparre-Ulrich%26aufirst%3DA.%2BH.%26aulast%3DLocati%26aufirst%3DM.%26aulast%3DLuster%26aufirst%3DA.%2BD.%26aulast%3DMantovani%26aufirst%3DA.%26aulast%3DMatsushima%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DP.%2BM.%26aulast%3DNibbs%26aufirst%3DR.%26aulast%3DNomiyama%26aufirst%3DH.%26aulast%3DPower%26aufirst%3DC.%2BA.%26aulast%3DProudfoot%26aufirst%3DA.%2BE.%26aulast%3DRosenkilde%26aufirst%3DM.%2BM.%26aulast%3DRot%26aufirst%3DA.%26aulast%3DSozzani%26aufirst%3DS.%26aulast%3DThelen%26aufirst%3DM.%26aulast%3DYoshie%26aufirst%3DO.%26aulast%3DZlotnik%26aufirst%3DA.%26atitle%3DInternational%2520union%2520of%2520basic%2520and%2520clinical%2520pharmacology.%2520LXXXIX.%2520Update%2520on%2520the%2520extended%2520family%2520of%2520chemokine%2520receptors%2520and%2520introducing%2520a%2520new%2520nomenclature%2520for%2520atypical%2520chemokine%2520receptors%26jtitle%3DPharmacol.%2520Rev.%26date%3D2014%26volume%3D66%26issue%3D1%26spage%3D1%26epage%3D79%26doi%3D10.1124%2Fpr.113.007724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Cotarelo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Moreira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criado-Garcia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Fernandez, J. L.</span></span> <span> </span><span class="NLM_article-title">Beyond chemoattraction: multifunctionality of chemokine receptors in leukocytes</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">941</span>, <span class="refDoi"> DOI: 10.1016/j.it.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.it.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28935522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=927-941&issue=12&author=P.+Lopez-Cotareloauthor=C.+Gomez-Moreiraauthor=O.+Criado-Garciaauthor=L.+Sanchezauthor=J.+L.+Rodriguez-Fernandez&title=Beyond+chemoattraction%3A+multifunctionality+of+chemokine+receptors+in+leukocytes&doi=10.1016%2Fj.it.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond Chemoattraction: Multifunctionality of Chemokine Receptors in Leukocytes</span></div><div class="casAuthors">Lopez-Cotarelo, Pilar; Gomez-Moreira, Carolina; Criado-Garcia, Olga; Sanchez, Lucas; Rodriguez-Fernandez, Jose Luis</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-941</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The word chemokine is a combination of the words chemotactic and cytokine, in other words cytokines that promote chemotaxis.  Hence, the term chemokine receptor refers largely to the ability to regulate chemoattraction.  However, these receptors can modulate addnl. leukocyte functions, as exemplified by the case of CCR7 which, apart from chemotaxis, regulates survival, migratory speed, endocytosis, differentiation and cytoarchitecture.  We present evidence highlighting that multifunctionality is a common feature of chemokine receptors.  Based on the activities that they regulate, we suggest that chemokine receptors can be classified into inflammatory (which control both inflammatory and homeostatic functions) and homeostatic families.  The information accrued also suggests that the non-chemotactic functions controlled by chemokine receptors may contribute to optimizing leukocyte functioning under normal physiol. conditions and during inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqERh6uUdiQvbVg90H21EOLACvtfcHk0lgnyFHekgjavw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnE&md5=8930a5b6d6d9e26235d406fe90cb0441</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Cotarelo%26aufirst%3DP.%26aulast%3DGomez-Moreira%26aufirst%3DC.%26aulast%3DCriado-Garcia%26aufirst%3DO.%26aulast%3DSanchez%26aufirst%3DL.%26aulast%3DRodriguez-Fernandez%26aufirst%3DJ.%2BL.%26atitle%3DBeyond%2520chemoattraction%253A%2520multifunctionality%2520of%2520chemokine%2520receptors%2520in%2520leukocytes%26jtitle%3DTrends%2520Immunol.%26date%3D2017%26volume%3D38%26issue%3D12%26spage%3D927%26epage%3D941%26doi%3D10.1016%2Fj.it.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schall, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, A. E.</span></span> <span> </span><span class="NLM_article-title">Overcoming hurdles in developing successful drugs targeting chemokine receptors</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">363</span>, <span class="refDoi"> DOI: 10.1038/nri2972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnri2972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=21494268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslCmtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=355-363&issue=5&author=T.+J.+Schallauthor=A.+E.+Proudfoot&title=Overcoming+hurdles+in+developing+successful+drugs+targeting+chemokine+receptors&doi=10.1038%2Fnri2972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming hurdles in developing successful drugs targeting chemokine receptors</span></div><div class="casAuthors">Schall, Thomas J.; Proudfoot, Amanda E. I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-363</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemokines and their receptors are central to the inflammatory process and are attractive therapeutic targets.  Drugs that inhibit chemokine receptors are approved for the treatment of HIV infection and for stem cell mobilization, but none have been approved yet for the treatment of inflammatory and/or autoimmune diseases.  We analyze the challenges of developing chemokine receptor antagonists, and propose that inappropriate target selection and ineffective dosing, not the 'redundancy' of the chemokine system, are the main barriers to their use as anti-inflammatory therapies.  We highlight evidence suggesting that chemokine receptor inhibition will prove to be an effective therapy in inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJxZhyoAteTLVg90H21EOLACvtfcHk0lg85RNgu20tng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslCmtb0%253D&md5=3173fc889bd22b7d7e19bd138a599897</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnri2972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2972%26sid%3Dliteratum%253Aachs%26aulast%3DSchall%26aufirst%3DT.%2BJ.%26aulast%3DProudfoot%26aufirst%3DA.%2BE.%26atitle%3DOvercoming%2520hurdles%2520in%2520developing%2520successful%2520drugs%2520targeting%2520chemokine%2520receptors%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2011%26volume%3D11%26issue%3D5%26spage%3D355%26epage%3D363%26doi%3D10.1038%2Fnri2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toraskar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span> <span> </span><span class="NLM_article-title">Bias in chemokine receptor signalling</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/j.it.2014.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.it.2014.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24679437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlt1elurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=243-252&issue=6&author=A.+J.+Zweemerauthor=J.+Toraskarauthor=L.+H.+Heitmanauthor=A.+P.+IJzerman&title=Bias+in+chemokine+receptor+signalling&doi=10.1016%2Fj.it.2014.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bias in chemokine receptor signalling</span></div><div class="casAuthors">Zweemer, Annelien J. M.; Toraskar, Jimita; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">243-252</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Chemokine receptors are widely expressed on a variety of immune cells and play a crucial role in normal physiol. as well as in inflammatory and infectious diseases.  The existence of 23 chemokine receptors and 48 chemokine ligands guarantees a tight control and fine-tuning of the immune system.  Here, we discuss the multiple regulatory mechanisms of chemokine signalling at a systemic, cellular, and mol. level.  In particular, we focus on the impact of biased signalling at the receptor level; an emerging concept in mol. pharmacol.  An improved understanding of these mechanisms may provide a framework for more effective drug discovery and development at a target class that is so relevant for immune function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqru6UHEP6GorVg90H21EOLACvtfcHk0lg85RNgu20tng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlt1elurw%253D&md5=f2aaab7e9b3c0a8f52af3eafd5a69579</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2014.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2014.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%26aulast%3DToraskar%26aufirst%3DJ.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DBias%2520in%2520chemokine%2520receptor%2520signalling%26jtitle%3DTrends%2520Immunol.%26date%3D2014%26volume%3D35%26issue%3D6%26spage%3D243%26epage%3D252%26doi%3D10.1016%2Fj.it.2014.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koelink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Kruijf, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkerts, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraneveld, A. D.</span></span> <span> </span><span class="NLM_article-title">Targeting chemokine receptors in chronic inflammatory diseases: an extensive review</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.pharmthera.2011.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=21839114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=1-18&issue=1&author=P.+J.+Koelinkauthor=S.+A.+Overbeekauthor=S.+Braberauthor=P.+de+Kruijfauthor=G.+Folkertsauthor=M.+J.+Smitauthor=A.+D.+Kraneveld&title=Targeting+chemokine+receptors+in+chronic+inflammatory+diseases%3A+an+extensive+review&doi=10.1016%2Fj.pharmthera.2011.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting chemokine receptors in chronic inflammatory diseases: An extensive review</span></div><div class="casAuthors">Koelink, Pim J.; Overbeek, Saskia A.; Braber, Saskia; de Kruijf, Petra; Folkerts, Gert; Smit, Martine J.; Kraneveld, Aletta D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-18</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The traffic of the different types of immune cells is an important aspect in the immune response.  Chemokines are sol. peptides that are able to attract cells by interaction with chemokine receptors on their target cells.  Several different chemokines and receptors exist enabling the specific trafficking of different immune cells.  In chronic inflammatory disorders there is abundance of immune cells present at the inflammatory site.  This review focuses on the role of chemokine receptors in chronic inflammatory disorders of the lungs, intestine, joints, skin and nervous system and the potential of targeting these receptors as therapeutic intervention in these disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ8i7YbGTlCrVg90H21EOLACvtfcHk0ljUHiXO_whuxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FO&md5=165ed0c4dde555d6e89262ee6ae7fc2a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DKoelink%26aufirst%3DP.%2BJ.%26aulast%3DOverbeek%26aufirst%3DS.%2BA.%26aulast%3DBraber%26aufirst%3DS.%26aulast%3Dde%2BKruijf%26aufirst%3DP.%26aulast%3DFolkerts%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DKraneveld%26aufirst%3DA.%2BD.%26atitle%3DTargeting%2520chemokine%2520receptors%2520in%2520chronic%2520inflammatory%2520diseases%253A%2520an%2520extensive%2520review%26jtitle%3DPharmacol.%2520Ther.%26date%3D2012%26volume%3D133%26issue%3D1%26spage%3D1%26epage%3D18%26doi%3D10.1016%2Fj.pharmthera.2011.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greaves, D. R.</span></span> <span> </span><span class="NLM_article-title">CC chemokine receptors and chronic inflammation—therapeutic opportunities and pharmacological challenges</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1124/pr.111.005074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fpr.111.005074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=23300131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjt1Sktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2013&pages=47-89&issue=1&author=G.+E.+Whiteauthor=A.+J.+Iqbalauthor=D.+R.+Greaves&title=CC+chemokine+receptors+and+chronic+inflammation%E2%80%94therapeutic+opportunities+and+pharmacological+challenges&doi=10.1124%2Fpr.111.005074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">CC chemokine receptors and chronic inflammation-therapeutic opportunities and pharmacological challenges</span></div><div class="casAuthors">White, Gemma E.; Iqbal, Asif J.; Greaves, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-89</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Chemokines are a family of low mol. wt. proteins with an essential role in leukocyte trafficking during both homeostasis and inflammation.  The CC class of chemokines consists of at least 28 members (CCL1-28) that signal through 10 known chemokine receptors (CCR1-10).  CC chemokine receptors are expressed predominantly by T cells and monocytemacrophages, cell types assocd. predominantly with chronic inflammation occurring over weeks or years.  Chronic inflammatory diseases including rheumatoid arthritis, atherosclerosis, and metabolic syndrome are characterized by continued leukocyte infiltration into the inflammatory site, driven in large part by excessive chemokine prodn.  Over years or decades, persistent inflammation may lead to loss of tissue architecture and function, causing severe disability or, in the case of atherosclerosis, fatal outcomes such as myocardial infarction or stroke.  Despite the existence of several clin. strategies for targeting chronic inflammation, these diseases remain significant causes of morbidity and mortality globally, with a concomitant economic impact.  Thus, the development of novel therapeutic agents for the treatment of chronic inflammatory disease continues to be a priority.  In this review we introduce CC chemokine receptors as crit. mediators of chronic inflammatory responses and explore their potential role as pharmacol. targets.  We discuss functions of individual CC chemokine receptors based on in vitro pharmacol. data as well as transgenic animal studies.  Focusing on three key forms of chronic inflammation-rheumatoid arthritis, atherosclerosis, and metabolic syndrome-we describe the pathol. function of CC chemokine receptors and their possible relevance as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0NGFhZPIXbbVg90H21EOLACvtfcHk0ljUHiXO_whuxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjt1Sktbg%253D&md5=1c06c5629ef1e866e98c39e012722c95</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005074%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DG.%2BE.%26aulast%3DIqbal%26aufirst%3DA.%2BJ.%26aulast%3DGreaves%26aufirst%3DD.%2BR.%26atitle%3DCC%2520chemokine%2520receptors%2520and%2520chronic%2520inflammation%25E2%2580%2594therapeutic%2520opportunities%2520and%2520pharmacological%2520challenges%26jtitle%3DPharmacol.%2520Rev.%26date%3D2013%26volume%3D65%26issue%3D1%26spage%3D47%26epage%3D89%26doi%3D10.1124%2Fpr.111.005074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woollard, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanmogne, G. D.</span></span> <span> </span><span class="NLM_article-title">Maraviroc: a review of its use in HIV infection and beyond</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5447</span>– <span class="NLM_lpage">5468</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S90580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.2147%2FDDDT.S90580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=26491256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXls1Sjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=5447-5468&author=S.+M.+Woollardauthor=G.+D.+Kanmogne&title=Maraviroc%3A+a+review+of+its+use+in+HIV+infection+and+beyond&doi=10.2147%2FDDDT.S90580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Maraviroc: a review of its use in HIV infection and beyond</span></div><div class="casAuthors">Woollard, Shawna M.; Kanmogne, Georgette D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5447-5468</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">The human immunodeficiency virus-1 (HIV-1) enters target cells by binding its envelope glycoprotein gp120 to the CD4 receptor and/or coreceptors such as C-C chemokine receptor type 5 (CCR5; R5) and C-X-C chemokine receptor type 4 (CXCR4; X4), and R5-tropic viruses predominate during the early stages of infection.  CCR5 antagonists bind to CCR5 to prevent viral entry.  Maraviroc (MVC) is the only CCR5 antagonist currently approved by the United States Food and Drug Administration, the European Commission, Health Canada, and several other countries for the treatment of patients infected with R5-tropic HIV-1.  MVC has been shown to be effective at inhibiting HIV-1 entry into cells and is well tolerated.  With expanding MVC use by HIV-1-infected humans, different clin. outcomes post-approval have been obsd. with MVC monotherapy or combination therapy with other antiretroviral drugs, with MVC use in humans infected with dual-R5- and X4-tropic HIV-1, infected with different HIV-1 genotype or infected with HIV-2.  This review discuss the role of CCR5 in HIV-1 infection, the development of the CCR5 antagonist MVC, its pharmacokinetics, pharmacodynamics, drug-drug interactions, and the implications of these interactions on treatment outcomes, including viral mutations and drug resistance, and the mechanisms assocd. with the development of resistance to MVC.  This review also discusses available studies investigating the use of MVC in the treatment of other diseases such as cancer, graft-vs.-host disease, and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV-IihM4Xq0LVg90H21EOLACvtfcHk0lg4dTkc3t4OdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXls1Sjtr8%253D&md5=7e43ebf96fe63773cd8e153a2c233a86</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S90580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S90580%26sid%3Dliteratum%253Aachs%26aulast%3DWoollard%26aufirst%3DS.%2BM.%26aulast%3DKanmogne%26aufirst%3DG.%2BD.%26atitle%3DMaraviroc%253A%2520a%2520review%2520of%2520its%2520use%2520in%2520HIV%2520infection%2520and%2520beyond%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2015%26volume%3D9%26spage%3D5447%26epage%3D5468%26doi%3D10.2147%2FDDDT.S90580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span> <span> </span><span class="NLM_article-title">Chemokine receptor antagonists: overcoming developmental hurdles</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nrd2734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnrd2734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=19079127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt12j" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=23-33&issue=1&author=R.+Horuk&title=Chemokine+receptor+antagonists%3A+overcoming+developmental+hurdles&doi=10.1038%2Fnrd2734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemokine receptor antagonists: overcoming developmental hurdles</span></div><div class="casAuthors">Horuk, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-33</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemokine receptors have a key role in the pathogenesis of autoimmune diseases, inflammation and viral infection.  However, with the exception of selective CCR5 antagonists for HIV, the promise of obtaining new therapeutics related to chemokine receptors has not yet been realized.  This article highlights some of the recent failures in the clin. trials of chemokine receptor antagonists and explores possible reasons as to why this might have occurred.  Such reasons include the lack of predictability of animal models and redundancy of the target.  A potential soln. could be to develop drugs that target more than one receptor - known as polypharmacol. - which could be a novel way to generate effective therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFMos9Ccf_G7Vg90H21EOLACvtfcHk0lg4dTkc3t4OdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt12j&md5=831294beadc398d0dc3007a26da96d39</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd2734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2734%26sid%3Dliteratum%253Aachs%26aulast%3DHoruk%26aufirst%3DR.%26atitle%3DChemokine%2520receptor%2520antagonists%253A%2520overcoming%2520developmental%2520hurdles%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26issue%3D1%26spage%3D23%26epage%3D33%26doi%3D10.1038%2Fnrd2734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span> <span> </span><span class="NLM_article-title">Promiscuous drugs as therapeutics for chemokine receptors</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">e1</span>, <span class="refDoi"> DOI: 10.1017/S1462399409000921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1017%2FS1462399409000921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=19123963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FltlCmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=e1&author=R.+Horuk&title=Promiscuous+drugs+as+therapeutics+for+chemokine+receptors&doi=10.1017%2FS1462399409000921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous drugs as therapeutics for chemokine receptors</span></div><div class="casAuthors">Horuk Richard</div><div class="citationInfo"><span class="NLM_cas:title">Expert reviews in molecular medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemokine receptor antagonists that held much promise for the treatment of autoimmune and inflammatory diseases have recently performed poorly in clinical trials, resulting in disappointment for both pharmaceutical companies and patients.  This review focuses on the redundancy of the molecular target as one potential reason for the failure of some of these antagonists to fulfil their initial promise, and discusses the use of drugs that are capable of interacting with more than one drug target - so-called promiscuous drugs - as possible approaches to overcome this difficulty.  Several clinically approved promiscuous drugs, such as aspirin and olanzapine, are already used successfully.  This review discusses examples of promiscuous drugs for G-protein-coupled receptors, including progress in developing dual-specific chemokine receptor antagonists, and considers evidence for the possible therapeutic utility of such drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQ5kVOYrQTyNLIORRPd5lbfW6udTcc2eZTfo4AOPGN3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FltlCmsQ%253D%253D&md5=8c4bf8bc9017136bdeb1489ebce6f4c7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1017%2FS1462399409000921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399409000921%26sid%3Dliteratum%253Aachs%26aulast%3DHoruk%26aufirst%3DR.%26atitle%3DPromiscuous%2520drugs%2520as%2520therapeutics%2520for%2520chemokine%2520receptors%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2009%26volume%3D11%26spage%3De1%26doi%3D10.1017%2FS1462399409000921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratziu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelmalek, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aithal, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caballeria, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francque, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowdley, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craxi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchor-Khan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vest, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiens, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seyedkazemi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, V. W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loomba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanyal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, E.</span></span> <span> </span><span class="NLM_article-title">A randomized, placebo-controlled trial of cenicriviroc for treatment of nonalcoholic steatohepatitis with fibrosis</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1754</span>– <span class="NLM_lpage">1767</span>, <span class="refDoi"> DOI: 10.1002/hep.29477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1002%2Fhep.29477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28833331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotlCms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=1754-1767&issue=5&author=S.+L.+Friedmanauthor=V.+Ratziuauthor=S.+A.+Harrisonauthor=M.+F.+Abdelmalekauthor=G.+P.+Aithalauthor=J.+Caballeriaauthor=S.+Francqueauthor=G.+Farrellauthor=K.+V.+Kowdleyauthor=A.+Craxiauthor=K.+Simonauthor=L.+Fischerauthor=L.+Melchor-Khanauthor=J.+Vestauthor=B.+L.+Wiensauthor=P.+Vigauthor=S.+Seyedkazemiauthor=Z.+Goodmanauthor=V.+W.-S.+Wongauthor=R.+Loombaauthor=F.+Tackeauthor=A.+Sanyalauthor=E.+Lefebvre&title=A+randomized%2C+placebo-controlled+trial+of+cenicriviroc+for+treatment+of+nonalcoholic+steatohepatitis+with+fibrosis&doi=10.1002%2Fhep.29477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A randomized, placebo-controlled trial of cenicriviroc for treatment of nonalcoholic steatohepatitis with fibrosis</span></div><div class="casAuthors">Friedman, Scott L.; Ratziu, Vlad; Harrison, Stephen A.; Abdelmalek, Manal F.; Aithal, Guruprasad P.; Caballeria, Juan; Francque, Sven; Farrell, Geoffrey; Kowdley, Kris V.; Craxi, Antonio; Simon, Krzysztof; Fischer, Laurent; Melchor-Khan, Liza; Vest, Jeffrey; Wiens, Brian L.; Vig, Pamela; Seyedkazemi, Star; Goodman, Zachary; Wong, Vincent Wai-Sun; Loomba, Rohit; Tacke, Frank; Sanyal, Arun; Lefebvre, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1754-1767</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The aim of this study was to evaluate cenicriviroc (CVC), a dual antagonist of C-C chemokine receptor types 2 and 5, for treatment of nonalcoholic steatohepatitis (NASH) with liver fibrosis (LF).  A randomized, double-blind, multinational phase 2b study enrolled subjects with NASH, a nonalcoholic fatty liver disease activity score (NAS) ≥4, and LF (stages 1-3, NASH Clin. Research Network) at 81 clin. sites.  Subjects (N = 289) were randomly assigned CVC 150 mg or placebo.  Primary outcome was ≥2-point improvement in NAS and no worsening of fibrosis at year 1.  Key secondary outcomes were: resoln. of steatohepatitis (SH) and no worsening of fibrosis; improvement in fibrosis by ≥1 stage and no worsening of SH.  Biomarkers of inflammation and adverse events were assessed.  Full study recruitment was achieved.  The primary endpoint of NAS improvement in the intent-to-treat population and resoln. of SH was achieved in a similar proportion of subjects on CVC (N = 145) and placebo (N = 144; 16% vs. 19%, P = 0.52 and 8% vs. 6%, P = 0.49, resp.).  However, the fibrosis endpoint was met in significantly more subjects on CVC than placebo (20% vs. 10%; P = 0.02).  Treatment benefits were greater in those with higher disease activity and fibrosis stage at baseline.  Biomarkers of systemic inflammation were reduced with CVC.  Safety and tolerability of CVC were comparable to placebo.  Conclusion: After 1 yr of CVC treatment, twice as many subjects achieved improvement in fibrosis and no worsening of SH compared with placebo.  Given the urgent need to develop antifibrotic therapies in NASH, these findings warrant phase 3 evaluation. (Hepatol. 2018;67:1754-1767).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_4ETuDf9irVg90H21EOLACvtfcHk0liauUVV9KijYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotlCms7g%253D&md5=a712985806b627b992895637d9f59b58</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fhep.29477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.29477%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%26aufirst%3DS.%2BL.%26aulast%3DRatziu%26aufirst%3DV.%26aulast%3DHarrison%26aufirst%3DS.%2BA.%26aulast%3DAbdelmalek%26aufirst%3DM.%2BF.%26aulast%3DAithal%26aufirst%3DG.%2BP.%26aulast%3DCaballeria%26aufirst%3DJ.%26aulast%3DFrancque%26aufirst%3DS.%26aulast%3DFarrell%26aufirst%3DG.%26aulast%3DKowdley%26aufirst%3DK.%2BV.%26aulast%3DCraxi%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DK.%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DMelchor-Khan%26aufirst%3DL.%26aulast%3DVest%26aufirst%3DJ.%26aulast%3DWiens%26aufirst%3DB.%2BL.%26aulast%3DVig%26aufirst%3DP.%26aulast%3DSeyedkazemi%26aufirst%3DS.%26aulast%3DGoodman%26aufirst%3DZ.%26aulast%3DWong%26aufirst%3DV.%2BW.-S.%26aulast%3DLoomba%26aufirst%3DR.%26aulast%3DTacke%26aufirst%3DF.%26aulast%3DSanyal%26aufirst%3DA.%26aulast%3DLefebvre%26aufirst%3DE.%26atitle%3DA%2520randomized%252C%2520placebo-controlled%2520trial%2520of%2520cenicriviroc%2520for%2520treatment%2520of%2520nonalcoholic%2520steatohepatitis%2520with%2520fibrosis%26jtitle%3DHepatology%26date%3D2018%26volume%3D67%26issue%3D5%26spage%3D1754%26epage%3D1767%26doi%3D10.1002%2Fhep.29477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">Dual targeting of CCR2 and CCR5: therapeutic potential for immunologic and cardiovascular diseases</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>88</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1189/jlb.1009671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1189%2Fjlb.1009671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=20360402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVWqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2010&pages=41-55&issue=1&author=Q.+Zhao&title=Dual+targeting+of+CCR2+and+CCR5%3A+therapeutic+potential+for+immunologic+and+cardiovascular+diseases&doi=10.1189%2Fjlb.1009671"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting of CCR2 and CCR5: therapeutic potential for immunologic and cardiovascular diseases</span></div><div class="casAuthors">Zhao, Qihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review.  A cardinal feature of inflammation is the tissue recruitment of leukocytes, a process that is mediated predominantly by chemokines via their receptors on migrating cells.  CCR2 and CCR5, two CC chemokine receptors, are important players in the trafficking of monocytes/macrophages and in the functions of other cell types relevant to disease pathogenesis.  This review provides a brief overview of the biol. actions of CCR2 and CCR5 and a comprehensive summary of published data that demonstrate the involvement of both receptors in the pathogenesis of immunol. diseases (RA, CD, and transplant rejection) and cardiovascular diseases (atherosclerosis and AIH).  In light of the potential for functional redundancy of chemokine receptors in mediating leukocyte trafficking and the consequent concern over insufficient efficacy offered by pharmacol. inhibiting one receptor, this review presents evidence supporting dual targeting of CCR2 and CCR5 as a more efficacious strategy than targeting either receptor alone.  It also examines potential safety issues assocd. with such dual targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc5RCcoXQyu7Vg90H21EOLACvtfcHk0liauUVV9KijYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVWqs78%253D&md5=f9ff14b43951bee94c0ffea6da30e7d4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1189%2Fjlb.1009671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.1009671%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DDual%2520targeting%2520of%2520CCR2%2520and%2520CCR5%253A%2520therapeutic%2520potential%2520for%2520immunologic%2520and%2520cardiovascular%2520diseases%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D2010%26volume%3D88%26issue%3D1%26spage%3D41%26epage%3D55%26doi%3D10.1189%2Fjlb.1009671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fantuzzi, L.</span>; <span class="NLM_string-name">Tagliamonte, M.</span>; <span class="NLM_string-name">Gauzzi, M. C.</span>; <span class="NLM_string-name">Lopalco, L.</span></span> <span> </span><span class="NLM_article-title">Dual CCR5/CCR2 targeting: opportunities for the cure of complex disorders</span>.  <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1007/s00018-019-03255-6</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1007%2Fs00018-019-03255-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=31377844" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=L.+Fantuzzi&author=M.+Tagliamonte&author=M.+C.+Gauzzi&author=L.+Lopalco&title=Dual+CCR5%2FCCR2+targeting%3A+opportunities+for+the+cure+of+complex+disorders&doi=10.1007%2Fs00018-019-03255-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00018-019-03255-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-019-03255-6%26sid%3Dliteratum%253Aachs%26aulast%3DFantuzzi%26aufirst%3DL.%26atitle%3DDual%2520CCR5%252FCCR2%2520targeting%253A%2520opportunities%2520for%2520the%2520cure%2520of%2520complex%2520disorders%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2019%26doi%3D10.1007%2Fs00018-019-03255-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Junker, A.</span>; <span class="NLM_string-name">Kokornaczyk, A. K.</span>; <span class="NLM_string-name">Strunz, A. K.</span>; <span class="NLM_string-name">Wünsch, B.</span></span> <span> </span><span class="NLM_article-title">Selective and dual targeting of CCR2 and CCR5 receptors: a current overview</span>. In  <i>Chemokines: Chemokines and Their Receptors in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>, Ed.; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Cham, Switzerland</span>, <span class="NLM_year">2015</span>; Vol.  <span class="NLM_volume">14</span>, pp  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">241</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=187-241&author=A.+Junker&author=A.+K.+Kokornaczyk&author=A.+K.+Strunz&author=B.+W%C3%BCnschauthor=N.+Tschammer&title=Chemokines%3A+Chemokines+and+Their+Receptors+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJunker%26aufirst%3DA.%26atitle%3DSelective%2520and%2520dual%2520targeting%2520of%2520CCR2%2520and%2520CCR5%2520receptors%253A%2520a%2520current%2520overview%26btitle%3DChemokines%253A%2520Chemokines%2520and%2520Their%2520Receptors%2520in%2520Drug%2520Discovery%26aulast%3DTschammer%26aufirst%3DN.%26pub%3DSpringer%2520International%2520Publishing%26date%3D2015%26volume%3D14%26spage%3D187%26epage%3D241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kothandan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadhe, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. J.</span></span> <span> </span><span class="NLM_article-title">Structural insights from binding poses of CCR2 and CCR5 with clinically important antagonists: a combined in silico study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e32864</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0032864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1371%2Fjournal.pone.0032864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=22479344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Cnt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e32864&issue=3&author=G.+Kothandanauthor=C.+G.+Gadheauthor=S.+J.+Cho&title=Structural+insights+from+binding+poses+of+CCR2+and+CCR5+with+clinically+important+antagonists%3A+a+combined+in+silico+study&doi=10.1371%2Fjournal.pone.0032864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights from binding poses of CCR2 and CCR5 with clinically important antagonists: a combined in silico study</span></div><div class="casAuthors">Kothandan, Gugan; Gadhe, Changdev G.; Cho, Seung Joo</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e32864</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Chemokine receptors are G protein-coupled receptors that contain seven transmembrane domains.  In particular, CCR2 and CCR5 and their ligands have been implicated in the pathophysiol. of a no. of diseases, including rheumatoid arthritis and multiple sclerosis.  Based on their roles in disease, they have been attractive targets for the pharmaceutical industry, and furthermore, targeting both CCR2 and CCR5 can be a useful strategy.  Owing to the importance of these receptors, information regarding the binding site is of prime importance.  Structural studies have been hampered due to the lack of X-ray crystal structures, and templates with close homologs for comparative modeling.  Most of the previous models were based on the bovine rhodopsin and β2-adrenergic receptor.  In this study, based on a closer homolog with higher resoln. (CXCR4, PDB code: 3ODU 2.5 Å), we constructed three-dimensional models.  The main aim of this study was to provide relevant information on binding sites of these receptors.  Mol. dynamics simulation was done to refine the homol. models and PROCHECK results indicated that the models were reasonable.  Here, binding poses were checked with some established inhibitors of high pharmaceutical importance against the modeled receptors.  Anal. of interaction modes gave an integrated interpretation with detailed structural information.  The binding poses confirmed that the acidic residues Glu291 (CCR2) and Glu283 (CCR5) are important, and we also found some addnl. residues.  Comparisons of binding sites of CCR2/CCR5 were done sequentially and also by docking a potent dual antagonist.  Our results can be a starting point for further structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ql7HNG7LfLVg90H21EOLACvtfcHk0liNdK-0MoO4MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Cnt7g%253D&md5=aee8357749939e2f9ec46e469b78ac8c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0032864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0032864%26sid%3Dliteratum%253Aachs%26aulast%3DKothandan%26aufirst%3DG.%26aulast%3DGadhe%26aufirst%3DC.%2BG.%26aulast%3DCho%26aufirst%3DS.%2BJ.%26atitle%3DStructural%2520insights%2520from%2520binding%2520poses%2520of%2520CCR2%2520and%2520CCR5%2520with%2520clinically%2520important%2520antagonists%253A%2520a%2520combined%2520in%2520silico%2520study%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D3%26spage%3De32864%26doi%3D10.1371%2Fjournal.pone.0032864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz Zacarías, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherney, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tebben, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handel, T. M.</span></span> <span> </span><span class="NLM_article-title">Structure of CC chemokine receptor 2 with orthosteric and allosteric antagonists</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span> (<span class="NLM_issue">7633</span>),  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1038/nature20605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnature20605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=27926736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=458-461&issue=7633&author=Y.+Zhengauthor=L.+Qinauthor=N.+V.+Ortiz+Zacar%C3%ADasauthor=H.+de+Vriesauthor=G.+W.+Hanauthor=M.+Gustavssonauthor=M.+Dabrosauthor=C.+Zhaoauthor=R.+J.+Cherneyauthor=P.+Carterauthor=D.+Stamosauthor=R.+Abagyanauthor=V.+Cherezovauthor=R.+C.+Stevensauthor=A.+P.+IJzermanauthor=L.+H.+Heitmanauthor=A.+Tebbenauthor=I.+Kufarevaauthor=T.+M.+Handel&title=Structure+of+CC+chemokine+receptor+2+with+orthosteric+and+allosteric+antagonists&doi=10.1038%2Fnature20605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of CC chemokine receptor 2 with orthosteric and allosteric antagonists</span></div><div class="casAuthors">Zheng, Yi; Qin, Ling; Zacarias, Natalia V. Ortiz; de Vries, Henk; Han, Gye Won; Gustavsson, Martin; Dabros, Marta; Zhao, Chunxia; Cherney, Robert J.; Carter, Percy; Stamos, Dean; Abagyan, Ruben; Cherezov, Vadim; Stevens, Raymond C.; IJzerman, Adriaan P.; Heitman, Laura H.; Tebben, Andrew; Kufareva, Irina; Handel, Tracy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">458-461</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CC chemokine receptor 2 (CCR2) is one of 19 members of the chemokine receptor subfamily of human class A G-protein-coupled receptors.  CCR2 is expressed on monocytes, immature dendritic cells, and T-cell subpopulations, and mediates their migration towards endogenous CC chemokine ligands such as CCL2 (ref. 1).  CCR2 and its ligands are implicated in numerous inflammatory and neurodegenerative diseases including atherosclerosis, multiple sclerosis, asthma, neuropathic pain, and diabetic nephropathy, as well as cancer.  These disease assocns. have motivated numerous preclin. studies and clin. trials (see http://www.clinicaltrials.gov) in search of therapies that target the CCR2-chemokine axis.  To aid drug discovery efforts, here we solve a structure of CCR2 in a ternary complex with an orthosteric (BMS-681 (ref. 6)) and allosteric (CCR2-RA-[R]7) antagonist.  BMS-681 inhibits chemokine binding by occupying the orthosteric pocket of the receptor in a previously unseen binding mode.  CCR2-RA-[R] binds in a novel, highly druggable pocket that is the most intracellular allosteric site obsd. in class A G-protein-coupled receptors so far; this site spatially overlaps the G-protein-binding site in homologous receptors.  CCR2-RA-[R] inhibits CCR2 non-competitively by blocking activation-assocd. conformational changes and formation of the G-protein-binding interface.  The conformational signature of the conserved microswitch residues obsd. in double-antagonist-bound CCR2 resembles the most inactive G-protein-coupled receptor structures solved so far.  Like other protein-protein interactions, receptor-chemokine complexes are considered challenging therapeutic targets for small mols., and the present structure suggests diverse pocket epitopes that can be exploited to overcome obstacles in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPPLlhgBLBUrVg90H21EOLACvtfcHk0lguoFl3LpJTfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurbP&md5=32e8631bdbf7a125d9c7d2333555041e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature20605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20605%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26aufirst%3DN.%2BV.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DGustavsson%26aufirst%3DM.%26aulast%3DDabros%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DCherney%26aufirst%3DR.%2BJ.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DStamos%26aufirst%3DD.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DTebben%26aufirst%3DA.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DHandel%26aufirst%3DT.%2BM.%26atitle%3DStructure%2520of%2520CC%2520chemokine%2520receptor%25202%2520with%2520orthosteric%2520and%2520allosteric%2520antagonists%26jtitle%3DNature%26date%3D2016%26volume%3D540%26issue%3D7633%26spage%3D458%26epage%3D461%26doi%3D10.1038%2Fnature20605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oswald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dore, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deflorian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Intracellular allosteric antagonism of the CCR9 receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span> (<span class="NLM_issue">7633</span>),  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1038/nature20606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnature20606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=27926729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=462-465&issue=7633&author=C.+Oswaldauthor=M.+Rappasauthor=J.+Keanauthor=A.+S.+Doreauthor=J.+C.+Erreyauthor=K.+Bennettauthor=F.+Deflorianauthor=J.+A.+Christopherauthor=A.+Jazayeriauthor=J.+S.+Masonauthor=M.+Congreveauthor=R.+M.+Cookeauthor=F.+H.+Marshall&title=Intracellular+allosteric+antagonism+of+the+CCR9+receptor&doi=10.1038%2Fnature20606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular allosteric antagonism of the CCR9 receptor</span></div><div class="casAuthors">Oswald, Christine; Rappas, Mathieu; Kean, James; Dore, Andrew S.; Errey, James C.; Bennett, Kirstie; Deflorian, Francesca; Christopher, John A.; Jazayeri, Ali; Mason, Jonathan S.; Congreve, Miles; Cooke, Robert M.; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">462-465</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chemokines and their G-protein-coupled receptors play a diverse role in immune defense by controlling the migration, activation and survival of immune cells.  They are also involved in viral entry, tumor growth and metastasis and hence are important drug targets in a wide range of diseases.  Despite very significant efforts by the pharmaceutical industry to develop drugs, with over 50 small-mol. drugs directed at the family entering clin. development, only two compds. have reached the market: maraviroc (CCR5) for HIV infection and plerixafor (CXCR4) for stem-cell mobilization.  The high failure rate may in part be due to limited understanding of the mechanism of action of chemokine antagonists and an inability to optimize compds. in the absence of structural information.  CC chemokine receptor type 9 (CCR9) activation by CCL25 plays a key role in leukocyte recruitment to the gut and represents a therapeutic target in inflammatory bowel disease.  The selective CCR9 antagonist vercirnon progressed to phase 3 clin. trials in Crohn's disease but efficacy was limited, with the need for very high doses to block receptor activation.  Here we report the crystal structure of the CCR9 receptor in complex with vercirnon at 2.8 Å resoln.  Remarkably, vercirnon binds to the intracellular side of the receptor, exerting allosteric antagonism and preventing G-protein coupling.  This binding site explains the need for relatively lipophilic ligands and describes another example of an allosteric site on G-protein-coupled receptors that can be targeted for drug design, not only at CCR9, but potentially extending to other chemokine receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHomIAvUxar7Vg90H21EOLACvtfcHk0lguoFl3LpJTfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM&md5=253b630bd2b9b65fc192ff86de9df61a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature20606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20606%26sid%3Dliteratum%253Aachs%26aulast%3DOswald%26aufirst%3DC.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DDore%26aufirst%3DA.%2BS.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DBennett%26aufirst%3DK.%26aulast%3DDeflorian%26aufirst%3DF.%26aulast%3DChristopher%26aufirst%3DJ.%2BA.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DIntracellular%2520allosteric%2520antagonism%2520of%2520the%2520CCR9%2520receptor%26jtitle%3DNature%26date%3D2016%26volume%3D540%26issue%3D7633%26spage%3D462%26epage%3D465%26doi%3D10.1038%2Fnature20606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedjai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, D. J.</span></span> <span> </span><span class="NLM_article-title">Cytokines and chemokines: at the crossroads of cell signalling and inflammatory disease</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1843</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2563</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.bbamcr.2014.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24892271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlWrur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=2563-2582&issue=11&author=M.+D.+Turnerauthor=B.+Nedjaiauthor=T.+Hurstauthor=D.+J.+Pennington&title=Cytokines+and+chemokines%3A+at+the+crossroads+of+cell+signalling+and+inflammatory+disease&doi=10.1016%2Fj.bbamcr.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cytokines and chemokines: At the crossroads of cell signalling and inflammatory disease</span></div><div class="casAuthors">Turner, Mark D.; Nedjai, Belinda; Hurst, Tara; Pennington, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2582</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Inflammation occurs as a result of exposure of tissues and organs to harmful stimuli such as microbial pathogens, irritants, or toxic cellular components.  The primary phys. manifestations of inflammation are redness, swelling, heat, pain, and loss of function to the affected area.  These processes involve the major cells of the immune system, including monocytes, macrophages, neutrophils, basophils, dendritic cells, mast cells, T-cells, and B-cells.  However, examn. of a range of inflammatory lesions demonstrates the presence of specific leukocytes in any given lesion.  That is, the inflammatory process is regulated in such a way as to ensure that the appropriate leukocytes are recruited.  These events are in turn controlled by a host of extracellular mol. regulators, including members of the cytokine and chemokine families that mediate both immune cell recruitment and complex intracellular signalling control mechanisms that characterize inflammation.  This review will focus on the role of the main cytokines, chemokines, and their receptors in the pathophysiol. of auto-inflammatory disorders, pro-inflammatory disorders, and neurol. disorders involving inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXDUTKEa7ptrVg90H21EOLACvtfcHk0lgg1mqPvLeRfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlWrur%252FI&md5=6a5667f3148836d1dcab9bef2eef3bef</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DM.%2BD.%26aulast%3DNedjai%26aufirst%3DB.%26aulast%3DHurst%26aufirst%3DT.%26aulast%3DPennington%26aufirst%3DD.%2BJ.%26atitle%3DCytokines%2520and%2520chemokines%253A%2520at%2520the%2520crossroads%2520of%2520cell%2520signalling%2520and%2520inflammatory%2520disease%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26issue%3D11%26spage%3D2563%26epage%3D2582%26doi%3D10.1016%2Fj.bbamcr.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
Zacarias, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handel, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span> <span> </span><span class="NLM_article-title">Intracellular receptor modulation: novel approach to target GPCRs</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2018.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.tips.2018.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29653834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVCjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=547-559&issue=6&author=N.+V.+Ortiz%0AZacariasauthor=E.+B.+Lenselinkauthor=A.+P.+IJzermanauthor=T.+M.+Handelauthor=L.+H.+Heitman&title=Intracellular+receptor+modulation%3A+novel+approach+to+target+GPCRs&doi=10.1016%2Fj.tips.2018.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular Receptor Modulation: Novel Approach to Target GPCRs</span></div><div class="casAuthors">Ortiz Zacarias, Natalia V.; Lenselink, Eelke B.; IJzerman, Adriaan P.; Handel, Tracy M.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">547-559</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent crystal structures of multiple G protein-coupled receptors (GPCRs) have revealed a highly conserved intracellular pocket that can be used to modulate these receptors from the inside.  This novel intracellular site partially overlaps with the G protein and β-arrestin binding site, providing a new manner of pharmacol. intervention.  Here we provide an update of the architecture and function of the intracellular region of GPCRs, until now portrayed as the signaling domain.  We review the available evidence on the presence of intracellular binding sites among chemokine receptors and other class A GPCRs, as well as different strategies to target it, including small mols., pepducins, and nanobodies.  Finally, the potential advantages of intracellular (allosteric) ligands over orthosteric ligands are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk_hgIKJ-qnrVg90H21EOLACvtfcHk0lgg1mqPvLeRfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVCjtLY%253D&md5=28b63bab9f1dd44769fcc3728be0177f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2018.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2018.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2BZacarias%26aufirst%3DN.%2BV.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHandel%26aufirst%3DT.%2BM.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DIntracellular%2520receptor%2520modulation%253A%2520novel%2520approach%2520to%2520target%2520GPCRs%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2018%26volume%3D39%26issue%3D6%26spage%3D547%26epage%3D559%26doi%3D10.1016%2Fj.tips.2018.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
Zacarías, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Veldhoven, J. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Spronsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenhuizen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Velden, W. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oenema, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span> <span> </span><span class="NLM_article-title">Pyrrolone derivatives as intracellular allosteric modulators for chemokine receptors: selective and dual-targeting inhibitors of CC chemokine receptors 1 and 2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9146</span>– <span class="NLM_lpage">9161</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9146-9161&issue=20&author=N.+V.+Ortiz%0AZacar%C3%ADasauthor=J.+P.+D.+van+Veldhovenauthor=L.+Portnerauthor=E.+van+Spronsenauthor=S.+Ulloauthor=M.+Veenhuizenauthor=W.+J.+C.+van+der+Veldenauthor=A.+J.+M.+Zweemerauthor=R.+M.+Kreekelauthor=K.+Oenemaauthor=E.+B.+Lenselinkauthor=L.+H.+Heitmanauthor=A.+P.+IJzerman&title=Pyrrolone+derivatives+as+intracellular+allosteric+modulators+for+chemokine+receptors%3A+selective+and+dual-targeting+inhibitors+of+CC+chemokine+receptors+1+and+2&doi=10.1021%2Facs.jmedchem.8b00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolone Derivatives as Intracellular Allosteric Modulators for Chemokine Receptors: Selective and Dual-Targeting Inhibitors of CC Chemokine Receptors 1 and 2</span></div><div class="casAuthors">Ortiz Zacarias, Natalia V.; van Veldhoven, Jacobus P. D.; Portner, Laura; van Spronsen, Eric; Ullo, Salviana; Veenhuizen, Margo; van der Velden, Wijnand J. C.; Zweemer, Annelien J. M.; Kreekel, Roy M.; Oenema, Kenny; Lenselink, Eelke B.; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9146-9161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent crystal structures of CC chemokine receptors 2 and 9 (CCR2 and CCR9) have provided structural evidence for an allosteric, intracellular binding site.  The high conservation of residues involved in this site suggests its presence in most chemokine receptors, including the close homolog CCR1.  By using [3H]CCR2-RA-[R], a high-affinity, CCR2 intracellular ligand, we report an intracellular binding site in CCR1, where this radioligand also binds with high affinity.  In addn., we report the synthesis and biol. characterization of a series of pyrrolone derivs. for CCR1 and CCR2, which allowed us to identify several high-affinity intracellular ligands, including selective and potential multitarget antagonists.  Evaluation of selected compds. in a functional [35S]GTPγS assay revealed that they act as inverse agonists in CCR1, providing a new manner of pharmacol. modulation.  Thus, this intracellular binding site enables the design of selective and multitarget inhibitors as a novel therapeutic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3HeUOWen2U7Vg90H21EOLACvtfcHk0ljrAV_WqOhOQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM&md5=bf05c560b983659de579074242d299d1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00605%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26aufirst%3DN.%2BV.%26aulast%3Dvan%2BVeldhoven%26aufirst%3DJ.%2BP.%2BD.%26aulast%3DPortner%26aufirst%3DL.%26aulast%3Dvan%2BSpronsen%26aufirst%3DE.%26aulast%3DUllo%26aufirst%3DS.%26aulast%3DVeenhuizen%26aufirst%3DM.%26aulast%3Dvan%2Bder%2BVelden%26aufirst%3DW.%2BJ.%2BC.%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DKreekel%26aufirst%3DR.%2BM.%26aulast%3DOenema%26aufirst%3DK.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DPyrrolone%2520derivatives%2520as%2520intracellular%2520allosteric%2520modulators%2520for%2520chemokine%2520receptors%253A%2520selective%2520and%2520dual-targeting%2520inhibitors%2520of%2520CC%2520chemokine%2520receptors%25201%2520and%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D20%26spage%3D9146%26epage%3D9161%26doi%3D10.1021%2Facs.jmedchem.8b00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nederpelt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrieling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doornbos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span> <span> </span><span class="NLM_article-title">Multiple binding sites for small-molecule antagonists at the CC chemokine receptor 2</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">561</span>, <span class="refDoi"> DOI: 10.1124/mol.113.086850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.113.086850" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=23877010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVygt7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=551-561&issue=4&author=A.+J.+Zweemerauthor=I.+Nederpeltauthor=H.+Vrielingauthor=S.+Hafithauthor=M.+L.+Doornbosauthor=H.+de+Vriesauthor=J.+Abtauthor=R.+Grossauthor=D.+Stamosauthor=J.+Saundersauthor=M.+J.+Smitauthor=A.+P.+IJzermanauthor=L.+H.+Heitman&title=Multiple+binding+sites+for+small-molecule+antagonists+at+the+CC+chemokine+receptor+2&doi=10.1124%2Fmol.113.086850"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple binding sites for small-molecule antagonists at the CC chemokine receptor 2</span></div><div class="casAuthors">Zweemer, Annelien J. M.; Nederpelt, Indira; Vrieling, Hilde; Hafith, Sarah; Doornbos, Maarten L. J.; de Vries, Henk; Abt, Jeffrey; Gross, Raymond; Stamos, Dean; Saunders, John; Smit, Martine J.; IJzerman, Adriaan P.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">551-561</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptor CCR2 is a G protein-coupled receptor that is activated primarily by the endogenous CC chemokine ligand 2 (CCL2).  Many different small-mol. antagonists have been developed to inhibit this receptor, as it is involved in a variety of diseases characterized by chronic inflammation.  Unfortunately, all these antagonists lack clin. efficacy, and therefore a better understanding of their mechanism of action is warranted.  In this study, we examd. the pharmacol. properties of small-mol. CCR2 antagonists in radioligand binding and functional assays.  Six structurally different antagonists were selected for this study, all of which displaced the endogenous agonist 125I-CCL2 from CCR2 with nanomolar affinity.  Two of these antagonists, INCB3344 [N-(2-(((3S,4S)-1-((1r,4S)-4-(benzo[d][1,3]dioxol-5-yl)-4-hydroxycyclohexyl)-4-ethoxypyrrolidin-3-yl)amino)-2-oxoethyl)-3-(trifluoromethyl)benzamide] and CCR2-RA, were radiolabeled to study the binding site in greater detail.  We discovered that [3H]INCB3344 and [3H]CCR2-RA bind to distinct binding sites at CCR2, the latter being the first allosteric radioligand for CCR2.  Besides the binding properties of the antagonists, we examd. CCR2 inhibition in multiple functional assays, including a novel label-free whole-cell assay.  INCB3344 competitively inhibited CCL2-induced G protein activation, whereas CCR2-RA showed a noncompetitive or allosteric mode of inhibition.  These findings demonstrated that the CCR2 antagonists examd. in this study can be classified into two groups with different binding sites and thereby different modes of inhibition.  We have provided further insights in CCR2 antagonism, and these insights are important for the development of novel CCR2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyrR-A2sHKNLVg90H21EOLACvtfcHk0ljBFAafj9EJRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVygt7%252FF&md5=61e8e2f1d0d587d98833fa5f2d088351</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.086850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.086850%26sid%3Dliteratum%253Aachs%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%26aulast%3DNederpelt%26aufirst%3DI.%26aulast%3DVrieling%26aufirst%3DH.%26aulast%3DHafith%26aufirst%3DS.%26aulast%3DDoornbos%26aufirst%3DM.%2BL.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DAbt%26aufirst%3DJ.%26aulast%3DGross%26aufirst%3DR.%26aulast%3DStamos%26aufirst%3DD.%26aulast%3DSaunders%26aufirst%3DJ.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DMultiple%2520binding%2520sites%2520for%2520small-molecule%2520antagonists%2520at%2520the%2520CC%2520chemokine%2520receptor%25202%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26issue%3D4%26spage%3D551%26epage%3D561%26doi%3D10.1124%2Fmol.113.086850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenhuizen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miraglia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilums, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiele, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkilde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span> <span> </span><span class="NLM_article-title">Discovery and mapping of an intracellular antagonist binding site at the chemokine receptor CCR2</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1124/mol.114.093328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.114.093328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25024169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ynu7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=358-368&issue=4&author=A.+J.+Zweemerauthor=J.+Bunnikauthor=M.+Veenhuizenauthor=F.+Miragliaauthor=E.+B.+Lenselinkauthor=M.+Vilumsauthor=H.+de+Vriesauthor=A.+Gibertauthor=S.+Thieleauthor=M.+M.+Rosenkildeauthor=A.+P.+IJzermanauthor=L.+H.+Heitman&title=Discovery+and+mapping+of+an+intracellular+antagonist+binding+site+at+the+chemokine+receptor+CCR2&doi=10.1124%2Fmol.114.093328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and mapping of an intracellular antagonist binding site at the chemokine receptor CCR2</span></div><div class="casAuthors">Zweemer, Annelien J. M.; Bunnik, Julia; Veenhuizen, Margo; Miraglia, Fabiana; Lenselink, Eelke B.; Vilums, Maris; de Vries, Henk; Gibert, Arthur; Thiele, Stefanie; Rosenkilde, Mette M.; IJzerman, Adriaan P.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-368, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptor CCR2 is a G protein-coupled receptor that is involved in many diseases characterized by chronic inflammation, and therefore a large variety of CCR2 small mol. antagonists has been developed.  On the basis of their chem. structures these antagonists can roughly be divided into two groups with most likely two topog. distinct binding sites.  The aim of the current study was to identify the binding site of one such group of ligands, exemplified by three allosteric antagonists, CCR2-RA-[R], JNJ-27141491, and SD-24.  We first used a chimeric CCR2/CCR5 receptor approach to obtain insight into the binding site of the allosteric antagonists and addnl. introduced eight single point mutations in CCR2 to further characterize the putative binding pocket.  All constructs were studied in radioligand binding and/or functional IP turnover assays, providing evidence for an intracellular binding site for CCR2-RA-[R], JNJ-27141491, and SD-24.  For CCR2-RA-[R] the most important residues for binding were found to be the highly conserved tyrosine Y7.53 and phenylalanine F8.50 of the NPxxYx(5,6)F motif, as well as V6.36 at the bottom of TM-VI and K8.49 in helix-VIII.  These findings demonstrate for the first time the presence of an allosteric intracellular binding site for CCR2 antagonists.  This contributes to an increased understanding of the interactions of diverse ligands at CCR2 and may allow for a more rational design of future allosteric antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeUNOf8f0n9bVg90H21EOLACvtfcHk0ljBFAafj9EJRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ynu7bN&md5=9776ef16d97485c2f9c91ea0c0d1870c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.093328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.093328%26sid%3Dliteratum%253Aachs%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%26aulast%3DBunnik%26aufirst%3DJ.%26aulast%3DVeenhuizen%26aufirst%3DM.%26aulast%3DMiraglia%26aufirst%3DF.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DVilums%26aufirst%3DM.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DGibert%26aufirst%3DA.%26aulast%3DThiele%26aufirst%3DS.%26aulast%3DRosenkilde%26aufirst%3DM.%2BM.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DDiscovery%2520and%2520mapping%2520of%2520an%2520intracellular%2520antagonist%2520binding%2520site%2520at%2520the%2520chemokine%2520receptor%2520CCR2%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26issue%3D4%26spage%3D358%26epage%3D368%26doi%3D10.1124%2Fmol.114.093328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wreggett, K. A.</span></span> <span> </span><span class="NLM_article-title">An intracellular allosteric site for a specific class of antagonists of the CC chemokine G protein-coupled receptors CCR4 and CCR5</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">867</span>, <span class="refDoi"> DOI: 10.1124/mol.107.039321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.107.039321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18042736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1egsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=855-867&issue=3&author=G.+Andrewsauthor=C.+Jonesauthor=K.+A.+Wreggett&title=An+intracellular+allosteric+site+for+a+specific+class+of+antagonists+of+the+CC+chemokine+G+protein-coupled+receptors+CCR4+and+CCR5&doi=10.1124%2Fmol.107.039321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An intracellular allosteric site for a specific class of antagonists of the CC chemokine G protein-coupled receptors CCR4 and CCR5</span></div><div class="casAuthors">Andrews, Glen; Jones, Carolyn; Wreggett, Keith A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">855-867</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A novel mechanism for antagonism of the human chemokine receptors CCR4 and CCR5 has been discovered with a series of small-mol. compds. that seems to interact with an allosteric, intracellular site on the receptor.  The existence of this site is supported by a series of observations: (1) intracellular access of these antagonists is required for their activity; (2) specific, saturable binding of a radiolabeled antagonist requires the presence of CCR4; and (3) through engineering receptor chimeras by reciprocal transfer of C-terminal domains between CCR4 and CCR5, compd. binding and the selective structure-activity relationships for antagonism of these receptors seem to be assocd. with the integrity of that intracellular region.  Published antagonists from other chem. series do not seem to bind to the novel site, and their interaction with either CCR4 or CCR5 is not affected by alteration of the C-terminal domain.  The precise location of the proposed binding site remains to be detd., but the known close assocn. of the C-terminal domain, including helix 8, as a proposed intracellular region that interacts with transduction proteins (e.g., G proteins and β-arrestin) suggests that this could be a generic allosteric site for chemokine receptors and perhaps more broadly for class A G protein-coupled receptors.  The existence of such a site that can be targeted for drug discovery has implications for screening assays for receptor antagonists, which would need, therefore, to consider compd. properties for access to this intracellular site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAPVG6Nu0Wf7Vg90H21EOLACvtfcHk0lhMNj6zJbuUng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1egsLw%253D&md5=f3624c421bac61365f63ea1389dffea0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fmol.107.039321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.107.039321%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DC.%26aulast%3DWreggett%26aufirst%3DK.%2BA.%26atitle%3DAn%2520intracellular%2520allosteric%2520site%2520for%2520a%2520specific%2520class%2520of%2520antagonists%2520of%2520the%2520CC%2520chemokine%2520G%2520protein-coupled%2520receptors%2520CCR4%2520and%2520CCR5%26jtitle%3DMol.%2520Pharmacol.%26date%3D2008%26volume%3D73%26issue%3D3%26spage%3D855%26epage%3D867%26doi%3D10.1124%2Fmol.107.039321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bengtsson, B. A.</span>; <span class="NLM_string-name">Blackaby, W.</span>; <span class="NLM_string-name">Cumming, J.</span>; <span class="NLM_string-name">Faull, A. W.</span>; <span class="NLM_string-name">Larsson, J.</span>; <span class="NLM_string-name">Nash, I. A.</span>; <span class="NLM_string-name">Oldham, K.</span>; <span class="NLM_string-name">Pape, A.</span></span> <span> </span><span class="NLM_article-title">4H-[1, 2, 4] Triazolo [5, 1-b] Pyrimidin-7-one Derivatives As CCR2b Receptor Antagonists</span>. Patent <span class="NLM_patent">WO 2011/114148 A1</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=B.+A.+Bengtsson&author=W.+Blackaby&author=J.+Cumming&author=A.+W.+Faull&author=J.+Larsson&author=I.+A.+Nash&author=K.+Oldham&author=A.+Pape&title=4H-%5B1%2C+2%2C+4%5D+Triazolo+%5B5%2C+1-b%5D+Pyrimidin-7-one+Derivatives+As+CCR2b+Receptor+Antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBengtsson%26aufirst%3DB.%2BA.%26atitle%3D4H-%255B1%252C%25202%252C%25204%255D%2520Triazolo%2520%255B5%252C%25201-b%255D%2520Pyrimidin-7-one%2520Derivatives%2520As%2520CCR2b%2520Receptor%2520Antagonists%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, J. W.</span>; <span class="NLM_string-name">Meo, P.</span>; <span class="NLM_string-name">Higginbottom, M.</span>; <span class="NLM_string-name">Simpson, I.</span>; <span class="NLM_string-name">Mountford, D.</span>; <span class="NLM_string-name">Savory, E. D.</span></span> <span> </span><span class="NLM_article-title">7-Hydroxy-Pyrazolo [1,5-a] Pyrimidine Compounds And Their Use As CCR2 Receptor Antagonists</span>. Patent <span class="NLM_patent">WO 2012/041817 A1</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=J.+W.+Boyd&author=P.+Meo&author=M.+Higginbottom&author=I.+Simpson&author=D.+Mountford&author=E.+D.+Savory&title=7-Hydroxy-Pyrazolo+%5B1%2C5-a%5D+Pyrimidine+Compounds+And+Their+Use+As+CCR2+Receptor+Antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBoyd%26aufirst%3DJ.%2BW.%26atitle%3D7-Hydroxy-Pyrazolo%2520%255B1%252C5-a%255D%2520Pyrimidine%2520Compounds%2520And%2520Their%2520Use%2520As%2520CCR2%2520Receptor%2520Antagonists%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordalski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pin, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vriend, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">Generic GPCR residue numbers – aligning topology maps while minding the gaps</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.tips.2014.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25541108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSgu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=22-31&issue=1&author=V.+Isbergauthor=C.+de+Graafauthor=A.+Bortolatoauthor=V.+Cherezovauthor=V.+Katritchauthor=F.+H.+Marshallauthor=S.+Mordalskiauthor=J.-P.+Pinauthor=R.+C.+Stevensauthor=G.+Vriendauthor=D.+E.+Gloriam&title=Generic+GPCR+residue+numbers+%E2%80%93+aligning+topology+maps+while+minding+the+gaps&doi=10.1016%2Fj.tips.2014.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Generic GPCR residue numbers - aligning topology maps while minding the gaps</span></div><div class="casAuthors">Isberg, Vignir; de Graaf, Chris; Bortolato, Andrea; Cherezov, Vadim; Katritch, Vsevolod; Marshall, Fiona H.; Mordalski, Stefan; Pin, Jean-Philippe; Stevens, Raymond C.; Vriend, Gerrit; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-31</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Generic residue nos. facilitate comparisons of, for example, mutational effects, ligand interactions, and structural motifs.  The numbering scheme by Ballesteros and Weinstein for residues within the class A GPCRs (G protein-coupled receptors) has more than 1100 citations, and the recent crystal structures for classes B, C, and F now call for a community consensus in residue numbering within and across these classes.  Furthermore, the structural era has uncovered helix bulges and constrictions that offset the generic residue nos.  The use of generic residue nos. depends on convenient access by pharmacologists, chemists, and structural biologists.  We review the generic residue numbering schemes for each GPCR class, as well as a complementary structure-based scheme, and provide illustrative examples and GPCR database (GPCRDB) web tools to no. any receptor sequence or structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLIeBaLTmUebVg90H21EOLACvtfcHk0lgalsCppNm70w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVSgu7rK&md5=31e18cc8196fb05a12d04dd0402b2338</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DIsberg%26aufirst%3DV.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DPin%26aufirst%3DJ.-P.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DVriend%26aufirst%3DG.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DGeneric%2520GPCR%2520residue%2520numbers%2520%25E2%2580%2593%2520aligning%2520topology%2520maps%2520while%2520minding%2520the%2520gaps%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26issue%3D1%26spage%3D22%26epage%3D31%26doi%3D10.1016%2Fj.tips.2014.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanzaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiraishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aramaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okonogi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meguro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujino, M.</span></span> <span> </span><span class="NLM_article-title">A small-molecule, nonpeptide CCR5 antagonist with highly potent and selective anti-HIV-1 activity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5698</span>– <span class="NLM_lpage">5703</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.10.5698</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1073%2Fpnas.96.10.5698" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10318947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFCntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=5698-5703&issue=10&author=M.+Babaauthor=O.+Nishimuraauthor=N.+Kanzakiauthor=M.+Okamotoauthor=H.+Sawadaauthor=Y.+Iizawaauthor=M.+Shiraishiauthor=Y.+Aramakiauthor=K.+Okonogiauthor=Y.+Ogawaauthor=K.+Meguroauthor=M.+Fujino&title=A+small-molecule%2C+nonpeptide+CCR5+antagonist+with+highly+potent+and+selective+anti-HIV-1+activity&doi=10.1073%2Fpnas.96.10.5698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule, nonpeptide CCR5 antagonist with highly potent and selective anti-HIV-1 activity</span></div><div class="casAuthors">Baba, Masanori; Nishimura, Osamu; Kanzaki, Naoyuki; Okamoto, Mika; Sawada, Hidekazu; Iizawa, Yuji; Shiraishi, Mitsuru; Aramaki, Yoshio; Okonogi, Kenji; Ogawa, Yasuaki; Meguro, Kanji; Fujino, Masahiko</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5698-5703</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The β-chemokine receptor CCR5 is considered to be an attractive target for inhibition of macrophagetropic (CCR5-using or R5) HIV-1 replication because individuals having a nonfunctional receptor (a homozygous 32-bp deletion in the CCR5 coding region) are apparently normal but resistant to infection with R5 HIV-1.  In this study, we found that TAK-779, a nonpeptide compd. with a small mol. wt. (Mr 531.13), antagonized the binding of RANTES (regulated on activation, normal T cell expressed and secreted) to CCR5-expressing Chinese hamster ovary cells and blocked CCR5-mediated Ca2+ signaling at nanomolar concns.  The inhibition of β-chemokine receptors by TAK-779 appeared to be specific to CCR5 because the compd. antagonized CCR2b to a lesser extent but did not affect CCR1, CCR3, or CCR4.  Consequently, TAK-779 displayed highly potent and selective inhibition of R5 HIV-1 replication without showing any cytotoxicity to the host cells.  The compd. inhibited the replication of R5 HIV-1 clin. isolates as well as a lab. strain at a concn. of 1.6-3.7 nM in peripheral blood mononuclear cells, though it was totally inactive against T-cell line-tropic (CXCR4-using or X4) HIV-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJEftBToEjV7Vg90H21EOLACvtfcHk0lgalsCppNm70w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFCntb4%253D&md5=9af3dd556be298aa9995e601725f49aa</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.10.5698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.10.5698%26sid%3Dliteratum%253Aachs%26aulast%3DBaba%26aufirst%3DM.%26aulast%3DNishimura%26aufirst%3DO.%26aulast%3DKanzaki%26aufirst%3DN.%26aulast%3DOkamoto%26aufirst%3DM.%26aulast%3DSawada%26aufirst%3DH.%26aulast%3DIizawa%26aufirst%3DY.%26aulast%3DShiraishi%26aufirst%3DM.%26aulast%3DAramaki%26aufirst%3DY.%26aulast%3DOkonogi%26aufirst%3DK.%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DMeguro%26aufirst%3DK.%26aulast%3DFujino%26aufirst%3DM.%26atitle%3DA%2520small-molecule%252C%2520nonpeptide%2520CCR5%2520antagonist%2520with%2520highly%2520potent%2520and%2520selective%2520anti-HIV-1%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26issue%3D10%26spage%3D5698%26epage%3D5703%26doi%3D10.1073%2Fpnas.96.10.5698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokornaczyk, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Maria, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz Zacarias, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunsch, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of chemokine receptor ligands with 2-benzazepine scaffold</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.ejmech.2017.04.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28463783" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFSktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2017&pages=401-413&author=S.+Thumauthor=A.+K.+Kokornaczykauthor=T.+Sekiauthor=M.+De+Mariaauthor=N.+V.+Ortiz+Zacariasauthor=H.+de+Vriesauthor=C.+Weissauthor=M.+Kochauthor=D.+Schepmannauthor=M.+Kitamuraauthor=N.+Tschammerauthor=L.+H.+Heitmanauthor=A.+Junkerauthor=B.+Wunsch&title=Synthesis+and+biological+evaluation+of+chemokine+receptor+ligands+with+2-benzazepine+scaffold&doi=10.1016%2Fj.ejmech.2017.04.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of chemokine receptor ligands with 2-benzazepine scaffold</span></div><div class="casAuthors">Thum, Simone; Kokornaczyk, Artur K.; Seki, Tomoaki; De Maria, Monica; Ortiz Zacarias, Natalia V.; de Vries, Henk; Weiss, Christina; Koch, Michael; Schepmann, Dirk; Kitamura, Masato; Tschammer, Nuska; Heitman, Laura H.; Junker, Anna; Wuensch, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">401-413</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The development of the first probe-dependent pos. allosteric modulator (PAM) of CCR5 receptors I [R = H, 8-NO2, 8-NHAc, etc.] and II with a 2-benzazepine scaffold was presented.  Compd. I [R = H] activated the CCR5 receptor in a CCL4-dependent manner, but did not compete with [3H]TAK-779 binding at the CCR5.  Furthermore, introduction of a p-tolyl moiety at 7-position of the 2-benzazepine scaffold turns the CCR5 PAM I [R = H] into the selective CCR2 receptor antagonist I [R = 7-p-tolyl].  The structure affinity and activity relationships presented here offered new insights into ligand recognition by CCR2 and CCR5 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOKR_X2Kf9dLVg90H21EOLACvtfcHk0lh9QHywIqs9mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFSktLs%253D&md5=789b738eddd46aecb8cd455f21b1f73d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.046%26sid%3Dliteratum%253Aachs%26aulast%3DThum%26aufirst%3DS.%26aulast%3DKokornaczyk%26aufirst%3DA.%2BK.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DDe%2BMaria%26aufirst%3DM.%26aulast%3DOrtiz%2BZacarias%26aufirst%3DN.%2BV.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DWeiss%26aufirst%3DC.%26aulast%3DKoch%26aufirst%3DM.%26aulast%3DSchepmann%26aufirst%3DD.%26aulast%3DKitamura%26aufirst%3DM.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DJunker%26aufirst%3DA.%26aulast%3DWunsch%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520chemokine%2520receptor%2520ligands%2520with%25202-benzazepine%2520scaffold%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D135%26spage%3D401%26epage%3D413%26doi%3D10.1016%2Fj.ejmech.2017.04.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corbisier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huszagh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmentier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springael, J. Y.</span></span> <span> </span><span class="NLM_article-title">Biased signaling at chemokine receptors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">9542</span>– <span class="NLM_lpage">9554</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.596098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1074%2Fjbc.M114.596098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25614627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Ohs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=9542-9554&issue=15&author=J.+Corbisierauthor=C.+Galesauthor=A.+Huszaghauthor=M.+Parmentierauthor=J.+Y.+Springael&title=Biased+signaling+at+chemokine+receptors&doi=10.1074%2Fjbc.M114.596098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Signaling at Chemokine Receptors</span></div><div class="casAuthors">Corbisier, Jenny; Gales, Celine; Huszagh, Alexandre; Parmentier, Marc; Springael, Jean-Yves</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">9542-9554</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The ability of G protein-coupled receptors (GPCRs) to activate selective signaling pathways according to the conformation stabilized by bound ligands (signaling bias) is a challenging concept in the GPCR field.  Signaling bias has been documented for several GPCRs, including chemokine receptors.  However, most of these studies examd. the global signaling bias between G protein- and arrestin-dependent pathways, leaving unaddressed the potential bias between particular G protein subtypes.  Here, the authors investigated the coupling selectivity of chemokine receptors CCR2, CCR5, and CCR7 in response to various ligands with G protein subtypes by using bioluminescence resonance energy transfer biosensors monitoring directly the activation of G proteins.  The authors also compared data obtained with the G protein biosensors with those obtained with other functional readouts, such as β-arrestin-2 recruitment, cAMP accumulation, and calcium mobilization assays.  The authors showed that the binding of chemokines to CCR2, CCR5, and CCR7 activated the three Gαi subtypes (Gαi1, Gαi2, and Gαi3) and the two Gαo isoforms (Gαoa and Gαob) with potencies that generally correlate to their binding affinities.  In addn., the authors showed that the binding of chemokines to CCR5 and CCR2 also activated Gα12, but not Gα13.  For each receptor, the authors showed that the relative potency of various agonist chemokines was not identical in all assays, supporting the notion that signaling bias exists at chemokine receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq627lB4lvYh7Vg90H21EOLACvtfcHk0lh9QHywIqs9mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Ohs7c%253D&md5=fc598a1428f9a9b1d191c6fa34ab81dd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.596098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.596098%26sid%3Dliteratum%253Aachs%26aulast%3DCorbisier%26aufirst%3DJ.%26aulast%3DGales%26aufirst%3DC.%26aulast%3DHuszagh%26aufirst%3DA.%26aulast%3DParmentier%26aufirst%3DM.%26aulast%3DSpringael%26aufirst%3DJ.%2BY.%26atitle%3DBiased%2520signaling%2520at%2520chemokine%2520receptors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26issue%3D15%26spage%3D9542%26epage%3D9554%26doi%3D10.1074%2Fjbc.M114.596098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, H.</span></span> <span> </span><span class="NLM_article-title">Hill coefficients, dose-response curves and allosteric mechanisms</span>. <i>J. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1007/s12154-009-0029-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1007%2Fs12154-009-0029-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=19779939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A280%3ADC%252BC38zitlSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=37-44&issue=1&author=H.+Prinz&title=Hill+coefficients%2C+dose-response+curves+and+allosteric+mechanisms&doi=10.1007%2Fs12154-009-0029-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Hill coefficients, dose-response curves and allosteric mechanisms</span></div><div class="casAuthors">Prinz Heino</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-44</span>
        ISSN:<span class="NLM_cas:issn">1864-6158</span>.
    </div><div class="casAbstract">Hill coefficients (n(H)) derived from four parameter logistic fits to dose-response curves were compared to calculated realistic reaction schemes and related to experimental data: (1) Hill coefficients may give information on the number of interacting sites but cannot distinguish between competitive, non-competitive or ortho-, iso-, or allosteric mechanisms. (2) For enzymatic dose-inhibition curves, Hill coefficients smaller than one do not indicate anticooperative binding but show that at least one ternary complex has enzymatic activity. (3) Hill coefficients different from one are proof for multiple ligand binding.  The large variations of reported Hill coefficients corresponds to multiple allosteric binding, where induced conformational changes cause loss of the active conformation.  Such a denaturation mechanism is in stark contrast to the desired specificity of drugs.  The discussion is open.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTaD6e5pnko8WGmni0r9GoKfW6udTcc2eagunEgblj0z7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38zitlSqsg%253D%253D&md5=5882ac14ec82a4f7b0f5a4eae02c9410</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs12154-009-0029-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12154-009-0029-3%26sid%3Dliteratum%253Aachs%26aulast%3DPrinz%26aufirst%3DH.%26atitle%3DHill%2520coefficients%252C%2520dose-response%2520curves%2520and%2520allosteric%2520mechanisms%26jtitle%3DJ.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D3%26issue%3D1%26spage%3D37%26epage%3D44%26doi%3D10.1007%2Fs12154-009-0029-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collart, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grassi, J.</span></span> <span> </span><span class="NLM_article-title">Novel, acidic CCR2 receptor antagonists: from hit to lead</span>. <i>Lett. Drug. Des. Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.2174/157018007780077381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.2174%2F157018007780077381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsF2qtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=185-191&issue=3&author=D.+Zouauthor=H.-X.+Zhaiauthor=J.+Eckmanauthor=P.+Higginsauthor=M.+Gillardauthor=L.+Knerrauthor=S.+Carreauthor=P.+Pasauauthor=P.+Collartauthor=J.+Grassi&title=Novel%2C+acidic+CCR2+receptor+antagonists%3A+from+hit+to+lead&doi=10.2174%2F157018007780077381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Novel, acidic CCR2 receptor antagonists: from hit to lead</span></div><div class="casAuthors">Zou, D.; Zhai, H.-X.; Eckman, J.; Higgins, P.; Gillard, M.; Knerr, L.; Carre, S.; Pasau, P.; Collart, P.; Grassi, J.; Libertine, L.; Nicolas, J.-M.; Schwartz, C. E.</div><div class="citationInfo"><span class="NLM_cas:title">Letters in Drug Design & Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">185-191</span>CODEN:
                <span class="NLM_cas:coden">LDDDAW</span>;
        ISSN:<span class="NLM_cas:issn">1570-1808</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">High throughput screening identified a pyrrolidinone contg. acidic functionality as a CCR2 antagonist of modest affinity.  We describe initial SAR around this hit, including soln. phase parallel synthesis for analog prepn., and we describe identification and characterization of an analog subsequently selected as a viable lead mol. for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqICdOhOTrW0bVg90H21EOLACvtfcHk0lj-iJhsA4lrIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsF2qtb8%253D&md5=6c04d2c6b5daf6767966ff35737649d9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.2174%2F157018007780077381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157018007780077381%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DD.%26aulast%3DZhai%26aufirst%3DH.-X.%26aulast%3DEckman%26aufirst%3DJ.%26aulast%3DHiggins%26aufirst%3DP.%26aulast%3DGillard%26aufirst%3DM.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DCarre%26aufirst%3DS.%26aulast%3DPasau%26aufirst%3DP.%26aulast%3DCollart%26aufirst%3DP.%26aulast%3DGrassi%26aufirst%3DJ.%26atitle%3DNovel%252C%2520acidic%2520CCR2%2520receptor%2520antagonists%253A%2520from%2520hit%2520to%2520lead%26jtitle%3DLett.%2520Drug.%2520Des.%2520Discovery%26date%3D2007%26volume%3D4%26issue%3D3%26spage%3D185%26epage%3D191%26doi%3D10.2174%2F157018007780077381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ágoston, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pongor, S.</span></span> <span> </span><span class="NLM_article-title">The efficiency of multi-target drugs: the network approach might help drug design</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">178</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2005.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.tips.2005.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=15808341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivV2ht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=178-182&issue=4&author=P.+Csermelyauthor=V.+%C3%81gostonauthor=S.+Pongor&title=The+efficiency+of+multi-target+drugs%3A+the+network+approach+might+help+drug+design&doi=10.1016%2Fj.tips.2005.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The efficiency of multi-target drugs: the network approach might help drug design</span></div><div class="casAuthors">Csermely, Peter; Agoston, Vilmos; Pongor, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">178-182</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite considerable progress in genome- and proteome-based high-throughput screening methods and rational drug design, the no. of successful single-target drugs did not increase appreciably during the past decade.  Network models suggest that partial inhibition of a surprisingly small no. of targets can be more efficient than the complete inhibition of a single target.  This and the success stories of multi-target drugs and combinatorial therapies led us to suggest that systematic drug-design strategies should be directed against multiple targets.  The authors propose that the final effect of partial, but multiple, drug actions might often surpass that of complete drug action at a single target.  The future success of this novel drug-design paradigm will depend not only on a new generation of computer models to identify the correct multiple targets and their multi-fitting, low-affinity drug candidates but also on more-efficient in vivo testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQRoFMVKb5m7Vg90H21EOLACvtfcHk0liGf9CbuPy-uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivV2ht70%253D&md5=4f1383f4699221ef99041dffafe03e8c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2005.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2005.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3D%25C3%2581goston%26aufirst%3DV.%26aulast%3DPongor%26aufirst%3DS.%26atitle%3DThe%2520efficiency%2520of%2520multi-target%2520drugs%253A%2520the%2520network%2520approach%2520might%2520help%2520drug%2520design%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2005%26volume%3D26%26issue%3D4%26spage%3D178%26epage%3D182%26doi%3D10.1016%2Fj.tips.2005.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, J. R.</span></span> <span> </span><span class="NLM_article-title">The Challenges of Multi-Target Lead Optimization</span>. In  <i>Designing Multi-Target Drugs</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, J. R.</span>, <span class="NLM_string-name">Harris, C. J.</span></span>, Eds.; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">154</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1039%2F9781849734912-00141" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=141-154&author=J.+R.+Morphyauthor=J.+R.+Morphy&author=C.+J.+Harris&title=Designing+Multi-Target+Drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1039%2F9781849734912-00141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849734912-00141%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520Challenges%2520of%2520Multi-Target%2520Lead%2520Optimization%26btitle%3DDesigning%2520Multi-Target%2520Drugs%26aulast%3DMorphy%26aufirst%3DJ.%2BR.%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2012%26spage%3D141%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buntinx, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moechars, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilissen, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckx, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coesemans, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lommen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Wauwe, J. P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological profile of JNJ-27141491 [(S)-3-[3, 4-difluorophenyl)-propyl]-5-isoxazol-5-yl-2-thioxo-2, 3-dihydro-1H-imidazole-4-carboxyl acid methyl ester], as a noncompetitive and orally active antagonist of the human chemokine receptor CCR2</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>327</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1124/jpet.108.140723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fjpet.108.140723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18599682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1amsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2008&pages=1-9&issue=1&author=M.+Buntinxauthor=B.+Hermansauthor=J.+Goossensauthor=D.+Moecharsauthor=R.+A.+Gilissenauthor=J.+Doyonauthor=S.+Boeckxauthor=E.+Coesemansauthor=G.+Van+Lommenauthor=J.+P.+Van+Wauwe&title=Pharmacological+profile+of+JNJ-27141491+%5B%28S%29-3-%5B3%2C+4-difluorophenyl%29-propyl%5D-5-isoxazol-5-yl-2-thioxo-2%2C+3-dihydro-1H-imidazole-4-carboxyl+acid+methyl+ester%5D%2C+as+a+noncompetitive+and+orally+active+antagonist+of+the+human+chemokine+receptor+CCR2&doi=10.1124%2Fjpet.108.140723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological profile of JNJ-27141491 [(S)-3-[(3,4-difluorophenyl)propyl]-5-isoxazol-5-yl-2-thioxo-2,3-dihydro-1H-imidazole-4-carboxyl acid methyl ester], as a noncompetitive and orally active antagonist of the human chemokine receptor CCR2</span></div><div class="casAuthors">Buntinx, Mieke; Hermans, Bart; Goossens, Jan; Moechars, Dieder; Gilissen, Ron A. H. J.; Doyon, Julien; Boeckx, Staf; Coesemans, Erwin; Van Lommen, Guy; Van Wauwe, Jean P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The interaction between CC chemokine receptor 2 (CCR2) with monocyte chemoattractant proteins, such as MCP-1, regulates the activation and recruitment of inflammatory leukocytes.  In this study, we characterized JNJ-27141491 as a noncompetitive and orally active functional antagonist of human (h)CCR2.  JNJ-27141491 strongly suppressed hCCR2-mediated in vitro functions, such as MCP-1-induced guanosine 5'-O-(3-[35S]thio)triphosphate binding; MCP-1, -3, and -4-induced Ca2+ mobilization; and leukocyte chemotaxis toward MCP-1 (IC50 = 7-97 nM), whereas it had little or no effect on the function of other chemokine receptors tested.  The inhibition of CCR2 function was both insurmountable and reversible, consistent with a noncompetitive mode of action.  JNJ-27141491 blocked the binding of 125I-MCP-1 to human monocytes (IC50 = 0.4 μM), but it failed to affect MCP-1 binding to mouse, rat, and dog cells (IC50 > 10 μM).  Therefore, transgenic mice, in which the mouse (m)CCR2 gene was replaced by the human counterpart, were generated for in vivo testing.  In these mice, oral administration of JNJ-27141491 dose-dependently [5-40 mg/kg q.d. (once daily) or b.i.d.] inhibited monocyte and neutrophil recruitment to the alveolar space 48 h after intratracheal mMCP-1/lipopolysaccharide instillation.  Furthermore, treatment with JNJ-27141491 (20 mg/kg q.d.) significantly delayed the onset and temporarily reduced neurol. signs in an exptl. autoimmune encephalomyelitis model of multiple sclerosis.  Taken together, these results identify JNJ-27141491 as a noncompetitive, functional antagonist of hCCR2, capable of exerting oral anti-inflammatory activity in transgenic hCCR2-expressing mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCvC8mYaH0rVg90H21EOLACvtfcHk0liGf9CbuPy-uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1amsLjE&md5=5c4ed289656be37b72a76210b34b9eeb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.140723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.140723%26sid%3Dliteratum%253Aachs%26aulast%3DBuntinx%26aufirst%3DM.%26aulast%3DHermans%26aufirst%3DB.%26aulast%3DGoossens%26aufirst%3DJ.%26aulast%3DMoechars%26aufirst%3DD.%26aulast%3DGilissen%26aufirst%3DR.%2BA.%26aulast%3DDoyon%26aufirst%3DJ.%26aulast%3DBoeckx%26aufirst%3DS.%26aulast%3DCoesemans%26aufirst%3DE.%26aulast%3DVan%2BLommen%26aufirst%3DG.%26aulast%3DVan%2BWauwe%26aufirst%3DJ.%2BP.%26atitle%3DPharmacological%2520profile%2520of%2520JNJ-27141491%2520%255B%2528S%2529-3-%255B3%252C%25204-difluorophenyl%2529-propyl%255D-5-isoxazol-5-yl-2-thioxo-2%252C%25203-dihydro-1H-imidazole-4-carboxyl%2520acid%2520methyl%2520ester%255D%252C%2520as%2520a%2520noncompetitive%2520and%2520orally%2520active%2520antagonist%2520of%2520the%2520human%2520chemokine%2520receptor%2520CCR2%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2008%26volume%3D327%26issue%3D1%26spage%3D1%26epage%3D9%26doi%3D10.1124%2Fjpet.108.140723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczuka, A.</span></span> <span> </span><span class="NLM_article-title">Kinetic versus allosteric mechanisms to explain insurmountable antagonism and delayed ligand dissociation</span>. <i>Neurochem. Int.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">254</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/j.neuint.2007.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.neuint.2007.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=17606314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCmt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=254-260&issue=5&author=G.+Vauquelinauthor=A.+Szczuka&title=Kinetic+versus+allosteric+mechanisms+to+explain+insurmountable+antagonism+and+delayed+ligand+dissociation&doi=10.1016%2Fj.neuint.2007.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic versus allosteric mechanisms to explain insurmountable antagonism and delayed ligand dissociation</span></div><div class="casAuthors">Vauquelin, Georges; Szczuka, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry International</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">254-260</span>CODEN:
                <span class="NLM_cas:coden">NEUIDS</span>;
        ISSN:<span class="NLM_cas:issn">0197-0186</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The present review addresses the theories which have been advanced to explain exptl. observations dealing with insurmountable antagonism and accelerated radioligand dissocn. in the presence of an excess unlabeled ligand.  We came to the perception that, for each of these phenomena, the theories can be placed into two distinctive categories.  The "kinetic" interpretations attribute these phenomena to, resp., the ability of antagonists to form long-lasting complexes with their cognate receptor and the ability of dissocd. ligands to bind again to the same or neighboring receptors rather than to diffuse away from the cell surface.  On the other hand, these observations can also be explained by neg. allosteric interactions among topog. distinct ligand binding sites at the same receptor or di/multimeric receptor complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZubKDcax4HbVg90H21EOLACvtfcHk0lic05jjpPt8Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCmt7jO&md5=49a0753fcf25ac7be190fc45514baa49</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2007.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2007.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26aulast%3DSzczuka%26aufirst%3DA.%26atitle%3DKinetic%2520versus%2520allosteric%2520mechanisms%2520to%2520explain%2520insurmountable%2520antagonism%2520and%2520delayed%2520ligand%2520dissociation%26jtitle%3DNeurochem.%2520Int.%26date%3D2007%26volume%3D51%26issue%3D5%26spage%3D254%26epage%3D260%26doi%3D10.1016%2Fj.neuint.2007.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span> <span> </span><span class="NLM_article-title">Structure of the CCR5 chemokine receptor–HIV entry inhibitor maraviroc complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6152</span>),  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1126/science.1241475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1126%2Fscience.1241475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24030490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1387-1390&issue=6152&author=Q.+Tanauthor=Y.+Zhuauthor=J.+Liauthor=Z.+Chenauthor=G.+W.+Hanauthor=I.+Kufarevaauthor=T.+Liauthor=L.+Maauthor=G.+Fenaltiauthor=J.+Liauthor=W.+Zhangauthor=X.+Xieauthor=H.+Yangauthor=H.+Jiangauthor=V.+Cherezovauthor=H.+Liuauthor=R.+C.+Stevensauthor=Q.+Zhaoauthor=B.+Wu&title=Structure+of+the+CCR5+chemokine+receptor%E2%80%93HIV+entry+inhibitor+maraviroc+complex&doi=10.1126%2Fscience.1241475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the CCR5 Chemokine Receptor-HIV Entry Inhibitor Maraviroc Complex</span></div><div class="casAuthors">Tan, Qiuxiang; Zhu, Ya; Li, Jian; Chen, Zhuxi; Han, Gye Won; Kufareva, Irina; Li, Tingting; Ma, Limin; Fenalti, Gustavo; Li, Jing; Zhang, Wenru; Xie, Xin; Yang, Huaiyu; Jiang, Hualiang; Cherezov, Vadim; Liu, Hong; Stevens, Raymond C.; Zhao, Qiang; Wu, Beili</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6152</span>),
    <span class="NLM_cas:pages">1387-1390</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The CCR5 chemokine receptor acts as a co-receptor for HIV-1 viral entry.  Here we report the 2.7 angstrom-resoln. crystal structure of human CCR5 bound to the marketed HIV drug maraviroc.  The structure reveals a ligand-binding site that is distinct from the proposed major recognition sites for chemokines and the viral glycoprotein gp120, providing insights into the mechanism of allosteric inhibition of chemokine signaling and viral entry.  A comparison between CCR5 and CXCR4 crystal structures, along with models of co-receptor-gp120-V3 complexes, suggests that different charge distributions and steric hindrances caused by residue substitutions may be major determinants of HIV-1 co-receptor selectivity.  These high-resoln. insights into CCR5 can enable structure-based drug discovery for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt5VKeO5Hmb7Vg90H21EOLACvtfcHk0lic05jjpPt8Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ&md5=730ee5644a2a144cd49cd0e9a15eba2e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscience.1241475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1241475%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DB.%26atitle%3DStructure%2520of%2520the%2520CCR5%2520chemokine%2520receptor%25E2%2580%2593HIV%2520entry%2520inhibitor%2520maraviroc%2520complex%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6152%26spage%3D1387%26epage%3D1390%26doi%3D10.1126%2Fscience.1241475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilcken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckler, F. M.</span></span> <span> </span><span class="NLM_article-title">Principles and applications of halogen bonding in medicinal chemistry and chemical biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1363</span>– <span class="NLM_lpage">1388</span>, <span class="refDoi"> DOI: 10.1021/jm3012068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3012068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1363-1388&issue=4&author=R.+Wilckenauthor=M.+O.+Zimmermannauthor=A.+Langeauthor=A.+C.+Joergerauthor=F.+M.+Boeckler&title=Principles+and+applications+of+halogen+bonding+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Fjm3012068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Wilcken, Rainer; Zimmermann, Markus O.; Lange, Andreas; Joerger, Andreas C.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1363-1388</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Halogen bonding has been known in material science for decades, but until recently, halogen bonds in protein-ligand interactions were largely the result of serendipitous discovery rather than rational design.  In this Perspective, we provide insights into the phenomenon of halogen bonding, with special focus on its role in drug discovery.  We summarize the theor. background defining its strength and directionality, provide a systematic anal. of its occurrence and interaction geometries in protein-ligand complexes, and give recent examples where halogen bonding has been successfully harnessed for lead identification and optimization.  In light of these data, we discuss the potential and limitations of exploiting halogen bonds for mol. recognition and rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjholHfNZ9kbVg90H21EOLACvtfcHk0li0cJowFvMlyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ&md5=81ba024e4427fac3e7af0a4cd330909c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm3012068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3012068%26sid%3Dliteratum%253Aachs%26aulast%3DWilcken%26aufirst%3DR.%26aulast%3DZimmermann%26aufirst%3DM.%2BO.%26aulast%3DLange%26aufirst%3DA.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPrinciples%2520and%2520applications%2520of%2520halogen%2520bonding%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D4%26spage%3D1363%26epage%3D1388%26doi%3D10.1021%2Fjm3012068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salchow, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Press, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, M. E.</span></span> <span> </span><span class="NLM_article-title">A common intracellular allosteric binding site for antagonists of the CXCR2 receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>159</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00623.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1111%2Fj.1476-5381.2009.00623.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=20233217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Ojt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2010&pages=1429-1439&issue=7&author=K.+Salchowauthor=M.+E.+Bondauthor=S.+C.+Evansauthor=N.+J.+Pressauthor=S.+J.+Charltonauthor=P.+A.+Huntauthor=M.+E.+Bradley&title=A+common+intracellular+allosteric+binding+site+for+antagonists+of+the+CXCR2+receptor&doi=10.1111%2Fj.1476-5381.2009.00623.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A common intracellular allosteric binding site for antagonists of the CXCR2 receptor</span></div><div class="casAuthors">Salchow, K.; Bond, M. E.; Evans, S. C.; Press, N. J.; Charlton, S. J.; Hunt, P. A.; Bradley, M. E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1429-1439</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and purpose: We have previously shown that SB265610 (1-(2-bromo-phenyl)-3-(7-cyano-3H-benzotriazol-4-yl)-urea) behaves as an allosteric, inverse agonist at the C-X-C chemokine (CXCR)2 receptor.  The aim of this study was to det. whether SB265610, in addn. to two other known antagonists, bind to either of the two putative, topog. distinct, allosteric binding sites previously reported in the Literature.  Exptl. approach: Ten single point mutations were introduced into the CXCR2 receptor using site-directed mutagenesis.  Three CXCR2 antagonists were investigated, SB265610, Pteridone-1 (2-(2,3 difluoro-benzylsulphanyl)-4-((R)-2-hydroxy-1-methyl-ethylamino)-8H-pteridin-7-one) and Sch527123 (2-hydroxy-N,N-dimethyl-3-{2-[[(R)-1-(5-methyl-furan-2-yl)-propyl]amino]-3,4-dioxo-cyclobut-1enylamino}-benzamide), and the effect of these mutations on their binding affinity and ability to inhibit interleukin-8-stimulated binding of [35S]GTPγS was examd.  Key results: Seven of the nine mutations introduced into the C-terminal domain and intracellular loops of the receptor produced a significant redn. in affinity at least one of the antagonists tested.  Of those seven mutations, three produced a significant redn. in the affinity of all three antagonists, namely K320A, Y314A and D84N.  In all but one mutation, the changes obsd. on antagonist affinity were matched with effects on inhibition of interleukin-8-stimulated [35S]GTPγS binding.  Conclusions and implications: These antagonists bind to a common intracellular, allosteric, binding site of the CXCR2 receptor, which has been further delineated.  As many of these mutations are close to the site of G protein coupling or to a region of the receptor that is responsible for the transduction of the activation signal, our results suggest a mol. mechanism for the inhibition of receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOiWNq0sDt17Vg90H21EOLACvtfcHk0liU6Yhtk6nDvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Ojt7Y%253D&md5=f1d3541316672373f9aebe8c9cc79edd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00623.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00623.x%26sid%3Dliteratum%253Aachs%26aulast%3DSalchow%26aufirst%3DK.%26aulast%3DBond%26aufirst%3DM.%2BE.%26aulast%3DEvans%26aufirst%3DS.%2BC.%26aulast%3DPress%26aufirst%3DN.%2BJ.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26aulast%3DHunt%26aufirst%3DP.%2BA.%26aulast%3DBradley%26aufirst%3DM.%2BE.%26atitle%3DA%2520common%2520intracellular%2520allosteric%2520binding%2520site%2520for%2520antagonists%2520of%2520the%2520CXCR2%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2010%26volume%3D159%26issue%3D7%26spage%3D1429%26epage%3D1439%26doi%3D10.1111%2Fj.1476-5381.2009.00623.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brammall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grahames, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meghani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallinder, P. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a putative intracellular allosteric antagonist binding-site in the CXC chemokine receptors 1 and 2</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>74</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi"> DOI: 10.1124/mol.107.044610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.107.044610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18676678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlCntrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2008&pages=1193-1202&issue=5&author=D.+J.+Nichollsauthor=N.+P.+Tomkinsonauthor=K.+E.+Wileyauthor=A.+Brammallauthor=L.+Bowersauthor=C.+Grahamesauthor=A.+Gawauthor=P.+Meghaniauthor=P.+Sheltonauthor=T.+J.+Wrightauthor=P.+R.+Mallinder&title=Identification+of+a+putative+intracellular+allosteric+antagonist+binding-site+in+the+CXC+chemokine+receptors+1+and+2&doi=10.1124%2Fmol.107.044610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a putative intracellular allosteric antagonist binding-site in the CXC chemokine receptors 1 and 2</span></div><div class="casAuthors">Nicholls, David J.; Tomkinson, Nick P.; Wiley, Katherine E.; Brammall, Anne; Bowers, Lorna; Grahames, Caroline; Gaw, Alasdair; Meghani, Premji; Shelton, Philip; Wright, Tracey J.; Mallinder, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1193-1202</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptors CXCR1 and CXCR2 are G-protein-coupled receptors (GPCRs) implicated in mediating cellular functions assocd. with the inflammatory response.  Potent CXCR2 receptor antagonists have been discovered, some of which have recently entered clin. development.  The aim of this study was to identify key amino acid residue differences between CXCR1 and CXCR2 that influence the relative antagonism by two compds. that have markedly different chem. structures.  By investigating the effects of domain switching and point mutations, we found that the second extracellular loop, which contained significant amino acid sequence diversity, was not important for compd. antagonism.  We were surprised to find that switching the intracellular C-terminal 60 amino acid domains of CXCR1 and CXCR2 caused an apparent reversal of antagonism at these two receptors.  Further investigation showed that a single amino acid residue, lysine 320 in CXCR2 and asparagine 311 in CXCR1, plays a predominant role in describing the relative antagonism of the two compds.  Homol. modeling studies based on the structure of bovine rhodopsin indicated a potential intracellular antagonist binding pocket involving lysine 320.  We conclude that residue 320 in CXCR2 forms part of a potential allosteric binding pocket on the intracellular side of the receptor, a site that is distal to the orthosteric site commonly assumed to be the location of antagonist binding to GPCRs.  The existence of a common intracellular allosteric binding site at GPCRs related to CXCR2 may be of value in the design of novel antagonists for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE2keoFJBVXrVg90H21EOLACvtfcHk0liU6Yhtk6nDvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlCntrbJ&md5=1af3dc48be040ab10efb3b62213b4b0b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1124%2Fmol.107.044610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.107.044610%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DTomkinson%26aufirst%3DN.%2BP.%26aulast%3DWiley%26aufirst%3DK.%2BE.%26aulast%3DBrammall%26aufirst%3DA.%26aulast%3DBowers%26aufirst%3DL.%26aulast%3DGrahames%26aufirst%3DC.%26aulast%3DGaw%26aufirst%3DA.%26aulast%3DMeghani%26aufirst%3DP.%26aulast%3DShelton%26aufirst%3DP.%26aulast%3DWright%26aufirst%3DT.%2BJ.%26aulast%3DMallinder%26aufirst%3DP.%2BR.%26atitle%3DIdentification%2520of%2520a%2520putative%2520intracellular%2520allosteric%2520antagonist%2520binding-site%2520in%2520the%2520CXC%2520chemokine%2520receptors%25201%2520and%25202%26jtitle%3DMol.%2520Pharmacol.%26date%3D2008%26volume%3D74%26issue%3D5%26spage%3D1193%26epage%3D1202%26doi%3D10.1124%2Fmol.107.044610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Draper-Joyce, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoshouei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, A. T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furness, S. G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venugopal, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltos, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plitzko, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danev, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glukhova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Structure of the adenosine-bound human adenosine A1 receptor–Gi complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0236-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fs41586-018-0236-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29925945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=558&publication_year=2018&pages=559-563&issue=7711&author=C.+J.+Draper-Joyceauthor=M.+Khoshoueiauthor=D.+M.+Thalauthor=Y.-L.+Liangauthor=A.+T.+N.+Nguyenauthor=S.+G.+B.+Furnessauthor=H.+Venugopalauthor=J.-A.+Baltosauthor=J.+M.+Plitzkoauthor=R.+Danevauthor=W.+Baumeisterauthor=L.+T.+Mayauthor=D.+Woottenauthor=P.+M.+Sextonauthor=A.+Glukhovaauthor=A.+Christopoulos&title=Structure+of+the+adenosine-bound+human+adenosine+A1+receptor%E2%80%93Gi+complex&doi=10.1038%2Fs41586-018-0236-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the adenosine-bound human adenosine A1 receptor-Gi complex</span></div><div class="casAuthors">Draper-Joyce, Christopher J.; Khoshouei, Maryam; Thal, David M.; Liang, Yi-Lynn; Nguyen, Anh T. N.; Furness, Sebastian G. B.; Venugopal, Hariprasad; Baltos, Jo-Anne; Plitzko, Jurgen M.; Danev, Radostin; Baumeister, Wolfgang; May, Lauren T.; Wootten, Denise; Sexton, Patrick M.; Glukhova, Alisa; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">559-563</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The class A adenosine A1 receptor (A1R) is a G-protein-coupled receptor that preferentially couples to inhibitory Gi/o heterotrimeric G proteins, has been implicated in numerous diseases, yet remains poorly targeted.  Here we report the 3.6 Å structure of the human A1R in complex with adenosine and heterotrimeric Gi2 protein detd. by Volta phase plate cryo-electron microscopy.  Compared to inactive A1R, there is contraction at the extracellular surface in the orthosteric binding site mediated via movement of transmembrane domains 1 and 2.  At the intracellular surface, the G protein engages the A1R primarily via amino acids in the C terminus of the Gαi α5-helix, concomitant with a 10.5 Å outward movement of the A1R transmembrane domain 6.  Comparison with the agonist-bound β2 adrenergic receptor-Gs-protein complex reveals distinct orientations for each G-protein subtype upon engagement with its receptor.  This active A1R structure provides mol. insights into receptor and G-protein selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP8kpMZ0qB6rVg90H21EOLACvtfcHk0lgJZAsWwF9VwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL3P&md5=f31a757364c3ff512bd5548b31e7f1bd</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0236-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0236-6%26sid%3Dliteratum%253Aachs%26aulast%3DDraper-Joyce%26aufirst%3DC.%2BJ.%26aulast%3DKhoshouei%26aufirst%3DM.%26aulast%3DThal%26aufirst%3DD.%2BM.%26aulast%3DLiang%26aufirst%3DY.-L.%26aulast%3DNguyen%26aufirst%3DA.%2BT.%2BN.%26aulast%3DFurness%26aufirst%3DS.%2BG.%2BB.%26aulast%3DVenugopal%26aufirst%3DH.%26aulast%3DBaltos%26aufirst%3DJ.-A.%26aulast%3DPlitzko%26aufirst%3DJ.%2BM.%26aulast%3DDanev%26aufirst%3DR.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DMay%26aufirst%3DL.%2BT.%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DGlukhova%26aufirst%3DA.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DStructure%2520of%2520the%2520adenosine-bound%2520human%2520adenosine%2520A1%2520receptor%25E2%2580%2593Gi%2520complex%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D559%26epage%3D563%26doi%3D10.1038%2Fs41586-018-0236-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koehl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paggi, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latorraca, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matile, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schertler, G. F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skiniotis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure of the μ-opioid receptor–Gi protein complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0219-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fs41586-018-0219-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29899455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=558&publication_year=2018&pages=547-552&issue=7711&author=A.+Koehlauthor=H.+Huauthor=S.+Maedaauthor=Y.+Zhangauthor=Q.+Quauthor=J.+M.+Paggiauthor=N.+R.+Latorracaauthor=D.+Hilgerauthor=R.+Dawsonauthor=H.+Matileauthor=G.+F.+X.+Schertlerauthor=S.+Granierauthor=W.+I.+Weisauthor=R.+O.+Drorauthor=A.+Manglikauthor=G.+Skiniotisauthor=B.+K.+Kobilka&title=Structure+of+the+%CE%BC-opioid+receptor%E2%80%93Gi+protein+complex&doi=10.1038%2Fs41586-018-0219-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the μ-opioid receptor-Gi protein complex</span></div><div class="casAuthors">Koehl, Antoine; Hu, Hongli; Maeda, Shoji; Zhang, Yan; Qu, Qianhui; Paggi, Joseph M.; Latorraca, Naomi R.; Hilger, Daniel; Dawson, Roger; Matile, Hugues; Schertler, Gebhard F. X.; Granier, Sebastien; Weis, William I.; Dror, Ron O.; Manglik, Aashish; Skiniotis, Georgios; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">547-552</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The μ-opioid receptor (μOR) is a G-protein-coupled receptor (GPCR) and the target of most clin. and recreationally used opioids.  The induced pos. effects of analgesia and euphoria are mediated by μOR signalling through the adenylyl cyclase-inhibiting heterotrimeric G protein Gi.  Here we present the 3.5 Å resoln. cryo-electron microscopy structure of the μOR bound to the agonist peptide DAMGO and nucleotide-free Gi.  DAMGO occupies the morphinan ligand pocket, with its N terminus interacting with conserved receptor residues and its C terminus engaging regions important for opioid-ligand selectivity.  Comparison of the μOR-Gi complex to previously detd. structures of other GPCRs bound to the stimulatory G protein Gs reveals differences in the position of transmembrane receptor helix 6 and in the interactions between the G protein α-subunit and the receptor core.  Together, these results shed light on the structural features that contribute to the Gi protein-coupling specificity of the μOR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSgJZtgX6dkbVg90H21EOLACvtfcHk0lgJZAsWwF9VwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjP&md5=92cddedb69cfa5f1a78cb419ef8b4ae0</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0219-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0219-7%26sid%3Dliteratum%253Aachs%26aulast%3DKoehl%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DMaeda%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DQ.%26aulast%3DPaggi%26aufirst%3DJ.%2BM.%26aulast%3DLatorraca%26aufirst%3DN.%2BR.%26aulast%3DHilger%26aufirst%3DD.%26aulast%3DDawson%26aufirst%3DR.%26aulast%3DMatile%26aufirst%3DH.%26aulast%3DSchertler%26aufirst%3DG.%2BF.%2BX.%26aulast%3DGranier%26aufirst%3DS.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DSkiniotis%26aufirst%3DG.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520of%2520the%2520%25CE%25BC-opioid%2520receptor%25E2%2580%2593Gi%2520protein%2520complex%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D547%26epage%3D552%26doi%3D10.1038%2Fs41586-018-0219-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuybeda, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Waal, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Eps, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erramilli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melcher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossiakoff, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0215-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fs41586-018-0215-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29899450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=558&publication_year=2018&pages=553-558&issue=7711&author=Y.+Kangauthor=O.+Kuybedaauthor=P.+W.+de+Waalauthor=S.+Mukherjeeauthor=N.+Van+Epsauthor=P.+Dutkaauthor=X.+E.+Zhouauthor=A.+Bartesaghiauthor=S.+Erramilliauthor=T.+Morizumiauthor=X.+Guauthor=Y.+Yinauthor=P.+Liuauthor=Y.+Jiangauthor=X.+Mengauthor=G.+Zhaoauthor=K.+Melcherauthor=O.+P.+Ernstauthor=A.+A.+Kossiakoffauthor=S.+Subramaniamauthor=H.+E.+Xu&title=Cryo-EM+structure+of+human+rhodopsin+bound+to+an+inhibitory+G+protein&doi=10.1038%2Fs41586-018-0215-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span></div><div class="casAuthors">Kang, Yanyong; Kuybeda, Oleg; de Waal, Parker W.; Mukherjee, Somnath; Van Eps, Ned; Dutka, Przemyslaw; Zhou, X. Edward; Bartesaghi, Alberto; Erramilli, Satchal; Morizumi, Takefumi; Gu, Xin; Yin, Yanting; Liu, Ping; Jiang, Yi; Meng, Xing; Zhao, Gongpu; Melcher, Karsten; Ernst, Oliver P.; Kossiakoff, Anthony A.; Subramaniam, Sriram; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">553-558</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) comprise the largest family of mammalian transmembrane receptors.  They mediate numerous cellular pathways by coupling with downstream signalling transducers, including the heterotrimeric G proteins Gs (stimulatory) and Gi (inhibitory) and several arrestin proteins.  The structural mechanisms that define how G-protein-coupled receptors selectively couple to a specific type of G protein or arrestin remain unknown.  Here, using cryo-electron microscopy, we show that the major interactions between activated rhodopsin and Gi are mediated by the C-terminal helix of the Gi α-subunit, which is wedged into the cytoplasmic cavity of the transmembrane helix bundle and directly contacts the amino terminus of helix 8 of rhodopsin.  Structural comparisons of inactive, Gi-bound and arrestin-bound forms of rhodopsin with inactive and Gs-bound forms of the β2-adrenergic receptor provide a foundation to understand the unique structural signatures that are assocd. with the recognition of Gs, Gi and arrestin by activated G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo98ChG6dDQrVg90H21EOLACvtfcHk0ljNJ4iGpx5bJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF&md5=e6d8e79977e0420442fa16ec0c53c692</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0215-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0215-y%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DY.%26aulast%3DKuybeda%26aufirst%3DO.%26aulast%3Dde%2BWaal%26aufirst%3DP.%2BW.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DVan%2BEps%26aufirst%3DN.%26aulast%3DDutka%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DX.%2BE.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DErramilli%26aufirst%3DS.%26aulast%3DMorizumi%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DMelcher%26aufirst%3DK.%26aulast%3DErnst%26aufirst%3DO.%2BP.%26aulast%3DKossiakoff%26aufirst%3DA.%2BA.%26aulast%3DSubramaniam%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DCryo-EM%2520structure%2520of%2520human%2520rhodopsin%2520bound%2520to%2520an%2520inhibitory%2520G%2520protein%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D553%26epage%3D558%26doi%3D10.1038%2Fs41586-018-0215-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yefanov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Waal, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moeller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascal, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Eps, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caro, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnivetskiy, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suino-Powell, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy-Chowdhury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lisova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupitz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotjohann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Standfuss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carragher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbell, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melcher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of rhodopsin bound to arrestin by femtosecond X-ray laser</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>523</i></span> (<span class="NLM_issue">7562</span>),  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1038/nature14656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnature14656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=26200343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WksrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=523&publication_year=2015&pages=561-567&issue=7562&author=Y.+Kangauthor=X.+E.+Zhouauthor=X.+Gaoauthor=Y.+Heauthor=W.+Liuauthor=A.+Ishchenkoauthor=A.+Bartyauthor=T.+A.+Whiteauthor=O.+Yefanovauthor=G.+W.+Hanauthor=Q.+Xuauthor=P.+W.+de+Waalauthor=J.+Keauthor=M.+H.+Tanauthor=C.+Zhangauthor=A.+Moellerauthor=G.+M.+Westauthor=B.+D.+Pascalauthor=N.+Van+Epsauthor=L.+N.+Caroauthor=S.+A.+Vishnivetskiyauthor=R.+J.+Leeauthor=K.+M.+Suino-Powellauthor=X.+Guauthor=K.+Palauthor=J.+Maauthor=X.+Zhiauthor=S.+Boutetauthor=G.+J.+Williamsauthor=M.+Messerschmidtauthor=C.+Gatiauthor=N.+A.+Zatsepinauthor=D.+Wangauthor=D.+Jamesauthor=S.+Basuauthor=S.+Roy-Chowdhuryauthor=C.+E.+Conradauthor=J.+Coeauthor=H.+Liuauthor=S.+Lisovaauthor=C.+Kupitzauthor=I.+Grotjohannauthor=R.+Frommeauthor=Y.+Jiangauthor=M.+Tanauthor=H.+Yangauthor=J.+Liauthor=M.+Wangauthor=Z.+Zhengauthor=D.+Liauthor=N.+Howeauthor=Y.+Zhaoauthor=J.+Standfussauthor=K.+Diederichsauthor=Y.+Dongauthor=C.+S.+Potterauthor=B.+Carragherauthor=M.+Caffreyauthor=H.+Jiangauthor=H.+N.+Chapmanauthor=J.+C.+Spenceauthor=P.+Frommeauthor=U.+Weierstallauthor=O.+P.+Ernstauthor=V.+Katritchauthor=V.+V.+Gurevichauthor=P.+R.+Griffinauthor=W.+L.+Hubbellauthor=R.+C.+Stevensauthor=V.+Cherezovauthor=K.+Melcherauthor=H.+E.+Xu&title=Crystal+structure+of+rhodopsin+bound+to+arrestin+by+femtosecond+X-ray+laser&doi=10.1038%2Fnature14656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of rhodopsin bound to arrestin by femtosecond X-ray laser</span></div><div class="casAuthors">Kang, Yanyong; Zhou, X. Edward; Gao, Xiang; He, Yuanzheng; Liu, Wei; Ishchenko, Andrii; Barty, Anton; White, Thomas A.; Yefanov, Oleksandr; Han, Gye Won; Xu, Qingping; de Waal, Parker W.; Ke, Jiyuan; Tan, M. H. Eileen; Zhang, Chenghai; Moeller, Arne; West, Graham M.; Pascal, Bruce D.; Van Eps, Ned; Caro, Lydia N.; Vishnivetskiy, Sergey A.; Lee, Regina J.; Suino-Powell, Kelly M.; Gu, Xin; Pal, Kuntal; Ma, Jinming; Zhi, Xiaoyong; Boutet, Sebastien; Williams, Garth J.; Messerschmidt, Marc; Gati, Cornelius; Zatsepin, Nadia A.; Wang, Dingjie; James, Daniel; Basu, Shibom; Roy-Chowdhury, Shatabdi; Conrad, Chelsie E.; Coe, Jesse; Liu, Haiguang; Lisova, Stella; Kupitz, Christopher; Grotjohann, Ingo; Fromme, Raimund; Jiang, Yi; Tan, Minjia; Yang, Huaiyu; Li, Jun; Wang, Meitian; Zheng, Zhong; Li, Dianfan; Howe, Nicole; Zhao, Yingming; Standfuss, Jorg; Diederichs, Kay; Dong, Yuhui; Potter, Clinton S.; Carragher, Bridget; Caffrey, Martin; Jiang, Hualiang; Chapman, Henry N.; Spence, John C. H.; Fromme, Petra; Weierstall, Uwe; Ernst, Oliver P.; Katritch, Vsevolod; Gurevich, Vsevolod V.; Griffin, Patrick R.; Hubbell, Wayne L.; Stevens, Raymond C.; Cherezov, Vadim; Melcher, Karsten; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">523</span>
        (<span class="NLM_cas:issue">7562</span>),
    <span class="NLM_cas:pages">561-567</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) signal primarily through G proteins or arrestins.  Arrestin binding to GPCRs blocks G protein interaction and redirects signalling to numerous G-protein-independent pathways.  Here we report the crystal structure of a constitutively active form of human rhodopsin bound to a pre-activated form of the mouse visual arrestin, detd. by serial femtosecond X-ray laser crystallog.  Together with extensive biochem. and mutagenesis data, the structure reveals an overall architecture of the rhodopsin-arrestin assembly in which rhodopsin uses distinct structural elements, including transmembrane helix 7 and helix 8, to recruit arrestin.  Correspondingly, arrestin adopts the pre-activated conformation, with a ∼20° rotation between the amino and carboxy domains, which opens up a cleft in arrestin to accommodate a short helix formed by the second intracellular loop of rhodopsin.  This structure provides a basis for understanding GPCR-mediated arrestin-biased signalling and demonstrates the power of X-ray lasers for advancing the frontiers of structural biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomyJZ-ekzglLVg90H21EOLACvtfcHk0lhPQ9bOMMN1LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WksrvK&md5=574baa8ffa403be6aec8e9e1ffd81a7a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature14656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14656%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%2BE.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DYefanov%26aufirst%3DO.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3Dde%2BWaal%26aufirst%3DP.%2BW.%26aulast%3DKe%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DM.%2BH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMoeller%26aufirst%3DA.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DPascal%26aufirst%3DB.%2BD.%26aulast%3DVan%2BEps%26aufirst%3DN.%26aulast%3DCaro%26aufirst%3DL.%2BN.%26aulast%3DVishnivetskiy%26aufirst%3DS.%2BA.%26aulast%3DLee%26aufirst%3DR.%2BJ.%26aulast%3DSuino-Powell%26aufirst%3DK.%2BM.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DPal%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DZhi%26aufirst%3DX.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DRoy-Chowdhury%26aufirst%3DS.%26aulast%3DConrad%26aufirst%3DC.%2BE.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLisova%26aufirst%3DS.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DStandfuss%26aufirst%3DJ.%26aulast%3DDiederichs%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DPotter%26aufirst%3DC.%2BS.%26aulast%3DCarragher%26aufirst%3DB.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DErnst%26aufirst%3DO.%2BP.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26aulast%3DHubbell%26aufirst%3DW.%2BL.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DMelcher%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DCrystal%2520structure%2520of%2520rhodopsin%2520bound%2520to%2520arrestin%2520by%2520femtosecond%2520X-ray%2520laser%26jtitle%3DNature%26date%3D2015%26volume%3D523%26issue%3D7562%26spage%3D561%26epage%3D567%26doi%3D10.1038%2Fnature14656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic-Nikolic, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uliassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span> <span> </span><span class="NLM_article-title">A perspective on multi-target drug discovery and design for complex diseases</span>. <i>Clin Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1186/s40169-017-0181-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1186%2Fs40169-017-0181-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29340951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A280%3ADC%252BC1MvhsVaiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=3&issue=1&author=R.+R.+Ramsayauthor=M.+R.+Popovic-Nikolicauthor=K.+Nikolicauthor=E.+Uliassiauthor=M.+L.+Bolognesi&title=A+perspective+on+multi-target+drug+discovery+and+design+for+complex+diseases&doi=10.1186%2Fs40169-017-0181-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on multi-target drug discovery and design for complex diseases</span></div><div class="casAuthors">Ramsay Rona R; Popovic-Nikolic Marija R; Nikolic Katarina; Uliassi Elisa; Bolognesi Maria Laura</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and translational medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3</span>
        ISSN:<span class="NLM_cas:issn">2001-1326</span>.
    </div><div class="casAbstract">Diseases of infection, of neurodegeneration (such as Alzheimer's and Parkinson's diseases), and of malignancy (cancers) have complex and varied causative factors.  Modern drug discovery has the power to identify potential modulators for multiple targets from millions of compounds.  Computational approaches allow the determination of the association of each compound with its target before chemical synthesis and biological testing is done.  These approaches depend on the prior identification of clinically and biologically validated targets.  This Perspective will focus on the molecular and computational approaches that underpin drug design by medicinal chemists to promote understanding and collaboration with clinical scientists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJhSGsWpifCZcXNb62YGmbfW6udTcc2eb6xTD66pBJX7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvhsVaiug%253D%253D&md5=5b05a73a3bbc955c77b9b2df93deb946</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1186%2Fs40169-017-0181-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40169-017-0181-2%26sid%3Dliteratum%253Aachs%26aulast%3DRamsay%26aufirst%3DR.%2BR.%26aulast%3DPopovic-Nikolic%26aufirst%3DM.%2BR.%26aulast%3DNikolic%26aufirst%3DK.%26aulast%3DUliassi%26aufirst%3DE.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26atitle%3DA%2520perspective%2520on%2520multi-target%2520drug%2520discovery%2520and%2520design%2520for%2520complex%2520diseases%26jtitle%3DClin%2520Transl.%2520Med.%26date%3D2018%26volume%3D7%26issue%3D1%26spage%3D3%26doi%3D10.1186%2Fs40169-017-0181-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douguet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">46277</span>, <span class="refDoi"> DOI: 10.1038/srep46277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fsrep46277" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28397808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=46277&author=D.+Lagorceauthor=D.+Douguetauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=Computational+analysis+of+calculated+physicochemical+and+ADMET+properties+of+protein-protein+interaction+inhibitors&doi=10.1038%2Fsrep46277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span></div><div class="casAuthors">Lagorce, David; Douguet, Dominique; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46277</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The modulation of PPIs by low mol. wt. chem. compds., particularly by orally bioavailable mols., would be very valuable in numerous disease indications.  However, it is known that PPI inhibitors (iPPIs) tend to have properties that are linked to poor Absorption, Distribution, Metab., Excretion and Toxicity (ADMET) and in some cases to poor clin. outcomes.  Previously reported in silico analyses of iPPIs have essentially focused on physicochem. properties but several other ADMET parameters would be important to assess.  In order to gain new insights into the ADMET properties of iPPIs, computations were carried out on eight datasets collected from several databases.  These datasets involve compds. targeting enzymes, GPCRs, ion channels, nuclear receptors, allosteric modulators, oral marketed drugs, oral natural product-derived marketed drugs and iPPIs.  Several trends are reported that should assist the design and optimization of future PPI inhibitors, either for drug discovery endeavors or for chem. biol. projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw3PmL5znBCLVg90H21EOLACvtfcHk0lgIK4yWCxQvZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D&md5=162ddf6594d20c770c1016313182245c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fsrep46277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep46277%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DDouguet%26aufirst%3DD.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DComputational%2520analysis%2520of%2520calculated%2520physicochemical%2520and%2520ADMET%2520properties%2520of%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D46277%26doi%3D10.1038%2Fsrep46277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">FAF-Drugs3: a web server for compound property calculation and chemical library design</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W200</span>– <span class="NLM_lpage">W207</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1093%2Fnar%2Fgkv353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25883137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W200-W207&issue=W1&author=D.+Lagorceauthor=O.+Sperandioauthor=J.+B.+Baellauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=FAF-Drugs3%3A+a+web+server+for+compound+property+calculation+and+chemical+library+design&doi=10.1093%2Fnar%2Fgkv353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">FAF-Drugs3: a web server for compound property calculation and chemical library design</span></div><div class="casAuthors">Lagorce, David; Sperandio, Olivier; Baell, Jonathan B.; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W200-W207</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Drug attrition late in preclin. or clin. development is a serious economic problem in the field of drug discovery.  These problems can be linked, in part, to the quality of the compd. collections used during the hit generation stage and to the selection of compds. undergoing optimization.  Here, we present FAF-Drugs3, a web server that can be used for drug discovery and chem. biol. projects to help in prepg. compd. libraries and to assist decision-making during the hit selection/lead optimization phase.  Since it was first described in 2006, FAF-Drugs has been significantly modified.  The tool now applies an enhanced structure curation procedure, can filter or analyze mols. with user-defined or eight predefined physicochem. filters as well as with several simple ADMET (absorption, distribution, metab., excretion and toxicity) rules.  In addn., compds. can be filtered using an updated list of 154 hand-curated structural alerts while Pan Assay Interference compds. (PAINS) and other, generally unwanted groups are also investigated.  FAF-Drugs3 offers access to user-friendly html result pages and the possibility to download all computed data.  The server requires as input an SDF file of the compds.; it is open to all users and can be accessed without registration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxYZ-hCUhlmrVg90H21EOLACvtfcHk0lixZottDe3NQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM&md5=5cf8888c489b90f392117ab065986214</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv353%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DO.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DFAF-Drugs3%253A%2520a%2520web%2520server%2520for%2520compound%2520property%2520calculation%2520and%2520chemical%2520library%2520design%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DW1%26spage%3DW200%26epage%3DW207%26doi%3D10.1093%2Fnar%2Fgkv353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lixZottDe3NQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lager, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Østergaard
Nielsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liljefors, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterner, O.</span></span> <span> </span><span class="NLM_article-title">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABAA receptors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">6936</span>– <span class="NLM_lpage">6948</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.bmc.2008.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18541432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=6936-6948&issue=14&author=E.+Lagerauthor=J.+Nilssonauthor=E.+%C3%98stergaard%0ANielsenauthor=M.+Nielsenauthor=T.+Liljeforsauthor=O.+Sterner&title=Affinity+of+3-acyl+substituted+4-quinolones+at+the+benzodiazepine+site+of+GABAA+receptors&doi=10.1016%2Fj.bmc.2008.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABAA receptors</span></div><div class="casAuthors">Lager, Erik; Nilsson, Jakob; Ostergaard Nielsen, Elsebet; Nielsen, Mogens; Liljefors, Tommy; Sterner, Olov</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6936-6948</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The finding that alkyl 1,4-dihydro-4-oxoquinoline-3-carboxylate and N-alkyl-1,4-dihydro-4-oxoquinoline-3-carboxamide derivs. may be high-affinity ligands at the benzodiazepine binding site of the GABAA receptor, prompted a study of 3-acyl-1,4-dihydro-4-oxoquinoline (3-acyl-4-quinolones).  In general, the affinity of the 3-acyl derivs. was found to be comparable with the 3-carboxylate and the 3-carboxamide derivs., and certain substituents (e.g., benzyl) in position 6 were again shown to be important.  As it is believed that the benzodiazepine binding site is situated between an α- and a γ-subunit in the GABAA receptor, selected compds. were tested on the α1β2γ2s, α2β2γ2s and α3β2γ2s GABAA receptor subtypes.  The 3-acyl-4-quinolones display various degrees of selectivity for α1- vs. α2- and α3-contg. receptors, and high-affinity ligands essentially selective for α1 over α3 were developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocF9dBEu1TLrVg90H21EOLACvtfcHk0lhMFV0pNL4Ypg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D&md5=729802775ad271d534dccddae8f1968b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DLager%26aufirst%3DE.%26aulast%3DNilsson%26aufirst%3DJ.%26aulast%3D%25C3%2598stergaard%2BNielsen%26aufirst%3DE.%26aulast%3DNielsen%26aufirst%3DM.%26aulast%3DLiljefors%26aufirst%3DT.%26aulast%3DSterner%26aufirst%3DO.%26atitle%3DAffinity%2520of%25203-acyl%2520substituted%25204-quinolones%2520at%2520the%2520benzodiazepine%2520site%2520of%2520GABAA%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26issue%3D14%26spage%3D6936%26epage%3D6948%26doi%3D10.1016%2Fj.bmc.2008.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, F. J.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of 2-alkylpyrimidine-4,6-diol and 6-alkylpyridine-2,4-diol as potent GPR84 agonists</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=579-583&issue=6&author=Y.+Liuauthor=Q.+Zhangauthor=L.+H.+Chenauthor=H.+Yangauthor=W.+Luauthor=X.+Xieauthor=F.+J.+Nan&title=Design+and+synthesis+of+2-alkylpyrimidine-4%2C6-diol+and+6-alkylpyridine-2%2C4-diol+as+potent+GPR84+agonists&doi=10.1021%2Facsmedchemlett.6b00025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of 2-Alkylpyrimidine-4,6-diol and 6-Alkylpyridine-2,4-diol as Potent GPR84 Agonists</span></div><div class="casAuthors">Liu, Yang; Zhang, Qing; Chen, Lin-Hai; Yang, Hui; Lu, Wei; Xie, Xin; Nan, Fa-Jun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">579-583</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of alkylpyrimidine-4,6-diol derivs. were designed and synthesized as novel GPR84 agonists based on a high-throughput screening (HTS) hit I. 6-Nonylpyridine-2,4-diol was identified as the most potent agonist of GPR84 reported so far, with an EC50 of 0.189 nM.  These novel GPR84 agonists will provide valuable tools for the study of the physiol. functions of GPR84.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyEcxh2lX2Q7Vg90H21EOLACvtfcHk0lhMFV0pNL4Ypg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D&md5=52f50ffae4ab4bbfd79f9eeda0bb063b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00025%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DF.%2BJ.%26atitle%3DDesign%2520and%2520synthesis%2520of%25202-alkylpyrimidine-4%252C6-diol%2520and%25206-alkylpyridine-2%252C4-diol%2520as%2520potent%2520GPR84%2520agonists%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26issue%3D6%26spage%3D579%26epage%3D583%26doi%3D10.1021%2Facsmedchemlett.6b00025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. S.</span></span> <span> </span><span class="NLM_article-title">Nickel-catalyzed rearrangement of 1-acyl-2-vinylcyclopropanes. A mild synthesis of substituted dihydrofurans</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1021/ol052700k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052700k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvFKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=573-576&issue=4&author=R.+K.+Bowmanauthor=J.+S.+Johnson&title=Nickel-catalyzed+rearrangement+of+1-acyl-2-vinylcyclopropanes.+A+mild+synthesis+of+substituted+dihydrofurans&doi=10.1021%2Fol052700k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Nickel-Catalyzed Rearrangement of 1-Acyl-2-vinylcyclopropanes. A Mild Synthesis of Substituted Dihydrofurans</span></div><div class="casAuthors">Bowman, Roy K.; Johnson, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-576</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mild Ni(0)-catalyzed rearrangements of 1-acyl-2-vinylcyclopropanes have been developed.  The room-temp. isomerizations afford dihydrofuran products in yields regularly greater than 90%.  E.g., Ni(COD)-catalyzed rearrangements of 1-acyl-2-vinylcyclopropane I gave 97% dihydrofuran II.  A highly substituted, stereochem. defined cyclopropane was employed in the rearrangement to evaluate the reaction mechanism.  Product anal. indicates that the overall reaction proceeds with retention of configuration at the vinyl-bearing stereogenic center.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw0gn_2J4LdLVg90H21EOLACvtfcHk0lj6ENuWyglQDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvFKnsw%253D%253D&md5=2fd83a59ff122addd736a9c778f92696</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fol052700k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052700k%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DR.%2BK.%26aulast%3DJohnson%26aufirst%3DJ.%2BS.%26atitle%3DNickel-catalyzed%2520rearrangement%2520of%25201-acyl-2-vinylcyclopropanes.%2520A%2520mild%2520synthesis%2520of%2520substituted%2520dihydrofurans%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26issue%3D4%26spage%3D573%26epage%3D576%26doi%3D10.1021%2Fol052700k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuttle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J. A.</span></span> <span> </span><span class="NLM_article-title">Overturning established chemoselectivities: selective reduction of arenes over malonates and cyanoacetates by photoactivated organic electron donors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">10934</span>– <span class="NLM_lpage">10937</span>, <span class="refDoi"> DOI: 10.1021/ja4050168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4050168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFamsrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=10934-10937&issue=30&author=E.+Doniauthor=B.+Mondalauthor=S.+O%E2%80%99Sullivanauthor=T.+Tuttleauthor=J.+A.+Murphy&title=Overturning+established+chemoselectivities%3A+selective+reduction+of+arenes+over+malonates+and+cyanoacetates+by+photoactivated+organic+electron+donors&doi=10.1021%2Fja4050168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overturning Established Chemoselectivities: Selective Reduction of Arenes over Malonates and Cyanoacetates by Photoactivated Organic Electron Donors</span></div><div class="casAuthors">Doni, Eswararao; Mondal, Bhaskar; O'Sullivan, Steven; Tuttle, Tell; Murphy, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">10934-10937</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The prevalence of metal-based reducing reagents, including metals, metal complexes, and metal salts, has produced an empirical order of reactivity that governs our approach to chem. synthesis.  However, this reactivity may be influenced by stabilization of transition states, intermediates, and products through substrate-metal bonding.  This article reports that in the absence of such stabilizing interactions, established chemoselectivities can be overthrown.  Thus, photoactivation of the recently developed neutral org. superelectron donor 5 selectively reduces alkyl-substituted benzene rings in the presence of activated esters and nitriles, in direct contrast to metal-based redns., opening a new perspective on reactivity.  The altered outcomes arising from the org. electron donors are attributed to selective interactions between the neutral org. donors and the arene rings of the substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEFVT1Yr3ZZbVg90H21EOLACvtfcHk0lj6ENuWyglQDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFamsrnI&md5=8b10c1a7b9679fe1d53b9737fdf9beb1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja4050168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4050168%26sid%3Dliteratum%253Aachs%26aulast%3DDoni%26aufirst%3DE.%26aulast%3DMondal%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DS.%26aulast%3DTuttle%26aufirst%3DT.%26aulast%3DMurphy%26aufirst%3DJ.%2BA.%26atitle%3DOverturning%2520established%2520chemoselectivities%253A%2520selective%2520reduction%2520of%2520arenes%2520over%2520malonates%2520and%2520cyanoacetates%2520by%2520photoactivated%2520organic%2520electron%2520donors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26issue%3D30%26spage%3D10934%26epage%3D10937%26doi%3D10.1021%2Fja4050168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amupitan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beddoes, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, J. K.</span></span> <span> </span><span class="NLM_article-title">3-Methylcyclohex-2-enone derivatives as initiators of cyclisation. Part 4. Some bicyclisations</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>0</i></span>,  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1039/p19830000759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1039%2Fp19830000759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADyaL3sXlsFOqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=0&publication_year=1983&pages=759-763&author=J.+A.+Amupitanauthor=R.+L.+Beddoesauthor=O.+S.+Millsauthor=J.+K.+Sutherland&title=3-Methylcyclohex-2-enone+derivatives+as+initiators+of+cyclisation.+Part+4.+Some+bicyclisations&doi=10.1039%2Fp19830000759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">3-Methylcyclohex-2-enone derivatives as initiators of cyclization.  Part 4.  Some bicyclizations</span></div><div class="casAuthors">Amupitan, Joseph; Beddoes, Roy L.; Mills, Owen S.; Sutherland, James K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1:  Organic and Bio-Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">759-63</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    </div><div class="casAbstract">Hydrochrysene and hydrophenanthrene derivs. were prepd. by cyclization of 2-alkenyl-3-methylcyclohexen-2-ones and their epoxides.  E.g., cyclization of cyclohexenone I (R2 = bond, R1 = C6H4OMe-3) with (F3CCO)2O/F3CCO2H at O° gave tetracycles II [R = OMe, R1 = H (III); R = H, R1 = OMe] in 56 and 14% yield, resp., and cyclization of I (R2 = O, R1 = C≡CH) with BCl3 in CH2Cl2 at -78° gave 35% ketol IV.  The structures of III and IV were detd. by x-ray crystallog. anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQmt43i_sXOLVg90H21EOLACvtfcHk0lh1trpE7h0Y-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXlsFOqtb8%253D&md5=65ffaf93be128ada4c7962ae00b372d7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2Fp19830000759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp19830000759%26sid%3Dliteratum%253Aachs%26aulast%3DAmupitan%26aufirst%3DJ.%2BA.%26aulast%3DBeddoes%26aufirst%3DR.%2BL.%26aulast%3DMills%26aufirst%3DO.%2BS.%26aulast%3DSutherland%26aufirst%3DJ.%2BK.%26atitle%3D3-Methylcyclohex-2-enone%2520derivatives%2520as%2520initiators%2520of%2520cyclisation.%2520Part%25204.%2520Some%2520bicyclisations%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1983%26volume%3D0%26spage%3D759%26epage%3D763%26doi%3D10.1039%2Fp19830000759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span>; <span class="NLM_string-name">Tymianski, M.</span>; <span class="NLM_string-name">Garman, D.</span></span> <span> </span><span class="NLM_article-title">Agents And Methods For Treating Ischemic And Other Diseases</span>. Patent <span class="NLM_patent">WO 2012/174488 A2</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=X.+Sun&author=M.+Tymianski&author=D.+Garman&title=Agents+And+Methods+For+Treating+Ischemic+And+Other+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26atitle%3DAgents%2520And%2520Methods%2520For%2520Treating%2520Ischemic%2520And%2520Other%2520Diseases%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krohn, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermanson, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallia, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gartner, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzano, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimoto, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeke, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klenk, D. C.</span></span> <span> </span><span class="NLM_article-title">Measurement of protein using bicinchoninic acid</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>150</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(85)90442-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2F0003-2697%2885%2990442-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=3843705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADyaL2MXlsFKksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=1985&pages=76-85&issue=1&author=P.+K.+Smithauthor=R.+I.+Krohnauthor=G.+T.+Hermansonauthor=A.+K.+Malliaauthor=F.+H.+Gartnerauthor=M.+D.+Provenzanoauthor=E.+K.+Fujimotoauthor=N.+M.+Goekeauthor=B.+J.+Olsonauthor=D.+C.+Klenk&title=Measurement+of+protein+using+bicinchoninic+acid&doi=10.1016%2F0003-2697%2885%2990442-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of protein using bicinchoninic acid</span></div><div class="casAuthors">Smith, P. K.; Krohn, R. I.; Hermanson, G. T.; Mallia, A. K.; Gartner, F. H.; Provenzano, M. D.; Fujimoto, E. K.; Goeke, N. M.; Olson, B. J.; Klenk, D. C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-85</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A method for the colorimetric detn. of proteins is described which uses bicinchoninic acid to monitor the Cu produced during the biuret reaction.  The method (micro and macro) was applied to the detn. of 7 proteins.  Bicinchoninic acid forms a 2:1 complex with Cu, resulting in a stable, highly colored chromophore with an absorbance max. at 562 nm.  The color produced from this reaction is stable and increases in a proportional fashion over a broad range of increasing protein concns.  When compared to the method of O. H. Lowry et al. (1951), the results reported demonstrate a greater tolerance of the bicinchoninate reagent toward such commonly encountered interferences as nonionic detergents and simple buffer salts.  The stability of the reagent and resulting chromophore also allows for a simplified, 1-step anal. and an enhanced flexibility in protocol selection.  This new method maintains the high sensitivity and low protein-to-protein variation assocd. with the Lowry technique.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbD7DGwzaIm7Vg90H21EOLACvtfcHk0lh1trpE7h0Y-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXlsFKksL0%253D&md5=f4f012331cf4dafa50cdc2d345bade38</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2885%2990442-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252885%252990442-7%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DP.%2BK.%26aulast%3DKrohn%26aufirst%3DR.%2BI.%26aulast%3DHermanson%26aufirst%3DG.%2BT.%26aulast%3DMallia%26aufirst%3DA.%2BK.%26aulast%3DGartner%26aufirst%3DF.%2BH.%26aulast%3DProvenzano%26aufirst%3DM.%2BD.%26aulast%3DFujimoto%26aufirst%3DE.%2BK.%26aulast%3DGoeke%26aufirst%3DN.%2BM.%26aulast%3DOlson%26aufirst%3DB.%2BJ.%26aulast%3DKlenk%26aufirst%3DD.%2BC.%26atitle%3DMeasurement%2520of%2520protein%2520using%2520bicinchoninic%2520acid%26jtitle%3DAnal.%2520Biochem.%26date%3D1985%26volume%3D150%26issue%3D1%26spage%3D76%26epage%3D85%26doi%3D10.1016%2F0003-2697%2885%2990442-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span> <i>Maestro</i>, release <span class="NLM_edition">2017-1</span>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Maestro%2C+release+2017-1%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span> <i>Prime</i>, release <span class="NLM_edition">2017-1</span>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Prime%2C+release+2017-1%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">On the role of the crystal environment in determining protein side-chain conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>320</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(02)00470-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&issue=3&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+role+of+the+crystal+environment+in+determining+protein+side-chain+conformations&doi=10.1016%2FS0022-2836%2802%2900470-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0lhqF2GiZwl-Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520role%2520of%2520the%2520crystal%2520environment%2520in%2520determining%2520protein%2520side-chain%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26issue%3D3%26spage%3D597%26epage%3D608%26doi%3D10.1016%2FS0022-2836%2802%2900470-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pincus, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&issue=2&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0lhqF2GiZwl-Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26issue%3D2%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span> <i>Glide Release 2017-1; Schrödinger Suite Prime 2017-1
Induced Fit Docking Protocol</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide+Release+2017-1%3B+Schr%C3%B6dinger+Suite+Prime+2017-1%0AInduced+Fit+Docking+Protocol%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%2520Release%25202017-1%253B%2520Schr%25C3%25B6dinger%2520Suite%2520Prime%25202017-1%250AInduced%2520Fit%2520Docking%2520Protocol%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&issue=2&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0lgt9yz4MAiZ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D2%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span> <i>The PyMOL Molecular Graphics System</i>, version <span class="NLM_edition">1.8</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular+Graphics+System%2C+version+1.8%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Natalia V. Ortiz Zacarías, Kirti K. Chahal, Tereza Šimková, Cas van der Horst, Yi Zheng, Asuka Inoue, Emy Theunissen, Lloyd Mallee, Daan van der Es, Julien Louvel, Adriaan P. IJzerman, Tracy M. Handel, Irina Kufareva, <span class="NLM_string-name hlFld-ContribAuthor">Laura H. Heitman</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Characterization of an Intracellular Covalent Ligand for CC Chemokine Receptor 2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2608-2621. <a href="https://doi.org/10.1021/acs.jmedchem.0c01137" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01137</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01137%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BCharacterization%252Bof%252Ban%252BIntracellular%252BCovalent%252BLigand%252Bfor%252BCC%252BChemokine%252BReceptor%252B2%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26aufirst%3DNatalia%2BV.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D02072020%26date%3D18022021%26volume%3D64%26issue%3D5%26spage%3D2608%26epage%3D2621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pule  Seboletswe</span>, <span class="hlFld-ContribAuthor ">Paul  Awolade</span>, <span class="hlFld-ContribAuthor ">Parvesh  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Developments on the Synthesis and Biological Activities of Fused Pyrimidinone Derivatives. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (13)
                                     , 2050-2067. <a href="https://doi.org/10.1002/cmdc.202100083" title="DOI URL">https://doi.org/10.1002/cmdc.202100083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100083%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DRecent%252BDevelopments%252Bon%252Bthe%252BSynthesis%252Band%252BBiological%252BActivities%252Bof%252BFused%252BPyrimidinone%252BDerivatives%26aulast%3DSeboletswe%26aufirst%3DPule%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D13%26spage%3D2050%26epage%3D2067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of the orthosteric CCR2/CCR5 antagonist TAK-779 and the CCR2 intracellular ligands CCR2-RA-[<i>R</i>], SD-24, JNJ-27141491 and the triazolopyrimidinone derivative <b>8</b>. [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] was used in radioligand binding assays for CCR2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Scheme of the Triazolopyrimidinone Derivatives <b>6</b>–<b>43</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) NaH, <i>n</i>-BuLi, THF, overnight, 0 °C to rt (<b>1a</b>–<b>e</b>,<b>i</b>,<b>l</b>,<b>m</b> were commercially available); (ii) DIPEA, LiCl, THF, reflux, overnight; (iii) (<b>8</b>–<b>43</b>, R<sup>2</sup> = NH<sub>2</sub>) BMIM-PF6, 200 °C, 1 h or (<b>6</b>, R<sup>2</sup> = H<b>)</b> H<sub>3</sub>PO<sub>4</sub>, EtOH, 170 °C, 10 h or (<b>7</b>, R<sup>2</sup> = Me) <i>p</i>-toluenesulfonic acid monohydrate, 180 °C, 30 min.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characterization of ligands in U2OS-CCR2 and U2OS-CCR5. (a) [<sup>3</sup>H]-CCR2-RA-[<i>R</i>] displacement by increasing concentrations of triazolopyrimidinone derivatives <b>8</b>, <b>39</b>, and <b>43</b> in U2OS-CCR2 at 25 °C. Data are normalized to specific binding in the absence of compound (set as 100%). (b) Inhibition of CCL2-stimulated β-arrestin recruitment in U2OS-CCR2 by increasing concentrations of compounds <b>8</b>, <b>39</b>, and <b>43</b>, after stimulation with an EC<sub>80</sub> concentration of CCL2 (set as 100%). (c) Inhibition of CCL3-stimulated β-arrestin recruitment in U2OS-CCR5 by increasing concentrations of compounds <b>8</b>, <b>39</b>, and <b>43</b>, after stimulation with an EC<sub>80</sub> concentration of CCL3 (set as 100%). All data are from single, representative experiments performed in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of compounds <b>39</b> and <b>43</b> as insurmountable, negative allosteric modulators using a [<sup>35</sup>S]GTPγS binding assay in hCCR2 and hCCR5. Effect of increasing concentrations of <b>39</b> and <b>43</b> in a CCL2-stimulated [<sup>35</sup>S]GTPγS binding in U2OS-CCR2 (a, b) or in a CCL3-stimulated [<sup>35</sup>S]GTPγS binding in U2OS-CCR5 (c, d), at 25 °C. Parameters obtained from the concentration–response curves (pEC<sub>50</sub>, <i>E</i><sub>max</sub>) are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Data are presented as mean ± SEM values of three experiments performed in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/medium/jm9b00742_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Proposed binding mode of <b>43</b> in hCCR2b. (a) Overlay of <b>43</b> with the CCR2 intracellular ligand CCR2-RA-[<i>R</i>], showing that <b>43</b> interacts in a similar manner as CCR2-RA-[<i>R</i>]. (b) Docking of <b>43</b>, displaying the interactions with CCR2. The amino group in R<sup>2</sup> makes an extra hydrogen-bond interaction with E<sup>8×48</sup>, while the bromine group in R<sup>1</sup> makes an extra halogen bond with the backbone of V<sup>1×53</sup>, which might contribute to the improved affinity of <b>43</b> to this receptor. Model of hCCR2 is based on the crystal structure of CCR2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1A">5T1A</a>),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and amino acid residues are labeled according to their structure-based Ballesteros–Weinstein numbers.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-24/acs.jmedchem.9b00742/20191219/images/large/jm9b00742_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00742&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i112">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03688" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03688" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachelerie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combadiere, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farber, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparre-Ulrich, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luster, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Power, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkilde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sozzani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thelen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshie, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnik, A.</span></span> <span> </span><span class="NLM_article-title">International union of basic and clinical pharmacology. LXXXIX. Update on the extended family of chemokine receptors and introducing a new nomenclature for atypical chemokine receptors</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1124/pr.113.007724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fpr.113.007724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24218476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2014&pages=1-79&issue=1&author=F.+Bachelerieauthor=A.+Ben-Baruchauthor=A.+M.+Burkhardtauthor=C.+Combadiereauthor=J.+M.+Farberauthor=G.+J.+Grahamauthor=R.+Horukauthor=A.+H.+Sparre-Ulrichauthor=M.+Locatiauthor=A.+D.+Lusterauthor=A.+Mantovaniauthor=K.+Matsushimaauthor=P.+M.+Murphyauthor=R.+Nibbsauthor=H.+Nomiyamaauthor=C.+A.+Powerauthor=A.+E.+Proudfootauthor=M.+M.+Rosenkildeauthor=A.+Rotauthor=S.+Sozzaniauthor=M.+Thelenauthor=O.+Yoshieauthor=A.+Zlotnik&title=International+union+of+basic+and+clinical+pharmacology.+LXXXIX.+Update+on+the+extended+family+of+chemokine+receptors+and+introducing+a+new+nomenclature+for+atypical+chemokine+receptors&doi=10.1124%2Fpr.113.007724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">International union of basic and clinical pharmacology LXXXIX. Update on the extended family of chemokine receptors and introducing a new nomenclature for atypical chemokine receptors</span></div><div class="casAuthors">Bachelerie, Francoise; Ben-Baruch, Adit; Burkhardt, Amanda M.; Combadiere, Christophe; Farber, Joshua M.; Graham, Gerard J.; Horuk, Richard; Sparre-Ulrich, Alexander Hovard; Locati, Massimo; Luster, Andrew D.; Mantovani, Alberto; Matsushima, Kouji; Murphy, Philip M.; Nibbs, Robert; Nomiyama, Hisayuki; Power, Christine A.; Proudfoot, Amanda E. I.; Rosenkilde, Mette M.; Rot, Antal; Sozzani, Silvano; Thelen, Marcus; Yoshie, Osamu; Zlotnik, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-79, 80 pp.</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Sixteen years ago, the Nomenclature Committee of the International Union of Pharmacol. approved a system for naming human seven-transmembrane (7TM) G protein-coupled chemokine receptors, the large family of leukocyte chemoattractant receptors that regulates immune system development and function, in large part by mediating leukocyte trafficking.  This was announced in Pharmacol. Reviews in a major overview of the first decade of research in this field.  Since then, several new receptors have been discovered, and major advances have been made for the others in many areas, including structural biol., signal transduction mechanisms, biol., and pharmacol.  New and diverse roles have been identified in infection, immunity, inflammation, development, cancer, and other areas.  The first two drugs acting at chemokine receptors have been approved by the U.S. Food and Drug Administration (FDA), maraviroc targeting CCR5 in human immunodeficiency virus (HIV)/AIDS, and plerixafor targeting CXCR4 for stem cell mobilization for transplantation in cancer, and other candidates are now undergoing pivotal clin. trials for diverse disease indications.  In addn., a subfamily of atypical chemokine receptors has emerged that may signal through arrestins instead of G proteins to act as chemokine scavengers, and many microbial and invertebrate G protein-coupled chemokine receptors and sol. chemokine-binding proteins have been described.  Here, we review this extended family of chemokine receptors and chemokine-binding proteins at the basic, translational, and clin. levels, including an update on drug development.  We also introduce a new nomenclature for atypical chemokine receptors with the stem ACKR (atypical chemokine receptor) approved by the Nomenclature Committee of the International Union of Pharmacol. and the Human Genome Nomenclature Committee.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7EhMCJlz-zLVg90H21EOLACvtfcHk0lhjFUFr1iWAVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D&md5=44a2c14ddf2508ada59121610c5682cf</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fpr.113.007724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.113.007724%26sid%3Dliteratum%253Aachs%26aulast%3DBachelerie%26aufirst%3DF.%26aulast%3DBen-Baruch%26aufirst%3DA.%26aulast%3DBurkhardt%26aufirst%3DA.%2BM.%26aulast%3DCombadiere%26aufirst%3DC.%26aulast%3DFarber%26aufirst%3DJ.%2BM.%26aulast%3DGraham%26aufirst%3DG.%2BJ.%26aulast%3DHoruk%26aufirst%3DR.%26aulast%3DSparre-Ulrich%26aufirst%3DA.%2BH.%26aulast%3DLocati%26aufirst%3DM.%26aulast%3DLuster%26aufirst%3DA.%2BD.%26aulast%3DMantovani%26aufirst%3DA.%26aulast%3DMatsushima%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DP.%2BM.%26aulast%3DNibbs%26aufirst%3DR.%26aulast%3DNomiyama%26aufirst%3DH.%26aulast%3DPower%26aufirst%3DC.%2BA.%26aulast%3DProudfoot%26aufirst%3DA.%2BE.%26aulast%3DRosenkilde%26aufirst%3DM.%2BM.%26aulast%3DRot%26aufirst%3DA.%26aulast%3DSozzani%26aufirst%3DS.%26aulast%3DThelen%26aufirst%3DM.%26aulast%3DYoshie%26aufirst%3DO.%26aulast%3DZlotnik%26aufirst%3DA.%26atitle%3DInternational%2520union%2520of%2520basic%2520and%2520clinical%2520pharmacology.%2520LXXXIX.%2520Update%2520on%2520the%2520extended%2520family%2520of%2520chemokine%2520receptors%2520and%2520introducing%2520a%2520new%2520nomenclature%2520for%2520atypical%2520chemokine%2520receptors%26jtitle%3DPharmacol.%2520Rev.%26date%3D2014%26volume%3D66%26issue%3D1%26spage%3D1%26epage%3D79%26doi%3D10.1124%2Fpr.113.007724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Cotarelo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Moreira, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Criado-Garcia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Fernandez, J. L.</span></span> <span> </span><span class="NLM_article-title">Beyond chemoattraction: multifunctionality of chemokine receptors in leukocytes</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">927</span>– <span class="NLM_lpage">941</span>, <span class="refDoi"> DOI: 10.1016/j.it.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.it.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28935522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=927-941&issue=12&author=P.+Lopez-Cotareloauthor=C.+Gomez-Moreiraauthor=O.+Criado-Garciaauthor=L.+Sanchezauthor=J.+L.+Rodriguez-Fernandez&title=Beyond+chemoattraction%3A+multifunctionality+of+chemokine+receptors+in+leukocytes&doi=10.1016%2Fj.it.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond Chemoattraction: Multifunctionality of Chemokine Receptors in Leukocytes</span></div><div class="casAuthors">Lopez-Cotarelo, Pilar; Gomez-Moreira, Carolina; Criado-Garcia, Olga; Sanchez, Lucas; Rodriguez-Fernandez, Jose Luis</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">927-941</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The word chemokine is a combination of the words chemotactic and cytokine, in other words cytokines that promote chemotaxis.  Hence, the term chemokine receptor refers largely to the ability to regulate chemoattraction.  However, these receptors can modulate addnl. leukocyte functions, as exemplified by the case of CCR7 which, apart from chemotaxis, regulates survival, migratory speed, endocytosis, differentiation and cytoarchitecture.  We present evidence highlighting that multifunctionality is a common feature of chemokine receptors.  Based on the activities that they regulate, we suggest that chemokine receptors can be classified into inflammatory (which control both inflammatory and homeostatic functions) and homeostatic families.  The information accrued also suggests that the non-chemotactic functions controlled by chemokine receptors may contribute to optimizing leukocyte functioning under normal physiol. conditions and during inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqERh6uUdiQvbVg90H21EOLACvtfcHk0liHyCYO-8GIww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnE&md5=8930a5b6d6d9e26235d406fe90cb0441</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Cotarelo%26aufirst%3DP.%26aulast%3DGomez-Moreira%26aufirst%3DC.%26aulast%3DCriado-Garcia%26aufirst%3DO.%26aulast%3DSanchez%26aufirst%3DL.%26aulast%3DRodriguez-Fernandez%26aufirst%3DJ.%2BL.%26atitle%3DBeyond%2520chemoattraction%253A%2520multifunctionality%2520of%2520chemokine%2520receptors%2520in%2520leukocytes%26jtitle%3DTrends%2520Immunol.%26date%3D2017%26volume%3D38%26issue%3D12%26spage%3D927%26epage%3D941%26doi%3D10.1016%2Fj.it.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schall, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, A. E.</span></span> <span> </span><span class="NLM_article-title">Overcoming hurdles in developing successful drugs targeting chemokine receptors</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">363</span>, <span class="refDoi"> DOI: 10.1038/nri2972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnri2972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=21494268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslCmtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=355-363&issue=5&author=T.+J.+Schallauthor=A.+E.+Proudfoot&title=Overcoming+hurdles+in+developing+successful+drugs+targeting+chemokine+receptors&doi=10.1038%2Fnri2972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming hurdles in developing successful drugs targeting chemokine receptors</span></div><div class="casAuthors">Schall, Thomas J.; Proudfoot, Amanda E. I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-363</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemokines and their receptors are central to the inflammatory process and are attractive therapeutic targets.  Drugs that inhibit chemokine receptors are approved for the treatment of HIV infection and for stem cell mobilization, but none have been approved yet for the treatment of inflammatory and/or autoimmune diseases.  We analyze the challenges of developing chemokine receptor antagonists, and propose that inappropriate target selection and ineffective dosing, not the 'redundancy' of the chemokine system, are the main barriers to their use as anti-inflammatory therapies.  We highlight evidence suggesting that chemokine receptor inhibition will prove to be an effective therapy in inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJxZhyoAteTLVg90H21EOLACvtfcHk0liHyCYO-8GIww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslCmtb0%253D&md5=3173fc889bd22b7d7e19bd138a599897</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnri2972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2972%26sid%3Dliteratum%253Aachs%26aulast%3DSchall%26aufirst%3DT.%2BJ.%26aulast%3DProudfoot%26aufirst%3DA.%2BE.%26atitle%3DOvercoming%2520hurdles%2520in%2520developing%2520successful%2520drugs%2520targeting%2520chemokine%2520receptors%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2011%26volume%3D11%26issue%3D5%26spage%3D355%26epage%3D363%26doi%3D10.1038%2Fnri2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toraskar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span> <span> </span><span class="NLM_article-title">Bias in chemokine receptor signalling</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/j.it.2014.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.it.2014.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24679437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlt1elurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=243-252&issue=6&author=A.+J.+Zweemerauthor=J.+Toraskarauthor=L.+H.+Heitmanauthor=A.+P.+IJzerman&title=Bias+in+chemokine+receptor+signalling&doi=10.1016%2Fj.it.2014.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bias in chemokine receptor signalling</span></div><div class="casAuthors">Zweemer, Annelien J. M.; Toraskar, Jimita; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">243-252</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Chemokine receptors are widely expressed on a variety of immune cells and play a crucial role in normal physiol. as well as in inflammatory and infectious diseases.  The existence of 23 chemokine receptors and 48 chemokine ligands guarantees a tight control and fine-tuning of the immune system.  Here, we discuss the multiple regulatory mechanisms of chemokine signalling at a systemic, cellular, and mol. level.  In particular, we focus on the impact of biased signalling at the receptor level; an emerging concept in mol. pharmacol.  An improved understanding of these mechanisms may provide a framework for more effective drug discovery and development at a target class that is so relevant for immune function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqru6UHEP6GorVg90H21EOLACvtfcHk0lhAP2rBPPlm8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlt1elurw%253D&md5=f2aaab7e9b3c0a8f52af3eafd5a69579</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2014.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2014.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%26aulast%3DToraskar%26aufirst%3DJ.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DBias%2520in%2520chemokine%2520receptor%2520signalling%26jtitle%3DTrends%2520Immunol.%26date%3D2014%26volume%3D35%26issue%3D6%26spage%3D243%26epage%3D252%26doi%3D10.1016%2Fj.it.2014.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koelink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Kruijf, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkerts, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraneveld, A. D.</span></span> <span> </span><span class="NLM_article-title">Targeting chemokine receptors in chronic inflammatory diseases: an extensive review</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.pharmthera.2011.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=21839114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=1-18&issue=1&author=P.+J.+Koelinkauthor=S.+A.+Overbeekauthor=S.+Braberauthor=P.+de+Kruijfauthor=G.+Folkertsauthor=M.+J.+Smitauthor=A.+D.+Kraneveld&title=Targeting+chemokine+receptors+in+chronic+inflammatory+diseases%3A+an+extensive+review&doi=10.1016%2Fj.pharmthera.2011.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting chemokine receptors in chronic inflammatory diseases: An extensive review</span></div><div class="casAuthors">Koelink, Pim J.; Overbeek, Saskia A.; Braber, Saskia; de Kruijf, Petra; Folkerts, Gert; Smit, Martine J.; Kraneveld, Aletta D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-18</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The traffic of the different types of immune cells is an important aspect in the immune response.  Chemokines are sol. peptides that are able to attract cells by interaction with chemokine receptors on their target cells.  Several different chemokines and receptors exist enabling the specific trafficking of different immune cells.  In chronic inflammatory disorders there is abundance of immune cells present at the inflammatory site.  This review focuses on the role of chemokine receptors in chronic inflammatory disorders of the lungs, intestine, joints, skin and nervous system and the potential of targeting these receptors as therapeutic intervention in these disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ8i7YbGTlCrVg90H21EOLACvtfcHk0lhAP2rBPPlm8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Ciur%252FO&md5=165ed0c4dde555d6e89262ee6ae7fc2a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DKoelink%26aufirst%3DP.%2BJ.%26aulast%3DOverbeek%26aufirst%3DS.%2BA.%26aulast%3DBraber%26aufirst%3DS.%26aulast%3Dde%2BKruijf%26aufirst%3DP.%26aulast%3DFolkerts%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DKraneveld%26aufirst%3DA.%2BD.%26atitle%3DTargeting%2520chemokine%2520receptors%2520in%2520chronic%2520inflammatory%2520diseases%253A%2520an%2520extensive%2520review%26jtitle%3DPharmacol.%2520Ther.%26date%3D2012%26volume%3D133%26issue%3D1%26spage%3D1%26epage%3D18%26doi%3D10.1016%2Fj.pharmthera.2011.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greaves, D. R.</span></span> <span> </span><span class="NLM_article-title">CC chemokine receptors and chronic inflammation—therapeutic opportunities and pharmacological challenges</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1124/pr.111.005074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fpr.111.005074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=23300131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjt1Sktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2013&pages=47-89&issue=1&author=G.+E.+Whiteauthor=A.+J.+Iqbalauthor=D.+R.+Greaves&title=CC+chemokine+receptors+and+chronic+inflammation%E2%80%94therapeutic+opportunities+and+pharmacological+challenges&doi=10.1124%2Fpr.111.005074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">CC chemokine receptors and chronic inflammation-therapeutic opportunities and pharmacological challenges</span></div><div class="casAuthors">White, Gemma E.; Iqbal, Asif J.; Greaves, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-89</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Chemokines are a family of low mol. wt. proteins with an essential role in leukocyte trafficking during both homeostasis and inflammation.  The CC class of chemokines consists of at least 28 members (CCL1-28) that signal through 10 known chemokine receptors (CCR1-10).  CC chemokine receptors are expressed predominantly by T cells and monocytemacrophages, cell types assocd. predominantly with chronic inflammation occurring over weeks or years.  Chronic inflammatory diseases including rheumatoid arthritis, atherosclerosis, and metabolic syndrome are characterized by continued leukocyte infiltration into the inflammatory site, driven in large part by excessive chemokine prodn.  Over years or decades, persistent inflammation may lead to loss of tissue architecture and function, causing severe disability or, in the case of atherosclerosis, fatal outcomes such as myocardial infarction or stroke.  Despite the existence of several clin. strategies for targeting chronic inflammation, these diseases remain significant causes of morbidity and mortality globally, with a concomitant economic impact.  Thus, the development of novel therapeutic agents for the treatment of chronic inflammatory disease continues to be a priority.  In this review we introduce CC chemokine receptors as crit. mediators of chronic inflammatory responses and explore their potential role as pharmacol. targets.  We discuss functions of individual CC chemokine receptors based on in vitro pharmacol. data as well as transgenic animal studies.  Focusing on three key forms of chronic inflammation-rheumatoid arthritis, atherosclerosis, and metabolic syndrome-we describe the pathol. function of CC chemokine receptors and their possible relevance as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0NGFhZPIXbbVg90H21EOLACvtfcHk0lhX0v4beZIGVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjt1Sktbg%253D&md5=1c06c5629ef1e866e98c39e012722c95</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005074%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DG.%2BE.%26aulast%3DIqbal%26aufirst%3DA.%2BJ.%26aulast%3DGreaves%26aufirst%3DD.%2BR.%26atitle%3DCC%2520chemokine%2520receptors%2520and%2520chronic%2520inflammation%25E2%2580%2594therapeutic%2520opportunities%2520and%2520pharmacological%2520challenges%26jtitle%3DPharmacol.%2520Rev.%26date%3D2013%26volume%3D65%26issue%3D1%26spage%3D47%26epage%3D89%26doi%3D10.1124%2Fpr.111.005074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woollard, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanmogne, G. D.</span></span> <span> </span><span class="NLM_article-title">Maraviroc: a review of its use in HIV infection and beyond</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5447</span>– <span class="NLM_lpage">5468</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S90580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.2147%2FDDDT.S90580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=26491256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXls1Sjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=5447-5468&author=S.+M.+Woollardauthor=G.+D.+Kanmogne&title=Maraviroc%3A+a+review+of+its+use+in+HIV+infection+and+beyond&doi=10.2147%2FDDDT.S90580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Maraviroc: a review of its use in HIV infection and beyond</span></div><div class="casAuthors">Woollard, Shawna M.; Kanmogne, Georgette D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5447-5468</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">The human immunodeficiency virus-1 (HIV-1) enters target cells by binding its envelope glycoprotein gp120 to the CD4 receptor and/or coreceptors such as C-C chemokine receptor type 5 (CCR5; R5) and C-X-C chemokine receptor type 4 (CXCR4; X4), and R5-tropic viruses predominate during the early stages of infection.  CCR5 antagonists bind to CCR5 to prevent viral entry.  Maraviroc (MVC) is the only CCR5 antagonist currently approved by the United States Food and Drug Administration, the European Commission, Health Canada, and several other countries for the treatment of patients infected with R5-tropic HIV-1.  MVC has been shown to be effective at inhibiting HIV-1 entry into cells and is well tolerated.  With expanding MVC use by HIV-1-infected humans, different clin. outcomes post-approval have been obsd. with MVC monotherapy or combination therapy with other antiretroviral drugs, with MVC use in humans infected with dual-R5- and X4-tropic HIV-1, infected with different HIV-1 genotype or infected with HIV-2.  This review discuss the role of CCR5 in HIV-1 infection, the development of the CCR5 antagonist MVC, its pharmacokinetics, pharmacodynamics, drug-drug interactions, and the implications of these interactions on treatment outcomes, including viral mutations and drug resistance, and the mechanisms assocd. with the development of resistance to MVC.  This review also discusses available studies investigating the use of MVC in the treatment of other diseases such as cancer, graft-vs.-host disease, and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV-IihM4Xq0LVg90H21EOLACvtfcHk0lhX0v4beZIGVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXls1Sjtr8%253D&md5=7e43ebf96fe63773cd8e153a2c233a86</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S90580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S90580%26sid%3Dliteratum%253Aachs%26aulast%3DWoollard%26aufirst%3DS.%2BM.%26aulast%3DKanmogne%26aufirst%3DG.%2BD.%26atitle%3DMaraviroc%253A%2520a%2520review%2520of%2520its%2520use%2520in%2520HIV%2520infection%2520and%2520beyond%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2015%26volume%3D9%26spage%3D5447%26epage%3D5468%26doi%3D10.2147%2FDDDT.S90580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span> <span> </span><span class="NLM_article-title">Chemokine receptor antagonists: overcoming developmental hurdles</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1038/nrd2734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnrd2734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=19079127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt12j" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=23-33&issue=1&author=R.+Horuk&title=Chemokine+receptor+antagonists%3A+overcoming+developmental+hurdles&doi=10.1038%2Fnrd2734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemokine receptor antagonists: overcoming developmental hurdles</span></div><div class="casAuthors">Horuk, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-33</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemokine receptors have a key role in the pathogenesis of autoimmune diseases, inflammation and viral infection.  However, with the exception of selective CCR5 antagonists for HIV, the promise of obtaining new therapeutics related to chemokine receptors has not yet been realized.  This article highlights some of the recent failures in the clin. trials of chemokine receptor antagonists and explores possible reasons as to why this might have occurred.  Such reasons include the lack of predictability of animal models and redundancy of the target.  A potential soln. could be to develop drugs that target more than one receptor - known as polypharmacol. - which could be a novel way to generate effective therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFMos9Ccf_G7Vg90H21EOLACvtfcHk0lhX0v4beZIGVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt12j&md5=831294beadc398d0dc3007a26da96d39</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd2734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2734%26sid%3Dliteratum%253Aachs%26aulast%3DHoruk%26aufirst%3DR.%26atitle%3DChemokine%2520receptor%2520antagonists%253A%2520overcoming%2520developmental%2520hurdles%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26issue%3D1%26spage%3D23%26epage%3D33%26doi%3D10.1038%2Fnrd2734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span> <span> </span><span class="NLM_article-title">Promiscuous drugs as therapeutics for chemokine receptors</span>. <i>Expert Rev. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">e1</span>, <span class="refDoi"> DOI: 10.1017/S1462399409000921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1017%2FS1462399409000921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=19123963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FltlCmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=e1&author=R.+Horuk&title=Promiscuous+drugs+as+therapeutics+for+chemokine+receptors&doi=10.1017%2FS1462399409000921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous drugs as therapeutics for chemokine receptors</span></div><div class="casAuthors">Horuk Richard</div><div class="citationInfo"><span class="NLM_cas:title">Expert reviews in molecular medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemokine receptor antagonists that held much promise for the treatment of autoimmune and inflammatory diseases have recently performed poorly in clinical trials, resulting in disappointment for both pharmaceutical companies and patients.  This review focuses on the redundancy of the molecular target as one potential reason for the failure of some of these antagonists to fulfil their initial promise, and discusses the use of drugs that are capable of interacting with more than one drug target - so-called promiscuous drugs - as possible approaches to overcome this difficulty.  Several clinically approved promiscuous drugs, such as aspirin and olanzapine, are already used successfully.  This review discusses examples of promiscuous drugs for G-protein-coupled receptors, including progress in developing dual-specific chemokine receptor antagonists, and considers evidence for the possible therapeutic utility of such drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQ5kVOYrQTyNLIORRPd5lbfW6udTcc2eZ8Hcw8wBG4VLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FltlCmsQ%253D%253D&md5=8c4bf8bc9017136bdeb1489ebce6f4c7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1017%2FS1462399409000921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399409000921%26sid%3Dliteratum%253Aachs%26aulast%3DHoruk%26aufirst%3DR.%26atitle%3DPromiscuous%2520drugs%2520as%2520therapeutics%2520for%2520chemokine%2520receptors%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2009%26volume%3D11%26spage%3De1%26doi%3D10.1017%2FS1462399409000921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratziu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelmalek, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aithal, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caballeria, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francque, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowdley, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craxi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchor-Khan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vest, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiens, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seyedkazemi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, V. W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loomba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanyal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, E.</span></span> <span> </span><span class="NLM_article-title">A randomized, placebo-controlled trial of cenicriviroc for treatment of nonalcoholic steatohepatitis with fibrosis</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1754</span>– <span class="NLM_lpage">1767</span>, <span class="refDoi"> DOI: 10.1002/hep.29477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1002%2Fhep.29477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28833331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotlCms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=1754-1767&issue=5&author=S.+L.+Friedmanauthor=V.+Ratziuauthor=S.+A.+Harrisonauthor=M.+F.+Abdelmalekauthor=G.+P.+Aithalauthor=J.+Caballeriaauthor=S.+Francqueauthor=G.+Farrellauthor=K.+V.+Kowdleyauthor=A.+Craxiauthor=K.+Simonauthor=L.+Fischerauthor=L.+Melchor-Khanauthor=J.+Vestauthor=B.+L.+Wiensauthor=P.+Vigauthor=S.+Seyedkazemiauthor=Z.+Goodmanauthor=V.+W.-S.+Wongauthor=R.+Loombaauthor=F.+Tackeauthor=A.+Sanyalauthor=E.+Lefebvre&title=A+randomized%2C+placebo-controlled+trial+of+cenicriviroc+for+treatment+of+nonalcoholic+steatohepatitis+with+fibrosis&doi=10.1002%2Fhep.29477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A randomized, placebo-controlled trial of cenicriviroc for treatment of nonalcoholic steatohepatitis with fibrosis</span></div><div class="casAuthors">Friedman, Scott L.; Ratziu, Vlad; Harrison, Stephen A.; Abdelmalek, Manal F.; Aithal, Guruprasad P.; Caballeria, Juan; Francque, Sven; Farrell, Geoffrey; Kowdley, Kris V.; Craxi, Antonio; Simon, Krzysztof; Fischer, Laurent; Melchor-Khan, Liza; Vest, Jeffrey; Wiens, Brian L.; Vig, Pamela; Seyedkazemi, Star; Goodman, Zachary; Wong, Vincent Wai-Sun; Loomba, Rohit; Tacke, Frank; Sanyal, Arun; Lefebvre, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1754-1767</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The aim of this study was to evaluate cenicriviroc (CVC), a dual antagonist of C-C chemokine receptor types 2 and 5, for treatment of nonalcoholic steatohepatitis (NASH) with liver fibrosis (LF).  A randomized, double-blind, multinational phase 2b study enrolled subjects with NASH, a nonalcoholic fatty liver disease activity score (NAS) ≥4, and LF (stages 1-3, NASH Clin. Research Network) at 81 clin. sites.  Subjects (N = 289) were randomly assigned CVC 150 mg or placebo.  Primary outcome was ≥2-point improvement in NAS and no worsening of fibrosis at year 1.  Key secondary outcomes were: resoln. of steatohepatitis (SH) and no worsening of fibrosis; improvement in fibrosis by ≥1 stage and no worsening of SH.  Biomarkers of inflammation and adverse events were assessed.  Full study recruitment was achieved.  The primary endpoint of NAS improvement in the intent-to-treat population and resoln. of SH was achieved in a similar proportion of subjects on CVC (N = 145) and placebo (N = 144; 16% vs. 19%, P = 0.52 and 8% vs. 6%, P = 0.49, resp.).  However, the fibrosis endpoint was met in significantly more subjects on CVC than placebo (20% vs. 10%; P = 0.02).  Treatment benefits were greater in those with higher disease activity and fibrosis stage at baseline.  Biomarkers of systemic inflammation were reduced with CVC.  Safety and tolerability of CVC were comparable to placebo.  Conclusion: After 1 yr of CVC treatment, twice as many subjects achieved improvement in fibrosis and no worsening of SH compared with placebo.  Given the urgent need to develop antifibrotic therapies in NASH, these findings warrant phase 3 evaluation. (Hepatol. 2018;67:1754-1767).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_4ETuDf9irVg90H21EOLACvtfcHk0lgyk9wpFHbgHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotlCms7g%253D&md5=a712985806b627b992895637d9f59b58</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fhep.29477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.29477%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%26aufirst%3DS.%2BL.%26aulast%3DRatziu%26aufirst%3DV.%26aulast%3DHarrison%26aufirst%3DS.%2BA.%26aulast%3DAbdelmalek%26aufirst%3DM.%2BF.%26aulast%3DAithal%26aufirst%3DG.%2BP.%26aulast%3DCaballeria%26aufirst%3DJ.%26aulast%3DFrancque%26aufirst%3DS.%26aulast%3DFarrell%26aufirst%3DG.%26aulast%3DKowdley%26aufirst%3DK.%2BV.%26aulast%3DCraxi%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DK.%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DMelchor-Khan%26aufirst%3DL.%26aulast%3DVest%26aufirst%3DJ.%26aulast%3DWiens%26aufirst%3DB.%2BL.%26aulast%3DVig%26aufirst%3DP.%26aulast%3DSeyedkazemi%26aufirst%3DS.%26aulast%3DGoodman%26aufirst%3DZ.%26aulast%3DWong%26aufirst%3DV.%2BW.-S.%26aulast%3DLoomba%26aufirst%3DR.%26aulast%3DTacke%26aufirst%3DF.%26aulast%3DSanyal%26aufirst%3DA.%26aulast%3DLefebvre%26aufirst%3DE.%26atitle%3DA%2520randomized%252C%2520placebo-controlled%2520trial%2520of%2520cenicriviroc%2520for%2520treatment%2520of%2520nonalcoholic%2520steatohepatitis%2520with%2520fibrosis%26jtitle%3DHepatology%26date%3D2018%26volume%3D67%26issue%3D5%26spage%3D1754%26epage%3D1767%26doi%3D10.1002%2Fhep.29477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">Dual targeting of CCR2 and CCR5: therapeutic potential for immunologic and cardiovascular diseases</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>88</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.1189/jlb.1009671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1189%2Fjlb.1009671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=20360402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVWqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2010&pages=41-55&issue=1&author=Q.+Zhao&title=Dual+targeting+of+CCR2+and+CCR5%3A+therapeutic+potential+for+immunologic+and+cardiovascular+diseases&doi=10.1189%2Fjlb.1009671"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting of CCR2 and CCR5: therapeutic potential for immunologic and cardiovascular diseases</span></div><div class="casAuthors">Zhao, Qihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review.  A cardinal feature of inflammation is the tissue recruitment of leukocytes, a process that is mediated predominantly by chemokines via their receptors on migrating cells.  CCR2 and CCR5, two CC chemokine receptors, are important players in the trafficking of monocytes/macrophages and in the functions of other cell types relevant to disease pathogenesis.  This review provides a brief overview of the biol. actions of CCR2 and CCR5 and a comprehensive summary of published data that demonstrate the involvement of both receptors in the pathogenesis of immunol. diseases (RA, CD, and transplant rejection) and cardiovascular diseases (atherosclerosis and AIH).  In light of the potential for functional redundancy of chemokine receptors in mediating leukocyte trafficking and the consequent concern over insufficient efficacy offered by pharmacol. inhibiting one receptor, this review presents evidence supporting dual targeting of CCR2 and CCR5 as a more efficacious strategy than targeting either receptor alone.  It also examines potential safety issues assocd. with such dual targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc5RCcoXQyu7Vg90H21EOLACvtfcHk0lhWIUlLtS-nHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVWqs78%253D&md5=f9ff14b43951bee94c0ffea6da30e7d4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1189%2Fjlb.1009671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.1009671%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DDual%2520targeting%2520of%2520CCR2%2520and%2520CCR5%253A%2520therapeutic%2520potential%2520for%2520immunologic%2520and%2520cardiovascular%2520diseases%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D2010%26volume%3D88%26issue%3D1%26spage%3D41%26epage%3D55%26doi%3D10.1189%2Fjlb.1009671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fantuzzi, L.</span>; <span class="NLM_string-name">Tagliamonte, M.</span>; <span class="NLM_string-name">Gauzzi, M. C.</span>; <span class="NLM_string-name">Lopalco, L.</span></span> <span> </span><span class="NLM_article-title">Dual CCR5/CCR2 targeting: opportunities for the cure of complex disorders</span>.  <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1007/s00018-019-03255-6</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1007%2Fs00018-019-03255-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=31377844" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=L.+Fantuzzi&author=M.+Tagliamonte&author=M.+C.+Gauzzi&author=L.+Lopalco&title=Dual+CCR5%2FCCR2+targeting%3A+opportunities+for+the+cure+of+complex+disorders&doi=10.1007%2Fs00018-019-03255-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00018-019-03255-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-019-03255-6%26sid%3Dliteratum%253Aachs%26aulast%3DFantuzzi%26aufirst%3DL.%26atitle%3DDual%2520CCR5%252FCCR2%2520targeting%253A%2520opportunities%2520for%2520the%2520cure%2520of%2520complex%2520disorders%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2019%26doi%3D10.1007%2Fs00018-019-03255-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Junker, A.</span>; <span class="NLM_string-name">Kokornaczyk, A. K.</span>; <span class="NLM_string-name">Strunz, A. K.</span>; <span class="NLM_string-name">Wünsch, B.</span></span> <span> </span><span class="NLM_article-title">Selective and dual targeting of CCR2 and CCR5 receptors: a current overview</span>. In  <i>Chemokines: Chemokines and Their Receptors in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>, Ed.; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Cham, Switzerland</span>, <span class="NLM_year">2015</span>; Vol.  <span class="NLM_volume">14</span>, pp  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">241</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=187-241&author=A.+Junker&author=A.+K.+Kokornaczyk&author=A.+K.+Strunz&author=B.+W%C3%BCnschauthor=N.+Tschammer&title=Chemokines%3A+Chemokines+and+Their+Receptors+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJunker%26aufirst%3DA.%26atitle%3DSelective%2520and%2520dual%2520targeting%2520of%2520CCR2%2520and%2520CCR5%2520receptors%253A%2520a%2520current%2520overview%26btitle%3DChemokines%253A%2520Chemokines%2520and%2520Their%2520Receptors%2520in%2520Drug%2520Discovery%26aulast%3DTschammer%26aufirst%3DN.%26pub%3DSpringer%2520International%2520Publishing%26date%3D2015%26volume%3D14%26spage%3D187%26epage%3D241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kothandan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadhe, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. J.</span></span> <span> </span><span class="NLM_article-title">Structural insights from binding poses of CCR2 and CCR5 with clinically important antagonists: a combined in silico study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e32864</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0032864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1371%2Fjournal.pone.0032864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=22479344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Cnt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e32864&issue=3&author=G.+Kothandanauthor=C.+G.+Gadheauthor=S.+J.+Cho&title=Structural+insights+from+binding+poses+of+CCR2+and+CCR5+with+clinically+important+antagonists%3A+a+combined+in+silico+study&doi=10.1371%2Fjournal.pone.0032864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights from binding poses of CCR2 and CCR5 with clinically important antagonists: a combined in silico study</span></div><div class="casAuthors">Kothandan, Gugan; Gadhe, Changdev G.; Cho, Seung Joo</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e32864</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Chemokine receptors are G protein-coupled receptors that contain seven transmembrane domains.  In particular, CCR2 and CCR5 and their ligands have been implicated in the pathophysiol. of a no. of diseases, including rheumatoid arthritis and multiple sclerosis.  Based on their roles in disease, they have been attractive targets for the pharmaceutical industry, and furthermore, targeting both CCR2 and CCR5 can be a useful strategy.  Owing to the importance of these receptors, information regarding the binding site is of prime importance.  Structural studies have been hampered due to the lack of X-ray crystal structures, and templates with close homologs for comparative modeling.  Most of the previous models were based on the bovine rhodopsin and β2-adrenergic receptor.  In this study, based on a closer homolog with higher resoln. (CXCR4, PDB code: 3ODU 2.5 Å), we constructed three-dimensional models.  The main aim of this study was to provide relevant information on binding sites of these receptors.  Mol. dynamics simulation was done to refine the homol. models and PROCHECK results indicated that the models were reasonable.  Here, binding poses were checked with some established inhibitors of high pharmaceutical importance against the modeled receptors.  Anal. of interaction modes gave an integrated interpretation with detailed structural information.  The binding poses confirmed that the acidic residues Glu291 (CCR2) and Glu283 (CCR5) are important, and we also found some addnl. residues.  Comparisons of binding sites of CCR2/CCR5 were done sequentially and also by docking a potent dual antagonist.  Our results can be a starting point for further structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ql7HNG7LfLVg90H21EOLACvtfcHk0lg8JEVU0h_nig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Cnt7g%253D&md5=aee8357749939e2f9ec46e469b78ac8c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0032864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0032864%26sid%3Dliteratum%253Aachs%26aulast%3DKothandan%26aufirst%3DG.%26aulast%3DGadhe%26aufirst%3DC.%2BG.%26aulast%3DCho%26aufirst%3DS.%2BJ.%26atitle%3DStructural%2520insights%2520from%2520binding%2520poses%2520of%2520CCR2%2520and%2520CCR5%2520with%2520clinically%2520important%2520antagonists%253A%2520a%2520combined%2520in%2520silico%2520study%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D3%26spage%3De32864%26doi%3D10.1371%2Fjournal.pone.0032864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz Zacarías, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustavsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherney, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tebben, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handel, T. M.</span></span> <span> </span><span class="NLM_article-title">Structure of CC chemokine receptor 2 with orthosteric and allosteric antagonists</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span> (<span class="NLM_issue">7633</span>),  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1038/nature20605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnature20605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=27926736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=458-461&issue=7633&author=Y.+Zhengauthor=L.+Qinauthor=N.+V.+Ortiz+Zacar%C3%ADasauthor=H.+de+Vriesauthor=G.+W.+Hanauthor=M.+Gustavssonauthor=M.+Dabrosauthor=C.+Zhaoauthor=R.+J.+Cherneyauthor=P.+Carterauthor=D.+Stamosauthor=R.+Abagyanauthor=V.+Cherezovauthor=R.+C.+Stevensauthor=A.+P.+IJzermanauthor=L.+H.+Heitmanauthor=A.+Tebbenauthor=I.+Kufarevaauthor=T.+M.+Handel&title=Structure+of+CC+chemokine+receptor+2+with+orthosteric+and+allosteric+antagonists&doi=10.1038%2Fnature20605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of CC chemokine receptor 2 with orthosteric and allosteric antagonists</span></div><div class="casAuthors">Zheng, Yi; Qin, Ling; Zacarias, Natalia V. Ortiz; de Vries, Henk; Han, Gye Won; Gustavsson, Martin; Dabros, Marta; Zhao, Chunxia; Cherney, Robert J.; Carter, Percy; Stamos, Dean; Abagyan, Ruben; Cherezov, Vadim; Stevens, Raymond C.; IJzerman, Adriaan P.; Heitman, Laura H.; Tebben, Andrew; Kufareva, Irina; Handel, Tracy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">458-461</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CC chemokine receptor 2 (CCR2) is one of 19 members of the chemokine receptor subfamily of human class A G-protein-coupled receptors.  CCR2 is expressed on monocytes, immature dendritic cells, and T-cell subpopulations, and mediates their migration towards endogenous CC chemokine ligands such as CCL2 (ref. 1).  CCR2 and its ligands are implicated in numerous inflammatory and neurodegenerative diseases including atherosclerosis, multiple sclerosis, asthma, neuropathic pain, and diabetic nephropathy, as well as cancer.  These disease assocns. have motivated numerous preclin. studies and clin. trials (see http://www.clinicaltrials.gov) in search of therapies that target the CCR2-chemokine axis.  To aid drug discovery efforts, here we solve a structure of CCR2 in a ternary complex with an orthosteric (BMS-681 (ref. 6)) and allosteric (CCR2-RA-[R]7) antagonist.  BMS-681 inhibits chemokine binding by occupying the orthosteric pocket of the receptor in a previously unseen binding mode.  CCR2-RA-[R] binds in a novel, highly druggable pocket that is the most intracellular allosteric site obsd. in class A G-protein-coupled receptors so far; this site spatially overlaps the G-protein-binding site in homologous receptors.  CCR2-RA-[R] inhibits CCR2 non-competitively by blocking activation-assocd. conformational changes and formation of the G-protein-binding interface.  The conformational signature of the conserved microswitch residues obsd. in double-antagonist-bound CCR2 resembles the most inactive G-protein-coupled receptor structures solved so far.  Like other protein-protein interactions, receptor-chemokine complexes are considered challenging therapeutic targets for small mols., and the present structure suggests diverse pocket epitopes that can be exploited to overcome obstacles in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPPLlhgBLBUrVg90H21EOLACvtfcHk0lg8JEVU0h_nig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurbP&md5=32e8631bdbf7a125d9c7d2333555041e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature20605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20605%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26aufirst%3DN.%2BV.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DGustavsson%26aufirst%3DM.%26aulast%3DDabros%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DCherney%26aufirst%3DR.%2BJ.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DStamos%26aufirst%3DD.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DTebben%26aufirst%3DA.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DHandel%26aufirst%3DT.%2BM.%26atitle%3DStructure%2520of%2520CC%2520chemokine%2520receptor%25202%2520with%2520orthosteric%2520and%2520allosteric%2520antagonists%26jtitle%3DNature%26date%3D2016%26volume%3D540%26issue%3D7633%26spage%3D458%26epage%3D461%26doi%3D10.1038%2Fnature20605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oswald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dore, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deflorian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Intracellular allosteric antagonism of the CCR9 receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span> (<span class="NLM_issue">7633</span>),  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1038/nature20606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnature20606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=27926729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=462-465&issue=7633&author=C.+Oswaldauthor=M.+Rappasauthor=J.+Keanauthor=A.+S.+Doreauthor=J.+C.+Erreyauthor=K.+Bennettauthor=F.+Deflorianauthor=J.+A.+Christopherauthor=A.+Jazayeriauthor=J.+S.+Masonauthor=M.+Congreveauthor=R.+M.+Cookeauthor=F.+H.+Marshall&title=Intracellular+allosteric+antagonism+of+the+CCR9+receptor&doi=10.1038%2Fnature20606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular allosteric antagonism of the CCR9 receptor</span></div><div class="casAuthors">Oswald, Christine; Rappas, Mathieu; Kean, James; Dore, Andrew S.; Errey, James C.; Bennett, Kirstie; Deflorian, Francesca; Christopher, John A.; Jazayeri, Ali; Mason, Jonathan S.; Congreve, Miles; Cooke, Robert M.; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">462-465</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chemokines and their G-protein-coupled receptors play a diverse role in immune defense by controlling the migration, activation and survival of immune cells.  They are also involved in viral entry, tumor growth and metastasis and hence are important drug targets in a wide range of diseases.  Despite very significant efforts by the pharmaceutical industry to develop drugs, with over 50 small-mol. drugs directed at the family entering clin. development, only two compds. have reached the market: maraviroc (CCR5) for HIV infection and plerixafor (CXCR4) for stem-cell mobilization.  The high failure rate may in part be due to limited understanding of the mechanism of action of chemokine antagonists and an inability to optimize compds. in the absence of structural information.  CC chemokine receptor type 9 (CCR9) activation by CCL25 plays a key role in leukocyte recruitment to the gut and represents a therapeutic target in inflammatory bowel disease.  The selective CCR9 antagonist vercirnon progressed to phase 3 clin. trials in Crohn's disease but efficacy was limited, with the need for very high doses to block receptor activation.  Here we report the crystal structure of the CCR9 receptor in complex with vercirnon at 2.8 Å resoln.  Remarkably, vercirnon binds to the intracellular side of the receptor, exerting allosteric antagonism and preventing G-protein coupling.  This binding site explains the need for relatively lipophilic ligands and describes another example of an allosteric site on G-protein-coupled receptors that can be targeted for drug design, not only at CCR9, but potentially extending to other chemokine receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHomIAvUxar7Vg90H21EOLACvtfcHk0lgD1n5zGwsGOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM&md5=253b630bd2b9b65fc192ff86de9df61a</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature20606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20606%26sid%3Dliteratum%253Aachs%26aulast%3DOswald%26aufirst%3DC.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DDore%26aufirst%3DA.%2BS.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DBennett%26aufirst%3DK.%26aulast%3DDeflorian%26aufirst%3DF.%26aulast%3DChristopher%26aufirst%3DJ.%2BA.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DIntracellular%2520allosteric%2520antagonism%2520of%2520the%2520CCR9%2520receptor%26jtitle%3DNature%26date%3D2016%26volume%3D540%26issue%3D7633%26spage%3D462%26epage%3D465%26doi%3D10.1038%2Fnature20606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedjai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, D. J.</span></span> <span> </span><span class="NLM_article-title">Cytokines and chemokines: at the crossroads of cell signalling and inflammatory disease</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1843</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2563</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.bbamcr.2014.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24892271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlWrur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=2563-2582&issue=11&author=M.+D.+Turnerauthor=B.+Nedjaiauthor=T.+Hurstauthor=D.+J.+Pennington&title=Cytokines+and+chemokines%3A+at+the+crossroads+of+cell+signalling+and+inflammatory+disease&doi=10.1016%2Fj.bbamcr.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cytokines and chemokines: At the crossroads of cell signalling and inflammatory disease</span></div><div class="casAuthors">Turner, Mark D.; Nedjai, Belinda; Hurst, Tara; Pennington, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2582</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Inflammation occurs as a result of exposure of tissues and organs to harmful stimuli such as microbial pathogens, irritants, or toxic cellular components.  The primary phys. manifestations of inflammation are redness, swelling, heat, pain, and loss of function to the affected area.  These processes involve the major cells of the immune system, including monocytes, macrophages, neutrophils, basophils, dendritic cells, mast cells, T-cells, and B-cells.  However, examn. of a range of inflammatory lesions demonstrates the presence of specific leukocytes in any given lesion.  That is, the inflammatory process is regulated in such a way as to ensure that the appropriate leukocytes are recruited.  These events are in turn controlled by a host of extracellular mol. regulators, including members of the cytokine and chemokine families that mediate both immune cell recruitment and complex intracellular signalling control mechanisms that characterize inflammation.  This review will focus on the role of the main cytokines, chemokines, and their receptors in the pathophysiol. of auto-inflammatory disorders, pro-inflammatory disorders, and neurol. disorders involving inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXDUTKEa7ptrVg90H21EOLACvtfcHk0lgbnQ7kOJNh_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlWrur%252FI&md5=6a5667f3148836d1dcab9bef2eef3bef</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DM.%2BD.%26aulast%3DNedjai%26aufirst%3DB.%26aulast%3DHurst%26aufirst%3DT.%26aulast%3DPennington%26aufirst%3DD.%2BJ.%26atitle%3DCytokines%2520and%2520chemokines%253A%2520at%2520the%2520crossroads%2520of%2520cell%2520signalling%2520and%2520inflammatory%2520disease%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26issue%3D11%26spage%3D2563%26epage%3D2582%26doi%3D10.1016%2Fj.bbamcr.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
Zacarias, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handel, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span> <span> </span><span class="NLM_article-title">Intracellular receptor modulation: novel approach to target GPCRs</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2018.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.tips.2018.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29653834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVCjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=547-559&issue=6&author=N.+V.+Ortiz%0AZacariasauthor=E.+B.+Lenselinkauthor=A.+P.+IJzermanauthor=T.+M.+Handelauthor=L.+H.+Heitman&title=Intracellular+receptor+modulation%3A+novel+approach+to+target+GPCRs&doi=10.1016%2Fj.tips.2018.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular Receptor Modulation: Novel Approach to Target GPCRs</span></div><div class="casAuthors">Ortiz Zacarias, Natalia V.; Lenselink, Eelke B.; IJzerman, Adriaan P.; Handel, Tracy M.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">547-559</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent crystal structures of multiple G protein-coupled receptors (GPCRs) have revealed a highly conserved intracellular pocket that can be used to modulate these receptors from the inside.  This novel intracellular site partially overlaps with the G protein and β-arrestin binding site, providing a new manner of pharmacol. intervention.  Here we provide an update of the architecture and function of the intracellular region of GPCRs, until now portrayed as the signaling domain.  We review the available evidence on the presence of intracellular binding sites among chemokine receptors and other class A GPCRs, as well as different strategies to target it, including small mols., pepducins, and nanobodies.  Finally, the potential advantages of intracellular (allosteric) ligands over orthosteric ligands are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk_hgIKJ-qnrVg90H21EOLACvtfcHk0lgbnQ7kOJNh_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVCjtLY%253D&md5=28b63bab9f1dd44769fcc3728be0177f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2018.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2018.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2BZacarias%26aufirst%3DN.%2BV.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHandel%26aufirst%3DT.%2BM.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DIntracellular%2520receptor%2520modulation%253A%2520novel%2520approach%2520to%2520target%2520GPCRs%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2018%26volume%3D39%26issue%3D6%26spage%3D547%26epage%3D559%26doi%3D10.1016%2Fj.tips.2018.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
Zacarías, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Veldhoven, J. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Spronsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenhuizen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Velden, W. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oenema, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span> <span> </span><span class="NLM_article-title">Pyrrolone derivatives as intracellular allosteric modulators for chemokine receptors: selective and dual-targeting inhibitors of CC chemokine receptors 1 and 2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9146</span>– <span class="NLM_lpage">9161</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9146-9161&issue=20&author=N.+V.+Ortiz%0AZacar%C3%ADasauthor=J.+P.+D.+van+Veldhovenauthor=L.+Portnerauthor=E.+van+Spronsenauthor=S.+Ulloauthor=M.+Veenhuizenauthor=W.+J.+C.+van+der+Veldenauthor=A.+J.+M.+Zweemerauthor=R.+M.+Kreekelauthor=K.+Oenemaauthor=E.+B.+Lenselinkauthor=L.+H.+Heitmanauthor=A.+P.+IJzerman&title=Pyrrolone+derivatives+as+intracellular+allosteric+modulators+for+chemokine+receptors%3A+selective+and+dual-targeting+inhibitors+of+CC+chemokine+receptors+1+and+2&doi=10.1021%2Facs.jmedchem.8b00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolone Derivatives as Intracellular Allosteric Modulators for Chemokine Receptors: Selective and Dual-Targeting Inhibitors of CC Chemokine Receptors 1 and 2</span></div><div class="casAuthors">Ortiz Zacarias, Natalia V.; van Veldhoven, Jacobus P. D.; Portner, Laura; van Spronsen, Eric; Ullo, Salviana; Veenhuizen, Margo; van der Velden, Wijnand J. C.; Zweemer, Annelien J. M.; Kreekel, Roy M.; Oenema, Kenny; Lenselink, Eelke B.; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9146-9161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent crystal structures of CC chemokine receptors 2 and 9 (CCR2 and CCR9) have provided structural evidence for an allosteric, intracellular binding site.  The high conservation of residues involved in this site suggests its presence in most chemokine receptors, including the close homolog CCR1.  By using [3H]CCR2-RA-[R], a high-affinity, CCR2 intracellular ligand, we report an intracellular binding site in CCR1, where this radioligand also binds with high affinity.  In addn., we report the synthesis and biol. characterization of a series of pyrrolone derivs. for CCR1 and CCR2, which allowed us to identify several high-affinity intracellular ligands, including selective and potential multitarget antagonists.  Evaluation of selected compds. in a functional [35S]GTPγS assay revealed that they act as inverse agonists in CCR1, providing a new manner of pharmacol. modulation.  Thus, this intracellular binding site enables the design of selective and multitarget inhibitors as a novel therapeutic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3HeUOWen2U7Vg90H21EOLACvtfcHk0lg86q2AiODRZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM&md5=bf05c560b983659de579074242d299d1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00605%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26aufirst%3DN.%2BV.%26aulast%3Dvan%2BVeldhoven%26aufirst%3DJ.%2BP.%2BD.%26aulast%3DPortner%26aufirst%3DL.%26aulast%3Dvan%2BSpronsen%26aufirst%3DE.%26aulast%3DUllo%26aufirst%3DS.%26aulast%3DVeenhuizen%26aufirst%3DM.%26aulast%3Dvan%2Bder%2BVelden%26aufirst%3DW.%2BJ.%2BC.%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DKreekel%26aufirst%3DR.%2BM.%26aulast%3DOenema%26aufirst%3DK.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DPyrrolone%2520derivatives%2520as%2520intracellular%2520allosteric%2520modulators%2520for%2520chemokine%2520receptors%253A%2520selective%2520and%2520dual-targeting%2520inhibitors%2520of%2520CC%2520chemokine%2520receptors%25201%2520and%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D20%26spage%3D9146%26epage%3D9161%26doi%3D10.1021%2Facs.jmedchem.8b00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nederpelt, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrieling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doornbos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span> <span> </span><span class="NLM_article-title">Multiple binding sites for small-molecule antagonists at the CC chemokine receptor 2</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>84</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">561</span>, <span class="refDoi"> DOI: 10.1124/mol.113.086850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.113.086850" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=23877010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVygt7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2013&pages=551-561&issue=4&author=A.+J.+Zweemerauthor=I.+Nederpeltauthor=H.+Vrielingauthor=S.+Hafithauthor=M.+L.+Doornbosauthor=H.+de+Vriesauthor=J.+Abtauthor=R.+Grossauthor=D.+Stamosauthor=J.+Saundersauthor=M.+J.+Smitauthor=A.+P.+IJzermanauthor=L.+H.+Heitman&title=Multiple+binding+sites+for+small-molecule+antagonists+at+the+CC+chemokine+receptor+2&doi=10.1124%2Fmol.113.086850"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple binding sites for small-molecule antagonists at the CC chemokine receptor 2</span></div><div class="casAuthors">Zweemer, Annelien J. M.; Nederpelt, Indira; Vrieling, Hilde; Hafith, Sarah; Doornbos, Maarten L. J.; de Vries, Henk; Abt, Jeffrey; Gross, Raymond; Stamos, Dean; Saunders, John; Smit, Martine J.; IJzerman, Adriaan P.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">551-561</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptor CCR2 is a G protein-coupled receptor that is activated primarily by the endogenous CC chemokine ligand 2 (CCL2).  Many different small-mol. antagonists have been developed to inhibit this receptor, as it is involved in a variety of diseases characterized by chronic inflammation.  Unfortunately, all these antagonists lack clin. efficacy, and therefore a better understanding of their mechanism of action is warranted.  In this study, we examd. the pharmacol. properties of small-mol. CCR2 antagonists in radioligand binding and functional assays.  Six structurally different antagonists were selected for this study, all of which displaced the endogenous agonist 125I-CCL2 from CCR2 with nanomolar affinity.  Two of these antagonists, INCB3344 [N-(2-(((3S,4S)-1-((1r,4S)-4-(benzo[d][1,3]dioxol-5-yl)-4-hydroxycyclohexyl)-4-ethoxypyrrolidin-3-yl)amino)-2-oxoethyl)-3-(trifluoromethyl)benzamide] and CCR2-RA, were radiolabeled to study the binding site in greater detail.  We discovered that [3H]INCB3344 and [3H]CCR2-RA bind to distinct binding sites at CCR2, the latter being the first allosteric radioligand for CCR2.  Besides the binding properties of the antagonists, we examd. CCR2 inhibition in multiple functional assays, including a novel label-free whole-cell assay.  INCB3344 competitively inhibited CCL2-induced G protein activation, whereas CCR2-RA showed a noncompetitive or allosteric mode of inhibition.  These findings demonstrated that the CCR2 antagonists examd. in this study can be classified into two groups with different binding sites and thereby different modes of inhibition.  We have provided further insights in CCR2 antagonism, and these insights are important for the development of novel CCR2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyrR-A2sHKNLVg90H21EOLACvtfcHk0lg86q2AiODRZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVygt7%252FF&md5=61e8e2f1d0d587d98833fa5f2d088351</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.086850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.086850%26sid%3Dliteratum%253Aachs%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%26aulast%3DNederpelt%26aufirst%3DI.%26aulast%3DVrieling%26aufirst%3DH.%26aulast%3DHafith%26aufirst%3DS.%26aulast%3DDoornbos%26aufirst%3DM.%2BL.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DAbt%26aufirst%3DJ.%26aulast%3DGross%26aufirst%3DR.%26aulast%3DStamos%26aufirst%3DD.%26aulast%3DSaunders%26aufirst%3DJ.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DMultiple%2520binding%2520sites%2520for%2520small-molecule%2520antagonists%2520at%2520the%2520CC%2520chemokine%2520receptor%25202%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D84%26issue%3D4%26spage%3D551%26epage%3D561%26doi%3D10.1124%2Fmol.113.086850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenhuizen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miraglia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilums, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiele, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkilde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span> <span> </span><span class="NLM_article-title">Discovery and mapping of an intracellular antagonist binding site at the chemokine receptor CCR2</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1124/mol.114.093328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.114.093328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25024169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1ynu7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=358-368&issue=4&author=A.+J.+Zweemerauthor=J.+Bunnikauthor=M.+Veenhuizenauthor=F.+Miragliaauthor=E.+B.+Lenselinkauthor=M.+Vilumsauthor=H.+de+Vriesauthor=A.+Gibertauthor=S.+Thieleauthor=M.+M.+Rosenkildeauthor=A.+P.+IJzermanauthor=L.+H.+Heitman&title=Discovery+and+mapping+of+an+intracellular+antagonist+binding+site+at+the+chemokine+receptor+CCR2&doi=10.1124%2Fmol.114.093328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and mapping of an intracellular antagonist binding site at the chemokine receptor CCR2</span></div><div class="casAuthors">Zweemer, Annelien J. M.; Bunnik, Julia; Veenhuizen, Margo; Miraglia, Fabiana; Lenselink, Eelke B.; Vilums, Maris; de Vries, Henk; Gibert, Arthur; Thiele, Stefanie; Rosenkilde, Mette M.; IJzerman, Adriaan P.; Heitman, Laura H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-368, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptor CCR2 is a G protein-coupled receptor that is involved in many diseases characterized by chronic inflammation, and therefore a large variety of CCR2 small mol. antagonists has been developed.  On the basis of their chem. structures these antagonists can roughly be divided into two groups with most likely two topog. distinct binding sites.  The aim of the current study was to identify the binding site of one such group of ligands, exemplified by three allosteric antagonists, CCR2-RA-[R], JNJ-27141491, and SD-24.  We first used a chimeric CCR2/CCR5 receptor approach to obtain insight into the binding site of the allosteric antagonists and addnl. introduced eight single point mutations in CCR2 to further characterize the putative binding pocket.  All constructs were studied in radioligand binding and/or functional IP turnover assays, providing evidence for an intracellular binding site for CCR2-RA-[R], JNJ-27141491, and SD-24.  For CCR2-RA-[R] the most important residues for binding were found to be the highly conserved tyrosine Y7.53 and phenylalanine F8.50 of the NPxxYx(5,6)F motif, as well as V6.36 at the bottom of TM-VI and K8.49 in helix-VIII.  These findings demonstrate for the first time the presence of an allosteric intracellular binding site for CCR2 antagonists.  This contributes to an increased understanding of the interactions of diverse ligands at CCR2 and may allow for a more rational design of future allosteric antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeUNOf8f0n9bVg90H21EOLACvtfcHk0ljEuWg0QNm-Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1ynu7bN&md5=9776ef16d97485c2f9c91ea0c0d1870c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.093328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.093328%26sid%3Dliteratum%253Aachs%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%26aulast%3DBunnik%26aufirst%3DJ.%26aulast%3DVeenhuizen%26aufirst%3DM.%26aulast%3DMiraglia%26aufirst%3DF.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DVilums%26aufirst%3DM.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DGibert%26aufirst%3DA.%26aulast%3DThiele%26aufirst%3DS.%26aulast%3DRosenkilde%26aufirst%3DM.%2BM.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26atitle%3DDiscovery%2520and%2520mapping%2520of%2520an%2520intracellular%2520antagonist%2520binding%2520site%2520at%2520the%2520chemokine%2520receptor%2520CCR2%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26issue%3D4%26spage%3D358%26epage%3D368%26doi%3D10.1124%2Fmol.114.093328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wreggett, K. A.</span></span> <span> </span><span class="NLM_article-title">An intracellular allosteric site for a specific class of antagonists of the CC chemokine G protein-coupled receptors CCR4 and CCR5</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">867</span>, <span class="refDoi"> DOI: 10.1124/mol.107.039321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.107.039321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18042736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1egsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=855-867&issue=3&author=G.+Andrewsauthor=C.+Jonesauthor=K.+A.+Wreggett&title=An+intracellular+allosteric+site+for+a+specific+class+of+antagonists+of+the+CC+chemokine+G+protein-coupled+receptors+CCR4+and+CCR5&doi=10.1124%2Fmol.107.039321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An intracellular allosteric site for a specific class of antagonists of the CC chemokine G protein-coupled receptors CCR4 and CCR5</span></div><div class="casAuthors">Andrews, Glen; Jones, Carolyn; Wreggett, Keith A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">855-867</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A novel mechanism for antagonism of the human chemokine receptors CCR4 and CCR5 has been discovered with a series of small-mol. compds. that seems to interact with an allosteric, intracellular site on the receptor.  The existence of this site is supported by a series of observations: (1) intracellular access of these antagonists is required for their activity; (2) specific, saturable binding of a radiolabeled antagonist requires the presence of CCR4; and (3) through engineering receptor chimeras by reciprocal transfer of C-terminal domains between CCR4 and CCR5, compd. binding and the selective structure-activity relationships for antagonism of these receptors seem to be assocd. with the integrity of that intracellular region.  Published antagonists from other chem. series do not seem to bind to the novel site, and their interaction with either CCR4 or CCR5 is not affected by alteration of the C-terminal domain.  The precise location of the proposed binding site remains to be detd., but the known close assocn. of the C-terminal domain, including helix 8, as a proposed intracellular region that interacts with transduction proteins (e.g., G proteins and β-arrestin) suggests that this could be a generic allosteric site for chemokine receptors and perhaps more broadly for class A G protein-coupled receptors.  The existence of such a site that can be targeted for drug discovery has implications for screening assays for receptor antagonists, which would need, therefore, to consider compd. properties for access to this intracellular site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAPVG6Nu0Wf7Vg90H21EOLACvtfcHk0ljEuWg0QNm-Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1egsLw%253D&md5=f3624c421bac61365f63ea1389dffea0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fmol.107.039321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.107.039321%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DC.%26aulast%3DWreggett%26aufirst%3DK.%2BA.%26atitle%3DAn%2520intracellular%2520allosteric%2520site%2520for%2520a%2520specific%2520class%2520of%2520antagonists%2520of%2520the%2520CC%2520chemokine%2520G%2520protein-coupled%2520receptors%2520CCR4%2520and%2520CCR5%26jtitle%3DMol.%2520Pharmacol.%26date%3D2008%26volume%3D73%26issue%3D3%26spage%3D855%26epage%3D867%26doi%3D10.1124%2Fmol.107.039321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bengtsson, B. A.</span>; <span class="NLM_string-name">Blackaby, W.</span>; <span class="NLM_string-name">Cumming, J.</span>; <span class="NLM_string-name">Faull, A. W.</span>; <span class="NLM_string-name">Larsson, J.</span>; <span class="NLM_string-name">Nash, I. A.</span>; <span class="NLM_string-name">Oldham, K.</span>; <span class="NLM_string-name">Pape, A.</span></span> <span> </span><span class="NLM_article-title">4H-[1, 2, 4] Triazolo [5, 1-b] Pyrimidin-7-one Derivatives As CCR2b Receptor Antagonists</span>. Patent <span class="NLM_patent">WO 2011/114148 A1</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=B.+A.+Bengtsson&author=W.+Blackaby&author=J.+Cumming&author=A.+W.+Faull&author=J.+Larsson&author=I.+A.+Nash&author=K.+Oldham&author=A.+Pape&title=4H-%5B1%2C+2%2C+4%5D+Triazolo+%5B5%2C+1-b%5D+Pyrimidin-7-one+Derivatives+As+CCR2b+Receptor+Antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBengtsson%26aufirst%3DB.%2BA.%26atitle%3D4H-%255B1%252C%25202%252C%25204%255D%2520Triazolo%2520%255B5%252C%25201-b%255D%2520Pyrimidin-7-one%2520Derivatives%2520As%2520CCR2b%2520Receptor%2520Antagonists%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, J. W.</span>; <span class="NLM_string-name">Meo, P.</span>; <span class="NLM_string-name">Higginbottom, M.</span>; <span class="NLM_string-name">Simpson, I.</span>; <span class="NLM_string-name">Mountford, D.</span>; <span class="NLM_string-name">Savory, E. D.</span></span> <span> </span><span class="NLM_article-title">7-Hydroxy-Pyrazolo [1,5-a] Pyrimidine Compounds And Their Use As CCR2 Receptor Antagonists</span>. Patent <span class="NLM_patent">WO 2012/041817 A1</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=J.+W.+Boyd&author=P.+Meo&author=M.+Higginbottom&author=I.+Simpson&author=D.+Mountford&author=E.+D.+Savory&title=7-Hydroxy-Pyrazolo+%5B1%2C5-a%5D+Pyrimidine+Compounds+And+Their+Use+As+CCR2+Receptor+Antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBoyd%26aufirst%3DJ.%2BW.%26atitle%3D7-Hydroxy-Pyrazolo%2520%255B1%252C5-a%255D%2520Pyrimidine%2520Compounds%2520And%2520Their%2520Use%2520As%2520CCR2%2520Receptor%2520Antagonists%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordalski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pin, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vriend, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">Generic GPCR residue numbers – aligning topology maps while minding the gaps</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.tips.2014.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25541108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSgu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=22-31&issue=1&author=V.+Isbergauthor=C.+de+Graafauthor=A.+Bortolatoauthor=V.+Cherezovauthor=V.+Katritchauthor=F.+H.+Marshallauthor=S.+Mordalskiauthor=J.-P.+Pinauthor=R.+C.+Stevensauthor=G.+Vriendauthor=D.+E.+Gloriam&title=Generic+GPCR+residue+numbers+%E2%80%93+aligning+topology+maps+while+minding+the+gaps&doi=10.1016%2Fj.tips.2014.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Generic GPCR residue numbers - aligning topology maps while minding the gaps</span></div><div class="casAuthors">Isberg, Vignir; de Graaf, Chris; Bortolato, Andrea; Cherezov, Vadim; Katritch, Vsevolod; Marshall, Fiona H.; Mordalski, Stefan; Pin, Jean-Philippe; Stevens, Raymond C.; Vriend, Gerrit; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-31</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Generic residue nos. facilitate comparisons of, for example, mutational effects, ligand interactions, and structural motifs.  The numbering scheme by Ballesteros and Weinstein for residues within the class A GPCRs (G protein-coupled receptors) has more than 1100 citations, and the recent crystal structures for classes B, C, and F now call for a community consensus in residue numbering within and across these classes.  Furthermore, the structural era has uncovered helix bulges and constrictions that offset the generic residue nos.  The use of generic residue nos. depends on convenient access by pharmacologists, chemists, and structural biologists.  We review the generic residue numbering schemes for each GPCR class, as well as a complementary structure-based scheme, and provide illustrative examples and GPCR database (GPCRDB) web tools to no. any receptor sequence or structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLIeBaLTmUebVg90H21EOLACvtfcHk0ljIRuZSiYsijQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVSgu7rK&md5=31e18cc8196fb05a12d04dd0402b2338</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DIsberg%26aufirst%3DV.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DPin%26aufirst%3DJ.-P.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DVriend%26aufirst%3DG.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DGeneric%2520GPCR%2520residue%2520numbers%2520%25E2%2580%2593%2520aligning%2520topology%2520maps%2520while%2520minding%2520the%2520gaps%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26issue%3D1%26spage%3D22%26epage%3D31%26doi%3D10.1016%2Fj.tips.2014.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanzaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiraishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aramaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okonogi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meguro, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujino, M.</span></span> <span> </span><span class="NLM_article-title">A small-molecule, nonpeptide CCR5 antagonist with highly potent and selective anti-HIV-1 activity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5698</span>– <span class="NLM_lpage">5703</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.10.5698</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1073%2Fpnas.96.10.5698" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10318947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFCntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=5698-5703&issue=10&author=M.+Babaauthor=O.+Nishimuraauthor=N.+Kanzakiauthor=M.+Okamotoauthor=H.+Sawadaauthor=Y.+Iizawaauthor=M.+Shiraishiauthor=Y.+Aramakiauthor=K.+Okonogiauthor=Y.+Ogawaauthor=K.+Meguroauthor=M.+Fujino&title=A+small-molecule%2C+nonpeptide+CCR5+antagonist+with+highly+potent+and+selective+anti-HIV-1+activity&doi=10.1073%2Fpnas.96.10.5698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule, nonpeptide CCR5 antagonist with highly potent and selective anti-HIV-1 activity</span></div><div class="casAuthors">Baba, Masanori; Nishimura, Osamu; Kanzaki, Naoyuki; Okamoto, Mika; Sawada, Hidekazu; Iizawa, Yuji; Shiraishi, Mitsuru; Aramaki, Yoshio; Okonogi, Kenji; Ogawa, Yasuaki; Meguro, Kanji; Fujino, Masahiko</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5698-5703</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The β-chemokine receptor CCR5 is considered to be an attractive target for inhibition of macrophagetropic (CCR5-using or R5) HIV-1 replication because individuals having a nonfunctional receptor (a homozygous 32-bp deletion in the CCR5 coding region) are apparently normal but resistant to infection with R5 HIV-1.  In this study, we found that TAK-779, a nonpeptide compd. with a small mol. wt. (Mr 531.13), antagonized the binding of RANTES (regulated on activation, normal T cell expressed and secreted) to CCR5-expressing Chinese hamster ovary cells and blocked CCR5-mediated Ca2+ signaling at nanomolar concns.  The inhibition of β-chemokine receptors by TAK-779 appeared to be specific to CCR5 because the compd. antagonized CCR2b to a lesser extent but did not affect CCR1, CCR3, or CCR4.  Consequently, TAK-779 displayed highly potent and selective inhibition of R5 HIV-1 replication without showing any cytotoxicity to the host cells.  The compd. inhibited the replication of R5 HIV-1 clin. isolates as well as a lab. strain at a concn. of 1.6-3.7 nM in peripheral blood mononuclear cells, though it was totally inactive against T-cell line-tropic (CXCR4-using or X4) HIV-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJEftBToEjV7Vg90H21EOLACvtfcHk0ljU_7-T2XD0yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFCntb4%253D&md5=9af3dd556be298aa9995e601725f49aa</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.10.5698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.10.5698%26sid%3Dliteratum%253Aachs%26aulast%3DBaba%26aufirst%3DM.%26aulast%3DNishimura%26aufirst%3DO.%26aulast%3DKanzaki%26aufirst%3DN.%26aulast%3DOkamoto%26aufirst%3DM.%26aulast%3DSawada%26aufirst%3DH.%26aulast%3DIizawa%26aufirst%3DY.%26aulast%3DShiraishi%26aufirst%3DM.%26aulast%3DAramaki%26aufirst%3DY.%26aulast%3DOkonogi%26aufirst%3DK.%26aulast%3DOgawa%26aufirst%3DY.%26aulast%3DMeguro%26aufirst%3DK.%26aulast%3DFujino%26aufirst%3DM.%26atitle%3DA%2520small-molecule%252C%2520nonpeptide%2520CCR5%2520antagonist%2520with%2520highly%2520potent%2520and%2520selective%2520anti-HIV-1%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26issue%3D10%26spage%3D5698%26epage%3D5703%26doi%3D10.1073%2Fpnas.96.10.5698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokornaczyk, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Maria, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz Zacarias, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunsch, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of chemokine receptor ligands with 2-benzazepine scaffold</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.ejmech.2017.04.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28463783" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFSktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2017&pages=401-413&author=S.+Thumauthor=A.+K.+Kokornaczykauthor=T.+Sekiauthor=M.+De+Mariaauthor=N.+V.+Ortiz+Zacariasauthor=H.+de+Vriesauthor=C.+Weissauthor=M.+Kochauthor=D.+Schepmannauthor=M.+Kitamuraauthor=N.+Tschammerauthor=L.+H.+Heitmanauthor=A.+Junkerauthor=B.+Wunsch&title=Synthesis+and+biological+evaluation+of+chemokine+receptor+ligands+with+2-benzazepine+scaffold&doi=10.1016%2Fj.ejmech.2017.04.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of chemokine receptor ligands with 2-benzazepine scaffold</span></div><div class="casAuthors">Thum, Simone; Kokornaczyk, Artur K.; Seki, Tomoaki; De Maria, Monica; Ortiz Zacarias, Natalia V.; de Vries, Henk; Weiss, Christina; Koch, Michael; Schepmann, Dirk; Kitamura, Masato; Tschammer, Nuska; Heitman, Laura H.; Junker, Anna; Wuensch, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">401-413</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The development of the first probe-dependent pos. allosteric modulator (PAM) of CCR5 receptors I [R = H, 8-NO2, 8-NHAc, etc.] and II with a 2-benzazepine scaffold was presented.  Compd. I [R = H] activated the CCR5 receptor in a CCL4-dependent manner, but did not compete with [3H]TAK-779 binding at the CCR5.  Furthermore, introduction of a p-tolyl moiety at 7-position of the 2-benzazepine scaffold turns the CCR5 PAM I [R = H] into the selective CCR2 receptor antagonist I [R = 7-p-tolyl].  The structure affinity and activity relationships presented here offered new insights into ligand recognition by CCR2 and CCR5 receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOKR_X2Kf9dLVg90H21EOLACvtfcHk0ljU_7-T2XD0yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFSktLs%253D&md5=789b738eddd46aecb8cd455f21b1f73d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.046%26sid%3Dliteratum%253Aachs%26aulast%3DThum%26aufirst%3DS.%26aulast%3DKokornaczyk%26aufirst%3DA.%2BK.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DDe%2BMaria%26aufirst%3DM.%26aulast%3DOrtiz%2BZacarias%26aufirst%3DN.%2BV.%26aulast%3Dde%2BVries%26aufirst%3DH.%26aulast%3DWeiss%26aufirst%3DC.%26aulast%3DKoch%26aufirst%3DM.%26aulast%3DSchepmann%26aufirst%3DD.%26aulast%3DKitamura%26aufirst%3DM.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DJunker%26aufirst%3DA.%26aulast%3DWunsch%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520chemokine%2520receptor%2520ligands%2520with%25202-benzazepine%2520scaffold%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D135%26spage%3D401%26epage%3D413%26doi%3D10.1016%2Fj.ejmech.2017.04.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corbisier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huszagh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmentier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springael, J. Y.</span></span> <span> </span><span class="NLM_article-title">Biased signaling at chemokine receptors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">9542</span>– <span class="NLM_lpage">9554</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.596098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1074%2Fjbc.M114.596098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25614627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Ohs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=9542-9554&issue=15&author=J.+Corbisierauthor=C.+Galesauthor=A.+Huszaghauthor=M.+Parmentierauthor=J.+Y.+Springael&title=Biased+signaling+at+chemokine+receptors&doi=10.1074%2Fjbc.M114.596098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Signaling at Chemokine Receptors</span></div><div class="casAuthors">Corbisier, Jenny; Gales, Celine; Huszagh, Alexandre; Parmentier, Marc; Springael, Jean-Yves</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">9542-9554</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The ability of G protein-coupled receptors (GPCRs) to activate selective signaling pathways according to the conformation stabilized by bound ligands (signaling bias) is a challenging concept in the GPCR field.  Signaling bias has been documented for several GPCRs, including chemokine receptors.  However, most of these studies examd. the global signaling bias between G protein- and arrestin-dependent pathways, leaving unaddressed the potential bias between particular G protein subtypes.  Here, the authors investigated the coupling selectivity of chemokine receptors CCR2, CCR5, and CCR7 in response to various ligands with G protein subtypes by using bioluminescence resonance energy transfer biosensors monitoring directly the activation of G proteins.  The authors also compared data obtained with the G protein biosensors with those obtained with other functional readouts, such as β-arrestin-2 recruitment, cAMP accumulation, and calcium mobilization assays.  The authors showed that the binding of chemokines to CCR2, CCR5, and CCR7 activated the three Gαi subtypes (Gαi1, Gαi2, and Gαi3) and the two Gαo isoforms (Gαoa and Gαob) with potencies that generally correlate to their binding affinities.  In addn., the authors showed that the binding of chemokines to CCR5 and CCR2 also activated Gα12, but not Gα13.  For each receptor, the authors showed that the relative potency of various agonist chemokines was not identical in all assays, supporting the notion that signaling bias exists at chemokine receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq627lB4lvYh7Vg90H21EOLACvtfcHk0liCZFwP8OLBhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Ohs7c%253D&md5=fc598a1428f9a9b1d191c6fa34ab81dd</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.596098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.596098%26sid%3Dliteratum%253Aachs%26aulast%3DCorbisier%26aufirst%3DJ.%26aulast%3DGales%26aufirst%3DC.%26aulast%3DHuszagh%26aufirst%3DA.%26aulast%3DParmentier%26aufirst%3DM.%26aulast%3DSpringael%26aufirst%3DJ.%2BY.%26atitle%3DBiased%2520signaling%2520at%2520chemokine%2520receptors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26issue%3D15%26spage%3D9542%26epage%3D9554%26doi%3D10.1074%2Fjbc.M114.596098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, H.</span></span> <span> </span><span class="NLM_article-title">Hill coefficients, dose-response curves and allosteric mechanisms</span>. <i>J. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1007/s12154-009-0029-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1007%2Fs12154-009-0029-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=19779939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A280%3ADC%252BC38zitlSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=37-44&issue=1&author=H.+Prinz&title=Hill+coefficients%2C+dose-response+curves+and+allosteric+mechanisms&doi=10.1007%2Fs12154-009-0029-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Hill coefficients, dose-response curves and allosteric mechanisms</span></div><div class="casAuthors">Prinz Heino</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-44</span>
        ISSN:<span class="NLM_cas:issn">1864-6158</span>.
    </div><div class="casAbstract">Hill coefficients (n(H)) derived from four parameter logistic fits to dose-response curves were compared to calculated realistic reaction schemes and related to experimental data: (1) Hill coefficients may give information on the number of interacting sites but cannot distinguish between competitive, non-competitive or ortho-, iso-, or allosteric mechanisms. (2) For enzymatic dose-inhibition curves, Hill coefficients smaller than one do not indicate anticooperative binding but show that at least one ternary complex has enzymatic activity. (3) Hill coefficients different from one are proof for multiple ligand binding.  The large variations of reported Hill coefficients corresponds to multiple allosteric binding, where induced conformational changes cause loss of the active conformation.  Such a denaturation mechanism is in stark contrast to the desired specificity of drugs.  The discussion is open.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTaD6e5pnko8WGmni0r9GoKfW6udTcc2eY_NzAcB9Q72Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38zitlSqsg%253D%253D&md5=5882ac14ec82a4f7b0f5a4eae02c9410</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs12154-009-0029-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12154-009-0029-3%26sid%3Dliteratum%253Aachs%26aulast%3DPrinz%26aufirst%3DH.%26atitle%3DHill%2520coefficients%252C%2520dose-response%2520curves%2520and%2520allosteric%2520mechanisms%26jtitle%3DJ.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D3%26issue%3D1%26spage%3D37%26epage%3D44%26doi%3D10.1007%2Fs12154-009-0029-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collart, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grassi, J.</span></span> <span> </span><span class="NLM_article-title">Novel, acidic CCR2 receptor antagonists: from hit to lead</span>. <i>Lett. Drug. Des. Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.2174/157018007780077381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.2174%2F157018007780077381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsF2qtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2007&pages=185-191&issue=3&author=D.+Zouauthor=H.-X.+Zhaiauthor=J.+Eckmanauthor=P.+Higginsauthor=M.+Gillardauthor=L.+Knerrauthor=S.+Carreauthor=P.+Pasauauthor=P.+Collartauthor=J.+Grassi&title=Novel%2C+acidic+CCR2+receptor+antagonists%3A+from+hit+to+lead&doi=10.2174%2F157018007780077381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Novel, acidic CCR2 receptor antagonists: from hit to lead</span></div><div class="casAuthors">Zou, D.; Zhai, H.-X.; Eckman, J.; Higgins, P.; Gillard, M.; Knerr, L.; Carre, S.; Pasau, P.; Collart, P.; Grassi, J.; Libertine, L.; Nicolas, J.-M.; Schwartz, C. E.</div><div class="citationInfo"><span class="NLM_cas:title">Letters in Drug Design & Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">185-191</span>CODEN:
                <span class="NLM_cas:coden">LDDDAW</span>;
        ISSN:<span class="NLM_cas:issn">1570-1808</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">High throughput screening identified a pyrrolidinone contg. acidic functionality as a CCR2 antagonist of modest affinity.  We describe initial SAR around this hit, including soln. phase parallel synthesis for analog prepn., and we describe identification and characterization of an analog subsequently selected as a viable lead mol. for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqICdOhOTrW0bVg90H21EOLACvtfcHk0lgL2ZDuHMDukg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsF2qtb8%253D&md5=6c04d2c6b5daf6767966ff35737649d9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.2174%2F157018007780077381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157018007780077381%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DD.%26aulast%3DZhai%26aufirst%3DH.-X.%26aulast%3DEckman%26aufirst%3DJ.%26aulast%3DHiggins%26aufirst%3DP.%26aulast%3DGillard%26aufirst%3DM.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DCarre%26aufirst%3DS.%26aulast%3DPasau%26aufirst%3DP.%26aulast%3DCollart%26aufirst%3DP.%26aulast%3DGrassi%26aufirst%3DJ.%26atitle%3DNovel%252C%2520acidic%2520CCR2%2520receptor%2520antagonists%253A%2520from%2520hit%2520to%2520lead%26jtitle%3DLett.%2520Drug.%2520Des.%2520Discovery%26date%3D2007%26volume%3D4%26issue%3D3%26spage%3D185%26epage%3D191%26doi%3D10.2174%2F157018007780077381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ágoston, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pongor, S.</span></span> <span> </span><span class="NLM_article-title">The efficiency of multi-target drugs: the network approach might help drug design</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">178</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2005.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.tips.2005.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=15808341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivV2ht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=178-182&issue=4&author=P.+Csermelyauthor=V.+%C3%81gostonauthor=S.+Pongor&title=The+efficiency+of+multi-target+drugs%3A+the+network+approach+might+help+drug+design&doi=10.1016%2Fj.tips.2005.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The efficiency of multi-target drugs: the network approach might help drug design</span></div><div class="casAuthors">Csermely, Peter; Agoston, Vilmos; Pongor, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">178-182</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite considerable progress in genome- and proteome-based high-throughput screening methods and rational drug design, the no. of successful single-target drugs did not increase appreciably during the past decade.  Network models suggest that partial inhibition of a surprisingly small no. of targets can be more efficient than the complete inhibition of a single target.  This and the success stories of multi-target drugs and combinatorial therapies led us to suggest that systematic drug-design strategies should be directed against multiple targets.  The authors propose that the final effect of partial, but multiple, drug actions might often surpass that of complete drug action at a single target.  The future success of this novel drug-design paradigm will depend not only on a new generation of computer models to identify the correct multiple targets and their multi-fitting, low-affinity drug candidates but also on more-efficient in vivo testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQRoFMVKb5m7Vg90H21EOLACvtfcHk0lgL2ZDuHMDukg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivV2ht70%253D&md5=4f1383f4699221ef99041dffafe03e8c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2005.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2005.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3D%25C3%2581goston%26aufirst%3DV.%26aulast%3DPongor%26aufirst%3DS.%26atitle%3DThe%2520efficiency%2520of%2520multi-target%2520drugs%253A%2520the%2520network%2520approach%2520might%2520help%2520drug%2520design%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2005%26volume%3D26%26issue%3D4%26spage%3D178%26epage%3D182%26doi%3D10.1016%2Fj.tips.2005.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, J. R.</span></span> <span> </span><span class="NLM_article-title">The Challenges of Multi-Target Lead Optimization</span>. In  <i>Designing Multi-Target Drugs</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, J. R.</span>, <span class="NLM_string-name">Harris, C. J.</span></span>, Eds.; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">154</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1039%2F9781849734912-00141" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=141-154&author=J.+R.+Morphyauthor=J.+R.+Morphy&author=C.+J.+Harris&title=Designing+Multi-Target+Drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1039%2F9781849734912-00141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849734912-00141%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520Challenges%2520of%2520Multi-Target%2520Lead%2520Optimization%26btitle%3DDesigning%2520Multi-Target%2520Drugs%26aulast%3DMorphy%26aufirst%3DJ.%2BR.%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2012%26spage%3D141%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buntinx, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moechars, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilissen, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckx, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coesemans, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lommen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Wauwe, J. P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological profile of JNJ-27141491 [(S)-3-[3, 4-difluorophenyl)-propyl]-5-isoxazol-5-yl-2-thioxo-2, 3-dihydro-1H-imidazole-4-carboxyl acid methyl ester], as a noncompetitive and orally active antagonist of the human chemokine receptor CCR2</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>327</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1124/jpet.108.140723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fjpet.108.140723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18599682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1amsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2008&pages=1-9&issue=1&author=M.+Buntinxauthor=B.+Hermansauthor=J.+Goossensauthor=D.+Moecharsauthor=R.+A.+Gilissenauthor=J.+Doyonauthor=S.+Boeckxauthor=E.+Coesemansauthor=G.+Van+Lommenauthor=J.+P.+Van+Wauwe&title=Pharmacological+profile+of+JNJ-27141491+%5B%28S%29-3-%5B3%2C+4-difluorophenyl%29-propyl%5D-5-isoxazol-5-yl-2-thioxo-2%2C+3-dihydro-1H-imidazole-4-carboxyl+acid+methyl+ester%5D%2C+as+a+noncompetitive+and+orally+active+antagonist+of+the+human+chemokine+receptor+CCR2&doi=10.1124%2Fjpet.108.140723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological profile of JNJ-27141491 [(S)-3-[(3,4-difluorophenyl)propyl]-5-isoxazol-5-yl-2-thioxo-2,3-dihydro-1H-imidazole-4-carboxyl acid methyl ester], as a noncompetitive and orally active antagonist of the human chemokine receptor CCR2</span></div><div class="casAuthors">Buntinx, Mieke; Hermans, Bart; Goossens, Jan; Moechars, Dieder; Gilissen, Ron A. H. J.; Doyon, Julien; Boeckx, Staf; Coesemans, Erwin; Van Lommen, Guy; Van Wauwe, Jean P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The interaction between CC chemokine receptor 2 (CCR2) with monocyte chemoattractant proteins, such as MCP-1, regulates the activation and recruitment of inflammatory leukocytes.  In this study, we characterized JNJ-27141491 as a noncompetitive and orally active functional antagonist of human (h)CCR2.  JNJ-27141491 strongly suppressed hCCR2-mediated in vitro functions, such as MCP-1-induced guanosine 5'-O-(3-[35S]thio)triphosphate binding; MCP-1, -3, and -4-induced Ca2+ mobilization; and leukocyte chemotaxis toward MCP-1 (IC50 = 7-97 nM), whereas it had little or no effect on the function of other chemokine receptors tested.  The inhibition of CCR2 function was both insurmountable and reversible, consistent with a noncompetitive mode of action.  JNJ-27141491 blocked the binding of 125I-MCP-1 to human monocytes (IC50 = 0.4 μM), but it failed to affect MCP-1 binding to mouse, rat, and dog cells (IC50 > 10 μM).  Therefore, transgenic mice, in which the mouse (m)CCR2 gene was replaced by the human counterpart, were generated for in vivo testing.  In these mice, oral administration of JNJ-27141491 dose-dependently [5-40 mg/kg q.d. (once daily) or b.i.d.] inhibited monocyte and neutrophil recruitment to the alveolar space 48 h after intratracheal mMCP-1/lipopolysaccharide instillation.  Furthermore, treatment with JNJ-27141491 (20 mg/kg q.d.) significantly delayed the onset and temporarily reduced neurol. signs in an exptl. autoimmune encephalomyelitis model of multiple sclerosis.  Taken together, these results identify JNJ-27141491 as a noncompetitive, functional antagonist of hCCR2, capable of exerting oral anti-inflammatory activity in transgenic hCCR2-expressing mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCvC8mYaH0rVg90H21EOLACvtfcHk0limyRlFNQzX4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1amsLjE&md5=5c4ed289656be37b72a76210b34b9eeb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.140723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.140723%26sid%3Dliteratum%253Aachs%26aulast%3DBuntinx%26aufirst%3DM.%26aulast%3DHermans%26aufirst%3DB.%26aulast%3DGoossens%26aufirst%3DJ.%26aulast%3DMoechars%26aufirst%3DD.%26aulast%3DGilissen%26aufirst%3DR.%2BA.%26aulast%3DDoyon%26aufirst%3DJ.%26aulast%3DBoeckx%26aufirst%3DS.%26aulast%3DCoesemans%26aufirst%3DE.%26aulast%3DVan%2BLommen%26aufirst%3DG.%26aulast%3DVan%2BWauwe%26aufirst%3DJ.%2BP.%26atitle%3DPharmacological%2520profile%2520of%2520JNJ-27141491%2520%255B%2528S%2529-3-%255B3%252C%25204-difluorophenyl%2529-propyl%255D-5-isoxazol-5-yl-2-thioxo-2%252C%25203-dihydro-1H-imidazole-4-carboxyl%2520acid%2520methyl%2520ester%255D%252C%2520as%2520a%2520noncompetitive%2520and%2520orally%2520active%2520antagonist%2520of%2520the%2520human%2520chemokine%2520receptor%2520CCR2%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2008%26volume%3D327%26issue%3D1%26spage%3D1%26epage%3D9%26doi%3D10.1124%2Fjpet.108.140723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczuka, A.</span></span> <span> </span><span class="NLM_article-title">Kinetic versus allosteric mechanisms to explain insurmountable antagonism and delayed ligand dissociation</span>. <i>Neurochem. Int.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">254</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/j.neuint.2007.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.neuint.2007.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=17606314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVCmt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=254-260&issue=5&author=G.+Vauquelinauthor=A.+Szczuka&title=Kinetic+versus+allosteric+mechanisms+to+explain+insurmountable+antagonism+and+delayed+ligand+dissociation&doi=10.1016%2Fj.neuint.2007.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic versus allosteric mechanisms to explain insurmountable antagonism and delayed ligand dissociation</span></div><div class="casAuthors">Vauquelin, Georges; Szczuka, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry International</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">254-260</span>CODEN:
                <span class="NLM_cas:coden">NEUIDS</span>;
        ISSN:<span class="NLM_cas:issn">0197-0186</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The present review addresses the theories which have been advanced to explain exptl. observations dealing with insurmountable antagonism and accelerated radioligand dissocn. in the presence of an excess unlabeled ligand.  We came to the perception that, for each of these phenomena, the theories can be placed into two distinctive categories.  The "kinetic" interpretations attribute these phenomena to, resp., the ability of antagonists to form long-lasting complexes with their cognate receptor and the ability of dissocd. ligands to bind again to the same or neighboring receptors rather than to diffuse away from the cell surface.  On the other hand, these observations can also be explained by neg. allosteric interactions among topog. distinct ligand binding sites at the same receptor or di/multimeric receptor complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZubKDcax4HbVg90H21EOLACvtfcHk0lg4yuyHJ93Csg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVCmt7jO&md5=49a0753fcf25ac7be190fc45514baa49</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2007.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2007.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26aulast%3DSzczuka%26aufirst%3DA.%26atitle%3DKinetic%2520versus%2520allosteric%2520mechanisms%2520to%2520explain%2520insurmountable%2520antagonism%2520and%2520delayed%2520ligand%2520dissociation%26jtitle%3DNeurochem.%2520Int.%26date%3D2007%26volume%3D51%26issue%3D5%26spage%3D254%26epage%3D260%26doi%3D10.1016%2Fj.neuint.2007.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span> <span> </span><span class="NLM_article-title">Structure of the CCR5 chemokine receptor–HIV entry inhibitor maraviroc complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6152</span>),  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1126/science.1241475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1126%2Fscience.1241475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=24030490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1387-1390&issue=6152&author=Q.+Tanauthor=Y.+Zhuauthor=J.+Liauthor=Z.+Chenauthor=G.+W.+Hanauthor=I.+Kufarevaauthor=T.+Liauthor=L.+Maauthor=G.+Fenaltiauthor=J.+Liauthor=W.+Zhangauthor=X.+Xieauthor=H.+Yangauthor=H.+Jiangauthor=V.+Cherezovauthor=H.+Liuauthor=R.+C.+Stevensauthor=Q.+Zhaoauthor=B.+Wu&title=Structure+of+the+CCR5+chemokine+receptor%E2%80%93HIV+entry+inhibitor+maraviroc+complex&doi=10.1126%2Fscience.1241475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the CCR5 Chemokine Receptor-HIV Entry Inhibitor Maraviroc Complex</span></div><div class="casAuthors">Tan, Qiuxiang; Zhu, Ya; Li, Jian; Chen, Zhuxi; Han, Gye Won; Kufareva, Irina; Li, Tingting; Ma, Limin; Fenalti, Gustavo; Li, Jing; Zhang, Wenru; Xie, Xin; Yang, Huaiyu; Jiang, Hualiang; Cherezov, Vadim; Liu, Hong; Stevens, Raymond C.; Zhao, Qiang; Wu, Beili</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6152</span>),
    <span class="NLM_cas:pages">1387-1390</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The CCR5 chemokine receptor acts as a co-receptor for HIV-1 viral entry.  Here we report the 2.7 angstrom-resoln. crystal structure of human CCR5 bound to the marketed HIV drug maraviroc.  The structure reveals a ligand-binding site that is distinct from the proposed major recognition sites for chemokines and the viral glycoprotein gp120, providing insights into the mechanism of allosteric inhibition of chemokine signaling and viral entry.  A comparison between CCR5 and CXCR4 crystal structures, along with models of co-receptor-gp120-V3 complexes, suggests that different charge distributions and steric hindrances caused by residue substitutions may be major determinants of HIV-1 co-receptor selectivity.  These high-resoln. insights into CCR5 can enable structure-based drug discovery for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt5VKeO5Hmb7Vg90H21EOLACvtfcHk0lg4yuyHJ93Csg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCmu7vJ&md5=730ee5644a2a144cd49cd0e9a15eba2e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscience.1241475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1241475%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DB.%26atitle%3DStructure%2520of%2520the%2520CCR5%2520chemokine%2520receptor%25E2%2580%2593HIV%2520entry%2520inhibitor%2520maraviroc%2520complex%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6152%26spage%3D1387%26epage%3D1390%26doi%3D10.1126%2Fscience.1241475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilcken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckler, F. M.</span></span> <span> </span><span class="NLM_article-title">Principles and applications of halogen bonding in medicinal chemistry and chemical biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1363</span>– <span class="NLM_lpage">1388</span>, <span class="refDoi"> DOI: 10.1021/jm3012068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3012068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1363-1388&issue=4&author=R.+Wilckenauthor=M.+O.+Zimmermannauthor=A.+Langeauthor=A.+C.+Joergerauthor=F.+M.+Boeckler&title=Principles+and+applications+of+halogen+bonding+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Fjm3012068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Wilcken, Rainer; Zimmermann, Markus O.; Lange, Andreas; Joerger, Andreas C.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1363-1388</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Halogen bonding has been known in material science for decades, but until recently, halogen bonds in protein-ligand interactions were largely the result of serendipitous discovery rather than rational design.  In this Perspective, we provide insights into the phenomenon of halogen bonding, with special focus on its role in drug discovery.  We summarize the theor. background defining its strength and directionality, provide a systematic anal. of its occurrence and interaction geometries in protein-ligand complexes, and give recent examples where halogen bonding has been successfully harnessed for lead identification and optimization.  In light of these data, we discuss the potential and limitations of exploiting halogen bonds for mol. recognition and rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjholHfNZ9kbVg90H21EOLACvtfcHk0lhY-woxMvXLOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ&md5=81ba024e4427fac3e7af0a4cd330909c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm3012068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3012068%26sid%3Dliteratum%253Aachs%26aulast%3DWilcken%26aufirst%3DR.%26aulast%3DZimmermann%26aufirst%3DM.%2BO.%26aulast%3DLange%26aufirst%3DA.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPrinciples%2520and%2520applications%2520of%2520halogen%2520bonding%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D4%26spage%3D1363%26epage%3D1388%26doi%3D10.1021%2Fjm3012068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salchow, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Press, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, M. E.</span></span> <span> </span><span class="NLM_article-title">A common intracellular allosteric binding site for antagonists of the CXCR2 receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>159</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00623.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1111%2Fj.1476-5381.2009.00623.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=20233217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3cXks1Ojt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2010&pages=1429-1439&issue=7&author=K.+Salchowauthor=M.+E.+Bondauthor=S.+C.+Evansauthor=N.+J.+Pressauthor=S.+J.+Charltonauthor=P.+A.+Huntauthor=M.+E.+Bradley&title=A+common+intracellular+allosteric+binding+site+for+antagonists+of+the+CXCR2+receptor&doi=10.1111%2Fj.1476-5381.2009.00623.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A common intracellular allosteric binding site for antagonists of the CXCR2 receptor</span></div><div class="casAuthors">Salchow, K.; Bond, M. E.; Evans, S. C.; Press, N. J.; Charlton, S. J.; Hunt, P. A.; Bradley, M. E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1429-1439</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and purpose: We have previously shown that SB265610 (1-(2-bromo-phenyl)-3-(7-cyano-3H-benzotriazol-4-yl)-urea) behaves as an allosteric, inverse agonist at the C-X-C chemokine (CXCR)2 receptor.  The aim of this study was to det. whether SB265610, in addn. to two other known antagonists, bind to either of the two putative, topog. distinct, allosteric binding sites previously reported in the Literature.  Exptl. approach: Ten single point mutations were introduced into the CXCR2 receptor using site-directed mutagenesis.  Three CXCR2 antagonists were investigated, SB265610, Pteridone-1 (2-(2,3 difluoro-benzylsulphanyl)-4-((R)-2-hydroxy-1-methyl-ethylamino)-8H-pteridin-7-one) and Sch527123 (2-hydroxy-N,N-dimethyl-3-{2-[[(R)-1-(5-methyl-furan-2-yl)-propyl]amino]-3,4-dioxo-cyclobut-1enylamino}-benzamide), and the effect of these mutations on their binding affinity and ability to inhibit interleukin-8-stimulated binding of [35S]GTPγS was examd.  Key results: Seven of the nine mutations introduced into the C-terminal domain and intracellular loops of the receptor produced a significant redn. in affinity at least one of the antagonists tested.  Of those seven mutations, three produced a significant redn. in the affinity of all three antagonists, namely K320A, Y314A and D84N.  In all but one mutation, the changes obsd. on antagonist affinity were matched with effects on inhibition of interleukin-8-stimulated [35S]GTPγS binding.  Conclusions and implications: These antagonists bind to a common intracellular, allosteric, binding site of the CXCR2 receptor, which has been further delineated.  As many of these mutations are close to the site of G protein coupling or to a region of the receptor that is responsible for the transduction of the activation signal, our results suggest a mol. mechanism for the inhibition of receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOiWNq0sDt17Vg90H21EOLACvtfcHk0liDtA4eAtc6TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXks1Ojt7Y%253D&md5=f1d3541316672373f9aebe8c9cc79edd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00623.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00623.x%26sid%3Dliteratum%253Aachs%26aulast%3DSalchow%26aufirst%3DK.%26aulast%3DBond%26aufirst%3DM.%2BE.%26aulast%3DEvans%26aufirst%3DS.%2BC.%26aulast%3DPress%26aufirst%3DN.%2BJ.%26aulast%3DCharlton%26aufirst%3DS.%2BJ.%26aulast%3DHunt%26aufirst%3DP.%2BA.%26aulast%3DBradley%26aufirst%3DM.%2BE.%26atitle%3DA%2520common%2520intracellular%2520allosteric%2520binding%2520site%2520for%2520antagonists%2520of%2520the%2520CXCR2%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2010%26volume%3D159%26issue%3D7%26spage%3D1429%26epage%3D1439%26doi%3D10.1111%2Fj.1476-5381.2009.00623.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brammall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grahames, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meghani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallinder, P. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a putative intracellular allosteric antagonist binding-site in the CXC chemokine receptors 1 and 2</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>74</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi"> DOI: 10.1124/mol.107.044610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1124%2Fmol.107.044610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18676678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlCntrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2008&pages=1193-1202&issue=5&author=D.+J.+Nichollsauthor=N.+P.+Tomkinsonauthor=K.+E.+Wileyauthor=A.+Brammallauthor=L.+Bowersauthor=C.+Grahamesauthor=A.+Gawauthor=P.+Meghaniauthor=P.+Sheltonauthor=T.+J.+Wrightauthor=P.+R.+Mallinder&title=Identification+of+a+putative+intracellular+allosteric+antagonist+binding-site+in+the+CXC+chemokine+receptors+1+and+2&doi=10.1124%2Fmol.107.044610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a putative intracellular allosteric antagonist binding-site in the CXC chemokine receptors 1 and 2</span></div><div class="casAuthors">Nicholls, David J.; Tomkinson, Nick P.; Wiley, Katherine E.; Brammall, Anne; Bowers, Lorna; Grahames, Caroline; Gaw, Alasdair; Meghani, Premji; Shelton, Philip; Wright, Tracey J.; Mallinder, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1193-1202</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptors CXCR1 and CXCR2 are G-protein-coupled receptors (GPCRs) implicated in mediating cellular functions assocd. with the inflammatory response.  Potent CXCR2 receptor antagonists have been discovered, some of which have recently entered clin. development.  The aim of this study was to identify key amino acid residue differences between CXCR1 and CXCR2 that influence the relative antagonism by two compds. that have markedly different chem. structures.  By investigating the effects of domain switching and point mutations, we found that the second extracellular loop, which contained significant amino acid sequence diversity, was not important for compd. antagonism.  We were surprised to find that switching the intracellular C-terminal 60 amino acid domains of CXCR1 and CXCR2 caused an apparent reversal of antagonism at these two receptors.  Further investigation showed that a single amino acid residue, lysine 320 in CXCR2 and asparagine 311 in CXCR1, plays a predominant role in describing the relative antagonism of the two compds.  Homol. modeling studies based on the structure of bovine rhodopsin indicated a potential intracellular antagonist binding pocket involving lysine 320.  We conclude that residue 320 in CXCR2 forms part of a potential allosteric binding pocket on the intracellular side of the receptor, a site that is distal to the orthosteric site commonly assumed to be the location of antagonist binding to GPCRs.  The existence of a common intracellular allosteric binding site at GPCRs related to CXCR2 may be of value in the design of novel antagonists for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE2keoFJBVXrVg90H21EOLACvtfcHk0liDtA4eAtc6TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlCntrbJ&md5=1af3dc48be040ab10efb3b62213b4b0b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1124%2Fmol.107.044610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.107.044610%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DTomkinson%26aufirst%3DN.%2BP.%26aulast%3DWiley%26aufirst%3DK.%2BE.%26aulast%3DBrammall%26aufirst%3DA.%26aulast%3DBowers%26aufirst%3DL.%26aulast%3DGrahames%26aufirst%3DC.%26aulast%3DGaw%26aufirst%3DA.%26aulast%3DMeghani%26aufirst%3DP.%26aulast%3DShelton%26aufirst%3DP.%26aulast%3DWright%26aufirst%3DT.%2BJ.%26aulast%3DMallinder%26aufirst%3DP.%2BR.%26atitle%3DIdentification%2520of%2520a%2520putative%2520intracellular%2520allosteric%2520antagonist%2520binding-site%2520in%2520the%2520CXC%2520chemokine%2520receptors%25201%2520and%25202%26jtitle%3DMol.%2520Pharmacol.%26date%3D2008%26volume%3D74%26issue%3D5%26spage%3D1193%26epage%3D1202%26doi%3D10.1124%2Fmol.107.044610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Draper-Joyce, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoshouei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, A. T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furness, S. G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venugopal, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltos, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plitzko, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danev, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glukhova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Structure of the adenosine-bound human adenosine A1 receptor–Gi complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0236-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fs41586-018-0236-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29925945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=558&publication_year=2018&pages=559-563&issue=7711&author=C.+J.+Draper-Joyceauthor=M.+Khoshoueiauthor=D.+M.+Thalauthor=Y.-L.+Liangauthor=A.+T.+N.+Nguyenauthor=S.+G.+B.+Furnessauthor=H.+Venugopalauthor=J.-A.+Baltosauthor=J.+M.+Plitzkoauthor=R.+Danevauthor=W.+Baumeisterauthor=L.+T.+Mayauthor=D.+Woottenauthor=P.+M.+Sextonauthor=A.+Glukhovaauthor=A.+Christopoulos&title=Structure+of+the+adenosine-bound+human+adenosine+A1+receptor%E2%80%93Gi+complex&doi=10.1038%2Fs41586-018-0236-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the adenosine-bound human adenosine A1 receptor-Gi complex</span></div><div class="casAuthors">Draper-Joyce, Christopher J.; Khoshouei, Maryam; Thal, David M.; Liang, Yi-Lynn; Nguyen, Anh T. N.; Furness, Sebastian G. B.; Venugopal, Hariprasad; Baltos, Jo-Anne; Plitzko, Jurgen M.; Danev, Radostin; Baumeister, Wolfgang; May, Lauren T.; Wootten, Denise; Sexton, Patrick M.; Glukhova, Alisa; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">559-563</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The class A adenosine A1 receptor (A1R) is a G-protein-coupled receptor that preferentially couples to inhibitory Gi/o heterotrimeric G proteins, has been implicated in numerous diseases, yet remains poorly targeted.  Here we report the 3.6 Å structure of the human A1R in complex with adenosine and heterotrimeric Gi2 protein detd. by Volta phase plate cryo-electron microscopy.  Compared to inactive A1R, there is contraction at the extracellular surface in the orthosteric binding site mediated via movement of transmembrane domains 1 and 2.  At the intracellular surface, the G protein engages the A1R primarily via amino acids in the C terminus of the Gαi α5-helix, concomitant with a 10.5 Å outward movement of the A1R transmembrane domain 6.  Comparison with the agonist-bound β2 adrenergic receptor-Gs-protein complex reveals distinct orientations for each G-protein subtype upon engagement with its receptor.  This active A1R structure provides mol. insights into receptor and G-protein selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP8kpMZ0qB6rVg90H21EOLACvtfcHk0lgBT9sRed-aeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL3P&md5=f31a757364c3ff512bd5548b31e7f1bd</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0236-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0236-6%26sid%3Dliteratum%253Aachs%26aulast%3DDraper-Joyce%26aufirst%3DC.%2BJ.%26aulast%3DKhoshouei%26aufirst%3DM.%26aulast%3DThal%26aufirst%3DD.%2BM.%26aulast%3DLiang%26aufirst%3DY.-L.%26aulast%3DNguyen%26aufirst%3DA.%2BT.%2BN.%26aulast%3DFurness%26aufirst%3DS.%2BG.%2BB.%26aulast%3DVenugopal%26aufirst%3DH.%26aulast%3DBaltos%26aufirst%3DJ.-A.%26aulast%3DPlitzko%26aufirst%3DJ.%2BM.%26aulast%3DDanev%26aufirst%3DR.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DMay%26aufirst%3DL.%2BT.%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DGlukhova%26aufirst%3DA.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DStructure%2520of%2520the%2520adenosine-bound%2520human%2520adenosine%2520A1%2520receptor%25E2%2580%2593Gi%2520complex%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D559%26epage%3D563%26doi%3D10.1038%2Fs41586-018-0236-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koehl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paggi, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latorraca, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matile, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schertler, G. F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skiniotis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure of the μ-opioid receptor–Gi protein complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0219-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fs41586-018-0219-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29899455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=558&publication_year=2018&pages=547-552&issue=7711&author=A.+Koehlauthor=H.+Huauthor=S.+Maedaauthor=Y.+Zhangauthor=Q.+Quauthor=J.+M.+Paggiauthor=N.+R.+Latorracaauthor=D.+Hilgerauthor=R.+Dawsonauthor=H.+Matileauthor=G.+F.+X.+Schertlerauthor=S.+Granierauthor=W.+I.+Weisauthor=R.+O.+Drorauthor=A.+Manglikauthor=G.+Skiniotisauthor=B.+K.+Kobilka&title=Structure+of+the+%CE%BC-opioid+receptor%E2%80%93Gi+protein+complex&doi=10.1038%2Fs41586-018-0219-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the μ-opioid receptor-Gi protein complex</span></div><div class="casAuthors">Koehl, Antoine; Hu, Hongli; Maeda, Shoji; Zhang, Yan; Qu, Qianhui; Paggi, Joseph M.; Latorraca, Naomi R.; Hilger, Daniel; Dawson, Roger; Matile, Hugues; Schertler, Gebhard F. X.; Granier, Sebastien; Weis, William I.; Dror, Ron O.; Manglik, Aashish; Skiniotis, Georgios; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">547-552</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The μ-opioid receptor (μOR) is a G-protein-coupled receptor (GPCR) and the target of most clin. and recreationally used opioids.  The induced pos. effects of analgesia and euphoria are mediated by μOR signalling through the adenylyl cyclase-inhibiting heterotrimeric G protein Gi.  Here we present the 3.5 Å resoln. cryo-electron microscopy structure of the μOR bound to the agonist peptide DAMGO and nucleotide-free Gi.  DAMGO occupies the morphinan ligand pocket, with its N terminus interacting with conserved receptor residues and its C terminus engaging regions important for opioid-ligand selectivity.  Comparison of the μOR-Gi complex to previously detd. structures of other GPCRs bound to the stimulatory G protein Gs reveals differences in the position of transmembrane receptor helix 6 and in the interactions between the G protein α-subunit and the receptor core.  Together, these results shed light on the structural features that contribute to the Gi protein-coupling specificity of the μOR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSgJZtgX6dkbVg90H21EOLACvtfcHk0lhEVuCo4cSUmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjP&md5=92cddedb69cfa5f1a78cb419ef8b4ae0</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0219-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0219-7%26sid%3Dliteratum%253Aachs%26aulast%3DKoehl%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DMaeda%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DQu%26aufirst%3DQ.%26aulast%3DPaggi%26aufirst%3DJ.%2BM.%26aulast%3DLatorraca%26aufirst%3DN.%2BR.%26aulast%3DHilger%26aufirst%3DD.%26aulast%3DDawson%26aufirst%3DR.%26aulast%3DMatile%26aufirst%3DH.%26aulast%3DSchertler%26aufirst%3DG.%2BF.%2BX.%26aulast%3DGranier%26aufirst%3DS.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DSkiniotis%26aufirst%3DG.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520of%2520the%2520%25CE%25BC-opioid%2520receptor%25E2%2580%2593Gi%2520protein%2520complex%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D547%26epage%3D552%26doi%3D10.1038%2Fs41586-018-0219-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuybeda, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Waal, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Eps, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erramilli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melcher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossiakoff, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0215-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fs41586-018-0215-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29899450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=558&publication_year=2018&pages=553-558&issue=7711&author=Y.+Kangauthor=O.+Kuybedaauthor=P.+W.+de+Waalauthor=S.+Mukherjeeauthor=N.+Van+Epsauthor=P.+Dutkaauthor=X.+E.+Zhouauthor=A.+Bartesaghiauthor=S.+Erramilliauthor=T.+Morizumiauthor=X.+Guauthor=Y.+Yinauthor=P.+Liuauthor=Y.+Jiangauthor=X.+Mengauthor=G.+Zhaoauthor=K.+Melcherauthor=O.+P.+Ernstauthor=A.+A.+Kossiakoffauthor=S.+Subramaniamauthor=H.+E.+Xu&title=Cryo-EM+structure+of+human+rhodopsin+bound+to+an+inhibitory+G+protein&doi=10.1038%2Fs41586-018-0215-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span></div><div class="casAuthors">Kang, Yanyong; Kuybeda, Oleg; de Waal, Parker W.; Mukherjee, Somnath; Van Eps, Ned; Dutka, Przemyslaw; Zhou, X. Edward; Bartesaghi, Alberto; Erramilli, Satchal; Morizumi, Takefumi; Gu, Xin; Yin, Yanting; Liu, Ping; Jiang, Yi; Meng, Xing; Zhao, Gongpu; Melcher, Karsten; Ernst, Oliver P.; Kossiakoff, Anthony A.; Subramaniam, Sriram; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">553-558</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) comprise the largest family of mammalian transmembrane receptors.  They mediate numerous cellular pathways by coupling with downstream signalling transducers, including the heterotrimeric G proteins Gs (stimulatory) and Gi (inhibitory) and several arrestin proteins.  The structural mechanisms that define how G-protein-coupled receptors selectively couple to a specific type of G protein or arrestin remain unknown.  Here, using cryo-electron microscopy, we show that the major interactions between activated rhodopsin and Gi are mediated by the C-terminal helix of the Gi α-subunit, which is wedged into the cytoplasmic cavity of the transmembrane helix bundle and directly contacts the amino terminus of helix 8 of rhodopsin.  Structural comparisons of inactive, Gi-bound and arrestin-bound forms of rhodopsin with inactive and Gs-bound forms of the β2-adrenergic receptor provide a foundation to understand the unique structural signatures that are assocd. with the recognition of Gs, Gi and arrestin by activated G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo98ChG6dDQrVg90H21EOLACvtfcHk0lhEVuCo4cSUmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF&md5=e6d8e79977e0420442fa16ec0c53c692</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0215-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0215-y%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DY.%26aulast%3DKuybeda%26aufirst%3DO.%26aulast%3Dde%2BWaal%26aufirst%3DP.%2BW.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DVan%2BEps%26aufirst%3DN.%26aulast%3DDutka%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DX.%2BE.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DErramilli%26aufirst%3DS.%26aulast%3DMorizumi%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DMelcher%26aufirst%3DK.%26aulast%3DErnst%26aufirst%3DO.%2BP.%26aulast%3DKossiakoff%26aufirst%3DA.%2BA.%26aulast%3DSubramaniam%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DCryo-EM%2520structure%2520of%2520human%2520rhodopsin%2520bound%2520to%2520an%2520inhibitory%2520G%2520protein%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D553%26epage%3D558%26doi%3D10.1038%2Fs41586-018-0215-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yefanov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Waal, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moeller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascal, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Eps, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caro, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnivetskiy, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suino-Powell, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy-Chowdhury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lisova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupitz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotjohann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Standfuss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carragher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurevich, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbell, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melcher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of rhodopsin bound to arrestin by femtosecond X-ray laser</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>523</i></span> (<span class="NLM_issue">7562</span>),  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1038/nature14656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fnature14656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=26200343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WksrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=523&publication_year=2015&pages=561-567&issue=7562&author=Y.+Kangauthor=X.+E.+Zhouauthor=X.+Gaoauthor=Y.+Heauthor=W.+Liuauthor=A.+Ishchenkoauthor=A.+Bartyauthor=T.+A.+Whiteauthor=O.+Yefanovauthor=G.+W.+Hanauthor=Q.+Xuauthor=P.+W.+de+Waalauthor=J.+Keauthor=M.+H.+Tanauthor=C.+Zhangauthor=A.+Moellerauthor=G.+M.+Westauthor=B.+D.+Pascalauthor=N.+Van+Epsauthor=L.+N.+Caroauthor=S.+A.+Vishnivetskiyauthor=R.+J.+Leeauthor=K.+M.+Suino-Powellauthor=X.+Guauthor=K.+Palauthor=J.+Maauthor=X.+Zhiauthor=S.+Boutetauthor=G.+J.+Williamsauthor=M.+Messerschmidtauthor=C.+Gatiauthor=N.+A.+Zatsepinauthor=D.+Wangauthor=D.+Jamesauthor=S.+Basuauthor=S.+Roy-Chowdhuryauthor=C.+E.+Conradauthor=J.+Coeauthor=H.+Liuauthor=S.+Lisovaauthor=C.+Kupitzauthor=I.+Grotjohannauthor=R.+Frommeauthor=Y.+Jiangauthor=M.+Tanauthor=H.+Yangauthor=J.+Liauthor=M.+Wangauthor=Z.+Zhengauthor=D.+Liauthor=N.+Howeauthor=Y.+Zhaoauthor=J.+Standfussauthor=K.+Diederichsauthor=Y.+Dongauthor=C.+S.+Potterauthor=B.+Carragherauthor=M.+Caffreyauthor=H.+Jiangauthor=H.+N.+Chapmanauthor=J.+C.+Spenceauthor=P.+Frommeauthor=U.+Weierstallauthor=O.+P.+Ernstauthor=V.+Katritchauthor=V.+V.+Gurevichauthor=P.+R.+Griffinauthor=W.+L.+Hubbellauthor=R.+C.+Stevensauthor=V.+Cherezovauthor=K.+Melcherauthor=H.+E.+Xu&title=Crystal+structure+of+rhodopsin+bound+to+arrestin+by+femtosecond+X-ray+laser&doi=10.1038%2Fnature14656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of rhodopsin bound to arrestin by femtosecond X-ray laser</span></div><div class="casAuthors">Kang, Yanyong; Zhou, X. Edward; Gao, Xiang; He, Yuanzheng; Liu, Wei; Ishchenko, Andrii; Barty, Anton; White, Thomas A.; Yefanov, Oleksandr; Han, Gye Won; Xu, Qingping; de Waal, Parker W.; Ke, Jiyuan; Tan, M. H. Eileen; Zhang, Chenghai; Moeller, Arne; West, Graham M.; Pascal, Bruce D.; Van Eps, Ned; Caro, Lydia N.; Vishnivetskiy, Sergey A.; Lee, Regina J.; Suino-Powell, Kelly M.; Gu, Xin; Pal, Kuntal; Ma, Jinming; Zhi, Xiaoyong; Boutet, Sebastien; Williams, Garth J.; Messerschmidt, Marc; Gati, Cornelius; Zatsepin, Nadia A.; Wang, Dingjie; James, Daniel; Basu, Shibom; Roy-Chowdhury, Shatabdi; Conrad, Chelsie E.; Coe, Jesse; Liu, Haiguang; Lisova, Stella; Kupitz, Christopher; Grotjohann, Ingo; Fromme, Raimund; Jiang, Yi; Tan, Minjia; Yang, Huaiyu; Li, Jun; Wang, Meitian; Zheng, Zhong; Li, Dianfan; Howe, Nicole; Zhao, Yingming; Standfuss, Jorg; Diederichs, Kay; Dong, Yuhui; Potter, Clinton S.; Carragher, Bridget; Caffrey, Martin; Jiang, Hualiang; Chapman, Henry N.; Spence, John C. H.; Fromme, Petra; Weierstall, Uwe; Ernst, Oliver P.; Katritch, Vsevolod; Gurevich, Vsevolod V.; Griffin, Patrick R.; Hubbell, Wayne L.; Stevens, Raymond C.; Cherezov, Vadim; Melcher, Karsten; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">523</span>
        (<span class="NLM_cas:issue">7562</span>),
    <span class="NLM_cas:pages">561-567</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) signal primarily through G proteins or arrestins.  Arrestin binding to GPCRs blocks G protein interaction and redirects signalling to numerous G-protein-independent pathways.  Here we report the crystal structure of a constitutively active form of human rhodopsin bound to a pre-activated form of the mouse visual arrestin, detd. by serial femtosecond X-ray laser crystallog.  Together with extensive biochem. and mutagenesis data, the structure reveals an overall architecture of the rhodopsin-arrestin assembly in which rhodopsin uses distinct structural elements, including transmembrane helix 7 and helix 8, to recruit arrestin.  Correspondingly, arrestin adopts the pre-activated conformation, with a ∼20° rotation between the amino and carboxy domains, which opens up a cleft in arrestin to accommodate a short helix formed by the second intracellular loop of rhodopsin.  This structure provides a basis for understanding GPCR-mediated arrestin-biased signalling and demonstrates the power of X-ray lasers for advancing the frontiers of structural biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomyJZ-ekzglLVg90H21EOLACvtfcHk0lgXLCi8UiEo-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WksrvK&md5=574baa8ffa403be6aec8e9e1ffd81a7a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature14656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14656%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%2BE.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DYefanov%26aufirst%3DO.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3Dde%2BWaal%26aufirst%3DP.%2BW.%26aulast%3DKe%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DM.%2BH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMoeller%26aufirst%3DA.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DPascal%26aufirst%3DB.%2BD.%26aulast%3DVan%2BEps%26aufirst%3DN.%26aulast%3DCaro%26aufirst%3DL.%2BN.%26aulast%3DVishnivetskiy%26aufirst%3DS.%2BA.%26aulast%3DLee%26aufirst%3DR.%2BJ.%26aulast%3DSuino-Powell%26aufirst%3DK.%2BM.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DPal%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DZhi%26aufirst%3DX.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DRoy-Chowdhury%26aufirst%3DS.%26aulast%3DConrad%26aufirst%3DC.%2BE.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLisova%26aufirst%3DS.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DStandfuss%26aufirst%3DJ.%26aulast%3DDiederichs%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DPotter%26aufirst%3DC.%2BS.%26aulast%3DCarragher%26aufirst%3DB.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DErnst%26aufirst%3DO.%2BP.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DGurevich%26aufirst%3DV.%2BV.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26aulast%3DHubbell%26aufirst%3DW.%2BL.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DMelcher%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DCrystal%2520structure%2520of%2520rhodopsin%2520bound%2520to%2520arrestin%2520by%2520femtosecond%2520X-ray%2520laser%26jtitle%3DNature%26date%3D2015%26volume%3D523%26issue%3D7562%26spage%3D561%26epage%3D567%26doi%3D10.1038%2Fnature14656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic-Nikolic, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uliassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span> <span> </span><span class="NLM_article-title">A perspective on multi-target drug discovery and design for complex diseases</span>. <i>Clin Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.1186/s40169-017-0181-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1186%2Fs40169-017-0181-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=29340951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A280%3ADC%252BC1MvhsVaiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=3&issue=1&author=R.+R.+Ramsayauthor=M.+R.+Popovic-Nikolicauthor=K.+Nikolicauthor=E.+Uliassiauthor=M.+L.+Bolognesi&title=A+perspective+on+multi-target+drug+discovery+and+design+for+complex+diseases&doi=10.1186%2Fs40169-017-0181-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on multi-target drug discovery and design for complex diseases</span></div><div class="casAuthors">Ramsay Rona R; Popovic-Nikolic Marija R; Nikolic Katarina; Uliassi Elisa; Bolognesi Maria Laura</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and translational medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3</span>
        ISSN:<span class="NLM_cas:issn">2001-1326</span>.
    </div><div class="casAbstract">Diseases of infection, of neurodegeneration (such as Alzheimer's and Parkinson's diseases), and of malignancy (cancers) have complex and varied causative factors.  Modern drug discovery has the power to identify potential modulators for multiple targets from millions of compounds.  Computational approaches allow the determination of the association of each compound with its target before chemical synthesis and biological testing is done.  These approaches depend on the prior identification of clinically and biologically validated targets.  This Perspective will focus on the molecular and computational approaches that underpin drug design by medicinal chemists to promote understanding and collaboration with clinical scientists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJhSGsWpifCZcXNb62YGmbfW6udTcc2eZVCGL_U132mLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvhsVaiug%253D%253D&md5=5b05a73a3bbc955c77b9b2df93deb946</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1186%2Fs40169-017-0181-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40169-017-0181-2%26sid%3Dliteratum%253Aachs%26aulast%3DRamsay%26aufirst%3DR.%2BR.%26aulast%3DPopovic-Nikolic%26aufirst%3DM.%2BR.%26aulast%3DNikolic%26aufirst%3DK.%26aulast%3DUliassi%26aufirst%3DE.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26atitle%3DA%2520perspective%2520on%2520multi-target%2520drug%2520discovery%2520and%2520design%2520for%2520complex%2520diseases%26jtitle%3DClin%2520Transl.%2520Med.%26date%3D2018%26volume%3D7%26issue%3D1%26spage%3D3%26doi%3D10.1186%2Fs40169-017-0181-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douguet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">46277</span>, <span class="refDoi"> DOI: 10.1038/srep46277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1038%2Fsrep46277" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=28397808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=46277&author=D.+Lagorceauthor=D.+Douguetauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=Computational+analysis+of+calculated+physicochemical+and+ADMET+properties+of+protein-protein+interaction+inhibitors&doi=10.1038%2Fsrep46277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span></div><div class="casAuthors">Lagorce, David; Douguet, Dominique; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46277</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The modulation of PPIs by low mol. wt. chem. compds., particularly by orally bioavailable mols., would be very valuable in numerous disease indications.  However, it is known that PPI inhibitors (iPPIs) tend to have properties that are linked to poor Absorption, Distribution, Metab., Excretion and Toxicity (ADMET) and in some cases to poor clin. outcomes.  Previously reported in silico analyses of iPPIs have essentially focused on physicochem. properties but several other ADMET parameters would be important to assess.  In order to gain new insights into the ADMET properties of iPPIs, computations were carried out on eight datasets collected from several databases.  These datasets involve compds. targeting enzymes, GPCRs, ion channels, nuclear receptors, allosteric modulators, oral marketed drugs, oral natural product-derived marketed drugs and iPPIs.  Several trends are reported that should assist the design and optimization of future PPI inhibitors, either for drug discovery endeavors or for chem. biol. projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw3PmL5znBCLVg90H21EOLACvtfcHk0ljZ_U8X5HWirQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D&md5=162ddf6594d20c770c1016313182245c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fsrep46277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep46277%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DDouguet%26aufirst%3DD.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DComputational%2520analysis%2520of%2520calculated%2520physicochemical%2520and%2520ADMET%2520properties%2520of%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D46277%26doi%3D10.1038%2Fsrep46277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">FAF-Drugs3: a web server for compound property calculation and chemical library design</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W200</span>– <span class="NLM_lpage">W207</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1093%2Fnar%2Fgkv353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=25883137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W200-W207&issue=W1&author=D.+Lagorceauthor=O.+Sperandioauthor=J.+B.+Baellauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=FAF-Drugs3%3A+a+web+server+for+compound+property+calculation+and+chemical+library+design&doi=10.1093%2Fnar%2Fgkv353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">FAF-Drugs3: a web server for compound property calculation and chemical library design</span></div><div class="casAuthors">Lagorce, David; Sperandio, Olivier; Baell, Jonathan B.; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W200-W207</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Drug attrition late in preclin. or clin. development is a serious economic problem in the field of drug discovery.  These problems can be linked, in part, to the quality of the compd. collections used during the hit generation stage and to the selection of compds. undergoing optimization.  Here, we present FAF-Drugs3, a web server that can be used for drug discovery and chem. biol. projects to help in prepg. compd. libraries and to assist decision-making during the hit selection/lead optimization phase.  Since it was first described in 2006, FAF-Drugs has been significantly modified.  The tool now applies an enhanced structure curation procedure, can filter or analyze mols. with user-defined or eight predefined physicochem. filters as well as with several simple ADMET (absorption, distribution, metab., excretion and toxicity) rules.  In addn., compds. can be filtered using an updated list of 154 hand-curated structural alerts while Pan Assay Interference compds. (PAINS) and other, generally unwanted groups are also investigated.  FAF-Drugs3 offers access to user-friendly html result pages and the possibility to download all computed data.  The server requires as input an SDF file of the compds.; it is open to all users and can be accessed without registration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxYZ-hCUhlmrVg90H21EOLACvtfcHk0ljZ_U8X5HWirQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLfM&md5=5cf8888c489b90f392117ab065986214</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv353%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DO.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DFAF-Drugs3%253A%2520a%2520web%2520server%2520for%2520compound%2520property%2520calculation%2520and%2520chemical%2520library%2520design%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DW1%26spage%3DW200%26epage%3DW207%26doi%3D10.1093%2Fnar%2Fgkv353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&issue=7&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lh3VZ-LYlnbPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D7%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lager, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Østergaard
Nielsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liljefors, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterner, O.</span></span> <span> </span><span class="NLM_article-title">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABAA receptors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">6936</span>– <span class="NLM_lpage">6948</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2Fj.bmc.2008.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=18541432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=6936-6948&issue=14&author=E.+Lagerauthor=J.+Nilssonauthor=E.+%C3%98stergaard%0ANielsenauthor=M.+Nielsenauthor=T.+Liljeforsauthor=O.+Sterner&title=Affinity+of+3-acyl+substituted+4-quinolones+at+the+benzodiazepine+site+of+GABAA+receptors&doi=10.1016%2Fj.bmc.2008.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity of 3-acyl substituted 4-quinolones at the benzodiazepine site of GABAA receptors</span></div><div class="casAuthors">Lager, Erik; Nilsson, Jakob; Ostergaard Nielsen, Elsebet; Nielsen, Mogens; Liljefors, Tommy; Sterner, Olov</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6936-6948</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The finding that alkyl 1,4-dihydro-4-oxoquinoline-3-carboxylate and N-alkyl-1,4-dihydro-4-oxoquinoline-3-carboxamide derivs. may be high-affinity ligands at the benzodiazepine binding site of the GABAA receptor, prompted a study of 3-acyl-1,4-dihydro-4-oxoquinoline (3-acyl-4-quinolones).  In general, the affinity of the 3-acyl derivs. was found to be comparable with the 3-carboxylate and the 3-carboxamide derivs., and certain substituents (e.g., benzyl) in position 6 were again shown to be important.  As it is believed that the benzodiazepine binding site is situated between an α- and a γ-subunit in the GABAA receptor, selected compds. were tested on the α1β2γ2s, α2β2γ2s and α3β2γ2s GABAA receptor subtypes.  The 3-acyl-4-quinolones display various degrees of selectivity for α1- vs. α2- and α3-contg. receptors, and high-affinity ligands essentially selective for α1 over α3 were developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocF9dBEu1TLrVg90H21EOLACvtfcHk0lh3VZ-LYlnbPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVKqtr0%253D&md5=729802775ad271d534dccddae8f1968b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DLager%26aufirst%3DE.%26aulast%3DNilsson%26aufirst%3DJ.%26aulast%3D%25C3%2598stergaard%2BNielsen%26aufirst%3DE.%26aulast%3DNielsen%26aufirst%3DM.%26aulast%3DLiljefors%26aufirst%3DT.%26aulast%3DSterner%26aufirst%3DO.%26atitle%3DAffinity%2520of%25203-acyl%2520substituted%25204-quinolones%2520at%2520the%2520benzodiazepine%2520site%2520of%2520GABAA%2520receptors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26issue%3D14%26spage%3D6936%26epage%3D6948%26doi%3D10.1016%2Fj.bmc.2008.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, F. J.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of 2-alkylpyrimidine-4,6-diol and 6-alkylpyridine-2,4-diol as potent GPR84 agonists</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=579-583&issue=6&author=Y.+Liuauthor=Q.+Zhangauthor=L.+H.+Chenauthor=H.+Yangauthor=W.+Luauthor=X.+Xieauthor=F.+J.+Nan&title=Design+and+synthesis+of+2-alkylpyrimidine-4%2C6-diol+and+6-alkylpyridine-2%2C4-diol+as+potent+GPR84+agonists&doi=10.1021%2Facsmedchemlett.6b00025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of 2-Alkylpyrimidine-4,6-diol and 6-Alkylpyridine-2,4-diol as Potent GPR84 Agonists</span></div><div class="casAuthors">Liu, Yang; Zhang, Qing; Chen, Lin-Hai; Yang, Hui; Lu, Wei; Xie, Xin; Nan, Fa-Jun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">579-583</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of alkylpyrimidine-4,6-diol derivs. were designed and synthesized as novel GPR84 agonists based on a high-throughput screening (HTS) hit I. 6-Nonylpyridine-2,4-diol was identified as the most potent agonist of GPR84 reported so far, with an EC50 of 0.189 nM.  These novel GPR84 agonists will provide valuable tools for the study of the physiol. functions of GPR84.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyEcxh2lX2Q7Vg90H21EOLACvtfcHk0lgOVc3b6SRLTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D&md5=52f50ffae4ab4bbfd79f9eeda0bb063b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00025%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DF.%2BJ.%26atitle%3DDesign%2520and%2520synthesis%2520of%25202-alkylpyrimidine-4%252C6-diol%2520and%25206-alkylpyridine-2%252C4-diol%2520as%2520potent%2520GPR84%2520agonists%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26issue%3D6%26spage%3D579%26epage%3D583%26doi%3D10.1021%2Facsmedchemlett.6b00025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. S.</span></span> <span> </span><span class="NLM_article-title">Nickel-catalyzed rearrangement of 1-acyl-2-vinylcyclopropanes. A mild synthesis of substituted dihydrofurans</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1021/ol052700k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052700k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvFKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=573-576&issue=4&author=R.+K.+Bowmanauthor=J.+S.+Johnson&title=Nickel-catalyzed+rearrangement+of+1-acyl-2-vinylcyclopropanes.+A+mild+synthesis+of+substituted+dihydrofurans&doi=10.1021%2Fol052700k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Nickel-Catalyzed Rearrangement of 1-Acyl-2-vinylcyclopropanes. A Mild Synthesis of Substituted Dihydrofurans</span></div><div class="casAuthors">Bowman, Roy K.; Johnson, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-576</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mild Ni(0)-catalyzed rearrangements of 1-acyl-2-vinylcyclopropanes have been developed.  The room-temp. isomerizations afford dihydrofuran products in yields regularly greater than 90%.  E.g., Ni(COD)-catalyzed rearrangements of 1-acyl-2-vinylcyclopropane I gave 97% dihydrofuran II.  A highly substituted, stereochem. defined cyclopropane was employed in the rearrangement to evaluate the reaction mechanism.  Product anal. indicates that the overall reaction proceeds with retention of configuration at the vinyl-bearing stereogenic center.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw0gn_2J4LdLVg90H21EOLACvtfcHk0lgOVc3b6SRLTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvFKnsw%253D%253D&md5=2fd83a59ff122addd736a9c778f92696</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fol052700k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052700k%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DR.%2BK.%26aulast%3DJohnson%26aufirst%3DJ.%2BS.%26atitle%3DNickel-catalyzed%2520rearrangement%2520of%25201-acyl-2-vinylcyclopropanes.%2520A%2520mild%2520synthesis%2520of%2520substituted%2520dihydrofurans%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26issue%3D4%26spage%3D573%26epage%3D576%26doi%3D10.1021%2Fol052700k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doni, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuttle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J. A.</span></span> <span> </span><span class="NLM_article-title">Overturning established chemoselectivities: selective reduction of arenes over malonates and cyanoacetates by photoactivated organic electron donors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">10934</span>– <span class="NLM_lpage">10937</span>, <span class="refDoi"> DOI: 10.1021/ja4050168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4050168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFamsrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=10934-10937&issue=30&author=E.+Doniauthor=B.+Mondalauthor=S.+O%E2%80%99Sullivanauthor=T.+Tuttleauthor=J.+A.+Murphy&title=Overturning+established+chemoselectivities%3A+selective+reduction+of+arenes+over+malonates+and+cyanoacetates+by+photoactivated+organic+electron+donors&doi=10.1021%2Fja4050168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overturning Established Chemoselectivities: Selective Reduction of Arenes over Malonates and Cyanoacetates by Photoactivated Organic Electron Donors</span></div><div class="casAuthors">Doni, Eswararao; Mondal, Bhaskar; O'Sullivan, Steven; Tuttle, Tell; Murphy, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">10934-10937</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The prevalence of metal-based reducing reagents, including metals, metal complexes, and metal salts, has produced an empirical order of reactivity that governs our approach to chem. synthesis.  However, this reactivity may be influenced by stabilization of transition states, intermediates, and products through substrate-metal bonding.  This article reports that in the absence of such stabilizing interactions, established chemoselectivities can be overthrown.  Thus, photoactivation of the recently developed neutral org. superelectron donor 5 selectively reduces alkyl-substituted benzene rings in the presence of activated esters and nitriles, in direct contrast to metal-based redns., opening a new perspective on reactivity.  The altered outcomes arising from the org. electron donors are attributed to selective interactions between the neutral org. donors and the arene rings of the substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEFVT1Yr3ZZbVg90H21EOLACvtfcHk0lgOVc3b6SRLTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFamsrnI&md5=8b10c1a7b9679fe1d53b9737fdf9beb1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja4050168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4050168%26sid%3Dliteratum%253Aachs%26aulast%3DDoni%26aufirst%3DE.%26aulast%3DMondal%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DS.%26aulast%3DTuttle%26aufirst%3DT.%26aulast%3DMurphy%26aufirst%3DJ.%2BA.%26atitle%3DOverturning%2520established%2520chemoselectivities%253A%2520selective%2520reduction%2520of%2520arenes%2520over%2520malonates%2520and%2520cyanoacetates%2520by%2520photoactivated%2520organic%2520electron%2520donors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26issue%3D30%26spage%3D10934%26epage%3D10937%26doi%3D10.1021%2Fja4050168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amupitan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beddoes, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, J. K.</span></span> <span> </span><span class="NLM_article-title">3-Methylcyclohex-2-enone derivatives as initiators of cyclisation. Part 4. Some bicyclisations</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>0</i></span>,  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1039/p19830000759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1039%2Fp19830000759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADyaL3sXlsFOqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=0&publication_year=1983&pages=759-763&author=J.+A.+Amupitanauthor=R.+L.+Beddoesauthor=O.+S.+Millsauthor=J.+K.+Sutherland&title=3-Methylcyclohex-2-enone+derivatives+as+initiators+of+cyclisation.+Part+4.+Some+bicyclisations&doi=10.1039%2Fp19830000759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">3-Methylcyclohex-2-enone derivatives as initiators of cyclization.  Part 4.  Some bicyclizations</span></div><div class="casAuthors">Amupitan, Joseph; Beddoes, Roy L.; Mills, Owen S.; Sutherland, James K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1:  Organic and Bio-Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">759-63</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    </div><div class="casAbstract">Hydrochrysene and hydrophenanthrene derivs. were prepd. by cyclization of 2-alkenyl-3-methylcyclohexen-2-ones and their epoxides.  E.g., cyclization of cyclohexenone I (R2 = bond, R1 = C6H4OMe-3) with (F3CCO)2O/F3CCO2H at O° gave tetracycles II [R = OMe, R1 = H (III); R = H, R1 = OMe] in 56 and 14% yield, resp., and cyclization of I (R2 = O, R1 = C≡CH) with BCl3 in CH2Cl2 at -78° gave 35% ketol IV.  The structures of III and IV were detd. by x-ray crystallog. anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQmt43i_sXOLVg90H21EOLACvtfcHk0lhmMvdBnGEyvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXlsFOqtb8%253D&md5=65ffaf93be128ada4c7962ae00b372d7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2Fp19830000759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp19830000759%26sid%3Dliteratum%253Aachs%26aulast%3DAmupitan%26aufirst%3DJ.%2BA.%26aulast%3DBeddoes%26aufirst%3DR.%2BL.%26aulast%3DMills%26aufirst%3DO.%2BS.%26aulast%3DSutherland%26aufirst%3DJ.%2BK.%26atitle%3D3-Methylcyclohex-2-enone%2520derivatives%2520as%2520initiators%2520of%2520cyclisation.%2520Part%25204.%2520Some%2520bicyclisations%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1983%26volume%3D0%26spage%3D759%26epage%3D763%26doi%3D10.1039%2Fp19830000759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span>; <span class="NLM_string-name">Tymianski, M.</span>; <span class="NLM_string-name">Garman, D.</span></span> <span> </span><span class="NLM_article-title">Agents And Methods For Treating Ischemic And Other Diseases</span>. Patent <span class="NLM_patent">WO 2012/174488 A2</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=X.+Sun&author=M.+Tymianski&author=D.+Garman&title=Agents+And+Methods+For+Treating+Ischemic+And+Other+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26atitle%3DAgents%2520And%2520Methods%2520For%2520Treating%2520Ischemic%2520And%2520Other%2520Diseases%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krohn, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermanson, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallia, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gartner, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Provenzano, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimoto, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeke, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klenk, D. C.</span></span> <span> </span><span class="NLM_article-title">Measurement of protein using bicinchoninic acid</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>150</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(85)90442-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2F0003-2697%2885%2990442-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=3843705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADyaL2MXlsFKksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=1985&pages=76-85&issue=1&author=P.+K.+Smithauthor=R.+I.+Krohnauthor=G.+T.+Hermansonauthor=A.+K.+Malliaauthor=F.+H.+Gartnerauthor=M.+D.+Provenzanoauthor=E.+K.+Fujimotoauthor=N.+M.+Goekeauthor=B.+J.+Olsonauthor=D.+C.+Klenk&title=Measurement+of+protein+using+bicinchoninic+acid&doi=10.1016%2F0003-2697%2885%2990442-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of protein using bicinchoninic acid</span></div><div class="casAuthors">Smith, P. K.; Krohn, R. I.; Hermanson, G. T.; Mallia, A. K.; Gartner, F. H.; Provenzano, M. D.; Fujimoto, E. K.; Goeke, N. M.; Olson, B. J.; Klenk, D. C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-85</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A method for the colorimetric detn. of proteins is described which uses bicinchoninic acid to monitor the Cu produced during the biuret reaction.  The method (micro and macro) was applied to the detn. of 7 proteins.  Bicinchoninic acid forms a 2:1 complex with Cu, resulting in a stable, highly colored chromophore with an absorbance max. at 562 nm.  The color produced from this reaction is stable and increases in a proportional fashion over a broad range of increasing protein concns.  When compared to the method of O. H. Lowry et al. (1951), the results reported demonstrate a greater tolerance of the bicinchoninate reagent toward such commonly encountered interferences as nonionic detergents and simple buffer salts.  The stability of the reagent and resulting chromophore also allows for a simplified, 1-step anal. and an enhanced flexibility in protocol selection.  This new method maintains the high sensitivity and low protein-to-protein variation assocd. with the Lowry technique.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbD7DGwzaIm7Vg90H21EOLACvtfcHk0lhmMvdBnGEyvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXlsFKksL0%253D&md5=f4f012331cf4dafa50cdc2d345bade38</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2885%2990442-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252885%252990442-7%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DP.%2BK.%26aulast%3DKrohn%26aufirst%3DR.%2BI.%26aulast%3DHermanson%26aufirst%3DG.%2BT.%26aulast%3DMallia%26aufirst%3DA.%2BK.%26aulast%3DGartner%26aufirst%3DF.%2BH.%26aulast%3DProvenzano%26aufirst%3DM.%2BD.%26aulast%3DFujimoto%26aufirst%3DE.%2BK.%26aulast%3DGoeke%26aufirst%3DN.%2BM.%26aulast%3DOlson%26aufirst%3DB.%2BJ.%26aulast%3DKlenk%26aufirst%3DD.%2BC.%26atitle%3DMeasurement%2520of%2520protein%2520using%2520bicinchoninic%2520acid%26jtitle%3DAnal.%2520Biochem.%26date%3D1985%26volume%3D150%26issue%3D1%26spage%3D76%26epage%3D85%26doi%3D10.1016%2F0003-2697%2885%2990442-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span> <i>Maestro</i>, release <span class="NLM_edition">2017-1</span>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Maestro%2C+release+2017-1%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span> <i>Prime</i>, release <span class="NLM_edition">2017-1</span>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Prime%2C+release+2017-1%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">On the role of the crystal environment in determining protein side-chain conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>320</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(02)00470-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&issue=3&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+role+of+the+crystal+environment+in+determining+protein+side-chain+conformations&doi=10.1016%2FS0022-2836%2802%2900470-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0ljT6EGEzydIoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520role%2520of%2520the%2520crystal%2520environment%2520in%2520determining%2520protein%2520side-chain%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26issue%3D3%26spage%3D597%26epage%3D608%26doi%3D10.1016%2FS0022-2836%2802%2900470-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pincus, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&issue=2&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0ljaqSsf69VO3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26issue%3D2%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span> <i>Glide Release 2017-1; Schrödinger Suite Prime 2017-1
Induced Fit Docking Protocol</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide+Release+2017-1%3B+Schr%C3%B6dinger+Suite+Prime+2017-1%0AInduced+Fit+Docking+Protocol%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%2520Release%25202017-1%253B%2520Schr%25C3%25B6dinger%2520Suite%2520Prime%25202017-1%250AInduced%2520Fit%2520Docking%2520Protocol%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&issue=2&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Å.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Å; the RMSD is ≤1.8 Å for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Å, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0ljaqSsf69VO3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26issue%3D2%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span> <i>The PyMOL Molecular Graphics System</i>, version <span class="NLM_edition">1.8</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular+Graphics+System%2C+version+1.8%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1A','PDB','5T1A'); return false;">PDB: 5T1A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MBS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MBS','PDB','4MBS'); return false;">PDB: 4MBS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i108"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_02470"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b00742?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00742</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Characterization of ligands in a U2OS-CCR5 β-arrestin-recruitment assay (Figure S1); correlation between log <i>P</i> (cLogP) and affinity (p<i>K</i><sub>i</sub>) values in CCR2 (Figure S2); characterization of compounds <b>39</b> and <b>43</b> as potential inverse agonists in hCCR2 (Figure S3); docking of compounds <b>8</b>, <b>39</b>, <b>40</b>, and <b>43</b> (Figure S4); list of intermediate compounds <b>4aa</b>–<b>na</b>, <b>4bb</b>–<b>bq</b>, <b>4eq</b>–<b>ev</b> (Table S1); functional activity of TAK-779 and CCR2-RA-[<i>R</i>] in hCCR5, using a CCL3-induced β-arrestin recruitment assay (Table S2); and functional activity of compounds <b>8</b>, <b>39</b>, and <b>43</b> in hCCR2, using a CCL2-induced β-arrestin recruitment assay (Table S3) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_002.csv">CSV</a>)</p></li><li><p class="inline">Model of CCR2b (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_007.pdb">PDB</a>)</p></li><li><p class="inline">Model of compound <b>8</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Model of compound <b>39</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Model of compound <b>40</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Model of compound <b>43</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_006.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_001.pdf">jm9b00742_si_001.pdf (1.53 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_002.csv">jm9b00742_si_002.csv (2.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_003.pdb">jm9b00742_si_003.pdb (3.95 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_004.pdb">jm9b00742_si_004.pdb (4.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_005.pdb">jm9b00742_si_005.pdb (4.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_006.pdb">jm9b00742_si_006.pdb (4.14 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00742/suppl_file/jm9b00742_si_007.pdb">jm9b00742_si_007.pdb (377.37 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00742&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00742%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00742" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c4eddc5a258d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
