
<html lang="en"     class="pb-page"  data-request-id="e817a8a3-2973-4f52-bf35-fa5fb01816ea"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2016.59.issue-19;article:article:10.1021/acs.jmedchem.6b01033"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6" /></meta><meta name="dc.Creator" content="Yudao  Shen" /></meta><meta name="dc.Creator" content="Magdalena M.  Szewczyk" /></meta><meta name="dc.Creator" content="Mohammad S.  Eram" /></meta><meta name="dc.Creator" content="David  Smil" /></meta><meta name="dc.Creator" content="H. Ümit  Kaniskan" /></meta><meta name="dc.Creator" content="Renato  Ferreira de Freitas" /></meta><meta name="dc.Creator" content="Guillermo  Senisterra" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Matthieu  Schapira" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chemical tools for testing biological and therapeutic hypotheses. Based on 4, a fragment-like i..." /></meta><meta name="Description" content="Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chemical tools for testing biological and therapeutic hypotheses. Based on 4, a fragment-like i..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 15, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01033" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01033" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01033" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01033" /></link>
        
    
    

<title>Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01033" /></meta><meta property="og:title" content="Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0011.jpeg" /></meta><meta property="og:description" content="Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chemical tools for testing biological and therapeutic hypotheses. Based on 4, a fragment-like inhibitor of type I PRMTs, we conducted structure–activity relationship (SAR) studies and explored three regions of this scaffold. The studies led to the discovery of a potent, selective, and cell-active dual inhibitor of PRMT4 and PRMT6, 17 (MS049). As compared to 4, 17 displayed much improved potency for PRMT4 and PRMT6 in both biochemical and cellular assays. It was selective for PRMT4 and PRMT6 over other PRMTs and a broad range of other epigenetic modifiers and nonepigenetic targets. We also developed 46 (MS049N), which was inactive in biochemical and cellular assays, as a negative control for chemical biology studies. Considering possible overlapping substrate specificity of PRMTs, 17 and 46 are valuable chemical tools for dissecting specific biological functions and dysregulation of PRMT4 and PRMT6 in health and disease." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01033"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01033">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01033&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01033&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01033&amp;href=/doi/10.1021/acs.jmedchem.6b01033" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 19</span><span class="cit-fg-pageRange">, 9124-9139</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/19" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01031" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01046" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yudao++Shen">Yudao Shen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena+M.++Szewczyk">Magdalena M. Szewczyk</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mohammad+S.++Eram">Mohammad S. Eram</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Smil">David Smil</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=H.+%C3%9Cmit++Kaniskan">H. Ümit Kaniskan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Renato++Ferreira+de+Freitas">Renato Ferreira de Freitas</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guillermo++Senisterra">Guillermo Senisterra</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthieu++Schapira">Matthieu Schapira</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Princess Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span></div><div class="corresp-info"><strong>*</strong>J.L.: e-mail, <a href="/cdn-cgi/l/email-protection#5b1132353c7537322e1b36282836753e3f2e"><span class="__cf_email__" data-cfemail="3e745750591052574b7e534d4d53105b5a4b">[email protected]</span></a>; phone, (212) 659-5499.</div><div class="corresp-info"><strong>*</strong>M.V.: e-mail, <a href="/cdn-cgi/l/email-protection#d8b5f6aebdbcb9bcb198adacb7aab7b6acb7f6bbb9"><span class="__cf_email__" data-cfemail="93febde5f6f7f2f7fad3e6e7fce1fcfde7fcbdf0f2">[email protected]</span></a>; phone, (416) 976-0897.</div><div class="corresp-info"><strong>*</strong>J.J.: e-mail, <a href="/cdn-cgi/l/email-protection#375d5e5659195d5e59775a44445a19525342"><span class="__cf_email__" data-cfemail="d7bdbeb6b9f9bdbeb997baa4a4baf9b2b3a2">[email protected]</span></a>; phone, (212) 659-8699.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01033&amp;href=/doi/10.1021%2Facs.jmedchem.6b01033" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 19</span><span class="cit-pageRange">, 9124–9139</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 1, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 July 2016</li><li><span class="item_label"><b>Published</b> online</span>15 September 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 October 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01033" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01033</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9124%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYudao%2BShen%252C%2BMagdalena%2BM.%2BSzewczyk%252C%2BMohammad%2BS.%2BEram%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D19%26contentID%3Dacs.jmedchem.6b01033%26title%3DDiscovery%2Bof%2Ba%2BPotent%252C%2BSelective%252C%2Band%2BCell-Active%2BDual%2BInhibitor%2Bof%2BProtein%2BArginine%2BMethyltransferase%2B4%2Band%2BProtein%2BArginine%2BMethyltransferase%2B6%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9139%26publicationDate%3DOctober%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01033"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2068</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">40</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01033" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yudao&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;M. Szewczyk&quot;},{&quot;first_name&quot;:&quot;Mohammad&quot;,&quot;last_name&quot;:&quot;S. Eram&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Smil&quot;},{&quot;first_name&quot;:&quot;H.&quot;,&quot;last_name&quot;:&quot;Ümit Kaniskan&quot;},{&quot;first_name&quot;:&quot;Renato&quot;,&quot;last_name&quot;:&quot;Ferreira de Freitas&quot;},{&quot;first_name&quot;:&quot;Guillermo&quot;,&quot;last_name&quot;:&quot;Senisterra&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Matthieu&quot;,&quot;last_name&quot;:&quot;Schapira&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;9124-9139&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01033&quot;},&quot;abstract&quot;:&quot;Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chemical tools for testing biological and therapeutic hypotheses. Based on 4, a fragment-like inhibitor of type I PRMTs, we conducted structure–activity relationship (SAR) studies and explored three regions of this scaffold. The studies led to the discovery of a potent, selective, and cell-active dual inhibitor of PRMT4 and PRMT6, 17 (MS049). As compared to 4, 17 displayed much improved potency for PRMT4 and PRMT6 in both biochemical and cellular assays. It was selective for PRMT4 and PRMT6 over other PRMTs and a broad range of other epigenetic modifiers and nonepigenetic targets. We also developed 46 (MS049N), which was inactive in biochemical and cellular assays, as a negative control for chemical biology studies. Considering possible overlapping substrate specificity of PRMTs, 17 and 46 are valuable chemical tools for dissecting specific biological functions and dysregulation of PRMT4 and PRMT6 in health a&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01033&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01033" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01033&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01033" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01033&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01033" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01033&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01033&amp;href=/doi/10.1021/acs.jmedchem.6b01033" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01033" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01033" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01033%26sid%3Dliteratum%253Aachs%26pmid%3D27584694%26genre%3Darticle%26aulast%3DShen%26date%3D2016%26atitle%3DDiscovery%2Bof%2Ba%2BPotent%252C%2BSelective%252C%2Band%2BCell-Active%2BDual%2BInhibitor%2Bof%2BProtein%2BArginine%2BMethyltransferase%2B4%2Band%2BProtein%2BArginine%2BMethyltransferase%2B6%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D19%26spage%3D9124%26epage%3D9139%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291301" title="Salts">Salts</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/19" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/jmcmar.2016.59.issue-19/20161013/jmcmar.2016.59.issue-19.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chemical tools for testing biological and therapeutic hypotheses. Based on <b>4</b>, a fragment-like inhibitor of type I PRMTs, we conducted structure–activity relationship (SAR) studies and explored three regions of this scaffold. The studies led to the discovery of a potent, selective, and cell-active dual inhibitor of PRMT4 and PRMT6, <b>17</b> (MS049). As compared to <b>4</b>, <b>17</b> displayed much improved potency for PRMT4 and PRMT6 in both biochemical and cellular assays. It was selective for PRMT4 and PRMT6 over other PRMTs and a broad range of other epigenetic modifiers and nonepigenetic targets. We also developed <b>46</b> (MS049N), which was inactive in biochemical and cellular assays, as a negative control for chemical biology studies. Considering possible overlapping substrate specificity of PRMTs, <b>17</b> and <b>46</b> are valuable chemical tools for dissecting specific biological functions and dysregulation of PRMT4 and PRMT6 in health and disease.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54081" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54081" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Arginine residues contain a characteristic guanidinium moiety, which is protonated and positively charged at physiological conditions.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The arginine guanidinium group can interact with other polar groups through its five potential hydrogen bond donors. Consequently, through stable bidentate salt bridges and hydrogen bonds, arginine is frequently involved in protein/protein and protein/nucleic acid complex formations.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Arginine methylation, a major post-translational modification of arginine residues, does not change its cationic charge character.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Instead, this modification increases the bulkiness of the guanidinium moiety and alters the charge distribution, hydrophobicity, and hydrogen bond formation potential.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> As a result, arginine methylation can have dramatic effects on the interactions of the modified protein with other proteins and nucleic acids and thus its physiological function,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> including RNA processing, DNA repair, transcriptional activation/repression, signal transduction, cell differentiation, and embryonic development.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">As a common post-translational modification in eukaryotic cells,<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8-10)</a> arginine methylation is catalyzed by <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) dependent PRMTs.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Currently, nine PRMTs have been identified and grouped into three categories based on their product specificity.<a onclick="showRef(event, 'ref9 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref11 ref12">(9, 11, 12)</a> Type I PRMTs, including PRMT1–4, PRMT6, and PRMT8, catalyze arginine monomethylation and asymmetric dimethylation. Type II PRMTs, including PRMT5 and PRMT9, catalyze arginine monomethylation and symmetric dimethylation. PRMT7, which catalyzes only arginine monomethylation, is the sole type III PRMT.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> PRMTs have a vital role in the regulation of the arginine methylation pattern and level of a plethora of different substrates, including both histones and non-histone proteins.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Thus, it is not surprising that dysregulation of PRMTs has been linked to a variety of human diseases.<a onclick="showRef(event, 'ref7 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref11 ref12">(7, 11, 12)</a> For example, PRMT4 (also known as coactivator-associated arginine methyltransferase 1, CARM1) has a broad range of substrates including histones (e.g., H3R17, H3R26),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> transcriptional factors (e.g., SOX2, SOX9, PAX7, RUNX1),<a onclick="showRef(event, 'ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17">(14-17)</a> chromatin remodeling factor (e.g., BAF155),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> transcriptional coactivators (e.g., CBP, p300, SRC3),<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19-21)</a> RNA binding proteins (e.g., PABP1, HuR, HuD),<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22-24)</a> splicing factors (e.g., CA150, SAP49, SmB, U1C),<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> T cell-specific factor (e.g., TARPP),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> RNA polymerase II,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and mediator of RNA polymerase II transcription (e.g., Med12).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Overexpression of PRMT4 has been associated with AML,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> breast,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> prostate,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> lung,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and colorectal<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> cancers. The primary methylation targets of PRMT6 are histone H3R2<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and DNA polymerase β.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> PRMT6 can also methylate HIV Tat<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and HMGA1a.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> PRMT6 is overexpressed in melanoma,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> bladder,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> lung,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and prostate<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> cancers. Thus, growing evidence suggests PRMT4 and PRMT6 as potential therapeutic targets.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> In addition, both PRMT4 and PRMT6 dimethylate H3R42 in vitro and in vivo.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Furthermore, PRMT4 and PRMT6 show a synergistic effect in stimulating estrogen receptor α (ERα) dependent transcription.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Therefore, targeting PRMT4 and PRMT6 simultaneously may provide a beneficial therapeutic strategy.</div><div class="NLM_p">Potent, selective, and cell-active small-molecule inhibitors of PRMTs<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41, 42)</a> would be invaluable tools for investigating the biology of these protein targets and testing the disease and therapeutic hypotheses associated with them. Over the past decade, a number of small-molecule PRMT inhibitors have been reported,<a onclick="showRef(event, 'ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59 ref60">(43-60)</a> which include a series of PRMT4 inhibitors represented by compound <b>1</b><a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and a PRMT6 selective inhibitor, <b>2</b><a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). On the basis of the PRMT4 and PRMT6 inhibitors,<a onclick="showRef(event, 'ref50 ref57'); return false;" href="javascript:void(0);" class="ref ref50 ref57">(50, 57)</a> we previously discovered <b>3</b> (MS023), a potent, selective, and cell-active inhibitor of type I PRMTs (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Recently, benzylpiperidinylethanamine <b>4 (</b><a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B) was identified as a fragment-like inhibitor of type I PRMTs (IC<sub>50</sub> = 0.3–19 μM).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The crystal structure of PRMT6 in complex with <b>4</b> revealed that the ethylenediamino moiety is anchored in the substrate binding channel and serves as an arginine mimetic (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Here, we report structure–activity relationship (SAR) studies that focus on exploring three regions of the scaffold represented by compound <b>4</b>. We describe the design, synthesis, and biological evaluation of novel compounds, which resulted in the discovery of a potent, selective, and cell-active dual inhibitor of PRMT4 and PRMT6, <b>17</b> (MS049), and a negative control, <b>46</b> (MS049N).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structure and potency of compounds <b>1</b> (PRMT4 inhibitor), <b>2</b> (PRMT6 selective inhibitor), and <b>3</b> (selective type I PRMT inhibitor). (B) Structure and potency of compound <b>4</b>, a fragment-like inhibitor of type I PRMTs. (C) X-ray cocrystal structure of compound <b>4</b> (cyan) in the complex with PRMT6 (gray) and SAH (green) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS">5EGS</a>). The key H-bond interactions are shown in yellow dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35667" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35667" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Design and Synthesis</h3><div class="NLM_p">To improve potency and selectivity of <b>4</b>, we investigated the following three regions: (1) the right-hand side (RHS) piperidinylethanamine moiety, (2) the middle linker moiety, and (3) the left-hand side (LHS) aromatic moiety (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B).</div><div class="NLM_p">The piperidinylethanamine portion of <b>4</b> contains an ethylenediamino group, which is also present in the PRMT4 inhibitor <b>1</b>,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> PRMT6 selective inhibitor <b>2</b>,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and type I PRMT selective inhibitor <b>3</b>.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The cocrystal structures<a onclick="showRef(event, 'ref47 ref48 ref50 ref57'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref50 ref57">(47, 48, 50, 57)</a> of these inhibitors in complex with PRMT4 or PRMT6 clearly indicate that the ethylenediamino group forms direct and water-mediated hydrogen bonds (H-bonds) with the proteins in the arginine binding pockets, thus serving as an arginine mimetic. Previous SAR studies suggested that the ethylenediamino group is a key contributor to the PRMT4 and PRMT6 inhibitory activities.<a onclick="showRef(event, 'ref47 ref48 ref50 ref57'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref50 ref57">(47, 48, 50, 57)</a> We therefore designed a relatively small set of compounds (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A and <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) to explore the SAR of the piperidinylethanamine region. We focused on investigating substituents on the terminal amino group, the number of carbons between two nitrogen atoms, the substitution position at the piperidinyl ring, and the ring size. Synthetic routes for preparing compounds <b>5</b>–<b>10</b> are outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A. Compounds <b>5</b>–<b>7</b>, <b>9</b>, and <b>10</b> were synthesized via a standard sodium triacetoxyborohydride mediated reductive amination reaction of commercially available benzylpiperidines or benzylpyrrolidines with corresponding <i>tert</i>-butyloxycarbonyl (Boc) protected α- or β-aminoaldehydes, followed by the removal of the Boc protecting group under acidic conditions. Compound <b>8</b> was synthesized using a nucleophilic substitution reaction between benzylpiperidine and Boc protected aminomethylepoxide, followed by a deprotection reaction.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes for Preparing Designed Compounds: Synthesis of Compounds in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> (A), Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> (B), and Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> (C)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, DCM; (b) HCl in MeOH or dioxane, rt, 10–85% over two to four steps; (c) <i>i</i>-PrOH, 50 °C (d) TFAA, DCM, 0 °C to rt, 85% over two steps; (e) TFA, DCM, rt; (f) NaOH, MeOH, H<sub>2</sub>O, 50 °C; (g) NaH, DMF, 0 °C to rt.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the RHS Moiety<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0008.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> values were from ref <a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">48</a>.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate, and the values are presented as the mean ± SD.</p></div></div><div></div></div><div class="NLM_p">The X-ray crystal structure of PRMT6–compound <b>4</b>–<i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH, the cofactor product) complex showed that the middle linker and LHS phenyl moiety of compound <b>4</b> make hydrophobic interactions with PRMT6.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> However, it was not clear what was required for these hydrophobic interactions and whether additional interactions could be made. We therefore explored these two regions to potentially increase H-bond formations and/or polar interactions. For the middle linker, we explored different linker lengths (from no linker to a three-atom linker) as well as different linker types including various heteroatoms such as O, S, and NH (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B and <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). In addition, we investigated linkers containing a hydroxyl methylene, alkyl ether, alkylamine, ester, or amide functional group (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B and <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Synthetic routes for preparing <b>11</b>–<b>23</b> are outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B. Compounds <b>11</b>–<b>18</b> and <b>20</b>–<b>23</b> were synthesized utilizing reductive amination reactions of commercially available substituted piperidines with <i>N</i>-Boc (methylamino)acetaldehyde followed by the removal of the protecting group. Compound <b>19</b> was prepared by using a six-step synthetic sequence. First, reductive amination of aniline with Boc protected piperidine-4-carboxaldehyde provided Boc protected piperidinylmethylaniline, the aniline nitrogen of which was subsequently protected with a trifluoroactyl group. Selective removal of the Boc protecting group produced intermediate <b>I1</b>, which underwent another reductive amination reaction. Sequential removal of the protecting groups afforded the desired compound <b>19</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the Middle Linker<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0009.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate, and the values are presented as the mean ± SD.</p></div></div><div></div></div><div class="NLM_p">To investigate SAR of the LHS phenyl moiety, we selected compound <b>17</b> as the starting point because compound <b>17</b> was identified as the most potent dual inhibitor of PRMT4 and PRMT6 from our SAR studies on the middle linker region (see <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> and discussion below). We designed compounds <b>24</b>–<b>45</b> to explore a number of electron donating and withdrawing groups with different steric bulkiness at the 2-, 3-, or 4-postion at the phenyl ring (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>C and <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). These compounds were synthesized using a four-step parallel synthetic sequence. Nucleophilic substitution reactions of Boc protected 4-hydroxypiperidine with various substituted benzyl chlorides or bromides and the subsequent deprotection afforded 4-benzyloxypiperidine intermediates <b>I2</b>, which were then converted to the desired compounds <b>24</b>–<b>45</b> via reduction amination with <i>N</i>-Boc (methylamino)acetaldehyde, followed by the removal of the Boc protecting group.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the LHS Phenyl Moiety<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0010.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate, and the values are presented as the mean ± SD.</p></div></div><div></div></div><div class="NLM_p last">All of the synthesized compounds were evaluated in human PRMT4 and PRMT6 biochemical assays, which measure the potency of test compounds at inhibiting the transfer of the methyl group from <sup>3</sup>H-SAM to the peptide substrates. IC<sub>50</sub> values of the compounds are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> SAR of the RHS Piperidinylethanamine Moiety</h3><div class="NLM_p last">The X-ray crystal structure of <b>4</b> in complex with PRMT6 and SAH revealed that two nitrogen atoms of the piperidinylethanamine moiety formed direct and water mediated H-bonds with Glu155, Glu164, and His317 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Consistent with this observation, installation of a small substituent on the terminal primary amino group of <b>4</b>, which does not disturb these H-bond interactions, was well tolerated, while changing the distance between these two nitrogen atoms, which interrupts these interactions, greatly reduced the potency for both PRMT4 and PRMT6. Specifically, switching the terminal primary amino group (<b>4</b>) to a secondary methylamino group provided <b>5</b>, which displayed similar potency (PRMT4, IC<sub>50</sub> = 890 ± 140 nM; PRMT6, IC<sub>50</sub> = 170 ± 39 nM) as compound <b>4</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). On the other hand, a bulkier ethylamino group (<b>6</b>) decreased the potency for PRMT4 and PRMT6 by 3- to 4-fold, compared with <b>4</b>. Increasing the distance between those two nitrogen atoms using 1,3-diaminopropyl group (<b>7</b>) totally abolished the inhibitory activity against both enzymes. Interestingly, adding a free hydroxyl group to the 1,3-diaminopropyl group (<b>8</b>) partially recovered the potency for PRMT4 (IC<sub>50</sub> = 1220 ± 140 nM) and PRMT6 (IC<sub>50</sub> = 3040 ± 270 nM), presumably due to extra H-bond interaction(s). These results confirm that the ethylenediamino group is a preferred moiety for interacting with the arginine binding site. We also explored the piperidinyl region. Switching the 4-benzyl group at the piperidinyl ring to 3-position (<b>9</b>) increased potency for PRMT4 (IC<sub>50</sub> = 120 ± 28 nM) while maintaining potency for PRMT6 (IC<sub>50</sub> = 290 ± 30 nM). Replacement of the six-membered piperidinyl ring with the five-membered pyrrolidinyl ring (<b>10</b>) resulted in high potency for both PRMT4 (IC<sub>50</sub> = 80 ± 28 nM) and PRMT6 (IC<sub>50</sub> = 70 ± 6 nM). Compound <b>10</b> could be a valuable lead for future SAR studies. Because compounds <b>9</b> and <b>10</b> contain a chiral center and would require enantiomeric resolution to obtain pure enantiomers, we chose <b>5</b>, which is achiral, as the starting point to study SAR of the middle linker for the simplicity reason.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> SAR of the Middle Linker</h3><div class="NLM_p last">We next investigated the middle linker region by exploring various linker lengths and introducing various heteroatoms and functional groups. We postulated that changing the linker length could direct the phenyl group to different locations within the binding site, thus affecting inhibitor–protein interactions. Furthermore, heteroatoms such as O and NH and various functional groups could increase polar interactions or H-bond formations and in turn improve potency for PRMT4 and PRMT6. Without the methylene linker, <b>11</b> displayed 7-fold and 55-fold potency loss against PRMT4 and PRMT6, respectively, compared with <b>5</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Among analogs with a one-atom linker, thioether (<b>13</b>) and hydroxymethine (<b>15</b>) linkers slightly impaired the potency, while amino linker (<b>14</b>) showed similar potency for PRMT4 (IC<sub>50</sub> = 616 ± 70 nM) and PRMT6 (IC<sub>50</sub> = 186 ± 23 nM) as compound <b>5</b>. Interestingly, with an ether linker, <b>12</b> displayed higher potency (∼6-fold) for PRMT4 (IC<sub>50</sub> = 149 ± 24 nM) and similar potency for PRMT6 (IC<sub>50</sub> = 90 ± 16 nM) as compared to <b>5</b>. Except for an amide-linker containing compound (<b>22</b>), the synthesized compounds with a two-atom linker, such as ethylene (<b>16</b>), ethers (<b>17</b> and <b>18</b>), methyleneamine (<b>19</b>), ester (<b>20</b>), and amide (<b>21</b>), displayed higher potency for PRMT4, resulting in inhibitors with almost equal potency for PRMT4 and PRMT6. Among all derivatives, <b>17</b> was the most potent dual inhibitor of PRMT4 (IC<sub>50</sub> = 34 ± 10 nM) and PRMT6 (IC<sub>50</sub> = 43 ± 7 nM). Compared with <b>5</b>, <b>17</b> was significantly more potent (∼26-fold) for PRMT4 and had moderately improved potency (∼4-fold) for PRMT6. The compound with a three-atom ester-linker (<b>23</b>), however, was less potent than compound <b>18</b>, which has a two-atom ether-linker, and compound <b>20</b>, which has a two-atom ester-linker. Overall, a linker with two atoms between the phenyl and piperidinyl groups is preferred and the methyleneoxy linker in compound <b>17</b> yields the highest potency for PRMT4 and PRMT6. Thus, compound <b>17</b> was used as the starting scaffold to investigate the LHS phenyl moiety.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> SAR of the LHS Phenyl Moiety</h3><div class="NLM_p last">To examine the effect of substitution at the phenyl moiety of <b>17</b>, a number of electron-withdrawing groups, such as F, Cl, Br, and CF<sub>3</sub>, and electron donating groups, such as phenyl, were installed at the 2-, 3-, or 4-position of the phenyl ring. SAR results of these substituted phenyl compounds (<b>24</b>–<b>38</b>) showed that both electron-deficient and electron-rich groups were well tolerated (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). With the exception of <b>33</b>, 4-substituted phenyl groups (<b>26</b>, <b>29</b>, <b>32</b>, and <b>38</b>) slightly improved (by about 2-fold) potency for PRMT4, compared with <b>17</b>. We therefore explored a few additional substituents (<b>39</b>–<b>45</b>) at this position. Similar to the substituents in <b>24</b>–<b>38</b>, these additional substituents (in <b>39</b>–<b>45</b>) were also well tolerated. In general, substitution at the phenyl ring did not affect PRMT4 and PRMT6 potency drastically. Several compounds (e.g., <b>30</b>, <b>38</b>, <b>45</b>) exhibited about a 2-fold potency improvement over <b>17</b> and could serve as leads for future SAR studies. Because compound <b>17</b> has the simplest structure, we selected it for full characterization in the following studies.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Characterization of <b>17</b> in Selectivity and Biophysical and Mechanism of Action Studies</h3><div class="NLM_p">We evaluated selectivity of compound <b>17</b> and its two close analogs (<b>25</b> and <b>37</b>) against other PRMTs (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). We were pleased to find none of these compounds showed inhibition against type II (PRMT5) and type III (PRMT7) PRMTs. In addition, all three compounds were more potent against PRMT4 and PRMT6 than other type I PRMTs. For example, <b>17</b> and <b>25</b> had good selectivity over PRMT8 (>30-fold) and excellent selectivity over PRMT1 and PRMT3 (>300-fold). To further assess selectivity of <b>17</b>, we tested it against 25 protein lysine methyltransferases (PKMTs), DNA and RNA methyltransferases (DNMTs/RNMTs), three histone lysine demethylases (KDMs), and nine methyllysine and methylarginine reader proteins. We found that <b>17</b> did not inhibit any of the methyltransferases at 50 μM (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>) or demethylases (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>) at 10 μM and did not display appreciable binding to any of the methyllysine/methylarginine reader proteins using the DSF assay at 200 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). Moreover, we also evaluated the selectivity of <b>17</b> over a broad range of nonepigenetic targets in two selectivity profiling panels: CEREP and PDSP.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The CEREP platforms cover 100 GPCRs, ion channels, transporters, and kinases. Compound <b>17</b> showed no appreciable inhibition (no more than 50% inhibition at 10 000 nM) in this panel (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). The PDSP selectivity panel consists of 44 GPCRs. Compound <b>17</b> showed no more than 50% inhibition at 10 000 nM against 41 targets and >50% inhibition at 10 000 nM against 3 targets in the panel. <i>K</i><sub>i</sub> determinations in the radioligand binding assay for each of the 3 interacting targets were subsequently performed. Compound <b>17</b> had <i>K</i><sub>i</sub> values of 64 nM, 87 nM, and 574 nM for σ1, histamine H3, and σ2, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Table S3</a>). However, it is not clear whether <b>17</b> has activity in σ1, histamine H3, and σ2 functional assays. Overall, <b>17</b> demonstrated excellent selectivity over a broad range of epigenetic and nonepigenetic targets.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selectivity of <b>17</b> against methyltransferases. Selectivity of <b>17</b> against a panel of 25 PKMTs and DNMTs was determined at two compound concentrations of 10 μM (black) and 50 μM (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. PRMT Selectivity of Compounds <b>17</b>, <b>25</b>, and <b>37</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="7" align="center">IC<sub>50</sub> (nM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">PRMT1</th><th class="colsep0 rowsep0" align="center">PRMT3</th><th class="colsep0 rowsep0" align="center">PRMT4</th><th class="colsep0 rowsep0" align="center">PRMT5</th><th class="colsep0 rowsep0" align="center">PRMT6</th><th class="colsep0 rowsep0" align="center">PRMT7</th><th class="colsep0 rowsep0" align="center">PRMT8</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">>13000</td><td class="colsep0 rowsep0" align="left">>22000</td><td class="colsep0 rowsep0" align="left">34 ± 10</td><td class="colsep0 rowsep0" align="left">NI</td><td class="colsep0 rowsep0" align="left">43 ± 7</td><td class="colsep0 rowsep0" align="left">NI</td><td class="colsep0 rowsep0" align="left">1600 ± 81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">>16000</td><td class="colsep0 rowsep0" align="left">>28000</td><td class="colsep0 rowsep0" align="left">41 ± 6</td><td class="colsep0 rowsep0" align="left">NI</td><td class="colsep0 rowsep0" align="left">56 ± 7</td><td class="colsep0 rowsep0" align="left">NI</td><td class="colsep0 rowsep0" align="left">1850 ± 160</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">>5800</td><td class="colsep0 rowsep0" align="left">1970 ± 170</td><td class="colsep0 rowsep0" align="left">27 ± 9</td><td class="colsep0 rowsep0" align="left">NI</td><td class="colsep0 rowsep0" align="left">30 ± 5</td><td class="colsep0 rowsep0" align="left">NI</td><td class="colsep0 rowsep0" align="left">573 ± 61</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate, and the values are presented as the mean ± SD. NI: no inhibition.</p></div></div></div><div class="NLM_p">We next confirmed binding of <b>17</b> to PRMT4 using isothermal titration calorimetry (ITC) in the presence of SAH with a <i>K</i><sub>d</sub> value of 100 nM (<i>n</i> = 2) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). In the case of PRMT6 binding was confirmed by ITC in the presence of SAM with a <i>K</i><sub>d</sub> value of 87 ± 35 nM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Binding of <b>17</b> to PRMT4 and PRMT6 was confirmed by another orthogonal method, differential scanning fluorimetry (DSF).<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> In DSF experiments, an increase in melting temperature (Δ<i>T</i><sub>m</sub>) of PRMT4 (3.3 °C) and PRMT6 (11.3 °C) was observed upon binding of <b>17</b> at 200 μM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding confirmation and MOA of <b>17</b>. (A) Isothermal titration calorimetry (ITC) was used to confirm binding of <b>17</b> to PRMT4 with a <i>K</i><sub>d</sub> of 100 nM (B) and to PRMT6 with a <i>K</i><sub>d</sub> of 87 ± 35 nM. ITC experiments for PRMT4 and PRMT6 were performed in duplicate and triplicate, respectively. Differential scanning fluorimetry (DSF) was also used to confirm the binding of <b>17</b> to PRMT4 (C) and PRMT6 (D). Experiments were performed (●) in the absence of any compound as a negative control and in the presence of (○) 100 μM SAH as a positive control with PRMT4 (Δ<i>T</i><sub>m</sub> of 4 °C) and PRMT6 (Δ<i>T</i><sub>m</sub> of 8.4 °C). Presence of (▲) 200 μM <b>17</b> resulted in stabilization of both proteins with Δ<i>T</i><sub>m</sub> of 3.3 and 11 °C for PRMT4 and PRMT6, respectively. Presence of (□) 100 μM SAM plus 200 μM <b>17</b> had the highest effect with Δ<i>T</i><sub>m</sub> of 13 and 21 °C for PRMT4 and PRMT6, respectively. The inflection point of each transition curve is considered melting temperature (<i>T</i><sub>m</sub>) and the increase in <i>T</i><sub>m</sub> is an indication of binding. The IC<sub>50</sub> values for <b>17</b> with PRMT4 and PRMT6 were unchanged when determined under various peptide (E) and SAM (F) concentrations, suggesting an apparent noncompetitive pattern of inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To assess the mechanism of action (MOA) of <b>17</b>, we evaluated the effect of SAM and peptide concentrations on IC<sub>50</sub> values of <b>17</b> against PRMT4 and PRMT6. As illustrated in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>E and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>F, increasing the cofactor SAM or peptide substrate concentrations had no effects on IC<sub>50</sub> values of <b>17</b> for either PRMT4 or PRMT6, suggesting that this inhibitor is noncompetitive with both the cofactor SAM and peptide substrate. This is not surprising, as similar noncompetitive inhibition phenomena were observed for arginine binding site occupying PRMT inhibitors, such as compounds <b>2</b>, <b>3</b>, and <b>4</b>.<a onclick="showRef(event, 'ref47 ref48 ref57'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref57">(47, 48, 57)</a> Considering its similar chemical structure to <b>4</b>, compound <b>17</b> most likely occupies the substrate (arginine) binding site of PRMT4 and PRMT6. It is likely that the binding affinity of the peptide substrate mainly derived from the interactions outside of the arginine-binding site. Thus, <b>17</b> cannot compete off the peptide substrate.<a onclick="showRef(event, 'ref47 ref48 ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref63 ref64">(47, 48, 63, 64)</a> Another plausible interpretation is that the binding of <b>17</b> induces major protein conformational changes and traditional enzyme kinetics may not apply.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cell-Based Studies</h3><div class="NLM_p">We first assessed the effect of <b>17</b> on inhibition of ectopically expressed PRMT6 in HEK293 cells. Overexpression of the wild type (WT) PRMT6 but not catalytic mutant (V86/D88A) robustly increased the asymmetric dimethylation of histone H3 arginine 2 (H3R2me2a) level (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A, first two columns).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Compound <b>17</b> (20 h exposure) reduced the H3R2me2a mark in HEK293 cells in a concentration dependent manner (IC<sub>50</sub> = 0.97 ± 0.05 μM (<i>n</i> = 3)) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). The effect of the 8 μM <b>17</b> treatment matched with that of the catalytically inactive mutant (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). Recently, Med12 was identified as a non-histone substrate of PRMT4.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Compound <b>17</b> treatment (72 h exposure) inhibited endogenous PRMT4 methyltransferase activity in a concentration dependent manner resulting in reduced levels of cellular asymmetric arginine dimethylation of Med12 (Med12-Rme2a, IC<sub>50</sub> = 1.4 ± 0.1 μM (<i>n</i> = 3)) in HEK293 cells (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). To further confirm the cellular assay results, we designed and synthesized a negative control compound, <b>46</b>, by replacing the methyl amino group with the methyl ether group on the RHS of <b>17</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Scheme 1</a>). This single atom switch (from nitrogen to oxygen) completely abolished inhibitory activity against PRMT4 and PRMT6 in vitro (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). As expected, <b>46</b> did not decrease cellular levels of H3R2me2a or Med12me2a at concentrations up to 30 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Figure S2A and Figure S2B</a>). It is worth noting that in our experiments further characterizing compound <b>17</b>, we observed that <b>17</b> unexpectedly reduced asymmetric dimethylation of histone H4 arginine 3 (H4R3me2a) in HEK293 cells while compound <b>46</b> did not (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Figure S2C</a>). It was shown previously that PRMT1 is the major isoform responsible for methylation of H4R3; however PRMT6 is also able to methylate H4R3 in vitro<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and in cells when overexpressed.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> It is unclear what contributes to the inconsistency between the biochemical and cellular assay results, which will be a subject of future investigations. Lastly, to evaluate cell toxicity of <b>17</b> and <b>46</b>, we investigated effects of <b>17</b> and <b>46</b> on growth inhibition of HEK293 cells over 4 days (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">Supporting Information Figure S3</a>). Both compounds did not display significant cell toxicity at up to 50 μM within the time of the treatment.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of <b>17</b> on inhibiting PRMT6 and PRMT4 in cells. (A) Compound <b>17</b> inhibits PRMT6 methyltransferase activity in HEK293 cells. HEK293 cells were transfected with FLAG-tagged PRMT6 (wt) or its catalytically inactive mutant V86K/D88A (mut) and treated with <b>17</b> at indicated concentrations for 20 h. H3R2me2a levels were determined by Western blot. The graphs represent nonlinear fits of H3R2me2a signal intensities normalized to total histone H3. The results are the mean ± SEM of three replicates. (B) Compound <b>17</b> inhibits endogenous PRMT4 methyltransferase activity in HEK293 cells. HEK293 cells were treated with <b>17</b> at indicated concentrations for 72 h, and Med12-Rme2a levels were determined by Western blot. The graphs represent nonlinear fits of Med12-Rme2a signal intensities normalized to total Med12. The results are the mean ± SEM of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>46</b>, a close analog of compound <b>17</b>, has no inhibitory activity against PRMT4 and PRMT6. NI: no inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07260" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07260" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We conducted SAR studies and explored three regions of the fragment-like inhibitor of PRMTs, <b>4</b>. By optimizing the three regions of this hit, we discovered compound <b>17</b> as a potent, selective, and cell-active dual inhibitor of PRMT4 and PRMT6. In biochemical assays, <b>17</b> was highly potent and selective for PRMT4 and PRMT6 over a broad range of epigenetic modifiers, including other PRMTs, PKMTs, DNMTs, KDMs, and methyllysine/methylarginine reader proteins and nonepigenetic targets, including GPCRs, ion channels, transporters, and kinases. In cellular assays, <b>17</b> potently inhibited the methyltransferase activity of PRMT4 and PRMT6 and reduced levels of Med12me2a and H3R2me2a in HEK293 cells. We also discovered <b>46</b>, a close analog of <b>17</b>, which was inactive in biochemical and cellular assays, as a negative control for chemical biology studies. Both <b>17</b> and <b>46</b> were not toxic and did not affect the growth of HEK293 cells. Therefore, <b>17</b> and <b>46</b> are valuable chemical tools for the biomedical community to test biological and therapeutic hypotheses concerning PRMT4 and PRMT6 in cellular systems.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50645" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50645" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="secz" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry. General Procedures</h3><div class="NLM_p last">HPLC spectra for all compounds were acquired using an Agilent 1200 series system with DAD detector. Chromatography was performed on a 2.1 mm × 150 mm Zorbax 300SB-C<sub>18</sub> 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). High-resolution mass spectra (HRMS) data were acquired in positive ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were acquired on either a Bruker DRX-600 spectrometer (600 MHz <sup>1</sup>H, 150 MHz <sup>13</sup>C) or a Bruker Avance-III 500 MHz spectrometer (500 MHz <sup>1</sup>H, 125 MHz <sup>13</sup>C). Chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with UV detector set to 254 nm. Samples were injected into a Phenomenex Luna 75 mm × 30 mm, 5 μm, C<sub>18</sub> column at room temperature. The flow rate was 40 mL/min. A linear gradient was used with 10% (or 50%) of MeOH (A) in H<sub>2</sub>O (with 0.1% TFA) (B) to 100% of MeOH (A). HPLC was used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC methods described above. Because compound <b>20</b> was not very stable when treated with HCl, the free base form was used in biological assays. All of the rest of compounds were tested in biological assays using their HCl salt forms.</div></div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedure A for Reductive Amination (Step a)</h3><div class="NLM_p last">To a solution of the amine (1 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (0.13 M solution) was added aldehyde (1.1 equiv). The resulting solution was stirred at room temperature for 15 min prior to the addition of sodium triacetoxyborohydride (1.4 equiv), after which the solution was stirred for an additional 16 h. The reaction was then quenched with aqueous sodium bicarbonate solution and extracted with EtOAc (3×). The combined organic extracts were dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then purified by silica gel column chromatography to afford the Boc protected amine. (Note: When the amine starting material is in the HCl salt form, Et<sub>3</sub>N (1.1 equiv) was added to the reaction before aldehyde addition.)</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure B for Boc Deprotection (Step b)</h3><div class="NLM_p last">To a solution of the Boc protected amine (1 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (0.20 M solution) was added HCl/dioxane (4 M solution, 10 equiv). The resulting solution was stirred at room temperature for 2 h prior to removal of all solvents under reduced pressure and dissolution of the residue in water. The aqueous solution was frozen and lyophilized for 16 h to afford the hydrochloride salt. Alternatively, HCl/MeOH (3 M solution, 10 equiv) was used to remove the Boc protecting group at rt for 12 h.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure C for Benzyl Ether Formation (Step g)</h3><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate (1 equiv) in DMF (0.13 M solution) was added NaH (60% dispersion in mineral oil, 2 equiv) in portions at 0 °C. After 15 min, a solution of benzyl bromide or chloride (2 equiv) in DMF (1 M solution) was added dropwise. The resulting mixture was slowly warmed to room temperature and stirred overnight, before being quenched with water. The reaction was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3×). The combined organic extracts were dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The resulting residue was purified by silica gel column chromatography to afford the desired benzyl ether.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedure D for Converting Compounds <b>17</b> and <b>24</b>–<b>46</b> to Free Base Forms</h3><div class="NLM_p last">The <sup>1</sup>H NMR of the HCl salt contains different sets of signals, presumably due to different salt forms. A small portion of the HCl salt was converted to free base by washing with aqueous saturated NaHCO<sub>3</sub> and extraction with EtOAc. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The resulting residue was used directly to obtain NMR spectra. Note: The HPLC–MS spectra were identical for the HCl salt form and free base form. In addition, the HCl salt form and free-base form of compound <b>17</b> showed identical results in the biochemical assay.</div></div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 2-(4-Benzylpiperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>5</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (68% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.90 (br s, 1H), 10.18 (br s, 2H), 7.30–7.24 (m, 2H), 7.20–7.13 (m, 3H), 3.74–3.65 (m, 1H), 3.53–3.45 (m, 1H), 2.81–2.76 (m, 2H), 2.53–2.47 (m, 2H), 2.30 (t, <i>J</i> = 6.5 Hz, 2H), 2.26 (s, 3H), 1.85–1.75 (m, 2H), 1.54–1.48 (m, 2H), 1.48–1.41 (m, 1H), 1.22–1.10 (m, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 139.9, 129.4 (2), 128.8 (2), 126.5, 52.7, 52.2 (2), 42.7, 41.9, 35.3, 32.7, 29.1 (2). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>25</sub>N<sub>2</sub>, 233.2018; found, 233.2019.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 2-(4-Benzylpiperidin-1-yl)-<i>N</i>-ethylethan-1-amine (<b>6</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/MeOH) (64% over two steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.29 (t, <i>J</i> = 7.5 Hz, 2H), 7.23–7.13 (m, 3H), 3.66 (d, <i>J</i> = 12.2 Hz, 2H), 3.55–3.48 (m, 2H), 3.46 (t, <i>J</i> = 6.6 Hz, 2H), 3.15 (q, <i>J</i> = 7.2 Hz, 2H), 3.07–2.96 (m, 2H), 2.62 (d, <i>J</i> = 6.8 Hz, 2H), 1.96–1.84 (m, 3H), 1.74–1.59 (m, 2H), 1.36 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>27</sub>N<sub>2</sub>, 247.2174; found, 247.2176.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 3-(4-Benzylpiperidin-1-yl)-<i>N</i>-methylpropan-1-amine (<b>7</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/MeOH) (68% over two steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.29 (t, <i>J</i> = 7.6 Hz, 2H), 7.23–7.14 (m, 3H), 3.59 (d, <i>J</i> = 12.2 Hz, 2H), 3.20 (t, <i>J</i> = 8.1 Hz, 2H), 3.11 (t, <i>J</i> = 7.8 Hz, 2H), 2.97 (t, <i>J</i> = 12.8 Hz, 2H), 2.74 (s, 3H), 2.62 (d, <i>J</i> = 6.6 Hz, 2H), 2.24–2.11 (m, 2H), 1.98–1.82 (m, 3H), 1.66–1.51 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>27</sub>N<sub>2</sub>, 247.2174; found, 247.2165.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 1-(4-Benzylpiperidin-1-yl)-3-(methylamino)propan-2-ol (<b>8</b>)</h3><div class="NLM_p last">To a solution of 4-benzylpiperidine (0.053 g, 0.3 mmol) in <i>i</i>-PrOH (5 mL) was added <i>tert</i>-butyl methyl(oxiran-2-ylmethyl)carbamate (0.067 g, 0.36 mmol). The reaction solution was heated at 80 °C for 16 h before being concentrated and passed through a short silica gel plug. After concentration under reduced pressure, the Boc protecting group was removed using general procedure B (HCl/MeOH) (53% over two steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.29 (t, <i>J</i> = 7.6 Hz, 2H), 7.23–7.17 (m, 3H), 4.47–4.39 (m, 1H), 3.72–3.65 (m, 1H), 3.64–3.59 (m, 1H), 3.24–3.13 (m, 3H), 3.10–2.93 (m, 3H), 2.75 (s, 3H), 2.62 (d, <i>J</i> = 6.8 Hz, 2H), 1.96–1.84 (m, 3H), 1.70–1.53 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>27</sub>N<sub>2</sub>O, 263.2123; found, 263.2122.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 2-(3-Benzylpiperidin-1-yl)-<i>N</i>-methylethan-1-amine ((±)-<b>9</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (77% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.90 (br s, 1H), 9.53 (br s, 2H), 7.35–7.29 (m, 2H), 7.26–7.20 (m, 1H), 7.20–7.16 (m, 2H), 3.74–3.65 (dm, 1H), 3.57–3.40 (m, 6H), 2.90–2.80 (m, 1H), 2.80–2.70 (m, 1H), 2.65–2.45 (dm, 1H), 2.57 (s, 3H), 2.25–2.15 (m, 1H), 1.85–1.75 (m, 2H), 1.70–1.61 (m, 1H), 1.20–1.06 (m, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 138.9, 129.5 (2), 128.9 (2), 126.8, 57.2, 53.6, 52.4, 42.6, 39.1 (hidden by solvent), 35.6, 32.7, 27.8, 22.5. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>25</sub>N<sub>2</sub>, 233.2018; found, 233.2017.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 2-(3-Benzylpyrrolidin-1-yl)-<i>N</i>-methylethan-1-amine ((±)-<b>10</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (80% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.40 (br s, 1H), 9.42 (br s, 2H), 7.35–7.29 (m, 2H), 7.28–7.20 (m, 3H), 3.75–3.65 (m, 1H), 3.65–3.55 (m, 1H), 3.55–3.45 (m, 3H), 3.35–3.10 (m, 3H), 2.90–2.66 (m, 3H), 2.58 (s, 3H), 2.20–1.95 (m, 1H), 1.85–1.55 (m, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 140.1, 129.1 (2), 129.0 (2), 126.8, 60.7, 58.1, 52.9, 44.1, 44.0, 39.1, 38.4, 32.7. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>23</sub>N<sub>2</sub>, 219.1861; found, 219.1862.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>N</i>-Methyl-2-(4-phenylpiperidin-1-yl)ethan-1-amine (<b>11</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (72% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.99 (br s, 1H), 9.45 (br s, 2H), 7.38–7.32 (m, 2H), 7.28–7.22 (m, 3H), 3.73–3.65 (m, 2H), 3.50–3.52 (m, 4H), 3.20–3.09 (m, 2H), 2.90–2.81 (m, 1H), 2.62 (s, 3H), 2.16–2.06 (m, 2H), 2.04–1.96 (m, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 144.6, 129.1 (2), 127.1, 127.0 (2), 53.0, 52.2 (2), 42.8, 38.9, 32.8, 30.2 (2). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>23</sub>N<sub>2</sub>, 219.1861; found, 219.1859.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-Methyl-2-(4-phenoxypiperidin-1-yl)ethan-1-amine (<b>12</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (60% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.15 (br s, 1H), 9.52 (s, 2H), 7.35–7.28 (m, 2H), 7.08–6.93 (m, 3H), 3.70–3.63 (m, 2H), 3.54–3.43 (m, 5H), 3.27–3.11 (m, 2H), 2.60 (s, 3H), 2.30–2.15 (m, 2H), 2.11–1.92 (m, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.1, 130.1 (2), 121.8, 116.8 (2), 70.5, 51.1, 48.0 (2), 42.7, 32.7, 28.6, 26.7. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>23</sub>N<sub>2</sub>O, 235.1810; found, 235.1811.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-Methyl-2-(4-(phenylthio)piperidin-1-yl)ethan-1-amine (<b>13</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (81% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.26–10.94 (br s, 1H), 9.49 (s, 2H), 7.46–7.42 (m, 2H), 7.40–7.35 (m, 2H), 7.32–7.28 (m, 1H), 3.64–3.55 (m, 2H), 3.55–3.40 (m, 5H), 3.15–3.00 (m, 2H), 2.58 (s, 3H), 2.14–2.04 (m, 2H), 1.98–1.82 (m, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 133.4, 131.6 (2), 129.7 (2), 127.6, 52.4 (2), 51.9, 42.6, 40.0 (hidden by solvent), 32.6, 29.7 (2). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>23</sub>N<sub>2</sub>S, 251.1582; found, 251.1577.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 1-(2-(Methylamino)ethyl)-<i>N</i>-phenylpiperidin-4-amine (<b>14</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (78% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.00–10.80 (br s, 1H), 9.52 (s, 2H), 7.44–7.27 (m, 2H), 7.26–7.03 (m, 3H), 4.95 (br s, 1H), 3.70–3.63 (m, 2H), 3.52–3.45 (m, 1H), 3.45–3.35 (m, 4H), 3.15–3.04 (m, 2H), 2.59 (t, <i>J</i> = 5.0 Hz, 3H), 2.22–2.10 (m, 2H), 2.08–1.94 (m, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 137.7, 130.2 (2), 126.5, 121.6 (2), 66.8, 51.9, 51.3 (2), 42.7, 32.8, 26.8 (2). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>24</sub>N<sub>3</sub>, 234.1970; found, 234.1968.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (1-(2-(Methylamino)ethyl)piperidin-4-yl)(phenyl)methanol (<b>15</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (85% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.46 (br s, 1H), 9.39 (br s, 2H), 7.36–7.29 (m, 4H), 7.29–7.24 (m, 1H), 5.45 (br s, 1H), 4.30 (br s, 1H), 3.74–3.65 (dm, 1H), 3.62–3.54 (m, 2H), 3.54–3.44 (m, 2H), 2.95–2.80 (m, 2H), 2.58 (s, 3H), 1.98–1.90 (m, 1H), 1.80–1.70 (m, 1H), 1.70–1.57 (m, 2H), 1.46–1.37 (m, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 144.4, 128.5 (2), 127.5, 126.9 (2), 66.8, 52.8, 52.6, 52.2, 42.8, 41.0, 32.8, 26.2, 25.3. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>25</sub>N<sub>2</sub>O, 249.1967; found, 249.1966.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i>-Methyl-2-(4-phenethylpiperidin-1-yl)ethan-1-amine (<b>16</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (69% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.77 (br s, 1H), 9.56 (br s, 2H), 7.30–7.25 (m, 2H), 7.24–7.14 (m, 3H), 3.75–3.63 (m, 2H), 3.63–3.43 (m, 7H), 3.00–2.90 (m, 2H), 2.51 (s, 3H), 1.93–1.86 (m, 2H), 1.60–1.45 (m, 4H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 142.4, 128.8 (2), 128.7 (2), 126.2, 52.8 (2), 52.1, 42.7, 37.6, 32.8, 32.7, 32.5, 29.2 (2). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>27</sub>N<sub>2</sub>, 247.2174; found, 247.2172.</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 2-(4-(Benzyloxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>17</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A (with Et<sub>3</sub>N) and B (HCl/MeOH) (64% over two steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.39–7.30 (m, 4H), 7.29–7.22 (m, 1H), 4.55 (s, 2H), 3.54–3.44 (m, 1H), 2.84–2.74 (m, 2H), 2.71 (t, <i>J</i> = 6.8 Hz, 2H), 2.50 (t, <i>J</i> = 6.8 Hz, 2H), 2.41 (s, 3H), 2.28–2.16 (m, 2H), 2.00–1.89 (m, 2H), 1.71–1.59 (m, 2H). <sup>13</sup>C NMR (150 MHz, CD<sub>3</sub>OD): δ 140.2, 129.3, 128.8, 128.6, 75.4, 70.9, 58.2, 52.4, 49.3, 36.1, 32.0. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>25</sub>N<sub>2</sub>O, 249.1967; found, 249.1970.</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-Methyl-2-(4-(phenoxymethyl)piperidin-1-yl)ethan-1-amine (<b>18</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/MeOH) (67% over two steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.26 (t, <i>J</i> = 7.8 Hz, 2H), 6.96–6.88 (m, 3H), 3.91 (d, <i>J</i> = 5.6 Hz, 2H), 3.75 (d, <i>J</i> = 12.0 Hz, 2H), 3.59–3.47 (m, 4H), 3.14 (t, <i>J</i> = 12.5 Hz, 2H), 2.81 (s, 3H), 2.21–2.10 (m, 3H), 1.90–1.77 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>25</sub>N<sub>2</sub>O, 249.1967; found, 249.1969.</div></div><div id="sec4_4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i>-((1-(2-(Methylamino)ethyl)piperidin-4-yl)methyl)aniline (<b>19</b>)</h3><div class="NLM_p last">To a solution of aniline (0.027 mL, 0.3 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) was added <i>tert</i>-butyl 4-formylpiperidine-1-carboxylate (0.64 g, 0.3 mmol). The resulting solution was stirred for 30 min before being treated with sodium triacetoxyborohydride (0.095 g, 0.45 mmol) for 16 h. The reaction was quenched with aqueous saturated NaHCO<sub>3</sub> and the product was extracted with EtOAc (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The resulting residue was passed through a short silica gel plug to get the crude desired product. The resulting intermediate (0.087 g, 0.3 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL). To the solution was added triethylamine (0.084 mL, 0.6 mmol) followed by the addition of trifluoroacetic anhydride (0.054 mL, 0.36 mmol) at 0 °C. The reaction was allowed to warm to rt and stirred for 16 h at rt. The reaction was quenched with aqueous saturated NaHCO<sub>3</sub> and the product was extracted with EtOAc (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The resulting residue was purified by silica gel column to provide <i>tert</i>-butyl 4-((2,2,2-trifluoro-<i>N</i>-phenylacetamido)methyl)piperidine-1-carboxylate (0.098 g, 85% over 2 steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.54–7.42 (m, 3H), 7.36 (d, <i>J</i> = 7.4 Hz, 2H), 4.05 (d, <i>J</i> = 13.4 Hz, 2H), 3.71 (d, <i>J</i> = 7.3 Hz, 2H), 2.86–2.58 (m, 2H), 1.83–1.72 (m, 1H), 1.68 (d, <i>J</i> = 13.3 Hz, 2H), 1.45 (s, 9H), 1.22–1.10 (m, 2H). To a solution of <i>tert</i>-butyl 4-((2,2,2-trifluoro-<i>N</i>-phenylacetamido)methyl)piperidine-1-carboxylate (0.077g, 0.2 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was added trifluoroacetic acid (1 mL). The reaction was stirred for 2 h at rt before being concentrated under reduce pressure. After the resulting residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL), <i>tert</i>-butyl methyl(2-oxoethyl)carbamate (0.035 g, 0.2 mmol) was added. The solution was stirred for 30 min, before triethylamine (0.041 mL, 0.3 mmol) and sodium triacetoxyborohydride (0.064 g, 0.3 mmol) were added. The reaction was stirred for 16 h at rt. The reaction was quenched with aqueous saturated NaHCO<sub>3</sub>, and the product was extracted with EtOAc (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The resulting residue was passed through a short silica gel plug. After evaporation of solvents, the resulting residue was dissolved in methanol (2.5 mL) and aqueous NaOH (5 N, 0.5 mL). The reaction mixture was heated at 50 °C for 2 h before being diluted with EtOAc and water. The mixture was extracted with EtOAc (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The residue was purified by silica gel column to provide <i>tert</i>-butyl methyl(2-(4-((phenylamino)methyl)piperidin-1-yl)ethyl)carbamate. A solution of <i>tert</i>-butyl methyl(2-(4-((phenylamino)methyl)piperidin-1-yl)ethyl)carbamate in 3 N HCl (2 mL) was stirred for 12 h. After concentration of the reaction mixture under reduced pressure, the resulting residue was dissolved in MeOH (0.5 mL). The resulting solution was triturated with ethyl ether to precipitate the desired product as white solid (0.047 g, 66% over 4 steps). <sup>1</sup>H NMR (600 MHz, D<sub>2</sub>O): δ 7.43 (t, <i>J</i> = 7.7 Hz, 2H), 7.37 (t, <i>J</i> = 7.4 Hz, 1H), 7.30 (d, <i>J</i> = 7.9 Hz, 2H), 3.65–3.47 (br s, 2H), 3.40 (s, 4H), 3.31 (d, <i>J</i> = 6.6 Hz, 2H), 3.09–2.88 (m, 2H), 2.67 (s, 3H), 2.09–1.93 (m, 3H), 1.62–1.44 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>26</sub>N<sub>3</sub>, 248.2127; found, 248.2130.</div></div><div id="sec4_4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 1-(2-(Methylamino)ethyl)piperidin-4-yl Benzoate (<b>20</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/MeOH). The product was further purified by prep-HPLC. After evaporation of solvents, the residue was basified with aqueous saturated NaHCO<sub>3</sub> and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to provide desired product as a free base (65% over two steps). Note: Conversion the free base to its HCl salt was attempted but led to slow decomposition. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.06–7.99 (m, 2H), 7.66–7.57 (m, 1H), 7.48 (t, <i>J</i> = 7.8 Hz, 2H), 5.10–4.99 (m, 1H), 2.93 (t, <i>J</i> = 6.3 Hz, 2H), 2.84–2.74 (m, 2H), 2.61 (t, <i>J</i> = 6.3 Hz, 2H), 2.57 (s, 3H), 2.50–2.41 (m, 2H), 2.09–2.00 (m, 2H), 1.93–1.82 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>23</sub>N<sub>2</sub>O<sub>2</sub>, 263.1760; found, 263.1761.</div></div><div id="sec4_4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i>-(1-(2-(Methylamino)ethyl)piperidin-4-yl)benzamide (<b>21</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/MeOH). The product was further purified by prep-HPLC. After evaporation of solvents, the residue was basified with aqueous saturated NaHCO<sub>3</sub> and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The resulting residue was dissolved in 3 mL of MeOH and was treated with 3 N HCl in MeOH (0.5 mL) for 1 h. After concentration, the title compound was obtained as white solid (59% over two steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.83 (d, <i>J</i> = 7.7 Hz, 2H), 7.59–7.50 (m, 1H), 7.47 (t, <i>J</i> = 7.6 Hz, 2H), 4.20 (t, <i>J</i> = 12.2 Hz, 1H), 3.79 (d, <i>J</i> = 12.3 Hz, 2H), 3.59–3.52 (m, 4H), 3.25 (t, <i>J</i> = 12.8 Hz, 2H), 2.81 (s, 3H), 2.30–2.24 (m, 2H), 2.17–2.02 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>24</sub>N<sub>3</sub>O, 262.1919; found, 262.1920.</div></div><div id="sec4_4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 1-(2-(Methylamino)ethyl)-<i>N</i>-phenylpiperidine-4-carboxamide (<b>22</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/dioxane) (65% over two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.94 (br s, 1H), 10.67 (br s, 1H), 10.22 (br s, 2H), 7.57 (d, <i>J</i> = 8.0 Hz, 2H), 7.25–7.20 (m, 2H), 6.99–6.94 (m, 1H), 3.63–3.55 (m, 2H), 3.44–3.29 (m, 6H), 3.04–2.91 (m, 2H), 2.68–2.58 (m, 1H), 2.45–2.42 (m, 1H), 2.10–1.89 (m, 4H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.2, 139.6, 129.1 (2), 123.7, 119.7 (2), 52.3, 52.0 (2), 42.8, 38.7 (hidden by solvent), 32.8, 26.1 (2). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>24</sub>N<sub>3</sub>O, 262.1919; found, 262.1921.</div></div><div id="sec4_4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (1-(2-(Methylamino)ethyl)piperidin-4-yl)methyl Benzoate (<b>23</b>)</h3><div class="NLM_p last">The title compound was prepared according to general procedures A and B (HCl/MeOH). The product was further purified by prep-HPLC. After evaporation of solvents, the residue was basified with aqueous saturated NaHCO<sub>3</sub> and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The resulting residue was dissolved in 3 mL of MeOH and was treated with 3 N HCl in MeOH (0.5 mL) for 1 h. After concentration, the title compound was obtained as white solid (62% over two steps). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.10–8.02 (m, 2H), 7.65–7.60 (m, 1H), 7.49 (t, <i>J</i> = 7.8 Hz, 1H), 4.28 (d, <i>J</i> = 5.8 Hz, 2H), 3.75 (d, <i>J</i> = 12.1 Hz, 2H), 3.58–3.48 (m, 4H), 3.19–3.09 (m, 2H), 2.81 (s, 3H), 2.24–2.15 (m, 1H), 2.15–2.07 (m, 2H), 1.97–1.84 (m, 2H).</div></div><div id="sec4_4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 2-(4-((2-Fluorobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>24</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (35% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.44 (t, <i>J</i> = 7.1 Hz, 1H), 7.36–7.24 (m, 1H), 7.15 (t, <i>J</i> = 7.4 Hz, 1H), 7.06 (t, 1H), 4.60 (s, 2H), 3.58–3.39 (m, 1H), 2.86–2.73 (m, 2H), 2.71 (t, <i>J</i> = 6.8 Hz, 2H), 2.50 (t, <i>J</i> = 6.8 Hz, 2H), 2.41 (s, 3H), 2.30–2.16 (m, 2H), 2.06–1.84 (m, 2H), 1.77–1.58 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 267.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 2-(4-((3-Fluorobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>25</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (38% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.39–7.24 (m, 1H), 7.14 (d, <i>J</i> = 7.6 Hz, 1H), 7.08 (d, <i>J</i> = 9.8 Hz, 1H), 6.98 (t, <i>J</i> = 8.5 Hz, 1H), 4.55 (s, 2H), 3.55–3.39 (m, 1H), 2.89–2.66 (m, 4H), 2.52 (t, <i>J</i> = 6.7 Hz, 2H), 2.46 (s, 3H), 2.33–2.07 (m, 2H), 2.00–1.90 (m, 2H), 1.77–1.58 (m, 2H). <sup>13</sup>C NMR (150 MHz, CD<sub>3</sub>OD): δ 163.7, 162.1, 141.9, 129.7, 122.7, 113.7, 74.2, 68.6, 55.9, 50.8, 47.4, 34.1, 30.6. MS (ESI) <i>m</i>/<i>z</i> 267.2 [M + H]<sup>+</sup>. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>24</sub>FN<sub>2</sub>O, 267.1867; found 267.1874.</div></div><div id="sec4_4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 2-(4-((4-Fluorobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>26</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (44% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.58–7.24 (m, 2H), 7.23–6.72 (m, 2H), 4.51 (s, 2H), 3.64–3.40 (m, 1H), 2.89–2.73 (m, 4H), 2.54 (t, <i>J</i> = 6.6 Hz, 2H), 2.50 (s, 3H), 2.33–2.17 (m, 2H), 2.03–1.86 (m, 2H), 1.72–1.58 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 267.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 2-(4-((2-Chlorobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>27</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (30% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.51 (d, <i>J</i> = 7.1 Hz, 1H), 7.36 (d, <i>J</i> = 8.4 Hz, 1H), 7.32–7.23 (m, 2H), 4.62 (s, 2H), 3.68–3.45 (m, 1H), 2.93–2.71 (m, 4H), 2.54 (t, <i>J</i> = 6.5 Hz, 2H), 2.49 (s, 3H), 2.33–2.21 (m, 2H), 2.04–1.91 (m, 2H), 1.77–1.61 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 283.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 2-(4-((3-Chlorobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>28</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (35% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.36 (s, 1H), 7.33–7.28 (m, 1H), 7.28–7.23 (m, 2H), 4.53 (s, 2H), 3.53–3.43 (m, 1H), 2.85 (t, <i>J</i> = 6.5 Hz, 2H), 2.82–2.74 (m, 2H), 2.54 (t, <i>J</i> = 6.5 Hz, 2H), 2.52 (s, 3H), 2.35–2.18 (m, 2H), 2.01–1.87 (m, 2H), 1.74–1.58 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 283.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 2-(4-((4-Chlorobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>29</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (44% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.32 (s, 4H), 4.52 (s, 2H), 3.52–3.44 (m, 1H), 2.77 (t, <i>J</i> = 6.5 Hz, 4H), 2.51 (t, <i>J</i> = 6.7 Hz, 2H), 2.46 (s, 3H), 2.30–2.16 (m, 2H), 1.99–1.89 (m, 2H), 1.72–1.59 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 283.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 2-(4-((2-Bromobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>30</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (27% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.55 (d, <i>J</i> = 7.9 Hz, 1H), 7.50 (d, <i>J</i> = 7.3 Hz, 1H), 7.34 (t, <i>J</i> = 7.3 Hz, 1H), 7.18 (t, <i>J</i> = 8.2 Hz, 1H), 4.58 (s, 2H), 3.59–3.48 (m, 1H), 2.84–2.69 (m, 4H), 2.52 (t, <i>J</i> = 6.7 Hz, 2H), 2.46 (s, 3H), 2.32–2.18 (m, 2H), 2.04–1.88 (m, 2H), 1.78–1.60 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 327.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 2-(4-((3-Bromobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>31</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (38% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.51 (s, 1H), 7.41 (d, <i>J</i> = 7.8 Hz, 1H), 7.30 (d, <i>J</i> = 7.6 Hz, 1H), 7.24 (t, <i>J</i> = 7.8 Hz, 1H), 4.52 (s, 2H), 3.52–3.43 (m, 1H), 2.78 (t, <i>J</i> = 6.6 Hz, 4H), 2.52 (t, <i>J</i> = 6.6 Hz, 2H), 2.47 (s, 3H), 2.30–2.18 (m, 2H), 1.99–1.90 (m, 2H), 1.72–1.60 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 327.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 2-(4-((4-Bromobenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>32</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (30% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.48 (d, <i>J</i> = 8.3 Hz, 2H), 7.27 (d, <i>J</i> = 8.2 Hz, 2H), 4.51 (s, 2H), 3.53–3.43 (m, 1H), 2.81 (t, <i>J</i> = 6.5 Hz, 4H), 2.53 (t, <i>J</i> = 6.6 Hz, 2H), 2.48 (s, 3H), 2.30–2.17 (m, 2H), 2.04–1.88 (m, 2H), 1.72–1.60 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 327.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>N</i>-Methyl-2-(4-((2-(trifluoromethyl)benzyl)oxy)piperidin-1-yl)ethan-1-amine (<b>33</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (45% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.72 (d, <i>J</i> = 7.7 Hz, 1H), 7.66 (d, <i>J</i> = 7.8 Hz, 1H), 7.61 (t, <i>J</i> = 7.5 Hz, 1H), 7.44 (t, <i>J</i> = 7.6 Hz, 1H), 4.70 (s, 2H), 3.58–3.45 (m, 1H), 2.85–2.77 (m, 2H), 2.74 (t, <i>J</i> = 6.7 Hz, 2H), 2.51 (t, <i>J</i> = 6.7 Hz, 2H), 2.44 (s, 3H), 2.34–2.15 (m, 2H), 2.03–1.88 (m, 2H), 1.77–1.55 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>24</sub>F<sub>3</sub>N<sub>2</sub>O, 317.1835; found 317.1832.</div></div><div id="sec4_4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> <i>N</i>-Methyl-2-(4-((3-(trifluoromethyl)benzyl)oxy)piperidin-1-yl)ethan-1-amine (<b>34</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (50% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.65 (s, 1H), 7.60 (d, <i>J</i> = 7.6 Hz, 1H), 7.56 (d, <i>J</i> = 7.8 Hz, 1H), 7.52 (t, <i>J</i> = 7.7 Hz, 1H), 4.62 (s, 2H), 3.55–3.48 (m, 1H), 2.85–2.75 (m, 2H), 2.72 (t, <i>J</i> = 6.8 Hz, 2H), 2.50 (t, <i>J</i> = 6.8 Hz, 2H), 2.42 (s, 3H), 2.31–2.16 (m, 2H), 2.06–1.89 (m, 2H), 1.80–1.62 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>24</sub>F<sub>3</sub>N<sub>2</sub>O, 317.1835; found 317.1834.</div></div><div id="sec4_4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> <i>N</i>-Methyl-2-(4-((4-(trifluoromethyl)benzyl)oxy)piperidin-1-yl)ethan-1-amine (<b>35</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (44% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.62 (d, <i>J</i> = 8.0 Hz, 2H), 7.53 (d, <i>J</i> = 7.9 Hz, 2H), 4.63 (s, 2H), 3.56–3.46 (m, 1H), 2.88–2.73 (m, 4H), 2.54 (t, <i>J</i> = 6.6 Hz, 2H), 2.49 (s, 3H), 2.34–2.17 (m, 2H), 2.07–1.88 (m, 2H), 1.80–1.60 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>24</sub>F<sub>3</sub>N<sub>2</sub>O, 317.1835; found 317.1841.</div></div><div id="sec4_4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 2-(4-([1,1′-Biphenyl]-2-ylmethoxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>36</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (45% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.53–7.43 (m, 1H), 7.50–7.37 (m, 2H), 7.37–7.29 (m, 5H), 7.27–7.17 (m, 1H), 4.39 (s, 2H), 3.35–3.24 (m, 1H), 2.75–2.60 (m, 4H), 2.46 (t, <i>J</i> = 6.7 Hz, 2H), 2.41 (s, 3H), 2.21–2.03 (m, 2H), 1.84–1.71 (m, 2H), 1.59–1.45 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 325.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 2-(4-([1,1′-Biphenyl]-3-ylmethoxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>37</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (41% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.66–7.54 (m, 3H), 7.52 (d, <i>J</i> = 7.7 Hz, 1H), 7.49–7.36 (m, 3H), 7.36–7.26 (m, 2H), 4.59 (s, 2H), 3.54–3.46 (m, 1H), 2.85–2.73 (m, 4H), 2.51 (t, <i>J</i> = 6.6 Hz, 2H), 2.48 (s, 3H), 2.26–2.15 (m, 2H), 2.01–1.91 (m, 2H), 1.73–1.60 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 325.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 2-(4-([1,1′-Biphenyl]-4-ylmethoxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>38</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (40% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.63–7.56 (m, 4H), 7.45–7.39 (m, 4H), 7.32 (t, <i>J</i> = 7.3 Hz, 1H), 4.59 (s, 2H), 3.57–3.47 (m, 1H), 2.87–2.70 (m, 4H), 2.52 (t, <i>J</i> = 6.7 Hz, 2H), 2.46 (s, 3H), 2.35–2.15 (m, 2H), 2.02–1.89 (m, 2H), 1.76–1.62 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>29</sub>N<sub>2</sub>O, 325.2274; found 325.2269.</div></div><div id="sec4_4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <i>N</i>-Methyl-2-(4-((4-(trifluoromethoxy)benzyl)oxy)piperidin-1-yl)ethan-1-amine (<b>39</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (34% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.44 (d, <i>J</i> = 8.3 Hz, 2H), 7.24 (d, <i>J</i> = 8.2 Hz, 2H), 4.56 (s, 2H), 3.55–3.45 (m, 1H), 2.86–2.73 (m, 2H), 2.70 (t, <i>J</i> = 6.8 Hz, 2H), 2.50 (t, <i>J</i> = 6.8 Hz, 2H), 2.41 (s, 3H), 2.31–2.17 (m, 2H), 2.06–1.90 (m, 2H), 1.75–1.61 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>24</sub>F<sub>3</sub>N<sub>2</sub>O<sub>2</sub>, 333.1784; found 333.1776.</div></div><div id="sec4_4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <i>N</i>-Methyl-2-(4-((4-(2,2,2-trifluoroethoxy)benzyl)oxy)piperidin-1-yl)ethan-1-amine (<b>40</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (21% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.31 (d, <i>J</i> = 8.2 Hz, 2H), 6.98 (d, <i>J</i> = 8.5 Hz, 2H), 4.56–4.44 (m, 4H), 3.49–3.43 (m, 1H), 2.84–2.72 (m, 4H), 2.52 (t, <i>J</i> = 6.6 Hz, 2H), 2.47 (s, 3H), 2.28–2.15 (m, 2H), 1.98–1.89 (m, 2H), 1.70–1.59 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>26</sub>F<sub>3</sub>N<sub>2</sub>O<sub>2</sub>, 347.1941; found 347.1938.</div></div><div id="sec4_4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 2-(4-((4-Isobutylbenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>41</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (20% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.24 (d, <i>J</i> = 7.6 Hz, 2H), 7.12 (d, <i>J</i> = 7.6 Hz, 2H), 4.50 (s, 2H), 3.55–3.40 (m, 1H), 2.82–2.75 (m, 2H), 2.73 (t, <i>J</i> = 6.7 Hz, 2H), 2.50 (t, <i>J</i> = 6.7 Hz, 2H), 2.46 (d, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 2.25–2.17 (m, 2H), 1.97–1.91 (m, 2H), 1.85–1.82 (m, 1H), 1.68–1.59 (m, 2H), 0.89 (d, <i>J</i> = 6.6 Hz, 6H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>33</sub>N<sub>2</sub>O, 305.2587; found 305.2580.</div></div><div id="sec4_4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 2-(4-((4-Cyclohexylbenzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>42</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (10% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.24 (d, <i>J</i> = 7.9 Hz, 2H), 7.17 (d, <i>J</i> = 7.8 Hz, 2H), 4.49 (s, 2H), 3.50–3.43 (m, 1H), 2.85–2.69 (m, 4H), 2.50 (t, <i>J</i> = 6.8 Hz, 2H), 2.44 (s, 3H), 2.29–2.12 (m, 2H), 1.97–1.89 (m, 2H), 1.89–1.78 (m, 4H), 1.78–1.71 (m, 1H), 1.69–1.58 (m, 2H), 1.49–1.35 (m, 4H), 1.36–1.21 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>35</sub>N<sub>2</sub>O, 331.2744; found 331.2745.</div></div><div id="sec4_4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> <i>N</i>-Methyl-2-(4-((4-(<i>tert</i>-pentyl)benzyl)oxy)piperidin-1-yl)ethan-1-amine (<b>43</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (11% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.31 (d, <i>J</i> = 8.0 Hz, 2H), 7.26 (d, <i>J</i> = 8.1 Hz, 2H), 4.51 (s, 2H), 3.51–3.44 (m, 1H), 2.86–2.73 (m, 2H), 2.69 (t, <i>J</i> = 6.9 Hz, 2H), 2.49 (t, <i>J</i> = 6.8 Hz, 2H), 2.40 (s, 3H), 2.27–2.16 (m, 2H), 1.99–1.89 (m, 2H), 1.71–1.59 (m, 4H), 1.27 (s, 6H), 0.65 (t, <i>J</i> = 7.5 Hz, 3H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>35</sub>Cl<sub>2</sub>N<sub>2</sub>O, 319.2744; found 319.2744.</div></div><div id="sec4_4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 2-(4-((4-(2,3-Dichlorophenoxy)benzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>44</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (25% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.40–7.30 (m, 3H), 7.26 (t, <i>J</i> = 8.2 Hz, 1H), 6.97–6.85 (m, 3H), 4.53 (s, 2H), 3.55–3.45 (m, 1H), 2.85–2.76 (m, 2H), 2.72 (t, <i>J</i> = 6.8 Hz, 2H), 2.50 (t, <i>J</i> = 6.8 Hz, 2H), 2.42 (s, 3H), 2.29–2.19 (m, 2H), 2.01–1.89 (m, 2H), 1.73–1.60 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>27</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>, 409.1444; found 409.1439.</div></div><div id="sec4_4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 2-(4-((4-(3,5-Dimethyl-1<i>H</i>-pyrazol-1-yl)benzyl)oxy)piperidin-1-yl)-<i>N</i>-methylethan-1-amine (<b>45</b>)</h3><div class="NLM_p last">The title compound was prepared in four steps according to general procedures C, B (HCl/MeOH), A (with Et<sub>3</sub>N), and B (HCl/MeOH) (25% over four steps). A small portion of the HCl salt was converted to the free base form using general procedure D to obtain the NMR sample. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.50 (d, <i>J</i> = 8.1 Hz, 2H), 7.39 (d, <i>J</i> = 8.0 Hz, 2H), 6.06 (s, 1H), 4.62 (s, 2H), 3.57–3.47 (m, 1H), 2.85–2.76 (m, 2H), 2.74 (t, <i>J</i> = 6.8 Hz, 2H), 2.51 (t, <i>J</i> = 6.8 Hz, 2H), 2.43 (s, 2H), 2.30–2.18 (m, 8H), 2.02–1.91 (m, 2H), 1.75–1.62 (m, 2H). HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>30</sub>N<sub>4</sub>O, 343.2492; found 343.2490.</div></div><div id="sec4_4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 4-(Benzyloxy)-1-(2-methoxyethyl)piperidine (<b>46</b>)</h3><div class="NLM_p last">To a mixture of 4-(benzyloxy)piperidine hydrochloride (0.23 g, 1.0 mmol), potassium carbonate (0.28 g, 2.0 mmol), potassium iodide (0.016 g, 0.1 mmol) in acetonitrile (10 mL) was added 1-bromo-2-methoxyethane (0.15 g, 1.1 mmol). After being refluxed for overnight, the reaction mixture was cooled down to rt and quenched with water. The mixture was extracted with EtOAc (3×). Combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The resulting residue was purified by flash chromatography on silica gel column to provide desired product as yellow oil (20%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.69–7.28 (m, 4H), 7.28–7.22 (m, 1H), 4.53 (s, 2H), 3.51 (t, <i>J</i> = 5.6 Hz, 2H), 3.49–3.42 (m, 1H), 3.32 (s, 3H), 2.88–2.74 (m, 2H), 2.55 (t, <i>J</i> = 5.6 Hz, 2H), 2.32–2.18 (m, 2H), 1.98–1.87 (m, 2H), 1.73–1.59 (m, 2H). <sup>13</sup>C NMR (150 MHz, CD<sub>3</sub>OD): δ 138.7, 127.9, 127.3, 127.1, 73.8, 69.7, 69.4, 57.5, 57.1, 51.0, 30.3. MS (ESI) <i>m</i>/<i>z</i> 250.2 [M + H]<sup>+</sup>. HRMS (ESI-TOF) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>24</sub>NO<sub>2</sub>, 250.1802; found 250.1799.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> PRMT Biochemical Assays</h3><div class="NLM_p last">A radioactivity based scintillation proximity assay (SPA) was used to assess the effect of test compounds on inhibiting the methyl transfer reaction catalyzed by PRMTs as described previously.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The full length human PRMT4 (residues 1–608) and PRMT6 (residues 1–375) were expressed and purified as described previously<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a> and were used in all of the biochemical and biophysical assays. The tritiated <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (<sup>3</sup>H-SAM) obtained from PerkinElmer Life Sciences (catalog no. NET155 V001MC; specific activity range 12–18 Ci/mmol) served as the methyl donor. The (<sup>3</sup>H) methylated biotin labeled peptide would be captured in streptavidin/scintillant-coated scintillation proximity FlashPlates Plus (PerkinElmer Life Sciences) which brings the incorporated <sup>3</sup>H-methyl and the scintillant to close proximity resulting in light emission that is quantified by tracing the radioactivity signal (counts per minute) as measured by a TopCount NXT microplate scintillation and luminescence counter (PerkinElmer Life Sciences). When necessary, nontritiated SAM (AK Scientific, Union City, CA) was used to supplement the reactions. The reaction mixture for the PRMT6 assay (10 μL) contained 20 mM bis-tris-propane (pH 7.5) containing 0.01% Tween-20, 10 mM dithiothreitol (DTT), 25 nM PRMT6, 0.6 μM H4 1–24 peptide (SGRGKGGKGLGKGGAKRHRKVLRDK-biotin) and 2.3 μM SAM. For PRMT4 assay, the reaction mixture contained 20 mM bicine (pH 8.5), 0.01% Triton X-100, 25 nM PRMT4, 0.7 μM H3 1–25 peptide (ARTKQTARKSTGGKAPRKQLATKAAGK-biotin) and 1.9 μM SAM. For the IC<sub>50</sub> determination the compounds were serially diluted in the reaction buffer and were added to the reaction mixture before starting the reactions by addition of the substrate. The reactions were quenched by addition of equal volume of 7.5 M guanidine hydrochloride, and the volume was finalized to 120 μL by the addition of 20 mM Tris-HCl, pH 8.0. After the transfer to the SPA plates (FlashPlate PLUS; PerkinElmer Life Sciences), the plates were incubated for at least 3 h before measuring the signals using a TopCount NXT microplate scintillation and luminescence counter (PerkinElmer Life Sciences). The IC<sub>50</sub> values were determined under balanced conditions at <i>K</i><sub>m</sub> concentrations of both substrate and cofactor by titration of test compounds in the reaction mixture. The <sup>3</sup>H-SAM can be purchased only at low concentration (40–50 μM) in acidic solution which needs to be diluted at least in 1:10 ratio to prepare the final reaction mixture (maximum of 5 μM). All IC<sub>50</sub> values were determined at <i>K</i><sub>m</sub> of substrates. For enzymes that have <i>K</i><sub>m</sub> values higher than 5 μM for SAM (for example, PRMT3) we needed to increase the concentration of SAM by adding unlabeled SAM. Same limitation existed when assessing the competition with SAM in which saturation concentration of SAM was required. In the case of the enzymes with apparent SAM <i>K</i><sub>m</sub> below 5 μM, due to high cost of <sup>3</sup>H-SAM, we still diluted <sup>3</sup>H-SAM with unlabeled SAM as long as the signal-to-noise was reliable for assays.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Selectivity Assays</h3><div class="NLM_p last">The methyltransferase selectivity of <b>17</b> was assessed at two compound concentrations of 10 and 50 μM as described previously.<a onclick="showRef(event, 'ref67 ref68 ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref67 ref68 ref69 ref70">(67-70)</a> The inhibitory effect of <b>17</b> against three lysine demethylases, KDM1A (LSD1), KDM3A (JMJD1A), and KDM4A (JMJD2A) was assessed using the method described by Yu et al.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The selectivity also was tested against 9 reader proteins (UHRF1, BRPF1, WDR5, HGDF2, TDRD3, RBBP1, FXR1, EED, and SND1) using DSF and/or DSLS based assays as described elsewhere.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Selectivity assays of 100 nonepigenetic targets (kinases, GPCRs, ion channels, and transporters) were conducted by CEREP. The 5HT2B radiolabel binding assay was performed by the NIMH Psychoactive Drug Screening Program.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Mechanism of Action (MOA) Studies</h3><div class="NLM_p last">To determine the MOA of the <b>17</b>, the competition of the compound with SAM and peptide substrate was determined individually. In brief, the peptide concentration was kept at saturation and compound potency was monitored at different SAM concentrations (0.5, 1, 4, 8, 12, 16, 20, and 25 × <i>K</i><sub>m</sub>). To test the competition with peptide, the SAM concentration was kept at saturation and IC<sub>50</sub> values were determined at different peptide concentrations (0.5, 1, 2, 4, 8, 12, 16, and 20 × <i>K</i><sub>m</sub>).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Biophysical Assays</h3><div class="NLM_p last">Isothermal titration calorimetry (ITC) was performed in a Nano ITC instrument (TA Instruments, USA) following the procedure described previously.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The ITC titrations were performed at 25 °C by using 2 μL injections with a total of 25 injections. For PRMT6, the cell was loaded with 50 μM PRMT6, 100 μM SAM with final DMSO concentration of 2.5% DMSO in 50 mM Tris-HCl (pH 8.5) containing 150 mM NaCl. The syringe was loaded with 200 μM <b>17</b>, 100 μM SAM, and 2.5% DMSO prepared in the same buffer. For PRMT4, the cell was loaded with 35 μM PRMT4, 120 μM SAH, and 3% DMSO in phosphate buffered saline. The syringe was loaded with 300 μM <b>17</b>, 125 μM SAH, and 3% DMSO prepared in the same buffer. The data were fitted with a one-binding-site model using “Nano Analyze” software supplied by the instrument manufacturer. Differential scanning fluorimetry (DSF) was performed using a Light Cycler 480 II instrument from Roche Applied Science as described previously.<a onclick="showRef(event, 'ref47 ref72'); return false;" href="javascript:void(0);" class="ref ref47 ref72">(47, 72)</a> The experiments were done with 100 mM HEPES buffer (pH 7.4) containing 150 mM NaCl, 0.1 mg/mL PRMT6, and 5X Sypro Orange (Invitrogen). DSF was carried out by increasing the temperature by 4 °C/min from 20 to 95 °C, and data points were collected at 1 °C intervals. The temperature scan curves were fitted to a Boltzmann sigmoid function, and the <i>T</i><sub>m</sub> values were obtained from the midpoint of the transition as described previously.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Differential static light scattering (DSLS) was performed as previously described.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Cellular PRMT4 Assay</h3><div class="NLM_p last">HEK293 cells were grown in 12-well plates in DMEM supplemented with 10% FBS, penicillin (100 units/mL), and streptomycin (100 μg/mL). 30% confluent cells were treated with different concentrations of compounds in triplicates or DMSO control for 72 h. Cells were lysed in 100 μL of total lysis buffer (20 mM Tris-HCl, pH 8, 150 mM NaCl, 1 mM EDTA, 10 mM MgCl<sub>2</sub>, 0.5% TritonX-100, 12.5 U/mL Benzonase (Sigma), complete EDTA-free protease inhibitor cocktail (Roche)). After a 3 min incubation at rt, SDS was added to final 1% concentration. Lysates were run on SDS–PAGE, and immunoblotting was done as outlined below to determine Med12-Rme2a levels in Western blot.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Cellular PRMT6 Assay</h3><div class="NLM_p last">HEK293 cells were grown in 12-well plates in DMEM supplemented with 10% FBS, penicillin (100 U/mL), and streptomycin (100 μg/mL). 50% confluent cells were transfected with FLAG-tagged PRMT6 or mutant V86K/D88A PRMT6 (1 μg of DNA per well) using jetPRIME transfection reagent (Polyplus-Transfection), following manufacturer instructions. After 4 h the medium was removed and cells were treated with compounds at indicated concentrations or DMSO control. After 20 h, the medium was removed and cells were lysed in 100 μL of total lysis buffer. Lysates were run on SDS–PAGE, and immunoblotting was done as outlined below to determine H3R2me2a levels in Western blot.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Western Blot</h3><div class="NLM_p last">Total cell lysates were resolved in 4–12% Bis-Tris protein gels (Invitrogen) with MOPS buffer (Invitrogen) and transferred for 1.5 h (80 V) onto PVDF membrane (Millipore) in Tris-glycine transfer buffer containing 20% MeOH and 0.05% SDS. Blots were blocked for 1 h in blocking buffer (5% milk in 0.1% Tween 20 PBS) and incubated with primary antibodies: mouse anti-H4 (1:1000, Abcam no. 174628), rabbit anti-H4R3me2a (1:1000, Active Motif no. 39705), mouse anti-H3 (1:1000, Abcam no. ab10799), rabbit anti-H3R2me2a (1:1000, Millipore no. 04-808), rabbit anti-Rme2a (1:1000, Cell Signaling Technology no. 13522), mouse anti-Med12 (1:1000, Abnova no. H00009968-A01), rabbit anti-Med12-Rme2a (1:1000, gift from Dr. Mark Bedford) in blocking buffer overnight at 4 °C. After five washes with 0.1% Tween 20 PBS the blots were incubated with goat-anti rabbit (IR800 conjugated, LiCor no. 926-32211) and donkey anti-mouse (IR 680, LiCor no. 926-68072) antibodies (1:5000) in Odyssey blocking buffer (LiCor) for 1 h at rt and washed five times with 0.1% Tween 20 PBS. The signal was read on an Odyssey scanner (LiCor) at 800 and 700 nm.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Cell Growth Assay</h3><div class="NLM_p last">HEK293 cells grown in DMEM supplemented with 10% FBS, penicillin (100 U/mL), and streptomycin (100 μg/mL) were seeded on 96-well plates at density 3000/well and treated with <b>17</b> or <b>46</b> at 0, 1, 10, and 50 μM concentrations for 96 h. The confluency was measured using IncuCyte ZOOM live cell imaging device (Essen Bioscence) and analyzed with IncuCyte ZOOM (2015A) software based on phase contrast images.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01033">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33083" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33083" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01033" class="ext-link">10.1021/acs.jmedchem.6b01033</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">This material is available free of charge via the Internet at <a href="http://pubs.acs,org" class="extLink">http://pubs.acs,org</a>. <sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>17</b>; selectivity of <b>17</b> against KDMs and methyllysine and methylarginine reader proteins, CEREP, and PDS panels; Synthetic scheme of <b>46</b>; effect of cellular assay results of <b>46</b>; rffects of <b>17</b> and <b>46</b> on cell growth (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf">jm6b01033_si_001.pdf (2.39 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_002.csv">jm6b01033_si_002.csv (2.77 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01033" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3d7754535a135154487d504e4e5013585948"><span class="__cf_email__" data-cfemail="d49ebdbab3fab8bda194b9a7a7b9fab1b0a1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#563b7820333237323f162322392439382239783537"><span class="__cf_email__" data-cfemail="8be6a5fdeeefeaefe2cbfeffe4f9e4e5ffe4a5e8ea">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#761c1f1718581c1f18361b05051b58131203"><span class="__cf_email__" data-cfemail="462c2f2728682c2f28062b35352b68232233">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yudao Shen</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena M. Szewczyk</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammad S. Eram</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Smil</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">H. Ümit Kaniskan</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacological Sciences and Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renato Ferreira de Freitas</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guillermo Senisterra</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthieu Schapira</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Princess
Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.S., M.M.S., and M.S.E. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d183e4246-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i73">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07833" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Albina Bolotokova for compound management and Taraneh Hajian for protein purification. The research described here was supported by Grant R01GM103893 (to J.J.) from the U.S. National Institutes of Health. The SGC is a registered charity (No. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD Grant 115766], Janssen, Merck & Co., Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and the Wellcome Trust. 7TM, kinase, and ion channel off-target selectivity screening was kindly supplied by Eurofins-Cerep. Further <i>K</i><sub>i</sub> determinations and receptor binding profiles were generously provided by the National Institute of Mental Health’s Psychoactive Drug Screening Program, Contract HHSN-271-2013-00017-C (NIMH PDSP). The NIMH PDSP is directed by Bryan L. Roth MD, Ph.D., at the University of North Carolina at Chapel Hill and by Project Officer Jamie Driscoll at NIMH, Bethesda MD, U.S. Y.S. was supported by a postdoctoral fellowship from the SGC.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i74" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i74"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i75" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i75"> Abbreviations Used</h2><tr><td class="NLM_term">PMT</td><td class="NLM_def"><p class="first last">protein methyltransferase</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">CARM1</td><td class="NLM_def"><p class="first last">coactivator associated arginine methyltransferase 1</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last">S-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">DSLS</td><td class="NLM_def"><p class="first last">differential static light scattering</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyloxycarbonyl</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">DNMT</td><td class="NLM_def"><p class="first last">DNA methyltransferase</p></td></tr><tr><td class="NLM_term">RNMT</td><td class="NLM_def"><p class="first last">RNA methyltransferase</p></td></tr><tr><td class="NLM_term">KDM</td><td class="NLM_def"><p class="first last">histone lysine demethylase</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">Rme2a</td><td class="NLM_def"><p class="first last">asymmetric arginine dimethylation</p></td></tr><tr><td class="NLM_term">H3R2me2a</td><td class="NLM_def"><p class="first last">asymmetric dimethylation of histone H3 arginine 2</p></td></tr><tr><td class="NLM_term">H4R3me2a</td><td class="NLM_def"><p class="first last">asymmetric dimethylation of histone H4 arginine 3</p></td></tr><tr><td class="NLM_term">H4R3me2a</td><td class="NLM_def"><p class="first last">asymmetric dimethylation of histone H4 arginine 3</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01667" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01667" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 73 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Fuhrmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clancy, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, P. R.</span><span> </span><span class="NLM_article-title">Chemical biology of protein arginine modifications in epigenetic regulation</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">5413</span><span class="NLM_x">–</span> <span class="NLM_lpage">5461</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=5413-5461&author=J.+Fuhrmannauthor=K.+W.+Clancyauthor=P.+R.+Thompson&title=Chemical+biology+of+protein+arginine+modifications+in+epigenetic+regulation&doi=10.1021%2Facs.chemrev.5b00003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology of Protein Arginine Modifications in Epigenetic Regulation</span></div><div class="casAuthors">Fuhrmann, Jakob; Clancy, Kathleen W.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5413-5461</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Here we aim to summarize the current knowledge surrounding the posttranslational modification of histone arginine residues, focusing on enzyme classes that catalyze the citrullination and methylation of arginine residues as well as noncanonical arginine modifications such as phosphorylation, ADP-ribosylation, and arginylation.  The major focus will be given to the protein arginine deiminases and protein arginine methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdzgTmC4C4fbVg90H21EOLACvtfcHk0ljA0ZTZ0lvOsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D&md5=a6df90a99a606e1d0a7296fa95625202</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00003%26sid%3Dliteratum%253Aachs%26aulast%3DFuhrmann%26aufirst%3DJ.%26aulast%3DClancy%26aufirst%3DK.%2BW.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DChemical%2520biology%2520of%2520protein%2520arginine%2520modifications%2520in%2520epigenetic%2520regulation%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D5413%26epage%3D5461%26doi%3D10.1021%2Facs.chemrev.5b00003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Morales, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caceres, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevel, J. M.</span><span> </span><span class="NLM_article-title">Biochemistry and regulation of the protein arginine methyltransferases (PRMTs)</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">590</span><span class="NLM_x">, </span> <span class="NLM_fpage">138</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span><span class="refDoi"> DOI: 10.1016/j.abb.2015.11.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.abb.2015.11.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26612103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVemtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=590&publication_year=2016&pages=138-152&author=Y.+Moralesauthor=T.+Caceresauthor=K.+Mayauthor=J.+M.+Hevel&title=Biochemistry+and+regulation+of+the+protein+arginine+methyltransferases+%28PRMTs%29&doi=10.1016%2Fj.abb.2015.11.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemistry and regulation of the protein arginine methyltransferases (PRMTs)</span></div><div class="casAuthors">Morales, Yalemi; Caceres, Tamar; May, Kyle; Hevel, Joan M.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">590</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-152</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many key cellular processes can be regulated by the seemingly simple addn. of one, or two, Me groups to arginine residues by the nine known mammalian protein arginine methyltransferases (PRMTs).  The impact that arginine methylation has on cellular well-being is highlighted by the ever growing evidence linking PRMT dysregulation to disease states, which has marked the PRMTs as prominent pharmacol. targets.  This review is meant to orient the reader with respect to the structural features of the PRMTs that account for catalytic activity, as well as provide a framework for understanding how these enzymes are regulated.  An overview of what we understand about substrate recognition and binding is provided.  Control of product specificity and enzyme processivity are introduced as necessary but flexible features of the PRMTs.  Precise control of PRMT activity is a crit. component to eukaryotic cell health, esp. given that an arginine demethylase has not been identified.  We therefore conclude the review with a comprehensive discussion of how protein arginine methylation is regulated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpskzNjOitlXrVg90H21EOLACvtfcHk0ljA0ZTZ0lvOsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVemtb7L&md5=009f90f8b06a0e121a5e75ba11185338</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2015.11.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2015.11.030%26sid%3Dliteratum%253Aachs%26aulast%3DMorales%26aufirst%3DY.%26aulast%3DCaceres%26aufirst%3DT.%26aulast%3DMay%26aufirst%3DK.%26aulast%3DHevel%26aufirst%3DJ.%2BM.%26atitle%3DBiochemistry%2520and%2520regulation%2520of%2520the%2520protein%2520arginine%2520methyltransferases%2520%2528PRMTs%2529%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2016%26volume%3D590%26spage%3D138%26epage%3D152%26doi%3D10.1016%2Fj.abb.2015.11.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Lee, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span> </span><span class="NLM_article-title">Minireview: protein arginine methylation of nonhistone proteins in transcriptional regulation</span> <span class="citation_source-journal">Mol. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">433</span><span class="refDoi"> DOI: 10.1210/me.2008-0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1210%2Fme.2008-0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19164444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktlSiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=425-433&author=Y.+H.+Leeauthor=M.+R.+Stallcup&title=Minireview%3A+protein+arginine+methylation+of+nonhistone+proteins+in+transcriptional+regulation&doi=10.1210%2Fme.2008-0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Minireview: protein arginine methylation of nonhistone proteins in transcriptional regulation</span></div><div class="casAuthors">Lee, Young-Ho; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">425-433</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Endocrine regulation frequently culminates in altered transcription of specific genes.  The signal transduction pathways, which transmit the endocrine signal from cell surface to the transcription machinery, often involve posttranslational modifications of proteins.  Although phosphorylation has been by far the most widely studied protein modification, recent studies have indicated important roles for other types of modification, including protein arginine methylation.  Ten different protein arginine methyltransferase (PRMT) family members have been identified in mammalian cells, and numerous substrates are being identified for these PRMTs.  Whereas major attention has been focused on the methylation of histones and its role in chromatin remodeling and transcriptional regulation, there are many nonhistone substrates methylated by PRMTs.  This review primarily focuses on recent progress on the roles of the nonhistone protein methylation in transcription.  Protein methylation of coactivators, transcription factors, and signal transducers, among other proteins, plays important roles in transcriptional regulation.  Protein methylation may affect protein-protein interaction, protein-DNA or protein-RNA interaction, protein stability, subcellular localization, or enzymic activity.  Thus, protein arginine methylation is crit. for regulation of transcription and potentially for various physiol./pathol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWEPQ4itT-8LVg90H21EOLACvtfcHk0liANGnnmwVtAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktlSiu7g%253D&md5=9bfeb72a31dd3115130ce7ccde3bd4c3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1210%2Fme.2008-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2008-0380%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%2BH.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DMinireview%253A%2520protein%2520arginine%2520methylation%2520of%2520nonhistone%2520proteins%2520in%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Endocrinol.%26date%3D2009%26volume%3D23%26spage%3D425%26epage%3D433%26doi%3D10.1210%2Fme.2008-0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Tripsianes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madl, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machyna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fessas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Englbrecht, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neugebauer, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Structural basis for dimethylarginine recognition by the Tudor domains of human SMN and SPF30 proteins</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1414</span><span class="NLM_x">–</span> <span class="NLM_lpage">1420</span><span class="refDoi"> DOI: 10.1038/nsmb.2185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnsmb.2185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=22101937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2gt7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1414-1420&author=K.+Tripsianesauthor=T.+Madlauthor=M.+Machynaauthor=D.+Fessasauthor=C.+Englbrechtauthor=U.+Fischerauthor=K.+M.+Neugebauerauthor=M.+Sattler&title=Structural+basis+for+dimethylarginine+recognition+by+the+Tudor+domains+of+human+SMN+and+SPF30+proteins&doi=10.1038%2Fnsmb.2185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for dimethylarginine recognition by the Tudor domains of human SMN and SPF30 proteins</span></div><div class="casAuthors">Tripsianes, Konstantinos; Madl, Tobias; Machyna, Martin; Fessas, Dimitrios; Englbrecht, Clemens; Fischer, Utz; Neugebauer, Karla M.; Sattler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1414-1420</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Arginine dimethylation plays crit. roles in the assembly of ribonucleoprotein complexes in pre-mRNA splicing and piRNA pathways.  We report soln. structures of SMN and SPF30 Tudor domains bound to sym. and asym. dimethylated arginines (sDMA and aDMA, resp.) that are inherent in the RNP complexes.  An arom. cage in the Tudor domain mediates dimethylarginine recognition by electrostatic stabilization through cation-π interactions.  Distinct from extended Tudor domains, dimethylarginine binding by the SMN and SPF30 Tudor domains is independent of proximal residues in the ligand.  Yet, enhanced micromolar affinities are obtained by external cooperativity when multiple methylation marks are presented in arginine- and glycine-rich peptide ligands.  A hydrogen bond network in the SMN Tudor domain, including Glu134 and a tyrosine hydroxyl of the arom. cage, enhances cation-π interactions and is impaired by a mutation causing an E134K substitution assocd. with spinal muscular atrophy.  Our structural anal. enables the design of an optimized binding pocket and the prediction of DMA binding properties of Tudor domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMSpLGZyOX5bVg90H21EOLACvtfcHk0liANGnnmwVtAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2gt7rK&md5=7d8f06a6d5a907c233fa4d16ae8f71d4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2185%26sid%3Dliteratum%253Aachs%26aulast%3DTripsianes%26aufirst%3DK.%26aulast%3DMadl%26aufirst%3DT.%26aulast%3DMachyna%26aufirst%3DM.%26aulast%3DFessas%26aufirst%3DD.%26aulast%3DEnglbrecht%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DU.%26aulast%3DNeugebauer%26aufirst%3DK.%2BM.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520dimethylarginine%2520recognition%2520by%2520the%2520Tudor%2520domains%2520of%2520human%2520SMN%2520and%2520SPF30%2520proteins%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1414%26epage%3D1420%26doi%3D10.1038%2Fnsmb.2185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Evich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroeva, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germann, M. W.</span><span> </span><span class="NLM_article-title">Effect of methylation on the side-chain pKa value of arginine</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">479</span><span class="NLM_x">–</span> <span class="NLM_lpage">486</span><span class="refDoi"> DOI: 10.1002/pro.2838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fpro.2838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26540340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2js7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=479-486&author=M.+Evichauthor=E.+Stroevaauthor=Y.+G.+Zhengauthor=M.+W.+Germann&title=Effect+of+methylation+on+the+side-chain+pKa+value+of+arginine&doi=10.1002%2Fpro.2838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of methylation on the side-chain pKa value of arginine</span></div><div class="casAuthors">Evich, Marina; Stroeva, Ekaterina; Zheng, Yujun George; Germann, Markus W.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">479-486</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Arginine methylation is important in biol. systems.  Recent studies link the deregulation of protein arginine methyltransferases with certain cancers.  To assess the impact of methylation on interaction with other biomols., the pKa values of methylated arginine variants were detd. using NMR data.  The pKa values of monomethylated, sym. dimethylated, and asym. dimethylated arginine are similar to the unmodified arginine (14.2 ± 0.4).  Although the pKa value has not been significantly affected by methylation, consequences of methylation include changes in charge distribution and steric effects, suggesting alternative mechanisms for recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzjhmzL8VEpLVg90H21EOLACvtfcHk0liANGnnmwVtAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2js7%252FO&md5=0555aa47417940f135b05759e8cd81e5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fpro.2838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2838%26sid%3Dliteratum%253Aachs%26aulast%3DEvich%26aufirst%3DM.%26aulast%3DStroeva%26aufirst%3DE.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DGermann%26aufirst%3DM.%2BW.%26atitle%3DEffect%2520of%2520methylation%2520on%2520the%2520side-chain%2520pKa%2520value%2520of%2520arginine%26jtitle%3DProtein%2520Sci.%26date%3D2016%26volume%3D25%26spage%3D479%26epage%3D486%26doi%3D10.1002%2Fpro.2838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">Arginine methylation: An emerging regulator of protein function</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1016/j.molcel.2005.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2005.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=15866169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=263-272&author=M.+T.+Bedfordauthor=S.+Richard&title=Arginine+methylation%3A+An+emerging+regulator+of+protein+function&doi=10.1016%2Fj.molcel.2005.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation: An emerging regulator of protein function</span></div><div class="casAuthors">Bedford, Mark T.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Arginine methylation is now coming out of the shadows of protein phosphorylation and entering the mainstream, largely due to the identification of the family of enzymes that lay down this modification.  In addn., modification-specific antibodies and proteomic approaches have facilitated the identification of an array of substrates for protein arginine methyltransferases.  This review describes recent insights into the mol. processes regulated by arginine methylation in normal and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDYRpc_643mbVg90H21EOLACvtfcHk0ljzPMTxcbu9gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D&md5=63525cb1e96138be5563fbd897c2bde1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520methylation%253A%2520An%2520emerging%2520regulator%2520of%2520protein%2520function%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D263%26epage%3D272%26doi%3D10.1016%2Fj.molcel.2005.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Wei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mundade, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0ljzPMTxcbu9gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nott, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span> </span><span class="NLM_article-title">Deciphering arginine methylation: Tudor tells the tale</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">629</span><span class="NLM_x">–</span> <span class="NLM_lpage">642</span><span class="refDoi"> DOI: 10.1038/nrm3185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnrm3185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21915143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKqsb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=629-642&author=C.+Chenauthor=T.+J.+Nottauthor=J.+Jinauthor=T.+Pawson&title=Deciphering+arginine+methylation%3A+Tudor+tells+the+tale&doi=10.1038%2Fnrm3185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering arginine methylation: Tudor tells the tale</span></div><div class="casAuthors">Chen, Chen; Nott, Timothy J.; Jin, Jing; Pawson, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">629-642</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Proteins can be modified by post-translational modifications such as phosphorylation, methylation, acetylation and ubiquitylation, creating binding sites for specific protein domains.  Methylation has pivotal roles in the formation of complexes that are involved in cellular regulation, including in the generation of small RNAs.  Arginine methylation was discovered half a century ago, but the ability of methylarginine sites to serve as binding motifs for members of the Tudor protein family, and the functional significance of the protein-protein interactions that are mediated by Tudor domains, has only recently been appreciated.  Tudor proteins are now known to be present in PIWI complexes, where they are thought to interact with methylated PIWI proteins and regulate the PIWI-interacting RNA (piRNA) pathway in the germ line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFKEDr8FDL-bVg90H21EOLACvtfcHk0ljzPMTxcbu9gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKqsb7E&md5=9c5e8c1f0466b7772a2b8d7fc8e7b26a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrm3185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3185%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%26aulast%3DNott%26aufirst%3DT.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DPawson%26aufirst%3DT.%26atitle%3DDeciphering%2520arginine%2520methylation%253A%2520Tudor%2520tells%2520the%2520tale%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2011%26volume%3D12%26spage%3D629%26epage%3D642%26doi%3D10.1038%2Fnrm3185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0ljzPMTxcbu9gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Boffa, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vidali, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allfrey, V. G.</span><span> </span><span class="NLM_article-title">Distribution of Ng,Ng-dimethylarginine in nuclear protein-fractions</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">969</span><span class="NLM_x">–</span> <span class="NLM_lpage">976</span><span class="refDoi"> DOI: 10.1016/0006-291X(77)91613-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2F0006-291X%2877%2991613-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=1977&pages=969-976&author=L.+C.+Boffaauthor=J.+Karnauthor=G.+Vidaliauthor=V.+G.+Allfrey&title=Distribution+of+Ng%2CNg-dimethylarginine+in+nuclear+protein-fractions&doi=10.1016%2F0006-291X%2877%2991613-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2F0006-291X%2877%2991613-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-291X%252877%252991613-8%26sid%3Dliteratum%253Aachs%26aulast%3DBoffa%26aufirst%3DL.%2BC.%26aulast%3DKarn%26aufirst%3DJ.%26aulast%3DVidali%26aufirst%3DG.%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26atitle%3DDistribution%2520of%2520Ng%252CNg-dimethylarginine%2520in%2520nuclear%2520protein-fractions%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1977%26volume%3D74%26spage%3D969%26epage%3D976%26doi%3D10.1016%2F0006-291X%2877%2991613-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1629</span><span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+U.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0liWGWchU8TixA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Yang, Y. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Z.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0liWGWchU8TixA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Schurter, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunick, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harp, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henschen-Edman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aswad, D. W.</span><span> </span><span class="NLM_article-title">Methylation of histone H3 by coactivator-associated arginine methyltransferase 1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">5747</span><span class="NLM_x">–</span> <span class="NLM_lpage">5756</span><span class="refDoi"> DOI: 10.1021/bi002631b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi002631b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitlyntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=5747-5756&author=B.+T.+Schurterauthor=S.+S.+Kohauthor=D.+Chenauthor=G.+J.+Bunickauthor=J.+M.+Harpauthor=B.+L.+Hansonauthor=A.+Henschen-Edmanauthor=D.+R.+Mackayauthor=M.+R.+Stallcupauthor=D.+W.+Aswad&title=Methylation+of+histone+H3+by+coactivator-associated+arginine+methyltransferase+1&doi=10.1021%2Fbi002631b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Histone H3 by Coactivator-Associated Arginine Methyltransferase 1</span></div><div class="casAuthors">Schurter, Brandon T.; Koh, Stephen S.; Chen, Dagang; Bunick, Gerard J.; Harp, Joel M.; Hanson, B. Leif; Henschen-Edman, Agnes; Mackay, Douglas R.; Stallcup, Michael R.; Aswad, Dana W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5747-5756</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The preferential in vitro methylation of histone H3 by coactivator-assocd. arginine methyltransferase 1 (CARM1) has been proposed as a basis for its ability to enhance gene transcription [Chen, D., et al. (1999) Science 284, 2174-2177].  To further evaluate the significance of H3 methylation, we studied the kinetics and site specificity of its modification by CARM1.  Affinity-purified CARM1 methylated recombinant chick H3, which is free of posttranslational modifications, and calf thymus H3, which is heterogeneous with regard to preexisting modifications, equally well, exhibiting a Vmax of 4500 pmol min-1 (mg of enzyme)-1 and an apparent Km for H3 of ≤0.2 μM.  The catalytic efficiency (kcat/Km) of CARM1 toward H3 was at least 1000 times that toward R1 (GGFGGRGGFGG-amide), a highly effective substrate for protein arginine methyltransferase 1.  Peptide mapping of 3H-methyl-labeled H3 indicated methylation at Arg-2, Arg-17, and Arg-26 in the N-terminal region and at one or more of four arginines (128/129/131/134) at the C-terminus.  Two of the N-terminal sites, Arg-17 and Arg-26, occur in the sequence KAXRK and appear to be more efficiently methylated than Arg-2.  CARM1 catalyzed formation of NG,NG-dimethylarginine (asym.) but little or no NG,N'G-dimethylarginine (sym.) and no form of methyllysine.  Amino acid anal. of untreated calf thymus H3 revealed that 3.7% of the mols. naturally contain asym. dimethylarginine and/or monomethylarginine.  Our findings support the hypothesis that methylation of H3 may be involved in the mechanism of transcriptional coactivation by CARM1 of genes whose expression is under the control of nuclear receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyQC65J4FUyrVg90H21EOLACvtfcHk0liWGWchU8TixA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitlyntbc%253D&md5=8ba953dfa829e00a69343610ca6c460d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi002631b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi002631b%26sid%3Dliteratum%253Aachs%26aulast%3DSchurter%26aufirst%3DB.%2BT.%26aulast%3DKoh%26aufirst%3DS.%2BS.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DBunick%26aufirst%3DG.%2BJ.%26aulast%3DHarp%26aufirst%3DJ.%2BM.%26aulast%3DHanson%26aufirst%3DB.%2BL.%26aulast%3DHenschen-Edman%26aufirst%3DA.%26aulast%3DMackay%26aufirst%3DD.%2BR.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DAswad%26aufirst%3DD.%2BW.%26atitle%3DMethylation%2520of%2520histone%2520H3%2520by%2520coactivator-associated%2520arginine%2520methyltransferase%25201%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D5747%26epage%3D5756%26doi%3D10.1021%2Fbi002631b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Ito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furumatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taniguchi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchiya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lotz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asahara, H.</span><span> </span><span class="NLM_article-title">Arginine methyltransferase CARM1/PRMT4 regulates endochondral ossification</span> <span class="citation_source-journal">BMC Dev. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="refDoi"> DOI: 10.1186/1471-213X-9-47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1186%2F1471-213X-9-47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=47&author=T.+Itoauthor=N.+Yadavauthor=J.+Leeauthor=T.+Furumatsuauthor=S.+Yamashitaauthor=K.+Yoshidaauthor=N.+Taniguchiauthor=M.+Hashimotoauthor=M.+Tsuchiyaauthor=T.+Ozakiauthor=M.+Lotzauthor=M.+T.+Bedfordauthor=H.+Asahara&title=Arginine+methyltransferase+CARM1%2FPRMT4+regulates+endochondral+ossification&doi=10.1186%2F1471-213X-9-47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1186%2F1471-213X-9-47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-213X-9-47%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DT.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DFurumatsu%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DS.%26aulast%3DYoshida%26aufirst%3DK.%26aulast%3DTaniguchi%26aufirst%3DN.%26aulast%3DHashimoto%26aufirst%3DM.%26aulast%3DTsuchiya%26aufirst%3DM.%26aulast%3DOzaki%26aufirst%3DT.%26aulast%3DLotz%26aufirst%3DM.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DAsahara%26aufirst%3DH.%26atitle%3DArginine%2520methyltransferase%2520CARM1%252FPRMT4%2520regulates%2520endochondral%2520ossification%26jtitle%3DBMC%2520Dev.%2520Biol.%26date%3D2009%26volume%3D9%26spage%3D47%26doi%3D10.1186%2F1471-213X-9-47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Kawabe, Y. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinnell, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudnicki, M. A.</span><span> </span><span class="NLM_article-title">Carm1 regulates Pax7 transcriptional activity through MLL1/2 recruitment during asymmetric satellite stem cell divisions</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">333</span><span class="NLM_x">–</span> <span class="NLM_lpage">345</span><span class="refDoi"> DOI: 10.1016/j.stem.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.stem.2012.07.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=333-345&author=Y.+I.+Kawabeauthor=Y.+X.+Wangauthor=I.+W.+McKinnellauthor=M.+T.+Bedfordauthor=M.+A.+Rudnicki&title=Carm1+regulates+Pax7+transcriptional+activity+through+MLL1%2F2+recruitment+during+asymmetric+satellite+stem+cell+divisions&doi=10.1016%2Fj.stem.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DKawabe%26aufirst%3DY.%2BI.%26aulast%3DWang%26aufirst%3DY.%2BX.%26aulast%3DMcKinnell%26aufirst%3DI.%2BW.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DRudnicki%26aufirst%3DM.%2BA.%26atitle%3DCarm1%2520regulates%2520Pax7%2520transcriptional%2520activity%2520through%2520MLL1%252F2%2520recruitment%2520during%2520asymmetric%2520satellite%2520stem%2520cell%2520divisions%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2012%26volume%3D11%26spage%3D333%26epage%3D345%26doi%3D10.1016%2Fj.stem.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Vu, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perna, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voza, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueroa, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paietta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deblasio, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimer, S. D.</span><span> </span><span class="NLM_article-title">PRMT4 blocks myeloid differentiation by assembling a methyl-RUNX1-dependent repressor complex</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1625</span><span class="NLM_x">–</span> <span class="NLM_lpage">1638</span><span class="refDoi"> DOI: 10.1016/j.celrep.2013.11.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.celrep.2013.11.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24332853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOgs7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=1625-1638&author=L.+P.+Vuauthor=F.+Pernaauthor=L.+Wangauthor=F.+Vozaauthor=M.+E.+Figueroaauthor=P.+Tempstauthor=H.+Erdjument-Bromageauthor=R.+Gaoauthor=S.+Chenauthor=E.+Paiettaauthor=T.+Deblasioauthor=A.+Melnickauthor=Y.+Liuauthor=X.+Zhaoauthor=S.+D.+Nimer&title=PRMT4+blocks+myeloid+differentiation+by+assembling+a+methyl-RUNX1-dependent+repressor+complex&doi=10.1016%2Fj.celrep.2013.11.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT4 Blocks Myeloid Differentiation by Assembling a Methyl-RUNX1-Dependent Repressor Complex</span></div><div class="casAuthors">Vu, Ly P.; Perna, Fabiana; Wang, Lan; Voza, Francesca; Figueroa, Maria E.; Tempst, Paul; Erdjument-Bromage, Hediye; Gao, Rui; Chen, Sisi; Paietta, Elisabeth; Deblasio, Tony; Melnick, Ari; Liu, Yan; Zhao, Xinyang; Nimer, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1625-1638</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Defining the role of epigenetic regulators in hematopoiesis has become critically important, because recurrent mutations or aberrant expression of these genes has been identified in both myeloid and lymphoid hematol. malignancies.  We found that PRMT4, a type I arginine methyltransferase whose function in normal and malignant hematopoiesis is unknown, is overexpressed in acute myelogenous leukemia patient samples.  Overexpression of PRMT4 blocks the myeloid differentiation of human stem/progenitor cells (HSPCs), whereas its knockdown is sufficient to induce myeloid differentiation of HSPCs.  We demonstrated that PRMT4 represses the expression of miR-223 in HSPCs via the methylation of RUNX1, which triggers the assembly of a multiprotein repressor complex that includes DPF2.  As part of the feedback loop, PRMT4 expression is repressed posttranscriptionally by miR-223.  Depletion of PRMT4 results in differentiation of myeloid leukemia cells in vitro and their decreased proliferation in vivo.  Thus, targeting PRMT4 holds potential as a novel therapy for acute myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8etHAEewC_7Vg90H21EOLACvtfcHk0lhkLid3smfXrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOgs7vK&md5=b0df503d2ce52a520f6ed3f1d7b8e362</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2013.11.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2013.11.025%26sid%3Dliteratum%253Aachs%26aulast%3DVu%26aufirst%3DL.%2BP.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DVoza%26aufirst%3DF.%26aulast%3DFigueroa%26aufirst%3DM.%2BE.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DPaietta%26aufirst%3DE.%26aulast%3DDeblasio%26aufirst%3DT.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DPRMT4%2520blocks%2520myeloid%2520differentiation%2520by%2520assembling%2520a%2520methyl-RUNX1-dependent%2520repressor%2520complex%26jtitle%3DCell%2520Rep.%26date%3D2013%26volume%3D5%26spage%3D1625%26epage%3D1638%26doi%3D10.1016%2Fj.celrep.2013.11.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Zhao, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y. F.</span><span> </span><span class="NLM_article-title">CARM1 mediates modulation of Sox2</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e27026</span><span class="refDoi"> DOI: 10.1371/journal.pone.0027026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1371%2Fjournal.pone.0027026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e27026&author=H.+Y.+Zhaoauthor=Y.+J.+Zhangauthor=H.+Daiauthor=Y.+Zhangauthor=Y.+F.+Shen&title=CARM1+mediates+modulation+of+Sox2&doi=10.1371%2Fjournal.pone.0027026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027026%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DH.%2BY.%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DY.%2BF.%26atitle%3DCARM1%2520mediates%2520modulation%2520of%2520Sox2%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De27026%26doi%3D10.1371%2Fjournal.pone.0027026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wisinski, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahlquist, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span> </span><span class="NLM_article-title">CARM1 methylates chromatin remodeling factor BAF155 to enhance tumor progression and metastasis</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">21</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.1016/j.ccr.2013.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.ccr.2013.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24434208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFSisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=21-36&author=L.+Wangauthor=Z.+B.+Zhaoauthor=M.+B.+Meyerauthor=S.+Sahaauthor=M.+G.+Yuauthor=A.+L.+Guoauthor=K.+B.+Wisinskiauthor=W.+Huangauthor=W.+B.+Caiauthor=J.+W.+Pikeauthor=M.+Yuanauthor=P.+Ahlquistauthor=W.+Xu&title=CARM1+methylates+chromatin+remodeling+factor+BAF155+to+enhance+tumor+progression+and+metastasis&doi=10.1016%2Fj.ccr.2013.12.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">CARM1 Methylates Chromatin Remodeling Factor BAF155 to Enhance Tumor Progression and Metastasis</span></div><div class="casAuthors">Wang, Lu; Zhao, Zibo; Meyer, Mark B.; Saha, Sandeep; Yu, Menggang; Guo, Ailan; Wisinski, Kari B.; Huang, Wei; Cai, Weibo; Pike, J. Wesley; Yuan, Ming; Ahlquist, Paul; Xu, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-36</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1), a coactivator for various cancer-relevant transcription factors, is overexpressed in breast cancer.  To elucidate the functions of CARM1 in tumorigenesis, we knocked out CARM1 from several breast cancer cell lines using Zinc-Finger Nuclease technol., which resulted in drastic phenotypic and biochem. changes.  The CARM1 KO cell lines enabled identification of CARM1 substrates, notably the SWI/SNF core subunit BAF155.  Methylation of BAF155 at R1064 was found to be an independent prognostic biomarker for cancer recurrence and to regulate breast cancer cell migration and metastasis.  Furthermore, CARM1-mediated BAF155 methylation affects gene expression by directing methylated BAF155 to unique chromatin regions (e.g., c-Myc pathway genes).  Collectively, our studies uncover a mechanism by which BAF155 acquires tumorigenic functions via arginine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Fx4Ny0ZnD7Vg90H21EOLACvtfcHk0lhkLid3smfXrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFSisLs%253D&md5=4257dd8fe6bbc915fe1d32ba7af57afe</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DZ.%2BB.%26aulast%3DMeyer%26aufirst%3DM.%2BB.%26aulast%3DSaha%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DM.%2BG.%26aulast%3DGuo%26aufirst%3DA.%2BL.%26aulast%3DWisinski%26aufirst%3DK.%2BB.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DCai%26aufirst%3DW.%2BB.%26aulast%3DPike%26aufirst%3DJ.%2BW.%26aulast%3DYuan%26aufirst%3DM.%26aulast%3DAhlquist%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DCARM1%2520methylates%2520chromatin%2520remodeling%2520factor%2520BAF155%2520to%2520enhance%2520tumor%2520progression%2520and%2520metastasis%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D21%26epage%3D36%26doi%3D10.1016%2Fj.ccr.2013.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Xu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asahara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emerson, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montminy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, R. M.</span><span> </span><span class="NLM_article-title">A transcriptional switch mediated by cofactor methylation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">294</span><span class="NLM_x">, </span> <span class="NLM_fpage">2507</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span><span class="refDoi"> DOI: 10.1126/science.1065961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1126%2Fscience.1065961" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2001&pages=2507-2511&author=W.+Xuauthor=H.+Chenauthor=K.+Duauthor=H.+Asaharaauthor=M.+Tiniauthor=B.+M.+Emersonauthor=M.+Montminyauthor=R.+M.+Evans&title=A+transcriptional+switch+mediated+by+cofactor+methylation&doi=10.1126%2Fscience.1065961"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.1065961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1065961%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DK.%26aulast%3DAsahara%26aufirst%3DH.%26aulast%3DTini%26aufirst%3DM.%26aulast%3DEmerson%26aufirst%3DB.%2BM.%26aulast%3DMontminy%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26atitle%3DA%2520transcriptional%2520switch%2520mediated%2520by%2520cofactor%2520methylation%26jtitle%3DScience%26date%3D2001%26volume%3D294%26spage%3D2507%26epage%3D2511%26doi%3D10.1126%2Fscience.1065961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Malley, B. W.</span><span> </span><span class="NLM_article-title">Signaling within a coactivator complex: Methylation of SRC-3/AIB1 is a molecular switch for complex disassembly</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">7846</span><span class="NLM_x">–</span> <span class="NLM_lpage">7857</span><span class="refDoi"> DOI: 10.1128/MCB.00568-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1128%2FMCB.00568-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16923966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFemur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=7846-7857&author=Q.+Fengauthor=P.+Yiauthor=J.+M.+Wongauthor=B.+W.+O%E2%80%99Malley&title=Signaling+within+a+coactivator+complex%3A+Methylation+of+SRC-3%2FAIB1+is+a+molecular+switch+for+complex+disassembly&doi=10.1128%2FMCB.00568-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling within a coactivator complex: methylation of SRC-3/AIB1 is a molecular switch for complex disassembly</span></div><div class="casAuthors">Feng, Qin; Yi, Ping; Wong, Jiemin; O'Malley, Bert W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7846-7857</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Recent studies indicate that steroid receptor-mediated transcriptional initiation is a cyclical process involving multiple rounds of coactivator assembly and disassembly.  Steroid receptor coactivator 3 (SRC-3) coactivator phosphorylation has been shown to regulate coactivator complex assembly, but the mechanisms by which coactivator disassembly is triggered are not well understood.  In this study, we provide in vitro and in vivo evidence that members of the SRC coactivator family serve as substrates for the enzymic coactivator coactivator-assocd. arginine methyltransferase 1 (CARM1).  Methylation of SRC-3 was localized to an arginine in its CARM1 binding region and correlated with decreased estrogen receptor alpha-mediated transcription, as seen with both cell-based and in vitro transcription assays.  Consistent with this finding, we demonstrated that methylation promotes dissocn. of the SRC-3/CARM1 coactivator complex.  Methylation of SRC-3 is regulated by estrogen signaling in MCF7 cells and serves as a mol. switch for disassembly of the SRC-3 transcriptional coactivator complex.  We propose that CARM1 is a dual-function coactivator, as it not only activates transcription by modifying core histone tails but also terminates hormone signaling by disassembly of the coactivator complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7xFAwd4U-0bVg90H21EOLACvtfcHk0lhso5FP71tCMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFemur7F&md5=e73b458635c8dc1fc629c275f39e429d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FMCB.00568-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00568-06%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DYi%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DJ.%2BM.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DB.%2BW.%26atitle%3DSignaling%2520within%2520a%2520coactivator%2520complex%253A%2520Methylation%2520of%2520SRC-3%252FAIB1%2520is%2520a%2520molecular%2520switch%2520for%2520complex%2520disassembly%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D7846%26epage%3D7857%26doi%3D10.1128%2FMCB.00568-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Chevillard-Briet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trouche, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandel, L.</span><span> </span><span class="NLM_article-title">Control of CBP co-activating activity by arginine methylation</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">5457</span><span class="NLM_x">–</span> <span class="NLM_lpage">5466</span><span class="refDoi"> DOI: 10.1093/emboj/cdf548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1093%2Femboj%2Fcdf548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=12374746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvF2qu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=5457-5466&author=M.+Chevillard-Brietauthor=D.+Troucheauthor=L.+Vandel&title=Control+of+CBP+co-activating+activity+by+arginine+methylation&doi=10.1093%2Femboj%2Fcdf548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Control of CBP co-activating activity by arginine methylation</span></div><div class="casAuthors">Chevillard-Briet, Martine; Trouche, Didier; Vandel, Laurence</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5457-5466</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The histone acetyltransferases CREB binding protein (CBP) and the related p300 protein function as key transcriptional co-activators in multiple pathways.  In the case of transcriptional activation by nuclear receptors, ligand promotes the recruitment of co-activators of the p160 family, such as GRIP-1.  Subsequently, the p160 co-activators recruit other co-activators via two activation domains, AD1 and AD2.  AD1 binds CBP or p300, whereas AD2 has been shown to activate transcription through the recruitment of the arginine methyltransferase CARM1.  Recently, the KIX domain of CBP has been shown to be methylated by CARM1 in vitro.  Here, we report that another domain of CBP is specifically methylated by CARM1 on conserved arginine residues in vitro and in vivo.  We also provide functional evidence that arginine residues methylated by CARM1 play a crit. role in GRIP-1-dependent transcriptional activation and in hormone-induced gene activation.  Altogether, our data provide strong evidence that arginine methylation represents an important mechanism for modulating co-activator transcriptional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFw6WhlsWAkbVg90H21EOLACvtfcHk0lhso5FP71tCMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvF2qu7c%253D&md5=5ac0df871507f31a64c9890647478273</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdf548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdf548%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard-Briet%26aufirst%3DM.%26aulast%3DTrouche%26aufirst%3DD.%26aulast%3DVandel%26aufirst%3DL.%26atitle%3DControl%2520of%2520CBP%2520co-activating%2520activity%2520by%2520arginine%2520methylation%26jtitle%3DEMBO%2520J.%26date%3D2002%26volume%3D21%26spage%3D5457%26epage%3D5466%26doi%3D10.1093%2Femboj%2Fcdf548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">PABP1 identified as an arginine methyltransferase substrate using high-density protein arrays</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">268</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span><span class="refDoi"> DOI: 10.1093/embo-reports/kvf052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1093%2Fembo-reports%2Fkvf052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=11850402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD38XitlCmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=268-273&author=J.+Leeauthor=M.+T.+Bedford&title=PABP1+identified+as+an+arginine+methyltransferase+substrate+using+high-density+protein+arrays&doi=10.1093%2Fembo-reports%2Fkvf052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">PABP1 identified as an arginine methyltransferase substrate using high-density protein arrays</span></div><div class="casAuthors">Lee, Jaeho; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">268-273</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The arginine methyltransferases CARM1 and PRMT1 assoc. with the p160 family of nuclear hormone receptor coactivators.  This assocn. enhances transcriptional activation by nuclear receptors.  We describe a method for identifying arginine N-methyltransferase substrates using arrayed high-d. protein membranes to perform solid-phase supported enzyme reactions in the presence of the Me donor S-adenosyl-L-methionine.  Using this screen, we identified distinct substrates for CARM1 and PRMT1.  All PRMT1 substrates harbor the expected GGRGG methylation motif, whereas the peptide sequence comparisons of the CARM1 substrates revealed no such motif.  The predominant CARM1 substrate identified in this screen was PABP1.  We mapped the methylated region of this RNA binding mol. in vitro and demonstrate that PABP1 is indeed methylated in vivo.  Prior to these findings, the only known substrate for CARM1 was histone H3.  We broaden the no. of CARM1 targets and suggest a role for CARM1 in regulating transcription/translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOsRoAKGzNqrVg90H21EOLACvtfcHk0lhso5FP71tCMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitlCmt7o%253D&md5=6d0be1647bdb90edeab2a00c4715b70d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fembo-reports%2Fkvf052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fembo-reports%252Fkvf052%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DPABP1%2520identified%2520as%2520an%2520arginine%2520methyltransferase%2520substrate%2520using%2520high-density%2520protein%2520arrays%26jtitle%3DEMBO%2520Rep.%26date%3D2002%26volume%3D3%26spage%3D268%26epage%3D273%26doi%3D10.1093%2Fembo-reports%2Fkvf052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kilburn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jelinek, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henschen-Edman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aswad, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laird-Offringa, I. A.</span><span> </span><span class="NLM_article-title">Lipopolysaccharide-induced methylation of HuR, an mRNA-stabilizing protein, by CARM1. Coactivator-associated arginine methyltransferase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">44623</span><span class="NLM_x">–</span> <span class="NLM_lpage">44630</span><span class="refDoi"> DOI: 10.1074/jbc.M206187200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M206187200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=44623-44630&author=H.+Liauthor=S.+Parkauthor=B.+Kilburnauthor=M.+A.+Jelinekauthor=A.+Henschen-Edmanauthor=D.+W.+Aswadauthor=M.+R.+Stallcupauthor=I.+A.+Laird-Offringa&title=Lipopolysaccharide-induced+methylation+of+HuR%2C+an+mRNA-stabilizing+protein%2C+by+CARM1.+Coactivator-associated+arginine+methyltransferase&doi=10.1074%2Fjbc.M206187200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206187200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206187200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DKilburn%26aufirst%3DB.%26aulast%3DJelinek%26aufirst%3DM.%2BA.%26aulast%3DHenschen-Edman%26aufirst%3DA.%26aulast%3DAswad%26aufirst%3DD.%2BW.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DLaird-Offringa%26aufirst%3DI.%2BA.%26atitle%3DLipopolysaccharide-induced%2520methylation%2520of%2520HuR%252C%2520an%2520mRNA-stabilizing%2520protein%252C%2520by%2520CARM1.%2520Coactivator-associated%2520arginine%2520methyltransferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D44623%26epage%3D44630%26doi%3D10.1074%2Fjbc.M206187200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Fujiwara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koyama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yachi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshikawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohyama, M.</span><span> </span><span class="NLM_article-title">CARM1 regulates proliferation of PC12 cells by methylating HuD</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">2273</span><span class="NLM_x">–</span> <span class="NLM_lpage">2285</span><span class="refDoi"> DOI: 10.1128/MCB.26.6.2273-2285.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1128%2FMCB.26.6.2273-2285.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16508003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2273-2285&author=T.+Fujiwaraauthor=Y.+Moriauthor=D.+L.+Chuauthor=Y.+Koyamaauthor=S.+Miyataauthor=H.+Tanakaauthor=K.+Yachiauthor=T.+Kuboauthor=H.+Yoshikawaauthor=M.+Tohyama&title=CARM1+regulates+proliferation+of+PC12+cells+by+methylating+HuD&doi=10.1128%2FMCB.26.6.2273-2285.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">CARM1 regulates proliferation of PC12 cells by methylating HuD</span></div><div class="casAuthors">Fujiwara, Tatsuji; Mori, Yasutake; Chu, Dong Ling; Koyama, Yoshihisa; Miyata, Shingo; Tanaka, Hiroyuki; Yachi, Kohji; Kubo, Tateki; Yoshikawa, Hideki; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2273-2285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">HuD is an RNA-binding protein that has been shown to induce neuronal differentiation by stabilizing labile mRNAs carrying AU-rich instability elements.  Here, we show a novel mechanism of arginine methylation of HuD by coactivator-assocd. arginine methyltransferase 1 (CARM1) that affected mRNA turnover of p21cip1/waf1 mRNA in PC12 cells.  CARM1 specifically methylated HuD in vitro and in vivo and colocalized with HuD in the cytoplasm.  Inhibition of HuD methylation by CARM1 knockdown elongated the p21cip1/waf1 mRNA half-life and resulted in a slow growth rate and robust neuritogenesis in response to nerve growth factor (NGF).  Methylation-resistant HuD bound more p21cip1/waf1 mRNA than did the wild type, and its overexpression upregulated p21cip1/waf1 protein expression.  These results suggested that CARM1-methylated HuD maintains PC12 cells in the proliferative state by committing p21cip1/waf1 mRNA to its decay system.  Since the methylated population of HuD was reduced in NGF-treated PC12 cells, downregulation of HuD methylation is a possible pathway through which NGF induces differentiation of PC12 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71TYVPwUyRLVg90H21EOLACvtfcHk0liCSPpH_M4K7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D&md5=a557599ff6460f5ab365362323b87a2b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.6.2273-2285.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.6.2273-2285.2006%26sid%3Dliteratum%253Aachs%26aulast%3DFujiwara%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DChu%26aufirst%3DD.%2BL.%26aulast%3DKoyama%26aufirst%3DY.%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DYachi%26aufirst%3DK.%26aulast%3DKubo%26aufirst%3DT.%26aulast%3DYoshikawa%26aufirst%3DH.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DCARM1%2520regulates%2520proliferation%2520of%2520PC12%2520cells%2520by%2520methylating%2520HuD%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2273%26epage%3D2285%26doi%3D10.1128%2FMCB.26.6.2273-2285.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaaban, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span><span class="refDoi"> DOI: 10.1016/j.molcel.2006.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2006.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17218272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=71-83&author=D.+Chengauthor=J.+Coteauthor=S.+Shaabanauthor=M.+T.+Bedford&title=The+arginine+methyltransferase+CARM1+regulates+the+coupling+of+transcription+and+mRNA+processing&doi=10.1016%2Fj.molcel.2006.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span></div><div class="casAuthors">Cheng, Donghang; Cote, Jocelyn; Shaaban, Salam; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-83</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The coactivator-assocd. arginine methyl-transferase CARM1 is recruited by many different transcription factors as a pos. regulator.  To understand the mechanism by which CARM1 functions, we sought to isolate its substrates.  We developed a small-pool screening approach for this purpose and identified CA150, SAP49, SmB, and U1C as splicing factors that are specifically methylated by CARM1.  We further showed that CA150, a mol. that links transcription to splicing, interacts with the Tudor domain of the spinal muscular atrophy protein SMN in a CARM1-dependent fashion.  Expts. with an exogenous splicing reporter and the endogenous CD44 gene revealed that CARM1 promotes exon skipping in an enzyme-dependent manner.  The identification of splicing factors that are methylated by CARM1, and protein-protein interactions that are regulated by CARM1, strongly implicates this enzyme in the regulation of alternative splicing and points toward its involvement in spinal muscular atrophy pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqulZNAke_ZsrVg90H21EOLACvtfcHk0liCSPpH_M4K7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D&md5=29f6dccb415e1146e6455a03613f9fa2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DCote%26aufirst%3DJ.%26aulast%3DShaaban%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520arginine%2520methyltransferase%2520CARM1%2520regulates%2520the%2520coupling%2520of%2520transcription%2520and%2520mRNA%2520processing%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D71%26epage%3D83%26doi%3D10.1016%2Fj.molcel.2006.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richie, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Loss of CARM1 results in hypomethylation of thymocyte cyclic AMP-regulated phosphoprotein and deregulated early T cell development</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">25339</span><span class="NLM_x">–</span> <span class="NLM_lpage">25344</span><span class="refDoi"> DOI: 10.1074/jbc.M402544200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M402544200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=25339-25344&author=J.+Kimauthor=J.+Leeauthor=N.+Yadavauthor=Q.+Wuauthor=C.+Carterauthor=S.+Richardauthor=E.+Richieauthor=M.+T.+Bedford&title=Loss+of+CARM1+results+in+hypomethylation+of+thymocyte+cyclic+AMP-regulated+phosphoprotein+and+deregulated+early+T+cell+development&doi=10.1074%2Fjbc.M402544200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M402544200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M402544200%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DRichie%26aufirst%3DE.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DLoss%2520of%2520CARM1%2520results%2520in%2520hypomethylation%2520of%2520thymocyte%2520cyclic%2520AMP-regulated%2520phosphoprotein%2520and%2520deregulated%2520early%2520T%2520cell%2520development%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D25339%26epage%3D25344%26doi%3D10.1074%2Fjbc.M402544200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Sims, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rojas, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonasio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuller, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drury, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eick, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">The C-terminal domain of RNA polymerase II is modified by site-specific methylation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1126/science.1202663</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1126%2Fscience.1202663" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21454787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVyisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2011&pages=99-103&author=R.+J.+Simsauthor=L.+A.+Rojasauthor=D.+Beckauthor=R.+Bonasioauthor=R.+Schullerauthor=W.+J.+Druryauthor=D.+Eickauthor=D.+Reinberg&title=The+C-terminal+domain+of+RNA+polymerase+II+is+modified+by+site-specific+methylation&doi=10.1126%2Fscience.1202663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The C-Terminal Domain of RNA Polymerase II Is Modified by Site-Specific Methylation</span></div><div class="casAuthors">Sims, Robert J., III; Rojas, Luis Alejandro; Beck, David; Bonasio, Roberto; Schueller, Roland; Drury, William J., III; Eick, Dirk; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">6025</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The carboxy-terminal domain (CTD) of RNA polymerase II (RNAPII) in mammals undergoes extensive posttranslational modification, which is essential for transcriptional initiation and elongation.  Here, we show that the CTD of RNAPII is methylated at a single arginine (R1810) by the coactivator-assocd. arginine methyltransferase 1 (CARM1).  Although methylation at R1810 is present on the hyperphosphorylated form of RNAPII in vivo, Ser2 or Ser5 phosphorylation inhibits CARM1 activity toward this site in vitro, suggesting that methylation occurs before transcription initiation.  Mutation of R1810 results in the misexpression of a variety of small nuclear RNAs and small nucleolar RNAs, an effect that is also obsd. in Carm1-/- mouse embryo fibroblasts.  These results demonstrate that CTD methylation facilitates the expression of select RNAs, perhaps serving to discriminate the RNAPII-assocd. machinery recruited to distinct gene types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6UKJDVEFMO7Vg90H21EOLACvtfcHk0lg5MtPBESLmZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvVyisbo%253D&md5=f613b8e70477e245b52cf5dd5786ba3c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1202663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1202663%26sid%3Dliteratum%253Aachs%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DRojas%26aufirst%3DL.%2BA.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DBonasio%26aufirst%3DR.%26aulast%3DSchuller%26aufirst%3DR.%26aulast%3DDrury%26aufirst%3DW.%2BJ.%26aulast%3DEick%26aufirst%3DD.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520C-terminal%2520domain%2520of%2520RNA%2520polymerase%2520II%2520is%2520modified%2520by%2520site-specific%2520methylation%26jtitle%3DScience%26date%3D2011%26volume%3D332%26spage%3D99%26epage%3D103%26doi%3D10.1126%2Fscience.1202663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span> </span><span class="NLM_article-title">MED12 methylation by CARM1 sensitizes human breast cancer cells to chemotherapy drugs</span> <span class="citation_source-journal">Sci. Adv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1500463</span><span class="refDoi"> DOI: 10.1126/sciadv.1500463</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1126%2Fsciadv.1500463" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=e1500463&author=L.+Wangauthor=H.+Zengauthor=Q.+Wangauthor=Z.+Zhaoauthor=T.+G.+Boyerauthor=X.+Bianauthor=W.+Xu&title=MED12+methylation+by+CARM1+sensitizes+human+breast+cancer+cells+to+chemotherapy+drugs&doi=10.1126%2Fsciadv.1500463"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500463%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DBoyer%26aufirst%3DT.%2BG.%26aulast%3DBian%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DMED12%2520methylation%2520by%2520CARM1%2520sensitizes%2520human%2520breast%2520cancer%2520cells%2520to%2520chemotherapy%2520drugs%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26spage%3De1500463%26doi%3D10.1126%2Fsciadv.1500463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">El Messaoudi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbrizio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuchana, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauquier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theillet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sardet, C.</span><span> </span><span class="NLM_article-title">Coactivator-associated arginine methyltransferase 1 (CARM1) is a positive regulator of the Cyclin E1 gene</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">13351</span><span class="NLM_x">–</span> <span class="NLM_lpage">13356</span><span class="refDoi"> DOI: 10.1073/pnas.0605692103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.0605692103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16938873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVals7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=13351-13356&author=S.+El+Messaoudiauthor=E.+Fabbrizioauthor=C.+Rodriguezauthor=P.+Chuchanaauthor=L.+Fauquierauthor=D.+H.+Chengauthor=C.+Theilletauthor=L.+Vandelauthor=M.+T.+Bedfordauthor=C.+Sardet&title=Coactivator-associated+arginine+methyltransferase+1+%28CARM1%29+is+a+positive+regulator+of+the+Cyclin+E1+gene&doi=10.1073%2Fpnas.0605692103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Coactivator-associated arginine methyltransferase 1 (CARM1) is a positive regulator of the cyclin E1 gene</span></div><div class="casAuthors">El Messaoudi, Selma; Fabbrizio, Eric; Rodriguez, Carmen; Chuchana, Paul; Fauquier, Lucas; Cheng, Donghang; Theillet, Charles; Vandel, Laurence; Bedford, Mark T.; Sardet, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">13351-13356</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Cyclin E1 gene (CCNE1) is an ideal model to explore the mechanisms that control the transcription of cell cycle-regulated genes whose expression rises transiently before entry into S phase.  E2F-dependent regulation of the CCNE1 promoter was shown to correlate with changes in the level of H3-K9 acetylation/methylation of nucleosomal histones positioned at the transcriptional start site region.  Here we show that, upon growth stimulation, the same region is subject to variations of H3-R17 and H3-R26 methylation that correlate with the recruitment of coactivator-assocd. arginine methyltransferase 1 (CARM1) onto the CCNE1 and DHFR promoters.  Accordingly, CARM1-deficient cells lack these modifications and present lowered levels and altered kinetics of CCNE1 and DHFR mRNA expression.  Consistently, reporter gene assays demonstrate that CARM1 functions as a transcriptional coactivator for their E2F1/DP1-stimulated expression.  CARM1 recruitment at the CCNE1 gene requires activator E2Fs and ACTR, a member of the p160 coactivator family that is frequently overexpressed in human breast cancer.  Finally, we show that grade-3 breast tumors present coelevated mRNA levels of ACTR and CARM1, along with their transcriptional target CCNE1.  All together, our results indicate that CARM1 is an important regulator of the CCNE1 gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwAPzIlZVHFLVg90H21EOLACvtfcHk0lg5MtPBESLmZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVals7c%253D&md5=377d79e0870e107b55675fb11b204ae8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0605692103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0605692103%26sid%3Dliteratum%253Aachs%26aulast%3DEl%2BMessaoudi%26aufirst%3DS.%26aulast%3DFabbrizio%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DC.%26aulast%3DChuchana%26aufirst%3DP.%26aulast%3DFauquier%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DTheillet%26aufirst%3DC.%26aulast%3DVandel%26aufirst%3DL.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DSardet%26aufirst%3DC.%26atitle%3DCoactivator-associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%2520is%2520a%2520positive%2520regulator%2520of%2520the%2520Cyclin%2520E1%2520gene%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D13351%26epage%3D13356%26doi%3D10.1073%2Fpnas.0605692103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Hong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kao, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeng, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eble, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacLennan, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, L.</span><span> </span><span class="NLM_article-title">Aberrant expression of CARM1, a transcriptional coactivator of androgen receptor, in the development of prostate carcinoma and androgen-independent status</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1002/cncr.20327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fcncr.20327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=15221992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFCnu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=83-89&author=H.+Hongauthor=C.+H.+Kaoauthor=M.+H.+Jengauthor=J.+N.+Ebleauthor=M.+O.+Kochauthor=T.+A.+Gardnerauthor=S.+B.+Zhangauthor=L.+Liauthor=C.+X.+Panauthor=Z.+Q.+Huauthor=G.+T.+MacLennanauthor=L.+Cheng&title=Aberrant+expression+of+CARM1%2C+a+transcriptional+coactivator+of+androgen+receptor%2C+in+the+development+of+prostate+carcinoma+and+androgen-independent+status&doi=10.1002%2Fcncr.20327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Aberrant expression of CARM1, a transcriptional coactivator of androgen receptor, in the development of prostate carcinoma and androgen-independent status</span></div><div class="casAuthors">Hong, Heng; Kao, Chinghai; Jeng, Meei-Huey; Eble, John N.; Koch, Michael O.; Gardner, Thomas A.; Zhang, Shaobo; Li, Lang; Pan, Chong-Xian; Hu, Zhiqiang; MacLennan, Gregory T.; Cheng, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-89</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1) is a transcriptional coactivator of the androgen receptor (AR).  It is involved in the regulation of the biol. functions of the AR.  It remains to be detd. whether CARM1 is involved in prostatic carcinogenesis.  The expression of CARM1 in normal prostate epithelium, high-grade prostatic intraepithelial neoplasia (PIN), and prostate carcinoma tissue was examd. in 66 previously untreated patients with prostate carcinomas, as well as 12 patients with hormone-independent prostate carcinoma, using immunohistochem. methods.  Cell staining was obsd. in the cytoplasm and the nucleus.  In 66 patients without previous hormonal treatment, the percentage of cells that stained pos. for CARM1 in benign prostate tissue specimens (mean values: cytoplasm, 23%; nucleus, 16%) was statistically significantly less than the percentage of pos. stained cells in PIN (mean values: cytoplasm, 56%; nucleus, 30%) and in prostatic adenocarcinoma specimens (mean values: cytoplasm, 79%; nucleus, 53%).  The difference between adenocarcinoma and PIN also was statistically significant.  The staining intensity for CARM1 was significantly lower in benign prostate tissue specimens compared with PIN and prostatic adenocarcinoma specimens.  In the 12 patients with androgen-independent prostatic adenocarcinoma, the expression of CARM1 was significantly increased when compared with patients without previous hormonal treatment.  Expression of CARM1 was not correlated with age, Gleason score sum, pathol. stage, lymph node metastasis, extraprostatic extension, surgical margin status, vascular invasion, or perineural invasion.  Thus, the authors found that overexpression of CARM1 was involved in the development of prostate carcinoma as well as androgen-independent prostate carcinoma.  Since CARM1 is functionally different from most other transcriptional coactivators of the AR, it may serve as a new target for the treatment of hormone-independent prostate carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSyswrcJiDBLVg90H21EOLACvtfcHk0liiyg1FsPtbcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFCnu7k%253D&md5=143b9ca402cbd7c0223ed07db991f503</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fcncr.20327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.20327%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DH.%26aulast%3DKao%26aufirst%3DC.%2BH.%26aulast%3DJeng%26aufirst%3DM.%2BH.%26aulast%3DEble%26aufirst%3DJ.%2BN.%26aulast%3DKoch%26aufirst%3DM.%2BO.%26aulast%3DGardner%26aufirst%3DT.%2BA.%26aulast%3DZhang%26aufirst%3DS.%2BB.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DC.%2BX.%26aulast%3DHu%26aufirst%3DZ.%2BQ.%26aulast%3DMacLennan%26aufirst%3DG.%2BT.%26aulast%3DCheng%26aufirst%3DL.%26atitle%3DAberrant%2520expression%2520of%2520CARM1%252C%2520a%2520transcriptional%2520coactivator%2520of%2520androgen%2520receptor%252C%2520in%2520the%2520development%2520of%2520prostate%2520carcinoma%2520and%2520androgen-independent%2520status%26jtitle%3DCancer%26date%3D2004%26volume%3D101%26spage%3D83%26epage%3D89%26doi%3D10.1002%2Fcncr.20327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Kim, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N. T. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, C.</span><span> </span><span class="NLM_article-title">Differential CARM1 expression in prostate and colorectal cancers</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="refDoi"> DOI: 10.1186/1471-2407-10-197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1186%2F1471-2407-10-197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20462455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A280%3ADC%252BC3cvmsVSjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=197&author=Y.+R.+Kimauthor=B.+K.+Leeauthor=R.+Y.+Parkauthor=N.+T.+X.+Nguyenauthor=J.+A.+Baeauthor=D.+D.+Kwonauthor=C.+Jung&title=Differential+CARM1+expression+in+prostate+and+colorectal+cancers&doi=10.1186%2F1471-2407-10-197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Differential CARM1 expression in prostate and colorectal cancers</span></div><div class="casAuthors">Kim Young-Rang; Lee Byung Kook; Park Ra-Young; Nguyen Nguyen Thi Xuan; Bae Jeong A; Kwon Dong Deuk; Jung Chaeyong</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">197</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Coactivator-associated arginine methyltransferase 1 (CARM1) functions as a transcriptional coactivator of androgen receptor (AR)-mediated signaling.  Correspondingly, overexpression of CARM1 has been associated with the development of prostate cancer (PCa) and its progression to androgen-independent PCa.  In our preliminary study, however, the promoting effects of CARM1, with regard to androgen-stimulated AR target gene expression were minimal.  These results suggested that the AR target gene expression associated with CARM1 may result primarily from non-hormone dependent activity.  The goal of this study was to confirm the pattern of expression of CARM1 in human tumors and determine the mechanism of action in CARM1 overexpressed tumors.  METHODS:  Tissue microarray was used to determine the pattern of expression of CARM1 in human cancers by immunohistochemistry.  CARM1 expression was also evaluated in prostate and colorectal surgical specimens and the clinical records of all cases were reviewed.  In addition, a reporter transcription assay using the prostate-specific antigen (PSA) promoter was used to identify the signaling pathways involved in non-hormone-mediated signal activation associated with CARM1.  RESULTS:  The tissue microarray showed that CARM1 was particularly overexpressed in the colorectal cancers while CARM1 expression was not prevalent in the prostate and breast cancers.  Further studies using surgical specimens demonstrated that CARM1 was highly overexpressed in 75% of colorectal cancers (49 out of 65) but not in the androgen-independent PCa.  In addition, CARM1's coactivating effect on the entire PSA promoter was very limited in both androgen-dependent and androgen-independent PCa cells.  These results suggest that there are other factors associated with CARM1 expression in PSA regulation.  Indeed, CARM1 significantly regulated both p53 and NF-kappaB target gene transcription.  CONCLUSIONS:  The results of this study suggest that, in addition to its role in activation of steroid receptors, CARM1 functions as a transcriptional modulator by altering the activity of many transcriptional factors, especially with regard to androgen independent PCa and colorectal cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNfngOqNkUpvcsfMJQC1ZdfW6udTcc2eaWsNBUzIa_-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cvmsVSjuw%253D%253D&md5=d10897a9e65d3c395f5114eb22fc4650</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-10-197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-10-197%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%2BR.%26aulast%3DLee%26aufirst%3DB.%2BK.%26aulast%3DPark%26aufirst%3DR.%2BY.%26aulast%3DNguyen%26aufirst%3DN.%2BT.%2BX.%26aulast%3DBae%26aufirst%3DJ.%2BA.%26aulast%3DKwon%26aufirst%3DD.%2BD.%26aulast%3DJung%26aufirst%3DC.%26atitle%3DDifferential%2520CARM1%2520expression%2520in%2520prostate%2520and%2520colorectal%2520cancers%26jtitle%3DBMC%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D197%26doi%3D10.1186%2F1471-2407-10-197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Guccione, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bassi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinato, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cesaroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuchlautz, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amati, B.</span><span> </span><span class="NLM_article-title">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">933</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1038/nature06166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnature06166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17898714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=933-937&author=E.+Guccioneauthor=C.+Bassiauthor=F.+Casadioauthor=F.+Martinatoauthor=M.+Cesaroniauthor=H.+Schuchlautzauthor=B.+Luscherauthor=B.+Amati&title=Methylation+of+histone+H3R2+by+PRMT6+and+H3K4+by+an+MLL+complex+are+mutually+exclusive&doi=10.1038%2Fnature06166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span></div><div class="casAuthors">Guccione, Ernesto; Bassi, Christian; Casadio, Fabio; Martinato, Francesca; Cesaroni, Matteo; Schuchlautz, Henning; Luescher, Bernhard; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">933-937</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Eukaryotic genomes are organized into active (euchromatic) and inactive (heterochromatic) chromatin domains.  Post-translational modifications of histones (or 'marks') are key in defining these functional states, particularly in promoter regions.  Mutual regulatory interactions between these marks-and the enzymes that catalyze them-contribute to the shaping of this epigenetic landscape, in a manner that remains to be fully elucidated.  We previously obsd. that asym. di-methylation of histone H3 arginine 2 (H3R2me2a) counter-correlates with di- and tri- methylation of H3 lysine 4 (H3K4me2, H3K4me3) on human promoters.  Here we show that the arginine methyltransferase PRMT6 catalyzes H3R2 di-methylation in vitro and controls global levels of H3R2me2a in vivo.  H3R2 methylation by PRMT6 was prevented by the presence of H3K4me3 on the H3 tail.  Conversely, the H3R2me2a mark prevented methylation of H3K4 as well as binding to the H3 tail by an ASH2/WDR5/MLL-family methyltransferase complex.  Chromatin immunopptn. showed that H3R2me2a was distributed within the body and at the 3' end of human genes, regardless of their transcriptional state, whereas it was selectively and locally depleted from active promoters, coincident with the presence of H3K4me3.  Hence, the mutual antagonism between H3R2 and H3K4 methylation, together with the assocn. of MLL-family complexes with the basal transcription machinery, may contribute to the localized patterns of H3K4 tri-methylation characteristic of transcriptionally poised or active promoters in mammalian genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWqE8pDpHmKLVg90H21EOLACvtfcHk0liiyg1FsPtbcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK&md5=f4cc9371491088af7b74b614c90d6bd4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnature06166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06166%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DMartinato%26aufirst%3DF.%26aulast%3DCesaroni%26aufirst%3DM.%26aulast%3DSchuchlautz%26aufirst%3DH.%26aulast%3DLuscher%26aufirst%3DB.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DMethylation%2520of%2520histone%2520H3R2%2520by%2520PRMT6%2520and%2520H3K4%2520by%2520an%2520MLL%2520complex%2520are%2520mutually%2520exclusive%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D933%26epage%3D937%26doi%3D10.1038%2Fnature06166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">El-Andaloussi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valovka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toueille, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinacher, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Focke, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehrig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassa, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubscher, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hottiger, M. O.</span><span> </span><span class="NLM_article-title">Arginine methylation regulates DNA polymerase beta</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">62</span><span class="refDoi"> DOI: 10.1016/j.molcel.2006.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2006.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16600869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=51-62&author=N.+El-Andaloussiauthor=T.+Valovkaauthor=M.+Toueilleauthor=R.+Steinacherauthor=F.+Fockeauthor=P.+Gehrigauthor=M.+Covicauthor=P.+O.+Hassaauthor=P.+Scharauthor=U.+Hubscherauthor=M.+O.+Hottiger&title=Arginine+methylation+regulates+DNA+polymerase+beta&doi=10.1016%2Fj.molcel.2006.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates DNA polymerase β</span></div><div class="casAuthors">El-Andaloussi, Nazim; Valovka, Taras; Toueille, Magali; Steinacher, Roland; Focke, Frauke; Gehrig, Peter; Covic, Marcela; Hassa, Paul O.; Schar, Primo; Hubscher, Ulrich; Hottiger, Michael O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-62</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Alterations in DNA repair lead to genomic instability and higher risk of cancer.  DNA base excision repair (BER) corrects damaged bases, apurinic sites, and single-strand DNA breaks.  Here, a regulatory mechanism for DNA polymerase β (Pol β) is described.  Pol β was found to form a complex with the protein arginine methyltransferase 6 (PRMT6) and was specifically methylated in vitro and in vivo.  Methylation of Pol β by PRMT6 strongly stimulated DNA polymerase activity by enhancing DNA binding and processivity, while single nucleotide insertion and dRP-lyase activity were not affected.  Two residues, R83 and R152, were identified in Pol β as the sites of methylation by PRMT6.  Genetic complementation of Pol β knockout cells with R83/152K mutant revealed the importance of these residues for the cellular resistance to DNA alkylating agent.  Based on our findings, we propose that PRMT6 plays a role as a regulator of BER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqui7wNvLvrVg90H21EOLACvtfcHk0lirzBq5Nqb8Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D&md5=987419489fb0439acb13f3b78d2d5a1e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Andaloussi%26aufirst%3DN.%26aulast%3DValovka%26aufirst%3DT.%26aulast%3DToueille%26aufirst%3DM.%26aulast%3DSteinacher%26aufirst%3DR.%26aulast%3DFocke%26aufirst%3DF.%26aulast%3DGehrig%26aufirst%3DP.%26aulast%3DCovic%26aufirst%3DM.%26aulast%3DHassa%26aufirst%3DP.%2BO.%26aulast%3DSchar%26aufirst%3DP.%26aulast%3DHubscher%26aufirst%3DU.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26atitle%3DArginine%2520methylation%2520regulates%2520DNA%2520polymerase%2520beta%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D22%26spage%3D51%26epage%3D62%26doi%3D10.1016%2Fj.molcel.2006.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Boulanger, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1128/JVI.79.1.124-131.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1128%2FJVI.79.1.124-131.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=15596808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=124-131&author=M.+C.+Boulangerauthor=C.+Liangauthor=R.+S.+Russellauthor=R.+Linauthor=M.+T.+Bedfordauthor=M.+A.+Wainbergauthor=S.+Richard&title=Methylation+of+Tat+by+PRMT6+regulates+human+immunodeficiency+virus+type+1+gene+expression&doi=10.1128%2FJVI.79.1.124-131.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span></div><div class="casAuthors">Boulanger, Marie-Chloe; Liang, Chen; Russell, Rodney S.; Lin, Rongtuan; Bedford, Mark T.; Wainberg, Mark A.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-131</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The human immunodeficiency virus (HIV) transactivator protein, Tat, stimulates transcription from the viral long terminal repeats via an arginine-rich transactivating domain.  Since arginines are often known to be methylated, authors investigated whether HIV type 1 (HIV-1) Tat was a substrate for known protein arginine methyltransferases (PRMTs).  Here authors identify Tat as a substrate for the arginine methyltransferase, PRMT6.  Tat is specifically assocd. with and methylated by PRMT6 within cells.  Overexpression of wild-type PRMT6, but not a methylase-inactive PRMT6 mutant, decreased Tat transactivation of an HIV-1 long terminal repeat luciferase reporter plasmid in a dose-dependent manner.  Knocking down PRMT6 consistently increased HIV-1 prodn. in HEK293T cells and also led to increased viral infectiousness as shown in multinuclear activation of a galactosidase indicator assays.  This study demonstrates that arginine methylation of Tat neg. regulates its transactivation activity and that PRMT6 acts as a restriction factor for HIV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0JF5C5Agwq7Vg90H21EOLACvtfcHk0lirzBq5Nqb8Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D&md5=763ac9af247b3f81962ae4e19419493a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.1.124-131.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.1.124-131.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBoulanger%26aufirst%3DM.%2BC.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DR.%2BS.%26aulast%3DLin%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DMethylation%2520of%2520Tat%2520by%2520PRMT6%2520regulates%2520human%2520immunodeficiency%2520virus%2520type%25201%2520gene%2520expression%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D124%26epage%3D131%26doi%3D10.1128%2FJVI.79.1.124-131.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Sgarra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tessari, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spolaore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giancotti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manfioletti, G.</span><span> </span><span class="NLM_article-title">The AT-hook of the chromatin architectural transcription factor high mobility group A1a is arginine-methylated by protein arginine methyltransferase 6</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">3764</span><span class="NLM_x">–</span> <span class="NLM_lpage">3772</span><span class="refDoi"> DOI: 10.1074/jbc.M510231200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M510231200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16293633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=3764-3772&author=R.+Sgarraauthor=J.+Leeauthor=M.+A.+Tessariauthor=S.+Altamuraauthor=B.+Spolaoreauthor=V.+Giancottiauthor=M.+T.+Bedfordauthor=G.+Manfioletti&title=The+AT-hook+of+the+chromatin+architectural+transcription+factor+high+mobility+group+A1a+is+arginine-methylated+by+protein+arginine+methyltransferase+6&doi=10.1074%2Fjbc.M510231200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The AT-hook of the Chromatin Architectural Transcription Factor High Mobility Group A1a Is Arginine-methylated by Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Sgarra, Riccardo; Lee, Jaeho; Tessari, Michela A.; Altamura, Sandro; Spolaore, Barbara; Giancotti, Vincenzo; Bedford, Mark T.; Manfioletti, Guidalberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3764-3772</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The HMGA1a protein belongs to the high mobility group A (HMGA) family of architectural nuclear factors, a group of proteins that plays an important role in chromatin dynamics.  HMGA proteins are multifunctional factors that assoc. both with DNA and nuclear proteins that have been involved in several nuclear processes, such as transcriptional regulation, viral integration, DNA repair, RNA processing, and chromatin remodeling.  The activity of HMGA proteins is finely modulated by a variety of post-translational modifications.  Arginine methylation was recently demonstrated to occur on HMGA1a protein, and it correlates with the apoptotic process and neoplastic progression.  Methyltransferases responsible for these modifications are unknown.  Here we show that the protein arginine methyltransferase PRMT6 specifically methylates HMGA1a protein both in vitro and in vivo.  By mass spectrometry, the sites of methylation were unambiguously mapped to Arg57 and Arg59, two residues which are embedded in the second AT-hook, a region crit. for both protein-DNA and protein-protein interactions and whose modification may cause profound alterations in the HMGA network.  The in vivo assocn. of HMGA and PRMT6 place this yet functionally uncharacterized methyltransferase in the well established functional context of the chromatin structure organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKv8D5yCRQ4rVg90H21EOLACvtfcHk0lirzBq5Nqb8Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D&md5=803d5cd75cc725a9c381220bc067d330</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M510231200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M510231200%26sid%3Dliteratum%253Aachs%26aulast%3DSgarra%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTessari%26aufirst%3DM.%2BA.%26aulast%3DAltamura%26aufirst%3DS.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DGiancotti%26aufirst%3DV.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DManfioletti%26aufirst%3DG.%26atitle%3DThe%2520AT-hook%2520of%2520the%2520chromatin%2520architectural%2520transcription%2520factor%2520high%2520mobility%2520group%2520A1a%2520is%2520arginine-methylated%2520by%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D3764%26epage%3D3772%26doi%3D10.1074%2Fjbc.M510231200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Limm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oefner, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellerbrand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosserhoff, A. K.</span><span> </span><span class="NLM_article-title">Deregulation of protein methylation in melanoma</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1305</span><span class="NLM_x">–</span> <span class="NLM_lpage">1313</span><span class="refDoi"> DOI: 10.1016/j.ejca.2012.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.ejca.2012.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23265702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=1305-1313&author=K.+Limmauthor=C.+Ottauthor=S.+Wallnerauthor=D.+W.+Muellerauthor=P.+Oefnerauthor=C.+Hellerbrandauthor=A.+K.+Bosserhoff&title=Deregulation+of+protein+methylation+in+melanoma&doi=10.1016%2Fj.ejca.2012.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of protein methylation in melanoma</span></div><div class="casAuthors">Limm, Katharina; Ott, Corinna; Wallner, Susanne; Mueller, Daniel W.; Oefner, Peter; Hellerbrand, Claus; Bosserhoff, Anja-Katrin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1305-1313</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Loss of methylthioadenosine phosphorylase (MTAP) expression and a concomitant accumulation of 5'-methyl-thioadenosine (MTA) characterize several tumor entities including malignant melanoma.  MTA affects cellular signalling, proliferation and migration not only of cancer but also surrounding cells including lymphocytes and stromal fibroblasts.  The mode of action of MTA is still not known.  Interestingly, MTA is a known potent inhibitor of protein arginine methyltransferases (PRMTs) and is used as a tool in studying activity and impact of PRMTs.  This study aimed at analyzing PRMTs in melanoma and the potential impact of MTA on tumorigenesis.Our findings demonstrate that expression of PRMT4/CARM1 and PRMT6 is deregulated in melanoma, whereas expression of the remaining PRMTs stays unchanged.  General PRMT activity and, consequently, sym. and asym. protein methylation are reduced significantly in melanoma cells and tissues.  This is due to a loss of MTAP expression and accumulation of MTA.  Redn. of protein methylation by MTA affects cell signalling and leads, for example, to an activation of extracellular signal-regulated kinase (ERK) activity.  The effects of endogeneous MTA on PRMTs as presented in this study can strongly support the migratory and invasive phenotype of melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqujaHA2MQGG7Vg90H21EOLACvtfcHk0lhjy00fduejpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF&md5=a29a9efe0acd75da35bb7af04bd26dc4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2012.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2012.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DLimm%26aufirst%3DK.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DWallner%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DD.%2BW.%26aulast%3DOefner%26aufirst%3DP.%26aulast%3DHellerbrand%26aufirst%3DC.%26aulast%3DBosserhoff%26aufirst%3DA.%2BK.%26atitle%3DDeregulation%2520of%2520protein%2520methylation%2520in%2520melanoma%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2013%26volume%3D49%26spage%3D1305%26epage%3D1313%26doi%3D10.1016%2Fj.ejca.2012.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maehara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Dysregulation of PRMT1 and PRMT6, Type I arginine methyltransferases, is involved in various types of human cancers</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">573</span><span class="refDoi"> DOI: 10.1002/ijc.25366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fijc.25366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20473859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=562-573&author=M.+Yoshimatsuauthor=G.+Toyokawaauthor=S.+Hayamiauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=J.+D.+Kellyauthor=D.+E.+Nealauthor=Y.+Maeharaauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Dysregulation+of+PRMT1+and+PRMT6%2C+Type+I+arginine+methyltransferases%2C+is+involved+in+various+types+of+human+cancers&doi=10.1002%2Fijc.25366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dysregulation of PRMT1 and PRMT6, Type I arginine methyltransferases, is involved in various types of human cancers</span></div><div class="casAuthors">Yoshimatsu Masanori; Toyokawa Gouji; Hayami Shinya; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Kelly John D; Neal David E; Maehara Yoshihiko; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">562-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methylation is a novel post-translational modification regulating a diversity of cellular processes, including histone functions, but the roles of protein arginine methyltransferases (PRMTs) in human cancer are not well investigated.  To address this issue, we first examined expression levels of genes belonging to the PRMT family and found significantly higher expression of PRMT1 and PRMT6, both of which are Type I PRMTs, in cancer cells of various tissues than in non-neoplastic cells.  Abrogation of the expression of these genes with specific siRNAs significantly suppressed growth of bladder and lung cancer cells.  Expression profile analysis using the cells transfected with the siRNAs indicated that PRMT1 and PRMT6 interplay in multiple pathways, supporting regulatory roles in the cell cycle, RNA processing and also DNA replication that are fundamentally important for cancer cell proliferation.  Furthermore, we demonstrated that serum asymmetric dimethylarginine (ADMA) levels of a number of cancer cases are significantly higher than those of nontumor control cases.  In summary, our results suggest that dysregulation of PRMT1 and PRMT6 can be involved in human carcinogenesis and that these Type I arginine methyltransferases are good therapeutic targets for various types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WjEwO064Z95rfW6udTcc2eZrh7Bm-s9SGLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D&md5=34889d958208a4d852b0833e21a045de</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fijc.25366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25366%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DMaehara%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDysregulation%2520of%2520PRMT1%2520and%2520PRMT6%252C%2520Type%2520I%2520arginine%2520methyltransferases%252C%2520is%2520involved%2520in%2520various%2520types%2520of%2520human%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D562%26epage%3D573%26doi%3D10.1002%2Fijc.25366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Vieira, F. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa-Pinheiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramalho-Carvalho, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menezes, F. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antunes, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carneiro, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrique, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeronimo, C.</span><span> </span><span class="NLM_article-title">Deregulated expression of selected histone methylases and demethylases in prostate carcinoma</span> <span class="citation_source-journal">Endocr.-Relat. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1530/ERC-13-0375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1530%2FERC-13-0375" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=51-61&author=F.+Q.+Vieiraauthor=P.+Costa-Pinheiroauthor=J.+Ramalho-Carvalhoauthor=A.+Pereiraauthor=F.+D.+Menezesauthor=L.+Antunesauthor=I.+Carneiroauthor=J.+Oliveiraauthor=R.+Henriqueauthor=C.+Jeronimo&title=Deregulated+expression+of+selected+histone+methylases+and+demethylases+in+prostate+carcinoma&doi=10.1530%2FERC-13-0375"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1530%2FERC-13-0375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FERC-13-0375%26sid%3Dliteratum%253Aachs%26aulast%3DVieira%26aufirst%3DF.%2BQ.%26aulast%3DCosta-Pinheiro%26aufirst%3DP.%26aulast%3DRamalho-Carvalho%26aufirst%3DJ.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DMenezes%26aufirst%3DF.%2BD.%26aulast%3DAntunes%26aufirst%3DL.%26aulast%3DCarneiro%26aufirst%3DI.%26aulast%3DOliveira%26aufirst%3DJ.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJeronimo%26aufirst%3DC.%26atitle%3DDeregulated%2520expression%2520of%2520selected%2520histone%2520methylases%2520and%2520demethylases%2520in%2520prostate%2520carcinoma%26jtitle%3DEndocr.-Relat.%2520Cancer%26date%3D2014%26volume%3D21%26spage%3D51%26epage%3D61%26doi%3D10.1530%2FERC-13-0375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeRoy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muir, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">H3R42me2a is a histone modification with positive transcriptional effects</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">14894</span><span class="NLM_x">–</span> <span class="NLM_lpage">14899</span><span class="refDoi"> DOI: 10.1073/pnas.1312925110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.1312925110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23980157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFWrsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=14894-14899&author=F.+Casadioauthor=X.+Luauthor=S.+B.+Pollockauthor=G.+LeRoyauthor=B.+A.+Garciaauthor=T.+W.+Muirauthor=R.+G.+Roederauthor=C.+D.+Allis&title=H3R42me2a+is+a+histone+modification+with+positive+transcriptional+effects&doi=10.1073%2Fpnas.1312925110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">H3R42me2a is a histone modification with positive transcriptional effects</span></div><div class="casAuthors">Casadio, Fabio; Lu, Xiangdong; Pollock, Samuel B.; LeRoy, Gary; Garcia, Benjamin A.; Muir, Tom W.; Roeder, Robert G.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">14894-14899,S14894/1-S14894/4</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Histone post-translational modification leads to downstream effects indirectly by allowing or preventing docking of effector mols., or directly by changing the intrinsic biophys. properties of local chromatin.  To date, little has been done to study post-translational modifications that lie outside of the unstructured tail domains of histones.  Core residues, and in particular arginines in H3 and H4, mediate key interactions between the histone octamer and DNA in forming the nucleosomal particle.  Using mass spectrometry, we find that one of these core residues, arginine 42 of histone H3 (H3R42), is dimethylated in mammalian cells by the methyltransferases coactivator arginine methyltransferase 1 (CARM1) and protein arginine methyltransferase 6 (PRMT6) in vitro and in vivo, and we demonstrate that methylation of H3R42 stimulates transcription in vitro from chromatinized templates.  Thus, H3R42 is a new, "nontail" histone methylation site with pos. effects on transcription.  We propose that methylation of basic histone residues at the DNA interface may disrupt histone:DNA interactions, with effects on downstream processes, notably transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBUDvvIfjKRbVg90H21EOLACvtfcHk0lg9BAvZtW5E2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFWrsr3L&md5=eda5ad0f47a66b06d3309fdcaf05d47a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1312925110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1312925110%26sid%3Dliteratum%253Aachs%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPollock%26aufirst%3DS.%2BB.%26aulast%3DLeRoy%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DMuir%26aufirst%3DT.%2BW.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DH3R42me2a%2520is%2520a%2520histone%2520modification%2520with%2520positive%2520transcriptional%2520effects%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D14894%26epage%3D14899%26doi%3D10.1073%2Fpnas.1312925110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Harrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowhan, D. H.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">2201</span><span class="NLM_x">–</span> <span class="NLM_lpage">2216</span><span class="refDoi"> DOI: 10.1093/nar/gkp1203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1093%2Fnar%2Fgkp1203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20047962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=2201-2216&author=M.+J.+Harrisonauthor=Y.+H.+Tangauthor=D.+H.+Dowhan&title=Protein+arginine+methyltransferase+6+regulates+multiple+aspects+of+gene+expression&doi=10.1093%2Fnar%2Fgkp1203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span></div><div class="casAuthors">Harrison, Matthew J.; Tang, Yue Hang; Dowhan, Dennis H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2201-2216</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It is well established that transcription and alternative splicing events are functionally coupled during gene expression.  Here, we report that protein arginine N-methyltransferase 6 (PRMT6) may play a key role in this coupling process by functioning as a transcriptional coactivator that can also regulate alternative splicing.  PRMT6 coactivates the progesterone, glucocorticoid and estrogen receptors in luciferase reporter assays in a hormone-dependent manner.  In addn., small interfering RNA (siRNA) oligonucleotide duplex knockdown of PRMT6 disrupts estrogen-stimulated transcription of endogenous GREB1 and progesterone receptor in MCF-7 breast cancer cells, demonstrating the importance of PRMT6 in hormone-dependent transcription.  In contrast, the regulation of alternative splicing by PRMT6 is hormone independent.  SiRNA knockdown of PRMT6 increases the exon inclusion:skipping ratio of alternatively spliced exons in endogenous vascular endothelial growth factor and spleen tyrosine kinase RNA transcripts in both the presence and absence of estrogen.  These results demonstrate that PRMT6 has a dual role in regulating gene expression and that these two functions can occur independently of each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA8_-JQGR0vbVg90H21EOLACvtfcHk0lg9BAvZtW5E2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D&md5=471281cfc866a6db8be4a7f8c9a009e5</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1203%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DM.%2BJ.%26aulast%3DTang%26aufirst%3DY.%2BH.%26aulast%3DDowhan%26aufirst%3DD.%2BH.%26atitle%3DProtein%2520arginine%2520methyltransferase%25206%2520regulates%2520multiple%2520aspects%2520of%2520gene%2520expression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D2201%26epage%3D2216%26doi%3D10.1093%2Fnar%2Fgkp1203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span><span class="refDoi"> DOI: 10.1038/nchembio.296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe&doi=10.1038%2Fnchembio.296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lg9BAvZtW5E2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161%26doi%3D10.1038%2Fnchembio.296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Chemical probes of histone lysine methyltransferases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/cb500785t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500785t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=40-50&author=H.+U.+Kaniskanauthor=J.+Jin&title=Chemical+probes+of+histone+lysine+methyltransferases&doi=10.1021%2Fcb500785t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes of histone lysine methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-50</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that histone-lysine methyltransferases (HMTs, also known as protein methyltransferases (PMTs)) play an important role in diverse biol. processes and human diseases by regulating gene expression and the chromatin state.  Therefore, HMTs have been increasingly recognized by the biomedical community as a class of potential therapeutic targets.  High quality chem. probes of HMTs, as tools for deciphering their physiol. functions and roles in human diseases and testing therapeutic hypotheses, are crit. for advancing this promising field.  Here, the authors focus on the discovery, characterization, and biol. applications of chem. probes for HMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp8kb_c7Xl57Vg90H21EOLACvtfcHk0lg9BAvZtW5E2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE&md5=4eb8a696e2227c60686df8d3c60777ce</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fcb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500785t%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DChemical%2520probes%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D40%26epage%3D50%26doi%3D10.1021%2Fcb500785t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of protein arginine methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span><span class="refDoi"> DOI: 10.1016/j.str.2012.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+P.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+R.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=P.+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+allosteric+inhibitor+of+protein+arginine+methyltransferase+3&doi=10.1016%2Fj.str.2012.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0lhOPwCvFKZC0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%2BR.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435%26doi%3D10.1016%2Fj.str.2012.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+L.+Liauthor=A.+Q.+Maauthor=E.+Dobrovetskyauthor=A.+P.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lhOPwCvFKZC0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMa%26aufirst%3DA.%2BQ.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin-Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5166</span><span class="NLM_x">–</span> <span class="NLM_lpage">5170</span><span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+U.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+T.+Yuauthor=M.+S.+Eramauthor=X.+B.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+L.+Liauthor=E.+Dobrovetskyauthor=A.+P.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+H.+Zhaoauthor=J.+Jinauthor=M.+Vedadi&title=A+potent%2C+selective+and+cell-active+allosteric+inhibitor+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Uemit; Szewczyk, Magdalena M.; Yu, Zhengtian; Eram, Mohammad S.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Kennedy, Steven; Li, Fengling; Dobrovetsky, Elena; Dong, Aiping; Smil, David; Min, Sun-Joon; Landon, Melissa; Lin-Jones, Jennifer; Huang, Xi-Ping; Roth, Bryan L.; Schapira, Matthieu; Atadja, Peter; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Zhao, Kehao; Jin, Jian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-5170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biol.  Here we report the discovery of the first PRMT3 chem. probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochem., biophys., and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50=31±2 nM, KD=53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chem. probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTk4dFfkDLm7Vg90H21EOLACvtfcHk0lhOPwCvFKZC0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D&md5=283b0875813af0f7631235e8ba7811c2</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%2BT.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%2BB.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520potent%252C%2520selective%2520and%2520cell-active%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Discovery of a dual PRMT5-PRMT7 inhibitor</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">412</span><span class="refDoi"> DOI: 10.1021/ml500467h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500467h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=408-412&author=D.+Smilauthor=M.+S.+Eramauthor=F.+L.+Liauthor=S.+Kennedyauthor=M.+M.+Szewczykauthor=P.+J.+Brownauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+dual+PRMT5-PRMT7+inhibitor&doi=10.1021%2Fml500467h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Dual PRMT5-PRMT7 Inhibitor</span></div><div class="casAuthors">Smil, David; Eram, Mohammad S.; Li, Fengling; Kennedy, Steven; Szewczyk, Magdalena M.; Brown, Peter J.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-412</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein arginine methyltransferases PRMT7 and PRMT5, resp., monomethylate and sym. dimethylate arginine side-chains of proteins involved in diverse cellular mechanisms, including chromatin-mediated control of gene transcription, splicing, and the RAS to ERK transduction cascade.  It is believed that PRMT5 and PRMT7 act in conjunction to methylate their substrates, and genetic deletions support the notion that these enzymes derepress cell proliferation and migration in cancer.  Using available structures of PRMT5, we designed DS-437, a PRMT5 inhibitor with an IC50 value of 6 μM against both PRMT5 and PRMT7 that is inactive against 29 other human protein-, DNA-, and RNA-methyltransferases and inhibits sym. dimethylation of PRMT5 substrates in cells.  This compd. behaves as a cofactor competitor and represents a valid scaffold to interrogate the potential of the PRMT5-PRMT7 axis as a target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9J078gEcjRLVg90H21EOLACvtfcHk0lhknsCeZGmmiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D&md5=8eefe93068418c61881850d3b4d39676</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fml500467h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500467h%26sid%3Dliteratum%253Aachs%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520dual%2520PRMT5-PRMT7%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D408%26epage%3D412%26doi%3D10.1021%2Fml500467h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Speed, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dela Sena, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">781</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+Shenauthor=M.+M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+Liauthor=K.+V.+Butlerauthor=H.+U.+Kaniskanauthor=B.+A.+Speedauthor=C.+Dela+Senaauthor=A.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0lhknsCeZGmmiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Ferreira de Freitas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steuber, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchcock, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moosmayer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stegmann, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1176</span><span class="NLM_x">–</span> <span class="NLM_lpage">1183</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira+de+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0lgBP1J11vv0EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Ferreira de Freitas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Discovery of a Potent and Selective Coactivator Associated Arginine Methyltransferase 1 (CARM1) Inhibitor by Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">6838</span><span class="NLM_x">–</span> <span class="NLM_lpage">6847</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00668</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00668" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6838-6847&author=R.+Ferreira+de+Freitasauthor=M.+S.+Eramauthor=D.+Smilauthor=M.+M.+Szewczykauthor=S.+Kennedyauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+Potent+and+Selective+Coactivator+Associated+Arginine+Methyltransferase+1+%28CARM1%29+Inhibitor+by+Virtual+Screening&doi=10.1021%2Facs.jmedchem.6b00668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Coactivator Associated Arginine Methyltransferase 1 (CARM1) Inhibitor by Virtual Screening</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Smil, David; Szewczyk, Magdalena M.; Kennedy, Steven; Brown, Peter J.; Santhakumar, Vijayaratnam; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6838-6847</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) represent an emerging target class in oncol. and other disease areas.  So far, the most successful strategy to identify PRMT inhibitors has been to screen large to medium-size chem. libraries.  Attempts to develop PRMT inhibitors using receptor-based computational methods have met limited success.  Here, using virtual screening approaches, we identify 11 CARM1 (PRMT4) inhibitors with ligand efficiencies ranging from 0.28 to 0.84.  CARM1 selective hits were further validated by orthogonal methods.  Two structure-based rounds of optimization produced 27 (SGC2085), a CARM1 inhibitor with an IC50 of 50 nM and more than hundred-fold selectivity over other PRMTs.  These results indicate that virtual screening strategies can be successfully applied to Rossmann-fold protein methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF-4WiwX0trVg90H21EOLACvtfcHk0lgBP1J11vv0EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I&md5=6a5daefe79f879bd732df11cf5b31e8a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00668%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520Coactivator%2520Associated%2520Arginine%2520Methyltransferase%25201%2520%2528CARM1%2529%2520Inhibitor%2520by%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6838%26epage%3D6847%26doi%3D10.1021%2Facs.jmedchem.6b00668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Sack, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thieffine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandiera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fasolini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duke, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kish, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosettani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troiani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, J. A.</span><span> </span><span class="NLM_article-title">Structural basis for CARM1 inhibition by indole and pyrazole inhibitors</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">436</span><span class="NLM_x">, </span> <span class="NLM_fpage">331</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1042/BJ20102161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1042%2FBJ20102161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21410432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFGitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2011&pages=331-339&author=J.+S.+Sackauthor=S.+Thieffineauthor=T.+Bandieraauthor=M.+Fasoliniauthor=G.+J.+Dukeauthor=L.+Jayaramanauthor=K.+F.+Kishauthor=H.+E.+Kleiauthor=A.+V.+Purandareauthor=P.+Rosettaniauthor=S.+Troianiauthor=D.+L.+Xieauthor=J.+A.+Bertrand&title=Structural+basis+for+CARM1+inhibition+by+indole+and+pyrazole+inhibitors&doi=10.1042%2FBJ20102161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for CARM1 inhibition by indole and pyrazole inhibitors</span></div><div class="casAuthors">Sack, John S.; Thieffine, Sandrine; Bandiera, Tiziano; Fasolini, Marina; Duke, Gerald J.; Jayaraman, Lata; Kish, Kevin F.; Klei, Herbert E.; Purandare, Ashok V.; Rosettani, Pamela; Troiani, Sonia; Xie, Dianlin; Bertrand, Jay A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-339</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">CARM1 (co-activator-assocd. arginine methyltransferase 1) is a PRMT (protein arginine N-methyltransferase) family member that catalyzes the transfer of Me groups from SAM (S-adenosylmethionine) to the side chain of specific arginine residues of substrate proteins.  This post-translational modification of proteins regulates a variety of transcriptional events and other cellular processes.  Moreover, CARM1 is a potential oncol. target due to its multiple roles in transcription activation by nuclear hormone receptors and other transcription factors such as p53.  Here, crystal structures of the CARM1 catalytic domain in complex with cofactors [SAH (S-adenosyl-L-homocysteine) or SNF (sinefungin)] and indole or pyrazole inhibitors are presented.  Anal. of the structures reveals that the inhibitors bind in the arginine-binding cavity and the surrounding pocket that exists at the interface between the N- and C-terminal domains.  In addn., it is shown using ITC (isothermal titrn. calorimetry) that the inhibitors bind to the CARM1 catalytic domain only in the presence of the cofactor SAH.  Furthermore, sequence differences for select residues that interact with the inhibitors may be responsible for the CARM1 selectivity against PRMT1 and PRMT3.  Together, the structural and biophys. information should aid in the design of both potent and specific inhibitors of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeVM8tGrZkjrVg90H21EOLACvtfcHk0lgBP1J11vv0EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFGitro%253D&md5=029cfceb945caac01a19d39f247c4c0a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1042%2FBJ20102161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20102161%26sid%3Dliteratum%253Aachs%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DThieffine%26aufirst%3DS.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DFasolini%26aufirst%3DM.%26aulast%3DDuke%26aufirst%3DG.%2BJ.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DKish%26aufirst%3DK.%2BF.%26aulast%3DKlei%26aufirst%3DH.%2BE.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26aulast%3DRosettani%26aufirst%3DP.%26aulast%3DTroiani%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DD.%2BL.%26aulast%3DBertrand%26aufirst%3DJ.%2BA.%26atitle%3DStructural%2520basis%2520for%2520CARM1%2520inhibition%2520by%2520indole%2520and%2520pyrazole%2520inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2011%26volume%3D436%26spage%3D331%26epage%3D339%26doi%3D10.1042%2FBJ20102161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Allan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manku, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Styhler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulet, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petschner, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahil, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacLeod, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deziel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besterman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahhab, A.</span><span> </span><span class="NLM_article-title">N-Benzyl-1-heteroaryl-3-(trifluoromethyl)-1H-pyrazole-5-carboxamides as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1218</span><span class="NLM_x">–</span> <span class="NLM_lpage">1223</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.12.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2008.12.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19131248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Wruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1218-1223&author=M.+Allanauthor=S.+Mankuauthor=E.+Therrienauthor=N.+Nguyenauthor=S.+Styhlerauthor=M.+F.+Robertauthor=A.+C.+Gouletauthor=A.+J.+Petschnerauthor=G.+Rahilauthor=A.+R.+MacLeodauthor=R.+Dezielauthor=J.+M.+Bestermanauthor=H.+Nguyenauthor=A.+Wahhab&title=N-Benzyl-1-heteroaryl-3-%28trifluoromethyl%29-1H-pyrazole-5-carboxamides+as+inhibitors+of+co-activator+associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2008.12.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">N-Benzyl-1-heteroaryl-3-(trifluoromethyl)-1H-pyrazole-5-carboxamides as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Allan, Martin; Manku, Sukhdev; Therrien, Eric; Nguyen, Natalie; Styhler, Sylvia; Robert, Marie-France; Goulet, Anne-Christine; Petschner, Andrea J.; Rahil, Gabi; Robert MacLeod, A.; Deziel, Robert; Besterman, Jeffrey M.; Nguyen, Hannah; Wahhab, Amal</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1218-1223</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of N-benzyl-1-heteroaryl-3-(trifluoromethyl)-1H-pyrazole-5-carboxamides, e.g. I (R = Ph, 2-MeC6H4, 2-MeOC6H4, 2-FC6H4, 3-MeOC6H4, 2-MeO2CC6H4, 2-pyridyl), targeting co-activator assocd. arginine methyltransferase 1 (CARM1) was designed and synthesized.  The potency of these inhibitors was influenced by the nature of the heteroaryl fragment with the thiophene analogs being superior to thiazole, pyridine, isoindoline and benzofuran based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow4qukRyeUALVg90H21EOLACvtfcHk0lge1I5n9Lsxlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Wruro%253D&md5=68aece67d648b302a99f117eb872c2e2</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.12.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.12.075%26sid%3Dliteratum%253Aachs%26aulast%3DAllan%26aufirst%3DM.%26aulast%3DManku%26aufirst%3DS.%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DStyhler%26aufirst%3DS.%26aulast%3DRobert%26aufirst%3DM.%2BF.%26aulast%3DGoulet%26aufirst%3DA.%2BC.%26aulast%3DPetschner%26aufirst%3DA.%2BJ.%26aulast%3DRahil%26aufirst%3DG.%26aulast%3DMacLeod%26aufirst%3DA.%2BR.%26aulast%3DDeziel%26aufirst%3DR.%26aulast%3DBesterman%26aufirst%3DJ.%2BM.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DWahhab%26aufirst%3DA.%26atitle%3DN-Benzyl-1-heteroaryl-3-%2528trifluoromethyl%2529-1H-pyrazole-5-carboxamides%2520as%2520inhibitors%2520of%2520co-activator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1218%26epage%3D1223%26doi%3D10.1016%2Fj.bmcl.2008.12.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milite, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limongelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of carboxy analogues of arginine methyltransferase inhibitor 1 (AMI-1)</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">398</span><span class="NLM_x">–</span> <span class="NLM_lpage">414</span><span class="refDoi"> DOI: 10.1002/cmdc.200900459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fcmdc.200900459" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=398-414&author=S.+Castellanoauthor=C.+Militeauthor=R.+Ragnoauthor=S.+Simeoniauthor=A.+Maiauthor=V.+Limongelliauthor=E.+Novellinoauthor=I.+Bauerauthor=G.+Broschauthor=A.+Spannhoffauthor=D.+H.+Chengauthor=M.+T.+Bedfordauthor=G.+Sbardella&title=Design%2C+synthesis+and+biological+evaluation+of+carboxy+analogues+of+arginine+methyltransferase+inhibitor+1+%28AMI-1%29&doi=10.1002%2Fcmdc.200900459"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900459%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DMilite%26aufirst%3DC.%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DLimongelli%26aufirst%3DV.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520carboxy%2520analogues%2520of%2520arginine%2520methyltransferase%2520inhibitor%25201%2520%2528AMI-1%2529%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D398%26epage%3D414%26doi%3D10.1002%2Fcmdc.200900459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandiera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bindi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roletto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thieffine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenzi, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottardis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span> </span><span class="NLM_article-title">Optimization of pyrazole inhibitors of Coactivator Associated Arginine Methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2924</span><span class="NLM_x">–</span> <span class="NLM_lpage">2927</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2009.04.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19419866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2924-2927&author=T.+Huynhauthor=Z.+Chenauthor=S.+H.+Pangauthor=J.+P.+Gengauthor=T.+Bandieraauthor=S.+Bindiauthor=P.+Vianelloauthor=F.+Rolettoauthor=S.+Thieffineauthor=A.+Galvaniauthor=W.+Vaccaroauthor=M.+A.+Possauthor=G.+L.+Trainorauthor=M.+V.+Lorenziauthor=M.+Gottardisauthor=L.+Jayaramanauthor=A.+V.+Purandare&title=Optimization+of+pyrazole+inhibitors+of+Coactivator+Associated+Arginine+Methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2009.04.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Huynh, Tram; Chen, Zhong; Pang, Suhong; Geng, Jieping; Bandiera, Tiziano; Bindi, Simona; Vianello, Paola; Roletto, Fulvia; Thieffine, Sandrine; Galvani, Arturo; Vaccaro, Wayne; Poss, Michael A.; Trainor, George L.; Lorenzi, Matthew V.; Gottardis, Marco; Jayaraman, Lata; Purandare, Ashok V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2924-2927</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Design, synthesis, and SAR development led to the identification of the potent, novel, and selective pyrazole based inhibitor I of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphFardNepde7Vg90H21EOLACvtfcHk0lge1I5n9Lsxlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D&md5=c37a2738bbd001d4758f8480746626fb</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.075%26sid%3Dliteratum%253Aachs%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3DGeng%26aufirst%3DJ.%2BP.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DBindi%26aufirst%3DS.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DRoletto%26aufirst%3DF.%26aulast%3DThieffine%26aufirst%3DS.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DGottardis%26aufirst%3DM.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26atitle%3DOptimization%2520of%2520pyrazole%2520inhibitors%2520of%2520Coactivator%2520Associated%2520Arginine%2520Methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2924%26epage%3D2927%26doi%3D10.1016%2Fj.bmcl.2009.04.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Purandare, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oconnell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowak, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span> </span><span class="NLM_article-title">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4438</span><span class="NLM_x">–</span> <span class="NLM_lpage">4441</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2008.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=18619839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4438-4441&author=A.+V.+Purandareauthor=Z.+Chenauthor=T.+Huynhauthor=S.+Pangauthor=J.+Gengauthor=W.+Vaccaroauthor=M.+A.+Possauthor=J.+Oconnellauthor=K.+Nowakauthor=L.+Jayaraman&title=Pyrazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2008.06.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Purandare, Ashok V.; Chen, Zhong; Huynh, Tram; Pang, Suhong; Geng, Jieping; Vaccaro, Wayne; Poss, Michael A.; Oconnell, Jonathan; Nowak, Kimberly; Jayaraman, Lata</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4438-4441</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">This study reports the identification and Hits to Leads optimization of inhibitors of coactivator assocd. arginine methyltransferase (CARM1).  Compd. (I) is a potent, selective inhibitor of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjsEj-ubHXsrVg90H21EOLACvtfcHk0lhu8AtERXeX_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D&md5=0ef68fcfa0dedbe54d1fa5b8ae9ca376</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DOconnell%26aufirst%3DJ.%26aulast%3DNowak%26aufirst%3DK.%26aulast%3DJayaraman%26aufirst%3DL.%26atitle%3DPyrazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4438%26epage%3D4441%26doi%3D10.1016%2Fj.bmcl.2008.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larouche, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manku, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Styhler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulet, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besterman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahhab, A.</span><span> </span><span class="NLM_article-title">1,2-Diamines as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">6725</span><span class="NLM_x">–</span> <span class="NLM_lpage">6732</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.09.110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2009.09.110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19836951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6725-6732&author=E.+Therrienauthor=G.+Laroucheauthor=S.+Mankuauthor=M.+Allanauthor=N.+Nguyenauthor=S.+Styhlerauthor=M.+F.+Robertauthor=A.+C.+Gouletauthor=J.+M.+Bestermanauthor=H.+Nguyenauthor=A.+Wahhab&title=1%2C2-Diamines+as+inhibitors+of+co-activator+associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2009.09.110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">1,2-Diamines as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Therrien, Eric; Larouche, Guillaume; Manku, Sukhdev; Allan, Martin; Nguyen, Natalie; Styhler, Sylvia; Robert, Marie-France; Goulet, Anne-Christine; Besterman, Jeffrey M.; Nguyen, Hannah; Wahhab, Amal</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6725-6732</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have identified the N 1-benzyl-N 2-methylethane-1,2-diamine unit as a substitute for the (S)-alanine benzylamide moiety for the design of co-activator assocd. arginine methyltransferase 1 (CARM1) inhibitors.  The potency of these inhibitors is in the same order of magnitude as their predecessors and their clearance, vol. of distribution, and half lives were greatly improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_1G-CFEVWrVg90H21EOLACvtfcHk0lhu8AtERXeX_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7I&md5=808ff34d43b9beb4efbf149c253acf6c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.09.110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.09.110%26sid%3Dliteratum%253Aachs%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DLarouche%26aufirst%3DG.%26aulast%3DManku%26aufirst%3DS.%26aulast%3DAllan%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DStyhler%26aufirst%3DS.%26aulast%3DRobert%26aufirst%3DM.%2BF.%26aulast%3DGoulet%26aufirst%3DA.%2BC.%26aulast%3DBesterman%26aufirst%3DJ.%2BM.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DWahhab%26aufirst%3DA.%26atitle%3D1%252C2-Diamines%2520as%2520inhibitors%2520of%2520co-activator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6725%26epage%3D6732%26doi%3D10.1016%2Fj.bmcl.2009.09.110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Wan, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenzi, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottardis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span> </span><span class="NLM_article-title">Benzo[d]imidazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)-hit to lead studies</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">5063</span><span class="NLM_x">–</span> <span class="NLM_lpage">5066</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.07.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2009.07.040" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=5063-5066&author=H.+H.+Wanauthor=T.+Huynhauthor=S.+H.+Pangauthor=J.+P.+Gengauthor=W.+Vaccaroauthor=M.+A.+Possauthor=G.+L.+Trainorauthor=M.+V.+Lorenziauthor=M.+Gottardisauthor=L.+Jayaramanauthor=A.+V.+Purandare&title=Benzo%5Bd%5Dimidazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29-hit+to+lead+studies&doi=10.1016%2Fj.bmcl.2009.07.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.07.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.07.040%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DH.%2BH.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3DGeng%26aufirst%3DJ.%2BP.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DGottardis%26aufirst%3DM.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26atitle%3DBenzo%255Bd%255Dimidazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529-hit%2520to%2520lead%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D5063%26epage%3D5066%26doi%3D10.1016%2Fj.bmcl.2009.07.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Mitchell, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drew, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swinger, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moradei, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter-Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: identification of the first PRMT6 tool compound</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0lhu8AtERXeX_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuplast, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stickland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbash, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nurse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oza, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallagher, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0ljAyVRGqmlujw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">van Haren, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Ufford, L. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moret, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N. I.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of protein arginine N-methyltransferase inhibitors designed to simultaneously occupy both substrate binding sites</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">560</span><span class="refDoi"> DOI: 10.1039/C4OB01734J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1039%2FC4OB01734J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=25380215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSiu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=549-560&author=M.+van+Harenauthor=L.+Q.+van+Uffordauthor=E.+E.+Moretauthor=N.+I.+Martin&title=Synthesis+and+evaluation+of+protein+arginine+N-methyltransferase+inhibitors+designed+to+simultaneously+occupy+both+substrate+binding+sites&doi=10.1039%2FC4OB01734J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of protein arginine N-methyltransferase inhibitors designed to simultaneously occupy both substrate binding sites</span></div><div class="casAuthors">van Haren, Matthijs; van Ufford, Linda Quarles; Moret, Ed E.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein arginine N-methyltransferases (PRMTs) are a family of enzymes that function by specifically transferring a Me group from the cofactor S-adenosyl-L-methionine (AdoMet) to the guanidine group of arginine residues in target proteins.  The most notable is the PRMT-mediated methylation of arginine residues that are present in histone proteins which can lead to chromatin remodelling and influence gene transcription.  A growing body of evidence now implicates dysregulated PRMT activity in a no. of diseases including various forms of cancer.  The development of PRMT inhibitors may therefore hold potential as a means of developing new therapeutics.  We here report the synthesis and evaluation of a series of small mol. PRMT inhibitors designed to simultaneously occupy the binding sites of both the guanidino substrate and AdoMet cofactor.  Potent inhibition and surprising selectivity were obsd. when testing these compds. against a panel of methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcFfPMCcpn97Vg90H21EOLACvtfcHk0ljAyVRGqmlujw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSiu77N&md5=62b07b10e6cc1c288168e8bd0ef4d9f5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2FC4OB01734J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01734J%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%26aulast%3Dvan%2BUfford%26aufirst%3DL.%2BQ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520protein%2520arginine%2520N-methyltransferase%2520inhibitors%2520designed%2520to%2520simultaneously%2520occupy%2520both%2520substrate%2520binding%2520sites%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D549%26epage%3D560%26doi%3D10.1039%2FC4OB01734J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martini, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Design and synthesis of selective, small molecule inhibitors of coactivator-associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1039/C6MD00342G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1039%2FC6MD00342G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=H.+U.+Kaniskanauthor=M.+S.+Eramauthor=J.+Liuauthor=D.+Smilauthor=M.+L.+Martiniauthor=Y.+Shenauthor=V.+Santhakumarauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=J.+Jin&title=Design+and+synthesis+of+selective%2C+small+molecule+inhibitors+of+coactivator-associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1039%2FC6MD00342G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1039%2FC6MD00342G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00342G%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520selective%252C%2520small%2520molecule%2520inhibitors%2520of%2520coactivator-associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DMedChemComm%26date%3D2016%26doi%3D10.1039%2FC6MD00342G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0liOf6SSpkE6uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2212</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span><span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0liOf6SSpkE6uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Blat, Y.</span><span> </span><span class="NLM_article-title">Non-competitive inhibition by active site binders</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">535</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2010.00972.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1111%2Fj.1747-0285.2010.00972.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20374252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtlaru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=535-540&author=Y.+Blat&title=Non-competitive+inhibition+by+active+site+binders&doi=10.1111%2Fj.1747-0285.2010.00972.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Non-competitive inhibition by active site binders</span></div><div class="casAuthors">Blat, Yuval</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">535-540</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Classical enzymol. has been used for generations to understand the interactions of inhibitors with their enzyme targets.  Enzymol. tools enabled prediction of the biol. impact of inhibitors as well as the development of novel, more potent, ones.  Expts. designed to examine the competition between the tested inhibitor and the enzyme substrate(s) are the tool of choice to identify inhibitors that bind in the active site.  Competition between an inhibitor and a substrate is considered a strong evidence for binding of the inhibitor in the active site, while the lack of competition suggests binding to an alternative site.  Nevertheless, exceptions to this notion do exist.  Active site-binding inhibitors can display non-competitive inhibition patterns.  This unusual behavior has been obsd. with enzymes utilizing an exosite for substrate binding, isomechanism enzymes, enzymes with multiple substrates and/or products and two-step binding inhibitors.  In many of these cases, the mechanisms underlying the lack of competition between the substrate and the inhibitor are well understood.  Tools like alternative substrates, testing the enzyme reaction in the reverse direction and monitoring inhibition time dependence can be applied to enable distinction between 'badly behaving' active site binders and true exosite inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQh6oNO4kuy7Vg90H21EOLACvtfcHk0liOf6SSpkE6uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtlaru7g%253D&md5=fc61590b33e72e75999c2fdd0da716a6</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.00972.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.00972.x%26sid%3Dliteratum%253Aachs%26aulast%3DBlat%26aufirst%3DY.%26atitle%3DNon-competitive%2520inhibition%2520by%2520active%2520site%2520binders%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D75%26spage%3D535%26epage%3D540%26doi%3D10.1111%2Fj.1747-0285.2010.00972.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Mitchell, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomenius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riera, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span> </span><span class="NLM_article-title">Novel Oxindole Sulfonamides and Sulfamides: EPZ031686, the First Orally Bioavailable Small Molecule SMYD3 Inhibitor</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=134-138&author=L.+H.+Mitchellauthor=P.+A.+Boriack-Sjodinauthor=S.+Smithauthor=M.+Thomeniusauthor=N.+Riouxauthor=M.+Munchhofauthor=J.+E.+Millsauthor=C.+Klausauthor=J.+Totmanauthor=T.+V.+Rieraauthor=A.+Raimondiauthor=S.+L.+Jacquesauthor=K.+Westauthor=M.+Foleyauthor=N.+J.+Watersauthor=K.+W.+Kuntzauthor=T.+J.+Wigleauthor=M.+P.+Scottauthor=R.+A.+Copelandauthor=J.+J.+Smithauthor=R.+Chesworth&title=Novel+Oxindole+Sulfonamides+and+Sulfamides%3A+EPZ031686%2C+the+First+Orally+Bioavailable+Small+Molecule+SMYD3+Inhibitor&doi=10.1021%2Facsmedchemlett.5b00272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oxindole Sulfonamides and Sulfamides: EPZ031686, the First Orally Bioavailable Small Molecule SMYD3 Inhibitor</span></div><div class="casAuthors">Mitchell, Lorna H.; Boriack-Sjodin, P. Ann; Smith, Sherri; Thomenius, Michael; Rioux, Nathalie; Munchhof, Michael; Mills, James E.; Klaus, Christine; Totman, Jennifer; Riera, Thomas V.; Raimondi, Alejandra; Jacques, Suzanne L.; West, Kip; Foley, Megan; Waters, Nigel J.; Kuntz, Kevin W.; Wigle, Tim J.; Scott, Margaret Porter; Copeland, Robert A.; Smith, Jesse J.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-138</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMYD3 has been implicated in a range of cancers; however, until now no potent selective small mol. inhibitors have been available for target validation studies.  A novel oxindole series of SMYD3 inhibitors was identified through screening of the Epizyme proprietary histone methyltransferase-biased library.  Potency optimization afforded two tool compds., sulfonamide EPZ031686 and sulfamide EPZ030456, with cellular potency at a level sufficient to probe the in vitro biol. of SMYD3 inhibition.  EPZ031686 shows good bioavailability following oral dosing in mice making it a suitable tool for potential in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQtO_uOhxeYrVg90H21EOLACvtfcHk0ljuJ57XYm_tOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ&md5=3fd7612cb086796a0bcc5c6e0c1f1352</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00272%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DThomenius%26aufirst%3DM.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%26aulast%3DMills%26aufirst%3DJ.%2BE.%26aulast%3DKlaus%26aufirst%3DC.%26aulast%3DTotman%26aufirst%3DJ.%26aulast%3DRiera%26aufirst%3DT.%2BV.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DNovel%2520Oxindole%2520Sulfonamides%2520and%2520Sulfamides%253A%2520EPZ031686%252C%2520the%2520First%2520Orally%2520Bioavailable%2520Small%2520Molecule%2520SMYD3%2520Inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D134%26epage%3D138%26doi%3D10.1021%2Facsmedchemlett.5b00272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Hyllus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnabel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiltz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, U. M.</span><span> </span><span class="NLM_article-title">PRMT6-mediated methylation of R2 in histone H3 antagonizes H3 K4 trimethylation</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">3369</span><span class="NLM_x">–</span> <span class="NLM_lpage">3380</span><span class="refDoi"> DOI: 10.1101/gad.447007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1101%2Fgad.447007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=18079182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrsb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=3369-3380&author=D.+Hyllusauthor=C.+Steinauthor=K.+Schnabelauthor=E.+Schiltzauthor=A.+Imhofauthor=Y.+Douauthor=J.+Hsiehauthor=U.+M.+Bauer&title=PRMT6-mediated+methylation+of+R2+in+histone+H3+antagonizes+H3+K4+trimethylation&doi=10.1101%2Fgad.447007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT6-mediated methylation of R2 in histone H3 antagonizes H3 K4 trimethylation</span></div><div class="casAuthors">Hyllus, Dawin; Stein, Claudia; Schnabel, Kristin; Schiltz, Emile; Imhof, Axel; Dou, Yali; Hsieh, James; Bauer, Uta-Maria</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3369-3380</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The arginine methyltransferase PRMT6 (protein arginine methyltransferase 6) has been shown recently to regulate DNA repair and gene expression.  As arginine methylation of histones is an important mechanism in transcriptional regulation, we asked whether PRMT6 possesses activity toward histones.  We show here that PRMT6 methylates histone H3 at R2 and histones H4/H2A at R3 in vitro.  Overexpression and knockdown anal. identify PRMT6 as the major H3 R2 methyltransferase in vivo.  We find that H3 R2 methylation inhibits H3 K4 trimethylation and recruitment of WDR5, a subunit of the MLL (mixed lineage leukemia) K4 methyltransferase complex, to histone H3 in vitro.  Upon PRMT6 overexpression, transcription of Hox genes and Myc-dependent genes, both well-known targets of H3 K4 trimethylation, decreases.  This transcriptional repression coincides with enhanced occurrence of H3 R2 methylation and PRMT6 as well as reduced levels of H3 K4 trimethylation and MLL1/WDR5 recruitment at the HoxA2 gene.  Upon retinoic acid-induced transcriptional activation of HoxA2 in a cell model of neuronal differentiation, PRMT6 recruitment and H3 R2 methylation are diminished and H3 K4 trimethylation increases at the gene.  Our findings identify PRMT6 as the mammalian methyltransferase for H3 R2 and establish the enzyme as a crucial neg. regulator of H3 K4 trimethylation and transcriptional activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsPUkDIyRzr7Vg90H21EOLACvtfcHk0ljuJ57XYm_tOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrsb%252FE&md5=951b9c1612c9f95b20951dec04e098d2</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1101%2Fgad.447007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.447007%26sid%3Dliteratum%253Aachs%26aulast%3DHyllus%26aufirst%3DD.%26aulast%3DStein%26aufirst%3DC.%26aulast%3DSchnabel%26aufirst%3DK.%26aulast%3DSchiltz%26aufirst%3DE.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DHsieh%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DU.%2BM.%26atitle%3DPRMT6-mediated%2520methylation%2520of%2520R2%2520in%2520histone%2520H3%2520antagonizes%2520H3%2520K4%2520trimethylation%26jtitle%3DGenes%2520Dev.%26date%3D2007%26volume%3D21%26spage%3D3369%26epage%3D3380%26doi%3D10.1101%2Fgad.447007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Waldmann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamieniarz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindner, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittler, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span> </span><span class="NLM_article-title">Methylation of H2AR29 is a novel repressive PRMT6 target</span> <span class="citation_source-journal">Epigenet. Chromatin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="refDoi"> DOI: 10.1186/1756-8935-4-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1186%2F1756-8935-4-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21774791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCktLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=11&author=T.+Waldmannauthor=A.+Izzoauthor=K.+Kamieniarzauthor=F.+Richterauthor=C.+Voglerauthor=B.+Sargauthor=H.+Lindnerauthor=N.+L.+Youngauthor=G.+Mittlerauthor=B.+A.+Garciaauthor=R.+Schneider&title=Methylation+of+H2AR29+is+a+novel+repressive+PRMT6+target&doi=10.1186%2F1756-8935-4-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of H2AR29 is a novel repressive PRMT6 target</span></div><div class="casAuthors">Waldmann, Tanja; Izzo, Annalisa; Kamieniarz, Kinga; Richter, Florian; Vogler, Christine; Sarg, Bettina; Lindner, Herbert; Young, Nicolas L.; Mittler, Gerhard; Garcia, Benjamin A.; Schneider, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Covalent histone modifications are central to all DNA-dependent processes.  Modifications of histones H3 and H4 are becoming well characterized, however, our knowledge of how H2A modifications regulate chromatin dynamics and gene expression is still very limited.  Results: To understand the function of H2A modifications we performed a systematic anal. of the histone H2A methylation status.  We identified and functionally characterized two new methylation sites in H2A: R11 (H2AR11) and R29 (H2AR29).  By using an unbiased biochem. approach in combination with candidate assays we show that the arginine methyltransferase (PRMT) 1 and PRMT6 are unique in their ability to catalyze these modifications.  Importantly we find that H2AR29me2 is specifically enriched at genes repressed by PRMT6, implicating H2AR29me2 in transcriptional repression.  Conclusions: Our data establishes R11 and R29 as new arginine methylation sites in H2A.  We identified the specific modifying enzymes and uncovered a novel functional role of H2AR29me2 in gene silencing in vivo.  Thus this work reveals novel insights into the function of H2A methylation as well as in the mechanisms of PRMT6-mediated transcriptional repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw0BxwOba9E7Vg90H21EOLACvtfcHk0lgO5N9jaQp3ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCktLjL&md5=2c66fd635d53774195965cbb86b94c40</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-4-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-4-11%26sid%3Dliteratum%253Aachs%26aulast%3DWaldmann%26aufirst%3DT.%26aulast%3DIzzo%26aufirst%3DA.%26aulast%3DKamieniarz%26aufirst%3DK.%26aulast%3DRichter%26aufirst%3DF.%26aulast%3DVogler%26aufirst%3DC.%26aulast%3DSarg%26aufirst%3DB.%26aulast%3DLindner%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DN.%2BL.%26aulast%3DMittler%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DSchneider%26aufirst%3DR.%26atitle%3DMethylation%2520of%2520H2AR29%2520is%2520a%2520novel%2520repressive%2520PRMT6%2520target%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2011%26volume%3D4%26spage%3D11%26doi%3D10.1186%2F1756-8935-4-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustos, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Trimethylation of histone H3 lysine 36 by human methyltransferase PRDM9 protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">12177</span><span class="NLM_x">–</span> <span class="NLM_lpage">12188</span><span class="refDoi"> DOI: 10.1074/jbc.M113.523183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M113.523183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=12177-12188&author=M.+S.+Eramauthor=S.+P.+Bustosauthor=E.+Lima-Fernandesauthor=A.+Siarheyevaauthor=G.+Senisterraauthor=T.+Hajianauthor=I.+Chauauthor=S.+Duanauthor=H.+Wuauthor=L.+Dombrovskiauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Trimethylation+of+histone+H3+lysine+36+by+human+methyltransferase+PRDM9+protein&doi=10.1074%2Fjbc.M113.523183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523183%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DBustos%26aufirst%3DS.%2BP.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DTrimethylation%2520of%2520histone%2520H3%2520lysine%252036%2520by%2520human%2520methyltransferase%2520PRDM9%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D12177%26epage%3D12188%26doi%3D10.1074%2Fjbc.M113.523183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">1842</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2015.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bbagen.2015.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26002201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=1842-1848&author=M.+S.+Eramauthor=E.+Kuznetsovaauthor=F.+Liauthor=E.+Lima-Fernandesauthor=S.+Kennedyauthor=I.+Chauauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=Kinetic+characterization+of+human+histone+H3+lysine+36+methyltransferases%2C+ASH1L+and+SETD2&doi=10.1016%2Fj.bbagen.2015.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span></div><div class="casAuthors">Eram, Mohammad S.; Kuznetsova, Ekaterina; Li, Fengling; Lima-Fernandes, Evelyne; Kennedy, Steven; Chau, Irene; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dysregulation of methylation of lysine 36 on histone H3 (H3K36) have been implicated in a variety of diseases including cancers.  ASH1L and SETD2 are two enzymes among others that catalyze H3K36 methylation.  H3K4 methylation has also been reported for ASH1L.  Radioactivity-based enzyme assays, Western and immunoblotting using specific antibodies and mol. modeling were used to characterize substrate specificity of ASH1L and SETD2.  Here we report on the assay development and kinetic characterization of ASH1L and SETD2 and their substrate specificities in vitro.  Both enzymes were active with recombinant nucleosome as substrate.  However, SETD2 but not ASH1L methylated histone peptides as well, indicating that the interaction of the basic post-SET extension with substrate may not be crit. for SETD2 activity.  Neither enzyme was active with nucleosome contg. a H3K36A mutation, indicating their specificity for H3K36.  Analyzing the methylation state of the products of ASH1L and SETD2 reactions also confirmed that both enzymes mono- and dimethylate H3K36 and are inactive with H3K4 as substrate, and that only SETD2 is able to trimethylate H3K36 in vitro.  We detd. the kinetic parameters for ASH1L and SETD2 activity, enabling screening for inhibitors that can be used to further investigate the roles of these two proteins in health and disease.  Both ASH1L and SETD2 are H3K36-specific methyltransferases but only SETD2 can trimethylate this mark.  The basic post-SET extension is crit. for ASH1L but not SETD2 activity.  We provide full kinetic characterization of ASH1L and SETD2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproBY7GkmOLrVg90H21EOLACvtfcHk0lgO5N9jaQp3ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D&md5=64839475d5ee9cc2f20e6e71f8e8a6ea</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DKinetic%2520characterization%2520of%2520human%2520histone%2520H3%2520lysine%252036%2520methyltransferases%252C%2520ASH1L%2520and%2520SETD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bbagen.2015.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.-P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.-C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+S.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=R%29-PFI-2+is+a+potent+and+selective+inhibitor+of+SETD7+methyltransferase+activity+in+cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DR%2529-PFI-2%2520is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520SETD7%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gräslund, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A basic post-SET extension of NSDs is essential for nucleosome binding in vitro</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">928</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1177/1087057114525854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1177%2F1087057114525854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24595546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWgs7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=928-935&author=A.+Allali-Hassaniauthor=E.+Kuznetsovaauthor=T.+Hajianauthor=H.+Wuauthor=L.+Dombrovskiauthor=Y.+Liauthor=S.+Gr%C3%A4slundauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=A+basic+post-SET+extension+of+NSDs+is+essential+for+nucleosome+binding+in+vitro&doi=10.1177%2F1087057114525854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">A basic post-SET extension of nsds is essential for nucleosome binding in vitro</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Kuznetsova, Ekaterina; Hajian, Taraneh; Wu, Hong; Dombrovski, Ludmila; Li, Yanjun; Graslund, Susanne; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">928-935, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The nuclear receptor SET domain-contg. family of proteins (NSD1, NSD2, and NSD3) is known to mono- and di-methylate lysine 36 of histone H3 (H3K36).  Overexpression and translocation of NSDs have been widely implicated in a variety of diseases including cancers.  Although the substrate specificity of NSDs has been a subject of many valuable studies, the activity of these proteins has never been fully characterized in vitro.  In this study, we present full kinetic characterization of NSD1, NSD2, and NSD3 and provide robust in vitro assays suitable for screening these proteins in a 384-well format using nucleosome as a substrate.  Through monitoring the changes in substrate specificity of a series of NSD constructs and using mol. modeling, we show that a basic post-SET extension common to all three NSDs (corresponding to residues 1209 to 1226 of NSD2) is essential for proper positioning on nucleosome substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWv4LcXsVmILVg90H21EOLACvtfcHk0liUp2n3Kc2_iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWgs7jE&md5=d27aad340f0e2fac6cb2d21d322e92fe</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1177%2F1087057114525854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057114525854%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGr%25C3%25A4slund%26aufirst%3DS.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520basic%2520post-SET%2520extension%2520of%2520NSDs%2520is%2520essential%2520for%2520nucleosome%2520binding%2520in%2520vitro%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D928%26epage%3D935%26doi%3D10.1177%2F1087057114525854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szykowska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess-Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span> </span><span class="NLM_article-title">A scintillation proximity assay for histone demethylases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">463</span><span class="NLM_x">, </span> <span class="NLM_fpage">54</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.ab.2014.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.ab.2014.06.023" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2014&pages=54-60&author=W.+Yuauthor=M.+S.+Eramauthor=T.+Hajianauthor=A.+Szykowskaauthor=N.+Burgess-Brownauthor=M.+Vedadiauthor=P.+J.+Brown&title=A+scintillation+proximity+assay+for+histone+demethylases&doi=10.1016%2Fj.ab.2014.06.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2014.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2014.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DA%2520scintillation%2520proximity%2520assay%2520for%2520histone%2520demethylases%26jtitle%3DAnal.%2520Biochem.%26date%3D2014%26volume%3D463%26spage%3D54%26epage%3D60%26doi%3D10.1016%2Fj.ab.2014.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Grebien, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giambruno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avellino, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skucha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V. M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonegger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilban, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerlov, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the Wdr5-MLL interaction in C/EBP[alpha] N-terminal leukemia</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&issue=11&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+M+Huberauthor=A.+Schoneggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+targeting+of+the+Wdr5-MLL+interaction+in+C%2FEBP%5Balpha%5D+N-terminal+leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0liUp2n3Kc2_iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DSchonegger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520Wdr5-MLL%2520interaction%2520in%2520C%252FEBP%255Balpha%255D%2520N-terminal%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finerty, P. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ball, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hui, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigelt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span> </span><span class="NLM_article-title">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">15835</span><span class="NLM_x">–</span> <span class="NLM_lpage">15840</span><span class="refDoi"> DOI: 10.1073/pnas.0605224103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.0605224103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17035505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=15835-15840&author=M.+Vedadiauthor=F.+H.+Niesenauthor=A.+Allali-Hassaniauthor=O.+Y.+Fedorovauthor=P.+J.+Finertyauthor=G.+A.+Wasneyauthor=R.+Yeungauthor=C.+Arrowsmithauthor=L.+J.+Ballauthor=H.+Berglundauthor=R.+Huiauthor=B.+D.+Marsdenauthor=P.+Nordlundauthor=M.+Sundstromauthor=J.+Weigeltauthor=A.+M.+Edwards&title=Chemical+screening+methods+to+identify+ligands+that+promote+protein+stability%2C+protein+crystallization%2C+and+structure+determination&doi=10.1073%2Fpnas.0605224103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span></div><div class="casAuthors">Vedadi, Masoud; Niesen, Frank H.; Allali-Hassani, Abdellah; Fedorov, Oleg Y.; Finerty, Patrick J., Jr.; Wasney, Gregory A.; Yeung, Ron; Arrowsmith, Cheryl; Ball, Linda J.; Berglund, Helena; Hui, Raymond; Marsden, Brian D.; Nordlund, Paer; Sundstrom, Michael; Weigelt, Johan; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15835-15840</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 3D structures of human therapeutic targets are enabling for drug discovery.  However, their purifn. and crystn. remain rate detg.  In individual cases, ligands have been used to increase the success rate of protein purifn. and crystn., but the broad applicability of this approach is unknown.  The authors implemented two screening platforms, based on either fluorometry or static light scattering, to measure the increase in protein thermal stability upon binding of a ligand without the need to monitor enzyme activity.  In total, 221 different proteins from humans and human parasites were screened against one or both of two sorts of small-mol. libraries.  The first library comprised different salts, pH conditions, and commonly found small mols. and was applicable to all proteins.  The second comprised compds. specific for protein families of particular interest (e.g., protein kinases).  In 20 cases, including nine unique human protein kinases, a small mol. was identified that stabilized the proteins and promoted structure detn.  The methods are cost-effective, can be implemented in any lab., promise to increase the success rates of purifying and crystg. human proteins significantly, and identify new ligands for these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4duhcFpFn-bVg90H21EOLACvtfcHk0lifkMRxss79sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN&md5=6515225c1613291deb7bb774d900f605</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0605224103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0605224103%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%2BY.%26aulast%3DFinerty%26aufirst%3DP.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DYeung%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBall%26aufirst%3DL.%2BJ.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DWeigelt%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DChemical%2520screening%2520methods%2520to%2520identify%2520ligands%2520that%2520promote%2520protein%2520stability%252C%2520protein%2520crystallization%252C%2520and%2520structure%2520determination%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D15835%26epage%3D15840%26doi%3D10.1073%2Fpnas.0605224103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 40 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Fengling Li, Magdalena M. Szewczyk, Levon Halabelian, Kwang-su Park, Irene Chau, Aiping Dong, Hong Zeng, He Chen, Fanye Meng, Dalia Barsyte-Lovejoy, Cheryl H. Arrowsmith, Peter J. Brown, Jing Liu, Masoud Vedadi, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5477-5487. <a href="https://doi.org/10.1021/acs.jmedchem.0c00406" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BFirst-in-Class%252BProtein%252BArginine%252BMethyltransferase%252B6%252B%252528PRMT6%252529%252BCovalent%252BInhibitor%26aulast%3DShen%26aufirst%3DYudao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09032020%26date%3D14052020%26date%3D05052020%26volume%3D63%26issue%3D10%26spage%3D5477%26epage%3D5487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zuhao Guo, Zhuqing Zhang, Hong Yang, Danyan Cao, Xiaowei Xu, Xingling Zheng, Danqi Chen, Qi Wang, Yanlian Li, Jian Li, Zhiyan Du, Xin Wang, Lin Chen, Jian Ding, Jingkang Shen, Meiyu Geng, Xun Huang, <span class="NLM_string-name hlFld-ContribAuthor">Bing Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of Potent, Selective Inhibitors of Protein Arginine Methyltransferase 4 against Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (11)
                                     , 5414-5433. <a href="https://doi.org/10.1021/acs.jmedchem.9b00297" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00297</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00297%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252BPotent%25252C%252BSelective%252BInhibitors%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B4%252Bagainst%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DGuo%26aufirst%3DZuhao%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15022019%26date%3D04062019%26date%3D22052019%26volume%3D62%26issue%3D11%26spage%3D5414%26epage%3D5433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zahid Q. Bonday, Guillermo S. Cortez, Michael J. Grogan, Stephen Antonysamy, Ken Weichert, Wayne P. Bocchinfuso, Fengling Li, Steven Kennedy, Binghui Li, Mary M. Mader, Cheryl H. Arrowsmith, Peter J. Brown, Mohammad S. Eram, Magdalena M. Szewczyk, Dalia Barsyte-Lovejoy, Masoud Vedadi, Ernesto Guccione, <span class="NLM_string-name hlFld-ContribAuthor">Robert M. Campbell</span>. </span><span class="cited-content_cbyCitation_article-title">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (7)
                                     , 612-617. <a href="https://doi.org/10.1021/acsmedchemlett.8b00014" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLLY-283%25252C%252Ba%252BPotent%252Band%252BSelective%252BInhibitor%252Bof%252BArginine%252BMethyltransferase%252B5%25252C%252BPRMT5%25252C%252Bwith%252BAntitumor%252BActivity%26aulast%3DBonday%26aufirst%3DZahid%2BQ.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D10012018%26date%3D23042018%26date%3D04052018%26date%3D23042018%26volume%3D9%26issue%3D7%26spage%3D612%26epage%3D617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Léa  Bouché</span>, <span class="hlFld-ContribAuthor ">Clara D.  Christ</span>, <span class="hlFld-ContribAuthor ">Stephan  Siegel</span>, <span class="hlFld-ContribAuthor ">Amaury E.  Fernández-Montalván</span>, <span class="hlFld-ContribAuthor ">Simon J.  Holton</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Antonius  ter Laak</span>, <span class="hlFld-ContribAuthor ">Tatsuo  Sugawara</span>, <span class="hlFld-ContribAuthor ">Detlef  Stöckigt</span>, <span class="hlFld-ContribAuthor ">Cynthia  Tallant</span>, <span class="hlFld-ContribAuthor ">James  Bennett</span>, <span class="hlFld-ContribAuthor ">Octovia  Monteiro</span>, <span class="hlFld-ContribAuthor ">Laura  Díaz-Sáez</span>, <span class="hlFld-ContribAuthor ">Paulina  Siejka</span>, <span class="hlFld-ContribAuthor ">Julia  Meier</span>, <span class="hlFld-ContribAuthor ">Vera  Pütter</span>, <span class="hlFld-ContribAuthor ">Jörg  Weiske</span>, <span class="hlFld-ContribAuthor ">Susanne  Müller</span>, <span class="hlFld-ContribAuthor ">Kilian V. M.  Huber</span>, <span class="hlFld-ContribAuthor ">Ingo V.  Hartung</span>, and <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>  . </span><span class="cited-content_cbyCitation_article-title">Benzoisoquinolinediones as Potent and Selective Inhibitors of BRPF2 and TAF1/TAF1L Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (9)
                                     , 4002-4022. <a href="https://doi.org/10.1021/acs.jmedchem.7b00306" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00306</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00306%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBenzoisoquinolinediones%252Bas%252BPotent%252Band%252BSelective%252BInhibitors%252Bof%252BBRPF2%252Band%252BTAF1%25252FTAF1L%252BBromodomains%26aulast%3DBouch%25C3%25A9%26aufirst%3DL%25C3%25A9a%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D24022017%26date%3D01052017%26date%3D11052017%26date%3D12042017%26volume%3D60%26issue%3D9%26spage%3D4002%26epage%3D4022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Russell A.  Judge</span>, <span class="hlFld-ContribAuthor ">Haizhong  Zhu</span>, <span class="hlFld-ContribAuthor ">Anup K.  Upadhyay</span>, <span class="hlFld-ContribAuthor ">Pierre M.  Bodelle</span>, <span class="hlFld-ContribAuthor ">Charles W.  Hutchins</span>, <span class="hlFld-ContribAuthor ">Maricel  Torrent</span>, <span class="hlFld-ContribAuthor ">Violeta L.  Marin</span>, <span class="hlFld-ContribAuthor ">Wenyu  Yu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">William N.  Pappano</span>, <span class="hlFld-ContribAuthor ">Chaohong  Sun</span>, and <span class="hlFld-ContribAuthor ">Andrew M.  Petros</span>  . </span><span class="cited-content_cbyCitation_article-title">Turning a Substrate Peptide into a Potent Inhibitor for the Histone Methyltransferase SETD8. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>7 </em>
                                    (12)
                                     , 1102-1106. <a href="https://doi.org/10.1021/acsmedchemlett.6b00303" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.6b00303</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.6b00303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.6b00303%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DTurning%252Ba%252BSubstrate%252BPeptide%252Binto%252Ba%252BPotent%252BInhibitor%252Bfor%252Bthe%252BHistone%252BMethyltransferase%252BSETD8%26aulast%3DJudge%26aufirst%3DRussell%2BA.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D03082016%26date%3D11102016%26date%3D17102016%26date%3D08122016%26date%3D11102016%26volume%3D7%26issue%3D12%26spage%3D1102%26epage%3D1106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernesto  Guccione</span>, <span class="hlFld-ContribAuthor ">Megan  Schwarz</span>, <span class="hlFld-ContribAuthor ">Federico  Di Tullio</span>, <span class="hlFld-ContribAuthor ">Slim  Mzoughi</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer synthetic vulnerabilities to protein arginine methyltransferase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Pharmacology</span><span> <strong>2021,</strong> <em>59 </em>, 33-42. <a href="https://doi.org/10.1016/j.coph.2021.04.004" title="DOI URL">https://doi.org/10.1016/j.coph.2021.04.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coph.2021.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coph.2021.04.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Pharmacology%26atitle%3DCancer%252Bsynthetic%252Bvulnerabilities%252Bto%252Bprotein%252Barginine%252Bmethyltransferase%252Binhibitors%26aulast%3DGuccione%26aufirst%3DErnesto%26date%3D2021%26volume%3D59%26spage%3D33%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qin  Wu</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methylation: from enigmatic functions to therapeutic targeting. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (7)
                                     , 509-530. <a href="https://doi.org/10.1038/s41573-021-00159-8" title="DOI URL">https://doi.org/10.1038/s41573-021-00159-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-021-00159-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-021-00159-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DProtein%252Barginine%252Bmethylation%25253A%252Bfrom%252Benigmatic%252Bfunctions%252Bto%252Btherapeutic%252Btargeting%26aulast%3DWu%26aufirst%3DQin%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D7%26spage%3D509%26epage%3D530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jee Won  Hwang</span>, <span class="hlFld-ContribAuthor ">Yena  Cho</span>, <span class="hlFld-ContribAuthor ">Gyu-Un  Bae</span>, <span class="hlFld-ContribAuthor ">Su-Nam  Kim</span>, <span class="hlFld-ContribAuthor ">Yong Kee  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methyltransferases: promising targets for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Experimental & Molecular Medicine</span><span> <strong>2021,</strong> <em>53 </em>
                                    (5)
                                     , 788-808. <a href="https://doi.org/10.1038/s12276-021-00613-y" title="DOI URL">https://doi.org/10.1038/s12276-021-00613-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s12276-021-00613-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs12276-021-00613-y%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520%2526%2520Molecular%2520Medicine%26atitle%3DProtein%252Barginine%252Bmethyltransferases%25253A%252Bpromising%252Btargets%252Bfor%252Bcancer%252Btherapy%26aulast%3DHwang%26aufirst%3DJee%2BWon%26date%3D2021%26date%3D2021%26volume%3D53%26issue%3D5%26spage%3D788%26epage%3D808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mathew  Sutherland</span>, <span class="hlFld-ContribAuthor ">Alice  Li</span>, <span class="hlFld-ContribAuthor ">Anissa  Kaghad</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Panagopoulos</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Magdalena  Szewczyk</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Cora  Scholten</span>, <span class="hlFld-ContribAuthor ">Léa  Bouché</span>, <span class="hlFld-ContribAuthor ">Timo  Stellfeld</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Ingo V.  Hartung</span>, <span class="hlFld-ContribAuthor ">Holger  Steuber</span>, <span class="hlFld-ContribAuthor ">Robert  Britton</span>, <span class="hlFld-ContribAuthor ">Vijayaratnam  Santhakumar</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design and Synthesis of Selective PRMT4 Inhibitors: A New Chemotype for Development of Cancer Therapeutics**. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1116-1125. <a href="https://doi.org/10.1002/cmdc.202100018" title="DOI URL">https://doi.org/10.1002/cmdc.202100018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100018%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DRational%252BDesign%252Band%252BSynthesis%252Bof%252BSelective%252BPRMT4%252BInhibitors%25253A%252BA%252BNew%252BChemotype%252Bfor%252BDevelopment%252Bof%252BCancer%252BTherapeutics%252A%252A%26aulast%3DSutherland%26aufirst%3DMathew%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D7%26spage%3D1116%26epage%3D1125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Paul  Bryant</span>, <span class="hlFld-ContribAuthor ">John  Heiss</span>, <span class="hlFld-ContribAuthor ">Yeshavanth Kumar  Banasavadi-Siddegowda</span>. </span><span class="cited-content_cbyCitation_article-title">Arginine Methylation in Brain Tumors: Tumor Biology and Therapeutic Strategies. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2021,</strong> <em>10 </em>
                                    (1)
                                     , 124. <a href="https://doi.org/10.3390/cells10010124" title="DOI URL">https://doi.org/10.3390/cells10010124</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells10010124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells10010124%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DArginine%252BMethylation%252Bin%252BBrain%252BTumors%25253A%252BTumor%252BBiology%252Band%252BTherapeutic%252BStrategies%26aulast%3DBryant%26aufirst%3DJean-Paul%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D1%26spage%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Richart</span>, <span class="hlFld-ContribAuthor ">Raphaël  Margueron</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging histone methyltransferases in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>56 </em>, 51-62. <a href="https://doi.org/10.1016/j.cbpa.2019.11.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2019.11.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2019.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2019.11.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDrugging%252Bhistone%252Bmethyltransferases%252Bin%252Bcancer%26aulast%3DRichart%26aufirst%3DLaia%26date%3D2020%26volume%3D56%26spage%3D51%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alice Shi Ming  Li</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Albina  Bolotokova</span>, <span class="hlFld-ContribAuthor ">Carlo C.  dela Seña</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical probes for protein arginine methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2020,</strong> <em>175 </em>, 30-43. <a href="https://doi.org/10.1016/j.ymeth.2019.11.017" title="DOI URL">https://doi.org/10.1016/j.ymeth.2019.11.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2019.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2019.11.017%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DChemical%252Bprobes%252Bfor%252Bprotein%252Barginine%252Bmethyltransferases%26aulast%3DLi%26aufirst%3DAlice%2BShi%2BMing%26date%3D2020%26volume%3D175%26spage%3D30%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabio  Stossi</span>, <span class="hlFld-ContribAuthor ">Radhika D  Dandekar</span>, <span class="hlFld-ContribAuthor ">Maureen G  Mancini</span>, <span class="hlFld-ContribAuthor ">Guowei  Gu</span>, <span class="hlFld-ContribAuthor ">Suzanne A W  Fuqua</span>, <span class="hlFld-ContribAuthor ">Agostina  Nardone</span>, <span class="hlFld-ContribAuthor ">Carmine  De Angelis</span>, <span class="hlFld-ContribAuthor ">Xiaoyong  Fu</span>, <span class="hlFld-ContribAuthor ">Rachel  Schiff</span>, <span class="hlFld-ContribAuthor ">Mark T  Bedford</span>, <span class="hlFld-ContribAuthor ">Wei  Xu</span>, <span class="hlFld-ContribAuthor ">Hans E  Johansson</span>, <span class="hlFld-ContribAuthor ">Clifford C  Stephan</span>, <span class="hlFld-ContribAuthor ">Michael A  Mancini</span>. </span><span class="cited-content_cbyCitation_article-title">Estrogen-induced transcription at individual alleles is independent of receptor level and active conformation but can be modulated by coactivators activity. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2020,</strong> <em>48 </em>
                                    (4)
                                     , 1800-1810. <a href="https://doi.org/10.1093/nar/gkz1172" title="DOI URL">https://doi.org/10.1093/nar/gkz1172</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz1172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz1172%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DEstrogen-induced%252Btranscription%252Bat%252Bindividual%252Balleles%252Bis%252Bindependent%252Bof%252Breceptor%252Blevel%252Band%252Bactive%252Bconformation%252Bbut%252Bcan%252Bbe%252Bmodulated%252Bby%252Bcoactivators%252Bactivity%26aulast%3DStossi%26aufirst%3DFabio%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D4%26spage%3D1800%26epage%3D1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiwei  Liu</span>, <span class="hlFld-ContribAuthor ">Jingxin  Feng</span>, <span class="hlFld-ContribAuthor ">Lili  Li</span>, <span class="hlFld-ContribAuthor ">Luyao  Lin</span>, <span class="hlFld-ContribAuthor ">Jiafei  Ji</span>, <span class="hlFld-ContribAuthor ">Cong  Lin</span>, <span class="hlFld-ContribAuthor ">Lingxia  Liu</span>, <span class="hlFld-ContribAuthor ">Na  Zhang</span>, <span class="hlFld-ContribAuthor ">Dandan  Duan</span>, <span class="hlFld-ContribAuthor ">Zhongwei  Li</span>, <span class="hlFld-ContribAuthor ">Baiqu  Huang</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jun  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Arginine methylation‐dependent LSD1 stability promotes invasion and metastasis of breast cancer. </span><span class="cited-content_cbyCitation_journal-name">EMBO reports</span><span> <strong>2020,</strong> <em>21 </em>
                                    (2)
                                     <a href="https://doi.org/10.15252/embr.201948597" title="DOI URL">https://doi.org/10.15252/embr.201948597</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15252/embr.201948597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15252%2Fembr.201948597%26sid%3Dliteratum%253Aachs%26jtitle%3DEMBO%2520reports%26atitle%3DArginine%252Bmethylation%2525E2%252580%252590dependent%252BLSD1%252Bstability%252Bpromotes%252Binvasion%252Band%252Bmetastasis%252Bof%252Bbreast%252Bcancer%26aulast%3DLiu%26aufirst%3DJiwei%26date%3D2020%26date%3D2019%26volume%3D21%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sebastian  Scheer</span>, <span class="hlFld-ContribAuthor ">Suzanne  Ackloo</span>, <span class="hlFld-ContribAuthor ">Tiago S.  Medina</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Ward</span>, <span class="hlFld-ContribAuthor ">Andrew M.  Lewis</span>, <span class="hlFld-ContribAuthor ">Jeffrey P.  Northrop</span>, <span class="hlFld-ContribAuthor ">Paul L.  Richardson</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yudao  Shen</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">David  McLeod</span>, <span class="hlFld-ContribAuthor ">Carlos A.  Zepeda-Velazquez</span>, <span class="hlFld-ContribAuthor ">Minkui  Luo</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Kilian V. M.  Huber</span>, <span class="hlFld-ContribAuthor ">Daniel D.  De Carvalho</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Colby  Zaph</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>. </span><span class="cited-content_cbyCitation_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2019,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-018-07905-4" title="DOI URL">https://doi.org/10.1038/s41467-018-07905-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DA%252Bchemical%252Bbiology%252Btoolbox%252Bto%252Bstudy%252Bprotein%252Bmethyltransferases%252Band%252Bepigenetic%252Bsignaling%26aulast%3DScheer%26aufirst%3DSebastian%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuhei  Ohta</span>, <span class="hlFld-ContribAuthor ">Hiroo  Wakita</span>, <span class="hlFld-ContribAuthor ">Mitsuyasu  Kawaguchi</span>, <span class="hlFld-ContribAuthor ">Naoya  Ieda</span>, <span class="hlFld-ContribAuthor ">Shigehiro  Osada</span>, <span class="hlFld-ContribAuthor ">Hidehiko  Nakagawa</span>. </span><span class="cited-content_cbyCitation_article-title">Ratiometric assay of CARM1 activity using a FRET-based fluorescent probe. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (22)
                                     , 126728. <a href="https://doi.org/10.1016/j.bmcl.2019.126728" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.126728</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.126728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.126728%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRatiometric%252Bassay%252Bof%252BCARM1%252Bactivity%252Busing%252Ba%252BFRET-based%252Bfluorescent%252Bprobe%26aulast%3DOhta%26aufirst%3DYuhei%26date%3D2019%26volume%3D29%26issue%3D22%26spage%3D126728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James  Jarrold</span>, <span class="hlFld-ContribAuthor ">Clare C.  Davies</span>. </span><span class="cited-content_cbyCitation_article-title">PRMTs and Arginine Methylation: Cancer’s Best-Kept Secret?. </span><span class="cited-content_cbyCitation_journal-name">Trends in Molecular Medicine</span><span> <strong>2019,</strong> <em>25 </em>
                                    (11)
                                     , 993-1009. <a href="https://doi.org/10.1016/j.molmed.2019.05.007" title="DOI URL">https://doi.org/10.1016/j.molmed.2019.05.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molmed.2019.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molmed.2019.05.007%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Molecular%2520Medicine%26atitle%3DPRMTs%252Band%252BArginine%252BMethylation%25253A%252BCancer%2525E2%252580%252599s%252BBest-Kept%252BSecret%25253F%26aulast%3DJarrold%26aufirst%3DJames%26date%3D2019%26volume%3D25%26issue%3D11%26spage%3D993%26epage%3D1009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernesto  Guccione</span>, <span class="hlFld-ContribAuthor ">Stéphane  Richard</span>. </span><span class="cited-content_cbyCitation_article-title">The regulation, functions and clinical relevance of arginine methylation. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Molecular Cell Biology</span><span> <strong>2019,</strong> <em>20 </em>
                                    (10)
                                     , 642-657. <a href="https://doi.org/10.1038/s41580-019-0155-x" title="DOI URL">https://doi.org/10.1038/s41580-019-0155-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41580-019-0155-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41580-019-0155-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Molecular%2520Cell%2520Biology%26atitle%3DThe%252Bregulation%25252C%252Bfunctions%252Band%252Bclinical%252Brelevance%252Bof%252Barginine%252Bmethylation%26aulast%3DGuccione%26aufirst%3DErnesto%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D10%26spage%3D642%26epage%3D657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Dilworth</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein methylation: from chemical tools to precision medicines. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2967-2985. <a href="https://doi.org/10.1007/s00018-019-03147-9" title="DOI URL">https://doi.org/10.1007/s00018-019-03147-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03147-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03147-9%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DTargeting%252Bprotein%252Bmethylation%25253A%252Bfrom%252Bchemical%252Btools%252Bto%252Bprecision%252Bmedicines%26aulast%3DDilworth%26aufirst%3DDavid%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2967%26epage%3D2985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kwi‐Hoon  Jang</span>, <span class="hlFld-ContribAuthor ">Han‐Na  Yoon</span>, <span class="hlFld-ContribAuthor ">Jongeun  Lee</span>, <span class="hlFld-ContribAuthor ">Hayan  Yi</span>, <span class="hlFld-ContribAuthor ">Sang‐Yoon  Park</span>, <span class="hlFld-ContribAuthor ">So‐Young  Lee</span>, <span class="hlFld-ContribAuthor ">Younglan  Lim</span>, <span class="hlFld-ContribAuthor ">Hyoung‐Joo  Lee</span>, <span class="hlFld-ContribAuthor ">Jin‐Won  Cho</span>, <span class="hlFld-ContribAuthor ">Young‐Ki  Paik</span>, <span class="hlFld-ContribAuthor ">Williams S.  Hancock</span>, <span class="hlFld-ContribAuthor ">Nam‐On  Ku</span>. </span><span class="cited-content_cbyCitation_article-title">Liver disease–associated keratin 8 and 18 mutations modulate keratin acetylation and methylation. </span><span class="cited-content_cbyCitation_journal-name">The FASEB Journal</span><span> <strong>2019,</strong> <em>33 </em>
                                    (8)
                                     , 9030-9043. <a href="https://doi.org/10.1096/fj.201800263RR" title="DOI URL">https://doi.org/10.1096/fj.201800263RR</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1096/fj.201800263RR&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1096%2Ffj.201800263RR%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FASEB%2520Journal%26atitle%3DLiver%252Bdisease%2525E2%252580%252593associated%252Bkeratin%252B8%252Band%252B18%252Bmutations%252Bmodulate%252Bkeratin%252Bacetylation%252Band%252Bmethylation%26aulast%3DJang%26aufirst%3DKwi%25E2%2580%2590Hoon%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D8%26spage%3D9030%26epage%3D9043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shu-Ching M  Wang</span>, <span class="hlFld-ContribAuthor ">Dennis H  Dowhan</span>, <span class="hlFld-ContribAuthor ">George E O  Muscat</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic arginine methylation in breast cancer: emerging therapeutic strategies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Endocrinology</span><span> <strong>2019,</strong> <em>62 </em>
                                    (3)
                                     , R223-R237. <a href="https://doi.org/10.1530/JME-18-0224" title="DOI URL">https://doi.org/10.1530/JME-18-0224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1530/JME-18-0224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1530%2FJME-18-0224%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Endocrinology%26atitle%3DEpigenetic%252Barginine%252Bmethylation%252Bin%252Bbreast%252Bcancer%25253A%252Bemerging%252Btherapeutic%252Bstrategies%26aulast%3DWang%26aufirst%3DShu-Ching%2BM%26date%3D2019%26volume%3D62%26issue%3D3%26spage%3DR223%26epage%3DR237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Small‐Molecule Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 201-220. <a href="https://doi.org/10.1002/9783527809257.ch9" title="DOI URL">https://doi.org/10.1002/9783527809257.ch9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch9%26sid%3Dliteratum%253Aachs%26atitle%3DSelective%252BSmall%2525E2%252580%252590Molecule%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2019%26date%3D2019%26spage%3D201%26epage%3D220%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthijs J.  van Haren</span>, <span class="hlFld-ContribAuthor ">Nathaniel I.  Martin</span>. </span><span class="cited-content_cbyCitation_article-title">PRMT Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 159-196. <a href="https://doi.org/10.1007/7355_2019_73" title="DOI URL">https://doi.org/10.1007/7355_2019_73</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_73&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_73%26sid%3Dliteratum%253Aachs%26atitle%3DPRMT%252BInhibitors%26aulast%3Dvan%2BHaren%26aufirst%3DMatthijs%2BJ.%26date%3D2019%26date%3D2019%26spage%3D159%26epage%3D196%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sabrina  Samuel</span>, <span class="hlFld-ContribAuthor ">Alistair  Marsden</span>, <span class="hlFld-ContribAuthor ">Srihari  Deepak</span>, <span class="hlFld-ContribAuthor ">Francisco  Rivero</span>, <span class="hlFld-ContribAuthor ">John  Greenman</span>, <span class="hlFld-ContribAuthor ">Pedro  Beltran-Alvarez</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibiting Arginine Methylation as a Tool to Investigate Cross-Talk with Methylation and Acetylation Post-Translational Modifications in a Glioblastoma Cell Line. </span><span class="cited-content_cbyCitation_journal-name">Proteomes</span><span> <strong>2018,</strong> <em>6 </em>
                                    (4)
                                     , 44. <a href="https://doi.org/10.3390/proteomes6040044" title="DOI URL">https://doi.org/10.3390/proteomes6040044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/proteomes6040044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fproteomes6040044%26sid%3Dliteratum%253Aachs%26jtitle%3DProteomes%26atitle%3DInhibiting%252BArginine%252BMethylation%252Bas%252Ba%252BTool%252Bto%252BInvestigate%252BCross-Talk%252Bwith%252BMethylation%252Band%252BAcetylation%252BPost-Translational%252BModifications%252Bin%252Ba%252BGlioblastoma%252BCell%252BLine%26aulast%3DSamuel%26aufirst%3DSabrina%26date%3D2018%26date%3D2018%26volume%3D6%26issue%3D4%26spage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Subhankar  Biswas</span>, <span class="hlFld-ContribAuthor ">C. Mallikarjuna  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic tools (The Writers, The Readers and The Erasers) and their implications in cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2018,</strong> <em>837 </em>, 8-24. <a href="https://doi.org/10.1016/j.ejphar.2018.08.021" title="DOI URL">https://doi.org/10.1016/j.ejphar.2018.08.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2018.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2018.08.021%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DEpigenetic%252Btools%252B%252528The%252BWriters%25252C%252BThe%252BReaders%252Band%252BThe%252BErasers%252529%252Band%252Btheir%252Bimplications%252Bin%252Bcancer%252Btherapy%26aulast%3DBiswas%26aufirst%3DSubhankar%26date%3D2018%26volume%3D837%26spage%3D8%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caroline  Bouchard</span>, <span class="hlFld-ContribAuthor ">Peeyush  Sahu</span>, <span class="hlFld-ContribAuthor ">Marion  Meixner</span>, <span class="hlFld-ContribAuthor ">René Reiner  Nötzold</span>, <span class="hlFld-ContribAuthor ">Marco B.  Rust</span>, <span class="hlFld-ContribAuthor ">Elisabeth  Kremmer</span>, <span class="hlFld-ContribAuthor ">Regina  Feederle</span>, <span class="hlFld-ContribAuthor ">Gene  Hart-Smith</span>, <span class="hlFld-ContribAuthor ">Florian  Finkernagel</span>, <span class="hlFld-ContribAuthor ">Marek  Bartkuhn</span>, <span class="hlFld-ContribAuthor ">Soni  Savai Pullamsetti</span>, <span class="hlFld-ContribAuthor ">Andrea  Nist</span>, <span class="hlFld-ContribAuthor ">Thorsten  Stiewe</span>, <span class="hlFld-ContribAuthor ">Sjaak  Philipsen</span>, <span class="hlFld-ContribAuthor ">Uta-Maria  Bauer</span>. </span><span class="cited-content_cbyCitation_article-title">Genomic Location of PRMT6-Dependent H3R2 Methylation Is Linked to the Transcriptional Outcome of Associated Genes. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2018,</strong> <em>24 </em>
                                    (12)
                                     , 3339-3352. <a href="https://doi.org/10.1016/j.celrep.2018.08.052" title="DOI URL">https://doi.org/10.1016/j.celrep.2018.08.052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2018.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2018.08.052%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DGenomic%252BLocation%252Bof%252BPRMT6-Dependent%252BH3R2%252BMethylation%252BIs%252BLinked%252Bto%252Bthe%252BTranscriptional%252BOutcome%252Bof%252BAssociated%252BGenes%26aulast%3DBouchard%26aufirst%3DCaroline%26date%3D2018%26volume%3D24%26issue%3D12%26spage%3D3339%26epage%3D3352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tharindumala  Abeywardana</span>, <span class="hlFld-ContribAuthor ">Myungeun  Oh</span>, <span class="hlFld-ContribAuthor ">Lei  Jiang</span>, <span class="hlFld-ContribAuthor ">Ying  Yang</span>, <span class="hlFld-ContribAuthor ">Mei  Kong</span>, <span class="hlFld-ContribAuthor ">Jikui  Song</span>, <span class="hlFld-ContribAuthor ">Yanzhong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">CARM1 suppresses de novo serine synthesis by promoting PKM2 activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2018,</strong> <em>293 </em>
                                    (39)
                                     , 15290-15303. <a href="https://doi.org/10.1074/jbc.RA118.004512" title="DOI URL">https://doi.org/10.1074/jbc.RA118.004512</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA118.004512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA118.004512%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DCARM1%252Bsuppresses%252Bde%252Bnovo%252Bserine%252Bsynthesis%252Bby%252Bpromoting%252BPKM2%252Bactivity%26aulast%3DAbeywardana%26aufirst%3DTharindumala%26date%3D2018%26volume%3D293%26issue%3D39%26spage%3D15290%26epage%3D15303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea  Di Pietro</span>, <span class="hlFld-ContribAuthor ">Kim L.  Good-Jacobson</span>. </span><span class="cited-content_cbyCitation_article-title">Disrupting the Code: Epigenetic Dysregulation of Lymphocyte Function during Infectious Disease and Lymphoma Development. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Immunology</span><span> <strong>2018,</strong> <em>201 </em>
                                    (4)
                                     , 1109-1118. <a href="https://doi.org/10.4049/jimmunol.1800137" title="DOI URL">https://doi.org/10.4049/jimmunol.1800137</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4049/jimmunol.1800137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4049%2Fjimmunol.1800137%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Immunology%26atitle%3DDisrupting%252Bthe%252BCode%25253A%252BEpigenetic%252BDysregulation%252Bof%252BLymphocyte%252BFunction%252Bduring%252BInfectious%252BDisease%252Band%252BLymphoma%252BDevelopment%26aulast%3DDi%2BPietro%26aufirst%3DAndrea%26date%3D2018%26date%3D2018%26volume%3D201%26issue%3D4%26spage%3D1109%26epage%3D1118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander-Thomas  Hauser</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic small molecule modulators of histone and DNA methylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>45 </em>, 73-85. <a href="https://doi.org/10.1016/j.cbpa.2018.03.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.03.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.03.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DEpigenetic%252Bsmall%252Bmolecule%252Bmodulators%252Bof%252Bhistone%252Band%252BDNA%252Bmethylation%26aulast%3DHauser%26aufirst%3DAlexander-Thomas%26date%3D2018%26volume%3D45%26spage%3D73%26epage%3D85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katerina  Cermakova</span>, <span class="hlFld-ContribAuthor ">H.  Hodges</span>. </span><span class="cited-content_cbyCitation_article-title">Next-Generation Drugs and Probes for Chromatin Biology: From Targeted Protein Degradation to Phase Separation. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (8)
                                     , 1958. <a href="https://doi.org/10.3390/molecules23081958" title="DOI URL">https://doi.org/10.3390/molecules23081958</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23081958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23081958%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNext-Generation%252BDrugs%252Band%252BProbes%252Bfor%252BChromatin%252BBiology%25253A%252BFrom%252BTargeted%252BProtein%252BDegradation%252Bto%252BPhase%252BSeparation%26aulast%3DCermakova%26aufirst%3DKaterina%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D8%26spage%3D1958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emily  Smith</span>, <span class="hlFld-ContribAuthor ">Wei  Zhou</span>, <span class="hlFld-ContribAuthor ">Polina  Shindiapina</span>, <span class="hlFld-ContribAuthor ">Said  Sif</span>, <span class="hlFld-ContribAuthor ">Chenglong  Li</span>, <span class="hlFld-ContribAuthor ">Robert A.  Baiocchi</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in targeting protein arginine methyltransferase enzymes in cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2018,</strong> <em>22 </em>
                                    (6)
                                     , 527-545. <a href="https://doi.org/10.1080/14728222.2018.1474203" title="DOI URL">https://doi.org/10.1080/14728222.2018.1474203</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14728222.2018.1474203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14728222.2018.1474203%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DRecent%252Badvances%252Bin%252Btargeting%252Bprotein%252Barginine%252Bmethyltransferase%252Benzymes%252Bin%252Bcancer%252Btherapy%26aulast%3DSmith%26aufirst%3DEmily%26date%3D2018%26date%3D2018%26volume%3D22%26issue%3D6%26spage%3D527%26epage%3D545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kazuhide  Nakayama</span>, <span class="hlFld-ContribAuthor ">Magdalena M.  Szewczyk</span>, <span class="hlFld-ContribAuthor ">Carlo  dela Sena</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Hong  Zeng</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Renato Ferreira  de Freitas</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Yuji  Baba</span>, <span class="hlFld-ContribAuthor ">Mihoko  Kunitomo</span>, <span class="hlFld-ContribAuthor ">Douglas R.  Cary</span>, <span class="hlFld-ContribAuthor ">Michiko  Tawada</span>, <span class="hlFld-ContribAuthor ">Akihiro  Ohashi</span>, <span class="hlFld-ContribAuthor ">Yasuhiro  Imaeda</span>, <span class="hlFld-ContribAuthor ">Kumar Singh  Saikatendu</span>, <span class="hlFld-ContribAuthor ">Charles E.  Grimshaw</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Atsushi  Kiba</span>, <span class="hlFld-ContribAuthor ">Daisuke  Tomita</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2018,</strong> <em>9 </em>
                                    (26)
                                     , 18480-18493. <a href="https://doi.org/10.18632/oncotarget.24883" title="DOI URL">https://doi.org/10.18632/oncotarget.24883</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.24883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.24883%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DTP-064%25252C%252Ba%252Bpotent%252Band%252Bselective%252Bsmall%252Bmolecule%252Binhibitor%252Bof%252BPRMT4%252Bfor%252Bmultiple%252Bmyeloma%26aulast%3DNakayama%26aufirst%3DKazuhide%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D26%26spage%3D18480%26epage%3D18493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jan  Kramer</span>, <span class="hlFld-ContribAuthor ">Veronika  Désor</span>, <span class="hlFld-ContribAuthor ">Steffen  Brunst</span>, <span class="hlFld-ContribAuthor ">Sandra K.  Wittmann</span>, <span class="hlFld-ContribAuthor ">Jörn  Lausen</span>, <span class="hlFld-ContribAuthor ">Jan  Heering</span>, <span class="hlFld-ContribAuthor ">Anna  Proschak</span>, <span class="hlFld-ContribAuthor ">Ewgenij  Proschak</span>. </span><span class="cited-content_cbyCitation_article-title">A coupled fluorescence-based assay for the detection of protein arginine N-methyltransferase 6 (PRMT6) enzymatic activity. </span><span class="cited-content_cbyCitation_journal-name">Analytical Biochemistry</span><span> <strong>2018,</strong> <em>547 </em>, 7-13. <a href="https://doi.org/10.1016/j.ab.2018.01.023" title="DOI URL">https://doi.org/10.1016/j.ab.2018.01.023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ab.2018.01.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ab.2018.01.023%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Biochemistry%26atitle%3DA%252Bcoupled%252Bfluorescence-based%252Bassay%252Bfor%252Bthe%252Bdetection%252Bof%252Bprotein%252Barginine%252BN-methyltransferase%252B6%252B%252528PRMT6%252529%252Benzymatic%252Bactivity%26aulast%3DKramer%26aufirst%3DJan%26date%3D2018%26volume%3D547%26spage%3D7%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennifer I.  Brown</span>, <span class="hlFld-ContribAuthor ">Timo  Koopmans</span>, <span class="hlFld-ContribAuthor ">Jolinde  van Strien</span>, <span class="hlFld-ContribAuthor ">Nathaniel I.  Martin</span>, <span class="hlFld-ContribAuthor ">Adam  Frankel</span>. </span><span class="cited-content_cbyCitation_article-title">Kinetic Analysis of PRMT1 Reveals Multifactorial Processivity and a Sequential Ordered Mechanism. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2018,</strong> <em>19 </em>
                                    (1)
                                     , 85-99. <a href="https://doi.org/10.1002/cbic.201700521" title="DOI URL">https://doi.org/10.1002/cbic.201700521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201700521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201700521%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DKinetic%252BAnalysis%252Bof%252BPRMT1%252BReveals%252BMultifactorial%252BProcessivity%252Band%252Ba%252BSequential%252BOrdered%252BMechanism%26aulast%3DBrown%26aufirst%3DJennifer%2BI.%26date%3D2018%26date%3D2017%26volume%3D19%26issue%3D1%26spage%3D85%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Allison E.  Drew</span>, <span class="hlFld-ContribAuthor ">Oscar  Moradei</span>, <span class="hlFld-ContribAuthor ">Suzanne L.  Jacques</span>, <span class="hlFld-ContribAuthor ">Nathalie  Rioux</span>, <span class="hlFld-ContribAuthor ">Ann P.  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Christina  Allain</span>, <span class="hlFld-ContribAuthor ">Margaret Porter  Scott</span>, <span class="hlFld-ContribAuthor ">Lei  Jin</span>, <span class="hlFld-ContribAuthor ">Alejandra  Raimondi</span>, <span class="hlFld-ContribAuthor ">Jessica L.  Handler</span>, <span class="hlFld-ContribAuthor ">Heidi M.  Ott</span>, <span class="hlFld-ContribAuthor ">Ryan G.  Kruger</span>, <span class="hlFld-ContribAuthor ">Michael T.  McCabe</span>, <span class="hlFld-ContribAuthor ">Christopher  Sneeringer</span>, <span class="hlFld-ContribAuthor ">Thomas  Riera</span>, <span class="hlFld-ContribAuthor ">Gideon  Shapiro</span>, <span class="hlFld-ContribAuthor ">Nigel J.  Waters</span>, <span class="hlFld-ContribAuthor ">Lorna H.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">Mikel P.  Moyer</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>, <span class="hlFld-ContribAuthor ">Jesse  Smith</span>, <span class="hlFld-ContribAuthor ">Richard  Chesworth</span>, <span class="hlFld-ContribAuthor ">Scott A.  Ribich</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a CARM1 Inhibitor with Potent In Vitro and In Vivo Activity in Preclinical Models of Multiple Myeloma. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-18446-z" title="DOI URL">https://doi.org/10.1038/s41598-017-18446-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-18446-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-18446-z%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DIdentification%252Bof%252Ba%252BCARM1%252BInhibitor%252Bwith%252BPotent%252BIn%252BVitro%252Band%252BIn%252BVivo%252BActivity%252Bin%252BPreclinical%252BModels%252Bof%252BMultiple%252BMyeloma%26aulast%3DDrew%26aufirst%3DAllison%2BE.%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Boehm</span>, <span class="hlFld-ContribAuthor ">Melanie  Ott</span>. </span><span class="cited-content_cbyCitation_article-title">Host Methyltransferases and Demethylases: Potential New Epigenetic Targets for HIV Cure Strategies and Beyond. </span><span class="cited-content_cbyCitation_journal-name">AIDS Research and Human Retroviruses</span><span> <strong>2017,</strong> <em>33 </em>
                                    (S1)
                                     , S-8-S-22. <a href="https://doi.org/10.1089/aid.2017.0180" title="DOI URL">https://doi.org/10.1089/aid.2017.0180</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/aid.2017.0180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Faid.2017.0180%26sid%3Dliteratum%253Aachs%26jtitle%3DAIDS%2520Research%2520and%2520Human%2520Retroviruses%26atitle%3DHost%252BMethyltransferases%252Band%252BDemethylases%25253A%252BPotential%252BNew%252BEpigenetic%252BTargets%252Bfor%252BHIV%252BCure%252BStrategies%252Band%252BBeyond%26aulast%3DBoehm%26aufirst%3DDaniela%26date%3D2017%26volume%3D33%26issue%3DS1%26spage%3DS-8%26epage%3DS-22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in developing selective inhibitors of protein methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 100-108. <a href="https://doi.org/10.1016/j.cbpa.2017.06.013" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.06.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.06.013%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Bprogress%252Bin%252Bdeveloping%252Bselective%252Binhibitors%252Bof%252Bprotein%252Bmethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2017%26volume%3D39%26spage%3D100%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Suzanne  Ackloo</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Susanne  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical probes targeting epigenetic proteins: Applications beyond oncology. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 378-400. <a href="https://doi.org/10.1080/15592294.2017.1279371" title="DOI URL">https://doi.org/10.1080/15592294.2017.1279371</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2017.1279371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2017.1279371%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DChemical%252Bprobes%252Btargeting%252Bepigenetic%252Bproteins%25253A%252BApplications%252Bbeyond%252Boncology%26aulast%3DAckloo%26aufirst%3DSuzanne%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D378%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Structure and potency of compounds <b>1</b> (PRMT4 inhibitor), <b>2</b> (PRMT6 selective inhibitor), and <b>3</b> (selective type I PRMT inhibitor). (B) Structure and potency of compound <b>4</b>, a fragment-like inhibitor of type I PRMTs. (C) X-ray cocrystal structure of compound <b>4</b> (cyan) in the complex with PRMT6 (gray) and SAH (green) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS">5EGS</a>). The key H-bond interactions are shown in yellow dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes for Preparing Designed Compounds: Synthesis of Compounds in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> (A), Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> (B), and Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> (C)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaBH(OAc)<sub>3</sub>, DCM; (b) HCl in MeOH or dioxane, rt, 10–85% over two to four steps; (c) <i>i</i>-PrOH, 50 °C (d) TFAA, DCM, 0 °C to rt, 85% over two steps; (e) TFA, DCM, rt; (f) NaOH, MeOH, H<sub>2</sub>O, 50 °C; (g) NaH, DMF, 0 °C to rt.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selectivity of <b>17</b> against methyltransferases. Selectivity of <b>17</b> against a panel of 25 PKMTs and DNMTs was determined at two compound concentrations of 10 μM (black) and 50 μM (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding confirmation and MOA of <b>17</b>. (A) Isothermal titration calorimetry (ITC) was used to confirm binding of <b>17</b> to PRMT4 with a <i>K</i><sub>d</sub> of 100 nM (B) and to PRMT6 with a <i>K</i><sub>d</sub> of 87 ± 35 nM. ITC experiments for PRMT4 and PRMT6 were performed in duplicate and triplicate, respectively. Differential scanning fluorimetry (DSF) was also used to confirm the binding of <b>17</b> to PRMT4 (C) and PRMT6 (D). Experiments were performed (●) in the absence of any compound as a negative control and in the presence of (○) 100 μM SAH as a positive control with PRMT4 (Δ<i>T</i><sub>m</sub> of 4 °C) and PRMT6 (Δ<i>T</i><sub>m</sub> of 8.4 °C). Presence of (▲) 200 μM <b>17</b> resulted in stabilization of both proteins with Δ<i>T</i><sub>m</sub> of 3.3 and 11 °C for PRMT4 and PRMT6, respectively. Presence of (□) 100 μM SAM plus 200 μM <b>17</b> had the highest effect with Δ<i>T</i><sub>m</sub> of 13 and 21 °C for PRMT4 and PRMT6, respectively. The inflection point of each transition curve is considered melting temperature (<i>T</i><sub>m</sub>) and the increase in <i>T</i><sub>m</sub> is an indication of binding. The IC<sub>50</sub> values for <b>17</b> with PRMT4 and PRMT6 were unchanged when determined under various peptide (E) and SAM (F) concentrations, suggesting an apparent noncompetitive pattern of inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effect of <b>17</b> on inhibiting PRMT6 and PRMT4 in cells. (A) Compound <b>17</b> inhibits PRMT6 methyltransferase activity in HEK293 cells. HEK293 cells were transfected with FLAG-tagged PRMT6 (wt) or its catalytically inactive mutant V86K/D88A (mut) and treated with <b>17</b> at indicated concentrations for 20 h. H3R2me2a levels were determined by Western blot. The graphs represent nonlinear fits of H3R2me2a signal intensities normalized to total histone H3. The results are the mean ± SEM of three replicates. (B) Compound <b>17</b> inhibits endogenous PRMT4 methyltransferase activity in HEK293 cells. HEK293 cells were treated with <b>17</b> at indicated concentrations for 72 h, and Med12-Rme2a levels were determined by Western blot. The graphs represent nonlinear fits of Med12-Rme2a signal intensities normalized to total Med12. The results are the mean ± SEM of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/medium/jm-2016-01033v_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>46</b>, a close analog of compound <b>17</b>, has no inhibitory activity against PRMT4 and PRMT6. NI: no inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-19/acs.jmedchem.6b01033/20161007/images/large/jm-2016-01033v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01033&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52022" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52022" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 73 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Fuhrmann, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clancy, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, P. R.</span><span> </span><span class="NLM_article-title">Chemical biology of protein arginine modifications in epigenetic regulation</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">5413</span><span class="NLM_x">–</span> <span class="NLM_lpage">5461</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=5413-5461&author=J.+Fuhrmannauthor=K.+W.+Clancyauthor=P.+R.+Thompson&title=Chemical+biology+of+protein+arginine+modifications+in+epigenetic+regulation&doi=10.1021%2Facs.chemrev.5b00003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology of Protein Arginine Modifications in Epigenetic Regulation</span></div><div class="casAuthors">Fuhrmann, Jakob; Clancy, Kathleen W.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5413-5461</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Here we aim to summarize the current knowledge surrounding the posttranslational modification of histone arginine residues, focusing on enzyme classes that catalyze the citrullination and methylation of arginine residues as well as noncanonical arginine modifications such as phosphorylation, ADP-ribosylation, and arginylation.  The major focus will be given to the protein arginine deiminases and protein arginine methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdzgTmC4C4fbVg90H21EOLACvtfcHk0lg2KHsKe5tlcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlCisLc%253D&md5=a6df90a99a606e1d0a7296fa95625202</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00003%26sid%3Dliteratum%253Aachs%26aulast%3DFuhrmann%26aufirst%3DJ.%26aulast%3DClancy%26aufirst%3DK.%2BW.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DChemical%2520biology%2520of%2520protein%2520arginine%2520modifications%2520in%2520epigenetic%2520regulation%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D5413%26epage%3D5461%26doi%3D10.1021%2Facs.chemrev.5b00003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Morales, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caceres, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevel, J. M.</span><span> </span><span class="NLM_article-title">Biochemistry and regulation of the protein arginine methyltransferases (PRMTs)</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">590</span><span class="NLM_x">, </span> <span class="NLM_fpage">138</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span><span class="refDoi"> DOI: 10.1016/j.abb.2015.11.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.abb.2015.11.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26612103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVemtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=590&publication_year=2016&pages=138-152&author=Y.+Moralesauthor=T.+Caceresauthor=K.+Mayauthor=J.+M.+Hevel&title=Biochemistry+and+regulation+of+the+protein+arginine+methyltransferases+%28PRMTs%29&doi=10.1016%2Fj.abb.2015.11.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemistry and regulation of the protein arginine methyltransferases (PRMTs)</span></div><div class="casAuthors">Morales, Yalemi; Caceres, Tamar; May, Kyle; Hevel, Joan M.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">590</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">138-152</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many key cellular processes can be regulated by the seemingly simple addn. of one, or two, Me groups to arginine residues by the nine known mammalian protein arginine methyltransferases (PRMTs).  The impact that arginine methylation has on cellular well-being is highlighted by the ever growing evidence linking PRMT dysregulation to disease states, which has marked the PRMTs as prominent pharmacol. targets.  This review is meant to orient the reader with respect to the structural features of the PRMTs that account for catalytic activity, as well as provide a framework for understanding how these enzymes are regulated.  An overview of what we understand about substrate recognition and binding is provided.  Control of product specificity and enzyme processivity are introduced as necessary but flexible features of the PRMTs.  Precise control of PRMT activity is a crit. component to eukaryotic cell health, esp. given that an arginine demethylase has not been identified.  We therefore conclude the review with a comprehensive discussion of how protein arginine methylation is regulated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpskzNjOitlXrVg90H21EOLACvtfcHk0lj0Vq-3PYm4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVemtb7L&md5=009f90f8b06a0e121a5e75ba11185338</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2015.11.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2015.11.030%26sid%3Dliteratum%253Aachs%26aulast%3DMorales%26aufirst%3DY.%26aulast%3DCaceres%26aufirst%3DT.%26aulast%3DMay%26aufirst%3DK.%26aulast%3DHevel%26aufirst%3DJ.%2BM.%26atitle%3DBiochemistry%2520and%2520regulation%2520of%2520the%2520protein%2520arginine%2520methyltransferases%2520%2528PRMTs%2529%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2016%26volume%3D590%26spage%3D138%26epage%3D152%26doi%3D10.1016%2Fj.abb.2015.11.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Lee, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span> </span><span class="NLM_article-title">Minireview: protein arginine methylation of nonhistone proteins in transcriptional regulation</span> <span class="citation_source-journal">Mol. Endocrinol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">433</span><span class="refDoi"> DOI: 10.1210/me.2008-0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1210%2Fme.2008-0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19164444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktlSiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=425-433&author=Y.+H.+Leeauthor=M.+R.+Stallcup&title=Minireview%3A+protein+arginine+methylation+of+nonhistone+proteins+in+transcriptional+regulation&doi=10.1210%2Fme.2008-0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Minireview: protein arginine methylation of nonhistone proteins in transcriptional regulation</span></div><div class="casAuthors">Lee, Young-Ho; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">425-433</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Endocrine regulation frequently culminates in altered transcription of specific genes.  The signal transduction pathways, which transmit the endocrine signal from cell surface to the transcription machinery, often involve posttranslational modifications of proteins.  Although phosphorylation has been by far the most widely studied protein modification, recent studies have indicated important roles for other types of modification, including protein arginine methylation.  Ten different protein arginine methyltransferase (PRMT) family members have been identified in mammalian cells, and numerous substrates are being identified for these PRMTs.  Whereas major attention has been focused on the methylation of histones and its role in chromatin remodeling and transcriptional regulation, there are many nonhistone substrates methylated by PRMTs.  This review primarily focuses on recent progress on the roles of the nonhistone protein methylation in transcription.  Protein methylation of coactivators, transcription factors, and signal transducers, among other proteins, plays important roles in transcriptional regulation.  Protein methylation may affect protein-protein interaction, protein-DNA or protein-RNA interaction, protein stability, subcellular localization, or enzymic activity.  Thus, protein arginine methylation is crit. for regulation of transcription and potentially for various physiol./pathol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWEPQ4itT-8LVg90H21EOLACvtfcHk0lj0Vq-3PYm4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktlSiu7g%253D&md5=9bfeb72a31dd3115130ce7ccde3bd4c3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1210%2Fme.2008-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2008-0380%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%2BH.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DMinireview%253A%2520protein%2520arginine%2520methylation%2520of%2520nonhistone%2520proteins%2520in%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Endocrinol.%26date%3D2009%26volume%3D23%26spage%3D425%26epage%3D433%26doi%3D10.1210%2Fme.2008-0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Tripsianes, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madl, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machyna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fessas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Englbrecht, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neugebauer, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Structural basis for dimethylarginine recognition by the Tudor domains of human SMN and SPF30 proteins</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1414</span><span class="NLM_x">–</span> <span class="NLM_lpage">1420</span><span class="refDoi"> DOI: 10.1038/nsmb.2185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnsmb.2185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=22101937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2gt7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1414-1420&author=K.+Tripsianesauthor=T.+Madlauthor=M.+Machynaauthor=D.+Fessasauthor=C.+Englbrechtauthor=U.+Fischerauthor=K.+M.+Neugebauerauthor=M.+Sattler&title=Structural+basis+for+dimethylarginine+recognition+by+the+Tudor+domains+of+human+SMN+and+SPF30+proteins&doi=10.1038%2Fnsmb.2185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for dimethylarginine recognition by the Tudor domains of human SMN and SPF30 proteins</span></div><div class="casAuthors">Tripsianes, Konstantinos; Madl, Tobias; Machyna, Martin; Fessas, Dimitrios; Englbrecht, Clemens; Fischer, Utz; Neugebauer, Karla M.; Sattler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1414-1420</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Arginine dimethylation plays crit. roles in the assembly of ribonucleoprotein complexes in pre-mRNA splicing and piRNA pathways.  We report soln. structures of SMN and SPF30 Tudor domains bound to sym. and asym. dimethylated arginines (sDMA and aDMA, resp.) that are inherent in the RNP complexes.  An arom. cage in the Tudor domain mediates dimethylarginine recognition by electrostatic stabilization through cation-π interactions.  Distinct from extended Tudor domains, dimethylarginine binding by the SMN and SPF30 Tudor domains is independent of proximal residues in the ligand.  Yet, enhanced micromolar affinities are obtained by external cooperativity when multiple methylation marks are presented in arginine- and glycine-rich peptide ligands.  A hydrogen bond network in the SMN Tudor domain, including Glu134 and a tyrosine hydroxyl of the arom. cage, enhances cation-π interactions and is impaired by a mutation causing an E134K substitution assocd. with spinal muscular atrophy.  Our structural anal. enables the design of an optimized binding pocket and the prediction of DMA binding properties of Tudor domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMSpLGZyOX5bVg90H21EOLACvtfcHk0lj0Vq-3PYm4HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2gt7rK&md5=7d8f06a6d5a907c233fa4d16ae8f71d4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2185%26sid%3Dliteratum%253Aachs%26aulast%3DTripsianes%26aufirst%3DK.%26aulast%3DMadl%26aufirst%3DT.%26aulast%3DMachyna%26aufirst%3DM.%26aulast%3DFessas%26aufirst%3DD.%26aulast%3DEnglbrecht%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DU.%26aulast%3DNeugebauer%26aufirst%3DK.%2BM.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520dimethylarginine%2520recognition%2520by%2520the%2520Tudor%2520domains%2520of%2520human%2520SMN%2520and%2520SPF30%2520proteins%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1414%26epage%3D1420%26doi%3D10.1038%2Fnsmb.2185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Evich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroeva, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germann, M. W.</span><span> </span><span class="NLM_article-title">Effect of methylation on the side-chain pKa value of arginine</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">479</span><span class="NLM_x">–</span> <span class="NLM_lpage">486</span><span class="refDoi"> DOI: 10.1002/pro.2838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fpro.2838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26540340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvV2js7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=479-486&author=M.+Evichauthor=E.+Stroevaauthor=Y.+G.+Zhengauthor=M.+W.+Germann&title=Effect+of+methylation+on+the+side-chain+pKa+value+of+arginine&doi=10.1002%2Fpro.2838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of methylation on the side-chain pKa value of arginine</span></div><div class="casAuthors">Evich, Marina; Stroeva, Ekaterina; Zheng, Yujun George; Germann, Markus W.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">479-486</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Arginine methylation is important in biol. systems.  Recent studies link the deregulation of protein arginine methyltransferases with certain cancers.  To assess the impact of methylation on interaction with other biomols., the pKa values of methylated arginine variants were detd. using NMR data.  The pKa values of monomethylated, sym. dimethylated, and asym. dimethylated arginine are similar to the unmodified arginine (14.2 ± 0.4).  Although the pKa value has not been significantly affected by methylation, consequences of methylation include changes in charge distribution and steric effects, suggesting alternative mechanisms for recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzjhmzL8VEpLVg90H21EOLACvtfcHk0ljoHtNlCF1zew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvV2js7%252FO&md5=0555aa47417940f135b05759e8cd81e5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fpro.2838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2838%26sid%3Dliteratum%253Aachs%26aulast%3DEvich%26aufirst%3DM.%26aulast%3DStroeva%26aufirst%3DE.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DGermann%26aufirst%3DM.%2BW.%26atitle%3DEffect%2520of%2520methylation%2520on%2520the%2520side-chain%2520pKa%2520value%2520of%2520arginine%26jtitle%3DProtein%2520Sci.%26date%3D2016%26volume%3D25%26spage%3D479%26epage%3D486%26doi%3D10.1002%2Fpro.2838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">Arginine methylation: An emerging regulator of protein function</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">272</span><span class="refDoi"> DOI: 10.1016/j.molcel.2005.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2005.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=15866169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=263-272&author=M.+T.+Bedfordauthor=S.+Richard&title=Arginine+methylation%3A+An+emerging+regulator+of+protein+function&doi=10.1016%2Fj.molcel.2005.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation: An emerging regulator of protein function</span></div><div class="casAuthors">Bedford, Mark T.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Arginine methylation is now coming out of the shadows of protein phosphorylation and entering the mainstream, largely due to the identification of the family of enzymes that lay down this modification.  In addn., modification-specific antibodies and proteomic approaches have facilitated the identification of an array of substrates for protein arginine methyltransferases.  This review describes recent insights into the mol. processes regulated by arginine methylation in normal and diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDYRpc_643mbVg90H21EOLACvtfcHk0ljoHtNlCF1zew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Cgtb4%253D&md5=63525cb1e96138be5563fbd897c2bde1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DArginine%2520methylation%253A%2520An%2520emerging%2520regulator%2520of%2520protein%2520function%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D263%26epage%3D272%26doi%3D10.1016%2Fj.molcel.2005.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Wei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mundade, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0ljoHtNlCF1zew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nott, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span> </span><span class="NLM_article-title">Deciphering arginine methylation: Tudor tells the tale</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">629</span><span class="NLM_x">–</span> <span class="NLM_lpage">642</span><span class="refDoi"> DOI: 10.1038/nrm3185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnrm3185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21915143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKqsb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=629-642&author=C.+Chenauthor=T.+J.+Nottauthor=J.+Jinauthor=T.+Pawson&title=Deciphering+arginine+methylation%3A+Tudor+tells+the+tale&doi=10.1038%2Fnrm3185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Deciphering arginine methylation: Tudor tells the tale</span></div><div class="casAuthors">Chen, Chen; Nott, Timothy J.; Jin, Jing; Pawson, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">629-642</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Proteins can be modified by post-translational modifications such as phosphorylation, methylation, acetylation and ubiquitylation, creating binding sites for specific protein domains.  Methylation has pivotal roles in the formation of complexes that are involved in cellular regulation, including in the generation of small RNAs.  Arginine methylation was discovered half a century ago, but the ability of methylarginine sites to serve as binding motifs for members of the Tudor protein family, and the functional significance of the protein-protein interactions that are mediated by Tudor domains, has only recently been appreciated.  Tudor proteins are now known to be present in PIWI complexes, where they are thought to interact with methylated PIWI proteins and regulate the PIWI-interacting RNA (piRNA) pathway in the germ line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFKEDr8FDL-bVg90H21EOLACvtfcHk0ljoHtNlCF1zew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKqsb7E&md5=9c5e8c1f0466b7772a2b8d7fc8e7b26a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrm3185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3185%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%26aulast%3DNott%26aufirst%3DT.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DPawson%26aufirst%3DT.%26atitle%3DDeciphering%2520arginine%2520methylation%253A%2520Tudor%2520tells%2520the%2520tale%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2011%26volume%3D12%26spage%3D629%26epage%3D642%26doi%3D10.1038%2Fnrm3185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0lj1cdQxzC-lhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Boffa, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vidali, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allfrey, V. G.</span><span> </span><span class="NLM_article-title">Distribution of Ng,Ng-dimethylarginine in nuclear protein-fractions</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">969</span><span class="NLM_x">–</span> <span class="NLM_lpage">976</span><span class="refDoi"> DOI: 10.1016/0006-291X(77)91613-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2F0006-291X%2877%2991613-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=1977&pages=969-976&author=L.+C.+Boffaauthor=J.+Karnauthor=G.+Vidaliauthor=V.+G.+Allfrey&title=Distribution+of+Ng%2CNg-dimethylarginine+in+nuclear+protein-fractions&doi=10.1016%2F0006-291X%2877%2991613-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2F0006-291X%2877%2991613-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-291X%252877%252991613-8%26sid%3Dliteratum%253Aachs%26aulast%3DBoffa%26aufirst%3DL.%2BC.%26aulast%3DKarn%26aufirst%3DJ.%26aulast%3DVidali%26aufirst%3DG.%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26atitle%3DDistribution%2520of%2520Ng%252CNg-dimethylarginine%2520in%2520nuclear%2520protein-fractions%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1977%26volume%3D74%26spage%3D969%26epage%3D976%26doi%3D10.1016%2F0006-291X%2877%2991613-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of protein methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1629</span><span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+U.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+inhibitors+of+protein+methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0lj1cdQxzC-lhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520inhibitors%2520of%2520protein%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Yang, Y. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Z.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0lj1cdQxzC-lhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%2BZ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Schurter, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunick, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harp, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henschen-Edman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aswad, D. W.</span><span> </span><span class="NLM_article-title">Methylation of histone H3 by coactivator-associated arginine methyltransferase 1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">5747</span><span class="NLM_x">–</span> <span class="NLM_lpage">5756</span><span class="refDoi"> DOI: 10.1021/bi002631b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi002631b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitlyntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=5747-5756&author=B.+T.+Schurterauthor=S.+S.+Kohauthor=D.+Chenauthor=G.+J.+Bunickauthor=J.+M.+Harpauthor=B.+L.+Hansonauthor=A.+Henschen-Edmanauthor=D.+R.+Mackayauthor=M.+R.+Stallcupauthor=D.+W.+Aswad&title=Methylation+of+histone+H3+by+coactivator-associated+arginine+methyltransferase+1&doi=10.1021%2Fbi002631b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Histone H3 by Coactivator-Associated Arginine Methyltransferase 1</span></div><div class="casAuthors">Schurter, Brandon T.; Koh, Stephen S.; Chen, Dagang; Bunick, Gerard J.; Harp, Joel M.; Hanson, B. Leif; Henschen-Edman, Agnes; Mackay, Douglas R.; Stallcup, Michael R.; Aswad, Dana W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5747-5756</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The preferential in vitro methylation of histone H3 by coactivator-assocd. arginine methyltransferase 1 (CARM1) has been proposed as a basis for its ability to enhance gene transcription [Chen, D., et al. (1999) Science 284, 2174-2177].  To further evaluate the significance of H3 methylation, we studied the kinetics and site specificity of its modification by CARM1.  Affinity-purified CARM1 methylated recombinant chick H3, which is free of posttranslational modifications, and calf thymus H3, which is heterogeneous with regard to preexisting modifications, equally well, exhibiting a Vmax of 4500 pmol min-1 (mg of enzyme)-1 and an apparent Km for H3 of ≤0.2 μM.  The catalytic efficiency (kcat/Km) of CARM1 toward H3 was at least 1000 times that toward R1 (GGFGGRGGFGG-amide), a highly effective substrate for protein arginine methyltransferase 1.  Peptide mapping of 3H-methyl-labeled H3 indicated methylation at Arg-2, Arg-17, and Arg-26 in the N-terminal region and at one or more of four arginines (128/129/131/134) at the C-terminus.  Two of the N-terminal sites, Arg-17 and Arg-26, occur in the sequence KAXRK and appear to be more efficiently methylated than Arg-2.  CARM1 catalyzed formation of NG,NG-dimethylarginine (asym.) but little or no NG,N'G-dimethylarginine (sym.) and no form of methyllysine.  Amino acid anal. of untreated calf thymus H3 revealed that 3.7% of the mols. naturally contain asym. dimethylarginine and/or monomethylarginine.  Our findings support the hypothesis that methylation of H3 may be involved in the mechanism of transcriptional coactivation by CARM1 of genes whose expression is under the control of nuclear receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyQC65J4FUyrVg90H21EOLACvtfcHk0ljd7sTUCjRzsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitlyntbc%253D&md5=8ba953dfa829e00a69343610ca6c460d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi002631b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi002631b%26sid%3Dliteratum%253Aachs%26aulast%3DSchurter%26aufirst%3DB.%2BT.%26aulast%3DKoh%26aufirst%3DS.%2BS.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DBunick%26aufirst%3DG.%2BJ.%26aulast%3DHarp%26aufirst%3DJ.%2BM.%26aulast%3DHanson%26aufirst%3DB.%2BL.%26aulast%3DHenschen-Edman%26aufirst%3DA.%26aulast%3DMackay%26aufirst%3DD.%2BR.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DAswad%26aufirst%3DD.%2BW.%26atitle%3DMethylation%2520of%2520histone%2520H3%2520by%2520coactivator-associated%2520arginine%2520methyltransferase%25201%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D5747%26epage%3D5756%26doi%3D10.1021%2Fbi002631b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Ito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furumatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taniguchi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchiya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lotz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asahara, H.</span><span> </span><span class="NLM_article-title">Arginine methyltransferase CARM1/PRMT4 regulates endochondral ossification</span> <span class="citation_source-journal">BMC Dev. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="refDoi"> DOI: 10.1186/1471-213X-9-47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1186%2F1471-213X-9-47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=47&author=T.+Itoauthor=N.+Yadavauthor=J.+Leeauthor=T.+Furumatsuauthor=S.+Yamashitaauthor=K.+Yoshidaauthor=N.+Taniguchiauthor=M.+Hashimotoauthor=M.+Tsuchiyaauthor=T.+Ozakiauthor=M.+Lotzauthor=M.+T.+Bedfordauthor=H.+Asahara&title=Arginine+methyltransferase+CARM1%2FPRMT4+regulates+endochondral+ossification&doi=10.1186%2F1471-213X-9-47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1186%2F1471-213X-9-47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-213X-9-47%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DT.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DFurumatsu%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DS.%26aulast%3DYoshida%26aufirst%3DK.%26aulast%3DTaniguchi%26aufirst%3DN.%26aulast%3DHashimoto%26aufirst%3DM.%26aulast%3DTsuchiya%26aufirst%3DM.%26aulast%3DOzaki%26aufirst%3DT.%26aulast%3DLotz%26aufirst%3DM.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DAsahara%26aufirst%3DH.%26atitle%3DArginine%2520methyltransferase%2520CARM1%252FPRMT4%2520regulates%2520endochondral%2520ossification%26jtitle%3DBMC%2520Dev.%2520Biol.%26date%3D2009%26volume%3D9%26spage%3D47%26doi%3D10.1186%2F1471-213X-9-47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Kawabe, Y. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinnell, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudnicki, M. A.</span><span> </span><span class="NLM_article-title">Carm1 regulates Pax7 transcriptional activity through MLL1/2 recruitment during asymmetric satellite stem cell divisions</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">333</span><span class="NLM_x">–</span> <span class="NLM_lpage">345</span><span class="refDoi"> DOI: 10.1016/j.stem.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.stem.2012.07.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=333-345&author=Y.+I.+Kawabeauthor=Y.+X.+Wangauthor=I.+W.+McKinnellauthor=M.+T.+Bedfordauthor=M.+A.+Rudnicki&title=Carm1+regulates+Pax7+transcriptional+activity+through+MLL1%2F2+recruitment+during+asymmetric+satellite+stem+cell+divisions&doi=10.1016%2Fj.stem.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DKawabe%26aufirst%3DY.%2BI.%26aulast%3DWang%26aufirst%3DY.%2BX.%26aulast%3DMcKinnell%26aufirst%3DI.%2BW.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DRudnicki%26aufirst%3DM.%2BA.%26atitle%3DCarm1%2520regulates%2520Pax7%2520transcriptional%2520activity%2520through%2520MLL1%252F2%2520recruitment%2520during%2520asymmetric%2520satellite%2520stem%2520cell%2520divisions%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2012%26volume%3D11%26spage%3D333%26epage%3D345%26doi%3D10.1016%2Fj.stem.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Vu, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perna, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voza, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueroa, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paietta, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deblasio, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimer, S. D.</span><span> </span><span class="NLM_article-title">PRMT4 blocks myeloid differentiation by assembling a methyl-RUNX1-dependent repressor complex</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1625</span><span class="NLM_x">–</span> <span class="NLM_lpage">1638</span><span class="refDoi"> DOI: 10.1016/j.celrep.2013.11.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.celrep.2013.11.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24332853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOgs7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=1625-1638&author=L.+P.+Vuauthor=F.+Pernaauthor=L.+Wangauthor=F.+Vozaauthor=M.+E.+Figueroaauthor=P.+Tempstauthor=H.+Erdjument-Bromageauthor=R.+Gaoauthor=S.+Chenauthor=E.+Paiettaauthor=T.+Deblasioauthor=A.+Melnickauthor=Y.+Liuauthor=X.+Zhaoauthor=S.+D.+Nimer&title=PRMT4+blocks+myeloid+differentiation+by+assembling+a+methyl-RUNX1-dependent+repressor+complex&doi=10.1016%2Fj.celrep.2013.11.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT4 Blocks Myeloid Differentiation by Assembling a Methyl-RUNX1-Dependent Repressor Complex</span></div><div class="casAuthors">Vu, Ly P.; Perna, Fabiana; Wang, Lan; Voza, Francesca; Figueroa, Maria E.; Tempst, Paul; Erdjument-Bromage, Hediye; Gao, Rui; Chen, Sisi; Paietta, Elisabeth; Deblasio, Tony; Melnick, Ari; Liu, Yan; Zhao, Xinyang; Nimer, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1625-1638</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Defining the role of epigenetic regulators in hematopoiesis has become critically important, because recurrent mutations or aberrant expression of these genes has been identified in both myeloid and lymphoid hematol. malignancies.  We found that PRMT4, a type I arginine methyltransferase whose function in normal and malignant hematopoiesis is unknown, is overexpressed in acute myelogenous leukemia patient samples.  Overexpression of PRMT4 blocks the myeloid differentiation of human stem/progenitor cells (HSPCs), whereas its knockdown is sufficient to induce myeloid differentiation of HSPCs.  We demonstrated that PRMT4 represses the expression of miR-223 in HSPCs via the methylation of RUNX1, which triggers the assembly of a multiprotein repressor complex that includes DPF2.  As part of the feedback loop, PRMT4 expression is repressed posttranscriptionally by miR-223.  Depletion of PRMT4 results in differentiation of myeloid leukemia cells in vitro and their decreased proliferation in vivo.  Thus, targeting PRMT4 holds potential as a novel therapy for acute myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8etHAEewC_7Vg90H21EOLACvtfcHk0ljd7sTUCjRzsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOgs7vK&md5=b0df503d2ce52a520f6ed3f1d7b8e362</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2013.11.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2013.11.025%26sid%3Dliteratum%253Aachs%26aulast%3DVu%26aufirst%3DL.%2BP.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DVoza%26aufirst%3DF.%26aulast%3DFigueroa%26aufirst%3DM.%2BE.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DPaietta%26aufirst%3DE.%26aulast%3DDeblasio%26aufirst%3DT.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DPRMT4%2520blocks%2520myeloid%2520differentiation%2520by%2520assembling%2520a%2520methyl-RUNX1-dependent%2520repressor%2520complex%26jtitle%3DCell%2520Rep.%26date%3D2013%26volume%3D5%26spage%3D1625%26epage%3D1638%26doi%3D10.1016%2Fj.celrep.2013.11.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Zhao, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y. F.</span><span> </span><span class="NLM_article-title">CARM1 mediates modulation of Sox2</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e27026</span><span class="refDoi"> DOI: 10.1371/journal.pone.0027026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1371%2Fjournal.pone.0027026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e27026&author=H.+Y.+Zhaoauthor=Y.+J.+Zhangauthor=H.+Daiauthor=Y.+Zhangauthor=Y.+F.+Shen&title=CARM1+mediates+modulation+of+Sox2&doi=10.1371%2Fjournal.pone.0027026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027026%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DH.%2BY.%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DY.%2BF.%26atitle%3DCARM1%2520mediates%2520modulation%2520of%2520Sox2%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De27026%26doi%3D10.1371%2Fjournal.pone.0027026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wisinski, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahlquist, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span> </span><span class="NLM_article-title">CARM1 methylates chromatin remodeling factor BAF155 to enhance tumor progression and metastasis</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">21</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.1016/j.ccr.2013.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.ccr.2013.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24434208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFSisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=21-36&author=L.+Wangauthor=Z.+B.+Zhaoauthor=M.+B.+Meyerauthor=S.+Sahaauthor=M.+G.+Yuauthor=A.+L.+Guoauthor=K.+B.+Wisinskiauthor=W.+Huangauthor=W.+B.+Caiauthor=J.+W.+Pikeauthor=M.+Yuanauthor=P.+Ahlquistauthor=W.+Xu&title=CARM1+methylates+chromatin+remodeling+factor+BAF155+to+enhance+tumor+progression+and+metastasis&doi=10.1016%2Fj.ccr.2013.12.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">CARM1 Methylates Chromatin Remodeling Factor BAF155 to Enhance Tumor Progression and Metastasis</span></div><div class="casAuthors">Wang, Lu; Zhao, Zibo; Meyer, Mark B.; Saha, Sandeep; Yu, Menggang; Guo, Ailan; Wisinski, Kari B.; Huang, Wei; Cai, Weibo; Pike, J. Wesley; Yuan, Ming; Ahlquist, Paul; Xu, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-36</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1), a coactivator for various cancer-relevant transcription factors, is overexpressed in breast cancer.  To elucidate the functions of CARM1 in tumorigenesis, we knocked out CARM1 from several breast cancer cell lines using Zinc-Finger Nuclease technol., which resulted in drastic phenotypic and biochem. changes.  The CARM1 KO cell lines enabled identification of CARM1 substrates, notably the SWI/SNF core subunit BAF155.  Methylation of BAF155 at R1064 was found to be an independent prognostic biomarker for cancer recurrence and to regulate breast cancer cell migration and metastasis.  Furthermore, CARM1-mediated BAF155 methylation affects gene expression by directing methylated BAF155 to unique chromatin regions (e.g., c-Myc pathway genes).  Collectively, our studies uncover a mechanism by which BAF155 acquires tumorigenic functions via arginine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Fx4Ny0ZnD7Vg90H21EOLACvtfcHk0lg0USqmVlyRIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFSisLs%253D&md5=4257dd8fe6bbc915fe1d32ba7af57afe</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DZ.%2BB.%26aulast%3DMeyer%26aufirst%3DM.%2BB.%26aulast%3DSaha%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DM.%2BG.%26aulast%3DGuo%26aufirst%3DA.%2BL.%26aulast%3DWisinski%26aufirst%3DK.%2BB.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DCai%26aufirst%3DW.%2BB.%26aulast%3DPike%26aufirst%3DJ.%2BW.%26aulast%3DYuan%26aufirst%3DM.%26aulast%3DAhlquist%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DCARM1%2520methylates%2520chromatin%2520remodeling%2520factor%2520BAF155%2520to%2520enhance%2520tumor%2520progression%2520and%2520metastasis%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D21%26epage%3D36%26doi%3D10.1016%2Fj.ccr.2013.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Xu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asahara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emerson, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montminy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, R. M.</span><span> </span><span class="NLM_article-title">A transcriptional switch mediated by cofactor methylation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">294</span><span class="NLM_x">, </span> <span class="NLM_fpage">2507</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span><span class="refDoi"> DOI: 10.1126/science.1065961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1126%2Fscience.1065961" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2001&pages=2507-2511&author=W.+Xuauthor=H.+Chenauthor=K.+Duauthor=H.+Asaharaauthor=M.+Tiniauthor=B.+M.+Emersonauthor=M.+Montminyauthor=R.+M.+Evans&title=A+transcriptional+switch+mediated+by+cofactor+methylation&doi=10.1126%2Fscience.1065961"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1126%2Fscience.1065961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1065961%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DK.%26aulast%3DAsahara%26aufirst%3DH.%26aulast%3DTini%26aufirst%3DM.%26aulast%3DEmerson%26aufirst%3DB.%2BM.%26aulast%3DMontminy%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26atitle%3DA%2520transcriptional%2520switch%2520mediated%2520by%2520cofactor%2520methylation%26jtitle%3DScience%26date%3D2001%26volume%3D294%26spage%3D2507%26epage%3D2511%26doi%3D10.1126%2Fscience.1065961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Malley, B. W.</span><span> </span><span class="NLM_article-title">Signaling within a coactivator complex: Methylation of SRC-3/AIB1 is a molecular switch for complex disassembly</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">7846</span><span class="NLM_x">–</span> <span class="NLM_lpage">7857</span><span class="refDoi"> DOI: 10.1128/MCB.00568-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1128%2FMCB.00568-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16923966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFemur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=7846-7857&author=Q.+Fengauthor=P.+Yiauthor=J.+M.+Wongauthor=B.+W.+O%E2%80%99Malley&title=Signaling+within+a+coactivator+complex%3A+Methylation+of+SRC-3%2FAIB1+is+a+molecular+switch+for+complex+disassembly&doi=10.1128%2FMCB.00568-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling within a coactivator complex: methylation of SRC-3/AIB1 is a molecular switch for complex disassembly</span></div><div class="casAuthors">Feng, Qin; Yi, Ping; Wong, Jiemin; O'Malley, Bert W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7846-7857</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Recent studies indicate that steroid receptor-mediated transcriptional initiation is a cyclical process involving multiple rounds of coactivator assembly and disassembly.  Steroid receptor coactivator 3 (SRC-3) coactivator phosphorylation has been shown to regulate coactivator complex assembly, but the mechanisms by which coactivator disassembly is triggered are not well understood.  In this study, we provide in vitro and in vivo evidence that members of the SRC coactivator family serve as substrates for the enzymic coactivator coactivator-assocd. arginine methyltransferase 1 (CARM1).  Methylation of SRC-3 was localized to an arginine in its CARM1 binding region and correlated with decreased estrogen receptor alpha-mediated transcription, as seen with both cell-based and in vitro transcription assays.  Consistent with this finding, we demonstrated that methylation promotes dissocn. of the SRC-3/CARM1 coactivator complex.  Methylation of SRC-3 is regulated by estrogen signaling in MCF7 cells and serves as a mol. switch for disassembly of the SRC-3 transcriptional coactivator complex.  We propose that CARM1 is a dual-function coactivator, as it not only activates transcription by modifying core histone tails but also terminates hormone signaling by disassembly of the coactivator complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7xFAwd4U-0bVg90H21EOLACvtfcHk0lg0USqmVlyRIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFemur7F&md5=e73b458635c8dc1fc629c275f39e429d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FMCB.00568-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00568-06%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DYi%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DJ.%2BM.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DB.%2BW.%26atitle%3DSignaling%2520within%2520a%2520coactivator%2520complex%253A%2520Methylation%2520of%2520SRC-3%252FAIB1%2520is%2520a%2520molecular%2520switch%2520for%2520complex%2520disassembly%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D7846%26epage%3D7857%26doi%3D10.1128%2FMCB.00568-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Chevillard-Briet, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trouche, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandel, L.</span><span> </span><span class="NLM_article-title">Control of CBP co-activating activity by arginine methylation</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">5457</span><span class="NLM_x">–</span> <span class="NLM_lpage">5466</span><span class="refDoi"> DOI: 10.1093/emboj/cdf548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1093%2Femboj%2Fcdf548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=12374746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvF2qu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=5457-5466&author=M.+Chevillard-Brietauthor=D.+Troucheauthor=L.+Vandel&title=Control+of+CBP+co-activating+activity+by+arginine+methylation&doi=10.1093%2Femboj%2Fcdf548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Control of CBP co-activating activity by arginine methylation</span></div><div class="casAuthors">Chevillard-Briet, Martine; Trouche, Didier; Vandel, Laurence</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5457-5466</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The histone acetyltransferases CREB binding protein (CBP) and the related p300 protein function as key transcriptional co-activators in multiple pathways.  In the case of transcriptional activation by nuclear receptors, ligand promotes the recruitment of co-activators of the p160 family, such as GRIP-1.  Subsequently, the p160 co-activators recruit other co-activators via two activation domains, AD1 and AD2.  AD1 binds CBP or p300, whereas AD2 has been shown to activate transcription through the recruitment of the arginine methyltransferase CARM1.  Recently, the KIX domain of CBP has been shown to be methylated by CARM1 in vitro.  Here, we report that another domain of CBP is specifically methylated by CARM1 on conserved arginine residues in vitro and in vivo.  We also provide functional evidence that arginine residues methylated by CARM1 play a crit. role in GRIP-1-dependent transcriptional activation and in hormone-induced gene activation.  Altogether, our data provide strong evidence that arginine methylation represents an important mechanism for modulating co-activator transcriptional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFw6WhlsWAkbVg90H21EOLACvtfcHk0lgzT-7AAYG5Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvF2qu7c%253D&md5=5ac0df871507f31a64c9890647478273</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdf548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdf548%26sid%3Dliteratum%253Aachs%26aulast%3DChevillard-Briet%26aufirst%3DM.%26aulast%3DTrouche%26aufirst%3DD.%26aulast%3DVandel%26aufirst%3DL.%26atitle%3DControl%2520of%2520CBP%2520co-activating%2520activity%2520by%2520arginine%2520methylation%26jtitle%3DEMBO%2520J.%26date%3D2002%26volume%3D21%26spage%3D5457%26epage%3D5466%26doi%3D10.1093%2Femboj%2Fcdf548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">PABP1 identified as an arginine methyltransferase substrate using high-density protein arrays</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">268</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span><span class="refDoi"> DOI: 10.1093/embo-reports/kvf052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1093%2Fembo-reports%2Fkvf052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=11850402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD38XitlCmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=268-273&author=J.+Leeauthor=M.+T.+Bedford&title=PABP1+identified+as+an+arginine+methyltransferase+substrate+using+high-density+protein+arrays&doi=10.1093%2Fembo-reports%2Fkvf052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">PABP1 identified as an arginine methyltransferase substrate using high-density protein arrays</span></div><div class="casAuthors">Lee, Jaeho; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">268-273</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The arginine methyltransferases CARM1 and PRMT1 assoc. with the p160 family of nuclear hormone receptor coactivators.  This assocn. enhances transcriptional activation by nuclear receptors.  We describe a method for identifying arginine N-methyltransferase substrates using arrayed high-d. protein membranes to perform solid-phase supported enzyme reactions in the presence of the Me donor S-adenosyl-L-methionine.  Using this screen, we identified distinct substrates for CARM1 and PRMT1.  All PRMT1 substrates harbor the expected GGRGG methylation motif, whereas the peptide sequence comparisons of the CARM1 substrates revealed no such motif.  The predominant CARM1 substrate identified in this screen was PABP1.  We mapped the methylated region of this RNA binding mol. in vitro and demonstrate that PABP1 is indeed methylated in vivo.  Prior to these findings, the only known substrate for CARM1 was histone H3.  We broaden the no. of CARM1 targets and suggest a role for CARM1 in regulating transcription/translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOsRoAKGzNqrVg90H21EOLACvtfcHk0lgzT-7AAYG5Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitlCmt7o%253D&md5=6d0be1647bdb90edeab2a00c4715b70d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fembo-reports%2Fkvf052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fembo-reports%252Fkvf052%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DPABP1%2520identified%2520as%2520an%2520arginine%2520methyltransferase%2520substrate%2520using%2520high-density%2520protein%2520arrays%26jtitle%3DEMBO%2520Rep.%26date%3D2002%26volume%3D3%26spage%3D268%26epage%3D273%26doi%3D10.1093%2Fembo-reports%2Fkvf052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kilburn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jelinek, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henschen-Edman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aswad, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stallcup, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laird-Offringa, I. A.</span><span> </span><span class="NLM_article-title">Lipopolysaccharide-induced methylation of HuR, an mRNA-stabilizing protein, by CARM1. Coactivator-associated arginine methyltransferase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">44623</span><span class="NLM_x">–</span> <span class="NLM_lpage">44630</span><span class="refDoi"> DOI: 10.1074/jbc.M206187200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M206187200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=44623-44630&author=H.+Liauthor=S.+Parkauthor=B.+Kilburnauthor=M.+A.+Jelinekauthor=A.+Henschen-Edmanauthor=D.+W.+Aswadauthor=M.+R.+Stallcupauthor=I.+A.+Laird-Offringa&title=Lipopolysaccharide-induced+methylation+of+HuR%2C+an+mRNA-stabilizing+protein%2C+by+CARM1.+Coactivator-associated+arginine+methyltransferase&doi=10.1074%2Fjbc.M206187200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206187200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206187200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DKilburn%26aufirst%3DB.%26aulast%3DJelinek%26aufirst%3DM.%2BA.%26aulast%3DHenschen-Edman%26aufirst%3DA.%26aulast%3DAswad%26aufirst%3DD.%2BW.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26aulast%3DLaird-Offringa%26aufirst%3DI.%2BA.%26atitle%3DLipopolysaccharide-induced%2520methylation%2520of%2520HuR%252C%2520an%2520mRNA-stabilizing%2520protein%252C%2520by%2520CARM1.%2520Coactivator-associated%2520arginine%2520methyltransferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D44623%26epage%3D44630%26doi%3D10.1074%2Fjbc.M206187200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Fujiwara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koyama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yachi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshikawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tohyama, M.</span><span> </span><span class="NLM_article-title">CARM1 regulates proliferation of PC12 cells by methylating HuD</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">2273</span><span class="NLM_x">–</span> <span class="NLM_lpage">2285</span><span class="refDoi"> DOI: 10.1128/MCB.26.6.2273-2285.2006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1128%2FMCB.26.6.2273-2285.2006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16508003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2273-2285&author=T.+Fujiwaraauthor=Y.+Moriauthor=D.+L.+Chuauthor=Y.+Koyamaauthor=S.+Miyataauthor=H.+Tanakaauthor=K.+Yachiauthor=T.+Kuboauthor=H.+Yoshikawaauthor=M.+Tohyama&title=CARM1+regulates+proliferation+of+PC12+cells+by+methylating+HuD&doi=10.1128%2FMCB.26.6.2273-2285.2006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">CARM1 regulates proliferation of PC12 cells by methylating HuD</span></div><div class="casAuthors">Fujiwara, Tatsuji; Mori, Yasutake; Chu, Dong Ling; Koyama, Yoshihisa; Miyata, Shingo; Tanaka, Hiroyuki; Yachi, Kohji; Kubo, Tateki; Yoshikawa, Hideki; Tohyama, Masaya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2273-2285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">HuD is an RNA-binding protein that has been shown to induce neuronal differentiation by stabilizing labile mRNAs carrying AU-rich instability elements.  Here, we show a novel mechanism of arginine methylation of HuD by coactivator-assocd. arginine methyltransferase 1 (CARM1) that affected mRNA turnover of p21cip1/waf1 mRNA in PC12 cells.  CARM1 specifically methylated HuD in vitro and in vivo and colocalized with HuD in the cytoplasm.  Inhibition of HuD methylation by CARM1 knockdown elongated the p21cip1/waf1 mRNA half-life and resulted in a slow growth rate and robust neuritogenesis in response to nerve growth factor (NGF).  Methylation-resistant HuD bound more p21cip1/waf1 mRNA than did the wild type, and its overexpression upregulated p21cip1/waf1 protein expression.  These results suggested that CARM1-methylated HuD maintains PC12 cells in the proliferative state by committing p21cip1/waf1 mRNA to its decay system.  Since the methylated population of HuD was reduced in NGF-treated PC12 cells, downregulation of HuD methylation is a possible pathway through which NGF induces differentiation of PC12 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71TYVPwUyRLVg90H21EOLACvtfcHk0lgzT-7AAYG5Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xislygtbo%253D&md5=a557599ff6460f5ab365362323b87a2b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FMCB.26.6.2273-2285.2006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.26.6.2273-2285.2006%26sid%3Dliteratum%253Aachs%26aulast%3DFujiwara%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DY.%26aulast%3DChu%26aufirst%3DD.%2BL.%26aulast%3DKoyama%26aufirst%3DY.%26aulast%3DMiyata%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DH.%26aulast%3DYachi%26aufirst%3DK.%26aulast%3DKubo%26aufirst%3DT.%26aulast%3DYoshikawa%26aufirst%3DH.%26aulast%3DTohyama%26aufirst%3DM.%26atitle%3DCARM1%2520regulates%2520proliferation%2520of%2520PC12%2520cells%2520by%2520methylating%2520HuD%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2006%26volume%3D26%26spage%3D2273%26epage%3D2285%26doi%3D10.1128%2FMCB.26.6.2273-2285.2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cote, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaaban, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span><span class="refDoi"> DOI: 10.1016/j.molcel.2006.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2006.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17218272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=71-83&author=D.+Chengauthor=J.+Coteauthor=S.+Shaabanauthor=M.+T.+Bedford&title=The+arginine+methyltransferase+CARM1+regulates+the+coupling+of+transcription+and+mRNA+processing&doi=10.1016%2Fj.molcel.2006.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The arginine methyltransferase CARM1 regulates the coupling of transcription and mRNA processing</span></div><div class="casAuthors">Cheng, Donghang; Cote, Jocelyn; Shaaban, Salam; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-83</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The coactivator-assocd. arginine methyl-transferase CARM1 is recruited by many different transcription factors as a pos. regulator.  To understand the mechanism by which CARM1 functions, we sought to isolate its substrates.  We developed a small-pool screening approach for this purpose and identified CA150, SAP49, SmB, and U1C as splicing factors that are specifically methylated by CARM1.  We further showed that CA150, a mol. that links transcription to splicing, interacts with the Tudor domain of the spinal muscular atrophy protein SMN in a CARM1-dependent fashion.  Expts. with an exogenous splicing reporter and the endogenous CD44 gene revealed that CARM1 promotes exon skipping in an enzyme-dependent manner.  The identification of splicing factors that are methylated by CARM1, and protein-protein interactions that are regulated by CARM1, strongly implicates this enzyme in the regulation of alternative splicing and points toward its involvement in spinal muscular atrophy pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqulZNAke_ZsrVg90H21EOLACvtfcHk0lh40bbyqzyeyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSnur8%253D&md5=29f6dccb415e1146e6455a03613f9fa2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DCote%26aufirst%3DJ.%26aulast%3DShaaban%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520arginine%2520methyltransferase%2520CARM1%2520regulates%2520the%2520coupling%2520of%2520transcription%2520and%2520mRNA%2520processing%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D71%26epage%3D83%26doi%3D10.1016%2Fj.molcel.2006.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richie, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Loss of CARM1 results in hypomethylation of thymocyte cyclic AMP-regulated phosphoprotein and deregulated early T cell development</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">25339</span><span class="NLM_x">–</span> <span class="NLM_lpage">25344</span><span class="refDoi"> DOI: 10.1074/jbc.M402544200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M402544200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=25339-25344&author=J.+Kimauthor=J.+Leeauthor=N.+Yadavauthor=Q.+Wuauthor=C.+Carterauthor=S.+Richardauthor=E.+Richieauthor=M.+T.+Bedford&title=Loss+of+CARM1+results+in+hypomethylation+of+thymocyte+cyclic+AMP-regulated+phosphoprotein+and+deregulated+early+T+cell+development&doi=10.1074%2Fjbc.M402544200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M402544200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M402544200%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DRichie%26aufirst%3DE.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DLoss%2520of%2520CARM1%2520results%2520in%2520hypomethylation%2520of%2520thymocyte%2520cyclic%2520AMP-regulated%2520phosphoprotein%2520and%2520deregulated%2520early%2520T%2520cell%2520development%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D25339%26epage%3D25344%26doi%3D10.1074%2Fjbc.M402544200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Sims, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rojas, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonasio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuller, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drury, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eick, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">The C-terminal domain of RNA polymerase II is modified by site-specific methylation</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1126/science.1202663</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1126%2Fscience.1202663" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21454787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVyisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2011&pages=99-103&author=R.+J.+Simsauthor=L.+A.+Rojasauthor=D.+Beckauthor=R.+Bonasioauthor=R.+Schullerauthor=W.+J.+Druryauthor=D.+Eickauthor=D.+Reinberg&title=The+C-terminal+domain+of+RNA+polymerase+II+is+modified+by+site-specific+methylation&doi=10.1126%2Fscience.1202663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The C-Terminal Domain of RNA Polymerase II Is Modified by Site-Specific Methylation</span></div><div class="casAuthors">Sims, Robert J., III; Rojas, Luis Alejandro; Beck, David; Bonasio, Roberto; Schueller, Roland; Drury, William J., III; Eick, Dirk; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">6025</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The carboxy-terminal domain (CTD) of RNA polymerase II (RNAPII) in mammals undergoes extensive posttranslational modification, which is essential for transcriptional initiation and elongation.  Here, we show that the CTD of RNAPII is methylated at a single arginine (R1810) by the coactivator-assocd. arginine methyltransferase 1 (CARM1).  Although methylation at R1810 is present on the hyperphosphorylated form of RNAPII in vivo, Ser2 or Ser5 phosphorylation inhibits CARM1 activity toward this site in vitro, suggesting that methylation occurs before transcription initiation.  Mutation of R1810 results in the misexpression of a variety of small nuclear RNAs and small nucleolar RNAs, an effect that is also obsd. in Carm1-/- mouse embryo fibroblasts.  These results demonstrate that CTD methylation facilitates the expression of select RNAs, perhaps serving to discriminate the RNAPII-assocd. machinery recruited to distinct gene types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6UKJDVEFMO7Vg90H21EOLACvtfcHk0lh40bbyqzyeyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjvVyisbo%253D&md5=f613b8e70477e245b52cf5dd5786ba3c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1202663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1202663%26sid%3Dliteratum%253Aachs%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DRojas%26aufirst%3DL.%2BA.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DBonasio%26aufirst%3DR.%26aulast%3DSchuller%26aufirst%3DR.%26aulast%3DDrury%26aufirst%3DW.%2BJ.%26aulast%3DEick%26aufirst%3DD.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520C-terminal%2520domain%2520of%2520RNA%2520polymerase%2520II%2520is%2520modified%2520by%2520site-specific%2520methylation%26jtitle%3DScience%26date%3D2011%26volume%3D332%26spage%3D99%26epage%3D103%26doi%3D10.1126%2Fscience.1202663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span> </span><span class="NLM_article-title">MED12 methylation by CARM1 sensitizes human breast cancer cells to chemotherapy drugs</span> <span class="citation_source-journal">Sci. Adv.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1500463</span><span class="refDoi"> DOI: 10.1126/sciadv.1500463</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1126%2Fsciadv.1500463" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=e1500463&author=L.+Wangauthor=H.+Zengauthor=Q.+Wangauthor=Z.+Zhaoauthor=T.+G.+Boyerauthor=X.+Bianauthor=W.+Xu&title=MED12+methylation+by+CARM1+sensitizes+human+breast+cancer+cells+to+chemotherapy+drugs&doi=10.1126%2Fsciadv.1500463"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500463%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DBoyer%26aufirst%3DT.%2BG.%26aulast%3DBian%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DMED12%2520methylation%2520by%2520CARM1%2520sensitizes%2520human%2520breast%2520cancer%2520cells%2520to%2520chemotherapy%2520drugs%26jtitle%3DSci.%2520Adv.%26date%3D2015%26volume%3D1%26spage%3De1500463%26doi%3D10.1126%2Fsciadv.1500463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">El Messaoudi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbrizio, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuchana, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fauquier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theillet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sardet, C.</span><span> </span><span class="NLM_article-title">Coactivator-associated arginine methyltransferase 1 (CARM1) is a positive regulator of the Cyclin E1 gene</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">13351</span><span class="NLM_x">–</span> <span class="NLM_lpage">13356</span><span class="refDoi"> DOI: 10.1073/pnas.0605692103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.0605692103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16938873" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVals7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=13351-13356&author=S.+El+Messaoudiauthor=E.+Fabbrizioauthor=C.+Rodriguezauthor=P.+Chuchanaauthor=L.+Fauquierauthor=D.+H.+Chengauthor=C.+Theilletauthor=L.+Vandelauthor=M.+T.+Bedfordauthor=C.+Sardet&title=Coactivator-associated+arginine+methyltransferase+1+%28CARM1%29+is+a+positive+regulator+of+the+Cyclin+E1+gene&doi=10.1073%2Fpnas.0605692103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Coactivator-associated arginine methyltransferase 1 (CARM1) is a positive regulator of the cyclin E1 gene</span></div><div class="casAuthors">El Messaoudi, Selma; Fabbrizio, Eric; Rodriguez, Carmen; Chuchana, Paul; Fauquier, Lucas; Cheng, Donghang; Theillet, Charles; Vandel, Laurence; Bedford, Mark T.; Sardet, Claude</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">13351-13356</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Cyclin E1 gene (CCNE1) is an ideal model to explore the mechanisms that control the transcription of cell cycle-regulated genes whose expression rises transiently before entry into S phase.  E2F-dependent regulation of the CCNE1 promoter was shown to correlate with changes in the level of H3-K9 acetylation/methylation of nucleosomal histones positioned at the transcriptional start site region.  Here we show that, upon growth stimulation, the same region is subject to variations of H3-R17 and H3-R26 methylation that correlate with the recruitment of coactivator-assocd. arginine methyltransferase 1 (CARM1) onto the CCNE1 and DHFR promoters.  Accordingly, CARM1-deficient cells lack these modifications and present lowered levels and altered kinetics of CCNE1 and DHFR mRNA expression.  Consistently, reporter gene assays demonstrate that CARM1 functions as a transcriptional coactivator for their E2F1/DP1-stimulated expression.  CARM1 recruitment at the CCNE1 gene requires activator E2Fs and ACTR, a member of the p160 coactivator family that is frequently overexpressed in human breast cancer.  Finally, we show that grade-3 breast tumors present coelevated mRNA levels of ACTR and CARM1, along with their transcriptional target CCNE1.  All together, our results indicate that CARM1 is an important regulator of the CCNE1 gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwAPzIlZVHFLVg90H21EOLACvtfcHk0lj7VATRXulHzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVals7c%253D&md5=377d79e0870e107b55675fb11b204ae8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0605692103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0605692103%26sid%3Dliteratum%253Aachs%26aulast%3DEl%2BMessaoudi%26aufirst%3DS.%26aulast%3DFabbrizio%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DC.%26aulast%3DChuchana%26aufirst%3DP.%26aulast%3DFauquier%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DTheillet%26aufirst%3DC.%26aulast%3DVandel%26aufirst%3DL.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DSardet%26aufirst%3DC.%26atitle%3DCoactivator-associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%2520is%2520a%2520positive%2520regulator%2520of%2520the%2520Cyclin%2520E1%2520gene%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D13351%26epage%3D13356%26doi%3D10.1073%2Fpnas.0605692103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Hong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kao, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeng, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eble, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacLennan, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, L.</span><span> </span><span class="NLM_article-title">Aberrant expression of CARM1, a transcriptional coactivator of androgen receptor, in the development of prostate carcinoma and androgen-independent status</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1002/cncr.20327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fcncr.20327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=15221992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFCnu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=83-89&author=H.+Hongauthor=C.+H.+Kaoauthor=M.+H.+Jengauthor=J.+N.+Ebleauthor=M.+O.+Kochauthor=T.+A.+Gardnerauthor=S.+B.+Zhangauthor=L.+Liauthor=C.+X.+Panauthor=Z.+Q.+Huauthor=G.+T.+MacLennanauthor=L.+Cheng&title=Aberrant+expression+of+CARM1%2C+a+transcriptional+coactivator+of+androgen+receptor%2C+in+the+development+of+prostate+carcinoma+and+androgen-independent+status&doi=10.1002%2Fcncr.20327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Aberrant expression of CARM1, a transcriptional coactivator of androgen receptor, in the development of prostate carcinoma and androgen-independent status</span></div><div class="casAuthors">Hong, Heng; Kao, Chinghai; Jeng, Meei-Huey; Eble, John N.; Koch, Michael O.; Gardner, Thomas A.; Zhang, Shaobo; Li, Lang; Pan, Chong-Xian; Hu, Zhiqiang; MacLennan, Gregory T.; Cheng, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-89</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1) is a transcriptional coactivator of the androgen receptor (AR).  It is involved in the regulation of the biol. functions of the AR.  It remains to be detd. whether CARM1 is involved in prostatic carcinogenesis.  The expression of CARM1 in normal prostate epithelium, high-grade prostatic intraepithelial neoplasia (PIN), and prostate carcinoma tissue was examd. in 66 previously untreated patients with prostate carcinomas, as well as 12 patients with hormone-independent prostate carcinoma, using immunohistochem. methods.  Cell staining was obsd. in the cytoplasm and the nucleus.  In 66 patients without previous hormonal treatment, the percentage of cells that stained pos. for CARM1 in benign prostate tissue specimens (mean values: cytoplasm, 23%; nucleus, 16%) was statistically significantly less than the percentage of pos. stained cells in PIN (mean values: cytoplasm, 56%; nucleus, 30%) and in prostatic adenocarcinoma specimens (mean values: cytoplasm, 79%; nucleus, 53%).  The difference between adenocarcinoma and PIN also was statistically significant.  The staining intensity for CARM1 was significantly lower in benign prostate tissue specimens compared with PIN and prostatic adenocarcinoma specimens.  In the 12 patients with androgen-independent prostatic adenocarcinoma, the expression of CARM1 was significantly increased when compared with patients without previous hormonal treatment.  Expression of CARM1 was not correlated with age, Gleason score sum, pathol. stage, lymph node metastasis, extraprostatic extension, surgical margin status, vascular invasion, or perineural invasion.  Thus, the authors found that overexpression of CARM1 was involved in the development of prostate carcinoma as well as androgen-independent prostate carcinoma.  Since CARM1 is functionally different from most other transcriptional coactivators of the AR, it may serve as a new target for the treatment of hormone-independent prostate carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSyswrcJiDBLVg90H21EOLACvtfcHk0lj7VATRXulHzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFCnu7k%253D&md5=143b9ca402cbd7c0223ed07db991f503</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fcncr.20327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.20327%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DH.%26aulast%3DKao%26aufirst%3DC.%2BH.%26aulast%3DJeng%26aufirst%3DM.%2BH.%26aulast%3DEble%26aufirst%3DJ.%2BN.%26aulast%3DKoch%26aufirst%3DM.%2BO.%26aulast%3DGardner%26aufirst%3DT.%2BA.%26aulast%3DZhang%26aufirst%3DS.%2BB.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DC.%2BX.%26aulast%3DHu%26aufirst%3DZ.%2BQ.%26aulast%3DMacLennan%26aufirst%3DG.%2BT.%26aulast%3DCheng%26aufirst%3DL.%26atitle%3DAberrant%2520expression%2520of%2520CARM1%252C%2520a%2520transcriptional%2520coactivator%2520of%2520androgen%2520receptor%252C%2520in%2520the%2520development%2520of%2520prostate%2520carcinoma%2520and%2520androgen-independent%2520status%26jtitle%3DCancer%26date%3D2004%26volume%3D101%26spage%3D83%26epage%3D89%26doi%3D10.1002%2Fcncr.20327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Kim, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N. T. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, C.</span><span> </span><span class="NLM_article-title">Differential CARM1 expression in prostate and colorectal cancers</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="refDoi"> DOI: 10.1186/1471-2407-10-197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1186%2F1471-2407-10-197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20462455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A280%3ADC%252BC3cvmsVSjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=197&author=Y.+R.+Kimauthor=B.+K.+Leeauthor=R.+Y.+Parkauthor=N.+T.+X.+Nguyenauthor=J.+A.+Baeauthor=D.+D.+Kwonauthor=C.+Jung&title=Differential+CARM1+expression+in+prostate+and+colorectal+cancers&doi=10.1186%2F1471-2407-10-197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Differential CARM1 expression in prostate and colorectal cancers</span></div><div class="casAuthors">Kim Young-Rang; Lee Byung Kook; Park Ra-Young; Nguyen Nguyen Thi Xuan; Bae Jeong A; Kwon Dong Deuk; Jung Chaeyong</div><div class="citationInfo"><span class="NLM_cas:title">BMC cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">197</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Coactivator-associated arginine methyltransferase 1 (CARM1) functions as a transcriptional coactivator of androgen receptor (AR)-mediated signaling.  Correspondingly, overexpression of CARM1 has been associated with the development of prostate cancer (PCa) and its progression to androgen-independent PCa.  In our preliminary study, however, the promoting effects of CARM1, with regard to androgen-stimulated AR target gene expression were minimal.  These results suggested that the AR target gene expression associated with CARM1 may result primarily from non-hormone dependent activity.  The goal of this study was to confirm the pattern of expression of CARM1 in human tumors and determine the mechanism of action in CARM1 overexpressed tumors.  METHODS:  Tissue microarray was used to determine the pattern of expression of CARM1 in human cancers by immunohistochemistry.  CARM1 expression was also evaluated in prostate and colorectal surgical specimens and the clinical records of all cases were reviewed.  In addition, a reporter transcription assay using the prostate-specific antigen (PSA) promoter was used to identify the signaling pathways involved in non-hormone-mediated signal activation associated with CARM1.  RESULTS:  The tissue microarray showed that CARM1 was particularly overexpressed in the colorectal cancers while CARM1 expression was not prevalent in the prostate and breast cancers.  Further studies using surgical specimens demonstrated that CARM1 was highly overexpressed in 75% of colorectal cancers (49 out of 65) but not in the androgen-independent PCa.  In addition, CARM1's coactivating effect on the entire PSA promoter was very limited in both androgen-dependent and androgen-independent PCa cells.  These results suggest that there are other factors associated with CARM1 expression in PSA regulation.  Indeed, CARM1 significantly regulated both p53 and NF-kappaB target gene transcription.  CONCLUSIONS:  The results of this study suggest that, in addition to its role in activation of steroid receptors, CARM1 functions as a transcriptional modulator by altering the activity of many transcriptional factors, especially with regard to androgen independent PCa and colorectal cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNfngOqNkUpvcsfMJQC1ZdfW6udTcc2eaBlOEDWLo2Tbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cvmsVSjuw%253D%253D&md5=d10897a9e65d3c395f5114eb22fc4650</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-10-197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-10-197%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%2BR.%26aulast%3DLee%26aufirst%3DB.%2BK.%26aulast%3DPark%26aufirst%3DR.%2BY.%26aulast%3DNguyen%26aufirst%3DN.%2BT.%2BX.%26aulast%3DBae%26aufirst%3DJ.%2BA.%26aulast%3DKwon%26aufirst%3DD.%2BD.%26aulast%3DJung%26aufirst%3DC.%26atitle%3DDifferential%2520CARM1%2520expression%2520in%2520prostate%2520and%2520colorectal%2520cancers%26jtitle%3DBMC%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D197%26doi%3D10.1186%2F1471-2407-10-197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Guccione, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bassi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinato, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cesaroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuchlautz, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amati, B.</span><span> </span><span class="NLM_article-title">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">933</span><span class="NLM_x">–</span> <span class="NLM_lpage">937</span><span class="refDoi"> DOI: 10.1038/nature06166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnature06166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17898714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=933-937&author=E.+Guccioneauthor=C.+Bassiauthor=F.+Casadioauthor=F.+Martinatoauthor=M.+Cesaroniauthor=H.+Schuchlautzauthor=B.+Luscherauthor=B.+Amati&title=Methylation+of+histone+H3R2+by+PRMT6+and+H3K4+by+an+MLL+complex+are+mutually+exclusive&doi=10.1038%2Fnature06166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span></div><div class="casAuthors">Guccione, Ernesto; Bassi, Christian; Casadio, Fabio; Martinato, Francesca; Cesaroni, Matteo; Schuchlautz, Henning; Luescher, Bernhard; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">933-937</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Eukaryotic genomes are organized into active (euchromatic) and inactive (heterochromatic) chromatin domains.  Post-translational modifications of histones (or 'marks') are key in defining these functional states, particularly in promoter regions.  Mutual regulatory interactions between these marks-and the enzymes that catalyze them-contribute to the shaping of this epigenetic landscape, in a manner that remains to be fully elucidated.  We previously obsd. that asym. di-methylation of histone H3 arginine 2 (H3R2me2a) counter-correlates with di- and tri- methylation of H3 lysine 4 (H3K4me2, H3K4me3) on human promoters.  Here we show that the arginine methyltransferase PRMT6 catalyzes H3R2 di-methylation in vitro and controls global levels of H3R2me2a in vivo.  H3R2 methylation by PRMT6 was prevented by the presence of H3K4me3 on the H3 tail.  Conversely, the H3R2me2a mark prevented methylation of H3K4 as well as binding to the H3 tail by an ASH2/WDR5/MLL-family methyltransferase complex.  Chromatin immunopptn. showed that H3R2me2a was distributed within the body and at the 3' end of human genes, regardless of their transcriptional state, whereas it was selectively and locally depleted from active promoters, coincident with the presence of H3K4me3.  Hence, the mutual antagonism between H3R2 and H3K4 methylation, together with the assocn. of MLL-family complexes with the basal transcription machinery, may contribute to the localized patterns of H3K4 tri-methylation characteristic of transcriptionally poised or active promoters in mammalian genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWqE8pDpHmKLVg90H21EOLACvtfcHk0ljh7e1bUSPTbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK&md5=f4cc9371491088af7b74b614c90d6bd4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnature06166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06166%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DMartinato%26aufirst%3DF.%26aulast%3DCesaroni%26aufirst%3DM.%26aulast%3DSchuchlautz%26aufirst%3DH.%26aulast%3DLuscher%26aufirst%3DB.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DMethylation%2520of%2520histone%2520H3R2%2520by%2520PRMT6%2520and%2520H3K4%2520by%2520an%2520MLL%2520complex%2520are%2520mutually%2520exclusive%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D933%26epage%3D937%26doi%3D10.1038%2Fnature06166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">El-Andaloussi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valovka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toueille, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinacher, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Focke, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehrig, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassa, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubscher, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hottiger, M. O.</span><span> </span><span class="NLM_article-title">Arginine methylation regulates DNA polymerase beta</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">62</span><span class="refDoi"> DOI: 10.1016/j.molcel.2006.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.molcel.2006.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16600869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=51-62&author=N.+El-Andaloussiauthor=T.+Valovkaauthor=M.+Toueilleauthor=R.+Steinacherauthor=F.+Fockeauthor=P.+Gehrigauthor=M.+Covicauthor=P.+O.+Hassaauthor=P.+Scharauthor=U.+Hubscherauthor=M.+O.+Hottiger&title=Arginine+methylation+regulates+DNA+polymerase+beta&doi=10.1016%2Fj.molcel.2006.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates DNA polymerase β</span></div><div class="casAuthors">El-Andaloussi, Nazim; Valovka, Taras; Toueille, Magali; Steinacher, Roland; Focke, Frauke; Gehrig, Peter; Covic, Marcela; Hassa, Paul O.; Schar, Primo; Hubscher, Ulrich; Hottiger, Michael O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-62</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Alterations in DNA repair lead to genomic instability and higher risk of cancer.  DNA base excision repair (BER) corrects damaged bases, apurinic sites, and single-strand DNA breaks.  Here, a regulatory mechanism for DNA polymerase β (Pol β) is described.  Pol β was found to form a complex with the protein arginine methyltransferase 6 (PRMT6) and was specifically methylated in vitro and in vivo.  Methylation of Pol β by PRMT6 strongly stimulated DNA polymerase activity by enhancing DNA binding and processivity, while single nucleotide insertion and dRP-lyase activity were not affected.  Two residues, R83 and R152, were identified in Pol β as the sites of methylation by PRMT6.  Genetic complementation of Pol β knockout cells with R83/152K mutant revealed the importance of these residues for the cellular resistance to DNA alkylating agent.  Based on our findings, we propose that PRMT6 plays a role as a regulator of BER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqui7wNvLvrVg90H21EOLACvtfcHk0ljh7e1bUSPTbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D&md5=987419489fb0439acb13f3b78d2d5a1e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Andaloussi%26aufirst%3DN.%26aulast%3DValovka%26aufirst%3DT.%26aulast%3DToueille%26aufirst%3DM.%26aulast%3DSteinacher%26aufirst%3DR.%26aulast%3DFocke%26aufirst%3DF.%26aulast%3DGehrig%26aufirst%3DP.%26aulast%3DCovic%26aufirst%3DM.%26aulast%3DHassa%26aufirst%3DP.%2BO.%26aulast%3DSchar%26aufirst%3DP.%26aulast%3DHubscher%26aufirst%3DU.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26atitle%3DArginine%2520methylation%2520regulates%2520DNA%2520polymerase%2520beta%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D22%26spage%3D51%26epage%3D62%26doi%3D10.1016%2Fj.molcel.2006.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Boulanger, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span> </span><span class="NLM_article-title">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1128/JVI.79.1.124-131.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1128%2FJVI.79.1.124-131.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=15596808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=124-131&author=M.+C.+Boulangerauthor=C.+Liangauthor=R.+S.+Russellauthor=R.+Linauthor=M.+T.+Bedfordauthor=M.+A.+Wainbergauthor=S.+Richard&title=Methylation+of+Tat+by+PRMT6+regulates+human+immunodeficiency+virus+type+1+gene+expression&doi=10.1128%2FJVI.79.1.124-131.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span></div><div class="casAuthors">Boulanger, Marie-Chloe; Liang, Chen; Russell, Rodney S.; Lin, Rongtuan; Bedford, Mark T.; Wainberg, Mark A.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-131</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The human immunodeficiency virus (HIV) transactivator protein, Tat, stimulates transcription from the viral long terminal repeats via an arginine-rich transactivating domain.  Since arginines are often known to be methylated, authors investigated whether HIV type 1 (HIV-1) Tat was a substrate for known protein arginine methyltransferases (PRMTs).  Here authors identify Tat as a substrate for the arginine methyltransferase, PRMT6.  Tat is specifically assocd. with and methylated by PRMT6 within cells.  Overexpression of wild-type PRMT6, but not a methylase-inactive PRMT6 mutant, decreased Tat transactivation of an HIV-1 long terminal repeat luciferase reporter plasmid in a dose-dependent manner.  Knocking down PRMT6 consistently increased HIV-1 prodn. in HEK293T cells and also led to increased viral infectiousness as shown in multinuclear activation of a galactosidase indicator assays.  This study demonstrates that arginine methylation of Tat neg. regulates its transactivation activity and that PRMT6 acts as a restriction factor for HIV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0JF5C5Agwq7Vg90H21EOLACvtfcHk0ljh7e1bUSPTbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D&md5=763ac9af247b3f81962ae4e19419493a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.1.124-131.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.1.124-131.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBoulanger%26aufirst%3DM.%2BC.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DR.%2BS.%26aulast%3DLin%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DMethylation%2520of%2520Tat%2520by%2520PRMT6%2520regulates%2520human%2520immunodeficiency%2520virus%2520type%25201%2520gene%2520expression%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D124%26epage%3D131%26doi%3D10.1128%2FJVI.79.1.124-131.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Sgarra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tessari, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spolaore, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giancotti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manfioletti, G.</span><span> </span><span class="NLM_article-title">The AT-hook of the chromatin architectural transcription factor high mobility group A1a is arginine-methylated by protein arginine methyltransferase 6</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">3764</span><span class="NLM_x">–</span> <span class="NLM_lpage">3772</span><span class="refDoi"> DOI: 10.1074/jbc.M510231200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M510231200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=16293633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=3764-3772&author=R.+Sgarraauthor=J.+Leeauthor=M.+A.+Tessariauthor=S.+Altamuraauthor=B.+Spolaoreauthor=V.+Giancottiauthor=M.+T.+Bedfordauthor=G.+Manfioletti&title=The+AT-hook+of+the+chromatin+architectural+transcription+factor+high+mobility+group+A1a+is+arginine-methylated+by+protein+arginine+methyltransferase+6&doi=10.1074%2Fjbc.M510231200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The AT-hook of the Chromatin Architectural Transcription Factor High Mobility Group A1a Is Arginine-methylated by Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Sgarra, Riccardo; Lee, Jaeho; Tessari, Michela A.; Altamura, Sandro; Spolaore, Barbara; Giancotti, Vincenzo; Bedford, Mark T.; Manfioletti, Guidalberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3764-3772</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The HMGA1a protein belongs to the high mobility group A (HMGA) family of architectural nuclear factors, a group of proteins that plays an important role in chromatin dynamics.  HMGA proteins are multifunctional factors that assoc. both with DNA and nuclear proteins that have been involved in several nuclear processes, such as transcriptional regulation, viral integration, DNA repair, RNA processing, and chromatin remodeling.  The activity of HMGA proteins is finely modulated by a variety of post-translational modifications.  Arginine methylation was recently demonstrated to occur on HMGA1a protein, and it correlates with the apoptotic process and neoplastic progression.  Methyltransferases responsible for these modifications are unknown.  Here we show that the protein arginine methyltransferase PRMT6 specifically methylates HMGA1a protein both in vitro and in vivo.  By mass spectrometry, the sites of methylation were unambiguously mapped to Arg57 and Arg59, two residues which are embedded in the second AT-hook, a region crit. for both protein-DNA and protein-protein interactions and whose modification may cause profound alterations in the HMGA network.  The in vivo assocn. of HMGA and PRMT6 place this yet functionally uncharacterized methyltransferase in the well established functional context of the chromatin structure organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKv8D5yCRQ4rVg90H21EOLACvtfcHk0lh7gc-rywt2ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D&md5=803d5cd75cc725a9c381220bc067d330</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M510231200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M510231200%26sid%3Dliteratum%253Aachs%26aulast%3DSgarra%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTessari%26aufirst%3DM.%2BA.%26aulast%3DAltamura%26aufirst%3DS.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DGiancotti%26aufirst%3DV.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DManfioletti%26aufirst%3DG.%26atitle%3DThe%2520AT-hook%2520of%2520the%2520chromatin%2520architectural%2520transcription%2520factor%2520high%2520mobility%2520group%2520A1a%2520is%2520arginine-methylated%2520by%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D3764%26epage%3D3772%26doi%3D10.1074%2Fjbc.M510231200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Limm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oefner, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellerbrand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosserhoff, A. K.</span><span> </span><span class="NLM_article-title">Deregulation of protein methylation in melanoma</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1305</span><span class="NLM_x">–</span> <span class="NLM_lpage">1313</span><span class="refDoi"> DOI: 10.1016/j.ejca.2012.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.ejca.2012.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23265702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=1305-1313&author=K.+Limmauthor=C.+Ottauthor=S.+Wallnerauthor=D.+W.+Muellerauthor=P.+Oefnerauthor=C.+Hellerbrandauthor=A.+K.+Bosserhoff&title=Deregulation+of+protein+methylation+in+melanoma&doi=10.1016%2Fj.ejca.2012.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of protein methylation in melanoma</span></div><div class="casAuthors">Limm, Katharina; Ott, Corinna; Wallner, Susanne; Mueller, Daniel W.; Oefner, Peter; Hellerbrand, Claus; Bosserhoff, Anja-Katrin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1305-1313</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Loss of methylthioadenosine phosphorylase (MTAP) expression and a concomitant accumulation of 5'-methyl-thioadenosine (MTA) characterize several tumor entities including malignant melanoma.  MTA affects cellular signalling, proliferation and migration not only of cancer but also surrounding cells including lymphocytes and stromal fibroblasts.  The mode of action of MTA is still not known.  Interestingly, MTA is a known potent inhibitor of protein arginine methyltransferases (PRMTs) and is used as a tool in studying activity and impact of PRMTs.  This study aimed at analyzing PRMTs in melanoma and the potential impact of MTA on tumorigenesis.Our findings demonstrate that expression of PRMT4/CARM1 and PRMT6 is deregulated in melanoma, whereas expression of the remaining PRMTs stays unchanged.  General PRMT activity and, consequently, sym. and asym. protein methylation are reduced significantly in melanoma cells and tissues.  This is due to a loss of MTAP expression and accumulation of MTA.  Redn. of protein methylation by MTA affects cell signalling and leads, for example, to an activation of extracellular signal-regulated kinase (ERK) activity.  The effects of endogeneous MTA on PRMTs as presented in this study can strongly support the migratory and invasive phenotype of melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqujaHA2MQGG7Vg90H21EOLACvtfcHk0lh7gc-rywt2ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF&md5=a29a9efe0acd75da35bb7af04bd26dc4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2012.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2012.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DLimm%26aufirst%3DK.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DWallner%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DD.%2BW.%26aulast%3DOefner%26aufirst%3DP.%26aulast%3DHellerbrand%26aufirst%3DC.%26aulast%3DBosserhoff%26aufirst%3DA.%2BK.%26atitle%3DDeregulation%2520of%2520protein%2520methylation%2520in%2520melanoma%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2013%26volume%3D49%26spage%3D1305%26epage%3D1313%26doi%3D10.1016%2Fj.ejca.2012.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maehara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Dysregulation of PRMT1 and PRMT6, Type I arginine methyltransferases, is involved in various types of human cancers</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">573</span><span class="refDoi"> DOI: 10.1002/ijc.25366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fijc.25366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20473859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=562-573&author=M.+Yoshimatsuauthor=G.+Toyokawaauthor=S.+Hayamiauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=J.+D.+Kellyauthor=D.+E.+Nealauthor=Y.+Maeharaauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Dysregulation+of+PRMT1+and+PRMT6%2C+Type+I+arginine+methyltransferases%2C+is+involved+in+various+types+of+human+cancers&doi=10.1002%2Fijc.25366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dysregulation of PRMT1 and PRMT6, Type I arginine methyltransferases, is involved in various types of human cancers</span></div><div class="casAuthors">Yoshimatsu Masanori; Toyokawa Gouji; Hayami Shinya; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Kelly John D; Neal David E; Maehara Yoshihiko; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">562-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methylation is a novel post-translational modification regulating a diversity of cellular processes, including histone functions, but the roles of protein arginine methyltransferases (PRMTs) in human cancer are not well investigated.  To address this issue, we first examined expression levels of genes belonging to the PRMT family and found significantly higher expression of PRMT1 and PRMT6, both of which are Type I PRMTs, in cancer cells of various tissues than in non-neoplastic cells.  Abrogation of the expression of these genes with specific siRNAs significantly suppressed growth of bladder and lung cancer cells.  Expression profile analysis using the cells transfected with the siRNAs indicated that PRMT1 and PRMT6 interplay in multiple pathways, supporting regulatory roles in the cell cycle, RNA processing and also DNA replication that are fundamentally important for cancer cell proliferation.  Furthermore, we demonstrated that serum asymmetric dimethylarginine (ADMA) levels of a number of cancer cases are significantly higher than those of nontumor control cases.  In summary, our results suggest that dysregulation of PRMT1 and PRMT6 can be involved in human carcinogenesis and that these Type I arginine methyltransferases are good therapeutic targets for various types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WjEwO064Z95rfW6udTcc2eZxrnLg3HOyFLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D&md5=34889d958208a4d852b0833e21a045de</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fijc.25366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25366%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DMaehara%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDysregulation%2520of%2520PRMT1%2520and%2520PRMT6%252C%2520Type%2520I%2520arginine%2520methyltransferases%252C%2520is%2520involved%2520in%2520various%2520types%2520of%2520human%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D562%26epage%3D573%26doi%3D10.1002%2Fijc.25366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Vieira, F. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa-Pinheiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramalho-Carvalho, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menezes, F. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antunes, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carneiro, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrique, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeronimo, C.</span><span> </span><span class="NLM_article-title">Deregulated expression of selected histone methylases and demethylases in prostate carcinoma</span> <span class="citation_source-journal">Endocr.-Relat. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1530/ERC-13-0375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1530%2FERC-13-0375" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=51-61&author=F.+Q.+Vieiraauthor=P.+Costa-Pinheiroauthor=J.+Ramalho-Carvalhoauthor=A.+Pereiraauthor=F.+D.+Menezesauthor=L.+Antunesauthor=I.+Carneiroauthor=J.+Oliveiraauthor=R.+Henriqueauthor=C.+Jeronimo&title=Deregulated+expression+of+selected+histone+methylases+and+demethylases+in+prostate+carcinoma&doi=10.1530%2FERC-13-0375"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1530%2FERC-13-0375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FERC-13-0375%26sid%3Dliteratum%253Aachs%26aulast%3DVieira%26aufirst%3DF.%2BQ.%26aulast%3DCosta-Pinheiro%26aufirst%3DP.%26aulast%3DRamalho-Carvalho%26aufirst%3DJ.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DMenezes%26aufirst%3DF.%2BD.%26aulast%3DAntunes%26aufirst%3DL.%26aulast%3DCarneiro%26aufirst%3DI.%26aulast%3DOliveira%26aufirst%3DJ.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJeronimo%26aufirst%3DC.%26atitle%3DDeregulated%2520expression%2520of%2520selected%2520histone%2520methylases%2520and%2520demethylases%2520in%2520prostate%2520carcinoma%26jtitle%3DEndocr.-Relat.%2520Cancer%26date%3D2014%26volume%3D21%26spage%3D51%26epage%3D61%26doi%3D10.1530%2FERC-13-0375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeRoy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muir, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">H3R42me2a is a histone modification with positive transcriptional effects</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">14894</span><span class="NLM_x">–</span> <span class="NLM_lpage">14899</span><span class="refDoi"> DOI: 10.1073/pnas.1312925110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.1312925110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23980157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFWrsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=14894-14899&author=F.+Casadioauthor=X.+Luauthor=S.+B.+Pollockauthor=G.+LeRoyauthor=B.+A.+Garciaauthor=T.+W.+Muirauthor=R.+G.+Roederauthor=C.+D.+Allis&title=H3R42me2a+is+a+histone+modification+with+positive+transcriptional+effects&doi=10.1073%2Fpnas.1312925110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">H3R42me2a is a histone modification with positive transcriptional effects</span></div><div class="casAuthors">Casadio, Fabio; Lu, Xiangdong; Pollock, Samuel B.; LeRoy, Gary; Garcia, Benjamin A.; Muir, Tom W.; Roeder, Robert G.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">14894-14899,S14894/1-S14894/4</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Histone post-translational modification leads to downstream effects indirectly by allowing or preventing docking of effector mols., or directly by changing the intrinsic biophys. properties of local chromatin.  To date, little has been done to study post-translational modifications that lie outside of the unstructured tail domains of histones.  Core residues, and in particular arginines in H3 and H4, mediate key interactions between the histone octamer and DNA in forming the nucleosomal particle.  Using mass spectrometry, we find that one of these core residues, arginine 42 of histone H3 (H3R42), is dimethylated in mammalian cells by the methyltransferases coactivator arginine methyltransferase 1 (CARM1) and protein arginine methyltransferase 6 (PRMT6) in vitro and in vivo, and we demonstrate that methylation of H3R42 stimulates transcription in vitro from chromatinized templates.  Thus, H3R42 is a new, "nontail" histone methylation site with pos. effects on transcription.  We propose that methylation of basic histone residues at the DNA interface may disrupt histone:DNA interactions, with effects on downstream processes, notably transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBUDvvIfjKRbVg90H21EOLACvtfcHk0lhuOaquwBjNig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFWrsr3L&md5=eda5ad0f47a66b06d3309fdcaf05d47a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1312925110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1312925110%26sid%3Dliteratum%253Aachs%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPollock%26aufirst%3DS.%2BB.%26aulast%3DLeRoy%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DMuir%26aufirst%3DT.%2BW.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DH3R42me2a%2520is%2520a%2520histone%2520modification%2520with%2520positive%2520transcriptional%2520effects%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D14894%26epage%3D14899%26doi%3D10.1073%2Fpnas.1312925110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Harrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowhan, D. H.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">2201</span><span class="NLM_x">–</span> <span class="NLM_lpage">2216</span><span class="refDoi"> DOI: 10.1093/nar/gkp1203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1093%2Fnar%2Fgkp1203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20047962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=2201-2216&author=M.+J.+Harrisonauthor=Y.+H.+Tangauthor=D.+H.+Dowhan&title=Protein+arginine+methyltransferase+6+regulates+multiple+aspects+of+gene+expression&doi=10.1093%2Fnar%2Fgkp1203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 6 regulates multiple aspects of gene expression</span></div><div class="casAuthors">Harrison, Matthew J.; Tang, Yue Hang; Dowhan, Dennis H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2201-2216</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">It is well established that transcription and alternative splicing events are functionally coupled during gene expression.  Here, we report that protein arginine N-methyltransferase 6 (PRMT6) may play a key role in this coupling process by functioning as a transcriptional coactivator that can also regulate alternative splicing.  PRMT6 coactivates the progesterone, glucocorticoid and estrogen receptors in luciferase reporter assays in a hormone-dependent manner.  In addn., small interfering RNA (siRNA) oligonucleotide duplex knockdown of PRMT6 disrupts estrogen-stimulated transcription of endogenous GREB1 and progesterone receptor in MCF-7 breast cancer cells, demonstrating the importance of PRMT6 in hormone-dependent transcription.  In contrast, the regulation of alternative splicing by PRMT6 is hormone independent.  SiRNA knockdown of PRMT6 increases the exon inclusion:skipping ratio of alternatively spliced exons in endogenous vascular endothelial growth factor and spleen tyrosine kinase RNA transcripts in both the presence and absence of estrogen.  These results demonstrate that PRMT6 has a dual role in regulating gene expression and that these two functions can occur independently of each other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA8_-JQGR0vbVg90H21EOLACvtfcHk0lhuOaquwBjNig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVOrtro%253D&md5=471281cfc866a6db8be4a7f8c9a009e5</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1203%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DM.%2BJ.%26aulast%3DTang%26aufirst%3DY.%2BH.%26aulast%3DDowhan%26aufirst%3DD.%2BH.%26atitle%3DProtein%2520arginine%2520methyltransferase%25206%2520regulates%2520multiple%2520aspects%2520of%2520gene%2520expression%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D2201%26epage%3D2216%26doi%3D10.1093%2Fnar%2Fgkp1203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span><span class="refDoi"> DOI: 10.1038/nchembio.296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe&doi=10.1038%2Fnchembio.296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lhuOaquwBjNig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161%26doi%3D10.1038%2Fnchembio.296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Chemical probes of histone lysine methyltransferases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1021/cb500785t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500785t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=40-50&author=H.+U.+Kaniskanauthor=J.+Jin&title=Chemical+probes+of+histone+lysine+methyltransferases&doi=10.1021%2Fcb500785t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes of histone lysine methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-50</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that histone-lysine methyltransferases (HMTs, also known as protein methyltransferases (PMTs)) play an important role in diverse biol. processes and human diseases by regulating gene expression and the chromatin state.  Therefore, HMTs have been increasingly recognized by the biomedical community as a class of potential therapeutic targets.  High quality chem. probes of HMTs, as tools for deciphering their physiol. functions and roles in human diseases and testing therapeutic hypotheses, are crit. for advancing this promising field.  Here, the authors focus on the discovery, characterization, and biol. applications of chem. probes for HMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp8kb_c7Xl57Vg90H21EOLACvtfcHk0lhkO9cMTxYIQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOht7%252FE&md5=4eb8a696e2227c60686df8d3c60777ce</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fcb500785t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500785t%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DChemical%2520probes%2520of%2520histone%2520lysine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D40%26epage%3D50%26doi%3D10.1021%2Fcb500785t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of protein arginine methyltransferase 3</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1425</span><span class="NLM_x">–</span> <span class="NLM_lpage">1435</span><span class="refDoi"> DOI: 10.1016/j.str.2012.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.str.2012.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=22795084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1425-1435&author=A.+Siarheyevaauthor=G.+Senisterraauthor=A.+Allali-Hassaniauthor=A.+P.+Dongauthor=E.+Dobrovetskyauthor=G.+A.+Wasneyauthor=I.+Chauauthor=R.+Marcellusauthor=T.+Hajianauthor=F.+Liuauthor=I.+Korboukhauthor=D.+Smilauthor=Y.+Bolshanauthor=J.+R.+Minauthor=H.+Wuauthor=H.+Zengauthor=P.+Loppnauauthor=G.+Podaauthor=C.+Griffinauthor=A.+Amanauthor=P.+J.+Brownauthor=J.+Jinauthor=R.+Al-awarauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=An+allosteric+inhibitor+of+protein+arginine+methyltransferase+3&doi=10.1016%2Fj.str.2012.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">An Allosteric Inhibitor of Protein Arginine Methyltransferase 3</span></div><div class="casAuthors">Siarheyeva, Alena; Senisterra, Guillermo; Allali-Hassani, Abdellah; Dong, Aiping; Dobrovetsky, Elena; Wasney, Gregory A.; Chau, Irene; Marcellus, Richard; Hajian, Taraneh; Liu, Feng; Korboukh, Ilia; Smil, David; Bolshan, Yuri; Min, Jinrong; Wu, Hong; Zeng, Hong; Loppnau, Peter; Poda, Gennadiy; Griffin, Carly; Aman, Ahmed; Brown, Peter J.; Jin, Jian; Al-awar, Rima; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1425-1435</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PRMT3, a protein arginine methyltransferase, has been shown to influence ribosomal biosynthesis by catalyzing the dimethylation of the 40S ribosomal protein S2.  Although PRMT3 has been reported to be a cytosolic protein, it has been shown to methylate histone H4 peptide (H4 1-24) in vitro.  Here, we report the identification of a PRMT3 inhibitor (1-(benzo[d][1,2,3]thiadiazol-6-yl)-3-(2-cyclohexenylethyl)urea; compd. 1) with IC50 value of 2.5 μM by screening a library of 16,000 compds. using H4 (1-24) peptide as a substrate.  The crystal structure of PRMT3 in complex with compd. 1 as well as kinetic anal. reveals an allosteric mechanism of inhibition.  Mutating PRMT3 residues within the allosteric site or using compd. 1 analogs that disrupt interactions with allosteric site residues both abrogated binding and inhibitory activity.  These data demonstrate an allosteric mechanism for inhibition of protein arginine methyltransferases, an emerging class of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruLurDJu10tLVg90H21EOLACvtfcHk0lhkO9cMTxYIQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVektr3N&md5=c745acac0beb494318f4577293281ddf</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DJ.%2BR.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1425%26epage%3D1435%26doi%3D10.1016%2Fj.str.2012.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+L.+Liauthor=A.+Q.+Maauthor=E.+Dobrovetskyauthor=A.+P.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lhkO9cMTxYIQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMa%26aufirst%3DA.%2BQ.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin-Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5166</span><span class="NLM_x">–</span> <span class="NLM_lpage">5170</span><span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+U.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+T.+Yuauthor=M.+S.+Eramauthor=X.+B.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+L.+Liauthor=E.+Dobrovetskyauthor=A.+P.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+H.+Zhaoauthor=J.+Jinauthor=M.+Vedadi&title=A+potent%2C+selective+and+cell-active+allosteric+inhibitor+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Uemit; Szewczyk, Magdalena M.; Yu, Zhengtian; Eram, Mohammad S.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Kennedy, Steven; Li, Fengling; Dobrovetsky, Elena; Dong, Aiping; Smil, David; Min, Sun-Joon; Landon, Melissa; Lin-Jones, Jennifer; Huang, Xi-Ping; Roth, Bryan L.; Schapira, Matthieu; Atadja, Peter; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Zhao, Kehao; Jin, Jian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-5170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biol.  Here we report the discovery of the first PRMT3 chem. probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochem., biophys., and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50=31±2 nM, KD=53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chem. probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTk4dFfkDLm7Vg90H21EOLACvtfcHk0ljvyHMB8HCvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D&md5=283b0875813af0f7631235e8ba7811c2</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%2BT.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%2BB.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520potent%252C%2520selective%2520and%2520cell-active%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Discovery of a dual PRMT5-PRMT7 inhibitor</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">412</span><span class="refDoi"> DOI: 10.1021/ml500467h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500467h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=408-412&author=D.+Smilauthor=M.+S.+Eramauthor=F.+L.+Liauthor=S.+Kennedyauthor=M.+M.+Szewczykauthor=P.+J.+Brownauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+dual+PRMT5-PRMT7+inhibitor&doi=10.1021%2Fml500467h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Dual PRMT5-PRMT7 Inhibitor</span></div><div class="casAuthors">Smil, David; Eram, Mohammad S.; Li, Fengling; Kennedy, Steven; Szewczyk, Magdalena M.; Brown, Peter J.; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">408-412</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein arginine methyltransferases PRMT7 and PRMT5, resp., monomethylate and sym. dimethylate arginine side-chains of proteins involved in diverse cellular mechanisms, including chromatin-mediated control of gene transcription, splicing, and the RAS to ERK transduction cascade.  It is believed that PRMT5 and PRMT7 act in conjunction to methylate their substrates, and genetic deletions support the notion that these enzymes derepress cell proliferation and migration in cancer.  Using available structures of PRMT5, we designed DS-437, a PRMT5 inhibitor with an IC50 value of 6 μM against both PRMT5 and PRMT7 that is inactive against 29 other human protein-, DNA-, and RNA-methyltransferases and inhibits sym. dimethylation of PRMT5 substrates in cells.  This compd. behaves as a cofactor competitor and represents a valid scaffold to interrogate the potential of the PRMT5-PRMT7 axis as a target for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9J078gEcjRLVg90H21EOLACvtfcHk0ljvyHMB8HCvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Ggu7k%253D&md5=8eefe93068418c61881850d3b4d39676</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fml500467h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500467h%26sid%3Dliteratum%253Aachs%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520dual%2520PRMT5-PRMT7%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D408%26epage%3D412%26doi%3D10.1021%2Fml500467h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Speed, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dela Sena, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">781</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+Shenauthor=M.+M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+Liauthor=K.+V.+Butlerauthor=H.+U.+Kaniskanauthor=B.+A.+Speedauthor=C.+Dela+Senaauthor=A.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0ljvyHMB8HCvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Ferreira de Freitas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steuber, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchcock, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moosmayer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stegmann, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1176</span><span class="NLM_x">–</span> <span class="NLM_lpage">1183</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira+de+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0lgPs0HoJcBZpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Ferreira de Freitas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczyk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Discovery of a Potent and Selective Coactivator Associated Arginine Methyltransferase 1 (CARM1) Inhibitor by Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">6838</span><span class="NLM_x">–</span> <span class="NLM_lpage">6847</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00668</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00668" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6838-6847&author=R.+Ferreira+de+Freitasauthor=M.+S.+Eramauthor=D.+Smilauthor=M.+M.+Szewczykauthor=S.+Kennedyauthor=P.+J.+Brownauthor=V.+Santhakumarauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+Potent+and+Selective+Coactivator+Associated+Arginine+Methyltransferase+1+%28CARM1%29+Inhibitor+by+Virtual+Screening&doi=10.1021%2Facs.jmedchem.6b00668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Coactivator Associated Arginine Methyltransferase 1 (CARM1) Inhibitor by Virtual Screening</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Smil, David; Szewczyk, Magdalena M.; Kennedy, Steven; Brown, Peter J.; Santhakumar, Vijayaratnam; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6838-6847</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) represent an emerging target class in oncol. and other disease areas.  So far, the most successful strategy to identify PRMT inhibitors has been to screen large to medium-size chem. libraries.  Attempts to develop PRMT inhibitors using receptor-based computational methods have met limited success.  Here, using virtual screening approaches, we identify 11 CARM1 (PRMT4) inhibitors with ligand efficiencies ranging from 0.28 to 0.84.  CARM1 selective hits were further validated by orthogonal methods.  Two structure-based rounds of optimization produced 27 (SGC2085), a CARM1 inhibitor with an IC50 of 50 nM and more than hundred-fold selectivity over other PRMTs.  These results indicate that virtual screening strategies can be successfully applied to Rossmann-fold protein methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF-4WiwX0trVg90H21EOLACvtfcHk0lgPs0HoJcBZpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFamu77I&md5=6a5daefe79f879bd732df11cf5b31e8a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00668%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520Coactivator%2520Associated%2520Arginine%2520Methyltransferase%25201%2520%2528CARM1%2529%2520Inhibitor%2520by%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6838%26epage%3D6847%26doi%3D10.1021%2Facs.jmedchem.6b00668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Sack, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thieffine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandiera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fasolini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duke, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kish, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosettani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troiani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, J. A.</span><span> </span><span class="NLM_article-title">Structural basis for CARM1 inhibition by indole and pyrazole inhibitors</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">436</span><span class="NLM_x">, </span> <span class="NLM_fpage">331</span><span class="NLM_x">–</span> <span class="NLM_lpage">339</span><span class="refDoi"> DOI: 10.1042/BJ20102161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1042%2FBJ20102161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21410432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtFGitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2011&pages=331-339&author=J.+S.+Sackauthor=S.+Thieffineauthor=T.+Bandieraauthor=M.+Fasoliniauthor=G.+J.+Dukeauthor=L.+Jayaramanauthor=K.+F.+Kishauthor=H.+E.+Kleiauthor=A.+V.+Purandareauthor=P.+Rosettaniauthor=S.+Troianiauthor=D.+L.+Xieauthor=J.+A.+Bertrand&title=Structural+basis+for+CARM1+inhibition+by+indole+and+pyrazole+inhibitors&doi=10.1042%2FBJ20102161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for CARM1 inhibition by indole and pyrazole inhibitors</span></div><div class="casAuthors">Sack, John S.; Thieffine, Sandrine; Bandiera, Tiziano; Fasolini, Marina; Duke, Gerald J.; Jayaraman, Lata; Kish, Kevin F.; Klei, Herbert E.; Purandare, Ashok V.; Rosettani, Pamela; Troiani, Sonia; Xie, Dianlin; Bertrand, Jay A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-339</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">CARM1 (co-activator-assocd. arginine methyltransferase 1) is a PRMT (protein arginine N-methyltransferase) family member that catalyzes the transfer of Me groups from SAM (S-adenosylmethionine) to the side chain of specific arginine residues of substrate proteins.  This post-translational modification of proteins regulates a variety of transcriptional events and other cellular processes.  Moreover, CARM1 is a potential oncol. target due to its multiple roles in transcription activation by nuclear hormone receptors and other transcription factors such as p53.  Here, crystal structures of the CARM1 catalytic domain in complex with cofactors [SAH (S-adenosyl-L-homocysteine) or SNF (sinefungin)] and indole or pyrazole inhibitors are presented.  Anal. of the structures reveals that the inhibitors bind in the arginine-binding cavity and the surrounding pocket that exists at the interface between the N- and C-terminal domains.  In addn., it is shown using ITC (isothermal titrn. calorimetry) that the inhibitors bind to the CARM1 catalytic domain only in the presence of the cofactor SAH.  Furthermore, sequence differences for select residues that interact with the inhibitors may be responsible for the CARM1 selectivity against PRMT1 and PRMT3.  Together, the structural and biophys. information should aid in the design of both potent and specific inhibitors of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeVM8tGrZkjrVg90H21EOLACvtfcHk0libmLEEZKQ0Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtFGitro%253D&md5=029cfceb945caac01a19d39f247c4c0a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1042%2FBJ20102161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20102161%26sid%3Dliteratum%253Aachs%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DThieffine%26aufirst%3DS.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DFasolini%26aufirst%3DM.%26aulast%3DDuke%26aufirst%3DG.%2BJ.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DKish%26aufirst%3DK.%2BF.%26aulast%3DKlei%26aufirst%3DH.%2BE.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26aulast%3DRosettani%26aufirst%3DP.%26aulast%3DTroiani%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DD.%2BL.%26aulast%3DBertrand%26aufirst%3DJ.%2BA.%26atitle%3DStructural%2520basis%2520for%2520CARM1%2520inhibition%2520by%2520indole%2520and%2520pyrazole%2520inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2011%26volume%3D436%26spage%3D331%26epage%3D339%26doi%3D10.1042%2FBJ20102161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Allan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manku, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Styhler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulet, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petschner, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahil, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacLeod, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deziel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besterman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahhab, A.</span><span> </span><span class="NLM_article-title">N-Benzyl-1-heteroaryl-3-(trifluoromethyl)-1H-pyrazole-5-carboxamides as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1218</span><span class="NLM_x">–</span> <span class="NLM_lpage">1223</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.12.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2008.12.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19131248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Wruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1218-1223&author=M.+Allanauthor=S.+Mankuauthor=E.+Therrienauthor=N.+Nguyenauthor=S.+Styhlerauthor=M.+F.+Robertauthor=A.+C.+Gouletauthor=A.+J.+Petschnerauthor=G.+Rahilauthor=A.+R.+MacLeodauthor=R.+Dezielauthor=J.+M.+Bestermanauthor=H.+Nguyenauthor=A.+Wahhab&title=N-Benzyl-1-heteroaryl-3-%28trifluoromethyl%29-1H-pyrazole-5-carboxamides+as+inhibitors+of+co-activator+associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2008.12.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">N-Benzyl-1-heteroaryl-3-(trifluoromethyl)-1H-pyrazole-5-carboxamides as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Allan, Martin; Manku, Sukhdev; Therrien, Eric; Nguyen, Natalie; Styhler, Sylvia; Robert, Marie-France; Goulet, Anne-Christine; Petschner, Andrea J.; Rahil, Gabi; Robert MacLeod, A.; Deziel, Robert; Besterman, Jeffrey M.; Nguyen, Hannah; Wahhab, Amal</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1218-1223</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of N-benzyl-1-heteroaryl-3-(trifluoromethyl)-1H-pyrazole-5-carboxamides, e.g. I (R = Ph, 2-MeC6H4, 2-MeOC6H4, 2-FC6H4, 3-MeOC6H4, 2-MeO2CC6H4, 2-pyridyl), targeting co-activator assocd. arginine methyltransferase 1 (CARM1) was designed and synthesized.  The potency of these inhibitors was influenced by the nature of the heteroaryl fragment with the thiophene analogs being superior to thiazole, pyridine, isoindoline and benzofuran based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow4qukRyeUALVg90H21EOLACvtfcHk0libmLEEZKQ0Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Wruro%253D&md5=68aece67d648b302a99f117eb872c2e2</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.12.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.12.075%26sid%3Dliteratum%253Aachs%26aulast%3DAllan%26aufirst%3DM.%26aulast%3DManku%26aufirst%3DS.%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DStyhler%26aufirst%3DS.%26aulast%3DRobert%26aufirst%3DM.%2BF.%26aulast%3DGoulet%26aufirst%3DA.%2BC.%26aulast%3DPetschner%26aufirst%3DA.%2BJ.%26aulast%3DRahil%26aufirst%3DG.%26aulast%3DMacLeod%26aufirst%3DA.%2BR.%26aulast%3DDeziel%26aufirst%3DR.%26aulast%3DBesterman%26aufirst%3DJ.%2BM.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DWahhab%26aufirst%3DA.%26atitle%3DN-Benzyl-1-heteroaryl-3-%2528trifluoromethyl%2529-1H-pyrazole-5-carboxamides%2520as%2520inhibitors%2520of%2520co-activator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1218%26epage%3D1223%26doi%3D10.1016%2Fj.bmcl.2008.12.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milite, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limongelli, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of carboxy analogues of arginine methyltransferase inhibitor 1 (AMI-1)</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">398</span><span class="NLM_x">–</span> <span class="NLM_lpage">414</span><span class="refDoi"> DOI: 10.1002/cmdc.200900459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1002%2Fcmdc.200900459" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=398-414&author=S.+Castellanoauthor=C.+Militeauthor=R.+Ragnoauthor=S.+Simeoniauthor=A.+Maiauthor=V.+Limongelliauthor=E.+Novellinoauthor=I.+Bauerauthor=G.+Broschauthor=A.+Spannhoffauthor=D.+H.+Chengauthor=M.+T.+Bedfordauthor=G.+Sbardella&title=Design%2C+synthesis+and+biological+evaluation+of+carboxy+analogues+of+arginine+methyltransferase+inhibitor+1+%28AMI-1%29&doi=10.1002%2Fcmdc.200900459"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900459%26sid%3Dliteratum%253Aachs%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DMilite%26aufirst%3DC.%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DLimongelli%26aufirst%3DV.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%2BH.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520carboxy%2520analogues%2520of%2520arginine%2520methyltransferase%2520inhibitor%25201%2520%2528AMI-1%2529%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D398%26epage%3D414%26doi%3D10.1002%2Fcmdc.200900459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandiera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bindi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roletto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thieffine, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenzi, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottardis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span> </span><span class="NLM_article-title">Optimization of pyrazole inhibitors of Coactivator Associated Arginine Methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2924</span><span class="NLM_x">–</span> <span class="NLM_lpage">2927</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2009.04.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19419866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2924-2927&author=T.+Huynhauthor=Z.+Chenauthor=S.+H.+Pangauthor=J.+P.+Gengauthor=T.+Bandieraauthor=S.+Bindiauthor=P.+Vianelloauthor=F.+Rolettoauthor=S.+Thieffineauthor=A.+Galvaniauthor=W.+Vaccaroauthor=M.+A.+Possauthor=G.+L.+Trainorauthor=M.+V.+Lorenziauthor=M.+Gottardisauthor=L.+Jayaramanauthor=A.+V.+Purandare&title=Optimization+of+pyrazole+inhibitors+of+Coactivator+Associated+Arginine+Methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2009.04.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Huynh, Tram; Chen, Zhong; Pang, Suhong; Geng, Jieping; Bandiera, Tiziano; Bindi, Simona; Vianello, Paola; Roletto, Fulvia; Thieffine, Sandrine; Galvani, Arturo; Vaccaro, Wayne; Poss, Michael A.; Trainor, George L.; Lorenzi, Matthew V.; Gottardis, Marco; Jayaraman, Lata; Purandare, Ashok V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2924-2927</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Design, synthesis, and SAR development led to the identification of the potent, novel, and selective pyrazole based inhibitor I of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphFardNepde7Vg90H21EOLACvtfcHk0libmLEEZKQ0Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVekur4%253D&md5=c37a2738bbd001d4758f8480746626fb</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.075%26sid%3Dliteratum%253Aachs%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3DGeng%26aufirst%3DJ.%2BP.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DBindi%26aufirst%3DS.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DRoletto%26aufirst%3DF.%26aulast%3DThieffine%26aufirst%3DS.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DGottardis%26aufirst%3DM.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26atitle%3DOptimization%2520of%2520pyrazole%2520inhibitors%2520of%2520Coactivator%2520Associated%2520Arginine%2520Methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2924%26epage%3D2927%26doi%3D10.1016%2Fj.bmcl.2009.04.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Purandare, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oconnell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowak, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span> </span><span class="NLM_article-title">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4438</span><span class="NLM_x">–</span> <span class="NLM_lpage">4441</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2008.06.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=18619839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4438-4441&author=A.+V.+Purandareauthor=Z.+Chenauthor=T.+Huynhauthor=S.+Pangauthor=J.+Gengauthor=W.+Vaccaroauthor=M.+A.+Possauthor=J.+Oconnellauthor=K.+Nowakauthor=L.+Jayaraman&title=Pyrazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2008.06.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Purandare, Ashok V.; Chen, Zhong; Huynh, Tram; Pang, Suhong; Geng, Jieping; Vaccaro, Wayne; Poss, Michael A.; Oconnell, Jonathan; Nowak, Kimberly; Jayaraman, Lata</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4438-4441</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">This study reports the identification and Hits to Leads optimization of inhibitors of coactivator assocd. arginine methyltransferase (CARM1).  Compd. (I) is a potent, selective inhibitor of CARM1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjsEj-ubHXsrVg90H21EOLACvtfcHk0lhfjwO_QkE2HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptVCjsr8%253D&md5=0ef68fcfa0dedbe54d1fa5b8ae9ca376</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DJ.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DOconnell%26aufirst%3DJ.%26aulast%3DNowak%26aufirst%3DK.%26aulast%3DJayaraman%26aufirst%3DL.%26atitle%3DPyrazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4438%26epage%3D4441%26doi%3D10.1016%2Fj.bmcl.2008.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Therrien, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larouche, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manku, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Styhler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulet, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Besterman, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahhab, A.</span><span> </span><span class="NLM_article-title">1,2-Diamines as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">6725</span><span class="NLM_x">–</span> <span class="NLM_lpage">6732</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.09.110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2009.09.110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=19836951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6725-6732&author=E.+Therrienauthor=G.+Laroucheauthor=S.+Mankuauthor=M.+Allanauthor=N.+Nguyenauthor=S.+Styhlerauthor=M.+F.+Robertauthor=A.+C.+Gouletauthor=J.+M.+Bestermanauthor=H.+Nguyenauthor=A.+Wahhab&title=1%2C2-Diamines+as+inhibitors+of+co-activator+associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1016%2Fj.bmcl.2009.09.110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">1,2-Diamines as inhibitors of co-activator associated arginine methyltransferase 1 (CARM1)</span></div><div class="casAuthors">Therrien, Eric; Larouche, Guillaume; Manku, Sukhdev; Allan, Martin; Nguyen, Natalie; Styhler, Sylvia; Robert, Marie-France; Goulet, Anne-Christine; Besterman, Jeffrey M.; Nguyen, Hannah; Wahhab, Amal</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6725-6732</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have identified the N 1-benzyl-N 2-methylethane-1,2-diamine unit as a substitute for the (S)-alanine benzylamide moiety for the design of co-activator assocd. arginine methyltransferase 1 (CARM1) inhibitors.  The potency of these inhibitors is in the same order of magnitude as their predecessors and their clearance, vol. of distribution, and half lives were greatly improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_1G-CFEVWrVg90H21EOLACvtfcHk0lhfjwO_QkE2HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7I&md5=808ff34d43b9beb4efbf149c253acf6c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.09.110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.09.110%26sid%3Dliteratum%253Aachs%26aulast%3DTherrien%26aufirst%3DE.%26aulast%3DLarouche%26aufirst%3DG.%26aulast%3DManku%26aufirst%3DS.%26aulast%3DAllan%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DStyhler%26aufirst%3DS.%26aulast%3DRobert%26aufirst%3DM.%2BF.%26aulast%3DGoulet%26aufirst%3DA.%2BC.%26aulast%3DBesterman%26aufirst%3DJ.%2BM.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DWahhab%26aufirst%3DA.%26atitle%3D1%252C2-Diamines%2520as%2520inhibitors%2520of%2520co-activator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6725%26epage%3D6732%26doi%3D10.1016%2Fj.bmcl.2009.09.110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Wan, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huynh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poss, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trainor, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorenzi, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottardis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayaraman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purandare, A. V.</span><span> </span><span class="NLM_article-title">Benzo[d]imidazole inhibitors of coactivator associated arginine methyltransferase 1 (CARM1)-hit to lead studies</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">5063</span><span class="NLM_x">–</span> <span class="NLM_lpage">5066</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.07.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bmcl.2009.07.040" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=5063-5066&author=H.+H.+Wanauthor=T.+Huynhauthor=S.+H.+Pangauthor=J.+P.+Gengauthor=W.+Vaccaroauthor=M.+A.+Possauthor=G.+L.+Trainorauthor=M.+V.+Lorenziauthor=M.+Gottardisauthor=L.+Jayaramanauthor=A.+V.+Purandare&title=Benzo%5Bd%5Dimidazole+inhibitors+of+coactivator+associated+arginine+methyltransferase+1+%28CARM1%29-hit+to+lead+studies&doi=10.1016%2Fj.bmcl.2009.07.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.07.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.07.040%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DH.%2BH.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DS.%2BH.%26aulast%3DGeng%26aufirst%3DJ.%2BP.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DPoss%26aufirst%3DM.%2BA.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DGottardis%26aufirst%3DM.%26aulast%3DJayaraman%26aufirst%3DL.%26aulast%3DPurandare%26aufirst%3DA.%2BV.%26atitle%3DBenzo%255Bd%255Dimidazole%2520inhibitors%2520of%2520coactivator%2520associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529-hit%2520to%2520lead%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D5063%26epage%3D5066%26doi%3D10.1016%2Fj.bmcl.2009.07.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Mitchell, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drew, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swinger, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moradei, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riera, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter-Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: identification of the first PRMT6 tool compound</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0lgOMiJUsQlIyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuplast, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stickland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbash, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nurse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oza, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallagher, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lgOMiJUsQlIyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">van Haren, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Ufford, L. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moret, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, N. I.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of protein arginine N-methyltransferase inhibitors designed to simultaneously occupy both substrate binding sites</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">560</span><span class="refDoi"> DOI: 10.1039/C4OB01734J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1039%2FC4OB01734J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=25380215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSiu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=549-560&author=M.+van+Harenauthor=L.+Q.+van+Uffordauthor=E.+E.+Moretauthor=N.+I.+Martin&title=Synthesis+and+evaluation+of+protein+arginine+N-methyltransferase+inhibitors+designed+to+simultaneously+occupy+both+substrate+binding+sites&doi=10.1039%2FC4OB01734J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of protein arginine N-methyltransferase inhibitors designed to simultaneously occupy both substrate binding sites</span></div><div class="casAuthors">van Haren, Matthijs; van Ufford, Linda Quarles; Moret, Ed E.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein arginine N-methyltransferases (PRMTs) are a family of enzymes that function by specifically transferring a Me group from the cofactor S-adenosyl-L-methionine (AdoMet) to the guanidine group of arginine residues in target proteins.  The most notable is the PRMT-mediated methylation of arginine residues that are present in histone proteins which can lead to chromatin remodelling and influence gene transcription.  A growing body of evidence now implicates dysregulated PRMT activity in a no. of diseases including various forms of cancer.  The development of PRMT inhibitors may therefore hold potential as a means of developing new therapeutics.  We here report the synthesis and evaluation of a series of small mol. PRMT inhibitors designed to simultaneously occupy the binding sites of both the guanidino substrate and AdoMet cofactor.  Potent inhibition and surprising selectivity were obsd. when testing these compds. against a panel of methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcFfPMCcpn97Vg90H21EOLACvtfcHk0lgOMiJUsQlIyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSiu77N&md5=62b07b10e6cc1c288168e8bd0ef4d9f5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2FC4OB01734J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01734J%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%26aulast%3Dvan%2BUfford%26aufirst%3DL.%2BQ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520protein%2520arginine%2520N-methyltransferase%2520inhibitors%2520designed%2520to%2520simultaneously%2520occupy%2520both%2520substrate%2520binding%2520sites%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D549%26epage%3D560%26doi%3D10.1039%2FC4OB01734J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martini, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santhakumar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Design and synthesis of selective, small molecule inhibitors of coactivator-associated arginine methyltransferase 1 (CARM1)</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1039/C6MD00342G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1039%2FC6MD00342G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=H.+U.+Kaniskanauthor=M.+S.+Eramauthor=J.+Liuauthor=D.+Smilauthor=M.+L.+Martiniauthor=Y.+Shenauthor=V.+Santhakumarauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=J.+Jin&title=Design+and+synthesis+of+selective%2C+small+molecule+inhibitors+of+coactivator-associated+arginine+methyltransferase+1+%28CARM1%29&doi=10.1039%2FC6MD00342G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1039%2FC6MD00342G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00342G%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520selective%252C%2520small%2520molecule%2520inhibitors%2520of%2520coactivator-associated%2520arginine%2520methyltransferase%25201%2520%2528CARM1%2529%26jtitle%3DMedChemComm%26date%3D2016%26doi%3D10.1039%2FC6MD00342G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lioDL95jnZB_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">2212</span><span class="NLM_x">–</span> <span class="NLM_lpage">2221</span><span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lioDL95jnZB_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Blat, Y.</span><span> </span><span class="NLM_article-title">Non-competitive inhibition by active site binders</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">535</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2010.00972.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1111%2Fj.1747-0285.2010.00972.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=20374252" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtlaru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=535-540&author=Y.+Blat&title=Non-competitive+inhibition+by+active+site+binders&doi=10.1111%2Fj.1747-0285.2010.00972.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Non-competitive inhibition by active site binders</span></div><div class="casAuthors">Blat, Yuval</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">535-540</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Classical enzymol. has been used for generations to understand the interactions of inhibitors with their enzyme targets.  Enzymol. tools enabled prediction of the biol. impact of inhibitors as well as the development of novel, more potent, ones.  Expts. designed to examine the competition between the tested inhibitor and the enzyme substrate(s) are the tool of choice to identify inhibitors that bind in the active site.  Competition between an inhibitor and a substrate is considered a strong evidence for binding of the inhibitor in the active site, while the lack of competition suggests binding to an alternative site.  Nevertheless, exceptions to this notion do exist.  Active site-binding inhibitors can display non-competitive inhibition patterns.  This unusual behavior has been obsd. with enzymes utilizing an exosite for substrate binding, isomechanism enzymes, enzymes with multiple substrates and/or products and two-step binding inhibitors.  In many of these cases, the mechanisms underlying the lack of competition between the substrate and the inhibitor are well understood.  Tools like alternative substrates, testing the enzyme reaction in the reverse direction and monitoring inhibition time dependence can be applied to enable distinction between 'badly behaving' active site binders and true exosite inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQh6oNO4kuy7Vg90H21EOLACvtfcHk0lh2BtezTN-rxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtlaru7g%253D&md5=fc61590b33e72e75999c2fdd0da716a6</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.00972.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.00972.x%26sid%3Dliteratum%253Aachs%26aulast%3DBlat%26aufirst%3DY.%26atitle%3DNon-competitive%2520inhibition%2520by%2520active%2520site%2520binders%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D75%26spage%3D535%26epage%3D540%26doi%3D10.1111%2Fj.1747-0285.2010.00972.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Mitchell, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomenius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riera, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span> </span><span class="NLM_article-title">Novel Oxindole Sulfonamides and Sulfamides: EPZ031686, the First Orally Bioavailable Small Molecule SMYD3 Inhibitor</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">134</span><span class="NLM_x">–</span> <span class="NLM_lpage">138</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=134-138&author=L.+H.+Mitchellauthor=P.+A.+Boriack-Sjodinauthor=S.+Smithauthor=M.+Thomeniusauthor=N.+Riouxauthor=M.+Munchhofauthor=J.+E.+Millsauthor=C.+Klausauthor=J.+Totmanauthor=T.+V.+Rieraauthor=A.+Raimondiauthor=S.+L.+Jacquesauthor=K.+Westauthor=M.+Foleyauthor=N.+J.+Watersauthor=K.+W.+Kuntzauthor=T.+J.+Wigleauthor=M.+P.+Scottauthor=R.+A.+Copelandauthor=J.+J.+Smithauthor=R.+Chesworth&title=Novel+Oxindole+Sulfonamides+and+Sulfamides%3A+EPZ031686%2C+the+First+Orally+Bioavailable+Small+Molecule+SMYD3+Inhibitor&doi=10.1021%2Facsmedchemlett.5b00272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oxindole Sulfonamides and Sulfamides: EPZ031686, the First Orally Bioavailable Small Molecule SMYD3 Inhibitor</span></div><div class="casAuthors">Mitchell, Lorna H.; Boriack-Sjodin, P. Ann; Smith, Sherri; Thomenius, Michael; Rioux, Nathalie; Munchhof, Michael; Mills, James E.; Klaus, Christine; Totman, Jennifer; Riera, Thomas V.; Raimondi, Alejandra; Jacques, Suzanne L.; West, Kip; Foley, Megan; Waters, Nigel J.; Kuntz, Kevin W.; Wigle, Tim J.; Scott, Margaret Porter; Copeland, Robert A.; Smith, Jesse J.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">134-138</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SMYD3 has been implicated in a range of cancers; however, until now no potent selective small mol. inhibitors have been available for target validation studies.  A novel oxindole series of SMYD3 inhibitors was identified through screening of the Epizyme proprietary histone methyltransferase-biased library.  Potency optimization afforded two tool compds., sulfonamide EPZ031686 and sulfamide EPZ030456, with cellular potency at a level sufficient to probe the in vitro biol. of SMYD3 inhibition.  EPZ031686 shows good bioavailability following oral dosing in mice making it a suitable tool for potential in vivo target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQtO_uOhxeYrVg90H21EOLACvtfcHk0lh2BtezTN-rxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVahsrnJ&md5=3fd7612cb086796a0bcc5c6e0c1f1352</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00272%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DThomenius%26aufirst%3DM.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%26aulast%3DMills%26aufirst%3DJ.%2BE.%26aulast%3DKlaus%26aufirst%3DC.%26aulast%3DTotman%26aufirst%3DJ.%26aulast%3DRiera%26aufirst%3DT.%2BV.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DNovel%2520Oxindole%2520Sulfonamides%2520and%2520Sulfamides%253A%2520EPZ031686%252C%2520the%2520First%2520Orally%2520Bioavailable%2520Small%2520Molecule%2520SMYD3%2520Inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D134%26epage%3D138%26doi%3D10.1021%2Facsmedchemlett.5b00272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Hyllus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnabel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiltz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, U. M.</span><span> </span><span class="NLM_article-title">PRMT6-mediated methylation of R2 in histone H3 antagonizes H3 K4 trimethylation</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">3369</span><span class="NLM_x">–</span> <span class="NLM_lpage">3380</span><span class="refDoi"> DOI: 10.1101/gad.447007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1101%2Fgad.447007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=18079182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrsb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=3369-3380&author=D.+Hyllusauthor=C.+Steinauthor=K.+Schnabelauthor=E.+Schiltzauthor=A.+Imhofauthor=Y.+Douauthor=J.+Hsiehauthor=U.+M.+Bauer&title=PRMT6-mediated+methylation+of+R2+in+histone+H3+antagonizes+H3+K4+trimethylation&doi=10.1101%2Fgad.447007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT6-mediated methylation of R2 in histone H3 antagonizes H3 K4 trimethylation</span></div><div class="casAuthors">Hyllus, Dawin; Stein, Claudia; Schnabel, Kristin; Schiltz, Emile; Imhof, Axel; Dou, Yali; Hsieh, James; Bauer, Uta-Maria</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3369-3380</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The arginine methyltransferase PRMT6 (protein arginine methyltransferase 6) has been shown recently to regulate DNA repair and gene expression.  As arginine methylation of histones is an important mechanism in transcriptional regulation, we asked whether PRMT6 possesses activity toward histones.  We show here that PRMT6 methylates histone H3 at R2 and histones H4/H2A at R3 in vitro.  Overexpression and knockdown anal. identify PRMT6 as the major H3 R2 methyltransferase in vivo.  We find that H3 R2 methylation inhibits H3 K4 trimethylation and recruitment of WDR5, a subunit of the MLL (mixed lineage leukemia) K4 methyltransferase complex, to histone H3 in vitro.  Upon PRMT6 overexpression, transcription of Hox genes and Myc-dependent genes, both well-known targets of H3 K4 trimethylation, decreases.  This transcriptional repression coincides with enhanced occurrence of H3 R2 methylation and PRMT6 as well as reduced levels of H3 K4 trimethylation and MLL1/WDR5 recruitment at the HoxA2 gene.  Upon retinoic acid-induced transcriptional activation of HoxA2 in a cell model of neuronal differentiation, PRMT6 recruitment and H3 R2 methylation are diminished and H3 K4 trimethylation increases at the gene.  Our findings identify PRMT6 as the mammalian methyltransferase for H3 R2 and establish the enzyme as a crucial neg. regulator of H3 K4 trimethylation and transcriptional activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsPUkDIyRzr7Vg90H21EOLACvtfcHk0lh2BtezTN-rxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrsb%252FE&md5=951b9c1612c9f95b20951dec04e098d2</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1101%2Fgad.447007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.447007%26sid%3Dliteratum%253Aachs%26aulast%3DHyllus%26aufirst%3DD.%26aulast%3DStein%26aufirst%3DC.%26aulast%3DSchnabel%26aufirst%3DK.%26aulast%3DSchiltz%26aufirst%3DE.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DHsieh%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DU.%2BM.%26atitle%3DPRMT6-mediated%2520methylation%2520of%2520R2%2520in%2520histone%2520H3%2520antagonizes%2520H3%2520K4%2520trimethylation%26jtitle%3DGenes%2520Dev.%26date%3D2007%26volume%3D21%26spage%3D3369%26epage%3D3380%26doi%3D10.1101%2Fgad.447007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Waldmann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamieniarz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindner, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittler, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span> </span><span class="NLM_article-title">Methylation of H2AR29 is a novel repressive PRMT6 target</span> <span class="citation_source-journal">Epigenet. Chromatin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="refDoi"> DOI: 10.1186/1756-8935-4-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1186%2F1756-8935-4-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=21774791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCktLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2011&pages=11&author=T.+Waldmannauthor=A.+Izzoauthor=K.+Kamieniarzauthor=F.+Richterauthor=C.+Voglerauthor=B.+Sargauthor=H.+Lindnerauthor=N.+L.+Youngauthor=G.+Mittlerauthor=B.+A.+Garciaauthor=R.+Schneider&title=Methylation+of+H2AR29+is+a+novel+repressive+PRMT6+target&doi=10.1186%2F1756-8935-4-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of H2AR29 is a novel repressive PRMT6 target</span></div><div class="casAuthors">Waldmann, Tanja; Izzo, Annalisa; Kamieniarz, Kinga; Richter, Florian; Vogler, Christine; Sarg, Bettina; Lindner, Herbert; Young, Nicolas L.; Mittler, Gerhard; Garcia, Benjamin A.; Schneider, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Covalent histone modifications are central to all DNA-dependent processes.  Modifications of histones H3 and H4 are becoming well characterized, however, our knowledge of how H2A modifications regulate chromatin dynamics and gene expression is still very limited.  Results: To understand the function of H2A modifications we performed a systematic anal. of the histone H2A methylation status.  We identified and functionally characterized two new methylation sites in H2A: R11 (H2AR11) and R29 (H2AR29).  By using an unbiased biochem. approach in combination with candidate assays we show that the arginine methyltransferase (PRMT) 1 and PRMT6 are unique in their ability to catalyze these modifications.  Importantly we find that H2AR29me2 is specifically enriched at genes repressed by PRMT6, implicating H2AR29me2 in transcriptional repression.  Conclusions: Our data establishes R11 and R29 as new arginine methylation sites in H2A.  We identified the specific modifying enzymes and uncovered a novel functional role of H2AR29me2 in gene silencing in vivo.  Thus this work reveals novel insights into the function of H2A methylation as well as in the mechanisms of PRMT6-mediated transcriptional repression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw0BxwOba9E7Vg90H21EOLACvtfcHk0ljNp9DjEX3pXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCktLjL&md5=2c66fd635d53774195965cbb86b94c40</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-4-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-4-11%26sid%3Dliteratum%253Aachs%26aulast%3DWaldmann%26aufirst%3DT.%26aulast%3DIzzo%26aufirst%3DA.%26aulast%3DKamieniarz%26aufirst%3DK.%26aulast%3DRichter%26aufirst%3DF.%26aulast%3DVogler%26aufirst%3DC.%26aulast%3DSarg%26aufirst%3DB.%26aulast%3DLindner%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DN.%2BL.%26aulast%3DMittler%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DSchneider%26aufirst%3DR.%26atitle%3DMethylation%2520of%2520H2AR29%2520is%2520a%2520novel%2520repressive%2520PRMT6%2520target%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2011%26volume%3D4%26spage%3D11%26doi%3D10.1186%2F1756-8935-4-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustos, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Trimethylation of histone H3 lysine 36 by human methyltransferase PRDM9 protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">12177</span><span class="NLM_x">–</span> <span class="NLM_lpage">12188</span><span class="refDoi"> DOI: 10.1074/jbc.M113.523183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1074%2Fjbc.M113.523183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=12177-12188&author=M.+S.+Eramauthor=S.+P.+Bustosauthor=E.+Lima-Fernandesauthor=A.+Siarheyevaauthor=G.+Senisterraauthor=T.+Hajianauthor=I.+Chauauthor=S.+Duanauthor=H.+Wuauthor=L.+Dombrovskiauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Trimethylation+of+histone+H3+lysine+36+by+human+methyltransferase+PRDM9+protein&doi=10.1074%2Fjbc.M113.523183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.523183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.523183%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DBustos%26aufirst%3DS.%2BP.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DTrimethylation%2520of%2520histone%2520H3%2520lysine%252036%2520by%2520human%2520methyltransferase%2520PRDM9%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D12177%26epage%3D12188%26doi%3D10.1074%2Fjbc.M113.523183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">1842</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2015.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.bbagen.2015.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26002201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=1842-1848&author=M.+S.+Eramauthor=E.+Kuznetsovaauthor=F.+Liauthor=E.+Lima-Fernandesauthor=S.+Kennedyauthor=I.+Chauauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=Kinetic+characterization+of+human+histone+H3+lysine+36+methyltransferases%2C+ASH1L+and+SETD2&doi=10.1016%2Fj.bbagen.2015.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span></div><div class="casAuthors">Eram, Mohammad S.; Kuznetsova, Ekaterina; Li, Fengling; Lima-Fernandes, Evelyne; Kennedy, Steven; Chau, Irene; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dysregulation of methylation of lysine 36 on histone H3 (H3K36) have been implicated in a variety of diseases including cancers.  ASH1L and SETD2 are two enzymes among others that catalyze H3K36 methylation.  H3K4 methylation has also been reported for ASH1L.  Radioactivity-based enzyme assays, Western and immunoblotting using specific antibodies and mol. modeling were used to characterize substrate specificity of ASH1L and SETD2.  Here we report on the assay development and kinetic characterization of ASH1L and SETD2 and their substrate specificities in vitro.  Both enzymes were active with recombinant nucleosome as substrate.  However, SETD2 but not ASH1L methylated histone peptides as well, indicating that the interaction of the basic post-SET extension with substrate may not be crit. for SETD2 activity.  Neither enzyme was active with nucleosome contg. a H3K36A mutation, indicating their specificity for H3K36.  Analyzing the methylation state of the products of ASH1L and SETD2 reactions also confirmed that both enzymes mono- and dimethylate H3K36 and are inactive with H3K4 as substrate, and that only SETD2 is able to trimethylate H3K36 in vitro.  We detd. the kinetic parameters for ASH1L and SETD2 activity, enabling screening for inhibitors that can be used to further investigate the roles of these two proteins in health and disease.  Both ASH1L and SETD2 are H3K36-specific methyltransferases but only SETD2 can trimethylate this mark.  The basic post-SET extension is crit. for ASH1L but not SETD2 activity.  We provide full kinetic characterization of ASH1L and SETD2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproBY7GkmOLrVg90H21EOLACvtfcHk0ljNp9DjEX3pXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D&md5=64839475d5ee9cc2f20e6e71f8e8a6ea</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DKinetic%2520characterization%2520of%2520human%2520histone%2520H3%2520lysine%252036%2520methyltransferases%252C%2520ASH1L%2520and%2520SETD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bbagen.2015.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.-P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.-C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+S.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=R%29-PFI-2+is+a+potent+and+selective+inhibitor+of+SETD7+methyltransferase+activity+in+cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DR%2529-PFI-2%2520is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520SETD7%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gräslund, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">A basic post-SET extension of NSDs is essential for nucleosome binding in vitro</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">928</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1177/1087057114525854</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1177%2F1087057114525854" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=24595546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWgs7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=928-935&author=A.+Allali-Hassaniauthor=E.+Kuznetsovaauthor=T.+Hajianauthor=H.+Wuauthor=L.+Dombrovskiauthor=Y.+Liauthor=S.+Gr%C3%A4slundauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=A+basic+post-SET+extension+of+NSDs+is+essential+for+nucleosome+binding+in+vitro&doi=10.1177%2F1087057114525854"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">A basic post-SET extension of nsds is essential for nucleosome binding in vitro</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Kuznetsova, Ekaterina; Hajian, Taraneh; Wu, Hong; Dombrovski, Ludmila; Li, Yanjun; Graslund, Susanne; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">928-935, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The nuclear receptor SET domain-contg. family of proteins (NSD1, NSD2, and NSD3) is known to mono- and di-methylate lysine 36 of histone H3 (H3K36).  Overexpression and translocation of NSDs have been widely implicated in a variety of diseases including cancers.  Although the substrate specificity of NSDs has been a subject of many valuable studies, the activity of these proteins has never been fully characterized in vitro.  In this study, we present full kinetic characterization of NSD1, NSD2, and NSD3 and provide robust in vitro assays suitable for screening these proteins in a 384-well format using nucleosome as a substrate.  Through monitoring the changes in substrate specificity of a series of NSD constructs and using mol. modeling, we show that a basic post-SET extension common to all three NSDs (corresponding to residues 1209 to 1226 of NSD2) is essential for proper positioning on nucleosome substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWv4LcXsVmILVg90H21EOLACvtfcHk0ljkZu2pR079Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWgs7jE&md5=d27aad340f0e2fac6cb2d21d322e92fe</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1177%2F1087057114525854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057114525854%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DGr%25C3%25A4slund%26aufirst%3DS.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520basic%2520post-SET%2520extension%2520of%2520NSDs%2520is%2520essential%2520for%2520nucleosome%2520binding%2520in%2520vitro%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D928%26epage%3D935%26doi%3D10.1177%2F1087057114525854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szykowska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess-Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span> </span><span class="NLM_article-title">A scintillation proximity assay for histone demethylases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">463</span><span class="NLM_x">, </span> <span class="NLM_fpage">54</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.ab.2014.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1016%2Fj.ab.2014.06.023" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2014&pages=54-60&author=W.+Yuauthor=M.+S.+Eramauthor=T.+Hajianauthor=A.+Szykowskaauthor=N.+Burgess-Brownauthor=M.+Vedadiauthor=P.+J.+Brown&title=A+scintillation+proximity+assay+for+histone+demethylases&doi=10.1016%2Fj.ab.2014.06.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2014.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2014.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DA%2520scintillation%2520proximity%2520assay%2520for%2520histone%2520demethylases%26jtitle%3DAnal.%2520Biochem.%26date%3D2014%26volume%3D463%26spage%3D54%26epage%3D60%26doi%3D10.1016%2Fj.ab.2014.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Grebien, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giambruno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avellino, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skucha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V. M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schonegger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilban, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerlov, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the Wdr5-MLL interaction in C/EBP[alpha] N-terminal leukemia</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&issue=11&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+M+Huberauthor=A.+Schoneggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+targeting+of+the+Wdr5-MLL+interaction+in+C%2FEBP%5Balpha%5D+N-terminal+leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0ljkZu2pR079Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DSchonegger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520Wdr5-MLL%2520interaction%2520in%2520C%252FEBP%255Balpha%255D%2520N-terminal%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niesen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finerty, P. J.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ball, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hui, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigelt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span> </span><span class="NLM_article-title">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">15835</span><span class="NLM_x">–</span> <span class="NLM_lpage">15840</span><span class="refDoi"> DOI: 10.1073/pnas.0605224103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=10.1073%2Fpnas.0605224103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=17035505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=15835-15840&author=M.+Vedadiauthor=F.+H.+Niesenauthor=A.+Allali-Hassaniauthor=O.+Y.+Fedorovauthor=P.+J.+Finertyauthor=G.+A.+Wasneyauthor=R.+Yeungauthor=C.+Arrowsmithauthor=L.+J.+Ballauthor=H.+Berglundauthor=R.+Huiauthor=B.+D.+Marsdenauthor=P.+Nordlundauthor=M.+Sundstromauthor=J.+Weigeltauthor=A.+M.+Edwards&title=Chemical+screening+methods+to+identify+ligands+that+promote+protein+stability%2C+protein+crystallization%2C+and+structure+determination&doi=10.1073%2Fpnas.0605224103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical screening methods to identify ligands that promote protein stability, protein crystallization, and structure determination</span></div><div class="casAuthors">Vedadi, Masoud; Niesen, Frank H.; Allali-Hassani, Abdellah; Fedorov, Oleg Y.; Finerty, Patrick J., Jr.; Wasney, Gregory A.; Yeung, Ron; Arrowsmith, Cheryl; Ball, Linda J.; Berglund, Helena; Hui, Raymond; Marsden, Brian D.; Nordlund, Paer; Sundstrom, Michael; Weigelt, Johan; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15835-15840</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 3D structures of human therapeutic targets are enabling for drug discovery.  However, their purifn. and crystn. remain rate detg.  In individual cases, ligands have been used to increase the success rate of protein purifn. and crystn., but the broad applicability of this approach is unknown.  The authors implemented two screening platforms, based on either fluorometry or static light scattering, to measure the increase in protein thermal stability upon binding of a ligand without the need to monitor enzyme activity.  In total, 221 different proteins from humans and human parasites were screened against one or both of two sorts of small-mol. libraries.  The first library comprised different salts, pH conditions, and commonly found small mols. and was applicable to all proteins.  The second comprised compds. specific for protein families of particular interest (e.g., protein kinases).  In 20 cases, including nine unique human protein kinases, a small mol. was identified that stabilized the proteins and promoted structure detn.  The methods are cost-effective, can be implemented in any lab., promise to increase the success rates of purifying and crystg. human proteins significantly, and identify new ligands for these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4duhcFpFn-bVg90H21EOLACvtfcHk0liVxQc3UfyY3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFymur%252FN&md5=6515225c1613291deb7bb774d900f605</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0605224103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0605224103%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%2BY.%26aulast%3DFinerty%26aufirst%3DP.%2BJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DYeung%26aufirst%3DR.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBall%26aufirst%3DL.%2BJ.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DWeigelt%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DChemical%2520screening%2520methods%2520to%2520identify%2520ligands%2520that%2520promote%2520protein%2520stability%252C%2520protein%2520crystallization%252C%2520and%2520structure%2520determination%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D15835%26epage%3D15840%26doi%3D10.1073%2Fpnas.0605224103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS','PDB','5EGS'); return false;">PDB: 5EGS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01033">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_77132"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01033">10.1021/acs.jmedchem.6b01033</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs,org">http://pubs.acs,org</a>. <sup>1</sup>H and <sup>13</sup>C NMR spectra of compound <b>17</b>; selectivity of <b>17</b> against KDMs and methyllysine and methylarginine reader proteins, CEREP, and PDS panels; Synthetic scheme of <b>46</b>; effect of cellular assay results of <b>46</b>; rffects of <b>17</b> and <b>46</b> on cell growth (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_001.pdf">jm6b01033_si_001.pdf (2.39 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01033/suppl_file/jm6b01033_si_002.csv">jm6b01033_si_002.csv (2.77 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-19%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01033" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01033" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992bff3cf83cfa","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
