
<html lang="en"     class="pb-page"  data-request-id="a2ba3536-2561-4a79-a508-1489ad89bdb7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b00017;website:website:acspubs;issue:issue:10.1021/jmcmar.2017.60.issue-9;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors" /></meta><meta name="dc.Creator" content="Le  Wang" /></meta><meta name="dc.Creator" content="John K.  Pratt" /></meta><meta name="dc.Creator" content="Todd  Soltwedel" /></meta><meta name="dc.Creator" content="George S.  Sheppard" /></meta><meta name="dc.Creator" content="Steven D.  Fidanze" /></meta><meta name="dc.Creator" content="Dachun  Liu" /></meta><meta name="dc.Creator" content="Lisa A.  Hasvold" /></meta><meta name="dc.Creator" content="Robert A.  Mantei" /></meta><meta name="dc.Creator" content="James H.  Holms" /></meta><meta name="dc.Creator" content="William J.  McClellan" /></meta><meta name="dc.Creator" content="Michael D.  Wendt" /></meta><meta name="dc.Creator" content="Carol  Wada" /></meta><meta name="dc.Creator" content="Robin  Frey" /></meta><meta name="dc.Creator" content="T. Matthew  Hansen" /></meta><meta name="dc.Creator" content="Robert  Hubbard" /></meta><meta name="dc.Creator" content="Chang H.  Park" /></meta><meta name="dc.Creator" content="Leiming  Li" /></meta><meta name="dc.Creator" content="Terrance J.  Magoc" /></meta><meta name="dc.Creator" content="Daniel H.  Albert" /></meta><meta name="dc.Creator" content="Xiaoyu  Lin" /></meta><meta name="dc.Creator" content="Scott E.  Warder" /></meta><meta name="dc.Creator" content="Peter  Kovar" /></meta><meta name="dc.Creator" content="Xiaoli  Huang" /></meta><meta name="dc.Creator" content="Denise  Wilcox" /></meta><meta name="dc.Creator" content="Rongqi  Wang" /></meta><meta name="dc.Creator" content="Ganesh  Rajaraman" /></meta><meta name="dc.Creator" content="Andrew M.  Petros" /></meta><meta name="dc.Creator" content="Charles W.  Hutchins" /></meta><meta name="dc.Creator" content="Sanjay C.  Panchal" /></meta><meta name="dc.Creator" content="Chaohong  Sun" /></meta><meta name="dc.Creator" content="Steven W.  Elmore" /></meta><meta name="dc.Creator" content="Yu  Shen" /></meta><meta name="dc.Creator" content="Warren M.  Kati" /></meta><meta name="dc.Creator" content="Keith F.  McDaniel" /></meta><meta name="dc.Description" content="Members of the BET family of bromodomain containing proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines. A two-dimensional NMR fragment s..." /></meta><meta name="Description" content="Members of the BET family of bromodomain containing proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines. A two-dimensional NMR fragment s..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 21, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00017" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00017" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00017" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00017" /></link>
        
    
    

<title>Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00017" /></meta><meta property="og:title" content="Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0023.jpeg" /></meta><meta property="og:description" content="Members of the BET family of bromodomain containing proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines. A two-dimensional NMR fragment screen for binders to the bromodomains of BRD4 identified a phenylpyridazinone fragment with a weak binding affinity (1, Ki = 160 μM). SAR investigation of fragment 1, aided by X-ray structure-based design, enabled the synthesis of potent pyridone and macrocyclic pyridone inhibitors exhibiting single digit nanomolar potency in both biochemical and cell based assays. Advanced analogs in these series exhibited high oral exposures in rodent PK studies and demonstrated significant tumor growth inhibition efficacy in mouse flank xenograft models." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00017"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00017">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00017&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00017&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00017&amp;href=/doi/10.1021/acs.jmedchem.7b00017" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 3828-3850</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00018" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00113" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Le++Wang">Le Wang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+K.++Pratt">John K. Pratt</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol "><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-2611-8480" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Todd++Soltwedel">Todd Soltwedel</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=George+S.++Sheppard">George S. Sheppard</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven+D.++Fidanze">Steven D. Fidanze</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dachun++Liu">Dachun Liu</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lisa+A.++Hasvold">Lisa A. Hasvold</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+A.++Mantei">Robert A. Mantei</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=James+H.++Holms">James H. Holms</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=William+J.++McClellan">William J. McClellan</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+D.++Wendt">Michael D. Wendt</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Carol++Wada">Carol Wada</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robin++Frey">Robin Frey</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=T.+Matthew++Hansen">T. Matthew Hansen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert++Hubbard">Robert Hubbard</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chang+H.++Park">Chang H. Park</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Leiming++Li">Leiming Li</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Terrance+J.++Magoc">Terrance J. Magoc</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+H.++Albert">Daniel H. Albert</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyu++Lin">Xiaoyu Lin</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Scott+E.++Warder">Scott E. Warder</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Kovar">Peter Kovar</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoli++Huang">Xiaoli Huang</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Denise++Wilcox">Denise Wilcox</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rongqi++Wang">Rongqi Wang</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ganesh++Rajaraman">Ganesh Rajaraman</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+M.++Petros">Andrew M. Petros</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Charles+W.++Hutchins">Charles W. Hutchins</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sanjay+C.++Panchal">Sanjay C. Panchal</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chaohong++Sun">Chaohong Sun</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven+W.++Elmore">Steven W. Elmore</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Shen">Yu Shen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Warren+M.++Kati">Warren M. Kati</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Keith+F.++McDaniel">Keith F. McDaniel</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div><div class="corresp-info"><strong>*</strong>L.W.: e-mail, <a href="/cdn-cgi/l/email-protection#573b32792036393017363535213e327934383a"><span class="__cf_email__" data-cfemail="dcb0b9f2abbdb2bb9cbdbebeaab5b9f2bfb3b1">[email protected]</span></a>; phone, 1-847-935-4952.</div><div class="corresp-info"><strong>*</strong>J.K.P.: e-mail, <a href="/cdn-cgi/l/email-protection#cda7a2a5a3e3a6e3bdbfacb9b98dacafafbba4a8e3aea2a0"><span class="__cf_email__" data-cfemail="563c393e38783d78262437222216373434203f337835393b">[email protected]</span></a>; phone, 1-847-937-0432.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00017&amp;href=/doi/10.1021%2Facs.jmedchem.7b00017" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 3828–3850</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 3, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 January 2017</li><li><span class="item_label"><b>Published</b> online</span>21 April 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 May 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00017" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00017</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editors’ Choice" /></img><span>ACS Editors’ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00017"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">6741</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">34</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00017" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Le&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;K. Pratt&quot;},{&quot;first_name&quot;:&quot;Todd&quot;,&quot;last_name&quot;:&quot;Soltwedel&quot;},{&quot;first_name&quot;:&quot;George&quot;,&quot;last_name&quot;:&quot;S. Sheppard&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;D. Fidanze&quot;},{&quot;first_name&quot;:&quot;Dachun&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;A. Hasvold&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;A. Mantei&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;H. Holms&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;J. McClellan&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;D. Wendt&quot;},{&quot;first_name&quot;:&quot;Carol&quot;,&quot;last_name&quot;:&quot;Wada&quot;},{&quot;first_name&quot;:&quot;Robin&quot;,&quot;last_name&quot;:&quot;Frey&quot;},{&quot;first_name&quot;:&quot;T.&quot;,&quot;last_name&quot;:&quot;Matthew Hansen&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;Hubbard&quot;},{&quot;first_name&quot;:&quot;Chang&quot;,&quot;last_name&quot;:&quot;H. Park&quot;},{&quot;first_name&quot;:&quot;Leiming&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Terrance&quot;,&quot;last_name&quot;:&quot;J. Magoc&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;H. Albert&quot;},{&quot;first_name&quot;:&quot;Xiaoyu&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;E. Warder&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Kovar&quot;},{&quot;first_name&quot;:&quot;Xiaoli&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Denise&quot;,&quot;last_name&quot;:&quot;Wilcox&quot;},{&quot;first_name&quot;:&quot;Rongqi&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ganesh&quot;,&quot;last_name&quot;:&quot;Rajaraman&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;M. Petros&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;W. Hutchins&quot;},{&quot;first_name&quot;:&quot;Sanjay&quot;,&quot;last_name&quot;:&quot;C. Panchal&quot;},{&quot;first_name&quot;:&quot;Chaohong&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;W. Elmore&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Warren&quot;,&quot;last_name&quot;:&quot;M. Kati&quot;},{&quot;first_name&quot;:&quot;Keith&quot;,&quot;last_name&quot;:&quot;F. McDaniel&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;3828-3850&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00017&quot;},&quot;abstract&quot;:&quot;Members of the BET family of bromodomain containing proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines. A two-dimensional NMR fragment screen for binders to the bromodomains of BRD4 identified a phenylpyridazinone fragment with a weak binding affinity (1, Ki = 160 μM). SAR investigation of fragment 1, aided by X-ray structure-based design, enabled the synthesis of potent pyridone and macrocyclic pyridone inhibitors exhibiting single digit nanomolar potency in both biochemical and cell based assays. Advanced analogs in these series exhibited high oral exposures in rodent PK studies and demonstrated significant tumor growth inhibition efficacy in mouse flank xenograft models.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00017&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00017" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00017&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00017" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00017&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00017" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00017&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00017&amp;href=/doi/10.1021/acs.jmedchem.7b00017" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00017" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00017" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00017%26sid%3Dliteratum%253Aachs%26pmid%3D28368119%26genre%3Darticle%26aulast%3DWang%26date%3D2017%26atitle%3DFragment-Based%252C%2BStructure-Enabled%2BDiscovery%2Bof%2BNovel%2BPyridones%2Band%2BPyridone%2BMacrocycles%2Bas%2BPotent%2BBromodomain%2Band%2BExtra-Terminal%2BDomain%2B%2528BET%2529%2BFamily%2BBromodomain%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D9%26spage%3D3828%26epage%3D3850%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291285" title="Ethers">Ethers</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/jmcmar.2017.60.issue-9/20170511/jmcmar.2017.60.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Members of the BET family of bromodomain containing proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines. A two-dimensional NMR fragment screen for binders to the bromodomains of BRD4 identified a phenylpyridazinone fragment with a weak binding affinity (<b>1</b>, <i>K</i><sub>i</sub> = 160 μM). SAR investigation of fragment <b>1</b>, aided by X-ray structure-based design, enabled the synthesis of potent pyridone and macrocyclic pyridone inhibitors exhibiting single digit nanomolar potency in both biochemical and cell based assays. Advanced analogs in these series exhibited high oral exposures in rodent PK studies and demonstrated significant tumor growth inhibition efficacy in mouse flank xenograft models.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Reversible lysine acetylation plays a central role in the regulation of chromatin structure and transcription of genes via modification of histone proteins and transcription factors.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> These changes in gene expression are modulated by three kinds of epigenetic regulatory proteins, “writers”, “erasers”, and “readers”. Histone acetyltransferases (HATs) function as “writers” to acetylate lysine. Histone deacetylases (HDACs) remove the acetyl group from acetylated lysine, acting as “erasers”.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> The third kind of epigenetic regulatory protein is the bromodomain (BRD) family of proteins that selectively bind to acetylated lysines,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> thus functioning as “readers” of the lysine acetylation state.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> Bromodomains consist of approximately 110 amino acid residues and can be found in 42 proteins in the human proteome. The bromodomain and extra-terminal (BET) family is a subset of related bromodomain-containing proteins that includes BRD2, BRD3, BRD4, and BRDT. All BET family members share the common configuration of two N-terminal bromodomains (BDI and BDII) and a C-terminal extra-terminal (ET) domain. The ET domain interacts with other cellular proteins that include cyclin-dependent kinase 9 (CDK9) and cyclin T to activate transcription of proximal genes into RNA. The BDI and BDII domains, respectively, are highly conserved across BET family members (>70% identity), whereas BDI and BDII from the same BET protein exhibit a larger degree of divergence (∼40% identity).</div><div class="NLM_p">The inhibition of bromodomain proteins was largely unknown until 2009 when Mitsubishi disclosed the first small molecule BET inhibitor <b>2a</b> (MS417)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. Two additional BET inhibitors, <b>2b</b> (JQ1)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and <b>2c</b> (I-BET762),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> were published in late 2010. <b>2b</b> is closely related to the molecules described in the initial Mitsubishi patent, whereas <b>2c</b> was discovered at GlaxoSmithKline (GSK) via a phenotypic screening campaign. All three compounds are potent and selective BET family protein inhibitors that were found to occupy the acetyl lysine (KAc) binding site,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> a well-defined pocket in the BET bromodomains. RVX-208 from Resverlogix Corp, a molecule in phase II clinical trials for cardiovascular disease, was also found to be a BET inhibitor, albeit with rather weak activity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> On the basis of these early BET inhibitors, rapid and significant progress has been made in understanding the role of BET proteins in multiple disease areas including oncology,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> inflammation,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> immunology,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and cardiovascular disease.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> From an oncology standpoint, the mechanisms-of-action for BET inhibitors fall into three broad categories: (1) disruption of cell cycle control leading to G1 arrest, (2) inhibition of genetic drivers of oncogenesis leading to apoptosis, and (3) inhibition of cytokine/chemokine signaling pathways that maintain a tumor-supportive microenvironment.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> From an immunology disease standpoint, the inhibition of NF-kB driven pathways contributes, at least in part, to anti-inflammatory properties exhibited by BET inhibitors.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> For example, the BET inhibitor <b>2d</b> (I-BET151) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) has been shown to down-regulate expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0001.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative BET inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The impressive biological activities of <b>2b</b>–<b>d</b> in a variety of therapeutic areas spurred the pharmaceutical industry to develop clinical quality BET inhibitors.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> The triazole diazepine binding motif found in <b>2b</b> is contained in clinical compounds developed by GSK (<b>2c</b>), OncoEthix/Merck (OTX-015/MK8628),<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Tensha/Roche (TEN010),<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and Constellation (CPI-610).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The 3,5-dimethylisoxazole motif, a KAc binding mimic, was first published by GSK as shown in <b>2d</b>.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Later, this moiety appeared in a number of BET inhibitor patent applications filed by several companies such as Incyte,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Gilead,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and Boehringer-Ingelheim.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In addition, GSK published a tetrahydroquinoline based BET inhibitor, THQ,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> while Pfizer disclosed its tool BET inhibitor PFI-1.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> In search of novel BET inhibitor chemotypes, we initiated a protein-based NMR fragment screen<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> carried out against the second bromodomain of BRD4 (BRD4-BDII). This fragment screen yielded a number of structurally distinct hits, including compound <b>1</b>, a novel small molecule that had not been previously reported to bind to bromodomains. Compound <b>1</b> showed a strong spectral shift in this NMR screen (see <a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information</a>). Subsequent testing revealed that compound <b>1</b> was a weak BRD4 inhibitor with a <i>K</i><sub>i</sub> of 160 μM, and its target binding was confirmed by a protein–inhibitor X-ray cocrystal structure. Medicinal chemistry efforts directed toward the generation of BET bromodomain inhibitors based on unique fragment <b>1</b> are reported in this publication. The design, syntheses, and SAR investigation of novel pyridones/pyridazinones and pyridone macrocycles as potent BET inhibitors will be discussed. The pharmacokinetics and in vivo antitumor activity of selected compounds will also be presented.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63005" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63005" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The general methodology for the syntheses of the pyridazinone and pyridone BET inhibitors described for this publication utilized a convergent synthetic approach. A key transformation in many of the synthetic sequences to prepare the acyclic inhibitors utilized a Suzuki coupling reaction, allowing flexibility in the choice of boronate/halide coupling partners.</div><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, methylation of commercially available <b>3</b> followed by nucleophilic displacement of the more reactive of the two aryl chlorides with sodium methoxide provided intermediate <b>5</b>. The aryl boronate <b>10</b> was prepared from commercially available 2-bromo-1-fluoro-4-nitrobenzene <b>6</b> in three steps. Nucleophilic aromatic substitution of the aryl fluoride of <b>6</b> with phenol using cesium carbonate in DMSO at elevated temperature provided <b>7</b>. Iron reduction of <b>7</b> in the presence of ammonium chloride provided aniline <b>8</b>. Conversion of the aryl bromide to the boron pinacolate ester gave Suzuki coupling partner <b>10</b>. Suzuki coupling of <b>5</b> with either commercially available boronate <b>9</b> or <b>10</b> provided <b>11</b> and <b>12</b>, respectively. Either Pd(PPh<sub>3</sub>)<sub>4</sub> in the presence of cesium fluoride or the combination of tris(dibenzylideneacetone)dipalladium(0) as the palladium catalyst and 1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane as the phosphine ligand was employed for the Suzuki coupling reactions. Aniline <b>12</b> was then treated with various electrophilic reagents to obtain final derivatives <b>13a</b>–<b>d</b>. Sulfonylation of anilines such as <b>12</b> often provided bis-sulfonylated intermediates. Removal of one sulfonyl group with aqueous hydroxide at elevated temperature gave the desired monosulfonylated products in good yield.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) MeI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 91%; (ii) Na, MeOH, 50 °C, 89%; (iii) phenol, Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 100 °C, 95%; (iv) Fe<sup>0</sup>, NH<sub>4</sub>Cl, EtOH/THF/water, 95 °C, 97%; (v) 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane), KOAc, Pd<sub>2</sub>(dba)<sub>3</sub>, (1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane), 1,4-dioxane, 80 °C, 70%; (vi) <b>5</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, CsF, DME/MeOH, 120 °C, 57–62%; (vii) (<b>13a</b>) AcCl, Et<sub>3</sub>N, DCM, 99%; (<b>13b</b>) MsCl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 68%; (<b>13c</b>) ClCOOEt, Et<sub>3</sub>N, DCM, 81%; (<b>13d</b>) EtNCO, Et<sub>3</sub>N, DCM, 51%.</p></p></figure><div class="NLM_p">Synthesis of the acyclic pyridone core (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) was very similar in nature to that described for the pyridazinone core. Diazotization of commercially available 5-bromo-4-chloropyridin-2-amine <b>14</b> followed by N-methylation gave the dihalogenated pyridone <b>16</b>. Nucleophilic displacement of the pyridone chloride with either methoxide or ethoxide provided compounds <b>17a</b> and <b>17b</b>. Suzuki couplings using boronate <b>10</b> followed by sulfonylation of the resulting aniline intermediates provided analogs <b>20a</b>, <b>20b</b>, and <b>21</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) NaNO<sub>2</sub>, H<sub>2</sub>SO<sub>4</sub>, NH<sub>4</sub>OH, water, 0 °C, quant; (ii) MeI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 91%; (iii) NaOMe in MeOH, or NaOEt in EtOH, 60 °C, 82–89%; (iv) <b>9</b> or <b>10</b> Pd(PPh<sub>3</sub>)<sub>4</sub>, CsF, 48–65%; (v) MsCl or ethanesulfonyl chloride, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 35–56%.</p></p></figure><div class="NLM_p"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Scheme S1 (Supporting Information)</a> is an expansion of <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> highlighting chemistry used to explore the SAR of the aryl ether region of the inhibitors (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_p">Exploration of various ethers incorporated on the pyridone core is shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Nucleophilic displacement of the fluorine of compound <b>6</b> followed by boronate ester formation provided <b>27</b>. Suzuki coupling gave <b>28</b> which was subsequently sulfonylated providing the common intermediate 4-chloropyridone <b>29</b>. Nucleophilic displacement of the chloride with commercially available or prepared alkoxides gave the final aliphatic ether derivatives <b>30a</b>–<b>f</b>.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 2,4-difluorophenol, Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C, quant; (ii) Fe<sup>0</sup>, NH<sub>4</sub>Cl, EtOH/THF/water, 100 °C, 82%; (iii) 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane), KOAc, Pd<sub>2</sub>(dba)<sub>3</sub>, (1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane), 1,4-dioxane, 80 °C, quant; (iv) <b>16</b>, K<sub>3</sub>PO<sub>4</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, (1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane), 1,4-dioxane/water, 60 °C, 86%; (v) ethanesulfonyl chloride, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 81%; (vi) NaOMe in MeOH, 60 °C for <b>30a</b>; NaH, ROH, 1,4-dioxane, 70 °C for <b>30b</b>–<b>f</b>, 11–87%.</p></p></figure><div class="NLM_p"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Schemes S2–S5 in Supporting Information</a> illustrate synthetic methods that were used to prepare individual analogs found in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, generally following the routes described above in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch3" aria-label="3">3</a>.</div><div class="NLM_p">One of the challenges associated with the exploration of macrocyclic frameworks for drug discovery is the difficulty in synthesizing such structures.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Several cyclization strategies employing the S<sub>N</sub>Ar displacement reaction developed for the synthesis of acyclic pyridone BET inhibitors were utilized successfully to generate macrocycles. In addition, two new macrocyclization routes using intramolecular Heck<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and Mitsunobu<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> reactions were also employed with modest to good yields. Syntheses of macrocycles <b>60a</b>–<b>c</b> and <b>61a</b>,<b>b</b> are described in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The key transformation was an intramolecular S<sub>N</sub>Ar displacement to form the macrocycle. Suzuki coupling between compound <b>16</b> and compound <b>6a</b> yielded compound <b>56</b>. Compounds <b>53</b> and <b>56</b> were reacted with various 2-iodophenols via nucleophilic displacement to give compounds <b>57a</b>–<b>d</b> in good yields. Sonogashira coupling between compounds <b>57a</b>–<b>d</b> and selected alkynyl alcohols gave compounds <b>58a</b>–<b>e</b> in excellent yields. Alkynes <b>58a</b>–<b>e</b> were then reduced in the presence of Pt and hydrogen to give compounds <b>59a</b>–<b>e</b>, which were subjected to intramolecular cyclization using sodium hydride to afford macrocycles <b>60a</b>–<b>e</b>. It is worth pointing out that this cyclization reaction usually gave modest yields with rather limited functional compatibility. For the larger ring sizes, such as a 13-membered ring, the cyclization failed to give the desired product. Compounds <b>60d</b> and <b>60e</b> were allowed to react with ethylsulfonyl chloride followed by treatment with aqueous sodium hydroxide to provide compounds <b>61a</b>,<b>b</b>.</div><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) <b>6a</b> or <b>52</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, CsF, THF, tri-<i>tert</i>-butylphosphonium tetrafluoroborate, 45 °C, 79%. (ii) (a) <b>53</b>: phenols, Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 100 °C, 6–62%. (b) <b>56</b>: phenols, Cs<sub>2</sub>CO<sub>3</sub>, 70–85 °C, DMSO, 69–75%. (iii) Et<sub>3</sub>N, CuI, but-3-yn-1-ol or prop-2-yn-1-ol, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, DMF, 47–100%; (iv) Pt, H<sub>2</sub>, THF, 80–95%; (v) NaH, 1,4-dioxane, 90 °C, 12–48%; (vi) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 58–81%.</p></p></figure><div class="NLM_p">Although the intramolecular displacement successfully yielded the 11- and 12-membered macrocycles described for <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, this strategy failed to form the 13-membered ring found in compound <b>66</b>. After several cyclization attempts, the intramolecular Heck reaction was found to provide a successful approach to this 13-membered ring, as described in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The nitro group of compound <b>57d</b> was reduced to the aniline <b>62</b>, which was then converted to the ethylsulfonamide <b>63</b>. In order to generate the iodoalkene <b>64</b> required for macrocycle formation, the chloride of compound <b>63</b> was displaced by the hydroxyl of pent-4-en-1-ol. The macrocycle formation was accomplished by an intramolecular Heck reaction between the terminal double bond and the iodide, catalyzed by Pd(PPh<sub>3</sub>)<sub>4</sub> and triethylamine in DMF, to give compounds <b>65a</b> and <b>65b</b> in roughly equal amounts. Hydrogenation of compound <b>65a</b> resulted in formation of compound <b>66</b> in excellent yield. This intramolecular Heck reaction was also applied to the formation of certain 12-membered ring analogs, such as <b>61b,</b> in reasonable yield.</div><figure id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0013.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Fe<sup>0</sup>, NH<sub>4</sub>Cl, THF/EtOH/water, 95 °C, 90%; (ii) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM then aq NaOH/1,4-dioxane, 90 °C, 80%; (iii) NaH, pent-4-en-1-ol, 1,4-dioxane, 90 °C, 75%; (iv) Pd(PPh<sub>3</sub>)<sub>4</sub>, Et<sub>3</sub>N, DMF, 100 °C. <b>65a</b>: 29%. <b>65b</b>: 30%. (v) Pt, H<sub>2</sub>, THF, 85%.</p></p></figure><div class="NLM_p">Although 12-membered ring macrocycles could be generated by intramolecular displacement of the chloropyridones <b>59</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>), the yields were modest at best. Because a strong base such as sodium hydride was required, functional group tolerability was limited. Therefore, an alternative route that relied on an intramolecular S<sub>N</sub>Ar reaction to form the biaryl ether moiety was developed for the preparation of the 12-membered macrocycles (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). Sonogashira coupling of compounds <b>67a</b>–<b>c</b> followed by hydrogenation resulted in compounds <b>69a</b>–<b>c</b>. Pyridone <b>70</b> was prepared in good yield by reaction of compound <b>16</b> and compound <b>69b</b>. Palladium-catalyzed borylation of compound <b>70</b> resulted in the formation of boronate ester <b>71</b>, which was allowed to react with three 3-bromo-2-chloro-5-substituted-pyridines (<b>77a</b>–<b>c</b>) to give pyridines <b>72a</b>–<b>c</b>. The required phenols <b>73a</b>–<b>c</b> were generated by reaction of compounds <b>72a</b>–<b>c</b> with boron tribromide. Intramolecular nucleophilic substitution mediated by Cs<sub>2</sub>CO<sub>3</sub> in acetonitrile successfully converted compounds <b>73a</b>–<b>c</b> to the macrocycles <b>74a</b>–<b>c</b> in excellent yields. Compound <b>74a</b> was then reduced to aniline <b>75</b> and converted to ethylsulfonamide <b>76</b>. Macrocycle <b>86</b> was prepared by an analogous route, as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information (Scheme S7)</a>.</div><figure id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0014.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Et<sub>3</sub>N, CuI, but-3-yn-1-ol, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, DMF, 83–96%; (ii) Pd/C, H<sub>2</sub>, THF, 68–100%; (iii) <b>16</b>, KO<sup><i>t</i></sup>Bu, 1,4-dioxane, 75%; (iv) 4,4,4′,4′,5,5,5′,5′-octamethyl -2,2′-bi(1,3,2-dioxaborolane), KOAc, dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 80 °C, 61%. (v) for <b>72a</b>: <b>77a</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, K<sub>3</sub>PO<sub>4</sub>, 1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane, 1,4-dioxane, water, 50 °C, 32%. For <b>72b</b> and <b>72c</b>: <b>77b</b> or <b>77c</b>, respectively, CsF, Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>tert</i>-butylphosphonium tetrafluoroborate, THF, 50 °C, 49–70%; (vi) BBr<sub>3</sub>, DCM, −78 °C to rt, 83–88%; (vii) Cs<sub>2</sub>CO<sub>3</sub>, MeCN, 75–95 °C, overnight, 77–90%; (viii) Fe<sup>0</sup>, NH<sub>4</sub>Cl, THF/EtOH/water, 90 °C, 94%; (ix) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 85 °C, 80%.</p></p></figure><div class="NLM_p">Synthesis of macrocycle <b>83</b> is shown in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>. Because nitropyridine <b>81</b> was likely to decompose under the strongly basic conditions required for displacement of a chloropyridone, as described above in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, this particular cyclization was accomplished using an intramolecular Mitsunobu reaction, albeit in modest yield. Biaryl ether <b>78</b> was prepared by the reaction between chloropyridine <b>77a</b> and phenol <b>69c</b>. Boronate <b>79</b> was prepared by displacement of the chloride of pyridone <b>16</b> by the hydroxyl group of 2,4-dimethoxybenzyl alcohol followed by palladium-catalyzed boronate formation. Suzuki coupling between bromide <b>78</b> and boronate <b>79</b> provided compound <b>80</b>. The latent hydroxyl group of pyridone <b>81</b> was unmasked by removal of the 2,4-dimethoxybenzyl group of compound <b>80</b> upon treatment with trifluoroacetic acid. Intramolecular Mitsunobu reaction of alcohol <b>81</b> in the presence of cyanomethylenetributylphosphorane in toluene gave macrocycle <b>82</b> in modest yield. Reduction of the nitro group followed by ethylsulfonamide formation resulted in compound <b>83</b>. Schemes and experimental procedures for the preparation of macrocycles <b>55</b>, <b>86</b>, and <b>94</b> can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information</a>. Macrocycles <b>55</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Scheme S6</a>) and <b>94</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Scheme S8</a>) were generated by an intramolecular Mitsunobu reaction sequence related to the approach utilized in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>.</div><figure id="sch7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0015.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 65 °C, 93%; (ii) 2,4-dimethoxybenzyl alcohol, KO<sup><i>t</i></sup>Bu, THF, 55%; (iii) KOAc, dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine, Pd<sub>2</sub>(dba)<sub>3</sub>, 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane), 1,4-dioxane, 80 °C, 60%; (iv) CsF, THF, Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>tert</i>-butylphosphonium tetrafluoroborate, 50 °C, 37%; (v) TFA, DCM, 100%; (vi) 2-(tributylphosphoranylidene)acetonitrile, toluene, 80 °C, 33%; (vii) (a) Fe<sup>0</sup>, NH<sub>4</sub>Cl, THF/EtOH/water, 90 °C, 81%; (b) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 80 °C, 67%.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87830" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87830" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A crucial component of the development of this novel series of pyridazinone- and pyridone-based BET inhibitors was the utilization of X-ray structure information to guide compound design. Although pyridazinone fragment <b>1</b> exhibited only weak binding to BRD4 (<i>K</i><sub>i</sub> = 160 μM), an X-ray cocrystal structure was solved (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a) confirming that fragment <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFO">5UFO</a>) occupied the acetylated lysine binding pocket of BRD4 BDII. Two key interactions were observed between <b>1</b> and the protein (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). The first crucial interaction is a 2.8 Å hydrogen bond between the carbonyl oxygen of the pyridazinone and the amino group of conserved Asn433 in the BET protein. The second interaction involves the <i>N</i>-methyl group of the pyridone situated in an amphoteric pocket created by a network of ordered water molecules. Together these two interactions function as an anchor for the <i>N</i>-methylpyridazinone core and served as a common binding feature upon which all subsequent SAR was based.</div><figure id="fig2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0002.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a, left panel) X-ray cocrystal structure of <b>1</b> in the binding pocket of the BRD4 BDII protein. (b, right panel) Hydrogen bonding of pyridone carbonyl oxygen to Asn433 and pyridone <i>N</i>-methyl substituent in amphoteric pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">An overlay of the crystal structures of fragment <b>1</b> and <b>2a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU">5UEU</a>) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>) confirmed that both molecules occupied the same binding site in the BRD4 BDII protein. However, fragment <b>1</b> lacked many of the important structural features of the more potent inhibitor <b>2a</b> (<i>K</i><sub>i</sub> = 16.7 nM). Analysis of the X-ray structure overlay revealed several potential vectors that might be useful for the incorporation of additional binding moieties leading to improved potency of fragment <b>1</b> analogs. The first vector was directed toward the hydrophobic WPF shelf of the protein, which is occupied by the <i>p</i>-chlorophenyl ring of <b>2a</b>. A second vector radiating away from the phenyl ring of <b>1</b> was directed toward an area of the protein with mixed hydrophilic/hydrophobic amino acid side chains. A third vector was directed toward a groove in the protein coming off of the pyridazinone ring in the same general direction as the methylamine of fragment <b>1</b>. This third vector essentially points toward the pocket occupied by the methyl ester of <b>2a</b>.</div><figure id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0003.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Overlay of X-ray cocrystal structures <b>1</b> in blue and <b>2a</b> in yellow bound to the BRD4 BDII protein. Vectors show opportunities for additional SAR exploration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Novel compounds were evaluated using a TR-FRET binding assay and two cellular assays, which are described in detail in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information</a>. A time-resolved fluorescence resonance energy transfer (TR-FRET) assay was used to determine the affinities (<i>K</i><sub>i</sub>) of compounds for the tandem bromodomain of BRD4. The impact of compounds on cancer cell proliferation was determined using the triple negative breast cancer cell line MX-1 (ATCC) in a 3-day proliferation assay. Target engagement in cells was measured with a luciferase reporter assay. A luciferase signal occurs only when a BET inhibitor blocks BRD4, thus providing a measure of target engagement. Good correlation was observed between the two cellular assays, indicating antiproliferation is due to target engagement.</div><div class="NLM_p">Efforts to improve the potency of <b>1</b> were initiated by adding a phenyl ether at the ortho position of the phenyl ring attached to the pyridazinone heterocycle. To avoid confusion in discussing these molecules, a lettering system was implemented for the three rings present in the inhibitors (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> header structure). The pyridazinone ring system was assigned the letter A. The phenyl ring attached to the pyridazinone was assigned letter B, and the newly introduced phenyl ether ring was assigned letter C. Through molecular modeling it was anticipated that the aromatic C-ring would be directed toward the WPF shelf of the protein. Simultaneous replacement of the methylamine substituent on the pyridazinone core with a methyl ether gave compound <b>11</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) These modifications improved the binding potency of <b>11</b> compared to fragment <b>1</b>, representing nearly a 170-fold improvement. Compound <b>11</b> also exhibited significant cellular activity, having an EC<sub>50</sub> of 4.3 μM in the MX-1 proliferation assay and 3 μM in the BRD4 engagement assay. The X-ray cocrystal structure of compound <b>11</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEZ">5UEZ</a>) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) clearly illustrates the phenyl ether C-ring occupying the hydrophobic WPF pocket.</div><figure id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0004.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray cocrystal structure of <b>11</b> in the BRD4 BDII protein illustrating the C-ring phenyl ether sitting in the WPF shelf pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical and Cellular Potency of Pyridazinone and Pyridone Phenyl Ether Derivatives</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0016.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center">TR-FRET</th><th class="rowsep1 colsep0" align="center" char=".">MX-1 proliferation</th><th class="rowsep1 colsep0" align="center" char=".">BRD4 engagement</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">970 ± 200</td><td class="colsep0 rowsep0" align="char" char=".">4300</td><td class="colsep0 rowsep0" align="char" char=".">3000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">1700 ± 120</td><td class="colsep0 rowsep0" align="char" char=".">7200</td><td class="colsep0 rowsep0" align="char" char=".">2900</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13a</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">-NHAc</td><td class="colsep0 rowsep0" align="left">900 ± 80</td><td class="colsep0 rowsep0" align="char" char=".">1300</td><td class="colsep0 rowsep0" align="char" char=".">370</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13b</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">-NHSO<sub>2</sub>Me</td><td class="colsep0 rowsep0" align="left">140 ± 10</td><td class="colsep0 rowsep0" align="char" char=".">730</td><td class="colsep0 rowsep0" align="char" char=".">230</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13c</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">-NHCO<sub>2</sub>Et</td><td class="colsep0 rowsep0" align="left">1100 ± 50</td><td class="colsep0 rowsep0" align="char" char=".">4500</td><td class="colsep0 rowsep0" align="char" char=".">4900</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13d</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">-NHCONHEt</td><td class="colsep0 rowsep0" align="left">1900 ± 450</td><td class="colsep0 rowsep0" align="char" char=".">13000</td><td class="colsep0 rowsep0" align="char" char=".">2800</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18a</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="left">450 ± 30 (2)</td><td class="colsep0 rowsep0" align="char" char=".">1500</td><td class="colsep0 rowsep0" align="char" char=".">2800</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20a</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">-NHSO<sub>2</sub>Me</td><td class="colsep0 rowsep0" align="left">58 ± 3</td><td class="colsep0 rowsep0" align="char" char=".">130</td><td class="colsep0 rowsep0" align="char" char=".">77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">-NHSO<sub>2</sub>Et</td><td class="colsep0 rowsep0" align="left">8.2 ± 0.5</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">67</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">TR-FRET BRD4 <i>K</i><sub>i</sub> values are reported as the geometric mean derived from three independent measurements except where indicated.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">EC<sub>50</sub> values are reported as the mean derived from two measurements.</p></div></div></div><div class="NLM_p">By use of the biochemical and cellular potencies of <b>11</b> as a benchmark, polar substituents were explored para to the phenyl ether to reach the aforementioned mixed hydrophilic/hydrophobic region of the BRD4 protein (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Aniline <b>12</b> was slightly weaker than <b>11</b> in the biochemical and MX-1 proliferation assays and equipotent in the BRD4 engagement cellular assay. Acylation of the aniline to give compound <b>13a</b> resulted in a small improvement in MX-1 cellular potency and a larger potency increase in the cellular proliferation assay. Methyl sulfonylation, <b>13b</b>, resulted in a 7-fold potency improvement over compound <b>11</b> in the biochemical assay (140 nM) and a 10-fold improvement in the cellular assays, with an EC<sub>50</sub> of 730 nM in the MX-1 proliferation assay and 230 nM in the BRD4 engagement assay. The ethyl carbamate <b>13c</b> and the ethyl urea <b>13d</b> did not show potency improvements when compared to <b>11</b>.</div><div class="NLM_p">A modification of the pyridazinone core of <b>11</b> to the pyridone core homolog<b>18a</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) provided a slight improvement in the biochemical and MX-1 proliferation assays but no improvement in the BRD4 engagement assay. Similar to what was observed in the pyridazinone series, potency of the pyridone series was improved through methyl sulfonylation (compound <b>20a</b>, <i>K</i><sub>i</sub> = 58 nM). Adding an additional carbon to form the ethylsulfonamide <b>20b</b> further improved the binding potency (<i>K</i><sub>i</sub> = 8.2 nM) and had a larger impact on MX-1 proliferation cellular potency (EC<sub>50</sub> = 16 nM). An X-ray cocrystal structure of <b>20a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEW">5UEW</a>) bound to BRD2 BDII (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>) illustrates key interactions of the inhibitor with the protein resulting in the large potency increase observed compared to compound <b>18a</b>. One of the sulfonamide oxygen atoms of <b>20a</b> replaces a water molecule in the water network surrounding the binding pocket and simultaneously forms a hydrogen bond with the backbone nitrogen of Asp377 in BRD2 BDII. The key pyridone carbonyl hydrogen bond to Asn429 and the position of the <i>N</i>-methyl group in the amphoteric pocket are maintained.</div><figure id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0005.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray cocrystal structure of <b>20a</b> bound to BRD2 BDII highlighting the sulfonamide oxygen hydrogen bond to Asp377 and the retained hydrogen bond to Asn429.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">SAR of the phenyl ether portion of the molecule was investigated next, with an emphasis on reducing the potential for oxidative metabolism of the phenyl ring while maintaining biochemical and cellular potency. By use of pyridone <b>21</b> as a starting point, derivatives <b>25a</b>–<b>e</b> with electron withdrawing groups attached at likely sites of oxidation were prepared (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Addition of these substituents had little effect on the binding affinity with the exception of the 4-cyano analog <b>25c</b> which was less potent in the biochemical assay. However, the cellular potencies of these analogs were weaker compared to <b>21</b>. The unbound clearances in human and mouse liver microsomes for <b>25a</b>–<b>e</b> were comparable to <b>21</b> with the exception of the trifluoromethoxy analog <b>25d</b>, which had much higher clearance in mouse liver microsomes. A positive attribute for this set of analogs was the dramatic improvement in rat microsomal stability for <b>25a</b>–<b>e</b> with clearance reduced to single digit values. Once again <b>25d</b> was the exception showing no improvement. The 3-pyridyl analog <b>25f</b> was weaker than <b>21</b> in both the binding and cellular assays but exhibited excellent microsomal stability in all three species. An aliphatic derivative <b>25g</b> was substantially less potent in all three assays. Compounds <b>25b</b> and <b>25e</b> demonstrated the best overall combination of potency and microsomal stability of the compounds shown. Further SAR studies based upon the 2,4-difluorophenyl ether derivative <b>25e</b> were conducted.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR and Microsomal Stability of Substituted Phenyl Ether Pyridone Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0017.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0018.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">TR-FRET BRD4 <i>K</i><sub>i</sub> values are reported as the geometric mean derived from three independent measurements except where indicated.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">EC<sub>50</sub> values are reported as the mean derived from two measurements.</p></div><div class="footnote" id="t2fn3"><sup>Table c</sup><p class="last">Cl<sub>int,u</sub>: scaled unbound intrinsic clearance [L h<sup>–1</sup> kg<sup>–1</sup>].</p></div></div><div></div></div><div class="NLM_p">Having improved the in vitro oxidative metabolism of the pyridone inhibitors, focus was shifted to the examination of the SAR of the 4-alkoxy substituent on the pyridone (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The methyl ether <b>30a</b> was slightly less potent than the ethyl ether <b>25e</b> in the biological assays. The 2,2,2-trifluoroethyl ether <b>30b</b> was less potent than <b>25e</b> in the biochemical and MX-1 proliferation assays. The isobutyl ether <b>30c</b> exhibited single digit binding potency but had weaker cell efficacy of 140 nM and 180 nM in the proliferation and BRD4 engagement assays, respectively. The oxetane ether <b>30d</b> was essentially indistinguishable from the ethyl ether in the biological assays, whereas the tertiary alcohol <b>30e</b> and morpholine ether <b>30f</b> were significantly less potent with cellular potencies in the single digit micromolar range. This set of data demonstrated that larger ethers such as the oxetane <b>30d</b> could maintain the biochemical and cellular potency of the smaller ethers; however, the extra molecular weight and metabolic liability of these larger ethers offered no advantage. Since the original fragment hit had a methylamine at this position on the pyridazinone ring, the ethylamine derivative <b>33</b> was synthesized for comparison. Compound <b>33</b> had a biochemical <i>K</i><sub>i</sub> of 14 nM and a MX-1 proliferation potency of 30 nM; however, the BRD4 engagement assay potency of this analog was weaker (140 nM) than the ether homolog <b>25e</b> (39 nM).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Expanded SAR of Pyridone Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0019.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0020.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">TR-FRET BRD4 <i>K</i><sub>i</sub> values are reported as the geometric mean derived from three independent measurements except where indicated.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">EC<sub>50</sub> values are reported as the mean derived from two measurements.</p></div></div><div></div></div><div class="NLM_p">Next, the effect of replacing the phenyl B ring with a heteroaromatic B ring was evaluated. The 3-pyridyl analog was prepared (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Scheme S3</a>) utilizing chemistry sequences similar to those used for the phenyl ring analogs. Compound <b>39</b> presented an activity profile very similar to <b>25e</b> exhibiting a binding <i>K</i><sub>i</sub> of 8.7 nM and EC<sub>50</sub> cellular potencies of 32 and 45 nM, respectively, in the proliferation and BRD4 engagement assays.</div><div class="NLM_p">As detailed previously, the alkylsulfonamide functional group forms a key hydrogen bond with the backbone nitrogen of Asp381. Minor modifications of this functional group that maintained the sulfonyl oxygen for hydrogen bonding were explored. Removing the nitrogen from the sulfonamide gave the direct linked sulfone <b>44</b>. The biochemical potency for <b>44</b> was approximately 3-fold weaker than that of <b>25e</b>, and the cellular potencies were slightly weaker. However, replacing the sulfonamide nitrogen with a methylene spacer provided <b>51</b> which exhibited significantly weaker potency than <b>25e</b> in all of the assays evaluated. A possible explanation for this reduced potency in TR-FRET is that the methylenesulfone adopts a less favorable conformation than the sulfonamide for hydrogen bonding to Asp381. An X-ray cocrystal structure of a methylenesulfone analog such as <b>51</b> was not able to be obtained to test this hypothesis.</div><div class="NLM_p">Physicochemical properties such as permeability, log <i>D</i>, and total polar surface area (TPSA) for the pyridones shown in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> and macrocyclic pyridones described later (<a class="ref internalNav" href="#tbl5" aria-label="Tables 5">Tables 5</a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>) were in acceptable ranges. TPSA values for these analogs ranged from 73 to 109, and experimentally determined log <i>D</i> values ranged from 2.8 to 4.3. Having optimized both the potency and ADME properties of the pyridone series inhibitors, examination of in vivo tumor growth inhibition (TGI) with a representative monocyclic pyridone BET inhibitor in a mouse xenograft model was undertaken. <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> compares the biological efficacy and PK profiles of several leading pyridone derivatives. Although no single compound clearly differentiated itself from the others, compound <b>30a</b> was chosen for an in vivo proof of concept study based primarily on the superior oral exposure and bioavailability demonstrated in mouse PK experiments.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Biological Efficacy and PK Profiles of Leading Pyridones</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">liver microsome</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" colspan="4" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center">TR-FRET</th><th class="rowsep1 colsep0" align="center">MX-1 proliferation</th><th class="rowsep1 colsep0" align="center">BRD4 engagement</th><th class="rowsep1 colsep0" align="center">human</th><th class="rowsep1 colsep0" align="center">rat</th><th class="rowsep1 colsep0" align="center">mouse</th><th class="rowsep1 colsep0" align="center">protein binding mouse</th><th class="rowsep1 colsep0" colspan="4" align="center">mouse PK (3 mg/kg iv and 10 mg/kg po)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> (nM)</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>f</i><sub>u,plasma</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>p,u</sub><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">AUC<sub>u</sub><a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">FaFg<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center"><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25e</b></td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left"><4.3</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left"><6.0</td><td class="colsep0 rowsep0" align="left">0.10</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">0.038</td><td class="colsep0 rowsep0" align="left">0.79</td><td class="colsep0 rowsep0" align="left">56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30a</b></td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">47</td><td class="colsep0 rowsep0" align="left">56</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">0.14</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">0.081</td><td class="colsep0 rowsep0" align="left">0.89</td><td class="colsep0 rowsep0" align="left">63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30d</b></td><td class="colsep0 rowsep0" align="left">9.0</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">48</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="left"><2.5</td><td class="colsep0 rowsep0" align="left">9.0</td><td class="colsep0 rowsep0" align="left">0.20</td><td class="colsep0 rowsep0" align="left">8.6</td><td class="colsep0 rowsep0" align="left">0.026</td><td class="colsep0 rowsep0" align="left">0.30</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left"><2.0</td><td class="colsep0 rowsep0" align="left">6.2</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">0.10</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">0.039</td><td class="colsep0 rowsep0" align="left">0.52</td><td class="colsep0 rowsep0" align="left">41</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Cl<sub>int,u</sub> [L h<sup>–1</sup> kg<sup>–1</sup>].</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last"><i>f</i><sub>u,plasma</sub> (unbound fraction in plasma).</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">Cl<sub>p,u</sub> [L h<sup>–1</sup> kg<sup>–1</sup>].</p></div><div class="footnote" id="t4fn4"><sup>d</sup><p class="last">AUC<sub>u</sub> dose adjusted to 1 mg/kg [μg·h/mL].</p></div><div class="footnote" id="t4fn5"><sup>e</sup><p class="last">FaFg = <i>F</i>/(1 – Cl<sub>p</sub>/Qh). Qh = liver blood flow for mouse: 5.2 L h<sup>–1</sup> kg<sup>–1</sup>.</p></div></div></div><div class="NLM_p">The in vivo efficacy of pyridone <b>30a</b> was evaluated in an OPM-2 mouse xenograft model of multiple myeloma that represents a cancer type previously reported to be responsive to BET inhibition therapy.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> As shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>, compound <b>30a</b>, dosed orally q.d. for 21 days consecutively at doses of 1 mg/kg and 3 mg/kg, demonstrated 66% tumor growth inhibition (TGI) at the 1 mg/kg dose and slightly higher inhibition (73%) at the 3 mg/kg dose, with acceptable animal weight loss. An estimate of an efficacious target concentration can be provided by adjusting the cellular antiproliferative potency (EC<sub>50</sub> of 0.023 μM in OPM-2 cells) for plasma protein binding (fraction unbound = 0.14) resulting in an efficacious target concentration of 0.16 μM. On the basis of extrapolated plasma exposure values obtained in a pharmacokinetic study in nontumor bearing animals (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information Figure S4-A</a>), the efficacious target was achieved daily for approximately 6.5 h for the 3 mg/kg dose group. Tumor growth resumed after the end of the treatment period, suggestive of reversible bromodomain inhibition. This in vivo efficacy verified that BET inhibitors based upon a pyridone core offered potential as valuable agents for hematology-based oncology indications.</div><figure id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0006.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Efficacy of compound <b>30a</b> in a multiple myeloma xenograft (OPM-2) model. Compound or vehicle was administered po daily for 21 days starting on day 0. Values represent mean tumor volume ± standard error (<i>n</i> = 10/group). WL: weight loss. RFS: removed from study due to morbidity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The encouraging tumor growth inhibition demonstrated by <b>30a</b> led to the investigation of novel ideas for new pyridone analogs. The spatial orientation of the aliphatic ether side chain on the pyridone ring of compound <b>20a</b> was observed to be in close proximity to the ortho carbon position of the phenyl C-ring (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). It was hypothesized that cyclic systems formed by tethering these sites might result in improved potency by reinforcement of a low energy binding conformation to the BRD protein and by reduction in the number of rotatable bonds. Conformational ring constraint<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and limiting the number of rotatable bonds<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> are frequently invoked strategies for medicinal chemistry design. Examination of the X-ray cocrystal structure of pyridone <b>20a</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>) appears to indicate that sufficient space exists in the binding pocket to accommodate this type of macrocycle.</div><div class="NLM_p">Linker lengths for the proposed macrocycle were examined empirically (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) with the intent of incorporating a macrocycle that would reinforce the optimized positioning of the C-ring and the B-ring substituent determined for the acyclic analogs and reduce the entropic cost of binding. Ten-membered ring analog <b>55</b>, bearing a three-atom linker, lost significant activity compared to the acyclic analog <b>44</b>. This decline in activity likely results from suboptimal positioning of the phenoxy C-ring in the WPF pocket. Addition of one and two more carbon atoms to the linker (11-membered ring <b>60a</b> and 12-membered ring <b>60b</b>, respectively) produced a substantial improvement in biochemical activity (13-fold for <b>60a</b> and 53-fold for <b>60b</b>) compared to <b>55</b>, along with an even more pronounced improvement in cellular activity. In particular, 12-membered ring sulfonamide <b>61b</b> demonstrated cellular activity in the single digit nanomolar range, which is 3- to 5-fold more potent than the corresponding acyclic pyridone <b>25b</b>. However, when a six-atom linker was incorporated, generating a 13-membered ring macrocycle, a decrease in cellular activity was observed (compound <b>61b</b> vs compound <b>66</b>), although biochemical activities were similar. In addition, compound <b>66</b> exhibited reduced microsomal stability and poor permeability (PAMPA = 0.89 × 10<sup>–6</sup> cm/s), perhaps due to slightly increased lipophilicity (elogD = 4.3). Thus, it was determined that a five-atom linker, generating a 12-membered ring, provided the optimal ring size for the macrocycle.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Examination of Macrocycle Linker Size</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0021.gif" alt="" id="fx6" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">liver microsome</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center">TR-FRET</th><th class="rowsep1 colsep0" align="center">MX-1 proliferation</th><th class="rowsep1 colsep0" align="center">BRD4 engagement</th><th class="rowsep1 colsep0" align="center">human</th><th class="rowsep1 colsep0" align="center">rat</th><th class="rowsep1 colsep0" align="center">mouse</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> (nM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="left">100 ± 4</td><td class="colsep0 rowsep0" align="left">710</td><td class="colsep0 rowsep0" align="left">630</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60a</b></td><td class="colsep0 rowsep0" align="left">7.3 ± 0.5</td><td class="colsep0 rowsep0" align="left">6.9</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">8.9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60b</b></td><td class="colsep0 rowsep0" align="left">1.9 ± 0.1</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">6.7</td><td class="colsep0 rowsep0" align="left">7.8</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61b</b></td><td class="colsep0 rowsep0" align="left">1.5 ± 0.01</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left"><13</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">140</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66</b></td><td class="colsep0 rowsep0" align="left">2.8 ± 0.3</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left">110</td><td class="colsep0 rowsep0" align="left">280</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">TR-FRET BRD4 <i>K</i><sub>i</sub> values are reported as the geometric mean derived from three independent measurements.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">EC<sub>50</sub> values are reported as the mean derived from two measurements.</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">Cl<sub>int,u</sub> [L h<sup>–1</sup> kg<sup>–1</sup>].</p></div></div></div><div class="NLM_p">X-ray structural analysis of 12-membered ring macrocycle <b>61a</b> supported the selection of this scaffold for further optimization. The X-ray cocrystal structure of macrocycle <b>61a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEX">5UEX</a>) (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>) overlaid with acyclic pyridone <b>25e</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEY">5UEY</a>) in BRD4 BDII (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>) shows that all important binding interactions, including the pyridone carbonyl-Asn433 NH<sub>2</sub> H-bond, the C-ring ether binding in the WPF pocket, and the sulfonamide-Asp381 binding, are preserved in the macrocycle and not altered by the presence of the linker. Although macrocycle <b>61a</b> showed improved biochemical activity over its acyclic counterpart <b>25e</b>, the improvement was rather limited (∼3-fold). Subsequently, the energetics of the acyclic compound <b>25e</b> and the macrocyclic analog <b>61a</b> were analyzed using FreeForm<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> based on their protein X-ray cocrystal structures. Acyclic compound <b>25e</b> with the conformation shown in the protein X-ray structure has a global strain energy of 8.0 kcal/mol, while macrocycle <b>61a</b> with the conformation shown in its protein X-ray structure has a global strain energy of 7.0 kcal/mol. The energy of the macrocycle is slightly less than that of the open analog which is in agreement with the experimental binding data.</div><figure id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0007.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of X-ray cocrystal structures <b>25e</b> (blue) and <b>61a</b> (yellow) in BRD4 BDII.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">After identification of the ideal macrocyclic linker length, further optimization of potency utilizing previously identified binding moieties and incorporation of halogens on the C-ring to reduce oxidative metabolism liabilities was carried out (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). In general, macrocycles that incorporated a B-ring sulfonamide (<b>61a</b>, <b>61b</b>, <b>76</b>, and <b>83</b>) were more potent than the analogous B-ring sulfones (<b>60c</b> and <b>74b</b>) and methylenesulfones (<b>86</b> and <b>94</b>), particularly with respect to cellular potency. The incorporation of a B-ring pyridine had little impact on the biochemical or cellular activity of the B-ring sulfonamides (<b>61b</b> vs <b>76</b>, <b>61a</b> vs <b>83</b>), although slightly decreased biochemical and cellular activity was noted for B-ring pyridine sulfones (<b>60c</b> vs <b>74b</b>) and methylenesulfones (<b>94</b> vs <b>86</b>) compared to their B-ring phenyl analogs. It was discovered that formation of 12-membered ring macrocycles always resulted in gains in biochemical and cellular activity compared to the corresponding acyclic analogs. For example, macrocycles <b>61a</b> and <b>61b</b> showed improvements in biochemical (1.6- to 3-fold) and cellular activity (3- to 5-fold) when compared to the acyclic counterparts <b>25e</b> and <b>25b</b>, respectively. When the B-ring aryl group is a pyridine, the macrocyclization resulted in even more substantial improvements in biochemical (9-fold) and cellular activity (7-fold) as demonstrated by comparison of acyclic B-ring pyridine <b>39</b> versus macrocyclic B-ring pyridine <b>83</b>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Detailed Comparison of Macrocyclic Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0022.gif" alt="" id="fx7" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" rowspan="2" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">liver microsome</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" align="center">TR-FRET</th><th class="rowsep1 colsep0" align="center">MX-1 proliferation</th><th class="rowsep1 colsep0" align="center">BRD4 engagement</th><th class="rowsep1 colsep0" align="center">human</th><th class="rowsep1 colsep0" align="center">rat</th><th class="rowsep1 colsep0" align="center">mouse</th><th class="rowsep1 colsep0" align="center">PAMPA</th><th class="rowsep1 colsep0" align="center">protein binding mouse</th><th class="rowsep1 colsep0" colspan="4" align="center">mouse PK (1 mg/kg iv and po)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> (nM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>int,u</sub><a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">10<sup>–6</sup> cm/s</th><th class="colsep0 rowsep0" align="center"><i>f</i><sub>u,plasma</sub><a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">Cl,<sub>p,u</sub><a class="ref internalNav" href="#t6fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">AUC<sub>u</sub><a class="ref internalNav" href="#t6fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">FaFg<a class="ref internalNav" href="#t6fn7" aria-label="g">g</a></th><th class="colsep0 rowsep0" align="center"><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60c</b></td><td class="colsep0 rowsep0" align="left">2.0 ± 0.2</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left">7.6</td><td class="colsep0 rowsep0" align="left">0.04</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">0.041</td><td class="colsep0 rowsep0" align="left">0.44</td><td class="colsep0 rowsep0" align="left">41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61a</b></td><td class="colsep0 rowsep0" align="left">1.5 ± 0.1</td><td class="colsep0 rowsep0" align="left">5.6</td><td class="colsep0 rowsep0" align="left">6.5</td><td class="colsep0 rowsep0" align="left">4.0</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">6.8</td><td class="colsep0 rowsep0" align="left">0.03</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.008</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61b</b></td><td class="colsep0 rowsep0" align="left">1.5 ± 0.01</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left"><13</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">140</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">0.01</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">0.016</td><td class="colsep0 rowsep0" align="left">0.41</td><td class="colsep0 rowsep0" align="left">39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74b</b></td><td class="colsep0 rowsep0" align="left">8.9 ± 0.6</td><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">4.4</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">0.04</td><td class="colsep0 rowsep0" align="left">8.4</td><td class="colsep0 rowsep0" align="left">0.10</td><td class="colsep0 rowsep0" align="left">0.90</td><td class="colsep0 rowsep0" align="left">84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>76</b></td><td class="colsep0 rowsep0" align="left">3.4 ± 0.2</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">9.3</td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">9.2</td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">7.2</td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">6.1</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">0.76</td><td class="colsep0 rowsep0" align="left">70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83</b></td><td class="colsep0 rowsep0" align="left">0.8 ± 0.5 (2)</td><td class="colsep0 rowsep0" align="left">4.2</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left"><1.7</td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">9.0</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="left">0.49</td><td class="colsep0 rowsep0" align="left">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>86</b></td><td class="colsep0 rowsep0" align="left">11 ± 1.3</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left">9.5</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">0.12</td><td class="colsep0 rowsep0" align="left">5.3</td><td class="colsep0 rowsep0" align="left">0.085</td><td class="colsep0 rowsep0" align="left">0.51</td><td class="colsep0 rowsep0" align="left">45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>94</b></td><td class="colsep0 rowsep0" align="left">3.8 ± 0.5</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">9.2</td><td class="colsep0 rowsep0" align="left">0.03</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">0.029</td><td class="colsep0 rowsep0" align="left">0.79</td><td class="colsep0 rowsep0" align="left">68</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">TR-FRET BRD4 <i>K</i><sub>i</sub> values are reported as the geometric mean derived from three independent measurements except where indicated.</p></div><div class="footnote" id="t6fn2"><sup>b</sup><p class="last">EC<sub>50</sub> values are reported as the mean derived from two measurements.</p></div><div class="footnote" id="t6fn3"><sup>c</sup><p class="last">Cl<sub>int,u</sub> [L h<sup>–1</sup> kg<sup>–1</sup>].</p></div><div class="footnote" id="t6fn4"><sup>d</sup><p class="last"><i>f</i><sub>u,plasma</sub> (unbound fraction in plasma).</p></div><div class="footnote" id="t6fn5"><sup>e</sup><p class="last">Cl<sub>p,u</sub> [L h<sup>–1</sup> kg<sup>–1</sup>].</p></div><div class="footnote" id="t6fn6"><sup>f</sup><p class="last">AUC<sub>u</sub>, 1 mg/kg po [μg·h/mL].</p></div><div class="footnote" id="t6fn7"><sup>g</sup><p class="last">FaFg = <i>F</i>/(1 – Cl<sub>p</sub>/Qh). Qh = liver blood flow for mouse: 5.2 L h<sup>–1</sup> kg<sup>–1</sup>.</p></div></div></div><div class="NLM_p">DMPK properties of macrocyclic BET inhibitors were evaluated to identify compounds with the best combination of potency and DMPK properties to enable in vivo efficacy studies. Initial investigations focused on mouse PK evaluation, with the results listed in <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>. Of the compounds examined, B-ring pyridylsulfonamide <b>76</b> and B-ring pyridylsulfone <b>74b</b> demonstrated the best oral exposure in mice due to low clearance, high absorption, and high bioavailability. The potent B-ring pyridinesulfonamide inhibitor <b>83</b> demonstrated good unbound microsomal stability and PAMPA permeability. However, the in vivo free exposure was lower than other compounds evaluated due to high Cl<sub>p,u</sub> (19 L h<sup>–1</sup> kg<sup>–1</sup>) combined with moderate absorption (49%). One of the most potent macrocycles, <b>61a</b>, also exhibited limited exposure when dosed orally in mice. Compound <b>61b</b>, another very potent macrocycle, showed moderate absorption (41%) and reasonable bioavailability despite poor mouse microsomal stability. Rat PK was then examined for these four compounds to determine the best overall PK profile (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>). Pyridylsulfone <b>74b</b> demonstrated the best properties among these four compounds with low clearance (Cl<sub>p,u</sub> = 14 L h<sup>–1</sup> kg<sup>–1</sup>), outstanding absorption (FaFg ≈ 1), and excellent oral bioavailability (<i>F</i> = 95%). It is worth noting that although compound <b>86</b> has reasonable rat microsomal stability compared to compound <b>74b</b>, the rat PK profile was significantly worse, suggesting metabolic pathways other than oxidative metabolism were probably involved. Although several macrocycles showed better cellular activity than compound <b>74b</b>, these compounds displayed inferior PK profiles in both mouse and rat. Thus, on the basis of its outstanding rodent PK and acceptable cellular activity, compound <b>74b</b> was chosen for examination in an in vivo efficacy experiment.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Rat PK of Select Macrocycles</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">rat PK (1 mg/kg iv and po)</th><th class="rowsep1 colsep0" align="center">protein binding</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">Cl<sub>p,u</sub><a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">AUC<sub>u</sub><a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">FaFg<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><i>F</i> (%)</th><th class="colsep0 rowsep0" align="center"><i>f</i><sub>u,plasma</sub><a class="ref internalNav" href="#t7fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61b</b></td><td class="colsep0 rowsep0" align="left">140</td><td class="colsep0 rowsep0" align="left">0.001</td><td class="colsep0 rowsep0" align="left">0.44</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74b</b></td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">0.069</td><td class="colsep0 rowsep0" align="left">∼1</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>76</b></td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">NA<a class="ref internalNav" href="#t7fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>86</b></td><td class="colsep0 rowsep0" align="left">8.5</td><td class="colsep0 rowsep0" align="left">0.002</td><td class="colsep0 rowsep0" align="left">0.42</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">0.24</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup>a</sup><p class="last">Cl<sub>p,u</sub> [L h<sup>–1</sup> kg<sup>–1</sup>].</p></div><div class="footnote" id="t7fn2"><sup>b</sup><p class="last">AUC<sub>u</sub>, 1 mg/kg po [μg·h/mL].</p></div><div class="footnote" id="t7fn3"><sup>c</sup><p class="last">FaFg = <i>F</i>/(1 – Cl<sub>p</sub>/Qh). Qh = liver blood flow for rat: 3.8 L h<sup>–1</sup> kg<sup>–1</sup>.</p></div><div class="footnote" id="t7fn4"><sup>d</sup><p class="last"><i>f</i><sub>u,plasma</sub> (unbound fraction in plasma).</p></div><div class="footnote" id="t7fn5"><sup>e</sup><p class="last">FaFg cannot be calculated due to Cl<sub>p</sub> > liver blood flow.</p></div></div></div><div class="NLM_p">Macrocycle <b>74b</b> was evaluated in a multiple myeloma (OPM-2) mouse xenograft model (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>) dosed orally q.d. for 21 consecutive days at 1 mg/kg and 2 mg/kg. It was well-tolerated at both doses and demonstrated dose-dependent tumor growth inhibition (TGI). At the higher 2 mg/kg dose, compound <b>74b</b> exhibited excellent TGI of 85%. A targeted concentration of 0.30 μM, based upon EC<sub>50</sub> = 0.012 μM in OPM-2 cells and adjusted for plasma protein binding (fraction unbound = 0.04), was likely achieved for >12 h after the 2 mg/kg dose (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information Figure S4-B</a>). Compared to the acyclic pyridone <b>30a</b>, macrocyclic inhibitor <b>74b</b> displayed a modest improvement in antitumor activity at a lower dose while maintaining tolerability.</div><figure id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0008.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Efficacy of compound <b>74b</b> in a multiple myeloma xenograft (OPM-2) model. Compound or vehicle was administered po daily for 21 days starting on day 0. Values represent mean tumor volume ± standard error (<i>n</i> = 8/group). WL: weight loss. RFS: removed from study due to morbidity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The pyridone and macrocyclic pyridones demonstrated no selectivity against the BET family proteins BRD2, BRD3, and BRDT in the TR-FRET assay (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). Compound <b>25e</b> was screened against a panel of 20 bromodome proteins to evaluate selectivity of the pyridone/macrocyclic pyridone inhibitor series. Weak inhibition in the 1–2 μM range was observed against BRD9, CREBBP, SMARCA4, and EP300. Inhibition of other proteins in the panel was >2 μM indicating high bromodome selectivity for the pyridone series inhibitors. Selectivity data can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information, Table S2</a>.</div><div class="NLM_p last">In summary, novel BET inhibitor fragment <b>1</b> was identified by an NMR-based fragment screen, and its binding to the BRD4 BDII protein was confirmed by the protein X-ray cocrystal structure. Medicinal chemistry optimization aided by protein X-ray crystallography resulted in the identification of a number of potent acyclic pyridone-based novel BET inhibitors. Two key hydrogen bond interactions, combined with a hydrophobic interaction with the WPF shelf of the BRD4 protein, were identified leading to increased potency. Macrocyclization of these acyclic derivatives resulted in inhibitors with further improvement in biochemical and cellular potency. This improvement is likely the result of increased rigidity of the desired binding conformation along with increased van der Waals interactions while maintaining the binding interactions observed with the acyclic inhibitors. Binding affinities exhibited by the most potent inhibitors represent a 100 000-fold increase over the original screening fragment <b>1</b> (160 μM). Potent compounds from both series demonstrated excellent rodent PK profiles. These profiles resulted in significant oral efficacy for both acyclic and macrocyclic BET inhibitors in a multiple myeloma OPM-2 mouse xenograft model, with macrocyclic inhibitor <b>74b</b> exhibiting somewhat superior antitumor activity compared to acyclic pyridone <b>30a</b>. Both the acyclic pyridone series and the macrocyclic pyridone series served as the framework for additional SAR investigations that will be reported in due course.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08245" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08245" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> NMR-Based Fragment Screen</h3><div class="NLM_p last">Recombinant, human, BRD4 BDII protein (residues 352–457), labeled with <sup>13</sup>C at the methyl groups of isoleucine (δ only), leucine, valine, and methionine, was first expressed in <i>E. coli</i> and purified as described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information</a>. Screening was then carried out by monitoring changes in the <sup>13</sup>C HSQC spectrum of this methyl-labeled protein upon addition of approximately 18 000 fragments. These fragments were first tested in mixtures of 30, and those mixtures that elicited the largest spectral perturbations were subsequently deconvoluted in order to find the individual fragment binders.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> BRD Protein Crystallization Method</h3><div class="NLM_p last">Human BRD4 BDII protein was concentrated to ∼4 mg/mL in 10 mM Bis-Tris, pH 6.8, 100 mM NaCl, 5 mM DTT buffer. Protein was incubated with compounds at a 3:1 mM ratio of compound to protein at 4 °C for 2 h. The protein–compound complexes were screened against SGC-1 and SGC Redwing custom screens (prepared by Rigaku) at 17 °C. Some protein–compound complexes were also screened against commercially available screens PEGRx and SaltRx (Hampton Research) at 17 °C. Vapor-diffusion sitting drops were prepared using a Mosquito liquid dispenser (TTP Labtech) in MRC 2 well crystallization plates (Hampton Research). The drops contained 0.3 μL of protein and 0.3 μL of reservoir solution over wells of 40 μL of reservoir solution.</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Synthetic Materials and Methods</h3><div class="NLM_p last">Unless otherwise specified, reactions were performed under an inert atmosphere of nitrogen and monitored by thin-layer chromatography (TLC) and/or LCMS. All reagents were purchased from commercial suppliers and used as provided. 3-Mercaptopropyl functionalized silica gel (Aldrich, catalog 538086) was routinely used to remove ionic palladium species during workup of Suzuki coupling reactions. 1,3,5,7-Tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane (Aldrich catalog 695459, CAS 97739-46-3) was a preferred ligand for Suzuki coupling reactions. Flash column chromatography was carried out on prepacked silica gel cartridges. Reverse phase chromatography samples were purified by preparative HPLC on a Phenomenex Luna C8(2) 5 μm 100 Å AXIA column (30 mm × 75 mm). A gradient of acetonitrile (A) and 0.1% trifluoroacetic acid in water (B) was used at a flow rate of 50 mL/min (0–0.5 min 10% A, 0.5–7.0 min linear gradient 10–95% A, 7.0–10.0 min 95% A, 10.0–12.0 min linear gradient 95–10% A). Samples were injected in 1.5 mL of DMSO:methanol (1:1). A custom purification system was used consisting of the following modules: Waters LC4000 preparative pump; Waters 996 diode-array detector; Waters 717+ autosampler; Waters SAT/IN module, Alltech Varex III evaporative light-scattering detector; Gilson 506C interface box; and two Gilson FC204 fraction collectors. The system was controlled using Waters Millennium32 software, automated using an AbbVie developed Visual Basic application for fraction collector control and fraction tracking. Fractions were collected based upon UV signal threshold and selected fractions subsequently analyzed by flow injection analysis mass spectrometry using positive APCI ionization on a Finnigan LCQ using 70:30 methanol/10 mM NH<sub>4</sub>OH (aq) at a flow rate of 0.8 mL/min. Loop-injection mass spectra were acquired using a Finnigan LCQ running LCQ Navigator 1.2 software and a Gilson 215 liquid handler for fraction injection controlled by an AbbVie developed Visual Basic application. All NMR spectra were recorded on 300–500 MHz instruments as specified with chemical shifts given in ppm (δ) and are referenced to an internal standard of tetramethylsilane (δ 0.00). <sup>1</sup>H–<sup>1</sup>H couplings are assumed to be first-order, and peak multiplicities are reported in the usual manner. HPLC purity determinations were performed on a Waters e2695 separation module/Waters 2489 UV/visible detector. Column types and elution methods are described in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">Supporting Information</a>. The purity of the biologically evaluated compounds was determined by HPLC to be >95%. Solvents used for HPLC analysis and sample preparation were HPLC grade.</div></div><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> 5,6-Dichloro-2-methylpyridazin-3(2<i>H</i>)-one (<b>4</b>)</h3><div class="NLM_p last">A mixture of <b>3</b> (2.1 g, 13 mmol) and cesium carbonate (4.98 g, 15.3 mmol) in DMF (25 mL) was treated with iodomethane (2.35 g, 16.6 mmol). The solution was stirred at room temperature for 2 h. The reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate three times. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash column chromatography on silica gel, eluting with 20% ethyl acetate in hexanes to give 2.08 g of the title compound (91%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.49 (s, 1H), 3.60 (s, 3H).</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> 6-Chloro-5-methoxy-2-methylpyridazin-3(2<i>H</i>)-one (<b>5</b>)</h3><div class="NLM_p last">Methanol (80 mL) was cooled to 0 °C. To this solvent was added sodium (0.804 g, 35.0 mmol). All sodium was dissolved completely within 1 h. To this solution was added <b>4</b> (6.2 g, 35 mmol). The reaction mixture was stirred at 50 °C for 2 h. The solvent was evaporated, and the residue was triturated with water. The solid was collected by filtration to give 5.41 g (89%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 6.44 (s, 1H), 3.84 (s, 3H), 3.55 (s, 3H).</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 2-Bromo-4-nitro-1-phenoxybenzene (<b>7</b>)</h3><div class="NLM_p last">Phenol (2.59 g, 27.5 mmol), 3-bromo-4-fluoronitrobenzene (<b>6</b>) (5.50 g, 25.0 mmol), and cesium carbonate (8.15 g, 25.0 mmol) were combined in DMSO (25 mL) under nitrogen and heated at 100 °C for 2 h. The mixture was cooled and partitioned between ethyl acetate and water. The organic layer was washed 3× with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated to provide an amber oil (7.0 g, 95%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.58 (d, <i>J</i> = 2.7 Hz, 1H), 8.12 (dd, <i>J</i> = 9.1, 2.7 Hz, 1H), 7.53–7.42 (m, 2H), 7.35–7.25 (m, 1H), 7.15–7.09 (m, 2H), 6.90–6.82 (m, 1H).</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 3-Bromo-4-phenoxyaniline (<b>8</b>)</h3><div class="NLM_p last">A mixture of <b>7</b> (2.94 g, 10.0 mmol), iron powder (2.79 g, 50.0 mmol), and ammonium chloride (0.802 g, 15.0 mmol) in a solvent mixture of ethanol (24 mL), THF (24 mL), water (8 mL) was heated at 95 °C with vigorous stirring for 1.5 h. The reaction mixture was cooled and filtered through a tight plug of Celite, and the Celite pad was washed repeatedly with ethanol and THF. The filtrate was concentrated and the residue dissolved in ethyl acetate, washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated to afford the title compound (2.56 g, 97%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.36–7.27 (m, 2H), 7.02 (ddt, <i>J</i> = 7.8, 7.1, 1.1 Hz, 1H), 6.93–6.86 (m, 2H), 6.84–6.76 (m, 2H), 6.62 (dd, <i>J</i> = 8.7, 2.7 Hz, 1H), 5.33 (s, 2H). MS (APCI+) <i>m</i>/<i>z</i> 264/266 (M + H)<sup>+</sup>.</div></div><div id="sec4_2_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 4-Phenoxy-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)aniline (<b>10</b>)</h3><div class="NLM_p last">Compound <b>8</b> (4.38 g, 16.6 mmol), 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane) (8.42 g, 33.2 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.456 g, 0.498 mmol), potassium acetate (3.58 g, 36.5 mmol), and 1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane (0.436 g, 1.49 mmol) were combined in a 250 mL round-bottom flask and sparged with argon for 30 min. Meanwhile 1,4-dioxane (83 mL) was sparged with nitrogen for 30 min and was added via syringe to the solids. The dark colored mixture was heated for 20 h at 80 °C. The mixture was cooled and partitioned between ethyl acetate and water. The organic layer was washed with water, saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by chromatography (silica, 0–40% ethyl acetate in hexanes) gave an oil (5.1 g) that was triturated in a minimal amount of hexanes. The resulting solid was collected, rinsed with additional hexanes, and dried to constant mass (3.64 g, 70%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.27–7.20 (m, 2H), 6.95–6.87 (m, 2H), 6.79–6.67 (m, 4H), 5.03 (s, 2H), 1.05 (s, 12H). MS (ESI+) <i>m</i>/<i>z</i> 310/312 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Suzuki Coupling Method A. 5-Methoxy-2-methyl-6-(2-phenoxyphenyl)pyridazin-3(2<i>H</i>)-one (<b>11</b>)</h3><div class="NLM_p last">A mixture of <b>5</b> (0.022 g, 0.13 mmol), 2-phenoxyphenylboronic acid <b>(9)</b> (0.035 g, 0.16 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (0.011 g, 5 mol %), and cesium fluoride (0.057 g, 0.38 mmol) in DME (2 mL) and methanol (1 mL) was heated under microwave conditions (120 °C, 40 min). The reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate three times. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash column chromatography on silica gel, eluting with 30% ethyl acetate in hexanes to give the title compound (0.022 g, 57%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.60–7.64 (m, 1H), 7.40–7.47 (m, 2H), 7.32–7.38 (m, 2H), 7.21–7.25 (m, 1H), 7.10 (t, <i>J</i> = 7.3 Hz, 1H), 6.91–6.98 (m, 3H), 6.29 (s, 1H), 3.61 (s, 3H), 3.57 (s, 3H). MS (DCI+) <i>m</i>/<i>z</i> 309.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 6-(5-Amino-2-phenoxyphenyl)-5-methoxy-2-methylpyridazin-3(2<i>H</i>)-one (<b>12</b>)</h3><div class="NLM_p last">A mixture of <b>5</b> (0.022 g, 0.13 mmol), <b>10</b> (0.050 g, 0.16 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (0.011 g, 5 mol %), and cesium fluoride (0.057 g, 0.38 mmol) in DME (2 mL) and methanol (1 mL) was heated under microwave conditions (120 °C, 40 min). The reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate three times. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash column chromatography on silica gel, eluting with 30% ethyl acetate in hexanes to give the title compound (0.020 g, 62%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.23–7.26 (m, 2H), 6.95 (t, <i>J</i> = 7.9 Hz, 1H), 6.76–6.81 (m, 3H), 6.64–6.67 (m, 1H), 6.56 (d, <i>J</i> = 2.7 Hz, 1H), 6.21 (s, 1H), 5.11 (s, 2H), 3.51 (s, 3H), 3.50 (s, 3H). MS (DCI+) <i>m</i>/<i>z</i> 324.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <i>N</i>-[3-(4-Methoxy-1-methyl-6-oxo-1,6-dihydropyridazin-3-yl)-4-phenoxyphenyl]acetamide (<b>13a</b>)</h3><div class="NLM_p last">A mixture of <b>12</b> (0.030 g, 0.093 mmol), acetyl chloride (0.0087 g, 0.11 mmol), and triethylamine (0.028 g, 0.28 mmol) in DCM (2 mL) was stirred for 2 h. The solvent was evaporated under reduced pressure. The residue was purified by reverse phase HPLC (C18, CH<sub>3</sub>CN/water (0.1% TFA), 0–100%) to afford the title compound (0.034 g, 99%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.11 (s, 1H), 7.89 (d, <i>J</i> = 2.7 Hz, 1H), 7.67–7.72 (m, 2H), 7.32–7.36 (m, 2H), 7.08 (t, <i>J</i> = 7.3 Hz, 1H), 7.00 (d, J = 8.8 Hz, 1H), 6.92–6.95 (m, 3H), 3.67 (s, 3H), 2.05 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 336.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>-(3-(4-Methoxy-1-methyl-6-oxo-1,6-dihydropyridazin-3-yl)-4-phenoxyphenyl)methanesulfonamide (<b>13b</b>)</h3><div class="NLM_p last">A mixture of <b>12</b> (0.030 g, 0.093 mmol), MsCl (0.021 g, 0.19 mmol), and triethylamine (0.036 g, 0.36 mmol) in DCM (1 mL) was stirred at ambient temperature for 1 h. The solvent was evaporated, and the residue was taken up in 1,4-dioxane (2 mL) and 1.0 N sodium hydroxide (1 mL). The reaction mixture was heated at 90 °C for 1 h. The solvents were partially evaporated, and the residue was partitioned between water and ethyl acetate. The aqueous layer was neutralized with 10% aqueous HCl solution and extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. The residue was purified by reverse phase HPLC (C18, CH<sub>3</sub>CN/water (0.1% TFA), 20–100%) to afford 0.025 g (68%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 7.29–7.35 (m, 1H), 7.22 (d, <i>J</i> = 2.7 Hz, 1H), 7.08 (t, <i>J</i> = 7.3 Hz, 1H), 6.99 (d, <i>J</i> = 8.8 Hz, 1H), 6.91–6.93 (m, 2H), 6.29 (s, 1H), 3.60 (s, 3H), 3.56 (s, 3H), 3.02 (s, 3H). MS (DCI+) <i>m</i>/<i>z</i> 402.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Ethyl [3-(4-Methoxy-1-methyl-6-oxo-1,6-dihydropyridazin-3-yl)-4-phenoxyphenyl]carbamate (<b>13c</b>)</h3><div class="NLM_p last">A mixture of <b>12</b> (0.030 g, 0.093 mmol), ethyl chloroformate (0.015 g, 0.14 mmol), and triethylamine (0.028 g, 0.28 mmol) in DCM (1 mL) was stirred at ambient temperature for 1 h. The solvent was evaporated, and the residue was purified by reverse phase HPLC (C18, CH<sub>3</sub>CN/water (0.1% TFA), 20–100%) to afford 0.030 g (81%) of the title compound. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.71 (s, 1H), 7.48–7.51 (m, 2H), 7.29–7.33 (m, 2H), 7.05 (t, <i>J</i> = 7.5 Hz, 1H), 6.96 (d, <i>J</i> = 8.8 Hz, 1H), 6.88 (d, <i>J</i> = 7.6 Hz, 2H), 6.27 (s, 1H), 4.12 (q, <i>J</i> = 7.0 Hz, 2H), 3.57 (s, 3H), 3.55 (s, 3H), 1.24 (t, <i>J</i> = 7.0 Hz, 3H). MS (DCI+) <i>m</i>/<i>z</i> 396.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 1-Ethyl-3-[3-(4-methoxy-1-methyl-6-oxo-1,6-dihydropyridazin-3-yl)-4-phenoxyphenyl]urea (<b>13d</b>)</h3><div class="NLM_p last">A mixture of <b>12</b> (0.030 g, 0.093 mmol), ethyl isocyanate (0.019 g, 0.27 mmol), and triethylamine (0.027 g, 0.27 mmol) in DCM (1 mL) was stirred at 43 °C overnight. The solvent was evaporated, and the residue was purified by reverse phase HPLC (C18, CH<sub>3</sub>CN/water (0.1% TFA), 0–100%) to afford the title compound (0.019 g, 51%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.54 (s, 1H), 7.52 (d, <i>J</i> = 2.4 Hz, 1H), 7.38 (dd, <i>J</i> = 8.8, 2.75 Hz, 1H), 7.28–7.31 (m, 2H), 7.03 (t, <i>J</i> = 7.3 Hz, 1H), 6.90 (d, <i>J</i> = 8.5 Hz, 1H), 6.87 (d, <i>J</i> = 7.6 Hz, 2H), 6.26 (s, 1H), 3.56 (s, 3H), 3.55 (s, 3H), 3.10 (q, <i>J</i> = 7.2 Hz, 2H), 1.05 (t, <i>J</i> = 7.2 Hz, 3H). MS (DCI+) <i>m</i>/<i>z</i> 395.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 5-Bromo-4-chloropyridin-2(1<i>H</i>)-one (<b>15</b>)</h3><div class="NLM_p last">Compound <b>14</b> (10 g, 48 mmol) in 75% (v/v) sulfuric acid (201 mL, 2.83 mol) was chilled in an ice bath and treated dropwise with a solution of sodium nitrite (11.0 g, 159 mmol) in water (101 mL, 5.58 mol). The mixture was stirred for 3 h and treated dropwise with concentrated aqueous ammonia (75 mL). The resulting white precipitate was collected by vacuum filtration, washed with water, and dried to constant mass to afford the title compound (11.4 g, quantitative). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.90 (s, 1H), 6.71 (s, 1H).</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 5-Bromo-4-chloro-1-methylpyridin-2(1<i>H</i>)-one (<b>16</b>)</h3><div class="NLM_p last">A mixture of <b>15</b> (20 g, 96 mmol) and cesium carbonate (9.84 g, 30.2 mmol) in DMF (240 mL) was treated dropwise with MeI (1.81 mL, 29.0 mmol). The mixture was stirred at ambient temperature for 2 h and partitioned between ethyl acetate and water. The combined organics were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was triturated with 10% ethyl acetate/heptanes to afford the title compound (20 g, 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.25 (d, <i>J</i> = 1.6 Hz, 1H), 6.74 (d, <i>J</i> = 2.0 Hz, 1H), 3.39 (s, 3H).</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 5-Bromo-4-methoxy-1-methylpyridin-2(1<i>H</i>)-one (<b>17a</b>)</h3><div class="NLM_p last">Compound <b>16</b> (1.11 g, 5.00 mmol) and 0.5 M sodium methoxide in methanol (20 mL, 10 mmol) were mixed together and heated at 60 °C for 16 h, cooled, and partitioned with ethyl acetate and water. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and evaporated. The residue was triturated with ethyl acetate/heptanes (10%) to give the title compound (0.895 g, 82%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.00 (s, 1H), 5.91 (s, 1H), 3.78 (s, 3H), 3.32 (s, 3H).</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 5-Bromo-4-ethoxy-1-methylpyridin-2(1<i>H</i>)-one (<b>17b</b>)</h3><div class="NLM_p last">Compound <b>16</b> (10 g, 45 mmol) in 2 M sodium ethoxide in ethanol (67.4 mL, 135 mmol) was heated at 60 °C for 1 h. The solution was cooled to ambient temperature, water (150 mL) added, and the aqueous layer was extracted with ethyl acetate (3 × 50 mL). The combined organics were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. Purification by flash chromatography (silica, 0–2% methanol in DCM) afforded 9.25 g (89%) of the title compound. <sup>1</sup>H NMR (501 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.00 (s, 1H), 5.89 (s, 1H), 4.05 (q, <i>J</i> = 7.0 Hz, 2H), 1.33 (t, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 4-Methoxy-1-methyl-5-(2-phenoxyphenyl)pyridin-2(1<i>H</i>)-one (<b>18a</b>)</h3><div class="NLM_p last">Compound <b>18a</b> was prepared from <b>17a</b> and <b>9</b> according to general Suzuki coupling method A (0.030 g, 48%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.57 (m, 1H), 7.30–7.37 (m, 4H), 7.16–7.20 (m, 1H), 7.07 (t, <i>J</i> = 7.3 Hz, 1H), 6.90–6.92.(m, 3H), 5.79 (s, 1H), 3.54 (s, 3H), 3.34 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 308.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 5-(5-Amino-2-phenoxyphenyl)-4-methoxy-1-methylpyridin-2(1<i>H</i>)-one (<b>18b</b>)</h3><div class="NLM_p last">Compound <b>18b</b> was prepared from <b>17a</b> and <b>10</b> according to general Suzuki coupling method A (0.030 g, 65%). MS (APCI+) <i>m</i>/<i>z</i> 323 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 5-(5-Amino-2-phenoxyphenyl)-4-ethoxy-1-methylpyridin-2(1<i>H</i>)-one (<b>19</b>)</h3><div class="NLM_p last">Compound <b>19</b> was prepared from <b>17b</b> and <b>10</b> according to general Suzuki coupling method A (0.027 g, 65%). MS (APCI+) <i>m</i>/<i>z</i> 337 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-[3-(4-Methoxy-1-methyl-6-oxo-1,6-dihydropyridin-3-yl)-4-phenoxyphenyl]methanesulfonamide (<b>20a</b>)</h3><div class="NLM_p last">Compound <b>20a</b> was prepared from <b>18a</b> according to the procedure used for the preparation of compound <b>13b</b> to provide the title compound (0.012 g, 39%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.70 (s, 1H), 7.58 (s, 1H), 7.29–7.33 (m, 2H), 7.20 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.13 (d, <i>J</i> = 2.7 Hz, 1H), 7.04 (t, <i>J</i> = 7.5 Hz, 1H), 6.94 (d, <i>J</i> = 8.8 Hz, 1H), 6.88 (d, <i>J</i> = 7.6 Hz, 2H), 5.79 (s, 1H), 3.53 (s, 3H), 3.33 (s, 3H), 3.02 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 401.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-(3-(4-Methoxy-1-methyl-6-oxo-1,6-dihydropyridin-3-yl)-4-phenoxyphenyl)ethanesulfonamide (<b>20b</b>)</h3><div class="NLM_p last">Compound <b>20b</b> was prepared from <b>18b</b> according to the procedure used for the preparation of compound <b>13b</b> substituting ethanesulfonyl chloride for MsCl to provide the title compound (0.035 g, 44%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.72 (s, 1H), 7.54 (s, 1H), 7.30–7.24 (m, 2H), 7.16 (dd, <i>J</i> = 8.8, 2.8 Hz, 1H), 7.10 (d, <i>J</i> = 2.7 Hz, 1H), 7.01 (tt, <i>J</i> = 7.3, 1.1 Hz, 1H), 6.90 (d, <i>J</i> = 8.7 Hz, 1H), 6.86–6.81 (m, 2H), 5.75 (s, 1H), 3.57 (s, 3H), 3.50 (s, 3H), 3.08 (q, <i>J</i> = 7.3 Hz, 2H), 1.20 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 415.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i>-(3-(4-Ethoxy-1-methyl-6-oxo-1,6-dihydropyridin-3-yl)-4-phenoxyphenyl)ethanesulfonamide (<b>21</b>)</h3><div class="NLM_p last">Compound <b>21</b> was prepared from <b>19</b> according to the procedure used for the preparation of compound <b>13b</b> substituting ethanesulfonyl chloride for MsCl to provide the title compound (0.046, 56%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.73 (s, 1H), 7.54 (s, 1H), 7.28 (t, <i>J</i> = 7.9 Hz, 2H), 7.20–7.10 (m, 2H), 7.01 (t, <i>J</i> = 7.3 Hz, 1H), 6.93 (d, <i>J</i> = 8.7 Hz, 1H), 6.84 (d, <i>J</i> = 8.0 Hz, 2H), 5.75 (s, 1H), 3.86 (q, <i>J</i> = 6.9 Hz, 2H), 3.30 (s, 3H), 3.09 (q, <i>J</i> = 7.3 Hz, 2H), 1.21 (t, <i>J</i> = 7.3 Hz, 3H), 1.09 (t, <i>J</i> = 7.0 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 429.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 2-Bromo-1-(2,4-difluorophenoxy)-4-nitrobenzene (<b>26</b>)</h3><div class="NLM_p last">Compound <b>6</b> (15 g, 68 mmol), 2,4-difluorophenol (10.6 g, 82.0 mmol), and cesium carbonate (26.7 g, 82.0 mmol) were combined in DMSO (75 mL) and then heated to 110 °C for 60 min. The mixture was cooled. Water (1000 mL) and saturated aqueous sodium chloride (1000 mL) were added, and the mixture was extracted with ethyl acetate (3 × 200 mL). The combined organics were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure to provide the title compound (22.5 g, quant). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.58 (d, <i>J</i> = 2.8 Hz, 1H), 8.20 (dd, <i>J</i> = 9.2, 2.8 Hz, 1H), 7.61 (ddd, <i>J</i> = 11.4, 8.8, 3.0 Hz, 1H), 7.53 (td, <i>J</i> = 9.2, 5.5 Hz, 1H), 7.30–7.22 (m, 1H), 7.00 (dd, <i>J</i> = 9.1, 1.0 Hz, 1H).</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 2-(2-(2,4-Difluorophenoxy)-5-nitrophenyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (<b>27</b>)</h3><div class="NLM_p">Compound <b>26</b> (22.5 g, 68.2 mmol), iron powder (19.0 g, 341 mmol), and ammonium chloride (7.30 g, 136 mmol) were dissolved in THF (117 mL), ethanol (117 mL), and water (39.0 mL) and then heated under reflux at 100 °C for 2 h. The mixture was cooled to just below reflux, filtered through Celite, and then the filter cake was washed with warm methanol (3 × 50 mL). The solution was concentrated under reduced pressure, neutralized to a pH of 8 with saturated NaHCO<sub>3</sub> (150 mL) and then the aqueous layer extracted with ethyl acetate (3 × 100 mL). The combined organics were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, concentrated, then purified (silica gel, ethyl acetate/hexanes 0–15%) to afford 3-bromo-4-(2,4-difluorophenoxy)aniline (16.8 g, 82%).</div><div class="NLM_p last">3-Bromo-4-(2,4-difluorophenoxy)aniline (14.3 g, 47.7 mmol), 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane) (24.2 g, 95.0 mmol), potassium acetate (10.3 g, 105 mmol), 1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane (1.39 g, 4.77 mmol), and tris(dibenzylideneacetone)dipalladium(0) (1.31 g, 1.43 mmol) were degassed under argon for 30 min. 1,4-Dioxane (200 mL), degassed with argon for 30 min, was then cannulated in and the mixture heated to 80 °C for 22 h. The cooled mixture was vacuum filtered through Celite, rinsed with ethyl acetate (100 mL), washed with saturated aqueous sodium chloride (150 mL) and water (150 mL), and then the aqueous layer was extracted with ethyl acetate (3 × 150 mL). The combined organics were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, gravity filtered, then concentrated. Purification by flash chromatography (silica, ethyl acetate/hexanes, 0–25%) afforded the title compound (16.8 g, quantitative). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.27 (ddd, <i>J</i> = 11.6, 8.8, 3.0 Hz, 1H), 6.92 (d, <i>J</i> = 2.8 Hz, 1H), 6.83 (dddd, <i>J</i> = 9.1, 8.2, 3.1, 1.7 Hz, 1H), 6.77–6.65 (m, 2H), 6.48 (td, <i>J</i> = 9.3, 5.5 Hz, 1H), 1.04 (s, 12H).</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 5-(5-Amino-2-(2,4-difluorophenoxy)phenyl)-4-chloro-1-methylpyridin-2(1<i>H</i>)-one (<b>28</b>)</h3><div class="NLM_p last">A mixture of <b>16</b> (2.23 g, 10.0 mmol), <b>27</b> (3.82 g, 11.0 mmol), 1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane (0.342 g, 1.17 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.275 g, 0.300 mmol), and potassium phosphate (5.31 g, 25.0 mmol) in 1,4-dioxane (32 mL) and water (8.00 mL) was degassed and backfilled with nitrogen several times. The reaction mixture was heated at 60 °C for 16 h. The reaction mixture was cooled and partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate three times. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash column chromatography (silica gel, 60% ethyl acetate in hexanes) to give the title compound (3.12 g, 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.80 (s, 1H), 7.28 (ddd, <i>J</i> = 11.6, 8.9, 3.0 Hz, 1H), 6.98–6.81 (m, 2H), 6.76 (d, <i>J</i> = 8.6 Hz, 1H), 6.61 (dd, <i>J</i> = 8.7, 2.8 Hz, 1H), 6.55 (s, 1H), 6.50 (d, <i>J</i> = 2.7 Hz, 1H), 5.13 (s, 2H), 3.38 (s, 2H). MS (ESI+) <i>m</i>/<i>z</i> 363 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(3-(4-Chloro-1-methyl-6-oxo-1,6-dihydropyridin-3-yl)-4-(2,4-difluorophenoxy)phenyl)ethanesulfonamide (<b>29</b>)</h3><div class="NLM_p last">A mixture of <b>28</b> (1.0 g, 2.8 mmol), ethanesulfonyl chloride (1.06 g, 8.27 mmol), and triethylamine (1.67 g, 16.5 mmol) in DCM (20 mL) was stirred at ambient temperature for 2 h. The solvent was evaporated, and the residue was treated with 2 M aqueous sodium hydroxide (5.51 mL, 11.0 mmol) and 1,4-dioxane (5 mL). The reaction mixture was heated at 90 °C for 2 h. The reaction mixture was cooled and partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate three times. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash column chromatography (silica gel, 0–70% ethyl acetate in hexanes) to give the title compound (1.02 g, 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.83 (s, 1H), 7.92 (s, 1H), 7.40 (ddd, <i>J</i> = 11.3, 8.8, 2.9 Hz, 1H), 7.24 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.16 (d, <i>J</i> = 2.7 Hz, 1H), 7.15–6.99 (m, 2H), 6.89 (d, <i>J</i> = 8.8 Hz, 1H), 6.63 (s, 1H), 3.43 (s, 3H), 3.11 (q, <i>J</i> = 7.3 Hz, 2H), 1.22 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 455 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>N</i>-[4-(2,4-Difluorophenoxy)-3-(4-methoxy-1-methyl-6-oxo-1,6-dihydropyridin-3-yl)phenyl]ethanesulfonamide (<b>30a</b>)</h3><div class="NLM_p last">Sodium (0.354 g, 15.4 mmol) was added to methanol (15 mL). The reaction mixture was stirred until all the sodium disappeared. To this solution was then added <b>29</b> (0.35 g, 0.77 mmol). The reaction mixture was heated at 60 °C overnight, cooled, and concentrated. The reaction mixture was partitioned between water and ethyl acetate. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. Purification by reverse phase preparative HPLC (C18, CH<sub>3</sub>CN/water (0.1% TFA), 20–100%) afforded the title compound (0.30 g, 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.78 (s, 1H), 7.63 (s, 1H), 7.42–7.33 (m, 1H), 7.20 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.14 (d, <i>J</i> = 2.7 Hz, 1H), 7.08–7.00 (m, 2H), 6.89 (d, <i>J</i> = 8.7 Hz, 1H), 5.84 (s, 1H), 3.61 (s, 3H), 3.38 (s, 3H), 3.12 (q, <i>J</i> = 7.3 Hz, 2H), 1.24 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.4, 162.7, 158.8, 158.7, 156.4, 156.2, 154.0, 153.9, 151.5, 151.4, 151.0, 140.4, 140.3, 140.3, 140.2, 139.1, 133.9, 125.6, 123.9, 121.4, 121.4, 121.3, 121.3, 121.2, 118.1, 111.8, 111.8, 111.6, 111.6, 108.7, 105.6, 105.4, 105.4, 105.1, 95.5, 55.5, 45.2, 35.8, 8.0. MS (ESI+) <i>m</i>/<i>z</i> 451.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> <i>N</i>-(4-(2,4-Difluorophenoxy)-3-(1-methyl-6-oxo-4-(2,2,2-trifluoroethoxy)-1,6-dihydropyridin-3-yl)phenyl)ethanesulfonamide (<b>30b</b>)</h3><div class="NLM_p last">Sodium hydride (0.016 g, 0.440 mmol, 60% in oil) was stirred in anhydrous 1,4-dioxane (2 mL). To this solution was added 2,2,2-trifluoroethanol (23.9 μL, 0.330 mmol) portionwise, and the suspension was stirred for 30 min. To this suspension was added <b>29</b> (0.05 g, 0.11 mmol), and the mixture was stirred for 2 h at 75 °C. Cold water (10 mL) was added and the aqueous layer extracted with ethyl acetate three times. The combined organics were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and then concentrated. Purification by reverse phase HPLC (C18, CH<sub>3</sub>CN/water (0.1% TFA), 20–100%) afforded 0.031 g (55%) of the title compound. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 7.69 (s, 1H), 7.41–7.30 (m, 1H), 7.20 (dd, <i>J</i> = 8.7, 2.7 Hz, 1H), 7.13 (d, <i>J</i> = 2.7 Hz, 1H), 7.05–6.86 (m, 3H), 6.03 (s, 1H), 4.70 (q, <i>J</i> = 8.8 Hz, 2H), 3.37 (s, 3H), 3.07 (q, <i>J</i> = 7.3 Hz, 2H), 1.20 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 519.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>N</i>-(4-(2,4-Difluorophenoxy)-3-(4-isobutoxy-1-methyl-6-oxo-1,6-dihydropyridin-3-yl)phenyl)ethanesulfonamide (<b>30c</b>)</h3><div class="NLM_p last">Compound <b>30c</b> was prepared according to the procedure used for the preparation of <b>30b</b>, substituting 2-methylpropan-1-ol for 2,2,2-trifluoroethanol (0.029 g, 54%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 7.58 (s, 1H), 7.38–7.17 (m, 2H), 7.13 (d, <i>J</i> = 2.7 Hz, 1H), 7.03–6.92 (m, 2H), 6.87 (td, <i>J</i> = 9.1, 5.6 Hz, 1H), 5.76 (s, 1H), 3.62 (d, <i>J</i> = 6.4 Hz, 2H), 3.33 (s, 3H), 3.07 (q, <i>J</i> = 7.3 Hz, 2H), 1.93–1.76 (m, 1H), 1.20 (t, <i>J</i> = 7.3 Hz, 3H), 0.81 (d, <i>J</i> = 6.7 Hz, 6H). MS (ESI+) <i>m</i>/<i>z</i> 493.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>-{4-(2,4-Difluorophenoxy)-3-[1-methyl-4-(oxetan-3-yloxy)-6-oxo-1,6-dihydropyridin-3-yl]phenyl}ethanesulfonamide (<b>30d</b>)</h3><div class="NLM_p last">Compound <b>30d</b> was prepared according to the procedure used for the preparation of <b>30b</b>, substituting oxetan-3-ol for 2,2,2-trifluoroethanol (0.035 g, 71%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.80 (s, 1H), 7.70 (s, 1H), 7.36–7.42 (m, 1H), 7.18–7.22 (m, 2H), 7.01–7.11 (m, 2H), 6.91 (d, <i>J</i> = 8.5 Hz, 1H), 5.48 (s, 1H), 5.15–5.21 (m, 1H), 4.80 (t, <i>J</i> = 6.9 Hz, 2H), 4.33 (dd, <i>J</i> = 7.6, 4.9 Hz, 2H), 3.36 (s, 3H), 3.12 (q, <i>J</i> = 7.3 Hz, 2H), 1.23 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 493.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <i>N</i>-{4-(2,4-Difluorophenoxy)-3-[4-(2-hydroxy-2-methylpropoxy)-1-methyl-6-oxo-1,6-dihydropyridin-3-yl]phenyl}ethanesulfonamide (<b>30e</b>)</h3><div class="NLM_p last">Compound <b>30e</b> was prepared according to the procedure used for the preparation of <b>30b</b>, substituting 2-methylpropane-1,2-diol for 2,2,2-trifluoroethanol (0.013 g, 11%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.78 (s, 1H), 7.59 (s, 1H), 7.31 (m, 1H), 7.22 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.13 (d, <i>J</i> = 2.8 Hz, 1H), 6.97 (d, <i>J</i> = 8.9 Hz, 1H), 6.95–6.87 (m, 2H), 5.74 (s, 1H), 3.58 (s, 2H), 3.08 (q, <i>J</i> = 7.3 Hz, 2H), 2.50 (s, 3H), 1.21 (t, <i>J</i> = 7.3 Hz, 3H), 1.01 (s, 6H). MS (ESI) 509.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <i>N</i>-(4-(2,4-Difluorophenoxy)-3-(1-methyl-4-(2-morpholinoethoxy)-6-oxo-1,6-dihydropyridin-3-yl)phenyl)ethanesulfonamide (<b>30f</b>)</h3><div class="NLM_p last">Compound <b>30f</b> was prepared according to the procedure used for the preparation of <b>30b</b>, substituting 2-morpholinoethanol for 2,2,2-trifluoroethanol to give the title compound (0.017 g, 28%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.81 (s, 1H), 7.65 (s, 1H), 7.39–7.31 (m, 1H), 7.21–7.10 (m, 2H), 7.04–6.91 (m, 3H), 5.93 (s, 1H), 4.26 (t, <i>J</i> = 4.6 Hz, 2H), 4.21–3.55 (m, 6H), 3.34 (s, 3H), 3.28–2.85 (m, 6H), 1.22 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 550.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 4-Chloro-5-(2-fluoro-5-nitrophenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>56</b>)</h3><div class="NLM_p last">A mixture of 2-(2-fluoro-5-nitrophenyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (<b>6a</b>) (6.41 g, 24.0 mmol), compound <b>16</b> (4.45 g, 20.0 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (0.348 g, 1.20 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.549 g, 0.600 mmol), and cesium fluoride (9.11 g, 60.0 mmol) in THF (80 mL) was subjected to vacuum and nitrogen gas cycle several times. The reaction mixture was heated at 45 °C overnight. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. The residue was purified by flash chromatography (silica gel, 4:6 acetone/heptanes) to give the title compound (4.5 g, 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.46–8.24 (m, 2H), 8.10 (s, 1H), 7.65 (t, <i>J</i> = 9.0 Hz, 1H), 6.72 (s, 1H), 3.45 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 283.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 4-Chloro-5-(5-(ethylsulfonyl)-2-(2-iodophenoxy)phenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>57a</b>)</h3><div class="NLM_p last">A mixture of compound <b>53</b> (0.46 g, 1.4 mmol), 2-iodophenol (0.307 g, 1.34 mmol), and cesium carbonate (0.454 g, 1.34 mmol) in DMSO (10 mL) was heated at 100 °C overnight. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was separated and extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash chromatography (silica gel, 1:1 ethyl acetate/hexanes) to give the title compound (0.46 g, 62%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.04 (s, 1H), 7.97–7.85 (m, 3H), 7.46 (td, <i>J</i> = 7.7, 1.6 Hz, 1H), 7.13 (dd, <i>J</i> = 8.1, 1.4 Hz, 1H), 7.06 (td, <i>J</i> = 7.6, 1.4 Hz, 1H), 6.92 (d, <i>J</i> = 8.5 Hz, 1H), 6.68 (s, 1H), 3.48 (s, 3H), 3.28–3.32 (m, 2H), 1.19–1.10 (m, 3H). MS (ESI+) <i>m</i>/<i>z</i> 530.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 4-Chloro-5-(2-(4-chloro-2-iodophenoxy)-5-(ethylsulfonyl)phenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>57b</b>)</h3><div class="NLM_p last">A mixture of compound <b>53</b> (0.50 g, 1.5 mmol), 4-chloro-2-iodophenol (0.386 g, 1.52 mmol), and cesium carbonate (0.494 g, 1.52 mmol) in DMSO (10 mL) was heated at 100 °C overnight. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate three times. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by prep HPLC (C18, 10–70% CH<sub>3</sub>CN/0.1% TFA water) to afford title compound (0.055 g, 6.4%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.00 (s, 1H), 7.97 (d, <i>J</i> = 2.6 Hz, 1H), 7.90–7.82 (m, 2H), 7.48 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.18–7.10 (m, 1H), 7.08 (d, <i>J</i> = 8.7 Hz, 1H), 6.96 (d, <i>J</i> = 8.4 Hz, 1H), 6.87 (d, <i>J</i> = 8.5 Hz, 1H), 6.64 (s, 1H), 3.43 (s, 3H), 3.30 (q, <i>J</i> = 7.3 Hz, 2H), 1.14–1.05 (m, 3H). MS (ESI+) <i>m</i>/<i>z</i> 564.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 4-Chloro-5-(2-(2,4-difluoro-6-iodophenoxy)-5-nitrophenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>57c</b>)</h3><div class="NLM_p last">A mixture of compound <b>56</b> (0.30 g, 1.0 mmol), 2,4-difluoro-6-iodophenol (0.258 g, 1.01 mmol), and cesium carbonate (0.493 g, 1.51 mmol) in DMSO (5 mL) was heated at 85 °C overnight. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was separated and extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash chromatography on silica gel, eluting with 1:1 ethyl acetate/hexanes to give the title compound (0.36 g, 69%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.29 (d, <i>J</i> = 2.9 Hz, 1H), 8.23 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 8.02 (s, 1H), 7.76 (ddd, <i>J</i> = 7.7, 2.9, 1.8 Hz, 1H), 7.61 (ddd, <i>J</i> = 10.4, 8.7, 2.9 Hz, 1H), 6.85 (dd, <i>J</i> = 9.2, 1.0 Hz, 1H), 6.71 (s, 1H), 3.48 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 519.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 4-Chloro-5-(2-(4-chloro-2-iodophenoxy)-5-nitrophenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>57d</b>)</h3><div class="NLM_p last">A mixture of compound <b>56</b> (2.83 g, 10.0 mmol), 4-chloro-2-iodophenol (3.46 g, 13.6 mmol), and cesium carbonate (4.43 g, 13.6 mmol) in DMSO (40 mL) was heated at 70 °C for 2 h. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash column chromatography on silica gel, eluting with 1:1 ethyl acetate/heptanes to afford the title compound (3.85 g, 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.31 (d, <i>J</i> = 2.8 Hz, 1H), 8.31–8.15 (m, 1H), 8.06 (s, 1H), 8.03 (d, <i>J</i> = 2.5 Hz, 1H), 7.55 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 1H), 6.93 (d, <i>J</i> = 9.1 Hz, 1H), 6.70 (s, 1H), 3.47 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 517.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 4-Chloro-5-{5-(ethylsulfonyl)-2-[2-(3-hydroxyprop-1-yn-1-yl)phenoxy]phenyl}-1-methylpyridin-2(1<i>H</i>)-one (<b>58a</b>)</h3><div class="NLM_p last">A mixture of compound <b>57a</b> (0.10 g, 0.19 mmol), prop-2-yn-1-ol (0.021 g, 0.38 mmol), copper(I) iodide (0.007 g, 0.038 mmol), triethylamine (0.540 mL, 3.78 mmol), and bis(triphenylphosphine)palladium(II) chloride (0.013 g, 0.019 mmol) in DMF (2 mL) was heated at 80 °C for 2 h. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was separated and extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash chromatography (silica gel, 2:1 ethyl acetate/hexanes) to give the title compound (0.055 g, 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.99 (s, 1H), 7.92–7.66 (m, 2H), 7.60–7.35 (m, 2H), 7.26 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.11 (dd, <i>J</i> = 8.3, 1.1 Hz, 1H), 6.94 (d, <i>J</i> = 8.7 Hz, 1H), 6.66 (s, 1H), 4.17 (s, 2H), 3.47 (s, 3H), 3.31 (q, <i>J</i> = 7.3 Hz, 2H), 1.13 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 457.9 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 4-Chloro-5-(5-(ethylsulfonyl)-2-(2-(4-hydroxybut-1-yn-1-yl)phenoxy)phenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>58b</b>)</h3><div class="NLM_p last">Compound <b>58b</b> was prepared in a similar manner described for the synthesis of compound <b>58a</b>, substituting but-3-yn-1-ol for prop-2-yn-1-ol (0.078 g, 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.94 (s, 1H), 7.87–7.77 (m, 2H), 7.51–7.33 (m, 2H), 7.20 (td, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.08 (dd, <i>J</i> = 8.2, 1.1 Hz, 1H), 6.87 (d, <i>J</i> = 8.7 Hz, 1H), 6.64 (s, 1H), 4.77 (t, <i>J</i> = 5.6 Hz, 2H), 3.43 (s, 3H), 3.37–3.22 (m, 4H), 2.37 (t, J = 6.9 Hz, 2H), 1.16–1.01 (m, 3H). MS (ESI+) <i>m</i>/<i>z</i> 472.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 4-Chloro-5-(2-(4-chloro-2-(4-hydroxybut-1-yn-1-yl)phenoxy)-5-(ethylsulfonyl)phenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>58c</b>)</h3><div class="NLM_p last">Compound <b>58c</b> was prepared in a similar manner described for the synthesis of compound <b>58a</b>, substituting but-3-yn-1-ol for prop-2-yn-1-ol, and compound <b>57b</b> for compound <b>57a</b>, respectively (0.072 g, 100%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.94 (s, 1H), 7.88–7.78 (m, 2H), 7.53 (d, <i>J</i> = 2.6 Hz, 1H), 7.42 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.09 (d, <i>J</i> = 8.8 Hz, 1H), 6.96 (d, <i>J</i> = 8.6 Hz, 1H), 6.64 (s, 1H), 4.80 (t, <i>J</i> = 5.6 Hz, 1H), 3.42 (s, 3H), 3.38–3.23 (m, 4H), 2.47–2.34 (m, 2H), 1.15–1.05 (m, 3H). MS (ESI+) <i>m</i>/<i>z</i> 506.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 4-Chloro-5-(2-(2,4-difluoro-6-(4-hydroxybut-1-yn-1-yl)phenoxy)-5-nitrophenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>58d</b>)</h3><div class="NLM_p last">Compound <b>58d</b> was prepared in a similar manner described for the synthesis of compound <b>58a</b>, substituting but-3-yn-1-ol for prop-2-yn-1-ol, and compound <b>57c</b> for compound <b>57a</b>, respectively (0.42 g, 47%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.28–8.17 (m, 2H), 7.98 (s, 1H), 7.54 (ddd, <i>J</i> = 10.5, 8.8, 3.0 Hz, 1H), 7.29 (ddd, <i>J</i> = 8.8, 3.1, 1.7 Hz, 1H), 6.86 (dd, <i>J</i> = 9.0, 0.9 Hz, 1H), 6.68 (s, 1H), 4.69 (s, 1H), 3.32 (s, 3H), 3.31 (t, <i>J</i> = 6.7 Hz, 3H), 2.38 (t, <i>J</i> = 6.7 Hz, 2H). MS (ESI+) <i>m</i>/<i>z</i> 461.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 4-Chloro-5-(2-(4-chloro-2-(4-hydroxybut-1-yn-1-yl)phenoxy)-5-nitrophenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>58e</b>)</h3><div class="NLM_p last">Compound <b>58e</b> was prepared in a similar manner described for the synthesis of compound <b>58a</b>, substituting but-3-yn-1-ol for prop-2-yn-1-ol, and compound <b>57d</b> for compound <b>57a</b>, respectively (0.76 g, 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.29–8.17 (m, 2H), 7.96 (s, 1H), 7.92 (s, 1H), 7.55 (d, <i>J</i> = 2.7 Hz, 1H), 7.45 (dd, <i>J</i> = 8.8, 2.6 Hz, 1H), 7.15 (d, <i>J</i> = 8.7 Hz, 1H), 6.89 (d, <i>J</i> = 8.8 Hz, 1H), 6.65 (s, 1H), 4.78 (t, <i>J</i> = 5.6 Hz, 1H), 3.43 (s, 3H), 3.32–3.35 (m, 2H), 2.37 (t, <i>J</i> = 6.8 Hz, 2H). MS (ESI+) <i>m</i>/<i>z</i> 459.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 4-Chloro-5-(5-(ethylsulfonyl)-2-(2-(3-hydroxypropyl)phenoxy)phenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>59a</b>)</h3><div class="NLM_p last">Compound <b>58a</b> (0.030 g, 0.066 mmol) and THF (10 mL) were added to 5% Pt/C wet, RD0585 (0.016 g, 0.033 mmol) in a 50 mL pressure bottle and stirred at 50 psi of hydrogen and ambient temperature until all the starting material was consumed. The solid was filtered, and the filtrate was concentrated under reduced pressure. The residue was purified by flash chromatography (silica gel, 100:5 ethyl acetate/methanol) to give the title compound (0.018 g, 60%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.04 (s, 1H), 7.92–7.81 (m, 2H), 7.40–7.16 (m, 3H), 7.07–6.96 (m, 1H), 6.84 (dd, <i>J</i> = 14.4, 8.6 Hz, 1H), 6.69 (s, 1H), 3.47 (s, 3H), 3.37–3.26 (m, 4H), 1.62–1.50 (m, 2H), 1.14 (t, <i>J</i> = 7.3 Hz, 5H). MS (ESI+) <i>m</i>/<i>z</i> 462.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 4-Chloro-5-(5-(ethylsulfonyl)-2-(2-(4-hydroxybutyl)phenoxy)phenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>59b</b>)</h3><div class="NLM_p last">Compound <b>59b</b> was prepared in a similar manner described for the synthesis of compound <b>59a</b>, substituting compound <b>58b</b> for compound <b>58a</b> (0.055 g, 84%). <sup>1</sup>H NMR (501 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.02 (s, 1H), 7.89–7.80 (m, 2H), 7.34 (td, <i>J</i> = 7.9, 1.8 Hz, 1H), 7.26 (tdd, <i>J</i> = 7.8, 5.5, 1.8 Hz, 1H), 7.18 (tdd, <i>J</i> = 7.4, 3.8, 1.3 Hz, 1H), 6.97 (dd, <i>J</i> = 8.0, 1.2 Hz, 1H), 6.86 (t, <i>J</i> = 8.8 Hz, 1H), 6.67 (s, 1H), 3.43 (s, 3H), 3.33–3.23 (m, 4H), 2.44 (t, <i>J</i> = 7.6 Hz, 2H), 1.53–1.36 (m, 2H), 1.31 (dt, <i>J</i> = 8.4, 6.4 Hz, 2H), 1.12 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 476.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 4-Chloro-5-(2-(4-chloro-2-(4-hydroxybutyl)phenoxy)-5-(ethylsulfonyl)phenyl)-1-methylpyridin-2(1<i>H</i>)-one (<b>59c</b>)</h3><div class="NLM_p last">Compound <b>59c</b> was prepared in a similar manner described for the synthesis of compound <b>59a</b>, substituting compound <b>58c</b> for compound <b>58a</b> (0.060 g, 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.98 (s, 1H), 7.88–7.78 (m, 2H), 7.46–7.36 (m, 1H), 7.28 (dd, <i>J</i> = 8.6, 2.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.5 Hz, 1H), 6.91 (d, <i>J</i> = 8.5 Hz, 1H), 6.65 (s, 1H), 3.56 (m, 2H), 3.42 (s, 3H), 3.31–3.20 (m, 2H), 2.42 (t, <i>J</i> = 7.4 Hz, 2H), 1.38 (q, <i>J</i> = 7.9 Hz, 2H), 1.32–1.17 (m, 2H), 1.10 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 510.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 5-(5-Amino-2-(2,4-difluoro-6-(4-hydroxybutyl)phenoxy)phenyl)-4-chloro-1-methylpyridin-2(1<i>H</i>)-one (<b>59d</b>)</h3><div class="NLM_p last">Compound <b>59d</b> was prepared in a similar manner described for the synthesis of compound <b>59a</b>, substituting compound <b>58d</b> for compound <b>58a</b> (0.26 g, 66%). <sup>1</sup>H NMR (501 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.86 (s, 1H), 7.15 (ddd, <i>J</i> = 10.7, 8.7, 3.0 Hz, 1H), 7.00 (ddd, <i>J</i> = 9.3, 3.1, 1.6 Hz, 1H), 6.61 (s, 1H), 6.50–6.42 (m, 2H), 6.26–6.19 (m, 1H), 4.87 (s, 2H), 4.31 (t, <i>J</i> = 5.2 Hz, 1H), 3.44 (s, 3H), 3.28 (td, <i>J</i> = 6.4, 5.1 Hz, 2H), 2.48–2.42 (m, 2H), 1.41 (tt, <i>J</i> = 8.9, 6.8 Hz, 2H), 1.34–1.25 (m, 2H). MS (ESI+) <i>m</i>/<i>z</i> 435.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 5-(5-Amino-2-(4-chloro-2-(4-hydroxybutyl)phenoxy)phenyl)-4-chloro-1-methylpyridin-2(1<i>H</i>)-one (<b>59e</b>)</h3><div class="NLM_p last">Compound <b>59e</b> was prepared in a similar manner described for the synthesis of compound <b>59a</b>, substituting compound <b>58e</b> for compound <b>58a</b> (0.40 g, 59%). <sup>1</sup>H NMR (501 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.91 (s, 1H), 7.38–7.31 (m, 1H), 7.29 (dd, <i>J</i> = 8.7, 2.8 Hz, 1H), 7.23 (td, <i>J</i> = 4.3, 2.6 Hz, 2H), 6.95 (dd, <i>J</i> = 8.7, 7.3 Hz, 1H), 6.81–6.74 (m, 1H), 6.65 (d, <i>J</i> = 6.1 Hz, 1H), 4.32 (t, <i>J</i> = 6.5 Hz, 2H), 3.42 (s, 3H), 2.51–2.52 (m, 2H), 1.67–1.59 (m, 2H), 1.57–1.49 (m, 2H). MS (ESI+) <i>m</i>/<i>z</i> 433.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 3-(Ethylsulfonyl)-6-methyl-11,12-dihydro-10<i>H</i>-dibenzo[4,5:7,8][1,6]dioxacycloundecino[3,2-<i>c</i>]pyridin-7(6<i>H</i>)-one (<b>60a</b>)</h3><div class="NLM_p last">A mixture of compound <b>59a</b> (0.018 g, 0.039 mmol) and 60% sodium hydride (0.005 g, 0.12 mmol) in 1,4-dioxane (3 mL) was heated at 90 °C overnight. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was separated and extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by prep HPLC (C18, 10–70% acetonitrile/water (0.1% TFA)) to afford the title compound (0.008 g, 48%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.84–7.85 (m, 2H), 7.76 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.43 (d, <i>J</i> = 7.0 Hz, 1H), 7.27–7.37 (m, 2H), 7.11 (d, <i>J</i> = 7.6 Hz, 1H), 6.63 (d, <i>J</i> = 8.5 Hz, 1H), 6.14 (s, 1H), 4.38–4.40 (m, 1H), 3.56–3.61 (m, 2H), 3.44 (s, 3H), 3.30 (q, <i>J</i> = 7.3 Hz, 2H), 2.57–2.63 (m, 1H), 2.10–2.14 (m, 1H), 1.61–1.66 (M, 1H), 1.14 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 426.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 17-(Ethylsulfonyl)-2-methyl-6,7,8,9-tetrahydrodibenzo[4,5:7,8][1,6]dioxacyclododecino[3,2-<i>c</i>]pyridin-3(2<i>H</i>)-one (<b>60b</b>)</h3><div class="NLM_p last">Compound <b>60b</b> was prepared in a similar manner described for the synthesis of compound <b>60a</b>, substituting compound <b>59b</b> for compound <b>59a</b> (0.011 g, 24%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.74–7.48 (m, 3H), 7.40 (dd, <i>J</i> = 7.3, 1.5 Hz, 1H), 7.31–7.35 (m, 1H), 7.23–7.26 (m, 1H), 7.11 (d, <i>J</i> = 7.0 Hz, 1H), 6.70 (d, <i>J</i> = 8.5 Hz, 1H), 5.91 (s, 1H), 4.09–4.16 (m, 2H), 3.43 (s, 3H), 3.28 (q, <i>J</i> = 7.3 Hz, 2H), 2.40–2.55 (m, 2H), 1.7 (m, 3H), 1.38–1.48 (m, 1H), 1.14 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 440.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 11-Chloro-17-(ethylsulfonyl)-2-methyl-6,7,8,9-tetrahydrodibenzo[4,5:7,8][1,6]dioxacyclododecino[3,2-<i>c</i>]pyridin-3(2<i>H</i>)-one (<b>60c</b>)</h3><div class="NLM_p last">Compound <b>60c</b> was prepared in a similar manner described for the synthesis of compound <b>60a</b>, substituting compound <b>59c</b> for compound <b>59a</b> (0.018 g, 32%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.75–7.79 (m, 3H), 7.51 (d, <i>J</i> = 2.7 Hz, 1H), 7.38 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 7.17 (d, <i>J</i> = 8.5 Hz, 1H), 6.74 (d, <i>J</i> = 8.5 Hz, 1H), 5.91 (s, 1H), 4.07–4.16 (m, 1H), 3.43 (s, 3H), 3.30 (q, <i>J</i> = 7.3 Hz, 2H), 2.42–2.54 (m, 2H), 2.43–2.48 (m, 1H), 1.69 (m, 3H), 1.36–1.38 (m, 1H), 1.14 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 474.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 17-Amino-11,13-difluoro-2-methyl-6,7,8,9-tetrahydrodibenzo[4,5:7,8][1,6]dioxacyclododecino[3,2-<i>c</i>]pyridin-3(2<i>H</i>)-one (<b>60d</b>)</h3><div class="NLM_p last">Compound <b>60d</b> was prepared in a similar manner described for the synthesis of compound <b>60a</b>, substituting compound <b>59d</b> for compound <b>59a</b> (0.039 g, 16%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.65 (s, 1H), 7.23 (ddd, <i>J</i> = 11.4, 8.8, 3.1 Hz, 1H), 7.08 (dt, <i>J</i> = 9.1, 2.2 Hz, 1H), 6.52–6.36 (m, 2H), 6.10 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 5.84 (s, 1H), 4.78 (s, 2H), 4.15–3.96 (m, 2H), 3.39 (s, 3H), 1.73 (br, 4H), 1.34 (br, 2H). MS (ESI+) <i>m</i>/<i>z</i> 399.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 17-Amino-11-chloro-2-methyl-6,7,8,9-tetrahydrodibenzo[4,5:7,8][1,6]dioxacyclododecino[3,2-<i>c</i>]pyridin-3(2<i>H</i>)-one (<b>60e</b>)</h3><div class="NLM_p last">Compound <b>60e</b> was prepared in a similar manner described for the synthesis of compound <b>60a</b>, substituting compound <b>59e</b> for compound <b>59a</b> (0.045 g, 12%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.96 (s, 2H), 7.65 (s, 1H), 7.47 (d, <i>J</i> = 2.6 Hz, 1H), 7.34 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 7.14–7.01 (m, 3H), 6.56–6.48 (m, 1H), 5.89 (s, 1H), 4.09 (s, 2H), 3.55 (s, 1H), 3.41 (s, 3H), 2.45–2.33 (m, 1H), 1.71–1.65 (m, 3H), 1.39–1.20 (m, 1H). MS (ESI+) <i>m</i>/<i>z</i> 397.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> <i>N</i>-(11,13-Difluoro-2-methyl-3-oxo-2,3,6,7,8,9-hexahydrodibenzo[4,5:7,8][1,6]dioxacyclododecino[3,2-<i>c</i>]pyridin-17-yl)ethanesulfonamide (<b>61a</b>)</h3><div class="NLM_p last">Compound <b>61a</b> was prepared in a similar manner described for the synthesis of compound <b>13b</b>, substituting compound <b>60d</b> for compound <b>12</b>, and ethanesulfonyl chloride for MsCl, respectively, to provide the title compound (0.028 g, 58%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.64 (s, 1H), 7.74 (s, 1H), 7.28–7.32 (m, 1H), 7.14 (d, <i>J</i> = 8.8 Hz, 1H), 7.10 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.06 (d, <i>J</i> = 2.7 Hz, 1H), 6.43 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 5.89 (s, 1H), 4.11 (m, 2H), 3.41 (s, 3H), 3.05–3.13 (m, 2H), 2.45–2.56 (m, 2H), (m, 1H), 1.65–1.70 (m, 3H), 1.35–1.39 (m, 1H), 1.22 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 491.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> <i>N</i>-(11-Chloro-2-methyl-3-oxo-2,3,6,7,8,9-hexahydrodibenzo[4,5:7,8][1,6]dioxacyclododecino[3,2-<i>c</i>]pyridin-17-yl)ethanesulfonamide (<b>61b</b>)</h3><div class="NLM_p last">Compound <b>61b</b> was prepared in a similar manner described for the synthesis of compound <b>13b</b>, substituting compound <b>60e</b> for compound <b>12</b>, and ethanesulfonyl chloride for MsCl, respectively, to provide the title compound (0.028 g, 81%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.60 (s, 1H), 7.67 (s, 1H), 7.45 (d, <i>J</i> = 2.7 Hz, 1H), 7.32 (dd, <i>J</i> = 8.5, 2.75 Hz, 1H), 7.06–7.10 (m, 3H), 6.47 (d, <i>J</i> = 8.8 Hz, 1H), 5.87 (s, 1H), 4.10 (t, <i>J</i> = 4.4 Hz, 2H), 3.41 (s, 3H), 3.04–3.11 (m, 2H), 2.42–2.50 (m, 2H), 1.67–1.70 (m, 3H), 1.34–1.36 (m, 1H), 1.22 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 489.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 5-(5-Amino-2-(4-chloro-2-iodophenoxy)phenyl)-4-chloro-1-methylpyridin-2(1<i>H</i>)-one (<b>62</b>)</h3><div class="NLM_p last">Compound <b>62</b> was prepared in a similar manner described for the synthesis of compound <b>8</b>, substituting compound <b>57d</b> for compound <b>7</b>, to provide the title compound (0.49 g, 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.18–7.72 (m, 2H), 7.27 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 6.89 (d, <i>J</i> = 8.6 Hz, 1H), 6.66 (dd, <i>J</i> = 8.7, 2.7 Hz, 1H), 6.61–6.41 (m, 3H), 6.61–6.41 (m, 3H), 5.23 (s, 2H), 3.36 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 487.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> <i>N</i>-(3-(4-Chloro-1-methyl-6-oxo-1,6-dihydropyridin-3-yl)-4-(4-chloro-2-iodophenoxy)phenyl)ethanesulfonamide (<b>63</b>)</h3><div class="NLM_p last">Compound <b>63</b> was prepared in a similar manner described for the synthesis of compound <b>13b</b>, substituting compound <b>62</b> for compound <b>12</b>, and ethanesulfonyl chloride for MsCl, respectively, to provide the title compound (0.83 g, 82%). <sup>1</sup>H NMR (501 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.93 (s, 1H), 7.90–7.82 (m, 2H), 7.33 (ddd, <i>J</i> = 21.5, 8.8, 2.7 Hz, 2H), 7.21 (d, <i>J</i> = 2.7 Hz, 1H), 7.08 (d, <i>J</i> = 8.8 Hz, 1H), 6.69 (d, <i>J</i> = 8.8 Hz, 1H), 6.57 (s, 1H), 3.41 (s, 3H), 3.14 (q, <i>J</i> = 7.3 Hz, 2H), 1.24 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 579.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> <i>N</i>-(4-(4-Chloro-2-iodophenoxy)-3-(1-methyl-6-oxo-4-(pent-4-en-1-yloxy)-1,6-dihydropyridin-3-yl)phenyl)ethanesulfonamide (<b>64</b>)</h3><div class="NLM_p last">A mixture of compound <b>63</b> (0.11 g, 0.19 mmol), pent-4-en-1-ol (0.049 g, 0.57 mmol), and sodium hydride (0.0460 g, 1.14 mmol) in 1,4-dioxane (3 mL) was heated at 90 °C overnight. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The pH of aqueous layer was adjusted to 5 with addition of 10% aqueous HCl. The aqueous layer was separated and extracted with additional ethyl acetate twice. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash chromatography on silica gel, eluting with 100:5 ethyl acetate/methanol to give the title compound (0.09 g, 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.85 (s, 1H), 7.81 (d, <i>J</i> = 2.6 Hz, 1H), 7.56 (s, 1H), 7.24–7.30 (m, 2H), 7.19 (d, <i>J</i> = 2.1 Hz, 1H), 7.11 (d, <i>J</i> = 8.8 Hz, 1H), 6.53 (d, <i>J</i> = 8.8 Hz, 1H), 5.83–5.57 (m, 2H), 5.51, 5.00–4.87 (m, 2H), 3.81 (q, <i>J</i> = 6.2 Hz, 2H), 3.11 (q, <i>J</i> = 7.4 Hz, 2H), 2.04–1.88 (m, 2H), 1.62 (dt, <i>J</i> = 8.3, 6.6 Hz, 2H), 1.22 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 628.9 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> <i>N</i>-[(8<i>E</i>)-12-Chloro-2-methyl-3-oxo-2,3,7,10-tetrahydro-6<i>H</i>-dibenzo[4,5:7,8][1,6]dioxacyclotridecino[3,2-<i>c</i>]pyridin-18-yl]ethanesulfonamide (<b>65a</b>)</h3><div class="NLM_p last">A mixture of compound <b>64</b> (0.062 g, 0.099 mmol), triethylamine (0.050 g, 0.49 mmol), and tetrakis(triphenylphosphine)palladium(0) (0.046 g, 0.039 mmol) in DMF (9 mL) was degassed and backfilled with nitrogen several times. The reaction mixture was heated at 100 °C overnight. After cooling to ambient temperature, the reaction mixture was partitioned between water and ethyl acetate. The aqueous layer was extracted with additional ethyl acetate several times. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by column flash chromatography on silica gel, eluting with 2% methanol in ethyl acetate to afford the crude product. This material was further purified by reverse phase prep HPLC (C18, 10–70% CH<sub>3</sub>CN/0.1% TFA water) to give the title compound (0.013 g, 29%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86 (s, 1H), 7.49 (s, 1H), 7.24–7.26 (m, 2H), 7.03–7.06 (m, 2H), 6.98 (d, <i>J</i> = 8.8 Hz, 1H), 6.20 (d, <i>J</i> = 8.8 Hz, 1H), 5.80 (s, 1H), 5.50 (m, 1H), 5.09 (m, 1H), 4.27 (m, 1H), 4.10 (m, 1H), 3.24 (s, 3H), 3.10–3.16 (m, 2H), 2.63 (m, 1H), 2.20 (m, 1H), 1.22–1.26 (m, 3H). MS (APCI+) <i>m</i>/<i>z</i> 501.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> (<i>E</i>)-<i>N</i>-(12-Chloro-2-methyl-3-oxo-3,6,7,8-tetrahydro-2<i>H</i>-dibenzo[4,5:7,8][1,6]dioxacyclotridecino[3,2-<i>c</i>]pyridin-18-yl)ethanesulfonamide (<b>65b</b>)</h3><div class="NLM_p last">Compound <b>65b</b> was isolated as the second eluting fraction in the preparation of compound <b>65a</b> (0.012 g, 30%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.60 (s, 1H), 7.62 (s, 1H), 7.40 (d, <i>J</i> = 2.7 Hz, 1H), 7.28 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 7.12 (dd, <i>J</i> = 8.7, 2.9 Hz, 1H), 7.04 (d, <i>J</i> = 2.7 Hz, 1H), 7.00 (d, <i>J</i> = 8.8 Hz, 1H), 6.52 (d, <i>J</i> = 8.8 Hz, 1H), 6.23–6.27 (m, 1H), 6.06–6.16 (m, 1H), 5.82 (s, 1H), 4.07–4.07 (m, 1H), 3.91–3.96 (m, 1H), 3.36 (s, 3H), 3.05–3.11 (m, 2H), 2.36–2.40 (m, 1H), 2.25–2.27 (m, 1H), 1.91–1.95 (m, 1H), 1.67–1.69 (m, 1H), 1.20–1.23 (m, 3H). MS (ESI+) <i>m</i>/<i>z</i> 501.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> <i>N</i>-(12-Chloro-2-methyl-3-oxo-2,3,7,8,9,10-hexahydro-6<i>H</i>-dibenzo[4,5:7,8][1,6]dioxacyclotridecino[3,2-<i>c</i>]pyridin-18-yl)ethanesulfonamide (<b>66</b>)</h3><div class="NLM_p last">Compound <b>66</b> was prepared in a similar manner described for the synthesis of compound <b>59a</b>, substituting compound <b>65a</b> for compound <b>58a</b>, to provide the title compound (0.068 g, 85%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.59 (s, 1H), 7.64 (s, 1H), 7.45 (d, <i>J</i> = 2.7 Hz, 1H), 7.28 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 7.07 (dd, <i>J</i> = 8.7, 2.7, Hz, 1H), 7.03 (d, <i>J</i> = 2.4 hz, 1H), 6.91 (d, <i>J</i> = 8.5 Hz, 1H), 6.29 (d, <i>J</i> = 8.8 Hz, 1H), 5.88 (s, 1H), 4.05 (t, <i>J</i> = 9.5 Hz, 1H), 3.92–3.95 (m, 1H), 3.40 (s, 3H), 3.07 (q, <i>J</i> = 7.3 Hz, 2H), 2.62–2.67 (m, 1H), 2.17–2.22 (m, 1H), 1.77–1.80 (m, 1H), 1.65–1.67 (m, 1H), 1.44–1.53 (m, 4H), 1.21 (t, <i>J</i> = 7.5 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 503.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 4-(3,5-Difluoro-2-methoxyphenyl)but-3-yn-1-ol (<b>68a</b>)</h3><div class="NLM_p last">A mixture of compound <b>67a</b> (4.3 g, 19 mmol), copper(I) iodide (0.367 g, 1.93 mmol), but-3-yn-1-ol (2.70 g, 38.6 mmol), bis(triphenylphosphine)palladium(II) chloride (0.677 g, 0.964 mmol), and triethylamine (53.7 mL, 386 mmol) in DMF (40 mL) was heated at 80 °C for 2 h. The reaction mixture was cooled to room temperature and diluted with ethyl acetate (250 mL) and then filtered through Celite and washed with additional ethyl acetate (200 mL, several times). The organic layer was washed with saturated aqueous sodium chloride twice (100 mL), dried over anhydrous sodium sulfate, filtered, and concentrated. The residue was purified by flash chromatography (silica gel, petroleum ether/ethyl acetate = 3/1) to give the title compound (3.6 g, 83%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, dhloroform-<i>d</i>) δ 6.93–6.73 (m, 2H), 3.96 (s, 3H), 3.85 (t, <i>J</i> = 6.4 Hz, 2H), 2.75 (t, <i>J</i> = 6.2 Hz, 2H), 2.21–2.14 (m, 1H). MS (ESI+) <i>m</i>/<i>z</i> (M + H)<sup>+</sup> ion was not obversed. LC/MS (APCI) <i>m</i>/<i>z</i> 213.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> 4-(5-Chloro-2-methoxyphenyl)but-3-yn-1-ol (<b>68b</b>)</h3><div class="NLM_p last">A solution of compound <b>67b</b> (2.0 g, 7.5 mmol), copper(I) iodide (0.142 g, 0.745 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (0.430 g, 0.372 mmol) in pyrrolidine (15 mL) was degassed with nitrogen. Separately, a solution of but-3-yn-1-ol (1.04 g, 14.9 mmol) in pyrrolidine (10 mL) was also degassed. The alkynol solution was added dropwise via syringe to the solution of the iodide and catalysts. The reaction mixture was stirred for 10 min at ambient temperature and then heated to 80 °C for 2 h. After cooling, the reaction mixture was poured into saturated aqueous ammonium chloride (300 mL) and extracted with ethyl acetate (3 × 150 mL). The combined extracts were washed with 1 N HCl (150 mL), water (150 mL), and saturated aqueous sodium chloride (150 mL), then dried over anhydrous sodium sulfate, filtered, and concentrated. The residue was purified by flash chromatography (silica gel, 0–50% ethyl acetate/heptanes) to give the title compound (1.38 g, 88%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.34 (dd, <i>J</i> = 6.9, 2.7 Hz, 2H), 7.09–6.98 (m, 1H), 4.87 (t, <i>J</i> = 5.7 Hz, 1H), 3.80 (s, 3H), 3.57 (td, <i>J</i> = 6.9, 5.6 Hz, 2H), 2.56 (t, <i>J</i> = 6.9 Hz, 2H). LC/MS <i>m</i>/<i>z</i> 211.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> 4-(2-(Benzyloxy)-3,5-difluorophenyl)but-3-yn-1-ol (<b>68c</b>)</h3><div class="NLM_p last">Compound <b>67c</b> (1.8 g, 6.0 mmol), but-3-yn-1-ol (0.844 g, 12.0 mmol), and triethylamine (8.39 mL, 60.2 mmol) were combined in DMF (15 mL) and degassed for 10 min. Pd(PPh<sub>3</sub>)<sub>4</sub> (0.348 g, 0.301 mmol) and copper(I) iodide (0.115 g, 0.602 mmol) were added together and degassed for another 5 min. The flask was fitted with a Vigreux column and heated at 80 °C for 3 h. After cooling to ambient temperature, the reaction mixture was poured into water and DCM. The aqueous layer was separated and extracted with DCM twice. The combined organics were dried over anhydrous magnesium sulfate, filtered, and concentrated. The residue was purified by flash chromatography (silica gel, 10–75% ethyl acetate/heptanes) to provide the title compound (1.67 g, 96%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.51–7.44 (m, 2H), 7.39 (ddt, <i>J</i> = 7.9, 6.2, 1.2 Hz, 2H), 7.39–7.30 (m, 2H), 7.11 (ddd, <i>J</i> = 9.0, 3.2, 1.8 Hz, 1H), 5.12 (s, 2H), 4.94 (t, <i>J</i> = 5.6 Hz, 1H), 3.60 (td, J = 6.8, 5.6 Hz, 2H), 2.61 (t, <i>J</i> = 6.8 Hz, 2H). MS (ESI+) <i>m</i>/<i>z</i> 305.9 (M + NH<sub>4</sub>)<sup>+</sup>.</div></div><div id="sec4_3_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> 4-(3,5-Difluoro-2-methoxyphenyl)butan-1-ol (<b>69a</b>)</h3><div class="NLM_p last">To the solution of compound <b>68a</b> (3.0 g, 14 mmol) in ethanol (40 mL) was added 10% Pd/C (3.01 g, 28.3 mmol) under argon, and then the mixture was degassed with hydrogen three times. The mixture was stirred at ambient temperature under hydrogen for 16 h. The reaction mixture was filtered through Celite and washed with ethyl acetate three times. The solvent was removed and the residue was azeotroped with toluene twice to remove additional water and dried under vacuum to give the title compound (2.6 g, 68%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 6.77–6.60 (m, 2H), 3.87 (s, 3H), 3.69 (t, <i>J</i> = 6.1 Hz, 2H), 2.67 (t, <i>J</i> = 7.3 Hz, 2H), 1.55–1.75 (m, 4H). LC/MS (APCI) <i>m</i>/<i>z</i> 217.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> 4-(5-Chloro-2-methoxyphenyl)butan-1-ol (<b>69b</b>)</h3><div class="NLM_p last">Compound <b>69b</b> was prepared in a similar manner described for the synthesis of compound <b>59a</b>, substituting compound <b>68b</b> for compound <b>58a</b>, to provide the title compound (8.52 g, 100%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.21–7.07 (m, 2H), 6.90 (d, <i>J</i> = 8.6 Hz, 1H), 4.35 (br, 1H), 3.73 (s, 3H), 1.58–1.15 (m, 6H). MS (ESI+) <i>m</i>/<i>z</i> 215.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> 2,4-Difluoro-6-(4-hydroxybutyl)phenol (<b>69c</b>)</h3><div class="NLM_p last">Compound <b>68c</b> (7.54 g, 26.2 mmol) was dissolved in THF (131 mL), and the solution was added to 20% Pd(OH)<sub>2</sub> (1.84 g, 2.62 mmol) in a 500 mL pressure bottle and stirred at ambient temperature for 16 h at 30 psi (H<sub>2</sub>). The mixture was filtered through a nylon membrane and the filtrate was concentrated by rotary evaporation to provide the title compound (5.18 g, 98%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.19 (s, 1H), 6.94 (ddd, <i>J</i> = 11.2, 8.7, 3.1 Hz, 1H), 6.75 (ddd, <i>J</i> = 9.4, 3.1, 1.8 Hz, 1H), 4.33 (s, 1H), 3.41–3.30 (m, 2H), 2.54 (t, <i>J</i> = 7.5 Hz, 2H), 1.58–1.45 (m, 2H), 1.48–1.33 (m, 2H). MS (ESI−) <i>m</i>/<i>z</i> 201.3 (M – H)<sup>−</sup>.</div></div><div id="sec4_3_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> 5-Bromo-4-(4-(5-chloro-2-methoxyphenyl)butoxy)-1-methylpyridin-2(1<i>H</i>)-one (<b>70</b>)</h3><div class="NLM_p last">Compound <b>69b</b> (10 g, 47 mmol) (azeotroped twice with toluene to remove adventitious water) and compound <b>16</b> (12.4 g, 55.9 mmol) were combined in anhydrous 1,4-dioxane (200 mL) and treated carefully with potassium <i>tert</i>-butoxide (1 N in THF, 60.6 mL, 60.6 mmol). The reaction mixture was stirred for 18 h, diluted with water (700 mL), and stirred for 1 h. The resulting solid was collected by filtration and triturated in diethyl ether (200 mL) to afford the title compound (14.1 g, 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.01 (s, 1H), 7.21 (d, <i>J</i> = 7.9 Hz, 2H), 6.96 (d, <i>J</i> = 8.4 Hz, 1H), 5.90 (s, 1H), 4.01 (t, <i>J</i> = 5.8 Hz, 2H), 3.77 (s, 3H), 3.33 (s, 3H), 2.60 (t, <i>J</i> = 7.0 Hz, 2H), 1.65–1.73 (m, 4H). MS (ESI+) <i>m</i>/<i>z</i> 401.9 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> 4-(4-(5-Chloro-2-methoxyphenyl)butoxy)-1-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-2(1<i>H</i>)-one (<b>71</b>)</h3><div class="NLM_p last">Dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine (0.297 g, 0.624 mmol), potassium acetate (1.41 g, 14.4 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.143 g, 0.156 mmol), compound <b>70</b> (2.5 g, 6.2 mmol), and 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane) (1.90 g, 7.49 mmol) were combined in anhydrous 1,4-dioxane (31.2 mL) and sparged with argon for 20 min. The mixture was then heated under nitrogen for 7 h at 80 °C, cooled, and partitioned between ethyl acetate and water. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered and concentrated. Purification by flash chromatography (silica gel, 25–50% of 3:1 ethyl acetate/ethanol in heptanes) afforded the title compound (1.9 g, 61%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.75 (s, 1H), 7.26–7.12 (m, 2H), 6.96 (d, <i>J</i> = 8.6 Hz, 1H), 5.70 (s, 1H), 3.99–3.85 (m, 2H), 3.77 (s, 3H), 3.136 (s, 3H), 2.59 (t, <i>J</i> = 7.3 Hz, 1H), 1.82–1.57 (m, 2H), 1.19 (s, 12H), 1.07 (s, 4H). MS (ESI+) <i>m</i>/<i>z</i> 448.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_63" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> 2′-Chloro-4-(4-(5-chloro-2-methoxyphenyl)butoxy)-1-methyl-5′-nitro-[3,3′-bipyridin]-6(1<i>H</i>)-one (<b>72a</b>)</h3><div class="NLM_p last">Compound <b>77a</b> (1.11 g, 4.67 mmol), compound <b>71</b> (1.90 g, 4.24 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.117 g, 0.127 mmol), 1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane (0.124 g, 0.424 mmol), and potassium phosphate (3.15 g, 14.9 mmol) were combined and sparged with argon for 15 min. Meanwhile a solution of 4:1 1,4-dioxane/water (25 mL) was sparged with nitrogen for 15 min and transferred by syringe into the reaction vessel under argon. The mixture was stirred for 2 h at 50 °C, cooled, and partitioned between ethyl acetate and water. The ethyl acetate layer was washed twice with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, treated with 3-mercaptopropyl functionalized silica gel for 20 min, filtered, and concentrated. Purification by flash chromatography (silica gel, 0.5–2.5% methanol in DCM) afforded the title compound (0.73 g, 32%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.22 (d, <i>J</i> = 2.7 Hz, 1H), 8.62 (d, <i>J</i> = 2.8 Hz, 1H), 7.83 (s, 1H), 7.18 (dd, <i>J</i> = 8.7, 2.7 Hz, 1H), 7.07 (d, <i>J</i> = 2.7 Hz, 1H), 6.93 (d, <i>J</i> = 8.8 Hz, 1H), 5.96 (s, 1H), 3.99 (t, <i>J</i> = 6.0 Hz, 2H), 3.73 (s, 3H), 3.40 (s, 3H), 2.46 (t, <i>J</i> = 7.2 Hz, 2H), 1.65–1.41 (m, 4H). MS (ESI+) <i>m</i>/<i>z</i> 478.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_64" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> 3-Bromo-2-chloro-5-(ethylsulfonyl)pyridine (<b>77b</b>)</h3><div class="NLM_p last">In a 500 mL round-bottomed flask were combined sodium sulfite (4.94 g, 39.2 mmol) and sodium bicarbonate (3.46 g, 41.2 mmol) in water (100 mL) to give a colorless solution. The mixture was heated to 75 °C. 3-Bromo-2-chloropyridine-5-sulfonyl chloride (6.00 g, 20.6 mmol) was added portionwise over 1 h. The reaction mixture was stirred at 75 °C for 1 h after complete addition. The mixture was concentrated under reduced pressure. To the reaction mixture were added DMF (37.5 mL), sodium bicarbonate (3.46 g, 41.2 mmol), and iodoethane (1.66 mL, 20.6 mmol). The resulting mixture was heated to 75 °C for 2 h and then cooled to ambient temperature. The mixture was partitioned between ethyl acetate and water. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous magnesium sulfate, filtered, and concentrated to a white solid. The solid was purified by flash chromatography (silica gel, 100% heptane to 40% ethyl acetate/heptane over 10 min, then to 100% ethyl acetate over 15 min) to give the title compound (3.15 g, 54%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.82 (d, <i>J</i> = 2.2 Hz, 1H), 8.64 (d, <i>J</i> = 2.3 Hz, 1H), 3.46 (q, <i>J</i> = 7.3 Hz, 2H), 1.12 (t, <i>J</i> = 7.4 Hz, 3H). MS (DCI+) <i>m</i>/<i>z</i> 302.9 (M + NH<sub>4</sub>)<sup>+</sup>.</div></div><div id="sec4_3_65" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> 2′-Chloro-4-(4-(5-chloro-2-methoxyphenyl)butoxy)-5′-(ethylsulfonyl)-1-methyl-[3,3′-bipyridin]-6(1<i>H</i>)-one (<b>72b</b>)</h3><div class="NLM_p last">Compound <b>77b</b> (0.818 g, 1.83 mmol), compound <b>71</b> (0.40 g, 1.4 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.045 g, 0.049 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (0.029 g, 0.098 mmol), and cesium fluoride (0.641 g, 4.22 mmol) were combined in anhydrous THF (14 mL), sparged with argon for 10 min, and stirred for 18 h at 50 °C. After cooling to ambient temperature, the reaction mixture was partitioned between ethyl acetate and water and filtered through Celite. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, treated with 3-mercaptopropyl functionalized silica, filtered, and concentrated. Purification by flash chromatography (silica gel, 30–60% of 3:1 ethyl acetate/ethanol in heptanes) afforded the title compound (0.36 g, 49%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.85 (d, <i>J</i> = 2.4 Hz, 1H), 8.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.84 (s, 1H), 7.18 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.09 (d, <i>J</i> = 2.7 Hz, 1H), 6.93 (d, <i>J</i> = 8.8 Hz, 1H), 5.95 (s, 1H), 4.10–3.93 (m, 2H), 3.73 (s, 3H), 3.62–3.39 (m, 5H), 1.68–1.40 (m, 4H), 1.14 (dt, <i>J</i> = 26.5, 7.2 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 525.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_66" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Methyl 2-Chloro-4′-(4-(5-chloro-2-methoxyphenyl)butoxy)-1′-methyl-6′-oxo-1′,6′-dihydro-[3,3′-bipyridine]-5-carboxylate (<b>72c</b>)</h3><div class="NLM_p last">Compound <b>71</b> (0.880 g, 1.96 mmol), methyl 5-bromo-6-chloronicotinate <b>77c</b> (0.541 g, 2.16 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.063 g, 0.069 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (0.040 g, 0.14 mmol), and cesium fluoride (0.896 g, 5.90 mmol) were combined in anhydrous THF (20 mL), sparged with argon for 10 min, and stirred for 3 h at 65 °C. The mixture was cooled to ambient temperature, partitioned between ethyl acetate and water, and filtered through Celite to remove elemental Pd. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, treated with 3-mercaptopropyl functionalized silica gel, filtered, and concentrated. The residue was purified by flash chromatography (silica gel, 35–60% 3:1 ethyl acetate/ethanol in heptanes) to afford the title compound (0.71 g, 74%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.88 (d, <i>J</i> = 2.4 Hz, 1H), 8.22 (d, <i>J</i> = 2.2 Hz, 1H), 7.79 (s, 1H), 7.18 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.04 (d, <i>J</i> = 2.7 Hz, 1H), 6.92 (d, <i>J</i> = 8.8 Hz, 1H), 5.93 (s, 1H), 4.13–3.89 (m, 2H), 3.87 (s, 3H), 3.72 (s, 3H), 22.44 (t, <i>J</i> = 7.3 Hz, 2H), 1.45–1.57 (m, 4H). MS (ESI+) <i>m</i>/<i>z</i> 491.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_67" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> 2′-Chloro-4-(4-(5-chloro-2-hydroxyphenyl)butoxy)-1-methyl-5′-nitro-[3,3′-bipyridin]-6(1<i>H</i>)-one (<b>73a</b>)</h3><div class="NLM_p last">To a solution of compound <b>72a</b> (0.790 g, 1.65 mmol) in DCM (17 mL) at −78 °C under nitrogen was added dropwise 1 M boron tribromide in DCM (8.26 mL, 8.26 mmol). The mixture was allowed to warm to 0 °C, stirred for 45 min, and carefully neutralized with saturated aqueous sodium bicarbonate to pH 10. The mixture was then diluted with DCM and water. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 50–100% ethyl acetate in heptanes) afforded the title compound (0.65 g, 85%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.48 (s, 1H), 9.21 (d, <i>J</i> = 2.7 Hz, 1H), 8.63 (d, <i>J</i> = 2.7 Hz, 1H), 7.83 (s, 1H), 7.04–6.93 (m, 2H), 6.74 (d, <i>J</i> = 8.5 Hz, 1H), 5.96 (s, 1H), 4.08–3.94 (m, 2H), 3.40 (s, 3H), 2.43 (t, <i>J</i> = 7.3 Hz, 2H), 1.63–1.44 (m, 4H). MS (ESI+) <i>m</i>/<i>z</i> 464.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_68" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> 2′-Chloro-4-(4-(5-chloro-2-hydroxyphenyl)butoxy)-5′-(ethylsulfonyl)-1-methyl-[3,3′-bipyridin]-6(1<i>H</i>)-one (<b>73b</b>)</h3><div class="NLM_p last">To a solution of compound <b>72b</b> (0.360 g, 0.685 mmol) in DCM (7 mL) at −78 °C under nitrogen was added dropwise 1 M boron tribromide in DCM (3.43 mL, 3.43 mmol). The mixture was allowed to warm to 0 °C, stirred for 45 min, and carefully neutralized with saturated aqueous sodium bicarbonate to pH 9. The mixture was then diluted with DCM and water and filtered to collect the title compound as a white solid that was dried to constant mass (0.29 g, 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.50 (s, 1H), 8.84 (d, <i>J</i> = 2.4 Hz, 1H), 8.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.84 (s, 1H), 7.00 (dd, <i>J</i> = 5.9, 2.8 Hz, 2H), 6.83–6.70 (m, 1H), 5.95 (s, 1H), 3.98 (t, <i>J</i> = 6.2 Hz, 2H), 3.53–3.38 (m, 5H), 2.43 (t, <i>J</i> = 7.4 Hz, 2H), 1.63–1.41 (m, 4H), 1.11 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 511.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_69" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Methyl 2-Chloro-4′-(4-(5-chloro-2-hydroxyphenyl)butoxy)-1′-methyl-6′-oxo-1′,6′-dihydro-[3,3′-bipyridine]-5-carboxylate (<b>73c</b>)</h3><div class="NLM_p last">To a solution of compound <b>72c</b> (0.710 g, 1.45 mmol) in DCM (14 mL) at −78 °C under nitrogen was added dropwise 1.0 M boron tribromide in DCM (7.22 mL, 7.22 mmol). The mixture was slowly allowed to warm to 0 °C over 4 h, recooled to −78 °C, and carefully treated with methanol (5 mL), added dropwise. The mixture was warmed to 0 °C and carefully neutralized to pH 7 by the addition of saturated aqueous sodium bicarbonate. The organic layer was washed with saturated aqueous sodium chloride, dried (anhydrous sodium sulfate), filtered, and concentrated. The resulting solid was triturated in 9:1 heptanes/ethyl acetate and collected by filtration to afford the title compound (0.61 g, 88%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.48 (s, 1H), 8.88 (d, <i>J</i> = 2.3 Hz, 1H), 8.23 (d, <i>J</i> = 2.2 Hz, 1H), 7.79 (s, 1H), 7.00 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 6.95 (d, <i>J</i> = 2.7 Hz, 1H), 6.74 (d, <i>J</i> = 8.5 Hz, 1H), 5.94 (s, 1H), 3.98 (t, <i>J</i> = 6.1 Hz, 2H), 3.87 (s, 3H), 3.39 (s, 3H), 2.42 (t, <i>J</i> = 7.4 Hz, 2H), 1.71–1.41 (m, 4H). MS (ESI+) <i>m</i>/<i>z</i> 477.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_70" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 11-Chloro-2-methyl-17-nitro-6,7,8,9-tetrahydrobenzo[11,12][1,6dioxacyclododecino[2,3-<i>b</i>:4,5-<i>c</i>′]dipyridin-3(2<i>H</i>)-one (<b>74a</b>)</h3><div class="NLM_p last">Compound <b>73a</b> (0.65 g, 1.4 mmol) and cesium carbonate (0.547 g, 1.68 mmol) were combined in acetonitrile (87 mL) under argon and heated at 75 °C for 1 h. The reaction mixture was cooled and concentrated to remove acetonitrile. The residue was partitioned between ethyl acetate and water. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 30–50% 3:1 ethyl acetate/ethanol in heptanes) afforded the title compound (0.46 g, 77%). LC/MS (APCI) <i>m</i>/<i>z</i> 427.6 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_71" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> 11-Chloro-17-(ethylsulfonyl)-2-methyl-6,7,8,9-tetrahydropyrido[3′,4′:4,5][1,6]benzodioxacyclododecino[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one (<b>74b</b>)</h3><div class="NLM_p last">Compound <b>73b</b> (0.29 g, 0.57 mmol) and cesium carbonate (0.24 g, 0.74 mmol) were combined in acetonitrile (50 mL) and DMSO (5 mL), heated at 95 °C for 2 h, and then cooled and concentrated. The residue was partitioned between ethyl acetate and water. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 1–4% methanol in DCM) afforded the title compound (0.237 g, 88%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.51 (d, <i>J</i> = 2.5 Hz, 1H), 8.16 (d, <i>J</i> = 2.4 Hz, 1H), 7.86 (s, 1H), 7.43 (d, <i>J</i> = 2.7 Hz, 1H), 7.32 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 7.14 (d, <i>J</i> = 8.7 Hz, 1H), 5.94 (s, 1H), 4.25–3.99 (m, 2H), 3.43 (s, 3H), 3.40 (t, <i>J</i> = 7.4 Hz, 2H), 2.65–2.53 (m, 1H), 2.39–2.24 (m, 1H), 1.81–1.53 (m, 3H), 1.51–1.32 (m, 1H), 1.17 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.5, 163.6, 163.0, 149.2, 146.9, 140.2, 140.1, 137.6, 130.1, 129.7, 129.5, 126.8, 125.2, 118.7, 107.2, 95.9, 67.3, 49.5, 35.9, 28.3, 26.7, 25.2, 7.0. MS (ESI+) <i>m</i>/<i>z</i> 475 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_72" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Methyl 11-Chloro-2-methyl-3-oxo-2,3,6,7,8,9-hexahydropyrido[3′,4′:4,5][1,6]benzodioxacyclododecino[2,3-<i>b</i>]pyridine-17-carboxylate (<b>74c</b>)</h3><div class="NLM_p last">Compound <b>73c</b> (0.60 g, 1.3 mmol) and cesium carbonate (0.491 g, 1.51 mmol) were combined in acetonitrile (80 mL) and DMSO (5 mL) under argon and heated at 85 °C for 5 h. The reaction mixture was cooled and concentrated to remove most of the acetonitrile. The concentrate was partitioned between ethyl acetate and water, and about 10 mL of methanol was added to give complete dissolution of solids. The aqueous layer was extracted three times with ethyl acetate. The organics were combined, washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 1–4% methanol in DCM) afforded the title compound (0.50 g, 90%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.60 (d, <i>J</i> = 2.4 Hz, 1H), 8.17 (d, <i>J</i> = 2.1 Hz, 1H), 7.83 (s, 1H), 7.41 (d, <i>J</i> = 2.7 Hz, 1H), 7.30 (dd, <i>J</i> = 8.5, 2.75 Hz, 1H), 7.11 (d, <i>J</i> = 8.8 Hz, 1H), 5.92 (s, 1H), 4.09–4.16 (m, 2H), 3.85 (s, 3H), 3.41 (s, 3H), 2.53–2.61 (m, 1H), 2.23–2.38 (m, 1H), 1.54–1.79 (m, 3H), 1.34–1.46 (m, 1H). MS (APCI+) <i>m</i>/<i>z</i> 441 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_73" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> 17-Amino-11-chloro-2-methyl-6,7,8,9-tetrahydropyrido[3′,4′:4,5][1,6]benzodioxacyclododecino[2,3-<i>b</i>]pyridin-3(2<i>H</i>)-one (<b>75</b>)</h3><div class="NLM_p last">Compound <b>74a</b> (0.40 g, 0.94 mmol), iron (0.261 g, 4.67 mmol), and ammonium chloride (0.100 g, 1.87 mmol) were combined in THF (10 mL), ethanol (10 mL), and water (3.3 mL) and heated at 90 °C for 2 h. The mixture was cooled, diluted with THF and methanol, and filtered through Celite to remove solids. The filtrate was concentrated. The residue was partitioned between ethyl acetate and saturated saturated aqueous sodium chloride. The ethyl acetate layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 1–5% methanol in DCM) afforded the title compound (0.35 g, 94%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.68 (s, 1H), 7.31–7.33 (m, 2H), 7.23 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 7.02 (d, <i>J</i> = 8.5 Hz, 1H), 6.94 (d, <i>J</i> = 2.7 Hz, 1H), 5.87 (s, 1H), 4.94 (s, 2H), 4.01–4.15 (m, 2H), 3.39 (s, 3H), 2.43–2.52 (m, 1H), 2.26–2.36 (m, <i>J</i> = 4.9 Hz, 1H), 1.60–1.77 (m, 3H), 1.30–1.40 (m, 1H). MS (ESI+) <i>m</i>/<i>z</i> 398 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_74" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> <i>N</i>-(11-Chloro-2-methyl-3-oxo-2,3,6,7,8,9-hexahydropyrido[3′,4′:4,5][1,6]benzodioxacyclododecino[2,3-<i>b</i>]pyridin-17-yl)ethanesulfonamide (<b>76</b>)</h3><div class="NLM_p last">Compound <b>75</b> (0.30 g, 0.75 mmol), ethanesulfonyl chloride (0.291 g, 2.26 mmol), and triethylamine (0.631 mL, 4.52 mmol) were combined in DCM (10 mL) and stirred for 1 h. The mixture was concentrated and azeotroped with DCM. The residue was dissolved in 1,4-dioxane (5 mL), treated with 1 M aqueous sodium hydroxide (2 mL), and heated at 80 °C for 1.5 h. The mixture was cooled and partitioned with ethyl acetate and water adjusting the pH to 9 with 1 M HCl. The ethyl acetate layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 1–5% methanol in DCM) afforded the title compound (0.294 g, 80%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.80 (s, 1H), 7.86 (d, <i>J</i> = 2.7 Hz, 1H), 7.76 (s, 1H), 7.53 (d, <i>J</i> = 2.7 Hz, 1H), 7.37 (d, <i>J</i> = 2.7 Hz, 1H), 7.27 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 7.07 (d, <i>J</i> = 8.5 Hz, 1H), 5.91 (s, 1H), 4.09–4.14 (m, 2H), 3.41 (s, 3H), 3.11–3.20 (m, 2H), 2.52–2.58 (m, 1H), 2.25–2.36 (m, 1H), 1.55–1.80 (m, 3H), 1.34–1.45 (m, 1H), 1.25 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 490 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_75" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> 4-(2-((3-Bromo-5-nitropyridin-2-yl)oxy)-3,5-difluorophenyl)butan-1-ol (<b>78</b>)</h3><div class="NLM_p last">A mixture of compound <b>69c</b> (0.40 g, 2.0 mmol), 3-bromo-2-chloro-5-nitropyridine <b>77a</b> (0.47 g, 2.0 mmol), and cesium carbonate (0.65 g, 2.0 mmol) in DMSO (5.0 mL) under nitrogen was heated at 65 °C for 1 h. The mixture was partitioned between ethyl acetate and water, and the layers were separated. The water layer was extracted again with ethyl acetate. The organics were combined, washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 20–50% ethyl acetate in heptanes) afforded the title compound (0.74 g, 93%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.04–8.96 (m, 2H), 7.35 (ddd, <i>J</i> = 10.4, 8.7, 3.0 Hz, 1H), 7.15 (ddd, <i>J</i> = 9.3, 3.0, 1.7 Hz, 1H), 4.29 (t, <i>J</i> = 5.2 Hz, 1H), 2.48–2.51 (m, 2H), 1.52 (tt, <i>J</i> = 8.2, 6.4 Hz, 2H), 1.32–1.39 (m, 2H). LC/MS <i>m</i>/<i>z</i> 404.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_76" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> 4-((2,4-Dimethoxybenzyl)oxy)-1-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-2(1<i>H</i>)-one (<b>79</b>)</h3><div class="NLM_p">1 M potassium <i>tert</i>-butoxide in THF (19.8 mL, 19.8 mmol) under nitrogen at 0 °C was treated dropwise with a solution of 2,4-dimethoxybenzyl alcohol (3.33 g, 19.8 mmol) in THF (5 mL). The reaction mixture was stirred for 15 min and treated dropwise with a solution of compound <b>16</b> (2.0 g, 9.0 mmol) dissolved in THF (15 mL). The mixture was stirred at ambient temperature for 2 h and partitioned between ethyl acetate and water. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by chromatography (silica gel, 40–100% ethyl acetate in heptanes) afforded 5-bromo-4-((2,4-dimethoxybenzyl)oxy)-1-methylpyridin-2(1<i>H</i>)-one (1.74 g, 55%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.01 (s, 1H), 7.30 (d, <i>J</i> = 8.3 Hz, 1H), 6.66–6.43 (m, 2H), 6.01 (s, 1H), 4.98 (s, 2H), 3.81 (s, 3H), 3.79 (s, 3H). MS (ESI+) <i>m</i>/<i>z</i> 355.6 (M + H)<sup>+</sup>.</div><div class="NLM_p last">A 250 mL flask was charged with 5-bromo-4-((2,4-dimethoxybenzyl)oxy)-1-methylpyridin-2(1<i>H</i>)-one (8.42 g, 23.8 mmol, prepared in the previous step), 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane) (9.05 g, 35.6 mmol), dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine (1.13 g, 2.38 mmol), potassium acetate (dried in vacuo at 60 °C for 18 h, 5.36 g, 54.6 mmol), and tris(dibenzylideneacetone)dipalladium (0.544 g, 0.594 mmol). The solids were sparged with argon for 5 min, and 1,4-dioxane (80 mL), degassed by argon for 15 min, was added. The mixture was degassed with argon for 10 min and heated at 80 °C overnight under nitrogen. The reaction mixture was concentrated onto silica gel (200 g). This material was loaded onto a funnel plug of silica gel (700 mL) and eluted with a 0–10% methanol/DCM gradient. These fractions containing the title compound were combined and concentrated. The resulting material was triturated in ethyl ether (300 mL) and the resulting solid was purified by flash chromatography (silica gel, 0–3% methanol in DCM) to give the title compound (5.71 g, 60%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.73 (d, <i>J</i> = 8.5 Hz, 1H), 7.66 (s, 1H), 6.55–6.41 (m, 2H), 6.00 (s, 1H), 5.00 (s, 2H), 3.82 (s, 3H), 3.81 (s, 3H), 3.49 (s, 3H), 1.34 (s, 12H). MS (DCI+) <i>m</i>/<i>z</i> 402.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_77" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> 2′-(2,4-Difluoro-6-(4-hydroxybutyl)phenoxy)-4-((2,4-dimethoxybenzyl)oxy)-1-methyl-5′-nitro-[3,3′-bipyridin]-6(1<i>H</i>)-one (<b>80</b>)</h3><div class="NLM_p last">Compound <b>78</b> (0.74 g, 1.8 mmol), compound <b>79</b> (0.810 g, 2.02 mmol), tris(dibenzylideneacetone)dipalladium(0) (0.059 g, 0.064 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (0.037 g, 0.13 mmol), and cesium fluoride (0.836 g, 5.51 mmol) were combined in anhydrous THF (9.0 mL) sparged with argon for 10 min and stirred for 18 h at 50 °C. The mixture was cooled, partitioned between ethyl acetate and water, and filtered through Celite to remove elemental Pd. The organic layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, treated with 3-mecaptopropyl functionalized silica, filtered, and concentrated. Purification by chromatography (silica gel, 1–4% methanol in DCM) afforded the title compound (0.41 g, 37%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.94 (d, <i>J</i> = 2.8 Hz, 1H), 8.58 (d, <i>J</i> = 2.7 Hz, 1H), 7.88 (s, 1H), 7.24 (ddd, <i>J</i> = 10.3, 8.6, 3.0 Hz, 1H), 7.15–7.01 (m, 2H), 6.54 (d, <i>J</i> = 2.3 Hz, 1H), 6.38 (dd, <i>J</i> = 8.4, 2.3 Hz, 1H), 6.02 (s, 1H), 5.00 (s, 2H), 4.28 (t, <i>J</i> = 5.1 Hz, 1H), 3.75 (s, 3H), 3.72 (s, 3H), 3.25–3.16 (m, 2H), 2.39 (t, <i>J</i> = 7.5 Hz, 2H), 1.37 (tt, <i>J</i> = 8.3, 6.4 Hz, 2H), 1.21–1.26 (m, 2H). MS (DCI+) <i>m</i>/<i>z</i> 598.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_78" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> 2′-(2,4-Difluoro-6-(4-hydroxybutyl)phenoxy)-4-hydroxy-1-methyl-5′-nitro-[3,3′-bipyridin]-6(1<i>H</i>)-one (<b>81</b>)</h3><div class="NLM_p last">A solution of compound <b>80</b> (0.41 g, 0.69 mmol) in DCM (20 mL) at 0 °C was treated dropwise with a solution of 2,2,2-trifluoroacetic acid (0.529 mL, 6.86 mmol) in DCM (1 mL). The reaction solution was stirred for 15 min. The cold solution was carefully treated with saturated aqueous sodium bicarbonate to a constant pH of 8. The mixture was filtered to remove white solid. The filtrate layers were separated and the aqueous layer was extracted with DCM (3 × 50 mL). The organics were combined, dried over anhydrous sodium sulfate, filtered, and concentrated to afford a light yellow powder (0.32 g, 100%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.07 (s, 1H), 8.97 (d, <i>J</i> = 2.7 Hz, 1H), 8.57 (d, <i>J</i> = 2.7 Hz, 1H), 7.86 (s, 1H), 7.27 (ddd, <i>J</i> = 10.4, 8.7, 3.0 Hz, 1H), 7.09 (dt, <i>J</i> = 9.2, 2.2 Hz, 1H), 5.79 (s, 1H), 4.35 (s, 1H), 3.34 (s, 3H), 3.23 (d, <i>J</i> = 8.1 Hz, 2H), 2.53 (m, 2H), 1.44 (tt, <i>J</i> = 8.6, 6.8 Hz, 2H), 1.33 (dt, <i>J</i> = 8.7, 6.2 Hz, 2H). MS (ESI+) <i>m</i>/<i>z</i> 448.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_79" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> 11,13-Difluoro-2-methyl-17-nitro-6,7,8,9-tetrahydrobenzo[11,12][1,6]dioxacyclododecino[2,3-<i>b</i>:4,5-<i>c</i>′]dipyridin-3(2<i>H</i>)-one (<b>82</b>)</h3><div class="NLM_p last">A mixture of compound <b>81</b> (0.32 g, 0.72 mmol) in toluene (71.5 mL) under argon was treated with 2-(tributylphosphoranylidene)acetonitrile (0.563 mL, 2.15 mmol), stirred at 80 °C for 2 h, and concentrated. Purification by flash chromatography (silica gel, 1–3% methanol in DCM) afforded the title compound (0.10 g, 33%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.01 (d, <i>J</i> = 2.7 Hz, 1H), 8.58 (d, <i>J</i> = 2.7 Hz, 1H), 7.96 (s, 1H), 7.29 (ddd, <i>J</i> = 10.4, 8.7, 3.0 Hz, 1H), 7.19–7.08 (m, 1H), 5.96 (s, 1H), 4.17 (d, <i>J</i> = 11.5 Hz, 2H), 3.43 (s, 3H), 2.71–2.58 (m, 2H), 2.44–2.33 (m, 2H), 1.60–1.36 (m, 2H). MS (ESI+) <i>m</i>/<i>z</i> 430.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_3_80" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> <i>N</i>-(11,13-Difluoro-2-methyl-3-oxo-2,3,6,7,8,9-hexahydropyrido[3′,4′:4,5][1,6]benzodioxacyclododecino[2,3-<i>b</i>]pyridin-17-yl)ethanesulfonamide (<b>83</b>)</h3><div class="NLM_p">Compound <b>82</b> (0.10 g, 0.23 mmol), iron powder (0.065 g, 1.2 mmol), and ammonium chloride (0.025 g, 0.47 mmol) were combined in THF (3 mL), ethanol (3 mL), and water (1 mL) and heated at 90 °C for 2 h. The mixture was cooled, diluted with THF and methanol, and filtered through Celite to remove solids. The filtrate was concentrated. The residue was partitioned between ethyl acetate and saturated aqueous sodium chloride. The ethyl acetate layer was dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 1–5% methanol in DCM) afforded 17-amino-11,13-difluoro-2-methyl-6,7,8,9-tetrahydrobenzo[11,12][1,6]dioxacyclododecino[2,3-b:4,5-<i>c</i>′]dipyridin-3(2<i>H</i>)-one (0.075 g, 81%). LC/MS (APCI+) <i>m</i>/<i>z</i> 399.7 (M<sup>+</sup>).</div><div class="NLM_p last">17-Amino-11,13-difluoro-2-methyl-6,7,8,9-tetrahydrobenzo[11,12][1,6]dioxacyclododecino[2,3-b:4,5-<i>c</i>′]dipyridin-3(2<i>H</i>)-one (0.075 g, 0.19 mmol, prepared in the previous step), ethanesulfonyl chloride (0.072 g, 0.56 mmol), and triethylamine (0.157 mL, 1.13 mmol) were combined in DCM (2.5 mL) and stirred for 1 h. The mixture was concentrated and azeotroped with DCM. The residue was dissolved in 1,4-dioxane (5 mL), treated with 1 M sodium hydroxide (2 mL), heated at 80 °C for 1.5 h, cooled, and partitioned with ethyl acetate and water, adjusting the pH to 8 with 1 M HCl. The ethyl acetate layer was washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, filtered, and concentrated. Purification by flash chromatography (silica gel, 1–5% methanol in DCM) afforded the title compound (0.062 g, 67%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.84 (s, 1H), 7.88 (d, <i>J</i> = 2.7 Hz, 1H), 7.84 (s, 1H), 7.54 (d, <i>J</i> = 2.4 Hz, 1H), 7.17–7.23 (m, 1H), 7.06 (d, <i>J</i> = 8.8 Hz, 1H), 5.92 (s, 1H), 4.09–4.16 (m, 2H), 3.41 (s, 3H), 3.12–3.20 (m, 2H), 2.56–2.65 (m, 1H), 2.32–2.39 (m, 1H), 1.65–1.79 (m, 2H), 1.52–1.61 (m, 1H), 1.37–1.47 (m, 1H), 1.25 (t, <i>J</i> = 7.3 Hz, 3H). MS (ESI+) <i>m</i>/<i>z</i> 492 (M + H)<sup>+</sup>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00017">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38857" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38857" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00017" class="ext-link">10.1021/acs.jmedchem.7b00017</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Description of all biophysical and biological methods, NMR-based fragment screen, bromodomain binding assay, TR-FRET assay conditions, MX-1 proliferation assay conditions, BRD4 engagement luciferase reporter assay conditions, murine multiple myeloma xenograft model, chemistry schemes, and experimental procedures for compounds <b>22</b>–<b>25</b>, <b>31</b>–<b>55</b>, <b>84</b>–<b>94</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf">jm7b00017_si_001.pdf (718.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_002.csv">jm7b00017_si_002.csv (3.52 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of BRD4 BDII bound to compounds <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFO">5UFO</a>), <b>2a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU">5UEU</a>), <b>11</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEZ">5UEZ</a>), <b>25e</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEY">5UEY</a>), and <b>61a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEX">5UEX</a>) and atomic coordinates of BRD2 BDII bound to compound <b>20a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEW">5UEW</a>) have been deposited with the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b00017" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16017" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16017" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Le Wang</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5d3138732a3c333a1d3c3f3f2b3438733e3230"><span class="__cf_email__" data-cfemail="eb878ec59c8a858cab8a89899d828ec5888486">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John K. Pratt</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2611-8480" title="Orcid link">http://orcid.org/0000-0003-2611-8480</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#97fdf8fff9b9fcb9e7e5f6e3e3d7f6f5f5e1fef2b9f4f8fa"><span class="__cf_email__" data-cfemail="c5afaaadabebaeebb5b7a4b1b185a4a7a7b3aca0eba6aaa8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Todd Soltwedel</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">George S. Sheppard</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven D. Fidanze</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dachun Liu</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa A. Hasvold</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert A. Mantei</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James H. Holms</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William J. McClellan</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael D. Wendt</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carol Wada</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robin Frey</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">T. Matthew Hansen</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert Hubbard</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chang H. Park</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leiming Li</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Terrance J. Magoc</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel H. Albert</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyu Lin</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott E. Warder</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Kovar</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoli Huang</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Denise Wilcox</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rongqi Wang</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ganesh Rajaraman</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew M. Petros</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles W. Hutchins</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sanjay C. Panchal</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chaohong Sun</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven W. Elmore</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Shen</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Warren M. Kati</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Keith F. McDaniel</span> - <span class="hlFld-Affiliation affiliation">AbbVie Inc., 1 North Waukegan Road, North Chicago, Illinois 60064, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.W. and J.K.P. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are current or former AbbVie employees. The design, study conduct, and financial support for this research were provided by AbbVie. AbbVie participated in the interpretation of data, review, and approval of the publication.<br /></br><p class="inlineNote">Former AbbVie employee.</p></div></li></ul></div><div class="ack" id="ACK-d7e7930-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i104">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46006" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46006" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research used resources at the Industrial Macromolecular Crystallography Association Collaborative Access Team (IMCA-CAT) beamline 17-ID, supported by the companies of the Industrial Macromolecular Crystallography Association through a contract with Hauptman-Woodward Medical Research Institute. This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract DE-AC02-06CH11357. The authors acknowledge Lance Bigelow for contributions to the crystallization discussion.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i105" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i105"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i106" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i106"> Abbreviations Used</h2><tr><td class="NLM_term">BD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">tumor growth inhibition</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence energy transfer</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i107">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38653" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38653" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 36 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lheIzMSUSQNAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lheIzMSUSQNAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Brownell, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranalli, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Tetrahymena histone acetyltransferase A: a homolog to yeast Gcn5p linking histone acetylation to gene activation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">843</span><span class="NLM_x">–</span> <span class="NLM_lpage">851</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81063-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0092-8674%2800%2981063-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1996&pages=843-851&author=J.+E.+Brownellauthor=J.+Zhouauthor=T.+Ranalliauthor=R.+Kobayashiauthor=D.+G.+Edmondsonauthor=S.+Y.+Rothauthor=C.+D.+Allis&title=Tetrahymena+histone+acetyltransferase+A%3A+a+homolog+to+yeast+Gcn5p+linking+histone+acetylation+to+gene+activation&doi=10.1016%2FS0092-8674%2800%2981063-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981063-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981063-6%26sid%3Dliteratum%253Aachs%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DRanalli%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DR.%26aulast%3DEdmondson%26aufirst%3DD.%2BG.%26aulast%3DRoth%26aufirst%3DS.%2BY.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTetrahymena%2520histone%2520acetyltransferase%2520A%253A%2520a%2520homolog%2520to%2520yeast%2520Gcn5p%2520linking%2520histone%2520acetylation%2520to%2520gene%2520activation%26jtitle%3DCell%26date%3D1996%26volume%3D84%26spage%3D843%26epage%3D851%26doi%3D10.1016%2FS0092-8674%2800%2981063-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassig, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A mammalian histone deacetylase related to the yeast transcriptional regulator Rpd3p</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1126/science.272.5260.408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1126%2Fscience.272.5260.408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=8602529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADyaK28XisVegsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1996&pages=408-411&author=J.+Tauntonauthor=C.+A.+Hassigauthor=S.+L.+Schreiber&title=A+mammalian+histone+deacetylase+related+to+the+yeast+transcriptional+regulator+Rpd3p&doi=10.1126%2Fscience.272.5260.408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A mammalian histone deacetylase related to the yeast transcriptional regulator Rpd3p</span></div><div class="casAuthors">Taunton, Jack; Hassig, Christian A.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">5260</span>),
    <span class="NLM_cas:pages">408-11</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Trapoxin is a microbially derived cyclotetrapeptide that inhibits histone deacetylation in vivo and causes mammalian cells to arrest in the cell cycle.  A trapoxin affinity matrix was used to isolate two nuclear proteins that copurified with histone deacetylase activity.  Both proteins were identified by peptide microsequencing, and a complementary DNA encoding the histone deacetylase catalytic subunit (HD1) was cloned from a human Jurkat T cell library.  As the predicted protein is very similar to the yeast transcriptional regulator Rpd3p, these results support a role for histone deacetylase as a key regulator of eukaryotic transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBh4pS_nPMRLVg90H21EOLACvtfcHk0li0bmxHwBAR1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XisVegsr8%253D&md5=fda244b40a10d5bc75f94fcc6fae189f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.272.5260.408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.272.5260.408%26sid%3Dliteratum%253Aachs%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DHassig%26aufirst%3DC.%2BA.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520mammalian%2520histone%2520deacetylase%2520related%2520to%2520the%2520yeast%2520transcriptional%2520regulator%2520Rpd3p%26jtitle%3DScience%26date%3D1996%26volume%3D272%26spage%3D408%26epage%3D411%26doi%3D10.1126%2Fscience.272.5260.408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Glozak, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, E.</span><span> </span><span class="NLM_article-title">Acetylation and deacetylation of non-histone proteins</span> <span class="citation_source-journal">Gene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">363</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span><span class="refDoi"> DOI: 10.1016/j.gene.2005.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.gene.2005.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=16289629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Klu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2005&pages=15-23&author=M.+A.+Glozakauthor=N.+Senguptaauthor=X.+Zhangauthor=E.+Seto&title=Acetylation+and+deacetylation+of+non-histone+proteins&doi=10.1016%2Fj.gene.2005.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation and deacetylation of non-histone proteins</span></div><div class="casAuthors">Glozak, Michele A.; Sengupta, Nilanjan; Zhang, Xiaohong; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-23</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Since the first report of p53 as a non-histone target of a histone acetyltransferase (HAT), there has been a rapid proliferation in the description of new non-histone targets of HATs.  Of these, transcription factors comprise the largest class of new targets.  The substrates for HATs extend to cytoskeletal proteins, mol. chaperones and nuclear import factors.  Deacetylation of these non-histone proteins by histone deacetylases (HDACs) opens yet another exciting new field of discovery in the role of the dynamic acetylation and deacetylation on cellular function.  This review will focus on these non-histone targets of HATs and HDACs and the consequences of their modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrghezJOUSImbVg90H21EOLACvtfcHk0li0bmxHwBAR1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Klu7%252FN&md5=c6befd12e4556a9154628d9fe9ebe53a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2005.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2005.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DGlozak%26aufirst%3DM.%2BA.%26aulast%3DSengupta%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DAcetylation%2520and%2520deacetylation%2520of%2520non-histone%2520proteins%26jtitle%3DGene%26date%3D2005%26volume%3D363%26spage%3D15%26epage%3D23%26doi%3D10.1016%2Fj.gene.2005.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Dhalluin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">399</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span><span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.-M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0lghfT6Wkf4lbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Bromodomain: an acetyl-lysine binding domain</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">513</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1016/S0014-5793(01)03309-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0014-5793%2801%2903309-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=11911891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=124-128&author=L.+Zengauthor=M.-M.+Zhou&title=Bromodomain%3A+an+acetyl-lysine+binding+domain&doi=10.1016%2FS0014-5793%2801%2903309-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain: an acetyl-lysine binding domain</span></div><div class="casAuthors">Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 43 refs.  Bromodomains, an extensive family of evolutionarily conserved protein modules originally found in proteins assocd. with chromatin and in nearly all nuclear histone acetyltransferases, have been recently discovered to function as acetyllysine-binding domains.  More recent structural studies of bromodomain/peptide ligand complexes have enriched the understanding of differences in ligand selectivity of bromodomains.  These new findings demonstrate that bromodomain/acetyllysine recognition can serve as a pivotal mechanism for regulating protein-protein interactions in numerous cellular processes including chromatin remodeling and transcriptional activation, and reinforce the concept that functional diversity of a conserved protein modular structure is achieved by evolutionary changes of amino acid sequences in the ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMESL8iZ9agLVg90H21EOLACvtfcHk0lghfT6Wkf4lbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D&md5=79070a63cd171f846ab577aa9f7092bb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2903309-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252903309-9%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DBromodomain%253A%2520an%2520acetyl-lysine%2520binding%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D124%26epage%3D128%26doi%3D10.1016%2FS0014-5793%2801%2903309-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The bromodomain interaction module</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">586</span><span class="NLM_x">, </span> <span class="NLM_fpage">2692</span><span class="NLM_x">–</span> <span class="NLM_lpage">2704</span><span class="refDoi"> DOI: 10.1016/j.febslet.2012.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+bromodomain+interaction+module&doi=10.1016%2Fj.febslet.2012.04.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0lghfT6Wkf4lbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520bromodomain%2520interaction%2520module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D2692%26epage%3D2704%26doi%3D10.1016%2Fj.febslet.2012.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Miyoshi, S.; Ooike, S.; Iwata, K.; Hikawa, H.; Sugahara, K.</span><span> </span><span class="NLM_article-title">Antitumor agent</span>. WO2009084693,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=S.+Miyoshi&author=S.+Ooike&author=K.+Iwata&author=H.+Hikawa&author=K.+Sugahara&title=Antitumor+agent"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMiyoshi%26aufirst%3DS.%26atitle%3DAntitumor%2520agent%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgJOH9SOpJZ1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lgJOH9SOpJZ1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jang, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padmanabhan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structural basis for acetylated histone H4 recognition by the human BRD2 bromodomain</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">7610</span><span class="NLM_x">–</span> <span class="NLM_lpage">7618</span><span class="refDoi"> DOI: 10.1074/jbc.M109.062422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1074%2Fjbc.M109.062422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=20048151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=7610-7618&author=T.+Umeharaauthor=Y.+Nakamuraauthor=M.+K.+Jangauthor=K.+Nakanoauthor=A.+Tanakaauthor=K.+Ozatoauthor=B.+Padmanabhanauthor=S.+Yokoyama&title=Structural+basis+for+acetylated+histone+H4+recognition+by+the+human+BRD2+bromodomain&doi=10.1074%2Fjbc.M109.062422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Acetylated Histone H4 Recognition by the Human BRD2 Bromodomain</span></div><div class="casAuthors">Umehara, Takashi; Nakamura, Yoshihiro; Jang, Moon-Kyoo; Nakano, Kazumi; Tanaka, Akiko; Ozato, Keiko; Padmanabhan, Balasundaram; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7610-7618</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Recognition of acetylated chromatin by the bromodomains and extra-terminal domain (BET) family proteins is a hallmark for transcriptional activation and anchoring viral genomes to mitotic chromosomes of the host.  One of the BET family proteins, BRD2, interacts with acetylated chromatin during mitosis and leads to transcriptional activation in culture cells.  Here, we report the crystal structures of the N-terminal bromodomain of human BRD2 (BRD2-BD1; residues 74-194) in complex with each of three different Lys-12-acetylated H4 peptides.  The BRD2-BD1 recognizes the H4 tail acetylated at Lys-12 (H4K12ac), whereas the side chain of hypoacetylated Lys-8 of H4 binds at the cavity of the dimer interface of BRD2-BD1.  From binding studies, we identified the BRD2-BD1 residues that are responsible for recognition of the Lys-12-acetylated H4 tail.  In addn., mutation to Lys-8 in the Lys-12-acetylated H4 tail decreased the binding to BRD2-BD1, implicating the crit. role of Lys-8 in the Lys-12-acetylated H4 tail for the recognition by BRD2-BD1.  Our findings provide a structural basis for deciphering the histone code by the BET bromodomain through the binding with a long segment of the histone H4 tail, which presumably prevents erasure of the histone code during the cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMfSa5EPdd57Vg90H21EOLACvtfcHk0lgJOH9SOpJZ1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D&md5=88df449d778d37724b9295ad36e0e188</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062422%26sid%3Dliteratum%253Aachs%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DNakano%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520for%2520acetylated%2520histone%2520H4%2520recognition%2520by%2520the%2520human%2520BRD2%2520bromodomain%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D7610%26epage%3D7618%26doi%3D10.1074%2Fjbc.M109.062422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">McLure, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesner, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujikawa, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharenko, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Attwell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campeau, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasiak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontano, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suto, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, N. C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span> </span><span class="NLM_article-title">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e83190</span><span class="refDoi"> DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e83190&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=N.+C.+W.+Wongauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=P.+R.+Young&title=RVX-208%2C+an+inducer+of+ApoA-I+in+humans%2C+is+a+BET+bromodomain+antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0lgcWrGqOrwNzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DRVX-208%252C%2520an%2520inducer%2520of%2520ApoA-I%2520in%2520humans%252C%2520is%2520a%2520BET%2520bromodomain%2520antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De83190%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span><span class="refDoi"> DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lgcWrGqOrwNzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Hytti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tokarz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Määttä, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piippo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korhonen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suuronen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honkakoski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaarniranta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lahtela-Kakkonen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauppinen, A.</span><span> </span><span class="NLM_article-title">Inhibition of BET bromodomains alleviates inflammation in human RPE cells</span> <span class="citation_source-journal">Biochem. Pharmacol. (Amsterdam, Neth.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">110–111</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span><span class="refDoi"> DOI: 10.1016/j.bcp.2016.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.bcp.2016.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110%E2%80%93111&publication_year=2016&pages=71-79&author=M.+Hyttiauthor=P.+Tokarzauthor=E.+M%C3%A4%C3%A4tt%C3%A4author=N.+Piippoauthor=E.+Korhonenauthor=T.+Suuronenauthor=P.+Honkakoskiauthor=K.+Kaarnirantaauthor=M.+Lahtela-Kakkonenauthor=A.+Kauppinen&title=Inhibition+of+BET+bromodomains+alleviates+inflammation+in+human+RPE+cells&doi=10.1016%2Fj.bcp.2016.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomains alleviates inflammation in human RPE cells</span></div><div class="casAuthors">Hytti, M.; Tokarz, P.; Maatta, E.; Piippo, N.; Korhonen, E.; Suuronen, T.; Honkakoski, P.; Kaarniranta, K.; Lahtela-Kakkonen, M.; Kauppinen, A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">110-111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-79</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are vital for controlling the expression of many pro-inflammatory genes.  Consequently, compds. capable of inhibiting specific bromodomain-facilitated protein-protein interactions would be predicted to alleviate inflammation, making them valuable agents in the treatment of diseases caused by dysregulated inflammation, such as age-related macular degeneration.  Here, the authors assessed the ability of known inhibitors JQ-1, PFI-1, and IBET-151 to protect from the inflammation and cell death caused by etoposide exposure in the human retinal pigment epithelial cell line, ARPE-19.  The potential anti-inflammatory effects of the bromodomain inhibitors were assessed by ELISA profiling.  The involvement of NF-κB and SIRT1 in inflammatory signaling was monitored by ELISA and western blotting.  Furthermore, SIRT1 was knocked down using a specific siRNA or inhibited by EX-527 to elucidate its role in the inflammatory reaction.  The bromodomain inhibitors effectively decreased etoposide-induced release of IL-6 and IL-8.  This anti-inflammatory effect was not related to SIRT1 activity, although all bromodomain inhibitors decreased the extent of acetylation of p53 at the SIRT1 deacetylation site.  Overall, since bromodomain inhibitors display anti-inflammatory properties in human retinal pigment epithelial cells, these compds. may represent a new way of alleviating the inflammation underlying the onset of age-related macular degeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocwtuJwbNv5rVg90H21EOLACvtfcHk0lgcWrGqOrwNzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D&md5=689201c89b6400fe064d3e20ac719bee</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DHytti%26aufirst%3DM.%26aulast%3DTokarz%26aufirst%3DP.%26aulast%3DM%25C3%25A4%25C3%25A4tt%25C3%25A4%26aufirst%3DE.%26aulast%3DPiippo%26aufirst%3DN.%26aulast%3DKorhonen%26aufirst%3DE.%26aulast%3DSuuronen%26aufirst%3DT.%26aulast%3DHonkakoski%26aufirst%3DP.%26aulast%3DKaarniranta%26aufirst%3DK.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26aulast%3DKauppinen%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520BET%2520bromodomains%2520alleviates%2520inflammation%2520in%2520human%2520RPE%2520cells%26jtitle%3DBiochem.%2520Pharmacol.%2520%2528Amsterdam%252C%2520Neth.%2529%26date%3D2016%26volume%3D110%25E2%2580%2593111%26spage%3D71%26epage%3D79%26doi%3D10.1016%2Fj.bcp.2016.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Bandukwala, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagnon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenbaum, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamperti, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molesworth, A. M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tough, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, A.</span><span> </span><span class="NLM_article-title">Selective inhibition of CD4<sup>+</sup> T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">14532</span><span class="NLM_x">–</span> <span class="NLM_lpage">14537</span><span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0ljcrb9pWATZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Spiltoir, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demos-Davies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinsey, T. A.</span><span> </span><span class="NLM_article-title">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span> <span class="citation_source-journal">J. Mol. Cell. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.yjmcc.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=23939492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=175-179&author=J.+Spiltoirauthor=M.+S.+Strattonauthor=M.+A.+Cavasinauthor=K.+Demos-Daviesauthor=B.+G.+Reidauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+A.+McKinsey&title=BET+acetyl-lysine+binding+proteins+control+pathological+cardiac+hypertrophy&doi=10.1016%2Fj.yjmcc.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span></div><div class="casAuthors">Spiltoir, Jessica I.; Stratton, Matthew S.; Cavasin, Maria A.; Demos-Davies, Kim; Reid, Brian G.; Qi, Jun; Bradner, James E.; McKinsey, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-179</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac hypertrophy is an independent predictor of adverse outcomes in patients with heart failure, and thus represents an attractive target for novel therapeutic intervention.  JQ1, a small mol. inhibitor of bromodomain and extraterminal (BET) acetyl-lysine reader proteins, was identified in a high throughput screen designed to discover novel small mol. regulators of cardiomyocyte hypertrophy.  JQ1 dose-dependently blocked agonist-dependent hypertrophy of cultured neonatal rat ventricular myocytes (NRVMs) and reversed the prototypical gene program assocd. with pathol. cardiac hypertrophy.  JQ1 also blocked left ventricular hypertrophy (LVH) and improved cardiac function in adult mice subjected to transverse aortic constriction (TAC).  The BET family consists of BRD2, BRD3, BRD4 and BRDT.  BRD4 protein expression was increased during cardiac hypertrophy, and hypertrophic stimuli promoted recruitment of BRD4 to the transcriptional start site (TSS) of the gene encoding atrial natriuretic factor (ANF).  Binding of BRD4 to the ANF TSS was assocd. with increased phosphorylation of local RNA polymerase II.  These findings define a novel function for BET proteins as signal-responsive regulators of cardiac hypertrophy, and suggest that small mol. inhibitors of these epigenetic reader proteins have potential as therapeutics for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUzCSA2tNtrVg90H21EOLACvtfcHk0ljcrb9pWATZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF&md5=ffe6d80a94c27e5d89b38baf33af365c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSpiltoir%26aufirst%3DJ.%26aulast%3DStratton%26aufirst%3DM.%2BS.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DDemos-Davies%26aufirst%3DK.%26aulast%3DReid%26aufirst%3DB.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26atitle%3DBET%2520acetyl-lysine%2520binding%2520proteins%2520control%2520pathological%2520cardiac%2520hypertrophy%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D63%26spage%3D175%26epage%3D179%26doi%3D10.1016%2Fj.yjmcc.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Bui, M. H.; Lin, X.; Albert, D. H.; Li, L.; Lam, L. T.; Faivre, E.; Warder, S.; Huang, X.; Wilcox, D.; Donawho, C.; Sheppard, G. S.; Wang, L.; Fidanze, S.; Pratt, J. K.; Liu, D.; Hasvold, L.; Uziel, T.; Fang, G.; Elmore, S. W.; Rosenberg, S. H.; McDaniel, K. F.; Kati, W.; Shen, Y.</span> Comprehensive in vitro and in vivo characterization of a novel BET family bromodomain inhibitor, ABBV-075, reveals sensitive cancer indications and combination strategies for BET inhibitors. Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bui%2C+M.+H.%3B+Lin%2C+X.%3B+Albert%2C+D.+H.%3B+Li%2C+L.%3B+Lam%2C+L.+T.%3B+Faivre%2C+E.%3B+Warder%2C+S.%3B+Huang%2C+X.%3B+Wilcox%2C+D.%3B+Donawho%2C+C.%3B+Sheppard%2C+G.+S.%3B+Wang%2C+L.%3B+Fidanze%2C+S.%3B+Pratt%2C+J.+K.%3B+Liu%2C+D.%3B+Hasvold%2C+L.%3B+Uziel%2C+T.%3B+Fang%2C+G.%3B+Elmore%2C+S.+W.%3B+Rosenberg%2C+S.+H.%3B+McDaniel%2C+K.+F.%3B+Kati%2C+W.%3B+Shen%2C+Y.+Comprehensive+in+vitro+and+in+vivo+characterization+of+a+novel+BET+family+bromodomain+inhibitor%2C+ABBV-075%2C+reveals+sensitive+cancer+indications+and+combination+strategies+for+BET+inhibitors.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DM.%2BH." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.-F.</span><span> </span><span class="NLM_article-title">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1375</span><span class="NLM_x">–</span> <span class="NLM_lpage">1387</span><span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=H.-D.+Yangauthor=M.-M.+Zhouauthor=K.+Ozatoauthor=L.-F.+Chen&title=Brd4+coactivates+transcriptional+activation+of+NF-%CE%BAB+via+specific+binding+to+acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0ljcrb9pWATZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DH.-D.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-%25CE%25BAB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Klein, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabala, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grabiec, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tak, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ospelt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reedquist, K. A.</span><span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span> <span class="citation_source-journal">Ann. Rheum. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">429</span><span class="refDoi"> DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+G.+Gayauthor=C.+Kollingauthor=L.-L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- α, interleukin (IL)-1β and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- α and IL-1β .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0ljpXgCTqWvogg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BG.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Ghoshal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yugandhar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, A. K.</span><span> </span><span class="NLM_article-title">BET inhibitors in cancer therapeutics: a patent review</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1517/13543776.2016.1159299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1517%2F13543776.2016.1159299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=26924192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Wksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=505-522&author=A.+Ghoshalauthor=D.+Yugandharauthor=A.+K.+Srivastava&title=BET+inhibitors+in+cancer+therapeutics%3A+a+patent+review&doi=10.1517%2F13543776.2016.1159299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors in cancer therapeutics: a patent review</span></div><div class="casAuthors">Ghoshal, Anirban; Yugandhar, D.; Srivastava, Ajay Kumar</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-522</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Introduction: Inhibition of Bromodomain and Extra Terminal (BET) proteins is an emerging approach for developing advanced cancer therapeutics.  In 2015, at least thirty patents have been published for developing cancer chemotherapeutics by targeting BET.  Currently there are seven small mol. BET inhibitors in various stages of clin. trials for the development of anti-cancer drugs.  Areas covered: Important patents focusing on development of BET inhibitors as potential cancer therapeutics published in 2015 have been covered.  The reports are presented together with a review of the related structural chem. space.  This review mainly focuses on the therapeutic applications, chem. class and structural modifications along with the mols. currently in clin. trials.  Expert opinion: BET sub-family proteins are one of the emerging targets to develop anti-cancer agents.  Although many research groups have demonstrated the rationality of BET inhibition to combat cancer, a detailed mol. study needs to be performed to investigate the affected biol. pathways.  Selectivity among BET proteins should be kept in mind while developing BET inhibitors.  In-silico mol. modeling studies can also provide valuable information for designing selective BET inhibitors towards anti-cancer drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoByvS8sb7HhLVg90H21EOLACvtfcHk0ljpXgCTqWvogg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Wksbw%253D&md5=02d7c0cc89a2a712990ceb2e19079f36</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1517%2F13543776.2016.1159299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2016.1159299%26sid%3Dliteratum%253Aachs%26aulast%3DGhoshal%26aufirst%3DA.%26aulast%3DYugandhar%26aufirst%3DD.%26aulast%3DSrivastava%26aufirst%3DA.%2BK.%26atitle%3DBET%2520inhibitors%2520in%2520cancer%2520therapeutics%253A%2520a%2520patent%2520review%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2016%26volume%3D26%26spage%3D505%26epage%3D522%26doi%3D10.1517%2F13543776.2016.1159299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: Progress in the development of bromodomain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1271</span><span class="NLM_x">–</span> <span class="NLM_lpage">1298</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+Progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0ljpXgCTqWvogg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520Progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lgfRcD2jiLsrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudgin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auger, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span><span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lgfRcD2jiLsrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Combs, A. P.; Yue, E. W.; Sparks, R. B.; Maduskuie, J. T. P.</span><span> </span><span class="NLM_article-title">Tricyclic heterocycles as BET protein inhibitors</span>. US20150011540 A1,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+P.+Combs&author=E.+W.+Yue&author=R.+B.+Sparks&author=J.+T.+P.+Maduskuie&title=Tricyclic+heterocycles+as+BET+protein+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCombs%26aufirst%3DA.%2BP.%26atitle%3DTricyclic%2520heterocycles%2520as%2520BET%2520protein%2520inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Aktoudianakis, I.; Chin, G.; Corkey, B. K.; Du, J.; Elbel, K.; Jiang, R. H.; Kobayashi, T.; Martinez, R.; Metobo, S. E.; Mish, M.; Shevick, S.; Sperandio, D.; Yang, H.; Zablocki, J.</span><span> </span><span class="NLM_article-title">Benzimidazolone derivatives as bromodomain inhibitors</span>. WO2014160873 A1,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=I.+Aktoudianakis&author=G.+Chin&author=B.+K.+Corkey&author=J.+Du&author=K.+Elbel&author=R.+H.+Jiang&author=T.+Kobayashi&author=R.+Martinez&author=S.+E.+Metobo&author=M.+Mish&author=S.+Shevick&author=D.+Sperandio&author=H.+Yang&author=J.+Zablocki&title=Benzimidazolone+derivatives+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAktoudianakis%26aufirst%3DI.%26atitle%3DBenzimidazolone%2520derivatives%2520as%2520bromodomain%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Engelhardt, H.; Gianni, D.; Mantoulidis, A.; Smethurst, C.</span><span> </span><span class="NLM_article-title">Indolinone analogues as BRD4 inhibitors</span>. WO2014154760 A1,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=H.+Engelhardt&author=D.+Gianni&author=A.+Mantoulidis&author=C.+Smethurst&title=Indolinone+analogues+as+BRD4+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEngelhardt%26aufirst%3DH.%26atitle%3DIndolinone%2520analogues%2520as%2520BRD4%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Demont, E. H.; Gosmini, R. L. M.</span><span> </span><span class="NLM_article-title">Thetrahydroquinolines derivatives as bromodomain inhibitors</span>. WO2011054848 A1,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=E.+H.+Demont&author=R.+L.+M.+Gosmini&title=Thetrahydroquinolines+derivatives+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThetrahydroquinolines%2520derivatives%2520as%2520bromodomain%2520inhibitors%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bish, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Primiano, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ralph, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciammetta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trzupek, J. D.</span><span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra c-terminal bromodomain inhibition through optimization of a fragment-derived hit</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9831</span><span class="NLM_x">–</span> <span class="NLM_lpage">9837</span><span class="refDoi"> DOI: 10.1021/jm3010515</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+c-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit&doi=10.1021%2Fjm3010515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lgDz4ZUi2C22g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520c-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837%26doi%3D10.1021%2Fjm3010515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augeri, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumeister, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betebenner, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bures, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landis, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of a potent inhibitor of the antiapoptotic protein Bcl-X L from NMR and parallel synthesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">656</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span><span class="refDoi"> DOI: 10.1021/jm0507532</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0507532" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=656-663&author=A.+M.+Petrosauthor=J.+Dingesauthor=D.+J.+Augeriauthor=S.+A.+Baumeisterauthor=D.+A.+Betebennerauthor=M.+G.+Buresauthor=S.+W.+Elmoreauthor=P.+J.+Hajdukauthor=M.+K.+Josephauthor=S.+K.+Landisauthor=D.+G.+Nettesheimauthor=S.+H.+Rosenbergauthor=W.+Shenauthor=S.+Thomasauthor=X.+Wangauthor=I.+Zanzeauthor=H.+Zhangauthor=S.+W.+Fesik&title=Discovery+of+a+potent+inhibitor+of+the+antiapoptotic+protein+Bcl-X+L+from+NMR+and+parallel+synthesis&doi=10.1021%2Fjm0507532"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm0507532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0507532%26sid%3Dliteratum%253Aachs%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DAugeri%26aufirst%3DD.%2BJ.%26aulast%3DBaumeister%26aufirst%3DS.%2BA.%26aulast%3DBetebenner%26aufirst%3DD.%2BA.%26aulast%3DBures%26aufirst%3DM.%2BG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DJoseph%26aufirst%3DM.%2BK.%26aulast%3DLandis%26aufirst%3DS.%2BK.%26aulast%3DNettesheim%26aufirst%3DD.%2BG.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZanze%26aufirst%3DI.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520inhibitor%2520of%2520the%2520antiapoptotic%2520protein%2520Bcl-X%2520L%2520from%2520NMR%2520and%2520parallel%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D656%26epage%3D663%26doi%3D10.1021%2Fjm0507532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span> </span><span class="NLM_article-title">Macrocycles are great cycles: Applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2004</span><span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+Applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lgDz4ZUi2C22g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520Applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Akaji, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teruya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akaji, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aimoto, S.</span><span> </span><span class="NLM_article-title">Synthesis of cyclic RGD derivatives via solid phase macrocyclization using the Heck reaction</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">–</span> <span class="NLM_lpage">2303</span><span class="refDoi"> DOI: 10.1016/S0040-4020(01)00113-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0040-4020%2801%2900113-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=2293-2303&author=K.+Akajiauthor=K.+Teruyaauthor=M.+Akajiauthor=S.+Aimoto&title=Synthesis+of+cyclic+RGD+derivatives+via+solid+phase+macrocyclization+using+the+Heck+reaction&doi=10.1016%2FS0040-4020%2801%2900113-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2801%2900113-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252801%252900113-2%26sid%3Dliteratum%253Aachs%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DTeruya%26aufirst%3DK.%26aulast%3DAkaji%26aufirst%3DM.%26aulast%3DAimoto%26aufirst%3DS.%26atitle%3DSynthesis%2520of%2520cyclic%2520RGD%2520derivatives%2520via%2520solid%2520phase%2520macrocyclization%2520using%2520the%2520Heck%2520reaction%26jtitle%3DTetrahedron%26date%3D2001%26volume%3D57%26spage%3D2293%26epage%3D2303%26doi%3D10.1016%2FS0040-4020%2801%2900113-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Steinman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtin, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garland, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidsen, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holms, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magoc, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagy, I. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchins, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summers, J. B.</span><span> </span><span class="NLM_article-title">The design, synthesis, and structure-activity relationships of a series of macrocyclic MMP inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2087</span><span class="NLM_x">–</span> <span class="NLM_lpage">2092</span><span class="refDoi"> DOI: 10.1016/S0960-894X(98)00396-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0960-894X%2898%2900396-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=2087-2092&author=D.+H.+Steinmanauthor=M.+L.+Curtinauthor=R.+B.+Garlandauthor=S.+K.+Davidsenauthor=H.+R.+Heymanauthor=J.+H.+Holmsauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=I.+B.+Nagyauthor=P.+A.+Marcotteauthor=J.+Liauthor=D.+W.+Morganauthor=C.+Hutchinsauthor=J.+B.+Summers&title=The+design%2C+synthesis%2C+and+structure-activity+relationships+of+a+series+of+macrocyclic+MMP+inhibitors&doi=10.1016%2FS0960-894X%2898%2900396-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33a&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2898%2900396-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252898%252900396-5%26sid%3Dliteratum%253Aachs%26aulast%3DSteinman%26aufirst%3DD.%2BH.%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DGarland%26aufirst%3DR.%2BB.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DHeyman%26aufirst%3DH.%2BR.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DNagy%26aufirst%3DI.%2BB.%26aulast%3DMarcotte%26aufirst%3DP.%2BA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMorgan%26aufirst%3DD.%2BW.%26aulast%3DHutchins%26aufirst%3DC.%26aulast%3DSummers%26aufirst%3DJ.%2BB.%26atitle%3DThe%2520design%252C%2520synthesis%252C%2520and%2520structure-activity%2520relationships%2520of%2520a%2520series%2520of%2520macrocyclic%2520MMP%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1998%26volume%3D8%26spage%3D2087%26epage%3D2092%26doi%3D10.1016%2FS0960-894X%2898%2900396-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit33b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Cherney, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arner, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardman, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasserman, Z. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaffee, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decicco, C. P.</span><span> </span><span class="NLM_article-title">Macrocyclic hydroxamate inhibitors of matrix metalloproteinases and TNF-alpha production</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1279</span><span class="NLM_x">–</span> <span class="NLM_lpage">1284</span><span class="refDoi"> DOI: 10.1016/S0960-894X(99)00178-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0960-894X%2899%2900178-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=1279-1284&author=R.+J.+Cherneyauthor=L.+Wangauthor=D.+T.+Meyerauthor=C.+B.+Xueauthor=E.+C.+Arnerauthor=R.+A.+Copelandauthor=M.+B.+Covingtonauthor=K.+D.+Hardmanauthor=Z.+R.+Wassermanauthor=B.+D.+Jaffeeauthor=C.+P.+Decicco&title=Macrocyclic+hydroxamate+inhibitors+of+matrix+metalloproteinases+and+TNF-alpha+production&doi=10.1016%2FS0960-894X%2899%2900178-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33b&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2899%2900178-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252899%252900178-X%26sid%3Dliteratum%253Aachs%26aulast%3DCherney%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeyer%26aufirst%3DD.%2BT.%26aulast%3DXue%26aufirst%3DC.%2BB.%26aulast%3DArner%26aufirst%3DE.%2BC.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCovington%26aufirst%3DM.%2BB.%26aulast%3DHardman%26aufirst%3DK.%2BD.%26aulast%3DWasserman%26aufirst%3DZ.%2BR.%26aulast%3DJaffee%26aufirst%3DB.%2BD.%26aulast%3DDecicco%26aufirst%3DC.%2BP.%26atitle%3DMacrocyclic%2520hydroxamate%2520inhibitors%2520of%2520matrix%2520metalloproteinases%2520and%2520TNF-alpha%2520production%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1999%26volume%3D9%26spage%3D1279%26epage%3D1284%26doi%3D10.1016%2FS0960-894X%2899%2900178-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Mallinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">1438</span><span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lhf2OlO95U4OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2615</span><span class="NLM_x">–</span> <span class="NLM_lpage">2623</span><span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.+Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lhAReTGd7qvsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.%2BY.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span>FreeForm; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FreeForm%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DFreeForm%26pub%3DOpenEye%2520Scientific%2520Software%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33a','cit33b'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 34 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Wenji Su, Rui Ge, Duanchen Ding, Wenhua Chen, Wenqing Wang, Hao Yan, Weikun Wang, Youlang Yuan, Huan Liu, Meng Zhang, Jiyuan Zhang, Qisheng Shu, Alexander L. Satz, <span class="NLM_string-name hlFld-ContribAuthor">Letian Kuai</span>. </span><span class="cited-content_cbyCitation_article-title">Triaging of DNA-Encoded Library Selection Results by High-Throughput Resynthesis of DNA–Conjugate and Affinity Selection Mass Spectrometry. </span><span class="cited-content_cbyCitation_journal-name">Bioconjugate Chemistry</span><span> <strong>2021,</strong> <em>32 </em>
                                    (5)
                                     , 1001-1007. <a href="https://doi.org/10.1021/acs.bioconjchem.1c00170" title="DOI URL">https://doi.org/10.1021/acs.bioconjchem.1c00170</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.bioconjchem.1c00170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.bioconjchem.1c00170%26sid%3Dliteratum%253Aachs%26jtitle%3DBioconjugate%2520Chemistry%26atitle%3DTriaging%252Bof%252BDNA-Encoded%252BLibrary%252BSelection%252BResults%252Bby%252BHigh-Throughput%252BResynthesis%252Bof%252BDNA%2525E2%252580%252593Conjugate%252Band%252BAffinity%252BSelection%252BMass%252BSpectrometry%26aulast%3DSu%26aufirst%3DWenji%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D30032021%26date%3D22042021%26date%3D29042021%26volume%3D32%26issue%3D5%26spage%3D1001%26epage%3D1007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">George S. Sheppard, Le Wang, Steven D. Fidanze, Lisa A. Hasvold, Dachun Liu, John K. Pratt, Chang H. Park, Kenton Longenecker, Wei Qiu, Maricel Torrent, Peter J. Kovar, Mai Bui, Emily Faivre, Xiaoli Huang, Xiaoyu Lin, Denise Wilcox, Lu Zhang, Yu Shen, Daniel H. Albert, Terrance J. Magoc, Ganesh Rajaraman, Warren M. Kati, <span class="NLM_string-name hlFld-ContribAuthor">Keith F. McDaniel</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5585-5623. <a href="https://doi.org/10.1021/acs.jmedchem.0c00628" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00628</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BN-Ethyl-4-%25255B2-%2525284-fluoro-2%25252C6-dimethyl-phenoxy%252529-5-%2525281-hydroxy-1-methyl-ethyl%252529phenyl%25255D-6-methyl-7-oxo-1H-pyrrolo%25255B2%25252C3-c%25255Dpyridine-2-carboxamide%252B%252528ABBV-744%252529%25252C%252Ba%252BBET%252BBromodomain%252BInhibitor%252Bwith%252BSelectivity%252Bfor%252Bthe%252BSecond%252BBromodomain%26aulast%3DSheppard%26aufirst%3DGeorge%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16042020%26date%3D07052020%26date%3D23042020%26volume%3D63%26issue%3D10%26spage%3D5585%26epage%3D5623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher R. Wellaway, Dominique Amans, Paul Bamborough, Heather Barnett, Rino A. Bit, Jack A. Brown, Neil R. Carlson, Chun-wa Chung, Anthony W. J. Cooper, Peter D. Craggs, Robert P. Davis, Tony W. Dean, John P. Evans, Laurie Gordon, Isobel L. Harada, David J. Hirst, Philip G. Humphreys, Katherine L. Jones, Antonia J. Lewis, Matthew J. Lindon, Dave Lugo, Mahnoor Mahmood, Scott McCleary, Patricia Medeiros, Darren J. Mitchell, Michael O’Sullivan, Armelle Le Gall, Vipulkumar K. Patel, Chris Patten, Darren L. Poole, Rishi R. Shah, Jane E. Smith, Kayleigh A. J. Stafford, Pamela J. Thomas, Mythily Vimal, Ian D. Wall, Robert J. Watson, Natalie Wellaway, Gang Yao, <span class="NLM_string-name hlFld-ContribAuthor">Rab K. Prinjha</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 714-746. <a href="https://doi.org/10.1021/acs.jmedchem.9b01670" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01670</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01670%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BBromodomain%252Band%252BExtraterminal%252BInhibitor%252Bwith%252Ba%252BLow%252BPredicted%252BHuman%252BDose%252Bthrough%252BSynergistic%252BUse%252Bof%252BEncoded%252BLibrary%252BTechnology%252Band%252BFragment%252BScreening%26aulast%3DWellaway%26aufirst%3DChristopher%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D08102019%26date%3D06012020%26volume%3D63%26issue%3D2%26spage%3D714%26epage%3D746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fei Jiang, Qinghua Hu, Zhimin Zhang, Hongmei Li, Huili Li, Dewei Zhang, Hanwen Li, Yu Ma, Jingjing Xu, Haifang Chen, Yong Cui, Yanle Zhi, Yanmin Zhang, Junyu Xu, Jiapeng Zhu, Tao Lu, <span class="NLM_string-name hlFld-ContribAuthor">Yadong Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (24)
                                     , 11080-11107. <a href="https://doi.org/10.1021/acs.jmedchem.9b01010" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BBenzo%25255Bcd%25255Dindol-2%2525281H%252529-ones%252Band%252BPyrrolo%25255B4%25252C3%25252C2-de%25255Dquinolin-2%2525281H%252529-ones%252Bas%252BBromodomain%252Band%252BExtra-Terminal%252BDomain%252B%252528BET%252529%252BInhibitors%252Bwith%252BSelectivity%252Bfor%252Bthe%252BFirst%252BBromodomain%252Bwith%252BPotential%252BHigh%252BEfficiency%252Bagainst%252BAcute%252BGouty%252BArthritis%26aulast%3DJiang%26aufirst%3DFei%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26062019%26date%3D13122019%26date%3D30112019%26volume%3D62%26issue%3D24%26spage%3D11080%26epage%3D11107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul N. Mortenson, Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, <span class="NLM_string-name hlFld-ContribAuthor">Christopher N. Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2017. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 3857-3872. <a href="https://doi.org/10.1021/acs.jmedchem.8b01472" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01472</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01472%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2017%26aulast%3DMortenson%26aufirst%3DPaul%2BN.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D21092018%26date%3D21112018%26volume%3D62%26issue%3D8%26spage%3D3857%26epage%3D3872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dean G. Brown, <span class="NLM_string-name hlFld-ContribAuthor">Jonas Boström</span>. </span><span class="cited-content_cbyCitation_article-title">Where Do Recent Small Molecule Clinical Development Candidates Come From?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (21)
                                     , 9442-9468. <a href="https://doi.org/10.1021/acs.jmedchem.8b00675" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00675</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00675%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DWhere%252BDo%252BRecent%252BSmall%252BMolecule%252BClinical%252BDevelopment%252BCandidates%252BCome%252BFrom%25253F%26aulast%3DBrown%26aufirst%3DDean%2BG.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D27042018%26date%3D09072018%26date%3D19062018%26volume%3D61%26issue%3D21%26spage%3D9442%26epage%3D9468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chong Qin, Yang Hu, Bing Zhou, Ester Fernandez-Salas, Chao-Yie Yang, Liu Liu, Donna McEachern, Sally Przybranowski, Mi Wang, Jeanne Stuckey, Jennifer Meagher, Longchuan Bai, Zhuo Chen, Mei Lin, Jiuling Yang, Danya N. Ziazadeh, Fuming Xu, Jiantao Hu, Weiguo Xiang, Liyue Huang, Siwei Li, Bo Wen, Duxin Sun, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6685-6704. <a href="https://doi.org/10.1021/acs.jmedchem.8b00506" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00506</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BQCA570%252Bas%252Ban%252BExceptionally%252BPotent%252Band%252BEfficacious%252BProteolysis%252BTargeting%252BChimera%252B%252528PROTAC%252529%252BDegrader%252Bof%252Bthe%252BBromodomain%252Band%252BExtra-Terminal%252B%252528BET%252529%252BProteins%252BCapable%252Bof%252BInducing%252BComplete%252Band%252BDurable%252BTumor%252BRegression%26aulast%3DQin%26aufirst%3DChong%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D30032018%26date%3D18072018%26volume%3D61%26issue%3D15%26spage%3D6685%26epage%3D6704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Keith F.  McDaniel</span>, <span class="hlFld-ContribAuthor ">Le  Wang</span>, <span class="hlFld-ContribAuthor ">Todd  Soltwedel</span>, <span class="hlFld-ContribAuthor ">Steven D.  Fidanze</span>, <span class="hlFld-ContribAuthor ">Lisa A.  Hasvold</span>, <span class="hlFld-ContribAuthor ">Dachun  Liu</span>, <span class="hlFld-ContribAuthor ">Robert A.  Mantei</span>, <span class="hlFld-ContribAuthor ">John K.  Pratt</span>, <span class="hlFld-ContribAuthor ">George S.  Sheppard</span>, <span class="hlFld-ContribAuthor ">Mai H.  Bui</span>, <span class="hlFld-ContribAuthor ">Emily J.  Faivre</span>, <span class="hlFld-ContribAuthor ">Xiaoli  Huang</span>, <span class="hlFld-ContribAuthor ">Leiming  Li</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Lin</span>, <span class="hlFld-ContribAuthor ">Rongqi  Wang</span>, <span class="hlFld-ContribAuthor ">Scott E.  Warder</span>, <span class="hlFld-ContribAuthor ">Denise  Wilcox</span>, <span class="hlFld-ContribAuthor ">Daniel H.  Albert</span>, <span class="hlFld-ContribAuthor ">Terrance J.  Magoc</span>, <span class="hlFld-ContribAuthor ">Ganesh  Rajaraman</span>, <span class="hlFld-ContribAuthor ">Chang H.  Park</span>, <span class="hlFld-ContribAuthor ">Charles W.  Hutchins</span>, <span class="hlFld-ContribAuthor ">Jianwei J.  Shen</span>, <span class="hlFld-ContribAuthor ">Rohinton P.  Edalji</span>, <span class="hlFld-ContribAuthor ">Chaohong C.  Sun</span>, <span class="hlFld-ContribAuthor ">Ruth  Martin</span>, <span class="hlFld-ContribAuthor ">Wenqing  Gao</span>, <span class="hlFld-ContribAuthor ">Shekman  Wong</span>, <span class="hlFld-ContribAuthor ">Guowei  Fang</span>, <span class="hlFld-ContribAuthor ">Steven W.  Elmore</span>, <span class="hlFld-ContribAuthor ">Yu  Shen</span>, and <span class="hlFld-ContribAuthor ">Warren M.  Kati</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (20)
                                     , 8369-8384. <a href="https://doi.org/10.1021/acs.jmedchem.7b00746" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00746</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BN-%2525284-%2525282%25252C4-Difluorophenoxy%252529-3-%2525286-methyl-7-oxo-6%25252C7-dihydro-1H-pyrrolo%25255B2%25252C3-c%25255Dpyridin-4-yl%252529phenyl%252529ethanesulfonamide%252B%252528ABBV-075%25252FMivebresib%252529%25252C%252Ba%252BPotent%252Band%252BOrally%252BAvailable%252BBromodomain%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BFamily%252BBromodomain%252BInhibitor%26aulast%3DMcDaniel%26aufirst%3DKeith%2BF.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D19052017%26date%3D12102017%26date%3D26102017%26date%3D26092017%26volume%3D60%26issue%3D20%26spage%3D8369%26epage%3D8384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Germano  Heinzelmann</span>, <span class="hlFld-ContribAuthor ">Michael K.  Gilson</span>. </span><span class="cited-content_cbyCitation_article-title">Automation of absolute protein-ligand binding free energy calculations for docking refinement and compound evaluation. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-80769-1" title="DOI URL">https://doi.org/10.1038/s41598-020-80769-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-80769-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-80769-1%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DAutomation%252Bof%252Babsolute%252Bprotein-ligand%252Bbinding%252Bfree%252Benergy%252Bcalculations%252Bfor%252Bdocking%252Brefinement%252Band%252Bcompound%252Bevaluation%26aulast%3DHeinzelmann%26aufirst%3DGermano%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aslı Deniz  Aydın</span>, <span class="hlFld-ContribAuthor ">Faruk  Altınel</span>, <span class="hlFld-ContribAuthor ">Hüseyin  Erdoğmuş</span>, <span class="hlFld-ContribAuthor ">Çağdaş Devrim  Son</span>. </span><span class="cited-content_cbyCitation_article-title">Allergen fragrance molecules: a potential relief for COVID-19. </span><span class="cited-content_cbyCitation_journal-name">BMC Complementary Medicine and Therapies</span><span> <strong>2021,</strong> <em>21 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12906-021-03214-4" title="DOI URL">https://doi.org/10.1186/s12906-021-03214-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12906-021-03214-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12906-021-03214-4%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Complementary%2520Medicine%2520and%2520Therapies%26atitle%3DAllergen%252Bfragrance%252Bmolecules%25253A%252Ba%252Bpotential%252Brelief%252Bfor%252BCOVID-19%26aulast%3DAyd%25C4%25B1n%26aufirst%3DAsl%25C4%25B1%2BDeniz%26date%3D2021%26date%3D2021%26volume%3D21%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert B. A.  Quinlan</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomics for drug discovery: clinical molecules from open access chemical probes. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>2 </em>
                                    (3)
                                     , 759-795. <a href="https://doi.org/10.1039/D1CB00016K" title="DOI URL">https://doi.org/10.1039/D1CB00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CB00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CB00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3DChemogenomics%252Bfor%252Bdrug%252Bdiscovery%25253A%252Bclinical%252Bmolecules%252Bfrom%252Bopen%252Baccess%252Bchemical%252Bprobes%26aulast%3DQuinlan%26aufirst%3DRobert%2BB.%2BA.%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D3%26spage%3D759%26epage%3D795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roderick E.  Hubbard</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment‐Based Lead Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-35. <a href="https://doi.org/10.1002/0471266949.bmc265" title="DOI URL">https://doi.org/10.1002/0471266949.bmc265</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc265%26sid%3Dliteratum%253Aachs%26atitle%3DFragment%2525E2%252580%252590Based%252BLead%252BDiscovery%26aulast%3DHubbard%26aufirst%3DRoderick%2BE.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D35%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vijay H.  Masand</span>, <span class="hlFld-ContribAuthor ">Meghshyam K.  Patil</span>, <span class="hlFld-ContribAuthor ">Nahed Nasser E.  El-Sayed</span>, <span class="hlFld-ContribAuthor ">Magdi E.A.  Zaki</span>, <span class="hlFld-ContribAuthor ">Zainab  Almarhoon</span>, <span class="hlFld-ContribAuthor ">Sami A  Al-Hussain</span>. </span><span class="cited-content_cbyCitation_article-title">Balanced QSAR analysis to identify the structural requirements of ABBV-075 (Mivebresib) analogues as bromodomain and extraterminal domain (BET) family bromodomain inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2021,</strong> <em>1229 </em>, 129597. <a href="https://doi.org/10.1016/j.molstruc.2020.129597" title="DOI URL">https://doi.org/10.1016/j.molstruc.2020.129597</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2020.129597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2020.129597%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DBalanced%252BQSAR%252Banalysis%252Bto%252Bidentify%252Bthe%252Bstructural%252Brequirements%252Bof%252BABBV-075%252B%252528Mivebresib%252529%252Banalogues%252Bas%252Bbromodomain%252Band%252Bextraterminal%252Bdomain%252B%252528BET%252529%252Bfamily%252Bbromodomain%252Binhibitor%26aulast%3DMasand%26aufirst%3DVijay%2BH.%26date%3D2021%26volume%3D1229%26spage%3D129597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaohua  Liu</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yalei  Li</span>, <span class="hlFld-ContribAuthor ">Juan  Wang</span>, <span class="hlFld-ContribAuthor ">Huaqian  Ding</span>, <span class="hlFld-ContribAuthor ">Wenjing  Huang</span>, <span class="hlFld-ContribAuthor ">Chunyong  Ding</span>, <span class="hlFld-ContribAuthor ">Hongchun  Liu</span>, <span class="hlFld-ContribAuthor ">Wenfu  Tan</span>, <span class="hlFld-ContribAuthor ">Ao  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of hedgehog pathway inhibitors by epigenetically targeting GLI through BET bromodomain for the treatment of medulloblastoma. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 488-504. <a href="https://doi.org/10.1016/j.apsb.2020.07.007" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDevelopment%252Bof%252Bhedgehog%252Bpathway%252Binhibitors%252Bby%252Bepigenetically%252Btargeting%252BGLI%252Bthrough%252BBET%252Bbromodomain%252Bfor%252Bthe%252Btreatment%252Bof%252Bmedulloblastoma%26aulast%3DLiu%26aufirst%3DXiaohua%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D488%26epage%3D504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohd.  Muddassir</span>, <span class="hlFld-ContribAuthor ">Kunjal  Soni</span>, <span class="hlFld-ContribAuthor ">Chetan B.  Sangani</span>, <span class="hlFld-ContribAuthor ">Abdullah  Alarifi</span>, <span class="hlFld-ContribAuthor ">Mohd.  Afzal</span>, <span class="hlFld-ContribAuthor ">Naaser A. Y.  Abduh</span>, <span class="hlFld-ContribAuthor ">Yongtao  Duan</span>, <span class="hlFld-ContribAuthor ">Poonam  Bhadja</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain and BET family proteins as epigenetic targets in cancer therapy: their degradation, present drugs, and possible PROTACs. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 612-636. <a href="https://doi.org/10.1039/D0RA07971E" title="DOI URL">https://doi.org/10.1039/D0RA07971E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA07971E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA07971E%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DBromodomain%252Band%252BBET%252Bfamily%252Bproteins%252Bas%252Bepigenetic%252Btargets%252Bin%252Bcancer%252Btherapy%25253A%252Btheir%252Bdegradation%25252C%252Bpresent%252Bdrugs%25252C%252Band%252Bpossible%252BPROTACs%26aulast%3DMuddassir%26aufirst%3DMohd.%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D612%26epage%3D636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Qin</span>, <span class="hlFld-ContribAuthor ">Jie‐Lian  Han</span>, <span class="hlFld-ContribAuthor ">Cheng‐Wei  Ju</span>, <span class="hlFld-ContribAuthor ">Dongbing  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Ring Expansion to 6‐, 7‐, and 8‐Membered Benzosilacycles through Strain‐Release Silicon‐Based Cross‐Coupling. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2020,</strong> <em>59 </em>
                                    (22)
                                     , 8481-8485. <a href="https://doi.org/10.1002/anie.202001539" title="DOI URL">https://doi.org/10.1002/anie.202001539</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202001539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202001539%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DRing%252BExpansion%252Bto%252B6%2525E2%252580%252590%25252C%252B7%2525E2%252580%252590%25252C%252Band%252B8%2525E2%252580%252590Membered%252BBenzosilacycles%252Bthrough%252BStrain%2525E2%252580%252590Release%252BSilicon%2525E2%252580%252590Based%252BCross%2525E2%252580%252590Coupling%26aulast%3DQin%26aufirst%3DYing%26date%3D2020%26date%3D2020%26volume%3D59%26issue%3D22%26spage%3D8481%26epage%3D8485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Qin</span>, <span class="hlFld-ContribAuthor ">Jie‐Lian  Han</span>, <span class="hlFld-ContribAuthor ">Cheng‐Wei  Ju</span>, <span class="hlFld-ContribAuthor ">Dongbing  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Ring Expansion to 6‐, 7‐, and 8‐Membered Benzosilacycles through Strain‐Release Silicon‐Based Cross‐Coupling. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2020,</strong> <em>132 </em>
                                    (22)
                                     , 8559-8563. <a href="https://doi.org/10.1002/ange.202001539" title="DOI URL">https://doi.org/10.1002/ange.202001539</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202001539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202001539%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DRing%252BExpansion%252Bto%252B6%2525E2%252580%252590%25252C%252B7%2525E2%252580%252590%25252C%252Band%252B8%2525E2%252580%252590Membered%252BBenzosilacycles%252Bthrough%252BStrain%2525E2%252580%252590Release%252BSilicon%2525E2%252580%252590Based%252BCross%2525E2%252580%252590Coupling%26aulast%3DQin%26aufirst%3DYing%26date%3D2020%26date%3D2020%26volume%3D132%26issue%3D22%26spage%3D8559%26epage%3D8563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">L.F.  Wang</span>, <span class="hlFld-ContribAuthor ">Y.  Wang</span>, <span class="hlFld-ContribAuthor ">Z.Y.  Yang</span>, <span class="hlFld-ContribAuthor ">J.  Zhao</span>, <span class="hlFld-ContribAuthor ">H.B.  Sun</span>, <span class="hlFld-ContribAuthor ">S.L.  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Revealing binding selectivity of inhibitors toward bromodomain-containing proteins 2 and 4 using multiple short molecular dynamics simulations and free energy analyses. </span><span class="cited-content_cbyCitation_journal-name">SAR and QSAR in Environmental Research</span><span> <strong>2020,</strong> <em>31 </em>
                                    (5)
                                     , 373-398. <a href="https://doi.org/10.1080/1062936X.2020.1748107" title="DOI URL">https://doi.org/10.1080/1062936X.2020.1748107</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/1062936X.2020.1748107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F1062936X.2020.1748107%26sid%3Dliteratum%253Aachs%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26atitle%3DRevealing%252Bbinding%252Bselectivity%252Bof%252Binhibitors%252Btoward%252Bbromodomain-containing%252Bproteins%252B2%252Band%252B4%252Busing%252Bmultiple%252Bshort%252Bmolecular%252Bdynamics%252Bsimulations%252Band%252Bfree%252Benergy%252Banalyses%26aulast%3DWang%26aufirst%3DL.F.%26date%3D2020%26date%3D2020%26volume%3D31%26issue%3D5%26spage%3D373%26epage%3D398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Márton  Nagyházi</span>, <span class="hlFld-ContribAuthor ">Gábor  Turczel</span>, <span class="hlFld-ContribAuthor ">Áron  Balla</span>, <span class="hlFld-ContribAuthor ">Gábor  Szálas</span>, <span class="hlFld-ContribAuthor ">Imre  Tóth</span>, <span class="hlFld-ContribAuthor ">Gyula Tamás  Gál</span>, <span class="hlFld-ContribAuthor ">Bombicz  Petra</span>, <span class="hlFld-ContribAuthor ">Paul T.  Anastas</span>, <span class="hlFld-ContribAuthor ">Róbert  Tuba</span>. </span><span class="cited-content_cbyCitation_article-title">Towards Sustainable Catalysis – Highly Efficient Olefin Metathesis in Protic Media Using Phase Labelled Cyclic Alkyl Amino Carbene (CAAC) Ruthenium Catalysts. </span><span class="cited-content_cbyCitation_journal-name">ChemCatChem</span><span> <strong>2020,</strong> <em>12 </em>
                                    (7)
                                     , 1953-1957. <a href="https://doi.org/10.1002/cctc.201902258" title="DOI URL">https://doi.org/10.1002/cctc.201902258</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cctc.201902258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcctc.201902258%26sid%3Dliteratum%253Aachs%26jtitle%3DChemCatChem%26atitle%3DTowards%252BSustainable%252BCatalysis%252B%2525E2%252580%252593%252BHighly%252BEfficient%252BOlefin%252BMetathesis%252Bin%252BProtic%252BMedia%252BUsing%252BPhase%252BLabelled%252BCyclic%252BAlkyl%252BAmino%252BCarbene%252B%252528CAAC%252529%252BRuthenium%252BCatalysts%26aulast%3DNagyh%25C3%25A1zi%26aufirst%3DM%25C3%25A1rton%26date%3D2020%26volume%3D12%26issue%3D7%26spage%3D1953%26epage%3D1957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">J.P.  Hilton-Proctor</span>, <span class="hlFld-ContribAuthor ">O.  Ilyichova</span>, <span class="hlFld-ContribAuthor ">Z.  Zheng</span>, <span class="hlFld-ContribAuthor ">I.G.  Jennings</span>, <span class="hlFld-ContribAuthor ">R.W.  Johnstone</span>, <span class="hlFld-ContribAuthor ">J.  Shortt</span>, <span class="hlFld-ContribAuthor ">S.J.  Mountford</span>, <span class="hlFld-ContribAuthor ">M.J.  Scanlon</span>, <span class="hlFld-ContribAuthor ">P.E.  Thompson</span>. </span><span class="cited-content_cbyCitation_article-title">Substituted 1-methyl-4-phenylpyrrolidin-2-ones – Fragment-based design of N-methylpyrrolidone-derived bromodomain inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>191 </em>, 112120. <a href="https://doi.org/10.1016/j.ejmech.2020.112120" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112120%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSubstituted%252B1-methyl-4-phenylpyrrolidin-2-ones%252B%2525E2%252580%252593%252BFragment-based%252Bdesign%252Bof%252BN-methylpyrrolidone-derived%252Bbromodomain%252Binhibitors%26aulast%3DHilton-Proctor%26aufirst%3DJ.P.%26date%3D2020%26volume%3D191%26spage%3D112120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando D.  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Current advances on the development of BET inhibitors: insights from computational methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 127-180. <a href="https://doi.org/10.1016/bs.apcsb.2020.06.002" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.06.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252Badvances%252Bon%252Bthe%252Bdevelopment%252Bof%252BBET%252Binhibitors%25253A%252Binsights%252Bfrom%252Bcomputational%252Bmethods%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%2BD.%26date%3D2020%26spage%3D127%26epage%3D180%26pub%3DElsevier%26date%3D2020%26volume%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanle  Zhi</span>, <span class="hlFld-ContribAuthor ">Shu  Wang</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Meihao  Liang</span>, <span class="hlFld-ContribAuthor ">Chixiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhen  Ma</span>, <span class="hlFld-ContribAuthor ">Zunyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Novel phenanthridin-6(5H)-one derivatives as potent and selective BET bromodomain inhibitors: Rational design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>179 </em>, 502-514. <a href="https://doi.org/10.1016/j.ejmech.2019.06.067" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.06.067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.06.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.06.067%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252Bphenanthridin-6%2525285H%252529-one%252Bderivatives%252Bas%252Bpotent%252Band%252Bselective%252BBET%252Bbromodomain%252Binhibitors%25253A%252BRational%252Bdesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DZhi%26aufirst%3DYanle%26date%3D2019%26volume%3D179%26spage%3D502%26epage%3D514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shyh-Ming  Yang</span>, <span class="hlFld-ContribAuthor ">Makoto  Yoshioka</span>, <span class="hlFld-ContribAuthor ">Jeffrey W.  Strovel</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Urban</span>, <span class="hlFld-ContribAuthor ">Xin  Hu</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Hall</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">David J.  Maloney</span>. </span><span class="cited-content_cbyCitation_article-title">Lead optimization and efficacy evaluation of quinazoline-based BET family inhibitors for potential treatment of cancer and inflammatory diseases. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (10)
                                     , 1220-1226. <a href="https://doi.org/10.1016/j.bmcl.2019.03.014" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.03.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.03.014%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLead%252Boptimization%252Band%252Befficacy%252Bevaluation%252Bof%252Bquinazoline-based%252BBET%252Bfamily%252Binhibitors%252Bfor%252Bpotential%252Btreatment%252Bof%252Bcancer%252Band%252Binflammatory%252Bdiseases%26aulast%3DYang%26aufirst%3DShyh-Ming%26date%3D2019%26volume%3D29%26issue%3D10%26spage%3D1220%26epage%3D1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dario  Valenti</span>, <span class="hlFld-ContribAuthor ">Stanimira  Hristeva</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Tzalis</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>. </span><span class="cited-content_cbyCitation_article-title">Clinical candidates modulating protein-protein interactions: The fragment-based experience. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 76-95. <a href="https://doi.org/10.1016/j.ejmech.2019.01.084" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.084</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.084%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DClinical%252Bcandidates%252Bmodulating%252Bprotein-protein%252Binteractions%25253A%252BThe%252Bfragment-based%252Bexperience%26aulast%3DValenti%26aufirst%3DDario%26date%3D2019%26volume%3D167%26spage%3D76%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingchao  Duan</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Guan</span>, <span class="hlFld-ContribAuthor ">Wenping  Qin</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhai</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Brd4 for cancer therapy: inhibitors and degraders. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2018,</strong> <em>9 </em>
                                    (11)
                                     , 1779-1802. <a href="https://doi.org/10.1039/C8MD00198G" title="DOI URL">https://doi.org/10.1039/C8MD00198G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8MD00198G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8MD00198G%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DTargeting%252BBrd4%252Bfor%252Bcancer%252Btherapy%25253A%252Binhibitors%252Band%252Bdegraders%26aulast%3DDuan%26aufirst%3DYingchao%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D11%26spage%3D1779%26epage%3D1802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyang  Li</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Leilei  Zhao</span>, <span class="hlFld-ContribAuthor ">Yifei  Yang</span>, <span class="hlFld-ContribAuthor ">Huibin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinpei  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and in vitro Biological Evaluation of 3,5-Dimethylisoxazole Derivatives as BRD4 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (13)
                                     , 1363-1368. <a href="https://doi.org/10.1002/cmdc.201800074" title="DOI URL">https://doi.org/10.1002/cmdc.201800074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800074%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252Bin%252Bvitro%252BBiological%252BEvaluation%252Bof%252B3%25252C5-Dimethylisoxazole%252BDerivatives%252Bas%252BBRD4%252BInhibitors%26aulast%3DLi%26aufirst%3DXiangyang%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D13%26spage%3D1363%26epage%3D1368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven D.  Fidanze</span>, <span class="hlFld-ContribAuthor ">Dachun  Liu</span>, <span class="hlFld-ContribAuthor ">Robert A.  Mantei</span>, <span class="hlFld-ContribAuthor ">Lisa A.  Hasvold</span>, <span class="hlFld-ContribAuthor ">John K.  Pratt</span>, <span class="hlFld-ContribAuthor ">George S.  Sheppard</span>, <span class="hlFld-ContribAuthor ">Le  Wang</span>, <span class="hlFld-ContribAuthor ">James H.  Holms</span>, <span class="hlFld-ContribAuthor ">Yujia  Dai</span>, <span class="hlFld-ContribAuthor ">Ana  Aguirre</span>, <span class="hlFld-ContribAuthor ">Andrew  Bogdan</span>, <span class="hlFld-ContribAuthor ">Justin D.  Dietrich</span>, <span class="hlFld-ContribAuthor ">Jasmina  Marjanovic</span>, <span class="hlFld-ContribAuthor ">Chang H.  Park</span>, <span class="hlFld-ContribAuthor ">Charles W.  Hutchins</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Lin</span>, <span class="hlFld-ContribAuthor ">Mai H.  Bui</span>, <span class="hlFld-ContribAuthor ">Xiaoli  Huang</span>, <span class="hlFld-ContribAuthor ">Denise  Wilcox</span>, <span class="hlFld-ContribAuthor ">Leiming  Li</span>, <span class="hlFld-ContribAuthor ">Rongqi  Wang</span>, <span class="hlFld-ContribAuthor ">Peter  Kovar</span>, <span class="hlFld-ContribAuthor ">Terrance J.  Magoc</span>, <span class="hlFld-ContribAuthor ">Ganesh  Rajaraman</span>, <span class="hlFld-ContribAuthor ">Daniel H.  Albert</span>, <span class="hlFld-ContribAuthor ">Yu  Shen</span>, <span class="hlFld-ContribAuthor ">Warren M.  Kati</span>, <span class="hlFld-ContribAuthor ">Keith F.  McDaniel</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and optimization of novel constrained pyrrolopyridone BET family inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (10)
                                     , 1804-1810. <a href="https://doi.org/10.1016/j.bmcl.2018.04.020" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.04.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.04.020%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252Boptimization%252Bof%252Bnovel%252Bconstrained%252Bpyrrolopyridone%252BBET%252Bfamily%252Binhibitors%26aulast%3DFidanze%26aufirst%3DSteven%2BD.%26date%3D2018%26volume%3D28%26issue%3D10%26spage%3D1804%26epage%3D1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqian  Xue</span>, <span class="hlFld-ContribAuthor ">Yan  Zhang</span>, <span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Maofeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiuping  Xiang</span>, <span class="hlFld-ContribAuthor ">Junjian  Wang</span>, <span class="hlFld-ContribAuthor ">Anhui  Wang</span>, <span class="hlFld-ContribAuthor ">Chenchang  Li</span>, <span class="hlFld-ContribAuthor ">Cheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Lingjiao  Zou</span>, <span class="hlFld-ContribAuthor ">Rui  Wang</span>, <span class="hlFld-ContribAuthor ">Shuang  Wu</span>, <span class="hlFld-ContribAuthor ">Yongzhi  Lu</span>, <span class="hlFld-ContribAuthor ">Hongwu  Chen</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>, <span class="hlFld-ContribAuthor ">Guohui  Li</span>, <span class="hlFld-ContribAuthor ">Yong  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Benzoxazinone-containing 3,5-dimethylisoxazole derivatives as BET bromodomain inhibitors for treatment of castration-resistant prostate cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>152 </em>, 542-559. <a href="https://doi.org/10.1016/j.ejmech.2018.04.034" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.04.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.04.034%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBenzoxazinone-containing%252B3%25252C5-dimethylisoxazole%252Bderivatives%252Bas%252BBET%252Bbromodomain%252Binhibitors%252Bfor%252Btreatment%252Bof%252Bcastration-resistant%252Bprostate%252Bcancer%26aulast%3DXue%26aufirst%3DXiaoqian%26date%3D2018%26volume%3D152%26spage%3D542%26epage%3D559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bing  Xiong</span>, <span class="hlFld-ContribAuthor ">Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Jingkang  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Discovery for Developing Inhibitors of Protein-Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 135-176. <a href="https://doi.org/10.1007/978-981-13-0773-7_6" title="DOI URL">https://doi.org/10.1007/978-981-13-0773-7_6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-13-0773-7_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-13-0773-7_6%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BDrug%252BDiscovery%252Bfor%252BDeveloping%252BInhibitors%252Bof%252BProtein-Protein%252BInteractions%26aulast%3DXiong%26aufirst%3DBing%26date%3D2018%26date%3D2018%26spage%3D135%26epage%3D176%26pub%3DSpringer%2520Singapore%26atitle%3DTargeting%252BProtein-Protein%252BInteractions%252Bby%252BSmall%252BMolecules%26aulast%3DSheng%26aufirst%3DChunquan%26date%3D2018%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sulejman  Alihodžić</span>, <span class="hlFld-ContribAuthor ">Mirjana  Bukvić</span>, <span class="hlFld-ContribAuthor ">Ivaylo J.  Elenkov</span>, <span class="hlFld-ContribAuthor ">Antun  Hutinec</span>, <span class="hlFld-ContribAuthor ">Sanja  Koštrun</span>, <span class="hlFld-ContribAuthor ">Dijana  Pešić</span>, <span class="hlFld-ContribAuthor ">Gordon  Saxty</span>, <span class="hlFld-ContribAuthor ">Linda  Tomašković</span>, <span class="hlFld-ContribAuthor ">Dinko  Žiher</span>. </span><span class="cited-content_cbyCitation_article-title">Current Trends in Macrocyclic Drug Discovery and beyond -Ro5. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 113-233. <a href="https://doi.org/10.1016/bs.pmch.2018.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2018.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2018.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2018.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252BTrends%252Bin%252BMacrocyclic%252BDrug%252BDiscovery%252Band%252Bbeyond%252B-Ro5%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DSulejman%26date%3D2018%26spage%3D113%26epage%3D233%26pub%3DElsevier%26date%3D2018%26volume%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vincent  Wagner</span>, <span class="hlFld-ContribAuthor ">Linda  Jantz</span>, <span class="hlFld-ContribAuthor ">Hans  Briem</span>, <span class="hlFld-ContribAuthor ">Kai  Sommer</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>, <span class="hlFld-ContribAuthor ">Clara D.  Christ</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Macrocyclization: From de novo Macrocycle Generation to Binding Affinity Estimation. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (22)
                                     , 1866-1872. <a href="https://doi.org/10.1002/cmdc.201700478" title="DOI URL">https://doi.org/10.1002/cmdc.201700478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700478%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DComputational%252BMacrocyclization%25253A%252BFrom%252Bde%2525E2%252580%252585novo%252BMacrocycle%252BGeneration%252Bto%252BBinding%252BAffinity%252BEstimation%26aulast%3DWagner%26aufirst%3DVincent%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D22%26spage%3D1866%26epage%3D1872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Damian  W Young</span>. </span><span class="cited-content_cbyCitation_article-title">Using Fragment Based Drug Discovery to Target Epigenetic Regulators in Cancer. </span><span class="cited-content_cbyCitation_journal-name">MOJ Bioequivalence & Bioavailability</span><span> <strong>2017,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.15406/mojbb.2017.04.00062" title="DOI URL">https://doi.org/10.15406/mojbb.2017.04.00062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15406/mojbb.2017.04.00062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15406%2Fmojbb.2017.04.00062%26sid%3Dliteratum%253Aachs%26jtitle%3DMOJ%2520Bioequivalence%2520%2526%2520Bioavailability%26atitle%3DUsing%252BFragment%252BBased%252BDrug%252BDiscovery%252Bto%252BTarget%252BEpigenetic%252BRegulators%252Bin%252BCancer%26aulast%3DW%2BYoung%26aufirst%3DDamian%26date%3D2017%26date%3D2017%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mostafa  Radwan</span>, <span class="hlFld-ContribAuthor ">Rabah  Serya</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Discovery in the Bromodomain and Extra-Terminal Domain Family. </span><span class="cited-content_cbyCitation_journal-name">Archiv der Pharmazie</span><span> <strong>2017,</strong> <em>350 </em>
                                    (8)
                                     , 1700147. <a href="https://doi.org/10.1002/ardp.201700147" title="DOI URL">https://doi.org/10.1002/ardp.201700147</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ardp.201700147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fardp.201700147%26sid%3Dliteratum%253Aachs%26jtitle%3DArchiv%2520der%2520Pharmazie%26atitle%3DFragment-Based%252BDrug%252BDiscovery%252Bin%252Bthe%252BBromodomain%252Band%252BExtra-Terminal%252BDomain%252BFamily%26aulast%3DRadwan%26aufirst%3DMostafa%26date%3D2017%26date%3D2017%26volume%3D350%26issue%3D8%26spage%3D1700147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan-Xuan  Du</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>, <span class="hlFld-ContribAuthor ">Chang-Long  Yang</span>, <span class="hlFld-ContribAuthor ">Jun  Lin</span>, <span class="hlFld-ContribAuthor ">Sheng-Jiao  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and evaluation of the antitumor activity of highly functionalised pyridin-2-ones and pyrimidin-4-ones. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2017,</strong> <em>7 </em>
                                    (64)
                                     , 40067-40073. <a href="https://doi.org/10.1039/C7RA06466G" title="DOI URL">https://doi.org/10.1039/C7RA06466G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7RA06466G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7RA06466G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSynthesis%252Band%252Bevaluation%252Bof%252Bthe%252Bantitumor%252Bactivity%252Bof%252Bhighly%252Bfunctionalised%252Bpyridin-2-ones%252Band%252Bpyrimidin-4-ones%26aulast%3DDu%26aufirst%3DXuan-Xuan%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D64%26spage%3D40067%26epage%3D40073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0001.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative BET inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) MeI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 91%; (ii) Na, MeOH, 50 °C, 89%; (iii) phenol, Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 100 °C, 95%; (iv) Fe<sup>0</sup>, NH<sub>4</sub>Cl, EtOH/THF/water, 95 °C, 97%; (v) 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane), KOAc, Pd<sub>2</sub>(dba)<sub>3</sub>, (1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane), 1,4-dioxane, 80 °C, 70%; (vi) <b>5</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, CsF, DME/MeOH, 120 °C, 57–62%; (vii) (<b>13a</b>) AcCl, Et<sub>3</sub>N, DCM, 99%; (<b>13b</b>) MsCl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 68%; (<b>13c</b>) ClCOOEt, Et<sub>3</sub>N, DCM, 81%; (<b>13d</b>) EtNCO, Et<sub>3</sub>N, DCM, 51%.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) NaNO<sub>2</sub>, H<sub>2</sub>SO<sub>4</sub>, NH<sub>4</sub>OH, water, 0 °C, quant; (ii) MeI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 91%; (iii) NaOMe in MeOH, or NaOEt in EtOH, 60 °C, 82–89%; (iv) <b>9</b> or <b>10</b> Pd(PPh<sub>3</sub>)<sub>4</sub>, CsF, 48–65%; (v) MsCl or ethanesulfonyl chloride, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 35–56%.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 2,4-difluorophenol, Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 110 °C, quant; (ii) Fe<sup>0</sup>, NH<sub>4</sub>Cl, EtOH/THF/water, 100 °C, 82%; (iii) 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane), KOAc, Pd<sub>2</sub>(dba)<sub>3</sub>, (1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane), 1,4-dioxane, 80 °C, quant; (iv) <b>16</b>, K<sub>3</sub>PO<sub>4</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, (1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane), 1,4-dioxane/water, 60 °C, 86%; (v) ethanesulfonyl chloride, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 81%; (vi) NaOMe in MeOH, 60 °C for <b>30a</b>; NaH, ROH, 1,4-dioxane, 70 °C for <b>30b</b>–<b>f</b>, 11–87%.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 4. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) <b>6a</b> or <b>52</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, CsF, THF, tri-<i>tert</i>-butylphosphonium tetrafluoroborate, 45 °C, 79%. (ii) (a) <b>53</b>: phenols, Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 100 °C, 6–62%. (b) <b>56</b>: phenols, Cs<sub>2</sub>CO<sub>3</sub>, 70–85 °C, DMSO, 69–75%. (iii) Et<sub>3</sub>N, CuI, but-3-yn-1-ol or prop-2-yn-1-ol, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, DMF, 47–100%; (iv) Pt, H<sub>2</sub>, THF, 80–95%; (v) NaH, 1,4-dioxane, 90 °C, 12–48%; (vi) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 90 °C, 58–81%.</p></p></figure><figure data-id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0013.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 5. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Fe<sup>0</sup>, NH<sub>4</sub>Cl, THF/EtOH/water, 95 °C, 90%; (ii) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM then aq NaOH/1,4-dioxane, 90 °C, 80%; (iii) NaH, pent-4-en-1-ol, 1,4-dioxane, 90 °C, 75%; (iv) Pd(PPh<sub>3</sub>)<sub>4</sub>, Et<sub>3</sub>N, DMF, 100 °C. <b>65a</b>: 29%. <b>65b</b>: 30%. (v) Pt, H<sub>2</sub>, THF, 85%.</p></p></figure><figure data-id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0014.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 6. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Et<sub>3</sub>N, CuI, but-3-yn-1-ol, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, DMF, 83–96%; (ii) Pd/C, H<sub>2</sub>, THF, 68–100%; (iii) <b>16</b>, KO<sup><i>t</i></sup>Bu, 1,4-dioxane, 75%; (iv) 4,4,4′,4′,5,5,5′,5′-octamethyl -2,2′-bi(1,3,2-dioxaborolane), KOAc, dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine, Pd<sub>2</sub>(dba)<sub>3</sub>, 1,4-dioxane, 80 °C, 61%. (v) for <b>72a</b>: <b>77a</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, K<sub>3</sub>PO<sub>4</sub>, 1,3,5,7-tetramethyl-6-phenyl-2,4,8-trioxa-6-phosphaadamantane, 1,4-dioxane, water, 50 °C, 32%. For <b>72b</b> and <b>72c</b>: <b>77b</b> or <b>77c</b>, respectively, CsF, Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>tert</i>-butylphosphonium tetrafluoroborate, THF, 50 °C, 49–70%; (vi) BBr<sub>3</sub>, DCM, −78 °C to rt, 83–88%; (vii) Cs<sub>2</sub>CO<sub>3</sub>, MeCN, 75–95 °C, overnight, 77–90%; (viii) Fe<sup>0</sup>, NH<sub>4</sub>Cl, THF/EtOH/water, 90 °C, 94%; (ix) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 85 °C, 80%.</p></p></figure><figure data-id="sch7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0015.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 7. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 65 °C, 93%; (ii) 2,4-dimethoxybenzyl alcohol, KO<sup><i>t</i></sup>Bu, THF, 55%; (iii) KOAc, dicyclohexyl(2′,4′,6′-triisopropyl-[1,1′-biphenyl]-2-yl)phosphine, Pd<sub>2</sub>(dba)<sub>3</sub>, 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane), 1,4-dioxane, 80 °C, 60%; (iv) CsF, THF, Pd<sub>2</sub>(dba)<sub>3</sub>, tri-<i>tert</i>-butylphosphonium tetrafluoroborate, 50 °C, 37%; (v) TFA, DCM, 100%; (vi) 2-(tributylphosphoranylidene)acetonitrile, toluene, 80 °C, 33%; (vii) (a) Fe<sup>0</sup>, NH<sub>4</sub>Cl, THF/EtOH/water, 90 °C, 81%; (b) EtSO<sub>2</sub>Cl, Et<sub>3</sub>N, DCM, then aq NaOH/1,4-dioxane, 80 °C, 67%.</p></p></figure><figure data-id="fig2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0002.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a, left panel) X-ray cocrystal structure of <b>1</b> in the binding pocket of the BRD4 BDII protein. (b, right panel) Hydrogen bonding of pyridone carbonyl oxygen to Asn433 and pyridone <i>N</i>-methyl substituent in amphoteric pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0003.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Overlay of X-ray cocrystal structures <b>1</b> in blue and <b>2a</b> in yellow bound to the BRD4 BDII protein. Vectors show opportunities for additional SAR exploration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0004.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray cocrystal structure of <b>11</b> in the BRD4 BDII protein illustrating the C-ring phenyl ether sitting in the WPF shelf pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0005.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray cocrystal structure of <b>20a</b> bound to BRD2 BDII highlighting the sulfonamide oxygen hydrogen bond to Asp377 and the retained hydrogen bond to Asn429.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0006.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Efficacy of compound <b>30a</b> in a multiple myeloma xenograft (OPM-2) model. Compound or vehicle was administered po daily for 21 days starting on day 0. Values represent mean tumor volume ± standard error (<i>n</i> = 10/group). WL: weight loss. RFS: removed from study due to morbidity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0007.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of X-ray cocrystal structures <b>25e</b> (blue) and <b>61a</b> (yellow) in BRD4 BDII.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/medium/jm-2017-00017p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0008.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Efficacy of compound <b>74b</b> in a multiple myeloma xenograft (OPM-2) model. Compound or vehicle was administered po daily for 21 days starting on day 0. Values represent mean tumor volume ± standard error (<i>n</i> = 8/group). WL: weight loss. RFS: removed from study due to morbidity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00017/20170505/images/large/jm-2017-00017p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00017&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i107">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84068" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84068" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 36 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lhEIz4nBxqg-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhEIz4nBxqg-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Brownell, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranalli, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Tetrahymena histone acetyltransferase A: a homolog to yeast Gcn5p linking histone acetylation to gene activation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">843</span><span class="NLM_x">–</span> <span class="NLM_lpage">851</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81063-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0092-8674%2800%2981063-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1996&pages=843-851&author=J.+E.+Brownellauthor=J.+Zhouauthor=T.+Ranalliauthor=R.+Kobayashiauthor=D.+G.+Edmondsonauthor=S.+Y.+Rothauthor=C.+D.+Allis&title=Tetrahymena+histone+acetyltransferase+A%3A+a+homolog+to+yeast+Gcn5p+linking+histone+acetylation+to+gene+activation&doi=10.1016%2FS0092-8674%2800%2981063-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981063-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981063-6%26sid%3Dliteratum%253Aachs%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DRanalli%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DR.%26aulast%3DEdmondson%26aufirst%3DD.%2BG.%26aulast%3DRoth%26aufirst%3DS.%2BY.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTetrahymena%2520histone%2520acetyltransferase%2520A%253A%2520a%2520homolog%2520to%2520yeast%2520Gcn5p%2520linking%2520histone%2520acetylation%2520to%2520gene%2520activation%26jtitle%3DCell%26date%3D1996%26volume%3D84%26spage%3D843%26epage%3D851%26doi%3D10.1016%2FS0092-8674%2800%2981063-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassig, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span> </span><span class="NLM_article-title">A mammalian histone deacetylase related to the yeast transcriptional regulator Rpd3p</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">408</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1126/science.272.5260.408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1126%2Fscience.272.5260.408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=8602529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADyaK28XisVegsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1996&pages=408-411&author=J.+Tauntonauthor=C.+A.+Hassigauthor=S.+L.+Schreiber&title=A+mammalian+histone+deacetylase+related+to+the+yeast+transcriptional+regulator+Rpd3p&doi=10.1126%2Fscience.272.5260.408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A mammalian histone deacetylase related to the yeast transcriptional regulator Rpd3p</span></div><div class="casAuthors">Taunton, Jack; Hassig, Christian A.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">5260</span>),
    <span class="NLM_cas:pages">408-11</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Trapoxin is a microbially derived cyclotetrapeptide that inhibits histone deacetylation in vivo and causes mammalian cells to arrest in the cell cycle.  A trapoxin affinity matrix was used to isolate two nuclear proteins that copurified with histone deacetylase activity.  Both proteins were identified by peptide microsequencing, and a complementary DNA encoding the histone deacetylase catalytic subunit (HD1) was cloned from a human Jurkat T cell library.  As the predicted protein is very similar to the yeast transcriptional regulator Rpd3p, these results support a role for histone deacetylase as a key regulator of eukaryotic transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBh4pS_nPMRLVg90H21EOLACvtfcHk0lhYGBtCEdvlrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XisVegsr8%253D&md5=fda244b40a10d5bc75f94fcc6fae189f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.272.5260.408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.272.5260.408%26sid%3Dliteratum%253Aachs%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DHassig%26aufirst%3DC.%2BA.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DA%2520mammalian%2520histone%2520deacetylase%2520related%2520to%2520the%2520yeast%2520transcriptional%2520regulator%2520Rpd3p%26jtitle%3DScience%26date%3D1996%26volume%3D272%26spage%3D408%26epage%3D411%26doi%3D10.1126%2Fscience.272.5260.408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Glozak, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, E.</span><span> </span><span class="NLM_article-title">Acetylation and deacetylation of non-histone proteins</span> <span class="citation_source-journal">Gene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">363</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span><span class="refDoi"> DOI: 10.1016/j.gene.2005.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.gene.2005.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=16289629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Klu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2005&pages=15-23&author=M.+A.+Glozakauthor=N.+Senguptaauthor=X.+Zhangauthor=E.+Seto&title=Acetylation+and+deacetylation+of+non-histone+proteins&doi=10.1016%2Fj.gene.2005.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation and deacetylation of non-histone proteins</span></div><div class="casAuthors">Glozak, Michele A.; Sengupta, Nilanjan; Zhang, Xiaohong; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-23</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Since the first report of p53 as a non-histone target of a histone acetyltransferase (HAT), there has been a rapid proliferation in the description of new non-histone targets of HATs.  Of these, transcription factors comprise the largest class of new targets.  The substrates for HATs extend to cytoskeletal proteins, mol. chaperones and nuclear import factors.  Deacetylation of these non-histone proteins by histone deacetylases (HDACs) opens yet another exciting new field of discovery in the role of the dynamic acetylation and deacetylation on cellular function.  This review will focus on these non-histone targets of HATs and HDACs and the consequences of their modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrghezJOUSImbVg90H21EOLACvtfcHk0lhYGBtCEdvlrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Klu7%252FN&md5=c6befd12e4556a9154628d9fe9ebe53a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2005.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2005.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DGlozak%26aufirst%3DM.%2BA.%26aulast%3DSengupta%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DAcetylation%2520and%2520deacetylation%2520of%2520non-histone%2520proteins%26jtitle%3DGene%26date%3D2005%26volume%3D363%26spage%3D15%26epage%3D23%26doi%3D10.1016%2Fj.gene.2005.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Dhalluin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">399</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span><span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.-M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0lhYGBtCEdvlrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Bromodomain: an acetyl-lysine binding domain</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">513</span><span class="NLM_x">, </span> <span class="NLM_fpage">124</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1016/S0014-5793(01)03309-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0014-5793%2801%2903309-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=11911891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=124-128&author=L.+Zengauthor=M.-M.+Zhou&title=Bromodomain%3A+an+acetyl-lysine+binding+domain&doi=10.1016%2FS0014-5793%2801%2903309-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain: an acetyl-lysine binding domain</span></div><div class="casAuthors">Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 43 refs.  Bromodomains, an extensive family of evolutionarily conserved protein modules originally found in proteins assocd. with chromatin and in nearly all nuclear histone acetyltransferases, have been recently discovered to function as acetyllysine-binding domains.  More recent structural studies of bromodomain/peptide ligand complexes have enriched the understanding of differences in ligand selectivity of bromodomains.  These new findings demonstrate that bromodomain/acetyllysine recognition can serve as a pivotal mechanism for regulating protein-protein interactions in numerous cellular processes including chromatin remodeling and transcriptional activation, and reinforce the concept that functional diversity of a conserved protein modular structure is achieved by evolutionary changes of amino acid sequences in the ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMESL8iZ9agLVg90H21EOLACvtfcHk0lhU8qNb3pgwkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D&md5=79070a63cd171f846ab577aa9f7092bb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2903309-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252903309-9%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DBromodomain%253A%2520an%2520acetyl-lysine%2520binding%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D124%26epage%3D128%26doi%3D10.1016%2FS0014-5793%2801%2903309-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The bromodomain interaction module</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">586</span><span class="NLM_x">, </span> <span class="NLM_fpage">2692</span><span class="NLM_x">–</span> <span class="NLM_lpage">2704</span><span class="refDoi"> DOI: 10.1016/j.febslet.2012.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.febslet.2012.04.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22710155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=2692-2704&author=P.+Filippakopoulosauthor=S.+Knapp&title=The+bromodomain+interaction+module&doi=10.1016%2Fj.febslet.2012.04.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain interaction module</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2692-2704</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The ε-N-acetylation of protein Lys residues (Kac) is one of the most abundant post-translation modifications (PTMs) in the human proteome.  In the nucleus, the acetylation of histones has been linked to transcriptional activation of genes but the functional consequences of most acetylation events and proteins recruited to these sites remains largely unknown.  Bromodomains (BRDs) are small helical interaction modules that specifically recognize acetylation sites in proteins.  BRDs have recently emerged as interesting targets for the development of specific protein interaction inhibitors, enabling a novel exiting strategy for the development of new therapies.  Here, the author provides an overview over sequence requirements of BRDs, known substrates, and the structural mechanisms of specific Kac recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7LaThygz_HrVg90H21EOLACvtfcHk0ljsRhNc-Vzpuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFahsbg%253D&md5=c02dee22c330b1d5c1ba39579360ebc5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520bromodomain%2520interaction%2520module%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D2692%26epage%3D2704%26doi%3D10.1016%2Fj.febslet.2012.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Miyoshi, S.; Ooike, S.; Iwata, K.; Hikawa, H.; Sugahara, K.</span><span> </span><span class="NLM_article-title">Antitumor agent</span>. WO2009084693,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=S.+Miyoshi&author=S.+Ooike&author=K.+Iwata&author=H.+Hikawa&author=K.+Sugahara&title=Antitumor+agent"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMiyoshi%26aufirst%3DS.%26atitle%3DAntitumor%2520agent%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljsRhNc-Vzpuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0ljfm6qUxlKE7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jang, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padmanabhan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structural basis for acetylated histone H4 recognition by the human BRD2 bromodomain</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">7610</span><span class="NLM_x">–</span> <span class="NLM_lpage">7618</span><span class="refDoi"> DOI: 10.1074/jbc.M109.062422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1074%2Fjbc.M109.062422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=20048151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=7610-7618&author=T.+Umeharaauthor=Y.+Nakamuraauthor=M.+K.+Jangauthor=K.+Nakanoauthor=A.+Tanakaauthor=K.+Ozatoauthor=B.+Padmanabhanauthor=S.+Yokoyama&title=Structural+basis+for+acetylated+histone+H4+recognition+by+the+human+BRD2+bromodomain&doi=10.1074%2Fjbc.M109.062422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Acetylated Histone H4 Recognition by the Human BRD2 Bromodomain</span></div><div class="casAuthors">Umehara, Takashi; Nakamura, Yoshihiro; Jang, Moon-Kyoo; Nakano, Kazumi; Tanaka, Akiko; Ozato, Keiko; Padmanabhan, Balasundaram; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7610-7618</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Recognition of acetylated chromatin by the bromodomains and extra-terminal domain (BET) family proteins is a hallmark for transcriptional activation and anchoring viral genomes to mitotic chromosomes of the host.  One of the BET family proteins, BRD2, interacts with acetylated chromatin during mitosis and leads to transcriptional activation in culture cells.  Here, we report the crystal structures of the N-terminal bromodomain of human BRD2 (BRD2-BD1; residues 74-194) in complex with each of three different Lys-12-acetylated H4 peptides.  The BRD2-BD1 recognizes the H4 tail acetylated at Lys-12 (H4K12ac), whereas the side chain of hypoacetylated Lys-8 of H4 binds at the cavity of the dimer interface of BRD2-BD1.  From binding studies, we identified the BRD2-BD1 residues that are responsible for recognition of the Lys-12-acetylated H4 tail.  In addn., mutation to Lys-8 in the Lys-12-acetylated H4 tail decreased the binding to BRD2-BD1, implicating the crit. role of Lys-8 in the Lys-12-acetylated H4 tail for the recognition by BRD2-BD1.  Our findings provide a structural basis for deciphering the histone code by the BET bromodomain through the binding with a long segment of the histone H4 tail, which presumably prevents erasure of the histone code during the cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMfSa5EPdd57Vg90H21EOLACvtfcHk0ljfm6qUxlKE7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D&md5=88df449d778d37724b9295ad36e0e188</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062422%26sid%3Dliteratum%253Aachs%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DNakano%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520for%2520acetylated%2520histone%2520H4%2520recognition%2520by%2520the%2520human%2520BRD2%2520bromodomain%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D7610%26epage%3D7618%26doi%3D10.1074%2Fjbc.M109.062422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">McLure, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gesner, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsujikawa, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharenko, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Attwell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campeau, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasiak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontano, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suto, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, N. C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span> </span><span class="NLM_article-title">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e83190</span><span class="refDoi"> DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e83190&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=N.+C.+W.+Wongauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=P.+R.+Young&title=RVX-208%2C+an+inducer+of+ApoA-I+in+humans%2C+is+a+BET+bromodomain+antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0ljfm6qUxlKE7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DRVX-208%252C%2520an%2520inducer%2520of%2520ApoA-I%2520in%2520humans%252C%2520is%2520a%2520BET%2520bromodomain%2520antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De83190%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span><span class="refDoi"> DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lgbDCMC_Yb4Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Hytti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tokarz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Määttä, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piippo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korhonen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suuronen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honkakoski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaarniranta, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lahtela-Kakkonen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauppinen, A.</span><span> </span><span class="NLM_article-title">Inhibition of BET bromodomains alleviates inflammation in human RPE cells</span> <span class="citation_source-journal">Biochem. Pharmacol. (Amsterdam, Neth.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">110–111</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span><span class="refDoi"> DOI: 10.1016/j.bcp.2016.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.bcp.2016.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110%E2%80%93111&publication_year=2016&pages=71-79&author=M.+Hyttiauthor=P.+Tokarzauthor=E.+M%C3%A4%C3%A4tt%C3%A4author=N.+Piippoauthor=E.+Korhonenauthor=T.+Suuronenauthor=P.+Honkakoskiauthor=K.+Kaarnirantaauthor=M.+Lahtela-Kakkonenauthor=A.+Kauppinen&title=Inhibition+of+BET+bromodomains+alleviates+inflammation+in+human+RPE+cells&doi=10.1016%2Fj.bcp.2016.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomains alleviates inflammation in human RPE cells</span></div><div class="casAuthors">Hytti, M.; Tokarz, P.; Maatta, E.; Piippo, N.; Korhonen, E.; Suuronen, T.; Honkakoski, P.; Kaarniranta, K.; Lahtela-Kakkonen, M.; Kauppinen, A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">110-111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-79</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are vital for controlling the expression of many pro-inflammatory genes.  Consequently, compds. capable of inhibiting specific bromodomain-facilitated protein-protein interactions would be predicted to alleviate inflammation, making them valuable agents in the treatment of diseases caused by dysregulated inflammation, such as age-related macular degeneration.  Here, the authors assessed the ability of known inhibitors JQ-1, PFI-1, and IBET-151 to protect from the inflammation and cell death caused by etoposide exposure in the human retinal pigment epithelial cell line, ARPE-19.  The potential anti-inflammatory effects of the bromodomain inhibitors were assessed by ELISA profiling.  The involvement of NF-κB and SIRT1 in inflammatory signaling was monitored by ELISA and western blotting.  Furthermore, SIRT1 was knocked down using a specific siRNA or inhibited by EX-527 to elucidate its role in the inflammatory reaction.  The bromodomain inhibitors effectively decreased etoposide-induced release of IL-6 and IL-8.  This anti-inflammatory effect was not related to SIRT1 activity, although all bromodomain inhibitors decreased the extent of acetylation of p53 at the SIRT1 deacetylation site.  Overall, since bromodomain inhibitors display anti-inflammatory properties in human retinal pigment epithelial cells, these compds. may represent a new way of alleviating the inflammation underlying the onset of age-related macular degeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocwtuJwbNv5rVg90H21EOLACvtfcHk0lgbDCMC_Yb4Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D&md5=689201c89b6400fe064d3e20ac719bee</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DHytti%26aufirst%3DM.%26aulast%3DTokarz%26aufirst%3DP.%26aulast%3DM%25C3%25A4%25C3%25A4tt%25C3%25A4%26aufirst%3DE.%26aulast%3DPiippo%26aufirst%3DN.%26aulast%3DKorhonen%26aufirst%3DE.%26aulast%3DSuuronen%26aufirst%3DT.%26aulast%3DHonkakoski%26aufirst%3DP.%26aulast%3DKaarniranta%26aufirst%3DK.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26aulast%3DKauppinen%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520BET%2520bromodomains%2520alleviates%2520inflammation%2520in%2520human%2520RPE%2520cells%26jtitle%3DBiochem.%2520Pharmacol.%2520%2528Amsterdam%252C%2520Neth.%2529%26date%3D2016%26volume%3D110%25E2%2580%2593111%26spage%3D71%26epage%3D79%26doi%3D10.1016%2Fj.bcp.2016.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Bandukwala, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagnon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Togher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenbaum, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamperti, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molesworth, A. M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithers, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tough, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, A.</span><span> </span><span class="NLM_article-title">Selective inhibition of CD4<sup>+</sup> T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">14532</span><span class="NLM_x">–</span> <span class="NLM_lpage">14537</span><span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0lgbDCMC_Yb4Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Spiltoir, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demos-Davies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinsey, T. A.</span><span> </span><span class="NLM_article-title">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span> <span class="citation_source-journal">J. Mol. Cell. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2Fj.yjmcc.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=23939492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=175-179&author=J.+Spiltoirauthor=M.+S.+Strattonauthor=M.+A.+Cavasinauthor=K.+Demos-Daviesauthor=B.+G.+Reidauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+A.+McKinsey&title=BET+acetyl-lysine+binding+proteins+control+pathological+cardiac+hypertrophy&doi=10.1016%2Fj.yjmcc.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span></div><div class="casAuthors">Spiltoir, Jessica I.; Stratton, Matthew S.; Cavasin, Maria A.; Demos-Davies, Kim; Reid, Brian G.; Qi, Jun; Bradner, James E.; McKinsey, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-179</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac hypertrophy is an independent predictor of adverse outcomes in patients with heart failure, and thus represents an attractive target for novel therapeutic intervention.  JQ1, a small mol. inhibitor of bromodomain and extraterminal (BET) acetyl-lysine reader proteins, was identified in a high throughput screen designed to discover novel small mol. regulators of cardiomyocyte hypertrophy.  JQ1 dose-dependently blocked agonist-dependent hypertrophy of cultured neonatal rat ventricular myocytes (NRVMs) and reversed the prototypical gene program assocd. with pathol. cardiac hypertrophy.  JQ1 also blocked left ventricular hypertrophy (LVH) and improved cardiac function in adult mice subjected to transverse aortic constriction (TAC).  The BET family consists of BRD2, BRD3, BRD4 and BRDT.  BRD4 protein expression was increased during cardiac hypertrophy, and hypertrophic stimuli promoted recruitment of BRD4 to the transcriptional start site (TSS) of the gene encoding atrial natriuretic factor (ANF).  Binding of BRD4 to the ANF TSS was assocd. with increased phosphorylation of local RNA polymerase II.  These findings define a novel function for BET proteins as signal-responsive regulators of cardiac hypertrophy, and suggest that small mol. inhibitors of these epigenetic reader proteins have potential as therapeutics for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUzCSA2tNtrVg90H21EOLACvtfcHk0ljgokKjHE6NrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF&md5=ffe6d80a94c27e5d89b38baf33af365c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSpiltoir%26aufirst%3DJ.%26aulast%3DStratton%26aufirst%3DM.%2BS.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DDemos-Davies%26aufirst%3DK.%26aulast%3DReid%26aufirst%3DB.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26atitle%3DBET%2520acetyl-lysine%2520binding%2520proteins%2520control%2520pathological%2520cardiac%2520hypertrophy%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D63%26spage%3D175%26epage%3D179%26doi%3D10.1016%2Fj.yjmcc.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Bui, M. H.; Lin, X.; Albert, D. H.; Li, L.; Lam, L. T.; Faivre, E.; Warder, S.; Huang, X.; Wilcox, D.; Donawho, C.; Sheppard, G. S.; Wang, L.; Fidanze, S.; Pratt, J. K.; Liu, D.; Hasvold, L.; Uziel, T.; Fang, G.; Elmore, S. W.; Rosenberg, S. H.; McDaniel, K. F.; Kati, W.; Shen, Y.</span> Comprehensive in vitro and in vivo characterization of a novel BET family bromodomain inhibitor, ABBV-075, reveals sensitive cancer indications and combination strategies for BET inhibitors. Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bui%2C+M.+H.%3B+Lin%2C+X.%3B+Albert%2C+D.+H.%3B+Li%2C+L.%3B+Lam%2C+L.+T.%3B+Faivre%2C+E.%3B+Warder%2C+S.%3B+Huang%2C+X.%3B+Wilcox%2C+D.%3B+Donawho%2C+C.%3B+Sheppard%2C+G.+S.%3B+Wang%2C+L.%3B+Fidanze%2C+S.%3B+Pratt%2C+J.+K.%3B+Liu%2C+D.%3B+Hasvold%2C+L.%3B+Uziel%2C+T.%3B+Fang%2C+G.%3B+Elmore%2C+S.+W.%3B+Rosenberg%2C+S.+H.%3B+McDaniel%2C+K.+F.%3B+Kati%2C+W.%3B+Shen%2C+Y.+Comprehensive+in+vitro+and+in+vivo+characterization+of+a+novel+BET+family+bromodomain+inhibitor%2C+ABBV-075%2C+reveals+sensitive+cancer+indications+and+combination+strategies+for+BET+inhibitors.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DBui%26aufirst%3DM.%2BH." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.-F.</span><span> </span><span class="NLM_article-title">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1375</span><span class="NLM_x">–</span> <span class="NLM_lpage">1387</span><span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=H.-D.+Yangauthor=M.-M.+Zhouauthor=K.+Ozatoauthor=L.-F.+Chen&title=Brd4+coactivates+transcriptional+activation+of+NF-%CE%BAB+via+specific+binding+to+acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0ljgokKjHE6NrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DH.-D.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-%25CE%25BAB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Klein, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabala, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grabiec, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolling, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, L.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tak, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ospelt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reedquist, K. A.</span><span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span> <span class="citation_source-journal">Ann. Rheum. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">422</span><span class="NLM_x">–</span> <span class="NLM_lpage">429</span><span class="refDoi"> DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+G.+Gayauthor=C.+Kollingauthor=L.-L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- α, interleukin (IL)-1β and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- α and IL-1β .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0li73-u-2vnwLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BG.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Ghoshal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yugandhar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, A. K.</span><span> </span><span class="NLM_article-title">BET inhibitors in cancer therapeutics: a patent review</span> <span class="citation_source-journal">Expert Opin. Ther. Pat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">522</span><span class="refDoi"> DOI: 10.1517/13543776.2016.1159299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1517%2F13543776.2016.1159299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=26924192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Wksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=505-522&author=A.+Ghoshalauthor=D.+Yugandharauthor=A.+K.+Srivastava&title=BET+inhibitors+in+cancer+therapeutics%3A+a+patent+review&doi=10.1517%2F13543776.2016.1159299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors in cancer therapeutics: a patent review</span></div><div class="casAuthors">Ghoshal, Anirban; Yugandhar, D.; Srivastava, Ajay Kumar</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-522</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Introduction: Inhibition of Bromodomain and Extra Terminal (BET) proteins is an emerging approach for developing advanced cancer therapeutics.  In 2015, at least thirty patents have been published for developing cancer chemotherapeutics by targeting BET.  Currently there are seven small mol. BET inhibitors in various stages of clin. trials for the development of anti-cancer drugs.  Areas covered: Important patents focusing on development of BET inhibitors as potential cancer therapeutics published in 2015 have been covered.  The reports are presented together with a review of the related structural chem. space.  This review mainly focuses on the therapeutic applications, chem. class and structural modifications along with the mols. currently in clin. trials.  Expert opinion: BET sub-family proteins are one of the emerging targets to develop anti-cancer agents.  Although many research groups have demonstrated the rationality of BET inhibition to combat cancer, a detailed mol. study needs to be performed to investigate the affected biol. pathways.  Selectivity among BET proteins should be kept in mind while developing BET inhibitors.  In-silico mol. modeling studies can also provide valuable information for designing selective BET inhibitors towards anti-cancer drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoByvS8sb7HhLVg90H21EOLACvtfcHk0li73-u-2vnwLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Wksbw%253D&md5=02d7c0cc89a2a712990ceb2e19079f36</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1517%2F13543776.2016.1159299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2016.1159299%26sid%3Dliteratum%253Aachs%26aulast%3DGhoshal%26aufirst%3DA.%26aulast%3DYugandhar%26aufirst%3DD.%26aulast%3DSrivastava%26aufirst%3DA.%2BK.%26atitle%3DBET%2520inhibitors%2520in%2520cancer%2520therapeutics%253A%2520a%2520patent%2520review%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2016%26volume%3D26%26spage%3D505%26epage%3D522%26doi%3D10.1517%2F13543776.2016.1159299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Romero, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Côté, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, S.</span><span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: Progress in the development of bromodomain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1271</span><span class="NLM_x">–</span> <span class="NLM_lpage">1298</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+Progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0li73-u-2vnwLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520Progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lgegQo1MoB1Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudgin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auger, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span><span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lgegQo1MoB1Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Combs, A. P.; Yue, E. W.; Sparks, R. B.; Maduskuie, J. T. P.</span><span> </span><span class="NLM_article-title">Tricyclic heterocycles as BET protein inhibitors</span>. US20150011540 A1,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+P.+Combs&author=E.+W.+Yue&author=R.+B.+Sparks&author=J.+T.+P.+Maduskuie&title=Tricyclic+heterocycles+as+BET+protein+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCombs%26aufirst%3DA.%2BP.%26atitle%3DTricyclic%2520heterocycles%2520as%2520BET%2520protein%2520inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Aktoudianakis, I.; Chin, G.; Corkey, B. K.; Du, J.; Elbel, K.; Jiang, R. H.; Kobayashi, T.; Martinez, R.; Metobo, S. E.; Mish, M.; Shevick, S.; Sperandio, D.; Yang, H.; Zablocki, J.</span><span> </span><span class="NLM_article-title">Benzimidazolone derivatives as bromodomain inhibitors</span>. WO2014160873 A1,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=I.+Aktoudianakis&author=G.+Chin&author=B.+K.+Corkey&author=J.+Du&author=K.+Elbel&author=R.+H.+Jiang&author=T.+Kobayashi&author=R.+Martinez&author=S.+E.+Metobo&author=M.+Mish&author=S.+Shevick&author=D.+Sperandio&author=H.+Yang&author=J.+Zablocki&title=Benzimidazolone+derivatives+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAktoudianakis%26aufirst%3DI.%26atitle%3DBenzimidazolone%2520derivatives%2520as%2520bromodomain%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Engelhardt, H.; Gianni, D.; Mantoulidis, A.; Smethurst, C.</span><span> </span><span class="NLM_article-title">Indolinone analogues as BRD4 inhibitors</span>. WO2014154760 A1,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=H.+Engelhardt&author=D.+Gianni&author=A.+Mantoulidis&author=C.+Smethurst&title=Indolinone+analogues+as+BRD4+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEngelhardt%26aufirst%3DH.%26atitle%3DIndolinone%2520analogues%2520as%2520BRD4%2520inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Demont, E. H.; Gosmini, R. L. M.</span><span> </span><span class="NLM_article-title">Thetrahydroquinolines derivatives as bromodomain inhibitors</span>. WO2011054848 A1,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=E.+H.+Demont&author=R.+L.+M.+Gosmini&title=Thetrahydroquinolines+derivatives+as+bromodomain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThetrahydroquinolines%2520derivatives%2520as%2520bromodomain%2520inhibitors%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bish, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Primiano, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ralph, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciammetta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trzupek, J. D.</span><span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra c-terminal bromodomain inhibition through optimization of a fragment-derived hit</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9831</span><span class="NLM_x">–</span> <span class="NLM_lpage">9837</span><span class="refDoi"> DOI: 10.1021/jm3010515</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+c-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit&doi=10.1021%2Fjm3010515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lgF1K7OYbdD_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520c-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837%26doi%3D10.1021%2Fjm3010515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinges, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augeri, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumeister, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betebenner, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bures, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landis, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nettesheim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zanze, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of a potent inhibitor of the antiapoptotic protein Bcl-X L from NMR and parallel synthesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">656</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span><span class="refDoi"> DOI: 10.1021/jm0507532</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0507532" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=656-663&author=A.+M.+Petrosauthor=J.+Dingesauthor=D.+J.+Augeriauthor=S.+A.+Baumeisterauthor=D.+A.+Betebennerauthor=M.+G.+Buresauthor=S.+W.+Elmoreauthor=P.+J.+Hajdukauthor=M.+K.+Josephauthor=S.+K.+Landisauthor=D.+G.+Nettesheimauthor=S.+H.+Rosenbergauthor=W.+Shenauthor=S.+Thomasauthor=X.+Wangauthor=I.+Zanzeauthor=H.+Zhangauthor=S.+W.+Fesik&title=Discovery+of+a+potent+inhibitor+of+the+antiapoptotic+protein+Bcl-X+L+from+NMR+and+parallel+synthesis&doi=10.1021%2Fjm0507532"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm0507532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0507532%26sid%3Dliteratum%253Aachs%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DDinges%26aufirst%3DJ.%26aulast%3DAugeri%26aufirst%3DD.%2BJ.%26aulast%3DBaumeister%26aufirst%3DS.%2BA.%26aulast%3DBetebenner%26aufirst%3DD.%2BA.%26aulast%3DBures%26aufirst%3DM.%2BG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DJoseph%26aufirst%3DM.%2BK.%26aulast%3DLandis%26aufirst%3DS.%2BK.%26aulast%3DNettesheim%26aufirst%3DD.%2BG.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZanze%26aufirst%3DI.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520inhibitor%2520of%2520the%2520antiapoptotic%2520protein%2520Bcl-X%2520L%2520from%2520NMR%2520and%2520parallel%2520synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D656%26epage%3D663%26doi%3D10.1021%2Fjm0507532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. L.</span><span> </span><span class="NLM_article-title">Macrocycles are great cycles: Applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1961</span><span class="NLM_x">–</span> <span class="NLM_lpage">2004</span><span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+Applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lgF1K7OYbdD_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520Applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Akaji, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teruya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akaji, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aimoto, S.</span><span> </span><span class="NLM_article-title">Synthesis of cyclic RGD derivatives via solid phase macrocyclization using the Heck reaction</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">–</span> <span class="NLM_lpage">2303</span><span class="refDoi"> DOI: 10.1016/S0040-4020(01)00113-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0040-4020%2801%2900113-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=2293-2303&author=K.+Akajiauthor=K.+Teruyaauthor=M.+Akajiauthor=S.+Aimoto&title=Synthesis+of+cyclic+RGD+derivatives+via+solid+phase+macrocyclization+using+the+Heck+reaction&doi=10.1016%2FS0040-4020%2801%2900113-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2801%2900113-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252801%252900113-2%26sid%3Dliteratum%253Aachs%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DTeruya%26aufirst%3DK.%26aulast%3DAkaji%26aufirst%3DM.%26aulast%3DAimoto%26aufirst%3DS.%26atitle%3DSynthesis%2520of%2520cyclic%2520RGD%2520derivatives%2520via%2520solid%2520phase%2520macrocyclization%2520using%2520the%2520Heck%2520reaction%26jtitle%3DTetrahedron%26date%3D2001%26volume%3D57%26spage%3D2293%26epage%3D2303%26doi%3D10.1016%2FS0040-4020%2801%2900113-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Steinman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtin, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garland, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidsen, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holms, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magoc, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagy, I. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchins, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Summers, J. B.</span><span> </span><span class="NLM_article-title">The design, synthesis, and structure-activity relationships of a series of macrocyclic MMP inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2087</span><span class="NLM_x">–</span> <span class="NLM_lpage">2092</span><span class="refDoi"> DOI: 10.1016/S0960-894X(98)00396-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0960-894X%2898%2900396-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=2087-2092&author=D.+H.+Steinmanauthor=M.+L.+Curtinauthor=R.+B.+Garlandauthor=S.+K.+Davidsenauthor=H.+R.+Heymanauthor=J.+H.+Holmsauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=I.+B.+Nagyauthor=P.+A.+Marcotteauthor=J.+Liauthor=D.+W.+Morganauthor=C.+Hutchinsauthor=J.+B.+Summers&title=The+design%2C+synthesis%2C+and+structure-activity+relationships+of+a+series+of+macrocyclic+MMP+inhibitors&doi=10.1016%2FS0960-894X%2898%2900396-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33a&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2898%2900396-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252898%252900396-5%26sid%3Dliteratum%253Aachs%26aulast%3DSteinman%26aufirst%3DD.%2BH.%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DGarland%26aufirst%3DR.%2BB.%26aulast%3DDavidsen%26aufirst%3DS.%2BK.%26aulast%3DHeyman%26aufirst%3DH.%2BR.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DNagy%26aufirst%3DI.%2BB.%26aulast%3DMarcotte%26aufirst%3DP.%2BA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMorgan%26aufirst%3DD.%2BW.%26aulast%3DHutchins%26aufirst%3DC.%26aulast%3DSummers%26aufirst%3DJ.%2BB.%26atitle%3DThe%2520design%252C%2520synthesis%252C%2520and%2520structure-activity%2520relationships%2520of%2520a%2520series%2520of%2520macrocyclic%2520MMP%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1998%26volume%3D8%26spage%3D2087%26epage%3D2092%26doi%3D10.1016%2FS0960-894X%2898%2900396-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit33b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Cherney, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arner, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covington, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardman, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasserman, Z. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaffee, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decicco, C. P.</span><span> </span><span class="NLM_article-title">Macrocyclic hydroxamate inhibitors of matrix metalloproteinases and TNF-alpha production</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1279</span><span class="NLM_x">–</span> <span class="NLM_lpage">1284</span><span class="refDoi"> DOI: 10.1016/S0960-894X(99)00178-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.1016%2FS0960-894X%2899%2900178-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1999&pages=1279-1284&author=R.+J.+Cherneyauthor=L.+Wangauthor=D.+T.+Meyerauthor=C.+B.+Xueauthor=E.+C.+Arnerauthor=R.+A.+Copelandauthor=M.+B.+Covingtonauthor=K.+D.+Hardmanauthor=Z.+R.+Wassermanauthor=B.+D.+Jaffeeauthor=C.+P.+Decicco&title=Macrocyclic+hydroxamate+inhibitors+of+matrix+metalloproteinases+and+TNF-alpha+production&doi=10.1016%2FS0960-894X%2899%2900178-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33b&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2899%2900178-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252899%252900178-X%26sid%3Dliteratum%253Aachs%26aulast%3DCherney%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeyer%26aufirst%3DD.%2BT.%26aulast%3DXue%26aufirst%3DC.%2BB.%26aulast%3DArner%26aufirst%3DE.%2BC.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCovington%26aufirst%3DM.%2BB.%26aulast%3DHardman%26aufirst%3DK.%2BD.%26aulast%3DWasserman%26aufirst%3DZ.%2BR.%26aulast%3DJaffee%26aufirst%3DB.%2BD.%26aulast%3DDecicco%26aufirst%3DC.%2BP.%26atitle%3DMacrocyclic%2520hydroxamate%2520inhibitors%2520of%2520matrix%2520metalloproteinases%2520and%2520TNF-alpha%2520production%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1999%26volume%3D9%26spage%3D1279%26epage%3D1284%26doi%3D10.1016%2FS0960-894X%2899%2900178-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Mallinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1409</span><span class="NLM_x">–</span> <span class="NLM_lpage">1438</span><span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lhm9hFuBjHACg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2615</span><span class="NLM_x">–</span> <span class="NLM_lpage">2623</span><span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.+Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lhm9hFuBjHACg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.%2BY.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span>FreeForm; <span class="NLM_publisher-name">OpenEye Scientific Software</span>: <span class="NLM_publisher-loc">Santa Fe, NM</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=FreeForm%3B+OpenEye+Scientific+Software%3A+Santa+Fe%2C+NM%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DFreeForm%26pub%3DOpenEye%2520Scientific%2520Software%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFO','PDB','5UFO'); return false;">PDB: 5UFO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU','PDB','5UEU'); return false;">PDB: 5UEU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEZ','PDB','5UEZ'); return false;">PDB: 5UEZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEW','PDB','5UEW'); return false;">PDB: 5UEW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEX','PDB','5UEX'); return false;">PDB: 5UEX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEY','PDB','5UEY'); return false;">PDB: 5UEY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i100"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00017">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73841"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00017">10.1021/acs.jmedchem.7b00017</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Description of all biophysical and biological methods, NMR-based fragment screen, bromodomain binding assay, TR-FRET assay conditions, MX-1 proliferation assay conditions, BRD4 engagement luciferase reporter assay conditions, murine multiple myeloma xenograft model, chemistry schemes, and experimental procedures for compounds <b>22</b>–<b>25</b>, <b>31</b>–<b>55</b>, <b>84</b>–<b>94</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_001.pdf">jm7b00017_si_001.pdf (718.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00017/suppl_file/jm7b00017_si_002.csv">jm7b00017_si_002.csv (3.52 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of BRD4 BDII bound to compounds <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UFO">5UFO</a>), <b>2a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEU">5UEU</a>), <b>11</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEZ">5UEZ</a>), <b>25e</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEY">5UEY</a>), and <b>61a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEX">5UEX</a>) and atomic coordinates of BRD2 BDII bound to compound <b>20a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UEW">5UEW</a>) have been deposited with the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00017%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-9%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00017" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994982fbb63d42","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
