
<html lang="en"     class="pb-page"  data-request-id="688c09c1-42eb-4dd5-bdf3-acc855090e27"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2012.55.issue-9;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/jm201609k;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="De Novo Peptide Design with C3a Receptor Agonist and Antagonist Activities: Theoretical Predictions and Experimental Validation" /></meta><meta name="dc.Creator" content="Meghan L.  Bellows-Peterson" /></meta><meta name="dc.Creator" content="Ho Ki  Fung" /></meta><meta name="dc.Creator" content="Christodoulos A.  Floudas" /></meta><meta name="dc.Creator" content="Chris A.  Kieslich" /></meta><meta name="dc.Creator" content="Li  Zhang" /></meta><meta name="dc.Creator" content="Dimitrios  Morikis" /></meta><meta name="dc.Creator" content="Kathryn J.  Wareham" /></meta><meta name="dc.Creator" content="Peter N.  Monk" /></meta><meta name="dc.Creator" content="Owen A.  Hawksworth" /></meta><meta name="dc.Creator" content="Trent M.  Woodruff" /></meta><meta name="dc.Description" content="Targeting the complement component 3a receptor (C3aR) with selective agonists or antagonists is believed to be a viable therapeutic option for several diseases such as stroke, heart attack, reperfu..." /></meta><meta name="Description" content="Targeting the complement component 3a receptor (C3aR) with selective agonists or antagonists is believed to be a viable therapeutic option for several diseases such as stroke, heart attack, reperfu..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 20, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm201609k" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2012 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm201609k" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm201609k" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm201609k" /></link>
        
    
    

<title>De Novo Peptide Design with C3a Receptor Agonist and Antagonist Activities: Theoretical Predictions and Experimental Validation | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm201609k" /></meta><meta property="og:title" content="De Novo Peptide Design with C3a Receptor Agonist and Antagonist Activities: Theoretical Predictions and Experimental Validation" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0004.jpeg" /></meta><meta property="og:description" content="Targeting the complement component 3a receptor (C3aR) with selective agonists or antagonists is believed to be a viable therapeutic option for several diseases such as stroke, heart attack, reperfusion injuries, and rheumatoid arthritis. We designed a number of agonists, partial agonists, and antagonists of C3aR using our two-stage de novo protein design framework. Of the peptides tested using a degranulation assay in C3aR-transfected rat basophilic leukemia cells, two were prominent agonists (EC50 values of 25.3 and 66.2 nM) and two others were partial agonists (IC50 values of 15.4 and 26.1 nM). Further testing of these lead compounds in a calcium flux assay in U937 cells yielded similar results although with reduced potencies compared to transfected cells. The partial agonists also displayed full antagonist activity when tested in a C3aR inhibition assay. In addition, the electrostatic potential profile was shown to potentially discriminate between full agonists and partial agonists." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm201609k"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm201609k">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm201609k&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm201609k&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm201609k&amp;href=/doi/10.1021/jm201609k" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4159-4168</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm201624h" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm2014315" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">De Novo Peptide Design with C3a Receptor Agonist and Antagonist Activities: Theoretical Predictions and Experimental Validation</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Meghan+L.++Bellows-Peterson">Meghan L. Bellows-Peterson</a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ho+Ki++Fung">Ho Ki Fung</a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christodoulos+A.++Floudas">Christodoulos A. Floudas</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chris+A.++Kieslich">Chris A. Kieslich</a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Li++Zhang">Li Zhang</a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dimitrios++Morikis">Dimitrios Morikis</a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kathryn+J.++Wareham">Kathryn J. Wareham</a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+N.++Monk">Peter N. Monk</a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Owen+A.++Hawksworth">Owen A. Hawksworth</a></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Trent+M.++Woodruff">Trent M. Woodruff</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Department of Chemical and Biological Engineering, Princeton University, Princeton, New Jersey 08544, United States</span></div><div class="aff-info" id="aff2"><span class="aff-text">Department of Bioengineering, University of California at Riverside, Riverside, California, United States</span></div><div class="aff-info" id="aff3"><span class="aff-text">Department of Infection and Immunity, The University of Sheffield Medical School, Sheffield, U.K.</span></div><div class="aff-info" id="aff4"><span class="aff-text">School of Biomedical Sciences, The University of Queensland, Brisbane, Australia</span></div><div class="corresp-info"><strong>*</strong>Phone: (609) 258-4595. E-mail: <a href="/cdn-cgi/l/email-protection#cbada7a4beafaab88bbfa2bfaaa5e5bbb9a2a5a8aebfa4a5e5aeafbe"><span class="__cf_email__" data-cfemail="b5d3d9dac0d1d4c6f5c1dcc1d4db9bc5c7dcdbd6d0c1dadb9bd0d1c0">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm201609k&amp;href=/doi/10.1021%2Fjm201609k" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4159â4168</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 13, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 October 2011</li><li><span class="item_label"><b>Published</b> online</span>20 April 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 May 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm201609k" title="DOI URL">https://doi.org/10.1021/jm201609k</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4159%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMeghan%2BL.%2BBellows-Peterson%252C%2BHo%2BKi%2BFung%252C%2BChristodoulos%2BA.%2BFloudas%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D55%26issueNum%3D9%26contentID%3Djm201609k%26title%3DDe%2BNovo%2BPeptide%2BDesign%2Bwith%2BC3a%2BReceptor%2BAgonist%2Band%2BAntagonist%2BActivities%253A%2BTheoretical%2BPredictions%2Band%2BExperimental%2BValidation%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4168%26publicationDate%3DMay%2B2012">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm201609k"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1669</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">27</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm201609k" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;De Novo Peptide Design with C3a Receptor Agonist and Antagonist Activities: Theoretical Predictions and Experimental Validation&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Meghan&quot;,&quot;last_name&quot;:&quot;L. Bellows-Peterson&quot;},{&quot;first_name&quot;:&quot;Ho&quot;,&quot;last_name&quot;:&quot;Ki Fung&quot;},{&quot;first_name&quot;:&quot;Christodoulos&quot;,&quot;last_name&quot;:&quot;A. Floudas&quot;},{&quot;first_name&quot;:&quot;Chris&quot;,&quot;last_name&quot;:&quot;A. Kieslich&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Dimitrios&quot;,&quot;last_name&quot;:&quot;Morikis&quot;},{&quot;first_name&quot;:&quot;Kathryn&quot;,&quot;last_name&quot;:&quot;J. Wareham&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;N. Monk&quot;},{&quot;first_name&quot;:&quot;Owen&quot;,&quot;last_name&quot;:&quot;A. Hawksworth&quot;},{&quot;first_name&quot;:&quot;Trent&quot;,&quot;last_name&quot;:&quot;M. Woodruff&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;4159-4168&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm201609k&quot;},&quot;abstract&quot;:&quot;Targeting the complement component 3a receptor (C3aR) with selective agonists or antagonists is believed to be a viable therapeutic option for several diseases such as stroke, heart attack, reperfusion injuries, and rheumatoid arthritis. We designed a number of agonists, partial agonists, and antagonists of C3aR using our two-stage de novo protein design framework. Of the peptides tested using a degranulation assay in C3aR-transfected rat basophilic leukemia cells, two were prominent agonists (EC50 values of 25.3 and 66.2 nM) and two others were partial agonists (IC50 values of 15.4 and 26.1 nM). Further testing of these lead compounds in a calcium flux assay in U937 cells yielded similar results although with reduced potencies compared to transfected cells. The partial agonists also displayed full antagonist activity when tested in a C3aR inhibition assay. In addition, the electrostatic potential profile was shown to potentially discriminate between full agonists and partial agonists.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm201609k&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201609k" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm201609k&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201609k" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm201609k&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201609k" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm201609k&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm201609k&amp;href=/doi/10.1021/jm201609k" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm201609k" /></input><a href="/doi/pdf/10.1021/jm201609k" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (1 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm201609k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm201609k%26sid%3Dliteratum%253Aachs%26pmid%3D22500977%26genre%3Darticle%26aulast%3DBellows-Peterson%26date%3D2012%26atitle%3DDe%2BNovo%2BPeptide%2BDesign%2Bwith%2BC3a%2BReceptor%2BAgonist%2Band%2BAntagonist%2BActivities%253A%2BTheoretical%2BPredictions%2Band%2BExperimental%2BValidation%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D9%26spage%3D4159%26epage%3D4168%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292589" title="Agonists">Agonists</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jmcmar.2012.55.issue-9/production/jmcmar.2012.55.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Targeting the complement component 3a receptor (C3aR) with selective agonists or antagonists is believed to be a viable therapeutic option for several diseases such as stroke, heart attack, reperfusion injuries, and rheumatoid arthritis. We designed a number of agonists, partial agonists, and antagonists of C3aR using our two-stage de novo protein design framework. Of the peptides tested using a degranulation assay in C3aR-transfected rat basophilic leukemia cells, two were prominent agonists (EC<sub>50</sub> values of 25.3 and 66.2 nM) and two others were partial agonists (IC<sub>50</sub> values of 15.4 and 26.1 nM). Further testing of these lead compounds in a calcium flux assay in U937 cells yielded similar results although with reduced potencies compared to transfected cells. The partial agonists also displayed full antagonist activity when tested in a C3aR inhibition assay. In addition, the electrostatic potential profile was shown to potentially discriminate between full agonists and partial agonists.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37075" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37075" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The complement system is an important part of the immune system and is involved in aiding clearance of immune complexes, debris removal, opsonization, inflammation, and cell lysis.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Improper activation of the complement system, however, can cause tissue injury in various pathological conditions and contributes to several immune diseases, including stroke, heart attack, adult respiratory distress syndrome, septic shock, inflammatory bowl disease, reperfusion injuries, asthma, rheumatoid arthritis, psoriasis, and rejection of xenotransplantation.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> One possibility to prevent this improper activation is to target the complement component 3a receptor, or C3aR.</div><div class="NLM_p">C3aR is a G protein-coupled receptor protein that is targeted by C3a, an anaphylatoxin that mediates the proinflammatory activities of the complement system. C3a is a 77-residue cationic peptide that is derived from the cleavage of the amino-terminus of the Î±-chains of C3. Besides mediating proinflammatory activities, C3a possibly has opposing immunological roles in some cellular systems.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> C3a is known to be a chemoattractant and secretagogue for eosinophils and mast cells<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8">(4-8)</a> and a mediator of smooth muscle contraction.<a onclick="showRef(event, 'ref4 ref9'); return false;" href="javascript:void(0);" class="ref ref4 ref9">(4, 9)</a> In addition, in vitro studies have shown that C3a can suppress or enhance cytokine production which may indicate that C3a is involved in secondary cytokine release and thus inflammatory and immune effects at local sites of inflammation.<a onclick="showRef(event, 'ref4 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref4 ref10 ref11 ref12">(4, 10-12)</a></div><div class="NLM_p">C3a is a potent mediator considering that only micromolar or submicromolar concentrations of C3a are sufficient to elicit its biological effects.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Earlier work by Ember et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> identified a potent C3aR agonist: WWGKKYRASKLGLAR. More recently, Scully et al.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> have reported potent hexapeptide agonists and antagonists that are selective for C3aR: FLPLAR, FIPLAR, FWTLAR, FLTLAR (agonists), and FLTChaAR (antagonist). Given the potential immunomodulatory role of C3a in several diseases, targeting the C3aR with selective agonists or antagonists is a viable therapeutic option.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p">Our de novo protein design framework was applied to the design of novel agonists and antagonists of C3aR, and our best predictions were experimentally validated using a rat basophilic leukemia cell degranulation assay and a monocytic cell calcium flux assay. The de novo protein design framework is a first-principles, computational predictive framework that determines amino acid sequences that are energetically favorable in a given three-dimensional (3D) template structure.<a onclick="showRef(event, 'ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19">(16-19)</a> The sequences can be further validated to be candidate binders of a target protein.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> For the design of C3aR agonists and antagonists, we utilized the 3D structure of the endogenous ligand for C3aR, C3a. The two-stage framework first generates low-energy amino acid sequences and then ranks these sequences based on a fold specificity or an approximate binding affinity. Fold specificity measures how likely the amino acid sequence will adopt the template structure while approximate binding affinity determines how likely the designed peptide or protein will bind to a target protein. For this design, we used only the fold specificity validation metric because structural information of the C3a:C3aR complex, which is necessary to compute the approximate binding affinity, was unavailable in the open literature.</div><div class="NLM_p last">Key features of the de novo protein design framework include the use of deterministic global optimization for sequence selection (determining amino acid sequences that will fold into given template structure),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> a computationally efficient fold specificity metric that can sample hundreds of conformational structures in minutes,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and an approximate binding affinity metric that is a measure of the functionality of the designed sequences.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> Both stages incorporate protein flexibility by introducing the flexibility of the template structure in the sequence selection stage and by generating thousands of flexible conformations of the designed sequences in the validation stage. Our framework has achieved numerous successes including the full sequence design of human-Î²-defensin-2<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and numerous experimentally validated peptides including compstatin inhibitors of human C3,<a onclick="showRef(event, 'ref20 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref22">(20, 22)</a> HIV-1 gp41 inhibitors,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and Bak inhibitors of Bcl-x<sub>L</sub> and Bcl-2.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> These successes have wide applicability to many diseases, including immune diseases, AIDS, and cancer.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57272" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57272" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Computational Design</h3><div class="NLM_p">The de novo protein design framework was used to design agonists and antagonists of C3aR by designing amino acid sequences that would adopt the C3a fold. In this case, because structural information of the C3a:C3aR complex was unknown, we were only able to use our fold specificity ranking metric to determine likely C3aR agonists and antagonists. A summary of this method is provided in the <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a> section, and details can be found in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p">Five separate design runs were performed, each one differing in what mutations were allowed. Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> presents the designed sequences from the latter two runs, along with the sequences designed by Ember et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (E1 and E2) and two sequences from our preliminary work on this system (S3 and S4).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Details on all five runs can be found in the <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a> section, and designed sequences from all five runs can be found in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. The de novo protein design method is freely available to the scientific community at <a href="http://www.proteinwisdom.org" class="extLink">www.proteinwisdom.org</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Sequence Selection and Fold Specificity Results for C3a Agonists/Antagonists<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0008.gif" alt="" id="GRAPHIC-d1914e314-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">Rankings are given for sequence selection (lowest energy =1, <i>E</i>) and fold specificity (highest specificity = 1, <i>f</i><sub>spec</sub>). Mutations away from native C3a are indicated in bold. * Indicates experimentally tested compound.</p></div></div><div></div></div><div class="NLM_p last">In run 4, position 71 consists solely of negatively charged amino acids while position 70 is very varied and mutates to Thr, Tyr, Leu, Ala, Gly, Cys, and Val. Run 5 is interesting in that it ranks negatively charged amino acid mutations in position 65 first and second. This disrupts the +3 charge seen in the native sequence. In fact, the native amino acid of Arg is ranked 10 out of 12 possible sequences.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Experimental Validation</h3><div class="NLM_p">Seven of the designed sequences were selected for synthesis (SQ074â4, SQ062â4, SQ110â4, SQ060â4, SQ007â5, SQ002â5, SQ010â5) in addition to S3, S4, and the two peptides by Ember et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (E1 and E2) for comparison. Peptides were selected from runs 4 and 5. Runs 1â3 provided information on what positions to fix to S4 or mutate, which then motivated runs 4 and 5 (described in the <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a> section). The top three sequences from run 4 based on fold specificity were selected along with the eighth ranked sequences because it retained the native Ala in position 70. The top two sequences from run 5 based on fold specificity were selected in addition to the sixth ranked sequence based upon inspection.</div><div class="NLM_p">Both the agonist and antagonist activity of the designed sequences were measured. Experiments were carried out in the rat basophilic leukemia cell line (RBL-2H3), which resembles a rat serosal mast cell. This cell line is transfected with human receptors for C5a (C5aR/CD88) or C3a (C3aR). Cells that have been exposed to a stimulus (such as the C3a peptides) secrete Î²-hexosaminidase from intracellular granules, providing a read-out of receptor activity. The level of secretion induced by each peptide is shown relative to the maximal secretion stimulated by a high dose of recombinant human C3a or an analogue of the C-terminus of C3a, FLPLAR, and is reported as EC<sub>50</sub>, the half-maximal effective concentration (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>A). Antagonist activity was measured as the inhibition of the response to the same high dose of either C3a or FLPLAR, following a preincubation with peptide and is reported as IC<sub>50</sub>, the half-minimal inhibitory concentration (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>B). Dose response curves for two selected agonists and two antagonists are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Agonist and Antagonist Activities of Peptide Sequences<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center" char=".">Log EC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">SEM</th><th class="colsep0 rowsep0" align="center">significance</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub>(nM)</th><th class="colsep0 rowsep0" align="center" char=".">M (%)</th><th class="colsep0 rowsep0" align="center" char=".">SEM</th><th class="colsep0 rowsep0" align="center">significance</th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="10" align="left">(A)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">S4</td><td class="colsep0 rowsep0" align="char" char=".">â6.71</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">194</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">S3</td><td class="colsep0 rowsep0" align="char" char=".">â6.00</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">1001</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">E1</td><td class="colsep0 rowsep0" align="char" char=".">â6.77</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="center">170</td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">E2</td><td class="colsep0 rowsep0" align="char" char=".">â6.17</td><td class="colsep0 rowsep0" align="char" char=".">0.20</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">675</td><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="char" char=".">5.9</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">SQ074â4</td><td class="colsep0 rowsep0" align="char" char=".">â6.47</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">337</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">SQ062â4</td><td class="colsep0 rowsep0" align="char" char=".">â6.15</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">715</td><td class="colsep0 rowsep0" align="char" char=".">45</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left"><b>SQ110â4</b></td><td class="colsep0 rowsep0" align="char" char="."><b>â7.60</b></td><td class="colsep0 rowsep0" align="char" char="."><b>0.20</b></td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="center"><b>25.3</b></td><td class="colsep0 rowsep0" align="char" char="."><b>72</b></td><td class="colsep0 rowsep0" align="char" char="."><b>4.7</b></td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left"><b>SQ060â4</b></td><td class="colsep0 rowsep0" align="char" char="."><b>â7.18</b></td><td class="colsep0 rowsep0" align="char" char="."><b>0.16</b></td><td class="colsep0 rowsep0" align="left"><b>ns</b></td><td class="colsep0 rowsep0" align="center"><b>66.2</b></td><td class="colsep0 rowsep0" align="char" char="."><b>79</b></td><td class="colsep0 rowsep0" align="char" char="."><b>4.7</b></td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">SQ007â5</td><td class="colsep0 rowsep0" align="char" char=".">â6.65</td><td class="colsep0 rowsep0" align="char" char=".">0.81</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">225</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">SQ002â5</td><td class="colsep0 rowsep0" align="char" char=".">â7.45</td><td class="colsep0 rowsep0" align="char" char=".">0.50</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">35.5</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">SQ010â5</td><td class="colsep0 rowsep0" align="char" char=".">â6.83</td><td class="colsep0 rowsep0" align="char" char=".">1.09</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">146</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">S4-noAc</td><td class="colsep0 rowsep0" align="char" char=".">â7.09</td><td class="colsep0 rowsep0" align="char" char=".">0.30</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">81.6</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" valign="bottom">no.</th><th class="colsep0 rowsep0" align="center" valign="bottom">name</th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">LogâIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">SEM</th><th class="colsep0 rowsep0" align="center" valign="bottom">significance</th><th class="colsep0 rowsep0" align="center" valign="bottom">IC<sub>50</sub>(nM)</th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">Mâ(%)</th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">SEM</th><th class="colsep0 rowsep0" align="center" valign="bottom">significance</th><th class="colsep0 rowsep0" align="center" valign="bottom" char="."><i>n</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="10" align="left">(B)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">S4</td><td class="colsep0 rowsep0" align="char" char=".">â7.30</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">50.2</td><td class="colsep0 rowsep0" align="char" char=".">47</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">S3</td><td class="colsep0 rowsep0" align="char" char=".">â7.63</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">23.4</td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">E1</td><td class="colsep0 rowsep0" align="char" char=".">â7.96</td><td class="colsep0 rowsep0" align="char" char=".">0.40</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="center">10.9</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">4.2</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">E2</td><td class="colsep0 rowsep0" align="char" char=".">â8.73</td><td class="colsep0 rowsep0" align="char" char=".">1.40</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">1.85</td><td class="colsep0 rowsep0" align="char" char=".">62.5</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">SQ074â4</td><td class="colsep0 rowsep0" align="char" char=".">â8.30</td><td class="colsep0 rowsep0" align="char" char=".">0.55</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">5.06</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">SQ062â4</td><td class="colsep0 rowsep0" align="char" char=".">â8.11</td><td class="colsep0 rowsep0" align="char" char=".">0.54</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">7.80</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">SQ110â4</td><td class="colsep0 rowsep0" align="char" char=".">â8.04</td><td class="colsep0 rowsep0" align="char" char=".">0.83</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">9.20</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">SQ060â4</td><td class="colsep0 rowsep0" align="char" char=".">â8.34</td><td class="colsep0 rowsep0" align="char" char=".">1.37</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">4.58</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">5.9</td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left"><b>SQ007â5</b></td><td class="colsep0 rowsep0" align="char" char="."><b>â7.81</b></td><td class="colsep0 rowsep0" align="char" char="."><b>0.30</b></td><td class="colsep0 rowsep0" align="left"><b>ns</b></td><td class="colsep0 rowsep0" align="center"><b>15.4</b></td><td class="colsep0 rowsep0" align="char" char="."><b>24</b></td><td class="colsep0 rowsep0" align="char" char="."><b>5.9</b></td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char="."><b>4</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left"><b>SQ002â5</b></td><td class="colsep0 rowsep0" align="char" char="."><b>â7.58</b></td><td class="colsep0 rowsep0" align="char" char="."><b>0.30</b></td><td class="colsep0 rowsep0" align="left"><b>ns</b></td><td class="colsep0 rowsep0" align="center"><b>26.1</b></td><td class="colsep0 rowsep0" align="char" char="."><b>35</b></td><td class="colsep0 rowsep0" align="char" char="."><b>6.1</b></td><td class="colsep0 rowsep0" align="left">***</td><td class="colsep0 rowsep0" align="char" char="."><b>5</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">SQ010â5</td><td class="colsep0 rowsep0" align="char" char=".">â7.20</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">63.0</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="left">**</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">S4-noAc</td><td class="colsep0 rowsep0" align="char" char=".">â7.91</td><td class="colsep0 rowsep0" align="char" char=".">0.54</td><td class="colsep0 rowsep0" align="left">ns</td><td class="colsep0 rowsep0" align="center">12.4</td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="left">*</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">(A) Agonist activity of peptides using RBL-C3aR cells is shown as Log EC<sub>50</sub> (Â± SEM), EC<sub>50</sub> in nM, and as a calculated maximum (M Â± SEM) relative to levels of secretion stimulated by an optimal concentration of C3a or FLPLAR. (B) Antagonist activity of peptides using RBL-C3aR cells is shown as Log IC<sub>50</sub> (Â± SEM), IC<sub>50</sub> in nM, and as a calculated minimum (M Â± SEM) relative to levels of secretion stimulated by an optimal concentration of C3a or FLPLAR. S4-noAC was the same sequence as S4 but with no N-terminal acetylation. Significance calculated using one-way ANOVA with Dunnetts multiple comparison post-test. ns >0.05, * < 0.05, ** < 0.005, *** < 0.001, compared to no. 3, peptide E1.</p></div></div></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Agonist and antagonist activities of selected peptides. (A) Agonist activity. RBL-C3aR cells were treated with the indicated concentrations of peptides and receptor activation measured by Î²-hexosaminidase secretion. Data are the means of a representative experiment performed in duplicate and are shown as a percentage of the maximal response. (B) Antagonist activity. RBL-C3aR cells were treated with the indicated concentrations of peptides before the addition of an optimal concentration of C3aR agonist peptide FLPLAR. Data are the means Â± SEM of five experiments performed in duplicate and are shown as a percentage of the response to FLPLAR alone. A one-sample <i>t</i> test was used to determine if means were significantly different to 100%; ***: <i>p</i> < 0.0001; **: <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">All the peptides had a degree of agonist activity, but two peptides showed strong agonistic properties (SQ110â4, SQ060â4), with EC<sub>50</sub> values of 25.3 nM and 66.2 nM, respectively, stimulating secretion to 72% and 79% of the maximal response (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>A). The full doseâresponse curves for these peptides are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A. In contrast, the previously reported âsuperagonistâ peptide E1<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> stimulated only 38% of the maximal response, with an EC<sub>50</sub> of 170 nM (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>A). We then tested the peptides for their ability to antagonize C3a activity and, again, all of the peptides had some antagonist activity at low nanomolar concentrations. Two peptides (SQ007â5, SQ002â5) that suppressed C3a activity to 24% and 35% of the maximal response, with IC<sub>50</sub> values of 15.4 and 26.1 nM, respectively, were selected for further study. Their weak agonist effects are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A, and their suppression of the response to FLPLAR is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B. The suppression of the peptide response was less than for C3a, at 48% and 61% of the maximal, but with IC<sub>50</sub> values of 0.95 and 1.6 nM, respectively (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>B).</div><div class="NLM_p">When tested on RBL cells expressing human C5aR, none of the peptides had any agonist or antagonist activity at concentrations of up to 1 Î¼M (data not shown), indicating that the peptides are selective for C3aR.</div><div class="NLM_p">To further test the activities of the peptides SQ110â4, SQ060â4, SQ007â5, and SQ002â5, experiments were performed in dbcAMP differentiated U937 cells. As with the Î²-hexosaminidase assays in RBL-transfected cells, peptides SQ110â4 and SQ060â4 again displayed strong agonistic activity, with calculated EC<sub>50</sub> values of 0.80 and 0.49 Î¼M, respectively, and inducing 81% and 74% of the maximal response (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>A, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). Again, similar to the transfected system, peptides SQ007â5 and SQ002â5 displayed more partial agonist effects in U937 cells, inducing 62% and 59% of the maximal response, with calculated EC<sub>50</sub> values of 2.0 and 0.66 Î¼M, respectively (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>A, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B). Although displaying mixed partial agonist responses, both SQ007â5 and SQ002â5 had full antagonistic activity, with 100 Î¼M of the peptides completely blocking a FLPLAR induced response. Antagonist IC<sub>50</sub> values of 8.95 and 1.12 Î¼M were calculated for SQ110â4 and SQ060â4, respectively (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>B, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Agonist and Antagonist Activities of Peptide Sequences of Interest<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">Log EC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub>(nM)</th><th class="colsep0 rowsep0" align="center" char=".">M (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="left">(A)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">SQ110â4</td><td class="colsep0 rowsep0" align="left">â6.10Â (0.18)</td><td class="colsep0 rowsep0" align="center">797</td><td class="colsep0 rowsep0" align="char" char=".">81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">SQ060â4</td><td class="colsep0 rowsep0" align="left">â6.31Â (0.22)</td><td class="colsep0 rowsep0" align="center">487.4</td><td class="colsep0 rowsep0" align="char" char=".">74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">SQ007â5</td><td class="colsep0 rowsep0" align="left">â4.58Â (0.39)</td><td class="colsep0 rowsep0" align="center">2002</td><td class="colsep0 rowsep0" align="char" char=".">62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">SQ002â5</td><td class="colsep0 rowsep0" align="left">â6.18Â (0.31)</td><td class="colsep0 rowsep0" align="center">654.9</td><td class="colsep0 rowsep0" align="char" char=".">59</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" valign="bottom">no.</th><th class="colsep0 rowsep0" align="center" valign="bottom">name</th><th class="colsep0 rowsep0" align="center" valign="bottom">Log IC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" valign="bottom">IC<sub>50</sub>(nM)</th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">Mâ(%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="left">(B)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">SQ007â5</td><td class="colsep0 rowsep0" align="left">â5.05Â (0.21)</td><td class="colsep0 rowsep0" align="center">8946</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">SQ002â5</td><td class="colsep0 rowsep0" align="left">â5.95Â (0.34)</td><td class="colsep0 rowsep0" align="center">1121</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup>a</sup><p class="last">(A) Agonist activity of peptides in U937 cells is displayed as Log EC<sub>50</sub> (Â± SEM), EC<sub>50</sub> in nM, and as a calculated maximum (M) relative to the response of the cells to 10 Î¼M of the C3aR agonist, FLPLAR. (B) Antagonists activity of peptides in U937 cells is displayed as Log IC<sub>50</sub> (Â±SEM), IC<sub>50</sub> in nM, and as a calculated minimum (M) relative to the response of the cells to 10 Î¼M of the C3aR agonist, FLPLAR, following treatment with the peptides of interest. <i>n</i> = 3â6.</p></div></div></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Agonist and antagonist activities of selected peptides. Agonist activity was measured for peptides SQ110â4 and SQ060â4 (A) and SQ007â5 and SQ002â5 (B). Dibutyryl cAMP differentiated U937 cells were treated with the peptides of interest and receptor activation measured by intracellular calcium mobilization. Following treatment with peptides SQ007â5 and SQ002â5, 10 Î¼M of the C3aR agonist, FLPLAR, was added and antagonist activity of the peptides measured (C). Data is displayed as the mean Â± SEM <i>n</i> = 3â6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Electrostatic Contributions to the Mechanism of Binding</h3><div class="NLM_p">Charge plays significant role in the function of C3a. First, C3a is a cationic protein, and second, C3a function is mediated by its C-terminal amino acid Arg77, which is positively charged. It is known that the C-terminal arginine of C3a (and homologous C5a) is indispensable for binding and biological activity (e.g., Morikis et al.,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Ember et al.,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Scully et al.,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and references therein). In addition, Ember et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> points out the importance of charge because of the presence of conserved arginines at the C-terminus of C3a from various species. These observations indicate the significance of considering charge in the design of our peptides. In addition, in a study of short cyclic peptides derived from the C-terminus of C5a, the C-terminal arginine was replaced by the neutral arginine-like amino acid citruline in two peptides, PMX200 and PMX201. This replacement in PMX200 resulted in 87-fold loss of binding affinity and 335-fold loss of antagonist activity compared to its parent peptide PMX205. The same replacement in PMX201 resulted in 16-fold loss of binding affinity and 293-fold loss of antagonist activity compared to its parent peptide PMX53.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">The 15-amino acid C-terminal segment of C3a, used for our peptide design, is highly cationic with side chain net charge +4 across residues 63â77 and +3 across residues 63â72 (assuming His67 and His72 are neutral). The +3 charge across residues 63â72 was maintained in runs 1â4 of our design by imposing a physicochemical constraint, however, this constraint was dropped for run 5. Because of the role of charge in the function of C3a and in the agonist/antagonist activities of C3a-derived peptides, we present here an analysis of electrostatic potentials. Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>â<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> show that the location of charged amino acids in the peptide discriminates agonist from partial agonist activities. For example, the two full agonists, SQ110â4 and SQ060â4, have a negatively charged amino acid at position 71, whereas the two partial agonists, SQ007â5 and SQ002â5, have a negatively charged amino acid at position 65. In both cases, full and partial agonists, the side chain net charge is +4 across residues 63â77, which makes charge nondiscriminatory for agonism/antagonism. Electrostatic potentials are more informative than net charge or the location of individual charges within the peptide sequence. This is because electrostatic potentials account for the variable dielectric environment of the proteinâsolvent system and for the ionic environment of the solvent, with both dielectric and ionic properties influencing (screening) electrostatic interactions within the peptides. In addition, proximity of charges within the structure can produce additive or subtractive effects in the electrostatic potentials which affect ligandâreceptor recognition. This is information not available when examining charges alone. Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> shows hierarchical clustering of the electrostatic potentials of the designed peptides at ionic strengths corresponding to salt concentrations of 0 and 150 mM. By performing electrostatic clustering analysis with no counterions, electrostatic potentials are more exaggerated presenting global differences in electrostatics, which can affect long-range recognition. Where electrostatic clustering performed with 150 mM of counterions, on the other hand, captures local differences that may affect specific interactions involved in binding. Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> depicts the two full agonists, SQ110â4 and SQ060â4, cluster together and the two partial agonists, SQ007â5 and SQ002â5, cluster together, but separately from the full agonists, at both 0 and 150 mM. Because the agonists and partial agonists cluster separately in both the 0 and 150 mM clustering, this indicates that both local and global effects contribute to agonist and partial agonist activities and suggests a key role for electrostatics. Therefore, the electrostatic potential profile has potentially predictive value, together with the locations of specific amino acid types within the peptide sequences, for the determination of agonist or partial agonist activity. For completion, we have included in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> all 12 peptides of Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> for which we collected experimental data, and three additional sequences of the native peptide in which each or both histidines, His67 and His72, are protonated (charged). The alternative charged states of the native C3a sequence were also analyzed to identify any similarities between the designed analogues and the four potential charged states of the parent peptide. Histidine protonation can result in peptide net charge increase to +5 (single protonated histidine) or +6 (two protonated histidines). On the basis of the electrostatic clustering results, the electrostatic potentials of all of the newly designed sequences are distinct from that of the native sequence, even when considering alternative charge states of histidine.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Hierarchical clustering dendrogram of spatial distributions of electrostatic potentials for the 12 experimentally tested peptides of Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and four native peptides with different combinations of histidine tautomers. Electrostatic potentials were calculated at 0 mM ionic strength (A) and 150 mM ionic strength (B). Isopotential surfaces are shown at two different orientations rotated about the vertical axis and are plotted at Â±2.5 <i>k</i><sub>B</sub><i>T</i>/<i>e</i> (0 mM) and Â±1 <i>k</i><sub>B</sub><i>T</i>/<i>e</i> (150 mM), with blue and red denoting positive and negative electrostatic potential, respectively. The net charge (<i>Q</i>) of the sequences is given and is calculated by taking into account the charged side chains and the negatively charged backbone at the unblocked C-termini (and the positively charged backbone at the N-terminus of S4-noAc). Native peptide His67 and His72 protonation is as follows: (i) C3a-cterm, His67/His72 neutral; (ii) C3a-cterm HIP5, His67 charged, His72 neutral; (iii) C3a-cterm HIP10, His67 neutral, His72 charged; and (iv) C3a-cterm HIP5/10, His67/His72 charged.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53718" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53718" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The de novo protein design framework was applied to the design of C3aR agonists and antagonists. Because structural information on the C3a:C3aR complex was unknown, the design employed the structure of C3a and identified short sequences (15 residues) that were favorable in the C3a folded structure.</div><div class="NLM_p">The computational results provided a number of strong patterns in the mutations of C3a. In particular, the introduction of negatively charged amino acids in positions 65 and 71 elucidated a number of potent agonists and partial agonists. For the majority of the computational runs, a charge of +3 across residues 63â69 was imposed to mimic the charge of the native peptide. However, the best antagonists have either Asp or Glu in position 65, bringing the side chain net charge across residues 63â69 down to +1.</div><div class="NLM_p">Of the seven designed peptides initially tested in our transfected cell system, two were prominent agonists while two others were partial agonists with prominent antagonist activity. These peptides were selected for further testing using a more direct measure of receptor activation with a cell line natively expressing C3aR.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Both systems were able to distinguish between the prominent agonists and partial agonists, although potency of these compounds in the native C3aR expressing cell line was lower than that in the transfected cell system.</div><div class="NLM_p">The two partial agonist peptides were also able to inhibit the activity of both intact C3a and an analogue of the C-terminus of C3a. Ligand binding to C3aR and C5aR involves the cooperation of at least two sites on the receptor. For C3aR, one site comprises charged residues in the very large second extracellular loop but binding here does not lead to receptor activation. Instead, a second binding site located in the pore formed by the helical transmembrane domains must be engaged.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> For C5aR, the first site is located at the receptor N-terminus but the second site has similarities with that of C3aR and some charged residues have been identified that are common to both receptors.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> For C5aR, it is clear that the C-terminus of C5a binds in the transmembrane pores, and so C-terminal peptides of C5a will activate the receptor even when the N-terminus of the receptor has been removed. By analogy, we would expect the C-terminus of C3a to interact with C3aR in a similar way and there is some experimental data to support this view.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> It is thus likely that the family of peptides described here interact with residues in the transmembrane domains of C3aR because they are similar to the C-terminus of C3a and also can antagonize the activity of another peptide (FLPLAR) derived from the C3a C-terminus.</div><div class="NLM_p">Examination of the sequences of the four most potent peptides (nos. 7, 8, 9 and 10) allowed us to gain insight into specific physicochemical properties of their function. In addition to the LGLAR C-terminal sequence, the potent agonists (nos. 7 and 8) contained the N-terminal sequence WWTRRWR and only differed in position 70, with no. 7 mutating to the small amino acid Gly and no. 8 retaining the native Ala. The presence of two N-terminal Trp and an additional aromatic amino acid (Trp) at position 68 suggests the possibility for aromatic ring Ï-stacking that stabilizes the structure of the peptides. This type of interaction may be necessary to stabilize the C-terminal sequence LGLAR, while allowing some flexibility, in a conformation that facilitates interaction with C3aR. It is likely that these aromatic amino acids act as a scaffold, in the absence of the remaining C3a sequence, to orient Arg77 in a proper conformation for interaction with C3aR amino acids. The aromatic amino acids may form hydrophobic and/or Ï-cation contacts with C5aR. We expect the remaining amino acids of LGLAR to form favorable contacts because they are part of the C3a binding site. In the case of the two better partial agonists (nos. 9 and 10), the only difference was Asp65 for no. 9 and Glu65 for no. 10. Despite this difference, both sequences had a negatively charged amino acid in position 65.</div><div class="NLM_p">Of particular interest is the net charge of the potent peptides. All four peptides had a side chain net charge of +4 across the entire 15-residue peptide. However the agonists had a charge of +3 across residues 63â69 while the partial agonists had a charge of +2 across these positions. This difference in charged areas was seen in hierarchical clustering of electrostatic potentials, where the two agonists clustered together and the two partial agonists clustered together, but separately from the agonists.</div><div class="NLM_p">On the basis of the above analysis, the main characteristics for agonist activity were: (i) C-terminal sequence LGLAR, (ii) N-terminal sequence WWTRRWR, (iii) Gly or Ala in position 70, and (iv) side chain net charge of +4 with a charge of +3 across positions 63â69. Similarly, the main characteristics for partial agonist activity were: (i) C-terminal sequence LGLAR, (ii) N-terminal sequence WW, (iii) negatively charged amino acid (D/E) in position 65, and (iv) side chain net charge of +4 with a charge of +2 across positions 63â69. These findings are important for further sequence optimization using our computational protocol in which sequence segments W63W(D/E)65 and L73GLAR77 may be considered indispensable for activity if the side chain net charge is +4, whereas the remaining seven amino acids may be amenable to further optimization. Incorporation of non-natural amino acids may be necessary to improve antagonist activity.</div><div class="NLM_p last">This work highlights the success of the protein design framework. In this case, structural information on the binding site was unknown, so the design was driven by fold specificities. Because structure often implies function, this is a good measure to use when structural binding data is unavailable.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Computational De Novo Design</h3><div class="NLM_p">A two-stage de novo protein design framework<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20 ref21">(16-21)</a> was applied to the design of C3a analogues. The first stage selects amino acid sequences using either a single template or multiple templates defined only by either the C<sup>Î±</sup> positions or the side chain centroids of the template(s). The second stage ranks the sequences from stage one based upon their fold specificity, a measure of how well the sequences assume the template fold. In addition to the design template, mutation sets, biological constraints, and a forcefield are also specified for the calculations. Components specific to the design of C3aR agonists and antagonists are described below, while details about the framework can be found in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p last">Five separate design runs were performed. For each run, the design template and forcefield were kept constant but the allowed mutations were varied. For runs 1â3, 1000 sequences were generated in stage one. For run 4 and run 5, all sequences were generated because the computational complexity was so small (132 and 12 sequences total, respectively). Once the sequences were generated in stage one, fold specificity calculations from stage two were performed in order to rank the sequences based upon the likelihood of assuming the template fold.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31 ref32 ref33 ref34">(28-34)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Design Template</h3><div class="NLM_p">The three-dimensional structure of C3a was first elucidated by Huber et al.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> using X-ray crystallography at a relatively low resolution of 3.2 Ã. However, the structure did not encompass residues 1â12, owing to the presence of either a disordered or mobile ordered structure with respect to the rest of C3a.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">The overall structure of C3a resembles a drumstick that has very little core. Disulfide bridges exist between Cys22âCys49, Cys23âCys56, and Cys36âCys57. These SâS bonds keep the shape of the head of the drumstick structure, and the long carboxy-terminal helix (residues 47â69) forms the stick of the drumstick. Besides the one present in the carboxy-terminus, helices are also found in residue segments 17â23 and 37â41 in the crystal structure. Salt bridges that contribute to packing interactions in C3a are: Glu24âArg20, Asp25âArg28, Glu62âArg65, and weaker ones are observed in Asp25âArg39, Asp55âLys51, Glu24âLys17, and Glu47âLys50.</div><div class="NLM_p">A flexible design template was generated using molecular dynamics (MD) simulations with explicit water molecules as solvent. The initial structure for the MD simulations was constructed as a composite of the C3a domain of the crystal structure of C3 for amino acids Val1âAla69 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A73">2A73</a>)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and the crystal structure of C3a for amino acids Ser70âArg76 (coordinates kindly provided by Prof. Huber; in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> of Morikis et al.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a>). Construction of the composite structure was necessary because the C3a structure is missing the first Î± helix, whereas the C3 structure is missing the 7 C-terminal amino acids. The MD simulation was performed for 10 ns, using the program CHARMM.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> MD snapshots were saved every 1 ns.</div><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> shows the flexible design template. Because previous potent analogues of C3a consisted of only the last 21 or 15 residues of C3a<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and the current binding model of C3a to C3aR suggests interaction between residues 69â77 of C3a to helices 4 and 5â7 of C3aR,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> only these last residues were considered for the design. These last 15 residues are shown in red in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Flexible design template of C3a (superposition of 10 MD snapshots). The last 15 residues are colored in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Mutation Sets</h3><div class="NLM_p">The mutation set specifies which positions can mutate and to what set of amino acids. In the most general sense, every position could be allowed to mutate to all 20 amino acids. To decrease the computational complexity, solvent accessible surface areas of each position or experimental knowledge are used to constrain the number of mutations allowed at each position. As previously stated, only positions 63â77 were designed. The last five residues (LGLAR) of C3a are highly conserved among animal species<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and so were fixed to the native sequence. These residues are considered to be the primary binding site of C3a to C3aR. Previous computational work by Fung<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> elucidated potential candidates for C3a agonists/antagonists with sequences of WWTRRYRASKLGLAR (S3) and WWTRRWRASKLGLAR (S4). Upon the basis of these two sequences, the two mutations to Trp in positions 63 and 64 were imposed and the remaining positions (65â72) were allowed to mutate in five separate computational runs.</div><div class="NLM_p">The first mutation set (MS1) allowed positions 65â68 to mutate. The second mutation set (MS2) allowed positions 70â72 to mutate. The third mutation set (MS3) allowed positions 65 and 70â71 to mutate. The fourth and fifth mutation sets (MS4 and MS5, respectively) split MS3 into two separate runs: MS4 allowed only positions 70 and 71 to mutate, while MS5 allowed only position 65 to mutate. The allowable amino acids at each mutable position were determined by solvent accessible surface area. If a position was more than 50% exposed to solvent, hydrophilic amino acids were allowed, if a position was less than 20% exposed to solvent, hydrophobic amino acids were allowed. Finally, if the exposure was between 20% and 50%, all amino acids were allowed. Positions that were not allowed to mutate were fixed to the sequence of S4. Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a> provides details about each mutation set along with the native sequence of C3a, the sequence of S4, and the sequence by Ember et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (E1). Positions 1â62 were fixed as the native sequence.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Mutation Sets for C3a<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0009.gif" alt="" id="GRAPHIC-d1914e1444-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup>Table a</sup><p class="last">Two sequences (native and S4) are given for reference. A dash (â) indicates that position is fixed with respect to the amino acid in S4. Sequence numbering is with respect to the full C3a sequence. Mutations away from the native sequence of C3a are indicated in bold. For MS1âMS5, <i>B</i> indicates hydrophobic amino acids (A, C, G, I, L, M, F, T, W, Y, V) and <i>Z</i> indicates hydrophilic amino acids (A, R, N, D, Q, E, G, H, K, P, S, T). The native residue was also always allowed at each mutable position. Computational complexity is defined as <i>b</i><sup><i>n</i></sup><i>z</i><sup><i>m</i></sup>, where <i>b</i> and <i>z</i> are the number of amino acids in each hydrophobic and hydrophilic set, respectively, and <i>n</i> and <i>m</i> the number of their respective mutable sequence positions.</p></div></div><div></div></div><div class="NLM_p last">MS4 and MS5 were motivated by the fact that some of the results using MS1 exhibited the mutations seen in the S4 and S3 sequences from Fung.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Using MS1, positions 66 and 67 showed a predominance of Arg, which appears in positions 66 and 67 in both S4 and S3. Thus we decided to fix these two positions to Arg. Position 68 was fixed to Trp because a large hydrophobic amino acid was present in this position in S3, S4, and the âsuperagonistâ elucidated by Ember et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> MS2 showed mutations to Asp and Glu in position 71 and some retention of the native amino acid in position 70. We decided to allow these two positions to mutate but fixed position 72 to Lys, which is the mutation identified by Ember et al.,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> S3, and S4. The remaining mutable positions (65, 70, and 71) were allowed to vary in MS3, which was then split into two separate runs, allowing only position 65 to mutate in MS5 and only positions 70 and 71 to mutate in MS4. Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> provides the top 10 sequences from runs 4 and 5 (using MS4 and MS5, respectively) ranked according to fold specificity. Additionally, the sequences of native C3a, S3, S4, and two potent sequences from Ember et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (E1 and E2) are provided for comparison.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Biological Constraints</h3><div class="NLM_p last">One biological constraint was imposed on the design. For runs 1â4 (corresponding to MS1âMS4), a charge of +3 between residues 63 and 69 was imposed. This is the charge between those residues in the native sequence.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Peptide Synthesis for Experimental Studies</h3><div class="NLM_p last">The peptides were synthesized by GenScript (Piscataway, NJ) with acetylated N-termini, unless otherwise noted, and in quantities of 1â4 mg at >95% purity. Purity was confirmed with HPLC. The C-terminus was unblocked in all peptides.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Rat Basophilic Leukemia Cell Degranulation Assay</h3><div class="NLM_p last">RBL-2H3 cells transfected with either human C5aR or human C3aR<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> were routinely cultured in Dulbeccos Modified Eagles Medium (DMEM), 10% (v/v) fetal calf serum, and 400 mg/Liter G-418 at 37 C in 5% CO<sub>2</sub>. Degranulation was measured by assaying for Î²-hexosaminidase activity in the cell supernatant, as described previously.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In agonist assays, degranulation was calculated as a percentage of maximal activity in response to 200 nM recombinant human C3a, human C5a,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> or 10 nM agonist hexapeptide, FLPLAR.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> For antagonist assays, cells were pretreated with peptides for 10 min prior to the addition of C3a, C5a, or FLPLAR. EC<sub>50</sub>, IC<sub>50</sub>, and standard error values were obtained by iterative curve fitting in GraphPad Prism v5.0.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Calcium Flux Assay</h3><div class="NLM_p last">The human monocytic lymphoma cell line, U937, was maintained in RPMI-1640 medium supplemented with 10% v/v fetal calf serum at 37 Â°C and 5% CO<sub>2</sub>. Prior to experimentation, U937 cells were incubated for 48 h in 1 mM dibutyryl cAMP (Sigma-Aldrich, USA) in order to differentiate the cells into a C3a-responsive macrophage-like cell.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Receptor activation was assessed using a Flexstation 3 microplate reader (Molecular Devices, USA) and Fluo-4 dye (Invitrogen, USA) to detect increases in intracellular calcium indicative of GPCR activation. For the agonist assays, responses were calculated as a percentage of maximal activity compared to 10 Î¼M of the C3aR agonist, FLPLAR. For antagonist assays, cells were pretreated with peptides for 10 min prior to the addition of 10 Î¼M FLPLAR. Doseâresponse curves were plotted and EC<sub>50</sub> and IC<sub>50</sub> values calculated using GraphPad Prism v5.0.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> PoissonâBoltzmann Electrostatics</h3><div class="NLM_p last">For electrostatic clustering, the structures of the experimentally tested C3a peptide analogues were generated using homology modeling with the Automodel module of Modeler 9v9.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The template used for the homology modeling consisted of the 15 C-terminal residues from the last snapshot of the 10 ns MD simulation of whole C3a, described above. Sequence alignments between the native peptide (named C3a-cterm), and the 12 analogues were generated using the R<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> package Bio3D<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and supplied as input to Modeler. The generated models were then prepped using PDB2PQR<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> in order to add atomic radii and partial charges to the coordinate files according to the PARSE<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> forcefield. When adding charges in PDBPQR, the N-termini of all peptides were neutralized because of the addition of the acetyl group, with the exception of S4-noAc, which was positively charged, whereas all C-termini were negatively charged. PDB2PQR was also used to generate four alternative charged states of the native peptide by different combinations of charges for His67 and His72. The number of peptide structures used for electrostatic calculations was 16, including eight newly designed sequences, four known sequences, and four native peptides with alternative protonation state combinations for His67 and His72. Electrostatic potentials were calculated using the linearized PoissonâBoltzmann equation implemented in APBS.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The dielectric surface was defined as the contact surface using a probe sphere with 1.4 Ã radius, resembling a water molecule, and the dielectric coefficients were 20 for the protein interior and 78.54 for the solvent (justification for dielectric of 20 for the protein interior is given in Gorham et al.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>). Electrostatic potentials were calculated by embedding the peptides in a grid with 65 Ã 65 Ã 65 grid points and dimensions of 52 Ã Ã 38 Ã Ã 40 Ã. The peptide structures were superimposed, and the same centering was used in the grid prior to performing the calculations. Two sets of electrostatic potentials were generated, corresponding to counterions at 0 and 150 mM ionic strength. The ion accessibility surface was defined using a probe sphere with a 2 Ã radius, resembling a monovalent ion. Electrostatic similarity calculations and clustering were performed for each set of electrostatic potentials using the Analysis of Electrostatic Similarities of Proteins (AESOP) framework.<a onclick="showRef(event, 'ref47 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49">(47-49)</a> A 16 Ã 16 electrostatic distance matrix was determined for each set of potentials using the pairwise localized difference (LD) measure (eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a>).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a><span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/jm-2011-01609k_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span>where LD<sub>AB</sub> is the similarity between proteins A and B, <i>N</i> is the number of grid points, and Ï<sub>A</sub>(<i>i</i>,<i>j</i>,<i>k</i>) and Ï<sub>B</sub>(<i>i</i>,<i>j</i>,<i>k</i>) refer to the electrostatic potentials of proteins A and B, respectively, at identical grid points (<i>i</i>,<i>j</i>,<i>k</i>). The summation is over all grid points. This LD corresponds to the average of the normalized error between all corresponding grid points. According to this scheme, two peptides are identical if they have LD 0 and totally different if they have LD 2. Hierarchical clustering was performed based on the generated electrostatic distance matrices using average linkage, and the clustering results are presented as dendrograms, using R. The spatial distributions of electrostatic potential were visualized and plotted as isopotential contours using the molecular graphics program Chimera.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20"><a href="/doi/suppl/10.1021/jm201609k">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67300" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67300" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Details about the mathematical model and formulations for the de novo design framework and sequence results from all five computational design runs.  This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm201609k/suppl_file/jm201609k_si_001.pdf">jm201609k_si_001.pdf (204.98 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm201609k" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christodoulos A. Floudas</span> - <span class="hlFld-Affiliation affiliation">Department of Chemical and Biological Engineering,
Princeton University, Princeton, New Jersey 08544, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7f1913100a1b1e0c3f0b160b1e11510f0d16111c1a0b1011511a1b0a"><span class="__cf_email__" data-cfemail="4d2b212238292c3e0d3924392c23633d3f24232e2839222363282938">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meghan L. Bellows-Peterson</span> - <span class="hlFld-Affiliation affiliation">Department of Chemical and Biological Engineering,
Princeton University, Princeton, New Jersey 08544, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ho Ki Fung</span> - <span class="hlFld-Affiliation affiliation">Department of Chemical and Biological Engineering,
Princeton University, Princeton, New Jersey 08544, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chris A. Kieslich</span> - <span class="hlFld-Affiliation affiliation">Department of Bioengineering, University
of California
at Riverside, Riverside, California, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Zhang</span> - <span class="hlFld-Affiliation affiliation">Department of Bioengineering, University
of California
at Riverside, Riverside, California, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dimitrios Morikis</span> - <span class="hlFld-Affiliation affiliation">Department of Bioengineering, University
of California
at Riverside, Riverside, California, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kathryn J. Wareham</span> - <span class="hlFld-Affiliation affiliation">Department
of Infection and Immunity, The University
of Sheffield Medical School, Sheffield, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter N. Monk</span> - <span class="hlFld-Affiliation affiliation">Department
of Infection and Immunity, The University
of Sheffield Medical School, Sheffield, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Owen A. Hawksworth</span> - <span class="hlFld-Affiliation affiliation">School
of Biomedical Sciences, The University of Queensland,
Brisbane, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Trent M. Woodruff</span> - <span class="hlFld-Affiliation affiliation">School
of Biomedical Sciences, The University of Queensland,
Brisbane, Australia</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d1914e1778-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants from NSF (C.A.F. and D.M.), NIH (R01 GM52032, C.A.F.; R24 GM069 736, C.A.F. and D.M.), University of California Tobacco-Related Disease Research Program (D.M. and C.A.K.), the Beckman Initiative for Macular Research (D.M.), and the US Environmental Protection Agency, EPA (R 832721-010, C.A.F.). Although the research described in the article has been funded in part by the US Environmental Protection Agencyâs STAR program, it has not been subjected to any EPA review and therefore does not necessarily reflect the views of the Agency, and no official endorsement should be inferred. A portion of this research was made possible with Government support by DoD, Air Force Office of Scientific Research, National Defense Science and Engineering Graduate (NDSEG) Fellowship, 32 CFR 168a (MLBP).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i22" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i22"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i23" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i23"> Abbreviations Used</h2><tr><td class="NLM_term">C3a</td><td class="NLM_def"><p class="first last">complement component 3a receptor</p></td></tr><tr><td class="NLM_term">C3aR</td><td class="NLM_def"><p class="first last">complement component 3a receptor</p></td></tr><tr><td class="NLM_term">3-D</td><td class="NLM_def"><p class="first last">3-dimensional</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mutation set</p></td></tr><tr><td class="NLM_term">RBL-2H3</td><td class="NLM_def"><p class="first last">rat basophilic leukemia cell line</p></td></tr><tr><td class="NLM_term">C5a</td><td class="NLM_def"><p class="first last">complement component 5a</p></td></tr><tr><td class="NLM_term">C5aR</td><td class="NLM_def"><p class="first last">complement component 5a receptor</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Tamamis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archontis, G.</span><span> </span><span class="NLM_article-title">Species specificity of the complement inhibitor compstatin investigated by all-atom molecular dynamics simulations</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2655</span><span class="NLM_x">â</span> <span class="NLM_lpage">2667</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=2655-2667&issue=12&author=P.+Tamamisauthor=D.+Morikisauthor=C.+A.+Floudasauthor=G.+Archontis&title=Species+specificity+of+the+complement+inhibitor+compstatin+investigated+by+all-atom+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTamamis%26aufirst%3DP.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DArchontis%26aufirst%3DG.%26atitle%3DSpecies%2520specificity%2520of%2520the%2520complement%2520inhibitor%2520compstatin%2520investigated%2520by%2520all-atom%2520molecular%2520dynamics%2520simulations%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2010%26volume%3D78%26issue%3D12%26spage%3D2655%26epage%3D2667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Sahu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Structure and biology of complement protein C3, a connecting link between innate and acquired immunity</span> <span class="citation_source-journal">Immunol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">180</span><span class="NLM_x">, </span> <span class="NLM_fpage">35</span><span class="NLM_x">â</span> <span class="NLM_lpage">48</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1034%2Fj.1600-065X.2001.1800103.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=11414361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvFegsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2001&pages=35-48&author=A.+Sahuauthor=J.+Lambris&title=Structure+and+biology+of+complement+protein+C3%2C+a+connecting+link+between+innate+and+acquired+immunity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and biology of complement protein C3, a connecting link between innate and acquired immunity</span></div><div class="casAuthors">Sahu, Arvind; Lambris, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">180</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-48</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">0105-2896</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Complement protein C3 is a central mol. in the complement system whose activation is essential for all the important functions performed by this system.  After four decades of research it is now well established that C3 functions like a double-edged sword on the one hand it promotes phagocytosis, supports local inflammatory responses against pathogens, and instructs the adaptive immune response to select the appropriate antigens for a humoral response; on the other hand its unregulated activation leads to host cell damage.  In addn., its interactions with the proteins of foreign pathogens may provide a mechanism by which these microorganisms evade complement attack.  Therefore, a clear knowledge of the mol. and its interactions at the mol. level not only may allow the rational design of mol. adjuvants but may also lead to the development of complement inhibitors and new therapeutic agents against infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVtCdSnM10J7Vg90H21EOLACvtfcHk0lhzFXtC_FWFPw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvFegsrc%253D&md5=5b3d6457364b9b4c042eb3713ab4b1d0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1034%2Fj.1600-065X.2001.1800103.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1034%252Fj.1600-065X.2001.1800103.x%26sid%3Dliteratum%253Aachs%26aulast%3DSahu%26aufirst%3DA.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520biology%2520of%2520complement%2520protein%2520C3%252C%2520a%2520connecting%2520link%2520between%2520innate%2520and%2520acquired%2520immunity%26jtitle%3DImmunol.%2520Rev.%26date%3D2001%26volume%3D180%26spage%3D35%26epage%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Morikis, D.; Holland, M. C. H.; Lambris, J. D.</span><span> </span><span class="NLM_article-title">Structure of the anaphylatoxins C3a and C5a</span>. In  <span class="citation_source-book">Structural Biology of the Complement System</span>; <span class="NLM_contrib-group">Morikis, D.; Lambris, J. D.</span>, Eds.; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, FL</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">, </span><div class="note"><p class="first last">Chapter 7</p></div>, pp  <span class="NLM_fpage">161</span><span class="NLM_x">â</span> <span class="NLM_lpage">177</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1201%2F9780849350368" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=161-177&author=D.+Morikis&author=M.+C.+H.+Holland&author=J.+D.+Lambrisauthor=D.+Morikis&author=J.+D.+Lambris&title=Structural+Biology+of+the+Complement+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1201%2F9780849350368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1201%252F9780849350368%26sid%3Dliteratum%253Aachs%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DStructure%2520of%2520the%2520anaphylatoxins%2520C3a%2520and%2520C5a%26btitle%3DStructural%2520Biology%2520of%2520the%2520Complement%2520System%26aulast%3DMorikis%26aufirst%3DD.%26pub%3DCRC%2520Press%26date%3D2005%26spage%3D161%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Proctor, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monk, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanderson, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T. M.</span><span> </span><span class="NLM_article-title">Complement factors C3a and C5a have distinct hemodynamic effects in the rat</span> <span class="citation_source-journal">Int. Immunopharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">800</span><span class="NLM_x">â</span> <span class="NLM_lpage">806</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1016%2Fj.intimp.2009.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=19285573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsVequrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=800-806&author=L.+M.+Proctorauthor=T.+A.+Mooreauthor=P.+N.+Monkauthor=S.+D.+Sandersonauthor=S.+M.+Taylorauthor=T.+M.+Woodruff&title=Complement+factors+C3a+and+C5a+have+distinct+hemodynamic+effects+in+the+rat"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Complement factors C3a and C5a have distinct hemodynamic effects in the rat</span></div><div class="casAuthors">Proctor, Lavinia M.; Moore, Tyson A.; Monk, Peter N.; Sanderson, Sam D.; Taylor, Stephen M.; Woodruff, Trent M.</div><div class="citationInfo"><span class="NLM_cas:title">International Immunopharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">800-806</span>CODEN:
                <span class="NLM_cas:coden">IINMBA</span>;
        ISSN:<span class="NLM_cas:issn">1567-5769</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the rat, complement C5a infusion mediates well-defined effects including hypotension and neutropenia.  Conversely, the comparative effect of C3a in the rat is not yet defined.  Here, the authors investigated C3a receptor (C3aR) activation in the rat, using recombinant human C3a, the C3aR agonist WWGKKYRASKLGLAR, which is a C-terminal analog of C3a, and a non-peptide C3aR antagonist SB-290157, as pharmacol. tools.  In vitro, C3a and WWGKKYRASKLGLAR selectively bound to C3aRs and induced degranulation of C3aR-transfected RBL-2H3 cells.  C3a or WWGKKYRASKLGLAR-induced degranulation was dose-dependently antagonized in a surmountable fashion by the non-peptide C3aR antagonist.  I.v. infusion of C3a and WWGKKYRASKLGLAR to rats induced a rapid, transient and concn.-dependent hypertensive response, which was mediated by C3aR-induced prostanoid release.  C3a and WWGKKYRASKLGLAR caused a small drop in circulating neutrophils, but a rise in circulating neutrophils was evident after 90-120 min.  In contrast to C3a, C5a infusion resulted in hypotension, and rapid and transient neutropenia.  Thus, C3a and C5a mediate distinct effects on blood pressure and circulating polymorphonuclear leukocytes in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq91jiRORQJvLVg90H21EOLACvtfcHk0lix2Oe8uVuuzg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsVequrs%253D&md5=9b40aa2b1c555fe65b4517517a9322c7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.intimp.2009.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.intimp.2009.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DProctor%26aufirst%3DL.%2BM.%26aulast%3DMoore%26aufirst%3DT.%2BA.%26aulast%3DMonk%26aufirst%3DP.%2BN.%26aulast%3DSanderson%26aufirst%3DS.%2BD.%26aulast%3DTaylor%26aufirst%3DS.%2BM.%26aulast%3DWoodruff%26aufirst%3DT.%2BM.%26atitle%3DComplement%2520factors%2520C3a%2520and%2520C5a%2520have%2520distinct%2520hemodynamic%2520effects%2520in%2520the%2520rat%26jtitle%3DInt.%2520Immunopharmacol.%26date%3D2009%26volume%3D9%26spage%3D800%26epage%3D806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Daffern, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeifer, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ember, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span> </span><span class="NLM_article-title">C3a is a chemotaxin for human eosinophils but not for neutrophils. I. C3a stimulation of neutrophils is secondary to eosinophil activation</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">181</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">2119</span><span class="NLM_x">â</span> <span class="NLM_lpage">2127</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=1995&pages=2119-2127&issue=6&author=P.+J.+Daffernauthor=P.+H.+Pfeiferauthor=J.+A.+Emberauthor=T.+E.+Hugli&title=C3a+is+a+chemotaxin+for+human+eosinophils+but+not+for+neutrophils.+I.+C3a+stimulation+of+neutrophils+is+secondary+to+eosinophil+activation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDaffern%26aufirst%3DP.%2BJ.%26aulast%3DPfeifer%26aufirst%3DP.%2BH.%26aulast%3DEmber%26aufirst%3DJ.%2BA.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26atitle%3DC3a%2520is%2520a%2520chemotaxin%2520for%2520human%2520eosinophils%2520but%2520not%2520for%2520neutrophils.%2520I.%2520C3a%2520stimulation%2520of%2520neutrophils%2520is%2520secondary%2520to%2520eosinophil%2520activation%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D1995%26volume%3D181%26issue%3D6%26spage%3D2119%26epage%3D2127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Elsner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czech, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¶pf, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norgauer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapp, A.</span><span> </span><span class="NLM_article-title">C3a activates reactive oxygen radical species production and intracellular calcium transients in human eosinophils</span> <span class="citation_source-journal">Eur. J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">518</span><span class="NLM_x">â</span> <span class="NLM_lpage">522</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1994&pages=518-522&issue=3&author=J.+Elsnerauthor=M.+Oppermannauthor=W.+Czechauthor=G.+Dobosauthor=E.+Sch%C3%B6pfauthor=J.+Norgauerauthor=A.+Kapp&title=C3a+activates+reactive+oxygen+radical+species+production+and+intracellular+calcium+transients+in+human+eosinophils"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DElsner%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DM.%26aulast%3DCzech%26aufirst%3DW.%26aulast%3DDobos%26aufirst%3DG.%26aulast%3DSch%25C3%25B6pf%26aufirst%3DE.%26aulast%3DNorgauer%26aufirst%3DJ.%26aulast%3DKapp%26aufirst%3DA.%26atitle%3DC3a%2520activates%2520reactive%2520oxygen%2520radical%2520species%2520production%2520and%2520intracellular%2520calcium%2520transients%2520in%2520human%2520eosinophils%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D1994%26volume%3D24%26issue%3D3%26spage%3D518%26epage%3D522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Hartmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henz, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KrÃ¼ger-Krasagakes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KÃ¶hl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guhl, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haase, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippert, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuberbier, T.</span><span> </span><span class="NLM_article-title">C3a and C5a stimulate chemotaxis of human mast cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2863</span><span class="NLM_x">â</span> <span class="NLM_lpage">2870</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1997&pages=2863-2870&issue=8&author=K.+Hartmannauthor=B.+M.+Henzauthor=S.+Kr%C3%BCger-Krasagakesauthor=J.+K%C3%B6hlauthor=R.+Burgerauthor=S.+Guhlauthor=I.+Haaseauthor=U.+Lippertauthor=T.+Zuberbier&title=C3a+and+C5a+stimulate+chemotaxis+of+human+mast+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DK.%26aulast%3DHenz%26aufirst%3DB.%2BM.%26aulast%3DKr%25C3%25BCger-Krasagakes%26aufirst%3DS.%26aulast%3DK%25C3%25B6hl%26aufirst%3DJ.%26aulast%3DBurger%26aufirst%3DR.%26aulast%3DGuhl%26aufirst%3DS.%26aulast%3DHaase%26aufirst%3DI.%26aulast%3DLippert%26aufirst%3DU.%26aulast%3DZuberbier%26aufirst%3DT.%26atitle%3DC3a%2520and%2520C5a%2520stimulate%2520chemotaxis%2520of%2520human%2520mast%2520cells%26jtitle%3DBlood%26date%3D1997%26volume%3D89%26issue%3D8%26spage%3D2863%26epage%3D2870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Takafuji, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tadokoro, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">lto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahinden, C. A.</span><span> </span><span class="NLM_article-title">Degranulation from human eosinophils stimulated with C3a and C5a</span> <span class="citation_source-journal">Int. Arch. Allergy Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">27</span><span class="NLM_x">â</span> <span class="NLM_lpage">29</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1159%2F000236743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=8156000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK2cXjt1Whtbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=1994&pages=27-29&issue=Suppl.+1&author=S.+Takafujiauthor=K.+Tadokoroauthor=K.+ltoauthor=C.+A.+Dahinden&title=Degranulation+from+human+eosinophils+stimulated+with+C3a+and+C5a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Degranulation from human eosinophils stimulated with C3a and C5a</span></div><div class="casAuthors">Takafuji, Shigeru; Tadokoro, Kenji; Ito, Koji; Dahinden, Clemens A.</div><div class="citationInfo"><span class="NLM_cas:title">International Archives of Allergy and Immunology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">SUPPL. 1</span>),
    <span class="NLM_cas:pages">27-9</span>CODEN:
                <span class="NLM_cas:coden">IAAIEG</span>;
        ISSN:<span class="NLM_cas:issn">1018-2438</span>.
    </div><div class="casAbstract">It is unclear what actually induces eosinophil degranulation in vivo.  The authors examd. eosinophil cationic protein (ECP) release from normal human eosinophils (Eos) in response to C3a and C5a.  C3a and C5a induced remarkable ECP release when Eos were preincubated with cytochalasin B.  ECP release induced by C3a or C5a was greater than that induced by PAF at a concn. of 10-7 M.  The ED50 for ECP release was 3Ã10-8 M and 3Ã10-9 M for C3a and C5a, resp.  C3a or C5a elicited a rapid and transient rise in [Ca2+]i.  Thus, C3a and C5a may contribute to hypersensitivity diseases by inducing eosinophil degranulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0MNKtke8ZF7Vg90H21EOLACvtfcHk0ljYCA8-IKUvvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjt1Whtbg%253D&md5=a36b5462c62c62e290ed646820ff7609</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1159%2F000236743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000236743%26sid%3Dliteratum%253Aachs%26aulast%3DTakafuji%26aufirst%3DS.%26aulast%3DTadokoro%26aufirst%3DK.%26aulast%3Dlto%26aufirst%3DK.%26aulast%3DDahinden%26aufirst%3DC.%2BA.%26atitle%3DDegranulation%2520from%2520human%2520eosinophils%2520stimulated%2520with%2520C3a%2520and%2520C5a%26jtitle%3DInt.%2520Arch.%2520Allergy%2520Immunol.%26date%3D1994%26volume%3D104%26issue%3DSuppl.%25201%26spage%3D27%26epage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Marceau, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundberg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span> </span><span class="NLM_article-title">Effects of the anaphylatoxins in circulation</span> <span class="citation_source-journal">Immunopharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">67</span><span class="NLM_x">â</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1016%2F0162-3109%2887%2990031-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=2828271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaL1cXktlar" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1987&pages=67-84&issue=2&author=F.+Marceauauthor=C.+Lundbergauthor=T.+E.+Hugli&title=Effects+of+the+anaphylatoxins+in+circulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of the anaphylatoxins on circulation</span></div><div class="casAuthors">Marceau, Francois; Lundberg, Claes; Hugli, Tony E.</div><div class="citationInfo"><span class="NLM_cas:title">Immunopharmacology</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-84</span>CODEN:
                <span class="NLM_cas:coden">IMMUDP</span>;
        ISSN:<span class="NLM_cas:issn">0162-3109</span>.
    </div><div class="casAbstract">A review with 87 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEpPEJ0vvgK7Vg90H21EOLACvtfcHk0ljYCA8-IKUvvQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktlar&md5=9f2e1971589d924b9ec941e5329210c5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2F0162-3109%2887%2990031-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0162-3109%252887%252990031-2%26sid%3Dliteratum%253Aachs%26aulast%3DMarceau%26aufirst%3DF.%26aulast%3DLundberg%26aufirst%3DC.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26atitle%3DEffects%2520of%2520the%2520anaphylatoxins%2520in%2520circulation%26jtitle%3DImmunopharmacology%26date%3D1987%26volume%3D14%26issue%3D2%26spage%3D67%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Fischer, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagels, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span> </span><span class="NLM_article-title">Regulation of IL-6 synthesis in human peripheral blood mononuclear cells by C3a and C3adesArg</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">453</span><span class="NLM_x">â</span> <span class="NLM_lpage">459</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=1999&pages=453-459&issue=1&author=W.+H.+Fischerauthor=M.+A.+Jagelsauthor=T.+E.+Hugli&title=Regulation+of+IL-6+synthesis+in+human+peripheral+blood+mononuclear+cells+by+C3a+and+C3adesArg"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DW.%2BH.%26aulast%3DJagels%26aufirst%3DM.%2BA.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26atitle%3DRegulation%2520of%2520IL-6%2520synthesis%2520in%2520human%2520peripheral%2520blood%2520mononuclear%2520cells%2520by%2520C3a%2520and%2520C3adesArg%26jtitle%3DJ.%2520Immunol.%26date%3D1999%26volume%3D162%26issue%3D1%26spage%3D453%26epage%3D459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Takabayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vannier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tompkins, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelfand, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, B. D.</span><span> </span><span class="NLM_article-title">Both C3a and C3a desArg regulate Interleukin-6 synthesis in human peripheral blood mononuclear cells</span> <span class="citation_source-journal">J. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">177</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1622</span><span class="NLM_x">â</span> <span class="NLM_lpage">1628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1086%2F515316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=9607842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK1cXjslKitrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=1998&pages=1622-1628&issue=6&author=T.+Takabayashiauthor=E.+Vannierauthor=J.+F.+Burkeauthor=R.+G.+Tompkinsauthor=J.+A.+Gelfandauthor=B.+D.+Clark&title=Both+C3a+and+C3a+desArg+regulate+Interleukin-6+synthesis+in+human+peripheral+blood+mononuclear+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Both C3a and C3adesArg regulate interleukin-6 synthesis in human peripheral blood mononuclear cells</span></div><div class="casAuthors">Takabayashi, Tsukasa; Vannier, Edouard; Burke, John F.; Tompkins, Ronald G.; Gelfand, Jeffrey A.; Clark, Burton D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1622-1628</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Synthesis of complement components is part of the acute-phase response.  Interleukin-6 (IL-6) is a crit. mediator of the acute-phase response during infections and injuries.  Plasma levels of C3a and IL-6 have been proposed as prognostic indicators in sepsis and trauma.  The effects of C3a and C3adesArg on IL-6 gene expression and protein prodn. in human peripheral blood mononuclear cells (PBMC) were investigated.  Neither C3a nor C3adesArg alone induced detectable IL-6 protein or mRNA levels.  However, C3a and C3adesArg affected endotoxin-induced IL-6 synthesis in a dose-dependent manner.  In nonadherent PBMC, C3a or C3adesArg suppressed, while in adherent PBMC, C3a or C3adesArg enhanced IL-6 protein and mRNA levels.  Thus, C3a and C3adesArg may provide a control mechanism of acute-phase responses by enhancing IL-6 synthesis in adherent monocytes at local inflammatory sites and by inhibiting IL-6 synthesis in circulating monocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfaUH9cD90SbVg90H21EOLACvtfcHk0lgnRAA9sbTVFA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjslKitrs%253D&md5=158b02ca08e2f0ee388ac7d005bdf6ad</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1086%2F515316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F515316%26sid%3Dliteratum%253Aachs%26aulast%3DTakabayashi%26aufirst%3DT.%26aulast%3DVannier%26aufirst%3DE.%26aulast%3DBurke%26aufirst%3DJ.%2BF.%26aulast%3DTompkins%26aufirst%3DR.%2BG.%26aulast%3DGelfand%26aufirst%3DJ.%2BA.%26aulast%3DClark%26aufirst%3DB.%2BD.%26atitle%3DBoth%2520C3a%2520and%2520C3a%2520desArg%2520regulate%2520Interleukin-6%2520synthesis%2520in%2520human%2520peripheral%2520blood%2520mononuclear%2520cells%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D1998%26volume%3D177%26issue%3D6%26spage%3D1622%26epage%3D1628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Takabayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vannier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinarello, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelfand, J.</span><span> </span><span class="NLM_article-title">A new biologic role for C3a and C3a desArg: regulation of TNF-alpha and IL-1 beta synthesis</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">156</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">3455</span><span class="NLM_x">â</span> <span class="NLM_lpage">3460</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=1996&pages=3455-3460&issue=9&author=T.+Takabayashiauthor=E.+Vannierauthor=B.+Clarkauthor=N.+Margolisauthor=C.+Dinarelloauthor=J.+Burkeauthor=J.+Gelfand&title=A+new+biologic+role+for+C3a+and+C3a+desArg%3A+regulation+of+TNF-alpha+and+IL-1+beta+synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTakabayashi%26aufirst%3DT.%26aulast%3DVannier%26aufirst%3DE.%26aulast%3DClark%26aufirst%3DB.%26aulast%3DMargolis%26aufirst%3DN.%26aulast%3DDinarello%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DJ.%26aulast%3DGelfand%26aufirst%3DJ.%26atitle%3DA%2520new%2520biologic%2520role%2520for%2520C3a%2520and%2520C3a%2520desArg%253A%2520regulation%2520of%2520TNF-alpha%2520and%2520IL-1%2520beta%2520synthesis%26jtitle%3DJ.%2520Immunol.%26date%3D1996%26volume%3D156%26issue%3D9%26spage%3D3455%26epage%3D3460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Ember, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T.</span><span> </span><span class="NLM_article-title">Designing synthetic superagonists of C3a anaphylatoxin</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3603</span><span class="NLM_x">â</span> <span class="NLM_lpage">3612</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi00229a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=3603-3612&author=J.+Emberauthor=N.+Johansenauthor=T.+Hugli&title=Designing+synthetic+superagonists+of+C3a+anaphylatoxin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi00229a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00229a003%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DJ.%26aulast%3DJohansen%26aufirst%3DN.%26aulast%3DHugli%26aufirst%3DT.%26atitle%3DDesigning%2520synthetic%2520superagonists%2520of%2520C3a%2520anaphylatoxin%26jtitle%3DBiochemistry%26date%3D1991%26volume%3D30%26spage%3D3603%26epage%3D3612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Scully, C. C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blakeney, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoang, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abenante, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlie, D. P.</span><span> </span><span class="NLM_article-title">Selective hexapeptide agonists and antagonists for human complement C3a receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4938</span><span class="NLM_x">â</span> <span class="NLM_lpage">4948</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1003705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntFyksbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4938-4948&author=C.+C.+G.+Scullyauthor=J.+S.+Blakeneyauthor=R.+Singhauthor=H.+N.+Hoangauthor=G.+Abenanteauthor=R.+C.+Reidauthor=D.+P.+Fairlie&title=Selective+hexapeptide+agonists+and+antagonists+for+human+complement+C3a+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Hexapeptide Agonists and Antagonists for Human Complement C3a Receptor</span></div><div class="casAuthors">Scully, Conor C. G.; Blakeney, Jade S.; Singh, Ranee; Hoang, Huy N.; Abbenante, Giovanni; Reid, Robert C.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4938-4948</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human anaphylatoxin C3a, formed through cleavage of complement protein C3, is a potent effector of innate immunity via activation of its G protein coupled receptor, human C3aR.  Previously reported short peptide ligands for this receptor either have low potency or lack receptor selectivity.  Here the authors report the first small peptide agonists that are both potent and selective for human C3aR, derived from structure-activity relationships of peptides based on the C-terminus of C3a.  Affinity for C3aR was examd. by competitive binding with 125I-labeled C3a to human macrophages, agonist vs. antagonist activity measured using fluorescence detection of intracellular calcium, and general selectivity monitored by C3a-induced receptor desensitization.  An NMR structure for an agonist in DMSO showed a Î²-turn motif that may be important for C3aR binding and activation.  Derivatization produced a noncompetitive and insurmountable antagonist of C3aR.  Small mol. C3a agonists and antagonists may be valuable probes of immunity and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9WeiihoJlNbVg90H21EOLACvtfcHk0ljy9VV5nmeRmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntFyksbo%253D&md5=a52f9cf07b0e4b5df3b2ee362d1e004f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm1003705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1003705%26sid%3Dliteratum%253Aachs%26aulast%3DScully%26aufirst%3DC.%2BC.%2BG.%26aulast%3DBlakeney%26aufirst%3DJ.%2BS.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DAbenante%26aufirst%3DG.%26aulast%3DReid%26aufirst%3DR.%2BC.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DSelective%2520hexapeptide%2520agonists%2520and%2520antagonists%2520for%2520human%2520complement%2520C3a%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4938%26epage%3D4948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Holland, M. C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Synthetic small-molecule complement inhibitors</span> <span class="citation_source-journal">Curr. Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1164</span><span class="NLM_x">â</span> <span class="NLM_lpage">1173</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=1164-1173&issue=11&author=M.+C.+H.+Hollandauthor=D.+Morikisauthor=J.+Lambris&title=Synthetic+small-molecule+complement+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHolland%26aufirst%3DM.%2BC.%2BH.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DSynthetic%2520small-molecule%2520complement%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Invest.%2520Drugs%26date%3D2004%26volume%3D5%26issue%3D11%26spage%3D1164%26epage%3D1173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsokos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argyropoulos, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spruce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Integrated computational and experimental approach for lead optimization and design of compstatin variants with improved activity</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">8422</span><span class="NLM_x">â</span> <span class="NLM_lpage">8423</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja034846p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=8422-8423&author=J.+Klepeisauthor=C.+Floudasauthor=D.+Morikisauthor=C.+Tsokosauthor=E.+Argyropoulosauthor=L.+Spruceauthor=J.+Lambris&title=Integrated+computational+and+experimental+approach+for+lead+optimization+and+design+of+compstatin+variants+with+improved+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja034846p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034846p%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DFloudas%26aufirst%3DC.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DTsokos%26aufirst%3DC.%26aulast%3DArgyropoulos%26aufirst%3DE.%26aulast%3DSpruce%26aufirst%3DL.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DIntegrated%2520computational%2520and%2520experimental%2520approach%2520for%2520lead%2520optimization%2520and%2520design%2520of%2520compstatin%2520variants%2520with%2520improved%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D8422%26epage%3D8423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsokos, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J. D.</span><span> </span><span class="NLM_article-title">Design of peptide analogs with improved activity using a novel de novo protein design approach</span> <span class="citation_source-journal">Ind. Eng. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3817</span><span class="NLM_x">â</span> <span class="NLM_lpage">3826</span></span><div class="citationLinks">[<a href="/doi/10.1021/ie0340995" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=3817-3826&author=J.+L.+Klepeisauthor=C.+A.+Floudasauthor=D.+Morikisauthor=C.+G.+Tsokosauthor=J.+D.+Lambris&title=Design+of+peptide+analogs+with+improved+activity+using+a+novel+de+novo+protein+design+approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fie0340995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fie0340995%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DTsokos%26aufirst%3DC.%2BG.%26aulast%3DLambris%26aufirst%3DJ.%2BD.%26atitle%3DDesign%2520of%2520peptide%2520analogs%2520with%2520improved%2520activity%2520using%2520a%2520novel%2520de%2520novo%2520protein%2520design%2520approach%26jtitle%3DInd.%2520Eng.%2520Chem.%2520Res.%26date%3D2004%26volume%3D43%26spage%3D3817%26epage%3D3826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Novel formulations for the sequence selection problem in de novo protein design with flexible templates</span> <span class="citation_source-journal">Optimization Methods Software</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">â</span> <span class="NLM_lpage">71</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1080%2F10556780600882041" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2007&pages=51-71&author=H.+K.+Fungauthor=M.+S.+Taylorauthor=C.+A.+Floudas&title=Novel+formulations+for+the+sequence+selection+problem+in+de+novo+protein+design+with+flexible+templates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F10556780600882041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10556780600882041%26sid%3Dliteratum%253Aachs%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DNovel%2520formulations%2520for%2520the%2520sequence%2520selection%2520problem%2520in%2520de%2520novo%2520protein%2520design%2520with%2520flexible%2520templates%26jtitle%3DOptimization%2520Methods%2520Software%26date%3D2007%26volume%3D22%26spage%3D51%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">Toward full-sequence de novo protein design with flexible templates for human beta-defensin-2</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">584</span><span class="NLM_x">â</span> <span class="NLM_lpage">599</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2008&pages=584-599&author=H.+K.+Fungauthor=C.+A.+Floudasauthor=M.+S.+Taylorauthor=L.+Zhangauthor=D.+Morikis&title=Toward+full-sequence+de+novo+protein+design+with+flexible+templates+for+human+beta-defensin-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DToward%2520full-sequence%2520de%2520novo%2520protein%2520design%2520with%2520flexible%2520templates%2520for%2520human%2520beta-defensin-2%26jtitle%3DBiophys.%2520J.%26date%3D2008%26volume%3D94%26spage%3D584%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Bellows, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez de Victoria, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">New compstatin variants through two de novo protein design frameworks</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">2337</span><span class="NLM_x">â</span> <span class="NLM_lpage">2346</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2010&pages=2337-2346&issue=10&author=M.+L.+Bellowsauthor=H.+K.+Fungauthor=C.+A.+Floudasauthor=A.+Lopez+de+Victoriaauthor=D.+Morikis&title=New+compstatin+variants+through+two+de+novo+protein+design+frameworks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBellows%26aufirst%3DM.%2BL.%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DLopez%2Bde%2BVictoria%26aufirst%3DA.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DNew%2520compstatin%2520variants%2520through%2520two%2520de%2520novo%2520protein%2520design%2520frameworks%26jtitle%3DBiophys.%2520J.%26date%3D2010%26volume%3D98%26issue%3D10%26spage%3D2337%26epage%3D2346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Bellows, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siliciano, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Discovery of entry inhibitors for HIV-1 via a new de novo protein design framework</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">3445</span><span class="NLM_x">â</span> <span class="NLM_lpage">3453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2010&pages=3445-3453&author=M.+L.+Bellowsauthor=M.+S.+Taylorauthor=P.+A.+Coleauthor=L.+Shenauthor=R.+F.+Silicianoauthor=H.+K.+Fungauthor=C.+A.+Floudas&title=Discovery+of+entry+inhibitors+for+HIV-1+via+a+new+de+novo+protein+design+framework"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBellows%26aufirst%3DM.%2BL.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DSiliciano%26aufirst%3DR.%2BF.%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DDiscovery%2520of%2520entry%2520inhibitors%2520for%2520HIV-1%2520via%2520a%2520new%2520de%2520novo%2520protein%2520design%2520framework%26jtitle%3DBiophys.%2520J.%26date%3D2010%26volume%3D99%26spage%3D3445%26epage%3D3453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">LÃ³pez de Victoria, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorham, R. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellows-Peterson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">A new generation of potent complement inhibitors of the compstatin family</span> <span class="citation_source-journal">Chem. Biol. Drug. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">â</span> <span class="NLM_lpage">440</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1111%2Fj.1747-0285.2011.01111.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=21352502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslWqsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=431-440&author=A.+L%C3%B3pez+de+Victoriaauthor=R.+D.+Gorhamauthor=M.+L.+Bellows-Petersonauthor=J.+Lingauthor=D.+D.+Loauthor=C.+A.+Floudasauthor=D.+Morikis&title=A+new+generation+of+potent+complement+inhibitors+of+the+compstatin+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A new generation of potent complement inhibitors of the Compstatin family</span></div><div class="casAuthors">Lopez de Victoria, Aliana; Gorham, Ronald D., Jr.; Bellows-Peterson, Meghan L.; Ling, Jun; Lo, David D.; Floudas, Christodoulos A.; Morikis, Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">431-440</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Compstatin family peptides are potent inhibitors of the complement system and promising drug candidates against diseases involving under-regulated complement activation.  Compstatin is a 13-residue cyclized peptide that inhibits cleavage of complement protein C3, preventing downstream complement activation.  We present three new compstatin variants, characterized by tryptophan replacement at positions 1 and/or 13.  Peptide design was based on physicochem. reasoning and was inspired by earlier work, which identified tryptophan substitutions at positions 1 and 13 in peptides with predicted C3c binding abilities.  The new variants preserve distinct polar and nonpolar surfaces of compstatin, but have altered local interaction capabilities with C3.  All three peptides exhibited potent C3 binding by surface plasmon resonance and potent complement inhibition by ELISA.  We also present ELISA data and detailed surface plasmon resonance kinetic data of three peptides from previous computational design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZWQ1bOVHUhrVg90H21EOLACvtfcHk0lhvgIp3azEVGQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslWqsbg%253D&md5=f987241f1aa8c6c7a2cb564c6d7d80d0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01111.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01111.x%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B3pez%2Bde%2BVictoria%26aufirst%3DA.%26aulast%3DGorham%26aufirst%3DR.%2BD.%26aulast%3DBellows-Peterson%26aufirst%3DM.%2BL.%26aulast%3DLing%26aufirst%3DJ.%26aulast%3DLo%26aufirst%3DD.%2BD.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DA%2520new%2520generation%2520of%2520potent%2520complement%2520inhibitors%2520of%2520the%2520compstatin%2520family%26jtitle%3DChem.%2520Biol.%2520Drug.%2520Des.%26date%3D2011%26volume%3D77%26spage%3D431%26epage%3D440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Sun, J.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdeljabbar, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellows, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Link, A. J.</span><span> </span><span class="NLM_article-title">Reconstitution and engineering of apoptotic protein interactions on the bacterial cell surface</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">394</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">â</span> <span class="NLM_lpage">305</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2009&pages=297-305&author=J.-J.+Sunauthor=D.+M.+Abdeljabbarauthor=N.+L.+Clarkeauthor=M.+L.+Bellowsauthor=C.+A.+Floudasauthor=A.+J.+Link&title=Reconstitution+and+engineering+of+apoptotic+protein+interactions+on+the+bacterial+cell+surface"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.-J.%26aulast%3DAbdeljabbar%26aufirst%3DD.%2BM.%26aulast%3DClarke%26aufirst%3DN.%2BL.%26aulast%3DBellows%26aufirst%3DM.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DLink%26aufirst%3DA.%2BJ.%26atitle%3DReconstitution%2520and%2520engineering%2520of%2520apoptotic%2520protein%2520interactions%2520on%2520the%2520bacterial%2520cell%2520surface%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D394%26spage%3D297%26epage%3D305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Fung, H. K.</span>Computational De Novo Protein Design with Flexible Templates. Ph.D. Thesis. <span class="NLM_publisher-name">Princeton University</span><span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=H.+K.+Fung&title=Computational+De+Novo+Protein+Design+with+Flexible+Templates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFung%26aufirst%3DH.%2BK.%26jtitle%3DComputational%2520De%2520Novo%2520Protein%2520Design%2520with%2520Flexible%2520Templates%26pub%3DPrinceton%2520University%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Proctor, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiels, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S.</span><span> </span><span class="NLM_article-title">Transdermal pharmacology of small molecule cyclic C5a antagonists</span> <span class="citation_source-journal">Adv. Exp. Med. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">586</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">â</span> <span class="NLM_lpage">345</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2006&pages=329-345&author=L.+Proctorauthor=T.+Woodruffauthor=P.+Sharmaauthor=I.+Shielsauthor=S.+Taylor&title=Transdermal+pharmacology+of+small+molecule+cyclic+C5a+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DProctor%26aufirst%3DL.%26aulast%3DWoodruff%26aufirst%3DT.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DShiels%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26atitle%3DTransdermal%2520pharmacology%2520of%2520small%2520molecule%2520cyclic%2520C5a%2520antagonists%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2006%26volume%3D586%26spage%3D329%26epage%3D345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Burg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rheinheimer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bautsch, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klos, A.</span><span> </span><span class="NLM_article-title">Differential regulation of the C3a and C5a receptors (CD88) by IFN-gamma and PMA in U937 cells and related myeloblastic cell lines</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">5574</span><span class="NLM_x">â</span> <span class="NLM_lpage">5581</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=1996&pages=5574-5581&author=M.+Burgauthor=U.+Martinauthor=D.+Bockauthor=C.+Rheinheimerauthor=J.+Kohlauthor=W.+Bautschauthor=A.+Klos&title=Differential+regulation+of+the+C3a+and+C5a+receptors+%28CD88%29+by+IFN-gamma+and+PMA+in+U937+cells+and+related+myeloblastic+cell+lines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBurg%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DU.%26aulast%3DBock%26aufirst%3DD.%26aulast%3DRheinheimer%26aufirst%3DC.%26aulast%3DKohl%26aufirst%3DJ.%26aulast%3DBautsch%26aufirst%3DW.%26aulast%3DKlos%26aufirst%3DA.%26atitle%3DDifferential%2520regulation%2520of%2520the%2520C3a%2520and%2520C5a%2520receptors%2520%2528CD88%2529%2520by%2520IFN-gamma%2520and%2520PMA%2520in%2520U937%2520cells%2520and%2520related%2520myeloblastic%2520cell%2520lines%26jtitle%3DJ.%2520Immunol.%26date%3D1996%26volume%3D157%26spage%3D5574%26epage%3D5581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ember, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, T.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuoka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, R. D.</span><span> </span><span class="NLM_article-title">Identification of ligand effector binding sites in transmembrane regions of the human G protein-coupled C3a receptor</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2304</span><span class="NLM_x">â</span> <span class="NLM_lpage">2311</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1110%2Fps.8.11.2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=10595533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFSiu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1999&pages=2304-2311&issue=11&author=J.+Sunauthor=J.+A.+Emberauthor=T.-H.+Chaoauthor=T.+E.+Hugliauthor=Y.+Fukuokaauthor=R.+D.+Ye&title=Identification+of+ligand+effector+binding+sites+in+transmembrane+regions+of+the+human+G+protein-coupled+C3a+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of ligand effector binding sites in transmembrane regions of the human G protein-coupled C3a receptor</span></div><div class="casAuthors">Sun, Jianzhong; Ember, Julia A.; Chao, Ta-Hsiang; Fukuoka, Yoshihiro; Ye, Richard D.; Hugli, Tony E.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2304-2311</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The human C3a anaphylatoxin receptor (C3aR) is a G protein-coupled receptor (GPCR) composed of seven transmembrane Î±-helixes connected by hydrophilic loops.  Previous studies of chimeric C3aR/C5aR and loop deletions in C3aR demonstrated that the large extracellular loop2 plays an important role in non-effector ligand binding; however, the effector binding site for C3a has not been identified.  In this study, selected charged residues in the transmembrane regions of C3aR were replaced by Ala using site-directed mutagenesis, and mutant receptors were stably expressed in the RBL-2H3 cell line.  Ligand binding studies demonstrated that R161A (helix IV), R340A (helix V), and D417A (helix VII) showed no binding activity, although full expression of these receptors was established by flow cytometric anal.  C3a induced very weak intracellular calcium flux in cells expressing these three mutant receptors.  H81A (helix II) and K96A (helix III) showed decreased ligand binding activity.  The calcium flux induced by C3a in H81A and K96A cells was also consistently reduced.  These findings suggest that the charged transmembrane residues Arg161, Arg340, and Asp417 in C3aR are essential for ligand effector binding and/or signal coupling, and that residues His81 and Lys96 may contribute less directly to the overall free energy of ligand binding.  These transmembrane residues in C3aR identify specific mol. contacts for ligand interactions that account for C3a-induced receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMeO-VRrMmQbVg90H21EOLACvtfcHk0lhvgIp3azEVGQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFSiu7Y%253D&md5=6a77ea26411f1aec85965839bc289e89</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1110%2Fps.8.11.2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.8.11.2304%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DEmber%26aufirst%3DJ.%2BA.%26aulast%3DChao%26aufirst%3DT.-H.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26aulast%3DFukuoka%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DR.%2BD.%26atitle%3DIdentification%2520of%2520ligand%2520effector%2520binding%2520sites%2520in%2520transmembrane%2520regions%2520of%2520the%2520human%2520G%2520protein-coupled%2520C3a%2520receptor%26jtitle%3DProtein%2520Sci.%26date%3D1999%26volume%3D8%26issue%3D11%26spage%3D2304%26epage%3D2311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Free energy calculations for peptides via deterministic global optimization</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7491</span><span class="NLM_x">â</span> <span class="NLM_lpage">7512</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1999&pages=7491-7512&author=J.+L.+Klepeisauthor=C.+A.+Floudas&title=Free+energy+calculations+for+peptides+via+deterministic+global+optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DFree%2520energy%2520calculations%2520for%2520peptides%2520via%2520deterministic%2520global%2520optimization%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1999%26volume%3D110%26spage%3D7491%26epage%3D7512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Predicting peptide structures using NMR data and deterministic global optimization</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1354</span><span class="NLM_x">â</span> <span class="NLM_lpage">1370</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1002%2F%28SICI%291096-987X%28199910%2920%3A13%3C1354%3A%3AAID-JCC3%3E3.0.CO%3B2-N" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK1MXmt1Ggtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1999&pages=1354-1370&author=J.+L.+Klepeisauthor=C.+A.+Floudasauthor=D.+Morikisauthor=J.+Lambris&title=Predicting+peptide+structures+using+NMR+data+and+deterministic+global+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting peptide structures using NMR data and deterministic global optimization</span></div><div class="casAuthors">Klepeis, J. L.; Floudas, C. A.; Morikis, D.; Lambris, J. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1354-1370</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The ability to analyze large mol. structures by NMR techniques requires efficient methods for structure calcn.  Currently, there are several widely available methods for tackling these problems, which, in general, rely on the optimization of penalty-type target functions to satisfy the conformational restraints.  Typically, these methods combine simulated annealing protocols with mol. dynamics and local minimization, either in distance or torsional angle space.  In this work, both a novel formulation and algorithmic procedure for the soln. of the NMR structure prediction problem is outlined.  First, the unconstrained, penalty-type structure prediction problem is reformulated using nonlinear constraints, which can be individually enumerated for all, or subsets, of the distance restraints.  In this way, the violation can be controlled as a constraint, in contrast to the usual penalty-type restraints.  In addn., the customary simplified objective function is replaced by a full atom force field in the torsional angle space.  This guarantees a better description of at. interactions, which dictate the native structure of the mol. along with the distance restraints.  The second novel portion of this work involves the soln. method.  Rather than pursue the typical simulated annealing procedure, this work relies on a deterministic method, which theor. guarantees that the global soln. can be located.  This branch and bound technique, based on the Î±BB algorithm, has already been successfully applied to the identification of global min. energy structures of peptides modeled by full atom force fields.  Finally, the approach is applied to the Compstatin structure prediction, and it is found to possess some important merits when compared to existing techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwBhVpptx3-bVg90H21EOLACvtfcHk0ljhoy8al5SOrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmt1Ggtrw%253D&md5=7c78e2c9e93ac10a21eaade4def8f94e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199910%2920%3A13%3C1354%3A%3AAID-JCC3%3E3.0.CO%3B2-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199910%252920%253A13%253C1354%253A%253AAID-JCC3%253E3.0.CO%253B2-N%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DPredicting%2520peptide%2520structures%2520using%2520NMR%2520data%2520and%2520deterministic%2520global%2520optimization%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1999%26volume%3D20%26spage%3D1354%26epage%3D1370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Ab initio prediction of helical segments of polypeptides</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">246</span><span class="NLM_x">â</span> <span class="NLM_lpage">266</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=246-266&author=J.+L.+Klepeisauthor=C.+A.+Floudas&title=Ab+initio+prediction+of+helical+segments+of+polypeptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DAb%2520initio%2520prediction%2520of%2520helical%2520segments%2520of%2520polypeptides%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D246%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Prediction of beta-sheet topology and disulfide bridges in polypeptides</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">â</span> <span class="NLM_lpage">208</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=191-208&author=J.+L.+Klepeisauthor=C.+A.+Floudas&title=Prediction+of+beta-sheet+topology+and+disulfide+bridges+in+polypeptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DPrediction%2520of%2520beta-sheet%2520topology%2520and%2520disulfide%2520bridges%2520in%2520polypeptides%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D191%26epage%3D208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span> </span><span class="NLM_article-title">ASTRO-FOLD: A combinatorial and global optimization framework for ab initio prediction of three-dimensional structures of proteins from the amino acid sequence</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2119</span><span class="NLM_x">â</span> <span class="NLM_lpage">2146</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2003&pages=2119-2146&author=J.+Klepeisauthor=C.+Floudas&title=ASTRO-FOLD%3A+A+combinatorial+and+global+optimization+framework+for+ab+initio+prediction+of+three-dimensional+structures+of+proteins+from+the+amino+acid+sequence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DFloudas%26aufirst%3DC.%26atitle%3DASTRO-FOLD%253A%2520A%2520combinatorial%2520and%2520global%2520optimization%2520framework%2520for%2520ab%2520initio%2520prediction%2520of%2520three-dimensional%2520structures%2520of%2520proteins%2520from%2520the%2520amino%2520acid%2520sequence%26jtitle%3DBiophys.%2520J.%26date%3D2003%26volume%3D85%26spage%3D2119%26epage%3D2146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieja, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span> </span><span class="NLM_article-title">A new class of hybrid global optimization algorithms for peptide structure prediction: Integrated hybrids</span> <span class="citation_source-journal">Comput. Phys. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003a</span><span class="NLM_x">, </span> <span class="NLM_volume">151</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">â</span> <span class="NLM_lpage">140</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1016%2FS0010-4655%2802%2900735-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1ygtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2003a&pages=121-140&author=J.+Klepeisauthor=M.+Piejaauthor=C.+Floudas&title=A+new+class+of+hybrid+global+optimization+algorithms+for+peptide+structure+prediction%3A+Integrated+hybrids"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A new class of hybrid global optimization algorithms for peptide structure prediction: integrated hybrids</span></div><div class="casAuthors">Klepeis, J. L.; Pieja, M. J.; Floudas, C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-140</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A novel class of hybrid global optimization methods for application to the structure prediction in protein-folding problem is introduced.  These optimization methods take the form of a hybrid between a deterministic global optimization algorithm, the Î±BB, and a stochastically based method, conformational space annealing (CSA), and attempt to combine the beneficial features of these two algorithms.  The Î±BB method as previously extant exhibits consistency, as it guarantees convergence to the global min. for twice-continuously differentiable constrained nonlinear programming problems, but can benefit from improvements in the computational front.  Computational studies for met-enkephalin demonstrate the promise for the proposed hybrid global optimization method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCRa_oT6V-47Vg90H21EOLACvtfcHk0ljhoy8al5SOrA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1ygtg%253D%253D&md5=adc573297e231dde19be34d83835e796</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0010-4655%2802%2900735-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0010-4655%252802%252900735-X%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DPieja%26aufirst%3DM.%26aulast%3DFloudas%26aufirst%3DC.%26atitle%3DA%2520new%2520class%2520of%2520hybrid%2520global%2520optimization%2520algorithms%2520for%2520peptide%2520structure%2520prediction%253A%2520Integrated%2520hybrids%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2003a%26volume%3D151%26spage%3D121%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieja, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span> </span><span class="NLM_article-title">Hybrid global optimization algorithms for protein structure prediction: alternating hybrids</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003b</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">â</span> <span class="NLM_lpage">882</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2003b&pages=869-882&author=J.+Klepeisauthor=M.+Piejaauthor=C.+Floudas&title=Hybrid+global+optimization+algorithms+for+protein+structure+prediction%3A+alternating+hybrids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DPieja%26aufirst%3DM.%26aulast%3DFloudas%26aufirst%3DC.%26atitle%3DHybrid%2520global%2520optimization%2520algorithms%2520for%2520protein%2520structure%2520prediction%253A%2520alternating%2520hybrids%26jtitle%3DBiophys.%2520J.%26date%3D2003b%26volume%3D84%26spage%3D869%26epage%3D882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Huber, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholze, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paques, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deisenhofer, J.</span><span> </span><span class="NLM_article-title">Crystal structure analysis and molecular model of human C3a anaphylatoxin</span> <span class="citation_source-journal">Hoppe Seylers Z. Physiol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">361</span><span class="NLM_x">, </span> <span class="NLM_fpage">1389</span><span class="NLM_x">â</span> <span class="NLM_lpage">1399</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=1980&pages=1389-1399&author=R.+Huberauthor=H.+Scholzeauthor=E.+Paquesauthor=J.+Deisenhofer&title=Crystal+structure+analysis+and+molecular+model+of+human+C3a+anaphylatoxin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DScholze%26aufirst%3DH.%26aulast%3DPaques%26aufirst%3DE.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520analysis%2520and%2520molecular%2520model%2520of%2520human%2520C3a%2520anaphylatoxin%26jtitle%3DHoppe%2520Seylers%2520Z.%2520Physiol.%2520Chem.%26date%3D1980%26volume%3D361%26spage%3D1389%26epage%3D1399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Janssen, B. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huizinga, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raaijmakers, H. C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daha, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson-Ekdahl, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gros, P.</span><span> </span><span class="NLM_article-title">Structures of complement component C3 provide insights into the function and evolution of immunity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">â</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1038%2Fnature04005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=16177781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVajs7vI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=505-511&author=B.+J.+C.+Janssenauthor=E.+G.+Huizingaauthor=H.+C.+A.+Raaijmakersauthor=A.+Roosauthor=M.+R.+Dahaauthor=K.+Nilsson-Ekdahlauthor=B.+Nilssonauthor=P.+Gros&title=Structures+of+complement+component+C3+provide+insights+into+the+function+and+evolution+of+immunity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of complement component C3 provide insights into the function and evolution of immunity</span></div><div class="casAuthors">Janssen, Bert J. C.; Huizinga, Eric G.; Raaijmakers, Hans C. A.; Roos, Anja; Daha, Mohamed R.; Nilsson-Ekdahl, Kristina; Nilsson, Bo; Gros, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7058</span>),
    <span class="NLM_cas:pages">505-511</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mammalian complement system is a phylogenetically ancient cascade system that has a major role in innate and adaptive immunity.  Activation of component C3 (1641 residues) is central to the three complement pathways and results in inflammation and elimination of self and non-self targets.  Here the authors present crystal structures of native C3 and its final major proteolytic fragment C3c.  The structures reveal thirteen domains, nine of which were unpredicted, and suggest that the proteins of the Î±2-macroglobulin family evolved from a core of eight homologous domains.  A double mechanism prevents hydrolysis of the thioester group, essential for covalent attachment of activated C3 to target surfaces.  Marked conformational changes in the Î±-chain, including movement of a crit. interaction site through a ring formed by the domains of the Î²-chain, indicate an unprecedented, conformation-dependent mechanism of activation, regulation and biol. function of C3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJPI74p1TderVg90H21EOLACvtfcHk0lhK-xcdtYgbaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVajs7vI&md5=7b229e0ea9d3771f000855c687b1a7ab</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature04005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04005%26sid%3Dliteratum%253Aachs%26aulast%3DJanssen%26aufirst%3DB.%2BJ.%2BC.%26aulast%3DHuizinga%26aufirst%3DE.%2BG.%26aulast%3DRaaijmakers%26aufirst%3DH.%2BC.%2BA.%26aulast%3DRoos%26aufirst%3DA.%26aulast%3DDaha%26aufirst%3DM.%2BR.%26aulast%3DNilsson-Ekdahl%26aufirst%3DK.%26aulast%3DNilsson%26aufirst%3DB.%26aulast%3DGros%26aufirst%3DP.%26atitle%3DStructures%2520of%2520complement%2520component%2520C3%2520provide%2520insights%2520into%2520the%2520function%2520and%2520evolution%2520of%2520immunity%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D505%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Brooks, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruccoleri, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olafson, B.</span><span> </span><span class="NLM_article-title">CHARMMâa program for macromolecular energy, minimization, and dynamics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">â</span> <span class="NLM_lpage">217</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1983&pages=187-217&author=B.+Brooksauthor=R.+Bruccoleriauthor=B.+Olafson&title=CHARMM%E2%80%94a+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%26aulast%3DBruccoleri%26aufirst%3DR.%26aulast%3DOlafson%26aufirst%3DB.%26atitle%3DCHARMM%25E2%2580%2594a%2520program%2520for%2520macromolecular%2520energy%252C%2520minimization%252C%2520and%2520dynamics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1983%26volume%3D4%26spage%3D187%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Cain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monk, P.</span><span> </span><span class="NLM_article-title">The orphan receptor C5L2 has high affinity binding sites for complement fragments C5a and C5a of Arg74</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">7165</span><span class="NLM_x">â</span> <span class="NLM_lpage">7169</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=7165-7169&author=S.+Cainauthor=P.+Monk&title=The+orphan+receptor+C5L2+has+high+affinity+binding+sites+for+complement+fragments+C5a+and+C5a+of+Arg74"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCain%26aufirst%3DS.%26aulast%3DMonk%26aufirst%3DP.%26atitle%3DThe%2520orphan%2520receptor%2520C5L2%2520has%2520high%2520affinity%2520binding%2520sites%2520for%2520complement%2520fragments%2520C5a%2520and%2520C5a%2520of%2520Arg74%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D7165%26epage%3D7169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Monk, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, M.</span><span> </span><span class="NLM_article-title">C5a stimulus-secretion coupling in rat basophilic leukemia (RBL-2H3) cells transfected with the human C5a receptor is mediated by pertussis and cholera toxin-sensitive G-proteins</span> <span class="citation_source-journal">Biochem. Mol. Biol. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">13</span><span class="NLM_x">â</span> <span class="NLM_lpage">20</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1994&pages=13-20&issue=1&author=P.+Monkauthor=J.+Peaseauthor=M.+Barker&title=C5a+stimulus-secretion+coupling+in+rat+basophilic+leukemia+%28RBL-2H3%29+cells+transfected+with+the+human+C5a+receptor+is+mediated+by+pertussis+and+cholera+toxin-sensitive+G-proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMonk%26aufirst%3DP.%26aulast%3DPease%26aufirst%3DJ.%26aulast%3DBarker%26aufirst%3DM.%26atitle%3DC5a%2520stimulus-secretion%2520coupling%2520in%2520rat%2520basophilic%2520leukemia%2520%2528RBL-2H3%2529%2520cells%2520transfected%2520with%2520the%2520human%2520C5a%2520receptor%2520is%2520mediated%2520by%2520pertussis%2520and%2520cholera%2520toxin-sensitive%2520G-proteins%26jtitle%3DBiochem.%2520Mol.%2520Biol.%2520Int.%26date%3D1994%26volume%3D32%26issue%3D1%26spage%3D13%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Eswar, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti-Renom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madhusudhan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eramian, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieper, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Comparative protein structure modeling with MODELLER</span> <span class="citation_source-journal">Curr. Protoc. Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2.9.9</span><span class="NLM_x">â</span> <span class="NLM_lpage">2.9.30</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2.9.9-2.9.30&author=N.+Eswarauthor=M.+Marti-Renomauthor=B.+Webbauthor=M.+Madhusudhanauthor=D.+Eramianauthor=M.+Shenauthor=U.+Pieperauthor=A.+Sali&title=Comparative+protein+structure+modeling+with+MODELLER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26aulast%3DMarti-Renom%26aufirst%3DM.%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DMadhusudhan%26aufirst%3DM.%26aulast%3DEramian%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520with%2520MODELLER%26jtitle%3DCurr.%2520Protoc.%2520Protein%2520Sci.%26date%3D2007%26volume%3D50%26spage%3D2.9.9%26epage%3D2.9.30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">R Development Core Team</span><span class="NLM_x">. </span><span> </span><span class="NLM_article-title">R: A language and environment for statistical computing</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=R+Development+Core+Team&title=R%3A+A+language+and+environment+for+statistical+computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DR%253A%2520A%2520language%2520and%2520environment%2520for%2520statistical%2520computing%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Grant, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ElSawy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caves, L.</span><span> </span><span class="NLM_article-title">Bio3d: an R package for the comparative analysis of protein structures</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">â</span> <span class="NLM_lpage">2696</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=2695-2696&author=B.+Grantauthor=A.+Rodriguesauthor=K.+ElSawyauthor=J.+McCammonauthor=L.+Caves&title=Bio3d%3A+an+R+package+for+the+comparative+analysis+of+protein+structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DB.%26aulast%3DRodrigues%26aufirst%3DA.%26aulast%3DElSawy%26aufirst%3DK.%26aulast%3DMcCammon%26aufirst%3DJ.%26aulast%3DCaves%26aufirst%3DL.%26atitle%3DBio3d%253A%2520an%2520R%2520package%2520for%2520the%2520comparative%2520analysis%2520of%2520protein%2520structures%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D22%26spage%3D2695%26epage%3D2696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Dolinsky, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czodrowski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, N.</span><span> </span><span class="NLM_article-title">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">W522</span><span class="NLM_x">â</span> <span class="NLM_lpage">W525</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1093%2Fnar%2Fgkm276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=17488841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W522-W525&author=T.+Dolinskyauthor=P.+Czodrowskiauthor=H.+Liauthor=J.+Nielsenauthor=J.+Jensenauthor=G.+Klebeauthor=N.+Baker&title=PDB2PQR%3A+expanding+and+upgrading+automated+preparation+of+biomolecular+structures+for+molecular+simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span></div><div class="casAuthors">Dolinsky Todd J; Czodrowski Paul; Li Hui; Nielsen Jens E; Jensen Jan H; Klebe Gerhard; Baker Nathan A</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W522-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Real-world observable physical and chemical characteristics are increasingly being calculated from the 3D structures of biomolecules.  Methods for calculating pK(a) values, binding constants of ligands, and changes in protein stability are readily available, but often the limiting step in computational biology is the conversion of PDB structures into formats ready for use with biomolecular simulation software.  The continued sophistication and integration of biomolecular simulation methods for systems- and genome-wide studies requires a fast, robust, physically realistic and standardized protocol for preparing macromolecular structures for biophysical algorithms.  As described previously, the PDB2PQR web server addresses this need for electrostatic field calculations (Dolinsky et al., Nucleic Acids Research, 32, W665-W667, 2004).  Here we report the significantly expanded PDB2PQR that includes the following features: robust standalone command line support, improved pK(a) estimation via the PROPKA framework, ligand parameterization via PEOE_PB charge methodology, expanded set of force fields and easily incorporated user-defined parameters via XML input files, and improvement of atom addition and optimization code.  These features are available through a new web interface (http://pdb2pqr.sourceforge.net/), which offers users a wide range of options for PDB file conversion, modification and parameterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqIxhnOQkIHOxfW6udTcc2eaL3jE7dDd7OLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D&md5=18883d2f9a6f30dba143a4a2ca80b0d8</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm276%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DNielsen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DBaker%26aufirst%3DN.%26atitle%3DPDB2PQR%253A%2520expanding%2520and%2520upgrading%2520automated%2520preparation%2520of%2520biomolecular%2520structures%2520for%2520molecular%2520simulations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW522%26epage%3DW525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Sitkoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Accurate calculation of hydration free energies using macroscopic solvent models</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1978</span><span class="NLM_x">â</span> <span class="NLM_lpage">88</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100058a043" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1994&pages=1978-88&author=D.+Sitkoffauthor=K.+Sharpauthor=B.+Honig&title=Accurate+calculation+of+hydration+free+energies+using+macroscopic+solvent+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Calculation of Hydration Free Energies Using Macroscopic Solvent Models</span></div><div class="casAuthors">Sitkoff, Doree; Sharp, Kim A.; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1978-88</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">A method and parametrization scheme which allow fast and accurate calcns. of hydration free energies are described.  The solute is treated as a polarizable cavity of a shape defined by the mol. surface, contg. point charges at the location of at. nuclei.  Electrostatic contributions to solvation are derived from finite difference solns. of the Poisson equation (FDPB method).  Nonpolar (cavity/van der Waals) energies are added as a surface area dependent term, with a single surface tension coeff. (Î³) derived from hydrocarbon soly. in water.  At. charges and radii are obtained by modifying existing force-field- or quantum-mech.-derived values, by fitting to exptl. solvation energies of small org. mols.  A new, simple parameter set (parameters for solvation energy, PARSE) is developed specifically for the FDPB/Î³ method, by choosing at. charges and radii which reproduce the estd. contributions to solvation of simple functional groups.  The PARSE parameters reproduce hydration free energies for a test set of 67 mols. with an av. error of 0.4 kcal/mol.  For amino acid side chain and peptide backbone analogs the av. error is only 0.1 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCKnU2H8vArVg90H21EOLACvtfcHk0lgTgSRUgdssMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D&md5=78ad4721dced796d92eb27c15f98aba5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fj100058a043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100058a043%26sid%3Dliteratum%253Aachs%26aulast%3DSitkoff%26aufirst%3DD.%26aulast%3DSharp%26aufirst%3DK.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DAccurate%2520calculation%2520of%2520hydration%2520free%2520energies%2520using%2520macroscopic%2520solvent%2520models%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1994%26volume%3D98%26spage%3D1978%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Baker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sept, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holst, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J.</span><span> </span><span class="NLM_article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10037</span><span class="NLM_x">â</span> <span class="NLM_lpage">10041</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1073%2Fpnas.181342398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=11517324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10037-10041&author=N.+Bakerauthor=D.+Septauthor=S.+Josephauthor=M.+Holstauthor=J.+McCammon&title=Electrostatics+of+nanosystems%3A+application+to+microtubules+and+the+ribosome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatics of nanosystems: application to microtubules and the ribosome</span></div><div class="casAuthors">Baker, Nathan A.; Sept, David; Joseph, Simpson; Holst, Michael J.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10037-10041</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Evaluation of the electrostatic properties of biomols. has become a std. practice in mol. biophysics.  Foremost among the models used to elucidate the electrostatic potential is the Poisson-Boltzmann equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calcns. to relatively small biomol. systems.  Here we present the application of numerical methods to enable the trivially parallel soln. of the Poisson-Boltzmann equation for supramol. structures that are orders of magnitude larger in size.  As a demonstration of this methodol., electrostatic potentials have been calcd. for large microtubule and ribosome structures.  The results point to the likely role of electrostatics in a variety of activities of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_CAlhE0uX7Vg90H21EOLACvtfcHk0lgTgSRUgdssMA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D&md5=1b861999ef12c6972e82e8ada0f387cb</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.181342398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.181342398%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%26aulast%3DSept%26aufirst%3DD.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DHolst%26aufirst%3DM.%26aulast%3DMcCammon%26aufirst%3DJ.%26atitle%3DElectrostatics%2520of%2520nanosystems%253A%2520application%2520to%2520microtubules%2520and%2520the%2520ribosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D10037%26epage%3D10041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Gorham, R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sausman, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foronda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">An evaluation of PoissonâBoltzmann electrostatic free energy calculations through comparison with experimental mutagenesis</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">746</span><span class="NLM_x">â</span> <span class="NLM_lpage">754</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2011&pages=746-754&author=R.+Gorhamauthor=C.+Kieslichauthor=A.+Nicholsauthor=N.+Sausmanauthor=M.+Forondaauthor=D.+Morikis&title=An+evaluation+of+Poisson%E2%80%93Boltzmann+electrostatic+free+energy+calculations+through+comparison+with+experimental+mutagenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGorham%26aufirst%3DR.%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DNichols%26aufirst%3DA.%26aulast%3DSausman%26aufirst%3DN.%26aulast%3DForonda%26aufirst%3DM.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DAn%2520evaluation%2520of%2520Poisson%25E2%2580%2593Boltzmann%2520electrostatic%2520free%2520energy%2520calculations%2520through%2520comparison%2520with%2520experimental%2520mutagenesis%26jtitle%3DBiopolymers%26date%3D2011%26volume%3D94%26spage%3D746%26epage%3D754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunopulos, D.</span><span> </span><span class="NLM_article-title">Automated computational framework for the analysis of electrostatic similarities of proteins</span> <span class="citation_source-journal">Biotechnol. Prog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="NLM_x">â</span> <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1002%2Fbtpr.541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=21485028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Gis7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=316-325&author=C.+Kieslichauthor=D.+Morikisauthor=J.+Yangauthor=D.+Gunopulos&title=Automated+computational+framework+for+the+analysis+of+electrostatic+similarities+of+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Automated computational framework for the analysis of electrostatic similarities of proteins</span></div><div class="casAuthors">Kieslich, Chris A.; Morikis, Dimitrios; Yang, Jianfeng; Gunopulos, Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Progress</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">316-325</span>CODEN:
                <span class="NLM_cas:coden">BIPRET</span>;
        ISSN:<span class="NLM_cas:issn">1520-6033</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Charge plays an important role in protein-protein interactions.  In the case of excessively charged proteins, their electrostatic potentials contribute to the processes of recognition and binding with other proteins or ligands.  We present an automated computational framework for detg. the contribution of each charged amino acid to the electrostatic properties of proteins, at at. resoln. level.  This framework involves computational alanine scans, calcn. of Poisson-Boltzmann electrostatic potentials, calcn. of electrostatic similarity distances (ESDs), hierarchical clustering anal. of ESDs, calcn. of solvation free energies of assocn., and visualization of the spatial distributions of electrostatic potentials.  The framework is useful to classify families of mutants with similar electrostatic properties and to compare them with the parent proteins in the complex.  The alanine scan mutants introduce perturbations in the local electrostatic properties of the proteins and aim in delineating the contribution of each mutated amino acid in the spatial distribution of electrostatic potential, and in biol. function when electrostatics is a dominant contributing factor in protein-protein interactions.  The framework can be used to design new proteins with tailored electrostatic properties, such as immune system regulators, inhibitors, and vaccines, and in guiding exptl. studies.  We present an example for the interaction of the immune system protein C3d (the d-fragment of complement protein C3) with its receptor CR2, and we discuss our data in view of a binding site controversy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ316FjcDZubVg90H21EOLACvtfcHk0lizXhf-jEyTrQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Gis7o%253D&md5=92ce43efe6e241446e475132fd6eea4d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fbtpr.541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbtpr.541%26sid%3Dliteratum%253Aachs%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DGunopulos%26aufirst%3DD.%26atitle%3DAutomated%2520computational%2520framework%2520for%2520the%2520analysis%2520of%2520electrostatic%2520similarities%2520of%2520proteins%26jtitle%3DBiotechnol.%2520Prog.%26date%3D2011%26volume%3D27%26spage%3D316%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Gorham, R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">Electrostatic clustering and free energy calculations provide a foundation for protein design and optimization</span> <span class="citation_source-journal">Ann. Biomed. Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1252</span><span class="NLM_x">â</span> <span class="NLM_lpage">1263</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1007%2Fs10439-010-0226-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=21140293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A280%3ADC%252BC3Mzot1WmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1252-1263&author=R.+Gorhamauthor=C.+Kieslichauthor=D.+Morikis&title=Electrostatic+clustering+and+free+energy+calculations+provide+a+foundation+for+protein+design+and+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic clustering and free energy calculations provide a foundation for protein design and optimization</span></div><div class="casAuthors">Gorham Ronald D Jr; Kieslich Chris A; Morikis Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Annals of biomedical engineering</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1252-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Electrostatic interactions are ubiquitous in proteins and dictate stability and function.  In this review, we discuss several methods for the analysis of electrostatics in protein-protein interactions.  We discuss alanine-scanning mutagenesis, Poisson-Boltzmann electrostatics, free energy calculations, electrostatic similarity distances, and hierarchical clustering of electrostatic potentials.  Our recently developed computational framework, known as Analysis of Electrostatic Similarities Of Proteins (AESOP), incorporates these tools to efficiently elucidate the role of electrostatic potentials in protein interactions.  We present the application of AESOP to several proteins and protein complexes, for which charge is purported to facilitate protein association.  Specifically, we illustrate how recent work has shaped the formulation of electrostatic calculations, the correlation of electrostatic free energies and electrostatic potential clustering results with experimental binding and activity data, the pH dependence of protein stability and association, the design of mutant proteins with enhanced immunological activity, and how AESOP can expose deficiencies in structural models and experimental data.  This integrative approach can be utilized to develop mechanistic models and to guide experimental studies by predicting mutations with desired physicochemical properties and function.  Alteration of the electrostatic properties of proteins offers a basis for the design of proteins with optimized binding and activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTVo5QVfbcq4IJKJbLTgsvCfW6udTcc2ebOY5ez5S1AeLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mzot1WmtQ%253D%253D&md5=3794585d99f75112a3c6f2ebb25d9334</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10439-010-0226-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10439-010-0226-9%26sid%3Dliteratum%253Aachs%26aulast%3DGorham%26aufirst%3DR.%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DElectrostatic%2520clustering%2520and%2520free%2520energy%2520calculations%2520provide%2520a%2520foundation%2520for%2520protein%2520design%2520and%2520optimization%26jtitle%3DAnn.%2520Biomed.%2520Eng.%26date%3D2011%26volume%3D39%26spage%3D1252%26epage%3D1263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorham, R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">Is the rigid-body assumption reasonable? Insights into the effects of dynamics on the electrostatic analysis of barnase-barstar</span> <span class="citation_source-journal">J. Non-Cryst. Solids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">â</span> <span class="NLM_lpage">716</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2011&pages=707-716&author=C.+Kieslichauthor=R.+Gorhamauthor=D.+Morikis&title=Is+the+rigid-body+assumption+reasonable%3F+Insights+into+the+effects+of+dynamics+on+the+electrostatic+analysis+of+barnase-barstar"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DGorham%26aufirst%3DR.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DIs%2520the%2520rigid-body%2520assumption%2520reasonable%253F%2520Insights%2520into%2520the%2520effects%2520of%2520dynamics%2520on%2520the%2520electrostatic%2520analysis%2520of%2520barnase-barstar%26jtitle%3DJ.%2520Non-Cryst.%2520Solids%26date%3D2011%26volume%3D357%26spage%3D707%26epage%3D716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Pettersen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T.</span><span> </span><span class="NLM_article-title">UCSF chimeraâA visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">â</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+Pettersenauthor=T.+Goddardauthor=C.+Huangauthor=G.+Couchauthor=D.+Greenblattauthor=E.+Mengauthor=T.+Ferrin&title=UCSF+chimera%E2%80%94A+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%26aulast%3DGoddard%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DCouch%26aufirst%3DG.%26aulast%3DGreenblatt%26aufirst%3DD.%26aulast%3DMeng%26aufirst%3DE.%26aulast%3DFerrin%26aufirst%3DT.%26atitle%3DUCSF%2520chimera%25E2%2580%2594A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 27 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ronald D.  Gorham, Jr.</span>, <span class="hlFld-ContribAuthor ">David L.  Forest</span>, <span class="hlFld-ContribAuthor ">George A.  Khoury</span>, <span class="hlFld-ContribAuthor ">James  Smadbeck</span>, <span class="hlFld-ContribAuthor ">Consuelo N.  Beecher</span>, <span class="hlFld-ContribAuthor ">Evangeline D.  Healy</span>, <span class="hlFld-ContribAuthor ">Phanourios  Tamamis</span>, <span class="hlFld-ContribAuthor ">Georgios  Archontis</span>, <span class="hlFld-ContribAuthor ">Cynthia K.  Larive</span>, <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>, <span class="hlFld-ContribAuthor ">Monte J.  Radeke</span>, <span class="hlFld-ContribAuthor ">Lincoln V.  Johnson</span>, and <span class="hlFld-ContribAuthor ">Dimitrios  Morikis</span>  . </span><span class="cited-content_cbyCitation_article-title">New Compstatin Peptides Containing N-Terminal Extensions and Non-Natural Amino Acids Exhibit Potent Complement Inhibition and Improved Solubility Characteristics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (2)
                                     , 814-826. <a href="https://doi.org/10.1021/jm501345y" title="DOI URL">https://doi.org/10.1021/jm501345y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm501345y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm501345y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNew%252BCompstatin%252BPeptides%252BContaining%252BN-Terminal%252BExtensions%252Band%252BNon-Natural%252BAmino%252BAcids%252BExhibit%252BPotent%252BComplement%252BInhibition%252Band%252BImproved%252BSolubility%252BCharacteristics%26aulast%3DGorham%26aufirst%3DRonald%2BD.%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D01092014%26date%3D29122014%26date%3D22012015%26date%3D10122014%26volume%3D58%26issue%3D2%26spage%3D814%26epage%3D826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George A.  Khoury</span>, <span class="hlFld-ContribAuthor ">James  Smadbeck</span>, <span class="hlFld-ContribAuthor ">Phanourios  Tamamis</span>, <span class="hlFld-ContribAuthor ">Andrew C.  Vandris</span>, <span class="hlFld-ContribAuthor ">Chris A.  Kieslich</span>, and <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>  . </span><span class="cited-content_cbyCitation_article-title">Forcefield_NCAA: Ab Initio Charge Parameters to Aid in the Discovery and Design of Therapeutic Proteins and Peptides with Unnatural Amino Acids and Their Application to Complement Inhibitors of the Compstatin Family. </span><span class="cited-content_cbyCitation_journal-name">ACS Synthetic Biology</span><span> <strong>2014,</strong> <em>3 </em>
                                    (12)
                                     , 855-869. <a href="https://doi.org/10.1021/sb400168u" title="DOI URL">https://doi.org/10.1021/sb400168u</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/sb400168u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fsb400168u%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Synthetic%2520Biology%26atitle%3DForcefield_NCAA%25253A%252BAb%252BInitio%252BCharge%252BParameters%252Bto%252BAid%252Bin%252Bthe%252BDiscovery%252Band%252BDesign%252Bof%252BTherapeutic%252BProteins%252Band%252BPeptides%252Bwith%252BUnnatural%252BAmino%252BAcids%252Band%252BTheir%252BApplication%252Bto%252BComplement%252BInhibitors%252Bof%252Bthe%252BCompstatin%252BFamily%26aulast%3DKhoury%26aufirst%3DGeorge%2BA.%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D21102013%26date%3D14012014%26date%3D19122014%26date%3D06012014%26volume%3D3%26issue%3D12%26spage%3D855%26epage%3D869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert C.  Reid</span>, <span class="hlFld-ContribAuthor ">Mei-Kwan  Yau</span>, <span class="hlFld-ContribAuthor ">Ranee  Singh</span>, <span class="hlFld-ContribAuthor ">Johan K.  Hamidon</span>, <span class="hlFld-ContribAuthor ">Junxian  Lim</span>, <span class="hlFld-ContribAuthor ">Martin J.  Stoermer</span>, and <span class="hlFld-ContribAuthor ">David P.  Fairlie</span>  . </span><span class="cited-content_cbyCitation_article-title">Potent Heterocyclic Ligands for Human Complement C3a Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (20)
                                     , 8459-8470. <a href="https://doi.org/10.1021/jm500956p" title="DOI URL">https://doi.org/10.1021/jm500956p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500956p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500956p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPotent%252BHeterocyclic%252BLigands%252Bfor%252BHuman%252BComplement%252BC3a%252BReceptor%26aulast%3DReid%26aufirst%3DRobert%2BC.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D24062014%26date%3D08102014%26date%3D23102014%26date%3D26092014%26volume%3D57%26issue%3D20%26spage%3D8459%26epage%3D8470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Phanourios  Tamamis</span> and <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>  . </span><span class="cited-content_cbyCitation_article-title">Elucidating a Key Component of Cancer Metastasis: CXCL12 (SDF-1Î±) Binding to CXCR4. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (4)
                                     , 1174-1188. <a href="https://doi.org/10.1021/ci500069y" title="DOI URL">https://doi.org/10.1021/ci500069y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500069y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500069y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DElucidating%252Ba%252BKey%252BComponent%252Bof%252BCancer%252BMetastasis%25253A%252BCXCL12%252B%252528SDF-1%2525CE%2525B1%252529%252BBinding%252Bto%252BCXCR4%26aulast%3DTamamis%26aufirst%3DPhanourios%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D03022014%26date%3D08042014%26date%3D28042014%26date%3D24032014%26volume%3D54%26issue%3D4%26spage%3D1174%26epage%3D1188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George A.  Khoury</span>, <span class="hlFld-ContribAuthor ">Jeff P.  Thompson</span>, <span class="hlFld-ContribAuthor ">James  Smadbeck</span>, <span class="hlFld-ContribAuthor ">Chris A.  Kieslich</span>, and <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>  . </span><span class="cited-content_cbyCitation_article-title">Forcefield_PTM: Ab Initio Charge and AMBER Forcefield Parameters for Frequently Occurring Post-Translational Modifications. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2013,</strong> <em>9 </em>
                                    (12)
                                     , 5653-5674. <a href="https://doi.org/10.1021/ct400556v" title="DOI URL">https://doi.org/10.1021/ct400556v</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ct400556v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fct400556v%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DForcefield_PTM%25253A%252BAb%252BInitio%252BCharge%252Band%252BAMBER%252BForcefield%252BParameters%252Bfor%252BFrequently%252BOccurring%252BPost-Translational%252BModifications%26aulast%3DKhoury%26aufirst%3DGeorge%2BA.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D28062013%26date%3D02122013%26date%3D10122013%26date%3D22112013%26volume%3D9%26issue%3D12%26spage%3D5653%26epage%3D5674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pooja  Shivshankar</span>, <span class="hlFld-ContribAuthor ">YiâDong  Li</span>, <span class="hlFld-ContribAuthor ">Stacey L.  MuellerâOrtiz</span>, <span class="hlFld-ContribAuthor ">Rick A.  Wetsel</span>. </span><span class="cited-content_cbyCitation_article-title">In response to complement anaphylatoxin peptides C3a and C5a, human vascular endothelial cells migrate and mediate the activation of Bâcells and polarization of Tâcells. </span><span class="cited-content_cbyCitation_journal-name">The FASEB Journal</span><span> <strong>2020,</strong> <em>34 </em>
                                    (6)
                                     , 7540-7560. <a href="https://doi.org/10.1096/fj.201902397R" title="DOI URL">https://doi.org/10.1096/fj.201902397R</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1096/fj.201902397R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1096%2Ffj.201902397R%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FASEB%2520Journal%26atitle%3DIn%252Bresponse%252Bto%252Bcomplement%252Banaphylatoxin%252Bpeptides%252BC3a%252Band%252BC5a%25252C%252Bhuman%252Bvascular%252Bendothelial%252Bcells%252Bmigrate%252Band%252Bmediate%252Bthe%252Bactivation%252Bof%252BB%2525E2%252580%252590cells%252Band%252Bpolarization%252Bof%252BT%2525E2%252580%252590cells%26aulast%3DShivshankar%26aufirst%3DPooja%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D6%26spage%3D7540%26epage%3D7560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David J.  Diller</span>, <span class="hlFld-ContribAuthor ">Jon  Swanson</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Bayden</span>, <span class="hlFld-ContribAuthor ">Chris J.  Brown</span>, <span class="hlFld-ContribAuthor ">Dawn  Thean</span>, <span class="hlFld-ContribAuthor ">David P.  Lane</span>, <span class="hlFld-ContribAuthor ">Anthony W.  Partridge</span>, <span class="hlFld-ContribAuthor ">Tomi K.  Sawyer</span>, <span class="hlFld-ContribAuthor ">Joseph  Audie</span>. </span><span class="cited-content_cbyCitation_article-title">Rigorous Computational and Experimental Investigations on MDM2/MDMX-Targeted Linear and Macrocyclic Peptides. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (24)
                                     , 4586. <a href="https://doi.org/10.3390/molecules24244586" title="DOI URL">https://doi.org/10.3390/molecules24244586</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24244586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24244586%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRigorous%252BComputational%252Band%252BExperimental%252BInvestigations%252Bon%252BMDM2%25252FMDMX-Targeted%252BLinear%252Band%252BMacrocyclic%252BPeptides%26aulast%3DDiller%26aufirst%3DDavid%2BJ.%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D24%26spage%3D4586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stacey L.  Mueller-Ortiz</span>, <span class="hlFld-ContribAuthor ">Daniel G.  Calame</span>, <span class="hlFld-ContribAuthor ">Nancy  Shenoi</span>, <span class="hlFld-ContribAuthor ">Yi-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Rick A.  Wetsel</span>. </span><span class="cited-content_cbyCitation_article-title">The Complement Anaphylatoxins C5a and C3a Suppress IFN-Î² Production in Response to
              Listeria monocytogenes
              by Inhibition of the Cyclic DinucleotideâActivated Cytosolic Surveillance Pathway. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Immunology</span><span> <strong>2017,</strong> <em>198 </em>
                                    (8)
                                     , 3237-3244. <a href="https://doi.org/10.4049/jimmunol.1601420" title="DOI URL">https://doi.org/10.4049/jimmunol.1601420</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4049/jimmunol.1601420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4049%2Fjimmunol.1601420%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Immunology%26atitle%3DThe%252BComplement%252BAnaphylatoxins%252BC5a%252Band%252BC3a%252BSuppress%252BIFN-%2525CE%2525B2%252BProduction%252Bin%252BResponse%252Bto%252BListeria%252Bmonocytogenes%252Bby%252BInhibition%252Bof%252Bthe%252BCyclic%252BDinucleotide%2525E2%252580%252593Activated%252BCytosolic%252BSurveillance%252BPathway%26aulast%3DMueller-Ortiz%26aufirst%3DStacey%2BL.%26date%3D2017%26date%3D2017%26volume%3D198%26issue%3D8%26spage%3D3237%26epage%3D3244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel E  Croker</span>, <span class="hlFld-ContribAuthor ">Peter N  Monk</span>, <span class="hlFld-ContribAuthor ">Reena  Halai</span>, <span class="hlFld-ContribAuthor ">Geraldine  Kaeslin</span>, <span class="hlFld-ContribAuthor ">Zoe  Schofield</span>, <span class="hlFld-ContribAuthor ">Mike CL  Wu</span>, <span class="hlFld-ContribAuthor ">Richard J  Clark</span>, <span class="hlFld-ContribAuthor ">Mark AT  Blaskovich</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Morikis</span>, <span class="hlFld-ContribAuthor ">Christodoulos A  Floudas</span>, <span class="hlFld-ContribAuthor ">Matthew A  Cooper</span>, <span class="hlFld-ContribAuthor ">Trent M  Woodruff</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of functionally selective C5aR2 ligands: novel modulators of C5a signalling. </span><span class="cited-content_cbyCitation_journal-name">Immunology & Cell Biology</span><span> <strong>2016,</strong> <em>94 </em>
                                    (8)
                                     , 787-795. <a href="https://doi.org/10.1038/icb.2016.43" title="DOI URL">https://doi.org/10.1038/icb.2016.43</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/icb.2016.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Ficb.2016.43%26sid%3Dliteratum%253Aachs%26jtitle%3DImmunology%2520%2526%2520Cell%2520Biology%26atitle%3DDiscovery%252Bof%252Bfunctionally%252Bselective%252BC5aR2%252Bligands%25253A%252Bnovel%252Bmodulators%252Bof%252BC5a%252Bsignalling%26aulast%3DCroker%26aufirst%3DDaniel%2BE%26date%3D2016%26date%3D2016%26volume%3D94%26issue%3D8%26spage%3D787%26epage%3D795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Po-Ssu  Huang</span>, <span class="hlFld-ContribAuthor ">Kaspar  Feldmeier</span>, <span class="hlFld-ContribAuthor ">Fabio  Parmeggiani</span>, <span class="hlFld-ContribAuthor ">D Alejandro  Fernandez Velasco</span>, <span class="hlFld-ContribAuthor ">Birte  HÃ¶cker</span>, <span class="hlFld-ContribAuthor ">David  Baker</span>. </span><span class="cited-content_cbyCitation_article-title">De novo design of a four-fold symmetric TIM-barrel protein with atomic-level accuracy. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2016,</strong> <em>12 </em>
                                    (1)
                                     , 29-34. <a href="https://doi.org/10.1038/nchembio.1966" title="DOI URL">https://doi.org/10.1038/nchembio.1966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nchembio.1966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnchembio.1966%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DDe%252Bnovo%252Bdesign%252Bof%252Ba%252Bfour-fold%252Bsymmetric%252BTIM-barrel%252Bprotein%252Bwith%252Batomic-level%252Baccuracy%26aulast%3DHuang%26aufirst%3DPo-Ssu%26date%3D2016%26date%3D2015%26volume%3D12%26issue%3D1%26spage%3D29%26epage%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David J  Diller</span>, <span class="hlFld-ContribAuthor ">Mark  Jarosinski</span>, <span class="hlFld-ContribAuthor ">Tomi K  Sawyer</span>, <span class="hlFld-ContribAuthor ">Joseph  Audie</span>. </span><span class="cited-content_cbyCitation_article-title">Peptide drug discovery: innovative technologies and transformational medicines. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 8-27. <a href="https://doi.org/10.4155/fseb2013.14.105" title="DOI URL">https://doi.org/10.4155/fseb2013.14.105</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fseb2013.14.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffseb2013.14.105%26sid%3Dliteratum%253Aachs%26atitle%3DPeptide%252Bdrug%252Bdiscovery%25253A%252Binnovative%252Btechnologies%252Band%252Btransformational%252Bmedicines%26aulast%3DDiller%26aufirst%3DDavid%2BJ%26date%3D2015%26date%3D2015%26spage%3D8%26epage%3D27%26pub%3DFuture%2520Science%2520Ltd%26atitle%3DAdvances%252Bin%252Bthe%252BDiscovery%252Band%252BDevelopment%252Bof%252BPeptide%252BTherapeutics%26aulast%3DKruger%26aufirst%3DGert%26date%3D2015%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David J  Diller</span>, <span class="hlFld-ContribAuthor ">Jon  Swanson</span>, <span class="hlFld-ContribAuthor ">Alexander S  Bayden</span>, <span class="hlFld-ContribAuthor ">Mark  Jarosinski</span>, <span class="hlFld-ContribAuthor ">Joseph  Audie</span>. </span><span class="cited-content_cbyCitation_article-title">Rational, computer-enabled peptide drug design: principles, methods, applications and future directions. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2015,</strong> <em>7 </em>
                                    (16)
                                     , 2173-2193. <a href="https://doi.org/10.4155/fmc.15.142" title="DOI URL">https://doi.org/10.4155/fmc.15.142</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.142%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRational%25252C%252Bcomputer-enabled%252Bpeptide%252Bdrug%252Bdesign%25253A%252Bprinciples%25252C%252Bmethods%25252C%252Bapplications%252Band%252Bfuture%252Bdirections%26aulast%3DDiller%26aufirst%3DDavid%2BJ%26date%3D2015%26volume%3D7%26issue%3D16%26spage%3D2173%26epage%3D2193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Shultis</span>, <span class="hlFld-ContribAuthor ">Gregory  Dodge</span>, <span class="hlFld-ContribAuthor ">Yang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal structure of designed PX domain from cytokine-independent survival kinase and implications on evolution-based protein engineering. </span><span class="cited-content_cbyCitation_journal-name">Journal of Structural Biology</span><span> <strong>2015,</strong> <em>191 </em>
                                    (2)
                                     , 197-206. <a href="https://doi.org/10.1016/j.jsb.2015.06.009" title="DOI URL">https://doi.org/10.1016/j.jsb.2015.06.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jsb.2015.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jsb.2015.06.009%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Structural%2520Biology%26atitle%3DCrystal%252Bstructure%252Bof%252Bdesigned%252BPX%252Bdomain%252Bfrom%252Bcytokine-independent%252Bsurvival%252Bkinase%252Band%252Bimplications%252Bon%252Bevolution-based%252Bprotein%252Bengineering%26aulast%3DShultis%26aufirst%3DDavid%26date%3D2015%26volume%3D191%26issue%3D2%26spage%3D197%26epage%3D206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Phanourios  Tamamis</span>, <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>. </span><span class="cited-content_cbyCitation_article-title">Elucidating a Key Anti-HIV-1 and Cancer-Associated Axis: The Structure of CCL5 (Rantes) in Complex with CCR5. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2015,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep05447" title="DOI URL">https://doi.org/10.1038/srep05447</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep05447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep05447%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DElucidating%252Ba%252BKey%252BAnti-HIV-1%252Band%252BCancer-Associated%252BAxis%25253A%252BThe%252BStructure%252Bof%252BCCL5%252B%252528Rantes%252529%252Bin%252BComplex%252Bwith%252BCCR5%26aulast%3DTamamis%26aufirst%3DPhanourios%26date%3D2015%26date%3D2014%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liam G.  Coulthard</span>, <span class="hlFld-ContribAuthor ">Trent M.  Woodruff</span>. </span><span class="cited-content_cbyCitation_article-title">Is the Complement Activation Product C3a a Proinflammatory Molecule? Re-evaluating the Evidence and the Myth. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Immunology</span><span> <strong>2015,</strong> <em>194 </em>
                                    (8)
                                     , 3542-3548. <a href="https://doi.org/10.4049/jimmunol.1403068" title="DOI URL">https://doi.org/10.4049/jimmunol.1403068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4049/jimmunol.1403068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4049%2Fjimmunol.1403068%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Immunology%26atitle%3DIs%252Bthe%252BComplement%252BActivation%252BProduct%252BC3a%252Ba%252BProinflammatory%252BMolecule%25253F%252BRe-evaluating%252Bthe%252BEvidence%252Band%252Bthe%252BMyth%26aulast%3DCoulthard%26aufirst%3DLiam%2BG.%26date%3D2015%26date%3D2015%26volume%3D194%26issue%3D8%26spage%3D3542%26epage%3D3548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Reena  Halai</span>, <span class="hlFld-ContribAuthor ">Meghan L  Bellows-Peterson</span>, <span class="hlFld-ContribAuthor ">Will  Branchett</span>, <span class="hlFld-ContribAuthor ">James  Smadbeck</span>, <span class="hlFld-ContribAuthor ">Chris A  Kieslich</span>, <span class="hlFld-ContribAuthor ">Daniel E  Croker</span>, <span class="hlFld-ContribAuthor ">Matthew A  Cooper</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Morikis</span>, <span class="hlFld-ContribAuthor ">Trent M  Woodruff</span>, <span class="hlFld-ContribAuthor ">Christodoulos A  Floudas</span>, <span class="hlFld-ContribAuthor ">Peter N  Monk</span>. </span><span class="cited-content_cbyCitation_article-title">Derivation of ligands for the complement C3a receptor from the C-terminus of C5a. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2014,</strong> <em>745 </em>, 176-181. <a href="https://doi.org/10.1016/j.ejphar.2014.10.041" title="DOI URL">https://doi.org/10.1016/j.ejphar.2014.10.041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2014.10.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2014.10.041%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DDerivation%252Bof%252Bligands%252Bfor%252Bthe%252Bcomplement%252BC3a%252Breceptor%252Bfrom%252Bthe%252BC-terminus%252Bof%252BC5a%26aulast%3DHalai%26aufirst%3DReena%26date%3D2014%26volume%3D745%26spage%3D176%26epage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James  Smadbeck</span>, <span class="hlFld-ContribAuthor ">Kiat Hwa  Chan</span>, <span class="hlFld-ContribAuthor ">George A.  Khoury</span>, <span class="hlFld-ContribAuthor ">Bo  Xue</span>, <span class="hlFld-ContribAuthor ">Robert C.  Robinson</span>, <span class="hlFld-ContribAuthor ">Charlotte A. E.  Hauser</span>, <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>, . </span><span class="cited-content_cbyCitation_article-title">De Novo Design and Experimental Characterization of Ultrashort Self-Associating Peptides. </span><span class="cited-content_cbyCitation_journal-name">PLoS Computational Biology</span><span> <strong>2014,</strong> <em>10 </em>
                                    (7)
                                     , e1003718. <a href="https://doi.org/10.1371/journal.pcbi.1003718" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1003718</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1003718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1003718%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520Computational%2520Biology%26atitle%3DDe%252BNovo%252BDesign%252Band%252BExperimental%252BCharacterization%252Bof%252BUltrashort%252BSelf-Associating%252BPeptides%26aulast%3DSmadbeck%26aufirst%3DJames%26date%3D2014%26date%3D2014%26volume%3D10%26issue%3D7%26spage%3De1003718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Phanourios  Tamamis</span>, <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>, . </span><span class="cited-content_cbyCitation_article-title">Molecular Recognition of CCR5 by an HIV-1 gp120 V3 Loop. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (4)
                                     , e95767. <a href="https://doi.org/10.1371/journal.pone.0095767" title="DOI URL">https://doi.org/10.1371/journal.pone.0095767</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0095767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0095767%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DMolecular%252BRecognition%252Bof%252BCCR5%252Bby%252Ban%252BHIV-1%252Bgp120%252BV3%252BLoop%26aulast%3DTamamis%26aufirst%3DPhanourios%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D4%26spage%3De95767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James  Smadbeck</span>, <span class="hlFld-ContribAuthor ">Meghan B.  Peterson</span>, <span class="hlFld-ContribAuthor ">Barry M.  Zee</span>, <span class="hlFld-ContribAuthor ">Shivani  Garapaty</span>, <span class="hlFld-ContribAuthor ">Aashna  Mago</span>, <span class="hlFld-ContribAuthor ">Christina  Lee</span>, <span class="hlFld-ContribAuthor ">Athanassios  Giannis</span>, <span class="hlFld-ContribAuthor ">Patrick  Trojer</span>, <span class="hlFld-ContribAuthor ">Benjamin A.  Garcia</span>, <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>, . </span><span class="cited-content_cbyCitation_article-title">De Novo Peptide Design and Experimental Validation of Histone Methyltransferase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (2)
                                     , e90095. <a href="https://doi.org/10.1371/journal.pone.0090095" title="DOI URL">https://doi.org/10.1371/journal.pone.0090095</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0090095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0090095%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DDe%252BNovo%252BPeptide%252BDesign%252Band%252BExperimental%252BValidation%252Bof%252BHistone%252BMethyltransferase%252BInhibitors%26aulast%3DSmadbeck%26aufirst%3DJames%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D2%26spage%3De90095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George A.  Khoury</span>, <span class="hlFld-ContribAuthor ">James  Smadbeck</span>, <span class="hlFld-ContribAuthor ">Chris A.  Kieslich</span>, <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>. </span><span class="cited-content_cbyCitation_article-title">Protein folding and de novo protein design for biotechnological applications. </span><span class="cited-content_cbyCitation_journal-name">Trends in Biotechnology</span><span> <strong>2014,</strong> <em>32 </em>
                                    (2)
                                     , 99-109. <a href="https://doi.org/10.1016/j.tibtech.2013.10.008" title="DOI URL">https://doi.org/10.1016/j.tibtech.2013.10.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tibtech.2013.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tibtech.2013.10.008%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Biotechnology%26atitle%3DProtein%252Bfolding%252Band%252Bde%252Bnovo%252Bprotein%252Bdesign%252Bfor%252Bbiotechnological%252Bapplications%26aulast%3DKhoury%26aufirst%3DGeorge%2BA.%26date%3D2014%26volume%3D32%26issue%3D2%26spage%3D99%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ronald D.  Gorham</span>, <span class="hlFld-ContribAuthor ">David L.  Forest</span>, <span class="hlFld-ContribAuthor ">Phanourios  Tamamis</span>, <span class="hlFld-ContribAuthor ">Aliana  LÃ³pez de Victoria</span>, <span class="hlFld-ContribAuthor ">MÃ¡rta  Kraszni</span>, <span class="hlFld-ContribAuthor ">Chris A.  Kieslich</span>, <span class="hlFld-ContribAuthor ">Christopher D.  Banna</span>, <span class="hlFld-ContribAuthor ">Meghan L.  Bellows-Peterson</span>, <span class="hlFld-ContribAuthor ">Cynthia K.  Larive</span>, <span class="hlFld-ContribAuthor ">Christodoulos A.  Floudas</span>, <span class="hlFld-ContribAuthor ">Georgios  Archontis</span>, <span class="hlFld-ContribAuthor ">Lincoln V.  Johnson</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Morikis</span>. </span><span class="cited-content_cbyCitation_article-title">Novel compstatin family peptides inhibit complement activation by drusen-like deposits in human retinal pigmented epithelial cell cultures. </span><span class="cited-content_cbyCitation_journal-name">Experimental Eye Research</span><span> <strong>2013,</strong> <em>116 </em>, 96-108. <a href="https://doi.org/10.1016/j.exer.2013.07.023" title="DOI URL">https://doi.org/10.1016/j.exer.2013.07.023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.exer.2013.07.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.exer.2013.07.023%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520Eye%2520Research%26atitle%3DNovel%252Bcompstatin%252Bfamily%252Bpeptides%252Binhibit%252Bcomplement%252Bactivation%252Bby%252Bdrusen-like%252Bdeposits%252Bin%252Bhuman%252Bretinal%252Bpigmented%252Bepithelial%252Bcell%252Bcultures%26aulast%3DGorham%26aufirst%3DRonald%2BD.%26date%3D2013%26volume%3D116%26spage%3D96%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pralay  Mitra</span>, <span class="hlFld-ContribAuthor ">David  Shultis</span>, <span class="hlFld-ContribAuthor ">Jeffrey R.  Brender</span>, <span class="hlFld-ContribAuthor ">Jeff  Czajka</span>, <span class="hlFld-ContribAuthor ">David  Marsh</span>, <span class="hlFld-ContribAuthor ">Felicia  Gray</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>, <span class="hlFld-ContribAuthor ">Yang  Zhang</span>, . </span><span class="cited-content_cbyCitation_article-title">An Evolution-Based Approach to De Novo Protein Design and Case Study on Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">PLoS Computational Biology</span><span> <strong>2013,</strong> <em>9 </em>
                                    (10)
                                     , e1003298. <a href="https://doi.org/10.1371/journal.pcbi.1003298" title="DOI URL">https://doi.org/10.1371/journal.pcbi.1003298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pcbi.1003298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pcbi.1003298%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520Computational%2520Biology%26atitle%3DAn%252BEvolution-Based%252BApproach%252Bto%252BDe%252BNovo%252BProtein%252BDesign%252Band%252BCase%252BStudy%252Bon%252BMycobacterium%252Btuberculosis%26aulast%3DMitra%26aufirst%3DPralay%26date%3D2013%26date%3D2013%26volume%3D9%26issue%3D10%26spage%3De1003298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yee Siew  Choong</span>, <span class="hlFld-ContribAuthor ">Gee Jun  Tye</span>, <span class="hlFld-ContribAuthor ">Theam Soon  Lim</span>. </span><span class="cited-content_cbyCitation_article-title">Minireview: Applied Structural Bioinformatics in Proteomics. </span><span class="cited-content_cbyCitation_journal-name">The Protein Journal</span><span> <strong>2013,</strong> <em>32 </em>
                                    (7)
                                     , 505-511. <a href="https://doi.org/10.1007/s10930-013-9514-1" title="DOI URL">https://doi.org/10.1007/s10930-013-9514-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10930-013-9514-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10930-013-9514-1%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Protein%2520Journal%26atitle%3DMinireview%25253A%252BApplied%252BStructural%252BBioinformatics%252Bin%252BProteomics%26aulast%3DChoong%26aufirst%3DYee%2BSiew%26date%3D2013%26date%3D2013%26volume%3D32%26issue%3D7%26spage%3D505%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Phanourios  Tamamis</span>, <span class="hlFld-ContribAuthor ">ChristodoulosÂ A.  Floudas</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Recognition of CXCR4 by a Dual Tropic HIV-1 gp120 V3 Loop. </span><span class="cited-content_cbyCitation_journal-name">Biophysical Journal</span><span> <strong>2013,</strong> <em>105 </em>
                                    (6)
                                     , 1502-1514. <a href="https://doi.org/10.1016/j.bpj.2013.07.049" title="DOI URL">https://doi.org/10.1016/j.bpj.2013.07.049</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bpj.2013.07.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bpj.2013.07.049%26sid%3Dliteratum%253Aachs%26jtitle%3DBiophysical%2520Journal%26atitle%3DMolecular%252BRecognition%252Bof%252BCXCR4%252Bby%252Ba%252BDual%252BTropic%252BHIV-1%252Bgp120%252BV3%252BLoop%26aulast%3DTamamis%26aufirst%3DPhanourios%26date%3D2013%26volume%3D105%26issue%3D6%26spage%3D1502%26epage%3D1514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqiang  Huang</span>, <span class="hlFld-ContribAuthor ">Kehang  Han</span>, <span class="hlFld-ContribAuthor ">Yushan  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic optimization model and algorithm for binding sequence selection in computational enzyme design. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2013,</strong> <em>22 </em>
                                    (7)
                                     , 929-941. <a href="https://doi.org/10.1002/pro.2275" title="DOI URL">https://doi.org/10.1002/pro.2275</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.2275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.2275%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DSystematic%252Boptimization%252Bmodel%252Band%252Balgorithm%252Bfor%252Bbinding%252Bsequence%252Bselection%252Bin%252Bcomputational%252Benzyme%252Bdesign%26aulast%3DHuang%26aufirst%3DXiaoqiang%26date%3D2013%26date%3D2013%26volume%3D22%26issue%3D7%26spage%3D929%26epage%3D941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mike C. L.  Wu</span>, <span class="hlFld-ContribAuthor ">Faith H.  Brennan</span>, <span class="hlFld-ContribAuthor ">Jason P. L.  Lynch</span>, <span class="hlFld-ContribAuthor ">Susanna  Mantovani</span>, <span class="hlFld-ContribAuthor ">Simon  Phipps</span>, <span class="hlFld-ContribAuthor ">Rick A.  Wetsel</span>, <span class="hlFld-ContribAuthor ">Marc J.  Ruitenberg</span>, <span class="hlFld-ContribAuthor ">Stephen M.  Taylor</span>, <span class="hlFld-ContribAuthor ">Trent M.  Woodruff</span>. </span><span class="cited-content_cbyCitation_article-title">The receptor for complement component C3a mediates protection from intestinal ischemia-reperfusion injuries by inhibiting neutrophil mobilization. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2013,</strong> <em>110 </em>
                                    (23)
                                     , 9439-9444. <a href="https://doi.org/10.1073/pnas.1218815110" title="DOI URL">https://doi.org/10.1073/pnas.1218815110</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1218815110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1218815110%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DThe%252Breceptor%252Bfor%252Bcomplement%252Bcomponent%252BC3a%252Bmediates%252Bprotection%252Bfrom%252Bintestinal%252Bischemia-reperfusion%252Binjuries%252Bby%252Binhibiting%252Bneutrophil%252Bmobilization%26aulast%3DWu%26aufirst%3DMike%2BC.%2BL.%26date%3D2013%26date%3D2013%26volume%3D110%26issue%3D23%26spage%3D9439%26epage%3D9444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Li</span>, <span class="hlFld-ContribAuthor ">Wuding  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Anaphylatoxins in organ transplantation. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Immunology</span><span> <strong>2013,</strong> <em>25 </em>
                                    (1)
                                     , 20-28. <a href="https://doi.org/10.1016/j.smim.2013.04.013" title="DOI URL">https://doi.org/10.1016/j.smim.2013.04.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.smim.2013.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.smim.2013.04.013%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Immunology%26atitle%3DAnaphylatoxins%252Bin%252Borgan%252Btransplantation%26aulast%3DLi%26aufirst%3DKe%26date%3D2013%26volume%3D25%26issue%3D1%26spage%3D20%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Agonist and antagonist activities of selected peptides. (A) Agonist activity. RBL-C3aR cells were treated with the indicated concentrations of peptides and receptor activation measured by Î²-hexosaminidase secretion. Data are the means of a representative experiment performed in duplicate and are shown as a percentage of the maximal response. (B) Antagonist activity. RBL-C3aR cells were treated with the indicated concentrations of peptides before the addition of an optimal concentration of C3aR agonist peptide FLPLAR. Data are the means Â± SEM of five experiments performed in duplicate and are shown as a percentage of the response to FLPLAR alone. A one-sample <i>t</i> test was used to determine if means were significantly different to 100%; ***: <i>p</i> < 0.0001; **: <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Agonist and antagonist activities of selected peptides. Agonist activity was measured for peptides SQ110â4 and SQ060â4 (A) and SQ007â5 and SQ002â5 (B). Dibutyryl cAMP differentiated U937 cells were treated with the peptides of interest and receptor activation measured by intracellular calcium mobilization. Following treatment with peptides SQ007â5 and SQ002â5, 10 Î¼M of the C3aR agonist, FLPLAR, was added and antagonist activity of the peptides measured (C). Data is displayed as the mean Â± SEM <i>n</i> = 3â6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Hierarchical clustering dendrogram of spatial distributions of electrostatic potentials for the 12 experimentally tested peptides of Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and four native peptides with different combinations of histidine tautomers. Electrostatic potentials were calculated at 0 mM ionic strength (A) and 150 mM ionic strength (B). Isopotential surfaces are shown at two different orientations rotated about the vertical axis and are plotted at Â±2.5 <i>k</i><sub>B</sub><i>T</i>/<i>e</i> (0 mM) and Â±1 <i>k</i><sub>B</sub><i>T</i>/<i>e</i> (150 mM), with blue and red denoting positive and negative electrostatic potential, respectively. The net charge (<i>Q</i>) of the sequences is given and is calculated by taking into account the charged side chains and the negatively charged backbone at the unblocked C-termini (and the positively charged backbone at the N-terminus of S4-noAc). Native peptide His67 and His72 protonation is as follows: (i) C3a-cterm, His67/His72 neutral; (ii) C3a-cterm HIP5, His67 charged, His72 neutral; (iii) C3a-cterm HIP10, His67 neutral, His72 charged; and (iv) C3a-cterm HIP5/10, His67/His72 charged.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/medium/jm-2011-01609k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Flexible design template of C3a (superposition of 10 MD snapshots). The last 15 residues are colored in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-9/jm201609k/production/images/large/jm-2011-01609k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm201609k&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76173" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76173" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Tamamis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archontis, G.</span><span> </span><span class="NLM_article-title">Species specificity of the complement inhibitor compstatin investigated by all-atom molecular dynamics simulations</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2655</span><span class="NLM_x">â</span> <span class="NLM_lpage">2667</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=2655-2667&issue=12&author=P.+Tamamisauthor=D.+Morikisauthor=C.+A.+Floudasauthor=G.+Archontis&title=Species+specificity+of+the+complement+inhibitor+compstatin+investigated+by+all-atom+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTamamis%26aufirst%3DP.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DArchontis%26aufirst%3DG.%26atitle%3DSpecies%2520specificity%2520of%2520the%2520complement%2520inhibitor%2520compstatin%2520investigated%2520by%2520all-atom%2520molecular%2520dynamics%2520simulations%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2010%26volume%3D78%26issue%3D12%26spage%3D2655%26epage%3D2667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Sahu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Structure and biology of complement protein C3, a connecting link between innate and acquired immunity</span> <span class="citation_source-journal">Immunol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">180</span><span class="NLM_x">, </span> <span class="NLM_fpage">35</span><span class="NLM_x">â</span> <span class="NLM_lpage">48</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1034%2Fj.1600-065X.2001.1800103.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=11414361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvFegsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2001&pages=35-48&author=A.+Sahuauthor=J.+Lambris&title=Structure+and+biology+of+complement+protein+C3%2C+a+connecting+link+between+innate+and+acquired+immunity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and biology of complement protein C3, a connecting link between innate and acquired immunity</span></div><div class="casAuthors">Sahu, Arvind; Lambris, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">180</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-48</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">0105-2896</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Complement protein C3 is a central mol. in the complement system whose activation is essential for all the important functions performed by this system.  After four decades of research it is now well established that C3 functions like a double-edged sword on the one hand it promotes phagocytosis, supports local inflammatory responses against pathogens, and instructs the adaptive immune response to select the appropriate antigens for a humoral response; on the other hand its unregulated activation leads to host cell damage.  In addn., its interactions with the proteins of foreign pathogens may provide a mechanism by which these microorganisms evade complement attack.  Therefore, a clear knowledge of the mol. and its interactions at the mol. level not only may allow the rational design of mol. adjuvants but may also lead to the development of complement inhibitors and new therapeutic agents against infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVtCdSnM10J7Vg90H21EOLACvtfcHk0lj51gI1Oid69g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvFegsrc%253D&md5=5b3d6457364b9b4c042eb3713ab4b1d0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1034%2Fj.1600-065X.2001.1800103.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1034%252Fj.1600-065X.2001.1800103.x%26sid%3Dliteratum%253Aachs%26aulast%3DSahu%26aufirst%3DA.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520biology%2520of%2520complement%2520protein%2520C3%252C%2520a%2520connecting%2520link%2520between%2520innate%2520and%2520acquired%2520immunity%26jtitle%3DImmunol.%2520Rev.%26date%3D2001%26volume%3D180%26spage%3D35%26epage%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Morikis, D.; Holland, M. C. H.; Lambris, J. D.</span><span> </span><span class="NLM_article-title">Structure of the anaphylatoxins C3a and C5a</span>. In  <span class="citation_source-book">Structural Biology of the Complement System</span>; <span class="NLM_contrib-group">Morikis, D.; Lambris, J. D.</span>, Eds.; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, FL</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">, </span><div class="note"><p class="first last">Chapter 7</p></div>, pp  <span class="NLM_fpage">161</span><span class="NLM_x">â</span> <span class="NLM_lpage">177</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1201%2F9780849350368" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=161-177&author=D.+Morikis&author=M.+C.+H.+Holland&author=J.+D.+Lambrisauthor=D.+Morikis&author=J.+D.+Lambris&title=Structural+Biology+of+the+Complement+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1201%2F9780849350368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1201%252F9780849350368%26sid%3Dliteratum%253Aachs%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DStructure%2520of%2520the%2520anaphylatoxins%2520C3a%2520and%2520C5a%26btitle%3DStructural%2520Biology%2520of%2520the%2520Complement%2520System%26aulast%3DMorikis%26aufirst%3DD.%26pub%3DCRC%2520Press%26date%3D2005%26spage%3D161%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Proctor, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monk, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanderson, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T. M.</span><span> </span><span class="NLM_article-title">Complement factors C3a and C5a have distinct hemodynamic effects in the rat</span> <span class="citation_source-journal">Int. Immunopharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">800</span><span class="NLM_x">â</span> <span class="NLM_lpage">806</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1016%2Fj.intimp.2009.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=19285573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsVequrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=800-806&author=L.+M.+Proctorauthor=T.+A.+Mooreauthor=P.+N.+Monkauthor=S.+D.+Sandersonauthor=S.+M.+Taylorauthor=T.+M.+Woodruff&title=Complement+factors+C3a+and+C5a+have+distinct+hemodynamic+effects+in+the+rat"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Complement factors C3a and C5a have distinct hemodynamic effects in the rat</span></div><div class="casAuthors">Proctor, Lavinia M.; Moore, Tyson A.; Monk, Peter N.; Sanderson, Sam D.; Taylor, Stephen M.; Woodruff, Trent M.</div><div class="citationInfo"><span class="NLM_cas:title">International Immunopharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">800-806</span>CODEN:
                <span class="NLM_cas:coden">IINMBA</span>;
        ISSN:<span class="NLM_cas:issn">1567-5769</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the rat, complement C5a infusion mediates well-defined effects including hypotension and neutropenia.  Conversely, the comparative effect of C3a in the rat is not yet defined.  Here, the authors investigated C3a receptor (C3aR) activation in the rat, using recombinant human C3a, the C3aR agonist WWGKKYRASKLGLAR, which is a C-terminal analog of C3a, and a non-peptide C3aR antagonist SB-290157, as pharmacol. tools.  In vitro, C3a and WWGKKYRASKLGLAR selectively bound to C3aRs and induced degranulation of C3aR-transfected RBL-2H3 cells.  C3a or WWGKKYRASKLGLAR-induced degranulation was dose-dependently antagonized in a surmountable fashion by the non-peptide C3aR antagonist.  I.v. infusion of C3a and WWGKKYRASKLGLAR to rats induced a rapid, transient and concn.-dependent hypertensive response, which was mediated by C3aR-induced prostanoid release.  C3a and WWGKKYRASKLGLAR caused a small drop in circulating neutrophils, but a rise in circulating neutrophils was evident after 90-120 min.  In contrast to C3a, C5a infusion resulted in hypotension, and rapid and transient neutropenia.  Thus, C3a and C5a mediate distinct effects on blood pressure and circulating polymorphonuclear leukocytes in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq91jiRORQJvLVg90H21EOLACvtfcHk0lgjkRnnJde70A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsVequrs%253D&md5=9b40aa2b1c555fe65b4517517a9322c7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.intimp.2009.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.intimp.2009.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DProctor%26aufirst%3DL.%2BM.%26aulast%3DMoore%26aufirst%3DT.%2BA.%26aulast%3DMonk%26aufirst%3DP.%2BN.%26aulast%3DSanderson%26aufirst%3DS.%2BD.%26aulast%3DTaylor%26aufirst%3DS.%2BM.%26aulast%3DWoodruff%26aufirst%3DT.%2BM.%26atitle%3DComplement%2520factors%2520C3a%2520and%2520C5a%2520have%2520distinct%2520hemodynamic%2520effects%2520in%2520the%2520rat%26jtitle%3DInt.%2520Immunopharmacol.%26date%3D2009%26volume%3D9%26spage%3D800%26epage%3D806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Daffern, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeifer, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ember, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span> </span><span class="NLM_article-title">C3a is a chemotaxin for human eosinophils but not for neutrophils. I. C3a stimulation of neutrophils is secondary to eosinophil activation</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">181</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">2119</span><span class="NLM_x">â</span> <span class="NLM_lpage">2127</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=1995&pages=2119-2127&issue=6&author=P.+J.+Daffernauthor=P.+H.+Pfeiferauthor=J.+A.+Emberauthor=T.+E.+Hugli&title=C3a+is+a+chemotaxin+for+human+eosinophils+but+not+for+neutrophils.+I.+C3a+stimulation+of+neutrophils+is+secondary+to+eosinophil+activation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDaffern%26aufirst%3DP.%2BJ.%26aulast%3DPfeifer%26aufirst%3DP.%2BH.%26aulast%3DEmber%26aufirst%3DJ.%2BA.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26atitle%3DC3a%2520is%2520a%2520chemotaxin%2520for%2520human%2520eosinophils%2520but%2520not%2520for%2520neutrophils.%2520I.%2520C3a%2520stimulation%2520of%2520neutrophils%2520is%2520secondary%2520to%2520eosinophil%2520activation%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D1995%26volume%3D181%26issue%3D6%26spage%3D2119%26epage%3D2127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Elsner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czech, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">SchÃ¶pf, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norgauer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapp, A.</span><span> </span><span class="NLM_article-title">C3a activates reactive oxygen radical species production and intracellular calcium transients in human eosinophils</span> <span class="citation_source-journal">Eur. J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">518</span><span class="NLM_x">â</span> <span class="NLM_lpage">522</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1994&pages=518-522&issue=3&author=J.+Elsnerauthor=M.+Oppermannauthor=W.+Czechauthor=G.+Dobosauthor=E.+Sch%C3%B6pfauthor=J.+Norgauerauthor=A.+Kapp&title=C3a+activates+reactive+oxygen+radical+species+production+and+intracellular+calcium+transients+in+human+eosinophils"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DElsner%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DM.%26aulast%3DCzech%26aufirst%3DW.%26aulast%3DDobos%26aufirst%3DG.%26aulast%3DSch%25C3%25B6pf%26aufirst%3DE.%26aulast%3DNorgauer%26aufirst%3DJ.%26aulast%3DKapp%26aufirst%3DA.%26atitle%3DC3a%2520activates%2520reactive%2520oxygen%2520radical%2520species%2520production%2520and%2520intracellular%2520calcium%2520transients%2520in%2520human%2520eosinophils%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D1994%26volume%3D24%26issue%3D3%26spage%3D518%26epage%3D522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Hartmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henz, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KrÃ¼ger-Krasagakes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">KÃ¶hl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guhl, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haase, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippert, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuberbier, T.</span><span> </span><span class="NLM_article-title">C3a and C5a stimulate chemotaxis of human mast cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2863</span><span class="NLM_x">â</span> <span class="NLM_lpage">2870</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1997&pages=2863-2870&issue=8&author=K.+Hartmannauthor=B.+M.+Henzauthor=S.+Kr%C3%BCger-Krasagakesauthor=J.+K%C3%B6hlauthor=R.+Burgerauthor=S.+Guhlauthor=I.+Haaseauthor=U.+Lippertauthor=T.+Zuberbier&title=C3a+and+C5a+stimulate+chemotaxis+of+human+mast+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DK.%26aulast%3DHenz%26aufirst%3DB.%2BM.%26aulast%3DKr%25C3%25BCger-Krasagakes%26aufirst%3DS.%26aulast%3DK%25C3%25B6hl%26aufirst%3DJ.%26aulast%3DBurger%26aufirst%3DR.%26aulast%3DGuhl%26aufirst%3DS.%26aulast%3DHaase%26aufirst%3DI.%26aulast%3DLippert%26aufirst%3DU.%26aulast%3DZuberbier%26aufirst%3DT.%26atitle%3DC3a%2520and%2520C5a%2520stimulate%2520chemotaxis%2520of%2520human%2520mast%2520cells%26jtitle%3DBlood%26date%3D1997%26volume%3D89%26issue%3D8%26spage%3D2863%26epage%3D2870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Takafuji, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tadokoro, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">lto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahinden, C. A.</span><span> </span><span class="NLM_article-title">Degranulation from human eosinophils stimulated with C3a and C5a</span> <span class="citation_source-journal">Int. Arch. Allergy Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">27</span><span class="NLM_x">â</span> <span class="NLM_lpage">29</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1159%2F000236743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=8156000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK2cXjt1Whtbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=1994&pages=27-29&issue=Suppl.+1&author=S.+Takafujiauthor=K.+Tadokoroauthor=K.+ltoauthor=C.+A.+Dahinden&title=Degranulation+from+human+eosinophils+stimulated+with+C3a+and+C5a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Degranulation from human eosinophils stimulated with C3a and C5a</span></div><div class="casAuthors">Takafuji, Shigeru; Tadokoro, Kenji; Ito, Koji; Dahinden, Clemens A.</div><div class="citationInfo"><span class="NLM_cas:title">International Archives of Allergy and Immunology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">SUPPL. 1</span>),
    <span class="NLM_cas:pages">27-9</span>CODEN:
                <span class="NLM_cas:coden">IAAIEG</span>;
        ISSN:<span class="NLM_cas:issn">1018-2438</span>.
    </div><div class="casAbstract">It is unclear what actually induces eosinophil degranulation in vivo.  The authors examd. eosinophil cationic protein (ECP) release from normal human eosinophils (Eos) in response to C3a and C5a.  C3a and C5a induced remarkable ECP release when Eos were preincubated with cytochalasin B.  ECP release induced by C3a or C5a was greater than that induced by PAF at a concn. of 10-7 M.  The ED50 for ECP release was 3Ã10-8 M and 3Ã10-9 M for C3a and C5a, resp.  C3a or C5a elicited a rapid and transient rise in [Ca2+]i.  Thus, C3a and C5a may contribute to hypersensitivity diseases by inducing eosinophil degranulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0MNKtke8ZF7Vg90H21EOLACvtfcHk0lgjkRnnJde70A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjt1Whtbg%253D&md5=a36b5462c62c62e290ed646820ff7609</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1159%2F000236743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000236743%26sid%3Dliteratum%253Aachs%26aulast%3DTakafuji%26aufirst%3DS.%26aulast%3DTadokoro%26aufirst%3DK.%26aulast%3Dlto%26aufirst%3DK.%26aulast%3DDahinden%26aufirst%3DC.%2BA.%26atitle%3DDegranulation%2520from%2520human%2520eosinophils%2520stimulated%2520with%2520C3a%2520and%2520C5a%26jtitle%3DInt.%2520Arch.%2520Allergy%2520Immunol.%26date%3D1994%26volume%3D104%26issue%3DSuppl.%25201%26spage%3D27%26epage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Marceau, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundberg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span> </span><span class="NLM_article-title">Effects of the anaphylatoxins in circulation</span> <span class="citation_source-journal">Immunopharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">67</span><span class="NLM_x">â</span> <span class="NLM_lpage">84</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1016%2F0162-3109%2887%2990031-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=2828271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaL1cXktlar" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1987&pages=67-84&issue=2&author=F.+Marceauauthor=C.+Lundbergauthor=T.+E.+Hugli&title=Effects+of+the+anaphylatoxins+in+circulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of the anaphylatoxins on circulation</span></div><div class="casAuthors">Marceau, Francois; Lundberg, Claes; Hugli, Tony E.</div><div class="citationInfo"><span class="NLM_cas:title">Immunopharmacology</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-84</span>CODEN:
                <span class="NLM_cas:coden">IMMUDP</span>;
        ISSN:<span class="NLM_cas:issn">0162-3109</span>.
    </div><div class="casAbstract">A review with 87 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEpPEJ0vvgK7Vg90H21EOLACvtfcHk0lhH-hXSGjtolA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktlar&md5=9f2e1971589d924b9ec941e5329210c5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2F0162-3109%2887%2990031-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0162-3109%252887%252990031-2%26sid%3Dliteratum%253Aachs%26aulast%3DMarceau%26aufirst%3DF.%26aulast%3DLundberg%26aufirst%3DC.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26atitle%3DEffects%2520of%2520the%2520anaphylatoxins%2520in%2520circulation%26jtitle%3DImmunopharmacology%26date%3D1987%26volume%3D14%26issue%3D2%26spage%3D67%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Fischer, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagels, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span> </span><span class="NLM_article-title">Regulation of IL-6 synthesis in human peripheral blood mononuclear cells by C3a and C3adesArg</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">453</span><span class="NLM_x">â</span> <span class="NLM_lpage">459</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=1999&pages=453-459&issue=1&author=W.+H.+Fischerauthor=M.+A.+Jagelsauthor=T.+E.+Hugli&title=Regulation+of+IL-6+synthesis+in+human+peripheral+blood+mononuclear+cells+by+C3a+and+C3adesArg"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DW.%2BH.%26aulast%3DJagels%26aufirst%3DM.%2BA.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26atitle%3DRegulation%2520of%2520IL-6%2520synthesis%2520in%2520human%2520peripheral%2520blood%2520mononuclear%2520cells%2520by%2520C3a%2520and%2520C3adesArg%26jtitle%3DJ.%2520Immunol.%26date%3D1999%26volume%3D162%26issue%3D1%26spage%3D453%26epage%3D459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Takabayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vannier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tompkins, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelfand, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, B. D.</span><span> </span><span class="NLM_article-title">Both C3a and C3a desArg regulate Interleukin-6 synthesis in human peripheral blood mononuclear cells</span> <span class="citation_source-journal">J. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">177</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1622</span><span class="NLM_x">â</span> <span class="NLM_lpage">1628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1086%2F515316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=9607842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK1cXjslKitrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=1998&pages=1622-1628&issue=6&author=T.+Takabayashiauthor=E.+Vannierauthor=J.+F.+Burkeauthor=R.+G.+Tompkinsauthor=J.+A.+Gelfandauthor=B.+D.+Clark&title=Both+C3a+and+C3a+desArg+regulate+Interleukin-6+synthesis+in+human+peripheral+blood+mononuclear+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Both C3a and C3adesArg regulate interleukin-6 synthesis in human peripheral blood mononuclear cells</span></div><div class="casAuthors">Takabayashi, Tsukasa; Vannier, Edouard; Burke, John F.; Tompkins, Ronald G.; Gelfand, Jeffrey A.; Clark, Burton D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1622-1628</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Synthesis of complement components is part of the acute-phase response.  Interleukin-6 (IL-6) is a crit. mediator of the acute-phase response during infections and injuries.  Plasma levels of C3a and IL-6 have been proposed as prognostic indicators in sepsis and trauma.  The effects of C3a and C3adesArg on IL-6 gene expression and protein prodn. in human peripheral blood mononuclear cells (PBMC) were investigated.  Neither C3a nor C3adesArg alone induced detectable IL-6 protein or mRNA levels.  However, C3a and C3adesArg affected endotoxin-induced IL-6 synthesis in a dose-dependent manner.  In nonadherent PBMC, C3a or C3adesArg suppressed, while in adherent PBMC, C3a or C3adesArg enhanced IL-6 protein and mRNA levels.  Thus, C3a and C3adesArg may provide a control mechanism of acute-phase responses by enhancing IL-6 synthesis in adherent monocytes at local inflammatory sites and by inhibiting IL-6 synthesis in circulating monocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfaUH9cD90SbVg90H21EOLACvtfcHk0lhIxynOdXJ9Jw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjslKitrs%253D&md5=158b02ca08e2f0ee388ac7d005bdf6ad</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1086%2F515316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F515316%26sid%3Dliteratum%253Aachs%26aulast%3DTakabayashi%26aufirst%3DT.%26aulast%3DVannier%26aufirst%3DE.%26aulast%3DBurke%26aufirst%3DJ.%2BF.%26aulast%3DTompkins%26aufirst%3DR.%2BG.%26aulast%3DGelfand%26aufirst%3DJ.%2BA.%26aulast%3DClark%26aufirst%3DB.%2BD.%26atitle%3DBoth%2520C3a%2520and%2520C3a%2520desArg%2520regulate%2520Interleukin-6%2520synthesis%2520in%2520human%2520peripheral%2520blood%2520mononuclear%2520cells%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D1998%26volume%3D177%26issue%3D6%26spage%3D1622%26epage%3D1628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Takabayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vannier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinarello, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelfand, J.</span><span> </span><span class="NLM_article-title">A new biologic role for C3a and C3a desArg: regulation of TNF-alpha and IL-1 beta synthesis</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">156</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">3455</span><span class="NLM_x">â</span> <span class="NLM_lpage">3460</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=1996&pages=3455-3460&issue=9&author=T.+Takabayashiauthor=E.+Vannierauthor=B.+Clarkauthor=N.+Margolisauthor=C.+Dinarelloauthor=J.+Burkeauthor=J.+Gelfand&title=A+new+biologic+role+for+C3a+and+C3a+desArg%3A+regulation+of+TNF-alpha+and+IL-1+beta+synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTakabayashi%26aufirst%3DT.%26aulast%3DVannier%26aufirst%3DE.%26aulast%3DClark%26aufirst%3DB.%26aulast%3DMargolis%26aufirst%3DN.%26aulast%3DDinarello%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DJ.%26aulast%3DGelfand%26aufirst%3DJ.%26atitle%3DA%2520new%2520biologic%2520role%2520for%2520C3a%2520and%2520C3a%2520desArg%253A%2520regulation%2520of%2520TNF-alpha%2520and%2520IL-1%2520beta%2520synthesis%26jtitle%3DJ.%2520Immunol.%26date%3D1996%26volume%3D156%26issue%3D9%26spage%3D3455%26epage%3D3460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Ember, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T.</span><span> </span><span class="NLM_article-title">Designing synthetic superagonists of C3a anaphylatoxin</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3603</span><span class="NLM_x">â</span> <span class="NLM_lpage">3612</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi00229a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=3603-3612&author=J.+Emberauthor=N.+Johansenauthor=T.+Hugli&title=Designing+synthetic+superagonists+of+C3a+anaphylatoxin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi00229a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00229a003%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DJ.%26aulast%3DJohansen%26aufirst%3DN.%26aulast%3DHugli%26aufirst%3DT.%26atitle%3DDesigning%2520synthetic%2520superagonists%2520of%2520C3a%2520anaphylatoxin%26jtitle%3DBiochemistry%26date%3D1991%26volume%3D30%26spage%3D3603%26epage%3D3612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Scully, C. C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blakeney, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoang, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abenante, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlie, D. P.</span><span> </span><span class="NLM_article-title">Selective hexapeptide agonists and antagonists for human complement C3a receptor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4938</span><span class="NLM_x">â</span> <span class="NLM_lpage">4948</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1003705" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntFyksbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4938-4948&author=C.+C.+G.+Scullyauthor=J.+S.+Blakeneyauthor=R.+Singhauthor=H.+N.+Hoangauthor=G.+Abenanteauthor=R.+C.+Reidauthor=D.+P.+Fairlie&title=Selective+hexapeptide+agonists+and+antagonists+for+human+complement+C3a+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Hexapeptide Agonists and Antagonists for Human Complement C3a Receptor</span></div><div class="casAuthors">Scully, Conor C. G.; Blakeney, Jade S.; Singh, Ranee; Hoang, Huy N.; Abbenante, Giovanni; Reid, Robert C.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4938-4948</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human anaphylatoxin C3a, formed through cleavage of complement protein C3, is a potent effector of innate immunity via activation of its G protein coupled receptor, human C3aR.  Previously reported short peptide ligands for this receptor either have low potency or lack receptor selectivity.  Here the authors report the first small peptide agonists that are both potent and selective for human C3aR, derived from structure-activity relationships of peptides based on the C-terminus of C3a.  Affinity for C3aR was examd. by competitive binding with 125I-labeled C3a to human macrophages, agonist vs. antagonist activity measured using fluorescence detection of intracellular calcium, and general selectivity monitored by C3a-induced receptor desensitization.  An NMR structure for an agonist in DMSO showed a Î²-turn motif that may be important for C3aR binding and activation.  Derivatization produced a noncompetitive and insurmountable antagonist of C3aR.  Small mol. C3a agonists and antagonists may be valuable probes of immunity and inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9WeiihoJlNbVg90H21EOLACvtfcHk0lgdn286aw9Ppg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntFyksbo%253D&md5=a52f9cf07b0e4b5df3b2ee362d1e004f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm1003705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1003705%26sid%3Dliteratum%253Aachs%26aulast%3DScully%26aufirst%3DC.%2BC.%2BG.%26aulast%3DBlakeney%26aufirst%3DJ.%2BS.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DAbenante%26aufirst%3DG.%26aulast%3DReid%26aufirst%3DR.%2BC.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DSelective%2520hexapeptide%2520agonists%2520and%2520antagonists%2520for%2520human%2520complement%2520C3a%2520receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4938%26epage%3D4948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Holland, M. C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Synthetic small-molecule complement inhibitors</span> <span class="citation_source-journal">Curr. Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">1164</span><span class="NLM_x">â</span> <span class="NLM_lpage">1173</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=1164-1173&issue=11&author=M.+C.+H.+Hollandauthor=D.+Morikisauthor=J.+Lambris&title=Synthetic+small-molecule+complement+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHolland%26aufirst%3DM.%2BC.%2BH.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DSynthetic%2520small-molecule%2520complement%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Invest.%2520Drugs%26date%3D2004%26volume%3D5%26issue%3D11%26spage%3D1164%26epage%3D1173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsokos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argyropoulos, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spruce, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Integrated computational and experimental approach for lead optimization and design of compstatin variants with improved activity</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">8422</span><span class="NLM_x">â</span> <span class="NLM_lpage">8423</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja034846p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=8422-8423&author=J.+Klepeisauthor=C.+Floudasauthor=D.+Morikisauthor=C.+Tsokosauthor=E.+Argyropoulosauthor=L.+Spruceauthor=J.+Lambris&title=Integrated+computational+and+experimental+approach+for+lead+optimization+and+design+of+compstatin+variants+with+improved+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja034846p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja034846p%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DFloudas%26aufirst%3DC.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DTsokos%26aufirst%3DC.%26aulast%3DArgyropoulos%26aufirst%3DE.%26aulast%3DSpruce%26aufirst%3DL.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DIntegrated%2520computational%2520and%2520experimental%2520approach%2520for%2520lead%2520optimization%2520and%2520design%2520of%2520compstatin%2520variants%2520with%2520improved%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D8422%26epage%3D8423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsokos, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J. D.</span><span> </span><span class="NLM_article-title">Design of peptide analogs with improved activity using a novel de novo protein design approach</span> <span class="citation_source-journal">Ind. Eng. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3817</span><span class="NLM_x">â</span> <span class="NLM_lpage">3826</span></span><div class="citationLinks">[<a href="/doi/10.1021/ie0340995" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2004&pages=3817-3826&author=J.+L.+Klepeisauthor=C.+A.+Floudasauthor=D.+Morikisauthor=C.+G.+Tsokosauthor=J.+D.+Lambris&title=Design+of+peptide+analogs+with+improved+activity+using+a+novel+de+novo+protein+design+approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fie0340995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fie0340995%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DTsokos%26aufirst%3DC.%2BG.%26aulast%3DLambris%26aufirst%3DJ.%2BD.%26atitle%3DDesign%2520of%2520peptide%2520analogs%2520with%2520improved%2520activity%2520using%2520a%2520novel%2520de%2520novo%2520protein%2520design%2520approach%26jtitle%3DInd.%2520Eng.%2520Chem.%2520Res.%26date%3D2004%26volume%3D43%26spage%3D3817%26epage%3D3826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Novel formulations for the sequence selection problem in de novo protein design with flexible templates</span> <span class="citation_source-journal">Optimization Methods Software</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">â</span> <span class="NLM_lpage">71</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1080%2F10556780600882041" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2007&pages=51-71&author=H.+K.+Fungauthor=M.+S.+Taylorauthor=C.+A.+Floudas&title=Novel+formulations+for+the+sequence+selection+problem+in+de+novo+protein+design+with+flexible+templates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F10556780600882041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10556780600882041%26sid%3Dliteratum%253Aachs%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DNovel%2520formulations%2520for%2520the%2520sequence%2520selection%2520problem%2520in%2520de%2520novo%2520protein%2520design%2520with%2520flexible%2520templates%26jtitle%3DOptimization%2520Methods%2520Software%26date%3D2007%26volume%3D22%26spage%3D51%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">Toward full-sequence de novo protein design with flexible templates for human beta-defensin-2</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">584</span><span class="NLM_x">â</span> <span class="NLM_lpage">599</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2008&pages=584-599&author=H.+K.+Fungauthor=C.+A.+Floudasauthor=M.+S.+Taylorauthor=L.+Zhangauthor=D.+Morikis&title=Toward+full-sequence+de+novo+protein+design+with+flexible+templates+for+human+beta-defensin-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DToward%2520full-sequence%2520de%2520novo%2520protein%2520design%2520with%2520flexible%2520templates%2520for%2520human%2520beta-defensin-2%26jtitle%3DBiophys.%2520J.%26date%3D2008%26volume%3D94%26spage%3D584%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Bellows, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez de Victoria, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">New compstatin variants through two de novo protein design frameworks</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">2337</span><span class="NLM_x">â</span> <span class="NLM_lpage">2346</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2010&pages=2337-2346&issue=10&author=M.+L.+Bellowsauthor=H.+K.+Fungauthor=C.+A.+Floudasauthor=A.+Lopez+de+Victoriaauthor=D.+Morikis&title=New+compstatin+variants+through+two+de+novo+protein+design+frameworks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBellows%26aufirst%3DM.%2BL.%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DLopez%2Bde%2BVictoria%26aufirst%3DA.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DNew%2520compstatin%2520variants%2520through%2520two%2520de%2520novo%2520protein%2520design%2520frameworks%26jtitle%3DBiophys.%2520J.%26date%3D2010%26volume%3D98%26issue%3D10%26spage%3D2337%26epage%3D2346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Bellows, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siliciano, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Discovery of entry inhibitors for HIV-1 via a new de novo protein design framework</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">3445</span><span class="NLM_x">â</span> <span class="NLM_lpage">3453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2010&pages=3445-3453&author=M.+L.+Bellowsauthor=M.+S.+Taylorauthor=P.+A.+Coleauthor=L.+Shenauthor=R.+F.+Silicianoauthor=H.+K.+Fungauthor=C.+A.+Floudas&title=Discovery+of+entry+inhibitors+for+HIV-1+via+a+new+de+novo+protein+design+framework"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBellows%26aufirst%3DM.%2BL.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DSiliciano%26aufirst%3DR.%2BF.%26aulast%3DFung%26aufirst%3DH.%2BK.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DDiscovery%2520of%2520entry%2520inhibitors%2520for%2520HIV-1%2520via%2520a%2520new%2520de%2520novo%2520protein%2520design%2520framework%26jtitle%3DBiophys.%2520J.%26date%3D2010%26volume%3D99%26spage%3D3445%26epage%3D3453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">LÃ³pez de Victoria, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorham, R. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellows-Peterson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lo, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">A new generation of potent complement inhibitors of the compstatin family</span> <span class="citation_source-journal">Chem. Biol. Drug. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">77</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">â</span> <span class="NLM_lpage">440</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1111%2Fj.1747-0285.2011.01111.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=21352502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslWqsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=431-440&author=A.+L%C3%B3pez+de+Victoriaauthor=R.+D.+Gorhamauthor=M.+L.+Bellows-Petersonauthor=J.+Lingauthor=D.+D.+Loauthor=C.+A.+Floudasauthor=D.+Morikis&title=A+new+generation+of+potent+complement+inhibitors+of+the+compstatin+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A new generation of potent complement inhibitors of the Compstatin family</span></div><div class="casAuthors">Lopez de Victoria, Aliana; Gorham, Ronald D., Jr.; Bellows-Peterson, Meghan L.; Ling, Jun; Lo, David D.; Floudas, Christodoulos A.; Morikis, Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">431-440</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Compstatin family peptides are potent inhibitors of the complement system and promising drug candidates against diseases involving under-regulated complement activation.  Compstatin is a 13-residue cyclized peptide that inhibits cleavage of complement protein C3, preventing downstream complement activation.  We present three new compstatin variants, characterized by tryptophan replacement at positions 1 and/or 13.  Peptide design was based on physicochem. reasoning and was inspired by earlier work, which identified tryptophan substitutions at positions 1 and 13 in peptides with predicted C3c binding abilities.  The new variants preserve distinct polar and nonpolar surfaces of compstatin, but have altered local interaction capabilities with C3.  All three peptides exhibited potent C3 binding by surface plasmon resonance and potent complement inhibition by ELISA.  We also present ELISA data and detailed surface plasmon resonance kinetic data of three peptides from previous computational design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZWQ1bOVHUhrVg90H21EOLACvtfcHk0lgprmOxBev4Kw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslWqsbg%253D&md5=f987241f1aa8c6c7a2cb564c6d7d80d0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01111.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01111.x%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B3pez%2Bde%2BVictoria%26aufirst%3DA.%26aulast%3DGorham%26aufirst%3DR.%2BD.%26aulast%3DBellows-Peterson%26aufirst%3DM.%2BL.%26aulast%3DLing%26aufirst%3DJ.%26aulast%3DLo%26aufirst%3DD.%2BD.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DA%2520new%2520generation%2520of%2520potent%2520complement%2520inhibitors%2520of%2520the%2520compstatin%2520family%26jtitle%3DChem.%2520Biol.%2520Drug.%2520Des.%26date%3D2011%26volume%3D77%26spage%3D431%26epage%3D440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Sun, J.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdeljabbar, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellows, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Link, A. J.</span><span> </span><span class="NLM_article-title">Reconstitution and engineering of apoptotic protein interactions on the bacterial cell surface</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">394</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">â</span> <span class="NLM_lpage">305</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=2009&pages=297-305&author=J.-J.+Sunauthor=D.+M.+Abdeljabbarauthor=N.+L.+Clarkeauthor=M.+L.+Bellowsauthor=C.+A.+Floudasauthor=A.+J.+Link&title=Reconstitution+and+engineering+of+apoptotic+protein+interactions+on+the+bacterial+cell+surface"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.-J.%26aulast%3DAbdeljabbar%26aufirst%3DD.%2BM.%26aulast%3DClarke%26aufirst%3DN.%2BL.%26aulast%3DBellows%26aufirst%3DM.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DLink%26aufirst%3DA.%2BJ.%26atitle%3DReconstitution%2520and%2520engineering%2520of%2520apoptotic%2520protein%2520interactions%2520on%2520the%2520bacterial%2520cell%2520surface%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D394%26spage%3D297%26epage%3D305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Fung, H. K.</span>Computational De Novo Protein Design with Flexible Templates. Ph.D. Thesis. <span class="NLM_publisher-name">Princeton University</span><span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=H.+K.+Fung&title=Computational+De+Novo+Protein+Design+with+Flexible+Templates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFung%26aufirst%3DH.%2BK.%26jtitle%3DComputational%2520De%2520Novo%2520Protein%2520Design%2520with%2520Flexible%2520Templates%26pub%3DPrinceton%2520University%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Proctor, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodruff, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiels, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S.</span><span> </span><span class="NLM_article-title">Transdermal pharmacology of small molecule cyclic C5a antagonists</span> <span class="citation_source-journal">Adv. Exp. Med. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">586</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">â</span> <span class="NLM_lpage">345</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2006&pages=329-345&author=L.+Proctorauthor=T.+Woodruffauthor=P.+Sharmaauthor=I.+Shielsauthor=S.+Taylor&title=Transdermal+pharmacology+of+small+molecule+cyclic+C5a+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DProctor%26aufirst%3DL.%26aulast%3DWoodruff%26aufirst%3DT.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DShiels%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26atitle%3DTransdermal%2520pharmacology%2520of%2520small%2520molecule%2520cyclic%2520C5a%2520antagonists%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2006%26volume%3D586%26spage%3D329%26epage%3D345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Burg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rheinheimer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bautsch, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klos, A.</span><span> </span><span class="NLM_article-title">Differential regulation of the C3a and C5a receptors (CD88) by IFN-gamma and PMA in U937 cells and related myeloblastic cell lines</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">5574</span><span class="NLM_x">â</span> <span class="NLM_lpage">5581</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=1996&pages=5574-5581&author=M.+Burgauthor=U.+Martinauthor=D.+Bockauthor=C.+Rheinheimerauthor=J.+Kohlauthor=W.+Bautschauthor=A.+Klos&title=Differential+regulation+of+the+C3a+and+C5a+receptors+%28CD88%29+by+IFN-gamma+and+PMA+in+U937+cells+and+related+myeloblastic+cell+lines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBurg%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DU.%26aulast%3DBock%26aufirst%3DD.%26aulast%3DRheinheimer%26aufirst%3DC.%26aulast%3DKohl%26aufirst%3DJ.%26aulast%3DBautsch%26aufirst%3DW.%26aulast%3DKlos%26aufirst%3DA.%26atitle%3DDifferential%2520regulation%2520of%2520the%2520C3a%2520and%2520C5a%2520receptors%2520%2528CD88%2529%2520by%2520IFN-gamma%2520and%2520PMA%2520in%2520U937%2520cells%2520and%2520related%2520myeloblastic%2520cell%2520lines%26jtitle%3DJ.%2520Immunol.%26date%3D1996%26volume%3D157%26spage%3D5574%26epage%3D5581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ember, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, T.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hugli, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuoka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, R. D.</span><span> </span><span class="NLM_article-title">Identification of ligand effector binding sites in transmembrane regions of the human G protein-coupled C3a receptor</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">11</span><span class="NLM_x">) </span> <span class="NLM_fpage">2304</span><span class="NLM_x">â</span> <span class="NLM_lpage">2311</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1110%2Fps.8.11.2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=10595533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK1MXnsFSiu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1999&pages=2304-2311&issue=11&author=J.+Sunauthor=J.+A.+Emberauthor=T.-H.+Chaoauthor=T.+E.+Hugliauthor=Y.+Fukuokaauthor=R.+D.+Ye&title=Identification+of+ligand+effector+binding+sites+in+transmembrane+regions+of+the+human+G+protein-coupled+C3a+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of ligand effector binding sites in transmembrane regions of the human G protein-coupled C3a receptor</span></div><div class="casAuthors">Sun, Jianzhong; Ember, Julia A.; Chao, Ta-Hsiang; Fukuoka, Yoshihiro; Ye, Richard D.; Hugli, Tony E.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2304-2311</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The human C3a anaphylatoxin receptor (C3aR) is a G protein-coupled receptor (GPCR) composed of seven transmembrane Î±-helixes connected by hydrophilic loops.  Previous studies of chimeric C3aR/C5aR and loop deletions in C3aR demonstrated that the large extracellular loop2 plays an important role in non-effector ligand binding; however, the effector binding site for C3a has not been identified.  In this study, selected charged residues in the transmembrane regions of C3aR were replaced by Ala using site-directed mutagenesis, and mutant receptors were stably expressed in the RBL-2H3 cell line.  Ligand binding studies demonstrated that R161A (helix IV), R340A (helix V), and D417A (helix VII) showed no binding activity, although full expression of these receptors was established by flow cytometric anal.  C3a induced very weak intracellular calcium flux in cells expressing these three mutant receptors.  H81A (helix II) and K96A (helix III) showed decreased ligand binding activity.  The calcium flux induced by C3a in H81A and K96A cells was also consistently reduced.  These findings suggest that the charged transmembrane residues Arg161, Arg340, and Asp417 in C3aR are essential for ligand effector binding and/or signal coupling, and that residues His81 and Lys96 may contribute less directly to the overall free energy of ligand binding.  These transmembrane residues in C3aR identify specific mol. contacts for ligand interactions that account for C3a-induced receptor activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMeO-VRrMmQbVg90H21EOLACvtfcHk0lgprmOxBev4Kw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnsFSiu7Y%253D&md5=6a77ea26411f1aec85965839bc289e89</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1110%2Fps.8.11.2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.8.11.2304%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DEmber%26aufirst%3DJ.%2BA.%26aulast%3DChao%26aufirst%3DT.-H.%26aulast%3DHugli%26aufirst%3DT.%2BE.%26aulast%3DFukuoka%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DR.%2BD.%26atitle%3DIdentification%2520of%2520ligand%2520effector%2520binding%2520sites%2520in%2520transmembrane%2520regions%2520of%2520the%2520human%2520G%2520protein-coupled%2520C3a%2520receptor%26jtitle%3DProtein%2520Sci.%26date%3D1999%26volume%3D8%26issue%3D11%26spage%3D2304%26epage%3D2311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Free energy calculations for peptides via deterministic global optimization</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7491</span><span class="NLM_x">â</span> <span class="NLM_lpage">7512</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1999&pages=7491-7512&author=J.+L.+Klepeisauthor=C.+A.+Floudas&title=Free+energy+calculations+for+peptides+via+deterministic+global+optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DFree%2520energy%2520calculations%2520for%2520peptides%2520via%2520deterministic%2520global%2520optimization%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1999%26volume%3D110%26spage%3D7491%26epage%3D7512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambris, J.</span><span> </span><span class="NLM_article-title">Predicting peptide structures using NMR data and deterministic global optimization</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1354</span><span class="NLM_x">â</span> <span class="NLM_lpage">1370</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1002%2F%28SICI%291096-987X%28199910%2920%3A13%3C1354%3A%3AAID-JCC3%3E3.0.CO%3B2-N" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK1MXmt1Ggtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1999&pages=1354-1370&author=J.+L.+Klepeisauthor=C.+A.+Floudasauthor=D.+Morikisauthor=J.+Lambris&title=Predicting+peptide+structures+using+NMR+data+and+deterministic+global+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting peptide structures using NMR data and deterministic global optimization</span></div><div class="casAuthors">Klepeis, J. L.; Floudas, C. A.; Morikis, D.; Lambris, J. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1354-1370</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The ability to analyze large mol. structures by NMR techniques requires efficient methods for structure calcn.  Currently, there are several widely available methods for tackling these problems, which, in general, rely on the optimization of penalty-type target functions to satisfy the conformational restraints.  Typically, these methods combine simulated annealing protocols with mol. dynamics and local minimization, either in distance or torsional angle space.  In this work, both a novel formulation and algorithmic procedure for the soln. of the NMR structure prediction problem is outlined.  First, the unconstrained, penalty-type structure prediction problem is reformulated using nonlinear constraints, which can be individually enumerated for all, or subsets, of the distance restraints.  In this way, the violation can be controlled as a constraint, in contrast to the usual penalty-type restraints.  In addn., the customary simplified objective function is replaced by a full atom force field in the torsional angle space.  This guarantees a better description of at. interactions, which dictate the native structure of the mol. along with the distance restraints.  The second novel portion of this work involves the soln. method.  Rather than pursue the typical simulated annealing procedure, this work relies on a deterministic method, which theor. guarantees that the global soln. can be located.  This branch and bound technique, based on the Î±BB algorithm, has already been successfully applied to the identification of global min. energy structures of peptides modeled by full atom force fields.  Finally, the approach is applied to the Compstatin structure prediction, and it is found to possess some important merits when compared to existing techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwBhVpptx3-bVg90H21EOLACvtfcHk0lg_m1hn0VDZUQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmt1Ggtrw%253D&md5=7c78e2c9e93ac10a21eaade4def8f94e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199910%2920%3A13%3C1354%3A%3AAID-JCC3%3E3.0.CO%3B2-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199910%252920%253A13%253C1354%253A%253AAID-JCC3%253E3.0.CO%253B2-N%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DLambris%26aufirst%3DJ.%26atitle%3DPredicting%2520peptide%2520structures%2520using%2520NMR%2520data%2520and%2520deterministic%2520global%2520optimization%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1999%26volume%3D20%26spage%3D1354%26epage%3D1370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Ab initio prediction of helical segments of polypeptides</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">246</span><span class="NLM_x">â</span> <span class="NLM_lpage">266</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=246-266&author=J.+L.+Klepeisauthor=C.+A.+Floudas&title=Ab+initio+prediction+of+helical+segments+of+polypeptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DAb%2520initio%2520prediction%2520of%2520helical%2520segments%2520of%2520polypeptides%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D246%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">Prediction of beta-sheet topology and disulfide bridges in polypeptides</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">â</span> <span class="NLM_lpage">208</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2003&pages=191-208&author=J.+L.+Klepeisauthor=C.+A.+Floudas&title=Prediction+of+beta-sheet+topology+and+disulfide+bridges+in+polypeptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DPrediction%2520of%2520beta-sheet%2520topology%2520and%2520disulfide%2520bridges%2520in%2520polypeptides%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2003%26volume%3D24%26spage%3D191%26epage%3D208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span> </span><span class="NLM_article-title">ASTRO-FOLD: A combinatorial and global optimization framework for ab initio prediction of three-dimensional structures of proteins from the amino acid sequence</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">2119</span><span class="NLM_x">â</span> <span class="NLM_lpage">2146</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2003&pages=2119-2146&author=J.+Klepeisauthor=C.+Floudas&title=ASTRO-FOLD%3A+A+combinatorial+and+global+optimization+framework+for+ab+initio+prediction+of+three-dimensional+structures+of+proteins+from+the+amino+acid+sequence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DFloudas%26aufirst%3DC.%26atitle%3DASTRO-FOLD%253A%2520A%2520combinatorial%2520and%2520global%2520optimization%2520framework%2520for%2520ab%2520initio%2520prediction%2520of%2520three-dimensional%2520structures%2520of%2520proteins%2520from%2520the%2520amino%2520acid%2520sequence%26jtitle%3DBiophys.%2520J.%26date%3D2003%26volume%3D85%26spage%3D2119%26epage%3D2146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieja, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span> </span><span class="NLM_article-title">A new class of hybrid global optimization algorithms for peptide structure prediction: Integrated hybrids</span> <span class="citation_source-journal">Comput. Phys. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003a</span><span class="NLM_x">, </span> <span class="NLM_volume">151</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">â</span> <span class="NLM_lpage">140</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1016%2FS0010-4655%2802%2900735-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1ygtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2003a&pages=121-140&author=J.+Klepeisauthor=M.+Piejaauthor=C.+Floudas&title=A+new+class+of+hybrid+global+optimization+algorithms+for+peptide+structure+prediction%3A+Integrated+hybrids"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A new class of hybrid global optimization algorithms for peptide structure prediction: integrated hybrids</span></div><div class="casAuthors">Klepeis, J. L.; Pieja, M. J.; Floudas, C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-140</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A novel class of hybrid global optimization methods for application to the structure prediction in protein-folding problem is introduced.  These optimization methods take the form of a hybrid between a deterministic global optimization algorithm, the Î±BB, and a stochastically based method, conformational space annealing (CSA), and attempt to combine the beneficial features of these two algorithms.  The Î±BB method as previously extant exhibits consistency, as it guarantees convergence to the global min. for twice-continuously differentiable constrained nonlinear programming problems, but can benefit from improvements in the computational front.  Computational studies for met-enkephalin demonstrate the promise for the proposed hybrid global optimization method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCRa_oT6V-47Vg90H21EOLACvtfcHk0lg_m1hn0VDZUQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1ygtg%253D%253D&md5=adc573297e231dde19be34d83835e796</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0010-4655%2802%2900735-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0010-4655%252802%252900735-X%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DPieja%26aufirst%3DM.%26aulast%3DFloudas%26aufirst%3DC.%26atitle%3DA%2520new%2520class%2520of%2520hybrid%2520global%2520optimization%2520algorithms%2520for%2520peptide%2520structure%2520prediction%253A%2520Integrated%2520hybrids%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2003a%26volume%3D151%26spage%3D121%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Klepeis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieja, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C.</span><span> </span><span class="NLM_article-title">Hybrid global optimization algorithms for protein structure prediction: alternating hybrids</span> <span class="citation_source-journal">Biophys. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003b</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">â</span> <span class="NLM_lpage">882</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2003b&pages=869-882&author=J.+Klepeisauthor=M.+Piejaauthor=C.+Floudas&title=Hybrid+global+optimization+algorithms+for+protein+structure+prediction%3A+alternating+hybrids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKlepeis%26aufirst%3DJ.%26aulast%3DPieja%26aufirst%3DM.%26aulast%3DFloudas%26aufirst%3DC.%26atitle%3DHybrid%2520global%2520optimization%2520algorithms%2520for%2520protein%2520structure%2520prediction%253A%2520alternating%2520hybrids%26jtitle%3DBiophys.%2520J.%26date%3D2003b%26volume%3D84%26spage%3D869%26epage%3D882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Huber, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholze, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paques, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deisenhofer, J.</span><span> </span><span class="NLM_article-title">Crystal structure analysis and molecular model of human C3a anaphylatoxin</span> <span class="citation_source-journal">Hoppe Seylers Z. Physiol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">361</span><span class="NLM_x">, </span> <span class="NLM_fpage">1389</span><span class="NLM_x">â</span> <span class="NLM_lpage">1399</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=1980&pages=1389-1399&author=R.+Huberauthor=H.+Scholzeauthor=E.+Paquesauthor=J.+Deisenhofer&title=Crystal+structure+analysis+and+molecular+model+of+human+C3a+anaphylatoxin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DScholze%26aufirst%3DH.%26aulast%3DPaques%26aufirst%3DE.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520analysis%2520and%2520molecular%2520model%2520of%2520human%2520C3a%2520anaphylatoxin%26jtitle%3DHoppe%2520Seylers%2520Z.%2520Physiol.%2520Chem.%26date%3D1980%26volume%3D361%26spage%3D1389%26epage%3D1399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Janssen, B. J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huizinga, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raaijmakers, H. C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daha, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson-Ekdahl, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gros, P.</span><span> </span><span class="NLM_article-title">Structures of complement component C3 provide insights into the function and evolution of immunity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">â</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1038%2Fnature04005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=16177781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVajs7vI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=505-511&author=B.+J.+C.+Janssenauthor=E.+G.+Huizingaauthor=H.+C.+A.+Raaijmakersauthor=A.+Roosauthor=M.+R.+Dahaauthor=K.+Nilsson-Ekdahlauthor=B.+Nilssonauthor=P.+Gros&title=Structures+of+complement+component+C3+provide+insights+into+the+function+and+evolution+of+immunity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of complement component C3 provide insights into the function and evolution of immunity</span></div><div class="casAuthors">Janssen, Bert J. C.; Huizinga, Eric G.; Raaijmakers, Hans C. A.; Roos, Anja; Daha, Mohamed R.; Nilsson-Ekdahl, Kristina; Nilsson, Bo; Gros, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7058</span>),
    <span class="NLM_cas:pages">505-511</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The mammalian complement system is a phylogenetically ancient cascade system that has a major role in innate and adaptive immunity.  Activation of component C3 (1641 residues) is central to the three complement pathways and results in inflammation and elimination of self and non-self targets.  Here the authors present crystal structures of native C3 and its final major proteolytic fragment C3c.  The structures reveal thirteen domains, nine of which were unpredicted, and suggest that the proteins of the Î±2-macroglobulin family evolved from a core of eight homologous domains.  A double mechanism prevents hydrolysis of the thioester group, essential for covalent attachment of activated C3 to target surfaces.  Marked conformational changes in the Î±-chain, including movement of a crit. interaction site through a ring formed by the domains of the Î²-chain, indicate an unprecedented, conformation-dependent mechanism of activation, regulation and biol. function of C3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJPI74p1TderVg90H21EOLACvtfcHk0lg0SXNyNWu-AQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVajs7vI&md5=7b229e0ea9d3771f000855c687b1a7ab</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature04005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04005%26sid%3Dliteratum%253Aachs%26aulast%3DJanssen%26aufirst%3DB.%2BJ.%2BC.%26aulast%3DHuizinga%26aufirst%3DE.%2BG.%26aulast%3DRaaijmakers%26aufirst%3DH.%2BC.%2BA.%26aulast%3DRoos%26aufirst%3DA.%26aulast%3DDaha%26aufirst%3DM.%2BR.%26aulast%3DNilsson-Ekdahl%26aufirst%3DK.%26aulast%3DNilsson%26aufirst%3DB.%26aulast%3DGros%26aufirst%3DP.%26atitle%3DStructures%2520of%2520complement%2520component%2520C3%2520provide%2520insights%2520into%2520the%2520function%2520and%2520evolution%2520of%2520immunity%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D505%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Brooks, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruccoleri, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olafson, B.</span><span> </span><span class="NLM_article-title">CHARMMâa program for macromolecular energy, minimization, and dynamics calculations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">â</span> <span class="NLM_lpage">217</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1983&pages=187-217&author=B.+Brooksauthor=R.+Bruccoleriauthor=B.+Olafson&title=CHARMM%E2%80%94a+program+for+macromolecular+energy%2C+minimization%2C+and+dynamics+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%26aulast%3DBruccoleri%26aufirst%3DR.%26aulast%3DOlafson%26aufirst%3DB.%26atitle%3DCHARMM%25E2%2580%2594a%2520program%2520for%2520macromolecular%2520energy%252C%2520minimization%252C%2520and%2520dynamics%2520calculations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1983%26volume%3D4%26spage%3D187%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Cain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monk, P.</span><span> </span><span class="NLM_article-title">The orphan receptor C5L2 has high affinity binding sites for complement fragments C5a and C5a of Arg74</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">7165</span><span class="NLM_x">â</span> <span class="NLM_lpage">7169</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=7165-7169&author=S.+Cainauthor=P.+Monk&title=The+orphan+receptor+C5L2+has+high+affinity+binding+sites+for+complement+fragments+C5a+and+C5a+of+Arg74"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCain%26aufirst%3DS.%26aulast%3DMonk%26aufirst%3DP.%26atitle%3DThe%2520orphan%2520receptor%2520C5L2%2520has%2520high%2520affinity%2520binding%2520sites%2520for%2520complement%2520fragments%2520C5a%2520and%2520C5a%2520of%2520Arg74%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D7165%26epage%3D7169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Monk, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, M.</span><span> </span><span class="NLM_article-title">C5a stimulus-secretion coupling in rat basophilic leukemia (RBL-2H3) cells transfected with the human C5a receptor is mediated by pertussis and cholera toxin-sensitive G-proteins</span> <span class="citation_source-journal">Biochem. Mol. Biol. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">13</span><span class="NLM_x">â</span> <span class="NLM_lpage">20</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1994&pages=13-20&issue=1&author=P.+Monkauthor=J.+Peaseauthor=M.+Barker&title=C5a+stimulus-secretion+coupling+in+rat+basophilic+leukemia+%28RBL-2H3%29+cells+transfected+with+the+human+C5a+receptor+is+mediated+by+pertussis+and+cholera+toxin-sensitive+G-proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMonk%26aufirst%3DP.%26aulast%3DPease%26aufirst%3DJ.%26aulast%3DBarker%26aufirst%3DM.%26atitle%3DC5a%2520stimulus-secretion%2520coupling%2520in%2520rat%2520basophilic%2520leukemia%2520%2528RBL-2H3%2529%2520cells%2520transfected%2520with%2520the%2520human%2520C5a%2520receptor%2520is%2520mediated%2520by%2520pertussis%2520and%2520cholera%2520toxin-sensitive%2520G-proteins%26jtitle%3DBiochem.%2520Mol.%2520Biol.%2520Int.%26date%3D1994%26volume%3D32%26issue%3D1%26spage%3D13%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Eswar, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti-Renom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madhusudhan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eramian, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pieper, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Comparative protein structure modeling with MODELLER</span> <span class="citation_source-journal">Curr. Protoc. Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2.9.9</span><span class="NLM_x">â</span> <span class="NLM_lpage">2.9.30</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2.9.9-2.9.30&author=N.+Eswarauthor=M.+Marti-Renomauthor=B.+Webbauthor=M.+Madhusudhanauthor=D.+Eramianauthor=M.+Shenauthor=U.+Pieperauthor=A.+Sali&title=Comparative+protein+structure+modeling+with+MODELLER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26aulast%3DMarti-Renom%26aufirst%3DM.%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DMadhusudhan%26aufirst%3DM.%26aulast%3DEramian%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520with%2520MODELLER%26jtitle%3DCurr.%2520Protoc.%2520Protein%2520Sci.%26date%3D2007%26volume%3D50%26spage%3D2.9.9%26epage%3D2.9.30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">R Development Core Team</span><span class="NLM_x">. </span><span> </span><span class="NLM_article-title">R: A language and environment for statistical computing</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=R+Development+Core+Team&title=R%3A+A+language+and+environment+for+statistical+computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DR%253A%2520A%2520language%2520and%2520environment%2520for%2520statistical%2520computing%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Grant, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ElSawy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caves, L.</span><span> </span><span class="NLM_article-title">Bio3d: an R package for the comparative analysis of protein structures</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2695</span><span class="NLM_x">â</span> <span class="NLM_lpage">2696</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=2695-2696&author=B.+Grantauthor=A.+Rodriguesauthor=K.+ElSawyauthor=J.+McCammonauthor=L.+Caves&title=Bio3d%3A+an+R+package+for+the+comparative+analysis+of+protein+structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DB.%26aulast%3DRodrigues%26aufirst%3DA.%26aulast%3DElSawy%26aufirst%3DK.%26aulast%3DMcCammon%26aufirst%3DJ.%26aulast%3DCaves%26aufirst%3DL.%26atitle%3DBio3d%253A%2520an%2520R%2520package%2520for%2520the%2520comparative%2520analysis%2520of%2520protein%2520structures%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D22%26spage%3D2695%26epage%3D2696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Dolinsky, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czodrowski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, N.</span><span> </span><span class="NLM_article-title">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">W522</span><span class="NLM_x">â</span> <span class="NLM_lpage">W525</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1093%2Fnar%2Fgkm276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=17488841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W522-W525&author=T.+Dolinskyauthor=P.+Czodrowskiauthor=H.+Liauthor=J.+Nielsenauthor=J.+Jensenauthor=G.+Klebeauthor=N.+Baker&title=PDB2PQR%3A+expanding+and+upgrading+automated+preparation+of+biomolecular+structures+for+molecular+simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span></div><div class="casAuthors">Dolinsky Todd J; Czodrowski Paul; Li Hui; Nielsen Jens E; Jensen Jan H; Klebe Gerhard; Baker Nathan A</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W522-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Real-world observable physical and chemical characteristics are increasingly being calculated from the 3D structures of biomolecules.  Methods for calculating pK(a) values, binding constants of ligands, and changes in protein stability are readily available, but often the limiting step in computational biology is the conversion of PDB structures into formats ready for use with biomolecular simulation software.  The continued sophistication and integration of biomolecular simulation methods for systems- and genome-wide studies requires a fast, robust, physically realistic and standardized protocol for preparing macromolecular structures for biophysical algorithms.  As described previously, the PDB2PQR web server addresses this need for electrostatic field calculations (Dolinsky et al., Nucleic Acids Research, 32, W665-W667, 2004).  Here we report the significantly expanded PDB2PQR that includes the following features: robust standalone command line support, improved pK(a) estimation via the PROPKA framework, ligand parameterization via PEOE_PB charge methodology, expanded set of force fields and easily incorporated user-defined parameters via XML input files, and improvement of atom addition and optimization code.  These features are available through a new web interface (http://pdb2pqr.sourceforge.net/), which offers users a wide range of options for PDB file conversion, modification and parameterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqIxhnOQkIHOxfW6udTcc2ebOzR-TjyUo6rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGltQ%253D%253D&md5=18883d2f9a6f30dba143a4a2ca80b0d8</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm276%26sid%3Dliteratum%253Aachs%26aulast%3DDolinsky%26aufirst%3DT.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DNielsen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DBaker%26aufirst%3DN.%26atitle%3DPDB2PQR%253A%2520expanding%2520and%2520upgrading%2520automated%2520preparation%2520of%2520biomolecular%2520structures%2520for%2520molecular%2520simulations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW522%26epage%3DW525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Sitkoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">Accurate calculation of hydration free energies using macroscopic solvent models</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">1978</span><span class="NLM_x">â</span> <span class="NLM_lpage">88</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100058a043" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1994&pages=1978-88&author=D.+Sitkoffauthor=K.+Sharpauthor=B.+Honig&title=Accurate+calculation+of+hydration+free+energies+using+macroscopic+solvent+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Calculation of Hydration Free Energies Using Macroscopic Solvent Models</span></div><div class="casAuthors">Sitkoff, Doree; Sharp, Kim A.; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1978-88</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">A method and parametrization scheme which allow fast and accurate calcns. of hydration free energies are described.  The solute is treated as a polarizable cavity of a shape defined by the mol. surface, contg. point charges at the location of at. nuclei.  Electrostatic contributions to solvation are derived from finite difference solns. of the Poisson equation (FDPB method).  Nonpolar (cavity/van der Waals) energies are added as a surface area dependent term, with a single surface tension coeff. (Î³) derived from hydrocarbon soly. in water.  At. charges and radii are obtained by modifying existing force-field- or quantum-mech.-derived values, by fitting to exptl. solvation energies of small org. mols.  A new, simple parameter set (parameters for solvation energy, PARSE) is developed specifically for the FDPB/Î³ method, by choosing at. charges and radii which reproduce the estd. contributions to solvation of simple functional groups.  The PARSE parameters reproduce hydration free energies for a test set of 67 mols. with an av. error of 0.4 kcal/mol.  For amino acid side chain and peptide backbone analogs the av. error is only 0.1 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCKnU2H8vArVg90H21EOLACvtfcHk0liuleR5uAGHWw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsVGrsrs%253D&md5=78ad4721dced796d92eb27c15f98aba5</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fj100058a043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100058a043%26sid%3Dliteratum%253Aachs%26aulast%3DSitkoff%26aufirst%3DD.%26aulast%3DSharp%26aufirst%3DK.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DAccurate%2520calculation%2520of%2520hydration%2520free%2520energies%2520using%2520macroscopic%2520solvent%2520models%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1994%26volume%3D98%26spage%3D1978%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Baker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sept, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holst, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCammon, J.</span><span> </span><span class="NLM_article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10037</span><span class="NLM_x">â</span> <span class="NLM_lpage">10041</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1073%2Fpnas.181342398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=11517324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10037-10041&author=N.+Bakerauthor=D.+Septauthor=S.+Josephauthor=M.+Holstauthor=J.+McCammon&title=Electrostatics+of+nanosystems%3A+application+to+microtubules+and+the+ribosome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatics of nanosystems: application to microtubules and the ribosome</span></div><div class="casAuthors">Baker, Nathan A.; Sept, David; Joseph, Simpson; Holst, Michael J.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10037-10041</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Evaluation of the electrostatic properties of biomols. has become a std. practice in mol. biophysics.  Foremost among the models used to elucidate the electrostatic potential is the Poisson-Boltzmann equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calcns. to relatively small biomol. systems.  Here we present the application of numerical methods to enable the trivially parallel soln. of the Poisson-Boltzmann equation for supramol. structures that are orders of magnitude larger in size.  As a demonstration of this methodol., electrostatic potentials have been calcd. for large microtubule and ribosome structures.  The results point to the likely role of electrostatics in a variety of activities of these structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_CAlhE0uX7Vg90H21EOLACvtfcHk0liuleR5uAGHWw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWisbc%253D&md5=1b861999ef12c6972e82e8ada0f387cb</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.181342398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.181342398%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%26aulast%3DSept%26aufirst%3DD.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DHolst%26aufirst%3DM.%26aulast%3DMcCammon%26aufirst%3DJ.%26atitle%3DElectrostatics%2520of%2520nanosystems%253A%2520application%2520to%2520microtubules%2520and%2520the%2520ribosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2001%26volume%3D98%26spage%3D10037%26epage%3D10041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Gorham, R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sausman, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foronda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">An evaluation of PoissonâBoltzmann electrostatic free energy calculations through comparison with experimental mutagenesis</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">746</span><span class="NLM_x">â</span> <span class="NLM_lpage">754</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2011&pages=746-754&author=R.+Gorhamauthor=C.+Kieslichauthor=A.+Nicholsauthor=N.+Sausmanauthor=M.+Forondaauthor=D.+Morikis&title=An+evaluation+of+Poisson%E2%80%93Boltzmann+electrostatic+free+energy+calculations+through+comparison+with+experimental+mutagenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGorham%26aufirst%3DR.%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DNichols%26aufirst%3DA.%26aulast%3DSausman%26aufirst%3DN.%26aulast%3DForonda%26aufirst%3DM.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DAn%2520evaluation%2520of%2520Poisson%25E2%2580%2593Boltzmann%2520electrostatic%2520free%2520energy%2520calculations%2520through%2520comparison%2520with%2520experimental%2520mutagenesis%26jtitle%3DBiopolymers%26date%3D2011%26volume%3D94%26spage%3D746%26epage%3D754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunopulos, D.</span><span> </span><span class="NLM_article-title">Automated computational framework for the analysis of electrostatic similarities of proteins</span> <span class="citation_source-journal">Biotechnol. Prog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="NLM_x">â</span> <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1002%2Fbtpr.541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=21485028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A528%3ADC%252BC3MXls1Gis7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=316-325&author=C.+Kieslichauthor=D.+Morikisauthor=J.+Yangauthor=D.+Gunopulos&title=Automated+computational+framework+for+the+analysis+of+electrostatic+similarities+of+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Automated computational framework for the analysis of electrostatic similarities of proteins</span></div><div class="casAuthors">Kieslich, Chris A.; Morikis, Dimitrios; Yang, Jianfeng; Gunopulos, Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Progress</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">316-325</span>CODEN:
                <span class="NLM_cas:coden">BIPRET</span>;
        ISSN:<span class="NLM_cas:issn">1520-6033</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Charge plays an important role in protein-protein interactions.  In the case of excessively charged proteins, their electrostatic potentials contribute to the processes of recognition and binding with other proteins or ligands.  We present an automated computational framework for detg. the contribution of each charged amino acid to the electrostatic properties of proteins, at at. resoln. level.  This framework involves computational alanine scans, calcn. of Poisson-Boltzmann electrostatic potentials, calcn. of electrostatic similarity distances (ESDs), hierarchical clustering anal. of ESDs, calcn. of solvation free energies of assocn., and visualization of the spatial distributions of electrostatic potentials.  The framework is useful to classify families of mutants with similar electrostatic properties and to compare them with the parent proteins in the complex.  The alanine scan mutants introduce perturbations in the local electrostatic properties of the proteins and aim in delineating the contribution of each mutated amino acid in the spatial distribution of electrostatic potential, and in biol. function when electrostatics is a dominant contributing factor in protein-protein interactions.  The framework can be used to design new proteins with tailored electrostatic properties, such as immune system regulators, inhibitors, and vaccines, and in guiding exptl. studies.  We present an example for the interaction of the immune system protein C3d (the d-fragment of complement protein C3) with its receptor CR2, and we discuss our data in view of a binding site controversy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ316FjcDZubVg90H21EOLACvtfcHk0liuleR5uAGHWw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXls1Gis7o%253D&md5=92ce43efe6e241446e475132fd6eea4d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fbtpr.541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbtpr.541%26sid%3Dliteratum%253Aachs%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DMorikis%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DGunopulos%26aufirst%3DD.%26atitle%3DAutomated%2520computational%2520framework%2520for%2520the%2520analysis%2520of%2520electrostatic%2520similarities%2520of%2520proteins%26jtitle%3DBiotechnol.%2520Prog.%26date%3D2011%26volume%3D27%26spage%3D316%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Gorham, R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">Electrostatic clustering and free energy calculations provide a foundation for protein design and optimization</span> <span class="citation_source-journal">Ann. Biomed. Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1252</span><span class="NLM_x">â</span> <span class="NLM_lpage">1263</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm201609k&amp;key=10.1007%2Fs10439-010-0226-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm201609k&amp;key=21140293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm201609k&amp;key=1%3ACAS%3A280%3ADC%252BC3Mzot1WmtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1252-1263&author=R.+Gorhamauthor=C.+Kieslichauthor=D.+Morikis&title=Electrostatic+clustering+and+free+energy+calculations+provide+a+foundation+for+protein+design+and+optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic clustering and free energy calculations provide a foundation for protein design and optimization</span></div><div class="casAuthors">Gorham Ronald D Jr; Kieslich Chris A; Morikis Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Annals of biomedical engineering</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1252-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Electrostatic interactions are ubiquitous in proteins and dictate stability and function.  In this review, we discuss several methods for the analysis of electrostatics in protein-protein interactions.  We discuss alanine-scanning mutagenesis, Poisson-Boltzmann electrostatics, free energy calculations, electrostatic similarity distances, and hierarchical clustering of electrostatic potentials.  Our recently developed computational framework, known as Analysis of Electrostatic Similarities Of Proteins (AESOP), incorporates these tools to efficiently elucidate the role of electrostatic potentials in protein interactions.  We present the application of AESOP to several proteins and protein complexes, for which charge is purported to facilitate protein association.  Specifically, we illustrate how recent work has shaped the formulation of electrostatic calculations, the correlation of electrostatic free energies and electrostatic potential clustering results with experimental binding and activity data, the pH dependence of protein stability and association, the design of mutant proteins with enhanced immunological activity, and how AESOP can expose deficiencies in structural models and experimental data.  This integrative approach can be utilized to develop mechanistic models and to guide experimental studies by predicting mutations with desired physicochemical properties and function.  Alteration of the electrostatic properties of proteins offers a basis for the design of proteins with optimized binding and activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTVo5QVfbcq4IJKJbLTgsvCfW6udTcc2eag8ZH72Wjwkbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mzot1WmtQ%253D%253D&md5=3794585d99f75112a3c6f2ebb25d9334</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10439-010-0226-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10439-010-0226-9%26sid%3Dliteratum%253Aachs%26aulast%3DGorham%26aufirst%3DR.%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DElectrostatic%2520clustering%2520and%2520free%2520energy%2520calculations%2520provide%2520a%2520foundation%2520for%2520protein%2520design%2520and%2520optimization%26jtitle%3DAnn.%2520Biomed.%2520Eng.%26date%3D2011%26volume%3D39%26spage%3D1252%26epage%3D1263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Kieslich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorham, R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikis, D.</span><span> </span><span class="NLM_article-title">Is the rigid-body assumption reasonable? Insights into the effects of dynamics on the electrostatic analysis of barnase-barstar</span> <span class="citation_source-journal">J. Non-Cryst. Solids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">â</span> <span class="NLM_lpage">716</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2011&pages=707-716&author=C.+Kieslichauthor=R.+Gorhamauthor=D.+Morikis&title=Is+the+rigid-body+assumption+reasonable%3F+Insights+into+the+effects+of+dynamics+on+the+electrostatic+analysis+of+barnase-barstar"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKieslich%26aufirst%3DC.%26aulast%3DGorham%26aufirst%3DR.%26aulast%3DMorikis%26aufirst%3DD.%26atitle%3DIs%2520the%2520rigid-body%2520assumption%2520reasonable%253F%2520Insights%2520into%2520the%2520effects%2520of%2520dynamics%2520on%2520the%2520electrostatic%2520analysis%2520of%2520barnase-barstar%26jtitle%3DJ.%2520Non-Cryst.%2520Solids%26date%3D2011%26volume%3D357%26spage%3D707%26epage%3D716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Pettersen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T.</span><span> </span><span class="NLM_article-title">UCSF chimeraâA visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">â</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+Pettersenauthor=T.+Goddardauthor=C.+Huangauthor=G.+Couchauthor=D.+Greenblattauthor=E.+Mengauthor=T.+Ferrin&title=UCSF+chimera%E2%80%94A+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%26aulast%3DGoddard%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DCouch%26aufirst%3DG.%26aulast%3DGreenblatt%26aufirst%3DD.%26aulast%3DMeng%26aufirst%3DE.%26aulast%3DFerrin%26aufirst%3DT.%26atitle%3DUCSF%2520chimera%25E2%2580%2594A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A73" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A73','PDB','2A73'); return false;">PDB: 2A73</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i20"><a href="/doi/suppl/10.1021/jm201609k">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_06102"></div></div></div></div></div><hr /></hr><p class="last">Details about the mathematical model and formulations for the de novo design framework and sequence results from all five computational design runs.  This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm201609k/suppl_file/jm201609k_si_001.pdf">jm201609k_si_001.pdf (204.98 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm201609k&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-9%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Fjm201609k%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm201609k" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                1MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d6536b083d66","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
